<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1263" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1263.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1263.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1263.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1263.atom" />
<meta name="HW.pisa" content="ggg;5/6/1263" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" />
<meta name="DC.Identifier" content="10.1534/g3.115.017616" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Giri et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Introgression is the transfer of genes or genomic regions from one species into another via hybridization and back-crosses. We have introgressed four translocations ( EB4 , IBj5 , UK14-1 , and B362i ) from Neurospora crassa into N. tetrasperma . This enabled us to construct two general types of heterokaryons with mat-A and mat-a nuclei of different genotypes: one type is \[ T + N \] (with one translocation nucleus and one normal sequence nucleus), and the other is \[ Dp + Df \] (with one nucleus carrying a duplication of the translocation region and the other being deleted for the translocation region). Self-crossing these heterokaryons again produced [ T + N ] and [ Dp + Df ] progeny. From conidia (vegetative spores) produced by the heterokaryotic mycelia, we obtained self-fertile (heterokaryotic) and self-sterile (homokaryotic) derivative strains. [ T + N ] heterokaryons produced homokaryotic conidial derivatives of both mating types, but [ Dp + Df ] heterokaryons produced viable conidial homokaryons of only the mating type of the Dp nucleus. All four [ T + N ] heterokaryons and three [ Dp + Df ] heterokaryons produced both self-sterile and self-fertile conidial derivatives, but the [ Dp(B362i) + Df(B362i) ] heterokaryons produced only self-sterile ones. Conceivably, the Df(B362i) nuclei may be deleted for a nucleus-limited gene required for efficient mitosis or nuclear division, and whose deficit is not complemented by the neighboring Dp(B362i) nuclei. A cross involving Dp(EB4) showed repeat-induced point mutation (RIP). Because RIP can occur in self-crosses of [ Dp + Df ] but not [ T + N ] heterokaryons, RIP alteration of a translocated segment would depend on the relative numbers of [ Dp + Df ] vs. [ T + N ] ancestors." />
<meta name="DC.Contributor" content="Dev Ashish Giri" />
<meta name="DC.Contributor" content="S. Rekha" />
<meta name="DC.Contributor" content="Durgadas P. Kasbekar" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Introgression is the transfer of genes or genomic regions from one species into another via hybridization and back-crosses. We have introgressed four translocations (&lt;i&gt;EB4&lt;/i&gt;, &lt;i&gt;IBj5&lt;/i&gt;, &lt;i&gt;UK14-1&lt;/i&gt;, and &lt;i&gt;B362i&lt;/i&gt;) from &lt;i&gt;Neurospora crassa&lt;/i&gt; into &lt;i&gt;N. tetrasperma&lt;/i&gt;. This enabled us to construct two general types of heterokaryons with &lt;i&gt;mat-A&lt;/i&gt; and &lt;i&gt;mat-a&lt;/i&gt; nuclei of different genotypes: one type is [&lt;i&gt;T&lt;/i&gt; + &lt;i&gt;N&lt;/i&gt;] (with one translocation nucleus and one normal sequence nucleus), and the other is [&lt;i&gt;Dp&lt;/i&gt; + &lt;i&gt;Df&lt;/i&gt;] (with one nucleus carrying a duplication of the translocation region and the other being deleted for the translocation region). Self-crossing these heterokaryons again produced [&lt;i&gt;T&lt;/i&gt; + &lt;i&gt;N&lt;/i&gt;] and [&lt;i&gt;Dp&lt;/i&gt; + &lt;i&gt;Df&lt;/i&gt;] progeny. From conidia (vegetative spores) produced by the heterokaryotic mycelia, we obtained self-fertile (heterokaryotic) and self-sterile (homokaryotic) derivative strains. [&lt;i&gt;T&lt;/i&gt; + &lt;i&gt;N&lt;/i&gt;] heterokaryons produced homokaryotic conidial derivatives of both mating types, but [&lt;i&gt;Dp&lt;/i&gt; + &lt;i&gt;Df&lt;/i&gt;] heterokaryons produced viable conidial homokaryons of only the mating type of the &lt;i&gt;Dp&lt;/i&gt; nucleus. All four [&lt;i&gt;T&lt;/i&gt; + &lt;i&gt;N&lt;/i&gt;] heterokaryons and three [&lt;i&gt;Dp&lt;/i&gt; + &lt;i&gt;Df&lt;/i&gt;] heterokaryons produced both self-sterile and self-fertile conidial derivatives, but the [&lt;i&gt;Dp(B362i)&lt;/i&gt; + &lt;i&gt;Df(B362i)&lt;/i&gt;] heterokaryons produced only self-sterile ones. Conceivably, the &lt;i&gt;Df(B362i)&lt;/i&gt; nuclei may be deleted for a nucleus-limited gene required for efficient mitosis or nuclear division, and whose deficit is not complemented by the neighboring &lt;i&gt;Dp(B362i)&lt;/i&gt; nuclei. A cross involving &lt;i&gt;Dp(EB4)&lt;/i&gt; showed repeat-induced point mutation (RIP). Because RIP can occur in self-crosses of [&lt;i&gt;Dp&lt;/i&gt; + &lt;i&gt;Df&lt;/i&gt;] but not [&lt;i&gt;T&lt;/i&gt; + &lt;i&gt;N&lt;/i&gt;] heterokaryons, RIP alteration of a translocated segment would depend on the relative numbers of [&lt;i&gt;Dp&lt;/i&gt; + &lt;i&gt;Df&lt;/i&gt;] &lt;i&gt;vs.&lt;/i&gt; [&lt;i&gt;T&lt;/i&gt; + &lt;i&gt;N&lt;/i&gt;] ancestors.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1263" />
<meta name="citation_id" content="5/6/1263" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1263" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1263.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1263.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1263.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017616" />
<meta name="citation_pmid" content="25897010" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1263" />
<meta name="citation_lastpage" content="1272" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dev Ashish Giri" />
<meta name="citation_author_institution" content="Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India" />
<meta name="citation_author_institution" content="Graduate Studies, Manipal University, Manipal 576104, Karnataka, India" />
<meta name="citation_author" content="S. Rekha" />
<meta name="citation_author_institution" content="Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India" />
<meta name="citation_author" content="Durgadas P. Kasbekar" />
<meta name="citation_author_institution" content="Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India" />
<meta name="citation_author_email" content="kas@cdfd.org.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5269-8020" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=E. G. Burton;citation_author=R. L. Metzenberg;citation_title=Novel Mutation Causing Derepression of Several Enzymes of Sulfur Metabolism in Neurospora crassa;citation_pages=140-151;citation_volume=109;citation_year=1972;citation_issue=1;citation_pmid=4257980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Calhoun;citation_author=H. B. Howe;citation_title=GENETIC ANALYSIS OF EIGHT-SPORED ASCI PRODUCED BY GENE E IN NEUROSPORA TETRASPERMA;citation_pages=449-459;citation_volume=60;citation_year=1968;citation_issue=3;citation_pmid=5728739" />
<meta name="citation_reference" content="citation_journal_abbrev=Fungal Biol;citation_author=P. Corcoran;citation_author=D. J. Jacobson;citation_author=M. I. Bidartondo;citation_author=P. C. Hickey;citation_author=J. F. Kerekes;citation_title=Quantifying functional heterothallism in the pseudohomothallic ascomycete Neurospora tetrasperma.;citation_pages=962-975;citation_volume=116;citation_year=2012;citation_issue=9;citation_pmid=22954339;citation_doi=10.1016/j.funbio.2012.06.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Czaja;citation_author=K. Y. Miller;citation_author=B. L. Miller;citation_title=Novel sexual-cycle specific gene silencing in Aspergillus nidulans.;citation_volume=193;citation_year=2013;citation_pmid=23341415;citation_doi=10.1534/genetics.112.147546" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=R. H. Davis;citation_title=Adaptation in pantothenate-requiring Neurospora. II. Nuclear competition during adaptation.;citation_pages=648-654;citation_volume=47;citation_year=1960;citation_doi=10.2307/2439515" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=R. H. Davis;citation_author=F. J. De Serres;citation_title=Genetic and microbiological research techniques for Neurospora crassa.;citation_pages=79-143;citation_volume=17;citation_year=1970;citation_doi=10.1016/0076-6879(71)17168-6" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. Demeter;citation_author=S. E. Lee;citation_author=J. E. Haber;citation_author=T. Stearns;citation_title=The DNA damage checkpoint signal in budding yeast is nuclear limited.;citation_pages=487-492;citation_volume=6;citation_year=2000;citation_issue=2;citation_pmid=10983994;citation_doi=10.1016/S1097-2765(00)00047-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. E. Ellison;citation_author=J. E. Stajich;citation_author=D. J. Jacobson;citation_author=D. O. Nativig;citation_author=A. Lapidus;citation_title=Massive changes in genome architecture accompany the transition to self-fertility in the filamentous fungus Neurospora tetrasperma.;citation_volume=189;citation_year=2011;citation_pmid=21750257;citation_doi=10.1534/genetics.111.130690" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Gallegos;citation_author=D. J. Jacobson;citation_author=N. B. Raju;citation_author=M. P. Skupski;citation_author=D. O. Natvig;citation_title=Suppressed Recombination and a Pairing Anomaly on the Mating-Type Chromosome of Neurospora tetrasperma;citation_pages=623-633;citation_volume=154;citation_year=2000;citation_issue=2;citation_pmid=10655216" />
<meta name="citation_reference" content="citation_journal_title=Journal of genetics;citation_journal_abbrev=J Genet;citation_author=S. V. Iyer;citation_author=M. Ramakrishnan;citation_author=D. P. Kasbekar;citation_title=Neurospora crassa fmf-1 encodes the homologue of the Schizosaccharomyces pombe Ste11p regulator of sexual development.;citation_pages=33-39;citation_volume=88;citation_year=2009;citation_issue=1;citation_pmid=19417542;citation_doi=10.1007/s12041-009-0005-2" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=J. L. Jinks;citation_title=Heterokaryosis: A System of Adaptation in Wild Fungi;citation_pages=83-99;citation_volume=140;citation_year=1952;citation_issue=898;citation_pmid=13003914;citation_doi=10.1098/rspb.1952.0046" />
<meta name="citation_reference" content="citation_title=Neurospora: Genomics and Molecular Biology;citation_title=Neurospora duplications, and genome defense by RIP and meiotic silencing;citation_author=D. P. Kasbekar;citation_pages=109-127;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Biosci.;citation_author=D. P. Kasbekar;citation_title=Are any fungal genes nucleus-limited?;citation_pages=341-346;citation_volume=39;citation_year=2014;citation_pmid=24845498;citation_doi=10.1007/s12038-014-9419-y" />
<meta name="citation_reference" content="citation_title=Genetic Transformation Systems in Fungi;citation_author=D. P. Kasbekar;citation_title=What have we learned by doing transformations in Neurospora tetrasperma?;citation_pages=47-52;citation_volume=2;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=D. P. Kasbekar;citation_author=P. K. Singh;citation_author=M. Ramakrishnan;citation_author=K. B. Raj;citation_title=Carrefour Mme. Gras: a wild-isolated Neurospora crassa strain that suppresses meiotic silencing by unpaired DNA and uncovers a novel ascospore stability defect.;citation_pages=612-620;citation_volume=48;citation_year=2011;citation_issue=6;citation_pmid=21295150;citation_doi=10.1016/j.fgb.2011.01.012" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=R. L. Metzenberg;citation_title=Vogel’s Medium N salts: avoiding the need for ammonium nitrate.;citation_pages=14;citation_volume=50;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Neurospora Newslett.;citation_author=R. L. Metzenberg;citation_author=S. K. Ahlgren;citation_title=Hybrid strains useful in transferring genes from one species of Neurospora to another.;citation_pages=9-10;citation_volume=15;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Canadian journal of genetics and cytology.  Journal canadien de gÃ©nÃ©tique et de cytologie;citation_journal_abbrev=Can J Genet Cytol;citation_author=D. Newmeyer;citation_title=A suppressor of the heterokaryon-incompatibility associated with mating type in Neurospora crassa.;citation_pages=914-926;citation_volume=12;citation_year=1970;citation_issue=4;citation_pmid=5512565;citation_doi=10.1139/g70-115" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=D. D. Perkins;citation_title=Transfer of genes and translocations from Neurospora crassa to N. tetrasperma.;citation_pages=84;citation_volume=38;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=D. D. Perkins;citation_title=Chromosome rearrangements in Neurospora and other filamentous fungi.;citation_pages=239-398;citation_volume=36;citation_year=1997;citation_pmid=9348657;citation_doi=10.1016/S0065-2660(08)60311-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. D. Perkins;citation_author=B. S. Margolin;citation_author=E. U. Selker;citation_author=S. D. Haedo;citation_title=Occurrence of Repeat Induced Point Mutation in Long Segmental Duplications of Neurospora;citation_pages=125-136;citation_volume=147;citation_year=1997;citation_issue=1;citation_pmid=9286673" />
<meta name="citation_reference" content="citation_title=The Neurospora Compendium Chromosomal Loci;citation_author=D. D. Perkins;citation_author=A. Radford;citation_author=M. S. Sachs;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. B. Raju;citation_title=CYTOGENETIC BEHAVIOR OF SPORE KILLER GENES IN NEUROSPORA;citation_pages=607-623;citation_volume=93;citation_year=1979;citation_issue=3;citation_pmid=17248974" />
<meta name="citation_reference" content="citation_journal_title=European journal of cell biology;citation_journal_abbrev=Eur J Cell Biol;citation_author=N. B. Raju;citation_title=Meiosis and ascospore genesis in Neurospora.;citation_pages=208-223;citation_volume=23;citation_year=1980;citation_issue=1;citation_pmid=6450683" />
<meta name="citation_reference" content="citation_journal_title=Canadian journal of genetics and cytology.  Journal canadien de gÃ©nÃ©tique et de cytologie;citation_journal_abbrev=Can J Genet Cytol;citation_author=N. B. Raju;citation_title=Ascus development in two temperature-sensitive four-spore mutants of Neurospora crassa.;citation_pages=982-990;citation_volume=28;citation_year=1986;citation_issue=6;citation_pmid=2950989;citation_doi=10.1139/g86-136" />
<meta name="citation_reference" content="citation_journal_title=Mycol. Res.;citation_author=N. B. Raju;citation_title=Functional heterothallism resulting from homokaryotic conidia and ascospores in Neurospora tetrasperma.;citation_pages=103-116;citation_volume=96;citation_year=1992;citation_doi=10.1016/S0953-7562(09)80923-4" />
<meta name="citation_reference" content="citation_journal_title=Developmental genetics;citation_journal_abbrev=Dev Genet;citation_author=N. B. Raju;citation_author=D. D. Perkins;citation_title=Diverse programs of ascus development in pseudohomothallic species of Neurospora, Gelasinospora, and Podospora.;citation_pages=104-118;citation_volume=15;citation_year=1994;citation_issue=1;citation_pmid=8187347;citation_doi=10.1002/dvg.1020150111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. B. Raju;citation_author=R. L. Metzenberg;citation_author=P. K. T. Shiu;citation_title=Neurospora spore killers Sk-2 and Sk-3 suppress meiotic silencing by unpaired DNA.;citation_volume=176;citation_year=2007;citation_pmid=17339226;citation_doi=10.1534/genetics.106.069161" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=M. Ramakrishnan;citation_author=T. N. Sowjanya;citation_author=K. B. Raj;citation_author=D. P. Kasbekar;citation_title=Meiotic silencing by unpaired DNA is expressed more strongly in the early than the late perithecia of crosses involving most wild-isolated Neurospora crassa strains and in self-crosses of N. tetrasperma.;citation_pages=1146-1152;citation_volume=48;citation_year=2011;citation_issue=12;citation_pmid=22056520;citation_doi=10.1016/j.fgb.2011.10.002" />
<meta name="citation_reference" content="citation_title=Glossary of Genetics Classical and Molecular;citation_author=R. Rieger;citation_author=A. Michaelis;citation_author=M. M. Green;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=N. Samils;citation_author=J. Oliva;citation_author=H. Johannesson;citation_title=Nuclear interactions in a heterokaryon: insight from the model Neurospora tetrasperma.;citation_volume=281;citation_year=2014;citation_pmid=24850920;citation_doi=10.1098/rspb.2014.0084" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. K. Shiu;citation_author=N. L. Glass;citation_title=Molecular Characterization of tol, a Mediator of Mating-Type-Associated Vegetative Incompatibility in Neurospora crassa;citation_pages=545-555;citation_volume=151;citation_year=1999;citation_issue=2;citation_pmid=9927450" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. K. Shiu;citation_author=N. B. Raju;citation_author=D. Zickler;citation_author=R. L. Metzenberg;citation_title=Meiotic silencing by unpaired DNA.;citation_pages=905-916;citation_volume=107;citation_year=2001;citation_issue=7;citation_pmid=11779466;citation_doi=10.1016/S0092-8674(01)00609-2" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. U. Selker;citation_title=Premeiotic Instability Of Repeated Sequences In Neurospora Crassa;citation_pages=579-613;citation_volume=24;citation_year=1990;citation_issue=1;citation_pmid=2150906;citation_doi=10.1146/annurev.ge.24.120190.003051" />
<meta name="citation_reference" content="citation_journal_title=Journal of biosciences;citation_journal_abbrev=J Biosci;citation_author=P. K. Singh;citation_author=S. V. Iyer;citation_author=T. N. Sowjanya;citation_author=B. K. Raj;citation_author=D. P. Kasbekar;citation_title=Translocations used to generate chromosome segment duplications in Neurospora can disrupt genes and create novel open reading frames.;citation_pages=539-546;citation_volume=35;citation_year=2010;citation_issue=4;citation_pmid=21289436;citation_doi=10.1007/s12038-010-0062-y" />
<meta name="citation_reference" content="citation_title=Neurospora: Genomics and Molecular Biology;citation_title=The fungal sense of non-self;citation_author=M. L. Smith;citation_author=D. L. Lafontaine;citation_pages=9-21;citation_year=2013" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="Introgression is the transfer of genes or genomic regions from one species into another via hybridization and back-crosses. We have introgressed four translocations ( EB4 , IBj5 , UK14-1 , and B362i ) from Neurospora crassa into N. tetrasperma . This enabled us to construct two general types of heterokaryons with mat-A and mat-a nuclei of different genotypes: one type is \[ T + N \] (with one translocation nucleus and one normal sequence nucleus), and the other is \[ Dp + Df \] (with one nucleus carrying a duplication of the translocation region and the other being deleted for the translocation region). Self-crossing these heterokaryons again produced [ T + N ] and [ Dp + Df ] progeny. From conidia (vegetative spores) produced by the heterokaryotic mycelia, we obtained self-fertile (heterokaryotic) and self-sterile (homokaryotic) derivative strains. [ T + N ] heterokaryons produced homokaryotic conidial derivatives of both mating types, but [ Dp + Df ] heterokaryons produced viable conidial homokaryons of only the mating type of the Dp nucleus. All four [ T + N ] heterokaryons and three [ Dp + Df ] heterokaryons produced both self-sterile and self-fertile conidial derivatives, but the [ Dp(B362i) + Df(B362i) ] heterokaryons produced only self-sterile ones. Conceivably, the Df(B362i) nuclei may be deleted for a nucleus-limited gene required for efficient mitosis or nuclear division, and whose deficit is not complemented by the neighboring Dp(B362i) nuclei. A cross involving Dp(EB4) showed repeat-induced point mutation (RIP). Because RIP can occur in self-crosses of [ Dp + Df ] but not [ T + N ] heterokaryons, RIP alteration of a translocated segment would depend on the relative numbers of [ Dp + Df ] vs. [ T + N ] ancestors." />
<meta name="og-title" property="og:title" content="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1263" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Introgression is the transfer of genes or genomic regions from one species into another via hybridization and back-crosses. We have introgressed four translocations ( EB4 , IBj5 , UK14-1 , and B362i ) from Neurospora crassa into N. tetrasperma . This enabled us to construct two general types of heterokaryons with mat-A and mat-a nuclei of different genotypes: one type is \[ T + N \] (with one translocation nucleus and one normal sequence nucleus), and the other is \[ Dp + Df \] (with one nucleus carrying a duplication of the translocation region and the other being deleted for the translocation region). Self-crossing these heterokaryons again produced [ T + N ] and [ Dp + Df ] progeny. From conidia (vegetative spores) produced by the heterokaryotic mycelia, we obtained self-fertile (heterokaryotic) and self-sterile (homokaryotic) derivative strains. [ T + N ] heterokaryons produced homokaryotic conidial derivatives of both mating types, but [ Dp + Df ] heterokaryons produced viable conidial homokaryons of only the mating type of the Dp nucleus. All four [ T + N ] heterokaryons and three [ Dp + Df ] heterokaryons produced both self-sterile and self-fertile conidial derivatives, but the [ Dp(B362i) + Df(B362i) ] heterokaryons produced only self-sterile ones. Conceivably, the Df(B362i) nuclei may be deleted for a nucleus-limited gene required for efficient mitosis or nuclear division, and whose deficit is not complemented by the neighboring Dp(B362i) nuclei. A cross involving Dp(EB4) showed repeat-induced point mutation (RIP). Because RIP can occur in self-crosses of [ Dp + Df ] but not [ T + N ] heterokaryons, RIP alteration of a translocated segment would depend on the relative numbers of [ Dp + Df ] vs. [ T + N ] ancestors." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"I1ZUHbVSy6pUpV-kBvtUkUKDtvL_5Nq2JLrC5CmfK1Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459965","apath":"\/ggg\/5\/6\/1263.atom","pisa":"ggg;5\/6\/1263","ac":{"\/ggg\/5\/6\/1263.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1263.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1263"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1263"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1263":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RvgfLuWDN1XUhLwkcEQdYra3WEeH\\\\\\\/krlsmcw8YyLpgOWhw8x4pwe479g4N9fx2Q4mL2Dsle9b7uJO0UGjPGLfy7qPoPk3FFwsSFArdAWBbXVKKFzqVMWizlKUR5sKH2RUEc0NCx1fn5lcwlQ\\\\\\\/UdEyexBaWYidZhOilTQLjKCmUwXApa6\\\\\\\/B0dG3ULJcxhthJQpPDn2du58jOrLHwKceYCyWuI5fsxLdqopbYuScoKTAfCsam16GvsfunmxW6xPjwu4CyxzCQyvXVrRWZh2ztjLiF4MeoFG2vePwTYBGSayTEn4dU4VblvCPqXC\\\\\\\/LvcK4d09tAOvc97vnOCKb00tPFxZ9j0MtH6KgqUAvB49CQA10VON\\\\\\\/jP1ln9Qk7+57MABbg4rXReHRg4Dj6jaDoo8m8dRFY0rldVuZOja0xR3L8nISaj9tLyp0mSwUdTINCzXm4phuTmtepJzsAqj5wiFNXi2\\\\\\\/IzDYWmSHxIcBPvbX0pFkcm8Fyu1I\\\\\\\/t6JPVX049D0JtS+3EV9fvk2H6jvMMoVrjAZaVwVOhfKGfu+cjPO53ww8U1MhorHteI4w98HgXdfYmiY5lutq\\\\\\\/PC+Eagjql8KB8SlxSSVkydYVCRQuRw1zyb0tZ50\\\\\\\/hLsith8cnZNJoJKagA4oyytIVwBTZQtyhjoxYtuIkwu8FFuE4prrBhAa8tbIZpabr0AyTwzJxQEV4KNHPUw\\\\\\\/IJZmarSx3\\\\\\\/TsrN6vwNL64zhFV0V34DcMouXmE\\\\\\\/2hlY7OfXcCvpA0w3db50zWCacVuQOkGtahPfR0usKuG31vfsDaZA+5XokoxHvtcr8w7fXBLuP0yjW\\\\\\\/KwkBqksmuqIISlzj\\\\\\\/pRIvfXBTMLYYBg5t2l0vgDus5FBUnkUqoWrNiG1gUQJ\\\\\\\/KmWugcjitZnKfQCyay5ZMHVEyv6MgXK0quJxsi5nP\\\\\\\/5j8fu\\\\\\\/EaX\\\\\\\/IqKdWjg8KDVvjYygYBcBjvkyH\\\\\\\/yXFV2FQ5aKAv1rziz8KexdMEgRe\\\\\\\/l5lkV7jqPfm3l1VSuK3u4elSUSqlF89K0KS\\\\\\\/c9Xux0tUWbNqsDhjYrtN0UzVGhoByuWS45akoNarjPYqvHRZpipgUG5J+mwbhybKYo0WbaDxIyzocbcqN4yirteG\\\\\\\/0vWM9tNNLvrugUuBALRyNsea3eGeOsmB8D1YjK+soear5XjItpSFsUgoQntG3DVvdBj5HOmiw02Xejm6bubf7BMB2KsX54Xs8bf58kAQcizDpX1Ccoj+cN06ie5r0wxQj2eeuUMCoCDlEf3k3VK9+KjEObN78bkqefW\\\u0022,\\\u0022iv\\\u0022:\\\u0022vV5qz98BJsXCIXkGCU51xQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022732eb858b484d7fdbe9e7a534b0164e41574edc56b53ad6a292d235f7dda92ff\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eXQkiSGhkNJ2x2hyaSqihvOITvEzWQElP8Ljy5TvkVmKd16lqPCqMTKRy\\\\\\\/bUeqK5E65t+bmkDRJ5GMAdNGCn9B2VoRfg9KBz6mMmNZxx\\\\\\\/3dmwg9m25ivPM13Ni+\\\\\\\/lF75s2lWXpp5jh05iHDjrZe+KDXwe5AD5GD1bYIGfMCz2sSK2fzB20S+VT50MQ\\\\\\\/qxb0JN4ccCLTrGAqbcdbM8XdQlyIwA26azjTIC9PnkuGMuqpo1\\\\\\\/LazWYdn6CiR9qilraKQ2tv3snmO0ud6rrXqUD1MSLA19f+6+MEjJW2PshUXATNh1BqdsAw6mydjPWz3odA78UENDhNViDO7uqPKtlnt5CwaoGoQpbh1RniQ4bXEv1Kcnt\\\\\\\/SL7zHEV52HK0379gGU2c5w0xt\\\\\\\/li83sEfnmBzCaWzdHMfFIIPBYCuLu7LxQubvjK7H0gORmmyBBMbbTWEqj1AUgGzlQ1KPxkBzGJohRi5t01Purh\\\\\\\/+trYuA3u9DaoYrTg5ZjAX+ndLPEsh8Pi19qOe0cis4PW+tpkr4bKEISHG4RLxbLDeOUZTfrL5FH0KmuXnx7xzJkOilN9zCsJvbFDUneYI8pv3QvWw70Gw1E1MMwf9uciIkKweQHzX7upJ3v5G8ZoeIDYju5VFoMg4TnugaMZ9dFtxYqZgwQnv1B8gRXdEMeFsq4dItBBtvy3jOK3MMKJIWlRchRfKxK3yvA3bm36FjbJQrXoM0Y6yh\\\\\\\/CUldhxWYOTKHQ1YVCoOldxaRv3P5g8GkNWKJ5sum0+5ILf6LGXh8\\\\\\\/9zMCse8qqt0iNakSvtte2SJ8kZS9IDW7JME57vSl+5PkXNkzfNAWl3SNmq+V2nTWMB6YjJRXx\\\\\\\/m5LexF2a4T9jQqiTAgBXfzk8CQ0FiDKGix5O69\\\\\\\/El0FgUQga3mXnDpPXRVJ+ER\\\\\\\/JqR\\\\\\\/qWuCQE7fHTc39T29wWJoYIwmyWepvGlo23Gc0WQ1mzE7kB8tHy8aBFdXdU2eS67wWwPVSyJuHm6Y9IiWQGeRJDe+e2cTtBaGPCfNA3RqJslBog0\\\\\\\/ABwc2RMVYgG5fezoSdsQrrzyNhcJnC3pg39b7vrsQoi0fXY3LYIrGLUh+UIvwRZKZFFaSZJqzpfSeEpGZ+z7daP7do05Y4yyfTYZEvC6MMWaLmf8kmW5VVqLUBtz\\\\\\\/y5u\\\\\\\/KpKscFTCFKjItCpOCIQQ4rbr5t14a61hBQ35SX7mNTKQJAVorUE73NW\\\\\\\/ZOOhsClGFrW9tjHbESzBgc3FibVZBQ77+li3lLaJ8paKb7Y4\\\\\\\/2J7wsitaP9YZ7PtrYrC9jQtOfxMIttev5L44UunhO3\\\\\\\/kwEf7l1nmDKrLPFv32B554HjzdUpCWBM\\\\\\\/4OaHBB77xWlYxbmYxL1i6ILSFYR9M0ZgdlUrfT8tAjj3w1EsyuQIqT4pl4uqySf+B1pJPqr5mLLx3C8k01PXDA8wJ7vQlaXtn841m1bf9wt2+v+pF+fPQKMSItkr5CdbzLfrdM6IhiVk+HxDSlEhTuIpgmnEWOcmoc2YmmkYQe8W7kXg67nhDgCcFo1aw61UgcaLlLQtT6VgiCo5S8vTt6WvdErf7bdwhT6v5wmVg46eWkN0OfFoLBEeG+E+QjJlg4oDzIXEyKihzAxOzd3HWRhoJRjpwkZQsHzJt6SA701gHMDbHCQYiZnRNFmJHwlDopv2QdOjH2\\\\\\\/IqUw8FHXhJ3kHui8KK6djW3XIFY3PWwL4oV+1LSSMBu6rmBWT1uHlmT7GxgSXNe07XW6BTGkVLl+CQPFVwGZqL4R+gzclQtStTlBg3Qld692\\\\\\\/0RblvG8WeHalQZjaRQr0dhqiS18mo4\\\\\\\/2sDyT6guyFxj19rInBoXz3a6H5aC+jQgySSqkWjYtzlN07GwPeaHtDP3Qb4f1Boog3LStvKIySXS2JVv6uMBd8FgnX14yGxgJ8fUuQsckx+Tk1aiwN05xnj3xEB2ZMtM17SX\\\\\\\/kEUDjyDyhLK7ubrC9lhumVeb3L82y2O4PQK5Zn0sohiXyrFflIFZyz0y8Nn2EGhUeMzRupph12dfa74IqXvXHtc9R704SyLjAIuTG4hD\\\\\\\/xiiVgAUyYJUO4YEUrXHZmzYRP\\\\\\\/yOxvVywXMiwmLBbiA+Son8hVSzn7F2jf\\\\\\\/4DVrp6Gb2nbL9UZ1HV4A9fEjdfnJS8RP2MOG8Pd7JuExLBUcOnVvXUonzSAn3ehYQNjVAhA9L8EyzMsvdmBSQlInCNz70I32nrSeWpx3t\\\\\\\/yHpTFzuDj8A4ZxBoxNAJ8K41W5iwerBpcbhm+sp5Iy33xKoO0s8hkBRR6ms+QThHkcUlLI0S\\\\\\\/WqAs\\\\\\\/xdb3oX7jdCuhQdHnHlJ+m3nma8DPDmir2SQGYGQx4x5QalH5ttqqb1m154KtwNGw4D8Silwc2uyyK9byOTBjlQwxbajIPZLhmn5vj+GxDdUPwNY9znGE7WmfCVyYu7lySJ3J3BDZCEgduuk1sjgieKPyH6Rhy8G7GJ1wflk7TeQEcxeGgczy4hurvEz0LGPIIycp1vaM0jei7NPgcN1M0miT3AANNljmvhLbxfNrehHdAtl9L0GsIYdL3ZMe\\\\\\\/XItX3\\\\\\\/UObgdlA1oo6a+lRJhSmp3x\\\\\\\/49tQwK69VpSZY+4B9NOv\\\\\\\/fHNpQCwtIc2EIvXGzeGgx1pbmMA+YA\\\\\\\/VK2qkm7cixbTSixL8x4z536PRN\\\\\\\/NNsNhqJ60Ty6KsyAGMhgPmNRCNTA+xQlyj+NqcMqIxa4MWIHtesCIC8+rsHWMRFh9X3+MjYqUWrOdxR\\\\\\\/8twgrOyh6FaH68l4HJb1KfsKRxaUr72Y5VzNVJHmRtdyTnpOOEdKTahiDsVFXzW9VdEy63J3lvVwBphCIhYPKLGHx+pG42YRnXo6FIZ1l2aORTfWHkhz6eaTExYUAx8ay57M3sEGICUsLj6YWwQxYzbbrNm8N08VPxKMslBWnMLQO2DoSmC3NHdP77QzPplpmuf6j9VqZp0UynHSpvtvkik5Xv8M8knzhJLRT+UnV53y7YLC9wzq+c9yPLqGX2QRMWCpEv8j0232nuoKGHLRYY81WOiDVvk00mBjL5tYLvihH+I4ROoCoiULVz+BYtADaNMXP\\\\\\\/c58yaM+QnbBeELwbIoX\\\\\\\/R+E28de0eLRSA\\\\\\\/LpbBObgZzwWmnEAilHwPUBqLuprVEoc1LdUkKaTjaSKEZJ7OO1c02Tag0fJKFaMnMbqzFYKnxyJIZ2J0bErCxqCYkwvy4Cp+3TrZ8ivq9uBPnVzt\\\\\\\/hQeHE5L4Lsw4zdX9ro0FHvXCMqk6MEVSzxMhe\\\\\\\/5S0ugQCr0rw6vS+sOJ627CJ2Y5H0R+tkNdNDXGeiV1LszM0kOsvKRO+5NF3NteqDdoC9KMpkyz0kf+6TKV5nVfzO1XST\\\\\\\/bTzmHQWqpkIJMneYzwtwplUyql5oJJNa1BcWlrNasQCyCujtUeT\\\\\\\/a6UfL0aWeZU7o+ciIN5GCgRsUOgx7yMVNFtSx0sx2sMREwiG9wT253kA3\\\\\\\/OLmcVRouG7sf5c0Q7iJTsDXnvou3OI744uP1Wv+1PMMIgg9H1TlBVK4lvFb2bMaBWmzimJwrpBDM1\\\\\\\/cOzkwGbVj5jfXdrOToawVlGl84SEym0N+NC6Rx\\\\\\\/nuEAj5TJXJPs9a7lVCVriH6D\\\\\\\/X7hatj0pXMsnEXpei+CHc24PO8DIUqN2eFABkIsqQGS0JWHjmjJQGyNGFANmipli8bj5UB7FY2RTx7J9bNgBOw7FB27ZX7cqmKIcHDfb+e2w5Z4XZ9Wt1p9FZNxDucjUDG9urDrjNKU1w+86GEGzHgP7lmB0Jw+You7mQJFjbxLWhBZJmLOjVFiqcSbYa7KboveN4HoJgB73rhTEUUymFV76sW+RJ1XLgGdhnwktkUTHgNL4CD1vMwR2UyqFxTuBzEW0w+xdN2TzTyruB9oU0L9wS3fUizPRYbjKsear0KxU+chTKgZCw7xkc3Oz4rPezbEUP4Dn5g1nyIUS6HKzjEb4aU+zqtFJU1LiL0seHM3L26GUUpTQ6pCBQy\\\\\\\/ld3wnsEnRJTFVK13n\\\\\\\/tVbcXLQB04owdfi9X0iloyJYlokFVLJNQq03KKTxI7bRoZjrqAvXpL2PympJL0JvCHy711sSmEJpMMa1J1+7UwZ31hz+46VPk+ArFaK+dp4C14t9H+hHXsFOXBa7tP9tcPIUiIISuF1JPi3mPPF1iwpNvfg\\\\\\\/Bxq8hZJ9Vksv5ppqPOPcjB6ijhQ2RQiWSMRVOJN5Vj19Jg29Ia+ZBpgEawZF0AjtxTHAq30ycvriUOUIDLc3fkJyzreMbvITF+HvJ8oqnudIEXD\\\\\\\/Nc82krmHGroHHzgFkgfv3DTVLNjuRKKG2i0iemkRk0hBjXcAKATwTYWOcG2K67M3Kp1tBX+A\\\\\\\/UvCh2Lk\\\\\\\/1C+bttNLqYRW+8Z7bs8Bokmokr\\\\\\\/BKKJoLN6ITr0QsahoNkEgTf6kJX2w1xniEYsr2KQWAJUfcAAG276Yz62gqwBaX2GvKe6oPIgxY0nbVBHW+V1eeP3WNoMq1Cr8089g6XFUuq38dgRwNCwszqP4bYHuZCnIezdUtIMqIZGTcKneMZO01UrQvik2vuuUFuhbL0uzC51FtWpN9fdFgHoLcaxLfdcU9xgONTxCqfvjhBZ0NlRsESUOkxYJZRuInJhDR9CYbz7aApuUfmqh+EmdgsG+DKMgIO8i2FRhCwRoR4s8NVaKLGH3ZlyR1Fr7s6JAG0mItgiBkzBnZmXDad30G3pvMQzdsivRySKj+BkiGokN5EUTGzYSRysgC7ORqsgQYSLuTAUVI+BUCOEx7StqAiKejuG7HKRnMBnVG+x0cijl6oWjLaOojAAPJ3XdnJXHm9xOlhJeirqpdI8NfCsYwIgOVqbohumY7lpMYeQUYDXDiaS9v8oVRZ5DRbXn+ckxPVTmdtGKqZZ+RHuG9cyIhj0butyZubbq8RIkNdyS3pV1PGaT8CHS5t0qvJeRe8NC7fMW5I9KpsRrYrKXR6PkVQZQaXN3zcNgNgtlBZPYJBNS9+iwgn7EzdM99XIw4yGhl4oNWPYX488tGF+QDu+RMOEQNq7k9n3bWs06UZp3iLSIR3Wt2I\\\\\\\/xNbDAVJbOC0uMyNnXDhYH973TJ+Gvq4+wP3k+3MZX0Qhmjoftfe0FrqmvnLm9Iml2PbcRyn7fV+5jKnYkpbDmjhTpoZu+6qmoBsSXELjLwAuUxCPAXvhLpryoTJgRbGz+cFFGHWgVgJFG6552IZ582nYRMzxMespiXcgFlPLSbK2mA59ogrYOGpin0YqYg3WE+3wZGjjSMX7GAOOIYE2\\\\\\\/4gHhXY6EFBRPHq\\\\\\\/Epg7JkJrseC+rnR+\\\\\\\/KbWPBszRLdAfBCu9mReDNtShTDMGpnfDbaOQZc6ob+d+wnrJQ4UpSC29fo+3Nh2wDVJsExeIyt42CiyoU0gWxTmSFN2Hlw6VZSk++8+TrRbpfRJPL3uMOu8wJtjIALhvTXsiK+iNEfS+2p5PyUXFEZSWONj4hXrkRrGwGOt3ztD6sjUh8RW3o9Lr0UAk\\\\\\\/h2VG9GBfNItFqbfy02U8gQ3qt61iEdw6gqtTqBVWJnde\\\\\\\/V3nuxo1tRu29eR1OwAWKvWEaRJyosD8qILuIJxebiZ8CCK1DSfBm2Rr2knAZKp6dI77zM5GsOQ9TCo9lNW6qT+GKBhhxCzbV3wE6D\\\\\\\/ViUefef3fJBYnVREcscQgqiEepB9n7Nu9C+X7vitWUodHBjlspXb3wFV4hvJn2pMXt3lp6E7e5JAEPBfqjFbFyFHfSOOjkimTJ3NP+vHmUnDC70l4iU9S4Cveq2pIeg30UOFXwNP7scgggHdPJJZaUCuYcQfQNT6Rl8myprjTeS+VuA\\\\\\\/JhfmGCCWUSnRYKKy0HXjzuqheVGulCO80jzfGmfMSfA1trKr14hg5lf+XsJioBlHBOIebLbAMJoCtUbDd3I396BuqYLYCZH6T+AixYO9xG1HrYlvzwQK7cSMbdtigqwZhT7SQmOtgzM2seKnXTyKpqxLfD2gXI4V52SrMWGiQ8U+FngN7zJzkOGQAEz2MRIN3thwX1Th5lVNrI9MwmD\\\\\\\/20ffaUhNb2w6BrZpsoxNBr+Vufu\\\\\\\/tw9KUjjMl+CbX1AtbIdSlJPLw57qyjlkuOllfueVIaOyzItEb0ut+rkx1\\\\\\\/NVuLNwuni6B4DFkXbCz\\\\\\\/wxM7b89P3B9NNazaOnKALbtBkseA2NH94X8F38Y6gwhPoGTjmV1k+keXmrsQN0JiZzjiCydi\\\\\\\/MwHqQR881GjHT+P18cT1u7vS80fkAVr7eJz5N3GrBBINOUDMsKfQHiUTUltIvUd0RoJz6qYlimDU7YOiwi\\\\\\\/vJtoLy2Lfli5P12eSQHR9\\\\\\\/wxdEGw9yWuK5nUzeUTVh9wnq3ttbyRdlzrHiv\\\\\\\/+UynHDCPFskOItou51ZlFY2SH7cDUP76vS5JiqVwwzP\\\\\\\/a4r2dr09\\\\\\\/\\\\\\\/ERFTNldX0b\\\\\\\/nva3SMYYy7prCH6OV4WjB6Y8X8LsoqeE5XgXzmJRu2\\\\\\\/U7bnLDA7Xf6qkSDaaYWAks2ypqQqa9+Pp9sDiU3nlokxVQXRGni5MKLPkHKqfe0JbOu3DeYECV3mNYNECHkg8jbBysjL5\\\\\\\/QCBrOF56tNLGfLMfG2c7iyNtBIcDwJZyKbpjanmTTUMEDRzWqugj6u3IFeDRcHUM8+muCCENlvFlhY3Cv52tXzH+wg6+g+i1vjA67ehkHfvDzwuLkhlnNhgHVF0kZeYMrLtNuweFyZDr8js0TXTzZ4KAuZ67tSkvSztiwqGMzQjodEbz67n60HPpeJUGahg7crVLJnS6QTSNkJmsyFcAEc9ffl9jZodsq7QtJo6GcvzGoCVwSdCJ7XK6GvLdvNJOYx9WJzof0HEOBJDNQ+4TmmZr\\\\\\\/VLJknz0JBemtaFuIJcR77AiWdDGxFH8Z9sLandnwwtq2XmPqY1ye2+KW60I2d\\\\\\\/DK2g2tUbLMKYXoNfvCe9ww1ujUHxhPtqu7G3PlAye9nR1ixX8Dw3Je5LfTlKtN07AgVHDGxng8OrqLiX65yojiUq7eGVNdR1eWaHbseHqfSgbYxZMemWTYYKrsAgq\\\\\\\/NylzMHylM7Rp3gsSyFwTv+U1U3873aiVwz3fF0zn7ykgbDTKYGpF00kRNUuCdR369umbfLT0llPx\\\\\\\/laxjbUPR0XP0yp4Oy9mYg0wjts3yhalt8yoAN1cP\\\\\\\/ovD9XM+bd\\\\\\\/hTRThSFBnMB5WL0YQbu0mmgg9LtErevloEUJKBAR2HjyYv5p\\\\\\\/bEWxwB0X95yHBCendYbJvNHOVez1zmTr80DUhEK5JZ1xhrhiHCZKc7Ma2WES4HikYiw7wFnXdr+tggDOOyFWwnUMDF1CJ0LCqsBloIkfVJ6eYK7aQtZtQBKqUeC7CluHOV2A+gAgwRVcezVGcvianEowWCZztobotSTuD6djM2WRTZ+tvMXZ6rg6xI\\\\\\\/4Oph1plKmLPClf8GA6hCKOC9KWKx0WcCEtL5vJsqSiSXuYk\\\\\\\/yt0gNRB6k66321lS180sFlfVL0WDf49Nc7ZVZDpHuZRG+JpYjUqjJMPWp6xQ7M0JyaqKWAKt20gePlJW3uxUBtE9VWVSfdA8oR9BaRemtVnbEYjFnqYet\\\\\\\/7Ocd+9iIvTBVhCqkeNV8iWVlByYdq3YIbck7YGsh7VvEo\\\\\\\/+f+5vp9+5MxfrXSyX85QHP3cikjJB9J1AQLqFXC62cSCxWx93GtDhoXBv7r+SbZPmT\\\\\\\/7u62Z5+7K8AOjHl13F5TjmuW8RkgyPUjVHlKjd6AxaboZqi0Xmozt3wRFwua\\\\\\\/uTfIl85SKT1dMuC9w6FmQyrCZXn5An8ZiEoYtJBcJKYCAOEkt5QhjtRVuaK4suvCdetu3eTnEXWJn5SI+3Hq8TxCYIrUC27TI5na2e79z7ZLu\\\\\\\/\\\\\\\/MQt7Ys\\\\\\\/JcOTcioe+uYzJp2NA9zNq1eJOeqalevtUPi1H3TDGuRR5ZDFOzgT6Ly+UfFtj4mEl87nk+F7NUyH3VIbLvRNFCv8q\\\\\\\/VE7\\\\\\\/XYVgZg7ZY4aFt6k4AXdRiAmZhWwjCPpak3bvucGmHnoXdPt3NLrWQqtrdOeLSxiFYRAlmSPMPjJLJ+SopV7L\\\\\\\/q0Ia0igJSwLdzesgyiY4jHOrDR7xqR1ZV3iffQiVNffR4M6F61FbFM69v3F2u2LUvZb7APPlNAOLhOfphWcbGAiir+meVPAxJeph83WuJvhO5\\\\\\\/cX9k8qigwbvs+bnK0LzN074Nvi0Y4l\\\\\\\/O5eQ8WS6YKtIgUHkknHpnnIV\\\\\\\/qojkAot39C48NB3oF4X7x+TnMHCwpnLHuWzV3uoJI5RMKk7xM29R811jtdcFTpPawzFTkqUfR9oBBcConAap7GbskkIRYorgqeMG2v0CdgVvuduNuNwC34Ax7iABEQJyEDaiUznr\\\\\\\/AJVsdWHJ+iwlhnK0qpcuXW5JE+tqQ5fC5ubF9zDChwQvyp6BlXfVrve+orPd6835DOSTX0IZmEUuW1u7BH+CKrd1VPaWEWHdbPTPSWwz1f\\\\\\\/31jeeGjBCmgxXsjeaNKVLAGyLvoh09UtyCKAr77OEYjlJgzQYivJcpH2SQRr7lAnucayEGQEWJxT9QII\\\\\\\/uKDLVd0Qnd8ffHVaTLqgTGjHfNeAQeUZTR3GfLHNxirDTYoUhhpki8Anttd57yko+z7aZFVt\\\\\\\/fQSJxRv08XK3XHNkZIKP5Y1ABPUDzZd+vI8oAJ3yLP8vxf\\\\\\\/h5i18jn8By5ZgbeFASN3Dxtpp21zcRLrEjSb3GRdGe0xf38Yk3SResarbqhRD\\\\\\\/dClDt\\\\\\\/gRBWz+6TjEuVptQj5acTjYaoqyHiz+vlgBRfceG5pWjQ7RPgvJlVQnX+eKa0tQUkFF1pZZgtZT1ysof4B1KQ8iIezg3DavI51pzJd26LyxFBlMXtqUUEahxZSh2rK6bGEKJEBXnWlCUGmnrWQnNet\\\\\\\/Yo5ZRFIcsJdlWU9N2sJMDccJ7dgeGQh0jEvHQdduB3nDiqciWKjKLfqNkDooBQNA4Uww6VK+OW0QiRQ5zW9LPmQ5WegCu\\\\\\\/LK5IeQZbxqLoOcf6PL6ld2rvITBwLpmK9kUgLlGG+hPe7zkhX3H2YYfH43gP7tz7qJohkkGTFDX2wV0z7KVb\\\\\\\/Y9yIS587bhSmL+FIgzUud72DFQyTM\\\\\\\/vd5dNfSi5Hh0o1T+yist+JTK9WTC81gFi4EBsWbWvmMFrgFOVgbLq4DaOHrfySJzXW7aBiTH9Fw6wrSetc1snbzz69f0vhTFrWsGzzfkwmZFx+u7nBStN+Wvrwh6709G8DHx\\\\\\\/rw+jDWdwhGzJKyQEZeJgU18+Hjd4KghLxq0bOO2I1dPY+87QRJQtbEjhj\\\\\\\/wEwR1klw6kntHSWU6+w1CF1VWcLMsAnBKUOPyVrQ\\\\\\\/9QQ2wgEGbLR0\\\\\\\/K2Ct9I5B+aPYbxw6q1LnLbdT7iIz8oocfw+XYOF\\\\\\\/8K0m56Qw5onqSNPQFJ\\\\\\\/YI8Ixa7NCsBywfAYYjC5mFg1FwYxQhitGVNOZ9pZzVGbD0tYfoPabGjkBLR4Ovb3ZB+FNeK7o1ShIFry7Kzvh26Dfb2Vet7baoyrwXufS9fZC2qXvS+cy5O8xTsbkp2gZbUwU8o31Yd5MT6ivdfORMxzxIBJwvhKvvqqfpmGobkrjfWvcDrtjkQ19U5wZZ8+ob7\\\\\\\/PvzUCt8Kvt\\\\\\\/hZIsaYy3v0sD5Vg26gNpB7LQbD0PMaityymiywZ\\\\\\\/iM2FbdAqvq+iVB76n8Hv0aOvG7QjH8peGymGGLUNzns2TTRDaij4XdNIy6LuH6jwZ0UqhAX6qlXMJuTo2qSvHlPsBgHlQzm4kLrOCzj4if7a15Ih+G1Yv1csHr3XaNqdvv2dcjdn2NNLuQR79fo34eDzJtvxuRt08Fl7Xv2L\\\\\\\/MYSY5eqXAujFqB5jh\\\\\\\/2BMdglpOKuc7tjU+7psq8uZKUhMZjbNPscv7bUgmVb2uny96WJVCHijaFEsLSkiywtBNoVKTqR5Ka80OAKhGmkryd93if3j8X\\\\\\\/1U2+K\\\\\\\/5rvuX42XeH5JvsxBas49t9eC8OjrSXtPG9owqccZDSxoasHIcs2BY\\\\\\\/FJt+ER2gbflGgVKsy75HFkPwLASOnIQ18BwudO3io0UuDRZKGmA4MPNNl3OEboyBrwUMnbhWqt7CFERROBjQm9hSHor5AaXwnefVlxrkiEacHiB3n6vLJpqlRzmT3JlRcQe8ba6rAoEi4HYon8rDt8XPVcPqYvflU6tkVExzfDHV\\\\\\\/BbMdi6FnOhUdhNs4wgf1A2drvi2UrOaREYldxmdFdIm4kBuBpYmRfWbLS61Low6wIsNOBJEf2G+jASN\\\\\\\/0rK6ib+bT8\\\\\\\/roO+LPmqreylcXFdWD+FfkDXiPHHuf5sSSYVnlUa7TxQuKhajFUP86cHo4AMdEFGaJLTxACVB3iSYEWIh\\\\\\\/vX4AJ9p2W5vbxTN35sjXBzeoBwq6hEjFgCR9M1GT9sfZY8\\\\\\\/h53hxTlfpmBp0wNfP8UrLOoTiNUzKFCzbJmJ4zk1i5G4rOyvHNmZUuCjRuG4SI\\\\\\\/yBNnocltMrDcUZb3EoaZKXA6wPr91qA3qkIYexxmZpeH0v\\\\\\\/Fo21HW6sycIPqJW1tnW8EF2YQi3\\\\\\\/\\\\\\\/2GyvKx2Y86mBzuyWlbPQm8ebW0MzGOwwd+Mz7wkwIRGF5nDdjb0NxFEIUGsB8iOdhzUrO\\\\\\\/OVY7ADRd2WeM05hpQB5M6cgehax4e1vqTiBCLrqTYbiQ+zago32tAfYarmyr+aSrr6P1WHtzyn6DNNodp74PHS\\\\\\\/Wbacj5qlNun+31LWfp\\\\\\\/F1ag7dw6s1yaz+Sm\\\\\\\/XiBPxVdqMiRCiO7fJXZ\\\\\\\/tS0NR1YGUC\\\\\\\/6KLLRXSRWagBSo0\\\\\\\/hN\\\\\\\/FzIMOIWMZ91qqPi73dkWi0aV5z6Lxk2UOg1z2fUNcmP9K6Xe0XNVBImkm5N80zYZrFsslm+Iu4Gp64e6MQz\\\\\\\/JkvUeAMcfz1iTvyhuPnpbcjWvxU0gIgaJQ6OKOyjpVwk3A9KkA67VIM1oDhESBJnSi9l1Vb5lHp3GqkBicmM00nX0hw5SzG2C+ZF4k5bYOPcrQeOgBXkz\\\\\\\/\\\\\\\/yMMjByG6LIqjieegnuawpw6x01K3hZVi\\\\\\\/Vn3+jNCfpIMRLBPbkWqmOI8RLDP17fzkhGx2MT7oICrCZgmPeCzcXS4OR\\\\\\\/ypu1ssgJBoMmxSyCObsOArBjeLG55CDjEzU\\\\\\\/jnRKkaSX+7fcsyh7lwsUlSKNNCmbXAX\\\\\\\/7Pn2EhkNZNEYZ6fWcKbUfcQg5fIP7Uj0vMiMUNysDoJNGnrEUFTlqN9vW5Nefl5GlO+GbZ\\\\\\\/gix1MGuQTe7UbbvlLGGd4Z4vE5Pk0pL2uaQi0wwhj5A+MsDgCrohzXsrSYuouzOMzhNMpo4pe\\\\\\\/ob368ioto5y0B3F+pBGmrx2FPb+BkaJlPOMWymGuCo1h3tKUymaceyqDz0YLnNZ43gWyuv349JId2ZYBnwU4D5fSLcvTuarqZMtuqPHCqHeogAxELm18PTvWDYALsKIBTm7bmWe8Q7DhoCMtn\\\\\\\/eQiqzZUNnxa2Uq72qGbtcZdjk2O2gQSIZq1clBMmyrGOSXjcbWCFUnnr7Dk7Sj8mDfV0E0cluYuy+tnanuYuxxrM7KDvkpR983LXnJEvCvSYMHwvb3rb4owD+JiJ2gC+ToSYUjWy6BCbiyTMfZASB6JG27JVSYBxV383YuVT0W5FGXRc\\\\\\\/aNzOHtytfenIwiMDt0sXyTt0IYLKeoXyJYpR\\\\\\\/CcHHhIsSjLKkinZsptB+2SpkR2mmGAZD+N\\\\\\\/8Zt562xVZNcZyQUueJ035ijlwxx5A6Wil2DnxvvSTWrhTjSX2pDGXznP5rXKg59cKh3ftCrC1lOGSywrI+xj3aVgA\\\\\\\/U0ERGIf+3hYhJhwPqUyw7LENrJr2j2LvaQnhSPjlI2d7CFCc3868gL2khQH2+FD04MoKoxE7Pt0skG0jKU2JowGjpQ4x8BtgqWOHDdMa4\\\\\\\/yNadLOpL8yoeikcFvAU0AYOUGm0UE447OXF2p2qy3m4SWD93aw6GugsipCgjYUhaHCF6E4kI9eVHdc1OYxNQY9MVo5\\\\\\\/38pohX82IDkXVEJNVGxEAn+WQYc27SPBqjJAy\\\\\\\/2c47qz6vyb4gsqbA\\\\\\\/UBFsQEFTxRrfePNn7z5x1ACUHtlqPfFw1NTky8oLBZHWOYnAaCF8ie1\\\\\\\/s+PMZhMTCtgGWRaQjN7rbDoyih4Oev6JyTQoGR4pAR\\\\\\\/nuL\\\\\\\/Ne80MavdBIAPHVJX\\\\\\\/TwaF0ZhcJWKuz1fBDGG3WT3QZkWSJzmwhU2K90H7WC077V1Y9G\\\\\\\/5CvZr5tOPp5sr6\\\\\\\/RuYDtlOOYe7YXnG9Aaum0f4Eg5ZquSCKlFcl2iopEt\\\\\\\/Mu1aSK+QMzKoB9eZG97dKh50G49\\\\\\\/encwATPpGI3lkYWOU8OsFOmmJquLc3WHN5wkaOyatqoanpbKHPS2VshI\\\\\\\/U\\\\\\\/SmF6u2pLfhEzpvJ58wZkq74rnQ39MDSmHN6BTqyNCeVOvXjuWzcUO2f5L+Atb\\\\\\\/wKc1\\\\\\\/+e+CxbU43vVjDCoxLeSIkm4x5bq8hx4EthTwdNwTVa8wFyMftG+x9it5jT5Voh+AL7e79RnaADpX3l6g60Lp+ojrOI2VJ290DGtKSDNreXFZB5r3nBphJLRLaRddthz0mFKFU8vQa4vx3M\\\\\\\/f+g8etPmwS7oSsNBvVcmv2oJoiDVrkTKGxG3hivkVO0vyBEb6nyRGdH8ORGfxsmyvAK3HS\\\\\\\/yXoedgcIELWb+wu67+KCWSZMZdOZv\\\\\\\/1Rz3YjitWqVyO6huQgV7h0fwB1Fo4nEsCEI4UgoOprfqaCjTeZutHFCtUIj0o0+Jc06JkKxOzIbUTW7JzKr9kQxDEokGdgfrVitVZ0KrJEhlKHc4lU1JQvYZGJvHdg7Cn1UaiRWcCxffUmB22MgnNU2NG8T9i7pp9wQqkGfDHb1eopzQXiVbitVsrjc+qWC+CKsKWHSoNnwBw1QLqCiO4es8cTcRyyYZkrxDTgqGQno8XYZHelQRmBYRjdxntGKDCyQFy4lCAPfHcXbT0RGcl74b023TqikAZYtexElzXcTnbBmpNfqqRLjU+Bq2gjlq58sp2zGgC2UVUOKoRnP5ASKMzqt3niQivSt1LPZcMpxo\\\\\\\/WJwXuwXVhX7kkAPrkkvlljchhs6ypYaQNqwjQ1Liky5eJpcmjhDbxqgFtx\\\\\\\/Jt\\\\\\\/mG3mp0HMVgDbiyDvZSbv4Mec9U2iALJA+s4724iHfLpfIzXmTxinaljeQE41zgl58tY4TMQ5BdnbglyOrYbQQ7sLDvF71yOm58IJ3G9D+Gq95xS6OqN0CM07ta1\\\\\\\/k6Y\\\\\\\/flBeOlJkMLHcOQcpd5wU0X1tasEvKDNRUzeQ6hXncYUUD8401VRVsYF0RwKxoBMqdgCaV9YHaAECrjcV5+8ndLxnPCWU+TzUD0kJtm+\\\\\\\/\\\\\\\/3iyp4XyJ\\\\\\\/e8M8w1W2XaB0cZl5hmJheyJWpU7ZG+\\\\\\\/EY\\\\\\\/No3uFdyR95oOPGzsCyXbU9X7rQ8QbNsTtxKqrrazkObGX\\\\\\\/JRHTbv5MrA9bC0aVqMQNde\\\\\\\/0940\\\\\\\/siYlbJEODWbxr4ekVc33d+anoYX1gQRKf5ZkWLTWL3NM368Sxro3935sVXlgtvY\\\\\\\/oWsbLal+qvivUdmwYYS8+J45Lw9TCZUa+ynvZOSGip5TiZOGVYLaqGvReRYwYtFAW42DGGflwoqoPLtMFSoRZHT40HB\\\\\\\/znsKBRrpnSpKQr0nOnkTcJCRxhWbHwUnf4HwaIinIi08hgRNAV\\\\\\\/092ppjoh2PtqVu\\\\\\\/GCt59i2LPkezRoSbySpWnTZ3OOqp\\\\\\\/l0xft1EVj5BPB0rsBDTfsQy9NakDpEIBvKg49SWZae95jm4UD7a2A1PAvpTMsrp7cLgDluyeNssw+FmK2Hnj8s89a52wLh7217CTzkPHNJjSmFmQav5yMvXJxLISABY2iGGM7sC4McgjJr8Znt1ehKScxUE95ToSkAWY4ldLIML3OSp0mkqVZvzF7PTEy8VCm0TA5GLV0usSMbhm9pLLqdQHj2exCnpSS3R+4qmC9GorssONLL+7r37Vn5HVepgqAE0Q8lZh6a9C1o3KzLLTVrDJaD8jXi3Vp27rq9kwnXLomfZZJuoF\\\\\\\/N5V+edpCmBW+8fPY+PQaxxk\\\\\\\/SFaVFy08fKacH7vnoOlx1FZ74j8Wn3qoet8kzT44i+Oq6jHoK2Z2+9qR2IXqaPRNa+dNsOS1QMNZTyBNp5Xmv\\\\\\\/zp79qBk0Lo7WVHVyUvFbJ8rrztbR4c9OnhxbGX\\\\\\\/fieSptc\\\\\\\/xWHmqwN9xE2RIq6HSBtQyfOUcrBq0W1hvBnaU+r4jLomQM9nBnHi6pDSrw6mql3qunAJSSVvJvJQWmHu0TXHsbA748E6fp9iZQUoQVLTw58Hlwbaw9VZYPPlIaPMrh5dh\\\\\\\/SXPU\\\\\\\/pcVvAFGCOOyf2tGWMMJnMwW79GuhXNF+Ov3BHtyDZUcF8MwpALAkhcEHQHrlnGGJWWMVnsh4Lk4ad6ffuBhSuIdU2Z+00YJmsrixwboFTdhaY4mw18sQ8ZvMt3QEqsnrxOLqX+cqNLNIvrJwqX69VafAS5FgRilrRDSBs6ISEhcRR8sVgjDWCYyw9zsBQ6nutHfnkguxJo5d3w5\\\\\\\/NDk2ttoHIUV26flGxlGJzYnChp6yfIosBBOucwL8EENIOPNEjdlNRU4GNQ2OmRbeq2JiJ3Hq7M3rGBnfslrXf12agH8wVUlCE2E2Kf61V46mpCgUV5+Xj6HpDQnOqH\\\\\\\/ZGkpqmtwtoj6xvhERFqQSV1+l8wuk\\\\\\\/11G49uOUqKNGroyexDqgqBupmne6FR9DgLWMZt6mFS2jQpo6R\\\\\\\/j7WsNEmCwDni\\\\\\\/LflP8\\\\\\\/NaPUe6g0jjOcXsa9hwffqt95h1YLammvim0xd8Hes5urLfIslwLi61xuitGtPDSVnV8VjeuEtD6llKGe5\\\\\\\/7I0HMsRRUjakXN\\\\\\\/Y\\\\\\\/4wN\\\\\\\/iMVatgWxKie9Zrrd6kx9mnjBRHryClEWaG0lyDeko71YHa9vehNxufQDrnV5Gn4CkqSCTcuJm6xIAOsxwwXvRfmHLA3eDORhvyHEp0y0p+OnWshIrv08LgjApPCixSfqQVjNVDE\\\\\\\/qi3+6qybCTITc1mvBp3B3\\\\\\\/A14PLILZp5otOZgbdNPwOyldjxKUwRpjpJ16DgpawmCaSSSUChnEIYZ9zCKvryCQAxrdWA33fq9EhpuXvm3gj3lpVuEqeg3G46ECmMaWJd2EukjPIxjrcKnB1Um\\\\\\\/trLPOO3KPMrRaDaQArBWoMk4la3+E6gBS\\\\\\\/xtsNGoBtOdV9w3rRg4jNLOxTQ5nt2wuNc\\\\\\\/F\\\\\\\/ZzKGzqXyj+0YjS21ZtGHocYZ+ASYlr78Y7VAHmJKNbtQG\\\\\\\/L\\\\\\\/g\\\\\\\/5beh1fqT5V8\\\\\\\/LxJzc2sxWYSeoc1Rxn+i89y1SUI0nR0JRGBU7hVXSPuocycyo6yRgUwCDj7ORsuccqu17zigzuxnQQiexedzrr4qfJj01wM3861nhjr6wIFrvpKX2KQtiQofAjz0\\\\\\\/YcXM1PowcXnFACzjYNNH50rwtsGyk67Zn8Q7Qbxxpi6V6QHIRcs7KgIjVSTLTe5q4aY3l3AhPt35RgXkgtSmW8Li086coPvGBb4ty00diQsgXftrpYrxJRrZmPUhFMmA5lBo8fX6N3xha1awuQK\\\\\\\/YfQayxKsa7t5HFXBwlZ+AB8J2jbB\\\\\\\/DyqJNh7juh8AeY\\\\\\\/0CDB1\\\\\\\/BsYEpWgEhnk9TrX7T5SsGK\\\\\\\/+HO\\\\\\\/BzzmbuwCRVzDrOmpnmuXETxRwOZT3qPUZN6f6qJbfRRW1QIdk2aG9sIO7eiVDQmad23KX97EHB1+JkjIbIgJD0fbE+fq2bG+GEWhZT\\\\\\\/svpgCoSpYNcFcTjMn1zn33VsYrPCvC\\\\\\\/Yi4nmVzmrOok7fgRrR5TZXSJawDhmmorjkgdgU4aPl85rCwp5ZyhjkdtRK6CRwoVY0uMdpsd9JmE2WkL7j+0GiDJqXCysJYKlG2Ay401fSsOuE9LH3OSFEBpmHgusMTzdw7rVxje\\\\\\\/cdD32ZIcDHn3rckL8x6pxIYGP61ym9r61Qle7NSgRGyJlN7uXgr8WIY6OW8qG0+ZIEarFNnlwQbLjBomuJPg\\\\\\\/xgjm5J\\\\\\\/QOmYuVVIqPH6ev9\\\\\\\/R3qzHeYay3IoUEQqUOKU9kl7UHbJOr+76osnK7vCW+8m2BA8VTlcif1\\\\\\\/ZuVk2dg\\\\\\\/EUfsaCdIGnLypNFHa58Ghpu+4VLf1pAVZaCMN4Tm0klx0t6\\\\\\\/TqCji0MInJBF+yrn4psEW2BNWCLLrELB8mwbIKymW4K4TokNzPoCSsKwdA\\\\\\\/ly9jTRnYmsTSyJ9oZI\\\\\\\/mFMpJ4QvFatjp0Q1O5PpPa1JipzokWiD9Xv4umFx5DWU\\\\\\\/VF+kfo1V11CAOBEj17JThRobq4KV7VPWZxx4sGhmx6S6iHQs+4qIxHoNOfyyGN+ONklAYE\\\\\\\/nxy\\\\\\\/FvdQFNL92+g3B38+qTL3Ra4hbJKnyJpvmBdVIoFWWaft6+peyPKj8gwKutB2mu6lNVp9UgUXSrOjE+2jujss0vB+ihcMAACl0XF8kRAvU9VLy3mucztcvIMi40PQnlQMVUqeTTVQPOGvi8ZAouqM30DfDaJwWShBcyspQvVZdXutllQED2TpZ+X8QF4\\\\\\\/DGfeXeQib2Kh05bOFllOemFLymMfPWes6J+b9uoE1jES7fZztRiRRN0mxsimG4nmzcUlug0n03sACAPuU3yRmXuAVDD7h9TY\\\\\\\/B0rRvNOKux2i3HXzQ95RexzUC50lj30OeXfYKW3SlWUROdA63QhqmtMRJt\\\\\\\/1LGvEUXMTedbKcw0AC5072W4hmWF8SDJIQc+VJXosySnvbVBhkg13v4N88sKUrIL+dSNw8VamR9kMgF9bnRhsAl24VGX61tDK05hswRNhLF91cua6cqvJ4MzlGXVh7p4PetxWSI9cpxuaikKTjqadMJrveBSUgrEWlYq69bup18IW2+Y1DbUaV7xQxx\\\\\\\/QMBZXyeMx\\\\\\\/hjaQvnFIB5jITIEbExgWt0b4UBRj9W3DCSgb9GKDH5+2\\\\\\\/T+PVrn7QUYBj1oynrv6QS9zGePj4Rk0oBJs9aWQajuXMfdy+gsOZ53Fd1QAcJzs7Rbuj9by\\\\\\\/\\\\\\\/CRrhOY4VRJ\\\\\\\/FBJm0\\\\\\\/PPhOVuPQQwT+ot6XOwghSoKwWUGjO0StZsVhnvFV\\\\\\\/M9QsG5XubLWjGEpKSr5HIMjKG6BQ3Qj2SVGFM+3u6TXTrfkAOBtgRXutSSmzo0QTH+grLuYu+0B6SoBbs5S1HFfbIItrDz3KA8yA0Kvrfb5Mo0ylDdhXCY6C2C+wCIgC\\\\\\\/po4CsyVY7SMkV9aAiDAWmYcYAfA9VY6t2vAhBfhQ5IQpamWf76EVfiXQlAl5ZuxX0MZu1HpSCcE83Ma28jCbrSCWn1vrD+tuCydVBfOy7w9T2qJXdiv3DD7VdJpfLJmcqV+cdQgHoScpOx3HciyRFqmFSAAoVt\\\\\\\/8f2fRhPMe3Oy6TbqH+C0qdOl9bjGGsoWLscWJpOPramLnLYCoqD3xxsDa9FMvWK0o+w9+KMwo4a+VL37h5eqRLeSAalGgDBLgyrcAiIOHvHrImZCR+JQAR24Y7DqaVPBDSS997qbdaHY1yNRQCocoxWRNMTqCLKKfhmdSN5RZc+LCVuHtjOFhVO9+iaV8ufYdmsAIBjCIlP7OmrETJx4s5CxUEGrmzWYCHL4sM\\\\\\\/LazfXSL64e0IpVUB+NQJTr8BQknlxa7NKmIB64DZwIV5jxxMvd5tPOZzV3WNVKOKmw0wMHDuCVPu+SWI0LqN0uk2l3I4AujmYwMCMlnJ0oYO7MCU\\\\\\\/VjZHpCyfnDDhjP+6GsHEKUn+sh+OMhFpif31ZTT\\\\\\\/ZWp+JLBHiyJF7ePtMaL7LQGp+ZIXHlZcjpgU+g868prc\\\\\\\/dJ\\\\\\\/0FVRQEHe9RssAoRMoEzL96WXuMWj0VpVE93d\\\\\\\/ySK9frbAGWI4XpbN9dsYo1KiPKjlQDaFjzvOJkrjeqktB0zbbqBhKnwvpne3lzxBKeYwFCeHZ74XFHIc3w8P1rQMDWb0SSLaUgv\\\\\\\/rU7bcVYkdhSj6sJrpOH14zNKYbF+Rvcqr96Tc3LZhqJ5hNin6fROqdztoN59YVIjetM84brp1TCnM7992n\\\\\\\/kbhspVZZijybrbIF\\\\\\\/x+8s7yRPKCQdnKT6saz\\\\\\\/fL8IhBg2LuNtGsuvSqtBK9T0fjQTdZsJeiy2pm6n6GQOJrdDWpUuwiwDDEYtW+x+svpgvtVRwdfdpVqsqV5nFRjRxuCwlyUpmfmYX0ZxRZNob7E12\\\\\\\/wSkXeJFRwMsPFapHctNYC+ryGZI1oVv1ZKfXmD7I+6rIaEYwD4NwVxRVoChLmLfAgkdVX8uoGs2I7wE6EBH95r4u1Gskyk1xKLTbhZzfp48wfRlsPlKk6AKQ2faMPWziOJAVwMVz5e4lkHDuRt4sGfZd6DWytWQ4TgvEDmqpt6EOXyO7jfFPqXYLdrgo48FYXEbWQI64hPKrpZ5SLP1HdFniqNkPkvSCqKNs5KqCYD6mttHG1bhwDzpXP22YxwECkYk99EfSbkX9JS8AQPglOmvKh8lYWJvBjJGV+88swqdNYnjRhRBb2SHnuonvth2m4JGMrQDrxHxjYI6N+aRsq6Jy\\\\\\\/rfPdWWJ\\\\\\\/03fNQ7apBpZl4mm1dNuvdsm9UWcpdgVdnySxFa+6cyH4Om\\\\\\\/4AhrCwjX91kVFYD7WaN2xqeZMPmp589UtHB7E4xMExdWzeFHg5Sv8kJ2uo\\\\\\\/WrOb\\\\\\\/ylAs34RuG9K7w8MWGOsGoin2dz\\\\\\\/JtrXIHzqk8i5I82p8889O17k0Yf7OBpTFebUJJBNYy9mNmh1X3gmk4rxGiM+tQOyy5vK40eRJ525WT34J94w4XL1\\\\\\\/TaH7YvKpVbLf6unUr9QvKqqq1jod54WJaiuJTpV4nAMEwAgBknZ6exEPdFW39Y1jSofl+qlv01fwSHtY61px4Il\\\\\\\/0GoAAjf4EsyQLeCV\\\\\\\/7Ith7j3UftkcE1TbpZ0AUuIX66qVSiUiQH0b0jclnUVJR0zV2IEpJn4YrpQF+BpAGDRE+bEKtxtLHADa+Md3kEItDzansbswbqdsdXbWJE9he3zAppIysa2NR5RJIMOTSQlCJZ4tb6YkH9FgsavD+DJcCxYQkrr1fwauLYAcgSKtMECnhsgrY28q\\\\\\\/7MM45t+OsjG\\\\\\\/Mep7EVNrNrn03q4wKUCHlWpaiCG\\\\\\\/ZV1ZecaRiVHVBy0iB72+VJMnDeIjSq8L9DIaUKmkoTxCea2sh6mbbvgeSkKTRZyj\\\\\\\/W7zQb82zKl6VzaQvnLQlaFyDvRwMDEZ7tA1ynoZymjV9GeReMtkEXb4MyQGmCSnvWjTdp7og9nLkv7GEdltzwi\\\\\\\/93hUxRf1pVD1D3pQdhz+bn\\\\\\\/V9Uz\\\\\\\/KzXIy8\\\\\\\/9AwTqKxOtlsfC4Wdp9xLoqNS6dtzW5UN7MFFmjqtBzjKJ8Vi20V7ce8fDvi8kkjhUqr9BNSnpZEHgb4oEd1bUMC+\\\\\\\/5U\\\\\\\/4Mxlv3gz6cACQVVIjsg9y+vZCoW2Q75sNX0pB6jXl0anOZLy9OTXfKrpo6+yrhsGF2LwJGO25LVc1jdEAl96HfCURJD8P+kFF2VrSRgixLBjHLZuyqjsQbbEySkiOCSXaDtf0d614h2wZEmWdyjvdGj5e45TyrlAX+G3jdSuixlvRic9RiIXXkTIvE+cxyOeWsDJxuFdTKeq1r2Adxxj\\\\\\\/n99N84CcUg5eOTit66U\\\\\\\/yl2Af8XWWyFME95uzKNZPbv+LckNRPvDHum36+P1JEoH2Ek9w8rj9JNdaV0Z6wY3Jg02kj0vkb0GAM25J8SYE4G0PgWgRIgCdNFDVNP81C08M5Ek7pVXi+lECSpFv8j7bgAkdUkv9iwirHHwkZTlVBP10X3yb4VJ4AS8Xata6Qh8ByzFS0lorY4eATs3leXujRmoC5kniyNl04zje5BvhtL4tvqTubNNaldkrddaRgzhyrHGB42Z5z5wxBMY+inEsSzMC0sVeY\\\\\\\/G7gaQyizadzrEjDzrfpiZAMvVkRsdSZB93ZA+WpBUgzZy5lXjrJgjf12zaUxvVRWb32tjNVn9rorEotFpoYOlzdeudPBTdsvr3nHrYyM8Jaq\\\\\\\/ZARMoVH1JVV00BqsxVtkHGAcXm1UHYPYqUew06ObzsACuuHaw6tWduf7W50Asqzv9YYxxxFJ4wdf80LKei5FRLa14VgTnKpCOc8yVtv6RP\\\\\\\/4yWr9CDwmmI1YP9j9rtOLf1WHXgqRgj0RKv8tzbKf3RW3mLeS4Z8t6kQGPO9VRglWUODXl23A4M\\\\\\\/s9Kfco2KZNG2eSr9oDzDvkMJLH6FqwoNLR3BAa5ibstBFNej8HXNE9\\\\\\\/LanBZlGLL1q1ikGA99pzO0DvI2ouFxcz5zOPdG72qOT+GqAsyipbyR2p185x5lKizhS2tU+zr\\\\\\\/5es\\\\\\\/05NTAdOODgZcYyu1wLvBOSW5y7V8uxy4B0ibChhBN1I9d81j+m80FEKbfkMBs70dfSQdEb4vDpKOwe51CpNXYXrDnze\\\\\\\/1pXrpFav0\\\\\\\/BDH1ZARowclVX1MzVUy+0181BWcS628+JdNaep37rtPjIrtB9xteWPFhn7DP224XXu90x15gs\\\\\\\/GM5nWVrrfHIzt4\\\\\\\/+ap6UsJyVGjCX+eZlOHcS\\\\\\\/vl\\\\\\\/6iZlHCmOcnonlsrM0x4DijsIBcXXobXAfHtjT7lQqYiG8uYkl4\\\\\\\/5+2tc1HFztw6nXuIctDX6vDCF1g\\\\\\\/OtuJhEeeqUmPPhk6\\\\\\\/LYcwOAg+nPOTj5ECOpMCrNWct+6nUkDTYgnsM0DRY99pbPrReClgwKqzYPaeOeeF7aZORmOU4O7WyvLAKx6Ki7JJ8qEnUCeppssVe\\\\\\\/Jk1tWnTLqk\\\\\\\/GVLMn+R2hzFKxjdTtsuBUlHzLYsWB6Fuj4wpWZHG4va9smet1nwLqTw\\\\\\\/BSAMKOKQaqiLoC28PtEtW\\\\\\\/3XBccu\\\\\\\/Ewn8wa+LCMRW0KeEN3GOhTp6j02IG2g\\\\\\\/ebZ9ZiMSLsgoD4QRx3vjYeUc3Hic0oZoJS3uPjmxkGK+vQZGkZm1SxwI63Dcdd0pEMO+46\\\\\\\/4PmQNAFn1ZSHuCAcSUDrksIbAozc\\\\\\\/UceFhaliN5DzhijrxdbFN2CJzTk7XSELIewOfTxH3HupH5vLcX2kFjDrWF0N4UmH4LeE39AsxnaX9LjzP6bSo3emPuRrsVM6C+5VMBcpPu9LNheWdWmduLXrwbM+U4u7z1hEJ2Gi0i68jX1kZ3T75qJVFcZTSlNSBBGsbdC3JNL7euOCv9seTTkrPfcfPT6fRxCCviQ\\\\\\\/EFBJrn\\\\\\\/tZoGf6Z6pDhdWuoNlFwDtE\\\\\\\/E\\\\\\\/rQFqaQpEnV52wnHq3cZyaDmoZDIUXULM5A6n6vfPTQt5HTSqfaX\\\\\\\/Y20txjxTpNj7+fztD1ix7Fh9mqI+RDr2XnjKqpHbWqbgzFYOb8ilFqOIPtQfTOOUrsNEDRHw9f3kWzfv3+TCH+HrTXeo7AOUwcSRu1ngtCFtxIKzJQPURVNScvQOPAdvhRd\\\\\\\/cjFKPxYMxrWblRD719F4hO5Z3tTddyA0yoLvJwkz4JQUc5EOsdkILBsWXNZzaTiWTYxPvoUFvSVhTYI2Gwl8qeljpGKIxKlet1T+durzQRk0P7dkX3TbcHnP8Y1BVuqoKlaGmRcIgFzkdof873bqlGzZ8ZDVfJ6QD9E5YKe7j+pQJl7gzsM4GNJXRWTtH7\\\\\\\/zihHOA24cyxLoKNA6JNaRsQE8EfmlsPRzvap6J9yX40A3U7qQIb8xgUYqP+sAYj0f33wRNkTp9C\\\\\\\/DO4dnHG7FZVgreWRcu3n\\\\\\\/StOBPQCIy\\\\\\\/aleRGSv4NBCHaBUrpw0vHoK4VS98omr0zCgdD9dFz2hepTqk+u66WIFXQe5d3UfqXEtNdKL5MBqqUQ112iKNuaO1Zorgo2ubIaBOnDql6jCeqg2GHdI2eLsv5E6xzObv7z84UiEYq4rawqmknv9XNg7y1VPE\\\\\\\/UY9J88hec\\\\\\\/BlObod5trgluFcpiTw28ByVSnF+zyP\\\\\\\/SfJXIfzeLIXnX\\\\\\\/t4t7SjHa8RPtE9NtYXSAl4QcadQh7LshGXjWzxmJC2ypziPGRcHRKz6qm4tQlgy6Mcus8gZcf60KMw3k\\\\\\\/f8d3+8mvzzevZlBwROCfV6PgEUz2127wBSI+h9Mp+p++dzwMtIMRbT4AUwkrce+xwdeB2mhTjkbca5EwmHZmyJ\\\\\\\/A4xBg83p8MKL4P\\\\\\\/0chnHq+idXpKgjB\\\\\\\/syfS0wiBRMU+eDlVJJ\\\\\\\/mpQXF9zid4aYh4vYO1As8NyBAlhRJPA06BQAq8dNunZ3BibEKo6+eePZHI1yMmfW2H\\\\\\\/Q4ZzKurk+izXJq\\\\\\\/W5jomRDdZ2MSZIWYtW8GUAakSn+UEFlFubh1+J1NpEAwF+Hw+kt9iUVB2AvE4\\\\\\\/vh1No7GAgdQKJPLVzEpxpcr1Ud1k6Xf4l\\\\\\\/AoTlFICa1Zzibpep240FZMiGG3Y33k1WK4Fu8cUAMGH2Ub9V0TnG4eZhAnmbpOHol8flcLlr8oba38x5zNTMcLGWROSMHyUa9kdVJlENc8yQ8r+pP5FqjTmLhtbvMLMSJ8N5q2INRFFUoQ27r3x3Fg+Q4F+Gb4RiUfQ+qJRbyJwg+tszLqU5v0UcJI5sMBKjaB5azFQszfFh8B5et9pHzSxk7U1gbi0npcyRMhFU9q8dEczn2ADREMbn\\\\\\\/cPVx8EAcPFmnhnWik3MjUvkV\\\\\\\/DLIASl\\\\\\\/wX53O9WYuhAd1MvnfJ0lWGARLBDQDSH5cGSB37stQjFQG4iaWZ3FQCYG24auqDWEOaqatK92zq6SXrVdsRZV5RHtcaTgYNeRKBw\\\\\\\/Zyh2ZGKzv4Ndx\\\\\\\/I+pAuFvRBKR3F8dmhrELa33S2TLlbUviwBkXY8k2f+guXM8glGOQQ2cU4IN23hORfY9TLEKzD8sihaZj2a+i2DLBq2D0lYpQi+PtaKAM9kijBDzfY6uY0WvXzI1YgIizUXpkXnwdZ+AZXVpQIz\\\\\\\/G7OFdFVPBxAfoQio4m2Si1caakZJtkqXYrVKgCFvviA0qKuyHk4D1zTTYKQj\\\\\\\/YkadWW+\\\\\\\/3X4toRkaD6t9MQD++QkLeOUz8EFGuSYFFZk0WkMNLTxzFAhhpOhLAyv5TPs0OYJ0Xm3Q9ypn2TlnmY04D9ir2Y6zYtrfWLR5VQhn5WT+ZPlYZlb9m9e4J4LxvejAOSn8U6AXHuJ9z9NQHPAAD2TLz3wvsd82dukxabkrejrm5B2lz59dV0mDZhVadga64\\\\\\\/rWBpYpS2aAgcHuz2pu+AcZikYDDWi7hmgQQa3zYszXtgxdI942BDegmDsS3Zx2IMT3ssfPvi6M6L1HozTkw6RuD5ZsJYofkKO0WDUvzyiIuE3oXlUoinJ2TzVGPkmzlBpRh1inZ1GPBFuB102faSdDF34GVxSGeXdIDRGKGjmaNZ9v66S\\\\\\\/bb5GK6IX8ddmP367rj23lf\\\\\\\/qyMeJJMA6szKGg1nflXTuFGaIhAmRIrtI4OXnPmCBwEIdYc39ErCaGwLB+oNH+16CoR0hehHvL1kLdZHsGlzQY7H5gzfGmxL1w9kLLRYkI05LWlP9zmO+zqQMe7GR7GH3deqnThPdn4Ok6NyZsNNqCPi0sYFIS0BNunHLdURaBKAERlOpCbuUDp2L8DBnv\\\\\\\/X83gk27FMSEvXcHY3yTdkVV24zkvZnX1+AZGfhKtnQO3CKqGZA\\\\\\\/LhHWOHi0Bls7Xa8++06qq1Kqe4\\\\\\\/nBwuP2AxKrFsbihEsvEgxban4zRqcSZMD8dOUHsBjijU0skG2Sa\\\\\\\/AEZjEUmLBRtTu3GnPmXZsScaCcHw+wc7zP6HRiHBq4RzWZvP06Tmr1hz3YdczmX++7yZGdN+FuGAl+ZXRTJT2zdsVV8claSHrwQcKDoYjO3KHYC5Z0JZ7hjWLVFo+Mq104CzfYUrvNp6KOuprsZKS08ul+v9LIAauXr00+Yfezc6KJl9kyp5RUXnRqvdKgx5a4\\\\\\\/HvSkgITV6wb1571dmGENAJPvU7XBakcdwGolvw+o26AJmDLQzXg1+YqfAlt66HAuHRQ6tFRsPIIMXS7nDlLgWQ102jBdbE+kl+nCkzMdenn9FOSaTrEBasEKDPgJEpRUOUszzbs23KrqC97UrFEEsKjbzXtMPPORkOEXQpCfLkU651pA+9uCcHZ7VlZtGKp5pYSzZhwSZMkTW55SspPbWOGzroJB4fA8\\\\\\\/cNwHcsR9O0Zm\\\\\\\/xX1+yWNTBii0xcSI0XiYBGFYAjRU+32kxZfvfrKeBzL7kU0TXCja1vvgCCO\\\\\\\/PWSUdJQLfUkYC60RYUp12+630uVyUvxE9DJNPc5WT1zcWUiS+j0Bv2pl0ua2MZ3XxJ5b3MElnwo0a\\\\\\\/09oQHuIBJQjUKT1+rOykj4GuCiz+f7F\\\\\\\/NUDAZtZZNt1P6R2nKZdZVdSBeYWOwQf+LGWQs\\\\\\\/vcbbx9\\\\\\\/sEpCIHZlOSFEAm5v1y9jDo\\\\\\\/wQNeZ4BUOi3j+UQWUFwgZYUkTtEZ2oWd4Z\\\\\\\/ABc9G7YohLT7VO5pzAoan4Xecwg8CaAOptP91k4gFtQuZ8H+l+L3CgSoQphxXLTj9ppx0N6uMGAP3kuVNR+TUbdLhSAPAzUIzCp+iHg2Sb6F1Ggf6ZP4gtwtyzA56k3FcTN7e3vdT7dPuy9GGC5Je2+DX1lhXxQdNgABC19yiEBl+bRNkfgWD7rZf8HTVtJPaFN2G+fb0gpyV1BjX+wtqEL00AfXw4LMQAFzA80Sakk1v1GSaVPL37oU6SWy+OWP03Rp+Cz412co0LsKxl+z6i99iEdp+Qr7V1\\\\\\\/rZUcTzoPdgYMol+OW\\\\\\\/l3f7TyWP7j3sitTT9oACyTgL+h\\\\\\\/Ird7tVSOWFUo5ja5ftoxRsdq7B25uW1eLVoIpmD3YOzgLxS6sPhzyKRL5pZ+Kkm47TWHLgOt3KNlk0F1Vw0h53AVNWB6bLza2VD6h5TtUb41vvKV81MkOUhnzjpovBh0oYwKvoCmUGO1bz1Kv562dIK42uIW+3g78dKDHcKm5rfBCThHjwbCkchjSy69tu9rl7ECkb+L7oA3wELEr3e50TicsACjbobrJszvInb0J+6Rh28xuqXTqKuVYQF4SjHznJOMOT6IwjHBGz9SKLNj7OVZifLLA\\\\\\\/tjtSqUdEr07Bd2oYi0muVWFSzDoyP6nRTnnLk94xUtzkQG5nICyMKDnc2dw6jzwXYQCto8SenGWGIjm0m8oQEdBSPaKGP22YxWb3kqWfzPCGcpVYZSkkLfUS1hoFtigM8mICaZRanA78YdDXVWUpjM3Q087itl85SbRMon8iFGHhHXUnILNTqIGEiaKH1jEYkkbb6id1hrpTT15VQ6hvuRqse\\\\\\\/oGTdQCFi3JDGURNGydTOMghjdKNBlcGCMGv8hzzU8oMs4G\\\\\\\/0SNdL3HOINZ+r8VU2erEuTBDpwoV12X\\\\\\\/AlJbuXP3duyrYdhMzsaUTktIKAA2GfPJ0evUDAx2QnzOVKru9hyxOodCcc6BpkxxB\\\\\\\/bgobrSjqqrkXC+rvjcRUHFmnIMZ3GEAysUQoJU3koxP0akVmZL0alnw2r2BbjxWDCeBdJj8f6G1iwTNJhregpThhetSplP2WHF017O3jmgf\\\\\\\/7H11okrGT30jEUQBa1N26CDMLAGK+uBFiwjK2Gjgld8FMXBQPT1nCjP+ucFT5oIovny9fwUeM0HYeK4PsoFS30akIOzcuuoc\\\\\\\/YBGgsOPZeDPx2Xgn58SC\\\\\\\/LWS0RiXEIfLq+CDaStghudW6ghrelVSwc5SrV4q6dmxEwjqVIvoUcflP7yQ1lK+EcAoq5ulz2oMIlqvX2xHaDhb6\\\\\\\/GpnM7wn1sIGMYxC2i\\\\\\\/KFVvHRNc93H1bC+734oax4Qa33Z2hAlmcYb2ytMAAHdr0EAPBliLlE04JFXX0dOvTF9g47s4rcBKVn6a+\\\\\\\/eFdr9\\\\\\\/eac9TZoNwZRrZnimgXrC2BtID9x\\\\\\\/js7g+vS2Ho7iQqVnRK5HbC7DoBl4nSh623genIHZhJe3AY8t\\\\\\\/6LodTXRnwep1SlkXQ\\\\\\\/IAfAsVM5Kya8lK7K+e2\\\\\\\/AhpyoVfdrRgjfl1pvgf+RFNW3QPGYOfyU3mLoGUxcLE6Q63gqPMQd4YWsQ1bkVVFhvRg6DGx2RIMMR30R+rRGrihYzxX8IkVUANt4Oi9Au1rBCyurRmTUhER5aPsVDhCpjyte5MqRxIKRvGqfvTVWkj+HW2RAgR8+8dBbcgIvMH07Q53axODGoHF1bVOw2YUgivgcP88u1z2tdpFNoCLjUqDVFBLKb52+FDyOV17VYOImPrTfg0ppEYzoIyAPrbvwOzvm1vaTPFnbvYhzb1tjjPWDQjxX5Nf5mL00C16MngmSwYwcP9QeRxYDosXUMuNKhiQeh+I0vwSo8ffgGJ6ELJ7khgr4oFavAvx6rbVSvFkfclQ\\\\\\\/rNXLFYjfPewSc1EXjs+GS5O6bGDMY+RWejYvyVNwktAmv\\\\\\\/s2C5XL9w5GfgsY33GZyeCrsj5BCuxGeRpVIP7rhpNDlRkxqBmyltZ8zEAHlFPNpAD7ThM1iuRYHD1d3Dbtzrv8sPRO\\\\\\\/O1xYbC36telFdA9QSCzcqmVs0ViyYFMAgwMUoqkWZMfslssnwc0\\\\\\\/CcUsiEgjDpZ\\\\\\\/dCMk0DyVw3xqjloAr3vsVL429jj93Macpz6L1C6ykpkuej+0YTuhCs1fKbEeZRlmcn+XXQoCho28uNbZ230H8NBsipt9Hx4Zt71rKRCHMQ8wGNxoceg06PLgOa8k2QbsJS88ND3N+uO4k++SF0xUPgLm23mZ5ASjdGU4Z2UfMjsKk0aA0iHE0cCwhAc9w8L598FBlxboiw\\\\\\\/vNXCXykKLDTvc1fuqwtPrquLFBp5WBZ56H4CuDwhVIAl4wkIPN4raED90+Hvf+JxXn+zMQ5Vuj+iZOsYzrXxeM0neQ1d\\\\\\\/hETHr+6TbanNoTL3vlBIFL9cUgNcSvysKBasLxvrE9mKTlGvEf1PTEYAvjjFrkhOyyh+tUvj8NYBNxPjsLV78+RJUB3t9HZgdSGiT98hEs85RQTeVzir9ciyMg0nPuVDBDL36VumhcQcXPt\\\\\\\/X5\\\\\\\/GXoLLCrQ74VJPjQfTcVw8Te0lzvn0S9mMWDs6SI+\\\\\\\/ZZQjaQEKqb9eNcezfAke3EBQoaxy0KLHDFplmKB9ypArP4xAkJXbVo7Wc\\\\\\\/jAg\\\\\\\/r\\\\\\\/WzJ4E6RAuQlwaN8M7LmZps29V2TTNJ9at\\\\\\\/WUloZNq0Sayc6GIVHPSLJto2s2aSN56ujUJrB9xiFvCADI0BKTtihCITDQqDpJ45ebLEfd\\\\\\\/rIKJUn652vfH9iykwjt0NKHZIcge3VPUtrc9K8193zIBKPdFX3hgJDJADw5yKmhfpqpy7OPXidY3G9d9F70dFQxFjt1dPmAV6lndnPMkY0Zpz7m8qyZWw\\\\\\\/VAHnQzSAQH1BOJmfBLfOfco8EDqL9ZTi5fbF1I88Cuh+LGXicE46d\\\\\\\/BU6xlRLsLdA6\\\\\\\/lGGQ6Y0GlmtOGaaCY9WX1B7oRee+jkNbyoe8PDH+N+DT25\\\\\\\/6HE0K1RJEZ0LDCpGWn9iA4+dQxLB+wvO9aYnDFA\\\\\\\/BtjfOGb7RECEhsxGUJbR9du+yVd70mGEH2TwODDEt35BUBVvAzV56FVruliTR74hn+vCy\\\\\\\/iaAQgWPP8b1DJFP8ogZDWAXMHsJm5IJ4LvN4\\\\\\\/GgYSvaqC1tRdLGI7jBi7SKWK+zYCFDZfEIW+RCUxRIGfLUcNCVpt\\\\\\\/Nzt+i4Bx2qvKT1YBNpSIabF4OPhEEq6ZrYRFSGpU+nqu+BzgR5eisCaI\\\\\\\/XXU7Q67m+BPONLwTFU0iZ6dVF\\\\\\\/ictSk9GbDBG4G6qi01w5EWIivOzu6OvvfDrFrtsbJUy9AXxvC05ZcBJF3ciYxJs5PH1P5kIwos\\\\\\\/Y3pauyI4IrljsEKT0HdmS7EWb1R4In+z88JumBWt6ZdlLBIjXWb3sJKaCK9KCb2bhghTNfVZycF8V8R2a+uy6TQ44GpMJ+QjB\\\\\\\/1n9PK\\\\\\\/rND+kZjCKBBAM45N\\\\\\\/6Srnmfw6fBryqNziJMN+Vv5bSVbtmhxXWxPExO7dCoXZ+KIjSoz\\\\\\\/3HISC7zB1\\\\\\\/3bm3EAW6o3t3Wv1HuXXS35hCAFpbmzJ7S3E3QNwCoOHVi7jhzj3tu9fGhY1hBpt4NKJAyWDOhd6ISaYAuWcg1Px\\\\\\\/3Tj1ANRusdPcCPTyKPtyeGQ6sfj2XXfUlanzcF3gmUtO8Nf6vhB5q9qGaPiRfZmc7UDxCTRDp5Rs3PvtdJJ5oiH4qEgY1+dW9E3gUpnL9XkSSwgjfEPX\\\\\\\/3zi\\\\\\\/vQ\\\\\\\/mMJ1Pd2XF1vjfBzuPsnJMLZo5Sv16G9+xmwzEEejQ2s4r\\\\\\\/MZ8aglj44bFYo2yr3Y\\\\\\\/yL\\\u0022,\\\u0022iv\\\u0022:\\\u0022xAeQOyD8LKizuSJstObWSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228b795604fa7deef44433a5e531a512138815f09300bf13b94abdb4eacfc92160\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022W+NGs3uMbrmBq\\\\\\\/K3\\\\\\\/SRP9\\\\\\\/UAB\\\\\\\/Zxh3efaPWEslJ\\\\\\\/cV4mbT5qPlfKNvUE01Hxu3b+jOMbl2mZD60nDglfsG6BHzepceYRfXfnn4ey\\\\\\\/Snjz3VztqGZs7RiEgLyVM3or\\\\\\\/O2XmA2S4NG3xUtulLbcJbwMqEun5I7yKnTIerP2mFjmFjBzttPOwHrP3\\\\\\\/+DIras8eRj+0tIdPvD1JsrtlQffw7eaBtwnzSl7Arbh2YK50yyij\\\\\\\/dMZVWqJpsQy6oyNryC1o+z2HdM\\\\\\\/11BWJ78U47\\\\\\\/t2odHEnptS0mCGJfRMmYVjdjSLaSh2kOMD\\\\\\\/ExeScsTkXeJMTc\\\\\\\/3Jcwln+deQzcZqp4Of4VQDKoblbk4laPT5nF1FAoh0jUDTVeoJ5xprmc\\\\\\\/+91zw9HQqGCS0SD9rfj9HkB\\\\\\\/EFiFbTnox6t9ZZGKOBjUfqmZ7LflgTV5+jB87j1VRSOFB+7uFsQMdnzm\\\\\\\/pWbG4WGqEywuY5zgBkuD2sQpo+IeoGvAHxeEtmz2\\\\\\\/JJfLm\\\\\\\/pB074sKgQlCErgKpbQiv\\\\\\\/DCapw9Q9bxT7N3t45eC2OtOnKyDOLisxyL2H9nQwjbkzK4ppHmhlCoSJ8YlVErknK3pTBEy98tLJd3e94xxO9x8Ky4MoGaq\\\\\\\/qSL1SUDvHTKHCPUOof3yEEVtp6ZsuqpT1+B+W9MY8Y5KANqGDjndK+ezebs2nmccsx3IamWrx8M+WN033tPJBqQoT9D1rhkK393oug3OtpZNYgQNp7wWr9j1gB9zLourhOSr0vROGCRDZzxCJr5VI4tigtULLJY+A9pcMEJNShPUBeZmnZfyZPMeLsOigtAxZFwBOy6awDuruBOgjyyn\\\\\\\/zBKoPzKpfLXheFkfoLxFqaCMV0f\\\\\\\/9qfUVy7g9rbLSjpP9d6DTybSzjVHvTIdekB+zult2b+1YAwO74be0pNWNmClSYn3bZC3E0CKp7Awl5QsCYwJhpvkTWGZih\\\\\\\/bzvZodBIi0UdrCKFxGlG4Q4qcmVXrbsYCLy287drbHuca7Lu5M5ExiImhH2Y7uQcrIRfThiR3yfGhT2Wqq4LdO\\\\\\\/uK4VVmOzYT58ahX507Na+W+xkP8a1TKt0KZ67u1yca5PxEt4zVpILqAohAjsCoYustkzpuNeGdd0mjD4O0UJJ9A6W5DQkLS4m2gml+q8TxJto+XOwqMCE2ZFxMPPBtiyQwf6JPsCXE8f6Jbhp5J+4Rqzos9\\\\\\\/YJ\\\\\\\/vgxyR2Wj2SHhnkZgbIWiZZZTRDM7AUx7yXYhQlU4OACNmoSyckYrJv52WMSrJ905E0N7GiCtLPs9MCvE4iAsw7q7sUqg3kFproCc4GEzTa3FzJR1+H4O3vH5GtSAdrEPS0pm00PlAGuysB14E7upW+KrU8Cgsb0aGCt1wTuFq8HVJ\\\\\\\/0hGtu2Xt5vNBOoi0tGdrbJGs7IYbwUZXR9kdO1DxnXPi7Lr3rNwAfQ7x6Pm9P6+2nbXW7F39sv9SeskMPdQJ7gwMoO5zyT6YUTXD2\\\\\\\/3xbieyjY7+yYoAFIUqlFXupPVPYJKfiLvIDF4w0xzAsltw6CNcwYil0\\\\\\\/YS1F+MVoKyJYqjml27EYAX4BdTHlDJrZj8PVeF7nrIJq93sDWN5yrdlpK3rTkarojpwX79Fth3EvaimT69VnqEPoi+jQ4R0JyCW7\\\\\\\/xulPYFAxolJzNCRm3n28aqqo28RzUG\\\\\\\/OWC5M2Lr8izmUWq+mLTHa1MgUFvDHS+NzsQCVrLbUQ5O129SjeVDYWb4nLvRd+kwh5oQL\\\\\\\/GVfCWDbpOnOAbVITQqylgBM85SQV++9Nu+bmFFyxaRqfhP2W7P+b6jyiJ\\\\\\\/UF53NZYtwJKbHLCfrbEgdvgC27+GKjvT6BtzNyajX6YoNBpvRfuvoWKuUQlgH5R98iQCK1VhTSCopYYdEKusSuyho6sYzmq7bZtj7bzYVSyqVLd3c+dokpWTo74j5Vg9fVOrxF15+1bTTRJs1x89HRJaiUb9un+hPkv92ovJg7EMZtfsT3ZmLmEXHN5zGvn6IPNlFvrkY6KkoNKpMByUqVxHH1N5QyXS6dw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022qYtQ9fUNIF4zNTjMSb0iDQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d118ae614c898a18d0ba8324fa8905716fcdcb9fc8154e9eed3e7a0501c60fc7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459965 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1263" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YLi-r2ZBAZAYV-fwOYD4qM-bqGpJPMcHxDDfHfOHrrs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vfnjSRK-YCL-rOCInyYF7IGrWWWN7y2_fJmdAKWLo3o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1253" title="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1273" title="A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459965" id="node459965" data-pisa="ggg;5/6/1263" data-pisa-master="ggg;g3.115.017616" data-apath="/ggg/5/6/1263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Neurospora</em> Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dev Ashish Giri</span>, <span class="highwire-citation-author" data-delta="1">S. Rekha</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Durgadas P. Kasbekar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1263-1272;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017616 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459965" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dev Ashish Giri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Studies, Manipal University, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BAshish%2BGiri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giri%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BAshish%2BGiri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">S. Rekha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BRekha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rekha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BRekha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Durgadas P. Kasbekar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Durgadas%2BP.%2BKasbekar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasbekar%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADurgadas%2BP.%2BKasbekar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Durgadas P. Kasbekar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kas@cdfd.org.in" class="" data-icon-position="" data-hide-link-title="0">kas@cdfd.org.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1263" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459965" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459965/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1263.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459965" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459965/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1263.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459965" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459965/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1263.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459965" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459965/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1263.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459965" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459965/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Introgression is the transfer of genes or genomic regions from one species into another via hybridization and back-crosses. We have introgressed four translocations (<em>EB4</em>, <em>IBj5</em>, <em>UK14-1</em>, and <em>B362i</em>) from <em>Neurospora crassa</em> into <em>N. tetrasperma</em>. This enabled us to construct two general types of heterokaryons with <em>mat-A</em> and <em>mat-a</em> nuclei of different genotypes: one type is [<em>T</em> + <em>N</em>] (with one translocation nucleus and one normal sequence nucleus), and the other is [<em>Dp</em> + <em>Df</em>] (with one nucleus carrying a duplication of the translocation region and the other being deleted for the translocation region). Self-crossing these heterokaryons again produced [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] progeny. From conidia (vegetative spores) produced by the heterokaryotic mycelia, we obtained self-fertile (heterokaryotic) and self-sterile (homokaryotic) derivative strains. [<em>T</em> + <em>N</em>] heterokaryons produced homokaryotic conidial derivatives of both mating types, but [<em>Dp</em> + <em>Df</em>] heterokaryons produced viable conidial homokaryons of only the mating type of the <em>Dp</em> nucleus. All four [<em>T</em> + <em>N</em>] heterokaryons and three [<em>Dp</em> + <em>Df</em>] heterokaryons produced both self-sterile and self-fertile conidial derivatives, but the [<em>Dp(B362i)</em> + <em>Df(B362i)</em>] heterokaryons produced only self-sterile ones. Conceivably, the <em>Df(B362i)</em> nuclei may be deleted for a nucleus-limited gene required for efficient mitosis or nuclear division, and whose deficit is not complemented by the neighboring <em>Dp(B362i)</em> nuclei. A cross involving <em>Dp(EB4)</em> showed repeat-induced point mutation (RIP). Because RIP can occur in self-crosses of [<em>Dp</em> + <em>Df</em>] but not [<em>T</em> + <em>N</em>] heterokaryons, RIP alteration of a translocated segment would depend on the relative numbers of [<em>Dp</em> + <em>Df</em>] <em>vs.</em> [<em>T</em> + <em>N</em>] ancestors.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromosome-translocation" class="hw-term hw-article-keyword hw-article-keyword-chromosome-translocation" rel="nofollow">chromosome translocation</a></li><li class="kwd"><a href="/keyword/introgression" class="hw-term hw-article-keyword hw-article-keyword-introgression" rel="nofollow">introgression</a></li><li class="kwd"><a href="/keyword/pseudohomothallism" class="hw-term hw-article-keyword hw-article-keyword-pseudohomothallism" rel="nofollow">pseudohomothallism</a></li><li class="kwd"><a href="/keyword/rip" class="hw-term hw-article-keyword hw-article-keyword-rip" rel="nofollow">RIP</a></li><li class="kwd"><a href="/keyword/msud" class="hw-term hw-article-keyword hw-article-keyword-msud" rel="nofollow">MSUD</a></li></ul><p id="p-3"><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Perkins (1997)</a> described three elementary chromosome translocation (<em>T</em>) types in <em>Neurospora crassa</em>, namely, insertional (<em>IT</em>), quasiterminal (<em>QT</em>), and reciprocal (<em>RT</em>). <em>IT</em>s transfer a segment of a donor chromosome into a recipient chromosome without any reciprocal exchange (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); <em>QT</em>s transfer a distal segment of a donor chromosome to the tip of a recipient chromosome, distal to any essential gene, and presumably the donor chromosome breakpoint is capped with the tip from the recipient chromosome; and <em>RT</em>s reciprocally interchange the terminal segments of two chromosomes. Other chromosome rearrangements are essentially variants of these, <em>e.g.</em>, an intrachromosomal transposition (<em>Tp</em>) is an <em>IT</em> in which the same chromosome is both donor and recipient, an inversion (<em>In</em>) is a <em>Tp</em> in which a chromosome segment is re-inserted in opposite orientation into the site from which it was derived, and there are complex rearrangements such as linked <em>RT</em> and <em>IT</em>. Three breakpoint junctions define an <em>IT</em>: junction A created by the deletion on the donor chromosome, and junctions B and C (proximal and distal), created by the insertion into the recipient chromosome; however, two breakpoint junctions define a <em>QT</em> or <em>RT</em>: junction A, between the breakpoint-proximal segment on the donor chromosome and the tip from the recipient chromosome, and junction B, between the breakpoint-proximal sequence on the recipient chromosome and the donor segment grafted onto it (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Singh <em>et al.</em> 2010</a>). In the cross of a translocation strain with normal sequence, the chromosomes can segregate in one of two ways in meiosis I: alternate or adjacent 1 (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> for <em>IT</em> × <em>N</em>). Alternate segregation produces eight parental-type ascospores (<em>i.e.</em>, 4 <em>IT</em> + 4 <em>N</em>), whereas adjacent 1 segregation produces eight nonparental ascospores, namely, four viable ascospores containing a duplication (<em>Dp</em>) of the translocation segment and four inviable ones with the complementary deficiency (<em>Df</em>) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Viable ascospores blacken (B), whereas inviable ones remain white (W). Therefore, alternate and adjacent 1 segregation produce, respectively, 8B:0W and 4B:4W asci (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Perkins 1997</a>). Because both segregations are equally likely, <em>IT</em> × <em>N</em> and <em>QT</em> × <em>N</em> crosses are characterized by 8B:0W = 4B:4W, whereas isosequential crosses (<em>i.e.</em>, <em>N</em> × <em>N</em> or <em>T</em> × <em>T</em>) produce mostly 8B:0W asci (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Perkins 1997</a>). In an <em>RT</em>, the two chromosomes that underwent reciprocal interchange of their terminal segments can be designated as <em>T<sup>1</sup></em> and <em>T<sup>2</sup></em> and their normal sequence homologs can be designated as <em>N<sup>1</sup></em> and <em>N<sup>2</sup></em>. In an <em>RT</em> × <em>N</em> cross, alternate segregation moves <em>T<sup>1</sup></em> and <em>T<sup>2</sup></em> to one spindle pole and <em>N<sup>1</sup></em> and <em>N<sup>2</sup></em> to the other to produce eight parental-type ascospores (<em>i.e.</em>, 4 <em>RT</em> + 4 <em>N</em>) that are viable and black. Adjacent 1 segregation moves <em>N<sup>1</sup></em> and <em>T<sup>2</sup></em> to one pole and <em>N<sup>2</sup></em> and <em>T<sup>1</sup></em> to the other to generate only inviable white ascospores bearing complementary duplications and deficiencies (<em>i.e.</em>, <em>Dp2</em>/<em>Df1</em> and <em>Dp1</em>/<em>Df2</em>), and the asci are 0B:8W. In other words, obtaining 8B:0W = 0B:8W signals an <em>RT</em> × <em>N</em>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1263/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Alternate and adjacent 1 segregation in IT &#xD7; N. The donor and recipient chromosomes of the IT are designated as TD and TR and their normal sequence homologs are designated as ND and NR. In alternate segregation (ALT), TD and TR segregate to one spindle pole, and ND and NR segregate to the other. Subsequently, meiosis II and postmeiotic mitosis generate eight parental-type nuclei, namely, four IT + four N. In adjacent 1 segregation (ADJ), ND and TR segregate to one pole and TD and NR segregate to the other to ultimately produce eight nonparental nuclei: four Dp + four Df." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-81004736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Alternate and adjacent 1 segregation in &lt;em&gt;IT&lt;/em&gt; &#xD7; &lt;em&gt;N&lt;/em&gt;. The donor and recipient chromosomes of the &lt;em&gt;IT&lt;/em&gt; are designated as &lt;em&gt;T&lt;sup&gt;D&lt;/sup&gt;&lt;/em&gt; and &lt;em&gt;T&lt;sup&gt;R&lt;/sup&gt;&lt;/em&gt; and their normal sequence homologs are designated as &lt;em&gt;N&lt;sup&gt;D&lt;/sup&gt;&lt;/em&gt; and &lt;em&gt;N&lt;sup&gt;R&lt;/sup&gt;&lt;/em&gt;. In alternate segregation (ALT), &lt;em&gt;T&lt;sup&gt;D&lt;/sup&gt;&lt;/em&gt; and &lt;em&gt;T&lt;sup&gt;R&lt;/sup&gt;&lt;/em&gt; segregate to one spindle pole, and &lt;em&gt;N&lt;sup&gt;D&lt;/sup&gt;&lt;/em&gt; and &lt;em&gt;N&lt;sup&gt;R&lt;/sup&gt;&lt;/em&gt; segregate to the other. Subsequently, meiosis II and postmeiotic mitosis generate eight parental-type nuclei, namely, four &lt;em&gt;IT&lt;/em&gt; + four &lt;em&gt;N&lt;/em&gt;. In adjacent 1 segregation (ADJ), &lt;em&gt;N&lt;sup&gt;D&lt;/sup&gt;&lt;/em&gt; and &lt;em&gt;T&lt;sup&gt;R&lt;/sup&gt;&lt;/em&gt; segregate to one pole and &lt;em&gt;T&lt;sup&gt;D&lt;/sup&gt;&lt;/em&gt; and &lt;em&gt;N&lt;sup&gt;R&lt;/sup&gt;&lt;/em&gt; segregate to the other to ultimately produce eight nonparental nuclei: four &lt;em&gt;Dp&lt;/em&gt; + four &lt;em&gt;Df&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1263/F1.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1263/F1.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1263/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1263/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466178" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-4" class="first-child">Alternate and adjacent 1 segregation in <em>IT</em> × <em>N</em>. The donor and recipient chromosomes of the <em>IT</em> are designated as <em>T<sup>D</sup></em> and <em>T<sup>R</sup></em> and their normal sequence homologs are designated as <em>N<sup>D</sup></em> and <em>N<sup>R</sup></em>. In alternate segregation (ALT), <em>T<sup>D</sup></em> and <em>T<sup>R</sup></em> segregate to one spindle pole, and <em>N<sup>D</sup></em> and <em>N<sup>R</sup></em> segregate to the other. Subsequently, meiosis II and postmeiotic mitosis generate eight parental-type nuclei, namely, four <em>IT</em> + four <em>N</em>. In adjacent 1 segregation (ADJ), <em>N<sup>D</sup></em> and <em>T<sup>R</sup></em> segregate to one pole and <em>T<sup>D</sup></em> and <em>N<sup>R</sup></em> segregate to the other to ultimately produce eight nonparental nuclei: four <em>Dp</em> + four <em>Df</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-5"><em>Dp</em> strains (<em>i.e.</em>, the viable segregants from 4B:4W asci) are recognizable by the characteristic barren phenotype of <em>Dp</em> × <em>N</em> crosses, wherein normal-looking perithecia are made, but only a few exceptional ascospores are produced (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Perkins 1997</a>). Barrenness is caused by meiotic silencing by unpaired DNA (MSUD), an RNAi-mediated process that eliminates the transcripts of any gene not properly paired with a homolog during meiosis (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Shiu <em>et al.</em> 2001</a>). Presumably, <em>Dp</em>-borne genes, including those underlying ascus and ascospore development, fail to be properly paired in a <em>Dp</em> × <em>N</em> cross, and their silencing by MSUD renders the cross barren. The breakpoint junctions of several <em>IT</em>s, <em>QT</em>s, and <em>RT</em>s were defined in our laboratory (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Singh 2010</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Singh <em>et al.</em> 2010</a>). PCR with breakpoint junction-specific primers can now be used to distinguish the <em>Dp</em> progeny from their <em>T</em> and <em>N</em> siblings. <em>IT</em> progeny contain all three breakpoints (A, B, and C), <em>Dp</em> progeny contain B and C, but not A, and <em>N</em> progeny contain none. Although <em>Dp</em>s have been extensively studied (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Perkins 1997</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kasbekar 2013</a>), the use of <em>Df</em>s was limited to flagging the <em>Dp</em>-bearing 4B:4W asci. We now report the generation of [<em>Dp</em> + <em>Df</em>] heterokaryons with complementing duplications and deficiencies in their constituent nuclei. They were obtained by introgressing <em>N. crassa IT</em>s (and a <em>QT</em>) into <em>N. tetrasperma</em>. Introgression is the transfer of genes or genomic regions from one species into another (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rieger <em>et al.</em> 1991</a>).</p><p id="p-6">Eight ascospores form per ascus in <em>N. crassa</em>, whereas four are formed in <em>N. tetrasperma</em>. In both species the parental <em>mat A</em> and <em>mat a</em> nuclei fuse in the ascogenous cell to produce a diploid zygote nucleus that immediately undergoes meiosis and a postmeiotic mitosis to generate eight haploid nuclei (four <em>mat A</em> + four <em>mat a</em>). In <em>N. crassa</em> the nuclei are partitioned into the eight initially uninucleate ascospores (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Raju 1980</a>). In contrast, <em>N. tetrasperma</em> ascospores are initially binucleate, receiving a pair of nonsister nuclei (one <em>mat A</em> + one <em>mat a</em>) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Raju and Perkins 1994</a>). Thus, <em>N. crassa</em> ascospores produce homokaryotic mycelia that are either <em>mat A</em> or <em>mat a</em> in mating type, whereas <em>N. tetrasperma</em> ascospores can produce heterokaryotic mycelia containing nuclei of both mating types. In <em>N. crassa</em>, a sexual cross perforce requires mycelia from two different ascospores, one <em>mat A</em> and the other <em>mat a</em>, thus making the lifecycle “heterothallic”; but a heterokaryotic <em>N. tetrasperma</em> mycelium from a single ascospore bearing nuclei of both mating types is competent to undergo a self-cross, making the lifecycle “pseudohomothallic.” However, a subset of conidia (vegetative spores) produced by a heterokaryotic <em>N. tetrasperma</em> mycelium can be homokaryotic by chance, and <em>N. tetrasperma</em> ascogenesis occasionally produces five or more (up to eight) ascospores by replacement of one or more dikaryotic ascospore by a pair of smaller homokaryotic ones (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Raju 1992</a>). The dominant <em>Eight-spore</em> (<em>E</em>) mutant increases the frequency of such replacement, although <em>E</em>-homozygous crosses are infertile (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Calhoun and Howe 1968</a>). Mycelia from homokaryotic conidia or ascospores can cross with like mycelia of the opposite mating type. Therefore, <em>N. tetrasperma</em> is actually a facultatively heterothallic species.</p><p id="p-7">Although [<em>mat A</em> + <em>mat a</em>] heterokaryons can form quite easily in <em>N. tetrasperma</em>, their formation is prevented in <em>N. crassa</em> by mating type–mediated vegetative incompatibility (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Newmeyer 1970</a>). The <em>N. crassa</em> allele of the <em>tolerant</em> gene (<em>tol<sup>C</sup></em>) is a key regulator of this incompatibility (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Shiu and Glass 1999</a>). If <em>tol<sup>C</sup></em> is replaced either by a recessive allele <em>tol</em> or by the <em>N. tetrasperma</em> allele (<em>tol<sup>T</sup></em>), then heterokaryons of genotype [(<em>tol mat A</em>) + (<em>tol mat a</em>)] can form in <em>N. crassa</em> and are stable, provided that they are homokaryotic for <em>het</em> loci that determine other vegetative heterokaryon incompatibilities (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Smith and Lafontaine 2013</a>). <em>N. crassa</em> strains of the same mating type and sharing the same alleles at the other <em>het</em> loci can fuse to form a culture that has both nuclear types in a common cytoplasm. Heterokaryon formation between two strains with different auxotrophic mutations can be “forced” by plating them together on minimal medium. The <em>helper-1</em> strain (genotype <em>a<sup>m1</sup> ad-3B cyh-1</em>) is capable of forming vigorous heterokaryons with either of the mating type strains of the standard laboratory Oak Ridge (OR) background, because the <em>mat a</em> allele <em>a<sup>m1</sup></em> is inactive and does not elicit mating type–mediated vegetative incompatibility. The <em>a<sup>m1</sup></em> allele also makes the <em>helper-1</em> component of such a heterokaryon a passive partner when the heterokaryon is used in a cross (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Perkins <em>et al.</em> 2001</a>). We have used <em>helper-1</em> to genetically map the <em>fmf-1</em> mutation (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Iyer <em>et al.</em> 2009</a>), which has a unique female-sterile and male-sterile phenotype. Heterokaryons have also been used for the analysis of complementation, and to “rescue” recessive lethal mutations. Additionally, heterokaryosis in <em>N. crassa</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Davis 1960</a>) and <em>Penicillium cyclopium</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Jinks 1952</a>) has long been known to confer phenotypic plasticity that enables the fungus to respond to changes in environmental conditions by changes in the ratios of the constituent nuclei, but how this happens is still not understood. Recent studies from Dr. Hanna Johannesson’s laboratory have shown that the <em>mat A</em>/<em>mat a</em> nuclear ratio can change during the <em>N. tetrasperma</em> lifecycle (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Samils <em>et al.</em> 2014</a>). The ratio is biased for <em>mat A</em>-nuclei during mycelial growth and becomes more balanced only during sexual development. There was also a bias in expression for <em>mat A</em>–linked genes during mycelial growth that switched during the sexual stage into a bias for genes of the <em>mat a</em> nuclei. These findings were interpreted to suggest that <em>N. tetrasperma mat A</em> and <em>mat a</em> nuclei have co-evolved to optimize their relative fitness in the heterokaryon by altering their ratios and by regulating gene expression. Previous studies from this group showed that that wild-isolated <em>N. tetrasperma</em> strains from United Kingdom generally produced a greater proportion of homokaryotic conidia than strains isolated from New Zealand, and some even showed a strong bias in the homokaryotic conidia for one of the mating types (<em>e.g.</em>, 0 <em>mat A</em>: 17 <em>mat a</em>; 7 <em>mat A</em>: 0 <em>mat a</em>; and 0 <em>mat A</em>: 9 <em>mat a</em>). However, all the strains made self-fertile conidia (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Corcoran <em>et al.</em> 2012</a>). The high proportion of single-mating-type conidia was attributed to a putative nonrandom distribution of nuclei in the heterokaryotic mycelium and/or stronger ability of certain nuclei to be packaged into conidia.</p><p id="p-8">Here, we have introgressed three <em>N. crassa IT</em>s (<em>EB4</em>, <em>IBj5</em>, and <em>B362i</em>) and one <em>QT</em> (<em>UK14-1</em>) into <em>N. tetrasperma</em>, and we have shown that <em>T</em> × <em>N</em> crosses can produce both [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] heterokaryotic progeny. The [<em>Dp</em> + <em>Df</em>] progeny allowed us to ask whether the <em>Df</em> deletes any gene with a nucleus-limited function. A gene may be considered to be nucleus-limited if nuclei bearing its null allele (<em>Δ</em>) are not complemented by wild-type nuclei (<em>WT</em>) in a [<em>Δ</em> + <em>WT</em>] heterokaryon (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kasbekar 2014</a>). If a <em>Df</em> deletes a nucleus-limited gene, then one might expect to see a phenotype difference between [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] heterokaryons. Although no nucleus-limited genes have been reported as yet, their existence in fungi is not ruled out, especially given the putative nucleus-limited behavior of the <em>N. crassa scon<sup>c</sup></em> mutant (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Burton and Metzenberg 1972</a>), the DNA damage checkpoint signal in <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Demeter <em>et al.</em> 2000</a>), and the MatIS gene silencing process in <em>Aspergillus nidulans</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Czaja <em>et al.</em> 2013</a>). We found that although [<em>T(B362i) A</em>+ <em>N a</em>] heterokaryons could produce both heterokaryotic and homokaryotic conidia, the [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] heterokaryons produced only homokaryotic conidia, possibly because a putative “nucleus-limited” gene required for efficient packaging of nuclei into conidia is absent from the <em>Df(B362i)</em> nuclei. Second, we found evidence that suggests that the <em>N. tetrasperma E</em> mutant contains a recessive mutation affecting alternate but not adjacent 1 segregation. Third, we show that a <em>Dp</em>-heterozygous cross can exhibit RIP (repeat-induced point mutation), the sexual stage-specific process that induces G:C to A:T mutations in duplicated DNA (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Selker 1990</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Neurospora strains and general genetic manipulations</h3><p id="p-9">All Neurospora strains were obtained from the Fungal Genetics Stock Center, Kansas State University, Manhatten, Kansas USA, unless otherwise indicated. <em>N. crassa</em> OR <em>A</em> (FGSC 987) and OR <em>a</em> (FGSC 988) are the standard laboratory Oak Ridge strains; the translocation strains are <em>T</em>(<em>VR→VII</em>)<em>EB4 A</em> (FGSC 3046), <em>T(VIL→IR) IBj5 cpc-1 A</em> (FGSC 4433), <em>T(VIR &gt; VL) UK14-1 A</em> (FGSC 6958), and <em>T(IV→I)B362i A</em> (FGSC 2935) [abbreviated <em>T(EB4)</em>, <em>T(IBj5)</em>, <em>T(UK14-1)</em>, and <em>T(B362i)</em>]. <em>T(EB4)</em>, <em>T(IBj5)</em>, and <em>T(B362i)</em> are <em>IT</em>s, whereas <em>T(UK14-1)</em> is a <em>QT</em>. These translocations have been described by <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Perkins (1997)</a> and <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Singh (2010)</a>. <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Perkins (1997)</a> reported that <em>T(IBj5)</em> translocates a chromosome VIL segment to IR, linked to <em>al-2</em> and <em>un-18</em>. However, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Singh <em>et al.</em> (2010)</a> showed that the VIL segment is, in fact, inserted into a chromosome IVR sequence. Therefore, <em>T(IBj5)</em> might include an additional rearrangement (possibly an <em>RT</em>) involving IVR and IR, whose breakpoints are as yet undetermined. Because <em>T(B362i)</em> also involves chromosomes I and IV, the cross <em>T(IBj5)</em> × <em>T(B362i)</em> would resemble an <em>RT</em>-heterozygous cross.</p><p id="p-10">The semi-dominant MSUD suppressor strains <em>Sad-1 A</em> (FGSC 8740) and <em>Sad-1 a</em> (FGSC 8741) were gifted by the late Robert L. Metzenberg and are described by <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Shiu <em>et al.</em> (2001)</a>. The <em>sad-1</em> locus encodes an RNA-dependent RNA polymerase essential for MSUD, and the <em>Sad-1</em> suppressor allele is presumed to prevent proper pairing of its wild-type homolog, thus inducing it to autogenously silence itself (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Shiu <em>et al.</em> 2001</a>). The MSUD testers <em>pan-2</em>; <em>his-3</em>::<em>his-3<sup>+</sup> Bml<sup>r</sup> A</em> (FGSC 8755); <em>pan-2</em>; <em>his-3</em>::<em>his-3<sup>+</sup> Bml<sup>r</sup> a</em> (FGSC 8756); <em>pan-2</em>; <em>his-3</em>::<em>his-3<sup>+</sup> mei-3<sup>+</sup> A</em> (FGSC 8759); and <em>pan-2</em>; <em>his-3</em>::<em>his-3<sup>+</sup> mei-3<sup>+</sup> a</em> (FGSC 8760) (hereafter designated as :: <em>Bml<sup>r</sup> A</em>, :: <em>Bml<sup>r</sup> a</em>, ::<em>mei-3 A</em>, and ::<em>mei-3 a</em>) are described by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Raju <em>et al.</em> (2007)</a>. Another tester, <em>rid his-3</em>; <em>VIIL</em>::<em>r<sup>ef2</sup>-hph A</em> (ISU3117) (hereafter ::<em>r<sup>ef2</sup></em> ), was a gift from Dr. Tom Hammond (Illinois State University). The ::<em>Bml<sup>r</sup></em> and ::<em>mei-3</em> testers have an extra copy of the <em>bml</em> (β-tubulin) or <em>mei-3</em> gene inserted ectopically in the <em>his-3</em> locus in chromosome 1, whereas the tester strain ::<em>r<sup>ef2</sup></em> has a copy of the <em>r</em> (<em>round spores</em>) gene inserted ectopically into chromosome 7. In a cross of the tester with wild-type, the ectopic copy remains unpaired in meiosis and results in elimination of all its homologous transcripts, including from the paired endogenous copies. In crosses of ::<em>Bml<sup>r</sup></em>, ::<em>mei-3</em>, and ::<em>r<sup>ef2</sup></em> with the wild-type, the <em>bml</em>, <em>mei-3</em>, and <em>r</em> genes, respectively, are silenced. Silencing of the <em>bml</em> or <em>mei-3</em> gene arrests normal ascus development (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Raju <em>et al.</em> 2007</a>; Kasbekar <em>et al.</em> 2011), and silencing of <em>r</em> causes all eight ascospores to be round instead of having the normal spindle shape. Homozygous <em>tester A</em> × <em>tester a</em> crosses do not show MSUD, nor do crosses of the testers with the <em>Sad-1</em> suppressor of MSUD, and the asci developed normally (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Raju <em>et al.</em> 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kasbekar <em>et al.</em> 2011</a>).</p><p id="p-11"><em>N. tetrasperma</em> has standard strains 85 <em>A</em> (FGSC 1270) and 85 <em>a</em> (FGSC 1271); the <em>E</em> mutants are <em>lwn</em>; <em>al(102)</em>, <em>E A</em> (FGSC 2783) and <em>lwn</em>; <em>al(102)</em>, <em>E a</em> (FGSC 2784) (hereafter <em>E A</em> and <em>E a</em>). <em>N. crassa</em>/<em>N. tetrasperma</em> hybrid strain is <em>C4,T4 a</em> (FGSC 1778). The <em>C4,T4 a</em> strain has four <em>N. crassa</em> great-grandparents and four <em>N. tetrasperma</em> great-grandparents (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Metzenberg and Ahlgren 1969</a>). The <em>N. crassa</em> great-grandparents were of the OR background, whereas the <em>N. tetrasperma</em> great-grandparents were of the 343.6 <em>A E</em> background (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Metzenberg and Ahlgren 1969</a>).</p><p id="p-12">Neurospora genetic analysis was performed essentially as described by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Davis and De Serres (1970)</a>. The alternative recipe of <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Metzenberg (2003)</a> was used for making Medium N.</p></div><div id="sec-3" class="subsection"><h3>Oligonucleotide primers used for PCR</h3><p id="p-13"><a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017616.DC1/017616SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017616.DC1/TableS1.pdf">Table S1</a> lists the sequences of the oligonucleotide primers used to test for the translocation breakpoint junctions and to amplify sequences of the <em>ad-7 RIP3C</em> and <em>RIP3T</em> alleles.</p></div><div id="sec-4" class="subsection"><h3>Outline of the introgression crosses and characterization of the resultant strains</h3><p id="p-14">Crosses between <em>N. crassa</em> and <em>N. tetrasperma</em> strains are almost completely sterile. However, both <em>N. crassa</em> strain OR <em>A</em> and <em>N. tetrasperma</em> strain 85 <em>A</em> can cross with the <em>N. crassa</em>/<em>N. tetrasperma</em> hybrid strain <em>C4,T4 a</em> and produce viable progeny (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Perkins 1991</a>; also see <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> of this article); therefore, we used the <em>C4,T4 a</em> strain as a bridging strain for the initial introgression crosses. The <em>N. crassa T</em> strains were crossed with <em>C4T4 a</em> and <em>T</em> progeny from these crosses (designated <em>T<sup>1xC4T4</sup></em>) were distinguished from their <em>Dp</em> and <em>N</em> siblings by PCR with breakpoint junction-specific primers. Nominally, 50% of the genome of <em>T<sup>1xC4T4</sup></em> progeny is derived from the <em>C4,T4 a</em> parent. The <em>T<sup>1xC4T4</sup>A</em> strains were crossed with <em>C4,T4 a</em> to obtain the <em>T<sup>2xC4T4</sup></em> progeny in a similar manner. Crosses of <em>T<sup>2xC4T4</sup></em> with the opposite mating-type derivative of strain <em>85</em> were productive, and their <em>T</em> progeny were designated as <em>T<sup>1x85</sup></em>. Likewise, <em>T<sup>1x85</sup></em> × <em>85</em> yielded <em>T<sup>2x85</sup></em>, etc. After two to three iterations of the crosses with <em>85</em>, we recovered progeny ascospores that produced mycelium of dual mating specificity characteristic of <em>N. tetrasperma</em>. That is, the resulting mycelium could cross with both 85<em>A</em> and <em>a</em>, and it could also undergo a self-cross. A heterokaryotic strain containing all three breakpoints (A, B, and C) is potentially of genotype [<em>T</em> + <em>N</em>] or [<em>Dp</em> + <em>Df</em>].</p><p id="p-15">The [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] heterokaryons are distinguishable, because the former produces homokaryotic conidial derivatives of both mating types, whereas the latter produces viable homokaryons of only the mating type of the <em>Dp</em> nucleus. Conidia from self-fertile heterokaryotic strains were streaked onto Vogel’s FGS medium, and well-isolated conidial germlings were transferred to SCM to distinguish self-fertile (heterokaryotic) from self-sterile (homokaryotic) conidial derivatives. The mating type of the self-sterile conidial derivatives was determined by crossing to the single mating type derivatives <em>85 a</em> and <em>85 A</em>. If all the self-sterile conidial derivatives are of a single mating type, then the heterokaryon from which they were derived is likely [<em>Dp</em> + <em>Df</em>], else it is [<em>T</em> + <em>N</em>]. The results were confirmed by PCR with primers for the breakpoint junctions and <em>mat</em> ideomorphs and DNA of the homokaryotic conidial derivatives.</p><p id="p-16">Four homokaryotic <em>T</em>-type conidial derivatives from the self-fertile [<em>T</em> + <em>N</em>] heterokaryons, namely, <em>T(EB4)<sup>Nt</sup> a</em> from the heterokaryon 3E1 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> serial number 3), <em>T(IBj5)<sup>Nt</sup> a</em> from I4 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> serial number 18), <em>T(UK14-1)<sup>Nt</sup> a</em> from U9 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> serial number 24), and <em>T(B362i)<sup>Nt</sup>A</em> from 19B7 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> serial number 35), were used in the experiments and results are summarized in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>.</p></div><div id="sec-5" class="subsection"><h3>A note on strain nomenclature</h3><p id="p-17">The different self-fertile strains listed in <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a> were named using the following rules. The letters E, I, U, or B in the name identify strains derived from introgressions of, respectively, the <em>N. crassa</em> translocations <em>T(EB4)</em>, <em>T(IBj5)</em>, <em>T(UK14-1)</em>, and <em>T(B362i)</em>. The strains E1, U9, B7, and I1-I5 were self-fertile heterokaryons obtained from the cross of homokaryotic <em>T(EB4)</em>, <em>T(UK14-1)</em>, <em>T(B362i)</em>, and <em>T(IBj5)</em> strains with either <em>85 a</em> or <em>85 A</em> (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Strains 1E1, 2E1, 3E1, etc., were derived from the self-cross of strain E1; likewise, strains 1I1, 2I1, 3I1, etc., were from the self-cross of strain I1; 1U9, 2U9, 3U9, etc., were from the self-cross of strain U9; and 11B7, 18B7, and 19B7 were from self-crosses of strain B7. Further, strains 1(1U9), 2(1U9), 3(1U9), etc., were from self-cross of strain 1U9; 6(19B7) was from self-cross of strain 19B7; and 1[6(19B7)], 2[6(19B7)], etc., were from the self-cross of strain 6(19B7).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1263/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Introgression crosses. T(EB4) A, T(IBj5) A, T(UK14-1) A, and T(B362i) A strains of N. crassa were crossed with the C4T4 a hybrid strain. Bent arrows represent PCR with breakpoint junction-specific primers to distinguish the translocation progeny (e.g., T1xC4T4) from their Dp and N siblings. T1xC4T4A &#xD7; C4T4 a yielded T2xC4T4 A or T2xC4T4 a strains, which were productive in crosses with opposite mating-type homokaryotic derivatives of N. tetrasperma strain 85. T1x85 progeny were crossed with 85 a or 85 A to obtain the self-fertile heterokaryotic strains I1-I5 (for IBj5) and U9 (for UK14-1), or the T2x85 strains (for EB4 and B362i). Crosses of T2x85 with 85 a produced the heterokaryons E1 and B7. From self-cross of the heterokaryons, we obtained self-fertile progeny that were genotyped as [T + N] or [Dp + Df] (see Table 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-81004736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Introgression crosses. &lt;em&gt;T(EB4) A&lt;/em&gt;, &lt;em&gt;T(IBj5) A&lt;/em&gt;, &lt;em&gt;T(UK14-1) A&lt;/em&gt;, and &lt;em&gt;T(B362i) A&lt;/em&gt; strains of &lt;em&gt;N. crassa&lt;/em&gt; were crossed with the &lt;em&gt;C4T4 a&lt;/em&gt; hybrid strain. Bent arrows represent PCR with breakpoint junction-specific primers to distinguish the translocation progeny (&lt;em&gt;e.g.&lt;/em&gt;, &lt;em&gt;T&lt;sup&gt;1xC4T4&lt;/sup&gt;&lt;/em&gt;) from their &lt;em&gt;Dp&lt;/em&gt; and &lt;em&gt;N&lt;/em&gt; siblings. &lt;em&gt;T&lt;sup&gt;1xC4T4&lt;/sup&gt;A&lt;/em&gt; &#xD7; &lt;em&gt;C4T4 a&lt;/em&gt; yielded &lt;em&gt;T&lt;sup&gt;2xC4T4&lt;/sup&gt; A&lt;/em&gt; or &lt;em&gt;T&lt;sup&gt;2xC4T4&lt;/sup&gt; a&lt;/em&gt; strains, which were productive in crosses with opposite mating-type homokaryotic derivatives of &lt;em&gt;N. tetrasperma&lt;/em&gt; strain &lt;em&gt;85&lt;/em&gt;. &lt;em&gt;T&lt;sup&gt;1x85&lt;/sup&gt;&lt;/em&gt; progeny were crossed with &lt;em&gt;85 a&lt;/em&gt; or &lt;em&gt;85 A&lt;/em&gt; to obtain the self-fertile heterokaryotic strains I1-I5 (for &lt;em&gt;IBj5&lt;/em&gt;) and U9 (for &lt;em&gt;UK14-1&lt;/em&gt;), or the &lt;em&gt;T&lt;sup&gt;2x85&lt;/sup&gt;&lt;/em&gt; strains (for &lt;em&gt;EB4&lt;/em&gt; and &lt;em&gt;B362i&lt;/em&gt;). Crosses of &lt;em&gt;T&lt;sup&gt;2x85&lt;/sup&gt;&lt;/em&gt; with &lt;em&gt;85 a&lt;/em&gt; produced the heterokaryons E1 and B7. From self-cross of the heterokaryons, we obtained self-fertile progeny that were genotyped as [&lt;em&gt;T&lt;/em&gt; + &lt;em&gt;N&lt;/em&gt;] or [&lt;em&gt;Dp&lt;/em&gt; + &lt;em&gt;Df&lt;/em&gt;] (see Table 2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1263/F2.medium.gif" width="278" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1263/F2.medium.gif" width="278" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1263/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1263/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466306" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Introgression crosses. <em>T(EB4) A</em>, <em>T(IBj5) A</em>, <em>T(UK14-1) A</em>, and <em>T(B362i) A</em> strains of <em>N. crassa</em> were crossed with the <em>C4T4 a</em> hybrid strain. Bent arrows represent PCR with breakpoint junction-specific primers to distinguish the translocation progeny (<em>e.g.</em>, <em>T<sup>1xC4T4</sup></em>) from their <em>Dp</em> and <em>N</em> siblings. <em>T<sup>1xC4T4</sup>A</em> × <em>C4T4 a</em> yielded <em>T<sup>2xC4T4</sup> A</em> or <em>T<sup>2xC4T4</sup> a</em> strains, which were productive in crosses with opposite mating-type homokaryotic derivatives of <em>N. tetrasperma</em> strain <em>85</em>. <em>T<sup>1x85</sup></em> progeny were crossed with <em>85 a</em> or <em>85 A</em> to obtain the self-fertile heterokaryotic strains I1-I5 (for <em>IBj5</em>) and U9 (for <em>UK14-1</em>), or the <em>T<sup>2x85</sup></em> strains (for <em>EB4</em> and <em>B362i</em>). Crosses of <em>T<sup>2x85</sup></em> with <em>85 a</em> produced the heterokaryons E1 and B7. From self-cross of the heterokaryons, we obtained self-fertile progeny that were genotyped as [<em>T</em> + <em>N</em>] or [<em>Dp</em> + <em>Df</em>] (see <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3><em>N. tetrasperma</em> [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] strains can switch genotype via self-crosses</h3><p id="p-19">The salient features of the <em>N. crassa</em> translocations <em>T(EB4)</em>, <em>T(IBj5)</em>, <em>T(UK14-1)</em>, and <em>T(B362i)</em> are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, along with the accession numbers of their breakpoint junction sequences. The introgression of these translocations into <em>N. tetrasperma</em> is outlined in the <em>Materials and Methods</em> section, and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> schematically presents the actual crosses performed.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466340/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466340/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Translocations used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Introgression of <em>T(EB4)</em> yielded the self-fertile heterokaryotic strain designated E1 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Using E1 genomic DNA as template and <em>T(EB4)</em> breakpoint junction-specific primers, all three breakpoint junctions (A, B, and C) of <em>T(EB4)</em> could be amplified by PCR (data not shown). A heterokaryon possessing all the three breakpoints is potentially of genotype [<em>T(EB4)</em> + <em>N</em>] or [<em>Dp(EB4)</em> + <em>Df(EB4)</em>]. Heterokaryons of genotype [<em>T(EB4)</em> + <em>T(EB4)</em>], [<em>T(EB4)</em> + <em>Dp(EB4)</em>], or [<em>T(EB4)</em> + <em>Df(EB4)</em>] also fulfill this criterion, but, they were deemed to be less likely because one or more crossover is required to generate them. Eight self-fertile progeny from the self-cross of E1 were analyzed and the results, which are summarized in <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a> (serial numbers 1–8), established that progeny 2E1, 4E1, and 6E1 were of the [<em>T(EB4)</em> + <em>N</em> ] genotype, whereas 1E1, 9E1, and 12E1 were [<em>Dp(EB4)</em> + <em>Df(EB4)</em>]. Only six self-sterile conidial derivatives were obtained for 3E1, and PCR revealed all to be type <em>T(EB4) a</em>, suggesting that 3E1 has the genotype [<em>T(EB4) a</em> + <em>Df(EB4) A</em>]. However, given the small numbers tested, and given the possibility of skewed segregation, our results do not exclude the [<em>T(EB4) a</em> + <em>N A</em>] genotype. No self-sterile conidial derivatives were obtained from 13E1; therefore, its genotype was not determined.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466230/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466230/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genotype of self-fertile strains</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">From self-crosses of strains 6E1 and 9E1 (see above), we examined 39 and 24 progeny and found that 12 and 9, respectively, were self-fertile. From a subset of self-fertile progeny, we obtained self-sterile (<em>i.e.</em>, homokaryotic) conidial derivatives and determined their mating type by crossing with 85 <em>a</em> and 85 <em>A</em>. Of three self-fertile progeny tested from strain 6E1, two appeared to be [<em>T(EB4) a</em> + <em>N A</em>] and one was [<em>Dp(EB4) a</em> + <em>Df(EB4) A</em>]; of three self-fertile progeny tested from strain 9E1, one appeared to be [<em>Dp(EB4) a</em> + <em>Df(EB4) A</em>] and the other two appeared to be [<em>T(EB4) a</em> + <em>N A</em>]. We used PCR to confirm the genotype of the [<em>Dp(EB4) a</em> + <em>Df(EB4) A</em>] progeny of 6E1, and the genotype of one [<em>T(EB4) a</em> + <em>N A</em>] progeny of 9E1 (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>, serial numbers 10 and 12). These results showed that self-crosses of both [<em>T(EB4)</em> + <em>N</em>] and [<em>Dp(EB4)</em> + <em>Df(EB4)</em>] heterokaryons can again generate [<em>T(EB4)</em> + <em>N</em>] and [<em>Dp(EB4)</em> + <em>Df(EB4)</em>] progeny.</p><p id="p-31">The translocations <em>T(IBj5)</em>, <em>T(UK14-1)</em>, and <em>T(B362i)</em> were introgressed in a similar manner (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>), and we recovered heterokaryons of genotypes [<em>T(IBj5)</em> + <em>N</em>] and [<em>Dp(IBj5)</em> + <em>Df(IBj5)</em>] (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a>, serial numbers 15, 18-21, and 25), [<em>T(UK14-1)</em> + <em>N</em>] and [<em>Dp(UK14-1)</em> + <em>Df(UK14-1)</em>] (<a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Table 2</a>, serial number 26, 27, 31, 33, and 34), and [<em>T(B362i)</em> + <em>N</em>] and [<em>Dp(B362i)</em> + <em>Df(B362i)</em>] (<a id="xref-table-wrap-2-11" class="xref-table" href="#T2">Table 2</a>, serial numbers 35-41, 43, 45, and 47). Again, self-crosses of the [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] heterokaryons produced progeny of the alternative genotype. Specifically, self-cross of the [<em>Dp(IBj5)</em> + <em>Df(IBj5)</em>] type strain I1 produced the [<em>T(IBj5)</em> + <em>N</em>] type progeny strain 1I1 (<a id="xref-table-wrap-2-12" class="xref-table" href="#T2">Table 2</a>, serial numbers 15 and 20); of the [<em>T(UK14-1)</em> + <em>N</em>] type strain, U9 produced the [<em>Dp(UK14-1)</em> + <em>Df(UK14-1)</em>] type progeny strain 1U9, whose self-cross, in turn, produced the [<em>T(UK14-1)</em> + <em>N</em>] type strain 2(1U9) (<a id="xref-table-wrap-2-13" class="xref-table" href="#T2">Table 2</a>, serial numbers 26, 27, and 33); and of the [<em>Dp(B362i)</em> + <em>Df(B362i)</em>] type strain, B7 produced the [<em>T(B362i5)</em> + <em>N</em>] type progeny strain 19B7, whose self-cross, in turn, produced the [<em>Dp(B362i)</em> + <em>Df(B362i)</em>] type progeny strain 6(19B7), whose self-cross produced the [<em>T(B362i5)</em> + <em>N</em>] type progeny 2[6(19B7)] and 5[6(19B7)] (<a id="xref-table-wrap-2-14" class="xref-table" href="#T2">Table 2</a>, serial numbers 35, 38, 43, 45 and 47). In sum, our results show that [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] genotypes can be interchanged through self-crosses.</p><p id="p-32">The genotype of two heterokaryons was found to be putatively [<em>Df(IBj5) a</em> + <em>N A</em>] and [<em>Df(IBj5) a</em> + <em>T(IBj5)A</em>] (<a id="xref-table-wrap-2-15" class="xref-table" href="#T2">Table 2</a>, serial numbers 16 and 17), but our results do not rule out the possibility that skewed segregation in small numbers might account for the absence of self-sterile <em>T(IBj5)</em> or <em>N</em> conidial types, respectively, from what in fact might be [<em>T(IBj5)</em> + <em>N</em>] heterokaryons. One heterokaryon was found to contain three nuclear types, and its genotype was [<em>T(IBj5) a</em> + <em>NA</em> + <em>Df(IBj5)A</em>] (<a id="xref-table-wrap-2-16" class="xref-table" href="#T2">Table 2</a>, serial number 22). It got flagged because two of the seven <em>mat A</em> self-sterile conidial derivatives examined possessed only the A, but not B, junction of <em>T(IBj5)</em>, whereas the other five did not possess either junction. A homokaryon bearing only the A junction is not expected to be viable because it contains an uncomplemented <em>Df</em> chromosome; therefore, we presume the two self-sterile <em>mat A</em> derivatives were [<em>NA</em> + <em>Df(IBj5)A</em>] heterokaryons and we infer that the genotype of the self-fertile strain was [<em>T(IBj5) a</em> + <em>NA</em> + <em>Df(IBj5)A</em>]. We defer to the <em>Discussion</em> section a consideration of how such a strain might have arisen.</p></div><div id="sec-8" class="subsection"><h3>[<em>Dp(B362i)</em> + <em>Df(B362i)</em>] heterokaryons yield only self-sterile conidial derivatives</h3><p id="p-33">The search for self-sterile homokaryotic conidial derivatives (above) was expected to inevitably also identify self-fertile ones. A majority of conidial derivatives from both [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] strains of <em>EB4</em>, <em>IBj5</em>, and <em>UK14-1</em> were self-fertile (<a id="xref-table-wrap-2-17" class="xref-table" href="#T2">Table 2</a>, serial numbers 1-33), and self-fertile conidial derivatives (or derivatives heterokaryotic for mating type) were also obtained from [<em>T(B362i)</em> + <em>N</em>] (<a id="xref-table-wrap-2-18" class="xref-table" href="#T2">Table 2</a>, serial numbers 38, 45, and 47). Therefore, we were very surprised to find that all the 117 conidial derivatives examined from seven different [<em>Dp(B362i)</em> + <em>Df(B362i)</em>] heterokaryons were self-sterile homokaryons (<a id="xref-table-wrap-2-19" class="xref-table" href="#T2">Table 2</a>, serial numbers 35–37, 39–41, and 43).</p><p id="p-34">As an additional control we performed the cross <em>Dp(B362i) a</em> × <em>85 A</em> and tested 61 progeny, of which 39 proved to be self-fertile (<em>i.e.</em>, heterokaryotic). We examined 10 to 25 conidial derivatives from each of 16 self-fertile progeny, and in every case we found some self-fertile conidial derivatives. Additionally, for three progeny, we also identified self-sterile conidial derivatives of both mating types, and by PCR we could establish that the genotype of one progeny was [<em>Dp(B362i) a</em> + <em>Dp(B362i) A +N A</em>], and the other two were [<em>Dp(B362i) a</em> + <em>Dp(B362i) A</em>]. Thus, unlike their [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] counterparts, the [<em>Dp(B362i)</em> + <em>N</em>] heterokaryons were able to make self-fertile heterokaryotic conidia. Additionally, among the progeny from the self-cross of a [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] self-fertile heterokaryon, we recovered one that was [<em>N a + Dp(B362i) A</em>] type (<a id="xref-table-wrap-2-20" class="xref-table" href="#T2">Table 2</a>, serial number 44), and of four conidial derivatives examined, one was self-fertile. Therefore, the absence of self-fertile conidia from [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] heterokaryons appears to be exceptional. The implication of this phenotype is considered in the <em>Discussion</em> section.</p></div><div id="sec-9" class="subsection"><h3>Characterizing the <em>T</em>-type homokaryons</h3><p id="p-35">The homokaryotic <em>T</em>-type conidial derivatives designated as <em>T(EB4)<sup>Nt</sup></em> , <em>T(IBj5)<sup>Nt</sup></em>, <em>T(UK14-1)<sup>Nt</sup></em>, and <em>T(B362i)<sup>Nt</sup></em> were obtained from the self-fertile [<em>T</em> + <em>N</em>] heterokaryons (see <em>Materials and Methods</em>) and were found to behave like <em>bona fide</em> <em>N. tetrasperma</em> strains. That is, their crosses with opposite mating-type derivatives of <em>N. tetrasperma</em> strain <em>85</em> were fertile, whereas their crosses with <em>N. crassa</em> OR strains of opposite mating type were as infertile as an OR × <em>85</em> interspecies cross (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Control crosses of the <em>N. crassa T</em> strains (<em>T<sup>Nc</sup></em> ) with the OR strains of the opposite mating type were productive, but the crosses of the <em>T<sup>Nc</sup></em> with the opposite mating type derivatives of strain <em>85</em> were sterile (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The <em>C4,T4 a</em> hybrid strain produced viable ascospores in crosses with both OR <em>A</em> and <em>85 A</em> (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466277/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466277/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Ascospore productivity in crosses of <em>T</em> strains with OR and 85</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">To obtain larger numbers of eight-spored asci, we crossed the <em>T<sup>Nt</sup></em> strains with <em>E</em> strains of the opposite mating type. The <em>T(EB4)<sup>Nt</sup> a</em> × <em>E A</em> and <em>T(UK14-1)<sup>Nt</sup> A</em> × <em>E a</em> crosses were productive and produced 8:0 and 4:4 ascus types at comparable frequencies (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), which is a characteristic of <em>IT</em> × <em>N</em> and <em>QT</em> × <em>N</em> crosses in <em>N. crassa</em>. Surprisingly, the crosses <em>T(IBj5)<sup>Nt</sup> a</em> × <em>E A</em> and <em>T(B362i)<sup>Nt</sup> A</em> × <em>E a</em> did not produce any 8:0 or 6:2 asci, and most ascospores from these crosses (∼90 and 70%, respectively) were white and inviable (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). However, the crosses <em>T(IBj5)<sup>Nt</sup>A</em> × <em>85 a</em> and <em>T</em>(<em>B362i</em>)<em><sup>Nt</sup>A</em> × <em>85 a</em> produced a few eight-spored asci, and in them the 8:0 and 4:4 frequencies appeared to be comparable (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466304/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466304/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Ascus (octets) types from <em>T</em> × <em>N</em> crosses in <em>N. crassa</em> (<em>Nc</em>) and <em>N. tetrasperma</em> (<em>Nt</em>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">Results summarized in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> show that the crosses of <em>T</em>(<em>EB4</em>)<em><sup>Nt</sup>a</em> and <em>T</em>(<em>IBj5</em>)<em><sup>Nt</sup>a</em> with <em>T</em>(<em>UK14-1</em>)<em><sup>Nt</sup>A</em> and <em>T</em>(<em>B362i</em>)<em><sup>Nt</sup>A</em> produced many four-spored asci, consistent with the conclusion that the <em>T<sup>Nt</sup></em> strains behave like <em>bona fide</em> <em>N. tetrasperma</em> strains. However, the crosses also produced many nonfour spored asci, probably because the <em>T<sup>Nt</sup></em> strains still retain a significant fraction of the ancestral non-<em>85</em> genetic background (<em>N. crassa T</em> or <em>C4T4 a</em>). All ascospores produced in the cross <em>T</em> (<em>IBj5</em>)<em><sup>Nt</sup> a</em> × <em>T</em> (<em>B362i</em>)<em><sup>Nt</sup> A</em> were white and inviable. A model to explain this result is considered in the <em>Discussion</em> section.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466318/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466318/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Ascus types from <em>N. tetrasperma T</em> × <em>T</em> crosses</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>RIP in a <em>Dp</em>-heterozygous <em>N. tetrasperma</em> cross</h3><p id="p-43">In <em>N. crassa</em>, crosses involving <em>Dp</em> strains can generate RIP-induced mutant progeny (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Perkins <em>et al.</em> 1997</a>). Therefore, we expected crosses of <em>Dp</em> strains in <em>N. tetrasperma</em> also to yield RIP-induced mutants. <em>Dp(EB4)</em> duplicates the <em>ad-7</em> (<em>adenine requiring-7</em>) gene (<a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Perkins 1997</a>). Ascospores from <em>Dp(EB4) a</em> × <em>E A</em> were germinated on adenine-supplemented Vogel’s FGS medium, 130 germlings were picked to adenine-supplemented Vogel’s glucose medium, and then their growth was tested on unsupplemented Vogel’s glucose medium. Three adenine-requiring auxotrophic strains were identified among 125 progeny examined. Two (RIP1 and RIP2) were <em>N</em>-type homokaryons; their <em>ad-7</em> locus, derived from strain <em>85 A</em>, was found to be altered by several RIP mutations (G:C to A:T transitions) (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). The third auxotrophic strain (RIP3) was a heterokaryon for mating type, showing that <em>E</em> has incomplete penetrance. Both the <em>mat A</em> and <em>mat a</em> nuclei of this heterokaryon must contain RIP-induced mutant <em>ad-7</em> alleles, and its genotype potentially can be [<em>Dp</em> + <em>N</em>], [<em>Dp</em> + <em>Dp</em>], or [<em>N</em> + <em>N</em>], with the latter two resulting from second-division segregation. Both copies of the 145-kb duplicated segment that participated in RIP are “captured” in the [<em>Dp</em> + <em>N</em>] and [<em>Dp</em> + <em>Dp</em>] types. The <em>T</em> recipient chromosome has <em>N. crassa</em>–derived sequences, whereas the <em>N</em> homolog of the <em>T</em> donor chromosome has 85 <em>A</em>-derived sequences; therefore, their <em>ad-7</em> mutant alleles, <em>RIP3C</em> and <em>RIP3T</em>, are distinguishable. Both <em>RIP3C</em> and <em>RIP3T</em> showed evidence of RIP (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). Interestingly, all the <em>RIP3T</em> mutations were G to A changes, whereas those in <em>RIP3C</em> were C to T changes.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466182/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466182/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">RIP-induced <em>ad-7</em> mutants from <em>Dp(EB4)</em> × <em>E</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Crosses involving <em>C4,T4</em> a and <em>85 A/a</em> show relatively weak MSUD</h3><p id="p-56"><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ramakrishnan <em>et al.</em> (2011)</a> classified wild-isolated <em>N. crassa</em> strains into three types based on the strength of MSUD in their crosses with tester strains, namely, “OR” (which includes the standard laboratory strains OR <em>A</em> and OR <em>a</em>), “Sad” (which includes the semi-dominant <em>Sad</em> suppressors of MSUD obtained in the OR background), and “Esm.” MSUD was strongest in crosses with the OR type, of intermediate strength in crosses with the Esm type, and weakest in crosses with the Sad type. We performed crosses of <em>C4,T4 a</em> with the MSUD tester strains. The results summarized in <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017616.DC1/FigureS1.pdf">Figure S1</a> show that, like the Esm-type strains, <em>C4,T4 a</em> also partially suppresses MSUD. The Esm phenotype of <em>C4,T4 a</em> must have come from its 343.6 <em>A E</em> ancestor.</p><p id="p-57">In <em>N. crassa</em>, crosses of <em>Dp(EB4)</em> and <em>Dp(IBj5)</em>strains with the OR-type strains were barren, whereas their crosses with the Sad-type strains were fertile (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Ramakrishnan <em>et al.</em> 2011</a>). Most Esm-type strains gave a fertile cross with <em>Dp(EB4)</em> and gave a barren cross with <em>Dp(IBj5)</em>. Some Esm-type strains gave barren crosses with both the <em>Dp</em>s, and fewer still gave fertile crosses with both the <em>Dp</em>s. Other results of <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Ramakrishnan <em>et al.</em> (2011)</a> suggested that <em>N. tetrasperma</em> strain 85 is the Esm type. We crossed the <em>N. tetrasperma Dp(EB4)</em> and <em>Dp(IBj5)</em> strains with opposite mating-type derivatives of strain 85 and found the crosses were as productive as crosses of <em>T(EB4)</em> and <em>T(IBj5)</em> with strain 85 (data not shown). This is consistent with the classification of strain 85 as the Esm type or the Sad type.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-58">We have made self-fertile heterokaryotic [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] Neurospora strains by introgressing translocations from <em>N. crassa</em> into <em>N. tetrasperma</em>. Both types of heterokaryons when self-crossed again produced [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] progeny. [<em>T</em> + <em>N</em>] and [<em>Dp</em> + <em>Df</em>] mycelia were distinguishable because the former yielded self-sterile homokaryotic conidial derivatives of both mating types, whereas the latter produced homokaryons of only the mating type of the <em>Dp</em> component. The introgressions depended on the unambiguous identification, by PCR with breakpoint junction-specific primers, of the translocation (<em>T</em>) progeny from each round of crosses, as represented by the bent arrows in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>. In 1984, N. B. Raju crossed a <em>N. crassa</em> rearrangement strain with the giant-spore <em>Banana</em> (<em>Ban</em>) mutant with the hope that he could rescue the [<em>Dp</em> + <em>Df</em>] heterokaryon. He had previously used this approach to rescue <em>Sk<sup>S</sup></em> nuclei from a <em>Sk<sup>S</sup></em> × <em>Sk<sup>K</sup></em> cross in [<em>Sk<sup>S</sup></em> + <em>Sk<sup>K</sup></em>] heterokaryotic ascospores (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Raju 1979</a>); however, analysis of progeny nuclei in the mixed cultures from the later experiment was not easy, and his efforts were inconclusive (N. B. Raju, personal communication). D. D. Perkins also alluded to his obtaining [<em>Dp</em> + <em>Df</em>] heterokaryons (<a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Perkins 1997</a>), but his results remained unpublished. Therefore, to the best of our knowledge, this is the first report of heterokaryons with complementary duplications and deficiencies in their constituent nuclei.</p><p id="p-59">The introgressions were facilitated by use of the <em>C4,T4 a</em> bridging strain (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Perkins 1991</a>). We found that <em>C4,T4 a</em> is a moderate suppressor of MSUD (<a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017616.DC1/017616SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017616.DC1/FigureS1.pdf">Figure S1</a>). <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Shiu <em>et al.</em> (2001)</a> had shown that the <em>N. crassa</em> MSUD suppressor <em>Sad-1</em> can partially breach the <em>N. crassa</em>/<em>N. tetrasperma</em> interspecies barrier. Although <em>C4,T4 a</em> does not suppress MSUD as strongly as <em>Sad-1 a</em>, its moderate suppressor phenotype might contribute to increase ascospore production in crosses with both <em>N. crassa</em> and <em>N. tetrasperma</em> strains. The <em>N. crassa T</em> strains were first crossed with <em>C4T4 a</em>, and <em>T A</em> progeny from this cross were crossed with <em>C4T4 a</em>. <em>T</em> progeny from the latter crosses were used to initiate two or three additional rounds of crosses with opposite mating-type derivatives of <em>N. tetrasperma</em> strain <em>85</em>. Although this sequence of introgression crosses can be indefinitely continued, we stopped once they began yielding self-fertile progeny. Nominally, one nucleus in the B7, E1, I1, I2, I3, I4, I5, and U9 self-fertile progeny of <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> derives one-eighth or one-fourth of its genome from the non-<em>85</em> genetic background (<em>N. crassa T</em> or <em>C4T4 a</em>) and, depending on the translocation, this fraction is enriched for sequences from the translocation chromosomes (1, 4, 5, 6, or 7). Therefore, in principle, by sequencing one should be able to identify strain <em>85</em> genome segments that are replaceable by non-<em>85</em> sequences without losing the pseudohomothallic life cycle.</p><p id="p-60">Self-crosses of the [<em>T</em> + <em>N</em>] / [<em>Dp</em> + <em>Df</em>] strains also appeared to produce [<em>T</em> + <em>Df</em> ], [<em>N</em> + <em>Dp</em>], and [<em>N</em> + <em>Df</em>] genotypes. These genotypes can be generated by a breakpoint-proximal crossover on the <em>T</em> donor or recipient chromosome, followed by second-division segregation of the resulting <em>Df</em>-bearing or <em>N</em>-bearing chromosome into both <em>mat a</em> and <em>mat A</em> nuclei. Breakpoint-proximal crossover in <em>IT</em> × <em>N</em> and <em>QT</em> × <em>N</em> crosses in <em>N. crassa</em> and in <em>IT</em> × <em>E</em> and <em>QT</em> × <em>E</em> crosses in <em>N. tetrasperma</em> produces 6:2 ascus types that have two ascospores each of <em>T</em>, <em>N</em>, <em>Dp</em>, and <em>Df</em> types (see <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> in <a id="xref-ref-20-10" class="xref-bibr" href="#ref-20">Perkins 1997</a>). One self-fertile heterokaryon had the genotype [<em>T a</em> + <em>NA</em> + <em>Df A</em>]. This triple heterokaryon could have arisen from an interstitial crossover; that in <em>N. crassa</em> would have produced a 6:2 ascus with two each of the <em>T a</em>, <em>N A</em>, <em>Dp a</em>, and <em>Df A</em> nuclei. Presumably, during ascus development, three nuclei (<em>T a</em>, <em>N A</em>, and <em>Df A</em>) instead of two were partitioned into the ascospore from which this strain was derived. In <em>N. crassa</em> crosses heterozygous for the <em>Fsp-1</em> (<em>Four-spore-1</em>) or <em>Fsp-2</em> (<em>Four-spore-2</em>) mutant, rare two-spored and three-spored asci are occasionally formed that contain heterokaryotic ascospores (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Raju 1986</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Perkins <em>et al.</em> 2001</a>). Our [<em>T a</em> + <em>NA</em> + <em>Df A</em>] strain may have had a similar provenance.</p><p id="p-61">The 8:0 and 4:4 ascus types in <em>IT</em> × <em>N</em> crosses in <em>N. crassa</em>, and now among the eight-spored asci from <em>IT</em> × <em>N</em> crosses in <em>N. tetrasperma</em>, arise, respectively, from alternate and adjacent 1 segregation. The 6:2 asci arise from crossing-over in either of the interstitial regions between centromeres and breakpoints, followed by alternate or adjacent 1 segregation (<a id="xref-ref-20-11" class="xref-bibr" href="#ref-20">Perkins 1997</a>). The eight-spored asci from <em>T(IBj5)<sup>Nt</sup> A</em> × 85 <em>a</em> and <em>T</em> (<em>B362i</em>)<em><sup>Nt</sup> A</em> × 85 <em>a</em> included 8:0, 6:2, and 4:4 types (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>), and showed that both alternate and adjacent 1 segregation occur at comparable frequencies. Therefore, the absence of 8:0 and 6:2 asci from <em>T(IBj5)<sup>Nt</sup> a</em> × <em>E A</em> and <em>T(B362i)<sup>Nt</sup> A</em> × <em>E a</em> was quite surprising (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). In these crosses, the viable ascospores came only from the 4:4 and 2:6 asci. Because the genome of both the <em>C4,T4 a</em>, and <em>E</em> strains is partly derived from the <em>N. tetrasperma</em> 343.6 <em>A</em> strain (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Metzenberg and Ahlgren 1969</a>), we suggest that the <em>T(IBj5)<sup>Nt</sup> a</em> × <em>E A</em> and <em>T(B362i)<sup>Nt</sup> A</em> × <em>E a</em> crosses may have become homozygous for a 343.6 <em>A</em>–derived recessive mutation that specifically affects alternate segregation, whereas the mutation may be heterozygous in the crosses <em>T</em> (<em>EB4</em>)<em><sup>Nt</sup> a</em> × <em>E A</em> and <em>T</em> (<em>UK14-1</em>)<em><sup>Nt</sup> A</em> × <em>E a</em>. To our best knowledge, such a mutant phenotype has not previously been reported. It is not detectable in an <em>N</em> × <em>N</em> cross, and an <em>RT</em> × <em>N</em> cross in the mutant background would produce only inviable ascospores. Given the possibility that <em>T(IBj5)</em> might be a complex rearrangement that includes a (I ; IV) <em>RT</em> (see <em>Materials and Methods</em>), homozygosity for the mutation can explain why <em>T</em>(<em>IBj5</em>)<em><sup>Nt</sup>a</em> × <em>T</em>(<em>B362i</em>)<em><sup>Nt</sup>A</em> yielded only inviable ascospores (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). In principle, it should be possible by genome sequencing to identify a <em>343.6 A</em>–derived mutation present in <em>T(IBj5)<sup>Nt</sup> a</em>, <em>T(B362i)<sup>Nt</sup> A</em> and <em>E</em>, but absent in <em>T</em> (<em>EB4</em>)<em><sup>Nt</sup>a</em> or <em>T</em> (<em>UK14-1</em>)<em><sup>Nt</sup>A</em>.</p><p id="p-62">The <em>Dp(EB4)</em>-borne <em>ad-7</em> gene was alterable by RIP in a <em>Dp(EB4)</em> × <em>E</em> cross. Our results suggest that when a duplicated segment undergoes a single round of RIP, one copy of it suffers G to A mutations, whereas the other suffers C to T mutations. Because self-crosses of [<em>Dp</em> + <em>Df</em>], but not [<em>T</em> + <em>N</em>], heterokaryons can undergo RIP, the extent to which a translocated segment is altered would depend on the number of adjacent 1 <em>vs.</em> alternate segregations in “ancestral self-crosses.”</p><p id="p-63">All four [<em>T</em> + <em>N</em>] heterokaryons (<em>i.e.</em>, <em>EB4</em>, <em>IBj5</em>, <em>UK14-1</em>, and <em>B362i</em>) and three of the [<em>Dp</em> + <em>Df</em>] heterokaryons (<em>i.e.</em>, <em>EB4</em>, <em>IBj5</em>, and <em>UK14-1</em>) produced both self-fertile and self-sterile conidial derivatives; in most cases, the proportion of self-fertile conidial derivatives was &gt;50% (<a id="xref-table-wrap-2-21" class="xref-table" href="#T2">Table 2</a>). The [<em>N a + T(B362i) A</em>] and [<em>N a + Dp(B362i) A</em>] heterokaryons also produced self-fertile and self-sterile conidial derivatives, although in the [<em>N a + T(B362i) A</em>] strains the proportion of self-fertile ;conidial derivatives was &lt;50%. Therefore, the inability of all seven [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] strains examined to produce any self-fertile conidial derivatives was exceptional. It may be that the <em>Df(B362i) A</em> nuclei divide much less efficiently than the <em>Dp(B362i) a</em> nuclei; therefore, their fraction rapidly dwindles in the [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] hyphae, making them increasingly less likely to be packaged along with the <em>Dp(B362i) a</em> nuclei during conidiation. Even if a few heterokaryotic [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] conidia did form, the <em>Df(B362i) A</em> nuclei would be less likely to divide on conidial germination, thus biasing even the heterokaryotic conidia to produce homokaryotic <em>Dp(B362i) a</em> germlings. This hypothesis provides an explanation for an otherwise perturbing result that we had obtained in the early stages of this study. A strain was initially scored as [<em>Dp(B362i)</em> + <em>Df(B362i)</em>], because of its self-fertility and the ability of its DNA to support PCR amplification of all three <em>B362i</em>-specific junction fragments. However, later it behaved like a <em>Dp(B362i)</em> homokaryon, because it had become self-sterile and its newly isolated DNA failed to amplify the A fragment (D. A. Giri, unpublished results). A self-cross selectively restores the <em>Df(B362i)</em> nuclear fraction back to 50%, but self-crosses are not possible once all the <em>Df(B362i)</em> nuclei are lost. One model to account for the putative nuclear division defect of <em>Df(B362i) A</em> nuclei in the [<em>Dp(B362i) a</em> + <em>Df(B362i) A</em>] hyphae is that introgression of <em>T(B362i)</em>-linked chromosome 1 sequences from <em>N. crassa</em> disrupts the internuclear interactions of the <em>N. tetrasperma mat A</em> and <em>mat a</em> nuclei that were presumed to have co-evolved to optimize their fitness in [<em>mat A</em> + <em>mat a</em>] heterokaryons (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Samils <em>et al.</em> 2014</a>). The mating-type chromosome of <em>N. tetrasperma</em> has a recombination (crossing over) block over most of its length that ensures that ascospores get one nucleus of each mating type (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gallegos <em>et al.</em> 2000</a>). The block has been correlated with large inversions on the <em>mat A</em> chromosome relative to the <em>mat a</em> chromosome, which has a gene arrangement like that of <em>N. crassa</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ellison <em>et al.</em> 2011</a>). The introgression of <em>T(B362i) A</em> chromosome 1 sequences might produce a <em>N. tetrasperma mat A</em> chromosome with more synteny with the <em>mat a</em> chromosome over most or all of its length, thereby disrupting the interactions between the <em>mat A</em> and <em>mat a</em> nuclei. Another model is that the <em>Df(B362i)</em> nuclei might simply lack a gene required for efficient nuclear division and whose null phenotype is not complemented by the wild-type allele in the neighboring <em>Dp(B362i)</em> nuclei, that is, a nucleus-limited gene. Studies showing the nucleus-limited nature of DNA damage checkpoint signal in <em>S. cerevisiae</em> (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Demeter <em>et al.</em> 2000</a>) are germane to the second model because they demonstrated that of two nuclei sharing the same cytoplasm, the one with damaged DNA arrests in mitosis without impeding progression through mitosis of the other with undamaged DNA. If the “disruption of co-evolved interactions” model is correct, then we might expect to see the same or similar phenotype following the introgression of another <em>N. crassa IT</em> whose recipient chromosome is chromosome 1, but if the “deletion of nucleus-limited gene” model is correct, then this phenotype might be <em>T(B362i)</em>-specific.</p><p id="p-64">Introgression of additional <em>N. crassa</em> translocations into <em>N. tetrasperma</em> holds out the prospect of uncovering more genotypes with nucleus-limited effects. Alternatively, <em>N. tetrasperma</em> can now be transformed (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kasbekar 2015</a>); therefore, a quicker way to screen for nucleus-limited genes might be to engineer targeted integration of yeast <em>Frt</em> sites into the different chromosome arms, and then use FLP recombinase to induce crossover and produce defined <em>RT</em>s. The [<em>Dp1</em>/<em>Df2</em> + <em>Dp2</em>/<em>Df1</em>] and [<em>RT</em> + <em>N</em>] heterokaryons emerging from <em>RT</em> × <em>N</em> crosses would enable us to screen two (or more) <em>Df</em>s in a single experiment.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">The authors thank K. Sreethi Reddy for preparing <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1, and B</a>. Navitha for technical assistance. D.A.G. was supported by a CSIR-UGC Junior Research Fellowship. D.P.K. holds the Haldane Chair of the Centre for DNA Fingerprinting and Diagnostics (CDFD). This work was supported by a grant from the Department of Science and Technology, Government of India, and by CDFD Core Funds (to D.P.K.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-27"><p id="p-66">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017616/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017616/-/DC1</a></p></li><li class="fn" id="fn-28"><p id="p-67">Accession numbers of nucleotide sequences determined in this work: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KP006652&amp;atom=%2Fggg%2F5%2F6%2F1263.atom">KP006652</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KP006653&amp;atom=%2Fggg%2F5%2F6%2F1263.atom">KP006653</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KP006654&amp;atom=%2Fggg%2F5%2F6%2F1263.atom">KP006654</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KR349720&amp;atom=%2Fggg%2F5%2F6%2F1263.atom">KR349720</a>, and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KR349721&amp;atom=%2Fggg%2F5%2F6%2F1263.atom">KR349721</a>.</p></li><li class="fn" id="fn-29"><p id="p-68">Dedicated to the memory of Suresh Kumar Mahajan (1942–2014), an outstanding teacher of microbial genetics.</p></li><li class="fn" id="fn-30"><p id="p-69"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-02-22"><span class="received-label">Received </span>February 22, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-15"><span class="accepted-label">Accepted </span>April 15, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Giri <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Novel mutation causing derepression of several enzymes of sulfur metabolism in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">140</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DBurton%26rft.auinit1%253DE.%2BG.%26rft.volume%253D109%26rft.issue%253D1%26rft.spage%253D140%26rft.epage%253D151%26rft.atitle%253DNovel%2BMutation%2BCausing%2BDerepression%2Bof%2BSeveral%2BEnzymes%2Bof%2BSulfur%2BMetabolism%2Bin%2BNeurospora%2Bcrassa%26rft_id%253Dinfo%253Apmid%252F4257980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTA5LzEvMTQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calhoun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">H. B.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1968</span> <span class="cit-article-title">Genetic analysis of eight-spored asci produced by gene <em>E</em> in <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalhoun%26rft.auinit1%253DF.%26rft.volume%253D60%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D459%26rft.atitle%253DGENETIC%2BANALYSIS%2BOF%2BEIGHT-SPORED%2BASCI%2BPRODUCED%2BBY%2BGENE%2BE%2BIN%2BNEUROSPORA%2BTETRASPERMA%26rft_id%253Dinfo%253Apmid%252F5728739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2MC8zLzQ0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.3" data-doi="10.1016/j.funbio.2012.06.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corcoran</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bidartondo</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerekes</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantifying functional heterothallism in the pseudohomothallic ascomycete <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Biol.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">962</span>–<span class="cit-lpage">975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFungal%2BBiol%26rft.aulast%253DCorcoran%26rft.auinit1%253DP.%26rft.volume%253D116%26rft.issue%253D9%26rft.spage%253D962%26rft.epage%253D975%26rft.atitle%253DQuantifying%2Bfunctional%2Bheterothallism%2Bin%2Bthe%2Bpseudohomothallic%2Bascomycete%2BNeurospora%2Btetrasperma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.funbio.2012.06.006%26rft_id%253Dinfo%253Apmid%252F22954339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.funbio.2012.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22954339&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310038800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.4" data-doi="10.1534/genetics.112.147546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Czaja</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Novel sexual-cycle specific gene silencing in <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147546%26rft_id%253Dinfo%253Apmid%252F23341415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzExNDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.5" data-doi="10.2307/2439515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">Adaptation in pantothenate-requiring Neurospora. II. Nuclear competition during adaptation.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D47%26rft.spage%253D648%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2439515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2439515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1960WN12300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.6" data-doi="10.1016/0076-6879(71)17168-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Serres</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Genetic and microbiological research techniques for <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D17%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252871%252917168-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(71)17168-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.7" data-doi="10.1016/S1097-2765(00)00047-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demeter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stearns</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The DNA damage checkpoint signal in budding yeast is nuclear limited.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDemeter%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D487%26rft.epage%253D492%26rft.atitle%253DThe%2BDNA%2Bdamage%2Bcheckpoint%2Bsignal%2Bin%2Bbudding%2Byeast%2Bis%2Bnuclear%2Blimited.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252900047-2%26rft_id%253Dinfo%253Apmid%252F10983994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)00047-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10983994&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089166100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.8" data-doi="10.1534/genetics.111.130690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nativig</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapidus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Massive changes in genome architecture accompany the transition to self-fertility in the filamentous fungus <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130690%26rft_id%253Dinfo%253Apmid%252F21750257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg5LzEvNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallegos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skupski</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natvig</span>  <span class="cit-name-given-names">D. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Suppressed recombination and a pairing anomaly on the mating-type chromosome of <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGallegos%26rft.auinit1%253DA.%26rft.volume%253D154%26rft.issue%253D2%26rft.spage%253D623%26rft.epage%253D633%26rft.atitle%253DSuppressed%2BRecombination%2Band%2Ba%2BPairing%2BAnomaly%2Bon%2Bthe%2BMating-Type%2BChromosome%2Bof%2BNeurospora%2Btetrasperma%26rft_id%253Dinfo%253Apmid%252F10655216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzIvNjIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.10" data-doi="10.1007/s12041-009-0005-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Neurospora crassa fmf-1</em> encodes the homologue of the <em>Schizosaccharomyces pombe</em> Ste11p regulator of sexual development.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgenetics%26rft.stitle%253DJ%2BGenet%26rft.aulast%253DIyer%26rft.auinit1%253DS.%2BV.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D39%26rft.atitle%253DNeurospora%2Bcrassa%2Bfmf-1%2Bencodes%2Bthe%2Bhomologue%2Bof%2Bthe%2BSchizosaccharomyces%2Bpombe%2BSte11p%2Bregulator%2Bof%2Bsexual%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12041-009-0005-2%26rft_id%253Dinfo%253Apmid%252F19417542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12041-009-0005-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19417542&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.11" data-doi="10.1098/rspb.1952.0046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinks</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">Heterokaryosis: a system of adaptation in wild fungi.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DJinks%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D140%26rft.issue%253D898%26rft.spage%253D83%26rft.epage%253D99%26rft.atitle%253DHeterokaryosis%253A%2BA%2BSystem%2Bof%2BAdaptation%2Bin%2BWild%2BFungi%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1952.0046%26rft_id%253Dinfo%253Apmid%252F13003914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMDoiMTQwLzg5OC84MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1263.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Neurospora duplications, and genome defense by RIP and meiotic silencing</span>, pp. <span class="cit-fpage">109</span>–<span class="cit-lpage">127</span> in <span class="cit-source">Neurospora: Genomics and Molecular Biology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P. </span></span>, <span class="cit-ed"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Caister Academic Press</span>, <span class="cit-publ-loc">Norfolk, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.13" data-doi="10.1007/s12038-014-9419-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Are any fungal genes nucleus-limited?</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiosci.%26rft.volume%253D39%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12038-014-9419-y%26rft_id%253Dinfo%253Apmid%252F24845498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12038-014-9419-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24845498&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1263.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">What have we learned by doing transformations in <em>Neurospora tetrasperma</em>?</span> pp. <span class="cit-fpage">47</span>–<span class="cit-lpage">52</span> in <span class="cit-source">Genetic Transformation Systems in Fungi</span>, Vol. <span class="cit-vol">2</span>, edited by <span class="cit-ed"><span class="cit-name-surname">van den Berg</span>  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Maruthachalam</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Springer International Publishing</span>, <span class="cit-publ-loc">Switzerland</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BTransformation%2BSystems%2Bin%2BFungi%26rft.volume%253D2%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.15" data-doi="10.1016/j.fgb.2011.01.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">K. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Carrefour Mme. Gras: A wild-isolated <em>Neurospora crassa</em> strain that suppresses meiotic silencing by unpaired DNA and uncovers a novel ascospore stability defect.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DKasbekar%26rft.auinit1%253DD.%2BP.%26rft.volume%253D48%26rft.issue%253D6%26rft.spage%253D612%26rft.epage%253D620%26rft.atitle%253DCarrefour%2BMme.%2BGras%253A%2Ba%2Bwild-isolated%2BNeurospora%2Bcrassa%2Bstrain%2Bthat%2Bsuppresses%2Bmeiotic%2Bsilencing%2Bby%2Bunpaired%2BDNA%2Band%2Buncovers%2Ba%2Bnovel%2Bascospore%2Bstability%2Bdefect.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2011.01.012%26rft_id%253Dinfo%253Apmid%252F21295150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2011.01.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21295150&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Vogel’s Medium N salts: avoiding the need for ammonium nitrate.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D50%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahlgren</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Hybrid strains useful in transferring genes from one species of Neurospora to another.</span> <abbr class="cit-jnl-abbrev">Neurospora Newslett.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurospora%2BNewslett.%26rft.volume%253D15%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.18" data-doi="10.1139/g70-115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newmeyer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">A suppressor of the heterokaryon-incompatibility associated with mating type in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">914</span>–<span class="cit-lpage">926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2Bjournal%2Bof%2Bgenetics%2Band%2Bcytology.%2B%2BJournal%2Bcanadien%2Bde%2Bg%25C3%25A9n%25C3%25A9tique%2Bet%2Bde%2Bcytologie%26rft.stitle%253DCan%2BJ%2BGenet%2BCytol%26rft.aulast%253DNewmeyer%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D914%26rft.epage%253D926%26rft.atitle%253DA%2Bsuppressor%2Bof%2Bthe%2Bheterokaryon-incompatibility%2Bassociated%2Bwith%2Bmating%2Btype%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg70-115%26rft_id%253Dinfo%253Apmid%252F5512565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g70-115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5512565&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Transfer of genes and translocations from <em>Neurospora crassa</em> to <em>N. tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D38%26rft.spage%253D84%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.20" data-doi="10.1016/S0065-2660(08)60311-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Chromosome rearrangements in Neurospora and other filamentous fungi.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DPerkins%26rft.auinit1%253DD.%2BD.%26rft.volume%253D36%26rft.spage%253D239%26rft.epage%253D398%26rft.atitle%253DChromosome%2Brearrangements%2Bin%2BNeurospora%2Band%2Bother%2Bfilamentous%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960311-9%26rft_id%253Dinfo%253Apmid%252F9348657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60311-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9348657&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margolin</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haedo</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Occurrence of repeat induced point mutation in long segmental duplications of Neurospora.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPerkins%26rft.auinit1%253DD.%2BD.%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D125%26rft.epage%253D136%26rft.atitle%253DOccurrence%2Bof%2BRepeat%2BInduced%2BPoint%2BMutation%2Bin%2BLong%2BSegmental%2BDuplications%2Bof%2BNeurospora%26rft_id%253Dinfo%253Apmid%252F9286673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ3LzEvMTI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1263.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source">The Neurospora Compendium Chromosomal Loci</span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Cytogenetic behavior of Spore killer genes in Neurospora.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRaju%26rft.auinit1%253DN.%2BB.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D607%26rft.epage%253D623%26rft.atitle%253DCYTOGENETIC%2BBEHAVIOR%2BOF%2BSPORE%2BKILLER%2BGENES%2BIN%2BNEUROSPORA%26rft_id%253Dinfo%253Apmid%252F17248974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTMvMy82MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Meiosis and ascospore genesis in Neurospora.</span> <abbr class="cit-jnl-abbrev">Eur. J. Cell Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bcell%2Bbiology%26rft.stitle%253DEur%2BJ%2BCell%2BBiol%26rft.aulast%253DRaju%26rft.auinit1%253DN.%2BB.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D208%26rft.epage%253D223%26rft.atitle%253DMeiosis%2Band%2Bascospore%2Bgenesis%2Bin%2BNeurospora.%26rft_id%253Dinfo%253Apmid%252F6450683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6450683&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980KV73900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.25" data-doi="10.1139/g86-136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Ascus-development in two temperature-sensitive Four-spore mutants of <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2Bjournal%2Bof%2Bgenetics%2Band%2Bcytology.%2B%2BJournal%2Bcanadien%2Bde%2Bg%25C3%25A9n%25C3%25A9tique%2Bet%2Bde%2Bcytologie%26rft.stitle%253DCan%2BJ%2BGenet%2BCytol%26rft.aulast%253DRaju%26rft.auinit1%253DN.%2BB.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D982%26rft.epage%253D990%26rft.atitle%253DAscus%2Bdevelopment%2Bin%2Btwo%2Btemperature-sensitive%2Bfour-spore%2Bmutants%2Bof%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg86-136%26rft_id%253Dinfo%253Apmid%252F2950989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g86-136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2950989&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.26" data-doi="10.1016/S0953-7562(09)80923-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Functional heterothallism resulting from homokaryotic conidia and ascospores in <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Mycol. Res.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycol.%2BRes.%26rft.volume%253D96%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0953-7562%252809%252980923-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0953-7562(09)80923-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.27" data-doi="10.1002/dvg.1020150111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Diverse programs of ascus development in pseudohomothallic strains of Neurospora, Gelasinospora, and Podospora.</span> <abbr class="cit-jnl-abbrev">Dev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bgenetics%26rft.stitle%253DDev%2BGenet%26rft.aulast%253DRaju%26rft.auinit1%253DN.%2BB.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D104%26rft.epage%253D118%26rft.atitle%253DDiverse%2Bprograms%2Bof%2Bascus%2Bdevelopment%2Bin%2Bpseudohomothallic%2Bspecies%2Bof%2BNeurospora%252C%2BGelasinospora%252C%2Band%2BPodospora.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.1020150111%26rft_id%253Dinfo%253Apmid%252F8187347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.1020150111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8187347&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MZ20400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.28" data-doi="10.1534/genetics.106.069161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Neurospora spore killers Sk-2 and Sk-3 suppress meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.069161%26rft_id%253Dinfo%253Apmid%252F17339226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc2LzEvNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.29" data-doi="10.1016/j.fgb.2011.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sowjanya</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Meiotic silencing by unpaired DNA is expressed more strongly in the early than the late perithecia of crosses involving most wild-isolated <em>Neurospora crassa</em> strains and in self-crosses of <em>N. tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1146</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DRamakrishnan%26rft.auinit1%253DM.%26rft.volume%253D48%26rft.issue%253D12%26rft.spage%253D1146%26rft.epage%253D1152%26rft.atitle%253DMeiotic%2Bsilencing%2Bby%2Bunpaired%2BDNA%2Bis%2Bexpressed%2Bmore%2Bstrongly%2Bin%2Bthe%2Bearly%2Bthan%2Bthe%2Blate%2Bperithecia%2Bof%2Bcrosses%2Binvolving%2Bmost%2Bwild-isolated%2BNeurospora%2Bcrassa%2Bstrains%2Band%2Bin%2Bself-crosses%2Bof%2BN.%2Btetrasperma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2011.10.002%26rft_id%253Dinfo%253Apmid%252F22056520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2011.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22056520&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1263.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rieger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-source">Glossary of Genetics Classical and Molecular</span>, <span class="cit-edition">Ed. 5th</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.31" data-doi="10.1098/rspb.2014.0084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samils</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliva</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannesson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Nuclear interactions in a heterokaryon: insight from the model <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20140084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.0084%26rft_id%253Dinfo%253Apmid%252F24850920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3ODYvMjAxNDAwODQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Molecular characterization of <em>tol</em>, a mediator of mating-type-associated vegetative incompatibility in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShiu%26rft.auinit1%253DP.%2BK.%2BT.%26rft.volume%253D151%26rft.issue%253D2%26rft.spage%253D545%26rft.epage%253D555%26rft.atitle%253DMolecular%2BCharacterization%2Bof%2Btol%252C%2Ba%2BMediator%2Bof%2BMating-Type-Associated%2BVegetative%2BIncompatibility%2Bin%2BNeurospora%2Bcrassa%26rft_id%253Dinfo%253Apmid%252F9927450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzIvNTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.33" data-doi="10.1016/S0092-8674(01)00609-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShiu%26rft.auinit1%253DP.%2BK.%26rft.volume%253D107%26rft.issue%253D7%26rft.spage%253D905%26rft.epage%253D916%26rft.atitle%253DMeiotic%2Bsilencing%2Bby%2Bunpaired%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900609-2%26rft_id%253Dinfo%253Apmid%252F11779466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00609-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11779466&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173024200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.34" data-doi="10.1146/annurev.ge.24.120190.003051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Premeiotic instability of repeated sequences in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DSelker%26rft.auinit1%253DE.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D579%26rft.epage%253D613%26rft.atitle%253DPremeiotic%2BInstability%2BOf%2BRepeated%2BSequences%2BIn%2BNeurospora%2BCrassa%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.24.120190.003051%26rft_id%253Dinfo%253Apmid%252F2150906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.24.120190.003051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2150906&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EN96900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1263.35" data-doi="10.1007/s12038-010-0062-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sowjanya</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translocations used to generate chromosome segment duplications in Neurospora can disrupt genes and create novel open reading frames.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiosciences%26rft.stitle%253DJ%2BBiosci%26rft.aulast%253DSingh%26rft.auinit1%253DP.%2BK.%26rft.volume%253D35%26rft.issue%253D4%26rft.spage%253D539%26rft.epage%253D546%26rft.atitle%253DTranslocations%2Bused%2Bto%2Bgenerate%2Bchromosome%2Bsegment%2Bduplications%2Bin%2BNeurospora%2Bcan%2Bdisrupt%2Bgenes%2Band%2Bcreate%2Bnovel%2Bopen%2Breading%2Bframes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12038-010-0062-y%26rft_id%253Dinfo%253Apmid%252F21289436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12038-010-0062-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21289436&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-thesis" id="cit-5.6.1263.36"><div class="cit-metadata unstructured">Singh, P. K., 2010 Genetic and molecular analysis of Neurospora duplications and duplication-generating translocations. PhD Thesis, Jawaharlal Nehru University, New Delhi.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1263.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafontaine</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The fungal sense of non-self</span>, pp. <span class="cit-fpage">9</span>–<span class="cit-lpage">21</span> in <span class="cit-source">Neurospora: Genomics and Molecular Biology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span>, <span class="cit-ed"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Caister Academic Press</span>, <span class="cit-publ-loc">Norfolk, UK</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1263.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1253" title="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1273" title="A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OHXaRj0OTAep3PYnvH0HTmgiMQeD9yG0L3SuzLr_6Tk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1263" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1263" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1263" class="active" data-icon-position="" data-hide-link-title="0">Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459965" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0X6BV162ousz7K6sBUbqzrqEWSfi0TJPWPpbnPEDftk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1263.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1263" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-y1rd9pTbRhzNhlvKrmyvNjfAh8TM0mIVHDBO5K8Ws34" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1263" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1263.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459965" id="node459965--2" data-pisa="ggg;5/6/1263" data-pisa-master="ggg;g3.115.017616" data-apath="/ggg/5/6/1263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Neurospora</em> Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dev Ashish Giri</span>, <span class="highwire-citation-author" data-delta="1">S. Rekha</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Durgadas P. Kasbekar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1263-1272;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017616 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459965--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dev Ashish Giri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Studies, Manipal University, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BAshish%2BGiri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giri%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BAshish%2BGiri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">S. Rekha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BRekha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rekha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BRekha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Durgadas P. Kasbekar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Durgadas%2BP.%2BKasbekar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasbekar%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADurgadas%2BP.%2BKasbekar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Durgadas P. Kasbekar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kas@cdfd.org.in" class="" data-icon-position="" data-hide-link-title="0">kas@cdfd.org.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1263" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1263&amp;title=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1263&amp;title=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1263&amp;title=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1263&amp;text=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1263&amp;title=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1263&amp;t=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1263&amp;title=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1263&amp;title=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459965/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459965" id="node459965--3" data-pisa="ggg;5/6/1263" data-pisa-master="ggg;g3.115.017616" data-apath="/ggg/5/6/1263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Neurospora</em> Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dev Ashish Giri</span>, <span class="highwire-citation-author" data-delta="1">S. Rekha</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Durgadas P. Kasbekar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1263-1272;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017616 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459965--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dev Ashish Giri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Studies, Manipal University, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BAshish%2BGiri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giri%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BAshish%2BGiri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">S. Rekha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BRekha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rekha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BRekha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Durgadas P. Kasbekar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500001, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Durgadas%2BP.%2BKasbekar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasbekar%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADurgadas%2BP.%2BKasbekar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Durgadas P. Kasbekar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kas@cdfd.org.in" class="" data-icon-position="" data-hide-link-title="0">kas@cdfd.org.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1263&amp;rft.epage=1272&amp;rft.atitle=Neurospora%20Heterokaryons%20with%20Complementary%20Duplications%20and%20Deficiencies%20in%20Their%20Constituent%20Nuclei%20Provide%20an%20Approach%20to%20Identify%20Nucleus-Limited%20Genes&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Giri%2C+Dev+Ashish&amp;rft.au=Rekha%2C+S.&amp;rft.au=Kasbekar%2C+Durgadas+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459965/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459965/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459965/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459965/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459965/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459965/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459965/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459965/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459965/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459965/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459965/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459965/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1263" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1263.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1263.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1263.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1263.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
