<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1075" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1075.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1075.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1075.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/6/1075/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1075.atom" />
<meta name="HW.pisa" content="ggg;5/6/1075" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Gene Network Reconstruction by Integration of Prior Biological Knowledge" />
<meta name="DC.Identifier" content="10.1534/g3.115.018127" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2015 Li and Jackson. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="With the development of high-throughput genomic technologies, large, genome-wide datasets have been collected, and the integration of these datasets should provide large-scale, multidimensional, and insightful views of biological systems. We developed a method for gene association network construction based on gene expression data that integrate a variety of biological resources. Assuming gene expression data are from a multivariate Gaussian distribution, a graphical lasso (glasso) algorithm is able to estimate the sparse inverse covariance matrix by a lasso (L1) penalty. The inverse covariance matrix can be seen as direct correlation between gene pairs in the gene association network. In our work, instead of using a single penalty, different penalty values were applied for gene pairs based on a priori knowledge as to whether the two genes should be connected. The a priori information can be calculated or retrieved from other biological data, e.g. , Gene Ontology similarity, protein-protein interaction, gene regulatory network. By incorporating prior knowledge, the weighted graphical lasso (wglasso) outperforms the original glasso both on simulations and on data from Arabidopsis. Simulation studies show that even when some prior knowledge is not correct, the overall quality of the wglasso network was still greater than when not incorporating that information, e.g. , glasso." />
<meta name="DC.Contributor" content="Yupeng Li" />
<meta name="DC.Contributor" content="Scott A. Jackson" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Gene Network Reconstruction by Integration of Prior Biological Knowledge" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;With the development of high-throughput genomic technologies, large, genome-wide datasets have been collected, and the integration of these datasets should provide large-scale, multidimensional, and insightful views of biological systems. We developed a method for gene association network construction based on gene expression data that integrate a variety of biological resources. Assuming gene expression data are from a multivariate Gaussian distribution, a graphical lasso (glasso) algorithm is able to estimate the sparse inverse covariance matrix by a lasso (L&lt;sub&gt;1&lt;/sub&gt;) penalty. The inverse covariance matrix can be seen as direct correlation between gene pairs in the gene association network. In our work, instead of using a single penalty, different penalty values were applied for gene pairs based on &lt;i&gt;a priori&lt;/i&gt; knowledge as to whether the two genes should be connected. The &lt;i&gt;a priori&lt;/i&gt; information can be calculated or retrieved from other biological data, &lt;i&gt;e.g.&lt;/i&gt;, Gene Ontology similarity, protein-protein interaction, gene regulatory network. By incorporating prior knowledge, the weighted graphical lasso (wglasso) outperforms the original glasso both on simulations and on data from Arabidopsis. Simulation studies show that even when some prior knowledge is not correct, the overall quality of the wglasso network was still greater than when not incorporating that information, &lt;i&gt;e.g.&lt;/i&gt;, glasso.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1075" />
<meta name="citation_id" content="5/6/1075" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1075" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1075.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1075.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1075.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.018127" />
<meta name="citation_pmid" content="25823587" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="5" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1075" />
<meta name="citation_lastpage" content="1079" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yupeng Li" />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia" />
<meta name="citation_author_institution" content="Department of Statistics, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0843-7592" />
<meta name="citation_author" content="Scott A. Jackson" />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia" />
<meta name="citation_author_email" content="sjackson@uga.edu" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Anjum;citation_author=A. Doucet;citation_author=C. C. Holmes;citation_title=A boosting approach to structure learning of graphs with and without prior knowledge.;citation_volume=25;citation_year=2009;citation_pmid=19696047;citation_doi=10.1093/bioinformatics/btp485" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. L. Barabasi;citation_author=Z. N. Oltvai;citation_title=Network biology: understanding the cell's functional organization.;citation_pages=101-113;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14735121;citation_doi=10.1038/nrg1272" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=J. Besag;citation_title=Spatial interaction and the statistical analysis of lattice systems.;citation_pages=192-236;citation_volume=36;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Cai;citation_author=W. D. Liu;citation_author=X. Luo;citation_title=A constrained L(1) minimization approach to sparse precision matrix estimation.;citation_pages=594-607;citation_volume=106;citation_year=2011;citation_doi=10.1198/jasa.2011.tm10155" />
<meta name="citation_reference" content="citation_journal_title=Stat Appl Genet Mol;citation_author=C. Charbonnier;citation_author=J. Chiquet;citation_author=C. Ambroise;citation_title=Weighted-lasso for structured network inference from time course data.;citation_pages=15Article;citation_volume=9;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.;citation_author=G. C. Chen;citation_author=M. J. Cairelli;citation_author=H. Kilicoglu;citation_author=D. Shin;citation_author=T. C. Rindflesch;citation_title=Augmenting microarray data with literature-based knowledge to enhance gene regulatory network inference.;citation_pages=e1003666;citation_volume=10;citation_year=2014;citation_pmid=24921649;citation_doi=10.1371/journal.pcbi.1003666" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Christley;citation_author=Q. Nie;citation_author=X. H. Xie;citation_title=Incorporating existing network information into gene network inference.;citation_pages=e6799-e6799;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19710931;citation_doi=10.1371/journal.pone.0006799" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. P. Dempster;citation_title=Covariance selection.;citation_pages=157-175;citation_volume=28;citation_year=1972;citation_doi=10.2307/2528966" />
<meta name="citation_reference" content="citation_title=Adv. Neural Info. Process. Syst.;citation_author=R. Foygel;citation_author=M. Drton;citation_title=Extended Bayesian information criteria for Gaussian graphical models;citation_pages=604-612;citation_volume=23;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Sparse inverse covariance estimation with the graphical lasso.;citation_volume=9;citation_year=2008;citation_pmid=18079126;citation_doi=10.1093/biostatistics/kxm045" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. Friedman;citation_title=Inferring Cellular Networks Using Probabilistic Graphical Models;citation_pages=799-805;citation_volume=303;citation_year=2004;citation_issue=5659;citation_pmid=14764868;citation_doi=10.1126/science.1094068" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. D. Hawkins;citation_author=G. C. Hon;citation_author=B. Ren;citation_title=Next-generation genomics: an integrative approach.;citation_pages=476-486;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20531367" />
<meta name="citation_reference" content="citation_journal_title=Journal of bioinformatics and computational biology;citation_journal_abbrev=J Bioinform Comput Biol;citation_author=S. Imoto;citation_author=T. Higuchi;citation_author=T. Goto;citation_author=K. Tashiro;citation_author=S. Kuhara;citation_title=Combining microarrays and biological knowledge for estimating gene networks via bayesian networks.;citation_pages=77-98;citation_volume=2;citation_year=2004;citation_issue=1;citation_pmid=15272434;citation_doi=10.1142/S021972000400048X" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=A. R. Joyce;citation_author=B. O. Palsson;citation_title=The model organism as a system: integrating 'omics' data sets.;citation_pages=198-210;citation_volume=7;citation_year=2006;citation_issue=3;citation_pmid=16496022;citation_doi=10.1038/nrm1857" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=G. Karlebach;citation_author=R. Shamir;citation_title=Modelling and analysis of gene regulatory networks.;citation_pages=770-780;citation_volume=9;citation_year=2008;citation_issue=10;citation_pmid=18797474;citation_doi=10.1038/nrm2503" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. Langfelder;citation_author=S. Horvath;citation_title=WGCNA: an R package for weighted correlation network analysis.;citation_pages=559-559;citation_volume=9;citation_year=2008;citation_pmid=19114008;citation_doi=10.1186/1471-2105-9-559" />
<meta name="citation_reference" content="citation_title=Graphical Models;citation_author=S. L. Lauritzen;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=I. Lee;citation_author=B. Ambaru;citation_author=P. Thakkar;citation_author=E. M. Marcotte;citation_author=S. Y. Rhee;citation_title=Rational association of genes with traits using a genome-scale gene network for Arabidopsis thaliana.;citation_pages=149-156;citation_volume=28;citation_year=2010;citation_issue=2;citation_pmid=20118918;citation_doi=10.1038/nbt.1603" />
<meta name="citation_reference" content="citation_title=Stability approach to regularization selection (StARS) for high dimensional graphical models.;citation_author=H. Liu;citation_author=K. Roeder;citation_author=L. Wasserman;citation_pages=1432-1440;citation_volume=23;citation_year=2010" />
<meta name="citation_reference" content="citation_title=J. Machine Learning Res. Proc. Track;citation_author=Q. Liu;citation_author=A. T. Ihler;citation_title=Learning scale free networks by reweighted L1 regularization.;citation_pages=40-48;citation_volume=15;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. Marbach;citation_author=J. C. Costello;citation_author=R. Kuffner;citation_author=N. M. Vega;citation_author=R. J. Prill;citation_title=Wisdom of crowds for robust gene network inference.;citation_pages=796-804;citation_volume=9;citation_year=2012;citation_issue=8;citation_pmid=22796662;citation_doi=10.1038/nmeth.2016" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=B. W. Matthews;citation_title=Comparison of the predicted and observed secondary structure of T4 phage lysozyme.;citation_pages=442-451;citation_volume=405;citation_year=1975;citation_issue=2;citation_pmid=1180967;citation_doi=10.1016/0005-2795(75)90109-9" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=N. Meinshausen;citation_author=P. Buhlmann;citation_title=High-dimensional graphs and variable selection with the lasso.;citation_pages=1436-1462;citation_volume=34;citation_year=2006;citation_doi=10.1214/009053606000000281" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Mostafavi;citation_author=D. Ray;citation_author=D. Warde-Farley;citation_author=C. Grouios;citation_author=Q. Morris;citation_title=GeneMANIA: a real-time multiple association network integration algorithm for predicting gene function.;citation_pages=S4;citation_volume=9;citation_year=2008;citation_pmid=18613948;citation_doi=10.1186/gb-2008-9-s1-s4" />
<meta name="citation_reference" content="citation_journal_title=Electron J Stat;citation_author=P. Ravikumar;citation_author=M. J. Wainwright;citation_author=G. Raskutti;citation_author=B. Yu;citation_title=High-dimensional covariance estimation by minimizing L(1)-penalized log-determinant divergence.;citation_pages=935-980;citation_volume=5;citation_year=2011;citation_doi=10.1214/11-EJS631" />
<meta name="citation_reference" content="citation_journal_title=Stat Appl Genet Mol;citation_author=J. Schafer;citation_author=K. Strimmer;citation_title=A shrinkage approach to large-scale covariance matrix estimation and implications for functional genomics.;citation_pages=Article32;citation_volume=4;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=J Roy Stat Soc B Met;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the lasso.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=C. Uhler;citation_title=Geometry of maximum likelihood estimation in Gaussian graphical models.;citation_pages=238-261;citation_volume=40;citation_year=2012;citation_doi=10.1214/11-AOS957" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. von Mering;citation_author=L. J. Jensen;citation_author=B. Snel;citation_author=S. D. Hooper;citation_author=M. Krupp;citation_title=STRING: known and predicted protein-protein associations, integrated and transferred across organisms;citation_pages=D433-D437;citation_volume=33;citation_year=2005;citation_issue=suppl_1;citation_pmid=15608232;citation_doi=10.1093/nar/gki005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. X. Wang;citation_author=W. L. Xu;citation_author=F. A. S. Lucas;citation_author=Y. Liu;citation_title=Incorporating prior knowledge into gene network study.;citation_volume=29;citation_year=2013;citation_pmid=23956306;citation_doi=10.1093/bioinformatics/btt443" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Wille;citation_author=P. Zimmermann;citation_author=E. Vranova;citation_author=A. Furholz;citation_author=O. Laule;citation_title=Sparse graphical Gaussian modeling of the isoprenoid gene network in Arabidopsis thaliana.;citation_pages=R92-R92;citation_volume=5;citation_year=2004;citation_issue=11;citation_pmid=15535868;citation_doi=10.1186/gb-2004-5-11-r92" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. J. Wolfe;citation_author=I. S. Kohane;citation_author=A. J. Butte;citation_title=Systematic survey reveals general applicability of &amp;quot;guilt-by-association&amp;quot; within gene coexpression networks.;citation_pages=227-227;citation_volume=6;citation_year=2005;citation_pmid=16162296;citation_doi=10.1186/1471-2105-6-227" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. C. Yu;citation_author=F. Li;citation_author=Y. D. Qin;citation_author=X. C. Bo;citation_author=Y. B. Wu;citation_title=GOSemSim: an R package for measuring semantic similarity among go terms and gene products.;citation_volume=26;citation_year=2010;citation_pmid=20179076;citation_doi=10.1093/bioinformatics/btq064" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=M. Yuan;citation_title=High dimensional inverse covariance matrix estimation via linear programming.;citation_pages=2261-2286;citation_volume=11;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=T. Zhao;citation_author=H. Liu;citation_author=K. Roeder;citation_author=J. Lafferty;citation_author=L. Wasserman;citation_title=The huge package for high-dimensional undirected graph estimation in R.;citation_pages=1059-1062;citation_volume=13;citation_year=2012;citation_pmid=26834510" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Gene Network Reconstruction by Integration of Prior Biological Knowledge" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="With the development of high-throughput genomic technologies, large, genome-wide datasets have been collected, and the integration of these datasets should provide large-scale, multidimensional, and insightful views of biological systems. We developed a method for gene association network construction based on gene expression data that integrate a variety of biological resources. Assuming gene expression data are from a multivariate Gaussian distribution, a graphical lasso (glasso) algorithm is able to estimate the sparse inverse covariance matrix by a lasso (L1) penalty. The inverse covariance matrix can be seen as direct correlation between gene pairs in the gene association network. In our work, instead of using a single penalty, different penalty values were applied for gene pairs based on a priori knowledge as to whether the two genes should be connected. The a priori information can be calculated or retrieved from other biological data, e.g. , Gene Ontology similarity, protein-protein interaction, gene regulatory network. By incorporating prior knowledge, the weighted graphical lasso (wglasso) outperforms the original glasso both on simulations and on data from Arabidopsis. Simulation studies show that even when some prior knowledge is not correct, the overall quality of the wglasso network was still greater than when not incorporating that information, e.g. , glasso." />
<meta name="og-title" property="og:title" content="Gene Network Reconstruction by Integration of Prior Biological Knowledge" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1075" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="With the development of high-throughput genomic technologies, large, genome-wide datasets have been collected, and the integration of these datasets should provide large-scale, multidimensional, and insightful views of biological systems. We developed a method for gene association network construction based on gene expression data that integrate a variety of biological resources. Assuming gene expression data are from a multivariate Gaussian distribution, a graphical lasso (glasso) algorithm is able to estimate the sparse inverse covariance matrix by a lasso (L1) penalty. The inverse covariance matrix can be seen as direct correlation between gene pairs in the gene association network. In our work, instead of using a single penalty, different penalty values were applied for gene pairs based on a priori knowledge as to whether the two genes should be connected. The a priori information can be calculated or retrieved from other biological data, e.g. , Gene Ontology similarity, protein-protein interaction, gene regulatory network. By incorporating prior knowledge, the weighted graphical lasso (wglasso) outperforms the original glasso both on simulations and on data from Arabidopsis. Simulation studies show that even when some prior knowledge is not correct, the overall quality of the wglasso network was still greater than when not incorporating that information, e.g. , glasso." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Gene Network Reconstruction by Integration of Prior Biological Knowledge | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"60ngjBtEZHp8ZLSZGv2d45zH87eHPfo4k1w8HQVZ0dQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459962","apath":"\/ggg\/5\/6\/1075.atom","pisa":"ggg;5\/6\/1075","ac":{"\/ggg\/5\/6\/1075.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1075.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1075"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1075"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1075.long":true,"\/content\/5\/6\/1075":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229gR92OjcHg819zClZdmVAlL1s9TVkp9BjNa29wDBCUxaHGXurqK5PsWTpgPNXQ\\\\\\\/K7Xw1zaoGlPhthilrSgk3MRkGSHynRlJmoM6aaOS7ukHmOWoQP\\\\\\\/39fq+OthEGF\\\\\\\/wLqBaenn9Im3BlBnH4AI\\\\\\\/8xqakl8VbQ7gWvvA3NNysTMWkBOi79U\\\\\\\/Bzeo\\\\\\\/yNdbDEhIaZDq80Iy0t5e0gINi2+5vrdG1T9nC5crAjq6u51McqgOkzcNsMRsjh9GZRK6Wsaysy1UMtrL68z\\\\\\\/Fhwr70FgffaWtrYgGYVuqCOr2vD\\\\\\\/KXL3T2pnaIVgcvZO2E8URtQ3BR\\\\\\\/gfDsHCNBDlI94GTLoayE13MsVWO1cEgn1P9n4g1YhAs+xw\\\\\\\/udPvbUPqVuCPeNh2eJ4Ta\\\\\\\/LcLu8u8oJeOBQItOhAX\\\\\\\/EJrjFCfxEtqHD03j19P7EbHxkQVCmTPI4ruwwGe+f8n9qNI3u6WYm1PlIix\\\\\\\/rWXEBJcyiB0kcucSK7\\\\\\\/22n\\\\\\\/fk8F0HQy4dphRguypPh1UeyXhihIfiAzs4qv\\\\\\\/OKCQ+uzSl4kPNsMXm57FI4HWpNkOdIPWz8NsBMYjKtl0QrG\\\\\\\/0UHgsksM45RHLNIwC1GGLCD\\\\\\\/quzE4jSyV\\\\\\\/X2rThreKh5QbEAaD74MlER0ceJM\\\\\\\/v0awGFBUHDQ4Cjjr0g7bYJFFL+Efm+lbc3SzsIHGpbSvo4shRW5eSBi+5jdrhvNBtHi4EgoEvrPxAW6DhBMmzP8k+qKQ2jRHCpxa2MQT5uoIJyE7IDc\\\\\\\/aJbdu9l6N7z4mUYTJtQ2IxUGssV58JHl8DvTqltvL10TrOqDuLGkLD80IRrgrt7WYE7VX6T0GiKAvqztD0AQpS7Wiq8aampnVdNPhtdijHG\\\\\\\/aDjuJ5NZIBvhQ1biaPsofngtRw+0CE7df3L4znwKAW7N4339ujSCv8Osmh60vCfKbTluds\\\\\\\/OJ5HxFSVKfTeNNQLmQJ6B7Y4P1y2IatdqKOaVwaq\\\\\\\/leZg\\\\\\\/X7GPKnIxmmhUjsKD1TN+3XC86aEmxbTSbMMx\\\\\\\/ffkh4GnPAUsMf\\\\\\\/JJaSXuZtU+\\\\\\\/FjaWZS8+3r15r3Zqudq912hXcMzN852mqftSlJdclmGjGDS7HOae8\\\\\\\/vBZII108W1fb9tWVRl8B96OjLU3DADdHHYBBK2\\\\\\\/rpuEbzgyhsiF7tiYRCvn8MEBKvyhd6vgWY7aUwrSa48u+VR160pkeyHCufuZleAdzjkWPZkj2epN4JOyG3fSI5CEpdz7B5OzXuAuTEiZcACZgTTvY6PdGmRpd+e2gDPEKOrnFemwBdPYtj\\\u0022,\\\u0022iv\\\u0022:\\\u0022z3rC3rWgMm5oOI+K6ZMQ3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227b798bba7082193282da1d4b6100b933b7d1bdc83f88473d177e2cc81c499106\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227cuGS3Sqx5DXqXyhjnUAyfRpYN8jIa9ZhljGZLBDD64L10Q3L4mxqBmZWEHCJve9z5Niu7uN5QA3mig98P8DLDt+SkN0hzldr4EJXSHXssco7OtceIM15a\\\\\\\/a9uFqnH4a4DaflkodgL+XCKrU+U\\\\\\\/vmV1vCbq9+\\\\\\\/K4DlSD4KCPP2z656CE+Gw74BsJQs4L2VxJLEOZfM7ouj3zxpO76WGZZcDFZYbn\\\\\\\/mtk6IG4vi7d3DN1yDpe5+MQj5xLnOidEyStKQFiVBjnhENItbWfURakSIdWcPspDjqCDgYpgwMN9P7v7N6R+tT0BNzbFRRJZtVOQ+OcLsODYdNaup37AvX1PVma13zpVYd5YJGYfkTnJWKRsaG1LRnrYVokau7m4MpaHisHGnjYZu4m8cKSAjWcjfJwdQ1iJR7DugphhOJcA+9ruj5WskHBMFKHV49YwGVmyT0elVfzIXRxySIxMMiuLqjaWc+I\\\\\\\/UXmSEG2050gsh6uaO27NSnyXraKlzpFFFtrVTjWox+jKG3pL5rga3Ne91Jb\\\\\\\/2ppwxbJvl9wa9uwI563OSCATHv2Oo4mGXmeiO8X\\\\\\\/XTjkxpCzS6NvIDLi3vvuAwAYX2n2cJ10XXyerunhfUG2r2BEBCckfqFLthfMpmLPJhgomwdexduS\\\\\\\/qopUWhtMq4QfkFsl\\\\\\\/J7UM7Oo+szjXRW96e\\\\\\\/BVakK4WVWx\\\\\\\/oHpCeiD0Ta5E3AoEUYprepHwS2Gv0CwrbVcTksFJNAfiW3tqk6DHsvevBJh0Beof+aFzWOMrONeiV+lEjHFwkeRqhfKZzqNGG3GvcpYeosyZdbqJForHuVzqu7BO0EOEbd+OoD5oQCq+4HLaeSemZ9OrMyITpNFfzKqAZ8ikO2lkgT2fRVhjONbxW31qGsYEE4+5g4hjNsoMYFVdNGUgv9G4wGqLwybgaR8ifvuLql2te7VhgIjdaVTkgZgiskWpIsdAL5ryMiEd1jYNIGflwra6fthv38IhTJVrRYYEKWUF\\\\\\\/1K6Qw31ljnrZjLBhHhbCQbL\\\\\\\/jMTtCQkfpSAgRORKKLuWrh7uWnuqBj2Pm8XUe8B1of5Gce5zpc748nrOWUgOYHW9bFA9hlnNC2bCYzMuLmYX40YpClEnq54B4MomT1wJh1E5rE7ganbSKyVQ9X9Qnog4wbrk7dU\\\\\\\/Ma7vsgfTgdpEIjS\\\\\\\/owSasdqDnuFzMGafsRkgByszzLwwpEVUoekjnD\\\\\\\/rPyGiKlQm4MT56XZgqcXKXNGLcf83rl9bkGvmQtkRwMuLWHz43n6qDZNq8u0Z4sRBHKw+gFZv1SeW6n+q1putkEBt0IXzrFNG2xJCUiq0pLQZto4QhSO9z663NjiSUoL5wO+\\\\\\\/eGLCv7NP\\\\\\\/o+iHHlHZBTrt7dTH10Za9G3+D\\\\\\\/IKjMX0QNa707X6MXw500d2IAPkYfeCOh3+7ePkSop2DnYpxTQlINzxwpvRSg\\\\\\\/IuFqFGN8ziZC9OmUbXTIl6xzVmECoa7drpdD5i3MuGTeIkVK7raeXp+vxa20nQWWwlhZjPlN2ZA\\\\\\\/a8jhWG+TIGlP\\\\\\\/T+cXeYvtMloxUW53rY47+NdfPy3i4EaLIiwwGb44uPlnTOHmdhTjAWjTVTRHCdgxcgsFy2J8wTIJuNWrT+R0D1\\\\\\\/Csnq5h8bNj6Wr7bon+MI5a5NLItqfT00zTqPQ95iohHy4DpPnbv+TpgoZ1alQDqNsxkJcMPJFLZAeS5c+KJLLYxDbVxnJ3fAgm0KNIhYKDg9NZKomngurNc\\\\\\\/HBTejT1FV1VGgj5\\\\\\\/cOOOz8a12kLKy3jNAJs1FhDN2WLfyyvEhlpTzPuNYBq7Qvnc3DV63DDQYq9rRetW4MTBxMzsctYSM0dNv8A\\\\\\\/7+SSGyoykSOzCyAARLr2FUlnqoi0TOUz0P6ErxY1c56mVhHll9fiiFECfIL0vJEn5\\\\\\\/w1PACGB0GlY86C7cEattxArtZ+5lN\\\\\\\/hEyz+XZaTW6UG83obeGcXp\\\\\\\/QmGxOMsE\\\\\\\/ReEI2nzKaZ8\\\\\\\/EyIPCOask5wUfXTw4nbznxlk1omqrCCgTMUav4OGO3ayCPqG9SyK7ZeaG\\\\\\\/4bjv3WV292wnvkU3fUHgp2Hh7bNtz7SAH1ZHwqO6Uej0XrUsjE5Gv8daLNmBJ9M7fRRnal0sUWl8k042MWwHphXtsMT\\\\\\\/MDzyYw08idpFl5+dMYrgoMiLFzvHV4xLc\\\\\\\/sbMVRO+q9HkfBM6KgjRkA4qBWUfGdqf1J0f1I+dQU8w9Oa+H+SAzF4Rf7FNSq+OyeiupR85TVj3uUXi2bm8KyZ9Deu\\\\\\\/juE\\\\\\\/aUokoN2nKCqGrMDKajjW6rq8xVjwDIJYlPCsrzgf4gG38J3bQcFFIZcEM6sB3cd880nO59K1mU7TuY1NqRca4LlP3VAlYem95msiaxKj\\\\\\\/LrW4iSCt+mr45yW8B98RaCJiKSGtzzC07RFv8VfvFyZYFx2\\\\\\\/M8I4xqM1HQlUsfhv4gB6y9VJaaO4kKB6\\\\\\\/5L5WsZ79oOqvqTYCG0EfqClTwbumaTf00osfbXbZ5IXMpjJTzO8M\\\\\\\/drx7I7fEDNCibha+39QJ2rOvinw8ksvtXHO+QhHr9gHYsw\\\\\\\/TZ+RgsEgF\\\\\\\/01F1Iy5zaGNej3KbXKqypnuY0Uv27ISJFD3mDasa\\\\\\\/xIehkP2PnXkOc6DlKrvs+b\\\\\\\/3QdTttDG7tXN3TYSeRjAkmBqGN9ZzvT8Mwqtd2gNQgELe135mk6tAaJ2qHwhdxh9xsOE\\\\\\\/N6qQcevgQz5N4h6DfJb8D3EG3Csst+DOwYI7Qhrl3Jf31qqR8+cZc36sWT4Sk+e+r6e4gz4FzPLvX4yzr\\\\\\\/sNke2S\\\\\\\/8c\\\\\\\/Ztc0f9FdVWz+\\\\\\\/Li5ZXayWAFA9YxScQ5gvCexyolwg+lG5mH5o+xfgKczT3ifmrL379HBJZVy+TlWzqBoQfhbs+0UM\\\\\\\/wvjPPCXBD1b1\\\\\\\/LUllrt4inIBfEb9A9aKLY025vqATmvc12a0+MqKn4jdIdtu2IxMgeV37TYGcLMK90Z7P7ZH1ifO5UUxLopLZ4IcIHtE6YOwAzF+iu2BOD5SC3eXbu863LV9Oz6yA5\\\\\\\/rrhwc3Wk9vFKIpL5\\\\\\\/LrWC\\\\\\\/4LU+R4elqSdepCNuKZbC\\\\\\\/jejNnWDHHJFcjKg1RZT318i50O9AqIYPPKpf2DBjhbbjEN6kT3+RD+omXiNvuvtbgifvvHerO5pgx4C\\\\\\\/qV4v\\\\\\\/enAZATDODZ0nhFlQzFtJi+rkVeysPz2HQE1HoIXxHbCyXx09dGwRA0rVMnicPiqqNhqNc8EnS1LmSOwd\\\\\\\/ZAo3xQ9Krsk0DdtGne6cmEf8O4sLBDUzwDCXYL2lsufhg2v+xP+87hbTHmnP7PvwESwzhdJIvOzso6LG0qKJSHaLoUIheEco3ZFh2vtKFCHZi+yI+QKMSyQhvAiq0iOlWodpNAaX+wd0g2smIxofmlwJXB3RXUHiM5hBEgo1S35mbASRc9Gbg9DgnkNUlU5MOs2MYv3jDFrbOZoa9xP3SLcLXShphfsoefFXF1K+ipk\\\\\\\/6\\\\\\\/RRD\\\\\\\/TTbUbfTw0DzHExXii9K\\\\\\\/Vd48NDXmUZn8A1ozc2BkGzx69cjH3+lRn8hzqG0Oc0wke2SHZMd+WWtR2Un43mfkUdss4kZsJKk54QyJbdvo7Kz9rxFZb3FnQjlX75g7TYMmlOlvO2qI4+jsd1RVIAhHF3IHfiVmTw1u8vGjYghVqqfdPPbCyHK8Zf3cLcD9tRJwA9LxNU25nndR+DIx5eFQ+PQU+P6oTY6vGgJwhdhytf8BgzJSkZImdo01r\\\\\\\/UB7VRxXpcXJkMf9Y1elmbkCDONJMzqb9azTQR+1Kg4H2YSE4c8mhyIKP0DZ8+GZcONE6c7Z5EsaCYR0qkXcYNcgFxMif0YiaeuxVQ2bpR87o\\\\\\\/TIStwjh+7FfSu\\\\\\\/iTns3iLGNjZg2Ql3qrnsfT77hY0CvIVSzlgIqb1BuReOgLM86rwGgHkFRf8OjiRmac3UugKrlTIUdfafYR3SLlzbSJWjmXpH2LYHf4KLpl6G4uKZg2EKKfKzb\\\\\\\/oiB8fuC8bK+mrRBA96V241oIgY4Q7zHmfcWtUyAkaI\\\\\\\/LL5KSQmq\\\\\\\/Md3JzrjofT1q4CjATa8sJ67UemsToVzvPYqQRm25cRS7cN9O8mFTBjRC+P4hA09L+ZhybuSNbYuYZgDl+kghBhG\\\\\\\/RN+55BbYtuoPD3wFAM9N5HqwjfDFIzEfuzVnAIjs6qc1UCAnVTnEke9EqDDOjotGhFN1LXIgM7jn\\\\\\\/YjxZHnNX2b3gthn6rVnjDSi+02IYe4spPZAqGBEPwgQf8iLNhVziNEf7kD2xE\\\\\\\/Y0EtTcgYn6Al8WrRqgs5g6E+hEcU3hBfIIh4qalHpO6e3E3av\\\\\\\/BVh4lxaKdOyHPwgMxKoMns65JMHY7fJV6+8ngn46hZ1iwGLs6vzITuhGo7Cguj0XgawPbEkKempbsYAFCs2DwlmAFO7AB3IUnch4fX0aKgVDvWxlpEjPvX1dnp10NdsxkMTxn2o1GUCeiYFecmtSUtW\\\\\\\/vUgcKBqRLCI7J0osYNGve7rgm0vuLooId3A5QTyQrQWPHHXI2UzXAvTi2TC9uJ2nFo8OvtzcRqDNCYHQympk1DyhZY52bMG45G0va3aUw8XLJK6A20uqATD8MsUpOrsuC6J3rJVrG\\\\\\\/QPUBzzxwyLEahHFfucOiOnqkie6eN7w6PmuKf9T+F4huigH2lngVsrUfEZ\\\\\\\/CyA5OmRuZfVbESXijrDMkEs+deeHoGZuiKmbhX2T2IYnVgxKxHFpCU+K3igOo0zz3OiHOWjA00sGp9R\\\\\\\/dOzrVbfOnNG2za3sJSp+RSBS5YDY7sZ1A1qbJwA\\\\\\\/BDQFFceD9\\\\\\\/nQj74ejjYwJK56Nea+AO9J6DRztklW4TFQ48TP8oAuZ7CwJ4BRMegBFcDkzlYAdBwkjWmDjXA2h3NgPdkrqPpLy4gu5rl8CDv6EgSQHDKkrq0zN+B4q9jSMxNlQCh0j23U06gxAtDSZGzS19CnUAZgvQRgNvKG9oJRvpxxyJ5Dh+L\\\\\\\/RxKUOownkJrmMp3ATpItjL\\\\\\\/9mVZNLvGvhT9FIVfwoJofHjYdMw1DazZV1IJxLNJUmG+IHP4Gs8mRD6gI0wEqswfILq9+NpOutO2E5ZHKBtAPjmXKyqCn6C6rTpdPvycTLHa6a4oG4RCQ2VHwIxR0ciJg1C6KlNzQmEr5wsbRnphWcMo7\\\\\\\/yg2NPXQFuv9GWHIdzLMv4QAmt0NA5dOLEVflLfHLiBYD7wWU2OOSXnFa01Y7t6yYOXaQ2kvtGS4G6plnaB9\\\\\\\/ecIgCp0nndFgIHLXs89OJTxrbyNIchyzY0rgbSmayacba3G8D8G27RXqKI4yHpFPogPR4KVrNkCjr1K1tQ32Se4IvkJzgn5wQYsb02w1a6Md8DaQ80AuOQCt6oe7vx0QZNNFMWV9nAindrJPAeqFxi5eC6bnT8d7J8iBWsXM0stXDaT0GMBL8JnlTibFc2iJBeCxqEbv4h3qk51CSy\\\\\\\/Dgt9aSX5+XC1TMuDg\\\\\\\/aBb1jhPLyhgeIrbNB25wxgW3ch82nK0DinWbEaHOLvQeVUelBxribCS6cTv\\\\\\\/EwwwR1GZWDg0yyipFKENJgKTtTyRARQsp8Pfy9OkGJarcLLY50xoeJD8H8WQCxe1d5Xd0F5jHlMH4qM9xpNNhRFDOBGWUUu3P+zKbkIY82enr+hmLfqL4DKDsp7H4VdEHmXF1K5SiMs+lyaFUL+M3Di\\\\\\\/Pe\\\\\\\/Ykzw1MxUtPkwKjK5N8YGoJgbGPjrGxHs9C9BOoHd34\\\\\\\/8e1dR4RKzd8Y1NpFV3sr25rplPCoCLJT6n86ZR9HrpFqEeLKr2NmEb9nqDOdYWU5M8MPK6lG56AsDW4nnqEfIe+zQ3TbRHV4HYTJ\\\\\\\/mhcXoughcCKRUp4rY1IkgGso4WXfz4LmKou9obkVCLJEIks5OBh9iblMy7NXuVEy+G624zfiBMIPCC6NcHbrxMixxXr4i9UlvAtiJGQ10LglrOn6Umczu7kGiSL+EmijX70HyA02LM4XAIAgiRTvYQ\\\\\\\/SD35hwqjEmkaG+uoez4TUuveVB0FZVVZ7v+VpIf1YOgjcIrZ+xJsImxD0aoi65eKAu1nM3ijluh7qTEWOwhidU3worEctQXwPGpQD4AGZyper07rrj+OuIAonAl\\\\\\\/+tQ2HNTXs40qKxbsjuBPIEHbKYap2wx\\\\\\\/KuSaY\\\\\\\/2520eCWa0\\\\\\\/OVFWUFiJg2od7\\\\\\\/zqe2IagtipO9RNxvMn0f8IW1abX3fpz1M3GBKVBdvqz3JxYP8Olj\\\\\\\/k5e5U4bXoCKIf5BqUygUmaopW2PdXbjxnPVcOoim2SFTcu36zTtZSN3Wt6U5QAHG3htsU0Bvmg8QmskkJbHhSO20gxmurGtkmOpzrYEHu55Qg1IoegHFzd1Lk5Cpi+mZwbOktXkdLBQufCeeqAR4M+qqRAMyZ+OmOq+t3+opOn\\\\\\\/1TIDu+5skKway7rIXDys+RXzYwAxU4mpJF5hApRV+cutG+BsaRCw5n0A1MEdBgW2Ao77Dd05a2kySIw9j3cCvnXXH10vWlXNQKy5DTbLXcLBWclrUvq25mH7U5l49PvfcHv9LWjbWrpVrK6ttBITMLPTLWtn3dFSZXJ6Xgg+\\\\\\\/npZYJHYlado+a7s3F6i\\\\\\\/5agAi21iZy31kubTQtVi0bpdwpA701J7W4aAV0ukE8YD+5YG90kFVRaxm8p8LbUYejIdFuDBudTWUjCjfkrkkI9nXsziTwtH0XW6swGN0aXT1UtcKIqHwF6586G1ZMp29vu4d8ciBXEdVDUvMx\\\\\\\/cXYpr7B6GHDKahqKQ8Q3p8ROFBL06YxHxLfqKDDmejDtGNASwEFwt\\\\\\\/SIRJRKpBf5eYlwhMwA+gCT0ONV+BxVXicgduT1cC9pSG2jrwapO7ZnhEA4n5S\\\\\\\/cwqzeG+GzUC\\\\\\\/sPpdB\\\\\\\/pXrAmOatZw+3JYCIoWYUQz6\\\\\\\/RmuGrWQI\\\\\\\/viWlJ8j0C1PGDC6srq+Fxm7q1Cj1KEjhHvkWS9B6SmZ5lyGY6VOqmWPhRswn42EwQLNXSEwFc6ISfq4YEgl+mgZZLipWvZldsJX4oju8BZnFsRVjsfbRmy7Iy873KuICivv2hKTI4r1VLT4v9KTxpyuPLMoY2i\\\\\\\/9r0Roeb+ml\\\\\\\/9J4is4JPIKkUVYghYnFtaYL1dQRLRGRUrnVSEpqgHczdSzWRcl88yYShmdvpozFzbg3xxMKfE0sq+RM3tbbH3AEdmk2gIDzxo15ldauhcr2aYu\\\\\\\/mamXl+iZe2+QR+D5d9qrDDAgL1aMNbKTzIlE8dw1lOplmNQ1EZui2NcKcGjwXOBCFvd\\\\\\\/j9g7laLdN1\\\\\\\/qzdi4tQEKFweSW5Na+aR4GTZOK9eRHPtaon\\\\\\\/1l5zOh57HHT2fxHcqgNTb3z5NquS0enYsE6rx9NUzPIHxYokQ1KzdBJlND2ryQm2rRguZgxjAN0bY0c1D7e8CtvF8cTC72lxfQynLTSww1wE5CjKrXJMAJQ28IwYnr6WYOUqwIVjv48OqEkrboP5gtZAQe1ihk5vX1k86BVpz+g\\\\\\\/VdAriUfbTQE\\\\\\\/\\\\\\\/cJFS8EPSG1hNAskcDBE3qcVpbyMbb\\\\\\\/bH7WwNgq3+drrxoJEkSRgCV7sGaVptDeXjrQox1639QaRIYiqD6vbbZFS7hYj7AkfPGEeKnr2yrM4V2FfX0U\\\\\\\/Gg\\\\\\\/b\\\\\\\/pf821rgYF9BtViVPPCDvaQMXdCfbbUVq0CC5utBS7FNie5ZjroOfPKDgbGgcorRvgg+q5\\\\\\\/ql7BX868PYHkrRhWBhyDiKUBLyz6C8\\\\\\\/VV7bmkfcnCJTYO7iNwwLSKeLR6blQKLyAInbxTfUiHCQUkFL2VK\\\\\\\/iWDvKbG5sAYdJBmCK1BnbarD8opxS3Qx7OODy+Xys+obl09UYlFTZoKrMxnTgZ8oP84naPmt4hrdNkt3Lbamiu+hQrZZqB8se+tWcqbbyB8F0QFD5XZYEqorvK9pvD5aC+62pKkdg5mADeB7WmaMi+5nDD6dG7WDoPAxpuicdyoKyffoKCe\\\\\\\/O2IJao5LfTbU+AUn5M26M2pbq3E6vaOyKw\\\\\\\/csrobgKHYTfgdXL+Hgl0B\\\\\\\/b1xe9ZTgUGor4UfQKGU2xaayVBk0BwwTdejI1e69eRf+sThv7MKJ\\\\\\\/xlNVRJKUFtX9dK6Vfd\\\\\\\/RWwICSPmG3Gb4MhxCL2fm+WtUi8gm2y6GzS9sVP+vdlYfDbuUoIns50+BNrC++J+4MN35iO9U3vsKWfmznxi\\\\\\\/0h+WogQRSEEq30xThhx0VcPtKiEA20vQJ4WB6PNjsc\\\\\\\/ztxzLqtVbSJWdtjavNzBq0W2pvBFi28XOYaLm0LBuDijWvxTD1oy5OcFPZ8ooXg1lSOGgPHmSlyEi51asofapGiOySF3ui0ytsoFO5RfS4w7Jbg9bPEySsAJTuifEkF2evFqcnuWVNyLi+WJGKmI1SwY\\\\\\\/\\\\\\\/mXEA4CiUxBOhhKbwn86vGYo7VSs1kFyFltgVDtEMCItQZZb7SnhdaWaMmCUBz0hgJ6STHvOcfFdgAkcu05t7zC9H1GvjPfj4pSZXAllMkkw8K5tq9tQP0mvsqIEQxzjyv66EsCgrkPkFQPANu7qJBvYLhUGztkuX0sIpqkgxkAg1OTK7TBT1XTD8A53LU6\\\\\\\/bolz6ZeXMD8JySpXFjEB5YYEN3J3Ng4ycioSA\\\\\\\/jXSemzdjKtAwxnf3wfahjq6YnkwaCQm0m\\\\\\\/vNsHsiF9faOYftXH3qZNNE89NHFrSN75umxMPrd1Tw1X5m+wFEdJTujrpYNpM+Pgv2HfDBXKOLQP6puWa13\\\\\\\/mn\\\\\\\/fpZgwbITI50aC8pHPFLmFbZ7PLZOmumc6XALvch+TVwYPrZtHncE2gZhHEsD\\\\\\\/1+DYZ+ZXLUAhRgUpVVQ6MlBEHDgj7CqNOghyV0zopMxzuJ308W\\\\\\\/NAvDt0CKgpiRVXsaBSMOVRnLunqQ6rcbkcsw4AIYWNIibACuShHidpWY0Kknz0FYTpKSFdlN8KTg+9w8qKPgmXOJFsJioewn6rKFwIDYBBvaopkl+JDfFzgNVMutsZshg68MP0oeccjxrjpmY2ueWLqMT1+B+YMHnXfaKw4JR0ZhvLq+N9DJjSOFpkxdYuIh2z8a0XQVkg17eJ\\\\\\\/Sj1D4+CCdANc\\\\\\\/xdHI1O+OvcvwvuJY1SyM4XA9Kfzfjw9wxP9yppk8o4Kh6pYtMOatX57CzUi4nhPls5\\\\\\\/u3650YjbEPH9rLEC+HQY26Y\\\\\\\/ntB2jDxScOanileVjDoqIL7Ze0owpaYc+S9qH+gNtWEQttnpXuXd\\\\\\\/riTvvcDLu2G6mdXPf5teGF6u4aBnvcDIkU28GU8hMrhRHFqosBikDpj7bG25nCtOYs0oVtG26u\\\\\\\/bH4clpj0CX\\\\\\\/bKvhVdNhJ3laS7KHu1yzQICdI9GFwM7UuC5BjQZ+X5V43UeAomjKKevINjWrNCQNYsbfX0VWBXTRzaZ9jDvqRLpgGTh3DNHBpHkQfhcCF0fJNOW3a\\\\\\\/914j7iEEa1o\\\\\\\/aFj81hYNkNbfs5FjoiQbfvFp+3wGVVe\\\\\\\/m6DQpSXJ7EmebuP+LD\\\\\\\/AQp1GBu04eVZsOCmSeU1dvpa3yWUfPYnRKfHS2ufPwFfRqHarDGV0OcPV414yOErJ9zlorvpIqWpMj2EU9X42hrPO+tOWTxTWUPx0hRK\\\\\\\/kea4pFPY8txgjAsjFfxOuVR\\\\\\\/q3R1UQo8uiNtjaDiXybJJ1g3mzFQyjUeUbFude55JeRkbiAe8+LvzpytmR1ulusiF+OMHaRbJqfO+iuxXNhLhXtatXxNEVSYwcWp8G8lqvzgQQCi5niPSMsu9NYO\\\\\\\/OXMW59ry60QK17mEg\\\\\\\/gt1Kc+IeQOAT8z9jnqvmuD8a5U75hyNZKx85rmQ9mXuYG53RxF7LjwSOb1RSGCrdVA9zf7ZgFMAsKDXQKqtSR8wgErGJ1vV7thCwdYHJhC4QTVfVzUZdr\\\\\\\/bz1O1IDyD4QjTMe1CX903Sr1Usi5ZStcDFtTHzMrro13yj3gGN9TLeYzRay6osWTSMqSwNVZrRBLXM2vga5l7zB8i1EDIRe0eNVGYKZhaWuPbyhUouW0iyoBLDP\\\\\\\/o65K4rKNshjWAmwhyOAuJwaqhNMpO9xfBeClo5cMTNW9y6az\\\\\\\/WQGTXO9e\\\\\\\/ZGFt6GXNY+6Qj2T6sANXADtB1aP0iMK3pifYvI2cRpD5SAXX+i4trPXCiqlliCy2Fj25Cjn0rU0FNF9RRwjWkKiK8jcZyTZhPAQctp3TpmterkbpwJD0sDDL84fdtotcHpZT3jSBwgUvId9iRrWkHzXOK4nJ0fH6Hy9SWGBOkI6iuxOg4vE9mDE\\\\\\\/bhvb3Ks5t\\\\\\\/O4tpFRExEyx68+sjRGJM2f4BU0l8WzBNKOoSX9ZdvOmOp9Jh7rqqFT8sda2GmZssDf3YOurC8kZi5g9Hc9mvSy6f\\\\\\\/hldx69y\\\\\\\/gzic+cuafN9XprvlpwJZ3OlKAv+MEWWYXPjSQ+nVOVSm85b2O+f8lcU0ZkbiSmodtWAUgHUEXI9ccNfzmRgoffwIG4u9E08nwThMtiC+VP1i9uvI0r\\\\\\\/WdWHuFKGZwAqV7VwejS4GfYrd9b5WBknYVFpDy38QdzrvCIo1J3GfxlhDQURnvzVAEQTtj2GmDyS\\\\\\\/vs235ZryCS98asBQOb1\\\\\\\/xuLFrUH65txfD+uZ7keyOZz2PYcCPRcEYXq1TmU2lYxBbc3eTcXx+a2Al7P3dzpMeqsi\\\\\\\/CXw74AXvQwb3LEWYYIaCKF3HQN8zx2QI94GZtlxzPOdzRn7wOY4RuN+PAj18jwjrhwNG4tpFP0SNVtkN9D2Q+i9V2mHv2wsiVVQAcwOPdgycLzf7E7EkfOqoduie\\\\\\\/A0pdB7IPjrJEpxU5Ev4eDUKqicmGT9oZjIVJsP9OlD98KGkVXSIE7Zqdgz1z02mEVl0DiO+t6zJCYDAbYQ28v0CkugUkQAskdCLjykdcNMZWdhgLcy391CSeU6wRhrHdRyJWBjug5V2+TbvY2Ysxi58rfr5lfWCrbYSXPQ9zxWMDkMExlv808Gq+K\\\\\\\/ZVLwsQVRvYbyEGNcddpKvk6jvtGdkiPPl96PeZtw46p65hEuQQTXmgNgzON57EmLfT5ykMSmfyqj3UQ9BVFOC4mXW8jj+mjchvHHmDEkJdPG3U6j+LiqqbyujHdU3wtExWr0YfHcAObRjoSQ4oWJVodFmewy51d4NTr0Q\\\\\\\/70AuaQokpS1Q0mQMvcoJyB+1iX0twwGRcQgE8iI9cqQG+lURb3Jv0aIFEVxJpICo2kUI4w9jiHxV6lXMp4t\\\\\\\/R3nE+u3RzU07p8OLLW7DE67jxvPhHKeZe9mUksQDM2jUs8M7Ahl3Ne8AT+\\\\\\\/vM95hrLAlh8TngawspNOENYApSBAiJEi4exlLEavJpibqT0rUw979xDyvxr9CSiTvLMNSCcjaPd0wvInVWwuhLupkyPGEoMQScop\\\\\\\/A806wvKLmTZSusMOS8BQY9x\\\\\\\/g7QEHeDEMZ8K5uaAQoldHbxnbD0dOUnSdsluR\\\\\\\/at0+fUo+Ee8sb0jq243LSLjjyvPBtsnMQ2nTshDE5gXqgquWY+poigzz158iBrsttMyvhpfYHgyORblmymqLc2VIJQ\\\\\\\/yexfmk2bKOCUoocEYoWKbVQh1HxCkRilT84yZGJjEBcHeMf3Lx4WPRb+iaKCLyVbdEOeayTipRDJ2Rxn5dwxsssOimgkalF63kesFykzBpGnSO6jQNccmFc\\\\\\\/E3QR34Rh8xoG10vdBK98JYKbohxU4eniagKO7aUIMJ9IGaf0amLc2jm9mbbIbrVvwmVYMjoQ2n92yPEmMY+F\\\\\\\/7kao2owlg8sENtAD1YwrFjbMaEYAZs8oHHmQajAt6mI5onuMeYCguhlvxQO7NCYeyfhKpxI9MeTj0DqOjN80htelFL8CMBEknoN\\\\\\\/A3gaH443RS4w263ZW9jttggvd64JqKaLovbb8hOjpwVcOtd0Y9XvXqrIcSOHdYeGbZQBxjboNGLh16tMezE+4uJ53mWZN8wyH5iMiMBWw0GFxZP\\\\\\\/Lqd34R8VtEH9LH\\\\\\\/GJ2WZv8Mf8msqJC4IkgnXvfIwO5owYPXUys\\\\\\\/v6qSUv2wlvWxG0vVXRADOTbUgd1dKwQJS+syGNkD+jQCjpWIBXRvBBXTwpevlEyeHIEx5FzwfTEnk7JZEEJYMQQQyzSDR95AfGIVIyKMCThk2NPsLyQksNagK\\\\\\\/gpxE2xvT9\\\\\\\/5PKiKlRIllWUpDRCFZvNIkDy4N06YskNa2sQShGSatb0hwjY3efnlx2K76zXsgfSqZXlkQK3ES2+3X19fAibbpS7xxZpxXV0oHXpPhGkHDRMgbMNVZM2Znp3KtG9vatu3waAtmVT1fRsyR1QkQ2onaG+jRiUg\\\\\\\/pzIKdH\\\\\\\/pAgDPG5p4weiY2bLhvATAAks\\\\\\\/bjW0mJJY+44tLLUB1AtxPy3JDmSzg5fSLU0r2FyLb73F+tjuRjKh9cv1\\\\\\\/5SodM1MqcwV\\\\\\\/TUii9dWjrqF0Yo1apPCT71S\\\\\\\/BbhO16ckMfncvBwxXQzO\\\\\\\/iTB4SN5x6LWStDw4u1Mj7kEW7VXCoHOTiBKuy1GyHynIPGloeIrzRv67p27B+wrBWypID0VGdJsNvwFqvQQLSU9RXw9lmLdZ+utPHtlKFwBpCSyqKsl6J6jsD5qDTQGGHu8jUiCTZZyHEMlCU1sTDsyFwPAfnVp5uZ9r8dW2q6xdNfUEMBoh6aHgwmEteC+ovIHMAGzV4cNI8bequFK\\\\\\\/vyQFkP3CMUMS1Iao0TPsjJe3dI5bt0MuG3Q87PBaY41elmHwdf8800cgzn\\\\\\\/2B2iVz+kCobzWUcTv6bqTUzd9lINAFXMO7DEEF99Fg+R8r008XD6DDdmASxR9PheQEr7WZMQNhZEsw\\\\\\\/dS49mEBmU474fYOe03OM3Jj\\\\\\\/jdFiPIvr2c91s2Bw3qAQmIL6apWySVPBCJ4YqdijDhRsiwNc4AmWadk3LOnUPLIK1MDACjHHa\\\\\\\/wFsKWRFv29TeWLjdM6KuBmkFuDPLNbj\\\\\\\/gGSWQIjMOCazWcd5F4wojhk3z7EMkbhTRufIFpo6DHjd5zlZ7B3W59ENtWwezbaVuTvWbF23ECR5k3TO32xchTIU7g3v5caaxNOjOuG4bncVPRdH+7qwsotK8nw74goQY1Wp1YsmajflEay70VmYHsI7xYF3GxR8ehUgU4eTanHfWIU8zgdQJau1VxhBckipS2NOUNXcO9jtcvKroun+1ZxkJ8sgjcLwDoKZ2HtWro6tn816vt+kdgxqjo6dAiuzpUJQwfkgzuXQW7S0RlJcI4QXgy3olfvNJJyHCjMUVymgEHbJ5lNjh7FuSLV38RlfGb13wBGoK4Nxk\\\\\\\/OyQRUrQsAWUbtMU711t7ZS6MDyFaKuhvLQvL7+9fRk0cVvNb2OZBMRB98umn+uZBefMQY6epFpjXJ2+srAV9PyfSzezuhBAej7gwx+Res1rMLF+tpN+ENfZLrmPkbrlkuUuCeirwgOJlcD8KODNlnMBSdQCvG7zbwAOSUwCjYOBvx5jMNa1buHgDE03i6JWOWMGHtWm+CDkUQeHh8PEVWGW4tyxwUxdm5Z5da1luiGjl1GJYZLnvDzGXkF0dvsNDQvJ47CXR1oR5jd\\\\\\\/+e2ii9IfBLgQC84pMtOcDQofbwsO2iW\\\\\\\/9wSkvNc4Gn44KdxJMPuCVfd\\\\\\\/xZpOY+2eHzZJh\\\\\\\/ewBlK6M6Z\\\\\\\/nZpc2K5HmnatSQOiks2FaWtWzD+\\\\\\\/rJ2qPt+OasVnM9kgnRf9cbrH9LhUB4qsyioraYt0wOlikAoopkyRERSp96vgjLrXkJiL4ZrNlTETcPb5WUXJ+w4pvuFbqlq4FnLrjDkHc6hOVNLnBKBV5F6Na1yRLGbvgl5SC6i+G8tZdeGrXkLMCqhKI7+Kj4gYUdL5tDOjS1oG0VGG0TPFDWn1HZGb5pyl2wYt9wCnej24kNp+eE6\\\\\\\/EluRWqxWfPCjEFlvMAek5QJwZLABeLnczqyiCgwxF8z+O6b\\\\\\\/FtOr66ex39sJBLURnTzpPbBBtV+6zfcE1y92OP+OGEsRmH0RDBwHQ+8qAYIxVvjHj3GqbZtsV2PjR5QCKi9Qo753oizmGZuauH4mkE+EtrgZ6tz+omO4OWD8BB7gVA6i\\\\\\\/yeE7KGSQh\\\\\\\/QJIz15AaF62HNtab4lmq7MFWjIX3pJbuaafxyugosYOVtQJg19wEB1CTOtRXejUX5MRyJelQEGstq2Uq\\\\\\\/k+Xa\\\\\\\/XngovIeutSkxO0VuXL3K2yOt3diD7Qwl60EZBQqMo\\\\\\\/Q4lKQgci4FcHAVud2GCXmwWnkBxJnO\\\\\\\/KEFuKD+CMNM5P0QjRNkWL4m+Uhyo5EIHiFEP5R4LfDtsouvzS+Rg+vZw6uCt0PfQyAgV3lMl08SbSmWn6rwWTOX+snNYVIR3VmIIUECBP+9DgUevP28DxrNi8nxTFe4rCl1mPa8CJX5a9cw6YV\\\\\\\/FqLVGJZ6mXPAKJmBfzQfTAuzDrt3UDXiPA6UY5RH3IRGXNkgU7K0z51atCg\\\\\\\/InS6ZARcBWh9\\\\\\\/E0iDovRy\\\\\\\/oHVnAzNpCuG2OKbx6z+CAuRnP4NtctTTJF46xidasQh\\\\\\\/5xBLS6N\\\\\\\/5e6e6VYo6YLDpyRqNuf6zwLLfJDwpP4Z\\\\\\\/7UbjgouC9Tp14KaUAVVyf6ZvYracbbGCgdC4KxezztYwpDFdwoP7EgYMTPAPGkJqlAeyN7pg\\\\\\\/6D70KsvCZ6Ee4V37F3PNo1tYKY4Jz\\\\\\\/scc5O+vm9AU1VDJFzNRDIiy64BewPY9w0NlcSDAVKAnNSxdQsB+Sz5lEAoIbCz9NRagcMgh1pZusDQNG\\\\\\\/UPV7YtRpnpwhHNMgEvpKQI8a2WXFQA4I+Z5CHfSrKJpxpEygerODhr0FEraOzsTLeZcj5mG0Nbfep9dWvDABCPkNqKlhsp4VQ1eB70mF7D73VJ7QutDRuQeb7MV8r9as9p7fBWkl5LT9lZcNztKiQEW1fk1No6fhvRa5HtSGtaviO+fvN1Hd21AfqPUCmgRGCRLJ+EeAYJ8ZxnX9xrUNZ9jJQ4\\\\\\\/2LvVzyuTjoCzNiTcFwp1ZUyLgpObzWvik0VVRB94R+f0Cv3NUE1z2D2ia3yP80sH91s5hklni+p2PlI0BxanHlltVezV2bXLA5dPdrR5P2\\\\\\\/f+BqoHZYixdtQPMjDu\\\\\\\/W+kTesxO5rqjLC2L6wmgqennNR1CNvC+osxmBcEvnAIAFrZcAQxh3cUk0p5WvqKyO6mgDUsohJHnjtjXHkBlOT1TP28T9o\\\\\\\/Mz8LBLhDC3AZje64+WE43KJH9U+A0XgbWhSpZTLPw6BeeWhyRN5ZLDMcKp\\\\\\\/0qxVjazg2LyRHBkSFIYjumc2NmIXZqG9YMAPc4Y+qrBdDWkvYwGb5J38fgpGozhIRSPg\\\\\\\/6DI64qgJVAZfJvj\\\\\\\/ah2W0bOsKLm76KRzdCp9la1Br6FrT5g2aMSj3FyQwmVkgbYEoY+7JCdkulQIkEJcsWfuBnfQbOMsJrQJ+Cl3Hrxu9qlWdM6w0uFMQ+pMKDVumjHTWWXWpbOr8sQwb\\\\\\\/+Y2T7WCqRnSjxt5SGp46FQj+ImDjNZM0UBcG4lHW5+08ACWVmClO4S3eBv9cCfGNjrFJZUyh+sutx+SswehwuEOO115jIWklU3wwVQFTo0v+f5vUmYZEOi\\\\\\\/tvBtIqBwbggTEc3M0jIp\\\\\\\/DiU0FiUS7\\\\\\\/bdkqiqmmxq7\\\\\\\/kE9m3w6xBXS1QT4QiTCqkFIQftj3v\\\\\\\/XG5fL2HJUZKMmZJoXVLeaJ7riC9KsIT0iLbswvFgOaqmntZKGD6U0N8tmxVRYqnq5omZaxxzgWDwQ6qTJfUzqArC5GeBtkF1KDDs7VMT7CR81hzAtq+gz\\\\\\\/oQmtQkLZ0wJgWnCM2TEk1WG8JijN\\\\\\\/N\\\\\\\/ODn8WwTqS6\\\\\\\/1WOCk6s5FOy3K0ZPcP3cPL\\\\\\\/xDSch978cIr987vZknDfxq+5XdCUwMmOLLpux2PoUdykgK24xKnmeNzE23Jy1umXyawKKl1oA6M+qCS4jgO1l+XU0S9y7b+lP62JJ9v5aapKT6X52MZIIoHoxllw+Vnyy7SdCfQv216Lt+mnpaYKZ7ixwmuYXAPEpTEE+RRV0oSRbi+HaMH8tJK9M7ep4cNajvcwOCc5MVBnoebVBbS5NWgeXjs3WpVBCMr6iy7Ke93EOyxYQRajwf29UtkPEAgRyE8QOcMIyO8a2SGch1F+yJicZ+bqOyaALMdhIPXXjJC+EswKYNmrYRds049+a\\\\\\\/xvJK6utfKRknmnxBU1ukwdsXDj8Xk1dnn4N6X2z8Voo3V+debegsz3lLpZj6kOkUfzCUKbwUXIZVRJz5h217+I7JExhLy\\\\\\\/9m2xvUyY93z+Jfth0VPmZtFGKQrK4Vv+3EyXef\\\\\\\/4rGzW553AQD3g03BBY0Oc7pwLrSxHfj9Vra4BG10\\\\\\\/GTVg9CUH5kOFi0t\\\\\\\/opau+gC\\\\\\\/3+WPUg3l3sP\\\\\\\/yEv1Xi7V6OTRkcAxBm9+jWDKZe8VCdtv49Rk3RO89nvIWXfKLGmToYxsnAFvhBBd2dvAO6D0IV73S1sa37bUddE2Opx6x2AyCGsdwG7oY0c9sEvkPjdqe2GZuTGZFauR9wIwxxIkf0\\\\\\\/Cl5WSfFEQdpQD9hWQnTU\\\\\\\/p\\\\\\\/MLtoK76Tbqf6V7ZaqSxRsyam3cWxlKFsjo\\\\\\\/1N25YOiwNudegLoSQlwXhoVWROtLMadYyFqyauSZxOMdYlYIIWWbqKIs\\\\\\\/WkDll5+g3Q7dsjGxm+7OgqN8vFMEruRsLEB1Un2xSxx7ReGZ914cNsNIZtViUc1dqs7DuUOGHyKG5+2ueCR3S+266BOhk\\\\\\\/OhZ9oU45xEk4mVNWYLMqMm\\\\\\\/cFZ7OQCw9qpdLSmHZbXm9GdDRUkw\\\\\\\/OoRrHnFc+rtKsaFkCB7hm27KwtUArR+rnmuRG7McPJ9yGwjzwhhFUGEABaXABAWfHgAqVng2t2DB1E7mgkT\\\\\\\/1P74c0M57h\\\\\\\/XG\\\\\\\/LjbwHpJxfUWRBvqOJcQFdT0BOELYFNSNEih0mLoyeWHORRVGyZJf8KVQm7NBo2\\\\\\\/XkLMRNGle\\\\\\\/P+hGpS33PlRA50vT81l7VWfs6h6RV4J+QVyDzHsxgArhOUnPl5QWSxWGh21lZy+Bc5GTcZYATLWMmMyJvFhmKngsQCVCX7iSSSJw6w\\\\\\\/+IVyb1xHjm0rHaivfpN14ihkAf\\\\\\\/Soz7H+V3bCuhzULTqJILNtNyMurh+uM6XTAQQY7Yp6JUfse0yX\\\\\\\/D3hfL9IpRpHXRi0q32t1YmTLJPjbFTJluMQsnPXZThTshsVcsssctl8TX8P+eKGdwXZzGA1auP3gwfyZtp7sF2b5fEjNoilpsuaBxNjnFHEXBFF3uggGAG1UoZ35zLQnKVt1pP37Ap+Va5zZU+F7MhJDdSF0PfzQHNsYQUjlaQEfSCAyXbyKGZcMoKRS5bKc78\\\\\\\/FhP3FVAW4h5QUyHyFiESgX+6\\\\\\\/CQji9eeRTmt6KllZkLESu4PEY4pLlu3AEiar8vgw3sRN+8cHwea9YOlHKJAE3DzgUiXV4RuvqymwI\\\\\\\/lsUoeVj49M+G\\\\\\\/z8WNmMMtDh1WlT04Lvdoj7FLDuHV7mwqPeVtrc+8CFKtQscL7PWV4\\\\\\\/qZQvDv5V+nbOkcerEX8DG0RaLp2IGew8og8q0thCtKSQq7qN0qhRMwX3ma1UC0Xj8RNDAXzD2DKeEQZaXlyaZPyqm9WwpjmOniHwuDQn8OrkPAAzsYO+\\\\\\\/0e2RUC7g2N9u13TCz55tDIwOrYVyDUvN3310E5qYMk3ixdxKY2SQ83bd6su73hBI0ho4C6+DliNiL2uD3jXQeBdFy\\\\\\\/WRk\\\\\\\/IdYyZA5MZGFQeGWKFH0CTUoqYA1554jX24ksF5bpxaAZsBpEbRz7XIpZ6U1yIP4Fmp+HRnx3x7NDAQoBPb15SpQN9pXRlcP+WRjty3Kl8g9uxABpCHyp6MlxGtW4pl2LStj9g5QPciJ1AoO8O6iR8aQ7YGULISvM9Bmi3cgiTtGWFnXTTN06rqMMNlz93RikT7j40dv72NDUSvmjLcBRudJEBq0TrrE+cwES7BBJ0fsg5jMqLc0CG+Uqd2Xrre\\\\\\\/X2l07eFOZS\\\\\\\/lfolhIdJr4ItWPLu27Yi133zGfamRb8XDWAnWFXQ3NfQy+pfqN4fCwm1raZvMlvKVjN\\\\\\\/Bunx7SCs4tRU5GYJe6RcAovlB1AUSv4KYYGj6CzpmqWPeR5LenGMZP+OnuUj6whi\\\\\\\/loSJE28jFH\\\\\\\/serFf4\\\\\\\/uLsABPs\\\\\\\/KHNtoK0J\\\\\\\/78fjsojXrDT9szcnmg2J1vwRyFCeRE\\\\\\\/U2E+P\\\\\\\/4K\\\\\\\/K\\\\\\\/pha1+3RD0LJBycpMGZkmqKLRHQhK3GtCw7GSkH2A7vNxpVSGwP1OSbXoA+Kr6wd6JcgcPo8bpxL5lXnAqzyGi4AkqpC6KgeIv5+\\\\\\\/QZXLT08sbppO35ZrGCaeLGiG2a\\\\\\\/SUnVGzQ1AtxRKyuIJdgg0xntWVMsQF+QmhxLCMw+gVZcEKEITP5zr9He9ooXmHWy9SO22528RMSynCRgxnsUpzX+q1L1Wn8SAslVAa4EZ1Q5QRTNYtDTA2nIzAlItgVhB87bL+kr2jY6UCQa4bRhxR7S8cbaA+DG1TRyqoo1qNkj8kybxHUt7ljk8iatZ4ryB3x4\\\\\\\/G+NEYOv\\\\\\\/UdM8MVHI7H2RtszI9OrxC2gcg6w\\\\\\\/5B0yylaIj\\\\\\\/QMlTCYKXkyYTBQIPRzK1RskYa66JsK1viSGOXteZQxtccQ2S06CAoY09SeWUIzfWfOPz1HOrbtSJ8OkYvBwlB9KKV8l3suMzIVpeBflWSAIUucX9nPUs0KW98Xaq4j\\\\\\\/JNhWsR4C9vnrPtslK9FbCZn51E1rTuVNTwdFcYrJpg0Bn4+rkNAUk5TxWBFlu0lmPHrqj1tFdSrVwMnBtl2BJhLD04Dv0z3n6StA5RM4bclS\\\\\\\/LPG63lo7Y+YIC5\\\\\\\/l3+Un16vrM1jzgTiE9lHBFjWyWqm4XZU0mra\\\\\\\/h8E3wA+mATMrJfxFfxDX60RUVGw+NYrCEJIHhfAdplG2QqL7B\\\\\\\/Jl2S\\\\\\\/ywT6Ex9R8GgFz3\\\\\\\/T66nb9v0uzqn09rq9O7MuablpM44o08bAMyHstljC2jOua94S12iNF18l9bxylKsQY2U0d5q0VmPzv6slWf9ayaRoREMlwlBnvpApBDhsaPJweUIPibnjZMdYdheZSIzUEGTsIIm7IdGdXxWOSxuZXtUekqmEuwBmLxr\\\\\\\/BqG7aq9pWj\\\\\\\/6jNSpznDjB+nT9UfXXruHWCmLsf2LdQCXD+vRGZx8bmawXVpR02tzvB6QMbNZvAzrSKG0J+gi+icvm+CgM8ow878sbxE8lglCrmh7rqgUqkIPlyRLIP1Xi+PST5dYJEomLfCbA1e6Mvcyo6zKMzbI\\\\\\\/B9pQyLHUUYSEL\\\\\\\/UbglAFXaHKyE6HR03ImZC\\\\\\\/HBLvQG7MghBaDZ4Z\\\\\\\/kgLf5DPsJerP9qQmB5OjnAZp0PjZEAknHJ6eOwmWToYj0FoTW8RLZYzeaJMrVvvvRGhmIykeOeUWq86fhGtQO2puBKrWjd8DYHJTbWKTXR9QI9io+srMf7togq2BGAqOUSOy+\\\\\\\/ggJj+8dWkBd7+\\\\\\\/DXAFD+01XZ\\\\\\\/F2ZpQmbXbe9zUmbSsz43FmEeZt1HmQFc4tQ2jLW56wIr\\\\\\\/uXmoa1tTlDajcQsJ5m2BC6VWGgg7d9IYRybV4GaTo6jfy9Icxpk\\\\\\\/Ih1k2S9jLtpyABs6\\\\\\\/HmBZuRr6jktr8j3w6YawhYM0zx8ZqbP4v8JFmP6XHcZBY3mzmXoSgo31HgwNOppGwbMA\\\\\\\/9cvMnJPJm5kRNXDzOyHBXg\\\\\\\/DvSr7hL0g2jqPLFDYNQHZSCZd+SA4RUU1t7VzKA+5IK5l+nanx6H0BVsQ5U1K9sRI0gWUVWcPtPmJa1lw+1tQnALoXPvkkT5EQ9mjyl\\\\\\\/JYERMj6BJJikqrVsxkeYP+UJ\\\\\\\/vJ2InnHUePzTnxQ4YE27+pvx4NEtAw9AQfGMQGgShrh+5lNARn52hRw3Q0TmyAfZIigK7uWQqoWCi0qICknMICH71C\\\\\\\/8ESaaVnP2Io9cQZQuYyDor53jswgBE4IHiFqAIZsJEAVrEdSVaCWywkxIXvfxCE9bLM6m3L9dxu+fdIV3OW61+MBUruh6uJmo56ccwgdZINaj+UEmZDaZ2zjfc6v9dDN3yGvb5Q8BKpTmz\\\\\\\/ImZGNLQ4BPXUP7NpwTJNY\\\\\\\/x\\\\\\\/PBCVJpi11QnySRi0m45xyJbaX0PNFB\\\\\\\/yifX5+mHkIGJnlchtkwul+Tio8HrAKrhbVWRNj9cbbV8EVZ8xdy7ArnfXmTThHq2K1AZGV+XILP8oI1hCFmc7UqREWVRMboQwa87VhLSUHtOFk1Ovq3s+DHRadQt+XteDIypJDTR4pSWvzCkLNR2IagCS\\\\\\\/fhpQli1KAgRksGIsRJ5saJPyYlLtYRQBltDXl6+RSg1mvxn4hm99FtCyZhtzpTcXvqJ\\\\\\\/wtIgU6xMrIMaXBe\\\\\\\/MBXlczzQppuwEJy53w3i3nHjO2TUg\\\\\\\/t4j2tmCZaLHi5EBP6vkbL9pKYP+2UqkSBCyAmoyOdcP4neJR7016sqXLAYr0oIqIM24\\\\\\\/IHJnpjQbHxo3MyogQxQ6XTas3jJ\\\\\\\/sJrytfCxc9KUHbetCwpU9Stg\\\\\\\/sM0bwxdV6JMxPil\\\\\\\/\\\\\\\/UX6zU2QEUqGlszEuqT7tpagVGOkUlNypdj8PwIoFvYTt4Xdj1ftAnoKlyEfbhjkPHtPxVx\\\\\\\/2cZm\\\\\\\/SO1J3HF8PTAAdHzdDHRNHH6HrWDyXFraawz1MV9mgPR7TE+f4WTVGvS7DRTr1dFEgCyvrQkZId9GIbTg\\\\\\\/lrg8+jribt66plQjbxiFSpOWdKbDs9hJfeqUOGAJo1+8DZU6K68kJSHCs7K2IQBHdh2AYxZyWXXTWfGjuSdF5wN4I9tcBxqMZ9qxCP6G0BVyYlTevMbr07u\\\\\\\/2EQ1KSpB2yqR7g+2oln5iaSRFEfN5hKMqSGtBS\\\\\\\/C+Nb0vFFi8EXebfTqdCDkwIkI5KtSJxQfDkT77Dz\\\\\\\/jRTdmaVlDpEPk9Di1yIzdbXLBaSayi1LI0hSGJuCiOx4i6MxUPxJj9gpt7VbNNBhhSL1JhQY1xYPjs0G59fZN6WSJ87TlkfObn9gnGHhbQdfnC13JMlcCc4FkyRUGgEgRaNUJ0L9h+syT3qPmiNTi10XaXs2fstQY5DL\\\\\\\/YvtuwHvH6iapo83ArmnTxwrmXqU2bfqOIUN\\\\\\\/j3D41nRg4xDn7xoaddZcb3mRzRMuBnlrOQKq3rxCQD3ChdbMv+R60v9oTTCVWEK4Z57\\\\\\\/XAsYftpaW2cEUyTvCkZP3sHwCdc47XGiqQtWIFnBl8kbvcKtbcn\\\\\\\/xc1jQNlfJ4hABzy2Xff2yI6\\\\\\\/+hyl09vI0W4rg5rSn15YZhwrXw7tBIOmCOMsn31rq7EIQ\\\\\\\/VXsz2eCvYnDeK4zxZ1gBKa8ewipU+Ol3FXtMJyzEOTYcYRo2fsshr85SYkXXKl81V0pON+Ftq1QpF8rHmhtKJLG1ygZwC3x3LLxueRLYEN7kASmLUT\\\\\\\/yb9uplVuSHXe5T3EYrQwKeEx1b4Owy999zXBZYlUir51xQ9jXrS3VK6RYlJeTVXba0nthxKYYp7tZB3plVf4p9pxVPoSiZO4j2PzwqOTnYx+NMCkB1cUwP5XQ12\\\\\\\/wQ1vfQ3eZy+QuNviuIDPgmHMB7\\\\\\\/ofit0iaCni0a+PVpPd1aoIBXjY62BaseoZVcfQF8AJ27c3M1oumK\\\\\\\/woxmTPOU9y+glTesOqptTTdkQ5MzrpfONkSUm656c3dtiXKyCWsdJmLjibWV9DZkr65o\\\\\\\/Mu4FG3AbUVfmSIzKePjcKkLXL9hPEW7FI1mudEj2+Q0O0AZbAEDkGKtBLegW9iiU3SqSHXA000mQIkjDvcdUqtDyIxoo4DB4oaXfuvIdisHn7O7BvPSe5ThwG3jeWyYAq4GDTZLubDwaej+TwfUwqaW+++Flb\\\\\\\/CXobwfX6Thydgign1DK\\\\\\\/1WxI9dIEyQnX9DtEl\\\\\\\/zSlOUAi\\\\\\\/w+xUkZAion4gWrYn5jNV7rasynsaWbIzbw1XYGJ8Qwm6vwD9QcGreDwGyN7831djIaFEg7KMrJ862C4tB7+JAIR6\\\\\\\/syxkb0OO1bd63IlbyLhBpsqG\\\\\\\/STuxhd5EKDCuuAY6RTYbG6rbaAxFQHhXBPo1DAemglVwAgJfIg2zY0+TO2oeJkbm6ALtXsA7LT76vIH4WYZ825iBYWjLjqfGgHnva0Fn0lHZWFz\\\\\\\/3nyRaDfX+nR0PEkeXp+N2AMo4\\\\\\\/fohfdMgfxdg21ZFZYjw3E5VVMKGyrskqH\\\\\\\/IK2XHmxytpcrGNU+Z9WY\\\\\\\/qHdUNqVrGLnHgTFXV\\\\\\\/+zegqDZVhyRBbcqtnNOISiV1wGVnegwFfyV\\\\\\\/410Fj5tJ1YkUS+FqSG1fLOk9MlSTvoitps06WbkEnTe5Uj7SF64L8mjj\\\\\\\/vThN0h6oH3a+NE96eUht3\\\\\\\/UvkOO8VQwSCAyt0IOMHMCw5\\\\\\\/FCYTEJEJsBQps1HAyL5ffdeRA79L9eha4yJupLv3jtKcSTZ8cyIQ8xQDvCgcXRndQ\\\\\\\/G2KNRqcEBw2TTBKN8uXEP6AK+oTLFDdmP2Ctywal+172h\\\\\\\/eSsmIo6GHytO0DHM9NsUAv1ZjtQ0N+\\\\\\\/3o2FXbVqgevmL\\\\\\\/bVsHCi5RZsGL3gv0r4iqCQdOQxnn4lCr5ptWKNMqGx3pdBked5fJJUDs1uuFk++33ND9rl1jamyZARhjBVjSASPufm0B+tnAbts631ZohRIjAb4VvALfOdgvEgnVfS+Gzia2Qaedhc5JT9hoyF33b3de0oYf7\\\\\\\/oqLv906Qhk2ITQcwBHMlcASCMlJoKdazedlXLydBP2+G3LI76RmrH8kTyf9SZAQtrLBZUy\\\\\\\/JSuOswEB6CbFm\\\\\\\/uy6GbGO2oxzQdt2ozmzsgQRZZv8NY+\\\\\\\/U7SpD560fLjXS+C0mJBac5WgGKkWJlOHmWxyLbWUe2hUii78DyNVIiJgVchyxwlr+DRk4AOXAOmzRC37VHvztdgkPcM09pWgLbILxmOT1nciBCnOKnnq7JRcDuh5utVXTw5XNSC3yg\\\\\\\/K\\\\\\\/hJe4AYepZ7D80pS66GPTaofcfSl6bgdHGew34\\\\\\\/aFCdBrTGZlMBRDsSvpVmwt0piBRDZG2pDTgVGOjvEfDybufCgl88R7pZ+0y2Js99mKFEFIxaZduHpI0kZSgosSDHwOx9yjGDw3j+4OLXK1hRQW761zaFYrBkiiGzA+daBsB\\\\\\\/SgkmTqZabuZhDMRHGpAbMlXgfK741EfkkzTxHnw2C6SHw6GpjWEGwJKmomjb9EmOH+YG+SSdXiriPG6NrwEqI6LMiB3bqWPu2lAYw4sg+AtENgundnob4iNKgbLaPXCN4oB6LNNThVNmjxE+l2PTT8J2fQ\\\\\\\/MMEbBftoLs10n1SvFogiSpw2CxRXppWtQdkEDpbQJktne7EkuOTdotFi1AuksofEi45\\\\\\\/doSfy\\\\\\\/5m+gr4B4ONZq\\\\\\\/4U8uAmGRJESVpXr9c4CrChc\\\\\\\/\\\\\\\/vPh7PgqkgUrJ69pD9iUKR3QBu9P1UrgKvRvEmMBD+pL8anE3gDGD1gkKwtw1+md1Uh2idfYeAO0YMRVI\\\\\\\/+9a3Eg0WFsWc5h0919KO\\\\\\\/YizL\\\\\\\/uoFnNvExBbqrVChrXKb5NcKWZ+ltPsEB0LaMRagP4JEuARYwCDEMBptUxqTSqq+B82HX5fnXTKopDnIFxPrg1jcrmUt3RQIthus4anbYGmJg\\\\\\\/5pcoJhjiHzEIy0DDRMhbJ\\\\\\\/tH0pcopqsoXIDsB678mCknNVdfWldH3jWBf25sb2rq\\\\\\\/lkKmcVMyATFndusTPFyb51F667Ycx1JBtLRnXLmSlhWShSJrIJBVsZHXeD97p\\\\\\\/9R6mzOza26xyyHoAai3D\\\\\\\/CAMnsQWME0eeZNbjQyGhxQ\\\\\\\/mf8wIBf8H6mWVkIqFWcn+GTaNW+chUQxp4pP+XeyAQTz1t0RRUIm2X6nSzXW6egOSTD8L0G9CQOQtTBebGGeciHEkTQTrEhVodYGVktIFmxQEXmVpKurXWcF6QXkB\\\\\\\/FhsdsgqzySOe\\\\\\\/eU9OxgnyHuyE2bmcHAm+Y6NwQ2Eah6bgdIsp9nxs3\\\\\\\/LWvcN+JZxc90k1UIZHDrchK8zHwSpTaKEVH9ALpZKwfszvBrccgbo7b8l0jkwjHNZGb6rjzK5ecuWzwvTKjXR1H0OCmEDpGvYA0XVcD\\\\\\\/RreP2gQC2wnSg0\\\\\\\/LvxUsJkMR69mMT4kUZ5xu2JOlDxr9LikkAqIK+PsTJqGlZBuWQd+L9n564hkdgh2+AHGiXmCA7xIn9CqFYFcEmy+spWvX3QV6fdWVVaLrVH7JReyLZSSh7mijoCVVSW6PEyviAMcJzm\\\\\\\/Yh6PiLNP9NmdcUAzXYfgyQck8fagz9UTF6\\\\\\\/C1X3lFNXljRKf4OXUoIpuRrvq+wW\\\\\\\/NlDxrIa\\\\\\\/xupRAYZggNgjGxjCnomSlctJZ9+LoS9hpMHoyBoWPDWqdb1Obp75hxBRsU8RPulsZ77RQVgXhN\\\\\\\/6dSLSsY4LlSKLHdUREdghBGu8kpQTMAHQCos3EkdTJpdbbu7xVxY\\\\\\\/s1gIr1U4Ubpacc674iQb4YSbzXqVHzADehBV1Wb5MWz02tj6qtJjDFxiGtwFZFXWuqrJkeE337yx4Iq47so22kI04Y\\\\\\\/Z0o6RC9To709rzP4WJIMBLDS2KgpLLT5HT95uuUdIuLbtDRn9Oj3s8q4BexZJqYrJEdqAG7U3+2erh0NydE+mu6Z8XizUAvGfdqaUHqJTjiqrj725ZyyvNDH5B+d8BwO3Z5ZliunZvI5KsKFMMgzwOG+5KcMN5nWJJdkvGFjylkjC3x6LeqJ7\\\\\\\/GARpkFd+ce+4wtN9irafteypRM3fssM5OciuqwDt9i4Vurm\\\\\\\/QK4yRLNQ2Om4aHoBb8NjWTx93mXNyao0HFlPuLRQZyUeuYOkCahLqS77gCFDNwCWBH90u+65OPqFjYEx3OukwQZaQM2v\\\\\\\/oCCYF3sS6xII+pQXMgKq\\\\\\\/yRxpkm4QG\\\\\\\/Dpfvaby3dejCtOBBoqj9UuzX14KJxU66TstrqDHJZqEhm20c49maxD8\\\\\\\/B6uec8XswdZA33Oy0p1TyYwSeVbTQ3Pb10fujXzInJyc8oV4HDxRAz9y0fiKemRWrAwfyyVbEJUsRMAmNGrHGNpyJgiq\\\\\\\/N3mRkCs\\\\\\\/Zsb8H5\\\\\\\/xt1nsKYaFtfz4iUVFjy9VFxWO\\\\\\\/mS\\\\\\\/g4TmE8DfWfkrDFnlpkLP0rr+r80C9S7pgBt1JiJDdCeg24DwnIX9OrNbuMqZnxFWMPBYXXEOmQ7RuxTjHzQq4ntG2NATpw20eVsIcpX9IPXGT+pFpRvXHGNp7CFKs49qmFEpELGm1L0RsG2OBOEXt9FaD\\\\\\\/hjuyKrYuk5tv8gpx88BnwXAH81WttFp7VjSixnXdi9sR3HMnkhf3FTZL\\\\\\\/L1YEyPdhqLGerTK\\\\\\\/IVQZqX7zM3ASfVVAaek\\\\\\\/p1cnR5YZtjNFIFu6oPK4RmPOuljy53lCKq2v5wELdw4GMLq9m\\\\\\\/VJ\\\\\\\/6QgTGj6YHWWBu0AxhC8J8sjkcrR9qbgudwivoAo4b03phYLF3z\\\\\\\/3I\\\\\\\/Y23tjc5NtCTPD\\\\\\\/bjLJHZybaRSEzYqoaJuc8bKpKqk9uhRlMRYhTYVUt\\\\\\\/GwYzeaoftqojjGkp1AN3aIKb4jiz1kR2S\\\\\\\/bCImdgJTT8LMg2PDvp9IwaS8wnr+A+KgOkGikJOkplYyq1Oz0odK7cyFcve40wlhTbax7b4DvOvB17sufzNYiuFma\\\\\\\/Z\\\\\\\/b1Ck7DXF9PDfOC\\\\\\\/7Q4ZubdN372I7QqhL883b+vimo9kQjZoiz1lN\\\\\\\/D5XE\\\\\\\/T\\\\\\\/0jFemplM5vorvbicambuGm+MUheCsrsvV\\\\\\\/wFB60BuexLU8FHQbmVAhdc08rd+Pc2XTUdY7HDDM\\\\\\\/KfqDzK34URZ\\\\\\\/Y9Z+n2bSNgK8GJKGecn3cu2TPlnMV4pjzU1IwZwOumLveC\\\\\\\/9tomhnpCn14ATtfqrhS3fQu6Ue0ynjXQZGLMwESxxJz7UW+hL1ag+8IOmFopqGBgBQtS8zfn9JsFD4SihWSRkX5wR7DEOMvlLXKysQBscCt4V4oeXQxJu+ahxjP9Egfnr8Dtd1Ps97EjNooSNSwtNmPjXxk08rdqamU9YRQWBTCBATyi2NpNde0GO0idk2DBdp56ky3Jv+zVUPRCF\\\\\\\/pt+uVyDqVTOueRkCbbos\\\\\\\/n2OHPbSb5xkgHPwU0lBwx1JlP1JFwvTlycgRa8+GBvpkIdL\\\\\\\/UceYzrmwjRM5BAM\\\\\\\/GlKmI1OitRn4gu0XCa9bwW4Sp2iMG1GbYSJN4S0gDwr3lOa4WKdXLG\\\\\\\/5psO\\\\\\\/KZAm8MYUBg3jBadcGo60Z\\\\\\\/t1ooqUImwugCzH+ROGJDrAkhyFCejIaLKdBE8aGLYTSfeZe8cr2\\\\\\\/nJqQkrZhtlyHpEmeitbjqkzljIkqX8pGfMntHYyUPDHow7FvH9Uj4FwXWEyLxs9fhSbBaLU6J5L5BV0+RNw7RjbM2ReBxptSOnznyVBrxed7kHL1vfn5\\\\\\\/VygN44GKq0rvMTF0rwg1tlmMyiLRajlE5B2ok8sj\\\\\\\/\\\\\\\/UG9\\\\\\\/SAzAzcSTYhz0zZ7br0tMDweRhWiWYp9PD3Vewbq5lEdy1\\\\\\\/kN3b5TsmN2voQzx1pIc7h74R8pOJJnXO0B8t2VlstKvGYluM40\\\\\\\/lbQvwEWV+1zyEOlswSxSNifcTxDmS7mxfBs4qrptrZkXGVMg+yt4KiNZ4ikee18pywoxEXGIPIyCCcvdY4yzusZK94atNRrHz6i84GlEIYZBks2Vgot6nJ0CvGwbtQI5HnYpFH1B4GMi1tIVISFTppZZRFXNz7xTw0LHU+Xam+tJvIAZNh2biiVAXmRM1A1LEf581PWKIv8OZbu2vrwwWBLJ\\\\\\\/FfedF5KtnBWUUfo3n804zXtNXq1R8Fx2EQjguCl5eV4CDDx6aQuzdaVW0MQDvKslMp6w\\\\\\\/feAOcxoN3xYyxUZq9oeZE15astldqARV8maIpZuNTHlULVhN+uxdxCprX+ot6P14+A7tXHOqGa3qERrlgQE4JIkObPMMS61A+M8pFaGYc9p6522JbpsyESzQ2ExDOImA5ohuk9ZTKauLB3Ij8tXD0QOvNcWV1zs8D+eAGjILUiCJkPWvHjtyBaXVq3pmFvoESDXq18jkQtDRWYGtBkjeEnnFvp3soeAi3RuVBkkNUhhOkXOq2Q56POThqFscAPYPf4vt9d3yPeF9DwNcnpQprc0T6zbhSXs30AW52piv4Hav9+MuES0uRnSAipwk5vWzjuAnfYyqbj9HX8VvXjBl1jHSbGRhKHtoPl6NWwgWtJyUaPQ6sk98hqrW8vLlM9+631nvifLBL18bsGeUV0d6k4UNeRCoXYilkqTXRb3W3l4V3Yyivgx7hA2Nll6J58StQHPqh+KI4dgQZVT03Fcdm3MZ1sTbnyY\\\\\\\/rmc0eyja\\\\\\\/f1MhtoXMtuMHO6Bv5otP3DCFOJWOKu0AOTvBYpefMDtlDPaz5i2pQU\\\\\\\/uEQ9mpFAWNHVbd6PIjfF\\\\\\\/2laMMpU\\\\\\\/+\\\\\\\/lehdH6qnP6qLsvNNpmexn\\\\\\\/K8uO+COQO76w70kPNOY\\\\\\\/bRm377ricvj\\\\\\\/ATBOMuRcX3d7OJYN0qn\\\\\\\/nlKLut8tMVx7xwVjpbETIFzUTNXiBWYRXl+AezFrntUhu1lc+xSzRAC7i5fSslPPqhoccU\\\\\\\/v3AZ0CNpKcsEQc8Gk96w3LHDn6Ujv4gZ2uOLOuHQ78EAfKIzhPCeJ393E5Ymhvm6fugZvICmhNFttpf5efMu2Nlh+39oq2Ogr0NWY3ga37c+EVA4TJ1SN17jrTNwES5So7AC8i1c0mrX\\\\\\\/NJkE\\\\\\\/0DujYZ8+vhcdIOXIhq4R0dxxf62WZdjqcSDKEsenPKrmnXWB3HdQxd6YBN5JPDmEK2Fiyp3gpa5FYR0aGOcbIaVPOrq8P1jyYIPFgwieq8qYM8f9kwu8dZtuP0ynreZ\\\\\\\/yiS1pdYjLYEGBYR69H\\\\\\\/5lxYbnIlUVkL617vr9SDktIC8cuaLuoawQ\\\\\\\/eeiOENXR6mxJCy+fsTO8yHCEIh64cCSyF7KI4Q8vbUo4S32nmPZozrPqb2feIxuTkEWIxQXgXWzx0\\\\\\\/8CBvCVtke7IchMVzDQXFDn4XVON70l4HDt4cunuwKGoatAyjnqRwkwKAhiKM\\\\\\\/sXiHv\\\\\\\/1a5sXnep8zBnbPshAFepWRB7YDNkaw8rAe78w0q+le\\\\\\\/dyeWKXNc5PDDc1+JyBZrpvVOS0Zo23ZIFUV0k0FnEgvR833bjCSgo80TKG5W8jAFjk76i9kIc4x9dxhFNW8\\\\\\\/dDoUt6VCOGMt7SFcILhcA+xs0xXt1dc+CdPaV8e0KJ06yEAgJ43UoN7Vph1upCk3v0M49bDiOwl5Y6CoYUnonC5x+yM7K9\\\\\\\/+wQ76buHJQrIcnIww8oAjpaF4uRJieJrJHzE2\\\u0022,\\\u0022iv\\\u0022:\\\u0022YwsizBrCaAYSn5licJtzAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221a4630dd38971a7ff224ab8d3e744ecf62b4154ac251cbce899a320baee2f369\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gLpw\\\\\\\/9lRwqJbLLPE53FyRy7T+qWL8UjFjgIJuizqC+HDIcQtdMBGnfWEjKjPkvY1m7f90uXV3eCmAhryT2zq6SxTQexNPtOB631Dxh861V6JDZldGHp+fUmSwlIZQiVBh1JbJ1UpynM3bTcTaeSMdX3RtxBNZIwdddUlV7hOsywsEhBmHW9mkiJGMCtmceN9q766mRqSsods9ANMIZIq\\\\\\\/N5+vVWZJWWopdBkbZZRDr4SswzVLKyJcG1b9TR1MnzvrFtDyU1kq5+6OQ7ZopkTIWnBQJcQ4bwbck4mxm0k2C2saUAIFglgU+74oAPdaJr9+pJK9Bllm4J0uhl\\\\\\\/95PpqotPWZasP6BsHsqarXmjC6+WCtwCNA7OcRsVbLEjjfeSduwPetmSqT5aR\\\\\\\/tLf57ELXZH0QLUnZENh0tpW0\\\\\\\/4+OBbN9LBOISVzT1kk43PPQZJnnSMQmpZJp8T1mfuIQJtrdVx2Qf8lOd4+MGVCpJxiHfdjbpzGRSrFOLxXw2sNlgoisBRwOniNUs35xk2+PYUdrHSoIH\\\\\\\/226i6skv+5E\\\\\\\/hi9NnSz1HhVW++BhmAqDpV+M513vJ1T46fsR\\\\\\\/jQ9uH43TQ2rIxBYPKFqaYibxDJhKPnV8VL8WSup5w+SZ4ZeBlpJ1bC\\\\\\\/Scq6gR5rUhIjK7ESQdbh0iEyTPK8ioTIkbMxI2s1PfazxuMiSfy3ydDeinCaXOScdmoZRwKhXkd6CJKZn4aZZsLkBW2fZohy+V+sIRxQ1CSz6\\\\\\\/BWESlI\\\\\\\/n6z1vsqKmCXh6sLokavrA6H0EaH1Xx07Mr8ypHPpWxa6asspO64nRATRvn+iSf0qcSbyKtbSJFy9aPwO4Dptxsust3RYakSAxK+iIMjg42RynRUdvZOTILeRNLmMC3QVAIg\\\\\\\/3rR+qG2QwuNwxhh\\\\\\\/LaAfBdzSR1Rg1p3tg0bJOL7YGU3ls+oTpxlN91lO\\\\\\\/Bg7Nj56xjxE4F08VmZ9zAG+I4r1cE0Shssx+Jqb0mVIbvz2UcRsGfNmB2ewWQrv4uLPnTwGhuflv41quPzyCFHCH6DPMy8r47ZdRcGhD8Wne3\\\\\\\/AuwW93vTTBsAuEIU0hN7+rZJtQXErgNQr5q1XMM+2nZpZqP4lqLusLW14KJDX5ST4J2703r+H0kwWymDdENf+T41QuBi4Rwi7RISgoLEMzxNJsbTLSqbRmQT0en8p+LrOY\\\\\\\/+ud8BMATWiv8ACQYzZR3bfttMjflwf1yb+8UbyKRHaJYsB97RsRdyuhpfNnFTDbI0VH4Yzp8CnpbmHSYm016e4TLfEhnapqhaQ3VJyQ4jd27sCSsTRWXYHRBPaE3f23cQwcn6iMSBRog15Iy8GZYOWpqJxj58w5y+OM2tViTUlc+wY+tnJILXqOjMkwX43oC\\\\\\\/X+9nvCIxNWGii+ngZxCx2IbylwX19CAggPb\\\\\\\/CPDDV+4nGl1\\\\\\\/YvGFeC+txmQstZKwiNbG3O8j\\\\\\\/NeGLOCXeVSk+MgjOxxUYiZPTdaUVUb\\\\\\\/nM3ZWk8v\\\\\\\/V49rCahIhOt0iDzI4VCnXr6B+EdveQNqwW6GmKrKz7T9Jq7dWiKAUxklj9KKx9x5M+dyGw\\\\\\\/6Ll5RizMfOyCbqyazpkNkbSnHu1S4jZwQqafm2MlM\\\\\\\/m8CIkZ9hb+g72\\\\\\\/Yp6PQ6gAetIBNH0gRt5vPWei\\\\\\\/JDEb5K91BaR26gnoKlt7kG4rgeVZSmfff3CtwBAc3kAZ1S1ZqpJeeRBnSQO1MUu1HBB\\\\\\\/i1adF2FjFPknGP8rXIUq4Yh68QQz\\\\\\\/tWgGL\\\\\\\/HIwp4DMJsvlVbwvfms6uhwqnYO5PPL\\\\\\\/TRtiW0o7zNcku6LoR+RgRptifmECIPILJOfVGBhqZxR1RcpSZEU4o9iX8cg7hoD6cYg5HojnTD5HtHcHl9sIDySo84Di5fasKtsdtzDL4kJmAOKKbryrHwJ\\\\\\\/JZOVdhxjWR76KzLUD2KlK2UWdf2RWlTZUwMOagAS9cmUtHoJvzr+gH9qw8amBGaqn9i1lvpoXqrxSdKs4mcvglvf\\\\\\\/5yTcwh3JRcEeIMcA\\\\\\\/4vL0sym8L9frPBWwkPIBPfDlbp6FlN3U91eDvbE\\\u0022,\\\u0022iv\\\u0022:\\\u0022th93H1vbQWM7XEa3A4v4Hw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e845e38ff291ad73a01b98ceec8271ec64f705d102d34161eff54de4a292b450\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459962 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1075" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e_CtlTVLgB4-EN1tBa9Dn6DDDkCcXkpmXbt135S8nSo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1075" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r6flMHMN4DRoSLPPE_PXBn2b84kvc4ZHMP8ghPlDkuI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1065" title="Response of Polygenic Traits Under Stabilizing Selection and Mutation When Loci Have Unequal Effects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1081" title="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459962" id="node459962" data-pisa="ggg;5/6/1075" data-pisa-master="ggg;g3.115.018127" data-apath="/ggg/5/6/1075.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Network Reconstruction by Integration of Prior Biological Knowledge</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0843-7592" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yupeng Li</span> and <span class="highwire-citation-author" data-delta="1">Scott A. Jackson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1075-1079;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018127 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459962" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yupeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Statistics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yupeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYupeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0843-7592" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yupeng Li</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Scott A. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BA.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BA.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sjackson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">sjackson@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1075" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459962" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459962/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1075.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459962" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459962/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1075.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459962" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459962/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1075.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459962" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459962/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">With the development of high-throughput genomic technologies, large, genome-wide datasets have been collected, and the integration of these datasets should provide large-scale, multidimensional, and insightful views of biological systems. We developed a method for gene association network construction based on gene expression data that integrate a variety of biological resources. Assuming gene expression data are from a multivariate Gaussian distribution, a graphical lasso (glasso) algorithm is able to estimate the sparse inverse covariance matrix by a lasso (L<sub>1</sub>) penalty. The inverse covariance matrix can be seen as direct correlation between gene pairs in the gene association network. In our work, instead of using a single penalty, different penalty values were applied for gene pairs based on <em>a priori</em> knowledge as to whether the two genes should be connected. The <em>a priori</em> information can be calculated or retrieved from other biological data, <em>e.g.</em>, Gene Ontology similarity, protein-protein interaction, gene regulatory network. By incorporating prior knowledge, the weighted graphical lasso (wglasso) outperforms the original glasso both on simulations and on data from Arabidopsis. Simulation studies show that even when some prior knowledge is not correct, the overall quality of the wglasso network was still greater than when not incorporating that information, <em>e.g.</em>, glasso.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-network" class="hw-term hw-article-keyword hw-article-keyword-gene-network" rel="nofollow">gene network</a></li><li class="kwd"><a href="/keyword/graphical-model" class="hw-term hw-article-keyword hw-article-keyword-graphical-model" rel="nofollow">graphical model</a></li><li class="kwd"><a href="/keyword/prior-knowledge" class="hw-term hw-article-keyword hw-article-keyword-prior-knowledge" rel="nofollow">prior knowledge</a></li><li class="kwd"><a href="/keyword/systems-biology-and-gene-expression" class="hw-term hw-article-keyword hw-article-keyword-systems-biology-and-gene-expression" rel="nofollow">systems biology and gene expression</a></li></ul><p id="p-3">A key challenge for biology is to understand the complex molecular interactions of genes in a living cell (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barabasi and Oltvai 2004</a>). Analysis of gene networks provides a global view of these interactions and can provide biologists with a better understanding of complex biological systems (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Friedman 2004</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Karlebach and Shamir 2008</a>). Also, by the use of gene networks, the guilt-by-association paradigm can be applied to infer the biological function of unknown genes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wolfe <em>et al.</em> 2005</a>).</p><p id="p-4">Gene expression data, which is relatively easy to generate or to collect from databases, has been used to infer gene networks. A variety of gene network reconstruction methods based on gene expression data have been developed, <em>e.g.</em>, regression, mutual information, correlation Bayesian network, meta predicators, and others (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Marbach <em>et al.</em> 2012</a>). Here, we focus on the Gaussian graphical model (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dempster 1972</a>). In this approach, gene expression is assumed to follow a multivariate Gaussian distribution <em>N(Î¼</em>, Î£<em>)</em>, and Gaussian Markov random fields have been used to infer the structure of networks from gene expression data (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Liu and Ihler 2011</a>). A gene association network can be seen as an undirected graph <em>G = (V</em>, <em>E)</em>, where <em>V = {v<sub>p</sub>}</em> is the vertex set representing genes and <em>E = {e<sub>ij</sub>}</em> is the edge set representing association relations between pairs of genes. In an unweighted network, if <em>e<sub>ij</sub></em> = 0, genes <em>i</em> and <em>j</em> are conditionally independent given all other genes. The HammersleyâClifford theorem implies that zeros in the inverse covariance matrix of a multivariate Gaussian distribution indicate absent edges in the corresponding graphical model (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Besag 1974</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lauritzen 1996</a>). Therefore, the problem of estimating the gene association network based on gene expression data can be transferred to estimating <em>Î£<sup>â1</sup></em> or selecting nonzero entries in <em>Î£<sup>â1</sup></em>. Many studies, however, have used Pearson correlation coefficients between pairs of genes to infer network structure; Pearson correlation coefficients correspond to the covariance matrix Î£ and cannot infer the true structure.</p><p id="p-5">An accurate inference of biological network using Gaussian graphical model is challenging for two main reasons. The first is that most genome-scale datasets are highly dimensional. Given <em>p</em> genes, there are possible <em>p(p â 1)/2</em> edges, but gene expression data often have a limited number of samples. When the gene or locus number is much higher than the sample size in a dataset, a traditional maximum-likelihood estimate for covariance variance is often not appropriate (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Schafer and Strimmer 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Uhler 2012</a>). Alternative methods have been developed for highly dimensional datasets (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meinshausen and Buhlmann 2006</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Friedman <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yuan 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cai <em>et al.</em> 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ravikumar <em>et al.</em> 2011</a>), most of which have used Lasso, a shrinkage and selection method using <em>L<sub>1</sub></em> regularization, a popular approach to deal with highly dimensional datasets (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tibshirani 1996</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hastie <em>et al.</em> 2009</a>). These methods have been shown to be able to asymptotically and consistently estimate the set of non-zero elements of <em>Î£<sup>â1</sup></em>.</p><p id="p-6">The second major difficulty is the lack of efficient methods to integrate multiple levels of biological data to enhance model accuracy. Increasing amounts of biological data have been collected, especially with the development of high-throughput technologies, <em>e.g.</em>, microarrays and next-generation sequencing, which provide an unprecedented opportunity to explore biological systems. We now have access to genomic, epigenomic, transcriptomic, proteomic, metabolomic, and phenomic data, and careful analysis and integration of these genome-scale datasets should provide large-scale, multidimensional, and insightful views into biological systems (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Joyce and Palsson 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hawkins <em>et al.</em> 2010</a>). Also, we have a variety of resources that serve as indicators of the functional association of any two genes, <em>e.g.</em>, Kyoto Encyclopedia of Genes and Genomes pathway, Gene Ontology (GO) similarity, proteinâprotein interaction, co-occurrence in literature, gene network generated from other methods, and association of orthologous genes. Integrating potentially reliable information from other sources should increase the accuracy of network reconstruction (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Imoto <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mostafavi <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christley <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2014</a>). Here, we present a statistical algorithm based on the Gaussian graphical model for gene association network reconstruction using gene expression data, which is able to solve these two challenges to allow a more thorough understanding of complex biological systems.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">Let <em>Î = Î£<sup>â1</sup></em> and S be the empirical covariance matrix, then by applying <em>L<sub>1</sub></em> penalty to the original log-likelihood for estimating <em>Î</em>, the problem of estimating <em>Î</em> becomes finding the <em>Î</em> which maximize the formula<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-1.gif"/></noscript></span></span>where <em>tr</em> is the trace, <em>i.e.</em>, the sum of the elements on the matrix diagonal, and <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-2.gif"/></noscript></span></span> is the <em>L<sub>1</sub></em> norm, <em>i.e.</em>, the sum of the absolute values of the elements of <em>Î£<sup>â1</sup></em>, and <em>Ï</em> is the penalty parameter. When <em>Ï = 0</em>, it is the normal maximum likelihood. When <em>Ï &gt; 0</em>, some elements in <em>Î</em> will be shrunk to zero. The sparsity of the estimated graph increases when <em>Ï</em> increases. For a fixed <em>Ï</em>, the graphical lasso (glasso) algorithm can be used to quickly solve the equation using a block-coordinate method (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Friedman <em>et al.</em> 2008</a>). The optimal <em>Ï</em> can be chosen empirically or tuned by cross-validation, Bayesian information criterion (BIC), or other methods (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Friedman <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Foygel and Drton 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Liu <em>et al.</em> 2010</a>).</p><p id="p-8">The Lasso regression can be interpreted as a Bayesian regression with a Laplace prior distribution, <em>Lp(0</em>, <em>Ï)</em>. In Bayesian statistics, prior information can be integrated into the model by changing the parameter in the prior distribution, and the posterior distribution should better approximate the true distribution of the data. Therefore, for the glasso algorithm, instead of using a single penalty parameter, it is reasonable to specify different amounts of penalties for different elements in <em>Î</em> based on <em>a priori</em> information as to whether two genes are associated, or not. A smaller penalty can be given if <em>a priori</em> information indicates that they are linked. Then, the log-likelihood becomes<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-3.gif"/></noscript></span></span>where <em>P</em> is the prior information matrix and * indicates component-wise multiplication, and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-4.gif"/></noscript></span></span>. Larger values for an element in <em>P</em> generate larger penalty values and represent weaker association of two genes based on <em>prior</em>i information. We name this updated version of glasso as weighted graphical lasso (wglasso), which is not only stable for high-dimensional datasets by utilization of Lasso but also more accurate than the original glasso through the integration of prior information.</p><p id="p-9">The prior matrix can be obtained in numerous ways. For example, the GO semantic similarity between genes can be calculated using tools GOSemSim (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Yu <em>et al.</em> 2010</a>) or GOssTo (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Yu <em>et al.</em> 2010</a>), Then, the inverse of similarities can be implemented in the prior matrix, as a high similarity means a low penalty in our model. Other types of networks, <em>e.g.</em>, protein-protein interaction and gene regulatory networks, can be directly transferred to a prior matrix after inverse transformation of the weights of the network. Some databases estimate the functional association between genes using data mining and text mining and can be valuable and reliable resources for generating a prior matrix, <em>e.g.</em>, STING (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Von Mering <em>et al.</em> 2005</a>) and AraNet (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lee <em>et al.</em> 2010</a>).</p><p id="p-10">Although previous efforts to use lasso for incorporating <em>a priori</em> knowledge have been made, none is identical to our proposed method. For example, most of the previously described algorithms partitioned the gene network reconstruction into a number of linear regression problems (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anjum <em>et al.</em> 2009</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Charbonnier <em>et al.</em> 2010</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2013</a>); however, it has been shown this type of algorithms is less accurate than glasso (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Friedman <em>et al.</em> 2008</a>).</p></div><div class="section results" id="sec-2"><h2 class="">Results and Discussions</h2><div id="sec-3" class="subsection"><h3>Simulation studies demonstrate that wglasso is more reliable than glasso</h3><p id="p-11">Because most biological networks are scale-free (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barabasi and Oltvai 2004</a>), a scale-free network with <em>p</em> genes and gene expression data with <em>n</em> samples based on the network topology are generated using the âhugeâ package in R (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zhao <em>et al.</em> 2012</a>). A prior network with precision ratio <em>x</em>, <em>x â [0,1]</em> also was generated. A prior network is a weighted network, in which each edge corresponds to the prior information that two genes are functionally associated and the edge weight indicates the strength of the information. Precision ratio = <em>x</em> with <em>x &gt; 0</em> means <em>p*100</em> percent of prior edges are correct, and the remaining <em>(1 â p)*100</em> edges are incorrect. The edge number of the prior network is set to the same number of true edges. The edge weights of the prior network were randomly generated from the uniform distribution, <em>U(0,1)</em>. A special case is precision ratio = 0, which indicates no prior information and the prior network will not contain any edges. In this case, wglasso is equivalent to glasso. Once the empirical covariance matrix <em>S</em> is calculated from the expression data and the prior information matrix <em>P</em> retrieved from the prior network, we estimate the true network using glasso and wglasso. In order to find the optimal penalty parameter, <em>Ï</em>, wglasso networks were estimated under a sequence of <em>Ï</em> values from 0 to 1 with 0.01 intervals. Then we selected the network most similar to the true network. The Matthews correlation coefficient (MCC) was used to measure the similarity between the estimated and true networks (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Matthews 1975</a>), and was calculated as follows based on 2 Ã 2 contingency table,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-5.gif"/></noscript></span></span>where TP, TN, FP, and FN are the number of true positives, true negatives, false positives, and false negatives, respectively. A penalty parameter that maximized MCC (maxMCC) was considered optimal. A single simulation with 100 genes, 50 samples and a 0.7 precision ratio is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Demonstration of a single simulation using glasso and wglasso. (A) The true network with scale-free property. (B) Heatmap of gene expression data. The X-axis represents genes and the Y-axis represents samples. (C) The prior network, representing prior information of genes&#x2019; associations with precision ratio 0.7. Black edges are correct association information; edges that exist in prior information but not in the true network are in red, and gray edges are missed associations in the prior information. (D) The estimated network using glasso. The edge colors have the same meaning as prior network. (E) The estimated network using wglasso. The edge colors have the same meaning as prior network." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508628168" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Demonstration of a single simulation using glasso and wglasso. (A) The true network with scale-free property. (B) Heatmap of gene expression data. The X-axis represents genes and the Y-axis represents samples. (C) The prior network, representing prior information of genes&#x2019; associations with precision ratio 0.7. Black edges are correct association information; edges that exist in prior information but not in the true network are in red, and gray edges are missed associations in the prior information. (D) The estimated network using glasso. The edge colors have the same meaning as prior network. (E) The estimated network using wglasso. The edge colors have the same meaning as prior network.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1075/F1.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1075/F1.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466191" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Demonstration of a single simulation using glasso and wglasso. (A) The true network with scale-free property. (B) Heatmap of gene expression data. The X-axis represents genes and the Y-axis represents samples. (C) The prior network, representing prior information of genesâ associations with precision ratio 0.7. Black edges are correct association information; edges that exist in prior information but not in the true network are in red, and gray edges are missed associations in the prior information. (D) The estimated network using glasso. The edge colors have the same meaning as prior network. (E) The estimated network using wglasso. The edge colors have the same meaning as prior network.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Parameter selection process of glasso and wglasso Matthews correlation coefficient (MCC; solid lines) and Bayesian information criterion (BIC; dashed lines) values of the estimated networks under different penalty parameters (Rho) using glasso (blue lines) and wglasso (red lines). Triangles are points that maximize MCC values and minimize BIC values under the corresponding penalty parameters, considered to be optimal penalty parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508628168" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Parameter selection process of glasso and wglasso Matthews correlation coefficient (MCC; solid lines) and Bayesian information criterion (BIC; dashed lines) values of the estimated networks under different penalty parameters (Rho) using glasso (blue lines) and wglasso (red lines). Triangles are points that maximize MCC values and minimize BIC values under the corresponding penalty parameters, considered to be optimal penalty parameters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1075/F2.medium.gif" width="440" height="223"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1075/F2.medium.gif" width="440" height="223"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466288" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-13" class="first-child">Parameter selection process of glasso and wglasso Matthews correlation coefficient (MCC; solid lines) and Bayesian information criterion (BIC; dashed lines) values of the estimated networks under different penalty parameters (Rho) using glasso (blue lines) and wglasso (red lines). Triangles are points that maximize MCC values and minimize BIC values under the corresponding penalty parameters, considered to be optimal penalty parameters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Additional simulations were performed with a variety of sample sizes and precision ratios in order to systematically evaluate wglasso. The simulation was repeated 100 times for each combination of sample size and precision ratio. The results show that reconstructed networks with <em>a priori</em> information had a significantly greater maxMCC than without <em>a priori</em> information, indicating superior performance of wglasso (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In most cases, wglasso outperformed glasso, even when most of the prior knowledge was incorrect. If the sample size was high and the precision ratio low, <em>e.g.</em>, sample size = 300 and precision ratio = 0.2, it is possible that an excess of incorrect prior information would be harmful. However, in real situations, the sample size is often much lower than the gene number and misleading <em>a priori</em> information based on experimental studies is likely to be low. Moreover, highly accurate prior knowledge results in more accurately reconstructed networks. In practice, it would be possible to integrate multiple resources to increase the reliability of prior information.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation to compare glasso and wglasso. The simulation was repeated 100 times for each combination of sample sizes (n) and precision ratio of the prior information. Gene number = 100. In each simulation, the maximum Matthews correlation coefficient (maxMCC) of estimated networks from different penalty parameters is recorded. The Y-axis shows the mean of maxMCC from the 100 simulations. The error bars are 95% confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508628168" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation to compare glasso and wglasso. The simulation was repeated 100 times for each combination of sample sizes (n) and precision ratio of the prior information. Gene number = 100. In each simulation, the maximum Matthews correlation coefficient (maxMCC) of estimated networks from different penalty parameters is recorded. The Y-axis shows the mean of maxMCC from the 100 simulations. The error bars are 95% confidence intervals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1075/F3.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1075/F3.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466253" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-15" class="first-child">Simulation to compare glasso and wglasso. The simulation was repeated 100 times for each combination of sample sizes (n) and precision ratio of the prior information. Gene number = 100. In each simulation, the maximum Matthews correlation coefficient (maxMCC) of estimated networks from different penalty parameters is recorded. The Y-axis shows the mean of maxMCC from the 100 simulations. The error bars are 95% confidence intervals.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Tuning the penalty parameter using BIC</h3><p id="p-16">In real situations, the true network is unknown; thus, maxMCC cannot be used to select the optimal penalty parameter. Instead, one can use cross-validation, BIC, extended BIC (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Foygel and Drton 2010</a>), and a stability approach for regularization selection (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Liu <em>et al.</em> 2010</a>). Cross-validation, stability approach for regularization selection, and BIC conduct many subsampling or permutations and thus are computationally intensive. Extended BIC tends to select very sparse networks, and often no edges are found. For a scale-free network, another method is to test whether the log transformed degree distribution has a linear relationship (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Langfelder and Horvath 2008</a>). The R-squared values from linear regressions generally increase as the network gets sparser, and the optimal fit is selected at the point where the increase trend slows down. The selection is usually done visually, so the method is not practical for simulations. Also, the optimal parameter varies by dataset and individual interpretation.</p><p id="p-17">BIC is one of the standard methods for choosing the regularization parameter, and works well in our scale-free network simulations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The BIC for Gaussian graphical model takes the form<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-6.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-7.gif"/></noscript></span></span> is the edge number, n the sample size, and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-8.gif"/></noscript></span></span> the log-likelihood function simplified from<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-9.gif"/></noscript></span></span>Simulation showed that the difference between the MCC of reconstructed networks based on BIC and the maxMCC was small, especially when sample size was relatively small (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Selection of the optimal penalty parameter based on minimum Bayesian information criterion (BIC) simulation was repeated 100 times for each combination of sample sizes (n) and precision ratio of the prior information. Gene number = 100. For each simulation, the maximum Matthews correlation coefficient (MCC) value (maxMCC) of estimated networks from different penalty parameters is shown. The MCC value of the estimated network based on minimum BIC is also recorded, called BICMCC. The difference between two values (maxMCC &#x2212; BICMCC) is used to evaluate two parameter selection methods. The Y-axis shows the mean of maxMCC &#x2212; BICMCC from the 100 simulations. The error bars represent 95% confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508628168" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Selection of the optimal penalty parameter based on minimum Bayesian information criterion (BIC) simulation was repeated 100 times for each combination of sample sizes (n) and precision ratio of the prior information. Gene number = 100. For each simulation, the maximum Matthews correlation coefficient (MCC) value (maxMCC) of estimated networks from different penalty parameters is shown. The MCC value of the estimated network based on minimum BIC is also recorded, called BICMCC. The difference between two values (maxMCC &#x2212; BICMCC) is used to evaluate two parameter selection methods. The Y-axis shows the mean of maxMCC &#x2212; BICMCC from the 100 simulations. The error bars represent 95% confidence intervals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1075/F4.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1075/F4.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1075/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466286" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-18" class="first-child">Selection of the optimal penalty parameter based on minimum Bayesian information criterion (BIC) simulation was repeated 100 times for each combination of sample sizes (n) and precision ratio of the prior information. Gene number = 100. For each simulation, the maximum Matthews correlation coefficient (MCC) value (maxMCC) of estimated networks from different penalty parameters is shown. The MCC value of the estimated network based on minimum BIC is also recorded, called BICMCC. The difference between two values (maxMCC â BICMCC) is used to evaluate two parameter selection methods. The Y-axis shows the mean of maxMCC â BICMCC from the 100 simulations. The error bars represent 95% confidence intervals.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Application to experimental data</h3><p id="p-19">We applied wglasso to gene expression data from the Eukaryotic species, <em>Arabidopsis thaliana</em>. Gene expression data of 795 genes related to isoprenoid pathways from 118 microarray experiments were collected (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wille <em>et al.</em> 2004</a>). Prior information was obtained from AraNet, a probabilistic network of functional associations among 19,647 <em>Arabidopsis</em> genes (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Lee <em>et al.</em> 2010</a>). A total of 701 of 795 genes have functional associations in AraNet. The edge weight is the likelihood score, calculated from variety of resources that indicates a functional association between two genes. The likelihood scores range from 0 to 5, so they were rescaled to a range of 0.5 to 1 using the formula,<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1075/embed/mml-math-10.gif"/></noscript></span></span>High likelihood scores will become low penalty values after rescaling. Pairs of genes without prior information have zero values in the prior matrix. The optimal penalty parameter was selected based on BIC. The network reconstructed using wglasso has 16,759 edges, whereas the glasso network has 19,331 edges. The MCC values were calculated by comparing the reconstructed networks with an independent benchmark network, which is the same benchmark set that was used to test the prediction performance of AraNet (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Lee <em>et al.</em> 2010</a>). The MCC of the estimated network with wglasso was 0.184, higher than that with glasso, which was 0.033.</p><p id="p-20">By incorporating prior knowledge, weighted graphical lasso (wglasso) outperforms glasso both on simulation studies and data from <em>Arabidopsis</em>. Simulation studies showed that even when some prior knowledge was incorrect, the overall quality of network from wglasso network was higher than that from glasso. Moreover, the more accurate the prior knowledge, the better the reconstructed network. This method increases gene network reconstruction accuracy and will allow researchers to better study networks in complex biological systems and their interaction with external factors, <em>e.g.</em>, the environment.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-21">We would like to acknowledge funding from the National Science Foundation (MCB 1339194).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-22"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-02-05"><span class="received-label">Received </span>February 5, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-28"><span class="accepted-label">Accepted </span>March 28, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2015 Li and Jackson</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.1" data-doi="10.1093/bioinformatics/btp485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anjum</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doucet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A boosting approach to structure learning of graphs with and without prior knowledge.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2929</span>â<span class="cit-lpage">2936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp485%26rft_id%253Dinfo%253Apmid%252F19696047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjIvMjkyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.2" data-doi="10.1038/nrg1272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barabasi</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oltvai</span>  <span class="cit-name-given-names">Z. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Network biology: understanding the cellâs functional organization.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">101</span>â<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBarab%25C3%25A1si%26rft.auinit1%253DA.%2BL.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D101%26rft.epage%253D113%26rft.atitle%253DNetwork%2Bbiology%253A%2Bunderstanding%2Bthe%2Bcell%2527s%2Bfunctional%2Borganization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1272%26rft_id%253Dinfo%253Apmid%252F14735121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14735121&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188602400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besag</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span>â<span class="cit-article-title">Spatial interaction and the statistical analysis of lattice systems.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">192</span>â<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D36%26rft.spage%253D192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.4" data-doi="10.1198/jasa.2011.tm10155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A constrained L(1) minimization approach to sparse precision matrix estimation.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">594</span>â<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D106%26rft.spage%253D594%26rft_id%253Dinfo%253Adoi%252F10.1198%252Fjasa.2011.tm10155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/jasa.2011.tm10155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charbonnier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiquet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambroise</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Weighted-lasso for structured network inference from time course data.</span> <abbr class="cit-jnl-abbrev">Stat Appl Genet Mol</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">Article 15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat%2BAppl%2BGenet%2BMol%26rft.volume%253D9%26rft.spage%253D15Article%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.6" data-doi="10.1371/journal.pcbi.1003666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairelli</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilicoglu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rindflesch</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Augmenting microarray data with literature-based knowledge to enhance gene regulatory network inference.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1003666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol.%26rft.volume%253D10%26rft.spage%253De1003666%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003666%26rft_id%253Dinfo%253Apmid%252F24921649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24921649&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.7" data-doi="10.1371/journal.pone.0006799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Incorporating existing network information into gene network inference.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChristley%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6799%26rft.epage%253De6799%26rft.atitle%253DIncorporating%2Bexisting%2Bnetwork%2Binformation%2Binto%2Bgene%2Bnetwork%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006799%26rft_id%253Dinfo%253Apmid%252F19710931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006799&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19710931&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.8" data-doi="10.2307/2528966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempster</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span>â<span class="cit-article-title">Covariance selection.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">157</span>â<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D28%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972L821100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-book" id="cit-5.6.1075.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foygel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drton</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Extended Bayesian information criteria for Gaussian graphical models</span>. <span class="cit-source">Adv. Neural Info. Process. Syst.</span> <span class="cit-vol">23</span>: <span class="cit-fpage">604</span>â<span class="cit-lpage">612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BNeural%2BInfo.%2BProcess.%2BSyst.%26rft.volume%253D23%26rft.spage%253D604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.10" data-doi="10.1093/biostatistics/kxm045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Sparse inverse covariance estimation with the graphical lasso.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">432</span>â<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxm045%26rft_id%253Dinfo%253Apmid%252F18079126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjc6IjkvMy80MzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.11" data-doi="10.1126/science.1094068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Inferring cellular networks using probabilistic graphical models.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">799</span>â<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFriedman%26rft.auinit1%253DN.%26rft.volume%253D303%26rft.issue%253D5659%26rft.spage%253D799%26rft.epage%253D805%26rft.atitle%253DInferring%2BCellular%2BNetworks%2BUsing%2BProbabilistic%2BGraphical%2BModels%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1094068%26rft_id%253Dinfo%253Apmid%252F14764868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1OS83OTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-book" id="cit-5.6.1075.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-source">The Elements of Statistical Learning</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Next-generation genomics: an integrative approach.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">476</span>â<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHawkins%26rft.auinit1%253DR.%2BD.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D476%26rft.epage%253D486%26rft.atitle%253DNext-generation%2Bgenomics%253A%2Ban%2Bintegrative%2Bapproach.%26rft_id%253Dinfo%253Apmid%252F20531367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20531367&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.14" data-doi="10.1142/S021972000400048X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tashiro</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Combining microarrays and biological knowledge for estimating gene networks via Bayesian networks.</span> <abbr class="cit-jnl-abbrev">J. Bioinform. Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">77</span>â<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbioinformatics%2Band%2Bcomputational%2Bbiology%26rft.stitle%253DJ%2BBioinform%2BComput%2BBiol%26rft.aulast%253DImoto%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D98%26rft.atitle%253DCombining%2Bmicroarrays%2Band%2Bbiological%2Bknowledge%2Bfor%2Bestimating%2Bgene%2Bnetworks%2Bvia%2Bbayesian%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1142%252FS021972000400048X%26rft_id%253Dinfo%253Apmid%252F15272434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/S021972000400048X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15272434&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.15" data-doi="10.1038/nrm1857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palsson</span>  <span class="cit-name-given-names">B. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">The model organism as a system: integrating âomicsâ data sets.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">198</span>â<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DJoyce%26rft.auinit1%253DA.%2BR.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D198%26rft.epage%253D210%26rft.atitle%253DThe%2Bmodel%2Borganism%2Bas%2Ba%2Bsystem%253A%2Bintegrating%2B%2527omics%2527%2Bdata%2Bsets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1857%26rft_id%253Dinfo%253Apmid%252F16496022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1857&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16496022&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235590500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.16" data-doi="10.1038/nrm2503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlebach</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shamir</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Modelling and analysis of gene regulatory networks.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">770</span>â<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DKarlebach%26rft.auinit1%253DG.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D770%26rft.epage%253D780%26rft.atitle%253DModelling%2Band%2Banalysis%2Bof%2Bgene%2Bregulatory%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2503%26rft_id%253Dinfo%253Apmid%252F18797474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18797474&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259402800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.17" data-doi="10.1186/1471-2105-9-559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">WGCNA: an R package for weighted correlation network analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLangfelder%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.spage%253D559%26rft.epage%253D559%26rft.atitle%253DWGCNA%253A%2Ban%2BR%2Bpackage%2Bfor%2Bweighted%2Bcorrelation%2Bnetwork%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-559%26rft_id%253Dinfo%253Apmid%252F19114008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19114008&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-book" id="cit-5.6.1075.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lauritzen</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-source">Graphical Models</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.19" data-doi="10.1038/nbt.1603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambaru</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thakkar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhee</span>  <span class="cit-name-given-names">S. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Rational association of genes with traits using a genome-scale gene network for <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">149</span>â<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLee%26rft.auinit1%253DI.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D149%26rft.epage%253D156%26rft.atitle%253DRational%2Bassociation%2Bof%2Bgenes%2Bwith%2Btraits%2Busing%2Ba%2Bgenome-scale%2Bgene%2Bnetwork%2Bfor%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1603%26rft_id%253Dinfo%253Apmid%252F20118918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20118918&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274317200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-book" id="cit-5.6.1075.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Stability approach to regularization selection (StARS) for high dimensional graphical models.</span> Adv. Neural Info. Process. Syst. <span class="cit-vol">23</span>: <span class="cit-fpage">1432</span>â<span class="cit-lpage">1440</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-confproc" id="cit-5.6.1075.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ihler</span>  <span class="cit-name-given-names">A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Learning scale free networks by reweighted L1 regularization.</span> <span class="cit-source">J. Machine Learning Res. Proc. Track</span>Â <span class="cit-vol">15</span>: <span class="cit-fpage">40</span>â<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMachine%2BLearning%2BRes.%2BProc.%2BTrack%26rft.volume%253D15%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.22" data-doi="10.1038/nmeth.2016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marbach</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costello</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuffner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prill</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Wisdom of crowds for robust gene network inference.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">796</span>â<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMarbach%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D796%26rft.epage%253D804%26rft.atitle%253DWisdom%2Bof%2Bcrowds%2Bfor%2Brobust%2Bgene%2Bnetwork%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2016%26rft_id%253Dinfo%253Apmid%252F22796662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22796662&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307015700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.23" data-doi="10.1016/0005-2795(75)90109-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span>â<span class="cit-article-title">Comparison of the predicted and observed secondary structure of T4 phage lysozyme.</span> <abbr class="cit-jnl-abbrev">Biochim .Biophys. Acta</abbr> <span class="cit-vol">405</span>: <span class="cit-fpage">442</span>â<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DMatthews%26rft.auinit1%253DB.%2BW.%26rft.volume%253D405%26rft.issue%253D2%26rft.spage%253D442%26rft.epage%253D451%26rft.atitle%253DComparison%2Bof%2Bthe%2Bpredicted%2Band%2Bobserved%2Bsecondary%2Bstructure%2Bof%2BT4%2Bphage%2Blysozyme.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0005-2795%252875%252990109-9%26rft_id%253Dinfo%253Apmid%252F1180967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0005-2795(75)90109-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1180967&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AU09000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.24" data-doi="10.1214/009053606000000281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meinshausen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buhlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">High-dimensional graphs and variable selection with the lasso.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1436</span>â<span class="cit-lpage">1462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D34%26rft.spage%253D1436%26rft_id%253Dinfo%253Adoi%252F10.1214%252F009053606000000281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/009053606000000281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239471300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.25" data-doi="10.1186/gb-2008-9-s1-s4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mostafavi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warde-Farley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grouios</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">GeneMANIA: a real-time multiple association network integration algorithm for predicting gene function.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">S4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D9%26rft.spage%253DS4%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-s1-s4%26rft_id%253Dinfo%253Apmid%252F18613948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-s1-s4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18613948&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.26" data-doi="10.1214/11-EJS631"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ravikumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wainwright</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raskutti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">High-dimensional covariance estimation by minimizing L(1)-penalized log-determinant divergence.</span> <abbr class="cit-jnl-abbrev">Electron J Stat</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">935</span>â<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron%2BJ%2BStat%26rft.volume%253D5%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.1214%252F11-EJS631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/11-EJS631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">A shrinkage approach to large-scale covariance matrix estimation and implications for functional genomics.</span> <abbr class="cit-jnl-abbrev">Stat Appl Genet Mol</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">Article 32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat%2BAppl%2BGenet%2BMol%26rft.volume%253D4%26rft.spage%253DArticle32%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Regression shrinkage and selection via the lasso.</span> <abbr class="cit-jnl-abbrev">J Roy Stat Soc B Met</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>â<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BRoy%2BStat%2BSoc%2BB%2BMet%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.29" data-doi="10.1214/11-AOS957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uhler</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Geometry of maximum likelihood estimation in Gaussian graphical models.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">238</span>â<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D40%26rft.spage%253D238%26rft_id%253Dinfo%253Adoi%252F10.1214%252F11-AOS957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/11-AOS957&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.30" data-doi="10.1093/nar/gki005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Mering</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooper</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krupp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">STRING: known and predicted proteinâprotein associations, integrated and transferred across organisms.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">D433</span>â<span class="cit-lpage">D437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253Dvon%2BMering%26rft.auinit1%253DC.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD433%26rft.epage%253DD437%26rft.atitle%253DSTRING%253A%2Bknown%2Band%2Bpredicted%2Bprotein-protein%2Bassociations%252C%2Bintegrated%2Band%2Btransferred%2Bacross%2Borganisms%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki005%26rft_id%253Dinfo%253Apmid%252F15608232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8xL0Q0MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.31" data-doi="10.1093/bioinformatics/btt443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas</span>  <span class="cit-name-given-names">F. A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Incorporating prior knowledge into gene network study.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2633</span>â<span class="cit-lpage">2640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt443%26rft_id%253Dinfo%253Apmid%252F23956306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMjAvMjYzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.32" data-doi="10.1186/gb-2004-5-11-r92"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wille</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vranova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furholz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laule</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Sparse graphical gaussian modeling of the isoprenoid gene network in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DWille%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253DR92%26rft.epage%253DR92%26rft.atitle%253DSparse%2Bgraphical%2BGaussian%2Bmodeling%2Bof%2Bthe%2Bisoprenoid%2Bgene%2Bnetwork%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-11-r92%26rft_id%253Dinfo%253Apmid%252F15535868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-11-r92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15535868&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.33" data-doi="10.1186/1471-2105-6-227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohane</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butte</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Systematic survey reveals general applicability of âguilt-by-associationâ within gene coexpression networks.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DWolfe%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D6%26rft.spage%253D227%26rft.epage%253D227%26rft.atitle%253DSystematic%2Bsurvey%2Breveals%2Bgeneral%2Bapplicability%2Bof%2B%2526quot%253Bguilt-by-association%2526quot%253B%2Bwithin%2Bgene%2Bcoexpression%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-227%26rft_id%253Dinfo%253Apmid%252F16162296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16162296&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.34" data-doi="10.1093/bioinformatics/btq064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bo</span>  <span class="cit-name-given-names">X. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">GOSemSim: an R package for measuring semantic similarity among go terms and gene products.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">976</span>â<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq064%26rft_id%253Dinfo%253Apmid%252F20179076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzk3NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">High dimensional inverse covariance matrix estimation via linear programming.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2261</span>â<span class="cit-lpage">2286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D11%26rft.spage%253D2261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1075.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafferty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The huge package for high-dimensional undirected graph estimation in R.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1059</span>â<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D13%26rft.spage%253D1059%26rft_id%253Dinfo%253Apmid%252F26834510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26834510&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1075.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1075.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1065" title="Response of Polygenic Traits Under Stabilizing Selection and Mutation When Loci Have Unequal Effects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1081" title="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1075.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ppK7ITG8yLIOgb98It3sYMwxIW83rE9LHXPqTvw4Zew" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1075" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1075" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1075" class="active" data-icon-position="" data-hide-link-title="0">Gene Network Reconstruction by Integration of Prior Biological Knowledge</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459962" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-reFYUc0-4Zugr96V2tFgilwjSO2qaQR5SOVM9VZzscw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1075.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1075" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form--7POuOd4tj_eIpZeAKAKL4JyhciBO-t3P3FqLw6XLLg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1075" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1075.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459962" id="node459962--2" data-pisa="ggg;5/6/1075" data-pisa-master="ggg;g3.115.018127" data-apath="/ggg/5/6/1075.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Network Reconstruction by Integration of Prior Biological Knowledge</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0843-7592" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yupeng Li</span> and <span class="highwire-citation-author" data-delta="1">Scott A. Jackson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1075-1079;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018127 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459962--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yupeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Statistics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yupeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYupeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0843-7592" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yupeng Li</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Scott A. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BA.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BA.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sjackson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">sjackson@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1075" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1075&amp;title=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1075&amp;title=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1075&amp;title=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1075&amp;text=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1075&amp;title=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1075&amp;t=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1075&amp;title=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1075&amp;title=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459962/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459962" id="node459962--3" data-pisa="ggg;5/6/1075" data-pisa-master="ggg;g3.115.018127" data-apath="/ggg/5/6/1075.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Network Reconstruction by Integration of Prior Biological Knowledge</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0843-7592" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yupeng Li</span> and <span class="highwire-citation-author" data-delta="1">Scott A. Jackson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1075-1079;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018127 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459962--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yupeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Statistics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yupeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYupeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0843-7592" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yupeng Li</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Scott A. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute of Plant Breeding, Genetics and Genomics, University of Georgia, Athens, Georgia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BA.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BA.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sjackson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">sjackson@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1075&amp;rft.epage=1079&amp;rft.atitle=Gene%20Network%20Reconstruction%20by%20Integration%20of%20Prior%20Biological%20Knowledge&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Li%2C+Yupeng&amp;rft.au=Jackson%2C+Scott+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459962/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459962/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459962/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459962/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459962/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459962/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459962/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459962/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459962/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459962/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459962/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459962/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1075" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1075.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1075.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/6/1075.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
