<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1187" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1187.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1187.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1187.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1187.atom" />
<meta name="HW.pisa" content="ggg;5/6/1187" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of Gossypium spp." />
<meta name="DC.Identifier" content="10.1534/g3.115.018416" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Hulse-Kemp et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-throughput genotyping arrays provide a standardized resource for plant breeding communities that are useful for a breadth of applications including high-density genetic mapping, genome-wide association studies (GWAS), genomic selection (GS), complex trait dissection, and studying patterns of genomic diversity among cultivars and wild accessions. We have developed the CottonSNP63K, an Illumina Infinium array containing assays for 45,104 putative intraspecific single nucleotide polymorphism (SNP) markers for use within the cultivated cotton species Gossypium hirsutum L. and 17,954 putative interspecific SNP markers for use with crosses of other cotton species with G. hirsutum . The SNPs on the array were developed from 13 different discovery sets that represent a diverse range of G. hirsutum germplasm and five other species: G. barbadense L., G. tomentosum Nuttal × Seemann, G. mustelinum Miers × Watt, G. armourianum Kearny, and G. longicalyx J.B. Hutchinson and Lee. The array was validated with 1,156 samples to generate cluster positions to facilitate automated analysis of 38,822 polymorphic markers. Two high-density genetic maps containing a total of 22,829 SNPs were generated for two F2 mapping populations, one intraspecific and one interspecific, and 3,533 SNP markers were co-occurring in both maps. The produced intraspecific genetic map is the first saturated map that associates into 26 linkage groups corresponding to the number of cotton chromosomes for a cross between two G. hirsutum lines. The linkage maps were shown to have high levels of collinearity to the JGI G. raimondii Ulbrich reference genome sequence. The CottonSNP63K array, cluster file and associated marker sequences constitute a major new resource for the global cotton research community." />
<meta name="DC.Contributor" content="Amanda M. Hulse-Kemp" />
<meta name="DC.Contributor" content="Jana Lemm" />
<meta name="DC.Contributor" content="Joerg Plieske" />
<meta name="DC.Contributor" content="Hamid Ashrafi" />
<meta name="DC.Contributor" content="Ramesh Buyyarapu" />
<meta name="DC.Contributor" content="David D. Fang" />
<meta name="DC.Contributor" content="James Frelichowski" />
<meta name="DC.Contributor" content="Marc Giband" />
<meta name="DC.Contributor" content="Steve Hague" />
<meta name="DC.Contributor" content="Lori L. Hinze" />
<meta name="DC.Contributor" content="Kelli J. Kochan" />
<meta name="DC.Contributor" content="Penny K. Riggs" />
<meta name="DC.Contributor" content="Jodi A. Scheffler" />
<meta name="DC.Contributor" content="Joshua A. Udall" />
<meta name="DC.Contributor" content="Mauricio Ulloa" />
<meta name="DC.Contributor" content="Shirley S. Wang" />
<meta name="DC.Contributor" content="Qian-Hao Zhu" />
<meta name="DC.Contributor" content="Sumit K. Bag" />
<meta name="DC.Contributor" content="Archana Bhardwaj" />
<meta name="DC.Contributor" content="John J. Burke" />
<meta name="DC.Contributor" content="Robert L. Byers" />
<meta name="DC.Contributor" content="Michel Claverie" />
<meta name="DC.Contributor" content="Michael A. Gore" />
<meta name="DC.Contributor" content="David B. Harker" />
<meta name="DC.Contributor" content="Md S. Islam" />
<meta name="DC.Contributor" content="Johnie N. Jenkins" />
<meta name="DC.Contributor" content="Don C. Jones" />
<meta name="DC.Contributor" content="Jean-Marc Lacape" />
<meta name="DC.Contributor" content="Danny J. Llewellyn" />
<meta name="DC.Contributor" content="Richard G. Percy" />
<meta name="DC.Contributor" content="Alan E. Pepper" />
<meta name="DC.Contributor" content="Jesse A. Poland" />
<meta name="DC.Contributor" content="Krishan Mohan Rai" />
<meta name="DC.Contributor" content="Samir V. Sawant" />
<meta name="DC.Contributor" content="Sunil Kumar Singh" />
<meta name="DC.Contributor" content="Andrew Spriggs" />
<meta name="DC.Contributor" content="Jen M. Taylor" />
<meta name="DC.Contributor" content="Fei Wang" />
<meta name="DC.Contributor" content="Scott M. Yourstone" />
<meta name="DC.Contributor" content="Xiuting Zheng" />
<meta name="DC.Contributor" content="Cindy T. Lawley" />
<meta name="DC.Contributor" content="Martin W. Ganal" />
<meta name="DC.Contributor" content="Allen Van Deynze" />
<meta name="DC.Contributor" content="Iain W. Wilson" />
<meta name="DC.Contributor" content="David M. Stelly" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of Gossypium spp." />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-throughput genotyping arrays provide a standardized resource for plant breeding communities that are useful for a breadth of applications including high-density genetic mapping, genome-wide association studies (GWAS), genomic selection (GS), complex trait dissection, and studying patterns of genomic diversity among cultivars and wild accessions. We have developed the CottonSNP63K, an Illumina Infinium array containing assays for 45,104 putative intraspecific single nucleotide polymorphism (SNP) markers for use within the cultivated cotton species &lt;i&gt;Gossypium hirsutum&lt;/i&gt; L. and 17,954 putative interspecific SNP markers for use with crosses of other cotton species with &lt;i&gt;G. hirsutum&lt;/i&gt;. The SNPs on the array were developed from 13 different discovery sets that represent a diverse range of &lt;i&gt;G. hirsutum&lt;/i&gt; germplasm and five other species: &lt;i&gt;G. barbadense&lt;/i&gt; L., &lt;i&gt;G. tomentosum&lt;/i&gt; Nuttal × Seemann, &lt;i&gt;G. mustelinum&lt;/i&gt; Miers × Watt, &lt;i&gt;G. armourianum&lt;/i&gt; Kearny, and &lt;i&gt;G. longicalyx&lt;/i&gt; J.B. Hutchinson and Lee. The array was validated with 1,156 samples to generate cluster positions to facilitate automated analysis of 38,822 polymorphic markers. Two high-density genetic maps containing a total of 22,829 SNPs were generated for two F&lt;sub&gt;2&lt;/sub&gt; mapping populations, one intraspecific and one interspecific, and 3,533 SNP markers were co-occurring in both maps. The produced intraspecific genetic map is the first saturated map that associates into 26 linkage groups corresponding to the number of cotton chromosomes for a cross between two &lt;i&gt;G. hirsutum&lt;/i&gt; lines. The linkage maps were shown to have high levels of collinearity to the JGI &lt;i&gt;G. raimondii&lt;/i&gt; Ulbrich reference genome sequence. The CottonSNP63K array, cluster file and associated marker sequences constitute a major new resource for the global cotton research community.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1187" />
<meta name="citation_id" content="5/6/1187" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1187" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1187.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1187.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1187.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.018416" />
<meta name="citation_pmid" content="25908569" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1187" />
<meta name="citation_lastpage" content="1209" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Amanda M. Hulse-Kemp" />
<meta name="citation_author_institution" content="Department of Soil &amp; Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Interdisciplinary Degree Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9670-9433" />
<meta name="citation_author" content="Jana Lemm" />
<meta name="citation_author_institution" content="TraitGenetics GmbH, 06466 Gatersleben, Germany" />
<meta name="citation_author" content="Joerg Plieske" />
<meta name="citation_author_institution" content="TraitGenetics GmbH, 06466 Gatersleben, Germany" />
<meta name="citation_author" content="Hamid Ashrafi" />
<meta name="citation_author_institution" content="Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2094-2367" />
<meta name="citation_author" content="Ramesh Buyyarapu" />
<meta name="citation_author_institution" content="Dow AgroSciences, Trait Genetics and Technologies, Indianapolis, Indiana 46268" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5343-9960" />
<meta name="citation_author" content="David D. Fang" />
<meta name="citation_author_institution" content="USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0036-5459" />
<meta name="citation_author" content="James Frelichowski" />
<meta name="citation_author_institution" content="USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845" />
<meta name="citation_author" content="Marc Giband" />
<meta name="citation_author_institution" content="CIRAD, UMR AGAP, Montpellier, F34398, France" />
<meta name="citation_author_institution" content="EMBRAPA, Algodão, Nucleo Cerrado, 75.375-000 Santo Antônio de Goias, GO, Brazil" />
<meta name="citation_author" content="Steve Hague" />
<meta name="citation_author_institution" content="Department of Soil &amp; Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Lori L. Hinze" />
<meta name="citation_author_institution" content="USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845" />
<meta name="citation_author" content="Kelli J. Kochan" />
<meta name="citation_author_institution" content="Department of Animal Science, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8124-8283" />
<meta name="citation_author" content="Penny K. Riggs" />
<meta name="citation_author_institution" content="Interdisciplinary Degree Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Department of Animal Science, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3296-320X" />
<meta name="citation_author" content="Jodi A. Scheffler" />
<meta name="citation_author_institution" content="USDA-ARS, Jamie Whitten Delta States Research Center, Stoneville, Mississippi 38776" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7897-2916" />
<meta name="citation_author" content="Joshua A. Udall" />
<meta name="citation_author_institution" content="Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602" />
<meta name="citation_author" content="Mauricio Ulloa" />
<meta name="citation_author_institution" content="USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415" />
<meta name="citation_author" content="Shirley S. Wang" />
<meta name="citation_author_institution" content="USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845" />
<meta name="citation_author" content="Qian-Hao Zhu" />
<meta name="citation_author_institution" content="CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6505-7417" />
<meta name="citation_author" content="Sumit K. Bag" />
<meta name="citation_author_institution" content="CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8907-2841" />
<meta name="citation_author" content="Archana Bhardwaj" />
<meta name="citation_author_institution" content="CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India" />
<meta name="citation_author" content="John J. Burke" />
<meta name="citation_author_institution" content="USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415" />
<meta name="citation_author" content="Robert L. Byers" />
<meta name="citation_author_institution" content="Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602" />
<meta name="citation_author" content="Michel Claverie" />
<meta name="citation_author_institution" content="CIRAD, UMR AGAP, Montpellier, F34398, France" />
<meta name="citation_author" content="Michael A. Gore" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6896-8024" />
<meta name="citation_author" content="David B. Harker" />
<meta name="citation_author_institution" content="Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602" />
<meta name="citation_author" content="Md S. Islam" />
<meta name="citation_author_institution" content="USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124" />
<meta name="citation_author" content="Johnie N. Jenkins" />
<meta name="citation_author_institution" content="USDA-ARS, Genetics and Precision Agriculture Research, Mississippi State, Mississippi 39762" />
<meta name="citation_author" content="Don C. Jones" />
<meta name="citation_author_institution" content="Cotton Incorporated, Agricultural Research, Cary, North Carolina 27513" />
<meta name="citation_author" content="Jean-Marc Lacape" />
<meta name="citation_author_institution" content="CIRAD, UMR AGAP, Montpellier, F34398, France" />
<meta name="citation_author" content="Danny J. Llewellyn" />
<meta name="citation_author_institution" content="CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia" />
<meta name="citation_author" content="Richard G. Percy" />
<meta name="citation_author_institution" content="USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845" />
<meta name="citation_author" content="Alan E. Pepper" />
<meta name="citation_author_institution" content="Interdisciplinary Degree Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Department of Biology, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Jesse A. Poland" />
<meta name="citation_author_institution" content="Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Krishan Mohan Rai" />
<meta name="citation_author_institution" content="CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India" />
<meta name="citation_author" content="Samir V. Sawant" />
<meta name="citation_author_institution" content="CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7635-5049" />
<meta name="citation_author" content="Sunil Kumar Singh" />
<meta name="citation_author_institution" content="CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4897-8812" />
<meta name="citation_author" content="Andrew Spriggs" />
<meta name="citation_author_institution" content="CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia" />
<meta name="citation_author" content="Jen M. Taylor" />
<meta name="citation_author_institution" content="CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia" />
<meta name="citation_author" content="Fei Wang" />
<meta name="citation_author_institution" content="Department of Soil &amp; Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Scott M. Yourstone" />
<meta name="citation_author_institution" content="Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602" />
<meta name="citation_author" content="Xiuting Zheng" />
<meta name="citation_author_institution" content="Department of Soil &amp; Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8574-4708" />
<meta name="citation_author" content="Cindy T. Lawley" />
<meta name="citation_author_institution" content="Illumina Inc., San Francisco, California 94158" />
<meta name="citation_author" content="Martin W. Ganal" />
<meta name="citation_author_institution" content="TraitGenetics GmbH, 06466 Gatersleben, Germany" />
<meta name="citation_author" content="Allen Van Deynze" />
<meta name="citation_author_institution" content="Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2093-0577" />
<meta name="citation_author" content="Iain W. Wilson" />
<meta name="citation_author_institution" content="CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia" />
<meta name="citation_author" content="David M. Stelly" />
<meta name="citation_author_institution" content="Department of Soil &amp; Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Interdisciplinary Degree Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_email" content="stelly@tamu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3468-4119" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=H. Ashrafi;citation_author=A. M. Hulse-Kemp;citation_author=F. Wang;citation_author=S. S. Yang;citation_author=X. Guan;citation_title=A long-read transcriptome assembly of cotton (Gossypium hirsutum L.) and intraspecific single nucleotide polymorphism discovery.;citation_pages=1-14;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=L. Bianco;citation_author=A. Cestaro;citation_author=D. J. Sargent;citation_author=E. Banchi;citation_author=S. Derdak;citation_title=Development and validation of a 20K single nucleotide polymorphism (SNP) whole genome genotyping array for apple (Malus × domestica Borkh).;citation_pages=e110377;citation_volume=9;citation_year=2014;citation_pmid=25303088;citation_doi=10.1371/journal.pone.0110377" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Blenda;citation_author=D. D. Fang;citation_author=J. F. Rami;citation_author=O. Garsmeur;citation_author=F. Luo;citation_title=A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.;citation_pages=e45739-e45739;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23029214;citation_doi=10.1371/journal.pone.0045739" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. S. Brown;citation_title=Identification of the chromosomes of Gossypium hirsutum L. by means of translocations;citation_pages=266-274;citation_volume=71;citation_year=1980;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. L. Byers;citation_author=D. B. Harker;citation_author=S. M. Yourstone;citation_author=P. J. Maughan;citation_author=J. A. Udall;citation_title=Development and mapping of SNP assays in allotetraploid cotton.;citation_pages=1201-1214;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22252442;citation_doi=10.1007/s00122-011-1780-8" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_title=A high-density SNP genotyping array for rice biology and molecular breeding.;citation_author=H. Chen;citation_author=W. Xie;citation_author=H. He;citation_author=H. Yu;citation_author=W. Chen;citation_volume=7;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Funct. Integr. Genomics;citation_author=J. Dalton-Morgan;citation_author=A. Hayward;citation_author=S. Alamery;citation_author=R. Tollenaere;citation_author=A. S. Mason;citation_title=A high-throughput SNP array in the amphidiploid species Brassica napus shows diversity in resistance genes.;citation_pages=643-655;citation_volume=14;citation_year=2014;citation_pmid=25147024;citation_doi=10.1007/s10142-014-0391-2" />
<meta name="citation_reference" content="citation_author=A. Desai;citation_author=P. W. Chee;citation_author=J. Rong;citation_author=O. L. May;citation_author=A. H. Paterson;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Berard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=Journal of biosciences;citation_journal_abbrev=J Biosci;citation_author=M. W. Ganal;citation_author=A. Polley;citation_author=E. M. Graner;citation_author=J. Plieske;citation_author=R. Wieseke;citation_title=Large SNP arrays for genotyping in crop plants.;citation_pages=821-828;citation_volume=37;citation_year=2012;citation_issue=5;citation_pmid=23107918;citation_doi=10.1007/s12038-012-9225-3" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Goecks;citation_author=A. Nekrutenko;citation_author=J. Taylor;citation_title=Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.;citation_pages=R86-R86;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20738864;citation_doi=10.1186/gb-2010-11-8-r86" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Registrations;citation_author=M. A. Gore;citation_title=Registration of the TM-1/NM24016 cotton recombinant inbred mapping population.;citation_pages=124-127;citation_volume=6;citation_year=2012;citation_doi=10.3198/jpr2011.06.0334crmp" />
<meta name="citation_reference" content="citation_author=Michael A. Gore;citation_pages=1;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=C. E. Grover;citation_author=X. Zhu;citation_author=K. K. Grupp;citation_author=J. J. Jareczek;citation_author=J. P. Gallagher;citation_title=Molecular confirmation of species status for the allopolyploid cotton species, Gossypium ekmanianum Wittmack.;citation_pages=1-12;citation_volume=62;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Hamilton;citation_author=C. N. Hansey;citation_author=B. R. Whitty;citation_author=K. Stoffel;citation_author=A. N. Massa;citation_title=Single nucleotide polymorphism discovery in elite North American potato germplasm.;citation_pages=302-302;citation_volume=12;citation_year=2011;citation_pmid=21658273;citation_doi=10.1186/1471-2164-12-302" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. L. Hinze;citation_author=D. D. Fang;citation_author=M. A. Gore;citation_author=B. E. Scheffler;citation_author=J. Z. Yu;citation_title=Molecular characterization of the Gossypium Diversity Reference Set of the US National Cotton Germplasm Collection.;citation_pages=313-327;citation_volume=128;citation_year=2015;citation_pmid=25431191;citation_doi=10.1007/s00122&amp;ndash;014&amp;ndash;2431&amp;ndash;7" />
<meta name="citation_reference" content="citation_author=A. M. Hulse-Kemp;citation_author=H. Ashrafi;citation_author=K. Stoffel;citation_author=X. Zheng;citation_author=A. C. Saski;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. M. Hulse-Kemp;citation_author=H. Ashrafi;citation_author=X. Zheng;citation_author=F. Wang;citation_author=K. A. Hoegenauer;citation_title=Development and bin mapping of gene-associated interspecific SNPs for cotton (Gossypium hirsutum L.) introgression breeding efforts.;citation_pages=945;citation_volume=15;citation_year=2014;citation_pmid=25359292;citation_doi=10.1186/1471&amp;ndash;2164&amp;ndash;15&amp;ndash;945" />
<meta name="citation_reference" content="citation_author=M. S. Islam;citation_pages=1-10;citation_volume=8;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. J. Kohel;citation_author=T. R. Richmond;citation_author=C. F. Lewis;citation_title=Texas Marker-1. Description of a Genetic Standard for Gossypium hirsutum L.;citation_pages=670-671;citation_volume=10;citation_year=1970;citation_issue=6;citation_doi=10.2135/cropsci1970.0011183X001000060019x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. M. Lacape;citation_author=M. Claverie;citation_author=R. O. Vidal;citation_author=M. F. Carazzolle;citation_author=G. A. Guimaraes Pereira;citation_title=Deep sequencing reveals differences in the transcriptional landscapes of fibers from two cultivated species of cotton.;citation_pages=e48855-e48855;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23166598;citation_doi=10.1371/journal.pone.0048855" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. M. Lacape;citation_author=T. B. Nguyen;citation_author=S. Thibivilliers;citation_author=B. Bojinov;citation_author=B. Courtois;citation_title=A combined RFLP-SSR-AFLP map of tetraploid cotton based on a Gossypium hirsutum x Gossypium barbadense backcross population.;citation_pages=612-626;citation_volume=46;citation_year=2003;citation_issue=4;citation_pmid=12897870;citation_doi=10.1139/g03&amp;ndash;050" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=F. G. Li;citation_author=G. Y. Fan;citation_author=K. B. Wang;citation_author=F. M. Sun;citation_author=Y. L. Yuan;citation_title=Genome sequence of the cultivated cotton Gossypium arboreum.;citation_pages=567-572;citation_volume=46;citation_year=2014;citation_pmid=24836287;citation_doi=10.1038/ng.2987" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=K. F. Manly;citation_author=R. H. Cudmore;citation_author=J. M. Meer;citation_title=Map Manager QTX, cross-platform software for genetic mapping.;citation_pages=930-932;citation_volume=12;citation_year=2001;citation_issue=12;citation_pmid=11707780;citation_doi=10.1007/s00335-001-1016-3" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=K. M. Rai;citation_author=S. K. Singh;citation_author=A. Bhardwaj;citation_author=V. Kumar;citation_author=D. Lakhwani;citation_title=Large-scale resource development in Gossypium hirsutum L. by 454 sequencing of genic-enriched libraries from six diverse genotypes.;citation_pages=953-963;citation_volume=11;citation_year=2013;citation_pmid=23782852;citation_doi=10.1111/pbi.12088" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Rong;citation_author=C. A. Abbey;citation_author=J. E. Bowers;citation_author=C. L. Brubaker;citation_author=C. H. Chang;citation_title=A 3347-Locus Genetic Recombination Map of Sequence-Tagged Sites Reveals Features of Genome Organization, Transmission and Evolution of Cotton (Gossypium);citation_pages=389-417;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020432;citation_doi=10.1534/genetics.166.1.389" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=X. L. Shen;citation_author=W. Guo;citation_author=Q. Lu;citation_author=X. Zhu;citation_author=Y. Yuan;citation_title=Genetic mapping of quantitative trait loci for fiber quality and yield trait by RIL approach in Upland cotton.;citation_pages=371-380;citation_volume=155;citation_year=2007;citation_doi=10.1007/s10681-006-9338-6" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=Y. Shi;citation_author=W. Li;citation_author=A. Li;citation_author=R. Ge;citation_author=B. Zhang;citation_title=Constructing a high-density linkage map for Gossypium hirsutum × Gossypium barbadense and identifying QTLs for lint percentage.;citation_pages=450-467;citation_volume=57;citation_year=2014;citation_pmid=25263268" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.;citation_pages=e54985;citation_volume=8;citation_year=2013;citation_pmid=23372807;citation_doi=10.1371/journal.pone.0054985" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=S. Tang;citation_author=Z. Teng;citation_author=T. Zhai;citation_author=X. Fang;citation_author=F. Liu;citation_title=Construction of genetic map and QTL analysis of fiber quality traits for Upland cotton (Gossypium hirsutum L.).;citation_pages=195-213;citation_volume=201;citation_year=2014" />
<meta name="citation_reference" content="citation_author=N. A. Tinker;citation_author=S. Chao;citation_author=G. R. Lazo;citation_author=R. E. Oliver;citation_author=Y. Huang;citation_year=2014" />
<meta name="citation_reference" content="citation_author=M. J. Truco;citation_author=H. Ashrafi;citation_author=A. Kozik;citation_author=H. van Leeuwen;citation_author=J. Bowers;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Tyagi;citation_author=M. A. Gore;citation_author=D. T. Bowman;citation_author=B. T. Campbell;citation_author=J. A. Udall;citation_title=Genetic diversity and population structure in the US Upland cotton (Gossypium hirsutum L.).;citation_pages=283-295;citation_volume=127;citation_year=2014;citation_pmid=24170350;citation_doi=10.1007/s00122-013-2217-3" />
<meta name="citation_reference" content="citation_journal_title=Botany;citation_author=M. Ulloa;citation_author=I. Y. Abdurakhmonov;citation_author=C. Perez-M;citation_author=R. Percy;citation_author=J. M. Stewart;citation_title=Genetic diversity and population structure of cotton (Gossypium spp.) of the New World assessed by SSR markers.;citation_pages=251-259;citation_volume=91;citation_year=2013;citation_doi=10.1139/cjb-2012-0192" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=M. Lee;citation_author=T. A. Wilkins;citation_author=A. Kozik;citation_title=Sampling nucleotide diversity in cotton.;citation_pages=125-125;citation_volume=9;citation_year=2009;citation_pmid=19840401;citation_doi=10.1186/1471-2229-9-125" />
<meta name="citation_reference" content="citation_title=“JoinMap 4.” Software for the calculation of genetic linkage maps in experimental populations;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Wang;citation_author=Z. Wang;citation_author=F. Li;citation_author=W. Ye;citation_author=J. Wang;citation_title=The draft genome of a diploid cotton Gossypium raimondii.;citation_pages=1098-1103;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22922876;citation_doi=10.1038/ng.2371" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. Wang;citation_author=D. Wong;citation_author=K. Forrest;citation_author=A. Allen;citation_author=S. Chao;citation_title=Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.;citation_pages=787-796;citation_volume=12;citation_year=2014;citation_pmid=24646323;citation_doi=10.1111/pbi.12183" />
<meta name="citation_reference" content="citation_title=Genetics and Genomics of Cotton;citation_title=Evolution and natural history of the cotton genus;citation_author=J. F. Wendel;citation_author=C. Brubaker;citation_author=I. Alvarez;citation_author=R. Cronn;citation_author=J. M. Stewart;citation_pages=3-22;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Yan;citation_author=T. Shah;citation_author=M. L. Warburton;citation_author=E. S. Buckler;citation_author=M. D. McMullen;citation_title=Genetic characterization and linkage disequilibrium estimation of a global maize collection using SNP markers.;citation_pages=e8451-e8451;citation_volume=4;citation_year=2009;citation_issue=12;citation_pmid=20041112;citation_doi=10.1371/journal.pone.0008451" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Yu;citation_author=S. Jung;citation_author=C. H. Cheng;citation_author=S. P. Ficklin;citation_author=T. Lee;citation_title=CottonGen: a genomics, genetics and breeding database for cotton research.;citation_volume=42;citation_year=2014;citation_issue=Database issue;citation_pmid=24203703;citation_doi=10.1093/nar/gkt1064" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. Z. Yu;citation_author=R. J. Kohel;citation_author=D. D. Fang;citation_author=J. Cho;citation_author=A. Van Deynze;citation_title=A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome;citation_pages=43-58;citation_volume=2;citation_year=2012;citation_issue=1;citation_pmid=22384381;citation_doi=10.1534/g3.111.001552" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=K. Zhang;citation_author=J. Zhang;citation_author=J. Ma;citation_author=S. Tang;citation_author=D. Liu;citation_title=Genetic mapping and quantitative trait locus analysis of fiber quality traits using a three-parent composite population in upland cotton (Gossypium hirsutum L.).;citation_pages=335-348;citation_volume=29;citation_year=2012;citation_doi=10.1007/s11032-011-9549-y" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=Z. Zhang;citation_author=M. Hu;citation_author=J. Zhang;citation_author=D. Liu;citation_author=J. Zheng;citation_title=Construction of a comprehensive PCR-based marker linkage map and QTL mapping for fiber quality traits in upland cotton (Gossypium hirsutum L.).;citation_pages=49-61;citation_volume=24;citation_year=2009;citation_doi=10.1007/s11032-009-9271-1" />
<meta name="citation_reference" content="citation_author=Q. H. Zhu;citation_author=A. Spriggs;citation_author=J. M. Taylor;citation_author=D. Llewellyn;citation_author=I. Wilson;citation_year=2014;citation_pmid=25106949;citation_doi=10.1534/g3.114.012542" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of Gossypium spp." />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="High-throughput genotyping arrays provide a standardized resource for plant breeding communities that are useful for a breadth of applications including high-density genetic mapping, genome-wide association studies (GWAS), genomic selection (GS), complex trait dissection, and studying patterns of genomic diversity among cultivars and wild accessions. We have developed the CottonSNP63K, an Illumina Infinium array containing assays for 45,104 putative intraspecific single nucleotide polymorphism (SNP) markers for use within the cultivated cotton species Gossypium hirsutum L. and 17,954 putative interspecific SNP markers for use with crosses of other cotton species with G. hirsutum . The SNPs on the array were developed from 13 different discovery sets that represent a diverse range of G. hirsutum germplasm and five other species: G. barbadense L., G. tomentosum Nuttal × Seemann, G. mustelinum Miers × Watt, G. armourianum Kearny, and G. longicalyx J.B. Hutchinson and Lee. The array was validated with 1,156 samples to generate cluster positions to facilitate automated analysis of 38,822 polymorphic markers. Two high-density genetic maps containing a total of 22,829 SNPs were generated for two F2 mapping populations, one intraspecific and one interspecific, and 3,533 SNP markers were co-occurring in both maps. The produced intraspecific genetic map is the first saturated map that associates into 26 linkage groups corresponding to the number of cotton chromosomes for a cross between two G. hirsutum lines. The linkage maps were shown to have high levels of collinearity to the JGI G. raimondii Ulbrich reference genome sequence. The CottonSNP63K array, cluster file and associated marker sequences constitute a major new resource for the global cotton research community." />
<meta name="og-title" property="og:title" content="Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of Gossypium spp." />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1187" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-throughput genotyping arrays provide a standardized resource for plant breeding communities that are useful for a breadth of applications including high-density genetic mapping, genome-wide association studies (GWAS), genomic selection (GS), complex trait dissection, and studying patterns of genomic diversity among cultivars and wild accessions. We have developed the CottonSNP63K, an Illumina Infinium array containing assays for 45,104 putative intraspecific single nucleotide polymorphism (SNP) markers for use within the cultivated cotton species Gossypium hirsutum L. and 17,954 putative interspecific SNP markers for use with crosses of other cotton species with G. hirsutum . The SNPs on the array were developed from 13 different discovery sets that represent a diverse range of G. hirsutum germplasm and five other species: G. barbadense L., G. tomentosum Nuttal × Seemann, G. mustelinum Miers × Watt, G. armourianum Kearny, and G. longicalyx J.B. Hutchinson and Lee. The array was validated with 1,156 samples to generate cluster positions to facilitate automated analysis of 38,822 polymorphic markers. Two high-density genetic maps containing a total of 22,829 SNPs were generated for two F2 mapping populations, one intraspecific and one interspecific, and 3,533 SNP markers were co-occurring in both maps. The produced intraspecific genetic map is the first saturated map that associates into 26 linkage groups corresponding to the number of cotton chromosomes for a cross between two G. hirsutum lines. The linkage maps were shown to have high levels of collinearity to the JGI G. raimondii Ulbrich reference genome sequence. The CottonSNP63K array, cluster file and associated marker sequences constitute a major new resource for the global cotton research community." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of Gossypium spp. | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ivb_Q_q6EkmmERox03jFh8GsFI0QFaytpzJTx2RwbGs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459913","apath":"\/ggg\/5\/6\/1187.atom","pisa":"ggg;5\/6\/1187","ac":{"\/ggg\/5\/6\/1187.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1187.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1187"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1187"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1187.short":true,"\/content\/5\/6\/1187":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AFwktbPe10rlHcgwyySh9bQPCz+rT\\\\\\\/LwkJxA3CK2FpSBPkqrEdL8Iin3gIxDBCFgPblYLMn1pE4GIVNS7OkvG8UXaKhMwQfjvlrJzU5ZobSciqQght\\\\\\\/tVdC9YdsZFhmubBi7g56IDutadAQPAswqpYrfNXzKmV2Ufd8bT3YYHOqBIaK7KIQueG1son3jtKs3hn5TVMeB9bS\\\\\\\/zKwLMW8ZR+O3X7sqcwyFGigIGn2Pn\\\\\\\/BrJQSf4wLh0wEXelQPodIt6aL82FuXWEOLfJJ8LGi2zE418iF5dHJa9yI1U3tK4DF\\\\\\\/nwlY30ZBXddSpwFAeB73dlV\\\\\\\/n\\\\\\\/H\\\\\\\/V4NBJnNrfxZnxh7ArOtHNmo2UlUm3WORoM06SCixrXr05YraBuP0eeQo8hKiih42dol4E3yhHbZDCMnku6QNE4VjiWXftAKuViAajySwbupKWzZwvzovInOeWc3TAySilVX+94XejEknuaDQoM\\\\\\\/4BVQiBlfG62dl1dvDFo1kW126C7aF1CHp3R3SBNCO1djUG8uquhE+jNZGTKbvRyAcRwGvxIv6N\\\\\\\/zlRdoc6xHWf1E8ymhq2m58L2KOoo1Z6XocNUhJ1WhSf7Th+RcIpZi9HGMD20rfcfzp9QAnqMCqzsTxDArm3NwLRjmKBkiRyyt8Tj\\\\\\\/avkYsc+5w1zJIjr6sL0sPXhS\\\\\\\/20RPdHE7LfRkGuSAsnHmMLqmNKzgenbZ7iLbVM4ldwK2NSoxWCFeRMiTW\\\\\\\/JwiK+ixxkbSg9NGaY4E0fIScqn90fa\\\\\\\/BHnmqMTI0qo2hdkRDqipZ\\\\\\\/Y5MeJ6LzoKe6ztFEstCq\\\\\\\/Duk7E4Igq44ROWQQt38b4jwIpFwXfG9GPJOLICOzdrTV7971zFVmRTW1sCYL1Ab1UjV5Aal8PD7WdipUOveZl0GD1JESMCjPxiB2JOKxDFJK3sYInP3ZNsevtVmRk1J9QFmmbsfKGKO6u5VUrUnpC+s3f5l61I+TzE4Qm+gSAJo7W6ML5PtofUWxpCeKx5Zk2ak++X42bpVe2TJ7CVdOv\\\\\\\/JcfjORqEO56OXZV6Agf3kE4aChixXVSp0XYpaVViCArzUbAxVlr9sbTHbL7mcnKRkszjY7o8suHx6FeAPVYytR\\\\\\\/d\\\\\\\/5IdGBqqWRLMFJV09vShraO2lhul+pSJlBPtEEpODxyncnfWgdVgtYCbUU+WdPhBwlST11w1Z6HtCANs3I4FxkX4dHDUI4fC04FujOSny8M8gWrm4Uh+so0lKCosQld1O2baC6fIyFZ\\\\\\\/pSZQINUTsO0yKoEI9mNX2tO0AVfSuh\\\u0022,\\\u0022iv\\\u0022:\\\u00223ImeR+DrPtw7wITclBriPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224ec7b3bdfdf1f6773efaba780d237b7d1cd463c94877517217fa2a561416e97d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227AgTW8AFdV1umAXabZ8IJChd39+dDHzraw5RsxtKL7H3tWM4zr5xgNpDcpxa6crGx5+MsPF8vLzW3R6qPg7uHRgl5QM8WTJ3+4Cz9jWveZ2gEAZRDGhvGtmdYjxUJaZ1rbG5HMH+L20AKJzOaYIwcEfclNZJUBqAboY4tZL19xpg8f9\\\\\\\/9I8z7Z1JFLKOQx9igAXsQj9v\\\\\\\/NneV1LJ5I8o9p0AN2HeE7euyZjzjeq\\\\\\\/YMep5yD7kZBuVOVVdNRb8PYM1Fwe7KVONh57YqPXDMfdjOLIy6kwskdu39glS6mIlUr0EwAo4lXGyNc45hMEtfautt4ARO+5V5kx6z860BZA7baOXowA5AlnM67be+yFEraq+McHtO39VQj9nPcMG+eaX3hhY2F8L\\\\\\\/MkcrBvJSzt6kYUihWrbus1hLgy3zjY8n0rcW3brcVGvD2zHKevHUZ3KmxsV0Y\\\\\\\/gOe8TvueZzn7Mo\\\\\\\/c540A3vE5jn5mAiC+kElcb7Fl+eCQbVENxa9ngB+voZG\\\\\\\/szGhqF8HG2FSSWSZauBLxtOAgqXB2i3iKPMKRCScsJHsa1TWeSS1IGUyuzQChXJ1WCjlsdglUizEsH6Ng8PWGrkJ\\\\\\\/TlWLKuHAxznA+Slc\\\\\\\/dM\\\\\\\/pmfEySUqEsKe+RDtk\\\\\\\/1\\\\\\\/9dX3zOeRYHUn57xn\\\\\\\/U2kEhiyhIlJoHGDPWbw1mj0jjfA4RUkMxh7U0X1HVPOmighjTVVg4yLy8gX77G+D6iHPtlZa64nlXAjxftcHXTWGtiuEsmrNNkFqAb8tu8kKkttVyOpm3rErBSXTAe6i2E56uh+b9TBi9O8PQ8VORqYC8l8d2mnuMyFtxbM5B+Y0HxJgN1al1G3sNm0PKJ0mx7uGrCM13W2qqfn8oH\\\\\\\/N5tC8x55RAYM8InAZn3AIMoKeMrkmYgrgi4f6S68dAGNYjbTdsgGcUJfmjFnEb0u2LNyZy53VZOFhiiGFEqvX8LjrAWPaNQFnt1DaB7nnTyS8eTdExkkZRhYf4XKppvKMBxaGCNr6d4wB\\\\\\\/8zfXY8TV98MUbOnvPEalhlmewoi3grWllb\\\\\\\/1835ANINzaiplZrGdQGoXxpU3XoYI2GvedlzyTBrphVnm0lzlcUafe7Nv4pLsVgfzruxJgUaUupNAc4PjgsHit\\\\\\\/O+BBwm4css3B+y9MnZsoZ0ATDIIl5vKJlVrZZsF7YKTNq3OOHSvvy6X8fv0ml68tKaN1niYWmen7kq9UivxZpEpQelp33SceQJoW2Gh25PaFaIkhibQKCy4QdRDhFXzTZ+nsSHAO0CeW6Aul438r+zBwdoAvRDg0BERJI8gufmceCD5X8owgOoobK6R5IGQvNpLD8QExand620cteML\\\\\\\/QfMcxiWJnrIuH2Nd9j0zIMhtkPJHLvjcraNChWLv46PeD9bQEK4QZuLg95U9ZJtinK1t4hj2QPazDu5eANKVduJq3N6ICnm2HCbByeOuqnxjnGkQpSsazzzitL1hJ2ziQenuwozLaVP7D8Zvs77AdE4mMsFIsT3tekvVicRDkEqkBClym3sFHUTrQsHqtL3s5aPiUDrknJVjuetCJh33TlrNLFJvfzELUJCEpln7GM\\\\\\\/bRmcZ7POeUSxmlAgiVP+H2ti9gM6I5+FmhSLO94o6Y1RncgE9wB9vC\\\\\\\/NCdcvtvnNHe+Bk9ck+3jCwlt4ThPtVIPr8S1TNXdqp6Qc6RI8nxVt5d\\\\\\\/d3mwxlNrH9ra5bmjBBj6oxi4xnJYdTw2Tvc1Thw+4x45jtzdAuv5zo4IjMexrdpIUv9Dvr8mnU2h+LUbFJ35dDgh5zlpevFCog3wOqLr4SM4\\\\\\\/VnofxxYI\\\\\\\/m82CRtlEXUbXinzIpq1Ps+4zxQp+vQCnMc2sUOBhVMm6eAkyuQWHdT9lFLZoBp\\\\\\\/urkuPNr9grxvBxo\\\\\\\/IP7SwfRaR\\\\\\\/M0QSzMzKTWn+ipK58HNffjxJVLnOIogG1Rb1Q8vQoT+5vhZRPa0cljFojvPa6YwfCDBazkzqK8kzFw+ljsIHajiM+hopxTQ9eCPSU7e9KACbofJSnjyrfgs23aQDV00RB5HaORMnjOQGsfyH30tfV65YACdCpCjsSpwdSDoP7OSvDdxjr3ZPNvEfh8yoqNZt3jZrzUKz0zyHj\\\\\\\/vS1QJas5aV2YjPSoyzSSZYOW5GMFRAC4hk3T5bouAbdwXorGajR\\\\\\\/lWC2cR1+BXmvCAsBczH1FaGs4+ImLKx3\\\\\\\/bGHJc5rGboEsUY5JkwGH6bUfioBFpG\\\\\\\/GM\\\\\\\/FWgujaCyhVlJRsQa2FC9xbkZzHRSpX9gHgXZsP4RllBVO2v5HJQTJhnsaBGll6eCkgwYKAfGu1mQUGoKWdztXTBtzFnDPAC6FYkNn40TMuUxqxoRIPNDbvCu2nCDpduluF6Vunf0zdi\\\\\\\/+fqUsm9BOO+aPaote5PLH9yzklB8CjloOKfH7f4zE7bIe+la7tSRIqqtPTc0HfolVl4HB7Dacwvre\\\\\\\/h0Cs6uyA52jWNou\\\\\\\/gorCyAdTQyJRAJRh\\\\\\\/MrboK7YFQa1gEE\\\\\\\/zzeJeFYbiKLhGvgqLJfXDUKbQvi7pa6Fsjz0\\\\\\\/4fU5OLe7NcdV5\\\\\\\/TaA5X5NLEETc2wNSvoemANdM0RQoV3bh\\\\\\\/VkDB2rG21Mst7aOk0hoPdS9dVzgAULo2QJ9sgfQiHbEERxsjc5V4EpufDPbLKK5d8qQRN5bT2GiNsV8WJLIuxA5ZYVMEaAOibuW4mlubegrrHI3RjmBO8aTbLQBoPDKPnhc21vxY3fC46eroV7E4zsn3v\\\\\\\/R0IVP2SMkCAj0VupZguHNoAdXHled71koeYQchwAl3uM+QApRXJtjEtrppfSwiELwSjChSAIIPdzbv\\\\\\\/B\\\\\\\/9YqYmWyeTy3sEA1ZVuqECCykdbwrq8qtc5+MfyExopAIHgZM01OmgxO\\\\\\\/nDh5ywSeZ4ZOMnDJvVEiQpE+bYvHxegLA2lW8pk0WTX2t6hMFoabgvzarclrlDrskrghyRlaEI6ULZ7gQtyBm5dJd6kqeP5cEcl90HlCsPZmDGaiPIeexby1oRLwDmwtQMAGYu3snYnEJEZ+e5n3AljdXEMop368vMW8jZab0iEmqmQp6vwH+EEGbpcsdtcqCmxUsMdEcJvHmUTh30ddzKL0wFLcYaJiD4uOSBdVZ4uho61Z7hhRf0wkbj0EAUZchRcPCD\\\\\\\/uc7bznutdvFZtA8On\\\\\\\/4Xf9ila6YTWmIyLbzlKwvgMWofWCtMQbaqSzUMS6DiBO7dF2vlUSku\\\\\\\/j9xLdJeYGtSMOzi3MHuTodmYSzX8pS4SHIAtkTDNseKKpBJ9hiRDRWcCx4RRXve0m\\\\\\\/OD9l3rUjZpot7jCic\\\\\\\/hyuYpk920cn7AHBxKWDDUQvjzy3r9FYqMl1+eD+KRq5w2VLnlvKcU45hTOYMw7R6Plj8eTnsCbCQ3xe8EiVqGAstwX69vnVRPSG4EZj04dH+ZNYuhNovCCl+4M1dX\\\\\\\/Fuzzz\\\\\\\/hFOz6\\\\\\\/qSLFtMl7KSMuStu\\\\\\\/PJfYekhRhgzrpMi86TCtM6CnhY6NYhUyVirzMjDbbOOx1EPh0IAHwEMUm3qPaMlabsPPJKXm4ZHQkTZq\\\\\\\/UdfvZLSLzuCPGZLB2M66YTzzT9kQtjDnjiHZAXLmI3rlBkvwIL+U58Q5WhCoTzjjLsuZoQoMNa+cIXHMn+lkfuGS5obchkDfZVogRFHeROkLedvbPZvQBBGkbMm+cWjCFECPKMQMD+3P6qGuO2w3YF5eSFQPwBjxUSKS5n6Ik1WeJtEREijkHaFgesZQHBd4a1UvW6eDOdDQwUDm9NkN9Y\\\\\\\/ESoIWmsKaReKT4Svs\\\\\\\/ZXb\\\\\\\/XT4KJoDlGShYdQdiIV\\\\\\\/N+zewji+sLpSPh\\\\\\\/szcC60LqabE0mNczuwebp3EYgo200vbwPwC6ZHNcHfyJlpC4CMJ6xnhDedogeKSaYxuXq6slEgeGjyb8rAWEKy8asq7wFt8PzvJYfk3Ysfy7IAvQ5psZdn4\\\\\\\/I3CjTyRjKrwQ4B\\\\\\\/zxNxrt5aYjHd5k+qbTWa4YWE+tue5+0Fyf3jra9EWIm67Qzgw0ggECTrQF9hqe9smvRBK7RHk0d5K6iKnE9RVncY9rcnMrYEIRLInQ4pzp8XIF6Wc0+OKSzeKAtDRZU7\\\\\\\/AnW4D1YcwyjdYaSxhXJtiO58UYODp+uSBAayJJ2p2YtuXBEvJIopVkb+BChfEbdVuSxXaLuW4vVIbd5msRQPKYHl09CEEouOvDRT1z+5csG9Q7frR5emVp4QwT4G5Y2OJ8cW3jv3pVq5KyXkGj+mj9GSLyNxFFowzE4aJAfvTfW5gI+tyTgCzG5LUSMqeGQsMMyA9bc+S2TWAUENoCs0iPSnhWdZxPZpQI34fBCgBKbgyNxfEJ0rK1F62X+Znz7jaWuYsvIUKed5QGmeMXiqnurG\\\\\\\/FAd34LB\\\\\\\/+7RYKCvNJ5hgSPCN0IQnO1CE4Uarvnx3fuqwLFxaket3IOc8cCKXv4wtm8CcJqADXaHpU+iUYCzO9LChwgKYn60q+1YYezqgDYEL\\\\\\\/oJLXGCrux3f9B++WLFzZDW4v3V60n+xN0Pm4eiJVYg9WZr2OuN6eHZZpiXbHX0clySg8BPUVRUpYxPfuI8Ik9RwpWmeMwA0xxxhaRxpnvUhqD8bIyY74HcnwM\\\\\\\/80TK3dP5BJR+ZP4koF1oKkKRLhEf4e1xofGip+8RhnMX1is45ubfpE+ZoEocurx2Xqqp1l10zzmHthQPx9ZyvdAMWbMhWpFCRkHBa1uZ\\\\\\\/pqq2yeudwM90OG4+pVQY3pk0EZpWyU6K+dWJRvz05zIqJbviWCyvbInQ3rBHlCW6a3iwEmliPfC9eim0aACdNJ7OF+EdU6k8TE14Z3bBIwTUMQwuxhyoHfOrjeslI8fDwX2VAY0GbqIkGA94RY6dscdYK\\\\\\\/7eHPNdK89nSgD+NrB5In2cf7s54iog2uvJN\\\\\\\/XCCf+MWjq1FdQS+HVqZJS\\\\\\\/XgrzJVOZYndUtiC1FIt8W\\\\\\\/722DAtKbwrfUIY\\\\\\\/+RsgWNGObXDHdPIfGq8sxGYaPJBhOB7UI4FujIaTIIQgbbJoT1PRx0kfJD0sW1jQJfesL4Tp0dzsOSoc+Vm6PfSslQ8RjH8+yrFGAedqxBFgbhKg9TwI\\\\\\\/3ciO8tc5lTNYn+lomJRj9dnaQZI4ySuyPnvir95LCatCkanzlWU8hRw7yEA1iNK\\\\\\\/oSXlOzSKddxWacWm4P9t3KuX0AnR4IMfSehZEma9a824wQnIaJg8dpKp3h570vUMTj+bKF52jzuRDHY\\\\\\\/YLEBosiioyLpI1N0YwOf1boyCVFfSvdIG5pqsRxD1qN5XHttKgIdSf5clcZe\\\\\\\/Zu\\\\\\\/ws99RyGTR0Y2u2aG7GStwVQf5I9Tkoj2Dfxk+PP5BVO4Nax1ujoqDno5sSp5fwjERsxq6zN\\\\\\\/kL9NQrKXIuSjj9urIWlYthSiLlU5JgcG8rQ2tnKxKR+nf0Z7SOs4WaF3rFKTX+G6vCmdCQVhVrN8rfi6h82JZwz6+EA88M9Zl8iB8Jbe7dpDUAmK7W9wTej6PCXY+R1BmZlu8WLX5CqeAf9X2YniS\\\\\\\/KFvcQJLNQOHu7vrOZNI5LzjLrKPHYGvjHhTmPLIDnhDHKK4fozdcP\\\\\\\/OgAsIf+J8Kt\\\\\\\/UiBV1kdXvu0ecTRXhOWCkCMlk\\\\\\\/tdhMmyGpQ4ldBwFM1XWrLDzszn0wSOrCYyYrimzcKIXGRhhfzmJiykPXHz21ysqEA1B8fOwlaiAtdjnDObgai1KRLslLeM2jwaBhcCGAGplj+lFgUKl7TD88xrZfkBCKNBWCAqo4LF4kJO9u7KI\\\\\\\/d7voUc5ohM3naxnLtvluZ5Erork2ybfgsZOyd39qdqIDrLMmKF0ttivTs3j2GyQEGkNIeiETZRSela\\\\\\\/cK\\\\\\\/k3Ecgf6vnN8DdZcbe3O\\\\\\\/q9eQ4SO9SeWchsJ1EOeptZ5bPAa0Kt10PJXP264PZi6IGLMr6PafFz6hDh0tei1X+xBSXUYqvu2RrWvEF2E8pu4E37Bm7Gn7YDPrkPX1MwOEclC6QmUadB\\\\\\\/FUR3r21ftLsvxVbmJaQPUGfTRcN2U2FSt18QItsphaZaVKZDN4+6hdxLk1v70Q9UMTE512GzwxjZiGkEYZC1Dfk8ppgY4JDO4QTJ7sCi2E3IMYH5dJIYhys8xmSip+8NvXprHSKqF59qP7MxIKVJKpKFk7QM2mdUj4ZFqBUypEDCsJkFtLALpFBeKAzJ0tNDzMSDdGNFZquoc7xZ9Z7Ql1lIs5OqIGX5D8TAbbyoekKzWrRazDfn52ZvtaIj2gEfiLF3guUwb1eg91fP7ziLsyCkmpBm\\\\\\\/kXIFpGO7nPy8reUXQFexn+h6D4efOBa3\\\\\\\/DuMjAtEzd7gmZBDWmf\\\\\\\/HoFXvvSN+HYfEsBCUKIOuMgvG\\\\\\\/Ep\\\\\\\/T034THJ2lZ42rIL0byulaQE4GD43FFvpt8qRrj5AZnnDSO53oTrlGuMB5Y6iBH3gDTAsT50aVX7gfuTvF+ee6ju0FUds1LS9qWE83QWwbO7GMExyUwADiZOMuuLhLC7tuzqd0JIcChUfYmvN56AMnwizH+xTjfOcoKL7ScblA+\\\\\\\/yPftP3f3DKDTfgK1twrQhrcEznwmOm7o3IU4LBr55OI0OW6Ew8qF0f6ttNNUThyGdNDAgUcdYjz8ujpgDLLnnc9qIGfLlnJNuVOAtNvAVCdffA+27s4LMU2DQ7VilhaY86CaNqCOxTnE+\\\\\\\/mgAuGy09TmcxyC8OdQDlY7xuBq+FKqyuRuX6q8F6UVVs\\\\\\\/veI2LSM0y3VeWkzdhMgZxhbxSKX+JMSZEZiajQ9lsyvXgK3oMv6WzXq2BkNsW1k2O68Fch8aYiYtpafs9mUjBUGN0xLMnJa4V5dtK0FbtzAZTy71RuTeiyE+GEx5l38JiZ97Dl3+xVNOaTwqh7zMIvL8WLrReGqIDQsa9TDRYTbGqV0RUx2hBiqzr7BAtYJPgmE1LmV8qoGlGCASB8jo\\\\\\\/UtNEv6FlMmdSQ0kHv2wxYSvD7u5YjvFxjPf7ipQeXbXaIo4e4wdJAnxlEgb6O\\\\\\\/kPUINTXdycNSi7Z0WpecRYPV9d\\\\\\\/WX1zLNvX9wvbYjpYCyfdwhlOmGzWrqJGvnYtXhZfksEOD2W7v07mre4xGsEJLLWRx8gaKYkL8jnslKpbeZMVSd2xmAJGiIOsIT\\\\\\\/kQngCvErv0snmdTcsvCZoXVSbK42HZitCZCr4iQ8dBUJ3dXnHbaU8bFv2KUnsjye\\\\\\\/FpXrsGg\\\\\\\/hBScUsh65gqFYqXWRHaHySZjZDd92AuCYZhQOU\\\\\\\/2s07ayON7eZwEwILCN1+V55LAhOc6tI5+PQ5g37mTEK5MfHQAisLm3Ukcw5cU56EJSb3DcK39F4SjPXv3Ij\\\\\\\/\\\\\\\/rsxYq8uWTt2mz8QIvE7fbh4QtuZh4\\\\\\\/OvNxGpW7fOE4FvFf5q7S5xDMtJyYVM75DkWzct2+l\\\\\\\/4Lsaq9fC+yrIKxwn8pVaTNKf6F+kdC19aSmmU45ABQhCIYH5fMhL+ilKj1zS09LyIE7O7KkbHfrUXBKI\\\\\\\/qDiFmh0mys9\\\\\\\/rcPRV6RIKRI9MRXzsFywVjL8pYDxKAE0D1DbNk7dfOLMvA2otPm8+XeCQX917ZQCg6w3RsOc8Ctxxfck9LP7xZcQLCzo8jg6y4ii3uH+rLJU3dz9zKdQBs+Ys7A6RI8ZEBDzMHYHAXu+qxuXvkYtvQq4PpgpSr367R6SXaLRgjaxSF2Gc2syqKhBvxk1HDllFhgzObuVFg1Yvc5dUohweH+qYZDPAkKRIOTCPPZnhdhMWkDrHSLe96PIBOoHxsFJmpFR4p9SBJ9aWmqfImw3ZQZjU6VIsrsST34E7QxdGMyzM9sRtiiU1LoYhG\\\\\\\/AjYFrvHOcAPf2753sf43YnbJMjxRFQePxp7ZUSSJhAv0pcrYpYbE1sjPl87RAu\\\\\\\/zhKkUC2FCSo5yAGuhbwzAJ3fba+Vmr+Sy1OD1CtXDX2jbxWxThsYfZWuC72Jsa7wmT5\\\\\\\/WaHg0ZRXGrD7VDWKsTVTipz3YnwU2md2bnVds8kMFGtP8USFphHRL48WgHrGfC0\\\\\\\/V5agnX6oZ8mXxZhMl5ttLfYSu2ayictmMTAB9abMmzPJ9p3lAH6Wu9KDIVlNvjUkym\\\\\\\/t0rMFj9p3ic+FTeZvUpDOXDAABPIMrlAL74Aw0\\\\\\\/ugHZUXkebJJyTUlwPdgHsjuKZmxOGqEJMqFEkB2\\\\\\\/dZo0WBwlmy+eq8aR8zSsnJL9vcBSN\\\\\\\/PSvLZiyoRZLd\\\\\\\/AGyinTu4KyNOtm1nQuirKsayKeZDgF4h6JgA7zg1pv\\\\\\\/d3Cb6rFidxxEK8tKlSVpD6o2C4iMqzhmzazx8WHqvi0MNvl438ep1PGCNgm5RelCqWhSLEnzAPqb2NM75l37T6Vv5G5MV3AaUacPMRQAwhNr1o1QukWzuYeUkionvEL21Pf2fYDgBH2baeANSTOa0Wnl0n0EPdsSxSx\\\\\\\/veutPmCUdMh8K5A6ZDd5hZVIf\\\\\\\/o4FdhCygrABHPXV4vw38SdAx7ukrDgBcgDNP04sc1VILI6WEGOitg+weJ5OPAvR08PrgLfKDOpqkTgRPq2IdaiAC+l37PyCyLxfdfk80dRJiN6BOr\\\\\\\/R5XxplyBoiMEf13YfktrXrKjSIqGkPjApXXFJQSDrxehfKBaU4SFX1UWRFyScYPgKYtb6SI+TeZMDJvtU7AkG4gs5om86iHiHeBdJUOYChCjhOA2cQxzeWgaDMwSJ8o3GZxzUc0KRkOrU3YtUA5RGHs+7OQyTqmL1coocZN7oQ7Qb2V27WUNXEAu0ChbsNLbfafYNwxs4gTFtmIg1aOj03FKR1sMPOUZ7bOcOH6Du82e4gjXUEOQF4an8kVjPeUozthwoWwIfbwuchd9I4WXWK+hJamS9zhpgk8OBich6tpgAPlEmxHQYdlwvZatQFihwVAckfwlEzZ7I0gMgme1\\\\\\\/EHntcNj7M45XGdnfjZVt9nnaU1O0NDoIASut7akrJnIlVrBSZa+by\\\\\\\/q+WHWzF0MbgxKreIHDA3FAAtf5BkmyReaoyPlQHYe\\\\\\\/3aEFWbTdp\\\\\\\/bE3LEZkDLqFr8z1Eh8xoazmucbdKNzKhJxjLqmmSUT+C9mCMTyCWWwSdl0VcH8ZEwlQZ\\\\\\\/B8H55jpix0fx\\\\\\\/DVsTlIMnLIihmrW7rxQ00hp\\\\\\\/GONMXFYND+t4yRUUkjWJcQSaAmc+ov94xIVk7mdQ3zSxSTwuuz4e88NLnCh+g2cbxfdgF+TreOwxSQjdsfQfHGJj+cbu4QnEeKb2hk+dkmIyVQ94MfcEOYurHJB9bwvic0BtbCQHNYaZzJ3x6Po1jQgz2RwH63BIUgm0mIylA+vtM2TH6f8omULU26oFrQuyI42bv+dOmPLJpMtDoNtkfgwhOiq5vtPmpkhEcGuDlAWN7L9zKeVS\\\\\\\/\\\\\\\/xj+cdw25pPz83MPuAp7L+JCn4wrGemsRGpcclk+vzW5mRuWHzDiG+rLa1iTrJ1I0eyUx\\\\\\\/0n26YxaxSAXCkrHbWFieNUT\\\\\\\/+TGxjCOuaT0bGE9VD+m2P2Wumo1+3Z5cQfaz7alIZNA\\\\\\\/lhHijUtHeQf48rgTM3oEv\\\\\\\/WYWGZPU0KkpFGpiNLpX\\\\\\\/HIDO4O2awQK4EEo8LhCN1GFbaxWMSvyJu15F5PxXWOHPYXEU1ylbFx04qRJNiQ84F96muYeQhh9ZkXrnlqbMztG9JbgM2frM18AEMK\\\\\\\/DOyh0tQWH\\\\\\\/A6aV1Kd+8lze7dsZLspaV1RtwJluiEsOrgOc+SByDXCkLdEhWNcT6WGuwWBlF2+lo7M5U0n1XiL0dSuR7GWqRwYLbpwKQV6L9abiWWS+LYzSHPT9ySTrIAKNGe9DW4ot9HHTQd0+O58N8o5Tg65ZJnZMDrGqSxcq7pNGnaYj+iZy9XUbL1bsnPueKqaKfUA6n9y+Q\\\\\\\/PplYIri3gcZT\\\\\\\/+MES0bxKxoxe819vbyjh2ms5uzQimXaGn\\\\\\\/FGNHSOKlOxbQN0qTx8fbRPGxtaH4zwZ7FqCR2Hdh994Qp0iJTnxb37RemBpgNJNNG0KJq+jHiH2T6oq9t97k3u\\\\\\\/2uqeYIgpD+iAx+N8\\\\\\\/MaFClKNXuG0nt7LVsmjID\\\\\\\/YRbMgvhaZN8Xbz7q66Qob+xsb9Vl7gzNW3IO+KHaiURBK\\\\\\\/e+y5OwFWLT9eqFRW3a87NIBR3zP8xU6wn8mZ8PeFWhrmfAp626JVh6eVPzERATqMsUtAgYF2snkLMJc5Vl4ILH1JQyTtzMPGnLjjaccKuXEsJo+KjbTTv1MtvZbSi9kbWJ9KtYGDZzcrd\\\\\\\/R9hLi9vb1OpMakeu\\\\\\\/RUR69NdQQPIW\\\\\\\/ZcEQPm6nTd\\\\\\\/gpcFnMd4hnqiDek1awaybyWOirUseWeiH9Sj69zyHRPN7etfiGizKAL78OMt93qyjox6EynXwdcqEyeNMFF\\\\\\\/Ye4obw3SFxFgQThit9EoQ4gViApKEWzgt0wjFVS2VOGenDMzSj+mD6UdghW64GXR4I0wLbqDbYkdMwEpJbpDMFzdvbGzyUVo\\\\\\\/DGmVDdUOKZIZO19GdwiwCsxx5atvYZHzSWFNf\\\\\\\/oeihLztFxGVAqkqtRMRAwvFEZVu3VIaKYdXuSkTE0lY4tlWDydEtsgU9GX\\\\\\\/4VaySC\\\\\\\/a9JfqNHTCXQPd3Zrcz\\\\\\\/PzFIX06i5Bzk2n6ClyLZ2ClSTKCuux9nwmIPzCuWnKMWanKioPC+AGiaekadBOuRdDQ+yrrj0cwwap\\\\\\\/BjtvGQxowNIOCahFV5V11RVVPowJqEVfDdGetx5vJSr+ahNUAEyNVbpBsBuO0azG0H6itNTk581O6cM3lihG+SjGYYruREKDRh4KW+F9j77G7Y4GfN0G6ySykuqICOeBY+KgRZQ00JF6WaKpF0q\\\\\\\/tnQlCBIsfRIpUjsuA5mBSKBLxaf23GT\\\\\\\/WNrtPCS+53mSOx77QwmnYJVnibMJtNTE7xR33R\\\\\\\/6U0wiHrySxFPINJWseS1iacLQnG1msIxeGuU6Us0cIH\\\\\\\/YL5TUMVcg0RHpifA+1jwRNBeFMorhhcVzH25fYNNCZmiTV\\\\\\\/8bt\\\\\\\/NG9fv+IFmeQxa2oJOnvwGg5cUtnS2qigxZBUjUYJ6l\\\\\\\/xiBWjVnepdLAB6NRODJUDXgXo3WdcpfZF0B8MqHOsAi77GJPVMvdb9io1vwTAftheh0UIYKY8IFh\\\\\\\/cP53FHcNLvxsadgB3b+MpJUc48nS9OnSDXpg5ItOP4fjvhVJU+BfnUZD\\\\\\\/\\\\\\\/4wiNanWpXoEmvrgQ\\\\\\\/UEWFzDYGl1F6ZtLqS4wkbm09TAsv9i0KmPbz7h0zSNh63h5jav0JEPaqSrCWHAZ0fuba2XHOXBjc6hoH0870a71NnAgDd98sAap4MQ5w4k2RZ5dGnl16PO6kkO5YKlUa7dnXwLupkEwm0SeRBJ6j+TxUrv5Q4WSzZxi0a7\\\\\\\/4+XqC+2oYre1gt9EfGZw+svMlxkCWnJtDtXFMiu1rlMYOJ\\\\\\\/oXjPZH1jeh3odqAbFqV3p32EoFFzFTKACLus8rN5SOUpqU6JQZGRAMV4aaXk\\\\\\\/Pn5u1HVCsVoS2oDlx15LF+5fTQuhV8W7VzMdcn93ofLg8MniqtpOFfMS2LHE+SwdhWBVI3sKIjOfmQb\\\\\\\/1F2\\\\\\\/cstfnO66dN8qOgWDUOJq8AJjoS+6CMTBrOZSA3dfmUS\\\\\\\/qwAQ\\\\\\\/noY081kWhVsM84D2CXmZDczkrLAl51O+yg4vzspaZDyP0qDU1UeGAzruH4RGXv\\\\\\\/Xji1fxb2fzMJmPjFYKlncfqYUcYxFyF3rD\\\\\\\/2MlR39otVh5tB0btZDMREkE\\\\\\\/DLa7pJMFv4OmkJBdqse+ct27y38mlH+4DZqdZ0CyURmbO0oEG2T7KYCXvGKLH+uMDB2e8kkwXNnTT2AsajSN0bAx982A4iIEmL\\\\\\\/\\\\\\\/zoVJwBZCHj7ISm98zQ8XeHFwDiAeizwoTh9EN1hqFUKz6DMYKRR3TJkg73DwpbVpe8qbYixFOiZ2vUtRLwEB27wkYi0qC0tG9myqsj8nIqcTwQSYLTQWupwn9N3U\\\\\\\/NVCbSGnErwpIiNXaEg8sLrpadlod2sP42pSqW38TE0GXZk5HkdwyQa4iaw+V4ajTgA7hJFgd2mXLgOPNpcl4kuODsjPHVsb22IaclbUdmCEVR1mho4Som8LnXWuPYneGnor4P+LRdokQrNIlosCt+KF8Fj0swdOCVTyd7B15m+E++g8hVXmQtiQGCGk8JnGzohhHbX\\\\\\\/DpIExYJWzbeSuuJoaYIpsqwXvQWkhxGwNcRsH+DoPcIz05pWDChObOWS9oPTAaiZ9iGCux75krSk3RETFcRGBjTSFGQ5rhni1ls69N0eAKfW3e5gnvDZlRq9NzX7\\\\\\\/LFMiqwuxIijVu\\\\\\\/hYmE48pGyDELY2re2CIYmM\\\\\\\/mYMevh+hs9G5bJ3GBmrBw8IwCinSxJFDg0ZO6tSoNue7tDe35Xi5\\\\\\\/nR1OHcVOsUSqmRRO6sCUaU9paPcRhOYK4R18dWafTMHABe19imGHcxTcq787VxPCiVeNn3uqXFfouE0x68YdJIHvmKoFnaTek0e\\\\\\\/vEirw9IB81w6qQzCsr4n4n1\\\\\\\/x9RYvBmxbxvGeSmNQVXh+5KMzD2KlTWWDlIY1cjHwBI58e0y36J22AZ3JMHogS04rqG3qkOo4dyJow5GR7EnXOppK9YgW5JOTIttT326pJo0JSZ3s6cuAoy1KfVU\\\\\\\/gtusrj8wqP3VobvSuMrpRud+3cTCVAi0C62hfiwVk+0k0IFkfXSdL036IioR8hQFK\\\\\\\/gpy7OqvstX86JPRhi2kR9X9h4pYZHFSf2P7TX5Cy2XbB\\\\\\\/iZu95jQmmuuPSNylsxB1DSHm45pBy7wD81h+ITqFVbzphL8dQQqili\\\\\\\/jcRD+6B5wbcuLu26WblmFZ3ilmeVDIe6rAL3EoyBlQ3p9j1ni1QfYZZL2jFTd3DvqfkYjezmQN7qPBMgvFBn0r+JTDTGG2xoTUuPbhqRz+TewNBIA9PU2cYNVmt1vugPbIXwqSytw7hLdt+U9Esky0ePmLv1VsrY4T20WoWCXikaoA30wwe7hMbKDPwqj7mxq9tdGdhbZyW\\\\\\\/gC33goLswyyvUYCSBN9rWFyRk6SReQmYITBMp\\\\\\\/zce6o0JvCKKpyLDGO3q2EDFKUQwgucziZgNaSu\\\\\\\/dOkP2huAUPe0AQHBle3iOCgdq1mI1qYE3\\\\\\\/1koAarbPcPfWYxnAw7pxu2kgVClsLts752nEEBlYUELz1Gr9o9eFBPLNVNO4PBPFPlnYk\\\\\\\/fEgQvZdTVaZ5VgNfxG8ZvfXmT2o+zTM8u4klqWRYXfaR2wS9CASqzuJG7u2CkJydpNrlx9FZ01CnJhU43xu9eBl2IPjPby2UPQScl7f765N+1wDeramXUUiRgKzH2t6ONoz3G6VMnRdng9Z9qZ9pq60hi9aHQbHz8Ci6O7dAE18OKWMttfR7Y4LwlKDsYpi7FUJNwLI6Y9QWkHrjU6A2luP9x1cfQ5s\\\\\\\/mZ5yuL+I6CCNHvIaIZOzzw1mU7gLg6Pe7CAJqZFRKuisamt0dC8GUjgogFW7njPB9XKouuRVHu3lgRGGzKqccEIkCK8FWIGmML4FQX3gLd1JSjmvZCc7TiBis9i8oOBIgU+R28Cf\\\\\\\/JcKJp1Ult6h9oiFWxBggpaNY1ybXMWK1lbPlGjRv8A+5nqTZcaYkHTm5Jgng8AnJGvehooAaeHTipa58C7FMYV17muCgVF\\\\\\\/jfFctl\\\\\\\/vnF1WgxyUVuJac5IY8UGHEuhBjrlCR7n+snMYrydk0vyIZVRrQIACLiGzbQ4YylYCVfz3srSXYCGk4nr9W0wNiFuF6scNPqlarPoUeu4KViIUt\\\\\\\/KD7dqRVKyQsdnTDsdTG0Y3NoeQ\\\\\\\/e7sKwOplRH5\\\\\\\/ljnTe8q8zTkuSZ2xK7IL1HpBJ6EFawQbC2nx6pxK8J6uObk9KWmpLKjsfR0H0fW9hH3dBg2lwDEws4Raaf1siofRC5NTkHeq\\\\\\\/YEXE8LsqFAjqXkTHX5\\\\\\\/AVIPGR4SsGZlXMNqcFM6byIKjbH6fSeMHoXumgP4dn3ntxeH36xU9hhlXyRCk5u3O9haVzg\\\\\\\/TBPgxNoik\\\\\\\/Ryi5XAGaM6uHeq7rno4FtfChtC5qFmMQ4ND6WNrAIFuwmC\\\\\\\/qDZIPWLRBCMMeToET16q7qSbpO9vgnIII09dFt4oCDhrYqD3lWipZ4IUOxz5gVMKyyn9wDntKw7cCNkV0G2IBv7QxlDSf\\\\\\\/IPcH\\\\\\\/hPKTGFsQ9OWMrj9i+qqaESsCUcN9L\\\\\\\/VbY0ZEPplIbhkv5kbn55Fy7Nv\\\\\\\/\\\\\\\/joQdVDhqOLAIhiC2BhcvfHvOg7N1w\\\\\\\/cpMQIZPnBHnE45pK7TGM+wvITVTuBjom+XJ1EJopiKV9DW4pWSJBpUqhyAhnGXCYkXdCMH2DN0yN1ph8C2CpLgtQrcgb7LUIh\\\\\\\/9SpbM13tVCiGwH5VB9PIB+mybJfbse6DEm0tFlwoHP97lGtQk\\\\\\\/LbyI9CQmwSZBIzGRoDQHf\\\\\\\/lyk8liHNR7zol0zPOP7rvUrHBVAt7Ac+v2OXN5rG3G4gsN238JpXbpEEhc8ybSyUfX\\\\\\\/ThGcKDAiq8SCB2HpmoHHZkAJ4gM98glZmGGKNaieoPgPWtUXgsWEySPsMKsnRdwI0RAbIza0vIvdsp\\\\\\\/ZfSLxV2KXzcTayYKlYOpIeFGbtzDDvIOmRiG\\\\\\\/HVQlDHiFeuEYHBeRpMNMIXyuIQdWPMscjrVU41Prd2jNfMf8lTMPDWmJ9E4LOBIcZ15rBDo472H8srfJVtOV6RFSvG9TL72D4EKnoGk7S6LuJTf02PJwiFwtG9vrFCmODAB48VzOHDKF\\\\\\\/UuxsaEpgaRKHk70XQ\\\\\\\/c94CsgYTc\\\\\\\/qE68xLLxyPtbc3i769eLaqiPqsIMpUtIcprwbs92lOAcJCKLQ4JVepPcsdnfRci6PSUx2lDBSY+xD8hrxV23rPGBTgC3S33sGa+izFE5cAkY954Q8ZtE0cN4hkkahtnXoOzKRwTJPAfb3a\\\\\\\/l\\\\\\\/rk7lTNr8Wcocgeggm7ljMdy\\\\\\\/RdHU7IwXNQXyGyOxsaOfKigWsKmU0iGI4lZzTin7o4HzQ6Vt2GFeaNkRfLwFlChliHVxsrlboWtPkOGxLkZgIYuPYjz8V2oWbJqIoWUqJJ6+YN4p2XmlEREMoeddUXO43x7vFq2K1KaeM8mcV\\\\\\\/gKrF2XAQDOusa6\\\\\\\/uq9XZdzLnjYBDl\\\\\\\/1gl0yvuInqwlTgeRLeilZfRGyN64m1dI+Cy\\\\\\\/gAoIinIAHYAdaPIt7t\\\\\\\/KyBcrjgGsvxlht3wQ4amIWf\\\\\\\/cLPhTFDYq9dOT\\\\\\\/Kk9pqVDcYINjha+NjKPh1TxerkmeXYHu7hiT4D2SK8cEYoUvzVLFd0DkYych\\\\\\\/Cim8gOCmlsghghZM+g8XH9fw\\\\\\\/pqOsyVUdsquSg2WGjPl4KWeLdXe\\\\\\\/PTnrF7wc6wWS56\\\\\\\/kqaRHAQ3dOixNe4pK5gmvQcCIZHAYLObNcKq7GNDTzstR3LDANawcrvyZMIApoxq6PbU5RkSUIvv7aSVH8L4jdvRDD\\\\\\\/SEJAfwwrHEMqT+u3x52+gnoHt75Ea2S4EmxEiudpsVkx6VeGaJvskEt92yUuRku0tfiTKvWGMOky01rRX+0ZLQ9DgiubLLdyfjhwZ7Q+DK\\\\\\\/a4o7xKRL4KW47D8w28iGCHUZLNZvSAYv7xEqFozEUHEM7sEmbmJ0BcmAbo2HhSxzqlGs5G0nlN+lLgC0Kl8g\\\\\\\/BiEkwr\\\\\\\/OXMPYkJtlkhACXQwXByXwRqKUewReID20+eqbJr3rwojJs+i9oQVHX96QM\\\\\\\/tBMdx10xijl758u48i2+jDBiLCTFVGrd+Hg9mXGkdrY2qo2aLIQQwg731eHmwdKVfBwKuR4jXksd1iFyIVrpu8hGkP31JN1JqCvu8xBCyOOqHYsZq4h+1nIXmtG5texK8272jjUgZBmeJRX5YVB5kcmCHYQkY0bS5Zgw2K+fEi22nlzQeUs42ogKmIqFNcih73L\\\\\\\/obYgT48x7vxP8ra54HQH4i\\\\\\\/oXSVZlo4mPjYVkYCCFYPoM8biG8qUFQOZFU2PJvF6+jEySYS6fJSHaCCB+D78RWbjOlJBBW9gzFR+CXMPvTx6CNoZunHnh2BWCm0xfrqf3OpEM2aeOdsGzIrKJMNl4EW9DztMaxMAwPMcsctSZIuBqiXvkhZY9xYclcJhdaGeRl3aVYJ7H1ecZjyRKghohv+Y0D3P3S1Ot5iHIsA5tlL18BtTH+8hkdcVml+Oaz\\\\\\\/SKSLOZR2QeyRZH7fkfIDZl1izAptblxr2WuzYRs57UDNpIwgmjvk+ALxKIfgnZyn6Kwk\\\\\\\/g0Zk98ggZxhlRKUktcF84BTaC9LqZy8AC0O3gl0f9q9CxXOeTdip6baVnjY+WQ1zc0TtuYjXcsZknEvGvr282zfNhqGgSjK2VL\\\\\\\/jBIst418n\\\\\\\/1n83i2mfJD3F07mF+\\\\\\\/TXCwWSX3ijyr4f6ZmIU8qlBQnX6naNDxWSZi\\\\\\\/B0MO\\\\\\\/PUvZr7v3Us+9BVMc+qHy6bEhiSVtdQviYbJ6jc\\\\\\\/zKTyzoYMB4zhWQEnInvBC1\\\\\\\/1VGM6F9xH7POdqA\\\\\\\/Q3qd+MCzo4fixrpElmxPaWoVzGzkHdQ6c6UPWmWnUor11Ea+XU4P9FwD6J0E4Ol0\\\\\\\/9zWH7Xt08DvFPiVNjDelVI3L4Y9N3tVOAfdeXbLJ6CyqQsb7ExtWPn0k21exFkkZ+czNu3APdf1XdxffqRCbibigtDygFserfkJkZ+VFX7RiIE3bOFFPIbIH3d+J5RJ8kK6vHHfKFUDsVCBU0x0BfwLEQ016IRGj+O4UhqV1Dr46FfVcXsgRJ6WMAuc2XO\\\\\\\/PyCT4Dbbt6M5Rpc6XgFPFcNgH7lVklrOgApNIWrR5p7wd8kVkazRiP9jX1V\\\\\\\/f8qYbe4OmkCtqsbx4hZPXZzBNv5wXgDwwor7noB880KSDbkYoSE08ONiM\\\\\\\/th3siXhMCaQvoXPK6aD97t96EgLeAj7zXTWst9BOjd0s4sxhpeFQ4ZPTTIsEibHg5SFMCGGZXtpndPk0NusR11d8DjBCe+i9fvM5DQYNfo4icj\\\\\\\/zPTQN\\\\\\\/Z9ClSsWYopkQv+7BSRFS1IrzoCqQInvdogCz0HSrMjZOl+7wxAEycPcytfGFvs+8PHd9p5ZMFnPLnKkbUeODWU0N\\\\\\\/ossrw9KL3qKmhnt61QxppHyd8ukovAoiipl2vuDQUZNWPY9Tn8ykVm3YTjoespNh2PAj6FnIelEzNpZ6+IqlrAuRwsj3keDkypYh1UmI7N4ie6I9mXFr0BFkrlHT1OasZQhIDyVOQFYarZWWkPalMxGPqToyZiaPDnYkn8amrevPgMC+HHvImC54G3P3fEcIVEkCI+Rripeds+urISM+PXUZ48f2zZNB3JPZjMkHWXRf5gAWr4diVLJ\\\\\\\/dwyWOFPL\\\\\\\/p7KVD\\\\\\\/kMEwMUXH+OnY8yutwZNWh90KoczIEq9+s35\\\\\\\/gq73Hn\\\\\\\/kN5n5YRUuPGkQFJ63uHzpB\\\\\\\/3D3sLrX+egg+eO0s8Uo0ztGJ0xKFnlqjkeWJUyPEoLpUbHiUSRr+L\\\\\\\/nNKlXXIdJJn+i4txtfIBghfiEaB9dDBkaqnTCbU28vcYCPuQ\\\\\\\/38Xf7jrKtbbwsuFEIcUtXoTg2NArgOm3jqRVMYX+dE5nOWi0n44T0EpZ6cN+s2Bcy4sfencUxyQ5d9AXsZR87C1lO9jsVUP+9VwxHd\\\\\\\/wepwaj3vcAiz4W1XNVq+YvaOgcAYOrYiK6CfS46Bf3utxyHSnO+9KMBCt\\\\\\\/O7j45MgqaT2C3B47PlQYnuoM\\\\\\\/POKKg9vCd+jRZFfpOtwj3BFE84A+rU5sQF8iKiij+qkd9v7slfkDbVPI2TEm4SsikxQt1ziaZXSoBHIPihoOllQ3HzV3E\\\\\\\/rq\\\\\\\/TXFmJuw8KfeeUd5cdlyGfy0vsDC1BwWGWrxZTAVBSjI8eOA1u5eiuIp1k0QPg6lJ76Tf+6zBsRpgXvxBrJOliC59NwHsbHFRqVOZwv2CQEldR1He2PY4hLV6rE1W58l4d7x3JTIQR6kvGf6\\\\\\\/k8TxeapPME69Jv+IFI6Odt\\\\\\\/bWtH+VX1dLvdahwNLq9BVa5mddPJ6vdwTl0v04pEcetT02VlJMtWOsURG1bDwTpr3MqrKSzWLSWuRoXGGrtW3Tk40LYe966WVv2BMR6Yx18mYSqvEszenH9nz3trQZp1f6maNO\\\\\\\/iWct1Ma\\\\\\\/Rd3ukT\\\\\\\/BFhtuZPfskLYaAH8kRjgGkKKp0RCMjWfxu+96Qe1+t22Ap5RoxLsXwpV3JGqg4ZhTph2YjBa\\\\\\\/BQwAPz9gL7hsd91szUTNHpSpDjP6YIGqZ7kHg9dkh2PXVmllUCsVV2KomV0fnJSM5jUXpmedutxEHtT\\\\\\\/lFlkb2HJnMepitgKpT83ioEiLVLr19+orvHKb6jb2XRBUgSvXmwZt8d51ljWpegDdvj1BVl3hZd7CnJbZmiUZnbGJkdL3K\\\\\\\/U03VJZJzblv62rt+Ymft8KSp5nqKug\\\\\\\/fRNOmX\\\\\\\/xmcQAgs4WRckIeqdSekbrRfldFGakHF15ZZOdoeCnvAPl8ztlxIc13ZlimHd+lCLKO8CKOfCSHi4P6qtGjHFmBd5vUdDycS3sa6u3hgGvH+9LNTmIDXj7HZiGbNwTO8mPbVEHGxb3kqdrowyArZhHjUpAZorJEU9qxVbIQnXDVaUuMbs7Le9V5oTFPVXmOhTQH\\\\\\\/QdxWIFxT\\\\\\\/TMK5vkk0G\\\\\\\/HMzJthHJrO5eXB6nUAe0xTe9\\\\\\\/n\\\\\\\/EZjUoqKRGIwrBB4K2i0rmY2G\\\\\\\/wsEw2Fv7IMfYbwqq8Y67\\\\\\\/rY5WyJmTB2JZ2CUluj4fFey+bzmVxVNst9QsAuPcg6PjVa+tE0MZCAx5M\\\\\\\/SrXQXqSY2Q4mGhmC6ZBAku2h5B+\\\\\\\/5Im86PpUsme2u95zGGR8PjLnptr989gA9GSpElgpB8Suzyn4NnkjhDOjxA4k5Px4bUFY1FZHLNYwHYlXiMKIMJbz1HkWTGWux9FB0AjQBXVTx59ImFlacJxU\\\\\\\/jMut4+BnJkXDpna3\\\\\\\/YaqTddg821fdQhFHqPZM3QNKEXfsKEgNB2Axa1aXxAMVpF3IjtXg1SQa4360s\\\\\\\/snrn9rkYhGbEJHqtMiU7U+Q5tKweN9yXbzeRz+5ul5r8H1SiWbQCb8I07PclWi0ooDom5YyzSJKMatTen+FS+M6zuUD+\\\\\\\/qnCo2hjOZvXdrFQ3JXBX3yjWOncvNYMZBi53B719F+BQL1G493x0xOmzU21SIeQCybRXAbCdOssoJB8E7EXJj3TwsrGgrtdyeBYxWMw0ooCpZWiNeZ+dMCSYJRyo\\\\\\\/3+JB2t9+lGX5JFtWHemJUXG80dCUb8NcT1okTDJs8x5XUzDtCi+NmTp7bb\\\\\\\/+vNFdWyM1IoEuBeaqQAnyJjYDBVnWedjFt59eQdOcCDqszAZU6UHiGdjx9Ca42cpZ97YO2BvcnDKwibM\\\\\\\/28s9fObykITA0yfP40ZZDNECCXMQjdiQvaRVlYniN2hAXiYiEKrro3AU2t75u4ws\\\\\\\/VwDRzZuzTf2\\\\\\\/3lr+Vi9GtNf4ypbOY4rgFaKzsUf1VDSi18688RRgmYvZ5e5itD202nA34Bx30j4TEBn7La\\\\\\\/NFGVP2Xa+eN8g9V9uZw3GX8d7D\\\\\\\/axFjH4347cFs1n9cVp0wHTJDp9vhZqY9h\\\\\\\/sXOQevys+oCKtSVLNcM9ndTzq0D99JoIhP45ngBCNfUAMa2DBktAe+gSKqGOvvZs5HvlKqtgNg68TNWhT+YgDmy35Np2Eel51LVefutjtdkifQppPuukKQmfZs0xUQCp0oCUQuJQeu+GdGlsvl8cdcVoIRuSFRJMlOwKarS7jhY6YXkOf02+a7nUNi0LriFKzA6we2wwDElWfvF32yyZea9OFBEOCEaU\\\\\\\/1LAd+vQkt7YtWnUnrYdkPtgUdKBrXhLceWR\\\\\\\/3NMmvxTm6fbnDKGgi5Fj1JAlyy0BZUyrKXu8R6Ma\\\\\\\/AoEoqXj7b\\\\\\\/UA9BetusK29H09QJX4faqyp1IULVcnFn64l+ByzOlkJZm5iIXopMNBjofTIgtg8g1NPeew550XUQl2QuLYvmxqBWsio5cbBa5CaDcFhGVsF21WupN6rWHCRWUrGDN5p1FGnQnM+e5fSmshI5511G9kr4aQGF9miczZqCsltxO8kq+0fc7Mehgwja0oRF1EvMqJoqeQMPPjd\\\\\\\/H7Xyg8qO9SOgI5+1xBgnYhoM9OL3ojTjGJs+fEA7mh23GZVda5BBCj+EOSnzvSVJlBAUyGPcVcYP+NLrsklaFcgrqUtGxxN4TWlF9soj6v\\\\\\\/gSLjPiRXhJeD+S\\\\\\\/AAD78zCMqZkwTYo465MLbRkKf6eDw0dJuMuqCVaHYM2NJWYwoId7yXIqF9dqpoDuWSSIyJr0IuUb59DZo0t29U7fQEuGWl7ZFOYU8\\\\\\\/fIdpfH86LubHDj2hhs+kMw8lHEsokwhgejzWLA6b3pqXx+vjkdTMdHsLdu2k+u5H9egQWkzOs108FL5XBoD6sfyU23AvVWCSDXQKpXKyxOh89qb7E5Pdxnof6SB0f6ngxBhBwprKXW3IP5vJyLgFtEuHeGqeNqy++o489\\\\\\\/sML2NBEwRP1PWsG8dvXmzz\\\\\\\/bvOLBTu3bXh2yExEn304h6ghftRVuwVpN4kZaQntOvTR7DwrrIfkaCQ+fofCoP8mnT4pVehYD1u44XXOqFndPhhSYo4KloFvONIpspz6BCtbIvnlzQxWpYLTiC+FqevV+GjahIYrBzYmTnDYzRRI+1k1P7HrfsPmj0dqlnH3hvGnR0eqVpn9BKGRVyeunv9oTNZhWp0ghGnobVlrQhLJCw1FPTt8Ppq6MY2RO4hMsLrrL1MsEvViwhtdxhfT7pR8fLkgjA0a8dXJt3ed2YRxK0AZZuznKgGN3b3hTkr\\\\\\\/xipSIXn4HiZhaz8dKTjqsMVmrcy715vHuvEe3r2T7ZUywqbEvycUljEX5jhYteYdKNhnhLqEjeGDjZksJOYYRwdig9BQTxEChhEIRMI1YF2GQHrBgxRYWzU1itSiYotuW3GVeIS5gLv36+Rd3kGix5PRqvbJMV7v3lW51XbsF1dnATF7qsXLZwPHOhI8QN5KBk97WlMSv4TnickQA75BplxZ7BZHWry8qqBEzKyP0em96ro5m\\\\\\\/OruuSyr8eeT+OErLGEmStK5BSaVuaC25Vx7xfQTG5QRbRC+1UkMKdvnE4cb9Ybg8hB2W1Bk3ViQdnAb6O6kHbQqHiCp2ahIDQYUFjHWNSaWCsZA1qqoX65anhSnY9vxZGhZxgPK3jBFWV+NlyXmLaU1m5y6NOj6\\\\\\\/bDSfUYyBOmUvyjFtuN3Xbt2j9hiMg2w0c+TJ5z+btuU3h\\\\\\\/wygdT++xQfUCDoMS28puqq8MRRjuJ+lKyYaiYuaSEDrLlriWyVxB9QvwVPcRuSweGJKOPz0aTDOZZbzProVby1DJmN5L2t6muJLR2Qt0G6XxLoDjE7eVtPOUl10wNcqvCTtqjU8A9B5ZRI83eJSdLaaPn7j2cMvv2d9A3dhohbht2QSI4GBitXEVrnPiDcxQHmKdIkNeyvHrQpjx4OqlrHWVOlWfaB7fzRbbZRzzDR8VsijA3f+ZE\\\\\\\/+GmSPX4UcrZFzaW\\\\\\\/P+6kC1UB0NB\\\\\\\/0Do+\\\\\\\/ZnlhpC1bW3UJbGiGZggOH3vUNfaF8UXGLSeMiWrQ6q1USVnODwJ3sqEVBdKmbfmiRsJnxPB8XitEntMHbhmFwUThP1MazeX35rGmpLDQoTOtM1oyIpINsAReVWNOvAUJDBYxue82BmC74tNV9tFDQfYiQ5Zlkz3j3oRHSsyKLHpkzeSYvTzh+15jzQhWImImedtUYWbt9CGLiqV9el2cIDZ1hj+MVLOE7jEyxy8Cob6jBVb1QYBoCStvw8LqPfy0McOf3INuSA95WTA3D95mUEEIuiM7T6ULfA365MGV\\\\\\\/2UluOZ1wFKQ4xPdaI9yfJ1vSnw\\\\\\\/4hpqB9OVDgASX7Mh8prIBlssd2rY9DWXoJEsdamj99e6XIuXsl7VoYeRQKidTxVyWWfuavHpmOdE3bK8G7d3T7tW56VUqEkKs9o6Osh2sN7J99Cca8mtKC4WcIdLxqx0nVLy\\\\\\\/uLEe2LSrbWlZrU\\\\\\\/m03CJBTe8s0YeyKW7UqfLC1fTD5PKToA2fiU6LkUczUcWk06QVeSJyLEvnq0vXKXH18mLl+mdp8AFLiSTNqjPFXGLnHtzAdzbLgjdo6zz3ef6cbyd\\\\\\\/UdClNrot0WeshkI23FVX3sh8IUlKb\\\\\\\/Gf1tHrxxyiEr\\\\\\\/rsBuhMWFjoaq\\\\\\\/xTmTpns7IMu\\\\\\\/Xu3RSxMdyuJWLJSRxhrI0zt5GPfkm\\\\\\\/HYYyyOMaGwVHM9cxdLHVrK4KktX0qNN4wNsqlP3fqDdMR6\\\\\\\/rtEw9QzR1O4PDVnytvGab3RB49rJVzP0\\\\\\\/+uD5fMe+0PS7I5gREzRvRrtcGrbETNKonk3XazDrD2phYFWiRKT\\\\\\\/+aQj\\\\\\\/LK\\\\\\\/H64jqFkAYv9AuFfZV0GjHFRw1guFRviQHGdm69aeFHKGEj8KAf9TI9U8U4ATn0RROnQja+H0Ob\\\\\\\/3zNZAEifmJtiO6bGBsvQVlx+3OeuKOSFK0mr8vzktuku+7i56SK2\\\\\\\/iITyyw\\\\\\\/v6vX0tboxgPWJXYj83JaQHLoTOjS3pIoNfQWk1VmE2x3FWEZ9PS7og1m6y2Tr6rIWWoCUfo6kinmJwWTlKEgiOBjY3qX+0Q+luNAclWN7JZoi\\\\\\\/uLoaK6jcsXJiKCa3T9jruda1pWv+7f9Qdr9kUBkm9rwVvTR6ai7OUVo5Oedf5RjVXxSR+JKa6AGgb2OxdKwOyMfAb7tze+E7As2avGvGoa7NU0udUH2JDrleU68Kja9939\\\\\\\/0BjbXn255GvDBJGxpJDBroSXeaEU\\\\\\\/te3O5OAFKXMyS4uHgXtb\\\\\\\/YGNCUQvOjVy+BS+BJXX6NRW7wuZq3av3EXLbXcXaE3aYQBQaFgAtlFic6Twg5AsWYBvoo6k\\\\\\\/hCCbCY484UBE6mKgn0DzTn2CE1RYn4FxE9V5Vcg++rxh5zGLJAl9ovg8\\\\\\\/HEffHRDzO3xvScSYmR5TUOaj8TDx7EDDo4zqvCH7AVvQP0h+QoaiN4kn2EcaXVT+bKNMbbAsZM95pT0OnOTqLKGd1b9kompp30RRsPJTVh0rICMv3UzdjPe38uqNQeZHjUkb+Y3aAtlG7uaEItyU9lI5UdqVJ8M617sDoI6i+lz86mrY+nLzCN9+0kDrZLfbsDMfoIL8PaDFL47Rc2hbqq9mb\\\\\\\/+GeASpGSzzEb6UykTGWWd4FuuaFurHGR0ruhzss1tKyc\\\\\\\/p+eR5X46Y4OrkaBISfjGbbJE9Fto9K2+wbxrXvtEw8RDeixJY+reqRXQggBpoVdhx88gLmscMNEgrjm+wmQCl\\\\\\\/HHf1VChFx0FRyAiFTKm\\\\\\\/y9AQ+TIwQ+E+MUXJbWUs9Tz0d\\\\\\\/cfDbzdOXhnc8hKWjTMuaa9x++RRVKJhwNGarT9K2DWMFYOHZ61PbMlAfqqymcIhLvUqTspfoHBRDqksfnWBpgPsRBCFvuduhNfczmco1DIS0HPsyTuqjipL0bXDBKV\\\\\\\/pHcySkSQO0LgqCLk1X6p4RFIGyE+JHFAzBHnXXiY6ckTuP3\\\\\\\/K+vcL5OiSHkw20LG7xa6ieOcwn\\\\\\\/P1UdHX06Z1ois6zK4bHlxclWDktHB+Tmjmd8qUqbZL4QyK0fsHQ7qLxus9DqanwAMxsndHtzcfqTEo+ir5OwHd0LXqFZQHyk+gs0CjJOnTT6pFb4ZjmteLlruK561U5XPoHfqMcu77cmwkMRPDAw3ydzxeI5HX8Z7NAT3He4pIyIwkFvfwKbXkzIem\\\\\\\/yd38eOBi+mDGmkJC31wXL+KMfnFRQSFpD2rbuVuhi6wsNUBLgZUTHMrG\\\\\\\/lJO\\\\\\\/hC96hRkHIZ+UJgrzQketWYYBExV4Grmg\\\\\\\/qhgGTdbzL4Rl\\\\\\\/Q6NeVGp0h5C1Y7wGCQwOJ+XWqTp8O2gh9vNAfuLAYqLy7Cw8ZdIwtIq\\\\\\\/OHe4uZMwYA2GzhetPf8aDZVHYtfQYUZpyes9Myct\\\\\\\/kai3e0+fO\\\\\\\/kp3cck25fHtprzk95UaKGoZLkEqW\\\\\\\/gjti6+Ml1O1XbN3ZGD7BeOUFueCUR3knGSDdzpwWNxYqn3muIp+JY39EAYUcivuauTuqEmuKk4yTvOIQgG+Lq3PzNzDj9DKlPAvtOKKlVOEVgM9k21w+4ilgsKnDJqZk2XRXpQJLuOSOOK7Tb8eR0UKNCjE+KB\\\\\\\/V53nSH8eCkmfVu4HXjSH0rgAS3nqrSPLO9YfcySTcZwJWgK06pu1Q2G\\\\\\\/WLaIxtTaTHCJbSNxgHO9gr7sJIbxUi7TsNmCUnC74oL2Dyzg8YUYLU6hGBfjdnhODUGJMneBtEzgl2MlUnEqRX1qBlzbNNW\\\\\\\/n69A9xA6vuha9CcchKfQbSWicUVb12L6YmnN3J6h5kQ+3oWyISIzwozHGv01\\\\\\\/mTkD1kNbFSmXkwG1AJO6mvxnGZs5frYkjVgenH67yXfF\\\\\\\/7OC9SO3I1Zxcv1drnqsm7aHUVqxt26HapvmxTWV7V5qBkvstxdPRyLw3VU0IZ3gpm0HNet+Fcie81IZYE17LkcJYJc0PrXiw1i2jBvwlGdhYOWUYCIOxZjt4GHedtmGup8ZJRuHxvnx4UlD\\\\\\\/mj7LKBnKLYfXzO5dYclpGJ6FvgMBtWlV2vyHu9aP08E\\\\\\\/fck+i0qiifJFewOIA5ISPsFoaWZPX2SOKFDiM3iiD1BgGMJzSgBF31ktwtqiTj+tJLGTIrEceJ7gfFTRbFl5cbr7fklf6Fc0w5zQx8YkTTrwwJxIsKMPZ0N3pQCYXFSRg5U8u+PxM\\\\\\\/KjsusPa3dTuX88RkHRYxtCJj\\\\\\\/K58a8fYSYL7XaDq0BP6JDWn3yc2plObpCemnxS7y6WN4e\\\\\\\/oBeEtmqLbKHDWYuKMpPREl5RRvrn6o5Uk3UVIISNX1CwLya0347uvmb4LIMy0RpHf\\\\\\\/8ZgtWa8D3NcQNgEbEfsiP0kZ4zoYZBBNc\\\\\\\/ZeMJGzrS0y+dK6Kueii9vhR2fV73P5Iw4qjmsjEX0YZfE4SVxEM7zVfWJqeD87HaqxaTTTUJA0cQuUBzjscdz908TdUyH3MtiR4AMzE521CsjDzChs\\\\\\\/qwuedhJ5AbWROMWCE1bw3V4Y6Yzp6piF0sm+AjgmyvBRMTlNAt+R32s6FG8h7idn4MH4nmWbiaKdUCkB2fkMqO+GoYDZVysue3HAimtf0exOvBAFrlwMkV5C1b\\\\\\\/SCpYi99HV0xgVabd+C1phpxn6UF2Sm\\\\\\\/92xPv6ogpmuRX5TCyh0AKn6qvty0JSqfe6OYuPD+X1GNIoVryjk2Ou7r7EhmBkRUWK3Ml+qTa2OUvzxIalwJ8FyJ0FT8GdkbwFwoA80UJX5s1XBcLvVHW4QA46awgvwT1t3krZDPLs0qgmAYYUpcK2eNwXoVzdtq5xQP75DFSvM\\\\\\\/Kpx0AsI9\\\\\\\/8EeAJ0zuFdLkh3CR+XkXYij\\\\\\\/vHoSGsk28mmD2PSs3u86vCxNLFBFh7rHqODAGUcDON94IPycrXrlnEyNsXEnVwx+DoPyLqCgODezM5wwigR45zJmcU90uEcosAwIAurARD7dOw0iN\\\\\\\/vN9pot+gFmo1FJyJkFwH7Qm3DcccLrjaTCUNIVdBlD09kiNtmJwTcwH5v3at\\\\\\\/dDLA\\\\\\\/ygR+IVD4ZxjIHWnnnrUcoDYbIniUtVCHosSKPFAfKbFNHA7Wo52gMAO\\\\\\\/AoxVQ\\\\\\\/RPmRYNc49M\\\\\\\/88IMMe7BacEwAhp2elrEUtbUNxPlSqh7EH2tEa40e7a9whCThRRnIAHbNuMbzLjluLPcsf7XpCTGDVteFJJPu+GMt8Ot+Tr\\\\\\\/D1ecNNT1SAnViZ9YLK2B0lRtwCVksNhhaNxFV3+twoDRNBAb6TgMkPSOsnmjHa8uzdVUzukbCWD9Kqm+pyY8RqLrlMaJT8WmBuCxiEQcdhfWHfl4fKNwSBVjnkvTN8CrdiShbNICit+nBxj+8tZHY6tab9ayjHXWvIs6L9LZAPPV84ivK\\\\\\\/GqwkJxZe54w\\\\\\\/HoPJjGh0nMys9ucyvFE5FhGuz7p4UTcZXWJrsN+wJhtoD6mJzzWwgugAnX7h7CoSLcyHlpZvEqMBP88mCcRUq8hmlQVxP3uqS8gAIDx9YA6o64yB32sUlzkb3rjKpDayGQAl0O6G+h79U4WwSi9Jho425VfvafqH+r5t805DGwW4mZQ1oKBuP\\\\\\\/FOwWxYBC0xyoFTLt8YnCz7q453Fhqajs6vzKa1zUIhoGVSpkdonIE6MIPXqFUz6xcmc3u+AAq3fypeHbq9TaTGX6flRWjRd+dJU2Yu4sE8I45qYLXuagMmqexyGkds2F8tafPfIMTiAaVJD7tGlmw0hh0lAO6f+u19yQB7Hmd2WnpzUUD2aLinEXm3RAVvXXe8lJiCO9uep3vdYk5EYHRkbE2UD70xyIRS4SNWVvteU+dlnsvJdfADDZjpWCPbnspl09MvZQYm5RivehupQDnbYUDkDWvUT\\\\\\\/YI4Md4i+cr1ui1zuWR1Iht5tE0hLHoZWBKIAlyPoGCTe3H\\\\\\\/Q7\\\\\\\/EqEL8EPPKOzGsMKwiwB7PRI1nCp0eUDEr4La8GH9OZ6enY5gjO5NMCNpU2Wf+z3ppKa3ol05tm7a\\\\\\\/\\\\\\\/TEhUuQ1BehZsFUxu3VhZlJDsonNK2uha79EOJVYQxtJ0Hs5XYemnY67b46M2DRRVXF2XiUG\\\\\\\/lnbK\\\\\\\/DxnfmL6rit72jQdHJ7RqX2h74g1GlFFDni3CKFvFgHmCYsJJyiL1aANbQthoDHhLSDbfi3CoNO1NX7okuSqChKe\\\\\\\/Y7hDk08\\\\\\\/+ebnDepYC+ODm+jH15N2n9vAJBTgropmQRQwBIdL0vsUtacCj2Eb1cImwwJXV4JNX81XzBd48zdiqBFvD55LWVYJIPfiamWXvmeRRA+uIOmXLvmruWRN5kROgL5skHVfXBocQghzQI2XWD8S0B+JFRRs0W+L\\\\\\\/ZzqJWyF8EEww9ZCaHYB\\\\\\\/GfJHJ+om+cleIqo2g52JcW66wdV6kUyHBd1HenX+07+W\\\\\\\/jRMNIfMSlUuTJDn04KlpkL4Sb9OD2rAVFdoqKN2DseciDlEbVhL+2f3TI6wluFJHHXHRIJor93avAT57epjHujbf\\\\\\\/wBMCM81293tYRcDNoOwFiXE2AldVBwxSJsOPFftqbJmBfuKlqfjEbv+QyaaLPBwNvt5N7zu1yT5m3caarDUwbwI74CqgPvfYJmUi8mRZrZLw9dHYQDkj9Q5joPaGYg62M5j0Yd7FYJ\\\\\\\/CDwRLH0WcWPw4k0CxPnBizhYC8g8lHsEeWYiY95YCTyJK78u\\\\\\\/jf0hU9sQ9yX2u6jQcMd5aPVkecofXG7BcwL5GPTdlwBCGP3iYb8kyv0ILaoMffoXg9Mk4iQnKbCGl2sCPc0GdjtqyBGrpx68J3rY0BhYLqxaqJEcPlUKYNhIt1DJr+5yBmAzcY4+GHaN+ytiIaaPTWRK9ToefMhuX31FnzaGkHnwNmY+nKDRI5rYSeOTngLege5tBlspA6SCNmGO6W1w8n+UHPusvUdTmASty1cAjGuFYoZw4tD5GaFpikwy5Slgwgx\\\\\\\/lleUdU\\\\\\\/zvU5lIEzXgM\\\\\\\/h9jqKtmxH40xkRdnUU6gCy8WImNio4KLGnxhM7Uqpt2WDQn0mnBvYVBM3F6gzKRubFJZoBhDl6o\\\\\\\/gc8eu0jLFIYD+QPdp1i0s5aqNTW7gFPZ1P3\\\\\\\/UrKlQy+UrzL+FyOnu1roiDTM52sNLv6lt4uztyKwAq1FKa3AOOwXrMtO8k\\\\\\\/fVq2ZE2XsfXurflx9PMCUEKdwsaCu0phb7OTIE1pb8AABwDIT8ii5FEj1EGSXBEI9+QCrr1cuvzl9FOneMHrm3aaZm3aQVOwuFzz2ntd4z1ZY9\\\\\\\/5\\\\\\\/Xu4HcgJKGqJMyFrlsijDt9yG\\\\\\\/ouoOaYUbwGZAhyMbSgnsfhkux4cHDpwEZl\\\\\\\/iuDKIUy0ljgxYhcewZt6YoRZY9khcm2g9tTbNUDZAkwWauusK8OEU8Q4LOX9tyWTMEpkEb\\\\\\\/+fJdxiAreUxONUc6gIZxzXpP1tTpXWRvyhqJmFY3hfhZ5gcMkp5tSCOxS8pduQzU9G229LJ7Ab\\\\\\\/q1So5oWyru9GbvVvYX82vY\\\\\\\/\\\\\\\/AKv1DgSzYwwrNEYSUNAW3ECdsOmxYtDrpzEneXSR+n6CTDtWj8G8uj25RfCioaTbmispPe6OJCwDh1bb+F+I24cAKyEikFgJMO1T2uWBtBMe2O\\\\\\\/HaC+ASUZSrYBEZQQXEnwT\\\\\\\/NNijKlmHFFLFggzz5z3uyh3ul8T6aDFIkvz\\\\\\\/FHf\\\\\\\/VnwLItq0w3ywzdodlL7CYwHBh0t9mf6VwSy3wifNgQonq3xPTuR3XBlAqdw50R096\\\\\\\/GxHAKuIFI7TSP\\\u0022,\\\u0022iv\\\u0022:\\\u0022sDXzyYtR\\\\\\\/8NLCaJLfeZ6pg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222895a15799260c27b231f8c2c4ade74f41e052dcbd9102371f2918ade2603f8d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ySIZvsDPmHfI65Tn3mK3NDYC3B5HvoWUTbgdI4gSuFyDMxi1m1xvDBbYVN26\\\\\\\/sIL+4cUwnf0BBUgN8\\\\\\\/wlxl3y9F93w5nTcGwe8gkUZjdQpkxpTVW1U871cZXZYSmNRoi9NjSPI\\\\\\\/hhn282PZFcdv7H8fgiV5sPt2DTAWUskm7GNu17JeLGkvq\\\\\\\/PWh2PXd1ScNbj4ay80aN+W+7Ia6AvOyOQbHqbSJTf3EENInqxkEvX0YsBhM2vfROgx9sdvK7EQzty8Y11IBhEEYsTo\\\\\\\/zkk4csNrWhmk9QIQs2guzwlwnSMJDsiSZM\\\\\\\/ZiZSpQx6TQDAjxzvSJVcF5y0nD8WJz9ujfCUXurkfOeW1aD46kve2pR0GcypCRWKZruwEpTrGL5e+WPDmDGa0lQNlJu\\\\\\\/bGbceAb20yRIKig\\\\\\\/ur0GGk1HdHlvDxis79Xdqcvbho61NFdyQuwqPyWFpbBzfLPalkeRolZpFseWhI26YTMZ9EG81LJkQw61ciSt1yyyVBujH9Sx1uz6q8xiwhemt\\\\\\\/jGbWdPk14LyLO80WDnSCxwi4EPPm0Yj8LDyxFtO0MY8ZxMzLi+J5fsUbUDAaNQAxJlFQN7wxA0ejcG3ObEEs3FzmfbJpO++8VmhI0+AbIF+4CkY\\\\\\\/h3W+h3uwOipLA7eLu\\\\\\\/4ADCckN5XOB9LS+t33NaF81dcgdNgWIhXDwpFahtHssuT2YbHyQoF6zkIIOhO1l8JXpgmFpkAgz8\\\\\\\/g8pKfQRF6pkn9vL4t03zcKX0Bk1auPsCq+umAgKujlFHMf3NDfnUgGF0QgtF+nDw7yFQa6bw1+i8KBcZk9rdNMKaBtUKvfluX0peShREtNmUjIeyO+CUM9BFtZnACrGE0pi2t\\\\\\\/bVhs4muFgjvdmANHkqpgWwGqTr4B0+amUPikQ5yld2HY59bUqBY7NCLaaoTvb1aoRutf+Kkxuah\\\\\\\/XC\\\\\\\/DQtRGsweV0B\\\\\\\/+nKCG3Pt9l4T2HBBTiw5M\\\\\\\/SX1bqBQlWX9jwOo0Se+HO2gsm2cW5gBAYwIQXHHUnXFCi6xeDyuLePgQwk4WcT2gdmD\\\\\\\/1r5+\\\\\\\/NRXsBFwUpxF94gfLiyb9VfgbXoPwo5FgnbfBMNAdor7124++w6iogvZO8gt\\\\\\\/DqLRdfn\\\\\\\/lpg\\\\\\\/LUBOi5RDtMWuYyvIBwnxPrWX047OU1NU8uj9UwG7ofKreUTPrYfvyVzu0IiSx4NvJEPL7EY7C1FTli3kqe9ck\\\\\\\/gd50pwTeH++Z5c\\\\\\\/K3IFrm2RZRTZfKoho7G0hDxYQ2eZXMTo4n+b\\\\\\\/xOQUEX4TQ7ZTf\\\\\\\/CSYSc1FXngJCIqoGaSMSrqgPRkvHduwHGBKsIQa1Y6S\\\\\\\/qMw6scnTKwXFDecBuiHnqBioWYEkmghogQEB23898oOBZgxN37+zpwYfzUXOhmANqkhTyK8IGISnAir43HMzRKFGr\\\\\\\/ra\\\\\\\/J6Z6gZCxgs8c313lEjfVqv7Gep8G0FqVsJEB9x2PuauM143SlZ1icHj1siUaDHDGwaeiGF7d8to\\\\\\\/JuAmo9DewMgXL6LsQnxW82vEDPfr3sysQ0am8hdVyyDHO8yi20Y0KSMaDVQD7SlknVLoJb9Ez4zbA3kYavj2xul3LW56W3A2Z1NLtYNdxWxWLYOn6Z1ScVKchorK1jj6NeEr1aBgUcmuhBXya1vrt24ie2rqJMUZTh3ctR2t1Y5egnSpb2ZbC55TiqrK0YtaH9jPWZ6zPhGwa22moJGyar8T7YOonmDbZvSAiLd9rRF+dmSHlanDvifx2DJeA912zJeKebtroTWgZr3bhObq6Nl7CfdPDTvDYcssFmzI7OoNpAtUCm25piIH9aIUmVn7uaETf\\\\\\\/5HWNI+iscNJeAlvET6EHhgBFbAoWAULA12S7U7++hcx\\\\\\\/Wkg6\\\\\\\/1lZKhciPTnMpO0XTOZu8g04XKivVRNMl8QFICb95CBCeZRTFVfZiBDcHKGwUmsSj5tdAV2p42fazIvcFSyNNJNsHjWFW54eeH0QuXy64NS1c1v78FuTkRgdvlTdRJZvGejWNRnd32Ym9EytdMzhzqxLtTZyu2AaiXis=\\\u0022,\\\u0022iv\\\u0022:\\\u0022seS8mTmeSARC+atflq8h8w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fb79aafeffee91db468ac761c34c97a20097bb0b4bf5d5106c20e97589b5b94e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459913 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1187" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_sXffV-c8ZdTScp3BdAFIzNJgAJFaPujJ_6m0MMhiUQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1187" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Tf-OvBptyT-yjHTZclQ2EpfVMLVWPxgPLQC52rV8ySk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1177" title="Caenorhabditis elegans TBX-2 Directly Regulates Its Own Expression in a Negative Autoregulatory Loop" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1211" title="The Mosaic Mutants of Cucumber: A Method to Produce Knock-Downs of Mitochondrial Transcripts" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459913" id="node459913" data-pisa="ggg;5/6/1187" data-pisa-master="ggg;g3.115.018416" data-apath="/ggg/5/6/1187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of <em>Gossypium</em> spp.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda M. Hulse-Kemp</span>, <span class="highwire-citation-author" data-delta="1">Jana Lemm</span>, <span class="highwire-citation-author" data-delta="2">Joerg Plieske</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Ashrafi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-5343-9960" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ramesh Buyyarapu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David D. Fang</span>, <span class="highwire-citation-author" data-delta="6">James Frelichowski</span>, <span class="highwire-citation-author" data-delta="7">Marc Giband</span>, <span class="highwire-citation-author" data-delta="8">Steve Hague</span>, <span class="highwire-citation-author" data-delta="9">Lori L. Hinze</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-8124-8283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kelli J. Kochan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-3296-320X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Penny K. Riggs</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-7897-2916" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jodi A. Scheffler</span>, <span class="highwire-citation-author" data-delta="13">Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="14">Mauricio Ulloa</span>, <span class="highwire-citation-author" data-delta="15">Shirley S. Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qian-Hao Zhu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0001-8907-2841" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sumit K. Bag</span>, <span class="highwire-citation-author" data-delta="18">Archana Bhardwaj</span>, <span class="highwire-citation-author" data-delta="19">John J. Burke</span>, <span class="highwire-citation-author" data-delta="20">Robert L. Byers</span>, <span class="highwire-citation-author" data-delta="21">Michel Claverie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="22"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span>, <span class="highwire-citation-author" data-delta="23">David B. Harker</span>, <span class="highwire-citation-author" data-delta="24">Md S. Islam</span>, <span class="highwire-citation-author" data-delta="25">Johnie N. Jenkins</span>, <span class="highwire-citation-author" data-delta="26">Don C. Jones</span>, <span class="highwire-citation-author" data-delta="27">Jean-Marc Lacape</span>, <span class="highwire-citation-author" data-delta="28">Danny J. Llewellyn</span>, <span class="highwire-citation-author" data-delta="29">Richard G. Percy</span>, <span class="highwire-citation-author" data-delta="30">Alan E. Pepper</span>, <span class="highwire-citation-author" data-delta="31">Jesse A. Poland</span>, <span class="highwire-citation-author" data-delta="32">Krishan Mohan Rai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="33"><a href="http://orcid.org/0000-0002-7635-5049" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samir V. Sawant</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="34"><a href="http://orcid.org/0000-0002-4897-8812" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sunil Kumar Singh</span>, <span class="highwire-citation-author" data-delta="35">Andrew Spriggs</span>, <span class="highwire-citation-author" data-delta="36">Jen M. Taylor</span>, <span class="highwire-citation-author" data-delta="37">Fei Wang</span>, <span class="highwire-citation-author" data-delta="38">Scott M. Yourstone</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="39"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiuting Zheng</span>, <span class="highwire-citation-author" data-delta="40">Cindy T. Lawley</span>, <span class="highwire-citation-author" data-delta="41">Martin W. Ganal</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="42"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="43">Iain W. Wilson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="44"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David M. Stelly</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1187-1209;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018416 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459913" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda M. Hulse-Kemp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse-Kemp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse-Kemp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BM.%2BHulse-Kemp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda M. Hulse-Kemp</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jana Lemm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BLemm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemm%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJana%2BLemm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joerg Plieske </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joerg%2BPlieske%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plieske%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoerg%2BPlieske%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Ashrafi</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ramesh Buyyarapu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Dow AgroSciences, Trait Genetics and Technologies, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BBuyyarapu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buyyarapu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARamesh%2BBuyyarapu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5343-9960" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ramesh Buyyarapu</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David D. Fang</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James Frelichowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BFrelichowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frelichowski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BFrelichowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Marc Giband </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div><div class='author-affiliation'><span class='nlm-sup'>***</span>EMBRAPA, Algodão, Nucleo Cerrado, 75.375-000 Santo Antônio de Goias, GO, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BGiband%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giband%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BGiband%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Steve Hague </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BHague%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hague%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BHague%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Lori L. Hinze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lori%2BL.%2BHinze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hinze%20LL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALori%2BL.%2BHinze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kelli J. Kochan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Animal Science, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelli%2BJ.%2BKochan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kochan%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelli%2BJ.%2BKochan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8124-8283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelli J. Kochan</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Penny K. Riggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Animal Science, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penny%2BK.%2BRiggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riggs%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APenny%2BK.%2BRiggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3296-320X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Penny K. Riggs</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jodi A. Scheffler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡</span>USDA-ARS, Jamie Whitten Delta States Research Center, Stoneville, Mississippi 38776</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jodi%2BA.%2BScheffler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scheffler%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJodi%2BA.%2BScheffler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7897-2916" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jodi A. Scheffler</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Mauricio Ulloa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>****</span>USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BUlloa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ulloa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BUlloa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Shirley S. Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shirley%2BS.%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShirley%2BS.%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qian-Hao Zhu</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Sumit K. Bag </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sumit%2BK.%2BBag%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bag%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASumit%2BK.%2BBag%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8907-2841" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sumit K. Bag</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Archana Bhardwaj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Archana%2BBhardwaj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhardwaj%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArchana%2BBhardwaj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">John J. Burke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>****</span>USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BJ.%2BBurke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burke%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BJ.%2BBurke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Robert L. Byers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BByers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byers%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BL.%2BByers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Michel Claverie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michel%2BClaverie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Claverie%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichel%2BClaverie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§§</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-23"><div class="author-tooltip-name">David B. Harker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BHarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harker%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BHarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-24"><div class="author-tooltip-name">Md S. Islam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Md%2BS.%2BIslam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Islam%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMd%2BS.%2BIslam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-25"><div class="author-tooltip-name">Johnie N. Jenkins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*****</span>USDA-ARS, Genetics and Precision Agriculture Research, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johnie%2BN.%2BJenkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jenkins%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohnie%2BN.%2BJenkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-26"><div class="author-tooltip-name">Don C. Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††††</span>Cotton Incorporated, Agricultural Research, Cary, North Carolina 27513</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Don%2BC.%2BJones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jones%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADon%2BC.%2BJones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-27"><div class="author-tooltip-name">Jean-Marc Lacape </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Marc%2BLacape%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lacape%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Marc%2BLacape%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-28"><div class="author-tooltip-name">Danny J. Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BJ.%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BJ.%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-29"><div class="author-tooltip-name">Richard G. Percy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BPercy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Percy%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BPercy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-30"><div class="author-tooltip-name">Alan E. Pepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡‡</span>Department of Biology, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BPepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pepper%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BPepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-31"><div class="author-tooltip-name">Jesse A. Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§§§</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BA.%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BA.%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-32"><div class="author-tooltip-name">Krishan Mohan Rai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krishan%2BMohan%2BRai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohan%20Rai%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrishan%2BMohan%2BRai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-33"><div class="author-tooltip-name">Samir V. Sawant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samir%2BV.%2BSawant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sawant%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamir%2BV.%2BSawant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7635-5049" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samir V. Sawant</a></li></ul></div><div class="author-tooltip-34"><div class="author-tooltip-name">Sunil Kumar Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunil%2BKumar%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASunil%2BKumar%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4897-8812" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sunil Kumar Singh</a></li></ul></div><div class="author-tooltip-35"><div class="author-tooltip-name">Andrew Spriggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BSpriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spriggs%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BSpriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-36"><div class="author-tooltip-name">Jen M. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jen%2BM.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJen%2BM.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-37"><div class="author-tooltip-name">Fei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-38"><div class="author-tooltip-name">Scott M. Yourstone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BM.%2BYourstone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yourstone%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BM.%2BYourstone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-39"><div class="author-tooltip-name">Xiuting Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiuting%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiuting%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiuting Zheng</a></li></ul></div><div class="author-tooltip-40"><div class="author-tooltip-name">Cindy T. Lawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>******</span>Illumina Inc., San Francisco, California 94158</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cindy%2BT.%2BLawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawley%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACindy%2BT.%2BLawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-41"><div class="author-tooltip-name">Martin W. Ganal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BW.%2BGanal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ganal%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BW.%2BGanal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-42"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Allen Van Deynze</a></li></ul></div><div class="author-tooltip-43"><div class="author-tooltip-name">Iain W. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BW.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20IW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIain%2BW.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-44"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David M. Stelly</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stelly@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">stelly@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1187" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459913" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1187.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459913" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1187.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459913" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1187.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459913" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1187.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459913" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459913/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">High-throughput genotyping arrays provide a standardized resource for plant breeding communities that are useful for a breadth of applications including high-density genetic mapping, genome-wide association studies (GWAS), genomic selection (GS), complex trait dissection, and studying patterns of genomic diversity among cultivars and wild accessions. We have developed the CottonSNP63K, an Illumina Infinium array containing assays for 45,104 putative intraspecific single nucleotide polymorphism (SNP) markers for use within the cultivated cotton species <em>Gossypium hirsutum</em> L. and 17,954 putative interspecific SNP markers for use with crosses of other cotton species with <em>G. hirsutum</em>. The SNPs on the array were developed from 13 different discovery sets that represent a diverse range of <em>G. hirsutum</em> germplasm and five other species: <em>G. barbadense</em> L., <em>G. tomentosum</em> Nuttal × Seemann, <em>G. mustelinum</em> Miers × Watt, <em>G. armourianum</em> Kearny, and <em>G. longicalyx</em> J.B. Hutchinson and Lee. The array was validated with 1,156 samples to generate cluster positions to facilitate automated analysis of 38,822 polymorphic markers. Two high-density genetic maps containing a total of 22,829 SNPs were generated for two F<sub>2</sub> mapping populations, one intraspecific and one interspecific, and 3,533 SNP markers were co-occurring in both maps. The produced intraspecific genetic map is the first saturated map that associates into 26 linkage groups corresponding to the number of cotton chromosomes for a cross between two <em>G. hirsutum</em> lines. The linkage maps were shown to have high levels of collinearity to the JGI <em>G. raimondii</em> Ulbrich reference genome sequence. The CottonSNP63K array, cluster file and associated marker sequences constitute a major new resource for the global cotton research community.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-analysis" class="hw-term hw-article-keyword hw-article-keyword-linkage-analysis" rel="nofollow">linkage analysis</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/interspecific-snps" class="hw-term hw-article-keyword hw-article-keyword-interspecific-snps" rel="nofollow">interspecific SNPs</a></li><li class="kwd"><a href="/keyword/intraspecific-snps" class="hw-term hw-article-keyword hw-article-keyword-intraspecific-snps" rel="nofollow">intraspecific SNPs</a></li><li class="kwd"><a href="/keyword/breeding" class="hw-term hw-article-keyword hw-article-keyword-breeding" rel="nofollow">breeding</a></li></ul><p id="p-3">Cotton (<em>Gossypium</em> spp.) is the world’s most important renewable natural textile fiber crop and also a significant source of oilseed. Cotton is grown in more than 75 countries, with more than 118 million bales of cotton fiber produced in 2013 (National Cotton Council, <a href="http://www.cotton.org">www.cotton.org</a>). In the United States, the 2013 crop of 12.9 million bales was valued at $5.2 billion and had an estimated overall direct economic impact of $27.6 billion (US Department of Agriculture; National Agriculture Statistics Service, <a href="http://www.nass.usda.gov">www.nass.usda.gov</a>). Approximately 75% of cotton fiber is used for apparel products, 18% is used for home furnishings, and 7% is used for industrial products (National Cotton Council, <a href="http://www.cotton.org">www.cotton.org</a>). In the past, cottonseed was often regarded as waste, but recently it has become marketable as a protein source for livestock, including dairy cattle and poultry, as well as for production of cottonseed oil, which is used in the food product industry . Although 52 different <em>Gossypium</em> species have been identified, including 7 tetraploids designated AD<sub>#</sub> and 45 diploids classified into 8 genome designations (A–G and K) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wendel <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grover <em>et al.</em> 2014</a>), contemporary cotton production relies primarily on allotetraploid <em>Gossypium hirsutum</em> L. (2<em>n</em> = 4<em>x</em> = 52), which accounts for more than 95% of the world crop. The remaining production relies largely on <em>G. barbadense</em> L., another allotetraploid, and to a lesser extent on two A-genome diploids (2<em>n</em> = 2<em>x</em> = 26), <em>G. arboreum</em> L. and <em>G. herbaceum</em> L., primarily in Asia. Although diploid species of interest such as <em>G. longicalyx</em>, an F-genome diploid, and <em>G. armourianum</em>, a D-genome diploid, cannot be readily crossed with the primary cultivated species <em>G. hirsutum</em>, allotetraploid species such as <em>G. barbadense</em>, <em>G. tomentosum</em>, and <em>G. mustelinum</em> can be readily intercrossed.</p><p id="p-4">Allotetraploid cotton was formed approximately 1–2 million years ago (mya) (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wendel <em>et al.</em> 2009</a>) in a polyploidization event between two diploids: one with an A-like genome and the other with a D-like genome. The ancestral genomes resemble the genomes of extant diploids <em>G. arboreum</em> (A<sub>2</sub>) and <em>G. raimondii</em> (D<sub>5</sub>). The <em>G. hirsutum</em> genome is designated [AD]<sub>1</sub> and has a C-value of approximately 2.4 Gbp; the 26 chromosomes are numbered according to genome ancestry, where chromosomes 1–13 are of A-genome origin and 14–26 are of D-genome origin (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brown 1980</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Wendel <em>et al.</em> 2009</a>). Because the ancestral genomes of cotton diverged only 5–10 mya, and because the polyploidization event was 1–2 mya, initial efforts to develop single nucleotide polymorphism (SNP) markers were hindered by the co-identification of interlocus SNP variants between the two subgenomes in the tetraploid (homeo-SNPs). Recent developments from the cotton community including the publication of a high-quality genome reference sequence for <em>G. raimondii</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Paterson <em>et al.</em> 2012</a>) and draft sequences for <em>G. arboreum</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 2014</a>) and <em>G. raimondii</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2012</a>) have aided development of large SNP data sets in gene-based and genome-based identification efforts (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Van Deynze <em>et al.</em> 2009</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lacape <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rai <em>et al.</em> 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gore <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Islam <em>et al.</em> 2014</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhu <em>et al.</em> 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hulse-Kemp <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hulse-Kemp <em>et al.</em> 2015</a>). Increasingly larger genetic maps with greater resolution and saturation have been generated as numbers of primarily SSR markers increased, culminating in the recent publication of a consensus genetic map that comprises more than 8,200 loci based primarily on six interspecific (<em>G. hirsutum</em> × <em>G. barbadense</em>) populations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blenda <em>et al.</em> 2012</a>). Few of the published genetic maps contain SNP markers, and those that have included SNPs are limited to inclusion of a few hundred SNPs with SSRs (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yu <em>et al.</em> 2012</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gore <em>et al.</em> 2014</a>) to, at most, the inclusion of a few thousand SNPs (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Islam <em>et al.</em> 2014</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zhu <em>et al.</em> 2014</a>). SNPs have been difficult to develop <em>in silico</em> due to low polymorphism and low divergence among polyploid genomes (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Van Deynze <em>et al.</em> 2009</a>). The increasing efficiency of next-generation sequencing and improved <em>in silico</em> methods has allowed SNP development at the whole genome level, even for the 2.4-Gbp allotetraploid cotton genome (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hulse-Kemp <em>et al.</em> 2015</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Zhu <em>et al.</em> 2014</a>).</p><p id="p-5">In cultivated cotton, as well as in other crop species, there is considerable interest in being able to genotype a large number of SNP markers in a high-throughput manner. With advancing array technology, large-scale genotyping can be performed in a massively parallel fashion to assay thousands of loci simultaneously in a short time. Genotype data obtained from the high-throughput assay can then be utilized to evaluate genetic diversity, construct genetic linkage maps, dissect the genetic architecture of important traits (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Truco <em>et al.</em> 2013</a>), and in many more novel applications (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ganal <em>et al.</em> 2012</a>). SNP arrays for many crop species have recently been developed and have been utilized to advance breeding and discoveries in those crop systems (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ganal <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hamilton <em>et al.</em> 2011</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Truco <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Song <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tinker <em>et al.</em> 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bianco <em>et al.</em> 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dalton-Morgan <em>et al.</em> 2014</a>).</p><p id="p-6">The objectives of this study were as follows: to utilize recently identified SNPs to develop a standardized large-scale SNP genotyping array for cotton; to evaluate the performance and reproducibility of the array on a large set of samples; to develop a cluster file that can be used to help automate genotyping for allotetraploid cotton; and to produce high-density linkage maps based on two biparental F<sub>2</sub> populations in an intraspecific cross (<em>G. hirsutum</em> × <em>G. hirsutum</em>) and an interspecific cross (<em>G. barbadense</em> × <em>G. hirsutum</em>). Completion of these objectives will enable reliable standardized high-throughput SNP genotyping on large cotton populations for many diverse applications, including high-density genetic mapping, genome-wide associated studies, genomic selection, complete trait dissection, and studying patterns of genomic diversity among cotton cultivars and wild accessions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-7">Seeds for each line were planted in peat pellets (Jiffy, Canada) at Texas A&amp;M University, CIRAD/EMBRAPA, USDA-ARS-SPARC, USDA-ARS-SRRC, or CSIRO greenhouses. Plants were allowed to grow until first true leaves were available. Young true leaves were sampled and extracted using the Macherey-Nagel Plant Nucleo-spin kit (Pennsylvania) according to the manufacturer’s instructions. All DNA samples were quantified using PicoGreen and then diluted to 50 ng/µl. The samples included reference lines from the various SNP development efforts, duplicated DNA samples, individual plants and plant pools from the same seed source, individual plant samples from different seed sources, parent/F<sub>1</sub> combinations, segregating samples from wild <em>Gossypium</em> species, inbred cultivar lines, wild <em>G. hirsutum</em> lines, and two mapping populations, one intraspecific and one interspecific (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This material represented samples from most of the cross-compatible range of <em>G. hirsutum</em>. Mapping samples included 93 F<sub>2</sub> lines from a <em>G. hirsutum</em> cv. Phytogen 72 × <em>G. hirsutum</em> cv. Stoneville 474 population designated as “PS” for intraspecific mapping and 118 F<sub>2</sub> lines from a <em>G. barbadense</em> doubled haploid line 3-79 × <em>G. hirsutum</em> cv. Texas Marker-1 population designated as “T3” for interspecific mapping. Samples for the mapping populations were chosen randomly from germinated seed within each population.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466180/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466180/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Samples included for array validation and cluster file development</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Array design</h3><p id="p-10">SNP data sets were obtained for nine intraspecific (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) and four interspecific SNP development efforts (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Classification of SNPs into intraspecific or interspecific content on the array was based on where the SNP was identified in our data sets (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The datasets obtained included SNPs from the following: restriction enzyme double-digest procedure in <em>G. hirsutum</em> between a cultivated and wild accession (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Byers <em>et al.</em> 2012</a>); long-read 454 sequencing of restriction-based genic enrichment libraries of six <em>G. hirsutum</em> lines (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Rai <em>et al.</em> 2013</a>); genotyping-by-sequencing (GBS) mapping of a cross between two <em>G. hirsutum</em> lines (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gore <em>et al.</em> 2014</a>); transcriptome sequencing of five <em>G. hirsutum</em> lines (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ashrafi <em>et al.</em> 2015</a>); GBS mapping of a multi-parent population (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Islam <em>et al.</em> 2014</a>); transcriptome sequencing and restriction-based analysis of 18 <em>G. hirsutum</em> lines (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Zhu <em>et al.</em> 2014</a>); re-sequencing of 12 <em>G. hirsutum</em> lines (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Hulse-Kemp <em>et al.</em> 2015</a>) (H. Ashrafi <em>et al</em>., personal communication); unclassified markers mapped to known chromosomes (DOW AgroSciences, unpublished data); single-copy sequence-derived SNPs between <em>G. hirsutum</em> and <em>G. barbadense</em> (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Van Deynze <em>et al.</em> 2009</a>); differential expression analysis using RNA-sequencing of <em>G. hirsutum</em> vs. <em>G. barbadense</em> (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lacape <em>et al.</em> 2012</a>); transcriptome sequencing of <em>G. barbadense</em>, <em>G. tomentosum</em>, <em>G. mustelinum</em>, <em>G. armourianum</em>, and <em>G. longicalyx</em> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hulse-Kemp <em>et al.</em> 2014</a>); and re-sequencing of <em>G. barbadense</em> (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Hulse-Kemp <em>et al.</em> 2015</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466254/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466254/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Datasets utilized in intraspecific content design on the CottonSNP63K array</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466190/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466190/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Datasets utilized in inter-specific content design the CottonSNP63K array</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">The Illumina Infinium technology utilizes a bead-based approach in which 50 bp oligonucleotide probes are used to hybridize to SNP-adjacent sequences of a sample and then a single base pair extension is performed to assay the SNP base with fluorescently labeled nucleotides. The technology utilizes a two-fluorophore system, necessitating the use of two beadtypes to discriminate only those SNPs that target alleles sharing the same fluorophore (transversions) and only one beadtype to capture all other SNP types (transitions). Based on the Infinium technology, putative markers were filtered to maximize value from this technology, <em>i.e.</em>, to select for unique one-beadtype assays to populate the array. Putative SNPs were retained based on requirements of design score &gt;0.8 (Illumina, Inc), unique identity &gt;99% over 100 bp of flanking sequence, &gt;99% unique identity for designed probe sequence, with final SNPs retained based on precedent publication order in the hierarchy listed above. Subsequent filtering steps included prioritization for the following: one beadtype (Infinium II) assays; validated markers; experimental screening success rates; and representation of genic and nongenic SNPs to optimally cover the cotton genome to obtain a final set of 70,000 putative SNP markers for inclusion on the array (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018416.DC1/018416SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018416.DC1/TableS1.xlsx">Table S1</a>). All genic-based data sets were filtered during dataset development to eliminate SNPs near predicted intron boundaries (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Hulse-Kemp <em>et al.</em> 2014</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Ashrafi <em>et al.</em> 2015</a>; <a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Zhu <em>et al.</em> 2014</a>) so SNPs and flanking sequences could be aligned directly with genomic-based markers to filter putative SNPs as indicated above.</p><p id="p-14">Due to limited prior validation, 500 markers were randomly selected for inclusion from the UC-Davis/TAMU Genomic Set 2 and 386 markers were selected from the CSIR-NBRI data set, including 252 that were chosen randomly and 134 markers that were selected for inclusion based on overlap as identified with BLAST to have 100% identity with markers from other data sets over the entire length of SNP and flanking sequences. For USDA Set 2, markers were primarily included for contigs that had low missing data and were detected across multiple lines with minor allele frequency greater than 0.1. The markers chosen from the CIRAD set primarily represent markers from the most highly differentially expressed transcripts identified between <em>G. hirsutum</em> and <em>G. barbadense</em>. From the TAMU/UC-Davis Inter RNA-Seq data set, as many SNPs as possible that are common between the five species represented and <em>G. hirsutum</em> were selected for inclusion, as well as sets of private SNPs unique to each of the five species (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP markers shared across five species included on the CottonSNP63K array from TAMU/UC-Davis Inter RNA-seq discovery set (Hulse-Kemp et al. 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNP markers shared across five species included on the CottonSNP63K array from TAMU/UC-Davis Inter RNA-seq discovery set (Hulse-Kemp &lt;em&gt;et al.&lt;/em&gt; 2014).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F1.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1187/F1.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466219" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">SNP markers shared across five species included on the CottonSNP63K array from TAMU/UC-Davis Inter RNA-seq discovery set (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Hulse-Kemp <em>et al.</em> 2014</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Genotyping with the array</h3><p id="p-16">Standardized DNA at 50 ng/µl for each of the cotton lines described above was processed according to Illumina protocols and hybridized to the CottonSNP63K array at Texas A&amp;M University or CSIRO. Single-base extension was performed and the chips were scanned using the Illumina iScan. Image files were saved for cluster file analysis. All image files were uploaded into a single GenomeStudio project containing 1156 individual samples. Of the 70,000 SNPs targeted for manufacture on the array, 6942 markers failed to meet standards for bead representation and decoding metrics during the array construction process at Illumina and were removed from the manifest. Data for the remaining markers were clustered using the GenomeStudio Genotyping Module (V 1.9.4, Illumina, Inc.). All markers were viewed and manually curated, taking into account the sample type and known segregation ratios, for construction of the best cluster file for genotyping tetraploid cotton (available at <a href="http://www.cottongen.org/node/add/cotton-cluster-file-request">http://www.cottongen.org/node/add/cotton-cluster-file-request</a>).</p></div><div id="sec-5" class="subsection"><h3>Reproducibility and call rate in a diverse set of cotton samples</h3><p id="p-17">Three technical replicates (running the same DNA on different genotyping runs) were processed for an individual DNA sample of <em>G</em><em>. hirsutum</em> cultivar TM-1, <em>G. barbadense</em> inbred line 3-79 and for the F<sub>1</sub> individual from a <em>G. hirsutum</em> (TM-1) by <em>G. barbadense</em> (3-79) cross. Different individual plants from the same line (cultivar or accession) were analyzed for 11 lines where the individual plants were obtained from the same seed source and for 11 lines where the individual plants were obtained from different seed sources. Genotypes from these lines were compared to determine similarity across technical replicates, individuals from different seed sources, and individuals from the same seed source. Multiple individuals (12) were pooled into the same DNA sample and then compared to the genotype from a single individual to determine variability within a seed source as well as percent residual heterozygosity. Polymorphic SNPs were classified based on Illumina GenTrain score (proximity of clusters) and call frequencies across samples. Minor allele frequencies of polymorphic markers were determined using only inbred line samples.</p></div><div id="sec-6" class="subsection"><h3>Genetic linkage analysis: intraspecific and interspecific</h3><p id="p-18">Genotyping data were transformed into mapping data format (“ABH”) for the 93 intraspecific PS F<sub>2</sub> samples and the 118 interspecific T3 F<sub>2</sub> samples. Only markers that had opposite homozygous allele calls between parental samples and behaved co-dominantly were retained. Subsequently, the data files were initially mapped with JoinMap 4.0 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Van Ooijen 2006</a>) with respect to verification of segregation patterns, the formation of linkage groups, and the preliminary map position of the markers on the chromosomes using the default grouping settings and the maximum likelihood mapping algorithm.</p><p id="p-19">The map data resulting from the initial mapping analysis were manually curated to identify and remove problematic markers that caused elevated numbers of double crossovers and expansions in the length of the linkage group. After removing the problematic markers from the ABH mapping data file in Microsoft Excel, the revised ABH mapping data file was used to construct the final linkage groups in MapManager QTX (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Manly, Cudmore, and Meer 2001</a>) with the following settings: linkage evaluation F<sub>2</sub>, search linkage criterion <em>P</em> = 0.05, map function Kosambi, and line cross cross-type. Markers were eliminated from the final mapping if they caused unexpected expansion of the linkage group because of too many crossovers because this is likely the result of low scoring quality of the individual marker analysis, <em>i.e.</em>, markers with low GenTrain score and/or call frequency. The final maps were drawn with MapChart version 2.2 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Voorrips 2002</a>) with one marker per centimorgan (cM) to allow easier visualization; positions for all markers are listed in <a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018416.DC1/TableS1.xlsx">Table S1</a>. Linkage disequilibrium and visualization of crossovers for each determined linkage group were plotted using the CheckMatrix software (<a href="http://www.atgc.org/XLinkage/">www.atgc.org/XLinkage/</a>) to analyze the mapping population genotypes. The correlation between the interspecific and intraspecific map orders was visualized using MapChart. Discordant linkage groups in the interspecific map identified using CheckMatrix and MapChart were remapped in MapManager using a framework map from the corresponding intraspecific linkage group. The number of recombination events per each F<sub>2</sub> individual was calculated for both the intraspecific and the reordered interspecific linkage maps. Average numbers of recombination bins across both linkage maps were also calculated. Recombination bins were considered as the interval between one recombination breakpoint and the next breakpoint within the mapping populations.</p></div><div id="sec-7" class="subsection"><h3>Synteny analyses</h3><p id="p-20">All SNP marker sequences were aligned to the high-quality JGI <em>G. raimondii</em> (D<sub>5</sub>) reference genome (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Paterson <em>et al.</em> 2012</a>) using Burrows Wheeler Alignment (BWA) software in GALAXY (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Goecks, Nekrutenko, and Taylor 2010</a>) with default parameters. Linkage map positions were plotted against D<sub>5</sub> alignment position for both the interspecific and intraspecific mapped markers. The corresponding allotetraploid chromosomes of the linkage groups were identified using mapped markers (Dow AgroSciences, unpublished data) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yu <em>et al.</em> 2014</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Blenda <em>et al.</em> 2012</a>) and D<sub>5</sub> alignment information. Linkage groups were oriented using the cotton consensus map that incorporated SSR marker chromosome assignment information available (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Blenda <em>et al.</em> 2012</a>). All markers were also aligned to the BGI <em>G. arboreum</em> (A<sub>2</sub>) draft sequence (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 2014</a>) and plotted.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genotyping array content</h3><p id="p-21">An Illumina Infinium genotyping array was developed targeting 70,000 putative SNP markers (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018416.DC1/FileS1.pdf">File S1</a>). The 70,000 putative markers represent 50,000 SNPs that have been identified for use in intraspecific crosses between <em>G. hirsutum</em> lines and 20,000 SNPs for use in interspecific crosses between <em>G. hirsutum</em> and other <em>Gossypium</em> species, such as <em>G. barbadense</em>, <em>G. tomentosum</em>, <em>G. mustelinum</em>, <em>G. armourianum</em>, and <em>G. longicalyx</em>. The intraspecific set was developed from an initial set of 1,658,397 putative SNP markers; however, the largest data set (UC-Davis/TAMU Intra Genomic Set 2) had not been experimentally validated previously and another large data set (CSIR-NBRI) had limited validation, so only small random samples of SNPs for these sets were included. Therefore, selection of the intraspecific markers for the array was based primarily on 91,953 markers. All markers were submitted through the Illumina Design Tool to determine assay design scores for each marker. The data set was filtered to retain only Infinium II, or one-bead type assays that assay one SNP per bead, and those with design scores greater than 0.8. This resulted in 69,306 SNPs or 75.4% retention. Results of probe design for each of the data sets are shown in <a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018416.DC1/TableS2.xlsx">Table S2</a>. Duplicated SNPs were eliminated, as noted in <em>Materials and Methods</em>. All available SNP markers (18,348) within genes (genic) and a randomly selected set of genomic sequence-derived markers (31,396) were included, for a total of 50,000 intraspecific markers. The resulting intraspecific set comprises 36.7% genic SNPs, 62.8% nongenic SNPs, and 0.5% unclassified SNPs.</p><p id="p-22">The same selection and filtering method was used for the interspecific data set. The SNP selection process started with 314,894 potential markers. After the initial step of filtering based on assay design score and one-bead type markers, there were 234,370 markers. As many previously mapped markers as possible were included. When feasible, a single marker was selected per reference sequence or gene sequence. Additional markers were randomly selected from the filtered data set to complete the final set of 20,000 interspecific assays. A total of 14,689 markers in genes were included and 5311 genomic-derived markers were included. The interspecific final set comprises 73.5% genic SNPs and 26.5% nongenic markers. Within the interspecific set, the TAMU/UC-Davis Inter RNA-seq markers were selected to contain the maximum number of markers with polymorphism across multiple species (relative to <em>G. hirsutum</em>) as possible and approximately 2000 markers unique to each of the wild species. This allowed for a maximum number of markers that can be used for introgression breeding efforts while occupying as few beads as possible for cost purposes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Approximately 18% of the interspecific set is composed of markers chosen to support work for multiple species of germplasm introgression into <em>G. hirsutum</em>.</p><p id="p-23">Together, the selected intraspecific and interspecific sets resulted in 70,000 SNP markers submitted to manufacturing for inclusion on the array. All of these SNPs have been deposited in the CottonGen database (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Yu <em>et al.</em> 2014</a>; <a href="http://www.cottongen.org">www.cottongen.org</a>). Based on the characteristics of the utilized Illumina array format, a total of 90,000 beads can be assayed per sample; therefore, it is possible to add up to 20,000 assays, which can be included as private add-on content to allow for adaptability of the array.</p></div><div id="sec-10" class="subsection"><h3>Automated genotype calling through cluster definition</h3><p id="p-24">From the 70,000 markers sent into production, 63,058 markers passed Illumina manufacturing and quality control and were analyzed on 1156 samples for amenability to automated genotyping. Detailed analyses of the diversity and population structure represented by the germplasm used in this project are underway and will be reported separately. The proportion of samples amenable to genotyping, or “call frequency,” across all markers on the chip could be grouped into four distinct types (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A–D</a>) based on all 1156 samples that were included for cluster file development; the distribution of markers within the four types were examined (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2E</a>). Importantly, the polymorphic markers have an average call frequency of 0.99. The first type of classification represents failed markers that did not amplify in the majority of samples (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). The second type consists of markers that had many samples with uncalled genotypes and therefore had a call frequency between 0.50 and 0.99 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>), which may be caused by the presence of an additional null allele. Type 3 includes markers in which only a few samples remain uncalled (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). The last type is those markers in which all samples are called and are highly reproducible (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). Call frequency distribution of all synthesized SNP markers on the chip are shown in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2E</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Types of call frequency of SNP markers. NormTheta or relative amount of each of the two fluorophore signals is plotted on the X-axis, whereas NormR or signal intensity is plotted on the Y-axis. (A) Failed marker with call frequency = 0. (B) Call frequency 0.500&#x2013;0.990 with major sample deviations. (C) Call frequency 0.990&#x2013;0.999 with few uncalled samples. (D) Call frequency = 1 with all called samples. (E) Distribution of call frequencies for all SNP markers on the array." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Types of call frequency of SNP markers. NormTheta or relative amount of each of the two fluorophore signals is plotted on the X-axis, whereas NormR or signal intensity is plotted on the Y-axis. (A) Failed marker with call frequency = 0. (B) Call frequency 0.500&#x2013;0.990 with major sample deviations. (C) Call frequency 0.990&#x2013;0.999 with few uncalled samples. (D) Call frequency = 1 with all called samples. (E) Distribution of call frequencies for all SNP markers on the array.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F2.medium.gif" width="307" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1187/F2.medium.gif" width="307" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466307" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Types of call frequency of SNP markers. NormTheta or relative amount of each of the two fluorophore signals is plotted on the X-axis, whereas NormR or signal intensity is plotted on the Y-axis. (A) Failed marker with call frequency = 0. (B) Call frequency 0.500–0.990 with major sample deviations. (C) Call frequency 0.990–0.999 with few uncalled samples. (D) Call frequency = 1 with all called samples. (E) Distribution of call frequencies for all SNP markers on the array.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Successful markers primarily produced six distinct clustering patterns (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A–F</a>). In the first pattern type, essentially all samples fall in a single cluster. This type represents probe sequences that detect a monomorphic locus or loci (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). The second pattern type represents markers that are detecting two monomorphic loci, which are each homozygous for a different allele (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Markers in this category appear to be heterozygous in all lines and correspond to intergenomic SNPs or “homeo-SNPs,” which are differences between the two subgenomes of cotton and are often identified as false positives in SNP discovery. Polymorphic markers are attributed to the remaining four patterns; they can be classified according to their GenTrain score. Pattern type 3 are markers that showed three clearly definable clusters and behaved as a traditional co-dominant, diploid-like marker with three possible genotypes (AA, AB, BB) and homozygous genotype clusters located near 0 and 1 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). These types of markers did not require any significant manual adjustment of marker positions. Similarly, the fourth pattern type (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>) showed three clearly identifiable clusters, but the clusters were shifted toward one side of the plot with one homozygous cluster at 0.5 on the X-axis. Pattern 4 markers represent markers that detect two loci, most likely from the homeologous chromosomes: one polymorphic and one monomorphic resulting in three possible genotypes (AAAA, AAAB, AABB). Pattern 5 markers represent a pattern in which the three clusters are quite close together and likely represent three or more loci with one polymorphic locus and multiple monomorphic loci in the background (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3E</a>). Finally, pattern 6 markers represent markers that show extremely close clusters due to assaying a large number of loci (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3F</a>). Marker patterns 4 and 5 typically required manual adjustment of locus positions, whereas pattern 6 markers were frequently set as failed.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Classification of scorable SNP markers according to Illumina GenTrain score. NormTheta or relative amount of each of the two fluorophore signals is plotted on the X-axis, whereas NormR or signal intensity is plotted on the Y-axis. (A) Monomorphic marker. (B) Intergenomic or homeo-SNP marker. (C&#x2013;F) Classification of polymorphic markers based on Illumina GenTrain score. (C) Genome-specific marker representing a single polymorphic locus with GenTrain score &gt;0.60. (D) Marker with GenTrain score 0.30&#x2013;0.59 on half the plot representing two genomes, one monomorphic and one polymorphic locus. (E) Marker with GenTrain score 0.21&#x2013;0.29 representing multiple monomorphic loci and one polymorphic locus. (F) Marker with GenTrain score less than 0.20 representing many monomorphic loci and one polymorphic locus. (G) Distribution of cluster types in polymorphic markers based on GenTrain score." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Classification of scorable SNP markers according to Illumina GenTrain score. NormTheta or relative amount of each of the two fluorophore signals is plotted on the X-axis, whereas NormR or signal intensity is plotted on the Y-axis. (A) Monomorphic marker. (B) Intergenomic or homeo-SNP marker. (C&#x2013;F) Classification of polymorphic markers based on Illumina GenTrain score. (C) Genome-specific marker representing a single polymorphic locus with GenTrain score &gt;0.60. (D) Marker with GenTrain score 0.30&#x2013;0.59 on half the plot representing two genomes, one monomorphic and one polymorphic locus. (E) Marker with GenTrain score 0.21&#x2013;0.29 representing multiple monomorphic loci and one polymorphic locus. (F) Marker with GenTrain score less than 0.20 representing many monomorphic loci and one polymorphic locus. (G) Distribution of cluster types in polymorphic markers based on GenTrain score.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F3.medium.gif" width="247" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1187/F3.medium.gif" width="247" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466213" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Classification of scorable SNP markers according to Illumina GenTrain score. NormTheta or relative amount of each of the two fluorophore signals is plotted on the X-axis, whereas NormR or signal intensity is plotted on the Y-axis. (A) Monomorphic marker. (B) Intergenomic or homeo-SNP marker. (C–F) Classification of polymorphic markers based on Illumina GenTrain score. (C) Genome-specific marker representing a single polymorphic locus with GenTrain score &gt;0.60. (D) Marker with GenTrain score 0.30–0.59 on half the plot representing two genomes, one monomorphic and one polymorphic locus. (E) Marker with GenTrain score 0.21–0.29 representing multiple monomorphic loci and one polymorphic locus. (F) Marker with GenTrain score less than 0.20 representing many monomorphic loci and one polymorphic locus. (G) Distribution of cluster types in polymorphic markers based on GenTrain score.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">After evaluating and manually curating all markers, a total of 55,201 of the markers produced successful assays. The successful assays were further characterized into 38,822 polymorphic markers, 10,314 monomorphic markers, and 6065 intergenomic markers. Thus, the overall success rate of the chip is represented by a minimum of 38,822 polymorphic markers (in the samples assayed) out of the 63,058 markers that were synthesized on the chip (61.6%) that will be useful for genotyping cotton samples. Analysis of minor allele frequency (MAF) across polymorphic markers using only inbred samples showed that 66.8% of the polymorphic markers have MAF above 0.05, 55.8% have MAF above 0.10, and 40.0% have MAF above 0.2. The average MAF among polymorphic markers on the CottonSNP63K array was found to be 0.17. Distribution of minor allele frequencies of polymorphic markers in inbred samples is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of minor allele frequencies of all polymorphic SNPs on the CottonSNP63K array. Minor allele frequencies were determined using only inbred line samples; mapping samples and other noninbred line samples used for cluser file development were excluded from this analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of minor allele frequencies of all polymorphic SNPs on the CottonSNP63K array. Minor allele frequencies were determined using only inbred line samples; mapping samples and other noninbred line samples used for cluser file development were excluded from this analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F4.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1187/F4.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466206" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Distribution of minor allele frequencies of all polymorphic SNPs on the CottonSNP63K array. Minor allele frequencies were determined using only inbred line samples; mapping samples and other noninbred line samples used for cluser file development were excluded from this analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the distribution of the final markers and their success rates across the designed data sets. Although most data sets had a success rate of 50% or more, some had a much lower success rate. Discovery sets that did not have previous validation or had limited validation with PCR-based assays frequently resulted in lower success rates, <em>e.g.</em>, CSIR-NBRI and USDA Set 1. In the CSIR-NBRI data set, low success is likely due to homeo-SNPs because it showed the highest percentage of markers classified as 33.82% intergenomic markers. The other data set that had a low success rate of 8.65% was the USDA Set 1 containing markers previously mapped by GBS (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Gore <em>et al.</em> 2014</a>). Although the CSIR-NBRI data set showed elevated levels of intergenomic markers, the USDA Set 1 was primarily found to have monomorphic markers (77.88%). Because these markers had been previously mapped, and because their map positions correlate with alignment positions on JGI D<sub>5</sub> genome (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Gore <em>et al.</em> 2014</a>), these are likely true markers that are of very low MAF or specific to the individual bi-parental population used in their identification (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gore <em>et al.</em> 2012</a>). Although lines for the parents of the population were included in the study, this population has been shown to contain nonparental derived alleles (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gore <em>et al.</em> 2012</a>). Thus, without lines that represent the nonparental derived alleles, polymorphism would not be able to be identified and this would lead to the monomorphic classification of these markers.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466217/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466217/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Distribution of classified SNPs across the discovery sets and success rates of these SNPs on the CottonSNP63K array</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Reproducibility and call rate in different cotton samples</h3><p id="p-31">Replicates of samples were analyzed to determine reproducibility across genotyping runs as well as reproducibility across seed sources. To do this, four different types of replicates were used: technical replicates running the same DNA on different genotyping runs; individual plants from the same seed source; individual plants from different seed sources; and pooled DNA from multiple plants from the same seed source. The three technical replicates (three samples from each <em>G. hirsutum</em> line TM-1, <em>G. barbadense</em> line 3-79, and their F<sub>1</sub>) showed negligible inconsistencies between runs with average similarity of 99.93% ± 0.0007. Therefore, allele calls are highly reproducible across runs. Individual plants from the same seed source showed a range of similarity from 100% for Coker 315 provided by CSIRO to 73.39% for Lu Mien 14 samples provided by CSIRO. A larger inconsistency was seen with duplicated samples from different seed sources. The amount of inconsistency varied considerably across different lines, from 66.61% for VIR-6615/MCU-5 samples provided by USDA-ARS (College Station) and CSIRO to 99.36% for Stoneville 474 samples provided by USDA-ARS (New Orleans) and CIRAD. When DNA of a pool of individual plants developed from the same seed source was analyzed, the percentage of similarity varied between lines from 79.48% for LuMien 14 to 99.84% for Sicot 189. Rates determined for individual lines and replications of different types are shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466251/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466251/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Percent similarities for technical and biological replicates of lines</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Genetic map construction</h3><p id="p-34">An intraspecific map was generated from 93 F<sub>2</sub> samples from a cross between <em>G. hirsutum</em> lines Phytogen 72 and Stoneville 474. A total of 7171 SNP markers were mapped in 26 linkage groups representing 3499 cM (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). These represent 6938 markers from the <em>G. hirsutum</em> set and 235 markers from the other species sets. An average of 254 markers per linkage group was mapped on A-subgenome chromosomes and 298 markers per linkage group of the D-subgenome. These correspond to an overall average of 55 bins with 4.98 markers/bin per chromosome, 51 bins with 4.59 markers/bin for A-subgenome chromosomes, and 60 bins with 5.40 markers/bin for D-subgenome chromosomes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F2 individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F&lt;sub&gt;2&lt;/sub&gt; individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-5.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-5.medium.gif" width="345" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466174" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F2 individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F&lt;sub&gt;2&lt;/sub&gt; individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-6.medium.gif" width="364" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-6.medium.gif" width="364" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466174" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F2 individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F&lt;sub&gt;2&lt;/sub&gt; individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-7.medium.gif" width="374" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-7.medium.gif" width="374" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466174" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F2 individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F&lt;sub&gt;2&lt;/sub&gt; individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-8.medium.gif" width="324" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-8.medium.gif" width="324" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F5/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466174" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Intraspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 93 F<sub>2</sub> individuals from a Phytogen 72 by Stoneville 474 cross. Only one marker is listed on the right per Kosambi centiMorgan (cM) on the left, even if there were more markers co-segregating. Chromosomes are listed based on AD chromosome number.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">An initial interspecific map was generated using 118 F<sub>2</sub> samples from a single interspecific cross between <em>G. barbadense</em> line 3-79 and <em>G. hirsutum</em> standard line TM-1. A total of 19,198 markers mapped into 26 linkage groups initially representing 4439.6 cM. Markers were largely distributed across the genome with a few moderately sized gaps. Correlation analysis between the initial interspecific map and the intraspecific map showed that the moderately sized gaps were associated with inverted marker orders compared to the intraspecific map in nine linkage groups corresponding to chromosomes: Chr06, Chr13, Chr15, Chr18, Chr20, Chr21, Chr22, Chr23, and Chr26. Linkage disequilibrium and recombination plots of these linkage groups showed incorrect ordering near the gaps (see Chr06 example in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The most likely correct orientation is found in the intraspecific linkage groups, where the gaps are smaller. For example, in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, it is likely that the interspecific map is inverted due to low linkage across the gap but the intraspecific map does not have a corresponding gap and is likely the correct orientation. To correct this issue, the intraspecific map was used as a framework map to reorder the problematic interspecific linkage groups.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inconsistencies between initial de novo interspecific map and the intraspecific map. (A) Initial plots of interspecific map order and correlation with intraspecific map show area of incorrect placement in center of the linkage group. (B) Corrected interspecific linkage group and final plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inconsistencies between initial &lt;em&gt;de novo&lt;/em&gt; interspecific map and the intraspecific map. (A) Initial plots of interspecific map order and correlation with intraspecific map show area of incorrect placement in center of the linkage group. (B) Corrected interspecific linkage group and final plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F6.medium.gif" width="323" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/6/1187/F6.medium.gif" width="323" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466237" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-37" class="first-child">Inconsistencies between initial <em>de novo</em> interspecific map and the intraspecific map. (A) Initial plots of interspecific map order and correlation with intraspecific map show area of incorrect placement in center of the linkage group. (B) Corrected interspecific linkage group and final plots.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The final map includes the reordered linkage groups and contains 19,191 markers mapped to 26 linkage groups that collectively encompass 3854.3 cM (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). This map represents a reduction in size of 585.3 cM (13%) from the initial interspecific map. The mapped SNPs represent 11,452 markers from the <em>G. hirsutum</em> set and 7746 markers from the other species data sets (6785 for <em>G. barbadense</em>; 614 for <em>G. tomentosum</em>; 280 for <em>G. mustelinum</em>; 39 for <em>G. armourianum</em>; and 28 for <em>G. longicalyx</em>). The map contained an average of 162 recombination bins with 4.55 markers/bin per chromosome across the linkage groups, with an average of 166 bins with 4.52 markers per bin and 159 bins with 4.60 markers per bin in the A- and D-subgenome chromosomes, respectively. The average overall number of markers per linkage group was 738, with an average of 718 for A-subgenome chromosomes and 759 for D-subgenome chromosomes.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F2 individuals from a G. barbadense line 3-79 by G. hirsutum genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F&lt;sub&gt;2&lt;/sub&gt; individuals from a &lt;em&gt;G. barbadense&lt;/em&gt; line 3-79 by &lt;em&gt;G. hirsutum&lt;/em&gt; genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-10.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-10.medium.gif" width="343" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F2 individuals from a G. barbadense line 3-79 by G. hirsutum genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F&lt;sub&gt;2&lt;/sub&gt; individuals from a &lt;em&gt;G. barbadense&lt;/em&gt; line 3-79 by &lt;em&gt;G. hirsutum&lt;/em&gt; genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-11.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-11.medium.gif" width="345" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F2 individuals from a G. barbadense line 3-79 by G. hirsutum genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F&lt;sub&gt;2&lt;/sub&gt; individuals from a &lt;em&gt;G. barbadense&lt;/em&gt; line 3-79 by &lt;em&gt;G. hirsutum&lt;/em&gt; genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-12.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-12.medium.gif" width="346" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F2 individuals from a G. barbadense line 3-79 by G. hirsutum genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F&lt;sub&gt;2&lt;/sub&gt; individuals from a &lt;em&gt;G. barbadense&lt;/em&gt; line 3-79 by &lt;em&gt;G. hirsutum&lt;/em&gt; genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-13.medium.gif" width="290" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-13.medium.gif" width="290" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F7/graphic-13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-39" class="first-child">Interspecific linkage map of 26 allotetraploid cotton chromosomes. Map determined using 118 F<sub>2</sub> individuals from a <em>G. barbadense</em> line 3-79 by <em>G. hirsutum</em> genetic standard line Texas Marker -1 cross. One marker listed on the right per Kosambi centiMorgan (cM) on the left. Chromosomes are listed based on AD chromosome number.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">In the intraspecific and interspecific maps, the observed map lengths (cM) for the A-subgenome chromosomes (137.5 cM, 152.8 cM) were on average very similar to but slightly larger than those for the D-subgenome chromosomes (131.7 cM, 143.7 cM), respectively. For cotton chromosomes, accurate estimates and comparisons of genetic size with physical distance, <em>e.g.</em>, cM/Mbp, are not yet available. Accurate cytological estimates for individual allotetraploid chromosome sizes are not available. Although a BAC-by-BAC reference sequence for the D<sub>5</sub> is available, highly repetitive regions are under-represented, including subtelomeric and large peri-centromeric regions. Therefore, a D<sub>5</sub>-based estimate would be limited in scope and would underestimate physical size. However, the genomes of the closely related extant diploids, <em>G. arboreum</em> and <em>G. raimondii</em>, have been estimated to be 1600 and 800 Mb, respectively; sizes of those genomes and chromosomes correspond well cytologically with absolute and relative sizes of chromosomes in the <em>G. hirsutum</em> subgenomes. Using the <em>G. arboreum</em> and <em>G. raimondii</em> estimates as a guide for A- and D-subgenome sizes, 1 cM in the intraspecific map represents an average of 895 kb in A-subgenome chromosomes and 467 kb in D-subgenome chromosomes. In the interspecific map, 1 cM represents an average physical distance of 805 kb in A-subgenome chromosomes and 428 kb in D-subgenome chromosomes. Although sizes and physical distances in A- and D-subgenome chromosomes differ approximately two-fold, the rate of crossing over does not, reflecting that most crossovers occur in nonheterochromatic regions or gene space.</p><p id="p-41">Segregation distortion has been previously reported in cotton populations and, not unexpectedly, a number of markers in both the intraspecific and interspecific maps showed significant distortion from expected F<sub>2</sub> segregation ratios. In the intraspecific map, 612 markers of the 7171 (8.5%) mapped markers showed significant segregation distortion (<em>P</em> &lt; 0.05). Of these distorted markers, 256 (41.8%) show a decrease in amount of heterozygotes. Overall, when one parental allele is favored, generally an excess of the female parental allele (Phytogen 72) was observed at a 3.45-to-1 ratio. In the interspecific map, 3475 markers of the 19,191 (18.1%) mapped markers showed significant segregation distortion (<em>P</em> &lt; 0.05). Unlike the intraspecific markers, when one parental allele is favored, there is not a large bias toward one parent, and a favored ratio of 1.13 is observed for a male parent plant (<em>G. hirsutum</em>) to female parent (<em>G. barbadense</em>) comparison. A skew toward the <em>G. barbadense</em> parent was observed in work by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lacape <em>et al.</em> (2003)</a>. The amounts of distorted markers in these populations are within the range of previous reported studies from 8.5% to more than 50% (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lacape <em>et al.</em> 2003</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Shen <em>et al.</em> 2007</a>).</p></div><div id="sec-13" class="subsection"><h3>Distribution of recombination events</h3><p id="p-42">The high density of markers mapped in this study allowed the identification of almost all of the recombination events present in each F<sub>2</sub> individual. We calculated the number of crossovers per F<sub>2</sub> using all markers in both the intraspecific and interspecific linkage maps. In the intraspecific map, the number of crossovers across all chromosomes of an individual varied from 44 to 97, with an average of 67.5, ignoring two individuals with an abnormally high number of crossovers on a couple of linkage groups. These individuals were ignored; although they showed a normal number of crossovers across most chromosomes, they had much higher than two-times the number of crossovers as all other samples for one or more linkage groups. One individual had 34 crossovers on Chr13 and another individual had 24, 34, and 38 crossovers on Chr05, Chr19, and Chr14, respectively. An average of 2.65 crossovers per linkage group occurred across all individuals. Distribution of crossovers in the intraspecific population across linkage groups is shown in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distribution of the number of crossovers. Numbers of crossovers detected for each F2 individual per chromosome (0 to &gt;8) are displayed chromatically for each linkage group, which are organized by genetic size (longest at top, shortest at bottom). (A) Distribution of crossovers in the intraspecific mapping population. (B) Distribution of crossovers in the interspecific mapping population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency distribution of the number of crossovers. Numbers of crossovers detected for each F&lt;sub&gt;2&lt;/sub&gt; individual per chromosome (0 to &gt;8) are displayed chromatically for each linkage group, which are organized by genetic size (longest at top, shortest at bottom). (A) Distribution of crossovers in the intraspecific mapping population. (B) Distribution of crossovers in the interspecific mapping population.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F8.medium.gif" width="269" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/6/1187/F8.medium.gif" width="269" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466300" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-43" class="first-child">Frequency distribution of the number of crossovers. Numbers of crossovers detected for each F<sub>2</sub> individual per chromosome (0 to &gt;8) are displayed chromatically for each linkage group, which are organized by genetic size (longest at top, shortest at bottom). (A) Distribution of crossovers in the intraspecific mapping population. (B) Distribution of crossovers in the interspecific mapping population.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">In the interspecific map, the number of crossovers per individual ranged from 54 to 97, with an average of 75.8 (ignoring a single individual that had an abnormally high number of crossovers in Chr14). Similar to the intraspecific mapping population, there was an average of 2.92 crossovers per linkage group across individuals. The distribution of crossovers in the interspecific population across linkage groups is shown in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>.</p></div><div id="sec-14" class="subsection"><h3>Analyses of synteny</h3><p id="p-45">All of the 70,000 putative SNP markers that were used for production of the CottonSNP63K array were aligned to the D<sub>5</sub> reference genome using BWA in Galaxy; 59.7% of the markers produced alignments to the reference. Synteny was detected for a total of 4521 and 12,027 mapped SNPs for the intraspecific and interspecific maps, respectively. Dot plots of the linkage maps vs. the D<sub>5</sub> reference genome show high collinearity across linkage groups and chromosomes as expected, except for four A-subgenome chromosomes, which are known to contain translocations relative to D-chromosomes. Translocations involving chromosomes 2, 3, 4, and 5 were identified in both maps and are indicated in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9, A and B</a>.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dot plot of the syntenic positions of SNP markers in the allotetraploid linkage maps vs. the JGI G. raimondii reference genome. The 26 allotetraploid chromosomes are shown on the y-axis and the 13 chromosomes of G. raimondii are shown on the x-axis. Red arrows indicate translocation events relative to G. raimondii. (A) Intraspecific linkage map displaying positions of 4521 mapped SNP in G. hirsutum with alignments to G. raimondii. (B) Interspecific linkage map (G. barbadense line 3-79 by G. hirsutum genetic standard line Texas Marker -1) displaying positions of 12,027 mapped SNP with alignments to G. raimondii." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1782654033" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dot plot of the syntenic positions of SNP markers in the allotetraploid linkage maps &lt;em&gt;vs.&lt;/em&gt; the JGI &lt;em&gt;G. raimondii&lt;/em&gt; reference genome. The 26 allotetraploid chromosomes are shown on the y-axis and the 13 chromosomes of &lt;em&gt;G. raimondii&lt;/em&gt; are shown on the x-axis. Red arrows indicate translocation events relative to &lt;em&gt;G. raimondii&lt;/em&gt;. (A) Intraspecific linkage map displaying positions of 4521 mapped SNP in &lt;em&gt;G. hirsutum&lt;/em&gt; with alignments to &lt;em&gt;G. raimondii&lt;/em&gt;. (B) Interspecific linkage map (&lt;em&gt;G. barbadense&lt;/em&gt; line 3-79 by &lt;em&gt;G. hirsutum&lt;/em&gt; genetic standard line Texas Marker -1) displaying positions of 12,027 mapped SNP with alignments to &lt;em&gt;G. raimondii&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1187/F9.medium.gif" width="440" height="167"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/5/6/1187/F9.medium.gif" width="440" height="167"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1187/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466317" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-46" class="first-child">Dot plot of the syntenic positions of SNP markers in the allotetraploid linkage maps <em>vs.</em> the JGI <em>G. raimondii</em> reference genome. The 26 allotetraploid chromosomes are shown on the y-axis and the 13 chromosomes of <em>G. raimondii</em> are shown on the x-axis. Red arrows indicate translocation events relative to <em>G. raimondii</em>. (A) Intraspecific linkage map displaying positions of 4521 mapped SNP in <em>G. hirsutum</em> with alignments to <em>G. raimondii</em>. (B) Interspecific linkage map (<em>G. barbadense</em> line 3-79 by <em>G. hirsutum</em> genetic standard line Texas Marker -1) displaying positions of 12,027 mapped SNP with alignments to <em>G. raimondii</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">When all markers were aligned to the A<sub>2</sub> draft genome, 61.8% of the markers aligned to the reference. A total of 3863 and 11,344 SNPs that mapped were included in the intraspecific and interspecific maps, respectively. While a larger percentage of the SNPs mapped to A<sub>2</sub> compared to D<sub>5</sub>, expected collinearity was not observed when these A<sub>2</sub> alignment positions were plotted against position in the linkage maps (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018416.DC1/FigureS3.pdf">Figure S3, A and B</a>), as was seen with the high-quality BAC-by-BAC–derived D<sub>5</sub> reference genome. The strong collinearity with the high-quality D<sub>5</sub> genome suggests that our genetic maps are correctly ordered, whereas the low collinearity with the A<sub>2</sub> genome sequence suggests that the genome sequence may be improved using the maps produced in this effort and other genetic maps.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-48">The interspecific and intraspecific genetic maps developed with the CottonSNP63K represent the most saturated maps developed for cotton to date. The produced intraspecific genetic map is the first saturated map for a cross between two <em>G. hirsutum</em> lines. While one other effort has come close to generating a number of linkage groups equal to the 26 cotton chromosomes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhang <em>et al.</em> 2009</a>), to the best of our knowledge the intraspecific map developed here is the first that associates into 26 linkage groups corresponding to the number of cotton chromosomes. The array and maps provide a foundation for fine mapping and genetic dissection of agronomically and economically important traits, and facilitate the identification of causative genes underlying quantitative trait loci. This new tool will also foster map-based cloning and genome assembly efforts and contribute to advancements in marker-assisted selection and genomic selection in breeding programs. Although our initial interspecific map was very close to the estimated tetraploid map size of ∼4500 cM suggested by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> (2004)</a>, it showed sizeable gaps and discordant LD patterns. Once corrected, the size of both maps was well below the estimated value of ∼4500 cM at 3854.3 cM for the interspecific map and 3499 cM for the intraspecific map. A decrease in map size is expected when higher-density maps are generated, due to the improved ability to distinguish scoring errors from double recombinant events with high-density markers (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>). Although the sizes of the new interspecific and intraspecific maps are lower than expected, they both fall well within the range of previous map sizes of published interspecific maps 3380 to 5115 cM (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Yu <em>et al.</em> 2012</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Blenda <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Shi <em>et al.</em> 2014</a>) and within the range of 2061 to 4448 cM for reported intraspecific maps (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhang <em>et al.</em> 2012</a>; <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Gore <em>et al.</em> 2014</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Tang <em>et al.</em> 2014</a>).</p><p id="p-49">The mapping of a large number of markers from different discovery sets utilizing a large variety of different cotton lines, primarily cultivars as shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, along with a moderate percentage of markers showing intermediate to high MAFs suggests high transferability of markers across different sets of cotton germplasm. To have a wide applicability across cotton samples, the array was designed to provide the most comprehensive tool to date for cotton researchers and included SNPs that were discovered from technologies targeting both gene regions and genomic regions. This approach was applied so that the inclusion of markers outside of genes would generate a more even distribution across chromosomes because genic markers are unevenly distributed across chromosomes (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Hulse-Kemp <em>et al.</em> 2014</a>). Because the majority of the gene-based markers were provided by the CSIRO discovery set using the D<sub>5</sub> sequence assembly as a reference (<a id="xref-ref-47-7" class="xref-bibr" href="#ref-47">Zhu <em>et al.</em> 2014</a>), many A-subgenome-specific genes may be under-represented in the gene-based marker set. Fortunately, genomic markers located in LD with a gene of interest can also be of value in genome-wide association studies, and thus help compensate for the subgenomic bias among genic intraspecific markers. As the TAMU/UC-Davis Intra-Genomic Set 1 and TAMU/UC-Davis Inter-Genomic sets are markers derived from BAC-end sequences (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Hulse-Kemp <em>et al.</em> 2015</a>), these markers will provide an avenue for direct map-based cloning and fine-mapping of identified regions of interest through direct integration of the array with BAC-based resources.</p><p id="p-50">The array provides cotton research groups with a standardized set of markers that can be used to localize and replicate important previously identified markers, and it will also allow for investigating different types of sample inconsistencies. The replication analysis performed here showed that while technical DNA replicates show very high consistency, different levels of variability were seen with other replication types. As found in maize (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yan <em>et al.</em> 2009</a>), the array analysis revealed considerable variation within some cotton lines, not only between different seed sources but also within the same seed source. Notably, two different TM-1 samples, typically used as a <em>G. hirsutum</em> genetic standard line (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kohel, Richmond, and Lewis 1970</a>), show only 89.77% similarity. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hinze <em>et al.</em> (2015)</a> found similar inconsistencies using SSRs in the US National Cotton Germplasm Collection where accessions with a common name may or may not have been identical whereas other accessions with no reason for similarity were genetically identical. Such inconsistencies can create difficulties when trying to interpret or compare studies utilizing samples bearing a common name. The CottonSNP63K provides an efficient way to characterize these inconsistencies because it has negligible inconsistencies with reproducibility; thus, the similarity differences discovered between seed sources are due to variation of the sources, not a problem with genotyping on the array. The amount of residual heterozygosity within the same seed source shows that some lines are more heterogeneous than others, and comparisons across studies using heterogeneous lines will be less reproducible. However, residual heterozygosity in lines also will offer direct avenues for fine mapping of genes in primarily isogenic backgrounds (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Truco <em>et al.</em> 2013</a>), as well as provide additional diversity within <em>G. hirsutum</em> that has been noted to have low levels of diversity (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tyagi <em>et al.</em> 2014</a>).</p><p id="p-51">Due to the low diversity among cotton lines, multiple mapping populations will be required to map all of the polymorphic SNPs on the CottonSNP63K array. Analysis of the two mapping populations reported here allowed us to map a total of 22,829 total markers, with 3533 SNPs being placed into both linkage maps. Classification of SNPs into the intraspecific or interspecific groups was based on which of our data sets the SNP was identified in. The SNP classification does not indicate if it will or will not be a useful marker in other germplasm types. For example, 3533 of the SNP loci are dually represented in the intraspecific and the interspecific maps reported here. Utilizing mapped loci will be very straightforward, but utilizing the other markers will be more difficult, at least until they are mapped. Syntenic analyses with available high-quality related genome sequences such as the JGI D<sub>5</sub> genome provide some insight into localization information in tetraploid cotton (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Paterson <em>et al.</em> 2012</a>). Allotetraploid linkage groups determined here showed a linear alignment with the D<sub>5</sub> genome (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9, A and B</a>) and were also able to identify historic reciprocal translocation events between allotetraploid chromosomes 2/3 and 4/5 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Desai <em>et al.</em> 2006</a>). The breakpoints for the translocations can be roughly mapped to homeologous regions with the linkage maps to between 21.7 and 29.5 Mb in D<sub>5</sub> chromosome 3 and between 16.7 and 21 Mb in D<sub>5</sub> chromosome 5 for allotetraploid chromosomes 2/3, between 40.1 and 46.1 Mb in D<sub>5</sub> chromosome 9, and between 11.5 and 23.2 Mb in D<sub>5</sub> chromosome 12 for allotetraploid chromosomes 4/5. It has been suggested that these translocations involved complete arms (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Blenda <em>et al.</em> 2012</a>), which is not ostensibly discordant to the approximate breakpoints found here. However, additional mapping would be required to pinpoint exact locations relative to the D<sub>5</sub> chromosomes because map information near the breakpoints is still ambiguous and overlapping at points.</p><p id="p-52">Like SNP arrays developed for other crops, we found a relatively large number of markers that were difficult to score and required either manual adjustment of markers or eliminating the marker from the analysis completely. Compared to diploids, the success rates for SNPs in arrays are typically lower for polyploids due to the presence of duplicated loci in homeologous, paralagous regions, the low levels of divergence particularly between gene copies in different subgenomes, and also between paralagous regions in the same subgenomes. The 61.6% success rate found for the CottonSNP63K is quite comparable to the success rates of arrays generated for other polyploid crops, such as 61% for oat (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Tinker <em>et al.</em> 2014</a>) and 63% for wheat (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2014</a>). Most of the markers mapped in this study were localized to a single map position, as was also the case for the majority of markers mapped with the wheat 90K array (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2014</a>), indicating that most SNP assays are specific to a single locus and/or assaying segregation of a single locus. Similarly, we also found that success rates among discovery sets included on the array were highly variable (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The differences in success rates for discovery efforts as determined by this effort can provide insights for future SNP development efforts by examining parameters and pipelines used and the outcome of those discovery sets. The information provided by the array will enable future SNP discovery efforts in cotton to focus on increasing the success rate of SNP predictions.</p></div><div class="section conclusions" id="sec-16"><h2 class="">Conclusions</h2><p id="p-53">The CottonSNP63K and the accompanying cluster file provide a standardized high-throughput genotyping tool for the cotton community with limited ascertainment bias over a wide range of cotton germplasm. The high-density genetic maps and their synteny with the <em>G. raimondii</em> reference genome provide a useful resource for analyzing genome-wide variation in allotetraploid cotton. Once the allotetraploid cotton genome sequence is available, these maps will be a resource to validate and refine the genome assemblies, as well as provide a resource to directly integrate physical and genetic resources. The array and maps provide a foundation for the genetic dissection of agronomically and economically important traits and crop improvement through genomics-assisted selection. It will also foster positional cloning and genome assembly efforts. Efforts are underway to also provide a comprehensive diversity analysis of a wide range of cotton material, because the results will further extend utility of the array, component SNPs, and associated genomics resources.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">We thank Alexander Kozik for assistance with the CheckMatrix software, Kranti Konganti for implementing and troubleshooting the CheckMatrix software on the Texas Institute for Genome Sciences and Society HPC cluster, and Jing Yu for incorporation of the SNP data file into the CottonGen database. The authors thank Illumina, Inc. for contributing to expanding the scope of the study to allow for a broader impact across global cotton germplasm. This project was supported by National Science Foundation, Plant Genome Research Program (NSF-PGRP) grant No. IOS1025947, and Cotton Incorporated grants Nos. 03-430, 13-326, 08-386, and 13-694. Mention of trade names or commercial products in this manuscript is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the USDA. The U.S. Department of Agriculture and Texas A&amp;M University System are equal opportunity providers and employers. J.L., J.P., and M.W.G. have competing commercial interests as employees of TraitGenetics, which is a commercial company that performs molecular marker analysis services with this array. C.T.L. is an employee of Illumina, Inc. that manufactures the array. This does not alter the authors’ adherence to all of the G3 policies on sharing data and materials. There are no patents or products in development to declare. All remaining authors declare that they have no competing interests.</p><p id="p-55">Author contributions: A.M.H.K., D.M.S., A.V.D., I.W.W., C.T.L., M.W.G., H.A., J.A.U., and D.C.J. conceived and designed the experiments. D.M.S., A.M.H.K. and C.T.L. managed the inter-lab coordination efforts. A.M.H.K., J.L., J.P., H.A., K.J.K., and P.K.R. performed the experiments. A.M.H.K., J.L., and J.P. analyzed the data. A.M.H.K. wrote the manuscript. A.M.H.K., D.D.F., J.F., M.G., S.H., L.L.H., J.A.S., J.A.U., M.U., S.S.W., Q.H.Z., J.J.B., R.G.P., A.E.P., A.V.D., I.W.W., and D.M.S. provided germplasm samples. A.M.H.K., H.A., R.B., D.D.F., J.A.U., Q.H.Z., S.K.B., A.B., R.L.B., M.C., M.A.G., D.B.H., M.S.I., J.N.J., J.M.L., D.J.L., J.A.P., K.M.R., S.V.S., S.K.S., A.S., J.M.T., F.W., S.M.V., X.Z., A.V.D., I.W.W., and D.M.S. provided discovery SNP set data. All authors read and approved the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-7"><p id="p-56">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018416/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018416/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-57">Cotton sequences utilized for array construction are available at: <a href="http://www.cottongen.org">http://www.cottongen.org</a>.</p></li><li class="fn" id="fn-9"><p id="p-58"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-02-16"><span class="received-label">Received </span>February 16, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-11"><span class="accepted-label">Accepted </span>April 11, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Hulse-Kemp <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A long-read transcriptome assembly of cotton (<em>Gossypium hirsutum</em> L.) and intraspecific single nucleotide polymorphism discovery.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.2" data-doi="10.1371/journal.pone.0110377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargent</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derdak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a 20K single nucleotide polymorphism (SNP) whole genome genotyping array for apple (Malus × domestica Borkh).</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e110377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De110377%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0110377%26rft_id%253Dinfo%253Apmid%252F25303088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0110377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25303088&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.3" data-doi="10.1371/journal.pone.0045739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blenda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garsmeur</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45739</span> <span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0045739</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBlenda%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45739%26rft.epage%253De45739%26rft.atitle%253DA%2Bhigh%2Bdensity%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Bthrough%2Bmolecular%2Bmarker%2Bredundancy%2Bcheck.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045739%26rft_id%253Dinfo%253Apmid%252F23029214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029214&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Identification of the chromosomes of <em>Gossypium hirsutum</em> L. by means of translocations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">266</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBROWN%26rft.auinit1%253DM.%2BS.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D266%26rft.epage%253D274%26rft.atitle%253DIdentification%2Bof%2Bthe%2Bchromosomes%2Bof%2BGossypium%2Bhirsutum%2BL.%2Bby%2Bmeans%2Bof%2Btranslocations%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6ODoiNzEvNC8yNjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.5" data-doi="10.1007/s00122-011-1780-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harker</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and mapping of SNP assays in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122–011–1780–8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DByers%26rft.auinit1%253DR.%2BL.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1214%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSNP%2Bassays%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1780-8%26rft_id%253Dinfo%253Apmid%252F22252442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1780-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252442&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A high-density SNP genotyping array for rice biology and molecular breeding.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">7</span>: 541–543.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.7" data-doi="10.1007/s10142-014-0391-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dalton-Morgan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alamery</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollenaere</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-throughput SNP array in the amphidiploid species <em>Brassica napus</em> shows diversity in resistance genes.</span> <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BIntegr.%2BGenomics%26rft.volume%253D14%26rft.spage%253D643%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-014-0391-2%26rft_id%253Dinfo%253Apmid%252F25147024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-014-0391-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25147024&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">O. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>. <span class="cit-pub-date">2006</span> Chromosome structural changes in diploid and tetraploid A genomes of Gossypium. Genome 49: 336–345.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.9" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (<em>Zea mays</em> L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0028334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.10" data-doi="10.1007/s12038-012-9225-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graner</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wieseke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Large SNP arrays for genotyping in crop plants.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiosciences%26rft.stitle%253DJ%2BBiosci%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D821%26rft.epage%253D828%26rft.atitle%253DLarge%2BSNP%2Barrays%2Bfor%2Bgenotyping%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12038-012-9225-3%26rft_id%253Dinfo%253Apmid%252F23107918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12038-012-9225-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23107918&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.11" data-doi="10.1186/gb-2010-11-8-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goecks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGoecks%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DGalaxy%253A%2Ba%2Bcomprehensive%2Bapproach%2Bfor%2Bsupporting%2Baccessible%252C%2Breproducible%252C%2Band%2Btransparent%2Bcomputational%2Bresearch%2Bin%2Bthe%2Blife%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r86%26rft_id%253Dinfo%253Apmid%252F20738864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20738864&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.12" data-doi="10.3198/jpr2011.06.0334crmp"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, R. G. Percy, J. Zhang, D. D. Fang, and R. G. Cantrell, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Registration of the TM-1/NM24016 cotton recombinant inbred mapping population.</span> <abbr class="cit-jnl-abbrev">J. Plant Registrations</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegistrations%26rft.volume%253D6%26rft.spage%253D124%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2011.06.0334crmp%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2011.06.0334crmp&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">Michael A</span></span></li></ol><cite>, D. D. Fang, J. A. Poland, J. Zhang, R. G. Percy <em>et al</em>., <span class="cit-pub-date">2014</span>  Linkage map construction and quantitative trait locus analysis of agronomic and fiber quality traits in cotton. Plant Genome <span class="cit-vol">7</span>: <span class="cit-fpage">1</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grupp</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jareczek</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular confirmation of species status for the allopolyploid cotton species, <em>Gossypium ekmanianum</em> Wittmack.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D62%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.15" data-doi="10.1186/1471-2164-12-302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitty</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Single nucleotide polymorphism discovery in elite North American potato germplasm.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">302</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471–2164–12–302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHamilton%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D12%26rft.spage%253D302%26rft.epage%253D302%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bin%2Belite%2BNorth%2BAmerican%2Bpotato%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-302%26rft_id%253Dinfo%253Apmid%252F21658273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21658273&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.16" data-doi="10.1007/s00122&amp;ndash;014&amp;ndash;2431&amp;ndash;7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinze</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Molecular characterization of the <em>Gossypium</em> Diversity Reference Set of the US National Cotton Germplasm Collection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">327</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122–014–2431–7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B014%2526ndash%253B2431%2526ndash%253B7%26rft_id%253Dinfo%253Apmid%252F25431191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;014&amp;ndash;2431&amp;ndash;7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25431191&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saski</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> BAC-end sequence-based SNP mining in allotetraploid cotton (<em>Gossypium</em>) utilizing resequencing data, phylogenetic inferences, and perspectives for genetic mapping. G3 (Bethesda) 5 :1–11.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.18" data-doi="10.1186/1471&amp;ndash;2164&amp;ndash;15&amp;ndash;945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoegenauer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and bin mapping of gene-associated interspecific SNPs for cotton (<em>Gossypium hirsutum</em> L.) introgression breeding efforts.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">945</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471–2164–15–945</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D945%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471%2526ndash%253B2164%2526ndash%253B15%2526ndash%253B945%26rft_id%253Dinfo%253Apmid%252F25359292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471&amp;ndash;2164&amp;ndash;15&amp;ndash;945&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25359292&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, G. N. Thyssen, J. N. Jenkins, and D. D. Fang, <span class="cit-pub-date">2014</span> Detection, validation and application of genotyping-by-sequencing based single nucleotide polymorphisms in upland cotton (<em>Gossypium hirsutum</em> L.). Plant Genome <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.20" data-doi="10.2135/cropsci1970.0011183X001000060019x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Texas marker-1. Description of a genetic standard for <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">670</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DKohel%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D670%26rft.epage%253D671%26rft.atitle%253DTexas%2BMarker-1.%2BDescription%2Bof%2Ba%2BGenetic%2BStandard%2Bfor%2BGossypium%2Bhirsutum%2BL.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1970.0011183X001000060019x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1970.0011183X001000060019x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.21" data-doi="10.1371/journal.pone.0048855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claverie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carazzolle</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimaraes Pereira</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Deep sequencing reveals differences in the transcriptional landscapes of fibers from two cultivated species of cotton.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e48855</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0048855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48855%26rft.epage%253De48855%26rft.atitle%253DDeep%2Bsequencing%2Breveals%2Bdifferences%2Bin%2Bthe%2Btranscriptional%2Blandscapes%2Bof%2Bfibers%2Bfrom%2Btwo%2Bcultivated%2Bspecies%2Bof%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048855%26rft_id%253Dinfo%253Apmid%252F23166598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23166598&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.22" data-doi="10.1139/g03&amp;ndash;050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibivilliers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bojinov</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A combined RFLP-SSR-AFLP map of tetraploid cotton based on a <em>Gossypium hirsutum</em> x <em>Gossypium barbadense</em> backcross population.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">626</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g03–050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D612%26rft.epage%253D626%26rft.atitle%253DA%2Bcombined%2BRFLP-SSR-AFLP%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bbased%2Bon%2Ba%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%2Bbackcross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg03%2526ndash%253B050%26rft_id%253Dinfo%253Apmid%252F12897870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g03&amp;ndash;050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12897870&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.23" data-doi="10.1038/ng.2987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">572</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/Ng.2987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2987%26rft_id%253Dinfo%253Apmid%252F24836287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836287&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.24" data-doi="10.1007/s00335-001-1016-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cudmore</span>  <span class="cit-name-given-names">R. H.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meer</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Map Manager QTX, cross-platform software for genetic mapping.</span> <abbr class="cit-jnl-abbrev">Mammalian Genome</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">930</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DManly%26rft.auinit1%253DK.%2BF.%26rft.volume%253D12%26rft.issue%253D12%26rft.spage%253D930%26rft.epage%253D932%26rft.atitle%253DMap%2BManager%2BQTX%252C%2Bcross-platform%2Bsoftware%2Bfor%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-001-1016-3%26rft_id%253Dinfo%253Apmid%252F11707780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-001-1016-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11707780&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172358900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.25" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11798</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.26" data-doi="10.1111/pbi.12088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakhwani</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Large-scale resource development in <em>Gossypium hirsutum</em> L. by 454 sequencing of genic-enriched libraries from six diverse genotypes.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">963</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D953%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12088%26rft_id%253Dinfo%253Apmid%252F23782852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23782852&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.27" data-doi="10.1534/genetics.166.1.389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (Gossypium).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRong%26rft.auinit1%253DJ.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D417%26rft.atitle%253DA%2B3347-Locus%2BGenetic%2BRecombination%2BMap%2Bof%2BSequence-Tagged%2BSites%2BReveals%2BFeatures%2Bof%2BGenome%2BOrganization%252C%2BTransmission%2Band%2BEvolution%2Bof%2BCotton%2B%2528Gossypium%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.389%26rft_id%253Dinfo%253Apmid%252F15020432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzExODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.28" data-doi="10.1007/s10681-006-9338-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic mapping of quantitative trait loci for fiber quality and yield trait by RIL approach in Upland cotton.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D155%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-006-9338-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-006-9338-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000245825300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Constructing a high-density linkage map for <em>Gossypium hirsutum</em> × <em>Gossypium barbadense</em> and identifying QTLs for lint percentage.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D57%26rft.spage%253D450%26rft_id%253Dinfo%253Apmid%252F25263268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25263268&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.30" data-doi="10.1371/journal.pone.0054985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De54985%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054985%26rft_id%253Dinfo%253Apmid%252F23372807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372807&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Construction of genetic map and QTL analysis of fiber quality traits for Upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D201%26rft.spage%253D195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazo</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite> <em>et al</em>., <span class="cit-pub-date">2014</span> A SNP genotyping array for hexaploid oat. Plant Genome 7.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite> <em>et al</em>., <span class="cit-pub-date">2013</span> “An ultra-high-density, transcript-based, genetic map of lettuce.” G3 (Bethesda) 3:617–631.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.34" data-doi="10.1007/s00122-013-2217-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity and population structure in the US Upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2217-3%26rft_id%253Dinfo%253Apmid%252F24170350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2217-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24170350&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.6.1187.35" data-doi="10.1139/cjb-2012-0192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdurakhmonov</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-M</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic diversity and population structure of cotton (<em>Gossypium</em> spp.) of the New World assessed by SSR markers.</span> <abbr class="cit-jnl-abbrev">Botany</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBotany%26rft.volume%253D91%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjb-2012-0192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjb-2012-0192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.36" data-doi="10.1186/1471-2229-9-125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sampling nucleotide diversity in cotton.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">125</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471–2229–9-125</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BDeynze%26rft.auinit1%253DA.%2BE.%26rft.volume%253D9%26rft.spage%253D125%26rft.epage%253D125%26rft.atitle%253DSampling%2Bnucleotide%2Bdiversity%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-125%26rft_id%253Dinfo%253Apmid%252F19840401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840401&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1187.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">“<em>JoinMap 4.” Software for the calculation of genetic linkage maps in experimental populations</em></span>. <span class="cit-publ-name">Kyazma BV</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.38" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.39" data-doi="10.1038/ng.2371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1098</span>–<span class="cit-lpage">1103</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1098%26rft.epage%253D1103%26rft.atitle%253DThe%2Bdraft%2Bgenome%2Bof%2Ba%2Bdiploid%2Bcotton%2BGossypium%2Braimondii.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2371%26rft_id%253Dinfo%253Apmid%252F22922876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22922876&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.40" data-doi="10.1111/pbi.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D12%26rft.spage%253D787%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12183%26rft_id%253Dinfo%253Apmid%252F24646323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24646323&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339488200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1187.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution and natural history of the cotton genus</span>, pp. <span class="cit-fpage">3</span>–<span class="cit-lpage">22</span> in <span class="cit-source">Genetics and Genomics of Cotton</span>, edited by A. H. <span class="cit-ed"><span class="cit-name-surname">Paterson</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.42" data-doi="10.1371/journal.pone.0008451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warburton</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic characterization and linkage disequilibrium estimation of a global maize collection using SNP markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e8451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DYan%26rft.auinit1%253DJ.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8451%26rft.epage%253De8451%26rft.atitle%253DGenetic%2Bcharacterization%2Band%2Blinkage%2Bdisequilibrium%2Bestimation%2Bof%2Ba%2Bglobal%2Bmaize%2Bcollection%2Busing%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008451%26rft_id%253Dinfo%253Apmid%252F20041112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20041112&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.43" data-doi="10.1093/nar/gkt1064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ficklin</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CottonGen: a genomics, genetics and breeding database for cotton research.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D1229</span>–<span class="cit-lpage">D1236</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1064</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1064%26rft_id%253Dinfo%253Apmid%252F24203703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0Mi9EMS9EMTIyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.44" data-doi="10.1534/g3.111.001552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-pub-date">2012</span> A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome. G3 (Bethesda) 2: 43–58 doi: .<span class="cit-pub-id cit-pub-id-doi">10.1534/g3.111.001552</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DYu%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D58%26rft.atitle%253DA%2BHigh-Density%2BSimple%2BSequence%2BRepeat%2Band%2BSingle%2BNucleotide%2BPolymorphism%2BGenetic%2BMap%2Bof%2Bthe%2BTetraploid%2BCotton%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001552%26rft_id%253Dinfo%253Apmid%252F22384381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjIvMS80MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.45" data-doi="10.1007/s11032-011-9549-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic mapping and quantitative trait locus analysis of fiber quality traits using a three-parent composite population in upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D29%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-011-9549-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-011-9549-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.46" data-doi="10.1007/s11032-009-9271-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Construction of a comprehensive PCR-based marker linkage map and QTL mapping for fiber quality traits in upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D24%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-009-9271-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-009-9271-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1187.47" data-doi="10.1534/g3.114.012542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spriggs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llewellyn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>. <span class="cit-pub-date">2014</span> Transcriptome and complexity-reduced, DNA-based identification of intraspecies single-nucleotide polymorphisms in the polyploid <em>Gossypium hirsutum</em> L. G3 (Bethesda) 4: 1893–905 doi: .<span class="cit-pub-id cit-pub-id-doi">10.1534/g3.114.012542</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012542%26rft_id%253Dinfo%253Apmid%252F25106949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTAvMTg5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1187.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1177" title="Caenorhabditis elegans TBX-2 Directly Regulates Its Own Expression in a Negative Autoregulatory Loop" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1211" title="The Mosaic Mutants of Cucumber: A Method to Produce Knock-Downs of Mitochondrial Transcripts" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1187.short" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1247xIAl4h2XXod25fmpVY3VaNn8P_wB1fMsTFVOpV4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1187" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1187" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1187" class="active" data-icon-position="" data-hide-link-title="0">Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of Gossypium spp.</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459913" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jJ5FaSHlodcaZHmP58nAvgnCVvgPoNKBlt8-s0BrpEA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1187.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1187" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-f9F6NvZ0QTvzzY3uXBnfUjDHHk9L7_lDXggTv_wRYPw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1187" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1187.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459913" id="node459913--2" data-pisa="ggg;5/6/1187" data-pisa-master="ggg;g3.115.018416" data-apath="/ggg/5/6/1187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of <em>Gossypium</em> spp.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda M. Hulse-Kemp</span>, <span class="highwire-citation-author" data-delta="1">Jana Lemm</span>, <span class="highwire-citation-author" data-delta="2">Joerg Plieske</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Ashrafi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-5343-9960" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ramesh Buyyarapu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David D. Fang</span>, <span class="highwire-citation-author" data-delta="6">James Frelichowski</span>, <span class="highwire-citation-author" data-delta="7">Marc Giband</span>, <span class="highwire-citation-author" data-delta="8">Steve Hague</span>, <span class="highwire-citation-author" data-delta="9">Lori L. Hinze</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-8124-8283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kelli J. Kochan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-3296-320X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Penny K. Riggs</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-7897-2916" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jodi A. Scheffler</span>, <span class="highwire-citation-author" data-delta="13">Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="14">Mauricio Ulloa</span>, <span class="highwire-citation-author" data-delta="15">Shirley S. Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qian-Hao Zhu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0001-8907-2841" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sumit K. Bag</span>, <span class="highwire-citation-author" data-delta="18">Archana Bhardwaj</span>, <span class="highwire-citation-author" data-delta="19">John J. Burke</span>, <span class="highwire-citation-author" data-delta="20">Robert L. Byers</span>, <span class="highwire-citation-author" data-delta="21">Michel Claverie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="22"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span>, <span class="highwire-citation-author" data-delta="23">David B. Harker</span>, <span class="highwire-citation-author" data-delta="24">Md S. Islam</span>, <span class="highwire-citation-author" data-delta="25">Johnie N. Jenkins</span>, <span class="highwire-citation-author" data-delta="26">Don C. Jones</span>, <span class="highwire-citation-author" data-delta="27">Jean-Marc Lacape</span>, <span class="highwire-citation-author" data-delta="28">Danny J. Llewellyn</span>, <span class="highwire-citation-author" data-delta="29">Richard G. Percy</span>, <span class="highwire-citation-author" data-delta="30">Alan E. Pepper</span>, <span class="highwire-citation-author" data-delta="31">Jesse A. Poland</span>, <span class="highwire-citation-author" data-delta="32">Krishan Mohan Rai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="33"><a href="http://orcid.org/0000-0002-7635-5049" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samir V. Sawant</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="34"><a href="http://orcid.org/0000-0002-4897-8812" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sunil Kumar Singh</span>, <span class="highwire-citation-author" data-delta="35">Andrew Spriggs</span>, <span class="highwire-citation-author" data-delta="36">Jen M. Taylor</span>, <span class="highwire-citation-author" data-delta="37">Fei Wang</span>, <span class="highwire-citation-author" data-delta="38">Scott M. Yourstone</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="39"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiuting Zheng</span>, <span class="highwire-citation-author" data-delta="40">Cindy T. Lawley</span>, <span class="highwire-citation-author" data-delta="41">Martin W. Ganal</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="42"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="43">Iain W. Wilson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="44"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David M. Stelly</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1187-1209;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018416 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459913--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda M. Hulse-Kemp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse-Kemp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse-Kemp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BM.%2BHulse-Kemp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda M. Hulse-Kemp</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jana Lemm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BLemm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemm%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJana%2BLemm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joerg Plieske </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joerg%2BPlieske%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plieske%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoerg%2BPlieske%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Ashrafi</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ramesh Buyyarapu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Dow AgroSciences, Trait Genetics and Technologies, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BBuyyarapu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buyyarapu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARamesh%2BBuyyarapu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5343-9960" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ramesh Buyyarapu</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David D. Fang</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James Frelichowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BFrelichowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frelichowski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BFrelichowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Marc Giband </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div><div class='author-affiliation'><span class='nlm-sup'>***</span>EMBRAPA, Algodão, Nucleo Cerrado, 75.375-000 Santo Antônio de Goias, GO, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BGiband%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giband%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BGiband%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Steve Hague </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BHague%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hague%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BHague%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Lori L. Hinze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lori%2BL.%2BHinze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hinze%20LL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALori%2BL.%2BHinze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kelli J. Kochan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Animal Science, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelli%2BJ.%2BKochan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kochan%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelli%2BJ.%2BKochan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8124-8283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelli J. Kochan</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Penny K. Riggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Animal Science, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penny%2BK.%2BRiggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riggs%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APenny%2BK.%2BRiggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3296-320X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Penny K. Riggs</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jodi A. Scheffler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡</span>USDA-ARS, Jamie Whitten Delta States Research Center, Stoneville, Mississippi 38776</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jodi%2BA.%2BScheffler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scheffler%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJodi%2BA.%2BScheffler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7897-2916" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jodi A. Scheffler</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Mauricio Ulloa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>****</span>USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BUlloa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ulloa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BUlloa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Shirley S. Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shirley%2BS.%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShirley%2BS.%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qian-Hao Zhu</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Sumit K. Bag </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sumit%2BK.%2BBag%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bag%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASumit%2BK.%2BBag%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8907-2841" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sumit K. Bag</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Archana Bhardwaj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Archana%2BBhardwaj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhardwaj%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArchana%2BBhardwaj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">John J. Burke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>****</span>USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BJ.%2BBurke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burke%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BJ.%2BBurke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Robert L. Byers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BByers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byers%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BL.%2BByers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Michel Claverie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michel%2BClaverie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Claverie%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichel%2BClaverie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§§</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-23"><div class="author-tooltip-name">David B. Harker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BHarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harker%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BHarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-24"><div class="author-tooltip-name">Md S. Islam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Md%2BS.%2BIslam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Islam%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMd%2BS.%2BIslam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-25"><div class="author-tooltip-name">Johnie N. Jenkins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*****</span>USDA-ARS, Genetics and Precision Agriculture Research, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johnie%2BN.%2BJenkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jenkins%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohnie%2BN.%2BJenkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-26"><div class="author-tooltip-name">Don C. Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††††</span>Cotton Incorporated, Agricultural Research, Cary, North Carolina 27513</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Don%2BC.%2BJones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jones%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADon%2BC.%2BJones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-27"><div class="author-tooltip-name">Jean-Marc Lacape </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Marc%2BLacape%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lacape%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Marc%2BLacape%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-28"><div class="author-tooltip-name">Danny J. Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BJ.%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BJ.%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-29"><div class="author-tooltip-name">Richard G. Percy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BPercy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Percy%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BPercy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-30"><div class="author-tooltip-name">Alan E. Pepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡‡</span>Department of Biology, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BPepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pepper%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BPepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-31"><div class="author-tooltip-name">Jesse A. Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§§§</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BA.%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BA.%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-32"><div class="author-tooltip-name">Krishan Mohan Rai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krishan%2BMohan%2BRai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohan%20Rai%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrishan%2BMohan%2BRai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-33"><div class="author-tooltip-name">Samir V. Sawant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samir%2BV.%2BSawant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sawant%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamir%2BV.%2BSawant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7635-5049" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samir V. Sawant</a></li></ul></div><div class="author-tooltip-34"><div class="author-tooltip-name">Sunil Kumar Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunil%2BKumar%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASunil%2BKumar%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4897-8812" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sunil Kumar Singh</a></li></ul></div><div class="author-tooltip-35"><div class="author-tooltip-name">Andrew Spriggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BSpriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spriggs%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BSpriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-36"><div class="author-tooltip-name">Jen M. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jen%2BM.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJen%2BM.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-37"><div class="author-tooltip-name">Fei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-38"><div class="author-tooltip-name">Scott M. Yourstone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BM.%2BYourstone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yourstone%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BM.%2BYourstone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-39"><div class="author-tooltip-name">Xiuting Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiuting%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiuting%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiuting Zheng</a></li></ul></div><div class="author-tooltip-40"><div class="author-tooltip-name">Cindy T. Lawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>******</span>Illumina Inc., San Francisco, California 94158</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cindy%2BT.%2BLawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawley%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACindy%2BT.%2BLawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-41"><div class="author-tooltip-name">Martin W. Ganal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BW.%2BGanal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ganal%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BW.%2BGanal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-42"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Allen Van Deynze</a></li></ul></div><div class="author-tooltip-43"><div class="author-tooltip-name">Iain W. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BW.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20IW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIain%2BW.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-44"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David M. Stelly</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stelly@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">stelly@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1187" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1187&amp;title=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1187&amp;title=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1187&amp;title=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1187&amp;text=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1187&amp;title=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1187&amp;t=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1187&amp;title=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1187&amp;title=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp." id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459913/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459913" id="node459913--3" data-pisa="ggg;5/6/1187" data-pisa-master="ggg;g3.115.018416" data-apath="/ggg/5/6/1187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Development of a 63K SNP Array for Cotton and High-Density Mapping of Intraspecific and Interspecific Populations of <em>Gossypium</em> spp.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda M. Hulse-Kemp</span>, <span class="highwire-citation-author" data-delta="1">Jana Lemm</span>, <span class="highwire-citation-author" data-delta="2">Joerg Plieske</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Ashrafi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-5343-9960" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ramesh Buyyarapu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David D. Fang</span>, <span class="highwire-citation-author" data-delta="6">James Frelichowski</span>, <span class="highwire-citation-author" data-delta="7">Marc Giband</span>, <span class="highwire-citation-author" data-delta="8">Steve Hague</span>, <span class="highwire-citation-author" data-delta="9">Lori L. Hinze</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-8124-8283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kelli J. Kochan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-3296-320X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Penny K. Riggs</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-7897-2916" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jodi A. Scheffler</span>, <span class="highwire-citation-author" data-delta="13">Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="14">Mauricio Ulloa</span>, <span class="highwire-citation-author" data-delta="15">Shirley S. Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qian-Hao Zhu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0001-8907-2841" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sumit K. Bag</span>, <span class="highwire-citation-author" data-delta="18">Archana Bhardwaj</span>, <span class="highwire-citation-author" data-delta="19">John J. Burke</span>, <span class="highwire-citation-author" data-delta="20">Robert L. Byers</span>, <span class="highwire-citation-author" data-delta="21">Michel Claverie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="22"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span>, <span class="highwire-citation-author" data-delta="23">David B. Harker</span>, <span class="highwire-citation-author" data-delta="24">Md S. Islam</span>, <span class="highwire-citation-author" data-delta="25">Johnie N. Jenkins</span>, <span class="highwire-citation-author" data-delta="26">Don C. Jones</span>, <span class="highwire-citation-author" data-delta="27">Jean-Marc Lacape</span>, <span class="highwire-citation-author" data-delta="28">Danny J. Llewellyn</span>, <span class="highwire-citation-author" data-delta="29">Richard G. Percy</span>, <span class="highwire-citation-author" data-delta="30">Alan E. Pepper</span>, <span class="highwire-citation-author" data-delta="31">Jesse A. Poland</span>, <span class="highwire-citation-author" data-delta="32">Krishan Mohan Rai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="33"><a href="http://orcid.org/0000-0002-7635-5049" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samir V. Sawant</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="34"><a href="http://orcid.org/0000-0002-4897-8812" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sunil Kumar Singh</span>, <span class="highwire-citation-author" data-delta="35">Andrew Spriggs</span>, <span class="highwire-citation-author" data-delta="36">Jen M. Taylor</span>, <span class="highwire-citation-author" data-delta="37">Fei Wang</span>, <span class="highwire-citation-author" data-delta="38">Scott M. Yourstone</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="39"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiuting Zheng</span>, <span class="highwire-citation-author" data-delta="40">Cindy T. Lawley</span>, <span class="highwire-citation-author" data-delta="41">Martin W. Ganal</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="42"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="43">Iain W. Wilson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="44"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David M. Stelly</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1187-1209;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018416 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459913--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda M. Hulse-Kemp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse-Kemp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse-Kemp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BM.%2BHulse-Kemp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda M. Hulse-Kemp</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jana Lemm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BLemm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemm%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJana%2BLemm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joerg Plieske </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joerg%2BPlieske%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plieske%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoerg%2BPlieske%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Ashrafi</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ramesh Buyyarapu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Dow AgroSciences, Trait Genetics and Technologies, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BBuyyarapu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buyyarapu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARamesh%2BBuyyarapu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5343-9960" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ramesh Buyyarapu</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David D. Fang</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James Frelichowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BFrelichowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frelichowski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BFrelichowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Marc Giband </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div><div class='author-affiliation'><span class='nlm-sup'>***</span>EMBRAPA, Algodão, Nucleo Cerrado, 75.375-000 Santo Antônio de Goias, GO, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BGiband%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giband%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BGiband%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Steve Hague </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BHague%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hague%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BHague%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Lori L. Hinze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lori%2BL.%2BHinze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hinze%20LL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALori%2BL.%2BHinze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kelli J. Kochan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Animal Science, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelli%2BJ.%2BKochan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kochan%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelli%2BJ.%2BKochan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8124-8283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelli J. Kochan</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Penny K. Riggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Animal Science, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penny%2BK.%2BRiggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riggs%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APenny%2BK.%2BRiggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3296-320X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Penny K. Riggs</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jodi A. Scheffler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡</span>USDA-ARS, Jamie Whitten Delta States Research Center, Stoneville, Mississippi 38776</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jodi%2BA.%2BScheffler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scheffler%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJodi%2BA.%2BScheffler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7897-2916" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jodi A. Scheffler</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Mauricio Ulloa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>****</span>USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BUlloa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ulloa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BUlloa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Shirley S. Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shirley%2BS.%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShirley%2BS.%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qian-Hao Zhu</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Sumit K. Bag </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sumit%2BK.%2BBag%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bag%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASumit%2BK.%2BBag%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8907-2841" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sumit K. Bag</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Archana Bhardwaj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Archana%2BBhardwaj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhardwaj%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArchana%2BBhardwaj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">John J. Burke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>****</span>USDA-ARS, PA, Plant Stress and Germplasm Development Research Unit, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BJ.%2BBurke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burke%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BJ.%2BBurke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Robert L. Byers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BByers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byers%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BL.%2BByers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Michel Claverie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michel%2BClaverie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Claverie%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichel%2BClaverie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§§</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-23"><div class="author-tooltip-name">David B. Harker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BHarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harker%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BHarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-24"><div class="author-tooltip-name">Md S. Islam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS-SRRC, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Md%2BS.%2BIslam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Islam%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMd%2BS.%2BIslam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-25"><div class="author-tooltip-name">Johnie N. Jenkins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*****</span>USDA-ARS, Genetics and Precision Agriculture Research, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johnie%2BN.%2BJenkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jenkins%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohnie%2BN.%2BJenkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-26"><div class="author-tooltip-name">Don C. Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††††</span>Cotton Incorporated, Agricultural Research, Cary, North Carolina 27513</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Don%2BC.%2BJones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jones%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADon%2BC.%2BJones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-27"><div class="author-tooltip-name">Jean-Marc Lacape </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>CIRAD, UMR AGAP, Montpellier, F34398, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Marc%2BLacape%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lacape%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Marc%2BLacape%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-28"><div class="author-tooltip-name">Danny J. Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BJ.%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BJ.%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-29"><div class="author-tooltip-name">Richard G. Percy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA-ARS-SPARC, Crop Germplasm Research Unit, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BPercy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Percy%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BPercy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-30"><div class="author-tooltip-name">Alan E. Pepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡‡</span>Department of Biology, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BPepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pepper%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BPepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-31"><div class="author-tooltip-name">Jesse A. Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§§§</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BA.%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BA.%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-32"><div class="author-tooltip-name">Krishan Mohan Rai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krishan%2BMohan%2BRai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohan%20Rai%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrishan%2BMohan%2BRai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-33"><div class="author-tooltip-name">Samir V. Sawant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samir%2BV.%2BSawant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sawant%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamir%2BV.%2BSawant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7635-5049" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samir V. Sawant</a></li></ul></div><div class="author-tooltip-34"><div class="author-tooltip-name">Sunil Kumar Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡‡‡</span>CSIR-National Botanical Research Institute, Plant Molecular Biology Division, Lucknow-226001, UP, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunil%2BKumar%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASunil%2BKumar%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4897-8812" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sunil Kumar Singh</a></li></ul></div><div class="author-tooltip-35"><div class="author-tooltip-name">Andrew Spriggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BSpriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spriggs%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BSpriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-36"><div class="author-tooltip-name">Jen M. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jen%2BM.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJen%2BM.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-37"><div class="author-tooltip-name">Fei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-38"><div class="author-tooltip-name">Scott M. Yourstone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>Brigham Young University, Plant and Wildlife Science Department, Provo, Utah 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BM.%2BYourstone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yourstone%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BM.%2BYourstone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-39"><div class="author-tooltip-name">Xiuting Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiuting%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiuting%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiuting Zheng</a></li></ul></div><div class="author-tooltip-40"><div class="author-tooltip-name">Cindy T. Lawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>******</span>Illumina Inc., San Francisco, California 94158</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cindy%2BT.%2BLawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawley%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACindy%2BT.%2BLawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-41"><div class="author-tooltip-name">Martin W. Ganal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>TraitGenetics GmbH, 06466 Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BW.%2BGanal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ganal%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BW.%2BGanal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-42"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences and Seed Biotechnology Center, University of California-Davis, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Allen Van Deynze</a></li></ul></div><div class="author-tooltip-43"><div class="author-tooltip-name">Iain W. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††††</span>CSIRO Agriculture Flagship, Black Mountain Laboratories, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BW.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20IW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIain%2BW.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-44"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil & Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Degree Program in Genetics, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David M. Stelly</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stelly@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">stelly@tamu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1187&amp;rft.epage=1209&amp;rft.atitle=Development%20of%20a%2063K%20SNP%20Array%20for%20Cotton%20and%20High-Density%20Mapping%20of%20Intraspecific%20and%20Interspecific%20Populations%20of%20Gossypium%20spp.&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hulse-Kemp%2C+Amanda+M.&amp;rft.au=Lemm%2C+Jana&amp;rft.au=Plieske%2C+Joerg&amp;rft.au=Ashrafi%2C+Hamid&amp;rft.au=Buyyarapu%2C+Ramesh&amp;rft.au=Fang%2C+David+D.&amp;rft.au=Frelichowski%2C+James&amp;rft.au=Giband%2C+Marc&amp;rft.au=Hague%2C+Steve&amp;rft.au=Hinze%2C+Lori+L.&amp;rft.au=Kochan%2C+Kelli+J.&amp;rft.au=Riggs%2C+Penny+K.&amp;rft.au=Scheffler%2C+Jodi+A.&amp;rft.au=Udall%2C+Joshua+A.&amp;rft.au=Ulloa%2C+Mauricio&amp;rft.au=Wang%2C+Shirley+S.&amp;rft.au=Zhu%2C+Qian-Hao&amp;rft.au=Bag%2C+Sumit+K.&amp;rft.au=Bhardwaj%2C+Archana&amp;rft.au=Burke%2C+John+J.&amp;rft.au=Byers%2C+Robert+L.&amp;rft.au=Claverie%2C+Michel&amp;rft.au=Gore%2C+Michael+A.&amp;rft.au=Harker%2C+David+B.&amp;rft.au=Islam%2C+Md+S.&amp;rft.au=Jenkins%2C+Johnie+N.&amp;rft.au=Jones%2C+Don+C.&amp;rft.au=Lacape%2C+Jean-Marc&amp;rft.au=Llewellyn%2C+Danny+J.&amp;rft.au=Percy%2C+Richard+G.&amp;rft.au=Pepper%2C+Alan+E.&amp;rft.au=Poland%2C+Jesse+A.&amp;rft.au=Mohan+Rai%2C+Krishan&amp;rft.au=Sawant%2C+Samir+V.&amp;rft.au=Singh%2C+Sunil+Kumar&amp;rft.au=Spriggs%2C+Andrew&amp;rft.au=Taylor%2C+Jen+M.&amp;rft.au=Wang%2C+Fei&amp;rft.au=Yourstone%2C+Scott+M.&amp;rft.au=Zheng%2C+Xiuting&amp;rft.au=Lawley%2C+Cindy+T.&amp;rft.au=Ganal%2C+Martin+W.&amp;rft.au=Van+Deynze%2C+Allen&amp;rft.au=Wilson%2C+Iain+W.&amp;rft.au=Stelly%2C+David+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459913/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459913/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459913/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459913/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459913/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459913/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459913/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459913/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459913/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459913/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459913/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459913/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1187" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1187.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1187.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1187.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1187.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
