<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1223" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1223.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1223.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1223.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1223.atom" />
<meta name="HW.pisa" content="ggg;5/6/1223" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.115.017830" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Koh et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Changes in protein subcellular localization and abundance are central to biological regulation in eukaryotic cells. Quantitative measures of protein dynamics in vivo are therefore highly useful for elucidating specific regulatory pathways. Using a combinatorial approach of yeast synthetic genetic array technology, high-content screening, and machine learning classifiers, we developed an automated platform to characterize protein localization and abundance patterns from images of log phase cells from the open-reading frame−green fluorescent protein collection in the budding yeast, Saccharomyces cerevisiae . For each protein, we produced quantitative profiles of localization scores for 16 subcellular compartments at single-cell resolution to trace proteome-wide relocalization in conditions over time. We generated a collection of ∼300,000 micrographs, comprising more than 20 million cells and ∼9 billion quantitative measurements. The images depict the localization and abundance dynamics of more than 4000 proteins under two chemical treatments and in a selected mutant background. Here, we describe CYCLoPs (Collection of Yeast Cells Localization Patterns), a web database resource that provides a central platform for housing and analyzing our yeast proteome dynamics datasets at the single cell level. CYCLoPs version 1.0 is available at &lt;http://cyclops.ccbr.utoronto.ca&gt;. CYCLoPs will provide a valuable resource for the yeast and eukaryotic cell biology communities and will be updated as new experiments become available." />
<meta name="DC.Contributor" content="Judice L. Y. Koh" />
<meta name="DC.Contributor" content="Yolanda T. Chong" />
<meta name="DC.Contributor" content="Helena Friesen" />
<meta name="DC.Contributor" content="Alan Moses" />
<meta name="DC.Contributor" content="Charles Boone" />
<meta name="DC.Contributor" content="Brenda J. Andrews" />
<meta name="DC.Contributor" content="Jason Moffat" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Changes in protein subcellular localization and abundance are central to biological regulation in eukaryotic cells. Quantitative measures of protein dynamics &lt;i&gt;in vivo&lt;/i&gt; are therefore highly useful for elucidating specific regulatory pathways. Using a combinatorial approach of yeast synthetic genetic array technology, high-content screening, and machine learning classifiers, we developed an automated platform to characterize protein localization and abundance patterns from images of log phase cells from the open-reading frame−green fluorescent protein collection in the budding yeast, &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. For each protein, we produced quantitative profiles of localization scores for 16 subcellular compartments at single-cell resolution to trace proteome-wide relocalization in conditions over time. We generated a collection of ∼300,000 micrographs, comprising more than 20 million cells and ∼9 billion quantitative measurements. The images depict the localization and abundance dynamics of more than 4000 proteins under two chemical treatments and in a selected mutant background. Here, we describe CYCLoPs (Collection of Yeast Cells Localization Patterns), a web database resource that provides a central platform for housing and analyzing our yeast proteome dynamics datasets at the single cell level. CYCLoPs version 1.0 is available at http://cyclops.ccbr.utoronto.ca. CYCLoPs will provide a valuable resource for the yeast and eukaryotic cell biology communities and will be updated as new experiments become available.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1223" />
<meta name="citation_id" content="5/6/1223" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1223" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1223.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1223.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1223.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017830" />
<meta name="citation_pmid" content="26048563" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1223" />
<meta name="citation_lastpage" content="1232" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Judice L. Y. Koh" />
<meta name="citation_author_institution" content="The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author" content="Yolanda T. Chong" />
<meta name="citation_author_institution" content="The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author" content="Helena Friesen" />
<meta name="citation_author_institution" content="The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author" content="Alan Moses" />
<meta name="citation_author_institution" content="Department of Cell &amp; Systems Biology, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author" content="Charles Boone" />
<meta name="citation_author_institution" content="The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author_email" content="j.moffat@utoronto.ca" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_author" content="Brenda J. Andrews" />
<meta name="citation_author_institution" content="The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author_email" content="j.moffat@utoronto.ca" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_author" content="Jason Moffat" />
<meta name="citation_author_institution" content="The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1" />
<meta name="citation_author_email" content="j.moffat@utoronto.ca" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=M. Breker;citation_author=M. Gymrek;citation_author=M. Schuldiner;citation_title=A novel single-cell screening platform reveals proteome plasticity during yeast stress responses.;citation_volume=200;citation_year=2013;citation_pmid=23509072;citation_doi=10.1083/jcb.201301120" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Bagging predictors.;citation_pages=123-140;citation_volume=2;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. E. Carpenter;citation_author=T. R. Jones;citation_author=M. R. Lamprecht;citation_author=C. Clarke;citation_author=I. H. Kang;citation_title=CellProfiler: image analysis software for identifying and quantifying cell phenotypes.;citation_pages=R100-R100;citation_volume=7;citation_year=2006;citation_issue=10;citation_pmid=17076895;citation_doi=10.1186/gb-2006-7-10-r100" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Chen;citation_author=L. Shi;citation_author=R. Kelly;citation_author=R. Perkins;citation_author=H. Fang;citation_title=Selecting a single model or combining multiple models for microarray-based classifier development?–a comparative analysis based on large and diverse datasets generated from the MAQC-II project.;citation_pages=S3;citation_volume=12;citation_year=2011;citation_pmid=21342560;citation_doi=10.1186/1471-2105-12-S1-S3" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. C. Chen;citation_author=T. Zhao;citation_author=G. J. Gordon;citation_author=R. F. Murphy;citation_title=Automated image analysis of protein localization in budding yeast;citation_pages=i66-i71;citation_volume=23;citation_year=2007;citation_issue=13;citation_pmid=17646347;citation_doi=10.1093/bioinformatics/btm206" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_title=Yeast proteome dynamics from single cell imaging and automated analysis.;citation_author=Y. T. Chong;citation_author=J. L. Y. Koh;citation_author=H. Friesen;citation_author=S. Kaluarachchi Duffy;citation_author=M. J. Cox;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Dénervaud;citation_author=J. Becker;citation_author=R. Delgado-Gonzalo;citation_author=P. Damay;citation_author=A. S. Rajkumar;citation_title=A chemostat array enables the spatio-temporal analysis of the yeast proteome.;citation_volume=110;citation_year=2013;citation_pmid=24019481;citation_doi=10.1073/pnas.1308265110" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W. K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Cytometry. Part A :  the journal of the International Society for Analytical Cytology;citation_journal_abbrev=Cytometry A;citation_author=S. Huh;citation_author=D. Lee;citation_author=R. F. Murphy;citation_title=Efficient framework for automated classification of subcellular patterns in budding yeast.;citation_pages=934-940;citation_volume=75;citation_year=2009;citation_issue=11;citation_pmid=19753630" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. K. Huh;citation_author=J. V. Falvo;citation_author=L. C. Gerke;citation_author=A. S. Carroll;citation_author=R. W. Howson;citation_title=Global analysis of protein localization in budding yeast.;citation_pages=686-691;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562095;citation_doi=10.1038/nature02026" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Jin;citation_author=J. Li;citation_author=F. S. Vizeacoumar;citation_author=Z. Li;citation_author=R. Min;citation_title=PhenoM: a database of morphological phenotypes caused by mutation of essential genes in Saccharomyces cerevisiae.;citation_volume=40;citation_year=2012;citation_pmid=22009677;citation_doi=10.1093/nar/gkr827" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. Kals;citation_author=K. Natter;citation_author=G. G. Thallinger;citation_author=Z. Trajanoski;citation_author=S. D. Kohlwein;citation_title=YPL.db2: the Yeast Protein Localization database, version 2.0.;citation_pages=213-218;citation_volume=22;citation_year=2005;citation_issue=3;citation_pmid=15704222;citation_doi=10.1002/yea.1204" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=N. A. Kulak;citation_author=G. Pichler;citation_author=I. Paron;citation_author=N. Nagaraj;citation_author=M. Mann;citation_title=Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells.;citation_pages=319-324;citation_volume=11;citation_year=2014;citation_pmid=24487582;citation_doi=10.1038/nmeth.2834" />
<meta name="citation_reference" content="citation_title=Mol. Cell Proteomics;citation_author=N. Nagaraj;citation_author=N. A. Kulak;citation_author=J. Cox;citation_author=N. Neuhauser;citation_author=K. Mayr;citation_title=System-wide perturbation analysis with nearly complete coverage of the yeast proteome by single-shot ultra HPLC runs on a bench top Orbitrap.;citation_volume=11;citation_year=2012;citation_pmid=22021278;citation_doi=10.1074/mcp.M111.013722" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. Newman;citation_author=S. Ghaemmaghami;citation_author=J. Ihmels;citation_author=D. K. Breslow;citation_author=M. Noble;citation_title=Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise.;citation_pages=840-846;citation_volume=441;citation_year=2006;citation_issue=7095;citation_pmid=16699522;citation_doi=10.1038/nature04785" />
<meta name="citation_reference" content="citation_title=Advances in Kernel Methods - Support Vector Learning;citation_title=Fast training of support vector machines using sequential minimal optimationation;citation_author=J. Platt;citation_pages=185-208;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Expert Syst. Appl.;citation_author=M. Reboiro-Jato;citation_author=R. Laza;citation_author=H. López-Fernández;citation_author=D. Glez-Peña;citation_author=F. Díaz;citation_title=genEnsemble: A new model for the combination of classifiers and integration of biological knowledge applied to genomic data.;citation_pages=52-63;citation_volume=40;citation_year=2013;citation_doi=10.1016/j.eswa.2012.07.003" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Riffle;citation_author=T. N. Davis;citation_title=The Yeast Resource Center Public Image Repository: A large database of fluorescence microscopy images.;citation_pages=263-263;citation_volume=11;citation_year=2010;citation_pmid=20482811;citation_doi=10.1186/1471-2105-11-263" />
<meta name="citation_reference" content="citation_journal_title=Artif. Intell. Rev.;citation_author=L. Rokach;citation_title=Ensemble-based classifiers.;citation_pages=1-39;citation_volume=33;citation_year=2010;citation_doi=10.1007/s10462-009-9124-7" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=T. L. Saito;citation_author=M. Ohtani;citation_author=H. Sawai;citation_author=F. Sano;citation_author=A. Saka;citation_title=SCMD: Saccharomyces cerevisiae Morphological Database;citation_pages=D319-322;citation_volume=32;citation_year=2004;citation_issue=90001;citation_pmid=14681423;citation_doi=10.1093/nar/gkh113" />
<meta name="citation_reference" content="citation_journal_title=J. Opt. Soc. Am.;citation_author=M. R. Teague;citation_title=Image analysis via the general theory of moments.;citation_pages=920-930;citation_volume=70;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Wiwatwattana;citation_author=C. M. Landau;citation_author=G. J. Cope;citation_author=G. A. Harp;citation_author=A. Kumar;citation_title=Organelle DB: an updated resource of eukaryotic protein localization and function.;citation_volume=35;citation_year=2007;citation_pmid=17130152;citation_doi=10.1093/nar/gkl1000" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="Changes in protein subcellular localization and abundance are central to biological regulation in eukaryotic cells. Quantitative measures of protein dynamics in vivo are therefore highly useful for elucidating specific regulatory pathways. Using a combinatorial approach of yeast synthetic genetic array technology, high-content screening, and machine learning classifiers, we developed an automated platform to characterize protein localization and abundance patterns from images of log phase cells from the open-reading frame−green fluorescent protein collection in the budding yeast, Saccharomyces cerevisiae . For each protein, we produced quantitative profiles of localization scores for 16 subcellular compartments at single-cell resolution to trace proteome-wide relocalization in conditions over time. We generated a collection of ∼300,000 micrographs, comprising more than 20 million cells and ∼9 billion quantitative measurements. The images depict the localization and abundance dynamics of more than 4000 proteins under two chemical treatments and in a selected mutant background. Here, we describe CYCLoPs (Collection of Yeast Cells Localization Patterns), a web database resource that provides a central platform for housing and analyzing our yeast proteome dynamics datasets at the single cell level. CYCLoPs version 1.0 is available at &lt;http://cyclops.ccbr.utoronto.ca&gt;. CYCLoPs will provide a valuable resource for the yeast and eukaryotic cell biology communities and will be updated as new experiments become available." />
<meta name="og-title" property="og:title" content="CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1223" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Changes in protein subcellular localization and abundance are central to biological regulation in eukaryotic cells. Quantitative measures of protein dynamics in vivo are therefore highly useful for elucidating specific regulatory pathways. Using a combinatorial approach of yeast synthetic genetic array technology, high-content screening, and machine learning classifiers, we developed an automated platform to characterize protein localization and abundance patterns from images of log phase cells from the open-reading frame−green fluorescent protein collection in the budding yeast, Saccharomyces cerevisiae . For each protein, we produced quantitative profiles of localization scores for 16 subcellular compartments at single-cell resolution to trace proteome-wide relocalization in conditions over time. We generated a collection of ∼300,000 micrographs, comprising more than 20 million cells and ∼9 billion quantitative measurements. The images depict the localization and abundance dynamics of more than 4000 proteins under two chemical treatments and in a selected mutant background. Here, we describe CYCLoPs (Collection of Yeast Cells Localization Patterns), a web database resource that provides a central platform for housing and analyzing our yeast proteome dynamics datasets at the single cell level. CYCLoPs version 1.0 is available at &lt;http://cyclops.ccbr.utoronto.ca&gt;. CYCLoPs will provide a valuable resource for the yeast and eukaryotic cell biology communities and will be updated as new experiments become available." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"V11D80e4nx8Mo4jVQWffmqBTrcO_qy6BS2siV6Pt_-k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459956","apath":"\/ggg\/5\/6\/1223.atom","pisa":"ggg;5\/6\/1223","ac":{"\/ggg\/5\/6\/1223.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1223.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1223"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1223"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1223":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ines\\\\\\\/vg8vG2pv0FgVS4R0nPuCAXCpFcVJNPdL3pOIVCVGJESaQyirRERMLwQoPGIOSchpGZW\\\\\\\/ZvgQB00OF1DJ9HghDwihc0BmYK8pTgdSsBog0TklfXekJ2MDOsU6HU1YGsFKDpo2RKy4aa7\\\\\\\/IatLGyY1MAFdCYVpZqW5zCj+YBw5OQUUKjbdpgcjCfOISWlYa5SHxlgUFkVLckAG9h7hb4UKpaZLk7BicpFHS3cUSpdetT1X1A3RK7pzfyPe86YZaSDxXzku5NXxEQzrWC7KWnPi5\\\\\\\/Csn0Ohp1ER3ncqJPadSkK3gfCqWIaEWSV2ZeW7bOr4LynZtxbJlVXIs0qMuUEXKb4\\\\\\\/ZMzcQsEaY7Du\\\\\\\/ijAB++YG3Q2AYXTMzP19A+sBoUTTJhu4Y4idWHUh7EogLp2CFCwugf0qX25zvsqMOnBytKQDwiJYOXXib3p9ikSKABOppJRvH1N2TQJdn0kWjmIq\\\\\\\/jkf+hOGTWJcR1LOdOb\\\\\\\/t9Bke\\\\\\\/gxyNO3dNEnFf\\\\\\\/1UAxVmy2TGN5427vThRNo6Uf\\\\\\\/LLT6y1Wts+zmQFFi3yypDa8j0uZ1r3xtsX4rvBhsA9sIRjL66Ka+iun0+4ez0hyRLj4ppo0LH4SU\\\\\\\/tkrNo8HkRrcfoNEvmex4ynxq0yAewU9yHK2HVAlGKlzLFuRWI9cz4QiWqbfIOMl+l7uZqljPnXfkuFrR+4lLcGZAlrEM7dL4\\\\\\\/+kF1a0eqQWaJpR\\\\\\\/JNoWSa4APyIaXqT868dT2m32xMneSgjBbUzZzh+d6OZ8DReQjNKEqIitFwn7HHQgh2XHhDzPoaBD31yx\\\\\\\/cqOaA1c0RrveYn3xsHKFhIdPO5Ye+tWoAvVeuSOXiRgfDNo8mEkIdcaHM3B+AwgVCB62DazhgFcM1yv7nlDyzjpU66+Epjd6anwaDUamJv8j0\\\\\\\/s27GGLFeIS8vB4Lp5Ct8zy4mamOb3r79oOg+ctna6sy+q+dUiR+9HTl3qaZchxepg1+\\\\\\\/kufVO71dGJVENdTANWTKICU99o\\\\\\\/+Ju7H9MKVdxynIZMoiyM0EnU48OnbnTPc\\\\\\\/MA6JhpoBMbVfEk\\\\\\\/H5MEQb\\\\\\\/kPbf0y+49WG4Tbbzkv4BFNOTwujOkuzOoAPOvxCEFPc7WRa7jPNLJXl4PwG67RqTtIDL\\\\\\\/76Ps6gs2rPchl5IbGtzreFw7COlrbxXzbkZgre7lnTTtK4wp6EpP1n69OPYDkaPoEWUyV7mYubmkMUlv0XRR5DoJQnfkbhAuTDyjDlN+oDEJwqLlRbAPJorxT1w3UONTmt8VOzZLrDrJTq\\\u0022,\\\u0022iv\\\u0022:\\\u0022nabvrKWc9O0lYv8nS7Kd1Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225416ad3ad620751b34347626321e11adc9c3bc247eb16738b4b52d4c57926460\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223itqF7GrmZG1Droo5SEEqHQuJ9x+uTlOg5LuN4sR72MBz9qsSFTLQaBaq+iK0EiSOWSiCaYQ+Lb13b0gx+VqACuE\\\\\\\/+7WF5JddCW5X3Mxw3ih+Lr\\\\\\\/lxdL9qXn4B4ZzTPpZwOWteICz3su\\\\\\\/5R1STsQY0dLAWjxKusIjXHWY3la3UwFuWtUljk3rfzOPnhC4GPOmoOPIh5Nkf9tJs2hBrwqWRLG1obbcseqaQ5PHnrnzyijPt8l+uD3mnuVAy61cA6Tg9RRurBN9zDsw5ZbBjYiQ3x6hwwZGy26UL0zlASJuyd5Kl6ioUWf7OaF0U8K34uz3\\\\\\\/wrwYn0YoKhNIumAb9aFBMj5PGhbprwfi0bIP8204xzpKC+E5Qb+TSlIuW6LYHp+TJpXCsBKITsp42FvQQy9Vpf8HvHFrdpiY0tsu6qKFNN9dcW9I\\\\\\\/I1sFv\\\\\\\/+5HHN6lQGyosqT0pnm0nlj0fkP7cqeyLO6e1UUHI4WNpmnHZT56vfoA\\\\\\\/rxXHG42rC9xZExTV5+cNKPkYhXqpgztk6LLotawKFriOTOZgZhqK4n8aubR6lhDI7nu3ZclO3OkC7RVBSMpCJg9L1ysUpF4WkUnqEN+Q\\\\\\\/jLneyBhf20Wsp0pcWuFxyFVt8Xrk54kwIVUWvRA\\\\\\\/eikn80yB0qZI4BbnZJzHfwbyfKoH4ujgno6c9z+3hCMyC65Bna0qCt9lqjm4UdWdjGBu2S9iOomhikyyINqEK+veSahxQIbBGUw7uncMyTzTVWk8IiRx2leFVFq+F\\\\\\\/BqslmfWhzCI9UTO\\\\\\\/dcL9\\\\\\\/3XYxdKy73MxZ59Tb\\\\\\\/8388KPfTDBGIQBWBm7ny2k+8tdV43uHUNaweBvjYCBlsyaB8srY2nY9z+EYmNjcJeAVGd3cOS5IOoGI1F2XxRRC8poPsVzjjlSLBko0rLI9PZ+\\\\\\\/e0OYIeWnblbiEslZPF1WPMEHqPaq6QqaeefmuJvBkP0ic9wpmUBIiDi8LRHzLcS2Tez9Z1WU25UinK506AGt4UMv5twruUsR2rOgd5w\\\\\\\/nMTiD1jl0Omu64EuqqPQW2fkFm1VCSqSBe73uBuFkb4RIOH9qX3bq4VIx1jsvga8sJF55bLOexSqmUPWl+vRNL3YaTVDng3saNq+zYVABBzy0G6nTaiFhzwhXXfipzQkM3W+aq12xtZ7p3kDX22JCzOCk1of2Mvp9YMu8yraw7p5SZTzRjFgt1kxbwmNGvKTzwyv\\\\\\\/sfVYIhmN8d6LmPIc5Qp7h1lC118ne\\\\\\\/vAPPpxuvK\\\\\\\/SL9IBDyxoIO7C00AQbuL9TWJxY2F0VHf8LXf8Z0kcNaWrsYroOBZn7KdraUhYmTePq\\\\\\\/ci\\\\\\\/NJB5SECqIGM28ONAZeGWswrJh3woxv4rXR3y1H8itBW5j64NaxxoLxPSHc7wMvIYs1ZbPROKQJKZOMjfd3tQkAexKA6sJS069ZgzPHVHfX4sP\\\\\\\/KQtJYXG2Karn2Z4RqPtQ6jPpwHtcktHc19m+Vo0tLhpni4igjINAiZ7fUvpJvZvmHAaV7IfD3EbV37Vbg27MNCJvjL4hg8sQcG6+G1KNl3Nb9v3GNbQw87gPkGQEqbjoPqKc8Y2IwYWMA53MbDQY\\\\\\\/c6qul7xuH9IJLuEMmddt7QzNTxfPcfDQHe+fyS6Z\\\\\\\/4\\\\\\\/HTqk0+k7oAKckqoqo6Z8fKwcKT1ACHOvawDOlmQ3+\\\\\\\/9LNmoC8odz0y6iWAR5qhaGOVbGk7gBIPAp2JD5qhJROShiZiWxqEwTTc8CntkDw6cjBlMrI7AhZiNM7\\\\\\\/l5hn\\\\\\\/js1\\\\\\\/7AQPPvG9NqZcgw0S8Y+6K6Ww2ydWbiBaM5pHDiATFmPgaKaKLtwwjSJXQrJW67d0ntDU7Zo1\\\\\\\/tqAKzz6oOdO4WC151o1LcIVkOpZPR8Z2lWULMBfdVICOABOMI7JYQRT\\\\\\\/NP2HXSl7Bsgbu9Pp\\\\\\\/2unXh+wWj2jby1K5PBKWgC2Z6lQxvqM3BhIScumP6NKrFSnrTkcnV9zs2UxqshoeLMET1DMJmgl6GNuXDi+sHSv2sEiHDxhaRiFMZKAGl8BnSkMSxbvvkQPnLJrstnVb+8Y\\\\\\\/BAQS8yGkgnQkb1HPubVCAlMlaeIwDK3PFwSqro5dczKCbPsU5aD7yd9CkB5\\\\\\\/yuMgMhjPJDcN8WOXAtlTZZoWm77+SI3UEXEGjNZS1wAv3dilD0t7BV44\\\\\\\/ijApe748y83PQs9YfN+WXf+rmqWVHWnXFLL9EhRH6W2vIn+55LlUOYr+LoXdEdpszi2OFgkOj44ayMCHO\\\\\\\/bl9jK1ypRwjRpX6uhy2Ro4NdZk0wrb1Syb6lrg26I5MWmoluyBNfcb8XIfM06xJeMcpp0IbgHxHzBkybVtR8U\\\\\\\/dcr57PXcpJgSxgvHBDEdoMWNz58K5Cb\\\\\\\/Cd6S9cHL2oKvnaPrlz\\\\\\\/ogVEDdI4CL3MAG1RH8f96ISa\\\\\\\/uB382YC4RF5YgXswXIpbcv8I+7AdPk9ZvMdjE3A5scIOGl30uT5bDiKNJxKQVMMZyupXyDpGBOwqKv9e7egsxyf7qTReSFX8gsXpaSV9h+B\\\\\\\/JnsLNva5JmkH99R8Cw5ocrQSWG8og61GcLMnh7VmUmXigOrZUQN1soBnjHlktNlnQKUBLiU765Lv6ZdJoejslWpRTB2AF40e\\\\\\\/IHDmWXmgaLraLcbnP3W\\\\\\\/F53mCIOUMjz3NPGERnVdlb4WyuLpTXuqtbS6uBIbTFKhpkAzms29Dskk9SLz8MZjNfPSbecddgy94In\\\\\\\/HQNqiNCkpxyXOSKCnRtvgolFYozhaO5JW4+h7VyWfzym6Rxbrdjb6rkb0AXKeAyOyoU7V6AgtBTXiob2bCKCyRcSq6ecLCPnXPCHstLbRKqjg2VDX7JppB1m89Kuj8HaNUK6hbF9pMawT2BH4PoJzShKn2b0AkB+npVXeErBc6bVl2pSSOPHDOQ6iRM5ndLEEyc8OLhxZ3hZdnN0k7S2AXnXi7zqrr3pFz3MCTATr22+A7AdsVr7a3zyqdcQCDTlxrnRAFZxCRHsOV3mYgx1i3Xr6\\\\\\\/+hgEJ41QIU2PZwr\\\\\\\/NF5wDjARSgoAxmTEXfSFGI+3nmiaf2MojfCtIus23t0gfeWtexEkADdVXX6wC6JtxgJzy24autGoLl\\\\\\\/sM+3QF7MTJW+koSb0SVYD1TGd5iqdui5QbpAmTe0GMtjicYDIyKdDV0vf2ePaIrrD02EyOs48vVFJZE7gr8LXQuZWnn10UJmEXYFcy0aDQ+I7aNUhqihJN619lDHokH1RMUgoSuMgYyZQFDD1YzDpsLREgjefL2m15ldU\\\\\\\/CrzkoBsaQ4YL9Zbmai8Xkq7DxJdlwKHj5pGXVJCcy+w+Ie7zILIVakcg9ZOdH5BvOq7quE8NNgexjKBZSfF4jbxjhHBmON1XXCI09TP\\\\\\\/evJu5mafJ0iHev37slK11cxbNGAC917MUiiCTvJTz9Ry85GSozvrzH89klTgpNsBwGGCQgoJhZTVSg1A9QTW7Zaqr+E6oQ9CV76GVIyiA3YnbGPlOpI\\\\\\\/113dfnZ51LiZKlQ68E\\\\\\\/qTm1B93lp0Adm8MA\\\\\\\/W4YcTY8t26GiNS+OTwsnk2sgOzlbwMOrVpmE6XCycJg2jeq15e+gmVDzx28soXfSbm4nKfKPkeQMlopD6oWqpKYW8B3Wy\\\\\\\/iNIftj4QabtVwDfRcNLxGypajCOexB99rr7e7TFUh\\\\\\\/163nuSW8cKOxFZFlzBF3RQys\\\\\\\/KvdNB68ZJHvqeJuMZDP\\\\\\\/sAkTYE6gtIazf6I4HqsPdbVQwi73tf5GTghFcWNesKvbe4gWTtFw3W6SsvOven4o6AdawMbGAY23EdvrHeONPJ9aO1My3U2bbjME3AeE3NJoVh3nB8SsmeooiPSWqlYDE457N96jNvhajOh6mKZeBjt\\\\\\\/upATEbp9N37FU0PTSRwAUwykIvtHMdhIhHYhAXzEdVVkkxXs8O7uRC7NJ\\\\\\\/k1zyHdZ179ZiLx3CKbc2EKNLYMk7AonIMSaax2r+9M+OGzWdRQGgKwJGy+\\\\\\\/bCCR8hqHSf9pml5J4Ssjp9VUN60uklwHtz7Uu8klWLPFS8ouBJUPLgNAbyYxTSUsR99RrHmBNt+wv9u7x2A7IKOCA9pBX1dz0HpAwg1Q+KfiPg9X6CiHN2enodTBwYd3VogoM3HV\\\\\\\/DyrqXhT6BRtx5NPDMgIgbkBCI6OQeUn5a9lhC7JPzWIT7sC0VlWSXO0uXf7b3UrWeb7TP88EgjsqO0SO1RaHNnzhuoFJ4AV1inbZl9P5KRRojjRTHqWQPBLsLm\\\\\\\/QlkBraJ6VLhLHnn7LI\\\\\\\/h9x7BTaNJH9OpbOel2sEmOzxfbq1ppEfGBJgJT5YY4qEfcvCE5CGtNVbRpSSzYJXXOQan3RHmC7gLLf7jNpeoFMfqZwv2Xc4N86uC+PE\\\\\\\/GMOw5J+YHzamD81XvPj3onVq5N8029XrrILhFUuDSG7gbI9iW\\\\\\\/4RgQEEUJnrzT+f98NAZF2kdh8VVzhdOYKqX7Gb8UBIRCD3K1UKc\\\\\\\/9+281E6tPGIQRpUmrHzR7gx10DSD2gfAbPzEJXx4xezc2kZiH7nLFOcj91qvWIBQTGuHtG6WcohNn8P0viXEv7KeP+tkU6YY3Zc\\\\\\\/GuezMhNmTX0I\\\\\\\/7LtboRfFyaLdzFTJ\\\\\\\/P9RFSvZHzM7Yb7ymBXinK0tCmSiOCDC636v0KrjMbKsUyiMSCPLqksOlhZKpsarF+xgHlrxIgRYWs7UmctXTmWYDpifUBD5xZniu7Ke7zWd7VKQaXElXhXsF5uIhTWX9uWBvVBlGkvxE\\\\\\\/L2\\\\\\\/cfFAxv5zkMXxlkVHV8rx5l2zX3BzqZT9s2hNWi7RKDSyLIAPrRRuGvUVF2j2EVjakbGXpAb0RdX8Z64j7RxXqj3cG9wYkEMJoK8jUqjbcj8+lBI\\\\\\\/cUZ4o3\\\\\\\/6OwAzM3MxjeR8hlqOatAaIPA2HgMFkgjOZnJf3OUCeo\\\\\\\/puzwkYe6oTgSWHu9HS8GnXfMoH9FjupaVPVn66Q7qpIyeUPwx19ThR96ZZUXkVopvyEHpIIB3frSQm5DnKiM2RiIHYGvotmqmUrXIuJyRC\\\\\\\/S6IS6mh37t94g0FDZDpMCbtwOl5XljI2BhkERRChBrgR8PWsy9sUAlIhNzW\\\\\\\/hMPSDDwGxMMyP02ebSTfDb4epzRVHmq46xH4wrjydrbdiMr8hth4w3vo4gSLOKy7Vfh\\\\\\\/gyklgm53Ra1OWfcpTjlEGGAaZFwoyWa5FI3O59zgL6d4A5cDXfoIvMajo0JpbOx15y7bnGZTXhztfDu+cAJTbVVbEgYv+c0AS35IWhKGyIYFQgdbnzzj3LvcHA6IgapfIN+bz5yRsyrVrY5NVsqhKnwHta\\\\\\\/5iR4CzpifJuPEN9Zybj7wxqMtdicmdDupuE4PUYlFYyQKxAY2jU5rlgBt7Ay\\\\\\\/roIQQNrmBWDOna8SiJitwmphqwvn6rCKipV1iyYnq6IxDNANeU2nJenxeL78FsbINRe8jg97yPlew5rK7P3VIStU0HSJR44V2ne+m6kD8rh8B3XgadPEETN5livWdJ26kMDu0AeP9FvxJG6RQpvTUMnIoOfCv+cl87\\\\\\\/YG9Tr8uk6tIqQMSsIAn2E4HKS1IGw3SFZx8RdeW+vmOBq5SktiuNk3bjatA1tWhhIbSCRn8qEF9onDwd5nRXPY0FL0irSiTdmexEpucniEfNowtFNh2ePD9wQiEhiBjwqVDOVmYlilWIQSzpT+5f\\\\\\\/TNZ8V6KRxUlQXBLqP1aJuyoUT8On3HJzme0Ex736kDbvEh+P1Mls82\\\\\\\/T6PkSgZdTNS9yCsEj6IsGd0VZmnGSlWOUTpSVA7SNQI1GEMY6YWcu5vwZRMtA4y3cJ5qO1cF7Svp89d8qms\\\\\\\/nyMtAPX3Kr8QHu2U0wxtB9rqkibmDexLrbcTqFhFMBYCD2L1X0lRzRUvOuV1Ykc1VMFjWNq+FBJuUuG\\\\\\\/xox\\\\\\\/pKyfkb8Il1BHLOmS0i0jCHoinXBoUM+w09NnSHAc8eTk21HcYb7P5SpGxUOyGLOjpI9KeWmIGlfXxLxMBOsAkyY75HRUXKs80xt2WW4nMKKkFyrL3oKV+pUEG0Fq0rotOxgUe0drFR2TkkLQ5sNLk9ItDW4RyMjgMvLGw+64vVxXpHbx1imXQX7WX4tDNKpTVzoMOnuAr+TwPuuqVsQ0WFeazCuT8JDTXCUg+4jEzV\\\\\\\/z\\\\\\\/0ReAwVjyuK9khfDCP9Hy40yS\\\\\\\/uEgExt4IrZ0lEHiemBNK7BKt7jUYh0LmC+0sMYsK2H8QMUIEN0kN5eNrpEmzRRho0pgcMTSrSyEdeU1GDim1ULsUUUpRF7I\\\\\\\/g9\\\\\\\/lw8Zu0euDwyQ3wdaq9TYStKrlhXThott0zVHrQxLrb59wQOFjDGy4\\\\\\\/9jFVyAOywBh4e13ae\\\\\\\/oCysXBcmNrwbp0LuLpL4aR+JFBrwpJnZgsjX0nyAxv09s\\\\\\\/WTW1ZqvTX+GYb9VPRIEnlhj6axy0MBthGj06U6fp0z3D44luL4uDYAPUrJmzBFIykGXi0CxuFuLUfKQb7ttFgu1p0cUiiQEidjSOHpTV0\\\\\\\/NPH58T1gjoR1+okPcmuBc0xxT3P0C7lZTh8HzM1hq7hjvW+xJ84f3CKUP9xWDgIBwXW5mGkpVjjXYZoVyx3QP0mXuwyHfdh9hqhG3hoWo+kJCycPcL1Cqh5Xet5kU0yPOsvZzVe8b1E7X7fZ42kFnUMmD16s\\\\\\\/Ib2pmnxyuSA2e+rqBj3Q67yUKimXzffufmI+AFZ+ay+wDDoaIE3wKBtSsnd1Fqlc5WXSbAtWZz693RRx7kjlU2bkKqsvjnB6p1RJtEQmSWauSlpL0tod4i+O1tgIfO5Uj5p\\\\\\\/TjpjW1L3i90j9X5vjQ1LWlUIE0FnIOZYfL5ggNLNUpeYA69fxLPcYP\\\\\\\/NwfuqtdJf8BwlI9fOa+TXV16PldLfioupd06pyBndw7K6Bu0hPmIXNu4uZjTF0QqmBjEVAXeSyFxkVP3OQj4f7xmpJ\\\\\\\/SLmDXsmdhwik9G81cz4L1g8kwZSlpzmt7zwKk0ljlNYpXFljwCjDB6MiDjRi2aS6Hwb0TJOcxKM1ng19bxpY0q5Rtc0wCc9qzaelcnk03+X1Nu5k2slk\\\\\\\/PK0xFc9IkW\\\\\\\/tB\\\\\\\/kkRwcOW7PXV66TfvmpSdORhS+9hQoA2QdFyklgah9AqsV6V7m2EZQdFmpT1SNH4deAIOxPoK6WtHFaKzVSixOmnYZLCTITdZnW5YdF+7EOUtrOYKZtMAyZmFygrsGAGeym7JLiMyK9x+sM+d7MSeg66hDHgq+r4Mv33DfjmRfzU3\\\\\\\/oxEZqKGEHALVWVrM1x9EcE2+BWcKW7wTExqvp9dmkvWqbESnIkYybrxvnQYjy0P2vcv60OiXvlLxfSe3VORuGpg4GVwIx+\\\\\\\/dYgtROc9mSbhFR8yehS8kvilQdPE3EWOQYQDtcisADCcqGVdcqiQHESmT7aq\\\\\\\/SY4gCBYFEJmq\\\\\\\/9K9JDR4Eqmng4BqU+5GbySmm0d0WZGU5A81iJVF\\\\\\\/xoivXMc9NjTgem94hYPo82H1PlhjHfqjcKzvF1hvGhoVe1O3ZvW9jsL3xuto5bmo+w2jJ+AqvnW7RhsRHV+wMp\\\\\\\/KsuXI0oNKxHbYxl9aYOAeWYoyyVOUOFo6CMSpB9VJiuBWopmfwkQ5zERRjyIFlvo9Pex0uSYqTQ95IUz31773zQiewhGITjuwLUX\\\\\\\/yvw9gBc4Bmkxig9iAxs+euDbcERwCj3iETmeoUfgzaLIyQSngjQ8SLpXRh8TW2e97MoZTQQiNZbOpS7VgD+w8BlPFrM1OdyIEsR4YEG6M48t\\\\\\\/NazZjQkrsNL15igDwp317XeaUYzVE5RfUeKciMmoxp5wDOyBap0TrmkZ3JfuzXaoWpersjMH1voOZyp25j4qATKJk5GlrfElKqdIeK2DOKAT8W6CDrEI6fLwFbxDltMeOD5egXCCi75bsQpjCCssUemPp9ydCjENvKE7iWE9f3f4TDOpTl+CtqFE3xxQtRvLtJHSVo6MLJmWB2QsPdVYe3bSliW5V6225hfAabtDjraUurSaN0ehPFXlHEpRB5tGnYD+JCE61t4C51UDzCeHdlI7d63fE8q8JMX3r2j1lGeYw2jbQhpCF8JUtugsPNOnKW+IaH9osZgXcfGXqKmdTZIo\\\\\\\/FE2Km+WiXWMSk6PD7nv6Y+dXf8q5pbPjv8BP1+vxbrnsBmt9SZhadnzOwp18lZo2lYfWLRGztm4wG4+1PqWgHA+ITXwplXOsGyXekL9n3ZGWDAOKMC1ORf+61gFXljyy3cAzbUB165onR7Eku38c\\\\\\\/gwTRSQibv4jgrHNG2CfFI5srvk5DTAhd90VsZImM4BSwDOwkosNKfB5mNQcnnzyEUOPVVCzHhpfCBJLjcBsFgVdLCm11ObwoN9hRJwXDxfPFVr+Un6bwyI0OWrztk5PDW2UgOIowPRzcSaubkmTvKGOkEyVFi9Xi1My0qk2atP2Px8ysx\\\\\\\/l6VJwTU+HfNkMb8kYou6cNttLUP4S9oxsW06nh+O4JyvRzW3+2mSbQhFkksPdCBb\\\\\\\/o59sxUF2aUJe3gfAnBAWZ+AVdqTrGZXnPkkq\\\\\\\/\\\\\\\/FSThqhEqaFDzNArig0y7kPF3pSJEfAjAEkjfmiuvgoHGYbOxRanVLeD9XEpfrwZIQ6iC4oTjEvrFDg8G4o2z38JjI2GCyXvNp2Vpt042zhX9Xu1vJLl+NphSR\\\\\\\/qICLtqOwbOSQ4FY2LhCBux0rJXVNkkUqJCIdfeuF6tK+jF641Fpazj4f6MpALutnHUs8JDmL\\\\\\\/aOW8MUKS5OTfw4M8Lrm9Yd6iQ+pdvZW3T4DHz+LDNfYgFj4+nvzE5zK5sLT8QWqIwi\\\\\\\/ro9JWrjaba1HX7JlQVTuc2RGbFjbDsXh2R6EIwUj8D7vy7\\\\\\\/DuusDT2B5cnOmvUcHEuvnHLHwaTLk2RCqDn0D7a9PsHmMCdHkYmbccrSCRcQ9XnAqaYuaZyAXhGfy\\\\\\\/\\\\\\\/5ij55i0IjCni73mvVDPvENSEZpUA5n898CfdC3WWuRJo6m27GCeppnC6sh\\\\\\\/E7VdVTxRDB8NZSrBhj3j4mmBRcm2sVP3WkPF\\\\\\\/syVj74k7J64hoKfhFkwL2EdagvU7Gf5lwBh9Nhe6unIww9DTy1yFUA3syUw91fXqwCNoN2KN87BpqYueEGtSySCpm2UA5NDMzzk8b337r9RcprifMkVAUdKLoWu6yyWqjIsxkzuz+MdATRsN2ALBKOYdcRje+2ufCkUC\\\\\\\/iKHBJYfeoRyJlVGaAJ\\\\\\\/fUulEj0TYadWN6FEzBaXfPExNJ1DY8ygwuXIbE90hf5VKKEnM\\\\\\\/NDKKNc4h1dhTtuN7mFjMRQ1YpRQu5BllK0p8FCyqagdQYT9gsKstvzQ36dQcKTOjR+bTW\\\\\\\/v5zi5gzP2pduM7i+73ZmHSSAzpdIszwT18SbOFcORY29Fc9iFZwDxOLQxXkH0aAB6sN9Wa1DNprGx144mmBxoXe+2CEMHAPxiGQMsndOlo+aE0CahDvVKsyIbJPejtgf3yymsCyGDM9vWtKS2QiBEPkoeDhfCk8j8geewHfxwxVU9I77xVbKNCNlEOQ1uVLRwSo70jJdVYT+Ny1aQCB\\\\\\\/jaxBBNTS6mGmV8ld9ef83R8jSK1dNy4GEx5V0qJSYXjSfHD5IS65sehdKWCKLF0iAM4c7foG1jseRQ\\\\\\\/ae435aaEk\\\\\\\/n3uzLUrHlmrjDT3CGaRAqZ1mTr0AOwI\\\\\\\/9+Jc\\\\\\\/i+8GHeGoEOdvBtHaBUhCbn21qh4zRlAX4ux3Z1oEBc9LHI2WzIVcT+DkXDeETeO9uGy5LkKQuQ8152\\\\\\\/AIMLjseroXZtbg\\\\\\\/\\\\\\\/zAzwmhzqYvQNCMdel4S9UV2PI4MtbsUJnGa6pjPvqWQI9PTNPgW3mpKu9ToVT\\\\\\\/s0EpsrwfRcq6hH9LWpmHgNDwLSh182BckyQJsVgeXAC4Gvg5csoPXGiMIZHFxON705JiSN6qNKjvFrIztmrms0srqdZCzfwgDq0gpvy\\\\\\\/NJlB5R4k8ct872U2JKUw7bS4SrMMzfTHTlQiYVe1GBgr4M3\\\\\\\/TtHXBPlPVsC3jalRZB\\\\\\\/IxtAtWMky47SE0k\\\\\\\/Ik5l5j\\\\\\\/YpndTxPQmO6VI7DAXUVhj4MYdlGAKPtB6gblL\\\\\\\/hSfZfvrJBzACDLHtUVCh9BvNMyj9z8wjF8UNHEzV0B6ocwFXQ1WJQQlsGUszqk8Eh+iVY4A7aiwqSRGCCmjo18UXcqIShgHopi43hVryjXgL9D71j\\\\\\\/+OqG0NGpRs+69zOzh0Xn8BHg1bgDjmy14cv9\\\\\\\/6+A2y3rjDzoaMeS3QHOO4IUFzstY7RbEHMXGD81Vj0mRPehXaVZ6u8DD4JVSBfZ9Ro3LI2wHodpIilrd\\\\\\\/U3tLUkbd+g2vn74BFinu9QlbCWfGdooEGlu\\\\\\\/cNx4fsVc3xlC1pJohfr4RlU3SywU7bpyb1QqbERfem8bT2oDOXM21cE+SjEoWMNaHoGFeF8qo6uo+XPz+lg90ERifMcWjcfn6xADZC3pKJylcfs7cpc41YEOkqneBFHw9+\\\\\\\/8jQI8KhZ+QWG7+y9f1ENmFrejl+SnIUVrsQVs8lHKpr9ewRLqQpVhnpRZw6a3cVUp4gpN6cc5l4XQNxUr0I155Z99wbcPlX+ZVKSdMk+mKLfagRLLHIbYlytvmSMOEtZpcc4CsMHcjdEKKk39iMK1od2LRg3CEmxqymTZPpIKS69lI3D9VcKwayrMgzr+AezboHxuavw2oiWJLYA0KSp4HmsDbwb9ftfbfYR3FmGhUjt30qZ4XP9hPreXYwGqXJt5O+duU9ZzOOUHz2Bdg\\\\\\\/fsvxc8GUHKDR3X+Sq1MzVryIMILyWWFdpmHTghC1saRRgZ4KCAMMlsYGbxLt9T3o0scqy4S0hz61LeBbNRDEBJN2pQKcba7nqPjGvAH3plbBCSkXD7vpXflpVCnk8gA0EC6YwXcf4w+ZKwcKCKj5Ns6fULh80dCJG5eLvdCmHtJWCs7RZjY52velwHMvXpDaLE8Qyyz4DfEcHAhRfKYNzwl9s3WiIhwr3VwTXIGD9svOj78nvRQkY3xgUAbx6llKw1K9cgn9uwmwdiMrAyS\\\\\\\/2koXZ+WbaSW\\\\\\\/FRrfofAmyNIF08MAovpinxyjZCzt5gNbNuloUk0s1M\\\\\\\/g4S0too0n5JpTMLhEYlb4pU5VpqgcP5yTfsDPjT98vvZY7Ka6NeoEz\\\\\\\/GmlYfHA1X5bASsgQ9iMzYqfTr2ZWrHzGCDH0HGWbXufljHYJjZvufgi\\\\\\\/VfYHp\\\\\\\/2\\\\\\\/2SddnfZdD5AE25PsnxrNUkvTvmWFiAz3bkK\\\\\\\/1Ei\\\\\\\/+QNsWQ52mZGui52zc\\\\\\\/28ikwv1sKRkXS1jherSJQSq0HIcb9m+cOyPEHnLOz5ugQeZqQe\\\\\\\/PLbUd+5k\\\\\\\/\\\\\\\/XOrgD3S3GsLE8LcYDORTmqhTfQl0+B6gZ+\\\\\\\/HCwbk7QuFVANmaTAFTvvuBJcBRvx5XLn\\\\\\\/RaRsvsDIhSPpjrLMeA0xv3nmVTXmLNzvdkE77Ra0SaKdYtHuJxOoD4s\\\\\\\/boZwCaLfmgd3mAuKhEwC+ImmWUvO0EbSqITTazIQynwwrxhtCTiF9BPj4uMlPQX12QmlMYGwaJBl9rSDQj5oTP63etPpM5C+CIro1MFfCnsazxZlZ84uteY4fzYltoHgRp2r0mYsbkXwZ0KfN7vQjmWyl8PgbCqae6w9fRJZT2dYpnFRMbzkKu6lCnVpqrivRnwlEb0PoRsA5sEn5BP\\\\\\\/UIXT86MlMZlxnoir0NTJwSm80AXMXlUwznSoBecy52fYDmWGvpcvtlJ1vtFZe7tQH51w140ByFWr8d\\\\\\\/CZy96EC2VuAiuhPEor8Ne92VPdRhdIaIW9\\\\\\\/5mIeUIPATKr4752a+t0+hL61fipy7Mp60\\\\\\\/x4OeB\\\\\\\/ilOaAPfck0QZ9Ty7ELiCF+qkIPlUq1NtTLwoDGKaQfeqTrfdOi89GoSOqUvnMd\\\\\\\/L4BTNOWtkXsZ1DZd7fcIC8n4ngDikXaegwp+rjKVzQnkaFHn3CcyXMk9vUliOHLH3QHD5zxDUrGp1hmF2qsIFklWss+ntO274geWN4LJo+Iuf55KcAH7nE1uQrNmOtebj+hyadTXCS8z6vQsPMeiJvZpF6CJR05\\\\\\\/KNAzTJJLjoT\\\\\\\/twWIfa8x4uOxuLJmtsPQdoun4XvrsDBJ+FaIR4Pk3ilaHNHAiE5kbzBsPG2LFioZxExikP9EgkonEj\\\\\\\/Q6hwpKK0Bt4q8vY7a0giZuaCCaavKnfpZRfOhr5NaWAF4ifilfGRskwaHngV6sRjIJy9L6ADvpqdqR+t59COAA0Qa+bAhQS3QRzqcsOEgrWy4D5x69LgFi3n4KWTOhUYQB9x3imDEiGjDzyyUaIkDHA8iMQt7M\\\\\\\/7a9ZAzod57WOsDJ38ICj6L0mdI0WIkWi7uae6Ar3ugfC9F4Hf7cr\\\\\\\/CHo3a717FMyNh+sP4OoKbH5mA2dzYYE1mNN+yqRLwXRkFjwbh9MoLrGFXm3orm33cctLBTy2gNDgrYUVrGJ7tOopAiQ0\\\\\\\/tZAqMvE2p1a1clAAQaukCplEsSzkDugnZqQ5iKyOuE\\\\\\\/bLvPGjwj1SxgnGfUD+CK1DgvzHi07jRRq0awENypY7oDu1\\\\\\\/dK3J4pU80HR555neuYhj9n9fdG4bYxZSUJ5ZptjN4HtJE8leBpNDaj6xOJ9Yzeze\\\\\\\/qVVYGam2ZEKZXw88Uuzf5Delbx0Uy8FzwcNrhymGs5K874mB38H1JELtD2qgNa+nrttdCzVxbaysGeDWYGTJpCEMoRQQNUmQBd8K3FfbXzaZ3slJKbzN3nodJQE5fUWnhwI6Z3qztat7osQb\\\\\\\/FncFLn9QGvjzQFoeVhgjt+zgi1M08HPGfsvyA1BDVA3WEGNPQr3KlrD1GE1sCFYFOCy0ptWbLTFvgPGLQybfCyy5lcIrgv0zhTOfMwnuA2N8HQ+J91fwj95Vb85Alm\\\\\\\/SS6+FXvR1G0uoRUGHKn24mgfhsIhYYSEfUQf5VZ+gChgaaAYBMxtKf6k0qRink90W9FqGvZsaZKOKgLgpKalYOP6V64uzWaCEjeR3irOfkY8OzW1luyOZad+7kKDgDk91GvVrdJrUMLHk0fEKNO6JqAyipucrw6MuROMGMy2BajrtQRyqB4vdsnyx1YaNdF2zli98A+lbNG5MrCkwSoiPkUEsMUlO4BGGDL+n2msJTJhUfXMTlXdCUOp6ZxAwqbnr7r+BVWK5Nj4BzKI0Jl6Flsb2TEe4B1sw\\\\\\\/uuGdegQg7i8p9jXc25QK3GNQHYeFUutKc6nGLRrgxI8q3atTMmeNpny8rp2Lbky9ZfM\\\\\\\/gxBIYgeigc1tiy05hAT7nti9jup0F1GHWEe9qhGtdnmXLoi+2NgPgINOPRCCzH7zArdXCcxcoWIJF55khz8MTnWYPrUtBboNlerWBpJ5ZnaslQxw35Ft3I1HWWk7F1fWTdqrnCRa0AfRWtaBRgZsXiaGz\\\\\\\/HjLhxQMsq2wdAfkk3orawz1MkcpRYowSWVgAOwLxuIXkc8VuGIGsWhI1hGCKAExlYnCKgcAGhRcaaLQtlPxPGajHatrnVc4rirENBkJpw+WIZMITjJ30lfPskfcj8OeKl+k6v4AHQnSEyntmpLZKWIErFiDgLEpIZVRo\\\\\\\/NBOPGqNifYJYInsHag8cks7UZsTUtbyJixvw3Qo4Q3FdT7dnsx58cX6c8drlgHOeOd3gEPkKKtLZ1UiCUamvGqrgKP9ZfrMA6QGME\\\\\\\/G8kBG48QiszKWjUsUhYol3dgx0Zao0ITF+sNVNkrBV0nHGmNJZuWMxtnn1tB1XCrUML\\\\\\\/\\\\\\\/TubH8en9BPzTLgMGxXDpNfWib22R+YWlhnYt6rTWBxzcKtvdeXSKnSLjbZHBFTTEx05jrvEwFcjhy+pIGhDZnK9igxRQQBQo23sg6J\\\\\\\/vOSYkA2EiXRvoYZXxk\\\\\\\/D24v\\\\\\\/JZEBlpBjcB6R0Ldq5fajcGq81Xnn9DEZv2zqY1qK8KB8zl5GQoaAOHvW0jlGqi74tRITLweT2joreRlhfea5\\\\\\\/DODQf7z4QLxRLWtpW3FC881nTmdlTM5eh3MOvOAEGne6o8KnwB1Xuq9\\\\\\\/NWjtChp8STsHqn+sDjIAzwQdm6SkRLepM6BxvLw6WaWk6EH0nD1tLM4bsoouPd26Nzlg2PHGZxoZu9TacY1tBw+uE8x77JVjQqghO2msaujG5lLR\\\\\\\/jsp6QbjSwXszO7d1wm0lJ2Rbs5xB5GuSc36xdT1RLtKxeuev8xWy9waFhHqNzfNgv09qvDjxpA43xlUh9zlvo99zU7zFen+RMosZOlibckIOctPStUyVuEZAvnHIBT\\\\\\\/C8J26QUI8L\\\\\\\/4yavuL89w5bgFrN7znymMn1oMToQb0WKMEnSPi3ymsm9EzxW5Ll5hhzzDfQ7lXcoW5UzN+y7YtPjZAHU0+5A1VdVREgyZaARsVfeCNpQQK8wwBxhMXRFhRk\\\\\\\/NOEkoTrrwweFy54GBpJUl8+3Ol4b4ue3M5mTmiAHBIjdkMK8qXy2t8FxuQuVFF\\\\\\\/M8SD8zWTdB3rna7U321KsaMhGMp+IKCmCnk1uKu6Nviy0ry4K2avQF10incW6kVQi50DGj3gcXQFw+WHXTo6\\\\\\\/IxD4Lc\\\\\\\/hIXOP7BOcjhWWfqiKwdMFvtKHPlVRbxOyn6wRKNKLI154xy+1IrZllz9drTq+NC8T8eaeGzmVvg3wZBaJp55rc\\\\\\\/QpUNDCYZ7I\\\\\\\/HyXeC0JOt5gHWQ4RJKmRo5VCaT4PZJYQsayZOUD6hGHw51cctqiEQkc44cf2p84KhMtXYWRk\\\\\\\/H1KT164UbUGBW1b5AePlUGtsXX6IsEV64VClG80D8xMz3zF58kgqdNwthC+3VRDjEWvEM5Bc4\\\\\\\/MkaIz2\\\\\\\/R\\\\\\\/kWFA8CBgN+NNrFu+cpciysiKSb0Uu0hpH9A3k9bei5+EnRGEAfxBjnfiFIuo2QroNx81XFpMu++DQAEBWChq2ZcFp5Yyq8+MsMt5ncaPEqgKXTB8Mb1Mv\\\\\\\/SwUReqo5zgJEXXzsNn4LYok2d4MptrZ9aw7ojnzZP4QZ0ncVkRERZhuz0Dcix1G9F+3r+lWz\\\\\\\/iJ+taZvISrtAFcVr\\\\\\\/jq6+RrdLv795wTWBmJ3IliLcnufV99K6eGlWE\\\\\\\/4kVZAvl6kmy8r8r5W\\\\\\\/HP8ZkPD3gbTP+UwrvHCjfy489f6STKlQhsJaucsWnfBv+y8xWdAje19HdTXwTsY8d2Kbk4k\\\\\\\/O6sx0H8f28pPeHsxQwNjvYYAbLaP5xgpSoQHy2XSr2c\\\\\\\/Ckn+G4O+oCs\\\\\\\/zZFd50F61kPUqzX6u3eQUdAEVHCQIUd3i7eIktsnFJ5sG8RtlHrQYyVmZ7EKf2xMLPpy6NZtmCIwQiBMWleFUQyu3dBqUI+OGvqEYR5Sb+BcgEjnQRSKFdP8yMiRQKzzPVRzqR2pidEgipA7NftFFVLrWYcoh20AJJj2alHVgr3RAtuBAB4NRl+TZQmdduSqZ1J9zGoDEaEXoYQdNMHQ+nIIzDuaBa\\\\\\\/uCwbaiiu9ZV+PTmeTa3+wk\\\\\\\/FONtc24Gfi\\\\\\\/wN3Skttx1STfVoTWNH5lsURadOQAqV9Yq8qlXzT7cZtvatQPJmoelYFD7T1b88LtSBlXzaDGttN0LhwCkx1VfiYYJtxHuI8debqoElFgd1PhhCxSIanykEUuaN7VSm+REGXgpyreNGDcSePd9KOteA0mH0mA89jduiKPFVlV0DHjY0I3\\\\\\\/nD3anYAtAvXQoCLDCoGCoF+J+dgRXTRSz4gkbWxWWUfIbAp+Rs9KEOaA4nbHvsWCrS8w7uUMMk4heKWuYFdTMXZH8+hlrr+WDvUtWsZg82I83FRjK3rzRKS1qULsaU2FfJYxaG6Z3DcO3NBuBhYwv3g4OEPsS2NFtiw5tZ\\\\\\\/riUTNC+U4qbWVkvwA0Ha4p41h9zIon8k6MVC0Uqi3iy66kKeIuiHr8QEngQt67f3L+tJbb+m7GYOBQ8n2kbnvn\\\\\\\/yMUegtSmA5aD+f\\\\\\\/xV\\\\\\\/Wmk\\\\\\\/DcamPAD5Kgxu6nPGLWO0R0XtN6PXae9ENFvyPDSNlMS0SsYg1dlXYy1lBkzSaqcJAMWTa6rKq2THnEW3nFjz4wQHPIJ7mXOqtxUZqpc6Dmnat5aqVejQH4AgKjdZ6eT5ZOMmlk7EcXNfWXael2u2+ndH5cqJ6lQCSUxiMBZN35GTy0z25vc1mds0\\\\\\\/mhogbwBISshUWPhjhbeGact92UCOJsH8FjdLTjzqojKwEv9QdHk6d59aai9TbYYg3WnEEZIcDGZBEttLImPTINSEykW17noCA6YtJDT3uKAyLRJbOITwZi6Ik9DZaBjknU0ht1EICB+zgGQlHYUhsl7riUf+WBAkPG+9FL4Vqh0mmSLee5Ss0C8CM1Vc4GJdmXzX5TXDQyCB4L9fg+M0EoEyHiu4jOLxeLWL+u0+NmuCXSwcXWTz0dVeU1sMhCeXJPr74XOJ9NL0\\\\\\\/1hMNlnmSS2+Vxf8FN8b+aukPCXZXq86tWp5pJWrAaAfjZXbNus\\\\\\\/qTHDIrgXTOx+dyVWwedeBIUj1TtmRZLE7peOhKYeycB2lSytU3u552KEP9qQi+7+PqeafcRK4nwumPAOBMmmYR64AcVdtzk2YznHSse5FA4Ro+jg5IqeG7Uj0id1eLRqs5F3ZKtEwLIhNSAIkMap6\\\\\\\/cnpgX4R8kbeEJ1bQL+\\\\\\\/YHxJAkvNeJyYDQ2pAReGxEb9HzDnt9Xea2WLuxL8XNwBy9n81iwhdO0noOp0UHPqsbEcOSEp099xr9XSl\\\\\\\/SWo+0cINGrtZXrq8oA1wgoXhmR0vnqrB9FVa4+Lt0j\\\\\\\/jeMQCgC0AZ0zYL9go29KBLeoFmMRUNWf4ppxuzlB7Bbzqk7kjv1AtIgGOvdNGBaUQ8pcykKcQFLOlnHabJvLOaHAZwSh4iiw1iPnhb0vEFrER4jrjpq8yBOGIGFAO4Dqe\\\\\\\/1VKtzZtEUzC4NcwPx72hFB3ExwQVPd\\\\\\\/0D3VH+Ox5+EzoVCcLLmX5Gz1wHFfjeE8boI35ZXkQd12K+dH7bTLLkzct1m4p9\\\\\\\/gDl\\\\\\\/7veo8kolLn2d8jbIchmbHEmxMPTuF5fT\\\\\\\/MLFa1kHl2+pAJrfaooGhnZzO+c1wGd6lKd0Es1a3E7CWGfegkKu25\\\\\\\/hFIB8ww32lXXz39HdFDrzTiwNrR487v3BNrFojUA6pR7YrHrKu6xG71kFa6m0p4a3ve8M6O\\\\\\\/nkACALmRnisPqPBfCYWMR6\\\\\\\/qZPFTG92TQx4OB\\\\\\\/zLuLSEyFzBmVrgFLmU6gylH\\\\\\\/PpxRRqI5iKpNST2\\\\\\\/RJaQ8SEe+tKfimbrn0nGbxTKom8FvFhqOxbRp6FLOtcNrU1OnyVwotuubxPYPnhreDRMIkn9XJe1xUYkv5kU1M\\\\\\\/gxiLxBgFKHlRaqnFJ050e3eLEJuMEbVUrkleE0SN\\\\\\\/PaBPGtJJOhG7KfKGilAOyCrdBo0Yj2yN97G2rVFvlBOq3QHxkVkjoZbSmgyIJ34byGO9KMWvntSZ6HTjaV+AeLqgTo1ytNPHz3hGVBfHR8yjnGPT2n1foniqZXPedWyaPBb9Y0NOUSwnnGomHrRY0cTPWetSJw24ZpjlUUTp0MGP8vKUM6P4DK3420R6TsDUrcw\\\\\\\/4owoWGlx6ZZaft1O\\\\\\\/2zBWVKrT7YmFugQNoVUaVD2A7cDWK77kF58qOMDbmSeJltldZ2e1drtg6CZEgrplUomyOQK2zSMonOEn4csUiEEBoOiCk79PNhx2T50vQho95CzNizUTRG63JMYeiGTxPf\\\\\\\/3fsnS7PziuMsdkpoWb4ZODi\\\\\\\/4hQo+LuxSwOAtUJW6Ujg+ynqoWIivJlo4EQlfOfPgb\\\\\\\/7TBwfLAiJYB+0V\\\\\\\/ngNkGoTDwowWuI\\\\\\\/MYlcGsfbJJCZgyfjl1Eg6TThVZCl096+LvhkhFDso2YmCzsCbTQIfWlC+7l8ibYLnAFnHOGTscmkR1y\\\\\\\/pe56eCvw73gbIhNua5SeX7TSQ+vhWajQ5Z8Pslcaesi\\\\\\\/531SFsLZDOkmhSWYfHiu3HIH5Jigehf2LlpuytZXfNDJBUBk4vyrZtDfJFhCaB4xw+vwjPiVAWTWQ3OTKe5l\\\\\\\/0rIdcHKsVWmryBKTN0fCVVwLWkh4+AQ\\\\\\\/STIkDnmrIPwQaGOq+JYRX3Wa0ICs2NKoOkP2wNykQZbG1kDKIdMSYyak35qi72t\\\\\\\/\\\\\\\/8Rd7YUPK\\\\\\\/nWppyJjJzbGd6f\\\\\\\/oly53FUt6b2hWBXsyMY9CRd9DKAunJ+u69zc7\\\\\\\/UbYY7glxLgzkL4XsOzG\\\\\\\/Yx1pG8pIvw9OpWdHIAHafgU\\\\\\\/76MEZGon5PO+qZx3Yw0IZA8io0LTRkptjsLc268wACp8Wws6jv1Av83LG\\\\\\\/RsLnJMZZfKiiQGMAu3RlsYAUGqEbZ6EmD+t+L+nkClA6LtuY6nDFK32401TRUyQlBt0ZZB\\\\\\\/7rYgbUD+GBe9MKFYG+b0kzRlrxf8SZitDQ4aUfYqYCG5T8TSlat5M7aE0tahAUvlHtMhe1omyHn5jat1eCNk9o5VI0YzmY5FSOg46\\\\\\\/NxjmufdxDxEARL8NSjWXU+ModqTM8A4Ewd7EXz7aMQb0RdAhaHwzlso07ryejSd6ICMBwC6wUMr50SDn12F8I3smUy7aV7Bc58ZRh12QwL5npqzhFLCTSEt3z9FKWMcLnVkDerqV0OoLdJvMrSiDXa0OdUTw8RKsKK9kMhddP6YeTkX1K67BEwQNpMtPd8UW2eyC5aDQ5V79j2QFs3nsxPlFOl01pyz4T7UPt5ne428368Od83yFvMmHTsKuls+4QW3LMZWHzq7XZe\\\\\\\/69Y26vJ+6pL3DDgv8iWe0N\\\\\\\/nzQrb014c1iqElJTmPlsEciZLR7r7Li5jrOe7G3Bw3sHP0xiwDTKXmyQdHfHLHjrz1D5Av4irDuzHtU5sn+MCKxqOmhXHWBDACX5TvbNZYBevIf7tQwwYK2RGXlIp\\\\\\\/fgGTqhEAh4xD+q1YVdnvTFC6I29WEOwyTyYVrZgYTko6tqcOZxnNbbyJErTm9AEInbMevVUna6veac8ZezCry\\\\\\\/hR4mXwhSRMslnvtMSTuWXNlKBjWo9EdsmPXFqY6kOiv3HwmYcgKaDFXu1In9FAE4Qin9L2CNibNO6H2RSRGj2XrJMEPP7vtVWa2OtYnShJpvx9Zny\\\\\\\/tN2B+pqd5lUHPfwP2iex2NqW+CxEhbdld\\\\\\\/S\\\\\\\/TDvtGmmp0q+PbGR3kHy6vpx1a0Hhx1WCJPKwRTOi3vcGg9vWtRoU2p57fLpTEk24N7jkAOz4eCbvG1nKNCudRx1ArVnM9njiBgBV0DJnTUEKNky2jVmxd6yU9xlWhbKT8Ryr1mblUTi5oob7ITQ83mUDUJxNoAcI4IobkJFSIK5Z\\\\\\\/Bihm8SDMby7PV0kcTXuEyD5rAIjOghvs6IO71KX5oYXrxpbxtQp8uzQwb12epNjwaXyOkS7tz0zbMKtOZwNbbBMGi\\\\\\\/p6M9EBu1k5lvISSlioMgXpUDDWKncLuHzM2CFbXE85C10jWfXMDw2RiEVS0cKhl7+486plx\\\\\\\/qFF7mzZ+DiT4XFUKWDaDUDVTgq+k23RwhAvpTFIjoQuFd3p5R\\\\\\\/Xo3CyuUaVh45DrZzcXNsIvRUq4sW\\\\\\\/kOq+BAqtUHTlUtnRd4\\\\\\\/mSePbvw3TABBIJFPANAuQ3OYsLe7pi9FVKSbsSbwetQ\\\\\\\/Zn+S\\\\\\\/WAodqY\\\\\\\/n4IInh3lx9wqEbyFgyZg2xWb91XlTi8M\\\\\\\/BxuhXyO+06NZs9FRqbtX7NHZ5F5eNQk3\\\\\\\/kg4WW4UhSNuyNt12dISH5jT194u4aVZlv6iLGRfSvr6DjZUTr\\\\\\\/U9PGBqGY\\\\\\\/Eh7arhzTiPyNW+kCnTXuxyDDnNLgVUXK47Zow6I0IETjtjuhn8eYka\\\\\\\/8GJ4818LkPgTeA5MOPYfhMGgl4bjLldoQhZvSGavIzoQZVjiLbPXkiGgdIN0WPHyElNfhkkghCBsC5nxRCZ7D9eaji\\\\\\\/7riUB0EvZnlPKv9ZqQZJjljnfmC3HFYMYEqj6GS5ldpN\\\\\\\/5ypMCaG0kP5tPqpkPDMN1eSnDtyZcssftrBI67adcER6iYNNgxFSBSJsTbfx5i0qYWpNHcRDqOIIO1xLzQP3A2Nmrl\\\\\\\/CRlAwZjJcyM4ARucpIWK1ith1Iuq6\\\\\\\/2UCzILAjITk3xGtGiJBHM071KtEeXJUCxVvMV51KcUNaIwTGginoL3rqDtf\\\\\\\/MzVPf+a4E3ZiHLHDyX2UaxJxLQxFxD6R21lN1pfXf3k7kjNLERn4\\\\\\\/3gVueR1QzJqb7FMB2ukvn6ZMgqplmtDiSWNIEMkI10H+duJBVoqk2UK30D6sy4bLciykcmWYVwUlQ2lKeJ8DXzyh3QRRU2YK4C1pWQefmHaK5maETissfMbi5\\\\\\\/YCMp1VxSEkNSzrH+zRORoxjn2EP2XprN6ZyFLO7+wNkIeZHbCZlvjgkdw3kodiUJMXd5M24LjltitihOraQr\\\\\\\/mlJtFDS85PtdqwP8gLPeyHNTdHhVQEX+NQNjiYR16EoWEtSdLY8QI86pw0XELlVtH8DqS\\\\\\\/xl6uPX8zuT5j2YqoGXwLiooE2g4mVMS3SWEo+RdNtyW\\\\\\\/+sYEw45oOyAtYkXO+Mbwq9ndu4Pd9WDpF2msvJo+JHKm3X\\\\\\\/2uDU6DSsPEuNdC5LoX6nQvCeg5EXxi4MRJZfEwf\\\\\\\/\\\\\\\/hASxSqCqzpOzNjpjtHzkl9xWmlPkam8ulWVCqanLfRZ\\\\\\\/gYFWlXGAcJIatG1lHVOEnzPTF0pg1AQdXI6TKnHgr1X2ZfR1HBLpGSDEfqniPa3dnqYcK2Xx0qw2FvDxBeWMYbuC+ZDFXRJJH4lQxDMIDM64iml5trB9QHAj4BPRtiM2Pmte6Nr3Y+B0P+2ARe8Tk4xb9SoBnKmBz94lMMuAza70VJLWwAwOV+IQ7msWk3BN8h3TQM1SeG8OzR1SoJGG8gvF6pvjLAPfAeGAeDRpzE1Cb2oPquzKMibJQFw2tjC+V\\\\\\\/1\\\\\\\/ysG3pZ62XUMRSKgRpDpeUeWpPnfUfK8IleNP3rU6VqdzaTCnyj1O8AnoFEMrkolK1007b80uuuo11MztAsDSc\\\\\\\/fkqsVC0pvlumPAxX5bzhej8ZN1dp+MEWV880qDeyReEke62s1PRW1wWCLnlCwL6syn+Vc64R8X9+26pF11ovM\\\\\\\/cVBQ3UVdiijaAOygMBki9w\\\\\\\/\\\\\\\/ph+q65I+6uZD+hxAwzzCXmVMYwjoex+fmVHaTXxL7CyQQrdJZc26sI6TWZwTP4lRvjasTl69thRjfbDLB2pagstw7Gz1DZbOf\\\\\\\/jov4V01iiLvTn\\\\\\\/cQw\\\\\\\/d4iyG8YhTSIGQ9ej3t0jwkZbeC+TngZUK\\\\\\\/CCPVavdHJs24CDCyYmHc1Dwrxu5EkRiatuBC0PVAL+9gt+7MB9dfhJHIuDtcFKf0AXnxJIBh\\\\\\\/Lfvzu6QpvhlRoZcpMarWuNFGmfgoIUjjDNC4KnyUhp36yHGJa2w9lhxqavtvEV2B\\\\\\\/WTbF\\\\\\\/E0PpZDie7Iwo6wx8pK+XOGb93b9xYrHc6DeBVQK3kWYRkJyNKJZOMAyx\\\\\\\/fc17MjRk5xjQuM\\\\\\\/SJ4av4n\\\\\\\/2f2l54NgjRXJCIuZIjRPyXR2qD224o1X1t9OcUOmSJWcU4HId2mEwmKufpum56SlH+1wZY0UbrQ+WRBRCPXLxNfBwv6Xj88EtTtz9h5INDivM0zobVQvmGgZGjLgqCOI\\\\\\\/ww1ZpwSNwaZnd5F3PoRvCv1mmreCEnr2DNt8Y5Mkv7NEDpCS3o35fML6XjlRNd0OunVoDUlYvedSBYeo+tTjChiLksvnaIhh1\\\\\\\/wUDCf6oHsuA\\\\\\\/6BzlgCbzx0QIycC1uVXr1AvCb8TVcTJ\\\\\\\/PT4extR+Wxdl2ev97u9bw6FxdYOR0QugpfFPpBHDVDtVPcgfbh+SJDXQAXZBUtJyZUMYiRvEgnJRELKE\\\\\\\/ahs7w9E6e5ccu6uUv3RbxmfvIJ5IMCZWjBL8DQcaIF5N7IoaLaozzzIGmI2ffuvZzKYa9VdhCjKWNpRIbm0rs7php+QNw45GNqfrkR9I49bn1o2OlxgSkNv0MHmB02nt+aHlHpepW94O+ZTTT6Os+LUVAr244GTPoiRchSr96I2ZCQj5ohZNfnu8NpdRK5ePc9Mc1eUA53ErFrHQc67TF1qP6RPzk9P8pPU193voagICJCqZRf9tk4y4zqrBscSmMSb30I4\\\\\\\/V899OfiNf5os+hTx2rl17lkGm2hVXm\\\\\\\/kgTs2OTryPhQdbAt\\\\\\\/gTKZeN2EtSxwh2Bd44Ck1fJqPcAVuIBZPMzMKvBWILeqgkMMPanrsdx6BbIgzXa07HN2\\\\\\\/Z3+CqV+WaUidTjZQvaSGcA41evVSMtUVdL+8L94a0iMLT44BJS9vhkQ\\\\\\\/Rg5nQPPPmPJTDSLnQ7RPrL7fbl+1dnV7IM\\\\\\\/C4KTwe8Za+M4mzUSSN5psZEhI\\\\\\\/bzQYDlVyzUrTw3z1GrXhThb+3\\\\\\\/TAhvh0CKVUaoSSUCevgTxB9uMOE0dibbSDLmuCjy35B9+OaQ3ctZogM9laMNhcGj\\\\\\\/cm5+qT6YoXlr9dyN81wHdjwum9oUl7jjS3IYEbnm2iI1MAmnbI98lXlwUvYPWM4b9qvso+0bQiaqlXuK2c6\\\\\\\/+2oY5v3Os1RniKjM32q90S\\\\\\\/eB8X7qgLxgcumHsVJ9sRPAaehWEXdd8zmVjECjezvagHqtBeiWz3tE\\\\\\\/aY7sqi3\\\\\\\/wKXyNVMJxC41RLJy+5ys+6SAH+U2n0ktg+F6jl3\\\\\\\/xdYkoIR896vuUxgwGXjT3PkQ0LKxMcuCmSOCMdEj6mQ\\\\\\\/uBk0mmMKYk7NVD2wqbmPTao9lB1osG1aokOIHMhbkaeVRG5bWfpsnRfHcKxlz4A\\\\\\\/dSiDFBvon1Uy7VD1pCXV8A4JtyU48+OYUPt2ALt94rVriOiZvAgS0GwMaSOWEcCpaAWHHYD6ACNytYjoFrYKSrCkYeOwZVZ5AxsbD\\\\\\\/OIMWQ3MaQ4gK8qR9G3iqf2A5zduYuAFVtyVaeEzhTv6ITHSsSjEqVlK5g0GgUcPKiAe2te7aKA9DrBkOtz90oI7id9Kmkt1wSkjNEiN2y\\\\\\\/8t7YcW52w3rRWjRzlja6QAdJTS4yvzafdd\\\\\\\/3kkSBlxWs1LM0GMSCh5kfGlFmJtFw6Pi7VAXIIYwkNEnTVd4MZBynKX+\\\\\\\/D1luz2ixTlzRV5BN2fq2iyq\\\\\\\/kO6A7wAxDxOfucQWJl5I+FBAWG1SWNrjoovfqOcERvtj41FZoORjrNEZ1b+w7tpf7BbFcTyFado3KdVSuUcn1tnPJUNViNpEveegP47H49V\\\\\\\/lCZ1oA3ElxhfVGQfT+kHopND+p9c5GPeUqpNtk+0TJRZG9wAbQOe3ZsoxT416bpj6rEipHj9iQ+J3pwnmUbwJ3ApVRPQKx3s087YdFxnecJfURFPdg9VUfgMIQz\\\\\\\/RGR8bvixm0AlUxqAKr8MS4J9r05nNvwUoSO576x\\\\\\\/AJq2LCkN37oRa7h1eanuNl\\\\\\\/hh3GffI6fy3Pc6JsJ+AICSCN6rrJWoHh9H25UqWHbQflIWjWtRGLWLmHLdjxalUn1v5AbWeK7+F8\\\\\\\/+8YiaVTL5gnMmr4ZYSsP24ARmskB8qGc+ZdQkRfo3LpmqffTPV2wBaytix8JFYPprM1ujM1bSR6vN8fq8Bdq1GIIqkST6MJx6dXCMEa4O+9pVMxuq1rXI7jKfR+8w2SOORviFZ6XvaMb2XgfCMyujg6VyVgJHC0k7j33oHFdjE8kxaJuLY5GOKcGw3\\\\\\\/zzyEwVTqKz7jPT1O+RiUDJ5Or\\\\\\\/KbBBDs0LXhDMkM7SZBay2\\\\\\\/01uXyjjeNCdqSMZE6wkD8U7IqiUP+YR\\\\\\\/tHm\\\\\\\/2DUVgbD16JuO1nz+4MA2o5zFNtAEdIQlZuggJ9VFGPEF9wULZQxNEo78D4easnsPm4OgRqxF\\\\\\\/I6TuHJhwrUqBTsaoAR31U1Bv3KZPLix2Y2gpCkEE1H5DDdk\\\\\\\/kFudgGVo3F4lAmRGrGHqxo8e1TlO9k4A4\\\\\\\/gxfXeglSkOpiRohNQA1UigETo429jVIZiY5wSd78Dg9ub6qlVjYYYLqBXrcxOdb3iAiJMFqG75jOiMnlMpV01fv0\\\\\\\/vj85rY08ULwYLozBH+46UtjULxfFmcnThg7Tq+uBtdqY5Guh9HtkkTrPTrBPnd4Tk\\\\\\\/5rOUkxuiRJFfVFpR48GkpsaC2iWzSgv\\\\\\\/gEQ0Ick1CKW3apALOOd81i7tJti1kAB9VK44dr+guyhisQBUzSShJ9y0\\\\\\\/GnVz7wgZ6YD5AJ4TTPeig1HFrXVPX0ifLo16JiQs1S38iKCwXXtwHoKYuLRBTDGJDuld1vmEFEgwRUuTm5g0vzc1qIAKcsL75+BNi+B4IBcmyy45R61Nf+UDBKbPI+3LqX2ICy0KdOjBOVm2bldWsSKpV6Ng8nS2NKiraC7vQ+Shs3J6WXSvtJ4LJc5\\\\\\\/yCEQW9XGTWIQhclJCGWUtJ8iU+wbaT9MUOhoV7rL3uyW13N\\\\\\\/2+OFToRPCU3uFhnagKHAEJS3OpwDP5bb3+ng6oz\\\\\\\/4e2VZXkLARwbFmqh697Fp4EiPx5HYk29kwatWQPtZ+rJdZL9WFIofURrsH3R2n4kRqj1+E+Yv4AIf1Tgtk+cRFLgIpbpngZrPiPBW6b7fTM7lSYmNE\\\\\\\/gs6wQn1KOmLfzeikvjlrPoz\\\\\\\/CRf2Kanq8\\\\\\\/U1gmKaIw3tuKkhjjxaq8RTTYiSMXkd81M336sztak6YjZyjFaHPOPuA4Dx2wJi5grCpHPyK34lXdoxnHI4vwElvGvN8JXU2uUT+M7XCmVAUELcK4j3vRDmbK9ARIy03D+UAk6a1OASddYd\\\\\\\/7yeauBND0Iinqob42rfA6v+a9JkvBZeDCz1+I544dpAYDKb2mOMM0N29B7xihjV+p4AtwAUie6BfWxvf7\\\\\\\/RdvI9c+2tlbmxGomsO1d1HUsStzNEQomDmr8qSHT74A7el6R+dkb4sL5XzHGvgyKVwggeAN3wKQk7nNjEqMGYmbPscri2V5LjgKfZIVnUGtxi\\\\\\\/BISHfJkNSg\\\\\\\/Fi5IcPwnyV0GT\\\\\\\/EvLudPTG9JY3mkCj3XyckQ5ko1EP49TDmw9vBD34WkqXULPsStyizeh1qzL9K5lHhIg480n1bPhIhljZ4HgtqmiyEJ8ncZLd9WXkatsZnnmdzKRQQyEsJvwTbn7XuOVCCe7itTl8lgcI6nYMygFl6hdC20N0o+YDT2RVNihlqlrs7jk\\\\\\\/HL8r7mZpNs0VtLeg8NzX0vYweObYyE76D7Doi2fozRP8ZYzTSMWh\\\\\\\/GbokRMWQBm9El8jGh3wC4oD7Oyrw8ZVeQfISjKqYbLGhwiks9h9aXv8iK+xHy3GDgbv\\\\\\\/q1FzyRhdG6AUiXVFGolNM0frSfvGd0RCbGLumn2h0Gq47spKKcNXH0015jHNDASHfHOZsNozTfsFfmRjc\\\\\\\/f88AiVxTYGzeutqSE++CnTxcyBmwbiNTUGY78xv7h9eJmr6PvtblOGymbJ3rNd5XxVZZ06lYzEmfDyOX3WHWmbW74ulTm2Ajx9isUpByia\\\\\\\/8PBBXB3fIPUqmKU6rjnQ7DahVeLNUu2Je8c+wYjvDPr93One6WkBRWFNSPg5ygzalhEbwNCyRl39Mczp2NqHD2pDcr5OMsRSFeoKdOyVgG0nJ22O+gJKwpGKwF4S\\\\\\\/wdAUJO1myQjWQMIZodfh9KZYkF+bdo+VT7+5Q6QqoG5LHnrDBrfmq2hmI5xxfV5WC0TGuYTqHoORI56l6WP5z7KhvIqi7oics1dw7J9eoayoKewVDuEByuxHQcoKMtk4qO5qWB76G60u8V664kdw\\\\\\\/py2sMfVSJQfyzSuVFBrWjJNXPMGOaOMi0ycnXdo8hAGiNvFBUUvWx9T3TgL40+QKLUpZj98o0eiXOp73zQs\\\\\\\/9a1rNIRB0BceZIj7S9CcdCJcYUJCVL0Vu3OKJN04klNx4EosF8JQ276mAANLcv97VrytHR0NGktNru0v0ojO5pEuZ34kub9LOsRiA0XITV3VDXjrxldGU8ju9vHkzv3QOjMIsP6AHj7wPWYKO2wn8cy3M4u4rk\\\\\\\/bGQ09ce38e3ktZOHCqEacMP+rysF2+kaIgR5Hzz5FvafPWCNvBTfGfiLLC3mAzgXlGj946qmRYB7TMT98HJf\\\\\\\/tcaJbg29fC0Z2F5eNyjpZ7b0tGmkpnaMLEC9c7POxAhDXxZZqxBJgkcIcuijCplwgBLsYCskFa6VcVmqtHeg5CkHyRPvYv0p5Z6r5NHlkdOiI0b6CJo+fxHqq+hjC3PTK6ipwUjkSD17gyYvm\\\\\\\/yzWjV5cKG\\\\\\\/dUkeDWLUeFKdo92c8GE\\\\\\\/5Db44sYHoIpcn9pvk94rgK21iP4j59txHQS0re0GNijZzNUgezepIfec8sPI3b7PHNos\\\\\\\/Ibd2l8beLNDtpZ5n8EWgZU5mvBl5FbI3CXqz+xuGLDl2VAoDgOVkeC22EN7H\\\\\\\/BwbBXgg7pbUucxm0mO8KJ1dz\\\\\\\/lKyH+jkAOxL5kf8LC6b5qEoO5zgUO6u8fnc6cUcjNYSMvCxGTT77X88ZlgmoPDqA9iKM6JD6kJPMMEq\\\\\\\/6Nk9e2fF74bOAOnWCChM9iP6quveIsWEAUe7ST4pai5b2VMi1S2zyzEXZcrddfJJRIYvWB\\\\\\\/T\\\\\\\/LtyzdZ6RQWRoEc81GT12hHTuJsplAC3op7E0WkuMBIrux1gHIDlmmtopmGU1mVT6TfYEyhCq0xUN2FiSPJDsvVKVkGQMBUtlnPfDQ0aL02H4+7C40\\\\\\\/I0wcyQf5wHtcaxlNLamQCAE+BF+DnLWKyfSXYZv2JnQ1XC9wvyUUzZJW8\\\\\\\/t5CTbkW1v875X8DZ6wvwzAqj3FW9kZZ1wKCo0lb3wUVz0zyWH4bxLS4YdhG7iFknajlg8ao29\\\\\\\/k+00H+jfxhm1DAKPasXKg54WN8zewSphz9Ugm5I7T7n4JBK+U9PAZ\\\\\\\/u2Y5N3z4z4prJciDcZMSM3Jb0b\\\\\\\/4vRN3XkVKJl5zxmxlt5tVhoHJVJJWXF7j4ZoQYd92dZd201\\\\\\\/yuoDABEIzc16HauRtkollKJAfI5Yucpf6RelPTBunM1OQyqs0wRO2u34k75EwSYVnxG3+ccEpCfGQmYRPVa729perACXrCOIpJNCOOH3RRZ4wiPI1Tuk\\\\\\\/DqKjS1qHud+emwIejZFEPlU6lTHZy7vxlvHRrCHbl+TjaUBugtj2g9Ps7bX7kIjvRoUzyg0ZeLCqTmVESrmndg4CLrePwdlTZIVpd1PuzNf3UMKrItH48a9\\\\\\\/05ra4+Ayowc\\\\\\\/Vr0JgcP0+3Km+lf50ClWrggmWbUipHZWB01KlYEkwllEIU2X8ZRAVGrwG59PNALD5kVysjj8XFl8yGIEhx4mkYe9KEePGd1j7JDn3qL3np3wX5F4TH6gv4pAGdzK0N7Eiv6OuEm5dgl6rgiWvaipQ7of47F3SwWCfixvkYJXB6nEkql3l7a6vDtSnwuXu9hLf1S9+W\\\\\\\/\\\\\\\/Ccd4HuiQNzKGXRP\\\\\\\/MQ7WCRWiSpDNWi06S2catdteaHYBjnn6+wpYyIraKgYdH8Jh5ZRsyUecqS9PMC4JcGql6eXpDiqbgH37IaejZr7igy47t1sjnLm4h25lBXVKlK+cazel9Aq6ov7uLlauBfmHG3MSAM1VsS42kEWYRTMzBaiWxxVjBKExnF8\\\\\\\/syrXCtjaQGTp6lAXyezn\\\\\\\/gBStR\\\\\\\/olgQEyyuY5LcA\\\\\\\/ZEVRREGnA3Zyq1gEii38FeswkyHlrYCk4J+4EOf5SqqoEVFetmKb5HQLhb4j++5ZBGgyCezq31lEtloLs\\\\\\\/bZewKcT\\\\\\\/Nl+MvawBkLpg39AInnrXy2d4Nrkbv1t+Pj3S7VdJX0nYWnzSQZoyCdlYQEw15sK2qi2Edqmhy\\\\\\\/pyp+HLvRHHfBv2WyRybNMXZO7nKrc2SfMv0IlP5aUgNce0EfinB6WfcXOxNUSKYp8r+gyGRy44BOmS39jYqpLrlv6P9xZCtxT67K5MJIwWT0uSH52NrAQSEIEvTw7qgK3JUVNqD7ED571ytazWspH5ghzEMs3YCgO\\\\\\\/19k1ti\\\\\\\/MrbZKJ3uavUGUpyxoq0WoP1yzMtNPmJJCtLE6YBQaSWnXHnbzBKgu0yLxqRM2EhLM09W6irhq5Kx7xxw6\\\\\\\/MpAk\\\\\\\/ELap47HXlrayFerFH64l8esYKCsGSDlju\\\\\\\/I3peZTVWvwJwIEhMYrsZRlcyvu3JuALusdhtTr1DHJIB4mcErrLi9OKT90PCANiVeKhpDUs9G2lasYRkzBq9mQDySOnBgxrQgRUYdbQ2IOaRFRzxXC5BfN+tXUisTPfxyqzcOpmbGCPkcSSRaQ2MTURdbbEOR8RzDjuTEbFAV0DDTT1erHzDjtsSyd09wqqr0uBpGGAHonaxlryMZ4hFa74Rc3worYRbc+nA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pe6nk2BnaADXhQF9qg5xxA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a004895cbe09d0be9779ad50516fb972f353ea6e8bf76fc84eda1a3ddd8f8e6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eto95ByxKWmNbdPLTnC5d27kjMYZRNPpKQkrYbWK7jKhx6KXPdvbWOCoFvVLcp+PYrvDsZ3UQsqOUgZD8DNy7bhjnuRQa\\\\\\\/1rng9jKMYKPpEev6N3JiW1Y0b1I3DYZeiYsIQ76\\\\\\\/APTupVuRlmxWp\\\\\\\/Trh9Rbh\\\\\\\/NwBoaH6HmEEwUswJapssG\\\\\\\/UmZ55Mx2Li4jK6fEPfSXjyhmO1atT2daWjS3AzYp8oCZcIu8JPFtZeps5w+OCSDPWCSkw\\\\\\\/\\\\\\\/cZTCaFnfYzn8PK0D1Nnh2J5MFSSTPuXjLHFf4H\\\\\\\/bOTPAtt4ISz90kaGW6WRMnTKyFU8tM\\\\\\\/ckc29DJPj69icTurIEUpNEgO9Ry2DfGylRsMzvqA9\\\\\\\/Yh+XM2Ym0YBQXZjK2awPCGcgRrdM6pfGe+ZN6\\\\\\\/CYqETrnvcr7aKaXvjvltCn22KG8DmDpBqw3idmbTbbrG6RMgKoc9ICXv3hxiqLHNk3Au\\\\\\\/Fjk1dt9AR5vlEDI6uRz2wJpmpAZLaekBgRl527upvRpFjDDL8SF5ymtT+oOmgcVYuXEVJjdSs81+TJgUmyoGWMCSaG1Ws97IV45msEhuWLQhgAPz2BKiLuki+Keuq8CyQBG9BV+BZZbZ+gUuQnT\\\\\\\/Z8ok\\\\\\\/61phtJo2LaCVcX28jjsp7uepgJgMyxTR8vFJq\\\\\\\/My\\\\\\\/z1GToWcw072Gl90tN4swfQ5yYAwFiKNJY6IP9B1kO66wooPCrdDHW+nGv3ivUFKjCnsT5ZNMvTWRIWCmBV6MGLeTMjc4VkDCEyH3fslTGOKHUbiKCeBAXNHECnLohUTWNaDH7XUILSZNMm1uffkgECJRL0hHObHI8v9UC8y4ScJ1N+pC0CSBObSPjB7V3FFI8No+ADkhMWQMJEZdyPlJYtHFYN8paHlLQ56esNoS0apHBr3bAZPQSJG4dKxqmAJEG3fJpKxzvHTR2uO4eQHlXoDbuNvRaL0sNWCBsGzJrNVZOdQywDhd7ZroYObugnPO4npGdIc2QnCarUAY1ddC+04fkrKBJ7SFDcXK\\\\\\\/oKqih7ilPrJtJx5A+7Qft++9S8veLLteqvKORQGcftZVXGsYiukRnpn\\\\\\\/ntpkoo9BjUvsNcG6H0\\\\\\\/9AWGy5UDTvwwog5T2WWrcjeYNeKAuYYHECKIABpyPc9uCi6Zcg4jLK18cpazqhymO0N6XsX72XFipBuS9k+a5nFK4Xw7OuEBA9dVH2kzc0ailEfmStE1x\\\\\\\/8DZHnXDUT\\\\\\\/fQiIpm7\\\\\\\/YGM3tYLd3WATCaEC2q128TVNBGeVWx4\\\\\\\/6YGq1IsKEgJgFKF7KUG13tMk+TWATN6MIsMPV0v11oV\\\\\\\/9TxCrsytDlLK0G4ky5gPYnI7AqNFpr92d4nwI\\\\\\\/WsXztJ0K84OYS74CeQ18jZNEo4gt9U5QIvni1zg5ExswTj3gJcNOJbLzGcYHjy4nXF\\\\\\\/OcqStsbgCy5iDWf+YH6SL8ej+KJXZUIOZORLpreO7Es42ZeWcHofe2B0naE\\\\\\\/461bGAKMNxG\\\\\\\/9IwO6lFu5zrkfQ\\\\\\\/UFq7EgBpBSXnuTXYeOw6k97ERKNZmWcVYodHU\\\\\\\/8GxTRG3c3n14BUjNYQhQY\\\\\\\/mZPf3ZMDpzTDdBV4sMLFmYpwMUYX3k18zIHPZ8CqI8Osi0t8zbydbz06TO5kcebh5RGoqZibay+CYMX86PHc9KhX35IaUkftseiOwIuYZnYrkXACivKTYtKGgop3BIm3XwvaZs\\\\\\\/PqShqg4rcmRmlWVeMq6yhSamyuc+0ZWmFWQI0Aezcr+UzmzGi5YsZ9XL7I2rlGxsXEe0WplEi86Q91iowkB8EUA+dLLThREmR+o3shp4e02BlmSwQIpgFGPeSAQxb4gjzykHtl79W6z0CYeCHL2kr54mSBMyL766mOPCx7tdiOEQ5H8HB\\\\\\\/BbPbZuUuFxcFh95Wtg+iYNkXBK3IzCELjYzXo4vk\\\\\\\/sAfH2wFSOOdjKCcvSR2S+e\\\\\\\/F\\\\\\\/e1mGs8AoIfIgsjnIBYO6YFbnU67JZSuj7aUxI9fXPLhzcMPDrcgHmXOh95Xw9pYsOXWQvbRHRu1mB7IPnXkouBwmwthbMubhgtfub6zGiI\\\\\\\/9XOl\\\u0022,\\\u0022iv\\\u0022:\\\u0022O\\\\\\\/ECOp63e2QtNqOc+VLHtw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002247ab6510ffa68ca4e6e2be91bcfbeb0279854ffafae6d840370217d94a28ad13\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459956 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1223" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-V74dxmJ1bLKR30MCZqKNuecSlE9gmeNUQeuriHvUzYk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1223" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W0onLYBF2462sTfjohY1P4RdiA5p1jPGuRHnkp7kfFE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1211" title="The Mosaic Mutants of Cucumber: A Method to Produce Knock-Downs of Mitochondrial Transcripts" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1233" title="Functional Analysis of Developmentally Regulated Genes chs7 and sec22 in the Ascomycete Sordaria macrospora" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459956" id="node459956" data-pisa="ggg;5/6/1223" data-pisa-master="ggg;g3.115.017830" data-apath="/ggg/5/6/1223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Judice L. Y. Koh</span>, <span class="highwire-citation-author" data-delta="1">Yolanda T. Chong</span>, <span class="highwire-citation-author" data-delta="2">Helena Friesen</span>, <span class="highwire-citation-author" data-delta="3">Alan Moses</span>, <span class="highwire-citation-author" data-delta="4">Charles Boone</span>, <span class="highwire-citation-author" data-delta="5">Brenda J. Andrews</span> and <span class="highwire-citation-author" data-delta="6">Jason Moffat</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1223-1232;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017830 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459956" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Judice L. Y. Koh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judice%2BL.%2BY.%2BKoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koh%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudice%2BL.%2BY.%2BKoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yolanda T. Chong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yolanda%2BT.%2BChong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chong%20YT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYolanda%2BT.%2BChong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Helena Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helena%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelena%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alan Moses </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cell & Systems Biology, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BMoses%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moses%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BMoses%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brenda J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jason Moffat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BMoffat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moffat%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BMoffat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1223" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459956" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459956/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1223.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459956" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459956/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1223.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459956" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459956/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1223.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459956" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459956/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Changes in protein subcellular localization and abundance are central to biological regulation in eukaryotic cells. Quantitative measures of protein dynamics <em>in vivo</em> are therefore highly useful for elucidating specific regulatory pathways. Using a combinatorial approach of yeast synthetic genetic array technology, high-content screening, and machine learning classifiers, we developed an automated platform to characterize protein localization and abundance patterns from images of log phase cells from the open-reading frame−green fluorescent protein collection in the budding yeast, <em>Saccharomyces cerevisiae</em>. For each protein, we produced quantitative profiles of localization scores for 16 subcellular compartments at single-cell resolution to trace proteome-wide relocalization in conditions over time. We generated a collection of ∼300,000 micrographs, comprising more than 20 million cells and ∼9 billion quantitative measurements. The images depict the localization and abundance dynamics of more than 4000 proteins under two chemical treatments and in a selected mutant background. Here, we describe CYCLoPs (<span class="underline">C</span>ollection of <span class="underline">Y</span>east <span class="underline">C</span>ells <span class="underline">Lo</span>calization <span class="underline">P</span>attern<span class="underline">s</span>), a web database resource that provides a central platform for housing and analyzing our yeast proteome dynamics datasets at the single cell level. CYCLoPs version 1.0 is available at <a href="http://cyclops.ccbr.utoronto.ca">http://cyclops.ccbr.utoronto.ca</a>. CYCLoPs will provide a valuable resource for the yeast and eukaryotic cell biology communities and will be updated as new experiments become available.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gfp" class="hw-term hw-article-keyword hw-article-keyword-gfp" rel="nofollow">GFP</a></li><li class="kwd"><a href="/keyword/subcellular-localization" class="hw-term hw-article-keyword hw-article-keyword-subcellular-localization" rel="nofollow">subcellular localization</a></li><li class="kwd"><a href="/keyword/abundance" class="hw-term hw-article-keyword hw-article-keyword-abundance" rel="nofollow">abundance</a></li><li class="kwd"><a href="/keyword/images" class="hw-term hw-article-keyword hw-article-keyword-images" rel="nofollow">images</a></li><li class="kwd"><a href="/keyword/microscopy" class="hw-term hw-article-keyword hw-article-keyword-microscopy" rel="nofollow">microscopy</a></li></ul><p id="p-6">During the past decade, proteome-wide screens in a variety of experimental systems have begun to elucidate how protein networks are organized in eukaryotic cells (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Huh <em>et al.</em> 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Newman <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Nagaraj <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kulak <em>et al.</em> 2014</a>). We have developed a method for integrating systematic genetics, high-throughput microscopy, image analysis and pattern classification into an automated data acquisition and analysis platform for cell biological screens in budding yeast (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>). Our screening pipeline makes use of the yeast GFP (green fluorescent protein) collection, which consists of a series of haploid yeast strains in which each open-reading frame (ORF) is individually tagged, generating a full-length protein with a COOH-terminus GFP fusion, whose expression is driven by the endogenous ORF promoter (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Huh <em>et al.</em> 2003</a>). We worked with the set of 4144 strains from the original collection previously annotated as having a visible GFP signal and representing ∼71% of the yeast proteome. We used this collection to measure the subcellular localization and abundance of yeast proteins at the single-cell level in several conditions in time courses of up to 11 hr (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>).</p><p id="p-7">A number of existing databases present images of yeast cells from large-scale studies. Some of these studies assess phenotypes associated with analysis of a small number of morphologic characteristics or markers in a collection of mutants. Databases that house this type of data include SCMD (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Saito <em>et al.</em> 2004</a>) and PhenoM (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jin <em>et al.</em> 2012</a>). Other databases present images of a collection of GFP (or otherwise)-tagged proteins in one or a few genetic backgrounds or conditions. Examples of this type include the Yeast GFP Fusion Localization Database, YGFP (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Huh <em>et al.</em> 2003</a>), the Yeast Protein Localization Database, YPL (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kals <em>et al.</em> 2005</a>), Organelle DB (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wiwatwattana <em>et al.</em> 2007</a>), the Yeast Resource Center, YRC (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Riffle and Davis 2010</a>), the Localization and Quantitation Atlas of the Yeast Proteome, LOQATE (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Breker <em>et al.</em> 2013</a>), and Cellbase (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dénervaud <em>et al.</em> 2013</a>). Several of these databases present visually annotated protein localizations together with the images (YGFP, YPL, LOQATE), two quantify protein abundance (LOQATE, Cellbase), and one assesses the probability of each cell displaying any mixture of six main spatial patterns (Cellbase); however, none of them computationally defines a localization for each GFP protein.</p><p id="p-8">To enable easy access of our image compendium of subcellular localization and abundance profiles to the research community, we developed a web-accessible database called CYCLoPs (<span class="underline">C</span>ollection of <span class="underline">Y</span>east <span class="underline">C</span>ells and <span class="underline">Lo</span>calization <span class="underline">P</span>attern<span class="underline">s</span>) that allows retrieval and visualization of yeast cell images and permits queries of the subcellular localization and abundance profiles of the yeast proteome for each genetic or chemical perturbation in our survey. CYCLoPs currently contains a total of 330,248 images from three wild-type screens, three screens with a strain deleted for the gene encoding the conserved lysine deacetylase <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005274">Rpd3</a>, and time courses of two chemical treatments (hydroxyurea and rapamycin; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). CYCLoPs differs from existing databases in a number of ways: (1) whereas other databases provide searchable localization assignments for proteins that had been assessed visually, CYCLoPs contains computationally derived quantitative localization and abundance profiles; (2) CYCLoPs provides a searchable web graphical interface for proteins with localization and/or abundance changes of interest, which reflects the proteome flux in response to varying environmental cues and genetic backgrounds; (3) the subcellular localization data hosted on CYCLoPs were determined directly from the morphologic features of the cells and accommodate the reality that many proteins localize to multiple locations; and (4) CYCLoPs provides localization and abundance profiles for individual cells screened, thus enabling analysis at the single-cell level.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466312/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466312/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary statistics for 18 cell biological screens whose results are housed in CYCLoPs</span><div class="sb-div caption-clear"></div></div></div><div class="section results-discussion" id="sec-1"><h2 class="">Results and Discussion</h2><div id="sec-2" class="subsection"><h3>Microscopy data acquisition and analysis</h3><p id="p-9">Details of the experimental approach are described in <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> (2015)</a>. In summary, the yeast synthetic genetic array protocol (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Tong <em>et al.</em> 2001</a>) was coupled with a high-content microscopy platform to image an arrayed collection of 4144 arrayed strains carrying a C-terminal fusion of GFP to each ORF (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Huh <em>et al.</em> 2003</a>) and expressing a tdTomato fluorescent protein from the constitutive <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003725">RPL39</a></em> promoter. The tdTomato protein is localized to the cytoplasm and allows identification of cell boundaries during automated imaging. Micrographs were acquired using a high-throughput spinning-disc confocal microscope (Opera; PerkinElmer). Eight images were acquired from each strain, four in the red channel and four in the green channel, and analyzed via the CellProfiler, version 5811 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carpenter <em>et al.</em> 2006</a>). On average, 84 cells were captured from each micrograph; between 900,000 and 2.4 million cells were segmented from each experiment, translating to more than 13 billion numerical cell-level image measurements, which were stored in the database. For each protein, the four GFP and four red fluorescent protein (RFP) micrographs, along with the corresponding overlay GFP-RFP images, are available for visualization and download through CYCLoPs as lower resolution JPEG files.</p></div><div id="sec-3" class="subsection"><h3>Quantitative scoring of protein abundance</h3><p id="p-10">Protein abundance was extrapolated from the mean GFP intensity. For each cell, we calculated the ratio of integrated GFP intensity measured within the area defined by the segmented cell boundary, divided by the segmented area of the cell. The mean GFP intensity (<em>I</em><sub>g</sub>) of the protein was taken as the arithmetic mean of these ratios. The <em>I</em><sub>g</sub> measurements from our wild-type screen were highly correlated with protein abundance measurements from other techniques, namely flow cytometry, western blot analysis, and mass spectrometry (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>). For each strain, protein abundance changes (∂PL) were calculated as the fold-change, <em>i.e.</em>, <em>I</em><sub>g</sub> in the presence of treatment/mutant over <em>I</em><sub>g</sub> in wild type. The <em>I</em><sub>g</sub> and ∂PL values for every protein in all conditions are readily searchable in CYCLoPs.</p></div><div id="sec-4" class="subsection"><h3>Constructing the ensemble classifiers for quantification of subcellular localizations at single-cell level</h3><p id="p-11">Previous studies have shown that combining decisions from multiple computational classifier instances—a so-called “ensemble” strategy—can improve the predictive accuracy of the classification (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gashler <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Rokach 2010</a>). The ensemble approach is particularly useful in boosting the performance of weak learners and has been used in recent genomic studies (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2011</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Reboiro-Jato <em>et al.</em> 2013</a>). We constructed an ensemble of classifiers—ensLOC—to accurately assign each yeast cell to one or more of 16 pre-defined subcellular localization classes based on its morphological features. Our approach allows proteins to localize to more than one compartment or to remain unclassified, based on our 16 predefined morphologic classes. For every segmented cell in the compendium, ensLOC generated a 16-element vector, where each element is an independent assessment of the cell’s membership in a localization class.</p><p id="p-12">The ensLOC framework comprises several steps (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We first segmented the cells from micrographs obtained from our wild-type screen. A total of 430 image features, including area, shape, intensity, texture, and Zernike moments (projections of image functions based on a set of orthogonal Zernike polynomials; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Teague 1979</a>) were extracted from the segmented cells. For each classifier, we used the χ<sup>2</sup> test of independence (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Liu and Setiono 1995</a>) to select features that best discriminated the positive from the negative training instances. The filtered features were then used as input to construct the linear Support Vector Machine classifier (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Platt 1998</a>). Seventy thousand instances of cell images representative of the morphological signatures of 16 subcellular localizations were handpicked. The distribution of these training instances is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. </p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the ensLOC framework for quantifying subcellular localization of yeast proteins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1354212082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the ensLOC framework for quantifying subcellular localization of yeast proteins.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1223/F1.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1223/F1.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466226" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Overview of the ensLOC framework for quantifying subcellular localization of yeast proteins.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram illustrating the ensemble of 60 binary classifiers for protein localization and quantification (modified from Chong et al. 2015). Only cell images that were not filtered by the quality-control classifiers for dead cells and &#x201C;ghost&#x201D; objects were further classified. All cells were first classified into different cell-cycle stages using the unbudded and budded classifiers. The rest of the ensemble is organized into 20 subgroups. For some classifier groups, e.g., Cortical Patches and Cell Periphery, budded and unbudded cells were separately tested. The results from each subgroup of binary classifiers e.g., CoP, CoP-Mito, and CoP-Cyto, were consolidated with Bagging. The circles denote the percentage of 1,057,871 cells in the wild-type WT1 experiment that were assigned to each localization class, with darker green indicating a greater percentage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1354212082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagram illustrating the ensemble of 60 binary classifiers for protein localization and quantification (modified from Chong &lt;em&gt;et al.&lt;/em&gt; 2015). Only cell images that were not filtered by the quality-control classifiers for dead cells and &#x201C;ghost&#x201D; objects were further classified. All cells were first classified into different cell-cycle stages using the unbudded and budded classifiers. The rest of the ensemble is organized into 20 subgroups. For some classifier groups, &lt;em&gt;e.g.&lt;/em&gt;, Cortical Patches and Cell Periphery, budded and unbudded cells were separately tested. The results from each subgroup of binary classifiers &lt;em&gt;e.g.&lt;/em&gt;, CoP, CoP-Mito, and CoP-Cyto, were consolidated with Bagging. The circles denote the percentage of 1,057,871 cells in the wild-type WT1 experiment that were assigned to each localization class, with darker green indicating a greater percentage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1223/F2.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1223/F2.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466220" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Diagram illustrating the ensemble of 60 binary classifiers for protein localization and quantification (modified from <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>). Only cell images that were not filtered by the quality-control classifiers for dead cells and “ghost” objects were further classified. All cells were first classified into different cell-cycle stages using the unbudded and budded classifiers. The rest of the ensemble is organized into 20 subgroups. For some classifier groups, <em>e.g.</em>, Cortical Patches and Cell Periphery, budded and unbudded cells were separately tested. The results from each subgroup of binary classifiers <em>e.g.</em>, CoP, CoP-Mito, and CoP-Cyto, were consolidated with Bagging. The circles denote the percentage of 1,057,871 cells in the wild-type WT1 experiment that were assigned to each localization class, with darker green indicating a greater percentage.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466264/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466264/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The 60 binary classifiers used in the ensLOC framework</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">Predictions for each subcellular localization class were obtained through combining the predicted results of a set of binary classifiers (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). A binary classifier classifies elements of a given test set into only two groups. For example, a cell is assigned to the endoplasm reticulum (ER) class if it is assigned to ER in the ER-all, ER-Cytoplasm and ER-Vacuole/Vacuolar Membrane, and ER-Cell Periphery binary classifiers (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Our objective with this approach was to reduce misclassifications among subcellular localization classes with overlapping morphological patterns. In addition, an improvement of predictive accuracy was achieved through bootstrap aggregation (bagging) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Breiman 1996</a>), specifically by plurality voting. We generated 25 bag classifiers, each from 1000 random training instances (500 positive and 500 negative samples) with replacements. Decisions for the localization assignments of a binary classification step were reconciled from these 25 bag classifiers. Thus, in total the localization assignment for a segmented cell was determined through plurality voting of more than 1000 classifiers (25 bags × 60 binary classifiers). Because the computation was time intensive, we modified the algorithm to enable parallelized execution on a computer cluster. Both 10-fold cross-validation and visual inspection of random samples were independently conducted for each localization class to validate the accuracy of the classifier (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-17">We benchmarked our computationally derived localization assignments for one of our wild-type screens, WT1 (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>), to visually assigned localization annotations from YGFP (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Huh <em>et al.</em> 2003</a>) and found 94% agreement among the set of 1097 proteins assigned to a single location by both methods (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>). We also compared our computationally derived localization assignments with assignments made using other computational methods with images from YGFP (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2007</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Huh <em>et al.</em> 2009</a>). The ensLOC framework achieved greater mean accuracy (overlap proteins divided by number of proteins identified in YGFP) for proteins identified as having a single localization in both data sets. The ensLOC framework attained a mean classifier accuracy of 81%, an improvement of up to 20% across 12 subcellular localization classes defined in both methods (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). It is also worth noting that, unlike other methods, the ensLOC framework for quantifying subcellular localization does not restrict a protein to a single localization class.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Classifier accuracy of the ensLOC framework. The accuracy of the ensLOC framework (red) in assigning protein localization to 12 different subcellular compartments (X-axis) is compared with two other automated classification methods (Chen et al. 2007, green; Huh et al. 2009, blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1354212082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Classifier accuracy of the ensLOC framework. The accuracy of the ensLOC framework (red) in assigning protein localization to 12 different subcellular compartments (X-axis) is compared with two other automated classification methods (Chen &lt;em&gt;et al.&lt;/em&gt; 2007, green; Huh &lt;em&gt;et al.&lt;/em&gt; 2009, blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1223/F3.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1223/F3.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466261" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-18" class="first-child">Classifier accuracy of the ensLOC framework. The accuracy of the ensLOC framework (red) in assigning protein localization to 12 different subcellular compartments (X-axis) is compared with two other automated classification methods (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2007</a>, green; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Huh <em>et al.</em> 2009</a>, blue).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Quantitative scoring of subcellular localization using the ensLOC framework</h3><p id="p-19">In our automated imaging pipeline (<a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>), an experiment generally produced more than a million segmented cells, among which approximately 5% were of inadequate quality. These poor quality cell images were removed using a quality control step with classifiers designed to identify dead and “ghost” cells. “Ghost” objects are artifacts that get recognized in the background of an image, as a consequence of noise being recognized as signal and result in segmentation of a region containing no cells. The ensLOC framework was then applied independently to each filtered cell; that is, protein localization in each cell was predicted for up to 60 binary classifiers, where each classifier determined if a cell should be assigned positively or negatively to the class based on its morphological features. For example, the ER-Cytoplasm binary classifier determined whether a cell harbored the phenotypic signatures of ER localization class (positive) or Cytoplasm localization class (negative). To determine the subcellular localization assignment profile of a GFP-tagged protein at the single-cell level, we calculated the proportion of labeled cells that were assigned to each of the 16 subcellular localization classes. The localization profile of a protein is thus represented as a 16-element vector, where each element (“LOC-score”) reflects the proportion of “classifiable” cells (that is, assigned to at least one localization class) that are assigned to a specific localization class.</p><p id="p-20">To identify changes in subcellular localization for each protein following genetic or environmental perturbation, we assessed the statistical significance of the difference between the proportion of cells with a given localization in a condition (genetic/chemical perturbation) and the proportion of cells in wild type by using a metric we designated a z-LOC score. Cutoffs for significant localization changes were determined by fitting a “background” normal Gaussian model and a uniform “outlier” density model to the z-LOC score distribution. Cutoffs were chosen such that the number of true “outliers” was optimized (<a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>). The LOC-scores and z-LOC scores are readily searchable in CYCLoPs.</p></div><div id="sec-6" class="subsection"><h3>Database system construction</h3><p id="p-21">The relational database schema of CYCLoPs was developed to provide central storage and querying of different types of data generated from our systematic yeast imaging experiments. Our goal was to optimize the efficient and scalable querying of the micrographs, the LOC-score and z-LOC score profiles, and the abundance <em>I</em><sub>g</sub> and δPL score profiles of all proteins and conditions surveyed (database schema available through the CYCLoPs online documentation). The backend of CYCLoPs features a <em>mySQL</em> relational database management system, which comprises more than 100 experiment-specific tables, and the front-end web interface is hosted on an Apache 2.0 web server. The web interface was developed using a combination of HTML, CGI Perl, Perl DataBase Interface, Cascading Style Sheets, Javascript and R plotting libraries.</p></div><div id="sec-7" class="subsection"><h3>Database utility</h3><div id="sec-8" class="subsection"><h4>System interfaces and visualization:</h4><p id="p-22">CYCLoPs is primarily accessible via a Web interface, with a focus on providing easy and efficient access to a genome-wide database of quantitative descriptors of protein dynamics, and to assist biologists in experiment planning and hypothesis generation. A number of query and visualization tools are included in CYCLoPs version 1.0, including two custom-made micrograph viewers.</p><p id="p-23">Various search options are available. Search options for proteins include protein name, common name, alias, and ORF. A protein-centric search returns a general description, abundance scores, and localization scores for the best matching protein across all screens. For example, a quick search of “<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004613">Hxt2</a>” returns a report displaying: (1) the micrographs from three wild-type screens; (2) a subcellular localization table depicting the LOC-scores of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004613">Hxt2</a> for each of 16 localization classes (rows) across all screens (columns); (3) a protein abundance table showing the abundance <em>I</em><sub>g</sub> and changes δPL across the screens and; (4) a subcellular localization change table showing the localization changes, z-LOCs, for each localization class across all screens (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Search results and images may be downloaded and direct visualization of each individual cell in the compendium is available through the companion Image Viewer and Cell Viewer.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screen shot of sample search and result page generated by CYCLoPs is shown. A query of the Hxt2 protein produces images of yeast cells from three wild-type screens (WT1, WT2, WT3). The tables list numerical measurements of protein abundance (Ig), protein abundance changes (&#x3B4;PL), subcellular localization (LOC-scores), and subcellular localization changes (zLOC-scores) with localizations from the WT screens shown in red. The scores are highlighted using a color scale from white to yellow to red to allow the viewer to identify variances in a range of values with a quick glance and do not represent significance values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1354212082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Screen shot of sample search and result page generated by CYCLoPs is shown. A query of the Hxt2 protein produces images of yeast cells from three wild-type screens (WT1, WT2, WT3). The tables list numerical measurements of protein abundance (&lt;em&gt;I&lt;/em&gt;&lt;sub&gt;g&lt;/sub&gt;), protein abundance changes (&#x3B4;PL), subcellular localization (LOC-scores), and subcellular localization changes (zLOC-scores) with localizations from the WT screens shown in red. The scores are highlighted using a color scale from white to yellow to red to allow the viewer to identify variances in a range of values with a quick glance and do not represent significance values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1223/F4.medium.gif" width="309" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1223/F4.medium.gif" width="309" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466293" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Screen shot of sample search and result page generated by CYCLoPs is shown. A query of the Hxt2 protein produces images of yeast cells from three wild-type screens (WT1, WT2, WT3). The tables list numerical measurements of protein abundance (<em>I</em><sub>g</sub>), protein abundance changes (δPL), subcellular localization (LOC-scores), and subcellular localization changes (zLOC-scores) with localizations from the WT screens shown in red. The scores are highlighted using a color scale from white to yellow to red to allow the viewer to identify variances in a range of values with a quick glance and do not represent significance values.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h4>Image and cell viewer:</h4><p id="p-25">The Image Viewer facilitates visual inspection of pairs of micrographs. Users can toggle between 18 screens, four images per screen, and three image channels (RFP/GFP/GFP-RFP overlay). This tool is particularly useful for visual inspection of morphologic changes. For example, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows internalization of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004613">Hxt2</a>, a glucose transporter, in response to rapamycin treatment. Cells in the left micrograph (from a WT screen) display morphologic patterns that define a cell-periphery localization of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004613">Hxt2</a>, whereas most cells in the right micrograph (after 300 min of rapamycin treatment) exhibit an obvious localization to vacuole/vacuolar membrane. The Cell Viewer provides a detailed view of a specified micrograph by cropping it into individual cells (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The position coordinates of each cell image were obtained from the cell segmentation routine in our image analysis program. The localization labels of the cell were determined using our ensemble classifiers.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ImageViewer showing micrographs of a wild-type strain expressing Hxt2-GFP after growth in standard medium (left) and 300 min after treatment with rapamycin (RAP300, right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1354212082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ImageViewer showing micrographs of a wild-type strain expressing Hxt2-GFP after growth in standard medium (left) and 300 min after treatment with rapamycin (RAP300, right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1223/F5.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/6/1223/F5.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466269" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-26" class="first-child">ImageViewer showing micrographs of a wild-type strain expressing Hxt2-GFP after growth in standard medium (left) and 300 min after treatment with rapamycin (RAP300, right).</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cell viewer for inspecting individual cells in a micrograph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1354212082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cell viewer for inspecting individual cells in a micrograph.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1223/F6.medium.gif" width="353" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/6/1223/F6.medium.gif" width="353" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1223/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466218" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-27" class="first-child">Cell viewer for inspecting individual cells in a micrograph.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Single cell abundance measurements and localization assignments:</h4><p id="p-28">Both localization and abundance of a protein may vary in individual cells in a population (<a id="xref-ref-6-10" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2015</a>), and analysis of single cells can give important information about cell-cycle events and stochasticity. Because all of our data were acquired at the level of the individual cell, we are able to provide a function that allows the user to download abundance and localization data for single cells. This function may be found at <a href="http://cyclops.ccbr.utoronto.ca/DOWNLOAD/Download.html">http://cyclops.ccbr.utoronto.ca/DOWNLOAD/Download.html</a>. For a selected image in our compendium, this function generates a text file with the following columns:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-29">Object ID, X-coordinate, Y-coordinate, GFP Intensity (Ig), Localization.</p></li></ul></div><div id="sec-11" class="subsection"><h4>Querying top protein abundance and localization changes:</h4><p id="p-30">Users also may retrieve top-ranked proteins that are transported toward or away from any of the 16 subcellular compartments included in the database or that exhibit increase/decrease in protein abundance in the presence of drug treatment or gene deletion. Search results and images may be downloaded and direct visualization of each object/cell in the compendium is available through the companion Image Viewer and Cell Viewer.</p><p id="p-31">We have generated a compendium of RFP/GFP micrographs and quantitative measurements of subcellular localization and abundance changes covering ∼71% of the yeast proteome in response to genetic and chemical perturbations. To make this novel compendium available and useful to the research community, we have developed a web-based query system for accessing, visualizing and analyzing the data.</p><p id="p-32">CYCLoPs is intended to be an active resource for quantitative genome-wide localization and abundance measurements of <em>S. cerevisiae</em> made in multiple genetic backgrounds and following different chemical treatments. Future enhancements of CYCLoPs will involve automation processes for experimental updates. In revised versions of CYCLoPs, we hope to integrate data from other external sources for on-the-fly cross-dimensional comparisons and visualization. CYCLoPs is tightly integrated with our experimental and scoring platform, and will house data from future experiments designed to test the response of the yeast proteome to a variety of chemical and environmental perturbations.</p></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">We thank Supipi Kaluarachchi Duffy, Michael J. Cox, and other anonymous users of CYCLoPs beta version for testing and suggestions and Jeff Liu for system and network administration. This work was supported by grant MOP-97939 from the Canadian Institutes for Health Research to B.A. and C.B. and from the Ontario Research Fund (GL2 competition) to B.A., C.B., and J.M. Infrastructure for high-content imaging and computational analysis was purchased using funds from the Canadian Foundation for Innovation Leading Edge Fund (Grant 21475) and the Ontario Research Fund. J.M., C.B., and B.A. are Senior Fellows in the Genetic Networks program of the Canadian Institute for Advanced Research. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-34"><em>Communicating editor: S. Fields</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-09"><span class="received-label">Received </span>March 9, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-13"><span class="accepted-label">Accepted </span>April 13, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Koh <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.1" data-doi="10.1083/jcb.201301120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gymrek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A novel single-cell screening platform reveals proteome plasticity during yeast stress responses.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201301120%26rft_id%253Dinfo%253Apmid%252F23509072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIwMC82LzgzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Bagging predictors.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D2%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.3" data-doi="10.1186/gb-2006-7-10-r100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamprecht</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">I. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">CellProfiler: image analysis software for identifying and quantifying cell phenotypes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCarpenter%26rft.auinit1%253DA.%2BE.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253DR100%26rft.epage%253DR100%26rft.atitle%253DCellProfiler%253A%2Bimage%2Banalysis%2Bsoftware%2Bfor%2Bidentifying%2Band%2Bquantifying%2Bcell%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-10-r100%26rft_id%253Dinfo%253Apmid%252F17076895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-10-r100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17076895&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.4" data-doi="10.1186/1471-2105-12-S1-S3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Selecting a single model or combining multiple models for microarray-based classifier development?–a comparative analysis based on large and diverse datasets generated from the MAQC-II project.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>(<span class="cit-supplement">Suppl 10</span>): <span class="cit-fpage">S3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D12%26rft.spage%253DS3%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-S1-S3%26rft_id%253Dinfo%253Apmid%252F21342560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-S1-S3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21342560&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.5" data-doi="10.1093/bioinformatics/btm206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Automated image analysis of protein localization in budding yeast.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">i66</span>–<span class="cit-lpage">i71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DChen%26rft.auinit1%253DS.-C.%26rft.volume%253D23%26rft.issue%253D13%26rft.spage%253Di66%26rft.epage%253Di71%26rft.atitle%253DAutomated%2Bimage%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm206%26rft_id%253Dinfo%253Apmid%252F17646347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy8xMy9pNjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaluarachchi Duffy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Yeast proteome dynamics from single cell imaging and automated analysis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.7" data-doi="10.1073/pnas.1308265110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dénervaud</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delgado-Gonzalo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Damay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajkumar</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A chemostat array enables the spatio-temporal analysis of the yeast proteome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">15842</span>–<span class="cit-lpage">15847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1308265110%26rft_id%253Dinfo%253Apmid%252F24019481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzM5LzE1ODQyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.6.1223.8"><div class="cit-metadata unstructured">Gashler, M., C. Giraud-Carrier, and T. Martinez, 2008 Decision tree ensemble: small heterogeneous is better than large homogeneous. The Seventh International Conference on Machine Learning and Applications, pp. 900–905.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.9" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Efficient framework for automated classification of subcellular patterns in budding yeast.</span> <abbr class="cit-jnl-abbrev">Cytometry A</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">934</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytometry.%2BPart%2BA%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BAnalytical%2BCytology%26rft.stitle%253DCytometry%2BA%26rft.aulast%253DHuh%26rft.auinit1%253DS.%26rft.volume%253D75%26rft.issue%253D11%26rft.spage%253D934%26rft.epage%253D940%26rft.atitle%253DEfficient%2Bframework%2Bfor%2Bautomated%2Bclassification%2Bof%2Bsubcellular%2Bpatterns%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Apmid%252F19753630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19753630&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.11" data-doi="10.1038/nature02026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falvo</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein localization in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHuh%26rft.auinit1%253DW.%2BK.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D686%26rft.epage%253D691%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02026%26rft_id%253Dinfo%253Apmid%252F14562095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562095&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.12" data-doi="10.1093/nar/gkr827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Min</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PhenoM: a database of morphological phenotypes caused by mutation of essential genes in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D687</span>–<span class="cit-lpage">D694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr827%26rft_id%253Dinfo%253Apmid%252F22009677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9ENjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.13" data-doi="10.1002/yea.1204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kals</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thallinger</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trajanoski</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohlwein</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">YPL.db2: the Yeast Protein Localization database, version 2.0.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DKals%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D213%26rft.epage%253D218%26rft.atitle%253DYPL.db2%253A%2Bthe%2BYeast%2BProtein%2BLocalization%2Bdatabase%252C%2Bversion%2B2.0.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1204%26rft_id%253Dinfo%253Apmid%252F15704222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15704222&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227105300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.14" data-doi="10.1038/nmeth.2834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulak</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pichler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paron</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaraj</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2834%26rft_id%253Dinfo%253Apmid%252F24487582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487582&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332086100028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.6.1223.15"><div class="cit-metadata unstructured">Liu, H., and R. Setiono, 1995  Chi2: Feature selection and discretization of numeric attributes. Proc IEEE 7th International Conference on Tools with Artificial Intelligence: 338–391.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-5.6.1223.16" data-doi="10.1074/mcp.M111.013722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagaraj</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulak</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuhauser</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayr</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">System-wide perturbation analysis with nearly complete coverage of the yeast proteome by single-shot ultra HPLC runs on a bench top Orbitrap.</span> <span class="cit-source">Mol. Cell Proteomics</span> <span class="cit-vol">11</span>: <span class="cit-fpage">M111.013722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M111.013722%26rft_id%253Dinfo%253Apmid%252F22021278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjE2OiIxMS8zL00xMTEuMDEzNzIyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.17" data-doi="10.1038/nature04785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ihmels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breslow</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">441</span>: <span class="cit-fpage">840</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNewman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D441%26rft.issue%253D7095%26rft.spage%253D840%26rft.epage%253D846%26rft.atitle%253DSingle-cell%2Bproteomic%2Banalysis%2Bof%2BS.%2Bcerevisiae%2Breveals%2Bthe%2Barchitecture%2Bof%2Bbiological%2Bnoise.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04785%26rft_id%253Dinfo%253Apmid%252F16699522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16699522&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238254100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1223.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite> <span class="cit-pub-date">1998</span>  <span class="cit-article-title">Fast training of support vector machines using sequential minimal optimationation</span>, pp. <span class="cit-fpage">185</span>–<span class="cit-lpage">208</span> in <span class="cit-source">Advances in Kernel Methods - Support Vector Learning</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Scholkopf</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Burges</span>  <span class="cit-name-given-names">C. J. C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Smola</span>  <span class="cit-name-given-names">A. J.</span></span>. <span class="cit-publ-name">MIT Press</span>,<span class="cit-publ-loc"> Cambridge</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.19" data-doi="10.1016/j.eswa.2012.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reboiro-Jato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laza</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López-Fernández</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glez-Peña</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Díaz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">genEnsemble: A new model for the combination of classifiers and integration of biological knowledge applied to genomic data.</span> <abbr class="cit-jnl-abbrev">Expert Syst. Appl.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BSyst.%2BAppl.%26rft.volume%253D40%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.eswa.2012.07.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.eswa.2012.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.20" data-doi="10.1186/1471-2105-11-263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">T. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Yeast Resource Center Public Image Repository: a large database of fluorescence microscopy images.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DRiffle%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.spage%253D263%26rft.epage%253D263%26rft.atitle%253DThe%2BYeast%2BResource%2BCenter%2BPublic%2BImage%2BRepository%253A%2BA%2Blarge%2Bdatabase%2Bof%2Bfluorescence%2Bmicroscopy%2Bimages.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-263%26rft_id%253Dinfo%253Apmid%252F20482811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20482811&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.21" data-doi="10.1007/s10462-009-9124-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokach</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ensemble-based classifiers.</span> <abbr class="cit-jnl-abbrev">Artif. Intell. Rev.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArtif.%2BIntell.%2BRev.%26rft.volume%253D33%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10462-009-9124-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10462-009-9124-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000273807600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.22" data-doi="10.1093/nar/gkh113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">SCMD: <em>Saccharomyces cerevisiae</em> Morphological Database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">D319</span>–<span class="cit-lpage">D322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSaito%26rft.auinit1%253DT.%2BL.%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD319%26rft.epage%253D322%26rft.atitle%253DSCMD%253A%2BSaccharomyces%2Bcerevisiae%2BMorphological%2BDatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh113%26rft_id%253Dinfo%253Apmid%252F14681423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMi9zdXBwbF8xL0QzMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teague</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Image analysis via the general theory of moments.</span> <abbr class="cit-jnl-abbrev">J. Opt. Soc. Am.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">920</span>–<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BOpt.%2BSoc.%2BAm.%26rft.volume%253D70%26rft.spage%253D920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.24" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1223.25" data-doi="10.1093/nar/gkl1000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiwatwattana</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landau</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cope</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harp</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Organelle DB: an updated resource of eukaryotic protein localization and function.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D810</span>–<span class="cit-lpage">D814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl1000%26rft_id%253Dinfo%253Apmid%252F17130152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNS9zdXBwbF8xL0Q4MTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1223.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1211" title="The Mosaic Mutants of Cucumber: A Method to Produce Knock-Downs of Mitochondrial Transcripts" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1233" title="Functional Analysis of Developmentally Regulated Genes chs7 and sec22 in the Ascomycete Sordaria macrospora" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1223" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FBXlIK6II-1ztmmdOk1RIzrRDqvX2pytXl0eF-P2DY4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1223" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1223" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1223" class="active" data-icon-position="" data-hide-link-title="0">CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459956" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xM15Ni7E1f6TAPTKlZA9yCjsPWDJyD_vaeSZy9Fst68" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1223.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1223" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-f7GGW_Hk-EQVJyqOiuAzOaN005gTY2k2HEyJk_SS4zQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1223" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1223.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459956" id="node459956--2" data-pisa="ggg;5/6/1223" data-pisa-master="ggg;g3.115.017830" data-apath="/ggg/5/6/1223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Judice L. Y. Koh</span>, <span class="highwire-citation-author" data-delta="1">Yolanda T. Chong</span>, <span class="highwire-citation-author" data-delta="2">Helena Friesen</span>, <span class="highwire-citation-author" data-delta="3">Alan Moses</span>, <span class="highwire-citation-author" data-delta="4">Charles Boone</span>, <span class="highwire-citation-author" data-delta="5">Brenda J. Andrews</span> and <span class="highwire-citation-author" data-delta="6">Jason Moffat</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1223-1232;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017830 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459956--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Judice L. Y. Koh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judice%2BL.%2BY.%2BKoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koh%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudice%2BL.%2BY.%2BKoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yolanda T. Chong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yolanda%2BT.%2BChong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chong%20YT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYolanda%2BT.%2BChong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Helena Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helena%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelena%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alan Moses </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cell & Systems Biology, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BMoses%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moses%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BMoses%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brenda J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jason Moffat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BMoffat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moffat%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BMoffat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1223" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1223&amp;title=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1223&amp;title=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1223&amp;title=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1223&amp;text=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1223&amp;title=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1223&amp;t=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1223&amp;title=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1223&amp;title=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459956/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459956" id="node459956--3" data-pisa="ggg;5/6/1223" data-pisa-master="ggg;g3.115.017830" data-apath="/ggg/5/6/1223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CYCLoPs: A Comprehensive Database Constructed from Automated Analysis of Protein Abundance and Subcellular Localization Patterns in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Judice L. Y. Koh</span>, <span class="highwire-citation-author" data-delta="1">Yolanda T. Chong</span>, <span class="highwire-citation-author" data-delta="2">Helena Friesen</span>, <span class="highwire-citation-author" data-delta="3">Alan Moses</span>, <span class="highwire-citation-author" data-delta="4">Charles Boone</span>, <span class="highwire-citation-author" data-delta="5">Brenda J. Andrews</span> and <span class="highwire-citation-author" data-delta="6">Jason Moffat</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1223-1232;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017830 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459956--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Judice L. Y. Koh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judice%2BL.%2BY.%2BKoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koh%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudice%2BL.%2BY.%2BKoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yolanda T. Chong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yolanda%2BT.%2BChong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chong%20YT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYolanda%2BT.%2BChong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Helena Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helena%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelena%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alan Moses </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cell & Systems Biology, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BMoses%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moses%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BMoses%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brenda J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jason Moffat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Donnelly Centre, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada, M5S3E1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BMoffat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moffat%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BMoffat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.moffat@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">j.moffat@utoronto.ca</a>
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1223&amp;rft.epage=1232&amp;rft.atitle=CYCLoPs%3A%20A%20Comprehensive%20Database%20Constructed%20from%20Automated%20Analysis%20of%20Protein%20Abundance%20and%20Subcellular%20Localization%20Patterns%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Koh%2C+Judice+L.+Y.&amp;rft.au=Chong%2C+Yolanda+T.&amp;rft.au=Friesen%2C+Helena&amp;rft.au=Moses%2C+Alan&amp;rft.au=Boone%2C+Charles&amp;rft.au=Andrews%2C+Brenda+J.&amp;rft.au=Moffat%2C+Jason"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459956/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459956/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459956/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459956/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459956/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459956/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459956/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459956/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459956/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459956/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459956/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459956/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1223" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1223.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1223.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/6/1223.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
