<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/9/1815" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/9/1815.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/9/1815.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/9/1815.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/9/1815.atom" />
<meta name="HW.pisa" content="ggg;5/9/1815" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?" />
<meta name="DC.Identifier" content="10.1534/g3.115.019323" />
<meta name="DC.Date" content="2015-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Kopps et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Kinship analyses are important pillars of ecological and conservation genetic studies with potentially far-reaching implications. There is a need for power analyses that address a range of possible relationships. Nevertheless, such analyses are rarely applied, and studies that use genetic-data-based-kinship inference often ignore the influence of intrinsic population characteristics. We investigated 11 questions regarding the correct classification rate of dyads to relatedness categories (relatedness category assignments; RCA) using an individual-based model with realistic life history parameters. We investigated the effects of the number of genetic markers; marker type (microsatellite, single nucleotide polymorphism SNP, or both); minor allele frequency; typing error; mating system; and the number of overlapping generations under different demographic conditions. We found that (i) an increasing number of genetic markers increased the correct classification rate of the RCA so that up to &gt;80% first cousins can be correctly assigned; (ii) the minimum number of genetic markers required for assignments with 80 and 95% correct classifications differed between relatedness categories, mating systems, and the number of overlapping generations; (iii) the correct classification rate was improved by adding additional relatedness categories and age and mitochondrial DNA data; and (iv) a combination of microsatellite and single-nucleotide polymorphism data increased the correct classification rate if &lt;800 SNP loci were available. This study shows how intrinsic population characteristics, such as mating system and the number of overlapping generations, life history traits, and genetic marker characteristics, can influence the correct classification rate of an RCA study. Therefore, species-specific power analyses are essential for empirical studies." />
<meta name="DC.Contributor" content="Anna M. Kopps" />
<meta name="DC.Contributor" content="Jungkoo Kang" />
<meta name="DC.Contributor" content="William B. Sherwin" />
<meta name="DC.Contributor" content="Per J. Palsbøll" />
<meta name="article:published_time" content="2015-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Kinship analyses are important pillars of ecological and conservation genetic studies with potentially far-reaching implications. There is a need for power analyses that address a range of possible relationships. Nevertheless, such analyses are rarely applied, and studies that use genetic-data-based-kinship inference often ignore the influence of intrinsic population characteristics. We investigated 11 questions regarding the correct classification rate of dyads to relatedness categories (relatedness category assignments; RCA) using an individual-based model with realistic life history parameters. We investigated the effects of the number of genetic markers; marker type (microsatellite, single nucleotide polymorphism SNP, or both); minor allele frequency; typing error; mating system; and the number of overlapping generations under different demographic conditions. We found that (i) an increasing number of genetic markers increased the correct classification rate of the RCA so that up to &amp;gt;80% first cousins can be correctly assigned; (ii) the minimum number of genetic markers required for assignments with 80 and 95% correct classifications differed between relatedness categories, mating systems, and the number of overlapping generations; (iii) the correct classification rate was improved by adding additional relatedness categories and age and mitochondrial DNA data; and (iv) a combination of microsatellite and single-nucleotide polymorphism data increased the correct classification rate if &amp;lt;800 SNP loci were available. This study shows how intrinsic population characteristics, such as mating system and the number of overlapping generations, life history traits, and genetic marker characteristics, can influence the correct classification rate of an RCA study. Therefore, species-specific power analyses are essential for empirical studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/09/01" />
<meta name="citation_mjid" content="ggg;5/9/1815" />
<meta name="citation_id" content="5/9/1815" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/9/1815" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/9/1815.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/9/1815.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/9/1815.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019323" />
<meta name="citation_pmid" content="26134496" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1815" />
<meta name="citation_lastpage" content="1826" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anna M. Kopps" />
<meta name="citation_author_institution" content="Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands" />
<meta name="citation_author_institution" content="Evolution &amp; Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia" />
<meta name="citation_author_email" content="anna.kopps@gmx.com" />
<meta name="citation_author" content="Jungkoo Kang" />
<meta name="citation_author_institution" content="Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands" />
<meta name="citation_author_institution" content="IceLab, Umeå University, 901 87, Umeå, Sweden" />
<meta name="citation_author" content="William B. Sherwin" />
<meta name="citation_author_institution" content="Evolution &amp; Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia" />
<meta name="citation_author_institution" content="Murdoch University Cetacean Research Unit, Centre for Fish, Fisheries and Aquatic Ecosystems Research, School of Veterinary and Life Sciences, Murdoch University, Murdoch, WA 6150, Australia" />
<meta name="citation_author" content="Per J. Palsbøll" />
<meta name="citation_author_institution" content="Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands" />
<meta name="citation_reference" content="citation_journal_title=Forensic Science International;citation_journal_abbrev=Forensic Science International;citation_author=A. Amorim;citation_author=L. Pereira;citation_title=Pros and cons in the use of SNPs in forensic kinship investigation: a comparative analysis with STRs.;citation_pages=17-21;citation_volume=150;citation_year=2005;citation_issue=1;citation_pmid=15837005;citation_doi=10.1016/j.forsciint.2004.06.018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Anderson;citation_author=J. Garza;citation_title=The power of single-nucleotide polymorphisms for large-scale parentage inference.;citation_volume=172;citation_year=2006;citation_pmid=16387880;citation_doi=10.1534/genetics.105.048074" />
<meta name="citation_reference" content="citation_journal_title=Oikos;citation_author=J. C. Avise;citation_title=Molecular population structure and the biogeographic history of a regional fauna: a case history with lessons for conservation biology.;citation_pages=62-76;citation_volume=63;citation_year=1992;citation_doi=10.2307/3545516" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=T. Aykanat;citation_author=S. Johnston;citation_author=D. Cotter;citation_author=T. Cross;citation_author=R. Poole;citation_title=Molecular pedigree reconstruction and estimation of evolutionary parameters in a wild Atlantic salmon river system with incomplete sampling: a power analysis.;citation_pages=68;citation_volume=14;citation_year=2014;citation_pmid=24684698;citation_doi=10.1186/1471-2148-14-68" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. J. Benjamin;citation_author=D. Cesarini;citation_author=M. J. H. M. van der Loos;citation_author=C. T. Dawes;citation_author=P. D. Koellinger;citation_title=The genetic architecture of economic and political preferences.;citation_volume=109;citation_year=2012;citation_pmid=22566634;citation_doi=10.1073/pnas.1120666109" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=M. S. Blouin;citation_title=DNA-based methods for pedigree reconstruction and kinship analysis in natural populations.;citation_pages=503-511;citation_volume=18;citation_year=2003;citation_doi=10.1016/S0169-5347(03)00225-8" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=R. Bonduriansky;citation_title=Rethinking heredity, again.;citation_pages=330-336;citation_volume=27;citation_year=2012;citation_issue=6;citation_pmid=22445060;citation_doi=10.1016/j.tree.2012.02.003" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. F. Y. Brookfield;citation_author=D. T. Parkin;citation_title=Use of single-locus DNA probes in the establishment of relatedness in wild populations.;citation_pages=660-663;citation_volume=70;citation_year=1993;citation_doi=10.1038/hdy.1993.94" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. M. Brown;citation_author=A. M. Kopps;citation_author=S. J. Allen;citation_author=L. Bejder;citation_author=B. Littleford-Colquhoun;citation_title=Population differentiation and hybridisation of Australian Snubfin (Orcaella heinsohni) and Indo-Pacific humpback (Sousa chinensis) dolphins in North-Western Australia.;citation_pages=e101427;citation_volume=9;citation_year=2014;citation_pmid=24988113;citation_doi=10.1371/journal.pone.0101427" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. Browning;citation_author=B. Browning;citation_title=Identity by descent between distant relatives: detection and applications.;citation_pages=617-633;citation_volume=46;citation_year=2012;citation_pmid=22994355;citation_doi=10.1146/annurev-genet-110711-155534" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=D. W. Coltman;citation_author=J. G. Pilkington;citation_author=J. M. Pemberton;citation_title=Fine-scale genetic structure in a free-living ungulate population.;citation_pages=733-742;citation_volume=12;citation_year=2003;citation_issue=3;citation_pmid=12675828;citation_doi=10.1046/j.1365-294X.2003.01762.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. C. Cope;citation_author=J. M. Lanyon;citation_author=J. M. Seddon;citation_author=P. K. Pollett;citation_title=Development and testing of a genetic marker-based pedigree reconstruction system ‘PR-genie’ incorporating size-class data.;citation_pages=857-870;citation_volume=14;citation_year=2014;citation_pmid=24373173;citation_doi=10.1111/1755-0998.12219" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Lett.;citation_author=A. J. Crean;citation_author=A. M. Kopps;citation_author=R. Bonduriansky;citation_title=Revisiting telegony: offspring inherit an acquired characteristic of their mother’s previous mate.;citation_pages=1545-1552;citation_volume=17;citation_year=2014;citation_pmid=25270393;citation_doi=10.1111/ele.12373" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=É. Danchin;citation_title=Avatars of information: towards an inclusive evolutionary synthesis.;citation_pages=351-358;citation_volume=28;citation_year=2013;citation_pmid=23540765;citation_doi=10.1016/j.tree.2013.02.010" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. P. Epstein;citation_author=W. L. Duren;citation_author=M. Boehnke;citation_title=Improved inference of relationship for pairs of individuals.;citation_pages=1219-1231;citation_volume=67;citation_year=2000;citation_issue=5;citation_pmid=11032786;citation_doi=10.1086/321195" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. G. Gardner;citation_author=A. J. Fitch;citation_author=T. Bertozzi;citation_author=A. J. Lowe;citation_title=Rise of the machines—recommendations for ecologists when using next generation sequencing for microsatellite development.;citation_pages=1093-1101;citation_volume=11;citation_year=2011;citation_pmid=21679314;citation_doi=10.1111/j.1755-0998.2011.03037.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Greminger;citation_author=K. Stolting;citation_author=A. Nater;citation_author=B. Goossens;citation_author=N. Arora;citation_title=Generation of SNP datasets for orangutan population genomics using improved reduced-representation sequencing and direct comparisons of SNP calling algorithms.;citation_pages=16;citation_volume=15;citation_year=2014;citation_pmid=24405840;citation_doi=10.1186/1471-2164-15-16" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. M. Henn;citation_author=L. Hon;citation_author=J. M. Macpherson;citation_author=N. Eriksson;citation_author=S. Saxonov;citation_title=Cryptic distant relatives are common in both isolated and cosmopolitan genetic samples.;citation_pages=e34267-e34267;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22509285;citation_doi=10.1371/journal.pone.0034267" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=W. G. Hill;citation_author=I. M. S. White;citation_title=Identification of pedigree relationship from genome sharing.;citation_volume=3;citation_year=2013;citation_pmid=23893739;citation_doi=10.1534/g3.113.007500" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. Hoban;citation_title=An overview of the utility of population simulation software in molecular ecology.;citation_pages=2383-2401;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12741" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. D. Huff;citation_author=D. J. Witherspoon;citation_author=T. S. Simonson;citation_author=J. Xing;citation_author=W. S. Watkins;citation_title=Maximum-likelihood estimation of recent shared ancestry (ERSA).;citation_volume=21;citation_year=2011;citation_pmid=21324875;citation_doi=10.1101/gr.115972.110" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=O. R. Jones;citation_author=J. Wang;citation_title=COLONY: a program for parentage and sibship inference from multilocus genotype data.;citation_pages=551-555;citation_volume=10;citation_year=2010;citation_pmid=21565056;citation_doi=10.1111/j.1755-0998.2009.02787.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=S. T. Kalinowski;citation_title=How many alleles per locus should be used to estimate genetic distances?;citation_pages=62-65;citation_volume=88;citation_year=2002;citation_issue=1;citation_pmid=11813108;citation_doi=10.1038/sj.hdy.6800009" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. T. Kalinowski;citation_author=M. L. Taper;citation_author=T. C. Marshall;citation_title=Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment.;citation_pages=1099-1106;citation_volume=16;citation_year=2007;citation_issue=5;citation_pmid=17305863;citation_doi=10.1111/j.1365-294X.2007.03089.x" />
<meta name="citation_reference" content="citation_journal_title=Anim. Behav.;citation_author=A. M. Kopps;citation_author=W. B. Sherwin;citation_title=Modelling the emergence and stability of a vertically transmitted cultural trait in bottlenose dolphins.;citation_pages=1347-1362;citation_volume=84;citation_year=2012;citation_doi=10.1016/j.anbehav.2012.08.029" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet. Resour.;citation_author=A. Kopps;citation_author=P. McDonald;citation_author=L. Rollins;citation_title=Isolation and characterisation of polymorphic microsatellite loci for Noisy Miners Manorina melanocephala, with successful cross-amplification in Bell Miners M. melanophrys.;citation_pages=39-41;citation_volume=5;citation_year=2013;citation_doi=10.1007/s12686-012-9727-1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=A. M. Kopps;citation_author=C. Y. Ackermann;citation_author=W. B. Sherwin;citation_author=S. J. Allen;citation_author=L. Bejder;citation_title=Cultural transmission of tool use combined with habitat specialisations leads to fine-scale genetic structure in bottlenose dolphins;citation_volume=281;citation_year=2014;citation_pmid=24648223;citation_doi=10.1098/rspb.2013.3245" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=H. Li;citation_author=G. Glusman;citation_author=H. Hu;citation_author=J. Shankaracharya;citation_author=. Caballero;citation_title=Relationship estimation from whole-genome sequence data.;citation_pages=e1004144;citation_volume=10;citation_year=2014;citation_pmid=24497848;citation_doi=10.1371/journal.pgen.1004144" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=T. C. Marshall;citation_author=J. Slate;citation_author=L. E. B. Kruuk;citation_author=J. M. Pemberton;citation_title=Statistical confidence for likelihood-based paternity inference in natural populations.;citation_pages=639-655;citation_volume=7;citation_year=1998;citation_issue=5;citation_pmid=9633105;citation_doi=10.1046/j.1365-294x.1998.00374.x" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. A. Morin;citation_author=G. Luikart;citation_author=R. K. Wayne;citation_title=SNPs in ecology, evolution and conservation.;citation_pages=208-216;citation_volume=19;citation_year=2004;citation_doi=10.1016/j.tree.2004.01.009" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=P. J. Palsbøll;citation_author=M. Z. Peery;citation_author=M. Bérubé;citation_title=Detecting populations in the ‘ambiguous’ zone: kinship-based estimation of population structure at low genetic divergence.;citation_pages=797-805;citation_volume=10;citation_year=2010;citation_pmid=21565091;citation_doi=10.1111/j.1755-0998.2010.02887.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Peterson;citation_author=J. Weber;citation_author=E. Kay;citation_author=H. Fisher;citation_author=H. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=A. M. Polanowski;citation_author=J. Robbins;citation_author=D. Chandler;citation_author=S. N. Jarman;citation_title=Epigenetic estimation of age in humpback whales.;citation_pages=976-987;citation_volume=14;citation_year=2014;citation_pmid=24606053" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. C. Queller;citation_author=K. F. Goodnight;citation_title=Estimating relatedness using genetic-markers.;citation_pages=258-275;citation_volume=43;citation_year=1989;citation_doi=10.2307/2409206" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Riester;citation_author=P. F. Stadler;citation_author=K. Klemm;citation_title=FRANz: reconstruction of wild multi-generation pedigrees.;citation_volume=25;citation_year=2009;citation_pmid=19202194;citation_doi=10.1093/bioinformatics/btp064" />
<meta name="citation_reference" content="citation_journal_title=Electrophoresis;citation_journal_abbrev=Electrophoresis;citation_author=J. J. Sanchez;citation_author=C. Phillips;citation_author=C. Børsting;citation_author=K. Balogh;citation_author=M. Bogus;citation_title=A multiplex assay with 52 single nucleotide polymorphisms for human identification.;citation_pages=1713-1724;citation_volume=27;citation_year=2006;citation_issue=9;citation_pmid=16586411;citation_doi=10.1002/elps.200500671" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. W. Santure;citation_author=J. Stapley;citation_author=A. D. Ball;citation_author=T. R. Birkhead;citation_author=T. Burke;citation_title=On the use of large marker panels to estimate inbreeding and relatedness: empirical and simulation studies of a pedigreed zebra finch population typed at 771 SNPs.;citation_pages=1439-1451;citation_volume=19;citation_year=2010;citation_issue=7;citation_pmid=20149098;citation_doi=10.1111/j.1365-294X.2010.04554.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=H. J. Skaug;citation_author=M. Berube;citation_author=P. J. Palsbøll;citation_title=Detecting dyads of related individuals in large collections of DNA-profiles by controlling the false discovery rate.;citation_pages=693-700;citation_volume=10;citation_year=2010;citation_pmid=21565074;citation_doi=10.1111/j.1755-0998.2010.02833.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Speed;citation_author=D. J. Balding;citation_title=Relatedness in the post-genomic era: is it still useful?;citation_pages=33-44;citation_volume=16;citation_year=2015;citation_pmid=25404112" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=H. Taylor;citation_author=M. Kardos;citation_author=K. Ramstad;citation_author=F. Allendorf;citation_title=Valid estimates of individual inbreeding coefficients from marker-based pedigrees are not feasible in wild populations with low allelic diversity.;citation_pages=901-913;citation_volume=16;citation_year=2015;citation_doi=10.1007/s10592-015-0709-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Thomas;citation_author=W. Hill;citation_title=Estimating Quantitative Genetic Parameters Using Sibships Reconstructed From Marker Data;citation_pages=1961-1972;citation_volume=155;citation_year=2000;citation_issue=4;citation_pmid=10924488" />
<meta name="citation_reference" content="citation_title=Handbook of Statistics;citation_title=Estimation of Relationships from Genetic Data;citation_author=E. Thompson;citation_pages=255-269;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=E. A. Thompson;citation_title=The estimation of pairwise relationships.;citation_pages=173-188;citation_volume=39;citation_year=1975;citation_issue=2;citation_pmid=1052764;citation_doi=10.1111/j.1469-1809.1975.tb00120.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. Vignal;citation_author=D. Milan;citation_author=M. SanCristobal;citation_author=A. Eggen;citation_title=A review on SNP and other types of molecular markers and their use in animal genetics.;citation_pages=275-305;citation_volume=34;citation_year=2002;citation_issue=3;citation_pmid=12081799;citation_doi=10.1051/gse:2002009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Wang;citation_title=Sibship Reconstruction From Genetic Data With Typing Errors;citation_pages=1963-1979;citation_volume=166;citation_year=2004;citation_issue=4;citation_pmid=15126412;citation_doi=10.1534/genetics.166.4.1963" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. Wang;citation_title=Informativeness of genetic markers for pairwise relationship and relatedness inference.;citation_pages=300-321;citation_volume=70;citation_year=2006;citation_issue=3;citation_pmid=16388833;citation_doi=10.1016/j.tpb.2005.11.003" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Wang;citation_title=Triadic IBD coefficients and applications to estimating pairwise relatedness.;citation_pages=135-153;citation_volume=89;citation_year=2007;citation_issue=3;citation_pmid=17894908" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Wang;citation_title=A simulation module in the computer program colony for sibship and parentage analysis.;citation_pages=734-739;citation_volume=13;citation_year=2013;citation_pmid=23615269;citation_doi=10.1111/1755-0998.12106" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Wang;citation_title=Estimation of migration rates from marker-based parentage analysis.;citation_pages=3191-3213;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12806" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wang;citation_author=A. W. Santure;citation_title=Parentage and sibship inference from multilocus genotype data under polygamy.;citation_volume=181;citation_year=2009;citation_pmid=19221199;citation_doi=10.1534/genetics.108.100214" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. S. Weir;citation_author=A. D. Anderson;citation_author=A. B. Hepler;citation_title=Genetic relatedness analysis: modern data and new challenges.;citation_pages=771-780;citation_volume=7;citation_year=2006;citation_issue=10;citation_pmid=16983373;citation_doi=10.1038/nrg1960" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
<meta name="twitter:description" content="Kinship analyses are important pillars of ecological and conservation genetic studies with potentially far-reaching implications. There is a need for power analyses that address a range of possible relationships. Nevertheless, such analyses are rarely applied, and studies that use genetic-data-based-kinship inference often ignore the influence of intrinsic population characteristics. We investigated 11 questions regarding the correct classification rate of dyads to relatedness categories (relatedness category assignments; RCA) using an individual-based model with realistic life history parameters. We investigated the effects of the number of genetic markers; marker type (microsatellite, single nucleotide polymorphism SNP, or both); minor allele frequency; typing error; mating system; and the number of overlapping generations under different demographic conditions. We found that (i) an increasing number of genetic markers increased the correct classification rate of the RCA so that up to &gt;80% first cousins can be correctly assigned; (ii) the minimum number of genetic markers required for assignments with 80 and 95% correct classifications differed between relatedness categories, mating systems, and the number of overlapping generations; (iii) the correct classification rate was improved by adding additional relatedness categories and age and mitochondrial DNA data; and (iv) a combination of microsatellite and single-nucleotide polymorphism data increased the correct classification rate if &lt;800 SNP loci were available. This study shows how intrinsic population characteristics, such as mating system and the number of overlapping generations, life history traits, and genetic marker characteristics, can influence the correct classification rate of an RCA study. Therefore, species-specific power analyses are essential for empirical studies." />
<meta name="og-title" property="og:title" content="How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/9/1815" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Kinship analyses are important pillars of ecological and conservation genetic studies with potentially far-reaching implications. There is a need for power analyses that address a range of possible relationships. Nevertheless, such analyses are rarely applied, and studies that use genetic-data-based-kinship inference often ignore the influence of intrinsic population characteristics. We investigated 11 questions regarding the correct classification rate of dyads to relatedness categories (relatedness category assignments; RCA) using an individual-based model with realistic life history parameters. We investigated the effects of the number of genetic markers; marker type (microsatellite, single nucleotide polymorphism SNP, or both); minor allele frequency; typing error; mating system; and the number of overlapping generations under different demographic conditions. We found that (i) an increasing number of genetic markers increased the correct classification rate of the RCA so that up to &gt;80% first cousins can be correctly assigned; (ii) the minimum number of genetic markers required for assignments with 80 and 95% correct classifications differed between relatedness categories, mating systems, and the number of overlapping generations; (iii) the correct classification rate was improved by adding additional relatedness categories and age and mitochondrial DNA data; and (iv) a combination of microsatellite and single-nucleotide polymorphism data increased the correct classification rate if &lt;800 SNP loci were available. This study shows how intrinsic population characteristics, such as mating system and the number of overlapping generations, life history traits, and genetic marker characteristics, can influence the correct classification rate of an RCA study. Therefore, species-specific power analyses are essential for empirical studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
    <title>How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data? | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FZfnIbTISYLxoP4dskxRHJtikZ6DmEEu4Iwcv3cBqxo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460023","apath":"\/ggg\/5\/9\/1815.atom","pisa":"ggg;5\/9\/1815","ac":{"\/ggg\/5\/9\/1815.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/9\/1815.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/9\/1815"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/9\/1815"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/9\/1815.supplemental":true,"\/content\/5\/9\/1815":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pC+7EU7Cn1GBqHmWquYmlQ4F\\\\\\\/tNWIK1egErrLJP8cgfa+tkOQ7bop2Fbh934WkYm+KXokA522c2+rQf4IQz5z2dpc\\\\\\\/Cd\\\\\\\/xlq77StySjSBe1Xw3YoJITlhU+dlLNl+5FnhCJQYvYc57\\\\\\\/nrgd2ctX931ItvYHt+HHaAeS1J19HQo6zFBLu9LBed7M5olGlszflmEaukgkSqXPZI45UVdL0JUvSjDlcd1+o1weAlncmnS6nl8n5qOENYbeDxvOrL4F966rhAO3SlpYiDFzVbtauDzrL4dzN\\\\\\\/06NDwD7XHW7UR5A0Ibmes8wdIZmYJkmKXOonQ0c2jjzAfX\\\\\\\/gjtL980oJ25Un6X7leYodLR19MXKkPy+r1xvG4R3Bt08ArHYXpfjjEK1yCvXCRh\\\\\\\/8N2IBrTdJspDAPD3gFhKbP5QUKJ+lOOSKk7re\\\\\\\/5cuDSJLIxU41N7SRORWCZsr2w+XU9oQED9S7h7zuI7MENFThvTI6nb9CjGeRH8JmYw1+6klBXF6ufxyNHSvNZU0RUvvDkWd\\\\\\\/JyhWlua1Eo8cDux7BrOt4Mgr6B+uA4MFBUq3+wNcZ8ZhJ+TFGikueoI1Rknm\\\\\\\/Zd0LmdJ63J14ivdGiMUrFvg8ZSzdSmcGEMVflP4cxGOzJZK6HgWEg50ctC68qkbOqf9QEfAWTBbK9wwZCHlDA9HFN6TctKD0mi+QEZTFZwFIYv0TGiJ7wztiT4ykcM1tbyOiKVdPXkSQXyKeCM\\\\\\\/ZQuzabea8jLGxAReasbMBjvlI1NP1JWmWVehojNKfHT9JQ\\\\\\\/fKRlhsAopcdgulY5hLlBev4W8JQbPqteW\\\\\\\/CnFaOu4mTckfvvrkP9mzsKW9aF9V090IpytbJOOiq1HBU\\\\\\\/fYspQ7s9XvhpUDi8IZmKQIAEsw3JanX2H\\\\\\\/AvOS7VNtVQPkV38hZuYVtsJCYhmgOtJOBcrSMG6hX36VrnSxKJLkZ8KEDVL\\\\\\\/Js1GSY6wNF28NVR1+EXsNB2NdZbKcTWxryWNGc9s6hn0InS5leRccoOjjP+ef8qR+Hq0h7mJlxRl01QJeISL\\\\\\\/aCMO45rR0fG4LVHwMs4yBwVzsFAaIrsaoRCmIlvLjwJ\\\\\\\/fAsTD5QeSpgg3ufyaAXRQkTr7XJ16RHzvKk6NUVBIP5Dz4Yd8A7FIsm9kti\\\\\\\/k+H\\\\\\\/hiH1oXeT10FuyrLReuPm\\\\\\\/DoGF2NORrqUdSDBWzDgJMdw+PGmj5I9\\\\\\\/X2zM9OBCfZWPmJ8h6RQ9+UHRNiRI70Mt00yTSBo+3qOtlg22HtoRDqAqQfFQtxOsbbXVmWy60mO\\\u0022,\\\u0022iv\\\u0022:\\\u0022MNprom4+hKQ+1XoqaQ4tcw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227226b73fe26610fadb1a040bebebf347b26752a8085eba45fd83d99ed0ab17fd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bOh51hDbLwXEaS2lNNmsvqLCw5BAo2\\\\\\\/zwDQiyXsXtIgrOJdxIq9vGfjikwvo1wldrNP7xVz\\\\\\\/d3VM+JHHiqd3pnqCHFXe2A+WoEYdFAYwwFmpXJipXrn4B8KX77XbWbDrE6FuXY4Ja1VO3271VRQ4mPkmbMNLMnGJoIXSYYsdNTmJW0ZnLZKLD4sdGqOhVxxG0uPKaCxDFOjkWUsAeHBYqEvCJ29ZTdr2UFSHJIv0SGrzPQzZM0G5iLDAm48LK4v08EeNXMb0hQS1aMLaYfdwyUVJkuxqbxdoC3a4S87ssiRQMxguoJlNMdNYkfucD+HC64TXbe7Lv4pF4EHAoKVtRP4wPCMZOS1bGf+qLQtBxgcwrFm+Ynhrza1BzvNJS8K\\\\\\\/itaqPnqtnh67CkEOYR2biAG4ZsF3HxKjqCzPXKnaG50Tk\\\\\\\/h+PlYK47Vn9DFAI45k+7Tm2bTgCYHCl+aiEmSuwDDY\\\\\\\/gidtCOM+QGAySf5h3UGniZV0qLABrtoc\\\\\\\/mWXijSnakRX5lXVaeMVxXX\\\\\\\/MFX+q7OREmZZAQXY7xqxwFyLuUqJpvJdaet\\\\\\\/jrnelG3QcafwW1vXPNkuyXMfg8XIMCaR2zW\\\\\\\/vU8GliRKQ\\\\\\\/0wkMO5jeDH77J0sCvWAUlUzPbX\\\\\\\/uiOxvC6JAiIBtuNAR\\\\\\\/BlMmpSuqMeOmj527yaYuO1DubMW2EFBUneaXHEAhJsiJk0YlebIAPi4rKO6jtiyiVEj5s9SfWRw8gC9aGeWh2QoRxlBl5rUrrCwYDpE+ZSEnoTCWUeu64Wu5Nogximc7y1BmZ4AhV8OrHajS+ltMYc3DU7H\\\\\\\/3gqw9f1OB35yRzrofKngxehlBRZ0C+k\\\\\\\/HnfYWUfoRe1MxiDPU9HLFcrtdH\\\\\\\/GAS\\\\\\\/5DI3tNHizxhlhb0FrdM\\\\\\\/RZ80G98dCVKzsgoi2+N4Rr2oUfvcJeh6iATg1IboNntHjWvnDM3I+1sHZSQTeWhj4IXdAaxsMCwo2VpqnziWYA7XMVOZ1RRkzGTvy1RQp9KYP18HL4ts4sikmVbzwUZ67EnIOe6SKCDnuS0FDW7i\\\\\\\/P7jC2gR7UXA6t3VlD2+z0oGAph1abH7f9C58ezLy3ahT7r4DZrD61wbeDQchvDPigFT2LP2HqWXEUmgd8Q5duSmadwzsSP\\\\\\\/tJl0hHQ62OnBWfXTF1TssaTEJYsVhNXk5AFyPlupgGmw3ZlYSAA2sSBK2GErwcRFmZCoUaZa7\\\\\\\/ts+EVF8DP\\\\\\\/uXVSseePe4OzugezahNqqS2isd0OB3JQbIaUvxz2hzFMd6\\\\\\\/vjXTFbfIjVPYK0pWI2LOHdLDF0iJvAlOXkYdyZu+kNYhVS9SZlMU2bfGbgTpnaQPslnkJciKCAgEn89er\\\\\\\/jbWB2VLrzvpVvSrAmAW9ImTqmdNnndFn3igZLWcyL+2vQHcahr5p9FxpygD7y5KnV95N9YQtZYxFVQg91KSH7QLeRRgcYAtgcA2d0zab+JHRc1gf8ggFoOq\\\\\\\/wpktTU0Hcu1GDnGtU1c4C0B9BSrbeIRBdOiu1StecL480HhTQVy5xxjbV9DnqcTZ+WuoZkXAUqoBISJLC2QPaJ\\\\\\\/TVbbkZ6aNkgTtrHh0U9yei8saU4E+y3K9jHSS65FctomHn6LHv3joHmSDpSiVMIylP8KaGhoHuypMI1rWG+I3XsXgj7HAm+\\\\\\\/vxvSDAmSYNYwwKpO0ZX7UtXYBS0Ghf56v7C04q\\\\\\\/btqqh47ydgcz6fHKixCuGrWjNvwLvlnBzmwSwVRCN35zYel\\\\\\\/fTCE2V8swuEK2Xlode\\\\\\\/hlbEH\\\\\\\/mDTm8tX0aHShKESS0RvY9M9J\\\\\\\/Coph6GTFzfJrtdV30aM8KZ10ux8+xeyBmFuCy7AVW6qVO4HF7jbvSGElGdIkJ9fsBSAyq4cZIIy5nOmLTqOK7fxbAQVYXio7UYUpJi192Fk\\\\\\\/svw4+HnSyXtm0Iuh5jJY4vfaC7XJdlLPbaAHdU\\\\\\\/TW1l9yq9GIEIVDQzpF6sgVXzCEMniMNoqEbj9YavSxq2da6uGTgKpKEw3V3obRt4bCwFYZ7n2+VBWICjvS9ZVmW8ZjXaRd\\\\\\\/T4GYyctsKD18H+Rz6XQoI\\\\\\\/CSYMlOBDA3jVjUq1\\\\\\\/jvR9VfCGuTh6zldEzkg+LFCqJlX+E1svJ1Q+viY65u\\\\\\\/GYeDg3WLjWk92LcIpqp0rGa5DgKrF8vLBqzKKdyUoBeHxuwTQQZL0enap+EXGdpVQeOI9jHA9wbW+C7NN2AzP3naEZT4MG1H1ydAroxvTMB6FGUfYa+XbfkE\\\\\\\/badyYVGBZzD+Qr1DpkdGDQYamVkiZznEzB4bp7zu3B2rOwRbnGzqBapeiKf0mz0h70eTssFtzNY0URefoZejVWppIyhDAPDmnn5l9tsM47tuGNVkPjb5fQDSlz+wmGBbAue+iJxd8BsnISBS8mI0Mk7j7Hus3IZ+z03\\\\\\\/CLGeSt9IrpEEVv+klOx7urtaFZ31mQMWcUNWRBjqoZ5UKnfRCMpUFolTRwjsgYrj2Qzk2tEf7yBETLnijHfXh3kdAmdaWRoBrYCAnJa4vZgPKYPwsKqkEa2VKnCcbf7O00GSWc4hk\\\\\\\/YfaJOi3jYzMnPfwORoC\\\\\\\/oTBNQKR4e8E+HlfPrOcrmRPd+v\\\\\\\/RF7kN15CE0UqsvBvq\\\\\\\/w095Hvj6cbHmlm6UpEAyhf6PwF+0fSRaVZ+rsmzTRtLXXkGF6L1a0ayt0cHO3482lVrN1NrJ0niEZ5Y\\\\\\\/dNssoNcFA5J6b5Bn6XDtDUIZNci8yzVqASQjKqDF2MdYE4tE95GtL3dRsJSHwlZ1VMwdEXSwNkFhoZgW2HTf5ei\\\\\\\/MaWw7qLlObQlwYTx7sw7vMpWRauNZdujo713sEJvTMujeEvZRbCY6TeN3QykevwfheqPJNa0NcuObRQJjunPg9sVBNFJ9BCmpaJ\\\\\\\/C4omzlzRXDQ\\\\\\\/oC6Q2Vq8V5V1GYCOwqHQI8RjJhvhe2HtRo+fQJ85ZLDQ4Tp6msE1QsZ7rt1iw3HhLcRUAF7NJ+mykW9wyn79qC8B9O9hDU9Tl2FFcuDKC0cPrjF19OxEVTBjq\\\\\\\/4+s6kbz5ymN\\\\\\\/PNncog\\\\\\\/CZfM7ibR9OIeFP66CKWaOR45ZoMlqiK3MXVkpdu5pQxL\\\\\\\/NuOwT5GlPqki3OYnx5jiPuh3F4LVM2AKIIEprNlhUh+yeswrIfc59+lvR11FnEoXEzcedvfTatAHHObKpRcncBWQnNVKv1aZcOQNe6QmABzP1t5LeKOKESjeDpRdaZ2hYNltQVSIMre34NsJvgSvat4MUX5XTks+wHcSwk5zv4luYQehqm3kX3i3Pn7FewSxQryqUwl31dNu9MbxJhTKanN9Sroz+DtWX5C\\\\\\\/QHWL900g8ECXg66kq8ULqoXKU7p5scd+78FgF1DxTUp7y+4yzUIg\\\\\\\/PxQ7o0unLHuz0ElrRfdrKYeqB0EDpy\\\\\\\/dKrhacKSpL7OZ8vhVjUW\\\\\\\/CGtpkkURhKKUiiSaf76c+c+ZeAlfWQUvY5BITYj3\\\\\\\/9hFfvTaEv\\\\\\\/So1\\\\\\\/wcukdI7M\\\\\\\/T\\\\\\\/diiYz5seOE78mp4sUTGc6Sf6Ec8JUrTSkPrPU4Ra\\\\\\\/uX32MR8PYyLewPTDwtKIMs57z3aZTBTSvaDfxyI8TZ8oLIGe8DlDCG+JU5kXMK4WyQJLzheqQmygRkUh0LqL\\\\\\\/2FkUdTVz19WejhyW\\\\\\\/MYMsQdkG0IqY60yj0dbtHey\\\\\\\/hh6gMb0LiswtpAgOzwc4LtzB1mGeRVVvGlFqouMOFZGHH8IDTUk4DoEyZUroWf6TdINi5JLTZAwrYtJsPkmAPFBQJT8bU8s7VyfzEKLMM\\\\\\\/GIMUljGYjoWqv5AyMVoruxD1me0p2tEVT+bJaPjfwR1Uy8S4+O8Qmx1hEdoIE2gjUAsruT4yDGQc+N7ezhiMqvFJgwbHXg6YKCofkfycyps8W55dzsdUB1dGdl0dJPztDD+ah\\\\\\\/cDMFScZLiZ5j2v3RUdJttdoNGkOueyKmx3E9gCYRBDo\\\\\\\/qddw284xs5eb2HA6n63zVVQYdtbhk29A\\\\\\\/GocHxFqOQR265KGTm9jiCtJ4UNwZ31o+Wqu1GmIT7KVrgqE6A0Q1UOvC3tsK5LfoNst1M+GeJldxiXCvXC+UMgDzKQOkemAJh4nOvhkw5Syge9hcYvSHaBDVSOCjqbjmT\\\\\\\/xLQVnzcMOxa0iz7Wbknx0HyN\\\\\\\/bQAar8lvPGHBpehBOZatMOKHUM0yBmjcQn8B9x5HLvE2KcHUB5eacuCDDQgdoa7zo0ERo8tmSd8ofKIcMFjoITz+3\\\\\\\/ZgOX78vPiANSyZPbO6lNkQ4AY0Z17Chfx\\\\\\\/K8gfLI6mUt3jzW93MZoVsgCr8OpR6W9IiW0zjeBZMzoB7RyU17oZ9MWup+AgPeCcezIBdpe0cx1DshJ7H0rUTTBqOXh2NCS21Bb4T8WfM\\\\\\\/nqT5hFZch3rdEObEFdValydmOsfN2BIyO1WUHUyXiGzjgiwjSE6A9ZXb8bOp0gRd6rbeDiKreoaZ\\\\\\\/4Ym3wCH+LyZlxeTLYYFqu2AFu47TiEU6eEzU4pYgUSD9BlFTCf+wlbNxNaR6Ce2wI5tAece83fkUBYoVuSSGdjRrC39vNLXDVzHSsESX3xqCOWrb1WAj1YYrZbUQrFNE96m1yzXj+dyp\\\\\\\/lYqH40uWxjyvjycpmIxc2I4QfGi\\\\\\\/HD7F6TssNiSi2nCYIBrqeODfoslTLMNA1KLnIrnkB+2xDvPRiJL0hiLNrq24PlwsDts0QKl2kyOVcJpKaiwRIlyrLywFzo+9+hCgwHqW850ME0B0UaVPiSJJcjSn+tLgJIKSSSv5g+1bAbY0kQ0eoJ4ZlXrL3rNTF2s8LkZH4x85EpfQW5SPIPFIkmA05tEBMs0sJT2LGwx+AeSj3EQSaDvjv\\\\\\\/RWR0eatk3AwNj5uSAHG+e32A+owaChl88EXY5lUrVVYUkXuTtJXF\\\\\\\/1b7ONuAHGzOjwYzcnDKEBpv\\\\\\\/XTab\\\\\\\/QCkrLxW5VS83Fbi0tNJpl8yBknisPjvwzbXZs3hGOPDkcXiuJOk+6sTB9h0L+HscHp8vK\\\\\\\/RwbcoipsQLpWpa7MTMjdTWdb1S4t2gugzyFxWGsfyveUCpbsK7230+ZEEbwZNTU2+6BeVogiI1W9t4BDvFanhRBqtzkr4wR3WJUCXhN0qugXtdXxPs81W5jMU6Wd83kFMm8Yx9zERR2uX7CMwgGAcNb08ZmgqfsVBj1AoTU4hJW3BHcVYCwH2JeeyjXgPDR4dFs\\\\\\\/Su9h2liEbIFGe13hrf+C+dxH0xtx1q7J38wJV++PH6Xf4vgYxR8q5KKbDGQxigD2pz+qqvOxttw+lEVchmDRRFrN45wFWv9UQYTp7HoKeeLtCvfntZ+y416gS\\\\\\\/0nyKgh6SOMF6GiZAQUyqgbBWq9s3M5Ia37IPOywWlO+lKWhdDY8IQnTBf2ZcI0SQV28dqRu4iSzmJ+JesWtBAuzKDlpgTrknYK91SXsyK3L2z3i4tNs7SoUS\\\\\\\/07Rp2KSSm+yZ0huUa2gOgpJGdTCSUv2Gln14pSIcf\\\\\\\/hMzXqxml8MfdbPh4Hhe6sdmu4q5EEY23vMFwkaVcIMT141o5AbUaus+H7ylHTZuqeKGjh0vc36ijCyvbitqP6VexCi6v5Sekn1wwl3XBdSHDd\\\\\\\/FzWr1pGm+CSVu1fzUE4OEV+Z7LRQcY\\\\\\\/cqEPouSn+WJOmEpHWlXtnzvxkPHJPWWuJYj1cj4LqBUthAZrHNWrFd8XugrmFQ+WNQdDQINl3Luw+CV\\\\\\\/Y+0EpUF1NgpWAjorOtH9IikWGR4pmuSs2\\\\\\\/JUBKHvc3\\\\\\\/Pz5WX29F6O3LWflgQ8xcW6psnt3Vi1FzX5UYvDXZx87ujQVvyfOPOjIi7Rt1rgLI3gU7ETrpBvo8k97YFnd6Rnpf5vwRgwe8evcD5d9aCzIxSHlQS8qvKB22xmozjjLAUpqWsf9ja8jFeyZkZLh61q0wSLnoE0Sb5TYYtwYLQ2iO6Eb6qORAvEPEOgNHzqE7csKbYSFTNVJHKr5oMXYDvtHN9NiY9wifgEvp3GP7MhS3I7WLaLrdIW0IA9o4V3NXanbOPEy9DL56Rz2yeV87kO+zlLdoBI96VwHnYwWao3mE2MfwvoT7A\\\\\\\/JjnpI+URIfCnIneb+1XkcYNdtgLm5pkvUdMgdkmSfjpJpIEDuTm5qNgISS39cw3HZ\\\\\\\/rtXRfqhcgH3htDeMI2WXbxEfGYE9Lm7ztODcX9cjKaQwEq1F0RCyYJWHQ6crTKZwFR8SuppcrT0rMZMVt7hgUFQFR6zZ\\\\\\\/iKSz6Sc\\\\\\\/1pSFehzj8SWY8d7\\\\\\\/Nf9xWUomM4P84YXcVEK\\\\\\\/9oZyhzgsv5qEI18\\\\\\\/JsY7Yc+DWPYmZUwQ6\\\\\\\/dwAISt9DQuzvOayTBD3xyd3CgrZ9nBT+3vQCmgmKdC+5Pix4mEzMLwInErVl0+ahRCqOuSf\\\\\\\/eBp8zygkpneWJ9OIUV8qQnuVDb\\\\\\\/OpjvGuZ7Au9SJTEaEcSWx\\\\\\\/BzZj34v+iAjRLECV2bHjqZnd55jNAkuPy7eGKH8SGHCja\\\\\\\/VT9Cguxjtoz6xaXxy7Uof3evUpcJx6cZfl8X3hkE1VRGZAKijGUZhTBMze63vxjeaFEMNTZ94yvMqBuQtrFDmpXUgOdk5TUNW4+CIqKQ+chqXVI6yYBRbt5N\\\\\\\/Kc8aj6ynmiNaxKAbkMUG5S0W6KevuAQzr8ZrK3ESL7LAvSESUNGBZHI\\\\\\\/XMQYoJiKnkRhihtugh3q1jgg4TW3kZKYAR8TT00dV2SGu2eacWng15Jo8Gz6VfJH0XOInZGiPA8dNssrduGwQ4uzgh5wViCN7pJ\\\\\\\/7fCGSG4rnZu\\\\\\\/xn2mjhoOBYR51dDM0u\\\\\\\/b3I9TypjFT32R2bQ\\\\\\\/66qfEwNcyhbRfWV5TglFTLk7GouoCrHCc6dI6GMVpS52uyvrR\\\\\\\/\\\\\\\/LhVd0+F336npq0i68Y6bCDOrW3jwGyQegVeILX+wNKdmiqcPV0YlksTcYMVQIdFzOhNOtHrqbSfLK7ks+fks9D85+AlYEFkEiGKBPRLDRbADbmC0TNTFKZmoZYzCcr\\\\\\\/cv0yk\\\\\\\/K0h6b1hg1Et98Uccmu7eKY\\\\\\\/bzy2p4kSKZj7GuvUVVevrAOOSnhbhW4EJVbczG87lkTiP5es8esuC4SoqrrELGXztU2Th40yidMaJJY9g\\\\\\\/N\\\\\\\/hQCLFmkiR3wBw3AsbxRCLMZ6ZkidEtgo8Asnw8jKYvlrOfx9K7pF1+Fw4BO50S3bRM4oi\\\\\\\/EJPuLdRBLnMnTbyOw5WvWv8wFZi1HNNLUuKGv9PIY5XJ0exP4JfVxheZmihbfYEROEFXXRapYPZDEzFUKFnW\\\\\\\/MY9lsviJcs5xW7r6rxBdiIXLyrLdislulPEq5iv6HcXkP6amrZ6wEPH0l1h3tcyQsdkQQ2e+XdRROk8xMQiN0QSpZe9NVjmAyBzxztP2bU9gvIfVxNdYzyL7c2R2c4e8uGELNCeeslQE1YGFNXgLIzNbOSz\\\\\\\/zWnwlwhg6DYe+nhFbZjGJ1bZK8rD9Dp5y3Og8l1JQC27\\\\\\\/digT+8WyPLLOLrVfKBqQnK688vxboCkgoY\\\\\\\/AT2PnHwnsHHUTLqtSRm7OFj2renBXrewPXiT7zpM6WQRShgh4+9CFILeS3s488JYrBm3D7tBMObyaj1TTCf47dzqNUVF7NtkXvBsCOvXEfEC9uxUP9BpbM+241XtqIn6BQR6qLEW94ck2r7hEmcPw1mn8fMPJ4VifJvzKDobwtoy81eJwfE6aEEiOeGVfRoadVnn9cOn7ErlY31aGr6Sb5DSwM6Q+1PZu1F0WsDPyqTNPK\\\\\\\/FJv\\\\\\\/aFmjcPYJb9NSkN0E+H0QpdFFci8VnoYd2vkPA5AcKva1TWlVnhbKLSJ7UQlECe5n3iyy0mPxXr596ZivI2HHiwWui9FIsbRPwRxwbb6OyRynAlLmhYVoaMriOWHnFpQ5XxSiKImoVhEeGNb0Sl6kFN+DZQuY0jCs1ON+5bz8eDq17DQ7zkLBrc2ezxKvYe+tcWZLaIVqVHtfp2IkmRrgFn5kCMO7PJyRATCCfdHHszV2r\\\\\\\/tgKo6i045fojHjXptc3bOMihgtD\\\\\\\/ewqZECSuQdQp2P5AgVZ94VIjmDsZ1uOZOXLTX3CJUtB3Gxx7Pj9xEB9POzJV0pVQTnHOu33MeZYX5kQAxOsTU0EXhnfCAwrSjILtL0U8g6+EXqc1JoCiKBEOp0AGgsKBT\\\\\\\/ajNlM\\\\\\\/N\\\\\\\/Ny7k7sDhvzGu5UjVE6ouzSOJhyx\\\\\\\/zmQPf4mCcS4HC0xFal4Xy\\\\\\\/TFuYlHNC6l7Gu\\\\\\\/noN\\\\\\\/RkA3pH+u\\\\\\\/P8GigiymkX3umQzt\\\\\\\/gyS4TBkKqtH\\\\\\\/XnLqNBkr9L9PNWlL4PEgmNKeKpunL+L\\\\\\\/Pk0e2Wg4s+rOvtvMIrtKPuV3UmKtt\\\\\\\/xRXTfggp4vNx3LUQ44dJgHZek5Qqw\\\\\\\/Tt+rma+\\\\\\\/oAMUfRzYsjVxg5N2ive4Nm5LYI3KQPtvdXASCTO0vFrN0QbKAqz26lutMjtEZ\\\\\\\/jvZjouGvlEAJIHMzRx0zgDsVGEFZl\\\\\\\/KOGUM7dvjUJ056bg8FKCUEx8rLcN+nybcyiMC0KKh\\\\\\\/4cNLjrxu6wrremlebIQnziTCTGeZGCGqsCXZNGilMdFUgctDb+sKE5uUuQgfPVKI1eXtb7mfBYu7Rdg7y2gAQIls0ktwZjPp0tC4UeeKM7qB6FZ9GHgKbMnXQMOZJad7yrLUeDJd9A+uzGHbkFFZm7EAAnaRQIWqya7YtQYoCteYdKSDC8IBza3yhQAd7WqxbxEJqw7Tc\\\\\\\/qeA9pj8eUaO9WDIx+ok4ebM871Dtyq2Gb0xnjQGfTpgIjUKaXRcdTIAslAkryY2gm6Ma0FZMr0HXYhDHme9Ns6vN59Dz69iSIjC4yxqlyj8chpYSeh1tAPwEwzGrIc89papnryEEV6eziYsLigt0+MZ5mDpc2w0dGNCz7HrZ5vQ+3Sh1JRdaxxWT4nLe+t6tHM\\\\\\\/v8mANECQ7MAVGfElpghbsaJDdM0KamZm64mjFY93I7IeLtblWCAF3LWUnUdgF29dcBSE2mdJIjxz6cNv9uTMhd+kAdJgM902R8oJVmmUS9vmddcn7K3dpl5DzxDAoupqjZm6E0FDsYL7RfFBy5qqPtPQVK1d06l2JDIb1IFO7KFmYFit\\\\\\\/QTH6k1w2fHQD3fE0mQchwcGgmqEZRcDeZLy1nsUAPlf178ZH9GMEMHPH3xLxpSlN\\\\\\\/g\\\\\\\/el2Lv1DYn+aPD5Be\\\\\\\/qA2+hG6AFRl9+kbpv3vHG5v71WPYzCja14kmDBJnVfG7X+b\\\\\\\/Y8QIAfmROfcYHAGlqLyR4h9vFV3PzcNZu\\\\\\\/ieV8seH0g7TVdLKc1Dg3X2h17Xw+yHall0kmAJa8vLMy1DM3ZxV9QE\\\\\\\/+wCfsG59F3aDRQRoxkbBghNN\\\\\\\/IXGgBLCUoo0ls+iMEfrnq1Hj8yYtEiK67kgtJ\\\\\\\/RVacf1LEhuQ5tp9SM2zvgqjwjGGXMNhrt3S1W+1to\\\\\\\/f9StD1KCSnUnUysAYWrrbo4lMKuK3nTZJt8wvFCOBAS9yeJ6t01Hvy1c3P0MDXr4SEpAkMFou3OWP6XSa46cCNLrWvcLZNRRx1N1tEuQJNgqtdackm5Evx7uQ7eQwsXWUeJQ74\\\\\\\/yMIzoHJmq7ymy4kvl2m1HMgpHWc+ofapcYZhNE59uBdWTN0jDj7klWX4y5LBh7SHL81t38UPaDOlLohNDR3P3Hsh7OJhRQpYdqfHXtHPRzCFjzH8UBYLfLmahBJj0GQRDNrJnsq9gIcB8cSCM4gaBUqaScra\\\\\\\/yAOvlebKuJx2nVuk3CJzjh3CabSvdtGe3TiIHzu7\\\\\\\/rzuUxdwkIAMNUdAS+SEKwVGjS\\\\\\\/3oGYVdYoLH0tn6j3z8FFYjeOzLA1UTWD5\\\\\\\/3OCEJnFT7Z+YF5HfanwHsG2MIT8pJe5+ppv6XLQ4zgkqI7zBtwE8ooNNR5yOwLdXhjw8N04yirtrZeMRzRaPKECeGAiy3H8c\\\\\\\/aIciao7J85\\\\\\\/v3PRvrz\\\\\\\/cRo1YDQUn1X2oxcrZNoF2\\\\\\\/ZWZ+xGMObN10zHe\\\\\\\/O0qiBW\\\\\\\/r\\\\\\\/47m4b\\\\\\\/bbz0wYSeF1k4s5+A76AmsQD7XSqmVsNbm2kqZ2jzD4t7LQSr1o0Xg9LFYHarpjPO2yiIOlCiL8PVvsmbCaceqT9EI41xAR+VZbX8I09etBvZYjKD7kw56wHIaea8PjJUvhTfnwID+6iFZzDDZ4ggwzEtrklc\\\\\\\/cUJdx450k4OHNZCyrmkOluvTsjRJRJFHYqVxIaIdTAy60EcX22zHABmK3vc6JsvPwug8CMSNFE6u22SO3mfsD3qsETnb3crnEacqQr7RPPpm8Qog69v9t8kmvwnTg6z0YVozKRWpxO2p7GmCuD2JYEY3pYQRU3i9L\\\\\\\/Xs2heAQJn0bOLYbpLWYOyA5uvS\\\\\\\/9NVkZqqoSVbx0fZTkIC9o9fK9i6d75+g80cYtrtHD7eE56OyLu2b7PuNpinXHFswBGcmFILA+\\\\\\\/gE7IJNU7\\\\\\\/FCQ4NJgvwGTGozBNUv+DiiBlu1A\\\\\\\/sU7M6nWam5t3Ot5\\\\\\\/u+8T7qN3NiV7LvQsmpTp8gJ89J+X9dlX4jenOwroSHSdnbFYxw+NR32ryzLgqG2nazQse6gTWblI1gw5L8PpaP7lPtLNqcAufxBXvvjecy6wyT8heZMxh54KskTUsdYMtN508p1Yz8WnWk8aWnFTRyGEHuauYjtpqLGhD+dQHeI\\\\\\\/Xil0NPGqWhEitEPUBD3Lz9yRSwwJefrBZMGB8zh\\\\\\\/CGzreiLxtaiyxJO+CTR8b7bHdY1wNN58oGz22z+fS5TfjlRdcTdpcvZBTZzXHwRfsRXfPdxyLhOaWpsTvxcIKeWQph1BJqOpSg8J3YabS9cDm7xZpG0lSVuPZNQcNxAXiH9rQ9sZHGoJBSEBrhnjPOTUGhdKxCdzfVMsdrAyJyyju6pJ5qdiCh\\\\\\\/AzazxD3tBfB7zcAMBuwihhdb7m6STIifrlWCD3WcQk+atIE2fDywBXRL0hDpL\\\\\\\/kRpihZebilhqwzmXKHPl84\\\\\\\/Cfxpm9L3QqiSx4t8Sq8oM\\\\\\\/LSO5A7l7SHs+295frz2\\\\\\\/GROwO76y4zl1J\\\\\\\/E63MomtnNr9MGsyBgaFQcoi0zhlTE7ysTWciiylbWK0RB6G8klgfFedJcHFzUMMq7PB3D1w2kSGOcOqbzxqtwqn8Sh1tOysI3lopEq5F2sq3xvoyR8e8dlMI1wS\\\\\\\/nBw3bE75oBZmY6UczOOWM1X+rfPjq6H9pD1wVhmYC74li+L\\\\\\\/apz1RBfskByGdmWwE0Y3uDMMZfindTedCR8SEWvY83jp21zmB188Dt6zltgvPkLhySFFKlWCJXotfrH3n+IujjqyK9UIypmv18OpXmwDI5uKI8uDb6TSqZ83c\\\\\\\/SckXl489N+yZXqy9AJWwoQBb0K1QbKckLLGJZ1S+1aWPWSfnlvVAhdmxzld584hliAUqtuEM+UWJ\\\\\\\/oQS6SDJRERH0dOk1gMc7NWMieOSzzO3rMgVvB2w\\\\\\\/isUg4MGgCgrM8smhftiDDOxyMIBhbxuWh\\\\\\\/4\\\\\\\/UG8XK3L5M53snok4opw7rZn1VECvqL7zjxY+NCOVEyMECSrAOtRqWUdwiA492GIZLMsKmndkskJVePNXNrGvUKCGylG70xlZ\\\\\\\/PLEKelv4er7v1rhXLvCB\\\\\\\/7E9gMsJIUDaeg8YkMm3ZofAfewniR3JOxWC6ZEOdL+vR0dWyUQRGg8lefk9GZ6xaXNwdBHb0Sbtiju67pVGvm9eaIndLdUyfLiSwP8iB8kyalLtVxtDNGOzMupBMuJ8uwFYSgbKppcYP4RFplfZH5h3G+J\\\\\\\/0RhicCtuI9jSjzKWgNFAY58fKy9fSLg63PVUoW9XOO2YbQnVT6lU3+RnyO+VkDRcFy1WwLomKj3oF33kZtJY85KAtrDMSvjWOg5+sJaWm60IU6lAnlFz2Ldne1d7iCMGJxVxLoz4nVbvL35M1C3wfVdbPrQB8Lv6NxAF\\\\\\\/oi5IsfWANhb9LLw0qoy+HqpIigqMn8QHhCR92Y7hUCrp2LImYVm432bTg\\\\\\\/fUzgquRJhXCG5qdtCct6JhzixDaaOA4Hmo8NYyJjowcQsOWMN9YaLwWxnsL9x8VRars\\\\\\\/oYI49K3A9pNlOllYlb5dnI09+7kTRi8RCE8oY0HTUYDSotXmympnkQlUvsJDNOb82snoDpKlFhEk+DJ9nXCP279t8QsitvYfqErTYbA1+CcyGhArbwLYCtSFI5pwJw88AjSN\\\\\\\/EUkPSvfrcT3\\\\\\\/fG7Aj+YqVNHNaG48ZGsjsixzGje15pTnFZNsxG7hBbnzQslOWut67jWno4YKu+i4ou8871XlZn+fMPsDEKrbynqF5o\\\\\\\/NU6sbMxXI7FhcUR2FYIbXpt1n5FJVUwfORu831d28vhuG5BW\\\\\\\/XRp5ioMX3x6uFqjS03zQzdsIsnzs5jx0TBDcI8nEIj0tbh4MIRZzmVqty3MRwKx9IcEkmXA2ap6AxDE0RiRiqKLzOpQJuFc1i+ZhWpUvek4Ts1NZm5L9ZponZsHrIz0G98ek\\\\\\\/UBhNsx6qN7\\\\\\\/hgIP1wcEmSNmAtGBmNG7k\\\\\\\/+SCxCqX5P5NJ\\\\\\\/fnkZ0D8XisM\\\\\\\/wzUHgQvckyM4Lb+T\\\\\\\/C\\\\\\\/ko4HkzfzvgKx0leb5I\\\\\\\/Fsr8n0NNfFRXom3ZhwVmhv36NnaZKLVoEBF5+LDvSBoJd1ae0u9q+ERKGIWv37ERtFbGaaSZjZyU13+zCTnlVhQhTP9UvWj2INkKc+PoubE5SKQXwEhYS1QjjH6B7G8AwLJiQeQb\\\\\\\/XwH+WfZa7TYbbk0hhTP8+pQMMlFTn6SfadHRq4afSycNy2+5vzzWW7YaULw7vq5y6WFqZnUkzaUn56s27XZ8MFVVZyFLRyE0eN78z3Nk43ezfY4kqpNrO9AiYWUApioqdcwINjgfetXHLaj4xCR6ouDM7eu2Zxbqv1K1HeOwys7JkGgMJzKl0YxTntTFKTs4uHzf\\\\\\\/6\\\\\\\/8Uwij3VAgYh6kEAzb06y7scstA0TbpQGaAm4GNRdhj5do6Ut16MAHn31VC5Yjj421qa9dy1BxIGMATdgjCnjqE\\\\\\\/\\\\\\\/42Mz9Rj1JAAHGHYI1LSTK2aXmgkM\\\\\\\/za2G6K4Lz6x9AAOJbjTZMPUpuE1i5r72kIu9nbny8FDQKmJZxzmMNxCdQdMLdD8jtfTgli6UsIjyeJnUIH+ySkQ9MNnUpb0YfKnuNbhDbgv\\\\\\\/PMeVSqwtT8XJPJRMn8W9mcfusLdLqF8+XPz1Mnk\\\\\\\/j3FkV24lqTW7LI3DNnPKhP\\\\\\\/4CwAVorDT+KhJohtd0iRh7Q2wIJQU3hYz\\\\\\\/Hfy6ELr\\\\\\\/kYF2HEQ3FCbbR\\\\\\\/cLWZtwBRDkOsg+ro5gknp4mcGG392XBZQC3hpKk1WgDMl4SXlp3GoeJKA69nifEgGdGzfYkUkBQNePFe2T4UNrUITsNR\\\\\\\/lJjGEZ8U4kVCn6oZjVzBLPWOeQWL6mSzyndOPBtY0Oi1Jc5GqKIPHzNfI5EARK3C5btGDAN1cGXT2kk\\\\\\\/DRQdlfSIzCsPUAs7SWxfsNbn3s13ihdD07Dr16N0PVFl2P\\\\\\\/MvckEp2L67EezvlwELTlJPrOmDjIAElMJX77yH2HJeGgGzH6r1eGHv90HZKj8Ws7R9bgpeJKxeP75DufyigBk7TiU8FPuLJFNDwczf2GluZGVsyiIt7oXsEd0ZOA882p1ApulKsu48jsqItE3501iUOutNkFht99lGerBae6b2W2YhSAu1m0T8Wsjkhof9ixy8e2igztTSUNjp32a4N6CpTOZ8sdw9HwUrN0rfRYXNi7Jv8NLDjYRgJIEbdzAW1Z5lZwmHHJlqrZ8fO8uxCElMyzDQPV1qD92+O4NHYiyRgyH7oaNyotrQWX4rL1ILEcjUOZeSIFw0tzyX+zONzV\\\\\\\/Bng9nuApog6TWfEM263cXTjgXk3wzdv\\\\\\\/I7EOU3yf81Aa4iSY6T8\\\\\\\/X3OBuQ3AJZ99WhODiSdXvemKxxEladv\\\\\\\/L2omBH5jrB9L+9zDG9YAQHRKf860iq\\\\\\\/RJU1OpVbzya0hg85w5ycn8chgUp9EPLmdj0fFWYsO+FwC8sMUHVNm4Ag0\\\\\\\/0OkWOJ5KegrFnHKVsY\\\\\\\/4h+CfiuEgiDtNKMN4eJeULCQzbimP0Min29Mk53KXNbHmyM3njepb8voqVIaIFu6PFNyw+ES7OosxnYOr0okeesxAnUXMUNC+lU9kJBhehAxGeYFxkKT38ov0XmMSSSxYpMw4kr1rXsNKpB16\\\\\\\/WNm19ZNjCkALpgP72FgXZmRR27dS0FBkPhOKN\\\\\\\/bHegS1iSFHgBWWZ\\\\\\\/9kLz8BZmbZIeX7+5n3GbsICMB0BTOZTq4bjTJjW1CzDjGwVGdjPhhuOErO0cH0jp8ujXL9W\\\\\\\/whhUmFMnJihilL8k8bDnU49pY7H\\\\\\\/\\\\\\\/FoY1oejgR6o6tPoI+\\\\\\\/ZcGMg1kgIPsh\\\\\\\/0QVzsJS5RKPFqHg+03E7WJyGlLYNHbBwVBz5Up1RzHYZCy9FJJ8TK+AqeW29rAidPS0alDMGShhPUvK7juU5Z00okmbZuJhiHL5RFLwWQ6X1R\\\\\\\/tK1Y6Vuv5EPV1KQLCuWbFa9wpPwpkTXufjTKuMDUyKo84Owi5ZICS5H5xNqks8PJaclUIIDQGYXxvRCufxJsJCaiWJopQ9OZe5XUabsTMumDaVfYaLgRgayo1mY+iaYAtUblKCSKqKGRFwhwDGWjrCd6zHahgu+GjlCGdJwLFsk4JgopUReZck\\\\\\\/R+oJhYoS4vpCfaRr5qeVMcFwDza1lwgST2FFTXfb6\\\\\\\/9aEh0dY0JbiLQSIy6zGUuL9MSPWqYPHPpKtAitLJ83rzAWSOHuU9tDyWqhUaDEerwdt0T00LM0kgLLeonhlVeI6u+OnOWh3w860vV9yB2ugnBclEg8cuKRiirVthmkvUFtT12hyGQ+LfGRwoODvYbUWhYgDkn5FF3d\\\\\\\/p8zA+BTBMT+Bqp9xGjJbKk7xGcHC0hhymn9ik7iNyqHetdQktn\\\\\\\/OJY9mEBWi2iV9Bgcj6qiHwDDDXEvtxdVtX8sO6oyaae87uOooabRDJS0aOvsdnRr6WTSEgMM5rlqlMyn8t8s0ZBhe4lloARWdMkg9GV\\\\\\\/FmytXfV3Mnv97+1aJqz39mrsK1sPMpdOX4It0J5IxMrH0TKhQR5No6hfqQWb0NihvP9hcr8\\\\\\\/WWx61Y2LKFGH+FEDyxF2yvRyeCmEXOSD2MvWBkQabLi05iw7M78essliHu1GKZt3Luld0Mtha0ZzSQw1Y0xmAKqUpOp47lnb5AGH9bozi9\\\\\\\/szg7LSSNrrOHnHdApYAWnj\\\\\\\/p3YW5v1R+7N2edh\\\\\\\/wnVpGYle1W4IPzlIENuiPNQufHrfBYHbMuwgUun8TAWlhLTiL2\\\\\\\/61BaoCJuh14c157n3c+WB12S9JjoRh6ObNBYHLW7PVDDXWGNymEqrskVF1mMRWj3phh2uOoD8EeROeStnCfG8174QV6eAXmvZ0g5c+SdaD00iHbHbYFQFOvlA4HKZbzQratt0N4xEkI5+PTgAuVYm7rga4al1l09C9iFsTcjEIByKf\\\\\\\/K3HWMQNt8hl\\\\\\\/IMHlv4bZeedzXLyNa2wNj2zxtpM9\\\\\\\/eUd2N8W\\\\\\\/s\\\\\\\/Q4iGFhsGuhAB1qKn+eiUnOhIXEkP0cvKTsWSKLcCOCm8xYxVW+Icj5VbUT9xnaHTvBCxBlaF+WuEfgqeLrJc8iHkaSvW349NRkVT8UBe3hNn+CYsfozw6BAgjJ23EI2IM65egWYFsWlHpihh0+6R6wxjJXyVX6JgHcG5GnPCjHCVtlBGX0BsZdNX\\\\\\\/8D4jQBNbrcf1k8v33oLQGYhyz3lgge+DkxR+RX+licLBt4oXOLjzF\\\\\\\/tDpn9IE02LPwJyfDNk+GokSp2VxW1L47UQD8thD3vOhHAvHZgxTO5Qy2cy9b2QtT7sdmO1msKEG6b++3Nua893Hjzex0WHzHQJ3n6jYKU2FRHHmrCSGRSmzU+1FwUUNAMZhKM3cdMAqIQlTWlGPTT3W\\\\\\\/9gtgl\\\\\\\/uSKO+V0iOz81UJZwDJW8T595ci8odaxEu6BxFOPh3pzE8f1djb8lVQ1Wj6z9WbPxm6ci2Gj4pKDwTyaoAhEqHj6jE+fvrLXTCV\\\\\\\/FFFfOXUaf5WqWS9adTVDn6dsAXbj4uDp2uku5RopPP2sZ8fE6AsvK6AzxiozRji9tPbyUBiv\\\\\\\/Axdl6+d6IjgHhSXBYd+8cricRS3mD2K5o8PAT\\\\\\\/2Jdq96G9Ji8FjC8sPkTU1fEbc1GbPoWSWNn02DW4idB4Xk6BLutO\\\\\\\/pepqZy2OpAEk1vBY1pWwvCrdTvX8Sx6NfrrjXxGSIGcvrkSml\\\\\\\/035nnRJEQLJrwFbllnWZNo1GANUjzyy+iER1bg0yHgi0Ez2YjpBUleIadzxlL7ff4wrCkUjvpOXydZLyJkDo+TfsVIkJjvyhV31rpl\\\\\\\/IKohlY0e6F9AnnfUX3gX1fZ24QG1xree4zeuwo3T5Moo\\\\\\\/JnFK8+vjp08hCfB9fb+9h6+RpvN5WHbaP9SECpB5y6zkpZ6spqsiMxXonVZrG\\\\\\\/OPY1CcCBSpXaY9+NJNJlZPWCQEET0JEloEP908HwUSNpODthpOBsw\\\\\\\/eZfDS3cjueaoP4bNbhIwk5+ymdLkIF64xRwcKPtAr6eHIcUQXgzr0WXvZYNuejpS9gThh\\\\\\\/nYrEjwRlZLoyflEJyN+8wgvWz95obKpW9ImEy3duSiIXJuAOYktX8Djw\\\\\\\/9ByGCWG2V4AXHB376eYXmC8DlZEKc6ORmQGVbRflD\\\\\\\/ejs0xaudR0+aRNEqa4M7eTVsHslL9qhdXQZvtdTz9nvDnMTZdbWxatUwnejqCjpjA\\\\\\\/5UQOe8JDFpzqu9pWXvQzujzHdUA68ZDsVMKRINzZEohDDM9Lr+6XTwSh+fLfo2+mbwoJyhH2GHFFGmrhFK7POl0vCCBXjtLfkM1OTkXo5alxdoIJ+7MXeJl5hZzk3IMtPDI9uUZziJwUS\\\\\\\/BZUQ848Fui4ZqPgoP817JENDYh3rKruJ3XbBsU1hpuaYyyg6NvPV5G3vYxgmwBjqYtfh4TCRxOb4RtIacwul6IF6OnsY4zyTY7+dSY8q1BlEUse34YD9MGpdfRKRA6F2K\\\\\\\/ZJTY1I+\\\\\\\/HIZ\\\\\\\/tXTHxAOSDET+0xVA6v3hRAN7XNtiYD2Yjvi\\\\\\\/aV0jipPIs8V6RlqhSumVs\\\\\\\/UT+6+i2mfguL0OY4at8Bh60jozFJYGesnQeDPoEW6LiowjxzUPFlFG3ZasgoHdzD5TXRQQr8GlMG9p8DGC9fvjLS2ItrQZGl4Hpz+78n07uY9\\\\\\\/QgH\\\\\\\/uiRmu90Wjh1yFKD8lLGO8\\\\\\\/asBtmMTfygoB7mz98WsDZMpZqk998QJvyVoeRgZfXF+OzzQtsGTPOi3fqLUPWNWiNdOmc2XT3GGnURkPrOUsW+3Y9ZxJcjv630uwjJ5gH+\\\\\\\/ee7wtyuHvGyGlXcxWa9V2SXJkhK96JiKxgr9x7U1U8Gw5q+HnXv3PJdiSOr7IIqNirheJ9xeZSkMGUt5A6irldpdBmO5ffeZquK26fzJwrEvs+eVZHkCK1PTf0WpuVvhyY21S5n2mr7ETDH+vnM2LUgsNvKUlJU10z3OZC01rB2cCtKftpFvWWtmgdZh54Q+oVcFBIXKyAXK3CvBuJ3vq+rtjI+cMiC+XDUouPMElhcV5TEnsVVTzp1LofyB4VFpg2tgaHWDaCjNXKlKxcUZmyuP39owSqY\\\\\\\/9O9jLGHBE\\\\\\\/\\\\\\\/UYu1lUgRkcB6tZOW\\\\\\\/gE1DWnYLpY7aXa+codoaqeN\\\\\\\/or99opGcl74YS1iHqB6tiRLVNyL6LZi5IHR2uvh82lzCsbCrTwZabG3TrVG6P6BqNKynj\\\\\\\/GYpqIfFtXxnNA6W1eO27lHuqNv\\\\\\\/bUayR2ERbILAUaUX4BFnXgEqqQIE49sGERXKhe+evQKww2itidRatplvDW7kOMrwwt0uUsW3LH60I1Dp2s\\\\\\\/YF4qan\\\\\\\/i\\\\\\\/8yxUQ2QRlb7vktbLeQ3hiGbXP9ELwlPgZWpwYUpw8BOFfSFkTxuUGi5yl\\\\\\\/uGofLqAhWKjUfdjhedDiiDyy\\\\\\\/a+4SoFPcDIILWEh3DP0hi3Oe9F0osjNmKZy5TzvWki6y7N0+9uU8aESANqRzTd3PJF6jrBHJxzHWFdS8dixDfGy0egMPeA0ee5Xke60BKbnIdxQKywS6vBXFbQjH2SPMtFfAkBKn5KUs71PyeUvyTgQ7oTYDtse+r6WQU+ODvBdxuF8dtAK3lNyap0vnEwB512eEr+I3BTF5V3JPzJFmjaIQQIneNLLbK7YhAkY5Vl2bTTyUVxb+I\\\\\\\/89tgSBe6PNKP+iNf265iQlc6+kDvWBhaDoPTJXiduzj1ApSR4QmA0O4UkdPJ2eY156Vk636+dClmQvUflF0E6oIZ5s2XmentCixywbYRy2H0L0vpXbX8n+f5tF2jrjC5cHNJVynKqWPNK1wLwNn9J+QiMdCsPFSuTPl4PXyTbcWN1g3LIMQeuF\\\\\\\/XsLLspHGXmYQV5oSOEzeY17b3FO7yxumsR+E8832zmvZva1WL+Z76cJKXVWg5Lv8xEACtihp9V12\\\\\\\/LAUVf5EhkaN1zkh+LNFLhoLL2bAEQGajvBkRg+CRzkWtFgfPxy7eJWZuP\\\\\\\/MU1GHZdL1quBIqZvZpHMIDgit9i0nR+84TM8ZTx2fsi5KgaEPF1UvO8mZu93Cdy4RR3a0Pa+BsUP8J2hZOlpkb2MqTZ3LJ1xFR66Tk9bzLZ\\\\\\\/IAN+Ixf9kCcY693sy+RTIh1RXtTncPnxWHshyhCK7mGe9KrNHydlwkyefM1StcgqNEZCXnivvu7HPxb7PbGYJRnC4eu5jlV3FgTlYdY1a\\\\\\\/oo6ojkMU7oNZvl1k93JB5jQqs8\\\\\\\/oqur3MV6Mp6dbBXFfWbiXhqnqVMWspBI9Ohc1rTKwtZCCTrI6CX5mhtDCuM3WGl7zOAhCOGStc4s1upSwKViGxoZ\\\\\\\/T86bBJGCXwp9Ae5mb3ubLFDRcLh2CYlDa3jx\\\\\\\/YJGjv1pHlJ\\\\\\\/OMm\\\\\\\/DOFA1TdvBEFEFxaM1StXrVCQflyHMnyp+PpLjO1X5lK1uQcS1bBzlvKBko0sKC5b03xp+\\\\\\\/bEVMtxF\\\\\\\/PJ4ZgWgncKEkBGR5\\\\\\\/PVHm+4+oi0ytUg+dCMhCDrRJBILHMhJzuIPUml1KAkF+IU6LPPdfrvRzfZsPNbRTJB97Irnv06eh4ty8csaDCd5BWiaXuLN+n1eeQV+wjKa2hz7FI6CRzFE3aLq4xdm31GhQksBVzYw18xfOoWBG2VK8dZpiKHWdqVx6+vi7euB\\\\\\\/eD+veHSmuJLg3GPEiJVbk7xGbGIMsd\\\\\\\/XEXpql8LgPN0vSGDvW\\\\\\\/K8YFBBYZkj2eRT7VffAhrrZeYXs\\\\\\\/N0IiMND8fEhAE0FgGdF90izQSOTsYX2rL\\\\\\\/Cbs7CPiOM3JfLhSK8C8s8c1t45m76FOx1F6Fi3vP3e\\\\\\\/rCpoCwQZ6JIFQF\\\\\\\/ftbyD0tLHi4r\\\\\\\/ECZndayDeEKzbfUNndBg+vDu71BmPO4dr8iTjefybxUxwFEZxS9DSlVYe+vYaaEbH4xsBzMlslWN8mc1DoX6MEgduqiG+obD4RVBL1BonwyjKR8DdTVYLI3iPz3ovXs7OMTtq3cS+GEJ2qY8ch7dsmsX1UwXUxWvD8WTF8uO6+p7rwbcSDMIPb3NLEnp53e5cOcRNiYlV2kq2ZS+aMS\\\\\\\/z2XMjMuJQjuG7qNhWgLwasG\\\\\\\/q53NBBlORMF6bJBJmGDBXHfqJJDNjjwIW4Ic86g5r9TpWwV2YzgjRpf+VAIUPwRmnijYntrLA6++z3V3NIpWKZkP9E4\\\\\\\/Zlc0P7qctulPcfHToJG5rHvUQpQU5g1KwoihQZHrd6Ux6H0+L3wtf2RrExevfWEedwbLnKT\\\\\\\/6UvqDWe5CKmO3XYBproXdByyKDa4WSN1DZQcKpKeWCv5fJBxrV0KIb8UYTeFhZ2fe6n\\\\\\\/+Z97dChiTWrSQrIkPDoSpvAI71ZWbyp\\\\\\\/f34G6vHPbUrsQr4F9WNf6LM9sEDSBSknE9ibdnxM0BImXNhiFGIpbsA4MUABiR006IX7GfwJ9lW8Tdh4xH3eSAlG6y6a6HpEPoYbb\\\\\\\/4Ov9bTThksIaiHRhaAGM\\\\\\\/eLPFIBqUOZJlAJ0SN\\\\\\\/YWKu35hXS+G\\\\\\\/t+0IXqnnbN4FxKXgCMk4+a07wtIjaAc4vRCV2013oqg7zzuB6Ft8tpbU4pxkt7W7CTl3znw0aI0fUM51L7VJXHx0NgvnGdgoJmxmE+8ZLgFh4Tu+\\\\\\\/hrqKM3tMOnLJ5QHyR0211wIbWN5KcHn7G4bc5YJiQodd\\\\\\\/TUE1r9x3FhBRYG7xS0Y1N7Whd5D9LNQdtB0g3YfefgD1s+01gHjxYrfleCRQY9T6MPRsm8VuGe4i93lMwl5joxFrzAV6zmfFS1aCPU\\\\\\\/0PVgbVk+q7dtqaYFJ\\\\\\\/GA0wvLJQ6hzDo9MG2GWua5J+oqynUAVZI+dXEtaVolzJYQn\\\\\\\/J\\\\\\\/u7hHB7lmbnDxSsCuL+w51Vcn\\\\\\\/8pKASD1+GmbMAlfrkBe6PyaVELROs4dNnSxOsYs\\\\\\\/mXOkO5A3pZ7M32vQSa7K037CznQK+fZvb+rQv5\\\\\\\/lLLTAsBtjaDFsNw\\\\\\\/8GbUkqdkUs0WAkB+iSavHTHNBbgXkPpT7H1mRewgFx4X9IcbgBVx3jQfEpenJ76k50512ueQ7ayu3WLiIiRve7oQqpRWhW\\\\\\\/RXR7HvhPmy59CJSJY5ApWCfQmAtC0VK\\\\\\\/BEI5QXmePmcRullQ7R9sD5MFwF157DMVXgGPgNoT0wQcgrUZPXsXaa1w7EaSKMvzNu\\\\\\\/7GEZLXkNHE9j6QOHtIy1ljSEITpMcUnoyf4sTLOBynOcAZGV0\\\\\\\/ayugjeYB2YF5vXTEvhM9n0xsA8V9zb8beniN02GrHcvZ6Eo\\\\\\\/oqtlBvs9\\\\\\\/PRL37Eo07eO9RsH\\\\\\\/n8nj\\\\\\\/q0D3dYvJWxVKU86aVJVW7i04JdisVPy4JmdsftoJu4iX0KY8Y9ZkkyIWKTUsaPTQebClZWPfOH\\\\\\\/jG0yvtDSkdlXjwe8+5iHFqwG9NbP2739h0RZqVI+g\\\\\\\/9kSjr95YiVjxYYKM7UvLminsqiOhdhCxJel\\\\\\\/wL2gDBt83t2J0w5BDWFRQOtfo5hAe6Fy07\\\\\\\/gEBeVTH\\\\\\\/McXKp81xZANIzvwUt9DaB8qIElv6Fk\\\\\\\/2Dk0USx2pxuO\\\\\\\/SjFG\\\\\\\/mhzfsqUqVhzSaErhieKAJmQy0a5rVcbvylYudISPGnZNvReNrUgyvrrhwrQ8XKHv9Tl0Hmp2dmlzHzsvlQjvyBhfpUGqL3pKKZ+RV30M1wqRcOR8S\\\\\\\/nfMWPVonBFsObeVATwpGwr8J1pGcx7bhqiNyr7ZmdoOt2XQ4jzi1MFH1ieDtBpC05FgT+aBLGfdeSEJ+vhVvW4NHYkqgcUIG\\\\\\\/MGyKXES0uMuBN38BTTkfRxAn6u4x7mlg9GUuEjPOu12ZCrbsajW8wZYhPkdkucFB2VDjctJeNyChmkDuAY5+57X4Hev0XN3ALBrC2vKwcRtvkHdH4JzogXiDBbmhjCcGf3j8Mu2IggJRoqeTqgE1pa5w6x15HdbCGLFiM110JZUCTjWl7egAzJ\\\\\\\/mKbOLdXlKolxCs1U6ZDIKvSLMs16qrPR0qi\\\\\\\/V8Tr9dlcmQtNME+lOpSxuYarVdySJ4ExS1tmbsJjWD2J9i12AoJ51rN5EORQQFSOxl8ZSXV8mlp1Gld4tvHYbzRhO+2OFCC7CuMlCNk9tF+ruWMV62dTzhBQBg2qghCj18NwfYbZBSculvj9XfGBHBxWtsRYEhKGMYfwgPj8bFUHn9sEGXZP9H92LbCx3dzc9HJaQtrHLBTpYtXt76eIAc7yM\\\\\\\/rFYfgXITGxR\\\\\\\/9s7vz8T8ghuAsiZEsNLb4426XJMqpNmwEN0W7uoOvCMCgEjc\\\\\\\/o2pLkTLX43x5VvCSRuPdI49hB70z\\\\\\\/eBxOaGAU\\\\\\\/Wg4ULyylHROJKKZKg6Nn9PzBsv8snnRzPg4rKlkxlVa12NCol9sYR7JL1hyF47ry+wcDyhslb4PoFD13aDImXJI+kC+FkVR+ldMjuBrGP1O\\\\\\\/YDgktaErdmU\\\\\\\/vZA\\\\\\\/3pU0+iOlJAH84+TrGK8YPNO3p1T49x+rKfnIfRbin+Tu22igqaCff6LSPc3BBLXENfqPKwUXJUIne9gN048Tcp5PulkJoCdfk+Vu7jzi3p7E5WJtc9E+UbWdUXrsgAPd3SysnoGTrFYGnbudgTwLSMYOKOoiQG1UxqsbCp1tua0z4bmkxNiwQ6DoBUkhTPoX\\\\\\\/N+qLNNB1GFI5qPfXygkM8nUOeOvS6uzXgdUlA4jc4RG1DOdMRt\\\\\\\/qs4gYfJLm\\\\\\\/BzNsS+vODes0IPQpcfs6Z6vuyYaG6oL2YN\\\\\\\/voql\\\\\\\/3Od+3SsbPMvH7uEB65SpSz1v6JnO8OaVq9DJ+uXpLWE066rqilKsttXzr4w9lOp+BpQJ2g17549UGbIt0\\\\\\\/BATMxzbBiJDMRr8H2UNKl94HlejA8tAozg06afxp3idV4hRJukmrAPVL9ojgMm6JcoNRZn1siqNlm\\\\\\\/6KF8ssmL696vzMS6BLDr9ZIyXI5lRXfUmporp5MOnE\\\\\\\/pFXu57HJeVQUFHzLYkV2B7+H9fPN94cDErCoGpKprvk8WjLgKYcDU97crKjTPuS9QE1cg\\\\\\\/+J\\\\\\\/X\\\\\\\/9cfnW\\\\\\\/RqF7\\\\\\\/HP4eLldji1JUPHseqofoV\\\\\\\/GVEmIQYbdrWjEB8Nl\\\\\\\/Xd2KQb6YUK\\\\\\\/gDGJRvJhW6nNhJCYKH7ViqdogvF8OjhAcCFXJ2FOsQdxyE54v0zU0Kx1vK+XnH48Km9zVK9SLsFZ+vPIjX7XCFN4jLWTuMG53er3fB17rQF27mgYQtgh0wPWqEGVqEaV5mHzj8pZfWZk9E3ltNoId0GGcrGL\\\\\\\/mW9RSythcYf6lzypd8Bm6+XLPlBJbmUnfjW3hTMc6Z3dO7S3mr0YhTes2V+R3d9MwhflmIV1+ehW9rkcjM5idk62\\\\\\\/01NrqW+SEEiZbJc0i+8pjrDC1SnnM+W1Cqv2OVA55\\\\\\\/b40Ui8eZnjiBnDtnF3Qz+53XhwiViVF6Uqa00cSgRKJonGLV5gYB8RQtwUiTqXCdgQtjyum5f3M0eEkFtNHPB\\\\\\\/H6Gcv+5ASshp2jcLS71rutnzBm4ZdVAbAwERs\\\\\\\/960EcIBoum6CX3BKpJQdYcc8XLJ3AVMr6L3knAuYnSagv9IG0crNly0prD\\\\\\\/w0UVLR3EvHQCE59un7KRf6\\\\\\\/zT2ICHIGsNuWpElvt4fMOzsJw7Fg5YNx7nrGfMbAGEYrLGH1gA\\\\\\\/tyl63mAG5DboIOIEuYs9cW45OPmZj2B7kUpfENNa+3jw6RXqReKDMiVDyEh\\\\\\\/RaQzs\\\\\\\/hLQueZeKqbiRKKSomLFLPCoHMF5lh+LAAMv2k7cMafdMGhpe\\\\\\\/uCXj6o8SJfAX8EOpmLTaC4YZFjZZrQxGGzqxc6NfeuJ+Igp7d9kZnvEio5fLPX\\\\\\\/UvGzxpz6+aSgu+qn6Eu4goAf1zNxXvxEvpUcEYnR+beyaUgtQ+Ap0JklH\\\\\\\/5\\\\\\\/o7gp6cAaD3nNi3h0ipvY4gRzos8pNPwY6cV6xM2us71Uw87VOsZfoGhplFbcuO6gfOUOWpWCLQj\\\\\\\/q92OwTNDL9ekx+w5hBrwoD0JQtQ3QxlUcS4AyIsq8vRZOp2JCSQev8uDM558unLmXIJC4TnGL42ZLR8kIXRsLvOHXhlyPTzK9dhkPwsveqnpe5beQJF5bU5L5BNcjszJiCBn19nn8hTfYqx6w0MOvVuBhMGc0oIiVZRDnxcOEERxsE5Hs50N\\\\\\\/jNj026WhC0mPbCg2EmzFCwt9enpfA4iikNEzEne9ymToWNsZVKX2mhumu4UO0+eZaFZIP7hJ9c1Nz9nhp6BBGAbsuOZ4fn2EM1UWm1c\\\\\\\/el3uthqC7W6+B7Z+ReNsHPDh+bnAb91ppbWufwaH3RQu+XI3sjSqgYvsKk0aeG91+Ct7FyLfjeYY7G4eN8Q0jpUxR94n\\\\\\\/wj5T\\\\\\\/L16D2T5+AEC9CIr6Nsdpv\\\\\\\/BP\\\\\\\/1KHNWLxUfXnHas4MUa8iyG5N\\\\\\\/pXb2bZ1rdjiFtb4taANEpYFxiR+eDSXrQVPPsTQEq\\\\\\\/LXMpi96I0cR+VDZyoaGNeSJYajVzdfKeaXqLyYmiAPEPQlJjJqN2ehSibVGq7CtGHvSFGAZakeI+JOIreD5l63ILtzVk\\\\\\\/g9u39O3DzzcBeWz41L61ssNqnLpEazYEu5rz43FoWVgyxlNyV9X66MkOHqGL1pq7yWmvAwh8nn6D0NYz\\\\\\\/E9F\\\\\\\/r+XoE1vj6nOlWIHtxWmlDacbNbDMwy4jP8nQZSBZM3Mit1V0XKXM3oRRThKFo7xjfz9PrILZOI9CafLNq1uj0rEkleOXZG2Qebf56hbTYVSpJZgBPPF9aFkv3tQL2B0aVxt26Mcq4RtK4xFh8bDCHhs1KXNhuNkcopmsUXbdmHEXV\\\\\\\/6pFzyJ4lVACJqpB5MMvrOBXLf8HUVsw8p2+joHIVqHnTRKntViGtBtPVRMZ1cIdOJY5LO7\\\\\\\/mnHQDDnFfj2\\\\\\\/+AsdCRi7BjqqUNssvqy7cfgJZ6e2CeSaUBfgaO4XueLG0JIoLn2ytsrzxgodd2ZFd5AVX3Z01iChXKu4Ku9LVMvMB0BRRKDIVkFlpA0Wy\\\\\\\/pIJwabZOBkFY+tVvvUunD3EvMatgI5FyKz6Cz7eyEjkh5M0jF1vS66DK\\\\\\\/MNCoq0nEt0inID+Js4+rSzIklHfArfJdgsJHthKetCKJWyLxG6csRZRCfMhLjGlg2rZxSAnU+\\\\\\\/MpbQy5nyOkGWVdMqZ6TuSskhfSmC08jEhI8cHAwDdfr+oqdAzGvlT4OojzbCAVK7DMPB+OUAYqepAgPtCjOnnBrHN72cT0O5pDEN3C88PJU5xinXY2WkgLLaTPN4OUIQ8\\\\\\\/gLO0V1t3Lwd+sdcLbOKzV4ZdjVL9fN\\\\\\\/wScuZMiYMZz\\\\\\\/MlptjkYv+q3MbqKVUpo6Ds+yZJt8S7El8M8EfeTAIMzpaYjAievkKKmeSfbAASTDPxrhgUfb495dLySIP5saSYB1ZpK5e2bSX1s2xDU+U4YZSoWe7EVR7NLcQE3StGrkYrYKlVoEhnMMdtWb86iXZjyOlKr95S6ZvVTYkcQCtIkwLDEaFStscqXJdBwIg8zo32\\\\\\\/xVqqiONKd21NskXN0r6z5Q0kMW47rXVPOxCJ57GCmEhdC9kVxGegMM\\\\\\\/hVP4VoIEihBfWkx2v0buXisdfIUSUgyXC31pcvrzKxI2s2RHnAAZKWxk0McLHXkRxEGM37LQlTlivAwyjVwZgIotogrgvFTErQ+9kJyo6OvTDGQW1ofYBlCKjAj4q8zs8wBzbek5txtYT0PyID1lZ564CLK0v34T9ffdC0YFzKyUv29A1ys1u4G6hrCjoJoYUCPT4cft8DB000unTOr312PvNeeKuhYn7KEyIISlR88oTuDo2s8ZYhOkHvmgl\\\\\\\/O6hTzDEjy231W6x7vFY9XGOGj3HGx79FcGjl8Ucc4kQAs9NPPM5tj5a2UPo7Fj97kiH15HwH8WWz6W+a3DgAZuOlLKf9g6StkZ4wlSxr9S5KhQCesHSLt\\\\\\\/WzakDJZWRSVu7PVJCdbMU\\\\\\\/Ezftp6nQqpBOvp1UsUZR0sLtPMVgP56dAArIS14TMYUN1wQT6Bhx1U+\\\\\\\/C18g8\\\\\\\/W3TLkDydNed6+iQoICrJuG\\\\\\\/dk1\\\\\\\/NP5rlBnIUsssLfxHHd8K\\\\\\\/mTgku81KnQmfhfPzfyYDvr35xv2OCNQq8q9nTEmt3ma2zkHOYzY1PVQWfYPycSyXvMOR6d837hNjI5c5\\\\\\\/GF\\\\\\\/gRqu4TmgogDGhgxXog9tYwfZBvk0vtbQNUE8HyT3Uxm46scPKWyPVg\\\\\\\/SEV2AdUNNm7UBqm8APGDUDpFv67h4CAX3wHVs2VWaj+Dge8\\\\\\\/lKIxzZxyI6dHqoxR4JZgtXoJHMYyWVyKsc735Rrxz7FWWZIlUVeO73UnOuhVo2NCV70OfdQY2mRn5QZdScd+FpViNSfRaES8ZN1iyEC6H3F2jqgcTxTKiZy1fhp9CwKT4JY7tF58Jba+Tto1DmORw\\\\\\\/JTrCsxneH\\\\\\\/fi+7ZvjOrcwMN60jyqtcnxmbw3i0eFb1oTdzpkFdwhVVaxSTqrQ9BNy5ubFpotfwiB7U5fh0njDAl4G8V8Oj7FBMkKfy5CcsxD2gFUDtsOqf29TSAqr283\\\\\\\/blzR5MXMpw2UelRbIpWyC\\\\\\\/ZlvTgx18Ssu3QM0Jw5rbHD\\\\\\\/dmW\\\\\\\/JjVo+G3QGE3xmXuVuygQOePpY+aD7SgVq6ueNJ2\\\\\\\/9ZiWKeeEhEYpNKdwl1mY\\\\\\\/\\\\\\\/LODxo5kotolx2rrIbSeigxMNJZpMIDduF4pAzihQe6qIZDwut4fNhEO1PAkFuSB2G2SlvgsBKXBAuFDE15lglBl\\\\\\\/u1fKEnHFHK75eU+yfvDAi6orYaGcPoYvvNTi4UqZpmmT7AfFRMAMBh9Fem6YMK\\\\\\\/IwZJUX7VeXu\\\\\\\/RxgBjerJX6gA\\\\\\\/YyeJEmZbfGOZpWBx4XEx8e\\\\\\\/VG8cjxw2kIrFjL1JqXreLi0tsITcZ5xhKnvsrE7saYeqTGuiBIEW612WP4EsEIykWyaU\\\\\\\/XvCo5KJNvC6FYx29TOAMxWbEHCdwmDoCuAf3dSaUZjd7\\\\\\\/49XkoZBC5g1KRV12isbpmnN2SKkzegEGn7PuZ1qUMCL7ToK5LzieCR9gh8VauaVmg4ib6tN\\\\\\\/K6tcNRJGDknK68m0MzQ8U3RZ8459IaH9FWNlEHM5NoMKIbY86i2eItvKq0JWwWGVAXYfcVUOZaRLKWQ1x3B0K\\\\\\\/AfkQAEow6IcAxujr9czoRUYWMqGkmdnq29j2idhX7yXaGI2a+8ESbZ7Q8qD5Tim011+QYBZHUcxk0bLfkTFh1LP8qwFTUZA6XqNAvgLEdH6QIYxGa4G0xnFavvj52tszW4zwbJoZd0UEgWG3KpmyxLz\\\\\\\/rpLhkgYtdKhIY0B\\\\\\\/MGabdsbuVlBnkfwzFCRI3B0j82qSO09DmM8W2uHa8p\\\\\\\/NLrutF+rLMOqh0tymOsV8MnbXxtZs53fNkEI1V8roE8PKmIrYmRbYJP7y2hXpvgBpePToqomR2OQs2hrV11FZaR7VOV0AeQzMt\\\\\\\/J4uOBqM7UqADKNwxjNBVmKOf+B9+u5+cpUl8iuWYp4b1Em7+nXcIhqjHrrOJ\\\\\\\/\\\\\\\/+FKymyB9YYmpBbSDnX6fLhrCXMPArU+K5Zj\\\\\\\/9Qi7+QYksa\\\\\\\/2XUob7NkkIWhXn0ulOS7z5Hj3VHg0P7wWnPAsqKXad7k7TLR6xTxrTSKPDAdqvPLoZPq4uMwVj58vWt5tLfh\\\\\\\/snXQyH8KoPZE7jUWultyroNkWke7uCgs1tNKXuGwGWadp2\\\\\\\/bG30K4Am\\\\\\\/eMIx8+9HM+QANrb1PHVfmiElYrj1PQWirlVappCZQqX\\\\\\\/Gtymwx54UIzBip8jjnlm5kvF3yAWo6qW4LCoZ6Tx5f8RDOWbXVE4SPfTz1jJAX3SxHVSbBDx1r4\\\\\\\/ps+GFES2oUujCMadN6hWxmzUQcSnGCenJR2DXxEFQIOGB5wNviw3HR2OwJjSPeTyxstEgbrDX6WnHYITLgRdRaX1i6KM+7YqQc2LyX0OgY8UmP9IzpA20l5TCikXxxbTPymf57zVheOrJwbAaFkP0ucNH4Nbp3Ie9H6EwqAdTQKXlMohFg8jPaFt+IDmwxjGV3V3PKzD5KM\\\\\\\/l1UYBzveSDsyWNzixQ0EMezl+WE55nVoNhDFhMwOlSc0w2rERc0Ubkq5KpwVrFEOLcphsztxvKVrRyidE7YEChXDZ+s9iLWMi+uiLaoAwDdggATwZfzIIT6zOsB5zIFhbeUJSLQfZFJ2JHnoPtzeupPEEi3Nevljeqk+xPPjFFVy\\\\\\\/z4SDMn\\\\\\\/xsec\\\\\\\/TvYhp6t6+2\\\\\\\/oGiOAkedzammciJ0\\\\\\\/msp7ogoyNBd4J1bsv2y\\\\\\\/mKj6wXvSxls2QkFqzvUhZFT9Mti1s8fBQfNAH6\\\\\\\/IchhcHYj\\\\\\\/QqBr6pqULKaOujNYgO9UEo8zyrZS6tKKztKV6N+moBA6rvtAkY0POT7PnefLUQ3Xgkk5VPlpR6xJTzyxBAOTH64ia2qALVtJngstcuzZknawNg5pdrEx7KwqE7oGSkN8cqI1mk+Y4ekWThQjwwj9BfC6shkHAy+tXY989oDz2ys5MhGPNeBqW2wDnPM4k4VpA7moSbRnBW\\\\\\\/99YldW0daxQ6WW1OsNRBwdon3BXlep79je5E03SFa72dco4jdlmBByYt6VU2TzMuXWaA\\\\\\\/My+og6OKcx5HGBQNdHFExFmEVPpga5VUg7G5GO3GPozwSzzAs1NLNKTfc\\\\\\\/8d2K1mA9W1T5gwL3QLsq5MfX6bkoOPGg\\\\\\\/d7IvbjXGqjJ8N+aHn5X2QRxVojThIYszm8SuLWwgv5j8Jqk7QhkmrdIShYYDD\\\\\\\/UaaZY8IOmW1lXhFQpUCP9DEijC74VnD38Wf5d5wNzB91FqDlwBBzh0Jo0QuQb0UHj6ERnt1r2KHoSlLtDN\\\\\\\/c2iafWwsL\\\\\\\/qd3f7aFWD42CMZ3zPDEoWtGLVchbf4M8XvGXFp8Ar9Lj36LgBJt8hFs=\\\u0022,\\\u0022iv\\\u0022:\\\u0022NkCnFFM2gO0lX0Ey+S4Iuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca5e52987cf8d2b89fd5e175d03c39246ac2afca36499b296704395cb4150c0e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TDTySI+0qIFx4hX7LxW3wvqLxxZlewfwEjC7P\\\\\\\/VIgFhFdZN3MaQgt2vbcbbfSOIUNB5kRH17xUQjSd+oF1\\\\\\\/Xd9iEoRz5YGfBsjt8LVC7wsC8noLCB+uwTTMug5uNhoerCI5yHYSsBiuJh4N+tFvlQLzzQXP5JvJ+syKA6bBOASCoy4S5x0CzdKbpb0Ds89VoAnKyuOS15ayA83Vk8NP1VCVau+q1aoXwDegddzP1MpXp\\\\\\\/cRZqQ8KBj4p0EYLQG+xdhkA5zWKvuKtgUxUDHJ93iILdU3ApvPsLIeJ5ekdaPsfWiA4y35d8Nnbm1pC3edvY2BVpVnMJegYF+UCuwDPU9oINeJ7IiyV7V4ot6eYLhKVWhtdr8gUPwf7jwvCTxB6ooc2x5CSrz0IRk0GT4JOcpvLFuPxzYaeOgYaPRky2qJE+Mvg+QB4Wdh3AQfSDCh2MWU8vv6wBiEkcDtKNv81hKG9LmC9HquC3a5\\\\\\\/k1ACEMksP75npXIPYjeKh9fKNsao\\\\\\\/HM5Ec\\\\\\\/XPOTf+W6ydGldFrKyjnBI0317xTOTMeC1GIYx4dnSeSu2RPRocXSpVl73YSmXw\\\\\\\/jSF6AZ1kM+Baq4kpDWFtBG\\\\\\\/hvvYWbny44Jo9KCFw98juI\\\\\\\/aOYDsYbAHSDpztXz9YrshrZ1MLZ1oe51zjKGDLmCfbFwe8R9+k+wTi2FYc5WsJ1v8zU\\\\\\\/I1CGKLgGxlRc9kZedjxMNV00E0e6t4kW1vkIxxNtJBPUdhgzsR\\\\\\\/YjOMmOrEvzIq5R5o84ir90AcMWxaR5RpjiDeHcWRpRVrgZR3PWgQhJbogLzKxqaK7V5HzZGrVy5hqW\\\\\\\/TsNXX9jPGHcir28gt7WLfJJdSrGeHKddC5YC3jtWH9ahnqrnRJj1GRG3ZRVKqDZB98soxbolm32I1abB8GSzskRwoaQkX\\\\\\\/2uRbePNfK0U2qyh2cN7N9JF+sEFHOGeShDzd3ZfMAdRVcRBUrzvUDY23qe+hHOdoZ61ubLPu07yFJuCUdu\\\\\\\/PEai2QR5oJxiPsxZDK65kX89VxD8wEAyEVjOwsA\\\\\\\/kjusdbA2oVDI+aTD28c4pEj2JILB+Z2ExWcJtkYz040XUewywn63qrn++74Q89jtCMvoliTqmIb7QKvzki7VS4oAmpmeD6iOPN1OdNL3jwDL0pIGmmztUZNwTl9CQrqmkIKBKRrbxcBXgYfkI5qkmR9cTLXcDHzbIS1PEBCLVuQFQtplWfUYcflfMmQfF2wb7GW+HLjCobaV23DVJwm1zrn9agfGjZSUONSG+jKeaA9bNhFDjzwvgFsl8DYd\\\\\\\/2TegRavGgMpgtcX7WgrjO3X2KW7IjkWfS6lA5UxcFkKpU5HXOAgoILfgZitlhUyBRvwGPnTho9MqzhQRYorzpfVeoaIFvysCovMtmMki1do\\\\\\\/eHU5LE4iboVV8QraNsV\\\\\\\/ppsLT429WRGPsVqdZeP7HyYAT43S0q\\\\\\\/i0x850BOsRa7Ax7UexFGqyo81e0Y9oZrKMYRpUBpdpALAPaAZYUkaPQYkXYTKuSBQH6ZKRULa5l4D6rgWFxdBdaRzjBGQBuaYi4AHrr8rkbVbR1cBtcdRYk3VEv61yOXGEDP6VVXvV9oBVxkhYhtQaDTE81BprlAsL5LOprC9PKArnMMl\\\\\\\/jv8MOT4mH\\\\\\\/iAP2Gm6iJdiDRsUkdMORBg1giMvuJrL770WQia5OkBbLpjlUbhGltxU5cdRDkUnEHU2kAKvLki8LDdwHXWlUA\\\\\\\/T65V621Vlq+Mzqw6VHbqNTO2WIJCRzTCSPKGyoWnolHVMtRBi6tIFPSKKJXF3k2s2ImVmH5UKBUiPE7GK8Tt5RqU\\\\\\\/uk\\\\\\\/Q6qS1a7LbcrCUyaGlodzP7tR0ZH8NfKo7KuHWbMfhn1vB\\\\\\\/IFxlEqKq5lbBicgeDi7lYjQcrJJmAlD1RwndeNpDt\\\\\\\/SQdX0WfgIu8k\\\\\\\/wtcDbamtW47JuiOMtyNAX7nQ3aDFKcpfoWAVlEvWm3XMlfKGEd49iBwObDiNaqb1aFWq6ZnwXeJgmH1jZxFD\\\\\\\/x4yzlnOHNgPn6QxNnejhyPjyYbC3t6H2YF9NfoR\\\\\\\/4M\\\\\\\/Ap7YnueIs1gwiCnf1JOtIwEQ\\\\\\\/trXbaX6xJk+5K3b9M+NB2BSdAqjGGDzrIOAlVpvZ7x7JnOfCq5aMorf2Vxw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022gaL8iGyBQCTlzOTxUC6PAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002290765c794302306e5f2134adaa0856ea70e2c9d6f02c68dd6d415a5aec21620c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460023 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1815" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FJlrQmybdw7kWpLtALGCXD56bMeRyS5_1FhMtCiwq0w" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1815" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-g3NabwNMrUf5iGeUCPq62R6sdzClIGa-jjaNChvnPss" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/9/1805" title="Comparative Transcriptome Analysis of the Cosmopolitan Marine Fungus Corollospora maritima Under Two Physiological Conditions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/9/1827" title="Transcriptomic Analysis of Musca domestica to Reveal Key Genes of the Prophenoloxidase-Activating System" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460023" id="node460023" data-pisa="ggg;5/9/1815" data-pisa-master="ggg;g3.115.019323" data-apath="/ggg/5/9/1815.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna M. Kopps</span>, <span class="highwire-citation-author" data-delta="1">Jungkoo Kang</span>, <span class="highwire-citation-author" data-delta="2">William B. Sherwin</span> and <span class="highwire-citation-author" data-delta="3">Per J. Palsbøll</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1815-1826;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019323 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460023" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna M. Kopps </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Evolution & Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BM.%2BKopps%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kopps%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BM.%2BKopps%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.kopps@gmx.com" class="" data-icon-position="" data-hide-link-title="0">anna.kopps@gmx.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jungkoo Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>IceLab, Umeå University, 901 87, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jungkoo%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJungkoo%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William B. Sherwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Evolution & Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Murdoch University Cetacean Research Unit, Centre for Fish, Fisheries and Aquatic Ecosystems Research, School of Veterinary and Life Sciences, Murdoch University, Murdoch, WA 6150, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BB.%2BSherwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherwin%20WB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BB.%2BSherwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Per J. Palsbøll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Per%2BJ.%2BPalsb%C3%B8ll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palsb%C3%B8ll%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APer%2BJ.%2BPalsb%25C3%25B8ll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/9/1815" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460023" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1815.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460023" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1815.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460023" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1815.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460023" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/9/1815.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460023" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Kinship analyses are important pillars of ecological and conservation genetic studies with potentially far-reaching implications. There is a need for power analyses that address a range of possible relationships. Nevertheless, such analyses are rarely applied, and studies that use genetic-data-based-kinship inference often ignore the influence of intrinsic population characteristics. We investigated 11 questions regarding the correct classification rate of dyads to relatedness categories (relatedness category assignments; RCA) using an individual-based model with realistic life history parameters. We investigated the effects of the number of genetic markers; marker type (microsatellite, single nucleotide polymorphism SNP, or both); minor allele frequency; typing error; mating system; and the number of overlapping generations under different demographic conditions. We found that (i) an increasing number of genetic markers increased the correct classification rate of the RCA so that up to &gt;80% first cousins can be correctly assigned; (ii) the minimum number of genetic markers required for assignments with 80 and 95% correct classifications differed between relatedness categories, mating systems, and the number of overlapping generations; (iii) the correct classification rate was improved by adding additional relatedness categories and age and mitochondrial DNA data; and (iv) a combination of microsatellite and single-nucleotide polymorphism data increased the correct classification rate if &lt;800 SNP loci were available. This study shows how intrinsic population characteristics, such as mating system and the number of overlapping generations, life history traits, and genetic marker characteristics, can influence the correct classification rate of an RCA study. Therefore, species-specific power analyses are essential for empirical studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/identity-descent-ibd" class="hw-term hw-article-keyword hw-article-keyword-identity-by-descent-ibd" rel="nofollow">identity by descent (IBD)</a></li><li class="kwd"><a href="/keyword/relatedness" class="hw-term hw-article-keyword hw-article-keyword-relatedness" rel="nofollow">relatedness</a></li><li class="kwd"><a href="/keyword/pedigree-reconstruction" class="hw-term hw-article-keyword hw-article-keyword-pedigree-reconstruction" rel="nofollow">pedigree reconstruction</a></li><li class="kwd"><a href="/keyword/relatedness-category-assignment" class="hw-term hw-article-keyword hw-article-keyword-relatedness-category-assignment" rel="nofollow">relatedness category assignment</a></li><li class="kwd"><a href="/keyword/intrinsic-population-characteristics" class="hw-term hw-article-keyword hw-article-keyword-intrinsic-population-characteristics" rel="nofollow">intrinsic population characteristics</a></li></ul><p id="p-3">Knowledge about kinship (pedigree and relatedness) is central to our understanding of ecological and evolutionary processes and an integral part of the management and conservation of endangered populations, such as trend and distribution of abundance as well as dispersal rates and individual fitness (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Avise 1992</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Coltman <em>et al.</em> 2003</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Palsbøll <em>et al.</em> 2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wang 2014</a>). In the past, inferring relatedness from molecular genetic data was regarded as an approximation to the true pedigree relatedness. However, as molecular techniques improve, the pedigree is being regarded as only an approximation to the true relatedness or identity by descent (IBD, rather than IBS, or identity by state) that can be found in each part of the genome (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benjamin <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Speed and Balding 2015</a>). Of course, cases remain in which insights into the pedigree is important, such as niche inheritance, cultural inheritance, as well as epigenetic inheritance (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bonduriansky 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danchin 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kopps <em>et al.</em> 2014</a>). Therefore, it is possible to justify both reconstructed pedigrees and the approximation to true relatedness, and it is important to assess how well molecular and pedigree assessments of relatedness correspond to one another.</p><p id="p-4">Pedigrees can be estimated on the basis of genetic similarity among individuals of a population (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blouin 2003</a>). The equations used to assess genetic relatedness compare the observed genetic similarity between dyads to the population average (<em>e.g.</em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Queller and Goodnight 1989</a>) or provide likelihoods that a dyad belongs to several potential relationships based on Mendelian allele segregation (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thompson 1975</a>). The “correct classification rate” of these likelihood equations is defined as the frequency of dyads that are assigned to a category and that are true members of that category (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Thomas and Hill 2000</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wang 2004</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang and Santure 2009</a>). Most software used to assess genetic relationships based on genetic markers can deal with genotyping error (<em>e.g.</em>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marshall <em>et al.</em> 1998</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jones and Wang 2010</a>). However, except for parentage, there has been limited effort devoted to assessing the effects of intrinsic population characteristics and other aspects of the quality and quantity of data necessary to attain high correct classification rates in the estimation of relatedness among individuals (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson and Garza 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aykanat <em>et al.</em> 2014</a>).</p><p id="p-5">Intrinsic population characteristics such as mating system and number of overlapping generations, which is determined by the relationship between generation time and life span, are expected to affect a population’s “kinship composition,” that is, the distribution of the different relatedness categories in a population (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Wang 2004</a>). For example, full sibs are unexpected in a population with a promiscuous mating system. Also, dyads related as parent and offspring cannot be observed at the same time in species with nonoverlapping generations. The equations used for relatedness analyses are solely based on Mendelian probabilities and therefore do not take such life history parameters into account. Therefore the correct classification rate of these equations may depend on intrinsic population characteristics due to different kinship compositions. In this paper, we assessed the correct classification rate in one commonly used estimation of pairwise categories of relatedness in three different mating systems (monogamy, polygyny, and promiscuity) and one or three overlapping generations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) as a function of the number (and kind) of loci.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466711/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466711/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Questions, reasoning and results</span><div class="sb-div caption-clear"></div></div></div><p id="p-7">There are two main types of relatedness estimators (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Blouin 2003</a>). The first one, method of moments relatedness estimation (<em>e.g.</em>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Queller and Goodnight 1989</a>), assigns a value to each dyad based on IBD allele sharing. Typically, these values range from −1 (no similarity) to +1 (perfect match). Relatedness estimates and pedigree relatedness (mean expectations, <em>i.e.</em>, 0.5 for parent-offspring, 0.25 for half sibs, etc.) are highly correlated; however, the variance of relatedness estimates is usually high and in many cases the estimated relatedness does not agree with the actual pedigree (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Santure <em>et al.</em> 2010</a>). This means that relatedness estimates are more suitable for assessing the relatedness for a group of individuals rather than dyadic relatedness. Even though the variance of the relatedness of a group of individuals declines with increasing number of loci (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Queller and Goodnight 1989</a>) the correlation between pedigree relatedness and pairwise relatedness estimates did not exceed 0.86 even when relatedness estimates were based on 771 single-nucleotide polymorphisms (SNPs; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Santure <em>et al.</em> 2010</a>).</p><p id="p-8">The second type of relatedness estimator is the assignment of likelihood ratios to dyads belonging to certain relatedness categories {<em>i.e.</em>, parent-offspring [PO], full sibs [FS], relationships sharing on average a quarter of their genome IBD [R = 0.25: half sibs, grandparent−grandchild, avuncular (any of the four combinations: aunt/uncle-niece/nephew)], relationships sharing on average one eighth of their genome IBD [R = 0.125: first cousins], and unrelated}. The assignment of dyads to relatedness categories is based on the likelihood that a pair of individuals shares zero (Δ<sub>0</sub>), one (Δ<sub>1</sub>) or two (Δ<sub>2</sub>) alleles that are IBD (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Thompson 1975</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Thompson 1991</a>). This likelihood is calculated by multiplying probabilities across unlinked loci and it is different for each relatedness category. A dyad is assigned to the relatedness category for which it has the highest likelihood. We will focus on the relatedness category assignment (RCA) method in this paper because knowledge about relatedness categories provides essential information for niche inheritance, cultural inheritance, and epigenetic inheritance studies.</p><p id="p-9">To date, studies assessing the power to assign dyads to relatedness categories are based on a null hypothesis category <em>vs.</em> an alternative category (<em>e.g.</em>, full sib <em>vs.</em> half sib) (<em>e.g.</em>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brookfield and Parkin 1993</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Wang 2006</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wang and Santure 2009</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Santure <em>et al.</em> 2010</a>). This method is valid if previous knowledge is available, <em>e.g.</em>, are chicks in a nest full or half sibs? However, in a natural population and in the absence of any previous knowledge this kind of power analysis may be misleading in terms of underestimating the number of required loci for reliable RCAs. In addition to the relatedness category restrictions, the influence of intrinsic population characteristics on the correct classification rate of relationship assignments is often limited. For example, <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Wang and Santure (2009)</a> focused only on parentage and sibship inference in a polygamous population (here referred to as promiscuous) for studying the power of relationship assignments. We assessed the correct classification rate of RCA, including relatedness categories with relatively high IBD, and with a realistic degree of background relatedness in the study population.</p><p id="p-10">The advances in new massive parallel sequencing methods facilitate the genotyping of an increased number of genetic loci per individual (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gardner <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Peterson <em>et al.</em> 2012</a>), which may increase the correct classification rate of RCAs. In studies estimating pairwise categories of relatedness, the class of loci most commonly used have been short tandem repeats (STRs), but the use of SNPs has been increasing (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Weir <em>et al.</em> 2006</a>). Each class of loci has its specific advantages and disadvantages. STRs are more informative per locus because they are usually more polymorphic than the often biallelic SNPs (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Morin <em>et al.</em> 2004</a>). On the other hand, SNPs are more abundant and more suitable for automated data analyses than STRs (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Vignal <em>et al.</em> 2002</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Morin <em>et al.</em> 2004</a>).</p><p id="p-11">In this study, we assessed the correct classification rate for assignment of dyads to relatedness categories, and how the correct classification rate is influenced by number and type of genetic markers, minor allele frequency (MAF), genotyping error, mating system, and including or excluding overlap of generations by looking at 11 questions outlined in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-12">Eleven questions (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) were investigated in this study using an individual-based model in which pedigree relatedness was tracked and compared with the most likely relatedness category, which was assigned based on genetic markers. Individual-based models provide a means to investigate characteristics influenced by stochastic processes such as Mendelian segregation of alleles. We developed a model capable of simulating different natural systems, modified from a similar model originally designed by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kopps and Sherwin (2012)</a>, which was aimed at the Shark Bay bottlenose dolphin population (<em>Tursiops</em> sp.). For this study, the modifications to the original model include the implementation of additional mating systems, variable number of overlapping generations, an increase in the maximum number of (unlinked) genetic markers, and the tracking of pedigree relatedness. For each individual in a simulation (except at the start of the simulation) the parents, grandparents and great-grandparents were known. Pedigree-based unrelated individuals were individuals that did not share any common ancestors in the three previous generations. All simulations were run in Matlab R2012a (MathWorks, Natick, MA).</p><p id="p-13">The RCAs based on genetic markers were performed using the likelihood equations outlined in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Epstein <em>et al.</em> (2000)</a>. Each dyad was assigned to the relatedness category for which it had the highest likelihood. Unless otherwise stated, we assessed six relatedness categories: monozygotic twins, PO; FS; R = 0.25; R = 0.125; and unrelated.</p><p id="p-14">All simulations were initiated with a population size of 600 individuals as in <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kopps and Sherwin (2012)</a> and run for 100 time steps before assessing the correct classification rate of the relatedness estimation. One hundred time steps was considered a good compromise between being able to track three generations of ancestors and not losing much genetic variation due to genetic drift. Individuals in the simulation could lived for a maximum of 12 time steps (age class 12) and became sexually mature at age class 4. Unless otherwise stated, results shown were averaged over 10 independent simulations. Ten simulations proved sufficient because repeated sets of 10 simulations gave identical answers for the number of loci necessary to achieve a correct classification rate of 80 or 95% for assigning dyads to relatedness categories.</p><div id="sec-2" class="subsection"><h3>Mating system and generational overlap</h3><p id="p-15">Mating systems influence the genetic make-up of a population including the kinship composition, and thus may influence the performance of RCAs. We therefore simulated three different, general mating systems: monogamy, polygyny, and promiscuity. Based on the life history values used in all three scenarios (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/019323SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppldoi:10.1534/g3.115.019323/-/DC1/TableS1.pdf">Table S1</a>), the maximum number of overlapping generations during each time step was three, unless otherwise stated. In the case of the monogamy scenario, males and females were paired for life. In the event that a paired individual died, the surviving individual in the pair would be paired with an available, sexually mature individual of the opposite sex. In the polygynous scenario (<a href="http://www.g3journal.org/lookup/suppldoi:10.1534/g3.115.019323/-/DC1/TableS1.pdf">Table S1</a>), 60 territories were available and each was occupied by a single mature male. When a territory-holding male died he was replaced by a mature, nonterritory holding male, if possible from age classes 7 to 9. Females initially were assigned randomly to a male’s territory, where they remained for life. In the promiscuous scenario, males and females were mated randomly.</p><p id="p-16">With a small number of exceptions, monogamous, polygynous and promiscuous scenarios were modeled with identical life history parameter values (<a href="http://www.g3journal.org/lookup/suppldoi:10.1534/g3.115.019323/-/DC1/TableS1.pdf">Table S1</a>). The exceptions were required because of stochastic model constraints and included the alteration of the average number of offspring per female between simulations with different mating systems. We are aware that there is some artificiality in using the same life history data in scenarios with different mating systems. However, to investigate the effect of mating system on the correct classification rate of RCAs, it was essential to use the same life history data for each mating system scenario, in order to avoid confounding factors when drawing conclusions about the effect of the scenarios.</p><p id="p-17">Overlap of generations during the sampled time-period affects the kinship composition; for example, in populations with nonoverlapping generations there can be no detection of <em>trans</em>-generational kin such as parent-offspring, grandparent–grand offspring, avuncular. To test whether the absence of certain pedigree relationships in the population influences the correct classification rate of an RCA we also adapted each mating scenario so that generations would not overlap during a sampled time-period. All simulation parameters were the same as in simulations with overlapping generations, except for age at maturity, life expectancy and average number of offspring per female (<a href="http://www.g3journal.org/lookup/suppldoi:10.1534/g3.115.019323/-/DC1/TableS1.pdf">Table S1</a>). The population size was kept constant by letting 550-650 offspring survive each generation to produce the next generation.</p></div><div id="sec-3" class="subsection"><h3>Number of loci</h3><p id="p-18">We estimated the correct classification rates of RCAs from data sets with 10, 20, 40, and 80 STRs, or 50, 100, 200, 400, 800, 1600, or 3200 SNPs. Additionally we performed single simulations with 50,000 SNPs. Because many laboratories are in transition from STR to SNP genotyping, we also considered whether the use of a combination of the two marker types may increase the correct classification rate of an RCA compared with the use of a single marker type. For that purpose, we combined the relatedness category probabilities of 20 STR loci with 50, 100, 200, 400, 800, 1600, or 3200 SNPs. To allow for direct comparisons, we used the same conditions for the combined marker types as for the SNPs only analyses. For the STRs-only analyses we used the same simulations as for the SNPs MAF 0.5 scenarios.</p><p id="p-19">Note that, at least for population structure studies based on STRs, the number of alleles has been found to be more informative than the number of loci (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kalinowski 2002</a>). All SNP loci were biallelic and the 80 STR loci implemented in the simulation consisted of eight identical (but independent) sets of 10 STRs (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kopps and Sherwin 2012</a>) so that the number of alleles doubled with every duplication of the number of markers used. At the start of the simulations, these sets of 10 STRs had an average of 5.6 alleles/locus. This number is similar to that found in many empirical studies but in our simulations the loci had significantly higher expected heterozygosity (<em>e.g.</em>, <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Kopps and Sherwin 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kopps <em>et al.</em> 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2014</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Crean <em>et al.</em> 2014</a>).</p></div><div id="sec-4" class="subsection"><h3>Proportion of population sampled, MAF, and typing error</h3><p id="p-20">In most field studies, it is not feasible to sample the entire population. Accordingly, we ran scenarios in which 50, 25, 12.5, and 6.25% of the population were sampled to assess how the sampled fraction impacted the correct classification rate of the RCA relative to sampling the entire population (<em>i.e.</em>, perfect allele frequency estimates). This assessment was conducted with data from 400 and 3200 SNP loci, as well as 400 and 3200 SNP loci combined with 20 STR loci, and 80 STR loci.</p><p id="p-21">We also investigated the influence of genotyping errors and MAF upon the correct classification rate of RCAs. MAF was defined as the mean allele frequency of the rarer SNP allele at the start of the simulations, ranging from 0 to 0.5. We assessed the effect on correct classification rate of varying the mean MAF across loci at three different mean MAFs: 0.05; 0.25; 0.5. For some simulations, we implemented a typing error to assess any reduction in correct classification rate of RCAs. Each allele at each locus had the same probability of being mistyped (1%), leading to a 2% locus specific typing error rate, which is the error rate used in <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Wang (2006)</a>.</p></div><div id="sec-5" class="subsection"><h3>Exclusion and inclusion of relatedness categories for assessment</h3><p id="p-22">The correct classification rate of an RCA might be influenced by what categories of relatedness are chosen for assessment. We assessed whether the correct classification rate changed when the RCA did not attempt to identify certain categories of relatedness. For example, the proportion of false positives in the R = 0.125 category was high even when the estimation of relatedness was based upon data from 50,000 SNP loci. Therefore we tested whether not assessing the R = 0.125 category would increase the correct classification rate of the other relatedness categories.</p><p id="p-23">On the other hand, it is also possible that assessment of an additional relatedness category would allow dyads that were previously wrongly classified to some other category to be more appropriately classified to the new category. This would improve the correct classification rate to assign dyads to other categories. To test this, some simulations included the assessment of the category R = 0.0625 (half first cousins [sharing one grandparent], first cousins once removed, double second cousins). The probability of identity states for sharing zero, one and two alleles/locus used in the algorithm were: Δ<sub>0</sub> = 0.875, Δ<sub>1</sub> = 0.125, and Δ<sub>2</sub> = 0.</p></div><div id="sec-6" class="subsection"><h3>Incorporating demographic and mitochondrial data (mtDNA)</h3><p id="p-24">Incorporating demographic and mtDNA data might improve the correct classification rate of an RCA by reducing the number of false positives (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Riester <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cope <em>et al.</em> 2014</a>). We ran three scenarios in which sex, age (in age classes) and/or mitochondrial DNA haplotype mtDNA (five equifrequent haplotypes at the start of the simulation) were known. This led to the exclusion of particular relationships for certain dyads even if they had the highest likelihood, in which case that dyad was then assigned to the relatedness category with the second highest likelihood, according to the following criteria:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-25">mtDNA haplotype known. Individuals not sharing their mtDNA haplotypes could not be assigned to the category FS. Female-female dyads not sharing their mtDNA haplotypes could not be assigned to the category PO.</p></li><li id="list-item-2"><p id="p-26">Age known. Individuals whose age difference was less than the age at sexual maturity could not be assigned to PO.</p></li><li id="list-item-3"><p id="p-27">Age and mtDNA haplotype known. The same dyads were excluded as in (1) and (2). Additionally, dyads in which the older individual was female and which did not share their mtDNA haplotype could not be assigned to PO.</p></li></ol><p id="p-28">Although dyads could be excluded from being grandparent−grand offspring using age data, there were no age or mtDNA haplotype restrictions for R = 0.25 because this category included other relationships without age restrictions.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-29">The simulation code is available on DRYAD (<a href="http://dx.doi.org/10.5061/dryad.sr61r">http:/dx.doi.org/10.5061/dryad.sr61r</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-30">For a clear arrangement of our results, they were summarized together with the questions and reasoning in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. The correct classification rate of an RCA was affected by the number of loci, MAF, typing error rate, availability of additional data as well as intrinsic population characteristics (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS2.pdf">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS9.pdf">Figure S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS11.pdf">Figure S11</a>, <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, and <a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a>). Note that the formula for the likelihood for monozygotic twins was used in the simulations but we do not show the correct classification rate of assigning dyads to this category in the results section. However, we observed that a few dyads were assigned to this category when the MAF was low and 50 or 100 SNPs were used.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1815/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Promiscuity: correct classification rate of relatedness category assignment (RCA) in a promiscuous population (average over 10 simulations). Three different minor allele frequencies (MAF) for single-nucleotide polymorphisms (SNPs), seven different numbers of SNP loci (individual bars from left to right: 50, 100, 200, 400, 800, 1600, 3200), four different numbers of STR loci (from left to right: 10, 20, 40, 80), and a combination of SNP with 20 STR loci were simulated. On the left vertical axes, the proportion of the correct pedigree relatedness color in each category (PO: parent-offspring; FS: full sibs; unrel: unrelated) indicates the correct classification rate of the category-assignment based on the genetic loci. Other colors indicate source of erroneously assigned categories. The right vertical axes, and the lines in the subplots, indicate the number (No) of dyads that were assigned to each category (the true number of dyads can be inferred where almost 100% correct classification rates were achieved). The orders of magnitude at the top of the No dyads/category scale of the first row apply to all No dyads/category scales below it. Figure S1 and Figure S2 show the same plots for other mating systems. The variability between the 10 independent simulations is presented in Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1724573619" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Promiscuity: correct classification rate of relatedness category assignment (RCA) in a promiscuous population (average over 10 simulations). Three different minor allele frequencies (MAF) for single-nucleotide polymorphisms (SNPs), seven different numbers of SNP loci (individual bars from left to right: 50, 100, 200, 400, 800, 1600, 3200), four different numbers of STR loci (from left to right: 10, 20, 40, 80), and a combination of SNP with 20 STR loci were simulated. On the left vertical axes, the proportion of the correct pedigree relatedness color in each category (PO: parent-offspring; FS: full sibs; unrel: unrelated) indicates the correct classification rate of the category-assignment based on the genetic loci. Other colors indicate source of erroneously assigned categories. The right vertical axes, and the lines in the subplots, indicate the number (No) of dyads that were assigned to each category (the true number of dyads can be inferred where almost 100% correct classification rates were achieved). The orders of magnitude at the top of the No dyads/category scale of the first row apply to all No dyads/category scales below it. Figure S1 and Figure S2 show the same plots for other mating systems. The variability between the 10 independent simulations is presented in Table S2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1815/F1.medium.gif" width="291" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/9/1815/F1.medium.gif" width="291" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1815/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1815/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466659" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Promiscuity: correct classification rate of relatedness category assignment (RCA) in a promiscuous population (average over 10 simulations). Three different minor allele frequencies (MAF) for single-nucleotide polymorphisms (SNPs), seven different numbers of SNP loci (individual bars from left to right: 50, 100, 200, 400, 800, 1600, 3200), four different numbers of STR loci (from left to right: 10, 20, 40, 80), and a combination of SNP with 20 STR loci were simulated. On the left vertical axes, the proportion of the correct pedigree relatedness color in each category (PO: parent-offspring; FS: full sibs; unrel: unrelated) indicates the correct classification rate of the category-assignment based on the genetic loci. Other colors indicate source of erroneously assigned categories. The right vertical axes, and the lines in the subplots, indicate the number (No) of dyads that were assigned to each category (the true number of dyads can be inferred where almost 100% correct classification rates were achieved). The orders of magnitude at the top of the No dyads/category scale of the first row apply to all No dyads/category scales below it. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS2.pdf">Figure S2</a> show the same plots for other mating systems. The variability between the 10 independent simulations is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS2.pdf">Table S2</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1815/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of additional data on correct classification rate of relatedness category assignment in a monogamous population using 20 STRs. In addition to age and/or mtDNA haplotype, the sex of the individuals was known. Plotted are mean and range of the correct classification rate based on 10 independent simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1724573619" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of additional data on correct classification rate of relatedness category assignment in a monogamous population using 20 STRs. In addition to age and/or mtDNA haplotype, the sex of the individuals was known. Plotted are mean and range of the correct classification rate based on 10 independent simulations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1815/F2.medium.gif" width="440" height="205"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/9/1815/F2.medium.gif" width="440" height="205"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1815/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1815/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466654" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Effect of additional data on correct classification rate of relatedness category assignment in a monogamous population using 20 STRs. In addition to age and/or mtDNA haplotype, the sex of the individuals was known. Plotted are mean and range of the correct classification rate based on 10 independent simulations.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466661/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466661/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Minimum number of SNP and/or STR loci required per category for a relatedness category assignment with &gt;95% (&gt;80%) correct classification rates</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Population size averaged 598.67 individuals, with range of 482−703 individuals; note that these numbers of individuals are equal to 115,921 to 246,753 dyads. Within this large number of dyads, there were a few dyads that were assigned to more than one relatedness category based on pedigree (monogamy: mean = 0.63 dyads, range = 0−6; polygyny: 4.27, 0−13; promiscuity: 4.53, 0−13). This was possible because we assigned the dyads to the same categories based on pedigree as for the RCA. The most common categories that were shared were R = 0.25 and R = 0.125, <em>e.g.</em>, dyads shared one parent and two grandparents that were not the shared parent’s parent. Because of genetic drift, during the 100 simulated time steps of MAF 0.05 scenarios, a number of the 3200 SNP loci lost variation (became fixed for one allele): 25.8 (0.8%) in monogamy, 60.6 (1.9%) in polygyny, and 37.0 (1.2%) in promiscuity. No loci became fixed in scenarios with MAF = 0.25 or 0.5.</p></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-38">Several relatedness categories can be assigned with 80 or 95% correct classification rates if certain pitfalls are avoided by considering life history parameters, number of loci used, mean MAF, and typing error rate. As expected, there is always a positive correlation between correct classification rate of the RCA and the number of loci loci. Note that when using a realistic number of 800 SNPs or 20 STRs, more categories could be assigned at a given correct classification rate level by SNPs than by STRs (<a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Table 2</a>). This favors SNPs (or a combination of SNPs and STRs) over STRs. The assignment of more distantly related dyads than R = 0.25 (rarely R = 0.125) appears to be impossible using RCAs, even when 10,000s of SNP loci are available (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS3.pdf">Figure S3</a>), possibly because of stochasticity in allele segregation or deeper roots of coancestry.</p><p id="p-39">Even though few (around 100 SNPs or 10 STRs) loci are sufficient to correctly assign most real (pedigree-based) parent-offspring and full sib pairs to the correct relatedness categories, the high rate of false positives (SNPs MAF 0.05 and 0.25: PO: 0.81−0.85 and 0.11−0.19, FS 0.75−0.99 and 0.23−0.98; STRs: PO 0.85−0.89, FS 0.92−0.99 across all three mating systems) at these relatedness categories makes the use of this small number of loci problematic, as previously reported for parentage inference (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Anderson and Garza 2006</a>). A combined data set of SNP and STR loci may increase the correct classification rate of RCAs, especially for parent-offspring pairs when SNP data with low informativeness are available (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS6.pdf">Table S6</a>, and <a id="xref-table-wrap-2-11" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-40">For our estimation we did not consider any prior knowledge of possible relationships. This approach seems realistic for many ecological studies in natural populations. Because of the absence of assumptions about the potential relationships, the numbers of loci suggested here (≥ 40 STRs, 100−3200 SNPs) are much higher than the number of loci considered sufficient for the assignment of relatedness categories by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Brookfield and Parkin (1993</a>; 2−9+ STRs), <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Wang (2006</a>; 2−13 STRs, 11-92 SNPs 2006), or forensic studies (10−15 STRs, 51 SNPs, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amorim and Pereira 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sanchez <em>et al.</em> 2006</a>). This is because these studies’ estimates were based on only two hypothetical relationships: null relatedness <em>vs.</em> a single alternative relatedness category. Statistical significance for the support of the assignment of individual dyads to any relatedness category [by calculating LOD score (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kalinowski <em>et al.</em> 2007</a>) or FDR (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Skaug <em>et al.</em> 2010</a>)] was not inferred in our study, and, therefore, we might have underestimated the correct classification rate of the RCA. However, by assigning all dyads to the relatedness category with the greatest likelihood, all dyads were assigned to a relationship category.</p><p id="p-41">Some false-positive results may be identified through additional knowledge (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) such as mtDNA haplotype or age, which can be estimated from genetic samples (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Polanowski <em>et al.</em> 2014</a>). Besides increasing the correct classification rate of the category PO, age data also provide directionality to PO pairs, <em>i.e.</em>, identify who is the parent and who the offspring.</p><p id="p-42">Some false-positive results may be identified also by genetic data only: a polyadic approach, <em>i.e.</em>, comparing the compatibility of dyadic relationships by simultaneously assigning parentage and sibship, might filter out some incorrectly assigned dyads by identifying incompatible polyads. Borrowing the example from <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Wang and Santure (2009)</a>, if individuals A and B, and A and C, were assigned to the category full sibs the assignment of B and C to R = 0.25 is incompatible. A polyadic approach for sibship and parentage reconstruction was implemented in COLONY2 (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Wang and Santure 2009</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Jones and Wang 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wang 2013</a>). Another approach which uses a third individual as reference when assigning a dyad to a relatedness category seems to perform better than a purely dyadic approach (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wang 2007</a>) and could change the number of required loci for an RCA presented here.</p><p id="p-43">The exclusion from consideration of a single category (<em>e.g.</em>, R = 0.125, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS3.pdf">Table S3</a>) in simulations decreased the correct classification rate of an adjacent category (<em>e.g.</em>, R = 0.25) and is therefore not recommended. However, the addition of an extra category (<em>i.e.</em>, R = 0.0625) may be beneficial (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS7.pdf">Figure S7</a>). To our knowledge, the category R = 0.0625 has not been used in previous studies (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Thompson 1975</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Epstein <em>et al.</em> 2000</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Blouin 2003</a>; <a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Wang 2006</a>; <a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Wang and Santure 2009</a>) even though the expected proportion of shared genetic information is higher than for second cousins (R = 0.03125, described in <a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Wang 2006</a>).</p><p id="p-44">The categories R = 0.25 and R = 0.125 contain several relationships (half sibs/grand-parent-grand-child/avuncular and full cousins/half-avuncular, quadruple second cousins, respectively). Methods to separate pedigree relationships sharing the same relationship coefficient have been proposed. They are based on the relationship between the number of chromosomal segments (approximately 2 Mb, as opposed to individual loci) shared IBD by two individuals and the number of meioses on the path relating these individuals (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huff <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Browning and Browning 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hill and White 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2014</a>). With these methods, dyads may be assigned with reasonable confidence also to relatedness categories more distant than R = 0.25 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Henn <em>et al.</em> 2012</a>). These methods sound promising and feasible for non-model species if long scaffolds of their genomes are available for each individual. However, even with next-generation genotyping, most studies on nonmodel organisms do not have adequate mapping information to assign the IBD blocks upon which <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Browning and Browning (2012)</a> rely.</p><p id="p-45">The simulation data sets are unrealistically ideal in terms of completeness of sampling (or extent of random sampling), missing data (none), and typing error (none for most simulations). Genotyping error rates of current next-generation sequencing platforms are still substantial and the power of the conversion of the raw sequencing reads into genotypes depends on sequencing depth and SNP calling algorithms (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nielsen <em>et al.</em> 2011</a>). But even if three SNP calling algorithms agree on an individual genotype, the assigned genotype may be incorrect in 3.5% of individuals, as a recent study has shown (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Greminger <em>et al.</em> 2014</a>). Typing error should be taken into account because it can have profound effects on the power of kinship analyses. For most categories, with a 2% typing error rate, the number of loci required for any given correct classification rate of RCA is two to four times greater compared with error-free data (question 9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS5.pdf">Table S5</a>, <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, and <a id="xref-table-wrap-2-12" class="xref-table" href="#T2">Table 2</a>). PO and FS dyads are logically more susceptible to error than dyads of more distantly related categories; our data corroborate this (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/TableS5.pdf">Table S5</a> and <a id="xref-table-wrap-2-13" class="xref-table" href="#T2">Table 2</a>). Taking the ideal data sets generated by simulations into account, the number of loci we recommend to be used for RCAs are the minimum for best case data sets, meaning that researchers should estimate the error rate of their data and its impact on analyses. Also, it is important to bear in mind (and this is one aim of this study) that the necessary number of loci for an RCA to be able to have an adequate correct classification rate might differ for populations with other characteristics not present in the simulated populations.</p><p id="p-46">The decline in correct classification rate with increasing proportion of individuals sampled in the population (question 6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS4.pdf">Figure S4</a> and <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>) may be puzzling at first. Here it is important to note that the number of dyads increases as the square of the number of individuals sampled, with a faster increase of the number of unrelated dyads compared to related dyads (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS5.pdf">Figure S5</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Skaug <em>et al.</em> 2010</a>). It seems that the influence of misclassifying unrelated individuals outweighs any correct classification rate gain due to improved estimates of allele frequencies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1/FigureS6.pdf">Figure S6</a>) due to more complete sampling.</p><p id="p-47">Census population size is a parameter space that we did not explore, even though it might influence the correct classification rate of an RCA. Some effects may be inferred from the subsampling simulations: the correct classification rate of the RCA decreased with increasing number of pedigree-based unrelated dyads. This finding suggests that in comparison with small populations, relationship assignments of samples originating from large populations potentially may require more loci for similar correct classification rate and/or certain categories may not reach satisfactory correct classification rates.</p><p id="p-48">We would like to emphasize that, especially with the facilitated development of genetic markers, (i) the effect of intrinsic population characteristics on correct classification rate should be taken into account for RCAs, and (ii) 95% correct classification rate can be achieved and should be favored over the 80% threshold that is widely used in paternity studies (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Marshall <em>et al.</em> 1998</a>). Also, the correct classification rate of a set of loci should be evaluated at the beginning of a project and considered in downstream analyses and interpretation. Unfortunately, sorely-needed software simulating populations that have realistic population parameters and track pedigrees through time are very rare, making RCA power analyses difficult. Instead, available simulation software uses simple demographic and life history models (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hoban 2014</a>) or researchers use custom-made simulations (this study, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Taylor <em>et al.</em> 2015</a>). Until software that simulates populations with realistic parameters are available, the relationship between correct classification rate, population characteristics, and genetic marker characteristics suggested in this study could be used as a rough guideline for power estimates.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We are grateful to Bob Dröge from the Millipede team for his advice on running the simulations on a HPC cluster and to João B. L. Gusmão Junior for improving the resolution of <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>. We would like to thank two anonymous reviewers for their helpful comments. A.M.K. was supported by a fellowship for prospective researchers from the Swiss National Science Foundation. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-6"><p id="p-50">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019323/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-51"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-27"><span class="received-label">Received </span>March 27, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-06-23"><span class="accepted-label">Accepted </span>June 23, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Kopps <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.1" data-doi="10.1016/j.forsciint.2004.06.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amorim</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Pros and cons in the use of SNPs in forensic kinship investigation: a comparative analysis with STRs.</span> <abbr class="cit-jnl-abbrev">Forensic Sci. Int.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForensic%2BScience%2BInternational%26rft.stitle%253DForensic%2BScience%2BInternational%26rft.aulast%253DAmorim%26rft.auinit1%253DA.%26rft.volume%253D150%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D21%26rft.atitle%253DPros%2Band%2Bcons%2Bin%2Bthe%2Buse%2Bof%2BSNPs%2Bin%2Bforensic%2Bkinship%2Binvestigation%253A%2Ba%2Bcomparative%2Banalysis%2Bwith%2BSTRs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.forsciint.2004.06.018%26rft_id%253Dinfo%253Apmid%252F15837005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.forsciint.2004.06.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15837005&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.2" data-doi="10.1534/genetics.105.048074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garza</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The power of single-nucleotide polymorphisms for large-scale parentage inference.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2567</span>–<span class="cit-lpage">2582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048074%26rft_id%253Dinfo%253Apmid%252F16387880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI1NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTgxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.3" data-doi="10.2307/3545516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avise</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Molecular population structure and the biogeographic history of a regional fauna: a case history with lessons for conservation biology.</span> <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D63%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3545516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3545516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992HK90600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.4" data-doi="10.1186/1471-2148-14-68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aykanat</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular pedigree reconstruction and estimation of evolutionary parameters in a wild Atlantic salmon river system with incomplete sampling: a power analysis.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D14%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-14-68%26rft_id%253Dinfo%253Apmid%252F24684698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-14-68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24684698&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.5" data-doi="10.1073/pnas.1120666109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamin</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cesarini</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Loos</span>  <span class="cit-name-given-names">M. J. H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawes</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koellinger</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genetic architecture of economic and political preferences.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">8026</span>–<span class="cit-lpage">8031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1120666109%26rft_id%253Dinfo%253Apmid%252F22566634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzIxLzgwMjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTgxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.6" data-doi="10.1016/S0169-5347(03)00225-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blouin</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">DNA-based methods for pedigree reconstruction and kinship analysis in natural populations.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D18%26rft.spage%253D503%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252803%252900225-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(03)00225-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185728100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.7" data-doi="10.1016/j.tree.2012.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonduriansky</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rethinking heredity, again.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBonduriansky%26rft.auinit1%253DR.%26rft.volume%253D27%26rft.issue%253D6%26rft.spage%253D330%26rft.epage%253D336%26rft.atitle%253DRethinking%2Bheredity%252C%2Bagain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2012.02.003%26rft_id%253Dinfo%253Apmid%252F22445060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2012.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22445060&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305105000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.8" data-doi="10.1038/hdy.1993.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brookfield</span>  <span class="cit-name-given-names">J. F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Use of single-locus DNA probes in the establishment of relatedness in wild populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D70%26rft.spage%253D660%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1993.94%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1993.94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.9" data-doi="10.1371/journal.pone.0101427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kopps</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bejder</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littleford-Colquhoun</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population differentiation and hybridisation of Australian Snubfin (<em>Orcaella heinsohni</em>) and Indo-Pacific humpback (<em>Sousa chinensis</em>) dolphins in North-Western Australia.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e101427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101427%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101427%26rft_id%253Dinfo%253Apmid%252F24988113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24988113&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.10" data-doi="10.1146/annurev-genet-110711-155534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identity by descent between distant relatives: detection and applications.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D46%26rft.spage%253D617%26rft.epage%253D633%26rft.atitle%253DIdentity%2Bby%2Bdescent%2Bbetween%2Bdistant%2Brelatives%253A%2Bdetection%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-110711-155534%26rft_id%253Dinfo%253Apmid%252F22994355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-110711-155534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22994355&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311568300028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.11" data-doi="10.1046/j.1365-294X.2003.01762.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilkington</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Fine-scale genetic structure in a free-living ungulate population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">733</span>–<span class="cit-lpage">742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DColtman%26rft.auinit1%253DD.%2BW.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D733%26rft.epage%253D742%26rft.atitle%253DFine-scale%2Bgenetic%2Bstructure%2Bin%2Ba%2Bfree-living%2Bungulate%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2003.01762.x%26rft_id%253Dinfo%253Apmid%252F12675828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2003.01762.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12675828&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.12" data-doi="10.1111/1755-0998.12219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cope</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanyon</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seddon</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollett</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and testing of a genetic marker-based pedigree reconstruction system ‘PR-genie’ incorporating size-class data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D857%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12219%26rft_id%253Dinfo%253Apmid%252F24373173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24373173&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.13" data-doi="10.1111/ele.12373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crean</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kopps</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonduriansky</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Revisiting telegony: offspring inherit an acquired characteristic of their mother’s previous mate.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D17%26rft.spage%253D1545%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12373%26rft_id%253Dinfo%253Apmid%252F25270393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270393&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.14" data-doi="10.1016/j.tree.2013.02.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">É.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Avatars of information: towards an inclusive evolutionary synthesis.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D28%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.02.010%26rft_id%253Dinfo%253Apmid%252F23540765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23540765&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320742700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.15" data-doi="10.1086/321195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duren</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Improved inference of relationship for pairs of individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DEpstein%26rft.auinit1%253DM.%2BP.%26rft.volume%253D67%26rft.issue%253D5%26rft.spage%253D1219%26rft.epage%253D1231%26rft.atitle%253DImproved%2Binference%2Bof%2Brelationship%2Bfor%2Bpairs%2Bof%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321195%26rft_id%253Dinfo%253Apmid%252F11032786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11032786&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165091600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.16" data-doi="10.1111/j.1755-0998.2011.03037.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertozzi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rise of the machines—recommendations for ecologists when using next generation sequencing for microsatellite development.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D1093%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.03037.x%26rft_id%253Dinfo%253Apmid%252F21679314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.03037.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21679314&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.17" data-doi="10.1186/1471-2164-15-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greminger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stolting</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nater</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goossens</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arora</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Generation of SNP datasets for orangutan population genomics using improved reduced-representation sequencing and direct comparisons of SNP calling algorithms.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-16%26rft_id%253Dinfo%253Apmid%252F24405840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24405840&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.18" data-doi="10.1371/journal.pone.0034267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macpherson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxonov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cryptic distant relatives are common in both isolated and cosmopolitan genetic samples.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e34267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHenn%26rft.auinit1%253DB.%2BM.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De34267%26rft.epage%253De34267%26rft.atitle%253DCryptic%2Bdistant%2Brelatives%2Bare%2Bcommon%2Bin%2Bboth%2Bisolated%2Band%2Bcosmopolitan%2Bgenetic%2Bsamples.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0034267%26rft_id%253Dinfo%253Apmid%252F22509285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0034267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22509285&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-other" id="cit-5.9.1815.19" data-doi="10.1534/g3.113.007500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">I. M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of pedigree relationship from genome sharing.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">1553</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007500%26rft_id%253Dinfo%253Apmid%252F23893739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvOS8xNTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.20" data-doi="10.1111/mec.12741"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoban</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An overview of the utility of population simulation software in molecular ecology.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2383</span>–<span class="cit-lpage">2401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D2383%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12741&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.21" data-doi="10.1101/gr.115972.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huff</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witherspoon</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonson</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watkins</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Maximum-likelihood estimation of recent shared ancestry (ERSA).</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115972.110%26rft_id%253Dinfo%253Apmid%252F21324875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvNzY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.22" data-doi="10.1111/j.1755-0998.2009.02787.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">O. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">COLONY: a program for parentage and sibship inference from multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D10%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2009.02787.x%26rft_id%253Dinfo%253Apmid%252F21565056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2009.02787.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565056&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276407300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-2" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.23" data-doi="10.1038/sj.hdy.6800009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalinowski</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">How many alleles per locus should be used to estimate genetic distances?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DKalinowski%26rft.auinit1%253DS.%2BT.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D65%26rft.atitle%253DHow%2Bmany%2Balleles%2Bper%2Blocus%2Bshould%2Bbe%2Bused%2Bto%2Bestimate%2Bgenetic%2Bdistances%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800009%26rft_id%253Dinfo%253Apmid%252F11813108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11813108&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176214400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.24" data-doi="10.1111/j.1365-294X.2007.03089.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalinowski</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taper</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DKalinowski%26rft.auinit1%253DS.%2BT.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D1099%26rft.epage%253D1106%26rft.atitle%253DRevising%2Bhow%2Bthe%2Bcomputer%2Bprogram%2BCERVUS%2Baccommodates%2Bgenotyping%2Berror%2Bincreases%2Bsuccess%2Bin%2Bpaternity%2Bassignment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03089.x%26rft_id%253Dinfo%253Apmid%252F17305863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03089.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17305863&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244245300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.25" data-doi="10.1016/j.anbehav.2012.08.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kopps</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherwin</span>  <span class="cit-name-given-names">W. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Modelling the emergence and stability of a vertically transmitted cultural trait in bottlenose dolphins.</span> <abbr class="cit-jnl-abbrev">Anim. Behav.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1347</span>–<span class="cit-lpage">1362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BBehav.%26rft.volume%253D84%26rft.spage%253D1347%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.anbehav.2012.08.029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.anbehav.2012.08.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000311953200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.26" data-doi="10.1007/s12686-012-9727-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kopps</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rollins</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Isolation and characterisation of polymorphic microsatellite loci for Noisy Miners <em>Manorina melanocephala</em>, with successful cross-amplification in Bell Miners <em>M. melanophrys</em>.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet. Resour.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%2BResour.%26rft.volume%253D5%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12686-012-9727-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12686-012-9727-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.27" data-doi="10.1098/rspb.2013.3245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kopps</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackermann</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherwin</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bejder</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cultural transmission of tool use combined with habitat specialisations leads to fine-scale genetic structure in bottlenose dolphins</span>. <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20133245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.3245%26rft_id%253Dinfo%253Apmid%252F24648223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3ODIvMjAxMzMyNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTgxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.28" data-doi="10.1371/journal.pgen.1004144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glusman</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shankaracharya</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Relationship estimation from whole-genome sequence data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004144%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004144%26rft_id%253Dinfo%253Apmid%252F24497848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24497848&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.29" data-doi="10.1046/j.1365-294x.1998.00374.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Statistical confidence for likelihood-based paternity inference in natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMarshall%26rft.auinit1%253DT.%2BC.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D639%26rft.epage%253D655%26rft.atitle%253DStatistical%2Bconfidence%2Bfor%2Blikelihood-based%2Bpaternity%2Binference%2Bin%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.1998.00374.x%26rft_id%253Dinfo%253Apmid%252F9633105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.1998.00374.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9633105&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073810200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.30" data-doi="10.1016/j.tree.2004.01.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite> and the SNP workshop group, <span class="cit-pub-date">2004</span> <span class="cit-article-title">SNPs in ecology, evolution and conservation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D19%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2004.01.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2004.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220842400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.31" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.32" data-doi="10.1111/j.1755-0998.2010.02887.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palsbøll</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peery</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérubé</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detecting populations in the ‘ambiguous’ zone: kinship-based estimation of population structure at low genetic divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">797</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D10%26rft.spage%253D797%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02887.x%26rft_id%253Dinfo%253Apmid%252F21565091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02887.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565091&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.33" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polanowski</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robbins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarman</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epigenetic estimation of age in humpback whales.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">976</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D976%26rft_id%253Dinfo%253Apmid%252F24606053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24606053&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.35" data-doi="10.2307/2409206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Queller</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodnight</span>  <span class="cit-name-given-names">K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Estimating relatedness using genetic-markers.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D43%26rft.spage%253D258%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989T799700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.36" data-doi="10.1093/bioinformatics/btp064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klemm</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">FRANz: reconstruction of wild multi-generation pedigrees.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2134</span>–<span class="cit-lpage">2139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp064%26rft_id%253Dinfo%253Apmid%252F19202194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjEzNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.37" data-doi="10.1002/elps.200500671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Børsting</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balogh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A multiplex assay with 52 single nucleotide polymorphisms for human identification.</span> <abbr class="cit-jnl-abbrev">Electrophoresis</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1713</span>–<span class="cit-lpage">1724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.stitle%253DElectrophoresis%26rft.aulast%253DSanchez%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D27%26rft.issue%253D9%26rft.spage%253D1713%26rft.epage%253D1724%26rft.atitle%253DA%2Bmultiplex%2Bassay%2Bwith%2B52%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bfor%2Bhuman%2Bidentification.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.200500671%26rft_id%253Dinfo%253Apmid%252F16586411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elps.200500671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16586411&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237685600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.38" data-doi="10.1111/j.1365-294X.2010.04554.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santure</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkhead</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the use of large marker panels to estimate inbreeding and relatedness: empirical and simulation studies of a pedigreed zebra finch population typed at 771 SNPs.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1439</span>–<span class="cit-lpage">1451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSanture%26rft.auinit1%253DA.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D1439%26rft.epage%253D1451%26rft.atitle%253DOn%2Bthe%2Buse%2Bof%2Blarge%2Bmarker%2Bpanels%2Bto%2Bestimate%2Binbreeding%2Band%2Brelatedness%253A%2Bempirical%2Band%2Bsimulation%2Bstudies%2Bof%2Ba%2Bpedigreed%2Bzebra%2Bfinch%2Bpopulation%2Btyped%2Bat%2B771%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04554.x%26rft_id%253Dinfo%253Apmid%252F20149098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04554.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20149098&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275761300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.39" data-doi="10.1111/j.1755-0998.2010.02833.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skaug</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berube</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palsbøll</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detecting dyads of related individuals in large collections of DNA-profiles by controlling the false discovery rate.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D10%26rft.spage%253D693%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02833.x%26rft_id%253Dinfo%253Apmid%252F21565074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02833.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565074&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Relatedness in the post-genomic era: is it still useful?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D33%26rft_id%253Dinfo%253Apmid%252F25404112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25404112&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.41" data-doi="10.1007/s10592-015-0709-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramstad</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Valid estimates of individual inbreeding coefficients from marker-based pedigrees are not feasible in wild populations with low allelic diversity.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D16%26rft.spage%253D901%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10592-015-0709-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10592-015-0709-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimating quantitative genetic parameters using sibships reconstructed from marker data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1961</span>–<span class="cit-lpage">1972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThomas%26rft.auinit1%253DS.%2BC.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D1961%26rft.epage%253D1972%26rft.atitle%253DEstimating%2BQuantitative%2BGenetic%2BParameters%2BUsing%2BSibships%2BReconstructed%2BFrom%2BMarker%2BData%26rft_id%253Dinfo%253Apmid%252F10924488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzE5NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTgxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-5.9.1815.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Estimation of Relationships from Genetic Data</span>, pp. <span class="cit-fpage">255</span>–<span class="cit-lpage">269</span> in <span class="cit-source">Handbook of Statistics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Chakrabort</span>  <span class="cit-name-given-names">R.</span></span>. <span class="cit-publ-name">Elsevier Science, Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.44" data-doi="10.1111/j.1469-1809.1975.tb00120.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">The estimation of pairwise relationships.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DThompson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D188%26rft.atitle%253DThe%2Bestimation%2Bof%2Bpairwise%2Brelationships.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1975.tb00120.x%26rft_id%253Dinfo%253Apmid%252F1052764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1975.tb00120.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1052764&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AX72600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.45" data-doi="10.1051/gse:2002009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vignal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">SanCristobal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eggen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A review on SNP and other types of molecular markers and their use in animal genetics.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DVignal%26rft.auinit1%253DA.%26rft.volume%253D34%26rft.issue%253D3%26rft.spage%253D275%26rft.epage%253D305%26rft.atitle%253DA%2Breview%2Bon%2BSNP%2Band%2Bother%2Btypes%2Bof%2Bmolecular%2Bmarkers%2Band%2Btheir%2Buse%2Bin%2Banimal%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2002009%26rft_id%253Dinfo%253Apmid%252F12081799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2002009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12081799&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176561700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.46" data-doi="10.1534/genetics.166.4.1963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sibship reconstruction from genetic data with typing errors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D166%26rft.issue%253D4%26rft.spage%253D1963%26rft.epage%253D1979%26rft.atitle%253DSibship%2BReconstruction%2BFrom%2BGenetic%2BData%2BWith%2BTyping%2BErrors%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.4.1963%26rft_id%253Dinfo%253Apmid%252F15126412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni80LzE5NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTgxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-2" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.47" data-doi="10.1016/j.tpb.2005.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Informativeness of genetic markers for pairwise relationship and relatedness inference.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DSansone%26rft.auinit1%253DE.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D300%26rft.epage%253D321%26rft.atitle%253DInformativeness%2Bof%2Bgenetic%2Bmarkers%2Bfor%2Bpairwise%2Brelationship%2Band%2Brelatedness%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2005.11.003%26rft_id%253Dinfo%253Apmid%252F16388833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2005.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16388833&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Triadic IBD coefficients and applications to estimating pairwise relatedness.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D89%26rft.issue%253D3%26rft.spage%253D135%26rft.epage%253D153%26rft.atitle%253DTriadic%2BIBD%2Bcoefficients%2Band%2Bapplications%2Bto%2Bestimating%2Bpairwise%2Brelatedness.%26rft_id%253Dinfo%253Apmid%252F17894908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17894908&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250908100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.49" data-doi="10.1111/1755-0998.12106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A simulation module in the computer program colony for sibship and parentage analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">734</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D13%26rft.spage%253D734%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12106%26rft_id%253Dinfo%253Apmid%252F23615269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23615269&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.50" data-doi="10.1111/mec.12806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Estimation of migration rates from marker-based parentage analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">3191</span>–<span class="cit-lpage">3213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D3191%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.51" data-doi="10.1534/genetics.108.100214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santure</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Parentage and sibship inference from multilocus genotype data under polygamy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1579</span>–<span class="cit-lpage">1594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100214%26rft_id%253Dinfo%253Apmid%252F19221199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS80LzE1NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTgxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1815.52" data-doi="10.1038/nrg1960"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hepler</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic relatedness analysis: modern data and new challenges.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253D771%26rft.epage%253D780%26rft.atitle%253DGenetic%2Brelatedness%2Banalysis%253A%2Bmodern%2Bdata%2Band%2Bnew%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1960%26rft_id%253Dinfo%253Apmid%252F16983373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1960&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16983373&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1815.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241158700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/9/1815.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/9/1805" title="Comparative Transcriptome Analysis of the Cosmopolitan Marine Fungus Corollospora maritima Under Two Physiological Conditions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/9/1827" title="Transcriptomic Analysis of Musca domestica to Reveal Key Genes of the Prophenoloxidase-Activating System" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/9"><span>Volume 5 Issue 9, September 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/9.cover-source.jpg?itok=EE-GjT57" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1815.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yyjC43Ve-duBF0JBfxNNO-hsyZKZX745RCTNeBTiUR4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/9/1815" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/9/1815" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/9/1815" class="active" data-icon-position="" data-hide-link-title="0">How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460023" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qoynE4z5_s7nOHu-REbsHaigJG_2649u6k3E_WPPIGk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/9/1815.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/9/1815" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-iTsOguFslYpQCi1DrFKIxn3PgwMh3ERfCnXspiB7sF8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/9/1815" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/9/1815.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460023" id="node460023--2" data-pisa="ggg;5/9/1815" data-pisa-master="ggg;g3.115.019323" data-apath="/ggg/5/9/1815.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna M. Kopps</span>, <span class="highwire-citation-author" data-delta="1">Jungkoo Kang</span>, <span class="highwire-citation-author" data-delta="2">William B. Sherwin</span> and <span class="highwire-citation-author" data-delta="3">Per J. Palsbøll</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1815-1826;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019323 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460023--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna M. Kopps </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Evolution & Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BM.%2BKopps%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kopps%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BM.%2BKopps%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.kopps@gmx.com" class="" data-icon-position="" data-hide-link-title="0">anna.kopps@gmx.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jungkoo Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>IceLab, Umeå University, 901 87, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jungkoo%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJungkoo%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William B. Sherwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Evolution & Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Murdoch University Cetacean Research Unit, Centre for Fish, Fisheries and Aquatic Ecosystems Research, School of Veterinary and Life Sciences, Murdoch University, Murdoch, WA 6150, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BB.%2BSherwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherwin%20WB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BB.%2BSherwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Per J. Palsbøll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Per%2BJ.%2BPalsb%C3%B8ll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palsb%C3%B8ll%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APer%2BJ.%2BPalsb%25C3%25B8ll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/9/1815" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/9/1815&amp;title=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/9/1815&amp;title=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/9/1815&amp;title=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/9/1815&amp;text=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/9/1815&amp;title=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/9/1815&amp;t=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/9/1815&amp;title=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/9/1815&amp;title=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460023/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460023" id="node460023--3" data-pisa="ggg;5/9/1815" data-pisa-master="ggg;g3.115.019323" data-apath="/ggg/5/9/1815.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Well Do Molecular and Pedigree Relatedness Correspond, in Populations with Diverse Mating Systems, and Various Types and Quantities of Molecular and Demographic Data?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna M. Kopps</span>, <span class="highwire-citation-author" data-delta="1">Jungkoo Kang</span>, <span class="highwire-citation-author" data-delta="2">William B. Sherwin</span> and <span class="highwire-citation-author" data-delta="3">Per J. Palsbøll</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1815-1826;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019323 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460023--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna M. Kopps </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Evolution & Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BM.%2BKopps%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kopps%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BM.%2BKopps%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.kopps@gmx.com" class="" data-icon-position="" data-hide-link-title="0">anna.kopps@gmx.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jungkoo Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>IceLab, Umeå University, 901 87, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jungkoo%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJungkoo%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William B. Sherwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Evolution & Ecology Research Centre, School of Biological, Earth, and Environmental Sciences, University of New South Wales, Sydney, New South Wales 2052, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Murdoch University Cetacean Research Unit, Centre for Fish, Fisheries and Aquatic Ecosystems Research, School of Veterinary and Life Sciences, Murdoch University, Murdoch, WA 6150, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BB.%2BSherwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherwin%20WB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BB.%2BSherwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Per J. Palsbøll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Evolution and Conservation, Groningen Institute for Evolutionary Life Sciences, University of Groningen, 9747 AG Groningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Per%2BJ.%2BPalsb%C3%B8ll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palsb%C3%B8ll%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APer%2BJ.%2BPalsb%25C3%25B8ll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1815&amp;rft.epage=1826&amp;rft.atitle=How%20Well%20Do%20Molecular%20and%20Pedigree%20Relatedness%20Correspond%2C%20in%20Populations%20with%20Diverse%20Mating%20Systems%2C%20and%20Various%20Types%20and%20Quantities%20of%20Molecular%20and%20Demographic%20Data%3F&amp;rft.volume=5&amp;rft.issue=9&amp;rft.date=2015-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kopps%2C+Anna+M.&amp;rft.au=Kang%2C+Jungkoo&amp;rft.au=Sherwin%2C+William+B.&amp;rft.au=Palsb%C3%B8ll%2C+Per+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460023/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460023/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460023/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460023/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460023/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460023/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460023/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460023/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460023/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460023/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460023/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460023/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/9/1815" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/9/1815.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/9/1815.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/9/1815.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/9/1815.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
