<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/9/1865" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/9/1865.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/9/1865.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/9/1865.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/9/1865.atom" />
<meta name="HW.pisa" content="ggg;5/9/1865" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Enrichment of H3K9me2 on Unsynapsed Chromatin in Caenorhabditis elegans Does Not Target de Novo Sites" />
<meta name="DC.Identifier" content="10.1534/g3.115.019828" />
<meta name="DC.Date" content="2015-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Guo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Many organisms alter the chromatin state of unsynapsed chromosomes during meiotic prophase, a phenomenon hypothesized to function in maintaining germline integrity. In Caenorhabditis elegans , histone H3 lysine 9 dimethylation (H3K9me2) is detected by immunolabeling as enriched on unsynapsed meiotic chromosomes. Loss of the SET domain protein, [MET-2][1], greatly reduces H3K9me2 abundance and results in germline mortality. Here, we used [him-8][2] mutations to disable X chromosome synapsis and performed a combination of molecular assays to map the sites of H3K9me2 accumulation, evaluate H3K9me2 abundance in germline vs. whole animals, and evaluate the impact of H3K9me2 loss on the germline transcriptome. Our data indicate that H3K9me2 is elevated broadly across the X chromosome and at defined X chromosomal sites in [him-8][2] adults compared with controls. H3K9me2 levels are also elevated to a lesser degree at sites on synapsed chromosomes in [him-8][2] adults compared with controls. These results suggest that [MET-2][1] activity is elevated in [him-8][2] mutants generally as well as targeted preferentially to the unsynapsed X. Abundance of H3K9me2 and other histone H3 modifications is low in germline chromatin compared with whole animals, which may facilitate genome reprogramming during gametogenesis. Loss of H3K9me2 has a subtle impact on the [him-8][2] germline transcriptome, suggesting H3K9me2 may not be a major regulator of developmental gene expression in C. elegans . We hypothesize H3K9me2 may have a structural function critical for germline immortality, and a greater abundance of these marks may be required when a chromosome does not synapse.

 [1]: http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene" />
<meta name="DC.Contributor" content="Yiqing Guo" />
<meta name="DC.Contributor" content="Bing Yang" />
<meta name="DC.Contributor" content="Yini Li" />
<meta name="DC.Contributor" content="Xia Xu" />
<meta name="DC.Contributor" content="Eleanor M. Maine" />
<meta name="article:published_time" content="2015-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Enrichment of H3K9me2 on Unsynapsed Chromatin in Caenorhabditis elegans Does Not Target de Novo Sites" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Many organisms alter the chromatin state of unsynapsed chromosomes during meiotic prophase, a phenomenon hypothesized to function in maintaining germline integrity. In &lt;i&gt;Caenorhabditis elegans&lt;/i&gt;, histone H3 lysine 9 dimethylation (H3K9me2) is detected by immunolabeling as enriched on unsynapsed meiotic chromosomes. Loss of the SET domain protein, MET-2, greatly reduces H3K9me2 abundance and results in germline mortality. Here, we used &lt;i&gt;him-8&lt;/i&gt; mutations to disable X chromosome synapsis and performed a combination of molecular assays to map the sites of H3K9me2 accumulation, evaluate H3K9me2 abundance in germline &lt;i&gt;vs.&lt;/i&gt; whole animals, and evaluate the impact of H3K9me2 loss on the germline transcriptome. Our data indicate that H3K9me2 is elevated broadly across the X chromosome and at defined X chromosomal sites in &lt;i&gt;him-8&lt;/i&gt; adults compared with controls. H3K9me2 levels are also elevated to a lesser degree at sites on synapsed chromosomes in &lt;i&gt;him-8&lt;/i&gt; adults compared with controls. These results suggest that MET-2 activity is elevated in &lt;i&gt;him-8&lt;/i&gt; mutants generally as well as targeted preferentially to the unsynapsed X. Abundance of H3K9me2 and other histone H3 modifications is low in germline chromatin compared with whole animals, which may facilitate genome reprogramming during gametogenesis. Loss of H3K9me2 has a subtle impact on the &lt;i&gt;him-8&lt;/i&gt; germline transcriptome, suggesting H3K9me2 may not be a major regulator of developmental gene expression in &lt;i&gt;C. elegans&lt;/i&gt;. We hypothesize H3K9me2 may have a structural function critical for germline immortality, and a greater abundance of these marks may be required when a chromosome does not synapse.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/09/01" />
<meta name="citation_mjid" content="ggg;5/9/1865" />
<meta name="citation_id" content="5/9/1865" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/9/1865" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/9/1865.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/9/1865.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/9/1865.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019828" />
<meta name="citation_pmid" content="26156747" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1865" />
<meta name="citation_lastpage" content="1878" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yiqing Guo" />
<meta name="citation_author_institution" content="Department of Biology, Syracuse University, Syracuse, New York 13244" />
<meta name="citation_author" content="Bing Yang" />
<meta name="citation_author_institution" content="Department of Biology, Syracuse University, Syracuse, New York 13244" />
<meta name="citation_author" content="Yini Li" />
<meta name="citation_author_institution" content="Department of Biology, Syracuse University, Syracuse, New York 13244" />
<meta name="citation_author" content="Xia Xu" />
<meta name="citation_author_institution" content="Department of Biology, Syracuse University, Syracuse, New York 13244" />
<meta name="citation_author" content="Eleanor M. Maine" />
<meta name="citation_author_institution" content="Department of Biology, Syracuse University, Syracuse, New York 13244" />
<meta name="citation_author_email" content="emmaine@syr.edu" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=E. C. Andersen;citation_author=H. R. Horvitz;citation_title=Two C. elegans histone methyltransferases repress lin-3 EGF transcription to inhibit vulval development.;citation_volume=134;citation_year=2007;citation_pmid=17634190;citation_doi=10.1242/dev.009373" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. J. Bannister;citation_author=P. Zegerman;citation_author=J. F. Partridge;citation_author=E. A. Miska;citation_author=J. O. Thomas;citation_title=Selective recognition of methylated lysine 9 on histone H3 by the HP1 chromo domain.;citation_pages=120-124;citation_volume=410;citation_year=2001;citation_issue=6824;citation_pmid=11242054;citation_doi=10.1038/35065138" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. J. Bean;citation_author=C. E. Schaner;citation_author=W. G. Kelly;citation_title=Meiotic pairing and imprinted X chromatin assembly in Caenorhabditis elegans.;citation_pages=100-105;citation_volume=36;citation_year=2004;citation_issue=1;citation_pmid=14702046;citation_doi=10.1038/ng1283" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. B. Bessler;citation_author=E. C. Andersen;citation_author=A. M. Villeneuve;citation_title=Differential localization and independent acquisition of the H3K9me2 and H3K9me3 chromatin modifications in the Caenorhabditis elegans adult germ line.;citation_pages=e1000830-e1000830;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=20107519;citation_doi=10.1371/journal.pgen.1000830" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Brenner;citation_title=THE GENETICS OF CAENORHABDITIS ELEGANS;citation_pages=71-94;citation_volume=77;citation_year=1974;citation_issue=1;citation_pmid=4366476;citation_doi=10.1051/medsci/20031967725" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. M. Checchi;citation_author=J. Engebrecht;citation_title=Caenorhabditis elegans histone methyltransferase MET-2 shields the male X chromosome from checkpoint machinery and mediates meiotic sex chromosome inactivation.;citation_pages=e1002267-e1002267;citation_volume=7;citation_year=2011;citation_issue=9;citation_pmid=21909284;citation_doi=10.1371/journal.pgen.1002267" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. S. Cheung;citation_author=T. A. Down;citation_author=I. Latorre;citation_author=J. Ahringer;citation_title=Systematic bias in high-throughput sequencing data and its correction by BEADS.;citation_volume=39;citation_year=2011;citation_pmid=21646344;citation_doi=10.1093/nar/gkr425" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=I. Chiolo;citation_author=A. Minoda;citation_author=S. U. Colmenares;citation_author=A. Polyzos;citation_author=S. V. Costes;citation_title=Double-strand breaks in heterochromatin move outside of a dynamic HP1a domain to complete recombinational repair.;citation_pages=732-744;citation_volume=144;citation_year=2011;citation_issue=5;citation_pmid=21353298;citation_doi=10.1016/j.cell.2011.02.012" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. M. Claycomb;citation_author=P. J. Batista;citation_author=K. M. Pang;citation_author=W. Gu;citation_author=J. J. Vasale;citation_title=The Argonaute CSR-1 and its 22G-RNA cofactors are required for holocentric chromosome segregation.;citation_pages=123-134;citation_volume=139;citation_year=2009;citation_issue=1;citation_pmid=19804758;citation_doi=10.1016/j.cell.2009.09.014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. C. Conine;citation_author=P. J. Batista;citation_author=W. Gu;citation_author=J. M. Claycomb;citation_author=D. A. Chaves;citation_title=Argonautes ALG-3 and ALG-4 are required for spermatogenesis-specific 26G-RNAs and thermotolerant sperm in Caenorhabditis elegans.;citation_volume=107;citation_year=2010;citation_pmid=20133686;citation_doi=10.1073/pnas.0911685107" />
<meta name="citation_reference" content="citation_journal_title=Biol. Reprod.;citation_author=K. Deguchi;citation_author=G. Nagamatsu;citation_author=H. Miyachi;citation_author=Y. Kato;citation_author=S. Morita;citation_title=Posttranscriptional regulation of histone lysine methyltransferase GLP in embryonic male mouse germ cells.;citation_volume=88;citation_year=2013;citation_pmid=23284137;citation_doi=10.1095/biolreprod.112.103572" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=M. Di Giacomo;citation_author=S. Comazzetto;citation_author=H. Saini;citation_author=S. De Fazio;citation_author=C. Carrieri;citation_title=Multiple epigenetic mechanisms and the piRNA pathway enforce LINE1 silencing during adult spermatogenesis.;citation_pages=601-608;citation_volume=50;citation_year=2013;citation_pmid=23706823;citation_doi=10.1016/j.molcel.2013.04.026" />
<meta name="citation_reference" content="citation_journal_title=Epigenetics Chromatin;citation_author=M. Di Giacomo;citation_author=S. Comazzetto;citation_author=S. C. Sampath;citation_author=D. O’Carroll;citation_title=G9a co-suppresses LINE1 elements in spermatogonia.;citation_pages=24;citation_volume=7;citation_year=2014;citation_pmid=25276231;citation_doi=10.1186/1756-8935-7-24" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_title=Genome Res.;citation_author=J.M. Garrigues;citation_author=S. Sidoli;citation_author=B.A. Garcia;citation_author=S. Strome;citation_title=Defining heterochromatin in C. elegans through genome-wide analysis of the heterochromatin protein 1 homolog HPL-2.;citation_volume=25;citation_year=2015;citation_pmid=25467431;citation_doi=10.1101/gr.180489.114" />
<meta name="citation_reference" content="citation_journal_title=Electrophoresis;citation_author=M. Gassmann;citation_author=B. Grenacher;citation_author=B. Rohde;citation_author=J. Vogel;citation_title=Quantifying Western blots: pitfalls of densitometry.;citation_pages=1845-1855;citation_volume=30;citation_year=2009;citation_pmid=19517440;citation_doi=10.1002/elps.200800720" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Gassmann;citation_author=A. Rechtsteiner;citation_author=K. W. Yuen;citation_author=A. Muroyama;citation_author=T. Egelhofer;citation_title=An inverse relationship to germline transcription defines centromeric chromatin in C. elegans.;citation_pages=534-537;citation_volume=484;citation_year=2012;citation_issue=7395;citation_pmid=22495302;citation_doi=10.1038/nature10973" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. B. Gerstein;citation_author=Z. J. Lu;citation_author=E. L. Van Nostrand;citation_author=C. Cheng;citation_author=B. I. Arshinoff;citation_title=Integrative analysis of the Caenorhabditis elegans genome by the modENCODE project.;citation_volume=330;citation_year=2010;citation_pmid=21177976;citation_doi=10.1126/science.1196914" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=W. Gu;citation_author=M. Shirayama;citation_author=D. Conte;citation_author=J. Vasale;citation_author=P. J. Batista;citation_title=Distinct argonaute-mediated 22G-RNA pathways direct genome surveillance in the C. elegans germline.;citation_pages=231-244;citation_volume=36;citation_year=2009;citation_issue=2;citation_pmid=19800275;citation_doi=10.1016/j.molcel.2009.09.020" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. Handel;citation_title=Marking Xs, together and separately.;citation_pages=12-13;citation_volume=36;citation_year=2004;citation_issue=1;citation_pmid=14702033;citation_doi=10.1038/ng0104-12" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. W. Ho;citation_author=Y. L. Jung;citation_author=T. Liu;citation_author=B. H. Alver;citation_author=S. Lee;citation_title=Comparative analysis of metazoan chromatin organization.;citation_pages=449-452;citation_volume=512;citation_year=2014;citation_pmid=25164756;citation_doi=10.1038/nature13415" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Hodgkin;citation_author=H. R. Horvitz;citation_author=S. Brenner;citation_title=NONDISJUNCTION MUTANTS OF THE NEMATODE CAENORHABDITIS ELEGANS;citation_pages=67-94;citation_volume=91;citation_year=1979;citation_issue=1;citation_pmid=17248881" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Huang;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists.;citation_volume=37;citation_year=2009;citation_pmid=19033363;citation_doi=10.1093/nar/gkn923" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. Ikegami;citation_author=T. A. Egelhofer;citation_author=S. Strome;citation_author=J. D. Lieb;citation_title=Caenorhabditis elegans chromosome arms are anchored to the nuclear membrane via discontinuous association with LEM-2.;citation_pages=R120-R120;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21176223;citation_doi=10.1186/gb-2010-11-12-r120" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Jaramillo-Lambert;citation_author=J. Engebrecht;citation_title=A single unpaired and transcriptionally silenced X chromosome locally precludes checkpoint signaling in the Caenorhabditis elegans germ line.;citation_volume=184;citation_year=2010;citation_pmid=20008570;citation_doi=10.1534/genetics.109.110338" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Kaur;citation_author=M. V. Rockman;citation_title=Crossover heterogeneity in the absence of hotspots in Caenorhabditis elegans.;citation_volume=196;citation_year=2014;citation_pmid=24172135;citation_doi=10.1534/genetics.113.158857" />
<meta name="citation_reference" content="citation_journal_title=Epigenetics Chromatin;citation_author=W. G. Kelly;citation_title=Transgenerational epigenetics in the germline cycle of Caenorhabditis elegans.;citation_pages=6;citation_volume=7;citation_year=2014;citation_pmid=24678826;citation_doi=10.1186/1756-8935-7-6" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=W. G. Kelly;citation_author=R. Aramayo;citation_title=Meiotic silencing and the epigenetics of sex.;citation_pages=633-651;citation_volume=15;citation_year=2007;citation_issue=5;citation_pmid=17674151;citation_doi=10.1007/s10577-007-1143-0" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=W. G. Kelly;citation_author=C. E. Schaner;citation_author=A. F. Dernburg;citation_author=M. H. Lee;citation_author=S. K. Kim;citation_title=X-chromosome silencing in the germline of C. elegans.;citation_pages=479-492;citation_volume=129;citation_year=2002;citation_pmid=11807039" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. W. L’Hernault;citation_author=D. C. Shakes;citation_author=S. Ward;citation_title=Developmental genetics of chromosome I spermatogenesis-defective mutants in the nematode Caenorhabditis elegans.;citation_pages=435-452;citation_volume=120;citation_year=1988;citation_issue=2;citation_pmid=3197956" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=T. Liu;citation_author=A. Rechtsteiner;citation_author=T. A. Egelhofer;citation_author=A. Vielle;citation_author=I. Latorre;citation_title=Broad chromosomal domains of histone modification patterns in C. elegans.;citation_volume=21;citation_year=2011;citation_pmid=21177964;citation_doi=10.1101/gr.115519.110" />
<meta name="citation_reference" content="citation_journal_abbrev=Int Rev Cell Mol Biol;citation_author=E. M. Maine;citation_title=Meiotic silencing in Caenorhabditis elegans.;citation_pages=91-134;citation_volume=282;citation_year=2010;citation_pmid=20630467;citation_doi=10.1016/S1937-6448(10)82002-7" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=E. M. Maine;citation_author=J. Hauth;citation_author=T. Ratliff;citation_author=V. E. Vought;citation_author=X. She;citation_title=EGO-1, a putative RNA-dependent RNA polymerase, is required for heterochromatin assembly on unpaired dna during C. elegans meiosis.;citation_pages=1972-1978;citation_volume=15;citation_year=2005;citation_issue=21;citation_pmid=16271877;citation_doi=10.1016/j.cub.2005.09.049" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=B. L. Nelms;citation_author=W. Hanna-Rose;citation_title=C. elegans HIM-8 functions outside of meiosis to antagonize EGL-13 Sox protein function.;citation_pages=392-402;citation_volume=293;citation_year=2006;citation_issue=2;citation_pmid=16546157;citation_doi=10.1016/j.ydbio.2006.02.010" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Z. Ni;citation_author=E. Chen;citation_author=S. G. Gu;citation_title=Complex coding of endogenous siRNA, transcriptional silencing and H3K9 methylation on native targets of germline nuclear RNAi in C. elegans.;citation_pages=1157;citation_volume=15;citation_year=2014;citation_pmid=25534009;citation_doi=10.1186/1471-2164-15-1157" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=C. M. Phillips;citation_author=A. F. Dernburg;citation_title=A family of zinc-finger proteins is required for chromosome-specific pairing and synapsis during meiosis in C. elegans.;citation_pages=817-829;citation_volume=11;citation_year=2006;citation_issue=6;citation_pmid=17141157;citation_doi=10.1016/j.devcel.2006.09.020" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. M. Phillips;citation_author=C. Wong;citation_author=N. Bhalla;citation_author=P. M. Carlton;citation_author=P. Weiser;citation_title=HIM-8 binds to the X chromosome pairing center and mediates chromosome-specific meiotic synapsis.;citation_pages=1051-1063;citation_volume=123;citation_year=2005;citation_issue=6;citation_pmid=16360035;citation_doi=10.1016/j.cell.2005.09.035" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=C. M. Phillips;citation_author=X. Meng;citation_author=L. Zhang;citation_author=J. H. Chretien;citation_author=F. D. Urnov;citation_title=Identification of chromosome sequence motifs that mediate meiotic pairing and synapsis in C. elegans.;citation_pages=934-942;citation_volume=11;citation_year=2009;citation_issue=8;citation_pmid=19620970;citation_doi=10.1038/ncb1904" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Qiao;citation_author=J. L. Lissemore;citation_author=P. Shu;citation_author=A. Smardon;citation_author=M. B. Gelber;citation_title=Enhancers of glp-1, a gene required for cell-signaling in Caenorhabditis elegans, define a set of genes required for germline development.;citation_pages=551-569;citation_volume=141;citation_year=1995;citation_issue=2;citation_pmid=8647392" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. C. Reddy;citation_author=A. M. Villeneuve;citation_title=C. elegans HIM-17 links chromatin modification and competence for initiation of meiotic recombination.;citation_pages=439-452;citation_volume=118;citation_year=2004;citation_issue=4;citation_pmid=15315757;citation_doi=10.1016/j.cell.2004.07.026" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=V. Reinke;citation_author=H. E. Smith;citation_author=J. Nance;citation_author=J. Wang;citation_author=C. Van Doren;citation_title=A global profile of germline gene expression in C. elegans.;citation_pages=605-616;citation_volume=6;citation_year=2000;citation_issue=3;citation_pmid=11030340;citation_doi=10.1016/S1097-2765(00)00059-9" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=Y. Seki;citation_author=K. Hayashi;citation_author=K. Itoh;citation_author=M. Mizugaki;citation_author=M. Saitou;citation_title=Extensive and orderly reprogramming of genome-wide chromatin modifications associated with specification and early development of germ cells in mice.;citation_pages=440-458;citation_volume=278;citation_year=2005;citation_issue=2;citation_pmid=15680362;citation_doi=10.1016/j.ydbio.2004.11.025" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=M. Seth;citation_author=M. Shirayama;citation_author=W. Gu;citation_author=T. Ishidate;citation_author=D. Conte;citation_title=The C. elegans CSR-1 argonaute pathway counteracts epigenetic silencing to promote germline gene expression.;citation_pages=656-663;citation_volume=27;citation_year=2013;citation_pmid=24360782;citation_doi=10.1016/j.devcel.2013.11.014" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=X. She;citation_author=X. Xu;citation_author=A. Fedotov;citation_author=W. G. Kelly;citation_author=E. M. Maine;citation_title=Regulation of heterochromatin assembly on unpaired chromosomes during Caenorhabditis elegans meiosis by components of a small RNA-mediated pathway.;citation_pages=e1000624-e1000624;citation_volume=5;citation_year=2009;citation_issue=8;citation_pmid=19714217;citation_doi=10.1371/journal.pgen.1000624" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Shin;citation_author=T. Liu;citation_author=A. K. Manrai;citation_author=X. S. Liu;citation_title=CEAS: cis-regulatory element annotation system.;citation_volume=25;citation_year=2009;citation_pmid=19689956;citation_doi=10.1093/bioinformatics/btp479" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Sun;citation_author=B. L. Nelms;citation_author=S. F. Sleiman;citation_author=H. M. Chamberlin;citation_author=W. Hanna-Rose;citation_title=Modulation of Caenorhabditis elegans transcription factor activity by HIM-8 and the related Zinc-Finger ZIM proteins.;citation_volume=177;citation_year=2007;citation_pmid=17720937;citation_doi=10.1534/genetics.107.070847" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=Y. Takada;citation_author=C. Naruse;citation_author=Y. Costa;citation_author=T. Shirakawa;citation_author=M. Tachibana;citation_title=HP1{gamma} links histone methylation marks to meiotic synapsis in mice;citation_pages=4207-4217;citation_volume=138;citation_year=2011;citation_issue=19;citation_pmid=21896631;citation_doi=10.1242/dev.064444" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. D. Towbin;citation_author=C. Gonzalez-Aguilera;citation_author=R. Sack;citation_author=D. Gaidatzis;citation_author=V. Kalck;citation_title=Step-wise methylation of histone H3K9 positions heterochromatin at the nuclear periphery.;citation_pages=934-947;citation_volume=150;citation_year=2012;citation_issue=5;citation_pmid=22939621;citation_doi=10.1016/j.cell.2012.06.051" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=C. Trapnell;citation_author=D. G. Hendrickson;citation_author=M. Sauvageau;citation_author=L. Goff;citation_author=J. L. Rinn;citation_title=Differential analysis of gene regulation at transcript resolution with RNA-seq.;citation_pages=46-53;citation_volume=31;citation_year=2013;citation_pmid=23222703;citation_doi=10.1038/nbt.2450" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=J. M. Turner;citation_title=Meiotic sex chromosome inactivation.;citation_volume=134;citation_year=2007;citation_pmid=17329371;citation_doi=10.1242/dev.000018" />
<meta name="citation_reference" content="citation_journal_title=Advances in experimental medicine and biology;citation_journal_abbrev=Adv Exp Med Biol;citation_author=P. M. Van Wynsberghe;citation_author=E. M. Maine;citation_title=Epigenetic control of germline development.;citation_pages=373-403;citation_volume=757;citation_year=2013;citation_pmid=22872484;citation_doi=10.1007/978-1-4614-4015-4_13" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Vielle;citation_author=J. Lang;citation_author=Y. Dong;citation_author=S. Ercan;citation_author=C. Kotwaliwale;citation_title=H4K20me1 contributes to downregulation of X-linked genes for C. elegans dosage compensation.;citation_pages=e1002933-e1002933;citation_volume=8;citation_year=2012;citation_issue=9;citation_pmid=23028348;citation_doi=10.1371/journal.pgen.1002933" />
<meta name="citation_reference" content="citation_journal_title=Advances in experimental medicine and biology;citation_journal_abbrev=Adv Exp Med Biol;citation_author=J. T. Wang;citation_author=G. Seydoux;citation_title=Germ cell specification.;citation_pages=17-39;citation_volume=757;citation_year=2013;citation_pmid=22872473;citation_doi=10.1007/978-1-4614-4015-4_2" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=X. Wang;citation_author=Y. Zhao;citation_author=K. Wong;citation_author=P. Ehlers;citation_author=Y. Kohara;citation_title=Identification of genes expressed in the hermaphrodite germ line of C. elegans using SAGE.;citation_pages=213-213;citation_volume=10;citation_year=2009;citation_pmid=19426519;citation_doi=10.1186/1471-2164-10-213" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=C. J. Wedeles;citation_author=M. Z. Wu;citation_author=J. M. Claycomb;citation_title=Protection of germline gene expression by the C. elegans Argonaute CSR-1.;citation_pages=664-671;citation_volume=27;citation_year=2013;citation_pmid=24360783;citation_doi=10.1016/j.devcel.2013.11.016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. T. West;citation_author=Q. Li;citation_author=L. Ji;citation_author=S. R. Eichten;citation_author=J. Song;citation_title=Genomic distribution of H3K9me2 and DNA methylation in a maize genome.;citation_pages=e105267;citation_volume=9;citation_year=2014;citation_pmid=25122127;citation_doi=10.1371/journal.pone.0105267" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. C. Yasuhara;citation_author=B. T. Wakimoto;citation_title=Molecular landscape of modified histones in Drosophila heterochromatic genes and euchromatin-heterochromatin transition zones.;citation_pages=e16-e16;citation_volume=4;citation_year=2008;citation_issue=1;citation_pmid=18208336;citation_doi=10.1371/journal.pgen.0040016" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=Y. Zhang;citation_author=T. Liu;citation_author=C. A. Meyer;citation_author=J. Eeckhoute;citation_author=D. S. Johnson;citation_title=Model-based analysis of ChIP-Seq (MACS).;citation_pages=R137-R137;citation_volume=9;citation_year=2008;citation_issue=9;citation_pmid=18798982;citation_doi=10.1186/gb-2008-9-9-r137" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Enrichment of H3K9me2 on Unsynapsed Chromatin in Caenorhabditis elegans Does Not Target de Novo Sites" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
<meta name="twitter:description" content="Many organisms alter the chromatin state of unsynapsed chromosomes during meiotic prophase, a phenomenon hypothesized to function in maintaining germline integrity. In Caenorhabditis elegans , histone H3 lysine 9 dimethylation (H3K9me2) is detected by immunolabeling as enriched on unsynapsed meiotic chromosomes. Loss of the SET domain protein, [MET-2][1], greatly reduces H3K9me2 abundance and results in germline mortality. Here, we used [him-8][2] mutations to disable X chromosome synapsis and performed a combination of molecular assays to map the sites of H3K9me2 accumulation, evaluate H3K9me2 abundance in germline vs. whole animals, and evaluate the impact of H3K9me2 loss on the germline transcriptome. Our data indicate that H3K9me2 is elevated broadly across the X chromosome and at defined X chromosomal sites in [him-8][2] adults compared with controls. H3K9me2 levels are also elevated to a lesser degree at sites on synapsed chromosomes in [him-8][2] adults compared with controls. These results suggest that [MET-2][1] activity is elevated in [him-8][2] mutants generally as well as targeted preferentially to the unsynapsed X. Abundance of H3K9me2 and other histone H3 modifications is low in germline chromatin compared with whole animals, which may facilitate genome reprogramming during gametogenesis. Loss of H3K9me2 has a subtle impact on the [him-8][2] germline transcriptome, suggesting H3K9me2 may not be a major regulator of developmental gene expression in C. elegans . We hypothesize H3K9me2 may have a structural function critical for germline immortality, and a greater abundance of these marks may be required when a chromosome does not synapse.

 [1]: http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene" />
<meta name="og-title" property="og:title" content="Enrichment of H3K9me2 on Unsynapsed Chromatin in Caenorhabditis elegans Does Not Target de Novo Sites" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/9/1865" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Many organisms alter the chromatin state of unsynapsed chromosomes during meiotic prophase, a phenomenon hypothesized to function in maintaining germline integrity. In Caenorhabditis elegans , histone H3 lysine 9 dimethylation (H3K9me2) is detected by immunolabeling as enriched on unsynapsed meiotic chromosomes. Loss of the SET domain protein, [MET-2][1], greatly reduces H3K9me2 abundance and results in germline mortality. Here, we used [him-8][2] mutations to disable X chromosome synapsis and performed a combination of molecular assays to map the sites of H3K9me2 accumulation, evaluate H3K9me2 abundance in germline vs. whole animals, and evaluate the impact of H3K9me2 loss on the germline transcriptome. Our data indicate that H3K9me2 is elevated broadly across the X chromosome and at defined X chromosomal sites in [him-8][2] adults compared with controls. H3K9me2 levels are also elevated to a lesser degree at sites on synapsed chromosomes in [him-8][2] adults compared with controls. These results suggest that [MET-2][1] activity is elevated in [him-8][2] mutants generally as well as targeted preferentially to the unsynapsed X. Abundance of H3K9me2 and other histone H3 modifications is low in germline chromatin compared with whole animals, which may facilitate genome reprogramming during gametogenesis. Loss of H3K9me2 has a subtle impact on the [him-8][2] germline transcriptome, suggesting H3K9me2 may not be a major regulator of developmental gene expression in C. elegans . We hypothesize H3K9me2 may have a structural function critical for germline immortality, and a greater abundance of these marks may be required when a chromosome does not synapse.

 [1]: http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
    <title>Enrichment of H3K9me2 on Unsynapsed Chromatin in Caenorhabditis elegans Does Not Target de Novo Sites | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8zh-usUceTkpsvD2qWl_xgym6Ce5n4GKR4syafGTY6Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460026","apath":"\/ggg\/5\/9\/1865.atom","pisa":"ggg;5\/9\/1865","ac":{"\/ggg\/5\/9\/1865.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/9\/1865.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/9\/1865"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/9\/1865"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/9\/1865.abstract":true,"\/content\/5\/9\/1865":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022No3Frx\\\\\\\/HV9ng0GWuFbSG7nzi7JFyCrC4xrolDmHCIbTskSZBu8JsYNIUE9G2xty0C+wX65t\\\\\\\/l1TxZB9QqRfAdvvG1IU3RGLfi4E0x3luJe6tPE2MNAaupsphaE9xpkdlC1ak41uV5IOeoSwhrAQ1zVtOzAT9cmK5jeq4wRJmOAgjVEXoYyFij6IbEFmvmVMk53ldRkd8CIsDHnthR11ezfNNKhis6\\\\\\\/kcAShSWWKsgzLxJSWcZYo5Q+5KD0pq3\\\\\\\/jIQn7QkZwR0FY+T2g3v7xTXXF\\\\\\\/Sw+R9tSEuO\\\\\\\/ki3C+mrdqKke4buc4JVXa\\\\\\\/NkqsaXnv1XeUF4nb5umdsmV7EKgaZ1XkDhjVmbq+p0kC6nB14+fMUD3nOsxpkV3DZ\\\\\\\/JoqzYcXFPbr2wL1abFWPSXvpva\\\\\\\/pYyXBCkPNNukQVa\\\\\\\/1nynR8h7bdFfBBAT60p23YJUF6VfUsl2sDL\\\\\\\/LlEAlASXcfaUhNw7+i0TdukaL4e8X0UKflW0emrRl1bdaN0wkvoXdNXG6eLVJBKtkuRZ4ccum56mnRn5wWXilmPzknBLU9Ot0NJCdEJKr7ucaycFuftpg66dlvUWWIywl54u+FqCHRKwTb2jOY1odT3pU30svYfwQNtRtYd7mfR1U6v3QZPV\\\\\\\/bTfm7L98B1ci1UFRjfb39p4vRWUfSyB9VlMR6OGLG8NzcxwfSE+wmeXD0zrLAynGBzO6E8\\\\\\\/Pi797x+v1Zwx5E\\\\\\\/OaC\\\\\\\/HVl1BC6fSKK76RvZFZ7ouG4o5KjtrNOFOqnBAy6lBQwPK2oqEY7\\\\\\\/n6EurBIl8Hs3vGKK+JBIYhANR2I9AMuPSpQS8x\\\\\\\/8l94k3NQJqP+frDpTG2E1jPfJtG5ow7muhE2+Bde3W0dlJzp1kKZSx5DBOsRHnlQaorYEXJC5E2lByb\\\\\\\/Y++5Ilndd7\\\\\\\/NwSf5L40vZJuKAePEl8UuWi6phtvn9c7Ou\\\\\\\/8Iup8iwReG8ggx6HPpa704UD1a5+tErgNgTxXonbABi9GApAxRGBoJnyVmMAIJVt26hWnItUs7KP4ogBMrGOZ83jdYCmecIhAAWRLIYvM6u45YupiHhZMOBzqq+MxG64aRqLdB5nzXUolQj1KiISjCl07QYUDnYZMAPh50i0cIyDQFUQ4s1vAfhVSnsyOIUKxcNdH94lAsP7LljmLLwOR\\\\\\\/yGau5KdIoeA4NrKMZ5sVNr8SWGhASdSi\\\\\\\/\\\\\\\/5AZmHPGHcy04PCvngu+SwF+oynXVhX9748PdJc+C8l0tkGvUbJN5QJ9HNtItX8o340kMflbbcYJXy+XjZpVyrg\\\u0022,\\\u0022iv\\\u0022:\\\u0022FU1ZdKFEn0xe1J+CmXJi7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b60642cb625542dbc97b9a5a0dd34850f06bba159f9fa49a9abedab6fdb64b5f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022f1GeUBUARLhtT\\\\\\\/KmDWeV0e4z3fjM2GQWJAUxQxhEr6OnFAZL1ScJltzu827lULHkEHGmsIXRniuRkjqOB+CE2y3+ey3egKirPKX0cHSiiPx3Tk\\\\\\\/I3qRoq+JuklpcBZCp2JDNUSR3IKx9dZR3gHbm4lHqgqJugxsgXYpShFRHKF4e+b1MxKTDInift9UZzYxwcfOFn4CA539Ub0WjKaOYbc4Pm58UwrhP0MvN8d7LZUACsiB\\\\\\\/CGVcmltqUosJW688mIjB4EHvzFmQ17zooJIJYeT02Rb\\\\\\\/5DU58q994XFc0yA1erYXdRlCfAjyDmXetX2h1Gty9FuPXgVEsKohxy\\\\\\\/9QP8ylMY6Si2m3tAzmwfVXlxnlvDqvVZjGXtz82cMVwQTAFaS0mcVA8cbIo74yGDxBXGo16ftvmQi539wU3Fv\\\\\\\/4FukaZszUMimftKVeHXKxuVmJfp2Qiy7keFNOBbGpJvjV8ioWtas07UvViIBiV72YtVgKJfJFQaub\\\\\\\/ID1mDSco5MQV7mTsW92agNTVBpuk04v2YIoJAlzovp7Pn6iejpR9awlg91ukixMQX76UCjWTbsXjBaOSYr0MIXEm9iEnSOl\\\\\\\/WJ7yICWXqD9KOqRhaavzbw5h7mvgNZBkH8b+y9rNheWVNMciTAT8PhWanHoVgCf94DlDtwYwjNDB2V8j+sq9dm6IiMTrSSqAvkP\\\\\\\/S512FZIi\\\\\\\/CRw7SSpf83Q1zeHUvXgUPNUR+mt9BMqc06EeIYXVU5tucJJeh2BH1S8+2xMBQqxy37wC9wHjEDfucdOPLgTWXXP+VKMC1cJnBF7xOD2E8pypuQ7eqCu+HlFYSrklgxJ5ztS5rRK6Lh6Z67orcYUtjXaajZ\\\\\\\/+JkuVi9mditudWu5PexSmb2weEL5N8taqcqqXFT6ShY624V2apMO+qUqAZzWgddBjMnQu0PDTeCRp\\\\\\\/B\\\\\\\/6n\\\\\\\/ryyQ1nwgM3TE5CUnvGpEcPe9h3zh1GDd0gQ+Zs6pKEpbGxFobrsIg0+vCGU\\\\\\\/4oKOPx\\\\\\\/mdM+8vtbsIOh+2KiGDG2JzajHSKotowiThNDYLCCv1z+oiaBcJvQwpEy6152qdSzNr+aT94LbRGqEEQ3J57U5OGm\\\\\\\/qN2pgdbiLepe41Z5lYHhKxOXbmOqCnwlhPe3A2yUU6MyTYprsBo2rDyFEvSMGDIaVzZ6Ym4\\\\\\\/NpuqsEfcFDukBq8QXmR36oCdcv2kMh7f846uurMcAf6u+rvsZxftmuklEUYu7b99LvRAFWAB2lZ\\\\\\\/idACstCYMSQzlqDK00G9uS9i\\\\\\\/QFw+Fp33TrB9EbNsy80PU1yd5GrFOJCeW7bOFEX8TBl1qxuj9aMwRccOYfZDkH31+gcrg4bddth1heweri6YqZleYW0XkuJNg7DpxDDXldWl++4K4f9hqbL9KPRFYzV12HuVCEvQj7GLhRbfK9xUJRK\\\\\\\/fOgIPADlnO8P1qQpuNBJ4GYn37Qe3VSV+PvpYsLDIWrXRQ5EVGcCBqWlnxv7bk4ll5uJ7OP2qDJL9YgSSpRGqrKdGcdAVO0VgB2kHdRx0rjFFLs3DDEgMFDTP5wo4rbwFu94br+6bmhwHyy2Dow\\\\\\\/cs9HJNZuGs0Maj10cu1HpaPezHAuVK18IqyFi0hNjZHjRdCHkxyYY\\\\\\\/7y7Kkw8KrjvSgMXzZbBQMS2\\\\\\\/k3XdyJVvYmXByFXxfkiWjpggWm\\\\\\\/uUqRD7FnzWjbNOjPDUWGXxnDbM78\\\\\\\/fv6+DHGpHR0pU+wqX\\\\\\\/oIsWdL\\\\\\\/sgZ55K4MUuf\\\\\\\/f6oaSosE4XJQhxLGPm2LspHCbqgZhCAkXDz860pDq+FwtzuNTCsBUur2UG03G9i\\\\\\\/pvflXD3N9RxwaGOD3CFpiVO2wJ1EkAWGdvuFTNKBlhvTAtMc\\\\\\\/Uw5IL45G0wQRfCGC0NAPtk1PJYkPNxo5ddCqIvrqSi+3gYHPAlf\\\\\\\/a2WrIyZ+UPx6t\\\\\\\/eV3SXr7TZ85Vdq8BtiUWuNa5hl8tJUwu+1+gk3O6lZsfMuV63BxeEQH+LWatnh+tPpoqI4MtbYdEUV\\\\\\\/tE\\\\\\\/kBfLJegLP9sPB8RnX8DRcTKNAnu5uT3xoUBpKCqSjhXlLXJl+oxyvL+c0zPGzT1LNa4NTFNwcyiPExtK+aUF9NdhqUpJOJQu\\\\\\\/6x6Io1Sp4ANsHDyw8fnahM2QafU5S45RmmMvj33WN9DvndYl6W+mQ8YmCx7PPsXdK3iqChF\\\\\\\/ukHlgFmxLmvHjEH8nZFJYnYPcobkv7M4HL1GZeLdA4tSeuzLadY5mwJBeKnPSAmlQSPb+TO7fwOd+3EI\\\\\\\/egYfSFYn0DineCsCnQxRBxwCqTaqgzdVR36N8ObwY6BGk6M9Hw7vILkE8pKDCxXdQkBqCkP3Ug+qmWUj0bncpAzhPMhnzpCqjbesC1Dd\\\\\\\/iLVUOR6kOaZiAar9pea89jlXaxID8gMxQX052hHM9reCflWC+ZV5+bJEtH6yXrkVdx3O8XY6NWrTphofL6FOlc+1qLpATbrVFQKsLzb9dKyFNGW+pUH9mbyOuAgYmr0goBePhI8YEMRiVWRpWJpgFdmaaxcr+xjIFuR3iJ\\\\\\\/PHKOsDzVUgvgvf+DxiVynVfHmUjrfT4dednwyja9xf78EgsWuFrhSJ8ekOn37N3QpnSsle3XS\\\\\\\/DaccmX4iAVIi\\\\\\\/ZpTuUmRhaLDihZoEDYVYAc+ztfSLriZds1tbuXSG4Bv8CGinJ7J7XLtTix73pWXNKTuOjaayy6\\\\\\\/yHs2IEl6nM8FnfzmMBabv6UaLeVRswLI8yPwuP1r14DY9PjSJj+OZ7ME4jnodjp3xspRUNkqR+3o9YoeI5qh\\\\\\\/6gPAlhUDiH5q1JMqEqZDJqv76zPiveJsfgBswhYsjzHAsDRFrFuyCanZLb5EJYHsIB+bClje25eaoBqF3AwRgdlZSJ9DbEAKdqEg5KBKLWr+hmmL8A6LxavUCC2Q7DpB6Hkwh3JNC7yP5k9WEcI8lAyWSnBvfD+G3wl3qOgiUo7d3IzMWS0TsiopY8MocdV4JPZotnmZtRQbxckeRJtg9MWYPRe5O2VbPgKqgSMT0GvAD03PdzYo1b\\\\\\\/owTYC66\\\\\\\/O8xzH9cYlEy+k3nA20+5kEzw3HMZjeSLM9SYp\\\\\\\/1DQ7SEcYcZknBqbxdQbqbDwUn3IEVe4CIzc5sag49CxplktNXBO2FQIM8NcYYv48asXWE+eyujM7CMk87u+eZjGv9rl3F555LpU\\\\\\\/icMLvycrHtbgfz9ubvM+ymqL8An1yz8\\\\\\\/LM2MM302zRdskUfm\\\\\\\/fBBL2ce5ubEcHxpyaEYJsCrAr3XILoUuuC84GZuZsxYktaTcLjqk11atT0D8QdonlTgzCSa1J69pPGZZFwifw6VGQU90iVLimfzzsa9DzmwkL4U18TEUH2CgYsXuMln7L\\\\\\\/9EMKKiDhGMHt80T02LsNJKPQDfIS\\\\\\\/LH1m67G59g4M4VmBkeHHIFt4Khn8cnTMzOo7LeVV5nvkRcYBvRc4zPSGB2WyvEQzJf6ooHR0Dnt1xuzfuZ4jC5qDPAPvlLWRXt7I4kgdcC06cYZwkGk+Xxn4eY7gIwaMaKbUOKsJwyWJRN0s7OcLNmV8r9gphYpT7vexW\\\\\\\/MpLMz7ULPI+lYzk+PnEBKL+5whY7ejN8g\\\\\\\/fVlUkLtH0GEd80jBEmMODGD0DmaOCee1P6KWCTDFdojFb8cXd6G0TbGWgFgLQJ+tSQcBbEAg8JY76JxoXOz3brRlNIUQpvoC+Rh6RBvfF7AdbH\\\\\\\/uTNIYkFs+488CECOeOQQbjoivRzEPOFbC8Hleaw0gQ5ML8ns+iIQpDxTst+SeuyMnFwLP9xXXxlc1+v7wwB8hj6DseDMHPsYXdKEp\\\\\\\/dAB6Ruh6YB5bBN83M4By+dAGTzQj+yk2dsKcXbpX6YNWDehD0ZuEe7y15jz9udA93kKlXNC7E9++D56HZ7QNpz9hpXkIS0LSnRWboYrEGALKLsoHkt5WC+sHd5RZob71LxxkcRIFiLZglDcaF6ekPMt9kRQmiqfYHn5zVFgSsq2f\\\\\\\/h64H6CaY7yAsxoBwb0d9zAIlaHk60N1x28gCLVLyjZ9TRJCTfGJH5fxOk42di+OlJQneuTVAKu1lzfZsRCGg0YOMczZoh\\\\\\\/PnWpGTrxq\\\\\\\/OYJP5nWNw4VqD9iz4xx0nY6jNf4j7aF0GMvQhsmMz+UobtCThvu9iEAm+9ZHRTbkTvBLEMzPtzVPL97HlRYmvJQzZyRPy2pWJlQ29MzaLLYj\\\\\\\/7Q2ccsfVQOspMHug5S39hCYAhy8EGJAYrGbyTM8esNkgQZh\\\\\\\/Vi9TIDx6rofaWJXGho+VthaXT9vBjh+NvB3omODG2iyQFtA42psYX+uXtyJQJ0239ck+O4djN\\\\\\\/ZMH19rQ434bkdljkzdNTiybcB2OTflT2tjuJzws8PVrTVD96rrS1CpMpkT95JOO09lz24FeBszgYfa0VseOufPBQzgXq4t51syEiIVYzodmjxnfTjmfqWCj8IPPft4mr7dKbcnhCgLMcOm2F8czWXB6oOPoChwQlUeEX94ZtiTjppMr+PLHKdZ3xoDNO3s0Aqvc6b7NmaIkXpUwBzzL7P0OJ\\\\\\\/5PPj2zNidaNqFppAnwCR5y\\\\\\\/0GsjVurofTBg5n2UzMOc4wvlDv+GE5dRbvaeFZlnF3VjoWLU+GlRMY58jERlwJOhRHyguaDsm4I85Gl+QSgDknbqg1zjJ9\\\\\\\/VsFYWEMxIuI6VL2XsWd45K5OiBl18i78GIMAEKTaIFpH4dFjDEUpvMs+upWZlzTUf8lZaJX6Vd7\\\\\\\/jMjHqblSt+G6Vf0BJgwYVlSv+3EXTcD+yj8NH+tETLdYKWPFoOoA29e96trv\\\\\\\/\\\\\\\/XWyNxAzSryKMirVPU2g8aMuUBhTa+6M3dOUO+NElZpm0miH1dBCrEyNn7l167hPgotFCD4LZMR9PhVDqeQB4dN4q4FkeGdO0AgW31gjepfmxI642cghih1eozcEnYpgJejjwRhUe8KCz5aElw+3YEzwR3F8LfVAKBImMT1fKeUbc8JSZhjO1IN9IImb0WPQ+5RkBqvZXQM6ypQSrFud1V8PSZv8UcIL7HRtBVXXKhp7DWhEdLZeg6fYZoNOxLmbALbZbQfcVRqhYA+8qfjJ5\\\\\\\/TPZdk8KWwLYxsZfk9gvQFfSPMKnMATjdUF1wCpJysqxtX\\\\\\\/YJO\\\\\\\/+o8cyoLf807kj24XGW3ADShGLWEfFK4hrTA7S7h4etSgnhg6UAYRyuHVIaQLrXPHA5VDaO733egDwGbMwY\\\\\\\/d8oisSc\\\\\\\/ajBZAnniHDIukBPxspyY+uCY287G9E2p8aRKaDfPAmrv52CgSXAwFytt4\\\\\\\/UeSiBq7gJA0MPamOHMXyqvXIFUwZmwqdBSti5PskKESlnyYEdEmrGMELf\\\\\\\/wJgYSNy\\\\\\\/1sw8D+qEmb15pPFpimfLAWkNyd7YfSqMmLK8Vh2Bih\\\\\\\/ZNx9eju7NdeREX2GwbM+KCdILOcz9gcw+Qw8I1V0UkWfg+4u+BRg4ifZhArztNxQd0D6+L8f5Na9XBxB5N+XWJjNTqhsTiuGPZr5xhtZpME3ehgJ7D\\\\\\\/sQrUCcez6TFasTTGDrIuq8rgVHr4WcVI3X+MMXabg6a5ypeYdonIyTByJ8dbpYrhudRSaTLzuM8AeqVnB39NyKZOsul9SiAEGI8\\\\\\\/eTxDmUYa4dev9ZeFe1KJv+\\\\\\\/kh6\\\\\\\/9EnCkKBaYCB\\\\\\\/cTwq6hF+a9fNH2VwWCJef0BAXJRoAnsyaoEctjghNBC3p7o2WkQRqo47VUnJGE3hn3E9xJheNGj4WKJzJn3SkLDIh47nq7kbV6Z7eNfGibeV\\\\\\\/2Xlyg1ZuUMsp5jfG2PWxo5T\\\\\\\/pK2TA3rYvLuTzjU+S5UgxQQclvFQGZGQBZWfnY6HjohTakM4bjLR3dxwqKFi5usiI0ulW25jB1EBRBqb4JH6XwedgtXvFVSV3q3SLvwN\\\\\\\/+RNDKpxsf1GXa9GbJHYlipMkoyEgm8QO4g7FnGcdkdcQ0bF62UGaD5pH26AtVLkMbOe6z1SUslx8Hzj8hKIVarPwgZd8QrAIBIDYF0xRdk847X1ikxTjCBdpk4seSMSOMGTI9PaEXYrXmE5LOSsXLZDLSDUpzrFP2HkBM3UnpKDO6wjS9XzERekITcyVVNQfUZBrJm6v3HJeOu\\\\\\\/csPbCCNpdQEGpkYHXsGQkCZgogOwGyrIa4Zqt\\\\\\\/fi8aU9Iybg8Xw2C86ILnQQNje9HvfD9Rklw1dbCxzzdBMV8nui4aqlyjVFG5H8zPC5ABwHjRHNZBjR9pNDnGLa9pdi3yKkKUmljPb19nTx9Qw969dCOSGdGhe2GJhlXxme884Ss1fV5scpJKLUEKzmpSWhvGRQ2xc7QxsCW1R12tHUFcwn5UwAL\\\\\\\/tf\\\\\\\/WeFP5fPsRQ6BCjxZKwEQWsIxo2dedp4gR0y8xhMkOiDZ78oTFmUbxfrTMg\\\\\\\/j8HJZcv+M29NSSgufvBb1XjcbylPibOjDYH2y\\\\\\\/j2rnJdxc7oPUp3fBOjo34rUso4apufsmcL6jhNjo1f\\\\\\\/hmSZReK8xdG2t4p21uEFkSpS+Gts1l0vNFtDKblv2FEdvmH\\\\\\\/Yix6IXuVyiylyLB2pns2sSLtMxRAoOfgWznjyNf2j4bCdByQrANxn8er2zCB8mBP6SLltOGQffqyom4KBe\\\\\\\/VI7cgeQ8hAV11jRuLtbUerrDxiDlrHorswNIjKN72+v4eilFN05m6z28IO0Qwk3FS8fs1ClkbebTeqVTquaweVqKoshYJW4I8CGfDhKV0baaN1+HBs0K7i47BKteEB10yi4DZeTlM4WEDdeNmkFXJkTcovbKft78cVYZDFsvmisLXa+6WqjhzFnMwjAiFfBd6qdbURBz2t6R0EO2rZD\\\\\\\/b2ORyywTEaS9eDcpEjeHKzVU+sWPaRr394mToqVHWvrbXDN4pa3Ny0slwtiGcx+qWOSWBsNEc6fcy2cEp\\\\\\\/wWFT9t9TNuWPW0YFvrdm7VK6AwFRV78B+qsxMyopu0+7nGTT\\\\\\\/NtawKshalWQmJB4enc7pkbw55NLLDxKb+Raa61aLOyKfJTAJRoDDYxnsDmgeyOyiyxw2B16sKEKcuzYbsnPBuA3HALGjUzHduRjm\\\\\\\/a4Bo7pTNit3OC1Y5inw3dRwn9x6Ud4h1yRc42o11zyJ5WIhHZ\\\\\\\/RiFTI41d2REvTLOmyMgehSqb3fVu51b80m6vlIcc4lGwYDvK3gyX2SGLj+Ka6JHNNPwN1Sosh8oDh8nMxEJk8prscAS9QjLqfLPRnzPe+x+fzbzDKkxIIuNx\\\\\\\/VRBgFXhwbgBbjc0YY8E7eTZyHv5XMrkgDHarxCFXArhjBva+T+P+hwrKfLH64AKVBBV440cZQcS8vXk1ardcZhKpWcVH2VJ1q7fCKtAb3MyJ7Q3nfBuZJuBtYp3tOXB9EN4ahE\\\\\\\/qzEnYU5o14GflUpA\\\\\\\/X+9UlYAPvd7b5Q2pyWkXB0ARnZ+ZWFMXJDI7S\\\\\\\/zhWDQZnMLAv+VfT22FL9uMJidBR27lgauN\\\\\\\/cHiNDHF5ldNftpZ6Q+vXzQtTIhF\\\\\\\/L9eS93hBjyNaL94TuCl99Sh\\\\\\\/r\\\\\\\/hWOpAWOVF6hcQlFlO\\\\\\\/O2uhFXtrpS6KdzZKZ\\\\\\\/i0rvoHUOKL8R5oNnlnlTR+fKaDKo86O0mP3bmB1O36A9dsrCWdyEF5UFW9dC0d\\\\\\\/KsxUr2fijXqft9CFNtttqqhuQifI8VeS3uCevSoDILvvFpn5w5wty3nlLZ30+MNJnLa+QJq41RqeL5xW1chym4shlazmy8bziOvyn4o5+u9+7OfgAnTulspYIf0YFbKAM7wCifbYVPcPZKrGml04Kd1VSHCcvN\\\\\\\/100KtJKA9u0RtuYXgIvUxhVe20nK53CdArfoSF5VuAgZU+Tjj2Ln+4leymm+vFiDGYnwlaYswcsyB7sOn7UX4jj\\\\\\\/4L7upe9mOeopZtq0yAdQ8xfGcFKCXMuJqQ\\\\\\\/ZZxt8w1XU\\\\\\\/o5sskk4TVBwQCEo9kEirtqnCGicp2iptPfucR9F3mqq0V9sgVATl+FL\\\\\\\/kDfQB78tHaB9nMA8lODQNzU1pl7rYDmwvlIvBBYk07R9+a2bbYt8ldOPVOpPw5jEeR5Cwq8EeMVgBjbR2avcrQUYnfJ24bsJj6XM2dzfLzc2DQwDAc3J7OJFK\\\\\\\/ly0Fwp79ofsp6vqCQLYLpyCVv14Npqate6zJK2VEG43jfeYPwlQmK\\\\\\\/rVjifMwG9BFGP3AAeCDgv0E5y+o5TsvD6uNonHU0FDaBH5\\\\\\\/N860C3odxXIznGiEp62uq0v8gXi8jELfqZ20KhdZU5gyoMnlDFl9XzMIxsP\\\\\\\/81HGFzJeEL4zu+sb2QJKYrmSewBbbyH9wZ3Jnh3CGT+WAOosi42N+1OsPHaVwaPaY+UMHZFk9HGbCAlBTCez58yzIFX+8ZSjCFuFk86dYRCh0zlGM07cENdNp9lT0314TYIrp4CZVR8z8H9aSI1SA6jHh2ESBz8nIhM1FEgQ4ouuSzrIAuWk86bLwcQhSNliXM4xwkl\\\\\\\/ADB9syG\\\\\\\/gTnbfL3VzheQdYSxtrL+AWfafyKPtjm5USpXJtojUURNoEsJFgh++7wNMLgXqiJasL49C\\\\\\\/q8+AqS2ZLIYic96\\\\\\\/OxYdy7a4ZtkAAy2yQ2hZ2KF+LzwlFVFEaad7VcOjt1A+5GepAoe3jOiMJmPqfqF9Q1FN0lh11gg\\\\\\\/ii0vqE7XGkH5EAZQqDrCr7mzz6rCXdaHn9NVdhoM3BnkbfNS7T4MAowQwvX6YaAd667RdKTQpSVSDjOJN0eLKPYOHkawVKYrZ8essexwcG8nYXodxPBe7HJy9S02tLRajCl77UkvwXsu4iq+W2CkTiY1NIDt\\\\\\\/DsCiMk41kK\\\\\\\/ISPjC4usODNQ9eR8FiOsxfJJEFGOYbI31iYuWEzdhZDkd\\\\\\\/MVshbcZbMEqvHWNqGm46eck4ZZs8IF76wSalsDItyRAIzyNxQ+xKx0f1kovoAGfr8oY8pmv6Qhd3Y0TMwrQyRO0YMAWlSMZxv1uxh21KBNlbLXoHEXzlW8IbKCr\\\\\\\/zeG\\\\\\\/rDUDiVgYPlQYVT90jsrF5QFSNRwPiZlQgbaeh+7aCPDhCJ6QoB4h9lFoZjN4KdQ+hhxRIxrLcaUg6GCNFTrOeOAQBFHuk9QnX6+MQ5f9EQgXTtaU\\\\\\\/xDgN0hzp9oHaLyZdiCn8ea2nxTvj6gg4b0JQP73Abqpmp0QWnWvb1hOQW+dvgsK+ypAIQAAt65qGzq5VMxxa2eT\\\\\\\/MVXpGXir6vQ++TWwda9fsoGcmQOnZpR1w1dxaaVcI8sCt0ggMR+oWNRM08wZNkhK6uqPuPD40aYjxj+oITOuOlJGc0\\\\\\\/Koi7piUtl55jZogxSSvDzrsMoi8ZoTArZ07L2G3Egx9AoaaBu7+G2zXtY8gkr8yYzCMkBefonvZ5j2Qfnabx5q8jYzmyYQeJSBXlDbi6HuI8VFyKbxQVWgLUkNfaD70yCvHxtBuCYP\\\\\\\/hsfucs11EklGCmuXW4D4F0px89TRR6GZJDWqYrcvHhegCM5rJ3rxEV0vcrrXrD4ApwP0+2aT1PlqVSqG\\\\\\\/udK8WILWple\\\\\\\/SsroY67oBo6jgJgWC8CX6MrM3hcw\\\\\\\/\\\\\\\/CoXstTq5P6afLy\\\\\\\/SCGyodEA67v4CZLlaafE+8e9iMPOc1zMdxVIrH0YiL8B1\\\\\\\/rAz00b72LWJ4IGEvPFDmmj\\\\\\\/yZ3p1a885ay\\\\\\\/T+5e4AyqXCRhlzLiGlAD0nqVYHe94Q5pZimrf5kLYicbGDRFXkEt0omeGpwXb4dhnOYWhh5D5X+N3jl+theXola16WAgJbwlgClfPYfprHCsppvTRYJaTqtgNCqSccXeJQXorAeC9zDNalqDcIFaKWZtDWlYeqspKTX3q6p1N29GdNJp19JqIpTdii8tqFBEfzcxXu8MZJZ5p0RogQaGoVLM1ZF0zNLFDhZfQwsjP+BYYRoSgtQZIBCuTtfzqMX2v1mgCiJeN9+NJw4vLmcX\\\\\\\/a4qC7eByr+llJuB3zDRAx9I+kZCExk0ww\\\\\\\/Yt5t9MQDMTzUgGD2hUlZaB8M9j6sFjW8ZhLxpuXT0jjE2OmavU3xNWq30EeKQhb5qYO6RBplVcjY6Ly\\\\\\\/diNEf4JsMyxHxUkAZJUnqbU1it0t5HWd6Tbw\\\\\\\/WQHv18B8Ve28+moY2Py2KMRcvkPnNiY3e1QgnKUlOm\\\\\\\/ZChOjcK0oAujjcxNp93o0ibJT5YlHFfLhfB+4rpsx3H7MCCYHu\\\\\\\/fChWpLXIH4cYG7ZASwoiGld0baUHj+TAEkShZGtkIXhYK0l7ve1V6MJIZWOLh0mjR2SfiR2le3DQimKkyrg0FxcPgmLkHVgfLMP9UKKvE2lghSJSfKvAP3LmOpYQG05BWetW7OD4AsNbSENtxEymEJHarIoPGifUsEP\\\\\\\/ng6IEfGD5bi2OyzQVK7ALFEnrgUY5PpDMezj25nH203z0th8i8dv5Unob189KPFayJL2jsSOfhs\\\\\\\/\\\\\\\/q9uga1kUuDjBTtKk0IXuSly+EU7FizIJC8dlS5CODXY5+pm1k6f+aYmzO7p+HfOkSA0UyYoxnZGqKslkK7syzQu4aLE\\\\\\\/42r0XhhcPQ7LbecX50sCj7bdFdKzaA2qyW+j18+4G7qF9NCObSr6NKKr+k2VWCBacdh7YUQtwz1c7i+ETmd9xHHuFsIiIphscYOAEk8XcI6e4jIhNALrtofmm9xZ1NkJ6zTjm90OQ\\\\\\\/037VdYtD2NqdvGHoCQl\\\\\\\/RW\\\\\\\/i0nMlTlTQuIB2wAzTlgMyT4ZzKatA4R5P+PlUzQNF7rO0h868YcUMCkJTNMf228HllQWc39HUzJxPrn6ZGj+H\\\\\\\/GXgkJvz6b8hD\\\\\\\/23lFT8SjwjsYl6Z82QJLeRAq6TBMat5IOgRfX\\\\\\\/MAgbNfVuiR2u34sbv+DiXrhqkF2b0o54gZypoVSai0IJ4qk632lBT1DcTSgwt2vnHjcJVA5GhmP2GXtS93Wu+9nmN4va4z7guaLns\\\\\\\/Kw8+EIt7OhZAyotYIA5HRPq7TbIw\\\\\\\/OZjp0zb7ipsGC5hG\\\\\\\/ZrZ5I9pEIu7A54SJohpyhv5Mjgi+B5JzFnH5u0LdMKt05pv32U0VhaqggLPPmlomwK6NO80Da3C4DcqMxIXkTl6r2sVGdJLd+O2pXn3ZM6KcXFxDS8EdExG34gAKBH3HNntt\\\\\\\/Oa2Ep2KYL\\\\\\\/VAkqBOlA1yfnNs8sRc2jIvsfQpacKha0E8JMdjPOSxYLfZ8fzQOE3AO0AU1uP76gK2HYaHc7sx3qOilBF8WWsbRuN7BtEBYTWDxUgE+eLTwphsQITbjXyy0YSb9ohILu1j37S+Iy3BOVdJ4UhInOqzjY\\\\\\\/SZQWmrLkvow24B4GYgbsoj7HddBGRUk\\\\\\\/qA2nl2eyOgXoGD1tF0Wvsb72yQBxQfdLJAdCxVisqA\\\\\\\/fHSDyeEZnuT5FhVEkRFRkw0g+7k9Jm9hkZ3ByNSmIHL8q4DtAF8oJEpMflQ0PHmGoyUYsxDr+uUDvFDfwjkHv8HYvWtxH6oBxQ7nGenhm4fpQI2jyw+OW9y6Lng5R0QKHy+\\\\\\\/ssdkyc0Otw658izSpluNiLAGsOB6WGfOJMU6Wl3SOukHFmvypm7nGglw4Iz+ffrp\\\\\\\/58j2OSj9BkrzwPrWS9Ny2iD\\\\\\\/pjLizMNzoYMwQCc+ueA93ejuZxqJ\\\\\\\/oJu67LZ2gHIOaMlJGOktTTmh8Z+0KTVlTfed8zynvha+CFk\\\\\\\/cQLZsN0+BZ3c4G3xuT3HIgFOhpaSiNZqhYVVNwdX\\\\\\\/vZvxixx4ZomtWG1OBxsKVuRcdgbj2gvJXlFBV0LMX7+WWaFJLGMqp\\\\\\\/bJJDC3ccZ0\\\\\\\/t\\\\\\\/X7vlhzG66PR5IXp4XIs2HrQmsvolTIyXzJeB4koanCo3dBTbv7rimScPueqgGgSxausP\\\\\\\/X0uObaom8V5kXmDY8NigRpup7Hm9FqSSwwjN4NjQtKU7v1USgeDUe9H9lVUUC3XX5xFCujhoMMqN1i9IZZD6HwcxKPjViXUqNqhabqOE9Q1b1pb86iUiM\\\\\\\/Xuxu8mhwMUdg+Sur0K56jtlmKyPbebyyzD0lmlQuILW3pOcj2TELRBvxuVduR1BrsQzUdAEXgPJcr4L3TWAUTfm1Cg7UuJ5LIg\\\\\\\/vbf+cQ4d2B4OIVPXzNYB9nrvCVgcS5vAIlxOtbhlLA1IARp\\\\\\\/felzckBLoc911v8aTx+FEx84jW78\\\\\\\/Q+2DvsyjUjTO\\\\\\\/JQp8E109\\\\\\\/bp5aEgDHpb2En8ffKNDV+KSmGI1K2poqy4Iv81a6qV7KzDfFVpYm\\\\\\\/0M2hYaH\\\\\\\/xcYQfmk3KW0zbBfAkrkoqvofHIoR7C+bbnDHXiIAeEUgM4NfDXaSzUSdzSvwUFLUV0Aqro1gP7k3KylJ\\\\\\\/wyNfdpri1BR5hrwjUKyQ6WZ+en3CGmsESeM+TTpKt1EiL5K60Cbg21vQXdN17X2jKhmg5Gnlxd4DBaz8drLl3f5WDufka6mOJ11Xuf\\\\\\\/pRZIhIcilfvEPEw4mfnJXJ0DFablViCnqHzIpDPfyrIE8c2UBK0e9NdpEe0nldL9kwMVazdwL0Fd11\\\\\\\/dI0taRpDaKA5cnYDvUpxAgVDp1CG9kdvqXVOGs23GPSizVrDy2vjhZ0LDQeUONNI7ldAS2I4hP3B9+MOICWx1J8Rl2Lbnk1ZtwszCIS82ouKFjAXrhXloikUKsbiJe2V4N4x19CKKJEFdruAe4plpXUhjpWGAs+0AO6zVtUv8L37vICrYa\\\\\\\/73xN31zA66dXMyu5LUmEZX2Qylrc2oERrpaecv\\\\\\\/8SwJLB6gXwNzwDa4rfJoLcXJUkg8XS7LC6VfXqZAfzAPbLeiaNUEjjTNef++P7oqSLxsq82+Kk2RJkIO16a8iymZzeCbOYM2GKlJ8WlodELt4tZeLrR2gxockrRdczWpvGSq7Yl\\\\\\\/q9QKZk7vIsrqSOVTR9k8JnWHfHD9UNlkyWtl7mC9Zi5A7z3SHmO9sncLYKyeByHwHEJqlh\\\\\\\/Mgsh\\\\\\\/URgHjEXxpxZZzXlU0pAoyMPIWZDJbv066GQ4wJWBQ8BavE9C+zQZM9NsDRGrPoJm3I\\\\\\\/\\\\\\\/2BenZV9UTB1x+J3Re2fk\\\\\\\/zyZSw2MkpKAu7m+PmEqFM6FlEtwZ8zokIyTWnc1bblMmI7YGvyM3AOnZk31m8XrZcT3Yy9NFrjs3qQB\\\\\\\/f\\\\\\\/1M26UelAI1MIjKrw0OjWEXJjHmilLAfbf2gKOklhSOjjbfasr2zPIXWRo2NC9rVYoveJ5PQChZZU+lOyP3oY4MhQIkupTqDtgRJNDUUcchgzfQkVodpK111svVGY4sNDjxBUUW5qjT49XNFhB76L9XAJVKri0tY8Dyi11WWYegzgzPIMzD8byW0VinQT07xp8TvIpJJrBIo\\\\\\\/z95qU+AO2Uh1K2rpnwb+WKpnHD\\\\\\\/Y5ZpEfWqDrMtTI6yAGo7I4CckzQUYbVOd5efCOHhmvv\\\\\\\/RPgq7XhAGiav\\\\\\\/PiBJ9q2TBGwOXzKNqAWmvJAeLoYB2XqwsCf7f7rGg79oZQNBHjPoDbfoZT2rzTiPFWyUU38RajT92GVrWhQTui99z5diXu5AqxpRzJ\\\\\\\/QT03Q6wfhUegR9IloyvkXCQJC3E4TBVyR8YoOADzbsLBqyjueCsWGoO1dHGwzCFNAO426F\\\\\\\/hoe4BOd5hEOj5wZVOoEwcKBLAiL38CpSIaw5jSzLWjrsIB2z+VANg5pXmJ9vEvhwDE\\\\\\\/FCUvrfAraY\\\\\\\/RUMrnG4bU92A6\\\\\\\/1Kutr0GY21+doB\\\\\\\/rYmueOl9YNChtv7k1MzWxkOpl6XwqpWd+sLxBF8580iEOvDC3DdWOY+HTvKyv4wkkT5BiCHKzi0e6DxEgKHORK5Ua94u4zdju8kDPIkioyoSMzRz73rcMJ\\\\\\\/575MmODAlAHYJslL4pYMULfNHqKGB+scVBAlQipo+yrL+8WCIboNEEobLonAdR8FcJdQ4XmGCqZd7cGOBfMnttWOCbPz9H3Zb89EXQanvw+uvUTFzKZi+wVvw7gqjC7rB+kznTjz0yy\\\\\\\/6DGY3JdpvbVdxj3FM6kE0a+6gDsqAzi\\\\\\\/QftPCWIS56V3s3qfvRYMwF0z1FiJQL8kr6ddCwLfzRKaHn6NAvlAk5dfCSGy54PFCMhOvCvsrb1BSrGhmNYjwRc4+KjAydoaPUKTzI7K\\\\\\\/P7gDJVwA9X5uKjgD\\\\\\\/DaA1ZpRoIDT+mBHPlNNprXryERqcF7PgjXY1ftA2t3C6I26oTW6FMqMtTRTU35zbmJXEy4cjhgDch2y4w5DgqqNVkOk4tlcWY\\\\\\\/25e0Fq5Z0f4VidMFlFI8tYYVOzSp\\\\\\\/SCtSO6wC0um5hicO+osoF0odf2zHNonSjPyEMcq\\\\\\\/8ZvORMH1Yz4HSKo9mpqCKTN5W026EyG4C+BGUZDZjfRVjotEE3UcpE0wNReGMn4T70y+ABeq3KyEb5n+yYTruvHViiz64OlwM\\\\\\\/fGnQXsY7eXVKRgx6JGuv0vAJMF7dkNguZHOHwsI7NrlWnyMya+Xc\\\\\\\/qYABdfP+n0eH3Puk5c8kl6yEhufWkh7V+1uPEHydkZmuaIsZjUb5a4XebWLm9Y3TnnVEovg02Sy5726FJt0bHvPaVaLd8EVujtK2zfNO4NPN+0B8cA8CPyGzlUTpsDxyJpBahY5QZSAUvlFsPDIBDGi140fTWGY75BbsHF6\\\\\\\/2tPY7xcfBEMr3PeDZs+e\\\\\\\/XKx0LjyG462MtcWNXTzh2CgaVlBaoiiSg362Am7psKnU2a1qOjHBA2f\\\\\\\/SEsFEwtQI1HJYLUzEg3SviW9rLqskoqTfKPpLEVORLQQHxElkYYTqgPGRc83fgFe96SLvXbXc9wHYDNW3hAFhJHIbVrH13AOW0WaklPSow9m7CK8kQa5kIFag6dPjOL0eZlCX6o7eJl\\\\\\\/IuhWYhELnhOtDxewaLrJjECDxvmKBh2zeSIFKlCYgLjm96UbZzNdKro9K\\\\\\\/DUrEBHoXiSnDHJmjUcES3r2dnKDdGNdWeUsoVKKJR\\\\\\\/lOZKEuzUGApP4LFIcUYrwUy3R7vGzysjKGJO2iJ4ElQPc09tv87\\\\\\\/Q4tY1OJQHEVZ1PdI6x5t4VeDZKuX3rZ2lMwVqBcLAcst\\\\\\\/u6ByGE96zCtsZ\\\\\\\/ZDluC3cCmuHwQLJVy8qQYZ5T6RPqLFPZ3N\\\\\\\/qbgMdSuI6nBn3gMAE7lM8TQgeEC7MF5ocir6XYK7P14zknpJVK80tY+fU3q3650ja21euJl8ryudDC8bvDFHziETN542BmhcjjujXY8miig4P9IBmgWsHWoWM0il9HgKZDmaYOOxXIB3OFztuAsIyJZn10054LtT+gENO3oP1OB7rZQjE7ECPeZxkjE25z5WWOBSKzS1OoAovHjcYgUY9ZRu6OXywcKbCe+0LyUDBmKEZkC9tYwqTP4PwVKgoMofzBaQsXfWopP0v0vz+DYiKdn4fpOxCQfFqusJQCaogJyhadyzTgk4v6MGmugCEuL4QIljj2NhX7fbFJHU3Ri1MlZW\\\\\\\/epsIaJK5yAtOpghSm0l7uADW0As6ekw\\\\\\\/QZGV+NsVlq2upufyn56wpqFGFrBaE\\\\\\\/QY42prLKc8lFKxcwfKZUQjzu4k0MvDHkDPfEum84zfaGBZihEKCfEs3diFfTv3PUsrebmCtNck1ufliRJj5jukCCI2TtZowa+\\\\\\\/XLr1eHs+AP033p71WcsEobNsOF3C1NDNr9DtQHVDpB\\\\\\\/ZTiEn+3sUe9O9oqHWwJqc\\\\\\\/EMpc9f64zF19FAy\\\\\\\/7+9To8vr3WVa0bkUgC2dOqeZbbIjIHEpcKLmL5DOPwNcr0tA6WlM4irsXn4LQ2iYu2edEtVw+PQ\\\\\\\/2\\\\\\\/da76Vd2DgWjFXISZg44JNAJjsNSJFvApWebGdpLdw2tZMkiVmhxZ+XCrdI9znROdtm0y+ou3Ov5bG8M17syGryPCWTGNnGdn3QD7uhWHyT6BR+0LaVygfrfoqtr+t7uBOKPS\\\\\\\/HoRmv9vXv4hDlQQ1XT0FdIcEufPpq2gbhwDnAaOJhCg0pHNV1ZZyK5OTQHQzo5IM0HqooZXcP65MtIZdZDF3fdACHMM60yxlPFMTePnoPg046aDK2JMRpIi35jHZBcqqAgnyVlobU7qvBQMCszC1kGxTpVEwTRLHie3KDUPZAeot+jdJY10e+JH2N3RbaZ+TG77jl391kOuKj2M\\\\\\\/2p9eYKV5BA+7dNycmm8z8uOoQhuhfrzsp38boya5q9s861qs4j5ygTC2KjsH3+IDfTGpWbWtyNSdoKYnabPmWhGG9TXdFwRvIHOA8WN4NbeSMzOuJ8UU4myP3w+wGYsX5dWjpc7FFW58QsKzIdDCkPdFcSq6y\\\\\\\/7cvj02G7HDvNYZu6e0lU6KXVeqoWSJwlF37UjAG+BlXwm7GD5NvxHqnwh7ao7Gnr5Fw9zV+BQsizLnXRqu\\\\\\\/O58udNeVvc8MRH5BLrVtekxz61AkzeTaGZcCuu01uze9\\\\\\\/eQ1wIOSh+PJFTUR8QHz\\\\\\\/nrTM7+8Dbm5ruSSAMvnc7P1AvnGmcuNoMcsCdCSGTr5lAbnKsm0A9MHWf3kmFp8\\\\\\\/WCGFSeQVt13oh77ZfS\\\\\\\/3t4pT\\\\\\\/LYfBQh77TjivymrHSCNQV1nig08I0bHV8z6eVpuJdA4fIzgPl9d7I+8aGw2+shge5U09slvZ79bugHPVdALG55GQctiK6v2WBdqor2L4e2QZ3G7m9iVzRwbkPLxDbSk3tdBZhmpcy3did33ZU8X\\\\\\\/k+BlJjdcG55nR1zdw+eI3FQHDA+zzjt7jHpcWmYifGTY6XhGJXpqR+e4Cdls21uEOfwBY3oIcCCtpY+hjwWegRxZFAprYt6BsvZh163hWAaaxtIrOzT+vKYOWYZOtjsdPQS9nQSZBaPzqiHgi2kf+nYvLI+gom1H4sS0eQb7mDSCT1Ahdq+QckoY6kfUgpbHKlbZxLx\\\\\\\/7A55P+W92WudBHHuT7UmwuSF\\\\\\\/gtH768KcsZHg48w5NNo54NJsgPMl932a7405zX9\\\\\\\/1BJF069I6+X1mvh7HENmgYQHbVdeVpvVJKCBs+Ff2ZrEuezyXgGw8rM3MXhOatu41fjGN8KUpTADnHFwW9RKejy1oOBYkMdonmXCv\\\\\\\/0u+mzfTkSTgm807+NgEoNXfnuIM0HTtg8ybF01aVPiqVpT3ozbOGWBUjK1ySDQbTzeIeam81sSvjMkLm+8SsJ0SlqEdfFqLblx3nwSQAj52v9zUC7ChyXtLudorGjB6B\\\\\\\/yB0HDkUHym6Urlr491tc7Aa4nSPnG2b++qhtQCPdew7vMVma2wBtuEpZ1kB6GC1njpJVdAZxPQyZeASKTFIcATZ70hYqPmD3+pmYkDqju1MiQQX3gKmKPsjKo2k6ZkDeDWF3cIZH9Ss5YD1cRwzLY1HBUpqVpFTkWwXS4RadJeVhBKhTRHMow9JfPcRCkriT+gLpv8UZ9eNW+JGSjPzt4b9S9wExp4vsOGddSWmrzG8QMQ\\\\\\\/VASjomfn2C68dYRubDPJH6aLIpKk6niU\\\\\\\/x2hFm6GlDcaqhDldk2GnPmyqzNdnRGLxjdckZJXBES9Tcc110wVu6OuzV+JorWC14t5L4TxDJx70T9oqmPZbpFV5hlfRGaaZVmYbPzHNPYtqpxMsiy2kscuCMmXq62rBnfA0ebA\\\\\\\/lRSD5XxGUZy8ZEqtza4xuz5h564sgCqpK2FeQdtlJZYUDRh70Yw\\\\\\\/hN3CQNJaPVgvf01GYXFMuSgQ9KkjVjsKQAJTRC2FKAQ7hdl+rS5rAp71IpCgW+XVUqIDVxIfq2wd+FW2wCvU92VQh0PV5mmzGIHTaWmAhaZG7SWUdyAegtj33cDnxIfUa2ASqpZL2smU8EWP+hUYf1nkI5H\\\\\\\/CFR1VGPNmTuuR1B6Zs8ly5C8nyuEKfsSBfrD8TwhVUa2m16VvrmDKhowtWVxvq1nT7oEFBz4kpfEKBqG326IJ8+C+2H7PFYIf8xpw2sG2Z1F+76zNDOrQpEZoECSmAEY6\\\\\\\/Pb35uTr8Z8FIjtfavfl2R1izrHqlgxU61NBpfgQdox3VevC6+lcdz\\\\\\\/30bcSXFbQ1rXRXzLriyABl\\\\\\\/GIf2QyvnR1pqKk\\\\\\\/Eo5wYucKXkADmxYlvvsEwgBbEooDZiapPGCIcZs+FO03abxrb8Qicmhczbf727FlQ2GKnv4oOuZD57EGyvbi25LTszlyfkTvznq47\\\\\\\/JputCfwNEvc+abSFTri\\\\\\\/3K\\\\\\\/AdNxr6INk6GpvUt+BZTLdiD7ZkJIQb3G1WdNPtvt0\\\\\\\/c39BEJF\\\\\\\/Ht0d3PiZJtGGVZ2es9UP0jO6DrAhwiiJ0iqeSH1PcaOLlQmyG0X9GIp3\\\\\\\/6oC8Y6Z8fcoF7uHFfaOZq2nC\\\\\\\/cGw8dxZ4+w2iY46djB8ElhVDH1IAvOsLbEcpiybbOXJS6gDbjR8SCXvnqxX0FSatROJqdigRckagS7TYmJH7wJYTcB52McugunhiIzp8tpI6HRsmFcxCd7v7dunQZCR2nt+mAsGXfOonVRlJJwQCWOFcSkkwq32Px9KlYg7zwBBap\\\\\\\/gjBczsJUsBWm02HUTa0LVQMWukyLXdd3U4Rq0ZVpJAXUKOUkCvrClgjRgVu8il4KyJy3apDHtnbrTxTnphNPVsKHljWBNLImvJiGtXaMp7mISVyLlBIeEKWtzkD59GsSqUF5SQ47cJIJOdGA72D25h82Nist\\\\\\\/sFCXXy+ZkklVTm0s+yBxXCYZNNPL1B8y8aTn3gDeWp8l4WQoaKV0MaUK8iGtl8KG\\\\\\\/C7A+xrKQTzGsICU3JKxfQLuIC5dgrydY31RWO0emV8MU64g\\\\\\\/F076Z8XMFzgFUOZMVQcfyn\\\\\\\/HaDcOwGXv+rqVm7qke+y8waRqyWHRvl0VggG4ToYO8\\\\\\\/1Uau4BhR9ek17lsUANpgx61UAfJ2fgl9F5eX8+\\\\\\\/jNLmbFZ08yilQGMLSc5F7x8A37m7fDMtTL1vNdncN7+lyEKpIKXN1Pd5nSaqol40kj4Wz6bh+XzWsTuPGMBVUqLB98NURaKfgHWKIYy4Fp6tECvE6KVl2gerf2FBgTAWU2abL3DyaoexJUcOvm5wKTws0eJmUjctw+dIvSNGq2F5x8a918KzuqgdY2za0iBznY\\\\\\\/0Yx4J8WNweYSKygeFtk4Jz\\\\\\\/Y+i2RdYA9eRYmKVnpD6GNmW5emR7rJL\\\\\\\/gSLkUwETWzSFimUIsojbfzJIJHokcYBXU4zDqqBThjyitY6o4XlKTNkMhwtt1Y9pWMDlXfc2fmFDesIO7p1pAsGUJy2mUWd7533ealrzX7rOGtRWLnW2S3kJEyY6QuHHkL8WEPOiHiVdhYATpQNHVVAVe+0qoOUy0K6JYwc3pap6WHBKtEpiHtz3wcZJm\\\\\\\/13gzYc4W6s2mTWS9tCdvRXs+TAw1WQA60Q0JWEHG3vYwfTYac6HNCEKGMPmbUwuGZVN4Y1+NfJTo92EwDnGF7iGe2EQzz5Y4nCopTWh7LGQjubc3cFRRdW4RTlHBheh6v0Su0pZL2o2JWUfEWLU9BDoqHt8OpPdsH07nboHvVtrGETALcHDfsbAJj7fV8zLGApbfnTYR4ChYo4X7egiWp\\\\\\\/gvpmYmz8To3yGH1nOLicEZOq0gPJjve9\\\\\\\/IXShuLyZpRT1ynjWdXTu+bJZQCUyUHGq1YatBm3sfBNrTle9EM7cMqH+ue+guifdpX9mINoXz7NEihWR0D5aLyGw2mKqbiV+I70NK28EK60fo+iuhL5g7t\\\\\\\/5FKQ35YbIRg1Lwfl8WVejfMJSBhd5HAMJ1yFaiKP5b\\\\\\\/WdBs4D6mtHZN\\\\\\\/ysjoqJHCcEzuArudYi8ISKFqsE2uE3Oix5\\\\\\\/Ju6UCYe2NGSaspvE73YFE6whHmzQTZtqwQLs6dnj4fS5p\\\\\\\/O6aoyxopCa9iAsiyrPggvof7ma+QnC9ZCIb7IRz7hTLJQGO+UC\\\\\\\/7mOD3peZKUbWTHyBPD7u\\\\\\\/wI4+xmCIs\\\\\\\/W1efKGFsQMIpqdFp9p52hmBE9jGeW3w6UmSs5kXSHNHY2pcosCPbQ4vKD0\\\\\\\/iPDWcsFOMVqoaXzDU+q3qc1l0brV\\\\\\\/uVwLNSVfrbbiSzf8RTCI3Ncdimj8a1+SAV1Ue1vavrZQzlpJe43ODZfmVMqVuFauR0VHuAMY\\\\\\\/E1rYB7rSXVehYeragIWcca1wDaPX3cIj0nPApsfc+v446t9\\\\\\\/PFdMZie7t\\\\\\\/MwwJx6ZWjDlltaejID5BdJ+tbzCQI\\\\\\\/ZBsO28nTRLt72LjwISfa7E+rFF4sT10+G9PowyGNCwBwzc8QPtgsFOjvHUJtjSPmE10r+SuJV6BEBpzwBCbwqICEfqfoB0NuGHgs70arRKu1f60RPQ02kD2Bb61B+v018IpzZ9sS95TXTqv\\\\\\\/0t+j++LI8+JWfadUSdib80va0aUPXeXl6nam81qiCCVDFpdrTB4Lvut9hCtTyWgFx3upvDankI5LQToypgipP+MlPRLB\\\\\\\/3y0KkuWKrgBlbv8N7Gpv8y08hn\\\\\\\/onkQQRrCyGizJgZ+tsLTFljB+wfhLo0H\\\\\\\/Sjv4o2aBbz2SvBAI9V3wmD0f+pQ1h+Cbn24tNyv8OHaA5IjUlAr020npJvPkiork5b4YvMXGbzEO7HdSgAdyZjI\\\\\\\/haX95Jp+Um80lhyOQr1W1SyDBJzVCBBHQ4lVYokQmTmeeFen+Z9veNS3kFgSfxKx9yLtCLXbz5r+jR4X0Y3FLKDPQ3uxYnqytVoIaaDMwtc7mVJWvFWwlSAvHNXeTs+\\\\\\\/ENaih7ysx40PkWWgciM+dJOVX7Q9UnsC0iGVpkThRjW2zbbf1pzaTBYoPs3rh1ljc6IXbddxOi7VUFY2SCQEfs6bFrx8OXe0xTIDGfPQyRJzsPcn1QWz\\\\\\\/anyYg29nwf\\\\\\\/kYP8ayxms0LvH3qwJEKy4VbgE3NlVA2Wp+Kf40WY9ZHI2uJMdJ5V1LUgLCtbM9TxNjXzVZRKZTloOU6hmokjdyAvG2fA96gog2UPApRhCWwnSArYvq0CF3TBr63eNtXW0r58emKQr9nFGKmaEUezdBj7Mi6FWTzGXw99Kjc38FYxxGpQFsxeQexFZe+jIJPa8VtMWpb2Tnts+W6ubp2D1hkEp1ZESKxVCa4A4z6H+YsDfkjlJm6kzzjx0eicouruuiDWc+TrIw2qndwYItXvpeN0H4DZXK6gX9uVbbWzVYiTKsyr3KlXfuVn0Yji0bu4bqmd+fCL5CemJL9n88ngq8N2nk4ZJaJMKT80CO5HTGA\\\\\\\/klpY2XXd7+t2Q0TShuoccHz0dcSvVUFYRUGiPSQJbbG34ZNweg6XaHRNHrnMz9sqHM03krbCQmbI67Wq5PkKF5AgPADkywsUrAl0BGj2ycx6GBnPVx3piQphjpjszTa\\\\\\\/LYs+bOoxPHEnJIWTq6bN3GocVIYH8uRtGPIjCB+hP6rsvofDmkKyC6N+srIX\\\\\\\/t6TArfa1Nb5bzKAVX7stHhF7vhIyTLg+pPK+XolNQ6hZ8Tmj8KlDkU2YXNShQ907M91UtzU8TkxJQbuVRISUc4GHtO0Mi\\\\\\\/t2VFsN\\\\\\\/WkVdN4RQ2cE\\\\\\\/LoMWeVMgwSCIhZzOjJWla3rCsTqUVz8mkijIA8Ctua35z3CqyvzL+NTGJfJAgyC6dEyesd\\\\\\\/GtZT5Kiythz05ZvvnTIW5Y1uJefaTPzTSNcnZZyMM\\\\\\\/Y1EoMJ8u73is6hmfWAe7UxYSNB7uXHHwK6wkurGV9goqV+DddAgM\\\\\\\/GtfdnvUHaI215g0azCRz6J13vBIHH08Ld7\\\\\\\/OAA4PmBOmWM5WrCnYa\\\\\\\/qLcsFZxvBvAURYTEzNHAEN2lom9XPc8Q7rAFvgMCU75o1jPSvYjiTeXmiBRSqQF1jotg0+j1cppEMYMj1EyZ7GL8gac4HA9J6ormFET+WFmtHBgcjP2uFYeEZbjMqpThLj07bu+vMyRMcWB3qhPVl4bhAqZQKcwzr8+SWngOVSjtuXi\\\\\\\/Vh6mypipUU6++ni+K0QmhMp\\\\\\\/owJs0CKchu3L7fXnexRTv09MI98p73i0iKcIWTX18o1JjtchxJ4IxG1wheoM\\\\\\\/\\\\\\\/WNPr4dOPe1pcV3WC6YEq8qGPjj8fvi7wB\\\\\\\/4\\\\\\\/9mMvSdMuGJ02xBjhVOLHCIzfQB6eZN4vtimzTnhtPst1J73mFmjMXh4UmARwECvyUA+gWBf08PbBe+pqIM48LKXJZFlNgKkc32NM3DM\\\\\\\/K4yvv1hzReyxbzOkDsDIbgOwKfBuTd09K2Br577pz1A73YWKOmVbLECHH51bB2aF0xhzROri6G8Kj3IAE1Dei+hiWqPvvUX9ygxqOqwdDMWM8Gn2G\\\\\\\/Yetya\\\\\\\/hCWqmfB9jlXheW9LpSc2emcRXnGkdVIA6W8UtK\\\\\\\/z8PtrjiaUDNQR4jyL57hqoHsHwjm+dzCGsrGpvirLuP9BTySDMo8t5VtVydD6\\\\\\\/4D8CEGk5qZcVpsw3e8KYoryO3EcvvTHPJWaPwbYcfnWX1vApm\\\\\\\/pQFJ9a5wq0UutOrYr8JWtmiHJkMbDV\\\\\\\/MyDYNFc85BARtSwEA4lzFrbwsh563UAgNA26M5V\\\\\\\/7gZ1l9yQLY0ghmjNbTaoujov4ARwPTzmFdyqWMc0ojn\\\\\\\/h+xIjFW83YjLLd8vyPhpP+YFX3h6uRaOQKEMe5N7+nfSoYba8SXpUY4Ny0uraitqdDhbLhLU0TvMTv9a1p67VzkyntkwlEdTef8AiXdJMMhOeM\\\\\\\/WDJ618FhMuhwlcTmOJCu2DRKKzJPf0JDkCM0E\\\\\\\/KjtIOqKtRjiHXqilTtaCyJHqem41HAHO8DdRJ9Z9MJCenfuKOn+T6c0FYP4nCEp4SlvO35zMZI2GtHlJHh9LI2ArVsMKQfHox6cpoRCMnKGBaL8Meuz+HzTbcRz\\\\\\\/QVHg7AxaP1vzeSvdQmSJK5Melf2BP74j1qGq0ZyjMc4nQsnx8SAw\\\\\\\/La4tbz\\\\\\\/SX5dSosrl49eyygw3zrG1XXie2XoiUJE8H2osWeSDpMvrIx6EZZjYAzz+rQvYHGWTZGTnlNF7cPIka0d3f\\\\\\\/iXyZDc97PzAAyAOF5KH9o6x\\\\\\\/IhbTTbAhfBWPSTXvZU+1GtcQVX4IrjeJhLz+qg7+Llvj+QccYOHEmxTAkqrRc6aQHFtMHZixobnfnP9IsYgWRr6K2YaEqzxxOI7J7N4P9X0Z+EJ4CFUS7nIhGGCi1v0szEBjSGoBn1G\\\\\\\/Rbo8gxp1y8bbCjdTcaFJmzduA4Rg\\\\\\\/GzePbxIUi9LDDpmL3CCSYdVUhtheBDqUkkn77wVynn8SIPeGec8jBRL3UL7Y+oTaQKicozxKwIj4y68rkn22zhITZEguoq9YVrce5HQCymY0SKv5onfyJ+9KeyZ25qsEQBdnMINFdF+szLvubvIOj4Ygnch8YsUr+OCDtiMFdwhpn9pnzWEKI3s0XXGL5N7T8Oxb2UA3MxCl14AU\\\\\\\/QHeeJOZltNpFmYAMk9W0QTFWWPFvA3G5DqOQpJBLNw8uYFAzr9tHJ7wIg6PJUzNKGCXl4UOODdQGiCchgu8z5BuobKED9tmgjGezd7AsZkr3gJbakBXwAM4+8LEbS9QNvIPvBZAADY3f3v+\\\\\\\/r0H+1DgHBPhH8n5SjR7jPV1XjrE\\\\\\\/Wp0+ZIVxGq4hlJffgc+n+t3ivNDToVjt6QPyAyZkM0+KpPD0D\\\\\\\/l938fUVXRzWRwmRl8GX9q7a8Tk9IpKkup4Un9K\\\\\\\/A1b0AntkSKI4xXiS+5zsRKRjokYlwZss28qBnVGbG\\\\\\\/7PwIYw5uDpViUayRsgQQubvK10FwF3wGIgE6qreTY1ssiR9xZDZ4yvo0KXZ1zudh+xNUTwJWdvVp5Owz3ocuLcj+SE9+qaWHtZylmHpnvmFpr9nUU1aIe5OlJEKhMW8hTDeaW+TCVlYbyQkwzXOHoxZHqFjpyAAO0RCwfDLdQbm9hOgS4RdjOnMnGs3AAYF0NnnKmfnnnrEuebq6vMsZiNFAdALh\\\\\\\/IQs5M0MUfZzMYU9uLWjlBHt6FWLF56zo9OQd+JUSN69m8FFh0hn+6Rr9iQOsbqlJTS0AY5fviQC7aSgg3nlcwf5U6Hp1\\\\\\\/UWylxRYE9FWgdaqxn9qyiSK2OizgaDg3vxNQq3PDnHAW3+1McY7KgAEth8U59n4E54C6TaHsX8B2aU3ZOBzGshS+qezvi70rhR6rRs8cz3+7lsZlWJyjJaLhpbgfOxcCfYY1\\\\\\\/FKtqUGPwjxbo5DED8iq\\\\\\\/MNZF5EqHtrKXpHbwze9hUR46sCU8Ngx9ZLD3qyPy4rU21uOYyvAy7Ri1fXSRFXi9qjmOr81NOFG994l8WS6WpKjCcHWiNKHeSk8aPb4Eji\\\\\\\/QLrlQRBi0NprfW\\\\\\\/3rc8pbMMrTPHEjbCbGyfvFlXCD1FIXq2dnXsLdtFVhYKrXqbWrrcszirGovO1CAnSJryTqoINW8D8CWZa+kkOsHEEs0zXlnpIua9u7JzL4iNnOC9A5P3DKLvYhuof24VN72Xk5YmM8CCQMXLOqvngMwqsOvQ+j2v2pWlAGoJW4kVQlYjttQpi2+VztA9LvOANvg6W1AZvpJ6VG6P4BPP98t65devI0Qc6Pfoa0Z0VMgckGzZG5DhCmMejys6PCECAcf06+\\\\\\\/tI8GqGhGi7isPC1WejLT3REcSAjECmYX3fMZTraDXdxeHIy6ErKUjJvhrCH2+0jRFZu4I3qR5h2WXDt2G13ABaTRgejyaCY43dXeXcwr9c3Nxq7kJIPrgVyORFiO5rzp3GW3ZmeH5itVVWyH\\\\\\\/NXNVUR+O3JHD4Qqejb6E\\\\\\\/qAwGQu7fCS7pzgTmhqh6gb1DK5rYOA0EoBJejHARq+4pKc3kK4HCVwSdBXzfTFaFSltRcFQ4K0+hUCkCx3SZ21kruQPT8WNOZ1RTEar3qct3cLDSZAQAh6Wdk87DOLrTxlJSeGrRFXNK9Z6aLMDDDclioTGPWz9do7hoBSdmpdnhzpGjgKMRX46bSpThwLjb6BHe778+p0bWn6n43imt+TSl\\\\\\\/wKfGN4xpWuJQTCS0pqwUeDFtd8n+PdJsawh5\\\\\\\/ACaRptOJq9Zqnw0UAkf3f6kMUR9HaAIqr7mWoE\\\\\\\/BfGBFK4cwGEPu4zT9zndoqAGV0la1fAlcwFyuu6tIEDLv7JwZVSqpkAEOG6i784zV7zMkx1om3FKlQzhTcv8Xk0u8KxcKz+WxEDfIq6ZvJ1yEOhCxocOWcSCAWTtR9kJzan0pgS7bUwm+SODs3BT9BToqW5Os1fE7NiyYBWIkCHhVLmQu\\\\\\\/a+fKw0vhZ19Tn2\\\\\\\/pM6mah4TA8EeKrZED0epWcvYn6CLKO\\\\\\\/7bSWeQUMZElHE39uAUUQ8ca4uxwdOqrsErx41SgQ4KV6x6t+nZU1Wj2G5vhUAaQ07b7Pwglsm2RfMPazAyu38Wgky7isBmoP0FGXVqM5VzA2QNmLsAFya4PLDnETGs8ThMB5lEycxmLZaxLrxUJsOHTZ73H9CDLwWJuUC74F+4fELiIVOC\\\\\\\/8bhpiALKgctkm\\\\\\\/q845yL8EGVi+hbu\\\\\\\/0X+32uF5ZsZwIFbxiL6+rwdCGitgrgvRXcTcwU0FSjYif4P9LsCqVwQsIvSiEFEdRSivGENchZODKcuqA7VU0o8WQB3y3LHME0TBDgaCfb3D8aE3eovtkcJaByyeCp7OfwUPvcI+cWgnbjeIb769gN07hJ4k49Mh6U+u1uUNUPNXDZXEqsoFhPWksFogDI+\\\\\\\/2OL3KxMs0n3efvuqlM8zmfc0eZKBCWEzdEUrXX3Bgvdjnu8HI8ZWpBko8OEl0DtHzMNfgKhSQ5JedbuiNLQ3Kqc1bv9k4IWM7gWC1s9w+N\\\\\\\/fU77bDiTK7YA6qamwFgs9jbAae\\\\\\\/sXDFEkF4qBCXgXeMCaruXDHmP8X6x5VpxmqpB\\\\\\\/u8mdHkwOkDLnpvaE8Ndfu2TBvzeoGb2XuQI81YNTmc61hq8F1hWhobBy\\\\\\\/HyxxdsgnwDqskdtsMDqRw1dFvt0xy3Trj2KTK5MsyDfb\\\\\\\/NUd+QaiNrOykSATiAAPs0V+ttXLaYZPblOCXZBZBCPa7FHmWnHvBrHwfiDbdcM1EX+q0GDJ2w2NzwYdOw1aflV29pRoSXrhe76dgmaPc9YCiDjBoML4G6nLLwRF7bIuoQNSnJT4gCeaOYbke8RYFRDIoFEmINMwYh63M1kU8OBLCs72uMzOxfQPPf5JDNAT6d+xmL50BfQDmMiCVZBHjNnlFbz0bsdXHSQX4kOqEvaHgWNVQkj5m26dgY8nLxmxxuxzuYZQbIwY+PiqF36Jxq+8NaYUbqW9zupUazDKwwABjNXRYAsJ6dGBDGg3dT7n2Xf9kqErCy5DDlNnFI3rXVVcdzV4mOIgaxwmAh+EQMSn7WyZTq0FQvn+L30R7B9qMmZPJYwuJO5lXOUMdli87MYEqk88cl9DuxQ3uoIf2I00z3+bCH3jDH37vH9AsMr\\\\\\\/za7x4lCkltFos1XU9vlJSe4fIB0jfkMks8YWW2lGSdBaonFN1oOppPLEncw7fLEZ5tWkLEYLqDSe0BCOgjayLuakiUT70wDUDzFGHfbcu3IxrsNTiZUvPNvTRnpm5gLRyTzJB6Lu2a+i+YL\\\\\\\/aYo1QZ1LUL3CSfOiSzBGyd\\\\\\\/Q4Zwr3f1ASwxBuo8be1RsYXYhCKor4hpvWVyWdn8XWghJoHGnscBPh+qBod\\\\\\\/LLfuLGa9f8wg9Ko5zjOkenwjMV9Hv\\\\\\\/H9n8+4O8Jk4fezJ6ceOQbRje5B5oFj00IV1TVujDVP2Nik1wh3RNp3aBVN71rsSjqqdNaJgKfVvLRCDhyyyqKwJYFTVDro0omOgLBTKqDB7eInCyhA4vUqhvLD+yW6mgwEHY6RvoM\\\\\\\/\\\\\\\/YClGa4Oxelh+g+Sw5M2GongqHqGzm1pzmc06OM0sGWhCMHptG0GRUVITKbEqSWVUUOlGUqrV43gRwiWGd8nW7dEgTZk9Da4Q0Rr1RG2VE93flFww5FvN+3w2yyWbjB16AD6+8CrOtnZR567ab3mfXq2Isah50wLh3gJhuDq+L99tVf1kKA0wLI+4cBbrU2jdw4faEIjIMXXTUPETIvCSDw4pA0cx9ELCvoEeL2Byj22k4oVHa4GMdCMhpgU0N3I5iZaQ24GnKZvIWzdAhSz6r6xBL0JTRRL9i4y0CVA76rQIIOO4CKn1L0cjv5efKI7AEdS6ylabWSz8+hZeY7qgsBL0++8CJ5mmgc9DQZfAjvTPltEeaHAx8HJVQqG9PJUrod8AnFZx5OADmEQOw8oKClVlbZPV9rCoMe\\\\\\\/fnJDDxXqxV4nsqTcAO3rsn1W+xy+Ws+EJyFGmZc+sWOBSLWFp2QJryA9tgLoPN7YjJ1yg7AJP11CrOXPcVCiOQaSERJoUc5lA1npyOQcR1a+R1MLas30HA14qpOJFB1EJ8UQ7EnwJN2ts9aDge1A06sErhUPCwYuZoJ458n8b1MZyzxBDxIpe50+lDieuQ5WsLn+vg0lnYD8C28DO591F9e5qZKoNTspms354f1qqnNteylqbJ8ejmIIY2rM8QyaOHfWLoOrIm2ylD65s642Y9k1gzjKIMq\\\\\\\/i0Q875JJEZzfjQIy6BS\\\\\\\/7LeNBOjqlZptMetyOW7788qE5XVnGKbBA4weJQP3gGY7nECb9y8rDnPok5Pu2fm6D9ALYvbGbDoCguNuTAOxnaZqf2c6DcREk6KgMXJW76kTJJsIkgV+s2OQtIU3i9biZPS\\\\\\\/gt3ZZD81wiik5osm7XvhAYVwrYjwWnfX5k1Wv54jn7omyijZ0+xj8Cg7ORaGLeazm7DWl0ABezjZrqm3cLDArLU0Uqwiezeu6qzAsRZIFOZNmOsJ62meKjPugedPd3en8Ldt6m1UW4LDwGVVM6AUIregKmMm0e2z1Teyy0Dss8nrif\\\\\\\/IpYfLwVC7rBW68W\\\\\\\/dHyPXyqn0nBJ6jLTU7G0T6GzWct0DzQAKgtIaDcp0aC8ZO8FNQxT\\\\\\\/v5GwqOkr6KvAFL2t7jkjPBmDw1AOYZCT\\\\\\\/BlHoRqewfNIxkE2XpWo8i6fJGgRt\\\\\\\/YryCYoubuIZxNk8stqVKBnXylKu7XnTXvBNBVjdz8FXjLx14rNeJo\\\\\\\/gscso33H0gqKqkObPXEvV\\\\\\\/Wddcp36ik6sWZeZhqUsV5Nxc\\\\\\\/jYi7bgtGF5axLq2PpctoZ81bG0KuyydzXf4+VrEMihloYucscbvqlpexM2LvALBbBfrq4PICpwkpcrDKgUv6yCEWp3xA\\\\\\\/WSjVyXMQiLKEf6d+cYO9BPcc+vVAhxJkyiFC73UUlFQFXSXJruntRBfw3DMMYw4fTHtQpFdRhUVFCC8cWKfaK62Q1ZUfzy\\\\\\\/XdnlzqDdD9JNo99gRF0NjcHRKVMA3tOBje7dAQK5P\\\\\\\/aV9iy7IDWZQZ8O6wCHI9oMyi6siEahQ7sdb\\\\\\\/lU7pR8DlbV240uVMFXwWbuA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022daGliIvbrJntoSxiCSsOwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222dbdb4b9014f6b22b14cedf26d1b3753bb39802e195d0407ca7f77ab3ad1d584\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yvUzXPBYIRLpnupgMfFUMR6j1OjS24v4J4uqjkOXrzR+yqxlygPZ67IQ\\\\\\\/bCgs7dT30K136m+Ks3p9oDcj6BevqbE0hPMQEPq4gaOJisq7MYg4ged7FKnvLb\\\\\\\/UKRDzu2dh2+CEZmL6oFAb\\\\\\\/S+PbpgnefHFgVUaPq9XDxKc51Ht3EPofjtIZ3WD4aDngnTFX6Gygbm9VWc6+JL51QvcBxj6vAho6X+mHroYOpWIPl6lPLfBUB4aFnbeOcoBVdT\\\\\\\/g\\\\\\\/bbv6QDY6Lh98RErMNzmKfOq8A48tU99LuM+ZK3L5YqE2ttV4BLC4fFC4pxwASyYh9lF6OSePm0zcQcBTNMu0mpU0\\\\\\\/B9Udx6Rm0jC5+ETeJgufpT+EyP5rcE+uaZouUIMc2p\\\\\\\/6yHSpcvCIKGc52a7itGup6rGtPTncq8YAoZunQvGy\\\\\\\/7OWLYpUa5BscPS7rtYLKPPxrsmrblHDkhQIOjd3Uwyq08RkbAqEwm\\\\\\\/EMDc\\\\\\\/Pc4Br5hvvczTwRr7jbEQ6bchWVi2S64JYcuUuHIF\\\\\\\/9VPbdoMOxVvG0DoESO9sWAqHpQtdM1i4EIHNMwUVzAZk7aMapQn58378h2NxaToUdSjTUK2jG+qSWzTuRcXf6N9LDIP0XjX90E0cID9lQgGI2g\\\\\\\/Djos9i2fhXZD9lY6VFkyBg4t+9ELs6T09FiGiGIeb2RusywiUk5YY8g6fTjvFJW\\\\\\\/A0ijSyljnvPLxtdZ0lo6YYNMk6SWDj2h6bmj9kAhylHC1P\\\\\\\/dStLbllxErb7VyfMJbnsIYld5Emz+zQSXPiXkn6LLii6l1Zlz7p45sgEgLbi\\\\\\\/bmDuAvdvuvktxcod\\\\\\\/LBQir2T0aK2++edU9ausOr+HTQyxu\\\\\\\/qxp2EomUAIHQ7fd7J6jEonW0N5KFaf6rrfcHhYUAmYaKM8+q2ylXeYpJr5S0F8KvIrvdx4HuVtutDFiBl5gK2pGRqlbNL7UXrLxLpqs73onujpFgsp4WseMIqBejw1abb5J5GHfEAVZJqniF4y524ZU5bCuqycH1vRlCOshGBjOkrHeh7bLaCTlzHDn7Su5QWdFNvKW79fKeAVu5m3edPeRpU+zIM3BZLpkh7qDyD3bg1jSl6\\\\\\\/N3zJ9wDyFKI1MdpohJiJl4myfmCek\\\\\\\/EOFCSXTEFZMLtfOi9U\\\\\\\/P\\\\\\\/LS3ojj0Um3x8eaM+smbQby8CTTpeqnAaiFouwfrY8dlmnq9Hxn8jz7jwuImaBWOIB+83WSq+IJVyQiuxBUpS7vkLjcQy6BsvJLr4JER6iZ+TvK6dZFoOhy745RxMQr5P3f2n9JhGeMXkkpAroW6hapGVHu+1f2SF31IISlLs1n9E7aZpFjU87aC4NyNqPmWT\\\\\\\/yMRUwY+dwlOFTYGGY15nss26GAAtJar9LHmBYuLVbjFjiHU4dqMVG66h+DLwspEtJk7N5GKdLSNqA0CwXwuhN\\\\\\\/QV8Fg\\\\\\\/m5Wn16IuWUDQuO+AEE5IpGpbiaCWe1D49zR23Siu+OzkshnY8atRgQoJht7GWwSlmEr\\\\\\\/qNZm8y4j8BJSKH8ZotG80n6HpeKRHt5f3G\\\\\\\/ANcnBGqWCjsJkNU3Z5b9AFUmAhk9tb5s1A+jlioaL+D6KpHE27MX5QUOF7YZ6dzfxyoUMoYyhAFN7Bx62QKBqbnNurR6B6Y\\\\\\\/4pFgPSXoeo8V\\\\\\\/yRZeaQDqZd9yafocFkBkDKnhLYFX3hRHJPlNfhoVKWtFsk2aC7+G4g60Z1bf4dpdfBeO3Yv5h4v68RKzUIKamKx8BOlkfnhPkiTtocZtb8uZu5wtS464YDJx6PaGXk4mv5uZzqq16GhMN7Esy4ACbCXVlueiTpCZB+Iw53LWNHLbaDHhJSqmm8oenn6gJNrgaJTzwNw1GwOARGaMXvb4mOTrlU0GqKYlwdOreeirTjTqE+dC0\\\\\\\/htdbhYPMvy3MqExWUl\\\\\\\/G9YR7PXCRDhFfAORa0bci\\\\\\\/WcxfJaAyX0YKtMUek9ZVl6ms0r\\\\\\\/9k5xu9\\\\\\\/EttBgBYYnb081Xc1OxoHW+cFoMb\\\\\\\/25uBhkRnU7mA==\\\u0022,\\\u0022iv\\\u0022:\\\u00223XOQ+LDyyuPInynkYWY\\\\\\\/sw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022793880483b311254bf286a5e22522c513a7bc50cd0de33c0c718c574e9e3d97e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460026 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1865" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-T7O0FLHqJcPRzDIrpPp8-7lmSPeskNhfXoMvi2_apr0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1865" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uXGMXDV0br1dfIl-y6iXN63EigZi3PYD4Tt-l0if9aU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/9/1857" title="Targeted Modification of Gene Function Exploiting Homology-Directed Repair of TALEN-Mediated Double-Strand Breaks in Barley" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/9/1879" title="An Updated Collection of Sequence Barcoded Temperature-Sensitive Alleles of Yeast Essential Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460026" id="node460026" data-pisa="ggg;5/9/1865" data-pisa-master="ggg;g3.115.019828" data-apath="/ggg/5/9/1865.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Enrichment of H3K9me2 on Unsynapsed Chromatin in <em>Caenorhabditis elegans</em> Does Not Target <em>de Novo</em> Sites</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yiqing Guo</span>, <span class="highwire-citation-author" data-delta="1">Bing Yang</span>, <span class="highwire-citation-author" data-delta="2">Yini Li</span>, <span class="highwire-citation-author" data-delta="3">Xia Xu</span> and <span class="highwire-citation-author" data-delta="4">Eleanor M. Maine</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1865-1878;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019828 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460026" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yiqing Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yiqing%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYiqing%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bing Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bing%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABing%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yini Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yini%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYini%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xia Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xia%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXia%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eleanor M. Maine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleanor%2BM.%2BMaine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maine%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEleanor%2BM.%2BMaine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emmaine@syr.edu" class="" data-icon-position="" data-hide-link-title="0">emmaine@syr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/9/1865" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460026" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460026/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1865.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460026" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460026/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1865.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460026" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460026/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1865.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460026" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460026/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/9/1865.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460026" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460026/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Many organisms alter the chromatin state of unsynapsed chromosomes during meiotic prophase, a phenomenon hypothesized to function in maintaining germline integrity. In <em>Caenorhabditis elegans</em>, histone H3 lysine 9 dimethylation (H3K9me2) is detected by immunolabeling as enriched on unsynapsed meiotic chromosomes. Loss of the SET domain protein, <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a>, greatly reduces H3K9me2 abundance and results in germline mortality. Here, we used <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutations to disable X chromosome synapsis and performed a combination of molecular assays to map the sites of H3K9me2 accumulation, evaluate H3K9me2 abundance in germline <em>vs.</em> whole animals, and evaluate the impact of H3K9me2 loss on the germline transcriptome. Our data indicate that H3K9me2 is elevated broadly across the X chromosome and at defined X chromosomal sites in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> adults compared with controls. H3K9me2 levels are also elevated to a lesser degree at sites on synapsed chromosomes in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> adults compared with controls. These results suggest that <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a> activity is elevated in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants generally as well as targeted preferentially to the unsynapsed X. Abundance of H3K9me2 and other histone H3 modifications is low in germline chromatin compared with whole animals, which may facilitate genome reprogramming during gametogenesis. Loss of H3K9me2 has a subtle impact on the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> germline transcriptome, suggesting H3K9me2 may not be a major regulator of developmental gene expression in <em>C. elegans</em>. We hypothesize H3K9me2 may have a structural function critical for germline immortality, and a greater abundance of these marks may be required when a chromosome does not synapse.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/germ-cells" class="hw-term hw-article-keyword hw-article-keyword-germ-cells" rel="nofollow">germ cells</a></li><li class="kwd"><a href="/keyword/gene-regulation" class="hw-term hw-article-keyword hw-article-keyword-gene-regulation" rel="nofollow">gene regulation</a></li><li class="kwd"><a href="/keyword/histone-modification" class="hw-term hw-article-keyword hw-article-keyword-histone-modification" rel="nofollow">histone modification</a></li><li class="kwd"><a href="/keyword/met-2" class="hw-term hw-article-keyword hw-article-keyword-met-2" rel="nofollow">MET-2</a></li></ul><p id="p-4">In the germ line, as in all developing tissues, gene expression and chromosome segregation are in part regulated by chromatin structure. Moreover, as the tissue that gives rise to subsequent generations, the germ line undergoes distinctive processes during gamete formation. One distinctive aspect of germline chromatin regulation conserved in many animal species is a process called meiotic sex chromosome inactivation (MSCI), whereby the heterogametic sex chromosomes acquire histone modifications during first meiotic prophase that are consistent with a heterochromatic state (reviewed by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Handel 2004</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kelly and Aramayo 2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Maine 2010</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Turner 2007</a>). Similarly during first meiotic prophase, any homologous chromosomes that fail to synapse because of mutation, translocation, or duplication acquire elevated levels of heterochromatin-associated histone modifications in a process variably termed meiotic silencing of unpaired chromatin or meiotic silencing of unsynapsed chromosomes (MSUC) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Maine 2010</a>) . These meiosis-specific chromatin changes, observed in many animal species (including mammals, birds, nematodes, some insects), have been implicated in transcriptional and checkpoint control and have been hypothesized to play additional roles in regulating genome stability.</p><p id="p-5">In <em>C. elegans</em> (X0) males, the X chromosome acquires a relatively high level of histone H3 lysine 9 dimethylation (H3K9me2) during first meiotic prophase (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kelly <em>et al.</em> 2002</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bean <em>et al.</em> 2004</a>). H3K9me2 levels also increase during early meiosis on unsynapsed homologs present in certain mutant backgrounds, <em>e.g.</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutant hermaphrodites where the X chromosomes fail to synapse (because <a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">HIM-8</a> mediates X chromosome pairing), or on single-copy free chromosomal duplications, <em>e.g.</em>, <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bean <em>et al.</em> 2004</a>). <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a> (the SETDB1 ortholog, histone <span class="underline">met</span>hyltransferase-2) is primarily responsible for H3K9me2 marks in the <em>C. elegans</em> germ line, as H3K9me2 is not detectable by indirect immunofluorescence labeling of whole gonads isolated from <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> mutants (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bessler <em>et al.</em> 2010</a>). <em>In situ</em> hybridization analysis of X-linked oogenic genes, normally expressed in late pachytene, revealed a sharp reduction in mRNA levels in XO hermaphrodites (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bean <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jaramillo-Lambert and Engebrecht 2010</a>), suggesting the elevated H3K9me2 may limit transcription of X-linked genes.</p><p id="p-6">We previously showed that meiotic H3K9me2 distribution is severely altered when certain components of the germline small RNA network are mutated (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Maine <em>et al.</em> 2005</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). Specifically, H3K9me2 enrichment on unsynapsed chromosomes and chromosomal regions is reduced when components of the <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> (chromosome-segregation and RNAi deficient) Argonaute pathway are absent, including <a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene">EGO-1</a> RNA-directed RNA polymerase (RdRP); <a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene">DRH-3</a> helicase; the Tudor domain protein, <a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">EKL-1</a>; and <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a>. Unsynapsed chromosomes evaluated for H3K9me2 enrichment included the male X, hermaphrodite X chromosomes in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants, the large free duplication <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em>, and several extrachromosomal arrays (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Maine <em>et al.</em> 2005</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). In addition, ectopic H3K9me2 is detected on synapsed chromosomes in the absence of <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a>, leading us to propose that <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> activity might limit deposition of H3K9me2 marks at synapsed regions (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). This phenotype is consistent with the proposed role for <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> as a positive regulator of germline gene expression (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Claycomb <em>et al.</em> 2009</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Seth <em>et al.</em> 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wedeles <em>et al.</em> 2013</a>).</p><p id="p-7">To better understand the biological function of H3K9me2 enrichment as well as the mechanism by which these marks are targeted to particular genomic sites, we sought to map the sites of H3K9me2 enrichment on synapsed and unsynapsed chromosomes. We used chromatin immunoprecipitation followed by sequencing of precipitated DNA (chromatin immunoprecipitation sequencing; ChIP-seq) to evaluate the H3K9me2 pattern in control adult hermaphrodites and adult <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants where the X chromosomes fail to synapse during meiosis. The existing <em>C. elegans</em> literature regarding H3K9me2 distribution in the meiotic germ line is based on indirect immunofluorescence assays. The use of ChIP-seq allowed us to obtain a detailed picture of H3K9me2 distribution across the genome. We detect a similar pattern of H3K9me2 distribution in control and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> hermaphrodites, with a higher proportion of H3K9me2-associated sequence reads derived from the X chromosome in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> hermaphrodites compared with controls. In addition, H3K9me2 levels are higher in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> at many sites dispersed across the genome. Protein blotting similarly detects an increased H3K9me2 abundance in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> relative to controls. Taken together, these data indicate that H3K9me2 levels are generally elevated in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> genome, with an especial enrichment on the X. Messenger RNA-seq analysis reveals that loss of H3K9me2 has a subtle effect on the gonad transcriptome. Interestingly in light of the link to RNA mechanisms, H3K9me2 levels anti-correlate with candidate targets of <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a>/Argonaute activity and strongly correlate with candidate targets of <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> (worm Argonaute protein)/Argonaute activity.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains</h3><p id="p-8"><em>C. elegans</em> strains were cultured using standard methods (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brenner 1974</a>). <em>C. elegans</em> var. Bristol (<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) is the wild-type parent strain for all mutations used in this study. Information on specific genes, alleles, and nomenclature can be found at <a href="http://www.wormbase.org">www.wormbase.org</a>. Strains used for H3K9me2 ChIP-seq analysis were <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000367;class=Variation">b232ts</a>);<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00144038;class=Variation">e1489</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000367;class=Variation">b232ts</a>)</em>. Strains used for gonad mRNA-seq were <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00090812;class=Variation">n4256</a>);<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00249646;class=Variation">tm611</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00249646;class=Variation">tm611</a>)</em>. Strains carrying multiple mutations were generated by standard methods and confirmed by polymerase chain reaction (PCR) and/or sequencing. The strain deleted for all 12 <em>wago</em> clade genes (MAGO12) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>) was a gift from Darryl Conte. To obtain MAGO12 males for indirect immunofluorescence, L4 hermaphrodites were placed at 30° for 4 hr, males were collected from among their progeny and used to set up matings, and males from subsequent generations were immunolabeled.</p><p id="p-9">The two <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> alleles used in our strains disrupt production of functional <a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">HIM-8</a> protein and, consequently, prevent X chromosome synapsis and pairing (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Phillips <em>et al.</em> 2005</a>). We performed indirect immunofluorescence to confirm that H3K9me2 accumulated on the unsynapsed X chromosomes in each <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> strain and failed to accumulate in each <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene"><em>met</em>-2</a> strain, as expected (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/019828SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS1.pdf">Figure S1</a>). We used <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00249646;class=Variation">tm611</a>)</em> for mRNA-seq because it contains a small deletion that makes an excellent internal control for the sequencing data.</p></div><div id="sec-3" class="subsection"><h3>Indirect immunofluorescence and Western blotting</h3><p id="p-10">H3K9me2 indirect immunofluorescence was performed as described (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). Briefly, gonads were dissected in 0.1−0.2 mM levamisole/phosphate-buffered saline (PBS) in a deep-well slide and fixed by adding 200 μL of a 3% PFA/1× PBS solution to the well and incubating at room temperature (RT) for 5 min. Tissue was washed 3× in PBST (1× PBS/0.1% Tween-20), blocked in 30% goat serum/PBST for 1−2 hr, and incubated with anti-H3K9me2 (1:200 dilution, Abcam 1220) at 15° overnight. Tissue was washed 3× 15 min in PBST, incubated for 2 hr with Alexa488-conjugated goat antimouse secondary antibody (1:200 dilution, Invitrogen) at RT, and washed 3× in PBS; DAPI was included in the penultimate wash. Tissue was mounted on slides in VECTASHIELD medium (Vector Labs). Images were captured on a Zeiss Axioscope.</p><p id="p-11">Western blots were prepared using standard protocols. Gonads were dissected from hermaphrodites at L4 stage + ∼18−24 hr. Animals were washed 2X in M9 + protease inhibitor cocktail (Roche) and dissected in fresh M9 + protease inhibitor. Dissected gonads were transferred to a well with fresh solution; when ∼150 gonads were accumulated, material was centrifuged, excess solution removed, and tissue was frozen on dry ice. For whole worms, the specified number of animals was washed in M9, transferred to a tube, spun down, excess solution removed, and frozen on dry ice. To prepare samples, an appropriate amount of 4× loading buffer was added, and sample was boiled for 10 min, cooled down on ice, and centrifuged. Supernatant was transferred to a new tube and volume was measured. Total volume was adjusted by addition of 1× loading buffer, and appropriate amounts were loaded onto gels for histone detection. Typically, the adjusted volume was 32 μL; 16 μL was used for testing histone H3 modification, and 8 μL was used for testing histone H3. Pierce SuperSignal West Pico (#34080) or Femto (#34094) detection substrate was used to visualize protein. We noted that the Femto substrate consistently detected rarer modifications, H3K9me2 and H3K4me3, at a higher level relative to total H3 than did the Pico substrate. In contrast, we did not observe this difference when assaying protein that is present at a relatively greater level, <em>e.g.</em>, gonad H3K4me1 was detected at 80% of whole animal H3K4me1 by both the Pico and Femto substrates. We hypothesize that detection at the lower limit of the Pico substrate sensitivity is unreliable, and included only Femto detection data for H3K9me2 and H3K4me3 in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. Quantification was performed using Image J software (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gassmann <em>et al.</em> 2009</a>).</p></div><div id="sec-4" class="subsection"><h3>ChIP-seq</h3><p id="p-12">To obtain adults for ChIP, we collected embryos from animals grown at 15°, hatched them into M9 medium, collected the synchronized L1s, raised them in liquid medium at 22° for ∼60 hr, and harvested adults. For <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> strains, hermaphrodites were separated from males using 35 micrometer nylon mesh (Sefar). Animals were fixed in 1% formaldehyde for 10−20 min at 22° (RT), frozen in liquid nitrogen, and stored at −80°. Lysis buffer (50 mM Tris-HCl, 1% SDS, 10 mM ethylenediaminetetraacetic acid (EDTA), 1× protease inhibitor cocktail; Roche 04693132001) was added to frozen samples, which were then thawed on ice. Tissue was disrupted and chromatin was sheared with a probe sonicator (Sonic Dismembrator, Fisher Scientific) 8 × 10 sec on setting 7 (output power ∼24 Watts), with cooling on ice for 2 min after each pulse. Protein concentration was measured by Bradford assay. A total of 1−2 mg of total protein extract (100 μL) was added to 900 μL of dilution buffer (1.1% Triton X-100, 1.2 mM EDTA, 16.7 mM Tris-HCl, 167 mM NaCl, 1× protease inhibitor cocktail, Roche), and precleared by addition of DynaBeads. Anti-H3K9me2 (Abcam1220) was used to detected histone modification. Normal mouse IgG (Invitrogen; 10400C) was used as a nonspecific control. For each 1 mL of precleared lysate, ∼6 μg of antibody was added, and the sample was rotated at 4° overnight for ∼16 hr. Prewashed DynaBeads were added and the sample was rotated at 4° for 1 hr. Beads were collected by magnetic separation and washed with a series of buffers: low salt wash buffer [0.1% sodium dodecyl sulfate (SDS), 1% Triton X-100, 2mM EDTA, 20 mM Tris-HCl, 150 mM NaCl], high salt wash buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM Tris-HCl, 500 mM NaCl), and LiCl wash buffer (1% sodium deoxycholate, 1% NP-40, 1 mM EDTA, 10 mM Tris-HCl, 250 mM LiCl), 5 min each, and then washed in TE buffer once for 5 min. Antibody/protein was eluted in 200 μL of 50° elution buffer (1% SDS, 0.1 M NaHCO<sub>3</sub>); supernatant was removed, and a second 200-μL elution was performed. Cross-linking was reversed by addition of 20 μL of 5M NaCl to each 400-μL sample, and incubation at 65° for 4 hr. Then, 31 μL of PK solution (10 μL of 0.5M EDTA, 20 μL of 1 M Tris-HCl, pH 6.8, 1 μL of 20 mg/mL proteinase K) was added, and samples incubated at 45° for 1 hr. DNA was recovered by phenol/chloroform and isopropanol extraction. Before performing ChIP with adult chromatin extracts, we optimized conditions using chromatin prepared from synchronized L3 larvae and performed real-time PCR to assay genomic regions reported to be enriched/not enriched for H3K9me2 by modENCODE. We subsequently performed ChIP with synchronized adult hermaphrodites, as described.</p><p id="p-13">DNA libraries were generated using the NEBNext ChIP-seq Library Prep Master Mix Set for Illumina (NEB #E6240) following the manufacturer’s protocol and using QIAGEN column purification. Library concentration and size were evaluated via Qubit fluorometric quantification (Invitrogen) and on a Bioanalyzer (Agilent Tech), respectively. DNA was sequenced at Cornell University on an Illuminar Highsequation 2500. In total, four ChIP DNA libraries were generated, with an average of ∼10.6 million unique mapped reads per library (see the section <em>Elevated H3K9me2 levels are observed in</em> <a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a> <em>mutants</em>).</p></div><div id="sec-5" class="subsection"><h3>mRNA-seq</h3><p id="p-14">Gonad dissection protocol was adapted from (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2009</a>). Gonads were dissected from hermaphrodites at L4 stage + ∼18−24 hr. Animals were washed in “red PBS” (1× PBS + 1mM aurin tricarboxylic acid, an RNAse inhibitor) and dissected in red PBS with 0.2 mM Levamisole to immobilize worms. A total of 10−20 animals were dissected at a time; ∼15 gonad arms were collected (cutting at the spermatheca) per set of dissected animals. Dissected gonads were placed into Trizol (Invitrogen) on ice. Dissected material was stored for as many as several days at −80°. RNA was isolated following the standard Trizol protocol. Messenger RNA was isolated using NEBNext Poly(A) mRNA Magnetic Isolation Module (NEB#7490). Libraries for RNA-seq were constructed with the Illumina TruSeq RNA Sample Preparation Kit v2 (cat no.: RS-122-2001) following the protocol provided. Subsequent steps were the same as for ChIP-seq. In total, four mRNA libraries were generated; an average of 54 millions reads aligning perfectly to the <em>C. elegans</em> genome (see <em>Impact of H3K9me2 loss on the gonad transcriptome</em>) was obtained per library. The Pearson correlation coefficients for biological replicates of <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> are 0.87 and 0.97, respectively.</p></div><div id="sec-6" class="subsection"><h3>Analysis of DNA sequence data</h3><p id="p-15">ChIP-seq data were mapped to the genome using Bowtie (v1.0.0) or BWA (v0.7.7) aligner (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Langmead <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Durbin 2009</a>); only those reads mapping uniquely to a single position in the genome were analyzed further. Peaks were called using MACS2 (v2.0.10.20130306-beta) (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zhang <em>et al.</em> 2008</a>) with input as control (sample to input) or one of ChIP-seq datasets as control (sample to sample). BEADs analysis (v1.1) then was used to normalize the data as described (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheung <em>et al.</em> 2011</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Vielle <em>et al.</em> 2012</a>). Two biological replicates were averaged after BEADs standardization, and H3K9me2 distribution relative to important genomic features was characterized with the use of CEAS (v1.0.2) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Shin <em>et al.</em> 2009</a>). Bedtools (v2.17.0) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Quinlan and Hall 2010</a>) was used to assist in finding differential peaks. R (v3.1.1) was used to plot the results. RNA-seq data were mapped to the genome using STAR (v2.3.0) aligner (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dobin <em>et al.</em> 2013</a>). Cufflink (v2.2.1) and Cuffdiff (v2.2.1) were used to call RPKM values and determine which transcripts are present at significantly different levels in the various datasets (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Trapnell <em>et al.</em> 2013</a>). Bedtools (v2.17.0) was used to find genes located at H3K9me2-enriched sites. The IGV (v2.3.36) genome browser was used to visualize data (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Robinson <em>et al.</em> 2011</a>). All sequencing data have been submitted to the GEO database at NCBI as series GSE67030.</p><p id="p-16">We took two approaches to identify repeat regions. </p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-17">We screened the <em>C. elegans</em> genome (version WS 220) for annotated repeats listed in Repbase (version 20140131, <a href="http://www.girinst.org">www.girinst.org</a>) using RepeatMasker (version open-4-0-5, repeatmasker.org, search engine RMBlast). This procedure identified seven types of repetitive sequence, classified by Repbase as Retro elements, DNA transposons, Helitrons, unclassified interspersed repeats (unassigned repeats), satellite sequences, simple repeats, and low complexity repeats. We then screened the H3K9me2-enriched sequences identified in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> ChIP experiments using the same method. </p></li><li id="list-item-2"><p id="p-18">We downloaded the Wormbase repetitive DNA data set, which includes additional repetitive sequences not annotated by Repbase (listed as “other” in <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS1.pdf">Table S1</a>), and compared these sequences to our ChIP-seq data sets using Bedtools.</p></li></ol></div><div id="sec-7" class="subsection"><h3>Quantitative real-time PCR</h3><p id="p-19">Real-time PCR was used to assay H3K9me2 ChIP results for select sites before deep sequencing. Real-time PCRs were performed with the Bio-Rad CFX Connect Real-Time PCR Detection System and RT-PCR reagent (Invitrogen). Primers used are listed in <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS3.pdf">Table S3</a>.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-20">ChIP-seq and gene expression data are available at GEO with the accession number GSE67030. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> lists the distribution of H3K9me2 enrichment on different classes of repetitive sequence. <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS2.pdf">Table S2</a> summarizes the developmental phenotype associated with <em>wago-1;met-2</em> double mutants and controls. <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS1.pdf">Figure S1</a> contains representative examples of H3K9me2 labeling, as detected by indirect immunofluorescence, of all relevant strains used in the ChIP-seq experiments. <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS2.pdf">Figure S2</a> compares our adult ChIP-seq data with adult data from the modENCODE project for select sites. <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS3.pdf">Figure S3</a> compares our adult data with germline-specific ChIP-seq data from the modENCODE project for select sites. <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS4.pdf">Figure S4</a> contains data to indicate the low abundance of H3K9me2 in isolated gonads is not an artifact of the dissection process. <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS5.pdf">Figure S5</a> demonstrates that H3K9me2 labeling, as detected by indirect immuofluorescence, is normal in MAGO12 males.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466587/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466587/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Most sites of H3K9me2 enrichment are common to <em>fer-1;him-8</em> and <em>fer-1</em> adults</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Elevated H3K9me2 levels are observed in <em>him-8</em> mutants</h3><p id="p-24">To identify sites of elevated H3K9me2 on unsynapsed X chromosomes, we performed ChIP-seq analysis of control adult hermaphrodites (<em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em>) and adult hermaphrodites with an X chromosome synapsis defect (<em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants). The <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutation disrupts X chromosome pairing and synapsis (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hodgkin <em>et al.</em> 1979</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Phillips <em>et al.</em> 2005</a>) and the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>(ts)</em> mutation impairs spermatogenesis, thereby preventing embryo production (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">L’Hernault <em>et al.</em> 1988</a>). As shown in <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS1.pdf">Figure S1</a>, sites of high H3K9me2 enrichment are detected in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> meiotic nuclei via indirect immunofluorescence as expected given the unsynapsed X chromosomes. Meiotic germ cells comprise a substantial proportion (∼25%) of nuclei in the adult hermaphrodite (<em>e.g.</em>, see <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Qiao <em>et al.</em> 1995</a>), and therefore we expected the H3K9me2 signal on meiotic X chromosomes to add substantially to the total X chromosome reads in our ChIP-seq dataset. We performed ChIP-seq on chromatin extracts prepared from synchronized <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> adult hermaphrodites raised at the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> restrictive temperature, 22° (see the section <em>Materials and Methods</em>). We performed two biological replicates and obtained an average of ∼10.6 million unique reads mapped to the <em>C. elegans</em> genome per ChIP library for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> libraries (see the section <em>Materials and Methods</em>).</p><p id="p-25">We observed a low level of reads along the length of each chromosome and sequence enrichment (relative to input) at ∼8200 regions across the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> genome. Data are summarized in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. This H3K9me2 enrichment pattern is similar to those previously observed by the modENCODE consortium in embryos and L3 larvae (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gerstein <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2011</a>) (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS2.pdf">Figure S2</a>). The similarity of H3K9me2 distribution among developmental stages was striking, given that embryos and L3 larvae comprise mostly somatic cells whereas adults contain a large germ line. This finding suggested that either the H3K9me2 pattern is similar in germline and soma, or the level of H3K9me2 in the germ line is much lower than in the soma. H3K9me2-enriched regions tended to include repeated sequence (see <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS2.pdf">Figure S2</a>): 92% of the individual sequence reads at these enriched regions contained repetitive sequence, and 56.8% of the total sequence recovered is repetitive (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS1.pdf">Table S1</a>). In contrast, repeated sequence comprises ∼19.6% of the total genome (<a href="http://www.wormbase.org">www.wormbase.org</a> annotation WS220). This 19.6% includes all repeated sequences, ranging from simple repeats (as small as 11 nucleotides) to large repeated regions of ∼45 kb. When we examined the distribution of H3K9me2 relative to different classes of annotated repeat sequence, we found that interspersed repeats, including DNA transposons, helitrons, and retroelements, are substantially enriched for H3K9me2 marks (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS1.pdf">Table S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histone H3 lysine 9 dimethylation (H3K9me2) enrichment across the genome of fer-1 control and fer-1;him-8 adult hermaphrodites. Bars represent regions that are enriched for H3K9me2 compared with input in staged (A) fer-1 control and (B) fer-1;him-8 adults. The fer-1(ts) allele impairs spermatogenesis and therefore prevents production of embryos; the him-8 allele impairs X chromosome pairing and synapsis in XX animals. X-axis reflects chromosome length; Y-axis reflects the relative amount of H3K9me2 detected at each position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histone H3 lysine 9 dimethylation (H3K9me2) enrichment across the genome of &lt;em&gt;fer-1&lt;/em&gt; control and &lt;em&gt;fer-1;him-8&lt;/em&gt; adult hermaphrodites. Bars represent regions that are enriched for H3K9me2 compared with input in staged (A) &lt;em&gt;fer-1&lt;/em&gt; control and (B) &lt;em&gt;fer-1;him-8&lt;/em&gt; adults. The &lt;em&gt;fer-1(ts)&lt;/em&gt; allele impairs spermatogenesis and therefore prevents production of embryos; the &lt;em&gt;him-8&lt;/em&gt; allele impairs X chromosome pairing and synapsis in XX animals. X-axis reflects chromosome length; Y-axis reflects the relative amount of H3K9me2 detected at each position.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F1.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/9/1865/F1.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466622" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Histone H3 lysine 9 dimethylation (H3K9me2) enrichment across the genome of <em>fer-1</em> control and <em>fer-1;him-8</em> adult hermaphrodites. Bars represent regions that are enriched for H3K9me2 compared with input in staged (A) <em>fer-1</em> control and (B) <em>fer-1;him-8</em> adults. The <em>fer-1(ts)</em> allele impairs spermatogenesis and therefore prevents production of embryos; the <em>him-8</em> allele impairs X chromosome pairing and synapsis in XX animals. X-axis reflects chromosome length; Y-axis reflects the relative amount of H3K9me2 detected at each position.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Analysis of H3K9me2 distribution relative to gene features indicates overall enrichment in introns and depletion in exons, reflecting the presence of interspersed repeats in many introns and their absence from exons (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS2.pdf">Figure S2</a>). H3K9me2-enriched regions are underrepresented on the X chromosome relative to autosomes, as illustrated in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> where the proportion of total H3K9me2-enriched sequence in the genome is represented on a per chromosome basis (see the section <em>Materials and Methods</em>). This enrichment pattern may reflect (at least in part) the relative paucity of repetitive sequences on the X: 12.9% of X chromosomal sequences are repetitive compared with 18.9–24% of autosomal sequences (data from wormbase.org).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of histone H3 lysine 9 dimethylation (H3K9me2) chromatin immunoprecipitation&#xA0;sequencing reads relative to genome features. Pie charts indicate the percentages of H3K9me2-enriched regions that are located in the (predicted) promoter regions, 5&#x2032; untranslated regions (UTRs), coding exons, introns, 3&#x2032; UTRs, and intergenic regions. We note that promoter and 5&#x2032; UTR regions are as predicted in the current C. elegans genome database (WS 220) and in many cases are only approximate. Input is included for comparison. Overall, H3K9me2 is enriched in introns and depleted in exons with respect to input; other regions are similar to input. Each category is mutually exclusive." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of histone H3 lysine 9 dimethylation (H3K9me2) chromatin immunoprecipitation&#xA0;sequencing reads relative to genome features. Pie charts indicate the percentages of H3K9me2-enriched regions that are located in the (predicted) promoter regions, 5&#x2032; untranslated regions (UTRs), coding exons, introns, 3&#x2032; UTRs, and intergenic regions. We note that promoter and 5&#x2032; UTR regions are as predicted in the current &lt;em&gt;C. elegans&lt;/em&gt; genome database (WS 220) and in many cases are only approximate. Input is included for comparison. Overall, H3K9me2 is enriched in introns and depleted in exons with respect to input; other regions are similar to input. Each category is mutually exclusive.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F2.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/9/1865/F2.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466731" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Distribution of histone H3 lysine 9 dimethylation (H3K9me2) chromatin immunoprecipitation sequencing reads relative to genome features. Pie charts indicate the percentages of H3K9me2-enriched regions that are located in the (predicted) promoter regions, 5′ untranslated regions (UTRs), coding exons, introns, 3′ UTRs, and intergenic regions. We note that promoter and 5′ UTR regions are as predicted in the current <em>C. elegans</em> genome database (WS 220) and in many cases are only approximate. Input is included for comparison. Overall, H3K9me2 is enriched in introns and depleted in exons with respect to input; other regions are similar to input. Each category is mutually exclusive.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="X-chromosomal distribution of histone H3 lysine 9 dimethylation (H3K9me2) enrichment is elevated in fer-1;him-8. The total length of H3K9me2-enriched DNA in the genome was calculated, and the proportion of this enriched sequence present on each chromosome was determined. In the fer-1 control, 6.5% of the enriched sequence was on the X. In fer-1;him-8, 7.7% of total enriched sequence was on the X, an increase of 18% compared with control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;X-chromosomal distribution of histone H3 lysine 9 dimethylation (H3K9me2) enrichment is elevated in &lt;em&gt;fer-1;him-8&lt;/em&gt;. The total length of H3K9me2-enriched DNA in the genome was calculated, and the proportion of this enriched sequence present on each chromosome was determined. In the &lt;em&gt;fer-1&lt;/em&gt; control, 6.5% of the enriched sequence was on the X. In &lt;em&gt;fer-1;him-8&lt;/em&gt;, 7.7% of total enriched sequence was on the X, an increase of 18% compared with control.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F3.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/9/1865/F3.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466610" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">X-chromosomal distribution of histone H3 lysine 9 dimethylation (H3K9me2) enrichment is elevated in <em>fer-1;him-8</em>. The total length of H3K9me2-enriched DNA in the genome was calculated, and the proportion of this enriched sequence present on each chromosome was determined. In the <em>fer-1</em> control, 6.5% of the enriched sequence was on the X. In <em>fer-1;him-8</em>, 7.7% of total enriched sequence was on the X, an increase of 18% compared with control.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">When we compared the H3K9me2 distribution in control and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants, we observed a similar general distribution of marks across the genome and correlation with repetitive sequences (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS2.pdf">Figure S2</a>). In the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> dataset, 92% of enriched regions contained at least one repetitive sequence, and 53.3% of the enriched sequence was repetitive (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS1.pdf">Table S1</a>). When we compared the <em>total</em> number of sequence reads on the X chromosome with the average number of total sequence reads on the autosomes (see the section <em>Materials and Methods</em>), we detected a 20% increase in total X chromosomal reads, relative to autosomal reads, in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> adult hermaphrodites compared with controls (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). These reads are distributed broadly across the chromosome. When we specifically compared regions that are <em>enriched</em> for H3K9me2 on a per-chromosome basis, we see that the <em>proportion</em> of the total H3K9me2-enriched area on the X chromosome is 18% greater in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> relative to controls, whereas the proportion of H3K9me2-enriched area on the autosomes decreases (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Examining individual autosomes, the proportion of total H3K9me2-enriched area decreases or remains nearly unchanged on each autosome except for chromosome II; chromosome II behaved anomalously, with an ∼5% increase in the proportion of H3K9me2 coverage relative to controls (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Overall, these data indicate an increase in X-linked H3K9me2 marks in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> hermaphrodites.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Total histone H3 lysine 9 dimethylation (H3K9me2)-associated X-chromosome sequence read count is elevated in fer-1;him-8. Plots represent the H3K9me2 signal on each individual chromosome relative to the average autosomal signal. Plotted data have been corrected for biased input using the BEADS method (Cheung et al. 2011). Each box represents z-scores ranging from the 25th to 75th percentile; line indicates the mean value; whiskers indicate the 2.5th and 97.5th percentiles. Average number of X chromosomal reads is greater and variation is reduced in fer-1;him-8 relative to fer-1 controls." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Total histone H3 lysine 9 dimethylation (H3K9me2)-associated X-chromosome sequence read count is elevated in &lt;em&gt;fer-1;him-8&lt;/em&gt;. Plots represent the H3K9me2 signal on each individual chromosome relative to the average autosomal signal. Plotted data have been corrected for biased input using the BEADS method (Cheung &lt;em&gt;et al.&lt;/em&gt; 2011). Each box represents z-scores ranging from the 25th to 75th percentile; line indicates the mean value; whiskers indicate the 2.5th and 97.5th percentiles. Average number of X chromosomal reads is greater and variation is reduced in &lt;em&gt;fer-1;him-8&lt;/em&gt; relative to &lt;em&gt;fer-1&lt;/em&gt; controls.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F4.medium.gif" width="440" height="171"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/9/1865/F4.medium.gif" width="440" height="171"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466690" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Total histone H3 lysine 9 dimethylation (H3K9me2)-associated X-chromosome sequence read count is elevated in <em>fer-1;him-8</em>. Plots represent the H3K9me2 signal on each individual chromosome relative to the average autosomal signal. Plotted data have been corrected for biased input using the BEADS method (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cheung <em>et al.</em> 2011</a>). Each box represents z-scores ranging from the 25th to 75th percentile; line indicates the mean value; whiskers indicate the 2.5th and 97.5th percentiles. Average number of X chromosomal reads is greater and variation is reduced in <em>fer-1;him-8</em> relative to <em>fer-1</em> controls.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Although the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> datasets are similar, we observed differential enrichment at many sites. Overall, ∼99.9% of the enriched sites present in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> were also enriched in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em>. We were surprised to observe a statistically significant increase in enrichment at approximately 50% of these sites distributed across the genome (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). A statistically significant reduction in enrichment was observed at approximately 25% of the sites, and enrichment at the remainder of the sites was not significantly different in the two data sets (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). We also identified a small number of <em>de novo</em> sites that are present specifically in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em>. To do so, we first identified sites that might be unique to <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> or <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> using MACS2.1 (see the section <em>Materials and Methods</em>), and then visually inspected all sites that had been called as unique to one genotype to validate the peak calling. This approach identified a relatively small set of 348 <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em>−specific sites distributed across the genome (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). These <em>de novo</em> sites tend to be associated with repetitive sequences, although to a slightly lower degree than the conserved sites: 85% of the 348 <em>de novo</em> regions include at least one repetitive sequence, and repetitive sequences comprise 41% of the <em>de novo</em> enrichment. The net effect of the observed differential H3K9me2 deposition is a general increase in H3K9me2 level in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em>. Taken together, our data are consistent with a global increase in H3K9me2 across the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> hermaphrodite genome, and a further enhanced increase on the unsynapsed X chromosomes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of differential histone H3 lysine 9 dimethylation (H3K9me2) accumulation in the fer-1 vs. fer-1;him-8 genome. Tracks represent fer-1 control H3K9me2, fer-1;him-8 H3K9me2, differential sites, input reads, and annotated genes (Refseq). For H3K9me2 tracks, the average of two independent biological replicates is shown as normalized total reads. Representative genome browser tracts show differential sites that (A) have a greater signal in fer-1;him-8 than in fer-1 or (B) have a de novo him-8&#x2212;specific site of enrichment. Differential regions are boxed. Y-axis scale reflects the number of reads, ranging from a minimum of 0 to a maximum of &#x2265; 120." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Examples of differential histone H3 lysine 9 dimethylation (H3K9me2) accumulation in the &lt;em&gt;fer-1&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;fer-1;him-8&lt;/em&gt; genome. Tracks represent &lt;em&gt;fer-1&lt;/em&gt; control H3K9me2, &lt;em&gt;fer-1;him-8&lt;/em&gt; H3K9me2, differential sites, input reads, and annotated genes (Refseq). For H3K9me2 tracks, the average of two independent biological replicates is shown as normalized total reads. Representative genome browser tracts show differential sites that (A) have a greater signal in &lt;em&gt;fer-1;him-8&lt;/em&gt; than in &lt;em&gt;fer-1&lt;/em&gt; or (B) have a &lt;em&gt;de novo him-8&#x2212;&lt;/em&gt;specific site of enrichment. Differential regions are boxed. Y-axis scale reflects the number of reads, ranging from a minimum of 0 to a maximum of &#x2265; 120.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F5.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/9/1865/F5.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466681" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Examples of differential histone H3 lysine 9 dimethylation (H3K9me2) accumulation in the <em>fer-1</em> <em>vs.</em> <em>fer-1;him-8</em> genome. Tracks represent <em>fer-1</em> control H3K9me2, <em>fer-1;him-8</em> H3K9me2, differential sites, input reads, and annotated genes (Refseq). For H3K9me2 tracks, the average of two independent biological replicates is shown as normalized total reads. Representative genome browser tracts show differential sites that (A) have a greater signal in <em>fer-1;him-8</em> than in <em>fer-1</em> or (B) have a <em>de novo him-8−</em>specific site of enrichment. Differential regions are boxed. Y-axis scale reflects the number of reads, ranging from a minimum of 0 to a maximum of ≥ 120.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Evaluation of H3K9me2 levels in gonads <em>vs.</em> whole animals</h3><p id="p-34">To validate our ChIP-seq data analysis, we evaluated H3K9me2 levels in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> adults and isolated adult gonads by performing protein blots. Using an antibody reported to detect H3K9me2 reliably on protein blots (Antibody Validation Database, <a href="http://compbio.med.harvard.edu/antibodies/">http://compbio.med.harvard.edu/antibodies/</a>), we detected a greater level of H3K9me2 relative to pan-H3 in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> whole worms and dissected gonads than in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> control whole worms and dissected gonads (∼114% and ∼107%, respectively; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). These results confirm the ChIP-seq finding that H3K9me2 levels are elevated in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histone H3 lysine 9 dimethylation (H3K9me2) abundance is elevated in him-8 and low in the germline relative to the soma. (A, C) Representative protein blots probed with pan-H3 antibody and antibodies against several modified forms of histone H3. The number of whole adult hermaphrodites or dissected adult hermaphrodite gonad arms used to prepare extract for each lane is indicated. As the C. elegans gonad is primarily germ line, most of the histone H3 in gonad extracts is from germ cells (see Impact of H3K9me2 loss on the gonad transcriptome). Aliquots of a single extract were probed for total H3 and an individual H3 modification. Signal was quantified and the ratio of H3 modification to pan-H3 signal was calculated to determine the normalized value for each modification. (B) Histogram represents the ratio of normalized H3K9me2 signal in him-8 relative to wild-type (N2) whole animals and isolated gonads. H3K9me2 signal is greater in him-8 whole animals and gonads. (D) Histogram represents the ratio of normalized signal in isolated gonads relative to whole animals. Note that the Y-axis scale is different in B and D. Data represent three biological replicates for H3K4me1 and two biological replicates for each other mark (see the section Materials and Methods). Error bars, SEM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histone H3 lysine 9 dimethylation (H3K9me2) abundance is elevated in &lt;em&gt;him-8&lt;/em&gt; and low in the germline relative to the soma. (A, C) Representative protein blots probed with pan-H3 antibody and antibodies against several modified forms of histone H3. The number of whole adult hermaphrodites or dissected adult hermaphrodite gonad arms used to prepare extract for each lane is indicated. As the &lt;em&gt;C. elegans&lt;/em&gt; gonad is primarily germ line, most of the histone H3 in gonad extracts is from germ cells (see &lt;em&gt;Impact of H3K9me2 loss on the gonad transcriptome&lt;/em&gt;). Aliquots of a single extract were probed for total H3 and an individual H3 modification. Signal was quantified and the ratio of H3 modification to pan-H3 signal was calculated to determine the normalized value for each modification. (B) Histogram represents the ratio of normalized H3K9me2 signal in &lt;em&gt;him-8&lt;/em&gt; relative to wild-type (N2) whole animals and isolated gonads. H3K9me2 signal is greater in &lt;em&gt;him-8&lt;/em&gt; whole animals and gonads. (D) Histogram represents the ratio of normalized signal in isolated gonads relative to whole animals. Note that the Y-axis scale is different in B and D. Data represent three biological replicates for H3K4me1 and two biological replicates for each other mark (see the section &lt;em&gt;Materials and Methods&lt;/em&gt;). Error bars, SEM.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F6.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/9/1865/F6.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466724" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">Histone H3 lysine 9 dimethylation (H3K9me2) abundance is elevated in <em>him-8</em> and low in the germline relative to the soma. (A, C) Representative protein blots probed with pan-H3 antibody and antibodies against several modified forms of histone H3. The number of whole adult hermaphrodites or dissected adult hermaphrodite gonad arms used to prepare extract for each lane is indicated. As the <em>C. elegans</em> gonad is primarily germ line, most of the histone H3 in gonad extracts is from germ cells (see <em>Impact of H3K9me2 loss on the gonad transcriptome</em>). Aliquots of a single extract were probed for total H3 and an individual H3 modification. Signal was quantified and the ratio of H3 modification to pan-H3 signal was calculated to determine the normalized value for each modification. (B) Histogram represents the ratio of normalized H3K9me2 signal in <em>him-8</em> relative to wild-type (N2) whole animals and isolated gonads. H3K9me2 signal is greater in <em>him-8</em> whole animals and gonads. (D) Histogram represents the ratio of normalized signal in isolated gonads relative to whole animals. Note that the Y-axis scale is different in B and D. Data represent three biological replicates for H3K4me1 and two biological replicates for each other mark (see the section <em>Materials and Methods</em>). Error bars, SEM.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The H3K9me2 enrichment detected on <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> X chromosomes was less pronounced than we had expected <em>a priori</em> based on indirect immunofluorescence studies. We considered that the moderate X chromosomal H3K9me2 enrichment in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> might indicate that H3K9me2 marks are under-represented in germline chromatin. In this case, germ line contribution to the ChIP-seq signal would be lower than expected based on the proportion of meiotic nuclei present in adults. We analyzed the protein blot data to assess whether H3K9me2 marks were less prevalent in gonads than in whole animals. For both <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> animals, we detected H3K9me2 at a lower level relative to pan-H3 in isolated gonads than in whole animals (∼20%; <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). Hence, this mark is underrepresented in the germline compared to the soma. Given this result, it appears that only ∼20% of the H3K9me2 signal detected by H3K9me2 ChIP-seq is germline. Therefore, if the 20% increase in X-linked reads we detected in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants results primarily from an increase in the germ line, then this would represent a substantial increase in germline X-linked H3K9me2. A low abundance of H3K9me2 in the germ line is consistent with recent modENCODE data examining H3K9me2 in germ cell nuclei isolated from young adult hermaphrodites (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> 2014</a>); these data indicate a relatively even distribution of H3K9me2 marks across the genome compared with whole-animal data (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-37">We included a <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> (null) mutant as a negative control in our assays. Despite using a highly specific antibody, we detected a low H3K9me2 signal in protein extracts prepared from <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> whole worms (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>) and variably detected an extremely low signal in extracts prepared from dissected gonads (data not shown). As previously stated, H3K9me2 is not detected in gonads via indirect immunofluorescence (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bessler <em>et al.</em> 2010</a>). Given these apparently conflicting results, we hypothesize that either <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> mutants contain residual H3K9me2 not detected by IF or the anti-H3K9me2 antibody cross-reacts weakly on protein blots with other marks, <em>e.g.</em>, H3K9me3, that are unaffected in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> mutants. We note that <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> embryos have been reported to contain a residual level of H3K9me2 as detected by mass spectrometry (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Towbin <em>et al.</em> 2012</a>).</p><p id="p-38">We evaluated several other histone H3 modifications and observed that each is underrepresented in the germline relative to whole animals, although to widely varying degrees (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). These findings reveal two aspects of chromatin regulation in the <em>C. elegans</em> germ line: the abundance of certain histone modifications is (much) lower than in the soma overall, and the ratio of different modifications to each other is very different than in the soma overall. Whole-animal data obviously represent an averaging of signals from numerous tissues, and our results emphasize this point for the germ line.</p><p id="p-39">As a control, we considered that tissue dissection might trigger the removal of N-terminal histone tails, thus reducing our ability to detect certain marks. To investigate this idea, we performed a series of controls where we dissected adults, let the tissues sit in buffer 0, 1, 2, or 3 hr, and then analyzed the dissected tissue via protein blot. The histone signal did not decrease over time, and cleavage products did not appear, therefore we conclude that the low signal in dissected gonads is not an artifact of the dissection process (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS4.pdf">Figure S4</a>).</p></div><div id="sec-12" class="subsection"><h3>Impact of H3K9me2 loss on the gonad transcriptome</h3><p id="p-40">To evaluate the effect of H3K9me2 on the germline transcriptome, we isolated mRNA from <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> adult hermaphrodite gonads and performed deep sequencing (mRNA-seq; see the section <em>Materials and Methods</em>). We estimate that 90–95% of the nuclei in the dissected gonad arms are germline (∼325 germ cells per gonad arm <em>vs.</em> 10 somatic sheath cells and ≤24 somatic spermathecal cells; see <a href="http://www.wormatlas.org">www.wormatlas.org</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Qiao <em>et al.</em> 1995</a>). In addition, the hermaphrodite germ line generates high levels of maternal mRNA for inclusion in oocytes, which will increase the germline contribution to the steady state transcript level in the gonad. Therefore, gonad mRNA-seq data primarily reflect the germline transcriptome. We note that <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> mutants have a mortal germline (<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000966;class=Phenotype">Mrt</a>) phenotype: when maintained as homozygotes for many generations, they progressively lose their germline and eventually become sterile (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen and Horvitz 2007</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bessler <em>et al.</em> 2010</a>). In our experiments, we were interested in examining the effect of H3K9me2 loss over the short term; therefore, we used <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> animals recently derived from a balanced strain, which were fertile and did not exhibit a severely reduced germ line. Two biological replicates were performed wherein mRNA was isolated from separate sets of dissected gonads and used to generate libraries for sequencing (see the section <em>Materials and Methods</em>).</p><p id="p-41">We compared the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> gonad transcriptomes to each other. We calculated the RPKM (<span class="underline">R</span>eads <span class="underline">P</span>er <span class="underline">K</span>ilobase of transcript per <span class="underline">M</span>illion mapped reads) for each transcript in our mRNA-seq datasets. For purposes of comparing the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> transcriptomes, we considered transcripts present at a relatively permissive cut-off value of &gt;0.5 RPKM. A transcript was defined as present at a significantly different level in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> <em>vs.</em> <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> if the Cuffdiff q-value was &lt;0.05 (see the section <em>Materials and Methods</em>). The vast majority of transcripts were present at statistically similar levels in both datasets; however we did identify a set of 181 differentially expressed transcripts (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) that fall into two classes. Transcripts from 108 genes were detected in only one genotype (RPKM = 0 in the other genotype); we refer to these as genotype-specific transcripts. Transcripts from 73 genes were detected in both genotypes, but at significantly different levels; we refer to these as genotype-regulated transcripts. The log<sub>2</sub> fold-change of genotype-regulated transcripts ranged from 5.3 (40-fold) to 0.96 (1.9-fold). These 181 differentially regulated genes are distributed across the genome (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7D</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effect of histone H3 lysine 9 dimethylation (H3K9me2) loss on the adult hermaphrodite gonad transcriptome. (A) Scatter plot represents the transcriptome data from him-8 gonads vs. met-2;him-8 gonads. Transcripts present at a statistically different level are highlighted in red if more highly expressed in met-2;him-8 and blue if more highly expressed in him-8. X- and Y-axis scale is expressed in RPKM (see text). (B) The table lists the number of genes on each chromosome with transcripts that are differentially abundant in met-2; him-8 vs. him-8 gonads. These represent the highlighted transcripts in (A). (C) Screen shots of representative differential transcripts: clec-85 transcripts are elevated in met-2;him-8 and egl-30 transcripts are reduced in met-2;him-8. Transcripts from adjacent genes, Y54G2A.16 and emr-1, are statistically unchanged. (D) Chromosomal distribution of genes whose transcripts are differentially abundant in met-2;him-8 vs. him-8. Lines extending upward indicate transcripts that are more abundant in met-2; him-8, and lines extending downward indicate transcripts that are less abundant in met-2;him-8. Longer lines represent &#x201C;genotype-specific&#x201D; genes whose transcripts are detected in one genotype and not the other. Shorter lines represent genotype-regulated genes whose transcripts are more abundant in one genotype than the other; their length is proportional to the log2 value of the fold-change." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The effect of histone H3 lysine 9 dimethylation (H3K9me2) loss on the adult hermaphrodite gonad transcriptome. (A) Scatter plot represents the transcriptome data from &lt;em&gt;him-8&lt;/em&gt; gonads &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;met-2;him-8&lt;/em&gt; gonads. Transcripts present at a statistically different level are highlighted in red if more highly expressed in &lt;em&gt;met-2;him-8&lt;/em&gt; and blue if more highly expressed in &lt;em&gt;him-8&lt;/em&gt;. X- and Y-axis scale is expressed in RPKM (see text). (B) The table lists the number of genes on each chromosome with transcripts that are differentially abundant in &lt;em&gt;met-2&lt;/em&gt;; &lt;em&gt;him-8&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;him-8&lt;/em&gt; gonads. These represent the highlighted transcripts in (A). (C) Screen shots of representative differential transcripts: &lt;em&gt;clec-85&lt;/em&gt; transcripts are elevated in &lt;em&gt;met-2;him-8&lt;/em&gt; and &lt;em&gt;egl-30&lt;/em&gt; transcripts are reduced in &lt;em&gt;met-2;him-8&lt;/em&gt;. Transcripts from adjacent genes, Y54G2A.16 and &lt;em&gt;emr-1&lt;/em&gt;, are statistically unchanged. (D) Chromosomal distribution of genes whose transcripts are differentially abundant in &lt;em&gt;met-2;him-8&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;him-8&lt;/em&gt;. Lines extending upward indicate transcripts that are more abundant in &lt;em&gt;met-2&lt;/em&gt;; &lt;em&gt;him-8&lt;/em&gt;, and lines extending downward indicate transcripts that are less abundant in &lt;em&gt;met-2;him-8&lt;/em&gt;. Longer lines represent &#x201C;genotype-specific&#x201D; genes whose transcripts are detected in one genotype and not the other. Shorter lines represent genotype-regulated genes whose transcripts are more abundant in one genotype than the other; their length is proportional to the log&lt;sub&gt;2&lt;/sub&gt; value of the fold-change.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F7.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/9/1865/F7.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466619" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-42" class="first-child">The effect of histone H3 lysine 9 dimethylation (H3K9me2) loss on the adult hermaphrodite gonad transcriptome. (A) Scatter plot represents the transcriptome data from <em>him-8</em> gonads <em>vs.</em> <em>met-2;him-8</em> gonads. Transcripts present at a statistically different level are highlighted in red if more highly expressed in <em>met-2;him-8</em> and blue if more highly expressed in <em>him-8</em>. X- and Y-axis scale is expressed in RPKM (see text). (B) The table lists the number of genes on each chromosome with transcripts that are differentially abundant in <em>met-2</em>; <em>him-8</em> <em>vs.</em> <em>him-8</em> gonads. These represent the highlighted transcripts in (A). (C) Screen shots of representative differential transcripts: <em>clec-85</em> transcripts are elevated in <em>met-2;him-8</em> and <em>egl-30</em> transcripts are reduced in <em>met-2;him-8</em>. Transcripts from adjacent genes, Y54G2A.16 and <em>emr-1</em>, are statistically unchanged. (D) Chromosomal distribution of genes whose transcripts are differentially abundant in <em>met-2;him-8</em> <em>vs.</em> <em>him-8</em>. Lines extending upward indicate transcripts that are more abundant in <em>met-2</em>; <em>him-8</em>, and lines extending downward indicate transcripts that are less abundant in <em>met-2;him-8</em>. Longer lines represent “genotype-specific” genes whose transcripts are detected in one genotype and not the other. Shorter lines represent genotype-regulated genes whose transcripts are more abundant in one genotype than the other; their length is proportional to the log<sub>2</sub> value of the fold-change.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We compared our gonad transcriptome data with the results of a previously published serial analysis of gene expression (SAGE) study of gonad-expressed mRNA (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2009</a>). In this study, soma-expressed transcripts were identified by comparison with <em><a href="http://www.wormbase.org/db/get?name=WBGene00001611;class=Gene">glp-4</a></em> germ cell-less mutants; transcripts absent from the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001611;class=Gene">glp-4</a></em> mutants were deemed germline-specific. This approach identified transcripts from 4699 genes as present in at least one copy in 100,000 reads; using an average transcript size of 1.3 kb, this means the low detection cut-off was approximately equivalent to an average RPKM of 7.7. Our <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> dataset contains 6869 genes with an RPKM of &gt;7.7. As expected, these 6869 genes include &gt;91% of the total and 99.9% of the germline-specific genes identified in the SAGE analysis. [Transcripts from the remaining germline-expressed genes identified in <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> (2009)</a> were also expressed in our dataset, but at an RPKM of &lt;7.7.] Among the 181 transcripts we identified as differentially regulated in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> <em>vs.</em> <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>;him</em>-8, 40 were also identified in the SAGE dataset, and 17 of these were identified as germline-specific. The other 141 genes in are expressed at a low level (RPKM &lt;5) in our data, which is presumably why they are absent from the SAGE dataset.</p><p id="p-44">We also examined the identity of differentially expressed genes with respect to known and hypothesized gene function by performing Gene Ontology (GO) analysis. Using the DAVID algorithm (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2009</a>), we found that a large proportion of the differentially expressed genes (123/181) are not associated with a GO molecular function term. The remaining 58 genes are associated with a variety of GO molecular function terms and are not enriched for any particular term.</p></div><div id="sec-13" class="subsection"><h3>Distribution of H3K9me2 enrichment relative to germline-expressed genes</h3><p id="p-45">We performed meta-gene analysis (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Shin <em>et al.</em> 2009</a>) to evaluate the distribution of H3K9me2 marks with respect to transcript level in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> gonads. H3K9me2 levels were relatively low within a 2-kb window flanking transcription start and stop sites and higher in introns (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>) as observed previously for whole embryos and L3 larvae (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>) (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gerstein <em>et al.</em> 2010</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2011</a>). In addition, H3K9me2 levels were lowest at genes with a relatively high transcript level (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8A</a>) and were present at a similar level on genes expressed at a moderate or low level. This pattern is consistent with other studies reporting that H3K9me2 marks do not strictly correlate with transcriptional inactivity and are often present within the body of genes located within heterochromatic regions (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yasuhara and Wakimoto 2008</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> 2014</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">West <em>et al.</em> 2014</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Meta-gene analysis of the adult hermaphrodite histone H3 lysine 9 dimethylation (H3K9me2) profile across gonad-expressed genes and predicted Argonaute targets. Gene expression data are from our RNA-seq analysis of him-8 gonads. The meta-gene includes 1 kb of sequence upstream of the transcription start site (0), a 3 kb averaged coding region, and 1 kb downstream of the transcription termination site (3000). (A) The average H3K9me2 enrichment signals of the top 10%, middle 10%, and bottom 10% of expressed genes in the him-8 adult hermaphrodite gonad. (B) The average H3K9me2 enrichment of predicted CSR-1, WAGO-1, and ALG-3/4 target genes (Claycomb et al. 2009; Conine et al. 2010; Gu et al. 2009). The 96 most highly represented WAGO-1 targets, used in this analysis, are substantially enriched for H3K9me2 across the entire meta-gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1603503170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Meta-gene analysis of the adult hermaphrodite histone H3 lysine 9 dimethylation (H3K9me2) profile across gonad-expressed genes and predicted Argonaute targets. Gene expression data are from our RNA-seq analysis of &lt;em&gt;him-8&lt;/em&gt; gonads. The meta-gene includes 1 kb of sequence upstream of the transcription start site (0), a 3 kb averaged coding region, and 1 kb downstream of the transcription termination site (3000). (A) The average H3K9me2 enrichment signals of the top 10%, middle 10%, and bottom 10% of expressed genes in the &lt;em&gt;him-8&lt;/em&gt; adult hermaphrodite gonad. (B) The average H3K9me2 enrichment of predicted CSR-1, WAGO-1, and ALG-3/4 target genes (Claycomb &lt;em&gt;et al.&lt;/em&gt; 2009; Conine &lt;em&gt;et al.&lt;/em&gt; 2010; Gu &lt;em&gt;et al.&lt;/em&gt; 2009). The 96 most highly represented WAGO-1 targets, used in this analysis, are substantially enriched for H3K9me2 across the entire meta-gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1865/F8.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/9/1865/F8.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1865/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466624" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-46" class="first-child">Meta-gene analysis of the adult hermaphrodite histone H3 lysine 9 dimethylation (H3K9me2) profile across gonad-expressed genes and predicted Argonaute targets. Gene expression data are from our RNA-seq analysis of <em>him-8</em> gonads. The meta-gene includes 1 kb of sequence upstream of the transcription start site (0), a 3 kb averaged coding region, and 1 kb downstream of the transcription termination site (3000). (A) The average H3K9me2 enrichment signals of the top 10%, middle 10%, and bottom 10% of expressed genes in the <em>him-8</em> adult hermaphrodite gonad. (B) The average H3K9me2 enrichment of predicted CSR-1, WAGO-1, and ALG-3/4 target genes (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Claycomb <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Conine <em>et al.</em> 2010</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>). The 96 most highly represented WAGO-1 targets, used in this analysis, are substantially enriched for H3K9me2 across the entire meta-gene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">We hypothesized the loss of <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a> activity in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> background might disproportionally impact the level of X-linked transcripts. If H3K9me2 marks correlate with reduced transcription of nearby/adjacent genes, then we might expect to see a significant increase in transcript levels in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> mutant where those marks are absent, and this pattern might be especially obvious in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> background where H3K9me2 marks are elevated. Instead, only a minority (57 of 181, 31%) of the differentially expressed transcripts were elevated in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> mutants (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a>). This pattern is particularly striking for the genotype-specific genes: 89 of 108 (82%) of them are off in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> mutants (lacking H3K9me2) and only 19 of 108 (18%) of them are off in the control (containing H3K9me2). A total of 23 of the 181 differentially regulated transcripts are X-linked. Of these, 11 are elevated in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> background and 12 are reduced. Hence, there is no clear pattern with respect to H3K9me2 marks and germline transcript level.</p><p id="p-48">We also evaluated H3K9me2 enrichment, as detected by ChIP-seq, relative to germline-expressed genes. We first compared the H3K9me2 enrichment we detected in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> adults with the H3K9me2 enrichment in adult germ lines reported by <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> (2014)</a> to see whether the latter data set contained germline-specific H3K9me2 sites that had not been detected in our whole animal data set. To make this comparison, we analyzed the germ line data of <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> (2014)</a> in the same way we had analyzed our whole animal data. This analysis revealed ∼400 sites of H3K9me2 enrichment (as opposed to ∼8200 sites in whole animals), each of which we had detected as enriched in our <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em> whole animal dataset, and 98% of which contained repetitive sequence. These results are consistent with our finding that H3K9me2 abundance is low in the germ line. Because these data did not reveal germline-specific sites of H3K9me2 accumulation, we then used our whole genome dataset to ask whether the few <em>de novo</em> sites of H3K9me2 accumulation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> hermaphrodites correlated with genes whose transcript levels were altered in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> mutant germ cells. To do so, we compared the differential H3K9me2 peaks detected in our ChIP-seq experiments with differentially expressed genes detected in our mRNA-seq experiments by calculating the correlation coefficients between data sets (see the section <em>Materials and Methods</em>). We identified H3K9me2-enriched regions located within genes or adjacent to genes within 500 bp of the predicted transcription start site. This analysis revealed 40 H3K9me2-associated genes among the 181 genes differentially expressed genes in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> and control animals. We did not observe a correlation between <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a> activity (<em>i.e.</em>, H3K9me2) and transcript level among these genes: 16 were expressed more highly in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> animals and 24 were expressed more highly in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(+)</em> animals.</p></div><div id="sec-14" class="subsection"><h3>H3K9me2 distribution with respect to Argonaute targets</h3><p id="p-49">As described previously, H3K9me2 distribution in the germ line, as detected by IF, depends on activity of <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> Argonaute and proteins required for production of <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a>−associated 22G RNAs, <a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene">EGO-1</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene">DRH-3</a>, and <a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">EKL-1</a> (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). We were interested in evaluating the extent to which H3K9me2 marks might correlate with putative targets of Argonaute pathway activity. Consequently, we evaluated the H3K9me2 profile at putative <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a>, and <a href="http://www.wormbase.org/db/get?name=WBGene00011910;class=Gene">ALG-3</a>/4 targets as reported in the literature (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Claycomb <em>et al.</em> 2009</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Conine <em>et al.</em> 2010</a>) (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8B</a>). The H3K9me2 profile is relatively low for <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00011910;class=Gene">ALG-3</a>/4 targets, similar to levels observed for germline-expressed genes. In contrast, the H3K9me2 profile for <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> targets is notably greater overall and particularly elevated in sequences flanking the transcription start site and immediately downstream of the transcription termination site. <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> targets are reported to include repetitive elements, pseudogenes, intergenic regions, and some protein-coding genes (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>). In our meta-analysis, we used the top 96 reported <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> targets, 95 of which are protein-coding genes (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>); 67% (66/96) of these <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> target genes contain at least one repetitive region, although these regions are highly variable in size. We also analyzed supplemental data (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>) to compare transposon and other repeat sequence targets of <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> to repeat sequence targets of H3K9me2. SiRNAs corresponding to many retrotransposons and DNA transposons, but not helitrons, were identified in association with <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a>. Consequently, a subset of repeat sequences appears to be commonly targeted by <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> and H3K9me2.</p><p id="p-50">Previously, we surveyed the H3K9me2 distribution via immunolabeling of Argonaute single mutant males and observed a defect only in <em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">csr-1</a></em> mutants (<a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). H3K9me2 distribution appeared normal in males carrying a mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a></em> or any of the other individual WAGO clade genes. Interestingly, the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene">ego-1</a>(0)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">csr-1</a>(0)</em> phenotypes are not identical: in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene">ego-1</a>(0)</em> mutants, H3K9me2 foci are absent from unsynapsed chromosomes; in <em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">csr-1</a>(0)</em> mutants, weak foci are visible on unsynapsed chromosomes, and ectopic foci are visible on synapsed chromosomes (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Maine <em>et al.</em> 2005</a>; <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). <a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene">EGO-1</a> is required for biogenesis of siRNAs that associate with either <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> or the WAGOs (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>), and we wondered whether redundant WAGO activity might impact H3K9me2 distribution. We decided to evaluate meiotic H3K9me2 in males that carry MAGO12 (<a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>) to address this question. As shown in <a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/FigureS5.pdf">Figure S5</a>, H3K9me2 distribution in first meiotic prophase nuclei appears normal in MAGO12 males.</p><p id="p-51">To begin to investigate the relationship between H3K9me2 marks and <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> activity, we generated a <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a>(0)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> double mutant and evaluated the developmental phenotype. <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> single mutants are reported to develop an <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000966;class=Phenotype">Mrt</a> germline defect over 18−24 successive generations (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Andersen and Horvitz 2007</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bessler <em>et al.</em> 2010</a>). In our hands, the brood size of <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> double mutants is substantially reduced at 20° within ∼12 generations; however, the strain does not become sterile sooner than the 18−24 generations described for <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Andersen and Horvitz 2007</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bessler <em>et al.</em> 2010</a>), hence the <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000966;class=Phenotype">Mrt</a> phenotype does not appear to be enhanced.</p><p id="p-52">To evaluate developmental defects, we grew the strains at 20°, picked mid-L4 stage hermaphrodites to fresh plates, and DAPI-stained them 48 hr later. As previously reported, <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> mutants contain a variety of germline defects at a very low level (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Bessler <em>et al.</em> 2010</a>). The most common germline defects we observed in our <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> strain were (1) presence of endomitotic oocytes (an <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000668;class=Phenotype">Emo</a> phenotype), (2) the presence of clumped meiotic nuclei, and (3) the presence of some morphologically abnormal (typically small) nuclei in the meiotic region. We observed these defects at a very low level in our <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a></em> strain (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS2.pdf">Table S2</a>). The frequency of the <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000668;class=Phenotype">Emo</a> phenotype was reduced in <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> compared with <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em>, whereas the frequencies of nuclear clumping and morphologically abnormal nuclei were increased additively in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> double mutant compared with each single mutant. These results suggest <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> may act in parallel to impact some aspects of germ line development.</p><p id="p-53">The most prominent somatic phenotypic we observed is an egg-laying defect (animals filled with developing embryos) detected at a low level in all three strains (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS2.pdf">Table S2</a>). This defect was observed at the L4 + 48-hr time point, but not at the earlier L4 + 24-hr time point. Animals were visually examined on the culture plate and after DAPI staining and judged to have an <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000640;class=Phenotype">Egl</a> defect if at least a double row of embryos was observed in the uterus. The phenotype is more penetrant in <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> double mutants than in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a></em> single mutants (<a href="http://www.g3journal.org/content/suppl/2015/07/08/g3.115.019828.DC1/TableS2.pdf">Table S2</a>); the effects appear to be additive, suggesting that <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> activity act in parallel to impact egg laying.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-54">Our studies describe and compare the genome-wide distribution of H3K9me2 in adult <em>C. elegans</em> hermaphrodites with synapsed (<em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a></em>) <em>vs.</em> unsynapsed (<em><a href="http://www.wormbase.org/db/get?name=WBGene00001414;class=Gene">fer-1</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em>) X chromosomes. Immunolabeling assays consistently detect elevated H3K9me2 on unsynapsed chromosomes in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> hermaphrodites, and we performed ChIP-seq to distinguish whether the H3K9me2 enrichment accumulates at <em>de novo</em> sites on the unsynapsed meiotic X or to an elevated degree at sites normally present on synapsed chromosomes. Our data reveal that H3K9me2 is enriched primarily at existing sites, and at very few <em>de novo</em> sites, in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants. This X-enriched signal is relatively modest, most likely because H3K9me2 is underrepresented in germ cells compared with whole animals as detected in our protein blot experiments. Nonetheless, we interpret the increased X chromosome H3K9me2 signal as an elevated abundance of H3K9me2 on meiotic X chromosomes and a redistribution of <a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">MET-2</a> activity toward the unsynapsed Xs. Unexpectedly, in addition to detecting an increase in the proportion of H3K9me2 associated with the X chromosome in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants, we also observed a net increase in H3K9me2 abundance at sites distributed across the genome. Moreover, this increase is not specific to the germ line, but instead present in (at least some) somatic tissues, as well.</p><p id="p-55">To compare H3K9me2 levels on a set of chromosomes when synapsed <em>vs.</em> non-synapsed, we necessarily evaluated an example of MSUC. We chose to use <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> and evaluate the X chromosome because this is the best-studied situation in the literature. However, these choices raise two points. First, the effect of MSCI might be stronger than MSUC, both in terms of H3K9me2 abundance and effects on the transcriptome. It is possible that MSCI, as a naturally occurring process, causes a more robust increase in H3K9me2 level and has a more profound impact on the transcriptome than does MSUC, which arises in due to mutation or chromosomal duplication. Second, X-linked gene expression is limited in the <em>C. elegans</em> germ line relative to autosomal gene expression (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kelly <em>et al.</em> 2002</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Reinke <em>et al.</em> 2000</a>). It is possible that MSUC triggered by unsynapsed autosomes, such as chromosome V in <em><a href="http://www.wormbase.org/db/get?name=WBGene00011600;class=Gene">zim-2</a></em> mutants, would have a greater impact on the germline transcriptome. However, <em>zim</em> mutants are fertile and have essentially normal germline development (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Phillips and Dernburg 2006</a>), therefore it seems unlikely that transcript levels are radically altered in those mutant backgrounds.</p><p id="p-56">Another consideration is the residual H3K9me2 present in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> mutants. Analysis of H3K9me2 in embryos via mass spectrometry detected a low abundance of H3K9me2 in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> embryo extracts, a result is consistent with our protein blot data reported here (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Towbin <em>et al.</em> 2012</a>). Therefore, our analysis of <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> provides information about the transcriptome in germ lines with a very low abundance of H3K9me2 marks. If randomly distributed, these residual marks might be expected to have minimal impact. However, their distribution is not known. These other marks appear to depend on <a href="http://www.wormbase.org/db/get?name=WBGene00012802;class=Gene">SET-25</a> activity, as H3K9me2 was not detected in <em><a href="http://www.wormbase.org/db/get?name=WBGene00012802;class=Gene">set-25</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> double mutants (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Towbin <em>et al.</em> 2012</a>). <a href="http://www.wormbase.org/db/get?name=WBGene00012802;class=Gene">SET-25</a> also is responsible for H3K9me3, as these marks are eliminated in <em><a href="http://www.wormbase.org/db/get?name=WBGene00012802;class=Gene">set-25</a></em> mutants. One consequence of this dual activity is that we do not have a means to eliminate H3K9me2 marks entirely without also eliminating H3K9me3, and therefore we cannot easily test the impact of the <a href="http://www.wormbase.org/db/get?name=WBGene00012802;class=Gene">SET-25</a>-dependent H3K9me2 on the germline transcriptome.</p><p id="p-57">Although primarily studied for its role in meiosis, <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> is known to interact genetically with a number of somatically expressed transcriptional regulators (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nelms and Hanna-Rose 2006</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sun <em>et al.</em> 2007</a>). It is possible that <a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">HIM-8</a> impacts H3K9me2 level by modulating activity of one or more transcriptional regulators that, in turn, modulate expression of histone-modifying enzymes. As <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> mutants develop normally for the most part, we speculate that the elevated H3K9me2 level may be balanced out by elevated levels of other histone modifications, as well.</p><p id="p-58">We find that germline abundance of H3K9me2 and several other histone marks is low relative to abundance in the soma overall. We also observed that the relative level of histone H3 modifications in germline <em>vs.</em> soma varies widely, although each of the marks we assayed is underrepresented in adult germ lines relative to adult soma. Although it is to be expected that histone marks might vary among tissues and cells, our observations are important for interpreting ChIP data in that they indicate the germline signal varies widely depending on the mark being assayed. The low H3K9me2 abundance in <em>C. elegans</em> is reminiscent of the situation in mouse, where primordial germ cells lose H3K9me2 (and other) marks as they undergo epigenetic reprogramming (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Seki <em>et al.</em> 2005</a>). The abundance of H3K9me2 marks remains low in adult testes compared with somatic gonadal cells (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Deguchi <em>et al.</em> 2013</a>). Similarly sweeping changes in histone modification are observed in the <em>C. elegans</em> germ lineage (P4 and its daughters, Z2 and Z3) as it forms in the embryo (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wang and Seydoux 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kelly 2014</a>). The abundance of many marks, including H3K9me2 and others tested here, is low in Z2 and Z3 compared with somatic embryonic nuclei (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schaner and Kelly 2006</a>). Subsequent changes in histone modification pattern are observed as the germline develops and produces gametes, with dramatic increases in many histone activation marks (reviewed by <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Van Wynsberghe and Maine 2013</a>). The general paucity of histone H3 modifications that we detect in the adult germline may reflect on-going mitosis and differentiation in the adult germline, which is absent in the adult soma, and gamete-specific patterns of chromatin regulation important for meiotic events and/or early embryogenesis.</p><p id="p-59">The pattern of H3K9me2 accumulation across the adult XX genome correlates positively with reported <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> Argonaute targets and negatively with reported <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> Argonaute targets (<a id="xref-ref-19-9" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Claycomb <em>et al.</em> 2009</a>). A negative correlation with <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> targets is consistent with our earlier observation of ectopic H3K9me2 accumulation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">csr-1</a></em>−null mutants (<a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>). In other words, <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> activity may <em>exclude</em> deposition of H3K9me2 marks at actively expressed chromatin regions. Moreover, current thinking is that <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> functions in a mechanism to distinguish appropriate from inappropriate gene expression in the germ line (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Seth <em>et al.</em> 2013</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Wedeles <em>et al.</em> 2013</a>). In this sense, <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene">CSR-1</a> is thought to be a positive regulator of germline gene expression. Positive correlation of H3K9me2 with <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> targets suggests H3K9me2 and <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> may act together or in parallel to regulate a common set of targets. In our analysis, we used the top 96 <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> targets identified based on recovery of 22G siRNAs by <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> co-IP; 95 of these targets are protein-coding genes (<a id="xref-ref-19-10" class="xref-bibr" href="#ref-19">Gu <em>et al.</em> 2009</a>). Therefore, <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a> and H3K9me2 appear to have some common protein-coding gene targets. We also found that H3K9me2, like <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a>, preferentially modified retrotransposons and DNA transposons, although H3K9me2 marks also accumulate at helitrons and some other interspersed repeat sequences not targeted by <a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">WAGO-1</a>.</p><p id="p-60">Our genetic and immunolabeling data suggest that the two mechanisms act in parallel: <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> developmental defects are additive, and H3K9me2 deposition appears normally enriched on the X chromosome in <em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene">wago-1</a></em> (<a id="xref-ref-49-9" class="xref-bibr" href="#ref-49">She <em>et al.</em> 2009</a>) and MAGO12 meiotic male germ cells (this study). Studies in mouse suggest H3K9 methylation acts in parallel with other mechanisms, piRNA and DNA methylation, to silence certain transposons (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Di Giacomo <em>et al.</em> 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Di Giacomo <em>et al.</em> 2014</a>). In <em>C. elegans</em>, transposons are known to be variably targeted for silencing by the 21U (pi) RNA/PRG and/or 22G RNA/WAGO pathways and for H3K9 methylation (reviewed by <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Van Wynsberghe and Maine 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ni <em>et al.</em> 2014</a>). There is overlap among these mechanisms, for example the nuclear RNAi (NRDE) pathway targets H3K9me3 marks to retrotransposons (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ni <em>et al.</em> 2014</a>). Our data provide further evidence of parallel regulatory pathways targeting a common set of transposons.</p><p id="p-61">Although H3K9me2 was originally described as a “silencing” mark due to its association with heterochromatin (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bannister <em>et al.</em> 2001</a>), substantial data have accumulated to indicate that H3K9me2 can be present on expressed genes. In <em>C. elegans</em> embryos, H3K9me2 is associated with both active and inactive genes in nuclear envelope-associated chromatin (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ikegami <em>et al.</em> 2010</a>). In <em>Drosophila</em>, distinct patterns of H3K9me2 accumulation are observed across active genes located in heterochromatin domains (H3K9me2 was present) <em>vs.</em> active genes located in euchromatin domains (H3K9me2 was absent) (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Yasuhara and Wakimoto 2008</a>). This flexible H3K9me2 accumulation pattern is in contrast to many other marks whose accumulation correlates strongly with either an active (<em>e.g.</em>, H3K36me1) or repressed (<em>e.g.</em>, H3K27me3) state (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> 2014</a>). Consistent with published data from <em>C. elegans</em> whole-animal RNA-seq analysis (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> 2014</a>), our RNA-seq results do not indicate a strong correlation between H3K9me2 and transcript level. Although the most highly expressed genes tend to have a lower H3K9me2 level than other genes, this trend does not extend to genes expressed at a low-moderate level. The presence of H3K9me2 on active genes in <em>C. elegans</em> heterochromatic regions is consistent with the observations in <em>Drosophila</em>. In a recent study, Garrigues <em>et al.</em> showed that H3K9me2 is a mark of heterochromatin in <em>C. elegans</em> as in other organisms (Garrigues <em>et al.</em> 2014). Their results define heterochromatin as regions containing H3K9me1, H3K9me2, <a href="http://www.wormbase.org/db/get?name=WBGene00001996;class=Gene">HPL-2</a> (Heterochromatin Protein 1 ortholog), and a relatively high density of repetitive sequences. Most of the H3K9me2 enrichment we detect appears to be heterochromatic as it is associated with repeat sequences.</p><p id="p-62">Consistent with the idea that H3K9me2 serves another function in addition to or instead of transcriptional control, our data reveal only modest differences in the gonad transcriptome in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>(0)</em> mutants <em>vs.</em> controls. This finding is consistent with immunolabeling results obtained by Checchi and Engebrecht in their analysis of checkpoint surveillance (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Checchi and Engebrecht 2011</a>). When they evaluated RNAPII association with unsynapsed chromosomes via immunolabeling, they observed variable association of active RNAPII with the X chromosome in meiotic nuclei in both <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a>;<a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> hermaphrodites. The authors interpreted these results to mean that H3K9me2 was not responsible for limiting transcription in the unsynapsed X homologs. We do note that our data do not provide information about the <em><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene">met-2</a></em> germline transcriptome after successive homozygous generations, and we hypothesize that the transcriptome shifts as fertility decreases.</p><p id="p-63">Numerous alternative functions for H3K9 methylation have been proposed based on a variety of studies. For example, H3K9me2 at repetitive regions may promote a chromatin configuration to reduce unequal crossing-over between repetitive sequences located at different positions on the chromosome. Such a scenario is hypothesized for <em>Drosophila</em>, where double-strand breaks (DSBs) arising in heterochromatin are shifted outside of the heterochromatin domain before RAD51 can be recruited for homologous recombination repair (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chiolo <em>et al.</em> 2011</a>). On the basis of colocalization studies, DSBs that occur in heterochromatin domains attract early recombination markers, and then are relocated outside of the heterochromatin domain prior to RAD51 recruitment and strand repair (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chiolo <em>et al.</em> 2011</a>). Moreover, heterochromatin proteins themselves appear to prevent RAD51 association. A recent fine-structure analysis of recombination on <em>C. elegans</em> chromosome II did not detect a correlation between H3K9me2 density and recombination: an abrupt increase in recombination frequency on the chromosome arm did not correlate with a similar increase in H3K9me2 density (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kaur and Rockman 2014</a>). However, <em>C. elegans</em> recombination is well known to occur more frequently on autosomal arms than in the chromosome centers, and this pattern correlates with the relatively high repetitive sequence density on arms compared with centers (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gerstein <em>et al.</em> 2010</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2011</a>).</p><p id="p-64">In mouse, H3K9me2 accumulation at pericentric heterochromatin appears to contribute to meiotic pairing and synapsis through its association with heterochromatin protein 1 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Takada <em>et al.</em> 2011</a>). In <em>C. elegans</em>, where chromosomes are holocentric, H3K9me2 does not accumulate at centromeric heterochromatin (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gassmann <em>et al.</em> 2012</a>). However, chromosome pairing is mediated by a single pairing center at the end of one arm of each chromosome, and each pairing center contains a number of short sequence motifs responsible for recruiting pairing proteins (<em>e.g.</em>, <a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">HIM-8</a>) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Phillips <em>et al.</em> 2009</a>). Although chromatin in the vicinity of the pairing center contains substantial H3K9me2, these marks do not accumulate at the recruitment motifs themselves (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2011</a>). Consistent with nuclear envelope association of heterochromatic regions, pairing centers are brought together via interaction with <a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">HIM-8</a> and other pairing proteins at the nuclear envelope (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Phillips and Dernburg 2006</a>). While there is no direct evidence that H3K9me2 modulates pairing in <em>C. elegans</em>, there may be a relationship between recombination and H3K9me2 distribution because the chromatin factor, <a href="http://www.wormbase.org/db/get?name=WBGene00001874;class=Gene">HIM-17</a>, is required both for DSB formation and for robust H3K9me2 accumulation at certain sites (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Reddy and Villeneuve 2004</a>).</p><p id="p-65">Another postulated role for H3K9 methylation is in positioning heterochromatin at the nuclear periphery in association with the nuclear envelope, an association thought to limit gene expression. Heterochromatin in the <em>C. elegans</em> genome is concentrated on the autosomal arms and left end of X—coincident with H3K9me2 and repetitive sequences—and these regions associate with the nuclear envelope via <a href="http://www.wormbase.org/db/get?name=WBGene00002275;class=Gene">LEM-2</a> in a manner that depends on H3K9 methylation (Garrigues <em>et al.</em> 2014; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Ikegami <em>et al.</em> 2010</a>; <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Towbin <em>et al.</em> 2012</a>). We note that <a href="http://www.wormbase.org/db/get?name=WBGene00002275;class=Gene">LEM-2</a> binds preferentially to helitron sequences, a class of transposon also enriched for H3K9me2.</p><p id="p-66">In summary, the accumulation of H3K9me2 on unsynapsed X chromosomes may function in multiple ways to promote germline processes. The paucity of <em>de novo</em> H3K9me2 sites on unsynapsed chromosomes suggests that the increased level of H3K9me2 does not play a unique role on those chromosomes. Instead, assembly of heterochromatin on unsynapsed chromosomes may require an elevated level of H3K9me2 to confer the same control over chromosomal processes as obtained on synapsed chromosomes. Future studies may reveal the basis for this difference, and how H3K9me2 marks regulate chromosomal processes during meiosis.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-67">We thank Priscilla Van Wynsberghe for invaluable assistance in developing the ChIP-seq protocol, Bill Kelly for invaluable discussions during the course of this study and comments on the manuscript, Hiroshi Kimura for antibody used in the initial phase of this study, and Julie Ahringer, Kirill Borziak, Mike Cosgrove, Sarah Hall, Sylvia Lee, Jason Lieb, Susan Strome, Alex Twyford, Katherine Walstrom, Robert Waterston, and Yiou Xiao for discussions and technical suggestions. This work was supported by funds from the National Institutes of Health (R01 GM089818) to E.M.M. Some strains used in this study were provided by the <em>Caenorhabditis</em> Genetics Center, which is funded by the NIH Office of Research Infrastructure Programs (P40 OD010440).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-5"><p id="p-68">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019828/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019828/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-69">Sequencing data have been submitted to the GEO database at NCBI as series GSE67030.</p></li><li class="fn" id="fn-7"><p id="p-70"><em>Communicating editor: S. Lee</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-05-01"><span class="received-label">Received </span>May 1, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-07-06"><span class="accepted-label">Accepted </span>July 6, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Guo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.1" data-doi="10.1242/dev.009373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvitz</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Two <em>C. elegans</em> histone methyltransferases repress <em>lin-3</em> EGF transcription to inhibit vulval development.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">2991</span>–<span class="cit-lpage">2999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.009373%26rft_id%253Dinfo%253Apmid%252F17634190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM0LzE2LzI5OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTg2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.2" data-doi="10.1038/35065138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bannister</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zegerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Partridge</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miska</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Selective recognition of methylated lysine 9 on histone H3 by the HP1 chromo domain.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">410</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBannister%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D410%26rft.issue%253D6824%26rft.spage%253D120%26rft.epage%253D124%26rft.atitle%253DSelective%2Brecognition%2Bof%2Bmethylated%2Blysine%2B9%2Bon%2Bhistone%2BH3%2Bby%2Bthe%2BHP1%2Bchromo%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35065138%26rft_id%253Dinfo%253Apmid%252F11242054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35065138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11242054&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167194300055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.3" data-doi="10.1038/ng1283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Meiotic pairing and imprinted X chromatin assembly in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBean%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D100%26rft.epage%253D105%26rft.atitle%253DMeiotic%2Bpairing%2Band%2Bimprinted%2BX%2Bchromatin%2Bassembly%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1283%26rft_id%253Dinfo%253Apmid%252F14702046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14702046&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187666800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.4" data-doi="10.1371/journal.pgen.1000830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bessler</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential localization and independent acquisition of the H3K9me2 and H3K9me3 chromatin modifications in the <em>Caenorhabditis elegans</em> adult germ line.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBessler%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1000830%26rft.epage%253De1000830%26rft.atitle%253DDifferential%2Blocalization%2Band%2Bindependent%2Bacquisition%2Bof%2Bthe%2BH3K9me2%2Band%2BH3K9me3%2Bchromatin%2Bmodifications%2Bin%2Bthe%2BCaenorhabditis%2Belegans%2Badult%2Bgerm%2Bline.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000830%26rft_id%253Dinfo%253Apmid%252F20107519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20107519&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.5" data-doi="10.1051/medsci/20031967725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The genetics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrenner%26rft.auinit1%253DS.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D94%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fmedsci%252F20031967725%26rft_id%253Dinfo%253Apmid%252F4366476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiNzcvMS83MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.6" data-doi="10.1371/journal.pgen.1002267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Checchi</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engebrecht</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> Histone Methyltransferase MET-2 shields the male X chromosome from checkpoint machinery and mediates meiotic sex chromosome inactivation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChecchi%26rft.auinit1%253DP.%2BM.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002267%26rft.epage%253De1002267%26rft.atitle%253DCaenorhabditis%2Belegans%2Bhistone%2Bmethyltransferase%2BMET-2%2Bshields%2Bthe%2Bmale%2BX%2Bchromosome%2Bfrom%2Bcheckpoint%2Bmachinery%2Band%2Bmediates%2Bmeiotic%2Bsex%2Bchromosome%2Binactivation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002267%26rft_id%253Dinfo%253Apmid%252F21909284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21909284&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.7" data-doi="10.1093/nar/gkr425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Down</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latorre</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahringer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Systematic bias in high-throughput sequencing data and its correction by BEADS.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr425%26rft_id%253Dinfo%253Apmid%252F21646344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xNS9lMTAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.8" data-doi="10.1016/j.cell.2011.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiolo</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colmenares</span>  <span class="cit-name-given-names">S. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polyzos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costes</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Double-strand breaks in heterochromatin move outside of a dynamic HP1a domain to complete recombinational repair.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">732</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChiolo%26rft.auinit1%253DI.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D732%26rft.epage%253D744%26rft.atitle%253DDouble-strand%2Bbreaks%2Bin%2Bheterochromatin%2Bmove%2Boutside%2Bof%2Ba%2Bdynamic%2BHP1a%2Bdomain%2Bto%2Bcomplete%2Brecombinational%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.012%26rft_id%253Dinfo%253Apmid%252F21353298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21353298&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288007100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.9" data-doi="10.1016/j.cell.2009.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claycomb</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batista</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasale</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Argonaute CSR-1 and its 22G-RNA cofactors are required for holocentric chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DClaycomb%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D139%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D134%26rft.atitle%253DThe%2BArgonaute%2BCSR-1%2Band%2Bits%2B22G-RNA%2Bcofactors%2Bare%2Brequired%2Bfor%2Bholocentric%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.09.014%26rft_id%253Dinfo%253Apmid%252F19804758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19804758&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270388600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.10" data-doi="10.1073/pnas.0911685107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conine</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batista</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claycomb</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaves</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Argonautes ALG-3 and ALG-4 are required for spermatogenesis-specific 26G-RNAs and thermotolerant sperm in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">3588</span>–<span class="cit-lpage">3593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0911685107%26rft_id%253Dinfo%253Apmid%252F20133686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzgvMzU4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.11" data-doi="10.1095/biolreprod.112.103572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deguchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagamatsu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyachi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Posttranscriptional regulation of histone lysine methyltransferase GLP in embryonic male mouse germ cells.</span> <abbr class="cit-jnl-abbrev">Biol. Reprod.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BReprod.%26rft_id%253Dinfo%253Adoi%252F10.1095%252Fbiolreprod.112.103572%26rft_id%253Dinfo%253Apmid%252F23284137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImJpb2xyZXByb2QiO3M6NToicmVzaWQiO3M6NzoiODgvMi8zNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.12" data-doi="10.1016/j.molcel.2013.04.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Giacomo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comazzetto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saini</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Fazio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrieri</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiple epigenetic mechanisms and the piRNA pathway enforce LINE1 silencing during adult spermatogenesis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D601%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.04.026%26rft_id%253Dinfo%253Apmid%252F23706823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.04.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23706823&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319893600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.13" data-doi="10.1186/1756-8935-7-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Giacomo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comazzetto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sampath</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Carroll</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">G9a co-suppresses LINE1 elements in spermatogonia.</span> <abbr class="cit-jnl-abbrev">Epigenetics Chromatin</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpigenetics%2BChromatin%26rft.volume%253D7%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-8935-7-24%26rft_id%253Dinfo%253Apmid%252F25276231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-8935-7-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25276231&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.14" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiIyOS8xLzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-5.9.1865.15" data-doi="10.1101/gr.180489.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrigues</span>  <span class="cit-name-given-names">J.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidoli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">B.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strome</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Defining heterochromatin in C. elegans through genome-wide analysis of the heterochromatin protein 1 homolog HPL-2.</span> <span class="cit-source">Genome Res.</span> <span class="cit-vol">25</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.180489.114%26rft_id%253Dinfo%253Apmid%252F25467431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjI1LzEvNzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTg2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.16" data-doi="10.1002/elps.200800720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gassmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenacher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohde</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantifying Western blots: pitfalls of densitometry.</span> <abbr class="cit-jnl-abbrev">Electrophoresis</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1845</span>–<span class="cit-lpage">1855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.volume%253D30%26rft.spage%253D1845%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.200800720%26rft_id%253Dinfo%253Apmid%252F19517440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elps.200800720&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19517440&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267689900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.17" data-doi="10.1038/nature10973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gassmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rechtsteiner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muroyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egelhofer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An inverse relationship to germline transcription defines centromeric chromatin in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">534</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGassmann%26rft.auinit1%253DR.%26rft.volume%253D484%26rft.issue%253D7395%26rft.spage%253D534%26rft.epage%253D537%26rft.atitle%253DAn%2Binverse%2Brelationship%2Bto%2Bgermline%2Btranscription%2Bdefines%2Bcentromeric%2Bchromatin%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10973%26rft_id%253Dinfo%253Apmid%252F22495302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10973&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495302&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303200400055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.18" data-doi="10.1126/science.1196914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Nostrand</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arshinoff</span>  <span class="cit-name-given-names">B. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrative analysis of the <em>Caenorhabditis elegans</em> genome by the modENCODE project.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1775</span>–<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1196914%26rft_id%253Dinfo%253Apmid%252F21177976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMi8xNzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.19" data-doi="10.1016/j.molcel.2009.09.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirayama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">D.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasale</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batista</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Distinct argonaute-mediated 22G-RNA pathways direct genome surveillance in the <em>C. elegans</em> germline.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGu%26rft.auinit1%253DW.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D231%26rft.epage%253D244%26rft.atitle%253DDistinct%2Bargonaute-mediated%2B22G-RNA%2Bpathways%2Bdirect%2Bgenome%2Bsurveillance%2Bin%2Bthe%2BC.%2Belegans%2Bgermline.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.09.020%26rft_id%253Dinfo%253Apmid%252F19800275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.09.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19800275&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271801900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.20" data-doi="10.1038/ng0104-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Handel</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Marking Xs, together and separately.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHandel%26rft.auinit1%253DM.%2BA.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D13%26rft.atitle%253DMarking%2BXs%252C%2Btogether%2Band%2Bseparately.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0104-12%26rft_id%253Dinfo%253Apmid%252F14702033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0104-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14702033&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.21" data-doi="10.1038/nature13415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alver</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative analysis of metazoan chromatin organization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13415%26rft_id%253Dinfo%253Apmid%252F25164756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25164756&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340840600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvitz</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Nondisjunction Mutants of the Nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHodgkin%26rft.auinit1%253DJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D94%26rft.atitle%253DNONDISJUNCTION%2BMUTANTS%2BOF%2BTHE%2BNEMATODE%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Apmid%252F17248881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiOTEvMS82NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.23" data-doi="10.1093/nar/gkn923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn923%26rft_id%253Dinfo%253Apmid%252F19033363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjY6IjM3LzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.24" data-doi="10.1186/gb-2010-11-12-r120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikegami</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egelhofer</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strome</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> chromosome arms are anchored to the nuclear membrane via discontinuous association with LEM-2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DIkegami%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253DR120%26rft.epage%253DR120%26rft.atitle%253DCaenorhabditis%2Belegans%2Bchromosome%2Barms%2Bare%2Banchored%2Bto%2Bthe%2Bnuclear%2Bmembrane%2Bvia%2Bdiscontinuous%2Bassociation%2Bwith%2BLEM-2.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-12-r120%26rft_id%253Dinfo%253Apmid%252F21176223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-12-r120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21176223&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.25" data-doi="10.1534/genetics.109.110338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo-Lambert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engebrecht</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A single unpaired and transcriptionally silenced X chromosome locally precludes checkpoint signaling in the <em>Caenorhabditis elegans</em> germ line.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.110338%26rft_id%253Dinfo%253Apmid%252F20008570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzMvNjEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.26" data-doi="10.1534/genetics.113.158857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Crossover heterogeneity in the absence of hotspots in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158857%26rft_id%253Dinfo%253Apmid%252F24172135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.27" data-doi="10.1186/1756-8935-7-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transgenerational epigenetics in the germline cycle of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Epigenetics Chromatin</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpigenetics%2BChromatin%26rft.volume%253D7%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-8935-7-6%26rft_id%253Dinfo%253Apmid%252F24678826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-8935-7-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24678826&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.28" data-doi="10.1007/s10577-007-1143-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aramayo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Meiotic silencing and the epigenetics of sex.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DKelly%26rft.auinit1%253DW.%2BG.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D633%26rft.epage%253D651%26rft.atitle%253DMeiotic%2Bsilencing%2Band%2Bthe%2Bepigenetics%2Bof%2Bsex.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-007-1143-0%26rft_id%253Dinfo%253Apmid%252F17674151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-007-1143-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17674151&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249011500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dernburg</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">X-chromosome silencing in the germline of <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.volume%253D129%26rft.spage%253D479%26rft_id%253Dinfo%253Apmid%252F11807039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11807039&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173759100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.30" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">L’Hernault</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shakes</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Developmental genetics of chromosome I spermatogenesis-defective mutants in the nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DL%2527Hernault%26rft.auinit1%253DS.%2BW.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D435%26rft.epage%253D452%26rft.atitle%253DDevelopmental%2Bgenetics%2Bof%2Bchromosome%2BI%2Bspermatogenesis-defective%2Bmutants%2Bin%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F3197956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIwLzIvNDM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.32" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.33" data-doi="10.1101/gr.115519.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rechtsteiner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egelhofer</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vielle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latorre</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Broad chromosomal domains of histone modification patterns in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115519.110%26rft_id%253Dinfo%253Apmid%252F21177964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMjI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.34" data-doi="10.1016/S1937-6448(10)82002-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maine</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Meiotic silencing in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Int. Rev. Cell Mol. Biol.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BRev%2BCell%2BMol%2BBiol%26rft.aulast%253DMaine%26rft.auinit1%253DE.%2BM.%26rft.volume%253D282%26rft.spage%253D91%26rft.epage%253D134%26rft.atitle%253DMeiotic%2Bsilencing%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1937-6448%252810%252982002-7%26rft_id%253Dinfo%253Apmid%252F20630467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1937-6448(10)82002-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20630467&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.35" data-doi="10.1016/j.cub.2005.09.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maine</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratliff</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vought</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">She</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">EGO-1, a putative RNA-dependent RNA polymerase, is required for heterochromatin assembly on unpaired dna during <em>C. elegans</em> meiosis.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DMaine%26rft.auinit1%253DE.%2BM.%26rft.volume%253D15%26rft.issue%253D21%26rft.spage%253D1972%26rft.epage%253D1978%26rft.atitle%253DEGO-1%252C%2Ba%2Bputative%2BRNA-dependent%2BRNA%2Bpolymerase%252C%2Bis%2Brequired%2Bfor%2Bheterochromatin%2Bassembly%2Bon%2Bunpaired%2Bdna%2Bduring%2BC.%2Belegans%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.09.049%26rft_id%253Dinfo%253Apmid%252F16271877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.09.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16271877&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233422900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.36" data-doi="10.1016/j.ydbio.2006.02.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelms</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna-Rose</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>C. elegans</em> HIM-8 functions outside of meiosis to antagonize EGL-13 Sox protein function.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DNelms%26rft.auinit1%253DB.%2BL.%26rft.volume%253D293%26rft.issue%253D2%26rft.spage%253D392%26rft.epage%253D402%26rft.atitle%253DC.%2Belegans%2BHIM-8%2Bfunctions%2Boutside%2Bof%2Bmeiosis%2Bto%2Bantagonize%2BEGL-13%2BSox%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2006.02.010%26rft_id%253Dinfo%253Apmid%252F16546157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2006.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16546157&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.37" data-doi="10.1186/1471-2164-15-1157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Complex coding of endogenous siRNA, transcriptional silencing and H3K9 methylation on native targets of germline nuclear RNAi in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1157%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1157%26rft_id%253Dinfo%253Apmid%252F25534009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25534009&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.38" data-doi="10.1016/j.devcel.2006.09.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dernburg</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A family of zinc-finger proteins is required for chromosome-specific pairing and synapsis during meiosis in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">817</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DPhillips%26rft.auinit1%253DC.%2BM.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D817%26rft.epage%253D829%26rft.atitle%253DA%2Bfamily%2Bof%2Bzinc-finger%2Bproteins%2Bis%2Brequired%2Bfor%2Bchromosome-specific%2Bpairing%2Band%2Bsynapsis%2Bduring%2Bmeiosis%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2006.09.020%26rft_id%253Dinfo%253Apmid%252F17141157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2006.09.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17141157&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242664900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.39" data-doi="10.1016/j.cell.2005.09.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhalla</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlton</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiser</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">HIM-8 binds to the X chromosome pairing center and mediates chromosome-specific meiotic synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1051</span>–<span class="cit-lpage">1063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPhillips%26rft.auinit1%253DC.%2BM.%26rft.volume%253D123%26rft.issue%253D6%26rft.spage%253D1051%26rft.epage%253D1063%26rft.atitle%253DHIM-8%2Bbinds%2Bto%2Bthe%2BX%2Bchromosome%2Bpairing%2Bcenter%2Band%2Bmediates%2Bchromosome-specific%2Bmeiotic%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.09.035%26rft_id%253Dinfo%253Apmid%252F16360035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.09.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16360035&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234177200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.40" data-doi="10.1038/ncb1904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chretien</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urnov</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of chromosome sequence motifs that mediate meiotic pairing and synapsis in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">934</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DPhillips%26rft.auinit1%253DC.%2BM.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D934%26rft.epage%253D942%26rft.atitle%253DIdentification%2Bof%2Bchromosome%2Bsequence%2Bmotifs%2Bthat%2Bmediate%2Bmeiotic%2Bpairing%2Band%2Bsynapsis%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1904%26rft_id%253Dinfo%253Apmid%252F19620970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19620970&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268593200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lissemore</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smardon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelber</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Enhancers of <em>glp-1</em>, a gene required for cell-signaling in <em>Caenorhabditis elegans</em>, define a set of genes required for germline development.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DQiao%26rft.auinit1%253DL.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D551%26rft.epage%253D569%26rft.atitle%253DEnhancers%2Bof%2Bglp-1%252C%2Ba%2Bgene%2Brequired%2Bfor%2Bcell-signaling%2Bin%2BCaenorhabditis%2Belegans%252C%2Bdefine%2Ba%2Bset%2Bof%2Bgenes%2Brequired%2Bfor%2Bgermline%2Bdevelopment.%26rft_id%253Dinfo%253Apmid%252F8647392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzIvNTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.42" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.43" data-doi="10.1016/j.cell.2004.07.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>C. elegans</em> HIM-17 links chromatin modification and competence for initiation of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DReddy%26rft.auinit1%253DK.%2BC.%26rft.volume%253D118%26rft.issue%253D4%26rft.spage%253D439%26rft.epage%253D452%26rft.atitle%253DC.%2Belegans%2BHIM-17%2Blinks%2Bchromatin%2Bmodification%2Band%2Bcompetence%2Bfor%2Binitiation%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2004.07.026%26rft_id%253Dinfo%253Apmid%252F15315757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2004.07.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15315757&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223650900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.44" data-doi="10.1016/S1097-2765(00)00059-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinke</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nance</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Doren</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A global profile of germline gene expression in C. elegans.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DReinke%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D605%26rft.epage%253D616%26rft.atitle%253DA%2Bglobal%2Bprofile%2Bof%2Bgermline%2Bgene%2Bexpression%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252900059-9%26rft_id%253Dinfo%253Apmid%252F11030340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)00059-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11030340&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089578600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.45" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-other" id="cit-5.9.1865.46"><div class="cit-metadata unstructured">Schaner, C.E., and W.G. Kelly, 2006 Germline chromatin (January 24, 2006), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.73.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.47" data-doi="10.1016/j.ydbio.2004.11.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizugaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Extensive and orderly reprogramming of genome-wide chromatin modifications associated with specification and early development of germ cells in mice.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">440</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSeki%26rft.auinit1%253DY.%26rft.volume%253D278%26rft.issue%253D2%26rft.spage%253D440%26rft.epage%253D458%26rft.atitle%253DExtensive%2Band%2Borderly%2Breprogramming%2Bof%2Bgenome-wide%2Bchromatin%2Bmodifications%2Bassociated%2Bwith%2Bspecification%2Band%2Bearly%2Bdevelopment%2Bof%2Bgerm%2Bcells%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2004.11.025%26rft_id%253Dinfo%253Apmid%252F15680362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2004.11.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15680362&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226970000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.48" data-doi="10.1016/j.devcel.2013.11.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirayama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishidate</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">D.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The <em>C. elegans</em> CSR-1 argonaute pathway counteracts epigenetic silencing to promote germline gene expression.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D27%26rft.spage%253D656%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2013.11.014%26rft_id%253Dinfo%253Apmid%252F24360782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2013.11.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24360782&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329117600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.49" data-doi="10.1371/journal.pgen.1000624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">She</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedotov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maine</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regulation of heterochromatin assembly on unpaired chromosomes during <em>Caenorhabditis elegans</em> meiosis by components of a small RNA-mediated pathway.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DShe%26rft.auinit1%253DX.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De1000624%26rft.epage%253De1000624%26rft.atitle%253DRegulation%2Bof%2Bheterochromatin%2Bassembly%2Bon%2Bunpaired%2Bchromosomes%2Bduring%2BCaenorhabditis%2Belegans%2Bmeiosis%2Bby%2Bcomponents%2Bof%2Ba%2Bsmall%2BRNA-mediated%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000624%26rft_id%253Dinfo%253Apmid%252F19714217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000624&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19714217&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.50" data-doi="10.1093/bioinformatics/btp479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manrai</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CEAS: cis-regulatory element annotation system.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2605</span>–<span class="cit-lpage">2606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp479%26rft_id%253Dinfo%253Apmid%252F19689956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTkvMjYwNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.51" data-doi="10.1534/genetics.107.070847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelms</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sleiman</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlin</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna-Rose</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modulation of <em>Caenorhabditis elegans</em> transcription factor activity by HIM-8 and the related Zinc-Finger ZIM proteins.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070847%26rft_id%253Dinfo%253Apmid%252F17720937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8yLzEyMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTg2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.52" data-doi="10.1242/dev.064444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirakawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tachibana</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">HP1gamma links histone methylation marks to meiotic synapsis in mice.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">4207</span>–<span class="cit-lpage">4217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DTakada%26rft.auinit1%253DY.%26rft.volume%253D138%26rft.issue%253D19%26rft.spage%253D4207%26rft.epage%253D4217%26rft.atitle%253DHP1%257Bgamma%257D%2Blinks%2Bhistone%2Bmethylation%2Bmarks%2Bto%2Bmeiotic%2Bsynapsis%2Bin%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.064444%26rft_id%253Dinfo%253Apmid%252F21896631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM4LzE5LzQyMDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTg2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.53" data-doi="10.1016/j.cell.2012.06.051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Towbin</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Aguilera</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sack</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaidatzis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalck</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Step-wise methylation of histone H3K9 positions heterochromatin at the nuclear periphery.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">934</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTowbin%26rft.auinit1%253DB.%2BD.%26rft.volume%253D150%26rft.issue%253D5%26rft.spage%253D934%26rft.epage%253D947%26rft.atitle%253DStep-wise%2Bmethylation%2Bof%2Bhistone%2BH3K9%2Bpositions%2Bheterochromatin%2Bat%2Bthe%2Bnuclear%2Bperiphery.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.06.051%26rft_id%253Dinfo%253Apmid%252F22939621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.06.051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22939621&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308500200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.54" data-doi="10.1038/nbt.2450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrickson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Differential analysis of gene regulation at transcript resolution with RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2450%26rft_id%253Dinfo%253Apmid%252F23222703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23222703&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.55" data-doi="10.1242/dev.000018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Meiotic sex chromosome inactivation.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1823</span>–<span class="cit-lpage">1831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.000018%26rft_id%253Dinfo%253Apmid%252F17329371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM0LzEwLzE4MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTg2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.56" data-doi="10.1007/978-1-4614-4015-4_13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Wynsberghe</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maine</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Epigenetic control of germline development.</span> <abbr class="cit-jnl-abbrev">Adv. Exp. Med. Biol.</abbr> <span class="cit-vol">757</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.stitle%253DAdv%2BExp%2BMed%2BBiol%26rft.aulast%253DVan%2BWynsberghe%26rft.auinit1%253DP.%2BM.%26rft.volume%253D757%26rft.spage%253D373%26rft.epage%253D403%26rft.atitle%253DEpigenetic%2Bcontrol%2Bof%2Bgermline%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4614-4015-4_13%26rft_id%253Dinfo%253Apmid%252F22872484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4614-4015-4_13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22872484&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.57" data-doi="10.1371/journal.pgen.1002933"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vielle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ercan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotwaliwale</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">H4K20me1 contributes to downregulation of X-linked genes for <em>C. elegans</em> dosage compensation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVielle%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253De1002933%26rft.epage%253De1002933%26rft.atitle%253DH4K20me1%2Bcontributes%2Bto%2Bdownregulation%2Bof%2BX-linked%2Bgenes%2Bfor%2BC.%2Belegans%2Bdosage%2Bcompensation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002933%26rft_id%253Dinfo%253Apmid%252F23028348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002933&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028348&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.58" data-doi="10.1007/978-1-4614-4015-4_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seydoux</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Germ cell specification.</span> <abbr class="cit-jnl-abbrev">Adv. Exp. Med. Biol.</abbr> <span class="cit-vol">757</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.stitle%253DAdv%2BExp%2BMed%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D757%26rft.spage%253D17%26rft.epage%253D39%26rft.atitle%253DGerm%2Bcell%2Bspecification.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4614-4015-4_2%26rft_id%253Dinfo%253Apmid%252F22872473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4614-4015-4_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22872473&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.59" data-doi="10.1186/1471-2164-10-213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehlers</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of genes expressed in the hermaphrodite germ line of <em>C. elegans</em> using SAGE.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D10%26rft.spage%253D213%26rft.epage%253D213%26rft.atitle%253DIdentification%2Bof%2Bgenes%2Bexpressed%2Bin%2Bthe%2Bhermaphrodite%2Bgerm%2Bline%2Bof%2BC.%2Belegans%2Busing%2BSAGE.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-213%26rft_id%253Dinfo%253Apmid%252F19426519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19426519&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.60" data-doi="10.1016/j.devcel.2013.11.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wedeles</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claycomb</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Protection of germline gene expression by the <em>C. elegans</em> Argonaute CSR-1.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">664</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D27%26rft.spage%253D664%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2013.11.016%26rft_id%253Dinfo%253Apmid%252F24360783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2013.11.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24360783&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329117600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.61" data-doi="10.1371/journal.pone.0105267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic distribution of H3K9me2 and DNA methylation in a maize genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e105267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De105267%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0105267%26rft_id%253Dinfo%253Apmid%252F25122127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0105267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25122127&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.62" data-doi="10.1371/journal.pgen.0040016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yasuhara</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakimoto</span>  <span class="cit-name-given-names">B. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular landscape of modified histones in <em>Drosophila</em> heterochromatic genes and euchromatin-heterochromatin transition zones.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DYasuhara%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253De16%26rft.epage%253De16%26rft.atitle%253DMolecular%2Blandscape%2Bof%2Bmodified%2Bhistones%2Bin%2BDrosophila%2Bheterochromatic%2Bgenes%2Band%2Beuchromatin-heterochromatin%2Btransition%2Bzones.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040016%26rft_id%253Dinfo%253Apmid%252F18208336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18208336&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1865.63" data-doi="10.1186/gb-2008-9-9-r137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eeckhoute</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Model-based analysis of ChIP-Seq (MACS).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253DR137%26rft.epage%253DR137%26rft.atitle%253DModel-based%2Banalysis%2Bof%2BChIP-Seq%2B%2528MACS%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-9-r137%26rft_id%253Dinfo%253Apmid%252F18798982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-9-r137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18798982&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/9/1865.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/9/1857" title="Targeted Modification of Gene Function Exploiting Homology-Directed Repair of TALEN-Mediated Double-Strand Breaks in Barley" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/9/1879" title="An Updated Collection of Sequence Barcoded Temperature-Sensitive Alleles of Yeast Essential Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/9"><span>Volume 5 Issue 9, September 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/9.cover-source.jpg?itok=EE-GjT57" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1865.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dsxbvtBlmyYsh6o_CQYdFagRqQpT6Jl2LzhXATxbyFk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/9/1865" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/9/1865" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/9/1865" class="active" data-icon-position="" data-hide-link-title="0">Enrichment of H3K9me2 on Unsynapsed Chromatin in Caenorhabditis elegans Does Not Target de Novo Sites</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460026" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1oo26pNJc_t-B9U1_Dco1eIOQPKsvTNLyr41N9IIb2I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/9/1865.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/9/1865" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RI6MGhq4gPlPOmBxX3JnvaSWYcZ7oAmTzpJKTpgt8iY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/9/1865" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/9/1865.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460026" id="node460026--2" data-pisa="ggg;5/9/1865" data-pisa-master="ggg;g3.115.019828" data-apath="/ggg/5/9/1865.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Enrichment of H3K9me2 on Unsynapsed Chromatin in <em>Caenorhabditis elegans</em> Does Not Target <em>de Novo</em> Sites</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yiqing Guo</span>, <span class="highwire-citation-author" data-delta="1">Bing Yang</span>, <span class="highwire-citation-author" data-delta="2">Yini Li</span>, <span class="highwire-citation-author" data-delta="3">Xia Xu</span> and <span class="highwire-citation-author" data-delta="4">Eleanor M. Maine</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1865-1878;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019828 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460026--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yiqing Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yiqing%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYiqing%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bing Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bing%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABing%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yini Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yini%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYini%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xia Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xia%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXia%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eleanor M. Maine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleanor%2BM.%2BMaine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maine%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEleanor%2BM.%2BMaine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emmaine@syr.edu" class="" data-icon-position="" data-hide-link-title="0">emmaine@syr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/9/1865" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/9/1865&amp;title=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/9/1865&amp;title=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/9/1865&amp;title=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/9/1865&amp;text=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/9/1865&amp;title=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/9/1865&amp;t=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/9/1865&amp;title=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/9/1865&amp;title=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460026/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460026" id="node460026--3" data-pisa="ggg;5/9/1865" data-pisa-master="ggg;g3.115.019828" data-apath="/ggg/5/9/1865.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Enrichment of H3K9me2 on Unsynapsed Chromatin in <em>Caenorhabditis elegans</em> Does Not Target <em>de Novo</em> Sites</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yiqing Guo</span>, <span class="highwire-citation-author" data-delta="1">Bing Yang</span>, <span class="highwire-citation-author" data-delta="2">Yini Li</span>, <span class="highwire-citation-author" data-delta="3">Xia Xu</span> and <span class="highwire-citation-author" data-delta="4">Eleanor M. Maine</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1865-1878;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019828 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460026--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yiqing Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yiqing%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYiqing%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bing Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bing%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABing%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yini Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yini%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYini%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xia Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xia%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXia%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eleanor M. Maine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Syracuse University, Syracuse, New York 13244</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleanor%2BM.%2BMaine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maine%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEleanor%2BM.%2BMaine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emmaine@syr.edu" class="" data-icon-position="" data-hide-link-title="0">emmaine@syr.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1865&amp;rft.epage=1878&amp;rft.atitle=Enrichment%20of%20H3K9me2%20on%20Unsynapsed%20Chromatin%20in%20Caenorhabditis%20elegans%20Does%20Not%20Target%20de%20Novo%20Sites&amp;rft.volume=5&amp;rft.issue=9&amp;rft.date=2015-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Guo%2C+Yiqing&amp;rft.au=Yang%2C+Bing&amp;rft.au=Li%2C+Yini&amp;rft.au=Xu%2C+Xia&amp;rft.au=Maine%2C+Eleanor+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460026/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460026/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460026/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460026/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460026/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460026/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460026/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460026/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460026/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460026/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460026/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460026/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/9/1865" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/9/1865.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/9/1865.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/9/1865.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/9/1865.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
