<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/9/1797" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/9/1797.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/9/1797.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/9/1797.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/9/1797.atom" />
<meta name="HW.pisa" content="ggg;5/9/1797" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops" />
<meta name="DC.Identifier" content="10.1534/g3.115.019703" />
<meta name="DC.Date" content="2015-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Clevenger and Ozias-Akins. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-throughput next-generation sequence-based genotyping and single nucleotide polymorphism (SNP) detection opens the door for emerging genomics-based breeding strategies such as genome-wide association analysis and genomic selection. In polyploids, SNP detection is confounded by a highly similar homeologous sequence where a polymorphism between subgenomes must be differentiated from a SNP. We have developed and implemented a novel tool called SWEEP: Sliding Window Extraction of Explicit Polymorphisms. SWEEP uses subgenome polymorphism haplotypes as contrast to identify true SNPs between genotypes. The tool is a single command script that calls a series of modules based on user-defined options and takes sorted/indexed bam files or vcf files as input. Filtering options are highly flexible and include filtering based on sequence depth, alternate allele ratio, and SNP quality on top of the SWEEP filtering procedure. Using real and simulated data we show that SWEEP outperforms current SNP filtering methods for polyploids. SWEEP can be used for high-quality SNP discovery in polyploid crops." />
<meta name="DC.Contributor" content="Josh P. Clevenger" />
<meta name="DC.Contributor" content="Peggy Ozias-Akins" />
<meta name="article:published_time" content="2015-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-throughput next-generation sequence-based genotyping and single nucleotide polymorphism (SNP) detection opens the door for emerging genomics-based breeding strategies such as genome-wide association analysis and genomic selection. In polyploids, SNP detection is confounded by a highly similar homeologous sequence where a polymorphism between subgenomes must be differentiated from a SNP. We have developed and implemented a novel tool called SWEEP: Sliding Window Extraction of Explicit Polymorphisms. SWEEP uses subgenome polymorphism haplotypes as contrast to identify true SNPs between genotypes. The tool is a single command script that calls a series of modules based on user-defined options and takes sorted/indexed bam files or vcf files as input. Filtering options are highly flexible and include filtering based on sequence depth, alternate allele ratio, and SNP quality on top of the SWEEP filtering procedure. Using real and simulated data we show that SWEEP outperforms current SNP filtering methods for polyploids. SWEEP can be used for high-quality SNP discovery in polyploid crops.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/09/01" />
<meta name="citation_mjid" content="ggg;5/9/1797" />
<meta name="citation_id" content="5/9/1797" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/9/1797" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/9/1797.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/9/1797.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/9/1797.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019703" />
<meta name="citation_pmid" content="26153076" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1797" />
<meta name="citation_lastpage" content="1803" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Josh P. Clevenger" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793" />
<meta name="citation_author" content="Peggy Ozias-Akins" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793" />
<meta name="citation_author_email" content="pozias@uga.edu" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=N. Bassil;citation_author=T. M. Davis;citation_author=H. Zhang;citation_author=S. Ficklin;citation_author=M. Mittmann;citation_title=Development and preliminary evaluation of a 90 K Axiom SNP array for the allo-octoploid cultivated strawberry Fragaria × ananassa.;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864&amp;ndash;12015&amp;ndash;11310&amp;ndash;12861" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=D. J. Bertioli;citation_author=B. Vidigal;citation_author=S. Nielen;citation_author=M. B. Ratnaparkhe;citation_author=T. H. Lee;citation_title=The repetitive component of the A genome of peanut (Arachis hypogaea) and its role in remodelling intergenic sequence space since its evolutionary divergence from the B genome.;citation_volume=112;citation_year=2013;citation_pmid=23828319;citation_doi=10.1093/aob/mct128" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. J. Bertioli;citation_author=P. Ozias-Akins;citation_author=Y. Chu;citation_author=K. M. Dantas;citation_author=S. P. Santos;citation_title=The use of SNP markers for linkage mapping in diploid and tetraploid peanuts.;citation_volume=4;citation_year=2014;citation_pmid=24212082;citation_doi=10.1534/g3.113.007617" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. L. Byers;citation_author=D. B. Harker;citation_author=S. M. Yourstone;citation_author=P. J. Maughan;citation_author=J. A. Udall;citation_title=Development and mapping of SNP assays in allotetraploid cotton.;citation_pages=1201-1214;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22252442;citation_doi=10.1007/s00122-011-1780-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. Chao;citation_author=S. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=J. Clevenger;citation_author=C. Chavarro;citation_author=S. A. Pearl;citation_author=P. Ozias-Akins;citation_author=S. A. Jackson;citation_title=SNP identification in polyploids: a review, example and recommendations.;citation_year=2015;citation_doi=10.1016/j.molp.2015.1002.1002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C.A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=K. Devos;citation_author=J. Beales;citation_author=Y. Ogihara;citation_author=A. N. Doust;citation_title=Comparative sequence analysis of the phytochrome C gene and its upstream region in allohexaploid wheat reveals new data on the evolution of its three constituent genomes.;citation_pages=625-641;citation_volume=58;citation_year=2005;citation_issue=5;citation_pmid=16158239;citation_doi=10.1007/s11103-005-6801-z" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. S. Ersoz;citation_author=M. H. Wright;citation_author=J. L. Pangilinan;citation_author=M. J. Sheehan;citation_author=C. Tobias;citation_title=SNP Discovery with EST and NextGen Sequencing in Switchgrass (Panicum virgatum L.).;citation_pages=e44112-e44112;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23049744;citation_doi=10.1371/journal.pone.0044112" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Hamilton;citation_author=C. N. Hansey;citation_author=B. R. Whitty;citation_author=K. Stoffel;citation_author=A. N. Massa;citation_title=Single nucleotide polymorphism discovery in elite North American potato germplasm.;citation_pages=302-302;citation_volume=12;citation_year=2011;citation_pmid=21658273;citation_doi=10.1186/1471-2164-12-302" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Huang;citation_author=L. Li;citation_author=J. R. Myers;citation_author=G. T. Marth;citation_title=ART: a next-generation sequencing read simulator.;citation_volume=28;citation_year=2012;citation_pmid=22199392;citation_doi=10.1093/bioinformatics/btr708" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Huang;citation_author=L. Deng;citation_author=M. Guan;citation_author=J. Li;citation_author=K. Lu;citation_title=Identification of genome-wide single nucleotide polymorphisms in allopolyploid crop Brassica napus.;citation_pages=717;citation_volume=14;citation_year=2013;citation_pmid=24138473;citation_doi=10.1186/1471-2164-14-717" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Khera;citation_author=H. D. Upadhyaya;citation_author=M. K. Pandey;citation_author=M. Roorkiwal;citation_author=M. Sriswathi;citation_title=Single nucleotide polymorphism–based genetic diversity in the reference set of peanut (Arachis spp.) by developing and applying cost-effective competitive allele specific polymerase chain reaction genotyping assays.;citation_pages=1-11;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Leal-Bertioli;citation_author=K. Shirasawa;citation_author=B. Abernathy;citation_author=M. Moretzsohn;citation_author=C. Chavarro;citation_title=Tetrasomic recombination is surprisingly frequent in allotetraploid Arachis.;citation_volume=199;citation_year=2015;citation_pmid=25701284;citation_doi=10.1534/genetics.115.174607" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=Y. Lee;citation_author=N. Jeong;citation_author=J. H. Kim;citation_author=K. J. Lee;citation_author=K. H. Kim;citation_title=Development, validation and genetic analysis of a large soybean SNP genotyping array.;citation_pages=625-636;citation_volume=81;citation_year=2015;citation_pmid=25641104;citation_doi=10.1111/tpj.12755" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=X. Li;citation_author=A. Acharya;citation_author=A. D. Farmer;citation_author=J. A. Crow;citation_author=A. K. Bharti;citation_title=Prevalence of single nucleotide polymorphism among 27 diverse alfalfa genotypes as assessed by transcriptome sequencing.;citation_pages=568-568;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23107476;citation_doi=10.1186/1471-2164-13-568" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=I. Nagy;citation_author=S. Barth;citation_author=J. Mehenni-Ciz;citation_author=M. T. Abberton;citation_author=D. Milbourne;citation_title=A hybrid next generation transcript sequencing-based approach to identify allelic and homeolog-specific single nucleotide polymorphisms in allotetraploid white clover.;citation_pages=100;citation_volume=14;citation_year=2013;citation_pmid=23402685;citation_doi=10.1186/1471-2164-14-100" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_title=Combining transcriptome assemblies from multiple de novo assemblers in the allo-tetraploid plant Nicotiana benthamiana.;citation_author=K. Nakasugi;citation_author=R. Crowhurst;citation_author=J. Bally;citation_author=P. Waterhouse;citation_volume=9;citation_year=2014;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=S. Nielen;citation_author=B. S. Vidigal;citation_author=S. C. Leal-Bertioli;citation_author=M. Ratnaparkhe;citation_author=A. H. Paterson;citation_title=Matita, a new retroelement from peanut: characterization and evolutionary context in the light of the Arachis A-B genome divergence.;citation_pages=21-38;citation_volume=287;citation_year=2012;citation_issue=1;citation_pmid=22120641;citation_doi=10.1007/s00438-011-0656-6" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=A. Rafalski;citation_title=Applications of single nucleotide polymorphisms in crop genetics.;citation_pages=94-100;citation_volume=5;citation_year=2002;citation_issue=2;citation_pmid=11856602;citation_doi=10.1016/S1369-5266(02)00240-6" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=K. Shirasawa;citation_author=H. Fukuoka;citation_author=H. Matsunaga;citation_author=Y. Kobayashi;citation_author=I. Kobayashi;citation_title=Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato.;citation_volume=20;citation_year=2013;citation_pmid=23903436;citation_doi=10.1093/dnares/dst033" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=L. Song;citation_author=L. Florea;citation_author=B. Langmead;citation_title=Lighter: fast and memory-efficient sequencing error correction without counting.;citation_volume=15;citation_year=2014;citation_doi=10.1186/s13059&amp;ndash;13014&amp;ndash;10509&amp;ndash;13059" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Regis.;citation_author=S. P. Tallury;citation_author=T. G. Isleib;citation_author=S. C. Copeland;citation_author=M. Balota;citation_author=P. R. Anderson;citation_title=Registration of two multiple disease-resistant peanut germplasm lines derived from Arachis cardenasii Krapov. &amp; W.C. Gregory, GKP 10017 (PI262141).;citation_pages=86-89;citation_volume=8;citation_year=2014;citation_issue=1;citation_doi=10.3198/jpr2013.04.0017crg" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. G. A. M. L. Uitdewilligen;citation_author=A.-M. A. Wolters;citation_author=B. B. D’hoop;citation_author=T. J. A. Borm;citation_author=R. G. F. Visser;citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.;citation_pages=e62355;citation_volume=8;citation_year=2013;citation_pmid=23667470;citation_doi=10.1371/journal.pone.0062355" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergrasser;citation_author=I. Cutcutache;citation_author=T. Koressaar;citation_author=J. Ye;citation_author=B. C. Faircloth;citation_title=Primer3 - new capabilities and interfaces.;citation_volume=40;citation_year=2012;citation_pmid=22730293;citation_doi=10.1093/nar/gks596" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Wallace;citation_author=P. J. Bradbury;citation_author=N. Zhang;citation_author=Y. Gibon;citation_author=M. Stitt;citation_title=Association mapping across numerous traits reveals patterns of functional variation in maize.;citation_pages=e1004845;citation_volume=10;citation_year=2014;citation_pmid=25474422;citation_doi=10.1371/journal.pgen.1004845" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Wendel;citation_title=New World tetraploid cottons contain Old World cytoplasm;citation_pages=4132-4136;citation_volume=86;citation_year=1989;citation_issue=11;citation_pmid=16594050;citation_doi=10.1073/pnas.86.11.4132" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Zhou;citation_author=Y. Xia;citation_author=X. Ren;citation_author=Y. Chen;citation_author=L. Huang;citation_title=Construction of a SNP-based genetic linkage map in cultivated peanut based on large scale marker development using next-generation double-digest restriction-site-associated DNA sequencing (ddRADseq).;citation_pages=351;citation_volume=15;citation_year=2014;citation_pmid=24885639;citation_doi=10.1186/1471-2164-15-351" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
<meta name="twitter:description" content="High-throughput next-generation sequence-based genotyping and single nucleotide polymorphism (SNP) detection opens the door for emerging genomics-based breeding strategies such as genome-wide association analysis and genomic selection. In polyploids, SNP detection is confounded by a highly similar homeologous sequence where a polymorphism between subgenomes must be differentiated from a SNP. We have developed and implemented a novel tool called SWEEP: Sliding Window Extraction of Explicit Polymorphisms. SWEEP uses subgenome polymorphism haplotypes as contrast to identify true SNPs between genotypes. The tool is a single command script that calls a series of modules based on user-defined options and takes sorted/indexed bam files or vcf files as input. Filtering options are highly flexible and include filtering based on sequence depth, alternate allele ratio, and SNP quality on top of the SWEEP filtering procedure. Using real and simulated data we show that SWEEP outperforms current SNP filtering methods for polyploids. SWEEP can be used for high-quality SNP discovery in polyploid crops." />
<meta name="og-title" property="og:title" content="SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/9/1797" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-throughput next-generation sequence-based genotyping and single nucleotide polymorphism (SNP) detection opens the door for emerging genomics-based breeding strategies such as genome-wide association analysis and genomic selection. In polyploids, SNP detection is confounded by a highly similar homeologous sequence where a polymorphism between subgenomes must be differentiated from a SNP. We have developed and implemented a novel tool called SWEEP: Sliding Window Extraction of Explicit Polymorphisms. SWEEP uses subgenome polymorphism haplotypes as contrast to identify true SNPs between genotypes. The tool is a single command script that calls a series of modules based on user-defined options and takes sorted/indexed bam files or vcf files as input. Filtering options are highly flexible and include filtering based on sequence depth, alternate allele ratio, and SNP quality on top of the SWEEP filtering procedure. Using real and simulated data we show that SWEEP outperforms current SNP filtering methods for polyploids. SWEEP can be used for high-quality SNP discovery in polyploid crops." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/9.cover-source.jpg" />
    <title>SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"LBPaGwxP1wFL51ZIiTyJc32pEGH0ShI_6JbgdjrX31s","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460022","apath":"\/ggg\/5\/9\/1797.atom","pisa":"ggg;5\/9\/1797","ac":{"\/ggg\/5\/9\/1797.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/9\/1797.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/9\/1797"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/9\/1797"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/9\/1797":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Jyzndthf6c6pUqeYATXU8Mz6Qjyv9xFELZBnKseCZLVj4By\\\\\\\/\\\\\\\/5nQkvYtVNeYtdn3Kwu\\\\\\\/dK7I1FueTKgB+bPhiSbxDrz\\\\\\\/1sKtc1XeODbZJKgbhHazVw2t0OIW+XTV8d0kryIvMTcwC4glEz5wCA\\\\\\\/4kTzpLwcj7M0WVoZNrGyrkv4d8EqOFRM3ez1yXQzNiOcocEWfSgK0+bIrnFiN9H1FCPKp2p7\\\\\\\/ohkB5Qaj4R3Awi21\\\\\\\/8VxOtAFi7JTgzXmdgmB+wvwgN2uV1qQsGbfuUt9Iy+94sCSjbJgPU5rteLgam3Po1kg14PMQwi9gAYV3GeiOo\\\\\\\/ea0lZf5T2UgR7iOvEFzYmNX\\\\\\\/j5fPLCdzXb3FEJ9CrLchzhvdb9YvkIOyopHj29hzgBhwUlhZm0oPmTSjPdV\\\\\\\/8uez8yv4uluh9QmrQbaq+kRHfNQlI4Ik10eD8SJyrqJI9DD+up\\\\\\\/mtnh61cDKq7t7PwvCNXK6YAXJgoY8oJSovRv8qP9N+LqJVtAGZ7P2\\\\\\\/aOy+gC1lDSjuKu2QJy03bNcXtdBRutoLmyQOcSBkUj5themHmgw+cfdSPPuJhrHWTlliTYjpFfhECpNI2+vfxFf9Awa6pK9IM6swAbeHCvs2xHy63BsmM9dmHYtHxVHtxdvCYpSXRFQU9o5OB0Qzx8hsrcP7nrCOSwUfPcUntHYy6hQXaHxfuSQFFOx\\\\\\\/V6HOjtNScDvjIo7u1YKSdsX8fCnQWFgxICSqDFlzU48pY6ru7LN6DWT3TspyOLJoOlaAA7SqeTAfh03g8fPshr4QMgzjjj53FUsIhlA41eEF0IjV0Am2nHcHj2\\\\\\\/35gkAUNMWPvs70YYWWe6tkZTZlcRIyxqWqLQWfypQ+I8KzUlod3l4iArerha\\\\\\\/g5hwoki8JKj5Fn03+sLsw5UtU0Vdm3caX+H6IGeIBRctmeRngR\\\\\\\/sODi6cS91QQAfWpIQJFPhi4xc1WzF9lwdpcJSqBCQfDwwTiU1uU+XJ3a7okZZX4B6wIYPRX1Qt7Gg1MTPjdFIcqPaWhXGypdGdG7YOvEsbb8hLNjzKoeOrC+h0PdRebo\\\\\\\/2JXa+Xunmsi6qUh9XSLyd4KCt8IfvggRdenBDhFbyD7YVbd5AHf0xgYgz5qEZ\\\\\\\/zrLNsTQAaOhNPdD77nACnVxdDKRUFlHmL0xPKyRl9GVMXRk2CZ0DunfWglvQOQz6qgbyRzgQQoZw+15fIp+YMgjAhYvSDs5+axgonO7+76dnPzPXta7TFXDNdRIYgXe1wigF3+SXAb3hUVb0fT6Z3\\\\\\\/V6HZ\\\u0022,\\\u0022iv\\\u0022:\\\u0022u42XE7Nke+Wn5JgWLJu9Vg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dcc6481c53149eaf494fe8bc8cdc983122f48b6adc540af6dafc9ae0d3e0ed02\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222uVwWeq+QwT25ivLd5DoGs0RK2FCgTUnlnFBfx2FmzdEqnBZnkIJj7sHnqoQcH25uJ9FWbH7hJQxfGjpNBIeKqJVmLghwLev6DY45envX4FXAIu9hzBAKPHFTxfYqIONxBNdJ++hmYu3lVxYKzyQjLY0xPQRHV9zNmfQPmvAUDM4387+tRPDx1CJJ8Mv4Tt20e\\\\\\\/Flh6Wd+2dleuTLktrgZZ0wzbUkfZCFqj0D+6xlyXfhbB8WjuF9L+kCalI9sqqp6MKioBcGxThoCpBIuqkQhZVv2dPnonk7YSQULoAVtvZTr3h8\\\\\\\/+FQ1ANVo7plwLmh32AMJvYHGBuIgh8Z+aYzaVusCSbYD6FxbZ9\\\\\\\/ZXsV0I3FvT9WcrdslVtpL2OgLQlKNMv91KjLg3zRsk2UgOkFAUo4+5gxBALuUXmcPEko2cKDzysEhfwyBWbPmRDPfiOpTihSTUlPCkcsvlMFxWq29Xc\\\\\\\/Zb+Xewkuyyexzj0l2U4nmFT8uxty9HdIhyhjnqn4p6BjFOmgsLMQlCDF5mLVwonDuo1ib\\\\\\\/qas3L8L6Te\\\\\\\/f3AwT7KV20QcYSn5KoAAWdUx22z9WKir6N1kHlL2jBfkngdy5VHUhgWO5k7Wq9lBobDEpkaFzkPF8xtpQXnAclAwMWvMAk8iTZRUut5zy5Y8AW8TU3OoZToIookJR424P68TJBHlA0LPu6K3wozKsqpec0Io5j1elJ5k3yNdFYCPpeEhsB5m68Or8BtEng5CZJg3z\\\\\\\/4DcSE1Tt+j2PYdL4RDNb7Kn5xCqNCG3d0ZhOzI7s+9yNZPlkWy6VJkUCvq12qG5HLlFQG9cEG8u3PbLuKIepzw+l6nMMZYrq3VmCNTKWA9rhA3UM0KfIq4fXbkxQboyTBsJpG0jG3gHA4D\\\\\\\/nuDRYYM2Hhc1w9mD87OX82hASVAi3wcSFNRI8Y1e9wySH+\\\\\\\/jAqvU1D2iFjyJEaOEvbOgp\\\\\\\/qt19zV4bWC7GeDI6RW3KflAYKNRbRPEmO2VEPEHzAdugL1zfSPT6F5RFjLmKEgXHIIXHcjb0Y5o00qvmg9IxwZvsDYEMqvlSJntkM6ZuSme48ZyuVFS204pDVW5VgVQmzC0bVL23ttQI9Cvc3Ly41F2iRUWZPY\\\\\\\/zWhPJ81YRg\\\\\\\/nxlKY9iu6F\\\\\\\/DwtKGCvA2hrvivOv9SCdj9nVHyAoEhyBTKa\\\\\\\/7NsGyHRk8AG86HMgzcGTmKyiDwFUb4SXpJ\\\\\\\/81GEK\\\\\\\/cOqhAA27EaJGcGIcwQVT4ICYSfS7Yb8khobu05dmVfezuVIDNcuZ7grqgxCpymag\\\\\\\/JKCzzyb+H9QoFxzFfHV+VGKQJlL1SUCpaJKGQ+L6amzhsGWYthNsfOJekJzX\\\\\\\/cx+jjxJokzVY2QI7jIUBpbm1aGYKhkV38MQcf1TBtNPfBZKpG74tXLZvGn0oBKbve+EMYTkIocLxWDvMlSBAMgN8eJz3X5kxHUtfoN8GWIbeFKFhGFlztAFngEoRV8H39LhuzST6+qvRW52CEsRFFo6lApJHrAONrLDrPWwO2xYCKHmDzTjq+RjBHEl3gs0AIQKN\\\\\\\/2Tq1BiC53qbZcQY2vjTt6hJKnNVW9sEllv3MED+KMIa5WZooF3VBetWID4XqDBJc9kNjXybCEoFyWp3blaKsRw+nSM5Re5BebB0\\\\\\\/V\\\\\\\/zDqVn1bxQEvFxv5Uth+6rBTkiKDLd3eRAc5MBzbON66QBNap2lJnwvr0ohIZ\\\\\\\/3VkUPTacZ8r40b3gRelHlfsOHu6hbjWtTvSn\\\\\\\/xmNI9bXuLP+cE6XasGFLlUIG9gp6vu+AZ7dDvR1whNp6P0wXPG4OHrbuN\\\\\\\/ebuWTQBMQufva++gPNedCzRtkdTAC6iF\\\\\\\/q8fu5L+IIuDfSGP+vXe9afCogNHHUNDJZF68x7\\\\\\\/+huEfew+DJd+XUzzfF62Jx+EtG8yuG5zZIX6uS\\\\\\\/+FhuvP+j\\\\\\\/T75qT8tPFbCNF1gIbMnGGw\\\\\\\/E6iMsQYLz0NZphq2xNxGWpQa3dXx3G49PP7G+fu12iPg4k0waGf\\\\\\\/bNv14AF5I6YA9hwJsGBHBg\\\\\\\/uVJu38awu+FWHRYXSbG\\\\\\\/ba\\\\\\\/P1YPHbsFy5Ol2WowOUNSjJsNg8Dz6M10PWPeAKbQJT+cChn9L5B\\\\\\\/9z7nLuEHNIaBApTPE18NqJHwtdS1Mwk4IpydFA+m7xmS7TJ2mvSruZ7vg3gywhNnVjDQm69zM9LgWMww0EFPJE4Y1eDeWpR3WP73rvEpE5jLEt8Vw0cSjS3rUHUV0JEoWre6eqaWIuV0j+WU6TSOvL44mGxypY6tbMK\\\\\\\/8fGwA57MS4zJsY88FXTcs9rwatDFxqFfrTwWnMNfsGawUpah\\\\\\\/P1ma1a3jgBio3p\\\\\\\/5\\\\\\\/F6MduWedkPEOGPfgdg+g+LAPzx5wWCww16QPioNkqHUnvd\\\\\\\/Nkd5Sz8RPREfOiKEKYe6D57ECuiACyDGv6cZouQ6VpNHCFwxZTjp\\\\\\\/YZbL1g5bArWOuPhI\\\\\\\/o0C85kVdvavXwgNhY3gMeh0Bt2a1ZVml63DVFZ5MV4dG8KlfMMjvRXNVuNFOB1PiEpu3YejazbeVD4VuLMwK9B0hdAmZc8BxIZvOEu7NTxkqol+\\\\\\\/eXbD7+D6Vz8tGHiBc8G\\\\\\\/nZfbiRUwwnpkD3HbOddebao9SDp7+NWdHEpdEUivk7r3NxZ\\\\\\\/aa0PY62dpRNaTO8yMbHAnDLbzG280vJkQ3zTMhLMBjSVzBvQDBmobU5znwg9ZO2r8UlqqS1cHgnfX57ipkA9bW1OanywiMQ7nqO5lXhKs\\\\\\\/MOYks427z68nbRSyqSJDd1FZUfb\\\\\\\/XJAkGNz9wZGZHV0tQPoD6AO5LHXnyA6s\\\\\\\/IOtQUvDCwxVTBmtfrnhisT8ofGXUYgqcPnl5dsa0Ps9RDGSX3IjpCSJw89sUJqqXm+bn2oszq\\\\\\\/vMXg9xuJINtCTzrqJ7bg8EJBvBuoLyHlU6oUNCib9rOYp5aO1trF2DlNpHTyz6xokehf9uIfPcQabJDinsvTpxMWRgVHKmG4GWfCNLwL9cFLiHF02eqYiP1Rh57PUzUsOOqdWsrBiavv27TtRHw6jpcmH2nqrO9ph5rhuyr3yvZGNHtnwMvjPCH4omyXocsVB6HLiHfYIU\\\\\\\/mYS\\\\\\\/BFuh+nZn3Z71ROr1yH46t5bp0S0VQjbcJoWutcwxy5jSBUdB1DWm3\\\\\\\/M7oI8xOvKE6NuwEb3t4wtbsVjxy1nc9AouNZgcb1OcR6WfCpuK0eeOYZuCmLzVdLIJnDbj3lJsyscSHWeyFTFIDqIVgjJu5zWhkLxDtl9MVDN0LcSWHNv7Sz1+Z30oP8czeFpTZGfjPqRZdqbsAPiYJQxyLntOc82BUgKH8OpoKkIoZ9lxOjJmv0Asb68Csx\\\\\\\/7ghyGQtwl68OKU3gXKcHRfgar1RofERhCAzqD9CUhj0ROhP9rcrqMh0d+07C6vDsGeg9EVYX\\\\\\\/Y5a7XBX6pbx+krRVc6ErP1OPYAhnb3CLFk+YVxOP4ZIZHudHll+0ji0FzSUny5A+7wVxKpCy\\\\\\\/hJchwvRuFaG9sLfunPpG8tMb42KADseZ6YxRjEK7GOMXxs2rbBBXMCDEMuHroPx7jT+ZSIciv9CfqTopmXcQ\\\\\\\/TURWZ09a5X43DsbbEBgzN\\\\\\\/q6wpGhHTXoJt3ZT1lj1RUS0fyV8KDU5OvBF5yg9QNJfo4EXyLdEwyMWR0oiT62fuVoSxN9kwJcuwLfyUTFeox6N1P5z6LSnTFopzJVvWeMyj2gMFrbs8aDOKLe9qhzYFlZWHmnohAt9K9gAzhp6nIGa8SP+Ia95aBO\\\\\\\/+jG93esoRdaW+kIHuCBNGnL3z77alvG\\\\\\\/IvskNfUHVwZJffluxt0GXHzGbuEmaaH0UKHXSyopC0kGN3mkHPxvFXp3jHOY+COe2Ig23uYLIc22gffwiYYbfqoVEKGuNpfELnym0dNLv4nbBeCTj06Ma5E\\\\\\\/WvEu9lYy8pEqzETSaL35pfdTL7chCkUBuHMX\\\\\\\/CBPxQejD77Ri0WqE7M63fv\\\\\\\/Ci3xGxVZYTiKGF9fsyTP2iJvTIOYxDsxC7l5SWoT3XGITTMpTfRx4U4\\\\\\\/PzNKQzsxY2kGfKbN3r16H3M\\\\\\\/\\\\\\\/px0FjarXdKGF\\\\\\\/1fvfLtbT28UvxL7Rtqe4SDKKPi6JtvMA8PnMzLpWMLU+W840ug91H7n7ztA\\\\\\\/65jr+eJMX6T2XuWWb3qXDb4hkw\\\\\\\/5dzqnLDj\\\\\\\/AyWKEz+JCn\\\\\\\/RlrSuyC4s+xAfkLcPuQ5cqJ1U2\\\\\\\/s4IUC18taKe6RRkUHX0qWsAk9Tx871i4uKkbYhB24MGtgYWgVWj\\\\\\\/asoZOGyOdQS6moBh\\\\\\\/F1uaHRLLt7x7BCarUEE0zo4wd3xJi7DofOhMsIdWqHryoNDjfGPlqAxfwmpmj4uKsEiBdXCFh5+pwxff6240ot5iUEOhGjWl6L5Nei3fNMz4FfG69HxGtT9YfdHXcgTgVJdaYzXL3\\\\\\\/4wOwm0WaVT24sw+L+O2IWijQePfkbQODMELBbxa\\\\\\\/Q5z4n92gPuWFFdLLjGGxo6Fa9xRXlrVIPl+H41aloKQ9eI1LG2da6IifcPe94nQtHQMnPp8qUAxngr+VlqwGghGEKmBJVdz6kpS0v37xxiPAQHrKq4Q1u9OVLS3+33RTcmDUNty7OEH6MDztQe3KYySK3uBXG\\\\\\\/Dwarau41zfw32rVZRff\\\\\\\/HsuFIJKlGaRmHrNTYMyc3SevD3EqOwoJWdYiVsa13vgO4Z+9F8gRZqcxmsVoqDlb4CS1H1+IBovoHjLoj04Bi0NWZM8wdsIY+dTA6Dr48ep19oIC\\\\\\\/AsTp+Xxy5dVbhrnqlTQvrEl519BEqDw5UJI7gNt1iMPEHbgQBKlHPNTcGqh696XLTDY\\\\\\\/l56Rvdd8XkPSd\\\\\\\/TMPXET3667JuEiAOWYw5GL1R3MfMzQz9qtZjzTOUtE9ApYx3NuWnfeulHV83sdyGAC2OKktQQaP+mCH14psAWlOhPDGSVk1O6GP6OG0DoMVbMbJUVAbRCcws7DrX7AmZNZmJAM7+YRSsGvs+x4dbFmrIZL1LRAzLOc4wdRX1aLf0sBXhIDM\\\\\\\/2VnWVhmh4YyuOIvDsjHl91sUNdH28rZq6Rrjksmx8BGp8zHs8j8m+7b6fI65iLFlSDwEFHYHufGq8LqloOtFz9vE7weM4Bs292L2NiBQExWZmjPTRithRyA8hNnT4fMo5AqDtKezpocen3JJ8kug4Z\\\\\\\/8nK\\\\\\\/NWmu+J6RqmL2\\\\\\\/VzymDRFm\\\\\\\/IAea3NwK9i5tFZhynXad5VhM0Rl0mHWwdm0WXksbsWLiGkObwgzkI9TjBrkmpXhxSsqoI\\\\\\\/BYMmhePc+STKCuiBqn69euLNw2PZAMCfP+\\\\\\\/wd6pQsFXPoVbi48WyWCZVP2ncEoRyBd4zjf3T8UT7vIB4rRZ\\\\\\\/wWet3q3ZbfDSDcUjOX8E\\\\\\\/RvpT9YwLE03Wr9cbEIQCXHE+TcIbbA4Gp5BPB72N\\\\\\\/B5z5e5H8+3m8ZDUFhXJJVyTNi43i+4arI9BGQUzy\\\\\\\/e2e\\\\\\\/tY0aJKNGadu8iK+y7e82yngFlo\\\\\\\/fmiIjrG0u1Gt79tjSdkurNBlE8MPgHRpJ1ZLkuCW4xWb14JWKcL2Wei4jaPmlej3KKrhhHflIDokvG9LwElaEybzMfgrcmNjglorfTsnoIzGlB+XzJSCzHrlySlaaapsa9q98ykjWtnmvVdVAkh7DLBQrzyk7Wca91xp4exgcxaTirj6yJYtEx36VbfBNQAMjMcX5gKUQdOM8ApEMx+K2E3kE5iOjRst300RfyKFvYULwCYxmcZmxevegrh\\\\\\\/09LKeHyPAH4L6m1SWq8mDgB2\\\\\\\/S07JpqBtvAij4l+\\\\\\\/NFez8Q5ASrwnzrChCv55nkYURWb6so63RtW50cZaNPQQW+oDPiKit6q\\\\\\\/6sbeWQdLxRlI6hrZiOWg7uFQ\\\\\\\/vg0ajpdaq9iof2x8XBd8GaS2O4Ty5AfebbqBLPSRKkX7QxbjIBM3tRA6pIumzT4Me8CpNsr\\\\\\\/4ugXCeeUIQHubqYaa0b7ZN8kB1IJg1q1TfmgC0Wzc8CoykG4xqwQKW+tbfqYLCmNCCqnKAZKYb2FeRLOzpZIAAfd6xolnO5VXG5Z\\\\\\\/GwU+M7pmJEnGrn7dsJOsOg3GdZroP347y+W8XTGFBl6aIS+vtLlTwigX0uVafBsRgFEgTM9NqvLDD59nuCAwZfyZpnGW7cSoVYqixadiEYv3KRvwUOUuEC4OBUluVDOLgfVg9xGi88uBBZCp10avx33nFYakCqc1VqfvsX3NcXfm5vmSZCaY4h5FTMnBvfGimo9fkSOvJkaaJWCAAnQWWLUfuXYXxljYjrOeg6P9mSBE7xy6bZ6ihSVasFePgqJrBuEFRtfV0gU5wtyIvzOOai6kU7RAIUCR1iWETZy18MNU6ruslleiUH7sZVOg3XS51byaBj8OnTmUIz8IVmOj4TWmJGuUShg9xJNxj5KATVoItn9+XE2\\\\\\\/1AsZwDzEVfUVJZ\\\\\\\/2tt\\\\\\\/joYk6T\\\\\\\/jvOIHp\\\\\\\/BcQb0yWGk6GwZLFCu7EPafUXKvB4pqaHsxy9Ub8baAzlA1cIbiyAizDGEfDrjnJ7mV12ywNj4OGlGZ3lRb4kouVFWZWl3BkEiCiz4P+tFcCUMwV9C7vFctlM443CnNZ73VHayUHo5auCwIgJYKI3CDXHI7Q9pTNtP0\\\\\\\/w8MBVeUN23rfIUhoPe0qSDLXKYAGUpk01VIuxJIftTa1Dpk52ifmTdB3xZiUnJqJr+m1d8k7AenlvoOyuA5zhL55qMgdzymrYvzLpXzZgdXBgxYeFOiu3zTo+3cxBg1sGgUX1JWryNddy8mulIh7wnRh+qsqJyoogYPEQYDjXN6dl0U0qeg3d9FGE9OpguUgldMO0J6LGKFLwQniY9LauTUdtCvDlI9fRWwaH3wX4WDnmeK6rwHNQhLMl1AaGQk311IMszVddcUAfUDOVcwdy2iGLs58TQxPmUXrudj\\\\\\\/8u+vdsAkHwEfDpBmDiYbFPzx3fHJeWqv7XJ3dSaArIhd9Uf+ohV7KhiBm3iIvnNVWnpT\\\\\\\/eGtKs+tjYL\\\\\\\/DsNylXD3SX8XV8+lCeWc1U3RsmRbjUaFSVEw2FrXT01e0Dq1USRcPZH\\\\\\\/fC7udxHFNdZqKehi\\\\\\\/IhH\\\\\\\/t7eVAcD4zuOb+t74IK3FoWjtnIK\\\\\\\/gCIa3F2uTBPdbiJGXEppzlUv0IbyItnjl9Z6KTDkOCvyuVZPwnYbXuRpMT7AiDcm7583bK4fzBs30Lh5pMFuz1IJKToU8K4S9ZFcSkDVQYs7AEfA6VHEal\\\\\\\/owKwZATu6rL6EpftrPnDg3PxP0BGROVn+M2+DrCoTyuycsPCef950Sh2Rs4rccibtLmdIMW9xvx5TfXvY7nczJLTCRmCDJ0Fxb3VzbsjhkdOPA8AEqAoxxN78MJh6cNNr3T9hjX940mn+Q05yUdp5JqGa9XOZKV+MeqqFiQosvF3B7RPNE6qnT8+TUQf\\\\\\\/VG7x9s9ffxIGCqLBA4hJufVOQF1Td5Howj2x5Po4pdpPdUy72RmbjnfIV6HcOdBc+N6uqY0gKcZwjgElT7j\\\\\\\/xVoHSUVZ70qvbeX6NqOsirTyA\\\\\\\/taFSbuP\\\\\\\/biM6aQGq2pdKiVu0761rNLqbdTU62Z3Y3Qr71il+eYXxmMk5D0FGYmkO23luZR4YqDm8Xpqhh6M3U4JUJ7wE+C1ANW0LTUQiscyHXcCfzbRuT8QJXXoYxEy2g19po6A3XDs\\\\\\\/4IMBStr\\\\\\\/EDd5sXDhqWNUD+7Hk\\\\\\\/IZfKEU2+H5huj4mLl34\\\\\\\/x8Isjz+QHNOW6GKbyQ8cVPR9PH0nd\\\\\\\/+ZbXecetbB3Mq860nV3p1Ek\\\\\\\/E\\\\\\\/WCYx1NS06hbdBQdAQlFqnYE5NB+sNbBEWAipsqBYbDB6w9oCD8ZMYJaWHjS2l+NgZ6lxL6TQs+sZ13gP8D4aAqYfoCgz3mcqEcYlMrLiNEKJkbETqe+hzo48B81kIOdjgX9lz1KIwCifOzutwPuedmc7fO3XfdjhN1IU3G\\\\\\\/ifP9InrRbPxKSuuSMMV8Drhdn1uo3n9111G+0561PUagjyLKDol\\\\\\\/djzDRjzpNVcXnQyD9SUWTG2Tq\\\\\\\/MOWLjTv1tHBP5W\\\\\\\/Ha2ruyA9FrsAHYBOG7PNdx+JaUm52x7QvhtpJ6WQNAnuh0bCVpy0uYY5G\\\\\\\/TW\\\\\\\/TQsi5kyDs9tk2E06yvtrWh5bjlVYhUvIDP8q6NPzTsqoWxOyEFCWBtg+TFznXaYWMzhKXlOROsQrAU1HSIuttWqAoPcrrpyZORSiLQYZEZgIUdyO77940eevnog0\\\\\\\/lkLRj6l3kvGcsMi\\\\\\\/cY7RsnOLpYVB8Mf+gIVOvjjnzENQNrZG9NgBUS2BddA91VMd\\\\\\\/BKQ\\\\\\\/v4Nb0A\\\\\\\/ANAHab+xryUWXoijB0fgoAKl3G5jNHpnou61+8agi3dka1WqH64Db+\\\\\\\/WEfPbxcBZaOHxTmo917C8uUe9weNOLXV9M971U\\\\\\\/h+Oq9aWCYCnr2I4MYwum2UD+aqc27F5KgVHcW1xefCXPYUA\\\\\\\/yagqSE06qvcIi2u+r2T+tmnxR4OIlk\\\\\\\/FP4kBfSTjHUQ7czmXNo4bhQYPsXgwbOHf54zsEqfitZUxj69aO9esPAJAeEO\\\\\\\/ag\\\\\\\/ohyWyj+qDmNNQHDU1ppcPmSR\\\\\\\/UODDuRAYcXhc1nygeI7Xgmb57t4K7XBKCT5Ut4bRS8oXpRUZRWopooav3TEe2AczIlVfoTkP1gWb1OFGPICK3MxHLTL\\\\\\\/d6Ua46srmThacjzl0EGKqOoIC77XmaJA3DbfPSZ9zyC6BelYaYne68+r3YMLYic+isqifJAC1KbFKSVkxmD0SoxPfiequSuZei\\\\\\\/8GGqdHtZcYstLpPszNpoK4NCb\\\\\\\/HocBNtNIckLkdPisqYgz5EjhhBSgtkvvs2KZXdTCZNrlO3iayuNwR\\\\\\\/i59jL69Tl\\\\\\\/bmCp+f2sxF1X4RbSYpBtL\\\\\\\/sLk4Yr7QLVCESlzDfvpRPEreWWlEBTOCSxayyvf1fNn0hPAtlGM4CEjEI1hnPfmFZaB\\\\\\\/Z6trKmzdB+1yF76zR0BH95qHX22bP3NiQa8k6g6ahjqknS+p0W11b2IQyPaIZbF5RSEkPUs\\\\\\\/pziNMQBdpKW2t68kQf5d07REzAYhU1loRMLWwI\\\\\\\/lmsTgpxVmfh5hEAQhX\\\\\\\/owWHZrIgg1CDqlx9AZyYzUPOfSMsGlwajrGvMbpfr7PWTkBNLdqLbp19OE2V2S\\\\\\\/S2uFuve5RXhAgbK+yiqLrfBPhtsLlRhGV7SlolpJJXnDUAg3z9nIMaAzkAEzlOnXBew36LoeevlTzrX74ra2xbgliinWn+F2JHzKZsZz0Esm8A6w6Sc0nR9ifgMXDzrWLfxwleKE7QAff9o\\\\\\\/p517Pf6LHaGW8fiLA+5ztHwsKnlAWRaUBrjpOzz4r3Bo4ABxAZ4IUh4e6f0Zu5xXB8f20aCU3UkQe74me4EIFLWvfx4o5\\\\\\\/CnXZ4L+XTRxZ8jk4JoNl\\\\\\\/A7o\\\\\\\/d6OoVVr2o1KcKOEGxClo58stYcJlnZgjeFOS4KLoStaFIpvTjjZjyhjZFta11ZT9EJnHa0VtOr8LNmpgIIqczonTLcZ+yZtr6BKKPbbPhF3ixpWAgLw8umnHDUoRwAWNGbRXAgaBe0XRhMECqjdL9PlvqdxbFRIU\\\\\\\/JalxJIc2usQpk0gjHsIS8rpupDiVKgb0gABisQvJ1dM6a62qO+yhZRscMcbNuQWC13XQ+q9uyyO66uoRhWuOBhA09v9yqgVBSOyXpfOMJj1yBWWuBLEhwzCzSRSCCOPhzNlTbWzUoJszgOpXw95qB7QdrYsvtPwEPUz8SwNmrl1yr7l+tqq1JHBeuVNWfKIkiQc\\\\\\\/7Dn\\\\\\\/\\\\\\\/WiDiKiz6jL9pvjF5X2JyESb7j4js1OtFUdDqjZmwCzycCBT\\\\\\\/eoXee\\\\\\\/f\\\\\\\/cRtQoHDAZczDAvLZeTxQBH6vPD30De2pnW35QKdpsWI0Z3BL4Fyha19S5conqvva9UZ2UBDrjEZ1H3kCmiMoO3sJW7QuyX0M2VJelawTbjFex7Yt05p9GKeAGA5N\\\\\\\/\\\\\\\/Iz8vwyuIX4C3kIkvwoUFupVRrs13F6eOTq6X2zbaVxW1tpCcmh+43VA3XzzzqiTspnRCR\\\\\\\/GyCaZQfJmYHWRGjFbAu0DrApfviehKXlNlbAoRLLkPAkuo0bNVpL2kSeXP5UTLDnbTThUWsGHCbCDiQhMYjarvQZb6Dia9eAiBcVP+qn3pmTbeAnx6JVs31V\\\\\\\/LqvN\\\\\\\/VwmeRf8mNm9d9QXUN4oH+BdzNsXKizHUvQzxy1XNb7goncTxuwK8UNdOLuSsSC7W5VxnQ9TaxD6\\\\\\\/5lBZJiWNTH5DYmWm6kjh6uApEVs6\\\\\\\/1J2dpCkCF\\\\\\\/9BdOtm8XwD0dwW0E9HbxayJ0\\\\\\\/Mli6h4aUNB+\\\\\\\/vbnZVenhigu7V7KmEbrAJVNMd6w45nlgj8VAzHFG4vHI+bdLe99CWy2QeFnMzSQK1UgCqPvvxKnnSoNlGXutuML4yXRKW2lCGjHBq++LC4mV0kcSzT\\\\\\\/W4wZ8T5tzd0mriz5s1eRvs\\\\\\\/dWiF5AWY1kTvrWW73SfodGzi3eNL3P6qEEcHDGj7Mkc4ClGgaMsy2wlgvC2LC0K2e6dNyYX7ZlEIGUsRg0fvJjBL+0jK2sfmy+BOrf+D4Za3I8pNa3P\\\\\\\/nFkkYrUzOBiXSHaR3zACU9Xd8mXfzqXBuUSRTP0bEK9UX\\\\\\\/9sfNkPCHF2MBiSfXJASJxbs33CojFT+HkvoleIoS+ekb65eRyyN1kkGpczUNcT9w9WilE4I3w12fPY11mdOy4MtxhGHAk3BtrZdIbJgE0Zwj23etybW5CkOURoWe\\\\\\\/5M+JVceX8ga2KVrb03OsIoJlrdXI8Emnf9fNwWbxKmk28G3qP7dEgsSXtChQ4+eTdbGaAM0SF5LHkuwzlKQpR3YVWpLeXY6OTao74PTRKN80EXkXBwZfwAUhrSd2Tzd3R0HVwKl0MBK6Ur\\\\\\\/SEz5CHCNb4eeEOuvbA2dq4li5qsyrPwvvnlZNFtviUKyyDfbeDxTJhBywPFI3wtz289T5stiQ6r7sTuKbmZJPmAzYqQQWa6oSJMr0pNvIVuZTNR9oI6NO+j2H6OMKsN4fKtJ2YCaLHaT4khNmL21b7uMXgLb9KZML6YXymHx8bUZ4zxEK97J4Lro56CVp7vGQmGTHynJsKJ7FcvVaawCfih7kpe6lrm90yk50hWivJfzZIo03XjKv7jkzvrw6WWwdjOGnMMEtG09C0TIj1O1imQ3FKByqfbG9TzZ1nJVrtt7OSuBWCMVWiZj49bA1D500DALQO7hffES0kCxJZQTn0y17c2Ls1\\\\\\\/\\\\\\\/vflCEcWg1B5HLtIP2XeSqCPj1yM0IvJHQOcQZH9GUcpp41xXuXGy1MhxWuklm2P5K16kHIvx4q3HIFD5owvcG2Ptgn+LaZAhLxYjy2mnIuo9Kls+USxzfUGRlFvZPB7Ng2QyPtgOTua75PPR4zYld1KtjJKasxpgdXrN+Qm7IzT2KXPciNiT8zSjWDVYJPig\\\\\\\/RYuoLzao9GHxJNy3Y3SwMI72ZvKDTFKpMAZtG+huaGlMKE4cUQ9mqfFJGHxYtyKhFbxp1QghGf\\\\\\\/Sf0qS4EDfNQ3krLFNPAWggx\\\\\\\/eXVQbXOkE+oi1EW22W4B4XFObyGENBc6qGKrsBZ+tlcKzj5RpHmpP7cWcG3BxYjj1Z0YtZyFsMxnCXUFFhAaDxm6EVaJEOe0\\\\\\\/u8Hr7kKcI+zOIIpPuPKuPgeuw+CfDtoW\\\\\\\/vbEWIJS+w58pnPYniVVKEv30mjOCPVeuyYeQSrD9IevGuRE9hcI8fjAM29P4ap5Z\\\\\\\/QIO2Ien7MzGJZIiUBytnPwlQpZXeEI55R4r4WG92J31pJzUUyeOKDn0AxTeqmg1D1O7P6\\\\\\\/1JB4UwTEvvb9gdvCq2w0xvCjGSOuK9reG3xJrzrRiCzXRTots2TW6uXbBoh9GmVmAmiD0LWcBKaUS9pWDIalrvW2D8nApyD9IVx1ds7ZJ8OD0FCL97Sdrnu9NaclXLysZ896Be80Q8q0cDRHAu8BZTY2pw4oEru5oBOy3uiR9d3hc1LJILjQWD+0mo04nGDIDPZGFiOZ9vzSKSsQOfy\\\\\\\/NfR8VkV30I+yT4EAjwP9mIDt97quOw\\\\\\\/xX0o\\\\\\\/sScqgIHA4XvDaAWrgbFXJDKS9JmymoQwCqsU\\\\\\\/kTM81JBwTfavO26Nmw1anqfxBV6d5Q8A1emGbyVDqP1GJyf5VeI7\\\\\\\/uSSEDlgwsHtQxdYfcl4pIStjGYVJhTmuJ\\\\\\\/B+YIjbzpvdWIgGp6jZGVGl3VPOWm4TepHb5AKS9kP76VEsDKuV2XPjYL8u4e0oSRMwKimrh8PowB9yIvNFE+u7Z4O+IngR+CSOAKlcXvS8uUhRP44CnVo\\\\\\\/iGlQzG+QUOhKbEA+nWQOlzrZjT5eeXPhJC0QG3Y2mPIcNbexHE6yyS3bkHMVAcD\\\\\\\/9gDQ31R4FLHlRFAqibMYum3bYMu8F7TPJ2ZY5m635NMFyUG6N6ccc03P58KNgGJbTUEynUl0nmpNXkpAMNSfMj9\\\\\\\/obRT0KDN3udl8VLHQrP80GRNyX\\\\\\\/KXjV1iTKCs44Nm7afeX\\\\\\\/8WsDVyxP8L06A0r\\\\\\\/zJXpm9tl8si\\\\\\\/cDQg2OpcYxLdzM0bMCWQNHE8rS\\\\\\\/xJG7AQ58OUiKfcgdIE6c92yrSWaURUR69tVBWABeK7oYntEpufGoP7TatgbMsrRyEmK6m1SEwRSPDnTS34HwxpS3G7YEpGFMWUQKVvhvg7AJFfUzO0hYr6rFmUwFY8g9UTln0YwIXAI7oSkpsGc29y3Jk1eIH7L+a40wWjqMBSqQNj6MN6rcsnhWakhQ2KcJCYplcnLdb87orQIwNPMw8WyJTAaKvgh4FB\\\\\\\/yaDjGwotBNj9g+UGGTx3rSx6AblC8GMKXlUxxmFp1CNF56QVo8h4tuqBstDFvHSLycESIqTYs8O8\\\\\\\/RCCBrs+A7tURN3WzJq3IMaYA+nmEmj4O7JMTWCIREF2Mzb\\\\\\\/nKKDSKmVjdixty\\\\\\\/c0AIAHm5SdBETlF6Y\\\\\\\/Z0+wwULKTTfaic4ogLPZkCgzlRXeIHKk4uYfQsY2CR20i73xlX\\\\\\\/KVClpTo\\\\\\\/nu6upQajzR\\\\\\\/kaA1h9oyBLVRSX4nTk2c7WZTYfy72+ScV07MY71dUsPDLCn7UvjBV9zeHeMrHwg\\\\\\\/InLJYMz2Ni+jFvP1Vl3vnshiBGtIIoI29cMeIiYyKS6xjZN39d4eNZv3sFBWJ6ZAY45J46zJBH7Pg6Kz24yCg\\\\\\\/i9Uk5oeHoH8DFvW919mOhO3e0+eXCB0cUEkTXFYhWTvVQJwYdvaPAxMnOSHYYwlj7dGJ3gqsPOE1wGB9QmMpL4\\\\\\\/wCQfxjFB7f\\\\\\\/njcyty2M1iqZUXaLJh\\\\\\\/mXaS3LRI6E5h\\\\\\\/nEdcQuT0Qye+R0ySRd8O0\\\\\\\/ktTdKkSI6+SAFPYKg0R82rQq9dCvrIgQmsInZKHzt\\\\\\\/w4SBP156gCoPFkmaUITSfmtDrKZ8k1cF1W6yFepBV033pX35pSsAeH1y5RBrFWCM\\\\\\\/HBSSfyxPeBGBiWStF+Tks\\\\\\\/C\\\\\\\/S\\\\\\\/lu\\\\\\\/T9HKd\\\\\\\/2VzsK4CUOVHMNWbpxaJgfuBpAUe1hnsYoRvZJYNBAlSXxMUN\\\\\\\/5EHD9JoG7VnUDh8MSx7yGlq5Ih\\\\\\\/lf2XAdlbez+N19SVXwzSBUJiqTasy5Yc0y4Fh03bD2l1rY2B5U6bdB4Cvxzk9gd+7ZnhGcHJAFVdApAEcgT4oS3KgDLmOBjkAGwzRLENLLDL5cMEKgLkYwVSsNUKQbb7qZu76Fx7h6\\\\\\\/nx30uyRrRn5cuO5q5KccOeAEJTMZMIvcUv5f4MaRtWSO9GPh88Paig6qotpYtIpMNhpGe8m3MR09D2LpklzcN\\\\\\\/ki474kNsEwuduK176I8x2ugN3G8dnaqg9XzWHlbA\\\\\\\/4KzeVlIuNhbvDn+vWja5dxYviXDhEik5D8FRywN2sHcElyQU2Nt\\\\\\\/1ITJQAOXRRhHWiAJuiYSvv1F4FAnunG6FZ\\\\\\\/qTednAYOEIVKimH2RDqUdq\\\\\\\/kyJdpx4GgoETlDKMMLozTum\\\\\\\/esjHTxLIYSNCtMSyl5o5zBpx906mfdinq\\\\\\\/uNoiYC1RMUU4WJtwySboU\\\\\\\/D6UON5AfBEoZMuFUpLPbGXbRYj2z\\\\\\\/vztyyn6WAL2Wm17QVO3AQEt\\\\\\\/fCviE\\\\\\\/djGfUh33aIfxziJqA+mzF0I5rrvjCqx7E7P\\\\\\\/AxJYaw2Gecs3Gf1M0cONtMXyJ67Q0kAAHHxy0GA7YgyLRbVuRmCY8b4HLpbUtRCOtay372UtzGnk1gEtBaWc+cWSxM2h0UysnlOP+9SOzoLPTRHPwe2ScoTdhEwcY7EfRJUACB87JthyxtK2WD5k8iWI9XVkOE7VcSYw7GTMHgF5ZGKHiJBhbGsacBnrh5NdHe5LaofESp+cHefztJKwH32\\\\\\\/9pKEkmlM9GnKq7433r001HT6FCenXAwPUngv26mqIM1th9sxuqbYmQL3pH1ZFsnHcnduwty\\\\\\\/mGelx4CNVzl7U+16oGEM7mR8v3nclrA58sZWjMF1jtUMWLUg2Nkxh8mig8ZMYP3hlgX8WvFxn9jm6MnAdLezjv7i9lQDfR1OlOWQl0gmzIGl8uL0KRupq0DE4ESJSu0oyir8fcFYdCOi\\\\\\\/YFyu8XDswcpJI+gzVjzbysFo7cV\\\\\\\/f8tEVrOEJ\\\\\\\/o6WtsIDApJYr\\\\\\\/JNzkJ4\\\\\\\/r\\\\\\\/tqOLvoZV5Fkp7KMz5CZCivcSibx+pPMis1uMMPm+NWBGeZwhJn8jk6CovIP3SYDDoIY8\\\\\\\/RD8nQvrcwfAY1AifcCjFazNAOYzA9NfPfTGjZ6af2lhH8PvE8LqEX4c905CBgiCB0kZ9xcR2f7lzGMArnBIsi\\\\\\\/E6G8lPiYExufrg6HRHl3NO\\\\\\\/jJXLknrB2qNGnP8iedXoDQa0D6RKwu0e3W6rIHGyL8OO9yAaifYDwamCCR2LY\\\\\\\/wEBtgBfSxYe7IyP4e1BTD0uarYQelEoqro6vS6PnCxJ0aFf\\\\\\\/Rqs93YCTZ+BQtCuqstMNEtyiMIUWHuppRnP2b3Yjh5cUNoK8mr2k7m6u5PFaONG3NcC\\\\\\\/brH1c9s6mB\\\\\\\/THLE1Wj73ao\\\\\\\/2ybVkU1+6djw2CwTsUHvgqwhswu9x4jvy+jWbbUdUwfWe9Q70t+URzyJ77\\\\\\\/uLE1FTjQP+UvgoNXKiafqWy6SvYV3zIR8VYqx+e6+fBCJUI4sPYGGC+oTD5wwSoxiz3KLDRsBv04micbMSXJ3FIL39sMn1mvCZb8qzsdh5o0SdT065MNOTEOenZrwfwvLI2i3oU01iuOmLkW6pV0cBQzko8227BYMJ4cArimLm47+KfBFqYv8tXhsGobu8DsAdVousOgCHCXNtdu9LTTg7RaReiziwmh2uybldE0eRRrF03oMikX4RlJjgwqicBsDFDL34VdefL0R6gzB6PBirzACiE0VOnFfiDoUazoMWYogwgaVChRH8xH0eeuwlHdxSFi46M1HabjAxzq365v0294+CTAUdvZKgWjCwozdamSRAaD6jEuqpOAxzgV8Y3UDkos1bpXaOuv7bfJWTeLFWV8FDgZdsS1Z4AmjDjTI8FXeu4lb0CxehlZrHGVc0oWjPdn2CDwe+9vA7OUXye78\\\\\\\/OtHTRJfpO5aPQMcqwdfhUXl1J8XJRHWSj4VsjwA7RK+8cdLYs6xsat4w0V+PI+\\\\\\\/PoTu9KEE0NPF3R8K0bwex2QmymIw29GxCgov0vqGxuWOTmJFRp37HALCO5cRw7rWxHyBJBu9Lqrv1qAtH1Q6jaj0em4o+KNIuk6J0\\\\\\\/mH6EqLCw7kfpQu+5SjqIV3lgt4r\\\\\\\/Y+77Ga3BjPfXm0KVvv42LxAGL5q52+Bv5jNBkgHN1TaZUcFRsUwZxGhw1slSj574a18w+96dhX1eNZrs+tfMl0u7buDH8amP6o5PxhYQsFbmdcYVzlOwsLvxDrHyMPyp4GJMJnBk5xZoX5QPCs1eGRDY3sf93RDChgWEKqCkKiYm3lw44rBuWS+tgEDQlbGy81ZLdpCXpQhTdF1Lggb2Jnbm68mjSSxV8vzSAK6lPE8pOp1fbLnZ9n7F5uFcgzd3IXNKiwUI2rvlrpcRZ0\\\\\\\/5XFdLZXGIee6SW7Mo5GYNNwr2HEOrKoK6WlJJw0Wo9CmJ0YeY7+kVB7oB4b60y7Q+4LNobAXqIlTYcNC2jzgQtq5R9YQ+TQmlqPbHjQZzrCHuJHUfRhmf8ciaWPQOUQGeWamk6syBlfFT9w09aMi81YQuqPW9BRxuUhZRs3en4OnCvHT1wepXG5LeNmn0THDBzygLgOieVkBtnSRYf48Mn85U2Ndhm1EtQ9yqvvcjucjYZhQf+ivPagm3dGv3Cgw35YG9Zef2gc9qmoZ5SXGWHchHpI549IgG4HXM1ZWB6MP12ONH3r4sY31K+5zjSw\\\\\\\/W4oTQjEFFy+j+Vu4yZ7os9Ykn\\\\\\\/M\\\\\\\/VXWu6y\\\\\\\/A8+TJn8luY06xwcrJZA1wtIbXkG1nXMvyVcdJwSsQkyeyL21C1SqJfbSvPGuSO2WIzCWpdW6p9g1FtMjbyPS+Pixg1PEblL\\\\\\\/IsxwqvZDzUlFd9jvv+zkEjIJiOEHg0Y34CaUJPI4cgKE5ZgyEHfVj0ZdA8k8m3Qu0HgUH+gvAid\\\\\\\/1BWvU3h\\\\\\\/fegA1F+LACoSwIeL6B8hG7N8+GVy8l8kWAiiweLPf5UZE1dcTNKVqd8P0sSwEgVm8EMF2dhS2KO1\\\\\\\/FopuXTvzvetZ2FZMU6hGf7qcy6LZElbeOB6moYLcVtM0NlYD7XqlXXFEhxFDE4liFX1ZKHH37ONuFxC6R0JrEHkphJlu9\\\\\\\/P3TN2tXXU5GOo7kJYCN+E5emlmSKKh49SSaEJk4y16glfrOyVMJyD+P8M\\\\\\\/k+FQ5MWgAwc\\\\\\\/HGFMj3j5AJ4q4XWbzrieMkXKtzT9jjHdk4Yz9V31KGXS0Hu3SokwcZXtXq8NUq6KxLeQtlixanGmR+diGnkLfWxeh07x2gOsihhc2PAbMwcxBc9YPzPLR68CXuwFZC613qbV3RmErOJzf\\\\\\\/5VtPUZCNL2SqQcqU1l1Sz04Rfw7cm\\\\\\\/lsoWnjKygBnsgBxAn+fpBnAU8v4DlYVwwqCVSUW5j3X2y8vPY8bs\\\\\\\/ljAQRCNPE73+TzYihJmYLZQtyy1B\\\\\\\/ZDGoLSwxCmGWjQ3LSlvfCPT5Em8PVaSDmLQuKYLrRKlorD01n5GMGekWWslOlLpRqJSzxUxlTjM391lztfwboE25cVhTlZqYucYOKHm2e2nZBqPcii\\\\\\\/JNSNnhHgokDFrpuJUn+R2muxMTtMU+NB\\\\\\\/+mXeg+REkljgKOylCfnIiwbZVZSa0h\\\\\\\/yUxQgHM1p3Qb21XXYtQMTXjzBfg5aZmCSW1WevOeuESOj6KEk+OA5EmTd5VDN6Hz9E333D7ZqhkwLiXsuFGPcUM0S0I3y2j2rBDjger\\\\\\\/Rg3EpIX8h3HQDEHmvcuG\\\\\\\/P2JNVtO6qLr7DpLuMDM+uzM97spASMAP6Ep\\\\\\\/4k6hkSb3buHdpEFRO2GOekrgsbRxUs1ePW3Dxw3F+Jnq1nd\\\\\\\/Kq1JWdC\\\\\\\/vxoUgM7BBvcRPWcweoJdDFrE3y5ayvX28k5nizu2POaPuPyoiO05O51N+aQZls2+9n89HbhkffCp9kCIPYAGMqVnxM5naQ\\\\\\\/0N1vaTOk1gCnF0eSbCKO2Pklrbl9pFlGUT20xTUrua8p9pj0Ed7p6CWhiuvRwoFjJSgVeCwuhesGmEPDOIYlQphb1IgmHS9v3lZo5\\\\\\\/MfiGKCox4U4krG01Rx4PCwxUvoSve8mLKnqKruC731TGzSY2pxyX2o+vGNc82yMfinCBj8ZYoNQqlaxS+COJ1g2VoRGDr9Zncmh3IN5JDsUhmiW1gJyfhuIbNjCVYsDjyomod\\\\\\\/esDCSE6xFK3IYQKmvqN+dPFTY2JaUFUbzKftLRgQUMWeLsmslXyyhdObCuqZpBa24mfWfsZ6tMJdo70aQFrjqGrSmARr7549YryrqF4dXxeZBMGvZMlgG\\\\\\\/zexoWAk+qDC24btkZkH\\\\\\\/X9x7Yt9S0rT1ZCwsnxZxDVyJtoOYXtM45UBzcVfUCR4YCsp5WjGk\\\\\\\/dGBopchF0HO0UiUM7t2sQx0ZGd7PG4sfYKorMMp5c4TIofvlrdNtC\\\\\\\/IdBRFWbnXgNC2p49UN8pK2xfMpPTC7Eyw4kf7rmaZJGu3k+YUqa++UeO6UsXwkIRbGtbw0NO9k\\\\\\\/djpRmRxPDzOr3hmhcicx9VDR0hcLxu4ajKLfdRQ\\\\\\\/uOWziKZTtL60N9tis744+qgBsAt\\\\\\\/q9el5LLkdPaJsluc7x4cT7X6cEMsUI18E926KOByZJlYo6S41tkdePeEELCS1aOVSzDCaATXA\\\\\\\/MbnnZ94cycomsoZocdVahGI5Fr469pGaIn2E\\\\\\\/BhCBKqPtz9GYrJiSsCb4KGU8xJ6bwm8rTx\\\\\\\/qhTA2UTqOvGTcSkxb85RUcyLU5m4NkJan2XUYpEL5NMjbIvctjoaBj9seEZZm7ybSfkvA\\\\\\\/ZK6tWgk3RaIQ4Tdw6Ysa8sGRV+U0xdKn+qcHOP1xkLO0CrF4qSt4ufg4cwEWYu0UiZudHvr5ZgmKZRR9HB2UhkP17d2R3DLxumdVNd3Lbj6Ho4LgpO20HXdRN+r+f1\\\\\\\/YriYLAYzlQYC1HHwkdomb674yyGtQwz2bn5WJFYfFZxYc86VQiZhUPOofk+0RYFNIIomig+zVNLEcTTwvzSmaWRJIAl5dGuYly9q0jaEiJV7VygtXHnBLJ8\\\\\\\/HthYVpTJN\\\\\\\/s4Pu3m1pGOCnrRGXzwfNPSOHmGQrrHPvSh7SvY6RKBkU93tsmz60Xjt\\\\\\\/Df6N+ils7jzCf1u0kxwZZ0Ja5ST2+BiNPSCy4Q4ElCYwFs6dZOsFwTs0Rn1LCfxaia7Y7gqPASthlsSWZwgRG1c\\\\\\\/iNhdVH7EM\\\\\\\/Y4OiBBoAbiyDUWfacniSTl5dWmc\\\\\\\/7OZD5e8kJKGDcGezD2\\\\\\\/a\\\\\\\/nbYBuGgMST4fDSwsNDK7pGUdfRN9s5AqvqZwbAGJnlHBPwRk9CsyN8Uh0H8LuLLb57OP1ziduZrjT2Zoe3ZF+9Ve8wZ3AnPuSAKX3kj9Ff7zh6FziA4wKaKe1ldQIkrYU\\\\\\\/kS4MF4nGnXVF9leqWSzIZwq1DtSvvmQoDWc7RhNBOP9+otm8DTHx0CvnCTJmtQimMb1wVPSwBuR4KXhx8JdlEni7tN8Dmwxus3JLkhklHEYnY09V\\\\\\\/jzpBz73Y9UwiJyOLXhV0ryPir6sgtYwTwIq5LFeUgTDbPSLg3zo\\\\\\\/WpD4fbWdoFe2VgZCKP\\\\\\\/VF6rcYrSntpSPZX4r2LihIkDp9Q9gZ3eF3wOB6YlSVjwHsnMX0giYtJ7E5sq6lpXn\\\\\\\/ibWWzvs4g3biz9V4xwjzWPIuMxlTXqBf8JSkFwluzFMejqDxN7PkZ0cShW9U1HRV1ZCpzm7Qv5GJ+md+ZPmgnbWb64jdwGbVysRLHSiFDWrtWPPjhbU7b0x1PI0Ow5QpkLUmpWeEaEVNwGK7IjHDAw2vvBJE4COBMXFFnfq\\\\\\\/Ogrmb3557f40gp\\\\\\\/FbRmJ0rqkdoow5J4NTeOgJInXj6xSitJ1p6toa3oOLgOOlyVHI33sZEqBKFUmAnR0bZmdzqp4nny4E08jFLWNXudvJW9SMXC87qcgOuaGeFk1i7\\\\\\\/BSF+dUJ6EL+9peOlzK4RoQha2YuerilWtRD916Em5AieQFtCTwU2\\\\\\\/NYW0WCgqgZc57tzctdlOe0sCJI8jh3mU4SJWEaU1xtb4mMzAOZqJ3voCh8PP9\\\\\\\/6CsxCTpl4fpzntl8ngf5xHeaqXPSzmKwB0CzJfKMXv1ImGSpUZLGoQZj\\\\\\\/PwkRYM4VYlORxmIxn8Obh1OD191NRzqUU9miKXygFryGhVfBXtqeWiCqKXqmHn4QsvLv7+SytEeaYcbwYVPn3mhMpoHPBW\\\\\\\/Z+KHTBH0aIDx5cPpW\\\\\\\/YAt4WUUtSs3C7ZBoMujZSI7zglESquhAmZUw+gFgp3Vz7q7mUOjXyG7QKIXIBTbWCVHrnKvMXNEkqE19IoRpL2EM6YpIXBOTfsfBY3oLOhN5RnjZPT1+aWOynZpa6AlR7yDqfLnGwOLCeuVOsGHPaMTQRu0w0V\\\\\\\/SO\\\\\\\/gE+wBc6WbhPCw7pc1lfwr6MvYlzDyMa5VysAIHoSN9yUqArVmqSbzrym377CW9rz9m+nHu92RNmXZF4yha9BiCYu7bvFtlmPgxpgCPDav6n+LagHt0yrMlZ6PNFhJ97cWDT3epcNSC8ydWurZ8nVa6\\\\\\\/PSIwebJXZygF9WK7ppG\\\\\\\/TnVCGi3DTZr630KqAL9HSkJyZ\\\\\\\/FvcYqU6w8SPZnP1E\\\\\\\/S0MzkrIMV2VbCMEn4kwdhPXuf4hsVi5Htd2aH9qlO7cVA5GuxoQPohRT8Y6tqzlF5inkKToqbG93XZ+AuyFJh0dzuJeB+Nz29pUh9Z8Ueo1w11N4HReWq5ZOrexwBb9r\\\\\\\/Q+jNp7we4GYc1JQ9xr\\\\\\\/+Wp9Sz2Vh5tN0r7vE37PFjjUilW2bUR13ttMU1Pzj80xJE1WVtmoTinYT+SIN+qZwqfOzkQz1d6FHfFjjl+ySuv27j0HBtey7qgGg\\\\\\\/Xue7ctuWwklcgUnbiJDkZRJ4HJJcQpKBZKzvbF54kfA5dYLLgmUP9Gmna2eBLVCTMfTRaJCSRuWMdpUmwt7SAX4\\\\\\\/VQ0ONq3MUFGweYDyzAtKePfkSBVtPFZcRcHqd9pojuZl\\\\\\\/YtWlCQWH5j4nZ0\\\\\\\/ulcI9+mcYJQR1EKgSO\\\\\\\/BKNMZgvL03wPDLvIEXkBa8d+UaVKHjYJXs7VOtYvZSICylEdOOlsOgjUp4WMzLqieIDNYy6XH10eeKtDT6d\\\\\\\/uyasjWqV24\\\\\\\/ohdhGjxjyPiKh2sI5WSWPvEltThwr8mVFTHczzcT13Vo4F7ov\\\\\\\/GTWtTA7qbsXSP5b6s+l28Sow+7it2K3u2Ts0YZqxhZIQoKe+anU40a\\\\\\\/cStUXzCSac5M8xat7kjNztzOBJxrQ2cybhR8IMOEzQC\\\\\\\/DnGXoT1Sxhu6vs3hLz2hNvh\\\\\\\/XWJb80aYxNlzaegUg0Oxt4MtRYeR3u7C6V5BR3VyjcHOsXhP40sCY6gVymnk6GZliBjqZvZ\\\\\\\/fhjetbwbfzHZlfHy55O9hl3dG04giS767Fumu8lCIwOONKSxEan6ecJMl+MSRjCZKL1tRDnQltBnMRPOiD4ktTzqU1ySTU9cmDRk9pZsMSEP5kZ5t4PqY0kifDgODugCOr0ImC5ppYZ5ZvGyqBhdw6sRjLFYfOdyWSOb5x8+gRYN1lGxeoEIdl2gUGMl5Ol+fIbv1bUD0yJFvmR644mg+gC+v2aS3pQm8nS0Wa7rj4aK5WKciU2SSINYAYSXiZ6Pt6jqi7hbexU7MyZnO8jtFuC+Leop49kTvY9demKJg6080OYjlg8KgzxmxfnsbWsZSsXHqxY54bNCFTNHNuir4WV1U91tQiqyi1b0lUbd9EhGUdGQR4UmcGp4uyUws7NYegcOlGUZgOuj15WnSVTHkHsS99x0fYa5f\\\\\\\/uofMJ\\\\\\\/tEkPeWTnTGOjgiy0OLfgT0vm9h+WsXkb98B5nhCn9cfzLqt4Y0HO++o+L509mQulK4WP9WE15RNIYxv1kdzpPrg3auH\\\\\\\/3yr\\\\\\\/d5elPsax2Avv6BR40\\\\\\\/lXYKkebqecIsLkIeUpjdnXgP115\\\\\\\/Xe8136NM8vo+DAOiAx7zE9OHijsWagok4e1PXBgUlL+1gz5f0S329ioXn+PhEaIA8sl8UaArGZ2\\\\\\\/4uDWfwjRxoxOGz2rLFawa1iZyohn37Xni8T1nnqiqccwB5X9F82KBppEjYNLsUDHjMOX03G7y2rN87fKAppyufjGpzqzUqzVenJncW0W9RcgHILwkbMveb4FTubKRQC1x5rzcr8xIIykn01QoWVdJdC5Yf50fOgL9i5fWjchmpzJX\\\\\\\/uBmLXl\\\\\\\/vAnjkhnZ71O3BnZWRRPxfPMtyUxbSpYLvs+zb2\\\\\\\/EoCrb8HodYk4FBzGEEmX2+7wzePxkoE0DtlUbOWMFVj89D0HCWL+4ZJWw\\\\\\\/7A7yZ8RP4iMiLjPwjSNZUbUsH2EJXZgIQSbpCmnCZ1MMH9UGTYoM\\\\\\\/PNWQtbDIM6Kr5c9RjcNyP0ZuASZK+07+\\\\\\\/yRa\\\\\\\/3fFMhmnysinfDsPE6uAW4RMLDSmBTUHWo7EuAJG9Dr18jIkgjT\\\\\\\/GBYTYwHH9iqCSHd1dOHnRZMf1Li0Hr6bGKW+d32Ogc5N5mSTWJbEuh35yj8wJdHHz8ujemR0s2fm\\\\\\\/Wx1PY0pJW2ltI7NCb0rAKEBN9mvbZF3QAjBk\\\\\\\/HBlhLIf7Yw5d8KNAVBrX\\\\\\\/f6ggu1C8iZySBlMFqyNO0igJdycT2k2dd0kTbB12AaCTp8y4LYdtjgXOrg6hs+4TwlbI1WOZd4GOirs7aDTAVfQIE0z5KoFeCYhGWZLmbjM9AJTmOvQm1wvVYg7BFOHaK2cWEmYGZAb1SPH5\\\\\\\/Ym27igbzaj9ok6eReHjrtY4nSThWiHKidM4lu8C0\\\\\\\/AYAzeWAqiDp6w8wXE2J8rSy+qJXtADZflIAG7uxQ4wH53jL2h5My\\\\\\\/ytasCokCstwH2bVAYk3SBpE8Jl1FhSwijjggriwMrOWPxf+LIELWMFIctuYkVmlq236FJEUEPlU9gTr5HpiIHtL6LRh+5BuXvYay5fxfo+SD+\\\\\\\/cEPIjNlpvCcIxayrAWiRilx3W1Zcx7unxsvHC1bvBRAGddQg39SYdTjdPB\\\\\\\/txIMEGnfh2iIn7AlhCMFJVtLdc3OH\\\\\\\/muAFBfvRxF7009qExgTNetp3DhHI9P4KuH3054YNqCwmTgABLvpyW609bOWr1v2alRU6ksr+v1o7tGR58HxFIdk2q4WtT4ZWVoWgXS6QW24I6VO1fejdx5icfWueGtyj6S49dcYI0kfoo49m\\\\\\\/GaoV+uRNEvLOQpN\\\\\\\/gvGllRGhUh\\\\\\\/y2ObFv2kgtIHb9atXekHy5+Ob6bUkLdHCGJiYvE\\\\\\\/+\\\\\\\/erAmv6D4QYcVRy\\\\\\\/dWcLhR\\\\\\\/pLim36HBqzvQi9\\\\\\\/0JOOOtgWJfk2+Boc2m10pMCNfMP8mVlWOl4Het8czDNZ\\\\\\\/04\\\\\\\/JZGG93vlyLbAPbUb7q7XZJdTkSC\\\\\\\/Aftwxh2ZJtpZCAYBqtJfNY0KmM1pBUy34F+ygJU7c7CF8jONdXf+t3kRmtc15l0Ke6MyUH1MeN+Nvb7y+pIWUz0uYdoSltt4ltL\\\\\\\/\\\\\\\/yAPs7dHuQ70aCg4JtSbzhl2pQAPgQcLfWA9WQrCklLvC6hBwb5q\\\\\\\/JZtGI9l6OGzgnPD4aQbnHfovMMwuIdaXQNY7pPrQZLHwHNbiQiptkiSt+iu+5y8txZXJ7Xbhf4GrffW1+AjvST24sUGjGYURS0BDCcllR6ijOd6GM0ei43T4McangXzKxS5XNczqzOaMWOO9B0\\\\\\\/oKUKGgbjTKewaf98bk0\\\\\\\/P57yB3e1KyBO6OSVtjDGd1LfUF3umC9bVObHpgtbBkwD6vH7znd8g3sSdx8meIWykAA4rXwXX7RRAQmvEkP7DXK3NDJ9pblEtnlTpEfnX1jsBwLTdC43mM025NjCF4BAwa8NAXs4IfqOGflqPbU\\\\\\\/yqdoNizUmTO4J9EMY6TJ+WcCtUf3\\\\\\\/ftTtNlx67jh2iCNaaJecOmgGUp0s\\\\\\\/0veK6WA9yazAohiFim2pFkdCV0a3Omve0du+Ypj6ROvdHui7iDRAQz0MCMd5S4No5igs\\\\\\\/dL919x7MuAoGRHfzECVfqgz5+0NadRlN6rDfIKRLUpmIxvQLOP\\\\\\\/AeGYB6hD4GkLZGt31SI1NdsJr0qsofGVWsRCKICpJMIiUScx6HBXKZn3TuwQOQWuLmM\\\\\\\/zKUgejHNWOWtgqOQKECzPWFPaZrJp9yU8Y1fy\\\\\\\/y6By1rigHbJ+1h2Y6gYSMgasjE2BT0j09cqdRYHBgDESRwVU3JbKB36rFW9JOmpRnxAizHX6AfdURWN7AWGdaQ49AUF0cxDkKp8ioSaeKM+37jK2pnLRN3DeWbW8VwuCDDj\\\\\\\/j6WYGpws04WhivbM3fD9V8ukWmDuEsPg9dt4dbw69eNKtIsCo82wduYORd8SYjXrfL+wSD47gN8PYxqsCz9SKq8asrO6GY4xsrmhpmxTQs6pFS7TbO5yzrszMmZKviUItqovAf38bByMGKJV04PCNKcPBDMTrJi1+\\\\\\\/BkS5zT9MAybZsaVDxNlXOyDPuwZOF3McQ8ywaql534ierNwpb5sHhikCa22Ga8DdkCcuvGDGtl81p3LhWBak7ybvHVbDmscuo+qeGZQksqKciFfuizOm32\\\\\\\/7b02PQIPM4sXk1+o\\\\\\\/V9To9CbpKrGQHkqsu\\\\\\\/tCHvv1GyVl52ihIeC\\\\\\\/5D4yX9e0s3oHk027ZKDyjukwgizeqMyRiJxaGekrOKwa+RxIhXR2gL3HNpJ0yyMKRWrsfrCcPL1H1tAJ2F1\\\\\\\/30Fcazq8c01KmJ4l+zF0eg9UlcFk5039GL+71mfA9dt0VRCJkGotMymU74ggg+9mmowvfalHo5DKwTTJIdhYJkwTR1FgCFhZk5yLskQ8F2PwrHXcQcPEwup3k+Eel6hf+I0ZecXJ8E898j8JQfLytQOjPpeHojhLWodqBsRqyyp+aHQrCzq6sY2RtALSTTTcL9Z4NnjUeXlWh1LZ\\\\\\\/N7GFFnFu5hyBgeduq1CB3hnW9\\\\\\\/BchdleBYkNudIdvr265M93YU4fli2ODI+snMaZ2lI1M1r9fL7fkO6JvxgVq9qlk1HQKU+7OiCHAQAM8NX6ds0DHc0VYM\\\\\\\/jXJysCM\\\\\\\/DdmRtuoImqAMrn1Q7RelcZdCTF3CzqYtAfQiB1e7C656hn3\\\\\\\/fFJVaZhdbMsxru8DlQTuz\\\\\\\/ajSYj8DIahkI9B63ihtBZLGV9\\\\\\\/cF5e\\\\\\\/9k8S7bYQ3lwVD\\\\\\\/7vbVoKbfqCQR6+scqosLzpcJx9HkzbFPtBVUkl1vH8DcieNz86XAlRhV+rHtAF9yTWkszwaS4bN2Vqjqc\\\\\\\/8plysSyvAxQ3qGmIpUQy7UZ0+qhKIG\\\\\\\/RCVxRu3wna01Jy5zhZbzLRD2kU4s1HThkgZQTt293HmMergmv9TNx2ig4rs1CLwbPARzMpJO+FT3fR6XR8jQYqX2KfuXp8\\\\\\\/TDyUPPWb1OGdmKlbVdhLgsZvH+R\\\\\\\/y0co3yK8Ya+y3pEs2wNuOiFVicKBc8W9s6Y5VdplzCyt5ZOPMLbCIgHw9KraT6PgsBQtDYrDnHw5G2fZKAqz8Jx\\\\\\\/HrvMCqIFnRLflDsNZKZ\\\\\\\/lSnxPYEg6HCKiBFQhHxwwRximVaRgRsK40v9lKC2d7SyIlgi7D9H6IIGFp0PBQFpFawhINOx6BCddyf\\\\\\\/qmnuQF6mmS2ht8OEYhfGi60A6\\\\\\\/XaRj0FXmGvLtchvJD+VcJ1KVkfh60uMdkdTC6R\\\\\\\/RNJk\\\\\\\/7p9ovDIoRh2EnnqE5zrS9fPbYezq3iARJgy+FFaxK00F6KzQliC2eooM88bIvUbLodcFrbS9KaRy2LAmCxp9ZyLlLI8fswSlsCc1PG8i5AwqYX5Ee8\\\\\\\/hwgKB3q9Up2ufr0G06ewd8rSjMfvB73DieVWrFTf2jNgEzpYTpjJODN9NI6yy4YYdtF5yDcWF3ZlRl1mHN+asNqLT936Nh7y6aBUOA7SBLJeuZYarLlmGC\\\\\\\/JIRQl1m6LyRJU+z2m71qjLEid316ypjnzRzEiJ3Y5hJ98jiVAJ4Z2zZGhI+m6zMADq+Iz3DfH7N0x6EWEIaikcabOvs22AhkKRRtCS6VvElISWKIzxBflcm7tfAy\\\\\\\/ChnAX6mtE01675lwYdXYowfsTyvaSSOPRaAbAmxJ1bbVJaAkE55mu2Ncv5+VjEoPtMrTwfuTFaCrS8hhwvNWpEfXNKq9NWpRGWKjFdsfoPONoCQOvUtqpjKhD5wtCVOPYQK0xSfKL2eIl7xEe0DVd9A3FXjFW3Sa\\\\\\\/kzUrhLOmaOMUm1unNPCy4lRuw1aWjm51cZUoRocyUvQ\\\\\\\/t6\\\\\\\/xnvujxOUMQn9ppWIBCs5E4O78ZHfbDIH1FLQebPI1NjgzJECda4SgiVlaiuu\\\\\\\/AB5FFowIrlR+Gy1qzvsj5OQnYBKXXh3DqO4KIiA2mQZPCDZ0tKu51PRLcMpeBkUm12qNNApsnusx9Bwds4pu0LoyBUMxLxaHYw1wnqRVi35TxZbXPFWeN4gDj+QHoyqo6DqkVIfBVUqxnQaSEN1Sl5tlgOTWgA\\\\\\\/nRgNURKTQ88RmlmfvLnApNOMwhgLgedChniDQ6LAWHz\\\\\\\/HodkL7147MFzqP4l7gnc0ND\\\\\\\/ERpzLouDHYAZ+iJBYNX2xR9eW5FxBQRgZJ\\\\\\\/g3KWWZxHfGlmLgZfNiPWo9W4dCGinPaX6fSjxbRm67VuDPS\\\\\\\/VjG1wJ8IGPDFfidzo2dmNGWrnf03dCFOVDrf9QHR6TkmrmkL6qukgQ7xDNnM9q1bN6zh6FUs+HYrCKkiegJkSJ7CEEINKF7\\\\\\\/T3BoUM6OufxzwhE\\\\\\\/RNuz1jmCS3jUORD6uPHOIChg+gg+kcD8UTJbpX0dZDZxaXgU4bDPyuwQW\\\\\\\/5OxdrMSgFtihc+Wq8fTX7YgMZZW7I6I3tX2\\\\\\\/079lDbeuIdXe6maO+dXuudY7G1z3l1+WkGRYzu7+g6TOqs90jUw+0ebmmHpKTv8stjKZQlCHr+QNfd3oQ8EsiUH2P6PVSBsZlTJWnO2YssBi1rxPlzP7J2oVF2jX60gKjd7PqBUPkK9pK8nAL8UeJ5SiIZv\\\\\\\/W+y6m8tM1u8qzYRDHpgLstn1eC72kYwYRj4XqnTK2yYGmmAE3YuWNPKEWR1metp9sLG7CuQNmC8aburG+ybVX84SM0LkHjNLpSFPpqen23qkv5IELQRkGxzbA3GSkrf8DYXBrJ1mgUhNv4cJDkVHei\\\\\\\/j6A5P+cMmiuf3QCbq3PJANDr1n+rkh6rP3r2+JDKC9LNYibrSw5lgc6pFRfGeSTxNdeQnF7uFxBhpLfad1ngnwXbADcW7F8Xlip1CYEvW1D6MXv1P3YXlD2sq7X\\\\\\\/dolMCzDUPGIkL+e4uVffIktqONzfqtP5rhStyFNAWP5aOls9VJVy6Qqh3QyhCwAVRTExW1jFVd9u3s9t1wMNjbp1txXme+ABLAPVQ5W4Id7VypBeCefHXDti8FSzMCIknNYX2VWUPJu7+fB8ZiwBOecv25Zh5SvTUfZd9yawgRFKGVAgxZO8nBBWZzBYrpM5MdVZp8v1nih15CIeL+Wgu0z7Q2u7lyY\\\\\\\/3eho0mu+2Ehv1JEZVs5N7oQo6x64n+XyQ963NIiXDzAPkooJmArBui8eAu4yOdFA0ZNBMimjPoASj\\\\\\\/sSz6xzjvuzkiKWDdWdvTenoclOO8DaXH8j4Z922QJ+RAUKV5tK+QI9jjbLy1qHAwTXWWOaCONKqekNedPIm\\\\\\\/Dp\\\\\\\/+JNsbiowT+vf3GDOe57o3ubXGXiTThbNCF+8\\\\\\\/2bsXS\\\\\\\/3hTcqh6S9i0HZyor\\\\\\\/r+E2LxpeBOe5uyM9V6hmutI\\\\\\\/Yc1s9oq2ffBZW8RMsmM9wDOSbIsYM\\\\\\\/Q4yvj66uH3KG9wzfTpVWHiH00eyvY1ecsGyG2Ixvd60GP\\\\\\\/Noq43K8h7kBYdpadey5uhbj8Jme42d\\\\\\\/aCupPu9+QOP3PjX2QkgnhH\\\\\\\/jbGCul0FLPvMqey6NN5uxdcEKAazKxxKLpecYIRYWraCLTvODGpuCSzZ9a1mcQj6ZM+Kn3KThVIGSnPcDkqip2VFO+xgd6K3rzFRw90BzxaHG4xhkO66NkCgNnXX2krUKJOtppXQVUg6Dpqr3XznUIBek5xuc3O3ar2FEdfjfbzkFx+Q\\\\\\\/thKaPNmaAX\\\\\\\/ieCF0Sc9OY5f89VdOYKhG3A9rqkwoxESvXQjywKaJtq2Q2sFoc\\\\\\\/TlqLb4\\\\\\\/8E5spRK2SzeUgJZiLYw0PsK29hixM7goNBIwU84NPD103VNDFXW2dc98AylOoYN5AtUJC42uZdXdq4zjVvSjDWqkLYtIz8Ki8h7J7AFjXAh\\\\\\\/GSxCPzsmOKZVLiwmzvY7otpnYpLiU9Jux6E82Gp5vU+q3oG7eA9QB87nAUf4Sl2dcQcJ14cVY1dCB4lriTUPPLajOXIUHZ4WIn9UvbdOZgZDeOyXTgTVmN5Ke\\\\\\\/3kO8jENEw8Sinzu0EYLvNaJJqGBeO6prKuhouYGPB3h+AS2FxnYpI2GFK4GBi6uXBy2Bn8hNq1lClRyAy2PiOV7ssRObxGD9aL45n3efsxXcjIWAtRAomEQ+OMnPnvwTWP2NhVDHRihYWyBNmASnwMgOrEbdvVaeYMigOZwtE4eG1kII7FBW\\\\\\\/XuHOceQ8Kggj0y6iXHaw20PppPCDUQJ9LBUHLticy8SF69ASq\\\\\\\/JijXTByw8QlBbYyeXDx5sNFFwFu5Wy1Ev5uuHTska2qKQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ccUOZWjKMyfMQ9BKmtF1Rg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002238f2e62a716997d234059b18b30fd6b6fbb5e8864e832882b82df64b5094e583\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dF3UHnAkieILnhBikMlmBg1Qh5ncdeOnhqFDcKlSM7VWmtow7+i7Pej8AZ9Ell\\\\\\\/35C4iGAI1K4krMS+4bZ9Lfktw2nK2td1cLQstu9kqzYu0G2Df\\\\\\\/OBSlk8vLJJ2mgH92AyUNWU+AlAo1mpsttL7dohWVyHiofe5ZB+sWvuD25M3f8jDmegQwFcHj4XVhZbiRYELKcMQOWMtYZGXCt3BWs4D0omq8Kymfr9Jfp4Z6nG0O+T82gMPyOG5BrdJwD+SWPUOSHw\\\\\\\/0O8yUCw+4KUuDhsfwAnJAOU2L9ddQ3MsVqMFh8m1jCxze32sdrSjCc\\\\\\\/nBPZ4eRjPWLuVinkylCJY2q4jc8dS92BADHVLb1MHdYvq7F+EBZnlWZkXl60NS1F\\\\\\\/fBnDa65\\\\\\\/CZmGMwrrbx4NOANsGob0NXIDRZ2K3os+zxlc7r9PJMfLMnGoPJhFgz5uNWL0XkIKHx4bcwQfqe5GkfMbr5El+tQJa2ZLlBTWQXakul0L7kL8OwWgZ2psrEdQa5Z3O97Qcoj7qgRL7saOZ0o\\\\\\\/pgRAJJ7SJK5s4ClC\\\\\\\/vcdk7VJEAKYIS62kApR0pZTe9SRX7MJzutvA5Tz\\\\\\\/mKi0n9DuLSRvNwG2DaJXs1iRvpOjIBdzRin6W2hI8+BOPkV6ZlNghvjRZ8dAk5FF6s0\\\\\\\/Lnh4mShijvA1FTYBQ+JxQ4w2HikpHEMJMNFoqKqT3m062Co573L09bJPcUGoFCoRYoO1\\\\\\\/K8ffB6kyooQQ96+cH59DXWAWEHejsDpDwY8lgdkWYesx3C+CQuNnVoDUFYtem1nKaL7+flitoWiMwyvpnRYnsknn5vDmbqv5htk64u8tRZqW2l4IrvDkO0gtSf8Ti7vDkPb1EKvSKXBSzEuK5+sjszn9i38OJoNQJc5\\\\\\\/OUbfev+9KipzVOhXCYUpV1KIFdFMg5bhRgeV9Cgq7AyXNZrNs04P0DILS8M1jk+jONNnxUYshoMhq2zlR8nahHIbyjy+LNHBDoSV8FhXHfv2rmr5T3vldMSew7fr\\\\\\\/9dhITE5P5rx0MA3u8hmdd8g1I8MKPzByK5wfwVVTVIY6mnsg47bsCzfOC8XZ\\\\\\\/Acnr57qYxis329odlR17bcsrAhr8XuNzl57t3aMfMFAOHmZRci2tvnMwJUyRmMA+KoVXK+zjZg4JLRSW3OrYB06XmR1QAZneA+fGjN2OWGNGUuFim5deITW2SYgvNiA0fLoE8uoaNkr49Qhe35vO1h4gzlBrcEhYJLdU4XG1vPhUpjQ1i0OUuw1XSq1amsPXXF+TykWBXFphskIEvzMs8Pl9QZFKfD59nVLXjRWWQhvFtfqOUiCvcUFdQXIv\\\\\\\/J72bHF\\\\\\\/haIRotehzwGcIakZ9Qs5rJjXbZG2ty\\\\\\\/qf6k4QEFNar1mZHUARpnuUgNk8JP0Zq4o+ck0hKk\\\\\\\/XOsmtfaJ2MVwtRVWNJs7fxtX0vHJ1gW1pClEKkdmv1HN2pOPb5gkNpp6JIWI+nZZ+\\\\\\\/OMWZ2S8Ttcu4oUHBkdHcFiS4Le0rI3iOjscxF\\\\\\\/EeB2EhKl5Aeno71MV7Btjh0tu7omAHL3xKbZ55VNIdbVIO62JlN\\\\\\\/Hmo0NcQzTOsy5\\\\\\\/BAu5qgem0QUuRBkBooYS7iH6gICnPckNrc6uaMmVxqzGyJZ5FkgesPVQHJMeV\\\\\\\/Gu3npcWWRzGfRRxHtNwjAB5thCoZZoRVsvH+6mJLz6g6WUX6Tc4HUmBrYEbYA9ZqTy1KDhhx730vv9f72PHWY442\\\\\\\/ZZptKaAwWWp8GQV42dEsRW+t53qCJGPE6dvNdr0UdZQ9eIOTOpMOi2Yg5+DyPMU7QiPkxzbJyKW2N\\\\\\\/zz9g+8pwEpO2XtEkmCK5uraCJob0Ub\\\\\\\/H9FsXyndYK04WX2v8hmde2TI4dOqwxwOP5LW1siSrrKzPXQhB2eRTD5Y6Ob8hcqqmyGyI8cSJ8BPxTd0oOFpKs6MHFLWotCsyV+5o7gNk53RT7fGfUOh9SFEMQbaAO28upIZt7rLTx2I7hNh9MMSJMasWrBt0\\\\\\\/ESwBTw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022warxiLsTXkIaxSNLDEpU+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d4f34d9eea00e3945d7d9b0d1994283e674c1c9218eb8650846c8fb2ab98ab7e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460022 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1797" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QdXhZAzUkV96gJxiMmCmzpC8M7rGuYXJJLPct8Kwoxc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1797" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ypDadmpT55L98Ju78cxZkUnoRo4Af6jvuJjRVgeVonU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1783" title="A Genetic Screen for Saccharomyces cerevisiae Mutants That Fail to Enter Quiescence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/9/1805" title="Comparative Transcriptome Analysis of the Cosmopolitan Marine Fungus Corollospora maritima Under Two Physiological Conditions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460022" id="node460022" data-pisa="ggg;5/9/1797" data-pisa-master="ggg;g3.115.019703" data-apath="/ggg/5/9/1797.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Josh P. Clevenger</span> and <span class="highwire-citation-author" data-delta="1">Peggy Ozias-Akins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1797-1803;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019703 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460022" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Josh P. Clevenger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josh%2BP.%2BClevenger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clevenger%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosh%2BP.%2BClevenger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peggy Ozias-Akins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peggy%2BOzias-Akins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozias-Akins%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeggy%2BOzias-Akins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pozias@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pozias@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/9/1797" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460022" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460022/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1797.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460022" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460022/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1797.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460022" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460022/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/9/1797.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460022" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460022/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/9/1797.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460022" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460022/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">High-throughput next-generation sequence-based genotyping and single nucleotide polymorphism (SNP) detection opens the door for emerging genomics-based breeding strategies such as genome-wide association analysis and genomic selection. In polyploids, SNP detection is confounded by a highly similar homeologous sequence where a polymorphism between subgenomes must be differentiated from a SNP. We have developed and implemented a novel tool called SWEEP: Sliding Window Extraction of Explicit Polymorphisms. SWEEP uses subgenome polymorphism haplotypes as contrast to identify true SNPs between genotypes. The tool is a single command script that calls a series of modules based on user-defined options and takes sorted/indexed bam files or vcf files as input. Filtering options are highly flexible and include filtering based on sequence depth, alternate allele ratio, and SNP quality on top of the SWEEP filtering procedure. Using real and simulated data we show that SWEEP outperforms current SNP filtering methods for polyploids. SWEEP can be used for high-quality SNP discovery in polyploid crops.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">polyploidy</a></li><li class="kwd"><a href="/keyword/peanut" class="hw-term hw-article-keyword hw-article-keyword-peanut" rel="nofollow">peanut</a></li></ul><p id="p-3">Single nucleotide polymorphism (SNP) markers are powerful genomic tools because they are ubiquitous in the genome, codominant, and can be in coding regions and act as functional markers. In diploid organisms, including plants, SNPs have become the preferred marker type in genome-wide studies (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rafalski 2002</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Shirasawa <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wallace <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lee <em>et al.</em> 2015</a>). In polyploid crops, however, accurate SNP detection becomes challenging due to the ambiguous mapping of highly similar homeologous loci, which increases the false-positive rate of detected SNPs. Despite this difficulty, SNPs have been utilized in some auto-polyploid and allo-polyploid crops with great success (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ersoz <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cavanagh <em>et al.</em> 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nagy <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Uitdewilligen <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Huang <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Clevenger <em>et al.</em> 2015</a>). In peanut, SNP discovery and utilization as markers have been difficult. There is promise in using SNPs from wild diploid progenitor species to genotype cultivated germplasm when breeding with populations incorporating wild introgressions (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bertioli <em>et al.</em> 2014</a>), but SNP detection within cultivated germplasm has been limited. There has been one SNP-based <em>A. hypogaea</em> genetic map (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zhou <em>et al.</em> 2014</a>) and one genetic diversity study using SNP markers (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Khera <em>et al.</em> 2013</a>) published. Compared to other species, the number of SNP markers in those two studies was low, with 1621 and 96 SNPs, respectively. In contrast, from a 180K SNP array in soybean, as many as 50,000 SNPs are polymorphic in a biparental population (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Lee <em>et al.</em> 2015</a>). In potato, an 8300 SNP array was developed from a set of 69,011 high-quality SNPs identified from six cultivars (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hamilton <em>et al.</em> 2011</a>). Discovery of SNPs, array design, and GBS application has been successful in allohexaploid wheat (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cavanagh <em>et al.</em> 2013</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Poland <em>et al.</em> 2012</a>). This success has been due to using highly stringent mapping parameters for subgenome-specific mapping. Essentially, if subgenomes are diverged by 3%, only allowing reads that map with less than 3% differences will result in reads mapping to the correct subgenome and will alleviate the problem with false positives. In peanut, this strategy is not useful because the subgenomes comprising cultivated peanut are too similar.</p><p id="p-4">Although the exact age of the hybridization event between probable progenitor species <em>Arachis duranensis</em> (A genome) and <em>Arachis ipaensis</em> (B genome) that led to the origin of cultivated peanut is not known, these two diploids diverged from one another only 3–3.5 million years ago (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nielen <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bertioli <em>et al.</em> 2013</a>). As a comparison, the A and D genomes of cultivated cotton diverged between 5 and 10 million years ago (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wendel 1989</a>), and the A, B, and D genomes of allohexaploid wheat diverged approximately 6.9 million years ago (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Devos <em>et al.</em> 2005</a>). Their recent divergence exacerbates the problem of detecting false-positive SNP calls that are manifested by polymorphism between subgenomes. <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Khera <em>et al.</em> (2013)</a> could only utilize 96 out of 1536 selected SNPs after filtering for a false-positive rate of 93.8%. The false-positive SNP calls were actually polymorphisms between subgenomes and not between accessions. This was shown nicely as <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Leal-Bertioli <em>et al.</em> (2015)</a> used these false-positive SNP calls to find evidence for tetrasomic recombination in <em>Arachis hypogaea</em>. <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Zhou <em>et al.</em> (2014)</a> were only able to map 1621 out of 14,663 selected SNPs after filtering for a false-positive rate of 89%. It is clear that for SNP identification in peanut, new strategies need to be developed. Historical filtering strategies that have been successfully applied to other polyploid crops still result in excessive false-positive SNP calls in cultivated peanut.</p><p id="p-5">Here, we describe a novel SNP filtering method called SWEEP (<span class="underline">S</span>liding <span class="underline">W</span>indow <span class="underline">E</span>xtraction of <span class="underline">E</span>xplicit <span class="underline">P</span>olymorphisms) that can be used as a tool to filter out false positives from a set of SNP calls. Our method uses the ubiquitous false-positive SNP calls and transforms them from a weakness to a strength by using their information to pull out the true SNPs that are polymorphic between genotypes of interest. SWEEP is implemented in a Perl script that is easy to use; the user only needs to supply sorted and indexed bam files and the reference genome used to map sequence reads. In addition, previously generated vcf files can be used. SWEEP will call SNPs using Samtools (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2009</a>) and implement our novel filtering method. If desired, then the user can filter further based on additional common metrics. We show the accuracy and efficacy of our method in cultivated peanut by calling SNPs in leaf transcriptome RNAseq data from six <em>Arachis hypogaea</em> genotypes and also on simulated data. SWEEP is also applicable for other allopolyploid crops.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Leaf tissue RNA-seq and read mapping</h3><p id="p-6">Six accessions that are parents of F6:8 recombinant inbred line populations were assayed: Tifrunner (TR); Florida-07 (F07); SPT06-06 (SPT); NC 3033 (NC); New Mexico Valencia A (NM); and C76-16 (C76). SPT06-06 has been released as GP-NC WS 16 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tallury <em>et al.</em> 2014</a>). These data were previously published in <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Clevenger <em>et al.</em> (2015)</a>.</p><p id="p-7">Raw reads were trimmed for adaptor sequences. Sequence quality was assessed using FASTQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) and reads were further trimmed for base quality (&gt;30) and base usage bias. Due to nonrandom base usage in the first 10 bases of these libraries, 10 bases were trimmed from the 5′ end. Trimmed libraries were then filtered for rRNA contamination by mapping to a set of known rRNA sequences using Bowtie (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Langmead <em>et al.</em> 2009</a>) and allowing two mismatches in the 25 bp seed.</p><p id="p-8">A <em>de novo</em> assembly was constructed using New Mexico Valencia A and the Trinity software package for <em>de novo</em> assembly (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Haas <em>et al.</em> 2013</a>) with the following parameters: "–normalize_reads–CuffFly–min_glue 4." The assembly was then filtered for redundancy if any transcript covered more than 75% of a longer transcript with more than 95% nucleotide identity. Final assembly statistics are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/019703SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/TableS1.pdf">Table S1</a>.</p><p id="p-9">Processed sequences for each genotype were first normalized to a maximum kmer coverage of 30 using the insilico_read_normalization.pl script of Trinity. Normalized reads were then mapped to the <em>de novo</em> assembly using Bowtie2 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Langmead and Salzberg 2012</a>) and the following parameters to only allow proper and complete read pairs mapping with no clipping: "–end-to-end–no-mixed." Average read mapping across all libraries was 59.62% with 16.95% unique mapping. Mapping statistics are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/TableS2.pdf">Table S2</a>. SNPs were called using Samtools mpileup with default parameters (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2009</a>). No filtering is used in this step because all possible SNPs will subsequently be filtered using our SWEEP algorithm. SWEEP was used to filter called SNPs using default parameters. SWEEP is available on Github at <a href="https://github.com/jclev-uga/SWEEP/">https://github.com/jclev-uga/SWEEP/</a> as free software.</p></div><div id="sec-3" class="subsection"><h3>Traditional filtering</h3><p id="p-10">For the purposes of comparison with SWEEP, we have compiled a consensus filtering procedure. These criteria include filtering for mean read depth across all samples covering a SNP to be at least four reads, for minor allele frequency greater than 0.35, for SNP quality &gt;30, and for SNPs that are not within 35 bp of another SNP. To accomplish these filtering criteria we have used the freely available software, vcftools (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Danecek <em>et al.</em> 2011</a>), and custom scripts.</p><p id="p-11">VCF files were first filtered for depth (four reads per genotype covering base of interest) using a custom python script. Then vcftools (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Danecek <em>et al.</em> 2011</a>) was run with the following parameters: "–maf 0.35–minQ 30–remove-filtered-all–remove-indels –recode." Finally, SNPs within 35 bp of another called SNP were filtered out using a custom python script.</p></div><div id="sec-4" class="subsection"><h3>SNP validation by SANGER sequencing</h3><p id="p-12">A set of 28 randomly chosen SNPs were selected for validation using Sanger sequencing. Sequence was extracted from 300 bases upstream and downstream of the SNP of interest and primers were designed with the SNP in the center of the product using primer3 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Untergrasser <em>et al.</em> 2012</a>). Two accessions called as reference and two accessions called as alternate were sequenced for validation.</p></div><div id="sec-5" class="subsection"><h3>Analysis of SWEEP-filtered SNPs</h3><p id="p-13">All analyses of polymorphic SNPs and false-positive rates were performed using custom python scripts. False-positive rate was measured by using the Samtools-derived genotype probabilities for calling genotypes and then searching for the following genotypic pattern among the sets of five, four, three, and two genotypes:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-14"><strong>If</strong> every genotype has reads with the alternate base at the site of interest, then the called SNP is a false positive.</p></li><li id="list-item-2"><p id="p-15"><strong>If</strong> at least one genotype has all reads with the reference base at the site of interest, then the called SNP is determined to be a true allelic SNP.</p></li></ul></div><div id="sec-6" class="subsection"><h3>Simulation</h3><p id="p-16"><em>De novo</em> assembled transcripts of New Mexico Valencia A were reduced to a nonredundant set of 28,967 transcripts using the Evigene pipeline (<a href="http://arthropods.eugenes.org/genes2/about/EvidentialGene_trassembly_pipe.html">http://arthropods.eugenes.org/genes2/about/EvidentialGene_trassembly_pipe.html</a>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nakasugi <em>et al.</em> 2014</a>). A copied set of transcripts was then mutated randomly at 1% divergence using a custom python script. Illumina reads were simulated at 20× coverage of the mutated set of transcripts using ART (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Huang <em>et al.</em> 2012</a>) and mapped back to the original set of transcripts to identify the induced "homeologous" mutations. The set of original transcripts and the mutated set were then used to generate three genotypes, mutated randomly again at 0.01%, 0.02%, and 0.0001% divergence. Reads were simulated using ART 100 bp paired-end reads with an insertion size of 250 bp at 5×, 10×, 15×, and 20× coverage. Reads for each simulated genotype for all coverages were mapped to the original set of 28,967 transcripts and SNPs were called using Samtools mpileup with default parameters. GATK UnifiedGenotyper was used with the following parameters: "-stand_call_conf 30 –ploidy 4." Traditional filtering methods were performed as above. SWEEP filtering was performed with the following parameters: "-s 1 -d 5 -r 0 –ultimate."</p></div><div id="sec-7" class="subsection"><h3>Figure construction</h3><p id="p-17">All graphs were made using R statistical software (<a href="http://r-project.org">r-project.org</a>) and the package ggplot2.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-18">SWEEP is freely available under the MIT license at <a href="https://github.com/jclev-uga/SWEEP/">https://github.com/jclev-uga/SWEEP/</a>. Any other scripts used in this study for filtering, simulation, etc. are available upon request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/019703SI.pdf">Supplementary Information</a> contains <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/TableS1.pdf">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/TableS2.pdf">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/TableS3.pdf">Table S3</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Sweep</h3><p id="p-19">SWEEP is a Perl wrapper script that calls a series of python scripts to implement the sliding window strategy of identifying true allelic SNPs between polyploid genotypes. We define "allelic" SNP in a polyploid crop as originating from the same subgenome, within the same locus, and being polymorphic between genotypes. In contrast, a homeologous SNP we define as a polymorphism that exists at homeologous loci and is polymorphic between subgenomes within genotypes. The user can input sorted and indexed bam files from as many genotypes as needed, and SWEEP will use Samtools to call the SNPs, filter them using the sliding window strategy, and filter them based on metrics that can be good identifiers of high-quality SNP calls (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Clevenger <em>et al.</em> 2015</a>). Additionally, the user can input a vcf file generated by their tool of choice and SWEEP will filter the SNPs based on the supplied options. These metrics include genotypic likelihood, read depth, and ratio of reference base to alternate base containing reads spanning the locus.</p></div><div id="sec-11" class="subsection"><h3>Sliding window procedure</h3><p id="p-20">SWEEP tackles the uniquely polyploid issue of detecting false-positive SNPs that are polymorphisms between subgenomes. To do this, the program visits each called SNP using a sliding window of three called SNPs within a set window size. The user can adjust the size of this window. It is recommended that the size of this window should be set to the average length of the sequenced reads. Confidence in the actual sequence of that locus that the read represents declines if not contained in a continuous read. Ends of reads have a higher percentage of sequencing errors, so further confidence can be attained by setting a smaller window size (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Song <em>et al.</em> 2014</a>). The SNP of interest is anchored by called SNPs up-stream and down-stream. The procedure checks all genotypes and looks for a case where: (1) the SNP of interest is homozygous for the reference allele in at least one genotype while (2) the "anchor" SNPs are always heterozygous alternate allele. If a SNP of interest lies at the end of a known sequence or only has one anchor SNP within the window size, then SWEEP will use only one anchor. The elegance of the procedure is that it uses the false-positive, homeologous SNPs as a guide to find the true SNPs. A flow chart of the procedure is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> and a representative sequence example with a detected allelic SNP is shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Logic for SWEEP pipeline. (B) Example of detection of a SNP between genotypes. Blue bar represents the reference consensus sequence. Green bars represent one subgenome-derived sequence. Orange bars represent the alternative subgenome-derived sequence. Bases in red are within genome polymorphisms and in this instance are the anchor SNPs. Bases in yellow are the true between-genotype SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1767413613" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Logic for SWEEP pipeline. (B) Example of detection of a SNP between genotypes. Blue bar represents the reference consensus sequence. Green bars represent one subgenome-derived sequence. Orange bars represent the alternative subgenome-derived sequence. Bases in red are within genome polymorphisms and in this instance are the anchor SNPs. Bases in yellow are the true between-genotype SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1797/F1.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/9/1797/F1.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466689" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">(A) Logic for SWEEP pipeline. (B) Example of detection of a SNP between genotypes. Blue bar represents the reference consensus sequence. Green bars represent one subgenome-derived sequence. Orange bars represent the alternative subgenome-derived sequence. Bases in red are within genome polymorphisms and in this instance are the anchor SNPs. Bases in yellow are the true between-genotype SNPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>User-determined filtering</h3><p id="p-22">Samtools mpileup calculates the phred scale genotype probabilities for each SNP in the following format: AA/AB/BB, where A represents the reference allele and B represents the alternative allele. A value of 0 is the highest probability that the genotype represented is correct and the least likely probability will have a value of 255. For example, 0,255,255 represents the highest probability that the sample is homozygous for the reference allele. These scores are calculated based on a Bayesian statistical model for characteristics of the mapped reads that cover the base in question, including mapping quality, base quality, etc. The SWEEP filtering option is simple, allowing three levels of stringency for filtering. The default filters out homozygous genotype calls if the phred-based score of a heterozygous call is less than 20, medium stringency filters at less than 125, and high stringency filters at less than 200. The lowest stringency filter was used on the set of SNPs selected for sequencing and produces robust results. Higher stringency can be used when the number of SNPs between genotypes is high and only the best SNPs are needed.</p><p id="p-23">Read depth can be an important filter for high-quality SNP calls. It is more critical in an allopolyploid context where sufficient sequence information is needed to discern homeologous read mapping. SWEEP allows the user to filter based on per sample average read depth. For example, setting the option to "4" will filter out any SNPs that have less than four reads per sample covering the base.</p><p id="p-24">Increasing the ratio increases the likelihood that the alternate allele is present in more than one genotype, which will reduce rare SNPs. In the context of SNP detection in polyploids from next-generation sequence data, confidence in the veracity of a rare SNP is low. This filter is best utilized when the number of genotypes assayed is high and total SNPs is not a concern.</p></div><div id="sec-13" class="subsection"><h3>The –ultimate filter</h3><p id="p-25">SWEEP gives the user the option to include the ultimate filter. This option is more computationally intensive and will require longer run times (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/TableS3.pdf">Table S3</a>). The default SWEEP program relies heavily on Samtools-calculated genotype probabilities. Manual inspection of alignments revealed that in some cases, reads map to the locus with the alternate base even when the genotype probability score of 0 for homozygous reference is calculated. These reads may have lower mapping quality or have been clipped. In the case of a SNP where only one genotype has been called homozygous reference, it is important to make sure no single read in the experiment from that genotype contains the alternate base, no matter the mapping quality of the read. These cases can cause false-positive SNP calls and lead to spurious downstream analysis. As an alternative to manually checking every individual SNP, the user can implement the –ultimate function, which, after SWEEP filtering, checks every base from a genotype covering the SNP of interest that was called homozygous reference. If one read covering the SNP contains the alternate allele, then the SNP is filtered out. Running –ultimate is recommended in all cases for optimal quality.</p></div><div id="sec-14" class="subsection"><h3>Leaf transcriptome case study</h3><p id="p-26">As a case study we used RNAseq data from leaf tissue of six <em>A. hypogaea</em> genotypes (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Clevenger <em>et al.</em> 2015</a>). RNAseq data are suited for SNP discovery in repetitive, complex genomes due to inherent complexity reduction and have been used successfully in many polyploid crops (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Clevenger <em>et al.</em> 2015</a>). As a reference we used a <em>de novo</em> assembled transcriptome of New Mexico Valencia A, the genotype with the highest number of sequenced reads.</p></div><div id="sec-15" class="subsection"><h3>SWEEP filtering</h3><p id="p-27">Samtools called a total of 610,942 SNPs relative to the <em>de novo</em> reference transcriptome. After SWEEP filtering with default parameters, a total of 49,876 SNPs was retained. After using the –ultimate function, a final set of 5025 SNPs was retained. This result shows the importance of the –ultimate function because in some cases it is difficult to select a confident genotype likelihood cut-off. We selected 50 random SNP loci to validate for Sanger sequencing. We sequenced two genotypes that were designated as reference and two genotypes that were designated as alternate. Out of the 50 sequenced loci, we obtained reliable sequence data for 28. Of the 28 loci, 25 were confirmed as true SNPs called using SWEEP for all genotypes and three were false positives for 89% accuracy. During pilot experiments, we sequenced 21 randomly chosen loci and confirmed 17 of them for 85% accuracy (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Clevenger <em>et al.</em> 2014</a>). Combined, these data represent 49 sequenced loci from four different genotypes per locus with a combined 86% accuracy.</p></div><div id="sec-16" class="subsection"><h3>Traditional filtering</h3><p id="p-28">Traditionally, SNP identification in polyploid crops has relied on filtering using a set of criteria that has been identified as representing how a true SNP will be described in a SNP call (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Clevenger <em>et al.</em> 2015</a>). For purposes of comparison with SWEEP, we have compiled a consensus filtering procedure relying on read depth, SNP quality, minor allele frequency, and physical proximity to adjacent called SNPs. Traditional filtering resulted in 69,874 of the 610,942 called SNPs being retained.</p></div><div id="sec-17" class="subsection"><h3>SWEEP <em>vs.</em> traditional filtering</h3><p id="p-29">A comparison between the computational times for vcftools filtering and SWEEP of different numbers of genotypes showed that the two methods are comparable (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The –ultimate function requires more computational time and is an optional quality check (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1/TableS3.pdf">Table S3</a>). It is important to test for the efficacy of the filtering method. Without a known set of validated SNPs, we cannot fully verify the SNPs identified <em>in silico</em>, but we can evaluate them based on a set of assumptions. The assumption we make is that given a locus with a minimum number of reads mapping to it, if all the reads mapping to that locus have the reference base for a particular genotype, then that genotype does not have a SNP at that position. If, however, every genotype assayed contains at least one read that exhibits the alternate base at the position of interest, then we can reliably say that the base of interest is showing either a polymorphism in all genotypes relative to the reference or a polymorphism between subgenomes. In the latter case the SNP is not a true allelic SNP and can be determined to be a false positive. We compared the SWEEP filtering method with the traditional filtering method based on these assumptions and using different numbers of genotypes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). For SWEEP, the false-positive rate is near zero, ranging from 0.01% for five genotypes and 0.02% for two genotypes (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Using the traditional filtering method, the false positive rate is, in contrast, very high, ranging from 80% for five genotypes and 96.9% for two genotypes. SWEEP filtering results in good polymorphism between genotypes. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the polymorphism among detected SNPs with SWEEP. The pairwise SNP polymorphisms between genotypes ranged from 32% to 66%.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SWEEP filtering vs. traditional filtering methods. (A) Samtools-called SNPs were filtered using vcftools and SWEEP and evaluated for computational time using combinations of all five, four, three, and two genotypes. (B) SWEEP filtering and traditional filtering were evaluated for false-positive rate using combinations of all five, four, three, and two genotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1767413613" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SWEEP filtering &lt;em&gt;vs.&lt;/em&gt; traditional filtering methods. (A) Samtools-called SNPs were filtered using vcftools and SWEEP and evaluated for computational time using combinations of all five, four, three, and two genotypes. (B) SWEEP filtering and traditional filtering were evaluated for false-positive rate using combinations of all five, four, three, and two genotypes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1797/F2.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/9/1797/F2.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466694" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">SWEEP filtering <em>vs.</em> traditional filtering methods. (A) Samtools-called SNPs were filtered using vcftools and SWEEP and evaluated for computational time using combinations of all five, four, three, and two genotypes. (B) SWEEP filtering and traditional filtering were evaluated for false-positive rate using combinations of all five, four, three, and two genotypes.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise polymorphism between genotypes. The upper diagonal above shows pairwise fraction of polymorphic SNPs relative to total SNPs called. The heatmap in the lower diagonal reflects the range of pairwise polymorphic SNPs. Genotypes are Tifrunner (TR), NC3033 (NC), C76-16 (C76), Florida-07 (F07), and SPT06-06 (SPT)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1767413613" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pairwise polymorphism between genotypes. The upper diagonal above shows pairwise fraction of polymorphic SNPs relative to total SNPs called. The heatmap in the lower diagonal reflects the range of pairwise polymorphic SNPs. Genotypes are Tifrunner (TR), NC3033 (NC), C76-16 (C76), Florida-07 (F07), and SPT06-06 (SPT).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1797/F3.medium.gif" width="365" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/9/1797/F3.medium.gif" width="365" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466717" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Pairwise polymorphism between genotypes. The upper diagonal above shows pairwise fraction of polymorphic SNPs relative to total SNPs called. The heatmap in the lower diagonal reflects the range of pairwise polymorphic SNPs. Genotypes are Tifrunner (TR), NC3033 (NC), C76-16 (C76), Florida-07 (F07), and SPT06-06 (SPT).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>SWEEP <em>vs.</em> traditional filtering simulation</h3><p id="p-32">To show that SWEEP filtering performs better in an allopolyploid context using a set of known SNPs, we performed a simulation. For the simulation we used a compressed set of 28,967 transcripts after redundancy reduction using the Evigene pipeline (<a href="http://arthropods.eugenes.org/genes2/about/EvidentialGene_trassembly_pipe.html">http://arthropods.eugenes.org/genes2/about/EvidentialGene_trassembly_pipe.html</a>) (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Nakasugi <em>et al.</em> 2014</a>). We randomly mutated these transcripts with a divergence of 1% to form a set of homeologous transcripts; therefore, the location of all homeologous polymorphisms was predetermined. From these two sets we generated three distinct genotypes: one genotype contained additional SNPs at 0.01% divergence; a second genotype contained unique SNPs at 0.02% divergence; and the third genotype maintained the SNPs from genotype two and contained a unique set at 0.0001% divergence. Simulated Illumina reads from these three genotypes were mapped to the original set of 28,967 nonmutated transcripts with 5×, 10×, 15×, and 20× coverage. This scenario mimics a situation in which the reference a researcher is using contains “collapsed” sequences representing a consensus sequence between homeologous or paralogous genes. We evaluated SWEEP filtering against Samtools with traditional filtering and GATK (ploidy set to 4×) with traditional filtering (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). SWEEP filtering greatly outperformed both methods for percentage of SNPs recovered that are true SNPs relative to homeologous SNPs. At 5× coverage, SWEEP recovers true SNPs at a rate of 65%, and that increases to 94% at 10× coverage, 98% at 15× coverage, and 99% at 20× coverage. Samtools with traditional filtering surprisingly outperformed GATK with the ploidy option set to 4× with 8% true SNP recovery at 20× coverage. Neither Samtools nor GATK with traditional filtering recovered true SNPs at higher than 8% with any coverage. The lower percentage of true SNPs identified by SWEEP at 5× coverage results from lack of sampling both homeologous loci at low sequence coverage. In such cases, there is no way to discern, within the experiment, a homeologous SNP from an allelic SNP. The higher percentage of true SNPs recovered for SWEEP is not a consequence of identifying fewer true SNPs overall because SWEEP recovered a larger number of true SNPs than Samtools and GATK at 5×, 10×, and 15× coverage and retained a comparable amount to Samtools at 20× coverage (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of true SNPs compared to false-positive, homeologous SNPs called by filtering method. Samtools and GATK with traditional filtering methods were compared to SWEEP filtering in a simulation with 5&#xD7;, 10&#xD7;, 15&#xD7;, and 20&#xD7; coverage. (A) Recovery of true SNPs as percentage of total SNPs retained after SWEEP filtering and traditional filtering using GATK and Samtools. (B) Recovery of true SNPs as percentage of total simulated true SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1767413613" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percentage of true SNPs compared to false-positive, homeologous SNPs called by filtering method. Samtools and GATK with traditional filtering methods were compared to SWEEP filtering in a simulation with 5&#xD7;, 10&#xD7;, 15&#xD7;, and 20&#xD7; coverage. (A) Recovery of true SNPs as percentage of total SNPs retained after SWEEP filtering and traditional filtering using GATK and Samtools. (B) Recovery of true SNPs as percentage of total simulated true SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/9/1797/F4.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/9/1797/F4.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/9/1797/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466645" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Percentage of true SNPs compared to false-positive, homeologous SNPs called by filtering method. Samtools and GATK with traditional filtering methods were compared to SWEEP filtering in a simulation with 5×, 10×, 15×, and 20× coverage. (A) Recovery of true SNPs as percentage of total SNPs retained after SWEEP filtering and traditional filtering using GATK and Samtools. (B) Recovery of true SNPs as percentage of total simulated true SNPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Summary and Conclusions</h3><p id="p-34">We have shown that SWEEP is a reliable method for identifying true allelic SNPs among genotypes of <em>A. hypogaea</em> and is a major improvement of current methods. We have validated SWEEP-filtered SNPs at an accuracy rate of 85%. SWEEP outperforms traditional filtering methods relative to the false-positive rate. The false-positive rate of traditionally filtered SNPs ranges from 80% to 96%, whereas the false-positive rate of SWEEP-filtered SNPs ranges from 0.01% to 0.02%. Using a simulation, we further validated that SWEEP greatly outperforms current filtering methods, retaining a high percentage of true SNPs relative to homeologous polymorphisms (65%–99% depending on coverage) compared to other methods (2%–8%). A similar method was developed and implemented in parallel to design high-quality probes for a 90k SNP array for allo-octoploid cultivated strawberry <em>Fragaria</em> × <em>ananassa</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bassil <em>et al.</em> 2015</a>). Their method uses "destabilization" sites as a method of ploidy reduction in their array design. These destabilization sites are similar to our anchor SNPs in our sliding window design and confirm that using homeologous polymorphism haplotypes as contrast is an efficacious method for discerning true genotypic SNPs in polyploid species. Using SWEEP for outcrossing polyploids or autopolyploids with higher levels of ploidy may not be useful. However, it is clear from these data that SNP identification in peanut cannot rely on traditional methods and that SWEEP is a reliable tool for genotyping using GBS and for identifying SNPs for SNP array design for polyploids.</p></div></div><div class="section" id="sec-20"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">This project was supported by the Peanut Foundation, National Peanut Board, Georgia Peanut Commission, and Agriculture and Food Research Initiative competitive grant 2012-85117-19435 of the USDA National Institute of Food and Agriculture.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-36">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019703/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-37"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-04-21"><span class="received-label">Received </span>April 21, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-06-19"><span class="accepted-label">Accepted </span>June 19, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Clevenger and Ozias-Akins</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.1" data-doi="10.1186/s12864&amp;ndash;12015&amp;ndash;11310&amp;ndash;12861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassil</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ficklin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mittmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development and preliminary evaluation of a 90 K Axiom SNP array for the allo-octoploid cultivated strawberry <em>Fragaria</em> × <em>ananassa</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864–12015–11310–12861</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864%2526ndash%253B12015%2526ndash%253B11310%2526ndash%253B12861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864&amp;ndash;12015&amp;ndash;11310&amp;ndash;12861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.2" data-doi="10.1093/aob/mct128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidigal</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnaparkhe</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The repetitive component of the A genome of peanut (<em>Arachis hypogaea</em>) and its role in remodelling intergenic sequence space since its evolutionary divergence from the B genome.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmct128%26rft_id%253Dinfo%253Apmid%252F23828319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjExMi8zLzU0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xNzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.3" data-doi="10.1534/g3.113.007617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozias-Akins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dantas</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The use of SNP markers for linkage mapping in diploid and tetraploid peanuts.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007617%26rft_id%253Dinfo%253Apmid%252F24212082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjQvMS84OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xNzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.4" data-doi="10.1007/s00122-011-1780-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harker</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and mapping of SNP assays in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DByers%26rft.auinit1%253DR.%2BL.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1214%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSNP%2Bassays%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1780-8%26rft_id%253Dinfo%253Apmid%252F22252442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1780-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252442&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.5" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTc5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.9.1797.6"><div class="cit-metadata unstructured">Clevenger, J., Y. Guo, and P. Ozias-Akins, 2014 Single nucleotide polymorphism (SNP) detection in cultivated peanut using the diploid wild progenitor reference genomes. Proc Annu APRES Meeting 46: 55.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.7" data-doi="10.1016/j.molp.2015.1002.1002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clevenger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavarro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearl</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozias-Akins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">SNP identification in polyploids: a review, example and recommendations.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2015.1002.1002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2015.1002.1002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2015.1002.1002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.8" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, and 1000 Genomes Project Analysis Group, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xNzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.9" data-doi="10.1007/s11103-005-6801-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beales</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogihara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doust</span>  <span class="cit-name-given-names">A. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative sequence analysis of the phytochrome C gene and its upstream region in allohexaploid wheat reveals new data on the evolution of its three constituent genomes.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DDevos%26rft.auinit1%253DK.%2BM.%26rft.volume%253D58%26rft.issue%253D5%26rft.spage%253D625%26rft.epage%253D641%26rft.atitle%253DComparative%2Bsequence%2Banalysis%2Bof%2Bthe%2Bphytochrome%2BC%2Bgene%2Band%2Bits%2Bupstream%2Bregion%2Bin%2Ballohexaploid%2Bwheat%2Breveals%2Bnew%2Bdata%2Bon%2Bthe%2Bevolution%2Bof%2Bits%2Bthree%2Bconstituent%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-6801-z%26rft_id%253Dinfo%253Apmid%252F16158239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-6801-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16158239&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232555700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.10" data-doi="10.1371/journal.pone.0044112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pangilinan</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP discovery with EST and NextGen sequencing in switchgrass (<em>Panicum virgatum</em> L.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e44112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DErsoz%26rft.auinit1%253DE.%2BS.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De44112%26rft.epage%253De44112%26rft.atitle%253DSNP%2BDiscovery%2Bwith%2BEST%2Band%2BNextGen%2BSequencing%2Bin%2BSwitchgrass%2B%2528Panicum%2Bvirgatum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0044112%26rft_id%253Dinfo%253Apmid%252F23049744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0044112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23049744&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.11" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.12" data-doi="10.1186/1471-2164-12-302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitty</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Single nucleotide polymorphism discovery in elite North American potato germplasm.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHamilton%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D12%26rft.spage%253D302%26rft.epage%253D302%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bin%2Belite%2BNorth%2BAmerican%2Bpotato%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-302%26rft_id%253Dinfo%253Apmid%252F21658273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21658273&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.13" data-doi="10.1093/bioinformatics/btr708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ART: a next-generation sequencing read simulator.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr708%26rft_id%253Dinfo%253Apmid%252F22199392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOC80LzU5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xNzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.14" data-doi="10.1186/1471-2164-14-717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of genome-wide single nucleotide polymorphisms in allopolyploid crop <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-717%26rft_id%253Dinfo%253Apmid%252F24138473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-717&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24138473&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khera</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roorkiwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sriswathi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single nucleotide polymorphism–based genetic diversity in the reference set of peanut (<em>Arachis</em> spp.) by developing and applying cost-effective competitive allele specific polymerase chain reaction genotyping assays.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.16" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.17" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.18" data-doi="10.1534/genetics.115.174607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abernathy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavarro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tetrasomic recombination is surprisingly frequent in allotetraploid <em>Arachis</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174607%26rft_id%253Dinfo%253Apmid%252F25701284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OS80LzEwOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzkvMTc5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.19" data-doi="10.1111/tpj.12755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development, validation and genetic analysis of a large soybean SNP genotyping array.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D81%26rft.spage%253D625%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12755%26rft_id%253Dinfo%253Apmid%252F25641104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25641104&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.20" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> 1000 Genome Project Data Processing Subgroup, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xNzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.21" data-doi="10.1186/1471-2164-13-568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bharti</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prevalence of single nucleotide polymorphism among 27 diverse alfalfa genotypes as assessed by transcriptome sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D568%26rft.epage%253D568%26rft.atitle%253DPrevalence%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bamong%2B27%2Bdiverse%2Balfalfa%2Bgenotypes%2Bas%2Bassessed%2Bby%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-568%26rft_id%253Dinfo%253Apmid%252F23107476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23107476&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.22" data-doi="10.1186/1471-2164-14-100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehenni-Ciz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abberton</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milbourne</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A hybrid next generation transcript sequencing-based approach to identify allelic and homeolog-specific single nucleotide polymorphisms in allotetraploid white clover.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-100%26rft_id%253Dinfo%253Apmid%252F23402685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23402685&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakasugi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crowhurst</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bally</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Combining transcriptome assemblies from multiple <em>de novo</em> assemblers in the allo-tetraploid plant <em>Nicotiana benthamiana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): DOI: <a href="http://10.1371/journal.pone.0091776">10.1371/journal.pone.0091776</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.24" data-doi="10.1007/s00438-011-0656-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidigal</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnaparkhe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Matita, a new retroelement from peanut: characterization and evolutionary context in the light of the <em>Arachis</em> A-B genome divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DNielen%26rft.auinit1%253DS.%26rft.volume%253D287%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D38%26rft.atitle%253DMatita%252C%2Ba%2Bnew%2Bretroelement%2Bfrom%2Bpeanut%253A%2Bcharacterization%2Band%2Bevolutionary%2Bcontext%2Bin%2Bthe%2Blight%2Bof%2Bthe%2BArachis%2BA-B%2Bgenome%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-011-0656-6%26rft_id%253Dinfo%253Apmid%252F22120641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-011-0656-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22120641&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.25" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.26" data-doi="10.1016/S1369-5266(02)00240-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rafalski</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Applications of single nucleotide polymorphisms in crop genetics.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DRafalski%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D94%26rft.epage%253D100%26rft.atitle%253DApplications%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bin%2Bcrop%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5266%252802%252900240-6%26rft_id%253Dinfo%253Apmid%252F11856602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5266(02)00240-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11856602&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174097000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.27" data-doi="10.1093/dnares/dst033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuoka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsunaga</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdst033%26rft_id%253Dinfo%253Apmid%252F23903436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvNTkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE3OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.28" data-doi="10.1186/s13059&amp;ndash;13014&amp;ndash;10509&amp;ndash;13059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florea</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Lighter: fast and memory-efficient sequencing error correction without counting.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059–13014–10509–13059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059%2526ndash%253B13014%2526ndash%253B10509%2526ndash%253B13059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059&amp;ndash;13014&amp;ndash;10509&amp;ndash;13059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.29" data-doi="10.3198/jpr2013.04.0017crg"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tallury</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isleib</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copeland</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balota</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Registration of two multiple disease-resistant peanut germplasm lines derived from Arachis cardenasii Krapov. &amp; W.C. Gregory, GKP 10017 (PI262141).</span> <abbr class="cit-jnl-abbrev">J. Plant Regis.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">86</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegis.%26rft.volume%253D8%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2013.04.0017crg%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2013.04.0017crg&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.30" data-doi="10.1371/journal.pone.0062355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G. A. M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A.-M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borm</span>  <span class="cit-name-given-names">T. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062355%26rft_id%253Dinfo%253Apmid%252F23667470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667470&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.31" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergrasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer3 - new capabilities and interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC8xNS9lMTE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS85LzE3OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.32" data-doi="10.1371/journal.pgen.1004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Association mapping across numerous traits reveals patterns of functional variation in maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004845%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004845%26rft_id%253Dinfo%253Apmid%252F25474422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25474422&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.33" data-doi="10.1073/pnas.86.11.4132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">New World tetraploid cottons contain Old World cytoplasm.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">4132</span>–<span class="cit-lpage">4136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWendel%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D86%26rft.issue%253D11%26rft.spage%253D4132%26rft.epage%253D4136%26rft.atitle%253DNew%2BWorld%2Btetraploid%2Bcottons%2Bcontain%2BOld%2BWorld%2Bcytoplasm%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.86.11.4132%26rft_id%253Dinfo%253Apmid%252F16594050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODYvMTEvNDEzMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOS8xNzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.9.1797.34" data-doi="10.1186/1471-2164-15-351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Construction of a SNP-based genetic linkage map in cultivated peanut based on large scale marker development using next-generation double-digest restriction-site-associated DNA sequencing (ddRADseq).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-351%26rft_id%253Dinfo%253Apmid%252F24885639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24885639&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F9%2F1797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/9/1797.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1783" title="A Genetic Screen for Saccharomyces cerevisiae Mutants That Fail to Enter Quiescence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/9/1805" title="Comparative Transcriptome Analysis of the Cosmopolitan Marine Fungus Corollospora maritima Under Two Physiological Conditions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/9"><span>Volume 5 Issue 9, September 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/9.cover-source.jpg?itok=EE-GjT57" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/9/1797" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3cDVF4gX-KgZm3tywaV64cNZygbQ3DulllEITJndpB8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/9/1797" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/9/1797" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/9/1797" class="active" data-icon-position="" data-hide-link-title="0">SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460022" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cWq_cPEUF6LHZUcx43VR0Ky-nkmx93Nv2hP2BXPQYO8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/9/1797.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/9/1797" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-vVNJ_OZnho1pjKZVYYtH6n59Qdk-pK6Hs6LS-FfkLlg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/9/1797" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/9/1797.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460022" id="node460022--2" data-pisa="ggg;5/9/1797" data-pisa-master="ggg;g3.115.019703" data-apath="/ggg/5/9/1797.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Josh P. Clevenger</span> and <span class="highwire-citation-author" data-delta="1">Peggy Ozias-Akins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1797-1803;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019703 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460022--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Josh P. Clevenger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josh%2BP.%2BClevenger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clevenger%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosh%2BP.%2BClevenger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peggy Ozias-Akins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peggy%2BOzias-Akins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozias-Akins%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeggy%2BOzias-Akins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pozias@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pozias@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/9/1797" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/9/1797&amp;title=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/9/1797&amp;title=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/9/1797&amp;title=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/9/1797&amp;text=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/9/1797&amp;title=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/9/1797&amp;t=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/9/1797&amp;title=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/9/1797&amp;title=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460022/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460022" id="node460022--3" data-pisa="ggg;5/9/1797" data-pisa-master="ggg;g3.115.019703" data-apath="/ggg/5/9/1797.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Josh P. Clevenger</span> and <span class="highwire-citation-author" data-delta="1">Peggy Ozias-Akins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1797-1803;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019703 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460022--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Josh P. Clevenger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josh%2BP.%2BClevenger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clevenger%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosh%2BP.%2BClevenger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peggy Ozias-Akins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Genetics and Genomics, The University of Georgia, Tifton, Georgia 31793</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peggy%2BOzias-Akins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozias-Akins%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeggy%2BOzias-Akins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pozias@uga.edu" class="" data-icon-position="" data-hide-link-title="0">pozias@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1797&amp;rft.epage=1803&amp;rft.atitle=SWEEP%3A%20A%20Tool%20for%20Filtering%20High-Quality%20SNPs%20in%20Polyploid%20Crops&amp;rft.volume=5&amp;rft.issue=9&amp;rft.date=2015-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Clevenger%2C+Josh+P.&amp;rft.au=Ozias-Akins%2C+Peggy"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460022/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460022/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460022/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460022/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460022/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460022/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460022/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460022/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460022/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460022/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460022/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460022/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/9/1797" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/9/1797.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/9/1797.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/9/1797.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/9/1797.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
