<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3649" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3649.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3649.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3649.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/11/3649/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3649.atom" />
<meta name="HW.pisa" content="ggg;7/11/3649" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.)" />
<meta name="DC.Identifier" content="10.1534/g3.117.300121" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Giraud et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Identification of quantitative trait loci (QTL) involved in the variation of hybrid value is of key importance for cross-pollinated species such as maize ( Zea mays L.). In a companion paper, we illustrated a new QTL mapping population design involving a factorial mating between two multiparental segregating populations. Six biparental line populations were developed from four founder lines in the Dent and Flint heterotic groups. They were crossed to produce 951 hybrids and evaluated for silage performances. Previously, a linkage analysis (LA) model that assumes each founder line carries a different allele was used to detect QTL involved in General and Specific Combining Abilities (GCA and SCA, respectively) of hybrid value. This previously introduced model requires the estimation of numerous effects per locus, potentially affecting QTL detection power. Using the same design, we compared this “Founder alleles” model to two more parsimonious models, which assume that (i) identity in state at SNP alleles from the same heterotic group implies identity by descent (IBD) at linked QTL (“SNP within-group” model) or (ii) identity in state implies IBD, regardless of population origin of the alleles (“Hybrid genotype” model). This last model assumes biallelic QTL with equal effects in each group. It detected more QTL on average than the two other models but explained lower percentages of variance. The “SNP within-group” model appeared to be a good compromise between the two other models. These results confirm the divergence between the Dent and Flint groups. They also illustrate the need to adapt the QTL detection model to the complexity of the allelic variation, which depends on the trait, the QTL, and the divergence between the heterotic groups." />
<meta name="DC.Contributor" content="Héloïse Giraud" />
<meta name="DC.Contributor" content="Cyril Bauland" />
<meta name="DC.Contributor" content="Matthieu Falque" />
<meta name="DC.Contributor" content="Delphine Madur" />
<meta name="DC.Contributor" content="Valérie Combes" />
<meta name="DC.Contributor" content="Philippe Jamin" />
<meta name="DC.Contributor" content="Cécile Monteil" />
<meta name="DC.Contributor" content="Jacques Laborde" />
<meta name="DC.Contributor" content="Carine Palaffre" />
<meta name="DC.Contributor" content="Antoine Gaillard" />
<meta name="DC.Contributor" content="Philippe Blanchard" />
<meta name="DC.Contributor" content="Alain Charcosset" />
<meta name="DC.Contributor" content="Laurence Moreau" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Identification of quantitative trait loci (QTL) involved in the variation of hybrid value is of key importance for cross-pollinated species such as maize (&lt;i&gt;Zea mays&lt;/i&gt; L.). In a companion paper, we illustrated a new QTL mapping population design involving a factorial mating between two multiparental segregating populations. Six biparental line populations were developed from four founder lines in the Dent and Flint heterotic groups. They were crossed to produce 951 hybrids and evaluated for silage performances. Previously, a linkage analysis (LA) model that assumes each founder line carries a different allele was used to detect QTL involved in General and Specific Combining Abilities (GCA and SCA, respectively) of hybrid value. This previously introduced model requires the estimation of numerous effects per locus, potentially affecting QTL detection power. Using the same design, we compared this “Founder alleles” model to two more parsimonious models, which assume that (i) identity in state at SNP alleles from the same heterotic group implies identity by descent (IBD) at linked QTL (“SNP within-group” model) or (ii) identity in state implies IBD, regardless of population origin of the alleles (“Hybrid genotype” model). This last model assumes biallelic QTL with equal effects in each group. It detected more QTL on average than the two other models but explained lower percentages of variance. The “SNP within-group” model appeared to be a good compromise between the two other models. These results confirm the divergence between the Dent and Flint groups. They also illustrate the need to adapt the QTL detection model to the complexity of the allelic variation, which depends on the trait, the QTL, and the divergence between the heterotic groups.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3649" />
<meta name="citation_id" content="7/11/3649" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3649" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3649.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3649.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3649.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300121" />
<meta name="citation_pmid" content="28963164" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="3649" />
<meta name="citation_lastpage" content="3657" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Héloïse Giraud" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Cyril Bauland" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Matthieu Falque" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Delphine Madur" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Valérie Combes" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Philippe Jamin" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Cécile Monteil" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Jacques Laborde" />
<meta name="citation_author_institution" content="Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France" />
<meta name="citation_author" content="Carine Palaffre" />
<meta name="citation_author_institution" content="Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France" />
<meta name="citation_author" content="Antoine Gaillard" />
<meta name="citation_author_institution" content="Maïsadour Semences SA, F-40001 Mont de Marsan Cedex, France" />
<meta name="citation_author" content="Philippe Blanchard" />
<meta name="citation_author_institution" content="Euralis Semences, Domaine de Sandreau, F-31700 Mondonville, France" />
<meta name="citation_author" content="Alain Charcosset" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author" content="Laurence Moreau" />
<meta name="citation_author_institution" content="Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France" />
<meta name="citation_author_email" content="moreau@moulon.inra.fr" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Bardol;citation_author=M. Ventelon;citation_author=B. Mangin;citation_author=S. Jasson;citation_author=V. Loywick;citation_title=Combined linkage and linkage disequilibrium QTL mapping in multiple families of maize (Zea mays L.) line crosses highlights complementarities between models based on parental haplotype and single locus polymorphism.;citation_pages=2717-2736;citation_volume=126;citation_year=2013;citation_pmid=23975245;citation_doi=10.1007/s00122-013-2167-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Blanc;citation_author=A. Charcosset;citation_author=B. Mangin;citation_author=A. Gallais;citation_author=L. Moreau;citation_title=Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize.;citation_pages=206-224;citation_volume=113;citation_year=2006;citation_issue=2;citation_pmid=16791688;citation_doi=10.1007/s00122-006-0287-1" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_title=ASReml-R Reference Manual.;citation_author=D. Butler;citation_author=B. Cullis;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Crepieux;citation_author=C. Lebreton;citation_author=S. Servin;citation_author=G. Charmet;citation_title=Quantitative Trait Loci (QTL) Detection in Multicross Inbred Designs: Recovering QTL Identical-by-Descent Status Information From Marker Data;citation_pages=1737-1749;citation_volume=168;citation_year=2004;citation_issue=3;citation_pmid=15579720;citation_doi=10.1534/genetics.104.028993" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=X. Gao;citation_author=J. Starmer;citation_author=E. R. Martin;citation_title=A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms.;citation_pages=361-369;citation_volume=32;citation_year=2008;citation_issue=4;citation_pmid=18271029;citation_doi=10.1002/gepi.20310" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=V. Garin;citation_author=V. Wimmer;citation_author=S. Mezmouk;citation_author=M. Malosetti;citation_author=F. van Eeuwijk;citation_title=How do the type of QTL effect and the form of the residual term influence QTL detection in multi-parent populations? A case study in the maize EU-NAM population.;citation_pages=1753-1764;citation_volume=130;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Giraud;citation_author=C. Lehermeier;citation_author=E. Bauer;citation_author=M. Falque;citation_author=V. Segura;citation_title=Linkage disequilibrium with linkage analysis of multiline crosses reveals different multiallelic QTL for hybrid performance in the flint and dent heterotic groups of maize.;citation_volume=198;citation_year=2014;citation_pmid=25271305;citation_doi=10.1534/genetics.114.169367" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Giraud;citation_author=C. Bauland;citation_author=M. Falque;citation_author=D. Madur;citation_author=V. Combes;citation_title=Reciprocal genetics: identifying QTLs for general and specific combining abilities in hybrids between multiparental populations from two maize (Zea mays L.) heterotic groups.;citation_year=2017;citation_pmid=28971957;citation_doi=10.1534/genetics.117.300305" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Han;citation_author=H. F. Utz;citation_author=W. Liu;citation_author=T. A. Schrag;citation_author=M. Stange;citation_title=Choice of models for QTL mapping with multiple families and design of the training set for prediction of Fusarium resistance traits in maize.;citation_pages=431-444;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=G. Gorjanc;citation_author=R. K. Varshney;citation_author=C. Nettelblad;citation_title=Imputation of single nucleotide polymorphism genotypes in biparental, backcross, and topcross populations with a hidden Markov model.;citation_pages=1934-1946;citation_volume=55;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. L. Kump;citation_author=P. J. Bradbury;citation_author=R. J. Wisser;citation_author=E. S. Buckler;citation_author=A. R. Belcher;citation_title=Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.;citation_pages=163-168;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217757;citation_doi=10.1038/ng.747" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krämer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Leroux;citation_author=A. Rahmani;citation_author=S. Jasson;citation_author=M. Ventelon;citation_author=F. Louis;citation_title=Clusthaplo: a plug-in for MCQTL to enhance QTL detection using ancestral alleles in multi-cross design.;citation_pages=921-933;citation_volume=127;citation_year=2014;citation_pmid=24482114;citation_doi=10.1007/s00122-014-2267-1" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=J. Li;citation_author=A. Bus;citation_author=V. Spamer;citation_author=B. Stich;citation_title=Comparison of statistical models for nested association mapping in rapeseed (Brassica napus L.) through computer simulations.;citation_pages=26;citation_volume=16;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. Parisseaux;citation_author=R. Bernardo;citation_title=In silico mapping of quantitative trait loci in maize.;citation_pages=508-514;citation_volume=109;citation_year=2004;citation_issue=3;citation_pmid=15150690" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Rebaï;citation_author=P. Blanchard;citation_author=D. Perret;citation_author=P. Vincourt;citation_title=Mapping quantitative trait loci controlling silking date in a diallel cross among four lines of maize.;citation_pages=451-459;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050582" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. C. Reif;citation_author=F.-M. Gumpert;citation_author=S. Fischer;citation_author=A. E. Melchinger;citation_title=Impact of interpopulation divergence on additive and dominance variance in hybrid populations.;citation_volume=176;citation_year=2007;citation_pmid=17507673;citation_doi=10.1534/genetics.107.074146" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Agron.;citation_author=G. F. Sprague;citation_author=L. A. Tatum;citation_title=General vs. specific combining ability in single crosses of corn.;citation_pages=923-932;citation_volume=34;citation_year=1942;citation_doi=10.2134/agronj1942.00021962003400100008x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_author=P. J. Brown;citation_author=H. Hung;citation_author=Q. Sun;citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population.;citation_pages=159-162;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217756;citation_doi=10.1038/ng.746" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Tisné;citation_author=M. Denis;citation_author=D. Cros;citation_author=V. Pomiès;citation_author=V. Riou;citation_title=Mixed model approach for IBD-based QTL mapping in a complex oil palm pedigree.;citation_pages=798;citation_volume=16;citation_year=2015;citation_pmid=26472667;citation_doi=10.1186/s12864-015-1985-3" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. A. van Eeuwijk;citation_author=M. Boer;citation_author=L. R. Totir;citation_author=M. Bink;citation_author=D. Wright;citation_title=Mixed model approaches for the identification of QTLs within a maize hybrid breeding program.;citation_pages=429-440;citation_volume=120;citation_year=2010;citation_issue=2;citation_pmid=19921142;citation_doi=10.1007/s00122-009-1205-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Wei;citation_author=S. Z. Xu;citation_title=A Random-Model Approach to QTL Mapping in Multiparent Advanced Generation Intercross (MAGIC) Populations;citation_volume=202;citation_year=2016;citation_pmid=26715662;citation_doi=10.1534/genetics.115.179945" />
<meta name="citation_reference" content="citation_journal_abbrev=Biom J;citation_author=E. Williams;citation_author=H.-P. Piepho;citation_author=D. Whitaker;citation_title=Augmented p-rep designs.;citation_pages=19-27;citation_volume=53;citation_year=2011;citation_issue=1;citation_pmid=21259306;citation_doi=10.1002/bimj.201000102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Mapping quantitative trait loci by controlling polygenic background effects.;citation_volume=195;citation_year=2013;citation_pmid=24077303;citation_doi=10.1534/genetics.113.157032" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic design and statistical power of nested association mapping in maize.;citation_volume=178;citation_year=2008;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="Identification of quantitative trait loci (QTL) involved in the variation of hybrid value is of key importance for cross-pollinated species such as maize ( Zea mays L.). In a companion paper, we illustrated a new QTL mapping population design involving a factorial mating between two multiparental segregating populations. Six biparental line populations were developed from four founder lines in the Dent and Flint heterotic groups. They were crossed to produce 951 hybrids and evaluated for silage performances. Previously, a linkage analysis (LA) model that assumes each founder line carries a different allele was used to detect QTL involved in General and Specific Combining Abilities (GCA and SCA, respectively) of hybrid value. This previously introduced model requires the estimation of numerous effects per locus, potentially affecting QTL detection power. Using the same design, we compared this “Founder alleles” model to two more parsimonious models, which assume that (i) identity in state at SNP alleles from the same heterotic group implies identity by descent (IBD) at linked QTL (“SNP within-group” model) or (ii) identity in state implies IBD, regardless of population origin of the alleles (“Hybrid genotype” model). This last model assumes biallelic QTL with equal effects in each group. It detected more QTL on average than the two other models but explained lower percentages of variance. The “SNP within-group” model appeared to be a good compromise between the two other models. These results confirm the divergence between the Dent and Flint groups. They also illustrate the need to adapt the QTL detection model to the complexity of the allelic variation, which depends on the trait, the QTL, and the divergence between the heterotic groups." />
<meta name="og-title" property="og:title" content="Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3649" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Identification of quantitative trait loci (QTL) involved in the variation of hybrid value is of key importance for cross-pollinated species such as maize ( Zea mays L.). In a companion paper, we illustrated a new QTL mapping population design involving a factorial mating between two multiparental segregating populations. Six biparental line populations were developed from four founder lines in the Dent and Flint heterotic groups. They were crossed to produce 951 hybrids and evaluated for silage performances. Previously, a linkage analysis (LA) model that assumes each founder line carries a different allele was used to detect QTL involved in General and Specific Combining Abilities (GCA and SCA, respectively) of hybrid value. This previously introduced model requires the estimation of numerous effects per locus, potentially affecting QTL detection power. Using the same design, we compared this “Founder alleles” model to two more parsimonious models, which assume that (i) identity in state at SNP alleles from the same heterotic group implies identity by descent (IBD) at linked QTL (“SNP within-group” model) or (ii) identity in state implies IBD, regardless of population origin of the alleles (“Hybrid genotype” model). This last model assumes biallelic QTL with equal effects in each group. It detected more QTL on average than the two other models but explained lower percentages of variance. The “SNP within-group” model appeared to be a good compromise between the two other models. These results confirm the divergence between the Dent and Flint groups. They also illustrate the need to adapt the QTL detection model to the complexity of the allelic variation, which depends on the trait, the QTL, and the divergence between the heterotic groups." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uMWYo-Q3lJGxk4ozg5V-GXfD5N44KThgmpM_7_M-DI4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487104","apath":"\/ggg\/7\/11\/3649.atom","pisa":"ggg;7\/11\/3649","ac":{"\/ggg\/7\/11\/3649.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3649.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3649"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3649"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3649.full":true,"\/content\/7\/11\/3649":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022e5G9ONlNx43Ik5otUaanQ99pM0\\\\\\\/GM8cJiPs1yCATN\\\\\\\/zuf\\\\\\\/J57NWATcKbBuDV7Pur4Fe9rs2PD8mZQR0p9+kor57viy0oyNqAsHLNohoie33H59Tj2rZ2HTlOq3B9xNq0YU4YT6wMeUxtFGo34mICon4LyJWQIARZGBuALE6\\\\\\\/aGK\\\\\\\/op\\\\\\\/E3mmLiuyCb5kAATR4vHXaEcATBI7ly5aBtOiCnTO7UHSA+I6m9aKQhwCx9Va7CoQNwP1gMhCGDVDL3nbiFSfjTgntkSN7IwklzL4Kv7XAnlM5rUdThEIlXXHmDCwQUyFBx+QQd8DLoUpydDXWwypexBuSvYTLK5Lg2CWcuI+KBETlmoUJeeMZf1CyCKfcWsrzyB6ke7vtcdzGTQyoV6hBgQwXrSpBE7hUQ4tVHE7p+Ukt7cDkxyGKiJAG1tjAhRhwPc3Qr+e2hs1uDbhG+XYLCXeRl4yrxknk13dpVkHYRB8nWA5H6yfpMKhk7YVlhFOcJbXHJWYWZ7AA0KyPySec+9XvkPPR+SxdYIPFHhBkfYynZN9ee45jghhqNwVzfyR9CucK048sCj6oP9vcwx7YRR+8LofTBgmUbytl5z2ke9cB5lUlD98gzZSndmHNm7SjArHx1qaedovXKiR414oNVCykMw0zJB9wX8p7G1QUN8Fgh1C\\\\\\\/nqpIHFYy1CIBWPjRJB5mmUxsfAyYu1\\\\\\\/WIJMMoUiOZag\\\\\\\/hRhi7hgSoJndj502RhXfhJwqfE9NyIR9cYtAyogBG4hSsWpoSR\\\\\\\/5HquH8crN2yAG5yr8jZtYwBKvGqSoL9ieWoOim4E3vRhnCRnbrk9GCdziy5mMBdmnCbPT7eD\\\\\\\/DjR3BJdYgtyz2O21i6rKAEdYcC3g8iJm18BHGh7JpByvB2CIplVVSSWhSvXKV69QXFesL6\\\\\\\/A3QAmLBIKV5nXsgEdYAaIT+50YVNHn2aar7nS28NZfs3Ihx4X8tVVD6LyA\\\\\\\/p6cASLfgOOJiuF\\\\\\\/EKXM\\\\\\\/V3vniU9BYpTUmUJWuoeYLbvjsCZVaKgaR8xVbJp5sOi8gmg8jbibgpm8Q64gY88\\\\\\\/Z7hoZGmAneiSr+br2edgjPhAp4HS2UFjZHz0Q+KqXL0g6nws1ZmoVgXpIvFQp25PTESf778aqaQmDYH0JuIp2SnuFaMG89NA7pcC3O97ulqchtTbRGWf8Yo1VgUrl3rf4nlu21m6wyczWtYOhwNCgkMHpH1toDIBG+KgtQYkrZoUvsy+jmyM6tfHWcsiLHafgrzi4\\\\\\\/omk+ndohJbDAZD4zxFuwqpKLDBLV\\\u0022,\\\u0022iv\\\u0022:\\\u00225DzKYl3zUFl4DAY3i03L\\\\\\\/Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e62aa6479714a99712300537b9df453670289d094aed80eac3195a15c366074\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gQmnlCYwWxJ4bFWdXJa0FEGXDXmgbOpbUmkH0fIjuuz+cEg8uN3ex3RUwVeEgTqxX419WkL4SbALfE0Bq\\\\\\\/8h7TkkCqmlPQtbTtXsmeOBriTJ+71eRt8Do3nK6mDveufFoPH7Z8vv08S768mZ7ZQmZ0z4Dgj4759OpCMot5tPcJcUa\\\\\\\/O7qqSyBhGAc+ykWm183AbFRiurOf36Sno+c9ogQ2PdEXoroM\\\\\\\/GUKQzjxb43qOszc1kYzPEhSXxjmMUQgi6mNiTkQ9nmkSweu+C4ZY4Jx\\\\\\\/M9t4FDawrVPL17\\\\\\\/rV1Ex4yBlEr5Dj24nZFFZLRFGv3o5aymoDa8ltEAocUhIYyrBzqvtZcSWTOi+\\\\\\\/EVZ9hwaZYnRvTVK9vjwsCc6oaP3lNh4T1ZIkZxEzo3aKIJhI2Lqc2JpEY4H7mnz1s73FCAPTvPtm54HluMlz3oZATZKb37Kz4Hj9tu+ZUG+CCSalHMKAGCek2Cnkte1KARLL0284NqZCvwZDSgHYToFjuXv7ayEvgJ7khjPhsZePmGnyzpWJLPEUrxac2ZcKyfJLQO3eBQwJjkvfa5BsLZT+eVoQlZimUZdQ9aj+QrSnOq7raFk2KUq8ETMDFCLuZ5HetGNvsg\\\\\\\/HQSxNAK0fyHB+JvBbGujlMBzL7eth7Z2e0rq48\\\\\\\/7jlguBrCvGJ\\\\\\\/nON42+0DRiTaStOAkb+r+8M3WYkEZnCIIGK0LtoVbHcvNQObH2eQLxCiCwWUObUGnVUDTNP0k1GN+970GSCdIjEiXbPpMJkH0jsjmvBmwxGQu\\\\\\\/Vh8ycyeC92ZJNWkNrvvdn3uNSqNxlfGAF4QSb2ZUK3hr5yO45X6E+AkoT+sS6zcMxBB2WvEWytt+0qPRCPOWS0oeowuOrNgtAXJl3SaNLKyH1kuTwSVmwmFLj3J3hArtdt\\\\\\\/LoLhxIwMspVHGKmzEvpNyBePcwXth4pRyAgvHuFFt8C9RZ6SfDmZx5NZKynr7dRjd0kbsMrAjj4LofSqFs\\\\\\\/URpqGbvvlOQf7Ba9yL2cbIDIMBH0P5+2Cxapj0DM1FEGR9VrI3uOk3y7TnX78Z9oVBYQyUspNssSO69Uy6aXlnfVkbmVSTn9duwWZV6GyRx+mHo91yTOXNmYJHW3\\\\\\\/649ZHjkmE6sefX9HmMkROYeHArut3MbrJjY2683ooJS1GIFD3mgdkpOMsCUgPuBCblRSfH+RL+LLLcve5amRku1sSWkJ92NstiRFFmqnjG2yUOPGOIJ+OG4GAePa60SVSRji1TQGMirqLJ6Rk7UMu61aUXRUaB7glEKY0mMYqkzjBjIyydiFeblsIdhslE10H6Kj0fjX8bIhgygxIqcRCWmq42mCB02NGCQUUggaqlYrcAIFdOXu+0LQ7tvP1B3nyGHjwOVYOdP6lhFf1SuY5Nv\\\\\\\/sJx+nBCD3hfSbZq+c\\\\\\\/sfVYjPQrrqo5Y20eBBZwQyri06gU0na4WT\\\\\\\/2TwwQ5BsFdP1PSNEdOz0FqaEi3HKqb4RSXcaY59qSun2ouYCJ81ppKklSIePDflMkvDUcuEXYcJqsnoppOkeVez+VF8paolHNw9yBn0s\\\\\\\/xVShXRrIlIitDQTLKEP\\\\\\\/VCdA4v+YH01uTvSihjHcIuCNTMdyqDRWQ9+AsYjriA2VmXjvOF1AFeEkpIR3Kt+hxswLsniO79isv9LOQGqaBzmLjN+EYkSIOUIcTVlaFM9b54eeDpLB0WQH9tPpTgViqrc7Fn38+mUoHIpE79lEqVz7LBNrUmNnn8VzBZombIg+1uvRAU\\\\\\\/gT2NGxpbsc5Tx8mB8o76ReJvBZ4lQerG7TN1yh0syJUht3RV+XblwKIidfweJBrJ4nxudfLZSmNEEfrwprSjCRoJ0O2aknmasHE9ZL7fq4ib+w62y2GTat33OVwgHMNxi4HXx1kRuZt6oIMksWOEFgXni02cEuavJdPa9jlpMQR33+4q+QVCtq8N6wARPHsRQpVBOYwnMxXDcSiO5+E3TVRUc+jxn72Q8xX7S9pmleDQu192E9ezXTbE2NnzGbrUpaGofbCZQMOum5QmSI16pDwD1+hRBMW5nJyHxla8hTZlECZpcU6ZxG6zMsOs\\\\\\\/Eg3ouoqGPxSe3hBOFBT7jEaKWIJ098Af2TVJPQeOdcIr94g6xt7AiKdNCiyTXLeZa47lVY8paKWAx\\\\\\\/0csaK+cGCecWpklucl32dGohMq8l\\\\\\\/3MHZKeK8RigDRs2FwC9pKYv3QcC2gqR7DrtROEU9jWgPxPu2IW5dC1ZvwNfYkoGEXhW86LRPcL8eFO68osS08nnQgD2W3COg3CRrurXiILvNui\\\\\\\/cneNcnLtVmS9aEgaTQNqxRz7S4vfCv8c+J3GtGKC\\\\\\\/64DuOvPGvs2lzW1\\\\\\\/RC6xWlJDfL9q6Z3Wx9bLH8OmFvzufPeEcGCSphHi6Vb29W0oxmBirLYbBKV8HbegZerNsobJYyrrDlWkYygrqG68H5AzgGO0dS1jN1oQjKDWj0F3idRbZd84zPjtZqMlFnmzrCceHTWl8dE9IiKkGwQu6NMzRB66iN35vNgQ3p0bc+NQnmJ7OaBGIAJJPEVif+kZOjEZipIJocRBkkdHEOkixZ5YnzAgdEj4X3elWKOWR51suhrwAywHxQlnTiDuYc4FAdIBYoG8zQVyBEPyUdXmo2G8ysos30dhEaZS6XyXqiYcKIVaO\\\\\\\/iIXGfVvK8Hu771H5Xv4F8uzDofkP60AQm\\\\\\\/cOtaeI7n8Jrr7Pop9cvb3RENY25x+StP2aue5Xn\\\\\\\/6VYt8VaKAUgDwVcT8i4DAL5B76nTw590SONpUeJ3m1vBs6tYEM72tGYpBlVmC\\\\\\\/plB3A0WoZawYA0MRbhKc+15gv5AF3rbtF4mQdbS0q836r8YJKaIfn9AziuJeXLK0cVWxAroJx5LgLCFH7B\\\\\\\/9g5NwLtiVTYw2l\\\\\\\/cW7\\\\\\\/p+Q2LWKkaJR3PZTAMH\\\\\\\/JwH6vPyvI\\\\\\\/pPva05PF28QyvoVRuXNME9Uthqa1C2sLZdtIpZTRksb5ew6F6ZbkolcQCJur4mUoLFTHEC7sp4RhqSstIVqDW2FBQcqcFWzmXJqfpQjQ7+LNIQktSMUWAoKeCZw6MIpC4wSpFjMSbnM\\\\\\\/qPRyBX8C9UCG9A2w8ZxMqAwWBtxIEBV28lgKss3WAFqeNgO1rGI0tJB54Wxw+mosdBsKxIPnJeSpY+n6V1HyfqBjtaXLbRz5y5nA7QS4xLPYIyZvUTJ3qNmiVFMI8d6UC\\\\\\\/EqaaxmKA4jy2f4RMcIfJK5vxLlqdcwXVS0m9EncvKA+srL5n5hhhS7iypuPM41wpfPsBBbOe\\\\\\\/LSw+pA1\\\\\\\/JmV5jma4bf\\\\\\\/ejXBr0Yyq45RnocTaQd9oyiRHJWkuqyN\\\\\\\/8NaPkaAMCVeUBZHADUEEPDEhm4pNnm33yGfiozrOElvrg\\\\\\\/7QX\\\\\\\/x1ZzBSrIZrwePVCeFGqRhMJRH9ChXXeHKAIIov0uBTVnRQUSdOPdi4geyU2Whd0uU7eG9PXg83qufx1MZXWa31OvT9GDhSMzr0LP9sD+xluzvMw+yzuHwIvLRECRv7PMP4dD0HOcG5oFD7k5s44q\\\\\\\/kLdzYhu4uhShfHd1X8kOlxL8xAw2XPf19mJca0dVA0K98rC5XCMJqLCQxcJs7TOu0tSjlooE7JlQT6F3bbVwD02idQL\\\\\\\/SP95wj4Y4j\\\\\\\/nPqKXBFOgkfaFy7IIA0rKuXfAV5sCV\\\\\\\/I8UNywxOx+iDnZT9NPeyPGmnzJSVaUgXdj1wRcx1CPQ1Ds5WNfglPGMF7XxoEWG\\\\\\\/CRNBP2YzMQdTrnFLyaRV+64sPwO+bePtrz\\\\\\\/m1ipIeO1CFIVqdHenmFjS\\\\\\\/QNxv0LZT5YMGQA8kfXMj2C\\\\\\\/rQD95+hRDKwVQP1gTvcog+kioMj6sRZobQ+OUjDGuxmPwH\\\\\\\/\\\\\\\/0BJbw18S5NF+efP+Sv8DAMxJOGwiKetgDHExxVsgGVw4IZQtX2DrMROIySRA8ZuKiKTVITf9NOYjYSoKEqnlUeYP7emrOQnpXF9mh1yjmVRkg29X1J2gyyW36FzYgMKbQOUhoJjmDoZp1sAN7\\\\\\\/f+zRd3mSwSwnRBREZ+5htPN8O0VAq5wHPW596BdP7oajiKFKUK\\\\\\\/v33s6MldN1Yu9S7f+afylarU8t\\\\\\\/5BvcBq7rBa+EayiMFt2ZmoVxtQmo7+VxJgGcwskvWhSN8M3ssOFDnIeJabuaing52bo65B4KgC1nIPo4mIJ33RCLyH+DxWt6Ai4CtHF+n37INVJBCtoZRBE0zo0VMzlJ6I1cm9ZACSFbRr6DLlVq6KkqVgrUJj9VG6ZkODYiGxQzJUNZwugGavL3oLo\\\\\\\/AbqDEtu6KFiCnvhyp9\\\\\\\/V0tBgsPPKMeDWcQitoTGbjalGCZKmYJzrnG2AyQEGDIEVlfmSkvOOtp9szNyypoyM7JwLmNp\\\\\\\/eWyG58mizkb6hUZ25v3p3PpvV+9mzF\\\\\\\/buRiUnZxlAiTA3FUiFKJyMBNY7wbBFb5GKngOouariQgH4ottYwMn4A2fzPSSDFH0S4nseYQ017lTQeoFEYAGafjIWI9ybz4X13sMCLhg64SEAFwpgGZfODD3XaU2x0\\\\\\\/Ijlb+rDpTPdFivTeMJy2k6reehps1xlehKXfRWoAEGfX16Nuvy3tNYvyuPb4gMD+ij4wBzrc5LGmKJGWu2o8TBqZK0OVSMNWEl76LByjAkYlWndqsamdH\\\\\\\/4DNfTlWhlkVGjRu0Ru+Dnt5R9RJdvm1g8T706urkWJ+a9wK60DIRUiJnHTG8rMAcw3KnQDCRWQOAyywpQLgH3nGxRur\\\\\\\/WZwJVv1Cj\\\\\\\/mkItCxV1lCjn0ymxMXiTLjpr9AD0mPL1AAZ8fLk\\\\\\\/fjR7SIspA8jLpbP+tjP0yZFGMgES6V+NN2rrgvo8yrYdjPvzh1V8p\\\\\\\/z69wMDYjy1EBHHZcZJGq5KDuzi1Dorwukr8CIbfrWocnLEMLWbC1e3kfh1cc\\\\\\\/yL7NR69+OXTnNtnL3UVleJGByOmNGKeRRfevxGs9xy8X63Ki7EwFp0n\\\\\\\/SdfPvgIFsj61D8lfRfd7zKGnx70u2H9nVMSX8fghGz\\\\\\\/85SS00oKrZXy17FPpEp7figMVCJ91OwuivfDqZC6xL+IdiIL9LjSjjaZWHTOntkf2cPqg7TLNlry5dl2ndmIN7cWFzuvfdZfMly1vjHzvUgp3UVXeSoIzVLQ9B0ViFClGju4QT1TcQk4ZdQbZmrVXC7pC7LmpXWFGA51t0h2Krh2\\\\\\\/9nMKiLgcuXMMv41S98VWns\\\\\\\/alWFrFmPkDgqy8YdelusplUWmFVsOW8g0Ze5T8yIUQq8AghbFvjgM9wYdtp4bg\\\\\\\/fRoK\\\\\\\/crE2ym11eE17xxluR6XhCfy29Dhw5EUFVOYpT2VYTf2yiaZ5IjOLkhhr3dkbofpl1sr5sZP74335qYfVfe1vkjWwyMMadKWDVG9aZWy3ltsLDIAINZIYouYtyp2jsHj96XaX\\\\\\\/TPE7ZNW3DOoDVQh5NnqOJdzlD65q2fAYJZr+448EuuZ3moYxpnkH8Tq897OOyVPz\\\\\\\/BONKGmCQWNxnqP2FDteY469SJrxqNx\\\\\\\/ZminOCkIhDW3x0J+UZeYKZE6ky8ILVbAtNvoATS6+p+7HP12szKUqj\\\\\\\/29vx+waoSmBP6RIWGQ66Xf6pb+9DxLV8vJQBNMMMyTjwU9No8bkQICKcNohlQevpbMpdIQlthiLIqUhf\\\\\\\/yrqs0hNbZgClUYp1d+cn7WS7gvfuGawi4HYlH7BCZJ\\\\\\\/WStyACGUNptsBmDrYV0z5r++U10zXJYkqd5bntHMIbStZHIDP0nqZ7OkkAgC4k4aFvFlnWuDfUHuzIWW\\\\\\\/phpzBUch4wBSWO7DsiW\\\\\\\/HhWpG4uZ6w8gBPpZEDg0LnEkDkwD3rvHJ0Bio45irExrp9ORrJWjw\\\\\\\/4j6899K0rH0FGZhK++EGCxobkiiebzifkJwIg+dfeatexHiU2Q0\\\\\\\/hmj4pEvdPMRiTaSl1p90Lzr3+z3uZlmax3etMr26WDzR5+eHt9IWUbC7vkQKsP3tExFoznUXULudBN9Iukng4AXD0D6IqboWAbfoKskkKhhmh8bfDKdDRfPikRP0u2esRCAOMPo1o9y7nIeMb3FDUuQgw\\\\\\\/\\\\\\\/TO5JqNhBkMPQ+T9UoS0myduhn+jbgj3Tk1bMt21kbODCk5TiDBc\\\\\\\/xDQ5xyrPX4zKdBq3yhIH7Uw7y7Pl9zm62fvzu9U+tKfu83MMJXTsbXrdP9JSjB3jff4Vjsd9CTJ2+\\\\\\\/R\\\\\\\/IyePajr3cdQT6dK+ZSOm3S1Sa0f3lIl6H3pe1vEnuM8NzsDHkmceDbwqJt8flo9R4HLYhgpudu8S8OtiS8Meip9d0X2iDFz\\\\\\\/CSapz5VwY+rCIOPyb82pBb5Gt7qBquCoKV\\\\\\\/R4zUUsjIoRvVTbMwdFcQZ1psrO6o3czA2unjJd75qynSPZ0eNjn51aKri907JzrefIeslVT3L1OR9wmT0m47Y2Lsf+97S681CuCcuFQGYzVvoJgyFKQUrW3dyI2aT\\\\\\\/QPD92toYWetku6ufBuUVr3YyArNRG6Ephu09I1bUkQBfliI\\\\\\\/SoOJ6LjPTVnPrzNiXpyKGcPYLkffd47nwjCDuzz+PtF1aEEQro8YpbYAJbXl0Oz4ueeBok7Ct2oJRYqmIZVSXomu074ukufxV\\\\\\\/FaY8Dx5dHVo0HnWBLVmGmYOnM+yQKuv5vGVSmyhmDnoGzh9GaSzM0JW\\\\\\\/slfwLpOVIndLiFv\\\\\\\/pXECW5DZlgWhewO8YVrBeqzyfZ\\\\\\\/iQdfu8kk3G045dHdt0D4qnwqvLnY+Rx8mVLwd\\\\\\\/D9LvtDrAb1SlkD9AAbTQr6Q6TxRfFLoi+1UD3ZZYqDVjhGYL3F4IPGAbt\\\\\\\/4k2xjqb+x45AAHjp3loJcgqZmw1eip7RmQlGMgvD08vk+TbV24QMAi6oLXQmSHoRzJyrvgF80jr+Irkhcs\\\\\\\/GNhXiC0jmylbZ+KJ0hOgpRXPKMKRCE6ixZYkfLvuAIIx87\\\\\\\/mWHNuJaDRAmVnYoLHMmKUlCpWcxR8hO+vam8Q+pfSWH82PfbqJOUdAxqisNAnkTUansMEMbQxRgqTa77TqQ5geH6KxIRBz5gcPrFxrHW1lEz4cmUX+X9mLF\\\\\\\/G8HnWICgnjq2iAQq6o0yYYZOC9CKaPNmIjLtBr5He\\\\\\\/Nhe96A9xrq1VctMF95G\\\\\\\/2O0uLTT55V3WxjT48hLKrlSNWSOlW5Qu\\\\\\\/3ml5OqYOVF678\\\\\\\/HigoZ61SxcbRUCpwETfHz7pOXa+O\\\\\\\/m5tAeqSGB5e5mIUC+V2nfbrBk9qdpAjvQ7dlKkbXLyIEqoiufPQrUGKjM\\\\\\\/oc42cv\\\\\\\/bDek\\\\\\\/3yWfW8t\\\\\\\/WbW+KBilzh\\\\\\\/vICB6P1XbsRDMeYPke3PeqKjR7XyNUL1\\\\\\\/jh7jhox4N4r6RO\\\\\\\/SfiQ0YZxdBmRgHGoYmEzLsI8\\\\\\\/oY8cFiwEzK6N+yh791zDQi+NyWLinEY7k5zBwD+XOqZ6\\\\\\\/wK9DhE8uqV0HqqccUpEmgsT0ZtM00lo1If7tlfQA\\\\\\\/JKeyY7uhc0J6VXVUbVyNhw3iFP3YJyWcGamDSXLmG2NR9mA07LNEfZ4cx\\\\\\\/zcenidNQlea61DxdNw\\\\\\\/9w4F86yDwrru\\\\\\\/qF1Ic7ZeED\\\\\\\/0mPtoL7EliwvUq501HYg9rt4J9ICElJ\\\\\\\/vlEqXE3s87VJHCdWF\\\\\\\/gLWf+v+md20s9NFVV6mMo\\\\\\\/\\\\\\\/gF5Wcv1DRzH1j\\\\\\\/vA7p1DuSwkGYE9tb3lGOzgQYxdnxmFHq8zFJZi\\\\\\\/JY7Tx9qK71dG1NYkXXaCreXfxVEAXMdKOFb5Y7JxXkAKxfA+EsGSiE1vRyoA6GnnvW9d\\\\\\\/LLRj6SLxcL9fGYMmtntev618iem7fvfH8jkc70y0FTseNhqVBy7KvMCiwy+EWpAnwfBLIiJIJpYvceaJGlkX+vfXDObuqMUtB3kYx2wGMXHdjembQPowq2g6pADw8H0XdK6A1HGq7iyorZ+819W+Ulphv4Bbbj8f3a6FkDjSi0F\\\\\\\/\\\\\\\/YGrDE6gRwhq++OzayFnFyVWTEwK9LF39geE11Z0wjFSE8C9c\\\\\\\/+rnGOnDDOoWYSvw4iRAMMBqPROFgu9riS8mZSWTpJvHrm36hxkTa6YZOICvtkKei8V2\\\\\\\/g3YNs0psDpSLblLrQ9+IcGUMVJnsR35g16lbv\\\\\\\/MAl4x721VwPYFQ+1VOIDisZkg4Uhbos2cdlxZZ82bDpYKqU6B6rqYytb1s\\\\\\\/ileallZjUaohiVeF2k\\\\\\\/84o6OSTnMWAhi9W9oeZ3eZpUy2zokyTMRrHHe7UdQplK\\\\\\\/MYytpImQKd03bfyLqUumaAz0UKG2VScG0hG81z7wc1wujmhNkGmf8Sx3NTVD2a3i2B0xyJVnoSkzcKfooCQenHgicVP7WFSpiXxa0j0XxHjN5HSyRaP6ahC8oVc\\\\\\\/tvbZyEn6a1CIjydJRrHWkP5ZHE0zTd8o50PdIxv6H5vqNEc5b+CLD4XtpLt\\\\\\\/i+jYVpt47tMzn250Akjab1xfMgoPQFljKwyYFlEC8ZR82PTvI\\\\\\\/C7n4Al\\\\\\\/XR5IjGYwRjtmPBpjoxNC6rQSJhWuYfhHPxG4PhTslkmzTql+RnEJYWjDwA\\\\\\\/E1SCqkRhEdpcUGpRK4\\\\\\\/CsCyM8eMtqKtLl7x08PeV7eJLZ6Crw58VO6tpEfYMdL8zdNgOjzYSCMnBnmYm96YHpeRJ5MBvXVFYuZ48q7wPK2P6uDMhACwqDiuBA+NOlLVcwIUKZDf1wNrnynL4wL2DVzTvgKP3N86C8GvMb9IumkSO\\\\\\\/T5q8zW3iMp4PonmYhKBfv+HgYZsxUf82HgHUX9X7On64jsSN5pqvUCfyFuHfxHNE8Dgg3SUGgjiiFFXplZ1hU2j\\\\\\\/oVn1ULczl8EnqvRfjB+glGZICjFBJcR69BugYGHkSkpxn08jvLVkhK+aPucF4UyWqOrzPOFUxgQqiFPbcpj6u5lYjUC\\\\\\\/QU4I9CK\\\\\\\/7MOMltxZqxkCtGRTyuAheOF0Umliesd85aAn2+5UY8IaVRoTSiet+U\\\\\\\/ZIjZa43sgpVgh8gMHZdafN2fWmJsO0cgQVr4RV84AcnfSxAmRACTFhtcIgLMg\\\\\\\/a+1JmIrL4MzMVGQC6UtkqQbKH0sCDRjS74xUNgxgaayt+R+4\\\\\\\/aGZloC4jt9zCjED+0BwsG\\\\\\\/lBwwjeQVkoC7SLnxfFY34M4T5p2pT5+UFUdF2IQnr6vHScuyXJpt7gYiHXGZq2h8wdYJ8ULayP7gBTmwlOrWKa+AtyTMZYR6DKOavtdZsfw1fPO1cKTbqWMDk1rtE5TIJ1WWNmgu2pZfZ4PdYwuzBxJig33ZiUx3RXWVfWoOOe40rkgRn+Xu1YgAOytpoiPRFfpu7dXAqgl9g\\\\\\\/faUFUqKK17GbjOQlOzPmy7xpszJFPhPiNXiBg1agzP\\\\\\\/a5TovYPc1tteb2tPyi95IU2rEd+AovgvdbG0Ljt2LoGX5lbliuPK2MzC+NiWzIDq8sKORJmvnSJGCPqrMpSathKJ2kYml1FHDvHpFwLF74vE8pJXihDD1CjJOp7OxrLauN8IJH\\\\\\\/z1pugbO86\\\\\\\/r3a1sakNYsqnuTggJKBuFTD3NECohokY69uHxamxNDljUpzM5MhyVYD2VKFIue28075Rk7kvk2\\\\\\\/z15YTC9i1XdU4Q1CO9VbQLMQ1OKwjzLFOkJVSCLfDXccT3I2Z66nRZKPlAqho5tmYAcJTrJTRLqUxvHgqfyHY\\\\\\\/ZjKjl5+\\\\\\\/4W8oB+5OMVfPk8HY\\\\\\\/fP\\\\\\\/lztMx3As3bqLjd6MFDOAoTgv5Yi7kWagUUqQip3htyT6uQogOVfp0wgSI1xBkyOfLfsPJf0uuWwiKEQ89k98IZXtalWGIAvQ8HZjvzzh8fCbmhWuzVwO\\\\\\\/yE04nnlMjaiSn6XJnK8nBZcYOxPk09KmKdRdKBPXv72cuhhAKim2\\\\\\\/cmDj1wg4J24ZwmBfHG\\\\\\\/scqr0zOpZtlcheS9CqW4ByNV++S\\\\\\\/UmRQKlXkmzGoAbVpmqdUQlL9Fst3KPyILtXaEc+g+oDHd\\\\\\\/CQw88DOKsmPQbGOdUIak50qfF7iD+9CLZvTyeJcmTNYCMpQ5acMEPLSN070ME0V2bx2sYzVDLLJwCExzYd705WEdSP5uXYZpg44q0R6Sdf3znOe74OmdyaUvQa4FIaNmjQ0hx4XQ9efTgwMTbf8G8zof1iPiUT4DODbzH3skM4sb4hdNq2HFtkH97mBN1FCeRcxG5ys5c9xmw1gKHCA1FwljwBr81oydj2HBBec6U6+mZ1c6tIMzegaP5ikfTyTq19fPrlpX8FqndcI88Td3MwCzjaukZtDjXNWrqcDyGjJgu7+l56aZhXnNqypXmyNKwp3pVoKGdrZLRCjKg1SFneRihbJq7unxXeFEbZouGl7aTubWwIRi\\\\\\\/5Zto4hcUcew9BG76FV4oRNOPttWxkYj2\\\\\\\/6q7s7qqWK2m3WNFYqQCIrx6Hk\\\\\\\/k0U\\\\\\\/OuicXDv\\\\\\\/CQa0yI4pE3cARDRw\\\\\\\/cfJ6F0HLXpmFHTIc+it1UVQcOmjBAYV5xQZ\\\\\\\/tSrGZNxwy2MjHhPHr4ekZBaAxMafvBEsx87N04\\\\\\\/V\\\\\\\/AiNgdOcUL8XazqL9uBRcw+gdhbTeQekIxWa1SZuTDtn1+o4CLChw7NDIpKbDOI9F+63LHtHbEEQV4ELcIgBRnKiWQSNRkl4ES6s1SfkTJqS66sgw+DmzPdFmvk3FWlvTXN2OX3BoZIDveQ2N2APajdzm2UlFBIgpI4EoqnAW9RJw9UaKql0D\\\\\\\/\\\\\\\/+j+lFEsshQVWFwVaBSP7X0SHDI61seCrSc9Ed2BeprpKtL9ZBd8wu38D1gBgS5i7msxwAJ115H7IQjgMSh9eS1rOnYMX8CA88K7y7mogV89C6488o5zxTPfY6GGP3m7C+LfuEuFbQd2QH6eSnFWCjPW63wy8SOCSyNE5KX\\\\\\\/\\\\\\\/Yy4IgeypuhOdtg13Dlpx40aWQK7VaPwATyXpZbt3Yx8rQA+8mNZ4R989RLOfYGbwN9owZsjgDtq6\\\\\\\/kJRPes7uUl1vw0iE1zxFPeC5V9y4kEXUJCFWh+hN+QmhACKWMCOQGAtwCwSxssfG\\\\\\\/lZkMzvYbFWzYkvCjzIoxw1I0ORIf6KjUS2o4XPaM9BuIROHC9H842pzy6XBm2gbzwHyx9M4Dujsmt4qPKUfCV1lv3Jx3zXwNTlT6SenDNrwrOEIPe+FZw6mugUbWx1IBcwFDyt2PXfoZGNgXoLArcbFcpRUEFDaF6VvkXlAa7Y2xQhCWQzdZTqHeiFbW6aUOZ9TflPMZKaUhq0eP8DdbKw6b7fUQfinCF71tqAqBwb5+X8+WheiFXpapkCLXZasFRNGtSmgn8334ILCbK7D4NOFW\\\\\\\/qoEq5W53on5cuMHBzwn3BdaFLebGVNv8i4YNHXaVyJvFQ1w2vtrUbSOPniK5oB7hgH6a49SCKpzY\\\\\\\/SAYxMKNslykX9H9eVzqTcjPbUqyMYb9k8Ua\\\\\\\/pvbWk+GKRN6Xi\\\\\\\/hTQKcyVy526RM2FRpY5Kos0rhwbfh2hJNqD+ZwEMlNgOualfjfQPplbaRPQYM2oEoQApmkW+lfioLNwWiHb2gaMth3FUsf+vv1h8Zyyy3iIo8laSHm4d4miphAWJ0ymtdbzWWaJG3hBu66zpqtZ7q90yctJ5CnQfgCL5qeeC86XnDc2C8sIKZrPMbsv+B7seRPj2kTBeIaAeOGIvao+WBu98FjFTH2basc5WnDv3bG\\\\\\\/TIEW8h+jW3fVNtQIIli3Qb0rI++7YjaSbtfLVvQDAbmjEAXijzB4mP\\\\\\\/U7KaJw5s7mFk5vRkekqMfzTlUIIFPJliJJrorWJxpdbNUARwyVQhgrYdMK4QWYa\\\\\\\/AFWxLgNUpFciZD9M5P6FMlJhDp9EyDVicCElTzaV1nv0bIO7BmWwD3jlEkIc\\\\\\\/SLdEFegjl\\\\\\\/EH61K\\\\\\\/x0kXlI1X6hVZ7lTT\\\\\\\/k6sghWuH2CRASxdom\\\\\\\/7fiOIJCp38dh5IDVvRLVCeZsVPH8eTTs3rZZazQ44eW5oFGYhUeHm0773viRVEH2rdnM57CIs9ccvbd0vJNMN4Sn\\\\\\\/9z8DME5vD2zLkFfmrd+QmqQfO7eypCor1DoBNI7QdALz6Wh6\\\\\\\/UlF7+e0x3wCd2shkHhSPt5mVit1DceXHs0KRKP8JKwBbXfp6iHTMeTquKZydIJaySiRFAFxFxAWSRQho04iCzaq6vcMQIcv9ex1+xHcPLs8kmgAB4RWIkZkjmV8HUWWneaz0WQMAi3TpmLtwrrwQoR8ou09gYQIvSKX9dUEIlnqhzlnKP8QH+gAkl0xme+bDRNZsQGiOktW\\\\\\\/3XbzGbaPE8lUZVqDf6gHsrWj1ejlJ8Q78UA4qmVtIOH4Z1EGXVUlEKuQHurYqsSPhC6MXg8CN1JwOFFLv7+QqsVWDCABE\\\\\\\/GwMkg9VBlyAb2J\\\\\\\/7bEcBv6hgwl97tYisE7\\\\\\\/tS5VyRg1jLBcAulwk+jZ7yr4kAmTiNPictkaaATszdiaMhE02m9ZGirt1VL+V6CZFHHKOcjcU0LXvoZyFV7DKMiGehtcHpERQPiXaw4MmdfAla9JlydWx74w00eIAhlpb0gP66z13T3Dv79IcoWBQCCaOwn9D90knc8baZZ66EegxelicgJAEN6HMSe\\\\\\\/cRNrcTwFlLlBgjoDEikjEysdUTf4DwIEuCewN4JpgCaiFsqyPo07ZP5E+2EJ5ba+SwTajOSV9oXdCFdqsjx1JMhpQqwRbbUr228V3j7CMXQ\\\\\\\/8sCFJNogf4JTXsShGZFY7Fdfx8nz0kSytzjXPIN6hBgUoltkNi5jtsWn5ovn7pQ5RkrzFCr6Lzj5Xb+nANfg55R4jLqNppZU19SsUlgowUgh0LhTcUj491IGVNBpu+7UGXQc3kifaL2P9d5CqEsT2R0ehcVX++BHTgjE+c6iUGR4r+ejQoi3Q7ahvVWTGTEv0prtyDufoqJHU0kLx9IMHw8SMAWjn9mpfe8jGMygpGFhulQZ59\\\\\\\/I\\\\\\\/iT8VqIrb\\\\\\\/KsO2N3\\\\\\\/uK1MT29UNLog6Y3RoANMkiZY+vD1DMtB92gC8pOU\\\\\\\/OV4n64awBy8E47opJbUO6emzWYIQZ1DNgLM0EECbwepumQNTNnpXsOUC9orGghxXe1qA6qpJr2+e0XfBKdTVF0DakF\\\\\\\/lYnuRbYk1tBI1kqzNj4WM3PI+tTpP\\\\\\\/Km+R14aX+0X1qspEZWnsB6migqqWkKSa\\\\\\\/OtfK9LFiJvRkIJFSbegDtVfX+vHAMoDhnRBnZClFRd2N3XCV8VdCWSf5AghJenJy4yQzHMvKzmxoKbhYf2y+lmqQli4UMeTTEkdZZD\\\\\\\/wRauHfFHS0Um\\\\\\\/C2CzLl72C3Cug9BUwMaYlQe18jMelnOKuQaFqpIlvIa\\\\\\\/dk6u76qh7TQRDLNsh7MkJ3sYJ0tiEO3A8\\\\\\\/9CzzxsAGjGRtTfhsf2ShoWPEzUU4Re\\\\\\\/dojeiq4yrP\\\\\\\/hE3GP5gE7Q9P08Buie8JIAhfTDbu01ku9qnvaEKIwXnn\\\\\\\/ytFO9MT4nHx1z2G0AQuywBCPEBCUt6LYd1rw+Srpv7C0MEx24CBUGqws8j\\\\\\\/HCMfsmP5WVVy\\\\\\\/o12\\\\\\\/ThJykqpd92kzJtjQJ1AsT1owGzHMlb\\\\\\\/klnpjiENTqzWXCcite2+MoPAJK9hN4vjnH59RBnQlv0pwFTFVFZsm42I8j4jOSKRVFCUWZCDjIJGxsLOGNH9QXrFJFvnAJvAOwAZBCpxij8IonoRpLWcroOOs+LB55PqSd3xx6+OXO\\\\\\\/lpFNOeJ5k\\\\\\\/SqNjDY+fmQBbzmy97bLO6gg4Uy28GYYZNhcqiqvCiAERFkWUHjh2CQzkKpyMVaM7DyTAdZS7LL83vaRLT0abuCDDr9GlG1JOg5C\\\\\\\/K\\\\\\\/vSJeMyd9da2fYfAz+vzqKHvsu\\\\\\\/Kuh\\\\\\\/TWuIlgbZSTS08pJK3V+ScFq4a3zM1nwAMbZKdVpmIg377hTuhBS1OnFA38zB2mg2Oy+lYSeZOLtdD5IOKkHCeu1tGO4GCQjVAsWAXpL8+mI38CVpJrV23npejCNURFdesPX7\\\\\\\/mjreNw\\\\\\\/kwqtPk\\\\\\\/MCtPY4KT9HtqdVfVrrd2QAa5zsd3UWxXvbA6QzJc18RvCZuXT2fZqmq1dvi6XXBs3VPQCxa2U+1\\\\\\\/qETAGjQEqbG2ooWnj3LoG7M0kCEozu3vEcSnv+wKfP99WqKexOCINJex8l5jjdq9EfFdYl6qtMSek\\\\\\\/2ad5Va7njhrFV67Xy6zf6D9H1KP0N8UDELUoiyrMDpzlY5JEc+qewdcb1kqdIOoUmIVj\\\\\\\/bkT9Zv8XL7LBmPC4ZgUsnA6gk7gCrebhBV9C6e\\\\\\\/VECegnwyhxpWpmOZy0Wm8N7wJC4eGc59BwU7BVVU1xIghQu9ekfBg8E\\\\\\\/LZ8fl\\\\\\\/7P4H5w0EgWin5d0nQmEQes3GcAucDUm\\\\\\\/Kt4f6TCgNMPVwKMyO7eIPd1u2dLBUcSR0IbKzCk1rN6Geafk9dZa3HfXW9Hfk9LtI\\\\\\\/h63u9VTfxUW0pGD\\\\\\\/wykCqbaBnoU07I6EnDs1NoIdqL8evz\\\\\\\/ONLb\\\\\\\/96aIylmETaQ2EKmRsgZo7hl057m3CQKYVqOg8lx6L\\\\\\\/bbzjzSiVnMoWQ4rgCrBllmR0\\\\\\\/XhLaIm0oytV7RAVa2tMamT8XhrxTpBl+55WXuxBpc5nPkh9sMF8b76SNS+OpIChvZIpTxBEWUDxxSqPZDaBhnabq4hIAJ4OaEZ29lPX1Dm1HtwU66jY0t\\\\\\\/uZc7PzL9ebd0Uip0SowaoymuiHMBy5NJXYFuKOVoAK3N7wWA3OKfRpuEZ0ArKcDBw9MQfKq5t0VwHy3LVbS2aazFiMG2jkanY3iRkc4Kho7oLK8uJFajltu\\\\\\\/iekQIcpgq\\\\\\\/xix7JkqDu\\\\\\\/Jb1R2RYlv1UVrKlWF\\\\\\\/sDXqToLMFiKT9NkAQTMl7\\\\\\\/\\\\\\\/Ae5CYISMeaWinlTqjzbiolgoOrorS1V47EF15r7owgKiGgvb4Zz5b5rmW3HA48d0m2XgDuKY1henlmYT2iC+f6CRRCf5qNbGqMusuF1tYHSrDCGHrtD55EWJMbmdqXDZ6Zsaatg5B2dnQizJWuifCLvhNUuhc78GHe6o8zeyWmXGFyd5Sx4fuJnqEzENaSynxBg2urU2YVubh7K+wXksOeC3zUr4Vpj1mx0crj+RT4I6AA+ddB0W6Hx\\\\\\\/H7NUqViFfRsny4Yi9cetG2wBwzhOc3kYYTYo4ZfzEsj5uSH8H3GfN+iebQJ2jw0luzx0qK3wyXwT1r9N4NQojBTPl1FbQnGZnS77rx9Gy0UbFApq6U9UFZjMqtzmkoXf95vgMoC98JkWE01jX7S7hVQwyW5I\\\\\\\/vAXuZnqD4fm6OAEA8TZGcM956Rc9VXQVU2VsecpUukKvjDVBIW+0k\\\\\\\/goNWytYYs+FPmdQoE8PPXUpC42ovSrkZf8yiuHIqh7+lqFXZ\\\\\\\/+fjad19PJK5JDShire0wZfgivi1pQ9P0NZSsycHxnA1dlTYJi7VTtswx6xmyErREJO4NF5nEzVDWbos9N04ZntNS+fFHICvqzOaYXlFTGafG+xYxi03X7laSJef2mX6HYQK70QyfoXF+sCts3IvFFvP27c0qS9qon5eimORdDUYXu3NiAxdZcvMae3MhAt3JgSIsaGMibwYgYm\\\\\\\/si8ioKTe0kJMNoIIsiwDLhLp5jfVPJcJXMBhF3Yzp9fEl+\\\\\\\/FiN+95Hjcql4cb4lHFCGuaKF5jvyMQNQslbbev4WqdpTFTrtemrUOJw9P8EUx4HUC3l1NBSDxyYQJGrfgbS6Bpy3qsZSZsH6XBxROF1OVb\\\\\\\/As6tQ\\\\\\\/lqnoIEkN9wwLF8jXaNRaVOGojUaKlwLHCXcQ5JonJt+HP2MubFd3ACCWkzl0PXjp6CIxK+h4okWtaH8HS8MK3A6jZhJqQfQ2vD6G8HuFcLBYFpyL1As870mE4fZvraoulzLTF1N6ncnXIQkIPBhQvYAThG+E4lN\\\\\\\/S968Yr91gyas8ZIccgpMlflucr\\\\\\\/2mBIjSVMTJtzNK6LwrvyyltdgobUAFE5w2KURpKKQmxH7uGPIYD1MJA7WPKaYAdwIpP9YY8Pit79MA+7r2UU6G67etU85XW1S6HpJB4Zl\\\\\\\/7tYJl54NVF0T1XxnAF5urh\\\\\\\/NxCcM5AcLUGj0jRl26WdrNLNOGVhEAzH1vgy03w5iU5X5W7+xjRRQpNl0R\\\\\\\/2GtBYscDuFOCFwtK3Yvki5PL9DMobEpKFHt67uSddycie\\\\\\\/LYKwGS8h0UBargxSqBH1ZOVlzTlkPx+dt9\\\\\\\/zSLYSCMA+Rx4iS3Y6spRmbiBAy2yd9sNauWEbrpbzgvF2klcxi\\\\\\\/3FKh2BJi9RWygp8XPH\\\\\\\/+QX+P9aoRDN3jv5UH3M+VKIOqWon4j159b9oHyjZtmBVBu1FBaZNJeJTsksASnc+kdXW5y448ma6a9xd6tJtTrshYHFqSgJxcdWSzdN2vc7qcropUqcW6mds6fuSlloS2Ws7jPxq6khw\\\\\\\/EfC2CVR\\\\\\\/Erc9b6qT6v91h11sKyaB8Ap3TL5DRpUKnm4GIXcQ0xCXLY1Gx3rbRnJeISvYaWh18huXnMPBsjjwwT++81fI\\\\\\\/iPJ16ojJ2swveU8oEwu8FOgX+pIZ32mcH2MLdnjsMvBijuF5N6SgSsukSMLlvKrVNXQkt6bih85K3gKnqnBParja+8Bfp\\\\\\\/tcAIps6gzQ2x3Ek1YQcZUxML2j3btJ7SGbb3FTA7qfPjKKgnhPNeMQR0n4KyxnLLqwkIV3bNgXBlG4hBO5hyc2ItkNzVBUKGy0BRNSqFfSxxVDrsnsKXaQLKUrbqMc8\\\\\\\/0KVQW8rrWiOuTQ0IDF8\\\\\\\/GG0mHFRzfA+lIOV38hwQ\\\\\\\/9p3IzZZRj4P9W8mGI32DmOTHEGYywyNiqUy531bpKdmsOHlRnSCwKRQ\\\\\\\/sIiuDmon+3wQv32Q8amqH9DKEbdW1Zo4PSln1aKeRD83mjCQRWRr8b1t\\\\\\\/hJxSAqsndSxPlHGSzXUKjsweGM4zhdO+yC+y0BwQm\\\\\\\/EigX2CB\\\\\\\/WnSnJg1WUBhznerOce26nepdg19nM3PhQeLMzvTw1dYu3u0LQgXPTsCx0VCa0J3N0M9UCfvZkyJJPpeA7exHNgdSD1WmpTxWNbhuclvWwiSGyERymDBahExz5+XuZxFRLkyFIpUjTywUjGVWJL8MeexBBDm+xujo\\\\\\\/nWne81PIFqKk2ZRU6MBSsCjBMbDRMecNkpXK5fgmd\\\\\\\/AH7pP0vATwG5Wfs4YF7MmmtyA1\\\\\\\/QHgQbWboSGLdEFfIT8DXN6GhTFyvxVgi7KPcFFzbtHVty7ssuVsFnkQXOe7da6Zz2HgqmAPExIwaNnw9aqZ7mB0Q7no0eVMdH94sfzttw9287KEc9NzkF59pVkZbI1AfQlx8wXm44ipTtVOiLyml2nRgEyeQvwWGUds4bSkWuJgV56sFpHtzzW8HbsHMhT9lIlQtk9yL7iO2wCV3fTx8e1rkXakdO5K3dXSjJU0u076cw2VLGopjrI\\\\\\\/3xihrWvSj4srABOQyXBnhSalwokJEXt7ne9hOi6WIQlriw53ymPdIvKjYSpUsITZ8n7\\\\\\\/Q29lbHchWgMWFmLEbhdjpoCzSxPQpdtvUUKOu1d0wxTSAtX18C+QVs12U7T11\\\\\\\/Sx4pqSHsxXr9WUNaADfIykobOhoVtVL\\\\\\\/N+LPkqR\\\\\\\/qwZno+Cx1xlLUgCPE1dWbnBS3C\\\\\\\/SLv6ZmEHdcw1KYb51rbtCKJ06s1wjUgn8gONW+KIPkyAwZb3qxFGxQOUWTOQjM6MYwv+HJxhrVf170GJO733ItT19pshhWT5iyLQNreUSpKB100W\\\\\\\/3Unk37ONGkm7JBH2vDd60cLGRD55YlLSMU9RspH\\\\\\\/9BH5z6gGfABXHOoiqx9BM88XrzbdihkrIDHv2gZbDk2vObjqtODEHaPcyp4W\\\\\\\/kOqZ5jmDbsEe0CVWCKDw7ZIrTT306\\\\\\\/nWuzux8Alz8FnKYsjqRIy\\\\\\\/5rOwBhVj4KLv2fMUgaKL8TCqygc+BN6gnOiPrjEpfXDeHR\\\\\\\/4DY5zNXL2r+4LSP62w39NifzwU22bwOvxUSF\\\\\\\/w\\\\\\\/u5AyvqIuEjH2yjk4NUMePa0CCLLTrQgwGVxrZXieYuoaifsj\\\\\\\/kn3D2JbzZYknkOGjWsg17Wn47TSkP5csvct0Uj02AQk7+s2IyFEgrVv7V1R1\\\\\\\/NImaruV6\\\\\\\/oh5DX3PyPPpKC1OmktDb3CPNKxPYWVckdUk5P3Q3+cv0bHAHY1J2k2uwRw1NDXxB7hQpP5Y+2815wKOn\\\\\\\/SP3v+tJ3SJ3kDqAHxDSDCgTXMrEX6M3VxdMaXfZomDIzMKa95Bn4To59484w0NNo0Pl7kcIdbxBo26J0NolUamQqtOl\\\\\\\/HdVmv9DYkg+4uUYEOg2IGfBXT2PYyxl5faBRXyZLwoLmxS07QBghnC+nYrppRylhWrhIP0PWnZZDNtYB89ybzPNvrRKabx\\\\\\\/tpfFiwkwZfWlvJqCuogecymQdr+DTULWOJuqeQIop5GWZuvk3goKRzZfVilBWxBBwJnQPZOkVW+0AXHOuoY6WcKFikToTcu1x\\\\\\\/B+U0qGnYCPJ3NyAi9EgFrCdooPGIqA12q+IRWmc2Bg7RFh52MJLdmzjuOwwdQNavPLbRxhKKxszDVqlmuB\\\\\\\/6Zde3fIQWLsjXf5ZN9rqmCljFZfThlsQqLsl9Q81m6rtI1wOY8NZ5EJwFP+9D2BC\\\\\\\/hyLswVQs6\\\\\\\/7oW4EmrfDPpraCxYzCoUkRAk9IIbR6QVBLTOc781N74Fz1iypFee1mhXC\\\\\\\/oudubgh+Ous3kOEAMOVqlAasaQgAMKtjteHC7x1HW2DzF09BYJEgGCKI4gZnGUFw9qIDAU4n6WgWLoGn1k4O6SYvF+2BwlElBryAkcBZ6fPQj2PnPDXMBvTTbmbANG8+Bh\\\\\\\/cQUq2uX0ikJNVLGJNH14Vt4J3ZvP+Bjzr0dln+zcAx7C9Pd+4sAfdE9fO41QcymcJZShfJHZ5XH7QmmOFmq1JRz0aLb+v1ejh6J3adTr9iBYe5ibA1vlGmfSzivRoiOW0mSMlWL1CPyG1kOh6ubDsqTu0FTEsgH9n+a7Oki9dElj0o5\\\\\\\/OnbA0VwiAgZ3iqEeQ7rRkN4IT+pFOrH3eiQ0LojzhncDcsD67OZzLbLKki80gBDIeMQ84mBgmmjqf4+1Ew\\\\\\\/pSmVclB\\\\\\\/c\\\\\\\/JmyM4qZnn+d16VdpF9UUAx3\\\\\\\/nilpgY55WVvtM+x9azH7DHPLTi+rPiS8CyGXp6cKC29k1vtE2zjW3EQwkgxnsn2gyExEp+C6lceHkX7XuKnl+4jobzs6nF1oZLNfNdeqMywy61bqwCnpTRUEzCKBZ+LwogSfSn6LrjjnnLZgWSMhLPRUCnw8+zuNp2IlgZmBpTmMsY7btm7j2FMrG6yrO+GNzs4g4LCzPs2w0G+nUUmohXObYUro2Q5vQVIb5Qi8BipC88dm6YKer6Rh5oxyKm7Gk\\\\\\\/tmuOkRPDYlCUB0JQFwJbtmmmbQRku6CTBYDZtNpJUc8uSj5ItzuBYLDvDBIXUYnHfXTqSMpytsmIzgGcALFTges3fVUhUqTHJrVZEBKRODXOfAaYGMzgtUS3j9Paj3ooFbzIYinHr9\\\\\\\/WtY5eRGq4IHhx3TDHi4FUWzBOtpciBWOsBm0\\\\\\\/PKKm9EPHNukUcmSK0Gt2e\\\\\\\/wHlKuLwbZc1w2vK5ghq91lgljKN9iKEorqiL\\\\\\\/Sra6fzDH+1kzoXDENsQ+sY1gfcb2eHTWR6AnYdd4cgkoavnxjRcfnEhoBL9EeSvEijoKE38MvHqGKCkw5Xp5AuDoaaTEyn5LWaWSuK\\\\\\\/E315CtEawiEATzbq+rPgXiawxGYp2cm6XbsKCqNjfYFUNz\\\\\\\/pNSfhiXHQN\\\\\\\/6hPmMW9zjVWcoeTHX6XgNjyioir+9RiOnOXsvh0TRie+dWppQvIaUxHL1OPwUxqfaQRAiqzZtNG4udjIaLcQ3ULeHHPpFYTjemDrH1fwrdTC5S3gdVXW1YrX7+4mFR+4Lvp9TADpXft3Y1I16rZigd9O6sSr26Ey98RDXSLJ3s4Vja\\\\\\\/AOuDmmkhp23AGuinq8W9XPFaZYO4J3NQ2TEYq3ltXX+770zGtrYP1m3Dq3zfQaC9DwTVUNY243gdOUJ5sck7LMpDyuYxz+hId5VQiKl4+Jcwgsdft1Mlxdoc+mJCbfJJAt1pYBrAbA6jei9EX98VC2JaQU6L3sjY7ZszDBumYsHgFGPsMCPDweo0u0PQGP+40uhzX86fqmdeEcIgAIKk4J2NP3JnjGIPPbXUNdPUrY3Dl4\\\\\\\/uXU\\\\\\\/sVuykPb8ioqP3lgUIKitRNAfFlLTlGznnbVMnUKo5h8QA+xP9OTjtmfaAbiLxyjuwDOaWRCRLh1zLCdZnVS0NASmEcHl3U7n1lkfdPdWnbCH2FfNsnCTo\\\\\\\/H03ZKMUf8uAV6G\\\\\\\/cO6csECctbDJG+pUAkfu4Am6jeCdMQuRIe\\\\\\\/0brJg2MfaJyr9IZ2Pp1hnoMd8dFvPVJXMjKhy\\\\\\\/lzB1jJZ4KTB5P1baBTA2YD+GEPFYI\\\\\\\/hbMr4aY7P7KJWCSWYUBA+kOBLa+5Qi3iOxY+9dTmc5JsjGRel29xK2Hece7mp4\\\\\\\/Hq0lPgHqsp6DT9Uwpj6XIoCO7SDKKFKnnNRMGOvWRkZt9OYr5PYOnEH6XF+v4fdtIAF8ATpXJvkFQEA2vru+N1otrNBrILfpyRnSxp5OsBKEs9XZPVZERLtlUexBND29YfoXgEocQpcyojnAaQ75aKTKqeQnXqFEX1vGLlX1+NNJtTtUQrgTDQBVERQYAsA1Xbgg\\\\\\\/jaWDDnaxiLd27p3W1NrsEWY4TpStjZ8TTQ09bODwqe59a4rHBWd2O1BCGTKwtFaCGeU4xpvom59ADHBpeIZfgLFL6K\\\\\\\/eRe7ZCxBZQXTsOPXxRgFKbeiAxALDhHS2dewcyvsPsD4Tg2jhPqGhjPSBgjg40oya6UnkzmZPTfuX\\\\\\\/+x\\\\\\\/37URMSAjTpeVfHf4KPCXQBMFRbZgfHPvutDmR176v0Bi5LExGq6L3bUCIygrYJC92HtMfLFWhSSYcg0WEmHUQi2vQE9ULv0Li50NHaca6Bmc1uHQKeuDd+2f4LJF\\\\\\\/2MlBcWYTcie\\\\\\\/rU0+JwZJ3XW0VATSBQyVcnEEAyIfqyyJHhImLsH7UTp\\\\\\\/2TwUTdeKqIyqjh1NcuaaA3Sz5Cd+1Qg6yrt3CKHaybf5y3lYLe5hePd7YsR\\\\\\\/W9hzcfPYHLIhcQUCOdErw+f4w1jsaJ45yGl1urB33Qk9SSfSF5tGhMdwss84fEsM39dvhd4fi1j3pUq8uSGOkQTvGBqWclUML4CyuOzK0q6TQ8zjD8nnFfdmZmOnIuiEhN4sByFKSsg7p9+bt2tq9neoVJk5mo\\\\\\\/sHxgm9TO1OOD7sm8k1NEJ\\\\\\\/Z14xg6QaW+hWNGOgwE8HlPzSWoASSuAL5guYsa4cMX2ZC8cI+NYFnh1QIhaxk5fS8E0uqizapgpv9kShMDRFBMYBcy+aK5TpMkClkqKkRVZdFMye+193YWvJ\\\\\\\/Dv0LUVp8rWNsScUenMqIx8Nth79\\\\\\\/nhgeObZI+q9YkH6MEg5xTN78movowhLm1VB0LPJ\\\\\\\/dvlgOgsu0sCS1U24Og5wGk0K2q6dknCTEQ4KaDZQOqVjLEiv9V6nkazriuaBA6btEkT+XlYSbTOfNOdCOTzSurwJvvbKLuKprpvodiZ+AinhTkT0Fh79k6RQ8VDu\\\\\\\/wb8wU2JGqrxh2MLNG0ctV4i9CxLUhCjFV+TzRo9iGi9n6ofhGdTRSzbawrIcKqS8umX8DanZibmN8DtOcTNAyVluiWJZqwanQEb2YDkMJGNw0B7SccV+M5L8F\\\\\\\/euTj2cz0cwykAfHUPWZBwkcxVwUksKWTnDIqz5gOfNV+yPWv2MTWtjDi793vvdeJ1Qci8iVJKN+3C5uE8JRxffMnQe3sq6g5InGh3\\\\\\\/bijm9ySTn1FO1RK1HOMZ9AvvVNwU8+OBJJZh09EQVEuL6Sr6oOWDOtqtnbI2YxrO64RoXDJqJ+EKppe\\\\\\\/\\\\\\\/YPR1oJIuYM0v0z6tLpjijMGmFT0c52y5I0LsEUmRQZV9vuPDcvBj+Gl2wIas5sdr\\\\\\\/j0ZsS7FT99mq5fY1yBDXxadTGPr5bjb\\\\\\\/nkGVyUAyF8edU8zK5U4Ct9pS1fOHS9cEOO+N4k3BJjtHpNujvBmGvCl+MGizX62BLRdouJMjRkh+hTK8PPf7\\\\\\\/K3kYmOQB5PhmCrCp\\\\\\\/vXlyCGqK0VXPaDUAk6eby4+8jHteWUvqFZVrQBRxa9RGfy87RrB0WbRo+uPjGTab4Kg3nx4nHKqrLdTZTxLzph\\\\\\\/CYsdljoONfQk8VeOy7UtuOJMSqBoXfUUSLoJrRQG85c1FSH3We9vcUOOTvHrs4+gVnEaHp6BB5XZsjl0rbaGSPes8GP9NYg\\\\\\\/NRk2EXlPZtMyZ00pr2SS1SjnWMdK8Be570yxxvioRQb8dmsTQgn6Rq+orG7i8enIwjBOy1TrJssqGpP83p\\\\\\\/AuwRc\\\\\\\/0zNakqhLJaaQ96U5hmKcpNbF\\\\\\\/oGz5AZvj70gkHzZW42LvesEc9hYnSFDhrLFdr2gibReZRRF2lCJfDctm5ddPJIE9B\\\\\\\/XYZX7neQ3y6JR1WB4XkdEOZ4p9bcl\\\\\\\/SWI60KvH2nJlDyoF+Wc2XWXSYqoh4JDjk\\\\\\\/b006rTfOwLNkZP+GfO8NhnBdBeINc\\\\\\\/IAmTWzKAQF\\\\\\\/PT3QUujti83jOjtYITFN\\\\\\\/HsxFM\\\\\\\/okCAT3Zrd1j+oD81kVxaJyz6s6Wt4njYlhRcPZzNNWf8f02kgJiMb\\\\\\\/9oP1KZJ8We38BRt5ds5\\\\\\\/BtmmZB9YexPOSkAMsDvtyRJVDBALDf\\\\\\\/kzVOnTPrz9B\\\\\\\/C9JOjvGlT\\\\\\\/WPvXN15mzFsfsIyF7gVqOsDD3zw0UcmI5EcxW8YswiCTGWTF5rHOP1tChrHO7xpU9d53jLujGMAJknA0THBPqMJf2QgyJgpVSBGzJgwpOTkhoU8rEO54libu0pY0YRk1xMV2\\\\\\\/qBNUsl4KfqftZTmLTnUGyVAFIeXcGRDqqEd9F\\\\\\\/frWtC16k0nGxQ6v0zopiFM5duvs6+3SeYQsotigl1aOcAWOEbHPPZMj9aYBHiPjuTSjuNZI6GA3woKfctgKn6mNZIHyuTvqhsx4jZP\\\\\\\/lSddl7bH3hQPeBu8It0rqign\\\\\\\/MzPcuUYm80g0k4PMDeHqsM4AmIOTL\\\\\\\/1xPMIxZVlXMqVz5APzjqUm5C63kCPLpbfZj\\\\\\\/vogDr43H0G9vjsEZEgmfuRypvWFNko5CZOlsbYkCN3\\\\\\\/Lnzm3uVZdxqpoxA25wh5LQvbNq3Qh+\\\\\\\/ayJqLD8pdjJWCFC26Cak6xe9RLFRuf0yWKIvcZmxX4EHUeN9vvOWVZx6q8Ma+WpVRyDuJEbN+DkarNVCJQ\\\\\\\/b\\\\\\\/6CPzCdzZRpPhDNGxru7eYsE71lyTKkE7AcKwkEz71C9wpL54MqiU1B97fjek3AaOi\\\\\\\/7glViOatyiDnU+HSB0Xff9pIfYEgGP9LgAoFKBbLWN\\\\\\\/1Lzgxz2HzdzLLCmYR6CBUZCel4lcH4h6i5hbf7xX296xUiGqfgaWBA0gP1j1aakRuxRhExd852No5+4CIJMUyDRC9fRdmqI69iLuXkWtzr3ceacHTX05hnOLFYDW++p6qYseYPx7jO8d9gTxp0727V801eBUU5urgyQvT5NdIJiamHg+WRmmNuumFexT\\\\\\\/5quoFRkdAyQM+rw6dmDkGYJ7CcNPD8iRICfyNmFtvdCWe+1DK5jOSXY6MktH0n2KrUpCQQ14+0j15wwzTOgzO9ial4BUGPtG\\\\\\\/05\\\\\\\/nFirtoV4ev3B0kyOj72HUmkv4BJpLNoWDv49Nh8Btbj51M4QyUsDKR\\\\\\\/Cwd1f\\\\\\\/rdznEONIokKUhz+YeybHs9JdGiRPDuov5ch6R2cRUaq7j6v5i6VAVwvWSJh4pnbSSkeTiRcj4D5dvPzSnezHB+2buItiq+ljigpuqSKF+m5wWk\\\\\\\/VvoQA6Dgkme+X+tFhh+7iueXWtc3v7MYV50X2RMaUnK9gi\\\\\\\/koKlIVC0WASx9mi1AjKBlpPVO18uiQlUqROSNAlKJf1rRn01pC+XXdAz4dfFFVxKTKJvDBZsgW3X\\\\\\\/+tZxtziOZhBnKWCGifXVrMjVdRRd78goZUtBvICtifZyFADX9ZNu16mmU\\\\\\\/SEfq1z3Jlau5owtLT\\\\\\\/PZTFWmF0xU2Vj2CqczxOvw5H6mxOG0fvboyojgazbAOogTduorwIOXYFnlGHuaynq9CaNI\\\\\\\/19HKqFej0jpUsvl9BjidTZB2oq\\\\\\\/\\\\\\\/G+UXh5e47m3nSGxlB9RxiBD2I4d5Y2\\\\\\\/zFTp1omINz6oW1S+Y+Taak\\\\\\\/WQX2BjVJwyFqDAdZg1Q61y26w5SUhIJXdgjYXCDWDqLZ\\\\\\\/2vCt2P0zeNq4XiDesbbMa5VRETXWNMFHOC0hKe7F7Srg4PdKQNU71p6CQ\\\\\\\/4tKldpPFlvCI3qBBNDL+5g09YggFNU3rS9b+cuR3r6Zv5Hc83tEKTKqN+UUV+JvJ\\\\\\\/Zt8rYd8jRJdRlfEUBHhz1cx2X1WkiC\\\\\\\/gGTX60ZxaxP3VsKNrSULU9w7ZRqwqFY\\\\\\\/VkzMF8CCtF4O12ERNWfkeCc9w0xsplZMyQH3ZLhxG3mpeL+7WBjsaMchlY0XWGr2pzp+NrbgWyMHagtf6jX+TJj17UlmhSoV805lDuuC1Lfbh+GQf0XmuToSh8sMIFnqcePCVVMvrjmdflUOxJMpVFES2u59BGd\\\\\\\/31OmMKlwThTBBM+QL6+X1CiSoSmtqjJgX\\\\\\\/nCcLQDQ5yCiPRNv\\\\\\\/BpmKVD0to2F5gptJaIFxFsqmiD6F4tXMZzO0+kBR1UWPQei8m6ShlSzaPccx3kmEmVESOW8PGCpt5tJDmKL6WEMhXF9eJXxY7IZ256X2mk+uBRiZ1WcxSv2XKgqwJAArDcA5ZvD2vEksMXnHuT6\\\\\\\/0hZZ5Hd53oDLBw6BYxvcGEVRMX2aF1GddDVBg6OFIPpZAYmnpgNq\\\\\\\/pgXCttqlIk+GMUN683ngTGmolXSr38hS0T+GJEVINnb7C0AAbOPpfwNgtCh56W54VtIsbBS6jQWDRg\\\\\\\/874qBQjFSo4HvZ7scSFp3dnJqmdxrFZVzyMC+ETgPoLKEIF9MOgN8TxVBXVO7uXjS6SGt60ZlT7O+OoUPpk\\\\\\\/tSY3RC71DIIXqHBZH\\\\\\\/dgRgTd8LZSxw8hXe1\\\\\\\/PeFiHKvqSyDl\\\\\\\/L4L836wQLvDyN4Y\\\\\\\/6OczQ3fFscet7dJJ9Yg0i3fQD+e5ArBQIOUJ8\\\\\\\/\\\\\\\/GygM1Zb4Xv0AN1c0cV+aTPrH3w6aWw65j0Q8PjBxKF4wbKWaSeqIG+OAOiEEKgE6HxcyDNTOhbwLyfLp9yzboLiP4zJwqTUON0wxGvyBRa75TA6AfyO\\\\\\\/9YCl1Xk7KBf2XZ7X9AMDAbbuV\\\\\\\/ARVbXcN8fd5aERFkaeCq01fHKgDirrS4kfAtX8dDjD66w8I6JQtBqm8nsf+hgu9Rxz0z3gbwvhyRlho5V73ucnVoZILwHohm2V9TuRw0HKHfQQKAijmJT86MVfFBF\\\\\\\/d1yRjdOFM9cFd9+89L4c9Lp138+p39ZRHz1nkaNwKx2BwGA6R7K8CEppo4YmM4KGNuaQBUp7MEZNPNsQM3Fett34uQn4kcZUBAgVNOSE+BtuTJU0+tCUrSjcRs\\\\\\\/Z\\\\\\\/ngsgBahN3b6120qLSqmtyQLSl57pwx+koFWVGgk\\\\\\\/wtG\\\\\\\/7ELiaF3uYF63mVPzRn6GpCXyx\\\\\\\/t+42g+knAFDfo7OEhH7vZ0lpEtHNr7KABbeAGc9LquOlbAqnGKbT4uBpuEoFShNOjQGnHalui+x+Jx5A+1zaxn5zIor1GYF9vYNHOk55FBNyj8xgaDm33kioJJ7dZQZ5OosgNpAzS3cs2H+V\\\\\\\/KSpuctqhuluAOBp61trwProrr5zHHuaMZcA7Xcb+UWshCagNObkjlkM6vo9LPHhw+nuoLKkIkewol+FVeF+UAuiamr86RguANAblhvYLnhui0jQ9Ex+vkMF3UhHERFk+DQQqIag4UcdyLqzCzzEbifw7306gVlTycsGrWJZUEmJswsiLoLC6SCTjKi1aNTWlGrSEmLMWSrBF4lMwMn4UF8feZf9WAcEZGWobaWZDkioSFIi2yc5EiF0XoBTywWiTV5ttEud+W3NT3\\\\\\\/bVc1wYvZP1RbH+ZUbXc4+q4iOm\\\\\\\/3wIkLVfSNaPcdxz+CUUfSb0CldMyZxPDTp7VAJBzEOhaBmjSZFcX4L2s1RHhM3yaTdERLfRtz3iQ59ep3hyaRle3PLyzTCDA8qj7HvD\\\\\\\/alaoRt6K7g2gpONzvQ2acN18hCGer1O0mwT1ABRUVfo12dJlHnKWBrP9WvuTLUG08KYJ8e679exEaurr+nR6eTyY6fWc7alIjCmuYr0s9cWsVAoiIR02gOTlvWZQUNb6vLHcfwOF8L\\\\\\\/nLb9WC6rPdfW7bT+9EtP2usuItjO0C1ouLZzZuR2ty1E57KsZs66eYDThF10I8Z15mGOOHO9dcSsUphy18KxJpRlDcyHkKbiZXj0Le45fXhFZ9vj3ZeNNpnWnQGG5Kdaqk0ZIjjJKqLvUv+GbdlD3JH0QJjBZ1ERPnXIn1Vn+YF44UNoJD4Z9V9eeiuz\\\\\\\/GW4z0ZtUWilgxwENysxgcYa3UXi5SP5gW67H8Ub7YGY3KFRT3JJHRWZZoD20PWcr6o1K0xHDLsdNRhLziNYPD8f7u11GfW6QOcHfttsaIOwr0DAIoNKuOc5X7NCOwLft1KVqa7WDjR2pCoXKVu22I3cqCp6j\\\\\\\/DaVrRE2SsOLnoGd8+hgaaj11kvtSoZICT6DLwbTNNWmsWkXtUpMBQWWG6jyIAKjRWpNAPVJxei30fhP3XIPgJkxte+gA6zxIJi854oKXG\\\\\\\/37VYAfd56TjqvPYeyD2of57+mE+fxejumpESdTe72D5Y\\\\\\\/V2xFMI47vH5b4SjVkpnmp+yGxb+QdIJUSRxXCuL4hyJjzKcWPxI2YTD1MrTb8svBql5K3cm1lPBU+aUAm1zHRXesy2eQ3G6NnoxtIyizfkPrZilo1eMwK5oWF2rkvQ3HDOcqo5JomGgLIg74dHPwNi8mUeOmr8EGFtuCHsE2YbgoIPSO0DVxKQz9OkvZYSjJ6sSh0\\\\\\\/99sBw53tyUb3t1JHlEBSwiAvWXGdBDAy9htNZ0wKaR\\\\\\\/00WVFbR2rNyjITUK7MG4fZexRp+E9n+q1Oek+w7uqOp4yNU8ZNN0uHB19Iyzh9JnHjuPVErZmWFN5nf7xgKGU9tFd9Mkt7T6QYFMOesaRHPfshDw5B4jlVxZBfp+Zh53FoAFDaotbdI7zrOxuU7BpCHHqxwmnIRFTdposZ5iVGZyGGZ7vyF3io8xwk\\\\\\\/2yuCe4byakLDhPnyd7W4rnsNjlJSZZ\\\\\\\/yfQPJgL6TAgaymxkYt6yjMxPfFtRAXPxoOKSeU7GxQuDx9+QbizG03wJ9y0tRJ6aFvRLS8wqsn1KCnF5UIhPtc2PKLDlmjdV2JL64Hll90XFQBcZ4cH9X3EmaUFQCUxLmRd1R+SrpXm90pz1iqH6scMof4rlfY88tHeO+LZ9aGbAqThG+IZSUd\\\\\\\/1qTOHC4hFUC365u62dJbG9NiaJy9E5QXElgETcsjrHo7P4CGNQbrcFjDQRCK+SlovfLIhCftXxB0V+AvVvdgeW+gr5z+\\\\\\\/hEqVRdvUyS+SXGCj4crLz4Nra+2z33lg6KJoadXB+6CLUbNqI76QkJZXMEjMfasDbEosw\\\\\\\/CN8OQuBSLZUpub7lHzofpGJDxNJs2pmRLoK4ChXGsQlukOxjiJuzxLITytRcpmLD6o+rnMLT5TZXRK\\\\\\\/Qs1+iGbhPItSQaLK42IA3EBXQvNJceoXod02rCA3Dde0uiuQZzG9lnkghvbejX6WW1tCghojVpSPf9gGouWS1XtL727gBJPmK6666mk0DXb2JGd97oaXW5PhHGCtfnFqzDtFUMOyMkKR7on6VLD1QCzWrPF86lA+uHzSbQ4uPuOj80VFbgaZCjJRAq\\\\\\\/R+Zb\\\\\\\/RSaQXeB5fJXOP7UiZVZ4bdflfjKpH5gqhxmnNeUZ8X5CbjgotKU\\\\\\\/+4sQ7YjlmXniZzeMxYwnm\\\\\\\/H15IjnKWbuHWpEFPa8V+ndb1gEvk8cW2RR\\\\\\\/fO3UjRPBQMrWVzWrJ8jB6y\\\\\\\/tKILG\\\\\\\/GY0VuUcyaf+UOEjmd36LoKjnXFJS2BJoa9Nmv5LFa\\\\\\\/fZ+5rfb2ciKa3ZlV3Y9tYJqMmlMNLPRL+5qnE6X\\\\\\\/nwejlHLTlajgtXwTLfg2wdpxbM4NvwUQ9bXALTyUtvJ\\\\\\\/\\\\\\\/oYya4xq5cKFnus2hzDi2U8hdkhgknX1ZfyM+U9MFCvNMDzmRQpLubiCfdDJc0JxNxYn\\\\\\\/VIJQlUovGAp2hX3x8nACju54eugVd945\\\\\\\/u78vQSjkE8p4dkVs6QD3FGBLzKtO8gMduyKk\\\u0022,\\\u0022iv\\\u0022:\\\u0022sRM++7nJrjEr74hny4mclQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e26a36eff4f989300101185846811605f5a5ed13a7c4cf128d3b46a8517e331c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WtO8omH10JonUgVhRMrZ3zzNz6aLCrpqMiWQpLGpc1BAWcLNALQ4oBPQqCrCz3TopZhly3YeJ5NT\\\\\\\/3iJOiP28eG7g628EX9RKs0FGfaiwzqL7QDwEZZvHbBIFgFacEP0jGouiiWVfZjVTnrUu3Nr4WcIJxR\\\\\\\/FMICHzrSe1Ii+tc1ZrB7wMXLsiWiCqGjcSnn1IAl40z\\\\\\\/3UEqBuaQ6uA1DZvGhZntFqzkXJiCfWo7NPdNNOR0LQcGqgBWLhV6\\\\\\\/aii9ElXdV4t4\\\\\\\/3d5rzihQwA1bTYi0Kw6UVs\\\\\\\/3lznor3TnhHgI9wtScE0XFWPkx95WYvZ2aeN\\\\\\\/6AkRKPX+KOgNIbPkXqD\\\\\\\/N4+S4FN\\\\\\\/60+VSu+cVL3u5VCuHoW+OMMiWI2GhvagyPRaMepxMtTdIk47L0WP+vFpmFZyq5n0pKgSm9+D6u4cYfxeLUD8fKIPga40BVMeMU\\\\\\\/Rb7u+6A7B0w\\\\\\\/fKQE5C\\\\\\\/U85dL9UJVxK3aOJnz+pFCkPkVX1A\\\\\\\/DFQcZRlRpsWdFtlzccuoGzxgxHSaxZM1pRZ64IXau9o5ZGvczlFXJ\\\\\\\/RzzyrXJ4lMeAKhhne6Kb7\\\\\\\/HEeFIZSOXoXbe+4KmukuOviCPmk\\\\\\\/vEazKQR1oaC+0xxM2H\\\\\\\/UHTp4FgxF7PIDLgODwcOXDjK5OCa+8fBY\\\\\\\/9Jm4DwLnHzKwEcZXPDGJf\\\\\\\/JA\\\\\\\/TEVTuLEr93GgrRg121YyOlzX5ryl0LIXNuJxiKdx55Byk9kZf879InYcZBGI8bZhcGaYuxjYjsoNxUUrh1\\\\\\\/j1HlHYb59u8owbKVFGso2fz8iQZNstNHwOvqmU7k+\\\\\\\/6o4IzrWhc\\\\\\\/5VtpylnHsiwo3dlBnKjypmd4xL6FjviFq6mgz1ByZPuImNkRNEkdw75dUIDZQ2hk\\\\\\\/hwBo141JHJoEEJOup+xmQ4dJKAMW\\\\\\\/ujxyN6ILUSX9rZK0BVHJ42LgbYQwG92yIjCRKO\\\\\\\/IekSkixvnZvYdkO0ksiRCO4nAL\\\\\\\/UvdNiHfcCDp3hn9f1TWqYWz7u+kM\\\\\\\/9k6sBS1zMF3jC4b+tHDxJyvc21QVmdRm8\\\\\\\/Jwhqyk0o8wiyBHtiwXtOv8HMca\\\\\\\/XUb26R4zB8JXSH5G57u6H7Llg0v9YY19Aw8PmIQrkF7j5ZLKkKK+kuMuB23k7lo72tQUV5I+5Ad\\\\\\\/5e+CwlYRQRp4SHOgYxiRKjt3xrbq3+hQbBwGobLXIXuh\\\\\\\/TtCBE1YFny6DnutiEGqUCTrGcj1pS316bY1PVWk8fzrDVHEn91bj5A+C743xiRO1fIM8I74KZSWPvG69Hn9Wxg3TivAlDaHuqbcmvcvdORoEzTDMUAuoAqoMlmzrToK8Whkm1usNZK0bw4\\\\\\\/TZhFAvlS7TBBXYCVKtaSvrU01QPsEKwEofWkWbm6Iri1dR+xXCfATM3mVw6q5jzwh+cp6GEMX3KqwFf2jbeVJncNGQ10eW\\\\\\\/6IqkIMWqXV9XQJ2b5M9kovYtsqLq2BOGdOOPEBzmnPvft5qibweopDmluQ3cHD3vxFeQdg3Kxze1PWeqswLIlbVVWgfED0AkDNBYLdwGqAS6eEfY+bgwkREaz6vfYmxc2zxL3T3eGufYVdT2foS3E3g6TItfVHLkacm8U3ts5TQSniq\\\\\\\/8PIyn9B3ZYhAo50zt\\\\\\\/fGzCeYWdvCyIzbxcVxsZ+xDIc+SnBKGbTX80MvBiwPiV4CVjOrCiuJ3nxSboukOkmxV9YQMn3Y8QZ5cLHWbPQvuNprnBh3bfnu0okFxnRDi7bGso\\\\\\\/x78mt9xLfPSp45texmbunNF\\\\\\\/2fwGuMz2zJGB+0CKIcBCTcZhq6aXPWsaRlOdssR5D4IZDv573jGzZUPZnZtnV5kHPSso2F8rN9OuvY3F5aKhathcbLX6OxQBIKDleMkXZ1HfYngQPuPhyQp01fzT0SP75ZpYxxVPVY1VkpK6OMiJs7atS7R5c\\\\\\\/TvahkSEL5Fhue7JFvuQaG7O\\\\\\\/9P5GsOkW6bV3bh4U8lJnSkpBWes1ccxfDx2p64+cxNmPohtBI5zG0EAQMQ1S5xPZSIfV8vXNp6ucLlK2TiDUluU=\\\u0022,\\\u0022iv\\\u0022:\\\u00228eyYYyrV3KFArjrPUSpFVA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223432744e550d4d532cfdabcc04fa05812b9b6b8e3efb17e12c45ce451a35357b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487104 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3649" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PjrR9mLNqQd19x0XMFwWmcPJIANTkUZ83q8cTYJdBNw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3649" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G2Qcwj7jvr-pXwCgDmvG0H3OgrcHt2vmp4J4iiQ5hzI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3637" title="A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3659" title="The Effect of Common Inversion Polymorphisms In(2L)t and In(3R)Mo on Patterns of Transcriptional Variation in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487104" id="node487104" data-pisa="ggg;7/11/3649" data-pisa-master="ggg;g3.117.300121" data-apath="/ggg/7/11/3649.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Héloïse Giraud</span>, <span class="highwire-citation-author" data-delta="1">Cyril Bauland</span>, <span class="highwire-citation-author" data-delta="2">Matthieu Falque</span>, <span class="highwire-citation-author" data-delta="3">Delphine Madur</span>, <span class="highwire-citation-author" data-delta="4">Valérie Combes</span>, <span class="highwire-citation-author" data-delta="5">Philippe Jamin</span>, <span class="highwire-citation-author" data-delta="6">Cécile Monteil</span>, <span class="highwire-citation-author" data-delta="7">Jacques Laborde</span>, <span class="highwire-citation-author" data-delta="8">Carine Palaffre</span>, <span class="highwire-citation-author" data-delta="9">Antoine Gaillard</span>, <span class="highwire-citation-author" data-delta="10">Philippe Blanchard</span>, <span class="highwire-citation-author" data-delta="11">Alain Charcosset</span> and <span class="highwire-citation-author" data-delta="12">Laurence Moreau</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3649-3657;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487104" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Héloïse Giraud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9lo%C3%AFse%2BGiraud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giraud%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9lo%25C3%25AFse%2BGiraud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cyril Bauland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyril%2BBauland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bauland%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyril%2BBauland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthieu Falque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFalque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falque%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthieu%2BFalque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Delphine Madur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Delphine%2BMadur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Madur%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADelphine%2BMadur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valérie Combes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Val%C3%A9rie%2BCombes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Combes%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVal%25C3%25A9rie%2BCombes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Philippe Jamin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BJamin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jamin%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilippe%2BJamin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Cécile Monteil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9cile%2BMonteil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monteil%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9cile%2BMonteil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jacques Laborde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLaborde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laborde%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacques%2BLaborde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Carine Palaffre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carine%2BPalaffre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaffre%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarine%2BPalaffre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antoine Gaillard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maïsadour Semences SA, F-40001 Mont de Marsan Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BGaillard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gaillard%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntoine%2BGaillard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Philippe Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Euralis Semences, Domaine de Sandreau, F-31700 Mondonville, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilippe%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Alain Charcosset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BCharcosset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charcosset%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlain%2BCharcosset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Laurence Moreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BMoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreau%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurence%2BMoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:moreau@moulon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">moreau@moulon.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3649" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487104" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3649.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487104" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3649.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487104" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3649.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487104" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3649.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487104" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Identification of quantitative trait loci (QTL) involved in the variation of hybrid value is of key importance for cross-pollinated species such as maize (<em>Zea mays</em> L.). In a companion paper, we illustrated a new QTL mapping population design involving a factorial mating between two multiparental segregating populations. Six biparental line populations were developed from four founder lines in the Dent and Flint heterotic groups. They were crossed to produce 951 hybrids and evaluated for silage performances. Previously, a linkage analysis (LA) model that assumes each founder line carries a different allele was used to detect QTL involved in General and Specific Combining Abilities (GCA and SCA, respectively) of hybrid value. This previously introduced model requires the estimation of numerous effects per locus, potentially affecting QTL detection power. Using the same design, we compared this “Founder alleles” model to two more parsimonious models, which assume that (i) identity in state at SNP alleles from the same heterotic group implies identity by descent (IBD) at linked QTL (“SNP within-group” model) or (ii) identity in state implies IBD, regardless of population origin of the alleles (“Hybrid genotype” model). This last model assumes biallelic QTL with equal effects in each group. It detected more QTL on average than the two other models but explained lower percentages of variance. The “SNP within-group” model appeared to be a good compromise between the two other models. These results confirm the divergence between the Dent and Flint groups. They also illustrate the need to adapt the QTL detection model to the complexity of the allelic variation, which depends on the trait, the QTL, and the divergence between the heterotic groups.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/hybrids" class="hw-term hw-article-keyword hw-article-keyword-hybrids" rel="nofollow">hybrids</a></li><li class="kwd"><a href="/keyword/qtl-detection" class="hw-term hw-article-keyword hw-article-keyword-qtl-detection" rel="nofollow">QTL detection</a></li><li class="kwd"><a href="/keyword/additivity" class="hw-term hw-article-keyword hw-article-keyword-additivity" rel="nofollow">additivity</a></li><li class="kwd"><a href="/keyword/dominance" class="hw-term hw-article-keyword hw-article-keyword-dominance" rel="nofollow">dominance</a></li><li class="kwd"><a href="/keyword/silage-maize" class="hw-term hw-article-keyword hw-article-keyword-silage-maize" rel="nofollow">silage maize</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-4">Multiparental populations (MPP) have proved to be efficient for detecting loci involved in the variation of quantitative traits. Compared to biparental populations, they enable the exploration of more allelic diversity and improve the power and accuracy of QTL detection. Contrary to genome-wide association mapping based on panels of inbred lines, MPP designs composed of several biparental population families have a clear population structure. Controlling it in statistical analyses helps to prevent the risk of false positives due to associations between loci that are not physically linked. Such designs also permit alleles to be traced from the founder “parental lines” to the segregating populations, allowing the implementation of several detection models. The first joint analyses of several segregation populations considered that each founder line carried a different allele. In the case of Maize (<em>Zea mays</em> L.), this joint LA model led to the detection of allelic series for several traits of interest, <em>i.e.</em>, at least three significantly different parental allele effects (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Rebaï <em>et al.</em> 1997</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blanc <em>et al.</em> 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> 2014</a>). However, when the number of parents is high, this model may be overparametrized, especially when founder lines are related. Such founder relationships are a common feature in breeding programs, and lead to local IBD. In such a case, dense SNP genotyping of the founder parental lines enables application of a genome-wide association study model to reduce the number of parameters in the QTL detection and potentially increase the mapping resolution. This approach (often referred to as LDLA mapping) was first proposed by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Yu <em>et al.</em> (2008)</a> for NAM designs and was applied efficiently in the US-NAM design to fine-map QTL for several traits up to the gene level (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Tian <em>et al.</em> 2011</a>). It makes the implicit assumption that the QTL are biallelic and that the allelic effects do not depend on the genetic background. Several studies evaluated the properties of these different models by simulations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Li <em>et al.</em> 2016</a>) or empirically (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Tian <em>et al.</em> 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bardol <em>et al.</em> 2013</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> 2014</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Garin <em>et al.</em> 2017</a>) and consistently found that the models were complementary, with efficiencies depending on the trait, the design, and the QTL considered.</p><p id="p-5">In cross-pollinated species, heterosis is important for traits related to yield. Hybrids of agronomical interest are derived from crosses between unrelated individuals belonging to complementary genetic groups (heterotic groups). In this context, the value of a hybrid can be decomposed as the sum of the additive value of each of its parents (their GCA) and the interaction between its two parents (the SCA of the two parents) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Sprague and Tatum 1942</a>). Identifying loci involved in these two components is of interest in hybrid breeding. Most MPP designs evaluated so far for intergroup hybrid value have involved crosses of segregating individuals to a common unrelated line (“tester”). The variation of hybrid performances in such designs only reflects the allelic variation in the group of parental founder lines of the MPP. Also, as the GCA allelic effects are confounded with their SCA with the tester alleles, additive and dominant QTL effects cannot be distinguished and part of the variation of the MPP design can be masked by dominant tester alleles.</p><p id="p-6">In a companion paper, we proposed a new design to extend multiparental QTL detection designs to factorials between two heterotic groups (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> 2017</a>). This design relies on two multiparental maize populations corresponding to the two most important heterotic groups used in Europe (Dent and Flint). Within each heterotic group, a multiparental mapping population was developed from six connected biparental populations of segregating lines, issued from four founder lines. Instead of using “testers” (a common unrelated inbred line crossed to all experimental lines) to evaluate their hybrid values, segregating lines were crossed according to an incomplete factorial design to produce Dent-Flint hybrids. These hybrids were evaluated for biomass production. A first QTL detection was carried out to identify loci involved in the GCA and SCA components of hybrid value. This analysis was conducted considering allelic effects transmitted from each founder line. This model is an extension of joint LA to factorial models. It proved very encouraging to detect jointly GCA QTL in the two heterotic groups and test for SCA effects. The results suggested that most of the GCA QTL were specific to each group and no QTL with significant SCA effects at the genome level were found. This analysis raises issues regarding the number of parameters that need to be estimated at each QTL. With four founder lines per group, three independent GCA effects per group and nine independent SCA effects were estimated, leading to a total of 15 effects per locus. This might have negatively impacted the power of QTL detection. Therefore, the question of the potential of allele clustering appears to be particularly important in this context to (i) restrict the number of parameters to be estimated and (ii) test the consistency of allelic effects in the two groups. The first point is particularly critical for dominance effects in designs that would involve a high number of parents in each group.</p><p id="p-7">The objective of this study is therefore to propose alternative QTL detection models adapted to the analysis of factorials between segregating populations, and test their efficiency in the factorial design proposed by <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. Comparison of the different QTL models in such design can provide insight into the importance of allelic series and the divergence between the Dent and Flint groups.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genetic material</h3><p id="p-8">The experimental material consists of a Dent-Flint factorial between segregating lines used in the companion paper by <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. Four founder inbred lines were chosen within each heterotic group (Dent and Flint): one for its digestibility and the others for their yield potential. In each group, all crosses between pairs of founder lines were made to produce six biparental populations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In total, 931 Dent lines and 913 Flint lines were obtained by doubled haploidization and five to six generations of selfing, respectively. From these parental lines, 863 Dent lines and 879 Flint lines were crossed in an incomplete factorial design in order to produce 1044 experimental Dent-Flint hybrids. Each biparental population of one group was crossed with all the biparental populations of the other group, with the objective of balancing their contribution. The majority of lines (699 in the Dent and 732 in the Flint) contributed to only one hybrid, but some lines contributed twice (163 in the Dent group and 146 in the Flint group). Only two lines contributed to three or four hybrids. All founder lines of one group were crossed with the founder lines of the other group to create 16 hybrids that were used as checks.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the experimental design. The table shows the number of Dent-Flint hybrids retained for QTL detection for each of the 36 Dent-Flint combinations of biparental populations. QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1665393690" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of the experimental design. The table shows the number of Dent-Flint hybrids retained for QTL detection for each of the 36 Dent-Flint combinations of biparental populations. QTL, quantitative trait loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3649/F1.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3649/F1.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487241" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Schematic representation of the experimental design. The table shows the number of Dent-Flint hybrids retained for QTL detection for each of the 36 Dent-Flint combinations of biparental populations. QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Genotyping data</h3><p id="p-10">The founder lines were genotyped with an Illumina 50 K SNP array (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ganal <em>et al.</em> 2011</a>). The founder lines and the parental lines were genotyped for a subset of 18,480 SNPs with an Affymetrix array designed by Limagrain. For the analyses, we considered the Affymetrix genotyping data for the founder lines, and when possible replaced missing data by the genotypes obtained with the 50 K SNP array. To avoid ascertainment bias, we only considered the PANZEA markers (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Ganal <em>et al.</em> 2011</a>) that were polymorphic among the founder lines. We restricted the analyses to loci that had &lt;20% missing values within the Dent and Flint sets of parental lines. Markers with &gt;5% of heterozygosity among the Dent parental lines or in total, or &gt;10% of heterozygosity among the Flint parental lines (issued from single seed descent and therefore with some expected residual heterozygosity) were discarded. Markers with a Minor Allele Frequency (MAF) &lt;5% were discarded. After applying the above-mentioned criteria, 9643 markers were retained for further analyses.</p><p id="p-11">Genotyping failed for nine inbred lines. Genotype consistency between founder lines and parental lines was checked and off-type lines were excluded, as well as inbred lines showing a high level of heterozygosity (&gt;25 and 10% for the Flint and the Dent lines, respectively). In total, 875 Dent lines and 883 Flint lines were retained for further analyses. This data can be found in the supplemental material of <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>.</p><p id="p-12">These 1758 inbred lines were used to build 12 genetic maps, one for each of the 12 biparental populations as well as one Dent-Flint consensus map, established following the approach described in <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> (2014)</a>. Segregation distortion was tested within each population and markers with unexpected segregation were discarded prior to map building. Most allele frequencies ranged from 0.4 to 0.6 except for a few chromosome segments in some populations (for instance on chromosome 2 for the population D6), suggesting no strong involuntary selection during line development. On average, distances between consecutive map positions were &lt;2 cM but large gaps (&gt;10 cM) were observed (for instance on chromosome 4 for the population F2), corresponding to chromosome regions where the two founder lines of the population were IBD (Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>). The Dent-Flint consensus map comprised 9548 markers that were polymorphic in at least one Dent or one Flint population. This map has a total length of 1578.6 cM and contains 5216 unique positions (<em>cf</em>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS1.txt">File S1</a> for the consensus and individual population maps). Missing genotypes of parental lines were imputed with Beagle v3.0. (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Browning and Browning 2009</a>). As Dent lines were doubled haploids, heterozygous loci for Dent lines were considered as missing data and were imputed considering that only homozygous genotypes were possible. Imputations were performed within each population with the founder lines included. Phasing (for the Flint lines and the founder lines) and missing genotype imputation were done at the same time.</p></div><div id="sec-4" class="subsection"><h3>Field trial design and estimation of least squares-means (ls-means)</h3><p id="p-13">Hybrids were evaluated in eight different environments (four locations in 2013 and four in 2014) in the North of France and in Germany. Trials were conducted following common agricultural practices of the region. Four traits were measured: silage yield (DMY in tons of dry matter per ha), dry matter content at harvest (DMC in % of fresh weight), plant height (six environments) (PH in centimeters) and female flowering (DtSILK in days after January the first, scored as the date at which 50% of the plants of the elementary plot exhibited stigmas, referred to as “silks” in maize). The field experiments consisted of 1088 experimental units, each a field plot of two 5 m-long rows. The experimental design was laid out as an augmented p-rep design (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Williams <em>et al.</em> 2011</a>). The hybrids between founder lines and ∼17% of the experimental hybrids were evaluated twice per environment, whereas most of the hybrids between the parental inbred lines were evaluated only once. Trials were laid out in 68 incomplete blocks consisting of 16 plots each, with five to six plots used for repeated genotypes (experimental hybrids and checks). In total, 1044 hybrids were evaluated over the whole experimental design. After removing outlier observations (at the phenotypic and genotypic level), 951 hybrids were considered for QTL detection (950 for PH and DMY), involving 822 and 802 parental lines in the Flint and Dent group, respectively. The hybrids retained for the QTL detection belonged to all of the 36 hybrid populations corresponding to the crosses between the six Flint and the six Dent line populations [figure 1 and table 1 of <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>].</p><p id="p-14">QTL detection was based on the ls-means of each hybrid over the environments. For DMY, data from one of eight environments were excluded as they were not correlated with the other environments. For each trait, we first corrected single-plot values by spatial effects obtained by analyzing jointly all the field trials and considering for each trial the best spatial model (row-column model or block model). Ls-means of hybrids were derived from the following model: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-1.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-2.gif"/></noscript></span></span> is the performance of the hybrid <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-3.gif"/></noscript></span></span> located at position <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-4.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-5.gif"/></noscript></span></span>, and block <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-6.gif"/></noscript></span></span> in the environment <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-7.gif"/></noscript></span></span> corrected for the spatial field effects, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-8.gif"/></noscript></span></span> is the intercept, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-9.gif"/></noscript></span></span> is the fixed effect of the environment <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-10.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-11.gif"/></noscript></span></span> is the hybrid genetic effect considered as fixed. <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-12.gif"/></noscript></span></span> is the residual of the model <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-13.gif"/></noscript></span></span>. Details of the models used for spatial correction are described in <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. The R-script used to estimate ls-means is included in the supplemental material of <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>.</p></div><div id="sec-5" class="subsection"><h3>QTL detection</h3><p id="p-15">Three models were used for QTL detection differing by (i) the fact that they consider either the alleles transmitted by the parents of the hybrids or directly the hybrid genotypes, and (ii) the type of allele coding considered (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The population structure of the design was taken into account for all models. Also, we included random genetic effects corresponding to the parents of the hybrids to account for the fact that some parental inbred lines were involved in several hybrids.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow of data analysis for the phenotypic data, genotypic data and QTL detection. For each QTL model we indicated the number of d.f. corresponding to the QTL effects. Methods and results of the estimation of variance components are presented in Giraud et al. (2017). GCA, General Combining Ability; LD, linkage disequilibrium; ls-means, least squares-means; SCA, Specific Combining Ability; SNP, single nucleotide polymorphism; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1665393690" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Workflow of data analysis for the phenotypic data, genotypic data and QTL detection. For each QTL model we indicated the number of d.f. corresponding to the QTL effects. Methods and results of the estimation of variance components are presented in Giraud &lt;em&gt;et al.&lt;/em&gt; (2017). GCA, General Combining Ability; LD, linkage disequilibrium; ls-means, least squares-means; SCA, Specific Combining Ability; SNP, single nucleotide polymorphism; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3649/F2.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3649/F2.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487149" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-16" class="first-child">Workflow of data analysis for the phenotypic data, genotypic data and QTL detection. For each QTL model we indicated the number of d.f. corresponding to the QTL effects. Methods and results of the estimation of variance components are presented in <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. GCA, General Combining Ability; LD, linkage disequilibrium; ls-means, least squares-means; SCA, Specific Combining Ability; SNP, single nucleotide polymorphism; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The Founder alleles model is the one described in <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. It considers the founder alleles transmitted to the hybrids and makes the assumption that each of the eight founder lines carries a different allele.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-15.gif"/></noscript></span></span> is a (<em>N</em> × 1) vector of the ls-means of the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-16.gif"/></noscript></span></span> experimental hybrids phenotyped for the considered trait; <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-17.gif"/></noscript></span></span> is the intercept, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-18.gif"/></noscript></span></span> is a (<em>N</em> × 1) vector of 1. The term α (respectively β) is a (6 × 1) vector of the fixed effects of the Dent (respectively Flint) populations of origin of the Dent (Flint) parental line, (αβ is a (36 × 1) vector of the fixed interaction effects between the Dent and Flint populations of parental lines. <em>A</em>, <em>B</em>, and <em>C</em> are the corresponding design matrices. <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-19.gif"/></noscript></span></span> (respectively <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-20.gif"/></noscript></span></span>) is a (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-21.gif"/></noscript></span></span> × 1) [respectively (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-22.gif"/></noscript></span></span> × 1)] vector of the random effects of the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-23.gif"/></noscript></span></span> Dent (respectively <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-24.gif"/></noscript></span></span> Flint) parents, with <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-25.gif"/></noscript></span></span> (respectively<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-26.gif"/></noscript></span></span>). <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-27.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-28.gif"/></noscript></span></span>are the corresponding design matrices. <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-30.gif"/></noscript></span> </span>are GCA effects not accounted for by the QTL. <em>e</em> is a (<em>N</em> × 1) vector of the residuals of the model with <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-31.gif"/></noscript></span></span> The QTL effect is decomposed into three terms: <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-32.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-33.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-34.gif"/></noscript></span></span> The first term <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-35.gif"/></noscript></span></span> (respectively <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-36.gif"/></noscript></span></span>) is the (4 × 1) vector of the allelic effects at the marker associated with each Dent (Flint) founder line. These effects correspond to the GCA effects of the QTL. For each marker, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-37.gif"/></noscript></span></span> (respectively <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-38.gif"/></noscript></span></span>) is a (<em>N</em> × 4) matrix of the probabilities that the hybrid received its Dent (respectively Flint) allele from each of the four Dent (respectively Flint) founder lines. <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-39.gif"/></noscript></span></span> is the (16 × 1) vector of the interactions, or SCAs, between the founder alleles; <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-40.gif"/></noscript></span></span>is a (<em>N</em> × 16) matrix corresponding to the elementwise product between each column of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-41.gif"/></noscript></span></span>and each column of <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-42.gif"/></noscript></span></span> As the sum of probabilities for each allele equals 1, this model has three d.f. for the additive effects of the founder alleles (GCAs) in each group and nine d.f. for the interaction effects (SCA). Probabilities that a hybrid received one of the four Dent (respectively Flint) founder alleles were inferred for each position of the 9548 mapped markers based on the genotypes of its parental lines at the closest informative markers. These probabilities were computed with PlantImpute (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hickey <em>et al.</em> 2015</a>) using 10 iterations.</p><p id="p-18">The SNP within-group model considers the observed alleles at SNPs received from the parental inbred lines, assuming different effects in the two heterotic groups. This model assumes that two inbred lines from the same group sharing the same allele at a given SNP are IBD at this position and transmit the same QTL allele to the hybrids.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>All effects are defined as in the Founder alleles model, except that for the QTL effects two alleles segregate in each group instead of four. The model estimates the contrast between these two allelic effects by performing a regression on the SNP minor allele frequencies. The first term <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-44.gif"/></noscript></span></span>(respectively <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-45.gif"/></noscript></span></span>) is the GCA effect associated with the Dent (respectively Flint) minor allele, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-46.gif"/></noscript></span></span> (respectively <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-47.gif"/></noscript></span></span>) is a (<em>N</em> × 1) vector of marker genotypes for the Dent (respectively Flint) parent of the hybrid, coded as 0 for homozygotes for the major allele, 1 for homozygotes for the minor allele, and 0.5 for heterozygotes (only Flint lines, see above). <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-48.gif"/></noscript></span></span> is the SCA effect between the minor SNP marker alleles of each group, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-49.gif"/></noscript></span></span> is a (<em>N</em> × 1) column vector corresponding to the Hadamard product of <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-50.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-51.gif"/></noscript></span></span> This model has one d.f. for the GCA effect of each group and one d.f. for the SCA.</p><p id="p-19">The Hybrid genotype model considers the marker genotypes of the hybrids and ignores the group origin of the alleles transmitted by the parents. It assumes that the QTL effects are the same in both heterotic groups and decomposes the QTL effect into additivity and dominance terms.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-52.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Compared to the previous models, QTL effect is decomposed into two terms <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-54.gif"/></noscript></span></span>, which are, respectively, the additive and dominance effect at the marker. <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-55.gif"/></noscript></span></span> is a (<em>N</em> × 1) vector coded in −1, −0.5, 0, 0.5, and 1, corresponding to the genotypes of the hybrids, inferred from the genotypes of their parental lines. <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-56.gif"/></noscript></span></span> = −1 when the hybrid is homozygous for the major allele, 1 when it is homozygous for the minor allele, 0 when it is heterozygous, and −0.5 (respectively 0.5) when the Dent parent is homozygous for the major (respectively minor) allele and the Flint parent is heterozygous. <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-57.gif"/></noscript></span></span> is a (<em>N</em> × 1) vector coded in 0, 0.5, and 1. <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-58.gif"/></noscript></span></span> equals 0 when the hybrid is homozygous, 0.5 when the Dent parent is homozygous and the Flint parent is heterozygous (note that the reverse is not possible as Dent parents are doubled haploids), and 1 when the hybrid is heterozygous. This model has one d.f. for the additive effect and one d.f. for the dominance effect.</p><p id="p-20">QTL detection was performed with the package ASReml-R (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Butler <em>et al.</em> 2007</a>) of R (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">R Core Team, 2013</a>) considering the level of significance of the Wald test for the QTL effects. For the SNP within-group and the Hybrid genotype models, QTL detection was performed on the 4758 mapped markers that were polymorphic (MAF &gt; 5%) in both heterotic groups, whereas for the Founder alleles model it was performed on the 9548 mapped markers. For each model, we considered a 5% genome-wide significance threshold based on the number of effective markers (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gao <em>et al.</em> 2008</a>). The total effect at each marker position was tested using the “group” function of the ASReml-R package (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Butler <em>et al.</em> 2007</a>). After the first initial single-marker scan along the genome, a multimarker procedure was implemented using a forward and backward marker selection process, similarly to the one presented in <a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. Only markers with significant effects were kept in the final model. The R-scripts used to perform QTL detection are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS2.txt">File S2</a>.</p><p id="p-21">From the final multilocus model, we estimated the percentage of phenotypic variance explained by the detected QTL <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-59.gif"/></noscript></span></span> and the percentage of within-population phenotypic variance explained by the detected QTL, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-60.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3649/embed/mml-math-61.gif"/></noscript></span></span> is the percentage of variance explained by the population effects in a model without QTL. We also estimated the individual <em>R</em><sup>2</sup> of each QTL [see <a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>, for more details].</p><p id="p-22">To evaluate the quality of prediction of these models, we performed a cross-validation approach following the procedure described in <a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. Eighty percent of the data (training set) was sampled in each population and used to identify QTL, estimate the population and QTL effects, and predict the values of the hybrids on the remaining 20% (test set). Sampling was repeated 100 times. To limit computation time, for each sampling, the training set was used to test the significance of the QTL detected in the whole data set and only significant QTL were considered in the prediction model. The percentages of variance explained by the models were estimated by the squared correlation between the predicted and observed hybrid values of the test set. This procedure was conducted (i) without taking into account SCA/dominance QTL effects and (ii) taking them into account for QTL for which they were significant at a 5% individual risk level. A model including only the population effects and no QTL was also considered.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-23">The consensus map and individual maps are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS1.txt">File S1</a>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS2.txt">File S2</a> contains the R-scripts used to perform QTL detection. The p-values of each marker in the single-marker scans for all traits and QTL models are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS3.txt">File S3</a>. Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a> contains information about the genetic maps. Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a> present the QTL detection results of the final multimarker models. Figure S1, Figure S2, and Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a> show the comparison of the QTL results obtained with the different models for DMC, DtSILK, and PH. Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a> shows an overview of the QTL detected in this study. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS4.txt">File S4</a> contains a description of the genetic material and the supplemental material. Pedigrees of the segregating populations, raw phenotypic data, adjusted means of hybrid performances, and genotypic data of parental lines are available as supplemental files of the companion paper (<a id="xref-ref-11-13" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> 2017</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Thresholds for QTL detection</h3><p id="p-24">The thresholds at a 5% genome-wide level used for QTL detection were determined as −log(p-value) equal to 4.53 for the Hybrid genotype model, 4.40 for the SNP within-group model, and 3.84 for the Founder alleles model. These differences reflect that, in the Founder alleles model, genotypic data at closely linked loci are highly correlated, leading to a lower threshold.</p></div><div id="sec-9" class="subsection"><h3>Detection of QTL for all trait × model combinations</h3><p id="p-25">We detected QTL for all trait × model combinations. Note that results for the Founder alleles model are identical to those shown in a companion paper <a id="xref-ref-11-14" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>. As expected, we observed that the test statistics at adjacent positions were closer for the Founder alleles model than for the two others (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For a given trait and a given QTL model, the number of detected QTL in the final multilocus models varied between 9 (DtSILK, Founder alleles model, and DMY, SNP within-group model) and 16 QTL (DtSILK and Hybrid genotype model) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In total for the four studied traits, the SNP within-group model and the Hybrid genotype model detected more QTL (51 and 54, respectively) than the Founder alleles model, which detected in total only 42 QTL (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Nevertheless, the Founder alleles model detected more QTL for DMY. To compare the QTL detected by the different models (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and Figure S4, Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>), we considered that QTL detected within 10 cM of each other were identical. With this assumption, 59 QTL were specific to one model and only 16 QTL were detected with all the three models.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL detection for DMY with (A) the Founder alleles model, (B) the SNP within-group model, and (C) the Hybrid genotype model for the single-marker analysis. The chromosome number is indicated on the x-axis. For each model, graphics correspond to the test of the global effect (on the top) or of one component (Flint GCA, Dent GCA, and SCA effects for the Founder alleles and SNP within-group models; and additive and dominance effects for the Hybrid genotype model). The blue (black) dots correspond to positions that were above (below) the threshold in the single-marker analysis (see File S3). The red squares correspond to the &#x2212;log(p-value) of the QTL that were included in the final multilocus model, with tests conditioned by the other QTL effects of the model. DMY, Dry Matter Yield; GCA, General Combining Ability; LD, linkage disequilibrium; ls-means, least squares-means; SCA, Specific Combining Ability; SNP, single nucleotide polymorphism; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1665393690" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL detection for DMY with (A) the Founder alleles model, (B) the SNP within-group model, and (C) the Hybrid genotype model for the single-marker analysis. The chromosome number is indicated on the &lt;em&gt;x&lt;/em&gt;-axis. For each model, graphics correspond to the test of the global effect (on the top) or of one component (Flint GCA, Dent GCA, and SCA effects for the Founder alleles and SNP within-group models; and additive and dominance effects for the Hybrid genotype model). The blue (black) dots correspond to positions that were above (below) the threshold in the single-marker analysis (see File S3). The red squares correspond to the &#x2212;log(p-value) of the QTL that were included in the final multilocus model, with tests conditioned by the other QTL effects of the model. DMY, Dry Matter Yield; GCA, General Combining Ability; LD, linkage disequilibrium; ls-means, least squares-means; SCA, Specific Combining Ability; SNP, single nucleotide polymorphism; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3649/F3.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/11/3649/F3.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3649/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487236" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">QTL detection for DMY with (A) the Founder alleles model, (B) the SNP within-group model, and (C) the Hybrid genotype model for the single-marker analysis. The chromosome number is indicated on the <em>x</em>-axis. For each model, graphics correspond to the test of the global effect (on the top) or of one component (Flint GCA, Dent GCA, and SCA effects for the Founder alleles and SNP within-group models; and additive and dominance effects for the Hybrid genotype model). The blue (black) dots correspond to positions that were above (below) the threshold in the single-marker analysis (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS3.txt">File S3</a>). The red squares correspond to the −log(p-value) of the QTL that were included in the final multilocus model, with tests conditioned by the other QTL effects of the model. DMY, Dry Matter Yield; GCA, General Combining Ability; LD, linkage disequilibrium; ls-means, least squares-means; SCA, Specific Combining Ability; SNP, single nucleotide polymorphism; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487116/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487116/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">QTL detection results with the different detection models for the different traits</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">For all models, the majority of the QTL explained &lt;5% of the variation (see Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>). The only notable exception was a QTL detected by the three models on chromosome 10 at 44.5 cM, which explained ∼8% of the variance for DMC and 13% of the variance for DtSILK for all the detection models. Other QTL regions showed pleiotropic effects on different traits (Figure S4, Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>).</p></div><div id="sec-10" class="subsection"><h3>Decomposition of the global effect of the QTL in its different components</h3><p id="p-29">We tested the level of significance of GCA/SCA or additive/dominance components for all QTL that were detected (Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>). Whatever the model considered, none of the detected QTL showed a significant dominance/SCA effect at a 5% genome-wide level. Some markers had significant dominance effects in the single-marker QTL detection scan with the Hybrid genotype model but their effects were not significant in the final multilocus model (see <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> for DMY). However, some QTL were significant for SCA/dominance with an individual risk at 5%: 9 for the Hybrid genotype model, 6 for the SNP within-group model, and 13 for the Founder alleles model (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>). Among them some were significant at a 1% risk level. QTL showing SCA/dominance effects were located all over the genome. However, one region on chromosome 2, between 82.3 and 135.8 cM, stands out for presenting SCA for both DMC and DMY (Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>).</p><p id="p-30">The Founder alleles and the SNP within-group models decomposed the QTL effects into its Dent and Flint components. A majority of QTL appeared specific to one group. For the SNP within-group model, 9 QTL were significant for both GCA effects, 23 only for the Dent GCA effect, and 15 only for the Flint one (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>). For the Founder alleles model, seven QTL were significant for both GCA effects, 21 only for the Dent GCA effect, and 12 only for the Flint GCA effect (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1/FileS5.docx">File S5</a>). The other QTL correspond to QTL with significant global effect but no significant individual GCA component. For QTL detected at close positions with several models, GCA/additive QTL effects of the founder lines were consistent between models (result not shown).</p></div><div id="sec-11" class="subsection"><h3>Variation explained by the QTL</h3><p id="p-31">The detected QTL explained jointly between 19.7% (DMY, Hybrid genotype model, without dominance) and 37.6% (DtSILK, SNP within-group model, with SCA) of the total phenotypic variance, and between 26.8 and 47.1% of the within-population phenotypic variance (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The model that explained the largest fraction of the phenotypic variance was the Founder alleles model for DMY, DMC, and PH, and the SNP within-group model for DtSILK. The increase in percentage of explained phenotypic variance when taking into account dominance/SCA was low for the SNP within-group model (+0.2 for DMY to +1.9 for PH) and for the Hybrid genotype model (+0.6 for DMC to +1.5 for DtSILK), whereas it was more important for the Founder alleles model (+4.1 for PH to +6.5 for DMY) (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-32">Cross-validations were performed to eliminate potential bias in the <em>R</em><sup>2</sup> values of <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> that were computed on the data also used to estimate QTL parameters, potentially advantaging models with a high number of parameters. The highest <em>R</em><sup>2</sup> of models combining population and QTL effects were obtained with the Founder alleles model for PH and the SNP within-group model for DtSILK, and to a lesser extend DMC and DMY (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We observed a reduction of the <em>R</em><sup>2</sup> obtained by cross-validation compared to the <em>R</em><sup>2</sup> evaluated on the whole data set (<em>R</em><sup>2</sup><sub>pop+QTL</sub> column of <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>) for all the models. This reduction was stronger for the Founder alleles model. It has to be noted that, for this model, the number of QTL found to be significant when considering only four-fifths of the data were lower than the number of QTL detected using the whole data set (results not shown). The same tendency was observed, but to a lesser extent, for the other QTL models. Taking into account the dominance/SCA for the QTL for which it was significant at a 5% individual risk always had a small negative impact on the <em>R</em><sup>2</sup> of the models, especially for the Founder alleles model.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487243/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487243/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Cross-validation estimates of the quality of prediction of different models (average <em>R</em><sup>2</sup> and its SD)</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Comparison of QTL detection models showed the predominance of group-specific GCA QTL</h3><p id="p-34">Compared to the Founder alleles model, the SNP within-group and the Hybrid genotypes models consider alleles defined at the level of SNPs (with the SNP within-group and Hybrid genotype models). These models are close to the ones used for association mapping (LD mapping), except that we used the known population structure of the design instead of a kinship matrix to control for false positives. They correspond to an extension of the models proposed for NAM designs (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Yu <em>et al.</em> 2008</a>) to the case of hybrids. The three models used for QTL detection performed differently depending on the trait and the genomic region considered. As they rely on different assumptions in terms of allelic effects, they are expected to perform differently depending on the actual distribution of QTL effects. The Hybrid genotype model considers only two d.f. per marker and is thus expected to be more powerful than the other models, which have more parameters per marker. However, it makes the strong assumptions that (i) the QTL are biallelic; (ii) they have the same effect in both heterotic groups; (iii) the marker-QTL phase is also conserved between groups; and (iv) there is no epistasis. The other models have more parameters but make fewer assumptions: (i) the effect of a given QTL and/or the marker QTL-phase depend on the heterotic group for the SNP within-group model and, in addition, (ii) each founder line has a different allele at the QTL for the Founder alleles model.</p><p id="p-35">The SNP within-group model found more QTL than the Founder alleles model for all traits but DMY. This is consistent with observations by <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> (2014)</a> for European NAM designs and supports the hypothesis that allelic series for yield are more complex than for other traits [see figure 3 from <a id="xref-ref-11-15" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>].</p><p id="p-36">The Hybrid genotype model detected the highest total number of QTL but it almost never explained the largest part of the genetic variance (considering direct adjustment of the data or cross-validations). The strong constraints it considers for estimating genetic effects therefore counterbalanced its advantages in terms of power. This is consistent with the detection of QTL specific to Dent or to Flint GCA by the other models. Thus, the Founder alleles and the SNP within-group models seem better adapted to QTL detection in such a design. This is in agreement with <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> (2014)</a>, who found different QTL in the Dent and Flint heterotic groups. The same conclusion was drawn by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">van Eeuwijk <em>et al.</em> (2010)</a> when analyzing a maize factorial between two other heterotic groups for ear height and by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Parisseaux and Bernardo (2004)</a> considering intergroup hybrids obtained by crossing lines issued from a total of nine different heterotic groups. Group-specific GCA QTL may be due to actual differences in QTL allelic variability but may also result from epistatic effects. Differences in linkage disequilibrium phases between heterotic groups [as found by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lehermeier <em>et al.</em> (2014)</a> between the Dent and Flint groups] may partly explain the lower efficiency of the Hybrid genotype model.</p><p id="p-37">Whatever the model considered, we did not detect QTL with SCA/dominance effects significant at a 5% genome-wide risk level. We nevertheless detected dominance and/or SCA effects significant at a 5% individual risk level for some QTL (at a 1% individual risk level for three of them). As for the Founder alleles model, cross-validation results showed that adding SCA QTL effects to the models slightly decreased the quality of prediction of hybrid values, suggesting that these moderate QTL SCA effects may not be well estimated in training sets whatever the model considered. Reducing the number of parameters via the use of the SNP within-group or the Hybrid genotype models instead of the Founder alleles model did not help to identify QTL involved in SCA. This suggests that SCA is due to numerous small effects that are difficult to detect and/or that they involve allelic interactions (dominance) that cannot be captured by a single parameter (as assumed by models based on SNP genotypes). Another explanation might be that SCA is due to epistasis that was not included in our detection models.</p><p id="p-38">Hence, results from the different models consistently show that performance of hybrids between lines from different heterotic groups is mostly affected by GCA QTL that are located at different positions in the two groups of interest. As discussed in <a id="xref-ref-11-16" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>, this result is consistent with the strong divergence between the two groups that were considered in this study and with GCA explaining 80% of hybrid variation (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Reif <em>et al.</em> 2007</a>). The small percentage of SCA variance certainly partly explains why we did not detect QTL with significant SCA at the genome level. Therefore, it would be interesting to compare the efficiency of our different models for detecting dominance/SCA effects in other experiments showing a higher contribution of SCA/dominance to the total hybrid variation. This may also be done by simulations but this is beyond the scope of this paper.</p></div><div id="sec-14" class="subsection"><h3>Potential improvement of QTL detection models for a higher number of founder lines</h3><p id="p-39">Our results are consistent with those of <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bardol <em>et al.</em> (2013)</a> and <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> (2014)</a>, who also found that the model considering that each founder line carried a different allele (the Founder alleles model) was more adapted to complex traits such as yield than to simpler traits such as flowering time. As discussed in <a id="xref-ref-11-17" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> (2017)</a>, one of the main drawbacks of this QTL detection model is that it requires the estimation of many parameters. This phenomenon is reinforced in the present hybrid design, with 35 d.f. for the combinations between the Dent and Flint populations and nine d.f. for the SCA per QTL, in addition to six d.f. for the GCA effects per QTL. This certainly explains the strong reduction of <em>R</em><sup>2</sup> observed for this model in cross-validation results compared to other models. Use of this model makes it necessary in practice to develop large segregating populations from few founder lines to get enough power and accurate QTL effect estimates. The SNP within-group model that was tested seems to be a good alternative for more complex designs, at least for traits that are expected to show “simple” allelic series (such as DtSILK). An intermediate strategy between the Founder alleles and the SNP within-group models would be to cluster the parental alleles based on their local similarities, as proposed by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Leroux <em>et al.</em> (2014)</a> and evaluated experimentally by <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Bardol <em>et al.</em> (2013)</a>, <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> (2014)</a>, and <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Han <em>et al.</em> (2016)</a>. Recently, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Garin <em>et al.</em> (2017)</a> reanalyzed part of the Dent NAM design of <a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Giraud <em>et al.</em> (2014)</a> and showed that it can be useful to mix in the same model QTL with different types of effects (parental, ancestral based on allele clustering, or biallelic). It would be interesting to adapt this strategy to factorial designs. Another possibility for more complex pedigree could be to consider QTL effects as random and use markers to compute local similarity, as done by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crepieux <em>et al.</em> (2004)</a> on wheat inbred line data and recently by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Tisné <em>et al.</em> (2015)</a> in oil palm hybrids. <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">van Eeuwijk <em>et al.</em> (2010)</a> performed QTL detection in a factorial design issued from a private breeding program that was derived by crossing a large number of parental lines (but not structured in balanced families as in our design). Their analyses were based on a Bayesian model that used both molecular markers and pedigree to trace back ancestral founder alleles at QTL, assumed to be biallelic, and compute local similarity. SCA was not included in their analysis. Another improvement could be to use markers to compute kinship matrices to handle covariances between individuals due both to the population structure and the effect of unlinked QTL, as proposed by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Xu (2013)</a>, and applied to QTL mapping in MAGIC populations by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wei and Xu (2016)</a>. Nevertheless, this approach needs to be adapted to factorial designs.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We thank T. Mary-Huard for helpful discussions and advice for the estimation of <em>R</em><sup>2</sup>. We are grateful to anonymous reviewers and editors L. McIntyre, J. Holland, and D. J. de Koning for very helpful comments. We thank Caussade Semences, Euralis Semences, Limagrain Europe, Maïsadour Semences, Pioneer Genetics, R2n, and Syngenta Seeds grouped in the frame of the ProMais SAM-MCR program for the funding, inbred lines development, hybrid production, and phenotyping. We are also grateful to scientists from these companies for helpful discussions on the results. We thank the Institut National de la Recherche Agronomique metaprogram SelGen—and specifically the X-Gen and SelDir projects coordinated, respectively, by Andrés Legarra and Hélène Gilbert—for partial funding and scientific feedback on the results. H.G. was funded by the Université Paris Sud through a scholarship from the Ministry of Research given by the Doctoral School ABIES (Agriculture Food Biology Environment Health).</p><p id="p-41"> <em>Note added in proof:</em> See H. <a id="xref-ref-11-18" class="xref-bibr" href="#ref-11">Giraud <em>et al.</em> 2017</a> (pp. @@@–@@@) in Genetics 207-3, for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-42">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300121/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-43"><em>Communicating editor: L. McIntyre</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-07"><span class="received-label">Received </span>August 7, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-09-05"><span class="accepted-label">Accepted </span>September 5, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Giraud <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.1" data-doi="10.1007/s00122-013-2167-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bardol</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventelon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loywick</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Combined linkage and linkage disequilibrium QTL mapping in multiple families of maize (<em>Zea mays</em> L.) line crosses highlights complementarities between models based on parental haplotype and single locus polymorphism.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2717</span>–<span class="cit-lpage">2736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2717%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2167-9%26rft_id%253Dinfo%253Apmid%252F23975245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2167-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23975245&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.2" data-doi="10.1007/s00122-006-0287-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBlanc%26rft.auinit1%253DG.%26rft.volume%253D113%26rft.issue%253D2%26rft.spage%253D206%26rft.epage%253D224%26rft.atitle%253DConnected%2Bpopulations%2Bfor%2Bdetecting%2Bquantitative%2Btrait%2Bloci%2Band%2Btesting%2Bfor%2Bepistasis%253A%2Ban%2Bapplication%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0287-1%26rft_id%253Dinfo%253Apmid%252F16791688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0287-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16791688&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238503200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.3" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.4" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3649.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source"><em>ASReml-R Reference Manual</em>.</span> <span class="cit-publ-name">Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane, QLD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.6" data-doi="10.1534/genetics.104.028993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crepieux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebreton</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait loci (QTL) detection in multicross inbred designs: recovering QTL identical-by-descent status information from marker data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1737</span>–<span class="cit-lpage">1749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCrepieux%26rft.auinit1%253DS.%26rft.volume%253D168%26rft.issue%253D3%26rft.spage%253D1737%26rft.epage%253D1749%26rft.atitle%253DQuantitative%2BTrait%2BLoci%2B%2528QTL%2529%2BDetection%2Bin%2BMulticross%2BInbred%2BDesigns%253A%2BRecovering%2BQTL%2BIdentical-by-Descent%2BStatus%2BInformation%2BFrom%2BMarker%2BData%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.028993%26rft_id%253Dinfo%253Apmid%252F15579720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC8zLzE3MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.7" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.8" data-doi="10.1002/gepi.20310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starmer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DGao%26rft.auinit1%253DX.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D361%26rft.epage%253D369%26rft.atitle%253DA%2Bmultiple%2Btesting%2Bcorrection%2Bmethod%2Bfor%2Bgenetic%2Bassociation%2Bstudies%2Busing%2Bcorrelated%2Bsingle%2Bnucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20310%26rft_id%253Dinfo%253Apmid%252F18271029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18271029&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255471100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mezmouk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">How do the type of QTL effect and the form of the residual term influence QTL detection in multi-parent populations? A case study in the maize EU-NAM population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D1753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.10" data-doi="10.1534/genetics.114.169367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage disequilibrium with linkage analysis of multiline crosses reveals different multiallelic QTL for hybrid performance in the flint and dent heterotic groups of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169367%26rft_id%253Dinfo%253Apmid%252F25271305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.11" data-doi="10.1534/genetics.117.300305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madur</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Combes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Reciprocal genetics: identifying QTLs for general and specific combining abilities in hybrids between multiparental populations from two maize (<em>Zea mays L</em>.) heterotic groups.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.117.300305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300305%26rft_id%253Dinfo%253Apmid%252F28971957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy8zLzExNjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Choice of models for QTL mapping with multiple families and design of the training set for prediction of <em>Fusarium</em> resistance traits in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettelblad</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Imputation of single nucleotide polymorphism genotypes in biparental, backcross, and topcross populations with a hidden Markov model.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1934</span>–<span class="cit-lpage">1946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.14" data-doi="10.1038/ng.747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kump</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcher</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKump%26rft.auinit1%253DK.%2BL.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D168%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bquantitative%2Bresistance%2Bto%2Bsouthern%2Bleaf%2Bblight%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.747%26rft_id%253Dinfo%253Apmid%252F21217757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217757&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.15" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krämer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (<em>Zea mays</em> L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzY0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.16" data-doi="10.1007/s00122-014-2267-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leroux</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahmani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventelon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Clusthaplo: a plug-in for MCQTL to enhance QTL detection using ancestral alleles in multi-cross design.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D921%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2267-1%26rft_id%253Dinfo%253Apmid%252F24482114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2267-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24482114&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spamer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparison of statistical models for nested association mapping in rapeseed (<em>Brassica napus</em> L.) through computer simulations.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D16%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parisseaux</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">In silico mapping of quantitative trait loci in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DParisseaux%26rft.auinit1%253DB.%26rft.volume%253D109%26rft.issue%253D3%26rft.spage%253D508%26rft.epage%253D514%26rft.atitle%253DIn%2Bsilico%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bin%2Bmaize.%26rft_id%253Dinfo%253Apmid%252F15150690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15150690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223097700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3649.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em></span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.20" data-doi="10.1007/s001220050582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebaï</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perret</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vincourt</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mapping quantitative trait loci controlling silking date in a diallel cross among four lines of maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XV26800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.21" data-doi="10.1534/genetics.107.074146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gumpert</span>  <span class="cit-name-given-names">F.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Impact of interpopulation divergence on additive and dominance variance in hybrid populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1931</span>–<span class="cit-lpage">1934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074146%26rft_id%253Dinfo%253Apmid%252F17507673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE5MzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.22" data-doi="10.2134/agronj1942.00021962003400100008x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sprague</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatum</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1942</span> <span class="cit-article-title">General <em>vs.</em> specific combining ability in single crosses of corn.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Agron.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BAgron.%26rft.volume%253D34%26rft.spage%253D923%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1942.00021962003400100008x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1942.00021962003400100008x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.23" data-doi="10.1038/ng.746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTian%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D162%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bleaf%2Barchitecture%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.746%26rft_id%253Dinfo%253Apmid%252F21217756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217756&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.24" data-doi="10.1186/s12864-015-1985-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tisné</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomiès</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riou</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mixed model approach for IBD-based QTL mapping in a complex oil palm pedigree.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D798%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1985-3%26rft_id%253Dinfo%253Apmid%252F26472667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1985-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26472667&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.25" data-doi="10.1007/s00122-009-1205-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totir</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed model approaches for the identification of QTLs within a maize hybrid breeding program.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253Dvan%2BEeuwijk%26rft.auinit1%253DF.%2BA.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D429%26rft.epage%253D440%26rft.atitle%253DMixed%2Bmodel%2Bapproaches%2Bfor%2Bthe%2Bidentification%2Bof%2BQTLs%2Bwithin%2Ba%2Bmaize%2Bhybrid%2Bbreeding%2Bprogram.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1205-0%26rft_id%253Dinfo%253Apmid%252F19921142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1205-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19921142&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272803700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.26" data-doi="10.1534/genetics.115.179945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Random-Model Approach to QTL Mapping in Multiparent Advanced Generation Intercross (MAGIC) Populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179945%26rft_id%253Dinfo%253Apmid%252F26715662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNDcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.27" data-doi="10.1002/bimj.201000102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Augmented p-rep designs.</span> <abbr class="cit-jnl-abbrev">Biom. J.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiom%2BJ%26rft.aulast%253DWilliams%26rft.auinit1%253DE.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DAugmented%2Bp-rep%2Bdesigns.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.201000102%26rft_id%253Dinfo%253Apmid%252F21259306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.201000102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21259306&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.28" data-doi="10.1534/genetics.113.157032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping quantitative trait loci by controlling polygenic background effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.157032%26rft_id%253Dinfo%253Apmid%252F24077303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3649.29" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3649.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3637" title="A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3659" title="The Effect of Common Inversion Polymorphisms In(2L)t and In(3R)Mo on Patterns of Transcriptional Variation in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3649.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iElutRnObDteWE8qhjDTDv3DCDg-ZNm0y26kVW3F6qI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3649" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3649" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3649" class="active" data-icon-position="" data-hide-link-title="0">Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487104" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xjj8Quez3fPQY8QEMTwROBFNvL1-U4_qvVvxFnNfRMA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3649.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3649" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VmxywiQli-XYfe1Ic9zgQHjR0tO1KjPs6BYjwITDBf0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3649" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3649.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487104" id="node487104--2" data-pisa="ggg;7/11/3649" data-pisa-master="ggg;g3.117.300121" data-apath="/ggg/7/11/3649.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Héloïse Giraud</span>, <span class="highwire-citation-author" data-delta="1">Cyril Bauland</span>, <span class="highwire-citation-author" data-delta="2">Matthieu Falque</span>, <span class="highwire-citation-author" data-delta="3">Delphine Madur</span>, <span class="highwire-citation-author" data-delta="4">Valérie Combes</span>, <span class="highwire-citation-author" data-delta="5">Philippe Jamin</span>, <span class="highwire-citation-author" data-delta="6">Cécile Monteil</span>, <span class="highwire-citation-author" data-delta="7">Jacques Laborde</span>, <span class="highwire-citation-author" data-delta="8">Carine Palaffre</span>, <span class="highwire-citation-author" data-delta="9">Antoine Gaillard</span>, <span class="highwire-citation-author" data-delta="10">Philippe Blanchard</span>, <span class="highwire-citation-author" data-delta="11">Alain Charcosset</span> and <span class="highwire-citation-author" data-delta="12">Laurence Moreau</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3649-3657;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487104--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Héloïse Giraud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9lo%C3%AFse%2BGiraud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giraud%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9lo%25C3%25AFse%2BGiraud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cyril Bauland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyril%2BBauland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bauland%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyril%2BBauland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthieu Falque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFalque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falque%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthieu%2BFalque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Delphine Madur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Delphine%2BMadur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Madur%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADelphine%2BMadur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valérie Combes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Val%C3%A9rie%2BCombes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Combes%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVal%25C3%25A9rie%2BCombes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Philippe Jamin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BJamin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jamin%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilippe%2BJamin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Cécile Monteil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9cile%2BMonteil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monteil%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9cile%2BMonteil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jacques Laborde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLaborde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laborde%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacques%2BLaborde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Carine Palaffre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carine%2BPalaffre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaffre%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarine%2BPalaffre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antoine Gaillard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maïsadour Semences SA, F-40001 Mont de Marsan Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BGaillard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gaillard%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntoine%2BGaillard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Philippe Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Euralis Semences, Domaine de Sandreau, F-31700 Mondonville, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilippe%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Alain Charcosset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BCharcosset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charcosset%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlain%2BCharcosset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Laurence Moreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BMoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreau%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurence%2BMoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:moreau@moulon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">moreau@moulon.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3649" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3649&amp;title=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3649&amp;title=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3649&amp;title=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3649&amp;text=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3649&amp;title=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3649&amp;t=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3649&amp;title=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3649&amp;title=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487104/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487104" id="node487104--3" data-pisa="ggg;7/11/3649" data-pisa-master="ggg;g3.117.300121" data-apath="/ggg/7/11/3649.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Héloïse Giraud</span>, <span class="highwire-citation-author" data-delta="1">Cyril Bauland</span>, <span class="highwire-citation-author" data-delta="2">Matthieu Falque</span>, <span class="highwire-citation-author" data-delta="3">Delphine Madur</span>, <span class="highwire-citation-author" data-delta="4">Valérie Combes</span>, <span class="highwire-citation-author" data-delta="5">Philippe Jamin</span>, <span class="highwire-citation-author" data-delta="6">Cécile Monteil</span>, <span class="highwire-citation-author" data-delta="7">Jacques Laborde</span>, <span class="highwire-citation-author" data-delta="8">Carine Palaffre</span>, <span class="highwire-citation-author" data-delta="9">Antoine Gaillard</span>, <span class="highwire-citation-author" data-delta="10">Philippe Blanchard</span>, <span class="highwire-citation-author" data-delta="11">Alain Charcosset</span> and <span class="highwire-citation-author" data-delta="12">Laurence Moreau</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3649-3657;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487104--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Héloïse Giraud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9lo%C3%AFse%2BGiraud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giraud%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9lo%25C3%25AFse%2BGiraud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cyril Bauland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyril%2BBauland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bauland%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyril%2BBauland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthieu Falque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFalque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falque%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthieu%2BFalque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Delphine Madur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Delphine%2BMadur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Madur%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADelphine%2BMadur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valérie Combes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Val%C3%A9rie%2BCombes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Combes%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVal%25C3%25A9rie%2BCombes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Philippe Jamin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BJamin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jamin%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilippe%2BJamin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Cécile Monteil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9cile%2BMonteil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monteil%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9cile%2BMonteil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jacques Laborde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLaborde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laborde%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacques%2BLaborde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Carine Palaffre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Unité Expérimentale 0394 Saint-Martin-de-Hinx Maïs, Institut National de la Recherche Agronomique, F-40390 Saint-Martin-de-Hinx, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carine%2BPalaffre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaffre%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarine%2BPalaffre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antoine Gaillard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maïsadour Semences SA, F-40001 Mont de Marsan Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BGaillard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gaillard%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntoine%2BGaillard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Philippe Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Euralis Semences, Domaine de Sandreau, F-31700 Mondonville, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilippe%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Alain Charcosset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BCharcosset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charcosset%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlain%2BCharcosset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Laurence Moreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Quantitative et Évolution-Le Moulon, Institut National de la Recherche Agronomique, Université Paris-Sud, Centre Nationnal de la Recherche Scientifique, AgroParisTech, Université Paris-Saclay, F-91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BMoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreau%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurence%2BMoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:moreau@moulon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">moreau@moulon.inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3649&amp;rft.epage=3657&amp;rft.atitle=Linkage%20Analysis%20and%20Association%20Mapping%20QTL%20Detection%20Models%20for%20Hybrids%20Between%20Multiparental%20Populations%20from%20Two%20Heterotic%20Groups%3A%20Application%20to%20Biomass%20Production%20in%20Maize%20%28Zea%20mays%20L.%29&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Giraud%2C+H%C3%A9lo%C3%AFse&amp;rft.au=Bauland%2C+Cyril&amp;rft.au=Falque%2C+Matthieu&amp;rft.au=Madur%2C+Delphine&amp;rft.au=Combes%2C+Val%C3%A9rie&amp;rft.au=Jamin%2C+Philippe&amp;rft.au=Monteil%2C+C%C3%A9cile&amp;rft.au=Laborde%2C+Jacques&amp;rft.au=Palaffre%2C+Carine&amp;rft.au=Gaillard%2C+Antoine&amp;rft.au=Blanchard%2C+Philippe&amp;rft.au=Charcosset%2C+Alain&amp;rft.au=Moreau%2C+Laurence"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487104/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487104/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487104/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487104/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487104/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487104/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487104/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487104/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487104/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487104/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487104/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487104/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3649" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3649.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3649.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3649.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3649.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
