<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3775" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3775.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3775.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3775.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3775.atom" />
<meta name="HW.pisa" content="ggg;7/11/3775" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events" />
<meta name="DC.Identifier" content="10.1534/g3.117.300262" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Omer et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Okazaki fragments that are formed during lagging strand DNA synthesis include an initiating primer consisting of both RNA and DNA. The RNA fragment must be removed before the fragments are joined. In Saccharomyces cerevisiae , a key player in this process is the structure-specific flap endonuclease, [Rad27p][1] (human homolog FEN1). To obtain a genomic view of the mutational consequence of loss of [RAD27][1] , a S. cerevisiae [rad27][1] Δ strain was subcultured for 25 generations and sequenced using Illumina paired-end sequencing. Out of the 455 changes observed in 10 colonies isolated the two most common types of events were insertions or deletions (INDELs) in simple sequence repeats (SSRs) and INDELs mediated by short direct repeats. Surprisingly, we also detected a previously neglected class of 21 template-switching events. These events were presumably generated by quasi-palindrome to palindrome correction, as well as palindrome elongation. The formation of these events is best explained by folding back of the stalled nascent strand and resumption of DNA synthesis using the same nascent strand as a template. Evidence of quasi-palindrome to palindrome correction that could be generated by template switching appears also in yeast genome evolution. Out of the 455 events, 55 events appeared in multiple isolates; further analysis indicates that these loci are mutational hotspots. Since [Rad27][1] acts on the lagging strand when the leading strand should not contain any gaps, we propose a mechanism favoring intramolecular strand switching over an intermolecular mechanism. We note that our results open new ways of understanding template switching that occurs during genome instability and evolution.

 [1]: http://www.yeastgenome.org/locus/S000001596/overview" />
<meta name="DC.Contributor" content="Sumita Omer" />
<meta name="DC.Contributor" content="Bar Lavi" />
<meta name="DC.Contributor" content="Piotr A. Mieczkowski" />
<meta name="DC.Contributor" content="Shay Covo" />
<meta name="DC.Contributor" content="Einat Hazkani-Covo" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Okazaki fragments that are formed during lagging strand DNA synthesis include an initiating primer consisting of both RNA and DNA. The RNA fragment must be removed before the fragments are joined. In &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, a key player in this process is the structure-specific flap endonuclease, Rad27p (human homolog FEN1). To obtain a genomic view of the mutational consequence of loss of &lt;i&gt;RAD27&lt;/i&gt;, a &lt;i&gt;S. cerevisiae rad27&lt;/i&gt;Δ strain was subcultured for 25 generations and sequenced using Illumina paired-end sequencing. Out of the 455 changes observed in 10 colonies isolated the two most common types of events were insertions or deletions (INDELs) in simple sequence repeats (SSRs) and INDELs mediated by short direct repeats. Surprisingly, we also detected a previously neglected class of 21 template-switching events. These events were presumably generated by quasi-palindrome to palindrome correction, as well as palindrome elongation. The formation of these events is best explained by folding back of the stalled nascent strand and resumption of DNA synthesis using the same nascent strand as a template. Evidence of quasi-palindrome to palindrome correction that could be generated by template switching appears also in yeast genome evolution. Out of the 455 events, 55 events appeared in multiple isolates; further analysis indicates that these loci are mutational hotspots. Since Rad27 acts on the lagging strand when the leading strand should not contain any gaps, we propose a mechanism favoring intramolecular strand switching over an intermolecular mechanism. We note that our results open new ways of understanding template switching that occurs during genome instability and evolution.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3775" />
<meta name="citation_id" content="7/11/3775" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3775" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3775.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3775.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3775.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300262" />
<meta name="citation_pmid" content="28974572" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3775" />
<meta name="citation_lastpage" content="3787" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sumita Omer" />
<meta name="citation_author_institution" content="Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel" />
<meta name="citation_author_institution" content="Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel" />
<meta name="citation_author" content="Bar Lavi" />
<meta name="citation_author_institution" content="Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel" />
<meta name="citation_author_institution" content="Department of Cell Research and Immunology, George S. Wise Faculty of Life Sciences, Tel Aviv University, 69978, Israel" />
<meta name="citation_author" content="Piotr A. Mieczkowski" />
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Shay Covo" />
<meta name="citation_author_institution" content="Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel" />
<meta name="citation_author_email" content="shay.covo@mail.huji.ac.il" />
<meta name="citation_author_email" content="einatco@openu.ac.il" />
<meta name="citation_author" content="Einat Hazkani-Covo" />
<meta name="citation_author_institution" content="Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel" />
<meta name="citation_author_email" content="shay.covo@mail.huji.ac.il" />
<meta name="citation_author_email" content="einatco@openu.ac.il" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. H. Bae;citation_author=Y. S. Seo;citation_title=Characterization of the enzymatic properties of the yeast Dna2 helicase/endonuclease suggests a new model for Okazaki fragment processing.;citation_volume=275;citation_year=2000;citation_pmid=10984490;citation_doi=10.1074/jbc.M006513200" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=S. H. Bae;citation_author=E. Choi;citation_author=K. H. Lee;citation_author=J. S. Park;citation_author=S. H. Lee;citation_title=Dna2 of Saccharomyces cerevisiae Possesses a Single-stranded DNA-specific Endonuclease Activity That Is Able to Act on Double-stranded DNA in the Presence of ATP;citation_pages=26880-26890;citation_volume=273;citation_year=1998;citation_issue=41;citation_pmid=9756935;citation_doi=10.1074/jbc.273.41.26880" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. H. Bae;citation_author=K. H. Bae;citation_author=J. A. Kim;citation_author=Y. S. Seo;citation_title=RPA governs endonuclease switching during processing of Okazaki fragments in eukaryotes.;citation_pages=456-461;citation_volume=412;citation_year=2001;citation_issue=6845;citation_pmid=11473323;citation_doi=10.1038/35086609" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=R. A. Bambara;citation_author=R. S. Murante;citation_author=L. A. Henricksen;citation_title=Enzymes and Reactions at the Eukaryotic DNA Replication Fork;citation_pages=4647-4650;citation_volume=272;citation_year=1997;citation_issue=8;citation_pmid=9081985;citation_doi=10.1074/jbc.272.8.4647" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. Benson;citation_title=Tandem repeats finder: a program to analyze DNA sequences.;citation_volume=27;citation_year=1999;citation_pmid=9862982;citation_doi=10.1093/nar/27.2.573" />
<meta name="citation_reference" content="citation_journal_title=Frontiers in bioscience :  a journal and virtual library;citation_journal_abbrev=Front Biosci;citation_author=J. J. Bissler;citation_title=DNA inverted repeats and human disease.;citation_pages=d408-d418;citation_volume=3;citation_year=1998;citation_pmid=9516381" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. J. Brewer;citation_author=C. Payen;citation_author=M. K. Raghuraman;citation_author=M. J. Dunham;citation_title=Origin-dependent inverted-repeat amplification: a replication-based model for generating palindromic amplicons.;citation_pages=e1002016-e1002016;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21437266;citation_doi=10.1371/journal.pgen.1002016" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Brudno;citation_author=C. B. Do;citation_author=G. M. Cooper;citation_author=M. F. Kim;citation_author=E. Davydov;citation_title=LAGAN and multi-LAGAN: efficient tools for large-scale multiple alignment of genomic DNA.;citation_volume=13;citation_year=2003;citation_pmid=12654723;citation_doi=10.1101/gr.926603" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=P. M. J. Burgers;citation_title=Polymerase dynamics at the eukaryotic DNA replication fork.;citation_volume=284;citation_year=2009;citation_pmid=18835809;citation_doi=10.1074/jbc.R800062200" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Biochem.;citation_author=P. M. J. Burgers;citation_author=T. A. Kunkel;citation_title=Eukaryotic DNA replication fork.;citation_pages=417-438;citation_volume=86;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Butler;citation_author=L. E. Yasuda;citation_author=M. C. Yao;citation_title=Induction of large DNA palindrome formation in yeast: implications for gene amplification and genome stability in eukaryotes.;citation_pages=1115-1122;citation_volume=87;citation_year=1996;citation_issue=6;citation_pmid=8978615;citation_doi=10.1016/S0092-8674(00)81805-X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Bzymek;citation_author=S. T. Lovett;citation_title=Evidence for Two Mechanisms of Palindrome-Stimulated Deletion in Escherichia coli: Single-Strand Annealing and Replication Slipped Mispairing;citation_pages=527-540;citation_volume=158;citation_year=2001;citation_issue=2;citation_pmid=11404319" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=G. Frank;citation_author=J. Qiu;citation_author=M. Somsouk;citation_author=Y. Weng;citation_author=L. Somsouk;citation_title=Partial Functional Deficiency of E160D Flap Endonuclease-1 Mutantin Vitro and in Vivo Is Due to Defective Cleavage of DNA Substrates;citation_pages=33064-33072;citation_volume=273;citation_year=1998;citation_issue=49;citation_pmid=9830061;citation_doi=10.1074/jbc.273.49.33064" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Gouy;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building.;citation_volume=27;citation_year=2010;citation_pmid=19854763;citation_doi=10.1093/molbev/msp259" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Guenthoer;citation_author=S. J. Diede;citation_author=H. Tanaka;citation_author=X. Chai;citation_author=L. Hsu;citation_title=Assessment of palindromes as platforms for DNA amplification in breast cancer.;citation_volume=22;citation_year=2012;citation_pmid=21752925;citation_doi=10.1101/gr.117226.110" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. J. Harrington;citation_author=M. R. Lieber;citation_title=Functional domains within FEN-1 and RAD2 define a family of structure-specific endonucleases: implications for nucleotide excision repair.;citation_pages=1344-1355;citation_volume=8;citation_year=1994;citation_issue=11;citation_pmid=7926735;citation_doi=10.1101/gad.8.11.1344" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. H. Jin;citation_author=R. Obert;citation_author=P. M. Burgers;citation_author=T. A. Kunkel;citation_author=M. A. Resnick;citation_title=The 3′→5′ exonuclease of DNA polymerase delta can substitute for the 5′ flap endonuclease Rad27/Fen1 in processing Okazaki fragments and preventing genome instability.;citation_volume=98;citation_year=2001;citation_pmid=11309502;citation_doi=10.1073/pnas.091095198" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. E. Johnson;citation_author=G. K. Kovvali;citation_author=L. Prakash;citation_author=S. Prakash;citation_title=Requirement of the yeast RTH1 5' to 3' exonuclease for the stability of simple repetitive DNA;citation_pages=238-240;citation_volume=269;citation_year=1995;citation_issue=5221;citation_pmid=7618086;citation_doi=10.1126/science.7618086" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. Jurka;citation_title=Repbase update: a database and an electronic journal of repetitive elements.;citation_pages=418-420;citation_volume=16;citation_year=2000;citation_issue=9;citation_pmid=10973072;citation_doi=10.1016/S0168-9525(00)02093-X" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=K. Katoh;citation_author=G. Asimenos;citation_author=H. Toh;citation_title=Multiple alignment of DNA sequences with MAFFT.;citation_pages=39-64;citation_volume=537;citation_year=2009;citation_pmid=19378139;citation_doi=10.1007/978-1-59745-251-9_3" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. Kim;citation_author=J. E. Cho;citation_author=Y. C. Li;citation_author=S. Jinks-Robertson;citation_title=RNA:DNA hybrids initiate quasi-palindrome-associated mutations in highly transcribed yeast DNA.;citation_pages=e1003924;citation_volume=9;citation_year=2013;citation_pmid=24244191;citation_doi=10.1371/journal.pgen.1003924" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. J. Kokoska;citation_author=L. Stefanovic;citation_author=H. T. Tran;citation_author=M. A. Resnick;citation_author=D. A. Gordenin;citation_title=Destabilization of Yeast Micro- and Minisatellite DNA Sequences by Mutations Affecting a Nuclease Involved in Okazaki Fragment Processing (rad27) and DNA Polymerase delta  (pol3-t);citation_pages=2779-2788;citation_volume=18;citation_year=1998;citation_issue=5;citation_pmid=9566897;citation_doi=10.1128/MCB.18.5.2779" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=T. A. Kunkel;citation_author=P. M. Burgers;citation_title=Dividing the workload at a eukaryotic replication fork.;citation_pages=521-527;citation_volume=18;citation_year=2008;citation_issue=11;citation_pmid=18824354;citation_doi=10.1016/j.tcb.2008.08.005" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Lee;citation_author=C. M. Carvalho;citation_author=J. R. Lupski;citation_title=A DNA replication mechanism for generating nonrecurrent rearrangements associated with genomic disorders.;citation_pages=1235-1247;citation_volume=131;citation_year=2007;citation_issue=7;citation_pmid=18160035;citation_doi=10.1016/j.cell.2007.11.037" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=B. Lisnić;citation_author=I. K. Svetec;citation_author=H. Sarić;citation_author=I. Nikolić;citation_author=Z. Zgaga;citation_title=Palindrome content of the yeast Saccharomyces cerevisiae genome.;citation_pages=289-297;citation_volume=47;citation_year=2005;citation_issue=5;citation_pmid=15776233;citation_doi=10.1007/s00294-005-0573-5" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=S. T. Lovett;citation_title=Encoded errors: mutations and rearrangements mediated by misalignment at repetitive DNA sequences.;citation_pages=1243-1253;citation_volume=52;citation_year=2004;citation_issue=5;citation_pmid=15165229;citation_doi=10.1111/j.1365-2958.2004.04076.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast.;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=T. Masuda-Sasa;citation_author=O. Imamura;citation_author=J. L. Campbell;citation_title=Biochemical analysis of human Dna2.;citation_pages=1865-1875;citation_volume=34;citation_year=2006;citation_issue=6;citation_pmid=16595800;citation_doi=10.1093/nar/gkl070" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=V. Narayanan;citation_author=P. A. Mieczkowski;citation_author=H. M. Kim;citation_author=T. D. Petes;citation_author=K. S. Lobachev;citation_title=The pattern of gene amplification is determined by the chromosomal location of hairpin-capped breaks.;citation_pages=1283-1296;citation_volume=125;citation_year=2006;citation_issue=7;citation_pmid=16814715;citation_doi=10.1016/j.cell.2006.04.042" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. A. Nick McElhinny;citation_author=D. A. Gordenin;citation_author=C. M. Stith;citation_author=P. M. Burgers;citation_author=T. A. Kunkel;citation_title=Division of labor at the eukaryotic replication fork.;citation_pages=137-144;citation_volume=30;citation_year=2008;citation_issue=2;citation_pmid=18439893;citation_doi=10.1016/j.molcel.2008.02.022" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. T. Nishant;citation_author=W. Wei;citation_author=E. Mancera;citation_author=J. L. Argueso;citation_author=A. Schlattl;citation_title=The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.;citation_pages=e1001109;citation_volume=6;citation_year=2010;citation_pmid=20838597;citation_doi=10.1371/journal.pgen.1001109" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. J. Rattray;citation_author=B. K. Shafer;citation_author=B. Neelam;citation_author=J. N. Strathern;citation_title=A mechanism of palindromic gene amplification in Saccharomyces cerevisiae;citation_pages=1390-1399;citation_volume=19;citation_year=2005;citation_issue=11;citation_pmid=15937224;citation_doi=10.1101/gad.1315805" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. S. Reagan;citation_author=C. Pittenger;citation_author=W. Siede;citation_author=E. C. Friedberg;citation_title=Characterization of a mutant strain of Saccharomyces cerevisiae with a deletion of the RAD27 gene, a structural homolog of the RAD2 nucleotide excision repair gene.;citation_pages=364-371;citation_volume=177;citation_year=1995;citation_issue=2;citation_pmid=7814325;citation_doi=10.1128/jb.177.2.364-371.1995" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. Rice;citation_author=I. Longden;citation_author=A. Bleasby;citation_title=EMBOSS: the European Molecular Biology Open Software Suite.;citation_pages=276-277;citation_volume=16;citation_year=2000;citation_issue=6;citation_pmid=10827456;citation_doi=10.1016/S0168-9525(00)02024-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. S. Ripley;citation_title=Model for the participation of quasi-palindromic DNA sequences in frameshift mutation.;citation_pages=4128-4132;citation_volume=79;citation_year=1982;citation_issue=13;citation_pmid=7051004;citation_doi=10.1073/pnas.79.13.4128" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=W. A. Rosche;citation_author=L. S. Ripley;citation_author=R. R. Sinden;citation_title=Primer-template misalignments during leading strand DNA synthesis account for the most frequent spontaneous mutations in a quasipalindromic region in Escherichia coli.;citation_pages=633-646;citation_volume=284;citation_year=1998;citation_issue=3;citation_pmid=9826504;citation_doi=10.1006/jmbi.1998.2193" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Serero;citation_author=C. Jubin;citation_author=S. Loeillet;citation_author=P. Legoix-Né;citation_author=A. G. Nicolas;citation_title=Mutational landscape of yeast mutator strains.;citation_volume=111;citation_year=2014;citation_pmid=24449905;citation_doi=10.1073/pnas.1314423111" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=B. Shen;citation_author=P. Singh;citation_author=R. Liu;citation_author=J. Qiu;citation_author=L. Zheng;citation_title=Multiple but dissectible functions of FEN-1 nucleases in nucleic acid processing, genome stability and diseases.;citation_pages=717-729;citation_volume=27;citation_year=2005;citation_issue=7;citation_pmid=15954100;citation_doi=10.1002/bies.20255" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. T. Simpson;citation_author=K. Wong;citation_author=S. D. Jackman;citation_author=J. E. Schein;citation_author=S. J. Jones;citation_title=ABySS: a parallel assembler for short read sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19251739;citation_doi=10.1101/gr.089532.108" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. M. Stith;citation_author=J. Sterling;citation_author=M. A. Resnick;citation_author=D. A. Gordenin;citation_author=P. M. Burgers;citation_title=Flexibility of eukaryotic Okazaki fragment maturation through regulated strand displacement synthesis.;citation_volume=283;citation_year=2008;citation_pmid=18927077;citation_doi=10.1074/jbc.M806668200" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Strand;citation_author=M. C. Earley;citation_author=G. F. Crouse;citation_author=T. D. Petes;citation_title=Mutations in the MSH3 gene preferentially lead to deletions within tracts of simple repetitive DNA in Saccharomyces cerevisiae.;citation_pages=10418-10421;citation_volume=92;citation_year=1995;citation_issue=22;citation_pmid=7479796;citation_doi=10.1073/pnas.92.22.10418" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Cancer;citation_journal_abbrev=Nat Rev Cancer;citation_author=H. Tanaka;citation_author=M. C. Yao;citation_title=Palindromic gene amplification--an evolutionarily conserved role for DNA inverted repeats in the genome.;citation_pages=216-224;citation_volume=9;citation_year=2009;citation_issue=3;citation_pmid=19212324;citation_doi=10.1038/nrc2591" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Tanaka;citation_author=S. J. Tapscott;citation_author=B. J. Trask;citation_author=M. C. Yao;citation_title=Short inverted repeats initiate gene amplification through the formation of a large DNA palindrome in mammalian cells.;citation_volume=99;citation_year=2002;citation_pmid=12060719;citation_doi=10.1073/pnas.132275999" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Tanaka;citation_author=D. A. Bergstrom;citation_author=M. C. Yao;citation_author=S. J. Tapscott;citation_title=Widespread and nonrandom distribution of DNA palindromes in cancer cells provides a structural platform for subsequent gene amplification.;citation_pages=320-327;citation_volume=37;citation_year=2005;citation_issue=3;citation_pmid=15711546;citation_doi=10.1038/ng1515" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=H. Tanaka;citation_author=Y. Cao;citation_author=D. A. Bergstrom;citation_author=C. Kooperberg;citation_author=S. J. Tapscott;citation_title=Intrastrand annealing leads to the formation of a large DNA palindrome and determines the boundaries of genomic amplification in human cancer.;citation_volume=27;citation_year=2007;citation_pmid=17242211;citation_doi=10.1128/MCB.01313-06" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. X. Tishkoff;citation_author=N. Filosi;citation_author=G. M. Gaida;citation_author=R. D. Kolodner;citation_title=A novel mutation avoidance mechanism dependent on S. cerevisiae RAD27 is distinct from DNA mismatch repair.;citation_pages=253-263;citation_volume=88;citation_year=1997;citation_issue=2;citation_pmid=9008166;citation_doi=10.1016/S0092-8674(00)81846-2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. J. Turchi;citation_author=R. A. Bambara;citation_title=Completion of mammalian lagging strand DNA replication using purified proteins;citation_pages=15136-15141;citation_volume=268;citation_year=1993;citation_issue=20;citation_pmid=8392066" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. J. Turchi;citation_author=L. Huang;citation_author=R. S. Murante;citation_author=Y. Kim;citation_author=R. A. Bambara;citation_title=Enzymatic completion of mammalian lagging-strand DNA replication.;citation_pages=9803-9807;citation_volume=91;citation_year=1994;citation_issue=21;citation_pmid=7524089;citation_doi=10.1073/pnas.91.21.9803" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Viswanathan;citation_author=J. J. Lacirignola;citation_author=R. L. Hurley;citation_author=S. T. Lovett;citation_title=A novel mutational hotspot in a natural quasipalindrome in Escherichia coli.;citation_pages=553-564;citation_volume=302;citation_year=2000;citation_issue=3;citation_pmid=10986118;citation_doi=10.1006/jmbi.2000.4088" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Waga;citation_author=B. Stillman;citation_title=Anatomy of a DNA replication fork revealed by reconstitution of SV40 DNA replication in vitro.;citation_pages=207-212;citation_volume=369;citation_year=1994;citation_issue=6477;citation_pmid=7910375;citation_doi=10.1038/369207a0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Wei;citation_author=J. H. McCusker;citation_author=R. W. Hyman;citation_author=T. Jones;citation_author=Y. Ning;citation_title=Genome sequencing and comparative analysis of Saccharomyces cerevisiae strain YJM789.;citation_volume=104;citation_year=2007;citation_pmid=17652520;citation_doi=10.1073/pnas.0701291104" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=T. Weinert;citation_author=S. Kaochar;citation_author=H. Jones;citation_author=A. Paek;citation_author=A. J. Clark;citation_title=The replication fork's five degrees of freedom, their failure and genome rearrangements.;citation_pages=778-784;citation_volume=21;citation_year=2009;citation_issue=6;citation_pmid=19913398;citation_doi=10.1016/j.ceb.2009.10.004" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Cell Biol.;citation_author=L. Zheng;citation_author=B. Shen;citation_title=Okazaki fragment maturation: nucleases take centre stage.;citation_volume=3;citation_year=2011;citation_pmid=21278448;citation_doi=10.1093/jmcb/mjq048" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="Okazaki fragments that are formed during lagging strand DNA synthesis include an initiating primer consisting of both RNA and DNA. The RNA fragment must be removed before the fragments are joined. In Saccharomyces cerevisiae , a key player in this process is the structure-specific flap endonuclease, [Rad27p][1] (human homolog FEN1). To obtain a genomic view of the mutational consequence of loss of [RAD27][1] , a S. cerevisiae [rad27][1] Δ strain was subcultured for 25 generations and sequenced using Illumina paired-end sequencing. Out of the 455 changes observed in 10 colonies isolated the two most common types of events were insertions or deletions (INDELs) in simple sequence repeats (SSRs) and INDELs mediated by short direct repeats. Surprisingly, we also detected a previously neglected class of 21 template-switching events. These events were presumably generated by quasi-palindrome to palindrome correction, as well as palindrome elongation. The formation of these events is best explained by folding back of the stalled nascent strand and resumption of DNA synthesis using the same nascent strand as a template. Evidence of quasi-palindrome to palindrome correction that could be generated by template switching appears also in yeast genome evolution. Out of the 455 events, 55 events appeared in multiple isolates; further analysis indicates that these loci are mutational hotspots. Since [Rad27][1] acts on the lagging strand when the leading strand should not contain any gaps, we propose a mechanism favoring intramolecular strand switching over an intermolecular mechanism. We note that our results open new ways of understanding template switching that occurs during genome instability and evolution.

 [1]: http://www.yeastgenome.org/locus/S000001596/overview" />
<meta name="og-title" property="og:title" content="Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3775" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Okazaki fragments that are formed during lagging strand DNA synthesis include an initiating primer consisting of both RNA and DNA. The RNA fragment must be removed before the fragments are joined. In Saccharomyces cerevisiae , a key player in this process is the structure-specific flap endonuclease, [Rad27p][1] (human homolog FEN1). To obtain a genomic view of the mutational consequence of loss of [RAD27][1] , a S. cerevisiae [rad27][1] Δ strain was subcultured for 25 generations and sequenced using Illumina paired-end sequencing. Out of the 455 changes observed in 10 colonies isolated the two most common types of events were insertions or deletions (INDELs) in simple sequence repeats (SSRs) and INDELs mediated by short direct repeats. Surprisingly, we also detected a previously neglected class of 21 template-switching events. These events were presumably generated by quasi-palindrome to palindrome correction, as well as palindrome elongation. The formation of these events is best explained by folding back of the stalled nascent strand and resumption of DNA synthesis using the same nascent strand as a template. Evidence of quasi-palindrome to palindrome correction that could be generated by template switching appears also in yeast genome evolution. Out of the 455 events, 55 events appeared in multiple isolates; further analysis indicates that these loci are mutational hotspots. Since [Rad27][1] acts on the lagging strand when the leading strand should not contain any gaps, we propose a mechanism favoring intramolecular strand switching over an intermolecular mechanism. We note that our results open new ways of understanding template switching that occurs during genome instability and evolution.

 [1]: http://www.yeastgenome.org/locus/S000001596/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VEm-pSs0EtWJE7VaQgsPHTnMhk7NdlAHV5SZ4Tno9dI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487100","apath":"\/ggg\/7\/11\/3775.atom","pisa":"ggg;7\/11\/3775","ac":{"\/ggg\/7\/11\/3775.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3775.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3775"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3775"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3775":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SBb3ezKM01IzVkn4PWxyUlWMNGLqtJwLa7lPnn27fbgl\\\\\\\/jIQESCqS17anfgWd1hCzQbOcA00Cr2n5LBghfS4DRkHgY5nCNpJ8gsvuHkRqYABcdpNLleul7Kqt+Ei\\\\\\\/0tTL\\\\\\\/\\\\\\\/6pWuVNxxzf9TDpj7\\\\\\\/R8Yv6Com2hfshRgkjvC1m19ANxy7wXKSnRQkeBWXwOEzwqDN79Lq+QP9U5qTpHPyOD4RqwShMtmZ+6zubKPwy+KmTVwZZkYsp3eRrs8SrRKwVxgs5AzL+GVjCRQYk0t+sVmRMVfuKiO3441rSX2tqlVCOSv8shVYq80iY\\\\\\\/vuIONHN01ergrVmRl9HB7CqWmQXAFZdzd35TzQvPFlyKt+AU2x1Ef50oAZZMGA2LlbmxZ\\\\\\\/\\\\\\\/8+VAuNncQUjJiiiQnvyhpvUWaedhxK0O68STSKQVqxRivq0yNpme9B7+aMT6fuOHUD+2y\\\\\\\/vvEJYCtphHR9uk\\\\\\\/g3VKNT+NWUACzTOjfoY68+cXA7a\\\\\\\/WeoIFIcvF4GhmSwCPJERVN34GX+2y8MYxfis8BoHB5NIRFId3ihQxPOPbpiEYvINO2\\\\\\\/UB5Bn8geD5rQ\\\\\\\/HFAsYI3JXmGgFVQ9lKC6txfgedbNccUUBSeK728By8DTMRGA7LhT30Qz3n66GnFHhuFBQrrarMCRIz41G3v4q4QCxGTAtcGLsUdv2E+Bmjn9TGRbKMhHxg27XTz5fBMikgpZiftYoFz9bBo+lo6Bcy9vMK7TXHbG8\\\\\\\/o1J7kwY0jspGhiaCtuqpwxpUqAykq\\\\\\\/QDNZEZzjB740NVvcqTTCzasf6\\\\\\\/R0QitXtQ7cB6o7SDKKQWtXN\\\\\\\/GO3If++Tnj9mcV3CVbNxzng0fFKVczhPcOzivhLB0eSqJY4tDPSvjv6tix2ldsFr4UDWJs9PPmkix02Ht6KzAFsvAdOtKinBEOzLsTz0xxiOJ3RhPvvjgEXe6FeiCwjIeJA3M31Sd4xPS2DKyUbm0oxWmVOFj5rIp6x3uoc50uXiWkAUgKKlG5\\\\\\\/LwtSuJhUND8N6l2T4dBKnnOAiS\\\\\\\/mLS89vcwlIj+p2Rr8havnGLHh7jqIJQ7RQr+LepBt4TogWWWhlP1s9pCoQs7rYL7d0IfZ9MUHohXGC4j+\\\\\\\/nr7vhp4x7CLHjessHZU4BfOWRSY1bZhArnae8IiSiEU0SDlT7VaWBOYlF\\\\\\\/oNNwUCQ\\\\\\\/ew7hynW3w6yAs3Im6pDoB\\\\\\\/ipyzU04xAG11J9uv+rBTTPU+xUia6TLPXQsvteVfygKga54v6PgnxisOtRNLx5KdQSV42eiccxlxFzpM\\\u0022,\\\u0022iv\\\u0022:\\\u0022mmDXSh\\\\\\\/9L0Ns+NlvJRYEPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e1de5866a334c9e9c66c2efb78d5138a1b6040083e1fd72ec0e806af321162a2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oPvgcZBmDcd5hfyyD4dXJPlT5+SVNz2xLMDTMNDW82ZbeS8UiTONLNsdtTJd8bW1lcEuBA8lETfpLzvoD\\\\\\\/Bx1dubR+TxrFFb+lY0zH1sAncLmo\\\\\\\/H5NN\\\\\\\/QQVYHP7IAJuSds7\\\\\\\/wjjYXAOpMG\\\\\\\/\\\\\\\/HyO81IBeq5mOoKAj3aIIlK5deJ52XTBBJ7L8eFNoipAqYesP9q13B60RTE1o0Q2JXk1BI3lk81dKGtBpcaAMGMVNfmnEC5uNkgi47y78ChvLt267ueo\\\\\\\/fbcAmihsxq85uef47EA36GVIory+ES3AwqH6SK8\\\\\\\/JZBWu\\\\\\\/gdzs\\\\\\\/6PwuSSFCiDBWBPKwEgW3Bokb88p1Ethvn72wraMptopQbKQXbusK\\\\\\\/HrXKWmby1t8gxEyT5aknKikg33w5dbjXDqZMq6IYaqvM933fJ8TqHTWZQhopsSmMOnhrmlV+Q6v9xjXMTyujBkhZMOnODvj1AYRgNm3fRw\\\\\\\/sM+VreApStrIR3XtaZA7d6JLoiybHZx85NFER5YJmUE+sKC9aAIzP38M7nk6Zi7XheRIsnDUHPWTlmhUrECapr10mfDAL4yqedhOvWIhvN9gb7Sh5rYdgxSGeODX8l2C2qyevSLZneRfdZTi2zxYN5Px0I3r8xjJZQNDc+18UwpVEZwo0PrjMERvQnJJQ\\\\\\\/rKjfcRkcfJtJHu8HrTUtQ+IrFkmhOSA3oQV8sJ+5Andt1kS1fzFWJMWkbPQ3hqPiE5aVm\\\\\\\/2sPPeK25sKEZO9E65XtIerZ2d8ZOj5Qa7DvTAOZWhgyElF4PaR4dQ17Q2XxVE2T8q21VAvA9Z+ShGpGN8TZKWv15jeeyl8jFzQhjrMAu61MCQhlw2YFaXFkYvxFChdArHmXi17cVQ7LtKD9JRVdWujSVbmlCsDD2eJWih9ztoIH6PtMm+gnU70bhbNmJe0lVuijrK7ALqVo+xS0TKFGePnPjnKoWk\\\\\\\/W60MYeJG8ell6Kj3buSNEfAjudOfSPYiKYF6Km0zzF1eBCr\\\\\\\/ZlAZ7oAQ1vNJmqOD+teu6wGflzgGaVu+l1OtXXRB7mkcg+kROsBsccUxZJSseLEPa7dtsKCUgGDDCZCvCHisKJoSgS1BoRLhs4tauk+WIbz3UMpsbunvOco6YHcG8M7ipv3+Jd6BYC5NTN2whHY9a5qTwwByt5zycveFQ+taLJmDOlhk9ycr9XHg8Jyu1HvcYoQCsoz5Zy9qXguGOESI\\\\\\\/mZCW9AuW+lIN5f3zOriZhYV0B44DtH8VT4TjmFl0y4pc58LOjA1HFPq\\\\\\\/kSz7CCx1gWSIGjrH\\\\\\\/fsKYblGLkNUrxrqVY56u8fGjB1efUmR9owvS8Wj6tl4glzAhqUmk3eFY8M2fSqNvGUadsnjdX0ANQRWqUYOAT+usSlQEG+3B8CFZJhrIPCpxLhWgcmb\\\\\\\/ub\\\\\\\/cUtuKILSQaZTW7AJ6cH2vFHfZSGG52oJhWujw79Q3mm\\\\\\\/xivyz+nakeCCY8j7PeCtrgc3Gce0\\\\\\\/qxg0vczH3r6gjVLE8n80FQirAs8jvU9diz0jN79WeV7R5yAx4F7L+LD1d1X7H5MxiQNGEulAWyKsnI66vRSBTlTWi396TYAovmrE2KU8sMxanAvxXXeAZNhykchr86E6CiyAjLjAGAgFeg9JiFPY9ZCWJthm5okp7MPuMakIuv2sdjc2k9b0kU76pFHhh9vHa3lCI6ZfIBntC+GteLRJf6q4jg1UiHow+fFLPQ9P2rwLENFac430cbi2xtagtnxbUfofhwqG+fNk++iqqD9ulAkRXxd3zaWfCCWKRHyLfRyO5BeHUzULfViXh8shb0LM2CG3msW9yD1Xs\\\\\\\/5\\\\\\\/JRut0SWMa9R9j\\\\\\\/aXzhV9X54yI\\\\\\\/f1aXgaEU72mP141fMyQo+2mtDOkCDG7nRhmrtavh2ONgNDfgvWQSqWMbJ5bmvhQJLc\\\\\\\/Ek47qM4mBcdK+e8ZgsBXBdTzH2AwTCWMhn3EJ1Mv50Gts71fYysZyA8IQ9ksJ0u2f9VIV\\\\\\\/QRAjW+\\\\\\\/kp2swf6YknalqHA9No3sm8mDVp9swfh3QGCFcNYOtNn3nrhc1gAYjtP2aVL6dvludzmGn5jyJXlEojaF4bWLqFmIcYz8pW7P60LHbfKLgP\\\\\\\/+IJ8kBceJYwz4OFVTeU8esYi2l42qOnU0m5SWpYSNWnnnzwbgRKSaGqTyKz3hnbpLHT56hWK5A+7BToFztYk+VA1LVG+exdoLHBWJy0RxeXFrEv6TjS85ROQRCdrZxC\\\\\\\/Am+wD+RbNeHjejeAp+\\\\\\\/nUxRh6VkKSd2YDSM64pRCU\\\\\\\/hMHGABU0tB2TRd1Vqiwfp3W\\\\\\\/QFiZV145KoN4+Co37rQ1WaN60sPbp3znKJXRrcozaorWyot6ibexHH48jmvf3GchDdIGckYo\\\\\\\/LNq81P19ou09FOrjZcLdz\\\\\\\/JT79yqnEcxM5YTILXkRvBnxxGvq1issCHPKtaxRHb1VYWXJ5u9LRUhqakezQrqC3G42OgWmYE0BTOXRgNiuOLcshU8UDscNF4o9db5uBpjjAh6aLxc8RVFt6n6Bt4FSSJAeUEQ5Ebl18gQYRHErAECaRIfVFQW\\\\\\\/qAwcc7oeZtM7o+b+5SNSBvVbcU0thQNTtPcwRpeMcP6G0CVA9p0eI931b12TowSaBrZxwvNC351WDFc86UKcNE4kefITjNmG9QhUTrIFnhjpUltgd3LtWE2MwrwVXRnpd050rPEiwF10z+xP91DmAYlOdXXQuTYqNespG0SnHmamyP89UyzyWJ709SaW43dIA7gGnO4k361jRq\\\\\\\/brf0U7hKkSdIIBVMeFGI6sD7R0mCUGTNH1dy2PUU5vQ2n5aKt9sIs7Mts2qPvCQh0DO6730ectLOL9KU+UFkvH6eqVnDKrOUzG9wy8X4ays9Ma+HOw\\\\\\\/DpJ0iF3FZQDFCiWJCXaj1KIzcPhL1QPw\\\\\\\/rvXNDaRcIktkCkLORdJxbHYrow7Zpj7WtvZmVMOcKWIYHhkeDLWb72NMwnHY9XpslSx0dbU9ipNLCTFME\\\\\\\/YefcpNz37aDYnzizjpvGbFAE3sniriZg3A8ULH2swYZvbU4esDJSpS\\\\\\\/uv73vrLMTYHSk9JuZfQzW8W3KDa4Hizi0iLJVzErfbViIQEzBbp52VN4tWqESsCKhpQZOkE9QIy1MHZdu47b2JPpfWuUwh6e6pPzdui2olSR6\\\\\\\/pgf3gcs1bb4MDa8Jbsr1+nOJUqqckJYMmHOfxLZ68Aiy72CZKZmgkSjQYabgJPPu5ihKG0gH+qD6k\\\\\\\/03wzp\\\\\\\/9njxf5hRNmCbkx6cISpvuommJcWl65jvzH2tG9i49rAX4Ueenl\\\\\\\/1aetDG63LoVuWFb7oZ5mOX1aqvYxkbCugzOIr7\\\\\\\/UYk6WEdkwnmNA9YYtKtmbwZRA0g4A5v01pHoB+eyiRJ8CilEzQMPrGlyIhHIg7LlG92zoMIeBPvWK1IjBmx5Eg0tbN3clsGhnF\\\\\\\/JsAbWiT2KhnVPS64yyAr4sTn0q5IG3Ov\\\\\\\/h5myZ7f6Tycze0R\\\\\\\/7Zj+5ygiWxCxAIc6bQ0MejcapZdD05Au6871gtFgj6XHQLeKMMJ6wK0GiXp7WXUexsk5bbY0yWYTOML+OThwuXGaDkXShvjkDR78QwfveD+cHg8RtSkCcZZeJIkjMhK8WLsQKaM+83Ps\\\\\\\/4nuV8i9l3wD9hSXV2Eb2GJB8GfpNsy7EqVgIgJtDWPmVepkBBR\\\\\\\/D9jQ1pOUbT38xsjO1CMzK1J1b9n5wxyxLe0JquHSWNQbaVGi9S2HMzddxEeuLd8yFDjU0YG+997v\\\\\\\/EY5f143p1OukNr\\\\\\\/cUZS9oNIRYGlknk2voIeMeZoNqasuHU1NMKrfv6b\\\\\\\/48\\\\\\\/KrLKULbgtOy7KM81fVWxBALY8rNJ358sNfghr83xEthu8IcBowxIPizion70zE3GD\\\\\\\/kglDVBR5w93vuoNIwMUCfid11r6yxdoalqBHYtLgZQ\\\\\\\/lNo\\\\\\\/udX0WYJOdnovX2YAtLM4NitCC5\\\\\\\/8f83NYhpeoQOP0Q033NY2D8t7bfDSe\\\\\\\/\\\\\\\/kOB+NrCU901EcpsV8wcnNUx\\\\\\\/hSq\\\\\\\/hcuejLo7QoVPujUl2tLQDEPWn2WSaeskXMIznSeqI5+pNfSo6KaXmu2m3ZIR1qdEbj+IL3oJghvEAc3F9q0XDgrMQr18SeG61zBiEQdg+UOKnGUpJhi6jnCjShvQTQMK1bXpK3OpWFAjtE7roMTTUPnuavN\\\\\\\/Fjlg3Cw8opYkaU7jg53O6BiTvqFE3tCjPhHpjHP\\\\\\\/PtJpCeLhx+a3kY0np\\\\\\\/AfhZx5QKnak1ILgRw\\\\\\\/VHMS1oONoWSxCl8721tI9mJtOmR5uoF+T+e1FqDT5A5R6ChXhZ4j\\\\\\\/gntPS3gxda6ZMtSDcggfZWqAOhMnRoGcooYn41Q201OPPL2qdOIkQC3gkE0Y4PjW6W+mv02TbdwTJhe71uotogQv\\\\\\\/1M4RPjdMWDuofxWqwBi7WMZ50zxNCEXyrDEcIdU18xLkCHMg1yjcXyjbG1GrBfjQJ9Ed7Fln1XR3TJC0p8glpEzKRN5hgCXY2\\\\\\\/VPThrdVxruhnIELyVTROD8SzbNtaegb1pNFndLAPEypnaDaIXZI5x50b6RxcZelh0R4vBfmmzP5ITAQrUk0uE5Nlo6pNnd+AIZ8\\\\\\\/taJVOBHC7RI3xg3wGlWddfxfZTMsuIv67Rbc5Jsg9qg+soalPLrXfshjfldSd4ePR1D+Tc95VaqUvBXuUfBggpTqEf7ViCO095+utQlGVCkv43w9SbCo17LqxFFveGXrFdebZdu9PBg8Hfi3xsWFFZxSMqgG2HRX302yqYqqBhTwfyCyjaX59VIH\\\\\\\/KDs7WgBAW1L4F9cOkEl9wkOXnFVvuzdJnJKH6LEZdIzODfHw9X3UeSfNBrgSs0UUeXt84T+zc\\\\\\\/3DWJl0Ookf4Wwxbl5k+qwbGmTfRzhq4B6CiB1dgdFtm4dJCMRNQM\\\\\\\/MmihLtTlPA93C+Nz7B+tTcprCLIyrvc6EIw\\\\\\\/Yl2l\\\\\\\/ez3e1Ixo7LfBYIw2oFfS3P0TVu5a2vyggq1+2LjRWyEPvvejWWwv2kzIhwQYw7Fc39VIjJ0n75SJY1De\\\\\\\/oppI7bPWOT4Wg63eFBHeYhX79O+1ARDzEjjWD4PqTOkaE0JqFQjHhgZZIcKY0kXkJ82c0piWm2YlOqUzY83rMPoMxKBRuRxI7S6E3PzNNavmQfvnecDqa9kuaOgp3HWxRVedbCGckCSoN24fVX\\\\\\\/CzxfIavEgl\\\\\\\/MLPRJCcC4XafI2bJAUsyndDjECcgU\\\\\\\/REdkFeODHMUtPtF0bLJGLaRClMofruJmV90m8ocpt+LG10ydPEdrdPsubi3FGpaXBjctlvOJMs+vKjQfI3FN+tkjn\\\\\\\/\\\\\\\/XBDwLVGmDD7SeL27LLgCAX5o9iaInt3oORseD3\\\\\\\/+GFTHz1PsO1uP1wlncJEvlDsYHx5+INvL7DhAnA8ZAplC3nhvLuyBHo\\\\\\\/4xQMnR2rpMwSAfRuLzxKRYl4dpQFV5cZtBSWpSTAxInLPhPHnrp+XRAXxiyEnJ9nVP+qlQn1sdUZhcez\\\\\\\/mJQfTcgXeNikgn\\\\\\\/r0qNOGNWxFGHYJ6wvWpI4hhisSPaxFyHFuRap6elU2AZtVW24QnuQnJ6mQWm2KWuuQpTaAO+o9xIuxUJeaoE\\\\\\\/s\\\\\\\/ZehVlZnvBoRhVzS\\\\\\\/SP4raRnWE2WAjy2\\\\\\\/vvTZ6svv1WcxsPESJOxVZXi5b2fLV08S28Gl1lmzb13eapUUfLCdYqCqRnnpgBH2hrcjd+rwKsl88gSt+xskUpQ7\\\\\\\/7y4ZDhXuQcaZpMr\\\\\\\/SOfAaZKO18v+n1psXan0+5+de1m\\\\\\\/p5ch1mk\\\\\\\/iC+DHsnVaOwn\\\\\\\/RdscP+cdqGqpQhoF5F+DCRJ1zneLEU9ExUpqC+hWNM4VBdHBLD8UHxuEwTs\\\\\\\/pUCGTIQfXSIVDhHM6yWFVoAS1+YCBIWxsQlNIqfihKuaUEW3VSSovFgw1RkFTzPAr21HMLms64bXgWzhX1A1VZ9k63SsYtdZ2C7xV9FviPXeBNeEqxtPVYjG6Sl+J5c1ue9Lt82Ul8Q6\\\\\\\/BTGFjhK2VHXJx0D51SZcox429f3Q2wX4Q9pkSeDRNUtXAj0NNJvPQCHqA1sg4yQjjpG9ZP2OFrGsUtL0zfFcuAfgV6\\\\\\\/Z\\\\\\\/3qLUpqGAIkiAo\\\\\\\/L64FRs6kAvQtrJg1\\\\\\\/3uBEtsiTPrix1\\\\\\\/Ekdm8AdYawQNH4JQW6W3Fuo+pRVC1sdJRFxvfpI9QVCj75xbHuE3ScjZwZxm4fb4tgWLJlxm4rY7eoO+G9mQZGTPZGGPr7SDD3uBpUsMXla+bHAWxxVfDTVCPnUBJnTt1qp3yXr5tXZp0ZqoYZIGvKevMUv63qBWuLMUCUmqjVa4keHsNTkGGtn+7yA2i0yF11uFT3xm6QB+Z3ac2edl7GnfWFcjlFA2TrxFIQOIPhpBcPrH5O4tIeyrBjeOkvwTxIONoNkOvWFl9nT2OFROKuWWpm3o7eG3rxg82af7K4TWiBWnPBVtgm4OStPfC5hcthH2IxfufavrpkeavT3wY\\\\\\\/KHP2w7roRAfFWoP0HEufnh7NLyoPwRQqamnsw5nEFXQAwdy4AzdxOQ9P42Y6WHnQsGWC\\\\\\\/9AFCTbSakzh6smojsFBcppWj0zp\\\\\\\/ckAvvceAbBxPeszjzY6Hdg6rmGmNFB3TbxHmyo5J5zvPiXhmERSxQJwJ4T2XhXefnv0S6Z1OLN5HBWqYU0h9zuCOHG8MbJ8B+GZ7a5pT\\\\\\\/ZUu4tjCGVRey1I\\\\\\\/DSq\\\\\\\/0wCX31MPc95eFpjQ9aqg4A+pXsTVqZat5ZeU4A8ReJP2dW\\\\\\\/88TQNuKPYhfE5pAQNC7P58ANVtDt1MdYi+XOCt5ylQnREr++WztwWiEwR6Jg+mWqsxtoNypI9dbCcxyIEkdNHd53dM8QQ5QmhGyebX6e48VdS7o0Pnome0+hlArqxEXG99wu3cr4jOXr5TFeZ1Urcegyb7gzfJFy6IdLD6YorTLNOHE\\\\\\\/bSAY8CfVmQ+XD2d36e3iDKmIlRkDjz3lejlwLzKGifsc+\\\\\\\/euqza+Z3xJXSU7IWmPR3BjzywDS9UNTtR0SndLJN3mMVVSQK9mng9ekAb2pHSZFHqE83w23dq2Ij6rQNLgdN0gb7O3jvYBqCll3rhrP1t6cJhbsge4KFbflW8A4+YCRiLbuo1f9TNLy2teowI4AR2dEFf2yPyZpi2kAQ0nQCEEN4SdoN2N9p5K\\\\\\\/RBLRyC1j58+W9hMO31BH8lc6H9lm+DfdX++ZfRz4GCAksMbCREFRD7OYnpdvb9Q9ofiNHI2BR3oeRX\\\\\\\/BYKMkAS2l+DTLWea3vfgaWkDLSrUVcdrWYzTp+L8hxHBsKWyTn1nrS1yluuT9QSshMh9aWpuLXmOuM\\\\\\\/RhQuA+hnK7hVntd\\\\\\\/s6+01MY6tROFyqnm2jsUCX\\\\\\\/pwG8iibIFuCLqCrCmLO\\\\\\\/3zQcXouBsU6ebpzQpAA7sGeVMbIVXKJ9s6gnMzlbj+QSSNJBQPErW9vKXD99ocq6SEaSUCNJ5PQWC7WNAWIaAeDRVQN++y3pRNxTutHJ3i9FSBGRa4M4Y4VhRjO6qrIVbefR7ARXpgaA1ts60jaQZeybnZi6aWlaTnELAgQGycdRuUYE2eHDVMOJF68gt12qy2p7+KmxSewJxwtAIlag0D9MPSqNyCwitbTRRTqWqZA8crPk5YIepKVlHW9o8zkrT9u81sYsppWpWzrxykjufhfiLLf+jCY3YtLg2Wt\\\\\\\/c\\\\\\\/eTwbODdqs8l9+Qa0HwoVt8qF\\\\\\\/qX\\\\\\\/iVCeyt0yzE6d2zFHL+3voLwxliI3lBSsg2jNtevxBIM4\\\\\\\/ViZqb87EkyoCmc98R4zpUEtKkW65z360IAF6IM6FvWVxKWIJmEDN9SatSRcQENUvWXmgTkpLqU0W169Wr2\\\\\\\/5gZz4pYHG4\\\\\\\/ZQi2zXCGnHrnKnQDAPUTf7sJLkUTKVdhIJ+wHlyd2swCj16nMsZ1MCRe0u0jTy\\\\\\\/XU6ltrkAWViP8tkviSfcOI1b5M4sRxui9PseYX\\\\\\\/FgJydObPp+fMa2YjETv2cKdLxo2DURq+pFBSonbSwntQqsmIlteEt6BEqNYnSP8sSm9BfdVWy3db0A+y+xrg2ihcvpyTwHXw0AiRKX8EKjNC2fSNHdYLX7O1D4ocMs2TLXQzEbVK3nK6gmK8T\\\\\\\/tcu1em5fUU3dMPJ+dOhybfYEbKIBRTXDvNxuPHkZgx8d6yuPrH5if6gVUouUy61hYiR1B+H7ft3oU8kXpRo1MmOy\\\\\\\/B\\\\\\\/Ykv9zvOCV8TsuqUBNPhozxXfB551aitYsXvo+mk+tnRW\\\\\\\/DNj2HiUe6wR9qhWm+6y\\\\\\\/IQ1JzuLAEI+qK3eWaH3zAPEhPaxOGAsYlt09dP2R+r7imFdVPoyZP8sE0x7sRD9wPCvPANC6aMR7BzqHulY\\\\\\\/sp77rq3vbFDJbmrkKcMGKs5vZfeXYfcuOeLzpnBZv72tH9I2hYpvLOorxgyRrXffSANxFsNSWBSSIz7cqcCpHkXQyNzH\\\\\\\/lh96otYmUiEAXY8L7DcxAOipOxcOZGwaYEH68I5F4Abulr\\\\\\\/d+KxIwgXXeN2slBeaG1XLYKQDU45h4ziiJAc+EbvIGzHOZVcoEtYsqu1JnIoJ2gvZo9DOzoOAisiIPhl8kdDWZVtFAU96sIzr8w5nVlGKODvd3obbrpxQxJ2PmwDgKpukQteiPBL3A55BxBbkLazoIxdmIcCSSzs5UtOe0w7X+LBeRTXlxCWQx4vJY6PMk1qqt\\\\\\\/B4y7D0AXf8lJus6WfWXhkI4iegAKFztQKLD\\\\\\\/y3ZLfYJFrAU7Hc+msLPH2pkrm5QRcQ8loMm99X3wYZQHHJZe+JoiJxZzjUTnVmvTTwtYxnKrQA+ls8HUBHR4jRtZxDy9QMzNWuIaHqAGckiWONiAWCpcGY6Vj87qRHc2mHSGHMFdepmAj\\\\\\\/5FehvxoPfSzXRMLecBqCkcrTDz1qzjReuPMCLcC+h3dPTFvSk2yN6qrtpwxlYtDNfW1yRmPhvhKUO9nVufhJi4eo2BRBErbTzjGidUToO6kZRdivsCMzIrxtErBGsVzKOWLb\\\\\\\/XC10YVP6nrAv4fuSJhbn7U\\\\\\\/P3xp6iKPT5SA+n2OrqaxM9cqX0TMEw5EOOLlFQot5oCFEYMzFODqb9KG52VV1m7SyahkDNUgdAnimDvtakCnJXQwFILFCo8K9cDS\\\\\\\/0ntQv9rAnmLd9t4xeRdhuAsCWIPsncbvGDqxhLrjeHmZyxi3kEL\\\\\\\/1Pbld5bqucxQzgw\\\\\\\/DF9BY7VZCoLuRhhL0RmpvzxEWqkd5o4NlJGMLhoWlNzzpcO7SF6ejFf+6WR4w7ENoUf4IrUNyI66NtYMwX2s24tgdnGTjJ4Buu+LyBeUjSKA6eeiZtCYOVSIgUFvcagACeXGjsZKRu0Jc76iPHVQzYftnWx0C\\\\\\\/MpMms55dFxZyUll4byRDriPm1snX\\\\\\\/5DRFLQAuETKgae0HG0o4IN6WKWhZrH7gCtA+X5FPIrcPfyaAHOTts0RSderywwrtEUjzEv8C5mMW6zbfJ3tYYgDgqdq6CIq1uhQ462CIZxHYoTth8sW7hgohT41RRLWuNxXGNn0yH5+d+7xig6GWcujg5fL2xNaNNhUlPOqnLsKKyoxp6zFoCd1DTwjJ0fg58aLrZjh+hxCnDN+Nt94UNfQync+RtRWNzmu3E6ZdKQaj6jOgqNyycYtsd+xG4qcQysQXiC192gEePYENY7XtbYlk2M6e2CPi5PPB7SLg8HCVFVFe4TqVcztUtn2MjG3lniz2k4toyB7u9897nZwoTNvpsn54xkkw0hCxAqHaIQnn9cGmZQzTZpYBjPqaG\\\\\\\/HUrzCOGlXeoB7d7PcSqUE8+BH+tnpd1Z5yHivbn8gICl\\\\\\\/PiL4cgeGuoBlVeRrWNqZxLaGV38DWqUYzRbYPsgYW9jNX09YQcx9owV9D6ImnE\\\\\\\/dx0OrQe89utD5UV0YzjWnxKx6QzAqjF5B19m66ctmDjG1xAnsW1Dh5AcsvHDcWtHG6+OPp3w01ffxKUbtPZ0Mr694PYYmbcPsNTWi67WaMApT39sCkXXHbEXp778sefofk1kIu1wJU\\\\\\\/DzJoYpS8E2hq7msctY8\\\\\\\/G\\\\\\\/5fS4I2KnohnVceJVWCF7YnkHLlWvzh0iyYZmF5crir9UsUUz8gcIasl301rSbVTjgSvm7ASigGiRsDDcBWbwC5fbTXpB6XTxEpeG4jcnt56QOLIM6433D0MaElahYBOxcyVMKpLxlddtxSvgvbY12SiqgZvTXkl6DOp98Ckk+uCBNrEtuTfNmDvgrLOKtcTfQyZj\\\\\\\/PZBv7nH9bk4MHf98JvTaahrXyt3+XeQHH9Lg2l+Oo78VwwPd1t6IRxUlVeghdPgeQ7d44pRCBro8PwhelJlE\\\\\\\/m73VTPSAdDvGC6dGD0Fjx9KkX0dmY5lozTZoB7Ll7KcD1v4pcGqyZW7SqhAW5+p04PCQrtnABqlbHMsT2q+X+3mAuF092AOSLndyaTCI7G9UwUyUSGaq+WtXj7Aovl3+A0srOIOvvMyQrRDahvitGrzQ5Tl7HwZk7b\\\\\\\/HWrniA9kyQA1TL2meeKm5mq0OJxjR6svWTXE2pYRM3YWJN2wBYpnSPBWTAS352HcXxktdkjxT9L4NPyAQ0ELeSD9AQwcYCqcZrz8OGbCqqF\\\\\\\/Yc9jlJoXBy862l6YYd8yCkDEdgKpLuWi7L2qNemYKN\\\\\\\/Ev9i3S16QW\\\\\\\/AYOL6McA6R\\\\\\\/XTBFeJnBO3p+pcUZGTEKidrG7xqJk2vjMP84QtC8ickjqbB+ykEiRw6TqEWPZ+9NlNXbAiimjNwWbMkPk5zStA3fqoYHVCo24KFd882TQd1rpawK041m\\\\\\\/mJVMq2ki8DUzGbrwVrOKbFcPK3GOzwSW1Jg453oBTGOgnGxx+hq\\\\\\\/o7UPPZ\\\\\\\/4XgF7JIlPyINEXFqOb5fyzOR003NvgDyA0IbMWJskczfO9Tbiw2hF+pPfU3+t6i3r4V3iNF\\\\\\\/ee40QPJ02j9iiOvQZ2Y5WIbDUF4HDL4m60mWs\\\\\\\/teqeNeqHRt\\\\\\\/6xQWBAIaVcbc0O8IOW1GFsLQq3MjC5qxBB5LA45pcH0LMXtDG3tKe+Yo5bCH\\\\\\\/BUuDTSmXHSydb7mCQHaC0nZV5LLZeC9YVs+grBkIiY2YWkM95z3ykfGTyojTdHN\\\\\\\/TogchDm8YeeEslrkFjTB3bJ2nAv4oQuUnZyf5ZTymbgSL3PKamk5FwXUjmewbFGuq7qbY5t6P9+yiiM+jhtjrcdqJ3ScLTYblztAQp1zRvDrSgEWCi1Bmc7Dcr+xF9QUOoX8b6V9YGPruYoZl2tPMx\\\\\\\/30IuMvWvIyu3EjcAkQXHj9PM4Du3GIzg6xvKgZGaT4fdLA59oG21lMPJtGy7konPJu+sb9EgeBLh\\\\\\\/Zw2L5\\\\\\\/81S5O6R+DXe9orR09urGVe4p44Tk9HnVQeNzeBBxrxDTBaPu5EW\\\\\\\/TTsK0e6p5vIcvNhJhZry1QTQttU4l04N2ZIf5Am6RT3vfNy0KhwmGGMP4zxRvs0Au5jwYQIs3SEkIEXnf0UENHbL7P9nKvtNgBpkL462qc47thEF7ZoaZ5A7ZI0nsAv1nGpyx+cldszSjpLqMRnKPdPiEa7p\\\\\\\/A9dCaElYtoC1unW5rul+geR4kqDZt7fH9h6\\\\\\\/C79xE0+SX6fer0406Nrd3zoxUrDZb+6OfCRaG8\\\\\\\/toKXzFemhAUyIKsu7QulO3soqyT9AjVAND9Of0gE6gP28u5X94KFwI9tDV4xQvK9NyBw751L7gYDTHo6sfIYKSCyzkIBO3Yx2aj5jgXQJTNHzs3Vh6G2vE8BsoCKON1TSgKUx74N33E6t5\\\\\\\/7MNy\\\\\\\/ibSUWEbbyhV2vSNYOaduXLNBiyLOm5BZRTbbOC0yl\\\\\\\/W9ikxLYykC1kqCoGl+hHaynVPwztfDe390Fxieaw89Jfnv4OPVHU2QNcQiLN8ilvAb3BZ6tzgHdwbThQxb3o5lbo9rSVCnkf\\\\\\\/U53oPu2tNBk9Y0qfDy95oJ5OD2L3HoFQBIpfomSVEzraTYr7nUwNkw3rNS6fTMiRdIuNS8Nzqziz0oCyjYA+dBWF5H8kfOzJOIQCKswontX\\\\\\\/hFF3Aan0cro2hGS\\\\\\\/Nado9OdIYp2o5mmtjhGF6mo2LHtzHZn2+O63x0lUj\\\\\\\/UPY\\\\\\\/9W90jblPrgCD572ZjdTRTVd1rrKnKkrxIixLu0OALnVUAB1z20oSJ6WNiQHmllw9JGJQZYvBSWUSSK4X1NOHigjHxOUFoP+mbwReNnYPwccC9P0Xxmq1saMNPTx5rfGSDyM8PdSApod4bX7f3ZnhqyrS+i+m6hh+8\\\\\\\/by56AQA4Cz4OBxEaR43ZKi7KZqJIzDoZAPAZqKIiZpeSdjgl+4\\\\\\\/I\\\\\\\/B9YrX\\\\\\\/zCEGPM4MeGxefGecn3lHEuk6m6Z2lRJGahTryfmZOn4fvO23sNWImAIhQuNJtqhqhNVINHZHLm9bmDOYjllOeXFvMwTQWxeBWx8zvhR68IfU8nrCVzrF0jmvzClAFt0JZx6KKMPsIT8UlIQvZbCI6dtDKGWOU9PwnEeaIAOBp648J5R9ZCj3mTzq1il8OXTuY896QmGaPOwSH8uTz8e5JQ8PC+I31q7EHwwbWt6ttsiPLqT+W4UiOkX6BkDETRz\\\\\\\/HKE4v0Zed9aTUrwWHvxkGiNadHv7vqxJgQzANinJQ1J9RFkGVNv1jYcRnuxmOIHH8VnZuAbMF1XoBGURHHaVC54Lvl4aHpZTWbxsk87gmdoQA6AWF6pnMjYqSRhx6h+qEWiMxj2bsltaLpR5gbTv8Iwm4z1o57HuUyo6\\\\\\\/vJIj9JLqwBqEqQdFayIaQ+NVqrLQ2XLTSLydUfjAW4lEo71q\\\\\\\/y2BabgsuVRdw38NLDFIMKa0mXu4GGDDvkp2hUszWcIqsLxOR037s8X8iWxpsmIGQX8AhDxyyg07llyGi95DLdLp6g6PmcPhO4RJM3CGEI4VMNXV3T1TlemF8oL\\\\\\\/RLC\\\\\\\/73eZ7emEM7Zzz32zbNR6KwQFtopP\\\\\\\/pDquqKKsqIFUlBAUtEZWmV0ybzBV8X0jVcEmzkDigJRdckfqHyhZJL\\\\\\\/ZPjEm68kmsOW2ts3p81l\\\\\\\/P09Y4FaRyvhFEsG1sdlb5kNxlobQYLNlpIxxZi+3bEOGxZV9UNVhCk0LcaopEdrrqW8t\\\\\\\/W8Z5lB4XNDxHaGwCbMjZS+X6pgoh27J6A7cS6Bfhc5GstbzjjLZNEXEA5oLWha46DIBBNRYWIe7ElTvd33XZai7B8MtI1SivBi\\\\\\\/86ohRQ9MPvq9I\\\\\\\/Lb6Mhb6PC6Oot0gFmjYecnnxduSfzkE2JAH1osWeFEUvrVT9MwwCVCb\\\\\\\/S98aWg\\\\\\\/nppT9xui6giIOSm4hg1w7KkyDppkvDLldH8eqL3ocM69BLzNR04f4vDj16HYt5Vldq7jFrPv6lfDDHcGz6MJXWuDeFOFlZzS+uPEjlcFVFYYFBq5jEa4VnDTW\\\\\\\/GAkmzHDs1JoNTZUIs1YizhoW4QBpbbMbTDTSSfHERdcm9yUjZsnneC7ycq71pgjsfK02wH7+VhwwHCuZyYfV8PEWJv7B4nvM0tzilQN6r\\\\\\\/ZKfa6Jel+hCMvCBhf+poYelraeqsfOsZppTpR9Jz4Yeq4528eIEMq\\\\\\\/HtS9nonq5hPr\\\\\\\/RPgTJ497oRcS5D9ssycdOpPBjLmoyRw\\\\\\\/QeJtG1rAXwGraznho6Ljsj7btWtcA\\\\\\\/ut5LbEGb9VbfMajFp0zOrZWtjL+VwCa1PowFu2AWanz9ocP71T186Qd\\\\\\\/Gvgkk15zvERY8+tvDDAvT\\\\\\\/eGDsI92GQqPFewZlmYxEeU2G32Z\\\\\\\/Q7YS5WTyM2zgOEnUXTDk3Cp01yNhP7nk6W4dsjPviHoZR5Ritymj+NuYTr6\\\\\\\/sHi6v6zaRfZ9e0PsnEMXC\\\\\\\/4okW6V61s2gJRmGUYVoalQA6rXf7FPYiRFPf8h6qfWJxeYERs\\\\\\\/vPMV5Zl+liFGVHWF99PRoPHuRPr1Q\\\\\\\/aIU5D8ZUGIkiDct9TYCEGm7f5wsaFo26xtYZfo+ijtsEIzKFzg92YjOCbYzA07Tj62jmw9NEpSlLp7\\\\\\\/pkXWXhksQsOLJYDkSCmnYixqvPEWUOVTIfHLKBAp\\\\\\\/zO6gHVWRyQDoiKA5rAMh4Qaqgnvc\\\\\\\/f3D3B+UeSkmcMQLKRSyF4zAKHj3oG2ao9emhmGXGwLc3H+LUgAOdJ+HZ41difwae8\\\\\\\/v+3XwC0NUpNV9Gh1mJ8hmCsexDDnc6sShOmce0M8D6WY8DqY3GRciCzrk77Rbku8HfhJT97EMyda6r\\\\\\\/6qQg97e6o8fmkjQLQIxJnHh8h73b59pkLgOlu3geQi3VOvC5iz1HmUr3YXSzNV0gsvN7nvvlJ\\\\\\\/dnSyV5EEUAUDisq0lI1sGkWkQ10MXWDKYaD5gcuZmP6aIVatInZYiQ1B\\\\\\\/bxqE5sgWCffoQ+xMkuMFRwfXti9\\\\\\\/KMnDWCS3ARUiZ\\\\\\\/fu2CS9jwnJA\\\\\\\/WjBu2rNgrsfwJYX9yI3qJK+5XPhHOD2NqJ+YajPbKFFIAxbA8wj7RT3siWeIhjMTh0WmfkdDLqk28FS4X011VFl+49qLEWHD92TSpFBPRCaHx0ZfR8mD+TQ8w5k61jZQdg5qpUfpnDuLXQ1iM4yWuH0Hyr\\\\\\\/TTr5J8fGjxF2XeB\\\\\\\/\\\\\\\/Yf5u3k0IPkwdBxurkhOMQzot7aaY2NL3Q\\\\\\\/wHHHidF31c8BW\\\\\\\/94AtkseG8\\\\\\\/dz\\\\\\\/ulQ5+O+N4LMhVkGHwU4aX008G6GqTh+MD49u8vGWsUkIGGZ5npomY5\\\\\\\/nud20vdku72T1u0qeuIemg6GLdc1KBn1K8\\\\\\\/FLxZF4p2n5lxulupaTO9YVYsNe6RXms\\\\\\\/wHdxRy92lzSVMO8xtF1WKnBBuMAZ8IxpNGN\\\\\\\/8O+n2\\\\\\\/gMU2DCfRxT6Fcb4BBsRuv3qF6yvxsQMwF7w+\\\\\\\/yX73r\\\\\\\/yH0EKLTrdUWDHgJ\\\\\\\/RxIiJWH+dhj+rTSMUvl2WIUBg+wVN9khJ+xvWFiAbbUAcHVMoksBVa1KZgBqjfNeBbuKIammSsAwU6g1oRQ3suHa502kdU2LoYnC2ojroOjjJaX0iKW0mRfweMECwUYqkYQAQl2HW2Ica+gEQ60rvvW6MsuEJMZKscMrejE10ujbOZ\\\\\\\/ymiIPXelocievc8E7J9fs8eQQlkExWDaMquUi30kW\\\\\\\/+uXfpjgEmVUTMDDTLyzC2vqTA3wE\\\\\\\/E+RnxLqcNnLPpGpC6E4g7p18iXO6khg9AyICrPQgcR9aFiHgvy\\\\\\\/8xjXoI9Hmzu345ayOJKp3uuzvdubpQil39xm211vKK\\\\\\\/GtPkctNhGKHX8nR\\\\\\\/MvxzxkPqBYRhCYkWBIs\\\\\\\/22igM2hzTPy7lhAJXLFM68JKbt5yBslIlpekUJaeUaw2iMDiPORInm9kK\\\\\\\/VyY+fGXKA3y07HqD4nP3ruOM3sTtm9gXS8pfXXFKS\\\\\\\/1vJQK7NEYpadznlzc3rsg6aBn7IunuHF903h+5PsHyqRHzYc+RSg0of2gpVsde3IoAUM9gU+bjkJN+zhkqIQZkxVJMe98Z8sYOT0R12MAEwiyAaZs6SQLljihYvVtxPbIp+KOB4gHS8pIJCYgc4tGAEsgWX7VlPiiR1AV9MgLXUN61a08\\\\\\\/DZwafr\\\\\\\/K27+OAWonCdpwk4QBHYX5KpkqyAcCZA5jxvrUxBOCFnh8HQkP+yZ57UU+ONVw\\\\\\\/vn5lPrYXF8o\\\\\\\/l8wa6TI2gVuQ8+Rc9YtcngLg+Ht\\\\\\\/vowlssPFQsQozinG8+eZhXPv4LXyroGP7Zxlyh32yf\\\\\\\/rE5ElA2b97kOsLIXRXzAGVTxIEcf7cUIeDI5zpZTlXJOKULdN7Bvngfpq6mjEi6rQ3JenfmIxl3g168HCGHEdpN1sqrp42ExFt\\\\\\\/W24N1zjxkKFYIRpZpZupVmqxmYI93pcWVpa\\\\\\\/Q4oiPEkY+k5PuRvQbVEOA0VVFHFWdaoMSliasbShRq58c8Q8kDXmt0SsI+bP0iRjHgLxzJgdYu34HcA10yMdhbQ987oBc7KdrpL4LRDZa1fdlNrZMiLjah37GqOAMav1+hbrA89EE7g3qospg0bt5cDR2p0BBQICmwzUlGtqp0tEjMJwuDvN5fZesd8BkHLJcmP2IYQKmtHioSqh32iCd10fgI2sFDHTmae+Q2mqniASKReR7y4JWQRcRRZ6IC6d2wUYf2ispOtnpkgbI8D\\\\\\\/70qSphGU+yTxqUTYAuIQkMJ\\\\\\\/EcUxYv+dahZMZ9y2ZgOHaHC3HnI5ZXIu+AAwtI2ZZXN8+MO+xiPvytRZ4Ciur69DnUzq5ul0s\\\\\\\/8RuU0q54QG8moc0TWfOkgt8nv0vmGMSnI1X1NhEVIBZVEfJuMoUXMfFOAjYs3JnMzXDDqsbHOJIZ7eeC86oQ5NdgZCv2l5TX\\\\\\\/sUEs9fKZyHt1y\\\\\\\/WgVTUKlnAks+eVXRnvK2WcQUopXnLyNVpn4PCHp+AoaEGxLNa0iYkDdUe1F3YKn9ZuLrMPr\\\\\\\/wiKUNzJM7UWLyX+ADZJNKoGazkDRbYN+nTNW+vJziUmcwtG36GmRHUOt66\\\\\\\/fPFCGyKNemhCyyl5pOpoNG93P\\\\\\\/cmUUfW3RLsTTaZXJaAEYMGaMEMBATG0R3ND7s8pnnnffK8lUe6tPhWQ4v1Vf+ZUqcdalYEJUdJpcNzcPT4\\\\\\\/b0EAlQGj08yiqamFiS51qiqXehskWpDwywLH8JRfV\\\\\\\/dHDRLngqjo3ZlAyF+iGqZWDbFJ\\\\\\\/kNemIpzSy8XnOr2blltVmAF9a5yjJMS1HEbgk7SjY1fXyvuY91jgOnfOQwPHZBHwzqTZKJvfwNb3OlO3zRgP7QP6BuitdEmpIRFttd8wrAQC+s5G5R82BiV\\\\\\\/N+6aPDutJMgsOtQ9gfhOXvLx2TOd1NOTWpxY4f617+2XaoNhe4xPBtZFobbbJsALusD3CmvYhNMfmtS48umUQZRuvwnW2cnc1VZdUd984Q22+uQv6y3zaoPUmcoRfCTjisZpKlWnfqchMqhtSZLhHUt0JUKIG\\\\\\\/\\\\\\\/cF50E1yn9GXnGo8DuMcme3oBbFm8TYs21dHssvHVoFZ0xWKkcctsSNzbXIWh\\\\\\\/kkRELcUcya2oN70NgEcJv56RLC4qofjneJUSVlwAfnzCBFYQb6TaZIU7c0OP9pkL4a5LVKr9cg7SaDkv5OXaQtrGUepDR5Ken8zxFzah1mVtc4SxP3gdyZ0mfeciCMZuKxe5rThjwj8AV\\\\\\\/Du8W6MH55RlCqvBFF66IfP458Vk74QoyxMsne2RLJxG\\\\\\\/qfsfurl1SD9Olz6TkUJOSWA2BfE7IAh5bZcEEJ\\\\\\\/GNuiWS5lJrnSfSUVMxJoMyfcJLUMwZIycF0gjlsLFFkR+dZ29u7GHS7j1zrf7QVkLcTKU5Os6i3bfEzFOFyMOcSNM0iCmkGgiPzkaqUyhNsTrCiJzm59N+akOqFcMWUTIn8sy57tdwMHx+xC51lQIP4SCx\\\\\\\/HurfXkqpN\\\\\\\/+7Jpf+98IdSTnuej8vyKjNAWK50Ouo0SyUflUDpfTQVMgms6gJ4wTVLK31+4hwcGjdHuua9eFl+fiOeV5EP9L1iYxggIK\\\\\\\/d5TsXb\\\\\\\/vKlvJJdf2Epgk4iDdyMx45vn9vyJJckIeb0pUqyUWqRxMExkW9DAbVyOXeF\\\\\\\/\\\\\\\/LkxVfi9UJNiXZUqdIcboa5m52D4zpEh1IOlBCDTlUXHT53ntiiOpIOPEbAU09XbL8BDuJMe50mvIXCPxJDXnHsj5qYzpZRIVCxHSXoFMredqXoVwea\\\\\\\/C6tm0PaFzspJO8qKN6ugEesGqiSPr5FJy92KwSrwuINI717PdDglZ9HjJmc1QCN\\\\\\\/ALvkvaxeHlr5Myk+KK\\\\\\\/0rankdLq5Vyf4tdq60rz2XBhGsNoGYg5iGntC2VIuqx3lMZ\\\\\\\/atUxHpIBU8UvNtevHokau98dDWdn2oLfdUEvc3IvkbR9U1BcBAn19rIPVZOvSw4ePnSBUCI4IhFDfuxZ9BbMOpj1Si65J22\\\\\\\/VjD363fDo+Neuad+w2\\\\\\\/uTZpWxWPxaOMsuCWpxPTTOB1sXzcoCBNxVgdlzOroHhZhcdy7fwADO91IbNzWkF49O2EtRkZUccvMjfH3CigC9f3OSFnxhhAC9mTYaVZ9lIIbFXJNW+saLEEtgjjIzIVBEq2+R3ybkahnkfXZ9B+CkW6yfXOkKNbNTtfR\\\\\\\/FJn0qUV8Vr+5SgCHyUpdoILge8tLorSulhK04adblBUqQ1DhdRyz2zCRWJnQc1jLQugKfZUBHpbHV19MQl2kZhhm\\\\\\\/30QR9bbaDMzHfJyLLFbQxgaD1pOG+tXwksFwH48y1JepEPSilGBk1Kh8hCn220W1V3utxpOwfnkp3Vly828Zt+U3I+\\\\\\\/67nvTQIVneBp8Ew7OCEF9l2FJ6dT6D1OZ3TerbqjONR2InOhvymkpqc4h+\\\\\\\/OqQhW1WtULBLbffj2BUt0ftDf3Ed7KZ4vnGH4ZYskaoOtZJSTFmWRuSyUQCRgsQH82jGi0l0VVy9zZ1UMrbVayUYpvD\\\\\\\/L4QyKvv5GpHO3PAjYlCmFfMMHlVbEeeLcpKUVUUBLPvCk1Gi5g0FpVTFX5PSNiA7DQFFzEeY2LLwfYJ0v4ITpA2qZoQhZGVZQpd9SDvWGZnIOHLdlgXoKcuQVu8+K4clYRUXJjZh\\\\\\\/JMn6t7Ic1tM5yTOrsaegGANUoFTr8nr56h+jISFjQc8mwj1fSmDB7+VRqJExfvkJXOutmUw9ww68xZKxT9kobH1nol\\\\\\\/p53qujvC88W5tPcWY1mW44k\\\\\\\/wWanCXOAMEAe8rN2C+MkPywTJH9JPLIMVpNuFDMDx21z2d4ZfweEfqImyGQhySSrFIW4nLgA8Zm6lGA1KgGUbmr\\\\\\\/VS1VAcl7LRA1ekGOCwnIKx3yPq32F3iMEbFITMIrAYVDo+fcTSadBqq1xhpTB+3V5e7\\\\\\\/IqcVjtNyRsf2K7XPCz3zYNdAcAeWsT8BP0yHvPjUVZfDXcYQc3XptGzIIeHawpS\\\\\\\/Kv4L4E3QtR7IwjdVrIcY37ElRRiSMxK5tMZ8J5qhH+23ASE+zwgF77PUYxIsDeVfriPaNubjQaV2nlcGCdz2NqAY5htQ3kkKCvJ2oqaJTEYS7Ytix6tZZ1IZqeyFS85nIdzl2Q9mf+riR8O+xwQWOBRgDnSc0Uzxqa4ODN+qULqNj2PdF0Itz7\\\\\\\/NmlUrhCm1YYVjWW81oJNQKI5FsJF8s4FmdGczFYY80EK8nvNd1unjcwW8QM4zcokAyXSNxJE7+jS0\\\\\\\/JapoLMr0N+i4VXjG3hdjfRIjHFsmx2o9WbZzh8whVRPPBr7hweJ4w4hL36qSBDbTiFhV1QLhRRl7hf80uvr7pKsYgVZAxFeljaOVHi\\\\\\\/RkdChZgNv6lzRUeSxkf72y3pBiiUJTNJFd26rWdnB4juCvEO7PasqsZX7wY6Zoh6RUONGqRc\\\\\\\/e4EGTn8khOZxFODemZ\\\\\\\/IVY71GD\\\\\\\/ACq2D4IYsk0zIvjKq+pN9pF4fKihFvw+VbMNKFx311zqwOQzC5hugiKUZQeo1gaqLSRE30IH1F2O1Qquk7mDWGGWMdf2STlcC8QLKeaBU7hEvVBGnVF3LcT+VJw8AH4ROmcrctpzW8omq0zkTsHHEfOoa9S6BT1+eqxMIG3+qztF+JabbGtWGezZwwadMhYDgEbX7ujaYqrqxamj6xTD+SaW0XOuv1dAaVtZHMZu+I9LHB28UnPTPgzSIeDEBMAvfzHmYjGMFCw6fVuY3GbDwxQFxnWAc+gRN6UuF9Fhzbdajs2dzp7q7oruLkN7xl3X8MBxB4F6uN5CUCcAPXzOOUzxYFrxFQTRUALzvWa46TEBWqbbxEk1QWwvBmlGpdgJNu7RaZPlyWDXQI28m5wdUB8B23APNXqlN0W8f\\\\\\\/VHIy3rK7b49WJfbkwIt+UiAzJ7bXpZXEtOk27rh3pI0oTgOx\\\\\\\/Bks+W\\\\\\\/N5Qphd9REgdBrCn9THlpfwhMCyrHhw++LW3xW9YLipQ+7rJgvorQ9+AXYKBRugALrX1K3fSAM0vJQE4JvlvpeJ71arqHi5cZtLqnECEBNJRSZ2bnsgoIAPh+Tfny6wmJQKi7xBSABxGMQ6\\\\\\\/tdL7gIgOAzzG4Z6M6o7Fk7vz91V0rzuYmS96OBVbMUn0+MNKbiEl6NlAl8n3oBTR6C8H4bsO\\\\\\\/Aogf\\\\\\\/kgCkZeGBK0lBslOJHLXkt0tCOfV9SmDoivy0zKqQldEuDTjBpEkdyIsj\\\\\\\/nw6LxxpfEfQjQt3pD2Z4YeEQuG0TnzR2VdFXyndFHYU2tknfZDzP8pZAtqvGIexTnYQvvpYaEvp9IWS4cSjmIl1BuoAZuBCCRoGBJ9XtoSRlbt7FUk+J6pyee6M3vABVEyYjE6q5Z77r3j5Uk8+L18kQCVHeHeRQM9qqUmKv5T\\\\\\\/OCwJPxG+xEFqM8EdH+b2zNuyyTsVDWpzd\\\\\\\/hpBe9yJNu\\\\\\\/dYNdiiveLnCnY2SNT964\\\\\\\/2SWHhODov1SXZB+iIWGZDWk\\\\\\\/79tSr+lhyCw5WqOisq+9EOMBJEUkY2tRxJMH9DcW03Koy2foJCA01EO\\\\\\\/MQ+qdw89jbvdIRi6Cwjt7JMG6tY4YpVlXFpDbxFqccAmkBZxxIHlN1ngWhpkw5hJlx62qNb5mm1sx5BNAEulLtwVofu8cB9V8XUIScABrJ19BrUzgG+fhH6o7FuZSk9q26LQk8ii5GFJUG\\\\\\\/8sxTXy9bleSqNpUaCoMZ+qOkFoq38KtJS\\\\\\\/n5KZcZk\\\\\\\/xNeZLYgqawfHfqzS44jK1w5V\\\\\\\/TF6vZ2klOFEtIJNCxBYuQgH8qHoELyJ+KXidLz9KytGAJtW+paDetFbXP6NoNQk6BbtAwpGB7v8sdbUIYxzXV6mRsA6tZpgjQrosRn1LxsCauba7S08c+eBmwPUaDM24+TL3BhpsoxKKbxO9CXHk3vAFkL4o1ph7ZETD0rUPrK7uXrFgFY6dd9GXkluBMDYg4R94utwSlGWovSgbSXGlvPZ7p7nWvqas179+yKQtMKqZa4ApsT4R54rNfaV7wpvTmbzHeoGPFw5aHGvoWuvuiMfWjlDLaV15DFuH7UR4rZUwxI6sbQDGwRiRWFI\\\\\\\/pK7YhWk3R38A61CVJXEE7S9H\\\\\\\/JncOPZamkxzUKCLtJ6jTLky14NudWwI3xiKZf4slrukdbZ2vNUVBbXIoMj41RArC0NNAKthG2asKdUli3smddwk4\\\\\\\/sOLc7nZKLDsFoEe\\\\\\\/J7vvhjjSYlHPZqcEDf4kLtBbv05EsDjBXA4bJJV4sxPrreDORUXtc7ZLlM68uOlU3lbdw3Hs3vg07UPdgiLBZDjiOEEz9eb7ckJLnRVMZtH29l8ekiLSTzEZWY68VL7ZvQZNtVLnUtCqxPCmbs0CBfqvYpZ6kvtaMSIUiueqblTi5av\\\\\\\/rz2qCeDHs\\\\\\\/tlkvsovawbKqeeM1bn06lvG4pKIdcITsah6tscnHUQBQfBeqmDBmWRcMK+8Ip7NCXo6nHkxhyT0ySYoEiPx+iyiwWXMIXjKB2dNyS6V7xPwfhfUNjlSbwxb43eGvWVelIOF1HSd07A5Ez\\\\\\\/ZSexBmHr2nC01+w7ywBinuSQkS6coSw\\\\\\\/qegpLj\\\\\\\/RSdAdMeO+H9JE\\\\\\\/jKR67bogCwX\\\\\\\/pUC3rJhekg3VgWCXbgNlxXgkXtP0ee0CovXYl3a\\\\\\\/MT8k0j\\\\\\\/YjGT0ae36GFZITQbHO584Tr2yDRM4d2OFoAsWomVcYFtAiQVX7BDhQiWfYoonNpZHHIL2BAY\\\\\\\/s0eogyN4EkcMcMMlfhSTi8lBkYZl9zI7ca1vnL+tx5ASZ4AAYdnZ\\\\\\\/W3qLaQH1ENNOxeYHI9N0BXe6rYaYxs9ULmjgyqlCnVDSqowTnaR9vpfsbbkvY9ISEDb1fMJVnCnx7X54Wd4BYFTwRzHnsQsQrqzeGOQ+AMQao6NLQwOC88b8quX72xyeH3TXbuQJ0SnguDxmxiTi2wYrEQjahnmxKoH8VcVl76VXa8AoygAs8vcab\\\\\\\/5ykzSVbiSB8SHGm0LkXIcogIFDY6RcnHwdwlEUzaisiBobMufe0PZ1Ioy6PkzcKwm+YVws+RCp6wSn9aBZeDE3dGORHdi0dnkY5r\\\\\\\/q6Rx62Eqc5rOAmq368XSvYzIlPGbtFpNupZ9khejXm34suoaG0sEUgvFW9k4NcRapVZK0KsUvoE0K2GZeKpnDI6y5TIW5CQxnL2SDTVF\\\\\\\/k3cGytnvzrzAh7e9RJ7u4dgXeKWizywtDL\\\\\\\/cyWhhjxrQJjzSdkeQsow8m71KOixsZfPPsrpRYG6uQteCzIq2UrhD4VeNhJDjNY8rF04sqr0LdTD7lZwT\\\\\\\/cXtDHfOUWnK1iDTB29oBK4wEVFffmtQjthBMYe3K8dFpu1+9xiotvHsRH3sGHW3EqTKmL0ZF1HrXpBJSydJdaWoeUVehzkTdk0mQuVsO8UN\\\\\\\/qXBQ+n1Q2ED+o1UgZUQLKq+oNbwZ8V13PFh4+\\\\\\\/OVgNSAOwnfEp6QPRmxkx926xfBTIVqCxM4M\\\\\\\/B6lLPOPQOS5hC37Gmrlj4LQMQJ1iN0VGoSJDujA19QW3jW5XGUOKNzZ\\\\\\\/wpTYeZfQCJ3pd5IlT086SlWDnIpLFQqTIG4IAWGZ0e5Wog3LuZS+9X0aep0osLYqES4aWtxnwkir6N8CIEAPdFjiVUqmNn8HywxyCRK8Dm3UGnk\\\\\\\/aHM6ImgiEuIf50\\\\\\\/Kk4DTEv9HLZd+kNcz7\\\\\\\/6V8sjBhyTSiRCOkwD86MSw6VoEDBg29dGmc3RKXfa7I0YzDPByrC0CFHSIaNsFkPVFmLUHOt4ZG9qzCmlftrpfwzzhSGHx\\\\\\\/rcONLouFfAxF8s60YlUQNvGckOQ\\\\\\\/A0TUxwja4HERPZXdRYPIH4noTZXnDs7L9CgSykR76fjbr5oldOa8wcjlwh\\\\\\\/qqok9k+QtejpxlRP9eRsIKmHChsxq85fk+BCj3iB18EKseNsds8nlJxqJOIsAOKlsPDnymmLRZuRunKSvMPm8u1HKNFBDXasUcZSrGQsxDdJcUKZUMzny3V9imQhUmLcJOhoUYDYiRcI754CEPAR3dbQTelAdAsLkSiGsZywTW59E84M\\\\\\\/Dq9w73zSKCCw\\\\\\\/9jZtcv5D0kGGEqyXkZqVMUQANTTyrJw66OvXnKUvF\\\\\\\/TxUyZGf73uJXGClFOy8S9rWMfqjifyWvMfFJunjvohJUpTL2mgBlCBzP0BYwM8Qvn\\\\\\\/yLgk1y\\\\\\\/9FwreB8+rcI+OrzPhnRUoyqffaTIt4Yek\\\\\\\/2H4Jj+eP1VjrexxVoW3jjf6sJ8Daeiy2cT\\\\\\\/p5SZCEevI09DY3B8gWPhqPWxdZSK3RfF\\\\\\\/0Fn8X3nVr4CQrVqy9eVACKBKNkioayvlO9pKzF9dnv5W0kU8cZAO\\\\\\\/I5CyyyQDGcvEpE8ayzfQE9ZywYXwbiiBovUyMlLHa0rsuS4qf3CSBFWHoeCFGT+w3eUyefb4rKQ80SWg6wR\\\\\\\/F935dVG8bszYgzTdrxDuHDNA3uKmQfBGczjYrgF+MRbc\\\\\\\/s7B74lplGc9mGj2Czx9ZfXuakPzlJyx7eh098\\\\\\\/AGX8ZTgEpzj+K5co6WC3wJyo2YIcUS\\\\\\\/GdJf6SzEid7\\\\\\\/GnEZeBFWckXK4iIC3AThhpfZnJz5ryfkATb9mIzq7hfV5ktB5HysgT+kasZa\\\\\\\/MuVGjhgTMLnM4F1sQcv\\\\\\\/YmtXnzYuVL6bb9H5l72LGAI0YmSvcCaUJxEIM1EV1U0GeMXyF9vHiurMwWkcbqmYn+dXoLrr6Aw9c\\\\\\\/eGT50qWTvkYqrd2kKEKPO\\\\\\\/pCihdxt1mfFVEL5leF+CY6Sat055pX\\\\\\\/5b3dacyYutUHppCmKSWPG2lmnOdoMXq448\\\\\\\/6tP7QsZLW8JU88us+rjjLCtNEHTCD0qoGBwsTa1GqXMUQdgkCEYo9iuXwo+646PpNMqGGf1GcvjmqZsMk1lqHnX4p4OohqepvRGOHAJAStKiDkKvIzaaOQxX7Hp0nGZ8BsNmq5l33t+JckfiSyOHaAsoiWiVh29TuH4nXb6cjIc+AdsU86xDI5T3\\\\\\\/yJAzKH\\\\\\\/\\\\\\\/jRXkzXfwc9aJ9BrRsV19UyWd4RYReIAZFEHAM2kqfdamqpjV6iMrkL69f5w0M1uNgQO7iOEFh\\\\\\\/uLYAFKE0cjbLR1IZsRBEBmRoM8dnv2PujNkjUP\\\\\\\/vWedtCSPxe2LRYT16aQ+UwVCp9r2Ople0hHCHJvHDnpoXC4UxbkxV+PHhz2z22a2Eqi9MmFCEC2066NyHZ2ZEsBoOtKiBDDSEeJeTVa6OlzMpU62POVSLTzhi5oC\\\\\\\/\\\\\\\/a084ouB4G2PG2\\\\\\\/fGq\\\\\\\/SfmzxDTAwGIogYzcFWhzvvmJjrWAHE8zwADdTUYO1XLlxx+Qlm7EVCPdHMH7aasvO25c+az8+bN5EiZVbjEqMdLbtx7n1\\\\\\\/RfZ9cjzvZ2lb3SuBiYhebNbulOEOnlJEFqWWApg4YoCcEkoofZKPSWshor1u2sq8C8w\\\\\\\/TZYnJkXFOIi3xIMa5aNUgVSEi\\\\\\\/d2xeRVsosfd3bAnAjIiFLaxS6l0jbhJNj4GrA0E+bsBUHfN5N7Ma+tK4aP+KPcJ+N29wrplRzsNmllEtO27fmfbtJiW8jS33TrF2Q5bxtVPcCU09e8x8KJNzlzmwjN0OtMgR4KBj7U5omDcWkCTou0nfwffET9rM2CTCp1q2wIHW33iPhQJk\\\\\\\/hdUvffj70JkDhZpB18\\\\\\\/x+5iP\\\\\\\/u83swxe5pd3zQemrk\\\\\\\/FNXmqM5A768kTOMfmDOJNoMD8yXwXcUlbaVWN0ngnF5ej7X1SM2yUvSwsitBrzEotVPZiHFLQcXqbNMm0jwKLR\\\\\\\/5h6RrSX7OE7q4HKUM5Jjx2xnfWOL5l0TzurqiZ6Z1IArkSak7oVIAfzc\\\\\\\/pXCN1t9mX+1EzvO4eVrzQLgfwzcdSN+cnE+D3m\\\\\\\/LyG\\\\\\\/tw+K5aZfDAyxlo6q+6M0MIgUQF6Np9g0tnhnsjzRNZQ7nBo1CXZZQaQqVZM5KPLnW9t662VCbJxZy5WQc8p5bxy5YYvh718cl5EEjSVweMGM2gs0rCwjOgBhoydMYKFfmAQDvY3\\\\\\\/TedJSzMdvtrU2RWF5AlqPBmi8U\\\\\\\/J\\\\\\\/K6eJ2cF5CfasWl9pGkmoDHIcdVcPgtLazArZcI2d5CASVZZGW8YUVIBAopMToP4v9oZxIIdrjE09w\\\\\\\/3IHJr4jMQdl5AR9LaF2inScQpdT7qzdnwps0OIsQ+L807i75J7CCYWejF2KtlXAkJQHvfoaYLo0EZ4iB+gN+RLHOUsgV5XH\\\\\\\/tmprHYohbP7NICD+bNxEkTphEFjCbd2lHLRBM3Jt+rD+GmL8f7hC\\\\\\\/lzD3J7EkyUTtfcIC5a0z2T2nlp5ZBSM5OMIya+jROG1kN390JR7v3mPrwd8SRNDmTvEHiIdxuI+I5stSY1RYUvN0Bsl6tZKg108ZW2vrZHSs5iUxc3bfsmhjyBDOrIY+q1VYfziEFDAZqFMelDQphYfzi8WgxbmQ1GXZbr5R5dP3T6U4LNEcg445q6Wj5NCejycqXhTIZHpa4CNQp86\\\\\\\/cxnMN7q4Zi9XR5PbNmMVJhLAH0\\\\\\\/DJMEnMtNeDj05yJ1DgakU9P6AAqs1yVieORbdQOuvJfIoqCrM3wgIpd0I05YgRNFN\\\\\\\/u7cr8iaBwRi84I0C3fB09VkidFTLZos0lrDr+V9wkEC604w4oVi\\\\\\\/vP1iTbe3bVwfwkMC6vOBwcLn86+WI+4Y5UNBjL\\\\\\\/FoV3oRtmvZ3kysAH73wKGbf0z1Lw+ACDrD9OZbpLwX8YEyWMTaMkAewcBWJYnwavPyZDEVQzmloRUmvw8m2of5pk3XEeBXqMV93ewqNqqQz050rNpbVU47X9OUBQ8kuLOf8ds7pTQnXWvtevu16YIibU1r60liGfc7nUpcK\\\\\\\/hU\\\\\\\/+VSCCyWEWY0UJK7uShblckm2Evctv+8mfBHfVJ03uIR86LqRXxVI3HoenT9mF7oDRojIWD2PBsRJUvGlyZgYcl1+2RfHwO29isYu6FoCOHOtRqQCi9co4uN\\\\\\\/dVqzCX5q4hVYLwyUJno0Hidas90ebUX6jxwJlgSCITZ95eCkFhp1iQ6UgXFlRYrUvxwfKnUHS3qGkibckt\\\\\\\/NRAjHYM0OIuciwUeXPQFGPCYZRle4+tclIS9F0olgnYdN5KFUlgKASo8t8la2JD7JB2+4BenJJ+WY+hcSilFxzkylz6cfQb9eo9VplB\\\\\\\/c5mShtxJ868iuyPcpgn4j4OgX3\\\\\\\/mJWHDQX1DGdf3+5iwSdJBm2dhfBcV\\\\\\\/onQBW0tSGDQcbdNNnnCr3CB9ObVnOHcXuCAMsJu0OFE5sO+KnBKoQW7Hvml5jBvXnQxT\\\\\\\/ORxBVHsU7BADZ8H5\\\\\\\/Nusre940J\\\\\\\/xXlL+\\\\\\\/ufN+lryB1sy6WHx95lSZfPwLL9DU8IGfblmcG7n3GvMiWZuqqghaSRtDT9YyFqhVzNO2jDLciK\\\\\\\/tU8y42P0adzbHBJO9TtJrAl4qsxSsvotOxmzLaOH9JEoFaGlgInXe6PAT9lP7uFpq1+tp68FBPUwJZzVoz6TkLJBNJyC\\\\\\\/oVuoLlCqeUOWg3Nq9Vrn3yDf18n4ySrYxi+6dlVZnH4rbBeeQuyhvf3IVN5nJ5L6GJozXzsBgyW9EiBsW++k8C8FhtrrAGwdr5P1FbxvPV2XXy50eg5HjWCfbPuJk+vMxB191ype3syoTy\\\\\\\/wvII3ZASWvgQrd4uGX6+ksOI4PgLZRC0\\\\\\\/UWowyv61N05NdvAnjI0XBgYef4Tk+Dqj7CKcQZ+XrUoFZqvmHCBt43BaXtino8hFQGDMlcyKTTA8jn3XCMVEH39ZEBoqpLj\\\\\\\/4Zzng9IVcOXqU4UzXeEdJ\\\\\\\/Ncxs1ZjRRYG4B9NmZOIZPfmJiZ8v5EQ\\\\\\\/fnADYXA8bYJ9H8mMCobD\\\\\\\/ZezkFjRBirKpx961hFeylQGtVtqUVC4tlm2y6OPaJhzVE9D3tb+fz57J44wMUQqNUtmGrAJCG0CHM3eXF54IHRkVyj\\\\\\\/fs\\\\\\\/YqNPgIQnRBrE4EQVL3ufF0DkrWTuMCqwvyapBUbnO9aYzFRxHj9fM3Rfm+1+oqe5hRHH0cuwjHYaEovB2miNGUZhwWs51+regSpI7wUO8xT\\\\\\\/t88G2kZMsU04Gb66z0IcgDsL\\\\\\\/ISd43qjAO9HxexUYhw0nhi9CV75ZJfOqtzz9tucSQmwyTl5tltF7PYYmaDrliqiueIxbeH1H17LRILTMUsnhZqh5P1m6DY6Fqb0DiTjqFixn7rOxuYhmHU\\\\\\\/9BaIRhNrjBrL0ki6bNbqpNTN6Ml8PlF1nuzMfmpCHu4mzDg2sHElyrzispxkLDPVUW1iF4WO\\\\\\\/TuPs5QYn0cKedv0IO+KDl955v9wwiuQ+dSs47QZ8K6LMurCGXGH+uUHpsTiQsME+bT6W40yw49rHIN06qq34OZ2zfSTKzg7e2IDq7iZyhPBDrKlNbVpZxsK0C4qPHZDXIcplSVNym6OYMrOnzBcVrbIXUrngMEGhteijAX0zQ6+lldjD8dshtgu6ujM2W\\\\\\\/CHB9uaUp0Rl55NWh69JeP+pzq6fBOPrjwa4majqQlAUWMGfAhmNBVpeHB6plXcdH0Qspg+lzBTrdaQd7PwLm4XhUtR\\\\\\\/CkhDKN0msfUNAUfr8hb6v5qZGTpcrJ\\\\\\\/0gf3IJ+BIMDt4vEX5r6j5WMdd21DhZCFmYrtF8dpwvF4gIb\\\\\\\/NfX9Zik9aHkwHNOCvZHsh+6G\\\\\\\/0knZZ3sRmToAA1jwo\\\\\\\/2qkQALLB9aWfx30P2hCSayZKXB1ycU5HmA0tx7Y+CgZx029w3yFIegG2aXnuv4ErfhSLA0nESgPnGE5zSyE7KD0UoCI7mBLn34Z0f4BR2OACdfcXN4rEyjJ\\\\\\\/Zb581vIb7L0prxGrFfacSmjNgsirBRPT\\\\\\\/YNTOt4WBPj5Vk735tBkuLxM4pl1kmAPllcXc\\\\\\\/hPV7NINUcHJ\\\\\\\/LsrZjBFh3gv7+RuE0v7N4C3I1\\\\\\\/sUZh+hGPl2wqERunjEenQGnvJM8lhA2Rf22QUgf8ZSfjLAtJ8iAm7A\\\\\\\/arzLVnQZVUWzTCaEWhptGU8OpKox9OV2kNw2pRUnLWvYi3eckhBITUXM\\\\\\\/3y3oz2QHGLLdasWWfh1oSZPmwjEzjCH++eytnjv5jhL3+jxCmmvord+y+C8v4KAqx1tvOHjsAyH3luD3fbkByvRaAwzMTZecx3oMgmXPnpBytXVoKvz5zm0leIW1DX8qyCz1mUXCvDYsADPmwSGB7dNXsB9+JfxdcB8E2764LySfgOEn4w+Lo3qve6BkR6ePpqv+ABFHk0\\\\\\\/zpsJtmW2ISADWSRGkIyXWgD04NPJtVapNCQkgr6RbFnzaywLCMQSLyc+vxF6dU89FCy+xJNkdACMjbDi6dFygGR2DpUGx023v+pzkYDUkqp5PCPmrWXTgfXQw\\\\\\\/uj9XOmWp+AomxhibSOx6Sdmr75xBTSYotBt42YA+PWyQgn8xkqqjK4R7s894Uxxt4Rc9jJ6zHZcwS8QH8\\\\\\\/YDLJiENPXpRWei0kP+zEITaoMFD\\\\\\\/Gr6gBlPCK7CW43g2Vbge6lOyeANdO7V\\\\\\\/NkE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022NsIU\\\\\\\/dh34ymO\\\\\\\/U78Xpu2Ng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a956064fa392d9ce58088e0e23bb812a6028258885b8fd90c09a0ed8d8f9bc98\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022frO8Qm9vForrSTmW5uEkNzOwLqBTOAfBMEG05iArfCtvXxwPowKDbHLTj\\\\\\\/w+Yc0xZV+8X3pNKK7rp7SzzJlOaFX8nXw0BePaU7tmaTuenStBeWlGl32C+HtZy3\\\\\\\/nvSGV5wdbBOHfQwUra\\\\\\\/A5sx0yLYueUMEIuHJ33GREeiBzpGpywUp6bQdjI3QBvsTAzaentRaMO38X56M9xw0OSW8TvghSgS7Is6\\\\\\\/QeqHEMFfpXesWQbOB2SFYOuu3B+QO1WpMnCaNzEUQluOZRk6qGOWtbAJ69sUoJhyKe7rFW5aLyhe1XvaImF0GUMu5vk8dRlTX1KBGDnKqZNbYyH2FNsaW+C7tjCfpYxLBR3iez2I9NqRAqKFiNueKUZDVI5dtiEdJ4zJthkfozERJ1heYUf9hgv\\\\\\\/aYInh81rMyDRyXQud3YVReKYSJ2WL4vEnCZLa4YiZ0Xz+HbPeRS0QmGXaNJR2JfXUQiZULC+5Kk5OIFCnUbbnM6O7KNqTx8mrOjsU+2TmmGeGAFDhA0zkMXFnu0+7RdjqCYzBlVYFtcKu5e2gJWP35acZgCmg\\\\\\\/kh70IGciL3PZk7eKLMDSu9u2yYdmiy8Ipt1p2HFma7sY3LgPfO54dWTvC9qz0JgoCespy4YRS0dD7lsv0GT7E5JqtGZflenY4BVvfuK8Tr1fvaWFxv2cVsvrumJf90MR1cvOX2HCsVzGvdjn72AoezHBLKc8hJ5LYmkjKwkfBV3EqZPY2\\\\\\\/2wAnZAZ4ajeIGL78f6Ge9YihMEJc4sOSpeyZE1WCDNhdKPWxHTQXRjwao43lf90VPvWPgcQKDqcxIYswloM6T6Kj2YrkeCI0C6RAOxunH83j+Uze+w6o6p2wU4J3xoMMez3iOUSI9iH9AkSMno7a6poSo6KS+3VlRDwF9MI10ELWXivOIFtqZIOyeGukSgkRRNoPnXe3NRlu\\\\\\\/xK5lrQ2UUbSxvFBkx3wFgytwCPDGsh3RuAw3RVk+yXdOpH1JAH\\\\\\\/oAZeUnoonrnKVKsNFYy58n1xCskl9trUr2AWyuwlfflQ7uIqjtQ2bWdvUXPp2cOvxiMGjHh1xZGYdRbxfi3FGseWbIGhAi4tyobHDml47cmYWFS++G3TA3osrWdR0BP7jlOy3jVs1Gn+0nWCzBEO4wohs0\\\\\\\/ZVn2uWbbILR8EK\\\\\\\/5nFuLDzk9ydBcwhY4riE+cUxF5SPNxIiO3mSKodC190h8qKigmUu1lxvzTewmOUnKzBw1NKgQ1aNaz44JVAe7HqqoJaecfiNz59gD2IayXCwJLRu+hKmUgDku17kA0lQ1\\\\\\\/mW6RXd\\\\\\\/+CllcK1rTxYzOsec+qlK9+3XjlkIBmIs6pit43Wuu2iOnBIIiG0J5xsf3zcyu\\\\\\\/Dv4HadQCJd1G4EKiLchpB9\\\\\\\/BgkDhHFQmpzQV+gpQy9y0UJprJB2mN7q+XyxEGZ+bm1RFyO4YI27RBkDhv\\\\\\\/sGlgE5G+ek6x1bKMd4dNHUfMxxPVFHhdu67aN9DvTsj0hB1DOtzN2k0jN391gAPGEksQpvbxWS1gyJGiEi4RH+pt8CJ6vrglxZGKzJLwM7cpYzMi5doECGs27OtEXUyY\\\\\\\/jywWO1wnqKxJ05n+s8pVmjJo0FGlaWkSY03Ic4qPYTGxVTQ\\\\\\\/cwjnhce3iylh8PB5KXKaqkvCH5JkdFQEHi35uYjH58lCqXy4FYquyysDh0g+q67WN6tgDhWlSpMnoS36LToI5yjQzahC5fxV2BPGkJo1+L37LSa2qXWO0on+\\\\\\\/Fzv+i5q4mv4QlIbIjUCEEub58tWkylj7fJfPa3PgvBY1CaguSo3f\\\\\\\/Bu1lMjC8bU2RQHSw+0RcA+IDkNnacL+vLS3JLU6CSvJprOVCP+ANEQyUHvOBw\\\\\\\/NmXFOhOsfOg\\\\\\\/EB4MM2TLPhyiAI4F79hNJRbZOV2QKdCMW9iLi5l1KoHY1AXrIxQnJ8KC9\\\\\\\/AK4pw7GPj5D3biqJ2t2lgXaaQsjIdCJALHaFCrRiSD4tlAPy280i1XKmNToY5jswDJavhDarssnC2sS6+VxqrYS3YPPNesh1ULnEFxggX6OCEvQEzvl+j\\\\\\\/51dt\\\\\\\/pRIXt7aInWOXDHtuoY9RwhVYVmNNDLQQGiIdD24\\\\\\\/ll5dRvrcR8mzSObM0cR\\\\\\\/lfDSxzEkfk0e575IHA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WMv5WJnR7qRb0cHQBE\\\\\\\/oMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f2ab4bb438653fd30e358f428aafa4d9abe72980ebc6e372ee03c34c8888a27f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487100 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3775" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WJMcY88cmUmKf_UEKoqwnzEDuSWmFIVJpPLhca2bk5c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3775" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SEOM-vTSOXUbxQe08sSBOmOZocNEvWnOtW5dboveK_g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3757" title="Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3789" title="Pervasive, Genome-Wide Transcription in the Organelle Genomes of Diverse Plastid-Bearing Protists" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487100" id="node487100" data-pisa="ggg;7/11/3775" data-pisa-master="ggg;g3.117.300262" data-apath="/ggg/7/11/3775.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequence Analysis of Mutations Accumulated in <em>rad27</em><strong>Δ</strong> Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sumita Omer</span>, <span class="highwire-citation-author" data-delta="1">Bar Lavi</span>, <span class="highwire-citation-author" data-delta="2">Piotr A. Mieczkowski</span>, <span class="highwire-citation-author" data-delta="3">Shay Covo</span> and <span class="highwire-citation-author" data-delta="4">Einat Hazkani-Covo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3775-3787;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487100" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sumita Omer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sumita%2BOmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Omer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASumita%2BOmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bar Lavi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Cell Research and Immunology, George S. Wise Faculty of Life Sciences, Tel Aviv University, 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bar%2BLavi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lavi%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABar%2BLavi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Piotr A. Mieczkowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piotr%2BA.%2BMieczkowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mieczkowski%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiotr%2BA.%2BMieczkowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shay Covo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shay%2BCovo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShay%2BCovo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shay.covo@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">shay.covo@mail.huji.ac.il</a>
<a href="mailto:einatco@openu.ac.il" class="" data-icon-position="" data-hide-link-title="0">einatco@openu.ac.il</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Einat Hazkani-Covo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Einat%2BHazkani-Covo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hazkani-Covo%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEinat%2BHazkani-Covo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shay.covo@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">shay.covo@mail.huji.ac.il</a>
<a href="mailto:einatco@openu.ac.il" class="" data-icon-position="" data-hide-link-title="0">einatco@openu.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3775" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487100" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487100/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3775.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487100" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487100/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3775.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487100" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487100/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3775.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487100" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487100/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3775.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487100" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487100/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Okazaki fragments that are formed during lagging strand DNA synthesis include an initiating primer consisting of both RNA and DNA. The RNA fragment must be removed before the fragments are joined. In <em>Saccharomyces cerevisiae</em>, a key player in this process is the structure-specific flap endonuclease, <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27p</a> (human homolog FEN1). To obtain a genomic view of the mutational consequence of loss of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">RAD27</a></em>, a <em>S. cerevisiae <a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ strain was subcultured for 25 generations and sequenced using Illumina paired-end sequencing. Out of the 455 changes observed in 10 colonies isolated the two most common types of events were insertions or deletions (INDELs) in simple sequence repeats (SSRs) and INDELs mediated by short direct repeats. Surprisingly, we also detected a previously neglected class of 21 template-switching events. These events were presumably generated by quasi-palindrome to palindrome correction, as well as palindrome elongation. The formation of these events is best explained by folding back of the stalled nascent strand and resumption of DNA synthesis using the same nascent strand as a template. Evidence of quasi-palindrome to palindrome correction that could be generated by template switching appears also in yeast genome evolution. Out of the 455 events, 55 events appeared in multiple isolates; further analysis indicates that these loci are mutational hotspots. Since <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27</a> acts on the lagging strand when the leading strand should not contain any gaps, we propose a mechanism favoring intramolecular strand switching over an intermolecular mechanism. We note that our results open new ways of understanding template switching that occurs during genome instability and evolution.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/template-switching" class="hw-term hw-article-keyword hw-article-keyword-template-switching" rel="nofollow">template switching</a></li><li class="kwd"><a href="/keyword/rad27" class="hw-term hw-article-keyword hw-article-keyword-rad27" rel="nofollow">RAD27</a></li><li class="kwd"><a href="/keyword/okazaki-fragments" class="hw-term hw-article-keyword hw-article-keyword-okazaki-fragments" rel="nofollow">Okazaki fragments</a></li><li class="kwd"><a href="/keyword/inverted-repeats" class="hw-term hw-article-keyword hw-article-keyword-inverted-repeats" rel="nofollow">inverted repeats</a></li><li class="kwd"><a href="/keyword/fen1" class="hw-term hw-article-keyword hw-article-keyword-fen1" rel="nofollow">FEN1</a></li></ul><p id="p-3">Processing Okazaki fragments during the synthesis of the lagging DNA strand is an extremely complex task. On the lagging strand, as the DNA helix is opened up, Pol α-primase first synthesizes an RNA fragment of between 7 and 14 nucleotides and elongates it with a short amount of DNA. Pol δ then extends the primer until it reaches the synthesized downstream fragment, a length of ∼200 nucleotides (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kunkel and Burgers 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nick McElhinny <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Burgers 2009</a>). The polymerase then displaces the RNA primer of the previous Okazaki fragment, resulting in a 5′ single-stranded flap structure (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Burgers 2009</a>). The RNA primers must be removed from the 5′ end of the Okazaki fragments by a nuclease in order to ligate the nearby fragments into a continuous DNA molecule. Three pathways have been suggested to remove the RNA/DNA primer during Okazaki fragment maturation, and all of these pathways involve a structure-specific flap endonuclease, specifically the <em>Saccharomyces cerevisiae</em> <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27</a> or its human homolog FEN1 [reviewed in <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zheng and Shen (2011)</a>]. In the first pathway, RNase H removes most of the RNA moieties, yet frequently one RNA moiety remains attached to the DNA, preventing the ligation of two adjacent Okazaki pieces. The last ribonucleotide is then removed by the 5′ exonuclease activity of FEN1 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Turchi and Bambara 1993</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Turchi <em>et al.</em> 1994</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Waga and Stillman 1994</a>). However, as deletion of both yeast RNase <sup>1</sup>H and RNase H2 is viable, this is probably not the primary mechanism (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Frank <em>et al.</em> 1998</a>). In the second pathway, a short 2–10 nt flap is formed following cleavage by FEN1 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Harrington and Lieber 1994</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bambara <em>et al.</em> 1997</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shen <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stith <em>et al.</em> 2008</a>). In the third pathway, <em><a href="http://www.yeastgenome.org/locus/S000001207/overview">DNA2</a></em> first shortens a long flap, and only then does FEN1 remove the flap (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bae <em>et al.</em> 1998</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2001</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bae and Seo 2000</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Masuda-Sasa <em>et al.</em> 2006</a>). After <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27</a>/FEN1 removes the RNA/DNA primer, a ligatable nick is formed, enabling DNA ligase 1 to immediately seal it. With successful ligation of the nick, pol δ can be recycled to continue the replication of the lagging strand.</p><p id="p-4">Okazaki fragment maturation occurs as the next downstream fragment is already being synthesized, and this process needs to be in coordination with the progress of the replication fork. Even in a simple genome, such as that of <em>S. cerevisiae</em>, Okazaki fragment maturation occurs ∼100,000 times per replication cycle (and tens of million times per replication cycle in the human genome) (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Burgers 2009</a>). Due to this complexity, Okazaki fragment maturation poses a significant challenge to genome stability. However, it is still not clear if a checkpoint mechanism transduces delays in Okazaki fragment maturation and how the cell would act upon such a signal. For these reasons, mutations in the components responsible for Okazaki fragment maturation are likely to cause severe genome instability. Deletion of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">RAD27</a></em> is tolerated in yeast; therefore, it is likely that an alternative but potentially less effective nuclease is capable of processing Okazaki fragments in yeast (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Johnson <em>et al.</em> 1995</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Reagan <em>et al.</em> 1995</a>). Furthermore, it is not clear yet how loss of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">RAD27</a></em> influences instability at the genomic level.</p><p id="p-5">Based on genetic reporter assays that measure changes in the sequence of a single locus, it appears that loss of function of <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27</a> in yeast leads to a unique mutation signature. First, instability of microsatellites (small tandem repeats with units 10–60 bp in length) and mini satellites (larger repeat units) has been observed, and there is a tendency toward insertions rather than deletions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kokoska <em>et al.</em> 1998</a>). Second, <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ strains also accumulate insertions of up to 100 bp flanked by nontandem direct repeats (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tishkoff <em>et al.</em> 1997</a>). Finally, <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ strains show increased mitotic recombination (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Tishkoff <em>et al.</em> 1997</a>). Whether or not the mutational signature of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ is consistent across the entire yeast genome is unknown.</p><p id="p-6">Here, we used whole genome sequencing of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ strains to detect the spectrum of mutations of short (&lt;300 bp) events throughout the entire genome. We confirm that insertions are the most common events in <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ strains, and events in simple repeats are as frequent as insertions mediated by direct repeats. Interestingly, <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ strains undergo template-switching events, resulting in the creation of larger DNA palindromes. These template-switching events can be followed by DNA amplification, as well as the conversion of quasi-palindromic sequences into perfect palindromes. This latter finding was not previously reported, and to our knowledge this is one of the first examples of how whole genome sequencing of genetically modified strains can pinpoint regions that are prone to complex mutations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains</h3><p id="p-7">Analyses were conducted with the <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ haploid strain RJK56 (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kokoska <em>et al.</em> 1998</a>), which has the MS71 background (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Strand <em>et al.</em> 1995</a>). The genotype of RJK56 is <em>MATα <a href="http://www.yeastgenome.org/locus/S000003203/overview">ade5</a>-1 <a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a>-3 <a href="http://www.yeastgenome.org/locus/S000002414/overview">trp1</a>-239 <a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>-52 <a href="http://www.yeastgenome.org/locus/S000000452/overview">his7</a>-2 LEU+ <a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ <em>ρ+</em>. A single colony was picked from the subculture zero plate and streaked for single colonies on YPD. This plate was designated subculture one. After ∼7 d, the cells in the heavy growth were frozen away as subculture one for that colony. Another single colony was then picked and streaked for singles (subculture two). This process was then repeated for five subcultures. Cells from frozen subculture five were struck for single colonies on YPD. After 3 d, a single colony was picked from the plate and streaked for single colonies on YPD. This plate was designated subculture six. After ∼3 d, another single colony was picked and streaked for singles. This process was repeated for 25 subcultures.</p></div><div id="sec-3" class="subsection"><h3>Generation and analysis of high-throughput sequencing data</h3><p id="p-8">Fourteen colonies were sequenced using Illumina paired-end reads: two independent colonies from subculture zero (RJK56-1-0/PG1016, RJK56-2-0/PG1017), two colonies from subculture five (RJK56-1-5/PG1018, RJK56-2-5/PG1019), and 10 colonies from subculture 25 (RJK56-1-25.[1-5]/MD1020-MD1024 and RJK56-2-25.[1-5]/MD1025-MD1029). The DNA was prepared for sequencing using the protocol recommended by Illumina for the HiSeq2000. The samples were sequenced using an Illumina HiSeq2000 machine, generating paired-end reads of 100 bp for the subculture 25 colonies and reads of 93–100 bp for the subculture zero and five colonies. For the 14 sequenced samples, coverage varied from 65- to 289-fold.</p></div><div id="sec-4" class="subsection"><h3>Detecting insertions and deletions (INDELs)</h3><p id="p-9">INDELs were detected using a combination of two methods: Burrows-Wheeler Alignment Tool (BWA, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li and Durbin 2009</a>) was used for reference assembly and Abyss was used for <em>de novo</em> assembly (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Simpson <em>et al.</em> 2009</a>). During BWA assembly, reads were assembled on a previously sequenced and assembled MS71 strain (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Strand <em>et al.</em> 1995</a>), which was assembled with S288c as a reference. Since MS71 shows hybrid regions that resemble both S288c and YJM789, we decided not to assemble directly on the S288c sequence. However, all positions in our analysis are presented here in SGD coordinates. SAMtools was then used to extract “pileup” files out of the assembled files (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2009</a>), creating files that show the number of bases supported at each position. Only positions that did not overlap repeats, as determined by RepeatMasker using the <em>S. cerevisiae</em> library (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jurka 2000</a>), and positions that passed the vcf-annotate script from VCF-tools (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danecek <em>et al.</em> 2011</a>) were considered. During Abyss assembly, since the reference assembly can identify relatively small INDELs, we applied Abyss <em>de novo</em> assembly for our data. Each of the 14 strains was analyzed with multiple <em>C</em> parameters, and the <em>C</em> value was chosen that had resulted in the combination of the contig of maximal length and of the highest N50 (the N50 is the size of the smallest of all the large contigs covering 50% of the genome). For each strain, the Abyss Scaffold file was then masked with RepeatMasker (<em>e.g.</em>, Tys and LTRs) using the <em>S. cerevisiae</em> library and Lagan.pl (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brudno <em>et al.</em> 2003</a>) was used to place each of the <em>de novo</em> contigs into the reference chromosomes. For each strain, MAFFT (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Katoh <em>et al.</em> 2009</a>) was then used to align the contigs to chromosomes and produce an alignment. Finally, for each of the chromosomes, all strains were aligned using MAFFT with profile mode, and INDELs were extracted using in house PERL scripts. The final alignment was presented using SEAVIEW viewer (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gouy <em>et al.</em> 2010</a>).</p><p id="p-10">We looked for events that occurred in at least one of the strains that was subcultured for 25 generations but not shared among all of the subculture 25 strains. Abyss Events are only predicted if at least one of the subculture zero or five strains were assembled in the region. INDELs in telomeric regions and INDELs that overlapped Ns were also ignored. Small (&lt;9) Abyss events were selected if they were supported by BWA (even if they did not pass the strict classification in the first list). Integrative Genomic Viewer (IGV, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Robinson <em>et al.</em> 2011</a>) was used to manually inspect all cases.</p><p id="p-11">Each locus with an INDEL was assigned to the following classifications: (1) INDELs in SSRs (<em>e.g.</em>, mononucleotides, dinucleotide, <em>etc</em>.); (2) INDELs mediated by short direct repeats; (3) INDELs involved in palindromic expansion; (4) clusters of INDELs or clusters of INDELs and single nucleotide polymorphisms (SNPs) that can be explained by a quasi-palindrome correction; or (5) other INDELs. Each of these classes is further grouped to separate insertions, deletions, or unknown/mixed mutations.</p><p id="p-12">Classification of the INDELs was performed using in house PERL scripts and a combination of the following programs: Repeatmasker, tandem repeat finder (TDF, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benson 1999</a>), EMBOSS palindrome (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rice <em>et al.</em> 2000</a>), and BLAST of a sequence against itself (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>). In the classifications of duplications/deletions mediated by a direct repeat, events are either of the type <em>ABA</em> to <em>ABABA</em> (insertion) or <em>ABABA</em> to <em>ABA</em> (deletion), where <em>A</em> is the direct repeat and <em>B</em> is a unique sequence (<em>A</em>, <em>B</em> are regions of DNA). The program TDF was used to identify A and B regions. The length of A+B should equal the length of the INDEL. If TDF revealed multiple choices that fit the length of the INDEL, the duplications with the highest similarity between copies was chosen. Therefore, events in nonperfect simple repeat regions can end up as insertion mediated by tandem repeat.</p></div><div id="sec-5" class="subsection"><h3>Detecting mutations</h3><p id="p-13">Mutations were identified by two reference assembly methods: BWA and CLC Genomics. BWA mutations were identified as described above for INDELs. CLC Genomic Workbench (currently Qiagen) was also used to identify mutations. Reads from each of the 14 strains were assembled independently to the MS71 reference genome, and variant calling was performed with a minimum variant calling of 90%. A “filter against control reads” analysis was used to filter events shared with subculture five strains. IGV viewer (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Robinson <em>et al.</em> 2011</a>) was used to eliminate cases with SNPs that were shared by all strains by manually inspecting all events. SNPs in clusters with and without INDELs were tested by EMBOSS palindrome (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Rice <em>et al.</em> 2000</a>) and BLAST (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) for the identification of palindromes and inverted repeats (IRs).</p></div><div id="sec-6" class="subsection"><h3>Detecting template switching in Saccharomyces evolution</h3><p id="p-14">Inverted repeats with an arm length of at least five nucleotides and spacer up to 70 bp were identified in S288c coding genome using the EMBOSS palindrome package (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Rice <em>et al.</em> 2000</a>). An orthologous region for each IR was detected in the following <em>Saccharomyces</em> genomes: <em>S. eubayanus</em> (GCF_001298625.1), <em>S. uvarum</em> (GCA_000732305.1), <em>S. arboricola</em> (GCF_000292725.1), <em>S. kudriavzevii</em> (GCA_000167075.2), <em>S. mikatae</em> (GCA_000167055.1), and <em>S. paradoxus</em> (GCA_000166955.1). BLAST was used to identify orthologs (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) requiring at least one additional gene from each side of the query genes. Multiple sequence alignments (MSAs) were reconstructed using MAFFT V3.705 (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Katoh <em>et al.</em> 2009</a>). Finally, the number of perfect but different IR variants in an orthologous region was detected and compared to the number of different perfect IRs in 1000 simulated MSAs. Note, the evolutionary simulation was performed without a template-switching mechanism.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-15">Strains are available upon request. A list of all events reported is provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300262/-/DC1/FileS1.xlsx">File S1</a>. The sequencing data have been deposited in the National Center for Biotechnology Information Sequence Read Archive (accession no. SRP117788).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-16">Mutations in <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">RAD27</a></em> and other strains are typically studied using a single gene for which forward mutations can be selected. The rate of changes in this one gene, as well as the spectrum of mutations, is assumed to be consistent across the entire genome. Here, we used whole genome sequencing of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> strains to directly test the mutation spectrum. A single <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> haploid colony (RJK56, <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kokoska <em>et al.</em> 1998</a>) was picked from a subculture zero plate streaked for single colonies on YPD that was grown for ∼7 d. A colony from this plate was designated subculture one. Restreaking and growth on YPD was repeated until subculture five was obtained. Five independent colonies were picked from the subculture five plate, and each of these colonies was further streaked until subculture 25, growing each subculture for ∼3 d. Using Illumina deep sequencing technology, we sequenced two separate lines, each with the following seven colonies: one from subculture zero, one from subculture five, and five from subculture 25, for a total of 14 strains (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). HiSeq2000 was used to analyze the data. The genomic analysis focused on the spectrum of events ranging in size from 1 to 300 bp.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The experimental design of this study. The initiator strain was the RJK56 rad27&#x394; haploid (Kokoska et al. 1998). Two subculture zero plates were used and streaked for single colonies on YPD and grown for &#x223C;7 d. One colony from each of these plates was designated subculture one, and, after growing and streaking for singles, one colony from each plate was picked to be grown as subculture two. We repeated this process until subculture five. After subculture five was plated for single colonies, five independent colonies were picked, and the process was repeated until subculture 25, with each subculture grown for &#x223C;3 d. Using a HiSeq2000 machine, we sequenced two lines, each with seven colonies, for a total of 14 strains. Each line contained one sample from subculture zero, one sample from subculture five, and five samples from subculture 25." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The experimental design of this study. The initiator strain was the RJK56 &lt;em&gt;rad27&lt;/em&gt;&#x394; haploid (Kokoska &lt;em&gt;et al.&lt;/em&gt; 1998). Two subculture zero plates were used and streaked for single colonies on YPD and grown for &#x223C;7 d. One colony from each of these plates was designated subculture one, and, after growing and streaking for singles, one colony from each plate was picked to be grown as subculture two. We repeated this process until subculture five. After subculture five was plated for single colonies, five independent colonies were picked, and the process was repeated until subculture 25, with each subculture grown for &#x223C;3 d. Using a HiSeq2000 machine, we sequenced two lines, each with seven colonies, for a total of 14 strains. Each line contained one sample from subculture zero, one sample from subculture five, and five samples from subculture 25.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F1.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3775/F1.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487186" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">The experimental design of this study. The initiator strain was the RJK56 <em>rad27</em>Δ haploid (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Kokoska <em>et al.</em> 1998</a>). Two subculture zero plates were used and streaked for single colonies on YPD and grown for ∼7 d. One colony from each of these plates was designated subculture one, and, after growing and streaking for singles, one colony from each plate was picked to be grown as subculture two. We repeated this process until subculture five. After subculture five was plated for single colonies, five independent colonies were picked, and the process was repeated until subculture 25, with each subculture grown for ∼3 d. Using a HiSeq2000 machine, we sequenced two lines, each with seven colonies, for a total of 14 strains. Each line contained one sample from subculture zero, one sample from subculture five, and five samples from subculture 25.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We identified 455 loci with changes that occurred in at least one of the strains that was subcultured for 25 generations but were not shared among all of the subculture 25 strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300262/-/DC1/FileS1.xlsx">File S1</a>). The events were classified to different mutation categories as described below. The rate of mutation for each category was calculated as rate per colony/generation/base pair. The information is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300262/-/DC1/FileS1.xlsx">File S1</a>. Importantly, we provide a minimal estimation for the rates because some of the events can be reversal and some of them that present in multiple strains could have been created independently. The majority of events we observed were INDELs. Two types of INDELs were most common: INDELs in simple sequence repeats (SSRs, 183 loci) and INDELs mediated by short direct repeats (177 loci). An additional class consisted of INDELs and SNPs that are presumably the result of template-switching events, a class that was not previously associated with <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> strains (21 events). Other events were composed of SNPs and gene conversions between paralogous genes.</p><p id="p-19">Among INDELs, insertions are more common than deletions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Most events were unique, <em>i.e.</em>, they appeared in only one strain; however, 51 of the INDELs and 4 of the template-switching loci were found with events in multiple strains, indicating that these loci are prone to changes. Here we focus on INDELS mediated by short directed repeats (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Tishkoff <em>et al.</em> 1997</a>) and template switching. The full list of all events depicted are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300262/-/DC1/FileS1.xlsx">File S1</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stacked histogram depicting the lengths of 338 INDELs that appear only once in the 10 rad27&#x394; subculture-25 strains. The different classes of events include insertions (black) and deletions (gray) in SSRs, insertions (red) and deletions (orange) mediated by direct repeats, insertions (blue) and deletions in template-switching events, and other insertions (green)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Stacked histogram depicting the lengths of 338 INDELs that appear only once in the 10 &lt;em&gt;rad27&lt;/em&gt;&#x394; subculture-25 strains. The different classes of events include insertions (black) and deletions (gray) in SSRs, insertions (red) and deletions (orange) mediated by direct repeats, insertions (blue) and deletions in template-switching events, and other insertions (green).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F2.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3775/F2.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487170" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Stacked histogram depicting the lengths of 338 INDELs that appear only once in the 10 <em>rad27</em>Δ subculture-25 strains. The different classes of events include insertions (black) and deletions (gray) in SSRs, insertions (red) and deletions (orange) mediated by direct repeats, insertions (blue) and deletions in template-switching events, and other insertions (green).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-9" class="subsection"><h3>rad27∆ strains have hotspots for changes</h3><p id="p-21">In the case of insertions mediated by a direct repeat, the nonmutated, original locus is the form <em>ABA</em>, where <em>B</em> is a unique stretch of DNA sequence flanked by two short direct repeats, marked as <em>A</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, bottom sequence). After the insertion has occurred, the mutated locus would be in the form of <em>ABABA</em> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, top sequence). In the case of deletions, the opposite scenario occurs; <em>ABABA</em> is transformed to <em>ABA</em>. For insertion events, the size of the unique region <em>B</em> can range in size between 1 and 91 bp, and the size of the repeat <em>A</em> can range between 1 and 21 bp. This is a somewhat larger size range compared to the previously detected (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Tishkoff <em>et al.</em> 1997</a>) 5–108 bp, which was mediated by a repeat of 3–12 bp.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) An example for an insertion mediated by a direct repeat. The top sequence in this pairwise alignment includes an insertion of 51 bp compared to the other strains (represented here by a single strain MD1020). The bottom sequence represents the ancestral form with the structure of ABA, and the top sequence with the insertion has the structure of ABABA (SGD position 470,500 on chrXIII). (B) Three types of changes at ChrII:392888 that were observed in rad27&#x394; strains. A unit of 16 bp composed of an 8-bp direct repeat (indicated as A) and an 8-bp unique sequence (indicated as B). Most strains, as well as the two zero and two five-strains, included the ancestral form of three copies of AB (represented here by strain MD1021). However, two independent strains (MD1020, MD1028) included four copies of AB, and one strain (MD1029) included two copies of AB. S. cerevisiae strain JAY291 has one AB unit, and YJM789 contained three AB units in this locus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) An example for an insertion mediated by a direct repeat. The top sequence in this pairwise alignment includes an insertion of 51 bp compared to the other strains (represented here by a single strain MD1020). The bottom sequence represents the ancestral form with the structure of &lt;em&gt;ABA&lt;/em&gt;, and the top sequence with the insertion has the structure of &lt;em&gt;ABABA&lt;/em&gt; (SGD position 470,500 on chrXIII). (B) Three types of changes at ChrII:392888 that were observed in &lt;em&gt;rad27&lt;/em&gt;&#x394; strains. A unit of 16 bp composed of an 8-bp direct repeat (indicated as &lt;em&gt;A&lt;/em&gt;) and an 8-bp unique sequence (indicated as &lt;em&gt;B&lt;/em&gt;). Most strains, as well as the two zero and two five-strains, included the ancestral form of three copies of &lt;em&gt;AB&lt;/em&gt; (represented here by strain MD1021). However, two independent strains (MD1020, MD1028) included four copies of &lt;em&gt;AB&lt;/em&gt;, and one strain (MD1029) included two copies of &lt;em&gt;AB&lt;/em&gt;. &lt;em&gt;S. cerevisiae&lt;/em&gt; strain JAY291 has one &lt;em&gt;AB&lt;/em&gt; unit, and YJM789 contained three &lt;em&gt;AB&lt;/em&gt; units in this locus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F3.medium.gif" width="440" height="100"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/11/3775/F3.medium.gif" width="440" height="100"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487187" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">(A) An example for an insertion mediated by a direct repeat. The top sequence in this pairwise alignment includes an insertion of 51 bp compared to the other strains (represented here by a single strain MD1020). The bottom sequence represents the ancestral form with the structure of <em>ABA</em>, and the top sequence with the insertion has the structure of <em>ABABA</em> (SGD position 470,500 on chrXIII). (B) Three types of changes at ChrII:392888 that were observed in <em>rad27</em>Δ strains. A unit of 16 bp composed of an 8-bp direct repeat (indicated as <em>A</em>) and an 8-bp unique sequence (indicated as <em>B</em>). Most strains, as well as the two zero and two five-strains, included the ancestral form of three copies of <em>AB</em> (represented here by strain MD1021). However, two independent strains (MD1020, MD1028) included four copies of <em>AB</em>, and one strain (MD1029) included two copies of <em>AB</em>. <em>S. cerevisiae</em> strain JAY291 has one <em>AB</em> unit, and YJM789 contained three <em>AB</em> units in this locus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">One interesting example occurs around position 392,888 on chrII, between the <em>SML4</em> and <em><a href="http://www.yeastgenome.org/locus/S000000282/overview">ECM33</a></em> genes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). This locus includes three variants with two to four copies of a 16-bp AB unit. This example, verified by PCR, contains duplications of a 16-bp unit mediated by a 8-bp direct repeat (A) in two independent strains (RJK56-1-25.1/MD1020, RJK56-2-25.4/MD1028) and a deletion of 16 bp mediated by a direct repeat in an additional strain (RJK56-2-25.5/MD1029). Moreover, BLAST analysis of additional <em>S. cerevisiae</em> strains indicated that variation in this locus is common and that different wild-type strains have variation in the number of repeats. For example, one single AB copy is observed in the JAY291 strain that is used in bioethanol production, and three AB copies are observed in the YJM789 strain that was derived from the lung of an AIDS patient (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wei <em>et al.</em> 2007</a>).</p><p id="p-24">The appearance of identical events in the two independent <em>S</em>. <em>cerevisiae</em> lines is striking. The majority of events that appear in multiple strains are insertions mediated by direct repeats (29); 17 of the events are in SSR and 4 are template-switching events (see below). We analyzed 16 events that appear in multiple strains of which 13 were PCR-validated to appear in multi strains; one was identified to appear only in one strain, while another two events we analyzed were not resolved. Since events repeated in several isolates we hypothesized that the sites are prone to changes. To test this hypothesis, we streaked for singles colonies from subcultures zero and five from both lines and randomly picked 10 colonies from each of the four, a total of 40 colonies. In 70% of the 13 cases, at least one of the picked colonies showed an alternative length of the PCR product. As can be seen in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4B</a>, we could find evidence for sectoring: both the original allele and an evolved one appear in a culture grown from a single colony isolate. Thus, these events are highly unstable and during subculturing they can appear and disappear (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). To the best of our knowledge, this is one of the first examples of how NGS data provide prediction to loci of structural variability in the population.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structural variation hotspots in rad27&#x394; strains. (A) Several loci were shown to be different from the consensus sequence in multiple strains. A pie chart showing the number of variants differ from the consensus strain among 10 randomly picked colonies that were subcultured from two zero- and two five-strains in these loci is presented. In 70% of the 13 cases, at least one of the picked colonies showed the alternative length of a PCR product. (B) An example of an alternative length PCR product of one of the isolates from subculture zero at the locus 72410 on Chromosome II." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structural variation hotspots in &lt;em&gt;rad27&lt;/em&gt;&#x394; strains. (A) Several loci were shown to be different from the consensus sequence in multiple strains. A pie chart showing the number of variants differ from the consensus strain among 10 randomly picked colonies that were subcultured from two zero- and two five-strains in these loci is presented. In 70% of the 13 cases, at least one of the picked colonies showed the alternative length of a PCR product. (B) An example of an alternative length PCR product of one of the isolates from subculture zero at the locus 72410 on Chromosome II.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F4.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/11/3775/F4.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487185" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Structural variation hotspots in <em>rad27</em>Δ strains. (A) Several loci were shown to be different from the consensus sequence in multiple strains. A pie chart showing the number of variants differ from the consensus strain among 10 randomly picked colonies that were subcultured from two zero- and two five-strains in these loci is presented. In 70% of the 13 cases, at least one of the picked colonies showed the alternative length of a PCR product. (B) An example of an alternative length PCR product of one of the isolates from subculture zero at the locus 72410 on Chromosome II.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>rad27∆ strains show template-switching events</h3><p id="p-26">A template-switching mechanism where the replication fork presumably hops between templates on two DNA strands results in the most interesting class of events. Template-switching mediated events were found at a total of 21 loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300262/-/DC1/FileS1.xlsx">File S1</a>), of which 17 events were unique, and four events were detected in multiple strains. We verified by Sanger sequencing or PCR 15/16 unique template-switching events. The four template-switching events appearing in multiple strains were also verified as noted before. These events resulted in the following outcomes: (a) a cluster of SNPs and a combination of INDELs and SNPs that was formed by template switching between either the arms of imperfect IRs or quasi-palindromes, leading to the creation of a perfect IR or palindrome (10 events); (b) long insertions that were created by a combination of palindrome extensions and DNA amplification (six unique events, four events in multiple strains); and (c) palindrome expansion followed by exonuclease activity without DNA amplification (one event).</p><p id="p-27">Quasi-palindrome is a palindromic or IR sequence that bears mutations in one of the arms. A simple example is <em>5′AGAACAxxxxTcTTCT 3′</em> where <em>c</em> shows a mismatch between arms and <em>x</em> represents a spacer base. A conversion of this quasi-palindrome to a perfect palindrome means an appearance of identical arms in inverted orientations, thus 5′ AGAACAxxxxTGTTCT 3′. A mechanism of template switching resulting in a quasi-palindrome to perfect palindrome correction is illustrated in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. Two subculture 25 <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> strains are shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>. The top sequence is mutated and has a perfect IR with identical arms of 16 bp separated by a spacer of 1 bp. The bottom strain, which represents the original sequence, has an imperfect IR. The arms of the imperfect palindrome are different from each other in three points (one mismatch and a deletion of two bases, all of which are shown in large font in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). The most parsimonious way to explain how the top sequence originated is that the replication fork stalled and switched templates; the stalled replication fork folded back used the nascent DNA strand as a template. As shown in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>, replication starts using a template with the sequence of the bottom strain in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a>. The newly synthesized strand replicates the template of the 3′ arm of the IR and then begins replicating the second arm but stalls. Next, the newly synthesized strand forms a secondary structure, resulting in a template switch, where the recently synthesized IR arm is used as the template (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>). Finally, the replication fork realigns to the legitimate template (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5D</a>). At the end of this process, the bottom strand in <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5D</a> would resemble the top and mutated strain in <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5A</a>, but the top strand on <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5D</a> would resemble the bottom and original strain in <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5A</a>. At the next round of replication, two different daughter cells will be produced, one with and one without a perfect IR (as shown in <a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quasi-palindrome to palindrome correction in rad27&#x394; strains. (A) Variation between two rad27&#x394; strains from subculture 25. The top sequence represents the mutated strain that includes a perfect palindrome, and the bottom sequence represents the original (nonmutated) strain that includes a quasi-palindrome. (B) Replication starts on a template similar to the bottom strain in A. The fork synthesizes one arm and starts synthesis on the second arm (C). Then, the newly synthesized strand folds back using the homology between the arms. A template switch occurs, causing the nascent strand to use itself as a template, eliminating variation between the arms of the palindrome. Finally, (D) the nascent strand returns to the correct template. The next round of replication results in two daughter cells that resemble the variants depicted in A. Arrows indicate palindromes, and bases showing the differences between arms are shown in large font. The red strand is the nascent strand and the blue strand is the original template strand." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quasi-palindrome to palindrome correction in &lt;em&gt;rad27&lt;/em&gt;&#x394; strains. (A) Variation between two &lt;em&gt;rad27&lt;/em&gt;&#x394; strains from subculture 25. The top sequence represents the mutated strain that includes a perfect palindrome, and the bottom sequence represents the original (nonmutated) strain that includes a quasi-palindrome. (B) Replication starts on a template similar to the bottom strain in A. The fork synthesizes one arm and starts synthesis on the second arm (C). Then, the newly synthesized strand folds back using the homology between the arms. A template switch occurs, causing the nascent strand to use itself as a template, eliminating variation between the arms of the palindrome. Finally, (D) the nascent strand returns to the correct template. The next round of replication results in two daughter cells that resemble the variants depicted in A. Arrows indicate palindromes, and bases showing the differences between arms are shown in large font. The red strand is the nascent strand and the blue strand is the original template strand.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F5.medium.gif" width="349" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/11/3775/F5.medium.gif" width="349" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487184" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Quasi-palindrome to palindrome correction in <em>rad27</em>Δ strains. (A) Variation between two <em>rad27</em>Δ strains from subculture 25. The top sequence represents the mutated strain that includes a perfect palindrome, and the bottom sequence represents the original (nonmutated) strain that includes a quasi-palindrome. (B) Replication starts on a template similar to the bottom strain in A. The fork synthesizes one arm and starts synthesis on the second arm (C). Then, the newly synthesized strand folds back using the homology between the arms. A template switch occurs, causing the nascent strand to use itself as a template, eliminating variation between the arms of the palindrome. Finally, (D) the nascent strand returns to the correct template. The next round of replication results in two daughter cells that resemble the variants depicted in A. Arrows indicate palindromes, and bases showing the differences between arms are shown in large font. The red strand is the nascent strand and the blue strand is the original template strand.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">We also found template-switching events that caused DNA amplification and extension of the palindrome. The ratio of the length between the palindrome or IR expansion and the amount of DNA amplification varied between the events, depending on the point where the replicated strand returned to the original template. In some examples, like in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, the palindrome expansion segment is bigger than the amount of DNA amplification, while in other events, like <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, the opposite relationship was observed.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Palindrome expansion occurs via template switching in rad27&#x394; strains. Insertion at ChrII 715496. (A) The top line includes the mutated strain with an insertion mediated by a palindrome; the bottom line is the original sequence. Arrows indicate palindromes, and brackets indicate duplications. Numbers indicate important points in the mechanism. (B) DNA synthesis starts on a template similar to the bottom line in A and continues until the fork is stalled (point 1). (C) At point 2, the DNA folds back (template switch #1) and synthesizes until it arrives at point 3. (D) The synthesized strand aligns back to the chromosome at point 4 (template switch #2) and (E) continues synthesis in the original direction. The top strand resembles the bottom and original strain in A, and the bottom strand resembles the top and mutated strain in A. The next round of replication results in two daughter cells with the rearrangements depicted in A. The red strand is the nascent strand and the blue strand is the original template strand." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Palindrome expansion occurs via template switching in &lt;em&gt;rad27&lt;/em&gt;&#x394; strains. Insertion at ChrII 715496. (A) The top line includes the mutated strain with an insertion mediated by a palindrome; the bottom line is the original sequence. Arrows indicate palindromes, and brackets indicate duplications. Numbers indicate important points in the mechanism. (B) DNA synthesis starts on a template similar to the bottom line in A and continues until the fork is stalled (point 1). (C) At point 2, the DNA folds back (template switch #1) and synthesizes until it arrives at point 3. (D) The synthesized strand aligns back to the chromosome at point 4 (template switch #2) and (E) continues synthesis in the original direction. The top strand resembles the bottom and original strain in A, and the bottom strand resembles the top and mutated strain in A. The next round of replication results in two daughter cells with the rearrangements depicted in A. The red strand is the nascent strand and the blue strand is the original template strand.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F6.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/11/3775/F6.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487128" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-30" class="first-child">Palindrome expansion occurs via template switching in <em>rad27</em>Δ strains. Insertion at ChrII 715496. (A) The top line includes the mutated strain with an insertion mediated by a palindrome; the bottom line is the original sequence. Arrows indicate palindromes, and brackets indicate duplications. Numbers indicate important points in the mechanism. (B) DNA synthesis starts on a template similar to the bottom line in A and continues until the fork is stalled (point 1). (C) At point 2, the DNA folds back (template switch #1) and synthesizes until it arrives at point 3. (D) The synthesized strand aligns back to the chromosome at point 4 (template switch #2) and (E) continues synthesis in the original direction. The top strand resembles the bottom and original strain in A, and the bottom strand resembles the top and mutated strain in A. The next round of replication results in two daughter cells with the rearrangements depicted in A. The red strand is the nascent strand and the blue strand is the original template strand.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Palindrome-mediated DNA amplification occurs via template switching in rad27&#x394; strains. In this example, found at chrVII: 657,551, DNA amplification is the dominant outcome. (A) Two different subculture 25 strains are shown: the top line is the mutated sequence and includes an insertion mediated by a palindrome, and the bottom line is the original sequence. Arrows indicate palindromes, brackets indicate duplication, and numbers indicate important points in the mechanism. (B) DNA synthesis starts on a template similar to the bottom line in A and continues until the fork is stalled (point 1). (C) At point 2, the DNA folds back (template switch #1) (D) and synthesizes until it arrives at point 3. (E) The nascent strand aligns back to the top strand (template switch #2) at point 4 (F) and continues synthesis in the original direction. The top strand resembles the bottom and original sequence in A, and the bottom strand resembles the top and mutated sequence in A. The next round of replication results in two daughter cells with the rearrangements depicted in A. The red strand is the nascent strand and the blue strand is the original template strand." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Palindrome-mediated DNA amplification occurs via template switching in &lt;em&gt;rad27&lt;/em&gt;&#x394; strains. In this example, found at chrVII: 657,551, DNA amplification is the dominant outcome. (A) Two different subculture 25 strains are shown: the top line is the mutated sequence and includes an insertion mediated by a palindrome, and the bottom line is the original sequence. Arrows indicate palindromes, brackets indicate duplication, and numbers indicate important points in the mechanism. (B) DNA synthesis starts on a template similar to the bottom line in A and continues until the fork is stalled (point 1). (C) At point 2, the DNA folds back (template switch #1) (D) and synthesizes until it arrives at point 3. (E) The nascent strand aligns back to the top strand (template switch #2) at point 4 (F) and continues synthesis in the original direction. The top strand resembles the bottom and original sequence in A, and the bottom strand resembles the top and mutated sequence in A. The next round of replication results in two daughter cells with the rearrangements depicted in A. The red strand is the nascent strand and the blue strand is the original template strand.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F7.medium.gif" width="311" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/11/3775/F7.medium.gif" width="311" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487153" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-31" class="first-child">Palindrome-mediated DNA amplification occurs via template switching in <em>rad27</em>Δ strains. In this example, found at chrVII: 657,551, DNA amplification is the dominant outcome. (A) Two different subculture 25 strains are shown: the top line is the mutated sequence and includes an insertion mediated by a palindrome, and the bottom line is the original sequence. Arrows indicate palindromes, brackets indicate duplication, and numbers indicate important points in the mechanism. (B) DNA synthesis starts on a template similar to the bottom line in A and continues until the fork is stalled (point 1). (C) At point 2, the DNA folds back (template switch #1) (D) and synthesizes until it arrives at point 3. (E) The nascent strand aligns back to the top strand (template switch #2) at point 4 (F) and continues synthesis in the original direction. The top strand resembles the bottom and original sequence in A, and the bottom strand resembles the top and mutated sequence in A. The next round of replication results in two daughter cells with the rearrangements depicted in A. The red strand is the nascent strand and the blue strand is the original template strand.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">In <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>, two variants of subculture 25 <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> strains are depicted; the top strain contained the mutated sequence, which includes an insertion of 66 bp, the result of palindrome expansion. The bottom strain, which represents the original sequence, includes a palindrome of 15 bp with a center at point 2. The mechanism of mutation generation here is best explained by a folding-back mechanism that involves two steps of template switching. First, while replicating on a template with the sequence of the bottom strain, the replication fork is presumably blocked at point 1 (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). The nascent strand is then folded back to the center of a small, 15-bp palindrome (point 2, <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>) and synthesis occurs until point 3. After replicating a total of 55 bp, the nascent strand realigns back to the original template (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6D</a>) and replication resumes using the original template (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6E</a>). This transaction results in a 66-bp insertion and a short duplication. At the end of this process, the bottom strand in <a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6E</a> would resemble the top and mutated strain in <a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6A</a>, while the top strand on <a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6E</a> would resemble the bottom and original strain on <a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6A</a>. The next round of replication will result in two daughter cells with structures similar to the two forms in <a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6A</a>.</p><p id="p-33">A similar event was found at the <em><a href="http://www.yeastgenome.org/locus/S000003321/overview">NNF2</a></em> gene, near <a href="http://www.yeastgenome.org/locus/S000118463/overview">ARS727</a>, and is shown in <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>. Here, a duplication makes up a larger part of the insertion, although palindrome expansion is also involved. The top sequence in <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a> represents that of a mutated strain from subculture 25 with an insertion, and the bottom sequence in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a> represents the original form prior to insertion and was found in the other subculture 25 strains. First, while replicating on a template similar to the bottom strain in <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7A</a>, the replication fork stalls (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7B</a>, point 1 in <a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7A</a>). Next, the nascent strand folds back (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7C</a>, point 2 in <a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7A</a>) forming a stem loop structure due to imperfect IRs that are centered at point 2. DNA synthesis continues up to point 3 (<a id="xref-fig-7-10" class="xref-fig" href="#F7">Figure 7D</a>). The fork realigns to the original template at point 4 (<a id="xref-fig-7-11" class="xref-fig" href="#F7">Figure 7E</a>) and replication resumes (<a id="xref-fig-7-12" class="xref-fig" href="#F7">Figure 7F</a>). At the end of the process, the bottom strand will resemble the top and mutated strain on <a id="xref-fig-7-13" class="xref-fig" href="#F7">Figure 7A</a>, while the top strand on <a id="xref-fig-7-14" class="xref-fig" href="#F7">Figure 7F</a> will resemble the bottom and original strain in <a id="xref-fig-7-15" class="xref-fig" href="#F7">Figure 7A</a>. The next round of replication will result in a daughter cell with the insertion. We note that it is also possible that the fork does not return to the native template at point 4 but may fold back on itself a second time, using the short IRs (indicated in <a id="xref-fig-7-16" class="xref-fig" href="#F7">Figure 7A</a>). In this case, the fork would realign to the original template strand at point 5. Both options lead to the same final sequence.</p><p id="p-34">A fourth example of template switching without DNA amplification is shown in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>. Two subculture 25 strains are shown in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>. The top strain represents the mutated strain including a palindromic insertion, and the bottom strain in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8A</a> represents the original sequence. Here, the original sequence includes 17 bases that are missing in the mutant strain. The most parsimonious explanation is that these 17 bases were deleted by an exonuclease during the process of palindrome expansion. Removal of these bases enables the newly synthesized strand to fold back and form a secondary structure thus switching template and replicating the same strand resulting in palindromic expansion. The process leads to a bottom strand similar to the mutated strain and a top strand similar to the original strain. The next round of replication will result in two daughter cells with these different genotypes.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nascent strand processing allows palindrome amplification via template switching in rad27&#x394; strains. (A) Two different subculture 25 strains are depicted, and the top sequence is mutated and includes a palindrome expansion. The bottom sequence is unchanged and includes an additional 17 bp (boxed). Arrows indicate palindromes. (B) DNA synthesis starts on a template similar to bottom sequence in A and continues until the fork is stalled. (C and D) An exonuclease then removes 17 bases. (E) The newly synthesized strand folds due to the homology between arms. A template switching occurs causing the nascent strand to use the recently synthesized arm as a template, expanding the palindrome. (F) The two DNA strands now differ in the area of expansion: the top strand resembles the bottom and original sequence in A, and the bottom strand resembles the top and mutated sequence in A. The next round of replication results in two daughter cells with the sequence changes that appear in A. The red strand is the nascent strand and the blue strand is the original template strand." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nascent strand processing allows palindrome amplification via template switching in &lt;em&gt;rad27&lt;/em&gt;&#x394; strains. (A) Two different subculture 25 strains are depicted, and the top sequence is mutated and includes a palindrome expansion. The bottom sequence is unchanged and includes an additional 17 bp (boxed). Arrows indicate palindromes. (B) DNA synthesis starts on a template similar to bottom sequence in A and continues until the fork is stalled. (C and D) An exonuclease then removes 17 bases. (E) The newly synthesized strand folds due to the homology between arms. A template switching occurs causing the nascent strand to use the recently synthesized arm as a template, expanding the palindrome. (F) The two DNA strands now differ in the area of expansion: the top strand resembles the bottom and original sequence in A, and the bottom strand resembles the top and mutated sequence in A. The next round of replication results in two daughter cells with the sequence changes that appear in A. The red strand is the nascent strand and the blue strand is the original template strand.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F8.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/11/3775/F8.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487137" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-35" class="first-child">Nascent strand processing allows palindrome amplification via template switching in <em>rad27</em>Δ strains. (A) Two different subculture 25 strains are depicted, and the top sequence is mutated and includes a palindrome expansion. The bottom sequence is unchanged and includes an additional 17 bp (boxed). Arrows indicate palindromes. (B) DNA synthesis starts on a template similar to bottom sequence in A and continues until the fork is stalled. (C and D) An exonuclease then removes 17 bases. (E) The newly synthesized strand folds due to the homology between arms. A template switching occurs causing the nascent strand to use the recently synthesized arm as a template, expanding the palindrome. (F) The two DNA strands now differ in the area of expansion: the top strand resembles the bottom and original sequence in A, and the bottom strand resembles the top and mutated sequence in A. The next round of replication results in two daughter cells with the sequence changes that appear in A. The red strand is the nascent strand and the blue strand is the original template strand.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Template-switching events through Saccharomyces evolution</h3><p id="p-36">We found 21 events of template switching in <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> isolates. Such events were not reported for WT yeasts (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nishant <em>et al.</em> 2010</a>). Is template switching a phenomenon affecting normal yeast evolution? To answer this question, we looked for evidence of template switching causing quasi-palindrome to palindrome correction resulting in different but perfect orthologous IRs through <em>Saccharomyces</em> genes. We searched the SGD S288c reference genome for perfect IRs with arms of at least 5 bp and a spacer of a maximum 70 bp. For each perfect IR in S228c genes, we looked for the appearance of a slightly different perfect repeats in orthologous loci in seven fully sequenced <em>Saccharomyces</em> genomes with known phylogeny (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>). Out of 49,912 perfect palindromes in 543 genes, 3741 in 502 genes appear with at least two forms through phylogeny. This number includes 3 loci that appear in 4 forms through phylogeny (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a>), and 92 that appear in 3 forms through phylogeny (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9C</a>). When comparing to an evolutionary simulation assuming no template-switching mechanism, 431 out of the 502 were statistically significant (<em>p</em> value &lt;0.05). The meaning of such findings is that independent events of quasi-palindrome to palindrome conversion had occurred multiple times in the same locus during evolution. Thus, template switching is not only significant in the case of instability conditions, as in the case of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em>, but also as a source of variation throughout normal yeast evolution.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evidence for quasi-palindrome to palindrome conversion via template switching during evolution of S. cerevisiae. (A) A phylogenetic tree of seven Saccharomyces genomes. (B) Three perfect palindromes appear in S288c three coding regions. Different perfect palindromes appear at the same loci in three additional genomes. Each locus is shown in a column and different colors show different palindrome forms. Loci that are in a quasi-palindromic state are shown in gray. Cases in which the loci could not be identified are shown in black. Positions identical to S288c are shown with a dot and the spacer between arms is represented with a tilde sign. (C) Ninety-two S288c loci that appear in two additional genomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evidence for quasi-palindrome to palindrome conversion via template switching during evolution of &lt;em&gt;S. cerevisiae&lt;/em&gt;. (A) A phylogenetic tree of seven &lt;em&gt;Saccharomyces&lt;/em&gt; genomes. (B) Three perfect palindromes appear in S288c three coding regions. Different perfect palindromes appear at the same loci in three additional genomes. Each locus is shown in a column and different colors show different palindrome forms. Loci that are in a quasi-palindromic state are shown in gray. Cases in which the loci could not be identified are shown in black. Positions identical to S288c are shown with a dot and the spacer between arms is represented with a tilde sign. (C) Ninety-two S288c loci that appear in two additional genomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F9.medium.gif" width="440" height="58"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/11/3775/F9.medium.gif" width="440" height="58"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487178" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-37" class="first-child">Evidence for quasi-palindrome to palindrome conversion via template switching during evolution of <em>S. cerevisiae</em>. (A) A phylogenetic tree of seven <em>Saccharomyces</em> genomes. (B) Three perfect palindromes appear in S288c three coding regions. Different perfect palindromes appear at the same loci in three additional genomes. Each locus is shown in a column and different colors show different palindrome forms. Loci that are in a quasi-palindromic state are shown in gray. Cases in which the loci could not be identified are shown in black. Positions identical to S288c are shown with a dot and the spacer between arms is represented with a tilde sign. (C) Ninety-two S288c loci that appear in two additional genomes.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>rad27∆ strains have hotspots for changes</h3><p id="p-38"><a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27</a> is a key player in the assembly of Okazaki fragments on the lagging strand (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10A</a>). Insertions mediated by tandem repeats are common in <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> strains (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10B</a>) (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Tishkoff <em>et al.</em> 1997</a>). We found that there are genomic hotspots for these insertions in this genotype. We note that a model of an insertion of one repeat unit in SSR is similar (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10C</a>), as the repeat can easily realign in multiple places. <a id="xref-fig-10-4" class="xref-fig" href="#F10">Figure 10B</a> shows a model for an insertion mediated by a direct repeat, where realignment of the direct repeat in the wrong location enables the formation of a loop containing the unique sequence that forms the insertion in the next round of replication.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Models for the formation of the common events observed in rad27&#x394; strains. (A) Description of Okazaki fragment maturation. Primers produced by polymerase &#x3B1; are shown in a light gray zigzag, and the extension by polymerase &#x3B4; is represented by a darker shade of gray. The predicted position where Rad27/FEN1 acts to remove the RNA/DNA primer in wild-type strains is indicated by the small black arrow. The junction between two adjacent Okazaki fragments is enlarged in B&#x2013;E (green ellipse). (B) A model for an insertion mediated by a direct repeat, where realignment of the direct repeat in the wrong location enables the formation of a loop containing the unique sequence that forms the insertion in the next round of replication. (C) A model for an insertion of one repeat in SSR. We note that mechanisms B and C are similar, as the repeat can easily realign in multiple places. (D) An intramolecular template-switching model for the correction of a quasi-palindrome to a palindrome. The star indicates a change between the two copies of the IR. (E) A model for expansion of a palindrome through a template-switching mechanism. We suggest that the polymerase is stalled due to stabilization of the flap as a hairpin in the previously synthesized Okazaki fragment, or due to inefficient strand displacement. The polymerase continues synthesis using the nascent strand as a template. The red strand is the nascent strand and the blue strand is the original template strand." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899285364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Models for the formation of the common events observed in &lt;em&gt;rad27&lt;/em&gt;&#x394; strains. (A) Description of Okazaki fragment maturation. Primers produced by polymerase &#x3B1; are shown in a light gray zigzag, and the extension by polymerase &#x3B4; is represented by a darker shade of gray. The predicted position where Rad27/FEN1 acts to remove the RNA/DNA primer in wild-type strains is indicated by the small black arrow. The junction between two adjacent Okazaki fragments is enlarged in B&#x2013;E (green ellipse). (B) A model for an insertion mediated by a direct repeat, where realignment of the direct repeat in the wrong location enables the formation of a loop containing the unique sequence that forms the insertion in the next round of replication. (C) A model for an insertion of one repeat in SSR. We note that mechanisms B and C are similar, as the repeat can easily realign in multiple places. (D) An intramolecular template-switching model for the correction of a quasi-palindrome to a palindrome. The star indicates a change between the two copies of the IR. (E) A model for expansion of a palindrome through a template-switching mechanism. We suggest that the polymerase is stalled due to stabilization of the flap as a hairpin in the previously synthesized Okazaki fragment, or due to inefficient strand displacement. The polymerase continues synthesis using the nascent strand as a template. The red strand is the nascent strand and the blue strand is the original template strand.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3775/F10.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/7/11/3775/F10.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3775/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487271" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-39" class="first-child">Models for the formation of the common events observed in <em>rad27</em>Δ strains. (A) Description of Okazaki fragment maturation. Primers produced by polymerase α are shown in a light gray zigzag, and the extension by polymerase δ is represented by a darker shade of gray. The predicted position where Rad27/FEN1 acts to remove the RNA/DNA primer in wild-type strains is indicated by the small black arrow. The junction between two adjacent Okazaki fragments is enlarged in B–E (green ellipse). (B) A model for an insertion mediated by a direct repeat, where realignment of the direct repeat in the wrong location enables the formation of a loop containing the unique sequence that forms the insertion in the next round of replication. (C) A model for an insertion of one repeat in SSR. We note that mechanisms B and C are similar, as the repeat can easily realign in multiple places. (D) An intramolecular template-switching model for the correction of a quasi-palindrome to a palindrome. The star indicates a change between the two copies of the IR. (E) A model for expansion of a palindrome through a template-switching mechanism. We suggest that the polymerase is stalled due to stabilization of the flap as a hairpin in the previously synthesized Okazaki fragment, or due to inefficient strand displacement. The polymerase continues synthesis using the nascent strand as a template. The red strand is the nascent strand and the blue strand is the original template strand.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Stalled replication forks in rad27∆ strains are resolved by template switching</h3><p id="p-40">The current view of Okazaki fragment maturation assumes a competition between DNA polymerase and DNA ligase I on the 3′ OH of the nascent strand. In the absence of <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27</a>, the formation of ligatable nicks is severely reduced and thus the polymerase can rebind the 3′ OH and continue synthesis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Burgers and Kunkel 2017</a>). Our results indicate that when the legitimate single-stranded DNA template is shrinking, the polymerase may switch to other available templates rather than continue and displace the downstream Okazaki fragment. This is in agreement with biochemical data suggesting that strand displacement by pol δ is minimal (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Burgers and Kunkel 2017</a>). Therefore, here we relate to the point where the template of DNA synthesis is no longer the original template as a point of stalled replication fork. It is possible that <em>in vivo</em> strand displacement can progress further than the point of template switching but the nascent DNA is processed by nuclease as described here in <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a> and by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jin <em>et al.</em> (2001)</a>.</p><p id="p-41">The most powerful aspect of using whole genome sequencing in the study of mutagenesis is to reveal the contribution of the sequence context. Detecting that <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a>∆</em> strains undergo template switching was only possible because of the use of whole genome sequencing. Moreover, a previous analysis of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em> null mutants was unable to detect these events (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Serero <em>et al.</em> 2014</a>). Quasi-palindrome to palindrome correction in one arm of an IR, which then serves as a template for the synthesis of the second arm, has been demonstrated in multiple organisms including bacteria and yeasts (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bzymek and Lovett 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lisnić <em>et al.</em> 2005</a>). This phenomenon has been linked to both mutational hotspots (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rosche <em>et al.</em> 1998</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Viswanathan <em>et al.</em> 2000</a>) and genetic diseases (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bissler 1998</a>). It was also shown that obstacles in Okazaki fragments, such as RNA residues in the DNA of DNA-RNA hybrids, can also lead to template-switching events (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kim <em>et al.</em> 2013</a>).</p><p id="p-42">Either an intermolecular or an intramolecular template-switching model could be responsible for quasi-palindrome to palindrome correction (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ripley 1982</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lovett 2004</a>). In the intermolecular strand switching model, the unpaired 3′ end of the nascent strand pairs with a single-stranded template on the opposite strand. As <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27p</a> is assigned to the lagging strand and acts when the leading strand should not contain any gaps, a mechanism of intermolecular strand switching is very unlikely, and we prefer a mechanism of intramolecular strand switching (<a id="xref-fig-10-5" class="xref-fig" href="#F10">Figure 10D</a>).</p><p id="p-43">A template-switching mechanism similar to that of a quasi-palindrome correction can also achieve palindrome expansion. In the absence of <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">RAD27</a></em>, the legitimate DNA template can be blocked. Recent data strongly argue that the replication fork is dynamic and can adopt alternative templates when it is blocked to allow completion of DNA synthesis, even at the risk of a severe genome rearrangement (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lee <em>et al.</em> 2007</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Weinert <em>et al.</em> 2009</a>). Here, at least two alternative templates can be proposed for strand misalignment at the border between the newly synthesized Okazaki fragment and the previous fragment. First, when <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">RAD27</a></em> is missing, the flap structure of the old Okazaki fragment can create a hairpin, preventing the strand displacement by pol δ (<a id="xref-fig-10-6" class="xref-fig" href="#F10">Figure 10E</a>). Therefore, the newly synthesized Okazaki fragment will fold back and synthesize in the opposite direction. It was shown that palindromes can also be achieved when a double strand break occurs next to a short inverted repeat (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Butler <em>et al.</em> 1996</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rattray <em>et al.</em> 2005</a>) or by origin-dependent IR amplification (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brewer <em>et al.</em> 2011</a>). Forming longer palindromes can also be unstable (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Butler <em>et al.</em> 1996</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tanaka <em>et al.</em> 2002</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Brewer <em>et al.</em> 2011</a>), and their formation is associated with gene amplification events that can promote or contribute to carcinogenesis (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Narayanan <em>et al.</em> 2006</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tanaka <em>et al.</em> 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tanaka and Yao 2009</a>).</p><p id="p-44">In breast cancer, it was shown that palindrome formation is an early stage in gene amplification (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tanaka <em>et al.</em> 2005</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Tanaka and Yao 2009</a>), often associated with poor prognosis (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guenthoer <em>et al.</em> 2012</a>). Similarly, in our <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ analysis we found that palindrome expansion is primarily associated with DNA amplification. We note, however, that gene amplification in cancer includes also events much longer than we can detect in our <em><a href="http://www.yeastgenome.org/locus/S000001596/overview">rad27</a></em>Δ system. In a broader context, our data support a more frequent phenomenon of primer extension when the template is unavailable, presumably due to a stalled replication fork, that causes the polymerase to switch to the nascent strand and use it as the template (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lee <em>et al.</em> 2007</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Weinert <em>et al.</em> 2009</a>).</p><p id="p-45">We also show the first evidences of perfect IR representation in orthologous regions through <em>Saccharomyces</em> evolution. This finding suggests that a template-switching mechanism is important not only in instability conditions but also through evolution and normal organism life.</p><p id="p-46">Finally, our results provide a mechanistic insight. It has not been clear how the 3′ end of an Okazaki fragment transfers between the polymerase and DNA ligase. The fact that we observe significant template switching indicates that the absence of a natural DNA template does not serve as a signal for the polymerase to fall off and cease DNA synthesis. It is tempting to speculate that <a href="http://www.yeastgenome.org/locus/S000001596/overview">Rad27p</a> itself may have a direct role in signaling for the switch between the polymerase and the ligase.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank Margaret Dominska, Patricia W. Greenwell, Marina Druseikis, and Aseelnor Abedalkhal for their help with yeast subculturing. This project started at the laboratory of Thomas D. Petes at Duke University and was supported by National Institutes of Health grants GM24110 and GM52319. P.A.M. is funded by the University of North Carolina at Chapel Hill University Cancer Research Fund. S.C. is supported by the Research Career Development Award of the Israel Cancer Research Foundation. E.H.-C. was supported by the Israel Cancer Association grant 20150038 and by the Open University of Israel Research fund.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-48">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300262/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300262/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-49"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-06"><span class="received-label">Received </span>August 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-09-14"><span class="accepted-label">Accepted </span>September 14, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Omer <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.2" data-doi="10.1074/jbc.M006513200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Characterization of the enzymatic properties of the yeast Dna2 helicase/endonuclease suggests a new model for Okazaki fragment processing.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">38022</span>–<span class="cit-lpage">38031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M006513200%26rft_id%253Dinfo%253Apmid%252F10984490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvNDgvMzgwMjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.3" data-doi="10.1074/jbc.273.41.26880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Dna2 of <em>Saccharomyces cerevisiae</em> possesses a single-stranded DNA-specific endonuclease activity that is able to act on double-stranded DNA in the presence of ATP.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">26880</span>–<span class="cit-lpage">26890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBae%26rft.auinit1%253DS.-H.%26rft.volume%253D273%26rft.issue%253D41%26rft.spage%253D26880%26rft.epage%253D26890%26rft.atitle%253DDna2%2Bof%2BSaccharomyces%2Bcerevisiae%2BPossesses%2Ba%2BSingle-stranded%2BDNA-specific%2BEndonuclease%2BActivity%2BThat%2BIs%2BAble%2Bto%2BAct%2Bon%2BDouble-stranded%2BDNA%2Bin%2Bthe%2BPresence%2Bof%2BATP%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.41.26880%26rft_id%253Dinfo%253Apmid%252F9756935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzMvNDEvMjY4ODAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.4" data-doi="10.1038/35086609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">RPA governs endonuclease switching during processing of Okazaki fragments in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">412</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBae%26rft.auinit1%253DS.%2BH.%26rft.volume%253D412%26rft.issue%253D6845%26rft.spage%253D456%26rft.epage%253D461%26rft.atitle%253DRPA%2Bgoverns%2Bendonuclease%2Bswitching%2Bduring%2Bprocessing%2Bof%2BOkazaki%2Bfragments%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35086609%26rft_id%253Dinfo%253Apmid%252F11473323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35086609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11473323&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170068200053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.5" data-doi="10.1074/jbc.272.8.4647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bambara</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murante</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henricksen</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Enzymes and reactions at the eukaryotic DNA replication fork.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">4647</span>–<span class="cit-lpage">4650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBambara%26rft.auinit1%253DR.%2BA.%26rft.volume%253D272%26rft.issue%253D8%26rft.spage%253D4647%26rft.epage%253D4650%26rft.atitle%253DEnzymes%2Band%2BReactions%2Bat%2Bthe%2BEukaryotic%2BDNA%2BReplication%2BFork%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.272.8.4647%26rft_id%253Dinfo%253Apmid%252F9081985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzIvOC80NjQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.6" data-doi="10.1093/nar/27.2.573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Tandem repeats finder: a program to analyze DNA sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.2.573%26rft_id%253Dinfo%253Apmid%252F9862982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.2.573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9862982&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078181500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bissler</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">DNA inverted repeats and human disease.</span> <abbr class="cit-jnl-abbrev">Front. Biosci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">d408</span>–<span class="cit-lpage">d418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bbioscience%2B%253A%2B%2Ba%2Bjournal%2Band%2Bvirtual%2Blibrary%26rft.stitle%253DFront%2BBiosci%26rft.aulast%253DBissler%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D3%26rft.spage%253Dd408%26rft.epage%253Dd418%26rft.atitle%253DDNA%2Binverted%2Brepeats%2Band%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Apmid%252F9516381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9516381&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.8" data-doi="10.1371/journal.pgen.1002016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Origin-dependent inverted-repeat amplification: a replication-based model for generating palindromic amplicons.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBrewer%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1002016%26rft.epage%253De1002016%26rft.atitle%253DOrigin-dependent%2Binverted-repeat%2Bamplification%253A%2Ba%2Breplication-based%2Bmodel%2Bfor%2Bgenerating%2Bpalindromic%2Bamplicons.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002016%26rft_id%253Dinfo%253Apmid%252F21437266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437266&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.9" data-doi="10.1101/gr.926603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brudno</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davydov</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">LAGAN and multi-LAGAN: efficient tools for large-scale multiple alignment of genomic DNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.926603%26rft_id%253Dinfo%253Apmid%252F12654723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzQvNzIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.10" data-doi="10.1074/jbc.R800062200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Polymerase dynamics at the eukaryotic DNA replication fork.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">4041</span>–<span class="cit-lpage">4045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R800062200%26rft_id%253Dinfo%253Apmid%252F18835809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODQvNy80MDQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Eukaryotic DNA replication fork.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiochem.%26rft.volume%253D86%26rft.spage%253D417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.12" data-doi="10.1016/S0092-8674(00)81805-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Induction of large DNA palindrome formation in yeast: implications for gene amplification and genome stability in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DButler%26rft.auinit1%253DD.%2BK.%26rft.volume%253D87%26rft.issue%253D6%26rft.spage%253D1115%26rft.epage%253D1122%26rft.atitle%253DInduction%2Bof%2Blarge%2BDNA%2Bpalindrome%2Bformation%2Bin%2Byeast%253A%2Bimplications%2Bfor%2Bgene%2Bamplification%2Band%2Bgenome%2Bstability%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981805-X%26rft_id%253Dinfo%253Apmid%252F8978615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81805-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8978615&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VY44700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bzymek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evidence for two mechanisms of palindrome-stimulated deletion in <em>Escherichia coli</em>: single-strand annealing and replication slipped mispairing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBzymek%26rft.auinit1%253DM.%26rft.volume%253D158%26rft.issue%253D2%26rft.spage%253D527%26rft.epage%253D540%26rft.atitle%253DEvidence%2Bfor%2BTwo%2BMechanisms%2Bof%2BPalindrome-Stimulated%2BDeletion%2Bin%2BEscherichia%2Bcoli%253A%2BSingle-Strand%2BAnnealing%2Band%2BReplication%2BSlipped%2BMispairing%26rft_id%253Dinfo%253Apmid%252F11404319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzIvNTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.14" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.15" data-doi="10.1074/jbc.273.49.33064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somsouk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somsouk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Partial functional deficiency of E160D flap endonuclease-1 mutant <em>in vitro</em> and <em>in vivo</em> is due to defective cleavage of DNA substrates.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">33064</span>–<span class="cit-lpage">33072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DFrank%26rft.auinit1%253DG.%26rft.volume%253D273%26rft.issue%253D49%26rft.spage%253D33064%26rft.epage%253D33072%26rft.atitle%253DPartial%2BFunctional%2BDeficiency%2Bof%2BE160D%2BFlap%2BEndonuclease-1%2BMutantin%2BVitro%2Band%2Bin%2BVivo%2BIs%2BDue%2Bto%2BDefective%2BCleavage%2Bof%2BDNA%2BSubstrates%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.49.33064%26rft_id%253Dinfo%253Apmid%252F9830061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzMvNDkvMzMwNjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.16" data-doi="10.1093/molbev/msp259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp259%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854763&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273704400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.17" data-doi="10.1101/gr.117226.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guenthoer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diede</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Assessment of palindromes as platforms for DNA amplification in breast cancer.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.117226.110%26rft_id%253Dinfo%253Apmid%252F21752925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzIvMjMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.18" data-doi="10.1101/gad.8.11.1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrington</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieber</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Functional domains within FEN-1 and RAD2 define a family of structure-specific endonucleases: implications for nucleotide excision repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1344</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHarrington%26rft.auinit1%253DJ%2BJ%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D1344%26rft.epage%253D1355%26rft.atitle%253DFunctional%2Bdomains%2Bwithin%2BFEN-1%2Band%2BRAD2%2Bdefine%2Ba%2Bfamily%2Bof%2Bstructure-specific%2Bendonucleases%253A%2Bimplications%2Bfor%2Bnucleotide%2Bexcision%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.11.1344%26rft_id%253Dinfo%253Apmid%252F7926735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOC8xMS8xMzQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.19" data-doi="10.1073/pnas.091095198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The 3′→5′ exonuclease of DNA polymerase delta can substitute for the 5′ flap endonuclease Rad27/Fen1 in processing Okazaki fragments and preventing genome instability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">5122</span>–<span class="cit-lpage">5127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.091095198%26rft_id%253Dinfo%253Apmid%252F11309502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC85LzUxMjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.20" data-doi="10.1126/science.7618086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovvali</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Requirement of the yeast RTH1 5′ to 3′ exonuclease for the stability of simple repetitive DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJohnson%26rft.auinit1%253DR.%26rft.volume%253D269%26rft.issue%253D5221%26rft.spage%253D238%26rft.epage%253D240%26rft.atitle%253DRequirement%2Bof%2Bthe%2Byeast%2BRTH1%2B5%2527%2Bto%2B3%2527%2Bexonuclease%2Bfor%2Bthe%2Bstability%2Bof%2Bsimple%2Brepetitive%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7618086%26rft_id%253Dinfo%253Apmid%252F7618086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvNTIyMS8yMzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.21" data-doi="10.1016/S0168-9525(00)02093-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Repbase update: a database and an electronic journal of repetitive elements.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D16%26rft.issue%253D9%26rft.spage%253D418%26rft.epage%253D420%26rft.atitle%253DRepbase%2Bupdate%253A%2Ba%2Bdatabase%2Band%2Ban%2Belectronic%2Bjournal%2Bof%2Brepetitive%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902093-X%26rft_id%253Dinfo%253Apmid%252F10973072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02093-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10973072&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089376900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.22" data-doi="10.1007/978-1-59745-251-9_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asimenos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Multiple alignment of DNA sequences with MAFFT.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">537</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D537%26rft.spage%253D39%26rft.epage%253D64%26rft.atitle%253DMultiple%2Balignment%2Bof%2BDNA%2Bsequences%2Bwith%2BMAFFT.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-251-9_3%26rft_id%253Dinfo%253Apmid%252F19378139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-251-9_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19378139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268335500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.23" data-doi="10.1371/journal.pgen.1003924"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA:DNA hybrids initiate quasi-palindrome-associated mutations in highly transcribed yeast DNA.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003924%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003924%26rft_id%253Dinfo%253Apmid%252F24244191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244191&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.24" data-doi="10.1128/MCB.18.5.2779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kokoska</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefanovic</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordenin</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Destabilization of yeast micro- and minisatellite DNA sequences by mutations affecting a nuclease involved in Okazaki fragment processing (<em>rad27</em>) and DNA polymerase δ (<em>pol3-t</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2779</span>–<span class="cit-lpage">2788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKokoska%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D2779%26rft.epage%253D2788%26rft.atitle%253DDestabilization%2Bof%2BYeast%2BMicro-%2Band%2BMinisatellite%2BDNA%2BSequences%2Bby%2BMutations%2BAffecting%2Ba%2BNuclease%2BInvolved%2Bin%2BOkazaki%2BFragment%2BProcessing%2B%2528rad27%2529%2Band%2BDNA%2BPolymerase%2Bdelta%2B%25A0%2528pol3-t%2529%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.18.5.2779%26rft_id%253Dinfo%253Apmid%252F9566897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE4LzUvMjc3OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.25" data-doi="10.1016/j.tcb.2008.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dividing the workload at a eukaryotic replication fork.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DKunkel%26rft.auinit1%253DT.%2BA.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D521%26rft.epage%253D527%26rft.atitle%253DDividing%2Bthe%2Bworkload%2Bat%2Ba%2Beukaryotic%2Breplication%2Bfork.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2008.08.005%26rft_id%253Dinfo%253Apmid%252F18824354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2008.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18824354&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261074400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.26" data-doi="10.1016/j.cell.2007.11.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A DNA replication mechanism for generating nonrecurrent rearrangements associated with genomic disorders.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLee%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D131%26rft.issue%253D7%26rft.spage%253D1235%26rft.epage%253D1247%26rft.atitle%253DA%2BDNA%2Breplication%2Bmechanism%2Bfor%2Bgenerating%2Bnonrecurrent%2Brearrangements%2Bassociated%2Bwith%2Bgenomic%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.11.037%26rft_id%253Dinfo%253Apmid%252F18160035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.11.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18160035&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252217200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.27" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.28" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.29" data-doi="10.1007/s00294-005-0573-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lisnić</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svetec</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarić</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikolić</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zgaga</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Palindrome content of the yeast <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DLisni%25C4%2587%26rft.auinit1%253DB.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D289%26rft.epage%253D297%26rft.atitle%253DPalindrome%2Bcontent%2Bof%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-005-0573-5%26rft_id%253Dinfo%253Apmid%252F15776233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-005-0573-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15776233&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229022100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.30" data-doi="10.1111/j.1365-2958.2004.04076.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Encoded errors: mutations and rearrangements mediated by misalignment at repetitive DNA sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLovett%26rft.auinit1%253DS.%2BT.%26rft.volume%253D52%26rft.issue%253D5%26rft.spage%253D1243%26rft.epage%253D1253%26rft.atitle%253DEncoded%2Berrors%253A%2Bmutations%2Band%2Brearrangements%2Bmediated%2Bby%2Bmisalignment%2Bat%2Brepetitive%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2004.04076.x%26rft_id%253Dinfo%253Apmid%252F15165229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2004.04076.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15165229&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221644200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.31" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.32" data-doi="10.1093/nar/gkl070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masuda-Sasa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imamura</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Biochemical analysis of human Dna2.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1865</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMasuda-Sasa%26rft.auinit1%253DT.%26rft.volume%253D34%26rft.issue%253D6%26rft.spage%253D1865%26rft.epage%253D1875%26rft.atitle%253DBiochemical%2Banalysis%2Bof%2Bhuman%2BDna2.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl070%26rft_id%253Dinfo%253Apmid%252F16595800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16595800&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237002900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.33" data-doi="10.1016/j.cell.2006.04.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobachev</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The pattern of gene amplification is determined by the chromosomal location of hairpin-capped breaks.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNarayanan%26rft.auinit1%253DV.%26rft.volume%253D125%26rft.issue%253D7%26rft.spage%253D1283%26rft.epage%253D1296%26rft.atitle%253DThe%2Bpattern%2Bof%2Bgene%2Bamplification%2Bis%2Bdetermined%2Bby%2Bthe%2Bchromosomal%2Blocation%2Bof%2Bhairpin-capped%2Bbreaks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.04.042%26rft_id%253Dinfo%253Apmid%252F16814715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.04.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16814715&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239104500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.34" data-doi="10.1016/j.molcel.2008.02.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nick McElhinny</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordenin</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stith</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Division of labor at the eukaryotic replication fork.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DNick%2BMcElhinny%26rft.auinit1%253DS.%2BA.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D137%26rft.epage%253D144%26rft.atitle%253DDivision%2Bof%2Blabor%2Bat%2Bthe%2Beukaryotic%2Breplication%2Bfork.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.02.022%26rft_id%253Dinfo%253Apmid%252F18439893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.02.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18439893&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255410200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.35" data-doi="10.1371/journal.pgen.1001109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001109%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001109%26rft_id%253Dinfo%253Apmid%252F20838597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838597&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.36" data-doi="10.1101/gad.1315805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rattray</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neelam</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A mechanism of palindromic gene amplification in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1390</span>–<span class="cit-lpage">1399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRattray%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D1390%26rft.epage%253D1399%26rft.atitle%253DA%2Bmechanism%2Bof%2Bpalindromic%2Bgene%2Bamplification%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1315805%26rft_id%253Dinfo%253Apmid%252F15937224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzExLzEzOTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.37" data-doi="10.1128/jb.177.2.364-371.1995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reagan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pittenger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siede</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedberg</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Characterization of a mutant strain of Saccharomyces cerevisiae with a deletion of the RAD27 gene, a structural homolog of the RAD2 nucleotide excision repair gene.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">364</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DReagan%26rft.auinit1%253DM.%2BS.%26rft.volume%253D177%26rft.issue%253D2%26rft.spage%253D364%26rft.epage%253D371%26rft.atitle%253DCharacterization%2Bof%2Ba%2Bmutant%2Bstrain%2Bof%2BSaccharomyces%2Bcerevisiae%2Bwith%2Ba%2Bdeletion%2Bof%2Bthe%2BRAD27%2Bgene%252C%2Ba%2Bstructural%2Bhomolog%2Bof%2Bthe%2BRAD2%2Bnucleotide%2Bexcision%2Brepair%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.177.2.364-371.1995%26rft_id%253Dinfo%253Apmid%252F7814325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTc3LzIvMzY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.38" data-doi="10.1016/S0168-9525(00)02024-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleasby</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">EMBOSS: the European molecular biology open software suite.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRice%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D276%26rft.epage%253D277%26rft.atitle%253DEMBOSS%253A%2Bthe%2BEuropean%2BMolecular%2BBiology%2BOpen%2BSoftware%2BSuite.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902024-2%26rft_id%253Dinfo%253Apmid%252F10827456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827456&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.39" data-doi="10.1073/pnas.79.13.4128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ripley</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Model for the participation of quasi-palindromic DNA sequences in frameshift mutation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">4128</span>–<span class="cit-lpage">4132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRipley%26rft.auinit1%253DL.%2BS.%26rft.volume%253D79%26rft.issue%253D13%26rft.spage%253D4128%26rft.epage%253D4132%26rft.atitle%253DModel%2Bfor%2Bthe%2Bparticipation%2Bof%2Bquasi-palindromic%2BDNA%2Bsequences%2Bin%2Bframeshift%2Bmutation.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.13.4128%26rft_id%253Dinfo%253Apmid%252F7051004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMTMvNDEyOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.40" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.41" data-doi="10.1006/jmbi.1998.2193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosche</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripley</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinden</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Primer-template misalignments during leading strand DNA synthesis account for the most frequent spontaneous mutations in a quasipalindromic region in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DRosche%26rft.auinit1%253DW.%2BA.%26rft.volume%253D284%26rft.issue%253D3%26rft.spage%253D633%26rft.epage%253D646%26rft.atitle%253DPrimer-template%2Bmisalignments%2Bduring%2Bleading%2Bstrand%2BDNA%2Bsynthesis%2Baccount%2Bfor%2Bthe%2Bmost%2Bfrequent%2Bspontaneous%2Bmutations%2Bin%2Ba%2Bquasipalindromic%2Bregion%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.2193%26rft_id%253Dinfo%253Apmid%252F9826504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.2193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9826504&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077275600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.42" data-doi="10.1073/pnas.1314423111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jubin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeillet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legoix-Né</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mutational landscape of yeast mutator strains.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1897</span>–<span class="cit-lpage">1902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1314423111%26rft_id%253Dinfo%253Apmid%252F24449905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzUvMTg5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.43" data-doi="10.1002/bies.20255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Multiple but dissectible functions of FEN-1 nucleases in nucleic acid processing, genome stability and diseases.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DShen%26rft.auinit1%253DB.%26rft.volume%253D27%26rft.issue%253D7%26rft.spage%253D717%26rft.epage%253D729%26rft.atitle%253DMultiple%2Bbut%2Bdissectible%2Bfunctions%2Bof%2BFEN-1%2Bnucleases%2Bin%2Bnucleic%2Bacid%2Bprocessing%252C%2Bgenome%2Bstability%2Band%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20255%26rft_id%253Dinfo%253Apmid%252F15954100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15954100&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230255200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.44" data-doi="10.1101/gr.089532.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">ABySS: a parallel assembler for short read sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089532.108%26rft_id%253Dinfo%253Apmid%252F19251739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTExNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.45" data-doi="10.1074/jbc.M806668200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stith</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordenin</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Flexibility of eukaryotic Okazaki fragment maturation through regulated strand displacement synthesis.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">34129</span>–<span class="cit-lpage">34140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M806668200%26rft_id%253Dinfo%253Apmid%252F18927077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvNDkvMzQxMjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.46" data-doi="10.1073/pnas.92.22.10418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Earley</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crouse</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Mutations in the MSH3 gene preferentially lead to deletions within tracts of simple repetitive DNA in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">10418</span>–<span class="cit-lpage">10421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DStrand%26rft.auinit1%253DM.%26rft.volume%253D92%26rft.issue%253D22%26rft.spage%253D10418%26rft.epage%253D10421%26rft.atitle%253DMutations%2Bin%2Bthe%2BMSH3%2Bgene%2Bpreferentially%2Blead%2Bto%2Bdeletions%2Bwithin%2Btracts%2Bof%2Bsimple%2Brepetitive%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.22.10418%26rft_id%253Dinfo%253Apmid%252F7479796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTIvMjIvMTA0MTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.47" data-doi="10.1038/nrc2591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Palindromic gene amplification – an evolutionarily conserved role for DNA inverted repeats in the genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DTanaka%26rft.auinit1%253DH.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D216%26rft.epage%253D224%26rft.atitle%253DPalindromic%2Bgene%2Bamplification--an%2Bevolutionarily%2Bconserved%2Brole%2Bfor%2BDNA%2Binverted%2Brepeats%2Bin%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc2591%26rft_id%253Dinfo%253Apmid%252F19212324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc2591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212324&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.48" data-doi="10.1073/pnas.132275999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tapscott</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trask</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Short inverted repeats initiate gene amplification through the formation of a large DNA palindrome in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">8772</span>–<span class="cit-lpage">8777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.132275999%26rft_id%253Dinfo%253Apmid%252F12060719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTMvODc3MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.49" data-doi="10.1038/ng1515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tapscott</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Widespread and nonrandom distribution of DNA palindromes in cancer cells provides a structural platform for subsequent gene amplification.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTanaka%26rft.auinit1%253DH.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D320%26rft.epage%253D327%26rft.atitle%253DWidespread%2Band%2Bnonrandom%2Bdistribution%2Bof%2BDNA%2Bpalindromes%2Bin%2Bcancer%2Bcells%2Bprovides%2Ba%2Bstructural%2Bplatform%2Bfor%2Bsubsequent%2Bgene%2Bamplification.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1515%26rft_id%253Dinfo%253Apmid%252F15711546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15711546&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227297800025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.50" data-doi="10.1128/MCB.01313-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kooperberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tapscott</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Intrastrand annealing leads to the formation of a large DNA palindrome and determines the boundaries of genomic amplification in human cancer.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1993</span>–<span class="cit-lpage">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01313-06%26rft_id%253Dinfo%253Apmid%252F17242211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI3LzYvMTk5MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.51" data-doi="10.1016/S0092-8674(00)81846-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">D. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filosi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaida</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A novel mutation avoidance mechanism dependent on S. cerevisiae <em>RAD27</em> is distinct from DNA mismatch repair.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTishkoff%26rft.auinit1%253DD.%2BX.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D253%26rft.epage%253D263%26rft.atitle%253DA%2Bnovel%2Bmutation%2Bavoidance%2Bmechanism%2Bdependent%2Bon%2BS.%2Bcerevisiae%2BRAD27%2Bis%2Bdistinct%2Bfrom%2BDNA%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981846-2%26rft_id%253Dinfo%253Apmid%252F9008166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81846-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9008166&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WE97000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turchi</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bambara</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Completion of mammalian lagging strand DNA replication using purified proteins.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">15136</span>–<span class="cit-lpage">15141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DTurchi%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D268%26rft.issue%253D20%26rft.spage%253D15136%26rft.epage%253D15141%26rft.atitle%253DCompletion%2Bof%2Bmammalian%2Blagging%2Bstrand%2BDNA%2Breplication%2Busing%2Bpurified%2Bproteins%26rft_id%253Dinfo%253Apmid%252F8392066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjgvMjAvMTUxMzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.53" data-doi="10.1073/pnas.91.21.9803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turchi</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murante</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bambara</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Enzymatic completion of mammalian lagging-strand DNA replication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">9803</span>–<span class="cit-lpage">9807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTurchi%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D91%26rft.issue%253D21%26rft.spage%253D9803%26rft.epage%253D9807%26rft.atitle%253DEnzymatic%2Bcompletion%2Bof%2Bmammalian%2Blagging-strand%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.21.9803%26rft_id%253Dinfo%253Apmid%252F7524089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTEvMjEvOTgwMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.54" data-doi="10.1006/jmbi.2000.4088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viswanathan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacirignola</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurley</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A novel mutational hotspot in a natural quasipalindrome in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DViswanathan%26rft.auinit1%253DM.%26rft.volume%253D302%26rft.issue%253D3%26rft.spage%253D553%26rft.epage%253D564%26rft.atitle%253DA%2Bnovel%2Bmutational%2Bhotspot%2Bin%2Ba%2Bnatural%2Bquasipalindrome%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4088%26rft_id%253Dinfo%253Apmid%252F10986118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10986118&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089603600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.55" data-doi="10.1038/369207a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Anatomy of a DNA replication fork revealed by reconstitution of SV40 DNA replication in vitro.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">369</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaga%26rft.auinit1%253DS.%26rft.volume%253D369%26rft.issue%253D6477%26rft.spage%253D207%26rft.epage%253D212%26rft.atitle%253DAnatomy%2Bof%2Ba%2BDNA%2Breplication%2Bfork%2Brevealed%2Bby%2Breconstitution%2Bof%2BSV40%2BDNA%2Breplication%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F369207a0%26rft_id%253Dinfo%253Apmid%252F7910375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/369207a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7910375&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NM06700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.56" data-doi="10.1073/pnas.0701291104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome sequencing and comparative analysis of <em>Saccharomyces cerevisiae</em> strain YJM789.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">12825</span>–<span class="cit-lpage">12830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701291104%26rft_id%253Dinfo%253Apmid%252F17652520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzMxLzEyODI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.57" data-doi="10.1016/j.ceb.2009.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaochar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paek</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The replication fork’s five degrees of freedom, their failure and genome rearrangements.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">778</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DWeinert%26rft.auinit1%253DT.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D778%26rft.epage%253D784%26rft.atitle%253DThe%2Breplication%2Bfork%2527s%2Bfive%2Bdegrees%2Bof%2Bfreedom%252C%2Btheir%2Bfailure%2Band%2Bgenome%2Brearrangements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2009.10.004%26rft_id%253Dinfo%253Apmid%252F19913398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2009.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19913398&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272936500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3775.58" data-doi="10.1093/jmcb/mjq048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Okazaki fragment maturation: nucleases take centre stage.</span> <abbr class="cit-jnl-abbrev">J. Mol. Cell Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjmcb%252Fmjq048%26rft_id%253Dinfo%253Apmid%252F21278448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jmcb/mjq048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21278448&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3775.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3775.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3757" title="Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3789" title="Pervasive, Genome-Wide Transcription in the Organelle Genomes of Diverse Plastid-Bearing Protists" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3775" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IaiphuV4KhgB3H3L6j0EWof-ydIIRevvg4_VLLynjO4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3775" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3775" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3775" class="active" data-icon-position="" data-hide-link-title="0">Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487100" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yQqge103lVsoQQGC0QwSokM9FbcrbKw4PZkOeg3vb9s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3775.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3775" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-80PIvgPWbTCtfe8CM1BEIjD3U3Wuy_JVhK6u4JTztMc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3775" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3775.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487100" id="node487100--2" data-pisa="ggg;7/11/3775" data-pisa-master="ggg;g3.117.300262" data-apath="/ggg/7/11/3775.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequence Analysis of Mutations Accumulated in <em>rad27</em><strong>Δ</strong> Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sumita Omer</span>, <span class="highwire-citation-author" data-delta="1">Bar Lavi</span>, <span class="highwire-citation-author" data-delta="2">Piotr A. Mieczkowski</span>, <span class="highwire-citation-author" data-delta="3">Shay Covo</span> and <span class="highwire-citation-author" data-delta="4">Einat Hazkani-Covo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3775-3787;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487100--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sumita Omer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sumita%2BOmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Omer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASumita%2BOmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bar Lavi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Cell Research and Immunology, George S. Wise Faculty of Life Sciences, Tel Aviv University, 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bar%2BLavi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lavi%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABar%2BLavi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Piotr A. Mieczkowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piotr%2BA.%2BMieczkowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mieczkowski%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiotr%2BA.%2BMieczkowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shay Covo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shay%2BCovo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShay%2BCovo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shay.covo@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">shay.covo@mail.huji.ac.il</a>
<a href="mailto:einatco@openu.ac.il" class="" data-icon-position="" data-hide-link-title="0">einatco@openu.ac.il</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Einat Hazkani-Covo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Einat%2BHazkani-Covo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hazkani-Covo%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEinat%2BHazkani-Covo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shay.covo@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">shay.covo@mail.huji.ac.il</a>
<a href="mailto:einatco@openu.ac.il" class="" data-icon-position="" data-hide-link-title="0">einatco@openu.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3775" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3775&amp;title=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3775&amp;title=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3775&amp;title=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3775&amp;text=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3775&amp;title=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3775&amp;t=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3775&amp;title=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3775&amp;title=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487100/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487100" id="node487100--3" data-pisa="ggg;7/11/3775" data-pisa-master="ggg;g3.117.300262" data-apath="/ggg/7/11/3775.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequence Analysis of Mutations Accumulated in <em>rad27</em><strong>Δ</strong> Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sumita Omer</span>, <span class="highwire-citation-author" data-delta="1">Bar Lavi</span>, <span class="highwire-citation-author" data-delta="2">Piotr A. Mieczkowski</span>, <span class="highwire-citation-author" data-delta="3">Shay Covo</span> and <span class="highwire-citation-author" data-delta="4">Einat Hazkani-Covo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3775-3787;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487100--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sumita Omer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sumita%2BOmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Omer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASumita%2BOmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bar Lavi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Cell Research and Immunology, George S. Wise Faculty of Life Sciences, Tel Aviv University, 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bar%2BLavi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lavi%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABar%2BLavi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Piotr A. Mieczkowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piotr%2BA.%2BMieczkowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mieczkowski%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiotr%2BA.%2BMieczkowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shay Covo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology and Microbiology, Robert H. Smith Faculty of Agriculture, Food and Environment, Hebrew University, 76100 Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shay%2BCovo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShay%2BCovo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shay.covo@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">shay.covo@mail.huji.ac.il</a>
<a href="mailto:einatco@openu.ac.il" class="" data-icon-position="" data-hide-link-title="0">einatco@openu.ac.il</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Einat Hazkani-Covo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Natural and Life Sciences, The Open University of Israel, 4353701 Ra’anana, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Einat%2BHazkani-Covo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hazkani-Covo%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEinat%2BHazkani-Covo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shay.covo@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">shay.covo@mail.huji.ac.il</a>
<a href="mailto:einatco@openu.ac.il" class="" data-icon-position="" data-hide-link-title="0">einatco@openu.ac.il</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3775&amp;rft.epage=3787&amp;rft.atitle=Whole%20Genome%20Sequence%20Analysis%20of%20Mutations%20Accumulated%20in%20rad27%CE%94%20Yeast%20Strains%20with%20Defects%20in%20the%20Processing%20of%20Okazaki%20Fragments%20Indicates%20Template-Switching%20Events&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Omer%2C+Sumita&amp;rft.au=Lavi%2C+Bar&amp;rft.au=Mieczkowski%2C+Piotr+A.&amp;rft.au=Covo%2C+Shay&amp;rft.au=Hazkani-Covo%2C+Einat"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487100/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487100/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487100/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487100/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487100/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487100/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487100/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487100/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487100/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487100/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487100/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487100/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3775" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3775.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3775.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3775.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3775.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
