<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3669" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3669.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3669.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3669.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3669.atom" />
<meta name="HW.pisa" content="ggg;7/11/3669" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Dynamics of Hybrid Saccharomyces cerevisiae During Vegetative and Meiotic Divisions" />
<meta name="DC.Identifier" content="10.1534/g3.117.1135" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Dutta et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mutation and recombination are the major sources of genetic diversity in all organisms. In the baker’s yeast, all mutation rate estimates are in homozygous background. We determined the extent of genetic change through mutation and loss of heterozygosity (LOH) in a heterozygous Saccharomyces cerevisiae genome during successive vegetative and meiotic divisions. We measured genome-wide LOH and base mutation rates during vegetative and meiotic divisions in a hybrid (S288c/YJM789) S. cerevisiae strain. The S288c/YJM789 hybrid showed nearly complete reduction in heterozygosity within 31 generations of meioses and improved spore viability. LOH in the meiotic lines was driven primarily by the mating of spores within the tetrad. The S288c/YJM789 hybrid lines propagated vegetatively for the same duration as the meiotic lines, showed variable LOH (from 2 to 3% and up to 35%). Two of the vegetative lines with extensive LOH showed frequent and large internal LOH tracts that suggest a high frequency of recombination repair. These results suggest significant LOH can occur in the S288c/YJM789 hybrid during vegetative propagation presumably due to return to growth events. The average base substitution rates for the vegetative lines (1.82 × 10−10 per base per division) and the meiotic lines (1.22 × 10−10 per base per division) are the first genome-wide mutation rate estimates for a hybrid yeast. This study therefore provides a novel context for the analysis of mutation rates (especially in the context of detecting LOH during vegetative divisions), compared to previous mutation accumulation studies in yeast that used homozygous backgrounds." />
<meta name="DC.Contributor" content="Abhishek Dutta" />
<meta name="DC.Contributor" content="Gen Lin" />
<meta name="DC.Contributor" content="Ajith V. Pankajam" />
<meta name="DC.Contributor" content="Parijat Chakraborty" />
<meta name="DC.Contributor" content="Nahush Bhat" />
<meta name="DC.Contributor" content="Lars M. Steinmetz" />
<meta name="DC.Contributor" content="Koodali T. Nishant" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome Dynamics of Hybrid Saccharomyces cerevisiae During Vegetative and Meiotic Divisions" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mutation and recombination are the major sources of genetic diversity in all organisms. In the baker’s yeast, all mutation rate estimates are in homozygous background. We determined the extent of genetic change through mutation and loss of heterozygosity (LOH) in a heterozygous &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; genome during successive vegetative and meiotic divisions. We measured genome-wide LOH and base mutation rates during vegetative and meiotic divisions in a hybrid (S288c/YJM789) &lt;i&gt;S. cerevisiae&lt;/i&gt; strain. The S288c/YJM789 hybrid showed nearly complete reduction in heterozygosity within 31 generations of meioses and improved spore viability. LOH in the meiotic lines was driven primarily by the mating of spores within the tetrad. The S288c/YJM789 hybrid lines propagated vegetatively for the same duration as the meiotic lines, showed variable LOH (from 2 to 3% and up to 35%). Two of the vegetative lines with extensive LOH showed frequent and large internal LOH tracts that suggest a high frequency of recombination repair. These results suggest significant LOH can occur in the S288c/YJM789 hybrid during vegetative propagation presumably due to return to growth events. The average base substitution rates for the vegetative lines (1.82 × 10&lt;sup&gt;−10&lt;/sup&gt; per base per division) and the meiotic lines (1.22 × 10&lt;sup&gt;−10&lt;/sup&gt; per base per division) are the first genome-wide mutation rate estimates for a hybrid yeast. This study therefore provides a novel context for the analysis of mutation rates (especially in the context of detecting LOH during vegetative divisions), compared to previous mutation accumulation studies in yeast that used homozygous backgrounds.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3669" />
<meta name="citation_id" content="7/11/3669" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3669" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3669.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3669.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3669.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.1135" />
<meta name="citation_pmid" content="28916648" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3669" />
<meta name="citation_lastpage" content="3679" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Abhishek Dutta" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Gen Lin" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany" />
<meta name="citation_author" content="Ajith V. Pankajam" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Parijat Chakraborty" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Nahush Bhat" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Lars M. Steinmetz" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University, California 94305" />
<meta name="citation_author_institution" content="Stanford Genome Technology Center, Palo Alto, California 94304" />
<meta name="citation_author" content="Koodali T. Nishant" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author_institution" content="Center for Computation, Modelling, and Simulation, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author_email" content="nishantkt@iisertvm.ac.in" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=W. Amos;citation_title=Heterozygosity and mutation rate: evidence for an interaction and its implications: the potential for meiotic gene conversions to influence both mutation rate and distribution.;citation_pages=82-90;citation_volume=32;citation_year=2010;citation_issue=1;citation_pmid=19967709;citation_doi=10.1002/bies.200900108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Andersen;citation_author=Z. W. Nelson;citation_author=E. D. Hetrick;citation_author=D. E. Gottschling;citation_title=A genetic screen for increased loss of heterozygosity in Saccharomyces cerevisiae.;citation_volume=179;citation_year=2008;citation_pmid=18562670;citation_doi=10.1534/genetics.108.089250" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=Z. Antunovics;citation_author=H. V. Nguyen;citation_author=C. Gaillardin;citation_author=M. Sipiczki;citation_title=Gradual genome stabilisation by progressive reduction of the Saccharomyces uvarum genome in an interspecific hybrid with Saccharomyces cerevisiae.;citation_volume=5;citation_year=2005;citation_pmid=15982931;citation_doi=10.1016/j.femsyr.2005.04.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Argueso;citation_author=J. Wanat;citation_author=Z. Gemici;citation_author=E. Alani;citation_title=Competing Crossover Pathways Act During Meiosis in Saccharomyces cerevisiae;citation_pages=1805-1816;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611158;citation_doi=10.1534/genetics.104.032912" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Argueso;citation_author=M. F. Carazzolle;citation_author=P. A. Mieczkowski;citation_author=F. M. Duarte;citation_author=O. V. Netto;citation_title=Genome structure of a Saccharomyces cerevisiae strain widely used in bioethanol production.;citation_volume=19;citation_year=2009;citation_pmid=19812109;citation_doi=10.1101/gr.091777.109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Barbera;citation_author=T. D. Petes;citation_title=Selection and analysis of spontaneous reciprocal mitotic cross-overs in Saccharomyces cerevisiae.;citation_volume=103;citation_year=2006;citation_pmid=16908833;citation_doi=10.1073/pnas.0605778103" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Bergelson;citation_author=F. Roux;citation_title=Towards identifying genes underlying ecologically relevant traits in Arabidopsis thaliana.;citation_pages=867-879;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21085205;citation_doi=10.1038/nrg2896" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. R. Borneman;citation_author=B. A. Desany;citation_author=D. Riches;citation_author=J. P. Affourtit;citation_author=A. H. Forgan;citation_title=Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of Saccharomyces cerevisiae.;citation_pages=e1001287-e1001287;citation_volume=7;citation_year=2011;citation_issue=2;citation_pmid=21304888;citation_doi=10.1371/journal.pgen.1001287" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. K. Burke;citation_author=J. P. Dunham;citation_author=P. Shahrestani;citation_author=K. R. Thornton;citation_author=M. R. Rose;citation_title=Genome-wide analysis of a long-term evolution experiment with Drosophila.;citation_pages=587-590;citation_volume=467;citation_year=2010;citation_issue=7315;citation_pmid=20844486;citation_doi=10.1038/nature09352" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. K. Cavenee;citation_author=T. P. Dryja;citation_author=R. A. Phillips;citation_author=W. F. Benedict;citation_author=R. Godbout;citation_title=Expression of recessive alleles by chromosomal mechanisms in retinoblastoma.;citation_pages=779-784;citation_volume=305;citation_year=1983;citation_issue=5937;citation_pmid=6633649;citation_doi=10.1038/305779a0" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=U. Chakraborty;citation_author=E. Alani;citation_title=Understanding how mismatch repair proteins participate in the repair/anti-recombination decision.;citation_volume=16;citation_year=2016;citation_pmid=27573382;citation_doi=10.1093/femsyr/fow071" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=S. Y. Chen;citation_author=T. Tsubouchi;citation_author=B. Rockmill;citation_author=J. S. Sandler;citation_author=D. R. Richards;citation_title=Global analysis of the meiotic crossover landscape.;citation_pages=401-415;citation_volume=15;citation_year=2008;citation_issue=3;citation_pmid=18691940;citation_doi=10.1016/j.devcel.2008.07.006" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. A. Cubillos;citation_author=E. Billi;citation_author=E. Zorgo;citation_author=L. Parts;citation_author=P. Fargier;citation_title=Assessing the complex architecture of polygenic traits in diverged yeast populations.;citation_pages=1401-1413;citation_volume=20;citation_year=2011;citation_issue=7;citation_pmid=21261765;citation_doi=10.1111/j.1365-294X.2011.05005.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Dayani;citation_author=G. Simchen;citation_author=M. Lichten;citation_title=Meiotic recombination intermediates are resolved with minimal crossover formation during return-to-growth, an analogue of the mitotic cell cycle.;citation_pages=e1002083-e1002083;citation_volume=7;citation_year=2011;citation_issue=5;citation_pmid=21637791;citation_doi=10.1371/journal.pgen.1002083" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. R. Denver;citation_author=K. Morris;citation_author=M. Lynch;citation_author=L. L. Vassilieva;citation_author=W. K. Thomas;citation_title=High Direct Estimate of the Mutation Rate in the Mitochondrial Genome of Caenorhabditis elegans;citation_pages=2342-2344;citation_volume=289;citation_year=2000;citation_issue=5488;citation_pmid=11009418;citation_doi=10.1126/science.289.5488.2342" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. R. Denver;citation_author=K. Morris;citation_author=M. Lynch;citation_author=W. K. Thomas;citation_title=High mutation rate and predominance of insertions in the Caenorhabditis elegans nuclear genome.;citation_pages=679-682;citation_volume=430;citation_year=2004;citation_issue=7000;citation_pmid=15295601;citation_doi=10.1038/nature02697" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Dunham;citation_author=H. Badrane;citation_author=T. Ferea;citation_author=J. Adams;citation_author=P. O. Brown;citation_title=Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae.;citation_volume=99;citation_year=2002;citation_pmid=12446845;citation_doi=10.1073/pnas.242624799" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. Dunn;citation_author=T. Paulish;citation_author=A. Stanbery;citation_author=J. Piotrowski;citation_author=G. Koniges;citation_title=Recurrent rearrangement during adaptive evolution in an interspecific yeast hybrid suggests a model for rapid introgression.;citation_pages=e1003366;citation_volume=9;citation_year=2013;citation_pmid=23555283;citation_doi=10.1371/journal.pgen.1003366" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=R. Mott;citation_title=Finding the molecular basis of quantitative traits: successes and pitfalls.;citation_pages=437-445;citation_volume=2;citation_year=2001;citation_issue=6;citation_pmid=11389460;citation_doi=10.1038/35076585" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. M. Flynn;citation_author=F. J. Chain;citation_author=D. J. Schoen;citation_author=M. E. Cristescu;citation_title=Spontaneous mutation accumulation in Daphnia pulex in selection-free vs. competitive environments.;citation_volume=34;citation_year=2017;citation_pmid=27777284;citation_doi=10.1093/molbev/msw234" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. C. Gerstein;citation_author=A. Kuzmin;citation_author=S. P. Otto;citation_title=Loss-of-heterozygosity facilitates passage through Haldane’s sieve for Saccharomyces cerevisiae undergoing adaptation.;citation_pages=3819;citation_volume=5;citation_year=2014;citation_pmid=24804896" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Greig;citation_author=E. J. Louis;citation_author=R. H. Borts;citation_author=M. Travisano;citation_title=Hybrid Speciation in Experimental Populations of Yeast;citation_pages=1773-1775;citation_volume=298;citation_year=2002;citation_issue=5599;citation_pmid=12459586;citation_doi=10.1126/science.1076374" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. Gresham;citation_author=D. M. Ruderfer;citation_author=S. C. Pratt;citation_author=J. Schacherer;citation_author=M. J. Dunham;citation_title=Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray.;citation_volume=311;citation_year=2006;citation_pmid=16527929;citation_doi=10.1126/science.1123726" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Haag-Liautard;citation_author=M. Dorris;citation_author=X. Maside;citation_author=S. Macaskill;citation_author=D. L. Halligan;citation_title=Direct estimation of per nucleotide and genomic deleterious mutation rates in Drosophila.;citation_pages=82-85;citation_volume=445;citation_year=2007;citation_issue=7123;citation_pmid=17203060;citation_doi=10.1038/nature05388" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=D. L. Halligan;citation_author=P. D. Keightley;citation_title=Spontaneous mutation accumulation studies in evolutionary genetics.;citation_pages=151-172;citation_volume=40;citation_year=2009;citation_doi=10.1146/annurev.ecolsys.39.110707.173437" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. E. Hansche;citation_author=V. Beres;citation_author=P. Lange;citation_title=GENE DUPLICATION IN SACCHAROMYCES CEREVISIAE;citation_pages=673-687;citation_volume=88;citation_year=1978;citation_issue=4;citation_pmid=348562" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Heck;citation_author=J. L. Argueso;citation_author=Z. Gemici;citation_author=R. G. Reeves;citation_author=A. Bernard;citation_title=Negative epistasis between natural variants of the Saccharomyces cerevisiae MLH1 and PMS1 genes results in a defect in mismatch repair.;citation_volume=103;citation_year=2006;citation_pmid=16492773;citation_doi=10.1073/pnas.0510998103" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. B. Joseph;citation_author=D. W. Hall;citation_title=Spontaneous Mutations in Diploid Saccharomyces cerevisiae: More Beneficial Than Expected;citation_pages=1817-1825;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611159;citation_doi=10.1534/genetics.104.033761" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. D. Keightley;citation_author=U. Trivedi;citation_author=M. Thomson;citation_author=F. Oliver;citation_author=S. Kumar;citation_title=Analysis of the genome sequences of three Drosophila melanogaster spontaneous mutation accumulation lines.;citation_volume=19;citation_year=2009;citation_pmid=19439516;citation_doi=10.1101/gr.091231.109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. C. Kelly;citation_author=F. P. Shewmaker;citation_author=D. Kryndushkin;citation_author=R. B. Wickner;citation_title=Sex, prions, and plasmids in yeast.;citation_volume=109;citation_year=2012;citation_pmid=22949655;citation_doi=10.1073/pnas.1213449109" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=M. Knop;citation_title=Evolution of the hemiascomycete yeasts: on life styles and the importance of inbreeding.;citation_pages=696-708;citation_volume=28;citation_year=2006;citation_issue=7;citation_pmid=16929561;citation_doi=10.1002/bies.20435" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. N. Krishnaprasad;citation_author=M. T. Anand;citation_author=G. Lin;citation_author=M. M. Tekkedil;citation_author=L. M. Steinmetz;citation_title=Variation in crossover frequencies perturb crossover assurance without affecting meiotic chromosome segregation in Saccharomyces cerevisiae.;citation_volume=199;citation_year=2015;citation_pmid=25467183;citation_doi=10.1534/genetics.114.172320" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. Laureau;citation_author=S. Loeillet;citation_author=F. Salinas;citation_author=A. Bergstrom;citation_author=P. Legoix-Ne;citation_title=Extensive recombination of a yeast diploid hybrid through meiotic reversion.;citation_pages=e1005781;citation_volume=12;citation_year=2016;citation_pmid=26828862;citation_doi=10.1371/journal.pgen.1005781" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Lee;citation_author=E. Popodi;citation_author=H. Tang;citation_author=P. L. Foster;citation_title=Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing.;citation_volume=109;citation_year=2012;citation_pmid=22991466;citation_doi=10.1073/pnas.1210309109" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. S. Lee;citation_author=P. W. Greenwell;citation_author=M. Dominska;citation_author=M. Gawel;citation_author=M. Hamilton;citation_title=A fine-structure map of spontaneous mitotic crossovers in the yeast Saccharomyces cerevisiae.;citation_pages=e1000410-e1000410;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19282969;citation_doi=10.1371/journal.pgen.1000410" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=W. Li;citation_author=A. F. Averette;citation_author=M. Desnos-Ollivier;citation_author=M. Ni;citation_author=F. Dromer;citation_title=Genetic diversity and genomic plasticity of Cryptococcus neoformans AD hybrid strains.;citation_pages=83-97;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Liti;citation_author=D. B. Barton;citation_author=E. J. Louis;citation_title=Sequence diversity, reproductive isolation and species concepts in Saccharomyces.;citation_volume=174;citation_year=2006;citation_pmid=16951060;citation_doi=10.1534/genetics.106.062166" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Liu;citation_author=W. A. Gaines;citation_author=T. Callender;citation_author=V. Busygina;citation_author=A. Oke;citation_title=Down-regulation of Rad51 activity during meiosis in yeast prevents competition with Dmc1 for repair of double-strand breaks.;citation_pages=e1004005;citation_volume=10;citation_year=2014;citation_pmid=24465215;citation_doi=10.1371/journal.pgen.1004005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast.;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_title=Adv. Exp. Med. Biol.;citation_author=P. M. Magwene;citation_title=Revisiting Mortimer’s genome renewal hypothesis: heterozygosity, homothallism, and the potential for adaptation in yeast.;citation_pages=37-48;citation_volume=781;citation_year=2014;citation_pmid=24277294;citation_doi=10.1007/978-94-007-7347-9_3" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. M. Magwene;citation_author=Ö. Kayıkçı;citation_author=J. A. Granek;citation_author=J. M. Reininga;citation_author=Z. Scholl;citation_title=Outcrossing, mitotic recombination, and life-history trade-offs shape genome evolution in Saccharomyces cerevisiae.;citation_volume=108;citation_year=2011;citation_pmid=21245305;citation_doi=10.1073/pnas.1012544108" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Mancera;citation_author=R. Bourgon;citation_author=A. Brozzi;citation_author=W. Huber;citation_author=L. M. Steinmetz;citation_title=High-resolution mapping of meiotic crossovers and non-crossovers in yeast.;citation_pages=479-485;citation_volume=454;citation_year=2008;citation_issue=7203;citation_pmid=18615017;citation_doi=10.1038/nature07135" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=M. A. Mandegar;citation_author=S. P. Otto;citation_title=Mitotic recombination counteracts the benefits of genetic segregation;citation_pages=1301-1307;citation_volume=274;citation_year=2007;citation_issue=1615;citation_pmid=17360283;citation_doi=10.1098/rspb.2007.0056" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Martini;citation_author=V. Borde;citation_author=M. Legendre;citation_author=S. Audic;citation_author=B. Regnault;citation_title=Genome-wide analysis of heteroduplex DNA in mismatch repair-deficient yeast cells reveals novel properties of meiotic recombination pathways.;citation_pages=e1002305-e1002305;citation_volume=7;citation_year=2011;citation_issue=9;citation_pmid=21980306;citation_doi=10.1371/journal.pgen.1002305" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. H. McCusker;citation_author=K. V. Clemons;citation_author=D. A. Stevens;citation_author=R. W. Davis;citation_title=Genetic characterization of pathogenic Saccharomyces cerevisiae isolates.;citation_pages=1261-1269;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013903" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Montgomery;citation_author=S. M. Huang;citation_author=C. H. Langley;citation_author=B. H. Judd;citation_title=Chromosome rearrangement by ectopic recombination in Drosophila melanogaster: genome structure and evolution.;citation_pages=1085-1098;citation_volume=129;citation_year=1991;citation_issue=4;citation_pmid=1783293" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=L. Morales;citation_author=B. Dujon;citation_title=Evolutionary Role of Interspecies Hybridization and Genetic Exchanges in Yeasts;citation_pages=721-739;citation_volume=76;citation_year=2012;citation_issue=4;citation_pmid=23204364;citation_doi=10.1128/MMBR.00022-12" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=R. K. Mortimer;citation_author=P. Romano;citation_author=G. Suzzi;citation_author=M. Polsinelli;citation_title=Genome renewal: a new phenomenon revealed from a genetic study of 43 strains of Saccharomyces cerevisiae derived from natural fermentation of grape musts.;citation_pages=1543-1552;citation_volume=10;citation_year=1994;citation_issue=12;citation_pmid=7725789;citation_doi=10.1002/yea.320101203" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=K. T. Nishant;citation_author=N. D. Singh;citation_author=E. Alani;citation_title=Genomic mutation rates: what high-throughput methods can tell us.;citation_pages=912-920;citation_volume=31;citation_year=2009;citation_issue=9;citation_pmid=19644920;citation_doi=10.1002/bies.200900017" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. T. Nishant;citation_author=W. Wei;citation_author=E. Mancera;citation_author=J. L. Argueso;citation_author=A. Schlattl;citation_title=The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.;citation_pages=e1001109;citation_volume=6;citation_year=2010;citation_pmid=20838597;citation_doi=10.1371/journal.pgen.1001109" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Oke;citation_author=C. M. Anderson;citation_author=P. Yam;citation_author=J. C. Fung;citation_title=Controlling meiotic recombinational repair – specifying the roles of ZMMs, Sgs1 and Mus81/Mms4 in crossover formation.;citation_pages=e1004690;citation_volume=10;citation_year=2014;citation_pmid=25329811;citation_doi=10.1371/journal.pgen.1004690" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. Palmer;citation_author=E. Schildkraut;citation_author=R. Lazarin;citation_author=J. Nguyen;citation_author=J. A. Nickoloff;citation_title=Gene conversion tracts in Saccharomyces cerevisiae can be extremely short and highly directional;citation_pages=1164-1173;citation_volume=31;citation_year=2003;citation_issue=4;citation_pmid=12582235;citation_doi=10.1093/nar/gkg219" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=F. Paques;citation_author=J. E. Haber;citation_title=Multiple Pathways of Recombination Induced by Double-Strand Breaks in Saccharomyces cerevisiae;citation_pages=349-404;citation_volume=63;citation_year=1999;citation_issue=2;citation_pmid=10357855" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=L. P. Pryszcz;citation_author=T. Nemeth;citation_author=A. Gacser;citation_author=T. Gabaldon;citation_title=Genome comparison of Candida orthopsilosis clinical strains reveals the existence of hybrids between two distinct subspecies.;citation_volume=6;citation_year=2014;citation_pmid=24747362;citation_doi=10.1093/gbe/evu082" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Qi;citation_author=A. J. Wijeratne;citation_author=L. P. Tomsho;citation_author=Y. Hu;citation_author=S. C. Schuster;citation_title=Characterization of meiotic crossovers and gene conversion by whole-genome sequencing in Saccharomyces cerevisiae.;citation_pages=475-475;citation_volume=10;citation_year=2009;citation_pmid=19832984;citation_doi=10.1186/1471-2164-10-475" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=A. Querol;citation_author=U. Bond;citation_title=The complex and dynamic genomes of industrial yeasts.;citation_volume=293;citation_year=2009;citation_pmid=19175410;citation_doi=10.1111/j.1574-6968.2008.01480.x" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics: A Laboratory Course Manual;citation_author=M. D. Rose;citation_author=F. M. Winston;citation_author=P. Heiter;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. M. Rosen;citation_author=E. M. Younkin;citation_author=S. D. Miller;citation_author=A. M. Casper;citation_title=Fragile site instability in Saccharomyces cerevisiae causes loss of heterozygosity by mitotic crossovers and break-induced replication.;citation_pages=e1003817;citation_volume=9;citation_year=2013;citation_pmid=24068975;citation_doi=10.1371/journal.pgen.1003817" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. M. Ruderfer;citation_author=S. C. Pratt;citation_author=H. S. Seidel;citation_author=L. Kruglyak;citation_title=Population genomic analysis of outcrossing and recombination in yeast.;citation_pages=1077-1081;citation_volume=38;citation_year=2006;citation_issue=9;citation_pmid=16892060;citation_doi=10.1038/ng1859" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=D. Houle;citation_author=M. Lynch;citation_author=M. W. Hahn;citation_title=Rates and genomic consequences of spontaneous mutational events in Drosophila melanogaster.;citation_volume=194;citation_year=2013;citation_pmid=23733788;citation_doi=10.1534/genetics.113.151670" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=A. Selmecki;citation_author=A. Forche;citation_author=J. Berman;citation_title=Genomic plasticity of the human fungal pathogen Candida albicans.;citation_volume=9;citation_year=2010;citation_pmid=20495058;citation_doi=10.1128/EC.00060-10" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. St Charles;citation_author=T. D. Petes;citation_title=High-resolution mapping of spontaneous mitotic recombination hotspots on the 1.1 Mb arm of yeast chromosome IV.;citation_pages=e1003434;citation_volume=9;citation_year=2013;citation_pmid=23593029;citation_doi=10.1371/journal.pgen.1003434" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. M. Steinmetz;citation_author=H. Sinha;citation_author=D. R. Richards;citation_author=J. I. Spiegelman;citation_author=P. J. Oefner;citation_title=Dissecting the architecture of a quantitative trait locus in yeast.;citation_pages=326-330;citation_volume=416;citation_year=2002;citation_issue=6878;citation_pmid=11907579;citation_doi=10.1038/416326a" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=R. Stelkens;citation_author=M. Brockhurst;citation_author=G. Hurst;citation_author=E. Miller;citation_author=D. Greig;citation_title=The effect of hybrid transgression on environmental tolerance in experimental yeast crosses.;citation_pages=2507-2519;citation_volume=27;citation_year=2014;citation_pmid=25262771;citation_doi=10.1111/jeb.12494" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. N. Strathern;citation_author=B. K. Shafer;citation_author=C. B. McGill;citation_title=DNA synthesis errors associated with double-strand-break repair.;citation_pages=965-972;citation_volume=140;citation_year=1995;citation_issue=3;citation_pmid=7672595" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=S. Tang;citation_author=R. A. Okashah;citation_author=S. J. Knapp;citation_author=M. L. Arnold;citation_author=N. H. Martin;citation_title=Transmission ratio distortion results in asymmetric introgression in Louisiana Iris.;citation_pages=48-48;citation_volume=10;citation_year=2010;citation_pmid=20298609" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. L. Vassilieva;citation_author=A. M. Hook;citation_author=M. Lynch;citation_title=The fitness effects of spontaneous mutations in Caenorhabditis elegans.;citation_pages=1234-1246;citation_volume=54;citation_year=2000;citation_issue=4;citation_pmid=11005291;citation_doi=10.1554/0014-3820(2000)054[1234:TFEOSM]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Wei;citation_author=J. H. McCusker;citation_author=R. W. Hyman;citation_author=T. Jones;citation_author=Y. Ning;citation_title=Genome sequencing and comparative analysis of Saccharomyces cerevisiae strain YJM789.;citation_volume=104;citation_year=2007;citation_pmid=17652520;citation_doi=10.1073/pnas.0701291104" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. M. Wloch;citation_author=K. Szafraniec;citation_author=R. H. Borts;citation_author=R. Korona;citation_title=Direct Estimate of the Mutation Rate and the Distribution of Fitness Effects in the Yeast Saccharomyces cerevisiae;citation_pages=441-452;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606524" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=K. H. Wolfe;citation_title=Origin of the yeast whole-genome duplication.;citation_pages=e1002221;citation_volume=13;citation_year=2015;citation_pmid=26252643;citation_doi=10.1371/journal.pbio.1002221" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Yang;citation_author=L. Wang;citation_author=J. Huang;citation_author=X. Zhang;citation_author=Y. Yuan;citation_title=Parent-progeny sequencing indicates higher mutation rates in heterozygotes.;citation_pages=463-467;citation_volume=523;citation_year=2015;citation_pmid=26176923;citation_doi=10.1038/nature14649" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Yang;citation_author=J. Sterling;citation_author=F. Storici;citation_author=M. A. Resnick;citation_author=D. A. Gordenin;citation_title=Hypermutability of damaged single-strand DNA formed at double-strand breaks and uncapped telomeres in yeast Saccharomyces cerevisiae.;citation_pages=e1000264-e1000264;citation_volume=4;citation_year=2008;citation_issue=11;citation_pmid=19023402;citation_doi=10.1371/journal.pgen.1000264" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Yim;citation_author=K. E. O’Connell;citation_author=J. S. Charles;citation_author=T. D. Petes;citation_title=High-resolution mapping of two types of spontaneous mitotic gene conversion events in Saccharomyces cerevisiae.;citation_volume=198;citation_year=2014;citation_pmid=24990991;citation_doi=10.1534/genetics.114.167395" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Yin;citation_author=T. D. Petes;citation_title=Genome-wide high-resolution mapping of UV-induced mitotic recombination events in Saccharomyces cerevisiae.;citation_pages=e1003894;citation_volume=9;citation_year=2013;citation_pmid=24204306;citation_doi=10.1371/journal.pgen.1003894" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. O. Zhu;citation_author=M. L. Siegal;citation_author=D. W. Hall;citation_author=D. A. Petrov;citation_title=Precise estimates of mutation rate and spectrum in yeast.;citation_volume=111;citation_year=2014;citation_pmid=24847077;citation_doi=10.1073/pnas.1323011111" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome Dynamics of Hybrid Saccharomyces cerevisiae During Vegetative and Meiotic Divisions" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="Mutation and recombination are the major sources of genetic diversity in all organisms. In the baker’s yeast, all mutation rate estimates are in homozygous background. We determined the extent of genetic change through mutation and loss of heterozygosity (LOH) in a heterozygous Saccharomyces cerevisiae genome during successive vegetative and meiotic divisions. We measured genome-wide LOH and base mutation rates during vegetative and meiotic divisions in a hybrid (S288c/YJM789) S. cerevisiae strain. The S288c/YJM789 hybrid showed nearly complete reduction in heterozygosity within 31 generations of meioses and improved spore viability. LOH in the meiotic lines was driven primarily by the mating of spores within the tetrad. The S288c/YJM789 hybrid lines propagated vegetatively for the same duration as the meiotic lines, showed variable LOH (from 2 to 3% and up to 35%). Two of the vegetative lines with extensive LOH showed frequent and large internal LOH tracts that suggest a high frequency of recombination repair. These results suggest significant LOH can occur in the S288c/YJM789 hybrid during vegetative propagation presumably due to return to growth events. The average base substitution rates for the vegetative lines (1.82 × 10−10 per base per division) and the meiotic lines (1.22 × 10−10 per base per division) are the first genome-wide mutation rate estimates for a hybrid yeast. This study therefore provides a novel context for the analysis of mutation rates (especially in the context of detecting LOH during vegetative divisions), compared to previous mutation accumulation studies in yeast that used homozygous backgrounds." />
<meta name="og-title" property="og:title" content="Genome Dynamics of Hybrid Saccharomyces cerevisiae During Vegetative and Meiotic Divisions" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3669" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mutation and recombination are the major sources of genetic diversity in all organisms. In the baker’s yeast, all mutation rate estimates are in homozygous background. We determined the extent of genetic change through mutation and loss of heterozygosity (LOH) in a heterozygous Saccharomyces cerevisiae genome during successive vegetative and meiotic divisions. We measured genome-wide LOH and base mutation rates during vegetative and meiotic divisions in a hybrid (S288c/YJM789) S. cerevisiae strain. The S288c/YJM789 hybrid showed nearly complete reduction in heterozygosity within 31 generations of meioses and improved spore viability. LOH in the meiotic lines was driven primarily by the mating of spores within the tetrad. The S288c/YJM789 hybrid lines propagated vegetatively for the same duration as the meiotic lines, showed variable LOH (from 2 to 3% and up to 35%). Two of the vegetative lines with extensive LOH showed frequent and large internal LOH tracts that suggest a high frequency of recombination repair. These results suggest significant LOH can occur in the S288c/YJM789 hybrid during vegetative propagation presumably due to return to growth events. The average base substitution rates for the vegetative lines (1.82 × 10−10 per base per division) and the meiotic lines (1.22 × 10−10 per base per division) are the first genome-wide mutation rate estimates for a hybrid yeast. This study therefore provides a novel context for the analysis of mutation rates (especially in the context of detecting LOH during vegetative divisions), compared to previous mutation accumulation studies in yeast that used homozygous backgrounds." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>Genome Dynamics of Hybrid Saccharomyces cerevisiae During Vegetative and Meiotic Divisions | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__Ta-UXINC0YONAj3hv7M-Srfc-_B4Wn0KATTOQGieTps__agTSMicuoC50kRvu-VcTNoPubLJgflHIUuLq_Jt5dMY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aEyDJzQaWX-ltZlOOyoaFaAe2eUe9SpR3EJQ2CJlcrU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487109","apath":"\/ggg\/7\/11\/3669.atom","pisa":"ggg;7\/11\/3669","ac":{"\/ggg\/7\/11\/3669.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3669.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3669"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3669"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3669":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oYVo4HEHo8AYfVaChgd5d9TZ0LbB9WlbHTretehjNVUMRFMqxR9a5wLqZB\\\\\\\/MXo1rP0LjYXQ4S4jDDljv92XYsCKW4hvPLFool35Ykx3Zb5TlwExNIG+Uh1KejBNr0EH0V92oIkeEo17q1O\\\\\\\/bZC86hvmTKfzCT9yvlyCiR6cll0u4kwPr8ofQLHPMQ9F648Ba3k2EBtpm3HOryANOflw3mfZ\\\\\\\/18g7slAbFskdEvBGeiIzKsbsghRb08V2iRfk1Tg\\\\\\\/yDzqeO5j7AnGq7Ca\\\\\\\/tB1wu08eNpA56gZ8aAGfPcutt2NUzavQdWje\\\\\\\/D7LKmmvbnwB1MjPqW2woUxf+pZpOm1l\\\\\\\/RTRgriK5RpELf+X\\\\\\\/sH+r49iPiNCcmZLhWRrolXpSD4VC+QJiM3ocE+fYVYPZGpsLQNd863n3OQkRBxDrn0M6XKlSFsh07j2ox+ezIi2FFXg22cfTgnadcgJg8lJdFySqg4mRLPSpk3i2tGpDdH3RgCt5FVbs6wPpIK2YTakPuBxo5e0c1RSGEb6FOKk+j4cGTA49XOduLKhH3sJXsDwKnBjjnBhk2VEsCdlk2f\\\\\\\/nBs5t+wm1KtIV6qoml3ukiRjD558kgdG9fQqBUtp38w+\\\\\\\/fuwJZXubrfGxy2aMwYTz\\\\\\\/wOwQE+J+5wu2+nLwVYZPy49hGuY9K9LU3xTtvE0Pf3VlGyifwxvcmzTYrVCZAy00TpFxW7fRQQHW1YusUo5KR6Lru40EhewtyQVRZdmLCAygGFGAjHaD29yrTprfJtNxvJ6oa5LpAk1wAIKV4LJYYFX8yJLaebioPBfp1wkgcqUOpuL+ZJN+gAB7HY24tzks10M8winRa6gdRUVoGpeJD+C5SUeUeFCnF\\\\\\\/OP9uw6s2\\\\\\\/oJUGUBHgth16BaKrr1nECYx0Ln+Ic+GsmQG4xRogDnYBPX9\\\\\\\/SsBJ0VcxDiQnDClKHShm5QY2U5nlz29wFoJH7bLECSBqrzAxnadMlXxwSEI0IVgeFAnidgbj+W\\\\\\\/O9w4H\\\\\\\/hdEkQOlaP6bb1NUYanKT4ilN\\\\\\\/Mph4Csjsw6aZAt7WhykG02t09pyUEyI6fRXk5EFvw7Etqrq+NOoYO7ywuUj3gXaUB4pzhGCCWrpI9aqFuJjN2d9URjNfA6kZw+S+JNT3s\\\\\\\/r+WxvYkns5rm\\\\\\\/p537Pt+aJ0BglDdqDUWKT3ooH8m+PNANyF9jg0gtTi3SWUvCthg3EPGtsjkgk96PBpNQW6mtr9OYOJZsolt\\\\\\\/Ph5W+tAQDsUO0AOWHMr5xX1elyJ4mEz0ZERxohTPeP2VKz\\\\\\\/G2\\\u0022,\\\u0022iv\\\u0022:\\\u0022eE85RT5PvOShGZyZfAnWfA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226ef0d0d05c4cfbdfa0d0f33fe14da737ac2eea2ef783be4a4713ccb3f5c7735b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TyfZcM5dlfKRblG8QhGjEzK6N1vO2n4VTflzPSaKgDWNmEjWtX83B3gwV+uAoz0M6TpPI9M9v5Y+S3m\\\\\\\/RiDKb4XNtKx0Ttc6uqOAy7Tx7VMKOzfX+gLGU1DeSZeGGTYVu+Sf3CafH0svmJhMYvNmvKZD8uSk0ucHUq\\\\\\\/mOM9GMlGgQs2I5w6VVJ9nh4brEhaikAwFu6U7banByZGZqUJRuQVe6lsmvDcu8mKB6t1aOfX54dA9i2ahb+GN3w9U1rBW+XDrdtNd6WsxrQVFieFv67LcS8yHngzyzVYZaEBYs+Ua5M8hEwFM7Zxo7TpA5Bl6wjx1E\\\\\\\/12bL7g7p+xRmBHbgVPOZ0cdB9nkq5OWF1FOhAmzdNrqgM002COE4TYPsQz9tT5sVSloibGmxSmsEdeDcu9Q3ZnHD6dZNV78nmci7bduJpBHBKZPd776oD9nYO9LIWwqv6zvjuobrxUik0Ks+SPNGrAGG6AFBnhqL7ZgWr3oJEZJ\\\\\\\/uuY1jqdXzRtx0rRfg\\\\\\\/9XocbWuqxg6xmboPaJVBWlH+QdHOQwm0CNBcYJELshA+2tSEbIkGlJvzuGK9j4Vutn5vKwPLKgpw1BSsADH3QbS1bePI17V3BpfzmEZdKiEXzDxdk5AfQqQ5tKD73b4huYkEjhZzbJqWnCaDaT7qNfPAOdUmKXl4vuZUs7WXUZ9PEfNk9oMgSSnEiHSmRCc\\\\\\\/VzCUFa+L6eYlxXr\\\\\\\/vb8WyZTEYIZ12m1XEl4cXt\\\\\\\/7bFPetYYVTfWTtbC5FwtUWiqHhdy868sY3eCl\\\\\\\/o5mYG3VxqZKtUDJdkvDvc\\\\\\\/ytS\\\\\\\/vFYE8a\\\\\\\/0ZCrqDQ5FPxLXNuaOQgPGh+tUkyaPKZDQsBfBnIGWyVqUOPqcUi1ipavZ7SJkbQm0T9m1+WlydZh+EjiezKaXWiNdEosSpGa44y9\\\\\\\/iJ9l5x5t+jb0sgCvmgGQNPZTkwl92gGuuQ9htGdzbgGFW0KRg7vwtq9H89+PWuW8NIDx1CkgGnViyZRZ3e05WrZ2MFvSX9TWG9iE7LflDnQPgyny7todjN5EdtESbmJtysguz4QouWCvolwoRCcEitvOOtbN4FfeHJspHleHl88v6k5m5o9ey3alkNoEVVc0eR6HoSh3YHxrBNRPNMa413Jabq1SOzy+VIhH7CBempzIg+JOrKwhWUcZ9Mw+Tyz+TRygFxGwBJcJLUeQE1PcTm6mtnWT4k8jt0SHXDtcLOO+QJju1hczGXfVfmXWlr+o7x9UGjG6k3qlnmoh6uO59gXmKm5Pb\\\\\\\/AsUCd\\\\\\\/FKxiAyDl70MD7fK49LTbdLQ9xHb9kzy2mKuawE3m0UJPf8U2yYOLDrVsqRHWP6iOyaFl5ki+XRu\\\\\\\/BGHwBD8E4pKa5xUtr\\\\\\\/A2+AS+8sKMJqqx8\\\\\\\/ZEf6GaHmfJNfI2a2+F2wCVQ4osFFCucbuOZmgZrOr\\\\\\\/wmodvOD9VPBrL6g4IPk7pd4Q+0P6CB\\\\\\\/Brg4jxxsukaVQ3mXag1+LBe0e5SSf9TiNfP8CMVOUd48UcWPh0GvPrmBCxIwO7\\\\\\\/8To7tHr8qwdNwYrW\\\\\\\/6j77XL+DP0LTlV3rEHZK5tquJf1sIKXLXvNjI9NQMvppsT+P7TK6k8\\\\\\\/m00L9lMWMviSs0E0wpWkixC0X2WC20EGCB8mr1VlSIADkq0vwZVh76OxagambabAyw08xu\\\\\\\/zU39DIcQ6dZOWYEQO6e4uCjTnyHFbIR\\\\\\\/uekay0TJ0CMQz6G3znoTV9MCgDddk4QUcaxfqopH1SePWfIIiarRUYLMPmB04YqDctp\\\\\\\/7YDtCBNVadoZHYogy3z6cx8iQR3eY8fsBze\\\\\\\/MJjHuOEaDNqpI6eADjY3ZbHli+79FJEvNtlhXQhQOuAHZFpJS2Oxni+Jz5UryY7fudqJiquEt9UFPlcCfytjV45EYaCL79nduLEzvcdt7xCk9oZUxHNcJ2iCrotF9jQz2HiD3dE7lI81aKRTQgxTO0kZKpOsu1d+BJ7UGjjCaVgxQ4FKHx+bwkGIGS06YAgMSD2VkTZz7TUDwR218Gz8nLa2lXJM6BhCdIlSB2H3V9xl+DjAsPt\\\\\\\/b5JYRTGcN5kRfY85vKqWDPbTNu28Y3JdmPiWOFg9TnHXQVBY0Kcmpetule9Vd8UD\\\\\\\/ql0AoeWz0sxbOrY3F6Q9xI6MsOBTW5h2WkUeZ8VMWA77LRdWq9MY9frFU2zU\\\\\\\/bFCCLl6czx7DfyW1zkKLbQM6T6wZ5nFxGQ9\\\\\\\/fJrxkEcQLApMjSjS2CQ+FD\\\\\\\/tP0fw5suAPvQD9wnt0pGIwovsHnuib1ZkltazQIXPaJM4LCD\\\\\\\/yendK1qxjoZ3c+z4T8CPJr24nI7Hsp3TyW\\\\\\\/UeHR8McToycpJUlTnu4irSwK2E7neBZ+MgmajFRUA1hyCjdy5nhnFlIDDhHnhZFnuwBe6i7GKFsYyzp48kZn7Jq6jJ2CMG6fU176wLGDWjHuHJtMY3hvMxTX4No4l2bnC6LY\\\\\\\/wOqeizTP0MTmQn8etYKUj8QIyPBLXha5qMj6UIEMbfiAtFDwk8HH0jvNxvh5nqXG6NElhd5AHwb4Jg12LpMQ0zuucr4QzNNz5QkBKfRneN2oYpnLwr7ZVU7Q+3qh6hv2BQY812JuFmyoiGfvdH\\\\\\\/1AJMYc29NeTx1T\\\\\\\/wDrk1IhDQscTmTHAfhY\\\\\\\/BQmRhEwXLZjCRV6ogjodQcuV4wFgvbE0Xbm\\\\\\\/SACNiX+REav8HK6HJuPykuPQ2O6oL3PZ6JD52ndBOGkceFW\\\\\\\/YqdyCqcYuGh1\\\\\\\/Ir\\\\\\\/4ZrcOBpOmUxE9T87R8\\\\\\\/h56Nynu7M8JnUaXpS3i\\\\\\\/OCffj50zTop2tDX3kF2KHUUWDUgNTuIC8Xti9CUHg25Ry+QZFzCvssZPF+bwUTNiG\\\\\\\/oAblRI2NHAAdfdjWZnSTerTJqio5wvu1JliFJgZEiDdS14+O809QG6l67ImsrWn8PPFpE1HirZUW3eEHeCB\\\\\\\/Da81Q2MsW4Mwy5dcZgzHFLVT04NLfOQj7Foadb97hN9EfLkN6LelBPDZtDvhslgurFs1fLyN0K8ZruAN+O2EVFgUyfBVuMklACXE60gvEvZ6tgY6EWrJn7sB5sppSfa3vCAZwFZCO6Tx3sHJq71QUR2CePNXSV90JQO2bJVD2zvNyTtotrWlJRbeeNzk2YTnP1OowLgP32ahWFTpuiI+eGs5C3358Bi9OzOumI7NTTovgd0sa\\\\\\\/hKMnA2oN4oXCJ1IXPtd1CQqeMJL4jS0qeYZQE9HrxURn2xBJUXZEmWy8vA9u4LuO7ViEuNMCT2sTOwPAQIoRVg0R0QTQfjSAAfCmHG\\\\\\\/2SMHCniRghgPu\\\\\\\/Zic5MVYUf64v2s\\\\\\\/OxjqsTIALyteBp25bknAgLDKkf948t56c4a9nDtRhAplaH8Up7BH6Ynwn7xS5HbnpRnvV0quZ8Sak\\\\\\\/p8mY0YMdEXPN9QPgajJiNEm3\\\\\\\/3Wad9YDBGGm2kq+1UDmOrLjTRppPLLw3HxIfTzveDd3lhc7pRYIqVoSJ2w+4oXfKQp8AyLJyyAJ\\\\\\\/778f\\\\\\\/+Jf9AKaWz9ezBJpBHdYsjbjljCclFjh71cCEoWHgSAtxjrKCrthS\\\\\\\/TCnvBj8YLieVNQd9nOU3EEy1mxo582nahzppWubYI0YzfHRlZNEfP574sXfYtiaRVynUBZSp1cu9zCxdOEs93SxWZ6plFv17O6LvpVeDnieo5MJiWt+DU51RaM7eMQeIs\\\\\\\/p1Sdp+LJvPZAJHGqpRvPSu8R\\\\\\\/Pn46z8F6tjChl39hI8q9zhwBX729ckOT6\\\\\\\/\\\\\\\/odjJuaMwk5K8\\\\\\\/6u1KKCYTDJTbuH+HnNkMQoHpiJIjqy\\\\\\\/RQ\\\\\\\/KxkqYsAy8c8lc1YIy3KbFOXc+A2M\\\\\\\/kLm0cBK8PKV+scINJDrH6WSSl3XNaLAK0Zbt7XpYk1RlQgFWAc7h0ILRWJOv3dyBX3tYc4B8IwbWRGi4FACz3t+5U4TCWz4KHE8uuiypic07bpG7yEeU6F4tiGRzFbPTF4MHErjG7Y6zmvU30eLg83YEPnDV5Ur+mAY\\\\\\\/Sk5M6bwXg1YANYwNRPojlJDtHj4BsaUSWdgfk9jbb1OqgPrWDIWiKxvYnZIr56AhI1HwiIgdBayVUU\\\\\\\/QLS4Z+YTvCfcbdgEYolDkhbR1vpjwSsAE2xBH+nC4fR\\\\\\\/Gwhi2gPhpykzWjexV+QIapcAYwqAPbLZ6Fei37S7rpzW3EKQ74HdHjawiPQRfFait0QYYzbp+Y4wz+PlVm22VQ+eOIrJbZ5oIn+2sOQWVnDHVyVyaN8OT2xVmbs73A8dalHFZnCMu+1aXOf4HFd3cAf4y8baFVGdCR7+Ij2Rl4xcIRoaQ1IpheyH4oVV339B3QGc+RQb91i3pytN3b9xYQwlJxf0F59bMQZZqK9iYcbcAm6EEBmFydjHFl+1UqIKsx4\\\\\\\/jQpfWPCUXuxsLmgk9A8vEtfLKoAYjsDnTxTd75cBG2fNBl0o1ovHQRm40UTchM\\\\\\\/m8TDctZzx4JP2B5kmo8l14CG36gP1Evn4\\\\\\\/ayVuCdnzKsKsWEeQ79SuL1\\\\\\\/4OVdUzoP2q80W2Pe+9ZdTl+a160yjVwoJqQLvyl9VgmS8Obn+eU0uM3J12rCtVxylHhSiaJtYG0pMNld80VPj8XXafhgWaTiS+4I3Q6ZNcMCsTgvofJhIuaJzUCK6xk13AS2ChJE5jf3kFbjo+nxNIqgqHoJY2T5IA8TXO0OFSBOAS34teTnqs91DrICcr0syL5DUKOtiNLrjjPUubWJs20XdZupF6xS93yGw5UvSaHmGSggiQ\\\\\\\/8\\\\\\\/rruTXW+bYhtbRp1cQxNNYRvUf35UqkM7139JoLN5UtPAY+hA3ua8Vn+2ZqbGxXEei3dNsgJn+0iIZ6\\\\\\\/CmvUjnXnrmT8Ld6uqmw1yU\\\\\\\/NvaObABCd8\\\\\\\/PodDFosws8AaQKSzg7cI2eRbm8Ux0e1ONGu2KzbK1REjteAn9lrh\\\\\\\/AobBJycnrdBNSytMkh\\\\\\\/XcseXo70v05FL2VsR2QSJUPTau8GrM+VSWGunwxLtfTolYm5X3SFKNhgCSwfLzbp4TcdNma0lxEgeH5z4oqQo6OKX4GNJufnXqicQ5k3IyC4z+nu\\\\\\\/5PV5BHZmmZw0\\\\\\\/bgkgE9UrbYcfzCUKvbBPaU7800cd++KZWKt1njQ7XNZ1DLqze+T9GFO5CVKDWlp4+Ux4svtlKm3ZWdSOzrGYzQj2NuiD5CiIq6uG7wWtPulS6FdJcKvnaCEaTbhTiF1aU\\\\\\\/4rYDQxIXCS2B5HF1wA99ZRm1sL8aQPEKxiyE78SEM4YV4M2g2wXE8L7cA3jnFEU1\\\\\\\/3GBNWgvtiMRvO7OaMThTc9mH2+pFgm7DYYPrIE9aFMNr8YoIAmqpVXD9FuYBuWLDA\\\\\\\/3tTAJCWULSq524jUjvr6cqJegWxtfqnVFdVuyCakpTOpDZnhdBE39Nz9fymTSsDDCQTbPckvA1whzZePb4UbopQ97iuZBgE6PJTrPcC+CQ17vBfnWjJKO8KHqEWi+q9D55YInB4hzT4SmKiEbwC+S2vH4wNH2sUrrB8Dm33bspWvRAhyRBc3+J5TS2qrKDMCwXFNkmDRkUKITu7rHMBgmGqm\\\\\\\/HBCEUGtdIYBMisD+qRiVufGRJDmIrCjBt+7f6V0UN5GCXDvBok1Hw2bA8NQ+byA3dzxwFTtKtdOuQO2J2zGyNkA4NavYNNXUeqGJ4GvIUZfzzLHF0xsEbsZ5uKyKfQC1wcV5TQzIUZhSouAexpDBQQbVUTJLs1mzDTfekTxycP9MDj9wTB\\\\\\\/0sW87AKTCYeiE4PEHhVE93\\\\\\\/lZxkjrhLZpCi3z65voKXkNN4Ihjb3U3\\\\\\\/V+GDUBluT6XvRaGj+GT48EX9JjoA9a9WYp1nkM9w68Arf5IRnN5rXSww2n9VCKai+cVK7P2TVOw3z\\\\\\\/x9oJ5NfvTOBD7itAgd\\\\\\\/boRHfdAzL0NZae8BeoO92cY06e6R9Zoz7IN3qtCPuX+n6LKgFZQYHv1rGhG80WmNJDCQ+tUaYSfaoa09gjCtklNGbo43b5nrZZJWdrdP\\\\\\\/lQ3UimvU\\\\\\\/TSuVsTn2ZpWZO9qbCSxgnZLRISVcq75YnlD0cqkzmpiQs80nMlMfJ1ANDc\\\\\\\/gBd5lVs2YDKSK\\\\\\\/PWpqjYW9NB84t9T\\\\\\\/qsdf1K6MJaLrOuSU1FL7JFowiMnEV3pm+Tkuu7IJE5lbRvPYCLu9JK4u3UpeLfeJXCHN797qJf4RaBog6FcaeKaBfQTnfWpalUq7rfqlVfuMLyt+JulKeQF0pDPY3jaDl2lcLi0FsQw73Q4Vfi6DqSzESWOO4VHzogMPDs4Bj8048+ocu9WzuPMMaJPMyvHFcMSUJhhlraQBNRutUn1IYzD1h67YuXtZBTy8AhBFRi2lcBclaurvXaCpyK\\\\\\\/PIfcfekvXwptadsZ9WClemSwyZ6GHcjF8ZlNGhlNPurrR+lEq3P1Y5ks8tWqyP0Q7VAEjd6JpHG\\\\\\\/AlcMYsfh+ehja9oF\\\\\\\/JYofxrv\\\\\\\/Wj2fU4nR+5STL3UYm7zjdxcJScvk5UQyR1ZsGqYixcC9qNgfJJ9awosL8d9GVkr7saAMUNw+jI2gFqyaFJ3rKZ+B69WDrhth0+VLWEUKkx9TONhuq9e2PePoGXuervMSZPtxufi\\\\\\\/LCTh+7I+\\\\\\\/PxA7SnlKRDaQBEXexWue89UM4sT5DOGOWJOxqs5izM5ocKjS9YYVL8\\\\\\\/2Hxf+JHFIDLhgDqXKRPAwVjgLmAqD4c1I5MvPvkzq+ZMA6I0yiL4uc\\\\\\\/WD99Lex24d6fP6vvEvH9sDlC4XliEB6UJHitnmyb1K6fdTiMb8SYGjT386ClEZA8XB518+Hj8Zp\\\\\\\/E7iD5Yyt2LYzp2LfV8ZYsrLpEtCT\\\\\\\/TScD6f6GNkv9VT4qkmceNhhorGGzT4rp6t+htcmUEvA1QbSkEJ6fa1vknsF4x5V4c6NCRWsEfdzeKk4T75BLQ1C8RnNa2z862ZhImAT5C2HSb8NlWcXIPNpP6Id12GAvYX6oC+RXp+I+Xt95Adf02AJlnhgvJsuA2KKL0hc134+3kcu433knnJI7tzeP3dCbVTdF\\\\\\\/Qcx4b8DJlaS+EhiRsh85MJtxNKUqRZh1daJFNAvPGWwnyQ8uzB1OzrUdblxHeZyKFfsHevNrt+aWNQYrek2BCS6qPXaz\\\\\\\/BCVqhEd+ZJszTRoC0s3ah6KWRSiepIp8dF8+7UQ4+IWL4SYfm8qR0T7J7+Ug9PPvJi3lxa5LxuDTl66LlxTJtZt9tmpJL\\\\\\\/W2NR6cGib49ak8U8xuHEXDeMqGAHf7ZI0WEbs6HjBmJg4n0eBVJKJV5r3WVqe1\\\\\\\/tJBkBqXhwE+240i2ENglXeLhhYNsWd\\\\\\\/tmDdExZHMcHMIFif2XIRixHxEyHGN4o24XcnHuTcakxy0h48MM3MvFnyWLA7jZ\\\\\\\/I+agQ8T0SFd4vxFDoYe3liRMq61gFt20UomOMy\\\\\\\/YSQ9yWSy4ZewvGLzWyiBwLYn5aPF1rbSEQ55Kh6EC6E6DSiB14Mnd6UMayX0ttTugWupK\\\\\\\/WVXnicMzXWyFKdZWEEbKKEv6S5u8t11fsQySZJrhJO32vGJZPxy0Iky1e6dxmE2LIMlbBWIsc+NiQ7CBqvwtDeH\\\\\\\/GckrzM0CaUVoxMQ7XsrmArcVH6QUg6VZ+Fy9vAT3FgLZ9hfeXnMw6PlX+D72l\\\\\\\/Fua4KHw+xirBxa5ebDnWlSU2nrSservKa3HUBALosB6jSvMwMc1oY2to1KwRaMQZqyxFK2VtAkpht7uQSjqkeGHR4m6mtLw8JrsTqOtF42jly56YEiwVsg0FBv0gJiRB0tRVV8SgKotP\\\\\\\/d\\\\\\\/6o1fgV3C3DjRg\\\\\\\/0P15DUXomGntenJBnBI3bcJw8v7ZfgtuC7ulTpUtMUz5U67hN3e1m1f+l8WwkqPaoxDc9ZkHeoOaKCJL3lmE4F52EgsGzWYBq50BeSdr3UgNexh5zr8WHDfOII6nVWKXzr1wZQPXuGamW+twQjQwGNOT4GdJSiqjOIDhuu4YimsTnFHf9lgBoB+7YdwTsIaRatzVqXQGnjAazszq5LlYgDSVyvXQqq9y16mrtc\\\\\\\/N9+VuacFqmFppUcin4rKwDHXz+090jTF0RG1JWWy3ETRBRqVMRD2A2oKWt827iTn8wyCZja9MfmVCa5bxQ\\\\\\\/Xi5o\\\\\\\/4dYPXlHhx7nY3esUV+Bbkwlk+GU9j6W4S26Hrjv\\\\\\\/AjnDugDizKC0AT2iVGyf7FUh32UnqKTJNm3VpAORNeNs9AHDkaR1A0gt4hxZJMmKo4o\\\\\\\/TeZZFgTTT\\\\\\\/rUTatoeF\\\\\\\/PsAvE\\\\\\\/YuEQ2uCGboPnANVIkk0ckhFxX\\\\\\\/9H7DOEWZe1d3Xttpw\\\\\\\/ua6iWlDDRNPC9dwlvJ41l3mzJ2miHpS2InpOHqbl66Cvgzug18kVglYmmzrXJm4A8ztQrBayo91MEF7b40eqDk5WbR4c4RB89l6YuBgM5XgdTXC+uXZgpLh2D8kwBUBvf4WuwOQAPLBbn3VOvOUOq\\\\\\\/e+VpWRpBNN6cmBb4znbMLlc62X0ZbbHfqq4i7U+vBMO+fW2mP4xOO8rsRHRNlCwPFn9ZftFDQszgYFhVvnWgkHINvWDbPUxc4q2ULR1gPuX0K2ePoBVYZcvQ+ExxNKYQ5bwMbZwFERBZjRkzJjGtdmfQS3dgAqRbHOuQ2pwMgerxqyRgCwuMs9IFU0GCpvdSlYpc8xzGKjdCdHDqcj5Tie3xeqcV8lLQs2K29uExVCgdl4GMvF9iogYj3GcuWqIlwHNiFI+r8eX6N6L53PecTm12n4CfFZ4tns8MODxkqExnZj97Fwzlmu6ZiaYuwooSz5V51RDnZL67xAc36HdeieJ5+Z4qsBDhbABuk1bXF+WCKjzjRy5z4\\\\\\\/D1YbERFNhhhJgvJjDkwczjp6esVjeYMLn+nV\\\\\\\/Nh4J6rxbnUMezcqNG6KEB2av1RTBwmvKAD7QbGJIHYwBk34z974pvYtTcHZfLcx8jUwTEINuaSLJaZ2VBUwNzUNOsGeXRnMef5Ug8oTmxMdl3TvSWYvZVh4bZdGjBrTwA5\\\\\\\/etvc+SdDP8R6I7oXnwEULjVPwm65WzRUn5tCjyab7SEjNLuiOabhlcuVk1gcnuMLoixmotrGhoWnLsjdlEtySc7TpM0iQr7\\\\\\\/oqLIhQHSEG8j+p4veeqkTT5pv\\\\\\\/WwuP7VIMENi81uOnEve3UUSnCl7bQRKXktoA8mUlqJBJsM5xO40vqM75S4SGHvlAmOdG6FZEKa3FRYTaUyzCCXfbfQrOjMBSQYa7MUkNpxqRFrptemF5HgUnXvwtd6n0vCNIpMcHKxnoOab7JjofV3ACmoSO6tI74LiCy4eAifqvE7rVBDM6Q331qfqxMP5ywywt+rxo0mcynbT5pkgvvN5kzG+EA1wal5Jk5dyjJKaMtfcBzvftoD4st7Zc8xgR0iPwtXfuofaIIA803HsddIX0HiZDbpsZ3NdLO4oVKOhyR\\\\\\\/PgEuMUwZZbYvef3+yQuXtkP8pneKNbU1\\\\\\\/QajdqACa63wqbQMpZovs3TN4ycSU\\\\\\\/8K0D+BHtXfDEM\\\\\\\/SBQmaddXT60rtttdTbK2nP6P07hgmozBlqpifaRWgBtfPH+xiHuydm2+v9V45CO326Jpp7rEbalRoAcedR1bai9aR35ZknD2QbxSRq22JNivGITsUm24Eq73pNIaq5SkG5vsJWdu+WKAAUOsFGkfIu5YbRpHo4ZID+wZmt0MIgE0b9MQTWnsq8HHF\\\\\\\/SI\\\\\\\/+p+QuvKDfOBF4woJrIyDM2wftd3GaQFN93n5YGU6HUP0wtGC86aT85p69BTF3fzoglZMNR3hzqkz5ULZ+a+XLHxAus6st7Q1ZRbMCn1eN9fhAk1F32LssGmcSW0nBuydGDxZCWissNsEqvPtodUp9Pt13bj8RO7+xrlpESX\\\\\\\/v7ItN7GcJL2OKOFkpMJnMpRlECFCPTrqrTNs6EXhL8uAaxB7aXRC7IU6PEt3S77hP\\\\\\\/QJFCEykMeAZxPjNwd8DIz2GI4Q+KtX3zZae+fr0APWgTAjSdbxgdCsBcpF7Cmv1rtb8IfuD3icdGWkDGQc\\\\\\\/8NZHbtJE\\\\\\\/yL0cSRDhiMEwmo53iajaHXvlQ6tqyZuIxt077c5J7HDolwe3z5AgzBmKCh6y9SGjM39Ic+5J7UJz9PBqXiYrQwm1z2+AWJG8KMoibUQxzgbX9jPde7mJiqS\\\\\\\/idI3IYifSZURufhBQF9gWkEqpL2TdLmMFQbA63bIkiujGnJxEJhxihwcSiEzkDp1OCPgKszbsZrVCV3pwX5BbBG2qaJl95DjL9S6aytClZzNqHq5tuDPNPyuEqexMwMiL8SZd75L6UBi\\\\\\\/NY3XK6gF4Tn2\\\\\\\/F7AolXpTzUHhrdnaacqC+Im7O9ABrpKVpZlHGC0Rswetag\\\\\\\/lriv3fBhW4OVcd0GkOpYAeJWhIIi0Jgnfq3ju3ddYvqY27MRt8gMHWA7O8crRhnswRQDc\\\\\\\/KIuUqQFat60MsQx4AgS73\\\\\\\/Lq5LD2TUQJTMTLQQhy1BFzL46t7HHhksLHxUJ3oHdNRhZ4RVz9T2XgqflDPRSEa2Qpqk1MYMnTL6ghGBukXoon53EKlupePlR+7QtWENQILeGvXxi+KItCrFB1GA0PL8D5g6TDKZEFJhuSbHMy+l973ALAJxU05\\\\\\\/gkN8bmIhqCqMyhy3YQDP1muDJKu9d1AMPf4\\\\\\\/\\\\\\\/9N9D1agT16mdOFlCAZi+Q5CSvZw1E5kHUyBnXfnFfRu6M6827PCIGoPj5IVwhzCz2hI7J+VzVre0FPexzElp6yNrCQK8+bbcYZyHGjwDHfxTyUbiGsroMxIoa2RoTmQDD1IIJG+rbsFuXw\\\\\\\/KpBHi6WiuBo\\\\\\\/ta2o+iKsy9lNC4Sm3b4FhZukn6vJKcyb\\\\\\\/O3sXJ+yjEEzSDybmUAyjtAkWmsqr53yKkN3o1Xv5lhVftq9BZ1vjbbz7y7Z9+K+l38FpZ\\\\\\\/xy4D7WVinv3Enu43P6Q54SZzqlm9haq0cIDOLWiFelSewCOnM\\\\\\\/Xe59mKGSOEST+OufT6MBeReS5Z2vjyvqaydB6L\\\\\\\/ZWr9Uqs8MqfV+HRX7Bhe0pV+ZZWiZDJMEEvPz35KQ8eDxQMHPAZm6W2PiLEzuaukhbBTBn5u9HprWbs77k8+uIX6wFDUk1JiS1+\\\\\\\/B+nFbnQgkqlvY7qvjMBvMOMDPBe\\\\\\\/3ynFRPPkuhipKom9xoSAwGHWstyKXoDVjoDIZbizgNfBWn1C4KX0fNNtl8vWOMzIHLxmrnNuqDV12TLHsQgCXdZLMl6uPWNyTI5YGSUvtzDKMzsHz0Y+Bvw1bbjy7j7ONIT2QxNKXFTEDNrOjwyildt5NmKY1JG3EY3WqmtOFiNbz5npSfNkuwH7UEMnEYuTNCMiaaMgnh0RQXAnx\\\\\\\/vaBE0rEwlAaSdv9inrzhFX\\\\\\\/jeF7JSJowpVmDseGiaYNIVA4FQVaSIiXA3BGivfCt\\\\\\\/GTOoUOJdZNNaSUuIQhRXL\\\\\\\/ouyOhZ5D9Zb9F1WrQ29nUJnOsPNTTZkQQHmrpnYATkdFPqjE8gn821JOtxViYWIwgHfDz4rzTHthVp9vC6btg5MFHGsl0lpb80KJReqt1yXVbSFlrdYk6bspN4lSF28fAN3tuqxfBRztICm3A1qq\\\\\\\/xS8xcaOIvuVjHxVYXHumhbBIV5P8q9X6P5Co\\\\\\\/vxz1r+wg+O0jEKs4cKmpOI6qJk7CEScSKBXN8nX3OCdu\\\\\\\/fjIJB\\\\\\\/LkWhvlk\\\\\\\/1e\\\\\\\/tY+sDdth+5QUewYUCNzUj6\\\\\\\/zx0Bln5RkGmx0dLS0my71bN2Zvm1nWdbIr4X4SrfPhwZST\\\\\\\/dYl7xEqVuqdwTggxriF\\\\\\\/fBZ+l1ZqmW1Tma9mnbg\\\\\\\/au8HsieOBLT4majz6WVy+lsHT7YkjWUTVaox83CjIxkgnCKl7lo2ZVbEOxZ5RjKMI+fKyiueFyFrXwnOXDROJhvpuZvUFEH3ZaNKbOOGyuQnARD+qy72eTsF2g1+X5Ys2AkujcFV5AGEl2I39X6QjMJjLj3YW+FcTTPd4AC4p+sl8ZDzenGCvgfk4yr3gqIlOhRmpgKJWld8XAANccv1hNSniNtXnDESp7hQREbl5CldIJ9wRmPp\\\\\\\/xilYHPy0HtkBbYKwt0dZ\\\\\\\/ZwqiLxqRvnZ4cvbBtk7VWkPkHwKnxhOVhSW1\\\\\\\/CH440PAlcEzUUWMP16b325exKJTFgfnZKrM2MFFkROd6\\\\\\\/QB4eskfGqfQNf1FZhzaBqa\\\\\\\/ysYVlEKbSvjJBt0ZY3Tr4vOX7a\\\\\\\/BdECmMRCohPUKUy14htq1k4vSxUR7M8JHJslBG7LBT9GY\\\\\\\/9v\\\\\\\/OHNyt2b8AAzn4GK4BX9D+srbxhyphftYy3TMAT\\\\\\\/dqsgXcHO7tIKlP6aeETOWJLytdP\\\\\\\/OqCvEu9Fs9uKTZK+1jSeawQ\\\\\\\/mYxgLAekrJBiTfy2HxgJV+olsWFFCmKkt+80uQmGUYBCRoSODhZu7Qayh7zmHp7swb0UGKZkSLKwMOknxJEyoS1ExgiGX3fn4w4tfsvynk4RxmQJVOerFaI+HTgc1mdXzY67zo1GDndFQe7spt4yOsOZsS+EQUacdXpklmPdg42lD1o9p9HnHkV7mFo+dw\\\\\\\/8wUF8mm2oiBVaIhoK+wVuOLM9\\\\\\\/Hlh\\\\\\\/7T01yYUfU7hdbppkt1P9sRKN2SX6wKABe+jzvwX8PML2kWiEtdQHGkUg\\\\\\\/Cq1r51lsyiF0b84Jpz1Ef\\\\\\\/fYSD4sZWhn9qVm231osFyL4H66wyi+hH1ZkJsF5z\\\\\\\/tGKQFZTfAuO51a8k1NTXwCq+UPlVtanUwWWsWaK7+yoQ\\\\\\\/8JfNJF29EV0OwoWEgodm6PmzNVjZSV\\\\\\\/hg4VVVHHKsBriM3c3wKTmc1pP6YWzdtU4NTR+DxzL5WNdfZaUWDW1K7jusYBJiNnRFPY9U+2JEa8fysaV0gPbH9gt0oNTBgj4lwJNn61KBiLl4m7pUwlAtnnDnHwBLn4cpdq0ZhrcM38sGlkITAqbTj5\\\\\\\/vCKN56f8EVsy6I556A2hTI6Vgo8AZMHB+w+pdRLF1FcbAOFkW\\\\\\\/rZXK7vuRI3C0SslFa1HUjQ+LxgFXXm\\\\\\\/rQz7r6wbwqeo5T8yFigmuhYq+h+glD\\\\\\\/OIdLjChsAaCpdMNC44VRjwGDcOzQbxrMy0cdNvAPHVAtXGXgP48sSzLEccSRNeek6xBAGF+tSEq83P+xxe5Jh8efQOyx2lA58aR9aKa49WsybCtYJtxS2mrpYo9QHYze2X0u9TLI4XGKwPN4+Q0TEKx8U6Ba8kP5llZ6gGouHKP8JX+Uwua3W+4mywvfbbxctfBDkEA2AXBN7gVHepARXibbsmFl0MU57cxuF0Zrs78PDqGR8ZIlMw7pPH\\\\\\\/kjZNisOQaLNe3+cOl9\\\\\\\/zxSPZLo6dEkjErzSFEWQ5gQf7em\\\\\\\/mbZFSbxwNAupFIUHBOHF6qt6mDXVjLB2KAjupN2VSma\\\\\\\/Z+cfWJc+A\\\\\\\/lfuqSGuxCZZcl9Y4SB2bPWeWlF6gtuzv1q2WAtTFLV9YOqQwUwYBYbMNHGxDqJ1w5w3E6fuu\\\\\\\/l+i1jgdIWXl3yTU9r9r98HzrYHP8iX\\\\\\\/H9MV\\\\\\\/JeHNryaR70uGNfiYYtQ6PaBmzzpH\\\\\\\/0+fC1nuZkp2Ry0lyS7Ns83WUKNZrebP36WqkOAgP+ZHVWm3DySMttUx1g8t41KWPo5EyPv3BoqtxbZ7uXNGTU\\\\\\\/Uu40S8AN6czpWZqXZ5bluDeeQp3Nx2v6BdKIQnOH0GBkfF3KPNIs7GMVICzNru97wJ76OcAPj\\\\\\\/YG27oEdykg9hdyLTfoz7B\\\\\\\/JuREo4Tx464QP3XTuyMWX+ERqM8LawtPuLCXM0M+O+LfFWLCxHiYRXlWcV6kHh+sXTMxwOtTggBJwxde1z8wfZKF4wAbYTThQBKQ8bd67YcHnW5mK6VOf3WWsp72xKnIpSHzzy8vkp65Cn1M8krePgyNx6zRXusW9hJVLfNfveRtlz97SuLv0ypM0rnbbE5WXjRUgx5Zw2H44xBO9fVtlMjxd1xbfobdOtORh50w\\\\\\\/puuR5OF4J4aMWBoeMrmfvGZWojOZx4B4GNibwkf0qx90F2l7v50MkPiDRV+dNhoPL8vrYyjSsMymck6NkahOVckOSNs3DOEV2EPekxlcuVxW\\\\\\\/kahyqu9pnX1qx+SK4kZqPynJCv6z8x7QidhqrmBn9uzT84pmwqH9FBILf6tbcTI2RbTj2pQ++ihJOxO4TRhao3Mq0mGUG9AC+lP4Af7JLZHB36hU8zznJaeGv4a5GhL\\\\\\\/XIKGIOwMIxn8dALyJdN1K9yWmIjzlTBoU1tsnjcLYzUdIQW0qKb01Y\\\\\\\/Kzfq+8hEsLzjiQhRz2eT96HggJWsJO8zm1L\\\\\\\/fItRfUpc0tAnodCwzz2jRHy3FA+1dxak8kFlU94g5rmHh0\\\\\\\/qedDBZxzO9SBRFyyEwz9pytCyd4ER\\\\\\\/HVXTiBNpvkgir20UXK3EbqXIZBjfEag+lRTDDCq18zQPwWsuemevnHM7xhI95S3E4168CYnF322k9iobuIt4exEgvDnJA8aCkaMVm3gsRxmNvt7+\\\\\\\/K2LN3lULQIlfv1+terGjKC57zsooavkAmNCEArGrooNW641YbG8ZDJb+txBmJifst0XVdJiQEb2aQ7TGlJEm4ST96dRijUjWh8+guhT+VnnztaCDe7eMcfesEvtFyP25Z9T3tBjn25lIHa6o2W0Lpj4nhxVdo\\\\\\\/lowHh94ahCukd7Do1Z\\\\\\\/Sh9UE\\\\\\\/hpbwL0d8EIy3Qv7XaDzkleM1VVTmHg4wUQVfWqgYYJQwYTv2bS1OJLUdurFUvuDqmAq3Kiu5N8v7b8tSlQryLyNiUdnlwtExccp41GXRu8HruYParOqPU9J6zShVEN92z98jzLzYECsKmKbgfk4J1YNWSPyTWfyZrfDM1IphTPxs2Upgeh3vem1BNXldnc94UituWKDuf+nmA9sNBkACGB0kQCctT5meYQMJ4xzgeUKwOxiMxRW5NbHnxEqQMHgVccv3J75nppn4DqBc+WuHD\\\\\\\/lmzjVnuAb9vblsDDmIo+SjiNu9hsUr0FWE7iQ7kiensQvmc61ywPH\\\\\\\/UKULr7PFMGzy+1xKmwnCEzHqrZOIZL\\\\\\\/cFEK6t2CY7LO3k19XhtWWDW5Wu\\\\\\\/r+cyY4ZssTzrE9qWbaP8AXcNs591oi4+TMuRHi7Yl29UkjwUHkm7wSm0YfF9Eub86KkKHSVs+1wEunxYbdJXDoauA1jxX1AhyTRZ8vi67XyY7df2EtipHrOoCCP75cVe\\\\\\\/qx74r6fQ6lB28I5WXAH4xzuDQja+LUKWHvKGUlk6dscM+skz5t4mG5DX\\\\\\\/j6uhuBt+Gr9jMLePpaeaVqq09utDlyTD\\\\\\\/3e4\\\\\\\/ecoHcTxda6j4V\\\\\\\/ZCudU9yMjeEKC9kdAJUbq63d8PFOYRUfarpr1B2YZItTx7u7qWFvqQPaWysdrYnSwxZB6cgOzmsAsdy9TKVPNd2LuuQiTZcTM+4OqBknVPA37zpqeTwrwlWRF0Xjy8gyARpKH2QOaq74udxyTODgLjft3MOuSKqIL7uYcR9WE78uNRjmMqm4VFPdOJM1qJX4I\\\\\\\/YHQpzGQBjtTpGaEDA5hu6+hnndHEysdf2O4n3UnJ+xcJWEpkgnTLHNehyvZPqRvY61zj\\\\\\\/Yuw\\\\\\\/uoluDQPTfk\\\\\\\/uSCxWAeHg31YjohjncDuBFGt4U0yxlXNy21RlxRmjkb36bFEosscdT3MMF\\\\\\\/vc3rMgmAPwbVpISkiChPP+SO3GfTgpua6lmArYuRuVpC1J9JAF9fcxu\\\\\\\/RfB6h33JiPdjpOM8AwA0QBL8OeQlcwUVuxyQUqbfRAkceVgiG6zw8ZAvPMnez6I2M1fRDrLRdd0qZeTe1BlL+0d8uKS+AYMIYO4fs5zNGYFiOqpn7KfJFkyCYVXdXf\\\\\\\/sFp1ozJ7HJKUzsaRDLq2XY21qNXtOq5vC6h5iFSzM3JZeQLtyUqkTtOvRwJSxlJieByOs1mlPSw0BwL04h7e1Ymak2sX6P4NqpZSnNXht9+VPAo9tF4Ld45tu4LtyX7PfDwzzuyawjUk\\\\\\\/ey\\\\\\\/Q8U4bAJWUWdC1pgVOYh940YsG9hhDiuGedhBOTvxG2hdEXZFFErw252Lp3pGxcUl7WkstEkIGXD50DoPtXWkp463wsX7te\\\\\\\/HtYWb4BVyhGUc851nbvdA9fOixbossjAwGWMAsW5GdWZBNRLDDaU\\\\\\\/4XKp1ewR2uzOvfPKBWZBxO4xfE7bq\\\\\\\/SeVW7kXGWeQT7q+rS6EDj4cb0InL7B3OAw5PNejx+fvSe0RC4QvsnzrIpMVAbqKN0wBNvXPO3Ky8wUSq6r3\\\\\\\/awkXvwfNszN+tgElRqpvJP7SbazSyTPGQokqHnRvkFEDaE3O8VA+hr76CTqahIqEiba8WVfHkDW4ZryQ77OAkZFQNFCwKXcUqy2wt0LxrC+XSP\\\\\\\/myn7Xs0FYIVqwvenJOuqj4Ud1OHKWmjyWbLbfAlr8HwcC1zMj4PhtW7M+bI+2CYaVPhHZvvTryGTR+wgZ336aeyAlgDKXM0tiHJGZWsuNdbAuV+jmyHKqprx5MRsPPYRYqSLHu+qC8EILVccpvhrGW3W0TgiQgN9BtEPvy8pR6FTK+YyII\\\\\\\/eT9YWAvn9vtR4vPGQn\\\\\\\/NOXcm8mRUTr4gFknnRRpLEbuzpPuLiwJrQ3qwfGRmdom54kl5mHzEd9nXb9CdLorVC2WPgP1jxlE7CeiV3L35dLYqgoFP\\\\\\\/9EEK+yJiu0lPEHEiCJbVLIZQJQ7RK5wnXSwEPA3ulrQ6USCqU3lwOLKfrwunoMj+kGhHPzozjfbyOwg743m8NnUrgTnO+4+9o0VLY9ZCJxjNR6MtkLyhHP6Qa6EwA3Kf+q1Klj1iZbVNapQItEMlyg6pik4eOxdtVHfelMgBS4PrrlJp9GZcHVWBKQ2kOBXKhhagsFcl+Jxe0OYyOm1u2OL5oW0eUyAtlfCEh7q+n3CVkvs6lSy1Q02YBNTGKyICFPhAbf+p+PFr\\\\\\\/S8I8uatvAKqIDrN5y\\\\\\\/CGx6pynpF7OyCO6CXWBb47wM7Sl1yaaqN\\\\\\\/qw1U1VXFvFQ+hc6XWExoN2F8OpKmYuVHVMuzVfrLjdPo5j4QNgjjPcOP8EY11+4AJCkwov4FCTGo2PskttL6hiD0n9ljyoyny7pvV4XgqjE4y\\\\\\\/B2ZsWpMhXACaWV77Ojb6lz7DoarCXh4qcDfPq\\\\\\\/CT7dQhIbEh2BtIXB5b9JKHQQLjhicUL6YOpoUGTyzr98zbYa9Ty3eYGLKZm7pWpnCtRia9FpZ6mW1wmtI7EBiIrJeyJI3JhUOPn00up7aOGR\\\\\\\/49ONKna+MDeiQU3qsYbTCACmSGj9yixQ9aN7Vb8bPH2jdxPyrlg4ek3i1T+CoEONZcr0SKwT7GmD6aKCjVmsS40ZjY8pK3a3YUWUoejL+JubyKptr4\\\\\\\/\\\\\\\/Q2qD5mmI8bPQzMtL\\\\\\\/iXYfZQljdmRuzuIVnMLqCx24TXEKakZxFA79LXQHXSDVhhSdM8Jxcdqsya4nqLrIlYJf2Z3NhwkMJmbAZWoY3kiuI65hwfoAGPc8AtlPQIF29023aip4k\\\\\\\/35WRLEiXODHmxgGQQtEOIdWYDXFJ\\\\\\\/1zEfu5bqOv5B0eAD\\\\\\\/srhlvHNjEW+w0hHsalqa5xtv5\\\\\\\/wK8e2LxmCgNAYWNd0OD4PRkEMt4ShXSAJ2TUcjYXyNnxpVivP4A0VHE8dhvyuG7dQnvdm\\\\\\\/JseH2cbhc6pjab8LYz9w9i5yeiYTaVPgQ+DvvvnvCvB9+esjAkFRdHTI\\\\\\\/rM\\\\\\\/wu\\\\\\\/JfwUtEJhGTwB+wMYU\\\\\\\/TXeKtVtv+dIOeGOzDgWKe0ACxWArkBjq6Nxn4DEwy7vi55878VJiKtFi+nLiEVGFadcCp+Xsix4FI7rPDNOzPE+N9DqlpBaVgmPQlZIVhfZqc+gxfq\\\\\\\/07ktBppTTbkXqpL0kLYL8GrmLpdhToJ9K1a05F+r9+Uszc9+MQE0lukp\\\\\\\/uIxeU05LAS7KSko+CSFnY3bRIYzL6q0KEbwqDVb7EqDYV5XvwNtEWW6QbloetFpXzq5k+2Jg\\\\\\\/vXKav+LI1+IKN2Np+ductQhAYVU5GMYBGsBnnD0LEmEIm4CtDe0RbgPcz+2tLR4aZc6kplyLk1T3JJxAY+6bc4yteGBC4gbXPP+xBdInb\\\\\\\/lWnlkcszxoKi4mSu2ypywcJ4P04\\\\\\\/WWEBXLkyS6iTjp7+\\\\\\\/YXNHjKWMyDKt8ISbZt4R3sblel9\\\\\\\/JFk7Sjjuz\\\\\\\/9Pjro8Q9i1t6eWylagXCfKXqCLAwCpfLGvX6xKsuTUKTy7hajZswzQ6J8vLULKRRsmdeM6I9Bsaiy4mvgBBHzK15rS4HBCyPKj22Z1FQysxWTpHHf3njqW5EOHFqU51WTzmrM1k\\\\\\\/yCaV04k+9US2AGMTYkPWgkPCNU2oNSqvSqKugNZZpPxb4CIIyF9yOmIZL78x8VgIZvvA8OVzmS3Hm6JQZnQdZHIKSKXx8QpakwAJwLoI5XPkFNhLKX0+1e3soS1AYAdhaXgB5ZFlf5yU0MwjdZIwZPTcxnQF0es1lZgYc8sCOes+CFBzjPd1mo0netoZjy0d3j1qCYlokq5HTE8AJDx4gO4nDfpF4GROIJzOdYMmbmUZd\\\\\\\/60LjnQzG9QyQie8kDId0hrKmzc\\\\\\\/Aw5n\\\\\\\/mi8uWeaxGdd\\\\\\\/eH5w1NrkqdNjaDa34iqsnb9ibGErai7AYNm6\\\\\\\/aqg24XoegtuNwDPEF7jR4ZU1XQQDF\\\\\\\/UanjAAjBMcw\\\\\\\/h3AeCeOaKMvsilQ3QRRFTJHla\\\\\\\/xeuWYb+6Bq7gxytxrCVGr1\\\\\\\/luQaB+ryusOlyO93drKwud3k6W6Pepdagms0ib66sXUG87NqGIsagIlT3\\\\\\\/0pV2MQN9CCzY1X0cZ5EIpWmIOqY1BVLflztg+3zoafm6rlTWtbWqbjtSYzH8pQDEcGDjLQ1ceyAeTAKWo1Y7+6D9UOR03a8CwAQkQ9CQjPU4DEvGZ4Q+qAhDtf0CvDDhB7aSeXQvslP74g0eGzSAC5un58GATtRCoJLqGMgavoMvesiRueSSfoX32wGgejf\\\\\\\/5WO5HufJhcGPH6GlLF\\\\\\\/fXKkR1ZljFhz6pNWg5VEzcbAq+FtmnU+caB8dzM\\\\\\\/91YYwrmCvBS48dI97gjolHvmzTD7r82BEkCsYwDfBaSnK+UY8rfHA1XzyKDjzXxdHdMDifI2mnT7XVZb\\\\\\\/4H6RLiNV5yxGfZr5C11PfpS7v0+VtS6VyKeIlw4\\\\\\\/jdGvgKGyFf9y+nYN+yKEsSEt4\\\\\\\/KBNZfnTyhL8cH\\\\\\\/di7NzPvhINQNoezzdqr8eBVBlWOQSfxl5NI+ev8RCEdXufT2qrWJjm9Mmhw6f3itp\\\\\\\/+f0jDIfF5+ui5G2UmWRimO\\\\\\\/fBw6+uaoIdMqMDb5uNYiEK9xsC+RvHTwE\\\\\\\/IKKKiqc4Jizf3cxRKLR2WQy8Za618tra09rJwpo36zJXdFtTtpAWZOFFQJxm0tJlrDnuT9KJUngrpnLE6V88rImt\\\\\\\/i5d7I3xQIilYaH0AxZMjsnASRa1xLJcPhEKXM\\\\\\\/c2SNo309XCnaF1\\\\\\\/Atxba71pMqAxjWybpssl70EMlEdL6p2Jl9rai1N39DJF50Ci6WZtNyKjZ+p9Y3+nRixthVAZKsTX\\\\\\\/So9gZIzBqKpR3cBzLXZZGT15MfxNF7F354rBN\\\\\\\/TcZuCPZX6ib+01gki0Qmirsh44z3gDuGB2rLNoAPInLjLVEuIjpMumXU0NFMavrYUIs5OION6Cb3xIOPE2JhJRiWezNhHFCxCOiIyvzzo8g+1ruRC4ETXPlLty0GfVB0MJJYQNhq2CXxvtNM7TKQX9YKJqjB86qhiKqENDVRLpBpbm7pmt4E\\\\\\\/WF1wSug2+UX\\\\\\\/nTxC2TIcOtIUAhGkIzUW934c28snEt0+mgZjF+QpeJMt0\\\\\\\/2XH6IrsvhOtJIE7eFGYOff85zq8rhREHr95iWaP+5fL8RKZh07gveyC0fQt0R6e\\\\\\\/v1FX8mJUNHUCZGvmpB4RDy7d9hUO8rq2Tsv0TId0qR5hBPmZeyoIaWeGtsUaTntryLF0flMXu\\\\\\\/4EsRXu\\\\\\\/Z+hx1kHqgSkw8wE6D8LqauH0Evu1vr0Zy3ikAvCF+\\\\\\\/UgppTVAR+WMuaJkZHnhsK+ig9bIpldbnMm79+gc4XeNvDEnyrY\\\\\\\/90qOPZ9dfGlb5gM4mVl9SsvN2YittBkO8wzy8BJG9RwZXaWQAFq\\\\\\\/BkreKf3bA5u\\\\\\\/WBSWcxYepO80ueEXgnucfzvjVyWtG7iTuMKW4VJnM+h0PMgARc32besAkrH1vYCFTIWbp8sVCQgE36DGEZjR06v1EI0qTNxmCbQjSfWXr3J8ojK0zrAaUTyx8d9HzOB4EZj7eHxqmnIE5I2zbdRaDDw7uHSFyhY8ULTv63SoyvUS88yll1a\\\\\\\/3yzz2FcAi9CubevoZuapewIkvBFRY6\\\\\\\/ci2TQuWlk3IjLBOEO4+2zJTIAbg7uI3AyeUnwf616P2CKWCHKPoKLgPQXK26yH9+7OlBP8je5bZZKrvC7dnqVk6VJxoYtmeQVv1MGKIwS0dixWS8\\\\\\\/Wv1hWHcgpVzE5f5sEks7zOO\\\\\\\/bhXwky4sAVBeAnDz4CMLKLyzMK9LwHVtaePsPp1BSHa0FTWQ61dMrrjIrKzYNN7CyS5s+P8zXwStumf5HKwV1WsrYTBkC+ofZvQZlckYU8uFyovFglL1fnQNSj7z8oQxnBtJj1G3tZWRCjIEXmaeUhSV+W3f0QeG258wYXmjAR3KQrTKkyUpI0ok3sHwvpXzFvmlgnRh4HOzyTmLmDIXVDXGCXITlKnfcJZZmirg2AFHa3A1KUwRWV04q3iFa1Kp1F2lbYm6ftOmVgR\\\\\\\/MzbQP98LKu3gO6YCiOQzwnnSrQh2RZttJY3Om3sLT4gZR8HBBF9iWhHXhYHrDMBYQimN8x++dBOBn+eO+WjPgIGvDP6wuTsQIvYDbQ4OlyLfmH0AXLXAmBBp4C6MbIteMY5IZ3tDx+B2zTbxsftl0wgGfBJKNM1eVhkY6nz75FWRDKLUV41q6PUlwi+mr\\\\\\\/LWLdU+X31ofNX8jY3\\\\\\\/zkVOcKRjCcQ607OJpfoguIYuxjbr36nsp9yE13DUSNa20Qjao5XtebZCAVKJIaS0POOUMSCbI5wwuw7mYup37TugCQXh4J3r7NYMHeeWR2rZYfkmq4hpXRPieOMGtW\\\\\\\/K3HSSAoMWkseATet3zzGfHl80\\\\\\\/9OHvWTixo0jear7\\\\\\\/pZ3HrhADf5\\\\\\\/awh1CtZKI\\\\\\\/Q253wr+aNwgrAhVAqmgjWseps2gYO91izY\\\\\\\/UJ\\\\\\\/15Nmb2+6uO0iAdvms\\\\\\\/M\\\\\\\/WAGiLh8IlHN+92NEbu0rgjXkxraGjIgC6vADOeVHMTaU2RQjK96a4RafMesa9YvB1Ifl89Rbq7LJgevYBe0J53U+mAQvTIJ3iHEM848iMvEjriEwfDwrcATMbfbhWNxfLNMGARjH\\\\\\\/nCEd9znh8KzD4AXTag720L9gGOZlawlDGZ988FiJP4g52jq3EqElUZIkmo8TuzR0\\\\\\\/rewUzXrbPAqc9UZ29hT54gbj1i2t7Yx0qHc\\\\\\\/IByfJbyfxrCSwDSrwLx3ItXIkCCd0i\\\\\\\/ADU5dj5pdmt7NO7uyWT4DPPAQ7DcSQxiAx+vnw8kCGtPqjfW0OrFKxIUb306LjkcDB75NhBXWPy7VvRn\\\\\\\/10jy7CuFlSIUOa+rAzXSYyk2U9iTMy92S6P4I\\\\\\\/GUM5H3Lgf+V3+G4lIU9xMcnDWhzcbWPRVWnT5a\\\\\\\/gIywrIV2DnOLJaHK\\\\\\\/TaOBOTbhuQ\\\\\\\/gkh7GfHvjqGaZZ7cwSLG9Q44j\\\\\\\/E0vFXI2qt+DrYu0uRBueLqQm2kqBmAMf4EI6G3UDKtw3B372svZk+SfQdGdJ6qQj19Tkz1miortcb9mqKBptMGxAjChRNn3FhNwKf8tcXrx1e0CMPBOZEVaxGm7p\\\\\\\/kID5kwnD6iVBT+DPuqyMdtj07e0\\\\\\\/vD4rTdvLqQEcefdMF86kO2Rf0sDHi6tiBm9iQZZinX8CUPfLnPPU7fWA8AbHtDqJUzdZNwKsEj\\\\\\\/X8Y+nTKWg4uxW\\\\\\\/KxX1e0bwq8WJC64GGNbrxM2v8My05UJ8YSoqWmKPBSzVnLZsjmHO9UAcYCI3RkJoIdU7e37XJ7jROhmHJOsZKM+b2A3Rlb9vUciqWyGC7hg4szn\\\\\\\/DDwYahxdEwciBs9owRdYotsuBEk6MNi\\\\\\\/CKKbn49B6sW99HZQegQzXN0cUgiAFKM2EXQbLNEUVhOITN414PnxwmJWUVG24GqvwrJXSEVOJ3MQmIJPU97T1RYqX1PFFKBD3SgyhddVQBiJcS2HShmSdvsnsdf364wcRPoCd4AGRxHhF7NE5ZSoe1MVCT7\\\\\\\/kgAP4hNFAF6YbU9fIPU7RvyyMcgi4fTPb82e08trAm2Gu28VdUhinhGWVG21D1MNBLZIQZVlKjm1gjRe1gas2KzagQ1BB7YMoz0Ie8+7l3pdrWFpPtUqOV2vWVMWBbTaILMgwkUFp64Wb13g9fZ1s3XYY8dvpGez8tI95seduShiSJyrGC+Mz7a8UPIJylbyF5Xs9RWdZdBqBkFg8tH03KkYGLblRNpzpjDcYGWJb7PGUgy7wNqYMs\\\\\\\/Je8fzVzU\\\\\\\/d+plD4EeM2b9z3aSDBvwCipRvUkjwUD95VyHpgrrsUgiWXJtVgO0Kta9d3sl7Bc0anFlHF6gv1lUCINJYE4V++jR3tt1q7GFIYjIuUIlMcVPLqEouRtHGKxpeJtU\\\\\\\/3UYUeZlfNP127SMFuj2RUf5NTUMnqqcJ63s4wP3E0egG3HQOs2VyAW4emRtPqnrvmEqmwCeAQIq02ZfEfrFf9OVjHpSY79g9WHCfIM0jAAcyzeP5W6TE9jgS4pHozZ2KV6XcUT2xTn5Rq9vZ6eEIG2NA9\\\\\\\/u5ZR1JrayBi4Owd7\\\\\\\/cCL8LwGWquF9v1RR2mvhe\\\\\\\/DtxTOl6VomQlz3FvK4SV54YtF09Zt8nWO6+odX6OY7QGi6dv4qpBmmmLJOKRgWn3ig1l0J4PYTc2xhOiS66RcW7t6htv\\\\\\\/hpRVNc5A9GYb5\\\\\\\/+FS\\\\\\\/LaX6sYywCbkiFZsLWSeU9vb\\\\\\\/v2uWXjXR6N6o30MRYFIUbMY4Do+3fU0ke52N9oy\\\\\\\/mB1n3GdF6+9TbrSFEjpjs3\\\\\\\/pyGMd0H\\\\\\\/3Utaq0mqxFNvGr+q03njESlVh051Zrl9yoFA15ANTrI\\\\\\\/1pVMa4P8uyKwKRLLpHbqMzMYwddBdJSFhlJkqAQOvkSONTJHUf+1NVWIS1iZ0TzPsI01BUtHtzj7zZ2ow68NwIoMFWletu3vYPao3IjUsWpy0HT6mqy8CCpA6n3uZwOIXfqJ46w+NcRDDE16BuqU9Sf12R5LxrrNrMRxg93b4hlnUzzmOizb7feWR4mayqs3tOD5NspXQo+SDS+ptCTILNVUv1kk8NANwRJXz6n5yodvGVgbDsGYThzYwRwMG5Eit0JWs2XUZpPavz\\\\\\\/9tzo7YffBNCa6CoVV4CVGGoZBDXZOHDyh4uaHds4+r5Z1uryz4HXa0n18iGfWcrVoZFSVvHCGDb3fn8Asws8V3mQPC7QM4x1ZGuSEgmaY7qslRJY\\\\\\\/Sl8jpEQJ61YfDvcouvu0xembP34wy8ZxbLkyhLAG5bSr2vuYaJe5YpC3WbSJZyyEBuvhRUblV42s58hkwyso6\\\\\\\/088\\\\\\\/MVzsAKvns6ENhaNDwcTGixpVDMUHX5d2QIoxWBuaqmqiA8WvxWSylyqdjWlBHB91iBHo1GcwWV4l2XLDTp7XI\\\\\\\/phx1yK5iC1etChH+bRlQspeG16q6RElF47QtlPYyGmbcMUDVpnSd87YRDCamQCEP6u9MViURck0vWYtKssW8hlF0Jb4zkDIGq+3vzHZwnYVs7448el3ah1yDx1ilxPfGe9KdhBDdAhJHk+eJdD9iABF+hYozL9cjZZU9QCLMxAL4wOWn68wE1bjQ+QY5TGxIMsF4uhItEnM5RPftp0EtVEZxZ1VSXbdmqxAQrTzuEUQ8c9+gT2ckNiKvdiPL4z5wG5V1E29XXW2+Ei5\\\\\\\/qk+GWSBR9VA0qtHeLTwuN4FnvU20zo0ARPw+Um45R25TnkYGG1rDB1STTaqGtFkFd0fAa9hAYX+R8lqBplLyrDXhhbdsArpFav7NSQRtmLROIVMGUrfBQK9Xmkdau8aUQSDYxj6PBwPBSWyASksDKF8zMGsTaeDDLuLIrWtq54YxixBrucagnm1HUxQ6slf2jLW6zMgZ+adbT7E72\\\\\\\/NXMDQzHVWdiYJ6a0JymdQ2prlg9IBBRjhKHMQ5fPlZcsaP7gf0oABSgDjYoeAg7bIV1vX\\\\\\\/O8P0J7LvrYzH423ymhkHU3VuDzo7csopGqa+rNCYmr4d2i+xr5e+\\\\\\\/8\\\\\\\/p4zSjURfTdxqxAwGq4jyeKwzeJZr2gJ7wVbIxUyx8lzDN2BJJKaP7kCdHjAAG7696wb0uhGaKZlZiPFt+y9myydeXBUmeW9oL57JMOgDlr2RS9LibW0VtaostRiUeJhXXNuRFmwuA3lHxU40L+NZHyW5nVcVikLazzmshRM\\\\\\\/BbTAFUiGOElDW7NFIZ3kXsQo\\\\\\\/JVltTWyaAiOQOqQJAFUfV0XGJN86y9dZaxFptHvO6WpdbmHp4qt+m+jNqYV\\\\\\\/QEBZpWo22sKvp5g9eAwyhruB016X1YoYGTIHLhCzEFWuqyeBc53Q0DVQx7QWGI2wD3a4sucTdIz1qk\\\\\\\/10G6xJRz\\\\\\\/5ftRcivpazMkO2w3WyvATYE7DTxNwm7UDLuO+OabRA4tCb8HeKFXFWlrj+YwDnCzXxXo7vw8g1jpJetl4tKNrM8xxcuM7NbNCfD3TTFzbCvA\\\\\\\/ff7y65vWolG21YS9kEArqi6lT2fuhBTgfKl6Qh4rsx65Bb7aOKS5ffMBcRzdi4DXVrXlNuOsfXlhbm1ZGGUIf9zVKCrhW29oeBKGj9jrWFyjYCU+XYPMn0OJ9vARYHmE+8qneHCDVnsQBnDydLjIOCYpphBaXrxalAQ1K7Xo2+LDU748W3SHU3K4kuXQBybOT+83kJMPAB5WNfBWbptYawd+IPNyCASmJoBM1luJuDAzhV6Ys\\\\\\\/m4lhLLa4WoaNPlONF9myRHaqltE+MNleM+SwCrcgoPehLxnJ3dy5z5nJSVSKRLGz76OBgoy6wy0LT8JfhjpxcepVgjNkqRgiaUHxZvmcT25MT7\\\\\\\/dIQfgmbIJaM4G5s1s4zc9+UVvcsF1Rg8DGuiw5gr2U\\\\\\\/GYLw2XYN6AleKPnI5fDuAXPWNBoPayF+6RA9cSJZBRsz6IrVCYnEMqnmYbDd71L0q+wsB4FpXiGq\\\\\\\/6tFvipqD7jtsiA+APcI9Z16DQ492UF4kKzBCf3H516to6FJbonJzhy8Zw7DjQln3EZQ8JSr27KuerX6XLl534Ru\\\\\\\/IWz8Xkc+ukWXXNW0fDa2yyhzpCGlCZxXT04RrmdTjZHARFDInrjtqFS9bjuYYVKn286V6wS8\\\\\\\/1WazpO7r64IQe6KSst24e0uGP313vqo3Ysprm2Y8cSPOgtheDPbkLgi994EIljO\\\\\\\/J\\\\\\\/ucnXSKGAXLj7doGBJWxxmxn3\\\\\\\/cuM+vY33wBiNF\\\\\\\/Jb4xf2aIC9RzKMNMFI\\\\\\\/nw6ajatNmFjlgQgWmL943gI324dKgwF464xvXVJFuNReUresJmTfiKa0154ECZE7e0cPEflUU4hAHQDcJtwN7cE\\\\\\\/mk+Aca4Ipnk9DQIu3QdaFuqWIJaF3frH9Eg0yZ49xIlIPe\\\\\\\/ribC8t9Z9t7nUH9WOKeQwkJRjMzUpdydQ0kI7S2C3Vx3A+BL48fowOQfEzCM+ju6NDc0cHXvXFZlmrewE6yn5Ea\\\\\\\/+O8PtFBoctZkIexxplTu1Nio7AbZ\\\\\\\/XVsgx8oClnS\\\\\\\/FgaYl0ocNs2a0p0KsanoEWrEqGA+OtOtfX0GM7jQ6\\\\\\\/n\\\\\\\/Ajldzx5MKwGzCF7uzMpSxPMmVJQRYBYYe3MyGzqMHBgUB8LhxJjqT\\\\\\\/dMwndvCMNyY131y71kBNeQOQBZkkqZ2MErCqHO8VpdDILhCXIbAp2bfs8vHRXOhXr88vtIeP9rr7S8gzitlxf8JplCn6+wqQBKLcKEs55wmCZhUVpPeMGgu5MdjWW0tN77rLaZZhmO\\\\\\\/7I41m0Tw0wfEuXcCtx7lMG3X86gygKiNIpDq36rYVRipLUJqLAZxW+G1bzrydN2arttZBO0HWtiMFkkqnbWuzCne1n0ajny7lMjCq0m++Qjm7\\\\\\\/T5FV4Ou0T55MmmxeIbUh9yX+1IkZ0aYCYtdLi+Rz7HIr1n28iX98xIrTSTAMmzU7sjWR6uG\\\\\\\/Ci9eT\\\\\\\/+leLuDwXDeNgh8UD1BgvYb7vVxxbQCSvRiBzD7oJ+ow9uUf4srFHuAFoKougwzV24UCWS3fNLHXnsrBUJRl0ca6g40CbpYioj5lJ\\\\\\\/BgLP9PlRBoHiL5jyJ\\\\\\\/wbfvfOyCSiGCregWPXI1uI2gzw7yQ+21m7AJI+9ZHAXNwYeF51fmzIMTbMQfdY8Mp\\\\\\\/yDOJKJb3u9Te7wicaSxoBXtcSQrh5O3W7moGEwj2IDsIj21NaChzoloMdL5ce9c05ojvoOj8oSvjrq\\\\\\\/Ty6FvzjQpfg+89XlNVIWzpBus76IARcAVjj7iO3LbIONbTvFix7G1ipJlgn72LlQFnKd9+fQeQ\\\\\\\/SxsIvJ4QP9gKC63VidLgV4+CrLwAg\\\\\\\/zb+MTitcgWOIMAg336i9bFiDcBTq0zMCNEh83Mhe44WHtNCEcbxwvralqw\\\\\\\/XvOeuFCk7Vmyg9Xcxpwsaxp\\\\\\\/om8r2yRcS7EEOuxOPVKUiO\\\\\\\/4ae2FQax2\\\\\\\/VVwyiK8xw29Qy6rSo2V9EM8ajU2YKojXxbvEAQ4C3ykhtq3LzEGSXcCa4LJnpokTvfB1cIsogV7eEctXdn57D523fxw8DQKQOCrgYrjrhUKxwSOBwZYFFh6t6oA8V6L0BPd3dg8RFSNg6BxJWFAFe7\\\\\\\/+0IjgVLP31XzveT9QRlE0WJ1\\\\\\\/ZkLrH7qz2TjuyWaqi9kq1KC0XNcAVBNtP4NdROKfYIOT9AjYXgQQ9YnFnQ4f4Qar+K4E0\\\\\\\/Jv6AucDxB\\\\\\\/VQTzOmR4YRnsXtxTghmKcrJ5vjpFMMWqCippsmc2nv6Pp+k6quAZpFXPDPmQaGpyvJhAszBMffr+HGZtHYIbKl0eJKZ9EUBg+3qjVKZrqmzmx\\\\\\\/XoxmvHtm5nnaKIwhyQw1XJRhAUINIv9aevLKzfnp7fGhH\\\\\\\/27rxQlnAGeHL2Of2LcL+tdOW2on0ey+IA3yePDYAwhA++AYCzrbCkyY6tsei7cnAMwRftL10kjs4xjaxSaHcSMJWcGmZVl2IcwvlZK\\\\\\\/l11iY2A8G0AEiuwa5CWTItPY7biOZ+nxV\\\\\\\/ZwWmt7ExbBji75EnpdE0cshui+5p7NYbwJzTidOT+6rOBTSrT0zOqI2J4eLrJ+8zbfKEsLCReP5A8hkZr4JrzhU9RE+qR+h2GJ0OvRRZD+EDIVXOtfPxRXq52eMoKGjjdjbLsk0PyBQhMb0+dHisB4UiQHJ9+Rx6vidA03epHDaVWfbjcz7VcYu\\\\\\\/7kzfPHd05Lvag7qKJ+qBT8Jt9CQCUMqTumh2E+aG+zPmZ7B\\\\\\\/Xd6bBqxBI7BIvkn4tVHUqt54I0Ww9SIQixWoE7p3XmZN8TpJCZFAEn+oiyI6bglnN1t\\\\\\\/Q0N5uWZkpVS2tbW4vbzDYV\\\\\\\/GIwC\\\\\\\/IN+pvstDCPdZHA2OnTdgH5qWqS3\\\\\\\/SgwRYeSwEQYKl5EItPhQnKHlod8isBVE4VYQ\\\\\\\/CuCoUgtdmNb9Cdk7lLkB+y8PM\\\\\\\/zJ81yGwGCSPXsBPK+KX327ZDIIP\\\\\\\/t0CJqhMT8AtT4NFAUZzFw8aRSDmqCM6iwuhmJZKA\\\\\\\/rBPYKd6gbcyyzLpUz\\\\\\\/EO6un3Aa5JFBLnNa7IKI662Qq+bczTOt2z4olfd1m5T+Ezb\\\\\\\/t1xmabC0LI3YFUBj1YALiB6QqjmeFDWFRW9W6aDWDg68GdOmGNk6lZFTN8+RlmhpHsZifGWiGTSXKlysFqLCZEIDJ9bUsNBdz+UQUAeh5Oh7gtxA\\\\\\\/2MhACcw02zV3CxJGGdpleWwr5AsAbsWd1lyz3\\\\\\\/4N95QjwhP832U7Fk2qltMkA6cu6MOYKR8CtrM8jeECszzxUamUskTHoelPIAts1sW\\\\\\\/Dux95AOH4jPMgRDy8hGmnfGOkn4ho8FzRTJ+0qrTiDw1poXHaJoOOuSjerw3GzzWEjnCWJ\\\\\\\/qmaFCthyDMCW2vnJmxFUYjh0hQuXAY5QBKlb\\\\\\\/NbS6IRu2RaY2+sWaDQVuTwLcytFQoZ9gYThLJwsn22dw2TeaptqOIiDLlQc50kxXhQtqeBXwNnd7+zN1a1RPwOIqAd\\\\\\\/64fnb988Or9Fd1sSgvJ+k3uiw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xsiECqhgT8KwWKJjOXFYbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222b8ab25a8244a89fc95eb8100420e27b78016c5a0b28804d45accac134763889\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dtfP9rh4xNIJjg51lwIPok6LyOI3AF3+wz2svPBXR6mJHhQZMywMib6csJ2qbssGIiDlf5spm2Otpl1WdoFOCIt1e6h4KS2RtXw0q80XE2+newhDK8Nt54T2zVwjNKxPLr9EkMGPmDwM1LhnRFYuatxUh4NkwsmJie\\\\\\\/+H3eWNdmNbJMXyTjMfD0hJPBn5sDGsPkfBCScCYqcS\\\\\\\/u02CLtIEU2FwDGxV6OiBu1lrgoW5KaUNBfngUKw+1TOOPn\\\\\\\/fYOMtAPsFYtqJdDCM3ZGmQHOj3RJEJoboa9dGK02XNZ1CEyoVu85t6K8\\\\\\\/mcN9\\\\\\\/Y+TiNzrNv2QBHMziBgP7\\\\\\\/o\\\\\\\/vPc5VkLQrQ73SAzTVEEYG84s2BPiVrURnlZDaTPQk78ZKO0Yh0ww1udE6FG7HWUgr1+\\\\\\\/RSDRaLNd+h7kph05rcv\\\\\\\/S4cLmI9gNY2lKaQCHMiDA4iK34KVJTa7WSt6p3hOhsFI6cbg6VN9NWgPsvEKnjjB\\\\\\\/QYAkviU2gaIQYKon8lSiISXep3jp\\\\\\\/jspBmX0MsiLVCBjEmdlMrPbvdfZK9sZ+sZt5R5nXacqxYA7211z7VqXeUjC0LWlbSSnYbbnbcHasBk2UF3Zy15hkhVNp1CzhLGkxM5ZjGNXHAcegbK252WkykrjHPuN26vRFFKBo+gjbpzQWo89ahWXI5wz7vtmxLn2c3LhtGkQoqzoM0R\\\\\\\/4bpU88\\\\\\\/rh8HpYcLm8mVPLpx\\\\\\\/Kx2NKAgygmwg50DEANqmcnBDNaxp54dH1qr\\\\\\\/tOT8Cqh+kO4X2LMELNGjpbDA4aj+hXmoC6MNKdRYsh0ubCQhMln3dIRAZXsPYdAMZbBJ4nOoeHywyPIK+GYIyRfxWXRAtb0nQVKRN5Iins1W4kiE1WHSzRaBx\\\\\\\/sM6aWFTmINQcP\\\\\\\/2JQG2Y2IC65+0U69\\\\\\\/xkxrVxzOSPQ+1hJdfsGsK7M2L0\\\\\\\/+hhLQr0B+M\\\\\\\/mL50taLWTehUnzNx9pWurt\\\\\\\/edlqimHmZJtxlCuuVjxPyXlIr2KRsbIVV9AWYHJeAuP1nwdVyi3qCpfHyIHTb\\\\\\\/aFNCdaPHoiSYhjpLYRxKkdJK1L8i+WbDCiN2xhfCGnpHX59Ry89Baqe5GY2LPOWEHJ4rnRn17XWBGOuZ8\\\\\\\/s5xL+69OKPwDPrLdcSCoF4FNN5QzSgyn+rPqe4CF\\\\\\\/k003GYMjfQyL+Fr3NWQRo2WQWrvqNX+ob0x4HseXP+gr9sG\\\\\\\/OOwwwvsEBV5jH6fExkSUsu2kXBEUc68ahQxG1QKn5XCddWHypCAXKXUcPZyWT9u73U127fvvhEm6z0hRWRTtEdAIcv5wKeuox+Y+cKgQJ1OZQjBGN8ziuV5+4Yphee0Bsh9niNt1aQh9RFvAJLr\\\\\\\/ewOQnPlxqJLjxrI8VQ6kkn2DeCiGZXvsHFhHhhqII6yfyGf6XTQdIAM8Co071db7X+WHqw6Vx0T6icN9gNTpwIIMAutVtrAFuNyxONiZwK7e5KUICgs5QR9ptNbjuYHU2bjjS91LKw9b8LeobUZkWkVz1+i3XQX41axd8bt17CsuWe90N0fc0XJBIlvlN2SLcsTkcdBVBjdpv+8k3iJ4TTvN+3zX7v5sU3haU5tVnv8nqyYIhGnqSXS9\\\\\\\/R30nqDx4eigK+FK2FkSQ\\\\\\\/AlU4nBkk4HhFcJ66imy7gAEIiH9VMQ8xZITMKoMrxXgTNG2ucQVAP0rlshHUvMUJa51sUuzy0tbs0y\\\\\\\/B6JpZiZR9Up4nxoXYxdcFnH8lQ2N+wxzvkXC+tNpy9JW+CRZHmcLMWh+K1rJdxs1bfZGB2MeCmw6RUNmv5W6VVafEVn61GgylUcMJQ0Sq6wcZVRJRWxOSf1luD42Z77bxP848EdAV3XHHuImiVjoUolM\\\\\\\/6UxWZb0AYeQfWbM+Egm8cYGiHjr4\\\\\\\/my4PePyNMnF9F74Y6wPCmijlyf7xBHxXopHWNsRgn9yAb8DikKcBdOSTbp4qKabfMtuTRpDHv3gRBhXuGbm5GIvGoN\\\\\\\/IGArHtQM+Rl6CVbks3kWZMrmcj5yaw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sg\\\\\\\/EWrm75H1ZKtFXuqT1Zg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b9993d495759ecb6567f0bef3a7978faa4d1d93b2339d5991b96980e375e7716\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487109 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3669" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HU3AczWRscp9jIvNukJW6HuVbT4N8umymXC-seRVizI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-f1lgb5feUwSGlSpfzx1UFAbEeegBApipRhQ4LEmyPGY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3659" title="The Effect of Common Inversion Polymorphisms In(2L)t and In(3R)Mo on Patterns of Transcriptional Variation in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3681" title="Differential Expression of miRNAs in the Respiratory Tree of the Sea Cucumber Apostichopus japonicus Under Hypoxia Stress" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487109" id="node487109" data-pisa="ggg;7/11/3669" data-pisa-master="ggg;g3.117.1135" data-apath="/ggg/7/11/3669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Dynamics of Hybrid <em>Saccharomyces cerevisiae</em> During Vegetative and Meiotic Divisions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Abhishek Dutta</span>, <span class="highwire-citation-author" data-delta="1">Gen Lin</span>, <span class="highwire-citation-author" data-delta="2">Ajith V. Pankajam</span>, <span class="highwire-citation-author" data-delta="3">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="4">Nahush Bhat</span>, <span class="highwire-citation-author" data-delta="5">Lars M. Steinmetz</span> and <span class="highwire-citation-author" data-delta="6">Koodali T. Nishant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3669-3679;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1135 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487109" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Abhishek Dutta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BDutta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dutta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhishek%2BDutta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gen Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gen%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGen%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nahush Bhat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nahush%2BBhat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhat%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANahush%2BBhat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford University, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Stanford Genome Technology Center, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computation, Modelling, and Simulation, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nishantkt@iisertvm.ac.in" class="" data-icon-position="" data-hide-link-title="0">nishantkt@iisertvm.ac.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3669" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487109" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3669.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487109" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3669.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487109" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3669.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487109" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3669.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487109" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Mutation and recombination are the major sources of genetic diversity in all organisms. In the baker’s yeast, all mutation rate estimates are in homozygous background. We determined the extent of genetic change through mutation and loss of heterozygosity (LOH) in a heterozygous <em>Saccharomyces cerevisiae</em> genome during successive vegetative and meiotic divisions. We measured genome-wide LOH and base mutation rates during vegetative and meiotic divisions in a hybrid (S288c/YJM789) <em>S. cerevisiae</em> strain. The S288c/YJM789 hybrid showed nearly complete reduction in heterozygosity within 31 generations of meioses and improved spore viability. LOH in the meiotic lines was driven primarily by the mating of spores within the tetrad. The S288c/YJM789 hybrid lines propagated vegetatively for the same duration as the meiotic lines, showed variable LOH (from 2 to 3% and up to 35%). Two of the vegetative lines with extensive LOH showed frequent and large internal LOH tracts that suggest a high frequency of recombination repair. These results suggest significant LOH can occur in the S288c/YJM789 hybrid during vegetative propagation presumably due to return to growth events. The average base substitution rates for the vegetative lines (1.82 × 10<sup>−10</sup> per base per division) and the meiotic lines (1.22 × 10<sup>−10</sup> per base per division) are the first genome-wide mutation rate estimates for a hybrid yeast. This study therefore provides a novel context for the analysis of mutation rates (especially in the context of detecting LOH during vegetative divisions), compared to previous mutation accumulation studies in yeast that used homozygous backgrounds.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate" rel="nofollow">mutation rate</a></li><li class="kwd"><a href="/keyword/hybrid-yeast" class="hw-term hw-article-keyword hw-article-keyword-hybrid-yeast" rel="nofollow">hybrid yeast</a></li><li class="kwd"><a href="/keyword/loss-heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-loss-of-heterozygosity" rel="nofollow">loss of heterozygosity</a></li><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/mitotic-recombination" class="hw-term hw-article-keyword hw-article-keyword-mitotic-recombination" rel="nofollow">mitotic recombination</a></li></ul><p id="p-3">Most diploid organisms in nature possess heterozygous genomes. In budding yeast, homozygosity is thought to be the default state as seen from wild isolates that come from environments that are undisturbed by humans (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Magwene 2014</a>). Isolates of <em>Saccharomyces cerevisiae</em> from human-associated (industrial, agricultural, or clinical) environments are observed to have low (&lt;1000 SNPs) to high (&gt;30,000 SNPs) levels of heterozygosity (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Argueso <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Borneman <em>et al.</em> 2011</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Magwene 2014</a>). These heterozygosities may be generated due to the accumulation of heterozygous mutations during vegetative divisions, by outcrossing during infrequent sexual cycles [once every 50,000–100,000 mitotic generations (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ruderfer <em>et al.</em> 2006</a>), but see <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kelly <em>et al.</em> 2012</a>] or hybridization events mediated through human activities. The heterozygosity may be further maintained in natural environments by selection. On the other hand, mitotic recombination during vegetative divisions, intratetrad mating, which is common during <em>S. cerevisiae</em> meiosis, or selection on genetic variants may contribute to the loss of heterozygosity (LOH).</p><p id="p-4">The baker’s yeast <em>S</em><em>. cerevisiae</em> can undergo vegetative and meiotic divisions rapidly under laboratory conditions. The short generation time and small genome size of <em>S. cerevisiae</em> (12 Mb) facilitate analysis of genotypic changes through mutation and recombination processes over a large number of generations (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Halligan and Keightley 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nishant <em>et al.</em> 2009</a>). Most mutation accumulation studies in yeast and other microbes involve propagation of isogenic asexual lineages (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lee <em>et al.</em> 2012</a>), though sometimes the sexual cycle has also been incorporated (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>). The effect of natural selection is minimized through the use of inbred lines and bottlenecks at each generation where a limited number of individuals are randomly selected to produce the next generation. However, the homozygosity of such isogenic lines can mask the detection of genotypic changes induced by mitotic crossovers, gene conversions, and other types of DNA repair processes. Intra and interspecific hybrids of yeast and other organisms have been analyzed during experimental evolution. In addition, a recent study analyzed mutation accumulation during asexual propagation in the microcrustacean <em>Daphnia</em> that is naturally heterozygous (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Flynn <em>et al.</em> 2017</a>). Such hybrid genomes when propagated show large-scale aneuploidies, gross chromosomal rearrangements, and LOH that create a more homogeneous genome (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Antunovics <em>et al.</em> 2005</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Querol and Bond 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burke <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Morales and Dujon 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dunn <em>et al.</em> 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Stelkens <em>et al.</em> 2014</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Flynn <em>et al.</em> 2017</a>). Much of this drive toward homogeneity is due to the selection on heterozygous alleles and to purge out genetic incompatibilities (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Greig <em>et al.</em> 2002</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Dunn <em>et al.</em> 2013</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wolfe 2015</a>). Similar changes in copy number and genome rearrangements mediated by mitotic recombination have been observed during experimental evolution of isogenic <em>S. cerevisiae</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hansche <em>et al.</em> 1978</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dunham <em>et al.</em> 2002</a>). The mechanisms and distributions of mitotic recombination events are well characterized in yeast (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lee <em>et al.</em> 2009</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rosen <em>et al.</em> 2013</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">St Charles and Petes 2013</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yin and Petes 2013</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yim <em>et al.</em> 2014</a>). But it is not clear to what extent genotypic changes occur in hybrid yeast over a large number of generations through these somatic DSB repair processes especially when selection is minimized. There are also no measures of mutation rates associated with mitotic and meiotic divisions in hybrid yeast.</p><p id="p-5">We experimentally measured genome-wide LOH and base mutations in an artificial <em>S. cerevisiae</em> hybrid strain (S288c × YJM789) propagated through successive vegetative and meiotic divisions. External selection was minimal with the only constraint that the meiotic lines sporulate efficiently and that the spores are viable. The S288c/YJM789 hybrid has ∼60,000 heterozygous SNP markers distributed uniformly across the genome (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wei <em>et al.</em> 2007</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mancera <em>et al.</em> 2008</a>). Since these SNPs are well characterized, we used them to track genotype changes occurring in the S288c/YJM789 hybrid during vegetative and meiotic divisions. We addressed the following questions: (1) What is the extent and pattern of LOH during vegetative and meiotic divisions? (2) Is heterozygosity preserved on specific chromosomal regions and are there potential fixation biases toward the parental S288c or YJM789 alleles during LOH? (3) How does heterozygosity affect phenotypes such as spore viability? (4) What is the mutation rate associated with vegetative and meiotic divisions in the S288c/YJM789 hybrid?</p><p id="p-6">We observed that LOH through intratetrad mating was rapid in the meiotic lines (70% loss in three generations of meiosis) and associated with improved spore viability. A few of the vegetative lines showed extensive LOH suggesting the occurrence of abortive meiosis and return to growth events during vegetative propagation in laboratory conditions. The base mutation rates in the <em>S. cerevisiae</em> S288c/YJM789 hybrid were similar to previous estimates in other <em>S. cerevisiae</em> strains suggesting the S288c/YJM789 hybrid is not mutagenic.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Media and strains</h3><p id="p-7">The meiotic (M) and vegetative (V) lines were grown on either YPD (yeast extract, peptone, and dextrose) or synthetic complete (SC) media at 30° (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rose <em>et al.</em><span class="sc">1990</span></a>). For inducing meiosis, diploids cells were patched on sporulation media (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Argueso <em>et al.</em><span class="sc">2004</span></a>). After 72 hr on sporulation media, tetrads were isolated on YPD or SC using a Zeiss dissection microscope. To generate the parent diploid, cells from overnight patches of YJM789 (<em>MATα</em>, <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>::<em>hisG <a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a></em>, <em>cyh</em>) and S288c (<em>MATa</em>, <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003122/overview">lys5</a></em>) were crossed on SC plates to form diploids. They were then streaked on YPD and 12 single colonies were picked and patched on sporulation plates. After 3 d on sporulation media, one of the single colonies that sporulated was stocked and labeled as the parent diploid hybrid strain (KTY162). The KTY162 strain was used to generate the V and M lines.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and sequencing</h3><p id="p-8">Diploid colonies or spore colonies from tetrads were independently cultured overnight at 30° in YPD liquid medium. Genomic DNA was extracted from each culture using the PrepEase DNA isolation kit from Affymetrix following the manufacturer’s protocol. Whole genome sequencing was performed on Illumina HiSequation 2500 machines at Fasteris SA, Switzerland.</p></div><div id="sec-4" class="subsection"><h3>Read mapping, genotyping of whole-genome sequencing data</h3><p id="p-9">The sequence reads were mapped to the S288c genome (version 64-1-1, 2011) using bowtie2 (version 2.1.0) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Langmead and Salzberg 2012</a>). Uniquely mapped reads were only considered for the SNP calling (duplicate reads were removed using picardtools). SNPs defined in <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Mancera <em>et al.</em> (2008)</a> were used for all analysis. In order to reduce misalignment due to indels, we performed a local indel realignment after mapping the reads to the reference genome using GATK IndelRealigner. SNPs were called with multiple samples for M line and V lines using GATK unified genotype caller. R package was used for data visualization and downstream statistical analysis. To detect conserved fixed SNPs in the M lines with high spore viability, we subsetted the genotype matrix of all lines from M5 onwards and collapsed consecutive markers that have no change in genotype across all samples. Markers with same genotype across all samples were identified as the boundaries of conserved regions.</p></div><div id="sec-5" class="subsection"><h3>Analysis of LOH tracts</h3><p id="p-10">LOH tracts from sequencing of V1_57, V3_57, V4_57, V5_57 diploids were compared with the sequence data from the four haploid spores from these lines. None of the diploid LOH tracts supported by 10 or more SNPs were invalidated when compared with the haploid sequence data from the V1_57, V3_57, V4_57, V5_57 lines. But for LOH tracts supported by &lt;10 SNPs (and especially those supported by only two or three SNPs), some of the SNPs were sometimes called differently in the diploid and haploid data sets (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS1.pptx">Figure S1</a>). This problem may be due to genotyping issues in the diploid sequence data. Therefore, LOH tracts were called in all the V_57 diploid lines only if supported by 10 or more SNPs.</p></div><div id="sec-6" class="subsection"><h3>Analysis of new mutations</h3><p id="p-11">To detect new mutations from the M and V lines sequence data sets, we sequenced the parent diploid as well as the S288c and YJM789 strains used to generate the parent diploid. We recalibrated the base qualities in these bam files (generated using alignment with the S288c reference genome) using GATK. SNP positions where the coverage deviates from the median coverage of the sample (due to copy number variation or mapping issues) were excluded. More specifically, at a position, if coverage/median coverage &gt;1.65 || &lt;0.35, the SNP is filtered away. We also filtered away SNPs where QD (Quality by depth) ≤10. For each sample at each potential SNP, we consider the SNP as a mutation if it differs from the genotype in the parent diploid. We checked that it follows a non-mendelian inheritance pattern from the parents (YJM789 and S288c), and this corrects for potential genotyping error in the parent diploid. Next we filtered away sites if the genotype call is not optimal (Genotype quality &lt;30) in either the sample itself, or any of the parent diploid, S288c, or YJM789 strains. All new mutations were verified by Sanger sequencing.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-12">All vegetative and meiotic lines listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS1.xlsx">Table S1</a> are available upon request. Sequence data are available from National Center for Biotechnology Information Sequence Read Archive under accession number: SRP098673. The Data S1, SNP segregation files, and the custom R scripts are available online at the Dryad digital repository (<a href="http://dx.doi.org/10.5061/dryad.s14m0">http://dx.doi.org/10.5061/dryad.s14m0</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Whole genome sequencing analysis of the S288c/YJM789 hybrid vegetative and meiotic lines</h3><p id="p-13">Vegetative and meiotic lines of the S288c/YJM789 parent diploid strain were set up as described in <em>Materials and Methods</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> and <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>). For the meiotic lines, 20 diploid colonies derived from the parent diploid were patched on sporulation media for 3 d and a single complete tetrad from each colony was isolated. Each tetrad was placed on rich media and the germinated spores of opposite mating type mated to form diploids. The resulting colony was then sporulated and the bottleneck repeated. These lines were labeled M1_N to M20_N where N indicates the number of meioses. Most of the M lines showed reduced sporulation after successive rounds of meiosis and could not be propagated with a 3 d sporulation schedule. After seven generations of meiosis and intervening 140 vegetative divisions, only two lines could be maintained as the rest could not sporulate in 3 d (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). The two lines were further continued until M_31 (corresponding to 31 generations of meiosis). Whole genome sequence data were obtained from the two lines after the third (M1_3, M2_3), fifth (M1_5, M2_5), seventh (M1_7, M2_7), 10th (M1_10, M2_10), 15th (M1_15, M2_15), and 31st (M1_31, M2_31) generation of meiosis. Six additional lines after five generations of meiosis (M3_5, M5_5, M6_5, M7_5, M8_5, M9_5) were also sequenced to increase the sample size. To accurately genotype heterozygous sites, sequencing was performed on the diploid colonies as well as on the haploid spores obtained by sporulating the diploids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS1.pdf">File S1</a>). In parallel, 12 vegetative (V) lines were bottlenecked to single cells from a colony, every 2 d (20 generations). These were initially propagated for a total of 19 bottlenecks (380 generations) that correspond to the amount of time taken for the M_7 lines. These lines were labeled V1_N to V12_N, where N indicates the number of bottlenecks. Five of these lines were propagated further for a total of 57 bottlenecks (1140 mitotic generations) that correlate with the same length of time as the M_31 lines. But the number of mitotic generations in the M_31 lines (620) is fewer due to the intervening meiotic divisions. The V_57 lines (V1_57, V3_57, V4_57, and V5_57) were also sequenced as diploids as well as haploid spores. Due to a recessive lethal mutation in V2_57 (49% spore viability, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), sequencing of haploid spores was not performed for this line. Sequencing details for all M and V lines are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS1.xlsx">Table S1</a>. The average sequencing depth was ∼70× (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS1.xlsx">Table S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Experimental set up of the vegetative (V) and meiotic (M) lines from the parent diploid. The numbers in brackets indicate the number of V and M lines available at the vegetative bottleneck (V_19, V_57) or meiotic generations (M_3, M_5, M_7, M_10, M_15, M_31). LOH events are observed in both V and M lines. (B) LOH in the M lines following each generation of meiosis and intratetrad mating. Continuous lines (red and green) show the observed number of heterozygous SNPs in two M lines (Line 1 and 2). Black dashes show the number of heterozygous SNPs expected assuming wild-type S. cerevisiae recombination rates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-514106301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Experimental set up of the vegetative (V) and meiotic (M) lines from the parent diploid. The numbers in brackets indicate the number of V and M lines available at the vegetative bottleneck (V_19, V_57) or meiotic generations (M_3, M_5, M_7, M_10, M_15, M_31). LOH events are observed in both V and M lines. (B) LOH in the M lines following each generation of meiosis and intratetrad mating. Continuous lines (red and green) show the observed number of heterozygous SNPs in two M lines (Line 1 and 2). Black dashes show the number of heterozygous SNPs expected assuming wild-type &lt;em&gt;S. cerevisiae&lt;/em&gt; recombination rates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3669/F1.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3669/F1.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487211" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">(A) Experimental set up of the vegetative (V) and meiotic (M) lines from the parent diploid. The numbers in brackets indicate the number of V and M lines available at the vegetative bottleneck (V_19, V_57) or meiotic generations (M_3, M_5, M_7, M_10, M_15, M_31). LOH events are observed in both V and M lines. (B) LOH in the M lines following each generation of meiosis and intratetrad mating. Continuous lines (red and green) show the observed number of heterozygous SNPs in two M lines (Line 1 and 2). Black dashes show the number of heterozygous SNPs expected assuming wild-type <em>S. cerevisiae</em> recombination rates.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487252/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487252/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Heterozygous SNP counts from sequencing of diploid M and V lines</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Meiotic lines show almost complete LOH and improved viability</h3><p id="p-16">Intratetrad mating of spores is expected to reduce heterozygosity in the M lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS2.pdf">File S2</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Knop 2006</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>). Sequence information from diploid colonies of the two M lines, line 1 (M1_3, M1_5, M1_7, M1_10, M1_15, M1_31) and line 2 (M2_3, M2_5, M2_7, M2_10, M2_15, M2_31), were analyzed for heterozygous SNP markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS2.xlsx">Table S2</a>). We focused on 46,281 SNPs that could be called out from all the M lines.</p><p id="p-17">The number of heterozygous SNPs rapidly declined with increasing number of meiotic generations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We traced the lineage of each of the 46,281 SNPs for two M lines (1 and 2). By the third round of meiosis, ∼30% of the SNPs were heterozygous and by the seventh round it was ∼10% in these two lines (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). M1_7 and M2_7 contained 4688 and 5800 heterozygous SNPs, respectively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide distribution of SNPs in the two M lines (1 and 2) after 3rd, 7th, 15th, and 31st generation of meiosis. Homozygous SNPs are shown in orange while heterozygous SNPs are shown in cyan. More than 99% of the heterozygous sites are fixed by M_31." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-514106301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide distribution of SNPs in the two M lines (1 and 2) after 3rd, 7th, 15th, and 31st generation of meiosis. Homozygous SNPs are shown in orange while heterozygous SNPs are shown in cyan. More than 99% of the heterozygous sites are fixed by M_31.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3669/F2.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3669/F2.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487273" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Genome-wide distribution of SNPs in the two M lines (1 and 2) after 3rd, 7th, 15th, and 31st generation of meiosis. Homozygous SNPs are shown in orange while heterozygous SNPs are shown in cyan. More than 99% of the heterozygous sites are fixed by M_31.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">We were curious to know whether all heterozygous segregating sites would eventually be lost in the M lines (except for the <em>MAT</em> locus, which is under selection). To address this question, we propagated the two lines until M_31. Sequence analysis of M1_31 and M2_31 showed the presence of 146 and 135 heterozygous sites, respectively (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS2.xlsx">Table S2</a>). Most of the heterozygous SNPs in M1_31 (85 out of 146) and M2_31 (102 out of 135) were in proximity to the <em>MAT</em> locus on chromosome III, suggesting that they were retained in the heterozygous state because of passive linkage to the <em>MAT</em> locus. The rate of LOH was similar for the two lines and maximal in the initial rounds of meiosis when heterozygous markers were maximum. The rate of LOH gradually declined in subsequent generations of meiosis and intratetrad mating. Similar results were observed from the sequencing data of haploid spores from the M lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS3.xlsx">Table S3</a>).</p><p id="p-20">Since meiotic recombination parameters of the S288c/YJM789 hybrid are well characterized, we mathematically analyzed the LOH in the two M lines (1 and 2) by incorporating high-resolution genome-wide meiotic crossover data of the S288c/YJM789 hybrid (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS2.pdf">File S2</a>). The M lines are expected to retain on average 98 SNPs by the 31st meiotic generation. This number is close to the experimentally observed number of heterozygous sites in the two M lines. The excess of heterozygous alleles in the M lines compared to the expectation may be because of altered recombination (enhanced) beyond M_5 as the lines become extremely homozygous or due to factors other than recombination (<em>e.g.</em>, chromosomal structural features or genomic loci besides the <em>MAT</em> locus that affect mating) that contribute to the maintenance of residual heterozygosity in the M lines. Besides the <em>MAT</em> locus, the distribution of the residual heterozygous sites between the two lines were unique (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-21">The M lines showed rapid fixation of alleles toward the S288c or YJM789 state. In order to statistically test biased fixation toward S288c or YJM789 alleles, we used the sequence data from the eight M5 lines (M1_5, M2_5, M3_5, M5_5, M6_5, M7_5, M8_5, M9_5). Five of the eight M_5 lines showed excess fixation toward the YJM789 alleles. Across these eight lines, an average of 18,276 SNPs were fixed toward YJM789 and 16,897 SNPs were fixed toward S288c (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS2.pptx">Figure S2</a>A and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS2.xlsx">Table S2</a>). We simulated five successive rounds of meiosis with intraspore mating, using crossover locations from <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Mancera <em>et al.</em> (2008)</a> and <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> (2014)</a> to test if the fixations toward S288c and YJM789 SNPs are significant. We tabulated the number of fixed YJM789 SNPs in the eight M_5 lines, over 5000 simulations. The average number of YJM789 fixed SNPs in the eight M_5 lines (18,276 SNPs) was within the expected range based on simulation data and suggest these biased fixations are not statistically significant (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS2.pptx">Figure S2</a>B). Asymmetric fixation of alleles derived from either parent in a hybrid has been observed previously (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Tang <em>et al.</em> 2010</a>).</p><p id="p-22">The S288c × YJM789 hybrid has a spore viability of 85% unlike the S288c diploid that has a spore viability &gt;97% (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">McCusker <em>et al.</em> 1994</a>). The presence of ∼60,000 SNPs along with indels can create incompatibilities in the S288c/YJM789 hybrid causing reduced spore viability (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Wei <em>et al.</em> 2007</a>). Meiotic lines at M_3 and beyond showed significantly improved spore viability (&gt;92%) compared to the parent diploid (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). This result is interesting because it suggests that the significant loss in heterozygosity observed by the third generation of meiosis or fixation of certain S288c and YJM789 alleles may contribute to the improved spore viability in the M lines (see below).</p></div><div id="sec-11" class="subsection"><h3>Variable LOH in hybrid vegetative lines</h3><p id="p-23">Hybrid vegetative lines can be used to detect LOH during mitotic divisions. Hybrid <em>S. cerevisiae</em> strains have been previously used to map LOH events involving mitotic crossovers and break induced replication (BIR) events on specific chromosomes III, IV, and V and genome wide (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lee <em>et al.</em> 2009</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Rosen <em>et al.</em> 2013</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">St Charles and Petes 2013</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Yin and Petes 2013</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Yim <em>et al.</em> 2014</a>). These studies have generated considerable insights into the mechanisms and distributions of mitotic recombination. The mitotic recombination events are thought to occur with ∼10<sup>5</sup>-fold less frequency than meiotic recombination and therefore require a selection system for their detection (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lee <em>et al.</em> 2009</a>). But it is possible that over a large number of divisions they can cause significant genotypic changes that can be detected and is relevant to measure, given the ratio of mitotic to meiotic cycles in yeast (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Ruderfer <em>et al.</em> 2006</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kelly <em>et al.</em> 2012</a>). Diploid colonies from five vegetative lines (V1–V5) were whole-genome sequenced after 57 bottlenecks (corresponding to the 31st generation of meiosis for the M lines) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS4.xlsx">Table S4</a>). We focused on the 47,954 SNPs that are common to all the five V lines. In the lines V1_57 and V2_57, only 2–3% of the SNPs showed LOH compared to &gt;99% of the heterozygous sites that became homozygous in the meiotic lines propagated for the same length of time (M_31) (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). In V4_57, 8% of the SNPs showed LOH, which was primarily due to a single large terminal LOH tract on chromosome XII (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In the most extreme cases in V3_57 and V5_57, up to 35 and 22% of the heterozygous sites became homozygous. LOH in the V lines likely results from the repair of DNA lesions using the homologous chromosome followed by segregation of sister chromatids. Mitotic crossovers, local gene conversions, BIR, and chromosome loss are implicated in LOH events during mitosis (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Paques and Haber 1999</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barbera and Petes 2006</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide plots of LOH and new base mutations in the five V_57 lines. Regions showing loss of heterozygosity are in orange (2:0 or 0:2). Regions in cyan are heterozygous (1:1). Asterisk (*) symbols show the position of the new base mutations in the V_57 lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-514106301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide plots of LOH and new base mutations in the five V_57 lines. Regions showing loss of heterozygosity are in orange (2:0 or 0:2). Regions in cyan are heterozygous (1:1). Asterisk (*) symbols show the position of the new base mutations in the V_57 lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3669/F3.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/11/3669/F3.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487285" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Genome-wide plots of LOH and new base mutations in the five V_57 lines. Regions showing loss of heterozygosity are in orange (2:0 or 0:2). Regions in cyan are heterozygous (1:1). Asterisk (*) symbols show the position of the new base mutations in the V_57 lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We analyzed the SNP distribution data in the V_57 lines to map the accumulated LOH tracts genome wide. Sequence data from both the diploid V_57 lines as well as their haploid derivatives (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS5.xlsx">Table S5</a>) were used to call LOH tracts accurately (<em>Materials and Methods</em>). Genome-wide distribution of 141 LOH tracts in the V lines is shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS3.pptx">Figure S3</a>A along with information on chromosomal location and tract sizes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS6.xlsx">Table S6</a>). Since the LOH tracts in the vegetative lines are summed over 1140 mitotic generations, these tracts may have been generated through multiple DSB repair mechanisms that are difficult to distinguish. The tract sizes showed a broad distribution from &lt;100 bp to over 100 kb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS3.pptx">Figure S3</a>, B and C and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS6.xlsx">Table S6</a>). The tracts were on average closer to telomeres than centromeres. The average distance of the LOH tracts to the nearest telomere was 126.2 kb compared to 244.7 kb for the centromere (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS6.xlsx">Table S6</a>). This is consistent with the increase in mitotic recombination rates away from the centromere in <em>S. cerevisiae</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mandegar and Otto 2007</a>). Centromere distal regions have also been shown to be more prone to spontaneous LOH events in <em>S. cerevisiae</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2008</a>). V1_57, V2_57, and V4_57 showed fewer LOH tracts than V3_57 and V5_57. The long LOH tracts in V1_57, V2_57, and V4_57 lines were primarily terminal as expected to arise during mitotic crossovers or BIR (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS3.pptx">Figure S3</a>A). In contrast, V3_57 and V5_57 have frequent long internal LOH tracts that might arise due to high levels of DNA damage and repeated rounds of mitotic recombination events creating chimeric chromosomes that are fixed alternatively toward S288c or YJM789 (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Morales and Dujon 2012</a>). The high frequency of long internal LOH tracts also suggest the possibility that V3_57 and V5_57 may have been through a cycle of abortive meiosis and return to growth involving DSB repair (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dayani <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Laureau <em>et al.</em> 2016</a>). Twenty-eight 2:0/0:2 LOH events &gt;100 kb were observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS6.xlsx">Table S6</a>) with the largest one ∼827 kb on chromosome XII in V4_57. Chromosome XII has been previously also observed to have large LOH tracts (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Magwene <em>et al.</em> 2011</a>). The short LOH tracts (&lt;1 kb) are likely to have arisen through local gene conversions not associated with crossovers (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Palmer <em>et al.</em> 2003</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Laureau <em>et al.</em> 2016</a>). Chromosomal aneuploidies contributing to LOH were not observed in the V lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS4.pdf">Figure S4</a>), which suggests the S288c/YJM789 hybrid does not show chromosomal instability during mitotic divisions.</p><p id="p-26">During experimental evolution of interspecific hybrids, biased elimination of one of the parent genomes as well as specific genome changes driven by selection are often observed (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Antunovics <em>et al.</em> 2005</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Dunn <em>et al.</em> 2013</a>). When the V line’s SNP data were analyzed for biased fixation, 81 LOH tracts were fixed toward S288c compared to 60 tracts toward YJM789 (<em>P</em> = 0.092, binomial test) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS6.xlsx">Table S6</a>). Despite random fixation of the LOH tracts toward S288c or YJM789, we observed more fixed YJM789 SNPs as most of the longer LOH tracts are fixed toward YJM789 alleles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS2.pptx">Figure S2</a>A and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS6.xlsx">Table S6</a>).</p><p id="p-27">Unlike the significant improvement in spore viability observed in all meiotic lines, the spore viability of two of the vegetative lines (V1_57 and V3_57) was not significantly different from the parent diploid strain (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). V2_57 showed the presence of a recessive lethal mutation. V4_57 and V5_57 showed improvement in spore viability compared to the parent diploid (0.05 &lt; <em>P</em> &gt; 0.01), but the differences were less significant when compared to the meiotic lines (<em>P</em> &lt; 0.0001). These results suggest LOH did not affect the spore viability of the hybrid vegetative lines.</p></div><div id="sec-12" class="subsection"><h3>Base mutations in the hybrid vegetative and meiotic lines</h3><p id="p-28">While LOH events and intratetrad mating can reduce heterozygosity, new mutations can add to the number of heterozygous sites in the genome. We estimated the number of base substitutions of the S288c/YJM789 hybrid vegetative (V1_57, V2_57, V3_57, V4_57, V5_57) and meiotic lines (M1_31, M2_31) with reference to the parent diploid genome. We detected one mutation in the meiotic line (M1_31) and 15 mutations across three vegetative lines (eight in V3_57, five in V4_57, and two in V5_57) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). No base substitution mutations were observed in M2_31 and V1_57, V2_57. It is possible that the recessive lethal mutation in V2_57 is caused by an in-del or other structural variation. All the 16 mutations were validated by Sanger sequencing. All the new mutations were heterozygous and did not occur in the coding region of the genome. Only one mutation was shared between two of the vegetative lines V3_57 and V4_57 (C &gt; A on Chr IV). The vegetative lines were propagated for a total of 1140 mitotic divisions (57 bottlenecks) and the 24.04 Mb genome was sequenced at 99.9% sequence coverage. The mutation rates are 2.92 × 10<sup>−10</sup> for V3_57; 1.82 × 10<sup>−10</sup> for V4_57, and 0.73 × 10<sup>−10</sup> for V5_57. The average mutation rate for the three vegetative lines is therefore 1.82 × 10<sup>−10</sup> per base per division. The average mutation rates in the S288c/YJM789 hybrid are similar to mutation rate estimates from a large set of 145 diploid <em>S. cerevisiae</em> vegetative mutation accumulation lines from <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zhu <em>et al.</em> (2014)</a> (1.67 × 10<sup>−10</sup> per base per generation). They are also comparable to the homozygous <em>S. cerevisiae</em> SK1 diploid mutation rates (2.9 × 10<sup>−10</sup>) in vegetative lines (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487148/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487148/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genomic locations of the 16 base mutations identified in the M and V lines</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">To calculate mutation rates for the S288c/YJM789 hybrid meiotic lines, we multiplied the number of mutations observed in the meiotic line by two, as half of the mutations in the meiotic lines are expected to be lost during intratetrad mating (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>). The base substitution rate for M_31 is estimated to be 1.33 × 10<sup>−10</sup> per base per division (31 meiotic divisions + intervening 620 mitotic divisions and 99.9% sequence coverage of the 24.04 Mb diploid genome). For comparison, the base substitution rate in <em>S. cerevisiae</em> SK1 meiotic lines was 3.9 × 10<sup>−10</sup> after 50 meiotic divisions and 1000 intervening mitotic divisions (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>).</p><p id="p-31">Higher mutation rates have been observed in heterozygotes compared to homozygotes based on sequence analysis of parent–progeny in <em>Arabidopsis</em> and rice (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2015</a>). Since we do not have mutation rate data for <em>S. cerevisiae</em> meiotic lines that are homozygous for S288c or YJM789, we cannot infer whether the same is true in <em>S. cerevisiae</em> as well. It is also important to note that S288c/YJM789 is an artificial hybrid with uniform distribution of heterozygous markers and these heterozygosities were progressively reduced during the meiotic divisions. In the meiotic lines, heterozygosity is lost by 70% within three generations of meiosis. In addition, the meiotic lines undergo both vegetative and meiotic divisions and there is 50% loss of mutations during intratetrad mating of spores (<a id="xref-ref-51-7" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>). It is therefore likely that more mutations were not observed in the S288c/YJM789 hybrid meiotic lines due to the study design (<a id="xref-ref-51-8" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>; see <em>Discussion</em>). In the vegetative lines, no base mutations were observed in V1_57 and V2_57 that have 2–3% loss of heterozygous SNPs. All of the base mutations were observed in the V3_57, V4_57, and V5_57 lines, which have a significantly higher percentage of LOH events (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). This is consistent with previous observations in yeast, where the repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease-induced DSBs causes high mutation frequency proximal to the break site (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Strathern <em>et al.</em> 1995</a>). But only two of the 16 mutations observed in the vegetative lines were located within the LOH tracts (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). Since all the 16 mutations are heterozygous, it is possible that the other mutations happened in locations where the mitotic recombination events were repaired using the sister chromatid; the DSB repair through other mechanisms did not leave an LOH signature or the SNP density was not sufficient to detect an LOH. It is also possible that some of the mutations in the LOH tracts may have been fixed in favor of the parent allele, and so the mutation event is not detected. It is also possible that at least for V3_57 and V5_57, mutations may be associated with recombination repair of DSBs during partial entry into meiosis followed by return to growth.</p></div><div id="sec-13" class="subsection"><h3>Heterozygosity determines spore viability in the hybrid meiotic lines</h3><p id="p-32">The meiotic lines showed enhanced spore viability. We tested if the improved spore viability is due to the biased fixation toward S288c/YJM789 alleles or due to a general reduction in heterozygosity. We observed specific conserved chromosomal regions that were fixed either toward S288c or YJM789 alleles in the meiotic lines. We identified 20 such regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS2.pptx">Figure S2</a>C). The largest region was observed on chromosome XV, and it showed fixation towards the S288c alleles. We backcrossed M2_15 spores (1104 SNPs and 100% SV) with the parent S288c and YJM789 strains. The backcross increases the heterozygosity and reintroduces the incompatibilities that might have existed in the parent hybrid. The heterozygous SNPs in the M2_15 × S288c cross increased to 22,923 SNPs and in the YJM789 cross to ∼19,838 SNPs. In addition, the backcross with the parent S288c creates an S288c/YJM789 hybrid set for the YJM789 fixed candidate regions, while the backcross with YJM789 created an S288c/YJM789 hybrid set for the S288c fixed candidate regions. In both the crosses, we observed a spore viability of ∼95% (<em>n</em> = 60 tetrads), which is significantly higher than that for the wild-type hybrid parent (85%, Fischer exact test, <em>P</em> &lt; 0.0001). Similarly, spores from M6_5 (12,383 SNPs, 92% spore viability) crossed with S288c (26,693 SNPs) and YJM789 (17,004 SNPs) showed a similar spore viability of 90%. These results suggest that the regions fixed either in favor of S288c or YJM789 may have no role in the improved spore viability of the M lines. Instead, the spore viability of the M lines is affected by the heterozygosity of the lines. Previous studies have observed a negative correlation of heterozygosity with sporulation efficiency as well as spore viability in <em>Saccharomyces</em> wild isolates (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mortimer <em>et al.</em> 1994</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Liti <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cubillos <em>et al.</em> 2011</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Magwene <em>et al.</em> 2011</a>).</p><p id="p-33">We further tested this hypothesis using different <em>S. cerevisiae</em> artificial hybrid combinations of S288c, SK1, YJM789, and RM11-1a as well as data from the M lines (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The heterozygosity in these hybrids and the M lines is well defined facilitating a calibrated comparison of spore viability with the level of heterozygosity. Among the different <em>S. cerevisiae</em> artificial hybrid combinations, the hybrid with the least heterozygosity, <em>i.e.</em>, RM11-1a × YJM789 (∼30,000 SNPs; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gresham <em>et al.</em> 2006</a>), showed the maximum spore viability (90%). Hybrids with higher heterozygosity, <em>e.g</em>., S288c × RM11-1a (∼46,000 SNPs; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Qi <em>et al.</em> 2009</a>), S288c × SK1 (∼62,000 SNPs; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Martini <em>et al.</em> 2011</a>), SK1 × YJM789 (∼65,000 SNPs), and SK1 × RM11-a (∼69,000 SNPs), showed lower spore viability (85, 73, 77, and 76% respectively) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Overall a strong negative correlation was observed between heterozygosity and spore viability (<em>r</em> = −0.94, <em>P</em> = 2.5 × 10<sup>−13</sup>). These observations suggest the high spore viability phenotype of the M lines is due to the reduction in the heterozygous load. Reduced heterozygosites can result in less heteroduplex rejection during recombination repair of DSBs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chakraborty and Alani 2016</a>). The improved repair outcomes and fewer genetic incompatibilities (<em>e.g.</em>, between S288c-Mlh1 and SK1-Pms1; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Heck <em>et al.</em> 2006</a>) may increase spore viability in homozygous backgrounds.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterozygosity is negatively correlated with spore viability. The number of heterozygous SNPs and spore viability are plotted against each other for the S288c &#xD7; YJM789 diploid M lines and for hybrid crosses involving other S. cerevisiae strains. Data for the M lines are from Table 1. Data for other hybrid crosses are shown in the text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-514106301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heterozygosity is negatively correlated with spore viability. The number of heterozygous SNPs and spore viability are plotted against each other for the S288c &#xD7; YJM789 diploid M lines and for hybrid crosses involving other &lt;em&gt;S. cerevisiae&lt;/em&gt; strains. Data for the M lines are from Table 1. Data for other hybrid crosses are shown in the text.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3669/F4.medium.gif" width="440" height="362"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/11/3669/F4.medium.gif" width="440" height="362"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487147" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Heterozygosity is negatively correlated with spore viability. The number of heterozygous SNPs and spore viability are plotted against each other for the S288c × YJM789 diploid M lines and for hybrid crosses involving other <em>S. cerevisiae</em> strains. Data for the M lines are from <a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>. Data for other hybrid crosses are shown in the text.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Generation of a panel of heterogeneous inbred families (HIFs) from the hybrid meiotic lines</h3><p id="p-35">Crosses between inbred lines can facilitate fine mapping of QTL (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Flint and Mott 2001</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bergelson and Roux 2010</a>). We experimentally determined the mating type of all spores derived from the M lines. These spores were mated <em>in silico</em> and the genome sequence information was used to generate a total of 1369 <em>in silico</em> diploid genomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/TableS7.xlsx">Table S7</a>). These <em>in silico</em> diploid genomes show varying numbers of SNP counts and retain heterozygosity at specific regions on the genome [Data S1, available online at the Dryad digital repository (<a href="http://dx.doi.org/10.5061/dryad.s14m0">http://dx.doi.org/10.5061/dryad.s14m0</a>)]. The rest of the genome remains homozygous for S288c or YJM789. This collection of 1369 <em>in silico</em> diploid genomes represents a panel of HIFs that can be experimentally generated by crossing the haploid spores from the M lines. The distribution of SNPs for two representative HIF strains that were also experimentally analyzed for spore viability is shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. The two diploid strains, M2BD_3 and M2DA_10, were generated by crossing two spores from M2_3 and M2_10 each. They have spore viability ≥90% and different numbers of heterozygous markers. M2BD_3 has 8765 heterozygous SNPs while M2DA_10 has 1108 heterozygous SNPs. Further, M2DA_10 has nine chromosomes (XIII, XI, X, IX, VIII, VI, V, IV, and I) that are completely homozygous while the rest of the chromosomes have heterozygous regions. Such HIF strains can be used to study the effects of varying levels of heterozygosity on different biological processes including spore viability (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) and meiotic recombination. Since SNP markers are lost as the lines become homozygous, cytological, physical, or biochemical methods can be used to test if genome-wide meiotic recombination is enhanced. The HIF lines are also useful for mapping QTL for traits that differ between S288c and YJM789. The S288c × YJM789 hybrid displays heterosis across many phenotypes including high-temperature growth and sporulation (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Steinmetz <em>et al.</em> 2002</a>). If a trait of interest segregates in such a diploid, it is relatively easy to identify the causative SNPs as the regions of heterozygosity are limited. If candidate regions for the QTL are already known, one can use the sequence information of the spores from the M lines to generate hybrid diploids that are heterozygous specifically in the candidate regions and thus fine map the QTL (Table S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative in silico genomes generated by crossing spores from M_3 and M_10 lines. White regions on the chromosome are homozygous for S288c or YJM789. Red and Blue indicate S288c and YJM789 SNPs respectively. The diploids generated from the cross have good viability (&gt;90%) and are heterozygous at specific chromosomal regions. The green and gray regions indicate centromere positions and chromosome boundaries, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-514106301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Representative &lt;em&gt;in silico&lt;/em&gt; genomes generated by crossing spores from M_3 and M_10 lines. White regions on the chromosome are homozygous for S288c or YJM789. Red and Blue indicate S288c and YJM789 SNPs respectively. The diploids generated from the cross have good viability (&gt;90%) and are heterozygous at specific chromosomal regions. The green and gray regions indicate centromere positions and chromosome boundaries, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3669/F5.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/11/3669/F5.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3669/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487268" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Representative <em>in silico</em> genomes generated by crossing spores from M_3 and M_10 lines. White regions on the chromosome are homozygous for S288c or YJM789. Red and Blue indicate S288c and YJM789 SNPs respectively. The diploids generated from the cross have good viability (&gt;90%) and are heterozygous at specific chromosomal regions. The green and gray regions indicate centromere positions and chromosome boundaries, respectively.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-37">An artificial hybrid such as S288c × YJM789 provides a unique opportunity to study the interaction between two independent genomes that have never existed together, in successive vegetative and meiotic generations. Unlike isogenic strains that can be propagated for successive meiotic divisions for large numbers of generations (<a id="xref-ref-51-9" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>), we observe that hybrid strains like the S288c/YJM789 pose unique problems during meiotic propagation. Only 2 of the 20 M lines could be propagated beyond the seventh generation of meiosis due to poor sporulation of the other lines. The most likely explanation is that both S288c and YJM145 (the diploid progenitor of YJM789) are poor sporulators (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">McCusker <em>et al.</em> 1994</a>). In comparison, the S288c × YJM789 hybrid efficiently sporulates within 3 d. Reduction in sporulation efficiency of many of the meiotic lines suggest that alleles from both S288c and YJM789 contribute to the good sporulation phenotype of the S288c/YJM789 hybrid. During the meiotic propagation of the lines, some of the alleles contributing to the high sporulation efficiency may be lost through fixation of the opposite allele. Consistent with this, even after five generations of meiosis, the sporulation efficiency of the M_5 lines was variable (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS5.pptx">Figure S5</a>).</p><p id="p-38">Whole-genome sequence analysis of meiotic lines showed a significant decline in heterozygosity as expected from intratetrad mating. A significant decrease in heterozygosity was also observed in some of the vegetative lines (V3_57, V4_57, V5_57). These results are relevant for interpreting heterozygosity in <em>S. cerevisiae</em> populations, since meiosis occurs infrequently in <em>S. cerevisiae</em> and most diploids that arise are produced by intratetrad mating (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Magwene <em>et al.</em> 2011</a>). Previous studies have suggested a very low outcrossing frequency of 0.5 per 10<sup>5</sup> mitotic divisions (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Ruderfer <em>et al.</em> 2006</a>). However, a recent study (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kelly <em>et al.</em> 2012</a>) shows mating and outcrossing frequency in <em>S. cerevisiae</em> is much higher (1 in 100 mitotic divisions). These new estimates of outcrossing can better explain heterozygosity in wild <em>S. cerevisiae</em>, given the extent of LOH in a few of the vegetative lines and during intratetrad mating that we observed in this study. Residual heterozygosity observed in these two M lines was conserved only at the <em>MAT</em> locus since we were selecting for hybrids that can sporulate. Selection acting on standing genetic variation can cause fixation as well as LOH of linked regions (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Burke <em>et al.</em> 2010</a>). We show that almost all heterozygous sites in the S288c/YJM789 hybrid yeast genome can be fixed even with minimal external selection. The fixation of SNPs occurred in large blocks after each meiosis consistent with fixation driven primarily by mating comprising recombinant spores (Figure S6, Figure S7, and Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-39">The fixation of the heterozygous sites in the meiotic lines can occur randomly toward S288c or YJM789 alleles or may show biases, either local or genome wide, especially if selection favoring either of the alleles is involved (<em>e.g.</em>, high temperature where the YJM789 genotype grows better; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Wei <em>et al.</em> 2007</a>). Even without apparent external selection, the presence of two different genomes (S288c and YJM789) in the hybrid could create fixation of either genotype through biased selection. Such regions were observed in the M lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS2.pptx">Figure S2</a>C). The fixation of these specific regions toward S288c and YJM789 alleles may arise due to differences in expression from the S288c and YJM789 alleles of the same gene; epigenetic reasons; preferential initiation of meiotic DSBs from chromosomal regions containing S288c or YJM789 SNPs followed by biased MMR; or selection for good sporulators (Figure S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS2.pdf">File S2</a>).</p><p id="p-40">Heterozygous SNPs in the S288c/YJM789 hybrid facilitated the analysis of genome dynamics as cells cycle through vegetative divisions. Genetic changes like LOH can uncover mutations in tumor suppressor genes commonly associated with cancer development (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cavenee <em>et al.</em> 1983</a>). LOH is shown to create extensive genome variation in <em>Cryptococcus</em> hybrids enabling pathogenesis (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2012</a>). Similarly, LOH events have been associated with the adaptation of <em>S. cerevisiae</em> and <em>Candida albicans</em> to stressful conditions, such as exposure to antifungals (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Selmecki <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gerstein <em>et al.</em> 2014</a>). Given the role of LOH in evolution and disease, it is useful to understand the extent of LOH over a large number of mitotic divisions. Previous analysis of mitotic LOH events in <em>S. cerevisiae</em> that allowed for selection of daughter cells containing the products of mitotic recombination have provided mechanistic insights into mitotic crossovers and gene conversion (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Lee <em>et al.</em> 2009</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Rosen <em>et al.</em> 2013</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">St Charles and Petes 2013</a>; <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Yim <em>et al.</em> 2014</a>). Our study design does not facilitate mechanistic analysis, as the outcome of the DSB repair processes are analyzed after many generations. Instead our study design provides information on the scale of genotypic changes brought about by the accumulated effects of various DSB repair processes over 1140 vegetative divisions (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). In our study, we have observed: (1) long LOH events extending toward the ends of chromosomes, probably as a result of BIR or mitotic crossovers; (2) small LOH tracts (&lt;1 kb) as a result of local gene conversions; and (3) internal large LOH events that are probably a consequence of repeated mitotic recombination events in response to DNA damage during the propagation of the vegetative lines (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pryszcz <em>et al.</em> 2014</a>). Alternatively, they may arise during DSB repair in response to Spo11-mediated DSBs during abortive meiosis. This is supported by the observation that the number and distribution of LOH tracts in V3_57 and V5_57 are distinct from V1_57, V2_57, and V4_57 (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FigureS3.pptx">Figure S3</a>, B and C). Extrapolating the spontaneous mitotic crossover rate of 6 × 10<sup>−5</sup> per division for the right arm of chromosome IV (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">St Charles and Petes 2013</a>) to the whole genome, only 5–6 crossover/conversion events are expected after 1000 mitotic divisions. The number of long LOH tracts observed in V1_57, V2_57, and V4_57 are consistent with these numbers (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>). But V3_57 and V5_57 have many more long LOH tracts that are also frequently internal. The pattern of LOH in V3_57 and V5_57 would either require a high frequency of DNA damage during vegetative growth or can also be explained, for example, by a double crossover during return to growth. The latter possibility suggests that yeast cells may enter the meiotic program and return to growth during vegetative propagation perhaps in response to nutrient stress. The extensive LOH observed in some of the vegetative lines (presumably due to return to growth) support the idea that LOH is an important tool for the evolution of vegetatively propagating cells by facilitating the fixation of beneficial alleles (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Mandegar and Otto 2007</a>). Given the applications of hybrid yeast strains in the industry, knowledge of the scale of genome dynamics in hybrid yeast will also be useful for the design of hybrid yeast genomes (Table S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1/FileS2.pdf">File S2</a>).</p><div id="sec-16" class="subsection"><h3>Mutation rates in the hybrid yeast genome</h3><p id="p-41">Mutation accumulation experiments have been previously used in a number of model organisms like <em>S. cerevisiae</em>, <em>Drosophila</em>, and <em>C. elegans</em> to estimate spontaneous mutation rates (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Denver <em>et al.</em> 2000</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2004</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Vassilieva <em>et al.</em> 2000</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wloch <em>et al.</em> 2001</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Joseph and Hall 2004</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Haag-Liautard <em>et al.</em> 2007</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Keightley <em>et al.</em> 2009</a>; <a id="xref-ref-51-10" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>). Mutation accumulation studies in diploid yeast have been carried out in homozygous backgrounds (<a id="xref-ref-51-11" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Zhu <em>et al.</em> 2014</a>). The use of homozygous lines can affect the mutation process in the cells. For example, fixation of recessive mutator alleles during generation of the homozygous lines can enhance mutation rates, while on the other hand genome rearrangements via ectopic recombination are significantly reduced in homozygous backgrounds compared to heterozygous ones (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Montgomery <em>et al.</em> 1991</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Schrider <em>et al.</em> 2013</a>). Given that most organisms have heterozygous genomes, there is a need for accurate estimation of mutation rates in a hybrid context. Our estimate of the average mitotic base mutation rate observed in the S288c/YJM789 hybrid is similar to other homozygous <em>S. cerevisiae</em> strains (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-51-12" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Zhu <em>et al.</em> 2014</a>). These observations suggest that the S288c/YJM789 diploid hybrid is not mutagenic. This is relevant, since the S288c/YJM789 hybrid is used extensively for genome-wide mapping of meiotic recombination events (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Oke <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Krishnaprasad <em>et al.</em> 2015</a>).</p><p id="p-42">Previous studies by <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> (2015)</a> have shown a 3.4-fold increase in base mutation rates in the F2 progeny of heterozygous F1 <em>Arabidopsis</em> plants compared to the homozygous progeny of the selfed plants. A similar 3.4-fold increase in mutation rates was observed in rice heterozygotes over homozygotes (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2015</a>). The increased mutagenesis in heterozygous genomes is due to the cis effects of heterozygosity, since homozygous regions within these genomes do not show higher mutation rates. The enhanced mutagenesis is thought to be due to the potential for new mutations during gene conversions associated with meiotic recombination in heterozygous regions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amos 2010</a>; <a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2015</a>). It may also reflect the enhanced mutagenicity of ssDNA tracts that are generated during meiotic recombination (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yang <em>et al.</em> 2008</a>). In order to determine if the heterozygosity of the S288c/YJM789 hybrid contributes to enhanced mutations in meiosis, we need to compare several meiotic lines of the S288c/YJM789 hybrid with isogenic diploid S288c and YJM789 strains, preferably after a single division to avoid the effects of LOH as well as the loss of mutations during intratetrad mating (<a id="xref-ref-51-13" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>). Reduced mutation rates were observed in the F3 and F4 selfed <em>Arabidopsis</em> plants due to a reduction in heterozygosity (<a id="xref-ref-72-5" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2015</a>). In artificial yeast hybrids, interactions between DNA repair proteins may also play a stronger role in determining the mutation rate. It is possible <em>S. cerevisiae</em> hybrid meiotic lines may have elevated mutation rates compared to their homozygous reference strains, due to incompatibility in mismatch repair genes or heterozygosity (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Heck <em>et al.</em> 2006</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Amos 2010</a>; <a id="xref-ref-72-6" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2015</a>).</p><p id="p-43">DNA synthesis at specific loci undergoing mitotic DSB repair in <em>S. cerevisiae</em> shows more mutations (Stathern <em>et al.</em> 1995). In heterozygous genomes, these effects may be enhanced due to mismatch triggered gene conversions that require additional DNA synthesis similar to the mechanisms proposed for enhanced mutagenesis in heterozygous regions during meiosis (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Amos 2010</a>; <a id="xref-ref-72-7" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2015</a>). All the base mutations were observed in the vegetative lines V3_57, V4_57, and V5_57 that showed significant LOH. But a causal link between mutation rate and frequency of LOH in these lines is hindered by the small sample size and lack of tight correlation between the frequency of LOH and the number of mutations (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>). It is also possible that LOH in V3_57 and V5_57 may have been caused by DSBs induced and repaired to a different extent during abortive meiosis followed by completion of the repair during return to growth. In such a scenario, the results would suggest mutagenicity of the meiotic/mitotic recombination process in a heterozygous genome. Mutation rate estimates in other <em>S. cerevisiae</em> hybrids will be useful to further consolidate these observations on the mutagenicity of the DSB repair process in a heterozygous context.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Eric Alani, Michael Lichten, Lucas Argueso, Tom Petes, and Michael Arnold for discussions and helpful comments on the manuscript. K.T.N. and N.B. were supported by a Wellcome Trust-DBT India Alliance Intermediate fellowship (IA/I/11/2500268). K.T.N. was also supported by Indian Institute of Science Education and Research Thiruvananthapuram intramural funds. The study was financially supported by a European Research Council Advanced Investigator grant (AdG-294542) to L.M.S. A.V.P. and P.C. are supported by a University Grants Commission fellowship. A.D. is supported by a fellowship from the Indian Institute of Science Education and Research Thiruvananthapuram.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-45">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1135/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-46"><em>Communicating editor: D. Gresham</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-07-24"><span class="received-label">Received </span>July 24, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-09-06"><span class="accepted-label">Accepted </span>September 6, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Dutta <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.1" data-doi="10.1002/bies.200900108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Heterozygosity and mutation rate: evidence for an interaction and its implications: the potential for meiotic gene conversions to influence both mutation rate and distribution.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DAmos%26rft.auinit1%253DW.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D82%26rft.epage%253D90%26rft.atitle%253DHeterozygosity%2Band%2Bmutation%2Brate%253A%2Bevidence%2Bfor%2Ban%2Binteraction%2Band%2Bits%2Bimplications%253A%2Bthe%2Bpotential%2Bfor%2Bmeiotic%2Bgene%2Bconversions%2Bto%2Binfluence%2Bboth%2Bmutation%2Brate%2Band%2Bdistribution.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.200900108%26rft_id%253Dinfo%253Apmid%252F19967709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.200900108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19967709&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273506800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.2" data-doi="10.1534/genetics.108.089250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hetrick</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genetic screen for increased loss of heterozygosity in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089250%26rft_id%253Dinfo%253Apmid%252F18562670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzExNzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.3" data-doi="10.1016/j.femsyr.2005.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antunovics</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaillardin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sipiczki</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gradual genome stabilisation by progressive reduction of the <em>Saccharomyces uvarum</em> genome in an interspecific hybrid with <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.femsyr.2005.04.008%26rft_id%253Dinfo%253Apmid%252F15982931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.femsyr.2005.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15982931&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233691700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.4" data-doi="10.1534/genetics.104.032912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemici</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Competing crossover pathways act during meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1805</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DArgueso%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D1805%26rft.epage%253D1816%26rft.atitle%253DCompeting%2BCrossover%2BPathways%2BAct%2BDuring%2BMeiosis%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032912%26rft_id%253Dinfo%253Apmid%252F15611158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzE4MDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.5" data-doi="10.1101/gr.091777.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carazzolle</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duarte</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Netto</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome structure of a <em>Saccharomyces cerevisiae</em> strain widely used in bioethanol production.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2258</span>–<span class="cit-lpage">2270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091777.109%26rft_id%253Dinfo%253Apmid%252F19812109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi8yMjU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.6" data-doi="10.1073/pnas.0605778103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barbera</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Selection and analysis of spontaneous reciprocal mitotic cross-overs in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">12819</span>–<span class="cit-lpage">12824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605778103%26rft_id%253Dinfo%253Apmid%252F16908833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM0LzEyODE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.7" data-doi="10.1038/nrg2896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Towards identifying genes underlying ecologically relevant traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBergelson%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D867%26rft.epage%253D879%26rft.atitle%253DTowards%2Bidentifying%2Bgenes%2Bunderlying%2Becologically%2Brelevant%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2896%26rft_id%253Dinfo%253Apmid%252F21085205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085205&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.8" data-doi="10.1371/journal.pgen.1001287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desany</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riches</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBorneman%26rft.auinit1%253DA.%2BR.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001287%26rft.epage%253De1001287%26rft.atitle%253DWhole-genome%2Bcomparison%2Breveals%2Bnovel%2Bgenetic%2Belements%2Bthat%2Bcharacterize%2Bthe%2Bgenome%2Bof%2Bindustrial%2Bstrains%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001287%26rft_id%253Dinfo%253Apmid%252F21304888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304888&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.9" data-doi="10.1038/nature09352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahrestani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of a long-term evolution experiment with <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBurke%26rft.auinit1%253DM.%2BK.%26rft.volume%253D467%26rft.issue%253D7315%26rft.spage%253D587%26rft.epage%253D590%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Ba%2Blong-term%2Bevolution%2Bexperiment%2Bwith%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09352%26rft_id%253Dinfo%253Apmid%252F20844486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844486&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282273100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.10" data-doi="10.1038/305779a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavenee</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dryja</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benedict</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godbout</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Expression of recessive alleles by chromosomal mechanisms in retinoblastoma.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCavenee%26rft.auinit1%253DW.%2BK.%26rft.volume%253D305%26rft.issue%253D5937%26rft.spage%253D779%26rft.epage%253D784%26rft.atitle%253DExpression%2Bof%2Brecessive%2Balleles%2Bby%2Bchromosomal%2Bmechanisms%2Bin%2Bretinoblastoma.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F305779a0%26rft_id%253Dinfo%253Apmid%252F6633649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/305779a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6633649&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RN21400038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.11" data-doi="10.1093/femsyr/fow071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Understanding how mismatch repair proteins participate in the repair/anti-recombination decision.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">fow071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffow071%26rft_id%253Dinfo%253Apmid%252F27573382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/fow071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27573382&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.12" data-doi="10.1016/j.devcel.2008.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Global analysis of the meiotic crossover landscape.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DChen%26rft.auinit1%253DS.%2BY.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D401%26rft.epage%253D415%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bthe%2Bmeiotic%2Bcrossover%2Blandscape.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2008.07.006%26rft_id%253Dinfo%253Apmid%252F18691940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2008.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691940&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259358100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.13" data-doi="10.1111/j.1365-294X.2011.05005.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fargier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assessing the complex architecture of polygenic traits in diverged yeast populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCubillos%26rft.auinit1%253DF.%2BA.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D1401%26rft.epage%253D1413%26rft.atitle%253DAssessing%2Bthe%2Bcomplex%2Barchitecture%2Bof%2Bpolygenic%2Btraits%2Bin%2Bdiverged%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05005.x%26rft_id%253Dinfo%253Apmid%252F21261765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05005.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21261765&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288705300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.14" data-doi="10.1371/journal.pgen.1002083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dayani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simchen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Meiotic recombination intermediates are resolved with minimal crossover formation during return-to-growth, an analogue of the mitotic cell cycle.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDayani%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1002083%26rft.epage%253De1002083%26rft.atitle%253DMeiotic%2Brecombination%2Bintermediates%2Bare%2Bresolved%2Bwith%2Bminimal%2Bcrossover%2Bformation%2Bduring%2Breturn-to-growth%252C%2Ban%2Banalogue%2Bof%2Bthe%2Bmitotic%2Bcell%2Bcycle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002083%26rft_id%253Dinfo%253Apmid%252F21637791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21637791&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.15" data-doi="10.1126/science.289.5488.2342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vassilieva</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">High direct estimate of the mutation rate in the mitochondrial genome of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">2342</span>–<span class="cit-lpage">2344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDenver%26rft.auinit1%253DD.%2BR.%26rft.volume%253D289%26rft.issue%253D5488%26rft.spage%253D2342%26rft.epage%253D2344%26rft.atitle%253DHigh%2BDirect%2BEstimate%2Bof%2Bthe%2BMutation%2BRate%2Bin%2Bthe%2BMitochondrial%2BGenome%2Bof%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5488.2342%26rft_id%253Dinfo%253Apmid%252F11009418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODkvNTQ4OC8yMzQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.16" data-doi="10.1038/nature02697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">High mutation rate and predominance of insertions in the <em>Caenorhabditis elegans</em> nuclear genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDenver%26rft.auinit1%253DD.%2BR.%26rft.volume%253D430%26rft.issue%253D7000%26rft.spage%253D679%26rft.epage%253D682%26rft.atitle%253DHigh%2Bmutation%2Brate%2Band%2Bpredominance%2Bof%2Binsertions%2Bin%2Bthe%2BCaenorhabditis%2Belegans%2Bnuclear%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02697%26rft_id%253Dinfo%253Apmid%252F15295601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15295601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223085400046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.17" data-doi="10.1073/pnas.242624799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badrane</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characteristic genome rearrangements in experimental evolution of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16144</span>–<span class="cit-lpage">16149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.242624799%26rft_id%253Dinfo%253Apmid%252F12446845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjUvMTYxNDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.18" data-doi="10.1371/journal.pgen.1003366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulish</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanbery</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piotrowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koniges</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recurrent rearrangement during adaptive evolution in an interspecific yeast hybrid suggests a model for rapid introgression.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003366%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003366%26rft_id%253Dinfo%253Apmid%252F23555283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555283&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.19" data-doi="10.1038/35076585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Finding the molecular basis of quantitative traits: successes and pitfalls.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253D437%26rft.epage%253D445%26rft.atitle%253DFinding%2Bthe%2Bmolecular%2Bbasis%2Bof%2Bquantitative%2Btraits%253A%2Bsuccesses%2Band%2Bpitfalls.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35076585%26rft_id%253Dinfo%253Apmid%252F11389460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35076585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11389460&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169144300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.20" data-doi="10.1093/molbev/msw234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flynn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chain</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cristescu</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Spontaneous mutation accumulation in <em>Daphnia pulex</em> in selection-free <em>vs.</em> competitive environments.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw234%26rft_id%253Dinfo%253Apmid%252F27777284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27777284&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Loss-of-heterozygosity facilitates passage through Haldane’s sieve for <em>Saccharomyces cerevisiae</em> undergoing adaptation.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3819%26rft_id%253Dinfo%253Apmid%252F24804896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24804896&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.22" data-doi="10.1126/science.1076374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travisano</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Hybrid speciation in experimental populations of yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">1773</span>–<span class="cit-lpage">1775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreig%26rft.auinit1%253DD.%26rft.volume%253D298%26rft.issue%253D5599%26rft.spage%253D1773%26rft.epage%253D1775%26rft.atitle%253DHybrid%2BSpeciation%2Bin%2BExperimental%2BPopulations%2Bof%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1076374%26rft_id%253Dinfo%253Apmid%252F12459586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTU5OS8xNzczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.23" data-doi="10.1126/science.1123726"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">311</span>: <span class="cit-fpage">1932</span>–<span class="cit-lpage">1936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1123726%26rft_id%253Dinfo%253Apmid%252F16527929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTEvNTc2OS8xOTMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.24" data-doi="10.1038/nature05388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haag-Liautard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorris</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maside</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macaskill</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Direct estimation of per nucleotide and genomic deleterious mutation rates in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHaag-Liautard%26rft.auinit1%253DC.%26rft.volume%253D445%26rft.issue%253D7123%26rft.spage%253D82%26rft.epage%253D85%26rft.atitle%253DDirect%2Bestimation%2Bof%2Bper%2Bnucleotide%2Band%2Bgenomic%2Bdeleterious%2Bmutation%2Brates%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05388%26rft_id%253Dinfo%253Apmid%252F17203060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17203060&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243225100040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.25" data-doi="10.1146/annurev.ecolsys.39.110707.173437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Spontaneous mutation accumulation studies in evolutionary genetics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D40%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansche</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beres</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Gene duplication in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHansche%26rft.auinit1%253DP.%2BE.%26rft.volume%253D88%26rft.issue%253D4%26rft.spage%253D673%26rft.epage%253D687%26rft.atitle%253DGENE%2BDUPLICATION%2BIN%2BSACCHAROMYCES%2BCEREVISIAE%26rft_id%253Dinfo%253Apmid%252F348562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODgvNC82NzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.27" data-doi="10.1073/pnas.0510998103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heck</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemici</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeves</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Negative epistasis between natural variants of the <em>Saccharomyces cerevisiae</em> MLH1 and PMS1 genes results in a defect in mismatch repair.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">3256</span>–<span class="cit-lpage">3261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0510998103%26rft_id%253Dinfo%253Apmid%252F16492773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAzLzkvMzI1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.28" data-doi="10.1534/genetics.104.033761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Spontaneous mutations in diploid <em>Saccharomyces cerevisiae</em>: more beneficial than expected.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJoseph%26rft.auinit1%253DS.%2BB.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D1817%26rft.epage%253D1825%26rft.atitle%253DSpontaneous%2BMutations%2Bin%2BDiploid%2BSaccharomyces%2Bcerevisiae%253A%2BMore%2BBeneficial%2BThan%2BExpected%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.033761%26rft_id%253Dinfo%253Apmid%252F15611159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzE4MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.29" data-doi="10.1101/gr.091231.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trivedi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Analysis of the genome sequences of three <em>Drosophila melanogaster</em> spontaneous mutation accumulation lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1195</span>–<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091231.109%26rft_id%253Dinfo%253Apmid%252F19439516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTE5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.30" data-doi="10.1073/pnas.1213449109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shewmaker</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kryndushkin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickner</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sex, prions, and plasmids in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E2683</span>–<span class="cit-lpage">E2690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213449109%26rft_id%253Dinfo%253Apmid%252F22949655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQwL0UyNjgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.31" data-doi="10.1002/bies.20435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolution of the hemiascomycete yeasts: on life styles and the importance of inbreeding.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DKnop%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D696%26rft.epage%253D708%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bhemiascomycete%2Byeasts%253A%2Bon%2Blife%2Bstyles%2Band%2Bthe%2Bimportance%2Bof%2Binbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20435%26rft_id%253Dinfo%253Apmid%252F16929561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16929561&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239322900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.32" data-doi="10.1534/genetics.114.172320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnaprasad</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anand</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekkedil</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Variation in crossover frequencies perturb crossover assurance without affecting meiotic chromosome segregation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172320%26rft_id%253Dinfo%253Apmid%252F25467183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzIvMzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.33" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.34" data-doi="10.1371/journal.pgen.1005781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laureau</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeillet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legoix-Ne</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extensive recombination of a yeast diploid hybrid through meiotic reversion.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005781%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005781%26rft_id%253Dinfo%253Apmid%252F26828862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26828862&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.35" data-doi="10.1073/pnas.1210309109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popodi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rate and molecular spectrum of spontaneous mutations in the bacterium <em>Escherichia coli</em> as determined by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E2774</span>–<span class="cit-lpage">E2783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1210309109%26rft_id%253Dinfo%253Apmid%252F22991466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQxL0UyNzc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.36" data-doi="10.1371/journal.pgen.1000410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenwell</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gawel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A fine-structure map of spontaneous mitotic crossovers in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DP.%2BS.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000410%26rft.epage%253De1000410%26rft.atitle%253DA%2Bfine-structure%2Bmap%2Bof%2Bspontaneous%2Bmitotic%2Bcrossovers%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000410%26rft_id%253Dinfo%253Apmid%252F19282969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19282969&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Averette</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desnos-Ollivier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dromer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic diversity and genomic plasticity of <em>Cryptococcus neoformans</em> AD hybrid strains.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D2%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.38" data-doi="10.1534/genetics.106.062166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Sequence diversity, reproductive isolation and species concepts in <em>Saccharomyces</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.062166%26rft_id%253Dinfo%253Apmid%252F16951060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc0LzIvODM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.39" data-doi="10.1371/journal.pgen.1004005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaines</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callender</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busygina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Down-regulation of Rad51 activity during meiosis in yeast prevents competition with Dmc1 for repair of double-strand breaks.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004005%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004005%26rft_id%253Dinfo%253Apmid%252F24465215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24465215&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.40" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.41" data-doi="10.1007/978-94-007-7347-9_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Revisiting Mortimer’s genome renewal hypothesis: heterozygosity, homothallism, and the potential for adaptation in yeast.</span> <abbr class="cit-jnl-abbrev">Adv. Exp. Med. Biol.</abbr> <span class="cit-vol">781</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BExp.%2BMed.%2BBiol.%26rft.volume%253D781%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-94-007-7347-9_3%26rft_id%253Dinfo%253Apmid%252F24277294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-94-007-7347-9_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24277294&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.42" data-doi="10.1073/pnas.1012544108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayıkçı</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reininga</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholl</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Outcrossing, mitotic recombination, and life-history trade-offs shape genome evolution in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1987</span>–<span class="cit-lpage">1992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012544108%26rft_id%253Dinfo%253Apmid%252F21245305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzUvMTk4NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.43" data-doi="10.1038/nature07135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourgon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of meiotic crossovers and non-crossovers in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMancera%26rft.auinit1%253DE.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bnon-crossovers%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07135%26rft_id%253Dinfo%253Apmid%252F18615017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18615017&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.44" data-doi="10.1098/rspb.2007.0056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mandegar</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mitotic recombination counteracts the benefits of genetic segregation.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DMandegar%26rft.auinit1%253DM.%2BA%26rft.volume%253D274%26rft.issue%253D1615%26rft.spage%253D1301%26rft.epage%253D1307%26rft.atitle%253DMitotic%2Brecombination%2Bcounteracts%2Bthe%2Bbenefits%2Bof%2Bgenetic%2Bsegregation%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.0056%26rft_id%253Dinfo%253Apmid%252F17360283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjc0LzE2MTUvMTMwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.45" data-doi="10.1371/journal.pgen.1002305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnault</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide analysis of heteroduplex DNA in mismatch repair-deficient yeast cells reveals novel properties of meiotic recombination pathways.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMartini%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002305%26rft.epage%253De1002305%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bheteroduplex%2BDNA%2Bin%2Bmismatch%2Brepair-deficient%2Byeast%2Bcells%2Breveals%2Bnovel%2Bproperties%2Bof%2Bmeiotic%2Brecombination%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002305%26rft_id%253Dinfo%253Apmid%252F21980306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21980306&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clemons</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic characterization of pathogenic <em>Saccharomyces cerevisiae</em> isolates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcCusker%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1261%26rft.epage%253D1269%26rft.atitle%253DGenetic%2Bcharacterization%2Bof%2Bpathogenic%2BSaccharomyces%2Bcerevisiae%2Bisolates.%26rft_id%253Dinfo%253Apmid%252F8013903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzEyNjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Judd</span>  <span class="cit-name-given-names">B. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Chromosome rearrangement by ectopic recombination in <em>Drosophila melanogaster</em>: genome structure and evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1085</span>–<span class="cit-lpage">1098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMontgomery%26rft.auinit1%253DE.%2BA.%26rft.volume%253D129%26rft.issue%253D4%26rft.spage%253D1085%26rft.epage%253D1098%26rft.atitle%253DChromosome%2Brearrangement%2Bby%2Bectopic%2Brecombination%2Bin%2BDrosophila%2Bmelanogaster%253A%2Bgenome%2Bstructure%2Band%2Bevolution.%26rft_id%253Dinfo%253Apmid%252F1783293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEyOS80LzEwODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.48" data-doi="10.1128/MMBR.00022-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morales</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary role of interspecies hybridization and genetic exchanges in yeasts.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DMorales%26rft.auinit1%253DL.%26rft.volume%253D76%26rft.issue%253D4%26rft.spage%253D721%26rft.epage%253D739%26rft.atitle%253DEvolutionary%2BRole%2Bof%2BInterspecies%2BHybridization%2Band%2BGenetic%2BExchanges%2Bin%2BYeasts%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00022-12%26rft_id%253Dinfo%253Apmid%252F23204364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Ni80LzcyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.49" data-doi="10.1002/yea.320101203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortimer</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzzi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polsinelli</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genome renewal: a new phenomenon revealed from a genetic study of 43 strains of <em>Saccharomyces cerevisiae</em> derived from natural fermentation of grape musts.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1543</span>–<span class="cit-lpage">1552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DMortimer%26rft.auinit1%253DR.%2BK.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D1543%26rft.epage%253D1552%26rft.atitle%253DGenome%2Brenewal%253A%2Ba%2Bnew%2Bphenomenon%2Brevealed%2Bfrom%2Ba%2Bgenetic%2Bstudy%2Bof%2B43%2Bstrains%2Bof%2BSaccharomyces%2Bcerevisiae%2Bderived%2Bfrom%2Bnatural%2Bfermentation%2Bof%2Bgrape%2Bmusts.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320101203%26rft_id%253Dinfo%253Apmid%252F7725789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320101203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7725789&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QF61300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.50" data-doi="10.1002/bies.200900017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic mutation rates: what high-throughput methods can tell us.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">912</span>–<span class="cit-lpage">920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DNishant%26rft.auinit1%253DK.%2BT.%26rft.volume%253D31%26rft.issue%253D9%26rft.spage%253D912%26rft.epage%253D920%26rft.atitle%253DGenomic%2Bmutation%2Brates%253A%2Bwhat%2Bhigh-throughput%2Bmethods%2Bcan%2Btell%2Bus.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.200900017%26rft_id%253Dinfo%253Apmid%252F19644920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.200900017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19644920&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.51" data-doi="10.1371/journal.pgen.1001109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001109%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001109%26rft_id%253Dinfo%253Apmid%252F20838597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838597&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.52" data-doi="10.1371/journal.pgen.1004690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Controlling meiotic recombinational repair – specifying the roles of ZMMs, Sgs1 and Mus81/Mms4 in crossover formation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004690%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004690%26rft_id%253Dinfo%253Apmid%252F25329811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25329811&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.53" data-doi="10.1093/nar/gkg219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schildkraut</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazarin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickoloff</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Gene conversion tracts in <em>Saccharomyces cerevisiae</em> can be extremely short and highly directional.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1164</span>–<span class="cit-lpage">1173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPalmer%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D1164%26rft.epage%253D1173%26rft.atitle%253DGene%2Bconversion%2Btracts%2Bin%2BSaccharomyces%2Bcerevisiae%2Bcan%2Bbe%2Bextremely%2Bshort%2Band%2Bhighly%2Bdirectional%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg219%26rft_id%253Dinfo%253Apmid%252F12582235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582235&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181115100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paques</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple pathways of recombination induced by double-strand breaks in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DPaques%26rft.auinit1%253DF.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D349%26rft.epage%253D404%26rft.atitle%253DMultiple%2BPathways%2Bof%2BRecombination%2BInduced%2Bby%2BDouble-Strand%2BBreaks%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F10357855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2My8yLzM0OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.55" data-doi="10.1093/gbe/evu082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryszcz</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemeth</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gacser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome comparison of <em>Candida orthopsilosis</em> clinical strains reveals the existence of hybrids between two distinct subspecies.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu082%26rft_id%253Dinfo%253Apmid%252F24747362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24747362&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.56" data-doi="10.1186/1471-2164-10-475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijeratne</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomsho</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterization of meiotic crossovers and gene conversion by whole-genome sequencing in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQi%26rft.auinit1%253DJ.%26rft.volume%253D10%26rft.spage%253D475%26rft.epage%253D475%26rft.atitle%253DCharacterization%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bgene%2Bconversion%2Bby%2Bwhole-genome%2Bsequencing%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-475%26rft_id%253Dinfo%253Apmid%252F19832984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-475&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19832984&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.57" data-doi="10.1111/j.1574-6968.2008.01480.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The complex and dynamic genomes of industrial yeasts.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2008.01480.x%26rft_id%253Dinfo%253Apmid%252F19175410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2008.01480.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19175410&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3669.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Methods in Yeast Genetics: A Laboratory Course Manual</em></span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.59" data-doi="10.1371/journal.pgen.1003817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Younkin</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casper</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fragile site instability in <em>Saccharomyces cerevisiae</em> causes loss of heterozygosity by mitotic crossovers and break-induced replication.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003817%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003817%26rft_id%253Dinfo%253Apmid%252F24068975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24068975&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.60" data-doi="10.1038/ng1859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population genomic analysis of outcrossing and recombination in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRuderfer%26rft.auinit1%253DD.%2BM.%26rft.volume%253D38%26rft.issue%253D9%26rft.spage%253D1077%26rft.epage%253D1081%26rft.atitle%253DPopulation%2Bgenomic%2Banalysis%2Bof%2Boutcrossing%2Band%2Brecombination%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1859%26rft_id%253Dinfo%253Apmid%252F16892060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16892060&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240112100028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.61" data-doi="10.1534/genetics.113.151670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rates and genomic consequences of spontaneous mutational events in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151670%26rft_id%253Dinfo%253Apmid%252F23733788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.62" data-doi="10.1128/EC.00060-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic plasticity of the human fungal pathogen <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">1008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00060-10%26rft_id%253Dinfo%253Apmid%252F20495058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI5LzcvOTkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.63" data-doi="10.1371/journal.pgen.1003434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St Charles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of spontaneous mitotic recombination hotspots on the 1.1 Mb arm of yeast chromosome IV.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003434%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003434%26rft_id%253Dinfo%253Apmid%252F23593029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23593029&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.64" data-doi="10.1038/416326a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegelman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oefner</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dissecting the architecture of a quantitative trait locus in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteinmetz%26rft.auinit1%253DL.%2BM.%26rft.volume%253D416%26rft.issue%253D6878%26rft.spage%253D326%26rft.epage%253D330%26rft.atitle%253DDissecting%2Bthe%2Barchitecture%2Bof%2Ba%2Bquantitative%2Btrait%2Blocus%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416326a%26rft_id%253Dinfo%253Apmid%252F11907579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416326a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11907579&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174482200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.65" data-doi="10.1111/jeb.12494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stelkens</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brockhurst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effect of hybrid transgression on environmental tolerance in experimental yeast crosses.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2507</span>–<span class="cit-lpage">2519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D27%26rft.spage%253D2507%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12494%26rft_id%253Dinfo%253Apmid%252F25262771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25262771&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGill</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">DNA synthesis errors associated with double-strand-break repair.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStrathern%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D140%26rft.issue%253D3%26rft.spage%253D965%26rft.epage%253D972%26rft.atitle%253DDNA%2Bsynthesis%2Berrors%2Bassociated%2Bwith%2Bdouble-strand-break%2Brepair.%26rft_id%253Dinfo%253Apmid%252F7672595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzMvOTY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okashah</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transmission ratio distortion results in asymmetric introgression in Louisiana Iris.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DTang%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DTransmission%2Bratio%2Bdistortion%2Bresults%2Bin%2Basymmetric%2Bintrogression%2Bin%2BLouisiana%2BIris.%26rft_id%253Dinfo%253Apmid%252F20298609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20298609&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.68" data-doi="10.1554/0014-3820(2000)054[1234:TFEOSM]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vassilieva</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hook</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The fitness effects of spontaneous mutations in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1234</span>–<span class="cit-lpage">1246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DVassilieva%26rft.auinit1%253DL.%2BL.%26rft.volume%253D54%26rft.issue%253D4%26rft.spage%253D1234%26rft.epage%253D1246%26rft.atitle%253DThe%2Bfitness%2Beffects%2Bof%2Bspontaneous%2Bmutations%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282000%2529054%255B1234%253ATFEOSM%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11005291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2000)054[1234:TFEOSM]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11005291&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089317900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.69" data-doi="10.1073/pnas.0701291104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome sequencing and comparative analysis of <em>Saccharomyces cerevisiae</em> strain YJM789.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">12825</span>–<span class="cit-lpage">12830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701291104%26rft_id%253Dinfo%253Apmid%252F17652520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzMxLzEyODI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wloch</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szafraniec</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korona</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Direct estimate of the mutation rate and the distribution of fitness effects in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWloch%26rft.auinit1%253DD.%2BM.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D441%26rft.epage%253D452%26rft.atitle%253DDirect%2BEstimate%2Bof%2Bthe%2BMutation%2BRate%2Band%2Bthe%2BDistribution%2Bof%2BFitness%2BEffects%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11606524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNDQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.71" data-doi="10.1371/journal.pbio.1002221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Origin of the yeast whole-genome duplication.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1002221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002221%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002221%26rft_id%253Dinfo%253Apmid%252F26252643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26252643&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.72" data-doi="10.1038/nature14649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Parent-progeny sequencing indicates higher mutation rates in heterozygotes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">523</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14649%26rft_id%253Dinfo%253Apmid%252F26176923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26176923&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.73" data-doi="10.1371/journal.pgen.1000264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storici</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordenin</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Hypermutability of damaged single-strand DNA formed at double-strand breaks and uncapped telomeres in yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De1000264%26rft.epage%253De1000264%26rft.atitle%253DHypermutability%2Bof%2Bdamaged%2Bsingle-strand%2BDNA%2Bformed%2Bat%2Bdouble-strand%2Bbreaks%2Band%2Buncapped%2Btelomeres%2Bin%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000264%26rft_id%253Dinfo%253Apmid%252F19023402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19023402&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.74" data-doi="10.1534/genetics.114.167395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yim</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charles</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution mapping of two types of spontaneous mitotic gene conversion events in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167395%26rft_id%253Dinfo%253Apmid%252F24990991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.75" data-doi="10.1371/journal.pgen.1003894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide high-resolution mapping of UV-induced mitotic recombination events in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003894%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003894%26rft_id%253Dinfo%253Apmid%252F24204306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003894&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204306&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3669.76" data-doi="10.1073/pnas.1323011111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Precise estimates of mutation rate and spectrum in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E2310</span>–<span class="cit-lpage">E2318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323011111%26rft_id%253Dinfo%253Apmid%252F24847077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzIyL0UyMzEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3669.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3659" title="The Effect of Common Inversion Polymorphisms In(2L)t and In(3R)Mo on Patterns of Transcriptional Variation in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3681" title="Differential Expression of miRNAs in the Respiratory Tree of the Sea Cucumber Apostichopus japonicus Under Hypoxia Stress" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-36iEdxpdhQkV7YZPGpChc2exBFw2q2MLpHi_I7SSnU4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3669" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3669" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3669" class="active" data-icon-position="" data-hide-link-title="0">Genome Dynamics of Hybrid Saccharomyces cerevisiae During Vegetative and Meiotic Divisions</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487109" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TiZ7JhZVOJZ2Tw8fUhmupCUcyqh8YrZ9eo4VA63OcAE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3669.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3669" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-n-iI-rwb8VJ6sKYfpqzl3FJTm1qBKk2Luj3ihxhhhuo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3669" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3669.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487109" id="node487109--2" data-pisa="ggg;7/11/3669" data-pisa-master="ggg;g3.117.1135" data-apath="/ggg/7/11/3669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Dynamics of Hybrid <em>Saccharomyces cerevisiae</em> During Vegetative and Meiotic Divisions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Abhishek Dutta</span>, <span class="highwire-citation-author" data-delta="1">Gen Lin</span>, <span class="highwire-citation-author" data-delta="2">Ajith V. Pankajam</span>, <span class="highwire-citation-author" data-delta="3">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="4">Nahush Bhat</span>, <span class="highwire-citation-author" data-delta="5">Lars M. Steinmetz</span> and <span class="highwire-citation-author" data-delta="6">Koodali T. Nishant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3669-3679;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1135 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487109--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Abhishek Dutta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BDutta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dutta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhishek%2BDutta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gen Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gen%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGen%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nahush Bhat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nahush%2BBhat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhat%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANahush%2BBhat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford University, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Stanford Genome Technology Center, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computation, Modelling, and Simulation, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nishantkt@iisertvm.ac.in" class="" data-icon-position="" data-hide-link-title="0">nishantkt@iisertvm.ac.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3669" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3669&amp;title=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3669&amp;title=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3669&amp;title=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3669&amp;text=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3669&amp;title=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3669&amp;t=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3669&amp;title=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3669&amp;title=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487109/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487109" id="node487109--3" data-pisa="ggg;7/11/3669" data-pisa-master="ggg;g3.117.1135" data-apath="/ggg/7/11/3669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Dynamics of Hybrid <em>Saccharomyces cerevisiae</em> During Vegetative and Meiotic Divisions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Abhishek Dutta</span>, <span class="highwire-citation-author" data-delta="1">Gen Lin</span>, <span class="highwire-citation-author" data-delta="2">Ajith V. Pankajam</span>, <span class="highwire-citation-author" data-delta="3">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="4">Nahush Bhat</span>, <span class="highwire-citation-author" data-delta="5">Lars M. Steinmetz</span> and <span class="highwire-citation-author" data-delta="6">Koodali T. Nishant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3669-3679;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1135 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487109--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Abhishek Dutta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BDutta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dutta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhishek%2BDutta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gen Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gen%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGen%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nahush Bhat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nahush%2BBhat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhat%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANahush%2BBhat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Stanford University, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Stanford Genome Technology Center, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computation, Modelling, and Simulation, Indian Institute of Science Education and Research Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nishantkt@iisertvm.ac.in" class="" data-icon-position="" data-hide-link-title="0">nishantkt@iisertvm.ac.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3669&amp;rft.epage=3679&amp;rft.atitle=Genome%20Dynamics%20of%20Hybrid%20Saccharomyces%20cerevisiae%20During%20Vegetative%20and%20Meiotic%20Divisions&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dutta%2C+Abhishek&amp;rft.au=Lin%2C+Gen&amp;rft.au=Pankajam%2C+Ajith+V.&amp;rft.au=Chakraborty%2C+Parijat&amp;rft.au=Bhat%2C+Nahush&amp;rft.au=Steinmetz%2C+Lars+M.&amp;rft.au=Nishant%2C+Koodali+T."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487109/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487109/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487109/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487109/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487109/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487109/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487109/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487109/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487109/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487109/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487109/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487109/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3669" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3669.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3669.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3669.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3669.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
