<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3571" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3571.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3571.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3571.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/11/3571/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3571.atom" />
<meta name="HW.pisa" content="ggg;7/11/3571" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations" />
<meta name="DC.Identifier" content="10.1534/g3.117.300076" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Schopp et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A major application of genomic prediction (GP) in plant breeding is the identification of superior inbred lines within families derived from biparental crosses. When models for various traits were trained within related or unrelated biparental families (BPFs), experimental studies found substantial variation in prediction accuracy (PA), but little is known about the underlying factors. We used SNP marker genotypes of inbred lines from either elite germplasm or landraces of maize ( Zea mays L.) as parents to generate in silico 300 BPFs of doubled-haploid lines. We analyzed PA within each BPF for 50 simulated polygenic traits, using genomic best linear unbiased prediction (GBLUP) models trained with individuals from either full-sib (FSF), half-sib (HSF), or unrelated families (URF) for various sizes (![Formula][1]&lt;/img&gt;) of the training set and different heritabilities (![Formula][2]&lt;/img&gt; In addition, we modified two deterministic equations for forecasting PA to account for inbreeding and genetic variance unexplained by the training set. Averaged across traits, PA was high within FSF (0.41–0.97) with large variation only for ![Formula][3]&lt;/img&gt; and ![Formula][4]&lt;/img&gt; ![Formula][5]&lt;/img&gt; For HSF and URF, PA was on average ∼40–60% lower and varied substantially among different combinations of BPFs used for model training and prediction as well as different traits. As exemplified by HSF results, PA of across-family GP can be very low if causal variants not segregating in the training set account for a sizeable proportion of the genetic variance among predicted individuals. Deterministic equations accurately forecast the PA expected over many traits, yet cannot capture trait-specific deviations. We conclude that model training within BPFs generally yields stable PA, whereas a high level of uncertainty is encountered in across-family GP. Our study shows the extent of variation in PA that must be at least reckoned with in practice and offers a starting point for the design of training sets composed of multiple BPFs.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif" />
<meta name="DC.Contributor" content="Pascal Schopp" />
<meta name="DC.Contributor" content="Dominik Müller" />
<meta name="DC.Contributor" content="Yvonne C. J. Wientjes" />
<meta name="DC.Contributor" content="Albrecht E. Melchinger" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A major application of genomic prediction (GP) in plant breeding is the identification of superior inbred lines within families derived from biparental crosses. When models for various traits were trained within related or unrelated biparental families (BPFs), experimental studies found substantial variation in prediction accuracy (PA), but little is known about the underlying factors. We used SNP marker genotypes of inbred lines from either elite germplasm or landraces of maize (&lt;i&gt;Zea&lt;/i&gt;&lt;i&gt;mays&lt;/i&gt; L.) as parents to generate &lt;i&gt;in silico&lt;/i&gt; 300 BPFs of doubled-haploid lines. We analyzed PA within each BPF for 50 simulated polygenic traits, using genomic best linear unbiased prediction (GBLUP) models trained with individuals from either full-sib (FSF), half-sib (HSF), or unrelated families (URF) for various sizes (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;t&lt;/mi&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;a&lt;/mi&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;n&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt;) of the training set and different heritabilities (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msup&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt; In addition, we modified two deterministic equations for forecasting PA to account for inbreeding and genetic variance unexplained by the training set. Averaged across traits, PA was high within FSF (0.41–0.97) with large variation only for &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me3&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;t&lt;/mi&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;a&lt;/mi&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;n&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo&gt;&amp;lt;&lt;/mo&gt;&lt;mn&gt;50&lt;/mn&gt;&lt;/mrow&gt;&lt;/math&gt; and &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me4&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msup&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/math&gt;&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me5&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mo&gt;&amp;lt;&lt;/mo&gt;&lt;mn&gt;0.6.&lt;/mn&gt;&lt;/mrow&gt;&lt;/math&gt; For HSF and URF, PA was on average ∼40–60% lower and varied substantially among different combinations of BPFs used for model training and prediction as well as different traits. As exemplified by HSF results, PA of across-family GP can be very low if causal variants not segregating in the training set account for a sizeable proportion of the genetic variance among predicted individuals. Deterministic equations accurately forecast the PA expected over many traits, yet cannot capture trait-specific deviations. We conclude that model training within BPFs generally yields stable PA, whereas a high level of uncertainty is encountered in across-family GP. Our study shows the extent of variation in PA that must be at least reckoned with in practice and offers a starting point for the design of training sets composed of multiple BPFs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3571" />
<meta name="citation_id" content="7/11/3571" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3571" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3571.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3571.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3571.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300076" />
<meta name="citation_pmid" content="28916649" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3571" />
<meta name="citation_lastpage" content="3586" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Pascal Schopp" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author" content="Dominik Müller" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author" content="Yvonne C. J. Wientjes" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University and Research, 6700 AH, The Netherlands" />
<meta name="citation_author" content="Albrecht E. Melchinger" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author_email" content="melchinger@uni-hohenheim.de" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J. I. Sanchez;citation_author=J.-L. Jannink;citation_title=Optimization of genomic selection training populations with a genetic algorithm.;citation_pages=38;citation_volume=47;citation_year=2015;citation_pmid=25943105;citation_doi=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=W. Astle;citation_author=D. Balding;citation_title=Population structure and cryptic relatedness in genetic association studies.;citation_pages=451-471;citation_volume=24;citation_year=2009;citation_doi=10.1214/09-STS307" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Mächler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting linear mixed-effects models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=D. Bustos-Korts;citation_author=M. Malosetti;citation_author=S. Chapman;citation_author=B. Biddulph;citation_author=F. van Eeuwijk;citation_title=Improvement of predictive ability by uniform coverage of the target genetic space.;citation_pages=3733-3747;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=L. Chen;citation_author=F. Schenkel;citation_author=M. Vinsky;citation_author=D. H. C. Jr;citation_author=C. Li;citation_title=Accuracy of predicting genomic breeding values for residual feed intake in angus and charolais beef cattle.;citation_pages=4669-4678;citation_volume=91;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. A. Clark;citation_author=J. M. Hickey;citation_author=H. D. Daetwyler;citation_author=J. H. J. van der Werf;citation_title=The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes.;citation_pages=4-4;citation_volume=44;citation_year=2012;citation_pmid=22321529;citation_doi=10.1186/1297-9686-44-4" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Accuracy of predicting the genetic risk of disease using a genome-wide approach.;citation_pages=e3395-e3395;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852893;citation_doi=10.1371/journal.pone.0003395" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_author=R. Pong-Wong;citation_author=G. de Los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics.;citation_author=D. F. Falconer;citation_author=T. S. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Giraud;citation_author=C. Lehermeier;citation_author=E. Bauer;citation_author=M. Falque;citation_author=V. Segura;citation_title=Linkage disequilibrium with linkage analysis of multiline crosses reveals different multiallelic QTL for hybrid performance in the flint and dent heterotic groups of maize.;citation_volume=198;citation_year=2014;citation_pmid=25271305;citation_doi=10.1534/genetics.114.169367" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_author=T. H. E. Meuwissen;citation_title=Using the genomic relationship matrix to predict the accuracy of genomic selection.;citation_pages=409-421;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059574;citation_doi=10.1111/j.1439-0388.2011.00964.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Habier;citation_author=J. Tetens;citation_author=F. Seefried;citation_author=P. Lichtner;citation_author=G. Thaller;citation_title=The impact of genetic relationship information on genomic breeding values in German Holstein cattle.;citation_pages=5-5;citation_volume=42;citation_year=2010;citation_pmid=20170500;citation_doi=10.1186/1297-9686-42-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. He;citation_author=A. W. Schulthess;citation_author=V. Mirdita;citation_author=Y. Zhao;citation_author=V. Korzun;citation_title=Genomic selection in a commercial winter wheat population.;citation_pages=641-651;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=S. Dreisigacker;citation_author=J. Crossa;citation_author=S. Hearne;citation_author=R. Babu;citation_title=Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.;citation_pages=1476-1488;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0195" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Jacobson;citation_author=L. Lian;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=General combining ability model for genomewide selection in a biparental cross.;citation_pages=895-905;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.11.0774" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Karoui;citation_author=M. J. Carabaño;citation_author=C. Díaz;citation_author=A. Legarra;citation_title=Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.;citation_pages=39;citation_volume=44;citation_year=2012;citation_pmid=23216664;citation_doi=10.1186/1297-9686-44-39" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. E. Kemper;citation_author=C. M. Reich;citation_author=P. J. Bowman;citation_author=C. J. Vander Jagt;citation_author=A. J. Chamberlain;citation_title=Improved precision of QTL mapping using a nonlinear Bayesian method in a multi-breed population leads to greater accuracy of across-breed genomic predictions.;citation_pages=29;citation_volume=47;citation_year=2015;citation_pmid=25887988;citation_doi=10.1186/s12711-014-0074-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krämer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lehermeier;citation_author=C.-C. Schön;citation_author=G. de los Campos;citation_title=Assessment of genetic heterogeneity in structured plant populations using multivariate whole-genome regression models.;citation_volume=201;citation_year=2015;citation_pmid=26122758;citation_doi=10.1534/genetics.115.177394" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=L. Lian;citation_author=A. Jacobson;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=Genomewide prediction accuracy within 969 maize biparental populations.;citation_pages=1514-1522;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.12.0856" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=Z. Lin;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Genomic selection in crops, trees and forages: a review.;citation_pages=1177-1191;citation_volume=65;citation_year=2014;citation_doi=10.1071/CP13363" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=A. J. Lorenz;citation_title=Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment.;citation_pages=481-491;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_title=Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley.;citation_pages=2657-2667;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.12.0827" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=J. J. Marulanda;citation_author=A. E. Melchinger;citation_author=T. Würschum;citation_title=Genomic selection in biparental populations: assessment of parameters for optimum estimation set design.;citation_pages=623-630;citation_volume=134;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. K. Matukumalli;citation_author=C. T. Lawley;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=M. F. Allan;citation_title=Development and characterization of a high density SNP genotyping assay for cattle.;citation_pages=e5350-e5350;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19390634;citation_doi=10.1371/journal.pone.0005350" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=A. E. Melchinger;citation_title=Expectation of means and variances of testcrosses produced from F2 and backcross individuals and their selfed progenies.;citation_pages=105-115;citation_volume=59;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. E. Melchinger;citation_author=P. Schopp;citation_author=D. Müller;citation_author=T. A. Schrag;citation_author=E. Bauer;citation_title=Safeguarding our genetic resources with libraries of doubled-haploid lines.;citation_volume=206;citation_year=2017;citation_pmid=28468909;citation_doi=10.1534/genetics.115.186205" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Meuwissen;citation_author=M. Goddard;citation_title=Accurate prediction of genetic values for complex traits by whole-genome resequencing.;citation_volume=185;citation_year=2010;citation_pmid=20308278;citation_doi=10.1534/genetics.110.116590" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Mohammadi;citation_author=T. Tiede;citation_author=K. P. Smith;citation_title=Popvar: a genome-wide procedure for predicting genetic variance and correlated response in biparental breeding populations.;citation_pages=2068-2077;citation_volume=55;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Introduction to the Theory of Statistics.;citation_author=A. Mood;citation_year=1974" />
<meta name="citation_reference" content="citation_title=Linear Models for the Prediction of Animal Breeding Values.;citation_author=R. A. Mrode;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Pérez-Enciso;citation_author=J. C. Rincón;citation_author=A. Legarra;citation_title=Sequence- vs. chip-assisted genomic selection: accurate biological information is advised.;citation_pages=43;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Riedelsheimer;citation_author=A. E. Melchinger;citation_title=Optimizing the allocation of resources for genomic selection in one breeding cycle.;citation_pages=2835-2848;citation_volume=126;citation_year=2013;citation_pmid=23982591;citation_doi=10.1007/s00122-013-2175-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Riedelsheimer;citation_author=J. B. Endelman;citation_author=M. Stange;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic predictability of interconnected biparental maize populations.;citation_volume=194;citation_year=2013;citation_pmid=23535384;citation_doi=10.1534/genetics.113.150227" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloë;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Schopp;citation_author=D. Müller;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=The Sustained Impact of Model Organisms--in Genetics and Epigenetics;citation_pages=1-4;citation_volume=205;citation_year=2017;citation_issue=1;citation_pmid=28049700;citation_doi=10.1534/genetics.116.187864" />
<meta name="citation_reference" content="citation_title=Numerical Taxonomy: The Principles and Practice of Numerical Classification.;citation_author=P. Sneath;citation_author=R. Sokal;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=I. Strandén;citation_author=O. F. Christensen;citation_title=Allele coding in genomic evaluation.;citation_pages=25-25;citation_volume=43;citation_year=2011;citation_pmid=21703021;citation_doi=10.1186/1297-9686-43-25" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. C. J. Wientjes;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_title=The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.;citation_volume=193;citation_year=2013;citation_pmid=23267052;citation_doi=10.1534/genetics.112.146290" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Y. C. J. Wientjes;citation_author=R. F. Veerkamp;citation_author=P. Bijma;citation_author=H. Bovenhuis;citation_author=C. Schrooten;citation_title=Empirical and deterministic accuracies of across-population genomic prediction.;citation_pages=5;citation_volume=47;citation_year=2015;citation_pmid=25885467;citation_doi=10.1186/s12711-014-0086-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. C. J. Wientjes;citation_author=P. Bijma;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_title=An equation to predict the accuracy of genomic values by combining data from multiple traits, populations, or environments.;citation_volume=202;citation_year=2016;citation_pmid=26637542;citation_doi=10.1534/genetics.115.183269" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="A major application of genomic prediction (GP) in plant breeding is the identification of superior inbred lines within families derived from biparental crosses. When models for various traits were trained within related or unrelated biparental families (BPFs), experimental studies found substantial variation in prediction accuracy (PA), but little is known about the underlying factors. We used SNP marker genotypes of inbred lines from either elite germplasm or landraces of maize ( Zea mays L.) as parents to generate in silico 300 BPFs of doubled-haploid lines. We analyzed PA within each BPF for 50 simulated polygenic traits, using genomic best linear unbiased prediction (GBLUP) models trained with individuals from either full-sib (FSF), half-sib (HSF), or unrelated families (URF) for various sizes (![Formula][1]&lt;/img&gt;) of the training set and different heritabilities (![Formula][2]&lt;/img&gt; In addition, we modified two deterministic equations for forecasting PA to account for inbreeding and genetic variance unexplained by the training set. Averaged across traits, PA was high within FSF (0.41–0.97) with large variation only for ![Formula][3]&lt;/img&gt; and ![Formula][4]&lt;/img&gt; ![Formula][5]&lt;/img&gt; For HSF and URF, PA was on average ∼40–60% lower and varied substantially among different combinations of BPFs used for model training and prediction as well as different traits. As exemplified by HSF results, PA of across-family GP can be very low if causal variants not segregating in the training set account for a sizeable proportion of the genetic variance among predicted individuals. Deterministic equations accurately forecast the PA expected over many traits, yet cannot capture trait-specific deviations. We conclude that model training within BPFs generally yields stable PA, whereas a high level of uncertainty is encountered in across-family GP. Our study shows the extent of variation in PA that must be at least reckoned with in practice and offers a starting point for the design of training sets composed of multiple BPFs.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif" />
<meta name="og-title" property="og:title" content="Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3571" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A major application of genomic prediction (GP) in plant breeding is the identification of superior inbred lines within families derived from biparental crosses. When models for various traits were trained within related or unrelated biparental families (BPFs), experimental studies found substantial variation in prediction accuracy (PA), but little is known about the underlying factors. We used SNP marker genotypes of inbred lines from either elite germplasm or landraces of maize ( Zea mays L.) as parents to generate in silico 300 BPFs of doubled-haploid lines. We analyzed PA within each BPF for 50 simulated polygenic traits, using genomic best linear unbiased prediction (GBLUP) models trained with individuals from either full-sib (FSF), half-sib (HSF), or unrelated families (URF) for various sizes (![Formula][1]&lt;/img&gt;) of the training set and different heritabilities (![Formula][2]&lt;/img&gt; In addition, we modified two deterministic equations for forecasting PA to account for inbreeding and genetic variance unexplained by the training set. Averaged across traits, PA was high within FSF (0.41–0.97) with large variation only for ![Formula][3]&lt;/img&gt; and ![Formula][4]&lt;/img&gt; ![Formula][5]&lt;/img&gt; For HSF and URF, PA was on average ∼40–60% lower and varied substantially among different combinations of BPFs used for model training and prediction as well as different traits. As exemplified by HSF results, PA of across-family GP can be very low if causal variants not segregating in the training set account for a sizeable proportion of the genetic variance among predicted individuals. Deterministic equations accurately forecast the PA expected over many traits, yet cannot capture trait-specific deviations. We conclude that model training within BPFs generally yields stable PA, whereas a high level of uncertainty is encountered in across-family GP. Our study shows the extent of variation in PA that must be at least reckoned with in practice and offers a starting point for the design of training sets composed of multiple BPFs.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"c8GBB82XjCo6vopKcZoW7jWpI8dZrQCbGgXAlkCwshc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487096","apath":"\/ggg\/7\/11\/3571.atom","pisa":"ggg;7\/11\/3571","ac":{"\/ggg\/7\/11\/3571.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3571.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3571"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3571"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3571":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U6yoycmRSkXrwVCdcIi3sNca5q9vGq9t9NVA3soiu2pogXOUtAsUZTYnSx\\\\\\\/vJUSP1wqaNMN6NZ41lMTl0hDfCdW2+iJ9PqQF6m6zkzCRibYTU1LLi4Ir6OTfflh8I72EdwuH6ne0D4lEYRGonBH6AnGDYQllHdIYUUuDta9hANbhx3j0Nbr7fpT0Yru4zbs2pDHNSre7TmQu9mO931N5NDqnEBtuLBYMfVcm5xWzicdqZVfY\\\\\\\/Y\\\\\\\/D5\\\\\\\/geMhwpv43ZSv3RnymL0qPsc7H4PUX8eSY7Gehye80jBwpwLB1WtyeAmzXY+g\\\\\\\/GrDaNmyVwIpbw7NHI89ZGLGYjCuuOejJ5TbcW5uMoENb4yqNMviLQ\\\\\\\/RcSgHEjNMog44R8nznoz1LIT50NfSMcS7026NgJvoTPMMYsrdKLzHHVFvwD+MZg16pud9keHmmQtDIRWY0TWwnAaQfKgGhP+jpfxEBy9Uev35\\\\\\\/iSt3yWAdo\\\\\\\/zGcI+RtMVU9AIvvNHWrH9f6xPSxFCgPk8QbhoA6r5XlhUXXt+u6N\\\\\\\/Ba\\\\\\\/IpMqgJPUNVWrhaYbOFDrNSIrHXZu+alUVmkxf\\\\\\\/AWZu8yK32lFf5leW3ybKN8rE4VKX4axLMKE4G5t1cwhOSf+4aWefNPMeY8YiY34Ly6NgOXiN7+mbPPBAtTBmulw+v3wGwk\\\\\\\/+\\\\\\\/FKQbjoGdMiAuDz1LzgzUBcRzSWdw1GGg176VQG5mRGXlCucqX4m7bAnDO2PXH9vbmNKT4wUCPiHtK3Tz1ntZ45NreV140b8Eqsaxo31C0wHYJNonil6EXEVezG+5nW91hSUklZEoXIhZMXB13Q16Con3\\\\\\\/IJTUlBYv0r\\\\\\\/eMlhrIXbgY6MQSfEqJq2z2Jvc8yZmxM8ZKuLqNrHg71MqQ9ffvq8\\\\\\\/S1uy1w37DP8p3C3DFFdUNjyYCsAwW3K6eIHVCvzhjThAkUCr0XnPc0wr8CBCPVUe2og6fCQVcwC5ppGNdEhdounbGlVlAd8V53s2hUuMH5W7\\\\\\\/OaWOZ0XGTcfewHm5wqYedIqCJd4KCPQpCcZz5+GwX4ymtstzTzako2QQW3O7mUDP54kL+pUj9HW\\\\\\\/o55b7ZJUn7GBbGH+aR6Wxf37STQyQh41R0x6gwmochuYTwvLbQKVtua7BCnh2pAZVb50bXsg4w4yCwMcBCHBh4W+\\\\\\\/+W4t+UE\\\\\\\/tpK4oT2\\\\\\\/4oGucWU9tkzb8AhpGsQG8g2UkM8IxVrotqp28JZFflyHSfXa\\\\\\\/qSy0WixFpj6PRzrp6iQNeObBTVpvGjwC28YeivmDqKZmOlxT1ggf\\\u0022,\\\u0022iv\\\u0022:\\\u0022NSpR1trhJZtI7Q8jtvTLfg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e1786ab8afdeb1ef41089a91a01a53b3fd599c71ab6d08839fdf0b39c9258a19\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WCdIpe0szLmVeyAxg0J7cxBLIuBaBntGxDpKmJUVkzBS2EyGMGhirPuHAKc4DcPuLd8CfOlAx6h\\\\\\\/+q78qJdkT0k42f8jbudtHFOZXniQXgoKg9hRGjp4Ii3cq02nKo2+Jc6IwhVI5HxH+2PfBL1HBbora8oTU2t6e2yyeQI3tsHFY4Dl1AdU1dFYuPfcYDWRTmork+Vz6pJxCjc5+lMhPH12ufKlmvv17eAetBfpMvRLisgUPlkJc+4V15lurJbyXLN\\\\\\\/Z+bJ8bsfAc8a\\\\\\\/UTdhe5UXFo5CoH75grIMsIBGOs017DTtNLK3FBm\\\\\\\/Exloai1\\\\\\\/rtUQD5UMgjQcP6xRYDQT0+9mOmynYHNqPpQNJdgwkkrLT6Im5erqFumbo\\\\\\\/U9yesi4TgMxQNXmG0AdWYQp4b74Jze+AjZJe+8hxu\\\\\\\/O99FT4lbPLzpmD8BR5Gn47SAWrBJPc8+PINcHcc9iqlzqK6bw7O+bh3J2ixiaySMtEVjbTIcv+X8wKw4C7Y9repUWMDsxcAfsfI8RKZFeP66K85pAth5VXboWeCD3h7jpP4PfdMrJ3uJCMxHr35UiMr1z6kqcAlZf\\\\\\\/Yq\\\\\\\/NOoA+ZKzl0hDxrt40KT+7NOqUa\\\\\\\/qgcJKa3XHXrORBXjmUk8\\\\\\\/WuaNrHgcildQ9WpOPWDXQcLMK2o5vij+ldsBhPPBWoyDKNFLZIxe\\\\\\\/ZJ2ikhI3P9zmnAhldZ3NerKSB9UxuDRSLXaKd7dfCrwCR8BzRL6kZrIBUqS+vc2l0aWLqeOhhnZ1RJTMca5au3XOiWQWZIADYSPzHoKLOTdKNQouRDzxmmtp1\\\\\\\/WB4T4e6IBoHf\\\\\\\/1Lvs1Qy6EYeSZWZKwkIbkCD7EprwfnOvEjcDENWsV7wS2xy2Y2zJA1ScQdtfwefCSNP5+LXPuycr1JMDve8jMIqyRrm3c81606TQRYx\\\\\\\/AM9d7DXynS8y0J5u00TDZ3aUNk5rVi7it+Ye9ted662BYPzDyHaehM4E2ve09icULpA4Lj\\\\\\\/J6ODZb+XnOimghCYnMVLMnKvmoVTLxbHkllcw8KkZrBSyYI5b7rWfOhFmHLcqZwiSrbL95msRRICULr4P4asgYuLZFLT4XdUY8+6pz4fbxEzvbrV1uuM\\\\\\\/QT9SFP+baypXGaaIA8JicOzeIt38viKaEi0+Q\\\\\\\/bO7SvmiQZjIoO6QhYGT8mD9ER2cl1czPL\\\\\\\/PRNbgUEdzJlbj7uEX59upnm9c1oH8e2k5JYjYCX2Yp7hL0UYaWFSU+\\\\\\\/y2GTv4TebdT+3\\\\\\\/vatkAtBDFywN9vS\\\\\\\/UFNssAboJUTvBjFwsqiaGfvp0MuUgmYMkwSGivzrFDre7kDuEYPWe6QNaNCceLYP6lBD7FPXXYRQNw6kh+kivJ4uEitHpuxEzRDJGM\\\\\\\/S2eRv0loicVNcBc0M7FN2yzPy2W27iiyFBNQCewFiZIkwrN0XUQ3Qs6l1xrhyGiHLihaDY06QUpRTVpcH+zNuEUUpP55pbPuErmuXwPNPlVAoh6Xxt7ivHG5mIQIkkzTtYaEajyVmj77jsTMQDrX+NVSiItwGUSeZTbQ9Wy0ynck3TIr6UrJSKRsjwdHpnix3vV6PKRq9nBkOGWkvilmeMi6u0S4y+h4qmSAlTJaPb\\\\\\\/OE+stuYFZb85OR2i4JM+Sde3XLIrCKDiLqmFEqVgitCW6Ts3AGVlulbz\\\\\\\/EwHPitfZG3mEqIt6C6wrNFoPeKsCaSGWW3FMAynLaNoZqfjo6ScO\\\\\\\/Fd46XproViO994JyKgMyP+ZZiTHFfIQ03poG5NXSbWC3QGNGyFxlRncVcigWYEfeWPBm+GiONQ2nfuQfYPPka7Acbajp\\\\\\\/m0fPlzHRJDGaC+OfW79GLg8FiIkWJdoIiveTxiOHrl4iZ7Sm\\\\\\\/DqNo4AipKoR0s6xlMdtEu\\\\\\\/N1locNcRJyJasie1tlp76z1lXi6gUsCIglngc2DN226b+aqad2Heazp8gI0OYQsbCGx0EHVuapZnvhawok4jJZhm3h9R6uY5dFBCzx9xjDlHxk0fXiCIZWdpG+LdKE3spNQEDZNoOgSfq+X7FZRbQ9klKr2UKH5zffr\\\\\\\/RDEkAWQyCnWCP1tTrhdDmdEv7oRI3vGtswHBnze1ARxeR6ZQSqXmCr3gByEiDWZ7\\\\\\\/958jN4OuhaUNptD13bxkPuYiDRNbrFWSSeiCkv3S2aLD+B2xvKdZnIj9S3haPwWMwr7bkDG9SyYp4abzfuXFtdMhS3KxPrPXKjKsaa+Wa0qrT4WtHxzznwQTx6xxY\\\\\\\/5aqHvlP5NTRD4lX1RTfyKst5SelVo+fUtaA8EhvCTIcPcdgILWXqoeTHr9s0qzkcVbV2yZ8ck5yZsktDehyqjZzDCG5R6iZMautkgypBjrYd\\\\\\\/ohYgYrFRaP4axn2zAAbLEJ+yeKNiOiB6T6Jk3Cdh2Z+46Lgb+alw9JNgy7MtnvbhkC5QqULMQxPMCu09TN5qIx9IjxMG6oOU\\\\\\\/JgzDuA3vWkhbBURKmijW1MOnb5+Ic3zSxzuc16bIM99jf5xS834AzR4ynCLqSjXj4\\\\\\\/DfiHb5n2oV2XmhWd4e3\\\\\\\/LUgAEwJPl1hKaXCq69RdOXIOQ9PU4+b+xIIuPDikpHMmYF2YLbnXnlZjUGOMUFfuLYGo6mesYJ5nwSZTYNpNGqbxL6oTdkOXuRMZHy21umcjGUWuJJ9QjVCNz0rW7BrM3dqkLuyV3F2DWfCNA4Wl+xNHV0KiMizawDQvi6Hf6phcnxDevSFE\\\\\\\/Yx1uv6rwe\\\\\\\/L+jNx2uT4Yz0C\\\\\\\/g+MiQYJa7be5PuvbqTAr3V4zK4UN0DPotHYdMXGrQfvuEEwp08OhtyCMyr4DG0yGO9HGy9iuGy1aF84qqyyeH1nQEReGudZWUkX8nw0yYSk+gJf4lentrpQvleV1f9AcRKmnPY2A\\\\\\\/hNRPnRaBc6xZhSzDfvaeYMT408krJ2MVIsTFF671xo3a9KWQXgJDyPgWYagbuhQwdg56Q1G8mTrqES+YUTODDJQ5gYCFNZ8dbXfOdOdxwDFF0e5p9H8RpB983uy6+9uvRbvFkpuL6EMYdRz0woUYtidobZb8xPbwGtSc7lu20mjN3ScPlKklHzylTFNePTrAFcz7qrKC1qDPXZyino055tUrhsKfLyfcJrzat0Dgc+ZFi8ZlKt8E+KTs3dp9PBU47dtBJZW58SiDKRO8VidSZVeO5ZnwVoOqrw0\\\\\\\/cS4kvpqBMyfc2CwPVBKQmkbw5cX\\\\\\\/jnUdsL3eXjRZt2ziTlS4KQmh6NPDNbllAch24EzNzQM\\\\\\\/2LRo5xHpdlIj5NSKODY5AHPDkBh3p8JpaRS+cPWCHvMcn6z3eWUFraBMtSS65ADYmG2UtQ2QNMDJXqJFTIkPsvwkGp44y5dwRR0bdvJh9ERvESs45xStupc9YIaG5OFmmHgDHU\\\\\\\/DIdvB1K5bDgfT9sUlQ18a1daTLyZLk3Q3\\\\\\\/2CI2iNJR0lZodeF12aQz\\\\\\\/lY8CCMdfeV7Bdtpbwzr1Tncs5kCPguWIv8r1j0dZGwYYWGlFZ3rP\\\\\\\/H8+7z1G9+F8iXrAU3JZb0LBqdE4K66A3Ttb9RZ4RysnW8vnwXQ6X7W5aJwJZWF+siP4wbwnbHJCZZYQ0c63h+oko1YY9J7oBWqJY3ZbQirto5gyvkPICLj2D7lA2qoPbYdm7768NjBbeAumTMD4teHx9eWKwvghQVhXTrmZyRKlpt5ZKpdpez794U3tNkdVeCx4ixKEAkPWlDwu6idnbDuoTtk3eIOVjfYZXKd8s4zIlqoKE2yboOtxg41flegBCXGZjbiftKpDD02yeSZwW6aNW4Y\\\\\\\/PHTklGkIudeLwLxAZv+ZqEw\\\\\\\/0VvTaIJJmnUKvFtlnZBw\\\\\\\/wP19SBP4FDajpq4iErmolmZZjDMXB\\\\\\\/4T7q1A5JBv4OyRariaLOJXWGBoEfKIPwMCo2HQLSLqaDkCa8bnA+r3s9KXZJxzqAo9ZMRbxahpn+Wb1kusueGXuUpnidfQ\\\\\\\/601y3IHbM24xrXSxj4gU6X4t314eY5k\\\\\\\/M9EvoNzO53+9SOlZZcgthFsbNEiT3c2kiGs8gYUSIVXYrqbTWfiKdCd2Q46KRIBU54ElO1fw+Os3V8Xv4AhaMl6Dsz31CtptCbGhxiFA9F6PJPfDU8qfj5YBR9DzSy8FSInWFC07ceGCcSmWg9VVMro3taDhMN8WunAnc5hVcNRs392Gx2d0FqjQCbNgdqYA+qKBlpVTGIWq+pT7UCqL6eVlJIAm9TJPM7jEJv94V3+0ob6QpeldMQQJCbN4+FTPj1ECXK9PGFomPqpBIyBeCVcqS5FUOQX3\\\\\\\/DfxxREP3bFN9RI9X8NMop8LSNWriRToKEBDXOPNPeML3kBObe5VC5xxffMdz913\\\\\\\/p37muXoaqdEHkllGzkAdJJLLWobZ22oxFy4UFYiBCd\\\\\\\/UDWavVQO6S10XHTWH4easEeDTM\\\\\\\/zzuyuOdrRxzktY3WVpUzDz4U40hHnYxNBzbkmLAgipFntJ1DcYm+iJedbBrAdx13o\\\\\\\/0oc78sY+2wcSXufGGugK9X0nfOFqP4DvAI8DNS2xwYkps\\\\\\\/\\\\\\\/YPCc5cSCFDBTmmOi34ISn6+BbEsUH8+KOvQS7zsK3QZqV4kL6gAIsD+Pu25qtR16Eft7XDlnJxYEohBhYyv4Rgzs4W5CY2CsTsEb1Q6dMSbnb\\\\\\\/U9HVxm4dhEVzzFDaP4HBhuFWD47WXUUa8tPH4Z3gX3WSeBTVEFHmOdgviEKEtTIul7iejqgdjQxf+WN\\\\\\\/3So0hnNIxWe66dW+275Q5IOOUtWbbBSdDa\\\\\\\/xkkb1HysqmZEE2HhWMmfXKHDGHuq9klpuUDJyfOxMCytZQEEyjSU9lRMdypz2SFADjm6WUoA4VmHHbijhwe+I2htN9AR1h5ZrmVwZCokk1sNbbmOvPuRkzsEHvjzkKCNzoU0N7KbH9HWozzZ19SjImbTeIHwodPw6+De52Vw9V3WqSoAcmL2RLV\\\\\\\/VbMqAbFxrCQrP5Q1e\\\\\\\/Z\\\\\\\/8lGmWngPPMrhM16aJI7qVO4+TFX\\\\\\\/Jeez8Vt+zBC8AubeiV6atqMUNJJiNOPVzov1wgF0OyKhE+AIvOiyJP2ewCpHBt+9QhXHQmxqtVQxq5jNrZQR\\\\\\\/qBJ1uXvB3KIBEciiF2RSnIG53j7NrTf5jak35aQcVu8Tqu5xpJ7NeyDZYsVwMfP3MkCkZG95RLIW1ArKQlMoY\\\\\\\/fkr9EtxLGnVfs8VXTSRfSj\\\\\\\/V4kiyi3f9HMqR0ThWMlkdNs+eDjkLsKqiO6d73zy8WbegXPp3j9bxsx4H6r+ynsr85u7UGthrxmgFkdo+6pbOQ1UH\\\\\\\/fgLH6tIRN63AdWsZqQPilbADPjQHw4jPI+NwmFNhtWXDeCU2oG5DsyPIy6c3eLA5FvanW4l3fzoRPe3pQJpSEQ69dpPVoyuSHP9FJoHCjmW\\\\\\\/EFKBO1grGhDQrug+7gJ1rpWypaSDrOIru8MnkmHsqvJ\\\\\\\/MfUK1j4W1\\\\\\\/9of7bY\\\\\\\/NJkIi8ZuKlfUC5RwG\\\\\\\/i0UT+N9PmHgunMOi0aa3H79td1lTIE2sbOsoUCpxxWI2Krt3huoryvBr+1kKvVh\\\\\\\/RwV4xpI8s5n3F0bxz1j18RFH8f9YuNnZ1Is4nklM0bPkQ9jWKpudEfJSHHLdX+yN8hbG3TvG++loGePs9sBdrhttm5+j+2j5rPCkc0nmA3BlyYQs+PyH3Ql74HSaq82dqjn9aTfLhCmBW+L2jPgGdzCbqe8vhATptNpShjKTIN+EOXPs4JyMT5cfrEDibSX2Pu9GOQkg+ENwDqUueqV7x2dr1p14UHbR9EJKxKmxpxgIqDvDkkDVJbKdvwc5x+dOxqcSFkbr\\\\\\\/rCbHQQNKGtIJUpagyAg8uRDYKAiECto4r6EEkavLjp11u8VgcGU2nArAuO\\\\\\\/s0jMUGfKdrlDjgKMEs6LQD8DAUJkep7D5WvZICuXSa1D9WYj\\\\\\\/bRJ8I1z9iAlNPehH2PHSaMOtb7Me8iAwPU78sWC8A3bNPcH\\\\\\\/UozTKhMDyxrA4b4T8DgJlNYf+3B7tPBWtnDqFhp9edMtCW0eRQ8mzcFIxoS5tEqJ3H9R7uLko639prXRfXftFEMvw7wpYIPDbtEPbc7zkbWPUAoSUBGUfDjY4xBETvZMXRje1c7p7bGhMwmIHQv\\\\\\\/AUAG\\\\\\\/g6BYJ0ofwwEb4vL34xervfyFNcv88tOT2TpoFxA8lIWvl\\\\\\\/gQAJAOexewRVKxa4z4U8KprrdMxuP\\\\\\\/zR8VDNswTZUx6tggjDalscanTx7smXfPZwj7P2dL+cAGgEQimXDwFuVNuJrwcldVprPeniwopUQ+p8tak001h6eLD\\\\\\\/zhzb6qLcwewXz0j3EQsODv7UeyD7GPOdJocub8ceQHLXAejh5GMAB4KZ5dg8RfQkH8mPFeEu+jNO1h0Ua\\\\\\\/WBQiRt7t1wA3tbdU2Q+Wftag4+LtNArkia8Ngjbrgh+fykasrp8hfG2Pm7C+3ViAisiHzCkOf1MELKLSmnVJ\\\\\\\/8o\\\\\\\/FSvUfkd1tXUGTUsVCryPpgOUyXrKiE2VjWOQ9n+1iwbAsNoQYiabcCcNxCSs6CulTgRD9F+346wvJv3pfvR7bSwKeQrD4mbqcrhFjKmTrMySQyCz6tR\\\\\\\/B05hk20kONJS3U86UAPR7WGurei1kufSt0pkNSJmUjbZPTffgdVgqVWsKudpNM24OmNQe6lMrH1ksijGCghXE8Rs0HjKlzGYVbzlbEIJ3AhoItyEolO3yG5LfxgKDqRzTfnyXIX\\\\\\\/2Ldl\\\\\\\/MHOPKu0I06LJLEiIYz+mhUVHomBaQppQvsvzaQJCyk0UULdm8nRHipo6SB+Ww31r\\\\\\\/HIs+WOUMbD+Hzhj+294dwlpVDf2S1C6wJoKqCHAN79hsXqMbY\\\\\\\/sXW6u+MYUcGzmRsSypHlq4iSHOTEqp\\\\\\\/8upXwxSbaS5h31mvL36XSJml9+Pg18jeD3dKgqPXnJXFr51JSsinmN725A\\\\\\\/quAY8CR8yTmXmUOM8jqRRtJjK64up981vz6XSQ4xWrQyZoH7Vw5WzyLbq82KzoQXXFdlkwUWQ7Edz3dNRDqTDmA8JK5PeHyUHvSeWCamlOyF9+Ty8W\\\\\\\/+XjltVZEwHLlIb9aEjZeP3yzXFXKpN62ohKdC4kbE+DvdZ48kHZzQkrMO+HeLv4DUdRtlOoGa+uwiiEcVdYob29J4zJWmexdJXkPx44FDprs+YMUOwDJ6v2yl33iq+aKAK+RFz6i\\\\\\\/N6x1NW63Kij8QaCexY+1YNMedI83pZTqVSy8BtJKU1gOd+dDCIBRvQQh2px4MJ3xYJMK62VXlJ7Av2liDYfIJ5eqVt+ZBMcFbLJj9lnZBMJsGJsGvj\\\\\\\/dOLmv+5YfIU8H9FrpNPweXG+pHxVZASluWJDu3iX4y99kNueZ8D+e3drmXKztqPJN1sEz\\\\\\\/P7\\\\\\\/dJcoKvVkLxCLSUFtsG5QYtxG\\\\\\\/O3\\\\\\\/Hxhajb88zV\\\\\\\/Fx6fPeco7uuBbcxWxFYXh35cKq0aPaU8lcI6snaS8+l7XEUkOOl2kY1SRTo2tvIb7cZ41HDx4NXVjk7xNTDGWeaQNrJR7y2GEliC231rlZOd5Hj6gIcML1O4ENcdq6ukNnAdJCnha8XAqM\\\\\\\/aRRbdXcHbxJqUp9D2p7UIZ6KzNNB8HRh92oY7syHSSH5mjARpIiXKgH0QSDXpZop4GfkOwp7\\\\\\\/vcJQZhzGiHDBC\\\\\\\/LcV9MWjOCvmjfNdBOD5jlW0J\\\\\\\/keqO0nUvyzH7Wv8xzr5K1kv0r5vwhINbfaRwdTrStanh8f9Y\\\\\\\/CAD3LkIMKIaq1kXQcqocJZsvDAJ34fov9W1aPQGjheZ4E5e6N0sJYPWK4tNRnMwcMN1\\\\\\\/FkzCdb3VkKXiR2IsxJlXhPPaHAvVzVMFpIJQu0AB5e+06fno8HpjNEZSjPa5h\\\\\\\/QT1uv6ilza8sEZ0MdptucTif6ZUPgELzgIKTfpapJoGrc+csTzjuVVN8s4RIMH\\\\\\\/EPVEABKpvRVlZwIjJjiMytpWK0Vx\\\\\\\/lUKVhTQtYBxghoOfmMDi8KgdRZANjc3HwYNGTxyrLk6eXGLjoolnPxDKGUY4W5MOkXv6dqo\\\\\\\/Yh+R3ogn24NS0TSNMsawVQ\\\\\\\/Is4RMto8m5kp58xTne+WyMemWDe2EE6S8fxATrAkOit+dZftwYBLtnQyrWug6c5jDuEhneOvszBN6tud0iieNPQhYwqe2phJT65RFG2ay+bv8YlajkZUcJ+7YKtgbPmMGeSEpPQXSjcZ6cg9DxTAEtHQVdlxqSiu5kpzcnLumCr9kBTkFnxhnt06qTFYLUSqWa5CF84+c47vA20IY4wZ\\\\\\\/SS2pK2DLV2A9rc6sDh+iir8j7hSHERMBc5frNBFB0e2\\\\\\\/MoCl7ejIc6huWni4v2LMdzvCUYfNGCidk9Fd2dMWOQqUO889aXN+M+VqZ3y3H3hK21UVPSkDmCmnDQO7WUZgg4tAiGEJQoCZpE5AUPuDFNwsXjP+pXT3xwuhzmxcGV3bhQFOeivZXtHXmE\\\\\\\/Weu\\\\\\\/ix1ruFF+I1zVxPBiXnxKWZRluUl577ArJMGUrxn1wbDy6g\\\\\\\/rI0fJm8PgBfRP+z9e71kZRdcU3HcERZKSp\\\\\\\/MaAW7En4VXSskwdv8jXVggiWc5Y8pkjNm6k1lUZU9xZTICdtt8tbsO5Yk5vjzJ3ZO+DbcAuVtch3G8Uf6xE8GHTGgwJvPmOVQvzIdmnuFhzkVsbFxCbcX7lzVTE+AVbdkwAW1Cp246qzbJLMugVRo4JuT8r3ZJFqrldrlhCV5KGeABWnkXpQg49z0zDZhzXhu4XlevvpI+7M680kI8YYiPpdmFuoYzBOmKCJWg8BgCtFg3X5Xt6dZa6086WybaiZKI+7BbPapEsgLPFCe3kIprrK\\\\\\\/H8P1dbxyfZ+MioC3uSYUyrRrgpdrlawL+5MtM6+GIngT+4R6cDGZCtZZ7j2JaKj9P9YINlXuc+4TuRFEkV9ZGHYLeRKHzH2v3mcTsZAEG3g7LVQBRHo9dIZ+uTIzj+CcFP64pVwd0+xjZ0\\\\\\\/jPh2JFy5obsNVW62Z0tumQwbrM0FTQRxQ9Dmsav1oyXXrcmFFocg6+j1AQLuelGGhtM0i\\\\\\\/qszdZ5YJ9HG1cXXG0wgbrpNsMEV\\\\\\\/KFxhK3ERX6kAcwe7DXwtwvUoaq9OAzCsCniNvbJwSIYEh0fba\\\\\\\/lWOUMmv19hbLC\\\\\\\/OiOR63JHN\\\\\\\/7+GkS6TkJYmRxAZ2kaqPw0XTo5X2cgggUcCIK8vyuGnjq2LJGdh7idkSa0NlsaIC2tl3aSzig+1N2vxj40Dc+LHpQYt30Mu33BstzRHmUZnrzCPtBCIDBwPUqUrfw4D+Jr0XuwmGvVhMLstJsF90hVJ1tzDwz6DUbiFHRgCSYDtmY9oKxOvWGdXeYqihJ11lIHLIRLq6jHQbIeS0s6EQ5hagaAT2pJY1HWGJl8AWIU4HXBxiCGsxNp7BWVQDHo5c7lhkLP0GeMHpZ5KeQnZ7m3AzVq6Yulsfcrw5bwGhGOpJVli04PV9VFbcAgmr1Kqd\\\\\\\/yzQL1vJnyhdqGRWsAmBoaU\\\\\\\/mS\\\\\\\/cJyLOg7gZwuKDCNaVSUue4hY3nMZTKf62GbCTUE8RIOpequIeFe0y2Ku9ZeVBWVmWe6qqMYFqYBMJl9dRLqJrgtek2AjMgMW1MeJPiIogFBS6Pu8gpzuFtNCQyywF\\\\\\\/wE9+dzZkW0rE7aW3LRycyJ5GxMSvublnaxnW4fZjGvNcDnsO+Vmt00e8oOyd2lb4lHVsKfZpbNQ6umfgmIrcPX\\\\\\\/u2q3fRJ\\\\\\\/VRKdYx8IeWEOYtaqNEMxbylds+BxfgsK82O7eE1\\\\\\\/rP86Uv8QbAqubODZhtFsVd3c38N7kwp9aXz\\\\\\\/f2L6kX05RAvVoPZIUCJatlPpXkp97MuGnM08xURXStikmV9VlUJrPQd6Nt1fzo0nDTD1Zu2nebTA8OQJMBtUlT3VnfGSqjU8NkmNxrgwCzVZjZ+XQ\\\\\\\/NOhB0oCrQYEKcoO54XP2CCurGR+WMZzo\\\\\\\/42DoXbDGyjG+hpQ6eJdKB5lgz4NkhJStk5FJzmWWdBLe\\\\\\\/REmErLLrhrsqrGiLE33CvFIKoNJzdV26g9C6KBk4x9YKDgCGn2hIBTekKQgSySpjn\\\\\\\/t3whYlBG91XrWJnaOC226qd403cnvRjhOvVM8Bdu6YoXgjpsrPRiVw1Aa6Ujy6IWtBq7Ba2nm5GajWbQslaGeqPTBLkQTEQOcFAO1bloJmI5yk0gxmi3rIzk7Z2SK8jQvYI\\\\\\\/IQkvx31O+bGYCCQVhNuulLEvAKLtGLW9MdSnJOu0oH9Hxxu4T8unYzYZgYnDt+v1bFfAiYpsYetGcMvfw4YKVbCGFcG08jaZILsL3MNerRU5m1AbmnBRe1fhUnnnNUVrpwFL97MJkzUz9dzKuXLA22yNKPcYdpHDCTOhSR5m3XvUlD8drd\\\\\\\/7pTyvVlb5awJ0OLmr2xFNF\\\\\\\/a9dqVr1jGQL7Qr2Djt\\\\\\\/XSh5DNkcpfn1qx2LBkYIMkZUzXuErvjCSkm2bsXSMhlcKwlN2duGu2XS7qHi8P2LWNmIIdIXVoYruc2BsDZphUiUryjAeE3m9R0nANLWBoWp+W1NnFG4YWI2wIe49qRxxa76jLspC29sKzWf4MhJd0D5GKXoXOe+X4oKliDQDc4yi6Ba+u2dhPwvxlusg2exlZi2OrwYzWMMMiquuS9IU2\\\\\\\/JjZ+7RUaIohx2TQr7T8jx3Z3Ct6qxG8K50QTt7gMHMtomeJiZArXDc7xzWhLB+tz3RLD3gez3Zt3Ermn6YaHupTEO42nbA00o9GC7jxGaMmd9AzqVfUCrfDjhD1B6mHAoanue4ykC2wFImuJdmrY0WB\\\\\\\/zG\\\\\\\/6nNd3RQdUFtaqNXncYamFMo\\\\\\\/\\\\\\\/fuOKrKT8Cu6eFSQ0pHpU5zVPk54j9SsjG\\\\\\\/Ine7C9NBtM\\\\\\\/nmrbdwDzfN9lsO63NYN5Ghh+PwgVKelRoHDnbkAIBMBRFzZytoeQlO40SZ5eCmJ\\\\\\\/J7bwepFvMciu5hssBDmCdKndimO0TXlCF2VOZOdJiYjUOGa315sL3K+ZucHOt+jHrmVud+vqWV37kjRahNh71x9YEACkqRcIEMXCqc\\\\\\\/fwzIQTkuANR4Dvw\\\\\\\/Ou9e2XEKdhwz4vgceuW30wIT\\\\\\\/LsKPIAjX8UZozu82b6+CG9Eu8XaLdK+rbJ3gUzAi8IL\\\\\\\/P+LOJnJv3LCdLYjB4vKBQcZ\\\\\\\/\\\\\\\/ccHxPu6AhwkP\\\\\\\/BztqXTqdnFmXTTMgzHQO3i\\\\\\\/JuhFF0PMlxrFyQMUvGbUL21PEKt1vh55WqkPO+v76K1YnbACGGfEKAySENi3Q0AXTlMqZt\\\\\\\/q6Q9KJQ2AXi8AiD2VUG7uVyvhuDDKM5D3Zd8xoWuHsTICJ1ttUMD8FIG8A6WM3U\\\\\\\/cSvBdhiM0uSNSnMFT3QrFzkT2xNR0szguR9HbrluGf1FkjsBcTUfUn6dPtM5FLLBcsV0jUhX3I49tvgCC2tdJsm\\\\\\\/JATQSiId6Fdi\\\\\\\/iqOBu2evc9V1U5FDylhyHCDMAfB2Ms79GKgBko02F9seAZPLBMgrcYplsDPtdc5QwU04HZj8aSGBsuLjhU5IIp2FoQUqoUWuKMkJnoXAUHcDqKuqFillJLGusB7Lax0c\\\\\\\/0majyHRcu9Tzl53kJnvZ37aMT5GBpGu2oFOaP9VX7AQMVCrl9uYpfDDgoHnthQS6vWuwVAbaLTDwhBDxn+zVlXt6XQzqwcl8FcPI8kU+HHjCCoaZ4CZcsfIR8O9JIpyRtPinFAPLD6ZTg3h\\\\\\\/GNwUCqW0+Aop6hEn65WtzMMS1oR3Q70r+k7JQGh\\\\\\\/d1FSVMints96lQqTi8F6T3midHScMzdRQ9fqU4hmar2D5pyx46v5LstiDaLZOAyBoqAwL1tfT45MbCetfrMNU\\\\\\\/U\\\\\\\/7ZfzKKXtJwq9W7Eg4vs7fDm2X3a6jt8L2RtNMVETSSt8s6ODFmAUXjpHXUehNsxR7xnjPkadCiOGUjDJPfLPzMDE5wTX2dses1oAFOPSYh4e0vegnhgcdqohQ7ZBw1hZPqW74ktiEForBRAYFxGom5\\\\\\\/+d5BJyuWekA5vssKC9d2jZl2nyQ8zsz\\\\\\\/Hw5QmMcWvQgKkq\\\\\\\/ENLcHUr\\\\\\\/zel+qzLsUvBBgpoLJT92XGmOKInOgsMN8sYb\\\\\\\/ld2f2G4Zw\\\\\\\/YRFUx164r2HFf+UJf+EdrW7ZJUnxkTM7KI\\\\\\\/vZtqSpp4xfI93dU9zAUpcDRcY2Gpa85ikbOPvIxRPr39Rk03RoLOkoqLbe8iHfQ09y0K7G\\\\\\\/zKIMpq6dIOD2VztYBYrtWykktN49C5LfNC97y5U7umqdFTmJdiB58U4oUE8EE\\\\\\\/+wk1RQxjnb+JedLNZGrElQiEk\\\\\\\/WawWl3Ahv5ZHh2sttr8iTWu878C+GQUp9gH1ZHOZPUVclB5EebUKmAiRsK2S+RikjaY+ld\\\\\\\/i1DYV6kf0bqPg5LfDI8B+v1TQgIE6+Sqkl51eovhftRlWCWcPPq+owCRk3OwLrPFcHtfbZQFFz0rEfQpEwedPspvKh6P75dIFjAGHyKdf1OYmzLXhEQvLjqibnDsANXulCEPRNkWsCd0FDbp8IFfpgbudGuNyOkFKgSXvrYSiQXuoIdq3H2omTlL2tvyvcpfsj64cytN4wE7oM0twuqeWBajet9388nLIzY2YeIysKheAv7xZibxZMD9bqB+rQ8qhk7+bpSZW6p0TYCcKSkEw\\\\\\\/gZy5H+o0sPmF5GckTA\\\\\\\/gUF1WT93u7AkyExKkjOFu7zVcgb2CP15bfXZVmcH\\\\\\\/a+txY0kXV8MWOCAIn\\\\\\\/jAPUUO0bFQfFhw5PzpRB5d7d+9\\\\\\\/VjTBheQsLQD5RQbTl+nmOCGwni6nlwY7Miou2gsf9ojc0wQcoENMwW\\\\\\\/n4pasmUrMJXQbY7n39L5TZxSmnbzq6h45rf73ARuSTdBuwV9zCPM5hRJucLMhiTajTNVKl9CBO5FK5KTTyySfH+sjhZ+tLy7kP826pV4jM3HXy9JHyGLOTezerKl9uQKRn9JpEcruuA52WqMOlwf82ULn+3DmJxBnSXRJ+UgwFNK8I6nxVxhMa04BOxqOsgyQPGPLB\\\\\\\/cqksbLooRcaCcFOgK6DfEZNTcqUB2sVn7H1T0hV18c+tNyhcU2jU3wOIJtf3otGiHy2uFPBbdXDGYBR\\\\\\\/8yG6Uhuv1oo85p9Q7WJ00\\\\\\\/0Ys2yZlJEd+nyxwXZG3zJH8MQpeV40Lqa\\\\\\\/oikZEBKc\\\\\\\/mhbhYp6pdOJWIMA5fePgtrWpJ65pZ2FvWJhH9g3SWTQBrrT4TXzFGONyAzVajVKBd2IDpaMcoxQSr5EEZVbtsZBAdo\\\\\\\/HBNUlKAhDqVZic7z8N2EToQbDPi4I29tUtV9oqs92P5JnlQMgmvhjxHRkhta5Hgpmv7GU+iz4lGKntsmfZun\\\\\\\/iL6cCc6PESWixt55DbKq93lrQ7MH5AWr2my6I3zDB+N6b4brg8uuQSYDk4YtL62ydM60chAprcVrVZ\\\\\\\/AfhGBCC\\\\\\\/NwF\\\\\\\/3+f57ErWhO2+bbEkE4g\\\\\\\/XbQelWWcNb9n6sADejav\\\\\\\/6RbEdQwuD64RM0Wng+nvrEoDT5yAXkyaBRKgTwIbSK2Nccr29Q7+Yp+A0blFWTjFBld6EIsYhtb6gVW1f3nXwzNrJhh6OJi2db6W946JS03DLgByxLimk3\\\\\\\/\\\\\\\/dyhag4LP7l4LsVRkgmTk77pePoFaxz2uQiTJ\\\\\\\/3uu4f7dK1Jx6fzNbkiM2Y3pxqieZ71rTTcRLZOisaCe7MbfPW7d9FXV\\\\\\\/K6cJ75hK6BiaBUcErA9qliU4\\\\\\\/A7E5upeCipacKdv9Sq5w8nyr06enGHcaHAXO3m5cCgMQElJZaMOROsAaXN63aO9Zl9DiSCHBJJ80plfcsMNeT5e19VKSvfp7vUYHO9Nv0Gc4XnzZJrNZYY93DYN2gWkmi5JGQC4nwTZ9u8dj0WNsQQCY+bwAQ20IBANVoFpg7tGGI9JCPujrNk\\\\\\\/ZEjsCaMvaUo2InIbGyFRxSESpy6n26jAtAfNgf\\\\\\\/6gmpP6LaJHBX2vo4QB1j1AnUyCj6MPJwzWlpGEJiK+UgZoTjo2qeoHS7axU1TdCJfrA6EN\\\\\\\/v1zPg9URn5wNU6fiGF1qpLb58p7RBRYH8j730HJe8K\\\\\\\/WbtHqzJEXYZpDiqruurRrsnVHIZOfiC0LGGbbT5AHHFi9pz0lz\\\\\\\/sv6coR3RakurY1iN4LM7Ui8YrVd+P8S08nkZxXODHrVDaF1oOtfRbCf\\\\\\\/UO2hQjZCK+7ZrVJPJ2k3YKNwBa0kdCVlASfvXzQPs3wP37VsYnzVi6F+M0SXHlomVXjcTuqRKb1Wjirn7yScrRyawAIW1xlvALgk56kmR2WXsmaDMIu6EmqSyonDKwtD4Q5ZkRnFuaDWCjQ7ENRtTXc3AdI5wNBOQOqFKz1SO2koFieoLvY5cV5uU6NQ5+s+dRM\\\\\\\/FBwFKm\\\\\\\/s2kZstowZUM\\\\\\\/Mn+uYIQEgdfLZ78+cy8PkiBaqPXWv1gEv4DZMfLGT1PwOkr7eBavsO\\\\\\\/fE7wwgNSQinxnU0pr+Lt\\\\\\\/5nU07XPbmrQsANdaTydFUVHZP3nZI4ElkSiVDBW7tvdOR+RuKEn5f\\\\\\\/fuNAYMmjIxibkwimGR6gNg8ePc3gis\\\\\\\/TuNgRbyN+RKqDfrQXm5ESQ2shleH1k0aEu3okIW0S3r+LX6uvX38y0jbVwoi9phC80mXq7+7N\\\\\\\/CL2NRPr\\\\\\\/UdTt3tj7VD8mMfWPxB6FNy08htCeM7Cn\\\\\\\/rbOXQrCiSJ6sUrE4gY1rUpLTT+y4JjpyI71YOd5nNyMuC9Pl7hfLzr5ueNcD\\\\\\\/\\\\\\\/MQkuOE4d1eth3CqUByfIwpbG2j5uJ3GjbhfGL5DxSj26odODCkVQy\\\\\\\/aCH1zM3Ke1aaOvXarwFSfuA0ttc1bamWu6swO1L8wMCxa2\\\\\\\/fxh\\\\\\\/pkwAIhrpdAsP1tS57GJwvb6fUjh8AdjQj2uASwzgzCAXjQzyeK3Agw4uJMlCRa5eX89Ord0jweU8NNsNA\\\\\\\/VkiU+oX81EHFaqVcXkeLlqO6a5rTk1QlSgMOwi7z9WhpUwS5UTI3vBPGvX4Q8oSft7LEDXQiuNHof+6aqIRiBrvpTZ2kp8nQ+WqA7ESXEfaNQZTpBB\\\\\\\/oh85VuANxtXxyB\\\\\\\/rPmL\\\\\\\/cLPe8PgxUOFWFD9AoxSkkadyPkgl26\\\\\\\/jbW42MFoYxWhBjee93bNvYixX9XxqTCLAl9UjSr6ly2vgCmaQYZjOQmjp6kTs8UUZ9W1ciJxl0qfyyvhc7+GUjxs6+KiVzVd5RNS9GltJZdNfJl+yJAE0b77JZAD2qSKyOML3Ml+HtcyCZm2umN8RI9cLjiCWhfB5Zbvk\\\\\\\/Jv7fUMCfxgqQSRRLlUPtgdERbZVKsG5s4+Oc9xLfHFjUlUe0L5DKdACtI8yp09ToQ8y\\\\\\\/PAnPwKJQ\\\\\\\/DfAVLiYGM8iTuF8MzImRp4hJ2FnZDPLMNizTHIc0IA2KJIpydmgno0uPNZ7r1D2mqiedoLHXS1U6N6FsFIeaQGjTT8BlETRaNBGxQ9Xh2vMVeing4wn6PI3uTImuQUD+Hb+REnGztL5mol3S81SXl1+Re8wOPO9xRLSfHdd6N8+3eIp3lyts+vE1Lryptq5E\\\\\\\/5j+nqGo\\\\\\\/SQUSkncl18+d+uuN2DxTujo6k\\\\\\\/qqzxOMLK9aZNBdYQZ3cCJuS5EGRJuEMblaihlwn6wQDhTLNyAxRghwAakWManjESRd3tsae11QQYTdO8chLLkRx9cwgQ87Gm0d0kiOlXu8Kpjzr6g5Zh5KeCFzFnUmiLh4rS0KzyYn6VxqZod2Ev5VeSbGZAd28ja5xI8l099dKMs9E8XmfyDveolZgnuBScvJey2IT15w++7ktpLZI1dER5L8cERoPnlFQw2qZEdjl6NqA\\\\\\\/+fNb4jFVQzJYqYVAEYtz15RFH7PQQNS7+oA2HVjRt7i076uG8OKw48IX2YqZ83qP5\\\\\\\/V8DsP0l3Kmm9NVYSDBtcCKcGegKT97KYNEBoo4ahvjv99dVAy18W1fV95KytX6sKVCNFwl77ezIP6sTMto14Vy0O3HRTmfx\\\\\\\/ywZpzzIOGQNWsi6UkWXcptQzR\\\\\\\/LtobfJoG1U\\\\\\\/0Q43b1DoPJfIxlaRPFSuxiDYfAywwyjys+GVUNsyuYz7bggI4U0kHdsbMjDO1fu0maEGp4p3jphcRprgExe1b0LeLzOGZBrWBM+WWj1zMZpM9\\\\\\\/AfJXgqpknim\\\\\\\/pOHXLo6QM7y\\\\\\\/UvARq2OwUZxtg70XfXQXGMjcZZ0DXm1ld99ByFCa794btkaFw4+cPg1gq3t7aiJ8fcPLjV2vrQVyPhjFoY33zcwh0Aa1iNH+LTzk5OSBlD9EgFB20ZL98vvzDI9Wg96sXyKJL\\\\\\\/AP446g3HOLGkVOZR9XAboye6NUXjzZvmrIyCmZ0\\\\\\\/o31hZRg\\\\\\\/MQNa9kebL\\\\\\\/84yRyucMsPN7tUdZ0zqTgTop3rTA1f1bhDgM9QgG\\\\\\\/H6QdUfsN\\\\\\\/\\\\\\\/Uhu8xQ0RuMJ5hJnf8OhYCzxN2LGw6J6MBhxXipCdFKxxleMxO57N\\\\\\\/2W+mI0uU0MPU9kdABbTiZM1k6\\\\\\\/MDKnAGfM6PXN+FWsOz3UW\\\\\\\/UmJZ8q15uyeAhYdhYJuycwadWAmQARHgZGbpzlTNBlpww+xT\\\\\\\/kCfCis4kzUMKQZYUXCUa72It+Edcbf8Tp2FIFQ4iDeA4+EpYtwn0hmI8whV3V4NyO5DTS+4wN139WuducwCEti0Jf0LST4gNEUcgXYwoZep65cSWUW4d5uSqP3JSmkIOMcU99i2Ui+X0UFcAOlYYIx4xT5DDDE\\\\\\\/Ce+hUE4+aT\\\\\\\/Acz5rMUHcEMzUKxX60GGsHxB7jsNziok4+LU7JFm2e2bCGJlIglX0uls7h8VqLzDGJi1394OG6KncpwNKWF7MPaBk3S25OU7Re+AxNa\\\\\\\/fr8YnONshcGb812vBEbDNVZQhooCcw8IL4\\\\\\\/irP2pB0R6vjQv7d+5ogF4Hzr4V60nr0Qz2BnZ4k5DcyRZ7FntarRKjw2BOtUeIvTzTmjjXqcj7YPdgFPa1dHVVE4F1+kMOALaXd6llwZ5aF9VEv9oOQ3rEdVmXrR7Lx+fZPvA5wXHp\\\\\\\/m8J95u7IYdAIsbqHZIffPzCmZWjz8+N2aGPim6k8bUHC2R\\\\\\\/lXnHMTly0Vv++zcrGkD1z+h6a09Q5aHyCNl0wDP8Wlo9Um0W4QPOYh9qX2MxiIA\\\\\\\/Cvw\\\\\\\/\\\\\\\/raMsBivGLTXaFgoHQd\\\\\\\/lkiea+z8b12ATp8Z3sir5bo3hjBzPh4ccnNxchWY\\\\\\\/ofD6pQYJOnIXOzTiL3L5qfN9XVgP35gx4yMQndayL8y3LEwbkkAQ4Mm26HOZlVGk5K53aEjkDyhD17Sbu2ra5IDrJESb4qiItEPkFfrMokoHPg26HG6swqCs\\\\\\\/okFObrfnPLwsRjk+Qp4RTuSCythbl1WTmxZ8BpbCELjshHGMGiBlDAbJ7bEQDEhPVPLoLV836j\\\\\\\/Fya19E1LizwwUoAg0k+uaZUir7T+8M04rQp0c75R78TkVGMY\\\\\\\/Z8Js8VnV8sU2kHHdjx4lxkIBRhKTyUsQZFjz2nn\\\\\\\/nvuda2sB\\\\\\\/0vt+jtt2Re0qIahu53tRwVMxsX5wJG1yd79egfw5QKpefI5HTU3lUbYPrEPSNvJdJogD6a9FJpnll4\\\\\\\/Ho2EoqM1AlVrX2tk\\\\\\\/idgWuUr08+bo4BxTCDZbhYqNSXNWZzhJbbv6UXIeyuwCFx9I014nOE8jr5vjppKRoyVxccwfRvdlxsvWBoGZIVvgA2nTw6BVmdiDGBbMRTirQgfx8hO3QsubMIjROZPCS8Nh0yoAQ2c0p4Xr0VCOHA4EEDNYYVJqHuGFUhXFspLUh4ZsjLPzT6eehztiqw+b7lM8d7yWCwC1ECCQaasrPZZ3SSKMphde7Qemyjh8JpLy9Nhw8pGoXig3Uz5x1+LoiRAtnkSBD+P2Y0mm01Be2iiPPDSJHmk4ZAZNtcxNUiIsV+tsKnOPMjKzYoMOB72oI+8Rqeu0m9zL2YNCdq+T3ruSOp2H0fFexb8s0iQxfCiMMfD7XsDoEXZer9aEM8iTqbGS1vwchfOz4DgX3GO4e9MXIAT76P99eC0l6TsWyjAvzBP2Ew4eFUS6jcxA2wVdbhwyecxR4hh7xa0sYvDmvQ9I\\\\\\\/fWdIMzgiCXU6FMtZZsnAyGrc4JXmrmxa0m+P1wz5dkVwitnmQth\\\\\\\/LPgfpkbLZkyzE+tTdLj9NHscG3mAN\\\\\\\/TQUwM29H+mRqXrGOzbr1VehXvPVgX98531VJP\\\\\\\/LUgfQwgr3wHxXcp3spUGYA7OH\\\\\\\/EnJkozolkIYkMNZmEE7evzt22awwpTYsUTz6gSImHjQThVXhSoI5f73PYxuNnPKQh8T6CHAQs1O+ymjNSYoQZ3csEubq4ufLhxVtrK4ycIwkMFWmz1AD1ghOY0Fu1WxMmbCKtijmAvljIoOd4tfuxrHFxVkGom2P38Wt4EWQOfHBhAopAP48If7wlJNmQbAaU4hNhYaYmVGA+dOXT+q\\\\\\\/dwko2Sz1emsBEWMY9R4NOH9zHqbDk+PzA5ubA1LJM7Av69N4auzC+TbaSuxeCEH6DHALd5wIquvq1Ac4t7ZL2z6UOtoFyG\\\\\\\/C3NltBYHk8CCU\\\\\\\/vhIUSADdsCzg8fuJqHzYoMvSAJs2gOfvVKI7I5K3GL3dEo1Vg9K4lntk7vCLqVvij5jNc4kFrFsIUm118kp0MWJscS9X+QVxXSpmG+dxHWKYifXxKGL7nyprk+AK50HuKnl0pKBMf+DPImjoO0xHA\\\\\\\/vXjohjxGaHox9scDuWy+FOVCZwQvOFjDmDOroqb\\\\\\\/\\\\\\\/MyrrVIu\\\\\\\/pZJQTKmDM7WDXsZq4LiqPt71rQYw7UcUFLCdU3sw2WKe3hxa+C4TXA6tKXdm2pEWFXloVYFGqGYjTSk0k53CEu\\\\\\\/LKTu7icepQLKEGsN\\\\\\\/YBMR0XV3gsKaq02dGiIJROi3W7LDbuIXK8XqBIeZ09BEgUnozgf0OpTWrMFr0RoKaPKjhcEFiE\\\\\\\/tZwA4sfmwm1dQsB7xUBE1eaY1RAwoNsgADlNuTvyK2PdI2gACAq41eru66WxGyP2eteFydp6YRA3h67A1dQ6X\\\\\\\/\\\\\\\/aTFwvBzZnAkPW9QSZbtnStSpL68to3MbTxa5FyZHH2t349TBDg8ISa5Fay9IVQr0EC2tKSjI3SfueDUChRKYHeRs03ixRuocOWdZate18tPbrgULrO65uBAyBC5kfyJUsqIWV+eBkubMUnRhPLJkPHZVqOvB8MXiuKMhKBKlrqmfrNW7BGjNaa2Amg1NGTE9csjxIUXfuB7OVEJkikp1sBlJiY+FFw\\\\\\\/flrON9FjWGCBFvrlXYO\\\\\\\/N+UFIGWigcMaiIkspJnSNpbP\\\\\\\/iaGmg3Y2ZzBWztoL2AUdpwgpG55SVI79zvIEbczBuebETLqZ\\\\\\\/r80tufNM8SRKVmbFfPVlzz511t+BfPQ\\\\\\\/PD4cI2u7HrKeHSVB41CQXL4R2UxSuMPpQENmUbioyZLTnUVFN+pxrBLwdaRF8QZsEKr41L11UGYtBBf0R9XBlDoqHamhyA6yYReO4G6FcGeNLtgBWoqb9Otn0xEGXi6d6a\\\\\\\/ZiuPr7YUOITynmBUDR0GAa3SRdeFm6wsS3wb61V4j2oNi4tbeG6ahS6cVhvzOnCcXR0rKQaQB6hpqXcxUtsIGb7ivDIDwVGv6oUVUwDcbEpA7J4Y5PzXOjWVD6kOsGzYaAl5jbLsqyMxBL4eOtJ4RZ\\\\\\\/ccbRHMgEZUY3uKkIIiIcEaRzuRKel9sHH+dJUFr\\\\\\\/rSXg9u1q6P\\\\\\\/rQUO5\\\\\\\/6wmnF2jenjVNazC5LYGVZNFoHZo1RmImeTTv5RSxGcOB9WxQt4JhDR2LfYkJUlqWMaX3gwRuTrkxNO7y9iWWzPqV5v5TGQfK6x5IYor+g9ZIeFcpLmWCiSyGAbFaB1MVk0eW5F3GZDb7Q+e6hdkWI\\\\\\\/crvV3+APL6XlKPtLYxJP+g2HKcY4Hl+ozT6DJ2lIKS6pEbHsGMgME6Y53C3h3NK3ddWajxOjNiWMY+b\\\\\\\/xXrmUr4J\\\\\\\/NsRxotdZGtJH9Hw4GvPDhtxRDC\\\\\\\/o85HGUxj1OmmLesxyp8PKRio3SfehA0ecHmmG5FsXVyxpW20R13zoVpV9Q224kMjMAnsEg5TJRpuxXOaXH7lLgqSV5MOrQ\\\\\\\/k1\\\\\\\/etQfrUh\\\\\\\/rl8mgBkdqYoVKWZWdbshzXcxzxXQD0qdodk6VmgcIi3e9D4MYtomto6u1\\\\\\\/JpfbmN3QDKhX7a00Y2mBjvYAmCCDWrfX5HD5T8\\\\\\\/Z+kZaqP4l2g4t48sZXZsfjoZS3zFDzTa5cZ0udBjkfZgAYcx\\\\\\\/\\\\\\\/8N+ecp6bh68BvwHRABaXDIjprg\\\\\\\/ynGkPESLrW7qmQrJpl346AcQ0onQ7ej55Pzmtr2MxjWCj5UkKetlo38tmBgsGYmIb13JV3NJS9hZdpIjg9zK8R4Ovco6pNRvMR06B4jvGXHEmy1PJpcw363UrzfGsgdr\\\\\\\/RMriqoZxnsH2b6LH+nt+wcjnmfynvvTY7YgwELRpu4+sRTAvRrvlKt9fAsLugTuY8EB2nmFnb7ASYghtVCsntISwrkAidkwiArQdUeaQ2Ok7PPn8vPYHEIJTD6pA6jonU9EBMhtmI8N8vuqN9+IsmK8mLzhSavhVVLW9IYksQwCC2OnWatCxtjAXot8raA43iLtXrsMvvHmHYKuaAzclEH9tGVWuCLzW1b87ZIzsLuhq69OO5EPyNyMs+XXPRKQWeEavb39zC9pYo1KdQcTMomNx0YoV8SXGhMuTL9wHn7OTRVSShQiG1TcADNA3Q2fyK2P+40SMBJcPKzlwnMbqDyXaFtx04Pbx\\\\\\\/uvyDPIdzDnmpFmgOCf5zcEv5WiQtg3\\\\\\\/IKOKISZ2tLN3CK0f0ZLC9EG4GB0moIEoqXdPmBIZPHJDoekhoNBmDlM4e8xKgLyuF7hOs5TYd1sD5oZ6VBDU53jmh44371Y7yMibUtxFiFFB5KRYWQP6Xa\\\\\\\/Uxr6+vcD5bDzhMDPMuit0ku6HHlwEHEVACaIGAFt1711iiaBOAxV4PyzG9\\\\\\\/cQ41OFXYOSeg8fQ09j+q957PV+458sv4Ps0cFBJiX59bHxRURf0W0Qez67sK2MplQeaIM8JZEuBc1cXjYF7B3LvvYGFMTG7VGGERu\\\\\\\/yFlOGFXs0mKd2grxR11AJLdCNmhf7qThyRb8EVmj8F7egJa1\\\\\\\/rJ3gXSDKGxnLYzbsK2Nxv0JSBmoKePB8\\\\\\\/Attpy8ewYtM7mJrVNFHO8JEONgf2BZazPn8mCAzEojlrYLqqmezRG0Q\\\\\\\/VEzaCngf0mM+qbi5HiIefTWHZ7qEIF9KDSxWGaIl07hhKBt46lFpmyFOJrF6ds51\\\\\\\/+pVl2RS96WaJBqinz3AE3H9if6WnNEDm7A54hdRdyifI\\\\\\\/oRXJ9YBd39gvPyqW1H1EXT94taTCBNRy2CIAfXGecaq+zRLBuL4HvUrwnkN0BKm2BVvkpe1qhC4niphClyFB6SF0ycN7kDsC\\\\\\\/u4lwHbL7KO7Q3zACWBEmmLO8xDx8v\\\\\\\/P2qJUFS5CafrvxuNDt0vbo7XBLEe+9vOseteKLJw4Hj+EAgGOL\\\\\\\/3rwidkYJRIorNeLeH\\\\\\\/VX33jQWL7nEDkmQxuq9mShI7zi1zUg8yG24Th1V4PVz0oFQ\\\\\\\/BOfasTbDBGvlvGChOUgW+RzQ6txcf5kjp5tvVIWzixROo0p7aGoVIJtb43R4Y\\\\\\\/+z5M+lzJfh8pb5EhrVuOc8D0zLs+YsFxFsiJ85IfvNhGRU1VDERKekm1PT76vK2nnQ2YB0VhGxUjsQFrncLL\\\\\\\/8GdPV1G9Umn5lG6k6TN00lcS\\\\\\\/qK\\\\\\\/yBchChTf0BOJmndZsGySgY7JWHATZlAVOIIutCbYETR6CPTYzkk5q0frL9Q9eySHF5dL+An7Ktf8jQU4vzKsMH9Ja6pI2rEjinNbwBSJY6khwZVzhDBXtgQ1XkumRI5LoL8tNQjp0D7CtR0DCFsmVGHfvMPpMG8l7UW7n0qS\\\\\\\/d4lDk99IA+iTxwZ29tO3AnDcAEULtuJUVf6Xqcx8NhcWNjxlthMZlbRaPTN49cdpIHEhbb5lv8YuFBHjduHH2WwsviB5lYyB7arBSrOwmFbNK0v7K6QgNL\\\\\\\/bXudRdqFt65k0C527CGGWoJ\\\\\\\/p6WILelXyTuxy97OCcyQ8mUgABH1h7MJoDiWXUEvvO1q9n1v2QS7m813v+63qdC+ai6K2ShcDO0k4e\\\\\\\/e5BCTYUTgjT9JwU9XrXBNV0pRwnjBgU\\\\\\\/TzKp0oQO38DVfemZxByC27Rx\\\\\\\/9D4hdaTMZCwihpTnGShEnRM18YEJFdeQnrH9\\\\\\\/nhsdvatMaDFFgFGUAv2q\\\\\\\/QU2J0vnLaeNJktN91xF0WaksVGtcTVOxDQ8Kqv7Obb7h9AtXceAbZl\\\\\\\/t7cVmZ73ZunnNE6XYQT2Si3QNu2n\\\\\\\/f0TG5EviQg2E9j\\\\\\\/bap4ZPhS\\\\\\\/g8NV\\\\\\\/bwSMpUNGBtWhMnmXyAJAgAOO2KU7ZYf67rGLxTx+NFyN+rmirU9Fm42idFs1+k76HIVtM5vfFfab5KI4NJxnOdgpsrL+gPiBOh+hnnYS1DFCLXwNZF84yMnFzjescbUrKP4VnUQj5x4uRFCktlR7XM04YudBo5RmbkRlshQcdiCdlnSeQNITI\\\\\\\/igRcLaBuXlUP5gOm1M8AUPZnZ7j616HgRZI7xjKEmMMqsY+a6VPo6sPKbuxd0p1wTXJv0agWrX+aGSaigYTeYM664sAyLHYDYz15s+S6rs8hlM0jvw\\\\\\\/cP6uLnqdbX6ZBvtu4BhJFO7dPVjm9i0Jph6b3TmOkmbx6yIKt4PEDWlnyE3UpgI7blWSa7t0DhKZ1CSdokEA+MiCwfqsDiGZ1qVIIkrgA59Lp4cbUfWA3DnZ587ycz64WLKGxnn2wBZYWD\\\\\\\/Wz0XPzjNNDz4mqxGx\\\\\\\/y2mMA3zhBKmXvLPVOBicDhQthAesQNmetZwHn\\\\\\\/gJRrR7Rec5i8KuwF74unEfKrAvMulLRD1EOapO6bolQWsJu0REFA6MAb6DLpsey6mdSxY92tNydtty+HDH0ci7kj2AvmXgW082M7N+di1Hyf8LeOgBVIbA3WNDBGiXngtdQRRoBpYA2uTX5tlnddA6KQsrdDn3QXjThhDW5GoUeWucgmtnKh08r1qEJY8w9ekRS4Dy8hkTTWDGklNRlmGzfjRshjgL4DunyKCC+tgXpQhKK9ij+sApsNZZQngMM+VJH3MnCfXYsANlPsKY2tnzv6O0LuETM4hLSHmMsBCAI09aCwfQa0tEmggcqYQu3eKClTP0soXLaZS8SuMJNNVfMQGSKYj2Vm34sUZbpeVrJsp6pA5Uj4EBCsZTKLOJ0bC5psk3cb6Si8CNlCNzk6dr1Z8ocFkM7uZixV8hcKnT7W0f4RrCApq1EWqqxGJSSIG1DcSudD+rNBjPYcPwslrnS+BXSbT9HkCLUoDhG\\\\\\\/oGfM4uJWzFbTGerbtjcyTTVXA\\\\\\\/T6xzbPpykPpWADimM5tjMibX7uY94IBbZ3A67KCiOmQ3DOfcfXdNOwoBER\\\\\\\/XGOQLMCvJEmhWAeMyJpGFudBRQ\\\\\\\/UESosk7ozodI3VyeKYJj76wqUaVHzjmabuw+Ffk7CtFWNB5CRit3OrdMQ0lymQ5qTU32KorcGPsD4ZeB0yXS0+WM26iR6FbjaQ0ArppqngQIpcXSxqwVwd9VaDT\\\\\\\/4XaSuroJ5GrK8yTrFgkl72VAxVmht4ds+YhtvQcPY8f6wyRWU8zuPfvB47XgDlxr0JatMHc2j1iwTnZHgrjSGZMNyK4uGWAcpiV7g1kpqTDSLfgYB3Ruxe4uduAxLVPQGk2fJZWygfRvYg1dnir+nEqWSffxV0\\\\\\\/91HO41SZqU1OXxkSk3mbaQmTqlhT\\\\\\\/tYCXNGuh7In18YKRrjjaCk1brbIjoJwMACOG2a9RC8PmF85alX2GlFHo4FJ604aHzFBuS85BanTaq5EK\\\\\\\/RL16k4u8Ul+62gSMzF\\\\\\\/5ly0b3lXYpTU5gTUcwOqAoW5zR5Ygn17Ux+87e66UzxagPt\\\\\\\/D+4yT+w0C6VErODl5PRhJ8jAgZS5MPbrajk5Cpzt4hcTV4vUz+bfqgKXYSThJDdfUSrAGGOw5vkmBEGVnVVNsAcURdAuDi71Bb9B9WYqPZkomx0ABWfZB3wqew9d+0VHb73j15ExYrOfr8p9e\\\\\\\/RnHA3niYJuvEeM9kBFDY4EXMxDxdmNvXDAFcyu+Xha3OqEwTRi7IYiWRNB657cpRxwXdlR2X+bH63nkizPktHFWlNS1LkTy7LHoGNWV8s9EXPKX55Lt0rnQcv4JyLHSpnNnObUktNMw45RmuXtTwwppbzz8DvzVFwuirScelQmdLcje4ZDKbKoEOlF3mG1kqa\\\\\\\/tktojtv1KtELz1UVUTYs7t2Kd92MP\\\\\\\/TN35BvSaNvuV9b3Iuv9AyZ1mKLnUpwxIMrrpkfY3XceFp4jwy\\\\\\\/+D7XUIIEFq8NSuwWbNMTX1AaAHA19kQ6uuKwjbYle+HDOSexOk2FEqFnzI4Dqoe6XtjeY3Ogcn63vsGNf6zYhwfCbLWSEYlx2ZTJmcKnc\\\\\\\/EO+uziJN39e7esejc422Cs8nta1b\\\\\\\/5RI+L7oLzLmqzPgXUfJdRwI\\\\\\\/ndHVzk7oYH1ttbTQGwP7jIAavoJQ5x7wWRepw\\\\\\\/CyxXQwHe5GaQyO45w+8VPQ5IgL1Vsua6obT2UL6Zgbm5NfS+iagtxkzojsSf++HkmhFZm9FtAxh55UZ4wJkmTPFwvXU2+O5TXp+5ZCp+RVWvPbSDjRzsl5a3GAe7tUQRZGIXn7qwAdKFUzCLNt2aBw6KUFUKnCodNHO9V1+N7zAp0rKShfB3L5QmmAqGNsUuu\\\\\\\/FBH7zT1P5C5zQY52QUvzFH2yDewSEd8hn77MhmcED31iTk628VVswIQOh+R1fhYdt2kH8stFbZHU1kGMLTBm5nyRgmxvfxkmApU2+VrWehe3E6UsFbzcgcumzJO7QKM0D\\\\\\\/zSQFiTcH74IX19F9+iepOu4Btd2mof\\\\\\\/SNpoLURscVdf2Nhy6neV1t3BnR12GH9xC8yMsdq1efY89VNWtGGs4ePeHkMqIkvl3DEB0fSu2k1GS2mytKE2QccvTMNVQoHK2KIB6CQDi1+fT3lD5j63eLES0yTWS7IyzZ64aettvgK8paR11f9QyIx\\\\\\\/N2PGZePuVgA81TcAp4aAmhWNiTeLZsXD9+MqGsl2bi4HkHWg16j371+Dp9OtyP7OQKgTh0AUBFvVcAlC9lVHFtrCHI1UpDNRu3XmhZsI0x0p12VZ7362PVgd0fS+bD\\\\\\\/r6q9hyuPz6prEmjsVJGeYNa2dndh+vv33UQjGM5VaaoJ5RNDgiZ36jIkSQBvUZHdaBNVVkeeg+XHoDCMMA7Cf6TFn4+sO\\\\\\\/feEBdg3t\\\\\\\/j0VLmxngJ0S076gsYLFpU1U2DkvmAckovJtSYCErHDAL08CTsu5VF7VCg+cwHPnuPIpOt9t21XTWw9u+ENMqjRHy8u7sYqKhIZR8UFI37TM+qAepLGjtV49ni1Jk4r9hV\\\\\\\/rb8uPGGY6mlT2akzu3IXETw7Di59dCfnEHUcEVQbM6QZph1HU8lu93XbBY\\\\\\\/OmoFtZFHwKTIoK\\\\\\\/6qXMd4QcFcICOk5aAN6DJEWcC3MJ77qMTfEVxmLEzUAFKeGNUOEAv3pBfUszVvt14+ZIIbi48zMU69wrCr4DUr0nNObCi8FTuLOXUFiyR9RPJgp42J60tM7hS+hvlR4usybtvijPTKqHA0AqRZdeRrPBrDBxPfY1u8HsaxtNPV1YLT11HtKNyXm6JQWNXGYOBiGw3CmI\\\\\\\/dmHuQlj5LBzXwDubFDrmtmPKC49qRBp0Wf3LTZMynNc7J\\\\\\\/9Ot\\\\\\\/XqhkuQVDIRwpDaaAuTHwddWcvikzL3DEtK3ix5VRMReEeOvsVtzhA9qBt7PhB\\\\\\\/56b39IqC\\\\\\\/tihjpOoFOpor9IJ6iumgnJYASkjVnwgPcdMUDZgREFVLMwlrFja+9Ms+krvQ1uMp9lQRTCBul8A0uzjnMkC8LC2E\\\\\\\/vKD05ooiRBBeI16JeB7hNB2fcYc4occqiRkRKElIpIw8lOkSpyzfR53of1lPxlP3nSGzWKRp7WR7omDMcXs2m0W6dUl7bcK6tccWNoGR8sPSo0xQ5B9yuZ9TYuDYkD++ZVroRtPYN77OhHiZYlxRfW0zaKa3hEC9hd9rcmpgeMrflBjzm1UaxYcOvNCn0hmlmH3jRJbgnACPeQlXw5ueNZIvGJAM5ASeo3MfhA8hDOH4PcN\\\\\\\/GYfu\\\\\\\/dI08ssRqR20ra\\\\\\\/KR3sCGd2y2kZDXBnTMW0+t+bFjLt1z9\\\\\\\/v6zZtYnT1NaGw6iyYGOYOnQZL2aAAnnt\\\\\\\/6v6+95X5XF1BIqfoLHz\\\\\\\/\\\\\\\/QYZOc+qaauPb9sOaz\\\\\\\/7tQkV3tfIxEXyxeTE9gBu0Z\\\\\\\/hOJzyWrUaUQDKAdJxR8I4jgyLn+SNaA9aSRf1E3m\\\\\\\/UbpduTwgR1+yCDyzbcKujNwnxPSkZbJroQbwDEGFfecksg1q\\\\\\\/dflEDaRsJQlpCyBOBIJeUCjDoMXgm7s7x7hqKI5zLjod+eHQ6jPC9nGoTC5UEUqCvyFOca8wYesxCAEA4vaAPj0JYz3UYhT6gsZ\\\\\\\/0m2TO\\\\\\\/ySuYihgzKCceqksCUvX7zZ8ZaZhQdSEWSFV7SSTEk6a+MnJxm6zkO8\\\\\\\/LGhyDvf2JUQcoFMofxjUTe3yd81sizRkgAZHkcb9xCS2da9lwtbUUWDfXPYOdZksgHSqWjoYAMM1\\\\\\\/ORjdJRwtauMCPGTrMyw2qeCbYFuxN0I5Xro2ESGOhIzZWQILzrDdo6VXsRwUBudqA8KsSZ\\\\\\\/4mo3wXdf\\\\\\\/9jN7WNuLDqf6PYhLT\\\\\\\/fWFndwwfJDWMmetIKr1FaX5Xh327VxcBgzL2s3vH5reLZnaknzQBeYfkkY8G\\\\\\\/a5UqW3gSDUc+lfHjhsW63mrAVB+ljBGgkJMhx3WS7x9\\\\\\\/zAKGGrk0IDVVXCQ4\\\\\\\/PoRzSpghGjdyv8OuR5kO7n+gLxK3hrDHcdTbDfMyAJGyvR4NnNfKSiTZg0SD5X4Vm8g+r5fX2o5hzMQ09LtDE7TdexKkqqQt+Zefee3m3sGYWtgQ9qrKDFnNzfgVCzXWBCVOyfo5Bpn\\\\\\\/txJFisOYxgZxLRwvrx9GnJ3B2MKYYJd2uD7fRs1NhUy9ZnpDQfGNb5fOcPJ95Ytc6i\\\\\\\/SglrK\\\\\\\/8O3xcXE2fjjkEqTD7NXongmZdAbN47rSNQDz9Hg0NjzTC4j1tPiPnEtfDYPMt7b\\\\\\\/heVcMFTSHZNzXPijd+qGOPhLLFFE4Foaes2CmpGEtjmkMBBLPBVAQ8a4WGeWAwikvuAKdZ5ZkL50XPtJSoyuu2IzAtvrBD+RXKo7LCAYkx3Z8BQR2MvFD1AS4Lkg5BLM+mbQzFKRSDWLtx4OzUu0e8\\\\\\\/TQrox5ewIzacj0DnEM0GtGYhDIH1EaCVEkH5ay61uBvifZnShdTr\\\\\\\/YUPKvY1PtsvSda\\\\\\\/qTaulU9kqSyxgsamoOjywearw1Dgm9LXSuLxMciKU3pCQtHNJ5xBZqFNuCwf9x45E8USNHvQd6JDwicpEwtq\\\\\\\/ldS3xQ\\\\\\\/dTZ8YhHJXFMrumvM1+5htOMdpyXuzaRihTHWDURV1Rp6+B5msENHOk5jhASgKV9cIZsPEY5XxeGzHphJUN+Jhgs7TeIhiA9O6YbkOK2EeIjPXZiAAq5s\\\\\\\/uIK2\\\\\\\/P9F135mnbPP093u1SN6AwXNry6HE51kThfgpaotLBTvIrT6Fq5bVg0cjs+c6428L63Zp3SSEOFwXEMOpoNyx09FFzJEyxcK1iwbAFLXj4HjDqcAugClCeY44+1jcauGzX\\\\\\\/8rVavBnhGGNixmIDrIWxMx+KPkSyTtgkxQIgw3+kwxU4LOno5iRXpAKvUtoZfSRT\\\\\\\/Q5puijpcn2SPqMgfkMU14K0hD+7t3HcpL6mzvHAi1XA4ZLptTkQM3d9OHITAC2Wrjrz8uxihYiLrrJUBMrPCIqpgCEWZbY0C6rKKTI3ntJ0j4iZzQ3D3wyhpmivMfc13\\\\\\\/LyX9bbB5gM8KlCjHJSqOQ59uyAQMoNTUtreOSESp4MnduUg5hmfSJyUTSRElg00Qb1MbvJou95uSXpx\\\\\\\/LaYcxlQNrJC8mo2ViYAfE1Kl3ubphRvLcXOrttNJfe3cI+jb6v\\\\\\\/bfxJFLXX4bVrJrrrKrEur5oozweWtk4cavHRP8y+\\\\\\\/07430u0nQV9+BImLR9Xh0I2CeTQRoUFrvgkSg8F9t1WpENJScgo4NzpAAImEw49Qc3s\\\\\\\/AH0MvgPh15PHN0+t9Xn24d+V02ptJ8AYoiz1hiKGrkh5WNAeLAI10l9cIsm8PJyd7IGZUhQsiw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022uxQG0cPeM4vwAGSAoxttQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f6bb1f201dfbf686af962168eb6c501361de1428cd85bd15a5d3f050eabd278f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Gjho81HLI8BnwAIus3SvwTCEzhma2R9m1IE\\\\\\\/2EqSjMylKqBoCJTT6ZcNsEQZSiAUgIhLJ\\\\\\\/piy7tAsXfet48C8Dh0tjtY3MYWBlvNbdgH5NASgKMmC56WqMFZsJ0jXqUhEN2\\\\\\\/\\\\\\\/OWOIb6tM0Aiy0nRvlpa+Imb4BzOViETZp8\\\\\\\/Zmdc3HjfdG7eP+8p2Jxm4uss2b56y+JDGS+aX2sLQdiKwjVThwplIm267yAcHJMY9j3brUUojQdTanPIXs\\\\\\\/Go9by67PD34K1N5gKL+ZASFwqoMpiSvXEAw32DEwhEWqHu7ybpyo4qNRUri3mXcytvVviLKebwYPI+zp5SaCM2cPtu76EsAOBtdcJrqm7KSu7idjEBABKUvzUkC5DLNv09XYQsKoats8Jiotf3hjScCphMTm1\\\\\\\/bdVUr3OXAgGgcy8Nix6PNjfVMoOIj3SeQ9URQZ\\\\\\\/qkfuVFSdzY7e+TFg9giHdZregD7RSTUYOw8m6WH4L\\\\\\\/+rUX0FUphvzNXhsoiDtJvmH\\\\\\\/X7Mb\\\\\\\/n2MsYvhnbzuHlI73+TrfUDAg2h8aEWLpRfI35xmgOsMKnUyGI0Udr1EF+u0Sk\\\\\\\/+WmeyRRT5m49x1qeELVue0+ALwYq0fPswSqBkjGCNJR4duaYQFnIpn6wUahIke1NpY8OtyLUx9FnHe8ZisJ132Z4S+kCnS\\\\\\\/Gv5y77Hb1\\\\\\\/zEdztwQKecjnDJ8GKvs+LJbZ\\\\\\\/BzxPlNej3cTsn0c18FIl8eE252ppPoY\\\\\\\/kqFD+30ygIHZAPULQWlzjr009+FJy9ELANvHZwRqEczK5tipZLoDWOJtvWME8+mDebjvbZ3UqcuJ02Oq4fzewggho1CbIJByOYsyxarW4Hj3o\\\\\\\/xRbBtbMY1v5W4PySH3Q0sy8\\\\\\\/2uf5einMQsajWzYTsIog\\\\\\\/mcL6lMuJPxUZVvVCSLKYRn+nVAbLXI5p7adypz8aYinIa5SUG\\\\\\\/seNSRPBX0mEIJ5W7YMnQDYLyrpjQ1EPUiBcM5J6cFBHVYeycc94hIeR+UajqxY\\\\\\\/IsWl634k+NA1PyTtOVM89Kw9WeZkH66Es\\\\\\\/8pUWfg0fSXUSnk6siDeoUyxDks1GUVIypRkTLytMB53d+JhsRN\\\\\\\/yme5ncdhLlBSEqvqbdShTB6P5Prhxl1F0GWDSfMm2t5dbdWFIDmsrrP+QfN3MjnuYgFDraimhBR+eJF499nH2mGfm0tCgUijasjbzOxrgAIXeYhzPot9ENqp1suLv1Uj+75lJjAYUeINlGJ9Sw7C0F+a1mANV3ZYpQ5ipjczZa5LwtHX+7S2WTxhkFshkf+TtZI11NUHWnsfWn6Ni2NNVr\\\\\\\/189jsnCPq\\\\\\\/bhN9NRumZATlb6MfJIIwUveJpfu9YALlUWp2uktlBCZEFclzLhef5EGcZC1boYz38V9JZXQpSiAD8TiN0ftrYONP6tllK+NdZalUtmhdFSYs1BMAHGt4sKupPtywwuzICkfSSg\\\\\\\/ei6ZA\\\\\\\/OwRAIs8IZ\\\\\\\/jYh24FCjj3PAXgWUkUpXZ6LdckuwBg7zsLvUXLJwEKeGbd3lZj4opO9C2NdRfgYalO8o2AvMlXOifcUiq2O9NAR8+WhTKyZ7CBAb3ltAl9wxjHdQWShsbUs8pe6936X3ZGjgHSlrqeABEub2MNw3JnrYVWlLhCnDkt2brKqKfxybc2b9lu9WrW\\\\\\\/+fccDlGNavRarf6bDQNOe67hzPqcFde7N0Xf3yPjK57+EnFNVTwVgGQPIFaYFaq\\\\\\\/LVeONpDxpYEmh3p2eQTv60ph+ze3KzfXi6Rqa36xRzSKtE4yHJ0nqaD3dGYxLfik7FwVhHiuRvR8MUv1dIaSOWx8NVR3rQ13aBD8b4pZ71rr9BHu+IyQgLX8kF1lGx+jh1kbwrmXa1zYIAaeEXPZmJidBnQTbTVD6K8dTtanQpls7Q8sPx\\\\\\\/CU8mRqAw\\\\\\\/Mt3F7rvBVKj\\\\\\\/wX7eL1ikRQu+GQFY2VnVdbbPH41xhdjECHHOlxBMqSyq+Iqu5RplP+c4Chb\\\\\\\/pxoU1EcXF9865wly5WpV0lTKbRA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hG\\\\\\\/Svcl+JEXo8W1XMv3rmA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ac9c2c8cff7634af44fb02dfdfbe476607ca31534e417040b17aed9c54213d2c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487096 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3571" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-glsPAVQAX5u9ZhVYhn3T8WOcPdHzPk3sNfpGBiI9YOA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3571" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7WjX-ZPbG6yQn_gI4N93tQ2U9RXumWCXEZzc_Ll-gwI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3559" title="Genetic Screening for EMS-Induced Maize Embryo-Specific Mutants Altered in Embryo Morphogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3587" title="Mapping Loci That Control Tuber and Foliar Symptoms Caused by PVY in Autotetraploid Potato (Solanum tuberosum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487096" id="node487096" data-pisa="ggg;7/11/3571" data-pisa-master="ggg;g3.117.300076" data-apath="/ggg/7/11/3571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pascal Schopp</span>, <span class="highwire-citation-author" data-delta="1">Dominik Müller</span>, <span class="highwire-citation-author" data-delta="2">Yvonne C. J. Wientjes</span> and <span class="highwire-citation-author" data-delta="3">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3571-3586;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300076 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487096" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pascal Schopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BSchopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schopp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BSchopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominik Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominik%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yvonne C. J. Wientjes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics, Wageningen University and Research, 6700 AH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BC.%2BJ.%2BWientjes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wientjes%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYvonne%2BC.%2BJ.%2BWientjes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3571" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487096" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487096/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3571.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487096" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487096/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3571.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487096" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487096/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3571.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487096" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487096/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3571.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487096" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487096/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">A major application of genomic prediction (GP) in plant breeding is the identification of superior inbred lines within families derived from biparental crosses. When models for various traits were trained within related or unrelated biparental families (BPFs), experimental studies found substantial variation in prediction accuracy (PA), but little is known about the underlying factors. We used SNP marker genotypes of inbred lines from either elite germplasm or landraces of maize (<em>Zea</em> <em>mays</em> L.) as parents to generate <em>in silico</em> 300 BPFs of doubled-haploid lines. We analyzed PA within each BPF for 50 simulated polygenic traits, using genomic best linear unbiased prediction (GBLUP) models trained with individuals from either full-sib (FSF), half-sib (HSF), or unrelated families (URF) for various sizes (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-1.gif"/></noscript></span></span>) of the training set and different heritabilities (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-2.gif"/></noscript></span></span> In addition, we modified two deterministic equations for forecasting PA to account for inbreeding and genetic variance unexplained by the training set. Averaged across traits, PA was high within FSF (0.41–0.97) with large variation only for <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-4.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-5.gif"/></noscript></span></span> For HSF and URF, PA was on average ∼40–60% lower and varied substantially among different combinations of BPFs used for model training and prediction as well as different traits. As exemplified by HSF results, PA of across-family GP can be very low if causal variants not segregating in the training set account for a sizeable proportion of the genetic variance among predicted individuals. Deterministic equations accurately forecast the PA expected over many traits, yet cannot capture trait-specific deviations. We conclude that model training within BPFs generally yields stable PA, whereas a high level of uncertainty is encountered in across-family GP. Our study shows the extent of variation in PA that must be at least reckoned with in practice and offers a starting point for the design of training sets composed of multiple BPFs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/biparental-families" class="hw-term hw-article-keyword hw-article-keyword-biparental-families" rel="nofollow">biparental families</a></li><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">plant breeding</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/deterministic-accuracy" class="hw-term hw-article-keyword hw-article-keyword-deterministic-accuracy" rel="nofollow">deterministic accuracy</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-4">With the advent of low-cost genome-wide SNP markers, genomic prediction (GP, see Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a> for full list of abbreviations) proposed by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Meuwissen <em>et al.</em> (2001)</a> has become a powerful tool in animal and plant breeding. The basic idea of GP is to combine the phenotypic and genotypic data of training individuals in a model for predicting the genetic merit of selection candidates that have only been genotyped. Complementing, or even replacing phenotyping can result in considerable cost savings and shortening of breeding cycles, thereby giving GP a big advantage over traditional selection methods (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo and Yu 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goddard and Hayes 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lin <em>et al.</em> 2014</a>). Particular challenges of GP in plant breeding arise from (i) the specific population structures mostly characterized by multiple related or unrelated segregating biparental families (BPFs) derived from crosses between inbred parents, and (ii) small samples sizes available for model training (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jannink <em>et al.</em> 2010</a>).</p><p id="p-5">In commercial breeding of line and hybrid cultivars, up to several hundred BPFs are newly generated every year. Depending on the species and size of the breeding program, each family can comprise a variable number (usually &lt;250) of lines, developed either by recurrent selfing or the doubled-haploid (DH) technology (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Albrecht <em>et al.</em> 2011</a>). Since expected differences among BPFs can be reliably predicted based on the mean performance of their parents (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Melchinger 1987</a>), GP applied to populations comprising multiple BPFs aims primarily at the identification of superior lines within these families (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>). Prediction models such as genomic best linear unbiased prediction (GBLUP) allow capturing Mendelian sampling—responsible for variation in the breeding values of siblings within BPFs—through cosegregation of SNP markers with quantitative trait loci (QTL) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> 2013</a>). While several studies have investigated the accuracy of GP within and across BPFs, more attention is needed to assess the mean and variation of PA for training sets taken from full-sib (FSF), half-sib (HSF) or unrelated families (URF). Experimental results available so far are confined by the number and size of BPFs (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>) and low marker density (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jacobson <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>).</p><p id="p-6">Model training with individual BPFs has been studied intensively, and PA has been generally more promising for “within-family GP” than “across-family GP” (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>). Various authors argued that for a given size of the training set, within-family GP would provide the highest possible PA owing to strong linkage disequilibrium (LD) between SNPs and QTL due to cosegregation and the same set of loci being polymorphic in the prediction and training set (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2014</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>). Nevertheless, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> (2014)</a> reported for within-family GP substantial variation in PA among 969 BPFs and various traits, in line with the results of other studies on BPFs (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jacobson <em>et al.</em> 2014</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>). However, a systematic investigation on the extent and factors determining the mean and variation in PA among BPFs and traits is, to the best of our knowledge, not available to date.</p><p id="p-7">Since PA increases with closer pedigree relationships between training and predicted individuals (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark <em>et al.</em> 2012</a>), one obvious strategy is to use HSFs with one common parent between the training family (BPF<sub>train</sub>) and the predicted family (BPF<sub>pred</sub>) in across-family GP. Compared to within-family GP, PA for this strategy was generally much lower with the same sample size, but can reach similar levels if the sample size is strongly extended (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>). By comparison, model training with only unrelated BPFs produced from the same ancestral population yields often poor or even negative PA (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Jacobson <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schopp <em>et al.</em> 2017</a>). Optimizing training set designs in GP with BPFs therefore requires better insights into how the pedigree relationship between BPFs, the sample size, and the heritability affect the mean and the variation in PA. Herein, we address these factors for the simple case of GP across individual pairs of BPFs, thereby providing a starting point for further investigations on the design of multi-family training sets in plant breeding.</p><p id="p-8">Forecasting PA based on existing molecular and phenotypic data could assist breeders in (i) choosing the most suitable BPFs for model training for prediction of existing or planned BPFs, and (ii) allocating resources to the training and prediction sets. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Daetwyler <em>et al.</em> (2008</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2010</a>) derived a deterministic equation for forecasting PA, which requires only population parameters (sample size <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-6.gif"/></noscript></span></span> heritability <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-7.gif"/></noscript></span></span> and the effective number of chromosome segments <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-8.gif"/></noscript></span></span> When averaged over several traits, empirical and deterministic accuracy agreed well within BPFs (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lorenz 2013</a>; <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>). There is little consensus, however, regarding the calculation of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-9.gif"/></noscript></span></span>in general (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goddard 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Meuwissen and Goddard 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wientjes <em>et al.</em> 2013</a>), and, specifically, for BPFs (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lorenz 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Riedelsheimer and Melchinger 2013</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>). Recently, Daetwyler’s equation was applied to both GP within and across cattle breeds (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Wientjes <em>et al.</em> 2013</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">2015</a>). The authors extended <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em>’s (2011)</a> approach for calculating <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-10.gif"/></noscript></span></span> from the variance of genomic relationship coefficients to multiple populations. Overestimation of PA was attributed to a violation of Daetwyler’s assumption that the genetic variance in the prediction set is fully explained by marker effects estimated in the training set. An aggravation of this problem is expected for across-family GP with BPFs due to a high fraction of QTL and markers that are not consistently polymorphic across BPFs. Herein, we propose to extend Daetwyler’s equation to cope with this problem and make the equation applicable to across-family GP in plant breeding.</p><p id="p-9">Alternatively, PA can be forecasted based on the estimated reliability of genomic-estimated breeding values (GEBVs) derived from selection index theory (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">VanRaden 2008</a>). However, this approach has rarely been applied in plant breeding (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">He <em>et al.</em> 2016</a>), and, to the best of our knowledge, not to GP of individual BPFs, despite promising results for GP within and across breeds of cattle (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Wientjes <em>et al.</em> 2013</a>, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">2015</a>). One problem is that the approach was developed for outbred populations, and needs modifications when applied to inbred genotypes. Moreover, several strict assumptions regarding the properties of the genomic relationship matrix must be satisfied to obtain meaningful results, which will be elaborated in this paper for the case of BPFs in plant breeding.</p><p id="p-10">The objectives of our study were to (i) investigate the mean and variation of empirical PA within and across BPFs of inbred lines, (ii) examine how the variation in PA is affected by differences in polymorphism at causal loci of polygenic traits between the training and prediction set, as well as by other factors (<em>e.g.</em>, level of ancestral LD, pedigree relationship between BPFs, sample size, heritability), and (iii) adapt equations for deterministic forecasting of PA in BPFs of inbred genotypes and demonstrate their usefulness in simulated data sets. To simulate realistic scenarios, we used SNP data of inbred lines taken either from a public maize breeding program or a DH library of a European maize landrace and generated <em>in silico</em> numerous BPFs of DH lines. Besides flexibility in the choice of sample sizes, and exclusion of nuisance factors uncontrollable in experimental studies, this allowed us to simulate traits with known genetic architecture for a profound analysis of the causal factors affecting PA of GP within and across BPFs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Ancestral populations</h3><p id="p-11">We considered two ancestral populations as source germplasm of parental genotypes for generating BPFs. Ancestral population <em>Elite</em> consisted of 72 elite inbred lines with medium long-range LD (Figure S1A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>) representative for the Flint heterotic group of the maize breeding program of the University of Hohenheim. Ancestral population <em>Landrace</em> consisted of 40 DH lines derived without any intentional selection from the German maize landrace “Gelber Badischer” with a rapid decay of LD to a low level (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Melchinger <em>et al.</em> 2017</a>). All lines were genotyped with the Illumina chip MaizeSNP50, containing 57,841 SNPs, and were expected to be fully homozygous. Markers monomorphic in the ancestral population or heterozygous in at least one individual were removed for further analysis. Physical map positions were converted into genetic map positions required for simulating meioses as described by <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Schopp <em>et al.</em> (2017)</a>. In total, we retained 19,204 and 16,171 SNPs for <em>Elite</em> and <em>Landrace</em>, respectively, distributed over the 10 maize chromosomes ranging in length from 137 to 276 cM (1913 cM in total). Individuals in the ancestral population were regarded as unrelated for defining pedigree relationships between subsequently generated BPFs.</p></div><div id="sec-3" class="subsection"><h3>Simulation of BFPs</h3><p id="p-12">For generating BPFs, we first sampled at random <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-11.gif"/></noscript></span></span> = 25 parent lines from each ancestral population, and intermated them according to a half-diallel design to generate all <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-12.gif"/></noscript></span></span> possible crosses. Subsequently, 1500 DH lines were derived from each F<sub>1</sub> cross to obtain the BPFs used for further analyses. According to the half-diallel, each predicted family (BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-13.gif"/></noscript></span></span>) was associated with several possible training families (BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-14.gif"/></noscript></span></span>) with different pedigree relationships to <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-15.gif"/></noscript></span></span> These were: one FSF, corresponding to <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-16.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-17.gif"/></noscript></span></span> HSF <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-18.gif"/></noscript></span></span> sharing one common parent with <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-19.gif"/></noscript></span></span>; and (iii) <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-20.gif"/></noscript></span></span> URF <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-21.gif"/></noscript></span></span> sharing no common parent with <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-22.gif"/></noscript></span></span> Meioses for <em>in silico</em> production of DH lines were simulated with the <em>R</em> package <em>Meiosis</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Müller and Broman 2017</a>).</p></div><div id="sec-4" class="subsection"><h3>Description of factors analyzed</h3><p id="p-13">For systematic assessment of the factors influencing the distribution of the empirical PA, we defined various fixed and random factors (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). As fixed factors, we considered (i) the ancestral population (<em>Elite</em> or <em>Landrace</em>), (ii) the pedigree relationship (FSF, HSF, or URF) between individuals in BPF<sub>pred</sub> and BPF<sub>train</sub>, (iii) the type of data (SNP marker genotypes or QTL genotypes) used to calculate the genomic relationship matrix <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-23.gif"/></noscript></span></span> for GBLUP, (iv) the sample size <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-24.gif"/></noscript></span></span>, and (v) the heritability of the trait <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-25.gif"/></noscript></span></span> The idealistic scenario <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-26.gif"/></noscript></span></span> was included to demonstrate how the variation in PA behaves when phenotypic accuracy is not a limiting factor. Random factors were the trait <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-27.gif"/></noscript></span></span> the BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-28.gif"/></noscript></span></span> the BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-29.gif"/></noscript></span></span> as well as the actual sample of training individuals <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-30.gif"/></noscript></span></span> taken from <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-31.gif"/></noscript></span></span></p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487154/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487154/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of factors with their corresponding levels analyzed in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">We simulated 50 truly polygenic traits <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-40.gif"/></noscript></span></span> = <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-41.gif"/></noscript></span></span> each governed by 1000 QTL. First, we sampled at random a subset of 5000 SNP markers from all SNPs available in the ancestral population, corresponding to a marker density of 2.61 SNPs cM<sup>−1</sup>. This fixed set of marker was used for GP of all traits, because resampling of SNP marker positions had a negligible influence on the results. Second, for each of the 50 traits we sampled at random the map positions of 1000 QTL from the remaining 14,204 and 12,171 SNPs in <em>Elite</em> and <em>Landrace</em>, respectively. Following <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Meuwissen <em>et al.</em> (2001)</a>, effects of each QTL were drawn from a Gamma distribution <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-42.gif"/></noscript></span></span> with equal probability of effect signs. Importantly, all traits were affected by the same number of loci, but differed in the position and effects of QTL. Thus, the realized number of polymorphic QTL loci could vary depending on the trait and the BPF<sub>pred</sub> and BPF<sub>train.</sub></p><p id="p-16">Phenotypes <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-43.gif"/></noscript></span></span>of training individuals were simulated according to the model <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-44.gif"/></noscript></span></span> (<em>cf</em>. <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> 2011</a>), where <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-45.gif"/></noscript></span></span> is the vector of true breeding values (TBVs) calculated as <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-46.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-47.gif"/></noscript></span></span> is the matrix of genotypic scores at QTL coded as 2 or 0, depending on whether a DH line was homozygous for the 1 or 0 allele, respectively, and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-48.gif"/></noscript></span></span>is the vector of QTL effects. Vector <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-49.gif"/></noscript></span></span> contains independent normally distributed environmental noise variables, where variance <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-50.gif"/></noscript></span></span> was assumed to be constant across BPFs derived from one ancestral population, implying independent environmental influence on the phenotypes. We calculated <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-51.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-52.gif"/></noscript></span></span> is the <em>a priori</em> specified heritability (<em>cf</em>. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-53.gif"/></noscript></span></span> is the genetic variance within a BPF, averaged across all 300 BPFs and 50 traits simulated.</p><p id="p-17">Finally, we sampled at random 50 out of the 300 BPFs, and considered them individually as the predicted family BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-54.gif"/></noscript></span></span> From the 1500 DH lines in each BPF<sub>pred</sub>, we estimated GEBVs for the first 500 lines. For within-family GP, training individuals were sampled from the remaining 1000 lines to predict individuals within the same family (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-55.gif"/></noscript></span></span> FSF). For across-family GP (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-56.gif"/></noscript></span></span> HSF or URF), 25 BPF<sub>train</sub> serving individually for model training were sampled from the 46 available HSFs and the 253 available URFs, respectively. For given BPF<sub>pred</sub> and BPF<sub>train</sub>, we sampled from BPF<sub>train</sub> three disjunct samples <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-57.gif"/></noscript></span></span> of individuals of size <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-58.gif"/></noscript></span></span> (according to the fixed factor “sample size,” <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) with which the prediction model was trained. To minimize variation in PA attributable to sampling individuals from the BPF<sub>pred</sub>, we chose <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-59.gif"/></noscript></span></span> By contrast, the numbers <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-60.gif"/></noscript></span></span> were of realistic magnitude, and analyzing repeated samples allowed us to quantify the variation in PA due to finite sampling in BPF<sub>train</sub>.</p></div><div id="sec-5" class="subsection"><h3>Genomic prediction model</h3><p id="p-18">The GBLUP model can be written as <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-61.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-62.gif"/></noscript></span></span> is the general mean, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-63.gif"/></noscript></span></span> is an incidence matrix linking phenotypes with breeding values, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-64.gif"/></noscript></span></span> is the vector of random breeding values with mean zero and variance-covariance matrix <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-65.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-66.gif"/></noscript></span></span> is the genomic relationship matrix and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-67.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-68.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-69.gif"/></noscript></span></span> are the additive variances in the noninbred reference population of BPF<sub>pred</sub> and BPF<sub>train</sub>, respectively, which correspond to their (outbred) F<sub>2</sub> generation. <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-70.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-71.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-72.gif"/></noscript></span></span> are matrices of 1’s,<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-73.gif"/></noscript></span></span> is the genetic correlation between populations <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-74.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-75.gif"/></noscript></span></span> which was assumed to be equal to 1 for reasons detailed in the discussion, and ∘ symbolizes the Hadamard product. Vector <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-76.gif"/></noscript></span></span> contains random residuals with mean zero and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-77.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-78.gif"/></noscript></span></span> is an identity matrix and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-79.gif"/></noscript></span></span> is the residual error variance. We used <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-80.gif"/></noscript></span></span> representing a modified version of the block-structured genomic relationship matrix devised by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> (2013)</a>, where the across-population blocks <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-81.gif"/></noscript></span></span> had elements<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-82.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-83.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-84.gif"/></noscript></span></span> are the genotypic scores of DH lines <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-85.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-86.gif"/></noscript></span></span> in population <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-87.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-88.gif"/></noscript></span></span> at locus <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-89.gif"/></noscript></span></span> respectively, coded as 2 and 0, and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-90.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-91.gif"/></noscript></span></span>are the allele frequencies at locus <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-92.gif"/></noscript></span></span> in <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-93.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-94.gif"/></noscript></span></span> respectively, where <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-95.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-96.gif"/></noscript></span></span> depending on whether QTL or SNPs were used to calculate <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-97.gif"/></noscript></span></span> (according to the fixed factor “data,” <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Submatrices <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-98.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-99.gif"/></noscript></span></span> are calculated accordingly, but here the denominator simplifies to <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-100.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-101.gif"/></noscript></span></span> respectively, corresponding to the standard <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-102.gif"/></noscript></span></span> matrix without subpopulation structure (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">VanRaden 2008</a>). Importantly, the denominator for matrix <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-103.gif"/></noscript></span></span> in Equation 1 is different from that in <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> (2013)</a>, who used <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-104.gif"/></noscript></span></span> Their approach effectively removes all loci that are monomorphic in <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-105.gif"/></noscript></span></span> and/or <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-106.gif"/></noscript></span></span> whereas our denominator retains these loci in the scaling of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-107.gif"/></noscript></span></span> yielding a better approximation of the true relationship matrix, as discussed below.</p><p id="p-19">In any BPF derived from fully homozygous parents, the expected allele frequency of a locus is known to be either 0, 0.5, or 1, depending on the genotypes of the parents. These expected frequencies were used in the computation of genomic relationships. Since, in our study, only population <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-108.gif"/></noscript></span></span> had phenotypes, we used a single-group GBLUP model. Although we allowed for heterogeneous genetic variances among BPFs in the general model (Equation 1) and the derivation of reliability described below (see Appendix B), <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-109.gif"/></noscript></span></span> enters the computation of GEBVs in <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-110.gif"/></noscript></span></span> as a constant factor (see Equation B4) and, hence, does not affect the empirical PA. Estimates <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-112.gif"/></noscript></span></span> for BPF<sub>train</sub> were obtained by restricted maximum likelihood from the <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-113.gif"/></noscript></span></span> individuals in the training set using the <em>mixed</em>.<em>solve</em> function from R-package <em>rrBLUP</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Endelman 2011</a>). The empirical PA was calculated as the correlation between GEBVs <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-114.gif"/></noscript></span></span> and the TBVs <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-115.gif"/></noscript></span></span> for the 500 predicted individuals in BPF<sub>pred</sub>.</p></div><div id="sec-6" class="subsection"><h3>Analysis of variance of empirical prediction accuracies</h3><p id="p-20">For each possible combination of fixed factors (<em>cf</em>. <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), we partitioned <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-116.gif"/></noscript></span></span> the total variance of the empirically observed PA <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-117.gif"/></noscript></span></span> into variance components caused by each random factor, where we assumed a hierarchical structure for BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-118.gif"/></noscript></span></span> BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-119.gif"/></noscript></span></span> and the training set sample <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-120.gif"/></noscript></span></span> as well as cross-classification with factor trait <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-121.gif"/></noscript></span></span> Estimates of the variance components were obtained from the following random-effects model using function <em>lmer</em> of <em>R</em> package <em>lme4</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bates <em>et al.</em> 2015</a>):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-122.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-123.gif"/></noscript></span></span> is the overall mean of PA for each of the three pedigree relationships (FSF, HSF, and URF) between individuals in <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-124.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-125.gif"/></noscript></span></span> analyzed; <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-126.gif"/></noscript></span></span> is the effect of the BPF<sub>pred</sub>; <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-127.gif"/></noscript></span></span> is the effect of the BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-128.gif"/></noscript></span></span> nested within <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-129.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-130.gif"/></noscript></span></span> is the effect of the <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-131.gif"/></noscript></span></span>th sample of training individuals from <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-132.gif"/></noscript></span></span> nested within <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-133.gif"/></noscript></span></span>; <em>T</em> is the effect of the trait, <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-134.gif"/></noscript></span></span> is the interaction effect of BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-135.gif"/></noscript></span></span> with trait <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-136.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-137.gif"/></noscript></span></span> is the interaction effect of BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-138.gif"/></noscript></span></span> nested within <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-139.gif"/></noscript></span></span> with trait <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-140.gif"/></noscript></span></span>; and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-141.gif"/></noscript></span></span> is the interaction effect of the training set sample nested within <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-142.gif"/></noscript></span></span> with trait <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-143.gif"/></noscript></span></span> which corresponds to the residual error of the model. In the case of FSF (<span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-144.gif"/></noscript></span></span>), all random factors involving <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-145.gif"/></noscript></span></span> were dropped. The degrees of freedom for each factor are shown in Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-7" class="subsection"><h3>Deterministic equations for forecasting prediction accuracy (PA)</h3><p id="p-21">We followed the theoretical framework of <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> (2015)</a> for forecasting PA within and across populations using two deterministic equations. Both equations assume that actual relationships regarding QTL are known, and were originally developed for outbred individuals. Hence, modifications are required to apply the equations to inbred individuals. As mentioned above, the outbred reference population corresponding to a BPF of fully inbred (DH) lines with an inbreeding coefficient of <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-146.gif"/></noscript></span></span> is the F<sub>2</sub> generation. The level of inbreeding in BPFs of DH lines is reflected in the diagonal elements of <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-147.gif"/></noscript></span></span> calculated according to Equation 1, yielding <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-148.gif"/></noscript></span></span> in the special case of BPFs derived from homozygous parents.</p><p id="p-22">The first approach is based on the reliability of GEBVs of each individual in <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-149.gif"/></noscript></span></span> (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">VanRaden 2008</a>; <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Wientjes <em>et al.</em> 2013</a>, <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">2015</a>). Using the formula for the reliability of a selection index given by <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mrode (2005</a>, p. 15) and replacing the genetic covariance matrices by the genomic relationship matrices [multiplied by the corresponding genetic (co)variance components] yields the following formula that accounts for inbreeding in the predicted individual (see Appendix B):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-150.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-151.gif"/></noscript></span></span> is the squared genetic correlation between <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-152.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-153.gif"/></noscript></span></span> (here <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-154.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-155.gif"/></noscript></span></span> is the vector of genomic relationships of individual <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-156.gif"/></noscript></span></span>in <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-157.gif"/></noscript></span></span>with all training individuals of <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-158.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-159.gif"/></noscript></span></span>is an identity matrix when assuming independent residual error variances<span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-160.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-161.gif"/></noscript></span></span> is the relationship of individual <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-162.gif"/></noscript></span></span> with itself, providing an estimate of <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-163.gif"/></noscript></span></span> Dividing by <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-164.gif"/></noscript></span></span> assures that reliabilities are correctly scaled, given that variance components and inbreeding refer to an outbred reference population, as is the case when calculating <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-165.gif"/></noscript></span></span> according to Equation 1 (see Appendix B). The deterministic PA in population <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-166.gif"/></noscript></span></span> was subsequently obtained by averaging over all individuals in <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-167.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-168.gif"/></noscript></span></span> where in our case <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-169.gif"/></noscript></span></span></p><p id="p-23">The second equation was proposed by <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Daetwyler <em>et al.</em> (2008</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">2010</a>) and is based solely on population parameters, which was modified to account for unexplained variance in <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-170.gif"/></noscript></span></span> by accounting for different markers segregating in <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-171.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-172.gif"/></noscript></span></span> (in cases where <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-173.gif"/></noscript></span></span>):<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-174.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>with <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-175.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-176.gif"/></noscript></span></span> is the number of markers that segregate in both <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-177.gif"/></noscript></span></span> and in <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-178.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-179.gif"/></noscript></span></span> is the number of markers that segregate in <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-180.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-181.gif"/></noscript></span></span> is the sample size, <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-182.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-183.gif"/></noscript></span></span> is the average inbreeding coefficient of the individuals in <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-184.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-185.gif"/></noscript></span></span> refers to the estimated additive variance in the (outbred) F<sub>2</sub> generation of <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-186.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-187.gif"/></noscript></span></span> is the effective number of chromosome segments. <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> (2015)</a> proposed an estimator for <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-188.gif"/></noscript></span></span> across outbred populations, which is calculated as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-189.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-190.gif"/></noscript></span></span> contains all genomic relationships between individuals <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-191.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-192.gif"/></noscript></span></span> and training individuals <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-193.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-194.gif"/></noscript></span></span> Given a uniform pedigree relationship between individuals in <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-195.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-196.gif"/></noscript></span></span> (<em>e.g.</em>, FSF, HSF, and URF), the denominator simplifies to <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-197.gif"/></noscript></span></span> because <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-198.gif"/></noscript></span></span> If the individuals <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-199.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-200.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-201.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-202.gif"/></noscript></span></span> have inbreeding coefficients <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-203.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-204.gif"/></noscript></span></span> respectively, we propose to use (<em>see</em> Appendix C):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-205.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>For DH lines from BPFs, <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-206.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-207.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-208.gif"/></noscript></span></span> which was herein used as estimator for <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-209.gif"/></noscript></span></span></p></div><div id="sec-8" class="subsection"><h3>Comparison of empirical and deterministic prediction accuracies</h3><p id="p-24">For all analyses except the ANOVA of <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-210.gif"/></noscript></span></span> we considered only one sample <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-211.gif"/></noscript></span></span> of training individuals and dropped index <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-212.gif"/></noscript></span></span> altogether. This simplifies the presentation of our results and corresponds to the realistic case of having only one specific sample of training individuals available. For comparison of PA between fixed factors (<em>e.g.</em>, between samples sizes, heritabilities or ancestral populations), as well as for evaluating the overall agreement of empirical and deterministic PAs, we calculated the general mean of PA across all random factors <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-213.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-214.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-215.gif"/></noscript></span></span> subsequently denoted as <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-216.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-217.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-218.gif"/></noscript></span></span> for the empirical PA and the two deterministic PAs, respectively.</p></div><div id="sec-9" class="subsection"><h3>Causal analysis of the variation in PA among traits in GP across BPFs</h3><p id="p-25">Preliminary analyses showed that PA varied substantially among traits in across-family GP for HSFs and URFs, although we assumed the same polygenic architecture for all 50 simulated traits. Therefore, we devised additional simulations to investigate the underlying cause(s), using assumptions warranting almost ideal conditions for GP to largely eliminate the influence of nuisance factors on PA. We restricted these simulations to HSFs to demonstrate the key points in a simple fashion. First, we chose at random (i) a pair of HSFs BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-219.gif"/></noscript></span></span>and BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-220.gif"/></noscript></span></span> produced from ancestral population <em>Elite</em>, and (ii) repeatedly sampled 1000 QTL positions from the entire set of 19,204 SNPs until we found a sample with <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-221.gif"/></noscript></span></span> corresponding to the average value of <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-222.gif"/></noscript></span></span> for HSF in our study (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Second, given <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-223.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-224.gif"/></noscript></span></span> and the 1000 QTL positions, we sampled 1000 sets of different QTL effects <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-225.gif"/></noscript></span></span> as described above. This resulted in 1000 traits with <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-226.gif"/></noscript></span></span> and identical QTL positions, but different QTL effects. Finally, assuming <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-227.gif"/></noscript></span></span> and known QTL genotypes, we used RR-BLUP—yielding equivalent GEBVs as GBLUP (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>)—to identify among the 1000 traits the two with lowest and highest PA and retrieved the corresponding QTL effect estimates.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487205/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487205/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean (<span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-228.gif"/></noscript></span></span>SD) of the estimated number of effective chromosome segments (<span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-229.gif"/></noscript></span></span>) and the proportion of polymorphic loci in the predicted family <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-230.gif"/></noscript></span></span> that also segregate in the training family <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-231.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-232.gif"/></noscript></span></span>) with different pedigree relationships (FSF, HSF, and URF) between <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-233.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/T2/embed/mml-math-234.gif"/></noscript></span></span> derived either from ancestral populations <em>Elite</em> or <em>Landrace</em>.</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">We surmised that variation in PA among traits arises from structural differences in the large chromosome segments containing cosegregating QTL alleles that DH lines inherit from their respective parents. To investigate this hypothesis, we analyzed the contribution of each chromosome segment along the entire genome to PA. The length of the chromosome segments within <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-238.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-239.gif"/></noscript></span></span> was taken as the expected genetic map distance at which the LD between two QTL in BPFs falls below <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-240.gif"/></noscript></span></span> (<em>cf</em>. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Giraud <em>et al.</em> 2014</a>), which amounted to <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-241.gif"/></noscript></span></span> cM (<em>cf</em>. File S3 in <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Schopp <em>et al.</em> 2017</a>). Using a sliding window approach, chromosome segments of this length moved in steps of 5 cM along each chromosome separately for each trait. Similar to <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> (2015)</a>, we subsequently calculated for each window <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-242.gif"/></noscript></span></span> the “local” TBV for all DH lines <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-243.gif"/></noscript></span></span> in the BPF<sub>pred</sub> as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-244.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-245.gif"/></noscript></span></span> is the genotypic score coded (2,0) for DH line <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-246.gif"/></noscript></span></span> at QTL <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-247.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-248.gif"/></noscript></span></span> is the corresponding QTL effect. Analogously, we calculated the local GEBV in the BPF<sub>pred</sub> as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-249.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-250.gif"/></noscript></span></span> is the estimate of <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-251.gif"/></noscript></span></span> obtained from RR-BLUP in BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-252.gif"/></noscript></span></span> provided <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-253.gif"/></noscript></span></span> segregated in <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-254.gif"/></noscript></span></span> and otherwise <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-255.gif"/></noscript></span></span> Subsequently, we calculated for each window <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-256.gif"/></noscript></span></span> the correlation between local TBVs and local GEBVs among all 500 DH lines in <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-257.gif"/></noscript></span></span></p><p id="p-27">Further, we defined <em>chromosome segment substitution effects</em> (<span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-258.gif"/></noscript></span></span>) for the parental chromosome segments of <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-259.gif"/></noscript></span></span> as the sum of allele substitution effects across all QTL <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-260.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-261.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-262.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-263.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-264.gif"/></noscript></span></span> are the parents of <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-265.gif"/></noscript></span></span> with<span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-266.gif"/></noscript></span></span> being the common parent of <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-267.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-268.gif"/></noscript></span></span> Thus, <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-269.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-270.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-271.gif"/></noscript></span></span> carry different alleles at QTL <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-272.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-273.gif"/></noscript></span></span> otherwise. Values <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-274.gif"/></noscript></span></span> were calculated analogously with respect to parents <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-275.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-276.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-277.gif"/></noscript></span></span> Note that <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-278.gif"/></noscript></span></span> if QTL <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-279.gif"/></noscript></span></span> segregates in both <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-280.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-281.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-282.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-283.gif"/></noscript></span></span> carry the same allele that is different from the allele in <span class="inline-formula" id="inline-formula-275"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-284.gif"/></noscript></span></span> In contrast, <span class="inline-formula" id="inline-formula-276"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-285.gif"/></noscript></span></span> implies that QTL <span class="inline-formula" id="inline-formula-277"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-286.gif"/></noscript></span></span> segregates in exactly one of the two HSFs <span class="inline-formula" id="inline-formula-278"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-287.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-279"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-288.gif"/></noscript></span></span> Thus, <span class="inline-formula" id="inline-formula-280"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-289.gif"/></noscript></span></span> only if <span class="inline-formula" id="inline-formula-281"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-290.gif"/></noscript></span></span> at one or more QTL <span class="inline-formula" id="inline-formula-282"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-291.gif"/></noscript></span></span> and the magnitude of this difference depends on (i) the subset of QTL <span class="inline-formula" id="inline-formula-283"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-292.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-284"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-293.gif"/></noscript></span></span> (ii) the relative size of <span class="inline-formula" id="inline-formula-285"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-294.gif"/></noscript></span></span> for each QTL in <span class="inline-formula" id="inline-formula-286"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-295.gif"/></noscript></span></span> compared with the effects of other QTL in the genome, and (iii) whether these effects have identical sign or not, which is important, especially for QTL that are closely linked. Altogether, the magnitude of <span class="inline-formula" id="inline-formula-287"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-296.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-296.gif"/></noscript></span></span> and its difference to <span class="inline-formula" id="inline-formula-288"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-297.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-297.gif"/></noscript></span></span> for each trait along the genome were expected to strongly influence the PA of GEBVs in BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-289"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-298.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-298.gif"/></noscript></span></span> estimated on the basis of BPF<sub>train</sub> <span class="inline-formula" id="inline-formula-290"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-299.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-299.gif"/></noscript></span></span></p><p id="p-28">All computations were carried out in the <em>R</em> statistical environment (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">R Core Team 2017</a>).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-29">Genotypic data of the ancestral populations is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS2.zip">File S2</a>. All R packages used for simulating the data are publicly available. All simulation steps and equations are fully described within the manuscript.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Means and variation of empirical PA</h3><p id="p-30"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> shows the distributions of empirical PA <span class="inline-formula" id="inline-formula-291"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-300.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-300.gif"/></noscript></span></span> For the standard scenario (ancestral population <em>Elite</em>, <span class="inline-formula" id="inline-formula-292"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-301.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-301.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-293"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-302.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-302.gif"/></noscript></span></span> calculated from SNP markers, <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), the mean PA (<span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-303.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-303.gif"/></noscript></span></span>) across all pairs of BPF<sub>pred</sub> and BPF<sub>train</sub> and traits was highest for FSF (0.79, Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>), and decreased by 43% for HSF (0.45) and by 60% for URF (0.32). A reverse trend was observed for the SD of <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-304.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-304.gif"/></noscript></span></span> which amounted to 0.09 for FSF and more than doubled for HSF (0.20) and URF (0.22). The 5 and 95% quantiles of <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-305.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-305.gif"/></noscript></span></span> ranged from 0.61 to 0.89 for FSF, but from <span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-306.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-306.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-298"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-307.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-307.gif"/></noscript></span></span> for HSF and from <span class="inline-formula" id="inline-formula-299"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-308.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-308.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-300"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-309.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-309.gif"/></noscript></span></span> for URF.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Boxplots of empirical prediction accuracies  in BPFs of DH lines, and (B) variance components of different factors influencing the variation of  Parents of BPFs were sampled from ancestral population Elite, and SNP markers were used to calculate the genomic relationship matrix  Results are shown for different pedigree relationships (FSF, HSF, and URF) between the predicted family (BPFpred)  and training family (BPFtrain)  as well as for different sample sizes and heritabilities " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-603183319" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;(A) Boxplots of empirical prediction accuracies &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-310&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in BPFs of DH lines, and (B) variance components of different factors influencing the variation of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-311&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Parents of BPFs were sampled from ancestral population &lt;em&gt;Elite&lt;/em&gt;, and SNP markers were used to calculate the genomic relationship matrix &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-312&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Results are shown for different pedigree relationships (FSF, HSF, and URF) between the predicted family (BPF&lt;sub&gt;pred&lt;/sub&gt;) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-313&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and training family (BPF&lt;sub&gt;train&lt;/sub&gt;) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-314&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; as well as for different sample sizes &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-315&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;and heritabilities &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-316&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3571/F1.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3571/F1.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487196" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">(A) Boxplots of empirical prediction accuracies <span class="inline-formula" id="inline-formula-301"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-310.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-310.gif"/></noscript></span></span> in BPFs of DH lines, and (B) variance components of different factors influencing the variation of <span class="inline-formula" id="inline-formula-302"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-311.gif"/></noscript></span></span> Parents of BPFs were sampled from ancestral population <em>Elite</em>, and SNP markers were used to calculate the genomic relationship matrix <span class="inline-formula" id="inline-formula-303"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-312.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-312.gif"/></noscript></span></span> Results are shown for different pedigree relationships (FSF, HSF, and URF) between the predicted family (BPF<sub>pred</sub>) <span class="inline-formula" id="inline-formula-304"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-313.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-313.gif"/></noscript></span></span> and training family (BPF<sub>train</sub>) <span class="inline-formula" id="inline-formula-305"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-314.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-314.gif"/></noscript></span></span> as well as for different sample sizes <span class="inline-formula" id="inline-formula-306"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-315.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-315.gif"/></noscript></span></span>and heritabilities <span class="inline-formula" id="inline-formula-307"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-316.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F1/embed/mml-math-316.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div><p id="p-32">For <span class="inline-formula" id="inline-formula-308"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-317.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-317.gif"/></noscript></span></span> reducing <span class="inline-formula" id="inline-formula-309"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-318.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-318.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-310"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-319.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-319.gif"/></noscript></span></span> to 25 resulted in <span class="inline-formula" id="inline-formula-311"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-320.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-320.gif"/></noscript></span></span>–<span class="inline-formula" id="inline-formula-312"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-321.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-321.gif"/></noscript></span></span> lower <span class="inline-formula" id="inline-formula-313"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-322.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-322.gif"/></noscript></span></span> and increasing <span class="inline-formula" id="inline-formula-314"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-323.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-323.gif"/></noscript></span></span> to 250 resulted in 12–18% higher <span class="inline-formula" id="inline-formula-315"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-324.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-324.gif"/></noscript></span></span> for all pedigree relationships (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). The SD increased for <span class="inline-formula" id="inline-formula-316"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-325.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-325.gif"/></noscript></span></span> by 84% for FSF, but only by <span class="inline-formula" id="inline-formula-317"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-326.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-326.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-318"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-327.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-327.gif"/></noscript></span></span> for HSF and URF, respectively, because it was already large under <span class="inline-formula" id="inline-formula-319"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-328.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-328.gif"/></noscript></span></span> For <span class="inline-formula" id="inline-formula-320"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-329.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-329.gif"/></noscript></span></span> the SD reduced by <span class="inline-formula" id="inline-formula-321"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-330.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-330.gif"/></noscript></span></span> for FSF, yet only by 6% for HSF and <span class="inline-formula" id="inline-formula-322"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-331.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-331.gif"/></noscript></span></span> for URF. Altering <span class="inline-formula" id="inline-formula-323"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-332.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-332.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-324"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-333.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-333.gif"/></noscript></span></span> affected the PA similarly as altering <span class="inline-formula" id="inline-formula-325"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-334.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-334.gif"/></noscript></span></span> under fixed <span class="inline-formula" id="inline-formula-326"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-335.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-335.gif"/></noscript></span></span> In comparison with <span class="inline-formula" id="inline-formula-327"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-336.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-336.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-328"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-337.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-337.gif"/></noscript></span></span> was reduced by<span class="inline-formula" id="inline-formula-329"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-338.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-338.gif"/></noscript></span></span>–<span class="inline-formula" id="inline-formula-330"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-339.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-339.gif"/></noscript></span></span> for<span class="inline-formula" id="inline-formula-331"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-340.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-340.gif"/></noscript></span></span> and increased by <span class="inline-formula" id="inline-formula-332"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-341.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-341.gif"/></noscript></span></span>–<span class="inline-formula" id="inline-formula-333"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-342.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-342.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-334"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-343.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-343.gif"/></noscript></span></span> depending on the pedigree relationship. The corresponding SDs changed considerably for FSF (+57 and −68%), but only marginally for HSF (8 and −11%) and URF (4 and −7%).</p><p id="p-33">Deriving BPFs from ancestral population <em>Landrace</em> instead of <em>Elite</em> generally reduced <span class="inline-formula" id="inline-formula-335"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-344.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-344.gif"/></noscript></span></span> by &lt;0.05, whereas the SD remained nearly identical (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). By comparison, calculating the <span class="inline-formula" id="inline-formula-336"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-345.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-345.gif"/></noscript></span></span> matrix from QTL instead of SNP data increased <span class="inline-formula" id="inline-formula-337"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-346.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-346.gif"/></noscript></span></span> by only 0.02, 0.03, and 0.05 for FSF, HSF, and URF, respectively, but hardly affected the SD, regardless of the pedigree relationship and the ancestral population.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Boxplots of empirical prediction accuracies  in BPFs of DH lines and (B) variance components of different factors influencing the variation of  Parents of BPFs were sampled from ancestral population Elite (left) or Landrace (right), and either genotypes at SNP markers or at QTL were used to calculate the genomic relationship matrix  Results are shown for different pedigree relationships (FSF, HSF, and URF) between the predicted family (BPFpred)  and training family (BPFtrain)  and refer to  and " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-603183319" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;(A) Boxplots of empirical prediction accuracies &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-347&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in BPFs of DH lines and (B) variance components of different factors influencing the variation of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-348&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Parents of BPFs were sampled from ancestral population &lt;em&gt;Elite&lt;/em&gt; (left) or &lt;em&gt;Landrace&lt;/em&gt; (right), and either genotypes at SNP markers or at QTL were used to calculate the genomic relationship matrix &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-349&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Results are shown for different pedigree relationships (FSF, HSF, and URF) between the predicted family (BPF&lt;sub&gt;pred&lt;/sub&gt;) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-350&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and training family (BPF&lt;sub&gt;train&lt;/sub&gt;) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-351&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and refer to &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-352&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-353&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3571/F2.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3571/F2.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487123" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">(A) Boxplots of empirical prediction accuracies <span class="inline-formula" id="inline-formula-338"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-347.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-347.gif"/></noscript></span></span> in BPFs of DH lines and (B) variance components of different factors influencing the variation of <span class="inline-formula" id="inline-formula-339"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-348.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-348.gif"/></noscript></span></span> Parents of BPFs were sampled from ancestral population <em>Elite</em> (left) or <em>Landrace</em> (right), and either genotypes at SNP markers or at QTL were used to calculate the genomic relationship matrix <span class="inline-formula" id="inline-formula-340"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-349.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-349.gif"/></noscript></span></span> Results are shown for different pedigree relationships (FSF, HSF, and URF) between the predicted family (BPF<sub>pred</sub>) <span class="inline-formula" id="inline-formula-341"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-350.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-350.gif"/></noscript></span></span> and training family (BPF<sub>train</sub>) <span class="inline-formula" id="inline-formula-342"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-351.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-351.gif"/></noscript></span></span> and refer to <span class="inline-formula" id="inline-formula-343"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-352.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-352.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-344"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-353.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F2/embed/mml-math-353.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Analysis of variance of random factors affecting the empirical PA</h3><p id="p-35">Estimates of <span class="inline-formula" id="inline-formula-345"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-354.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-354.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-346"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-355.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-355.gif"/></noscript></span></span> were of similar magnitude for HSF and URF, but generally much smaller for FSF (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). For the standard scenario, <span class="inline-formula" id="inline-formula-347"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-356.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-356.gif"/></noscript></span></span> was small for FSF (0.01) and primarily attributable to <span class="inline-formula" id="inline-formula-348"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-357.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-357.gif"/></noscript></span></span> By comparison, <span class="inline-formula" id="inline-formula-349"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-358.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-358.gif"/></noscript></span></span> was 5.3 and 6.6 times larger for HSF and URF, respectively, with &gt;50% contributed by <span class="inline-formula" id="inline-formula-350"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-359.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-359.gif"/></noscript></span></span> followed by the residual variance <span class="inline-formula" id="inline-formula-351"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-360.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-360.gif"/></noscript></span></span> (26 and 19%, respectively). All variance components not involving factor <span class="inline-formula" id="inline-formula-352"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-361.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-361.gif"/></noscript></span></span> were substantially smaller, with <span class="inline-formula" id="inline-formula-353"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-362.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-362.gif"/></noscript></span></span> contributing most for HSF (9%) and URF (6%).</p><p id="p-36">Decreasing <span class="inline-formula" id="inline-formula-354"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-363.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-363.gif"/></noscript></span></span> to 25 or <span class="inline-formula" id="inline-formula-355"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-364.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-364.gif"/></noscript></span></span> to 0.3 affected the relative importance and overall magnitude of the variance components similarly for the three pedigree relationships (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). The residual variances <span class="inline-formula" id="inline-formula-356"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-365.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-365.gif"/></noscript></span></span> (FSF) and <span class="inline-formula" id="inline-formula-357"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-366.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-366.gif"/></noscript></span></span> (HSF, URF) increased substantially, accompanied by a moderate increase in <span class="inline-formula" id="inline-formula-358"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-367.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-367.gif"/></noscript></span></span> for FSF and decrease in <span class="inline-formula" id="inline-formula-359"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-368.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-368.gif"/></noscript></span></span>for HSF and URF. Conversely, increasing <span class="inline-formula" id="inline-formula-360"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-369.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-369.gif"/></noscript></span></span> to 250 or <span class="inline-formula" id="inline-formula-361"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-370.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-370.gif"/></noscript></span></span> to 1.0 strongly reduced the residual variances and nearly eliminated <span class="inline-formula" id="inline-formula-362"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-371.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-371.gif"/></noscript></span></span> for FSF, whereas, for HSF and URF, <span class="inline-formula" id="inline-formula-363"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-372.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-372.gif"/></noscript></span></span> remained large owing to a high <span class="inline-formula" id="inline-formula-364"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-373.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-373.gif"/></noscript></span></span> even under these favorable conditions.</p><p id="p-37">Deriving BPFs from ancestral population <em>Landrace</em> instead of <em>Elite</em> had almost no effect on <span class="inline-formula" id="inline-formula-365"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-374.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-374.gif"/></noscript></span></span> and its components (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Calculating the <span class="inline-formula" id="inline-formula-366"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-375.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-375.gif"/></noscript></span></span> matrix from QTL instead of from SNP genotypes moderately reduced <span class="inline-formula" id="inline-formula-367"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-376.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-376.gif"/></noscript></span></span> by 5% for HSF and 10% for URF, mainly due to decreasing <span class="inline-formula" id="inline-formula-368"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-377.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-377.gif"/></noscript></span></span> In contrast to HSF and URF, <span class="inline-formula" id="inline-formula-369"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-378.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-378.gif"/></noscript></span></span> for FSF was already minor when using SNP genotypes, leaving less room for improvement when using QTL instead of SNP genotypes than for HSF and URF, which both showed bigger changes in the absolute magnitude of the variance components than FSF.</p></div><div id="sec-14" class="subsection"><h3>Comparison of empirical and deterministic prediction accuracies</h3><p id="p-38"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows scatter plots for empirical versus deterministic prediction accuracies for the standard scenario. In general, empirical and deterministic accuracies for single traits agreed relatively well for FSF (<span class="inline-formula" id="inline-formula-370"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-379.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-379.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-371"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-380.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-380.gif"/></noscript></span></span>), but rather weakly for HSF (<span class="inline-formula" id="inline-formula-372"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-381.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-381.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-373"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-382.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-382.gif"/></noscript></span></span>, respectively) and URF (<span class="inline-formula" id="inline-formula-374"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-383.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-383.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-375"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-384.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-384.gif"/></noscript></span></span>, respectively). By comparison, the correlations between the means of empirical and deterministic accuracies across the 50 traits increased for FSF (<span class="inline-formula" id="inline-formula-376"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-385.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-385.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-377"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-386.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-386.gif"/></noscript></span></span>), but even more so for HSF (0.94 and 0.92, respectively) and URF (0.89 and 0.88, respectively), indicating that trait-specific deviations from the mean empirical accuracy hampers the agreement with deterministic accuracies, particularly for HSF and URF.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical prediction accuracy  in BPFs of DH lines plotted against deterministic prediction accuracies  and  The top two graphs refer to observations for single traits ( for FSF and  otherwise), and the bottom row to means over traits ( for FSF and  otherwise). Parents of BPFs were sampled from ancestral population Elite and genotypes at SNP markers were used to calculate the genomic relationship matrix  Results are shown for a random sample of 10,000 data points,  and " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-603183319" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Empirical prediction accuracy &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-387&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in BPFs of DH lines plotted against deterministic prediction accuracies &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-388&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-389&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; The top two graphs refer to observations for single traits (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-390&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for FSF and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-391&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; otherwise), and the bottom row to means over traits (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-392&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for FSF and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-393&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; otherwise). Parents of BPFs were sampled from ancestral population &lt;em&gt;Elite&lt;/em&gt; and genotypes at SNP markers were used to calculate the genomic relationship matrix &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-394&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Results are shown for a random sample of 10,000 data points, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-395&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-396&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3571/F3.medium.gif" width="440" height="396"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/11/3571/F3.medium.gif" width="440" height="396"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487206" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Empirical prediction accuracy <span class="inline-formula" id="inline-formula-378"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-387.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-387.gif"/></noscript></span></span> in BPFs of DH lines plotted against deterministic prediction accuracies <span class="inline-formula" id="inline-formula-379"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-388.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-388.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-380"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-389.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-389.gif"/></noscript></span></span> The top two graphs refer to observations for single traits (<span class="inline-formula" id="inline-formula-381"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-390.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-390.gif"/></noscript></span></span> for FSF and <span class="inline-formula" id="inline-formula-382"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-391.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-391.gif"/></noscript></span></span> otherwise), and the bottom row to means over traits (<span class="inline-formula" id="inline-formula-383"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-392.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-392.gif"/></noscript></span></span> for FSF and <span class="inline-formula" id="inline-formula-384"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-393.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-393.gif"/></noscript></span></span> otherwise). Parents of BPFs were sampled from ancestral population <em>Elite</em> and genotypes at SNP markers were used to calculate the genomic relationship matrix <span class="inline-formula" id="inline-formula-385"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-394.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-394.gif"/></noscript></span></span> Results are shown for a random sample of 10,000 data points, <span class="inline-formula" id="inline-formula-386"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-395.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-395.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-387"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-396.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F3/embed/mml-math-396.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div><p id="p-40">For the general mean of empirical and deterministic PA across <span class="inline-formula" id="inline-formula-388"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-397.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-397.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-389"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-398.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-398.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-390"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-399.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-399.gif"/></noscript></span></span>matched very well with <span class="inline-formula" id="inline-formula-391"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-400.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-400.gif"/></noscript></span></span> for all pedigree relationships and values of <span class="inline-formula" id="inline-formula-392"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-401.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-401.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-393"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-402.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-402.gif"/></noscript></span></span> (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). By comparison, <span class="inline-formula" id="inline-formula-394"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-403.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-403.gif"/></noscript></span></span> generally underestimated <span class="inline-formula" id="inline-formula-395"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-404.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-404.gif"/></noscript></span></span> with increasing bias for HSF and URF as compared with <span class="inline-formula" id="inline-formula-396"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-405.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-405.gif"/></noscript></span></span> (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>), and particularly for smaller values of <span class="inline-formula" id="inline-formula-397"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-406.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-406.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-398"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-407.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-407.gif"/></noscript></span></span> (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). Calculating the <span class="inline-formula" id="inline-formula-399"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-408.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-408.gif"/></noscript></span></span> matrix from QTL instead of from SNP genotypes hardly influenced the bias of deterministic accuracies (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>) and the correlations with empirical accuracies.</p></div><div id="sec-15" class="subsection"><h3>Causal analysis of the variation in PA among traits</h3><p id="p-41"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> compares two traits <em>T1</em> and <em>T2</em> with divergent PA for one representative pair of HSFs. For both traits with identical QTL positions and QTL genotypes in the BPF<sub>pred</sub> <span class="inline-formula" id="inline-formula-400"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-409.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-409.gif"/></noscript></span></span> and BPF<sub>train</sub> <em>B</em>, but different QTL effects, 376 QTL segregated in <span class="inline-formula" id="inline-formula-401"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-410.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-410.gif"/></noscript></span></span> 286 in <span class="inline-formula" id="inline-formula-402"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-411.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-411.gif"/></noscript></span></span> and 151 of them jointly in <span class="inline-formula" id="inline-formula-403"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-412.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-412.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-404"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-413.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-413.gif"/></noscript></span></span> For trait <em>T1</em> with high <span class="inline-formula" id="inline-formula-405"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-414.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-414.gif"/></noscript></span></span> the differences between chromosome segment substitution effects (<em>CSSE</em>) in <span class="inline-formula" id="inline-formula-406"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-415.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-415.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-407"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-416.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-416.gif"/></noscript></span></span> were generally small across the entire genome, in particular on chromosomes 2, 3, and 9, with sizeable <em>CSSEs</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Conversely, for trait <em>T2</em> with low <span class="inline-formula" id="inline-formula-408"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-417.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-417.gif"/></noscript></span></span> the <em>CSSEs</em> in <span class="inline-formula" id="inline-formula-409"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-418.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-418.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-410"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-419.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-419.gif"/></noscript></span></span> differed substantially over large parts of the genome, and showed even opposite signs on several chromosomes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Chromosome segment substitution effects (CSSEA,W in red and CSSEB,W in blue) and correlation between local TBVs and local GEBVs in the predicted family  (green) averaged in sliding windows  (see Materials and Methods for definition). GEBVs were calculated from QTL effects estimated by RR-BLUP in training set (HSF)  Results are shown for  and two traits T1 and T2 with  and large differences in prediction accuracy  Both traits were generated from the same set of 1000 QTL with  but different QTL effects. (B) Correlation between local TBVs and local GEBVs (green lines) shown together with true QTL effects (diamonds) and estimated QTL effects (circles) for T1 and T2 in  on chromosome&#xA0;5. Colors indicate QTL segregating in both  and  (orange) or only in  (purple); grey bars in the background reflect the windows " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-603183319" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;(A) Chromosome segment substitution effects (&lt;em&gt;CSSE&lt;/em&gt;&lt;em&gt;&lt;sub&gt;A&lt;/sub&gt;&lt;/em&gt;,&lt;em&gt;&lt;sub&gt;W&lt;/sub&gt;&lt;/em&gt; in red and &lt;em&gt;CSSE&lt;/em&gt;&lt;em&gt;&lt;sub&gt;B&lt;/sub&gt;&lt;/em&gt;,&lt;em&gt;&lt;sub&gt;W&lt;/sub&gt;&lt;/em&gt; in blue) and correlation between local TBVs and local GEBVs in the predicted family &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-420&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green) averaged in sliding windows &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-421&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (see &lt;em&gt;Materials and Methods&lt;/em&gt; for definition). GEBVs were calculated from QTL effects estimated by RR-BLUP in training set (HSF) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-422&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Results are shown for &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-423&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and two traits &lt;em&gt;T1&lt;/em&gt; and &lt;em&gt;T&lt;/em&gt;2 with &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-424&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and large differences in prediction accuracy &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-425&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Both traits were generated from the same set of 1000 QTL with &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-426&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; but different QTL effects. (B) Correlation between local TBVs and local GEBVs (green lines) shown together with true QTL effects (diamonds) and estimated QTL effects (circles) for &lt;em&gt;T1&lt;/em&gt; and &lt;em&gt;T&lt;/em&gt;2 in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-427&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; on chromosome&#xA0;5. Colors indicate QTL segregating in both &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-428&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-429&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (orange) or only in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-430&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (purple); grey bars in the background reflect the windows &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-431&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3571/F4.medium.gif" width="269" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/11/3571/F4.medium.gif" width="269" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3571/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487222" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">(A) Chromosome segment substitution effects (<em>CSSE</em><em><sub>A</sub></em>,<em><sub>W</sub></em> in red and <em>CSSE</em><em><sub>B</sub></em>,<em><sub>W</sub></em> in blue) and correlation between local TBVs and local GEBVs in the predicted family <span class="inline-formula" id="inline-formula-411"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-420.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-420.gif"/></noscript></span></span> (green) averaged in sliding windows <span class="inline-formula" id="inline-formula-412"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-421.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-421.gif"/></noscript></span></span> (see <em>Materials and Methods</em> for definition). GEBVs were calculated from QTL effects estimated by RR-BLUP in training set (HSF) <span class="inline-formula" id="inline-formula-413"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-422.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-422.gif"/></noscript></span></span> Results are shown for <span class="inline-formula" id="inline-formula-414"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-423.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-423.gif"/></noscript></span></span> and two traits <em>T1</em> and <em>T</em>2 with <span class="inline-formula" id="inline-formula-415"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-424.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-424.gif"/></noscript></span></span> and large differences in prediction accuracy <span class="inline-formula" id="inline-formula-416"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-425.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-425.gif"/></noscript></span></span> Both traits were generated from the same set of 1000 QTL with <span class="inline-formula" id="inline-formula-417"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-426.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-426.gif"/></noscript></span></span> but different QTL effects. (B) Correlation between local TBVs and local GEBVs (green lines) shown together with true QTL effects (diamonds) and estimated QTL effects (circles) for <em>T1</em> and <em>T</em>2 in <span class="inline-formula" id="inline-formula-418"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-427.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-427.gif"/></noscript></span></span> on chromosome 5. Colors indicate QTL segregating in both <span class="inline-formula" id="inline-formula-419"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-428.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-428.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-420"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-429.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-429.gif"/></noscript></span></span> (orange) or only in <span class="inline-formula" id="inline-formula-421"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-430.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-430.gif"/></noscript></span></span> (purple); grey bars in the background reflect the windows <span class="inline-formula" id="inline-formula-422"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-431.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/F4/embed/mml-math-431.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The correlation between local TBVs and local GEBVs of the DH lines <span class="inline-formula" id="inline-formula-423"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-432.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-432.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-424"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-433.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-433.gif"/></noscript></span></span> were closely associated with the differences between the <em>CSSEs</em> for <span class="inline-formula" id="inline-formula-425"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-434.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-434.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-426"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-435.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-435.gif"/></noscript></span></span> in the corresponding windows <span class="inline-formula" id="inline-formula-427"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-436.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-436.gif"/></noscript></span></span> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). If the difference in the <em>CSSE</em> for a segment was small, the correlation was generally high, particularly if both <em>CSSEs</em> in <span class="inline-formula" id="inline-formula-428"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-437.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-437.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-429"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-438.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-438.gif"/></noscript></span></span> had large magnitude and identical sign (see chromosomes 2, 3 and 9 for trait <em>T1</em>). Conversely, if the <em>CSSEs</em> for a window <span class="inline-formula" id="inline-formula-430"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-439.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-439.gif"/></noscript></span></span> differed and had opposite sign in <span class="inline-formula" id="inline-formula-431"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-440.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-440.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-432"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-441.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-441.gif"/></noscript></span></span> the correlation between local TBV and local GEBV dropped substantially, and frequently became negative (see chromosomes 2, 5, and 8 for trait <em>T2</em>). Overall, the proportion of the genome showing low or even negative correlations was much smaller for trait <em>T1</em> with high PA than for trait <em>T2</em> with low PA.</p><p id="p-44">Zooming into chromosome 5—which had a large impact on the differences between the two traits—revealed that for trait <em>T1</em>, all large-effect QTL that segregated in <span class="inline-formula" id="inline-formula-433"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-442.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-442.gif"/></noscript></span></span> also segregated in <span class="inline-formula" id="inline-formula-434"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-443.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-443.gif"/></noscript></span></span> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). However, for trait <em>T2</em>, there was a large-effect QTL that segregated only in <span class="inline-formula" id="inline-formula-435"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-444.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-444.gif"/></noscript></span></span> in windows <span class="inline-formula" id="inline-formula-436"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-445.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-445.gif"/></noscript></span></span> with low correlation between local TBVs and local GEBVs. Neighboring windows not harboring this QTL showed higher correlations. The trends for this exemplary chromosome were consistent with other chromosomes and other HSF pairs <span class="inline-formula" id="inline-formula-437"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-446.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-446.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-438"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-447.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-447.gif"/></noscript></span></span> as well as other traits with high and low PA (results not shown).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-45">Experimental studies showed that PA can be highly variable for GP within, but even more so across BPFs. Moreover, PA was found to vary substantially among different target traits for distinct pairs of training and predicted families. Investigating the causes for this variability is hardly possible based on experimental data due to the limited number and sample size of available BPFs, and the generally unknown genetic architecture of agronomically important traits. Here, we used computer simulations to analyze in detail why PA varies among different combinations of training sets, prediction sets, and polygenic traits. Moreover, we demonstrate that modification of available deterministic equations enables accurate estimates of PA averaged across many polygenic traits for both within-family GP and across-family GP.</p><div id="sec-17" class="subsection"><h3>Variation in PA within and across biparental families</h3><p id="p-46">The average PA decreased under small <span class="inline-formula" id="inline-formula-439"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-448.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-448.gif"/></noscript></span></span> and low <span class="inline-formula" id="inline-formula-440"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-449.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-449.gif"/></noscript></span></span> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>) for all pedigree relationships, as expected from theory (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Daetwyler <em>et al.</em> 2008</a>). This was always accompanied by a large increase in the variation of PA (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>), which was mainly caused by inflated residual errors [<span class="inline-formula" id="inline-formula-441"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-450.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-450.gif"/></noscript></span></span> for FSF, <span class="inline-formula" id="inline-formula-442"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-451.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-451.gif"/></noscript></span></span> for HSF and URF, <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>]. These errors capture the variation in PA that arises due to the random sampling of (i) individuals (genotypes) from the BPF<sub>train</sub>, and (ii) their corresponding phenotypes for a specific trait. The larger residual errors in across-family GP are presumably due to incongruent sets of QTL segregating in pairs of HSFs and URFs, which can vary substantially across traits, as reflected by the SD of <span class="inline-formula" id="inline-formula-443"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-452.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-452.gif"/></noscript></span></span> (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The fact that predictions became much more robust under <span class="inline-formula" id="inline-formula-444"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-453.gif.backup.1540846410.5131"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-453.gif.backup.1540846410.5131"/></noscript></span></span>100 and <span class="inline-formula" id="inline-formula-445"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-454.gif.backup.1540846410.5599"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-454.gif.backup.1540846410.5599"/></noscript></span></span> illustrate that large sample sizes and heritabilities are mandatory to alleviate the trait-specific sampling variance in PA. Together with the generally optimal conditions in within-family GP (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2014</a>), this nearly eliminated all variation in PA for FSF (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-47">The predicted family BPF<sub>pred</sub> accounted only for a marginal proportion of variation in PA, irrespective of the pedigree relationship with BPF<sub>train</sub> (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1B</a>, <span class="inline-formula" id="inline-formula-446"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-455.gif.backup.1540846410.5939"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-455.gif.backup.1540846410.5939"/></noscript></span></span>). For within-family GP (where BPF<sub>train</sub> = BPF<sub>pred</sub>), this implies that the genetic distance between the parents of a BPF has at best marginal influence on the average PA across traits, in agreement with previous studies (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Marulanda <em>et al.</em> 2015</a>). This conclusion is further supported by the similar variation in PA among predicted families derived from the two ancestral populations (<span class="inline-formula" id="inline-formula-447"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-456.gif.backup.1540846410.6414"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-456.gif.backup.1540846410.6414"/></noscript></span></span> <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, FSF), despite the much weaker latent pedigree structure in <em>Landrace</em> compared with <em>Elite</em> (Figure S1B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). By comparison, the generally substantial influence of <span class="inline-formula" id="inline-formula-448"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-457.gif.backup.1540846410.6744"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-457.gif.backup.1540846410.6744"/></noscript></span></span> in FSF (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1B</a> and <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>) suggests that PA strongly depends on <span class="inline-formula" id="inline-formula-449"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-458.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-458.gif"/></noscript></span></span> in the training set (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>), which can be highly variable among BPF × trait combinations (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). This is in harmony with previous studies that attributed variation in PA partially to differences in the phenotypic variance of the training set (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Marulanda <em>et al.</em> 2015</a>).</p><p id="p-48">For across-family GP, the expected PA depends largely on the pedigree relationship (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>) and on the variation in across-family genomic relationships. Since genomic relationships across families have a zero mean (if calculated according to Equation 1), their variation is equal to the mean squared genomic relationship between training and predicted individuals (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Wientjes <em>et al.</em> 2013</a>). Generally, PA is expected to increase proportionally with these squared relationships. In the case of BPFs, genomic relationships between families are heavily influenced by the proportion of polymorphic markers in the BPF<sub>pred</sub> (<span class="inline-formula" id="inline-formula-450"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-459.gif.backup.1540846410.7037"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-459.gif.backup.1540846410.7037"/></noscript></span></span>) segregating also in the BPF<sub>train</sub> (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). Therefore, PA for across-family GP depends primarily on the magnitude of <span class="inline-formula" id="inline-formula-451"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-460.gif.backup.1540846410.7262"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-460.gif.backup.1540846410.7262"/></noscript></span></span> because larger <span class="inline-formula" id="inline-formula-452"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-461.gif.backup.1540846410.7428"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-461.gif.backup.1540846410.7428"/></noscript></span></span> implies that a greater proportion of the genetic variance in the BPF<sub>pred</sub> can be explained by the QTL in BPF<sub>train</sub>. Accordingly, the variation in <span class="inline-formula" id="inline-formula-453"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-462.gif.backup.1540846410.7756"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-462.gif.backup.1540846410.7756"/></noscript></span></span> among combinations of different HSFs or URFs (Figure S1D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>) was largely responsible for the notable contribution of <span class="inline-formula" id="inline-formula-454"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-463.gif.backup.1540846410.7933"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-463.gif.backup.1540846410.7933"/></noscript></span></span> to the total variation in PA (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1B</a>). Altogether, the much larger <span class="inline-formula" id="inline-formula-455"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-464.gif.backup.1540846410.8223"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-464.gif.backup.1540846410.8223"/></noscript></span></span> for across-family GP, compared to within-family GP, was mainly due to the overriding influence of <span class="inline-formula" id="inline-formula-456"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-465.gif.backup.1540846410.8468"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-465.gif.backup.1540846410.8468"/></noscript></span></span> besides the considerable contribution of <span class="inline-formula" id="inline-formula-457"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-466.gif.backup.1540846410.8769"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-466.gif.backup.1540846410.8769"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-458"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-467.gif.backup.1540846410.9107"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-467.gif.backup.1540846410.9107"/></noscript></span></span> (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1B</a>, FSF <em>vs.</em> HSF or URF). Unraveling the genetic causes for this complex interaction required additional analyses, which are discussed in depth in the next section.</p><p id="p-49">Sampling of training individuals from a given BPF<sub>train</sub> barely contributed to the variation in PA, for both within- and across-family GP (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1B</a>, <span class="inline-formula" id="inline-formula-459"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-468.gif.backup.1540846410.9376"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-468.gif.backup.1540846410.9376"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-460"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-469.gif.backup.1540846410.9663"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-469.gif.backup.1540846410.9663"/></noscript></span></span>). Thus, compared with structured populations or diversity panels, there is little room for improvement by applying optimization algorithms accounting for genomic relationships in the sampling of training individuals within BPFs (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bustos-Korts <em>et al.</em> 2016</a>), confirming previous findings (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lorenz and Smith 2015</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Marulanda <em>et al.</em> 2015</a>). This is because already modest sample sizes (<em>e.g.</em>, <em><span class="inline-formula" id="inline-formula-461"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-470.gif.backup.1540846410.9917"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-470.gif.backup.1540846410.9917"/></noscript></span></span></em>) enable the Mendelian sampling term in the BPF<sub>train</sub> to be sufficiently captured. Nevertheless, we recommend <span class="inline-formula" id="inline-formula-462"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-471.gif.backup.1540846411.1062"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-471.gif.backup.1540846411.1062"/></noscript></span></span> to achieve a high mean and small variance of PA (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>) arising from sampling of genotypes from a given BPF<sub>train</sub> (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-50">Previous experimental studies found generally higher levels of variation in PA, particularly for within-family GP (<a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>). This is most likely attributable to miscellaneous additional factors present in these studies, which were not accounted for in our simulations. These factors include (i) small prediction set size, (ii) analysis of different types of progeny (F<sub>2</sub> or backcross generations and DH lines derived from them), (iii) variation in QTL-SNP LD within BPFs due to low marker density, (iv) nonadditive gene action due to epistasis, and (v) estimation error in <span class="inline-formula" id="inline-formula-463"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-472.gif.backup.1540846411.1262"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-472.gif.backup.1540846411.1262"/></noscript></span></span> which affects calculation of PA from predictive ability. Further, the various agronomic traits investigated in the experimental studies differed likely in their genetic architecture, which further increases the total variation in PA compared with the polygenic traits simulated in our study (<span class="inline-formula" id="inline-formula-464"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-473.gif.backup.1540846411.2144"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-473.gif.backup.1540846411.2144"/></noscript></span></span> <a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1B</a>). Consequently, our results should be regarded as a lower bound for the variation in PA that must be expected in practice for a given <span class="inline-formula" id="inline-formula-465"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-474.gif.backup.1540846411.2398"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-474.gif.backup.1540846411.2398"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-466"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-475.gif.backup.1540846411.2783"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-475.gif.backup.1540846411.2783"/></noscript></span></span></p></div><div id="sec-18" class="subsection"><h3>Unraveling the variation among traits in across-family GP</h3><p id="p-51">We adopted the concept of local breeding values (<em>cf</em>. <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2015</a>) to investigate the relationship between the strong variation in PA among traits and the large chromosome segments that DH lines of BPF inherit from their parents. The latter entails strong LD between QTL alleles and consequently small <span class="inline-formula" id="inline-formula-467"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-476.gif.backup.1540846411.3156"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-476.gif.backup.1540846411.3156"/></noscript></span></span> (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), which is very different from the situation found in diverse populations such as cattle breeds (<span class="inline-formula" id="inline-formula-468"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-477.gif.backup.1540846411.3533"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-477.gif.backup.1540846411.3533"/></noscript></span></span>) (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Wientjes <em>et al.</em> 2013</a>). Thus, only a small number of local TBVs contribute to the “global” TBV of predicted individuals. Similarly, the PA can be thought of as the average accuracy of local GEBVs estimated from the training data, weighted by their relative contribution to the global TBV in the BPF<sub>pred</sub>. As a consequence of the small <span class="inline-formula" id="inline-formula-469"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-478.gif.backup.1540846411.378"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-478.gif.backup.1540846411.378"/></noscript></span></span> in BPFs, the accuracy of local GEBVs is prone to much larger sample variance than would be the case in more diverse populations. To illustrate this point, we examined for a given pair of HSFs exemplarily two traits with contrasting PA (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-52">Of all QTL, only those that segregated in the BPF<sub>pred</sub> (376/1, 000, <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>) contributed to the variance in local TBVs, which were estimated by local GEBVs from the training set. In our example, trait <span class="inline-formula" id="inline-formula-470"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-479.gif.backup.1540846411.4226"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-479.gif.backup.1540846411.4226"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-471"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-480.gif.backup.1540846411.4482"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-480.gif.backup.1540846411.4482"/></noscript></span></span> showed, on average, much higher correlations between local TBVs and local GEBVs in the BPF<sub>pred</sub> along the entire genome than trait <span class="inline-formula" id="inline-formula-472"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-481.gif.backup.1540846411.4784"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-481.gif.backup.1540846411.4784"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-473"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-482.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-482.gif"/></noscript></span></span> (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>). For the trait with low PA, we found a larger proportion of local GEBVs that provided a false prediction signal, in the sense that negative effects were estimated for favorable parental chromosome segments and vice versa. These discrepancies between local TBVs and local GEBVs trace back to different chromosome segment substitution effects (<em>CSSE</em>, Equation 9) between the BPF<sub>pred</sub> and BPF<sub>train</sub> (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4A</a>), which, in the case of HSFs, occur if their noncommon parent carries different alleles at one or more QTL on the segment. If this is the case, one of the two BPFs will be monomorphic for the respective QTL. The effect of such a QTL compared with other QTL on a chromosome segment that may be polymorphic in both the BPF<sub>pred</sub> and BPF<sub>train</sub> determines the difference in <em>CSSE</em> between two families. For instance, if the variance in local TBVs among predicted individuals is dominated by a large-effect QTL, which is monomorphic in the training set, the ranking of local GEBVs based on the other polymorphic QTL located on this segment might deviate substantially from the ranking of local TBVs, resulting in low local PA (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4B</a>, <span class="inline-formula" id="inline-formula-474"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-483.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-483.gif"/></noscript></span></span>). The frequency of inaccurate local GEBVs along the whole genome together with the variance explained by the corresponding local TBVs will finally determine the PA of across-family GP. Hence, two traits with the same number and positions of QTL might have very different PA, depending on the effects of QTL that are poly- or monomorphic across the training and prediction set. This explains also why <span class="inline-formula" id="inline-formula-475"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-484.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-484.gif"/></noscript></span></span> and thereby across-family genomic relationships, were closely associated with the average PA across many traits for different pairs of HSF and URF (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>), but poorly associated with PA <span class="inline-formula" id="inline-formula-476"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-485.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-485.gif"/></noscript></span></span> for individual traits (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Additional simulations showed further that reducing (i) the number of chromosomes on which QTL were located, or (ii) the total number of QTL, results in increased variation in PA (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). Both these alterations reduce the number of local TBVs discernible for a trait, which underlines the relevance of small <span class="inline-formula" id="inline-formula-477"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-486.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-486.gif"/></noscript></span></span> (<em>i.e.</em>, a low number of segments carrying QTL) for the variation in PA.</p><p id="p-53">In conclusion, the large variation in PA among traits observed for across-family GP is caused by the strong LD among linked QTL within BPFs, and the resulting small effective number of chromosome segments contributing to polygenic traits, in combination with different QTL segregating across BPFs. Our analyses exemplify that BPFs represent a special case regarding the possibly strong fluctuations in PA, which is—to this extent—not expected for genetically more diverse populations.</p></div><div id="sec-19" class="subsection"><h3>Influence of LD in the ancestral population on the expected accuracy of GP across BPFs</h3><p id="p-54">Differences in the extent of LD in ancestral populations <em>Elite</em> and <em>Landrace</em> (Figure S1A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>) translated into sizable differences in QTL-SNP linkage phase similarity among URFs derived from these populations (Figure S1C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). Surprisingly, this barely affected <span class="inline-formula" id="inline-formula-478"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-487.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-487.gif"/></noscript></span></span> across URFs (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a> and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). The low relevance of linkage phase similarity across URFs was confirmed by the similar PAs when substituting the SNP- with a QTL-derived <span class="inline-formula" id="inline-formula-479"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-488.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-488.gif"/></noscript></span></span> matrix (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>), which eliminates the influence of this factor. This reflects most likely the overriding influence of <span class="inline-formula" id="inline-formula-480"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-489.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-489.gif"/></noscript></span></span> on PA across URFs, because the mean <span class="inline-formula" id="inline-formula-481"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-490.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-490.gif"/></noscript></span></span> was similar for URFs derived from the two ancestral populations (Figure S1D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). Thus, the higher mean in PA for HSFs compared with URFs seems to be attributable to higher <span class="inline-formula" id="inline-formula-482"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-491.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-491.gif"/></noscript></span></span> values (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) rather than to the fact that QTL-SNP linkage phases are always consistent across HSF (<a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>), but not necessarily across URF. This corrects a conjecture of <a id="xref-ref-49-9" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> (2013)</a>, who suspected that low PA obtained from certain URFs was due to low linkage phase similarity with the respective BPF<sub>pred</sub>.</p></div><div id="sec-20" class="subsection"><h3>Deterministic equations for forecasting PA within and across BPFs</h3><p id="p-55">Forecasting PA based on estimated reliabilities of GEBVs requires that unrelated individuals have an expected genomic relationship of zero (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> 2011</a>; <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> 2015</a>). This can be achieved by a block-structured <span class="inline-formula" id="inline-formula-483"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-492.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-492.gif"/></noscript></span></span> matrix based on population-specific allele frequencies (<em>e.g.</em>, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2013</a>). Preliminary analyses showed that in the calculation of <span class="inline-formula" id="inline-formula-484"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-493.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-493.gif"/></noscript></span></span> (Equation A5), correct treatment of SNPs polymorphic only in either BPF<sub>train</sub> or in BPF<sub>pred</sub> is very important. Different from empirical PAs, which remain unaffected by <span class="inline-formula" id="inline-formula-485"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-494.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-494.gif"/></noscript></span></span> (see Appendix A), deterministic PAs across BPFs can be grossly inflated by ignoring <span class="inline-formula" id="inline-formula-486"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-495.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-495.gif"/></noscript></span></span> in the calculation of <span class="inline-formula" id="inline-formula-487"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-496.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-496.gif"/></noscript></span></span> (results not shown). While <span class="inline-formula" id="inline-formula-488"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-497.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-497.gif"/></noscript></span></span> is generally high across diverse populations such as breeds of cattle (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Matukumalli <em>et al.</em> 2009</a>), it can fall to &lt;0.4 across different BPFs produced from inbred parents in plant breeding (Figure S1D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a> and <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Calculating <span class="inline-formula" id="inline-formula-489"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-498.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-498.gif"/></noscript></span></span> according to our improved method (Equation 1) largely eliminated the bias in deterministic accuracies <span class="inline-formula" id="inline-formula-490"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-499.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-499.gif"/></noscript></span></span>attributable to <span class="inline-formula" id="inline-formula-491"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-500.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-500.gif"/></noscript></span></span> and is therefore a prerequisite for applying Equation 3 to GP across BPFs.</p><p id="p-56">Accounting for inbreeding (see Appendix B for derivation) in the original reliability equation, resulted together with the modifications on the <span class="inline-formula" id="inline-formula-492"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-501.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-501.gif"/></noscript></span></span> matrix in excellent agreement between empirical and deterministic accuracies <span class="inline-formula" id="inline-formula-493"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-502.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-502.gif"/></noscript></span></span> averaged across traits, which is consistent with the findings of <a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> (2015)</a> for cattle populations. However, the trait-dependent variation in empirical PA observed for GP across BPFs cannot be accounted for by <span class="inline-formula" id="inline-formula-494"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-503.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-503.gif"/></noscript></span></span> This is because for a given set of training and predicted individuals and two traits with the same <span class="inline-formula" id="inline-formula-495"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-504.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-504.gif"/></noscript></span></span> but different QTL effects, the deterministic accuracy would be identical yet the empirical accuracy can differ substantially as illustrated in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>.</p><p id="p-57">Forecasting PA within FSF by <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Daetwyler <em>et al.</em>’s (2008</a>, <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">2010</a>) equation based on population parameters has been widely used in plant breeding (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Lorenz 2013</a>; <a id="xref-ref-49-10" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>). However, estimates of <span class="inline-formula" id="inline-formula-496"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-505.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-505.gif"/></noscript></span></span> can differ substantially (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Riedelsheimer and Melchinger 2013</a>; <a id="xref-ref-55-7" class="xref-bibr" href="#ref-55">Wientjes <em>et al.</em> 2013</a>) between the various proposed formulas to estimate <span class="inline-formula" id="inline-formula-497"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-506.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-506.gif"/></noscript></span></span> from the effective population size <span class="inline-formula" id="inline-formula-498"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-507.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-507.gif"/></noscript></span></span> and genome length (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Goddard 2009</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Meuwissen and Goddard 2010</a>; <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> 2011</a>). Moreover, estimation of <span class="inline-formula" id="inline-formula-499"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-508.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-508.gif"/></noscript></span></span> itself is problematic, because it assumes a base population of unrelated founders, which is often impossible to define in practice (<em>cf</em>. Figure S1B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>, <em>Elite</em>). Following <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> (2011)</a>, we calculated <span class="inline-formula" id="inline-formula-500"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-509.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-509.gif"/></noscript></span></span> directly from the variance of genomic relationships, with extensions devised by <a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> (2015</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">2016</a>) for GP across populations (Equation 5). This has the advantage that <span class="inline-formula" id="inline-formula-501"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-510.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-510.gif"/></noscript></span></span> is computed from the actual genotypes for which the PA is to be forecasted. The calculation of <span class="inline-formula" id="inline-formula-502"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-511.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-511.gif"/></noscript></span></span> required in Equation 4 must account for inbreeding (Equation 6), because the variance in genomic relationships increases with the inbreeding coefficient <span class="inline-formula" id="inline-formula-503"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-512.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-512.gif"/></noscript></span></span> (<em>see</em> Appendix C). Ignoring inbreeding would result in underestimation of <span class="inline-formula" id="inline-formula-504"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-513.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-513.gif"/></noscript></span></span> and strong overestimation of the deterministic accuracy <span class="inline-formula" id="inline-formula-505"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-514.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-514.gif"/></noscript></span></span></p><p id="p-58">An important assumption of the equation of Daetwyler <em>et al.</em> is that the entire genetic variance in the prediction set is explained by QTL segregating in the training set (<em>cf. <span class="inline-formula" id="inline-formula-506"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-515.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-515.gif"/></noscript></span></span></em> in <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Wientjes <em>et al.</em> 2016</a>). This holds true for FSF (<span class="inline-formula" id="inline-formula-507"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-516.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-516.gif"/></noscript></span></span>), but is violated for GP across BPFs (<span class="inline-formula" id="inline-formula-508"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-517.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-517.gif"/></noscript></span></span><a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). As a solution for this problem, we propose multiplication with <span class="inline-formula" id="inline-formula-509"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-518.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-518.gif"/></noscript></span></span> in calculating <span class="inline-formula" id="inline-formula-510"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-519.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-519.gif"/></noscript></span></span> (Equation 4), which efficiently reduced the strong upward-bias observed otherwise (results not shown). With these modifications, empirical and deterministic accuracies <span class="inline-formula" id="inline-formula-511"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-520.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-520.gif"/></noscript></span></span> agreed reasonably well when averaged across traits, but forecasting was problematic for individual traits for the same reasons as discussed above for <span class="inline-formula" id="inline-formula-512"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-521.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-521.gif"/></noscript></span></span> (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Compared with previous experimental studies (<a id="xref-ref-49-11" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>), we found overall better agreement of <span class="inline-formula" id="inline-formula-513"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-522.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-522.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-514"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-523.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-523.gif"/></noscript></span></span> for single traits in within-family GP (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). We suppose that, in addition to the lower variation in empirical PA (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1</a>), this is likely attributable to smaller deviations between estimated and true <span class="inline-formula" id="inline-formula-515"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-524.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-524.gif"/></noscript></span></span> (<a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>) when dealing with real traits of diverse genetic architecture.</p><p id="p-59">An upward bias in deterministic PA must generally be expected if SNPs are not a good approximation of QTL due to incomplete QTL-SNP LD, (<em>cf. <span class="inline-formula" id="inline-formula-516"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-525.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-525.gif"/></noscript></span></span></em> <em>vs.</em> <span class="inline-formula" id="inline-formula-517"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-526.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-526.gif"/></noscript></span></span> in <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Wientjes <em>et al.</em> 2016</a>), leading to “missing heritability” in genomic studies (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yang <em>et al.</em> 2010</a>). This is because empirical PA decreases as less variance at QTL is explained by SNPs under incomplete LD, whereas deterministic PA is hardly affected (Figure S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). However, our results show that this is barely relevant in BPFs (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a> <em>vs.</em> Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>), if large chromosome segments are covered sufficiently by markers. Thus, a sizable reduction in empirical PA and overestimation of deterministic PA must only be expected under very low marker density (&lt;100 SNPs) as in the study of <a id="xref-ref-32-9" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> (2014)</a>. Although these authors argued that 100 SNPs were likely sufficient for within-family GP in maize, our results indicate that at least 1000 and 2500 SNPs should be used for within- and across-family GP, respectively, to obtain acceptable empirical PA and minimize the bias in deterministic PA (Figure S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). If such numbers are not available, deterministic equations must additionally account for incomplete LD (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Wientjes <em>et al.</em> 2016</a>), using, for example, multiplication with the average LD (<span class="inline-formula" id="inline-formula-518"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-527.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-527.gif"/></noscript></span></span>) between adjacent markers as proxy for the QTL-SNP LD (<a id="xref-ref-32-10" class="xref-bibr" href="#ref-32">Lian <em>et al.</em> 2014</a>).</p><p id="p-60">Besides low marker density, incomplete QTL-SNP LD can result from differences in the allele frequency distribution at QTL and SNPs (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> 2011</a>), <em>inter alia</em> due to ascertainment bias of SNP chips. These differences are in reality unknown, and, as treated herein, commonly not accounted for in simulation studies (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daetwyler <em>et al.</em> 2013</a>). For GP across BPFs, differences in allele frequencies at QTL and SNPs in the ancestral population (<em>cf</em>. Figure S1E in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>) would translate into different <span class="inline-formula" id="inline-formula-519"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-528.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-528.gif"/></noscript></span></span> values at SNPs and QTL across BPFs, because the smaller the minor allele frequency, the larger the chances of a locus being monomorphic in a BPF. Thus, calculation of <span class="inline-formula" id="inline-formula-520"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-529.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-529.gif"/></noscript></span></span> might be inflated by an upward-bias in <span class="inline-formula" id="inline-formula-521"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-530.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-530.gif"/></noscript></span></span> (Equation 5), in addition to the possible overestimation of across-family genomic relationships affecting both <span class="inline-formula" id="inline-formula-522"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-531.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-531.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-523"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-532.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-532.gif"/></noscript></span></span> (Equations 3 and 4). Further research is needed to show how strongly overestimation of <span class="inline-formula" id="inline-formula-524"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-533.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-533.gif"/></noscript></span></span> can affect application of deterministic equations in practice, for example, by comparing the equations under chip-based and sequencing-based genotyping (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pérez-Enciso <em>et al.</em> 2015</a>).</p><p id="p-61">We assumed in our derivations that the genetic correlation among BPFs = 1 (<em>see</em> Appendix B), which is expected to hold under a purely additive-genetic model, as applies in the absence of epistasis to (i) testcross performance for a given tester, and (ii) to <em>per se</em> performance of completely homozygous lines (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Melchinger 1987</a>). By comparison, in cattle breeds or diverse germplasm in plant breeding, genetic correlations between populations are typically &lt; 1 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Karoui <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lehermeier <em>et al.</em> 2015</a>). Accounting for genetic correlations is possible with multi-group models, but these require sufficient phenotypic data for the predicted population as well as estimating these correlations, which seems impractical in the case of GP of a single BPF.</p><p id="p-62">Despite generally promising results for both deterministic equations, we recommend using <span class="inline-formula" id="inline-formula-525"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-534.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-534.gif"/></noscript></span></span> (Equation 3), because it depended less on the relatedness between BPFs, <span class="inline-formula" id="inline-formula-526"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-535.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-535.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-527"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-536.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-536.gif"/></noscript></span></span> (Figures S2 and S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>), rendering it more robust across a wide range of scenarios. Since <span class="inline-formula" id="inline-formula-528"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-537.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-537.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-529"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-538.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-538.gif"/></noscript></span></span> (as implemented here) require genotypic data of both the training and predicted individuals, they can be applied only after obtaining genotypic data of the individuals to be predicted. Alternatively, for newly planned crosses we propose to use computer simulations to generate <em>in silico</em> virtual genotypic data of the corresponding BPFs using known genotypes of the parents and genetic map information of the markers, as conducted in this study (<em>cf</em>. <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mohammadi <em>et al.</em> 2015</a>). This would make both equations accessible prior to generating new crosses for use in optimizing training set designs and allocation of resources.</p></div><div id="sec-21" class="subsection"><h3>Conclusions and extensions to multi-family training sets</h3><p id="p-63">We demonstrated that the empirical PA in BPFs of inbred lines is prone to various sources of variation, which differ strongly in their relevance for GP within and across BPFs. It should be stressed that the conclusions drawn from our study do not only apply to DH lines, but also to inbreds developed by recurrent selfing and most likely also to partly inbred generations. Overall, our results corroborate within-family GP as a valuable and robust tool for the implementation of GP in plant breeding, provided the training set meets minimum standards for <span class="inline-formula" id="inline-formula-530"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-539.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-539.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-531"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-540.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-540.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-532"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-541.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-541.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-533"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-542.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-542.gif"/></noscript></span></span>0.3). However, the need for phenotypes from the predicted family represents the main drawback of within-family GP, because this increases both the costs and the time needed until selection can be applied.</p><p id="p-64">Our simulations on across-family GP were restricted to the simple strategy of using only a single HSF or URF for model training. This provided a manageable framework for analyzing the underlying causes affecting variation in PA. For a given BPF<sub>pred</sub>, we showed: (i) the PA in across-family GP expected across many traits differs systematically between different BPF<sub>train</sub>, even if they have the same pedigree relationship with the BPF<sub>pred</sub>, (ii) deterministic equations enable accurate forecasts of the PA across traits for given pairs of BPF<sub>pred</sub> and BPF<sub>train</sub>, and (iii) large variation in the PA among traits hampers the forecasting. Therefore, it is very unlikely to find a single BPF<sub>train</sub> that performs uniformly best across all target traits. This means that caution must be exercised when applying rules of thumb or deterministic equations for choosing the BPF<sub>train</sub> in GP of a specific trait given BPF<sub>pred</sub>. This issue can be even more severe if (i) traits deviate from the polygenic architecture assumed in our simulations, or (ii) <span class="inline-formula" id="inline-formula-534"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-543.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-543.gif"/></noscript></span></span> in the BPFs is smaller than <span class="inline-formula" id="inline-formula-535"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-544.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-544.gif"/></noscript></span></span> in maize due to fewer chromosomes and/or smaller genome size (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>). Thus, identification of useful, trait-specific BPF<sub>train</sub> might only be possible by directly evaluating the empirical PA for a small sample <span class="inline-formula" id="inline-formula-536"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-545.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-545.gif"/></noscript></span></span>of individuals from the BPF<sub>pred</sub>. However, this would largely eliminate the time- and cost-related advantages of genomic selection based on previously available data from BPFs.</p><p id="p-65">In practice, breeders generally do not rely on single-family training sets in GP across BPFs, but rather use multi-family training set designs for the sake of increasing sample size (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-49-12" class="xref-bibr" href="#ref-49">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hickey <em>et al.</em> 2014</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Jacobson <em>et al.</em> 2014</a>; <a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2014</a>). Another important advantage of multi-family over single-family training sets in across-family GP most likely stems from the increased proportion of causal loci segregating in both the BPF<sub>pred</sub> and the training set, which we identified as the core problem leading to the large variation of PA in GP across single BPFs. One critical question in this context is whether or not a single BPF<sub>train</sub> that is poorly predictive of a given BPF<sub>pred</sub> (<em>e.g.</em>, a HSFs that yields PA close to zero, <a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4</a>) is detrimental or harmless for PA if combined together with other predictive BPFs for extending the training set. The problem might exacerbate if URF are included in multi-family training sets (<em>cf</em>. <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Albrecht <em>et al.</em> 2011</a>), which might come at the expense of reduced linkage phase similarity (<em>cf</em>. Figure S1C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1/FileS1.pdf">File S1</a>) between a multifamily training set and the BPF<sub>pred</sub> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lorenz and Smith 2015</a>). Further research is warranted to investigate whether the current design of training sets can be improved by identifying and excluding adverse families to avoid disappointing outcomes of GP in BPFs.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">We thank Chris-Carolin Schön, Matthias Westhues, Tobias Schrag, and Willem Molenaar for valuable suggestions to improve the content of this manuscript. P.S. acknowledges Syngenta for partially funding this research by a Ph.D. fellowship, and A.E.M. acknowledges the financial contribution of the International Maize and Wheat Improvement Center/Gesellschaft für Internationale Zusammenarbeit (CIMMYT/GIZ) through the Climate Resilient Maize for Asia (CRMA) Project 15.78600.8-001-00.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-22"><h3>Genomic Relationships Between DH Lines from BFPs <span class="inline-formula" id="inline-formula-537"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-546.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-546.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-538"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-547.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-547.gif"/></noscript></span></span> Calculated with Different Methods</h3><p id="p-66">Suppose <span class="inline-formula" id="inline-formula-539"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-548.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-548.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-540"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-549.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-549.gif"/></noscript></span></span> are two DH lines from BPFs <span class="inline-formula" id="inline-formula-541"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-550.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-550.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-542"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-551.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-551.gif"/></noscript></span></span> respectively. Let <span class="inline-formula" id="inline-formula-543"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-552.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-552.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-544"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-553.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-553.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-545"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-554.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-554.gif"/></noscript></span></span> denote the set of loci (SNPs or QTL, depending on the context) that are polymorphic in <span class="inline-formula" id="inline-formula-546"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-555.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-555.gif"/></noscript></span></span>or in <span class="inline-formula" id="inline-formula-547"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-556.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-556.gif"/></noscript></span></span> polymorphic in <span class="inline-formula" id="inline-formula-548"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-557.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-557.gif"/></noscript></span></span> polymorphic in <span class="inline-formula" id="inline-formula-549"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-558.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-558.gif"/></noscript></span></span> and polymorphic in <span class="inline-formula" id="inline-formula-550"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-559.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-559.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-551"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-560.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-560.gif"/></noscript></span></span> respectively. Since <span class="inline-formula" id="inline-formula-552"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-561.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-561.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-553"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-562.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-562.gif"/></noscript></span></span> are BPFs, we have, under Mendelian inheritance,<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-563.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-563.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span>Thus,<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-564.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-564.gif"/></noscript></span><span class="disp-formula-label">(A2)</span></span>where <span class="inline-formula" id="inline-formula-554"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-565.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-565.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-555"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-566.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-566.gif"/></noscript></span></span> denote the number of elements in set <span class="inline-formula" id="inline-formula-556"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-567.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-567.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-557"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-568.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-568.gif"/></noscript></span></span> respectively. Defining <span class="inline-formula" id="inline-formula-558"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-569.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-569.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-559"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-570.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-570.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-560"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-571.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-571.gif"/></noscript></span></span> we get, with Equation 1, for completely homozygous lines<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-572.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-572.gif"/></noscript></span><span class="disp-formula-label">(A3)</span></span>For calculating the elements of the genomic relationship matrix <span class="inline-formula" id="inline-formula-561"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-573.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-573.gif"/></noscript></span></span> according to the modification proposed in Equation 1, we obtain<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-574.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-574.gif"/></noscript></span><span class="disp-formula-label">(A4)</span></span>where <span class="inline-formula" id="inline-formula-562"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-575.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-575.gif"/></noscript></span></span> refers to the simple matching coefficient (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sneath and Sokal 1973</a>), also known as the IBS (identity by state) coefficient (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Astle and Balding 2009</a>), between <span class="inline-formula" id="inline-formula-563"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-576.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-576.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-564"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-577.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-577.gif"/></noscript></span></span> with respect to the loci set <span class="inline-formula" id="inline-formula-565"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-578.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-578.gif"/></noscript></span></span> Using the original formula of <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> (2013)</a>, which extends Method 1 of <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">VanRaden (2008)</a> to the case of two populations, we obtain the genomic relationship matrix <span class="inline-formula" id="inline-formula-566"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-579.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-579.gif"/></noscript></span></span> with elements<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-580.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-580.gif"/></noscript></span><span class="disp-formula-label">(A5)</span></span>Extending Method 2 of <a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">VanRaden (2008)</a> to the case of two populations, we obtain the genomic relationship matrix <span class="inline-formula" id="inline-formula-567"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-581.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-581.gif"/></noscript></span></span> as follows<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-582.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-582.gif"/></noscript></span><span class="disp-formula-label">(A6)</span></span>where summation is only possible for <span class="inline-formula" id="inline-formula-568"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-583.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-583.gif"/></noscript></span></span> because for <span class="inline-formula" id="inline-formula-569"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-584.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-584.gif"/></noscript></span></span> the denominator is zero, where <span class="inline-formula" id="inline-formula-570"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-585.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-585.gif"/></noscript></span></span> denotes subset of <span class="inline-formula" id="inline-formula-571"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-586.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-586.gif"/></noscript></span></span>polymorphic in <span class="inline-formula" id="inline-formula-572"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-587.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-587.gif"/></noscript></span></span> but not in <span class="inline-formula" id="inline-formula-573"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-588.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-588.gif"/></noscript></span></span> or polymorphic in <span class="inline-formula" id="inline-formula-574"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-589.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-589.gif"/></noscript></span></span> but not in <span class="inline-formula" id="inline-formula-575"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-590.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-590.gif"/></noscript></span></span> Thus, we obtain<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-591.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-591.gif"/></noscript></span><span class="disp-formula-label">(A7)</span></span>in BPFs with allele frequencies equal to 0.5 at segregating loci, with <span class="inline-formula" id="inline-formula-576"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-592.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-592.gif"/></noscript></span></span> Consequently, <span class="inline-formula" id="inline-formula-577"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-593.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-593.gif"/></noscript></span></span> if and only if <span class="inline-formula" id="inline-formula-578"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-594.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-594.gif"/></noscript></span></span> (<em>e.g.</em>, if <span class="inline-formula" id="inline-formula-579"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-595.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-595.gif"/></noscript></span></span>), but otherwise <span class="inline-formula" id="inline-formula-580"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-596.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-596.gif"/></noscript></span></span> Note that the empirical PA of GBLUP is invariant to <span class="inline-formula" id="inline-formula-581"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-597.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-597.gif"/></noscript></span></span> (<em>cf</em>. <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Strandén and Christensen 2011</a>), but <span class="inline-formula" id="inline-formula-582"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-598.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-598.gif"/></noscript></span></span> affects uniformly the scaling of GEBVs and reliabilities thereof. Note also that <span class="inline-formula" id="inline-formula-583"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-599.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-599.gif"/></noscript></span></span> calculated with regard to all loci (set <span class="inline-formula" id="inline-formula-584"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-600.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-600.gif"/></noscript></span></span>) can deviate from <span class="inline-formula" id="inline-formula-585"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-601.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-601.gif"/></noscript></span></span> because<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-602.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-602.gif"/></noscript></span><span class="disp-formula-label">(A8)</span></span>and <span class="inline-formula" id="inline-formula-586"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-603.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-603.gif"/></noscript></span></span> as well as <span class="inline-formula" id="inline-formula-587"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-604.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-604.gif"/></noscript></span></span> can vary between pairs of <span class="inline-formula" id="inline-formula-588"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-605.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-605.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-589"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-606.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-606.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-590"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-607.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-607.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-591"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-608.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-608.gif"/></noscript></span></span> denote subsets of <span class="inline-formula" id="inline-formula-592"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-609.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-609.gif"/></noscript></span></span> polymorphic in <span class="inline-formula" id="inline-formula-593"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-610.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-610.gif"/></noscript></span></span> but not in <span class="inline-formula" id="inline-formula-594"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-611.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-611.gif"/></noscript></span></span> and polymorphic in <span class="inline-formula" id="inline-formula-595"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-612.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-612.gif"/></noscript></span></span> but not in <span class="inline-formula" id="inline-formula-596"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-613.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-613.gif"/></noscript></span></span> respectively.</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-23"><h3>Calculation of the Reliability of GP across Populations for Inbred Individuals</h3><p id="p-67">Assume two populations <em>A</em> (= prediction set) and <em>B</em> (= training set), which are not necessarily BPFs, and we consider across-population GP. Using well-known results about selection indices (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Mrode 2005</a>), the breeding value <span class="inline-formula" id="inline-formula-597"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-614.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-614.gif"/></noscript></span></span> for individual <span class="inline-formula" id="inline-formula-598"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-615.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-615.gif"/></noscript></span></span> which may be inbred, is predicted with information from its genotype and the phenotypic and genotypic information from the training set as:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-616.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-616.gif"/></noscript></span><span class="disp-formula-label">(B1)</span></span>in which <span class="inline-formula" id="inline-formula-599"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-617.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-617.gif"/></noscript></span></span> is the predicted breeding value, <span class="inline-formula" id="inline-formula-600"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-618.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-618.gif"/></noscript></span></span> is the true breeding value of individual <span class="inline-formula" id="inline-formula-601"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-619.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-619.gif"/></noscript></span></span> in population <span class="inline-formula" id="inline-formula-602"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-620.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-620.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-603"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-621.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-621.gif"/></noscript></span></span> is a vector with phenotypes of individuals from population <span class="inline-formula" id="inline-formula-604"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-622.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-622.gif"/></noscript></span></span> corrected for fixed effects.</p><p id="p-68">The covariance between the true breeding value of an individual from population <span class="inline-formula" id="inline-formula-605"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-623.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-623.gif"/></noscript></span></span> and the phenotypes of individuals from population <span class="inline-formula" id="inline-formula-606"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-624.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-624.gif"/></noscript></span></span> is:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-625.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-625.gif"/></noscript></span><span class="disp-formula-label">(B2)</span></span>where <span class="inline-formula" id="inline-formula-607"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-626.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-626.gif"/></noscript></span></span> is the genetic correlation between <span class="inline-formula" id="inline-formula-608"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-627.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-627.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-609"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-628.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-628.gif"/></noscript></span></span> (which represents the correlation between the breeding value in population <span class="inline-formula" id="inline-formula-610"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-629.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-629.gif"/></noscript></span></span> and the breeding value in population <span class="inline-formula" id="inline-formula-611"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-630.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-630.gif"/></noscript></span></span> for the individuals in <span class="inline-formula" id="inline-formula-612"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-631.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-631.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-613"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-632.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-632.gif"/></noscript></span></span> is the vector of genomic relationships between individual <span class="inline-formula" id="inline-formula-614"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-633.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-633.gif"/></noscript></span></span> and the training individuals from <span class="inline-formula" id="inline-formula-615"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-634.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-634.gif"/></noscript></span></span> that can be estimated by Equation 1 in the main text, and <span class="inline-formula" id="inline-formula-616"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-635.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-635.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-617"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-636.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-636.gif"/></noscript></span></span> are the square root of the additive variances in <span class="inline-formula" id="inline-formula-618"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-637.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-637.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-619"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-638.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-638.gif"/></noscript></span></span> respectively. Finally,<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-639.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-639.gif"/></noscript></span><span class="disp-formula-label">(B3)</span></span>where <span class="inline-formula" id="inline-formula-620"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-640.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-640.gif"/></noscript></span></span> is the genomic relationship matrix among training individuals in <span class="inline-formula" id="inline-formula-621"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-641.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-641.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-622"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-642.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-642.gif"/></noscript></span></span> is the covariance matrix of the errors <span class="inline-formula" id="inline-formula-623"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-643.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-643.gif"/></noscript></span></span> in the observation vector <span class="inline-formula" id="inline-formula-624"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-644.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-644.gif"/></noscript></span></span> and the breeding value <span class="inline-formula" id="inline-formula-625"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-645.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-645.gif"/></noscript></span></span> is predicted as:<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-646.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-646.gif"/></noscript></span><span class="disp-formula-label">(B4)</span></span>We are interested in the reliability<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-647.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-647.gif"/></noscript></span><span class="disp-formula-label">(B5)</span></span>for the estimated breeding value <span class="inline-formula" id="inline-formula-626"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-648.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-648.gif"/></noscript></span></span> of individual <span class="inline-formula" id="inline-formula-627"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-649.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-649.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-628"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-650.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-650.gif"/></noscript></span></span> being defined with respect to population <span class="inline-formula" id="inline-formula-629"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-651.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-651.gif"/></noscript></span></span> Since <span class="inline-formula" id="inline-formula-630"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-652.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-652.gif"/></noscript></span></span> together with Equation B2, we obtain<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-653.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-653.gif"/></noscript></span><span class="disp-formula-label">(B6)</span></span>so, the reliability is:</p><span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-654.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-654.gif"/></noscript></span></span></div></div><div class="section app" id="app-3"><h2 class="">Appendix C</h2><div class="section" id="sec-24"><h3>Calculation of <span class="inline-formula" id="inline-formula-631"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-655.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-655.gif"/></noscript></span></span> and the Variance of Genomic Relationships of Inbred Populations</h3><p id="p-69">Consider two populations <span class="inline-formula" id="inline-formula-632"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-656.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-656.gif"/></noscript></span></span> (=prediction set) and <span class="inline-formula" id="inline-formula-633"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-657.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-657.gif"/></noscript></span></span> (= training set) that are not necessarily BPFs. Based on the theory of <a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> (2011)</a>, <a id="xref-ref-56-9" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> (2015)</a> suggested to calculate the effective number of chromosome segments <span class="inline-formula" id="inline-formula-634"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-658.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-658.gif"/></noscript></span></span> shared between the two populations as (see their Equation 20)<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-659.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-659.gif"/></noscript></span><span class="disp-formula-label">(C1)</span></span>If all individuals in <span class="inline-formula" id="inline-formula-635"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-660.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-660.gif"/></noscript></span></span> have the same pedigree relationships with the individuals in <span class="inline-formula" id="inline-formula-636"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-661.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-661.gif"/></noscript></span></span> which holds true for pairs of BPFs, we have <span class="inline-formula" id="inline-formula-637"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-662.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-662.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-638"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-663.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-663.gif"/></noscript></span></span> If the genotypes of loci pairs are stochastically independent, as follows from the assumption of independent segregation in the definition of <span class="inline-formula" id="inline-formula-639"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-664.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-664.gif"/></noscript></span></span> (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Daetwyler <em>et al.</em> 2010</a>), and applies in reality to all loci pairs located on different chromosomes (<em>i.e.</em>, a fraction of at least <span class="inline-formula" id="inline-formula-640"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-665.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-665.gif"/></noscript></span></span> of all loci pairs, assuming <span class="inline-formula" id="inline-formula-641"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-666.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-666.gif"/></noscript></span></span> chromosomes with equal length and number of loci), we have<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-667.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-667.gif"/></noscript></span><span class="disp-formula-label">(C2)</span></span>where <span class="inline-formula" id="inline-formula-642"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-668.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-668.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-643"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-669.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-669.gif"/></noscript></span></span> are defined as in Appendix A, with <span class="inline-formula" id="inline-formula-644"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-670.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-670.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-645"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-671.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-671.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-646"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-672.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-672.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-647"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-673.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-673.gif"/></noscript></span></span>are stochastically independent because <span class="inline-formula" id="inline-formula-648"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-674.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-674.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-649"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-675.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-675.gif"/></noscript></span></span> are random samples from <span class="inline-formula" id="inline-formula-650"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-676.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-676.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-651"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-677.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-677.gif"/></noscript></span></span> respectively. Thus, using results about the product of two stochastically independent random variables (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mood 1974</a>), we obtain<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-678.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-678.gif"/></noscript></span><span class="disp-formula-label">(C3)</span></span>Under inbreeding with inbreeding coefficient <span class="inline-formula" id="inline-formula-652"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-679.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-679.gif"/></noscript></span></span> in population <span class="inline-formula" id="inline-formula-653"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-680.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-680.gif"/></noscript></span></span> applying well-known results on the effects of inbreeding on the additive genetic variance (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Falconer and Mackay 1996</a>), we obtain<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-681.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-681.gif"/></noscript></span><span class="disp-formula-label">(C4)</span></span>and a similar result for population <span class="inline-formula" id="inline-formula-654"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-682.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-682.gif"/></noscript></span></span> Combining Equations C2, C3, and C4 yields<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-683.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-683.gif"/></noscript></span><span class="disp-formula-label">(C5)</span></span>While, for inbred generations derived by recurrent selfing, this equation may hold only approximately true due to the assumption of stochastic independence among all loci pairs, a proof can be given that Equation C5 holds strictly true without this requirement for DH lines and F<sub>2</sub> individuals, <em>i.e.</em>,<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-684.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-684.gif"/></noscript></span><span class="disp-formula-label">(C6)</span></span>In the original publications (<a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">Goddard <em>et al.</em> 2011</a>; <a id="xref-ref-56-10" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> 2015</a>) connecting <span class="inline-formula" id="inline-formula-655"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-685.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-685.gif"/></noscript></span></span> with the variance in genomic relationships among individuals, it was assumed that the individuals were noninbred. However, if they are DH lines or from another inbred generation, this is expected to affect <span class="inline-formula" id="inline-formula-656"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-686.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-686.gif"/></noscript></span></span> so that for the case of fixed pedigree relationships between <span class="inline-formula" id="inline-formula-657"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-687.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-687.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-658"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-688.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-688.gif"/></noscript></span></span> the estimator of <span class="inline-formula" id="inline-formula-659"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-689.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-689.gif"/></noscript></span></span> becomes</p><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-690.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3571/embed/mml-math-690.gif"/></noscript></span><span class="disp-formula-label">(C7)</span></span></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-3"><p id="p-71">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300076/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-72"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-07-26"><span class="received-label">Received </span>July 26, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-08-22"><span class="accepted-label">Accepted </span>August 22, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Schopp <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.1" data-doi="10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimization of genomic selection training populations with a genetic algorithm.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711%2526ndash%253B015%2526ndash%253B0116%2526ndash%253B6%26rft_id%253Dinfo%253Apmid%252F25943105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25943105&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.2" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-based prediction of testcross values in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.3" data-doi="10.1214/09-STS307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Astle</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population structure and cryptic relatedness in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D24%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-STS307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-STS307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mächler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting linear mixed-effects models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.5" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bustos-Korts</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biddulph</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improvement of predictive ability by uniform coverage of the target genetic space.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3733</span>–<span class="cit-lpage">3747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D3733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jr</span>  <span class="cit-name-given-names">D. H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of predicting genomic breeding values for residual feed intake in angus and charolais beef cattle.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4669</span>–<span class="cit-lpage">4678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D91%26rft.spage%253D4669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.8" data-doi="10.1186/1297-9686-44-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DClark%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DThe%2Bimportance%2Bof%2Binformation%2Bon%2Brelatives%2Bfor%2Bthe%2Bprediction%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Band%2Bthe%2Bimplications%2Bfor%2Bthe%2Bmakeup%2Bof%2Breference%2Bdata%2Bsets%2Bin%2Blivestock%2Bbreeding%2Bschemes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-4%26rft_id%253Dinfo%253Apmid%252F22321529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22321529&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D112%26rft.spage%253D48%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.10" data-doi="10.1371/journal.pone.0003395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of predicting the genetic risk of disease using a genome-wide approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3395%26rft.epage%253De3395%26rft.atitle%253DAccuracy%2Bof%2Bpredicting%2Bthe%2Bgenetic%2Brisk%2Bof%2Bdisease%2Busing%2Ba%2Bgenome-wide%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003395%26rft_id%253Dinfo%253Apmid%252F18852893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852893&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.11" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.12" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3571.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Introduction to Quantitative Genetics</em>.</span> <span class="cit-publ-name">Longman, Pearson</span>, <span class="cit-publ-loc">Essex</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.15" data-doi="10.1534/genetics.114.169367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage disequilibrium with linkage analysis of multiline crosses reveals different multiallelic QTL for hybrid performance in the flint and dent heterotic groups of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169367%26rft_id%253Dinfo%253Apmid%252F25271305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.16" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.17" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.18" data-doi="10.1111/j.1439-0388.2011.00964.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using the genomic relationship matrix to predict the accuracy of genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D409%26rft.epage%253D421%26rft.atitle%253DUsing%2Bthe%2Bgenomic%2Brelationship%2Bmatrix%2Bto%2Bpredict%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00964.x%26rft_id%253Dinfo%253Apmid%252F22059574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00964.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059574&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.19" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.20" data-doi="10.1186/1297-9686-42-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichtner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic relationship information on genomic breeding values in German Holstein cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D42%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenetic%2Brelationship%2Binformation%2Bon%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BGerman%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-5%26rft_id%253Dinfo%253Apmid%252F20170500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20170500&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.21" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.22" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirdita</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection in a commercial winter wheat population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.25" data-doi="10.2135/cropsci2013.03.0195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1476</span>–<span class="cit-lpage">1488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.26" data-doi="10.2135/cropsci2013.11.0774"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">General combining ability model for genomewide selection in a biparental cross.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.11.0774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.11.0774&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.27" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.28" data-doi="10.1186/1297-9686-44-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karoui</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carabaño</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Díaz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D44%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-39%26rft_id%253Dinfo%253Apmid%252F23216664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23216664&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.29" data-doi="10.1186/s12711-014-0074-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vander Jagt</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improved precision of QTL mapping using a nonlinear Bayesian method in a multi-breed population leads to greater accuracy of across-breed genomic predictions.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0074-4%26rft_id%253Dinfo%253Apmid%252F25887988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0074-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25887988&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.30" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krämer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multi-parental populations of maize (<em>Zea mays</em> L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.31" data-doi="10.1534/genetics.115.177394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessment of genetic heterogeneity in structured plant populations using multivariate whole-genome regression models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177394%26rft_id%253Dinfo%253Apmid%252F26122758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzEvMzIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.32" data-doi="10.2135/cropsci2013.12.0856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide prediction accuracy within 969 maize biparental populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1514</span>–<span class="cit-lpage">1522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1514%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.12.0856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.12.0856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.33" data-doi="10.1071/CP13363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection in crops, trees and forages: a review.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D65%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP13363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP13363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D3%26rft.spage%253D481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.35" data-doi="10.2135/cropsci2014.12.0827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2657</span>–<span class="cit-lpage">2667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2657%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.12.0827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.12.0827&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marulanda</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection in biparental populations: assessment of parameters for optimum estimation set design.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D134%26rft.spage%253D623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.37" data-doi="10.1371/journal.pone.0005350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and characterization of a high density SNP genotyping assay for cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMatukumalli%26rft.auinit1%253DL.%2BK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5350%26rft.epage%253De5350%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bfor%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005350%26rft_id%253Dinfo%253Apmid%252F19390634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19390634&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Expectation of means and variances of testcrosses produced from F2 and backcross individuals and their selfed progenies.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D59%26rft.spage%253D105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.39" data-doi="10.1534/genetics.115.186205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Safeguarding our genetic resources with libraries of doubled-haploid lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1611</span>–<span class="cit-lpage">1619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.186205%26rft_id%253Dinfo%253Apmid%252F28468909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzE2MTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.40" data-doi="10.1534/genetics.110.116590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accurate prediction of genetic values for complex traits by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116590%26rft_id%253Dinfo%253Apmid%252F20308278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiede</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Popvar: a genome-wide procedure for predicting genetic variance and correlated response in biparental breeding populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2068</span>–<span class="cit-lpage">2077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3571.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mood</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-source"><em>Introduction to the Theory of Statistics</em>.</span> <span class="cit-publ-name">McGraw-Hill Education</span>, <span class="cit-publ-loc">Europe</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3571.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Linear Models for the Prediction of Animal Breeding Values</em>.</span> <span class="cit-publ-name">CABI</span>, <span class="cit-publ-loc">Wallingford, Oxfordshire, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-web" id="cit-7.11.3571.45"><div class="cit-metadata unstructured">Müller, D., and K. W. Broman, 2017 Meiosis: simulation of meiosis in plant breeding research. R Package. version 1.0.0. Available at: <a href="https://github.com/DominikMueller64/Meiosis">https://github.com/DominikMueller64/Meiosis</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rincón</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequence- <em>vs.</em> chip-assisted genomic selection: accurate biological information is advised.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-web" id="cit-7.11.3571.47"><div class="cit-metadata unstructured">R Core Team, 2017 <em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="https://www.r-project.org/">https://www.r-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.48" data-doi="10.1007/s00122-013-2175-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Optimizing the allocation of resources for genomic selection in one breeding cycle.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2835</span>–<span class="cit-lpage">2848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2835%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2175-9%26rft_id%253Dinfo%253Apmid%252F23982591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2175-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23982591&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.49" data-doi="10.1534/genetics.113.150227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic predictability of interconnected biparental maize populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150227%26rft_id%253Dinfo%253Apmid%252F23535384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.50" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (<em>Zea mays</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.51" data-doi="10.1534/genetics.116.187864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accuracy of genomic prediction in synthetic populations depending on the number of parents, relatedness and ancestral linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.volume%253D205%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D4%26rft.atitle%253DThe%2BSustained%2BImpact%2Bof%2BModel%2BOrganisms--in%2BGenetics%2Band%2BEpigenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.187864%26rft_id%253Dinfo%253Apmid%252F28049700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMjA1LzEvMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3571.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sneath</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokal</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-source"><em>Numerical Taxonomy: The Principles and Practice of Numerical Classification</em>.</span> <span class="cit-publ-name">Freeman</span>, <span class="cit-publ-loc">San Francisco, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.53" data-doi="10.1186/1297-9686-43-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Allele coding in genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D43%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DAllele%2Bcoding%2Bin%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-25%26rft_id%253Dinfo%253Apmid%252F21703021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21703021&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.54" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.55" data-doi="10.1534/genetics.112.146290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146290%26rft_id%253Dinfo%253Apmid%252F23267052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNjIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.56" data-doi="10.1186/s12711-014-0086-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovenhuis</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrooten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Empirical and deterministic accuracies of across-population genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0086-0%26rft_id%253Dinfo%253Apmid%252F25885467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0086-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25885467&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.57" data-doi="10.1534/genetics.115.183269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An equation to predict the accuracy of genomic values by combining data from multiple traits, populations, or environments.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183269%26rft_id%253Dinfo%253Apmid%252F26637542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3571.58" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3571.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3559" title="Genetic Screening for EMS-Induced Maize Embryo-Specific Mutants Altered in Embryo Morphogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3587" title="Mapping Loci That Control Tuber and Foliar Symptoms Caused by PVY in Autotetraploid Potato (Solanum tuberosum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3571" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TuwrVosINrQz4MYi7n4xyCD9TubbeCQ6v4v4nwGYEqo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3571" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3571" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3571" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487096" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AKTydBTBiZHV9Yd2LXmMs6z_oI1eRIim1NL_Vim1HXA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3571.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3571" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-d16gdkbRNbCdWZtU6zmXjRR63sQqZ8Y32709yL528RY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3571" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3571.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487096" id="node487096--2" data-pisa="ggg;7/11/3571" data-pisa-master="ggg;g3.117.300076" data-apath="/ggg/7/11/3571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pascal Schopp</span>, <span class="highwire-citation-author" data-delta="1">Dominik Müller</span>, <span class="highwire-citation-author" data-delta="2">Yvonne C. J. Wientjes</span> and <span class="highwire-citation-author" data-delta="3">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3571-3586;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300076 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487096--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pascal Schopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BSchopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schopp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BSchopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominik Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominik%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yvonne C. J. Wientjes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics, Wageningen University and Research, 6700 AH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BC.%2BJ.%2BWientjes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wientjes%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYvonne%2BC.%2BJ.%2BWientjes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3571" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3571&amp;title=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3571&amp;title=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3571&amp;title=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3571&amp;text=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3571&amp;title=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3571&amp;t=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3571&amp;title=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3571&amp;title=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487096/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487096" id="node487096--3" data-pisa="ggg;7/11/3571" data-pisa-master="ggg;g3.117.300076" data-apath="/ggg/7/11/3571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pascal Schopp</span>, <span class="highwire-citation-author" data-delta="1">Dominik Müller</span>, <span class="highwire-citation-author" data-delta="2">Yvonne C. J. Wientjes</span> and <span class="highwire-citation-author" data-delta="3">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3571-3586;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300076 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487096--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pascal Schopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BSchopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schopp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BSchopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominik Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominik%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yvonne C. J. Wientjes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics, Wageningen University and Research, 6700 AH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BC.%2BJ.%2BWientjes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wientjes%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYvonne%2BC.%2BJ.%2BWientjes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3571&amp;rft.epage=3586&amp;rft.atitle=Genomic%20Prediction%20Within%20and%20Across%20Biparental%20Families%3A%20Means%20and%20Variances%20of%20Prediction%20Accuracy%20and%20Usefulness%20of%20Deterministic%20Equations&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Schopp%2C+Pascal&amp;rft.au=M%C3%BCller%2C+Dominik&amp;rft.au=Wientjes%2C+Yvonne+C.+J.&amp;rft.au=Melchinger%2C+Albrecht+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487096/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487096/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487096/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487096/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487096/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487096/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487096/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487096/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487096/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487096/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487096/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487096/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3571" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix&#xA0;A</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="even"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix&#xA0;C</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3571.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3571.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3571.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3571.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
