<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3731" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3731.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3731.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3731.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3731.atom" />
<meta name="HW.pisa" content="ggg;7/11/3731" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.117.300227" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Metzger et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Protein degradation by the ubiquitin-proteasome system is essential to many processes. We sought to assess its involvement in the turnover of mitochondrial proteins in Saccharomyces cerevisiae . We find that deletion of a specific ubiquitin ligase (E3), [Psh1p][1], increases the abundance of a temperature-sensitive mitochondrial protein, [mia40][2]-4pHA, when it is expressed from a centromeric plasmid. Deletion of [Psh1p][1] unexpectedly elevates the levels of other proteins expressed from centromeric plasmids. Loss of [Psh1p][1] does not increase the rate of turnover of [mia40][2]-4pHA, affect total protein synthesis, or increase the protein levels of chromosomal genes. Instead, [psh1][1]Δ appears to increase the incidence of missegregation of centromeric plasmids relative to their normal 1:1 segregation. After generations of growth with selection for the plasmid, ongoing missegregation would lead to elevated plasmid DNA, mRNA, and protein, all of which we observe in [psh1][1]Δ cells. The only known substrate of [Psh1p][1] is the centromeric histone H3 variant [Cse4p][3], which is targeted for proteasomal degradation after ubiquitination by [Psh1p][1]. However, [Cse4p][3] overexpression alone does not phenocopy [psh1][1]Δ in increasing plasmid DNA and protein levels. Instead, elevation of [Cse4p][3] leads to an apparent increase in 1:0 plasmid segregation events. Further, 2 μm high-copy yeast plasmids also missegregate in [psh1][1]Δ , but not when [Cse4p][3] alone is overexpressed. These findings demonstrate that [Psh1p][1] is required for the faithful inheritance of both centromeric and 2 μm plasmids. Moreover, the effects that loss of [Psh1p][1] has on plasmid segregation cannot be accounted for by increased levels of [Cse4p][3].

 [1]: http://www.yeastgenome.org/locus/S000005415/overview
 [2]: http://www.yeastgenome.org/locus/S000001678/overview
 [3]: http://www.yeastgenome.org/locus/S000001532/overview" />
<meta name="DC.Contributor" content="Meredith B. Metzger" />
<meta name="DC.Contributor" content="Jessica L. Scales" />
<meta name="DC.Contributor" content="Mitchell F. Dunklebarger" />
<meta name="DC.Contributor" content="Allan M. Weissman" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Protein degradation by the ubiquitin-proteasome system is essential to many processes. We sought to assess its involvement in the turnover of mitochondrial proteins in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. We find that deletion of a specific ubiquitin ligase (E3), Psh1p, increases the abundance of a temperature-sensitive mitochondrial protein, mia40-4pHA, when it is expressed from a centromeric plasmid. Deletion of Psh1p unexpectedly elevates the levels of other proteins expressed from centromeric plasmids. Loss of Psh1p does not increase the rate of turnover of mia40-4pHA, affect total protein synthesis, or increase the protein levels of chromosomal genes. Instead, &lt;i&gt;psh1Δ&lt;/i&gt; appears to increase the incidence of missegregation of centromeric plasmids relative to their normal 1:1 segregation. After generations of growth with selection for the plasmid, ongoing missegregation would lead to elevated plasmid DNA, mRNA, and protein, all of which we observe in &lt;i&gt;psh1Δ&lt;/i&gt; cells. The only known substrate of Psh1p is the centromeric histone H3 variant Cse4p, which is targeted for proteasomal degradation after ubiquitination by Psh1p. However, Cse4p overexpression alone does not phenocopy &lt;i&gt;psh1Δ&lt;/i&gt; in increasing plasmid DNA and protein levels. Instead, elevation of Cse4p leads to an apparent increase in 1:0 plasmid segregation events. Further, 2&amp;nbsp;μm high-copy yeast plasmids also missegregate in &lt;i&gt;psh1Δ&lt;/i&gt;, but not when Cse4p alone is overexpressed. These findings demonstrate that Psh1p is required for the faithful inheritance of both centromeric and 2&amp;nbsp;μm plasmids. Moreover, the effects that loss of Psh1p has on plasmid segregation cannot be accounted for by increased levels of Cse4p.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3731" />
<meta name="citation_id" content="7/11/3731" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3731" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3731.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3731.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3731.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300227" />
<meta name="citation_pmid" content="28928274" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3731" />
<meta name="citation_lastpage" content="3743" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Meredith B. Metzger" />
<meta name="citation_author_institution" content="Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702" />
<meta name="citation_author_email" content="metzgermb@mail.nih.gov" />
<meta name="citation_author_email" content="weissmaa@mail.nih.gov" />
<meta name="citation_author" content="Jessica L. Scales" />
<meta name="citation_author_institution" content="Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702" />
<meta name="citation_author" content="Mitchell F. Dunklebarger" />
<meta name="citation_author_institution" content="Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702" />
<meta name="citation_author" content="Allan M. Weissman" />
<meta name="citation_author_institution" content="Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702" />
<meta name="citation_author_email" content="metzgermb@mail.nih.gov" />
<meta name="citation_author_email" content="weissmaa@mail.nih.gov" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=M. Akutsu;citation_author=I. Dikic;citation_author=A. Bremm;citation_title=Ubiquitin chain diversity at a glance.;citation_volume=129;citation_year=2016;citation_pmid=26906419;citation_doi=10.1242/jcs.183954" />
<meta name="citation_reference" content="citation_journal_title=Molecular cancer [electronic resource];citation_journal_abbrev=Mol Cancer;citation_author=A. Amato;citation_author=T. Schillaci;citation_author=L. Lentini;citation_author=A. Di Leonardo;citation_title=CENPA overexpression promotes genome instability in pRb-depleted human cells.;citation_pages=119-119;citation_volume=8;citation_year=2009;citation_pmid=20003272;citation_doi=10.1186/1476-4598-8-119" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. C. Au;citation_author=M. J. Crisp;citation_author=S. Z. DeLuca;citation_author=O. J. Rando;citation_author=M. A. Basrai;citation_title=Altered dosage and mislocalization of histone H3 and Cse4p lead to chromosome loss in Saccharomyces cerevisiae.;citation_volume=179;citation_year=2008;citation_pmid=18458100;citation_doi=10.1534/genetics.108.088518" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. C. Au;citation_author=A. R. Dawson;citation_author=D. W. Rawson;citation_author=S. B. Taylor;citation_author=R. E. Baker;citation_title=A novel role of the N terminus of budding yeast histone H3 variant Cse4 in ubiquitin-mediated proteolysis.;citation_volume=194;citation_year=2013;citation_pmid=23525333;citation_doi=10.1534/genetics.113.149898" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. H. Bengtson;citation_author=C. A. Joazeiro;citation_title=Role of a ribosome-associated E3 ubiquitin ligase in protein quality control.;citation_pages=470-473;citation_volume=467;citation_year=2010;citation_issue=7314;citation_pmid=20835226;citation_doi=10.1038/nature09371" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=P. Bragoszewski;citation_author=A. Gornicka;citation_author=M. E. Sztolsztener;citation_author=A. Chacinska;citation_title=The ubiquitin-proteasome system regulates mitochondrial intermembrane space proteins.;citation_volume=33;citation_year=2013;citation_pmid=23508107;citation_doi=10.1128/MCB.01579-12" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=O. Brandman;citation_author=R. S. Hegde;citation_title=Ribosome-associated protein quality control.;citation_pages=7-15;citation_volume=23;citation_year=2016;citation_pmid=26733220;citation_doi=10.1038/nsmb.3147" />
<meta name="citation_reference" content="citation_journal_title=Trends Cell Biol.;citation_author=R. J. Braun;citation_author=B. Westermann;citation_title=With the help of MOM: mitochondrial contributions to cellular quality control.;citation_pages=441-452;citation_volume=27;citation_year=2017;citation_doi=10.1016/j.tcb.2017.02.007" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Canzonetta;citation_author=S. Vernarecci;citation_author=M. Iuliani;citation_author=C. Marracino;citation_author=C. Belloni;citation_title=SAGA DUB-Ubp8 deubiquitylates centromeric histone variant Cse4.;citation_pages=287-298;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=A. Chacinska;citation_author=S. Pfannschmidt;citation_author=N. Wiedemann;citation_author=V. Kozjak;citation_author=L. K. Sanjuan Szklarz;citation_title=Essential role of Mia40 in import and assembly of mitochondrial intermembrane space proteins;citation_pages=3735-3746;citation_volume=23;citation_year=2004;citation_issue=19;citation_pmid=15359280;citation_doi=10.1038/sj.emboj.7600389" />
<meta name="citation_reference" content="citation_journal_title=Plasmid;citation_author=K. M. Chan;citation_author=Y. T. Liu;citation_author=C. H. Ma;citation_author=M. Jayaram;citation_author=S. Sau;citation_title=The 2 micron plasmid of Saccharomyces cerevisiae: a miniaturized selfish genome with optimized functional competence.;citation_pages=2-17;citation_volume=70;citation_year=2013;citation_pmid=23541845;citation_doi=10.1016/j.plasmid.2013.03.001" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=N. C. Chan;citation_author=T. Lithgow;citation_title=The peripheral membrane subunits of the SAM complex function codependently in mitochondrial outer membrane biogenesis.;citation_volume=19;citation_year=2008;citation_pmid=17978093;citation_doi=10.1091/mbc.E07-08-0796" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. Chau;citation_author=J. W. Tobias;citation_author=A. Bachmair;citation_author=D. Marriott;citation_author=D. J. Ecker;citation_title=A multiubiquitin chain is confined to specific lysine in a targeted short-lived protein;citation_pages=1576-1583;citation_volume=243;citation_year=1989;citation_issue=4898;citation_pmid=2538923;citation_doi=10.1126/science.2538923" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=H. Cheng;citation_author=X. Bao;citation_author=H. Rao;citation_title=The F-box protein Rcy1 is involved in the degradation of histone H3 variant Cse4 and genome maintenance.;citation_volume=291;citation_year=2016;citation_pmid=26975376;citation_doi=10.1074/jbc.M115.701813" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=T. W. Christianson;citation_author=R. S. Sikorski;citation_author=M. Dante;citation_author=J. H. Shero;citation_author=P. Hieter;citation_title=Multifunctional yeast high-copy-number shuttle vectors.;citation_pages=119-122;citation_volume=110;citation_year=1992;citation_issue=1;citation_pmid=1544568;citation_doi=10.1016/0378-1119(92)90454-W" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=L. Cipolla;citation_author=A. Maffia;citation_author=F. Bertoletti;citation_author=S. Sabbioneda;citation_title=The regulation of DNA damage tolerance by ubiquitin and ubiquitin-like modifiers.;citation_pages=105;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Clarke;citation_author=J. Carbon;citation_title=Isolation of a yeast centromere and construction of functional small circular chromosomes.;citation_pages=504-509;citation_volume=287;citation_year=1980;citation_issue=5782;citation_pmid=6999364;citation_doi=10.1038/287504a0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=M. M. Cohen;citation_author=G. P. Leboucher;citation_author=N. Livnat-Levanon;citation_author=M. H. Glickman;citation_author=A. M. Weissman;citation_title=Ubiquitin-proteasome-dependent degradation of a mitofusin, a critical regulator of mitochondrial fusion.;citation_volume=19;citation_year=2008;citation_pmid=18353967;citation_doi=10.1091/mbc.E08-02-0227" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genet.;citation_author=Q. Defenouillere;citation_author=M. Fromont-Racine;citation_title=The ribosome-bound quality control complex: from aberrant peptide clearance to proteostasis maintenance.;citation_pages=1165-1176;citation_volume=28;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. K. Doma;citation_author=R. Parker;citation_title=Endonucleolytic cleavage of eukaryotic mRNAs with stalls in translation elongation.;citation_pages=561-564;citation_volume=440;citation_year=2006;citation_issue=7083;citation_pmid=16554824;citation_doi=10.1038/nature04530" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=M. Escobar-Henriques;citation_author=B. Westermann;citation_author=T. Langer;citation_title=Regulation of mitochondrial fusion by the F-box protein Mdm30 involves proteasome-independent turnover of Fzo1.;citation_volume=173;citation_year=2006;citation_pmid=16735578;citation_doi=10.1083/jcb.200512079" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=S. Fang;citation_author=J. P. Jensen;citation_author=R. L. Ludwig;citation_author=K. H. Vousden;citation_author=A. M. Weissman;citation_title=Mdm2 Is a RING Finger-dependent Ubiquitin Protein Ligase for Itself and p53;citation_pages=8945-8951;citation_volume=275;citation_year=2000;citation_issue=12;citation_pmid=10722742;citation_doi=10.1074/jbc.275.12.8945" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=D. Finley;citation_author=S. Sadis;citation_author=B. P. Monia;citation_author=P. Boucher;citation_author=D. J. Ecker;citation_title=Inhibition of proteolysis and cell cycle progression in a multiubiquitination-deficient yeast mutant.;citation_pages=5501-5509;citation_volume=14;citation_year=1994;citation_issue=8;citation_pmid=8035826;citation_doi=10.1128/MCB.14.8.5501" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=N. Garcia-Rodriguez;citation_author=R. P. Wong;citation_author=H. D. Ulrich;citation_title=Functions of ubiquitin and SUMO in DNA replication and replication stress.;citation_pages=87;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00087" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Gonzalez;citation_author=H. He;citation_author=Q. Dong;citation_author=S. Sun;citation_author=F. Li;citation_title=Ectopic centromere nucleation by CENP–a in fission yeast.;citation_volume=198;citation_year=2014;citation_pmid=25298518;citation_doi=10.1534/genetics.114.171173" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=C. Gregg;citation_author=P. Kyryakov;citation_author=V. I. Titorenko;citation_title=PURIFICATION OF MITOCHONDRIA FROM YEAST CELLS.;citation_pages=1417;citation_volume=pii;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=S. Hajra;citation_author=S. K. Ghosh;citation_author=M. Jayaram;citation_title=The centromere-specific histone variant Cse4p (CENP-A) is essential for functional chromatin architecture at the yeast 2-microm circle partitioning locus and promotes equal plasmid segregation.;citation_volume=174;citation_year=2006;citation_pmid=16966420;citation_doi=10.1083/jcb.200603042" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. H. Hegemann;citation_author=J. H. Shero;citation_author=G. Cottarel;citation_author=P. Philippsen;citation_author=P. Hieter;citation_title=Mutational analysis of centromere DNA from chromosome VI of Saccharomyces cerevisiae.;citation_pages=2523-2535;citation_volume=8;citation_year=1988;citation_issue=6;citation_pmid=3043181" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=W. Heinemeyer;citation_author=A. Simeon;citation_author=H. H. Hirsch;citation_author=H. H. Schiffer;citation_author=U. Teichert;citation_title=Lysosomal and non-lysosomal proteolysis in the eukaryotic cell: studies on yeast.;citation_volume=19;citation_year=1991;citation_pmid=1783206;citation_doi=10.1042/bst0190724" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. Herrero;citation_author=P. H. Thorpe;citation_title=Synergistic control of kinetochore protein levels by Psh1 and Ubr2.;citation_pages=e1005855;citation_volume=12;citation_year=2016;citation_pmid=26891228;citation_doi=10.1371/journal.pgen.1005855" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=P. Heun;citation_author=S. Erhardt;citation_author=M. D. Blower;citation_author=S. Weiss;citation_author=A. D. Skora;citation_title=Mislocalization of the Drosophila centromere-specific histone CID promotes formation of functional ectopic kinetochores.;citation_pages=303-315;citation_volume=10;citation_year=2006;citation_issue=3;citation_pmid=16516834;citation_doi=10.1016/j.devcel.2006.01.014" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=G. Hewawasam;citation_author=M. Shivaraju;citation_author=M. Mattingly;citation_author=S. Venkatesh;citation_author=S. Martin-Brown;citation_title=Psh1 is an E3 ubiquitin ligase that targets the centromeric histone variant Cse4.;citation_pages=444-454;citation_volume=40;citation_year=2010;citation_issue=3;citation_pmid=21070970;citation_doi=10.1016/j.molcel.2010.10.014" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=G. S. Hewawasam;citation_author=M. Mattingly;citation_author=S. Venkatesh;citation_author=Y. Zhang;citation_author=L. Florens;citation_title=Phosphorylation by casein kinase 2 facilitates Psh1 protein-assisted degradation of Cse4 protein.;citation_volume=289;citation_year=2014;citation_pmid=25183013;citation_doi=10.1074/jbc.M114.580589" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. Hieter;citation_author=C. Mann;citation_author=M. Snyder;citation_author=R. W. Davis;citation_title=Mitotic stability of yeast chromosomes: a colony color assay that measures nondisjunction and chromosome loss.;citation_pages=381-392;citation_volume=40;citation_year=1985;citation_issue=2;citation_pmid=3967296;citation_doi=10.1016/0092-8674(85)90152-7" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. M. Hildebrand;citation_author=S. Biggins;citation_title=Regulation of budding yeast CENP-A levels prevents misincorporation at promoter nucleosomes and transcriptional defects.;citation_pages=e1005930;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. M. Hiller;citation_author=A. Finger;citation_author=M. Schweiger;citation_author=D. H. Wolf;citation_title=ER Degradation of a Misfolded Luminal Protein by the Cytosolic Ubiquitin-Proteasome Pathway;citation_pages=1725-1728;citation_volume=273;citation_year=1996;citation_issue=5282;citation_pmid=8781238;citation_doi=10.1126/science.273.5282.1725" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_journal_abbrev=Oncogene;citation_author=R. Honda;citation_author=H. Yasuda;citation_title=Activity of MDM2, a ubiquitin ligase, toward p53 or itself is dependent on the RING finger domain of the ligase.;citation_pages=1473-1476;citation_volume=19;citation_year=2000;citation_issue=11;citation_pmid=10723139;citation_doi=10.1038/sj.onc.1203464" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=C. C. Huang;citation_author=S. Hajra;citation_author=S. K. Ghosh;citation_author=M. Jayaram;citation_title=Cse4 (CenH3) association with the Saccharomyces cerevisiae plasmid partitioning locus in its native and chromosomally integrated states: implications in centromere evolution.;citation_volume=31;citation_year=2011;citation_pmid=21173161;citation_doi=10.1128/MCB.01191-10" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Cell Biol.;citation_author=R. D. Jones;citation_author=R. G. Gardner;citation_title=Protein quality control in the nucleus.;citation_pages=81-89;citation_volume=40;citation_year=2016;citation_pmid=27015023;citation_doi=10.1016/j.ceb.2016.03.002" />
<meta name="citation_reference" content="citation_journal_title=Mol Cell;citation_author=S. Juszkiewicz;citation_author=R. S. Hegde;citation_title=Initiation of quality control during poly(A) translation requires site-specific ribosome ubiquitination.;citation_pages=743-750.e4;citation_volume=65;citation_year=2017;citation_pmid=28065601;citation_doi=10.1016/j.molcel.2016.11.039" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Koshland;citation_author=J. C. Kent;citation_author=L. H. Hartwell;citation_title=Genetic analysis of the mitotic transmission of minichromosomes.;citation_pages=393-403;citation_volume=40;citation_year=1985;citation_issue=2;citation_pmid=3881185;citation_doi=10.1016/0092-8674(85)90153-9" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=D. Loayza;citation_author=A. Tam;citation_author=W. K. Schmidt;citation_author=S. Michaelis;citation_title=Ste6p Mutants Defective in Exit from the Endoplasmic Reticulum (ER) Reveal Aspects of an ER Quality Control Pathway in Saccharomyces cerevisiae;citation_pages=2767-2784;citation_volume=9;citation_year=1998;citation_issue=10;citation_pmid=9763443;citation_doi=10.1091/mbc.9.10.2767" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. Mann;citation_author=R. W. Davis;citation_title=Instability of dicentric plasmids in yeast.;citation_pages=228-232;citation_volume=80;citation_year=1983;citation_issue=1;citation_pmid=6337372;citation_doi=10.1073/pnas.80.1.228" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. B. Metzger;citation_author=M. J. Maurer;citation_author=B. M. Dancy;citation_author=S. Michaelis;citation_title=Degradation of a cytosolic protein requires endoplasmic reticulum-associated degradation machinery.;citation_volume=283;citation_year=2008;citation_pmid=18812321;citation_doi=10.1074/jbc.M806424200" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=M. B. Metzger;citation_author=Y.-H. Liang;citation_author=R. Das;citation_author=J. Mariano;citation_author=S. Li;citation_title=A structurally unique E2-binding domain activates ubiquitination by the ERAD E2, Ubc7p, through multiple mechanisms.;citation_pages=516-527;citation_volume=50;citation_year=2013;citation_pmid=23665230;citation_doi=10.1016/j.molcel.2013.04.004" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=M. B. Metzger;citation_author=J. N. Pruneda;citation_author=R. E. Klevit;citation_author=A. M. Weissman;citation_title=RING-type E3 ligases: master manipulators of E2 ubiquitin-conjugating enzymes and ubiquitination.;citation_pages=47-60;citation_volume=1843;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=L. Morey;citation_author=K. Barnes;citation_author=Y. Chen;citation_author=M. Fitzgerald-Hayes;citation_author=R. E. Baker;citation_title=The Histone Fold Domain of Cse4 Is Sufficient for CEN Targeting and Propagation of Active Centromeres in Budding Yeast;citation_pages=1533-1543;citation_volume=3;citation_year=2004;citation_issue=6;citation_pmid=15590827;citation_doi=10.1128/EC.3.6.1533-1543.2004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=J. M. Muller;citation_author=D. Milenkovic;citation_author=B. Guiard;citation_author=N. Pfanner;citation_author=A. Chacinska;citation_title=Precursor oxidation by Mia40 and Erv1 promotes vectorial transport of proteins into the mitochondrial intermembrane space.;citation_volume=19;citation_year=2008;citation_pmid=17978092;citation_doi=10.1091/mbc.E07-08-0814" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. W. Murray;citation_author=J. W. Szostak;citation_title=Construction and behavior of circularly permuted and telocentric chromosomes in Saccharomyces cerevisiae.;citation_pages=3166-3172;citation_volume=6;citation_year=1986;citation_issue=9;citation_pmid=3537731" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=T. Nolan;citation_author=R. E. Hands;citation_author=S. A. Bustin;citation_title=Quantification of mRNA using real-time RT-PCR.;citation_pages=1559-1582;citation_volume=1;citation_year=2006;citation_issue=3;citation_pmid=17406449;citation_doi=10.1038/nprot.2006.236" />
<meta name="citation_reference" content="citation_journal_title=EMBO Rep.;citation_author=H. F. O’Connor;citation_author=N. Lyon;citation_author=J. W. Leung;citation_author=P. Agarwal;citation_author=C. D. Swaim;citation_title=Ubiquitin-activated interaction traps (UBAITs) identify E3 ligase binding partners.;citation_volume=16;citation_year=2015;citation_pmid=26508657;citation_doi=10.15252/embr.201540620" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Ohkuni;citation_author=R. Abdulle;citation_author=K. Kitagawa;citation_title=Degradation of centromeric histone H3 variant Cse4 requires the Fpr3 peptidyl-prolyl Cis-Trans isomerase.;citation_volume=196;citation_year=2014;citation_pmid=24514906;citation_doi=10.1534/genetics.114.161224" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=K. Ohkuni;citation_author=Y. Takahashi;citation_author=A. Fulp;citation_author=J. Lawrimore;citation_author=W. C. Au;citation_title=SUMO-targeted ubiquitin ligase (STUbL) Slx5 regulates proteolysis of centromeric histone H3 variant Cse4 and prevents its mislocalization to euchromatin.;citation_volume=27;citation_year=2016;citation_pmid=26960795;citation_doi=10.1091/mbc.E15-12-0827" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. B. Pinder;citation_author=M. E. McQuaid;citation_author=M. J. Dobson;citation_title=Deficient sumoylation of yeast 2-micron plasmid proteins Rep1 and Rep2 associated with their loss from the plasmid-partitioning locus and impaired plasmid inheritance.;citation_pages=e60384;citation_volume=8;citation_year=2013;citation_pmid=23555963;citation_doi=10.1371/journal.pone.0060384" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=B. A. Pinsky;citation_author=S. Y. Tatsutani;citation_author=K. A. Collins;citation_author=S. Biggins;citation_title=An Mtw1 complex promotes kinetochore biorientation that is monitored by the Ipl1/Aurora protein kinase.;citation_pages=735-745;citation_volume=5;citation_year=2003;citation_issue=5;citation_pmid=14602074;citation_doi=10.1016/S1534-5807(03)00322-8" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=G. M. Preston;citation_author=J. L. Brodsky;citation_title=The evolving role of ubiquitin modification in endoplasmic reticulum-associated degradation.;citation_volume=474;citation_year=2017;citation_pmid=28159894;citation_doi=10.1042/BCJ20160582" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=P. Ranjitkar;citation_author=M. O. Press;citation_author=X. Yi;citation_author=R. Baker;citation_author=M. J. MacCoss;citation_title=An E3 ubiquitin ligase prevents ectopic localization of the centromeric histone H3 variant via the centromere targeting domain.;citation_pages=455-464;citation_volume=40;citation_year=2010;citation_issue=3;citation_pmid=21070971;citation_doi=10.1016/j.molcel.2010.09.025" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=T. Ravid;citation_author=M. Hochstrasser;citation_title=Autoregulation of an E2 enzyme by ubiquitin-chain assembly on its catalytic residue.;citation_pages=422-427;citation_volume=9;citation_year=2007;citation_issue=4;citation_pmid=17310239;citation_doi=10.1038/ncb1558" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Rev. Mutat. Res.;citation_author=X. Renaudin;citation_author=L. Koch Lerner;citation_author=C. F. Menck;citation_author=F. Rosselli;citation_title=The ubiquitin family meets the Fanconi anemia proteins.;citation_pages=36-46;citation_volume=769;citation_year=2016;citation_doi=10.1016/j.mrrev.2016.06.004" />
<meta name="citation_reference" content="citation_journal_title=Mob. Genet. Elements;citation_author=S. Sau;citation_author=Y. T. Liu;citation_author=C. H. Ma;citation_author=M. Jayaram;citation_title=Stable persistence of the yeast plasmid by hitchhiking on chromosomes during vegetative and germ-line divisions of host cells.;citation_pages=1-8;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=D. Scholz;citation_author=J. Fortsch;citation_author=S. Bockler;citation_author=T. Klecker;citation_author=B. Westermann;citation_title=Analyzing membrane dynamics with live cell fluorescence microscopy with a focus on yeast mitochondria.;citation_pages=275-283;citation_volume=1033;citation_year=2013;citation_doi=10.1007/978-1-62703-487-6_17" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_issue=1;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Stoler;citation_author=K. C. Keith;citation_author=K. E. Curnick;citation_author=M. Fitzgerald-Hayes;citation_title=A mutation in CSE4, an essential gene encoding a novel chromatin-associated protein in yeast, causes chromosome nondisjunction and cell cycle arrest at mitosis.;citation_pages=573-586;citation_volume=9;citation_year=1995;citation_issue=5;citation_pmid=7698647;citation_doi=10.1101/gad.9.5.573" />
<meta name="citation_reference" content="citation_journal_title=Mol Cell;citation_author=E. Sundaramoorthy;citation_author=M. Leonard;citation_author=R. Mak;citation_author=J. Liao;citation_author=A. Fulzele;citation_title=ZNF598 and RACK1 regulate mammalian ribosome-associated quality control function by mediating regulatory 40S ribosomal ubiquitylation.;citation_pages=751-760.e4;citation_volume=65;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=J. S. Thrower;citation_author=L. Hoffman;citation_author=M. Rechsteiner;citation_author=C. M. Pickart;citation_title=Recognition of the polyubiquitin proteolytic signal;citation_pages=94-102;citation_volume=19;citation_year=2000;citation_issue=1;citation_pmid=10619848;citation_doi=10.1093/emboj/19.1.94" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=B. Westermann;citation_author=W. Neupert;citation_title=Mitochondria-targeted green fluorescent proteins: convenient tools for the study of organelle biogenesis in Saccharomyces cerevisiae.;citation_pages=1421-1427;citation_volume=16;citation_year=2000;citation_issue=15;citation_pmid=11054823;citation_doi=10.1002/1097-0061(200011)16:15&lt;1421::AID-YEA624&gt;3.0.CO;2-U" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=T. Yao;citation_author=A. Ndoja;citation_title=Regulation of gene expression by the ubiquitin-proteasome system.;citation_pages=523-529;citation_volume=23;citation_year=2012;citation_issue=5;citation_pmid=22430757;citation_doi=10.1016/j.semcdb.2012.02.006" />
<meta name="citation_reference" content="citation_journal_title=Microbiol. Spectr.;citation_title=The partitioning and copy number control systems of the selfish yeast plasmid: an optimized molecular design for stable persistence in host cells.;citation_author=L. Yen Ting;citation_author=S. Sau;citation_author=C. H. Ma;citation_author=A. H. Kachroo;citation_author=P. A. Rowley;citation_volume=2;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. W. Yuen;citation_author=C. D. Warren;citation_author=O. Chen;citation_author=T. Kwok;citation_author=P. Hieter;citation_title=Systematic genome instability screens in yeast and their potential relevance to cancer.;citation_volume=104;citation_year=2007;citation_pmid=17360454;citation_doi=10.1073/pnas.0610642104" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Biochem.;citation_author=N. Zheng;citation_author=N. Shabek;citation_title=Ubiquitin ligases: structure, function, and regulation.;citation_pages=129-157;citation_volume=86;citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="Protein degradation by the ubiquitin-proteasome system is essential to many processes. We sought to assess its involvement in the turnover of mitochondrial proteins in Saccharomyces cerevisiae . We find that deletion of a specific ubiquitin ligase (E3), [Psh1p][1], increases the abundance of a temperature-sensitive mitochondrial protein, [mia40][2]-4pHA, when it is expressed from a centromeric plasmid. Deletion of [Psh1p][1] unexpectedly elevates the levels of other proteins expressed from centromeric plasmids. Loss of [Psh1p][1] does not increase the rate of turnover of [mia40][2]-4pHA, affect total protein synthesis, or increase the protein levels of chromosomal genes. Instead, [psh1][1]Δ appears to increase the incidence of missegregation of centromeric plasmids relative to their normal 1:1 segregation. After generations of growth with selection for the plasmid, ongoing missegregation would lead to elevated plasmid DNA, mRNA, and protein, all of which we observe in [psh1][1]Δ cells. The only known substrate of [Psh1p][1] is the centromeric histone H3 variant [Cse4p][3], which is targeted for proteasomal degradation after ubiquitination by [Psh1p][1]. However, [Cse4p][3] overexpression alone does not phenocopy [psh1][1]Δ in increasing plasmid DNA and protein levels. Instead, elevation of [Cse4p][3] leads to an apparent increase in 1:0 plasmid segregation events. Further, 2 μm high-copy yeast plasmids also missegregate in [psh1][1]Δ , but not when [Cse4p][3] alone is overexpressed. These findings demonstrate that [Psh1p][1] is required for the faithful inheritance of both centromeric and 2 μm plasmids. Moreover, the effects that loss of [Psh1p][1] has on plasmid segregation cannot be accounted for by increased levels of [Cse4p][3].

 [1]: http://www.yeastgenome.org/locus/S000005415/overview
 [2]: http://www.yeastgenome.org/locus/S000001678/overview
 [3]: http://www.yeastgenome.org/locus/S000001532/overview" />
<meta name="og-title" property="og:title" content="The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3731" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Protein degradation by the ubiquitin-proteasome system is essential to many processes. We sought to assess its involvement in the turnover of mitochondrial proteins in Saccharomyces cerevisiae . We find that deletion of a specific ubiquitin ligase (E3), [Psh1p][1], increases the abundance of a temperature-sensitive mitochondrial protein, [mia40][2]-4pHA, when it is expressed from a centromeric plasmid. Deletion of [Psh1p][1] unexpectedly elevates the levels of other proteins expressed from centromeric plasmids. Loss of [Psh1p][1] does not increase the rate of turnover of [mia40][2]-4pHA, affect total protein synthesis, or increase the protein levels of chromosomal genes. Instead, [psh1][1]Δ appears to increase the incidence of missegregation of centromeric plasmids relative to their normal 1:1 segregation. After generations of growth with selection for the plasmid, ongoing missegregation would lead to elevated plasmid DNA, mRNA, and protein, all of which we observe in [psh1][1]Δ cells. The only known substrate of [Psh1p][1] is the centromeric histone H3 variant [Cse4p][3], which is targeted for proteasomal degradation after ubiquitination by [Psh1p][1]. However, [Cse4p][3] overexpression alone does not phenocopy [psh1][1]Δ in increasing plasmid DNA and protein levels. Instead, elevation of [Cse4p][3] leads to an apparent increase in 1:0 plasmid segregation events. Further, 2 μm high-copy yeast plasmids also missegregate in [psh1][1]Δ , but not when [Cse4p][3] alone is overexpressed. These findings demonstrate that [Psh1p][1] is required for the faithful inheritance of both centromeric and 2 μm plasmids. Moreover, the effects that loss of [Psh1p][1] has on plasmid segregation cannot be accounted for by increased levels of [Cse4p][3].

 [1]: http://www.yeastgenome.org/locus/S000005415/overview
 [2]: http://www.yeastgenome.org/locus/S000001678/overview
 [3]: http://www.yeastgenome.org/locus/S000001532/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FA2Li_b_P1MUJf-32l5EANo2LqSyPoM1FAs240Do9FA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487094","apath":"\/ggg\/7\/11\/3731.atom","pisa":"ggg;7\/11\/3731","ac":{"\/ggg\/7\/11\/3731.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3731.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3731"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3731"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3731":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TZ+\\\\\\\/blPGojmSRscBB1yANbKqJvd11Ara+dRSmb\\\\\\\/TSHxXZSyKHnFlLEN4GrRmo61L0SkTwyCz\\\\\\\/PkXPJqGgu+s1K0NEw9XiZGsx9Zz0WOTQKQe+Jc\\\\\\\/LLFCV7ffbMatuRZliODzUrkkpIhlR8aB2fBYqi05qz04ucXtWvLWf0gwbWO3szahwSAeyXagMtz8lCyH3efXexsAYsh+7TuEJWjwPOneCgoUgcVyoUdzXcS+IBtNPOf4gLNSfSauh0a1ncocpmnPU1nkWVwp7mAX6fVNmUqy4K1tvDGSOa97KYRfzdvY9RoIrZj8lcB4wefEF5NCHHnYaqdakyPFfwGl4VwXfGTN4b9QmaqxjmYSazXd+UwEs2kdlosLZ9etFmZCrvx1niQgU9cilyXOzh9tESSvxvpnM9MDaIM+LmcJ976RekY+iSL3dDNBQEXVAwb2BPCFKnpGHdRjE4CEXeCgzC\\\\\\\/e1xRuOSOLPiBgR+g6oqJOhqeMzfyQBdqLBbmX5y5me5SHd3Y9M1o5izOAcDotQYAxBRbXb7zOnbBGAIOdKvQN8Yhynd982I3dS1JJH8\\\\\\\/F3WAlpAAI2NGf3rCiY\\\\\\\/\\\\\\\/PR2FVuXjd1iCF5WHa+2t539sRQu+eJnOFpgHG48JOR\\\\\\\/1NFtHtlCG8lkJ0hwf2D9bxuXYmBsT9Ruw5TFvu1p\\\\\\\/JbmLmBiv88Lx8qVCiId\\\\\\\/xoNNIp9GQcvd3NNo8xTGk0fYRdK4VkyXSgAYDTagmg\\\\\\\/O\\\\\\\/GkhG4vh6R5NIIqpZFTlnJlQxE+mVzdUVtK3uqZaefWKJXQ59A8CPFHhYowQfAAhf7Zrf9RTyzRQ9sOPU5gbGSmG+V0wAkBmCwEK8d69ptAKPn2xxC2q\\\\\\\/fuazTc+YjPqUnlDaSGSp9LK9Ijc0uNCkOBBtUF2d6fH+KlVHOm3I1fHeHVeKJuW8kP8E0Pf2jeZgkc9fJK\\\\\\\/fmDaFfTPbCjprvK8rjLvcjzScOjsmPAvaYHjfT7rCk3So1J6ABjWkUps087n0uHKvPlURiZPCTxu12Tn3Vt6FRWFok9aGCusR1Tn\\\\\\\/DhDc4+8BQfeyj1YBluwO1kd8GM\\\\\\\/uSYnuYQbKOGqUPBss7Hkkhkg4kR\\\\\\\/Rz7QWSpzlTxEYRnofufXQDM3HmFzJGIU7uuB2rPwJfzvB60raSXNvifCzOoiHOWCwpa5wjkNiKGKK1wxV14cCrmBn0afUeEOsWaaZfuDaPmLecAHWg9VyGIVJ5M0aRpgDTrIdJBvm1wFAGFDTVuk+pw8NK6lkel82ig3yWLZYNqvaOg4kwA1f2IiM\\\u0022,\\\u0022iv\\\u0022:\\\u0022SawmZ6riyLLDQstkoUa62g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223c9ea7cdc6f1bfc408923407a5fc2c455a5c1b28343e543fd2f6b1ed89421d6d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Mc072QApoHeyMdqsheWxkwdrlTkhwAQLjnC7C5Q8FxbXoqaB3xI16I8uz+XuO5SWoNUf6bzZL6WYK4xbqoKJOcEN0ksy+lFu9r8E5U+FDzn2SYBJEP2KWaccQaYp2hZ7+GTnysUqXgYRASd8uG14F62cKQloPmc2chQsPH3xjBOSlcHScOb\\\\\\\/Q8MMrTWPzfC4u4bSnItZLCpuolOasyMO98gz5yzCof23+QMSO+UaEjEvgIk5RJ6EuwCp9+p3hk0EMvDq9+N4hab+8y20wMNir1ISaVHab0J2srz8KaG+Gfjy6SMYD1m0L5wjKpuPtS6I0ZcTcAMQCu9mRvnP4EQ+BmgbrhG3J4VLtsFFzFzbOUIkILI5l9uWkAB1\\\\\\\/dBkAjn2HRvxY+vSoJMrsp1eWIw6+U6hM6cJAoVVX2NhaEBZkshaxyCQY5P7byQHtz+SkqibLH738ci7c6DJR8iFNq6g5zMsAJPJUagnHeUH273FUDfcCPp9oRDCt0GQXJz6p9bW1QVfE2+R2HgKZDeqty4UDyRAVno0NFsmJUy00XZlMM493k3+HWNwCeM08oCldU3XwsEgkhLeg1VnK5sSL5PZH0nqzEopas2iFpuqIQrn5xxRws3pDDMrE0zlgDRbqSYRZl4vURwiWAd\\\\\\\/NvaU9YGAI5L\\\\\\\/fxV8Ew2qZwku5MrOrv7mOB6fdfTtmTa6s\\\\\\\/ROEIRlmsGfeIQ3ttLCw9201n9Os4xbv5IkrE4fkCKP+bDMaHMAf5MDj4FXH0A3d5\\\\\\\/E3jeUwL26LuHIilQTa3VoTi6wy\\\\\\\/QLe1KCzdXMUWeQIyinYvCDJsV0YbY0Asm27RWpgcpr4Igeka4zKSguHAjp6\\\\\\\/kAyBjNdOI9r5VwPfqXHaKaeHmGZLFxcDEc5P1Uh5NH3n5OuiZOsYug3h00aMO\\\\\\\/QMoYkMd8pnrdOEDHsJ4i\\\\\\\/3SEEdYpEIWlqS+DXrV9g59ZlhAh3SY7rTX34di5v9xvyTN54G8QBCBZhUxPBMCZULaNxsZ1wt1hgbcFPFzL5o9D6RX2fURsP3xpTfTPK7Or4H4kNamsS3XCEEgU79KwxOm6uWhmZ2iNACVUTNBF3Q4z7pL5B+oV\\\\\\\/7SbDL16wbHZhwDrUXXbTphYIkV1XQWvmS9bwemIjNezOLoQ0E7A4rpI+PLVY+YKJMsVZM\\\\\\\/SbZQmInzkYneoCkJ+O1ZqWqREads10TP6KZIZJ8RfyOWrWU1ztAq8j9\\\\\\\/I632sH+It0\\\\\\\/3WLIzIUVCEbDATSHDxlB861davTZqSmlXu5RwSbubvUq0Sf8NgaJu2vjJB0IocNow6HsE+c1U+Vq8NyrlyNDD5sDgc09aFDLWNxUb\\\\\\\/l5t77BkO7+aG26GJ5VqcCOvOHjZUVjTKvm3rK0USx8cubKBJXJus3ixdqQ0k8nXyN46rkfk6OP9GVvR2gzlLo56c8YAl8nQA9OT1KFDEXCEBFyY34IIji3G4gUvgW5QpxBXhb8o2\\\\\\\/ZzqLVsdEEBHAWXlD3VGld\\\\\\\/Uf+\\\\\\\/Z6LZhk2BcQmeKzD9Ydk3Zy2JkBIcFvfFimXtgzG\\\\\\\/g2dBh73iIV1+Iwpy48zeiqMZ+lBn\\\\\\\/hxzzOPjnEpzS8CqCBQzDbvoVsqUKQJy0V7UZhrxEomt3j\\\\\\\/qu\\\\\\\/tr9gt\\\\\\\/CZQnCYXiLiyZ5\\\\\\\/usSKdWcFI4crvUzb1cpif7MJoowcaJGG3LM2BMF4ovpolUOZabOhTV3E9bGxfV6NqakFtvvcoL5qrHbrFvhbi+dF3ulqM5Vm2W0+v\\\\\\\/R0fXYcx8lA+FvhL+cjeXivx4dUOQmLbpn1FLCCGa8FwxrMJ8h37Cjqi1xUNw0gzXtX3R8DLoty73a770c5n6rqlteS\\\\\\\/lwOS3fkMrWfiWei1cVtZvQhB8Zq96SRgzdX3nzJD3yVtyaBTvUZbPx95nRl6yHomWXP73uau0wsrKulJbnnmwMOqaoeXHwZZ6ZHWoUUlSVwc5HSzKA+DjJOZ9LGbi0FAaJYOkBjpv5uD3RXF7oGhPNbnDrnaPcKze\\\\\\\/0L1PK5Y1Z8XiquG6pW3unBY4V1B0NRSSRHlFhXMEjq2L5VJe7lxm9uzI27EXOasxvM\\\\\\\/PpE1svRYmTnujR7qW7PmlLysWBuUDMydKyAxM2GEibH2g4XVgja5kJXu9r4bweiALYDGGUD7L0o\\\\\\\/a58T9fixcHKzqYF9d4\\\\\\\/c4nu1p68sphqoKGA5qLIH5PFry+\\\\\\\/YE4lVTceQuY6ckea50DHCgU57XgtTsDJefneEB7mnpWh9fWIIlWfLrGqnBW0gFXgEXTnuqsz\\\\\\\/Ae1m3\\\\\\\/5SgP3QgWVeChSb0q7opSn8S+wIZ0luZ8Mel4HpbC+IDivx1FVXGpv69CghuYwdqxurH8o4nElznv+Gvmwtp4MwJ3BlrI4Tv649lETMdqitEXKei0VrHrZ3Ghn7fWdQ5bGB+zU2DilVXrokRBBkSElUJSd\\\\\\\/lkj2ro3zGgxfi54FadNkAM2ZWMmFTga\\\\\\\/cqnZR7aUHCjJcwR3ZWKLKwcTC5Oysp7maIBrqxEXfSpTYv51xyPXPt0dmMcRF\\\\\\\/emQae859\\\\\\\/wK8VGrRjrLfDWBJ41GzJSPP0VoNZK2xAGOm+Bl81RnEvLrwLsrcfkdvSvIvyur87K8SeVaNex2wgEKJ8h0rOgCN6l4NxjU0zIDwvdTzT+NFPdxrA4\\\\\\\/s\\\\\\\/QaeXwKgETKNRamRD1RuA2SEwhszTZsUc5\\\\\\\/vtVenMTUzVgJeteOJnWbL1YOt7vp8C59CXlDk7zctHfg6PTdPGOkknqtRwYTXBGzQxpPOgmI+QIaHa3iIRa1xGbKHg8wIlghUQCXBGx9FxxjkOB4BdjoLHWVCDLwFVFT2E2VCWvgPnz+VxzZvg5EsyJcSwm9QXNElBfD60bNvWb4aXKKgzWF8aqFuvl9Shin6rEg0NMG08u1vrIufJCpkM0JRENJ9+tKb7pTILD187RPN3VO2w503Lcc69GYag+6ehK3vmj4EknqgM0dkivVxvBDSO\\\\\\\/hBuMTlc2BnFEQ5l4sIkKflnwcVpNbM6VfXwvd1jVJOWd469fvCBc\\\\\\\/sV79kns7oWfUNNJDBJW75AQU2FSlndZGri4IYKt2JUDxSMniKzqlz1jKskV7HQqv6U1drEiNuMPWe3pqNPqYgJBwbtRC\\\\\\\/Paj75FRY6VBDOPTgSqISEep60yBLitzqbJo0pFQwpmVnVoNCuv4oXlRZ8ObkNxbvqCRwS635n7G5SBAkS8l9AnHIW9\\\\\\\/kVLGOMHYBzUQK5gaMO9UNHD\\\\\\\/Ey4OFhUQOLOSNv\\\\\\\/AKSfBfWZa4i6FPPF9PSb6m1LKjyJ3QWN7tXxTMGX+VgkbgZg3KEICZeX1F9\\\\\\\/dzgDQgn1IE7nxIRlzSdmeI3g8UeX2t9YJyDouVfgeQG0d6KN08K9Rq4SDnzu\\\\\\\/ixeizUrXltGQ2AS5g9SslvKToBWPcciCGm7xEVfnIo\\\\\\\/nqrQipvLPKbZAWFWbyZ690P3Rem7kW7Zy4jibVe0rZjfqH2M\\\\\\\/H2R5agbN3aL+qScX1A9FRE5fX\\\\\\\/aqD9M4+\\\\\\\/asWeZ0DrxfZAzYFbuzTGFpDaTNSsezYB4sIey1DglIj7bBQ5JKzQUnUAMzHSE+v5rAyzxLj3EjntioEWc9wrcPHC5W+TJaQu7BzZL0fsPlOvOVS\\\\\\\/jyIflf4rTfIOdctSqcUYwra8goqfc6Ze31TPI0XVYtfu5NI6pMo9+Tzdsh7RJae\\\\\\\/Sfx7UU9PUDCuBKEqJdkgeQA1zbrCRhTEaWp9KEGcjMG6yZomzqWLYvaFV1OZXVMNIvuTMzyyGwJZ\\\\\\\/NVacx4lycwjBToBFZaMYsMg3VWSi5tggq8RPsWB9o2KlQIcqGgqsYeNoWh7yihC5W4oOkJ\\\\\\\/jS1Z26nGl\\\\\\\/v1daBgb0YZ9f6\\\\\\\/jtiJ8sPL1DAg\\\\\\\/ZJG1f+1oogw4LQjLNK9619oE4ZOzNGAYOxS63nCpilA7CBCgJAqE4NhVt9MBYc6HJvZskY4CX7+e9gppOmD0ZI8dbWgJHPwpHWHHUZuAPKbjlF7BNKz5sgPf+qKIdOWRtdrmxNmkIvZ+QdtqbRluoOK6BqfeUEH6TyHFZ9r+P2EZy9Xl\\\\\\\/oCPHcKelLoGVOcbtBO0P+3W+44BfA31mRm03QCG98BGFlWML41BLZmzCYpjWLqslo9LjLofIHnpq3S0U1fYjG0Sa2gYjwJdUIrreAFSRtnQn4OVByHbiIoC9qSO+5r+IgJEc5Ai8rFBVIqgjDKyiV0FQdEs8E+xThxA\\\\\\\/tx41xEBwSsCTbXEBU+4mcN\\\\\\\/b2zQO\\\\\\\/8pvDwAdfQMGW+8DRfW1oObGZfa0D2BWtl1YmM7Bw7fXao+dBHHP48ybGt+XVmQGtJH\\\\\\\/Ly7CFfXWq4AZyS2MqoVnQLR68EO6BrdAPxShqMStfRm950X2svk6lowvgksFYabT\\\\\\\/E5xQqq9HHBOCWDLFh1hHuZ4rTPVBnTn92Ne62J1OyXzv6gDJ7YAgOKPc9XsxgQ6FDNFC\\\\\\\/nVnfSMgng1uSF3SWk12A6qP1Eye1tzPgzlPibhiYlYsAmXBlN4u7rxh4VYA8KF6gt1kpVMHkeaDMp3s1ido+kbcxgqP\\\\\\\/Jajj02f1YqShjeElJOfVR7PIPKaRsG8q9veoswAN3I0oOKQM6D8LiC3Iri3wr6WYDo5COSY4U1uKTjdgNwOVH+ANPVi4uDFCMQOqAkgEFDhSM7eyA7ZboQUWaRskoZE3IHVLl5FlXMzE2M2L6\\\\\\\/dvUYrc6nbk8XXcj1KRSjkZCVAKFV4gJVO3\\\\\\\/nFP01WQ0SSQRfPKJV0OkgCcx2kC1I6WKGDq1IOVFK4t7QkSAUARfM6HtAtz41BJypOX5rqcbPnv2FXXvU4L5ZQetmnXoBGaQGBtMlkPnzKDT6DkKN3889c7Qpp0TLvwRkYH+b57I59ZEGXT+KaxXGtiFdcaj00rgcLctClOSd+WnU4MGAILxmMtw5ubG9JQv2Sh96DF8WQcYMFCTEcKnNkej9xWHsJi5Jwz0Cpq+Gp0PbLXXMZHAFnJRpcXICll962cG\\\\\\\/APm0WEb6fnJ3XXPkXFvQvQYbYhii3rKGS6ajVzTf9LSFWxrKqEN+33\\\\\\\/Ss0UADL6sUvnBZsv2imaGbhNXOKqLquomSglnq+DSVVVRkFLgKLP5oEfEcVAYPW43q5U2++EkAHsJc1LRpoG+hBIln5GNqrGrp6+arUAmBfZ4+2GfXhLIZo2GxLRutv8wxOsyZQrIwz1ZumHJnqaDc4CdNK\\\\\\\/f5gr30\\\\\\\/dxVk8NXanW1mo6sXHj+VOJwZdzRnDLSnfd0YonJ6DphkcUCIUt9JwwLatWO0Bn99fpVcmEbbSOVtb\\\\\\\/1boHQzqaFxv7zxHR2Q+0DRpMfObWUOdTmIiIjKCF7FcIQdiok2WYxEWVF2djSj+wM++jWl+00WlUZdR\\\\\\\/Dyh04NlG2jVottriZktMfTWbCnIMF83LU+2\\\\\\\/9Qs7MNQLywtiW5IfRfS301doPR6MTyWJ4QQ\\\\\\\/0cbr4fr5FPSdmIG0Y4prvYU6bR7Mbdj+FtEWGBfjnRMqziziMxqTCJ7G1SGqTL2MTeEKBiliVkHLS6t\\\\\\\/5PvDkUMtoKThB0TSvU46\\\\\\\/y8KuDOBIWBuTyJbaYOjVY3i5M9D9z21u7ZbeC3xtDgA9cluSm8dsRwupnydMK4xhJNJAiEtelSEBNB0vJ\\\\\\\/kXNfekL+tSGLBzOLRReuLM7cgrtEz8vNFeSgHmgzRaBsBQ14oPUJaJeXn4\\\\\\\/EpfiWaJciAKf+Gq8Df9ioucAY0SpGsGYvY17U2FT1Of+5sO57HNF1sNnCxApvpu1p3w2MUsIAlWEwxBl2jsUgg3egr17yrO60n4rvuEbhQx536Cc8SVJMAVBsae2hd3BHkkXOtDvk0puva\\\\\\\/QJLi\\\\\\\/Y2XA3TBDzMQ89mWkYWV1ZulRwEgDjPJc4XGzSa0PtONDj6N3qPRlvFwWMyEIVGtREOl4J38XtcY435ZXTM9T87G97SF1kHvMc0bav8Hn8lG2h5mqthuvCZxbW7dYufkyOa56oCfH4fsRNUlwOjwIj\\\\\\\/xaCiP0a8AVGSm1qZ78jRke4rvqGIgpQziGGKsly0g0ZIsjis9DKGiLF\\\\\\\/buYCD6Lz+TT839zrzMog5IP9GNCH1M35+igaO2BgbSpWI4DYPr3XBtZRZYlKwCrpityj1g5vc4kkTg+mk9nmHQNnLtNw\\\\\\\/ySTYfuRjUAvP7PEGNKlpzsSjmYUO90t4jFDV\\\\\\\/ZLMOipedWL1ik1e9zHSTp\\\\\\\/j7oJsTRKiaL+mxD70ySa7gDYWNI1vQJbZ5aCyc\\\\\\\/rSzb9B7T521cfxweRtamWuKwM7JBnwQPMBUGk2i6ejlO\\\\\\\/JVZTYrsNuaJvMUKAAnKz628Mavg+z5HH4zuC4z3yfmbo1sG\\\\\\\/BowFb+TGwXaiOf6gqPmr\\\\\\\/9Bl1kav9ZCFZ6S1\\\\\\\/2d2eSgNwMTU6PvFER3wI9rkjgRKwLKwHNe1YQThGz9SQxMHwMFPT4YdeRrg3n0en4NLi\\\\\\\/s0wF1ZEBTzsGyE82GDbotvSw7tGAkWrDOhEsqaCdB2oNZ7xCqkL\\\\\\\/cdpby5XjYXWrKWfTaiDp3uUpec8f10ixz8QhPvpIsjsp50hvYWk\\\\\\\/TlawIYAjhNz9lqipiJWAkz8sCn3BfI\\\\\\\/ts5FnIiJvQIni4K2tTC2wdfnqje+JdHc71l2A680G9tUH1YYXTfI91sgc6scJAPNv4Fu9\\\\\\\/QZ3UKuWdIx0cNRdXbk3iASvF7PaptXR\\\\\\\/RJwRYXamKccC0IBDtuEMBKt3kxG\\\\\\\/P\\\\\\\/ZG+ich9G7wyZHFlmGPXFmYfy15bYWWi5Svwk75lOs6mYt1G4vgoBh6yO3R8TmQXKTO8zf2DVsvbgFFdrR3Xd6xm+EF6M\\\\\\\/F363un\\\\\\\/dcKz6qqie5C\\\\\\\/gyyLgcvlxL9aLGQjNs3qcb9Ber8KmavzpL7+4P3SzHJrEluOJZeCD9RFszrlAvubEaRIxPPr4\\\\\\\/OSaoTN5GBk92H9QDR3gtKehEN+TFUx7fILj+Z2T0iiTqRQVV+\\\\\\\/gl0Ku72RoHxxs7st+xbmVuGreouFwG3uaLmz6h7lSIEdYUZbnxzZAbDWvQa3tOtuK1lG\\\\\\\/c4UZvmFJaeHbNoflauExNs6QGomusI7cLTN7mm5UGIa9XMvcKw9iKCLa7QE7h+8F3TIEaYQaHEQHUfTyo+msbHVbAMcyg9MRXcwFYG7N2cT0g3bXZ3oMLSUIuprgWjwJOMECLIf384CyhNSf7em6IKtJRfoArIH+OsOquc+PEchzReFVLPzElszOLGqR+P7xOoapOZgdcI3iiVhHNH6XEp4Iq3ZXaeYSTlFZKCcz6j4eUiykBxjPfxZHGitiLhEosM85vdhIOw0fOTzSCfO+ytl46q4r0VbyhrbnMk2XdVmc4K7P1RDdfSD1IGapEhO3mqY\\\\\\\/eH0nbfF0GBMa5ol21Id3T7zAKfpRC6Giz4x3j0FVmNyLLGrRZ5s0srkm4P0rxRuF4\\\\\\\/n7vzONNqj1v0aCzzIqSVHvqhU2UpgmjHDNec4FUDRWh+y6C6oJFrSkthQxfTVJpvZFSrDswld3cFBhNp6SMqvJOiszjcB1XhG76WdG3vziePQcdX1jK8X7Ocwa9zouAqp\\\\\\\/wSS7LdKLvWI6UW2byb9ntB4D9PkULBgFwyVaqaUThnemucjhSe+KLyWTgll7i4zK33h1yleA+Z4+V6SQhacR0Fo4WKTcYlNY4kOifXfc06MdgA\\\\\\\/NT\\\\\\\/RO5TeWgw2RdWOu5+xQQuxDc2SGaH96EBcHHaND+qcZOhak6nFGDu1kLTQe8vN3OgRPukhjLzc4HdIVnL8jx3bfZomN8Jbs32xTTNQLF2JzB27TTel78kstj1pB4J\\\\\\\/2xacv6fc21NIcrDBbdb\\\\\\\/LMXWmLnnLRr6E1n8N9d7EbXdKOLzFfJq4HzlpcDfRzXEKnhJzGEX2ZBcI7K5oUY52QndA8oyzu5B1nzbpFLaQvdctoXAzLwqrp4btOpF\\\\\\\/xVPg42YK7YIBBAR3euoMItAWOuaYoPebOT0HIzP9LXWZ+22CoJrVWEjO6u+TIgBbuhpQjjw9cyKeRnGeJOuOEgHecuUQ8wN3U3C2xii3MvT+Ryny6HiA8OYgJqylbQl\\\\\\\/hDKH5oSzLgnaP41nAF8HYfDjU6KTfTWbiM9qv99yEVpmKDqUxtmzAOxmD3R3Aoqsd+zWbwsfihYdFKTdB\\\\\\\/Bq+npAqn1ZDeIGgtdkY6vwGPoVXGSEcMej1DVXmQVo0RnAviEgJIzAeoajwZWNV5SIfZMANRoM+eV16xUqnTiklNMXUSlpxCu9ruxjOEYLKRrqy1AFhqEYPfNRNQ2rRREhQezE007dpQnVmZ98zV8nUgFsyOeG7nga2nt35LqKy+PqHlHh19HsnLJenHwH5n3Qv7b8GELOE2ifstYfpD2h0wvPo4jP1HE+Xz0dmMbhhGiAuDr113qC000xiavMQ\\\\\\\/82dyl7d4JwY\\\\\\\/l2B+z9pXqdJNVsxQtNMXI71VChzsaw3Ixb6XNfEbjQ\\\\\\\/ykpaRDmSA7PRwDKqMBw7hUoEVYagb1HSV5e0qXY+SE8rgoKCTP3gfW2\\\\\\\/vHwzgepYeUd0XzDTer88bVuAlzk2asHaHC1ASDCMLS8jOZs8NGOU1R9uxutbW7Ox33bEHCe3xMiRNxdrYUv0xn6KVoii7iXJTEqR6JUlNpTamUmh6\\\\\\\/\\\\\\\/CzFyjsJLlwgrhuCtyLqkzHBGf3wA3F2junzlqm0T2s3tnVGkfrXm0JJ5ZLgJDUXaVDgTVnsEPK7\\\\\\\/A09CZxxPlFw4Qffyd1ckeOghQgtcbTzpSh8N1RArVWvQmdWA7RiYCeGWR7XbEhz2eNsXNZ1gXjCFpWCrP19Tb8h77uHFzrIeLLZwlrijUo9pnu8Dc2+6MvAFR7+T6C\\\\\\\/Dho6HXCG05TOyeai79ZNs\\\\\\\/lY0KuXZCQt4SkaHRbQzXZoDH1kTmOO9j9Zz6nAm6E+wCBHNC+WdvCq+vdQB97RKYv1Okg5GZcREIgqVC24nJh1BFuX1bXOJkF9OAfZHvHbUJqqv5SNLiXJyFPLvt6+XpnvnBLKIeBFriDWVkwbh6u4Uodo2c9ozoGMRJ89BdhqoEbDKByETVDIeqwfnYV5L5DURc7g7AvR8a7HVaHn\\\\\\\/PnKoZNklgC1Nrq4+QWj3aQuMyv7LXLEMEDoSX4Sg0EuSY4igTiUqr6EwkjPzkZ9nRPqndoU5NqaPfR66etQNWBulNeJy5DTJ7+wL9ef5RRqBE0HoRIwv3wv0ucpIVQBbH0Rd7bKFxfbfxazQgqYPAeR56pwzwAYl8e2ylHhttieZWkf8VRiId9IVVspFLfkt+jZMlu95MeW73wZOMXnum6voCA9g42aQP9GNJrfL+++n36ug3oGMvcKIAp1Ag3e0jI7A23yA7RGkJtAPGj9POwD55E\\\\\\\/H7v58TNYCACJCsVW6JBXh1IIWa9E6BRZsbRLG2bHw4cvt6KEpKqjVvX5jtfpgXnYDp5DpSaqbq7ZpaZE6xeQo\\\\\\\/EsXYymLqYdQ0mMSr+3IS32Zz15kdtFjDNWBkZibO38IXFAL0oDIyHoDC1mP+KDnDgnof2\\\\\\\/GtBiiaG6ZCob1Sv1McoLIW1dTmxdhoptiEuttLxFSXcD5G3XIoS413InRBZpQBecmV8cyf53Jz3UClAecyMC5FAQuSu+8J1NdRA8WqOtT9EGxEydeeYnaeBFvVeAgTYRRWUk0hFVW+uNDlVO1LOm\\\\\\\/ohRpjoDJyjMKJwyW\\\\\\\/RkrG+s7EddO3FbBeai0eUeLpZC+YWBsTkdwSyaf\\\\\\\/Ic3hqUDvcMisk+IyANxgiD0CLPGopddnN6TL\\\\\\\/U2oiAfSqXD9+HPIfhv\\\\\\\/oKZCyDEAJZciokNMvWFJ+qIbD3dR+2NYJ3CxPa12mQDUAKbhIbTv\\\\\\\/IXEpHvW6Ej4xb94Crl0xaiYyBp3ZF9UUb\\\\\\\/t+DnVBQV825QXdXsCdIUk7nIjynyYCQWnBXTeV9MSXidNH\\\\\\\/bwJZSndQPB0xy4dmmLZMzBWe0m6K3xIIjz3Tg1yrxNZxmQgJ7FWuRRnc\\\\\\\/bURDSA4UjDCdhEDPjTiE6oblAx+a\\\\\\\/sCOFsyvIU75y4UevE1DvW+XqnKAepo21e+ApgUcPcB3L0fVuOINjpbpqLNCyD6bQUUuTzrxcmc9\\\\\\\/uh0VNIk6IM5JuYlo8lVJSR\\\\\\\/W+y6ZqJnhf67Wk\\\\\\\/T7HoO44lF1s3ZpwtmKtAVwXEXH+8OdCBf2Ubd87jKa15Q5u7lOIxwRxspAjpaRZQAynluntDB7SxaYQk9ozsSn42vh8IyqYH+pyEXU2+pSIUlJq3\\\\\\\/RCOQIgjgMHM5rK6gaU9tKyYFiynistH9xjVvKgHWwvyHL\\\\\\\/TY+zltzVAS68MNL9Wm+TSuM3zFm+dXXLFg+uXcP\\\\\\\/mVZjeDoyv3DAFkKMGTwxUKle7MkOfKM5HCXTzOt+XFVVdnohqO8ut7WNvwvlFYg5194Gx6Tt4PcYPHZX2m0A3YCJKVwxqMtRJVHSM3AReqGJGZ3dyM1dvTZL5odK8B3yjFPIJ+R0HxH0nMZVGNbuZ1UxOWLjT\\\\\\\/KGyTiH9jdcpBUYXUhz5VvM4XsKf79\\\\\\\/wD6KeBAJaAokSxiyXTTDRIqjjdb7elaYt4Zj7uV98KULnCcTWkHdrEu7LBWWe1uSpcXQXPOedLYPv6COCHYOVIB2Rz6OnMdsq1PwSuST\\\\\\\/41ruHqr9k2QBbbVNCc01szQBtMK7NYl5PYR9pcLOJi2hgn5h1Fzp7YBFoa1ig9umbzEaudXcjytl7wm+0YqXc5adgfo2sA438eIf4F8uQq13NpomtapHCFbTCuIxdc7pOtd6p11IyULTwRmJJoU925cL9MsrmvKHQ0P3DgFceGHaFjEoYMaE2FjCYYYyowwB4sBcvPdLfsA0V9XKdiCK3Fcg9MxEOeGlHEZ0stCxYKcJeve23\\\\\\\/rg9XhlfRMoRZXgRhdHJHIu3GpblimI2lbCti3l2keqcPn+2dz67EWI3KpGTC\\\\\\\/wFlTkuEa4qIZtS1iCPzQebGvtYhEDHwT5Jyze7D\\\\\\\/xsq7beC8X8MsmioorfbEcoAL5zjAJOXzgJR9MctbtFuGAgS34Z+X\\\\\\\/fwc+iRM9SBe9t62DytdUCeS\\\\\\\/P\\\\\\\/SmfBsm6SgmUc3RfzGRS70FcpJ2XJvdCfK8XVGdJLqZMZB5aDZOY1EylLLj3l2G+473aWWhjsGUvU0F\\\\\\\/H1l1qIUyeDGJZdLZh\\\\\\\/lOeXZmjA06EV+JWFPzF1ds6UW7VGxpWxAIRbWVP9yZWrEQRy6pTH27I9rrMDssdCC6\\\\\\\/p0fgt62rVnGG1CP\\\\\\\/lvqhTPJC0oaBnJitC6MX4HcDVFNKqY3rdXokZkuVnojml1sWm9L\\\\\\\/jP\\\\\\\/6iA6CgouykUuI8VRWJa+VgLo73z+F6Mpq+l9duSf+h52e7Ee7mHeduMaAAdzCifFLgEztfkjEzAyBgxaoytiTQ0oCZr4112\\\\\\\/vC8kWLaMYFjFAF5qwShYK9iX921o48FTJA7qqxhwlSYf9AqKn42AnTlxAzrFWzDAIQvLLUAlH4VI53dd9n2mHakCUKuxIzC3Pl1pANwasyeNZS2m1zkvspjLsnATyOCxXC3qcIs3lDkwGsrX0lygbuSo8ty5+Fitt97zHp3GYhGgbbRTtofqEMBl5b5n+DTtKtYuu7HwL7kwQ0885aEqXDWD1m5MOw3\\\\\\\/FroBpXVFW3dmKRukSphv5PYuXGU351fWOZToOhQsbWe12pw7rl0lnqrdzrgnpVgS4cM2cU2AmSjTvEGl88S\\\\\\\/THH0yvE4zsj40rvRnRpFXp6fh0JUoUhO9bTPyReKh8Ett9BaDsWVcCijcK6vLHr\\\\\\\/e3sywXeQQqroxhMoXhOVtUQhRFWUE5+krNXPNt8SFu6fXTwsN5MWkeoe5DtHbeWqn23tuk\\\\\\\/bAVyYBwglDCcdafsS8ljHsqVM6rcEGF6w+qGYvqjXE7zaKjSrIj1DqVZp5hZ5lut9CHCpbBi6oKU1R86BX5pFJ+cyz30M\\\\\\\/bnz0agPWYiFKEBxdCEfg3sGWbBEQKpr1cR8C5w9hOwBYeDw2lV+4CHRV8AtMju+3Ub3\\\\\\\/6jVbMtguQrWYqNqWk7nWCzNm0XcNe79pC4DoNWG7\\\\\\\/e8MVNVzpJ1Opaodthf2Y+IcB9oHvoMqu2fI1Yzlq0r5lfhwpA7OPne+xjcsoAjJt6Zs4AmQdNoZWRPWQzkhyJgw6oVVbMEp+jJpURm4QW2ToeGhXIFVQN278ybWKeVuwugrCygrLlC5O3fuyzDAZ2kbmwWZLV0k7UJhDMQ\\\\\\\/dK2FN5r9SyBIST+foF8ecRmFjrEx+SPdnzb+0fHGgyga6YwaElJtS+JiXx6SX7VGrqVG+r42i4pw5dm83RgBnV2oHvydGLP7tReIRtrXWWzsWKFWmE9IH47\\\\\\\/wnsKd5GogR4DqzOmOj0BQBwvGrDh09TRKU1lj5ugwzzAOAy8CyolSjRjiB6HJpHQHcX6WQKkRckDZwNy0UtVrHZsIvxYgDEINEWdJVwN51dko77OFy4d0BXU1Nu82RESe6CaeTZeBG5ISdCZ9RM8c3haZwCBibuuC01vS0TmwRKclu\\\\\\\/zrBf+mRLU7D5vB1aX8oeRkIcsfVu+d8N12YZBjnffwnuBwcZ2rDbGrpdyE4kNLbU27b7D6u8A4MVvMcLSn9DE7dWvEmPF8u9p2M1kzIGU4lghDnIPRisaxV5VZmr0f9ZnozJ6h6z2jLA3aoyBwCJarCLZRlo80MmN7fhxNVQMTIeCOdqkoju5\\\\\\\/2I0HRmesaHTQbgHB4VOOO0wIfNXbQTCONCKdSQubawVHl9qOk7hZSkbnSw3s\\\\\\\/9j1wKtVDh2GxtBHnoh7RcSPHOcTPQCC4l7oeEkTUeAf0PNHTlLEOBub1PJntSKxQitt9AXIR7mN2aX9\\\\\\\/\\\\\\\/7UEXmm3F\\\\\\\/upp+LFGKR8W5Hv6vS\\\\\\\/rA8+K7LRodDoZy+aOc9PzPfaw748cU9w4EQ87tdNDk0f6zJCttu4kQvXIgmQJ0XW1I+pzcvc+Q460RyhYwm2HjUDAbFhZVtM7m7eoOoMvGZkjjQfXMrF6Di\\\\\\\/67pK2vW0Ut98rjJywQq\\\\\\\/neV5uduQ+seppJpr5268lpTF8ZbXyt+vdRGA2U4G2BBYKOBpAzpW9LG2Lapj+O1yFeqSl6kqst16cH5+STM6ZC1jQKnizANzuN5kvOkAgBozqfsBnfRyt5FZmgsFtmxxqcxj83+CMMgAl1xEru0Q5oaUlwMms4bEC5EoAeUeTPmxoyTP2xlKuuJHl3KYUMlHJaamwT09wo1bgnrqzQ3NHjj+PHPdM+9Vhdh7tvto4AIkuXh6X2XY9uRfpkkE5\\\\\\\/+02ZxLPGkhLeohpDcQFH7iZFforWbZqa2C9mI4QMdtm+0faBzWF8ukKAMVkzjKAmnh+ZHNljAiclrEurzBwk9Te3dtdJOAkszrljU6J7lP2EOWVkENLTtyOtixLwRlYtoqgBxGjegwUBJgNeLKBwQeAj481WbMvVJVYc6uNfMbIMc3DJIBwHaH7ME4IdeiVqL3t8MqwyNDTS2ERCOoxwrNbmuep4bRW6Q\\\\\\\/A+rbM7ehf4+kfDntoEElMMVfnr3FE1+KU3gKatR2JHf0blweqCu9qBIU8s6aYJiMm84UlQOYSE5\\\\\\\/eUU5dUEsdunxANz\\\\\\\/6DmUdL0NLGpvg5kDceRUee\\\\\\\/bxxe8\\\\\\\/ICWexT0VlVlFY9N9IBrW7ICX+HWLscIVCwRJ9V7EOWeYN+6YmjGVar3MU2lR5eirhQ+GPXStY+hjznhu0T9szf1Mxr\\\\\\\/FJQWdf1+Xt4TT55U+2xyx6gBGOjXJ\\\\\\\/faTw1TkqT\\\\\\\/j\\\\\\\/C\\\\\\\/rjZwJ+unP5tcVrl2GdbH2OwVUmXbl4VHTXAX66nMS5UVWiTGhA9Woe2oI6bvbnGSqPAtEgHeWoDUhj9ayECuMoI9X3xy8rA8rAl8MkayL0gjNl6XKhltudvVYzaLZy2suGjQQ98E0I1IpvyZh5abKN+u5gpXD3NjpcnYAjl3xFHrrNl8I91y2yPVf1EBo+ZPv0hVaGmGrueEaJCCwp95s4XRpOaVwgK+Oval4DvShqBlfQjsOkGF\\\\\\\/Pmk6v1Q7A4mUKJqjiT0bDi9bIkmx\\\\\\\/GYX6FEfaIz3EUtXh+l2DswCo4tGUus0EOcUnhgbsknRwArvtc3IMT\\\\\\\/yMY8IA3dUWTDSXiS2G9b5cUUlk3BaE2pqebrINSXtGL1XuOojzXZIBzUqCgSrhfHKHCSpYbmlfwY53gTN+IapfUV\\\\\\\/bWMqIH\\\\\\\/i4+Nhy8emQQcxXxiOGN0Ctq4\\\\\\\/L6pmRM31RH9vGcmBi7NWxj4riCq2tO2HO6p4YkHAw5O0mLCpwFNkEmCmXg4qIlidjkzNfK4ISemq06dMKPDjDJCPKAnmZPBDJsLHHDvQ487b+sbh\\\\\\\/S7dFsnMGN0\\\\\\\/20RhWHTdW2YF1f9K3KBS1195WhTdxrYKC70S3Y0u4Iku0rz6uEtHwJsxlriPxccvCpSQMySkBSnwkjLczi1Md8HPjtQxDh0jRujwV6h\\\\\\\/86ooAdY6icOyneOjVPCgL5YzFoFDqyBy180yixPE\\\\\\\/5w7f0W6T1oUKcUPNutXs9OyIl7y1G3lIzzOzUDPxD3g6GOqowWBon+1eXD+kBC4rJgGzHc831f4cVjPWp1xaXfs8I7yN0xfFZvYEb6zr1PMWBpV8fn3pC0GRrUY\\\\\\\/Qx4MfgyWJyEw3H6T\\\\\\\/xRn1KVeYCXngO\\\\\\\/yl1RSfCH7fXGfm1R0FwRHYzVZeHx4+Yk7aIOJBGaVdXZ7B0r6sA7NhPjgktE45bzgqvmHjGd23p83Dn7dtqMSql4FWQ0t6FwwcjLwy3In6TJwmLLwL6W+0R8wsfMnRJ3NGBUkKsEFR0tGeIoPaZtADIscJG9pGfKb28NVg4PwvdEDuhDxX3xWKLjhJ9ctMGrufiMBCFmJ3BncU8WlnPVHzmDEkso+cMIx0jV6dBc892Uzsn+v0U+xGquCn\\\\\\\/nv6G2LvdFJPG6Gce\\\\\\\/rFQDbHCXl6SA140wQpXM+rjdl7UDO5FTXB+1p015rxPG48tN87zb6PwDLLg2CG2x8qyQRhy2CpXkvU0e9iJiYAQdPEnOxQfXzFyNrz9n8EOFLxnrSRcv1PX+Hugfl6knBjh7dt5B9tfO0zfUQj9UWcmm1jC+V4Vp0BDISSP3LmHO0du8gX66AyJsLFE90HGvhnkoP9+qU7avrONlgtD83I5YSqgmyDR2GQklhv5f5d0Uj1C\\\\\\\/fn0sTN2fr2x6hrrbUf9KVgi2bUm7NG9xsWV7822h7mjii7zn7xYwvINBlHwx+JnoYKqyuu\\\\\\\/xmDN2gQlgHcCil\\\\\\\/oZqYtF\\\\\\\/KCvIrLnObgLdb7NlU6\\\\\\\/wATmYbCAgMDraIHtfDX\\\\\\\/vRHdLsr66o5v8IkeC1u4AG0GLDflguB5IjJZ7xLX2pyQk9YxtpTKLEeE4OPxwr+w43bK5gICsFZuBdDe5+yRr98huCevcnPYPMkrcCUkiuLXBIHirigcwjXflURyjrIYyykdJaMfPPUQ2zqbn05QIxKIKjpYNjHmguAAAc9q8m62UBQqhFLjnniyeFKryF9AEdLdrQUb6nlF7Mjd4dxwRuIEfbFs9lxH\\\\\\\/jSq+mIOlj1jKQdIA1IHki2ovJKEw5lPi7w+PFFEYdchWZSScltGJPxIo5oBYIBptHsHSxVL8qgab9wMsvCVdM80gnPTIWyvNUU8DpwV22mBQn8OMQj9CZS5Baiyc4BmVFqdv8obzjCaKM0U3ajHJX\\\\\\\/aViX74F7hQG2wGdUnV6Uui3V10Yv1W5AYkPJvybHJQ0wBb\\\\\\\/1S49spUOTShk0VIfJIDNLJnr99LlKmdjqGyOzMZ\\\\\\\/rWG9NwAM0ybEgFFz3LgLHKfgCadVvd1svn+UQ78YGgT\\\\\\\/+kj\\\\\\\/k8Dv99k6iLNz4CrJqioKovrRlNBOrPJPbxjOt53XyjDUCaXI6KKwf+n409amI3f9FLZ5LBhK7gc4KctMounTLmPYdC2+tHHmm\\\\\\\/pcV9cr7+bubcXb7QddCHnNFu+DHe\\\\\\\/d4+PEJnbqvDC4UCWEpewy4VI1Ou1NuP0PAzvu6DE\\\\\\\/5wQE\\\\\\\/bPK0Yr1aejjOpudKI4RxNd4K4q+nXOne\\\\\\\/84zzdTEL8tCYNKE4g1ziLt4wX4PxrA5ueLE4GSbRfi05VshXrGmQSTX2y1HhrbQH2Bx+Pe7akXvpLVzpEA9TED49TbDUkkljmkkEvZ0Mp+C1CUatCRXEsqD79BTKTUre+uExpQ13zJ2wNtZrjAuNqge8aYPT+eEUtKEfQvIvhO\\\\\\\/TIoRaFImGPuzHmr2ViQ0X2EraOmMexrOrl8N5K+FBVjKy\\\\\\\/fO+nC+MnhDmFVGfKAYWvMGmthkav+oTvwVndW4KkMdEOJYwOthjSGEGx0EGAaWkpZyNgTakvhmS2Z\\\\\\\/jwrLIHL1xpO40eOwPFyk4C85Ubh+8WvQXDYFeqBMMdfdeGFWdyfwHheYh+opehyvk98hZ1n8\\\\\\\/z+9ggqtGVJTn8C\\\\\\\/5Z9l\\\\\\\/9Gh2ZAay79\\\\\\\/bneevDibs6YpBzrLyFjz8w\\\\\\\/acqeiSQLIRepMGM0N26+jUdjjUKqmA1CwIa6y9Df2RBexSR04ntn96AMoMTVDZos3EHGQGWIFcMCxCqKYBfsxIUF2B17npaxcE6pkXDyvLME4DlPuCn3aVY7HT3X96Gnm4tai0YC5GmLvmktDFFL59oSMqRJ7EjwqPMMbPR0ai4\\\\\\\/zs6C5bd+uwRIToE42jMVvNt+LyJIbZJunknfKNbVaRv4bPhdFqXGPnovOKlJEr7cgzOd84ZHRYd242VTLJaBoHrsOFWExjidAqf4X\\\\\\\/tHNrRPZbz5\\\\\\\/avcsiTKTjbMovHisc7\\\\\\\/+CY3NAYQt5eG7cK+eJzJR0okxt9hed4PouAvAjK93VSvHa2l9TOASoQKDHOVzvxOJSzeGeCKKxc9uevi3ZdfTV5\\\\\\\/SLIXEKHv3\\\\\\\/OgT9Y8IQ7Y7z8H2jBJcmZ6N3IiFoou37N2FNGuDttvzzoxG7t6IhGmnhRQ\\\\\\\/jW4vgaMO6\\\\\\\/2ILd5affhWGP5pFQkuSrRb7OwR3TWBOgL69E+705k2+QWMl4IYGZiW48lxmAZ7narhJAoKBOC6Vm5cn6SoR6zLLM4eXJQVQlJLVLJGGAq2MOoNnEGRdEVe9leIkpq7h+G9HqcLQ2\\\\\\\/UBtv8aJAfILg+0+85N7BG0S9dAx+NoSKfUw2pOGYNNlM2fXHsiGx\\\\\\\/YP4VrSY2ErbYjhkxod0geX8cngzvDmuh9ZgTVu5fqSstU79zakMBj4Yh6OcE5ji6aPqik+sQ5cfKqLcQIPW770Kn1SB\\\\\\\/+kCA7cbLIWRImlQO8EcIZVXT1INeM2ZcHZLFvEh9zHcNike6W+5NukJwwsj5ZbrQOAvMQtS4KuTZQanKdli46GR7Fb0FatmRMx696LTqhNEAwmnbmzpjMR0BVrfPJ3IKFbLtfUm22iv4DqFusLKxuuzDgpUV1ZQhlxMibsP7tf+l8cajS2qTGFAR2RpFC9uadnxB1yqgdosWAuSmfXTq4kGgauaWjCGt850tVnoWhvbATJFR0DSX5D\\\\\\\/0l\\\\\\\/dst6enPIo4Jxdw\\\\\\\/wWhglexGxi5dY4l1g0wGgTjoE2FTA2UnvPlYBvAzipjYBrxwpcd1WsjmY+LLA7IihURuqCLpoOEo8LF5sElA3Ynk2ZZkDcVw2wZ\\\\\\\/oF0ZmH1wTtzbJDgqMdWI19Ah04uA4rKq9Zucn3Hd\\\\\\\/5l9T1eSHFKvG1avNILReBW2A89p+DXVBA\\\\\\\/AZxbYH3tLRxXklt5H+F30C8kr548+RtesIcB7O++UknXzCeDfSYhY4MBlTLvCLgdFxGrtC1aJ\\\\\\\/Up9rnwOtoWAl2TdicIJsI3HLk7wGXJYGYz7kUN\\\\\\\/SstKYvcj9WL\\\\\\\/kuZH40kudC3BRAEEUqZF9R4ZN5JpkZrForFzvOfqXnFcJ5JsibZpfPMEkjlobud7GTkE\\\\\\\/gLnqOYrBWyKS6ft8AxrN0po+WzsCQVW+3Eo0SXjRVI9PM1Qo43+ZFmH7lkxlHspkpc1QhzfQEl5XFTUXEVEINiie1RssGKA1FSlamOf0GaVG7OMvQK\\\\\\\/NthKkHEQTln5um2UIKBVjnYdUutTtWU\\\\\\\/ePJLC+6h7CxY8TkZW2EOweepPw8kaxfFljKtlzKk23uztQmBcn8DAWlYNThLXenG\\\\\\\/OMo92ZKmpnCjuOtWelsZHqjvznIKRzGxDLsFByPgb2GsQujLAOjFnr+DyhJ3sguRtDlECoFiNyJwpihw2H\\\\\\\/10fCwa4+FtD\\\\\\\/bTUt25PdxD84xFf2oM4HWCYBsuc1bFjKJ9jT0uMUzEBzVFjFHvRqZdAZf7mAzwLH1zUdVeCoMH71+E4BJzNSETDyE0xwk0R9Ea46vrFLYq4sbss0+rK7ZC9UeuU9mJgqMfw78JOEA9sM+PxZox9JkV\\\\\\\/PYf5W\\\\\\\/pvkBfLP9BO\\\\\\\/n\\\\\\\/6jKoOHqMzA8toGeCalHVrBi6fR7l5EruDo6NKNJIX20kfWPc\\\\\\\/mPLCgIwWTgaMtosbOEHIjDiV+cWHCaElu22\\\\\\\/50W0eHgvVJwDfgY46GoHr9l3SWbsIQUbTJKSgFcfIc\\\\\\\/Xvz\\\\\\\/vQyBnn+xtbI1Ss6hdPDh32ZqodyQVlUX9nMQ2nLlEk1uHnJLVaisP8y0BGzxaA9NKB+8ecFt6YYvGp5N3FDspX8BLxFJu5cHUCzUNMfzvEmEDXzIUaWxQPlNI1moVtkbJS\\\\\\\/7QV5DLldvTuX3D+FEJLF\\\\\\\/QFph4mVXCh\\\\\\\/1dkJAWyBZ8p09EeLDRO9hADjMwxoQ1Nw00ByeT2C5hMmTOHgD5rtv0X34lvs6kuTUhmRf54isVDmDzKd7LhzkUYKQIfnVg2xbDQZ2S2bFAmA6mshx10pZ+eF1UhsAJ3NH37mLP0o5K2JIyKKUR+0VvKnmZWxUIpLaTMMgVf7WvqrA2BgrS\\\\\\\/KfRdafcN9HJaKEoAAZMBecNPz3brz0f8Mk6d6vMc26Qyz0wQG2wxmHkcVIfU1NhG5CRDjPCwm0+6wPsG\\\\\\\/QBODkFo36EyUYdGjC1gqFbU5Og7Hc8HcnXfMnP\\\\\\\/sO7PCTJiOwEJPpGe06X9gbmybFX6UqbMeJmEHcwHQkeyVc\\\\\\\/fNFOhA8PjkjzS57HenD6egHj9KLcetruKltMAe3tj6WVGnMymPzMflmswp9rSrBO+NddNbAlTe5f9qFMXU5UYCqg0cQColFCllBDv33+1n+Qvs82NDgpnGo3H6TerAfv3G0DfyVjmVwBFl+gosj6LUhmFqOmBYhQveQoUch747N\\\\\\\/vcQyuCQ7vlBeYla4Ul+OcDe\\\\\\\/MwICgLBkqkkR5jn35A39DR3LpTWhv8PZfEJjphVMHseaqz+nAdbO48Eat134iTGPmYVuQaDsOAwuhRZ3GgUf9lFLMj+vejBs\\\\\\\/RuQzDy8+yJ\\\\\\\/+kRYNzBykPGzxxkx3tG+NNALRHrc5YSpL+BZ7hpWFP8Id1LS73yJ34JKIgyf0h5Z9ylK5CKwd\\\\\\\/k9PuAID9Ds17QjBucAnQqHPwy7VZtW6KapaMujYXrRSGGsdXGWzzyMejSdiH4+KRoLLZSrpPpTu5GavVZJF8t\\\\\\\/5rON1MRJmmpYcNAjg0lLsxM8sUpQW1r818CETu5uQq11XTySfEkLxMm2R8MhUdRpZIJzRQ329EijUBjJr1sJ3jRnlkQ2r71dwzLrxmhp0elzIgeePUTmVewipGVc8oN4CSBkfg1QYyavTUYcKiyTUsBmiMk3XEmOPJh3NhVJigPGSvXPttDvhUJ3E6zph7JQDrqEzX07Fd+UUElwgulK7TABr9jQqaJq8UDsOwBOHJwhIU2gh18ckuhWt6H42odHxpgY8ISr+61rQyBqZPkUeKKbt2XEJFDLwT7G+jrciKllNkQHyS8Jo6VlqMD30Fp5nJIwyAG9fTd6j1D5d8Ra2VepwpBKiR9qb7tqz1sYSNhMWv8rAgogUEBY+qy\\\\\\\/OnG6\\\\\\\/3f3PXyC\\\\\\\/c0QzSd7gzVSWjvJA98yz86WM4AYk2GOEADoPlJtfG9Y57BTTQ+bsqMMqonskVv6SRRyoI1svbECbCg56Ma6xWQgL458av4tJqmOkQVlQag5KxGrneFTnS3srqWQLLNUxAFUAFDg98jc9u+3wy7wLvWfd4o8z9kXD\\\\\\\/YiFYUpaq7BoqCoTlQNyEj2J\\\\\\\/Dzfvx93bx8jM8RXzAwVVPSUWPb7SOLpaX7VaEl5s7o2fko+at6mLwdj8AXBS139u8GEsHZBHq6NMpQuZUz1NWr\\\\\\\/qQWyKS8nxxXpD4BxlB5wQD5kjJkPh5ryHyXQvsL8dqrpC3JL7ZCViej9Ub90jW2O\\\\\\\/EXW2Ai\\\\\\\/4q5atBaeEwNVTwG\\\\\\\/LcSrGgLoJewcxB011gjXNS4zZQVVLHvMjRV5xtDPtLWzUBMZnUj2\\\\\\\/48NGfEOdOztd8nOkFQSMWcdehuwfmeaLkCHmBn81+NQiBRpHfVqJ1LUSzQtiNA\\\\\\\/Z\\\\\\\/Z6A7w6ZkmBgPjCQ2dDMS6zvYNDe2sErpBYtC4QdLguIs2hRzGIWIEfQw\\\\\\\/v1Xvnqizfw4Z9eBC\\\\\\\/5CS62eyD2pnkIAVypcmSrpOO\\\\\\\/vm7+E+cG4PLDD8uiFzeicIo1qdrmM8W5nawALu7KEAniU1qMM\\\\\\\/DlOQmNQiujT3SmuXtScQte37Xy1b7kZTz1X98vyzpLfFI5yU6uqOj7ZwD0LBNp\\\\\\\/MzsSz3lggENJonJP0q8z4GA6U6EFjrgkjRaQWBE2ITP0qaNubMa+UV0oynyfFM37iPxpLRwqSU\\\\\\\/zJx2oigMcKOOzYv1SyZ3IeKIgtbWZ5Xax7cI+ySsVK7riSyCTOAVKp+TGRwCC4yJfI1cgNKKB8DlvSlLXsyaJ\\\\\\\/aURF5EKAktJi8vDPs75CISl7JpA4VQH6aqnP3TjGwZWATsP+Q\\\\\\\/+hkhhMmq8KyRn1WvpmVPoylejyHqnNs10eLbvIfdXB\\\\\\\/OUUOBX1ctUbR41Cy68d\\\\\\\/rTaF7gYv\\\\\\\/3Howul5yDwaoQIM7XGkCsj1PxPUYj5SaTnltUNCtwFESQ8FwQJ8CgFxvQuIEV2zwp8ra6dqNvVhie0K81+rw1HRCsw4wgxSVAGMFgRjsI1uNIkSeGr5uMFTbtU4FDmR2oXtZ1YCcDufU2LkpvjY6cNI\\\\\\\/CxDOGWfa786QKIfCFmqxtq73CwYJbWyPSAuMMWvz4RjMISjvOLcbSjJg898+XwRf3TMc5DdbzMTb3+99mSp31wAeUAc1RBa5+i+rIx4nrIUEIudaCT1lTYvoZzTAl6GrzE\\\\\\\/rbI\\\\\\\/Mco6MjVpSpC2xmR+0kHvRGWZr0Ovdp6vSHh65vaatZnDWslNhtZ\\\\\\\/gaFIU89eVxu8mkhEO0fnEL1xrigiucMw6sFNVOT\\\\\\\/sUfsvDlimSPn5imctu0moVKYzRY6g7iv+OZ3BPhGVLnkvfWRSt7wu1OeGs3QuA0r8bLjjYOy15GNx9KUsTkw3uDY646zTg0QqkJGpvQiw3p+YxaTFC6b2E3LUVLWLUpbgTu6msrg47IuTsR3vZBzKWVa3eyQfhWbGedf1I7LrstUW5DNqwSmDSxrRxCc8WPK1VO1RTIx6O9HOLlK210INHz2v9xndCpBj99lKQxJxuj7LD3YZbrIvL7MvIYcSzSC+3Fpw4vi64keb2N2JQPzdtQZ1jJ36zpsMMi7HUPj46Bh5MBvIPY3lO5Es\\\\\\\/XKQfDirDrfKhJxDjK\\\\\\\/IA5d\\\\\\\/TrEzsatxD\\\\\\\/lomRqu4omtJZker1YwdgBCYRNRgc250dt+cGr9uWMoSQnWUQeN8o7v5i1bzBTfvQBSS31XB2zpSghpUNp5IGVgl7S+98feKjcDU+F4odQjt5WaoUrGdwH2+qf2nxIh9M8YMYRDEIALjhFswIq7DpCnghWJt6HaciUQy7p0Qlk3vX4Bc7Aei2xYYuOrDYvCBvdCN8w2IKRNRMczCijxe1L5E662hChkDAauiFTJ0xsv0rFhskySSftM2GSBFpY1wTinzYYLIGnr7KDat8hAi\\\\\\\/xiu\\\\\\\/LETp9xogcJQNikPvcAV3HkApjGlppj1jIg77S92fkonzY4\\\\\\\/nFWxJ6RfNgmdlwEMgN9EpGyJuGGXoH\\\\\\\/y2QOKR1hNurasb9iGaTvZOC9t4p6YQWzW1Ez2K+I\\\\\\\/tkAeMfQE7lwvylBlVNBx2jDJOxBKMHkd801Vo2cuw2D+D+OUjqAs3UGhkn2xIitEgLghAbAEg1Uw0AcfAuQj0CdwPeNDkwBI3O1\\\\\\\/+AnZl2TCfY5zfjPUYIQqj226Bb3fKqQB49DmWUZdukreZPvIbSx54Ya803sZnT9SXaj0HJtx2UiYGtHWMpTX4Hwv+LLs6apELvW9aACXitvRYQTFni0tylvkJUF4Gdviadu+qQSU\\\\\\\/3N0exS25UtTgdC5l\\\\\\\/Hzzoo7Q6zH\\\\\\\/XcBcJuJtl+UYHIBjDN9y30zqGozu44xRF6nziZjykl71W92MfKFEak+9sjlGH4K2bMKUFzpOfp\\\\\\\/nW9uFBX\\\\\\\/gnCHJTijUZtC3DImui2QUUOJbhDY\\\\\\\/GYkTZ3sUONhnvtw3W2NLgMbXhr2CxfQQEIyCxeoPfcz6wDGGWkFrSWJGBvGb\\\\\\\/W\\\\\\\/tCRAiEJ4YatZIs564H\\\\\\\/CAZ3P9TRREZyCJiNIlxas5+nP8Dqx4VGVgB+FXTuFsid4IRt90tG2Xmv80znTaHgItajlgxFrJhbTy6RpNhDm2TV7d9R\\\\\\\/tOsFl+6tL9Npj6Ric3ciOhQuzp11Iktl9O8+09LZUZEKRLa9ScAFxpeedmKM\\\\\\\/cjotr3M3TEqR789zdKL5DjrZZbXFflW2yOpUSRd7uf4N7jbuaA6ojyoNVwbLOrBnhzOVPVNpH+cdzjY+7PpqbAGwU0UV\\\\\\\/UXJOaAlbdbqfGbMkb6w+\\\\\\\/4iwF0XuyxLiV8KN7CB6C1lwAJnU1WQM9NG\\\\\\\/WdylYg162LOFE5L5TdAZF5Isl9BQdIENPWdZJ0OfKeRujeN7XZVrIJp+vO3L0kgDKhNNSk8dLHQgxHoLzbBT09GtEUiATcQR0DEdXjYamWOjCB4Gi40yErcBqRk4nu2qryofXmlXaS9ZMDB8ixcj5ZXmkDgMt2XSJaKQhT8PYP5tbKmE0Tz2lEghuEUJpYrmL9cCJ1h8CD1i2KY4BOvLciBq5+KuGl1Wi\\\\\\\/g8OmDM+Dro6Ib+dLFh5er\\\\\\\/CSnv\\\\\\\/aYlgGWEwPhyAayogGADJ\\\\\\\/IcN+eNhHFhcjLVqyy6NON2Kg95n23LLUytIh4YiuCVuBbqf95kgLTwZW+glbrdVECLQZZQPqLeE0V8XyFlRhgrhfahw\\\\\\\/\\\\\\\/9GyJph4lDBfE2QCBbsVa+NA+hcm8adHaxyq0ZMSyBoGlC5Gjyo9tAtJicThDnbqdZ7tJiBrT4DRUeS27pqPPzRJt\\\\\\\/+q9f+5tbXhk+CDnONyLy2Rwm\\\\\\\/1z3HkahuH1CnW+ovf+D2W\\\\\\\/53GDXX7huSzppZXjIg6dS1cBOyA2zwD7d7w1VHoNnzIpYD16hCy2e9Nyg9nP6HTaSAA4UppOtVX221FJcJTOXO9kFFjgTne4rbU\\\\\\\/JI8lgZSMlhr5OWEJMUYIpTKuP5hKMfmQUtN67HCLtUP9vvgutFXOda0+ItlZbpNzuGLzYBkt3n4Grusf\\\\\\\/Ih5Ke8HO8yz\\\\\\\/Idg\\\\\\\/r4rjRN2Q7Pnuo+wWtHvlYJlFn5rUKgvBP\\\\\\\/LXCB9+AMBBtbgXYMswkhmM1irz1CDudAQtMII8W+jn+\\\\\\\/34zfcP2IPxCV84DiOHwgsPMnA9W78mnqrHM36F6xFhglBmcaO+OfUV7PdgpPBqK0d6s1GgKQUteX9n5G9\\\\\\\/hb+tjDwwJ+IRXai10qzMq9M\\\\\\\/neMt4Z8JatAva5KMyX9N+rI4CFwBF\\\\\\\/alOEdhBUu3JsRyOKE2sDk6jM8U6qFignxqz2OfVreJXdAgwsrmuFH1\\\\\\\/63aU0tKewxZx5PcuLyAWypa91\\\\\\\/y0iO48ryoste2ScT8KExTk8TQiL\\\\\\\/Viunll8j6eMq4LN7mVJKIo+J4LM9lNEAHbaiu+X5+faiIz+xbCVak0vIOxi5xPY0Y\\\\\\\/T43iF3Tv7jY8OeAOVxibTshIjFuZEhd04qecAIj4+WSoxoHcZovnb9n3PSkIsE4HtvFaFgE4pz9kH9CJlUIQkAb4lL60Z2kHT7Ljor9RzkDIY3mE1l9kI\\\\\\\/YD2kDyTgHJ3qN\\\\\\\/EKwM2h4KDi3sct8U4+EIalfizLteI\\\\\\\/3CE6REIRo6+veTjGx9p\\\\\\\/SzPVwU4LJiukCCpsqoYBe\\\\\\\/FnKU+yroYrx2Z73dCQWddrwoOsIprNmKrYoji\\\\\\\/v22g9cVPKhltQMY3h9pYtggvxqyW+ge6h\\\\\\\/orxYDqNkxzV7cQQ7q8IIC6mtj5r6dDlRhniuprJk35M1HOzZRTb8sI2b0BC34S+Fze+RftQkMYCMVUDxUsa0R97R6z3FhNF28goCyrpht\\\\\\\/hato\\\\\\\/hfOA32On9znBzimN7TZ\\\\\\\/NBaZxL4QXun5usMJkz+Q4ekTJKlpK5\\\\\\\/I50KnWxHx7f8TGAagztrVDnX92uaz9YTpGf9HqjYPJxWCDiyXg603lgC8Gt1jGJYOcPBjVlzYRKRyL6aA4eQQ+uNK2k5iRJXCjXwEuBUfvobJdBRyoIG9x6mZjPTSH+lLKRwR5ptwjFfVuiB6BXlIf99r\\\\\\\/MsJZl57sziuOiLb9adXLwy3GF7NYis1L71cHfXB5X72UsvdYp73IHSsCQtPitACBcmgKhJxdGiW8Y8doim2XGU0mDJylSsmSyhl15rze3D\\\\\\\/Aama35UniFC067D3GWLUdzaRM9mmfrj8rAdGQssQGIjG5mK8KYfyKRppMYRMrdi3UegiNzsTeYj7znwm9PnKgbAj\\\\\\\/7HWjRNM+36uxAqaFNjzDtpYyKp4bMXXdiVyRFsbddhbSEQsLkhhPTQhZ+jvfoh9plPpjac9jcWmE0FsscQXaqs3KawXT0igMbZH\\\\\\\/55DV\\\\\\\/T1xPlFoKuSWAAPj3QzJnJ4h10WvZodfvQvOIOKxaeEVSzVuEfPMVLuZ58mQLFUPkynGaKbtzJoXdvpAfrBeu+NjTF7+EfaDebMpeTavAjxfiFY6+\\\\\\\/5orewz\\\\\\\/7C57yCZbhVQcUJ48xWI56srpFzJMjl5tp4Fua+ObNBMcFYqcrc8JJWZzuyGh5IA2Bs3Hu+HnU+9jEIUvTzPd+w7nGvHJ8CN597kW14j8JDiasUhULdlgou1jNidHCQDPBRwGCT5Ka4Fb6SgqNSRpk8YqDmVZUZ0PkuGe78q9F\\\\\\\/5VGoNerh\\\\\\\/tD3MWdUBxZrKHv1vShEI46omA2L5rk51B6XIUZHPeBuAcws9X0ltB6BYySWb9aF7SP6COcqAO1vXJWQ7qmP7L5V1elrhctjmEHH+eZE3rggdoJwPfaxYt4tH7t4tb3QFXb\\\\\\\/F5YyFIOf882rnTqxcfedofh3xIhpFP0lfmyBn8KWrL5rKn0b9M5Odg7wo5tjXDJNsAZi06p87cC0JRAivH28J30ABdpu3RJXzBssPA1\\\\\\\/yjA0nL44u7S58u3IEAHQ3YAtEeka7HjXofirBw4sh9yATM1f8Xoh1caEMQULlm9btFvuTio\\\\\\\/udXi1Sm0R1DMbRlGhV1K0DydmAuBNSjA83eerheJoU1A1dZ7chne6RBPw1tIeeYwUTVrSJ2TMuBpTEUNIEE+At+Uo6XXty\\\\\\\/STAFDI4tHJGOuDqSnmtQIRPYNDpFDVwxlkW4YMQ32z6rGJm1DdGU36wWu3dA8PEzfT2d+Ap+ddpRCQon\\\\\\\/aWKqlgBRAIQ89nu7q3uQlU8UIB5gE9N1ltPLXApc21gtzCCFkGfJdgHF+434u\\\\\\\/YMUQBE+Yfp100utrTJZVJhOrhmVrkFIOBPwYGH3lgRdG\\\\\\\/zNRzUVJ5w5z\\\\\\\/+\\\\\\\/XSHCheC4YiNUXXP5CBt2PoEypv8IO5Up4yX6zX\\\\\\\/Vtz1HqZmat9Gig7f87LAYSTAyLWKHNaSo23lUGz6b4+u7dvbgvyzQ5BDYX91wliIzrga1jZy+L01et5kmaX1lv9cBtFDBMvgCMnaUir5DTB\\\\\\\/nRwRhWYpS1hNxQq2PPkOYaWy7WiJPJd4FYbyCvVq2T2H7K2f29rWGIhOEwf2Gmf2MlMFDnH1aroPRDqheWR7hLN3HDCICo1SwFf+6ILPwlLor5Sw0UGqvfvChuMGbZzG+0K5may5wf9TBIgGFumBzwvVkrUhje6Eyd6qMStbDWYGR7WDr+g8LDGPYDf6kqaexiI2s4rcj8VuN51sPnybN6i99aZMdIIdT7vMskw8W0W+easFVcXESFpG\\\\\\\/6E4\\\\\\\/GXDTxc2ZK\\\\\\\/KBuWPz1schCyq8yi8I2Uux9sbd+b162rWVA+VwlBR5SYGU+FvCRe0xCztO5WzdEDiU4CswiDpsxZPOu536Hc8Jsb83I5scBsC6ZFTQ0xkQ9dwSEJ6OdA8B1rDpv6eZartlRCSxhpx\\\\\\\/lNoXiNEnPpg6YgKifUNxVtMiIWqRjhnhUjOYiRCWggCA8dfbn96zQsubCqEe\\\\\\\/VdgOLMVEsYIJcQ9MMUR\\\\\\\/q9\\\\\\\/+\\\\\\\/slvPNSVqk6PRnak8SDblwNiv9XKQQCvWCa8umcvAdJMo0Tkrg1TrIRTgpTngyv\\\\\\\/7VdnSoylXWTTiPDBXyU+3DoSsat3EiZgbhuQLdiDvzxKQ4oTf\\\\\\\/zI38CnqfZvROEBCtwL2kvM2FF2yjSkaTMEc0DADjRI2KCDMJ92LGvqFpSb4iz2SuB5+OohVR\\\\\\\/C6L7PUqXv7+MMwECXhazg3e+L5HSy7xQcNzHH3Fge7LlgDLqF5Ylzbpbshrh7H5ptoMZvfOxrlfePbsvqXQzriefkVzV0y+zGUeu5dB841KwV3Vlxos\\\\\\\/SIBg\\\\\\\/BuKO69wJVABLnuXs0ejF5B8zNCa\\\\\\\/3+MSNuB5L4agQmhOT1lSQzoWePwRSEVje0cLZ9RPID1Z4+e9+ncTeSEHXXJPou7c8Sjo8a\\\\\\\/hVMOu+w\\\\\\\/VhC43eSc8jgJtk9FtuAqv9LBb2EgoCq6mxrVHwwzbZUq4NUENwE1XVfNQ9keEkgN9Qk5\\\\\\\/V1rpajlmUr0LwslGIciXEu6nvdMyOHccr6SYPNwJxrX8rpYUCZkzIYKNC2tubO8sAjZuZZ8COQQyE97MAwbRXOO\\\\\\\/b6S84TWgHg5cL14tSVIb5OhxyDS85bAPWVXPIZ8+V5zCBUxp5ZSKf8QNdr1gm2yaEda13ux6uJ9KAYsEZ8G35DV1ewVfcB4gTpVRX\\\\\\\/qVHv0SnAgeSAsJ35rAoGo+OYPkS13SqloaDb9sXaYOZBSzL3JhijJEe5jbCU6ATaLfvUYYnqREmzeyHC1KfKVTRh4Spf\\\\\\\/EhOFGsFSDyb\\\\\\\/U0Q5SuvyHG2c9hkFFUBqQv6CxxFofSVCHxt9nMDDNB+cUhsTeJVb9Qp6m5cRHJvdlPPkvBy06f2qCGP8XE8OrJbQItIx\\\\\\\/Y1TEZbPt\\\\\\\/d1EhsIF82fyEEc\\\\\\\/PAwbxcYyS5zqfchOPt1suhOZupKrIxbQQLUC6hGAYbMMIUCCDLPxD+MAcuvlzZIN8g586QKiOxPCaJU19Qces+WoMNZCYcy8B1\\\\\\\/zc+ouCP\\\\\\\/uEuNsyuTjbb6naU5UBYm7QtSPv+hw3Qs7yUEPPJb1VjcxwxiWPAp76idf\\\\\\\/pEIYJLpylBPWer+lW7s1RlCIgZLLyySQ6JlZX0YLThZdiFEcsFyUyqi1fOcwxgny5kSFE\\\\\\\/s1+X6hHh\\\\\\\/C1FU5wbEbu+WE+piU7WsYx4FtwUfx\\\\\\\/V2rYaDwpUnun3+c7UMSTds2FJFZrJrH0A5DEYJeWxovtN1zA5EmWZ3nBxiHNpC69rNhffLCe99LkqaRML4KCYv8MXAeRl29vTQgUO7THc0eHqN2vRsetEjhFV8YbtPO7XDB3X3R+leSJu2BbaIpqx0IhS5svsSw3nyURbb+lSdvKRThySOIpz2vG8B0cwCeU0TVzNVvsrjqDwsCUn1tH3BHS7gRQA5S7PAsJqt95AmRNUcVosch\\\\\\\/\\\\\\\/4WS7PUcF1ZwTbKmIeReZw+6k1DzJzCB6oXQOIzhawdYcBILVeGeDWfbCxHWsoh\\\\\\\/wCzUg+yYezEqig66jsQo2gF1NyNB8gU35WiBAiAXZEKq\\\\\\\/NRSfxZVUBe+NjU47DO3buJ+t6jVJODPD0bYWFc7yBGgnn1s4EFWJJDZF\\\\\\\/3+hGbWGCryfAtSjfxBZEy3XiEgrUzIal2P1BQ2SzRrqrdsjjSHy+D\\\\\\\/5EIsC2F3FHcdSJJf63g9L5gF7pZ44VrQfv0BCwh1FFFPZLCTp\\\\\\\/lHgiElk7zzl4UoMoruaNtKnROk8aIDVSQudfaFTCPlqpAGZt5EB3feZPd7G5AOXhFcteC649SqWDQm2Wybo6D3T+ZIEt+2Pv\\\\\\\/in0G7DRG5N19I8F6\\\\\\\/aqaKQcleLefTGh0MIZdC\\\\\\\/MDdTQAa1G\\\\\\\/fqE+OTjQV00Y5ew68ZpsF1WjMyunZwn4gz0QuXT5ID3K3yV0XUJpDrLeF8p+FRBv40G4XD3ug==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mKgokeXfHr+QeJMkBVVfwQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e843a491d45a0657c380d49f82391038e771a66ccca05cf94040e412d44c9947\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229J5pgtzdlMi5RYUNAtNTOXyt39g2cxKz6rJb2wwSyHvq8TQh84hZqL43IsAlBv3xXroQ9GmoQNNdjgvSxu6OAW1zfYeRl95k8QdhuH6JTm0C3rMTTWM5MWm1lgAI8ZuEixtpMknSZqCE9\\\\\\\/tGh+N1G2qKadXAbG9Nt4wkhpY1N5M8eJz3G\\\\\\\/IGdkQgImvCc1dF6g9RYy3t3LrLuXV7pd2fk7BkqBXhS9QUD7eHOkH9Tyz3asHD3HmHyA3BuCUkOScY2WCbiD8Pny0RbGzLe0aSbswU4Qfr\\\\\\\/sQenAYEAb3O4DMean+Dotde0ffUQDZ+nSETvNZ5zhZCEMyiZaWQXeEe8y2\\\\\\\/jUeMMBylwe7J\\\\\\\/qZ4b6tq94TUwX\\\\\\\/UJgdBb4hViYsrOx5Bvrd49Ga+4lyEVFUvtOLdAB95RkI7Uv8vydTQFeMBzrrrae6tWbgVRWe+HQeMDiXeefbDz7K0DKQFF\\\\\\\/y0NeFg10LQ25Stpxa0d5ztUDoumu0GU0ZMbcoE81LCOx\\\\\\\/3moz1PBEz7FLuDhlhtHkbiBY0j6A2kGe64NnlAn81DoSeLawE3mW3QvfG1DfKAT7jcgZVEiaDE9vP5ZcYKaRmc7S9D9QrIZwVdRCFH4m5DR\\\\\\\/4lfCcDq+urt+4YGFwSrYEqH7IgrrqVzpG3pugrPy0DSvjf5Yw2h4cDLEg9NauWZ+6aohyVjXYC\\\\\\\/YMUKyKxKy8jcl1vcwu94hMDY0xCzSO3KjuUEPvhK89Pe4IBn8gPYbr\\\\\\\/sgo0tMkeyoyyaLiWWYE+IiYtEn\\\\\\\/zbXshLyEy8K9c5DI27BdAr67eUfX+8eMMWpJQbOLgkKd82r00xBPEHEzU5xKHJyZ22U63Q10VnxHuUWUeh4MgLPQLRFCy7Ox7NGWPFnp2IsA+vvcgDtfZuCKtT72tH4kgCUHDk9Ev2Brjckbnm4UWtR33cEjmhybc3dgByOJW5s1kfPBNUIXbJJ11tVHERTVIQXWoJabtSuTwaZ9nl2LALvJHy+xXc6sSpmuX55EmdvN4BuZ\\\\\\\/ns8M9lw2vh13k6BKpkFy1d+Btg4QWuMFoiS7Xro2Bkx+cnCKdZdL6eoNurK3CgDFPXP1oKZFr0uc7SoXlxR6xUN6fC\\\\\\\/kWct8f19+Eu6rlB3xTdfMxCx32VlmoW4HffS01kk0ZyJZLyE\\\\\\\/U2lNfGkw0B6BEKbc4W6IBvsezSaMhJmdDa+o0o5L+Sr5mP\\\\\\\/3Ck5akS7+dwwIhA6JGYFRkAcqGD6+pRQMKR87ttiwwYDoK+NkMzniyqfKAIXEIAjDiuQsA8EU+C3fo+lMB8FJ3SHHlz75PoVa9\\\\\\\/Xrctzk4V9warkLJ\\\\\\\/crvuXzPX1cWXce4THS4Y4\\\\\\\/NLYyJXY3ZGxwceCTdA5Fk8W\\\\\\\/bU40dHMfzHtgA6LVxdfuN5x\\\\\\\/+55U2w+TvlbYKmiToL70o4TYkZX+mlYqw4nUCefe2FjVfbff+ddu8MgHDkVVL9zP3QxgUsnrdHoQLt07\\\\\\\/9zMLsyYSzTYYtlM3Hq30FQbqTCsERSleikezoqpXg+LmLdtYe6Ukys1Y3NudvzjfXx9\\\\\\\/UHqQRxEnQ8yx9N7CnGKtpgCJyl6zpbWAkrL2UNoUEJlW+CAGnTRMtuY7WSeYJ4qeb8kAQK+rRNZwHWDpDP7htMlhClncInvOUnzvtoirwDuGHZKHmvUGvnNyRoWxNyVFueufsB4rvnwJdMa18+9kkZxSnUc08e9FO\\\\\\\/6xgMo2OLDkhRSN+W+rdZdhMHc6fIGcRBKVgry8TgBDgTPdcv8STt5bN+hX3umAaFUkkLauO6Ajcd+b237sOYMtHCXNXHgs+2ZEc8sYSOn00DOIOHnUqgLjK+0PZwlpSo06hpwzpNV8a+zxIDdr+kz4uLa7I7PQsb8iSjaEVXUkaBCByR9dNbQ9wa4kiKTlQEe5SN8\\\\\\\/8Tnl81ALx3y\\\\\\\/je1NvL4Z9yv\\\\\\\/uK7Bc0P96u1myUQUKh38zBXrA+VCOL6JgM1QC+YPMxSIO+tPSoxpfmd6JDGum3j8ZL6Zx78Ive5ne4eRD\\\\\\\/Aw==\\\u0022,\\\u0022iv\\\u0022:\\\u00223gcKQLR4ZWKwUhV0SRHk4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022af4d8c36e07e40d9c6c13e56d57d9561402bca70f6e51e9a15c7cb7d4c57bcc5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487094 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3731" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OPnghKViyFS8e_FrYsV8OR_9a-UrVzEsXdDPjArdJsw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3731" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nJ-EFDK8nf9OwTRceSi6ZWUNBLQZ7ijEPsdk88PqtOA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3719" title="Mechanisms of Transmission Ratio Distortion at Hybrid Sterility Loci Within and Between Mimulus Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3745" title="Serotonin Drives Predatory Feeding Behavior via Synchronous Feeding Rhythms in the Nematode Pristionchus pacificus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487094" id="node487094" data-pisa="ggg;7/11/3731" data-pisa-master="ggg;g3.117.300227" data-apath="/ggg/7/11/3731.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Meredith B. Metzger</span>, <span class="highwire-citation-author" data-delta="1">Jessica L. Scales</span>, <span class="highwire-citation-author" data-delta="2">Mitchell F. Dunklebarger</span> and <span class="highwire-citation-author" data-delta="3">Allan M. Weissman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3731-3743;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487094" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Meredith B. Metzger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meredith%2BB.%2BMetzger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metzger%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMeredith%2BB.%2BMetzger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:metzgermb@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">metzgermb@mail.nih.gov</a>
<a href="mailto:weissmaa@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">weissmaa@mail.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jessica L. Scales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BL.%2BScales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scales%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BL.%2BScales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mitchell F. Dunklebarger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitchell%2BF.%2BDunklebarger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunklebarger%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMitchell%2BF.%2BDunklebarger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Allan M. Weissman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allan%2BM.%2BWeissman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weissman%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllan%2BM.%2BWeissman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:metzgermb@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">metzgermb@mail.nih.gov</a>
<a href="mailto:weissmaa@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">weissmaa@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3731" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487094" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487094/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3731.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487094" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487094/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3731.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487094" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487094/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3731.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487094" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487094/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3731.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487094" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487094/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Protein degradation by the ubiquitin-proteasome system is essential to many processes. We sought to assess its involvement in the turnover of mitochondrial proteins in <em>Saccharomyces cerevisiae</em>. We find that deletion of a specific ubiquitin ligase (E3), <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>, increases the abundance of a temperature-sensitive mitochondrial protein, <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA, when it is expressed from a centromeric plasmid. Deletion of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> unexpectedly elevates the levels of other proteins expressed from centromeric plasmids. Loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> does not increase the rate of turnover of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA, affect total protein synthesis, or increase the protein levels of chromosomal genes. Instead, <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> appears to increase the incidence of missegregation of centromeric plasmids relative to their normal 1:1 segregation. After generations of growth with selection for the plasmid, ongoing missegregation would lead to elevated plasmid DNA, mRNA, and protein, all of which we observe in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. The only known substrate of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is the centromeric histone H3 variant <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>, which is targeted for proteasomal degradation after ubiquitination by <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>. However, <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression alone does not phenocopy <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> in increasing plasmid DNA and protein levels. Instead, elevation of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> leads to an apparent increase in 1:0 plasmid segregation events. Further, 2 μm high-copy yeast plasmids also missegregate in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, but not when <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> alone is overexpressed. These findings demonstrate that <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is required for the faithful inheritance of both centromeric and 2 μm plasmids. Moreover, the effects that loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> has on plasmid segregation cannot be accounted for by increased levels of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ubiquitination" class="hw-term hw-article-keyword hw-article-keyword-ubiquitination" rel="nofollow">ubiquitination</a></li><li class="kwd"><a href="/keyword/cse4p" class="hw-term hw-article-keyword hw-article-keyword-cse4p" rel="nofollow">Cse4p</a></li><li class="kwd"><a href="/keyword/plasmid-missegregation" class="hw-term hw-article-keyword hw-article-keyword-plasmid-missegregation" rel="nofollow">plasmid missegregation</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/cen-plasmid" class="hw-term hw-article-keyword hw-article-keyword-cen-plasmid" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">CEN</em> plasmid</a></li><li class="kwd"><a href="/keyword/2%C2%A0%CE%BCm-plasmid" class="hw-term hw-article-keyword hw-article-keyword-2&#x3BC;m-plasmid" rel="nofollow">2 μm plasmid</a></li></ul><p id="p-3">Post-translational modification of proteins with the 76-amino acid protein ubiquitin targets substrates for a variety of fates. Ubiquitination requires the sequential action of three classes of enzymes: ubiquitin-activating enzyme (E1), a ubiquitin-conjugating enzyme (E2), and one of many different ubiquitin ligases (E3s), which confer exquisite substrate specificity to the process (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zheng and Shabek 2017</a>). Really Interesting New Gene (RING)-type E3s can mediate the transfer of ubiquitin directly from E2 to a substrate, generally onto a substrate lysine residue (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Metzger <em>et al.</em> 2014</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sundaramoorthy <em>et al.</em> 2017</a>). Substrates may be modified with a single ubiquitin or ubiquitin chains. Chains of four or more ubiquitins linked through lysine 48 (K48) of ubiquitin represent the archetypical targeting signal for degradation by the 26S proteasome (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chau <em>et al.</em> 1989</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Finley <em>et al.</em> 1994</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Thrower <em>et al.</em> 2000</a>). However, it is now evident that other ubiquitin chains can also target substrates for proteasomal degradation (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akutsu <em>et al.</em> 2016</a>).</p><p id="p-4">Although the ubiquitin-proteasome system (UPS) directly mediates protein degradation, it can have diverse cellular effects on RNA and DNA. The levels of many mRNAs are affected by UPS-mediated degradation of transcriptional activators or repressors (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yao and Ndoja 2012</a>); one example of this is the degradation of the tumor suppressor p53 by the E3 Mdm2 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fang <em>et al.</em> 2000</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Honda and Yasuda 2000</a>). The levels of specific mRNAs can also be affected by cotranslational protein quality control (QC), where monoubiquitination of 40S ribosomal proteins during ribosome stalling leads to degradation of both the mRNA and nascent polypeptide (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Doma and Parker 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bengtson and Joazeiro 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Juszkiewicz and Hegde 2017</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Sundaramoorthy <em>et al.</em> 2017</a>). The processes of DNA replication, segregation, and repair are also all known to be regulated by the UPS (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cipolla <em>et al.</em> 2016</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Garcia-Rodriguez <em>et al.</em> 2016</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Renaudin <em>et al.</em> 2016</a>).</p><p id="p-5">Chromosomal DNA replication and segregation are tightly regulated by cell cycle checkpoints, and errors can have catastrophic effects on cell viability. However, plasmid DNA levels can often be modulated without such effects. In <em>Saccharomyces cerevisiae</em>, two types of plasmids commonly used in the laboratory are centromere-containing (<em>CEN</em>) plasmids and 2 μm plasmids (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sikorski and Hieter 1989</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Christianson <em>et al.</em> 1992</a>). Both classes have been engineered to encode selectable marker genes that ensure plasmid maintenance under different selective growth conditions employed in the laboratory. <em>CEN</em> plasmids also contain point centromere DNA sequences required for 1:1 equal plasmid segregation into mother and daughter cells and an autonomously replicating sequence (ARS) required for plasmid replication once per cell division in synchrony with chromosome replication (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Sikorski and Hieter 1989</a>). These features of the <em>CEN</em> plasmid ensure that the plasmid remains, on average, at one copy per yeast cell, although the rate of mitotic loss of <em>CEN</em> plasmids is ∼1000 times greater than the rate of chromosome loss (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Clarke and Carbon 1980</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hieter <em>et al.</em> 1985</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Koshland <em>et al.</em> 1985</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Murray and Szostak 1986</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hegemann <em>et al.</em> 1988</a>).</p><p id="p-6">The 2 μm plasmids used for genetic manipulation in yeast contain DNA sequence derived from endogenous 2 μm circles found in the yeast nucleus. This sequence contains an origin of replication and plasmid partitioning elements that enable 2 μm plasmids to be stably maintained (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Yen Ting <em>et al.</em> 2014</a>). The 2 μm sequence also contains an amplification system, allowing these plasmids to remain at high copy number (∼10–30 copies per cell) uniformly across the population, despite missegregation events (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Christianson <em>et al.</em> 1992</a>).</p><p id="p-7">In this study, we set out to examine the role of the UPS in QC at yeast mitochondria, but unexpectedly discovered a role for the UPS in plasmid segregation. Loss of a ubiquitin ligase, <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>, increases the levels of proteins expressed from <em>CEN</em> plasmids without affecting their rates of degradation. Interestingly, we find that <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is required for the proper segregation of both <em>CEN</em> and 2 μm plasmids. Loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> results in plasmid alterations consistent with an increase in unequal plasmid segregation. This missegregation is distinct from what we observe herein upon overexpression of the only known <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> substrate, the centromeric histone H3 variant <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>. These results suggest that a previously unappreciated target or function of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is necessary for proper plasmid segregation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains, plasmids, and growth conditions</h3><p id="p-8"><em>S. cerevisiae</em> strains and plasmids are listed in Supplemental Material, Table S1 and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>, respectively. Cells were cultured at 30° in minimal media supplemented with 2% glucose and the appropriate amino acids, unless otherwise indicated. For experiments requiring galactose induction of <em>pGAL-FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">CSE4</a></em>, cells were cultured in 2% raffinose for 12–24 hr prior to the addition of 2% galactose for the indicated time. For spot growth assays, 10-fold serial dilutions beginning with 0.1 OD<sub>600</sub> units of cells were spotted to minimal media lacking leucine (SC-Leu) and grown at 25° for 3 d, or 37° for 2 d.</p></div><div id="sec-3" class="subsection"><h3>Antibodies</h3><p id="p-9">Rabbit polyclonal anti-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pinsky <em>et al.</em> 2003</a>) and rabbit polyclonal anti-Sam35p (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chan and Lithgow 2008</a>) were generous gifts from Sue Biggins and Trevor Lithgow, respectively. Commercial antibodies used were: mouse monoclonal <a href="http://www.yeastgenome.org/locus/S000007260/overview">Cox1p</a>, mouse monoclonal <a href="http://www.yeastgenome.org/locus/S000004249/overview">Hsp60p</a>, mouse monoclonal Porin and rabbit polyclonal Prc1p (all from abcam); mouse monoclonal myc (9E10) and mouse monoclonal green fluorescent protein (GFP; both from Santa Cruz Biotechnology); mouse monoclonal phosphoglycerate kinase (<a href="http://www.yeastgenome.org/locus/S000000605/overview">Pgk1p</a>; Life Technologies); and rat monoclonal peroxidase-conjugated anti-HA (3F10; Roche).</p></div><div id="sec-4" class="subsection"><h3>Immunoblotting and cycloheximide chase</h3><p id="p-10">Immunoblotting and cycloheximide (CHX) chase analysis were performed as described previously (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Metzger <em>et al.</em> 2013</a>), with the following changes. Some experiments were carried out at 25 or 37° as indicated. For CHX chases at 37°, cells were cultured at the permissive temperature of 25° until the addition of CHX, at which time the temperature was increased to 37° to accelerate the rate of turnover of the temperature-sensitive (ts-) protein.</p></div><div id="sec-5" class="subsection"><h3>Pulse-chase metabolic labeling and <sup>35</sup>S incorporation rate</h3><p id="p-11">Pulse-chase analyses were performed as described previously (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Metzger <em>et al.</em> 2013</a>), with the following exceptions: cells were pulsed with <sup>35</sup>S-labeled methionine/cysteine amino acid mix (EasyTagEXPRESS<sup>35</sup>S Protein Labeling Mix; Perkin Elmer) for 25 min at 25° (for <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA or when using strains WCG4a and WCG4-11/21a) or 30 min at 30° (for <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA), and cells were chased with 20 mM unlabeled methionine/cysteine at 37° (for <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA or when using strains WCG4a and WCG4-11/21a) or 30° (for <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA), for the indicated time points. Radioactive signal from three independent experiments was quantified as previously and fit to a one-phase decay curve using GraphPad Prism 7 (GraphPad Software).</p><p id="p-12">To calculate the relative amount of protein that incorporated <sup>35</sup>S, the radioactive signal at the zero time point was quantified in wild-type (WT) <em>vs.</em> <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells for three independent pulse-chases. The signal was normalized to the signal from WT cells. Prior to immunoprecipitation of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA or <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, an aliquot of total <sup>35</sup>S-labeled protein lysate was subject to TCA precipitation and counted using a scintillation counter to obtain values for incorporation into total protein. These values were also normalized to the signal from WT cells.</p><p id="p-13">The rate of <sup>35</sup>S incorporation in WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells was determined as follows. Cells were grown in minimal media supplemented with leucine, histidine, and uracil at 30° prior to splitting cultures in half. The first half of cells were labeled with EasyTagEXPRESS<sup>35</sup>S Protein Labeling Mix (2 μCi/OD<sub>600</sub> of yeast cells) and 0.5 μM unlabeled methionine at 30°. These cells were harvested at the time of addition (0 min) and every 10 min after up to 1 hr and then protein isolated like pulse-chase samples. An aliquot of <sup>35</sup>S-labeled protein was run on an SDS-PAGE gel and the radioactive signal was quantified. At the same time as the <sup>35</sup>S labeling, 0.5 μM unlabeled methionine was added to the second half of the cells and the OD<sub>600</sub> was measured at each time point. The radioactive signal was graphed relative to the cell density (OD<sub>600</sub>) at each time point.</p></div><div id="sec-6" class="subsection"><h3>Mitochondrial fractionation and proteinase K treatment</h3><p id="p-14">Mitochondria were isolated as described previously (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gregg <em>et al.</em> 2009</a>), with the following changes. Cultures were grown in minimal media containing the appropriate amino acids at 25°. Cells were incubated in DTT Buffer (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gregg <em>et al.</em> 2009</a>) for 30 min at 25° and in Zymolyase Buffer (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Gregg <em>et al.</em> 2009</a>) with Zymolyase-100T (MP Biomedicals) for 45 min at 25°, after which time the resulting spheroplasts were washed and resuspended in an equal volume of Zymolyase Buffer without Zymolyase-100T, either at 25° or prewarmed to 37°. CHX (100 μg/ml) was added and spheroplasts were incubated without shaking at 25 or 37° for 1 hr. An aliquot of spheroplasts was removed at this time (“Total”). The remainder were homogenized and fractionated at 12,000 × <em>g</em>, as per protocol. A portion of the supernatant resulting from the first 12,000 × <em>g</em> spin (“S”) was reserved. The resulting mitochondrial pellet (“P”) was resuspended in SDS-PAGE sample buffer. Total and “S” fractions were precipitated in 10% TCA and washed in acetone prior to resuspending in SDS-PAGE sample buffer. Equivalent proportions of Total, S, and P were analyzed by SDS-PAGE and immunoblotting.</p><p id="p-15">To determine whether <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA was mislocalized to the mitochondrial surface, mitochondria isolated as above were resuspended in SM buffer (10 mM MOPS/KOH pH 7.2, 250 mM sucrose). Equivalent amounts were aliquoted to tubes containing either SM buffer alone, SM buffer plus Proteinase K (PrK; 10 μg/ml), SM buffer plus Triton X-100 (Tx-100; 2%), or SM buffer plus both PrK and Tx-100, and incubated for 20 min on ice. Reactions were quenched by the addition of 1× protease inhibitor cocktail (Roche) and PMSF (1 mM), at which time mitochondria were reisolated by spinning at 12,500 × <em>g</em>, resuspended in SDS-PAGE sample buffer, and examined by SDS-PAGE and immunoblotting.</p></div><div id="sec-7" class="subsection"><h3>Microscopy</h3><p id="p-16">Logarithmically growing yeast cells were immobilized on coverslips coated with conconavalin A and imaged using the 100× objective (1.42 Plan Apo objective with 1.5× magnifying tube lens) of a Nikon Eclipse Ti inverted microscope, equipped with a 6.4 μm-pixel CoolSNAP HQ camera (Photometrics) and Intensilight C-HGFIE illuminator. Z-Sections of 200 nm thick each, spanning a total range of 6 μm, were acquired. ImageJ [National Institutes of Health (NIH)] was used to process the acquired images.</p></div><div id="sec-8" class="subsection"><h3>Real-time PCR (qPCR)</h3><p id="p-17">For qPCR analysis of WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strains, six OD<sub>600</sub> units of cells growing logarithmically at 30° in the appropriate glucose-containing media were harvested and processed. For qPCR analysis of WT, <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, and <em>pGAL-FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">CSE4</a></em> strains, two OD<sub>600</sub> units of cells growing logarithmically at 30° in selective media containing 2% raffinose/2% galactose for 24 hr were harvested and processed. For qPCR on genomic/plasmid DNA, yeast DNA was isolated via glass bead lysis in TSENT buffer (10 mM Tris-HCl pH 8, 100 mM NaCl, 1 mM EDTA, 1% SDS, 2% Tx-100), and phenol/chloroform/isoamyl alcohol (25:24:1), followed by ethanol precipitation of nucleic acids. RNA was removed by RNase A (Thermo Scientific) treatment, followed by reprecipitation of total DNA with ethanol. For reverse transcription (RT)-qPCR from cDNA, RNA was isolated from logarithmically growing cells using the RNeasy Mini Kit (Qiagen) following the protocol for “Purification of Total RNA from Yeast” via an enzymatic lysis with DNase treatment. The iScript Advanced cDNA Synthesis kit (Bio-Rad) was used to synthesize cDNA.</p><p id="p-18">Reactions for qPCR contained 10 ng of genomic/plasmid DNA or cDNA, SsoAdvanced universal SYBR Green supermix (Bio-Rad), and either PrimePCR probes to <em><a href="http://www.yeastgenome.org/locus/S000001855/overview">ACT1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005780/overview">SEC63</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000314/overview">ALG1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000383/overview">FZO1</a></em>, or <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">MIA40</a></em> (Bio-Rad) or primer pairs specific to <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em>, <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA</em>, or the pRS315 vector/<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a> junction, designed according to published recommendations (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nolan <em>et al.</em> 2006</a>). Reactions were performed using the PrimePCR cycling program (Bio-Rad) in a CFX96 Touch Real-Time PCR Detection System (Bio-Rad). At least three biological replicates and three technical replicates were done for each yeast strain, as well as the appropriate no RT and template controls. Normalized gene expression data were analyzed using CFX Manager Software with <em><a href="http://www.yeastgenome.org/locus/S000001855/overview">ACT1</a></em> (actin) as the reference target and normalized to the WT strain values. Data were graphed using GraphPad Prism using an unpaired <em>t</em>-test to obtain two-tailed <em>P</em>-values.</p></div><div id="sec-9" class="subsection"><h3>Plasmid loss assay</h3><p id="p-19">Plasmid loss of <em>CEN</em> or 2 μm plasmids was measured at 30°. For experiments with WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strains, cells were grown in selective media (SC-Leu) overnight, then an appropriate dilution was plated onto both rich medium (YEPD) and SC-Leu, and plates were incubated for 2–3 d, respectively (0 hr without selection). The remainder of the culture was diluted into YEPD and rediluted after 24 hr. After 48 hr total in nonselective growth media, an appropriate dilution was plated onto YEPD and SC-Leu plates and incubated as above. Colonies were counted and the fraction of plasmid-bearing cells was calculated as the number of colonies on SC-Leu relative to the number of colonies on YEPD for each time point. Three or more biological replicates were performed for each yeast strain. Data were analyzed using GraphPad Prism and an unpaired <em>t</em>-test was used to calculate two-tailed <em>P</em>-values.</p><p id="p-20">For plasmid loss experiments with <em>pGAL-FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">CSE4</a></em> and its isogenic WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strains, cells were grown in 2% raffinose/2% galactose-containing minimal media lacking leucine for 24 hr, then an appropriate dilution was plated onto YEPD and SC-Leu as above (0 hr without selection). The remainder of the culture was diluted into 2% raffinose/2% galactose-containing minimal media with leucine and rediluted after 24 hr. After a total of 48 hr without selection, an appropriate dilution was plated onto YEPD and SC-Leu plates and incubated as above. The fraction of plasmid-bearing cells was calculated and graphed as described above.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-21">Strains, plasmids, and reagents are available upon request.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>The temperature-sensitive protein mia40-4pHA is an unstable mitochondrial protein</h3><p id="p-22">The UPS plays well-characterized and critical roles in protein QC at the endoplasmic reticulum (ER), ribosome, nucleus, and cytosol (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brandman and Hegde 2016</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Jones and Gardner 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Defenouillere and Fromont-Racine 2017</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Preston and Brodsky 2017</a>). Recent studies have revealed a role for the UPS in QC at mitochondria as well (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Braun and Westermann 2017</a>), but far less is known regarding the mechanisms or scope of this type of degradation. To begin to explore the determinants of mitochondrial protein turnover, we analyzed a ts- allele of the inner membrane (IM) protein, <a href="http://www.yeastgenome.org/locus/S000001678/overview">Mia40</a>p, called <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chacinska <em>et al.</em> 2004</a>). For ease of analysis, we tagged ts- <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4</em> and WT <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">MIA40</a></em> with the hemagglutinin epitope (HA) and expressed them from <em>CEN</em> yeast plasmids. Haploid yeast cells deleted for <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">MIA40</a></em>, expressing only HA-tagged <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4</em> (<em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA</em>), are inviable at the nonpermissive temperature of 37°, while those expressing WT <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">MIA40</a>-HA</em> are not ts- (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). To assess protein stability, we performed CHX chase analysis, where the rate of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA turnover was analyzed after cells were treated with CHX to inhibit any new protein synthesis. <em>CEN</em> plasmid-expressed <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA is unstable at 37° relative to its turnover at the permissive temperature of 25° or the turnover of WT <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA at either 25 or 37° (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="mia40-4pHA is an unstable mitochondrial protein. (A) Ten-fold serial dilutions of mia40&#x394; cells expressing mia40-4-HA or MIA40-HA from CEN plasmids at permissive (25&#xB0;) or nonpermissive (37&#xB0;) temperatures were spotted to minimal media lacking leucine. (B) WT yeast were treated with CHX at 25 or 37&#xB0; for the indicated times to assess the degradation of CEN plasmid-expressed mia40-4pHA or Mia40pHA. Proteins were detected by immunoblotting with HA antibody. Phosphoglycerate kinase (PGK) served as a protein loading control. (C) Lysates from cells expressing Mia40pHA (top) or mia40-4pHA (bottom) from CEN plasmids at 25 or 37&#xB0; were fractionated into mitochondrial pellets (P) and postmitochondrial supernatants (S). Fractions were subject to immunoblotting with antibodies to HA, Porin, or PGK. (D) Differential interference contrast (DIC) and fluorescence microscopy at 25 and 37&#xB0; of cells coexpressing mia40-4pGFP or Mia40pGFP and a mitochondrial matrix-targeted RFP variant, mtERFP. Bar, 10&#xA0;&#x3BC;m and &#x201C;merge&#x201D; is an overlay of GFP and RFP channels. (E) Mitochondria isolated as in (C) were treated with proteinase K (PrK) alone, or in combination with Triton X-100 (Tx-100), and subject to immunoblotting with antibodies specific to the OM protein Sam35p, the IM protein Cox1p, or the matrix protein Hsp60p." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-196213172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;mia40-4pHA is an unstable mitochondrial protein. (A) Ten-fold serial dilutions of &lt;em&gt;mia40&#x394;&lt;/em&gt; cells expressing &lt;em&gt;mia40-4-HA&lt;/em&gt; or &lt;em&gt;MIA40-HA&lt;/em&gt; from &lt;em&gt;CEN&lt;/em&gt; plasmids at permissive (25&#xB0;) or nonpermissive (37&#xB0;) temperatures were spotted to minimal media lacking leucine. (B) WT yeast were treated with CHX at 25 or 37&#xB0; for the indicated times to assess the degradation of &lt;em&gt;CEN&lt;/em&gt; plasmid-expressed mia40-4pHA or Mia40pHA. Proteins were detected by immunoblotting with HA antibody. Phosphoglycerate kinase (PGK) served as a protein loading control. (C) Lysates from cells expressing Mia40pHA (top) or mia40-4pHA (bottom) from &lt;em&gt;CEN&lt;/em&gt; plasmids at 25 or 37&#xB0; were fractionated into mitochondrial pellets (P) and postmitochondrial supernatants (S). Fractions were subject to immunoblotting with antibodies to HA, Porin, or PGK. (D) Differential interference contrast (DIC) and fluorescence microscopy at 25 and 37&#xB0; of cells coexpressing mia40-4pGFP or Mia40pGFP and a mitochondrial matrix-targeted RFP variant, mtERFP. Bar, 10&#xA0;&#x3BC;m and &#x201C;merge&#x201D; is an overlay of GFP and RFP channels. (E) Mitochondria isolated as in (C) were treated with proteinase K (PrK) alone, or in combination with Triton X-100 (Tx-100), and subject to immunoblotting with antibodies specific to the OM protein Sam35p, the IM protein Cox1p, or the matrix protein Hsp60p.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3731/F1.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3731/F1.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487134" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child"><a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA is an unstable mitochondrial protein. (A) Ten-fold serial dilutions of <em><a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>Δ</em> cells expressing <em><a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4-HA</em> or <em><a href="https://www.yeastgenome.org/locus/S000001678">MIA40</a>-HA</em> from <em>CEN</em> plasmids at permissive (25°) or nonpermissive (37°) temperatures were spotted to minimal media lacking leucine. (B) WT yeast were treated with CHX at 25 or 37° for the indicated times to assess the degradation of <em>CEN</em> plasmid-expressed <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA or <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA. Proteins were detected by immunoblotting with HA antibody. Phosphoglycerate kinase (PGK) served as a protein loading control. (C) Lysates from cells expressing <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA (top) or <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA (bottom) from <em>CEN</em> plasmids at 25 or 37° were fractionated into mitochondrial pellets (P) and postmitochondrial supernatants (S). Fractions were subject to immunoblotting with antibodies to HA, Porin, or PGK. (D) Differential interference contrast (DIC) and fluorescence microscopy at 25 and 37° of cells coexpressing <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pGFP or <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pGFP and a mitochondrial matrix-targeted RFP variant, mtERFP. Bar, 10 μm and “merge” is an overlay of GFP and RFP channels. (E) Mitochondria isolated as in (C) were treated with proteinase K (PrK) alone, or in combination with Triton X-100 (Tx-100), and subject to immunoblotting with antibodies specific to the OM protein Sam35p, the IM protein Cox1p, or the matrix protein Hsp60p.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">We next confirmed the proper mitochondrial localization of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA by mitochondrial fractionation. Similar to WT <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA and the mitochondrial resident protein Porin, <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA is found exclusively in the 12,000 × <em>g</em> mitochondrial pellet (P) at both permissive and nonpermissive temperatures (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). A GFP-tagged version of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4p (<a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pGFP) also colocalizes at both temperatures with a mitochondria-targeted enhanced red fluorescent protein (mtERFP) (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Scholz <em>et al.</em> 2013</a>), nearly identically to WT <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pGFP. These data indicate that <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4p is found at mitochondria. To confirm that <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA is not mislocalized to the outside of mitochondria, mitochondria from cells expressing <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA were isolated and treated with proteinase K (PrK). Unlike a control outer membrane (OM) protein (<a href="http://www.yeastgenome.org/locus/S000001125/overview">Sam35p</a>), which is subject to proteolysis with PrK treatment alone, <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA is only subject to PrK proteolysis when detergent (Tx-100) is also added, similar to control IM (<a href="http://www.yeastgenome.org/locus/S000007260/overview">Cox1p</a>) and matrix (<a href="http://www.yeastgenome.org/locus/S000004249/overview">Hsp60p</a>) proteins. Thus, <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA is a mitochondrially localized, temperature-destabilized protein suitable for QC studies.</p></div><div id="sec-13" class="subsection"><h3>Loss of Psh1p increases levels of mia40-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, without affecting their rates of turnover</h3><p id="p-25">To identify UPS machinery that might play a role in the temperature-dependent instability of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA, we transformed the <em>CEN</em> plasmid expressing <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA into a collection of nonessential yeast deletion strains of genes with known or putative roles in the UPS (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ravid and Hochstrasser 2007</a>). We then screened by immunoblotting for deletions that increase the steady-state levels of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA, indicating potential involvement of these genes in its degradation. From this screen, we identified the ubiquitin ligase, <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>. Deletion of <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">PSH1</a></em> significantly and reproducibly increases the levels of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is a RING-type E3 whose only known substrate is a centromeric histone H3 variant, <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (CENP-A) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hewawasam <em>et al.</em> 2010</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ranjitkar <em>et al.</em> 2010</a>). To assess whether the increased levels of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4HA are a consequence of decreased <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA degradation in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells, we carried out CHX chase experiments (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Accounting for the increased steady-state level of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, 0 hr), it is not evident that deletion of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> decreases the turnover of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA, which would be expected if <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> was an E3 for <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA. This led us to assess the effect that loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> had on another mitochondrial protein, <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA. This mitofusin is an extensively studied UPS substrate targeted for degradation by <a href="http://www.yeastgenome.org/locus/S000004360/overview">Mdm30p</a> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Escobar-Henriques <em>et al.</em> 2006</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cohen <em>et al.</em> 2008</a>) as a consequence of ubiquitination by the E3, SCF<sup><a href="http://www.yeastgenome.org/locus/S000004360/overview">Mdm30p</a></sup> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Cohen <em>et al.</em> 2008</a>). The steady-state level of <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA expressed from a <em>CEN</em> plasmid is also increased in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). However, like <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA, CHX chase analysis of <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells does not show a clear effect on its turnover (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>, 0 hr).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of Psh1p affects the steady-state levels of mia40-4pHA and Fzo1pHA, without affecting their rates of turnover or total cellular protein levels. (A) The steady-state protein level of CEN plasmid-expressed mia40-4pHA was assessed at 37&#xB0; in WT and psh1&#x394; cells by immunoblotting with HA antibody. PGK serves as a control for equal loading. (B) CHX chase for the indicated times assessing turnover of mia40-4pHA expressed from a CEN plasmid in WT and psh1&#x394; yeast cells. Proteins were detecting by immunoblotting. (C) The steady-state protein levels of Fzo1pHA, analyzed as in (A) except at 30&#xB0;. (D) CHX chase of Fzo1pHA, analyzed as in (B) except at 30&#xB0;. (E) Representative 35S pulse-chase analysis to assess turnover of mia40-4pHA in WT and psh1&#x394; cells at 37&#xB0; at the indicated time points. The mean of three independent experiments is graphed below, with error bars depicting the SD. (F) Representative 35S pulse-chase analysis of the turnover of Fzo1pHA in WT and psh1&#x394; at 30&#xB0; at the indicated time points, analyzed and graphed as in (E). (G) Quantification of 35S incorporated into mia40-4pHA, Fzo1pHA, or total protein during a 30-min pulse with 35S-labeled methionine/cysteine in psh1&#x394; and WT strains. Values were normalized to the incorporation in the WT strain. The average and SD of three independent experiments is shown. (H) The rate of total protein synthesis in WT and psh1&#x394; strains was measured by analyzing 35S methionine/cysteine incorporation into total protein relative to cell density (OD600) over time. (I) Representative 35S pulse-chase analysis of the turnover of mia40-4pHA in a pre1-1 pre2-2 proteasome mutant strain and its PRE1 PRE2 isogenic WT strain, analyzed and graphed as in (E). (J) Representative 35S pulse-chase analysis of the turnover of Fzo1pHA in a pre1-1 pre2-2 proteasome mutant strain and its PRE1 PRE2 isogenic WT strain, analyzed and graphed as in (F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-196213172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Loss of Psh1p affects the steady-state levels of mia40-4pHA and Fzo1pHA, without affecting their rates of turnover or total cellular protein levels. (A) The steady-state protein level of &lt;em&gt;CEN&lt;/em&gt; plasmid-expressed mia40-4pHA was assessed at 37&#xB0; in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells by immunoblotting with HA antibody. PGK serves as a control for equal loading. (B) CHX chase for the indicated times assessing turnover of mia40-4pHA expressed from a &lt;em&gt;CEN&lt;/em&gt; plasmid in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; yeast cells. Proteins were detecting by immunoblotting. (C) The steady-state protein levels of Fzo1pHA, analyzed as in (A) except at 30&#xB0;. (D) CHX chase of Fzo1pHA, analyzed as in (B) except at 30&#xB0;. (E) Representative &lt;sup&gt;35&lt;/sup&gt;S pulse-chase analysis to assess turnover of mia40-4pHA in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells at 37&#xB0; at the indicated time points. The mean of three independent experiments is graphed below, with error bars depicting the SD. (F) Representative &lt;sup&gt;35&lt;/sup&gt;S pulse-chase analysis of the turnover of Fzo1pHA in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; at 30&#xB0; at the indicated time points, analyzed and graphed as in (E). (G) Quantification of &lt;sup&gt;35&lt;/sup&gt;S incorporated into mia40-4pHA, Fzo1pHA, or total protein during a 30-min pulse with &lt;sup&gt;35&lt;/sup&gt;S-labeled methionine/cysteine in &lt;em&gt;psh1&#x394;&lt;/em&gt; and WT strains. Values were normalized to the incorporation in the WT strain. The average and SD of three independent experiments is shown. (H) The rate of total protein synthesis in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; strains was measured by analyzing &lt;sup&gt;35&lt;/sup&gt;S methionine/cysteine incorporation into total protein relative to cell density (OD&lt;sub&gt;600&lt;/sub&gt;) over time. (I) Representative &lt;sup&gt;35&lt;/sup&gt;S pulse-chase analysis of the turnover of mia40-4pHA in a &lt;em&gt;pre1-1 pre2&lt;/em&gt;-2 proteasome mutant strain and its &lt;em&gt;PRE1 PRE2&lt;/em&gt; isogenic WT strain, analyzed and graphed as in (E). (J) Representative &lt;sup&gt;35&lt;/sup&gt;S pulse-chase analysis of the turnover of Fzo1pHA in a &lt;em&gt;pre1-1 pre2&lt;/em&gt;-2 proteasome mutant strain and its &lt;em&gt;PRE1 PRE2&lt;/em&gt; isogenic WT strain, analyzed and graphed as in (F).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3731/F2.medium.gif" width="260" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3731/F2.medium.gif" width="260" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487136" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Loss of Psh1p affects the steady-state levels of <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, without affecting their rates of turnover or total cellular protein levels. (A) The steady-state protein level of <em>CEN</em> plasmid-expressed <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA was assessed at 37° in WT and <em>psh1Δ</em> cells by immunoblotting with HA antibody. PGK serves as a control for equal loading. (B) CHX chase for the indicated times assessing turnover of <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA expressed from a <em>CEN</em> plasmid in WT and <em>psh1Δ</em> yeast cells. Proteins were detecting by immunoblotting. (C) The steady-state protein levels of <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, analyzed as in (A) except at 30°. (D) CHX chase of <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, analyzed as in (B) except at 30°. (E) Representative <sup>35</sup>S pulse-chase analysis to assess turnover of <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA in WT and <em>psh1Δ</em> cells at 37° at the indicated time points. The mean of three independent experiments is graphed below, with error bars depicting the SD. (F) Representative <sup>35</sup>S pulse-chase analysis of the turnover of <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA in WT and <em>psh1Δ</em> at 30° at the indicated time points, analyzed and graphed as in (E). (G) Quantification of <sup>35</sup>S incorporated into <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA, <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, or total protein during a 30-min pulse with <sup>35</sup>S-labeled methionine/cysteine in <em>psh1Δ</em> and WT strains. Values were normalized to the incorporation in the WT strain. The average and SD of three independent experiments is shown. (H) The rate of total protein synthesis in WT and <em>psh1Δ</em> strains was measured by analyzing <sup>35</sup>S methionine/cysteine incorporation into total protein relative to cell density (OD<sub>600</sub>) over time. (I) Representative <sup>35</sup>S pulse-chase analysis of the turnover of <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA in a <em>pre1-1 pre2</em>-2 proteasome mutant strain and its <em>PRE1 PRE2</em> isogenic WT strain, analyzed and graphed as in (E). (J) Representative <sup>35</sup>S pulse-chase analysis of the turnover of <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA in a <em>pre1-1 pre2</em>-2 proteasome mutant strain and its <em>PRE1 PRE2</em> isogenic WT strain, analyzed and graphed as in (F).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To quantitatively assess the rate of turnover of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, we examined their degradation in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> and WT strains by <sup>35</sup>S pulse-chase metabolic labeling. Quantification of these analyses reveals no significant difference in the rate of degradation of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2E</a>; t<sub>1/2</sub> = 55 ± 9 and 47 ± 8 min for WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, respectively) or <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2F</a>; t<sub>1/2</sub> = 85 ± 9 and 100 ± 10 min for WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, respectively) in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> compared to WT. However, there is a consistent increase in the amount of <sup>35</sup>S-labeled <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA in the <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strain (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2, E and F</a>, compare 0 hr time points for WT <em>vs.</em> <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>; quantified in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2G</a>), despite similar rates of growth. This indicates that more <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA has been synthesized in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>. Importantly, this is not reflective of an increase in the total radiolabeled protein pool in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2G</a>, total protein), so <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA appear to be differentially increased. The rate of total protein synthesis is also similar between the WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strains (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2H</a>).</p><p id="p-28">Since <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> does not appear to be targeting <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA for degradation, we wondered whether the proteasome was required for its degradation. To analyze this, <sup>35</sup>S pulse-chase analysis of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA was carried out in a <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a></em>-2 proteasome mutant strain (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Heinemeyer <em>et al.</em> 1991</a>). The rate of turnover of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA was also unchanged in this mutant strain (t<sub>1/2</sub> = 35 ± 7 and 33 ± 4 min for <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">PRE1</a> <a href="http://www.yeastgenome.org/locus/S000006307/overview">PRE2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-2</em>, respectively). Unlike <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, where there is an increase in the amount of protein synthesized during the pulse with <sup>35</sup>S-labeled methionine/cysteine, there is actually less protein labeled in the <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-2</em> mutant (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2I</a>, compare 0 hr time points), likely due to the impaired growth characteristic of this strain. The <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA protein is also not stabilized by mutations in a vacuolar peptidase (<em><a href="http://www.yeastgenome.org/locus/S000006075/overview">pep4</a>Δ</em>; Figure S1A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>); in generalized autophagy (<em><a href="http://www.yeastgenome.org/locus/S000006070/overview">atg5</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000000174/overview">atg8</a>Δ</em>) or mitophagy machinery (<em><a href="http://www.yeastgenome.org/locus/S000006253/overview">atg11</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000001408/overview">atg32</a>Δ</em>; Figure S1B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>); or in mitochondrial resident proteases (<em><a href="http://www.yeastgenome.org/locus/S000006228/overview">yme1</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000000819/overview">afg3</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000004695/overview">yta12</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000001795/overview">oma1</a>Δ</em>, or <em><a href="http://www.yeastgenome.org/locus/S000000118/overview">pim1</a>Δ</em>; Figure S1C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>). Having assessed the major cellular proteolytic systems, it remains unclear what is responsible for the instability of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA. <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA is stabilized in the proteasome mutant strain (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2J</a>; t<sub>1/2</sub> = 112 ± 15 and 649 ± 17 min for <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">PRE1</a> <a href="http://www.yeastgenome.org/locus/S000006307/overview">PRE2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-2</em>, respectively), as expected for this well-characterized UPS substrate. There is again no increase in the <sup>35</sup>S incorporation into <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA in the <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-2</em> strain (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2J</a>, 0 hr time points).</p></div><div id="sec-14" class="subsection"><h3>Loss of Psh1p (or Psh1p activity) leads to an increase in the steady-state levels of proteins expressed from CEN plasmids, but not those expressed from the chromosome</h3><p id="p-29">We examined whether the ubiquitin ligase activity of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is required to maintain the normal levels of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA proteins seen in WT cells. While re-expression of WT <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> reduces <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells, a previously described <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> RING domain mutant, which lacks E3 activity (<em><a href="http://www.yeastgenome.org/locus/S000005415/overview">PSH1</a></em><sup>C45S C50S</sup>) (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Ranjitkar <em>et al.</em> 2010</a>), does not significantly decrease either <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA or <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>). This indicates that the E3 activity of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is required to maintain the normal levels of these proteins seen in WT cells.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of Psh1p or its ubiquitin ligase activity affects the steady-state levels of many proteins when expressed from plasmids, but not when expressed from the chromosome. (A, B) The steady-state protein levels of CEN-plasmid expressed mia40-4pHA (A) or Fzo1pHA (B) were analyzed in psh1&#x394; cells coexpressing either vector, WT PSH1HA, or RING domain mutant PSH1HAC45S C50S growing at 37&#xB0; (mia40-4pHA) or 30&#xB0; (Fzo1pHA) by immunoblotting with HA antibody. Anti-PGK serves as a loading control. (C) Levels of the mitochondrial proteins Mia40pHA, erv1-2pHA, and mitochondrial-targeted GFP (mtGFP) expressed from CEN plasmids in WT and psh1&#x394; cells were assessed in WT and psh1&#x394; cells at 30&#xB0; as in (A). (D) CEN plasmid-expressed Ura3pHA-CL1, Ste6p*HA, and CPY*HA were analyzed in WT and psh1&#x394; cells at 30&#xB0; as in (A) except CPY*HA was visualized using anti-CPY. (E) Chromosomal proteins (CPY, PGK, and Porin) were analyzed in WT and psh1&#x394; cells by immunoblotting using antibodies specific to these targets. (F, G). Sam35pHA (F) or Sen2pHA (G) expressed from either the genome or a CEN plasmid were analyzed in WT and psh1&#x394; cells at 30&#xB0; as in (A). (H&#x2013;J) CEN plasmid-expressed Fzo1pHA under control of the ADH1 promoter (H) or mia40-4pHA and Mia40pHA under control of the GAL10 promoter (I) or with the native MIA40 3&#x2032; untranslated region (J) were assessed in psh1&#x394; and WT cells at 30&#xB0; (Fzo1pHA) or 37&#xB0; (mia40-4pHA), as in (A)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-196213172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Loss of Psh1p or its ubiquitin ligase activity affects the steady-state levels of many proteins when expressed from plasmids, but not when expressed from the chromosome. (A, B) The steady-state protein levels of &lt;em&gt;CEN&lt;/em&gt;-plasmid expressed mia40-4pHA (A) or Fzo1pHA (B) were analyzed in &lt;em&gt;psh1&#x394;&lt;/em&gt; cells coexpressing either vector, WT &lt;em&gt;PSH1HA&lt;/em&gt;, or RING domain mutant &lt;em&gt;PSH1HA&lt;/em&gt;&lt;sup&gt;C45S C50S&lt;/sup&gt; growing at 37&#xB0; (mia40-4pHA) or 30&#xB0; (Fzo1pHA) by immunoblotting with HA antibody. Anti-PGK serves as a loading control. (C) Levels of the mitochondrial proteins Mia40pHA, erv1-2pHA, and mitochondrial-targeted GFP (mtGFP) expressed from &lt;em&gt;CEN&lt;/em&gt; plasmids in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells were assessed in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells at 30&#xB0; as in (A). (D) &lt;em&gt;CEN&lt;/em&gt; plasmid-expressed Ura3pHA-CL1, Ste6p*HA, and CPY*HA were analyzed in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells at 30&#xB0; as in (A) except CPY*HA was visualized using anti-CPY. (E) Chromosomal proteins (CPY, PGK, and Porin) were analyzed in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells by immunoblotting using antibodies specific to these targets. (F, G). Sam35pHA (F) or Sen2pHA (G) expressed from either the genome or a &lt;em&gt;CEN&lt;/em&gt; plasmid were analyzed in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells at 30&#xB0; as in (A). (H&#x2013;J) &lt;em&gt;CEN&lt;/em&gt; plasmid-expressed Fzo1pHA under control of the &lt;em&gt;ADH1&lt;/em&gt; promoter (H) or mia40-4pHA and Mia40pHA under control of the &lt;em&gt;GAL10&lt;/em&gt; promoter (I) or with the native &lt;em&gt;MIA40&lt;/em&gt; 3&#x2032; untranslated region (J) were assessed in &lt;em&gt;psh1&#x394;&lt;/em&gt; and WT cells at 30&#xB0; (Fzo1pHA) or 37&#xB0; (mia40-4pHA), as in (A).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3731/F3.medium.gif" width="335" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/11/3731/F3.medium.gif" width="335" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487133" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Loss of Psh1p or its ubiquitin ligase activity affects the steady-state levels of many proteins when expressed from plasmids, but not when expressed from the chromosome. (A, B) The steady-state protein levels of <em>CEN</em>-plasmid expressed <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA (A) or <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA (B) were analyzed in <em>psh1Δ</em> cells coexpressing either vector, WT <em>PSH1HA</em>, or RING domain mutant <em>PSH1HA</em><sup>C45S C50S</sup> growing at 37° (<a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA) or 30° (<a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA) by immunoblotting with HA antibody. Anti-PGK serves as a loading control. (C) Levels of the mitochondrial proteins <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA, erv1-2pHA, and mitochondrial-targeted GFP (mtGFP) expressed from <em>CEN</em> plasmids in WT and <em>psh1Δ</em> cells were assessed in WT and <em>psh1Δ</em> cells at 30° as in (A). (D) <em>CEN</em> plasmid-expressed Ura3pHA-CL1, Ste6p*HA, and CPY*HA were analyzed in WT and <em>psh1Δ</em> cells at 30° as in (A) except CPY*HA was visualized using anti-CPY. (E) Chromosomal proteins (CPY, PGK, and Porin) were analyzed in WT and <em>psh1Δ</em> cells by immunoblotting using antibodies specific to these targets. (F, G). Sam35pHA (F) or Sen2pHA (G) expressed from either the genome or a <em>CEN</em> plasmid were analyzed in WT and <em>psh1Δ</em> cells at 30° as in (A). (H–J) <em>CEN</em> plasmid-expressed <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA under control of the <em>ADH1</em> promoter (H) or <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA under control of the <em>GAL10</em> promoter (I) or with the native <em><a href="https://www.yeastgenome.org/locus/S000001678">MIA40</a></em> 3′ untranslated region (J) were assessed in <em>psh1Δ</em> and WT cells at 30° (<a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA) or 37° (<a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA), as in (A).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">We next determined whether loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> affected the steady-state levels of other mitochondrial proteins. We expressed WT <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA, another ts- mitochondrial protein <a href="https://www.yeastgenome.org/locus/S000003261">erv1</a>-2pHA (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Muller <em>et al.</em> 2008</a>), and a mitochondrial matrix-targeted GFP (mtGFP) (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Westermann and Neupert 2000</a>) from <em>CEN</em> plasmids in WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. The levels of all of these proteins are increased when <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is deleted (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>). We also expressed other UPS substrates, including the cytosolic degron-containing protein <a href="https://www.yeastgenome.org/locus/S000000747">Ura3</a>pHA-CL1 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Metzger <em>et al.</em> 2008</a>), and the ER-localized misfolded proteins <a href="http://www.yeastgenome.org/locus/S000001692/overview">Ste6p</a>*HA and <a href="http://www.yeastgenome.org/locus/S000004912/overview">CPY</a>*HA (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hiller <em>et al.</em> 1996</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Loayza <em>et al.</em> 1998</a>), from <em>CEN</em> plasmids in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. All show a similar increase in protein abundance in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a>). When assessing <a href="http://www.yeastgenome.org/locus/S000004912/overview">CPY</a>* levels using <a href="http://www.yeastgenome.org/locus/S000004912/overview">CPY</a> antibody, we noticed that the protein level of endogenous chromosomal <a href="http://www.yeastgenome.org/locus/S000004912/overview">CPY</a> is unchanged in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>, <a href="http://www.yeastgenome.org/locus/S000004912/overview">CPY</a>). A more detailed analysis of proteins expressed from the genome (<a href="http://www.yeastgenome.org/locus/S000004912/overview">CPY</a>, PGK, and Porin) confirms that chromosomal targets are not significantly increased in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells relative to WT (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3E</a>).</p><p id="p-32">To more directly assess the effect that loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> was having on <em>CEN</em>-plasmid <em>vs.</em> chromosomal targets, we analyzed the steady-state levels of two mitochondrial proteins (<a href="https://www.yeastgenome.org/locus/S000001125">Sam35</a>pHA and <a href="https://www.yeastgenome.org/locus/S000004095">Sen2</a>pHA) when expressed from either a <em>CEN</em> plasmid or the genome in WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. Loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> increases the levels of Sam35pHA and Sen2pHA relative to WT cells only when they are expressed from a <em>CEN</em> plasmid (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, F and G</a>). Thus, the increases in protein abundance seen in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells are not related to protein location or type, but rather correlate with being expressed from CEN plasmids.</p><p id="p-33">To determine whether the <a href="https://www.yeastgenome.org/locus/S000005415">Psh1p</a>-dependent increase in steady-state plasmid protein levels was dependent on the specific promoters or terminators, we replaced the promoters and terminators in our <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA</em> and <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmids used in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2</a>, and <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>. <em>CEN</em> plasmid-encoded <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA was placed under control of the <em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em> promoter (rather than the <em><a href="http://www.yeastgenome.org/locus/S000000383/overview">FZO1</a></em> promoter) and <em>CEN</em> plasmid-encoded <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA were placed under control of the <em><a href="http://www.yeastgenome.org/locus/S000000223/overview">GAL10</a></em> promoter (rather than the <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">MIA40</a></em> promoter). We also replaced the terminator in our <em><a href="https://www.yeastgenome.org/locus/S000001678">MIA40</a>HA</em> and <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA CEN</em> plasmids with the <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">MIA40</a></em> 3′ untranslated region, rather than <em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em> terminator used in prior experiments. Regardless of the promoter or terminator, steady-state protein levels of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA, <a href="https://www.yeastgenome.org/locus/S000001678">Mia40</a>pHA, and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA all increased in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells relative to WT (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3, H–J</a>). Together, these data suggest that <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> increases the steady-state levels of proteins when they are expressed from <em>CEN</em> plasmids and not when they are expressed from the genome. This effect is a consequence of loss of the ubiquitin ligase activity of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>, and is independent of the promoter or terminator used.</p></div><div id="sec-15" class="subsection"><h3>Loss of Psh1p increases the mRNA of plasmid-expressed genes due to higher plasmid DNA levels resulting from plasmid missegregation</h3><p id="p-34">We next assessed whether the increase in <em>CEN</em> plasmid-derived protein in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells results from an increase in mRNA levels of the genes in question. Quantitative reverse transcription qPCR (RT-qPCR) with primers specific to genomic-expressed <em><a href="http://www.yeastgenome.org/locus/S000000383/overview">FZO1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">MIA40</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005780/overview">SEC63</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000314/overview">ALG1</a></em> shows no difference in the mRNA levels of WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strains when cells are grown in YPD medium and not expressing plasmid (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>; normalized to <em><a href="http://www.yeastgenome.org/locus/S000001855/overview">ACT1</a></em>). Similarly, there is no change in the relative mRNA levels of chromosomal <em><a href="http://www.yeastgenome.org/locus/S000000314/overview">ALG1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005780/overview">SEC63</a></em> in WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells expressing the <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>; genomic) or <em>CEN</em> <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA (data not shown) plasmid and growing in medium lacking leucine. In contrast, the mRNA levels of HA-tagged <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> and <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA</em> expressed from these <em>CEN</em> plasmids are significantly increased in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells relative to WT (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>; normalized to <em><a href="http://www.yeastgenome.org/locus/S000001855/overview">ACT1</a></em>). We also see similar increases in the mRNA levels of <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>, the selective marker gene on these plasmids required for their maintenance (and deleted from the chromosome of the yeast strain used here).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of Psh1p increases mRNA levels of genes expressed from CEN plasmids due to higher plasmid DNA content achieved through plasmid missegregation. (A) Normalized mRNA expression of the endogenous, genome-expressed genes FZO1, MIA40, SEC63, or ALG1 in WT and psh1&#x394; cells grown in YPD medium, not expressing a plasmid, was determined using RT-qPCR analysis. Expression was normalized to ACT1 levels and is graphed relative to WT cells; error bars represent the SD of at least three biological replicates with three technical replicates each. (B) Normalized mRNA expression of FZO1HA, mia40-4HA, and LEU2 in WT and psh1&#x394; cells carrying the indicated CEN plasmid, and normalized mRNA expression of genomic SEC63 and ALG1 in WT and psh1&#x394; cells expressing the CEN FZO1HA plasmid were determined as in (A). (C) Normalized DNA levels of CEN plasmid FZO1HA, mia40-4HA, and LEU2 in DNA harvested from WT and psh1&#x394; cells carrying the indicated CEN plasmid as determined by qPCR on DNA samples. Plasmid DNA content was normalized to the genomic level of ACT1 and graphed as in (A). (D) DNA levels of genome-expressed ACT1 and SEC63 in the DNA used in (C) from cells expressing the FZO1HA CEN plasmid as determined by qPCR. Values are graphed relative to WT cells; error bars represent the SD of at least three biological replicates with three technical replicates each. (E&#x2013;G) The fraction of FZO1HA CEN (E) mia40-4HA CEN (F) or FZO1HA 2&#xA0;&#x3BC;m plasmid-bearing WT and psh1&#x394; cells after 0 and 48&#xA0;hr without selection for the plasmid was assessed by comparing the number of cells able to grow on selective media to the cell number on YPD. Error bars represent the SD of at least three biological replicates. (H) Normalized DNA levels of 2&#xA0;&#x3BC;m and CEN plasmid FZO1HA and LEU2 in WT and psh1&#x394; cells, determined as in (C). (I) DNA levels of genome-expressed ACT1 and SEC63 in the DNA used in (H) as determined by qPCR, graphed and analyzed as in (D). For (A&#x2013;I) P-values are calculated from a two-tailed t-test and *&#xA0;P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-196213172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Loss of Psh1p increases mRNA levels of genes expressed from &lt;em&gt;CEN&lt;/em&gt; plasmids due to higher plasmid DNA content achieved through plasmid missegregation. (A) Normalized mRNA expression of the endogenous, genome-expressed genes &lt;em&gt;FZO1&lt;/em&gt;, &lt;em&gt;MIA40&lt;/em&gt;, &lt;em&gt;SEC63&lt;/em&gt;, or &lt;em&gt;ALG1&lt;/em&gt; in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells grown in YPD medium, not expressing a plasmid, was determined using RT-qPCR analysis. Expression was normalized to &lt;em&gt;ACT1&lt;/em&gt; levels and is graphed relative to WT cells; error bars represent the SD of at least three biological replicates with three technical replicates each. (B) Normalized mRNA expression of &lt;em&gt;FZO1HA&lt;/em&gt;, &lt;em&gt;mia40-4HA&lt;/em&gt;, and &lt;em&gt;LEU2&lt;/em&gt; in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells carrying the indicated &lt;em&gt;CEN&lt;/em&gt; plasmid, and normalized mRNA expression of genomic &lt;em&gt;SEC63&lt;/em&gt; and &lt;em&gt;ALG1&lt;/em&gt; in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells expressing the &lt;em&gt;CEN FZO1HA&lt;/em&gt; plasmid were determined as in (A). (C) Normalized DNA levels of &lt;em&gt;CEN&lt;/em&gt; plasmid &lt;em&gt;FZO1HA&lt;/em&gt;, &lt;em&gt;mia40-4HA&lt;/em&gt;, and &lt;em&gt;LEU2&lt;/em&gt; in DNA harvested from WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells carrying the indicated &lt;em&gt;CEN&lt;/em&gt; plasmid as determined by qPCR on DNA samples. Plasmid DNA content was normalized to the genomic level of &lt;em&gt;ACT1&lt;/em&gt; and graphed as in (A). (D) DNA levels of genome-expressed &lt;em&gt;ACT1&lt;/em&gt; and &lt;em&gt;SEC63&lt;/em&gt; in the DNA used in (C) from cells expressing the &lt;em&gt;FZO1HA CEN&lt;/em&gt; plasmid as determined by qPCR. Values are graphed relative to WT cells; error bars represent the SD of at least three biological replicates with three technical replicates each. (E&#x2013;G) The fraction of &lt;em&gt;FZO1HA CEN&lt;/em&gt; (E) &lt;em&gt;mia40-4HA CEN&lt;/em&gt; (F) or &lt;em&gt;FZO1HA&lt;/em&gt; 2&#xA0;&#x3BC;m plasmid-bearing WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells after 0 and 48&#xA0;hr without selection for the plasmid was assessed by comparing the number of cells able to grow on selective media to the cell number on YPD. Error bars represent the SD of at least three biological replicates. (H) Normalized DNA levels of 2&#xA0;&#x3BC;m and &lt;em&gt;CEN&lt;/em&gt; plasmid &lt;em&gt;FZO1HA&lt;/em&gt; and &lt;em&gt;LEU2&lt;/em&gt; in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; cells, determined as in (C). (I) DNA levels of genome-expressed &lt;em&gt;ACT1&lt;/em&gt; and &lt;em&gt;SEC63&lt;/em&gt; in the DNA used in (H) as determined by qPCR, graphed and analyzed as in (D). For (A&#x2013;I) &lt;em&gt;P&lt;/em&gt;-values are calculated from a two-tailed &lt;em&gt;t&lt;/em&gt;-test and *&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.05, **&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.01, ***&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.001, ****&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.0001, and ns&#xA0;=&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&gt;&#xA0;0.05.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3731/F4.medium.gif" width="293" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/11/3731/F4.medium.gif" width="293" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487218" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Loss of Psh1p increases mRNA levels of genes expressed from <em>CEN</em> plasmids due to higher plasmid DNA content achieved through plasmid missegregation. (A) Normalized mRNA expression of the endogenous, genome-expressed genes <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000001678">MIA40</a></em>, <em>SEC63</em>, or <em>ALG1</em> in WT and <em>psh1Δ</em> cells grown in YPD medium, not expressing a plasmid, was determined using RT-qPCR analysis. Expression was normalized to <em>ACT1</em> levels and is graphed relative to WT cells; error bars represent the SD of at least three biological replicates with three technical replicates each. (B) Normalized mRNA expression of <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em>, <em><a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4HA</em>, and <em>LEU2</em> in WT and <em>psh1Δ</em> cells carrying the indicated <em>CEN</em> plasmid, and normalized mRNA expression of genomic <em>SEC63</em> and <em>ALG1</em> in WT and <em>psh1Δ</em> cells expressing the <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid were determined as in (A). (C) Normalized DNA levels of <em>CEN</em> plasmid <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em>, <em><a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4HA</em>, and <em>LEU2</em> in DNA harvested from WT and <em>psh1Δ</em> cells carrying the indicated <em>CEN</em> plasmid as determined by qPCR on DNA samples. Plasmid DNA content was normalized to the genomic level of <em>ACT1</em> and graphed as in (A). (D) DNA levels of genome-expressed <em>ACT1</em> and <em>SEC63</em> in the DNA used in (C) from cells expressing the <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA CEN</em> plasmid as determined by qPCR. Values are graphed relative to WT cells; error bars represent the SD of at least three biological replicates with three technical replicates each. (E–G) The fraction of <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA CEN</em> (E) <em><a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4HA CEN</em> (F) or <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> 2 μm plasmid-bearing WT and <em>psh1Δ</em> cells after 0 and 48 hr without selection for the plasmid was assessed by comparing the number of cells able to grow on selective media to the cell number on YPD. Error bars represent the SD of at least three biological replicates. (H) Normalized DNA levels of 2 μm and <em>CEN</em> plasmid <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> and <em>LEU2</em> in WT and <em>psh1Δ</em> cells, determined as in (C). (I) DNA levels of genome-expressed <em>ACT1</em> and <em>SEC63</em> in the DNA used in (H) as determined by qPCR, graphed and analyzed as in (D). For (A–I) <em>P</em>-values are calculated from a two-tailed <em>t</em>-test and * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01, *** <em>P</em> &lt; 0.001, **** <em>P</em> &lt; 0.0001, and ns = <em>P</em> &gt; 0.05.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Since protein levels are increased by loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> even when different promoter and terminators are used (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3, H–J</a>), we hypothesized that the observed increases in mRNA levels did not result from changes in transcription or mRNA stability, but instead from increased plasmid DNA content in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strains. Analysis by qPCR reveals that the DNA levels of <em>CEN</em> plasmid <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em>, <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> are indeed increased in a <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strain relative to WT (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>; normalized to <em><a href="http://www.yeastgenome.org/locus/S000001855/overview">ACT1</a></em> levels). The DNA levels of genomic targets (<em><a href="http://www.yeastgenome.org/locus/S000001855/overview">ACT1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005780/overview">SEC63</a></em>) in these same <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells expressing <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>) or <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA (data not shown) are unaffected. Taken together, these data indicate that loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> increases <em>CEN</em> plasmid DNA content, thus leading to an increase in the mRNA and proteins encoded by these plasmids.</p><p id="p-37">The increase in <em>CEN</em> plasmid DNA levels in the population of cells seen when <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is absent could be resulting from an increased incidence of unequal plasmid segregation (<em>i.e.</em>, nondisjunction) during mitosis. This could initiate with 2:0 missegregation events following normal <em>CEN</em> plasmid replication, such that there is a net increase in plasmid DNA in one cell with a concurrent complete loss in the other. Previous experiments where we measure increased <em>CEN</em> plasmid DNA, mRNA, and protein levels in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells were performed under conditions where cells were grown in selective media lacking leucine to maintain the <em>CEN</em> plasmids. Any cells that lose <em>CEN</em> plasmid through missegregation would not be maintained in the population because they now lack the selective marker gene on the plasmids, and are inviable in selective media. Cells with one or more copies of the <em>CEN</em> plasmid would continue to replicate the plasmid and divide normally, thus leading to an increase in the overall plasmid DNA content within the population of cells over time.</p><p id="p-38">To examine whether the increased plasmid DNA levels measured in a population of <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells under selective conditions could be resulting from unequal plasmid segregation, we measured plasmid loss in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. The plasmid loss assay measures the fraction of cells maintaining plasmid after growth without selection, as assessed by the presence of plasmid-encoded <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> marker (see also <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> and legend). WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells expressing <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid were first grown in selective media, where there is no significant difference in the percentage of WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells that contain at least one copy of the <em>CEN</em> plasmid (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4E</a>, 0 hr). Cells were then grown in nonselective media for 48 hr, after which they were again plated on selective media; ∼65% of WT cells retain the <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid after 48 hr without selection, compared to &lt;40% of <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4E</a>, 48 hr). Using the same assay, we see a similar enhanced loss of <em>CEN <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA</em> plasmid (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4F</a>) or <em>CEN</em> empty vector plasmid (Figure S2A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>, compare WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, 48 hr) in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. This reduction in plasmid-bearing cells, in conjunction with the increase in overall plasmid DNA content when <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is absent, suggests that there is indeed an increased incidence of unequal <em>CEN</em> plasmid segregation.</p><p id="p-39">We wished to extend these findings to noncentromeric plasmids, so we next examined 2 μm plasmids. High-copy number, 2 μm plasmids appear to use a chromosome-coupled segregation mechanism, although the exact mechanism mediating this “hitchhiking” is unclear, and the fidelity of plasmid segregation is less than for <em>CEN</em> plasmids (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sau <em>et al.</em> 2015</a>). Interestingly, after 48 hr without selection, only 6% of <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells retain a 2 μm plasmid encoding <em><a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>HA</em> with a <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> selectable marker (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4G</a>), while 16% of WT cells do so. Expression of a 2 μm <em><a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4-HA</em> plasmid negatively affects cell viability (data not shown), precluding its inclusion in this analysis, but a 2 μm empty vector plasmid is lost similarly to the 2 μm <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid (Figure S2B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>, compare WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, 48 hr). Even with increased 2 μm plasmid loss, and in contrast to the <em>CEN</em> plasmids, we do not detect a change in the amount of 2 μm plasmid DNA in a <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strain compared to WT under selective media conditions (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4H</a>). The DNA levels of chromosomal genes are again also unaffected by <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> in cells expressing the <em>CEN</em> or 2 μm <em><a href="http://www.yeastgenome.org/locus/S000000383/overview">FZO1</a></em> plasmids (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4I</a>). In agreement with unchanged 2 μm plasmid DNA content, the levels of 2 μm plasmid-derived proteins are also not increased by loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> (Figure S2, C and D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>). The lack of a change in 2 μm DNA or protein levels is perhaps not surprising, as 2 μm plasmids are known to buffer changes in their levels by amplification using a recombination-induced rolling circle replication mechanism (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chan <em>et al.</em> 2013</a>). Regardless, the fidelity of segregation of both <em>CEN</em> and 2 μm plasmids appears to be affected by loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>.</p></div><div id="sec-16" class="subsection"><h3>Overexpression of the Psh1p degradation target Cse4p does not phenocopy loss of Psh1p in increasing CEN plasmid-derived protein levels</h3><p id="p-40">The only known target of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is the centromeric histone H3 variant <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (CENP-A). Several elegant studies have demonstrated that <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> ubiquitinates <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> that is not properly localized to the centromere, targeting this mislocalized pool of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> for degradation (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hewawasam <em>et al.</em> 2010</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Ranjitkar <em>et al.</em> 2010</a>). Ultimately, this ensures that <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> is exclusively centromere-restricted, which is required for proper chromosome segregation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hajra <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Au <em>et al.</em> 2008</a>). Loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> decreases the rate of turnover of endogenous <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> at all stages of the cell cycle (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Ranjitkar <em>et al.</em> 2010</a>). We confirm that our <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strain stabilizes <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> and, in darker exposures, we see higher molecular weight laddering that is suggestive of ubiquitination of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> in WT cells, but is absent in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overexpression of Cse4p does not phenocopy loss of Psh1p in increasing CEN plasmid protein levels. (A) CHX chase for the indicated times assessing turnover of myc-Cse4p in WT and psh1&#x394; yeast cells. Proteins were detecting by immunoblotting with myc antibody. Both long and short exposures are shown. Anti-PGK serves as a loading control. (B) The steady-state protein levels of CEN plasmid-expressed mia40-4pHA and Fzo1pHA, as well as endogenous genome-expressed Cse4p, were detected by immunoblotting with HA or Cse4p antibodies, in the indicated deletion strains. (C) Immunoblotting for Cse4p using Cse4p antibodies in WT cells, psh1&#x394; cells, or cells overexpressing Cse4p from the genome (pGAL-FLAG-CSE4) for the indicated time points after continuous galactose induction. (D) Immunoblotting using anti-HA for CEN plasmid-expressed mia40-4pHA or Fzo1pHA as in (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-196213172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overexpression of Cse4p does not phenocopy loss of Psh1p in increasing &lt;em&gt;CEN&lt;/em&gt; plasmid protein levels. (A) CHX chase for the indicated times assessing turnover of myc-Cse4p in WT and &lt;em&gt;psh1&#x394;&lt;/em&gt; yeast cells. Proteins were detecting by immunoblotting with myc antibody. Both long and short exposures are shown. Anti-PGK serves as a loading control. (B) The steady-state protein levels of &lt;em&gt;CEN&lt;/em&gt; plasmid-expressed mia40-4pHA and Fzo1pHA, as well as endogenous genome-expressed Cse4p, were detected by immunoblotting with HA or Cse4p antibodies, in the indicated deletion strains. (C) Immunoblotting for Cse4p using Cse4p antibodies in WT cells, &lt;em&gt;psh1&#x394;&lt;/em&gt; cells, or cells overexpressing Cse4p from the genome (&lt;em&gt;pGAL-FLAG-CSE4&lt;/em&gt;) for the indicated time points after continuous galactose induction. (D) Immunoblotting using anti-HA for &lt;em&gt;CEN&lt;/em&gt; plasmid-expressed mia40-4pHA or Fzo1pHA as in (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3731/F5.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/11/3731/F5.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487242" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Overexpression of Cse4p does not phenocopy loss of Psh1p in increasing <em>CEN</em> plasmid protein levels. (A) CHX chase for the indicated times assessing turnover of myc-Cse4p in WT and <em>psh1Δ</em> yeast cells. Proteins were detecting by immunoblotting with myc antibody. Both long and short exposures are shown. Anti-PGK serves as a loading control. (B) The steady-state protein levels of <em>CEN</em> plasmid-expressed <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA, as well as endogenous genome-expressed Cse4p, were detected by immunoblotting with HA or Cse4p antibodies, in the indicated deletion strains. (C) Immunoblotting for Cse4p using Cse4p antibodies in WT cells, <em>psh1Δ</em> cells, or cells overexpressing Cse4p from the genome (<em>pGAL-FLAG-CSE4</em>) for the indicated time points after continuous galactose induction. (D) Immunoblotting using anti-HA for <em>CEN</em> plasmid-expressed <a href="https://www.yeastgenome.org/locus/S000001678">mia40</a>-4pHA or <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA as in (C).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Since <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> misregulation leads to chromosome transmission defects (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Morey <em>et al.</em> 2004</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Au <em>et al.</em> 2008</a>), we wondered whether the increased <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> levels in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells could also be affecting <em>CEN</em> plasmid segregation and ultimately leading to increased <em>CEN</em> plasmid-derived protein levels. As <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> is essential for viability (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Stoler <em>et al.</em> 1995</a>), it cannot be deleted from <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells, preventing us from directly asking whether it is necessary for the plasmid missegregation phenotypes. However, other proteins have been implicated in increasing the levels of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> either through modulation of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> activity toward <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> or by affecting <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> levels independently of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>. We assessed <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA in deletions of these genes. These include mutants in sumoylation machinery (<a href="http://www.yeastgenome.org/locus/S000002817/overview">Siz1p</a>, <a href="http://www.yeastgenome.org/locus/S000005682/overview">Siz2p</a>, and <a href="http://www.yeastgenome.org/locus/S000002171/overview">Slx5p</a>), proline isomerases (<a href="http://www.yeastgenome.org/locus/S000004539/overview">Fpr3p</a> and <a href="http://www.yeastgenome.org/locus/S000004441/overview">Fpr4p</a>), a factor required for ubiquitin homeostasis (<a href="http://www.yeastgenome.org/locus/S000001696/overview">Doa1p</a>), and casein kinase 2 (<a href="http://www.yeastgenome.org/locus/S000005587/overview">Cka2p</a>) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Au <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hewawasam <em>et al.</em> 2014</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ohkuni <em>et al.</em> 2014</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">2016</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Canzonetta <em>et al.</em> 2015</a>). Deletion of an F-box protein, <a href="http://www.yeastgenome.org/locus/S000003740/overview">Rcy1p</a>, also affects <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> levels (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cheng <em>et al.</em> 2016</a>); this deletion was not assessed due to a severe growth defect. For most of these deletions, there is a significant increase in endogenous <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> relative to WT cells (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>; <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>). For deletions where <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> is unaltered, differences in strain background or growth conditions, or that we assessed endogenous <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>, may all be potential explanations. Importantly, the steady-state protein levels of <em>CEN</em> plasmid-based <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA do not significantly change in any of the deletion strains that manifest increased <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). This is in contrast to what is observed with <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>).</p><p id="p-43">To assess the effects of elevated <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> more directly, we used a well-characterized <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression system in WT cells to ask whether this could phenocopy <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. This strain contains a chromosomal, galactose-inducible FLAG-tagged <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (<em>pGAL-FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">CSE4</a></em>) (<a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Ranjitkar <em>et al.</em> 2010</a>). The pattern of occupancy of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> at the centromere is perturbed similarly by either <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> or by FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression using this strain (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hildebrand and Biggins 2016</a>). <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> is increased above WT levels by either deletion of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> or overexpression of FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> for 24 or 48 hr (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). Despite the substantial increase in <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> levels with FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression, the protein levels of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA and <a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>pHA are largely unchanged (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>). This is in contrast to the significant increase that we observe with <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>. Thus, increasing <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> to varying levels, using either previously characterized deletion strains or induced overexpression, does not phenocopy <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> in increasing <em>CEN</em> plasmid-derived protein levels.</p></div><div id="sec-17" class="subsection"><h3>Overexpression of Cse4p does not affect plasmid DNA levels or loss in the same manner as loss of Psh1p</h3><p id="p-44">To confirm that elevated <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> is alone insufficient to mediate the effects we see in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, we reassessed <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid DNA levels by qPCR after FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression. Some decrease in the amplification efficiency of genomic targets is observed (Figure S3A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>), perhaps related to reported <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>-mediated genomic instability (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Morey <em>et al.</em> 2004</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Au <em>et al.</em> 2008</a>). <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid DNA levels, in marked contrast to <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, are unchanged by FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>; <em><a href="https://www.yeastgenome.org/locus/S000000383">Fzo1</a>HA</em> and Vector/<em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> junction-normalized to <em><a href="http://www.yeastgenome.org/locus/S000001855/overview">ACT1</a></em>). The lack of an increase in plasmid DNA is consistent with the relative protein levels in <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5D</a>. These results indicate that, unlike loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>, the direct overexpression of its substrate <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> does not result in czhanges consistent with unequal plasmid missegregation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overexpression of Cse4p increases CEN, but not 2&#xA0;&#x3BC;m, plasmid loss and does not affect plasmid DNA content under selective growth conditions. (A) Normalized DNA levels of CEN FZO1HA plasmid were detected by qPCR using two plasmid-specific primer pairs (FZO1HA and vector/LEU2 junction) on DNA harvested from WT cells, psh1&#x394; cells, or cells overexpressing Cse4p from the chromosome (pGAL-FLAG-CSE4) for 24 hr in selective media. LEU2 primers could not be used, as the mutated leu2-3,112 allele is present in the genome of these strains. DNA content was normalized to ACT1 levels and graphed as in Figure 4C. See Figure S3A in File S1 legend for further detail. (B) The fraction of CEN FZO1HA plasmid-bearing was analyzed as in Figure 4D after 0 and 48&#xA0;hr growth in media without selection for the CEN plasmid. Cells were grown in medium containing galactose for 24&#xA0;hr prior to the shift to nonselective medium, and galactose induction was continued during growth without selection. Error bars represent the SD of at least three biological replicates. (C) The fraction of 2&#xA0;&#x3BC;m FZO1HA plasmid-bearing WT cells, psh1&#x394; cells, or cells overexpressing Cse4p from the chromosome (pGAL-FLAG-CSE4) was analyzed as in (B) For (A&#x2013;C) P-values are calculated from a two-tailed t-test; *&#xA0;P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-196213172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overexpression of Cse4p increases &lt;em&gt;CEN&lt;/em&gt;, but not 2&#xA0;&#x3BC;m, plasmid loss and does not affect plasmid DNA content under selective growth conditions. (A) Normalized DNA levels of &lt;em&gt;CEN FZO1HA&lt;/em&gt; plasmid were detected by qPCR using two plasmid-specific primer pairs (&lt;em&gt;FZO1HA&lt;/em&gt; and vector/&lt;em&gt;LEU2&lt;/em&gt; junction) on DNA harvested from WT cells, &lt;em&gt;psh1&#x394;&lt;/em&gt; cells, or cells overexpressing Cse4p from the chromosome (&lt;em&gt;pGAL-FLAG-CSE4&lt;/em&gt;) for 24 hr in selective media. &lt;em&gt;LEU2&lt;/em&gt; primers could not be used, as the mutated &lt;em&gt;leu2-3&lt;/em&gt;,&lt;em&gt;112&lt;/em&gt; allele is present in the genome of these strains. DNA content was normalized to &lt;em&gt;ACT1&lt;/em&gt; levels and graphed as in Figure 4C. See Figure S3A in File S1 legend for further detail. (B) The fraction of &lt;em&gt;CEN FZO1HA&lt;/em&gt; plasmid-bearing was analyzed as in Figure 4D after 0 and 48&#xA0;hr growth in media without selection for the &lt;em&gt;CEN&lt;/em&gt; plasmid. Cells were grown in medium containing galactose for 24&#xA0;hr prior to the shift to nonselective medium, and galactose induction was continued during growth without selection. Error bars represent the SD of at least three biological replicates. (C) The fraction of 2&#xA0;&#x3BC;m &lt;em&gt;FZO1HA&lt;/em&gt; plasmid-bearing WT cells, &lt;em&gt;psh1&#x394;&lt;/em&gt; cells, or cells overexpressing Cse4p from the chromosome (&lt;em&gt;pGAL-FLAG-CSE4&lt;/em&gt;) was analyzed as in (B) For (A&#x2013;C) &lt;em&gt;P&lt;/em&gt;-values are calculated from a two-tailed &lt;em&gt;t&lt;/em&gt;-test; *&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.05, **&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.01, ***&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.001, ****&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.0001, and ns&#xA0;=&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&gt;&#xA0;0.05.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3731/F6.medium.gif" width="215" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/11/3731/F6.medium.gif" width="215" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487163" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">Overexpression of Cse4p increases <em>CEN</em>, but not 2 μm, plasmid loss and does not affect plasmid DNA content under selective growth conditions. (A) Normalized DNA levels of <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid were detected by qPCR using two plasmid-specific primer pairs (<em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> and vector/<em>LEU2</em> junction) on DNA harvested from WT cells, <em>psh1Δ</em> cells, or cells overexpressing Cse4p from the chromosome (<em>pGAL-FLAG-CSE4</em>) for 24 hr in selective media. <em>LEU2</em> primers could not be used, as the mutated <em>leu2-3</em>,<em>112</em> allele is present in the genome of these strains. DNA content was normalized to <em>ACT1</em> levels and graphed as in <a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4C</a>. See Figure S3A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a> legend for further detail. (B) The fraction of <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid-bearing was analyzed as in <a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4D</a> after 0 and 48 hr growth in media without selection for the <em>CEN</em> plasmid. Cells were grown in medium containing galactose for 24 hr prior to the shift to nonselective medium, and galactose induction was continued during growth without selection. Error bars represent the SD of at least three biological replicates. (C) The fraction of 2 μm <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid-bearing WT cells, <em>psh1Δ</em> cells, or cells overexpressing Cse4p from the chromosome (<em>pGAL-FLAG-CSE4</em>) was analyzed as in (B) For (A–C) <em>P</em>-values are calculated from a two-tailed <em>t</em>-test; * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01, *** <em>P</em> &lt; 0.001, **** <em>P</em> &lt; 0.0001, and ns = <em>P</em> &gt; 0.05.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">We next repeated the plasmid loss assay in cells where FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> was overexpressed. As expected, loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> results in increased <em>CEN <a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid loss under nonselective growth conditions (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>, <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>). Interestingly, cells overexpressing FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> lose this <em>CEN</em> plasmid to an even greater extent than <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>, <em>pGAL-FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">CSE4</a></em>). Similar results are seen for a <em>CEN</em> plasmid without insert (Figure S2A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>). The levels of <em>CEN</em> plasmid DNA and protein from <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>-overexpressing cells are not elevated when they are grown under selection (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5D</a> and <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6A</a>), which is distinct from the increases seen in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells under the same conditions. This suggests that the plasmid loss resulting from elevation of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> alone is mechanistically distinct from that observed with loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>. In addition to plasmid loss by missegregation of replicated plasmid (<em>e.g.</em>, 2:0 plasmid missegregation), loss can also occur via a 1:0 mechanism. In this case, plasmid does not properly replicate or is destroyed before cell division; such a mechanism may be at play here for <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>-overexpressing cells.</p><p id="p-47">In addition to binding to centromeric DNA, <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> has been reported to bind at low levels to the <em>STB</em> DNA locus of 2 μm plasmids (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hajra <em>et al.</em> 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Huang <em>et al.</em> 2011</a>), perhaps playing a role in the hitchhiking segregation of this type of plasmid. Mutation of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> also affects <em>STB</em>-containing reporter plasmid segregation (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Hajra <em>et al.</em> 2006</a>). Since <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> leads to increased 2 μm plasmid loss, we wondered whether this was mediated by <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>’s effects on <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> levels or, like <em>CEN</em> plasmids, is different. Thus, we examined 2 μm <em><a href="https://www.yeastgenome.org/locus/S000000383">FZO1</a>HA</em> plasmid loss under nonselective galactose-induction conditions in WT, <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, and FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpressing cells. We again see 2 μm plasmid loss when <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is absent under these growth conditions. In contrast to the <em>CEN</em> plasmid, however, overexpression of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> alone using <em>pGAL-FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">CSE4</a></em> does not lead to increased 2 μm plasmid loss (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>), representing a striking dichotomy with <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>. Similar results are seen with a 2 μm vector lacking insert (Figure S2B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1/FileS1.docx">File S1</a>). These data argue against a direct role for <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> in 2 μm plasmid loss. Taken together, the distinct patterns of <em>CEN</em> and 2 μm plasmid missegregation and <em>CEN</em> plasmid DNA levels seen in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells strongly suggest a non-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>-mediated effect of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> on plasmid DNA maintenance.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-48">In this study, we demonstrate that loss of the ubiquitin ligase <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> increases the <em>CEN</em> plasmid DNA content in cells grown under selective pressure. This in turn elevates plasmid-encoded mRNA and protein. The net increase in plasmid DNA is coupled with a higher incidence of plasmid loss from cells that have undergone multiple rounds of division without selective pressure. This suggests that the <em>CEN</em> plasmid is missegregating more frequently in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> than in WT cells, where a 1:1 equal plasmid segregation generally occurs (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7, A and B</a>, shown is the simplest mechanism—ongoing 2:0 unequal plasmid segregation—but, as plasmid copy number increases in some cells, other permutations of missegregation are also possible, if not likely). Thus, the <em>CEN</em> plasmid is likely replicating properly but there is increased missegregation during mitosis. Ongoing missegregation would lead to some population of cells containing more than one copy of the plasmid, which increases the overall steady-state plasmid DNA content of the population. A similar plasmid missegregation could be responsible for the recently reported plasmid loss in a diploid <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> strain, which accompanies loss of heterozygosity (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Herrero and Thorpe 2016</a>). Interestingly, while overexpression of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>—the only known target of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>—leads to even greater plasmid loss than <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em>, their mechanisms are distinct. As there is no significant increase in <em>CEN</em> plasmid DNA levels when <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> is overexpressed under selective conditions, this suggests an increase in an apparent 1:0 mechanism of segregation wherein plasmid fails to replicate prior to mitosis or one copy of the replicated plasmid is lost from the nucleus or destroyed (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a>). The ARS and CEN DNA sequences are in close proximity on the plasmid, and <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression is known to lead to centromere spreading to proximal regions (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Hildebrand and Biggins 2016</a>), which could feasibly interfere with plasmid replication. Alternatively, the genomic instability observed with <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Morey <em>et al.</em> 2004</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Au <em>et al.</em> 2008</a>) could be increasing the rate of formation of dicentric plasmids, which break during DNA segregation and lead to plasmid instability and loss (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mann and Davis 1983</a>). This 1:0 mechanism of plasmid loss has been observed to occur at greater than twice the frequency of 2:0 events (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Koshland <em>et al.</em> 1985</a>), making the increased missegregation seen with loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> that much more significant.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Models for the distinct mechanisms of plasmid missegregation in haploid psh1&#x394; and Cse4p-overexpressing cells. (A) In WT cells, CEN/ARS-containing yeast plasmids generally accurately replicate and segregate in a 1:1 manner during cell division, and thus have a low frequency of plasmid loss. After generations of growth in media selective for cells containing the plasmid, cells typically maintain one copy of the plasmid and are viable. Thus, the overall plasmid DNA content in the population remains close to 1N in haploid yeast cells. (B) In haploid psh1&#x394; cells, CEN/ARS-containing plasmids are replicated, but have an apparent increased propensity for missegregation. One cell may receive two copies of the plasmid, while the other receives none. Under selective growth conditions, cells without plasmid are inviable. Yeast with one or more copies of plasmid would be viable and continue to divide, leading to an overall increase in the DNA content (&gt;1N) in the population after generations of growth with selection for the plasmid. Shown is a model for the simplest 2:0 missegregation of plasmid, but other mechanisms of missegregation (i.e., 4:0, 3:1, etc.) also become possible after initiating missegregation events have occurred. An increase in cells lacking plasmid is detected by the plasmid loss assay, which measures colony forming units on selective vs. nonselective media after generations of growth without selection for the plasmid. (C) In cells overexpressing Cse4p, there appears to be an increase in the incidence of 1:0 segregation of CEN/ARS-containing plasmids. This could occur because the plasmid fails to replicate properly, or if one copy of the plasmid is lost from the nucleus or degraded following replication. Regardless, the net result is that only one progeny cell receives a copy of plasmid. Again, the cell without plasmid would be inviable under selective growth conditions and this loss would be measured by the plasmid loss assay. The cell with one copy of the plasmid would remain viable and continue to divide, with the DNA content in the population remaining close to 1N after generations of growth in medium selective for the plasmid. For (A&#x2013;C) the nucleus is indicated by a dotted line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-196213172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Models for the distinct mechanisms of plasmid missegregation in haploid &lt;em&gt;psh1&#x394;&lt;/em&gt; and Cse4p-overexpressing cells. (A) In WT cells, &lt;em&gt;CEN&lt;/em&gt;/ARS-containing yeast plasmids generally accurately replicate and segregate in a 1:1 manner during cell division, and thus have a low frequency of plasmid loss. After generations of growth in media selective for cells containing the plasmid, cells typically maintain one copy of the plasmid and are viable. Thus, the overall plasmid DNA content in the population remains close to 1N in haploid yeast cells. (B) In haploid &lt;em&gt;psh1&#x394;&lt;/em&gt; cells, &lt;em&gt;CEN&lt;/em&gt;/ARS-containing plasmids are replicated, but have an apparent increased propensity for missegregation. One cell may receive two copies of the plasmid, while the other receives none. Under selective growth conditions, cells without plasmid are inviable. Yeast with one or more copies of plasmid would be viable and continue to divide, leading to an overall increase in the DNA content (&gt;1N) in the population after generations of growth with selection for the plasmid. Shown is a model for the simplest 2:0 missegregation of plasmid, but other mechanisms of missegregation (&lt;em&gt;i.e.&lt;/em&gt;, 4:0, 3:1, etc.) also become possible after initiating missegregation events have occurred. An increase in cells lacking plasmid is detected by the plasmid loss assay, which measures colony forming units on selective &lt;em&gt;vs.&lt;/em&gt; nonselective media after generations of growth without selection for the plasmid. (C) In cells overexpressing Cse4p, there appears to be an increase in the incidence of 1:0 segregation of &lt;em&gt;CEN&lt;/em&gt;/ARS-containing plasmids. This could occur because the plasmid fails to replicate properly, or if one copy of the plasmid is lost from the nucleus or degraded following replication. Regardless, the net result is that only one progeny cell receives a copy of plasmid. Again, the cell without plasmid would be inviable under selective growth conditions and this loss would be measured by the plasmid loss assay. The cell with one copy of the plasmid would remain viable and continue to divide, with the DNA content in the population remaining close to 1N after generations of growth in medium selective for the plasmid. For (A&#x2013;C) the nucleus is indicated by a dotted line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3731/F7.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/11/3731/F7.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3731/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487260" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-49" class="first-child">Models for the distinct mechanisms of plasmid missegregation in haploid <em>psh1Δ</em> and Cse4p-overexpressing cells. (A) In WT cells, <em>CEN</em>/ARS-containing yeast plasmids generally accurately replicate and segregate in a 1:1 manner during cell division, and thus have a low frequency of plasmid loss. After generations of growth in media selective for cells containing the plasmid, cells typically maintain one copy of the plasmid and are viable. Thus, the overall plasmid DNA content in the population remains close to 1N in haploid yeast cells. (B) In haploid <em>psh1Δ</em> cells, <em>CEN</em>/ARS-containing plasmids are replicated, but have an apparent increased propensity for missegregation. One cell may receive two copies of the plasmid, while the other receives none. Under selective growth conditions, cells without plasmid are inviable. Yeast with one or more copies of plasmid would be viable and continue to divide, leading to an overall increase in the DNA content (&gt;1N) in the population after generations of growth with selection for the plasmid. Shown is a model for the simplest 2:0 missegregation of plasmid, but other mechanisms of missegregation (<em>i.e.</em>, 4:0, 3:1, etc.) also become possible after initiating missegregation events have occurred. An increase in cells lacking plasmid is detected by the plasmid loss assay, which measures colony forming units on selective <em>vs.</em> nonselective media after generations of growth without selection for the plasmid. (C) In cells overexpressing Cse4p, there appears to be an increase in the incidence of 1:0 segregation of <em>CEN</em>/ARS-containing plasmids. This could occur because the plasmid fails to replicate properly, or if one copy of the plasmid is lost from the nucleus or degraded following replication. Regardless, the net result is that only one progeny cell receives a copy of plasmid. Again, the cell without plasmid would be inviable under selective growth conditions and this loss would be measured by the plasmid loss assay. The cell with one copy of the plasmid would remain viable and continue to divide, with the DNA content in the population remaining close to 1N after generations of growth in medium selective for the plasmid. For (A–C) the nucleus is indicated by a dotted line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">The differences in plasmid DNA, mRNA, and protein levels between <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> and <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression strongly suggest that the dysregulation of a distinct target or function of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is required to mediate these <em>CEN</em> plasmid missegregation events. The established <em>pGAL-FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">CSE4</a></em> overexpression system elevates <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> levels more than <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> does, making it difficult to completely exclude <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> dosage effects. However, direct overexpression of FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> or elevated levels of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> from loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> result in similar increases in the centromeric distribution of this histone H3 variant (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Hildebrand and Biggins 2016</a>), suggesting that, at least with respect to centromere loading of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>, the two are equivalent. We cannot rule out other differences between the two conditions, but, for several reasons, we favor a model where the loss of ubiquitination of a novel target of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is a requisite component of the missegregation mechanism. Varying the length of time of galactose induction of FLAG-<a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>, and thus increasing the number of rounds of division the cells have undergone, has no impact on <em>CEN</em> plasmid protein levels. Similarly, modulating <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> through gene deletions is without effect. Further, if <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> was a dedicated ubiquitin ligase for only <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>, one might reasonably expect that <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> would only be located where <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> is found, with the exception of the <em>CEN</em> localization of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>, where it is protected from Psh1p-mediated ubiquitination and degradation. Rather, there is evidence that <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> localizes to kinetochores independently of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>. Fractionation of micrococcal nuclease-treated chromatin reveals that <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> distributes almost equally between the soluble fraction and the insoluble pellet, while <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> remains entirely in the insoluble pellet, where the larger kinetochore protein complex fractionates (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Hewawasam <em>et al.</em> 2010</a>). Also, deletions that result in <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> being mislocalized to locations other than centromeres do not change the kinetochore association of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a>.</p><p id="p-51">Whether there is a <em>bona fide</em> target of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> at kinetochores, and what role <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> may play in its regulation, is an area for future investigation. Ubiquitin ligases rarely have a single substrate. However, one of the challenges in the field is the comprehensive identification of their substrates due to their transient and low-affinity substrate interactions. It will be important to identify <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> substrates other than <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> with the goal of understanding more completely the means by which this ligase affects not only plasmid, but potentially chromosome, inheritance. <em>CEN</em> plasmids are lost at least 100 times more frequently than a normal chromosome (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hieter <em>et al.</em> 1985</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Koshland <em>et al.</em> 1985</a>), making it unsurprising that we see no gross effect on genomic targets in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells. Also, chromosomal loss events would likely be deleterious to haploid cells, causing genomic instability and, in most cases, loss of viability. We and others see no measurable difference in the rate of growth or cell cycle progression between WT and <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells (data not shown) (<a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Ranjitkar <em>et al.</em> 2010</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Herrero and Thorpe 2016</a>), which would indicate relatively subtle effects on chromosome missegregation. Loss of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> has been shown to modulate kinetochore protein stoichiometry (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Herrero and Thorpe 2016</a>) with a resultant increase in genomic instability phenotypes (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yuen <em>et al.</em> 2007</a>). However, while increased loss of a particular genomic locus was seen in <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> cells in three independent genetic screens (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Yuen <em>et al.</em> 2007</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Herrero and Thorpe 2016</a>), loss of an entire artificial chromosome fragment was not observed (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Yuen <em>et al.</em> 2007</a>). In contrast, elevation of <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> or its orthologs lead to genomic instability and chromosome missegregation and loss (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Morey <em>et al.</em> 2004</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Heun <em>et al.</em> 2006</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Au <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amato <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gonzalez <em>et al.</em> 2014</a>), suggesting that, as is the case herein with plasmids, at the level of the genome, there are functional distinctions between deleting this ubiquitin ligase and its known target, <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a>.</p><p id="p-52">With respect to 2 μm plasmid loss, we see a clear difference between <em><a href="http://www.yeastgenome.org/locus/S000005415/overview">psh1</a>Δ</em> and <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression. <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> overexpression does not increase 2 μm plasmid loss, while deletion of <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> does, again arguing that the two are having distinct cellular effects. The precise mechanism of segregation of 2 μm plasmids is not clear, but one model is that they “hitchhike” along with chromosomes during mitosis (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Yen Ting <em>et al.</em> 2014</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Sau <em>et al.</em> 2015</a>). Thus, it is possible that 2 μm missegregation occurs because <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> is indeed having effects on chromosome segregation. Another possibility is that <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> plays a direct role in 2 μm plasmid segregation independent of its role in <em>CEN</em> plasmid or chromosome segregation. Interestingly, a study describing Ubiquitin-Activated Interaction Traps (UBAITs) technology, which can identify proteins in proximity to specific ubiquitin ligases, used <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> as a bait and identified a component of the 2 μm plasmid partitioning system (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">O’Connor <em>et al.</em> 2015</a>). This raises the intriguing possibility that <a href="http://www.yeastgenome.org/locus/S000005415/overview">Psh1p</a> may be in close proximity to 2 μm plasmids. Sumoylation of 2 μm plasmid-associated proteins has been previously demonstrated to be required for plasmid maintenance (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pinder <em>et al.</em> 2013</a>), so it is conceivable that other post-translational modifications like ubiquitin may also play a role.</p><p id="p-53">The original goal of this study was to identify and characterize UPS components acting on mitochondrial QC substrates. This is still an open and active area of investigation. While the <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA ts- protein is unstable at the nonpermissive temperature, we do not find it to be stabilized in the <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-2</em> mutant strain by pulse-chase under these conditions. Interestingly, a previous study showed that steady-state levels of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4p are increased by treatment with the proteasome inhibitor, MG132, but not by a proteasome mutant strain (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bragoszewski <em>et al.</em> 2013</a>). The turnover of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA is also not affected by mutations in autophagy or mitochondrial protease genes, including <a href="http://www.yeastgenome.org/locus/S000000118/overview">Pim1p</a> (Lon) and the mAAA- and iAAA-proteases. The mechanism of <a href="http://www.yeastgenome.org/locus/S000001678/overview">mia40</a>-4pHA degradation remains unclear and warrants continued investigation.</p><p id="p-54">Finally, this study raises a cautionary note in interpreting increased levels of plasmid-expressed mRNA and protein. Plasmids are a powerful tool to assess transcription and post-transcriptional regulation. Our work underscores that perturbation of cellular factors can lead to unexpected and unobvious alterations in plasmid levels and the genes they encode.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We would like to thank Sue Biggins, Nikolaus Pfanner, Trevor Lithgow, Mickael Cohen, Susan Michaelis, and Benedikt Westermann for their generous gifts of strains, antibodies, and plasmids; Jadranka Loncarek for assistance with microscopy; Ira Daar for helpful suggestions; and Sue Biggins and Yien Che Tsai for critical reading and comments on this manuscript. This study was funded by the Intramural Research Program at the National Cancer Institute, National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300227/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-57"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-02"><span class="received-label">Received </span>August 2, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-09-09"><span class="accepted-label">Accepted </span>September 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Metzger <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.1" data-doi="10.1242/jcs.183954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akutsu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dikic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bremm</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Ubiquitin chain diversity at a glance.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.183954%26rft_id%253Dinfo%253Apmid%252F26906419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6OToiMTI5LzUvODc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.2" data-doi="10.1186/1476-4598-8-119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schillaci</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lentini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Leonardo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CENPA overexpression promotes genome instability in pRb-depleted human cells.</span> <abbr class="cit-jnl-abbrev">Mol. Cancer</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcancer%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DMol%2BCancer%26rft.aulast%253DAmato%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.spage%253D119%26rft.epage%253D119%26rft.atitle%253DCENPA%2Boverexpression%2Bpromotes%2Bgenome%2Binstability%2Bin%2BpRb-depleted%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1476-4598-8-119%26rft_id%253Dinfo%253Apmid%252F20003272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1476-4598-8-119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003272&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262497100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.3" data-doi="10.1534/genetics.108.088518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisp</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeLuca</span>  <span class="cit-name-given-names">S. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rando</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basrai</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Altered dosage and mislocalization of histone H3 and Cse4p lead to chromosome loss in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088518%26rft_id%253Dinfo%253Apmid%252F18458100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvMjYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.4" data-doi="10.1534/genetics.113.149898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rawson</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A novel role of the N terminus of budding yeast histone H3 variant Cse4 in ubiquitin-mediated proteolysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.149898%26rft_id%253Dinfo%253Apmid%252F23525333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNTEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.5" data-doi="10.1038/nature09371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bengtson</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joazeiro</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Role of a ribosome-associated E3 ubiquitin ligase in protein quality control.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBengtson%26rft.auinit1%253DM.%2BH.%26rft.volume%253D467%26rft.issue%253D7314%26rft.spage%253D470%26rft.epage%253D473%26rft.atitle%253DRole%2Bof%2Ba%2Bribosome-associated%2BE3%2Bubiquitin%2Bligase%2Bin%2Bprotein%2Bquality%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09371%26rft_id%253Dinfo%253Apmid%252F20835226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20835226&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282090200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.6" data-doi="10.1128/MCB.01579-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bragoszewski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gornicka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sztolsztener</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chacinska</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The ubiquitin-proteasome system regulates mitochondrial intermembrane space proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2136</span>–<span class="cit-lpage">2148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01579-12%26rft_id%253Dinfo%253Apmid%252F23508107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMy8xMS8yMTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.7" data-doi="10.1038/nsmb.3147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandman</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Ribosome-associated protein quality control.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D23%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3147%26rft_id%253Dinfo%253Apmid%252F26733220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26733220&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.8" data-doi="10.1016/j.tcb.2017.02.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westermann</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">With the help of MOM: mitochondrial contributions to cellular quality control.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BCell%2BBiol.%26rft.volume%253D27%26rft.spage%253D441%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2017.02.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2017.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Canzonetta</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vernarecci</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iuliani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marracino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belloni</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">SAGA DUB-Ubp8 deubiquitylates centromeric histone variant Cse4.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.10" data-doi="10.1038/sj.emboj.7600389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chacinska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfannschmidt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiedemann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozjak</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanjuan Szklarz</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Essential role of Mia40 in import and assembly of mitochondrial intermembrane space proteins.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">3735</span>–<span class="cit-lpage">3746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DChacinska%26rft.auinit1%253DA.%26rft.volume%253D23%26rft.issue%253D19%26rft.spage%253D3735%26rft.epage%253D3746%26rft.atitle%253DEssential%2Brole%2Bof%2BMia40%2Bin%2Bimport%2Band%2Bassembly%2Bof%2Bmitochondrial%2Bintermembrane%2Bspace%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600389%26rft_id%253Dinfo%253Apmid%252F15359280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMzczNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.11" data-doi="10.1016/j.plasmid.2013.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jayaram</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sau</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The 2 micron plasmid of <em>Saccharomyces cerevisiae</em>: a miniaturized selfish genome with optimized functional competence.</span> <abbr class="cit-jnl-abbrev">Plasmid</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlasmid%26rft.volume%253D70%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plasmid.2013.03.001%26rft_id%253Dinfo%253Apmid%252F23541845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plasmid.2013.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23541845&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321319300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.12" data-doi="10.1091/mbc.E07-08-0796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lithgow</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The peripheral membrane subunits of the SAM complex function codependently in mitochondrial outer membrane biogenesis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-08-0796%26rft_id%253Dinfo%253Apmid%252F17978093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTI2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.13" data-doi="10.1126/science.2538923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chau</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachmair</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marriott</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A multiubiquitin chain is confined to specific lysine in a targeted short-lived protein.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">243</span>: <span class="cit-fpage">1576</span>–<span class="cit-lpage">1583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChau%26rft.auinit1%253DV%26rft.volume%253D243%26rft.issue%253D4898%26rft.spage%253D1576%26rft.epage%253D1583%26rft.atitle%253DA%2Bmultiubiquitin%2Bchain%2Bis%2Bconfined%2Bto%2Bspecific%2Blysine%2Bin%2Ba%2Btargeted%2Bshort-lived%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2538923%26rft_id%253Dinfo%253Apmid%252F2538923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNDMvNDg5OC8xNTc2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.14" data-doi="10.1074/jbc.M115.701813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The F-box protein Rcy1 is involved in the degradation of histone H3 variant Cse4 and genome maintenance.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">10372</span>–<span class="cit-lpage">10377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M115.701813%26rft_id%253Dinfo%253Apmid%252F26975376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTEvMTkvMTAzNzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.15" data-doi="10.1016/0378-1119(92)90454-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christianson</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shero</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Multifunctional yeast high-copy-number shuttle vectors.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DChristianson%26rft.auinit1%253DT.%2BW.%26rft.volume%253D110%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D122%26rft.atitle%253DMultifunctional%2Byeast%2Bhigh-copy-number%2Bshuttle%2Bvectors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252892%252990454-W%26rft_id%253Dinfo%253Apmid%252F1544568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(92)90454-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1544568&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HE64700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cipolla</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maffia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertoletti</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabbioneda</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The regulation of DNA damage tolerance by ubiquitin and ubiquitin-like modifiers.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.17" data-doi="10.1038/287504a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Isolation of a yeast centromere and construction of functional small circular chromosomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">504</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DClarke%26rft.auinit1%253DL.%26rft.volume%253D287%26rft.issue%253D5782%26rft.spage%253D504%26rft.epage%253D509%26rft.atitle%253DIsolation%2Bof%2Ba%2Byeast%2Bcentromere%2Band%2Bconstruction%2Bof%2Bfunctional%2Bsmall%2Bcircular%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F287504a0%26rft_id%253Dinfo%253Apmid%252F6999364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/287504a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6999364&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980KL19500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.18" data-doi="10.1091/mbc.E08-02-0227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leboucher</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Livnat-Levanon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glickman</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Ubiquitin-proteasome-dependent degradation of a mitofusin, a critical regulator of mitochondrial fusion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2457</span>–<span class="cit-lpage">2464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-02-0227%26rft_id%253Dinfo%253Apmid%252F18353967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMjQ1NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Defenouillere</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fromont-Racine</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The ribosome-bound quality control complex: from aberrant peptide clearance to proteostasis maintenance.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenet.%26rft.volume%253D28%26rft.spage%253D1165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.20" data-doi="10.1038/nature04530"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doma</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Endonucleolytic cleavage of eukaryotic mRNAs with stalls in translation elongation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDoma%26rft.auinit1%253DM.%2BK.%26rft.volume%253D440%26rft.issue%253D7083%26rft.spage%253D561%26rft.epage%253D564%26rft.atitle%253DEndonucleolytic%2Bcleavage%2Bof%2Beukaryotic%2BmRNAs%2Bwith%2Bstalls%2Bin%2Btranslation%2Belongation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04530%26rft_id%253Dinfo%253Apmid%252F16554824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16554824&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236176100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.21" data-doi="10.1083/jcb.200512079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Escobar-Henriques</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westermann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langer</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Regulation of mitochondrial fusion by the F-box protein Mdm30 involves proteasome-independent turnover of Fzo1.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200512079%26rft_id%253Dinfo%253Apmid%252F16735578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE3My81LzY0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.22" data-doi="10.1074/jbc.275.12.8945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludwig</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vousden</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mdm2 is a RING finger-dependent ubiquitin protein ligase for itself and p53.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">8945</span>–<span class="cit-lpage">8951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DFang%26rft.auinit1%253DS.%26rft.volume%253D275%26rft.issue%253D12%26rft.spage%253D8945%26rft.epage%253D8951%26rft.atitle%253DMdm2%2BIs%2Ba%2BRING%2BFinger-dependent%2BUbiquitin%2BProtein%2BLigase%2Bfor%2BItself%2Band%2Bp53%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.275.12.8945%26rft_id%253Dinfo%253Apmid%252F10722742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzUvMTIvODk0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.23" data-doi="10.1128/MCB.14.8.5501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monia</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Inhibition of proteolysis and cell cycle progression in a multiubiquitination-deficient yeast mutant.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5501</span>–<span class="cit-lpage">5509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFinley%26rft.auinit1%253DD%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D5501%26rft.epage%253D5509%26rft.atitle%253DInhibition%2Bof%2Bproteolysis%2Band%2Bcell%2Bcycle%2Bprogression%2Bin%2Ba%2Bmultiubiquitination-deficient%2Byeast%2Bmutant.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.8.5501%26rft_id%253Dinfo%253Apmid%252F8035826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE0LzgvNTUwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.24" data-doi="10.3389/fgene.2016.00087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Rodriguez</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulrich</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Functions of ubiquitin and SUMO in DNA replication and replication stress.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.25" data-doi="10.1534/genetics.114.171173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ectopic centromere nucleation by CENP–a in fission yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1433</span>–<span class="cit-lpage">1446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171173%26rft_id%253Dinfo%253Apmid%252F25298518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE0MzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kyryakov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Titorenko</span>  <span class="cit-name-given-names">V. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Purification of mitochondria from yeast cells.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">pii</span>: <span class="cit-fpage">1417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft.volume%253Dpii%26rft.spage%253D1417%26rft.atitle%253DPURIFICATION%2BOF%2BMITOCHONDRIA%2BFROM%2BYEAST%2BCELLS.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.27" data-doi="10.1083/jcb.200603042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hajra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jayaram</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The centromere-specific histone variant Cse4p (CENP-A) is essential for functional chromatin architecture at the yeast 2-microm circle partitioning locus and promotes equal plasmid segregation.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200603042%26rft_id%253Dinfo%253Apmid%252F16966420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE3NC82Lzc3OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hegemann</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shero</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cottarel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Mutational analysis of centromere DNA from chromosome VI of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2523</span>–<span class="cit-lpage">2535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHegemann%26rft.auinit1%253DJ%2BH%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D2523%26rft.epage%253D2535%26rft.atitle%253DMutational%2Banalysis%2Bof%2Bcentromere%2BDNA%2Bfrom%2Bchromosome%2BVI%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F3043181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjgvNi8yNTIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.29" data-doi="10.1042/bst0190724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heinemeyer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simeon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiffer</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teichert</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Lysosomal and non-lysosomal proteolysis in the eukaryotic cell: studies on yeast.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbst0190724%26rft_id%253Dinfo%253Apmid%252F1783206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo3OiJwcGJpb3N0IjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNzI0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.30" data-doi="10.1371/journal.pgen.1005855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herrero</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorpe</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Synergistic control of kinetochore protein levels by Psh1 and Ubr2.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005855%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005855%26rft_id%253Dinfo%253Apmid%252F26891228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26891228&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.31" data-doi="10.1016/j.devcel.2006.01.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heun</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erhardt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blower</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skora</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mislocalization of the <em>Drosophila</em> centromere-specific histone CID promotes formation of functional ectopic kinetochores.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DHeun%26rft.auinit1%253DP.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D303%26rft.epage%253D315%26rft.atitle%253DMislocalization%2Bof%2Bthe%2BDrosophila%2Bcentromere-specific%2Bhistone%2BCID%2Bpromotes%2Bformation%2Bof%2Bfunctional%2Bectopic%2Bkinetochores.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2006.01.014%26rft_id%253Dinfo%253Apmid%252F16516834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2006.01.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16516834&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236255500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.32" data-doi="10.1016/j.molcel.2010.10.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hewawasam</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shivaraju</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattingly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin-Brown</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Psh1 is an E3 ubiquitin ligase that targets the centromeric histone variant Cse4.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHewawasam%26rft.auinit1%253DG.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D444%26rft.epage%253D454%26rft.atitle%253DPsh1%2Bis%2Ban%2BE3%2Bubiquitin%2Bligase%2Bthat%2Btargets%2Bthe%2Bcentromeric%2Bhistone%2Bvariant%2BCse4.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.10.014%26rft_id%253Dinfo%253Apmid%252F21070970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.10.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21070970&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284556300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.33" data-doi="10.1074/jbc.M114.580589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hewawasam</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattingly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phosphorylation by casein kinase 2 facilitates Psh1 protein-assisted degradation of Cse4 protein.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">29297</span>–<span class="cit-lpage">29309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.580589%26rft_id%253Dinfo%253Apmid%252F25183013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNDIvMjkyOTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.34" data-doi="10.1016/0092-8674(85)90152-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Mitotic stability of yeast chromosomes: a colony color assay that measures nondisjunction and chromosome loss.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHieter%26rft.auinit1%253DP.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D381%26rft.epage%253D392%26rft.atitle%253DMitotic%2Bstability%2Bof%2Byeast%2Bchromosomes%253A%2Ba%2Bcolony%2Bcolor%2Bassay%2Bthat%2Bmeasures%2Bnondisjunction%2Band%2Bchromosome%2Bloss.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252885%252990152-7%26rft_id%253Dinfo%253Apmid%252F3967296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(85)90152-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3967296&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AFE4800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hildebrand</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biggins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Regulation of budding yeast CENP-A levels prevents misincorporation at promoter nucleosomes and transcriptional defects.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.36" data-doi="10.1126/science.273.5282.1725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hiller</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweiger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">ER degradation of a misfolded luminal protein by the cytosolic ubiquitin-proteasome pathway.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">1725</span>–<span class="cit-lpage">1728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHiller%26rft.auinit1%253DM.%2BM.%26rft.volume%253D273%26rft.issue%253D5282%26rft.spage%253D1725%26rft.epage%253D1728%26rft.atitle%253DER%2BDegradation%2Bof%2Ba%2BMisfolded%2BLuminal%2BProtein%2Bby%2Bthe%2BCytosolic%2BUbiquitin-Proteasome%2BPathway%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.273.5282.1725%26rft_id%253Dinfo%253Apmid%252F8781238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzMvNTI4Mi8xNzI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.37" data-doi="10.1038/sj.onc.1203464"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Activity of MDM2, a ubiquitin ligase, toward p53 or itself is dependent on the RING finger domain of the ligase.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1473</span>–<span class="cit-lpage">1476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DHonda%26rft.auinit1%253DR.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D1473%26rft.epage%253D1476%26rft.atitle%253DActivity%2Bof%2BMDM2%252C%2Ba%2Bubiquitin%2Bligase%252C%2Btoward%2Bp53%2Bor%2Bitself%2Bis%2Bdependent%2Bon%2Bthe%2BRING%2Bfinger%2Bdomain%2Bof%2Bthe%2Bligase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1203464%26rft_id%253Dinfo%253Apmid%252F10723139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1203464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10723139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085942700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.38" data-doi="10.1128/MCB.01191-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hajra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jayaram</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cse4 (CenH3) association with the <em>Saccharomyces cerevisiae</em> plasmid partitioning locus in its native and chromosomally integrated states: implications in centromere evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1030</span>–<span class="cit-lpage">1040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01191-10%26rft_id%253Dinfo%253Apmid%252F21173161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjMxLzUvMTAzMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.39" data-doi="10.1016/j.ceb.2016.03.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Protein quality control in the nucleus.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BCell%2BBiol.%26rft.volume%253D40%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2016.03.002%26rft_id%253Dinfo%253Apmid%252F27015023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2016.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27015023&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.40" data-doi="10.1016/j.molcel.2016.11.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juszkiewicz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Initiation of quality control during poly(A) translation requires site-specific ribosome ubiquitination.</span> <abbr class="cit-jnl-abbrev">Mol Cell</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">750.e4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%26rft.volume%253D65%26rft.spage%253D743%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2016.11.039%26rft_id%253Dinfo%253Apmid%252F28065601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2016.11.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28065601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.41" data-doi="10.1016/0092-8674(85)90153-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartwell</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Genetic analysis of the mitotic transmission of minichromosomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKoshland%26rft.auinit1%253DD.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D393%26rft.epage%253D403%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2Bmitotic%2Btransmission%2Bof%2Bminichromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252885%252990153-9%26rft_id%253Dinfo%253Apmid%252F3881185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(85)90153-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3881185&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AFE4800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.42" data-doi="10.1091/mbc.9.10.2767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loayza</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Ste6p mutants defective in exit from the endoplasmic reticulum (ER) reveal aspects of an ER quality control pathway in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2767</span>–<span class="cit-lpage">2784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DLoayza%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D2767%26rft.epage%253D2784%26rft.atitle%253DSte6p%2BMutants%2BDefective%2Bin%2BExit%2Bfrom%2Bthe%2BEndoplasmic%2BReticulum%2B%2528ER%2529%2BReveal%2BAspects%2Bof%2Ban%2BER%2BQuality%2BControl%2BPathway%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.9.10.2767%26rft_id%253Dinfo%253Apmid%252F9763443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjkvMTAvMjc2NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.43" data-doi="10.1073/pnas.80.1.228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Instability of dicentric plasmids in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMann%26rft.auinit1%253DC.%26rft.volume%253D80%26rft.issue%253D1%26rft.spage%253D228%26rft.epage%253D232%26rft.atitle%253DInstability%2Bof%2Bdicentric%2Bplasmids%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.1.228%26rft_id%253Dinfo%253Apmid%252F6337372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4MC8xLzIyOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.44" data-doi="10.1074/jbc.M806424200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dancy</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Degradation of a cytosolic protein requires endoplasmic reticulum-associated degradation machinery.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">32302</span>–<span class="cit-lpage">32316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M806424200%26rft_id%253Dinfo%253Apmid%252F18812321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvNDcvMzIzMDIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.45" data-doi="10.1016/j.molcel.2013.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariano</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A structurally unique E2-binding domain activates ubiquitination by the ERAD E2, Ubc7p, through multiple mechanisms.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D516%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.04.004%26rft_id%253Dinfo%253Apmid%252F23665230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23665230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319893600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pruneda</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klevit</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RING-type E3 ligases: master manipulators of E2 ubiquitin-conjugating enzymes and ubiquitination.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1843</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1843%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000330014200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.47" data-doi="10.1128/EC.3.6.1533-1543.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morey</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzgerald-Hayes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The histone fold domain of Cse4 is sufficient for CEN targeting and propagation of active centromeres in budding yeast.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DMorey%26rft.auinit1%253DL.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D1533%26rft.epage%253D1543%26rft.atitle%253DThe%2BHistone%2BFold%2BDomain%2Bof%2BCse4%2BIs%2BSufficient%2Bfor%2BCEN%2BTargeting%2Band%2BPropagation%2Bof%2BActive%2BCentromeres%2Bin%2BBudding%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.3.6.1533-1543.2004%26rft_id%253Dinfo%253Apmid%252F15590827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIzLzYvMTUzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.48" data-doi="10.1091/mbc.E07-08-0814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milenkovic</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guiard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfanner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chacinska</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Precursor oxidation by Mia40 and Erv1 promotes vectorial transport of proteins into the mitochondrial intermembrane space.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-08-0814%26rft_id%253Dinfo%253Apmid%252F17978092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMjI2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Construction and behavior of circularly permuted and telocentric chromosomes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3166</span>–<span class="cit-lpage">3172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMurray%26rft.auinit1%253DA%2BW%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D3166%26rft.epage%253D3172%26rft.atitle%253DConstruction%2Band%2Bbehavior%2Bof%2Bcircularly%2Bpermuted%2Band%2Btelocentric%2Bchromosomes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F3537731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8zMTY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.50" data-doi="10.1038/nprot.2006.236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hands</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustin</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Quantification of mRNA using real-time RT-PCR.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1559</span>–<span class="cit-lpage">1582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DNolan%26rft.auinit1%253DT.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D1559%26rft.epage%253D1582%26rft.atitle%253DQuantification%2Bof%2BmRNA%2Busing%2Breal-time%2BRT-PCR.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2006.236%26rft_id%253Dinfo%253Apmid%252F17406449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2006.236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17406449&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251155400062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.51" data-doi="10.15252/embr.201540620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swaim</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ubiquitin-activated interaction traps (UBAITs) identify E3 ligase binding partners.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1699</span>–<span class="cit-lpage">1712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembr.201540620%26rft_id%253Dinfo%253Apmid%252F26508657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6MTA6IjE2LzEyLzE2OTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.52" data-doi="10.1534/genetics.114.161224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohkuni</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdulle</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagawa</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Degradation of centromeric histone H3 variant Cse4 requires the Fpr3 peptidyl-prolyl Cis-Trans isomerase.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1041</span>–<span class="cit-lpage">1045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161224%26rft_id%253Dinfo%253Apmid%252F24514906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEwNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.53" data-doi="10.1091/mbc.E15-12-0827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohkuni</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrimore</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SUMO-targeted ubiquitin ligase (STUbL) Slx5 regulates proteolysis of centromeric histone H3 variant Cse4 and prevents its mislocalization to euchromatin.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1500</span>–<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E15-12-0827%26rft_id%253Dinfo%253Apmid%252F26960795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI3LzkvMTUwMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.54" data-doi="10.1371/journal.pone.0060384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinder</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McQuaid</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobson</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deficient sumoylation of yeast 2-micron plasmid proteins Rep1 and Rep2 associated with their loss from the plasmid-partitioning locus and impaired plasmid inheritance.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e60384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De60384%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0060384%26rft_id%253Dinfo%253Apmid%252F23555963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0060384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555963&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.55" data-doi="10.1016/S1534-5807(03)00322-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinsky</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatsutani</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biggins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">An Mtw1 complex promotes kinetochore biorientation that is monitored by the Ipl1/Aurora protein kinase.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DPinsky%26rft.auinit1%253DB.%2BA.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D735%26rft.epage%253D745%26rft.atitle%253DAn%2BMtw1%2Bcomplex%2Bpromotes%2Bkinetochore%2Bbiorientation%2Bthat%2Bis%2Bmonitored%2Bby%2Bthe%2BIpl1%252FAurora%2Bprotein%2Bkinase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1534-5807%252803%252900322-8%26rft_id%253Dinfo%253Apmid%252F14602074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1534-5807(03)00322-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14602074&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186544400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.56" data-doi="10.1042/BCJ20160582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The evolving role of ubiquitin modification in endoplasmic reticulum-associated degradation.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">474</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBCJ20160582%26rft_id%253Dinfo%253Apmid%252F28159894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDc0LzQvNDQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.57" data-doi="10.1016/j.molcel.2010.09.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranjitkar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Press</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacCoss</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An E3 ubiquitin ligase prevents ectopic localization of the centromeric histone H3 variant via the centromere targeting domain.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRanjitkar%26rft.auinit1%253DP.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D455%26rft.epage%253D464%26rft.atitle%253DAn%2BE3%2Bubiquitin%2Bligase%2Bprevents%2Bectopic%2Blocalization%2Bof%2Bthe%2Bcentromeric%2Bhistone%2BH3%2Bvariant%2Bvia%2Bthe%2Bcentromere%2Btargeting%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.09.025%26rft_id%253Dinfo%253Apmid%252F21070971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.09.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21070971&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284556300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.58" data-doi="10.1038/ncb1558"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ravid</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Autoregulation of an E2 enzyme by ubiquitin-chain assembly on its catalytic residue.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DRavid%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D422%26rft.epage%253D427%26rft.atitle%253DAutoregulation%2Bof%2Ban%2BE2%2Benzyme%2Bby%2Bubiquitin-chain%2Bassembly%2Bon%2Bits%2Bcatalytic%2Bresidue.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1558%26rft_id%253Dinfo%253Apmid%252F17310239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1558&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17310239&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245359400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.59" data-doi="10.1016/j.mrrev.2016.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaudin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koch Lerner</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menck</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosselli</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The ubiquitin family meets the <em>Fanconi anemia</em> proteins.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Rev. Mutat. Res.</abbr> <span class="cit-vol">769</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BRev.%2BMutat.%2BRes.%26rft.volume%253D769%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mrrev.2016.06.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mrrev.2016.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jayaram</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Stable persistence of the yeast plasmid by hitchhiking on chromosomes during vegetative and germ-line divisions of host cells.</span> <abbr class="cit-jnl-abbrev">Mob. Genet. Elements</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BGenet.%2BElements%26rft.volume%253D5%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.61" data-doi="10.1007/978-1-62703-487-6_17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortsch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bockler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klecker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westermann</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analyzing membrane dynamics with live cell fluorescence microscopy with a focus on yeast mitochondria.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1033</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1033%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-487-6_17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-487-6_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSikorski%26rft.auinit1%253DR.%2BS.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DA%2Bsystem%2Bof%2Bshuttle%2Bvectors%2Band%2Byeast%2Bhost%2Bstrains%2Bdesigned%2Bfor%2Befficient%2Bmanipulation%2Bof%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIyLzEvMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.63" data-doi="10.1101/gad.9.5.573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curnick</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzgerald-Hayes</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A mutation in CSE4, an essential gene encoding a novel chromatin-associated protein in yeast, causes chromosome nondisjunction and cell cycle arrest at mitosis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DStoler%26rft.auinit1%253DS%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D573%26rft.epage%253D586%26rft.atitle%253DA%2Bmutation%2Bin%2BCSE4%252C%2Ban%2Bessential%2Bgene%2Bencoding%2Ba%2Bnovel%2Bchromatin-associated%2Bprotein%2Bin%2Byeast%252C%2Bcauses%2Bchromosome%2Bnondisjunction%2Band%2Bcell%2Bcycle%2Barrest%2Bat%2Bmitosis.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.5.573%26rft_id%253Dinfo%253Apmid%252F7698647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiOS81LzU3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzczMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sundaramoorthy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulzele</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">ZNF598 and RACK1 regulate mammalian ribosome-associated quality control function by mediating regulatory 40S ribosomal ubiquitylation.</span> <abbr class="cit-jnl-abbrev">Mol Cell</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">751</span>–<span class="cit-lpage">760.e4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%26rft.volume%253D65%26rft.spage%253D751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.65" data-doi="10.1093/emboj/19.1.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thrower</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rechsteiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickart</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Recognition of the polyubiquitin proteolytic signal.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DThrower%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D94%26rft.epage%253D102%26rft.atitle%253DRecognition%2Bof%2Bthe%2Bpolyubiquitin%2Bproteolytic%2Bsignal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F19.1.94%26rft_id%253Dinfo%253Apmid%252F10619848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo3OiIxOS8xLzk0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.66" data-doi="10.1002/1097-0061(200011)16:15&lt;1421::AID-YEA624&gt;3.0.CO;2-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westermann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupert</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mitochondria-targeted green fluorescent proteins: convenient tools for the study of organelle biogenesis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1421</span>–<span class="cit-lpage">1427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWestermann%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D15%26rft.spage%253D1421%26rft.epage%253D1427%26rft.atitle%253DMitochondria-targeted%2Bgreen%2Bfluorescent%2Bproteins%253A%2Bconvenient%2Btools%2Bfor%2Bthe%2Bstudy%2Bof%2Borganelle%2Bbiogenesis%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0061%2528200011%252916%253A15%253C1421%253A%253AAID-YEA624%253E3.0.CO%253B2-U%26rft_id%253Dinfo%253Apmid%252F11054823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-0061(200011)16:15&lt;1421::AID-YEA624&gt;3.0.CO;2-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11054823&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165780900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.67" data-doi="10.1016/j.semcdb.2012.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ndoja</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Regulation of gene expression by the ubiquitin-proteasome system.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DYao%26rft.auinit1%253DT.%26rft.volume%253D23%26rft.issue%253D5%26rft.spage%253D523%26rft.epage%253D529%26rft.atitle%253DRegulation%2Bof%2Bgene%2Bexpression%2Bby%2Bthe%2Bubiquitin-proteasome%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2012.02.006%26rft_id%253Dinfo%253Apmid%252F22430757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2012.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22430757&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3731.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yen Ting</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kachroo</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowley</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The partitioning and copy number control systems of the selfish yeast plasmid: an optimized molecular design for stable persistence in host cells.</span> <abbr class="cit-jnl-abbrev">Microbiol. Spectr.</abbr> <span class="cit-vol">2</span>. Available at: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4274994/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4274994/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.69" data-doi="10.1073/pnas.0610642104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwok</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Systematic genome instability screens in yeast and their potential relevance to cancer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3925</span>–<span class="cit-lpage">3930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0610642104%26rft_id%253Dinfo%253Apmid%252F17360454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEwLzM5MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3731.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabek</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Ubiquitin ligases: structure, function, and regulation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiochem.%26rft.volume%253D86%26rft.spage%253D129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3731.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3719" title="Mechanisms of Transmission Ratio Distortion at Hybrid Sterility Loci Within and Between Mimulus Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3745" title="Serotonin Drives Predatory Feeding Behavior via Synchronous Feeding Rhythms in the Nematode Pristionchus pacificus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3731" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1bveUAEcc8NJI6tY-yRNB2Ch6Tkoz7xNWnEF5DQ7pl0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3731" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3731" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3731" class="active" data-icon-position="" data-hide-link-title="0">The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487094" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TCwZTdRBNvj1BvW07M_IVbJtvzgZ7op_eLYdKP8t1tQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3731.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3731" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-62ZKCtGpnQB4MJu3DMTwLr-Tv2aU9DRagJaEBJuYALg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3731" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3731.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487094" id="node487094--2" data-pisa="ggg;7/11/3731" data-pisa-master="ggg;g3.117.300227" data-apath="/ggg/7/11/3731.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Meredith B. Metzger</span>, <span class="highwire-citation-author" data-delta="1">Jessica L. Scales</span>, <span class="highwire-citation-author" data-delta="2">Mitchell F. Dunklebarger</span> and <span class="highwire-citation-author" data-delta="3">Allan M. Weissman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3731-3743;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487094--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Meredith B. Metzger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meredith%2BB.%2BMetzger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metzger%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMeredith%2BB.%2BMetzger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:metzgermb@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">metzgermb@mail.nih.gov</a>
<a href="mailto:weissmaa@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">weissmaa@mail.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jessica L. Scales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BL.%2BScales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scales%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BL.%2BScales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mitchell F. Dunklebarger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitchell%2BF.%2BDunklebarger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunklebarger%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMitchell%2BF.%2BDunklebarger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Allan M. Weissman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allan%2BM.%2BWeissman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weissman%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllan%2BM.%2BWeissman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:metzgermb@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">metzgermb@mail.nih.gov</a>
<a href="mailto:weissmaa@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">weissmaa@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3731" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3731&amp;title=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3731&amp;title=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3731&amp;title=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3731&amp;text=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3731&amp;title=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3731&amp;t=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3731&amp;title=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3731&amp;title=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487094/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487094" id="node487094--3" data-pisa="ggg;7/11/3731" data-pisa-master="ggg;g3.117.300227" data-apath="/ggg/7/11/3731.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Ubiquitin Ligase (E3) Psh1p Is Required for Proper Segregation of both Centromeric and Two-Micron Plasmids in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Meredith B. Metzger</span>, <span class="highwire-citation-author" data-delta="1">Jessica L. Scales</span>, <span class="highwire-citation-author" data-delta="2">Mitchell F. Dunklebarger</span> and <span class="highwire-citation-author" data-delta="3">Allan M. Weissman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3731-3743;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487094--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Meredith B. Metzger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meredith%2BB.%2BMetzger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metzger%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMeredith%2BB.%2BMetzger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:metzgermb@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">metzgermb@mail.nih.gov</a>
<a href="mailto:weissmaa@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">weissmaa@mail.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jessica L. Scales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BL.%2BScales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scales%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BL.%2BScales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mitchell F. Dunklebarger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitchell%2BF.%2BDunklebarger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunklebarger%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMitchell%2BF.%2BDunklebarger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Allan M. Weissman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Laboratory of Protein Dynamics and Signaling, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allan%2BM.%2BWeissman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weissman%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllan%2BM.%2BWeissman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:metzgermb@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">metzgermb@mail.nih.gov</a>
<a href="mailto:weissmaa@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">weissmaa@mail.nih.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3731&amp;rft.epage=3743&amp;rft.atitle=The%20Ubiquitin%20Ligase%20%28E3%29%20Psh1p%20Is%20Required%20for%20Proper%20Segregation%20of%20both%20Centromeric%20and%20Two-Micron%20Plasmids%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Metzger%2C+Meredith+B.&amp;rft.au=Scales%2C+Jessica+L.&amp;rft.au=Dunklebarger%2C+Mitchell+F.&amp;rft.au=Weissman%2C+Allan+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487094/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487094/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487094/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487094/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487094/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487094/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487094/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487094/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487094/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487094/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487094/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487094/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3731" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3731.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3731.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3731.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3731.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
