<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3637" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3637.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3637.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3637.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/11/3637/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3637.atom" />
<meta name="HW.pisa" content="ggg;7/11/3637" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.117.300169" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Sim et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Mediterranean fruit fly Ceratitis capitata (Wiedemann) is a destructive agricultural pest and the subject of exclusion efforts in many countries. Suppression and eradication of invasive populations to prevent its establishment is facilitated by the release of sterile males using the sterile insect technique (SIT). In SIT release areas, it is critical to accurately discriminate between released sterile males and wild individuals to detect extremely rare invasive individuals in areas inundated with millions of sterile male flies. Current methods for discrimination exist but are not always definitive, and a more reliable method is necessary. To address this, we developed a genotyping assay that can be used to discriminate between sterile males from the SIT strain and wild individuals. This was achieved by identifying single nucleotide polymorphisms (SNPs) linked to the maintained traits that facilitate male-only releases, white pupae ( wp ) and temperature-sensitive lethal ( tsl ), via QTL mapping. This resulted in the identification of one SNP that was in near-perfect linkage disequilibrium between genotype at this locus and the pupal color phenotype. Medfly from many SIT colonies and wild individuals from across its geographic range were genotyped for this locus, and results show its consistency in identifying SIT flies. In addition, linkage and QTL mapping of wp and tsl have larger impacts as they can serve as foundational tools to identify the genetic basis of traits that facilitate the separation of males from female flies, which can be used to develop SIT programs in related species." />
<meta name="DC.Contributor" content="Sheina B. Sim" />
<meta name="DC.Contributor" content="Raul Ruiz-Arce" />
<meta name="DC.Contributor" content="Norman B. Barr" />
<meta name="DC.Contributor" content="Scott M. Geib" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Mediterranean fruit fly &lt;i&gt;Ceratitis capitata&lt;/i&gt; (Wiedemann) is a destructive agricultural pest and the subject of exclusion efforts in many countries. Suppression and eradication of invasive populations to prevent its establishment is facilitated by the release of sterile males using the sterile insect technique (SIT). In SIT release areas, it is critical to accurately discriminate between released sterile males and wild individuals to detect extremely rare invasive individuals in areas inundated with millions of sterile male flies. Current methods for discrimination exist but are not always definitive, and a more reliable method is necessary. To address this, we developed a genotyping assay that can be used to discriminate between sterile males from the SIT strain and wild individuals. This was achieved by identifying single nucleotide polymorphisms (SNPs) linked to the maintained traits that facilitate male-only releases, white pupae (&lt;i&gt;wp&lt;/i&gt;) and temperature-sensitive lethal (&lt;i&gt;tsl&lt;/i&gt;), via QTL mapping. This resulted in the identification of one SNP that was in near-perfect linkage disequilibrium between genotype at this locus and the pupal color phenotype. Medfly from many SIT colonies and wild individuals from across its geographic range were genotyped for this locus, and results show its consistency in identifying SIT flies. In addition, linkage and QTL mapping of &lt;i&gt;wp&lt;/i&gt; and &lt;i&gt;tsl&lt;/i&gt; have larger impacts as they can serve as foundational tools to identify the genetic basis of traits that facilitate the separation of males from female flies, which can be used to develop SIT programs in related species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3637" />
<meta name="citation_id" content="7/11/3637" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3637" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3637.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3637.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3637.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300169" />
<meta name="citation_pmid" content="28889103" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3637" />
<meta name="citation_lastpage" content="3647" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sheina B. Sim" />
<meta name="citation_author_institution" content="USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0914-6914" />
<meta name="citation_author" content="Raul Ruiz-Arce" />
<meta name="citation_author_institution" content="Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541" />
<meta name="citation_author" content="Norman B. Barr" />
<meta name="citation_author_institution" content="Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541" />
<meta name="citation_author" content="Scott M. Geib" />
<meta name="citation_author_institution" content="USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720" />
<meta name="citation_author_email" content="scott.geib@ars.usda.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9511-5139" />
<meta name="citation_reference" content="citation_journal_title=Annual review of entomology;citation_journal_abbrev=Annu Rev Entomol;citation_author=H. G. Andrewartha;citation_author=L. C. Birch;citation_title=Some recent contributions to the study of the distribution and abundance of insects.;citation_pages=219-242;citation_volume=5;citation_year=1960;citation_doi=10.1146/annurev.en.05.010160.001251" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=N. B. Barr;citation_title=Pathway analysis of Ceratitis capitata (Diptera: Tephritidae) using mitochondrial DNA.;citation_volume=102;citation_year=2009;citation_pmid=19253662;citation_doi=10.1603/029.102.0153" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=A. Bassett;citation_author=J. L. Liu;citation_title=CRISPR/Cas9 mediated genome engineering in Drosophila.;citation_pages=128-136;citation_volume=69;citation_year=2014;citation_pmid=24576617;citation_doi=10.1016/j.ymeth.2014.02.019" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. W. Broman;citation_title=Mapping Quantitative Trait Loci in the Case of a Spike in the Phenotype Distribution;citation_pages=1169-1175;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663553" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses.;citation_pages=889890;citation_volume=19;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=S. J. Brown;citation_author=J. P. Mahaffey;citation_author=M. D. Lorenzen;citation_author=R. E. Denell;citation_author=J. W. Mahaffey;citation_title=Using RNAi to investigate orthologous homeotic gene function during development of distantly related insects.;citation_pages=11-15;citation_volume=1;citation_year=1999;citation_issue=1;citation_pmid=11324015;citation_doi=10.1046/j.1525-142x.1999.99013.x" />
<meta name="citation_reference" content="citation_title=Temperature Sensitive Lethal Factors and Puparial Colour Sex Separation Mechanisms in the Mediterranean Fruit Fly, Ceratitis capitata (Wied.);citation_author=E. Busch-Petersen;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Chen;citation_author=M. Li;citation_author=X. Feng;citation_author=S. Guang;citation_title=Targeted chromosomal translocations and essential gene knockout using CRISPR/Cas9 technology in Caenorhabditis elegans.;citation_volume=201;citation_year=2015;citation_pmid=26482793;citation_doi=10.1534/genetics.115.181883" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=M. Coon;citation_author=T. Nguyen;citation_author=L. Wang;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=M. J. Nueda;citation_author=A. Ferrer;citation_author=M. Talon;citation_title=maSigPro: a method to identify significantly differential expression profiles in time-course microarray experiments.;citation_volume=22;citation_year=2006;citation_pmid=16481333;citation_doi=10.1093/bioinformatics/btl056" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=W. Enkerlin;citation_author=L. Lopez;citation_author=H. Celedonio;citation_title=Increased accuracy in discrimination between captured wild unmarked and released dye-marked adults in fruit fly (Diptera: Tephritidae) sterile released programs.;citation_volume=89;citation_year=1996;citation_doi=10.1093/jee/89.4.946" />
<meta name="citation_reference" content="citation_title=Impact of Fruit Fly Control Programmes Using the Sterile Insect Technique.;citation_author=W. R. Enkerlin;citation_year=2005" />
<meta name="citation_reference" content="citation_title=Genetic Sexing Strains in Mediterranean Fruit Fly, an Example for Other Species Amenable to Large-Scale Rearing for the Sterile Insect Technique.;citation_author=G. Franz;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=G. Franz;citation_author=E. Gencheva;citation_author=P. Kerremans;citation_title=Improved stability of genetic sex-separation strains for the Mediterranean fruit fly, Ceratitis capitata.;citation_pages=72-82;citation_volume=37;citation_year=1994;citation_issue=1;citation_pmid=18470062" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=A. E. Friedland;citation_author=Y. B. Tzur;citation_author=K. M. Esvelt;citation_author=M. P. Colaiacovo;citation_author=G. M. Church;citation_title=Heritable genome editing in C. elegans via a CRISPR-Cas9 system.;citation_pages=741;citation_volume=10;citation_year=2013;citation_pmid=23817069;citation_doi=10.1038/nmeth.2532" />
<meta name="citation_reference" content="citation_title=Dominican Republic Uses Nuclear Technology to Win the War Against Fruit Flies. Technical Report bull58–2/5820405.;citation_author=L. Gil;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=Tassel-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_title=Programa mosca del Mediterráneo: Manual para la diferenciación de moscas del mediterráneo ceratitis capitata (WIED.) silvestres (fértiles) de moscas irradiadas (estériles). Programa Mosca del Mediterráneo.;citation_author=J. C. Guillén Aguilar;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Entomol.;citation_author=J. Hendrichs;citation_author=G. Franz;citation_author=P. Rendon;citation_title=Increased effectiveness and applicability of the sterile insect technique through male-only releases for control of Mediterranean fruit flies during fruiting seasons.;citation_pages=371-377;citation_volume=119;citation_year=1995;citation_doi=10.1111/j.1439-0418.1995.tb01303.x" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=C. L. Hughes;citation_author=T. C. Kaufman;citation_title=RNAi analysis of Deformed, proboscipedia and Sex combs reduced in the milkweed bug Oncopeltus fasciatus: novel roles for Hox genes in the hemipteran head;citation_pages=3683-3694;citation_volume=127;citation_year=2000;citation_issue=17;citation_pmid=10934013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=W. Y. Hwang;citation_author=Y. F. Fu;citation_author=D. Reyon;citation_author=M. L. Maeder;citation_author=P. Kaini;citation_title=Heritable and precise zebrafish genome editing using a CRISPR-Cas system.;citation_pages=e68708;citation_volume=8;citation_year=2013;citation_pmid=23874735;citation_doi=10.1371/journal.pone.0068708" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Jombart;citation_title=adegenet: a R package for the multivariate analysis of genetic markers.;citation_volume=24;citation_year=2008;citation_pmid=18397895;citation_doi=10.1093/bioinformatics/btn129" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Jombart;citation_author=I. Ahmed;citation_title=adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.;citation_volume=27;citation_year=2011;citation_pmid=21926124;citation_doi=10.1093/bioinformatics/btr521" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Kerremans;citation_author=G. Franz;citation_title=Use of a temperature-sensitive lethal mutation strain of medfly (Ceratitis capitata) for the suppression of pest populations.;citation_pages=511-518;citation_volume=90;citation_year=1995" />
<meta name="citation_reference" content="citation_title=Area-Wide Integrated Pest Management and the Sterile Insect Technique.;citation_author=W. Klassen;citation_year=2005" />
<meta name="citation_reference" content="citation_title=History of the Sterile Insect Technique.;citation_author=W. Klassen;citation_author=C. Curtis;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=E. F. Knipling;citation_title=Possibilities of insect control or eradication through the use of sexually sterile males.;citation_pages=4;citation_volume=48;citation_year=1955" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Entomol.;citation_author=E. S. Krafsur;citation_title=Sterile insect technique for suppressing and eradicating insect populations: 55 years and counting.;citation_pages=303-317;citation_volume=15;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Parasitology today (Personal ed.);citation_journal_abbrev=Parasitol Today;citation_author=E. S. Krafsur;citation_author=C. J. Whitten;citation_author=J. E. Novy;citation_title=Screwworm eradication in North and Central America.;citation_pages=131-137;citation_volume=3;citation_year=1987;citation_issue=5;citation_pmid=15462936;citation_doi=10.1016/0169-4758(87)90196-7" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. Langfelder;citation_author=S. Horvath;citation_title=WGCNA: an R package for weighted correlation network analysis.;citation_pages=559-559;citation_volume=9;citation_year=2008;citation_pmid=19114008;citation_doi=10.1186/1471-2105-9-559" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=P. Langfelder;citation_author=S. Horvath;citation_title=Intake_epis_food(): An R Function for Fitting a Bivariate Nonlinear Measurement Error Model to Estimate Usual and Energy Intake for Episodically Consumed Foods.;citation_pages=1-17;citation_volume=46;citation_year=2012;citation_issue=c03;citation_pmid=22837731" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. I. Love;citation_author=W. Huber;citation_author=S. Anders;citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.;citation_pages=550;citation_volume=15;citation_year=2014;citation_pmid=25516281;citation_doi=10.1186/s13059-014-0550-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. J. McCarthy;citation_author=Y. Chen;citation_author=G. K. Smyth;citation_title=Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation.;citation_volume=40;citation_year=2012;citation_pmid=22287627;citation_doi=10.1093/nar/gks042" />
<meta name="citation_reference" content="citation_journal_title=The Biological Bulletin;citation_journal_abbrev=Biol. Bull.;citation_author=T. H. Morgan;citation_title=NO CROSSING OVER IN THE MALE OF DROSOPHILA OF GENES IN THE SECOND AND THIRD PAIRS OF CHROMOSOMES;citation_pages=195-204;citation_volume=26;citation_year=1914;citation_issue=4;citation_doi=10.2307/1536193" />
<meta name="citation_reference" content="citation_journal_title=Annual review of entomology;citation_journal_abbrev=Annu Rev Entomol;citation_author=J. H. Myers;citation_author=A. Savoie;citation_author=E. van Randen;citation_title=ERADICATION AND PEST MANAGEMENT;citation_pages=471-491;citation_volume=43;citation_year=1998;citation_issue=1;citation_pmid=9444755;citation_doi=10.1146/annurev.ento.43.1.471" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=K. R. Norris;citation_title=A method of marking Calliphoridae (Diptera) during emergence from the puparium.;citation_pages=1002;citation_volume=180;citation_year=1957" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. J. Nueda;citation_author=S. Tarazona;citation_author=A. Conesa;citation_title=Next maSigPro: updating maSigPro bioconductor package for RNA-seq time series.;citation_volume=30;citation_year=2014;citation_pmid=24894503;citation_doi=10.1093/bioinformatics/btu333" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. A. Ouellette;citation_author=R. W. Reid;citation_author=J. Blanchard;citation_author=G. Steven;citation_author=C. R. Brouwer;citation_title=LinkageMapView - rendering high resolution linkage and QTL maps.;citation_year=2017;citation_doi=10.1093/bioinformatics/btx576" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Papanicolaou;citation_author=M. F. Schetelig;citation_author=P. Arensburger;citation_author=P. W. Atkinson;citation_author=J. B. Benoit;citation_title=The whole genome sequence of the Mediterranean fruit fly, Ceratitis capitata (Wiedemann), reveals insights into the biology and adaptive evolution of a highly invasive pest species.;citation_pages=192;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=A. Parubrub;citation_author=R. Reyes;citation_author=C. J. Smallridge;citation_author=B. Woods;citation_author=D. Haymer;citation_title=Mitochondrial single nucleotide polymorphisms in ceratitis capitata (Diptera: Tephritidae) can distinguish sterile, released flies from wild flies in various regions of the world.;citation_volume=108;citation_year=2015;citation_pmid=26470134;citation_doi=10.1093/jee/tou027" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Rastas;citation_author=L. Paulin;citation_author=I. Hanski;citation_author=R. Lehtonen;citation_author=P. Auvinen;citation_title=Lep-MAP: fast and accurate linkage map construction for large SNP datasets.;citation_volume=29;citation_year=2013;citation_pmid=24078685;citation_doi=10.1093/bioinformatics/btt563" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Rastas;citation_author=F. C. F. Calboli;citation_author=B. Guo;citation_author=T. Shikano;citation_author=J. Merilӓ;citation_title=Construction of ultradense linkage maps with Lep-MAP2: stickleback F2 recombinant crosses as an example.;citation_pages=78-93;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=P. Rendon;citation_author=D. McInnis;citation_author=D. Lance;citation_author=J. Stewart;citation_title=Medfly (Diptera: Tephritidae) genetic sexing: large-scale field comparison of males-only and bisexual sterile fly releases in Guatemala.;citation_volume=97;citation_year=2004;citation_pmid=15568342;citation_doi=10.1603/0022-0493-97.5.1547" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=Ann. Entomol. Soc. Am.;citation_author=Y. Rossler;citation_title=The genetics of the Mediterranean fruit fly: a ‘white pupae’ mutant.;citation_volume=72;citation_year=1979;citation_doi=10.1093/aesa/72.5.583" />
<meta name="citation_reference" content="citation_journal_title=Ann. Entomol. Soc. Am.;citation_author=Y. Rossler;citation_title=Recombination in males and females of the Mediterranean fruit-fly (Diptera, Tephritidae) with and without chromosomal-aberrations.;citation_volume=75;citation_year=1982;citation_doi=10.1093/aesa/75.6.619" />
<meta name="citation_reference" content="citation_journal_title=Journal of Economic Entomology;citation_journal_abbrev=J Econ Entomol;citation_author=V. San Andres;citation_author=A. Urbaneja;citation_author=B. Sabater-Munoz;citation_author=P. Castanera;citation_title=A novel molecular approach to assess mating success of sterile Ceratitis capitata (Diptera: Tephritidae) males in sterile insect technique programs.;citation_pages=1444-1449;citation_volume=100;citation_year=2007;citation_issue=4;citation_pmid=17849900;citation_doi=10.1603/0022-0493(2007)100[1444:ANMATA]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Schroder;citation_title=The genes orthodenticle and hunchback substitute for bicoid in the beetle Tribolium.;citation_pages=621-625;citation_volume=422;citation_year=2003;citation_issue=6932;citation_pmid=12687002;citation_doi=10.1038/nature01536" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=W. Schroeder;citation_author=R. Cunningham;citation_author=R. Miyabara;citation_author=G. Farias;citation_title=A fluorescent compound for marking Tephritidae.;citation_volume=65;citation_year=1972;citation_doi=10.1093/jee/65.4.1217" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. B. Sim;citation_author=S. M. Geib;citation_title=A chromosome-scale assembly of the Bactrocera cucurbitae genome provides insight to the genetic basis of white pupae.;citation_volume=7;citation_year=2017;citation_pmid=28450369;citation_doi=10.1534/g3.117.040170" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=L. F. Steiner;citation_title=A rapid method for identifying dye-marked fruit flies.;citation_volume=58;citation_year=1965;citation_doi=10.1093/jee/58.2.374" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. A. Sved;citation_author=Y. Z. Chen;citation_author=D. Shearman;citation_author=M. Frommer;citation_author=A. S. Gilchrist;citation_title=Extraordinary conservation of entire chromosomes in insects over long evolutionary periods.;citation_pages=229-234;citation_volume=70;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=qqman: an R package for visualizing GWAS results using Q-Q and manhattan plots.;citation_author=S. D. Turner;citation_year=2014" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. E. van’t Hof;citation_author=I. J. Saccheri;citation_title=Industrial melanism in the peppered moth is not associated with genetic variation in canonical melanisation gene candidates.;citation_pages=1-8;citation_volume=5;citation_year=2010;citation_pmid=20824070;citation_doi=10.1371/journal.pone.0010840" />
<meta name="citation_reference" content="citation_journal_title=World Crop Pests;citation_author=R. I. Vargas;citation_title=Rearing: mass production of tephritid fruit flies.;citation_pages=141-151;citation_volume=3B;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=Y. Q. Wang;citation_author=Z. Q. Li;citation_author=J. Xu;citation_author=B. S. Zeng;citation_author=L. Ling;citation_title=The CRISPR/Cas system mediates efficient genome engineering in Bombyx mori.;citation_pages=1414-1416;citation_volume=23;citation_year=2013;citation_pmid=24165890;citation_doi=10.1038/cr.2013.146" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Entomol.;citation_author=M. J. Whitten;citation_title=Automated sexing of pupae and its usefulness in control by sterile insects.;citation_pages=272-273;citation_volume=62;citation_year=1969" />
<meta name="citation_reference" content="citation_author=J. H. Wyss;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Z. Xu;citation_author=W. R. Atchley;citation_title=Mapping Quantitative Trait Loci for Complex Binary Diseases Using Line Crosses;citation_pages=1417-1424;citation_volume=143;citation_year=1996;citation_issue=3;citation_pmid=8807312" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=C. S. Zepeda-Cisneros;citation_author=J. S. Meza Hernandez;citation_author=V. Garcia-Martinez;citation_author=J. Ibanez-Palacios;citation_author=A. Zacharopoulou;citation_title=Development, genetic and cytogenetic analyses of genetic sexing strains of the Mexican fruit fly, Anastrepha ludens Loew (Diptera: Tephritidae).;citation_pages=S1;citation_volume=15;citation_year=2014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="The Mediterranean fruit fly Ceratitis capitata (Wiedemann) is a destructive agricultural pest and the subject of exclusion efforts in many countries. Suppression and eradication of invasive populations to prevent its establishment is facilitated by the release of sterile males using the sterile insect technique (SIT). In SIT release areas, it is critical to accurately discriminate between released sterile males and wild individuals to detect extremely rare invasive individuals in areas inundated with millions of sterile male flies. Current methods for discrimination exist but are not always definitive, and a more reliable method is necessary. To address this, we developed a genotyping assay that can be used to discriminate between sterile males from the SIT strain and wild individuals. This was achieved by identifying single nucleotide polymorphisms (SNPs) linked to the maintained traits that facilitate male-only releases, white pupae ( wp ) and temperature-sensitive lethal ( tsl ), via QTL mapping. This resulted in the identification of one SNP that was in near-perfect linkage disequilibrium between genotype at this locus and the pupal color phenotype. Medfly from many SIT colonies and wild individuals from across its geographic range were genotyped for this locus, and results show its consistency in identifying SIT flies. In addition, linkage and QTL mapping of wp and tsl have larger impacts as they can serve as foundational tools to identify the genetic basis of traits that facilitate the separation of males from female flies, which can be used to develop SIT programs in related species." />
<meta name="og-title" property="og:title" content="A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3637" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Mediterranean fruit fly Ceratitis capitata (Wiedemann) is a destructive agricultural pest and the subject of exclusion efforts in many countries. Suppression and eradication of invasive populations to prevent its establishment is facilitated by the release of sterile males using the sterile insect technique (SIT). In SIT release areas, it is critical to accurately discriminate between released sterile males and wild individuals to detect extremely rare invasive individuals in areas inundated with millions of sterile male flies. Current methods for discrimination exist but are not always definitive, and a more reliable method is necessary. To address this, we developed a genotyping assay that can be used to discriminate between sterile males from the SIT strain and wild individuals. This was achieved by identifying single nucleotide polymorphisms (SNPs) linked to the maintained traits that facilitate male-only releases, white pupae ( wp ) and temperature-sensitive lethal ( tsl ), via QTL mapping. This resulted in the identification of one SNP that was in near-perfect linkage disequilibrium between genotype at this locus and the pupal color phenotype. Medfly from many SIT colonies and wild individuals from across its geographic range were genotyped for this locus, and results show its consistency in identifying SIT flies. In addition, linkage and QTL mapping of wp and tsl have larger impacts as they can serve as foundational tools to identify the genetic basis of traits that facilitate the separation of males from female flies, which can be used to develop SIT programs in related species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7SEtWRgKbv8gTtRv-bAR3HhO4eiujrEu2ofrzTg1xAI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487097","apath":"\/ggg\/7\/11\/3637.atom","pisa":"ggg;7\/11\/3637","ac":{"\/ggg\/7\/11\/3637.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3637.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3637"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3637"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3637.supplemental":true,"\/content\/7\/11\/3637":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I6A0TeZ5rncuh7B+CkYwa\\\\\\\/Wnwe1lMEdvvQpA91V6btiliDMESEbDdK1O\\\\\\\/+U4M1UVzJoLf28U\\\\\\\/lWsOKRIi726q6BXxs1kqkOv3X9\\\\\\\/hJxsLwwzGboRRPuFkF+XeW9Kv+wiXYGpLSb\\\\\\\/lbuUia17OzKdwG0LgeSvuzKC2XwESYLxRop3wPb+mA5u878Zv4bagrPLqxnT2RisM65ZaFjzyxwA4LllccG+R1\\\\\\\/2eVo2XRELUChQrMVjV\\\\\\\/IfmiyJQH0UySG\\\\\\\/UtDaoFBoOjXi8e977mjApy9DSZxkFa6hbaNVTmF065Ulxy2OjYcbHkK3iofrPZAtHzSCPGg+7SKPkEY829\\\\\\\/79J5lt8hzoB7bPWNQSOQs9xN8DSOHEXMGd23qVGD8TY2N5rav96cVS7dRZ+5YJJl8imsxkNGQ67RgPv76ygoSf07hh4cUrRA1IX9FzsKwQbu1AcztpTc0aaE1JaeARefAXebK\\\\\\\/1rNiCCwk4XLZMX\\\\\\\/6bqSssrE\\\\\\\/FRXNiKqfWUIgNsT5gFoODq1\\\\\\\/xngvYTDEUAEfAiJ8KQnRC8yWGvpcj95OFlNNPwNygylCPi0FpAcf6w41r80FV3+h1nD15wO10UnCxcIP5s738NfRPShZUkZrR1tQPMfACINoKqVJFrB2V\\\\\\\/oZ0BDmx9BjnVFox2pVNKsqtHSzv9nfOw4ToYUm9gRswqaQL5ieZGcIv0ukXReZD4yd4IqzXc4C++acu7uDn6UVFOHlw3n2x06QOmgmHbfKtQHK1DB0O2GTtDqlrWPCtDXqAfV5jhCSGIENehjCXTOwzoN6GunVb8Dul2L+b2KioZcFLDRcaaTxBbVBS73QK6yxecFwNhG9HDi\\\\\\\/h01BWwq1epG2w8CzQlSYCCV\\\\\\\/JlnjYiDFlm9EOWHvE9RM+Caka2nc\\\\\\\/IFKgOVN7UTUaq3KJU3IbBUVs7UBrffTFir3CbDezws0bOIBsbUvmN8khV66kzuLJGlj8qihcBeDyVREifWz8cYH44B3Z8yY7YwIxqFKZlLS63A\\\\\\\/A\\\\\\\/OAWiqPUtk5j+w9xiktxn4TJTzhFM\\\\\\\/ZBvkDf\\\\\\\/9I\\\\\\\/NCqhD3J04SErCY+ctpLoLm7AlDqvxY+p7zrUuGE2L1L7vkMsqcIfAg4MdgOS1oz34BVSvKSxjFXH7lqs+5Pt3ZLYr5SB2g\\\\\\\/BLFqY+nn0qK0cusmMSsJK2alT3h7Jz1c4R\\\\\\\/z8pBc\\\\\\\/68BAQBbtSGc2PZojSbXS99TMOQW5QCdfzysAGcDuQE8j7ZUJmzaQZZBUsuWoDe0TGsltTlyUYVwY\\\\\\\/egqAlO6MOESfjkuP8\\\u0022,\\\u0022iv\\\u0022:\\\u0022X3NCIzUGoSlGUmrL89n7sw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aafe9ba6f8b1e93598123e1ffa4c6404b6a62d2fa189f1a810d79fe96c4b57b6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223ckT5ru+IfSTyBeJQ8PVzHe82V\\\\\\\/97qTOLccCrx2XhmTzXNm00aMwXSAMimmLuNPyqXrPbiQvhFDlZJXTvu17c1mcFG0Yue7aDrDOOyHwGy8OxM2T2g\\\\\\\/VzIs4xUiAutFIK87net1yI\\\\\\\/p3j\\\\\\\/wgLnBLczcxlg6lybg0pbHEXaLk6Y9oRl\\\\\\\/nvrkDB+uDtkRzJyLJlZIdD\\\\\\\/St88UOpyE8TsNFAI9\\\\\\\/yhTHIltMIy5nXMMnG0pbhGRWOphUyDgpaLjZ8xja32ZNUj5CGiP6G1JjdPlDjYewLDhZVsZ\\\\\\\/vqU5MbY7WiCGUMJtLfrjnJtJjX+lYYalG4f8Da315MLNnN+cw8T+vGX668WhEOHVTMDZIHWxxrTwvAKvQmRQm4dnvdKAZov9WGtdChFXQ\\\\\\\/jXFTV1UTzqZVyPnXaLqrRqNDP2TO2qJChHJpw57zxc8d3wrxyCKjltmqfxkxQkPXU1DHpCP8rxvYlbLbSr7ZAG\\\\\\\/0puZgpPwgKT5vB\\\\\\\/Pg0HF6SrhiZs7KstY7ncNDM9ydkaDSnMRANpwAvP8SBNpjSETkONvjSla6TXrt2OQQD3YQ4l5oihpGE4q72AzgsCHD8gzQWQz8bmp\\\\\\\/tw8aZvrbw\\\\\\\/fANOBLjW1Dji5EL6LqYqDLbYDgGYtOZOLWqurY7eFtzfjvzu0WoVjL8ZhW8uOvQjF1PWvES\\\\\\\/I4KxGGDntocXc\\\\\\\/AnChovv2oHIHntOj\\\\\\\/KQ1E6f7XbDWbjhsBcSK8s3\\\\\\\/W5qrIksTxjUDCe4IVNRwiW6l6rMW0y2M5RysiFv94\\\\\\\/a01En05G9tRqzqKjZv4TSskBUWgzvnG91PYhhJsJX9xO\\\\\\\/im\\\\\\\/8kGQO6aUVTafsk4CWZxoflUlwv\\\\\\\/lu3eyGGmeX5UufdXz861F9GI0NRKRzdRprS63rpjjHbEI\\\\\\\/HUqmOOYYxp9\\\\\\\/Jc4Ge8400Su5H2J8PkBRvEJMlVyKYaoqJD5GtBk4BjUX8R18geu7OEg09YbAGk1umU2kLhmk1C3ht91L2NM7CuTeZLIVHfkHeoii+UQFpdrPjQYm3S5O7+ebj9mi8Dje7EdW1LlhIHPDCNYkHHnocLbMLcZMHWCou3ba5X2tAJbUvAu24yeUDxa2bno\\\\\\\/c2+yuBnVCCWdPdOkz96x3ySIdnjY0NNxjdq6u9oMf0IdEBUniuV086ws8VVn\\\\\\\/YayoN\\\\\\\/7\\\\\\\/y7RH3qKoF0xXiqsF7TBN17\\\\\\\/xe\\\\\\\/vCILIFLQQnzEzPQxk1upNyjiD6EyREFK8gnKxaIObcGhsrXmi+qMa0N7w6T4kx12BR1+L7UrbVaMCn6d40e0Tl4kBu0QxbscNjBYXNVCXvXrTyLj5T0HOv7um7dZyJD\\\\\\\/+5ZTdDlRRAPPQm+aDxwZeaF3XAwPwXAo07JzxhkOKfh91ReJMUpn+dYopUtrZHhDTKdG8I43wzrA1vR8Ap5e9XByT6LPa2qrYLKsj6cdTc7bwge57SAom6Lg8tlzCrCGUFaXwLq78qnRzUgUcWwEfhiMDIGyr4hkWnfV0WMVOliAinBn9XYUGAGWLyqD0KtJ1r5NQispuVZRlJJ2+aKs5Pa3BVLUuxxF9M2QQMO1JQUQv2uxfwg8VCKPxe710uWn2e93Ci3X4sApnSodFfZBQebuZyraMfe\\\\\\\/ZSVf4EjhjQc5eRUHhb6q273U+eE2BPTckpfM4kSNsq1sTEVLPd\\\\\\\/NQwinW8yUNf\\\\\\\/72kDy9BUQcgJqgRk70JT47hA33vZK+5eghLRXSQhNIzLJJEha5QiNsuDA4Nf\\\\\\\/j12fj6PMP34BP16zfFR2WYRkvJy7FAILKIW\\\\\\\/3ohDPEzzlod7qlDCL4vlIGzpBN\\\\\\\/cxYYCTh+xXiGlVYwu+J0OKZdTAk1gN9hEPuYqluxV6wxYwItyWtKp2FUNCiFz6\\\\\\\/rwVWVKLUMjqF3DmpcE094+iy9pdhCc81DItj60VV7C7OzvFZXYZ3sMch44ngFwPZ+SS5KDPN0nv6XJYTnh2Mi5MLVUYryqosIHvity3lpTLm3RMZmbztgxrQZBRL7vurOlT4+8KaqFIJGe4q+Qcqi\\\\\\\/xSb68guWjlhjKiNturGDtbFP\\\\\\\/HebdmjBYeGc7BDjhmrJimU7b3xsfqpEF7RR8iYmDhVFKs\\\\\\\/CHh4r2iOEAsOrfL6QIxxkeqJGDKTVCbxahO07b5bQhZ\\\\\\\/3bQHVf4i5eUR9SDch6u3TzRJXA7Awo7tzhZy9jEruMZkkWXn2lodlvcMEdRell0tt5tqm8ur0rj3VdA+t1+PpZa3OVes1R0BLCBFLyU6ieOUkxWjnaUZAgncmtsPVvhLRHLg67s4v2DJ3yh2Ld6w0SFbA8cimSDaOG1wi\\\\\\\/rUYNJ9DD0aENY9mQj+ywM7Vd0qoYY\\\\\\\/HiquhjCni\\\\\\\/THXjA1yqkmfj6bmw1UO\\\\\\\/72lUlcQhoTZH5WSNuxyds99KOtPctu8epsgCLFK4PYZPVLGFyXVEp0ZOdKcPrahb8bqMopBYfShif5yJAZjwbAQEymqqrYnX09XC5jD0Ezvu8vqtoi4+O3wxA2aTR3Ar\\\\\\\/Jri78HgZu\\\\\\\/lprXEP1Md3AMERTXmlPEwfmkssNi0Gw0oiqPGT93l8+KyLDW3SsDUN2\\\\\\\/+T6zOvKX2hmPrv8HIimJWDaQa8jjxXK1EKrO4+RBbUMIJeHwkeGwM0eFzbnSKgmQ0+ez99FYFFWiKKhTr+sOsk3SvhvK9V5k6+GdEkexFNVx4DuMqfwCPODXVBG0Ip6lv+9vWuniR\\\\\\\/4i6fhiFJgUCgAMXS78A1dYF3Fu0p05EO+ZMTAzOoAQM4g9xWRf4ziy3X41o1K42xYjzmOmkxztN5Ob9zgtDw8o\\\\\\\/c9rQL2F6mMdkRnU+RbGtQ1j2Q5K9CLdhQE6jkSqmHawY+oVPGPBAFBW\\\\\\\/YUdgf0DulpPt7lL3nf90DwLoMrON9b0oCBNA+E9auSk1tJ4SjUIjWGbHotpytD6p4BqhO51ViNoAp48Cw8TTPjc\\\\\\\/4CZ+4BqNJTpkX3Jaatf0x2zX0pd0ZIQ6jMuRVNf1GYn403eNN0cui2fb+tdGNAcWqwnZqFHMYNEp7yAdiFu\\\\\\\/yHrI8STqwjAjPUSz8IujjIXKGDYer95hZeS\\\\\\\/TE0+i5+cBIUyQXbGDr3TkwvjHoZFw9C9iNt\\\\\\\/qdkwAI7PR4HF8MkGYPsHyE\\\\\\\/3uMH7ZWL3sbELKBvs6FEGatWOV2pmS6BoltV0bh5lFoof6++mC+thKVkUaT1vo1Iw21TI708dwcm77PxmoLvZPN0\\\\\\\/cZldOO3O7qvwdKGFqJQ9iaMcIoNVPTzE4BfSXjDXMqsBYd8Y18cDZY0sjTawzPpM4UGCHrw3SsipqLSNfT2e+nIP7ZLIQ7apjMZaaqVMrCS0hlx3bgBIM2Wtm2S8pAGiQz3ElK5RPokr+il37is0Uh9HnnB8GSYkFzKuQ5Eb2B1gK9m72KxAQioGyN5lPLmDvupU9lCn+T+6zCDWw5hAFYx53+4jyZ1jhrozVMtym7uDKpOX3wqhU1KYM+zzIx79E+pakM0\\\\\\\/rluvsmbilhE\\\\\\\/i7XvbpOvq69g\\\\\\\/AjAx5zTVofGpLXC1ed95gwu+sSId\\\\\\\/9xX4Z8phaBXNlilBK7JbKDQFrQFN0135BOhaWI+xBckCExd8X6mfP442wH0gJlskoyioLBLJFoGAYNhE+rwTEm9ucHkRDHx7AzLzhUQPaYs\\\\\\\/pkP32ygVBGfLHL2VyVPGGsIlbSoLf1PQqPOWd\\\\\\\/a\\\\\\\/L0\\\\\\\/rZghWVMEwtOMw6nldUGhl9piDMJ8GZfq5D4Eok\\\\\\\/5migTfSCaCHxsec4LbbGhJAhVfDTsTuhBPshGDxik+rnzhqtiJLPMcjYoQa+rOgKmiMEnWpyorx4dCiFg2WRUhNHsxY97R5q\\\\\\\/ahhqHTQDZHmpg9fwEi6XvGfSQvWaitiP61eJ2HHv+ngOMi3yeXZ9tEhBLH5cs+J0Q9CFm+\\\\\\\/ou1BOc0nNX4RHn2JSEOazHMOWBR7Bb35I8LZrYwnXpRg2ShPbBJsZ087R9YMBe341\\\\\\\/lTrgYP3fKxXZ4qQ+cj\\\\\\\/dHg80xlVUvGoDezKhPZKY875fgTr9MY\\\\\\\/n02BEHFBNCFQxdfuwnOiJNipVNDQk6U6nDgBO4kt9g40zmpPLiD3bwS3wb5FnAlSZUBfbhmqYVfSl4tMkolBjCh6irtBvHa6PCJspiMQcv7YHSevpWqzgir2ITwF83l1G4bdwXBZmOtIDptfdYtjFZ7ZdZVTZSfKDgF22N0tftv6RWm1PzF636JvoUUEYWQg7mO\\\\\\\/teo2jax8L8GcrWwe4k6qHOaU4MhIjfirldpS7j+eFikGWeBJUlbgSvLGNxP3nTpttm5LmWyZh3qLNmorUy6vw+LdslWCU1G2wgrmqWII9WNKIXmEwgfnUgwOejRYeE\\\\\\\/QQjHGlBqPVxk4\\\\\\\/KpFBx+yrGqVb0KZOo4eli9HXa6MpjV4VZ2pH1BV1w+sodLwsHOWOR84RLjwi7rpZ6qIpMkw2+sEdxfSttlNqYWwjlZuzcz\\\\\\\/WbD8SEdq9XeQ1YbBV1azgwy4VBtxElDFCn8Vf2yvJC2r7K3PJo5y\\\\\\\/104VKfv7Z9jXmv7k3FtkAXG4an7lkS\\\\\\\/yhWwZsNaY3wJ5JixDJMxHlW5kl0NNeC6SOkBGWoy2POfEtAnFsSmw840uKHnKZolFDv+m+5xoEvfoTufQpFVQFb1pR0MaSpmwqfribnYw8R609BDGPxtaqF9DaImACtQ2UhHJzeFuYvz4zsCZjzTCxtAzUDoSKIxHmhAAs8DOpy8kjfpZ+sjLbwQXe924wYygZH7AVnRrwuPlVAwdREuNbYr63COHLiOhvDit0BkMpwe3uGASECLVKiTdtQzdpHY5R3QY4Vt1HzG9sNK+sFqPoENFPjEZBJTPhHMlyFlpFHnnrZy5gn\\\\\\\/AbK\\\\\\\/REIpRZa2ejYSm+dFfmocC96\\\\\\\/N0hs20Tbq2tdmvXnomb2Pm2yDIDIw5Xl550rvqLATN9HBHS15bz5GUhz1QeOPL4FDYCHrSoVmRVuIvDUyZu9aSyHwdNPNhDnumolytG4EFqyfgi4\\\\\\\/6wLnm5Pi+XEb1oAD\\\\\\\/VFMo2KIYOPbjpgbCy8qW9mJUrcu5ijgGCr+3yL+ueOIimW+zwWomPwQ7lqY3vKnUNgmXThf218z3kZtQG4e9nOQjob7XsmsvgZKetCR90KACjDF7s8MMGXvs8myNQSSmQeXNR7EgoP59x4B\\\\\\\/T5neL+4XuJeGCYyoPJJ3IgNO+QsiyIL3EdMQmXQTe6PohsukJdn+eJsEgxnbFpFPlPb\\\\\\\/5odvOib49IhmT58gqtNrCeMhBEZhLoyCwppK6leRlYq8fo5yNUkMIuQuaHmeywc56dte+zLcE\\\\\\\/lIaItbzsM1K8SiHghbzovVMxWSjX+xOaA24X1Q0uo7kJLDmMdj0Xqu7SsZqMkqqjJ93dsvFnVXp5HPq0M3mVDOrvJtRZJhERcc7KgPzTma5mAPuuhz3vILsGpIIhMhXgQCl6sFyX37EXoU5wnBsCikeGFo4h1vSUuHpnn5uzljoHuR6Q82OpyfsWI\\\\\\\/20\\\\\\\/dgwHrrA5Bq+WxyFePI7G1fsQMaf8esWRAwwKc3YpIe5wCTBbx+KB8Xbc1kkYhhbl9C+rkFjKclRbmVcm7k0L\\\\\\\/WMJtbY9\\\\\\\/Yl0hExyNdMkTqFAkr1bNqpTkCasY5V0XJMfaz1GQByeSScvs9k\\\\\\\/+lwt\\\\\\\/HGm+bYAkBSF27JN6GxTQI1khzKTwXcXMiqSKi7TsBrp3HOO6OoDj7unLaC940W4vvNdqHbxD6Y00nkjfeyiOsf6b9gjp9ro+fQeeHOYJoplOCzjRsqKbUUKufIlOcvQi5LBWgDjsf8WyIS0kv9VCp+bDAiK3d7UUP\\\\\\\/3FSDqK9pv+9pNDZ1\\\\\\\/NleFsGHMU1SOwXwc\\\\\\\/t1qr5Y+UYKuHR3bUOGSFRIvt4aXV7mYiuotzoKCQRTDVUFISVZh0VVP1HUHPX\\\\\\\/f0D2RTAOh6J0lUwrDSxKcxaAj\\\\\\\/5a7wQMyku7s4JahEE8ZrfKgfGSOp9PINKgpoHbmLO4j1RpLrc\\\\\\\/7j5bCI3INNY06CXkLq6pZZ53n7XpEagiuIRccf8HZ5wSXE0vE8UQkFfG75M++HRQUspieR1yOTRksFyzcIQuqva8216d9rcgBoBglpwxay+gxM1sU12Nz+VROxUNDTLAGYrzeZd0YiIdbyuNTiMiKsBMvTRse2cg3DC0lqKjoiCSc5UAw+kPS1XZtO\\\\\\\/ryaKuoZli4rpZt9DV2lyRDzB+R0x0ADfvJKADlr7l53wSImGsaH1zwZclyc8ZnqggmTShRcxmBDZ4fGyLDdOugzsBPnhOm1Mqk3IlEde7EFGRVYP\\\\\\\/iIumOAZkSJZTWlb6Q9qgWDXZKb6mKkdBiuglCvIvCsG2KxZ8q+p5fB4ZpeVrmtv5zVPqEa2vuXqZ67FMtjRHCOui2Xpv7OE\\\\\\\/bVcM5A9gFDHk1pJXJ9RUtEjKHdrMYvvyQwguSQPBCEoj8uMjQoXOMhmbtP89B7\\\\\\\/TzJ2SShoNNe4yRfVU181LUP0Y0Ab2xQ017Z2cOxoF9gAX1SrYEK33N8+N33DH3x4w+rFH0j7BR3ecBrgN4aMDdcbIkXL9xCSriIzuO4IniwXueQGUWrL\\\\\\\/o5jWFEPp711XN8Zpg19Et3jgibpi6fFYGEar799kkVZeWNLmkMaLbkdIWmLfW8qC3g8yCTKeyHFCuMsyK\\\\\\\/xbWXAc9yS5nrpfi47w4Uh788EJ0tKrjGxwUj7hhqWI8Ybog5B87tn3PAkH6vjjA3oi\\\\\\\/H9kAhUIy+9FW9BwMyA+Lay3RxOmXNKyxUpP6rP6ncQJzvBVE6GCe9qBkysucK4qmmiRebwZqx2FxpKeQDKttv4Pfn4IEY\\\\\\\/A3Df9\\\\\\\/a5ShXgjADPPy5dHp6P4sah7NAE1U7vaKtZoHIqUi71W3Fe1dhyDHlVTDaxq1RnkyO5TXZsn+BKTujImGeTh5O5dze4YLUQZO8h3MFh6PQp4tvNiwIOvVIVAxZtsjsb5KXjt5x+KsSykB3FBfVOgai+jORpPjsY58M2FjQTrVgr7m+yxKtGp6hWB8hPF08GNeQR2piWivrbSeGgVEBR14srHLBfK7AwsntMC0iLj79QvMY+GOfq3nM2Zs6cr9iKMTM8NYcF0QNpODeCsMs77UtPFsYe57sKYkMWbo2iShxd8u07ddSvoCu9\\\\\\\/Xp4qa+HCpRx9551eRUMrZ7PVItiK0m+xGpqY30PAMhaGnlFksC8cF89A7fqwCu9DmNoYJ48YQ0tl5sM5LNQBbSgSDw6xx89SoNIU7bou8gFylpT\\\\\\\/8Z5qqJrn6LsYMV2yY16CBwdJ3tqVjUJ0MdxOw9GOpodj+buX\\\\\\\/ZZ+zfgx6tN5yaYcR193XGQP4HzuqaW0TcCe1pEn7mns1T74WnLOi5vrGCm9l1p1yvL15F0ha0jb+WWxvZwgqNbyoZ+GhUGZvEg7LDCB1sqo6gztZlojMt1HKKGzJ9ZNThcE1KLlGhvPsM2g+yoOK4wijf7qOL8b9+hJcUT\\\\\\\/B6QopXpNFCCujSpJ7ZCxyjRmnRgzmJTx1vOP3ApCuXbY12msK\\\\\\\/Q94qbGQD+AsTk\\\\\\\/MxXsR7YfERCY\\\\\\\/nMf7AW6MD+J0lEUaL6QlhTD9jk3UNzaCKeZ86vcr6mayoWhBIjSbr0SRPlUQNDqSTvmTZcfOgD8W0ciy6jVqtVnb6OWhVns1lcUcKNqLlNK9urNmLhL0jCvzkG\\\\\\\/jYD0ULOHccEFefpklH5IyoDcWSzspGRWkPmjgynOc8BKBCT1HwK7RY7\\\\\\\/nSIhAfKX\\\\\\\/aD8rbtpTV12ot+GJzT0w1jIaBoWz5KOOFHG2V1Y1zZ\\\\\\\/98QC5b3Cob49SJl6eddetz\\\\\\\/U3m16AB7N3if47Ayf4O45E+BIY+YsY8kMnnQYbXwWtSzJ8161UTPD007LMlqB8cYDDnD5K4ElPGnIdXwRJlsGAuDe9gTJoulmOplPupy4kq9soO0mwVI+drTO91zBTIgprUj9gVs4wWRKlCCWxCe0Ou6JhkQ6Xg7gjtxKfETymhKvS7IL2k2ZkTKqCSD1Irn9rr+jvZJ8rCgg7t9Xz+\\\\\\\/XZPW\\\\\\\/tKGUA2+7E5v+Tc66UL\\\\\\\/h3kNg\\\\\\\/RZJ57CDzZ6FSe7wnz9y16fPdyRIiC2p\\\\\\\/oxb0vJjySeJgbG8HmhU6Luwdk2wwlBZfD3FSHOon601X6dTN0c8vbUM7dK8gaWiAAB30HCsOy1FJfoApHG\\\\\\\/1MeOmFJoM418fX+PYsFh4VPijMajwcx0l4i77G21uPpsAPLpZ1ey\\\\\\\/EkIwtgD9DZVMWdWVpF+B\\\\\\\/5ppNAg4CDxVKhUEh6Hmx6gnyB49sqEdZnjosuTBsTY7LkG8AXqH02kl7c1EovmQSrNPo\\\\\\\/QArc4ytK5NQqTP0JNPmLl\\\\\\\/5TR\\\\\\\/r\\\\\\\/7UK0Pgw2KFWm4Tr18F1xcoYUv\\\\\\\/EIhBL6qcwexdUt14IQ7FafNyxJLlXBfwZSbC8e57JiYU6IID1PIZwKTgyW7I9NLaKef46vu0jmQUVrHdHDOedxcs+56iB5xUujUvwCP7FoBA\\\\\\\/ws89Fa3FJlr8LEBHSENs4XJxrG852ncfjMjBc0QE2EJM7IVQkzhRA055PH3DOb9BuC3GoSEgO59O4cSsXzANC+Rnp7G8OTiv+KOj9ZNTXnc54F28FgcmPWHex2DT\\\\\\\/nCMYAjq3ynFa3HkyJd7u9W48D2NE5Il6R7bhYG2uep+2Woo7tJE40X3pIsIFTZeGVRz78PJXSxrl7AKZ4aAU1W+eX4E+yI9oy4f4rFULM96bYX6PMYHkLwK+y4R58gThq35npbnRSpnTc2UGGvgPasfMlFSJRv3rbWRHQ\\\\\\\/+3sU6QwCStTtpVgAgoYbZJf5A6sSQDb824I7ZOGhNNCPLKfoYJ\\\\\\\/azATyQuQk5\\\\\\\/uVSkRlA5tF0FLbc5QiLG5ZKVtdo7CgVFH3HmMS+N1I5BvXa6ZGe6Cj151diqVzguINjTcqB7dj6+6JmbLWK+e57ppJzziB8v72G\\\\\\\/VAZoNlPf4S8aqw7sCEjhFIeRivPb2wK7ePKXjDOStu0yEpKcAKOT3SqSOmcTEYPqxCVesXvyVmvqZbs2toFYKmcLqN5Vzl3bQ7pVzCi6KhvvLf2IKQCOfnVln6992tqSgDU5lqS62jxxltVMHvPrqpv70VSALaG+Xp10EXHEnUM2IwnEcOG+6vsHRrZ2uj8ieeYgVklj2CQ\\\\\\\/I2tmObcx3Ru8ikzMyAFlPeUY8IFIphZvwkXsrIvDzVsN6D3R9KMBMscJu87S+9eLSUGGOPRbHwDoC8MwqFsEWhA4zkAf78\\\\\\\/MDhjGCQM2xwRAk2s5s16jZlRJj0Kf49qhtrNYbo3Oe0pypxoaILyU1cBwBxJ7zCAbAmKvVXY7VTgiRQ0tNIUGRqGFSVJjtqBKQEMaYNc3+Q3oEKKuyFp1ZZsNbZJbShpUUr4hFK6VS2d8aAaDzzjLm2QIB9+nmVS9PD8MRbqXtf+b9+6+WNuWt1axzZQA7\\\\\\\/sWcOlTvcQtm\\\\\\\/SuaBOi6mf4OP+vDIkrHvtUPbfEZ7mySN6qPzTxKNN7Dtsew+xzIGjDFPH1\\\\\\\/T4xPJIWXU\\\\\\\/wd0NQldcihuDLKAnQDavqA\\\\\\\/O0d5sy4UEVCUNDP9kvwEVoXad4QEpRmhO6d\\\\\\\/OENErHp\\\\\\\/MgRPcaPDie\\\\\\\/0oJHClq1mTwa3A4AS3zvY9QXv6an5DVzXsemZXDp1G56zXBXdOoKOqHeehUtD\\\\\\\/mI6qOT+Pwq379OjPtPwRRAC6lWb+56VEimOH6qK9TmK6HfW7MHPJt3IDpb7ZCYswm5r4tyiebl65EbkBRRpS+jhxiy5ykQMhhZm8mkxa\\\\\\\/QmdRWHDIDclUYJx9Kctt7FtUQkiU9km+vCnLvHQs2rHcGS617eVx5OawjWSoaPOZfdSBgyqdHQ5Zpk5wyJzNqkCVUj3y+BzKxwlU09aB2hbwvEwMmtltHbZxxzj3x5\\\\\\\/Ep\\\\\\\/X+R1\\\\\\\/mLPO57Rwft7sREWYHtCWdhWGDPgEnteziDjYTrwaJrbcq9Lsx704mnnqG5Lzx+8EQKQyy9V40A3VMhQRRZw3V0a27zbgqAaa5unQ1D\\\\\\\/zLh7ku6CECF7OO1HN1UVwtP\\\\\\\/jF8rmTvpurDkqhSFAqJiq9GRw34ycp01zHG6IqOh8ljAuhbqarkPtgyaSdPK\\\\\\\/KgDQ22MC8eUBdJi2w4ZQZpmhA3uqOVtXF7FgZODVC\\\\\\\/k9XwbszUWTanGozzAbInLejIcPerYNpHMbYduaAjnQHMkhfiapNT94\\\\\\\/wT0\\\\\\\/XXODAgkUfHYpuiJ07COfWgAy66KNK1OtTV+h7OJeCdhAnUMhVl6JAfJso9J\\\\\\\/7q\\\\\\\/6thHMFe30CDN\\\\\\\/aP0Gf9lH2QEwUBiOAWK6wrS6qP60X2FHMLsIf+J5SskWyybPwoZ1z+ItaDBjHu7ErG\\\\\\\/eOWha3sP2ZPkaB3T1bdtuRgSt3QjHAsjuFYSYfCnJS1avG7gMjWDmBmA\\\\\\\/jykLaPGcGNa7yH6ZdLSNbrh\\\\\\\/KpFyTZRo8JB1fCwQFRRIM24LB2xoFIalJpOyhBphMjicEXin+dhSacErmQUKDMXE1SFseWRxDpkHpl2RMgmsE+JV+hCy2nzK3bkywolRwkkwfAGt2WhbLnOkmmC5TC\\\\\\\/huldZvRUAxa1At3np9sQ07az8IOp7jZARfJ+XIk\\\\\\\/Ro8BPak\\\\\\\/jlu9VdTSrnCU9rzRHJ9HtSQT6ZaTcikg\\\\\\\/CMR0FFlRE6Nv\\\\\\\/t8mBJ9iM4U6OnoKbyc+05wp2nCA\\\\\\\/x0btDZRRYag42iUF1ZOfDGvFQ9NJz1e2Wn6X7gk+j2+uKCpyQS6eUZxtoQSemq28lgLYzzAbNMwLcLtVJmWT7ejOam3MQcxFBLGZxJvnhqsaGVyGAiuZOv8ziTwgoTitKZOAiLIirWMo49LGzYHn9CPm7SDiQ7wHCkKUa0HipfWLfQqqx7EsVqEJjKiqliSNmyzWncWwqzK4yaKBveVP7DfRJGTHA3C6j+MJzYKJ0JgDMIn+b3tkcDFokJYcxc3rtr3mUYAfOdIT\\\\\\\/gwXhL0Nz+O2L52pA9rWWhF7XejNwd3FXvtmJdxkso5Rscd+QSXr4sSPGNpc7bMWHhlVaBkqZ9\\\\\\\/Ppk0f7YgSTq6IFDwl\\\\\\\/SB7NS0LUqylDghT+9kGcx7bSrvh3g5Wa8lwQ75XSly7kbBnmqlmbLVGWv3+hyIUgChza16M4QN7GsPFu\\\\\\\/P42dpQ9Fcoq3m9oylmpTZfKodJq5O6Eybhiy\\\\\\\/Hd2HRho9uXpEzC2EOYcLqWYV7zP22x9L+yxtWlHNGmDH2LvBkohn+zeaR9mRbrNxKoZ3wVd6YRVFRB15g1X\\\\\\\/CRzrR1uYvnIojiFJoQDfh4W0Wpu8wLyYvdZvQtQqjAQl8g\\\\\\\/CuHfkNytTScNqdXiVktS1kC2rXvySqr3WyhmKEMWFDWchz1OEu3DxQYWoCoXAxilTWpl\\\\\\\/HWx077H5VUKy86H+thT9IsIonGFMcx4wf1tK3X4+0br0oq7sn6IyPrh3BOC8gSrylHThVPMtESakTnlxgD1Mf\\\\\\\/lp4W6Vax0px9YCt7WAiETCp1kfBmZUUm6m0H6x1GUyzeTQwSoJtVxSkm+CcIMRCKpXUtO\\\\\\\/WLBokjjgwAw3oL1fqGJuB2R+vDY6A3LVHJhdS8oHqFb6KK2huWK5X8byc14v7XtX410aLTXxkvPCSC0i57IQCcYQacBH2VndsMsA5LbSHsSzf3aW1riz4JLgjHVSofw2ZCQ44VniEBQkm8MlsnnexfUDB2acjYsDp\\\\\\\/pYypknL13TIMpENU0yYN15pjMLo7pk0AdB9zQKyiSaUenjvNlJYH\\\\\\\/aTKFQQMSK+TI\\\\\\\/J18+KI8vkMyalhcV+Bi0WlKsjZDyLqIkt6gjB25Poq2r044GU8E78VKujuMKKXo9391TnDdoj\\\\\\\/ciYr1BJzZnFC8hXd1JkYLLLBH3IPnKnKH7WVNqyW2qxRFZGct0vGkU+JAkLUypyLdaYyYz9Pv\\\\\\\/8D0folHyQOBxvvkgwdz4DpskN\\\\\\\/80CTK1lKZiFedsgH+yAcCHsJVwNmtj8qpwb4EmLe8MyuX52LCpuZegbJBnC97wVOP6aphl+D7sEqe1oVn48Y91IWj5ZtDd6sHLvz0pkGMetilGgjdtB0lBntapS6AI6eksckcw2GgVjbXobDgooHWR7cZoXeHc1sBTvSrHL\\\\\\\/1I4JcfNtg3ukuRegNVdgf6WF0dqYAW+WVtQVd6Gb3aPOw6sogLm+ZNtJA\\\\\\\/KF\\\\\\\/af1P9bk\\\\\\\/O72eDxfEU0LrgDgUfT5IC5+EgXEBXnUdpoaqT6AWXCCKgOexvNA83JqZalIIo+Aaxw4IWHKdPiuze7JrkCP\\\\\\\/Y0+pIqi9adiok+woawb5nGIqv54Z05h05vs8NO7EcrT3CJO7\\\\\\\/MWLqc9Sp9C\\\\\\\/16aqYCwsRMvM6chjHr\\\\\\\/Cm2DzyQ\\\\\\\/eqIu4kzf+LLedSF4dfNp4BO13Bqv7B4kjYUo6JnL4UZE0ximeKfq6LgVU2HHaKr1tr74X7NHBOgL8C3NktVYC0Tbklq9Hhmkfll6BlGPtQHdgisJTEy7jOkIeUuDOBR91edzhU41NrCYChmvd0NUVVRLgXv9qH8a4crNV2FNToPEsoZ7xbPQNEeCiR5n6DgcGjwnWim+CK6si0UBCD9W\\\\\\\/M3Knc\\\\\\\/MfLtxWbK+aJvRPh9J2hue0J1Lcx2bRpPvkOCXRYYTybOdnmX0nB+mQxtNo7jUOCgFW1H+7GAMP6m3jQnRkVaInYnfm+XE7QacvJTgA0HtD8rJV\\\\\\\/gzDYLblCnUvORlOyCpsz9Tmowmr6pTOFD1uDheYy0+bWUbU1O0UfF0z2u1Bwveez8admiM+YHKGkSrO7ESB0dnen4+UDsFHjql1sYcOJqNYtEqxuNmpt2Vk7qrhelQqoBlc1IBLVrVCUwWGhb58C7ikPPspDQfKhZc5kDm6oJ9jCcuUQqDF0Tyc3V9mGqdOqsdcTTYORC1AfqP++Cthvbd5EC\\\\\\\/lcSmhk5TqIMw9KX5LWiOPi83o0HDcB45YJJz37NcZ1j174\\\\\\\/SPDuPxOch0FRmnLjoFBiNw2XAaLNzsRSWOxuGiTN7D+acQ2bUqhEk\\\\\\\/bX4dEF\\\\\\\/WMKMXX1ZLDLCLl5jvSK3cRH6\\\\\\\/gnFUZQzzitrusH6aqZkOWe40C+jnu5XLZwpyI9K4s+LqqFapC3xh0M5OD\\\\\\\/\\\\\\\/SfjL4EI7OiQSkGMXiuZu\\\\\\\/fQ8nunwqzMmUoI\\\\\\\/BRCxd2Jge+vVLF\\\\\\\/wkk+KKEwJYPgALQkLb5xc5kd3kJOrhfUrVcChLeAaEf7HSTWJpbVmZLeWIa64kTdNJvZU3Vcav5KWhwMfMfMx9rb48dDi7VS0qcgv8WKhlOR4cNQdCIoffbAwdu162AWLJyIam6crKdVIy2\\\\\\\/xQLR5aS6HNFjqbzikaOg2M2mz2rP1lq4nYI79cli3wOq\\\\\\\/maSlLJO\\\\\\\/Uuj3BGG+3wYl0WUF2apIV1aecL6V3OdtT4dFFzUleiUaRvcrN7hS9voHUten1p2Q3w27Q7q3u2NXHKO52+AmLnafH+5D3alXPIff9CL7Zo\\\\\\\/JDfNPh64f+MYgUvOPIdexJdtxSQt\\\\\\\/Aw46utTsmx8PtWcQki0SuSdK7XmJXxm\\\\\\\/Xo29wIHTL5Ao3L3ylcFT8WIXqyRvGy4jJe3DkzXD\\\\\\\/WKJAFaa6jKPVdCFhUdw4p1JX1vnfLhZ2efrrDfQpV+WlfRmDDq61NKd1HW0Y+WDozosqMU\\\\\\\/qAEiCTqAh0Ass6dpSpaWLt8Q7F2t7h0n21\\\\\\\/kfbemg2EfBneEVkz4Yx195kMsy55PppFEcCoknucz68rcwtNRYz9uw1Nmru0vH8mIt48q+EMlb8Mr41QVisM+JqANQXXgZBYANdc7FyRC5hKvQRfBEpvpy+eVWcIThm8dNtXsnIE9QTp6urG4f+jZPrKm\\\\\\\/A\\\\\\\/82PpSeh59ATSaKWQNkQCJKN2eFrlkcI3V+9dS\\\\\\\/YCYhsbnBF9W73OPw4TSrOtPwVttscyqJ2zKfvnbRDr1uVOOr85PflX8yPio8Zbpfz1tTvfOEt4Qfd2TVJ0+o4C4noqZv\\\\\\\/k6fY\\\\\\\/lZUSHKM9s5ZtKLuij8SqcNK\\\\\\\/o0IVPD8yA7mzEq09bY8keHY1C3F5hTMv2d5JIDAximBT1NKdW6dRRFTvE\\\\\\\/19iI3G\\\\\\\/dowC0zFPh+GNiPKOjfinrmCWTgE3WZbGRzAG7eI\\\\\\\/sItAwus5Mb8wbOg2BNnpo+4KT0TaYOvxR\\\\\\\/EbxlgHJHXKn8EsVF1mMDhM2xMrH4TNEQKsJqKjvOyNoe\\\\\\\/A7\\\\\\\/3954N6WgMuKrR4oVNBAe6ueBdfmCVj+yYTb4qvKAOTdCCgk1rfFmYrPLwPbZlQx62\\\\\\\/mm\\\\\\\/XYmUsL41W9qNIvBHrbtsomKUVbG+PQINH4h8cKOIISUMB\\\\\\\/QG32arZlXsPEA89T4vzHYNCKzuQ+jCAR0BadeGWncfprpQC18kqXxFFW3z9jIYWMqNdlmLEp+Fmal99OwddO55Cc\\\\\\\/gzN6w3Gd6XuTpF89mDGJKuWCAt38JX6yC4IEr2+O1VoqvJ8p4X5C2HFEADkplEIHRlKLDTEcQvK0sFd+2aPgnelfwx4lcxLd\\\\\\\/X8xZFDax3jhxlcowPjCiDFofOlsvrQfI\\\\\\\/x9pjRM469xbNWYZmifP9qYztUJyPMdAeLOk+mHWIh2WjyiIlbI7ReJAZe+beXSfJgw+EShW+Gj6PXILZ1ccWghR+tkY0T4nRajWpLC2OXBgjmLb7gTfSfVE8Dq57bNhzwEOhmY9bKllkDPZwO1HO3zr+cp8z8ZpH+esEkexVxbaVIRYM2b6ozRpQIhJjZUF3\\\\\\\/eN44jNA7gPc86l8Og\\\\\\\/hWm+maUXYsowxgPcbYkMvTI5Bbd3JHfGEX62WVw6YDnEXuPtdLOlmfOUJyk+5t\\\\\\\/yRQc95svGkfdKbcwbOmZe9iPyaTx7XcFTT4aIbcH05efUhNiusF4sU7\\\\\\\/i6sODhS9NtPknkSA4U2CGyd7LpydjwDwh\\\\\\\/4s\\\\\\\/OyNBWtVALFttmDZ98LV31BChiDccwC5jPrykKwKt\\\\\\\/PdrUer6DOi5sCCh4IBeNRkk9VMV\\\\\\\/IkGHajRnXVJxO4\\\\\\\/CrzlQ93CqLs3\\\\\\\/mpq+pXlGST7Xyj5kjUyMFWlCKvEE7J9Q+NTdKxnM017vGAsQVeTM7pLDa6pw5te+RQbCh7iq\\\\\\\/wbPu6jzPs0W4CyulFR98FIY2Gb4qgWNHGoIpPp9\\\\\\\/L4N\\\\\\\/4XPLmRz4gBVUW4Hekj3OkyR1cHNs4teDUgZ1ejD9F0xnTEPLbNliYArGkFS4N8GlrQr\\\\\\\/Wiyl10nX6DXswAx9OSFTHIxZMa6Og4ZKm36pRL8BZ0Lw48Nv0jwBvJV9F1JbJQRmNSsKOSvrw+ZxkhtS4LJNiesG6vU+InnX2c1Ef3TBuBlsGPO11uHapMHsbcFAx0qgNRmek8DxFpKQES6Xy85gBJ\\\\\\\/MSANTDUnKapBKE8x\\\\\\\/zBkCJDG\\\\\\\/PO6ySWFMmpBoUXgRLEmdOLmWDY2ntXdGVzMQO2o+pyaLvv1935DsvSHingmATJXtlixyEzwK0ToFVlqc98j79rUCx7sNMUJJNUdu3bkDApb0aNZX6MEsvRx\\\\\\\/fu2O2qma159UBoTAjLRUyG+kJS42cZoR\\\\\\\/jiFVgod5bwNTwSbn6a0v58\\\\\\\/Sbylubz527lTZlzJZcskIPu1CCPxjtKcNL0ml+R796B1BiWR6RJyxr8kXFkKsy1+nVIPgvNGl4xhlJEDtaffk0WtxK3EwRUQ0hrdK7wlzBkxDKhZHsrinlvHcJ4sU8wKqHhQH29n0mP6roFyDDrj9AqL11rncPiFWEGBPHRVef9Wv9qDOXPxArvfM65HoambkJ39e71PAeIelFq99eB3A3+NRZfWtnlmd428Biqj6BEgkeVYgFnfnAon3JVm4g8xeIfLPz0LW93bR+pjTGSKnS2LFiWYnfTQXj4UsPajeBj7cyB1oCwT6vf\\\\\\\/Rc2\\\\\\\/Ka80RLeiy0HHY3uEIKdQVjONL5GRhagY3PUCuipjgNZS\\\\\\\/rOBs91+BtKE4txAeAppv9cU0Nsn2BM6Ipag2W3PREdqPXCrObxk867zPh5uT9i8xk5vW90AzMduo0bwM\\\\\\\/RxDcYj9cqowY6Wv2bDhO6vgKF2Jb64kmuEnnl+P0vW28dEpEyP7+x7dOIyjfM22I9ZIUc7d+rHJbQev2EiY3qFC8EjaJYnxgYAWl5zJH4vZMC1IM9ye5NitSbVb6WNR821qpOsrsdu5DhBs6jW4D88b\\\\\\\/REJSD3KL8LiRggD1F5zwcvCW3TG9sPsw7jrK1kQ3XlzCpoQNsh6EsmbJKPSLn4Mjx+YUJg3akPOvdmxi3FjI9pib0i+B2zQi+b9av8vogMALIO4HMt4UC1og1h4+JNyeL22D3WvEbNPGJxwVfLnGde3E7ffg3pRQ8JizJFXWfyCvjdxA7GjQZzTdOVQKYA6xeIUA2Zrlm+FCMwMmQK+q0tQEYh8jg18df2k967k8iQ\\\\\\\/Z\\\\\\\/sq8pLrcfg7ngbnuPWqFA\\\\\\\/yp1sXZgVjJP8KvGrQtKv194hT79xm9OO8pTsDIJ2kXK6B\\\\\\\/L4jnLx12+2Vdw64rvGGl6Y8FFIfv2cE+YMvrF6bq\\\\\\\/MJKzG2jOp5IZRE9\\\\\\\/HEwTAM1jTBXWr6p17GU81JjskmKveyyOmBLmBH5n8haZ\\\\\\\/AQH2YDQ8x6sn6TWzUUhlrVjXq4vxEEvJMTcGeNMVhky2h036Ew2RBh36OVR8K93PTSZLAU3eW43+jWAcu+N1fCDY8I41mVRVfdcDcneLLMpS690ar0ooZtUKn9GsRcGANDTP+U03XiRmZ5DcU64eYjBe7HIqXfW4u2ltCtjU5oW2hisF3EVQkMsZxT8fzWxrBhsMwV\\\\\\\/duDNcbxwGc4cFpcfMqdTSvGhz062O31FENryHDV3FnqekOqpJLNqrUCdcDEGISynri\\\\\\\/nyORn4HDB\\\\\\\/9laL2nm5G1tz16Nj8EK7Jgf2bE6\\\\\\\/+slDV4+NR\\\\\\\/PgIm2VJWHKLSbtWU7YzP7pqau3VrY73QU5oxmJw0VCmSYRlF4MatQZjXabZiDQjst7GSsk1lTH5scsDjTtq\\\\\\\/ysyCaDhgBXM6m63\\\\\\\/0sQDuxJgumteWEdxH90MUMABKkLs88hBVCcOnPl3\\\\\\\/Th0YYnsoGS9JKCdD0SwtY6o8nDJ5IeHbcMOM4jQkN\\\\\\\/U0PsrllPukEA8uNvqmUGDb29Cv3CEFDFcV6HHg9\\\\\\\/VA4xX85zWkA8qCWsD2QA0YVWI8QlPht0WB5ig7ty6h2DfhG\\\\\\\/BYfb+wzjikb5g9vSryMRKs8a3xg1evm5sP2NGq9Lz0ivtDVcAtIzSKk8MUb6YCUcJlshY6UPkFNF\\\\\\\/h\\\\\\\/oOmyGLU4v75RxBytHUE66PJ49a55Ycb9YBk23KbhQHrf8XQIol3YNEdfN2EeyXnElV6EFwToUttsHDZ5gj9cxIXYbmTkwd52RqrJd6KXhRUqq0k4Ai2q9\\\\\\\/ZkoZ0ExyG\\\\\\\/XUiK5+BEtRf2OPrNfOGV5FNMmKhmJ+JKAqz0NWkrhIIw1IsqiVP63+805UEWq5u8k2VliYWe1\\\\\\\/pgLJXAJvqGc1EizIZ7WuC2AyGW5URuzyNHgDIQxgW85oHzQgeht2yDrIGvEcth0ccaol3j6JVleCayBtzPg09rnplCf0ctXbsS9MykWIU10B2plwft0A+F0QT8dNVJkjpT9OZyNL8fMQNaJXn9oO3tzQf203f8w0Zh7uFXP\\\\\\\/qbyRoBd+KKX4bxI5ytunqOyP91tlaf9JnUffNkuOKvIIg9BrGjtv9+JIjq6\\\\\\\/BqUdRqmZpqtIxvKdKKF2jWmAH1Gp6ov5LXDi9KpuyUmTLWk9l\\\\\\\/+O2EpNt3sk+\\\\\\\/6lebAzXyvJ65GEiD\\\\\\\/TuRtPtr9AyXGX5h7Onn6EqSfGOOsxi7pOHGaniA3FuN\\\\\\\/tNDYFyGronCpwOmFvrxjVGoHPMgDomgkiyXs04ZJZg+XXjs\\\\\\\/XepwhTMFIMTZ5jSaJJuPXXldYF+zRlN34jjydgu\\\\\\\/yCRx\\\\\\\/Hn5LoZLH9CoDcDxKXdL9ZmiH1jHz4XPo2OZ0RCRGXTCEW2iYV1Gn9zoQ8rZRnDph1SdC\\\\\\\/72e907cNKhIYVgFSwVgTbNV8ZsmgM670ijJT\\\\\\\/12px8fwdjdE\\\\\\\/TBUAKo9\\\\\\\/wF4UaQXjkyy+SOmYj\\\\\\\/K23atsVRjkTYvvBJUuQwq8OYOR715YhGYAqaVGaMU\\\\\\\/NfpJkFy\\\\\\\/3H\\\\\\\/A9TM5P3O0DdUHYg0+3X0QSEjE4kKRQ49Wtf6K7YaTytreJ43dHXGMCYLjIsioKKGPO8coXK07fwBxOQ2kNim0XZw1IIoZy10549Y\\\\\\\/gfpI+w\\\\\\\/rz5CWhwdhose+vPWvliJp71w0bYB6\\\\\\\/tsjQgWzGFtlGBErapYC0GMzw5Wl6xGPJT2SK4kBHM\\\\\\\/Fbh6oprqGJnOV+5K8MhRf4C0DkJxYkBlxYGWrWaIJQDnDsoH1qODygbWEqESTpIHWlwn363ornzBUq6vPrRA8FSnXghsBn+YrY\\\\\\\/IBf1ks01HzjFX6TaMFx4qjDpc6ijEGIVcgfzeSrVsODadA6MLqtTQzCjB0CwNhhVCALt9Ia6+Os6FVa2wBUkzYWXJTt9uF2fLoaJ6CzO98gYDpFKo7DLIidTIQ7kCmAQSZSz2JTpy9vulWZ0jZ5j4eVLigNVTGhdHuOIleTOXD9fMXrnYe8viIXVpMfC0tPhyF8PdBFElIt2DNqHzTtAvXakZd03N6FxqaRZ6WlTzVg6kTh4h+VhoXiuLew5xSqqts04y3Z1saB13592wPNa8BzRIaCa73HmRvOnf6r\\\\\\\/30EwmlN7fuUKpEmcmbTKlOOKUs5TvErYk3uHt6Pa0Ttcnoted915CFQM3stce1iiEp\\\\\\\/NyZJMgEdb2282aW4BcuWEqiNdt44MPymi6vFQTT6AgVHADE\\\\\\\/p+iywBCXcMmhsMw2AB3o2d1XMrZxY759UsUSeBR\\\\\\\/9cCeWOsMoKweUrqkwANMofXIA18htKte\\\\\\\/EdY+dR8XKptvO+0KBpfdLRwneCyO7NO1lOgzE0eHC\\\\\\\/dl61r8JoD2g+5Tr4YtLa+rH6gYcLHih\\\\\\\/1Z9aOa\\\\\\\/43mJXU39cnqLGDCkP6Z+AE9nRaOPvTiICl3o9c\\\\\\\/lMBUCs2QJs2KaBJ0aXQMj8zfoMbn\\\\\\\/p1N0mr1Q+\\\\\\\/VaRIWUTzk\\\\\\\/g8+GmA1KgM8C5N4SKLXcn4RtITE\\\\\\\/\\\\\\\/8wEBJmIEQQBP2+EWj5lrjZcK2sh\\\\\\\/\\\\\\\/fXHxCzoETud5I0x7it04RVeV3ViboZlAeDSywfFJSvyoUpfk4eizRnGFASF\\\\\\\/5xjdfe2m49mtCYW8yei4lnoRxBYY4cNH5Ylqt814NiKcjCPfVxV1AEyS5DPaz9S+bympu8HsatMTGqI1pL65FIDkjVqWg3naSIrTKAP0XXhyUDx59FbaleV4filL6tyG4SWAYqN8\\\\\\\/mJ+zHy7ymxX+o7VPB5axpy7nrxDmg7u6QqUnKpDc9dFcpvUeiBBdi+K4HHPf5d9J5DM9pLGM4id27uZx6BnvLuxT1+hXsR76nR9jYkfQud7OXjwwvfKu\\\\\\\/RtOBYg7mVxmbAVWnOTDFw48IOChVGsF\\\\\\\/WRdNW9XyqdAJK3t+zpTc6J3xYOm9lfXIiYPmO5DCNUlGQnlHoalsCtnyqsota90XKWVxq6WvPy\\\\\\\/19m8EPrKDci4k5LfMu0qBa8CepTo1VSROLmia7cScASXAJG3bnupW7bVx7gFf4bNqzFw\\\\\\\/UxrOvjK69di60e8EO9OC1Ld4ttVZWmoVq8sS5jw3oxsxBmAnOTTEbC\\\\\\\/etG2WymBPjLwzc7Hthby4vFpfU\\\\\\\/+p\\\\\\\/Cq299\\\\\\\/hkxGcHHb84ox9HddSQrdPEl3s6trQhwPvNbe2XY\\\\\\\/ffkgRvw7FnqpUnFud7pUscFloXP9JlsffiHNUDrnDCYOX2mpYZj9ljRzwKhLyzSgyOJXNW0MEGblZEWEQ5NusMhHR7Oo2UENyrCX8vULtiWODDZbQg5cecqW4VMwHKrca6RkersTBXhDe8ATyUec98\\\\\\\/4AkH53bkkFjGuzQVa2TZMCMjNp41l5DOKJxPdn2ZXqG\\\\\\\/bwlKcMltRM+4lloMV7hwYL5DqSN2\\\\\\\/AuICASDMbrcO74rAoR4X7ruOOCSypJ+LMtaXW0j+kXRv5xXGVUIOv3xHm2pApGeWN3DNuXe8i2proUbzO3pA0vHpK2rSzfQGb1Cf9mY6migvXygi+V6\\\\\\\/Eq3QWHv\\\\\\\/l3u4DHX1iOn+a6HqE+siteHc3MEMiz+N4\\\\\\\/jXBVYCA+wYzuVlk8J4gv4PWKUYGsgNiWE+KzKkn0H7UqqgEPjo58BHrBoikxkOKSnFpNyJ5hQnos9OZySPaSCC0m3yKuKfeZAXDb1r4zKGFOZY1xIKtSO9EfocRtkCay7JYz9rr4ka3u0Co2pibQOuL3mSJslImJhWXIuiDXl8Hlmeu1Sy\\\\\\\/GM749iU\\\\\\\/CSj3HpXD7Q9EuIlnJLqrIiZ1tEgcj2c3YoVWYE79zn1h82cvt6I\\\\\\\/BFayvtZIt55vGF9Kb7ZlsUlyfe5pfi6\\\\\\\/nAPFRmsCJ9xxjT+sdbm4R3JmhFwl5qAFRqlEYx6OzfE0PKQrsNaQwbugSOaAd33YBfIo73FzB7uRVElA7WitA1riLi6himIB66bb\\\\\\\/94YomLmIERecwI438Nyrnvcf2mTvQp3Na3tGsowSPpvSLzms6ccRaHzmr54Tif4Sg8cgNejlsAKttKvcWDiqVc2SaLNMAjVbmpYnxJ9IFSZrk9b6vZhyhdqZCmYHL7guhDMen8jOiSXs0Ia7Dqsw8aqe9rmnGrbPRdivGTLZEX4TKvachtSlbMzp6IYgg3xTe8W9vvGhNBmuCDxBiyyp8ZpqMDNjEbvb2VNZtAY+0EhBG2s6\\\\\\\/rn9ro71qcpizEzvC6d1A7+RsWwdLlb8CfYd5MSKo0JjhpVLubWmvv6D5yo8TBst4fNqq+n0jLQlWb7g3+TkrXgbFbuVywWDtOH2WGUsQ7EIQmkNO37sO5r9GGUlTmwE6JwRbu95eNLTtkBAC0R\\\\\\\/IV2YtO8vOPxSAcf1uexPXK0aLNt6VZUdZacc3TwYPcSVY1ulvfTYp4qNwEZcy\\\\\\\/hdBb0uHi9lY3tg87EX1sEunsTKXK7btE0FKulEv\\\\\\\/T2VzcmrV7FMylBhlYxqJYFBXhLUhf55oy\\\\\\\/BjAEtqBmbVWiiHrdmoIMdvTz9w6NR26X1ZkaUCKCxOtqT1Z5eKH+LXmp+pEeBB3kOR9O5NCh7Et1lqjzc6PSc30tv2bJ9ggDWqRCN6jrUZGiHfzEQS4OLwjT6Fg9PDFb8aGwXTzz1sTHsG4hUyHtT6VjEeBdMXPKcjKDzvJU6avzdO0gx2TyP7MU0MQiyGdPMOOsXKM0djfhDDlridWH3Z3cnpoN61VjSRRcY00RVQWW28Rh8Cfvfq76uE3SRgqVV2RYQDT3D9WNjntEHCr2YGUH8ifRPDaNWmSOw4QFDxISckqb6A6rbhQxGfcAm\\\\\\\/fpO1MrDYMYGEtqvQTsGbSJY3Xdvqqq\\\\\\\/9YicyZV2Q3qniXRJmJVdJjcTi7cph1MIC25AI7A5BTBY0qs5WMDAsXgaYTQlZeZjzSHLPYIOh4anLXSuqbNxrLql5Tc+dhLR70fj2u8xZpybwB2Wq\\\\\\\/zXA3tBCIYHfk\\\\\\\/FisHRBnuaeTCMD3DOVEJGvylbJcAKsE5Ld80gcHqepqU1DtneCZZRxYjTj7DP1BzXHh+VF7mGqtAKcV4tERvOzQ1JlEH6DTk1L15CTaRzhd\\\\\\\/ChViuUVZ8Ri2hLty89wI3xUIhCLMBEq1+4jwwUtkHkjNCxqJLB7mMFyZLoGYM3wGlb+EzxhMjqoP105B3YNaRzQ1jLvdDzC4O5OlqA1gIZPIAPC9eOoEuOWeu1aIXBkIVndYb\\\\\\\/AT+\\\\\\\/Xvk7jSGQSDwqXL7Ujnpzyu+9nqj2\\\\\\\/s9NbWKqI4nzQeBgV0IeSqbyawCK174b\\\\\\\/0xB9XwBQa3dqNifeAxDQGCrpj0xmeOr49snjZfSqaOmRup61MlHi4nOHhnrOWLDtlDxYwwuKA\\\\\\\/fT6dRauYhJatWNgiCBV6nxkknLLfV4Gef\\\\\\\/+EqpNJ7BOQiGaFEuhtmSx+KPaw02BmzGoIWIhkUk0iWxmZ+oCcS1oCJMdAXICVdgdAWeMnwaDZjysZ3RkFr1vc7IvdcIFNjn7ALRimqqG\\\\\\\/WVcUOxbxtMUqbOMK\\\\\\\/elIi+VGWjPtFeRtZMJwR6Z7AgSM0ggN0fCvCnHXK5dKbcq5oGGHRmMMai1YMJHkV1PfeX79ASocKwEIDZy6vNgRQuFuXKU1+O82ywF4lLGCjBjzXRVpd7qMQo5ycf07EFTRh9K2i8SdoVsSmxhL5vBQ3\\\\\\\/qQyEvcQCJcuvNa7YN6UyLu0gf5Er3QgHSrgqBfamOVQO6jx7K1sTDLUsRggEApak7vjtJm98BdDtdUDkElm8Iv5oWZEDhE93JeuWJzc4nM8WQv5QzWamHltPtE39qnIKENirIykOQwrdpW2d28maER3XWEL5HkxsSToLDHqrJwdjXao3tmyeCj+bUVi\\\\\\\/GUe0SEQ67wY6eDGndxYGUWykrzscsgPjdY2jqElutiUpF8rj3MfBrPr1s3YYVGyXEV25JufO+KU0PlmA9Bg81WdnXGKqW21FbKGqidNpAS2ya8B3xwezSYdNJtct8gAhc9Dqcvg94fAlDHgClJqM59nPbCR9vYmWOozsMxpvamKn\\\\\\\/XCPkhQl4a2nHwG26tSbwubNwor2Cc4VawLIAZb5KJDO0rSkTNw8ENHkayfPVOLsIMVKY63bJeVC5avTJInq\\\\\\\/LzruVoiV0LvyVrsL6OSd1VD8UFhi5Kd2ngkL2+3TOdoipS+ioIpnNZRdHKpqk6hmUKgCiW4pkUUL5Ys6t6Lr0AH2otG5WtnCdSZwrGvU2+pR7WRNKp2jUFZ9iT3oUBZEBGloC7HOdfWlsvj+h1Vsof7iM85c4YIEm0oa1IOVjgQoXhXO7MbdpPirfkoe8ZENXwNZ5P1DbLGxkO9kLhdjjSymN0FUw+lgmr2GKIR4cjjfAjJc55fSCYPC9l2xQxURV5ACwzkgDE1M+4tk0H3Q2fWNJXjyl75Zdsr1OyXKi+hHsRg1VEF57uIDCH1DvwJlz7fQEK8Z16JmJaZ0\\\\\\\/yC3Qb3lrg3Sj\\\\\\\/mo9z86u8Rus8Mv12W1A4c5sPVd15qmZeBgkBcax9rX5CooryN3YVPViKK3k8dAGfu7zXK1R42lgS8Q13LDJJltRl7FWN9gxzj2Q25yuuHOWW8OAWScHYATNLjdaRpTzU6NnSEFWXSA5krqRsNbv2hDQCl\\\\\\\/YcP4DiVT6+EkrSfbEpXsS0gA4EgQKYVjIQY4KsLcLbxhjWg8Uq61V1NxU+fC2Do8DOPOMZaZoJSvwIQY8EhMSmFVsgHOuwZDuRoRMWCtaaxOJOvoZBz\\\\\\\/ZO59TNODcQiCfKMY9epn8WWXA7IqPRZ4KuLI+omOe1JN1jncSc4L9K2giHXI617bT0uYWQgzkmGYgT+0NooU\\\\\\\/9igit6JKxLd19cje4zPwRkfvVvWp2y4ij7MCSM6yJUTLL04Wwmo6NAT3Gq+X5sF4YXOeSPL0pH6FG3kf4sVRutkMDS0tO+A9czS9L014TpNd8q6keDVPw6zVlmRg\\\\\\\/NP2Ok9Z7d3F7rCcK2A4Ha9Emzgq\\\\\\\/PjRYS0rj5oP5t9OAcstEG8qlQ4yVjZyP89LvyAiE6\\\\\\\/ksqhmRCrxwqrIk59YChjrpprWHtFvRfr7XACxHB9mgmE5DQxvY7TdXH5NNp2cA9yNeU34CIw2LCuKs4RKSiCGBMWAlpZfYYUa6K\\\\\\\/iz0Y6kJujsVdwtMH+urF4hoYkFq9WgyutswrZSlLAyr6CdPavNFTXav8\\\\\\\/vQYo+Easno173XFcbZYIzfU150fnYr6sW4dGSlpu7dKs0V27Pv5RTfM9boSg29f8cgWJKe8qvKSyIg2JfLsDDEOlOxI2l2HNI1O3kpoVOpW4gHihDxq7wmN3Eljqx1TrJ0eCIu1fOVKNg6jaLb1jAT3a8Zy\\\\\\\/sqWVsIJUAqnkjGQCGNDDT7vzJV\\\\\\\/3EIP8QL3epgxHA1qusYn\\\\\\\/acMM2GpWkFNnNXOfcojYAEwTEYpHH+SXY2TKOULI34xdPJmdfBzul3sibixYB75mIrdrq8RnHfLTU3C\\\\\\\/lHrhJYm5bUhoDEyblKhcgNivlG3\\\\\\\/8DROw94hz0FuQ+Rm3c23rdYFcSwVO87Zy2eqcJGqksxn57HVmQybGcxQx+LIGdEG2yhO2fshyCB\\\\\\\/QJuSCnDQkFVaVCOC\\\\\\\/PTKvcnxDidrdLk1\\\\\\\/sjacW93SLYVQa\\\\\\\/21tswuQX++Jw885sOadpFKqTrp5wjItZbjNzd8\\\\\\\/+H5Mddh3eUqDpoV1iM1M746rATIpRpUN8LhWYub3p1uJibnghQcAKkldXgt3vFb4xr+5eM6Mxph2szduvI2a4FIuk5J7CMJLJa6FafvvIATcMKAOZJM1P9reViPr\\\\\\\/JrZysdXk7hSmaS2LhROl7FT8eGwfJRpCrAK7SzZUD8ud8ZAZ+8QMt3VF+2oc2YC37KZuayeC6DXXMq8ZXUEEM87+sQg1TIa96PKV+e7Wc\\\\\\\/tn6RulGY5pasQCEjIRfPK6+cR79Maf20sTrx\\\\\\\/0VAK6eaWD0K3a8QqBP0ab5nvUJZsXEDcmwkwvmUl6Gy5uq\\\\\\\/49IBAk\\\\\\\/lAiwLCd7YKzjBpMgcox15Po42O7DD6IKqNnOGEFOGlnFYPeCx1BKgtY7p5j9npDsqFVNLyLOaAAYC9oaNOU5tv2gIWNOXZyHZZ9R\\\\\\\/fRzidvhBBO7QBKgylEMj88+q1hv9VPniClvJSCRngvtCk5dvcKi3g+L0hc4Iwtz3+fBmn4QDypnBSlAyF7wYiRKBSwz7+tkxn1bR1u22E4sD2N61h1SYvEStOj9ER0tPVP3GDtvvWPgNAa5oPzdW2agA7LmzT9SLqeUC36sRlPMKRbuiR8X+ic\\\\\\\/3fApJducCZpG\\\\\\\/eJDeHIj1WkVdN1sX01\\\\\\\/6rWTXSq8G8pQctH5OnZ4CwRD2NPFA3njQOeqihKRV24Tv7KY+BK\\\\\\\/HtYlD+ybcWsU24FVIRMHiTeT\\\\\\\/TvkCTTgRB9eR4WZ8egUTj6hG0tanK\\\\\\\/tTLU1hCL4AfoTEeiZZs9Etkyf9vGaSJychZNcFheUKRZdOGFdaEjMOcPT2fLbhKzjakYu8nKR5OCXSEHmCuzMjDs+o1D7\\\\\\\/faSODVAA8NxZV3WlvuB1Q2Nysed1vzhb\\\\\\\/WTObr4stsuJ0XTHkvtfPqoG\\\\\\\/AP8qKGVqHaVroSq+4eEUqOTBfND0nwv8KUHpYebibhQgppjgN712lDhr0LxN8B5XvmRTWfHfSRH3OEHuxOrK+xjjHFEafYja3OD+MGKss6vs1XiDQormkcCTL09RE+gvW2vK25ue3SS5Kp8ScLnSCDHslCAUNimpnzUv2Rrqt2hoNtIUmxC1eR4T+K20txUJaBqft3FGiI2YAMn3qV7Jm0qj5AeigY8QrxfHz0yRukb0aozuWGAyMkwyA\\\\\\\/otHTgpfpDXcVc\\\\\\\/LT9o5oy8IoxOSfZoNLl9Bl+K7QNobIh\\\\\\\/B7iRTeWxnQkLFt8ZPdFi0BfrAaaN\\\\\\\/FSw5louItuVAk6Iodr1SsiYm26jlK7euwBj6cuYP1j9oGmfu9HpzAcZisbMwtTKfBCDRm1FkD9+61AX\\\\\\\/PpPzPdu0C2DPSFHxmh7kJqbJZhT1SvuXjDiEhTLvJk6hwXH707jycp0blHWP0iEQWxENwiMurN6vfUd+r3ljrrSNqWyRAXeuwCB3lRYwKUL2JOz4meoDsSs78W8jrKo4z3ygl86TEfS5HvyQN51R\\\\\\\/YOAIqrJCIHFzbLgVtQyUC0Lyx31kqhP8Vll+oxzkAmWb5pc3Z4uxSshftA4OIcX4M84ZZuEGNTKsdrqlQXOdCqkM8e1o0wsRJJfJGo2aaAcQAFFZ\\\\\\\/49y84ap\\\\\\\/mQ\\\\\\\/bMrgnjVOns1BNtwTifURvHHDVTrd5QcnKpFWJmORTmjzSeFv2CAzYxeSxrHqBKTcOHBkeWZKqBzPkWnSYhtBroIfZFH7KKNue6De0wdcC22Q6agpNJc45fP2jY9\\\\\\\/vAD0CUeRl+Z\\\\\\\/ysLUXAkHgM8bkkBecXuryEfRxPKUOVc6tZIxx4A7LHjShL5g2k42gbQUkMhX41YVJtTiBnixxD1+KoxnPzYIgk1NPWH3gfpS5jq5Qq1\\\\\\\/6SJ+Z2yAa77Cjb6t7Eu6+CAhEsbDSe9OlyycZV66280xxOkTe6MLf4FaT+zCHsATwYRDFO+t7sQyByLtbemFf4zqHgpKuMSYrA1\\\\\\\/lk+UM9hEkUqFLc40pYebWblFeR1vDnYCk8bIn9bgosHEsLX\\\\\\\/afljPwK67KPN9v7TlemtRu\\\\\\\/8X8yxVz+DkRj\\\\\\\/aLA7ZarIlKByT3TDP016L50sETrZQ65RIN+DNfpS1M0Ktt0+mlML4c+LPCi1yTIGMHCr02sDXz2K6oNpfEfI34DYrpXbeGFbfeZALKztV9RIDJ6+QfOmZgBaDP8UwCRaznWgg5o+21bQElUHcKGJyfW7imFlIOb90MOAsbajasOXKEvjnrO+FNsiHLrQQo6V0X+bvXNuze5go7tuABFWtvPdSIBKqDhdRJmqwFlTJeNc4fo0ome8xVb\\\\\\\/SDdgBhgGzSgXb0gdQ47Byv3GbeCY50ke6XiZYwgjgkiMQkJpPjk2Ibvs99j3hqWejnQBy5Z6G6EzpckzPucERYt9DM2Z0Zwwrax68MPEwR9NhAJixLbyYUV+RFY+brMhvgcrVpTd+yUbFf+yMAnz1v6bVlySDZTm56WHU2cP1oLvK9h62MUxHyOsbx+PFTR8CqR0qpBk9YI+Tm1Vo8CrUpL6SvCQWTuhB0pr5a9qubdwvFKdVJizdlUUxRsQWgwa9ceEj57XBdvpjQ6AcMF1r9YR5Bzd9con\\\\\\\/O2V0Dp8hqoyu1OQsRG0ugIox0NvYeJyKFmc9Ubbh0iuZ3gkLxCDeVGYPy5\\\\\\\/UhnI9Kxan\\\\\\\/PjUB8JOSugy4LY3d4FSmnfm7stTRJ3go1xif15pGbooZrASF3ByR4b6hSZeW1C4cGJewWLBftYRWOGScAHxTmVPM5AwDmzyvO0CmRbyj2aPpckJ8ucjp0ws9fGdg\\\\\\\/Pccak1DgGAdM\\\\\\\/SPRAHPxU3\\\\\\\/nD78qL+vHmYDdwtZzZw0jVmGEKfME6SeHFd1IpifjiIJkfRuKNmpKhp03BHI8xoDjasOYSPIH9TRhEwn+EB6wosf\\\\\\\/7d1Dk6YFx2p76e14zPtMotsRYO5+oS2chxYvDuCsiVd4AUox7KiPaasrSocVAAOagCIHwSX7xO\\\\\\\/H4ShK7WCwDq0\\\\\\\/qcJLmedbNrHytnMB2RZoXWuj03VLrhuwr99nFBpvRQniojoD4CuZVQggxVKtw+\\\\\\\/O63rQq1KNxsdfzCSUSgUdN8PQzE+KuY7939W\\\\\\\/5WA67MpfblDEGnmS+UNgdSljAEKSSK8CawmL1QY8vmJ9wX0AWbE64aW98l9nv\\\\\\\/5EkdSdliFrBph02aXmFwA0dGVXps2x6vp7\\\\\\\/loAEFZd+8Ad9L61KDUs2v9ZV7vBoMbvU7brtV2LB2C1\\\\\\\/lY6sCyKaP4wTX\\\\\\\/pTJnLQr3eugvclFDtNe\\\\\\\/PPDqNjS\\\\\\\/mMNEJMmXBNx72EPtguTbpWhYEomIwugPR7u0R9aFfIuaSY3Po1uZKQ0D5cDSXj5LPcfVXYpHLCNSO6ZV4QjH03nAd\\\\\\\/lu1884gIp5uWJx71j6AVIVkqg8q2uWnggcM1GpFtzs4FcH2wcXvo8Cgy+iiyLG3KHL3j3LJn+F7gvqjNZmOua6abt4Y0SraiwfiasIUisVe4k5rRJ9frN5btt5cqQlrkyV1Sas7sIZ850toLzVkQzrifbfHQe\\\\\\\/dv46cw6xlAA0SToivoffIKAnFdj6m8sNTtvZlIvUZluMROIExrsAaRygI1EYMJJ4Cm7kooVg4fVlmr4\\\\\\\/iRukzACf2+EKiS58ZdYWrPZdF8HctoR64QdKW7S3RuJy1GcODE2Db7Ov317I5h4h0uuZDn1D+iAR73OS\\\\\\\/4H4KD3Bs6W6RnNWvnGthPyj\\\\\\\/qmxESyoYIWIBJsjvBR8WOmNzVsTRn+mMZaJbE4rQ\\\\\\\/QDp8j9ytRspWR6RAjMF7xnABp8L2+xBlcwcS7P7DfTBJXtavxXUwUCRqw9wdTxQziahkNu5GwTdgTBgaW\\\\\\\/sFTfMS0qEFiYJcSU4c8dRJVmi2yLhaDmeSdZT9sMOxjsp0AHa3M1407BY78lCnj+On+2wB+Y\\\\\\\/YWimgyBJU22FcZVifzKD8wGU1q7SlWi5HTpoC7LoYp4KCgfW\\\\\\\/iFc3UI5rFrrDlo5DS0l4YYo+zBa76i05Ys20O4NYJZbaaQeozKZYfBxtMxiR9\\\\\\\/I+9qn3+fTaznuiwd5oz68qSXopeIBrgeolcpbVJ6mx469z2ndwk3v0bPjMNZrSNWXpwh8p\\\\\\\/ypwxIh1OBEZJhvJLmGZ9GMDIt90o\\\\\\\/U1ckpMw4eL++qZ8wYB2whbRE\\\\\\\/1cfl94O1gUT3WD0u1p5fGoYp9M4YUPiJ7VP3\\\\\\\/BvvAr7Hmt8ebQqt6pnkg3gt+T+DhAuTxM0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022bGGSoiAlQRiAJm8STYz4iw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ae16db38faaba7d559fc44e66ba86cfeb7821517af5e8406c33dbe90ddc37669\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Vvsu8RVRN4TRoy2b7Qf+XjOfFAQ+olNDRcTdI46zY4rvuazJ5GJcI+6hT4hEuoty+mJY29fi\\\\\\\/VMtTxBaB6BZ4btmx7W+qGntkJBb2P6mCSaRHRQiy1hXD8CNkGwl9pUdpAS6qtXDBxtUzC9Ac\\\\\\\/BUqESfB2gycYBCVbZpM4vz6yeW68HKkeDXReQHJJhE2WJSKbDWf3LfgiZidUFlXxZJoWvlOh\\\\\\\/kbfRFPtr2a\\\\\\\/ube3XT3CYJjVJGgbtQIg6Ug4C7m8eMkpEFhUc23H508cPXPfToINKFBAJb91d1cP6alc6xJB\\\\\\\/x1mZaQkMGQeSOQQASTQ42YT5tuqwmy2LsJpGLxjmYlZHWBcStWWM3f7\\\\\\\/lHdxC\\\\\\\/EFx4Awx43wKajnSImYUOhd8eucY0cWwg5XltbPBaQ\\\\\\\/kSTTukqgQjKbzVFoasPtwrdqBAEHRWL0QXltVwslmDDrcITB9DrDyigfUvpafDDCzGxO1SfyNNkmmxSm3N2PFTgktj2zOOew0WaKB6nGGzq6zNiPOtCaNGF3feSrIRyAtuscCzZYtp+JAR5JPcnoU2fyMxmZOVGqv\\\\\\\/c9OMn39qeUK6HonEH1CcFAE0fKm2iSxf4\\\\\\\/WSCCbpiRdgyCX+p\\\\\\\/qfTAgmWj2e+g322cPqEQdDU\\\\\\\/945aLKHURTWLfO6+NwpnxbXEINApZxNz8ikvdBvtbc7B0R24i6NfWkLiTdaSc0h0E+x602DUYQrCIcPP2ErWqC7GU126jsc6sPkFc60L1r9dVP3ptmx5LO3y2+kVBv51XJw6RB\\\\\\\/8VbIhmke5VeIFGcVHTUgO49vi6HQ\\\\\\\/MbOKAo63Nf\\\\\\\/Jka5AfzaZFW98TdYAGRJzMyYZRjDmq\\\\\\\/VnkeErOambPuMdBE0TfXRoS4pZqMF722oN6RvUrL\\\\\\\/UrEqIu\\\\\\\/gzgfPu9Q0d+UsUIXHuwTbRRJTbg+dYlGZnOcIX1mQUWKw84mcEb4g4akXl1XVPGhW\\\\\\\/Hwh2frUlLJjOB9Z035NwBGy2DMnaJVwDa2y2xoqwofw9uZ3d6oSTDAp1Nl8IKxMLA76gQWeP+hEypb5WLyOZSpDgqsRA5nUHVuSn9ekf5an9s+DsLUgkKgTtDZ+ccFrHQVt07WRUWtogTZQibfYrbRMACL0zNT6lWjfxq0HmMpbKW7BWtwPEk2WYCMvt5muLkZ8kM\\\\\\\/\\\\\\\/ojx4MwHlbEqjtW5H94A5hZuKHnPPj6trvdFPqrA3pAw3jV6Msl07n26vnLTyf+dMOW+zdogHUQV9pVnSfS+RV2AJ3lXhHkm2Q\\\\\\\/n3cstB8qeX385I2L3SOhu5ApYAqqrRhVInegMAbj3jvjavVIz8pDAw4W+eF82GmUty\\\\\\\/xmYsOoKNcphrAWkpbifpIS82c6BZ+1Kwy6jpw5xZHriSclDCVoQuFzF3XX+Hh\\\\\\\/uumzJhhGhmczVDsUdPfc3IFhelUm5THwJboMOKJs0dGT4ebTfQjWJDt64+R+u\\\\\\\/NMmgiBOieTR+DwubdL780OoLVaf0ps5k7SM6kpQZj8NBflRjgLU1sSebGaG3jfluBRqp4mEibU0P15xNXm1j1E5nwXoK20JzaizoVvKP+P+pVrk7nRoaoD8dW6SUb6dhMbTCIP2jYKXOBAmmLGYCYzytGGxynC1CTMfq5bQxK6bxLV5tJ1r5UYve02009mJyTRt93vscxIcMTyWv2F9IHmx\\\\\\\/nZ5NKxpd5opZbUBUvvbNpZy0eZSjS0BxEwzmlt+Sd0FAgOpZXu2oHFymvpA\\\\\\\/dtGGJanK1aV+TXTpbakNMbQP3rFokYwcjI3v5Ua1\\\\\\\/TIVqgV8497+VbUDi2KvuloFXPI\\\\\\\/3jxYkFhRTZwy3uq7jRDzpguqhDAG533vM55RNyPwH2\\\\\\\/FOmXgCRJQq\\\\\\\/6X\\\\\\\/HFGAvZ1\\\\\\\/ve2+S9HjEWGqn\\\\\\\/1XGykFb6s3uJ2O+TIECVTaRLA6nf7kbetyqYvP9RNHB+lExLV5jZi9CowR2vNrKO1qR9IYXPOyuak7hGHhvJxa9G\\\\\\\/vUok0fyc0vFQetjd4Y1el0Ab9\\\\\\\/K7CivO885LluWbJh3cmMrggcrfRr8eeFNSdqEZUfhu4BCFwnwi12sZXebVR9BaQik6Nh4mYYBg8JCc3O0WuhxNpDa+psrBbcKgdD8lLOFyrBF8vxGdcwCWoPTUO\\\\\\\/gW8WO0wZDrBwZW8IRRDOA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022G3DyuRjPFW8kK7dAwKpPxQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f9df5cf5fdb38a37310aa74c1f3b18dfb4ba0a4de0395da00f3d4b258c8cf419\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487097 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3637" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2SPGI5hSX5j69CV08F1TgWvdmhagVeCC4rPce_0essU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3637" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FubLRj5G6XxMz3SEtNchbMKKRRxysc8QX66vJxX20As" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3621" title="Genome-Wide Analysis of DNA Methylation During Ovule Development of Female-Sterile Rice fsv1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3649" title="Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487097" id="node487097" data-pisa="ggg;7/11/3637" data-pisa-master="ggg;g3.117.300169" data-apath="/ggg/7/11/3637.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Diagnostic Resource for <em>Ceratitis capitata</em> Strain Identification Based on QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0914-6914" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sheina B. Sim</span>, <span class="highwire-citation-author" data-delta="1">Raul Ruiz-Arce</span>, <span class="highwire-citation-author" data-delta="2">Norman B. Barr</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9511-5139" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott M. Geib</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3637-3647;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300169 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487097" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sheina B. Sim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheina%2BB.%2BSim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sim%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASheina%2BB.%2BSim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0914-6914" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sheina B. Sim</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Raul Ruiz-Arce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raul%2BRuiz-Arce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruiz-Arce%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaul%2BRuiz-Arce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Norman B. Barr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norman%2BB.%2BBarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorman%2BB.%2BBarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Scott M. Geib </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BM.%2BGeib%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geib%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BM.%2BGeib%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9511-5139" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott M. Geib</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:scott.geib@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">scott.geib@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3637" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487097" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487097/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3637.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487097" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487097/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3637.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487097" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487097/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3637.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487097" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487097/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3637.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487097" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487097/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The Mediterranean fruit fly <em>Ceratitis capitata</em> (Wiedemann) is a destructive agricultural pest and the subject of exclusion efforts in many countries. Suppression and eradication of invasive populations to prevent its establishment is facilitated by the release of sterile males using the sterile insect technique (SIT). In SIT release areas, it is critical to accurately discriminate between released sterile males and wild individuals to detect extremely rare invasive individuals in areas inundated with millions of sterile male flies. Current methods for discrimination exist but are not always definitive, and a more reliable method is necessary. To address this, we developed a genotyping assay that can be used to discriminate between sterile males from the SIT strain and wild individuals. This was achieved by identifying single nucleotide polymorphisms (SNPs) linked to the maintained traits that facilitate male-only releases, white pupae (<em>wp</em>) and temperature-sensitive lethal (<em>tsl</em>), via QTL mapping. This resulted in the identification of one SNP that was in near-perfect linkage disequilibrium between genotype at this locus and the pupal color phenotype. Medfly from many SIT colonies and wild individuals from across its geographic range were genotyped for this locus, and results show its consistency in identifying SIT flies. In addition, linkage and QTL mapping of <em>wp</em> and <em>tsl</em> have larger impacts as they can serve as foundational tools to identify the genetic basis of traits that facilitate the separation of males from female flies, which can be used to develop SIT programs in related species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-mapping" class="hw-term hw-article-keyword hw-article-keyword-linkage-mapping" rel="nofollow">linkage mapping</a></li><li class="kwd"><a href="/keyword/genome-wide-genotyping" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-genotyping" rel="nofollow">genome-wide genotyping</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/medfly" class="hw-term hw-article-keyword hw-article-keyword-medfly" rel="nofollow">medfly</a></li><li class="kwd"><a href="/keyword/sterile-insect-technique" class="hw-term hw-article-keyword hw-article-keyword-sterile-insect-technique" rel="nofollow">sterile insect technique</a></li><li class="kwd"><a href="/keyword/diagnostics" class="hw-term hw-article-keyword hw-article-keyword-diagnostics" rel="nofollow">diagnostics</a></li></ul><p id="p-3">The Mediterranean fruit fly (medfly), <em>Ceratitis capitata</em>, is an economically important agricultural pest that has the potential to inflict costly damage on a broad range of fruit and vegetable crops around the world and affect the multibillion-dollar U.S. specialty crop industry through the prohibition of fruit and vegetable exports from medfly established areas (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Enkerlin 2005</a>). One component of control programs used to prevent its establishment and contribute to its eradication in newly invaded agriculturally rich regions is the sterile insect technique (SIT). This technique operates on the premise that by inundating small invasive populations with sterile males, reproduction in the wild population will be suppressed and the population will be reduced and eventually eradicated (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Knipling 1955</a>). This process requires a mass-reared population in which sterility can be induced and subsequent sterile individuals can be released in large quantities. A classic example of this methodology was in the eradication of the New World screwworm from the United States in the mid-twentieth century (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wyss 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Krafsur 1998</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Myers <em>et al.</em> 1998</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Krafsur <em>et al.</em> 1987</a>), and most recently, the release of SIT flies to control a new introduction of <em>C. capitata</em> in the Dominican Republic on March 2015, resulting in its swift eradication in the majority of the affected areas within 10 months (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gil 2017</a>). Currently, SIT is employed to maintain the continental United States free from tephritid fruit fly pests such the medfly and Mexican fruit fly <em>Anastrepha ludens</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Franz 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Klassen 2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Klassen and Curtis 2005</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hendrichs <em>et al.</em> 1995</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">USDA-APHIS 2010</a>). The technique is most effective if a genetic sexing strain (GSS) is available for the target species, as a GSS facilitates the cost-effective and rapid separation of female and male flies before sterilization, resulting in male-only releases. Male-only releases are preferred when using SIT because sterile females sting host fruits in an attempt to oviposit (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hendrichs <em>et al.</em> 1995</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andrewartha and Birch 1960</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rossler 1979</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Whitten 1969</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zepeda-Cisneros <em>et al.</em> 2014</a>), causing unnecessary damage and potential infection of phytopathogenic bacteria and fruit rot fungi to crops. In addition, sterile females lower the efficacy of sterile males in successfully mating with wild females through mating competition (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Hendrichs <em>et al.</em> 1995</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rendon <em>et al.</em> 2004</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Zepeda-Cisneros <em>et al.</em> 2014</a>).</p><p id="p-4">In medfly, the Vienna strains (<em>e.g.</em>, Vienna-7, Vienna-8, and Vienna-8 D53-) are GSSs used for mass-release in SIT applications, particularly in Southern California and Central America. In Vienna strains, there are two sexually dimorphic and tightly linked traits: white pupae (<em>wp</em>) and temperature sensitive lethal (<em>tsl)</em>. Female Vienna medfly have a mutant white pupae phenotype, in contrast to males, which have a wild-type, brown pupae phenotype. The white pupae phenotype was first found in the fifth filial generation of an inbred colony that originated from a cross between an irradiated male and a wild-type female (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Rossler 1979</a>). In addition to being homozygous for <em>wp</em>, female Vienna medfly are also homozygous for a temperature-sensitive lethal mutation, a mutation originally induced into colony flies through the feeding of ethyl methanesulphonate (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Busch-Petersen 1990</a>). Homozygosity for <em>tsl</em> causes females to die at temperatures &gt; 33°, which is in contrast to males who are heterozygous at this locus and thus survive at elevated temperatures (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kerremans and Franz 1995</a>). When eggs produced by the Vienna lines undergo a high-temperature treatment (<em>e.g.</em>, incubation in an aerated water bath at 34° for 24 hr), the hatching larvae are essentially all male, and any surviving females can be identified at the pupal stage and excluded based on their white pupal color. Recombinants (brown pupae and temperature-sensitive individuals or white pupae nontemperature-sensitive individuals) are rare due to the tight linkage between the genes that govern these two traits and lowered recombination in males (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Morgan 1914</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rossler 1982</a>). Due to the artificial conditions under which these mutations and phenotypes were created, the likelihood of the mutations causing white pupae and temperature-sensitivity occurring in wild populations is very low.</p><p id="p-5">Although these two traits are autosomal recessive and should not be displayed in a sex-specific manner, sex linkage is maintained through a reciprocal translocation between the autosome carrying the two causative genes (<em>wp</em> and <em>tsl</em>, which are on chromosome 5) and the Y sex chromosome in males (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Franz 2005</a>). Vienna-7 and Vienna-8 flies differ in the location of the translocation breakpoint, and Vienna-8 flies contain a chromosomal inversion that places the gene causing <em>tsl</em> in closer physical proximity to the gene causing <em>wp</em>; this is in contrast to Vienna-8 D53- which lacks this inversion. The reciprocal translocation in Vienna flies, coupled with low recombination in males, preserves all males in a wild-type heterozygous state, and all females as homozygous recessive (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Franz <em>et al.</em> 1994</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Schematic of the sex chromosomes and the fifth chromosome containing the wp and tsl gene for female and male Vienna-8 GSS (genetic sexing strain) and HiMed C. capitata. The location of the white pupae and temperature-sensitive lethal gene has been crudely mapped to the tip of the fifth chromosome, and a translocation between the Y-chromosome and the fifth chromosome harbors a wild-type allele (wp+) in males and is heterozygous at the locus, in contrast to females who are homozygous for the wp and tsl mutations. (B) Drawing of the sex chromosomes and the fifth chromosome containing the genes for female and male wild-type C. capitata white pupae and wild-type temperature-sensitive lethal; wild-type individuals lack the wp and tsl variants. This figure is adapted from Franz (2005)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2049114201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Schematic of the sex chromosomes and the fifth chromosome containing the &lt;em&gt;wp&lt;/em&gt; and &lt;em&gt;tsl&lt;/em&gt; gene for female and male Vienna-8 GSS (genetic sexing strain) and HiMed &lt;em&gt;C. capitata&lt;/em&gt;. The location of the &lt;em&gt;white pupae&lt;/em&gt; and &lt;em&gt;temperature-sensitive lethal&lt;/em&gt; gene has been crudely mapped to the tip of the fifth chromosome, and a translocation between the Y-chromosome and the fifth chromosome harbors a wild-type allele (&lt;em&gt;wp&lt;/em&gt;&lt;sup&gt;+&lt;/sup&gt;) in males and is heterozygous at the locus, in contrast to females who are homozygous for the &lt;em&gt;wp&lt;/em&gt; and &lt;em&gt;tsl&lt;/em&gt; mutations. (B) Drawing of the sex chromosomes and the fifth chromosome containing the genes for female and male wild-type &lt;em&gt;C. capitata white pupae&lt;/em&gt; and wild-type &lt;em&gt;temperature-sensitive lethal&lt;/em&gt;; wild-type individuals lack the &lt;em&gt;wp&lt;/em&gt; and &lt;em&gt;tsl&lt;/em&gt; variants. This figure is adapted from Franz (2005).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3637/F1.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3637/F1.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487261" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">(A) Schematic of the sex chromosomes and the fifth chromosome containing the <em>wp</em> and <em>tsl</em> gene for female and male Vienna-8 GSS (genetic sexing strain) and HiMed <em>C. capitata</em>. The location of the <em>white pupae</em> and <em>temperature-sensitive lethal</em> gene has been crudely mapped to the tip of the fifth chromosome, and a translocation between the Y-chromosome and the fifth chromosome harbors a wild-type allele (<em>wp</em><sup>+</sup>) in males and is heterozygous at the locus, in contrast to females who are homozygous for the <em>wp</em> and <em>tsl</em> mutations. (B) Drawing of the sex chromosomes and the fifth chromosome containing the genes for female and male wild-type <em>C. capitata white pupae</em> and wild-type <em>temperature-sensitive lethal</em>; wild-type individuals lack the <em>wp</em> and <em>tsl</em> variants. This figure is adapted from <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Franz (2005)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Mass-produced sterilized males are released into areas at high risk of medfly invasion, and this practice is one of several components of an integrated pest management (IPM) program. An additional component of IPM is population monitoring, which is achieved by hanging a high density of yellow sticky card insect traps in high-risk areas (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Klassen 2005</a>). Due to the volume of traps that require attention, it is not practical to check traps daily, and the insects that are caught are subject to predation and weathering.</p><p id="p-8">In areas with an active SIT program, such as Southern California, several hundred million SIT flies are released each week on a continuous basis, and intensive trapping in these areas results in a recapture rate of 0.02–0.07%, depending on trapping density (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">USDA-APHIS 2014</a>). This amounts to 1–2 million flies that are recaptured yearly that must be differentiated from wild medfly captures, which may only be present at very low population levels. This is, in part, achieved through the marking of radiation-sterilized pupae with a powdered fluorescent dye, a marking method developed by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Norris (1957)</a> and subsequently modified for use in SIT programs by <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Steiner (1965)</a> and <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schroeder <em>et al.</em> (1972)</a> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Enkerlin <em>et al.</em> 1996</a>). Although the dye is eventually removed from the insect body through grooming, contact of the cephalic ptilinum sac with the dye upon adult emergence results in integration of the dye into the head of the fly when the sac is retrieved back into the head (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Enkerlin <em>et al.</em> 1996</a>). Because of the localization of the dye in the head, fluorescent marking can be unreliable upon the recapture of partial (headless) or weathered flies.</p><p id="p-9">Another method used by regulatory agencies to identify trapped material involves the dissection of gonadal tissue. This requires the examination of captured flies by a trained specialist to determine if its condition is consistent with that of irradiated flies (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guillén Aguilar 1983</a>). Unfortunately, this is a process that can become costly and time consuming if the number of captured flies are numerous. Though these two methods are useful in screening flies, additional techniques are needed to diagnose flies that are in poor condition and lack the diagnostic marking with dye.</p><p id="p-10">In an effort to address this need, molecular methods have been developed. However, these various methods have all relied upon mitochondrial haplotypes that are associated with Vienna flies (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">San Andres <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barr 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Parubrub <em>et al.</em> 2015</a>). For example, an assay based on a protocol reported by <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barr (2009)</a> requires sequencing of a 584 bp region of the mitochondria, which includes subunits 5 and 4 of the NADH gene. Examination of various Vienna strain flies from SIT colonies at multiple mass-rearing facilities showed that all reared Vienna flies exclusively have one of two haplotypes for the mitochondrial N5N4 locus; the M006 haplotype is found in the Vienna-8 strain reared by the Moscamed facility in El Piño, Guatemala, and both the M006 and M007 haplotypes are found in the Vienna-8 D53- strain reared by the California Department of Food and Agriculture (CDFA) in Waimanalo, HI. This method can diagnose a captured fly as a wild fly (<em>i.e.</em>, not a Vienna genetic sexing strain) if the unknown sample has any haplotype apart from M006 and M007 at the N5N4 locus. However, the low but detectable presence of the M006 and M007 haplotypes in the wild (R. Ruiz-Arce, unpublished data) prohibits the unequivocal designation of a captured fly as an Vienna strain fly. This limitation in diagnosis is also a problem for the other mitochondrial methods because the diagnostic genotypes reported are not private to Vienna strain colonies.</p><p id="p-11">The purpose of this project was to design a diagnostic assay that can be used in addition to the N5N4 locus to discriminate between wild and Vienna strain flies. To achieve this goal, linkage and QTL analysis was performed to identify a locus that is tightly linked to traits maintained in Vienna strain colonies. This resulted in the development of a single nucleotide polymorphism (SNP) genotyping assay that discriminates between individuals based on the white pupae phenotype, a phenotype with artificial origins, which increases its likelihood of being exclusive to the Vienna strain.</p><p id="p-12">In addition to working toward the development of a diagnostic assay to support the discrimination of wild flies from released individuals, the results of this study will be used as a foundation that can direct future genetic research on the medfly and related species. The identification of the loci closely linked to <em>wp</em> can be used as a starting point for the future identification the genes governing <em>wp</em> and <em>tsl</em>, the causative mutations underlying the mutations, and targets for recreating these phenotypes in new species to rapidly create genetic sexing strains in other important insect pests. In addition, this map can be applied to a recently published medfly genome (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Papanicolaou <em>et al.</em> 2016</a>) to further improve this assembly and place scaffold regions and genes into a chromosomal context.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Test crosses and fly husbandry</h3><p id="p-13">The GSS flies used to perform the genetic test crosses were Vienna-8 D53- strain flies obtained from the CDFA mass-rearing facility in Waimanalo, HI. Wild-type individuals were from a colony called “HiMed” that are reared and maintained at the United States Department of Agriculture-Agricultural Research Service (USDA-ARS) facility in Hilo, HI. Individual flies from the Vienna-8 D53- and HiMed lines were used as parents to create three sibships representing mapping populations. The crossing scheme employed (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), produced recombinant offspring with nonsex-linked variation in pupal color (white or wild-type brown) in an inbred, mostly wild-type background. To generate the mapping population to map <em>white pupae</em>, virgin Parental (P) generation Vienna-8 D53- white pupae females were mated with virgin wild-type HiMed males in isolated crosses. The resulting F1 population were wild-type and heterozygous for <em>wp</em>, and were intercrossed to recover the white pupae phenotype. The resulting white pupae F2 females were backcrossed to the wild-type HiMed colony males in isolated matings. The resulting F3s, which were heterozygous at the <em>wp</em> locus, were intercrossed to recover the white phenotype. This method of generating a mapping population decoupled the white pupae phenotype from sex due to the exclusion of the reciprocal translocated autosome and Y chromosome through usage of wild-type colony males, which have a wild-type chromosomal arrangement. This crossing scheme was replicated and produced many sibships, with the three most productive sibships used for genotyping experiments.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Virgin adult females from the C. capitata genetic sexing strain (Vienna-8 D53-) were mated in isolation with males from the wild-type laboratory colony (HiMed). The white pupae trait (wp) is autosomal recessive, so resulting F1 progeny will all have a wild-type brown pupal color phenotype. In F2 progeny from isolated intercrossing between F1 full-siblings, the pupal color phenotype will segregate at a 3:1 ratio of wild-type brown pupae to white pupae. White pupae F2 females were back-crossed to wild-type laboratory colony males. This increases the proportion of the wild-type alleles genome in subsequent offspring. Like the F1 progeny, the F3 progeny will all have a wild-type brown pupal color phenotype and full-sibs will be intercrossed to produce an F4 mapping population comprised of female and male wild-type brown pupae and white pupae individuals. Colors of the text boxes denote pupal color of individuals, white boxes represent white pupae individuals, and yellow boxes represent brown pupae individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2049114201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Virgin adult females from the &lt;em&gt;C. capitata&lt;/em&gt; genetic sexing strain (Vienna-8 D53-) were mated in isolation with males from the wild-type laboratory colony (HiMed). The &lt;em&gt;white pupae&lt;/em&gt; trait (wp) is autosomal recessive, so resulting F1 progeny will all have a wild-type brown pupal color phenotype. In F2 progeny from isolated intercrossing between F1 full-siblings, the pupal color phenotype will segregate at a 3:1 ratio of wild-type brown pupae to white pupae. White pupae F2 females were back-crossed to wild-type laboratory colony males. This increases the proportion of the wild-type alleles genome in subsequent offspring. Like the F1 progeny, the F3 progeny will all have a wild-type brown pupal color phenotype and full-sibs will be intercrossed to produce an F4 mapping population comprised of female and male wild-type brown pupae and white pupae individuals. Colors of the text boxes denote pupal color of individuals, white boxes represent white pupae individuals, and yellow boxes represent brown pupae individuals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3637/F2.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3637/F2.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487251" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Virgin adult females from the <em>C. capitata</em> genetic sexing strain (Vienna-8 D53-) were mated in isolation with males from the wild-type laboratory colony (HiMed). The <em>white pupae</em> trait (wp) is autosomal recessive, so resulting F1 progeny will all have a wild-type brown pupal color phenotype. In F2 progeny from isolated intercrossing between F1 full-siblings, the pupal color phenotype will segregate at a 3:1 ratio of wild-type brown pupae to white pupae. White pupae F2 females were back-crossed to wild-type laboratory colony males. This increases the proportion of the wild-type alleles genome in subsequent offspring. Like the F1 progeny, the F3 progeny will all have a wild-type brown pupal color phenotype and full-sibs will be intercrossed to produce an F4 mapping population comprised of female and male wild-type brown pupae and white pupae individuals. Colors of the text boxes denote pupal color of individuals, white boxes represent white pupae individuals, and yellow boxes represent brown pupae individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The adult flies used in this experiment were reared at the USDA-ARS under conditions similar to those described by <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Vargas (1989)</a>, but with minor amendments. Adult <em>C. capitata</em> were given a diet of a 3:1 ratio of white sugar and yeast hydrolysate and reared in 350 ml screen-covered cups. Each cup contained isolated male and female pairs, or male only or female only cohorts, from single sibships. Larvae were given a diet consisting of 26.3% wheat mill feed, 12.2% granulated white sugar, 3.4% torula yeast, 2.3% citric acid, 0.2% nipagen (preservative), 0.2% sodium benzoate (preservative), and 59.5% water. Individuals of all life stages were held in a room maintained at 25° with 65% humidity.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and genotype by sequencing library production</h3><p id="p-16">Individual F4 flies from the three most productive lines, along with their parents (from the F3 generation), F2 source matings, and the original parental (P generation) individuals were used for genotype by sequencing (GBS) library preparation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>). DNA extraction was performed on individual adult flies. Whole flies were homogenized in tissue lysis buffer using a FastPrep 24 homogenizer (MP Biomedical, Santa Ana, CA) for 20 sec at 4.0 m/sec. Homogenized samples were incubated in a 55° water bath for 3 hr, followed by DNA extraction on a Kingfisher Flex 96 automated extraction instrument (Thermo Scientific, Waltham, MA), using standard protocols with a Mag-Bind Tissue DNA KF Kit (Omega, BioTek, Norcross, GA). In addition to samples derived from the crossing scheme, an additional 169 samples were included representing flies derived from major <em>C. capitata</em> mass rearing colonies and a broad geographic distribution of wild flies (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1/TableS1.pdf">Table S1</a>). The quantity and quality of the extracted DNA samples were determined using the High Sensitivity Genomic DNA Analysis Kit on a Fragment Analyzer (Advanced Analytical, Ankeny, IA).</p><p id="p-17">DNA extracted from all individuals was assessed for quantity and integrity, and high-quality samples were sent for library preparation at the Institute for Diversity GBS facility at Cornell University, Ithaca, NY. Library preparation was performed by ligating barcodes unique within a plate of samples to genomic DNA digested with the restriction enzyme <em>Ape</em>KI. Illumina HiSeq 2500 was used to generate single-end 102 bp reads with in-line barcodes included in raw sequences. Raw sequences from the mapping population are available on NCBI, associated with BioProject PRJNA341700 and BioSamples SAMN05725882 to SAMN05725907 and SAMN05727687 to SAMN05727802. Raw sequences from samples from the wild populations and Vienna fly colonies are also available and associated with BioProject PRJNA341535 and BioSamples to SAMN05725834 to SAMN05725997 and SAMN05726002 to SAMN05726006. The BioSample accessions associated with the sequences from each individual are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-4" class="subsection"><h3>SNP discovery</h3><p id="p-18">Raw GBS reads for each sample were individually aligned to a <em>C. capitata</em> whole-genome assembly (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Papanicolaou <em>et al.</em> 2016</a>), made available through the 5000 Insect Genome Project (I5K) (<a href="https://i5k.nal.usda.gov/">https://i5k.nal.usda.gov/</a>, NCBI WGS accession AOHK00000000.1, BioProject accession PRJNA201381). GBS tags were aligned to the reference scaffold assembly using the Burrows–Wheeler Alignment Tool (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Durbin 2009</a>). SNP discovery and genotyping was performed using the <em>TASSEL</em> pipeline V3.0 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Glaubitz <em>et al.</em> 2014</a>). The plugins <em>FastqToTagCountPlugin</em>, <em>MergeMultipleTagCountPlugin</em>, <em>TagCountToFastqPlugin</em>, <em>SAMConverterPlugin</em>, <em>FastqToTBTPlugin</em>, <em>MergeTagsByTaxaFilesPlugin</em>, <em>TagsToSNPByAlignmentPlugin</em>, <em>MergeDuplicateSNPsPlugin</em>, and <em>GBSHapMapFiltersPlugin</em> were used to demultiplex GBS sequences, align fragments, identify SNPs, assign genotypes for every locus for each individual, and filter genotypes based on read depth and a minimum minor allele frequency. Default parameters were used, and SNP genotype filtering was based on a minimum minor allele frequency of 0.05, no linkage disequilibrium with at least one neighboring SNP, low taxon coverage, and high heterozygosity. Further filtering was performed using <em>VCFtools</em> v0.1.13 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Danecek <em>et al.</em> 2011</a>), and SNP genotypes were included in the analysis on the condition that there was ≤ 50% missing data after excluding individuals with low sequence coverage. Initial assessment of differentiation between the two colonies used in the mapping populations, HiMed (<em>N</em> = 6 females and 6 males) and Vienna-8 D53- (<em>N</em> = 12 males), was performed by estimating Weir and Cockerham’s Fst using <em>VCFtools</em> v0.1.13 (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Danecek <em>et al.</em> 2011</a>) for all identified SNP loci and a Manhattan plot of Fst values was generated using the R package <em>qqman</em> (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Turner 2014</a>).</p></div><div id="sec-5" class="subsection"><h3>Population genetic analysis of SNP genotypes from GBS library</h3><p id="p-19">After excluding individuals and SNP loci that contained &gt; 20% missing data, a population genetic analysis on 143 individuals from several sources was performed using their genotypes for 77,844 SNP loci. These 143 individuals were from several mass-rearing Vienna fly colonies, the wild-type colony HiMed, wild flies collected from regions where there are no current Vienna fly strain releases (the Azores, Brazil, Hawaii, Honduras, Morocco, Mozambique, Panama, Reunion Island, and Senegal), and flies collected from regions with active SIT, where Vienna fly strain are released (Australia, California, Guatemala, and Spain). A discriminant analysis of principle components (DAPC), which clusters individuals in a data set to maximize the variation between clusters and minimizes variation within clusters, was performed using the R package <em>adegenet</em> v.2.0.2 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jombart 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jombart and Ahmed 2011</a>). The number of genetic clusters that describe these 143 individuals was identified using the function <em>find.clusters</em> and assignment of each individual to a cluster was performed by the <em>dapc</em> function using 20 principle components and six discriminant functions.</p></div><div id="sec-6" class="subsection"><h3>Linkage and QTL mapping</h3><p id="p-20">The SNP genotypes obtained from the <em>TASSEL</em> pipeline were used to estimate a linkage map featuring chromosome-scale linkage groups. This analysis was performed using the recombination mapping algorithms of <em>LEP-Map2</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rastas <em>et al.</em> 2013</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">2015</a>), which simultaneously processes large amounts of SNP genotype data for several mapping populations. It was written specifically for use with genome-wide SNP data and organisms with achiasmatic meiosis, such as insects in the orders Lepidoptera and Diptera. Custom python scripts were used to modify existing SNP data in <em>.hmp</em> format into a <em>LEP-Map2</em>-compatible format. Using <em>Lep-MAP2</em>, genome-wide SNP genotypes from all parents and progeny from three mapping populations were filtered for Mendelian errors using custom python scripts and the <em>Filtering</em> module. Loci were then assigned to linkage groups using the <em>SeparateChromosomes</em> module. A minimum of 10 loci was required for a linkage group to be reported. Ungrouped SNP loci were included using the <em>JoinSingles</em> module, and the <em>OrderMarkers</em> module was used to order the markers within each linkage group defined by <em>SeparateChromosomes</em>. Due to the lowered and nearly absent recombination in male Diptera (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Morgan 1914</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Rossler 1982</a>), the initial recombination rate for males was set to <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-1.gif"/></noscript></span></span> in the <em>OrderMarkers</em> module, while for females it was set to 0.05.</p><p id="p-21">The outputs of <em>LEP-Map2</em> were modified with Python utilities to create files compatible with the <em>qtl</em> package in <em>R</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2003</a>) and visualized using the <em>LinkageMapView</em> package, also in <em>R</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ouellette <em>et al.</em> 2017</a>). Loci linked to the <em>wp</em> locus were identified through implementation of the single-locus binary QTL model, a model that is most appropriate for the <em>wp</em> trait and is described by <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Xu and Atchley (1996)</a>. Following this, a permutation test using 100,000 permutations was performed according to described methods to identify significant loci above the threshold of <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-2.gif"/></noscript></span></span> (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Broman 2003</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2003</a>). This shows the minimum logarithm of the odds (LOD) score a locus can have and still be considered significantly linked to the trait. The SNP loci displaying the strongest linkage to the <em>wp</em> phenotype as determined by a permutation test was used to design TaqMan SNP genotyping assays to differentiate between white pupae and wild-type brown pupae individuals.</p><p id="p-22">Population genetic differentiation analysis was performed between Vienna strain individuals from around the world and wild flies. To do so, Weir and Cockerham’s Fst was estimated between Vienna strain individuals (<em>N</em> = 63) collected from multiple mass-rearing facilities and flies collected from the wild (<em>N</em> = 88) in the established range of the medfly. For this analysis, only SNP loci in the linkage map were included to be able to identify which regions of the genome are highly diverged between Vienna strain flies and wild flies.</p></div><div id="sec-7" class="subsection"><h3>TaqMan genotyping assay design</h3><p id="p-23">The SNP locus displaying the strongest linkage to the <em>wp</em> phenotype as determined by a permutation test was located on scaffold NW_004523946.1 at base position 1,353,742, on the superscaffold that corresponds to chromosome 5. To confirm the validity of this analysis, a TaqMan SNP genotyping assay (Thermo Scientific) was designed around this marker. The assay, designed and identified as AHMSY8D by Thermo Scientific, was designed with the forward primer sequence 5′-GGAAAGGAAATAAGGTGGAGTAACCA-3′, reverse primer sequence 5′-GGTGTTGAAGTGTTGACTAGACATG-3′, wild-type allele reporter (VIC) sequence 5′-CAGCGCAGCACCGG-3′, and mutant allele reporter (FAM) sequence 5′-CAGCGCATCACCGG-3′. Based on SNP genotypes determined from GBS libraries for the mapping population, Vienna strain individuals, and wild individuals, the wild-type allele is the nucleotide G at the SNP locus in contrast to the nucleotide T, which is only present in Vienna strain individuals with Vienna strain females being homozygous for T and Vienna strain males being heterozygous.</p></div><div id="sec-8" class="subsection"><h3>Genotype confirmation of assay on Vienna colony flies and wild-derived flies</h3><p id="p-24">The initial genotype confirmation tests for TaqMan custom assay AHMSY8D took place at the USDA-ARS Daniel K. Inouye US PBARC laboratory in Hilo, HI. Amplifications were performed in triplicate in volumes of 10.0 μl containing 5.0 μl SensiFAST Hi-ROX Genotyping Kit (Bioline, London, UK), 0.5 μl 20× TaqMan SNP assay, 2.0 μl template DNA, and 2.5 μl DNAse-free <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-3.gif"/></noscript></span></span> The qPCR thermal cycling was performed on a StepOnePlus Real-Time PCR system (Thermo Scientific) using a fast protocol under the following cycling conditions: initial pre-PCR read stage at 25.0° for 30 sec; 95.0° for 20 sec; 40 cycles of 95.0° for 3 sec and 60° for 20 sec; and a final post-PCR read stage at 25.0° for 30 sec. Data analysis on the change <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-4.gif"/></noscript></span></span> in fluorescence for both reporters relative to the passive dye and assignment of genotype was performed on the StepOne Software v2.3 (Thermo Scientific). A total of 43 individuals from the Vienna-7 colony (<em>N</em> = 21) and the HiMed colony (<em>N</em> = 24) were genotyped. Additionally, three of the wild captures from Spain that clustered with mass-rearing Vienna strains in the DAPC analysis were genotyped to determine consistency between GBS and the TaqMan assay AHMSY8D in showing that genotypes observed in those flies were consistent with Vienna strain genotypes.</p></div><div id="sec-9" class="subsection"><h3>Cross-platform genotype confirmation</h3><p id="p-25">Additional genotype confirmation tests of the TaqMan assay AHMSY8D was performed on Vienna strain individuals collected from multiple mass-rearing facilities to demonstrate that the assay is consistent among the different Vienna colonies and that the assay is compatible across different qPCR platforms. At the USDA-APHIS CPHST Mission Laboratory in Edinburg, TX, qPCR amplifications were performed in replicates in volumes of 10.0 μl containing 5.0 μl 2× Perfect Real Time Premix ExTaq (Takara Bio USA, Mountain View, CA), 0.25 μl 40× Perfect Real TimePremix ExTaq Buffer, 3.75 μl DNAse-free <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-5.gif"/></noscript></span></span> and 1.0 μl template DNA. Thermal cycling for these reactions were performed on the CFX-96 Touch (Bio-Rad, Hercules, CA) using the same amplification conditions as what was performed by USDA-ARS described above. A total of 113 available individuals collected from several Vienna fly colonies reared in mass-rearing facilities in Austria (<em>N</em> = 4), Guatemala (<em>N</em> = 75), USA (<em>N</em> = 30), and Spain (<em>N</em> = 3) were tested using this assay.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-26">Demultiplexed GBS sequences are stored and curated at NCBI under BioProject accession numbers PRJNA341700 and PRJNA341535. Supporting data including the RQTL analysis file, raw linkage map, and <em>.vcf</em> are included among the supplemental files Table and Figure legends are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1/FileS1.pdf">File S1</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Genetic crosses</h3><p id="p-27">Genetic crosses between individual <em>wp</em> females from the Vienna-8 D53- GSS colony with individual males from the HiMed wild-type colony resulted in the segregation of <em>white pupae</em> alleles and placed the mutant white pupae and wild-type brown pupae phenotypes in a common genetic background. Three sibships were generated using this isolated-mating crossing scheme, and these sibships were comprised of white pupae and wild-type individuals. The proportion of wild-type brown pupae individuals to white pupae individuals was recorded at every generation. The parental generation was comprised of wild-type brown pupae males and white pupae females. As expected, 100% of the F1 and F3 generation exhibited the wild-type brown pupae phenotype. The F2 and F4 generation that were the progeny of F1 and F3 full-sibling intercrosses, respectively, was comprised of brown and white pupae individuals at a ratio that did not significantly deviate from 3:1 (adjusted <em>p</em>-value <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/embed/mml-math-6.gif"/></noscript></span></span>0.05) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) which is the ratio expected from single-heterozygote crosses. In each generation, there was no significant bias in sex-ratio. All sibships were derived from full-sibling matings and all full-sibling pairs were derived from one isolated mating pair.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487191/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487191/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Ratios of wild-type brown pupae to white pupae individuals for each of four genetic crosses (names of each cross are in ID column) in F2 and F4 sibships from the genetic crossing experiment</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>SNP discovery</h3><p id="p-29">To obtain needed sequence coverage from the samples in the GBS libraries, a total of three lanes of Illumina HiSeq were used for sequencing, and 95 individuals were sequenced on each lane. The number of raw reads, reads removed due to lack of restriction site, reads removed due to low quality, and number of reads retained is included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1/TableS1.pdf">Table S1</a>. A total of 77,844 SNPs were identified on 443 unique scaffolds from the <em>C. capitata</em> genome assembly. Genome-wide Fst values show 368 fixed loci (Fst = 1) and 781 additional nearly fixed loci (Fst &gt; 0.95) when comparing individuals from the wild-type HiMed colony to those from the Vienna-8 D53- GSS colony across the genome. The large number of fixed and nearly fixed SNPs that segregate between the wild-type colony and the Vienna fly colony demonstrates the need for performing a cross to isolate the differences that are attributed to the pupal color phenotype. For this application, a genome-wide association study would have identified many candidate loci that are differentially fixed in the two colonies. Furthermore, since Vienna strain males are heterozygous and thus contain a wild-type allele, the locus of interest is not expected to be differentially fixed between the two colonies when considering both males and females.</p></div><div id="sec-14" class="subsection"><h3>Population genetics</h3><p id="p-30">A DAPC analysis showed that the dataset, including 143 individuals from various SIT facilities that produce various Vienna strain flies, individuals from its native and established ranges in Australia, the Azores, Brazil, Hawaii, Morocco, Spain, Central America (Guatemala, Honduras, and Panama), and Africa (Mozambique, Reunion Island, Senegal, and South Africa), and three individuals collected from the southern California exclusion zone (with active SIT) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) was predicted to be from seven distinct clusters. These clusters are differentiated largely by principle components 1 and 2, which, together, explain 86% of the variance. The Vienna fly strains were divided into three clusters instead of being grouped into one (red in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This is potentially due to the practice of colony infusion of wild material that has been adopted by the mass-rearing facilities, as the introduction of wild genetic material from different sources will result in genotypic variation that is tied to the facility rather than the strain. Individuals from the wild-type HiMed colony (green) that were used in the genetic crosses are distinct, but are most similar to individuals from Australia, the Azores, Brazil, Hawaii, Spain, and Morocco, which themselves form a distinct cluster (yellow). Individuals from Guatemala, Honduras, and Panama are distinct and form a Central American cluster (violet). Lastly, individuals from sub-Saharan Africa are also very distinct (blue). Flies caught from California (triangles) appear to be from multiple geographic sources, including both Central America and the cluster containing flies from Australia, the Azores, Brazil, Hawaii, Morocco, and Spain, with no individuals appearing to be derived from Vienna flies released for SIT. In contrast, a total of 17 of the 24 captures from Spain (red diamonds) were assigned to the Vienna fly clusters, which indicates that they are released Vienna flies, likely as part of a suppression program, that were recaptured but designated as wild through other methods.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487244/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487244/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genotyping methods used to obtain genotypes of Vienna colony flies from multiple rearing facilities</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A discriminant analysis of principle components was performed on a total of 143 individuals from various sources using their genotypes for 77,844 single nucleotide polymorphism loci. All individuals have a membership probability of &lt; 99% to one of seven clusters, Vienna strain flies were assigned to three clusters (red), and wild individuals formed four distinct clusters (yellow, green, violet, and blue). Individuals from Spain (diamonds) are denoted as some were assigned to the same cluster as Vienna strain flies (red diamonds), and some were assigned to the wild fly cluster consisting of flies originating from Australia, the Azores, Brazil, Hawaii, and Morocco (yellow diamonds). Individuals from California (triangles) are denoted as one was assigned to the wild fly cluster consisting of flies originating from Australia, the Azores, Brazil, Hawaii, and Morocco (yellow triangle), while others were assigned to the cluster containing individuals from Central America (violet traingles). The box denoting eigenvalues shows the relative variance described by each of the principle components in descending order and that the clusters can be differentiated using principle components 1 and 2 (x-axis and y-axis, respectively), which together explains 86% of the variance. The clusters roughly correspond to a geographic location, with flies collected from sub-Saharan African countries being the most distinct." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2049114201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A discriminant analysis of principle components was performed on a total of 143 individuals from various sources using their genotypes for 77,844 single nucleotide polymorphism loci. All individuals have a membership probability of &lt; 99% to one of seven clusters, Vienna strain flies were assigned to three clusters (red), and wild individuals formed four distinct clusters (yellow, green, violet, and blue). Individuals from Spain (diamonds) are denoted as some were assigned to the same cluster as Vienna strain flies (red diamonds), and some were assigned to the wild fly cluster consisting of flies originating from Australia, the Azores, Brazil, Hawaii, and Morocco (yellow diamonds). Individuals from California (triangles) are denoted as one was assigned to the wild fly cluster consisting of flies originating from Australia, the Azores, Brazil, Hawaii, and Morocco (yellow triangle), while others were assigned to the cluster containing individuals from Central America (violet traingles). The box denoting eigenvalues shows the relative variance described by each of the principle components in descending order and that the clusters can be differentiated using principle components 1 and 2 (&lt;em&gt;x&lt;/em&gt;-axis and &lt;em&gt;y&lt;/em&gt;-axis, respectively), which together explains 86% of the variance. The clusters roughly correspond to a geographic location, with flies collected from sub-Saharan African countries being the most distinct.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3637/F3.medium.gif" width="440" height="275"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/11/3637/F3.medium.gif" width="440" height="275"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487245" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">A discriminant analysis of principle components was performed on a total of 143 individuals from various sources using their genotypes for 77,844 single nucleotide polymorphism loci. All individuals have a membership probability of &lt; 99% to one of seven clusters, Vienna strain flies were assigned to three clusters (red), and wild individuals formed four distinct clusters (yellow, green, violet, and blue). Individuals from Spain (diamonds) are denoted as some were assigned to the same cluster as Vienna strain flies (red diamonds), and some were assigned to the wild fly cluster consisting of flies originating from Australia, the Azores, Brazil, Hawaii, and Morocco (yellow diamonds). Individuals from California (triangles) are denoted as one was assigned to the wild fly cluster consisting of flies originating from Australia, the Azores, Brazil, Hawaii, and Morocco (yellow triangle), while others were assigned to the cluster containing individuals from Central America (violet traingles). The box denoting eigenvalues shows the relative variance described by each of the principle components in descending order and that the clusters can be differentiated using principle components 1 and 2 (<em>x</em>-axis and <em>y</em>-axis, respectively), which together explains 86% of the variance. The clusters roughly correspond to a geographic location, with flies collected from sub-Saharan African countries being the most distinct.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Linkage and QTL mapping</h3><p id="p-33">The three sibships in the F4 mapping population were comprised of white pupae and wild-type brown pupae individuals in a ratio that did not significantly deviate from the predicted ratio for three wild-type brown pupae to one white pupae individuals (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). All sibships were derived from full-sibling matings and all full-sibling pairs were derived from one isolated mating pair. Linkage analysis placed 980 SNPs from 133 unique scaffolds onto 11 linkage groups ranging from 2.1 to 144 Mb in size and represent 71.1% of the assembled genome. Linkage groups were anchored to chromosomes based on existing polytene chromosome maps using genes found in both maps. Unplaced linkage groups were assigned letters A–D (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1/FigureS1.pdf">Figure S1</a>). There are known to be five autosomes and two sex chromosomes (X and Y) in <em>C. capitata</em> and linkage groups from this data study were anchored to three of the five chromosomes.</p><p id="p-34">Quantitative trait locus analysis on the resulting linkage map was performed using the pupal color phenotype of each individual in the mapping populations. The single locus binary model revealed tight linkage between loci on scaffold NW_004523946.1 of the <em>C. capitata</em> genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), which is ∼3.3 MB in length. The single locus with the largest LOD score was on this scaffold at position 1,353,742 and was attributed a LOD score of 32.2. Manual inspection of the genotypes at this locus shows that out of 114 individuals included in the QTL analysis, 109 (96%) of the genotypes were in perfect LD with the phenotype, there was 1 (&lt; 1%) recombinant, and 4 (3.5%) genotypes were missing. The SNP locus with the highest LOD score is not located in a gene, and it is thus unlikely that the locus with the highest LOD score contains the causative mutation that confers the white pupae phenotype. However, it is likely that the gene for <em>white pupae</em> is one of the 113 genes located in the same scaffold (scaffold NW_004523946.1) as this QTL. Inspection of this scaffold for genes in the previously identified arthropod melanization pathway (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">van’t Hof and Saccheri 2010</a>) revealed that none are located on the same scaffold as the SNP with the highest LOD score.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative trait loci analysis using the binary interval mapping model. Results indicate that the pupal color phenotype is tightly linked to loci on autosome 5. The dotted line represents a logarithm of the odds (LOD) threshold for significance as determined by a permutation test performed with 100,000 permutations (p &lt; )." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2049114201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Quantitative trait loci analysis using the binary interval mapping model. Results indicate that the pupal color phenotype is tightly linked to loci on autosome 5. The dotted line represents a logarithm of the odds (LOD) threshold for significance as determined by a permutation test performed with 100,000 permutations (p &lt; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3637/F4.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/11/3637/F4.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487212" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Quantitative trait loci analysis using the binary interval mapping model. Results indicate that the pupal color phenotype is tightly linked to loci on autosome 5. The dotted line represents a logarithm of the odds (LOD) threshold for significance as determined by a permutation test performed with 100,000 permutations (p &lt; <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F4/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F4/embed/mml-math-9.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Calculation of Weir and Cockerham’s Fst as a measurement of differentiation between female flies from multiple Vienna colonies from around the world and wild individuals collected from the world-wide distribution of <em>C. capitata</em> shows a high level of differentiation for multiple loci. The SNP locus with the highest QTL score (the data point highlighted in red <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) is identified as nearly fixed (Fst = 0.99). In our sampling of wild flies, one individual from Brazil was heterozygous at SNP locus position 1353742 on scaffold NW_004523946.1. This indicates that the allele frequency of the SNP linked to <em>wp</em> is likely ≤ 0.5% in wild <em>C. capitata</em> based on the populations sampled in this study.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Weir and Cockerham&#x2019;s Fst calculated between wild individuals and only Vienna colony females that are homozygous for wp using the 981 single nucleotide polymorphism (SNP) loci in the linkage map. The SNP locus identified through quantitative trait loci analysis as tightly linked to wp was colored red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2049114201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Weir and Cockerham&#x2019;s Fst calculated between wild individuals and only Vienna colony females that are homozygous for &lt;em&gt;wp&lt;/em&gt; using the 981 single nucleotide polymorphism (SNP) loci in the linkage map. The SNP locus identified through quantitative trait loci analysis as tightly linked to &lt;em&gt;wp&lt;/em&gt; was colored red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3637/F5.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/11/3637/F5.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487142" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Weir and Cockerham’s Fst calculated between wild individuals and only Vienna colony females that are homozygous for <em>wp</em> using the 981 single nucleotide polymorphism (SNP) loci in the linkage map. The SNP locus identified through quantitative trait loci analysis as tightly linked to <em>wp</em> was colored red.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Genotype confirmation tests</h3><p id="p-38">Genotype confirmation tests of TaqMan assay AHMSY8D, designed to differentiate between homozygous wild-type (G/G), heterozygous (G/T), and homozygous for the <em>wp</em>-linked variant (T/T) at the <em>wp</em>-linked SNP locus, showed consistency between marker genotype and phenotype of tested individuals. A summary of the number of individuals genotyped using GBS and the developed assay can be found in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. In a preliminary test of nine Vienna-8 D53- females, 12 Vienna-8 D53- males, 12 HiMed females, 12 HiMed males, and one no template control, all females from the Vienna-8 D53- GSS colony, which have the white pupae phenotype, genotyped as homozygous for the <em>wp</em>-linked allele; all males from the Vienna-8 D53- GSS colony, which have the wild-type brown pupae phenotype, genotyped as heterozygous; and all females and males from the HiMed wild-type colony, which have a wild-type brown pupae phenotype, genotyped as homozygous for the wild-type allele (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The three wild captures from Spain, which were assigned to one of the clusters containing mass-reared Vienna strain flies based on genome-wide SNP data, showed consistency between the GBS genotype and the SNP assay genotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1/FigureS2.pdf">Figure S2</a>), and provides evidence that those wild-captures were Vienna strain flies released for SIT. Genotype confirmation testing of the TaqMan assay AHMSY8D on flies from several Vienna strain-rearing facilities showed that all the Vienna strain males genotyped were heterozygous and all the Vienna strain females were homozygous for the <em>wp</em>-linked allele (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allelic discrimination plot showing  Rn value for wild-type allele (T) on the x-axis vs. the  Rn value for the wp-linked allele (G) on the y-axis for individuals genotyped with assay AHMSY8D, where the  Rn value is the detected change between the initial read and the endpoint read in fluorescent signal of the reporter and the passive reference dye. Average values of three replicates are shown with minimum and maximum  Rn values for both alleles are shown as bars on their respective axes. Individuals that were assigned the genotype of homozygous for the wild-type allele are in orange, heterozygotes are in red, homozygous for the wp-linked allele are in blue, and the no template control (NTC) is in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2049114201" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Allelic discrimination plot showing &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Rn value for wild-type allele (T) on the &lt;em&gt;x&lt;/em&gt;-axis &lt;em&gt;vs.&lt;/em&gt; the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-11&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Rn value for the &lt;em&gt;wp&lt;/em&gt;-linked allele (G) on the &lt;em&gt;y&lt;/em&gt;-axis for individuals genotyped with assay AHMSY8D, where the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-12&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Rn value is the detected change between the initial read and the endpoint read in fluorescent signal of the reporter and the passive reference dye. Average values of three replicates are shown with minimum and maximum &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-13&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Rn values for both alleles are shown as bars on their respective axes. Individuals that were assigned the genotype of homozygous for the wild-type allele are in orange, heterozygotes are in red, homozygous for the &lt;em&gt;wp&lt;/em&gt;-linked allele are in blue, and the no template control (NTC) is in black.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3637/F6.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/11/3637/F6.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3637/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487181" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-39" class="first-child">Allelic discrimination plot showing <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-10.gif"/></noscript></span></span> Rn value for wild-type allele (T) on the <em>x</em>-axis <em>vs.</em> the <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-11.gif"/></noscript></span></span> Rn value for the <em>wp</em>-linked allele (G) on the <em>y</em>-axis for individuals genotyped with assay AHMSY8D, where the <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-12.gif"/></noscript></span></span> Rn value is the detected change between the initial read and the endpoint read in fluorescent signal of the reporter and the passive reference dye. Average values of three replicates are shown with minimum and maximum <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3637/F6/embed/mml-math-13.gif"/></noscript></span></span> Rn values for both alleles are shown as bars on their respective axes. Individuals that were assigned the genotype of homozygous for the wild-type allele are in orange, heterozygotes are in red, homozygous for the <em>wp</em>-linked allele are in blue, and the no template control (NTC) is in black.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487248/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487248/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genotypes of Vienna colony flies from multiple rearing facilities</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-41">The medfly is a major pest of worldwide economic importance. Their threat to high-value agriculture is so extensive that IPM programs are employed to prevent their long-term establishment in many countries worldwide. An important component of these IPM programs is the use of the SIT, which leverages the availability of GSS that are established for these species. When SIT flies are recaptured in area-wide monitoring efforts, it is necessary to be able to differentiate wild from SIT flies to prevent the initiation of unnecessary and costly quarantines when a SIT fly is mistaken for wild and also appropriately initiate quarantines when a wild fly is detected. Identification of loci linked to traits maintained in SIT released flies can be developed into a diagnostic assay, which will ensure that the appropriate management practices take place.</p><p id="p-42">To address this objective, we identified SNPs that are tightly linked to a trait maintained in Vienna fly colonies that are released for SIT. These SNPs were identified for the purpose of developing a diagnostic assay that is capable of distinguishing wild flies from intentionally released SIT genetic sexing flies. This was achieved through an experimental design that generated a mapping population that was genotyped for SNPs through sequencing and analysis of a GBS library. These genotypes were used to perform a population genetic analysis that showed very strong population structure between Vienna fly colony individuals and wild individuals, and between wild individuals collected from the breadth of the geographic range of <em>C. capitata</em>. The GBS genotypes were then used to assemble scaffolds into a linkage map in addition to performing a QTL analysis. The results of the QTL analysis were used to design a diagnostic SNP assay (TaqMan assay AHMSY8D), which is commercially available for purchase, but can also be genotyped using alternative methods using the sequence information reported. This assay was tested using known Vienna strain samples to assess the consistency of the locus and demonstrate that the assay is consistent in correctly identifying Vienna fly colony individuals from various mass-rearing facilities. Additional validation studies are planned for assay AHMSY8D to measure the probability of detecting a Vienna strain profile in wild flies and the exclusiveness of the <em>wp</em>-linked allele. In addition, SNP genotypes derived from GBS of wild individuals show that the presence of the <em>wp</em>-linked allele in the wild is ≤ 0.5% when 109 individual flies from 13 countries were sampled. Although we are not able to conclude that the SNP linked to the <em>wp</em> is unique to Vienna strains, this result demonstrates the ability of this assay to be effective in differentiating wild flies from Vienna flies in all of the geographic locations sampled. It also demonstrates that the combination of this assay with mitochondrial data such as the N5N4 locus will make an effective and robust diagnostic assay.</p><p id="p-43">Population genetic analysis using all 77,844 SNP loci that were determined through GBS shows two clusters of mass-rearing Vienna strain flies that are distinct, but females from both clusters are homozygous for the <em>wp</em> (T/T) allele and males are heterozygous (G/T) at the SNP assay locus. This demonstrates that the designed SNP assay can be used by all medfly diagnostic stations regardless of which Vienna-SIT strain is released. If diagnosis of particular Vienna strains is needed, then SNP profiles for additional loci would need to be generated and compared.</p><p id="p-44">In addition to working toward the development of a diagnostic assay, we assembled the medfly genome into a linkage map using the mapping population used to identify the loci linked to <em>white pupae</em>. This linkage map can be used to further scaffold the assembly into chromosome-scale superscaffolds and perform comparative genomic analysis to identify synteny of orthologous genes with the model species <em>Drosophila melanogaster</em> or other tephritid species for which chromosome-scale assemblies exist (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sim and Geib 2017</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sved <em>et al.</em> 2016</a>).</p><p id="p-45">Identification of the gene causing the <em>white pupae</em> mutation in medfly is important to the development of new strategies for genetic sexing strains in the future. A limitation of our QTL analysis through genotyping with GBS sequences is that restriction site-associated SNP genotypes are unlikely to be in coding regions and a SNP locus tightly linked to the trait of interest is unlikely to be in the gene governing the trait. Differential expression analysis of RNA sequences coupled with low-coverage resequencing of white and brown pupae individuals are currently being pursued to identify the gene causing <em>white pupae</em> in medfly. Differential expression analysis of brown and white pupae individuals collecting over a time-course will identify genes or coexpression networks that match the pupal color phenotype and identify the relative time frame at which the genes are expressed (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Love <em>et al.</em> 2014</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Robinson <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">McCarthy <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Langfelder and Horvath 2008</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Conesa <em>et al.</em> 2006</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nueda <em>et al.</em> 2014</a>). Low-coverage resequencing will identify all mutations in the genes or coexpression networks that are differentially expressed and the effects of these mutations can be identified (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cingolani <em>et al.</em> 2012</a>). Once the candidate genes are identified, they can be validated using RNAi (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schroder 2003</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hughes and Kaufman 2000</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 1999</a>) or targeted mutagenesis techniques such as CRISPR-Cas9 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Friedland <em>et al.</em> 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hwang <em>et al.</em> 2013</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bassett and Liu 2014</a>). These studies are facilitated by the development of the mapping population of the project presented here, which places the white pupae trait in a common background and is thus ideal for differential expression and dense variant identification because the analysis can be controlled for variation independent of the phenotype of interest.</p><div id="sec-18" class="subsection"><h3>Conclusions</h3><p id="p-46">The diagnostic tool developed based on a SNP locus tightly linked to the <em>wp</em> allele will hopefully be utilized to improve diagnostic capabilities for medfly, and help to increase the efficiency and accuracy of identification of wild medfly in regions subjected to SIT for suppression, eradication, and exclusion. By integrating this tool with existing genetic and morphological methodologies, fewer cases of inconclusive origin of recaptured flies will exist, reducing unnecessary quarantines caused by potentially recaptured Vienna SIT flies. While we expect this method to be robust, as this method is adopted and applied throughout the world to increasing numbers of geographic populations and mass-reared Vienna flies, the strength of this assay will continue to be tested and validated. This research is also the starting point to examining the causative mutations for the genetic sexing traits, and possible application of those traits to new species. In order to have an effective SIT program, a GSS that is cost-effective and facilitates the release of male-only sterile flies needs to be established for the species of interest. The identification of the <em>white pupae</em> gene in medfly can be used to identify its orthologous genes in other economic pests and the same genome editing tools used for validation can recreate this phenotype and sex linkage in other economically important species (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2015</a>). This project provides a foundation for the development of GSS lines for important tephritid pests that are currently not amenable to male-only mass rearing for SIT.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">Genotype by Sequencing (GBS) library preparation and sequencing was performed at the Genomic Diversity Facility, which is a part of the Cornell University Institute of Biotechnology. Samples used in this study were procured from the US Department of Agriculture Agricultural Research Service (USDA-ARS) Daniel K. Inouye Pacific Basin Agricultural Research Center (PBARC) rearing facility in Hilo, HI as well as the USDA Animal and Plant Health Inspection Service (APHIS) Plant Protection and Quarantine Science and Technology Mission Laboratory in Edinburg, TX. Mapping populations were generated by Steve Tam and Emmaleah Stauber. Laboratory support was provided by Nicole Yoneishi, Angela Kauwe, Terrance Todd, and Daniel Garza. Bioinformatic support was provided by Brian Hall and Theodore DeRego. Bioinformatic and data analysis were performed on the Moana high-performance computing cluster at the USDA-ARS Daniel K. Inouye PBARC. Samples were provided by Kostas Bourtzis, Pedro Rendon, Song-Min So, Todd Shelly, Bruce A. McPheron, Marc De Meyer, Shirani Poogoda, and Mario R. Vargas. We would like to thank Julian R. Dupuis, corresponding editor Kevin Thornton, and the anonymous reviewers for their thoughtful feedback. Funding for this project was provided by USDA-ARS and USDA-APHIS Farm Bill Section 10007 projects 3.0251.02 (FY 2014), 3.0256.01 (FY 2015), and 3.0392.02 (FY 2016) awarded to S.M.G. and N.B.B. S.B.S. was partially supported by the USDA National Institute of Food and Agriculture Agriculture and Food Research Initiative Education and Literacy Initiative 2017-67012-26087 (FY17-FY19), which was awarded to S.B.S. The USDA-ARS is an equal opportunity/affirmative action employer and all agency services are available without discrimination. Mention of commercial products and organizations in this manuscript is solely to provide specific information. It does not constitute endorsement by USDA-ARS over other products and organizations not mentioned.</p><p id="p-48">Author contributions: S.B.S., S.M.G., R.R.-A., and N.B.B. conceived and designed the experiment. S.B.S., S.M.G., and R.R.-A. performed the experiments, analyzed the data, and evaluated the conclusions. Wild <em>C. capitata</em> samples that were included in the GBS library were provided by N.B.B. and R.R.-A. S.B.S., R.R.-A., N.B.B., and S.M.G. contributed to writing the manuscript. The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-49">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300169/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-50"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-06-12"><span class="received-label">Received </span>June 12, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-09-04"><span class="accepted-label">Accepted </span>September 4, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Sim <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.1" data-doi="10.1146/annurev.en.05.010160.001251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrewartha</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birch</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">Some recent contributions to the study of the distribution and abundance of insects.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.volume%253D5%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.en.05.010160.001251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.en.05.010160.001251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.2" data-doi="10.1603/029.102.0153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barr</span>  <span class="cit-name-given-names">N. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pathway analysis of <em>Ceratitis capitata</em> (Diptera: Tephritidae) using mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1603%252F029.102.0153%26rft_id%253Dinfo%253Apmid%252F19253662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1603/029.102.0153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19253662&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.3" data-doi="10.1016/j.ymeth.2014.02.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassett</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CRISPR/Cas9 mediated genome engineering in Drosophila.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D69%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2014.02.019%26rft_id%253Dinfo%253Apmid%252F24576617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2014.02.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24576617&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping quantitative trait loci in the case of a spike in the phenotype distribution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D1169%26rft.epage%253D1175%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2Bin%2Bthe%2BCase%2Bof%2Ba%2BSpike%2Bin%2Bthe%2BPhenotype%2BDistribution%26rft_id%253Dinfo%253Apmid%252F12663553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2My8zLzExNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889–890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D19%26rft.spage%253D889890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.6" data-doi="10.1046/j.1525-142x.1999.99013.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahaffey</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenzen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denell</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahaffey</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Using RNAi to investigate orthologous homeotic gene function during development of distantly related insects.</span> <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DBrown%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D15%26rft.atitle%253DUsing%2BRNAi%2Bto%2Binvestigate%2Borthologous%2Bhomeotic%2Bgene%2Bfunction%2Bduring%2Bdevelopment%2Bof%2Bdistantly%2Brelated%2Binsects.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1525-142x.1999.99013.x%26rft_id%253Dinfo%253Apmid%252F11324015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1525-142x.1999.99013.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11324015&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086320200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Busch-Petersen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Temperature Sensitive Lethal Factors and Puparial Colour Sex Separation Mechanisms in the Mediterranean Fruit Fly</em>, <em>Ceratitis capitata (Wied.)</em></span>, Vol. 21. <span class="cit-publ-name">International Atomic Energy Agency</span>, <span class="cit-publ-loc">Vienna.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.8" data-doi="10.1534/genetics.115.181883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guang</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Targeted chromosomal translocations and essential gene knockout using CRISPR/Cas9 technology in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.181883%26rft_id%253Dinfo%253Apmid%252F26482793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS80LzEyOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.9" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.10" data-doi="10.1093/bioinformatics/btl056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nueda</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">maSigPro: a method to identify significantly differential expression profiles in time-course microarray experiments.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1096</span>–<span class="cit-lpage">1102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl056%26rft_id%253Dinfo%253Apmid%252F16481333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16481333&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236997600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.11" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.12" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.13" data-doi="10.1093/jee/89.4.946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enkerlin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celedonio</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Increased accuracy in discrimination between captured wild unmarked and released dye-marked adults in fruit fly (Diptera: Tephritidae) sterile released programs.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjee%252F89.4.946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jee/89.4.946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enkerlin</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Impact of Fruit Fly Control Programmes Using the Sterile Insect Technique</em>.</span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dordrecht, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Genetic Sexing Strains in Mediterranean Fruit Fly</em>, <em>an Example for Other Species Amenable to Large-Scale Rearing for the Sterile Insect Technique</em>.</span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dirdrecht, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gencheva</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerremans</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Improved stability of genetic sex-separation strains for the Mediterranean fruit fly, <em>Ceratitis capitata</em>.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DFranz%26rft.auinit1%253DG.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D72%26rft.epage%253D82%26rft.atitle%253DImproved%2Bstability%2Bof%2Bgenetic%2Bsex-separation%2Bstrains%2Bfor%2Bthe%2BMediterranean%2Bfruit%2Bfly%252C%2BCeratitis%2Bcapitata.%26rft_id%253Dinfo%253Apmid%252F18470062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470062&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.17" data-doi="10.1038/nmeth.2532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedland</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzur</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiacovo</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable genome editing in <em>C. elegans</em> via a CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D741%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2532%26rft_id%253Dinfo%253Apmid%252F23817069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817069&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322453600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gil</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source"><em>Dominican Republic Uses Nuclear Technology to Win the War Against Fruit Flies. Technical Report bull58–2/5820405</em>.</span> <span class="cit-publ-name">International Atomic Energy Agency</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.19" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tassel-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guillén Aguilar</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source"><em>Programa mosca del Mediterráneo: Manual para la diferenciación de moscas del mediterráneo ceratitis capitata (WIED.) silvestres (fértiles) de moscas irradiadas (estériles). Programa Mosca del Mediterráneo</em>.</span> <span class="cit-publ-name">Secretaría de Agricultura y Recursos Hidráulicos</span>, <span class="cit-publ-loc">México</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.21" data-doi="10.1111/j.1439-0418.1995.tb01303.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hendrichs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rendon</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Increased effectiveness and applicability of the sterile insect technique through male-only releases for control of Mediterranean fruit flies during fruiting seasons.</span> <abbr class="cit-jnl-abbrev">J. Appl. Entomol.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BEntomol.%26rft.volume%253D119%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0418.1995.tb01303.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0418.1995.tb01303.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TA74600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">RNAi analysis of deformed, proboscipedia and sex combs reduced in the milkweed bug <em>Oncopeltus fasciatus</em>: novel roles for Hox genes in the hemipteran head.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">3683</span>–<span class="cit-lpage">3694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DHughes%26rft.auinit1%253DC.%26rft.volume%253D127%26rft.issue%253D17%26rft.spage%253D3683%26rft.epage%253D3694%26rft.atitle%253DRNAi%2Banalysis%2Bof%2BDeformed%252C%2Bproboscipedia%2Band%2BSex%2Bcombs%2Breduced%2Bin%2Bthe%2Bmilkweed%2Bbug%2BOncopeltus%2Bfasciatus%253A%2Bnovel%2Broles%2Bfor%2BHox%2Bgenes%2Bin%2Bthe%2Bhemipteran%2Bhead%26rft_id%253Dinfo%253Apmid%252F10934013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI3LzE3LzM2ODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.23" data-doi="10.1371/journal.pone.0068708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeder</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaini</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable and precise zebrafish genome editing using a CRISPR-Cas system.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68708%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068708%26rft_id%253Dinfo%253Apmid%252F23874735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874735&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.24" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">adegenet: a R package for the multivariate analysis of genetic markers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397895&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.25" data-doi="10.1093/bioinformatics/btr521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3070</span>–<span class="cit-lpage">3071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr521%26rft_id%253Dinfo%253Apmid%252F21926124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21926124&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerremans</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Use of a temperature-sensitive lethal mutation strain of medfly (<em>Ceratitis capitata</em>) for the suppression of pest populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klassen</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Area-Wide Integrated Pest Management and the Sterile Insect Technique</em>.</span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dirdrecht, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klassen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtis</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>History of the Sterile Insect Technique</em>.</span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dordrecht, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knipling</span>  <span class="cit-name-given-names">E. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1955</span> <span class="cit-article-title">Possibilities of insect control or eradication through the use of sexually sterile males.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft.volume%253D48%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krafsur</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Sterile insect technique for suppressing and eradicating insect populations: 55 years and counting.</span> <abbr class="cit-jnl-abbrev">J. Agric. Entomol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BEntomol.%26rft.volume%253D15%26rft.spage%253D303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.31" data-doi="10.1016/0169-4758(87)90196-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krafsur</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novy</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Screwworm eradication in North and Central America.</span> <abbr class="cit-jnl-abbrev">Parasitol. Today</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasitology%2Btoday%2B%2528Personal%2Bed.%2529%26rft.stitle%253DParasitol%2BToday%26rft.aulast%253DKrafsur%26rft.auinit1%253DE.%2BS.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D131%26rft.epage%253D137%26rft.atitle%253DScrewworm%2Beradication%2Bin%2BNorth%2Band%2BCentral%2BAmerica.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-4758%252887%252990196-7%26rft_id%253Dinfo%253Apmid%252F15462936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-4758(87)90196-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15462936&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987H514600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.32" data-doi="10.1186/1471-2105-9-559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">WGCNA: an R package for weighted correlation network analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLangfelder%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.spage%253D559%26rft.epage%253D559%26rft.atitle%253DWGCNA%253A%2Ban%2BR%2Bpackage%2Bfor%2Bweighted%2Bcorrelation%2Bnetwork%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-559%26rft_id%253Dinfo%253Apmid%252F19114008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19114008&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast R functions for robust correlations and hierarchical clustering.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DPerez%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253Dc03%26rft.spage%253D1%26rft.epage%253D17%26rft.atitle%253DIntake_epis_food%2528%2529%253A%2BAn%2BR%2BFunction%2Bfor%2BFitting%2Ba%2BBivariate%2BNonlinear%2BMeasurement%2BError%2BModel%2Bto%2BEstimate%2BUsual%2Band%2BEnergy%2BIntake%2Bfor%2BEpisodically%2BConsumed%2BFoods.%26rft_id%253Dinfo%253Apmid%252F22837731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22837731&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.34" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.35" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.36" data-doi="10.1093/nar/gks042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">4288</span>–<span class="cit-lpage">4297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks042%26rft_id%253Dinfo%253Apmid%252F22287627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22287627&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304535500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.37" data-doi="10.2307/1536193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1914</span> <span class="cit-article-title">No crossing over in the male of <em>Drosophila</em> of genes in the second and third pairs of chromosomes.</span> <abbr class="cit-jnl-abbrev">Biol. Bull.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BBiological%2BBulletin%26rft.stitle%253DBiol.%2BBull.%26rft.aulast%253DMORGAN%26rft.auinit1%253DT.%2BH.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D195%26rft.epage%253D204%26rft.atitle%253DNO%2BCROSSING%2BOVER%2BIN%2BTHE%2BMALE%2BOF%2BDROSOPHILA%2BOF%2BGENES%2BIN%2BTHE%2BSECOND%2BAND%2BTHIRD%2BPAIRS%2BOF%2BCHROMOSOMES%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1536193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1536193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000201827800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.38" data-doi="10.1146/annurev.ento.43.1.471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savoie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Randen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Eradication and pest management.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.aulast%253DMyers%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D471%26rft.epage%253D491%26rft.atitle%253DERADICATION%2BAND%2BPEST%2BMANAGEMENT%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ento.43.1.471%26rft_id%253Dinfo%253Apmid%252F9444755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ento.43.1.471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9444755&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071438300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norris</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-article-title">A method of marking Calliphoridae (Diptera) during emergence from the puparium.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D180%26rft.spage%253D1002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.40" data-doi="10.1093/bioinformatics/btu333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nueda</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarazona</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Next maSigPro: updating maSigPro bioconductor package for RNA-seq time series.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2598</span>–<span class="cit-lpage">2602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu333%26rft_id%253Dinfo%253Apmid%252F24894503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24894503&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.41" data-doi="10.1093/bioinformatics/btx576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ouellette</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steven</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouwer</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">LinkageMapView - rendering high resolution linkage and QTL maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .org/</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx576</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schetelig</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arensburger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benoit</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The whole genome sequence of the Mediterranean fruit fly, <em>Ceratitis capitata</em> (Wiedemann), reveals insights into the biology and adaptive evolution of a highly invasive pest species.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.43" data-doi="10.1093/jee/tou027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parubrub</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smallridge</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haymer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mitochondrial single nucleotide polymorphisms in ceratitis capitata (Diptera: Tephritidae) can distinguish sterile, released flies from wild flies in various regions of the world.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjee%252Ftou027%26rft_id%253Dinfo%253Apmid%252F26470134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jee/tou027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26470134&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.44" data-doi="10.1093/bioinformatics/btt563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehtonen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auvinen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lep-MAP: fast and accurate linkage map construction for large SNP datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3128</span>–<span class="cit-lpage">3134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt563%26rft_id%253Dinfo%253Apmid%252F24078685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24078685&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327714700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calboli</span>  <span class="cit-name-given-names">F. C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilӓ</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of ultradense linkage maps with Lep-MAP2: stickleback F<sub>2</sub> recombinant crosses as an example.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D8%26rft.spage%253D78%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.46" data-doi="10.1603/0022-0493-97.5.1547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rendon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McInnis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lance</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Medfly (Diptera: Tephritidae) genetic sexing: large-scale field comparison of males-only and bisexual sterile fly releases in Guatemala.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">1547</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1603%252F0022-0493-97.5.1547%26rft_id%253Dinfo%253Apmid%252F15568342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1603/0022-0493-97.5.1547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15568342&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.47" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19910308&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273116100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.48" data-doi="10.1093/aesa/72.5.583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rossler</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">The genetics of the Mediterranean fruit fly: a ‘white pupae’ mutant.</span> <abbr class="cit-jnl-abbrev">Ann. Entomol. Soc. Am.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEntomol.%2BSoc.%2BAm.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faesa%252F72.5.583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aesa/72.5.583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.49" data-doi="10.1093/aesa/75.6.619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rossler</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Recombination in males and females of the Mediterranean fruit-fly (Diptera, Tephritidae) with and without chromosomal-aberrations.</span> <abbr class="cit-jnl-abbrev">Ann. Entomol. Soc. Am.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEntomol.%2BSoc.%2BAm.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faesa%252F75.6.619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aesa/75.6.619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.50" data-doi="10.1603/0022-0493(2007)100[1444:ANMATA]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">San Andres</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urbaneja</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabater-Munoz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castanera</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A novel molecular approach to assess mating success of sterile ceratitis capitata (Diptera: Tephritidae) males in sterile insect technique programs.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">1444</span>–<span class="cit-lpage">1449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEconomic%2BEntomology%26rft.stitle%253DJ%2BEcon%2BEntomol%26rft.aulast%253DSan%2BAndres%26rft.auinit1%253DV.%26rft.volume%253D100%26rft.issue%253D4%26rft.spage%253D1444%26rft.epage%253D1449%26rft.atitle%253DA%2Bnovel%2Bmolecular%2Bapproach%2Bto%2Bassess%2Bmating%2Bsuccess%2Bof%2Bsterile%2BCeratitis%2Bcapitata%2B%2528Diptera%253A%2BTephritidae%2529%2Bmales%2Bin%2Bsterile%2Binsect%2Btechnique%2Bprograms.%26rft_id%253Dinfo%253Adoi%252F10.1603%252F0022-0493%25282007%2529100%255B1444%253AANMATA%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F17849900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1603/0022-0493(2007)100[1444:ANMATA]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17849900&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.51" data-doi="10.1038/nature01536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schroder</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The genes orthodenticle and hunchback substitute for bicoid in the beetle <em>Tribolium</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchr%25C3%25B6der%26rft.auinit1%253DR.%26rft.volume%253D422%26rft.issue%253D6932%26rft.spage%253D621%26rft.epage%253D625%26rft.atitle%253DThe%2Bgenes%2Borthodenticle%2Band%2Bhunchback%2Bsubstitute%2Bfor%2Bbicoid%2Bin%2Bthe%2Bbeetle%2BTribolium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01536%26rft_id%253Dinfo%253Apmid%252F12687002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12687002&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.52" data-doi="10.1093/jee/65.4.1217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyabara</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farias</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">A fluorescent compound for marking Tephritidae.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjee%252F65.4.1217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jee/65.4.1217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.53" data-doi="10.1534/g3.117.040170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geib</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A chromosome-scale assembly of the <em>Bactrocera cucurbitae</em> genome provides insight to the genetic basis of <em>white pupae</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1927</span>–<span class="cit-lpage">1940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.040170%26rft_id%253Dinfo%253Apmid%252F28450369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xOTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.54" data-doi="10.1093/jee/58.2.374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">A rapid method for identifying dye-marked fruit flies.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjee%252F58.2.374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jee/58.2.374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sved</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shearman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frommer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilchrist</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extraordinary conservation of entire chromosomes in insects over long evolutionary periods.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D70%26rft.spage%253D229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">qqman: an R package for visualizing GWAS results using Q-Q and manhattan plots.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. Available at: <a href="https://doi.org/10.1101/005165">https://doi.org/10.1101/005165</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">USDA-APHIS and Mexico Lower Rio Grande Valley</span></li></ol><cite>, <span class="cit-pub-date">2010</span> Mexican Fruit Fly Eradication Program Review. Report. Retrieved from <a href="http://www-naweb.iaea.org/nafa/ipc/public/ipc-mexican-fruit-fly-eradication-program.pdf">http://www-naweb.iaea.org/nafa/ipc/public/ipc-mexican-fruit-fly-eradication-program.pdf</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3637.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">USDA-APHIS</span></li></ol><cite>, <span class="cit-pub-date">2014</span> Mediterranean fruit fly preventative release program; 2014 review of sterile insect release facilities. Report. Retrieved from <a href="https://www.aphis.usda.gov/plant_health/plant_pest_info/fruit_flies/downloads/2014-medfly-prp-review.pdf">https://www.aphis.usda.gov/plant_health/plant_pest_info/fruit_flies/downloads/2014-medfly-prp-review.pdf</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.59" data-doi="10.1371/journal.pone.0010840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van’t Hof</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saccheri</span>  <span class="cit-name-given-names">I. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Industrial melanism in the peppered moth is not associated with genetic variation in canonical melanisation gene candidates.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010840%26rft_id%253Dinfo%253Apmid%252F20824070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20824070&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">R. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Rearing: mass production of tephritid fruit flies.</span> <abbr class="cit-jnl-abbrev">World Crop Pests</abbr> <span class="cit-vol">3B</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWorld%2BCrop%2BPests%26rft.volume%253D3%26rft.spage%253D141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.61" data-doi="10.1038/cr.2013.146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The CRISPR/Cas system mediates efficient genome engineering in <em>Bombyx mori</em>.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1414</span>–<span class="cit-lpage">1416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D23%26rft.spage%253D1414%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.146%26rft_id%253Dinfo%253Apmid%252F24165890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2013.146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24165890&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328072600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitten</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Automated sexing of pupae and its usefulness in control by sterile insects.</span> <abbr class="cit-jnl-abbrev">J. Econ. Entomol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BEntomol.%26rft.volume%253D62%26rft.spage%253D272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wyss</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> Screwworm eradication in the Americas. Ann. N. Y. Acad. Sci. 916: 186–193.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atchley</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mapping quantitative trait loci for complex binary diseases using line crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1417</span>–<span class="cit-lpage">1424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D143%26rft.issue%253D3%26rft.spage%253D1417%26rft.epage%253D1424%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2Bfor%2BComplex%2BBinary%2BDiseases%2BUsing%2BLine%2BCrosses%26rft_id%253Dinfo%253Apmid%252F8807312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My8zLzE0MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3637.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zepeda-Cisneros</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meza Hernandez</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Martinez</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibanez-Palacios</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zacharopoulou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development, genetic and cytogenetic analyses of genetic sexing strains of the Mexican fruit fly, <em>Anastrepha ludens</em> Loew (Diptera: Tephritidae).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">S1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253DS1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3637.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3621" title="Genome-Wide Analysis of DNA Methylation During Ovule Development of Female-Sterile Rice fsv1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3649" title="Linkage Analysis and Association Mapping QTL Detection Models for Hybrids Between Multiparental Populations from Two Heterotic Groups: Application to Biomass Production in Maize (Zea mays L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3637.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-L9Gft3AZwlHT2r18CgCet8ze-Vab5jencyHXKJWCf0w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3637" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3637" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3637" class="active" data-icon-position="" data-hide-link-title="0">A New Diagnostic Resource for Ceratitis capitata Strain Identification Based on QTL Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487097" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-fZ3uQ1bD-OAvujmPUikeMSsRTMv281VTag2mbKk1ufU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3637.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3637" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kvDs5g-8XxCEhEu7I4UHFHn__6YFeJ_ei1mvtGB7ghY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3637" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3637.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487097" id="node487097--2" data-pisa="ggg;7/11/3637" data-pisa-master="ggg;g3.117.300169" data-apath="/ggg/7/11/3637.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Diagnostic Resource for <em>Ceratitis capitata</em> Strain Identification Based on QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0914-6914" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sheina B. Sim</span>, <span class="highwire-citation-author" data-delta="1">Raul Ruiz-Arce</span>, <span class="highwire-citation-author" data-delta="2">Norman B. Barr</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9511-5139" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott M. Geib</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3637-3647;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300169 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487097--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sheina B. Sim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheina%2BB.%2BSim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sim%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASheina%2BB.%2BSim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0914-6914" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sheina B. Sim</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Raul Ruiz-Arce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raul%2BRuiz-Arce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruiz-Arce%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaul%2BRuiz-Arce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Norman B. Barr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norman%2BB.%2BBarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorman%2BB.%2BBarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Scott M. Geib </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BM.%2BGeib%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geib%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BM.%2BGeib%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9511-5139" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott M. Geib</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:scott.geib@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">scott.geib@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3637" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3637&amp;title=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3637&amp;title=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3637&amp;title=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3637&amp;text=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3637&amp;title=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3637&amp;t=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3637&amp;title=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3637&amp;title=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487097/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487097" id="node487097--3" data-pisa="ggg;7/11/3637" data-pisa-master="ggg;g3.117.300169" data-apath="/ggg/7/11/3637.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Diagnostic Resource for <em>Ceratitis capitata</em> Strain Identification Based on QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0914-6914" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sheina B. Sim</span>, <span class="highwire-citation-author" data-delta="1">Raul Ruiz-Arce</span>, <span class="highwire-citation-author" data-delta="2">Norman B. Barr</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9511-5139" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott M. Geib</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3637-3647;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300169 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487097--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sheina B. Sim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheina%2BB.%2BSim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sim%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASheina%2BB.%2BSim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0914-6914" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sheina B. Sim</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Raul Ruiz-Arce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raul%2BRuiz-Arce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruiz-Arce%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaul%2BRuiz-Arce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Norman B. Barr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Plant Health Science and Technology, Mission Laboratory, USDA-APHIS, Edinburg, Texas 78541</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norman%2BB.%2BBarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorman%2BB.%2BBarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Scott M. Geib </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS Daniel K. Inouye U.S. Pacific Basin Agricultural Research Center, Tropical Crop and Commodity Protection Research Unit, Hilo, Hawaii 96720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BM.%2BGeib%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geib%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BM.%2BGeib%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9511-5139" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott M. Geib</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:scott.geib@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">scott.geib@ars.usda.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3637&amp;rft.epage=3647&amp;rft.atitle=A%20New%20Diagnostic%20Resource%20for%20Ceratitis%20capitata%20Strain%20Identification%20Based%20on%20QTL%20Mapping&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sim%2C+Sheina+B.&amp;rft.au=Ruiz-Arce%2C+Raul&amp;rft.au=Barr%2C+Norman+B.&amp;rft.au=Geib%2C+Scott+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487097/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487097/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487097/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487097/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487097/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487097/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487097/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487097/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487097/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487097/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487097/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487097/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3637" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3637.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3637.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3637.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3637.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
