<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3757" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3757.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3757.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3757.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3757.atom" />
<meta name="HW.pisa" content="ggg;7/11/3757" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.117.300223" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Rossbach et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="DDK, a conserved serine-threonine protein kinase composed of a regulatory subunit, [Dbf4][1], and a catalytic subunit, [Cdc7][2], is essential for DNA replication initiation during S phase of the cell cycle through MCM2-7 helicase phosphorylation. The biological significance of DDK is well characterized, but the full mechanism of how DDK associates with substrates remains unclear. [Cdc7][2] is bound to chromatin in the Saccharomyces cerevisiae genome throughout the cell cycle, but there is little empirical evidence as to specific [Cdc7][2] binding locations. Using biochemical and genetic techniques, this study investigated the specific localization of [Cdc7][2] on chromatin. The Calling Cards method, using Ty5 retrotransposons as a marker for DNA–protein binding, suggests [Cdc7][2] kinase is preferentially bound to genomic DNA known to replicate early in S phase, including centromeres and origins of replication. We also discovered [Cdc7][2] binding throughout the genome, which may be necessary to initiate other cellular processes, including meiotic recombination and translesion synthesis. A kinase dead [Cdc7][2] point mutation increases the Ty5 retrotransposon integration efficiency and a 55-amino acid C-terminal truncation of [Cdc7][2], unable to bind [Dbf4][1], reduces [Cdc7][2] binding suggesting a requirement for [Dbf4][1] to stabilize [Cdc7][2] on chromatin during S phase. Chromatin immunoprecipitation demonstrates that [Cdc7][2] binding near specific origins changes during S phase. Our results suggest a model where [Cdc7][2] is loosely bound to chromatin during G1. At the G1/S transition, [Cdc7][2] binding to chromatin is increased and stabilized, preferentially at sites that may become origins, in order to carry out a variety of cellular processes.

 [1]: http://www.yeastgenome.org/locus/S000002459/overview
 [2]: http://www.yeastgenome.org/locus/S000002175/overview" />
<meta name="DC.Contributor" content="Daniel Rossbach" />
<meta name="DC.Contributor" content="D. Suzi Bryan" />
<meta name="DC.Contributor" content="Jay R. Hesselberth" />
<meta name="DC.Contributor" content="Robert Sclafani" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;DDK, a conserved serine-threonine protein kinase composed of a regulatory subunit, Dbf4, and a catalytic subunit, Cdc7, is essential for DNA replication initiation during S phase of the cell cycle through MCM2-7 helicase phosphorylation. The biological significance of DDK is well characterized, but the full mechanism of how DDK associates with substrates remains unclear. Cdc7 is bound to chromatin in the &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; genome throughout the cell cycle, but there is little empirical evidence as to specific Cdc7 binding locations. Using biochemical and genetic techniques, this study investigated the specific localization of Cdc7 on chromatin. The Calling Cards method, using Ty5 retrotransposons as a marker for DNA–protein binding, suggests Cdc7 kinase is preferentially bound to genomic DNA known to replicate early in S phase, including centromeres and origins of replication. We also discovered Cdc7 binding throughout the genome, which may be necessary to initiate other cellular processes, including meiotic recombination and translesion synthesis. A kinase dead Cdc7 point mutation increases the Ty5 retrotransposon integration efficiency and a 55-amino acid C-terminal truncation of Cdc7, unable to bind Dbf4, reduces Cdc7 binding suggesting a requirement for Dbf4 to stabilize Cdc7 on chromatin during S phase. Chromatin immunoprecipitation demonstrates that Cdc7 binding near specific origins changes during S phase. Our results suggest a model where Cdc7 is loosely bound to chromatin during G&lt;sub&gt;1&lt;/sub&gt;. At the G&lt;sub&gt;1&lt;/sub&gt;/S transition, Cdc7 binding to chromatin is increased and stabilized, preferentially at sites that may become origins, in order to carry out a variety of cellular processes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3757" />
<meta name="citation_id" content="7/11/3757" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3757" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3757.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3757.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3757.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300223" />
<meta name="citation_pmid" content="28924058" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3757" />
<meta name="citation_lastpage" content="3774" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel Rossbach" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045" />
<meta name="citation_author" content="D. Suzi Bryan" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045" />
<meta name="citation_author" content="Jay R. Hesselberth" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045" />
<meta name="citation_author" content="Robert Sclafani" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045" />
<meta name="citation_author_email" content="robert.sclafani@ucdenver.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1100-6489" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=G. M. Alvino;citation_author=D. Collingwood;citation_author=J. M. Murphy;citation_author=J. Delrow;citation_author=B. J. Brewer;citation_title=Replication in hydroxyurea: it’s a matter of time.;citation_volume=27;citation_year=2007;citation_pmid=17636020;citation_doi=10.1128/MCB.00719-07" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=O. M. Aparicio;citation_author=D. M. Weinstein;citation_author=S. P. Bell;citation_title=Components and dynamics of DNA replication complexes in S. cerevisiae: redistribution of MCM proteins and Cdc45p during S phase.;citation_pages=59-69;citation_volume=91;citation_year=1997;citation_issue=1;citation_pmid=9335335;citation_doi=10.1016/S0092-8674(01)80009-X" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Biochem.;citation_author=S. P. Bell;citation_author=A. Dutta;citation_title=DNA replication in eukaryotic cells.;citation_volume=71;citation_year=2002;citation_pmid=12045100;citation_doi=10.1146/annurev.biochem.71.110601.135425" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. N. Brandao;citation_author=R. Ferguson;citation_author=I. Santoro;citation_author=S. Jinks-Robertson;citation_author=R. A. Sclafani;citation_title=The role of Dbf4-dependent protein kinase in DNA polymerase ζ-dependent mutagenesis in Saccharomyces cerevisiae.;citation_volume=197;citation_year=2014;citation_pmid=24875188;citation_doi=10.1534/genetics.114.165308" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=L. Cheng;citation_author=T. Collyer;citation_author=C. F. Hardy;citation_title=Cell Cycle Regulation of DNA Replication Initiator Factor Dbf4p;citation_pages=4270-4278;citation_volume=19;citation_year=1999;citation_issue=6;citation_pmid=10330168;citation_doi=10.1128/MCB.19.6.4270" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. J. Dowell;citation_author=P. Romanowski;citation_author=J. F. X. Diffley;citation_title=Interaction of Dbf4, the Cdc7 protein kinase regulatory subunit, with yeast replication origins in vivo;citation_pages=1243-1246;citation_volume=265;citation_year=1994;citation_issue=5176;citation_pmid=8066465;citation_doi=10.1126/science.8066465" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. F. Ferreira;citation_author=C. Santocanale;citation_author=L. S. Drury;citation_author=J. F. Diffley;citation_title=Dbf4p, an Essential S Phase-Promoting Factor, Is Targeted for Degradation by the Anaphase-Promoting Complex;citation_pages=242-248;citation_volume=20;citation_year=2000;citation_issue=1;citation_pmid=10594027;citation_doi=10.1128/MCB.20.1.242-248.2000" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. L. Forsburg;citation_title=Eukaryotic MCM Proteins: Beyond Replication Initiation;citation_pages=109-131;citation_volume=68;citation_year=2004;citation_issue=1;citation_pmid=15007098;citation_doi=10.1128/MMBR.68.1.109-131.2004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Frigola;citation_author=D. Remus;citation_author=A. Mehanna;citation_author=J. F. Diffley;citation_title=ATPase-dependent quality control of DNA replication origin licensing.;citation_pages=339-343;citation_volume=495;citation_year=2013;citation_pmid=23474987;citation_doi=10.1038/nature11920" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=C. F. Hardy;citation_title=Identification of Cdc45p, an essential factor required for DNA replication.;citation_pages=239-246;citation_volume=187;citation_year=1997;citation_issue=2;citation_pmid=9099887;citation_doi=10.1016/S0378-1119(96)00761-5" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. F. Hardy;citation_author=A. Pautz;citation_title=A novel role for Cdc5p in DNA replication;citation_pages=6775-6782;citation_volume=16;citation_year=1996;citation_issue=12;citation_pmid=8943332;citation_doi=10.1128/MCB.16.12.6775" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. F. J. Hardy;citation_author=O. Dryga;citation_author=S. Seematter;citation_author=P. M. B. Pahl;citation_author=R. A. Sclafani;citation_title=mcm5/cdc46-bob1 bypasses the requirement for the S phase activator Cdc7p;citation_pages=3151-3155;citation_volume=94;citation_year=1997;citation_issue=7;citation_pmid=9096361;citation_doi=10.1073/pnas.94.7.3151" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=L. H. Hartwell;citation_title=Genetic control of the cell division cycle in yeast. II. Genes controlling DNA replication and its initiation.;citation_pages=183-194;citation_volume=59;citation_year=1971;citation_issue=1;citation_pmid=5283752;citation_doi=10.1016/0022-2836(71)90420-7" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. C. Heller;citation_author=S. Kang;citation_author=W. M. Lam;citation_author=S. Chen;citation_author=C. S. Chan;citation_title=Eukaryotic origin-dependent DNA replication in vitro reveals sequential action of DDK and S-CDK kinases.;citation_pages=80-91;citation_volume=146;citation_year=2011;citation_issue=1;citation_pmid=21729781;citation_doi=10.1016/j.cell.2011.06.012" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=G. F. Hess;citation_author=R. F. Drong;citation_author=K. L. Weiland;citation_author=J. L. Sligthom;citation_author=R. A. Sclafani;citation_title=A human homolog of the yeast CDC7 gene is overexpressed in some tumors and transformed cell lines.;citation_pages=133-140;citation_volume=211;citation_year=1998;citation_issue=1;citation_pmid=9573348;citation_doi=10.1016/S0378-1119(98)00094-8" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. E. Hollingsworth;citation_author=R. A. Sclafani;citation_title=DNA metabolism gene CDC7 from yeast encodes a serine (threonine) protein kinase.;citation_pages=6272-6276;citation_volume=87;citation_year=1990;citation_issue=16;citation_pmid=2166954;citation_doi=10.1073/pnas.87.16.6272" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. E. Hollingsworth;citation_author=R. M. Ostroff;citation_author=M. B. Klein;citation_author=L. A. Niswander;citation_author=R. A. Sclafani;citation_title=Molecular Genetic Studies of the Cdc7 Protein Kinase and Induced Mutagenesis in Yeast;citation_pages=53-62;citation_volume=132;citation_year=1992;citation_issue=1;citation_pmid=1398063" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=T. M. Holzen;citation_author=R. Sclafani;citation_title=Genetic interaction of RAD53 protein kinase with histones is important for DNA replication.;citation_pages=4735-4747;citation_volume=9;citation_year=2010;citation_issue=23;citation_pmid=21099362;citation_doi=10.4161/cc.9.23.14091" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=S. Hughes;citation_author=F. Elustondo;citation_author=A. Di Fonzo;citation_author=F. G. Leroux;citation_author=A. C. Wong;citation_title=Crystal structure of human CDC7 kinase in complex with its activator DBF4.;citation_pages=1101-1107;citation_volume=19;citation_year=2012;citation_issue=11;citation_pmid=23064647;citation_doi=10.1038/nsmb.2404" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. L. Jackson;citation_author=P. M. B. Pahl;citation_author=K. Harrison;citation_author=J. Rosamond;citation_author=R. A. Sclafani;citation_title=Cell cycle regulation of the yeast Cdc7 protein kinase by association with the Dbf4 protein.;citation_pages=2899-2908;citation_volume=13;citation_year=1993;citation_issue=5;citation_pmid=8474449;citation_doi=10.1128/MCB.13.5.2899" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=L. H. Johnston;citation_author=H. Masai;citation_author=A. Sugino;citation_title=First the CDKs, now the DDKs.;citation_pages=249-252;citation_volume=9;citation_year=1999;citation_issue=7;citation_pmid=10370238;citation_doi=10.1016/S0962-8924(99)01586-X" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Hum. Genet.;citation_author=D. Karolchik;citation_author=A. S. Hinrichs;citation_author=W. J. Kent;citation_title=The UCSC genome browser.;citation_pages=Unit18.6;citation_volume=Chapter 18;citation_year=2011;citation_pmid=21975940" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=Y. Katou;citation_author=K. Kaneshiro;citation_author=H. Aburatani;citation_author=K. Shirahige;citation_title=Genomic approach for the understanding of dynamic aspect of chromosome behavior.;citation_pages=389-410;citation_volume=409;citation_year=2006;citation_pmid=16793414;citation_doi=10.1016/S0076-6879(05)09023-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. R. Knighton;citation_author=J. Zheng;citation_author=L. F. Ten Eyck;citation_author=A. Ashford;citation_author=N. Xuong;citation_title=Crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase;citation_pages=407-414;citation_volume=253;citation_year=1991;citation_issue=5018;citation_pmid=1862342;citation_doi=10.1126/science.1862342" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=I. Kogut;citation_author=J. Wang;citation_author=V. Guacci;citation_author=R. K. Mistry;citation_author=P. C. Megee;citation_title=The Scc2/Scc4 cohesin loader determines the distribution of cohesin on budding yeast chromosomes;citation_pages=2345-2357;citation_volume=23;citation_year=2009;citation_issue=19;citation_pmid=19797771;citation_doi=10.1101/gad.1819409" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=K. Labib;citation_title=How do Cdc7 and cyclin-dependent kinases trigger the initiation of chromosome replication in eukaryotic cells?;citation_pages=1208-1219;citation_volume=24;citation_year=2010;citation_issue=12;citation_pmid=20551170;citation_doi=10.1101/gad.1933010" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. Lei;citation_author=Y. Kawasaki;citation_author=M. R. Young;citation_author=M. Kihara;citation_author=A. Sugino;citation_title=Mcm2 is a target of regulation by Cdc7-Dbf4 during the initiation of DNA synthesis;citation_pages=3365-3374;citation_volume=11;citation_year=1997;citation_issue=24;citation_pmid=9407029;citation_doi=10.1101/gad.11.24.3365" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. A. Muller;citation_author=M. Hawkins;citation_author=R. Retkute;citation_author=S. Malla;citation_author=R. Wilson;citation_title=The dynamics of genome replication using deep sequencing.;citation_volume=42;citation_year=2014;citation_pmid=24089142;citation_doi=10.1093/nar/gkt878" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=T. Natsume;citation_author=C. A. Muller;citation_author=Y. Katou;citation_author=R. Retkute;citation_author=M. Gierlinski;citation_title=Kinetochores coordinate pericentromeric cohesion and early DNA replication by cdc7-dbf4 kinase recruitment.;citation_pages=661-674;citation_volume=50;citation_year=2013;citation_pmid=23746350;citation_doi=10.1016/j.molcel.2013.05.011" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. Nougarede;citation_author=F. Della Seta;citation_author=P. Zarzov;citation_author=E. Schwob;citation_title=Hierarchy of S-Phase-Promoting Factors: Yeast Dbf4-Cdc7 Kinase Requires Prior S-Phase Cyclin-Dependent Kinase Activation;citation_pages=3795-3806;citation_volume=20;citation_year=2000;citation_issue=11;citation_pmid=10805723;citation_doi=10.1128/MCB.20.11.3795-3806.2000" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=G. Oshiro;citation_author=J. C. Owens;citation_author=Y. Shellman;citation_author=R. A. Sclafani;citation_author=J. J. Li;citation_title=Cell Cycle Control of Cdc7p Kinase Activity through Regulation of Dbf4p Stability;citation_pages=4888-4896;citation_volume=19;citation_year=1999;citation_issue=7;citation_pmid=10373538;citation_doi=10.1128/MCB.19.7.4888" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=P. K. Patel;citation_author=N. Kommajosyula;citation_author=A. Rosebrock;citation_author=A. Bensimon;citation_author=J. Leatherwood;citation_title=The Hsk1(Cdc7) replication kinase regulates origin efficiency.;citation_volume=19;citation_year=2008;citation_pmid=18799612;citation_doi=10.1091/mbc.E08-06-0645" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Patterson;citation_author=R. A. Sclafani;citation_author=W. L. Fangman;citation_author=J. Rosamond;citation_title=Molecular characterization of cell cycle gene CDC7 from Saccharomyces cerevisiae.;citation_pages=1590-1598;citation_volume=6;citation_year=1986;citation_issue=5;citation_pmid=3537706;citation_doi=10.1128/MCB.6.5.1590" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=L. N. Princz;citation_author=P. Wild;citation_author=J. Bittmann;citation_author=F. J. Aguado;citation_author=M. G. Blanco;citation_title=Dbf4-dependent kinase and the Rtt107 scaffold promote Mus81-Mms4 resolvase activation during mitosis.;citation_volume=36;citation_year=2017;citation_pmid=28096179;citation_doi=10.15252/embj.201694831" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. K. Raghuraman;citation_author=E. A. Winzeler;citation_author=D. Collingwood;citation_author=S. Hunt;citation_author=L. Wodicka;citation_title=Replication Dynamics of the Yeast Genome;citation_pages=115-121;citation_volume=294;citation_year=2001;citation_issue=5540;citation_pmid=11588253;citation_doi=10.1126/science.294.5540.115" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=C. J. Ramey;citation_author=R. A. Sclafani;citation_title=Functional conservation of the pre-sensor one beta-finger hairpin (PS1-hp) structures in mini-chromosome maintenance proteins of Saccharomyces cerevisiae and archaea.;citation_pages=1319-1326;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. C. Randell;citation_author=A. Fan;citation_author=C. Chan;citation_author=L. I. Francis;citation_author=R. C. Heller;citation_title=Mec1 is one of multiple kinases that prime the Mcm2-7 helicase for phosphorylation by Cdc7.;citation_pages=353-363;citation_volume=40;citation_year=2010;citation_issue=3;citation_pmid=21070963;citation_doi=10.1016/j.molcel.2010.10.017" />
<meta name="citation_reference" content="citation_title=The Initiation of DNA Replication in Eukaryotes;citation_title=Role of DDK in replication initiation;citation_author=D. S. Rossbach;citation_author=R. A. Sclafani;citation_pages=279-296;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. Sasanuma;citation_author=K. Hirota;citation_author=T. Fukuda;citation_author=N. Kakusho;citation_author=K. Kugou;citation_title=Cdc7-dependent phosphorylation of Mer2 facilitates initiation of yeast meiotic recombination;citation_pages=398-410;citation_volume=22;citation_year=2008;citation_issue=3;citation_pmid=18245451;citation_doi=10.1101/gad.1626608" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=N. Sato;citation_author=K.-I. Arai;citation_author=H. Masai;citation_title=Human and Xenopus cDNAs encoding budding yeast Cdc7-related kinases: in vitro phosphorylation of MCM subunits by a putative human homologue of Cdc7;citation_pages=4340-4351;citation_volume=16;citation_year=1997;citation_issue=14;citation_pmid=9250678;citation_doi=10.1093/emboj/16.14.4340" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. A. Sclafani;citation_author=T. M. Holzen;citation_title=Cell cycle regulation of DNA replication.;citation_pages=237-280;citation_volume=41;citation_year=2007;citation_pmid=17630848;citation_doi=10.1146/annurev.genet.41.110306.130308" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y.-J. Sheu;citation_author=B. Stillman;citation_title=The Dbf4-Cdc7 kinase promotes S phase by alleviating an inhibitory activity in Mcm4.;citation_pages=113-117;citation_volume=463;citation_year=2010;citation_issue=7277;citation_pmid=20054399;citation_doi=10.1038/nature08647" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=L. Wan;citation_author=H. Niu;citation_author=B. Futcher;citation_author=C. Zhang;citation_author=K. M. Shokat;citation_title=Cdc28-Clb5 (CDK-S) and Cdc7-Dbf4 (DDK) collaborate to initiate meiotic recombination in yeast;citation_pages=386-397;citation_volume=22;citation_year=2008;citation_issue=3;citation_pmid=18245450;citation_doi=10.1101/gad.1626408" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Wang;citation_author=M. Johnston;citation_author=R. D. Mitra;citation_title=Calling cards for DNA-binding proteins.;citation_volume=17;citation_year=2007;citation_pmid=17623806;citation_doi=10.1101/gr.6510207" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=H. Wang;citation_author=M. E. Heinz;citation_author=S. D. Crosby;citation_author=M. Johnston;citation_author=R. D. Mitra;citation_title='Calling Cards' method for high-throughput identification of targets of yeast DNA-binding proteins.;citation_pages=1569-1577;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18802438;citation_doi=10.1038/nprot.2008.148" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Wang;citation_author=D. Mayhew;citation_author=X. Chen;citation_author=M. Johnston;citation_author=R. D. Mitra;citation_title=Calling Cards enable multiplexed identification of the genomic targets of DNA-binding proteins.;citation_volume=21;citation_year=2011;citation_pmid=21471402;citation_doi=10.1101/gr.114850.110" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=M. Weinreich;citation_author=B. Stillman;citation_title=Cdc7p-Dbf4p kinase binds to chromatin during S phase and is regulated by both the APC and the RAD53 checkpoint pathway;citation_pages=5334-5346;citation_volume=18;citation_year=1999;citation_issue=19;citation_pmid=10508166;citation_doi=10.1093/emboj/18.19.5334" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=X. Wu;citation_author=S. J. Noh;citation_author=G. Zhou;citation_author=J. E. Dixon;citation_author=K.-L. Guan;citation_title=Selective Activation of MEK1 but Not MEK2 by A-Raf from Epidermal Growth Factor-stimulated Hela Cells;citation_pages=3265-3271;citation_volume=271;citation_year=1996;citation_issue=6;citation_pmid=8621729;citation_doi=10.1074/jbc.271.6.3265" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=W. Xie;citation_author=X. Gai;citation_author=Y. Zhu;citation_author=D. C. Zappulla;citation_author=R. Sternglanz;citation_title=Targeting of the Yeast Ty5 Retrotransposon to Silent Chromatin Is Mediated by Interactions between Integrase and Sir4p;citation_pages=6606-6614;citation_volume=21;citation_year=2001;citation_issue=19;citation_pmid=11533248;citation_doi=10.1128/MCB.21.19.6606-6614.2001" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=W. Xu;citation_author=J. G. Aparicio;citation_author=O. M. Aparicio;citation_author=S. Tavare;citation_title=Genome-wide mapping of ORC and Mcm2p binding sites on tiling arrays and identification of essential ARS consensus sequences in S. cerevisiae.;citation_pages=276-276;citation_volume=7;citation_year=2006;citation_pmid=17067396;citation_doi=10.1186/1471-2164-7-276" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=N. Yabuki;citation_author=H. Terashima;citation_author=K. Kitada;citation_title=Mapping of early firing origins on a replication profile of budding yeast;citation_pages=781-789;citation_volume=7;citation_year=2002;citation_issue=8;citation_pmid=12167157;citation_doi=10.1046/j.1365-2443.2002.00559.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=S. C. Yang;citation_author=N. Rhind;citation_author=J. Bechhoefer;citation_title=Modeling genome-wide replication kinetics reveals a mechanism for regulation of replication timing;citation_pages=404-404;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=20739926;citation_doi=10.1038/msb.2010.61" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="DDK, a conserved serine-threonine protein kinase composed of a regulatory subunit, [Dbf4][1], and a catalytic subunit, [Cdc7][2], is essential for DNA replication initiation during S phase of the cell cycle through MCM2-7 helicase phosphorylation. The biological significance of DDK is well characterized, but the full mechanism of how DDK associates with substrates remains unclear. [Cdc7][2] is bound to chromatin in the Saccharomyces cerevisiae genome throughout the cell cycle, but there is little empirical evidence as to specific [Cdc7][2] binding locations. Using biochemical and genetic techniques, this study investigated the specific localization of [Cdc7][2] on chromatin. The Calling Cards method, using Ty5 retrotransposons as a marker for DNA–protein binding, suggests [Cdc7][2] kinase is preferentially bound to genomic DNA known to replicate early in S phase, including centromeres and origins of replication. We also discovered [Cdc7][2] binding throughout the genome, which may be necessary to initiate other cellular processes, including meiotic recombination and translesion synthesis. A kinase dead [Cdc7][2] point mutation increases the Ty5 retrotransposon integration efficiency and a 55-amino acid C-terminal truncation of [Cdc7][2], unable to bind [Dbf4][1], reduces [Cdc7][2] binding suggesting a requirement for [Dbf4][1] to stabilize [Cdc7][2] on chromatin during S phase. Chromatin immunoprecipitation demonstrates that [Cdc7][2] binding near specific origins changes during S phase. Our results suggest a model where [Cdc7][2] is loosely bound to chromatin during G1. At the G1/S transition, [Cdc7][2] binding to chromatin is increased and stabilized, preferentially at sites that may become origins, in order to carry out a variety of cellular processes.

 [1]: http://www.yeastgenome.org/locus/S000002459/overview
 [2]: http://www.yeastgenome.org/locus/S000002175/overview" />
<meta name="og-title" property="og:title" content="Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3757" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="DDK, a conserved serine-threonine protein kinase composed of a regulatory subunit, [Dbf4][1], and a catalytic subunit, [Cdc7][2], is essential for DNA replication initiation during S phase of the cell cycle through MCM2-7 helicase phosphorylation. The biological significance of DDK is well characterized, but the full mechanism of how DDK associates with substrates remains unclear. [Cdc7][2] is bound to chromatin in the Saccharomyces cerevisiae genome throughout the cell cycle, but there is little empirical evidence as to specific [Cdc7][2] binding locations. Using biochemical and genetic techniques, this study investigated the specific localization of [Cdc7][2] on chromatin. The Calling Cards method, using Ty5 retrotransposons as a marker for DNA–protein binding, suggests [Cdc7][2] kinase is preferentially bound to genomic DNA known to replicate early in S phase, including centromeres and origins of replication. We also discovered [Cdc7][2] binding throughout the genome, which may be necessary to initiate other cellular processes, including meiotic recombination and translesion synthesis. A kinase dead [Cdc7][2] point mutation increases the Ty5 retrotransposon integration efficiency and a 55-amino acid C-terminal truncation of [Cdc7][2], unable to bind [Dbf4][1], reduces [Cdc7][2] binding suggesting a requirement for [Dbf4][1] to stabilize [Cdc7][2] on chromatin during S phase. Chromatin immunoprecipitation demonstrates that [Cdc7][2] binding near specific origins changes during S phase. Our results suggest a model where [Cdc7][2] is loosely bound to chromatin during G1. At the G1/S transition, [Cdc7][2] binding to chromatin is increased and stabilized, preferentially at sites that may become origins, in order to carry out a variety of cellular processes.

 [1]: http://www.yeastgenome.org/locus/S000002459/overview
 [2]: http://www.yeastgenome.org/locus/S000002175/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gOCg3G8ElRE0CKgdXUUHm8xpjJsTgevtbbHe6uaFIWM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487101","apath":"\/ggg\/7\/11\/3757.atom","pisa":"ggg;7\/11\/3757","ac":{"\/ggg\/7\/11\/3757.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3757.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3757"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3757"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3757.full":true,"\/content\/7\/11\/3757":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022E9iQyo3y6XOpzntkieuwNIAARh3yfkUYPtvpti6Ajq9vKkxYR46oOc4NMju6+wGdhyn2th1yXIDQhztTlGqLy7fNNaGQpvUH1KU3jT\\\\\\\/rqi9ZYlKMq\\\\\\\/vEAAL8HgqW\\\\\\\/LbkSjcog8OVYNg47LSimIGvDmjavQjxlRwcISeWpwJg26zhgb8dtPlduvm+nz\\\\\\\/XtBdFwqXcVTRC3qyc7SiK3TS4xo8URAVPeELXDIqKZJvHrAc1Fm7D25AugMDjpeJQg2nbBa5gcjFrCvjd2gcWYT5w2LQ1x0du0l0NW6pCnnh52\\\\\\\/w88OuoUfrzqfOgNOmpcfziiMAeEb0pbr4YOyME7oUsfdYn0LvnOw\\\\\\\/MnsTNiK4LkPAjM+3\\\\\\\/fSlqgYY0QDmV+BchXM3eQn\\\\\\\/7\\\\\\\/9bIEDuzOcm\\\\\\\/BMpzN4kyb0eVDDarBRNhepYVfuGrF+xgpqScVfBmj0IEx7JkPlE+gC4pXTpJ7SYqzUs9tdRhQXVqrFvJ4pL4lZ3OxggWXOpILqqA2abHoUWFOfesyrmP6JqDVVTa1GGI6NjElf7klwpJQdHu4+XneHwWVjwYpkcz6KJ6fK9BbyYMC5Vaa8cMuHvgj7tfKAjaACRuINDTXxmH\\\\\\\/VMIOO+FVTZ4Bn\\\\\\\/lhG1tVQMB\\\\\\\/tsXxLxtZzCrBEDlVCo+gKALSyqVSrKeThAk02ZfcVZ6SEtg9jyC0Sop69nxl9NNAt0TtaHCR1clAUajRzlzAOt68Sy16mt\\\\\\\/4d\\\\\\\/k12GMyoydjQgJytM01dcHKPh\\\\\\\/PetOh5V4LV2J3ChPNuHvvq\\\\\\\/uvf1kXDmdpqbvynVuPcNZ3bCmahVUT94j9WRU2lZE\\\\\\\/h5D44750iXS2bHWznz1qXRG6yG7yWVdaXT5bbi1VJajBFr\\\\\\\/BUQzvSpoq7ZAaM\\\\\\\/4Lu0qjJ0vVNuVP5t1v00da3noLbgRZqCs6HFaJKdOP+GuFxYD0il+DRrWdhVXYT7NpnjMTRvhlLuwaZeWyKq8PKtSOkzQUgkLwO3TeHmWLTUSZ7oVpzhrQxo+3ISHHYEWrldb2Z+ZUrLvoRpU4fjjwSFHPH5zK9qZhFA\\\\\\\/q2M7s49KFgvt0RZPlc1xsOjIbaS5\\\\\\\/ZSstmErPH61FCurwJS43B3JCFBwzz\\\\\\\/ffpWN2zvk0aTTtFFVMMX0ecyIMJ8Dk40newJ6jKIn2l3BP2XfNXeU8wxd9ZuACSyPjuaO1UM5I1zi9Ah5JpK4p8bQFVvuJ4fcoyo7PrNRwqPHMZ\\\\\\\/Di0XqBdFfjmQgIL1PwIUKCTyzN4AJQ4iInjbGV3fEjfVp3jomty96B6ra32ti\\\u0022,\\\u0022iv\\\u0022:\\\u0022oKO1koUKiQ1Ic2KJ8xqGTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ce95799796d507aa3b82e01c56ffd6a55ff14950e8e574eb64baaa8c1a042578\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hLiO7vMUjHN7AYsjKkZKlmdeZBGQiUnnKU78wvzr8WoRPHdeSMGh8\\\\\\\/gK7rtoD17rB0S3F8MyTrt0L\\\\\\\/Fnc\\\\\\\/9kZILfVhidt62vaYV8Zw6Er1x5qw8fHfOKmHrYYVuxBf3nrttfhlXq2tMyWX6f5KCjiglY1sXA6iSFMdqGtc\\\\\\\/IHVHTmKi+EhHm5pCXusV3FPuZEM\\\\\\\/aDVD8s2pMchn4dsA86ayh\\\\\\\/YF5kc36aKCXxwvFZ284+CiW1zCPSghkdvtOjtC4HSARfxXE3wNHtIx3n1cxYmX5W+71ikIv9Bz1mmIOzJZ\\\\\\\/Q8W\\\\\\\/cwp11cFjvhp\\\\\\\/sf9gFyIngUDmdFMvHkWuN1ACRkYDCrP8rig8jcKuHhglp5PhpufRgV+PPQCVApKOQycwvhPSykTYbFTy1PC1dnrnksFWgE15VpwGkDTQ\\\\\\\/1Fq682XLq3EMwY10L5er9FUmxOczd0zcOOEzkXz\\\\\\\/raqAxHDZhQIR0OocDGZL1nkGah4EtFAsdqVRUJy6G25BABA4vwb8qjEVm8ukAdrWlt5iiKcuP3Rk1\\\\\\\/kHQ7GbA\\\\\\\/tjIS4VnSf4n55V\\\\\\\/foLgdjN2UU9htRN33icBD42b2ae\\\\\\\/7Vxu3YlY2artbiiojX0NxUMneh4uAuvt3k4z4iDD65Uyy7fTw\\\\\\\/SrOBEyj9kfs6huu0VwsBG1VlVISW0W55WF39\\\\\\\/f\\\\\\\/2jczxe4ZA35iEHujChYSZFHYX2KzBaTzHqpgD2X7X\\\\\\\/RnZuk5m5xMo7\\\\\\\/P3bbHSmqeGutJbBGw1UmGeQlmRo2gKJFdOdWzhmrKGDxwozCCX+9\\\\\\\/\\\\\\\/p179yVaCtTcNM+nQ\\\\\\\/zWDlc0PpEpUXLN2dONqW9xUVTTNLdG4j70qav3lEZFr0DJjEFctaHdnQ2\\\\\\\/KODnBZMNSb+fLlgc5iH7qcJQWfXyAcRpm0sn7qL9GaVm1E+KhJUCbhF2+52bYb0myYbYGyK\\\\\\\/E0s1P2gs9PcXP2isrWmM6WAc4QN436kOeWNqqoPGYqkeomZnc\\\\\\\/cGY\\\\\\\/h1PO1fYAQzL2cIXlC7Elzx6gkJiJontpTuxxSZF5mjhzbu3jYS5\\\\\\\/TJUPS4GuhK8ESyfL35l32\\\\\\\/yrhQXxkLVdxoFcyGyzGDLcKje\\\\\\\/iWlHlYUDaHOiISXeC4yT36rcvIhsouW8fYXuQCoNDvOj2oF3a5gk9ql6Sj8xG7FtfwWJ3jel9atcUC6P6sAznGNnU6D9OkmOVXqq7JDH6EK6gcMuKuMdkYUhlHP6e8hejj7l8uBzqY7C5FDvI9dpv402S4slVb80yIPc8r5Pv9wX6Qf0Sbk1Nv6WnbBTfDecWIYvvYg40T3aMu1XXg57uxKZVtOZ4nF5A0pzh15u\\\\\\\/sV+cMjgfWues\\\\\\\/GNB+EWvpUCGvvZdCGlAe9ci5djw4nSmuferudzb5IvnihQBQz0gUT+Gn+cSErAm87FupqyuCnrPjC\\\\\\\/pgMY+pi3RClV4LOmDIqcwfFrpSgY5YDxiopd\\\\\\\/q+Yj8gryqyegeIQB6CGJBLXQmZtVxgGWHHzUN\\\\\\\/O8Dx9cIMaC4tv09VqGro3k4dK0cVLuUHuJ2V\\\\\\\/2muwozu9ssrRKbnY4\\\\\\\/I\\\\\\\/1uva5+r6+k24BypAafTx5kUwTsr0SVnAOvHCSmNjHms\\\\\\\/uL+uWRZq9oKNgBkTrpuzwVZsLfT+9Yc9furrSG5PiZV0m6SVdh+3XEErjwohqr069X35\\\\\\\/XK7xAv8riOiJEqJ4lxGcFP2BVYLyMHXbFA9X8u+W2rw+a2Ullt7t9t\\\\\\\/7mlamPPXLynqLLmLQgtbPBgxUNP2F3ofNTY14xz3haRqnhyBDllSx9lRoRnRu3ItSUKLXTcRe\\\\\\\/TdlN14HyWTC6EvS+2H+XWeHR1oGs7rSKjIgMK0BLYaLulDsZZx50p7xZuh5iXsaNqCKt4xnXqJacsTrAdCZjrOAZrdqSRIlh8m8rTMLfVumOGEBxkjRYwocyg4hxyItzBgnlpxWczpnye1kUsqy62RwYC3z56Eacrtt7Nd9iJKs\\\\\\\/241Ujf3f9f2BN\\\\\\\/pSmnjht+ZFRElddnBYJ6UJk1dBxCEjbcCAndK29+B4sWsMzrziIt4jagMTYr\\\\\\\/T9dqWEnAql0\\\\\\\/vY3s3qsxMNb4HtgUIIra53VwdwUpVAJkNL9q7DjzEQIe4Q8HWQJJv4Hii99t0xNNCaZCpVL3HkiXtQlrq3dyJlYGzmCz1w3ZamXmhy+hddXgJcCokx6mt4HCb0Q1MIGKfCkWs59fD\\\\\\\/3kLKe+CNo8McsVq16Tw9m2p7lktuDrKFcPxEb6qVn2ztj+W8nJN49yVrWIh1\\\\\\\/88fkTemMpuXXLd53FaqlGMIJZPYR+EG2u1KHuP8ADNjIg+En7qb3pTV0JcdM3E4I1Skmk2438R+rPdJrv5miNuHaFBf2FfiJIacxErx9Ux5LUeWkrJ4kxwwWTsNrLOGhe\\\\\\\/PvEUOG\\\\\\\/Mmpb8QdoTKzAynYrv4j4yJYzfPPzl+2wesHgj22\\\\\\\/6vhXTnMzs4RRByEuyJ6PPrH152DmvSpWHgQBxNlqbqtPexRUwQ6sHiTvg1dNEhvP5qo893TPxxWOrY6AFLMSqM0W9VYfZSsWLu43dvc56hAPUThWQ1fl5Z7Hxuyk5wg1ILVyFGuoyiWSo8DrmlLHaXfIdiga4OsZlWDewVVIp08lOCMP2KOPq5zXy7AXjyjhDfUi68pquxQA7KIXt8G38PB27k\\\\\\\/G2YYonbfqMnoci6gxrmii4YsE1hTIUoQfhq3AL969YOZlwhbO1vSnlN7ThqjsMe0FduyaewoyFZ0hWyzav\\\\\\\/MbMGv\\\\\\\/AeR\\\\\\\/MooB1L3YSTFNrBYb\\\\\\\/HZEQxu9VyVviFFn5GTlC+gxVwivmBBKSqhofbKjgw8manZqyrfRPSVZyfZVdFd8wPglpHvQyMYbNepH6ASIv7xazVXRqBtQejw4y8wwdqMUmVgiyLz1jRblV7jWuW\\\\\\\/8DTpy697qUhXza4\\\\\\\/qyzM+YqQMRCja\\\\\\\/p33P+rV3v1mOI+QHPQ4qEO30TIlR0wiIyS8O8P+EkJnuEKZMMoqFwRKgsIjj4\\\\\\\/d0\\\\\\\/1zizowRrXpttbly88MfjHDBjiq3env927iAKrMdoug2N0oBN0rU+0MzPjStvFy+\\\\\\\/JAYxrpt\\\\\\\/FeVodSHTNZgoU1ZNxXBqYDafK9OZfiQN21UAx4XLyNK\\\\\\\/8FcVcXZKlvEw5keZocSYlwIlpDSUlTSAdkoX07slyFUV5gtuQDpMT3up2roZebXv6g72KED9NJGGK4qrAwS7gmGdbSBGkRELIdTaHwFoiYKJOADVTh7QTi1cpTt2nrFZDRGzJGalllRrn25h00Qir8Y1mOE7I98hbGavFX4BaEiO6wJyMeQKBMIHDrmqD75\\\\\\\/NthNvL5R6\\\\\\\/GURGTBmAnDRF8gbea72kaxVlz4mIvbWnu4fMEJW+nMGv3AcJ75wykaqc17eXGNBWn8yMio5T+1sGR7U0zcJyQcpakTpxgzC42zGtkShLrnESTIUnMVRli2Svk+AFJPdGvSzSsVWJxH36+uOXXrwgA6pldEZqwOmHg+Exk+H+hEcAilJMrThKbQ0Vaz\\\\\\\/N\\\\\\\/W3iSMvioe3KJtsZ3hb9wfW+8dX\\\\\\\/epUSzL+oQ3Y7RI8rMR0nedG1wVtfBn2TZOcxCtV6joodewfKcF9jx9bHZTeX69KHQ5m6BpD0rPiCtZ7ktB2uXPUoXvTB1KOfXX0Ik8Pw1HHdfkOvW6q75A6J2hSIFBq\\\\\\\/NtBoV3R+bS6Mjx+WSGTiObmjxN9tVnqWwxrYXqtMCALvYsY5YFttVoNTlvrfK54bteCcJwCB+j6TayCTcmc9UonwRAccFXYoemAnbzOJyJa6Y5b+CFZtQXJfeMadbxc6lszZTnwHSjJ9E5JcC5o+1m8TUhjO9E\\\\\\\/bZu8vHzsICs1ENTA08Bpv0kRWS3he65ZroZo3RJQk7Nu5qk\\\\\\\/7WsKHeOQxZ20Q236Js7BzNTeAB5d6eGIvRh+v9qSgazYY4s+rWleZAyAZXSA7ehn2IKYQ7kJMktKk4ubh3v34iXgj9haOpCQSnr7Vfi44u+0mmd1GUz3Hqt3gn\\\\\\\/i+SsGyMCCHb92\\\\\\\/jyAq26rresvJykkRSjlEjacqpW50yM466YLW75gfI9QEMPBT27HgkMKxfnCysjRfyLmQrH\\\\\\\/1P0Qx3ahZM\\\\\\\/XzEHRDyfEkGWihsE5X2fukHXKuOt\\\\\\\/\\\\\\\/reny7K6M+CdwoD7vCIARD\\\\\\\/5Bkng6iSTLG3US4a3ueIWoo08RhrWIQX23nQm5eFzf2\\\\\\\/iNoGbWWMNQo5tIY9w34r+wXNQyCnh0+ywBNoKiZTMPMNoMDdLynICSdHAG\\\\\\\/IDq1j7yVC9n6QOeIXoB4zlyi2+aGaeXKTzDPiAa2ioN9gQZkO5UajCWhJKGVgFG7GSAFPOA+pKErJyQdRhaYHiljWEnht90oKCDOC5bolVdlcAF4pCnM7PHtKiqVHaXD80j9qX7n5OcPQzJZcx1JmefnHWmiYDhvPKONOX4GLk6gOu3v3GGhA6RbbDnY+7t7vcR5wf6aka5hoOSQkgNZjpreNP3qwGBIDerQtoNH4LqjhD6\\\\\\\/lp0\\\\\\\/LQTZzisruc5h0vPOullT41Afss5msAigrK5N1rh+6KgKaqjayoTUACS48UoB7Ulj2qUkljNljlkch4VpmUQS0wEhEILq3I+RajVmFyQ1gjgwa8mXVu0skqDHZucI3MS\\\\\\\/rnu1YFYjqc0nUPgCSCEpC8wjXvobt2tVVvKuT9wdEJTvCm8vluG8dXKoye5fubEkqvoQ8z\\\\\\\/m5ur7zgqMOYZdLXkadF27Go\\\\\\\/+ZH69L8PbibgNaQNjo9wsYmipTpRsBWEWeu2r5GyM5f+cFEF46jD5efO+wdtqJzOMaqa\\\\\\\/9aoCmM\\\\\\\/mwE34mNlO6QU4mQyC2pQoRj8MfjfxO89BoAp6z2Oze4\\\\\\\/NYLp8mgkw\\\\\\\/48bLP2JyWq28vD+i3vIX3DmBFVxNm84Pyp3b6ucjwzPLi7R\\\\\\\/KnaGBXRfClNjprWFvBmrzeQzvg8SNakY0Yr0IwmNQGfN3ejJvvGjvNgz72qi8VMHemhGx+sDObCi9AGBojCIW8rm1m\\\\\\\/WqI7DVc3acYblsgEju2lEAuYA\\\\\\\/ck+mFlCIrM\\\\\\\/86HXP70MOOKBbueAq8HkvV7CUmjJhw7JdDfex3SDswz+SBjeQl028I6FzbsblJ0mVnYj6LpyvNb2TYNHXhHQlcGuiMjJAQplZATWzc4XAIKtDMyji4VSD2+NMSAOStQKlj7GnqAVvywk7U9H6nDH9AxSaAG70g+QvNwMEG6kHefylKli34A0o8atw5+\\\\\\\/uLDKwSSMIIn0JvWNCUWcQQ6w3ISeSvO1D8xrjWJ6v1GhCZ+RnlqIcS4kiBRpirDewAhbhE9QuwYdGQeDGmAaByoT63L9smNf8+2FWhE+db8PqchfD3qHS2plQbj81kXPVpaeLvLJ8feITXKqskJbLwja3MM27dEnvWlGvftwX9bSlLYCSs7I6iK2Tq2YW+1ZK7BnWEMYuPiHxr0pfQ7vl8vbRdrnaQnni1X3O99YTIJm3MXa5DIAEgYk2pXDNKEnCwMfjVbTa9LHO7Uqu7ZakQ\\\\\\\/KfJ3jXCxgN8lBJ2mRa7PmbkzTjIMVfxq65eWCiTS4vpJtvusFYGL7m7NQnVl30DppCAxptJUUnCnCCswI\\\\\\\/yjNerX19SOKUQSqFNZX83zneK9MWqOsECY2OEWSLx\\\\\\\/pPfYKbx\\\\\\\/HtnZ+GJSSGK1tpgA0uyGzIg\\\\\\\/2+eNfzEdq3eCtFbRVoDvfevOYQILldzyUCBp086yQDMQMdeX1nKmN3mk3qlNVBE6VPSF3mtLHxUE6OHB\\\\\\\/D3H917LL8B3YhKUOnByK8cGKFk7+\\\\\\\/EdhCVc9p82v034rjT8Fn4mcNPYbltCdwhfEvs4G5RMvssbOpVgpNLD1cHGs6NkOyv6v1utn0bWmURxnfXq7qDlhhPrcSoCYs2bpGk1i9gKAboM9lvq7gEuo548pDhbzdrbassIVOwX8G0cZT\\\\\\\/69w7XMzl\\\\\\\/4MpfLB11WIltHfDTrC0lN\\\\\\\/6WgVDX+oRo4Ij4+4OtJrkxS88yX8X9iUh\\\\\\\/x\\\\\\\/QisFWjbEput97XtHb3IP0NOl2PztDVOQMQgBZuiS+B2a\\\\\\\/AHLfZiyGfe456GSe5Aq9tipt2Sig1A92FEW5V6Ihiu5c6pFpGLpgmTN\\\\\\\/s7HM64c56UpTjwnqo\\\\\\\/9R112EnY8JLNlrqVR86TXNLt1ra8voD5S\\\\\\\/HcefXE3qvehpunw\\\\\\\/owITvUBJeOCZ4WGntQoApGM\\\\\\\/DzYEh7\\\\\\\/+yMJM7LG1bneNMjQCKXgeLav598jgaExU9yY1uDpy4oc\\\\\\\/RxNaabYeJ7DenML2fk+Qi5tkMQqNuCNqIzvGMPKbBI0yp9\\\\\\\/mmO6KceoqBrezT2SKzRceofu\\\\\\\/zLJm1KzRbifq0BqKkFuL6X5Qs\\\\\\\/ieNQLJeqHbj7EEmK2h\\\\\\\/pn57Us5FmEPXX\\\\\\\/09e3UAQn5ID3CbP\\\\\\\/Im44c9BuvJu\\\\\\\/aK+tLSARHok9cue3lgKHErNqPFJjaq19APMFj3JpLv2ztFQNFeudjCNqYuSZEQ\\\\\\\/oyRXGlE0Phb3ftK6KuXR8V+uHkiGMGUbw4Hn0\\\\\\\/WrINSJCR3d\\\\\\\/kjYRSbzyKLW2VKYHuUtyZ8PqppUoBhNR8YA1p06yIY+kfV56t7JhYVC7i52cYozKflyBouhJnxi1iZThRtA\\\\\\\/FvL5KX5DO5Mv7h6W77EHzywiJWSL2nL2CQoGCfmY2ZoU5XGz+CM3UQbESYEW\\\\\\\/kz5GFXTfKziR+OxpJB7kMpDDiW8xSoRnu0Gqq6Mviy1UFXwId9cGpCSOcATI+Z186Gobr6wBsF2W1JZOJtmjQw7xpZQiavooXZ580cWIbp0Tza8u2ULZx7uYuoRmzFKflkUYFdkrogpI9KH\\\\\\\/93FrgMn0V5YUBLL2gcEKOc90YjfnUeIZ7dYqegasCRaLR\\\\\\\/PnUiLOhld8qF6gz\\\\\\\/DQoAw910wNIkybCvnUB5gjb5fPy8SvYdskZbR7t0NdnJNIeZGvkhQGe+lZTLtWPi9d9cX9SiWMf4W3tIo6ogW45ASXa+j\\\\\\\/8icXx\\\\\\\/3bFsJtLWsZEyAa95KioXnygfZZe0tkb4eP0+I+iuLaVGCwsrqZ1Y2e5XpnL5Hg2xEcj5rPBlUq2xKGeCuCJcW\\\\\\\/0gmoOmVdlsimpvdQmmW1g2TcfPwY+zg0Vx05Q9WRQzryj82TwHq3gGKRhA0xrLepJGhXGUA\\\\\\\/WQryXVWXONWOAhzBzYDdhlpUH0HrhYw51nskjYioM4TStg1BV6e5+bWs5LHMMYPkpSKTQeVd2GsmVW5s0afTuwDiEVdW88k9IRsRvtEjMSlJMavp2pcGhu+hKkvlmT9\\\\\\\/INCsn6zcZ1PHOpTsVB7LttQRk\\\\\\\/rpgsMU5EqJLnpNWvjD8dkM5SU\\\\\\\/juXXGZlIsL0gRBYp7f7n2rhgSWf7B+H9ce9Mi0rvyOGqy1ko5uZna2drbPOsG12SOxESs+hn\\\\\\\/t3JxqqDbCGSURDeV+9XqvFmabFWgdXZKBBJQHT8o9D2HGts3ngF9FRkpFMTKEiv4JMiSAlO1\\\\\\\/ndSi5qT4JOyuWPJhvYzOreiAtplWZMPqhyDzpYhi\\\\\\\/LPlJA+Mbif2nokryW1SxYBVfaSpwMjXT3gEcgASW3zg4wexxbY96nbl5LPewpN8BlHTRVt+3FcqvMLGXzRRRsoDblvnaWc+4eOLEp\\\\\\\/tiJapvd7AwbYPyFPchEOAGl93gbBQEfojGpIuTVcPJA45tsJuMF1I\\\\\\\/\\\\\\\/SM2wENqLoMtDdzl4md41isD5sgMM4w5tPzDnyjDxBYMeCUJ+3uudYTyvpHEcm3tJ7Ebsuv5Z2ryThhdQb3CW3JB38\\\\\\\/kbD+2OqSEzmYzEu7jkhnGFZB9I7bocOWQJvHSgM63fVOmPwOor1pXq+Ea0qsvWpkMPl0dZBcMY+1Uo0zkqwZEHHZBMMUsbW77Go7tZhdvia5uLpi1drM7ZUOsbxuDaE\\\\\\\/VSnZkHnrXHk1tgm\\\\\\\/Bvi9+zTtb8ibkthppvm2Bl7GTn+5sBlK2MRZJPEHhpLFVMG2xZlwNcE+JQD819NknCkBjqMjsL4UHQ7\\\\\\\/dTXNTmZmqPoCbMqxCkhtdc6Obv+tNaHSaO97s7MzxA6vfcfWTn052B1wPh8bt7lf4V1lvZVJU9AONZmRMuGBHcYjyH661r9+pyEU0V5VK4+3gnDlX6SgB9irK8xR3ZYsOsgKr\\\\\\\/QqS34cFQXzyShucozVWPyQu4J5ATV\\\\\\\/QZwvT6NRzEYQRK7DftxBOfLZRZAJUoOox7YFnU3sbiAbuoXNpwjG1ytbJgXDd6GDnvh6lF6mzFb2BW+Y0sx\\\\\\\/uRUgYPickjxi8qu0SK1hww6hyjvc4wvEpUuUwE9TxeztVFJ1umtA8FYUYlR1IqkAiIMYd6octok2nVXobNMT2R3cl9tu22cbfnos1xs0MFWzDZVSZvg4I9hoFZKU8UuDM7MhDbTePOX75WwvQ+1h0QXWDs+e+CVQXjLRMofmHfYaJ0l3hcS3uakcq8l5jCUPSl5ise1JN\\\\\\\/9rDr8K3AkSll35zHmytzQGpgO5d6V+cyudKwoIQTR8beURPZM4RGVYRdavO+8GzQO6KyaB0vMrP9bdKzOcnlmD\\\\\\\/e7WjFTIjIgPF3IAlLfPvMdNGOCxD9ha6pjhX9+el6kiP\\\\\\\/QgAbOd6g8zhpzWMaOtpIIV5kNtj5AVFpZkMhRL31iF8Ue6WQsiWPrZ2EKQs2bKIJcpN5ZyYa5JvnAr8ebMNj+2KrSDhIE377MrNxoG4oGj358+IN2a7HIb5bGuiTTCwFAt8ZIAXn\\\\\\\/gxCcsJ9fVl+kEtYB7SVo\\\\\\\/Bt1TTg6FJBOpzXmLdHDBe9ziQGlxziFe2D5+7siDMCFAp6FRFBb3nS0OML4LHHS0NhGxfQbEzu7GxOq7R0sQPN\\\\\\\/QlfrqZfgihX4uUBG\\\\\\\/yjsqjIZ9T05svDYa5DIf6tw6KEPyvpBS5WBXh7TQmFHUWZ0G+vPuF389OjNopSByqovDK9HoxMYN5XnhxuF4kBMSffEN\\\\\\\/truKkdBnMnj5JNGXCig1CS\\\\\\\/JL2zwo35vyHov3PTc0bs8wKx+21GqiZzEJdPUV348OSrrowFO\\\\\\\/vfLdf6sXvtlW5oijXpSRhHjAIBxG3GVFr\\\\\\\/HmxSt8aBnfre30ckVZw3nSLU6M53GedeYQYPJcOraHIQxYoRDMnxMooVFbnBJAetpF5B+fJZk0hwaVeKay7sD4Kib31a9gTUwylkMoaUB0tEBscRU\\\\\\\/qPvzXz9CSGKsMw5o+eyaKe2\\\\\\\/70VpgOB0Cs1mBbK4GLH1Wr+\\\\\\\/JbGrsl\\\\\\\/xhoTtsky9rIpBvztNMs32XRGWaymjSwW+JBKONNDNFw+KcpDUKJELcVpSx3bQpgWeLa8OOlGEMm5oNtECSMGay4PLWH\\\\\\\/YCoaVnwNtZJzGG+9zgBVsNjgR\\\\\\\/RRZNImqyGGTBBQBd9BYNhcNq8foQBkp21zcFhlXFRYXR9yhavlgqfQCWD3KlAIpFfb+05w4sYtDz3Kww2P8cFWxuJM5ZRwZuXd5jYigz1bY2+fOXjjjJOKGgPskUGY1Mqug+5bRNOxrSDbtvnaO9JEpMprnVdoZVqpaTi5lWgo142\\\\\\\/oQzvh7qTyttuybLlZafCjzh03\\\\\\\/s1hE9MqnodHU5Y1o5X8G4oLlCu8ioCPdVm6dZgjhA+ZOd0F4m18Qs2PsyMp\\\\\\\/WCXgpuUvOHluocLu22sMM4ZIixo6fbxC21MttZ2jRpqJiL+vGLrDuuCumfJxwXR8BEVwMgSqWQKHxz6l+DJs9gERfCEN+PaRmHJYzjjHTG+bLP45VdeVSFRMgLIJNbKjqe\\\\\\\/cCmP6mG+ptEH8GwU+q8c56Cysea9TIoc2Z31xFzuPXHLNw3HPYxVjrcknrYstApGGs0uTaQ+4iBr1TfVWrnKHOtPNPWwXWBbDg\\\\\\\/sTl1DerCRygSl0vy62Q8RiaS2FEKPe5p7RDUcDN4YWFq4V1Q8Tj9Fea++bh4ULboMpsslhdTNTxQDn84Zv9ecckWLNqOcjFSwJLPZ578CUbW75aX41RyiVRPBd3pOiU\\\\\\\/VhPwQNwFmD1n6+8mDE6qqLegp3OfssCuICaY1Gou0riNxx8AtB5xc6S\\\\\\\/lMwuTWf2ayYzuyq+Sa\\\\\\\/QcmPyoHK067lpwY6+oBkBF9BdK2P3phOQt8\\\\\\\/+y+Ol3q9pnH+9\\\\\\\/pBZGYe1nBHDNPiD6wrLqN3mBSDnvWSSJKGTKwR5b8hOmArNC3j9GFdvPn9ZyqxLtQtnzQa5ebVPJKtz4gNj2KJoKLOFcDYoo1YTvpTNpCvxuF1cksRjs36c1o9OeYHqqdCqsW4uT7\\\\\\\/NR70vjK8fJS1qhIZ9O3HqZTk6vhl\\\\\\\/vo6vqVp6r++j6x4ruFwVN7KMsIVb5XChAH8v5K8LtBlUAH3K3PDAZ1dntffRvQ3TETdskvvMJyGGGQXbFWRvCbcN2Jt5Vt+lbeGsLrC8dYJVIjhuuG1OGewanza8NZn1156DzBCA8wqDwBPVEXNApU9favH1hsuR57RTqYq2e3Zz4xlOldeNhr42ei+w0msHcjuiwej12\\\\\\\/0L7e2BMsEwZ2IwE1nORvFx1H8yR548vdukOipjzTcVTaB07o6VgQL2OmZBOhTTVB0S4D7NvpKxIRkfLSRgYMMxYQVLTmc+W19UfaXwDZLePoH1RFwJoEj34G1uTd3Hp3k2\\\\\\\/PE6VXiG5i4zeWVKVEalWE0\\\\\\\/ms8ajFu6f3d\\\\\\\/ATJ\\\\\\\/lObpH4WOiqR+6YpDntt8v0XWOJ6L2+LJCTFmlZNZRC+Ofmickxb6oGJxDumx6VXFnVMLHtqFUDzSZ8YCmLn+kjl9ApbHDXK1L7S69nc5zIxd6PaPkjetWHX6TtpQpqXy0uYQRCeHGLrfvS2rJjdHGGzBWOBM2Tit\\\\\\\/u6n1Nr5l50PLHrDOJXVLuPUPdoJYaFyM5I2RxNNyVeT+AS3r59tzy1G21xkEQ+HeBSfnnKFqL7LE4GsJqEsY207sPjLK7HcJJW61u\\\\\\\/T0ZULnHAfbRSpPBmnR6yeUV\\\\\\\/mx14JGLmAUkHV0gln6db5TAB6bnKKyQWhIqjhve2CdnE6K679zFA7W36NQnC\\\\\\\/g+6tpvcTlcwybJNQg1fD0kjLtn5MpE3snvPAQd1HUOsX7vELxcRx+5W0P1tjRkCBnnNjOtL\\\\\\\/Dptfiv8PWCrN0HPe5YEBl\\\\\\\/pYHXWqzyRBDgdnYkY71uO+G29BOfeVYcfgiE51XVn8uqalsFd4hhN2NOHsKtCzqinIniMvEF8MR78BoKUkHRTeEaU8FfoE3d4VEtTRvesr+ClFtis4myiVCFetPo3hTDAN43zjAvFfQOSdTbqB2Ike3yLa+epUEvMQbut8W6TAXzdjAH4gxuebw7kDhzy3HiecU9+HIKtbPHlcQfPyGwNUjsLKF2h5bSlSTjGK3IUI1zuuIGg3fBbXvqMrUy3oGSvOGZt+reaEysMCAZB5OGQw5mPAPSsaNrVRFByA6yUX4yvhCnFIpXjDYK8F3fRogHr5GC0GIZvM1oSkiNnpN7sfp+KZzo1WI+8BVHnay3Ydnh\\\\\\\/w1mnPSixQBzMNhInrumijZU7rJJmn\\\\\\\/a36F1pDR0YutaQ+0VlhO2G3226LbfbYd2llZnkNumKrASDyyBKOHW8qLfYeLD4D2d1jrjjkEV82jPMGxQ0j+2h1iopYb+6ogl7NZmU3FRpK0f+eTiSSpx1NUZQmbAH46leZyc5ebevbneA3RHAYRj0IMVYtEDiEXoykxVccpWZk+ZVIDVvcLjrL96VxM2MU+shlCt7rONQ5YDEXVSUua6zQKvnm5Q3DBKuZpjrcyFwJu8Vs34mR7bv0nmXqklAPmh2HFc9JtyKO2uraEnfljk6nxT8CRhlrOt96CaaWTB6TRXlVGTaGTQ0LfayHF+KWIuQ6DRdhDolMhWd+VwbMrEdNbuLFNzjN3fr7R63JliOQ9pVaNqCwNCE4WrhiLyKpmW+kO9tKA5LT3aKBzxaqBmo7m0j8KtGOzSOg77WaTM6AWrbCvYiZvCpgvqhHwt17QIiye9rqXavOP3IqGj1tKaZS0J+lK4kz5MAmWvhVDHQ82vmPrMDjZtriCKVkTPXVXeRPjiUY1VPp5Rb68rXpJAG7HHZcQe+8jKnHwSQpD1up7\\\\\\\/ENEKNFifdIz5EhVZlO8O4Yz+7XGHeOaTcvdTgwd+NZYsuSDQzMjv+YWdgg3lpZiBpYSaF6xwWrdiHjrJ+TKrj0rgAXjrKcpSuLFC5M+mAD\\\\\\\/81WlMpnhCXo1P7DKKOA+qMpQqzGLO6G2W4sXu6RAy4XCaJufgYtX1EZjAqRAu7x5U+17xw6bQJ3RcocSJlXEM4iiKVl5fC\\\\\\\/jK2okBPl\\\\\\\/EZPZ8CEx0Ta6pYzmJf5n5oJ9O5ZZOCoqifKyclWJQXEANLEE8XWWRi\\\\\\\/eGhVEW5N+wYBNv4GhRaA01+4aLnuBC6LRXYiCdzfFBO6XOj9yRxNzZN4jblLvOtfSn0FHlgVt\\\\\\\/trs6sPsa+7l+3xw8Qnizf8YfQ+nDrof1IlaWDAdx4SHuZnWlxKzPWGdNMRQ5lgv2Yr6B6e54lOSeiCmXotsKyNtpxz3q7Q2btQSgsXSawt6rK0PVqjMXR9LnY+bxaITnkROE4rEH33mBrG1WYSGd37jiCHr7EnPUvwpcLxsafIm+nQ7S+LifOg4gyy2\\\\\\\/wgIHRk3ZvLir4PDpnpyndOi9vVOsgZL8NUidQj1hOsVLfV60T5AW5exYAI2cna0+UjOapleF3f9NquZVuIZWY8qwWOEb5SnYaatfSl2PVFWito\\\\\\\/dboyCWSFpsP6n8zIsIctmLRcLiONEDr2v1GY20ZXhpQIrfs\\\\\\\/dOXhgJV9J+QAbmzJ7cyV6LtzjnUI18E3aH2bh9T5PRhWS5B4EzXaqsCL+5USQkrqR0WIDXMoc8mX4V6hpRYiI6K6hubJ6DuA6\\\\\\\/cIAC5fF7jHOZB6Iq08jIwOm5uKe6NYjzQ3nx2FecvzCIoV3n6f\\\\\\\/5f2m\\\\\\\/WoH\\\\\\\/Ka2ByRkzlnYVgJqyGztg+7TVpOAgUUh0fJVxqtZbSw+mobUiuI7V5c8pVj3eDJNVoY6Y2skBvCNheDdWP7oaEgA2dDaeTFP4dDw09EJAJ3oW0g6uBuEECcrr8mpD3F1jW8vznBmAydX2WtqGjMCMWznz2DLfCmw1G9eanyE+7KqVww44rFDkybnTK0+lquClaAOPFr9KpeIn2MSEVHrs6YI4m47FTkbxX3vTAX0tflg2AhV4QrcVKfpwbsiPgnLYmxUjnMmsFzluWPncWfFHzjpwYyoqOcJBE8p3VijBYU8p\\\\\\\/bwj4SQAHCvKDtKiXpT6peqnT5JMir\\\\\\\/e4eTdjgbdN+bdtp7euU9IdOMuFs+BnjmgwgBJKeRTBEfsM6+4ZxLLcilJOUxdt23OdBRaPEW0DMG+fRSzyYD1vhjWBE2ed5N+YF1+GXIXZLotOz3IbMdA\\\\\\\/mUrwJFDpBSLCwmbc51EIjbnnz0X5i1kECL4FJ2ZSvAACSX+8xE8P18Qo1KUSZkQ4KMuX5Au8tpeQvNH+xoju79+CyXr3h1GNiMHYupLSQXl4WabHQtj1y8xpkGZW4f36goNJ67GldCrYVhjZTYslqj6BAMF78T9OZWqpKc7BOnna8lhvY2cohk+RvaoWBkNnnVIrj50KAkc4fUDpa\\\\\\\/hVjyfVFW0lWVcHmm3eivJ+xxKNyteX84pErM\\\\\\\/4SQfXaIkYXj3MgkXVb2E5DTbdMEiW9mmHVDo27vNimyUDavWWjrjeNCMpoAVtJqZQCxomCeIyIF2qvYh\\\\\\\/z8d2UdjXLIZ7FQtasCU4i1VYJiA9b4nElzq8wuT1gm7b5v9uBazripJzJiWVp66wj\\\\\\\/+\\\\\\\/QxVZBGdUTdBhaR4WzMHnjl9alOqEXUaey5s\\\\\\\/7MemlUBHxNPYetTLglzlmaUAoTgYQXTHnpBEFPlOg79INPTDfq9ZiudDuK5XGj6WpIqmEiols4jto48m7VUgNwYpgxL7MVRLc9g671zzoqjMGCRMAndE\\\\\\\/F80UxLDhVw0dM5Ps1UaA6jdzfvW1+MRcCvoDWnL94thYLCpq8uebrlE9e7A5n8+yIpmETt3ApPc9FMzy2Lqrvejatvd3wL\\\\\\\/isBqrW\\\\\\\/0o4AxS+8MFtAtGXRxOxK4LuMULV7YSqUMxcOIgnbzw+6D5JikeVXBILEV4f1uwgX6NQYCouieSRmrIvxKiIzeOLycXQKUkBoylk57f37\\\\\\\/wsl2Fe0spn8TwflQZxbxXgth7ya0rCa26zH1EKOJisELmHwbFhn0\\\\\\\/y0fO5kUqkBTQvSZ+qjb6Z4xTa2JE35rLcQIP1jR3AMB2UEfR33qJUp+oihszcHWYeekCWB3y2fDBMhj+dKiylxX9eN6RncSIEy00caAwqkXnO2eA\\\\\\\/6VZVsCPQCcEa18kFWLxLEGkX\\\\\\\/mv0V5z2iLc8Ap4CIe3jSv9Pft1uQ4xRTqkfOdkVySXH4wzFHMJ9HWSjRYO6QTHGWXwytk9juKXncGXml9PkPo7UZvdwmmFyL6E+KJ2khisCBU0o+DCnEmJWebpsM85C6u8SSLolr3EulWnc1D0T5Fo7qjeA5tKV0NjEEahgMXAGRvBulfeRx3+rkL6THsLxFV9cdYSuQi9l+PhPXNmcZZuoM9bwYA49cuKiTCyM8c+BJ9udXqQOKrCDUSZMW+cvtb8yIis+s0G9AJHqmzjit0wxx6oGrR\\\\\\\/cpVUB7ps0isls3sNmEckKqpt3Yh16hxQCh2PmNLXjRdrb+ylgt4sGoL8w6LFxaVdiww1fZl+iOLhdAg6Te0VGce7BZf7UwDO\\\\\\\/Dl2O13ua+UkJz7fkHluwRm1QTzUSoKIgqxeczggBrOr3m+DsWRkRp+mrbz9U8IrWx8dwhcMUexQG98LhlKEAYU\\\\\\\/aTm5kOF2YbDguVSIxYZofjPzfHyzrwPcVX1mfwnYsacZnZLtQTg2o\\\\\\\/Uz2hBeRVGZaVSjHMvIA0TRN7SJ9Th05U8ZD1cwoajKoElwzx9ZJVx5JKQSdH\\\\\\\/0j4QUOm89Qw78K5fznfU+zr1hRqOXKcIUNbB9AavRxJEBhAPaZzVdlmDYrh7j0ztwF8NkgdNUm\\\\\\\/DdnN0PLT0z+qashqNaKkgOdrLyzVJ\\\\\\\/tvpnLtiKJnwcn6I6gfkm87JzHlK0tSxen9wu1n6l2WGsP7ByfSp5kmgIRbuwzFLpPLuLOP1TnX\\\\\\\/GJXg8Va2CC56E15h+onYd\\\\\\\/9CjCfQmgVOVi+rtKywTRu06IKgb1w0DQ5OUdfER3g+95khL0V1yNcjKZd53rAxnb3G5Nl+xTIdZzpqvkYLdHT6bYj1We8Yrc2l3QvCA3ZS9dyDSY\\\\\\\/nmyr1IBtwUws3HWK+v99F4McHsq4oDePgxdCPQalfpIixl4Le0mCBYudy7Hl8g9MPl4CwIz5VkNCBsnuDtgWk28dtWEgkLdVYMssjuXY688DEeqGx+fwt5ThiJoiwR5n4R4XLHaY5WGXENhN5qr7Jp+kka\\\\\\\/XXvtVF2aN0K\\\\\\\/fXtAXGJ89MdAezRHFa2e\\\\\\\/6I7ba7vZpF2wp4woIe1ofp2mDbavDFgOH3M+Y9JDdBFyzzw2aXeX1an7kB81S8AP+E46jq8NosJW2t3qU8cid18CnddemheTsT0wOWGappdiEtk\\\\\\\/+gYaAfL7DO3N8GFd3OvPuiak1UQxm9PhUQmhaCoEMXXTPqcTyzq1o1yio53zm\\\\\\\/3C0ZRJ2jGA1Q4nR1dXtoo8K70KwvyHEvdfZVuxGQXlUavWSCe9Rl6psKWmG4qB4m1DRavxtpOXUb+s5u9X7lGEMXafFCv+pfleC1bNK3EOtiRFXrORfiXGeh\\\\\\\/u7+b4HLAzfwWkvTnuH9rbk9vrDSkq7QyT5Xk895sq54cZyjQKdzINYwGPR+J1rRu5+uDvBfpd8nXqZgkq1xuRMlTlX3lD7Q8LctHBN6FHj3HhkTdylj\\\\\\\/waBJ9IN1e9mXK9Gkzb377GwynGgoMFC2180dsO4AJa\\\\\\\/K\\\\\\\/b5licxxSg0HKCSbmY2wY1TWE2FbHC6H08VOo7xa0qTixPLg2wzS+MaDouheZHvxeY8q7BtrQAKa11vsHYTTScMT3ZE827D+kiBFTLAxNubTQPmLxSKYyhpd1VR2Q3py8XouLFiOjx0wP\\\\\\\/WIe9DKmi5IzQ36jDOa4UqXvVdbOT0ePYs5K3WvBB3xPY2O+0n0mWIRkKA86PIWhWBYf3ThoUwmYxVHpTEnuK7U3bAd0Px4q5E+D\\\\\\\/A4T4+Rfb39Omd\\\\\\\/UsvVKOQ77R0M4J7MK8t7IAceMB1cKfKBG98a8ZnQJPKIqTsqikTXW1SE\\\\\\\/DOmDNYPo7uJA3\\\\\\\/Nx6CDWy\\\\\\\/XuHTq\\\\\\\/IqWSUedQ4tWix+d0BhbsEdkTwwLH7jSoa0sNww7OZup0YX6R9IIItFZNNs2jZBwpjwOHfeIylBywfC2rEjwAlEqJP+HBvBqFIMJp7KB77NU0VT8hd59PAw1ukveogBGk\\\\\\\/R2cG4FshD7DZGtLiyoBNCu+qGaYmsswWbOevyAHN2t8fOdQrUvKDM3o0LBU4zHz\\\\\\\/9Lk89dYt7XYJsrcakVSXo1hBVmPIO8vs\\\\\\\/l8JQ72qc5lyovQvZ4YJSp7nKirrx31WNVB8a3C8GxbgbOBFeTy\\\\\\\/z4qHHB0uyi\\\\\\\/8wahjaVOY5HRtHlEKCPH0914Iy\\\\\\\/SOkku\\\\\\\/clmlYtO6y1jgCQuHjx7p9SkkmSAyPPlOXvv3R8OzOOVW1zy9Zrc2dDSYETaVNhksYNPXfmRdr4o2FHJej0E4SCJUSJeEP1ydso8xNNMvk9Ys\\\\\\\/Hkx5fG00PlujR1FWcety99Z06sNCxY0o7aVdtVtn+l8b+ySF8tZtZRSL69MRbDAdxL8qSP8dnakansNc8ppjTai3KUFW1kmXQc8N\\\\\\\/Q1SJ9oVV5yNqT6BifRB9hJ+tE3JTRKnVLzvIh7GaDDiSWKKkAo+dA5bVBK9lrDb+evldK0l+aUtQu4G3iXJ5nTGwkTt7EDRjmWhw4dth5qOtnzenutxDGcyqMmRewQBYun4dEVQEidLkrUd1tZyP5PAnvy1F\\\\\\\/jKUrdUcrAk5\\\\\\\/FJ4wTr3ictgMm3MQ\\\\\\\/pAI+HnU4REBst3mKjjqSPTma142lM45ouyVfougjwnAaXwr3mAOIQNgHKZKLJBVVUnR8SCR85Nyj0p9q\\\\\\\/QvTEcuZmIdQFlUIjwK8Vnc4F\\\\\\\/PvMf6MX5qDwIkIl+LkBfPD4+ufJ7o7WKPvJvCqtAaV\\\\\\\/7O2eneAhNM3SfMW4TLaXsiEH3AkIqStH6lg6PMznqwiyhl\\\\\\\/lhAOHHQoWAmUfJLdWeUyl42jtY4RbytD8NAB+FdB6dPdj3rCV5XBSk\\\\\\\/9dS5cY3S7SAcJ4yNjRxva5GpKak+URdl\\\\\\\/UvL\\\\\\\/RWaM3OouKjxIQetTA3EnLSxxnsw5W0k22Kg1fKjirziqC6ryO7zsSOA2PLdeyWZXaOdgmJ1+fQFMpncdM8iO4IcV1CpKuIgalTZ6v1VFu0EjuW0CpaEUZhDcANoddX7epzDbz0tkR3DoVpvtMz0gbAObgG5dgP\\\\\\\/dClluNUIvwMDvDyoC+WZ+1meqXKvksRHOdBvcaL0japLd+1QIOe6d\\\\\\\/6cBL+I4w9YSbAPzlj+WKeUG6beL8XG4olmVAlrTx2Zm1IvizeSJXNWq+o0BXgFnQqyEMFq6W6LdmZyq\\\\\\\/YL0NIksNjVQ1hwtyJ2ysl8h6HJuSOnF8cV1aRNRoD2sR\\\\\\\/FG6158wMQz9qpBUw3dxb5Hh7Uf7q6tZN7Z9L6tamEOiRnYi5oI+JDprTcXFqDj0njZI6jGYyQQe0A9Jue8lvmdiWadvh+IuQ2M2QtCkxt8\\\\\\\/e5EcHo7Lch6GYQ9ck1tbVZtFJ+r6tij5E3jiqryb0KDibQbL5b+VTGq5XHQXXIysdtgD67u9lOP8rgG0b9y5t3ezNUuY1e7qvV0thmbD+9L3jfkmyQCqms+KdlIHsy3MqBhb4xfucRKq\\\\\\\/l2LuZQ6PD8dgO5IfozJoLxjKwWAi+bBuagcx1ENCpqxl\\\\\\\/NtWc96SbuFDYPXheD86wAMMaLMMX6SRv\\\\\\\/6V47Ba4qpGXPW+m0stM2rD9HaQntEnAtWHY\\\\\\\/9dZCHFlScC4Gf42aecY5x8uHdHe7KK5tiwNwThu68sdtD2\\\\\\\/jDXjd0HncwObexQXW97+B0cfpy7LhMNC2IqZY2p5ovjirdNGKfJWaKLQUfrf4HPl3KoNrTITsFndHeHymZ7gZBWzTSWg9DREQuMhGMQMYA4Pj7sO46ijcqzWCxx7jcyxE4DJe4VOhmRjn0bWXZmb1y4Suv8CCQ7kGL9fh94AFSUGH0I3WeLDmgpS7j+6ErbzfhIh2gt1E8nO\\\\\\\/fwV\\\\\\\/dEwvNKE3wrKcJBsCK8Jal1FLqsFsTvP3QZUMpqiPk7tdPrm92DU1PBXixeCM+ipV2natJW44K1VdcXChZ1\\\\\\\/tEIldx5d8yi4sU0DrxDgTaN7V9JF8HuxyGx9a2\\\\\\\/yaELuNqgvsWV1RF3Eu5oSzN\\\\\\\/NW1RwSZ\\\\\\\/7rrFEc9FI1Ag4L60e\\\\\\\/bylT\\\\\\\/sD517koZ3mcSojq\\\\\\\/GN7jAdZH\\\\\\\/K0gzaJWEEy\\\\\\\/5Bh1bEGtO4q99yX1CTLq6ebrmzqbkNAgCb9Hjm4kVjnfS8XqkpZ21R\\\\\\\/B9zbC1lrNC2pFjp7fosxtavVaC9qSpf4JlSwDPspVVP7bdw1CedbtrkGduj15qAyyaduwW3QEj67C8rD\\\\\\\/d07Lt0F6bXgUmskFsYpgOQPdwRg29fBAiyn83I9uFn4L5BK\\\\\\\/L3NCCWQ\\\\\\\/iT8ZF5EZkJINe3meyiUe\\\\\\\/lsMmh9qEXDfhdB458T0PcnBnwZt5BP706aLiZHrAyDB2CnICF85Vh+Z9nrB6gqrDM1\\\\\\\/PDwnw4se4Np3j7ECyqqDTpjA2jRPHChz7LAtMEe+1ACfQceSRiPXJccYpTU9xYL2F02p1rlsyLjCB2TPc8+y6Kb+LES9poaFZ5x9qgowD3C\\\\\\\/Mji2iVD7\\\\\\\/CyteWMP1co41xzstIP5Y06yoLYAtK2hP0HPeluAFcpbOafG+2qL3hEARPGFqZjiEm\\\\\\\/35D0l40CezzirDCjeO\\\\\\\/hWI+QNYph9t7Ag48uXgq\\\\\\\/x6ej8D7q0wqW6UO6QtVcwC+l8qzq\\\\\\\/6rjisyI09dcGB3AJCxlVF3tOHb4JmI3PBbcrvyyvGeexaJru8U5zAoow30erOtnuwzxy\\\\\\\/4Dgi1vcoLNO3vpzl7cNc4cmG7w5V2WkdlDLQ03A2q\\\\\\\/CGGEMBl+e2GHbNE\\\\\\\/GLnS0xtylz4z9S02IGuyDROaV6AVfDrPv\\\\\\\/GXc602ZjGBuNKjkX9p0ddK8qlA5GuiMDxH1nptXadotRLlaZSfZdQFjugvc\\\\\\\/aSFfPIpnBl5zOSzC9x4LQomGHryaV7eCNDQKcGeDQD1woRWueon9h\\\\\\\/nin0Dc8zymG421nbgKa55daWTqMBKtCPDbi5GC4mNfMZocdqxa7gKP3Rjnlf8BVF1mcXza58hEcAlZVNPnJkTOspwR4dKMSMrEGd4zNyh2yfXRtAOhbKAO1GeYl1A8A2rDkqg1y5CFyfBmdQsb+66JvR39uFZlxxmA8iAgZwsWi7JL11Xjnjxciio06QSB1ywm2sSedCF9NCWcNP4V8Tb6BXnF5gqcqHlnF0OjQ15hJ5ntNcM\\\\\\\/2ymXhwZyMZPVi1FnRd9PkUm8iZIyS+8zQ7G0rHwODrTRCdhjsTyJ9GwgCVCVElfZtJL3ilWX87R0RIca+yNJ6BOcPLa3Qe2yRyxHZWk209nrEz6CUnt6NP2OJnswp07oIJyk2cVu+ErYp+GNmck2H+zbSdI8g0B0DABONdViQjLwsGrT9knAL\\\\\\\/1ArVthEfaXPZNY+C0l1UT1Vo9sQU5uUqGFdLp1YBGXkxBrUZ\\\\\\\/dLSZD4\\\\\\\/IJOuKDvNpSJ73liVRAAr1RfYOlJmt4lwj0eKwGnNhvaiPuPXuZ7FI1gY5lXA7Ajxn5nLaNmCMetDnycKONgAHguwZpVFVOLrEABUQzD\\\\\\\/Jd9rtodvq6YJYBzb6PA+zObj11bBpriwomzaUfwD0r3Dq1J2\\\\\\\/qK\\\\\\\/CMiSn9um4Jb0T3p1Xhu6Dovz7xp7wO+ivR2YtLBBQ\\\\\\\/Kcu6Kj9NP6jsBZJtCasQwX2HBHlZseoenmzoF8mHEXS0RrDZHexya8hKzRSxOw02YDSNz\\\\\\\/9RDg8YbDKUduj5IvvebSLUTMaLnGCebP7bugMOEmnu0XrdgEmqvoKl7M9ddL77Pnj+OZFA+br7Ej7t2C2aam69SBKEYMnnjSxnCFuLl2ABoaLPZGe8OpzMPDb0qCAi+jZX2a7AWbQhqqZTuDSk05Jk5fkKuAsZ5\\\\\\\/i5PxHASGS1d\\\\\\\/nM+4w32PkgYB\\\\\\\/M93deda0ERO640GNi2f5cZD6ymxTHjZCaZfe4kwdkWm2z6WTbvrlCGVm2VplbppgaKNCPwf+k6GXso48SGG7h9YNdKwCdnZOFFIJihx+jsh\\\\\\\/insQBABDOWZEYOMQIqjmpcdEBq52hYT+TbdqBJqmpwhv8eVa5TIwQ\\\\\\\/LTRA9gWKFB6DTITRKcFOWZ+tCv\\\\\\\/XjGE3dz0ysdoeHHJxksW+84YTO+KukAmWia7WMEvpy9dTL95mLOGNOcvoNO9hrwOu\\\\\\\/AXgLpZWTPX14fOadyiLT9sutgfH0knVldTrFmliH\\\\\\\/O2olRgMse7cwPHuz4muN7zNkLYXd1bq8ivA3WfBBzVDv9eCzeumitSuAnG8x7rRAHFC71p9k4NAK4Zfmkei+NGpjzCqbGFI17deZrlPVUAQ9JC1lGOx\\\\\\\/oHNoWWFCwH\\\\\\\/A6uMX7obpNmmeQus+rOZlRSE59Vb7fbmlw0\\\\\\\/l+dvqm\\\\\\\/UlLTmqD\\\\\\\/tCy8CO9iDKCCLr++n\\\\\\\/bbidNbCYi7q0OA1R8\\\\\\\/rNNHkc7l2NUDMjk+YpqOhi5J81rXrf6p8u6xiNa84aab6u3WL0+CwurP3oa7BQ8cxvDiFzaO3nls3XT+Fmj48i4AAar73lx0T83mymCv6YAq7XWj4RGt1JAMrt2mH61BJT3vtPX6H6O5bIIREPmY4tcAWbxLrArnz\\\\\\\/KXF7PsM6K5mEuqqjNi1MIl0M4trWaimMk5wWc+WYCkR7krbmPV6bdNEV9AoykIgtC\\\\\\\/r8oKYkqoElYnJZgCdbpLNQ1cMhba50FT8KLEWUs8gE9hfyeyTxkg6m\\\\\\\/xjD9ezWFuSOhLgwhSWrytM1qBBZQxEHtEwiqfv4xcBv4FbRRlvV4coimnvAkZfVfUBT7lx6dgw4wSXqmxnMFQ86c1VEnzRYcdLsypgl06KmTXinw4UffOOU4NEYieJ+ncSByPoDIeQDhOZPOzsKZFq\\\\\\\/RnZw0eBwqMbHmTmKiWMZaLUmy6YcitWFY1xFp6XwSlTwPcuwi9h8nOL\\\\\\\/d5SCR3oUywRd1z6fTGnXFy\\\\\\\/qMt7L9RPKMeTPxrFhYWWTO5b0zrCVnOmi01XNxXpSfrFqSOfEpZLBdZZYlcFK+zjqlzHPHm0FVOkAJUaqtyu4jgBptT2j8sCV5zRJct02EJcWvqwZU0WriwVDF\\\\\\\/zhwDc3BqGfDnzDtkh2rcUF7qyLOxnd7Vy1jeFtO4Tl8lMqnGiHCo5IzeeI46Ngx48AMLk6EpJ1xlzILY7PMBbPUrzNQ1K8DFenX4CWI70SEZg3+DDYbgXyHTWGtyrh\\\\\\\/cRcG6k0vtlXYnIglNao96czxvsgPHId35SeJ8t89X2N3hpv9c0TkjujMqns4gaFmLxlm5uubz\\\\\\\/0wp7gx24Iiynf13zA5m7MrgoHQfSZxL+kbuhOdHPPL03bkeZ\\\\\\\/gXzkvVnKNt2GtuYZEjpgZ\\\\\\\/o+\\\\\\\/1\\\\\\\/JQj+mhI6sJrK66QvLPiQjrXSBRCB8yJvNFkvxK7uLoHOz28Yri08x3Dk7l14wjqSdEd8rWFUzD11FXMP8lxFxDQWrqNxV5LU0Q\\\\\\\/7BrOOcGeNGh5gwTah9CB11Oh4NUIUz6UWsKU\\\\\\\/KqUkQ8UOQUcdIt1dvHCuq2IIKy9W1e40P42LnvcgWwbrjUHsf8FCjSu8rLdFWSxgASXn0RHDRNZ2p1GIcqtOJ2AmxnNlFz61UF3SBEwdYpAf8IRvrBGsABmhfPBpYImR64rJWGiv5aW4PspzYtrM8kkes+I3B1bSM8Amw8K+mi4Ct+WYHO3eW\\\\\\\/eQAMSyxn\\\\\\\/kpxa3O8cNEjqKzjDNQyZEjoIQKaaykAkdEPR+86+SI3KF+2e2uJmsFGRQybtQK+jKOKWxtQ\\\\\\\/mG7MjoZr3I7xdR+CX84rWSvmZo6sMEqEbtwz+qPMblx3P3m0XVwS2BsVvgvgwAZnP38lIw1bfm5t4tqJyIYyn8FIPDWy5sXFko4wL0Rpwvm09ZMJVGrzWG5wuJkryUUw5KQankgZFWB3cyIcJKrlX6VXDOsPAkQdf\\\\\\\/PDS0P4lbSEKjiUdQSGPbDyWlUJY+tT1g5G2uPTETOqljpYr7TR+dFg9dmcHomaNGnqlNfEpruGyegGujvve62977Zk2Y3RV97vmxKcGB5yToQBWFB8JhVEYeC71QkYYab8enTbRYMneDyAJ13pCJM0crM7KDFK21MngX6peUCZPeTHuX8ExlRH+FEgxdGuJ8MfofKtLaLMAlH0DQ6CfcPAaoZUhxiKQPdoTQ0EAmopJdSISTjoFRWpvxu+QtqLHokrsFGGRiRkPcQrWruUENsRlqFTPqugZ6sQ0xmV4Pv1fRl6GbloH2SjyVfoY8XjmybxH7L9BUWs3iZ50DOOHGEybA5w1KPBoGjhl1SCDo4s3monjCwJ2Un6jWXdRotUwyTXkP3JqdXCI4pj8uFZQRgF017uLHyu+Ex0ZZBTBzkCyUPtSFK54i\\\\\\\/lr7oJEWl7lGrmXq\\\\\\\/wSCI8efL2\\\\\\\/1Yz80skTdREVcAoR0LRVZkIP2\\\\\\\/bSJMdK++9OFZhEKFxIzFHaMQ\\\\\\\/S4j3qjXrMQjDC3yUC6ydwi2MD0tv2ZsvuitC9jX+Dw0RP9BBKJXa3X0JWP2iumj81ejdlfwSeDNjFEd37CIqtAvsekSPD3kRS9qbiuKF7\\\\\\\/gDHo3T05dApxVaIm98YAExGdSPMsVWiI43\\\\\\\/ScXZic7079j\\\\\\\/+7IRjXPrrbq7pDIfOrWMfi650x4zj+ShoNye\\\\\\\/HcXRrT763Wkgl81h8fNNVFsK1zWdp7rrOvOVIUS4ECnGyp3RPkTiN5sINhSMPQrx3AL9M7U7OVaaxlCpkR6EOs7ac+bKlLE2jedC5JMN89FGHiGSSnqkwKTSkvFe+e5qk+142G8TsVz0pvXbJIiZIAvWyl2i1GWC9tHl1h4h7OtxRZcmDT+HF3EKiRM0AW9sf7CdS634mMRNgPnhZRKmG2xI5SKEZNtDrllRN4DOrkCZdcK6taMLm\\\\\\\/ZTICAnPWFzIlobQeL\\\\\\\/uSZtnJizMPLdGe5tXBEA6Mwb8KBmui8KueaCWA+hHRGHplYl39Sq5sLosr4E7DsnHg4pydKrZqmpXfg49axvAYkxL76FlmtQt9kKnxyIsc0gHVK+Vi8q2y\\\\\\\/Is1wvkxoRXdtI4DRlzZHZSGEQEIk5e54PEh19gaCblkUW3qH1VsciTJq5j1PazJVVaTdbdiDKd6NJtax3nj\\\\\\\/bM1+qq\\\\\\\/xdHSGECcdJ+1ACbnXdHu9gvcxRdsrG840hrbsCbjODVDxbxZ9c4LYUQIGr8k7Ssmfbn1iRWWOFqmSKVHiI8PgHhwmoftr0yIy+s9MQFsYBYyp4hK0warSVuWWFnQv2CQwMgeMOxy7ZCuo6+KqhcpoDyTUsUuBuRMLKpDBA85haEyjY6nGb7dTYAjxEirKPFBUDj8FWlGoQtSRPN0ltWcFjzOVyRG+HnbWyP7qYOCmYUxt1d74EdnnZuOzazSU66nWCFvd+ZK+9mmEW4He4ozuuFtplSZvFV6RNwpGdAZ0m7xlVbv96IjTu1A6TUJwSrsKimudQtv9GF+7mH8Aqd+GxjuOvPxpGUiAOCZ18ekVk6SgFwrS2kD5k842tAs+4EtHZ6iT2dURyV7l\\\\\\\/TkZ9FOr9IDGTArnwYDveloEUJuoehUG59Sb\\\\\\\/Rl89WOEhnwZJGK+z+BwrimPkM1Zgf+Cyb9MHqfVj4NIrqr801zSBJDOYqbazxkgYxFRvkfwFyW17H3WU6vMHRLosAeXnilz1Tr5BhDemI1iLrmS8tMWnTiXYoOjsZVO3\\\\\\\/rZivWEc7Vo\\\\\\\/LsN8SNOc8j3MZ8wGc56xMhgUNuvPs2t0z6fFfzc03HX8xzhiSwZ0lmnSCSc\\\\\\\/LnNYluvD+FsDX\\\\\\\/z9zFFuLk4y\\\\\\\/6E7xcgSgNPhiISBHnR\\\\\\\/pK5lhnmFkVwLMRjVVuw70J8\\\\\\\/Ugd6xMWcp\\\\\\\/\\\\\\\/e3txdp6caBskQlheqArz1G8PelmPtbQoXvdAAh2aJApHTeyzHF3El8Rb49+Qa+dwEDOOB6wd266eL0w848EXC5e78kCPYAYWOf6v6sId2IYWNSJrnl1BM5nQOv\\\\\\\/+OqvF7Lq9zTRjBP3XLY0FSqB4hN+VPEBctKROqBsslmXzITKv8GjZP+p448zmLJTc57kl0hK3rpng9XMuO0c4Gxp+DEgvHC+Nykp8EOCJt7z\\\\\\\/9mXLOJl9S7Sg9XacmW3VWsWIJa0wrYgIEe8n6ERMTWd8q3F31Ghya2mNOmffoXKd9zKni63\\\\\\\/JSVIFhcksNh2aTMCiW4gzSJQREPfVBS962sCsXvH83KjGnrsSjc6mfwXEmh3NyVGzWM4dzkaDuM\\\\\\\/tg4uU0EhTjBjSjR4StfYn4sjXtWazWcBEPJcWD7T6+9i6Kr24+TZnIMe9P1euS0U6VWD4c0euSntQZ1NvWimvKTyOrjH5eN9IkCZO3AMDr7q0LeeNuVEksd\\\\\\\/55\\\\\\\/mfMS3u0aXZEhQRoAoxkO+LykKWl\\\\\\\/U6Bkli1Fxn6TrY7EUnuUYSTm\\\\\\\/NLPpg+GC2QUVrsbig0Rm+gDWlKr+zqNUMCuiIVppUh4977oNM\\\\\\\/mwe6Vyt9PoRLDJhfGwuddPlyV+5hbpS1ICE0WI3\\\\\\\/gQfYZgJK\\\\\\\/jMzcjpieHls8psvYZOPuj9uffaT4I2JTFfux4zGzw7kcq4Uz7KbrbazIbo2iW2PN\\\\\\\/dGS7QB4uMolt1WjBhcaBAyJ9J56TLKSs\\\\\\\/Apko1E0wN61ExUhyfbhcrPqT4elBOzBa2D3rbtZ1Tz3iFhNrqZUPR9nsW7T9+Qli+6jZqDiDjskEwABfGiSFANARvXfPTZpYxPqA2ZuiX4+zC1k\\\\\\\/uGcQhTEwedlbLLSRNt0B4\\\\\\\/GLQ2JY9hT+C+Jz3yK1jU5lvZLngpRxJRhSLdCnj9ZMsqBSJnTun5Gbw\\\\\\\/K\\\\\\\/1ZJiTItfghHqFKmRoLRfaW1\\\\\\\/fIdHL2CkICn54n6YJMf67hFrm0xrjlGHEWvCBajXvOxw+jxaEoJfWuV1M4gN8E3\\\\\\\/MGcxjVwJrOsHd6lOqFZpt3jV11LoE3lUXrUusowvKGbWncL4b6H\\\\\\\/g06vl3xU8R\\\\\\\/vpvLD\\\\\\\/vGb\\\\\\\/2Wbo0zhSO7M64sQtibzorBzfMNBCd7cppOsu1C3cXE08IiIg3juExxgcKkyrQZZDm9Yr\\\\\\\/tK4s8NwDUJfZXcLAnnMO7FBIWvHQUakrmjE1OgkrrjY663tTx0ygM0Qka1+VDFil1lcOFqQPwqOknT5tIW+DWktLrDWrxhd8SALZ2\\\\\\\/h3GOpNuCCalL4Iqm2jHevwY0g1zpGuihC5Z20rIQSjPgEA\\\\\\\/\\\\\\\/rHn6Yf1t1xjHW7KlwqNOs3vXHNd4rVYcZUq+GKBoRw0pLV87n+t8EkJG9yELg6TfJ8rCBpdlfmJTEDxrq03e2QB23XPkYbOkvfMyVcrbKF2f5MDATl6vHCIdx7qErMKRjmfKS2azDaKcWg\\\\\\\/fEYReXTQRc8W3fqSBh3dO1p1smlTSfHyg5q0400myqLkIXT04XOn+KntvdDhecnA3w2OOROkE2j2fWTZbe48M1QncR6etVhR9S7X+cUTPw2KtD6HVlGhP9sLhzxgIgEvVI3rPdUTzAO2bCjdAaQNY5nHe8UwhwdzSDEeHq2RWTIJZjG127gSf32tt6sK3mvlHRJq9TZRK8Vm+qkT6tP8E6kuXpAo0ZdZ4O2rFkbtgoLStClf+Objed0XkBWU\\\\\\\/sGsfJrMcq8t8HT0tNUoSVajZbgWchHYRRb\\\\\\\/WvLrnN0wGlChyf5uD70g\\\\\\\/4B8fkkL9BOlEVnOQ3FK48f6B0a56LQaat5FAuubKFxdvJxv5MFXkboaS7zeRgkpdjL5AXz06BwYOOBBmHwxZ9r43NwVEn7L0Ddvsn0eV+rOJ\\\\\\\/aSkbJBUSvdBgUprpnlg9m8pwFMYwxXsj0zPJuW38Sbiv7+aFa2yNao4xgCUfDVxDJC5AiiGFAWtFV7S3\\\\\\\/QL8zoKFyu3jCRDcyw7Exxy0IB1EEgWeMiMUBuWnQvb+Gfgf8x47hzdtMYK99JkhpKfF478WIDpw6im6YvR1V9xgCU1WlaW60t\\\\\\\/dd1Nx7r2lUWC77IlSzpSLLHp8mPinxwlXc97VkhTfq5YK9WF1Pw+7yN2TzW2sOURLbSUgsa3C6\\\\\\\/ZuwjnXcaXK4sZzT6CkueO9lKboUiJfwG1kJPTFkr6o2zDgO50y+izh9wFRTWXVptY5QePuz1sJBl1ivBcE2NznWffde59EdXwBd+fXf\\\\\\\/GmInym0gs3wqR0XEllDA3RRdBNzvtpCflLNIb1pLwjeoylpYRPbyAqREIQfhpdSHHEindFjm8ZHm8NPKLSM7j+lIbQEgX5y7l35AIwMbSaaxxk5t5eHwzLg+gdZG0oWGh979I4fRhA+t9bz7JU2aoxVQ1Vj2aPIPUV5MWrAtWfrl9CHvw2Pu\\\\\\\/I7HhwCIN9pF1TCe2HNkjSguiAYwE8Xe6OOiKdmy9ZdaaFR8GLVZLHKxVUu348fPflhnAQ3yicWQ8ytvaCb2n4gmXSDKcP9j9dpclZzNG1cXj\\\\\\\/K\\\\\\\/0dbchpHLO+Ly1r3A37vM4Q36IjFsfTNdCicVgvB6tTBLO0doAjyfQhPdF78rxyRrGk0T5R\\\\\\\/6kxjpscF6ZF9BU1nPKdGQVBgoJDRsX2VssUOF54Cd942Uo0dtlFPJafkZDNo2rujddICZKMEIjXeLvZaHyf6KjeGMaSZmqnCvRKi0yJkOvatH3kdINSseHBM+U85gvmBkNnT7Ee+eSCqhbHxlohaURJDHFHD5HlEsDsDCoLInni0jbzEs6YbR5pbEz6WhbD8nr2Xk1Q26Nh6nFZxt\\\\\\\/\\\\\\\/XlFgDrNkE0yreVgct1bN9DOsTceiCrkq9WtNGfI4c9pA\\\\\\\/xfV3Mmv6OrTP6ptdYkf0umJSKho\\\\\\\/WzLLHaDLXWFaWuMw9VYLAtREswGVWtTlOcBSJSx6cDLETvmKbqVm0YiXSSbX\\\\\\\/znzPHtHFMfgg0ZHt17e7o+O0BynGEf4vomC\\\\\\\/sIcK\\\\\\\/nnCoHZCgKZU0Y+mhsKgnPW9pjrhsaS\\\\\\\/DydrAY+Mf4RkOPt6mywi2RXUEfXduQ\\\\\\\/44h5JI27dn7\\\\\\\/8jGoPXekqYxhOaEKi9AoFqQrvCssGsLevGMVpNwG9jgWcgG+bqbOoiaSl2YQdG3QMweX8Az5wyaKpuPfi3NIpvBSxBTZuP7\\\\\\\/qGe7K\\\\\\\/l\\\\\\\/P\\\\\\\/n3JbeeyBIXgRKH6mkaoDO8Ef5kGE+hDUTzLsPm+gaWJTkfAcyMiEh460Ntyt69xxGaMP045TKbfVg+HP5n1VyG6OYY+rRh8930B2tUaWpDSCV8eVG+ExetWvdF6G7ISVbZpTNeQVn6StoT+jFySFMoG8lKiS\\\\\\\/h\\\\\\\/rc\\\\\\\/RXb8qX\\\\\\\/t9K0+M8et6EIzGGye2Q8fgNJov0YhIUQ+PbG\\\\\\\/bjLDKOFRqlnaoXBzUXoO4HR\\\\\\\/TG3b0D9KmhiZQuzg4O5fLsHrML\\\\\\\/\\\\\\\/dJabXj\\\\\\\/gYHQpAfetVeEwW0sEM3FuwmJUbYS1PVYjac+vok6QfuX4wCpKc7lqdAUjEqY4H9lqltS3jLWwPMmbNsi0N3VUcngkqJAYxZx2Ta8zIpx+kmO+O+Ys3IXDyGzW9MnzXtgngaCNLil42atYRjBcBe5JjTDyYaVKYluGGBe+8h0abpBL6hl+UrjfmvXZbXGt+ANWU6SwglgJFFaMxI9AjG4Svm0pFh8vDBqzwogORqJzti\\\\\\\/bce+yPJQfURUW08SYeD45uudndOmLlbbgq1\\\\\\\/KR+WeUqOX0I\\\\\\\/F\\\\\\\/YZOGdo6r+2ZgKeDAAeHMD\\\\\\\/xPnfa7L5BM+yJvpIgHz2jQlJYbQlol6cZLrS+zU+d6f+bNKxBEmIoINifGvCAfWfrxKVtH2O7cSSHl1XL4zH1nfGKpx1KOqemJ+W7+0QnStkU5wmtiiaJvCWnTuCn+fmL9by1dB8\\\\\\\/TvGJO9G\\\\\\\/GBWravZx7IRCLj6f\\\\\\\/txh7kG316NEJFn1hcAl9rheWOkg8gAo2VBr\\\\\\\/6W\\\\\\\/Cf6UuIIhegAkq36YWrKqKuCXBj0CTG8cl22RDOkN6JP\\\\\\\/hPKoskzKcllXVnTJ4zh2DdzEa4plJkOCr2sztevnLAkCCP0jYxMPWu0BrMW3dAVeNIESjdnKirDaTOBft3f4hr09EZQ9sR6p\\\u0022,\\\u0022iv\\\u0022:\\\u0022dvvCI1IPbGVI+lD8kzwrrw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002244f5ea552f89406a71c67f59ccd4296d89fdf56fefcec892ee3239d1995f1437\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dQmfDDKKvXJoNwLImd1xzqNG\\\\\\\/TtFvFc5NHIEdrrFIs\\\\\\\/F4VymhCP+lzmNobxod2hikN3jP7BHOBgRz2aa4txoLRwZUr82q9kRb9Gf3zIWwygR1qUKBbXSqWALrQLP1sBV1wLn7R36epUCb0HiXu+cBr3iJ69sBTKm6j3OgTKVZI+Qg1lZXHKNbHQ4YZDcBlf5noXz2uCcAguZHmqHfBN24pSy1TUKHvEWnlKnmoq5mzGyznx9Aii\\\\\\\/N2EjZ2z\\\\\\\/8hPULRovFQJsa4btlg3xD639O19MEiTh5oOMmtWFLcEXoiWr4DBaPEhhSbWEhj4TXfuFFHjU79Y+4obBir1dx3cTwUK8bI6WWl8Rvd2ts1ptWzxhMfUDHS34mVN4uNXzRP188j1WHhENeyiVgGSm2Qhj4H59C1HTq69HcpiBkRU3RTZ5ByMzdCQwMvZS05A7LLB\\\\\\\/m0gtrUXwePVSQdTAY97LIvzRO9zhO1bIl68hLaQfXE8foAarexRXbfwsbnVIrRO67PSXLwsMwZfgJW30vFvBYeb4uKy1fvhkJHWwhPbwhQszGxTP0zGoFO\\\\\\\/731Y+JoNSpNCfLxoW6xW2ElGeTQ8jTVFA+xTE5qV0JMl0Llu+9Mt0GcKoHkyoYMZrcoG9lQZZuvnuV0oCFD0DmGdrZnTmw5Klw26hPtBwwJmpSEYdstCDvJAN8KE\\\\\\\/\\\\\\\/JR5m6RsYP6RuGv9rPo+NcMnDsXSVF2Vn84UYN5OucUvoqZKl2oIBBZx0djj3Zim\\\\\\\/Bk77OeRQ3PjTYGRwtr3rtaIMLmlUcTGDrymeRQlVQ3bolx0r5zYLdMYBsr\\\\\\\/bjK3lCyo4TNlFKsgMSByDyo8oEOfkH060iWP\\\\\\\/3a0snZg9OzwuKdZCxH2Sxp+ydsXDVjJdfRaCdJBizlscWpiy7SlIK9ylLSOdzh6S0uCB8TdiQrH\\\\\\\/dBe0PmrHYnMyz3lFFFWk5JAjvnQScikXig2VFem+c1QlpI6Gg3h0\\\\\\\/F38r4FqJkHOFbNyjHtoJBku+Zq3R9SJ8xMDeVRnjHGORDjbVOZmd1A7deSAFazThnVnVK\\\\\\\/D1SrSKuBcpQfxJts1WCgawfKiJZjn1AP+vlodxcyxC\\\\\\\/QNbh8hYEWb9TRnl813nS\\\\\\\/9Tok3cIOIdGIlFdmHOD7xGGKd2KBdOsM5SCEJn4E4GdTn0OksiUtXHmDmvvTPG7ZSq2dFsjp\\\\\\\/Ju1tgbwm8MhlEGHGxZ9rwZAxGPpxE9+BvlsDXtkoOAtz98BMMBvMypGO2n4aXiTBEp4dBAVqcKVFGJXmJAkkOO1sQeVh8fG6YyVCJZhp4TdLc+lxZwu1hrhIl8SMVtwLHRyd4wQli3RbHxZlRQ+wz4aPF4szLLJhU1vMSjIIFgjMXOvDsqS0J4LQW5rcFUifz\\\\\\\/bMUPXpH+uJViv9xB6RFtDaQscsvVVHe7UcY2ajBrQo0XkMEZB6YOI1QzR3kQFXnnxN2dm+QkwuhkhhSENag4dGCupmXTLj3Q7yRcya+zqvEK+SgUCUqbFZs\\\\\\\/TRqgi454hx2UKDQ105zEjGuVYAL1bHnWeXTrXgKGY+Pacm8Zx8REYY7jyCGWl3WeDd33JVywiZWEtOnPTBroVExb7KEYw1FDdDZEWB4ciFD2b833kCLgPHlqvJiVz+9mXIFqwR9zyvCWXrIQihR4INZsIKXtKU1GVRKC8XCZgdVK+ToRt2GuOPqnDBf2xiPWYja49pbmJZVxdgUOBaiPotTbYEi3OvQwDWelMUgzeuABB1DlI4h6n+D7WMmJdfWDVs07ejm8j3wfyDazuOOId6MMC4MExPvPUp\\\\\\\/0fFb0Tcv8vXA8tLYb+ijeDn6uYEJOFIlPr4ao05sJT3plKV9ZpmWiG34hH3wlVUxoDGUI3pLP+NXUofzP\\\\\\\/QOcvDV3wojr4HGQ1ukCmz56p45SAxuPVZPqq7\\\\\\\/6ViedaYPU\\\\\\\/rdXSi+bB7LAyuugUMl+3UShVYNQOECgDAntsYZ2oDWzYeqrco6CD1MHr3qfShYpC5bKnhIas1SksWi2kE81yIugLtye+F+QTuyCItIc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022tT4ZY3cn7a+Dum3KaRKLfQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228b6668ba950b7afe6c44dd93be5cc3a55c92c391b5dddf834f39cfe2770e75d0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487101 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3757" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aiaZaQazwsrMLczNEheyjwFLQB4_s0MF9j4_uIKm9AE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3757" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Zf3K0f5TdRFwFPB-aA24eKdxSFJcQcEi_0WhAkII_Ts" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3745" title="Serotonin Drives Predatory Feeding Behavior via Synchronous Feeding Rhythms in the Nematode Pristionchus pacificus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3775" title="Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487101" id="node487101" data-pisa="ggg;7/11/3757" data-pisa-master="ggg;g3.117.300223" data-apath="/ggg/7/11/3757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Localization of Cdc7 Protein Kinase During DNA Replication in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Rossbach</span>, <span class="highwire-citation-author" data-delta="1">D. Suzi Bryan</span>, <span class="highwire-citation-author" data-delta="2">Jay R. Hesselberth</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-1100-6489" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert Sclafani</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3757-3774;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300223 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487101" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Rossbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BRossbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rossbach%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BRossbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. Suzi Bryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BSuzi%2BBryan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryan%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BSuzi%2BBryan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jay R. Hesselberth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jay%2BR.%2BHesselberth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hesselberth%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJay%2BR.%2BHesselberth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert Sclafani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BSclafani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sclafani%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BSclafani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1100-6489" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert Sclafani</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:robert.sclafani@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">robert.sclafani@ucdenver.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3757" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487101" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487101/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3757.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487101" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487101/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3757.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487101" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487101/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3757.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487101" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487101/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3757.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487101" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487101/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">DDK, a conserved serine-threonine protein kinase composed of a regulatory subunit, <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a>, and a catalytic subunit, <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>, is essential for DNA replication initiation during S phase of the cell cycle through MCM2-7 helicase phosphorylation. The biological significance of DDK is well characterized, but the full mechanism of how DDK associates with substrates remains unclear. <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is bound to chromatin in the <em>Saccharomyces cerevisiae</em> genome throughout the cell cycle, but there is little empirical evidence as to specific <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding locations. Using biochemical and genetic techniques, this study investigated the specific localization of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> on chromatin. The Calling Cards method, using Ty5 retrotransposons as a marker for DNA–protein binding, suggests <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> kinase is preferentially bound to genomic DNA known to replicate early in S phase, including centromeres and origins of replication. We also discovered <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding throughout the genome, which may be necessary to initiate other cellular processes, including meiotic recombination and translesion synthesis. A kinase dead <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> point mutation increases the Ty5 retrotransposon integration efficiency and a 55-amino acid C-terminal truncation of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>, unable to bind <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a>, reduces <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding suggesting a requirement for <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> to stabilize <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> on chromatin during S phase. Chromatin immunoprecipitation demonstrates that <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding near specific origins changes during S phase. Our results suggest a model where <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is loosely bound to chromatin during G<sub>1</sub>. At the G<sub>1</sub>/S transition, <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding to chromatin is increased and stabilized, preferentially at sites that may become origins, in order to carry out a variety of cellular processes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/replication" class="hw-term hw-article-keyword hw-article-keyword-replication" rel="nofollow">replication</a></li><li class="kwd"><a href="/keyword/kinase" class="hw-term hw-article-keyword hw-article-keyword-kinase" rel="nofollow">kinase</a></li><li class="kwd"><a href="/keyword/calling-cards" class="hw-term hw-article-keyword hw-article-keyword-calling-cards" rel="nofollow">calling cards</a></li><li class="kwd"><a href="/keyword/chromatin" class="hw-term hw-article-keyword hw-article-keyword-chromatin" rel="nofollow">chromatin</a></li><li class="kwd"><a href="/keyword/origins" class="hw-term hw-article-keyword hw-article-keyword-origins" rel="nofollow">origins</a></li></ul><p id="p-4">DNA replication in eukaryotes is a tightly regulated process that ensures the genome is duplicated once and only once during the cell cycle. Failure to control replication mechanisms leads to chromosome instability, aneuploidy, and mutations, all hallmarks of many human diseases (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rossbach and Sclafani 2016</a>). DNA replication is initiated at specific segments of genomic DNA, termed origins of replication, during S phase of the cell cycle. In budding yeast, <em>Saccharomyces</em> <em>cerevisiae</em>, origins are known as ARSs (autonomous replicating sequences) and contain an 11 bp consensus sequence termed the ACS (ARS consensus sequence) that serves as a docking site for replisome proteins (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sclafani and Holzen 2007</a>). Origins are classified by two key characteristics: efficiency and timing. Origin efficiency is determined by how frequently the origin initiates replication and whether it is used in a population of cells during every cellular division. Origin timing is a temporal reflection of when a given origin initiates replication during S phase. Origins can initiate replication throughout S phase and the time they initiate is quantified by the <em>T<sub>rep</sub></em> (timing of replication), the time it takes to duplicate 50% of the DNA (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Raghuraman <em>et al.</em> 2001</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yabuki <em>et al.</em> 2002</a>).</p><p id="p-5">Initiation of DNA replication is executed in distinct steps: origin licensing and origin firing. During origin licensing, ARSs are bound throughout the cell cycle by the ATP-dependent ORC (origin recognition complex) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bell and Dutta 2002</a>). The Pre-RC (prerecognition complex) is formed when the MCM complex (minichromosome maintenance helicase), composed of six paralogous proteins <a href="http://www.yeastgenome.org/locus/S000000119/overview">Mcm2</a>-7, is loaded onto double-stranded DNA in G<sub>1</sub> phase of the cell cycle by <a href="http://www.yeastgenome.org/locus/S000003807/overview">Cdt1</a> and <a href="http://www.yeastgenome.org/locus/S000003730/overview">Cdc6</a> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Labib 2010</a>). The MCM helicase loads as an inactive double hexamer during the licensing step of DNA replication (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Forsburg 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Frigola <em>et al.</em> 2013</a>). The origin firing step of DNA replication begins when the MCM helicase is activated in S phase by the sequential action of two kinases, CDK (cyclin-dependent kinase) and DDK (<a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a>-dependent kinase), forming the Pre-IC (preinitiation complex) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Johnston <em>et al.</em> 1999</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nougarede <em>et al.</em> 2000</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Heller <em>et al.</em> 2011</a>). The concerted action of the two kinases leads to recruitment of <a href="http://www.yeastgenome.org/locus/S000004093/overview">Cdc45</a>, the GINS complex, as well as the polymerases to form the active replisome. Once the replisome is formed, the MCM helicase unwinds DNA and uses polymerases to replicate DNA.</p><p id="p-6">DDK is a serine-threonine protein kinase, highly conserved in eukaryotes from yeast to humans (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sato <em>et al.</em> 1997</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hess <em>et al.</em> 1998</a>) and is composed of a regulatory subunit, <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a>, and a catalytic subunit, <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hartwell 1971</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Patterson <em>et al.</em> 1986</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hollingsworth and Sclafani 1990</a>). Each subunit of DDK contains conserved motifs and domains necessary for binding the other subunit and holding the complex together (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hughes <em>et al.</em> 2012</a>). In addition to essential kinase domains conserved in all eukaryotes, budding yeast <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> has a unique 55 amino acid C-terminal domain required for binding <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jackson <em>et al.</em> 1993</a>). This C-terminal domain is not found in homologous <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> proteins indicating there are alternate binding mechanisms required to bind <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> to <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> in other eukaryotes (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jackson <em>et al.</em> 1993</a>).</p><p id="p-7"><a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> and <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein subunits are regulated differently during the cell cycle. The <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein is stably expressed and subsequently bound to chromatin throughout the entire cell cycle (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Weinreich and Stillman 1999</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ferreira <em>et al.</em> 2000</a>). Conversely, <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein expression oscillates throughout the cell cycle due to transcriptional regulation and protein stability regulation by the APC (anaphase-promoting complex) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 1999</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Oshiro <em>et al.</em> 1999</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Weinreich and Stillman 1999</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Ferreira <em>et al.</em> 2000</a>). When APC-dependent degradation of <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> ceases upon the cells’ transition from G<sub>1</sub> to S phase, <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein is stable and associates with <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> and chromatin immediately. At the completion of S phase, <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein is rapidly degraded again by the APC. Mutations in essential APC subunits or a <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> N-terminal region resembling destruction boxes eliminates APC-mediated degradation (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 1999</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Ferreira <em>et al.</em> 2000</a>). DDK activity is regulated by expression of the <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> subunit in much the same way CDKs are regulated by cyclin activity. In G<sub>1</sub> phase, when <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein is not expressed, catalytic activity of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is low. As <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> expression increases at the G<sub>1</sub>/S transition, the catalytic activity of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> increases and remains high throughout S phase, before decreasing when <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> is degraded at the end of S phase (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 1999</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Oshiro <em>et al.</em> 1999</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Weinreich and Stillman 1999</a>; <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Ferreira <em>et al.</em> 2000</a>).</p><p id="p-8">The primary functional role of DDK is to initiate DNA replication through the phosphorylation of the MCM helicase, primarily on subunits <a href="http://www.yeastgenome.org/locus/S000000119/overview">Mcm2</a>, <a href="http://www.yeastgenome.org/locus/S000006223/overview">Mcm4</a>, and <a href="http://www.yeastgenome.org/locus/S000003169/overview">Mcm6</a> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lei <em>et al.</em> 1997</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sheu and Stillman 2010</a>). DDK phosphorylation of the MCM helicase is proposed to induce a conformational change, and together with CDK activity, permits recruitment of <a href="http://www.yeastgenome.org/locus/S000004093/overview">Cdc45</a>, the GINS complex, and DNA polymerases thereby forming an activated replisome (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Labib 2010</a>). The requirement for DDK to activate the helicase can be bypassed by expression of an <a href="http://www.yeastgenome.org/locus/S000004264/overview">Mcm5</a> mutant, <em><a href="http://www.yeastgenome.org/locus/S000004264/overview">mcm5</a>-bob1</em>, that potentially modifies the structure of <a href="http://www.yeastgenome.org/locus/S000004264/overview">Mcm5</a> and mimics the phosphorylation-induced conformational change (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hardy <em>et al.</em> 1997</a>). In addition to the <em><a href="http://www.yeastgenome.org/locus/S000004264/overview">mcm5</a>-bob1</em> mutant, deletion or phosphomimetic alterations of DDK phosphorylation sites in the N-terminus of <a href="http://www.yeastgenome.org/locus/S000006223/overview">Mcm4</a> can also bypass <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> loss (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Randell <em>et al.</em> 2010</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Sheu and Stillman 2010</a>). Recent experiments tethering DDK to origins with <a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a> results in overall increases in origin efficiency, while FRAP shows DDK diffuses rapidly in the nucleus such that it randomly interacts with origins during S phase to initiate productive phosphorylation events (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Patel <em>et al.</em> 2008</a>). While DDK’s most well characterized role is in DNA replication, it also functions to regulate other chromatin-bound substrates in meiotic recombination, translesion synthesis (TLS) during DNA-damage-induced mutagenesis, chromosome cohesion, and centrosome duplication (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sasanuma <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Natsume <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brandao <em>et al.</em> 2014</a>). Therefore, <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>’s functional role in the cell is that of a master regulator to promote recruitment of other proteins that carry out these processes.</p><p id="p-9"><a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein has been shown by one hybrid screen (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dowell <em>et al.</em> 1994</a>) and chromatin immunoprecipitation (ChIP) in G<sub>1</sub>-arrested cells (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Katou <em>et al.</em> 2006</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Natsume <em>et al.</em> 2013</a>) to directly associate with origins of replication. While this has provided useful information for the <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> regulatory subunit, similar methods have not been able to successfully identify where the <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> subunit binds within the genome. The inability of conventional methods to associate endogenous <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> with origins of replication is likely due to the transient interaction of the kinase with substrates. In this study, we adapt established methods and modify conventional ones to understand binding of the <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> subunit within the genome.The Calling Cards method was developed to map binding sites of transcription factors within the budding yeast genome (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2008</a>). This method takes advantage of the Ty5 retrotransposon in budding yeast where Ty5 mRNA is reverse transcribed into double-stranded cDNA and translocated to the nucleus where the Ty5 integrase reintegrates Ty5 retrotransposons into the genome near heterochromatin and telomeric regions under the direction of the <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> heterochromatin protein. Calling Cards fuses transcription factors to the domain of the <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> heterochromatin protein that interacts with the Ty5 integrase. Therefore, Ty5 retrotransposon cDNA is integrated into the genome near to where a given transcription factor binds, leaving a permanent mark of its association with that region of the genome (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xie <em>et al.</em> 2001</a>). Furthermore, barcodes can be added to transcription factor fusion constructs to allow for analysis of multiple transcription factors in a single study (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wang <em>et al.</em> 2007</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">2011</a>). While initial studies used this method primarily for transcription factor binding sites, its design permits modification for analyzing binding sites of any DNA binding protein including replication factors.</p><p id="p-10">Conventional methods have been unable to precisely determine <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein binding sites at specific genomic locations, so we adapted Calling Cards for replication proteins to look at genome wide <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding by means of where Ty5 retrotransposon permanent integrations are made. An additional benefit of Calling Cards is its ability to analyze protein binding sites in dividing cells over multiple generations, where previous studies have been limited to arrested populations. Calling Cards fusion constructs were generated fusing the domain of <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> that interacts with Ty5 integrase to DNA replication factors <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> and <a href="http://www.yeastgenome.org/locus/S000003081/overview">Sld3</a>. These fusion constructs were expressed in budding yeast containing a Ty5 donor plasmid under control of the <a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a> promoter as well as a deletion of the <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> protein. Growth of these strains on media plates containing galactose induces transcription of Ty5 mRNA from the donor plasmid that is then reversed transcribed into cDNA. The donor Ty5 plasmid contains a <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> marker permitting selection of cells with genomically integrated Ty5 retrotransposons. Genomic DNA can be isolated, fragmented by restriction digestion and circularized by intramolecular ligation thereby recovering the DNA immediately flanking a Ty5 retrotransposon integration. The DNA flanking Ty5 retrotransposon integrations is amplified using inverse PCR, subjected to Illumina Sequencing, and mapped back to the budding yeast genome. This provides qualitative information as to where replication proteins bind chromatin as well as a quantitative measurement of how many Ty5 retrotransposons integrations are made.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains, plasmids, and media</h3><p id="p-11">Yeast strains and plasmids are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, respectively. <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a></em> (wild type) open reading frame was amplified for Calling Cards from pRS277 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) using primers A and B (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hollingsworth <em>et al.</em> 1992</a>). <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a></em>(<em>N168A</em>) open reading frame was made by PCR mutagenesis of pCH766 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) with primers J and K (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) and amplified for Calling Cards by primers A and B. <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a></em>(<em>N168A</em>)(<em>1–452</em>) was amplified for Calling Cards from pCH777 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) with primers A and C (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Calling Cards plasmids containing DNA replication factors fused to the Ty5 integrase interacting domain of <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> were made using previously published methods (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2008</a>). Fusion construct homologous recombination was done in yeast strain RSY452 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and and Sanger sequencing with primers L and M (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Fusion constructs were verified for functionality by protein expression and complementation of <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a></em> temperature sensitive mutant in RSY302 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Verified fusion constructs were transformed into yeast strain RSY1354 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) containing the Ty5 donor plasmid and selected on Glucose–Ura–Trp media plates. Yeast strains were grown according to standard procedures and Calling Cards selection was carried out according to previous methods (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2008</a>). For ChIP strains, pCH766 and pCH777 plasmids (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>) were transformed into yeast strain RSY1294 (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) and selected on –His media plates.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487256/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487256/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><em>S. cerevisiae</em> strains used</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487129/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487129/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Recombinant plasmids used</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487230/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487230/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Primer sequences</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Calling Cards efficiency assay</h3><p id="p-12">Calling Cards strains were grown in Glucose–Ura–Trp media and then plated on Galactose–Ura–Trp plates to induce Ty5 transcription into mRNA. Cells were replicated to YPD plates and removed from plates. An Invitrogen Countess was used to count cells for serial dilutions. Serial dilutions were plated on either YPD plates (for total viable cells) or –His FOA plates (selecting for Ty5 transpositions only) and colonies were counted. The ratio of colonies grown on –His FOA to the total number of cells on YPD was calculated to measure the efficiency of Ty5 transposition.</p></div><div id="sec-4" class="subsection"><h3>Calling Cards library preparation</h3><p id="p-13">Calling Cards libraries were prepared according to previous methods (<a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2008</a>) with the following modifications. 100 OD<sub>600</sub> genomic DNA was harvested using the following protocol: cells were resuspended in a solution containing 1 M Sorbitol, 50 mM Tris-HCl, pH = 8.0, and 10 mM 2-Mercaptoethanol (BME). Cells were spheroplasted with 0.7 mg/ml 100 K Zymolyase (US Biological) and shaken at 30° for 1 hr. The resultant spheroplasts were resuspended in a solution containing 1 M Sorbitol, 1% SDS, 20 mM EDTA and incubated at 55° overnight. Phenol:Choloform:Isoamyl alcohol (25:24:1) extraction followed by Chloroform:Isoamyl alcohol (24:1) extraction and RNase treatment (10 mg/ml) was done and the DNA was then precipitated with 3 M Sodium Acetate and 100% Ethanol. One microgram of genomic DNA was digested with <em>Hin</em>dIII, <em>Hpa</em>II, or <em>Taq</em>α1 (New England BioLabs). Digested DNA (200 ng) was used for intramolecular ligation reaction by T4 Ligase (New England BioLabs). Ligated DNA (20 µl) was used for inverse PCR amplification with 0.25 mM dNTPs, 0.125 µM of primer D (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>), 0.125 μM of primer E, F, G, or H (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>), Taq polymerase, and Taq buffer (35 mM MgCl<sub>2</sub>, 750 mM KCl, 0.1% TritonX-100, pH = 8.7). Inverse PCR amplification was done with 31 cycles of 95° for 30 sec, 60° for 30 sec, and 72° for 3 min. Each PCR reaction was diluted to 10 nM and the three digestion reactions for a given Calling Cards strain were pooled and submitted for sequencing on the Illumina GAII or MiSeq with primer I (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-5" class="subsection"><h3>Analysis of Calling Cards sequencing</h3><p id="p-14">Sequences were analyzed by alignment to a reference genome (<em>sacCer1</em>) using Bowtie (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Langmead <em>et al.</em> 2009</a>) and SAMtools (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2009</a>), processed to bedGraph format using BEDTools (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Quinlan and Hall 2010</a>), and visualized in the UCSC genome browser (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Karolchik <em>et al.</em> 2011</a>). Coverage at each position was normalized by the signal at each genomic position relative to the number of aligned reads in the library (<em>i.e.</em>, reads per million, RPM) to get quantitative information or normalized to the mean number of unique insertions to get specific positional information. Software and pipeline used to analyze Calling Cards data can be found at <a href="https://github.com/hesselberthlab/callingcards">https://github.com/hesselberthlab/callingcards</a>.</p></div><div id="sec-6" class="subsection"><h3>ChIP and quantitative PCR</h3><p id="p-15">ChIP was performed as previously described (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kogut <em>et al.</em> 2009</a>) with the following modifications. Yeast strains were grown in –His 2% Raffinose media overnight. Cells were diluted in YEP + 2% Raffinose + 2% Galactose to induce <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein overexpression and grown to 0.8 OD<sub>600</sub>. Cells were synchronized with 50 nM α factor (αF) for 4 hr before being released into media containing 10 mg/ml pronase. Fifty milliliters of cells was harvested before addition of αF to serve as an asynchronous sample, at <em>T</em> = 0 when pronase was added to serve as a G<sub>1</sub>-arrested population and at 15-min intervals for 90 min after pronase addition to look at different S phase time points. Immunoprecipitations were set up using 300 µl of chromatin solution and incubated with 25 µg anti-HA antibody (12CA5; Roche) for 15 hr at 4°. Three hundred microliters of chromatin solution was mock treated as no antibody control and 1.5 ml of chromatin solution was used as input. Protein G sepharose beads were incubated with immunoprecipitations (IPs) and no antibody controls for 1 hr at 4°. Beads were washed according to previously described methods (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ramey and Sclafani 2014</a>) and resuspended in 200 µl ChIP elution buffer and 40 µl TE then incubated overnight at 65°. RNase (3 mg/ml) was added to IPs and no antibody controls and incubated for a minimum of 30 min at 37°. Four microliters of 20 mg/ml Proteinase K was added to IPs and controls and incubated for 2 hr at 42°. Organic extraction of IPs and controls using Phenol:Choloform:Isoamyl alcohol (25:24:1) was followed by ethanol precipitation. IP and control pellets were resuspended in 150 µl TE and input pellets were resuspended in 300 µl TE. ChIP chromatin was subjected to quantitative PCR, using 2× sybergreen master mix from (Roche) and a Roche Lightcycler 480 thermocycler. ChIP chromatin was subjected to qPCR analysis with primers N-U (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aparicio <em>et al.</em> 1997</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Ramey and Sclafani 2014</a>).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-16">Strains and plasmids are available upon request. Supplemental Material, Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a> contains UCSC genome browser Calling Cards data for <a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a> loci on Chromosomes 2, 4, and 12. Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a> contains UCSC genome browser Calling Cards data at specific origins of replication. Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a> contains comparison of Calling Cards insertions events and transcription start sites or centromeres.</p><p id="p-17">Sequencing data generated in this study has been deposited in NCBI GEO under accession GSE103943. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103943">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103943</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Cdc7 kinase dead fusion constructs efficiently integrate Ty5 transposons into the genome</h3><p id="p-18">Calling Cards has been established using fusion constructs as being a sensitive and quantitative method for analyzing genomic DNA binding sites of transcription factors (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Wang <em>et al.</em> 2007</a>, <a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">2008</a>). In our Calling Cards experiments, the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a>-myc</em> fusion construct (designed by the Mayhew laboratory at Washington University–St. Louis) was used as a positive control (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). This construct was used as a template to make fusion constructs containing wild-type <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a></em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a>). In addition to wild-type <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a></em>, Calling Cards constructs were made using mutant <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a></em> alleles carrying mutations affecting its binding to chromatin. A kinase dead construct with a -bp point mutation in the catalytic domain of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> changing asparagine (N) to alanine (A) at amino acid 168 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>) and a kinase dead construct that also contains a truncation of the final 55 amino acid C-terminal residues were made (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1E</a>). Homologous recombination of the replication factors into the fusion construct was verified by Sanger sequencing and the construct functionality was verified by Western blot analysis using the myc tag in the fusion and complementation of a <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a>-7</em> temperature sensitive strain (data not shown).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Calling Cards construct efficiency. (A) GAL4 open reading frame fused to the sir4 domain that interacts with Ty5 integrase. This construct was donated to us by David Mayhew (Washington University in St. Louis). (B) Full length CDC7 (507 aa) was inserted into the GAL4-sir4-myc plasmid backbone to create sir4-fusion. (C) SLD3 fused to sir4. (D) Full length cdc7KD (507 aa) fused to sir4 contains a N168A kinase dead point mutation. (E) cdc7KD&#x394;CT fused to sir4 contains same N168A mutation and additionally lacks 55 C-terminal residues. (F) Yeast strain containing Ty5 transposon donor plasmid was transformed with the indicated DNA replication factor fusion constructs and tested for efficiency to integrate Ty5 transposons into genomic DNA by measuring the relative amount of His+ 5FOA resistant cells, in which transposition occurred." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Calling Cards construct efficiency. (A) &lt;em&gt;GAL4&lt;/em&gt; open reading frame fused to the &lt;em&gt;sir4&lt;/em&gt; domain that interacts with Ty5 integrase. This construct was donated to us by David Mayhew (Washington University in St. Louis). (B) Full length &lt;em&gt;CDC7&lt;/em&gt; (507 aa) was inserted into the &lt;em&gt;GAL4-sir4-myc&lt;/em&gt; plasmid backbone to create &lt;em&gt;sir4&lt;/em&gt;-fusion. (C) &lt;em&gt;SLD3&lt;/em&gt; fused to &lt;em&gt;sir4&lt;/em&gt;. (D) Full length &lt;em&gt;cdc7KD&lt;/em&gt; (507 aa) fused to &lt;em&gt;sir4&lt;/em&gt; contains a N168A kinase dead point mutation. (E) &lt;em&gt;cdc7KD&#x394;CT&lt;/em&gt; fused to &lt;em&gt;sir4&lt;/em&gt; contains same N168A mutation and additionally lacks 55 C-terminal residues. (F) Yeast strain containing Ty5 transposon donor plasmid was transformed with the indicated DNA replication factor fusion constructs and tested for efficiency to integrate Ty5 transposons into genomic DNA by measuring the relative amount of His+ 5FOA resistant cells, in which transposition occurred.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F1.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3757/F1.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487183" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Calling Cards construct efficiency. (A) <em>GAL4</em> open reading frame fused to the <em>sir4</em> domain that interacts with Ty5 integrase. This construct was donated to us by David Mayhew (Washington University in St. Louis). (B) Full length <em>CDC7</em> (507 aa) was inserted into the <em>GAL4-sir4-myc</em> plasmid backbone to create <em>sir4</em>-fusion. (C) <em>SLD3</em> fused to <em>sir4</em>. (D) Full length <em>cdc7KD</em> (507 aa) fused to <em>sir4</em> contains a N168A kinase dead point mutation. (E) <em>cdc7KDΔCT</em> fused to <em>sir4</em> contains same N168A mutation and additionally lacks 55 C-terminal residues. (F) Yeast strain containing Ty5 transposon donor plasmid was transformed with the indicated DNA replication factor fusion constructs and tested for efficiency to integrate Ty5 transposons into genomic DNA by measuring the relative amount of His+ 5FOA resistant cells, in which transposition occurred.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">To validate the efficacy of Calling Cards using DNA replication factor proteins, the efficiency of each fusion construct to integrate Ty5 retrotransposons into the genome was determined (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1F</a>). Integration efficiency of each replication factor fusion construct was compared to the efficiency of a strain containing only the Ty5 donor plasmid without a <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> fusion construct as well as to the efficiency of the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> transcription factor fusion construct. Strain RSY1354 (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), containing only the <em>Ty5-<a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> donor plasmid under the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a></em> promoter without a <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> fusion construct, integrated Ty5 transposons inefficiently into the genome at a rate of 10<sup>−9</sup>/cell. In contrast, strain DRY110 (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>), containing the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion construct, integrated Ty5 transposons efficiently into the genome at a rate of 5 × 10<sup>−4</sup>/cell, comparable to the previously reported value 10<sup>−5</sup>/cell (<a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2008</a>).</p><p id="p-21">The efficiency of the DNA replication factor fusion constructs is expected to be within the range set by background Ty5 transposon integrations, the negative control, and <a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a>-directed Ty5 transposon integrations, the positive control. Strains DRY101 (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>), containing the wild-type <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a>-myc</em> fusion construct, and RSY1352, containing the <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a>-myc</em> fusion construct, integrate Ty5 transposons at rates of 4 × 10<sup>−5</sup>/cell and 7 × 10<sup>−5</sup>/cell, respectively. Compared to <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusions integrate Ty5 transposons less efficiently.</p><p id="p-22">Strain DRY113 (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>) contains the kinase dead <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a></em>(<em>N168A</em>)<em>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a>-myc</em> fusion construct (<em><a href="http://www.yeastgenome.org/locus/S000000222/overview">cdc7</a>KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>) that prevents <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> from phosphorylating MCM helicases (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Hardy and Pautz 1996</a>). This construct was used to potentially increase Ty5 transposon integration events compared to the wild-type <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a></em> fusion construct with the rationale that blocking substrate phosphorylation would result in <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein kinase remaining bound to the substrate longer. As shown originally in the crystal structure of Protein Kinase A, the conserved asparagine residue is essential for catalytic activity as it interacts directly with the catalytic basic residue (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Knighton <em>et al.</em> 1991</a>). Thus, the <em><a href="http://www.yeastgenome.org/locus/S000000222/overview">cdc7</a></em>(N168A) protein still binds ATP but is unable to transfer phosphate to the substrate. Increased binding of substrates <em>in vivo</em> by protein kinase dead mutants has been shown for both MEK5 kinase (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 1996</a>) and for <a href="http://www.yeastgenome.org/locus/S000006074/overview">Rad53</a> kinase (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Holzen and Sclafani 2010</a>). Ty5 integration efficiency of <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> was determined to be 3 × 10<sup>−4</sup>/cell, sevenfold more than wild-type <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and similar to the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> efficiency, consistent with increased binding by the <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> kinase dead protein.</p><p id="p-23">Strain DRY123 contains the kinase dead C-terminal truncation <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> fusion construct <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a></em>(<em>N168A</em>)(<em>1–452</em>)<em>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a>-myc</em> fusion (<em><a href="http://www.yeastgenome.org/locus/S000000222/overview">cdc7</a>KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>), which prevents essential <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> binding to <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>. This construct was used to investigate the role of <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> in localizing <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> to chromatin (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Jackson <em>et al.</em> 1993</a>). <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> had a Ty5 transposon integration efficiency of 1.69 × 10<sup>−4</sup>/cell, fourfold higher than <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a></em>. Our calculated efficiencies suggest <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> constructs are substantially better at integrating the Ty5 transposon into the genome compared to <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>. As postulated, the difference in efficiency is likely due to the extended binding of the kinase dead protein to substrates. Given their higher integration efficiencies, <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>, <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>, and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> were used in the Calling Cards experiment to determine <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding to chromatin in the budding yeast genome, while <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> will largely be ignored as a result of their low integration efficiency.</p></div><div id="sec-10" class="subsection"><h3>Cdc7 fusion constructs integrate Ty5 transposons throughout the genome</h3><p id="p-24">Upon high-throughput sequencing of genomic DNA flanking Ty5 integrations, the number of integration events retrieved for each construct was determined (<em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> ∼5000 insertions, <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> ∼9000 insertions, <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a></em> ∼5600 insertions and Ty5 background ∼2100 insertions). <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusions resulted in more Ty5 transposon integrations throughout the genome compared to the <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> constructs and to the strain containing only the Ty5 donor plasmid as expected due to their low integration inefficiencies. We show chromosome 4 from the UCSC genome browser as a representation for where each fusion construct integrated Ty5 retrotransposons (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Each insertion event represents a location where <a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a> or <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> proteins were associated with chromatin. We normalized the Calling Cards signal using two different methods. Initial Calling Cards analysis was done by taking the number of reads at each genomic position (the signal) and normalizing the signal to the total number of aligned reads in the library for a given construct which provides output as RPM. Here, each individual peak represents an insertion event at a specific location, while the height of each peak at a given location is the signal amplitude or the number of reads of that sequence due to either multiple insertion events at the same location or inverse PCR amplification bias of a single insertion event. As the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion construct was used as a template to create the other construct, the backbone of the constructs were not changed to include unique molecular identifiers that would have allowed separation between unique insertion events and insertion events that had inherent amplification bias in our PCR step. The second analysis normalized Calling Cards to the mean number of unique insertion events, which ignores quantitative information and only provides information as to where insertions occur. If an integration event occurs, the “count” is 1; if no integrations occur, the “count” is 0. Here, each peak is the same height, and we only look at unique positions where insertions occur.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Calling Cards sequences mapped to S. cerevisiae genome. Ty5 transposons integrated by DNA replication factor fusion constructs were isolated, sequenced, and mapped to the yeast genome. Chromosome 4 from the UCSC genome browser is used to show where Ty5 transposon integration marks occur for each DNA replication factor fusion construct. Orc1 and Mcm2 chromatin immunoprecipitation (Xu et al. 2006) and replication timing (Yabuki et al. 2002) are plotted for reference. (A) Calling Cards signal at each position normalized to the number of aligned reads in the library (reads per million, RPM). The X-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the y-axis and height of each line is the normalized signal at each position. (B) Normalized to mean number of unique insertions events. Removes quantitative information and only displays unique position of insertion events in the genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Calling Cards sequences mapped to &lt;em&gt;S. cerevisiae&lt;/em&gt; genome. Ty5 transposons integrated by DNA replication factor fusion constructs were isolated, sequenced, and mapped to the yeast genome. Chromosome 4 from the UCSC genome browser is used to show where Ty5 transposon integration marks occur for each DNA replication factor fusion construct. Orc1 and Mcm2 chromatin immunoprecipitation (Xu &lt;em&gt;et al.&lt;/em&gt; 2006) and replication timing (Yabuki &lt;em&gt;et al.&lt;/em&gt; 2002) are plotted for reference. (A) Calling Cards signal at each position normalized to the number of aligned reads in the library (reads per million, RPM). The &lt;em&gt;X&lt;/em&gt;-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the &lt;em&gt;y&lt;/em&gt;-axis and height of each line is the normalized signal at each position. (B) Normalized to mean number of unique insertions events. Removes quantitative information and only displays unique position of insertion events in the genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F2.medium.gif" width="399" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3757/F2.medium.gif" width="399" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487274" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Calling Cards sequences mapped to <em>S. cerevisiae</em> genome. Ty5 transposons integrated by DNA replication factor fusion constructs were isolated, sequenced, and mapped to the yeast genome. Chromosome 4 from the UCSC genome browser is used to show where Ty5 transposon integration marks occur for each DNA replication factor fusion construct. Orc1 and Mcm2 chromatin immunoprecipitation (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Xu <em>et al.</em> 2006</a>) and replication timing (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Yabuki <em>et al.</em> 2002</a>) are plotted for reference. (A) Calling Cards signal at each position normalized to the number of aligned reads in the library (reads per million, RPM). The <em>X</em>-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the <em>y</em>-axis and height of each line is the normalized signal at each position. (B) Normalized to mean number of unique insertions events. Removes quantitative information and only displays unique position of insertion events in the genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">When normalized to the number of aligned reads (RPM) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion, as a positive control for Calling Cards, targeted Ty5 transposon integration throughout the genome as well as at all of its known major binding sites. <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> inserted multiple Ty5 transposons at the promoters of each galactose metabolic gene including <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000223/overview">GAL10</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000222/overview">GAL7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004071/overview">GAL2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002416/overview">GAL3</a></em> as expected (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a>). In addition to the galactose metabolic genes, <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> integrated Ty5 transposons at other known <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a></em> binding regions including the promoters of <em><a href="http://www.yeastgenome.org/locus/S000000225/overview">FUR4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005646/overview">GCY1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003102/overview">PCL10</a></em> genes, among others (data not shown). The negative control strain containing the Ty5 donor plasmid without a <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> fusion construct reduced the signal of integrated Ty5 transposons in the genome dramatically compared to the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion. Due to their low integration efficiency, <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003081/overview">SLD3</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusions integrated very few Ty5 transposons into the genome. We predict there is an integration efficiency threshold required for genomic coverage of potential Ty5 transposon integration events and these two constructs do not achieve the necessary threshold. Due to their high integration efficiency, <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusions are above the threshold and provide enough insertion events to significantly cover the genome. The two kinase dead constructs integrated Ty5 transposons throughout the entire budding yeast genome, similar to the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion. To see the RPM signal of Ty5 transposon integrations across all 16 chromosomes, please use the following link to the interactive UCSC genome browser: <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&amp;hgS_otherUserName=drossbach&amp;hgS_otherUserSessionName=Cdc7%20Calling%20Card%20Paper">http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&amp;hgS_otherUserName=drossbach&amp;hgS_otherUserSessionName=Cdc7%20Calling%20Card%20Paper</a>.</p><p id="p-27">Upon normalization to the mean number of unique insertions events (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>), Ty5 transposon integrations are still made throughout entire chromosomes by the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>, <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>, and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusions. To see the mean number of unique insertions across all 16 chromosomes please use the following link to the interactive UCSC genome browser: <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&amp;hgS_otherUserName=drossbach&amp;hgS_otherUserSessionName=Calling%20Card%20Not%20%23%20reads%20paper">http://genome.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&amp;hgS_otherUserName=drossbach&amp;hgS_otherUserSessionName=Calling%20Card%20Not%20%23%20reads%20paper</a>. Using both normalization techniques, Ty5 transposon integrations are found throughout the genome and not specifically at origins of replication. <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding to chromatin at nonorigin DNA may be indicative of its role in other cellular processes such as chromatid cohesion, TLS, and meiotic recombination.</p><p id="p-28">As <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>’s main role is in DNA replication, we expect Ty5 transposon integrations to occur specifically at origins of replication as a result of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding to origins. Specific origins of replication were chosen in the UCSC genome browser to determine if Ty5 transposon integrations are made in relation to origins (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a>). This analysis showed some origins contain Ty5 transposon integrations nearby (<a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a>, <a href="http://www.yeastgenome.org/locus/S000028482/overview">ARS305</a>), while other origins do not contain integrations or contain integrations &gt;1 kb away (<a href="http://www.yeastgenome.org/locus/S000028483/overview">ARS306</a>, <a href="http://www.yeastgenome.org/locus/S000028447/overview">ARS1010</a>, <a href="http://www.yeastgenome.org/locus/S000007639/overview">ARS607</a>) from the origin, which likely indicates a different <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding site than the origin itself. Additionally, some Ty5 transposon integrations are made in genomic DNA that are known to bind the MCM complex, but do not bind the ORC complex (<a href="http://www.yeastgenome.org/locus/S000028485/overview">ARS308</a>). Using the UCSC genome browser, there does appear to be a pattern as to which origins are susceptible to Ty5 transposon integration events.</p><p id="p-29">As the UCSC genome browser is only useful for looking at chromosome-wide Ty5 transposon integrations, meta-analysis was performed to make further comparisons and determine if a difference exists in Ty5 transposon integration events from <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>. Our meta-analysis focused on whether <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>-directed Ty5 transposon integrations are dependent on <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> binding, if Ty5 integrations correlate to chromosomal replication features including timing, ARS sites, and replication proteins, and whether correlations exist with nonreplication chromosomal features including transcription start sites and centromeres.</p></div><div id="sec-11" class="subsection"><h3>Cdc7 directs Ty5 transposon insertions into early replicating DNA</h3><p id="p-30">To further understand the binding differences of kinase dead <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> constructs to chromatin, Ty5 transposon integration events were compared to known DNA replication features. Calling Cards data were first organized by timing of genomic DNA replication across entire chromosomes as defined by previous microarray data sets (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Raghuraman <em>et al.</em> 2001</a>). Newer sequence-based replication timing data (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Muller <em>et al.</em> 2014</a>) was looked at for comparison and as no significant differences existed from microarray data sets, Calling Cards data were not reanalyzed with these data. The replication timing data set was divided into 5-min windows by <em>T<sub>rep</sub></em> and Calling Cards signal in each window was determined.</p><p id="p-31">When normalized to the number of aligned reads (RPM) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A–C</a>), the negative control containing the Ty5 donor plasmid had low Ty5 transposon integration signal across each 5-min replication timing window as expected due to its low integration efficiency. Ty5 transposon integration signal due to the <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> positive control was higher than the negative control and highest in regions of genomic DNA that are known to replicate in mid-S phase and low in regions of genomic DNA that replicate early or late in S phase suggesting integration events due to <a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a> binding do not correlate well with replication timing. Ty5 transposon integration signal due to <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion was higher than background and highest in regions of DNA that are the earliest replicating. The signal steadily decreased in regions that replicate late in S phase. Previous experiments showed the majority of yeast origins initiate replication within the first 20–40 min of S phase and as S phase progresses fewer origins are needed to initiate replication (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yang <em>et al.</em> 2010</a>). Thus, in the same timing windows when the majority of origins initiate replication, the most Ty5 integrations were made due to <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding chromatin. <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion had a similar trend to <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> where Calling Cards signal peaked in regions of early replicating genomic DNA and was low in late replicating DNA. The Ty5 transposon integration signal from <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion was significantly lower than that of the <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion in the 15- to 30-min range. Removing the C-terminal of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>, a known essential domain for <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> binding, may have a negative effect on the amount of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> bound to chromatin suggesting a role for <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> in stabilizing <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>-chromatin binding.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ty5 transposons from Cdc7 insert into early replicating DNA. Ty5 transposon integrations are compared to timing of replication of the genome. Replication timing microarray data sets were used (Raghuraman et al. 2001). The genome was divided into 5-min windows based on the overall timing of replication. Each box is a representation of the Calling Cards signal of a given construct in a specific replication timing window. Calling Cards signal at each position was normalized to the number of aligned reads in the library (reads per million, RPM). The X-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the y-axis and height of each line is the normalized signal at each position. (A) RPM signal comparing each fusion construct to negative control Ty5 integrations. (B) RPM signal comparing Cdc7 fusion constructs to Gal4 fusion construct. (C) RPM signal comparing cdc7KD-sir4 fusion construct to cdc7KD&#x394;CT-sir4 fusion construct. Calling Cards was normalized to mean number of unique insertions events. (D) Mean number of unique insertions comparing each fusion construct to negative control Ty5 integrations. (E) Mean number of unique integrations comparing Cdc7 fusion constructs to Gal4 fusion construct. (F) Mean number of unique integrations comparing cdc7KD-sir4 fusion construct to cdc7KD&#x394;CT-sir4 fusion construct. Statistical analysis was done using the nonparametric Wilcoxon test as we were not convinced our data followed a normal distribution. *P = 0.01&#x2013;0.05; ** P = 0.001&#x2013;0.01; *** P &lt; 0.001; NS, not significant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ty5 transposons from Cdc7 insert into early replicating DNA. Ty5 transposon integrations are compared to timing of replication of the genome. Replication timing microarray data sets were used (Raghuraman &lt;em&gt;et al.&lt;/em&gt; 2001). The genome was divided into 5-min windows based on the overall timing of replication. Each box is a representation of the Calling Cards signal of a given construct in a specific replication timing window. Calling Cards signal at each position was normalized to the number of aligned reads in the library (reads per million, RPM). The &lt;em&gt;X&lt;/em&gt;-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the &lt;em&gt;y&lt;/em&gt;-axis and height of each line is the normalized signal at each position. (A) RPM signal comparing each fusion construct to negative control Ty5 integrations. (B) RPM signal comparing Cdc7 fusion constructs to Gal4 fusion construct. (C) RPM signal comparing &lt;em&gt;cdc7KD-sir4&lt;/em&gt; fusion construct to &lt;em&gt;cdc7KD&#x394;CT-sir4&lt;/em&gt; fusion construct. Calling Cards was normalized to mean number of unique insertions events. (D) Mean number of unique insertions comparing each fusion construct to negative control Ty5 integrations. (E) Mean number of unique integrations comparing Cdc7 fusion constructs to Gal4 fusion construct. (F) Mean number of unique integrations comparing &lt;em&gt;cdc7KD-sir4&lt;/em&gt; fusion construct to &lt;em&gt;cdc7KD&#x394;CT-sir4&lt;/em&gt; fusion construct. Statistical analysis was done using the nonparametric Wilcoxon test as we were not convinced our data followed a normal distribution. *&lt;em&gt;P&lt;/em&gt; = 0.01&#x2013;0.05; ** &lt;em&gt;P&lt;/em&gt; = 0.001&#x2013;0.01; *** &lt;em&gt;P&lt;/em&gt; &lt; 0.001; NS, not significant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-3.medium.gif" width="305" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-3.medium.gif" width="305" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ty5 transposons from Cdc7 insert into early replicating DNA. Ty5 transposon integrations are compared to timing of replication of the genome. Replication timing microarray data sets were used (Raghuraman et al. 2001). The genome was divided into 5-min windows based on the overall timing of replication. Each box is a representation of the Calling Cards signal of a given construct in a specific replication timing window. Calling Cards signal at each position was normalized to the number of aligned reads in the library (reads per million, RPM). The X-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the y-axis and height of each line is the normalized signal at each position. (A) RPM signal comparing each fusion construct to negative control Ty5 integrations. (B) RPM signal comparing Cdc7 fusion constructs to Gal4 fusion construct. (C) RPM signal comparing cdc7KD-sir4 fusion construct to cdc7KD&#x394;CT-sir4 fusion construct. Calling Cards was normalized to mean number of unique insertions events. (D) Mean number of unique insertions comparing each fusion construct to negative control Ty5 integrations. (E) Mean number of unique integrations comparing Cdc7 fusion constructs to Gal4 fusion construct. (F) Mean number of unique integrations comparing cdc7KD-sir4 fusion construct to cdc7KD&#x394;CT-sir4 fusion construct. Statistical analysis was done using the nonparametric Wilcoxon test as we were not convinced our data followed a normal distribution. *P = 0.01&#x2013;0.05; ** P = 0.001&#x2013;0.01; *** P &lt; 0.001; NS, not significant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ty5 transposons from Cdc7 insert into early replicating DNA. Ty5 transposon integrations are compared to timing of replication of the genome. Replication timing microarray data sets were used (Raghuraman &lt;em&gt;et al.&lt;/em&gt; 2001). The genome was divided into 5-min windows based on the overall timing of replication. Each box is a representation of the Calling Cards signal of a given construct in a specific replication timing window. Calling Cards signal at each position was normalized to the number of aligned reads in the library (reads per million, RPM). The &lt;em&gt;X&lt;/em&gt;-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the &lt;em&gt;y&lt;/em&gt;-axis and height of each line is the normalized signal at each position. (A) RPM signal comparing each fusion construct to negative control Ty5 integrations. (B) RPM signal comparing Cdc7 fusion constructs to Gal4 fusion construct. (C) RPM signal comparing &lt;em&gt;cdc7KD-sir4&lt;/em&gt; fusion construct to &lt;em&gt;cdc7KD&#x394;CT-sir4&lt;/em&gt; fusion construct. Calling Cards was normalized to mean number of unique insertions events. (D) Mean number of unique insertions comparing each fusion construct to negative control Ty5 integrations. (E) Mean number of unique integrations comparing Cdc7 fusion constructs to Gal4 fusion construct. (F) Mean number of unique integrations comparing &lt;em&gt;cdc7KD-sir4&lt;/em&gt; fusion construct to &lt;em&gt;cdc7KD&#x394;CT-sir4&lt;/em&gt; fusion construct. Statistical analysis was done using the nonparametric Wilcoxon test as we were not convinced our data followed a normal distribution. *&lt;em&gt;P&lt;/em&gt; = 0.01&#x2013;0.05; ** &lt;em&gt;P&lt;/em&gt; = 0.001&#x2013;0.01; *** &lt;em&gt;P&lt;/em&gt; &lt; 0.001; NS, not significant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-4.medium.gif" width="295" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-4.medium.gif" width="295" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ty5 transposons from Cdc7 insert into early replicating DNA. Ty5 transposon integrations are compared to timing of replication of the genome. Replication timing microarray data sets were used (Raghuraman et al. 2001). The genome was divided into 5-min windows based on the overall timing of replication. Each box is a representation of the Calling Cards signal of a given construct in a specific replication timing window. Calling Cards signal at each position was normalized to the number of aligned reads in the library (reads per million, RPM). The X-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the y-axis and height of each line is the normalized signal at each position. (A) RPM signal comparing each fusion construct to negative control Ty5 integrations. (B) RPM signal comparing Cdc7 fusion constructs to Gal4 fusion construct. (C) RPM signal comparing cdc7KD-sir4 fusion construct to cdc7KD&#x394;CT-sir4 fusion construct. Calling Cards was normalized to mean number of unique insertions events. (D) Mean number of unique insertions comparing each fusion construct to negative control Ty5 integrations. (E) Mean number of unique integrations comparing Cdc7 fusion constructs to Gal4 fusion construct. (F) Mean number of unique integrations comparing cdc7KD-sir4 fusion construct to cdc7KD&#x394;CT-sir4 fusion construct. Statistical analysis was done using the nonparametric Wilcoxon test as we were not convinced our data followed a normal distribution. *P = 0.01&#x2013;0.05; ** P = 0.001&#x2013;0.01; *** P &lt; 0.001; NS, not significant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ty5 transposons from Cdc7 insert into early replicating DNA. Ty5 transposon integrations are compared to timing of replication of the genome. Replication timing microarray data sets were used (Raghuraman &lt;em&gt;et al.&lt;/em&gt; 2001). The genome was divided into 5-min windows based on the overall timing of replication. Each box is a representation of the Calling Cards signal of a given construct in a specific replication timing window. Calling Cards signal at each position was normalized to the number of aligned reads in the library (reads per million, RPM). The &lt;em&gt;X&lt;/em&gt;-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the &lt;em&gt;y&lt;/em&gt;-axis and height of each line is the normalized signal at each position. (A) RPM signal comparing each fusion construct to negative control Ty5 integrations. (B) RPM signal comparing Cdc7 fusion constructs to Gal4 fusion construct. (C) RPM signal comparing &lt;em&gt;cdc7KD-sir4&lt;/em&gt; fusion construct to &lt;em&gt;cdc7KD&#x394;CT-sir4&lt;/em&gt; fusion construct. Calling Cards was normalized to mean number of unique insertions events. (D) Mean number of unique insertions comparing each fusion construct to negative control Ty5 integrations. (E) Mean number of unique integrations comparing Cdc7 fusion constructs to Gal4 fusion construct. (F) Mean number of unique integrations comparing &lt;em&gt;cdc7KD-sir4&lt;/em&gt; fusion construct to &lt;em&gt;cdc7KD&#x394;CT-sir4&lt;/em&gt; fusion construct. Statistical analysis was done using the nonparametric Wilcoxon test as we were not convinced our data followed a normal distribution. *&lt;em&gt;P&lt;/em&gt; = 0.01&#x2013;0.05; ** &lt;em&gt;P&lt;/em&gt; = 0.001&#x2013;0.01; *** &lt;em&gt;P&lt;/em&gt; &lt; 0.001; NS, not significant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-5.medium.gif" width="299" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-5.medium.gif" width="299" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F3/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Ty5 transposons from Cdc7 insert into early replicating DNA. Ty5 transposon integrations are compared to timing of replication of the genome. Replication timing microarray data sets were used (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Raghuraman <em>et al.</em> 2001</a>). The genome was divided into 5-min windows based on the overall timing of replication. Each box is a representation of the Calling Cards signal of a given construct in a specific replication timing window. Calling Cards signal at each position was normalized to the number of aligned reads in the library (reads per million, RPM). The <em>X</em>-axis and vertical lines represent Ty5 transposon integration events at specific positions within the chromosome and the <em>y</em>-axis and height of each line is the normalized signal at each position. (A) RPM signal comparing each fusion construct to negative control Ty5 integrations. (B) RPM signal comparing Cdc7 fusion constructs to Gal4 fusion construct. (C) RPM signal comparing <em>cdc7KD-sir4</em> fusion construct to <em>cdc7KDΔCT-sir4</em> fusion construct. Calling Cards was normalized to mean number of unique insertions events. (D) Mean number of unique insertions comparing each fusion construct to negative control Ty5 integrations. (E) Mean number of unique integrations comparing Cdc7 fusion constructs to Gal4 fusion construct. (F) Mean number of unique integrations comparing <em>cdc7KD-sir4</em> fusion construct to <em>cdc7KDΔCT-sir4</em> fusion construct. Statistical analysis was done using the nonparametric Wilcoxon test as we were not convinced our data followed a normal distribution. *<em>P</em> = 0.01–0.05; ** <em>P</em> = 0.001–0.01; *** <em>P</em> &lt; 0.001; NS, not significant.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">When Calling Cards data were normalized to the mean number of unique insertion events, providing only positional information (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, D–F</a>), similar trends were seen in comparison to replication timing. <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> created unique Ty5 integrations into early replicating regions of the genome and <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> integrated Ty5 primarily into mid-S regions, while the negative control integrations were low throughout all timing windows. However, with this normalization, <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> produced a higher mean number of unique insertions than <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>, a reversal of the RPM signal normalization. This suggests <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> made more unique insertions throughout each timing window, but <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> produced an overall higher average signal at specific locations.</p></div><div id="sec-12" class="subsection"><h3>Comparison of Cdc7 to ACS sites</h3><p id="p-34">As it is difficult and time consuming to use the UCSC genome browser to assess patterns at all origins of replication, meta-analysis was performed to analyze patterns of Ty5 integration near all confirmed, likely, or dubious ARSs. These ARS classifications are determined by ACS sites in the genome and whether they are known to initiate replication. Confirmed or likely sites are ACS sites that are known to initiate replication while dubious sites are ACS sites that are not explicitly known to initiate replication. The list of these sites was taken from the <em>S. cerevisiae</em> origin database (<a href="http://cerevisiae.oridb.org/data_output.php?main=sc_ori&amp;table=sc_ori&amp;ext_format=BED;FASTA;&amp;format=tab">http://cerevisiae.oridb.org/data_output.php?main=sc_ori&amp;table=sc_ori&amp;ext_format=BED;FASTA;&amp;format=tab</a>). For this analysis, we took the subset of integration events that occurred within the first 30 min according to the timing data and took a window containing 1-kb genomic DNA on either side of confirmed/likely ACS sites or dubious ACS sites to determine how <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>-chromatin binding correlates with ARSs. This 2-kb window around ACS sites was divided into 100-bp windows and the mean RPM or mean number of unique insertion events in each window was determined for the constructs.</p><p id="p-35">When Calling Cards data were normalized to the number of aligned reads (RPM) and comparing to confirmed/likely ACSs, <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> fusion construct produced a sharp increase in the Ty5 transposon integration signal within 100 bp of the center of the ACSs relative to the surrounding genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). This suggests <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binds preferentially near ACS sites known to initiate replication compared to surrounding genomic DNA. While <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> does bind preferentially near confirmed/likely ACS sites, there is still Ty5 integration signal when we move away from the ACS sites, evidence that <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is bound and needed at other places in the genome beyond just origins. <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> reduced the Ty5 integration signal at ACS sites by threefold suggesting a possible role for <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> in stabilizing <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> on chromatin or at least at specific sites. At dubious ACS sites, <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> produced a peak in Ty5 transposon integration signal that was shifted to 300 bp away from the center of the proposed ACS site compared to confirmed/likely ACS sites. As <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is still bound near dubious ACS sites, it is possible <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> still recognizes the sequence specificity of an ACS site as positions where it should bind even if that site doesn’t become a functional origin (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Similar to our <a href="http://www.yeastgenome.org/locus/S000028485/overview">ARS308</a> which binds MCM complex and not ORC, these dubious sites may still bind MCM to promote <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> association even when ORC is not bound. This sharp increase of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding specifically near ACS sites is consistent with the idea that <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is bound to specific sequences that will become origins of replication to promote replication initiation.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ty5 insertions integrate preferentially near ACS sites. Ty5 transposon insertions are compared to known ACS sites within the genome. A 1-kb window on either side of ACS sites was used to determine where Cdc7-directed integrations occur in relation to the ACS. (A) Calling Cards insertions signal compared to confirmed/likely ACS sites normalized to the number of aligned reads in the library (reads per million, RPM). The x-axis is the position relative to ACS sites and y-axis is the normalized signal. (B) Calling Cards insertions signal compared to dubious ACS sites normalized to the number of aligned reads in the library (RPM). (C) Calling Cards insertions compared to confirmed/likely ACS sites normalized to the mean number of unique insertions where the x-axis is the position relative to ACS sites and y-axis is the mean number of insertions for a given genomic position. (D) Calling Cards insertions compared to dubious ACS sites normalized to mean number of unique insertions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ty5 insertions integrate preferentially near ACS sites. Ty5 transposon insertions are compared to known ACS sites within the genome. A 1-kb window on either side of ACS sites was used to determine where Cdc7-directed integrations occur in relation to the ACS. (A) Calling Cards insertions signal compared to confirmed/likely ACS sites normalized to the number of aligned reads in the library (reads per million, RPM). The &lt;em&gt;x&lt;/em&gt;-axis is the position relative to ACS sites and &lt;em&gt;y&lt;/em&gt;-axis is the normalized signal. (B) Calling Cards insertions signal compared to dubious ACS sites normalized to the number of aligned reads in the library (RPM). (C) Calling Cards insertions compared to confirmed/likely ACS sites normalized to the mean number of unique insertions where the &lt;em&gt;x&lt;/em&gt;-axis is the position relative to ACS sites and &lt;em&gt;y&lt;/em&gt;-axis is the mean number of insertions for a given genomic position. (D) Calling Cards insertions compared to dubious ACS sites normalized to mean number of unique insertions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-6.medium.gif" width="305" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-6.medium.gif" width="305" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487227" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ty5 insertions integrate preferentially near ACS sites. Ty5 transposon insertions are compared to known ACS sites within the genome. A 1-kb window on either side of ACS sites was used to determine where Cdc7-directed integrations occur in relation to the ACS. (A) Calling Cards insertions signal compared to confirmed/likely ACS sites normalized to the number of aligned reads in the library (reads per million, RPM). The x-axis is the position relative to ACS sites and y-axis is the normalized signal. (B) Calling Cards insertions signal compared to dubious ACS sites normalized to the number of aligned reads in the library (RPM). (C) Calling Cards insertions compared to confirmed/likely ACS sites normalized to the mean number of unique insertions where the x-axis is the position relative to ACS sites and y-axis is the mean number of insertions for a given genomic position. (D) Calling Cards insertions compared to dubious ACS sites normalized to mean number of unique insertions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ty5 insertions integrate preferentially near ACS sites. Ty5 transposon insertions are compared to known ACS sites within the genome. A 1-kb window on either side of ACS sites was used to determine where Cdc7-directed integrations occur in relation to the ACS. (A) Calling Cards insertions signal compared to confirmed/likely ACS sites normalized to the number of aligned reads in the library (reads per million, RPM). The &lt;em&gt;x&lt;/em&gt;-axis is the position relative to ACS sites and &lt;em&gt;y&lt;/em&gt;-axis is the normalized signal. (B) Calling Cards insertions signal compared to dubious ACS sites normalized to the number of aligned reads in the library (RPM). (C) Calling Cards insertions compared to confirmed/likely ACS sites normalized to the mean number of unique insertions where the &lt;em&gt;x&lt;/em&gt;-axis is the position relative to ACS sites and &lt;em&gt;y&lt;/em&gt;-axis is the mean number of insertions for a given genomic position. (D) Calling Cards insertions compared to dubious ACS sites normalized to mean number of unique insertions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-7.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-7.medium.gif" width="306" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F4/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487227" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Ty5 insertions integrate preferentially near ACS sites. Ty5 transposon insertions are compared to known ACS sites within the genome. A 1-kb window on either side of ACS sites was used to determine where Cdc7-directed integrations occur in relation to the ACS. (A) Calling Cards insertions signal compared to confirmed/likely ACS sites normalized to the number of aligned reads in the library (reads per million, RPM). The <em>x</em>-axis is the position relative to ACS sites and <em>y</em>-axis is the normalized signal. (B) Calling Cards insertions signal compared to dubious ACS sites normalized to the number of aligned reads in the library (RPM). (C) Calling Cards insertions compared to confirmed/likely ACS sites normalized to the mean number of unique insertions where the <em>x</em>-axis is the position relative to ACS sites and <em>y</em>-axis is the mean number of insertions for a given genomic position. (D) Calling Cards insertions compared to dubious ACS sites normalized to mean number of unique insertions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">When Calling Cards data were normalized to the mean number of unique insertion events, <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> again produced a peak in the number of unique integrations near confirmed/likely ACS sites (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). As with replication timing, <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> made more unique integration events near confirmed/likely ACS sites than <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>. It is not known why the unique insertion events near dubious ACS sites is much more variable than that at confirmed/likely ACS sites, such that the mean number of integration events decreased immediately at the ACS site and contained no large peak within 300 bp as seen previously (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>). Comparing <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>-chromatin binding to known ACS sites as a marker for origins of replication indicates <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> has a preference for binding sequences that could become origins.</p></div><div id="sec-13" class="subsection"><h3>Comparison of Cdc7 to Orc1 binding sites</h3><p id="p-38">Calling Cards data were also compared to reported genomic binding sites of the replication protein <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aparicio <em>et al.</em> 1997</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Xu <em>et al.</em> 2006</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Ramey and Sclafani 2014</a>). Again, integrations that occurred within the first 30 min were used and analyzed in relation to 5-kb genomic DNA on either side of known <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> binding sites to determine how close to specific replication protein sites <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binds. This 10-kb genomic DNA around <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> sites was again broken into 100-bp windows where mean RPM and mean number of insertions was calculated.</p><p id="p-39"><a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a> has no specific role in DNA replication, and, as expected, <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> mean RPM signal shows little variability in intensity across each 100-bp window in the entire 10-kb genomic segments. Additionally, the Ty5 integration signal due to <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> was not increased substantially compared to the negative control containing only the Ty5 donor plasmid. <a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a> has no preference to bind near <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> binding sites. When normalized to the number of aligned reads (RPM), <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> and <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> had little variability in Ty5 transposon integration signal intensity across the genomic segment but rather appear to direct Ty5 transposon integrations across the entire 10-kb window to the same levels like <a href="http://www.yeastgenome.org/locus/S000006169/overview">Gal4</a> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). The signal intensity of <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> integrations was increased more than twofold compared to the negative control and <em><a href="http://www.yeastgenome.org/locus/S000006169/overview">GAL4</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>. <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> Ty5 transposon integration signal was reduced in the region around <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> by an order of magnitude compared to <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>, suggesting <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> binding affects <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding to chromatin. While <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> constructs increased the Ty5 transposon integration signal, the integrations still did not show a preference for inserting specifically near <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> binding sites as may be expected since <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> is bound to all functional origins.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ty5 transposons do not correlate with Orc1 binding sites. Ty5 transposon insertions compared to Orc1 binding sites. A 5-kb window on either side of all Orc1 binding sites was used to determine where integrations occur in relation to protein markers of replication. (A) Calling Cards insertions compared to Orc1 binding sites normalized to the number of aligned reads in the library (reads per million, RPM). The x-axis is the position relative to Orc1 binding sites and the y-axis is the normalized signal. (B) Calling Cards insertions compared to Orc1 binding sites normalized to mean number of unique insertions at a given position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ty5 transposons do not correlate with Orc1 binding sites. Ty5 transposon insertions compared to Orc1 binding sites. A 5-kb window on either side of all Orc1 binding sites was used to determine where integrations occur in relation to protein markers of replication. (A) Calling Cards insertions compared to Orc1 binding sites normalized to the number of aligned reads in the library (reads per million, RPM). The &lt;em&gt;x&lt;/em&gt;-axis is the position relative to Orc1 binding sites and the &lt;em&gt;y&lt;/em&gt;-axis is the normalized signal. (B) Calling Cards insertions compared to Orc1 binding sites normalized to mean number of unique insertions at a given position.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F5.medium.gif" width="301" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/11/3757/F5.medium.gif" width="301" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487253" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Ty5 transposons do not correlate with Orc1 binding sites. Ty5 transposon insertions compared to Orc1 binding sites. A 5-kb window on either side of all Orc1 binding sites was used to determine where integrations occur in relation to protein markers of replication. (A) Calling Cards insertions compared to Orc1 binding sites normalized to the number of aligned reads in the library (reads per million, RPM). The <em>x</em>-axis is the position relative to Orc1 binding sites and the <em>y</em>-axis is the normalized signal. (B) Calling Cards insertions compared to Orc1 binding sites normalized to mean number of unique insertions at a given position.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">When normalized to the mean number of insertion events, all constructs had little variability in Ty5 transposon integration signal near <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> binding sites (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). As with the other meta-analysis, <em>cdc7KDΔCT-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em> again showed a higher number of unique insertions around <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> binding sites relative to <em>cdc7KD-<a href="http://www.yeastgenome.org/locus/S000002635/overview">sir4</a></em>.</p><p id="p-42"><a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a>-directed insertions do not bind preferentially to <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> binding sites, indicating there are likely more binding sites for <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> than only at the ACS sites. As a protein, it is possible there are more possibilities for where <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> can bind and thus <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> insertions may be distributed more evenly in relation to these sites. In addition to specific DNA replication features, Calling Cards data were compared to other chromosomal features including transcription start sites (TSS) and centromeres (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a>). <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding does not correlate with TSS but does bind in pericentric genomic DNA, suggesting it is correlated with centromeric DNA to carry our chromatid cohesion.</p></div><div id="sec-14" class="subsection"><h3>Cdc7 is bound to early origins as cells are released from S phase</h3><p id="p-43">ChIP using endogenous <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein has been technically ineffective as reliable results have not been obtained, even when <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> binding was detected (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dowell <em>et al.</em> 1994</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Katou <em>et al.</em> 2006</a>). To further confirm whether <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> was bound to specific origins of replication, ChIP was used in yeast strains that contain overexpressed <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> wild-type and cdc7KD [Cdc7KD(N168A)] mutant proteins. Plasmids containing HA-tagged wild-type <em><a href="http://www.yeastgenome.org/locus/S000002175/overview">CDC7</a></em> or <em>cdc7KD</em> mutant were transformed into strain RSY1294 (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>) that maintains endogenous <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein. Flow cytometry confirmed overexpression of either <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> or cdc7KD does not affect progression through S phase of the cell cycle as seen previously with the <a href="http://www.yeastgenome.org/locus/S000002175/overview">cdc7</a>(N168A) mutant (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hardy 1997</a>). Cells containing either overexpression protein began to release from G<sub>1</sub>/S arrest between 30 and 45 min after addition of pronase and completed S phase at ∼90 min (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromatin immunoprecipitation (ChIP) analysis of Cdc7 at origins of replication. ChIP in strains overexpressing wild-type or kinase dead Cdc7 and strains lacking overexpression was performed on cells released from G1 arrest through S phase using an anti-HA antibody or no antibody negative control. Chromatin associated DNA was analyzed using quantitative PCR for specific DNA segments. (A) Flow cytometry analysis of wild-type and kinase dead Cdc7 overexpression plasmids. (B) ChIP analysis of wild-type Cdc7 using primers specific to origins of replication in a strain with overexpression or no overexpression. (C) Similar ChIP analysis of wild-type Cdc7 using primers specific to genomic regions 10 kb distant from ARS1 and 8 kb distant from ARS305. (D) ChIP analysis of kinase dead Cdc7 using primers specific to origins of replication in strains with or without overexpression. (E) ChIP analysis of kinase dead Cdc7 using primers specific to genomic regions distant from origins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromatin immunoprecipitation (ChIP) analysis of Cdc7 at origins of replication. ChIP in strains overexpressing wild-type or kinase dead Cdc7 and strains lacking overexpression was performed on cells released from G1 arrest through S phase using an anti-HA antibody or no antibody negative control. Chromatin associated DNA was analyzed using quantitative PCR for specific DNA segments. (A) Flow cytometry analysis of wild-type and kinase dead Cdc7 overexpression plasmids. (B) ChIP analysis of wild-type Cdc7 using primers specific to origins of replication in a strain with overexpression or no overexpression. (C) Similar ChIP analysis of wild-type Cdc7 using primers specific to genomic regions 10 kb distant from ARS1 and 8 kb distant from ARS305. (D) ChIP analysis of kinase dead Cdc7 using primers specific to origins of replication in strains with or without overexpression. (E) ChIP analysis of kinase dead Cdc7 using primers specific to genomic regions distant from origins.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-9.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-9.medium.gif" width="440" height="428"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487262" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromatin immunoprecipitation (ChIP) analysis of Cdc7 at origins of replication. ChIP in strains overexpressing wild-type or kinase dead Cdc7 and strains lacking overexpression was performed on cells released from G1 arrest through S phase using an anti-HA antibody or no antibody negative control. Chromatin associated DNA was analyzed using quantitative PCR for specific DNA segments. (A) Flow cytometry analysis of wild-type and kinase dead Cdc7 overexpression plasmids. (B) ChIP analysis of wild-type Cdc7 using primers specific to origins of replication in a strain with overexpression or no overexpression. (C) Similar ChIP analysis of wild-type Cdc7 using primers specific to genomic regions 10 kb distant from ARS1 and 8 kb distant from ARS305. (D) ChIP analysis of kinase dead Cdc7 using primers specific to origins of replication in strains with or without overexpression. (E) ChIP analysis of kinase dead Cdc7 using primers specific to genomic regions distant from origins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromatin immunoprecipitation (ChIP) analysis of Cdc7 at origins of replication. ChIP in strains overexpressing wild-type or kinase dead Cdc7 and strains lacking overexpression was performed on cells released from G1 arrest through S phase using an anti-HA antibody or no antibody negative control. Chromatin associated DNA was analyzed using quantitative PCR for specific DNA segments. (A) Flow cytometry analysis of wild-type and kinase dead Cdc7 overexpression plasmids. (B) ChIP analysis of wild-type Cdc7 using primers specific to origins of replication in a strain with overexpression or no overexpression. (C) Similar ChIP analysis of wild-type Cdc7 using primers specific to genomic regions 10 kb distant from ARS1 and 8 kb distant from ARS305. (D) ChIP analysis of kinase dead Cdc7 using primers specific to origins of replication in strains with or without overexpression. (E) ChIP analysis of kinase dead Cdc7 using primers specific to genomic regions distant from origins.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-10.medium.gif" width="356" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-10.medium.gif" width="356" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487262" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromatin immunoprecipitation (ChIP) analysis of Cdc7 at origins of replication. ChIP in strains overexpressing wild-type or kinase dead Cdc7 and strains lacking overexpression was performed on cells released from G1 arrest through S phase using an anti-HA antibody or no antibody negative control. Chromatin associated DNA was analyzed using quantitative PCR for specific DNA segments. (A) Flow cytometry analysis of wild-type and kinase dead Cdc7 overexpression plasmids. (B) ChIP analysis of wild-type Cdc7 using primers specific to origins of replication in a strain with overexpression or no overexpression. (C) Similar ChIP analysis of wild-type Cdc7 using primers specific to genomic regions 10 kb distant from ARS1 and 8 kb distant from ARS305. (D) ChIP analysis of kinase dead Cdc7 using primers specific to origins of replication in strains with or without overexpression. (E) ChIP analysis of kinase dead Cdc7 using primers specific to genomic regions distant from origins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromatin immunoprecipitation (ChIP) analysis of Cdc7 at origins of replication. ChIP in strains overexpressing wild-type or kinase dead Cdc7 and strains lacking overexpression was performed on cells released from G1 arrest through S phase using an anti-HA antibody or no antibody negative control. Chromatin associated DNA was analyzed using quantitative PCR for specific DNA segments. (A) Flow cytometry analysis of wild-type and kinase dead Cdc7 overexpression plasmids. (B) ChIP analysis of wild-type Cdc7 using primers specific to origins of replication in a strain with overexpression or no overexpression. (C) Similar ChIP analysis of wild-type Cdc7 using primers specific to genomic regions 10 kb distant from ARS1 and 8 kb distant from ARS305. (D) ChIP analysis of kinase dead Cdc7 using primers specific to origins of replication in strains with or without overexpression. (E) ChIP analysis of kinase dead Cdc7 using primers specific to genomic regions distant from origins.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-11.medium.gif" width="357" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-11.medium.gif" width="357" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F6/graphic-11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487262" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Chromatin immunoprecipitation (ChIP) analysis of Cdc7 at origins of replication. ChIP in strains overexpressing wild-type or kinase dead Cdc7 and strains lacking overexpression was performed on cells released from G1 arrest through S phase using an anti-HA antibody or no antibody negative control. Chromatin associated DNA was analyzed using quantitative PCR for specific DNA segments. (A) Flow cytometry analysis of wild-type and kinase dead Cdc7 overexpression plasmids. (B) ChIP analysis of wild-type Cdc7 using primers specific to origins of replication in a strain with overexpression or no overexpression. (C) Similar ChIP analysis of wild-type Cdc7 using primers specific to genomic regions 10 kb distant from ARS1 and 8 kb distant from ARS305. (D) ChIP analysis of kinase dead Cdc7 using primers specific to origins of replication in strains with or without overexpression. (E) ChIP analysis of kinase dead Cdc7 using primers specific to genomic regions distant from origins.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Early origins were used to determine association of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein with specific origins at various points during S phase. Both <a href="http://www.yeastgenome.org/locus/S000028483/overview">ARS306</a> and <a href="http://www.yeastgenome.org/locus/S000007639/overview">ARS607</a> have timing of replication between 10 and 13 min (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Raghuraman <em>et al.</em> 2001</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Yabuki <em>et al.</em> 2002</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alvino <em>et al.</em> 2007</a>). Wild-type <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein at these origins increased at 30 min after release, coinciding with the release into S phase from the G<sub>1</sub>/S arrest (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein signal at both origins peaked at ∼45 min parallel to the timing of replication of 10–13 min. Previous chromatin fractionation experiments showed an increase in <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein bound to bulk chromatin as cells enter into S phase (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Weinreich and Stillman 1999</a>). We used a negative control without an HA-tagged protein to show that increases during specific time points were specific to HA-<a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein. The negative control did not show signal increases at origins during S phase.</p><p id="p-46">Similar to wild-type <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein, Cdc7KD(N168A) protein signal increased at 30 min after release from S phase. However, Cdc7KD(N168A) signal peaked at ∼60 min, 15–30 min later than what was observed for wild type potentially due to Cdc7KD(N168A) binding longer. This kinase dead <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein is still able to bind but cannot phosphorylate the MCM complexes. Both <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> and Cdc7KD(N168A) signal decreased briefly ∼75 min before increasing again at 90 min (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6D</a>).</p><p id="p-47">Primers corresponding to DNA 10 kb away from <a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a> and 8 kb away from <a href="http://www.yeastgenome.org/locus/S000028482/overview">ARS305</a> were used to determine <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding to regions of the genome not containing origins (<a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>). While wild-type <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> could not be significantly detected at either of these nonorigin regions, Cdc7KD(N168A) was found at these regions specifically between 60 and 75 min (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6, C and E</a>), again likely reflecting Cdc7KD(N168A) interacting with the substrate longer and not exhibiting transient binding similar to wild-type <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-48">Using modified Calling Cards and ChIP methods in <em>S. cerevisiae</em>, we have begun to map <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein kinase binding to chromatin during DNA replication. Identifying <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding patterns and potential specificity within the genome furthers our understanding of the underlying molecular mechanisms of DNA replication initiation. It has been established that <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> activity is required throughout S phase to initiate replication from both early and late firing origins. Our Calling Cards results indicate the greatest amount of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> kinase is bound to early replicating regions of the genome when the majority of origins are used to initiate replication. As cells progress through S phase, fewer origins are activated and less <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> kinase is needed and bound in regions of late replicating DNA.</p><p id="p-49">Calling Cards analysis showed wild-type <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> or other replication proteins inefficiently integrate Ty5 transposons into the genome, likely due to the fact that <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> phosphorylation is a transient process as expected for an enzyme and the MCM helicases or other substrates are not bound long enough to allow for efficient Ty5 transposon integration. Kinase dead <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein binds the MCM helicases long enough to allow a more efficient integration of the Ty5 transposon. Each of our kinase dead <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> constructs binds to chromatin throughout the entire yeast genome as well as specifically at origins of replication. Furthermore, our results suggest that physical interaction between <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> and the <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> regulatory subunit protein may be required for <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding specificity in the genome. When <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> kinase dead protein lacking 55 C-terminal amino acids essential for binding <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> is used, more unique insertions are made, but the overall Ty5 transposon integration signal is reduced. This suggests <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> that cannot bind <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> has a wider distribution on chromatin, and direct interaction between the two subunits confers more specificity as to where <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is binding especially at origins of replication.</p><p id="p-50">Through Calling Cards meta-analysis, we determined <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> does bind preferentially near known ACS sites, whether they are confirmed or still in doubt. Additionally, <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding does not correlate with <a href="http://www.yeastgenome.org/locus/S000004530/overview">Orc1</a> binding sites indicating <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> recognizes specific sequences in the genome in order to bind rather than recognizing bound protein complexes or substrates. Using ChIP, we found <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding to specific early origins of replication. This association with early origins of replication changes throughout S phase such that <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is bound highest as cells release into S phase but decreases as cells complete S phase. The discrepancy between low wild-type <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding in Calling Cards and the more substantial binding seen in ChIP may be attributed to the formaldehyde fixation step in the ChIP protocol. In ChIP, we fix the proteins to chromatin and stabilize the interaction, while Calling Cards maintain the transient nature of the kinase interaction.</p><p id="p-51">Both Calling Cards and ChIP show <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> overexpression is critical to effectively look at the relationship between <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> and origins. Not only is <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> bound to ACS sites and origins of replication, but <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is also bound to nonorigin DNA. We predict <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> kinase activity is needed in other genomic DNA regions and not just origins to carry out functions in TLS and meiotic recombination in addition to replication and chromatid cohesion. Sequence specificity <em>vs.</em> substrate specificity is further characterized by correlation of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> binding with pericentric DNA. Centromeres are known to replicate early and our results provide evidence that <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is bound both in these early replicating regions as well as in pericentric DNA. We predict <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is bound in the centromeric flanking DNA to aid in chromatin cohesion.</p><p id="p-52">This study is consistent with the idea <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is bound specifically to origins to initiate DNA replication, but it also provides supports for a model in which <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> can be bound throughout the genome even in G<sub>1</sub> cells. Our model proposes that as cells enters the G<sub>1</sub>/S transition, <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein expression increases, stabilizes <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> on the chromatin, and allows preferential binding to origins. Precise localization of active <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> at origins is important for DNA replication, but it is equally important to increase the total amount of <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> bound to chromatin to initiate cellular processes beyond replication. This <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> mass action increases the potential for phosphorylation of nearby MCM complexes or substrates for TLS, meiotic recombination, and chromatid cohesion (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wan <em>et al.</em> 2008</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Brandao <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Princz <em>et al.</em> 2017</a>). Our model is consistent with previous bulk chromatin studies in which <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> was bound to chromatin throughout the cell cycle and increased after <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein appeared in S phase (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Weinreich and Stillman 1999</a>). Recent ChIP experiments in G<sub>1</sub>-arrested cells used tagged <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> proteins to show <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> is bound weakly to origins (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Katou <em>et al.</em> 2006</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Natsume <em>et al.</em> 2013</a>). However, levels of <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a> protein are very low in G<sub>1</sub> (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 1999</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Oshiro <em>et al.</em> 1999</a>; <a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Weinreich and Stillman 1999</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Ferreira <em>et al.</em> 2000</a>). In the previous studies, <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> was not used, making the comparison difficult until ChIP could be modified. In our studies, Calling Cards provided evidence for preferential binding to ACS sites, and our modified ChIP experiments using overexpressed-tagged <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> support this in which <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> is closely associated with specific origins. Furthermore, our experiments are not restricted to G<sub>1</sub>-arrested cells and our analysis demonstrates how the <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> interaction with origins changes at various time points in S phase. Together, our findings present a mechanism in which <a href="http://www.yeastgenome.org/locus/S000002175/overview">Cdc7</a> protein kinase binds throughout the genome using sequence-specific identification to carry out its many functions during the cell cycle and to promote DNA replication</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model of Cdc7 action during DNA replication. Cdc7 protein is bound to chromatin throughout the genome during G1 phase of the cell cycle in the absence of Dbf4 protein. As Dbf4 protein expression increases, Cdc7 protein is stabilized on chromatin throughout the genome and is preferentially bound to known ACS sites. Increases in binding increase the potential to phosphorylate substrates including minichromosome maintenance (MCM) helicase complexes. Phosphorylation of the MCM complexes by Cdc7-Dbf4 kinase (DDK) results in loading of Cdc45 and GINS to form the active MCM helicase and to load on the replisome, which replicates the DNA. DDK not bound preferentially to ACS sites is used to promote TLS, meiotic recombination, and chromatid cohesion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1421708399" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Model of Cdc7 action during DNA replication. Cdc7 protein is bound to chromatin throughout the genome during G1 phase of the cell cycle in the absence of Dbf4 protein. As Dbf4 protein expression increases, Cdc7 protein is stabilized on chromatin throughout the genome and is preferentially bound to known ACS sites. Increases in binding increase the potential to phosphorylate substrates including minichromosome maintenance (MCM) helicase complexes. Phosphorylation of the MCM complexes by Cdc7-Dbf4 kinase (DDK) results in loading of Cdc45 and GINS to form the active MCM helicase and to load on the replisome, which replicates the DNA. DDK not bound preferentially to ACS sites is used to promote TLS, meiotic recombination, and chromatid cohesion.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3757/F7.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/11/3757/F7.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3757/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487168" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-53" class="first-child">Model of Cdc7 action during DNA replication. Cdc7 protein is bound to chromatin throughout the genome during G1 phase of the cell cycle in the absence of Dbf4 protein. As Dbf4 protein expression increases, Cdc7 protein is stabilized on chromatin throughout the genome and is preferentially bound to known ACS sites. Increases in binding increase the potential to phosphorylate substrates including minichromosome maintenance (MCM) helicase complexes. Phosphorylation of the MCM complexes by Cdc7-Dbf4 kinase (DDK) results in loading of Cdc45 and GINS to form the active MCM helicase and to load on the replisome, which replicates the DNA. DDK not bound preferentially to ACS sites is used to promote TLS, meiotic recombination, and chromatid cohesion.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">We thank Marianne Tecklenburg, who provided excellent technical assistance, and Haoyi Wang and Rob Mitra (Washington University in St. Louis), Chris Hardy (Vanderbilt University), and Mark Johnston (University of Colorado) for strains and plasmids and technical information. This work was supported by a Public Health Service award to R.S. (R01 GM35078) and a Research Scholar grant (RSG-13-216-01-DMC) from the American Cancer Society to J.R.H. Shared Resource core facilities used were supported by the University of Colorado Cancer Center grant (P30CA046934).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> These authors are co-first authors.</p></li><li class="fn" id="fn-2"><p id="p-55">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023366/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023366/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-56"><em>Communicating editor: B. Cohen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-06"><span class="received-label">Received </span>April 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-09-09"><span class="accepted-label">Accepted </span>September 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Rossbach <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.1" data-doi="10.1128/MCB.00719-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvino</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delrow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Replication in hydroxyurea: it’s a matter of time.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">6396</span>–<span class="cit-lpage">6406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00719-07%26rft_id%253Dinfo%253Apmid%252F17636020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xOC82Mzk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.2" data-doi="10.1016/S0092-8674(01)80009-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstein</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Components and dynamics of DNA replication complexes in S. cerevisiae: redistribution of MCM proteins and Cdc45p during the S phase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAparicio%26rft.auinit1%253DO.%2BM.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D69%26rft.atitle%253DComponents%2Band%2Bdynamics%2Bof%2BDNA%2Breplication%2Bcomplexes%2Bin%2BS.%2Bcerevisiae%253A%2Bredistribution%2Bof%2BMCM%2Bproteins%2Band%2BCdc45p%2Bduring%2BS%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252980009-X%26rft_id%253Dinfo%253Apmid%252F9335335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)80009-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9335335&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XZ80900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.3" data-doi="10.1146/annurev.biochem.71.110601.135425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutta</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">DNA replication in eukaryotic cells.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiochem.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.71.110601.135425%26rft_id%253Dinfo%253Apmid%252F12045100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.71.110601.135425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12045100&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177352600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.4" data-doi="10.1534/genetics.114.165308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandao</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santoro</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The role of Dbf4-dependent protein kinase in DNA polymerase ζ-dependent mutagenesis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165308%26rft_id%253Dinfo%253Apmid%252F24875188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzExMTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.5" data-doi="10.1128/MCB.19.6.4270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collyer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Cell cycle regulation of DNA replication initiator factor Dbf4p.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4270</span>–<span class="cit-lpage">4278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DCheng%26rft.auinit1%253DL.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D4270%26rft.epage%253D4278%26rft.atitle%253DCell%2BCycle%2BRegulation%2Bof%2BDNA%2BReplication%2BInitiator%2BFactor%2BDbf4p%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.6.4270%26rft_id%253Dinfo%253Apmid%252F10330168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvNDI3MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.6" data-doi="10.1126/science.8066465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dowell</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romanowski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F. X.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Interaction of Dbf4, the Cdc7 protein kinase regulatory subunit, with yeast regulatory origins in vivo.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDowell%26rft.auinit1%253DS.%26rft.volume%253D265%26rft.issue%253D5176%26rft.spage%253D1243%26rft.epage%253D1246%26rft.atitle%253DInteraction%2Bof%2BDbf4%252C%2Bthe%2BCdc7%2Bprotein%2Bkinase%2Bregulatory%2Bsubunit%252C%2Bwith%2Byeast%2Breplication%2Borigins%2Bin%2Bvivo%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8066465%26rft_id%253Dinfo%253Apmid%252F8066465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjUvNTE3Ni8xMjQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.7" data-doi="10.1128/MCB.20.1.242-248.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santocanale</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drury</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Dbf4p, an essential S phase-promoting factor, is targeted for degradation by the anaphase-promoting complex.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGodinho%2BFerreira%26rft.auinit1%253DM.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D242%26rft.epage%253D248%26rft.atitle%253DDbf4p%252C%2Ban%2BEssential%2BS%2BPhase-Promoting%2BFactor%252C%2BIs%2BTargeted%2Bfor%2BDegradation%2Bby%2Bthe%2BAnaphase-Promoting%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.1.242-248.2000%26rft_id%253Dinfo%253Apmid%252F10594027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjIwLzEvMjQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.8" data-doi="10.1128/MMBR.68.1.109-131.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsburg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Eukaryotic MCM proteins: beyond replication initiation.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DForsburg%26rft.auinit1%253DS.%2BL.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D131%26rft.atitle%253DEukaryotic%2BMCM%2BProteins%253A%2BBeyond%2BReplication%2BInitiation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.68.1.109-131.2004%26rft_id%253Dinfo%253Apmid%252F15007098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2OC8xLzEwOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.9" data-doi="10.1038/nature11920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frigola</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehanna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ATPase-dependent quality control of DNA replication origin licensing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrigola%26rft.volume%253D495%26rft.spage%253D339%26rft.atitle%253DATPase-dependent%2Bquality%2Bcontrol%2Bof%2BDNA%2Breplication%2Borigin%2Blicensing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11920%26rft_id%253Dinfo%253Apmid%252F23474987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23474987&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.10" data-doi="10.1016/S0378-1119(96)00761-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Identification of Cdc45p, an essential factor required for DNA replication.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DHardy%26rft.auinit1%253DC.%2BF.%26rft.volume%253D187%26rft.issue%253D2%26rft.spage%253D239%26rft.epage%253D246%26rft.atitle%253DIdentification%2Bof%2BCdc45p%252C%2Ban%2Bessential%2Bfactor%2Brequired%2Bfor%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252896%252900761-5%26rft_id%253Dinfo%253Apmid%252F9099887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(96)00761-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9099887&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WR55700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-3" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.11" data-doi="10.1128/MCB.16.12.6775"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pautz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A novel role for Cdc5p in DNA replication.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">6775</span>–<span class="cit-lpage">6782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHardy%26rft.auinit1%253DC.%2BF.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D6775%26rft.epage%253D6782%26rft.atitle%253DA%2Bnovel%2Brole%2Bfor%2BCdc5p%2Bin%2BDNA%2Breplication%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.12.6775%26rft_id%253Dinfo%253Apmid%252F8943332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMi82Nzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.12" data-doi="10.1073/pnas.94.7.3151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">C. F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dryga</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seematter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pahl</span>  <span class="cit-name-given-names">P. M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title"><em>mcm5/cdc46-bob1</em> bypasses the requirement for the S phase activator Cdc7p.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3151</span>–<span class="cit-lpage">3155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHardy%26rft.auinit1%253DC.%2BF.%2BJ.%26rft.volume%253D94%26rft.issue%253D7%26rft.spage%253D3151%26rft.epage%253D3155%26rft.atitle%253Dmcm5%252Fcdc46-bob1%2Bbypasses%2Bthe%2Brequirement%2Bfor%2Bthe%2BS%2Bphase%2Bactivator%2BCdc7p%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.7.3151%26rft_id%253Dinfo%253Apmid%252F9096361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NC83LzMxNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.13" data-doi="10.1016/0022-2836(71)90420-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartwell</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Genetic control of the cell division cycle in yeast II. Genes controlling DNA replication and its initiation.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHartwell%26rft.auinit1%253DL.%2BH.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D183%26rft.epage%253D194%26rft.atitle%253DGenetic%2Bcontrol%2Bof%2Bthe%2Bcell%2Bdivision%2Bcycle%2Bin%2Byeast.%2BII.%2BGenes%2Bcontrolling%2BDNA%2Breplication%2Band%2Bits%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252871%252990420-7%26rft_id%253Dinfo%253Apmid%252F5283752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(71)90420-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5283752&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1971J879900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.14" data-doi="10.1016/j.cell.2011.06.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Eukaryotic origin-dependent DNA replication in vitro reveals sequential action of DDK and S-CDK kinases.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHeller%26rft.auinit1%253DR.%2BC.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D80%26rft.epage%253D91%26rft.atitle%253DEukaryotic%2Borigin-dependent%2BDNA%2Breplication%2Bin%2Bvitro%2Breveals%2Bsequential%2Baction%2Bof%2BDDK%2Band%2BS-CDK%2Bkinases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.06.012%26rft_id%253Dinfo%253Apmid%252F21729781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.06.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729781&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292539300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.15" data-doi="10.1016/S0378-1119(98)00094-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hess</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drong</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiland</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sligthom</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A human homolog of the yeast CDC7 gene is overexpressed in some tumors and transformed cell lines.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DHess%26rft.auinit1%253DG.%2BF.%26rft.volume%253D211%26rft.issue%253D1%26rft.spage%253D133%26rft.epage%253D140%26rft.atitle%253DA%2Bhuman%2Bhomolog%2Bof%2Bthe%2Byeast%2BCDC7%2Bgene%2Bis%2Boverexpressed%2Bin%2Bsome%2Btumors%2Band%2Btransformed%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252898%252900094-8%26rft_id%253Dinfo%253Apmid%252F9573348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(98)00094-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9573348&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073572000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.16" data-doi="10.1073/pnas.87.16.6272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">R. E.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">DNA metabolism gene CDC7 from yeast encodes a serine (threonine) protein kinase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">6272</span>–<span class="cit-lpage">6276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHollingsworth%26rft.auinit1%253DR.%2BE.%26rft.volume%253D87%26rft.issue%253D16%26rft.spage%253D6272%26rft.epage%253D6276%26rft.atitle%253DDNA%2Bmetabolism%2Bgene%2BCDC7%2Bfrom%2Byeast%2Bencodes%2Ba%2Bserine%2B%2528threonine%2529%2Bprotein%2Bkinase.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.16.6272%26rft_id%253Dinfo%253Apmid%252F2166954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODcvMTYvNjI3MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostroff</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niswander</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Molecular genetic studies of the Cdc7 protein kinase and induced mutagenesis in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHollingsworth-Jr.%26rft.auinit1%253DR.%2BE.%26rft.volume%253D132%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D62%26rft.atitle%253DMolecular%2BGenetic%2BStudies%2Bof%2Bthe%2BCdc7%2BProtein%2BKinase%2Band%2BInduced%2BMutagenesis%2Bin%2BYeast%26rft_id%253Dinfo%253Apmid%252F1398063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTMyLzEvNTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.18" data-doi="10.4161/cc.9.23.14091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holzen</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic interaction of RAD53 protein kinase with histones is important for DNA replication.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">4735</span>–<span class="cit-lpage">4747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DHolzen%26rft.auinit1%253DT.%2BM.%26rft.volume%253D9%26rft.issue%253D23%26rft.spage%253D4735%26rft.epage%253D4747%26rft.atitle%253DGenetic%2Binteraction%2Bof%2BRAD53%2Bprotein%2Bkinase%2Bwith%2Bhistones%2Bis%2Bimportant%2Bfor%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.9.23.14091%26rft_id%253Dinfo%253Apmid%252F21099362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.9.23.14091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21099362&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.19" data-doi="10.1038/nsmb.2404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elustondo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Fonzo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leroux</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Crystal structure of human CDC7 kinase in complex with its activator DBF4.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DHughes%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D1101%26rft.epage%253D1107%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bhuman%2BCDC7%2Bkinase%2Bin%2Bcomplex%2Bwith%2Bits%2Bactivator%2BDBF4.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2404%26rft_id%253Dinfo%253Apmid%252F23064647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23064647&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.20" data-doi="10.1128/MCB.13.5.2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pahl</span>  <span class="cit-name-given-names">P. M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosamond</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Cell cycle regulation of the yeast Cdc7 protein kinase by association with the Dbf4 protein.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2899</span>–<span class="cit-lpage">2908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DJackson%26rft.auinit1%253DA%2BL%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D2899%26rft.epage%253D2908%26rft.atitle%253DCell%2Bcycle%2Bregulation%2Bof%2Bthe%2Byeast%2BCdc7%2Bprotein%2Bkinase%2Bby%2Bassociation%2Bwith%2Bthe%2BDbf4%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.13.5.2899%26rft_id%253Dinfo%253Apmid%252F8474449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEzLzUvMjg5OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.21" data-doi="10.1016/S0962-8924(99)01586-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">First the CDKs, now the DDKs.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DJohnston%26rft.auinit1%253DL.%2BH.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D249%26rft.epage%253D252%26rft.atitle%253DFirst%2Bthe%2BCDKs%252C%2Bnow%2Bthe%2BDDKs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0962-8924%252899%252901586-X%26rft_id%253Dinfo%253Apmid%252F10370238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0962-8924(99)01586-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10370238&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081758900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karolchik</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The UCSC genome browser.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Hum. Genet.</abbr> <span class="cit-vol">Chapter 18</span>: <span class="cit-fpage">Unit18.6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BHum.%2BGenet.%26rft.volume%253D18%26rft.spage%253DUnit18.6%26rft_id%253Dinfo%253Apmid%252F21975940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21975940&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.23" data-doi="10.1016/S0076-6879(05)09023-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneshiro</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aburatani</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirahige</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic approach for the understanding of dynamic aspect of chromosome behavior.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DKatou%26rft.auinit1%253DY.%26rft.volume%253D409%26rft.spage%253D389%26rft.epage%253D410%26rft.atitle%253DGenomic%2Bapproach%2Bfor%2Bthe%2Bunderstanding%2Bof%2Bdynamic%2Baspect%2Bof%2Bchromosome%2Bbehavior.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252805%252909023-3%26rft_id%253Dinfo%253Apmid%252F16793414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(05)09023-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16793414&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238354800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.24" data-doi="10.1126/science.1862342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knighton</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ten Eyck</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xuong</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKnighton%26rft.auinit1%253DD.%26rft.volume%253D253%26rft.issue%253D5018%26rft.spage%253D407%26rft.epage%253D414%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bcatalytic%2Bsubunit%2Bof%2Bcyclic%2Badenosine%2Bmonophosphate-dependent%2Bprotein%2Bkinase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1862342%26rft_id%253Dinfo%253Apmid%252F1862342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTMvNTAxOC80MDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.25" data-doi="10.1101/gad.1819409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kogut</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Scc2/Scc4 cohesin loader determines the distribution of cohesin on budding yeast chromosomes.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2345</span>–<span class="cit-lpage">2357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKogut%26rft.auinit1%253DI.%26rft.volume%253D23%26rft.issue%253D19%26rft.spage%253D2345%26rft.epage%253D2357%26rft.atitle%253DThe%2BScc2%252FScc4%2Bcohesin%2Bloader%2Bdetermines%2Bthe%2Bdistribution%2Bof%2Bcohesin%2Bon%2Bbudding%2Byeast%2Bchromosomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1819409%26rft_id%253Dinfo%253Apmid%252F19797771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIzLzE5LzIzNDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.26" data-doi="10.1101/gad.1933010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labib</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How do Cdc7 and cyclin-dependent kinases trigger the initiation of chromosome replication in eukaryotic cells?</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1208</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLabib%26rft.auinit1%253DK.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D1208%26rft.epage%253D1219%26rft.atitle%253DHow%2Bdo%2BCdc7%2Band%2Bcyclin-dependent%2Bkinases%2Btrigger%2Bthe%2Binitiation%2Bof%2Bchromosome%2Breplication%2Bin%2Beukaryotic%2Bcells%253F%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1933010%26rft_id%253Dinfo%253Apmid%252F20551170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI0LzEyLzEyMDgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.27" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.28" data-doi="10.1101/gad.11.24.3365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawasaki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kihara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mcm2 is a target of regulation by Cdc7-Dbf4 during the initiation of DNA synthesis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">3365</span>–<span class="cit-lpage">3374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLei%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D24%26rft.spage%253D3365%26rft.epage%253D3374%26rft.atitle%253DMcm2%2Bis%2Ba%2Btarget%2Bof%2Bregulation%2Bby%2BCdc7-Dbf4%2Bduring%2Bthe%2Binitiation%2Bof%2BDNA%25A0synthesis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.24.3365%26rft_id%253Dinfo%253Apmid%252F9407029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzI0LzMzNjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.29" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.30" data-doi="10.1093/nar/gkt878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Retkute</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The dynamics of genome replication using deep sequencing.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">e3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt878%26rft_id%253Dinfo%253Apmid%252F24089142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24089142&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.31" data-doi="10.1016/j.molcel.2013.05.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Retkute</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gierlinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Kinetochores coordinate pericentromeric cohesion and early DNA replication by cdc7-dbf4 kinase recruitment.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D661%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.05.011%26rft_id%253Dinfo%253Apmid%252F23746350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.05.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746350&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320678500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.32" data-doi="10.1128/MCB.20.11.3795-3806.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nougarede</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Della Seta</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarzov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwob</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hierarchy of S-phase-promoting factors: yeast Dbf4-cdc7 kinase requires prior S-phase cyclin-dependent kinase activation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3795</span>–<span class="cit-lpage">3806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNougarede%26rft.auinit1%253DR.%26rft.volume%253D20%26rft.issue%253D11%26rft.spage%253D3795%26rft.epage%253D3806%26rft.atitle%253DHierarchy%2Bof%2BS-Phase-Promoting%2BFactors%253A%2BYeast%2BDbf4-Cdc7%2BKinase%2BRequires%2BPrior%2BS-Phase%2BCyclin-Dependent%2BKinase%2BActivation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.11.3795-3806.2000%26rft_id%253Dinfo%253Apmid%252F10805723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMS8zNzk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.33" data-doi="10.1128/MCB.19.7.4888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oshiro</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shellman</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Cell cycle control of cdc7p kinase activity through regulation of dbf4p stability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4888</span>–<span class="cit-lpage">4896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DOshiro%26rft.auinit1%253DG.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D4888%26rft.epage%253D4896%26rft.atitle%253DCell%2BCycle%2BControl%2Bof%2BCdc7p%2BKinase%2BActivity%2Bthrough%2BRegulation%2Bof%2BDbf4p%2BStability%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.7.4888%26rft_id%253Dinfo%253Apmid%252F10373538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvNDg4OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.34" data-doi="10.1091/mbc.E08-06-0645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kommajosyula</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosebrock</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensimon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leatherwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Hsk1(Cdc7) replication kinase regulates origin efficiency.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">5550</span>–<span class="cit-lpage">5558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-06-0645%26rft_id%253Dinfo%253Apmid%252F18799612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi81NTUwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.35" data-doi="10.1128/MCB.6.5.1590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosamond</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Molecular characterization of cell cycle gene CDC7 from Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1590</span>–<span class="cit-lpage">1598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DPatterson%26rft.auinit1%253DM%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D1590%26rft.epage%253D1598%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bcell%2Bcycle%2Bgene%2BCDC7%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.6.5.1590%26rft_id%253Dinfo%253Apmid%252F3537706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjYvNS8xNTkwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.36" data-doi="10.15252/embj.201694831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Princz</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wild</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bittmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguado</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dbf4-dependent kinase and the Rtt107 scaffold promote Mus81-Mms4 resolvase activation during mitosis.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">664</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembj.201694831%26rft_id%253Dinfo%253Apmid%252F28096179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIzNi81LzY2NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.37" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.38" data-doi="10.1126/science.294.5540.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wodicka</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Replication dynamics of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRaghuraman%26rft.auinit1%253DM.%2BK.%26rft.volume%253D294%26rft.issue%253D5540%26rft.spage%253D115%26rft.epage%253D121%26rft.atitle%253DReplication%2BDynamics%2Bof%2Bthe%2BYeast%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.294.5540.115%26rft_id%253Dinfo%253Apmid%252F11588253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTQvNTU0MC8xMTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramey</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Functional conservation of the pre-sensor one beta-finger hairpin (PS1-hp) structures in mini-chromosome maintenance proteins of <em>Saccharomyces cerevisiae</em> and archaea.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D1319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.40" data-doi="10.1016/j.molcel.2010.10.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Randell</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">L. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mec1 is one of multiple kinases that prime the Mcm2-7 helicase for phosphorylation by Cdc7.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRandell%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D353%26rft.epage%253D363%26rft.atitle%253DMec1%2Bis%2Bone%2Bof%2Bmultiple%2Bkinases%2Bthat%2Bprime%2Bthe%2BMcm2-7%2Bhelicase%2Bfor%2Bphosphorylation%2Bby%2BCdc7.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.10.017%26rft_id%253Dinfo%253Apmid%252F21070963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.10.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21070963&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284556300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3757.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rossbach</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Role of DDK in replication initiation</span>, pp. <span class="cit-fpage">279</span>–<span class="cit-lpage">296</span> in <span class="cit-source">The Initiation of DNA Replication in Eukaryotes</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">D. L.</span></span>. <span class="cit-publ-name">Springer International Publishing</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.42" data-doi="10.1101/gad.1626608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasanuma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirota</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakusho</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kugou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cdc7-dependent phosphorylation of Mer2 facilitates initiation of yeast meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSasanuma%26rft.auinit1%253DH.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D398%26rft.epage%253D410%26rft.atitle%253DCdc7-dependent%2Bphosphorylation%2Bof%2BMer2%2Bfacilitates%2Binitiation%2Bof%2Byeast%2Bmeiotic%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1626608%26rft_id%253Dinfo%253Apmid%252F18245451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjIvMy8zOTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.43" data-doi="10.1093/emboj/16.14.4340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arai</span>  <span class="cit-name-given-names">K.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masai</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Human and <em>Xenopus</em> cDNAs encoding budding yeast Cdc7-related kinases: <em>in vitro</em> phosphorylation of MCM subunits by a putative human homologue of Cdc7.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">4340</span>–<span class="cit-lpage">4351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DSato%26rft.auinit1%253DN.%26rft.volume%253D16%26rft.issue%253D14%26rft.spage%253D4340%26rft.epage%253D4351%26rft.atitle%253DHuman%2Band%2BXenopus%2BcDNAs%2Bencoding%2Bbudding%2Byeast%2BCdc7-related%2Bkinases%253A%2Bin%2Bvitro%2Bphosphorylation%2Bof%2BMCM%2Bsubunits%2Bby%2Ba%2Bputative%2Bhuman%2Bhomologue%2Bof%2BCdc7%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.14.4340%26rft_id%253Dinfo%253Apmid%252F9250678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTYvMTQvNDM0MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.44" data-doi="10.1146/annurev.genet.41.110306.130308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holzen</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cell cycle regulation of DNA replication.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DSclafani%26rft.auinit1%253DR.%2BA.%26rft.volume%253D41%26rft.spage%253D237%26rft.epage%253D280%26rft.atitle%253DCell%2Bcycle%2Bregulation%2Bof%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.41.110306.130308%26rft_id%253Dinfo%253Apmid%252F17630848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.41.110306.130308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17630848&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252359500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.45" data-doi="10.1038/nature08647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheu</span>  <span class="cit-name-given-names">Y.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Dbf4-Cdc7 kinase promotes S phase by alleviating an inhibitory activity in Mcm4.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSheu%26rft.auinit1%253DY.%2BJ.%26rft.volume%253D463%26rft.issue%253D7277%26rft.spage%253D113%26rft.epage%253D117%26rft.atitle%253DThe%2BDbf4-Cdc7%2Bkinase%2Bpromotes%2BS%2Bphase%2Bby%2Balleviating%2Ban%2Binhibitory%2Bactivity%2Bin%2BMcm4.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08647%26rft_id%253Dinfo%253Apmid%252F20054399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20054399&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273344900041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.46" data-doi="10.1101/gad.1626408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shokat</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cdc28-Clb5 (CDK-S) and Cdc7-Dbf4 (DDK) collaborate to initiate meiotic recombination in yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">386</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWan%26rft.auinit1%253DL.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D386%26rft.epage%253D397%26rft.atitle%253DCdc28-Clb5%2B%2528CDK-S%2529%2Band%2BCdc7-Dbf4%2B%2528DDK%2529%2Bcollaborate%2Bto%2Binitiate%2Bmeiotic%2Brecombination%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1626408%26rft_id%253Dinfo%253Apmid%252F18245450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjIvMy8zODYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.47" data-doi="10.1101/gr.6510207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Calling cards for DNA-binding proteins.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1202</span>–<span class="cit-lpage">1209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6510207%26rft_id%253Dinfo%253Apmid%252F17623806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE3LzgvMTIwMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.48" data-doi="10.1038/nprot.2008.148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinz</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">‘Calling Cards’ method for high-throughput identification of targets of yeast DNA-binding proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1569</span>–<span class="cit-lpage">1577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253D1569%26rft.epage%253D1577%26rft.atitle%253D%2527Calling%2BCards%2527%2Bmethod%2Bfor%2Bhigh-throughput%2Bidentification%2Bof%2Btargets%2Bof%2Byeast%2BDNA-binding%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.148%26rft_id%253Dinfo%253Apmid%252F18802438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18802438&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.49" data-doi="10.1101/gr.114850.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayhew</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Calling Cards enable multiplexed identification of the genomic targets of DNA-binding proteins.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114850.110%26rft_id%253Dinfo%253Apmid%252F21471402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvNzQ4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.50" data-doi="10.1093/emboj/18.19.5334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinreich</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Cdc7p-Dbf4p kinase binds to chromatin during S phase and is regulated by both the APC and the RAD53 checkpoint pathway.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">5334</span>–<span class="cit-lpage">5346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DWeinreich%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D19%26rft.spage%253D5334%26rft.epage%253D5346%26rft.atitle%253DCdc7p-Dbf4p%2Bkinase%2Bbinds%2Bto%2Bchromatin%2Bduring%2BS%2Bphase%2Band%2Bis%2Bregulated%2Bby%2Bboth%2Bthe%2BAPC%2Band%2Bthe%2BRAD53%2Bcheckpoint%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F18.19.5334%26rft_id%253Dinfo%253Apmid%252F10508166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTgvMTkvNTMzNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.51" data-doi="10.1074/jbc.271.6.3265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noh</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">K.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Selective activation of MEK1 but not Mek2 by A-Raf from epidermal growth factor-stimulated Hela cells.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">3265</span>–<span class="cit-lpage">3271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DWu%26rft.auinit1%253DX.%26rft.volume%253D271%26rft.issue%253D6%26rft.spage%253D3265%26rft.epage%253D3271%26rft.atitle%253DSelective%2BActivation%2Bof%2BMEK1%2Bbut%2BNot%2BMEK2%2Bby%2BA-Raf%2Bfrom%2BEpidermal%2BGrowth%2BFactor-stimulated%2BHela%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.271.6.3265%26rft_id%253Dinfo%253Apmid%252F8621729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzEvNi8zMjY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.52" data-doi="10.1128/MCB.21.19.6606-6614.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zappulla</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternglanz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Targeting of the yeast Ty5 retrotransposon to silent chromatin is mediated by interactions between integrase and Sir4p.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">6606</span>–<span class="cit-lpage">6614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DXie%26rft.auinit1%253DW.%26rft.volume%253D21%26rft.issue%253D19%26rft.spage%253D6606%26rft.epage%253D6614%26rft.atitle%253DTargeting%2Bof%2Bthe%2BYeast%2BTy5%2BRetrotransposon%2Bto%2BSilent%2BChromatin%2BIs%2BMediated%2Bby%2BInteractions%2Bbetween%2BIntegrase%2Band%2BSir4p%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.19.6606-6614.2001%26rft_id%253Dinfo%253Apmid%252F11533248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xOS82NjA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.53" data-doi="10.1186/1471-2164-7-276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide mapping of ORC and Mcm2p binding sites on tiling arrays and identification of essential ARS consensus sequences in <em>S. cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DXu%26rft.auinit1%253DW.%26rft.volume%253D7%26rft.spage%253D276%26rft.epage%253D276%26rft.atitle%253DGenome-wide%2Bmapping%2Bof%2BORC%2Band%2BMcm2p%2Bbinding%2Bsites%2Bon%2Btiling%2Barrays%2Band%2Bidentification%2Bof%2Bessential%2BARS%2Bconsensus%2Bsequences%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-276%26rft_id%253Dinfo%253Apmid%252F17067396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17067396&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.54" data-doi="10.1046/j.1365-2443.2002.00559.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yabuki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terashima</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitada</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mapping of early firing origins on a replication profile of budding yeast.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DYabuki%26rft.auinit1%253DN%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D781%26rft.epage%253D789%26rft.atitle%253DMapping%2Bof%2Bearly%2Bfiring%2Borigins%2Bon%2Ba%2Breplication%2Bprofile%2Bof%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.2002.00559.x%26rft_id%253Dinfo%253Apmid%252F12167157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.2002.00559.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12167157&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177365700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3757.55" data-doi="10.1038/msb.2010.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhind</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechhoefer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Modeling genome-wide replication kinetics reveals a mechanism for regulation of replication timing.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DYang%26rft.auinit1%253DS.%2BC.-H.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D404%26rft.epage%253D404%26rft.atitle%253DModeling%2Bgenome-wide%2Breplication%2Bkinetics%2Breveals%2Ba%2Bmechanism%2Bfor%2Bregulation%2Bof%2Breplication%2Btiming%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2010.61%26rft_id%253Dinfo%253Apmid%252F20739926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjYvMS80MDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3757.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3745" title="Serotonin Drives Predatory Feeding Behavior via Synchronous Feeding Rhythms in the Nematode Pristionchus pacificus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3775" title="Whole Genome Sequence Analysis of Mutations Accumulated in rad27Δ Yeast Strains with Defects in the Processing of Okazaki Fragments Indicates Template-Switching Events" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3757.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cUNwS-ujrNOVjojZPiP-3AwkkNMZs0_M36mOfeMzZ2Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3757" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3757" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3757" class="active" data-icon-position="" data-hide-link-title="0">Localization of Cdc7 Protein Kinase During DNA Replication in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487101" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LbwckRI7PHHJCw3HoKanoQ2MfhJ4Kw9-2V7NsncuTfc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3757.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3757" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1IMhftST2bfbhi98Uwjk1b7zq5v_TMzrfMuEAHMXtBs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3757" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3757.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487101" id="node487101--2" data-pisa="ggg;7/11/3757" data-pisa-master="ggg;g3.117.300223" data-apath="/ggg/7/11/3757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Localization of Cdc7 Protein Kinase During DNA Replication in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Rossbach</span>, <span class="highwire-citation-author" data-delta="1">D. Suzi Bryan</span>, <span class="highwire-citation-author" data-delta="2">Jay R. Hesselberth</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-1100-6489" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert Sclafani</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3757-3774;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300223 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487101--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Rossbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BRossbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rossbach%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BRossbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. Suzi Bryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BSuzi%2BBryan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryan%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BSuzi%2BBryan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jay R. Hesselberth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jay%2BR.%2BHesselberth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hesselberth%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJay%2BR.%2BHesselberth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert Sclafani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BSclafani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sclafani%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BSclafani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1100-6489" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert Sclafani</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:robert.sclafani@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">robert.sclafani@ucdenver.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3757" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3757&amp;title=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3757&amp;title=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3757&amp;title=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3757&amp;text=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3757&amp;title=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3757&amp;t=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3757&amp;title=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3757&amp;title=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487101/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487101" id="node487101--3" data-pisa="ggg;7/11/3757" data-pisa-master="ggg;g3.117.300223" data-apath="/ggg/7/11/3757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Localization of Cdc7 Protein Kinase During DNA Replication in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Rossbach</span>, <span class="highwire-citation-author" data-delta="1">D. Suzi Bryan</span>, <span class="highwire-citation-author" data-delta="2">Jay R. Hesselberth</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-1100-6489" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert Sclafani</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3757-3774;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300223 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487101--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Rossbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BRossbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rossbach%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BRossbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. Suzi Bryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BSuzi%2BBryan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryan%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BSuzi%2BBryan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jay R. Hesselberth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jay%2BR.%2BHesselberth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hesselberth%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJay%2BR.%2BHesselberth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert Sclafani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BSclafani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sclafani%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BSclafani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1100-6489" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert Sclafani</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:robert.sclafani@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">robert.sclafani@ucdenver.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3757&amp;rft.epage=3774&amp;rft.atitle=Localization%20of%20Cdc7%20Protein%20Kinase%20During%20DNA%20Replication%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rossbach%2C+Daniel&amp;rft.au=Bryan%2C+D.+Suzi&amp;rft.au=Hesselberth%2C+Jay+R.&amp;rft.au=Sclafani%2C+Robert"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487101/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487101/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487101/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487101/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487101/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487101/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487101/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487101/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487101/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487101/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487101/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487101/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3757" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3757.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3757.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3757.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3757.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
