<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/11/3605" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/11/3605.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/11/3605.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/11/3605.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/11/3605/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/11/3605.atom" />
<meta name="HW.pisa" content="ggg;7/11/3605" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories" />
<meta name="DC.Identifier" content="10.1534/g3.117.300259" />
<meta name="DC.Date" content="2017-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Beichman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Inference of demographic history from genetic data is a primary goal of population genetics of model and nonmodel organisms. Whole genome-based approaches such as the pairwise/multiple sequentially Markovian coalescent methods use genomic data from one to four individuals to infer the demographic history of an entire population, while site frequency spectrum (SFS)-based methods use the distribution of allele frequencies in a sample to reconstruct the same historical events. Although both methods are extensively used in empirical studies and perform well on data simulated under simple models, there have been only limited comparisons of them in more complex and realistic settings. Here we use published demographic models based on data from three human populations (Yoruba, descendants of northwest-Europeans, and Han Chinese) as an empirical test case to study the behavior of both inference procedures. We find that several of the demographic histories inferred by the whole genome-based methods do not predict the genome-wide distribution of heterozygosity, nor do they predict the empirical SFS. However, using simulated data, we also find that the whole genome methods can reconstruct the complex demographic models inferred by SFS-based methods, suggesting that the discordant patterns of genetic variation are not attributable to a lack of statistical power, but may reflect unmodeled complexities in the underlying demography. More generally, our findings indicate that demographic inference from a small number of genomes, routine in genomic studies of nonmodel organisms, should be interpreted cautiously, as these models cannot recapitulate other summaries of the data." />
<meta name="DC.Contributor" content="Annabel C. Beichman" />
<meta name="DC.Contributor" content="Tanya N. Phung" />
<meta name="DC.Contributor" content="Kirk E. Lohmueller" />
<meta name="article:published_time" content="2017-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Inference of demographic history from genetic data is a primary goal of population genetics of model and nonmodel organisms. Whole genome-based approaches such as the pairwise/multiple sequentially Markovian coalescent methods use genomic data from one to four individuals to infer the demographic history of an entire population, while site frequency spectrum (SFS)-based methods use the distribution of allele frequencies in a sample to reconstruct the same historical events. Although both methods are extensively used in empirical studies and perform well on data simulated under simple models, there have been only limited comparisons of them in more complex and realistic settings. Here we use published demographic models based on data from three human populations (Yoruba, descendants of northwest-Europeans, and Han Chinese) as an empirical test case to study the behavior of both inference procedures. We find that several of the demographic histories inferred by the whole genome-based methods do not predict the genome-wide distribution of heterozygosity, nor do they predict the empirical SFS. However, using simulated data, we also find that the whole genome methods can reconstruct the complex demographic models inferred by SFS-based methods, suggesting that the discordant patterns of genetic variation are not attributable to a lack of statistical power, but may reflect unmodeled complexities in the underlying demography. More generally, our findings indicate that demographic inference from a small number of genomes, routine in genomic studies of nonmodel organisms, should be interpreted cautiously, as these models cannot recapitulate other summaries of the data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/11/01" />
<meta name="citation_mjid" content="ggg;7/11/3605" />
<meta name="citation_id" content="7/11/3605" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/11/3605" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/11/3605.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/11/3605.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/11/3605.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300259" />
<meta name="citation_pmid" content="28893846" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3605" />
<meta name="citation_lastpage" content="3620" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Annabel C. Beichman" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6991-587X" />
<meta name="citation_author" content="Tanya N. Phung" />
<meta name="citation_author_institution" content="Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2685-6977" />
<meta name="citation_author" content="Kirk E. Lohmueller" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, California 90095" />
<meta name="citation_author_email" content="klohmueller@ucla.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3874-369X" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_title=A global reference for human genetic variation.;citation_pages=68-74;citation_volume=526;citation_year=2015;citation_pmid=26432245;citation_doi=10.1038/nature15393" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. M. Adams;citation_author=R. R. Hudson;citation_title=Maximum-Likelihood Estimation of Demographic Parameters Using the Frequency Spectrum of Unlinked Single-Nucleotide Polymorphisms;citation_pages=1699-1712;citation_volume=168;citation_year=2004;citation_issue=3;citation_pmid=15579718;citation_doi=10.1534/genetics.104.030171" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. A. Albert;citation_author=W. B. Barbazuk;citation_author=J. P. Der;citation_author=J. Leebens-Mack;citation_author=H. Ma;citation_title=The Amborella Genome and the Evolution of Flowering Plants;citation_pages=1241089-1241089;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357323;citation_doi=10.1126/science.1241089" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=L. Arbiza;citation_author=E. Zhong;citation_author=A. Keinan;citation_title=NRE: A tool for exploring neutral loci in the human genome.;citation_pages=301-301;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23151206;citation_doi=10.1186/1471-2105-13-301" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. Ardlie;citation_author=S. N. Liu-Cordero;citation_author=M. A. Eberle;citation_author=M. Daly;citation_author=J. Barrett;citation_title=Lower-than-expected linkage disequilibrium between tightly linked markers in humans suggests a role for gene conversion.;citation_pages=582-589;citation_volume=69;citation_year=2001;citation_issue=3;citation_pmid=11473344;citation_doi=10.1086/323251" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Bhaskar;citation_author=Y. R. Wang;citation_author=Y. S. Song;citation_title=Efficient inference of population size histories and locus-specific mutation rates from large-sample genomic variation data.;citation_volume=25;citation_year=2015;citation_pmid=25564017;citation_doi=10.1101/gr.178756.114" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. G. Blum;citation_author=M. Jakobsson;citation_title=Deep divergences of human gene trees and models of human origins.;citation_volume=28;citation_year=2011;citation_pmid=20930054;citation_doi=10.1093/molbev/msq265" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Boitard;citation_author=W. Rodriguez;citation_author=F. Jay;citation_author=S. Mona;citation_author=F. Austerlitz;citation_title=Inferring population size history from large samples of genome-wide molecular data-an approximate Bayesian computation approach.;citation_pages=e1005877;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Bosse;citation_author=H.-J. Megens;citation_author=O. Madsen;citation_author=L. A. Frantz;citation_author=Y. Paudel;citation_title=Untangling the hybrid nature of modern pig genomes: a mosaic derived from biogeographically distinct and highly divergent Sus scrofa populations.;citation_pages=4089-4102;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12807" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. R. Boyko;citation_author=S. H. Williamson;citation_author=A. R. Indap;citation_author=J. D. Degenhardt;citation_author=R. D. Hernandez;citation_title=Assessing the evolutionary impact of amino acid mutations in the human genome.;citation_pages=e1000083-e1000083;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18516229;citation_doi=10.1371/journal.pgen.1000083" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Bunnefeld;citation_author=L. A. F. Frantz;citation_author=K. Lohse;citation_title=Inferring bottlenecks from genome-wide samples of short sequence blocks.;citation_volume=201;citation_year=2015;citation_pmid=26341659;citation_doi=10.1534/genetics.115.179861" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B. Biol. Sci.;citation_author=J. A. Cahill;citation_author=A. E. Soares;citation_author=R. E. Green;citation_author=B. Shapiro;citation_title=Inferring species divergence times using pairwise sequential Markovian coalescent modelling and low-coverage genomic data.;citation_volume=371;citation_year=2016;citation_pmid=27325835;citation_doi=10.1098/rstb.2015.0138" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. K. Chen;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_title=Fast and flexible simulation of DNA sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19029539;citation_doi=10.1101/gr.083634.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Chikhi;citation_author=V. C. Sousa;citation_author=P. Luisi;citation_author=B. Goossens;citation_author=M. A. Beaumont;citation_title=The confounding effects of population structure, genetic diversity and the sampling scheme on the detection and quantification of population size changes.;citation_volume=186;citation_year=2010;citation_pmid=20739713;citation_doi=10.1534/genetics.110.118661" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. Drmanac;citation_author=A. B. Sparks;citation_author=M. J. Callow;citation_author=A. L. Halpern;citation_author=N. L. Burns;citation_title=Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays.;citation_volume=327;citation_year=2010;citation_pmid=19892942;citation_doi=10.1126/science.1181498" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Ewing;citation_author=J. Hermisson;citation_title=MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus.;citation_volume=26;citation_year=2010;citation_pmid=20591904;citation_doi=10.1093/bioinformatics/btq322" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Excoffier;citation_author=I. Dupanloup;citation_author=E. Huerta-Sánchez;citation_author=V. C. Sousa;citation_author=M. Foll;citation_title=Robust demographic inference from genomic and SNP data.;citation_pages=e1003905;citation_volume=9;citation_year=2013;citation_pmid=24204310;citation_doi=10.1371/journal.pgen.1003905" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. J. Fagundes;citation_author=N. Ray;citation_author=M. Beaumont;citation_author=S. Neuenschwander;citation_author=F. M. Salzano;citation_title=Statistical evaluation of alternative models of human evolution.;citation_volume=104;citation_year=2007;citation_pmid=17978179;citation_doi=10.1073/pnas.0708280104" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. R. Fitak;citation_author=E. Mohandesan;citation_author=J. Corander;citation_author=P. A. Burger;citation_title=The de novo genome assembly and annotation of a female domestic dromedary of North African origin.;citation_pages=314-324;citation_volume=16;citation_year=2016;citation_pmid=26178449;citation_doi=10.1111/1755-0998.12443" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. H. Freedman;citation_author=I. Gronau;citation_author=R. M. Schweizer;citation_author=D. Ortega-Del Vecchyo;citation_author=E. Han;citation_title=Genome sequencing highlights the dynamic early history of dogs.;citation_pages=e1004016;citation_volume=10;citation_year=2014;citation_pmid=24453982;citation_doi=10.1371/journal.pgen.1004016" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Frisse;citation_author=R. R. Hudson;citation_author=A. Bartoszewicz;citation_author=J. D. Wall;citation_author=J. Donfack;citation_title=Gene conversion and different population histories may explain the contrast between polymorphism and linkage disequilibrium levels.;citation_pages=831-843;citation_volume=69;citation_year=2001;citation_issue=4;citation_pmid=11533915;citation_doi=10.1086/323612" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=F. Gao;citation_author=A. Keinan;citation_title=Explosive genetic evidence for explosive human population growth.;citation_pages=130-139;citation_volume=41;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Garrigan;citation_author=M. F. Hammer;citation_title=Reconstructing human origins in the genomic era.;citation_pages=669-680;citation_volume=7;citation_year=2006;citation_issue=9;citation_pmid=16921345;citation_doi=10.1038/nrg1941" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. R. Garud;citation_author=P. W. Messer;citation_author=E. O. Buzbas;citation_author=D. A. Petrov;citation_title=Recent selective sweeps in North American Drosophila melanogaster show signatures of soft sweeps.;citation_pages=e1005004;citation_volume=11;citation_year=2015;citation_pmid=25706129;citation_doi=10.1371/journal.pgen.1005004" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=L. M. Gattepaille;citation_author=M. Jakobsson;citation_author=M. G. Blum;citation_title=Inferring population size changes with sequence and SNP data: lessons from human bottlenecks.;citation_pages=409-419;citation_volume=110;citation_year=2013;citation_pmid=23423148;citation_doi=10.1038/hdy.2012.120" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Gazave;citation_author=L. Ma;citation_author=D. Chang;citation_author=A. Coventry;citation_author=F. Gao;citation_title=Neutral genomic regions refine models of recent rapid human population growth.;citation_volume=111;citation_year=2014;citation_pmid=24379384;citation_doi=10.1073/pnas.1310398110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gravel;citation_author=B. M. Henn;citation_author=R. N. Gutenkunst;citation_author=A. R. Indap;citation_author=G. T. Marth;citation_title=Demographic history and rare allele sharing among human populations.;citation_volume=108;citation_year=2011;citation_pmid=21730125;citation_doi=10.1073/pnas.1019276108" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. A. M. Groenen;citation_author=A. L. Archibald;citation_author=H. Uenishi;citation_author=C. K. Tuggle;citation_author=Y. Takeuchi;citation_title=Analyses of pig genomes provide insight into porcine demography and evolution.;citation_pages=393-398;citation_volume=491;citation_year=2012;citation_issue=7424;citation_pmid=23151582;citation_doi=10.1038/nature11622" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. N. Gutenkunst;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.;citation_pages=e1000695-e1000695;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851460;citation_doi=10.1371/journal.pgen.1000695" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Han;citation_author=J. S. Sinsheimer;citation_author=J. Novembre;citation_title=Characterizing bias in population genetic inferences from low-coverage sequencing data.;citation_volume=31;citation_year=2014;citation_pmid=24288159;citation_doi=10.1093/molbev/mst229" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Han;citation_author=J. S. Sinsheimer;citation_author=J. Novembre;citation_title=Fast and accurate site frequency spectrum estimation from low coverage sequence data.;citation_volume=31;citation_year=2015;citation_pmid=25359894;citation_doi=10.1093/bioinformatics/btu725" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. C. Harpending;citation_author=M. A. Batzer;citation_author=M. Gurven;citation_author=L. B. Jorde;citation_author=A. R. Rogers;citation_title=Genetic traces of ancient demography;citation_pages=1961-1967;citation_volume=95;citation_year=1998;citation_issue=4;citation_pmid=9465125;citation_doi=10.1073/pnas.95.4.1961" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Harris;citation_author=R. Nielsen;citation_title=Inferring demographic history from a spectrum of shared haplotype lengths.;citation_pages=e1003521;citation_volume=9;citation_year=2013;citation_pmid=23754952;citation_doi=10.1371/journal.pgen.1003521" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. Hawks;citation_author=K. Hunley;citation_author=S.-H. Lee;citation_author=M. Wolpoff;citation_title=Population Bottlenecks and Pleistocene Human Evolution;citation_pages=2-22;citation_volume=17;citation_year=2000;citation_issue=1;citation_pmid=10666702;citation_doi=10.1093/oxfordjournals.molbev.a026233" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. Heller;citation_author=L. Chikhi;citation_author=H. R. Siegismund;citation_title=The confounding effect of population structure on Bayesian skyline plot inferences of demographic history.;citation_pages=1-10;citation_volume=8;citation_year=2013;citation_pmid=23620825;citation_doi=10.1371/journal.pone.0054583" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Henn;citation_author=L. L. Cavalli-Sforza;citation_author=M. W. Feldman;citation_title=The great human expansion.;citation_volume=109;citation_year=2012;citation_pmid=23077256;citation_doi=10.1073/pnas.1212380109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Henn;citation_author=L. R. Botigué;citation_author=S. Peischl;citation_author=I. Dupanloup;citation_author=M. Lipatov;citation_title=Distance from sub-Saharan Africa predicts mutational load in diverse human genomes.;citation_volume=113;citation_year=2016;citation_pmid=26712023;citation_doi=10.1073/pnas.1510805112" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. A. Holliday;citation_author=L. Zhou;citation_author=R. Bawa;citation_author=M. Zhang;citation_author=R. W. Oubida;citation_title=Evidence for extensive parallelism but divergent genomic architecture of adaptation along altitudinal and latitudinal gradients in Populus trichocarpa.;citation_pages=1240-1251;citation_volume=209;citation_year=2016;citation_pmid=26372471;citation_doi=10.1111/nph.13643" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Hsieh;citation_author=K. R. Veeramah;citation_author=J. Lachance;citation_author=S. A. Tishkoff;citation_author=J. D. Wall;citation_title=Whole-genome sequence analyses of Western Central African Pygmy hunter-gatherers reveal a complex demographic history and identify candidate genes under positive natural selection.;citation_volume=26;citation_year=2016;citation_pmid=26888263;citation_doi=10.1101/gr.192971.115" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. R. Hudson;citation_title=Generating samples under a Wright-Fisher neutral model of  genetic variation;citation_pages=337-338;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11847089;citation_doi=10.1093/bioinformatics/18.2.337" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C.-M. Hung;citation_author=P.-J. L. Shaner;citation_author=R. M. Zink;citation_author=W.-C. Liu;citation_author=T.-C. Chu;citation_title=Drastic population fluctuations explain the rapid extinction of the passenger pigeon.;citation_volume=111;citation_year=2014;citation_pmid=24979776;citation_doi=10.1073/pnas.1401526111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=E. Ibarra-Laclette;citation_author=E. Lyons;citation_author=G. Hernández-Guzmán;citation_author=C. A. Pérez-Torres;citation_author=L. Carretero-Paulet;citation_title=Architecture and evolution of a minute plant genome.;citation_pages=94-98;citation_volume=498;citation_year=2013;citation_pmid=23665961;citation_doi=10.1038/nature12132" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Jouganous;citation_author=W. Long;citation_author=A. P. Ragsdale;citation_author=S. Gravel;citation_title=Inferring the joint demographic history of multiple populations: beyond the diffusion approximation.;citation_volume=206;citation_year=2017;citation_pmid=28495960;citation_doi=10.1534/genetics.117.200493" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Keinan;citation_author=A. G. Clark;citation_title=Recent Explosive Human Population Growth Has Resulted in an Excess of Rare Genetic Variants;citation_pages=740-743;citation_volume=336;citation_year=2012;citation_issue=6082;citation_pmid=22582263;citation_doi=10.1126/science.1217283" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Keinan;citation_author=J. C. Mullikin;citation_author=N. Patterson;citation_author=D. Reich;citation_title=Measurement of the human allele frequency spectrum demonstrates greater genetic drift in East Asians than in Europeans.;citation_pages=1251-1255;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17828266;citation_doi=10.1038/ng2116" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. M. Kidd;citation_author=S. Gravel;citation_author=J. Byrnes;citation_author=A. Moreno-Estrada;citation_author=S. Musharoff;citation_title=Population genetic inference from personal genome data: impact of ancestry and admixture on human genomic variation.;citation_pages=660-671;citation_volume=91;citation_year=2012;citation_issue=4;citation_pmid=23040495;citation_doi=10.1016/j.ajhg.2012.08.025" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. Y. Kim;citation_author=K. E. Lohmueller;citation_author=A. Albrechtsen;citation_author=Y. Li;citation_author=T. Korneliussen;citation_title=Estimation of allele frequency and association mapping using next-generation sequencing data.;citation_pages=231-231;citation_volume=12;citation_year=2011;citation_pmid=21663684;citation_doi=10.1186/1471-2105-12-231" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=D. F. Gudbjartsson;citation_author=G. Masson;citation_author=A. Sigurdsson;citation_title=Fine-scale recombination rate differences between sexes, populations and individuals.;citation_pages=1099-1103;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981099;citation_doi=10.1038/nature09525" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Li;citation_author=R. Durbin;citation_title=Inference of human population history from individual whole-genome sequences.;citation_pages=493-496;citation_volume=475;citation_year=2011;citation_issue=7357;citation_pmid=21753753;citation_doi=10.1038/nature10231" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=X. Liu;citation_author=Y.-X. Fu;citation_title=Exploring population size changes using SNP frequency spectra.;citation_pages=555-559;citation_volume=47;citation_year=2015;citation_pmid=25848749;citation_doi=10.1038/ng.3254" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. E. Lohmueller;citation_author=C. D. Bustamante;citation_author=A. G. Clark;citation_title=Methods for human demographic inference using haplotype patterns from genomewide single-nucleotide polymorphism data.;citation_volume=182;citation_year=2009;citation_pmid=19255370;citation_doi=10.1534/genetics.108.099275" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. S. Malaspinas;citation_author=M. C. Westaway;citation_author=C. Muller;citation_author=V. C. Sousa;citation_author=O. Lao;citation_title=A genomic history of Aboriginal Australia.;citation_pages=207-214;citation_volume=538;citation_year=2016;citation_pmid=27654914;citation_doi=10.1038/nature18299" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Mallick;citation_author=H. Li;citation_author=M. Lipson;citation_author=I. Mathieson;citation_author=M. Gymrek;citation_title=The Simons Genome Diversity Project: 300 genomes from 142 diverse populations.;citation_pages=201-206;citation_volume=538;citation_year=2016;citation_pmid=27654912;citation_doi=10.1038/nature18964" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_title=Real-time PCR quantification of human complement C4A and C4B genes.;citation_pages=1-1;citation_volume=7;citation_year=2006;citation_pmid=16403222;citation_doi=10.1186/1471-2156-7-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. T. Marth;citation_author=E. Czabarka;citation_author=J. Murvai;citation_author=S. T. Sherry;citation_title=The Allele Frequency Spectrum in Genome-Wide Human Variation Data Reveals Signals of Differential Demographic History in Three Large World Populations;citation_pages=351-372;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020430;citation_doi=10.1534/genetics.166.1.351" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=O. Mazet;citation_author=W. Rodriguez;citation_author=L. Chikhi;citation_title=Demographic inference using genetic data from a single individual: separating population size variation from population structure.;citation_pages=46-58;citation_volume=104;citation_year=2015;citation_pmid=26120083;citation_doi=10.1016/j.tpb.2015.06.003" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=O. Mazet;citation_author=W. Rodriguez;citation_author=S. Grusea;citation_author=S. Boitard;citation_author=L. Chikhi;citation_title=On the importance of being structured: instantaneous coalescence rates and human evolution—lessons for ancestral population size inference?;citation_pages=362-371;citation_volume=116;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. C. McCoy;citation_author=N. R. Garud;citation_author=J. L. Kelley;citation_author=C. L. Boggs;citation_author=D. A. Petrov;citation_title=Genomic inference accurately predicts the timing and severity of a recent bottleneck in a nonmodel insect population.;citation_pages=136-150;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12591" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=G. A. McVean;citation_author=N. J. Cardin;citation_title=Approximating the coalescent with recombination;citation_pages=1387-1393;citation_volume=360;citation_year=2005;citation_issue=1459;citation_pmid=16048782;citation_doi=10.1098/rstb.2005.1673" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Meyer;citation_author=M. Kircher;citation_author=M.-T. Gansauge;citation_author=H. Li;citation_author=F. Racimo;citation_title=A high-coverage genome sequence from an archaic Denisovan individual.;citation_volume=338;citation_year=2012;citation_pmid=22936568;citation_doi=10.1126/science.1224344" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Nadachowska-Brzyska;citation_author=R. Burri;citation_author=P. I. Olason;citation_author=T. Kawakami;citation_author=L. A. Smeds;citation_title=Demographic divergence history of pied flycatcher and collared flycatcher inferred from whole-genome re-sequencing data.;citation_pages=e1003942;citation_volume=9;citation_year=2013;citation_pmid=24244198;citation_doi=10.1371/journal.pgen.1003942" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=K. Nadachowska-Brzyska;citation_author=C. Li;citation_author=L. Smeds;citation_author=G. Zhang;citation_author=H. Ellegren;citation_title=Temporal dynamics of avian populations during Pleistocene revealed by whole-genome sequences.;citation_pages=1375-1380;citation_volume=25;citation_year=2015;citation_pmid=25891404;citation_doi=10.1016/j.cub.2015.03.047" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. Nadachowska-Brzyska;citation_author=R. Burri;citation_author=L. Smeds;citation_author=H. Ellegren;citation_title=PSMC analysis of effective population sizes in molecular ecology and its application to black‐and‐white Ficedula flycatchers.;citation_pages=1058-1072;citation_volume=25;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. Nei;citation_author=T. Maruyama;citation_author=R. Chakraborty;citation_title=The bottleneck effect and genetic variability in populations.;citation_pages=1-10;citation_volume=29;citation_year=1975;citation_doi=10.2307/2407137" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. R. Nelson;citation_author=D. Wegmann;citation_author=M. G. Ehm;citation_author=D. Kessner;citation_author=P. S. Jean;citation_title=An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.;citation_volume=337;citation_year=2012;citation_pmid=22604722;citation_doi=10.1126/science.1217876" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Nielsen;citation_title=Estimation of Population Parameters and Recombination Rates From Single Nucleotide Polymorphisms;citation_pages=931-942;citation_volume=154;citation_year=2000;citation_issue=2;citation_pmid=10655242" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Nielsen;citation_author=M. J. Hubisz;citation_author=I. Hellmann;citation_author=D. Torgerson;citation_author=A. M. Andres;citation_title=Darwinian and demographic forces affecting human protein coding genes.;citation_volume=19;citation_year=2009;citation_pmid=19279335;citation_doi=10.1101/gr.088336.108" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Orlando;citation_author=A. Ginolhac;citation_author=G. Zhang;citation_author=D. Froese;citation_author=A. Albrechtsen;citation_title=Recalibrating Equus evolution using the genome sequence of an early middle Pleistocene horse;citation_pages=74-78;citation_volume=499;citation_year=2013;citation_issue=7456;citation_pmid=23803765;citation_doi=10.1038/nature12323;citation_georef=2013063355" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=P. Orozco-terWengel;citation_title=The devil is in the details: the effect of population structure on demographic inference.;citation_pages=349-350;citation_volume=116;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. Pagani;citation_author=D. J. Lawson;citation_author=E. Jagoda;citation_author=A. Mörseburg;citation_author=A. Eriksson;citation_title=Genomic analyses inform on migration events during the peopling of Eurasia.;citation_pages=238-242;citation_volume=538;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=E. Palkopoulou;citation_author=S. Mallick;citation_author=P. Skoglund;citation_author=J. Enk;citation_author=N. Rohland;citation_title=Complete genomes reveal signatures of demographic and genetic declines in the woolly mammoth.;citation_pages=1395-1400;citation_volume=25;citation_year=2015;citation_pmid=25913407;citation_doi=10.1016/j.cub.2015.04.007" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=B. M. Peter;citation_author=D. Wegmann;citation_author=L. Excoffier;citation_title=Distinguishing between population bottleneck and population subdivision by a Bayesian model choice procedure.;citation_pages=4648-4660;citation_volume=19;citation_year=2010;citation_issue=21;citation_pmid=20735743;citation_doi=10.1111/j.1365-294X.2010.04783.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. N. Phung;citation_author=C. D. Huber;citation_author=K. E. Lohmueller;citation_title=Determining the effect of natural selection on linked neutral divergence across species.;citation_pages=e1006199;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=V. Plagnol;citation_author=J. D. Wall;citation_title=Possible ancestral structure in human populations.;citation_pages=e105-e105;citation_volume=2;citation_year=2006;citation_issue=7;citation_pmid=16895447;citation_doi=10.1371/journal.pgen.0020105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Polanski;citation_author=M. Kimmel;citation_title=New Explicit Expressions for Relative Frequencies of Single-Nucleotide Polymorphisms With Application to Statistical Inference on Population Growth;citation_pages=427-436;citation_volume=165;citation_year=2003;citation_issue=1;citation_pmid=14504247" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Prado-Martinez;citation_author=P. H. Sudmant;citation_author=J. M. Kidd;citation_author=H. Li;citation_author=J. L. Kelley;citation_title=Great ape genetic diversity and population history.;citation_pages=471-475;citation_volume=499;citation_year=2013;citation_pmid=23823723;citation_doi=10.1038/nature12228" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Prufer;citation_author=F. Racimo;citation_author=N. Patterson;citation_author=F. Jay;citation_author=S. Sankararaman;citation_title=The complete genome sequence of a Neanderthal from the Altai Mountains;citation_pages=43-49;citation_volume=505;citation_year=2014;citation_issue=7481;citation_pmid=24352235;citation_doi=10.1038/nature12886;citation_georef=2014009693" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. E. Ptak;citation_author=M. Przeworski;citation_title=Evidence for population growth in humans is confounded by fine-scale population structure.;citation_pages=559-563;citation_volume=18;citation_year=2002;citation_issue=11;citation_pmid=12414185;citation_doi=10.1016/S0168-9525(02)02781-6" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Schiffels;citation_author=R. Durbin;citation_title=Inferring human population size and separation history from multiple genome sequences.;citation_pages=919-925;citation_volume=46;citation_year=2014;citation_pmid=24952747;citation_doi=10.1038/ng.3015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. A. Scholz;citation_author=T. C. Johnson;citation_author=A. S. Cohen;citation_author=J. W. King;citation_author=J. A. Peck;citation_title=East African megadroughts between 135 and 75 thousand years ago and bearing on early-modern human origins.;citation_volume=104;citation_year=2007;citation_pmid=17785420;citation_doi=10.1073/pnas.0703874104;citation_georef=2009081915" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. G. Schraiber;citation_author=J. M. Akey;citation_title=Methods and models for unravelling human evolutionary history.;citation_pages=727-740;citation_volume=16;citation_year=2015;citation_pmid=26553329" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=A. G. Shanku;citation_author=A. D. Kern;citation_title=Effects of linked selective sweeps on demographic inference and model selection.;citation_volume=204;citation_year=2016;citation_pmid=27605051;citation_doi=10.1534/genetics.116.190223" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Sjödin;citation_author=A. E. Sjöstrand;citation_author=M. Jakobsson;citation_author=M. G. Blum;citation_title=Resequencing data provide no evidence for a human bottleneck in Africa during the penultimate glacial period.;citation_volume=29;citation_year=2012;citation_pmid=22319141;citation_doi=10.1093/molbev/mss061" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Slatkin;citation_author=R. R. Hudson;citation_title=Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations.;citation_pages=555-562;citation_volume=129;citation_year=1991;citation_issue=2;citation_pmid=1743491" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Song;citation_author=E. Sliwerska;citation_author=S. Emery;citation_author=J. M. Kidd;citation_title=Modeling human population separation history using physically phased genomes.;citation_pages=385-395;citation_volume=205;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=M. G. Sovic;citation_author=B. C. Carstens;citation_author=H. L. Gibbs;citation_title=Genetic diversity in migratory bats: results from RADseq data for three tree bat species at an Ohio windfarm.;citation_pages=e1647;citation_volume=4;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=The effect of change in population size on DNA polymorphism.;citation_pages=597-601;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2599369" />
<meta name="citation_reference" content="citation_journal_title=Immunogenetics;citation_journal_abbrev=Immunogenetics;citation_author=N. Takahata;citation_author=Y. Satta;citation_title=Footprints of intragenic recombination at HLA loci.;citation_pages=430-441;citation_volume=47;citation_year=1998;citation_issue=6;citation_pmid=9553149;citation_doi=10.1007/s002510050380" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=N. Takahata;citation_author=Y. Satta;citation_author=J. Klein;citation_title=Divergence time and population size in the lineage leading to modern humans.;citation_pages=198-221;citation_volume=48;citation_year=1995;citation_issue=2;citation_pmid=7482371;citation_doi=10.1006/tpbi.1995.1026" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. A. Tennessen;citation_author=A. W. Bigham;citation_author=T. D. O’Connor;citation_author=W. Fu;citation_author=E. E. Kenny;citation_title=Evolution and functional impact of rare coding variation from deep sequencing of human exomes.;citation_volume=337;citation_year=2012;citation_pmid=22604720;citation_doi=10.1126/science.1219240" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Terhorst;citation_author=Y. S. Song;citation_title=Fundamental limits on the accuracy of demographic inference based on the sample frequency spectrum.;citation_volume=112;citation_year=2015;citation_pmid=26056264;citation_doi=10.1073/pnas.1503717112" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Terhorst;citation_author=J. A. Kamm;citation_author=Y. S. Song;citation_title=Robust and scalable inference of population history from hundreds of unphased whole genomes.;citation_pages=303-309;citation_volume=49;citation_year=2017;citation_pmid=28024154;citation_doi=10.1038/ng.3748" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Thornton;citation_author=P. Andolfatto;citation_title=Approximate Bayesian inference reveals evidence for a recent, severe bottleneck in a Netherlands population of Drosophila melanogaster.;citation_volume=172;citation_year=2006;citation_pmid=16299396;citation_doi=10.1534/genetics.105.048223" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=E. Trucchi;citation_author=P. Gratton;citation_author=J. D. Whittington;citation_author=R. Cristofari;citation_author=Y. Le Maho;citation_title=King penguin demography since the last glaciation inferred from genome-wide data.;citation_volume=281;citation_year=2014;citation_pmid=24920481;citation_doi=10.1098/rspb.2014.0528" />
<meta name="citation_reference" content="citation_title=Coalescent Theory: An Introduction;citation_author=J. Wakeley;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. D. Wall;citation_author=J. K. Pritchard;citation_title=Haplotype blocks and linkage disequilibrium in the human genome.;citation_pages=587-597;citation_volume=4;citation_year=2003;citation_issue=8;citation_pmid=12897771;citation_doi=10.1038/nrg1123" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=G.-D. Wang;citation_author=W. Zhai;citation_author=H.-C. Yang;citation_author=L. Wang;citation_author=L. Zhong;citation_title=Out of southern East Asia: the natural history of domestic dogs across the world.;citation_pages=21-33;citation_volume=26;citation_year=2016;citation_pmid=26667385;citation_doi=10.1038/cr.2015.147" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=D. B. Weissman;citation_author=O. Hallatschek;citation_title=Minimal-assumption inference from population-genomic data.;citation_pages=e24836;citation_volume=6;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Zhao;citation_author=P. Zheng;citation_author=S. Dong;citation_author=X. Zhan;citation_author=Q. Wu;citation_title=Whole-genome sequencing of giant pandas provides insights into demographic history and local adaptation.;citation_pages=67-71;citation_volume=45;citation_year=2012;citation_pmid=23242367;citation_doi=10.1038/ng.2494" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
<meta name="twitter:description" content="Inference of demographic history from genetic data is a primary goal of population genetics of model and nonmodel organisms. Whole genome-based approaches such as the pairwise/multiple sequentially Markovian coalescent methods use genomic data from one to four individuals to infer the demographic history of an entire population, while site frequency spectrum (SFS)-based methods use the distribution of allele frequencies in a sample to reconstruct the same historical events. Although both methods are extensively used in empirical studies and perform well on data simulated under simple models, there have been only limited comparisons of them in more complex and realistic settings. Here we use published demographic models based on data from three human populations (Yoruba, descendants of northwest-Europeans, and Han Chinese) as an empirical test case to study the behavior of both inference procedures. We find that several of the demographic histories inferred by the whole genome-based methods do not predict the genome-wide distribution of heterozygosity, nor do they predict the empirical SFS. However, using simulated data, we also find that the whole genome methods can reconstruct the complex demographic models inferred by SFS-based methods, suggesting that the discordant patterns of genetic variation are not attributable to a lack of statistical power, but may reflect unmodeled complexities in the underlying demography. More generally, our findings indicate that demographic inference from a small number of genomes, routine in genomic studies of nonmodel organisms, should be interpreted cautiously, as these models cannot recapitulate other summaries of the data." />
<meta name="og-title" property="og:title" content="Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/11/3605" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Inference of demographic history from genetic data is a primary goal of population genetics of model and nonmodel organisms. Whole genome-based approaches such as the pairwise/multiple sequentially Markovian coalescent methods use genomic data from one to four individuals to infer the demographic history of an entire population, while site frequency spectrum (SFS)-based methods use the distribution of allele frequencies in a sample to reconstruct the same historical events. Although both methods are extensively used in empirical studies and perform well on data simulated under simple models, there have been only limited comparisons of them in more complex and realistic settings. Here we use published demographic models based on data from three human populations (Yoruba, descendants of northwest-Europeans, and Han Chinese) as an empirical test case to study the behavior of both inference procedures. We find that several of the demographic histories inferred by the whole genome-based methods do not predict the genome-wide distribution of heterozygosity, nor do they predict the empirical SFS. However, using simulated data, we also find that the whole genome methods can reconstruct the complex demographic models inferred by SFS-based methods, suggesting that the discordant patterns of genetic variation are not attributable to a lack of statistical power, but may reflect unmodeled complexities in the underlying demography. More generally, our findings indicate that demographic inference from a small number of genomes, routine in genomic studies of nonmodel organisms, should be interpreted cautiously, as these models cannot recapitulate other summaries of the data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11.cover-source.jpg" />
    <title>Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jr-hNl6q5I0DbQzesS0g4RbD8Nj8BO2C3pBRfGqIAZo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487108","apath":"\/ggg\/7\/11\/3605.atom","pisa":"ggg;7\/11\/3605","ac":{"\/ggg\/7\/11\/3605.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/11\/3605.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3605"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/11\/3605"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/11\/3605":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VRq6CoYzYy\\\\\\\/tw+hx7DpustQxw8szDYeJaHQg9YnyMQyvQ0FLTF6y5As9oGtuJlEXjgGF7hl9RYtCCAdl+Yqv0LI+nVJ5fqCRYUIb+4UKTy5zk5zNMwTLg\\\\\\\/tujVwgRCPnsvDl0SikgtTrEekV2829zhwvw2xTT68\\\\\\\/jQe7hveX9R9KdMkirMGLCsvXtyqQhnjWSh+P+DySuEV\\\\\\\/C4HBxm4olYcUoXfjbkIwALlEgZfTVNo3Xt+nawhBNWZFCD6XIiEeF2jCUZjoXHjIXP8coBm9B7m2Jw7zreHBbdjlRTXf6LVPD1DIKNeDcWg3TyXLYM08E31kfFgTZpe8fCECmXldwZqSE++2VGVZdt7rn339IaHbce6HNetrOxyeo6qZ5DVwmQpplWnmB60EvDwVRe4IYvK3MhTl7TZIZ3gtOJ5EkKWFXhUCOTzL7mvaoICJONwpAFpywCbamkBuMMafbQgaYFrMWdvtZC3rLryunutNk2MVelptMJLU4ZXB7BqaZqKN64u97ikdwg4MIHaXZPBx1BlZoK2Cy440u0HcOvphUFS1Q+q3pFsIauvZsJa8K8rSkpH0cxXAXw2RfamE826HK2wgSWWpgjYOx9dVvv1f+n\\\\\\\/Q57lZdb7TXgGGi6fGU9B4D8yM8g8VSI6Fo59cQVfJYutuJ4SyKjMrtsc+gigDUGR2US5SMS43\\\\\\\/NzD59LeLL\\\\\\\/HpElr6sFKZSBM2Ff\\\\\\\/FRdbuoUG4hmcz5f98KnL9c9C4\\\\\\\/hgmIM9fVqeTv6n8t2PSFUT6VFKvEmggfNWb0LUB39u1P\\\\\\\/P1htLgRc6aPYQniG9hdhMTTwc8T6it7ickDyni4Zl1TVPwVpTyGtVc2LNkloFNKhfkiU8civs6\\\\\\\/zn0vQt2JMc\\\\\\\/A3rZO+SnwnKFF8nesChxAoAUBiEWfdtdr8shweTnEapUMtoUvrQiRrjCaUMJF2fnuWojarwVy3yKtTeAFx9S7XAH9VvxW90XhUoEieyPEyIMOO2BNFViWvfDDl90KIP0DXVvnSQiTjezPD9i74pwtgHuDpqf3EvXMCrwDIE4KEE7sSlnfCAzmBDKe+tYy9RByq6Om72iY7O95IfREFaccmg6q8cCj6Wje3ZJfRYb7gVlT1L84l9EKmBjYUi2ZoBwpauD2FiR9ApQGLqBhndsCugwEvdsAKdcwj\\\\\\\/2frNTq1EliI8gckG4C\\\\\\\/bjeuTRwnvXMtducOppylk3Z4nqg2uL5hD9Ov86Excp85X+f1wkper3raWSNXzV7qC8IzRHw+vEzbGA1G7HqTs5RRqJNhl\\\u0022,\\\u0022iv\\\u0022:\\\u00220ZJjyQbkgtq0DBWyNY9d+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c62d5d361ad47b12118848d2723064993be2ce56202dce15749cdc5fb82b2c74\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Gauvd6YwRqaoMckC6Rh11h92x6hQRSfuue3xqevVnPAShPLiOyRQ3yspm+gkrqdsV8IlSaPNgRuszUBn9wIo3hMUurtYcq22sDUQ8REwXyY+4NdRtJOCqmDO0G7AUr0rXN4JUb+oggSqvzz8IQc8JVrcHh3YgM1vdmo+rUGRHMrHPKSUsgRYeveVVdHxbkESkoJUd5Q2Y2enFFlvIgp+xZAAAyTs0mg8gY\\\\\\\/dksNtW4AX0cE+FLWPivWs7a2pkQoCGSdAzc3TYk7hADlkROI6i5qRIvdziil0ZdEwdYB6D95X+d4OYC1lB2RRIzdvxgQPfhqgRdeBHC\\\\\\\/e53wYujt3HHlzLgtmxhMIDkeG33zEvm2H705W2RyFQ\\\\\\\/j3TsiX9T5azwcXfJX2nag6NskzafGCgs0ybdTIYLgcKriYfKTM\\\\\\\/DuLGqRrXczlXeJkj6BN0\\\\\\\/oHjNCQUqT7PxcFe3uZRUp5p9oduRb+eZ0ItmDzaADJ7yh4X0gmSD+bUaR8LRhcvxRyZtkGLZ0L8udbh3YH4Zkxd+LSv6+C90zbVtYjjrbUDm\\\\\\\/GtBnm+HaGDJsbbDzpZOEK\\\\\\\/g52AoWVFzVFD46un46mcHCutmBFmnmieoBhj12hqhUsWh78A2c16YT0uoBJd8IzU8fWE\\\\\\\/7oFHP9on3sn29fXNA3+UlrJtsLLZRBZnH+YVKBlZSXPetHECxFfoENJIGKTVhp\\\\\\\/rLOY7wKkatSlgtD29AhB\\\\\\\/QJJRjETR3DgHlMntXviElkSpnY3fbDz2YQwjUglmiw2f55FzmTvdWAPFBCjE1umsKtpgeIQuOaLni8SqC1wbTYiuqQDaILqcLKaPAxHC23mdrgomjInHms6pcthD9Jy9DlXIrIrl7D5PYzx18WoY8vrfZO8MavnQ3eZ5tQ2A5yrisqd9s041QP90A7dmx+eqvgt+tpGpTvBXWfdCI8TGV5Fhq1d5EZlwlQ5PwJaFrdzhHv2UCXsC7w5HS4aBCOYGt82wGXesWzZ5sGYrPVK6VKq8vE+ADp6qJFkyq1OOS1tnKg7wfy0hNlTu7TPwZFhtwm3zyC4f8xh73o3mqJTm52Zmx5ey461yK2u9\\\\\\\/liKSmDdIXVtyfU96xF21C6ubQkAAaeyWOGrbdUlKRzw\\\\\\\/h8aqgcSkXupUGF6cQdXkEgfCRTmaZ5WoQMgFtml8th0C6HXg2U7BXQ\\\\\\\/WPuvGKeF5137AcdPAg9mwFOhqILtueOVOT1zexhIPlTGX9An4tcdXiHCZqulzTCIGzwg1jJdCvVYncJrymXYqR4wfhgFSDR0PRGbV8MOCW7SNhAmdQbOSXcIvUFYsGoczd3E4B5I\\\\\\\/mxC0fmoM69B4f5ETg3iwkqhH0HJpAdwktNxHdDl2o5y0H69k1Hg4q0xaBKpaKROiG9BU\\\\\\\/N2baMldYLQ\\\\\\\/dkLKjY+E\\\\\\\/+Uy7RT3FOrFUePmFqg2V2iX2yb+fZIdSb+36iG4KajElr2hnYnhYiQMx6QCsRP855W\\\\\\\/FYTsMz56634FgRrR5B5c\\\\\\\/xBegnHHUY2nsrZKmEKpfzsy3WJKEQGWFUlj4eLzCScCQDY5xhCSpbmiDFGLA3Et1LSJfwpsLoMKAoXrLuHNE2\\\\\\\/yBvOuYWrMPAyvPdq7tcIPklSuUgiPekIuQNBkPbQfYoPE5HwPUp6pGU4CQj2qzMbjPjdOJd9C69XEfTNOP0+3vQKQfMsL9u\\\\\\\/c95nYuoJtLYJctSUKmpoPIWfi0OXxEZDmCi\\\\\\\/F9TDIzXh3d6H4j0hhipdCJyF5sZMa0k2A46vpm7EW1Alc00g6mYWeAMGIEdQdRX5KAzq8S1RsMG2FBeLtgwHzKHw\\\\\\\/Iv\\\\\\\/BDsr1Wv5Y+3XcO1fBls2s0hnuImPTq2ts0XWk+1xM3KeoniNICroUUY+pA7dlDkFTMAuHWm+ll5Lrh9SGI5kdqv\\\\\\\/t2EBuCa9uIEXKZX7LQUFIHPm4OeeuGtRx4bUCCHvcbW+7\\\\\\\/4yqoSXJtnA1j8hSRETyvij7pBVMKgWde6ZDCtp3P\\\\\\\/LXNuKo3m9kUG4lrKozfen6ztQT6SdBSIjwVER1r5Eka1YChZ\\\\\\\/DxXHFve69B8qV8Y5IpaaNv23y1fuAxrpghZfLGksPdraogPHd0if6uHFGsJerMCgg6Zf36WHHskmtyWyIW9IScECdX6ziYyN9Kmc1a5T8uJtW5O51RlK0xaaoy1BDpvWhX5q5xw80OnrH1Vy0WhUQ4yuT+FfFdw1u4clpQ8zr8qHAMZSKBzbR6nNHOeNyVnrUvECA7l3C2w5+SSANY0VelfZYnvgkaZp2PZUUD6KTbYcSsR0SRejOal4cO\\\\\\\/kM0F6if6HyHs9FD3+i4CRl1IHtERTps+kTMHc1lEeHO0wlgx8U5Z2vrIjxUAkK2JGZ3U\\\\\\\/arICeSdrKGbac9Qk6TB2Gowy4vacJ\\\\\\\/TDGvWrUErw4dO2mifcc\\\\\\\/miTMATa825oqkx9P+B2HPpaGylGyH7alKl\\\\\\\/Ky+Br8u\\\\\\\/72MOD1XMmzePf7vyHk\\\\\\\/dHjLG3My5KkAXCh4uIPNld\\\\\\\/bFjoMPi\\\\\\\/Q61KqT4N16hExZIvsrMLyU9JDro\\\\\\\/2mDrCPSn8aKa\\\\\\\/0fphQuujDVAjdA7QCa3JrlUV8Bte0N+nr7CMQhicOqsaLZ1gOmsuG3hoB4ZQHJMI+ucqvFlI\\\\\\\/vG7ZHqxP3I4ojtkqNJMUt1EiBD\\\\\\\/Krt6xcwquG0nqsh7T+m3qUnMcpN8mfXKSTdVLKjXRCtcfValK1JCVEdInuKYLQA5oMGLFEBRbcEmPeV6A82KawfFsVwUJuDJB5+CVRIsRXUVjNbOlLYcB5QyYiO2DLubL+eowtR9VLArzapBrHnIyYmHXQBcX24ZkyKizqptyaeB64DuRUMAS\\\\\\\/JBfNwlQs8JWiYFoCt+4Gj0XtR\\\\\\\/ThO+nYCds4JnL6lqAHV2zkAcN8a8Oej\\\\\\\/9QT7JPslqKCtsjbJVRjdgAmxVWwzyAfbDapxl+Gx06drBvmgpC+HbXbvuKI3e51J1rgdIPlTeNXkEGmrugnBYnk3t6\\\\\\\/XQTQbpIZzweb\\\\\\\/dz2q0xs23o+3n2UdiozCBvSS9+Seb2ThBEqAalbwcHyLP4H9\\\\\\\/PZn7t2hHiLZ1TyBw5h7+pxoourrtnHeb\\\\\\\/AHOgRwdqKAT4lnDJYlqa4jxtgQFVA2HhVA\\\\\\\/6vBO7Vo+qnlITqM1xB+W3vhXM5PoLIRT8c\\\\\\\/nscYprsoyfZsmqJmahxMeZQMQO4P9ENx0VHS7ZPqIBpuK\\\\\\\/0LiaVE54f9axITXu6t96JZdqtkRgKreUQCXjghUnMF4lwF4a6EUamw\\\\\\\/Sd\\\\\\\/dQ3DjcJuur1gcx52ONW6iPKcJsUSAoRsngCcatjum73ANOUHcbtC0viRzRzPWJOfDTANtQ8h8VRsexJHFNclDgVfPH453ICuXyiZnJtKd1BMJ1j9V5wBWxtsqkWb\\\\\\\/v\\\\\\\/c5txVShq9JFUf\\\\\\\/vB5UK\\\\\\\/PU1KiuF2SnFqnLvx+QmQPxdzivEdTnyuKf1EHP\\\\\\\/Ku72TcIduNrMLPkQesq\\\\\\\/eVHpm1713hlB8tMHnesoaT+PBkwJW2GWMfH84pwiXz3Ng7pYWkL0Is\\\\\\\/VD7w+NyZwB0116jxhg2FOYCf4Y5G1qdWKig7GJV6QuDGl2fJpDAr7Gq9aXg0v1OEPSBhB4yVyv6QqtllIz+bDhQCpxYEjRwIxrkCJGzUemuBtmTHsO2z7gtVdKhpbfQdTxPdXJTfqODDZB7f+n5tb95OUXhMBeqAl5htu+FG9GuPEWweeDX+aQAYrOBDxjl4XKf9cL+GeI\\\\\\\/QeSgTXYQ4LIuaXcFBsWv7WvqJ1truPf5O0oqilPQIL9IpzvrQ2df6D0Zpce4uHbgjlRodkDOdIHR4rY4tSnkDpmGqm+4Xg3ucA+i9U7MNI+5KplKirM4ISMu+MBsBEkWYydCB7mlNZ8fu8whoPOvEqjOsAZts+PkceiKMCNrb5dgMu37r3hLIWhOgCiRhK1JcQrQZl6WWVj9jIIFafbGL3mxZLEEY5HiStgSPCaez2j4xNP+ogI7Xhgid\\\\\\\/TwtFqcs\\\\\\\/aL5ywL0nNVKxWNFfWEm1gbzuCwoVyI4rqLZKlu4vaEbH5WlSZl2XaA0psGCFYrbvFjLBXnJrR4GFD1wYtnBU0pKPk8GP0PnGYmD77bIKS1Q3gNxOjNOdjNCVGQPebdh4l8wE21XZ6cRVhDRjqey40en9zC3DWQ5EGaSs3U0puBqb23xd+3C7pRb3D5N9i0onI5aaFuEPbdqM0Y36C73tZjacNsSRXqyOIrXaSDjxffYn\\\\\\\/7lwYM5ynDIDIjwnpsaTcVxp+VdWeucdtvpy9UyJKuVMrKPbXYO8wpUxC6GAuuxojYDcfkwEaZ5SndlYdC9BdKUIr7UJc5EpRfwCp2AFC3AZy56TkbswlH2T5JGt+0dTj91FjO0wqO6uNlsFrrfeWjPgrvpYGY+XKy4nnwR2A+kkYTpnX8p0w80ctrRTxaa5NXwwPdv9bl\\\\\\\/W+FNm4r6EWSB4Xano9JaVHDtWVPUPCBNnoytj27s3\\\\\\\/L2PvuImm5Sl\\\\\\\/O4o7uo5nbGrrPqiG0vsfTZdleb1lBAmwc1RoonuFdzPiJoiqD7LqoFsDHo\\\\\\\/Z\\\\\\\/vuvUGl+GLqo\\\\\\\/Z2fSQD7HNss+Sfg7WUjeiS14wkEe0g7QuEMFWTWQl4AAHihjWdiS+P5LW7xbzW6v8T0lwYmSkEprktGmaX7Llsn8IvKjpXph64HO1Dm7h7qSufCCkz4sMRC+2kyDsb4Ucr5kL4UCwbbHg86Y2iItNqtFMzJB943Sx3YXTklTQDsjUtE5gx4BqL0oPCmONQRe6\\\\\\\/x3usd788rOFDtjFMj4NpSkeoHcarsnYtMPVfqwSb59wOZs0zStE8sAGivVHmuNMj8RW0EpnGVQuhrD\\\\\\\/zO7pf8uJK3w8qI+FBOR1LXmYKz6qAramBJl0u5F7CxZQM490uY+fP\\\\\\\/9Kk+m+0VJRMTU26\\\\\\\/xxOBCt+Elx0WD9xcMg+\\\\\\\/NGesgTmF9E36n8nJ+rZBCNGtgz2rQAlg6HZWYY3zAh1K\\\\\\\/NjAe5CBErWE4Ft7sotXiD7fYFVQWI\\\\\\\/LSdii\\\\\\\/JuTfglsKdd0a5FgDMLwgfzTMYjfnFqDYbw22o6pHltPJChtv32kfv6s4\\\\\\\/qzyL96A\\\\\\\/J42O7Ke7ZrnOcgEkEwOOEDpIGvL4L4ECz2hwxi5KqFNn59S5xOwuG6fTLaoFO8kiOxVA\\\\\\\/nJQCEQYxS2stxHFGMjy0ANhSeIg13aOaF1og3NhX67inIzX4UFGAtUNYmnQLORUeNDODDxO03EHRory68jzmcPmq7wuSiJbxilN2UdUbvHLBBgN2viyqz1RClirZwFcrUi7PGc\\\\\\\/6trGFr3lHIpZwGMVZvX3qrk32JodYQKT8oejh9VKn2fdQNDsQHQ0hKXlN8p8fOIOXn+PXRgdxPqW4v6DUiax38Xcj6Wx\\\\\\\/0DCa85ZADAERZG5+eDx9VnNPd5BrCPkTkLUh4UbptRDSxgFJmxH2XqFMNtSaTpue4egRV3czO0tActPssF4jE5+Kl7itb5Z\\\\\\\/WmiDKYEDCpb2H24PD\\\\\\\/2FgdGOBmft3rd+jM3zfiQPw0TLSJo1Z9ZUbrlr5\\\\\\\/K7XDtzioP10fdqUsADSB7f5Qun0Zj8I+U9Th9qxULdu\\\\\\\/VQgrUFWLGDKvNyoQBNgwiq7qiKUj9DIR4teAhxEcmEHUVsAWc48lh0hU7xfiSJdYMDi8TvFW1v\\\\\\\/v4Y\\\\\\\/1ANWSLi0vF4jFhHYUmckPgRYrGnQNvyKIsglX76c8rwbFXzOjT\\\\\\\/g7BW\\\\\\\/JJV\\\\\\\/HmxMywMDqAxtnVNpWZPMz75LHycyUh2dN8k16VA9AgeAMbildsACqmQYmADBtNGc6AVoAHyqINCki1g0qQjgWQO7PtiU2clarUTJ12mFqrlKXM\\\\\\\/2s3uhCLffI3l8qQ1fNJyCydakEZ7xB7zYBD4KFz4ol1BCCHjWNicG2bemL23E52Ev6\\\\\\\/TtLevUErJZbhj6fXPNiwe7Y6zf5pLob4Pr\\\\\\\/4+OqKmEraAHWk6h844Ndjfd1oQutmUAf5elIu9jA7MvVpnipOGpqFk2IYWttL3YqDe3tMIXLetHWKeeMmioAAiyCXjWJHDCxaV5AzUAq6Qamxx\\\\\\\/c67RaCMlWeDt+U3vL0h7\\\\\\\/pvVwpsNKoh\\\\\\\/MKA0z1hYqzAwQEiZue3pEXOTNwwxuqMHCAqlZ+ZQ7C0pOSzziTbCElQWZHnlLQDTVCocZbi2bL72yDBLvUXSKp45f1rGtwhkLVi\\\\\\\/B56ug7YggSE6etPngs2Y9vzQnioj2OYnXb83jG3\\\\\\\/P27qm9QOGQclDxufumRLXFYCJuekfm\\\\\\\/fqmKtRpe1LgAF89s4PC\\\\\\\/3\\\\\\\/sElWnP1WfJlebx4xh3RkeTCWrR\\\\\\\/e6g5qk+Fkh8y3YgKjUav4AEDJdGMfgjdvJkbFbxzg94nHME3miWInipyW90bznPARyATl0JiDSQ0xuLJAW4SrqayEp6\\\\\\\/U5+rvfZx+lParDeyTVY+6BnTzkZ10CI0ke9W2e3tmChRNALHLrcmwLpOXgI3\\\\\\\/6BSgd24yHVxL9VV6uolHJRFHk4tkFAnaws0iHEwev07dXuq4dTu3yT6K\\\\\\\/edw7JnRqLl5intNr1H3Mfs9glSOJir7agW4f3NTlreSmnJNkDqG31Gd7p6TAypw31l+HHg9lSfkTeScDfq9hUnv9lMHnRoqeEAGJVb73YyE55H0X\\\\\\\/lIL+94hf4UVSO7i3k4S32\\\\\\\/s9Nbw7rsn8Rfj4IuT5wtlro67UG6PTkNbkABFhbH8QIRBJa6VJ4x8L2wjmrXAkwrxJk+I+kcxanZZf6OjA+R5PwFIRglzspIvE0qvz49peRU9hKJmbfvnyOFLemH3P6gshzRfouJR0Y2crJ4ut\\\\\\\/d8P1pF5Q2qlpzQHuWHgX26pd52SMT22QOhNblzZQYl2Y\\\\\\\/jBrlE6BZUhYz5RTOP7ayPbd+R57sl5l6bm5Q2Kf6Ng4YMTp760pkHXcCla1KBp7m0mLXVIoqh2+WV00F4hfEoxswabpK\\\\\\\/QTc13ZSbn5K0bvgS38XiwbEoB1Ju6LpIlhCluPlu8r8g87+UJjCET3YKGk7vAF9T8\\\\\\\/OKtsHVIuX9DKhNzXKcLs93lY1XccqEFfHjKdDhTYKPY6bXaY+QK5acKgvVnGx8RpH38AeHIJUm0U344vrkyJ43ngUnqoYK4wTJQJdVPdNuLPcX65VNWm07RovKGOklqzEry+MXcbxn4rG3uRM+nAYT2qtErPdYmThn3TcMSz+WnQXN1uKKxfzLUet+COp91U674QxOySAnMv+MgcOjGDr6RqRC9C+2CusHvCpC4M0FdNRp7DYXUVf+CsxFmGo+Nl3e6JITDBd1FfwBIRASdDjBEQhL1O+XoaDS3QX5ZDM7rcndpSeduoPVzWKMbxKXYakN8VQ\\\\\\\/YT92zG1ENVLD7MNv3Ipk4wDt5p8qk8sAH7QyAyeFP6WdZPa2imymCfd6UEKB4lF1fQ9lTPMhcE2T3ynVcvOo66jX7nn\\\\\\\/LPKfbKgF6eXYOi6Lky0mrC3vy1GtXg5Z7mHNY2U53ouw0ZKffPi0kKnjaldDPRSWFXQM0cS3hCg7aMdGIZGLyG2DJaQBa84RzVR04aM8AE7O2g80Ca3KB5BFxTeIiovd71gdfvBmDimyQqnBjftlzD+QgEP93SFNs53dlZJ\\\\\\\/jjEnvzd00nTUKVrTgG+n18ar9gQgeRHAEqUsNLLleivJnQdou+bI9r2cIYuENxG2L70pxZQ5qUmxmNMLw\\\\\\\/aoKVqR5Cy+pTqZtSBaqB2vWy\\\\\\\/tA4u\\\\\\\/eNs39557R5I26bsUlXtu0cejY7rjGmgIibSgPfwIqKfJ36ozvOyIRe1hBJ9RjZJMJca36XqEUD5na8x9qcdAJT2co66qoTEv7CXPk9M7kxq3b+C2+yDpZj3ri6cRD2dv9TNaLezDTpKbUIMEU+4bx7NCrmYt27Gu\\\\\\\/36J4mdjTXV+refuNHTK5zYvuHT63d+5jd7QvM5XvnOAzGvlKZaeWvsZc1Qoach4Mmyg2OyRGn+nSqxs6evRa6kcx4wknuD\\\\\\\/VgxPRBxVfW+n9fKATXle6hZywqkm9RbxeDldO2J7\\\\\\\/BDM8+QLhK0ezeYZYnIJI0tRa+VSE+BRRnqWjUYMhrz6wFYaDRDhlIQUQGjnhJcpS0T208MfqkuMrhXbFhogHChIKtBogoP4Fds+LuHDWVQLwYnLwdS0GbhF3JGiQgAdvv+Mgt2cZy8MyFvxav9ZY60owY7aFDLTYOLY9DH7c+84zd2St8vHWK01tHdvP8C3C+Jq8qMQYgj+Taae7\\\\\\\/g3jSzQjyhiGUaAqD9h+dmy4\\\\\\\/d5LayNXApxcyQ6b+ZuSXo+2URxYsa4kH1cjSCYlV2vNhk7CknEzTIqEqK209KYjphsbjURlAPKgOtGbxMNTleQVmzG0pDHsdwClM76bsLnR6sDEv8gI80Ul61uB2sjt9oufP2tqJA1BhIrQAw+y\\\\\\\/VWWjf4t3pCaVzV0PNeOWrJXJ+PyFal9ZH6WhvXe6aOn7YfGlzGheivhp0Sf2d8gvH8KQfe4K2nieO+Po5Z\\\\\\\/ctjWL6X8H3GpRdBGsVB69EfhdDQ51sVfVcvnIkvGA22yGMvq17hSWxKeZ2DHLCHuhuHGvTT5R+pa+HpeJRlWMtNiKuex5fqVkmZpQZ8oaOs5SfCnN+nhbSLkT+YNAhoA8USVNomHLUGMjpxat7vQT6tnJ9VrqxE9KqIBJLu2sTyQ+hkvmnNtDlSlEHMCU9Cv\\\\\\\/8DHrmrfdL\\\\\\\/nWl4yfLr8oEeFweDmtiQDXjMLUj3zcT4pvKtxQDhOg2+HMazK\\\\\\\/k9kamt1\\\\\\\/MxVBLT1XcdeLiBfJHDuGqWWW2VEYuEGdNWVGSxSUmPc\\\\\\\/4vuU03SQ5yaJ7eHqSdcE6jMaB8DeGD4vLgItNuEz\\\\\\\/BdS39TgUfRGtDxfKn4WrEKFjKnjw1k3l\\\\\\\/NVrh6uhJFBGFrlszm1cquj1Pdlu6nuzC4Y1lVW3A1HFK1BzqzFIvuRuX+Cb5dSQ8Hu\\\\\\\/TjPt512PPWWHq5I+OuezlCCuq984f4IySDP+IjPOd29l0LyuSdJDKjS\\\\\\\/U71OqYY3tIJtQhJ+1p\\\\\\\/Z08JOlR2m3aJ8dFQzjzy25vKSO4DwKKbA+AIiq4bfeGNQMYt+MAeBu\\\\\\\/jt4KT4HU4z54lNuteUWwCIdMlkyMeINY5YYKKp+or9ypJYofIflYjD7AKl8iDp2lZ5nRmrKcbi3zv39oPpxRKsetMbJI08UBR6t\\\\\\\/fEPZb3y5EZ51h9UafQBXxZssqHZUmhCRGqRXnjNv\\\\\\\/wE4OFD+thcZ1SivaPlddqbcWNwg5tvanlLeDYeIYSqyozycfKqvVs5lX43gY5Vykhtt7UvrSnJycV0FSzTfu2MoP+JigoHgSJ9MGbIMdOcuSEg7\\\\\\\/WzgCRHrrlqYi5jZWUtWhQyIg9cWrC9DxOtfBTksof3I7Xsj1XevE4KFLr7Gx5Zt8ZOPuLfEmkAW4TqNHl7MxWt\\\\\\\/nWJSZ0Xr6scY4QmbW3hqt2ji5mn2OJMncSLEmSi+O14ekU8oB5JGI+tBegk1WcYPJxPU4hRKZapmJRkttuqpV7IlTm0Sbk3KujeuqVCyRD2EuAS2reqQD2b89Nq1ViJe67fApByYl1tektipSM77g2VR61N1ZMQHbjIqTEa4vWCD\\\\\\\/rBGApIihSVEUdh26n11xBMEpwg0W4QmZTC4cRCFgCAEm9LK53rYXy7ZXw5IY36sWV3WvemV254DbGReqs3vVt\\\\\\\/mQ1KTt9otTDcJihDn2SREbvVBqDMkRcPXde1RMugTulyuwkJ82C144PRTiBap38N9m6ChoeJmDf2ck0cLBJ4OKkgx4bqL73G3vhbEpBa2rCoIuAEbL+otZ++13wl60ZDQsXMLg\\\\\\\/p2sjmT9P7vnoHzbomHMPBC7bSUR2QLlWMDzXeqegGgB\\\\\\\/3xq0OKLmGQOro3Fq8FY++XoCA1ibOLW0o7eu7KwB3EzC2KCEcErc\\\\\\\/Tw4qnYt1dz+Mvu1teXAvCErjEF\\\\\\\/lAz1XqV90xsxrGUE\\\\\\\/hfwfR589FOX\\\\\\\/fz6Pd\\\\\\\/hEyfXEePT+Ycae7MYuNBkiBGNitlAa1fkUGqtnoePknhQe\\\\\\\/oDb3y8TEWKbjlstpWmFT5+1BD0pA1qZG275CstDV3Me6MZjnI5BQE9+Xk3ugioqFr\\\\\\\/G+Pw25S8B1hvhDdKWyLnoERfbO0cb7lNP9tM72Pk357hcZx7\\\\\\\/EaEfBEQRX8UNcPSfxbWgRSv9fepylZFMbXBQcswLTNGgpAd782YdmHmTVMhvr\\\\\\\/mGhnxTjipwxszaI3WMGlpM0Lji60v9\\\\\\\/4YqTHj5ZX7Bqye2K38pw3\\\\\\\/cMcaO37GukExV04Q\\\\\\\/Z0BMsdElEzLAXjwMCmvMsaaSe6Q6ZUx1N+iOkmx944FshyAgw3+VmXt\\\\\\\/sgtCOYzd4Eu4eyAs3AgtdY2tO6lzAukkFN3tKR50rGJL\\\\\\\/4s8NZtFxwC0Z4PiVrbB9p2RqwYYJmDkwUP+sQ0YMeijLUKAJ3oK91woUBPtEBAaNcGs8fFLTASUtjzWAiTuKLgBMt8ujOHllHwmDckuBhhbyu1tjNMiNm7MGvENHVUIc58rWR1CYPTsi6pxG5fGW8zGa3c5wqWvSfLzwtb04xNuITIAd83+0GLfwYwOzuyHS6UOdpEol0Y9iKllt+k7LvKFUI+A9WIt5vEV2o93QGT7Hp8HP0naXbf60SMlXYAtSnOW7q0UQYFQ0\\\\\\\/9ZGpW+jemgutFx8aZ6e2hNFBrG9oGFrzpefek4Dz1CGbsXEdgTddyIcDRIhxbae4v9qw6XxJrl2i4EQWZiNiaQ22lQDZoeFzhIf64OT08lNEJ\\\\\\\/anGI3cl\\\\\\\/hiiFiusewEff9Jr5T\\\\\\\/re2t4OMZ4XYF0wM+\\\\\\\/yoNwv16nubR4+GiREbfzc8DPybyeb9efuGDRXz8PGXd7oyGrkfkcjWB1wlpfPgzp3gs8w6kE5nZ8iublPbYrrD9behB6h\\\\\\\/HuxO9Xtqlsamh3vCuQMWQyy+pQi4u+yuEnGAN4ZGjzYnkbYL4O87owc8L6FQACgA5SufhFI4qTk8ChUUT+8nS9CqTOxRTEzfSyIkYgVl4qJCX1F6yKRc2t7jijxNbizXvTbXcz9+eXi13Ke5aVV29IFlUVwmZfIu3W2NpG0jiNS3tlTzToccDMaarO9Tv2dGZSWm1m\\\\\\\/Yk0CU\\\\\\\/iLoRroAQkAerZ2EBXVUfL8VCAqBt\\\\\\\/gHQx7Ax9ZqXPzDOWRucpUAdoQimJk0MyHK\\\\\\\/QcDZaAqK74pk2aPupI2PB4kIbi6vFgYp0iKI4AkDvIPfl\\\\\\\/66vTuHDeDx9inn\\\\\\\/o1ONvTPuhvMCYqr70KJPluXuAST6s2Km+jNKJnxgyYWBN+iJwPzXtx\\\\\\\/HVxRxcYtcn7RI2ou2mjhcTh8YoyBW0L9M\\\\\\\/c8BoYumVqRAKmZu9ZwKLvqBF7qk41jJpBFbbRbdsEcK5i2CzsMBHlgPV8RbPhpr+S5ZGe+2TRpVMk8uT8sv\\\\\\\/jxqSGgPNGbAvr+xUVwNdrpmmif5TnicfZMbU7P92E1Np1eSMQGJmb0nz2YXTYQJVXTDb2OCC+5w4m9Z9MDfZBB91+0QOMGZQ4S5OdBsQ6Rw\\\\\\\/0Xhr98pdYUCwWQM8gIr30g+\\\\\\\/yyderqNpamlkcg56FUBpiSo2qQinLhNyat9FaCgN42+5vyf6TSqRPnS6OJ7VLDD+rVBWVUwPp37rPAvTZlKwqGa9tjJrBeYmKjSqXaognPHH+mdpRlmv\\\\\\\/Nm1N8xd3zQjOh6zmteMicQrHueoZg0rVbki8jmT5xd6dXjQ0reB9tgcuTkP8Omfb4qDRP7lL+phunXIEmZBOGRP9FJHN2r70lx4GWXzApnJZ8froJVS9XMcfQtnCM8XbzXkFyhU+geLrIfwTpKw5xIvyVLhz5qFo4l3\\\\\\\/ugSHoOsHr\\\\\\\/DMuicOeOy7xq+UrNnoB9I2g6b+\\\\\\\/3ktEwd92Vkwp3V5Q9RuKsj2YwXiCnAaz2geyDsLq0fS1htsxKmE0CbQzklss0KThOkScaZdbhbvpTjKVtGLvpsfYGNOpc7zYt\\\\\\\/oOey2SxSGsK9xYX46PSGrcl9R5+AIGGbuFh7T\\\\\\\/QbJEI9T2RMguMTc+f2HNcdabvtRq9YdMJYSuhgjQ90PjFFngTelgFB78KwJRf\\\\\\\/KRtdVrQvw4mELYxJYbJU550\\\\\\\/vI2Wy64UFgiDEhJEOMdSM8nq0sLpm8IdId22uh0qjf2rt6JLXi9r4QtkfOme6eN5cnKReRi\\\\\\\/mF6HB1Y8vRSzVhTnZWZA30ixrbp\\\\\\\/WNZn00SrJdssX\\\\\\\/0bcxoTFWGaGwl6U0IkCUOLJtZ9Hp\\\\\\\/36gk6KR2ZmgbwMMi2nCjrSNWnzE6FkMtl3VYeUFPb+xKJRJivomhsFebo0hOCqudm8oRYiaN7uVLm70iPUANRjXgMvQ20YhY+4qm4n\\\\\\\/z9v11Ce1gmle\\\\\\\/UKK9Vm6qxMZeVC28hvLW\\\\\\\/mHCDE8UYaXcZek5J9FtutW\\\\\\\/um5vjaFONgN6KDcH+dp4WYC4BWOG6SokTOs0jALYkp+Y6d+MOvVwpY5kcOvKX\\\\\\\/AmvSGiJVRd5TNlk\\\\\\\/5KfH5JTqzxz\\\\\\\/x\\\\\\\/5N2vaJ3GFkskRutoxtbhCgnV7tpZn6NYl6UImvuadwMXAw+jl1Vqwsos3QHyzSUOCeYlkPPxM3Ac+asJhYbVrstZmE\\\\\\\/iZumFmuiLr2e1INiPog+Q1FwTMCUmwelZBVNUS6ce3fw+N\\\\\\\/ykXHV7tUM5YcyTTHKTj\\\\\\\/ymuJQFS7zbdha1xwGBV+dJsD9mvyzMjfdmryfprHGF7rYWSAqvJ21eYo39cEEpr6eWtIyEgpFezchaUItG7ZZY3nrgyVSHH7wPaSSo93556U+iPfEi8yHFxiYTgy93Xbz30fIshdYPSIoHryHj25ddlryN0iHu1oI0VHNWCq5phPmUdVzT8VMxlB9MalGpycLBDR2AhUWtOHttNkP7VADWKETpLv+8qI4eA8mS+\\\\\\\/Fh2aBeHZfqbgByV0ztiL2Tr8rbg0AMl0BxJM+RgtPqzP8NHVvZX7onklBp+w5Z3tHg9pDYbVDQc9KrBxY8QxgUzmBymJZGdKC0xX\\\\\\\/VMdNfuxMfvLpZx8n2oQYUkzyx1mYF\\\\\\\/52h52t248gWZLVhfjo7ZNtUB\\\\\\\/xwqK3WJB7rIW7PCOzVGtPvzgnSuQRYbfqzkSPXY64BnSDHwpeuLjWtqnMCR\\\\\\\/dV\\\\\\\/TmiKcXUBHsD8Um5CgPgh1UhHSPSnJXFc4OHp8oE+CkQBRFOuzmhmLQfBa\\\\\\\/Jlk4Jz2VRCrymPRXPDjJieklKCTtLUd8sdBParBgF+2LrQfR2FC2tgxAmvqhSHy0uaKJLiDbqoKX+Vc98fW4nJLSgY3CdAfEFsxnwru3xZQI8FTrviWOACq2iZg7yHH8mw03nfx5tVndL5EC4u0IoEF+5MNVxDTpxCL5dszBWGKgPzzNIniGbOrjrPKjyivXroFHJFktQHAFHkHM+QZEZSBkoH3YmQcWeC4+45YbfR\\\\\\\/yej4RN8frXQ70qI4rEBa9yWmrpnWvrioS5l6pMW0mTOeeb0M5kmuBgo6ML+rJ3FxuSFmBL8RyBbkudyTAyhLtWSdAFJoKHPxkmnoOms4Do+9kgR7myj7ap26j4MW1QLejIofpbXLvkU6iz8V5qQE8JfBWkkk9sEDXGfrl+LBtN3ircjFIzupLW12m1I4OAgsNUX0PsMSSKS2j4kzyKvsmTm+CnEevLu+1NxsnWU11mFUKNrco83aXY2zs5l9L9V\\\\\\\/tf8nshTboZK18VvxaZermfjkRqDvwwBNc3bdN40BGQSEVjNzisHWWRxepm+PCheWI1udji6t0BFYfL4zOFlqKdXmeswVoJSx\\\\\\\/OAX8D\\\\\\\/PL6WXf5\\\\\\\/sONPmHlr4OcmmY3xDMfzDFWdsDdn5JhVAcTgsTtvvfPHfSmcCVNTycDCpPOJkTn\\\\\\\/+NuMuMb9J8zXqR7vY7ArxzXSE3B3jsqKX5PkIXWf8uH9PreQNblMwLvlGOxw4bKh7IHF4ByuIS+TVAaBJ1kv\\\\\\\/ghrkqzOUEoHSo5zSoQZ1meMfRzpVC60w8UoGZkcALzsbDoT6fVXna+TOSttGggEVF8OJPESbrjIosiLvEPSAcx7L8MPxKLNEqRuF0xa+tmIGdUBwCKrF\\\\\\\/aBT++aK3L5qckFxa4RzP5pF+Sp6cdquWncGTuFiQ07qLfGdDBvZ5iDk8n+WnSoOgBjIxeyaqluz9pTIAn9am+XlVBo2AUNAlKTrylpmils8EW6RKh3wx\\\\\\\/\\\\\\\/o1ivwYrKJqNN\\\\\\\/rrJMcWdO4xQv7teqmL2OS44XIjc9m7bIPbndIvwXvey0Nys009wU88cQnfGUlYWY9ZSbOQGw9II8FbmmrWbESsABzhp9HpkbhATejiSSojvE792O\\\\\\\/q8T3DeHt3fntpJpA9HNtQR\\\\\\\/cGHvhxfKyFBVqTZvO3bDD+HR1sU\\\\\\\/pchFNzGu\\\\\\\/i3lBL2aCcOUMpHXJqHYgQV\\\\\\\/Aw95NI3\\\\\\\/u46XNdB3nDvGYF+D0\\\\\\\/cPX6pHl7uDISu4Jo7BqUnWrnyA18Fj1r2eV2yJrsHgZ\\\\\\\/F\\\\\\\/b6HFLyPxhsNQ3NRgORTNki9pLFl3RPYytkhEzmjNcaOwiDEET4rWigrTmiOYJ8Dz2p6OlpZvhqx4YXYox5xZNZZ4DESi7pNnAgG6ihqhIuvS\\\\\\\/NdQoC2k0hg78uvm3BSdT9YeIK5xnOw7xGPv4suU7jznGj8IBQgB+7zB4oafy+V1rTtDSDGGfE9NkmyhFcJ2kbZAY\\\\\\\/Mcx31h\\\\\\\/k26n3X89xtk+fHzTznIZ\\\\\\\/SLzmUltx6XedayuZW6t4OwgEpJrGvDdcPALXAN+tr51xFS+z6mGBBa1aIY38zyvGfQ8Y6M3DrdqdrSwJoNE2G5kDyR28KorFT0rF0v92Wvpu5XCMs037q9rAqA3F6VYY9LmpWKNQ1TdM5MAaVzLKCCH2CyJYhJaMWPSyb\\\\\\\/EwtvgrbxrgSAf18PYYyYYUbm075h50ltATO5cK9B9KmKH8nJMjgjfYQtUQsKvKmK7tM71nr\\\\\\\/tvi4e8gVMk8L8aoMoSiThfP5OpoWA2EKzJgbc905E04zsEaLc04Do6gmzXxCx9A\\\\\\\/T5GGP\\\\\\\/hSQUPiSVFwtdH7DIfk5wQmCc\\\\\\\/5mK+YfBOdsMVcdb5wfAsG3aHrFHlVPy7ZvLPBIOvITCby\\\\\\\/yELhXKe8zF1nD7Oq6G960b7mLgF8lyNIBOJHWMLB841GN6MX0e2X64z2hnTY33oulsWCO3sAyQXV67VLa7GKg9nFEWtSr9jCAoMjCFLpOfxqlfX19WdcEQNts5knUXc\\\\\\\/HvP6uruLgx5olw\\\\\\\/pwXJqMlD4FuODpGqixivmJDCAhKHO22UHe0pDnHatYUlUAl+HWvUW2po3Y3RQ2gxv\\\\\\\/ggJjHDbL98pJTZa3PtEIAF8AoUR9HtKNqV1qPwiXKBem6\\\\\\\/77OmuXUW+dIDE+F05KP3W8OZAbF9We2szWJ21v6YEgjz1rxOfn1EpHdj\\\\\\\/8J4WreSn1iAbrSskQ8bVqH9tqH\\\\\\\/XXcP+0f4XK2guKhzL8aTL\\\\\\\/kJ6tGC23xmm97qnYve7ZDyvKLRaTks9is2WI2c5jlunG143AuLxyY\\\\\\\/ywQIixfMRhYCA+uDKkQ9T5oZ\\\\\\\/SOCcMNzWPk1XekIttVgwiM6BfgN+uaBDM9+YK8DEdmcc2UvrzzFbfuqJkTCTDNl50Aacby+NiDGd9YKHph337+OmMYTqgoLITWRMVCITzd2y++knphJjXbo9FeQXbMoLb0ehofSeRNX+pLHg7gonFc7Lt3clZonC1WelHTnSUd3B38AGFNnae1qwX21MW\\\\\\\/0pZabAFayqBMiSKa6IFwn8gcrFxttKVo27OMYrHSrIhd0kLRfzGeOAFs1yJmksmSxvLy55a1kU7bRgeBTU\\\\\\\/Pb9RIWCLC7vXqQp9QG0u7o4xliOaQKCY8yMvwmV+0Z7z7+vg0ZgYwP\\\\\\\/Pwcj+PytClBNXoO4lNiv8nX9JoN+uOBtP70pgdT5oz4Yh\\\\\\\/951nOTkeMov7WwK7IM+xrYCnVCl2kEprnAD1IDHqfxLhBciQahAqC57661vvMjaEiv4BsHnvBqdbrlFSfYqWPPl0IDHTGralABoy9GLz9Fh4w3+0sAVqvpu\\\\\\\/9KyvwNRoJcDdVv\\\\\\\/oS7zIf2NON9AVFYqZLxDUUaMlegTs5bz2safwy2W4X8C0dnJv9Q29dkwU\\\\\\\/F5ADTO3X3j3A\\\\\\\/b8tl\\\\\\\/CvLC8IhL9BxE\\\\\\\/6mupue167l\\\\\\\/bFnUxJsklMt2OJS7Sr\\\\\\\/YeTg5kuzL\\\\\\\/8bKcl0Vekd2AwqvDwM4A8pD+lIo7yLUwwgDA9IEyfroVVW9RXeCavDa85ZNZvI6nySSyx8X+GO4DF65gVsh+gJ7BYqUDpPLZ0g0avYHIrGS7jjTdsiS93gwzcc\\\\\\\/cJ8trR5tvjTwWKButqwehoMuf6fFDUlSxfigkyinrs8sOfDD+YtH\\\\\\\/agL4K5tXew7k6QKqdYLmosuZXSiJBVj0Zyw+ctRTMX61ao94vydy9uMwz6hEbhcrKsI1VW+rIIkWSmTui\\\\\\\/Zl\\\\\\\/K1LmRZBkMxQ+M7JUN5z+eDW8kSteB9BuzBff2CoMOrZSCNwht4zD6F0JhBabO1fVCm4UBLMXoBba9tMYIx8QGwnjGQLha0VovGECnZwJL8vizoVeJAeBVGIZuV10KZLTcYeIZD5v2Vkld1FifMxV1Uy4xTYaknJiyhlfA0Jg8g4hPinMHHJIF0ac3sH+FajAGf52bZIN1CdzXG0o\\\\\\\/D2F8cZKlprWis9Dq4WyutM7ah2QqSWans2v7kJ2h+BypCr+4ATnwkU4Yl1qj0lvNOH6xWNLeCls9VJXE83lZhbiognPV2L6RPcklm4yPKmZAkNokKxKOQklp+CTjd0Tg1DvzV18SN7HPwuv3CDEkjoU34u+J6So56svgpZCOFGrMQd9rInjzQoRjpSo5Mq6O6pbkIQiusI5NkKL4NABl9qiRqDU2irkgPmaJJ3MQUzQLRPqEK1kwWZ10P0YJVDEJF57hVimfyQD3Zm6of3aa4Ljg\\\\\\\/OxL1BPgWTHdn7KXGccnTVKnuz7Ohwj5iC68+8OTeDUp0v+Lt0ZLwP0oNqvYnIayy1Nw2Y5bSHLR+am5JRq2MBcgYQGpgmqdwG+Mlybn248CFpzOkIULY8y5+UFVMExIXzkdPB20BYPTCp5Y1QgC+2RZPvdjXotLrwDwbL6o5BOKTTeLfuCf4zDNrQt2ZM+6A5YaZ+T61dXTqOODQfJkn+REPOftRHZI0wI+LlT\\\\\\\/MRelVcrz4p9obwiY7x3XRVdTJBOZVKVZn0N5qC48HThq\\\\\\\/rBmD4WB3OH1q3MrWnpQYwfT0wqjvThWTof+5\\\\\\\/DUiG7mjb4Eoz8fd870f95vDY7anfRgaSJqdE3ibw9jC5pGv8l1cUp+ca3trCIEV45zsECTxSXhDcEB9Z+QCAUTaQe82fsS2TeD+IJqpdG7TWBjaif3+U7z6680VLOsc0PiaiBEXLhP3U8I6VQKdDkkK45ZiyMLpLa+roIwnoq4jMz4tehxpQV1g0fwMiqiFzxPjawxLzvcFh7GObEcOcxpAcqFU3+OSmM1lcKed6IOg+CVOxV1V9opVFLPMBl9cJjEa6qTFuXWZWu688HbWlkaLgEwTk6QXJ+bVTMhqI0MQmo9ZSvkm4wsxabMfyj5J5p0JCoUp24eyk0L\\\\\\\/XZQtm\\\\\\\/6d6VWY99344IHVX7RfF64moEvrhBbww5WmHDJypMTVGhu4\\\\\\\/EQW4RXyusq9oqUOdtL7i3DYKZDQXyXcNiJXpq4Rw5KngR3kCxN\\\\\\\/qFd9shtqZhKIn3a9H6ZpEPgp69CHWQhXOHRlUjmeGUwFOC1z3aU0x+pjmigISMt2FxggS4hh0sMtPn4DM70rRZIk7yamSZOquVnZE8XU084KdgrH3hG10ORpBDlP4XZo5m2we9YNFcWsTbPPTTOnkdnBRA8\\\\\\\/UK6FmYL\\\\\\\/Q+zRkDuNFZ0FMn+BRGiG3\\\\\\\/jaLUM+qfq9+lCiDz\\\\\\\/T2Fm7mPXC7BjJIwJu4k4\\\\\\\/u3eIvP79jEkGnvLntsKCFlQoV0L4gJQfjC70nu9xr3YnBE\\\\\\\/BRqNG+V8IF0A9wETAESI4LbrdnjfLM0yuKTVpfCfle743YubSCS8CeEu5eEF7sSXhS5gJfM1J6ISXvRddd0tWTRORcESoa5DOP2WuvIyZGtWSDU9yrE1g9AO1DKn4fwvPQcKu2z0zdHxVhuosX1QyOO\\\\\\\/hIwfJDCJGPDr4+AXbO\\\\\\\/foUoXng79l2U\\\\\\\/EE9SXmsEhCqgJRhc1BidoAtmvFHEw1qkycJgDz5pcZGcas6\\\\\\\/QNJAKbDOCnP7scjiaTDPuj6liMN+NRZ4tPN9eKjzM3VA9qZCOUaV7wPDOawzjnUP26RpLt\\\\\\\/IwZKuzJAhm6H4VzAMDIt3K6MzakMye1ISPE7rdREliwmU2thUO7GwzBxhSl67P9ARIOGZrPh1\\\\\\\/NSD\\\\\\\/uoZRSKfnyZoWnucN91vuYFBLycq0\\\\\\\/XcYcPnkw7eu2PQV8\\\\\\\/I\\\\\\\/O5nL5\\\\\\\/I7sQy0Z7o\\\\\\\/JL2ACcKD2ST7c+nRcnVLsNWiPN1rExkSyLbnumrslDavhAKSpHjwYYPSxas\\\\\\\/1OaSBp86SPLTcR2y1IR25dOKCg+fj8\\\\\\\/z59YZNIa8FJmEMaGTP76rq\\\\\\\/rVHlDpWAI9iJvoNJkmedkO++TU1RI1EAAgdNmL7IpLDwumcX7XVKggO2ziJX5TwIX9571\\\\\\\/q+Ph5nRzb+LOy1fz0DOQxrb1TepwyhzAuF0SnnLi8oSKCrAj9HWugTqn\\\\\\\/WvkqYjvtvrDWmvwOt4MegWayqdCwGcXEqPONT6KX\\\\\\\/MY4z2NzJsaS517wVN0N7K6IG\\\\\\\/a0\\\\\\\/NxLOLSWrvGH0bpaA0uPWenpNB8keDGvys4u\\\\\\\/K7wIUOmL8UIuX8mzoA0KtPjlEGStbsxDmb+o\\\\\\\/oB\\\\\\\/\\\\\\\/ShEQ7DNRHI\\\\\\\/eIH\\\\\\\/NsO2hN1Jt7RYNMRwLQVHkPUeh911KZquJk1wp0KOsJvinGPc986S8BLPyA9Glxz3MZpjWdjNvKMRjvrQwDZIgZsQwNEKOb7HzHp9zueafGOO0ZU0hOdApwO1Rc4rP+5e3V8mcYuJkx2BX2BFRI7aabp8JH9IamQfv1b0VMJWd\\\\\\\/ddlZAfzrpM6WAz4zfURYRo0F983OJhg85KdK9aTxcShGtFPWwd45YqSJkB7R3pq1aho4YSE\\\\\\\/jn+qMDmf+l4k+1BUrAJgW1ZU9LPFU+zM+MFJUwUexLYUf1ryusBhFfKjArq7J7dOUvcbJQlRC+4ljRo4\\\\\\\/wzyhgh9MQS6szvBMUqYnydemc4zjYk0t0pl7IH1+PaCbLxrelM6cAwpHqlgsCIN1KNb49hnQs8ZwtrmDocn73yemXhdl7P+gsDQnkfUkyq4v0mrueZOJ8yT2bsYevZEn1DG3\\\\\\\/W8u6MOcIE3BygvAKzsPhYWyR\\\\\\\/Fd3Kob4iBIK5xi3qHj\\\\\\\/QX7S+WKuuKXRI+CLlR2Wkji8ueRd+sRWCEDnhoJScDsmrFz4QBBdP2EqN+Yw45cx8rx1cotyBH4OwzZU4TJAYSzLW4TEznbYronRCMGC2JjA0gyvLcVc2reYoKIjZ5DsOgcwEjsyq\\\\\\\/hvxUd2GdBj\\\\\\\/vEr8kiAHGdPJHZdGu+26Z9qQDDj44bQoGlTUz\\\\\\\/qf0Uy7DW9JWWaM0a5daDr\\\\\\\/ZkD83ea8fQBt+\\\\\\\/QLdRMOau4ctK5YKXjvtoTc9k1Q1RPW5VSZNjGiG4psOBVScbBoqkFexAj2SBq77VcWQgvfKW7rCuctS1+pGp7nPTyWPLXcnAj7DMJBfUReFx8IwLcvh1aC+jzaRjOl8Ytq3jqQJUsJMJ1RVeyo1YwspfO4\\\\\\\/q0st+pa3Zkw5IewlCY3RgFt4kl98\\\\\\\/tfklUPD2tuC6ZTHyFFL2IWfRR99SjUD719yXsdCQnv3BMmlvC6ctq\\\\\\\/lGd9ji7pClnjgQHmil6XHC6vD4LHUo8TZ7wUNYFkbXf21I2EX3SrdIr3mq14zHdImKd\\\\\\\/Yf72raQJYedrKZTepRJu31u8jGNDUFKHWfkcGgMDIvzgchXZe4lUyjWMPJ5XAPfDbCxvpu6vPemTlnKIbjgtZfHfbtcZacSyxHemoDwMcTkJuakVfwJKuQXH8r2sd5xfsWHcpH7AvJqumY4v+uAKAFmrS+uoC8WQ8GQhNycZXMvfOinzvjKObWI3yFciiAq8C9Vw\\\\\\\/q+kfE9Yg6utnqe5RDP\\\\\\\/YNjZn022iG+SvgNkJDiV4DsoZ\\\\\\\/7XctmKaXgNqCmfBYpuz4YTSgOhXgq10WgAuTSCMdpngWWPeeN2FN\\\\\\\/Jmnb0EvnEkbBrxDub44ZDXgdInJMmV\\\\\\\/zJ1eUwyozMDSQbz7fPdFXUDVLi07nCdFKok3uM6A40J7z7FdLYLaudjx6IB21M3i2tBMUyeEsDVrvr6DtWFluIKTHX0tEucAdynxJF9OiHjxHxrNIzAspfrUzEca4XxnrZfVTHT86jjWw9jMUuzDYMmWW0GLmprqmBSOxfDgp+zbDm7CBhN0U9Rh8rS3d0C5PhxEtXEDveyDHbMtclNaYDrGce\\\\\\\/PySO9tvzGQc4tYD7vsM+UYFsNBZXsItX12El1UFs9fb1yRPK7yhh6H2XNRT0rfh9GqvSlK\\\\\\\/d0CJNOP34xf8FczF2mnqp4awUg1BYqbtobtvh+d5b32QZY+vPqAxwGyB+41YwyigV8e5GiqaYyCx8\\\\\\\/BUYqFISWCLhyNToN+QL0e9oB6ESyDZ6KUq5sNUiKEoSYZecOUzLFmDq0uS5seFqKI6eu3md+sOHoXM3UOS0bEZFWWWeG2qlTogkVsD5oHCeS1WFfHDYhwXC+gemUUlXee5UDQ2Uj04wXPpiyeMdUh2PMw6lxs5ya6\\\\\\\/6GOtq\\\\\\\/xKfrm6h+ct\\\\\\\/nIxpmi31YGqAPAAqXAE0+7ciwDiW5kN3rcQey\\\\\\\/V9sQr3XemOYKLTOqtBosSkpwijim8g73IH\\\\\\\/Qk7j3QPwkw6I4U5FpHi+esxqxnynYpmBVY+skq+faED2brXK2i4UrsPwh+ejdSJYIgyghgqNRhb1ilI8lnI+kIpTAU437vgO9LPHEHzvebNhgcZbMhXRaKzfJD5EzgLIF5Nul\\\\\\\/LhUhPeuuc+RYHmVdwhENRpESx7+XvmUmjNE0ILkIv1OOsn3TFebs00yQUyf5NIHNuRPp0lLNdcd5MLeNaJiop13EPuLkzKPK4xhJ7hpRLWhymwE2JNyiCSbbuSN1gxxX+onpXt8H6dDedK2EwXQ\\\\\\\/ee3cBOPCgWLqgmQvXDOB\\\\\\\/k++IJEn9d4wO+b6XYu\\\\\\\/VJ6sPvsmq86CPnuRh8nomJdOSRtRSqT\\\\\\\/pqrPHpxfk+lqhDuO54VePyGMMuBPZAToEKeHYVufd+b73jEa7NTPqR\\\\\\\/oZYQyd12bvCTm\\\\\\\/QKwCeDS42sqGsU8B1evdwHJvrBi17ZM3DK\\\\\\\/wXnnwurndE6oZkptAIOFwrz4bqTAqMcCtqodXUIDUAEolPMBre0Fncswg26l5+NiwdUT2KS4BnH933ttZJg+t3BBReO+jKDU0q91l9JO7rJsvAr\\\\\\\/tFJy8k8Q2omna+SqLQQPSQEZTY3k00xe6bE6iXJNIf+p0yDTs1RN1sRm7UM1Ox+\\\\\\\/OPK4ajoXtABuJr2H9gDG6LOgmV81oxnnqzODaMoXDBH9kZ+NrI2qbdFMAfSIiemuSG\\\\\\\/68K4uoCp00MYlakeoDvbruSCZcQDu5fA3Aa4e4SVH8q4UQ6+EceeHYSwidXCpHfNPt24n0\\\\\\\/SI3l8M\\\\\\\/GtbfpqUzCaM5QSR6XC5uJyAhUOAQUdKMfJ246YY+pYKBzn4egE6K9Uzw+aQxLlSF42O7N\\\\\\\/hemYC8UJvqpkk8mNWAD\\\\\\\/zt5+Ltpcc0US1r\\\\\\\/HLk5g9hwrvmhKsz7S+JxmmuEvEs9WrkOcxBlHPcr38yHJZmb56Zlpb+28WGs\\\\\\\/rrzph5xSGC+5qRGrxUBSKvQU3\\\\\\\/iBwdaJ+F5cXLwo7Sw6QD547K4CHCQZkX9hTM38b8aQAyC\\\\\\\/oPRxiXxLLRdlsBFSwlejL0I+yr6zppZkQKV6D5V+4PBzLmtPKVB\\\\\\\/eSWQJ+isP\\\\\\\/XNU\\\\\\\/y146EugKqxpKG8C6AkQdVxdyZ8RMGkS\\\\\\\/goXUJUlIhb2upPflq06iY+r4\\\\\\\/PTM3SfyMbHI\\\\\\\/q27NJAHb3oHhuh8of7kak+EWdMAbb7e18MsUEm4\\\\\\\/aiKUAQMWOENljoNi2o8DhMLSQsJsXqdi0FLeZPNS5RvBq1lF8wdFJR+\\\\\\\/I0eyCmTSu7nzKimVDM6MgtxSW42BnVVpOQs2VtFgdIS4W4jkuOLGCXCpxko55fvvxrVd5+2jYGRc2jkt+vFokjWHs4EN8nPNw\\\\\\\/1KHYwQM17hpdPHILMIrJLfOebPxncT\\\\\\\/EN8l1I4+yQCoj4bm3s3i1w\\\\\\\/xJ7XBQ1JdPjqetiX9fR4W5zux\\\\\\\/TQxX0GWReQFeCrzMOfbLH42aqcjgaTNLaCR\\\\\\\/+mEVKnBSoa4UAnMjkO0S50DqK4\\\\\\\/BAi9RwRm4qAlQyh1D1siepjtTbusTL28u9+FyRtM4iZOm4tk5O0Q65PSN+MWcXXPYEKZxJvlFU9gBTb8ktRmeBZwMS4geVuSAFMjZ0HDo8y39JFs7os20btnaudrgsRCnyDWEk1gqMVo9Ebi7nhsqdDppby55+Eyb24UqrlcXl6mQcKhof+V5Je6C76rjSZ3HEKQBMItywq2GsXJPsbu2JrKc0xkvMzyi6d0zyKIYI6Pvtffiyru6jmbq7nlS9ugDcdeVaxMuN6R8yX++bC6+3yr87T\\\\\\\/Ykb79SnyZVZkLQamUffqrLkcfGKattbqv1hADnLy8dffi2ke+lljL+FmjkxqtLHDWirbTtQ5xzeUOig7kseWC5FDL0d2KzAZLFizib866dnOZxFCOthktRUf1pU83uflki597rok+CNHEPw2p2Twrm561SBBHoXkYaHINbLI3BGXdzQQ\\\\\\\/RJecLKia\\\\\\\/\\\\\\\/2x\\\\\\\/MtawlokNiqBew2Ab4YGs+lu\\\\\\\/UxdcEpjZMhyOauKJWIVMvFPSyoC9nM49rXE4d1DItUOTiMVkSeA8OQuO6vpRVckHOdh1MyzhZhXM\\\\\\\/7sK7QQtoI6ivHUDqEAI40NpqcE6rKA5T85BGANFHFxuLY35drbcm+SIo4wgFXxOoYJ7HwIKxEfNdTSlDeEP2nTZGAGMV7nX5yz6PQzlK0BgVaVg1YUYOEbTGGc8JQxpMtYYbkQDbG7LLRGb9HtOb8uyBVlTmjYToN453h3WVZ6c0Ipa0WNIldG6\\\\\\\/JI1ccLEou4XFtT2ontKkM3+aAQdPpId2we56NC9iPC3Wm+\\\\\\\/miqfFK2bqDD9API1qYTR3gfa91k4f68Flx31BQMssbRqwPUBoOBo2iySV9cNAaV9Ad76FmkObxItf2qtLXzWhJmHLcC\\\\\\\/3EsLE4u8oTA0KOUjnEmnRg6wnJOVSUk1bC9\\\\\\\/luhRcXH078aM+F5askcDoG0thKU87Gly\\\\\\\/shXBOGv2zf4kSr94tIoz5nFpC7MrYQLkwLxDYbkHObsvlOAgolmlbuxgPnvtMUN7ALSWa3S6LKDciSCk58RnHqfSCOzmKlsRT9XOlb8ulbEaUXuuJMZ1byOd+Fwf+JsbmTjCtDj7CEEoWcx10nMYwSmIAISCJcC1Qo9nqDUs2IFbpkqjKsS5xntu+gmQc9GXUf+deb2ZIBky3lhSBeJGc+8LjvnXDOxM3\\\\\\\/S5Ru11tGFAwzgYIQNVO3yfZ8nSd02+5EUD8MobCSySFxFnV\\\\\\\/58gMAn7uk6TrYce3RD\\\\\\\/ttHZDbBx97liatUMbdaNCWFRewsSphqJ8VuNx4XdGxJqPHrraAl9xxqP+45Gz0faNBFMw2pFQbWB7\\\\\\\/9UgeY+Kw5Uxkzj9ArcYpCMjbYJHk5iTm5ZvnUs+BiDJhZwARtOcVho4Z9uSHWxJPBeok0Y\\\\\\\/\\\\\\\/QTBTXEi+GcE8LqKNCHNfahThbM4qRdSigABUR0d7TCb7hQxaCuRmbIodpZS6F257s79a3iMU\\\\\\\/3iwIwaE\\\\\\\/YmMQQe1ie+EM0Z1sVMAbmT2AdOEzREBwiIEJzPxHRKR1QbB3V870r+fps4KnlqPHL6MGA7Qi\\\\\\\/Ek11eL49QTD42cA2oQMEUS12AleX1\\\\\\\/LWpxlAisvweIZsM7ZZ9Ws\\\\\\\/t8wvtBSLLCMheN5TRUs7BQfgU7iuC38TbdJ6Lmy4GZCGaTeV3YZSttFLyg0DQ49ayRWiB52E2iVcKCTvLhlK\\\\\\\/VQ4PUhCTBMyngG3lncclv10sZs6TekWbYadUslIrl4eVwD4+NfaCIYEzXARDWw3lWHEUzuUs7sA4CxMlEWmWiwTMsHJKivDtkcxCV3yaKgYRPQWg9falrdS2STZBc9UzMHCbqmCSbJ5Xkbqqkzd3N2friIlYD5qIhVsn+uJbv53Y8yePfbSJvcHXajB8fIM+RAqfvJc+2+VFDoQAYkIbY2f49r\\\\\\\/LdqK6ep\\\\\\\/QdeRu1BnQw9eRt5EHnvrTlJD9o4l\\\\\\\/Amda72875SRNySLeXv6ZK7\\\\\\\/AgHEb0vJYQIHsY+S7QBGn69S+HJ1fcDULuP+HGRJW88duQpub\\\\\\\/+AYVurTewVLG2Blm47QbWmE0eFzsD6fKDBap06x3KMIWo8RbpDPLlUtX2cebLH9F+9D9+19hRIa\\\\\\\/mw+E4cxwTy7DgbC7s0MJQBv1s88mBlEjH5lvtt6SOVwatwQq6Y\\\\\\\/2ZmGz9LUFrR4V1KVD3s84YTqTq77IzlUsADJo+KflFe7ko2bI5gYW6ME2NqOfvZT9UMNZUVJD8AxmoTmFuJvoU9\\\\\\\/uA6DLz6y9F3+QoHOIrwG3k9TXqwY4alOzuAAICg3YVk9xQjlRQvs4mhnP2Jm2bBnh2iH8iZ1F\\\\\\\/zmLI6EvrvnxiRmKyyJWNq2tTtugfTNpUeximIi4AYO9QrgPWsPqDKh6ZABqO+spinTjWvUCr3IEC7tf+KIq0mF6WLox4rzHHEe7Gynjz1jJbOPd\\\\\\\/V+HH7dh\\\\\\\/blWULq+bjwmMB21jQ41jRegigJaIzBPxpqqEsOBY36aEy1t8J2fg59oe7JKekhcD5C6\\\\\\\/uY+WWfGwdi7NdeXOw49UGMq9wlnxgF+QP0x3hYzmu9tz0xRRg0F1yVgvQnlSEempQVv5VPNr7n7wS57xwglCzPEgVr+vnhEK3dn9MztaoOdpCUVmb+2bBvN8oTQdxSYWYZPPmmEp8ETHwUW8fiDMw2aplaQ4VDpxIOcozJAmrbZoZx9Mec5uB534hzNjFn7AsE+K+tiA72Ee3pdQ5zQ+stivEj7Rup8vIRqVYLYoTaEL\\\\\\\/uTUMuhRrTWOEO\\\\\\\/DahaLgmFr4pcCmo0vtflfMJwUptS5ZQXPRseMVtiXy5TMj4oEfZZVs6H1iZ\\\\\\\/CvRrG+xDNKtlkNPOU4oZQClS31OcJ6usnPWTr50q1GcjYD\\\\\\\/siTTEVg+i9PVUaKzR315T7W7Nq8UpDfhccesbZMjEtP7wxg77dCx377yS\\\\\\\/4h1nXc9pA9Kg45vp+zQMvdO1bOwG+MnxRcL+BpOgQaI34VF0SPxk4ulvnumPh6KRiX9ELyJPm\\\\\\\/dcVhSLcIgLgsPvQ\\\\\\\/RTnsd\\\\\\\/5MjI7wmIwv9LmmXPizOBs2AHZRXVCXUc9hK3SpW7b7b9vaZnRdqAbph+lO8+7JMEHWb9kUJZpxsigrWzYW91n6ccscOyUgLvWm1a7fnn2WlpkKYOPGRjPgHPUxKyJGPIWlxiXxDob6NVG6Xp5A5i6\\\\\\\/P9N3hx\\\\\\\/i47wWLDMeYnBoS6A3MZxhMCT0PYkdE+0f77vAXS6H5\\\\\\\/7bpE8mUGLb5+nubAcfBcoModMTBVw6J01XhnEeGqzrCtZuF4M79phD4\\\\\\\/KfPZ50\\\\\\\/WX5Z2N7vpRjr6yJvd5FkSULc5FBgjxV8FKnaLA6lenvCypKtJKmbGcu1Rd\\\\\\\/np8yWkw76LHJvWLCgQEI9bNLMOw0Ah\\\\\\\/8qhu\\\\\\\/Be5rS97c\\\\\\\/FqeH5BLISMiAz4Jv13716zNX\\\\\\\/76afdNEtUf1Kq4DOIjAd03aL0hVNnqZp0nA5sLRjejL4gJvjL\\\\\\\/1q8\\\\\\\/g4DuqZNeLw2+TnPpQaTBlX\\\\\\\/YsQVlO2ckWVksgBterFlTF1EgKYdVHOI\\\\\\\/JLbQU5CLfBdE8iSg9zN7OL2YPrao74E10xNx8hoGDkLSi0cSUbrltmCJeCoN5+jbzZ8RNDfjrmaYBlm8Is2EWbq5Nuqn1LwtVEjZwcYnfkLC+EnDokzDaGtdYpKpemnceErpWpHsnenyHERlb8tsp3ZTUOXWDnpQ7AVP14GIjqzU3jQNH0pXRklhheAQnF5EPgg5yIxssikyQiX2CtjVc6QOaVkjhzbZ0xp5U4TXNR++DQQTR6S0Lq4Qs2obGZWVqqmfkQYHHgZUKKfchLXelsp4CI+maB3tYDvuo87\\\\\\\/LU5ydQStoaoQHlLkY3SVcmps+3GzOFEAlRM7RnAkleLTUAOCCOYJUrAwlcxsEPld4OBmtt7NMwyuFogtQnmkqm8dUh6IpbHgqM6O5o5XRkRJPQahPYU3h9fCYJjfhNsdRWuQSWFmou+BUnY5Hptn2xxgyjLRhtI\\\\\\\/PvGh+zYQOp7SOkpg6F78kgMce9ELWnu1t+tqpRbNs2CcbMuPq8a770Qtmvn93LYDENAlnY1wUyr0VcAQfOE+Bwj+alxHt1hRb0tKLAXVTLeLM+V6FUEXxC76ZIczW564Yi+QOnzxedHT4X9GXlyJausyKE1g0DxRlwzJwwdS5mUDMzBLQIoBldn6dihUhBqeouBe8OfuyItgYddvI6p8sVGxAUW+n6w0uXgbbkQUdFHAtKsBgDQ9719vrqc57fvPdAJxF8HScwV6RwR11yUDhlNObfHhd59QzdqaKpLujHByoWI8y4mnbdcWzS9tQmSSy7R4Jl\\\\\\\/eoZ+CiX+7xX9BdA0qzdhq25OtAFJpqe4Yxu\\\\\\\/JldznYkvQN4KxNccxm0pyI72aMYVHO9LwxTobDWTpSZXK+zIkSHCS2jBL\\\\\\\/oWbJk8YkUgoITzTRY8cupHvhLx9QEpyyuAgS8EOaiVAGXa+Nhn0J3VeRFlLPuGKonUpiFwfAJLs8YzqJZXSmHUTZtlW801qds2O9E7IlMiZLWRojItBYdMKa6z9slL986\\\\\\\/aMUESMH+PD0pdRvqcJAlpk7qXSECquMnB95QaMN2wwh8lT4a11YKNBhTYXau4e++sdu\\\\\\\/sy\\\\\\\/1gqLUBt1iUmMpXjpobx8X9NwOhBaFYgQWlkARaDa7BNnnqlxXGmd2acDqxBheRq7p5Avw2YlPSilq\\\\\\\/1\\\\\\\/Wq1LrAYnqgToluf4iuqtfG7KFFUrVSRf4tuq3rbaD+ez6eT4u+2Egf441GLI16MC3TD94t8+OSm8ATxvjAAZ4IMI4Mfgule5hMRyPZF+WJ4pL6s7Tdn7aYs5GgUpW1rtb6W0TDSHEINdbVzy8UbBXtctOCFzhtjhdBacF1PVtKkj+bSeywyspN4XmmoAqELMuigXA1qsFVEq22RqJlVMlkbKoTLGK14SBvJCXEcILac2jCKHcLzIbv3eVWUM65E1iHWgN74IEINoM1OXfC9ZtM1P\\\\\\\/r8BnP8XnTmNFYRZ93cXYOOagor+o9ar1q2RomOEIIi0dmG9mxDR9b8NChGeAbD4NUVYoYBks+\\\\\\\/EE4duaCwBUgKitVKtfZMC3BYbNFaEe0mGCn+zXIm3yxqCOopPGHqiAHK4nG\\\\\\\/iI3cn9yN\\\\\\\/R36oK3mKXsEkLpH4aHDKC3eYwhIqQYLwY8F3gJRqRKJsuPT\\\\\\\/BujKQSB+RXybae6r7zjkk2EoB2vkEpcgXnUp6ROfTflGocxadmUbI\\\\\\\/SpTe4lv0wKuAN2gpp9Dw2khGew0CVJ63b3cU51vmBVuOGxswPYiwA7wh7wpkjHiHGV4CXdBZkN2KfYNu5r5zyWkjUiysNO9yMLHhe3cgorvADCcZu941+E9+x926HM3MNhdatC+vUPZDMjssBwgoKy8mgNVmUm2Fb1hPWsy+lBulFl7eMkEcKjg7V0+tIDjGFEtag+skpRta4BzQ7jBigW5jyey3Xn7Wyw4nKFqILivqQgfOECxcp1sfCqEJAbiudFATbLJQITo\\\\\\\/YvyLE8bbwT4Mw6K6Qhq\\\\\\\/LCK27IbZmUQfMuM2PfgOxALQ3ZvG0+6fCvEnw0gaAdtpOUuhTSn9BNXf0OGivAwmbBjA2aUsYDq6EQiAFZon3HQKGrk87rMVSu7\\\\\\\/Tg7PdBxOCMqfc80W7WK1QM37y\\\\\\\/mBjBcOLvNC39cWdEjjUsY+dgeskQZDKSJP22pN9TypIKctwyz2cNe0kkcLe4Wv4fNB5IlCfyzMRkyuSn54MSKB7hQAYSGtEZwN0E0RtPzMRBmAw2vyl\\\\\\\/+KfqShkrvpZW5uFI1M5qnsnQgtS2cy\\\\\\\/auqVlU2NaPKMv7Z9Sod3NlAPcIgI9hLOOGsVbP4m8rMw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022k2Z3Joa5DhpsOJMKOYxKRg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d772437f1a84c6afa4f0ae26d726352ea67cf2bf9afbbd6db68a3e0ca206d51d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jTLoxpNCN21RGDQ4yAMNhHCBJ3gGHEg2S8tFQ+8Ynv7SEl78jSZZG4zvitugf7JEDHJ\\\\\\\/4wyJdqXleR7VD6tO8QLrkFdS6+3QfqzuK0UYduoLbQA8l0Qk0PJKcc+1TTIzbXNWKsHfNZM0+huDv53WZNLbay\\\\\\\/FIUfjMV1uWQ27q4ngNyxJ4P1IbWFcbGFSJJO\\\\\\\/cRnInMM9n\\\\\\\/eqqb6bx74Z0bc4ZTwBZHvOkPGLBjUlQ\\\\\\\/kRzeHp9dDh3Pw0ZIS3pQOM0Kl7+ci8BMdGMZfPLapv9Zc89TxmXsnQ3qAUKG+uVg4K8nHz1W0YoP9+n4H8sHmS1d23W4gpRbYCRsioFAjxYhRp1MQu2StZ2\\\\\\\/sRYzpEz6ZVR3qXMvPzA\\\\\\\/Yy\\\\\\\/el9\\\\\\\/RSiwN3Q8cTNyLmpr4x3F+\\\\\\\/AMpS5O262UvsUPIz1\\\\\\\/Jkb734H3GAtLL2FOwoCqIuj8ci7TdqJ\\\\\\\/SVKzrf0T+1nhQ9lf1HjA+LHrInF3X1oouAXFd31i+S8dF5HLPeII5qwGf7WQ9xS8j6PVX2UKiPIQdeCIfKprJY8n\\\\\\\/NnbcSIfpWU\\\\\\\/zGMaBBYRInvvHNPiTvxHwxS1\\\\\\\/bbGdU814qeRO\\\\\\\/ZdaI+X0CvbRWNAezBiADjz9f9Soc5M7SVOVisqCAR1zrXy4hI6RGHZeD3kmrWj2FD0kHWeEhOJ\\\\\\\/A7woZrtp1Ry3MVjEdQ02spPX6ebokjtplDoO3hnOnB65IJkLibWFR3qUf2fZpVkfKl5qCTM9tEUf\\\\\\\/RydHbDnhWzGPinWOu9PCdfGK\\\\\\\/gqugmxPZ\\\\\\\/1g+lY\\\\\\\/oChGhGHbY7iWeHucsuuukr4aeeiePi4lltE2pWlEQEiqYNQlQSg6FQBjwSjL1EzgtL+q4aRWNBgSOCJT4HgbJaDipjpMn8dF9oSBCyZRISaS4P+Bdd+8jsawOsmfQs0x2RfAVq0JpgQqaUR\\\\\\\/9afOV5tAiRHdN+z90nHDOUB4y8CIp6W3nq4yt94SMUrf71JxfLn1hg2UNyytrm93v5gzKzXabxeaRMSbY2XXoy9d\\\\\\\/OJJJcB6jEZpmkRA5vnvt9LAifAXRnerbIAyrWpoIUvn\\\\\\\/oPGsFUPbuuWQYmpFrQY13EBhdglKe2h8Tq7JhFBVPY3r5dlHYcdd6akAaVa1mQw7Bxx595+6iRhDuZyyRgvSHVQQB\\\\\\\/0dL4kCFOklE2srXdzwxdAgaoIdT8XBi8LHEN0\\\\\\\/8cJHD\\\\\\\/ej1HdTAW0tdRRleLHOl+IvpsYv9moOb+idvG86VHxDUlSrm\\\\\\\/KhqBwBKXZPkXJ3La2z2h6CTxW2eRX\\\\\\\/cORuzhSCXoi1hgbgflgmGkdar65sDcLR+cWAGqFiSTUdUCdt2t0YV\\\\\\\/PNz\\\\\\\/NbNSi59bg\\\\\\\/s4yuC1wSPryscDYw1J\\\\\\\/OuNlJ3lxB\\\\\\\/UweX27RgLuQROJv9FYGCLurz12yeDdjiqGKXUyQULm0o5hdUHISDdQtzP8bChcWNum0xEXxHUvQY8UWNJFZRj7v\\\\\\\/VtjAZsQE8ffI2bDh1MaEswpq1Vb5dT444YrHLhz8to1wvIGniBBjq0+9zAfjx1GTRU9GzC1UbUx+xq\\\\\\\/FlO7sdlOT75YqGbFRG3OZc1T3C8LmqK\\\\\\\/s8Jp1p3maPs19gbkowpsS6pmdAeJofxh+B1RLMTXr+2KXpQkQMOxnCA0DcLw1L04fB8JsImZDAP5tJdUCcLrMFNqQSUBky7FiiwJ29zxVoZC98BiJGIBvkdb\\\\\\\/7d2dydb1Lbm1vyLibSpN+4uwsCICAMS\\\\\\\/JXNnJdAGMKAKsevo\\\\\\\/FnN6Y0VykuRnQyw55o9qEobFn5xuaDJaFduKSFGuk8WG41y69HhPIPZtiTtK1m5r\\\\\\\/NT+gcu+aH\\\\\\\/ZyJm+pTSEkG0hbmsmil60Ny8uMRJLvb8Xfpdl98Ope9Hdp1vp791xJo0nQpZhh9dTUSe+SbJthTUox4opwTzNgt1HJNE8BrGtJ4B4hYtUYqKR1alAaOPgWLn+eFHGoIyBQydLfGheEIO3AvD+tWTAZEbC0AG4IJWAj7rN9eQvr3IQnW1FlJeDfvL8NIExixiw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022M5UWkqvOfwZfkr\\\\\\\/I2IF1Qw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f492b22e966dc544a1ddac1e1ff479a40f8eed004fb8215472c5d947dd8bc814\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487108 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3605" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ObWOcr45s-hoxMwGSNanWnd-pxsw8h3TXJI150ModC8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3605" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ikk8mk9o3ObEPsvr58HY-uisMUO_N8R8cZlWrnXdRJI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3597" title="The Transcriptional Response of Candida albicans to Weak Organic Acids, Carbon Source, and MIG1 Inactivation Unveils a Role for HGT16 in Mediating the Fungistatic Effect of Acetic Acid" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3621" title="Genome-Wide Analysis of DNA Methylation During Ovule Development of Female-Sterile Rice fsv1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487108" id="node487108" data-pisa="ggg;7/11/3605" data-pisa-master="ggg;g3.117.300259" data-apath="/ggg/7/11/3605.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6991-587X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Annabel C. Beichman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2685-6977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanya N. Phung</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3874-369X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kirk E. Lohmueller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3605-3620;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300259 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487108" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Annabel C. Beichman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabel%2BC.%2BBeichman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beichman%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnnabel%2BC.%2BBeichman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6991-587X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Annabel C. Beichman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tanya N. Phung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanya%2BN.%2BPhung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phung%20TN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanya%2BN.%2BPhung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2685-6977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanya N. Phung</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kirk E. Lohmueller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kirk%2BE.%2BLohmueller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lohmueller%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKirk%2BE.%2BLohmueller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3874-369X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kirk E. Lohmueller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:klohmueller@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">klohmueller@ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/11/3605" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487108" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3605.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487108" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3605.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487108" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/11/3605.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487108" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/11/3605.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487108" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Inference of demographic history from genetic data is a primary goal of population genetics of model and nonmodel organisms. Whole genome-based approaches such as the pairwise/multiple sequentially Markovian coalescent methods use genomic data from one to four individuals to infer the demographic history of an entire population, while site frequency spectrum (SFS)-based methods use the distribution of allele frequencies in a sample to reconstruct the same historical events. Although both methods are extensively used in empirical studies and perform well on data simulated under simple models, there have been only limited comparisons of them in more complex and realistic settings. Here we use published demographic models based on data from three human populations (Yoruba, descendants of northwest-Europeans, and Han Chinese) as an empirical test case to study the behavior of both inference procedures. We find that several of the demographic histories inferred by the whole genome-based methods do not predict the genome-wide distribution of heterozygosity, nor do they predict the empirical SFS. However, using simulated data, we also find that the whole genome methods can reconstruct the complex demographic models inferred by SFS-based methods, suggesting that the discordant patterns of genetic variation are not attributable to a lack of statistical power, but may reflect unmodeled complexities in the underlying demography. More generally, our findings indicate that demographic inference from a small number of genomes, routine in genomic studies of nonmodel organisms, should be interpreted cautiously, as these models cannot recapitulate other summaries of the data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pairwise-sequentially-markovian-coalescent" class="hw-term hw-article-keyword hw-article-keyword-pairwise-sequentially-markovian-coalescent" rel="nofollow">pairwise sequentially Markovian coalescent</a></li><li class="kwd"><a href="/keyword/site-frequency-spectrum" class="hw-term hw-article-keyword hw-article-keyword-site-frequency-spectrum" rel="nofollow">site frequency spectrum</a></li><li class="kwd"><a href="/keyword/population-genetics" class="hw-term hw-article-keyword hw-article-keyword-population-genetics" rel="nofollow">population genetics</a></li><li class="kwd"><a href="/keyword/demographic-inference" class="hw-term hw-article-keyword hw-article-keyword-demographic-inference" rel="nofollow">demographic inference</a></li><li class="kwd"><a href="/keyword/nonmodel-organisms" class="hw-term hw-article-keyword hw-article-keyword-nonmodel-organisms" rel="nofollow">nonmodel organisms</a></li></ul><p id="p-3">The pairwise sequentially Markovian coalescent (PSMC) and related methods have become a popular tool to estimate the history of a population from genetic variation data (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">McVean and Cardin 2005</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>). These methods use whole genome sequences from one to four individuals to infer the demographic history of an entire population. Specifically, they estimate the local time to the most recent common ancestor (TMRCA) for small regions in the genome and then use the distribution of these coalescent times to infer an overarching demographic history. For instance, if many regions of the genome coalesce at a specific time, it may be evidence for a population contraction, which would reduce the number of genetic lineages. The great appeal of these methods is that they do not rely on deep sequencing of multiple individuals in a population; instead, a single genome can be used to infer the demographic history of an entire population. PSMC and its successors have been used to infer the demographic histories and split times of many human populations (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kidd <em>et al.</em> 2012</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2015</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Henn <em>et al.</em> 2016</a>), and were recently featured in three prominent articles that reconstructed human history using whole genome sequencing data from over 20 populations (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Malaspinas <em>et al.</em> 2016</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Mallick <em>et al.</em> 2016</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Pagani <em>et al.</em> 2016</a>).</p><p id="p-4">PSMC plots have also become a cornerstone of many studies of nonmodel organisms lacking resources for the sequencing of numerous individuals, including archaic hominins (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Meyer <em>et al.</em> 2012</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Prufer <em>et al.</em> 2014</a>), great apes (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Prado-Martinez <em>et al.</em> 2013</a>), wild boars and domestic pigs (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Groenen <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bosse <em>et al.</em> 2014</a>), canids (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Freedman <em>et al.</em> 2014</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Wang <em>et al.</em> 2016</a>), horses (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Orlando <em>et al.</em> 2013</a>), over 38 bird species (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Nadachowska-Brzyska <em>et al.</em> 2013</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">2015</a>, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hung <em>et al.</em> 2014</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Murray <em>et al.</em> 2017</a>), pandas (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Zhao <em>et al.</em> 2012</a>), dromedaries (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fitak <em>et al.</em> 2016</a>), flowering plants (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Albert <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ibarra-Laclette <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Holliday <em>et al.</em> 2016</a>), and even woolly mammoths (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Palkopoulou <em>et al.</em> 2015</a>).</p><p id="p-5">Despite their widespread prominence, there is concern over the validity of demographic models obtained from this set of whole genome-based methods. Particularly, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Mazet <em>et al.</em> (2015b</a>) found that PSMC captures the inverse instantaneous coalescent rate (IICR) rather than an absolute measure of population size. The IICR corresponds to the effective population size if the population is panmictic, but it can differ from the population size due to gene flow and population structure, which affect the time to coalescence between subgroups. Thus, population structure can give a false signal of population growth or contraction, a notorious problem in demographic inference (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Ptak and Przeworski 2002</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chikhi <em>et al.</em> 2010</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Peter <em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gattepaille <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Heller <em>et al.</em> 2013</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Mazet <em>et al.</em> 2015a</a>,<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">b</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Orozco-terWengel 2016</a>). Given these possible confounders, the degree to which whole genome-based plots derived from PSMC and its successors correspond to actual population size changes, rather than other demographic phenomena, remains unclear.</p><p id="p-6">An alternative approach to infer population demography from genetic data uses the site frequency spectrum (SFS). The SFS represents the distribution of alleles at different frequencies in a sample of individuals from a population (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Nielsen 2000</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Wakeley 2009</a>). The distribution of single nucleotide polymorphisms (SNPs), ranging from rare “singletons” which appear only once in the sample, to high-frequency variants that may appear in the majority of individuals, is directly affected by the demographic history of the population (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Nielsen 2000</a>; <a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Wakeley 2009</a>). Population contractions (“bottlenecks”) can lead to a dearth of rare variants (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Nei <em>et al.</em> 1975</a>), whereas a rapid population expansion can lead to an overabundance (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Tajima 1989</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Slatkin and Hudson 1991</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Keinan and Clark 2012</a>). The SFS is a sufficient statistic for unlinked SNPs and has been used extensively in population genetic inference of demography (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Nielsen 2000</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Polanski and Kimmel 2003</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adams and Hudson 2004</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Marth <em>et al.</em> 2004</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Keinan <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gravel <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Excoffier <em>et al.</em> 2013</a>). SFS-based demographic inference has been implemented in programs such as ∂a∂i (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>), moments (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Jouganous <em>et al.</em> 2017</a>), fastsimcoal2 (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Excoffier <em>et al.</em> 2013</a>), stairway plot (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Liu and Fu 2015</a>), fastNeutrino (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bhaskar <em>et al.</em> 2015</a>), and others (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Schraiber and Akey 2015</a>). The SFS requires less sequence data per individual than the whole genome methods, but requires a greater number of individuals to be studied, with a minimum of 10 per population typically used (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Excoffier <em>et al.</em> 2013</a>). While the SFS is impractical if one can only sequence one or two individuals per population, population genomic studies based on many short loci scattered throughout the genome are beginning to be carried out on nonmodel organisms. RAD-seq data or gene transcript data from RNA-seq can readily be used for SFS-based demographic inference (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">McCoy <em>et al.</em> 2014</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Trucchi <em>et al.</em> 2014</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Sovic <em>et al.</em> 2016</a>).</p><p id="p-7">SFS-based and whole genome-based methods may have different strengths and weaknesses for demographic inference (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Schraiber and Akey 2015</a>). Theoretical and empirical data show that SFS-based approaches using large numbers of individuals can accurately estimate recent population growth (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Nelson <em>et al.</em> 2012</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Tennessen <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gazave <em>et al.</em> 2014</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bhaskar <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gao and Keinan 2016</a>). In contrast, whole genome-based methods are less able to do so (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>). Recently, however, <a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a> developed the multiple sequentially Markovian coalescent (MSMC), an extension to PSMC that uses the SMC′ algorithm (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Marjoram and Wall 2006</a>) and can infer demography from two, four, or eight haplotypes (also known as PSMC′ when inferring from two haplotypes). The incorporation of multiple genomes in MSMC is specifically meant to improve estimates of recent growth (<a id="xref-ref-82-4" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>).</p><p id="p-8">The SFS may be limited in the degree to which it can detect ancient bottlenecks &gt;2<em>N<sub>e</sub></em> (effective population size) generations ago and in its ability to detect population declines (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bunnefeld <em>et al.</em> 2015</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Terhorst and Song 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boitard <em>et al.</em> 2016</a>). Whole genome-based approaches are not constrained <em>a priori</em> by the number of population size changes as is common in the SFS-based approaches (but see the “stairway plot” approach of <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Liu and Fu 2015</a>). They therefore often give information about events occurring millions of years ago, but the reliability of those results remains uncertain (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>). Further, demographic models inferred from human populations using the SFS were unable to recapitulate the empirical distribution of identity-by-state (IBS) tracts across the genome, while PSMC-derived models and an IBS-derived model were better able to match the IBS tract distribution (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harris and Nielsen 2013</a>). However, the IBS-derived model did not predict the empirical SFS.</p><p id="p-9">Due to these different strengths and weaknesses of approaches using single types of data, new methods have been developed which attempt to combine linkage disequilibrium (LD) information and the SFS (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bunnefeld <em>et al.</em> 2015</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Boitard <em>et al.</em> 2016</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em> 2017</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Weissman and Hallatschek 2017</a>). One of the most recent methods is <a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em>’s (2017)</a> SMC++, which combines a PSMC-like approach with the SFS to condition an SFS calculated from many individuals on the distribution of TMRCA from a single unphased genome. This approach is fast and potentially very powerful, but has the same barrier to entry for those studying nonmodel organisms as the other SFS methods, as it requires sequence data from many individuals.</p><p id="p-10">Due to anthropological and biomedical interest, humans have been extensively studied using numerous demographic inference methods and provide a means to quantitatively compare these demographic inference approaches using the same empirical populations. <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> and <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Gravel <em>et al.</em> (2011)</a> carried out SFS-based inference of human demography using the diffusion approximation in ∂a∂i, while <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Li and Durbin (2011)</a> and <a id="xref-ref-82-5" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a> estimated human demography from the same populations using PSMC and MSMC, respectively. Although the results are in some ways generally similar, the demographic models inferred for three human populations using MSMC (<a id="xref-ref-82-6" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>) differ from demographic models for the same populations derived from SFS-based methods (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>). MSMC infers ancient ancestral sizes and periods of growth and decline (the characteristic “humps” in MSMC trajectories) that were not detected in the SFS-derived models as well as inferring greater recent growth (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The models inferred using MSMC also vary depending on the number of genomes used for the inference (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Demographic histories for the (A) CEU, (B) CHB, and (C) YRI populations. Trajectories are log scaled and in terms of physical units (diploid individuals and years). Models were either inferred using SFS-based methods (Gutenkunst) by Gutenkunst et al. (2009); from a sequentially Markovian coalescent-based approach (MSMC) from two, four, and eight haplotypes by Schiffels and Durbin (2014); or using a combined SFS and whole genome approach (SMC++) by Terhorst et al. (2017). The Gutenkunst models also include migration between all three populations, not depicted here. Models are scaled by the generation times used in each study [Gutenkunst et al. (2009): 25 yr/generation; Schiffels and Durbin (2014): 30 yr/generation; Terhorst et al. (2017): 29 yr/generation]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884691631" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Demographic histories for the (A) CEU, (B) CHB, and (C) YRI populations. Trajectories are log scaled and in terms of physical units (diploid individuals and years). Models were either inferred using SFS-based methods (Gutenkunst) by Gutenkunst &lt;em&gt;et al.&lt;/em&gt; (2009); from a sequentially Markovian coalescent-based approach (MSMC) from two, four, and eight haplotypes by Schiffels and Durbin (2014); or using a combined SFS and whole genome approach (SMC++) by Terhorst &lt;em&gt;et al.&lt;/em&gt; (2017). The Gutenkunst models also include migration between all three populations, not depicted here. Models are scaled by the generation times used in each study [Gutenkunst &lt;em&gt;et al.&lt;/em&gt; (2009): 25 yr/generation; Schiffels and Durbin (2014): 30 yr/generation; Terhorst &lt;em&gt;et al.&lt;/em&gt; (2017): 29 yr/generation].&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3605/F1.medium.gif" width="273" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/11/3605/F1.medium.gif" width="273" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487150" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Demographic histories for the (A) CEU, (B) CHB, and (C) YRI populations. Trajectories are log scaled and in terms of physical units (diploid individuals and years). Models were either inferred using SFS-based methods (Gutenkunst) by <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a>; from a sequentially Markovian coalescent-based approach (MSMC) from two, four, and eight haplotypes by <a id="xref-ref-82-7" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a>; or using a combined SFS and whole genome approach (SMC++) by <a id="xref-ref-95-3" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em> (2017)</a>. The Gutenkunst models also include migration between all three populations, not depicted here. Models are scaled by the generation times used in each study [<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a>: 25 yr/generation; <a id="xref-ref-82-8" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a>: 30 yr/generation; <a id="xref-ref-95-4" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em> (2017)</a>: 29 yr/generation].</p><div class="sb-div caption-clear"></div></div></div><p id="p-12"><a id="xref-ref-95-5" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em> (2017)</a> analyzed the same populations with the combined whole genome and SFS method, SMC++, finding an ancestral size more in line with <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em>’s (2009)</a> model, but with greater recent growth and ancestral bottlenecks more resembling the MSMC models (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The reasons why these approaches to demographic inference yield different estimates remain poorly understood.</p><p id="p-13">Here we leverage humans as a model system to perform an empirical comparison of the performance of whole genome, SFS, and combined methods of demographic inference. Specifically, we determine which published models of human demography described above (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) best fit the empirical distributions of genome-wide heterozygosity, LD decay, and the observed SFS.</p><p id="p-14">We find that the models inferred using the SFS or the combined method, SMC++, accurately recapitulate heterozygosity and the observed SFS. Among the MSMC models inferred by <a id="xref-ref-82-9" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a>, only the MSMC models based on a single genome were able to accurately recapitulate heterozygosity, and none of the MSMC models predicted an SFS that matched the empirical SFS. None of the demographic histories accurately predicted LD decay, but the histories derived from MSMC using four genomes (eight haplotypes), the SFS, and SMC++-based models fit better than the MSMC models based on one or two genomes. Our results provide a cautionary tale against the literal interpretation of demographic models inferred using one type of data, instead arguing for considering multiple summaries of the data when making detailed demographic inferences in nonmodel species.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Published demographic models used in this study</h3><p id="p-15">We determined which, if any, of the published models of human demography (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) described below could accurately predict multiple summaries of the genetic variation data. Demographic models that fit the data well should produce patterns of genetic variation that match the empirical patterns in the data. We focused on three human populations: Utah residents with Northern and Western European ancestry from the Centre d’Etude du Polymorphism Humain collection (CEU); Han Chinese in Beijing, China (CHB); and Yoruba in Ibadan, Nigeria (YRI).</p><p id="p-16">The first set of demographic models was jointly inferred for the three populations in ∂a∂i by <a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> using a three-population joint SFS based on data from intronic regions. Their model parameters were made available both in ∂a∂i and Hudson’s ms (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hudson 2002</a>) format, and include gene flow between the three populations (here referred to as the “Gutenkunst” model).</p><p id="p-17">The next nine models were inferred by <a id="xref-ref-82-10" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a> using whole genome Complete Genomics (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Drmanac <em>et al.</em> 2010</a>) sequence data of two, four, and eight statistically phased genomic haplotypes (one, two, and three individual genomes) per population to infer demographic histories using MSMC (here referred to as the “MSMC 2-haplotype,” “MSMC 4-haplotype,” and “MSMC 8-haplotype” models; Supplemental Material, Supplementary Note 1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-18">To analyze their models with ∂a∂i, we converted these nine demographic models (CEU, CHB, and YRI populations, each based on two, four, and eight haplotypes) into stepwise models of population size changes over small time intervals (Figure S1 and Supplementary Note 2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-19">The final set of models was inferred by <a id="xref-ref-95-6" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em> (2017)</a> in SMC++, a combined SFS plus whole genome approach. For the whole genome portion of the analysis, they used high coverage sequence data from Complete Genomics, and generated an SFS based on a combination of 1000 Genomes and Complete Genomics whole genome data for each population (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Drmanac <em>et al.</em> 2010</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2015</a>; <a id="xref-ref-95-7" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em> 2017</a>). We converted these SMC++ models to ∂a∂i and ms format in the same manner as the MSMC models (here referred to as the “SMC++” models; Supplementary Note 2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-3" class="subsection"><h3>Heterozygosity predicted by demographic models</h3><p id="p-20">We compared the distribution of expected heterozygosity from data simulated under each demographic model to empirical 1000 Genomes data from the same populations to determine which models most accurately predict this broad summary of the data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). While heterozygosity is a summary of the SFS, we considered it valuable to examine both statistics since information regarding the spatial correlation among SNPs along the genome is lost in the genome-wide SFS. The distribution of heterozygosity across windows of the genome retains some spatial information and is more similar to what is used by the MSMC inference approach.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Kernel density distribution of expected heterozygosity (&#x3C0; per site). Heterozygosity was calculated across 100-kb windows from whole genome 1000 Genomes Project data for (A) CEU, (B) CHB, and (C) YRI, and from 20,000 &#xD7; 100-kb blocks for data simulated under each demographic model. The black dot and bars indicate the mean &#xB1; 2 SD for each distribution. Note the log-10 scaling on the y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884691631" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Kernel density distribution of expected heterozygosity (&lt;em&gt;&#x3C0;&lt;/em&gt; per site). Heterozygosity was calculated across 100-kb windows from whole genome 1000 Genomes Project data for (A) CEU, (B) CHB, and (C) YRI, and from 20,000 &#xD7; 100-kb blocks for data simulated under each demographic model. The black dot and bars indicate the mean &#xB1; 2 SD for each distribution. Note the log-10 scaling on the &lt;em&gt;y&lt;/em&gt;-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3605/F2.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/11/3605/F2.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487239" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Kernel density distribution of expected heterozygosity (<em>π</em> per site). Heterozygosity was calculated across 100-kb windows from whole genome 1000 Genomes Project data for (A) CEU, (B) CHB, and (C) YRI, and from 20,000 × 100-kb blocks for data simulated under each demographic model. The black dot and bars indicate the mean ± 2 SD for each distribution. Note the log-10 scaling on the <em>y</em>-axis.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-4" class="subsection"><h4>Empirical heterozygosity:</h4><p id="p-22">1000 Genomes data from the CEU, CHB, and YRI populations were downloaded. 10 unrelated individuals per population (see Supplementary Note 3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a> for sequence IDs) were randomly chosen so that comparisons could be made with <a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em>’s (2009)</a> empirical SFS, described below. For all our empirical analyses, only sites that passed the 1000 Genomes “Strict Mask” filter were considered (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2015</a>).</p><p id="p-23">Expected heterozygosity per site (<em>π</em>) was calculated in nonoverlapping 100-kb windows from the whole genome data (Supplementary Note 3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>) as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-1.gif"/></noscript></span></span>where <em>p</em> is the frequency of one allele, <em>L</em> is the total number of callable sites in the window, and <em>n</em> is number of sampled chromosomes (<em>n</em> = 20 for 10 diploid individuals).</p><p id="p-24">Because genetic variation can be affected by linked natural selection (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gazave <em>et al.</em> 2014</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Schrider <em>et al.</em> 2016</a>), we also calculated expected heterozygosity for a set of 6333 × 10-kb neutral windows that were selected using the Neutral Region Explorer (NRE) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Arbiza <em>et al.</em> 2012</a>) (Figure S2 and Supplementary Note 3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The NRE is a useful tool that allows for the quick identification of putatively neutral regions that have high recombination rates and high <em>B</em>-values (indicating less linked selection). For the full set of parameters used in selection of putatively neutral regions, see Supplementary Note 3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-5" class="subsection"><h4>Simulated heterozygosity:</h4><p id="p-25">For each demographic model, whole genome data for 10 individuals were simulated in MaCS (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2009</a>) over 20,000 × 100-kb independent blocks, each with a different recombination rate drawn from the distribution of recombination rates calculated by <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Phung <em>et al.</em> (2016)</a> from the pedigree-based genetic map assembled by the deCODE project (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kong <em>et al.</em> 2010</a>). Additionally, 6300 × 10-kb independent blocks per 10 individuals were simulated for comparison to the neutral regions from the 1000 Genomes data set (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2015</a>). Each 10-kb block was simulated using a recombination rate matched to that of one of the empirically neutral 10-kb windows, linearly interpolated from the deCODE project (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Kong <em>et al.</em> 2010</a>). For both sets of simulations, the expected heterozygosity across the 10 individuals was calculated using the equation above in msstats (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Hudson 2002</a>).</p></div></div><div id="sec-6" class="subsection"><h3>LD decay predicted by demographic models</h3><p id="p-26">We calculated LD between pairs of SNPs using genotype data from 10 individuals from each of the four populations in the 1000 Genomes Project data. We removed singletons and sites where all 10 individuals were homozygous for the reference allele and then calculated genotype <em>r</em><sup>2</sup> using vcftools (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Danecek <em>et al.</em> 2011</a>). All pairs of SNPs were then placed into bins based on their physical distance (bp) between each other, from 0–1000 bp (bin 1) to 50,000–51,000 bp (bin 51). Within each bin, the average <em>r</em><sup>2</sup> was calculated by dividing the sum of <em>r</em><sup>2</sup> values of each pair of SNPs in the bin by the total number of SNP pairs in that bin.</p><p id="p-27">The same procedure was carried out for the data simulated in MaCS (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2009</a>) that were used for the calculations of heterozygosity above. The MaCS output was converted to vcf format using a custom bash script. Genotype <em>r</em><sup>2</sup> was calculated in vcftools (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Danecek <em>et al.</em> 2011</a>) for each 100-kb simulated window, the SNP pairs were binned by distance, and average <em>r</em><sup>2</sup> was calculated as described above. The MSMC 8-haplotype YRI and MSMC 4-haplotype CEU, CHB, and YRI models have extremely large ancestral sizes, and so their simulations involve so many SNPs that the LD calculations become highly computationally intensive. Therefore, for these models only 5000 × 100-kb blocks were used for LD decay calculations, with 20,000 × 100-kb blocks used for the other models. We experimented with down-sampling the results and found no change in the LD decay curve due to the smaller amount of data.</p><p id="p-28">To demonstrate that the use of the SMC′ approximation in the MaCS (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2009</a>) simulator was not biasing our estimates of LD, we simulated data in the manner described above under a simple model of extreme population decline (from 100,000 ancestral individuals to 1000) using both MaCS and MSMS (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ewing and Hermisson 2010</a>) (which does not use the SMC′ approximation) and ran it through the same LD decay pipeline used for our other simulated data (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-7" class="subsection"><h3>SFS predicted by demographic models</h3><p id="p-29">We used the diffusion approximation in ∂a∂i (<a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>) to calculate the expected SFSs under the Gutenkunst, MSMC 2-haplotype, MSMC 4-haplotype, MSMC 8-haplotype, and SMC++ models for the CEU, CHB, and YRI populations. We compared the SFSs expected under each of these models both to the empirical SFS used by <a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> to infer the demographic histories of these three populations [“observed (Gutenkunst),” <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>], as well as to the SFSs based on low-coverage 1000 Genomes whole genome sequencing data [“1000 Genomes (whole genome),” <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>] and SFSs based on putatively neutral regions in the 1000 Genomes data set [“1000 Genomes (neutral)”, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>]. We assessed the fit of different models to the observed SFS by comparing their log-likelihoods (see below, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, Supplementary Note 4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>, and Tables S2–S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487214/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487214/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Multinomial log-likelihoods comparing the fit of various models to the observed SFS derived from Sanger sequencing data and used by <a id="xref-ref-30-13" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> for their inference (SFSs in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>)</span><div class="sb-div caption-clear"></div></div></div><div id="sec-8" class="subsection"><h4>Empirical SFSs:</h4><p id="p-31">The primary empirical SFSs used in our comparisons were produced by <a id="xref-ref-30-16" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> and used to infer the joint demographic histories of CEU, CHB, and YRI populations in their study [observed (Gutenkunst)]. As described in their supplementary material, the joint SFS represents 4.04 Mb of Sanger sequencing data from 12 YRI, 12 CHB, and 22 CEU individuals projected down to 20 chromosomes for a total of 17,446 segregating SNPs polarized against chimp, with a correction for ancestral misidentification applied. We marginalized the SFS using ∂a∂i (<a id="xref-ref-30-17" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>) to have one SFS per population (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-32">To make sure our results were consistent with SFSs derived from other sequencing methodologies and different genomic regions, we also generated folded proportional genome-wide and neutral SFSs from the 1000 Genomes data described above [1000 Genomes (whole genome) and 1000 Genomes (neutral)] (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2015</a>; <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>, Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>, and Supplementary Note 3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-9" class="subsection"><h4>Expected SFSs under published demographic models:</h4><p id="p-33">Expected SFSs for a sample size of 10 diploid individuals were calculated in ∂a∂i (<a id="xref-ref-30-18" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>) for each of the published demographic models extrapolating calculations across three grid points (40, 50, and 60) (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). To test whether the effect of differences in mutation rate between the studies may be responsible for discrepancies, we also considered an alternative scaling of the MSMC models using a higher mutation rate (Supplementary Note 5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-34">We generated both the proportional (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> and Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>) and absolute (<em>i.e.</em>, SFS based on SNP counts) SFSs (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> and Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File</a>). The proportional SFS was calculated by dividing each bin of the SFS output by ∂a∂i by the sum of the bins. The absolute SFS was calculated by scaling the SFS output by ∂a∂i (which is relative to <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-2.gif"/></noscript></span></span>) by<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-3.gif"/></noscript></span></span>where <em>N<sub>Ai</sub></em> is the oldest ancestral size inferred in each model and <em>L</em> is the sequence length (4.04 Mbp) in <a id="xref-ref-30-19" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a>. <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-4.gif"/></noscript></span></span> for the Gutenkunst model used the authors’ preferred mutation rate of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-5.gif"/></noscript></span></span> = 2.35 × 10<sup>−8</sup> mutations per base per generation, and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-6.gif"/></noscript></span></span> for the MSMC and SMC++ models used the authors’ preferred mutation rate of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-7.gif"/></noscript></span></span> = 1.25 × 10<sup>−8</sup> mutations per base per generation (see Supplementary Note 5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a> for scaling using alternate mutation rates).</p></div><div id="sec-10" class="subsection"><h4>Assessing SFS fit:</h4><p id="p-35">Log-likelihoods were calculated for each proportional SFS relative to each of the three observed SFSs [observed (Gutenkunst), 1000 Genomes (whole genome), and 1000 Genomes (neutral)] using a multinomial log-likelihood (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, Supplementary Note 4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>, and Tables S2 and S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The fit of different models was compared by examining their decrease in log-likelihood compared to that of each of the observed SFSs to itself (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, Supplementary Note 4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>, and Tables S2 and S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). Due to the uncertainty of singleton SNP calls using high-throughput sequencing data, log-likelihoods were calculated both with singletons and with the SFS renormalized without the singletons category when comparing to the 1000 Genomes SFSs (Figure S7 and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-36">Log-likelihoods were calculated for each absolute SFS (in terms of SNP counts) using a Poisson likelihood relative to the observed (Gutenkunst) SFS (Supplementary Note 4 and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div></div><div id="sec-11" class="subsection"><h3>Effect of uncertainty in ancestral population size</h3><p id="p-37">To investigate whether changing the ancestral population size (<em>N<sub>A</sub></em>) in the MSMC trajectories would result in SFSs that better fit the observed SFS, we adjusted the CEU MSMC 2-haplotype model to have a variety of <em>N<sub>A</sub></em> values. We also trimmed the model to remove ancient events (older than 225.5 KYA) to better match the time period (in years) encompassed by the <a id="xref-ref-30-20" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em>’s (2009)</a> model. These adjusted stepwise models were then used to calculate the expected SFS in ∂a∂i, as above. Supplementary Note 7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a> describes the values of <em>N<sub>A</sub></em> used when testing the trimmed and untrimmed models (Figures S10–S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-12" class="subsection"><h3>MSMC population size trajectories for demographic models inferred from the SFS</h3><p id="p-38">To determine whether MSMC is capable of inferring a demography as complex as the one inferred in the Gutenkunst model, we used coalescent simulations to generate long chromosomal sequence data for each population under the <a id="xref-ref-30-21" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> inferred demographic model [see <a id="xref-ref-30-22" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em>’s (2009)</a> figure 2B and table 1 for full model], and then ran MSMC on these simulated data sets to assess whether the program is capable of recovering the underlying demographic model.</p><p id="p-39">Simulations were carried out using MaCS (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2009</a>). For each population, we simulated 50 replicate “genomes,” made up of 80 independent 30-Mb “chromosomes,” each made up of 300 linked 100-kb recombination blocks, with per-block recombination rates calculated by <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Phung <em>et al.</em> (2016)</a> from the pedigree-based genetic map assembled by the deCODE project (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Kong <em>et al.</em> 2010</a>).</p><p id="p-40">Each simulated genome was then used for a separate MSMC inference, using the default parameters (<a id="xref-ref-82-12" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). To determine whether these inferred MSMC trajectories would lead to SFSs matching those predicted by <a id="xref-ref-30-23" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em>’s (2009)</a> model, the MSMC trajectories were averaged and the average was converted into a stepwise ∂a∂i model. This model was then used to calculate the expected SFS under the averaged model based on simulated data (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7, B and C</a>). The multinomial and Poisson log-likelihoods for the proportional and SNP count SFSs were calculated as described in Supplementary Note 4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a> (Tables S2 and S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><div id="sec-13" class="subsection"><h4>Extreme recent growth and Neanderthal admixture:</h4><p id="p-41">We simulated data under more complex demographic histories, first to explore the relative abilities of the MSMC 2-haplotype and 8-haplotype models to infer extreme recent growth, and then to determine whether the addition of Neanderthal admixture may lead to MSMC trajectories resembling those inferred from real data by <a id="xref-ref-82-13" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a> (Supplementary Note 6 and Figures S8 and S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-42">All code to simulate data under each demographic model, calculate heterozygosity, and generate the SFS from simulated and empirical data are available on GitHub: github.com/LohmuellerLab/Compare_Demographic_Models.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><p id="p-43">We compared published models of demography for three human populations (CEU, CHB, and YRI) inferred using different methods for demographic inference: (1) using the SFS in ∂a∂i (Gutenkunst) (<a id="xref-ref-30-24" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>), (2) using whole genomes in MSMC (MSMC 2-, 4-, and 8-haplotype) (<a id="xref-ref-82-14" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>), and (3) using a combined SFS plus whole genome approach in SMC++ (SMC++) (<a id="xref-ref-95-9" class="xref-bibr" href="#ref-95">Terhorst <em>et al.</em> 2017</a>). The evaluation of the MSMC models involves three models per population because <a id="xref-ref-82-15" class="xref-bibr" href="#ref-82">Schiffels and Durbin’s (2014)</a> inference was carried out using two, three, or eight chromosomal haplotypes (from one, two, and four individuals), sometimes resulting in fundamentally different demographic parameter estimates. We evaluated whether the method’s performance was improved using certain numbers of haplotypes.</p><div id="sec-16" class="subsection"><h3>Heterozygosity predicted by demographic models</h3><p id="p-44">The distribution of expected heterozygosity across 100- and 10-kb blocks was calculated from data simulated under each published demographic model for each of the three populations and compared to empirical distributions of heterozygosity based on whole genome and putatively neutral sequence data from the 1000 Genomes Project.</p><p id="p-45">We find that the Gutenkunst demographic model inferred from the SFS, the MSMC 2-haplotype model, and the SMC++ model all yielded distributions of heterozygosity that resemble the empirical whole genome distribution of heterozygosity, with MSMC 2-haplotype fitting the mean most closely (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). However, we found that the higher haplotype MSMC models (MSMC 4-haplotype and 8-haplotype) yielded distributions of heterozygosity that were highly divergent from the empirical distribution (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-46">The MSMC 4-haplotype models fit worst due to their extremely high inferred ancestral size across all three populations (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>) (CEU, 187,514; CHB, 191,238; YRI, 205,845 individuals; compared to 4000–40,000 individuals in the other models), with mean whole genome heterozygosity distributions nearly seven times larger than that of the empirical whole genome distribution (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The MSMC 8-haplotye model for YRI infers a similarly large ancestral size and has a similarly high mean heterozygosity as the 4-haplotype YRI model. The MSMC 8-haplotype models for CEU and CHB, however, infer much lower ancestral sizes (CEU, 2147; CHB, 5666) (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). Due to the low ancestral size, these models also do not fit the empirical distribution well, yielding distributions of heterozygosity with means that are two to four times lower than the empirical distributions.</p><p id="p-47">When examining the 1000 Genomes data, we found that heterozygosity in the neutral regions was higher than that seen for the genome-wide distribution of heterozygosity calculated in 10-kb windows (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>; <em>e.g.</em>, CEU mean heterozygosity per site for whole genome: 7.8 × 10<sup>−4</sup> <em>vs.</em> neutral: 9.4 × 10<sup>−4</sup>), suggesting that natural selection has directly and/or indirectly affected genome-wide patterns of heterozygosity. When the published demographic models were compared to the neutral heterozygosity distributions, we found similar trends to those seen for the whole genome data (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-17" class="subsection"><h3>LD predicted by demographic models</h3><p id="p-48">None of the published demographic models could perfectly recapitulate the empirical LD decay curve (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For SNP pairs &lt;10 kb apart, the MSMC-8 haplotype model comes closest to the empirical curve for the CEU and CHB populations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, A and B</a>), but underestimates the amount of LD, while all other models predict too much LD. The Gutenkunst and SMC++ models predict similar LD curves and are closer to the empirical curve than the MSMC 2-haplotype and 4-haplotype models. For YRI SNP pairs &lt;10 kb apart, SMC++ and MSMC 8-haplotype predict similar LD decay curves and are close to the empirical distribution, with Gutenkunst still fitting better than MSMC 2-haplotype and 4-haplotype (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). At distances &gt;10 kb apart, all demographic models predict there to be more LD than seen in the empirical data (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LD decay patterns. LD decay was calculated across 100-kb windows from 1000 Genomes data and simulated data under each demographic model for (A) CEU, (B) CHB, and (C) YRI. Pairs of SNPs are binned based on physical distance (bp) between them, up to 51 kb. Average genotype r2 is calculated within each distance bin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884691631" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LD decay patterns. LD decay was calculated across 100-kb windows from 1000 Genomes data and simulated data under each demographic model for (A) CEU, (B) CHB, and (C) YRI. Pairs of SNPs are binned based on physical distance (bp) between them, up to 51 kb. Average genotype &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; is calculated within each distance bin.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3605/F3.medium.gif" width="308" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/11/3605/F3.medium.gif" width="308" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487208" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-49" class="first-child">LD decay patterns. LD decay was calculated across 100-kb windows from 1000 Genomes data and simulated data under each demographic model for (A) CEU, (B) CHB, and (C) YRI. Pairs of SNPs are binned based on physical distance (bp) between them, up to 51 kb. Average genotype <em>r</em><sup>2</sup> is calculated within each distance bin.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">We found that the lack of fit is not due to the use of the SMC′ approximation in the simulator MaCS (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2009</a>), as both MaCS and MSMS (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Ewing and Hermisson 2010</a>), a coalescent simulator which does not use the SMC′ approximation, yielded highly similar LD decay curves when simulating data under the same simple population contraction model (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-18" class="subsection"><h3>SFS predicted by demographic models</h3><p id="p-51">Lastly, we examined which of the demographic models could match the SFS of the empirical data. To account for the possibility of overfitting the SFS-based Gutenkunst model to the SFS it was inferred from, we also compared all models to empirical SFSs based on low-coverage, high-throughput 1000 Genomes sequence data from the same three populations.</p><div id="sec-19" class="subsection"><h4>Comparing to the observed Gutenkunst SFS:</h4><p id="p-52">For each population, the SFSs predicted by the three MSMC models do not match the empirical proportional SFS from <a id="xref-ref-30-25" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a>, regardless of the mutation rate or number of genomes used (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, and Figure S5 and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The expected SFS based on the <a id="xref-ref-30-26" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> demographic history matches the observed SFS closely, being only 9 log-likelihood units worse than the best possible fit (comparing the empirical SFS to itself) for CEU, 48 units worse for CHB, and 17 units worse for YRI (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). In comparison, the best-fitting MSMC models for each population are 152, 188, and 373 log-likelihood units below the best possible fit (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). The combined whole genome plus SFS method SMC++ has an intermediate fit, with a log-likelihood well below the Gutenkunst model, but consistently better than any of the MSMC models (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Unfolded proportional site frequency spectra for (A) CEU, (B) CHB (B), and (C) YRI populations. The &#x201C;observed&#x201D; SFS is from noncoding sequence used by Gutenkunst et al. (2009) to infer demographic histories for these three populations. See Figure S5 in File S1 for scaling using alternative mutation rates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884691631" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Unfolded proportional site frequency spectra for (A) CEU, (B) CHB (B), and (C) YRI populations. The &#x201C;observed&#x201D; SFS is from noncoding sequence used by Gutenkunst &lt;em&gt;et al.&lt;/em&gt; (2009) to infer demographic histories for these three populations. See Figure S5 in File S1 for scaling using alternative mutation rates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3605/F4.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/11/3605/F4.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487209" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-53" class="first-child">Unfolded proportional site frequency spectra for (A) CEU, (B) CHB (B), and (C) YRI populations. The “observed” SFS is from noncoding sequence used by <a id="xref-ref-30-27" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> to infer demographic histories for these three populations. See Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a> for scaling using alternative mutation rates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Interestingly, there is not a consistent improvement in fit to the observed SFS when increasing the number of individuals used for the MSMC inference. For each population, the 4-haplotype model has the worst fit (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). For CEU and YRI, the MSMC 2-haplotype models fit best out of the MSMC models, but both are &gt;100 log-likelihood units worse than the Gutenkunst model. For CHB, the 8-haplotype model fits best, but is still 140 units worse than the Gutenkunst model (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-55">The above comparisons considered the proportions of SNPs at specific frequencies in the sample. We also performed a comparison of the number of SNPs in each bin of the SFS, the absolute SFS, to the observed absolute SFS used in <a id="xref-ref-30-28" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em>’s (2009)</a> inference using a Poisson likelihood. The absolute SFS expected under the <a id="xref-ref-30-29" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> model fits the observed SFS best (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a> and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>), and is only 9, 49, and 17 log-likelihood units below the best possible fits for CEU, CHB, and YRI models, respectively. The SMC++ models have the next-best fit to the absolute SFS, but come 86 (CEU), 176 (CHB), and 193 (YRI) log-likelihood units below the best possible fit; followed by MSMC 2-haplotype which fell 278 (CEU), 378 (CHB), and 455 (YRI) log-likelihood units below the optimal fit (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). In all three populations, the MSMC 4-haplotype and 8-haplotype models are thousands of log-likelihood units worse than the best possible fit, showing no improvement based on using a larger number of individuals in the inference (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The overestimation of SNPs in the 4-haplotype model is due to the model’s extremely high predicted ancestral size (∼200,000 individuals for each population) (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP count SFSs using the counts of SNPs for the (A) CEU, (B) CHB, and (C) YRI populations. The &#x201C;observed&#x201D; SFS is from noncoding sequence used by Gutenkunst et al. (2009) to infer demographic histories for these three populations. SFSs are scaled using the ancestral population size given by each model, the mutation rate used to scale each model by the authors, and the sequence length of the empirical data set (4.04 Mb). See Figure S6 in File S1 for scaling using alternative mutation rates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884691631" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNP count SFSs using the counts of SNPs for the (A) CEU, (B) CHB, and (C) YRI populations. The &#x201C;observed&#x201D; SFS is from noncoding sequence used by Gutenkunst &lt;em&gt;et al.&lt;/em&gt; (2009) to infer demographic histories for these three populations. SFSs are scaled using the ancestral population size given by each model, the mutation rate used to scale each model by the authors, and the sequence length of the empirical data set (4.04 Mb). See Figure S6 in File S1 for scaling using alternative mutation rates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3605/F5.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/11/3605/F5.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487255" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-56" class="first-child">SNP count SFSs using the counts of SNPs for the (A) CEU, (B) CHB, and (C) YRI populations. The “observed” SFS is from noncoding sequence used by <a id="xref-ref-30-30" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> to infer demographic histories for these three populations. SFSs are scaled using the ancestral population size given by each model, the mutation rate used to scale each model by the authors, and the sequence length of the empirical data set (4.04 Mb). See Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a> for scaling using alternative mutation rates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">For both the proportional and absolute SFSs, we found that rescaling the models using a higher mutation rate did not produce large qualitative differences in how the MSMC models fit the observed (Gutenkunst) SFS (Supplementary Note 5 and Figures S4–S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-20" class="subsection"><h4>Comparing to the folded low-coverage 1000 Genomes SFS:</h4><p id="p-58">To avoid giving the Gutenkunst model an unfair advantage by fitting all models to the SFS used to infer that particular model, we also compared all models to proportional folded SFSs based on whole genome and neutral data from the 1000 Genomes Project (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> and Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The fit to the empirical singletons bin was poor for all models, except for SMC++, which was, in part, fit to an SFS based on 1000 Genomes data. Calling singletons is notoriously difficult in low-coverage data, making that bin the least reliable in the 1000 Genomes data (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Kim <em>et al.</em> 2011</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Nielsen <em>et al.</em> 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Han <em>et al.</em> 2014</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2015</a>). We therefore calculated likelihoods for all models relative to the data both with singletons included and again with the SFSs renormalized without the singletons category (Figure S7 and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Folded proportional SFSs for (A) CEU, (B) CHB, and (C) YRI populations. The &#x201C;1000 Genomes (WG)&#x201D; SFS is from low-coverage whole genome 1000 Genomes data, and the &#x201C;1000 Genomes (Neut)&#x201D; SFS is from 6333 &#xD7; 10-kb putatively neutral regions in the 1000 Genomes data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884691631" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Folded proportional SFSs for (A) CEU, (B) CHB, and (C) YRI populations. The &#x201C;1000 Genomes (WG)&#x201D; SFS is from low-coverage whole genome 1000 Genomes data, and the &#x201C;1000 Genomes (Neut)&#x201D; SFS is from 6333 &#xD7; 10-kb putatively neutral regions in the 1000 Genomes data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3605/F6.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/11/3605/F6.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487264" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-59" class="first-child">Folded proportional SFSs for (A) CEU, (B) CHB, and (C) YRI populations. The “1000 Genomes (WG)” SFS is from low-coverage whole genome 1000 Genomes data, and the “1000 Genomes (Neut)” SFS is from 6333 × 10-kb putatively neutral regions in the 1000 Genomes data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">For YRI, the Gutenkunst model is the best-fitting model for the whole genome and neutral 1000 Genomes SFSs, both with and without singletons, with all other models having a much worse fit (the next best model, SMC++, is hundreds to thousands of log-likelihood units below the fit of the Gutenkunst model) (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a> and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). For CEU and CHB, if singletons are included, SMC++ fits the whole genome and neutral 1000 Genomes SFSs best. For CEU, the Gutenkunst model then fits second best, with the MSMC models far behind (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6A</a> and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). For CHB, the MSMC 2-haplotype fits second best after SMC++, with the Gutenkunst model coming third, but both are &gt;10,000 log-likelihood units below SMC++ (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6B</a> and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). If singletons are excluded for CEU and CHB, then the Gutenkunst model fits best, with SMC++ coming in second, and the MSMC models all ranking far below (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div></div><div id="sec-21" class="subsection"><h3>Effect of uncertain ancestral population size</h3><p id="p-61">The accuracy of ancient ancestral population sizes, particularly &gt;3 MYA (&gt;100,000 generations), using the whole genome-based methods remains unclear (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>). As discussed above, the MSMC 2-haplotype and 4-haplotype models infer large ancestral sizes for each population that are not supported by previous inferences of human demographic history (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Adams and Hudson 2004</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Keinan <em>et al.</em> 2007</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boyko <em>et al.</em> 2008</a>; <a id="xref-ref-30-31" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Nielsen <em>et al.</em> 2009</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Gravel <em>et al.</em> 2011</a>). We hypothesized that these extreme ancestral sizes, as well as ancient bottlenecks and population growth (the signature humps of MSMC trajectories), which do not appear in demographic models inferred using other methods, could be artifacts that are causing the SFS predicted by these models to deviate from the true SFS.</p><p id="p-62">To test this hypothesis, we took the best fitting of the MSMC models, the CEU 2-haplotype model, and carried out a series of adjustment experiments to determine whether changes to the model could provide a better fit to the observed SFS. Without adjusting the time period encompassed by the model, we altered the ancestral population size to a variety of values including those inferred by <a id="xref-ref-30-32" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> (Supplementary Note 7 and Figures S10 and S11 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). We also truncated the MSMC trajectory to remove ancient events and better match the time period (in years) encompassed by the <a id="xref-ref-30-33" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> model. We again adjusted the ancestral population size to a variety of plausible values (Supplementary Note 7 and Figures S12 and S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-63">We found trimming away the ancient (older than ∼225 KYA) part of the demographic trajectory and lowering the ancestral population size to 10,000–12,300 (compared to 41,261 inferred initially) dramatically improved the fit of the proportional SFSs predicted under these adjusted models to the observed (Gutenkunst) SFS (Figure S12 and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The best-fit model with ancestral size (<em>N<sub>A</sub></em>) equal to 12,300 was brought to within 38 log-likelihood units of the best possible likelihood (Figure S12D and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>), only 29 units below the Gutenkunst model. When repeating this procedure using the SFS based on counts, the SFSs under these adjusted models showed a different pattern of improvement. Here the untrimmed models that did <em>not</em> have ancient events &gt;225 KYA trimmed away, but had a lowered ancestral population size of 7300–12,300, showed the most improvement (Figures S11 and S12 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). However, their fit was still &gt;100 log-likelihood units worse than the Gutenkunst model (Figure S12 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-22" class="subsection"><h3>MSMC population size trajectories for demographic models inferred from the SFS</h3><p id="p-64">Given that the SFSs predicted by the demographic models inferred using MSMC do not fit the observed SFS, we examined whether MSMC is capable of recovering a complex demography such as the one inferred by <a id="xref-ref-30-34" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> from a single simulated genome. We find that MSMC performs relatively well at inferring the underlying demography from the simulated data. <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a> shows the underlying Gutenkunst demographic model for each population (purple) (as in the other Gutenkunst model simulations, migration is included in the model, but is not depicted in our diagrams), with the results of 50 independent MSMC inferences on each 2-haplotype simulated data set coming close to the underlying demography. However, sharp bottlenecks are inferred as long population declines (as noted by <a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a> and <a id="xref-ref-82-16" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>). Additionally, we found evidence of MSMC detecting a false spurt of growth in the YRI population 1350 generations ago (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a>). Both of these phenomena were also noted by <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Bunnefeld <em>et al.</em> (2015)</a>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MSMC 2-haplotype can accurately infer the demographic model predicted by Gutenkunst et al. (2009). (A) The results of running MSMC 2-haplotype on 50 independent two-haplotype data sets simulated under the Gutenkunst et al. (2009) model of human demographic history (Gutenkunst, heavy purple line). The resulting MSMC 2-haplotype trajectories (&#x201C;MSMC Sim. Gut. Data,&#x201D; fine pink lines) show the MSMC trajectories inferred from these 50 data sets. Note that these trajectories accurately track the demographic model used to simulate the data. (B) and (C) show proportional and SNP count SFSs for each population, respectively. The gray bars (observed) denote the empirical SFS used by Gutenkunst et al. (2009). The purple bars denote the expected SFS under the inferred Gutenkunst demographic models. The pink bars denote the expected SFS under the average of the 50 MSMC 2-haplotype demographic model trajectories for each population. Note that these three SFSs agree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884691631" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MSMC 2-haplotype can accurately infer the demographic model predicted by Gutenkunst &lt;em&gt;et al.&lt;/em&gt; (2009). (A) The results of running MSMC 2-haplotype on 50 independent two-haplotype data sets simulated under the Gutenkunst &lt;em&gt;et al.&lt;/em&gt; (2009) model of human demographic history (Gutenkunst, heavy purple line). The resulting MSMC 2-haplotype trajectories (&#x201C;MSMC Sim. Gut. Data,&#x201D; fine pink lines) show the MSMC trajectories inferred from these 50 data sets. Note that these trajectories accurately track the demographic model used to simulate the data. (B) and (C) show proportional and SNP count SFSs for each population, respectively. The gray bars (observed) denote the empirical SFS used by Gutenkunst &lt;em&gt;et al.&lt;/em&gt; (2009). The purple bars denote the expected SFS under the inferred Gutenkunst demographic models. The pink bars denote the expected SFS under the average of the 50 MSMC 2-haplotype demographic model trajectories for each population. Note that these three SFSs agree.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/11/3605/F7.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/11/3605/F7.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/11/3605/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487284" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-65" class="first-child">MSMC 2-haplotype can accurately infer the demographic model predicted by <a id="xref-ref-30-35" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a>. (A) The results of running MSMC 2-haplotype on 50 independent two-haplotype data sets simulated under the <a id="xref-ref-30-36" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> model of human demographic history (Gutenkunst, heavy purple line). The resulting MSMC 2-haplotype trajectories (“MSMC Sim. Gut. Data,” fine pink lines) show the MSMC trajectories inferred from these 50 data sets. Note that these trajectories accurately track the demographic model used to simulate the data. (B) and (C) show proportional and SNP count SFSs for each population, respectively. The gray bars (observed) denote the empirical SFS used by <a id="xref-ref-30-37" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a>. The purple bars denote the expected SFS under the inferred Gutenkunst demographic models. The pink bars denote the expected SFS under the average of the 50 MSMC 2-haplotype demographic model trajectories for each population. Note that these three SFSs agree.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">The SFSs predicted by the demographic models inferred using MSMC on the simulated data fit the SFS expected under the Gutenkunst model and the observed Gutenkunst SFSs better than the MSMC demographic models inferred by <a id="xref-ref-82-17" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a> (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7, B and C</a>). The proportional MSMC simulated data SFSs were only 40, 74, and 10 log-likelihood units below the Gutenkunst model SFS (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>), with the SFSs based on SNP counts showing a similar pattern (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). Therefore, if the Gutenkunst model is the true demographic model for human history, MSMC accurately captures the population size changes and produces an appropriate SFS.</p><p id="p-67">It is well established that two-haplotype, whole genome-based inference (PSMC and MSMC 2-haplotype, also known as PSMC′) is not able to detect recent demographic events (<a id="xref-ref-50-8" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>; <a id="xref-ref-82-18" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>). However, the ability to detect recent growth by using more than two haplotypes in the inference is cited as a feature of MSMC (<a id="xref-ref-82-19" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>). We ran MSMC 2-haplotype and 8-haplotype on data sets simulated under the Gutenkunst model and a Gutenkunst model plus extreme recent growth (Figure S8 and Supplementary Note 6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). Unsurprisingly, MSMC 2-haplotype was not able to detect extreme recent growth. Its estimates of current population size were fairly accurate for the original Gutenkunst model (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7A</a>), but the method dramatically underestimated the growth for data simulated under the Gutenkunst plus growth model (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The results from 8-haplotype MSMC inference were most surprising. We found that for both models, MSMC 8-haplotype inferred extreme recent growth as many as four orders of magnitude beyond that in the underlying model, with a high degree of variance between replicates (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). Despite the high degree of variance, the average of the MSMC trajectories all showed a strong upward bias in estimates of the recent past (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). While the ability to detect recent growth is meant to be a feature of MSMC, our findings indicate that the magnitude of growth may not be estimated well.</p><p id="p-68">We had hypothesized that Neanderthal admixture could cause deviation between the MSMC and Gutenkunst demographic models, but found that the addition of Neanderthal admixture to our Gutenkunst model simulations did not substantively change the MSMC trajectories or expected SFSs (Figure S9, Supplementary Note 6, and Tables S2 and S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>).</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><p id="p-69">We tested which published models of human demographic history, inferred using either whole genome sequence data, the SFS, or a combined approach, can recapitulate multiple summaries of human genetic variation data. We found that no model was able to recapitulate all summaries of the data, but some models still performed better than others. In particular, none of the models was able recapitulate LD decay, but the Gutenkunst SFS-based models and the combined whole genome and SFS-based SMC++ models were able to recapitulate empirical heterozygosity and the SFS. MSMC 2-haplotype was able to recapitulate heterozygosity, but not the SFS, and MSMC 4-haplotype and 8-haplotype could fit neither heterozygosity nor the SFS, though MSMC 8-haplotype did fit LD decay slightly better than the other models. These results highlight the uncertainties of demographic inference from one, or even two, types of data and the need to assess the fit of demographic models using multiple summaries of the data.</p><p id="p-70">We found that the models based on MSMC inference from four or eight haplotypes did not improve the fit of the expected SFS compared to that based on two haplotypes; in fact, in most cases the 4- and 8-haplotype models fit much worse than the 2-haplotype models. The 4-haplotype models for CEU, CHB, and YRI and the 8-haplotype model for YRI appear to fit poorly due to their extremely high ancestral sizes and ancient humps of growth and decline (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>). The expected SFSs under the 8-haplotype models for CEU and CHB show a skew toward low-frequency variants that may be due to their low ancestral size followed by extreme recent growth (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>). We find that MSMC 8-haplotype vastly overestimates recent growth in simulated data, which may be contributing to the lack of fit to the SFS (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). This result is at odds with the findings of <a id="xref-ref-82-20" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a>, who suggested that using eight haplotypes instead of two should increase accuracy of population size inference in the recent past, though they also noted a bias toward smaller ancient population sizes when using an increased number of haplotypes. Changing the scaling of the mutation rate did not generally help the MSMC models to fit the expected SFS better (Figures S4–S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). It is worth noting that the model inferred in SMC++ used the same mutation rate as MSMC, yet fit the empirical SFSs much better (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>, and Tables S2–S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>), indicating that mutation rate differences between the whole genome- and SFS-based studies is not the source of the discrepancies.</p><p id="p-71">We found that in addition to not fitting the empirical SFS, the MSMC 4-haplotype and 8-haplotype models did not predict the genome-wide distribution of heterozygosity (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), which may be surprising as the genome-wide distribution of heterozygosity is a major feature of the data used by MSMC. As with the SFS, the reason for the lack of fit for these models appears to be the extremely high ancestral size inferred in the 4-haplotype models for all three populations and in the 8-haplotype YRI model, and the low ancestral size inferred in the 8-haplotype models for CEU and CHB (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-72">Since the most ancient size in the MSMC trajectory will have a large influence on heterozygosity and the SFS and the most ancient bin of the MSMC trajectory may be unreliable (<a id="xref-ref-50-9" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>; <a id="xref-ref-82-21" class="xref-bibr" href="#ref-82">Schiffels and Durbin 2014</a>), we explored the effect of altering this ancient size and removing ancient growth events in the CEU MSMC 2-haplotype model. We found that selective trimming could improve the fit to the SFS (Figures S10–S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). However, the final bin of the model cannot explain all of the lack of fit of the MSMC models to the data as the CEU and CHB MSMC 8-haplotype trajectories do not show the extreme ancestral sizes in the last bin, yet these models also dramatically deviate from empirical heterozygosity and the SFS. In other words, simple exclusion of the final high ancestral size is not sufficient to improve model fit to other summaries of the data. Our trimming experiments were only made possible by the abundance of human sequence data and demographic models previously fit to the data. Since many MSMC trajectories are calculated for species for which there is no prior information about ancient demographic history, the “informed trimming” we carried out is not a practicable solution to improve the reliability of MSMC inference.</p><p id="p-73">While our results indicate that features of MSMC trajectories, particularly ancient events, should be regarded with caution, we also found that MSMC 2-haplotype is able to accurately recapitulate a complex demography (with the exception of steep drops in population size, extreme recent growth, and some false periods of growth) from simulated data, supporting the validity of the method, at least for use on simulated data (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7</a>). Migration between populations did not appear to cause deviations in MSMC trajectories from the underlying model (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7</a>), nor did a small degree of Neanderthal admixture (Figure S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>), indicating that MSMC is robust to small amounts of gene flow. The fact that the 2-haplotype model based on real data did not fit the observed SFS very well (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>, and Tables S2–S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>) suggests that the true underlying pattern of human demography is more complex than either type of inference (∂a∂i or MSMC) is capturing, potentially revealing weaknesses in both methods.</p><p id="p-74">Alternatively, if the <a id="xref-ref-30-38" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> demographic model is largely accurate, biases or other factors that exist in real data but not in simulated data may be affecting MSMC inference, resulting in the method failing to recover an underlying demography that matches <a id="xref-ref-30-39" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em>’s (2009)</a> model. For example, <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Song <em>et al.</em> (2016)</a> found that statistical phasing could affect MSMC estimates of population split times; and <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Nadachowska-Brzyska <em>et al.</em> (2016)</a> found that per-site sequencing depth, mean genome coverage, and the amount of missing data led to differences in PSMC curve amplitudes, expansions and contractions, and the timing and values of <em>N<sub>e</sub></em>. They therefore recommended only using samples with a mean genome coverage of ≥18× and &lt;25% missing data, and employing a per-site sequencing depth filter of ≥10 (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Nadachowska-Brzyska <em>et al.</em> 2016</a>). The Complete Genomics genomes used by <a id="xref-ref-50-10" class="xref-bibr" href="#ref-50">Li and Durbin (2011)</a> were &gt;40× coverage (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Drmanac <em>et al.</em> 2010</a>), indicating that lack of coverage is not responsible for their divergence from estimates based on the SFS. However, the standards suggested by <a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Nadachowska-Brzyska <em>et al.</em> (2016)</a> may not always be attainable in <em>de novo</em> genome projects and, thus, data quality issues may affect nonmodel organism PSMC and MSMC inferences more acutely. Future work should also examine the impact of artifacts of genome assembly errors and structural variants on PSMC inference. For example, collapsing duplicate regions of the genome on top of each other could result in regions of the genome having excess heterozygosity, which could in turn affect inference of demography.</p><p id="p-75">We found that no model was able to accurately recapitulate the empirical distribution of LD decay. The lack of fit of the SFS-based models is perhaps unsurprising, as <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Harris and Nielsen (2013)</a> found that the Gutenkunst model cannot recapitulate empirical IBS distributions (a finer-scale summary of the data related to LD), and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Garud <em>et al.</em> (2015)</a> found that they could not recover empirical LD patterns in <em>Drosophila</em>, despite matching the SFS, number of segregating sites (S), and number of pairwise differences <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-8.gif"/></noscript></span></span> <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Garud <em>et al.</em> (2015)</a> suggested the lack of fit could either be due to linked positive selection or to an incompleteness of the demographic model, demonstrating how models that fit some summaries of the data may not recapitulate others. It is more surprising that the MSMC 2-haplotype and 4-haplotype models do not fit the data well, as the method uses LD information in its inference, though different summaries of LD may be affected by demography in distinct ways (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Plagnol and Wall 2006</a>). Other possible factors that could lead to the lack of fit of all models to empirical LD decay patterns include the absence of natural selection, gene conversion, and fine-scale recombination hotspots in our simulations (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ardlie <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Frisse <em>et al.</em> 2001</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Wall and Pritchard 2003</a>). Further, if the true mutation rate is actually smaller than the relatively high value used by Gutenkunst <em>et al.</em> (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-9.gif"/></noscript></span></span> 2.35 × 10<sup>−8</sup> mutations/bp/generation), then the population sizes would have to be larger than those estimated by <a id="xref-ref-30-40" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a>. Larger population sizes would yield larger values of the population scaled recombination rate <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-10.gif"/></noscript></span></span> than what was used in our simulations under the Gutenkunst model. Larger values of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/11/3605/embed/mml-math-11.gif"/></noscript></span></span> would then lead to a decrease in LD in the simulations, which might better match the empirical LD decay curves.</p><p id="p-76">Natural selection may affect both SFS- and whole genome-based methods of demographic inference. <a id="xref-ref-50-11" class="xref-bibr" href="#ref-50">Li and Durbin (2011)</a> found that masking exonic sequence did not alter PSMC trajectories. However, <a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Schrider <em>et al.</em> (2016)</a> examined the impact of selective sweeps on demographic inference using the SFS in ∂a∂i, approximate Bayesian computation (ABC), and PSMC and found that all three methods were influenced to varying degrees and in slightly different directions by the presence of selective sweeps, with ∂a∂i the most robust to these effects. This is a concern for published human demographic models as <a id="xref-ref-30-41" class="xref-bibr" href="#ref-30">Gutenkunst <em>et al.</em> (2009)</a> used noncoding sequence from autosomal genes in their study, which may be subject to linked selection (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Gazave <em>et al.</em> 2014</a>; <a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Schrider <em>et al.</em> 2016</a>). <a id="xref-ref-82-22" class="xref-bibr" href="#ref-82">Schiffels and Durbin (2014)</a> used whole genome sequences that included genic and nongenic regions, some of which are certainly under selection. Thus, the sensitivity of these methods to selection may partially explain why both perform well on simulated data without selection, yet have such divergent results when run on empirical data.</p><p id="p-77">Our results have implications for understanding human demographic history. First, there has been controversy concerning the presence of ancient bottlenecks (&gt;100 KYA) in human populations (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Takahata <em>et al.</em> 1995</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Harpending <em>et al.</em> 1998</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Takahata and Satta 1998</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hawks <em>et al.</em> 2000</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Garrigan and Hammer 2006</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fagundes <em>et al.</em> 2007</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Scholz <em>et al.</em> 2007</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blum and Jakobsson 2011</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Sjödin <em>et al.</em> 2012</a>). The inferred humps in the ancient portions of MSMC plots (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a>) tended to lend support to these ancient population size changes that appeared to be absent from SFS demographic estimates. Our results suggest that if these ancient population size changes did indeed occur, the resulting SFS would appear very different from the SFSs seen in human populations (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6</a>, and Figures S10–S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1/FileS1.pdf">File S1</a>). The fact that they are not seen in the observed SFS suggests that either the size changes did not occur, and the inferred size changes are artifacts, or instead, the true demography is more complex than currently modeled using either approach. Our conclusion of finding little evidence for the ancient population size changes is supported by the study of <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Sjödin <em>et al.</em> (2012)</a>. They employed an ABC approach to directly test models with ancient population size changes in Africa and found little support for such ancient bottlenecks.</p><p id="p-78">Deep ancestral structure has been put forward as explanation for the humps detected by the whole genome-based methods by the developers of PSMC and others (<a id="xref-ref-50-12" class="xref-bibr" href="#ref-50">Li and Durbin 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Henn <em>et al.</em> 2012</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Mazet <em>et al.</em> 2015a</a>,<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">b</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Orozco-terWengel 2016</a>). While <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Blum and Jakobsson (2011)</a> used the TMRCA to postulate an ancient bottleneck 150 KYA, they also were not able to reject a model of ancestral structure. Strikingly, <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Mazet <em>et al.</em> (2015b)</a> were able to perfectly recapitulate the human PSMC humps without invoking a single size change in the population by simulating data from a highly structured ancestral population (10 subpopulations) and modulating the amount of gene flow between these populations. Therefore, the large “population size changes” inferred in MSMC, which cause the models not to match the empirical SFS, may in fact be due to complex structure and large-scale changes in gene flow. This ancient structure may have a large effect on MSMC trajectories and LD patterns, but may not strongly influence the SFS (see figure 7 in <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lohmueller <em>et al.</em> 2009</a>), potentially resolving the discrepancy between the methods (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Henn <em>et al.</em> 2012</a>).</p><p id="p-79">Our work provides a cautionary tale for understanding population history in nonmodel organisms. Our results argue against a literal interpretation of humps and other jumps in MSMC plots as reflecting population size changes. This problem is exacerbated for putative ancient size changes. Given the ever-increasing generation of genomic data from nonmodel taxa and the application of whole genome-based approaches to such data (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Groenen <em>et al.</em> 2012</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Meyer <em>et al.</em> 2012</a>; <a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">Zhao <em>et al.</em> 2012</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Albert <em>et al.</em> 2013</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Ibarra-Laclette <em>et al.</em> 2013</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Nadachowska-Brzyska <em>et al.</em> 2013</a>, <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">2015</a>, <a id="xref-ref-65-5" class="xref-bibr" href="#ref-65">2016</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Orlando <em>et al.</em> 2013</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Prado-Martinez <em>et al.</em> 2013</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bosse <em>et al.</em> 2014</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Freedman <em>et al.</em> 2014</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Hung <em>et al.</em> 2014</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Prufer <em>et al.</em> 2014</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Palkopoulou <em>et al.</em> 2015</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Holliday <em>et al.</em> 2016</a>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Wang <em>et al.</em> 2016</a>), our findings are especially concerning. We recommend employing other model-based types of demographic inference leveraging either SFS-based or other summary statistics in an ABC framework to test whether important demographic features suggested by PSMC or MSMC plots can be recapitulated using other features in the data. We also recommend, as done in <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Thornton and Andolfatto (2006)</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Freedman <em>et al.</em> (2014)</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cahill <em>et al.</em> (2016)</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hsieh <em>et al.</em> (2016)</a>, and <a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Song <em>et al.</em> (2016)</a>, that the PSMC or MSMC plots and TMRCA estimates be used themselves as summary statistics for model comparison, rather than the actual population size estimates. In other words, more complex demographic models can be simulated and tested to see whether they recapitulate the observed whole genome-based trajectories. Of course, this approach will not be successful if the trajectories are strongly influenced by bioinformatic artifacts or other features not captured within the simulations, such as natural selection. For both PSMC/MSMC and SFS-based inference methods, we also recommend testing whether the estimated models can predict multiple features of the data. Specifically, researchers should check whether their inferred model can recapitulate the genome-wide distribution of heterozygosity. The genome-wide distribution of heterozygosity may be the most practical and useful statistic for studies of nonmodel organisms that only have a handful of genomes available to them. SMC++ and other new approaches that leverage multiple types of data (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Bunnefeld <em>et al.</em> 2015</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Boitard <em>et al.</em> 2016</a>; <a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Weissman and Hallatschek 2017</a>) are promising alternatives, though our results indicate that SMC++ still cannot recapitulate all summaries of the data.</p><p id="p-80">Testing more complex demographic scenarios using multiple summaries of the data may help to resolve uncertainties about our own species’ history and will improve demographic inference for nonmodel organisms. Incorporating the potential complexity of possible demographic histories to produce models that better recapitulate the data may in fact present the greatest challenge.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-81">We thank Ryan Gutenkunst for providing us with the SFS from his article, and Yun Song for his published demographic models. We also thank Diego Ortega Del Vecchyo and Bernard Kim for advice and Christian Huber, Robert K. Wayne, and Emilia Huerta-Sanchez for helpful comments on the manuscript. Joshua Schraiber and several anonymous reviewers made many insightful comments that strengthened our manuscript. This work was supported by National Institutes of Health (NIH) grant R35 GM-119856 to K.E.L. and NIH Training Grant in Genomic Analysis and Interpretation T32 HG-002536 and National Science Foundation Graduate Research Fellowship Program to A.C.B. T.N.P. was supported by NIH, under the Ruth L. Kirschestein National Research Service Award (T32 GM-008185).</p><p id="p-82">Author contributions: K.E.L. and A.C.B. conceived the study. A.C.B. carried out all analyses based on the demographic models, and T.N.P. carried out all empirical analyses based on the 1000 Genomes data. A.C.B. generated all figures. A.C.B., T.N.P., and K.E.L. all participated in manuscript preparation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-83">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300259/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-84"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-07-12"><span class="received-label">Received </span>July 12, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-09-01"><span class="accepted-label">Accepted </span>September 1, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Beichman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.1" data-doi="10.1038/nature15393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A global reference for human genetic variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432245&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.2" data-doi="10.1534/genetics.104.030171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Maximum-likelihood estimation of demographic parameters using the frequency spectrum of unlinked single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1699</span>–<span class="cit-lpage">1712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAdams%26rft.auinit1%253DA.%2BM.%26rft.volume%253D168%26rft.issue%253D3%26rft.spage%253D1699%26rft.epage%253D1712%26rft.atitle%253DMaximum-Likelihood%2BEstimation%2Bof%2BDemographic%2BParameters%2BUsing%2Bthe%2BFrequency%2BSpectrum%2Bof%2BUnlinked%2BSingle-Nucleotide%2BPolymorphisms%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.030171%26rft_id%253Dinfo%253Apmid%252F15579718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC8zLzE2OTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.3" data-doi="10.1126/science.1241089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Der</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leebens-Mack</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The <em>Amborella</em> genome and the evolution of flowering plants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1241089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAmborella%2BGenome%2BProject%26rft.auinit1%253D%2B%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1241089%26rft.epage%253D1241089%26rft.atitle%253DThe%2BAmborella%2BGenome%2Band%2Bthe%2BEvolution%2Bof%2BFlowering%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1241089%26rft_id%253Dinfo%253Apmid%252F24357323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDIvNjE2NS8xMjQxMDg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.4" data-doi="10.1186/1471-2105-13-301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arbiza</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NRE: a tool for exploring neutral loci in the human genome.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DArbiza%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D301%26rft.epage%253D301%26rft.atitle%253DNRE%253A%2BA%2Btool%2Bfor%2Bexploring%2Bneutral%2Bloci%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-301%26rft_id%253Dinfo%253Apmid%252F23151206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23151206&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.5" data-doi="10.1086/323251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ardlie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu-Cordero</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberle</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Lower-than-expected linkage disequilibrium between tightly linked markers in humans suggests a role for gene conversion.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">582</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DArdlie%26rft.auinit1%253DK.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D582%26rft.epage%253D589%26rft.atitle%253DLower-than-expected%2Blinkage%2Bdisequilibrium%2Bbetween%2Btightly%2Blinked%2Bmarkers%2Bin%2Bhumans%2Bsuggests%2Ba%2Brole%2Bfor%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F323251%26rft_id%253Dinfo%253Apmid%252F11473344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/323251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11473344&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170271400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.6" data-doi="10.1101/gr.178756.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhaskar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient inference of population size histories and locus-specific mutation rates from large-sample genomic variation data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.178756.114%26rft_id%253Dinfo%253Apmid%252F25564017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzIvMjY4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.7" data-doi="10.1093/molbev/msq265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deep divergences of human gene trees and models of human origins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq265%26rft_id%253Dinfo%253Apmid%252F20930054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msq265&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20930054&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287112300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jay</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mona</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austerlitz</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inferring population size history from large samples of genome-wide molecular data-an approximate Bayesian computation approach.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.9" data-doi="10.1111/mec.12807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bosse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paudel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Untangling the hybrid nature of modern pig genomes: a mosaic derived from biogeographically distinct and highly divergent <em>Sus scrofa</em> populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4089</span>–<span class="cit-lpage">4102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4089%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.10" data-doi="10.1371/journal.pgen.1000083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degenhardt</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Assessing the evolutionary impact of amino acid mutations in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBoyko%26rft.auinit1%253DA.%2BR.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000083%26rft.epage%253De1000083%26rft.atitle%253DAssessing%2Bthe%2Bevolutionary%2Bimpact%2Bof%2Bamino%2Bacid%2Bmutations%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000083%26rft_id%253Dinfo%253Apmid%252F18516229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516229&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.11" data-doi="10.1534/genetics.115.179861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bunnefeld</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">L. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inferring bottlenecks from genome-wide samples of short sequence blocks.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1157</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179861%26rft_id%253Dinfo%253Apmid%252F26341659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS8zLzExNTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.12" data-doi="10.1098/rstb.2015.0138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cahill</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soares</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inferring species divergence times using pairwise sequential Markovian coalescent modelling and low-coverage genomic data.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B. Biol. Sci.</abbr> <span class="cit-vol">371</span>: <span class="cit-fpage">20150138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2015.0138%26rft_id%253Dinfo%253Apmid%252F27325835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNzEvMTY5OS8yMDE1MDEzOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzYwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.13" data-doi="10.1101/gr.083634.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and flexible simulation of DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083634.108%26rft_id%253Dinfo%253Apmid%252F19029539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.14" data-doi="10.1534/genetics.110.118661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luisi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goossens</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The confounding effects of population structure, genetic diversity and the sampling scheme on the detection and quantification of population size changes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">983</span>–<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118661%26rft_id%253Dinfo%253Apmid%252F20739713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzMvOTgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.15" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.16" data-doi="10.1126/science.1181498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drmanac</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callow</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1181498%26rft_id%253Dinfo%253Apmid%252F19892942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjcvNTk2MS83OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzYwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.17" data-doi="10.1093/bioinformatics/btq322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2064</span>–<span class="cit-lpage">2065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq322%26rft_id%253Dinfo%253Apmid%252F20591904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20591904&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280703500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.18" data-doi="10.1371/journal.pgen.1003905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Sánchez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Robust demographic inference from genomic and SNP data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003905%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003905%26rft_id%253Dinfo%253Apmid%252F24204310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204310&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.19" data-doi="10.1073/pnas.0708280104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fagundes</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuenschwander</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzano</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Statistical evaluation of alternative models of human evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">17614</span>–<span class="cit-lpage">17619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0708280104%26rft_id%253Dinfo%253Apmid%252F17978179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ1LzE3NjE0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.20" data-doi="10.1111/1755-0998.12443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitak</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohandesan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The de novo genome assembly and annotation of a female domestic dromedary of North African origin.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D314%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12443%26rft_id%253Dinfo%253Apmid%252F26178449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26178449&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.21" data-doi="10.1371/journal.pgen.1004016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freedman</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gronau</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweizer</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortega-Del Vecchyo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing highlights the dynamic early history of dogs.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004016%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004016%26rft_id%253Dinfo%253Apmid%252F24453982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24453982&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.22" data-doi="10.1086/323612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frisse</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoszewicz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donfack</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Gene conversion and different population histories may explain the contrast between polymorphism and linkage disequilibrium levels.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DFrisse%26rft.auinit1%253DL.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D831%26rft.epage%253D843%26rft.atitle%253DGene%2Bconversion%2Band%2Bdifferent%2Bpopulation%2Bhistories%2Bmay%2Bexplain%2Bthe%2Bcontrast%2Bbetween%2Bpolymorphism%2Band%2Blinkage%2Bdisequilibrium%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F323612%26rft_id%253Dinfo%253Apmid%252F11533915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/323612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11533915&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171008200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Explosive genetic evidence for explosive human population growth.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D41%26rft.spage%253D130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.24" data-doi="10.1038/nrg1941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrigan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammer</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reconstructing human origins in the genomic era.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGarrigan%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D669%26rft.epage%253D680%26rft.atitle%253DReconstructing%2Bhuman%2Borigins%2Bin%2Bthe%2Bgenomic%2Bera.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1941%26rft_id%253Dinfo%253Apmid%252F16921345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16921345&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239941900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.25" data-doi="10.1371/journal.pgen.1005004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buzbas</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recent selective sweeps in North American <em>Drosophila melanogaster</em> show signatures of soft sweeps.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005004%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005004%26rft_id%253Dinfo%253Apmid%252F25706129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706129&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.26" data-doi="10.1038/hdy.2012.120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gattepaille</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring population size changes with sequence and SNP data: lessons from human bottlenecks.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.120%26rft_id%253Dinfo%253Apmid%252F23423148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23423148&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.27" data-doi="10.1073/pnas.1310398110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gazave</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coventry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Neutral genomic regions refine models of recent rapid human population growth.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1310398110%26rft_id%253Dinfo%253Apmid%252F24379384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTEvMi83NTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.28" data-doi="10.1073/pnas.1019276108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Demographic history and rare allele sharing among human populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">11983</span>–<span class="cit-lpage">11988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019276108%26rft_id%253Dinfo%253Apmid%252F21730125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI5LzExOTgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.29" data-doi="10.1038/nature11622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uenishi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuggle</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Analyses of pig genomes provide insight into porcine demography and evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGroenen%26rft.auinit1%253DM.%2BA.%26rft.volume%253D491%26rft.issue%253D7424%26rft.spage%253D393%26rft.epage%253D398%26rft.atitle%253DAnalyses%2Bof%2Bpig%2Bgenomes%2Bprovide%2Binsight%2Binto%2Bporcine%2Bdemography%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11622%26rft_id%253Dinfo%253Apmid%252F23151582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23151582&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311031600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.30" data-doi="10.1371/journal.pgen.1000695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000695%26rft.epage%253De1000695%26rft.atitle%253DInferring%2Bthe%2Bjoint%2Bdemographic%2Bhistory%2Bof%2Bmultiple%2Bpopulations%2Bfrom%2Bmultidimensional%2BSNP%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000695%26rft_id%253Dinfo%253Apmid%252F19851460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851460&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.31" data-doi="10.1093/molbev/mst229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterizing bias in population genetic inferences from low-coverage sequencing data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst229%26rft_id%253Dinfo%253Apmid%252F24288159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24288159&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332343000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.32" data-doi="10.1093/bioinformatics/btu725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fast and accurate site frequency spectrum estimation from low coverage sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu725%26rft_id%253Dinfo%253Apmid%252F25359894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25359894&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.33" data-doi="10.1073/pnas.95.4.1961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harpending</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzer</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurven</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic traces of ancient demography.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1961</span>–<span class="cit-lpage">1967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHarpending%26rft.auinit1%253DH.%2BC.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D1961%26rft.epage%253D1967%26rft.atitle%253DGenetic%2Btraces%2Bof%2Bancient%2Bdemography%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.4.1961%26rft_id%253Dinfo%253Apmid%252F9465125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NS80LzE5NjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.34" data-doi="10.1371/journal.pgen.1003521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring demographic history from a spectrum of shared haplotype lengths.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003521%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003521%26rft_id%253Dinfo%253Apmid%252F23754952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23754952&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.35" data-doi="10.1093/oxfordjournals.molbev.a026233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunley</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolpoff</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Population bottlenecks and Pleistocene human evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DHawks%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D2%26rft.epage%253D22%26rft.atitle%253DPopulation%2BBottlenecks%2Band%2BPleistocene%2BHuman%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026233%26rft_id%253Dinfo%253Apmid%252F10666702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10666702&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084917200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.36" data-doi="10.1371/journal.pone.0054583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegismund</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The confounding effect of population structure on Bayesian skyline plot inferences of demographic history.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054583%26rft_id%253Dinfo%253Apmid%252F23620825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23620825&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.37" data-doi="10.1073/pnas.1212380109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The great human expansion.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">17758</span>–<span class="cit-lpage">17764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1212380109%26rft_id%253Dinfo%253Apmid%252F23077256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ0LzE3NzU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.38" data-doi="10.1073/pnas.1510805112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botigué</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peischl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipatov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Distance from sub-Saharan Africa predicts mutational load in diverse human genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E440</span>–<span class="cit-lpage">E449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1510805112%26rft_id%253Dinfo%253Apmid%252F26712023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzQvRTQ0MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzYwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.39" data-doi="10.1111/nph.13643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bawa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oubida</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evidence for extensive parallelism but divergent genomic architecture of adaptation along altitudinal and latitudinal gradients in <em>Populus trichocarpa</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">1240</span>–<span class="cit-lpage">1251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D209%26rft.spage%253D1240%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13643%26rft_id%253Dinfo%253Apmid%252F26372471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13643&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26372471&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.40" data-doi="10.1101/gr.192971.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veeramah</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lachance</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole-genome sequence analyses of Western Central African Pygmy hunter-gatherers reveal a complex demographic history and identify candidate genes under positive natural selection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.192971.115%26rft_id%253Dinfo%253Apmid%252F26888263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI2LzMvMjc5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.41" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Generating samples under a Wright–Fisher neutral model of genetic variation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.2.337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847089&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174028100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.42" data-doi="10.1073/pnas.1401526111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">C.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaner</span>  <span class="cit-name-given-names">P.-J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zink</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">T.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Drastic population fluctuations explain the rapid extinction of the passenger pigeon.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">10636</span>–<span class="cit-lpage">10641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1401526111%26rft_id%253Dinfo%253Apmid%252F24979776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzI5LzEwNjM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.43" data-doi="10.1038/nature12132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ibarra-Laclette</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Guzmán</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Torres</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carretero-Paulet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Architecture and evolution of a minute plant genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">498</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D498%26rft.spage%253D94%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12132%26rft_id%253Dinfo%253Apmid%252F23665961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23665961&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319947800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.44" data-doi="10.1534/genetics.117.200493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jouganous</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragsdale</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations: beyond the diffusion approximation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1549</span>–<span class="cit-lpage">1567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.200493%26rft_id%253Dinfo%253Apmid%252F28495960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzE1NDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.45" data-doi="10.1126/science.1217283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recent explosive human population growth has resulted in an excess of rare genetic variants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>: <span class="cit-fpage">740</span>–<span class="cit-lpage">743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKeinan%26rft.auinit1%253DA.%26rft.volume%253D336%26rft.issue%253D6082%26rft.spage%253D740%26rft.epage%253D743%26rft.atitle%253DRecent%2BExplosive%2BHuman%2BPopulation%2BGrowth%2BHas%2BResulted%2Bin%2Ban%2BExcess%2Bof%2BRare%2BGenetic%2BVariants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1217283%26rft_id%253Dinfo%253Apmid%252F22582263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzYvNjA4Mi83NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.46" data-doi="10.1038/ng2116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullikin</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Measurement of the human allele frequency spectrum demonstrates greater genetic drift in East Asians than in Europeans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKeinan%26rft.auinit1%253DA.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1251%26rft.epage%253D1255%26rft.atitle%253DMeasurement%2Bof%2Bthe%2Bhuman%2Ballele%2Bfrequency%2Bspectrum%2Bdemonstrates%2Bgreater%2Bgenetic%2Bdrift%2Bin%2BEast%2BAsians%2Bthan%2Bin%2BEuropeans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2116%26rft_id%253Dinfo%253Apmid%252F17828266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828266&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.47" data-doi="10.1016/j.ajhg.2012.08.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrnes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musharoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genetic inference from personal genome data: impact of ancestry and admixture on human genomic variation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DKidd%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D91%26rft.issue%253D4%26rft.spage%253D660%26rft.epage%253D671%26rft.atitle%253DPopulation%2Bgenetic%2Binference%2Bfrom%2Bpersonal%2Bgenome%2Bdata%253A%2Bimpact%2Bof%2Bancestry%2Band%2Badmixture%2Bon%2Bhuman%2Bgenomic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.08.025%26rft_id%253Dinfo%253Apmid%252F23040495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.08.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23040495&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.48" data-doi="10.1186/1471-2105-12-231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimation of allele frequency and association mapping using next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKim%26rft.auinit1%253DS.%2BY.%26rft.volume%253D12%26rft.spage%253D231%26rft.epage%253D231%26rft.atitle%253DEstimation%2Bof%2Ballele%2Bfrequency%2Band%2Bassociation%2Bmapping%2Busing%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-231%26rft_id%253Dinfo%253Apmid%252F21663684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21663684&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.49" data-doi="10.1038/nature09525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigurdsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine-scale recombination rate differences between sexes, populations and individuals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1099%26rft.epage%253D1103%26rft.atitle%253DFine-scale%2Brecombination%2Brate%2Bdifferences%2Bbetween%2Bsexes%252C%2Bpopulations%2Band%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09525%26rft_id%253Dinfo%253Apmid%252F20981099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981099&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.50" data-doi="10.1038/nature10231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inference of human population history from individual whole-genome sequences.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D475%26rft.issue%253D7357%26rft.spage%253D493%26rft.epage%253D496%26rft.atitle%253DInference%2Bof%2Bhuman%2Bpopulation%2Bhistory%2Bfrom%2Bindividual%2Bwhole-genome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10231%26rft_id%253Dinfo%253Apmid%252F21753753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21753753&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293167900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.51" data-doi="10.1038/ng.3254"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.-X.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exploring population size changes using SNP frequency spectra.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D555%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3254%26rft_id%253Dinfo%253Apmid%252F25848749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25848749&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.52" data-doi="10.1534/genetics.108.099275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Methods for human demographic inference using haplotype patterns from genomewide single-nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.099275%26rft_id%253Dinfo%253Apmid%252F19255370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMjE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.53" data-doi="10.1038/nature18299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westaway</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lao</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomic history of Aboriginal Australia.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18299%26rft_id%253Dinfo%253Apmid%252F27654914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27654914&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.54" data-doi="10.1038/nature18964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathieson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gymrek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Simons Genome Diversity Project: 300 genomes from 142 diverse populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18964%26rft_id%253Dinfo%253Apmid%252F27654912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27654912&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.55" data-doi="10.1186/1471-2156-7-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Fast “coalescent” simulation.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DHamady%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DReal-time%2BPCR%2Bquantification%2Bof%2Bhuman%2Bcomplement%2BC4A%2Band%2BC4B%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-7-1%26rft_id%253Dinfo%253Apmid%252F16403222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-7-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16403222&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237351100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.56" data-doi="10.1534/genetics.166.1.351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czabarka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murvai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherry</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The allele frequency spectrum in genome-wide human variation data reveals signals of differential demographic history in three large world populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMarth%26rft.auinit1%253DG.%2BT.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D351%26rft.epage%253D372%26rft.atitle%253DThe%2BAllele%2BFrequency%2BSpectrum%2Bin%2BGenome-Wide%2BHuman%2BVariation%2BData%2BReveals%2BSignals%2Bof%2BDifferential%2BDemographic%2BHistory%2Bin%2BThree%2BLarge%2BWorld%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.351%26rft_id%253Dinfo%253Apmid%252F15020430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvMzUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.57" data-doi="10.1016/j.tpb.2015.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mazet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Demographic inference using genetic data from a single individual: separating population size variation from population structure.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D104%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2015.06.003%26rft_id%253Dinfo%253Apmid%252F26120083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2015.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26120083&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mazet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grusea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">On the importance of being structured: instantaneous coalescence rates and human evolution—lessons for ancestral population size inference?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D116%26rft.spage%253D362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.59" data-doi="10.1111/mec.12591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCoy</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boggs</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic inference accurately predicts the timing and severity of a recent bottleneck in a nonmodel insect population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000330950900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.60" data-doi="10.1098/rstb.2005.1673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardin</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Approximating the coalescent with recombination.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">1387</span>–<span class="cit-lpage">1393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D360%26rft.issue%253D1459%26rft.spage%253D1387%26rft.epage%253D1393%26rft.atitle%253DApproximating%2Bthe%2Bcoalescent%2Bwith%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2005.1673%26rft_id%253Dinfo%253Apmid%252F16048782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjAvMTQ1OS8xMzg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.61" data-doi="10.1126/science.1224344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gansauge</span>  <span class="cit-name-given-names">M.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-coverage genome sequence from an archaic Denisovan individual.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">338</span>: <span class="cit-fpage">222</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1224344%26rft_id%253Dinfo%253Apmid%252F22936568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzgvNjEwNC8yMjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-web" id="cit-7.11.3605.62"><div class="cit-metadata unstructured">Murray, G. G. R., A. E. R. Soares, B. J. Novak, N. K. Schaefer, J. A. Cahill <em>et al.</em>, 2017 Natural selection shaped the rise and fall of passenger pigeon genomic diversity. bioRxiv . Available at:: <a href="https://doi.org/10.1101/154294">https://doi.org/10.1101/154294</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.63" data-doi="10.1371/journal.pgen.1003942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadachowska-Brzyska</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olason</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Demographic divergence history of pied flycatcher and collared flycatcher inferred from whole-genome re-sequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003942%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003942%26rft_id%253Dinfo%253Apmid%252F24244198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244198&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.64" data-doi="10.1016/j.cub.2015.03.047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadachowska-Brzyska</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Temporal dynamics of avian populations during Pleistocene revealed by whole-genome sequences.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1375</span>–<span class="cit-lpage">1380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D25%26rft.spage%253D1375%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2015.03.047%26rft_id%253Dinfo%253Apmid%252F25891404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2015.03.047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25891404&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadachowska-Brzyska</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">PSMC analysis of effective population sizes in molecular ecology and its application to black‐and‐white <em>Ficedula</em> flycatchers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1058</span>–<span class="cit-lpage">1072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D1058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.66" data-doi="10.2307/2407137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruyama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">The bottleneck effect and genetic variability in populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D29%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2407137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2407137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1975W198800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.67" data-doi="10.1126/science.1217876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehm</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jean</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1217876%26rft_id%253Dinfo%253Apmid%252F22604722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5MC8xMDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimation of population parameters and recombination rates from single nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D154%26rft.issue%253D2%26rft.spage%253D931%26rft.epage%253D942%26rft.atitle%253DEstimation%2Bof%2BPopulation%2BParameters%2Band%2BRecombination%2BRates%2BFrom%2BSingle%2BNucleotide%2BPolymorphisms%26rft_id%253Dinfo%253Apmid%252F10655242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzIvOTMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.69" data-doi="10.1101/gr.088336.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andres</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Darwinian and demographic forces affecting human protein coding genes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">838</span>–<span class="cit-lpage">849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.088336.108%26rft_id%253Dinfo%253Apmid%252F19279335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvODM4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.70" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.71" data-doi="10.1038/nature12323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orlando</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginolhac</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froese</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recalibrating <em>Equus</em> evolution using the genome sequence of an early Middle Pleistocene horse.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">499</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOrlando%26rft.auinit1%253DL.%26rft.volume%253D499%26rft.issue%253D7456%26rft.spage%253D74%26rft.epage%253D78%26rft.atitle%253DRecalibrating%2BEquus%2Bevolution%2Busing%2Bthe%2Bgenome%2Bsequence%2Bof%2Ban%2Bearly%2Bmiddle%2BPleistocene%2Bhorse%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12323%26rft_id%253Dinfo%253Apmid%252F23803765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013063355&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=23803765&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321285600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The devil is in the details: the effect of population structure on demographic inference.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D116%26rft.spage%253D349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pagani</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jagoda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mörseburg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic analyses inform on migration events during the peopling of Eurasia.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.74" data-doi="10.1016/j.cub.2015.04.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palkopoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skoglund</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Complete genomes reveal signatures of demographic and genetic declines in the woolly mammoth.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1395</span>–<span class="cit-lpage">1400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D25%26rft.spage%253D1395%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2015.04.007%26rft_id%253Dinfo%253Apmid%252F25913407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2015.04.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25913407&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.75" data-doi="10.1111/j.1365-294X.2010.04783.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Distinguishing between population bottleneck and population subdivision by a Bayesian model choice procedure.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4648</span>–<span class="cit-lpage">4660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPeter%26rft.auinit1%253DB.%2BM.%26rft.volume%253D19%26rft.issue%253D21%26rft.spage%253D4648%26rft.epage%253D4660%26rft.atitle%253DDistinguishing%2Bbetween%2Bpopulation%2Bbottleneck%2Band%2Bpopulation%2Bsubdivision%2Bby%2Ba%2BBayesian%2Bmodel%2Bchoice%2Bprocedure.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04783.x%26rft_id%253Dinfo%253Apmid%252F20735743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04783.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20735743&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283163700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phung</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Determining the effect of natural selection on linked neutral divergence across species.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.77" data-doi="10.1371/journal.pgen.0020105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plagnol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Possible ancestral structure in human populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPlagnol%26rft.auinit1%253DV.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De105%26rft.epage%253De105%26rft.atitle%253DPossible%2Bancestral%2Bstructure%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020105%26rft_id%253Dinfo%253Apmid%252F16895447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16895447&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polanski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">New explicit expressions for relative frequencies of single-nucleotide polymorphisms with application to statistical inference on population growth.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPolanski%26rft.auinit1%253DA.%26rft.volume%253D165%26rft.issue%253D1%26rft.spage%253D427%26rft.epage%253D436%26rft.atitle%253DNew%2BExplicit%2BExpressions%2Bfor%2BRelative%2BFrequencies%2Bof%2BSingle-Nucleotide%2BPolymorphisms%2BWith%2BApplication%2Bto%2BStatistical%2BInference%2Bon%2BPopulation%2BGrowth%26rft_id%253Dinfo%253Apmid%252F14504247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzEvNDI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.79" data-doi="10.1038/nature12228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prado-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Great ape genetic diversity and population history.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">499</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12228%26rft_id%253Dinfo%253Apmid%252F23823723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23823723&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322157900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.80" data-doi="10.1038/nature12886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prufer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jay</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The complete genome sequence of a Neanderthal from the Altai Mountains.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPruefer%26rft.auinit1%253DK.%26rft.volume%253D505%26rft.issue%253D7481%26rft.spage%253D43%26rft.epage%253D49%26rft.atitle%253DThe%2Bcomplete%2Bgenome%2Bsequence%2Bof%2Ba%2BNeanderthal%2Bfrom%2Bthe%2BAltai%2BMountains%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12886%26rft_id%253Dinfo%253Apmid%252F24352235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2014009693&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=24352235&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329163300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.81" data-doi="10.1016/S0168-9525(02)02781-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ptak</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evidence for population growth in humans is confounded by fine-scale population structure.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DPtak%26rft.auinit1%253DS.%2BE.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D559%26rft.epage%253D563%26rft.atitle%253DEvidence%2Bfor%2Bpopulation%2Bgrowth%2Bin%2Bhumans%2Bis%2Bconfounded%2Bby%2Bfine-scale%2Bpopulation%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902781-6%26rft_id%253Dinfo%253Apmid%252F12414185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02781-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12414185&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178796300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.82" data-doi="10.1038/ng.3015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiffels</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Inferring human population size and separation history from multiple genome sequences.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D919%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3015%26rft_id%253Dinfo%253Apmid%252F24952747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24952747&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.83" data-doi="10.1073/pnas.0703874104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peck</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">East African megadroughts between 135 and 75 thousand years ago and bearing on early-modern human origins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">16416</span>–<span class="cit-lpage">16421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0703874104%26rft_id%253Dinfo%253Apmid%252F17785420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQyLzE2NDE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schraiber</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Methods and models for unravelling human evolutionary history.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D727%26rft_id%253Dinfo%253Apmid%252F26553329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26553329&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.85" data-doi="10.1534/genetics.116.190223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shanku</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effects of linked selective sweeps on demographic inference and model selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">1207</span>–<span class="cit-lpage">1223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.190223%26rft_id%253Dinfo%253Apmid%252F27605051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC8zLzEyMDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.86" data-doi="10.1093/molbev/mss061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sjödin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sjöstrand</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Resequencing data provide no evidence for a human bottleneck in Africa during the penultimate glacial period.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1851</span>–<span class="cit-lpage">1860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss061%26rft_id%253Dinfo%253Apmid%252F22319141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22319141&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305409000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D2%26rft.spage%253D555%26rft.epage%253D562%26rft.atitle%253DPairwise%2Bcomparisons%2Bof%2Bmitochondrial%2BDNA%2Bsequences%2Bin%2Bstable%2Band%2Bexponentially%2Bgrowing%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F1743491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzIvNTU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sliwerska</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Modeling human population separation history using physically phased genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D205%26rft.spage%253D385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sovic</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic diversity in migratory bats: results from RADseq data for three tree bat species at an Ohio windfarm.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D4%26rft.spage%253De1647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The effect of change in population size on DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D597%26rft.epage%253D601%26rft.atitle%253DThe%2Beffect%2Bof%2Bchange%2Bin%2Bpopulation%2Bsize%2Bon%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2599369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMS8zNjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.91" data-doi="10.1007/s002510050380"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satta</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Footprints of intragenic recombination at HLA loci.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.stitle%253DImmunogenetics%26rft.aulast%253DTakahata%26rft.auinit1%253DN.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D430%26rft.epage%253D441%26rft.atitle%253DFootprints%2Bof%2Bintragenic%2Brecombination%2Bat%2BHLA%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002510050380%26rft_id%253Dinfo%253Apmid%252F9553149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002510050380&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9553149&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073621900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.92" data-doi="10.1006/tpbi.1995.1026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Divergence time and population size in the lineage leading to modern humans.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DTakahata%26rft.auinit1%253DN.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D221%26rft.atitle%253DDivergence%2Btime%2Band%2Bpopulation%2Bsize%2Bin%2Bthe%2Blineage%2Bleading%2Bto%2Bmodern%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1995.1026%26rft_id%253Dinfo%253Apmid%252F7482371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1995.1026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7482371&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RZ41300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.93" data-doi="10.1126/science.1219240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tennessen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution and functional impact of rare coding variation from deep sequencing of human exomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1219240%26rft_id%253Dinfo%253Apmid%252F22604720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzcvNjA5MC82NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTEvMzYwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.94" data-doi="10.1073/pnas.1503717112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terhorst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fundamental limits on the accuracy of demographic inference based on the sample frequency spectrum.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">7677</span>–<span class="cit-lpage">7682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503717112%26rft_id%253Dinfo%253Apmid%252F26056264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzI1Lzc2NzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.95" data-doi="10.1038/ng.3748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terhorst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamm</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Robust and scalable inference of population history from hundreds of unphased whole genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3748%26rft_id%253Dinfo%253Apmid%252F28024154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28024154&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.96" data-doi="10.1534/genetics.105.048223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Approximate Bayesian inference reveals evidence for a recent, severe bottleneck in a Netherlands population of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1607</span>–<span class="cit-lpage">1619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048223%26rft_id%253Dinfo%253Apmid%252F16299396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE2MDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.97" data-doi="10.1098/rspb.2014.0528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trucchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gratton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittington</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cristofari</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Maho</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">King penguin demography since the last glaciation inferred from genome-wide data.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20140528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.0528%26rft_id%253Dinfo%253Apmid%252F24920481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3ODcvMjAxNDA1MjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzExLzM2MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-book" id="cit-7.11.3605.98"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>Coalescent Theory: An Introduction</em></span>. <span class="cit-publ-name">Roberts &amp; Co. Publishers</span>, <span class="cit-publ-loc">Greenwood Village, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.99" data-doi="10.1038/nrg1123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Haplotype blocks and linkage disequilibrium in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWall%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D587%26rft.epage%253D597%26rft.atitle%253DHaplotype%2Bblocks%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1123%26rft_id%253Dinfo%253Apmid%252F12897771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12897771&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184491300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.100" data-doi="10.1038/cr.2015.147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Out of southern East Asia: the natural history of domestic dogs across the world.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D26%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2015.147%26rft_id%253Dinfo%253Apmid%252F26667385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2015.147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26667385&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallatschek</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Minimal-assumption inference from population-genomic data.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D6%26rft.spage%253De24836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-7.11.3605.102" data-doi="10.1038/ng.2494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole-genome sequencing of giant pandas provides insights into demographic history and local adaptation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2494%26rft_id%253Dinfo%253Apmid%252F23242367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23242367&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F11%2F3605.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/11/3605.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3597" title="The Transcriptional Response of Candida albicans to Weak Organic Acids, Carbon Source, and MIG1 Inactivation Unveils a Role for HGT16 in Mediating the Fungistatic Effect of Acetic Acid" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/11/3621" title="Genome-Wide Analysis of DNA Methylation During Ovule Development of Female-Sterile Rice fsv1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/11"><span>Volume 7 Issue 11, November 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/11.cover-source.jpg?itok=-dAnYc96" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/11/3605" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gJswBDYNqQpba0trILcKQMqiqtNbnfa6aVCVFukULFg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/11/3605" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/11/3605" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/11/3605" class="active" data-icon-position="" data-hide-link-title="0">Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487108" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CLMCPvGh-QbnTDLH4Qtg7Y0eXvufvVFMrhqfwecI1zg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3605.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/11/3605" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-yBX2oE24ujLBzjZZf_AhMg9nPQ5LryG11nIMEHj6fsI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/11/3605" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/11/3605.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487108" id="node487108--2" data-pisa="ggg;7/11/3605" data-pisa-master="ggg;g3.117.300259" data-apath="/ggg/7/11/3605.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6991-587X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Annabel C. Beichman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2685-6977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanya N. Phung</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3874-369X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kirk E. Lohmueller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3605-3620;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300259 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487108--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Annabel C. Beichman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabel%2BC.%2BBeichman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beichman%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnnabel%2BC.%2BBeichman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6991-587X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Annabel C. Beichman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tanya N. Phung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanya%2BN.%2BPhung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phung%20TN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanya%2BN.%2BPhung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2685-6977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanya N. Phung</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kirk E. Lohmueller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kirk%2BE.%2BLohmueller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lohmueller%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKirk%2BE.%2BLohmueller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3874-369X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kirk E. Lohmueller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:klohmueller@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">klohmueller@ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/11/3605" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/11/3605&amp;title=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/11/3605&amp;title=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/11/3605&amp;title=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/11/3605&amp;text=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/11/3605&amp;title=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/11/3605&amp;t=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/11/3605&amp;title=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/11/3605&amp;title=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487108/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487108" id="node487108--3" data-pisa="ggg;7/11/3605" data-pisa-master="ggg;g3.117.300259" data-apath="/ggg/7/11/3605.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6991-587X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Annabel C. Beichman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2685-6977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanya N. Phung</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3874-369X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kirk E. Lohmueller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3605-3620;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300259 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487108--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Annabel C. Beichman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabel%2BC.%2BBeichman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beichman%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnnabel%2BC.%2BBeichman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6991-587X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Annabel C. Beichman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tanya N. Phung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanya%2BN.%2BPhung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phung%20TN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanya%2BN.%2BPhung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2685-6977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanya N. Phung</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kirk E. Lohmueller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kirk%2BE.%2BLohmueller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lohmueller%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKirk%2BE.%2BLohmueller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3874-369X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kirk E. Lohmueller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:klohmueller@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">klohmueller@ucla.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3605&amp;rft.epage=3620&amp;rft.atitle=Comparison%20of%20Single%20Genome%20and%20Allele%20Frequency%20Data%20Reveals%20Discordant%20Demographic%20Histories&amp;rft.volume=7&amp;rft.issue=11&amp;rft.date=2017-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Beichman%2C+Annabel+C.&amp;rft.au=Phung%2C+Tanya+N.&amp;rft.au=Lohmueller%2C+Kirk+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487108/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487108/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487108/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487108/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487108/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487108/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487108/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487108/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487108/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487108/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487108/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487108/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/11/3605" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/11/3605.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/11/3605.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/11/3605.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/11/3605.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
