<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3249" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3249.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3249.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3249.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3249.atom" />
<meta name="HW.pisa" content="ggg;9/10/3249" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" />
<meta name="DC.Identifier" content="10.1534/g3.119.400445" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Love et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Chromosomal inversion polymorphisms play an important role in adaptation to environmental heterogeneities. For mosquito species in the Anopheles gambiae complex that are significant vectors of human malaria, paracentric inversion polymorphisms are abundant and are associated with ecologically and epidemiologically important phenotypes. Improved understanding of these traits relies on determining mosquito karyotype, which currently depends upon laborious cytogenetic methods whose application is limited both by the requirement for specialized expertise and for properly preserved adult females at specific gonotrophic stages. To overcome this limitation, we developed sets of tag single nucleotide polymorphisms (SNPs) inside inversions whose biallelic genotype is strongly correlated with inversion genotype. We leveraged 1,347 fully sequenced An. gambiae and Anopheles coluzzii genomes in the Ag1000G database of natural variation. Beginning with principal components analysis (PCA) of population samples, applied to windows of the genome containing individual chromosomal rearrangements, we classified samples into three inversion genotypes, distinguishing homozygous inverted and homozygous uninverted groups by inclusion of the small subset of specimens in Ag1000G that are associated with cytogenetic metadata. We then assessed the correlation between candidate tag SNP genotypes and PCA-based inversion genotypes in our training sets, selecting those candidates with &gt;80% agreement. Our initial tests both in held-back validation samples from Ag1000G and in data independent of Ag1000G suggest that when used for in silico inversion genotyping of sequenced mosquitoes, these tags perform better than traditional cytogenetics, even for specimens where only a small subset of the tag SNPs can be successfully ascertained." />
<meta name="DC.Contributor" content="R. Rebecca Love" />
<meta name="DC.Contributor" content="Seth N. Redmond" />
<meta name="DC.Contributor" content="Marco Pombi" />
<meta name="DC.Contributor" content="Beniamino Caputo" />
<meta name="DC.Contributor" content="Vincenzo Petrarca" />
<meta name="DC.Contributor" content="Alessandra della Torre" />
<meta name="DC.Contributor" content="The Anopheles gambiae 1000 Genomes Consortium" />
<meta name="DC.Contributor" content="Nora J. Besansky" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Chromosomal inversion polymorphisms play an important role in adaptation to environmental heterogeneities. For mosquito species in the &lt;i&gt;Anopheles gambiae&lt;/i&gt; complex that are significant vectors of human malaria, paracentric inversion polymorphisms are abundant and are associated with ecologically and epidemiologically important phenotypes. Improved understanding of these traits relies on determining mosquito karyotype, which currently depends upon laborious cytogenetic methods whose application is limited both by the requirement for specialized expertise and for properly preserved adult females at specific gonotrophic stages. To overcome this limitation, we developed sets of tag single nucleotide polymorphisms (SNPs) inside inversions whose biallelic genotype is strongly correlated with inversion genotype. We leveraged 1,347 fully sequenced &lt;i&gt;An. gambiae&lt;/i&gt; and &lt;i&gt;Anopheles coluzzii&lt;/i&gt; genomes in the Ag1000G database of natural variation. Beginning with principal components analysis (PCA) of population samples, applied to windows of the genome containing individual chromosomal rearrangements, we classified samples into three inversion genotypes, distinguishing homozygous inverted and homozygous uninverted groups by inclusion of the small subset of specimens in Ag1000G that are associated with cytogenetic metadata. We then assessed the correlation between candidate tag SNP genotypes and PCA-based inversion genotypes in our training sets, selecting those candidates with &amp;gt;80% agreement. Our initial tests both in held-back validation samples from Ag1000G and in data independent of Ag1000G suggest that when used for &lt;i&gt;in silico&lt;/i&gt; inversion genotyping of sequenced mosquitoes, these tags perform better than traditional cytogenetics, even for specimens where only a small subset of the tag SNPs can be successfully ascertained.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3249" />
<meta name="citation_id" content="9/10/3249" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3249" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3249.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3249.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3249.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400445" />
<meta name="citation_pmid" content="31391198" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3249" />
<meta name="citation_lastpage" content="3262" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="R. Rebecca Love" />
<meta name="citation_author_institution" content="Eck Institute for Global Health &amp; Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8725-3929" />
<meta name="citation_author" content="Seth N. Redmond" />
<meta name="citation_author_institution" content="Infectious Disease and Microbiome Program, Broad Institute, Cambridge, MA 02142" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2653-760X" />
<meta name="citation_author" content="Marco Pombi" />
<meta name="citation_author_institution" content="Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy" />
<meta name="citation_author" content="Beniamino Caputo" />
<meta name="citation_author_institution" content="Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy" />
<meta name="citation_author" content="Vincenzo Petrarca" />
<meta name="citation_author_institution" content="Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7939-6786" />
<meta name="citation_author" content="Alessandra della Torre" />
<meta name="citation_author_institution" content="Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy" />
<meta name="citation_author" content="The Anopheles gambiae 1000 Genomes Consortium" />
<meta name="citation_author" content="Nora J. Besansky" />
<meta name="citation_author_institution" content="Eck Institute for Global Health &amp; Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556" />
<meta name="citation_author_email" content="nbesansk@nd.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0646-0721" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=P. Andolfatto;citation_author=F. Depaulis;citation_author=A. Navarro;citation_title=Inversion polymorphisms and nucleotide variability in Drosophila.;citation_pages=1-8;citation_volume=77;citation_year=2001;citation_issue=1;citation_pmid=11279826;citation_doi=10.1017/S0016672301004955" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_title=Genetic diversity of the African malaria vector Anopheles gambiae.;citation_pages=96-100;citation_volume=552;citation_year=2017;citation_pmid=29186111;citation_doi=10.1038/nature24995" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. Ayala;citation_author=P. Acevedo;citation_author=M. Pombi;citation_author=I. Dia;citation_author=D. Boccolini;citation_title=Chromosome inversions and ecological plasticity in the main African malaria mosquitoes.;citation_pages=686-701;citation_volume=71;citation_year=2017;citation_doi=10.1111/evo.13176" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=D. Ayala;citation_author=A. Ullastres;citation_author=J. Gonzalez;citation_title=Adaptation through chromosomal inversions in Anopheles.;citation_pages=129;citation_volume=5;citation_year=2014;citation_pmid=24904633;citation_doi=10.3389/fgene.2014.00129" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. Ayala;citation_author=S. Zhang;citation_author=M. Chateau;citation_author=C. Fouet;citation_author=I. Morlais;citation_title=Association mapping desiccation resistance within chromosomal inversions in the African malaria vector Anopheles gambiae.;citation_pages=1333-1342;citation_volume=28;citation_year=2019;citation_doi=10.1111/mec.14880" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=J. H. Bryan;citation_author=M. A. Di Deco;citation_author=V. Petrarca;citation_author=M. Coluzzi;citation_title=Inversion polymorphism and incipient speciation in Anopheles gambiae s. str. in The Gambia, West Africa.;citation_pages=167-176;citation_volume=59;citation_year=1982;citation_doi=10.1007/BF00056539" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. Caputo;citation_author=D. Nwakanma;citation_author=F. P. Caputo;citation_author=M. Jawara;citation_author=E. C. Oriero;citation_title=Prominent intra-specific genetic divergence within Anopheles gambiae sibling species triggered by habitat discontinuities across a riverine landscape.;citation_pages=4574-4589;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12866" />
<meta name="citation_reference" content="citation_journal_title=Malaria journal [electronic resource];citation_journal_abbrev=Malar J;citation_author=B. Caputo;citation_author=D. Nwakanma;citation_author=M. Jawara;citation_author=M. Adiamoh;citation_author=I. Dia;citation_title=Anopheles gambiae complex along The Gambia river, with particular reference to the molecular forms of An. gambiae s.s.;citation_pages=182-182;citation_volume=7;citation_year=2008;citation_pmid=18803885;citation_doi=10.1186/1475-2875-7-182" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Caputo;citation_author=F. Santolamazza;citation_author=J. L. Vicente;citation_author=D. C. Nwakanma;citation_author=M. Jawara;citation_title=The &quot;far-west&quot; of Anopheles gambiae molecular forms.;citation_pages=e16415-e16415;citation_volume=6;citation_year=2011;citation_issue=2;citation_pmid=21347223;citation_doi=10.1371/journal.pone.0016415" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=B. J. Cassone;citation_author=M. J. Molloy;citation_author=C. Cheng;citation_author=J. C. Tan;citation_author=M. W. Hahn;citation_title=Divergent transcriptional response to thermal stress by Anopheles gambiae larvae carrying alternative arrangements of inversion 2La.;citation_pages=2567-2580;citation_volume=20;citation_year=2011;citation_issue=12;citation_pmid=21535279;citation_doi=10.1111/j.1365-294X.2011.05114.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Cheng;citation_author=J. C. Tan;citation_author=M. W. Hahn;citation_author=N. J. Besansky;citation_title=A systems genetic analysis of inversion polymorphisms in the malaria mosquito Anopheles gambiae.;citation_volume=115;citation_year=2018;citation_pmid=29987007;citation_doi=10.1073/pnas.1806760115" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Cheng;citation_author=B. J. White;citation_author=C. Kamdem;citation_author=K. Mockaitis;citation_author=C. Costantini;citation_title=Ecological genomics of Anopheles gambiae along a latitudinal cline: a population-resequencing approach.;citation_volume=190;citation_year=2012;citation_pmid=22209907;citation_doi=10.1534/genetics.111.137794" />
<meta name="citation_reference" content="citation_journal_title=Zootaxa;citation_author=M. Coetzee;citation_author=R. H. Hunt;citation_author=R. Wilkerson;citation_author=A. della Torre;citation_author=M. B. Coulibaly;citation_title=Anopheles coluzzii and Anopheles amharicus, new members of the Anopheles gambiae complex.;citation_pages=246-274;citation_volume=3619;citation_year=2013;citation_pmid=26131476;citation_doi=10.11646/zootaxa.3619.3.2" />
<meta name="citation_reference" content="citation_journal_title=Parassitologia;citation_author=M. Coluzzi;citation_title=Cromosomi politenici delle cellule nutrici ovariche nel complesso gambiae del genere Anopheles.;citation_pages=179-183;citation_volume=10;citation_year=1968" />
<meta name="citation_reference" content="citation_journal_title=Boll. Zool.;citation_author=M. Coluzzi;citation_author=V. Petrarca;citation_author=M. A. DiDeco;citation_title=Chromosomal inversion intergradation and incipient speciation in Anopheles gambiae.;citation_pages=45-63;citation_volume=52;citation_year=1985;citation_doi=10.1080/11250008509440343" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Coluzzi;citation_author=A. Sabatini;citation_author=A. della Torre;citation_author=M. A. Di Deco;citation_author=V. Petrarca;citation_title=A polytene chromosome analysis of the Anopheles gambiae species complex.;citation_volume=298;citation_year=2002;citation_pmid=12364623;citation_doi=10.1126/science.1077769" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Trop. Med. Hyg.;citation_author=M. Coluzzi;citation_author=A. Sabatini;citation_author=V. Petrarca;citation_author=M. A. Di Deco;citation_title=Chromosomal differentiation and adaptation to human environments in the Anopheles gambiae complex.;citation_volume=73;citation_year=1979;citation_pmid=394408;citation_doi=10.1016/0035-9203(79)90036-1" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=R. Corbett-Detig;citation_author=I. Said;citation_author=M. Calzetta;citation_author=M. Genetti;citation_author=J. McBroome;citation_title=Fine-mapping complex inversion breakpoints and investigating somatic pairing in the Anopheles gambiae species complex using proximity-ligation sequencing.;citation_year=2019;citation_doi=10.1101/662114" />
<meta name="citation_reference" content="citation_journal_title=BMC ecology [electronic resource];citation_journal_abbrev=BMC Ecol;citation_author=C. Costantini;citation_author=D. Ayala;citation_author=W. M. Guelbeogo;citation_author=M. Pombi;citation_author=C. Y. Some;citation_title=Living at the edge: biogeographic patterns of habitat segregation conform to speciation by niche expansion in Anopheles gambiae.;citation_pages=16-16;citation_volume=9;citation_year=2009;citation_pmid=19460144;citation_doi=10.1186/1472-6785-9-16" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. B. Coulibaly;citation_author=N. F. Lobo;citation_author=M. C. Fitzpatrick;citation_author=M. Kern;citation_author=O. Grushko;citation_title=Segmental duplication implicated in the genesis of inversion 2Rj of Anopheles gambiae.;citation_pages=e849-e849;citation_volume=2;citation_year=2007;citation_issue=9;citation_pmid=17786220;citation_doi=10.1371/journal.pone.0000849" />
<meta name="citation_reference" content="citation_journal_title=Malaria journal [electronic resource];citation_journal_abbrev=Malar J;citation_author=M. B. Coulibaly;citation_author=M. Pombi;citation_author=B. Caputo;citation_author=D. Nwakanma;citation_author=M. Jawara;citation_title=PCR-based karyotyping of Anopheles gambiae inversion 2Rj identifies the BAMAKO chromosomal form.;citation_pages=133-133;citation_volume=6;citation_year=2007;citation_pmid=17908310;citation_doi=10.1186/1475-2875-6-133" />
<meta name="citation_reference" content="citation_journal_title=Med. Vet. Entomol.;citation_author=K. R. Dabire;citation_author=S. Sawadodgo;citation_author=A. Diabate;citation_author=K. H. Toe;citation_author=P. Kengne;citation_title=Assortative mating in mixed swarms of the mosquito Anopheles gambiae s.s. M and S molecular forms, in Burkina Faso, West Africa.;citation_pages=298-312;citation_volume=27;citation_year=2013;citation_pmid=23360106;citation_doi=10.1111/j.1365-2915.2012.01049.x" />
<meta name="citation_reference" content="citation_title=Molecular Biology of Disease Vectors: A Methods Manual;citation_author=A. della Torre;citation_title=Polytene chromosome preparation from anopheline mosquitoes;citation_year=1997;citation_doi=10.1007/978-94-009-1535-0_28" />
<meta name="citation_reference" content="citation_journal_title=Insect molecular biology;citation_journal_abbrev=Insect Mol Biol;citation_author=A. della Torre;citation_author=C. Fanello;citation_author=M. Akogbeto;citation_author=J. Dossou-yovo;citation_author=G. Favia;citation_title=Molecular evidence of incipient speciation within Anopheles gambiae s.s. in West Africa.;citation_pages=9-18;citation_volume=10;citation_year=2001;citation_issue=1;citation_pmid=11240632;citation_doi=10.1046/j.1365-2583.2001.00235.x" />
<meta name="citation_reference" content="citation_journal_title=Insect biochemistry and molecular biology;citation_journal_abbrev=Insect Biochem Mol Biol;citation_author=A. della Torre;citation_author=Z. Tu;citation_author=V. Petrarca;citation_title=On the distribution and genetic differentiation of Anopheles gambiae s.s. molecular forms.;citation_pages=755-769;citation_volume=35;citation_year=2005;citation_issue=7;citation_pmid=15894192;citation_doi=10.1016/j.ibmb.2005.02.006" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=A. Diabaté;citation_author=A. Dao;citation_author=A. S. Yaro;citation_author=A. Adamou;citation_author=R. Gonzalez;citation_title=Spatial swarm segregation and reproductive isolation between the molecular forms of Anopheles gambiae.;citation_volume=276;citation_year=2009;citation_pmid=19734189;citation_doi=10.1098/rspb.2009.1167" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. C. Fontaine;citation_author=J. B. Pease;citation_author=A. Steele;citation_author=R. M. Waterhouse;citation_author=D. E. Neafsey;citation_title=Extensive introgression in a malaria vector species complex revealed by phylogenomics.;citation_volume=347;citation_year=2015;citation_pmid=25431491;citation_doi=10.1126/science.1258524" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. Fouet;citation_author=E. Gray;citation_author=N. J. Besansky;citation_author=C. Costantini;citation_title=Adaptation to aridity in the malaria mosquito Anopheles gambiae: chromosomal inversion polymorphism and body size influence resistance to desiccation.;citation_pages=e34841-e34841;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22514674;citation_doi=10.1371/journal.pone.0034841" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=Z. L. Fuller;citation_author=G. D. Haynes;citation_author=S. Richards;citation_author=S. W. Schaeffer;citation_title=Genomics of natural populations: Evolutionary forces that establish and maintain gene arrangements in Drosophila pseudoobscura.;citation_pages=6539-6562;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14381" />
<meta name="citation_reference" content="citation_journal_title=Behav. Ecol.;citation_author=G. Gimonneau;citation_author=J. Bouyer;citation_author=S. Morand;citation_author=N. J. Besansky;citation_author=A. Diabate;citation_title=A behavioral mechanism underlying ecological divergence in the malaria mosquito Anopheles gambiae.;citation_volume=21;citation_year=2010;citation_pmid=22476108;citation_doi=10.1093/beheco/arq114" />
<meta name="citation_reference" content="citation_journal_title=Medical and veterinary entomology;citation_journal_abbrev=Med Vet Entomol;citation_author=G. Gimonneau;citation_author=M. Pombi;citation_author=M. Choisy;citation_author=S. Morand;citation_author=R. K. Dabire;citation_title=Larval habitat segregation between the molecular forms of the mosquito Anopheles gambiae in a rice field area of Burkina Faso, West Africa.;citation_pages=9-17;citation_volume=26;citation_year=2012;citation_issue=1;citation_pmid=21501199;citation_doi=10.1111/j.1365-2915.2011.00957.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Parasit Vectors;citation_author=G. Gimonneau;citation_author=M. Pombi;citation_author=R. K. Dabire;citation_author=A. Diabate;citation_author=S. Morand;citation_title=Behavioural responses of Anopheles gambiae sensu stricto M and S molecular form larvae to an aquatic predator in Burkina Faso.;citation_pages=65-65;citation_volume=5;citation_year=2012;citation_pmid=22463735;citation_doi=10.1186/1756-3305-5-65" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. I. Giraldo-Calderón;citation_author=S. J. Emrich;citation_author=R. M. MacCallum;citation_author=G. Maslen;citation_author=E. Dialynas;citation_title=VectorBase: an updated bioinformatics resource for invertebrate vectors and other organisms related with human diseases.;citation_volume=43;citation_year=2015;citation_pmid=25510499;citation_doi=10.1093/nar/gku1117" />
<meta name="citation_reference" content="citation_journal_title=Malaria journal [electronic resource];citation_journal_abbrev=Malar J;citation_author=E. M. Gray;citation_author=K. A. Rocca;citation_author=C. Costantini;citation_author=N. J. Besansky;citation_title=Inversion 2La is associated with enhanced desiccation resistance in Anopheles gambiae.;citation_pages=215-215;citation_volume=8;citation_year=2009;citation_pmid=19772577;citation_doi=10.1186/1475-2875-8-215" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. J. Hanemaaijer;citation_author=T. C. Collier;citation_author=A. Chang;citation_author=C. C. Shott;citation_author=P. D. Houston;citation_title=The fate of genes that cross species boundaries after a major hybridization event in a natural mosquito population.;citation_pages=4978-4990;citation_volume=27;citation_year=2018;citation_doi=10.1111/mec.14947" />
<meta name="citation_reference" content="citation_journal_abbrev=Annu Rev Ecol Evol Syst;citation_author=A. A. Hoffmann;citation_author=L. H. Rieseberg;citation_title=Revisiting the Impact of Inversions in Evolution: From Population Genetic Markers to Drivers of Adaptive Shifts and Speciation?;citation_pages=21-42;citation_volume=39;citation_year=2008;citation_pmid=20419035;citation_doi=10.1146/annurev.ecolsys.39.110707.173532" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=A. A. Hoffmann;citation_author=C. M. Sgro;citation_author=A. R. Weeks;citation_title=Chromosomal inversion polymorphisms and adaptation.;citation_pages=482-488;citation_volume=19;citation_year=2004;citation_issue=9;citation_pmid=16701311;citation_doi=10.1016/j.tree.2004.06.013" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. A. Holt;citation_author=G. M. Subramanian;citation_author=A. Halpern;citation_author=G. G. Sutton;citation_author=R. Charlab;citation_title=The Genome Sequence of the Malaria Mosquito Anopheles gambiae;citation_pages=129-149;citation_volume=298;citation_year=2002;citation_issue=5591;citation_pmid=12364791;citation_doi=10.1126/science.1076181" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Houle;citation_author=E. J. Marquez;citation_title=Linkage disequilibrium and inversion-typing of the Drosophila melanogaster genome reference panel.;citation_volume=5;citation_year=2015;citation_pmid=26068573;citation_doi=10.1534/g3.115.019554" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=484;citation_year=2012;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Joron;citation_author=L. Frezal;citation_author=R. T. Jones;citation_author=N. L. Chamberlain;citation_author=S. F. Lee;citation_title=Chromosomal rearrangements maintain a polymorphic supergene controlling butterfly mimicry.;citation_pages=203-206;citation_volume=477;citation_year=2011;citation_issue=7363;citation_pmid=21841803;citation_doi=10.1038/nature10341" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Kapun;citation_author=D. K. Fabian;citation_author=J. Goudet;citation_author=T. Flatt;citation_title=Genomic evidence for adaptive inversion clines in Drosophila melanogaster.;citation_volume=33;citation_year=2016;citation_pmid=26796550;citation_doi=10.1093/molbev/msw016" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. Kirkpatrick;citation_title=How and why chromosome inversions evolve.;citation_pages=e1000501;citation_volume=8;citation_year=2010;citation_pmid=20927412;citation_doi=10.1371/journal.pbio.1000501" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Kirkpatrick;citation_author=B. Barrett;citation_title=Chromosome inversions, adaptive cassettes and the evolution of species’ ranges.;citation_pages=2046-2055;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13074" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. Lemoine;citation_author=J. B. Domelevo Entfellner;citation_author=E. Wilkinson;citation_author=D. Correia;citation_author=M. Davila Felipe;citation_title=Renewing Felsenstein’s phylogenetic bootstrap in the era of big data.;citation_pages=452-456;citation_volume=556;citation_year=2018;citation_doi=10.1038/s41586-018-0043-0" />
<meta name="citation_reference" content="citation_journal_title=Malaria journal [electronic resource];citation_journal_abbrev=Malar J;citation_author=N. F. Lobo;citation_author=D. M. Sangare;citation_author=A. A. Regier;citation_author=K. R. Reidenbach;citation_author=D. A. Bretz;citation_title=Breakpoint structure of the Anopheles gambiae 2Rb chromosomal inversion.;citation_pages=293-293;citation_volume=9;citation_year=2010;citation_pmid=20974007;citation_doi=10.1186/1475-2875-9-293" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. R. Love;citation_author=A. M. Steele;citation_author=M. B. Coulibaly;citation_author=S. F. Traore;citation_author=S. J. Emrich;citation_title=Chromosomal inversions and ecotypic differentiation in Anopheles gambiae: the perspective from whole-genome sequencing.;citation_pages=5889-5906;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13888" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=D. B. Lowry;citation_author=J. H. Willis;citation_title=A widespread chromosomal inversion polymorphism contributes to a major life-history transition, local adaptation, and reproductive isolation.;citation_pages=e1000500;citation_volume=8;citation_year=2010;citation_pmid=20927411;citation_doi=10.1371/journal.pbio.1000500" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Ma;citation_author=C. I. Amos;citation_title=Investigation of inversion polymorphisms in the human genome using principal components analysis.;citation_pages=e40224-e40224;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22808122;citation_doi=10.1371/journal.pone.0040224" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. J. Main;citation_author=Y. Lee;citation_author=T. C. Collier;citation_author=L. C. Norris;citation_author=K. Brisco;citation_title=Complex genome evolution in Anopheles coluzzii associated with increased insecticide usage in Mali.;citation_pages=5145-5157;citation_volume=24;citation_year=2015;citation_pmid=26359110;citation_doi=10.1111/mec.13382" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. C. Manoukis;citation_author=J. R. Powell;citation_author=M. B. Touré;citation_author=A. Sacko;citation_author=F. E. Edillo;citation_title=A test of the chromosomal theory of ecotypic speciation in Anopheles gambiae.;citation_volume=105;citation_year=2008;citation_pmid=18287019;citation_doi=10.1073/pnas.0709806105" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=C. Marsden;citation_author=Y. Lee;citation_author=C. Neimen;citation_author=M. Sanford;citation_author=J. Dinis;citation_title=Asymmetric introgression between the M and S forms of the malaria vector, Anopheles gambiae, maintains divergence despite extensive hybridization.;citation_pages=4983-4994;citation_volume=20;citation_year=2011;citation_issue=23;citation_pmid=22059383;citation_doi=10.1111/j.1365-294X.2011.05339.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Navarro;citation_author=E. Betran;citation_author=A. Barbadilla;citation_author=A. Ruiz;citation_title=Recombination and Gene Flux Caused by Gene Conversion and Crossing Over in Inversion Heterokaryotypes;citation_pages=695-709;citation_volume=146;citation_year=1997;citation_issue=2;citation_pmid=9178017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. C. Nwakanma;citation_author=D. E. Neafsey;citation_author=M. Jawara;citation_author=M. Adiamoh;citation_author=E. Lund;citation_title=Breakdown in the process of incipient speciation in Anopheles gambiae.;citation_volume=193;citation_year=2013;citation_pmid=23335339;citation_doi=10.1534/genetics.112.148718" />
<meta name="citation_reference" content="citation_journal_title=Journal of Medical Entomology;citation_journal_abbrev=J Med Entomol;citation_author=E. Oliveira;citation_author=P. Salgueiro;citation_author=K. Palsson;citation_author=J. L. Vicente;citation_author=A. P. Arez;citation_title=High levels of hybridization between molecular forms of Anopheles gambiae from Guinea Bissau.;citation_pages=1057-1063;citation_volume=45;citation_year=2008;citation_issue=6;citation_pmid=19058629;citation_doi=10.1603/0022-2585(2008)45[1057:HLOHBM]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=F. Pedregosa;citation_author=G. Varoquaux;citation_author=A. Gramfort;citation_author=V. Michel;citation_author=B. Thirion;citation_title=Scikit-learn: Machine Learning in Python.;citation_pages=2825-2830;citation_volume=12;citation_year=2011;citation_pmid=23755062;citation_doi=10.1016/j.cpc.2010.04.018" />
<meta name="citation_reference" content="citation_journal_title=The American Journal of Tropical Medicine and Hygiene;citation_journal_abbrev=Am J Trop Med Hyg;citation_author=V. Petrarca;citation_author=J. C. Beier;citation_title=Intraspecific Chromosomal Polymorphism in the Anopheles Gambiae Complex as a Factor Affecting Malaria Transmission in the Kisumu Area of Kenya;citation_pages=229-237;citation_volume=46;citation_year=1992;citation_issue=2;citation_pmid=1539757;citation_doi=10.4269/ajtmh.1992.46.229" />
<meta name="citation_reference" content="citation_journal_title=Parassitologia (Roma);citation_journal_abbrev=Parassitologia (Roma);citation_author=V. Petrarca;citation_author=G. Sabatinelli;citation_author=M. A. Di Deco;citation_author=M. Papakay;citation_title=The Anopheles gambiae complex in the Federal Islamic Republic of Comoros (Indian Ocean): some cytogenetic and biometric data.;citation_pages=371-380;citation_volume=32;citation_year=1990;citation_issue=3;citation_pmid=2132451" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=M. Pombi;citation_author=B. Caputo;citation_author=F. Simard;citation_author=M. A. Di Deco;citation_author=M. Coluzzi;citation_title=Chromosomal plasticity and evolutionary potential in the malaria vector Anopheles gambiae sensu stricto: insights from three decades of rare paracentric inversions.;citation_pages=309-309;citation_volume=8;citation_year=2008;citation_pmid=19000304;citation_doi=10.1186/1471-2148-8-309" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=M. M. Riehle;citation_author=T. Bukhari;citation_author=A. Gneme;citation_author=W. M. Guelbeogo;citation_author=B. Coulibaly;citation_title=The Anopheles gambiae 2La chromosome inversion is associated with susceptibility to Plasmodium falciparum in Africa.;citation_volume=6;citation_year=2017;citation_doi=10.7554/eLife.25813" />
<meta name="citation_reference" content="citation_journal_title=Acta tropica;citation_journal_abbrev=Acta Trop;citation_author=N. Rishikesh;citation_author=M. A. Di Deco;citation_author=V. Petrarca;citation_author=M. Coluzzi;citation_title=Seasonal variations in indoor resting Anopheles gambiae and Anopheles arabiensis in Kaduna, Nigeria.;citation_pages=165-170;citation_volume=42;citation_year=1985;citation_issue=2;citation_pmid=2862779" />
<meta name="citation_reference" content="citation_journal_title=Malaria journal [electronic resource];citation_journal_abbrev=Malar J;citation_author=K. A. Rocca;citation_author=E. M. Gray;citation_author=C. Costantini;citation_author=N. J. Besansky;citation_title=2La chromosomal inversion enhances thermal tolerance of Anopheles gambiae larvae.;citation_pages=147-147;citation_volume=8;citation_year=2009;citation_pmid=19573238;citation_doi=10.1186/1475-2875-8-147" />
<meta name="citation_reference" content="Sangare, D. M., 2007 Breakpoint analysis of the Anopheles gambiae s.s. chromosome 2Rb, 2Rc, and 2Ru inversions in PhD Thesis, Graduate Program in Biological Sciences, University of Notre Dame. University of Notre Dame, Notre Dame, IN." />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. W. Schaeffer;citation_title=Selection in heterogeneous environments maintains the gene arrangement polymorphism of Drosophila pseudoobscura.;citation_pages=3082-3099;citation_volume=62;citation_year=2008;citation_issue=12;citation_pmid=18764919;citation_doi=10.1111/j.1558-5646.2008.00504.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. K. Seich Al Basatena;citation_author=C. J. Hoggart;citation_author=L. J. Coin;citation_author=P. F. O’Reilly;citation_title=The effect of genomic inversions on estimation of population genetic parameters from SNP data.;citation_volume=193;citation_year=2013;citation_pmid=23150602;citation_doi=10.1534/genetics.112.145599" />
<meta name="citation_reference" content="citation_journal_title=BMC ecology [electronic resource];citation_journal_abbrev=BMC Ecol;citation_author=F. Simard;citation_author=D. Ayala;citation_author=G. C. Kamdem;citation_author=M. Pombi;citation_author=J. Etouna;citation_title=Ecological niche partitioning between Anopheles gambiae molecular forms in Cameroon: the ecological side of speciation.;citation_pages=17-17;citation_volume=9;citation_year=2009;citation_pmid=19460146;citation_doi=10.1186/1472-6785-9-17" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=B. Tene Fossog;citation_author=D. Ayala;citation_author=P. Acevedo;citation_author=P. Kengne;citation_author=I. Ngomo Abeso Mebuy;citation_title=Habitat segregation and ecological character displacement in cryptic African malaria mosquitoes.;citation_pages=326-345;citation_volume=8;citation_year=2015;citation_pmid=25926878;citation_doi=10.1111/eva.12242" />
<meta name="citation_reference" content="citation_journal_title=Parassitologia (Roma);citation_journal_abbrev=Parassitologia (Roma);citation_author=Y. T. Toure;citation_author=V. Petrarca;citation_author=S. F. Traore;citation_author=A. Coulibaly;citation_author=H. M. Maiga;citation_title=The distribution and inversion polymorphism of chromosomally recognized taxa of the Anopheles gambiae complex in Mali, West Africa.;citation_pages=477-511;citation_volume=40;citation_year=1998;citation_issue=4;citation_pmid=10645562" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=A. D. Twyford;citation_author=J. Friedman;citation_title=Adaptive divergence in the monkey flower Mimulus guttatus is maintained by a chromosomal inversion.;citation_pages=1476-1486;citation_volume=69;citation_year=2015;citation_pmid=25879251;citation_doi=10.1111/evo.12663" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Weetman;citation_author=C. S. Wilding;citation_author=K. Steen;citation_author=J. Pinto;citation_author=M. J. Donnelly;citation_title=Gene flow-dependent genomic divergence between Anopheles gambiae M and S forms.;citation_volume=29;citation_year=2012;citation_pmid=21836185;citation_doi=10.1093/molbev/msr199" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=M. Wellenreuther;citation_author=L. Bernatchez;citation_title=Eco-Evolutionary Genomics of Chromosomal Inversions.;citation_pages=427-440;citation_volume=33;citation_year=2018;citation_doi=10.1016/j.tree.2018.04.002" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=M. Wellenreuther;citation_author=H. Rosenquist;citation_author=P. Jaksons;citation_author=K. W. Larson;citation_title=Local adaptation along an environmental cline in a species with an inversion polymorphism.;citation_pages=1068-1077;citation_volume=30;citation_year=2017;citation_doi=10.1111/jeb.13064" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=B. J. White;citation_author=F. H. Collins;citation_author=N. J. Besansky;citation_title=Evolution of Anopheles gambiae in relation to humans and malaria.;citation_pages=111-132;citation_volume=42;citation_year=2011;citation_doi=10.1146/annurev-ecolsys-102710-145028" />
<meta name="citation_reference" content="citation_journal_title=The American Journal of Tropical Medicine and Hygiene;citation_journal_abbrev=Am J Trop Med Hyg;citation_author=B. J. White;citation_author=F. Santolamazza;citation_author=L. Kamau;citation_author=M. Pombi;citation_author=O. Grushko;citation_title=MOLECULAR KARYOTYPING OF THE 2LA INVERSION IN ANOPHELES GAMBIAE;citation_pages=334-339;citation_volume=76;citation_year=2007;citation_issue=2;citation_pmid=17297045;citation_doi=10.4269/ajtmh.2007.76.334" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Chromosomal inversion polymorphisms play an important role in adaptation to environmental heterogeneities. For mosquito species in the Anopheles gambiae complex that are significant vectors of human malaria, paracentric inversion polymorphisms are abundant and are associated with ecologically and epidemiologically important phenotypes. Improved understanding of these traits relies on determining mosquito karyotype, which currently depends upon laborious cytogenetic methods whose application is limited both by the requirement for specialized expertise and for properly preserved adult females at specific gonotrophic stages. To overcome this limitation, we developed sets of tag single nucleotide polymorphisms (SNPs) inside inversions whose biallelic genotype is strongly correlated with inversion genotype. We leveraged 1,347 fully sequenced An. gambiae and Anopheles coluzzii genomes in the Ag1000G database of natural variation. Beginning with principal components analysis (PCA) of population samples, applied to windows of the genome containing individual chromosomal rearrangements, we classified samples into three inversion genotypes, distinguishing homozygous inverted and homozygous uninverted groups by inclusion of the small subset of specimens in Ag1000G that are associated with cytogenetic metadata. We then assessed the correlation between candidate tag SNP genotypes and PCA-based inversion genotypes in our training sets, selecting those candidates with &gt;80% agreement. Our initial tests both in held-back validation samples from Ag1000G and in data independent of Ag1000G suggest that when used for in silico inversion genotyping of sequenced mosquitoes, these tags perform better than traditional cytogenetics, even for specimens where only a small subset of the tag SNPs can be successfully ascertained." />
<meta name="og-title" property="og:title" content="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3249" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Chromosomal inversion polymorphisms play an important role in adaptation to environmental heterogeneities. For mosquito species in the Anopheles gambiae complex that are significant vectors of human malaria, paracentric inversion polymorphisms are abundant and are associated with ecologically and epidemiologically important phenotypes. Improved understanding of these traits relies on determining mosquito karyotype, which currently depends upon laborious cytogenetic methods whose application is limited both by the requirement for specialized expertise and for properly preserved adult females at specific gonotrophic stages. To overcome this limitation, we developed sets of tag single nucleotide polymorphisms (SNPs) inside inversions whose biallelic genotype is strongly correlated with inversion genotype. We leveraged 1,347 fully sequenced An. gambiae and Anopheles coluzzii genomes in the Ag1000G database of natural variation. Beginning with principal components analysis (PCA) of population samples, applied to windows of the genome containing individual chromosomal rearrangements, we classified samples into three inversion genotypes, distinguishing homozygous inverted and homozygous uninverted groups by inclusion of the small subset of specimens in Ag1000G that are associated with cytogenetic metadata. We then assessed the correlation between candidate tag SNP genotypes and PCA-based inversion genotypes in our training sets, selecting those candidates with &gt;80% agreement. Our initial tests both in held-back validation samples from Ag1000G and in data independent of Ag1000G suggest that when used for in silico inversion genotyping of sequenced mosquitoes, these tags perform better than traditional cytogenetics, even for specimens where only a small subset of the tag SNPs can be successfully ascertained." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FasYI8BEPpebUzBiFwjWnuTSSike6dQJEfkfCTVw-Sg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494275","apath":"\/ggg\/9\/10\/3249.atom","pisa":"ggg;9\/10\/3249","ac":{"\/ggg\/9\/10\/3249.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3249.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3249"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3249"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3249":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LIoTrIQ0BwYBOEd7QQtGbkfbvxo0VB2FmknC+MNFk9OnnflgGYsBFpBYTD2shB+9iKD9m7FuABgVFqBUTPOOlKc9ikbG9lZGT7kYL3YmGq0XQE7L30yvvqJ3T+odmZg+BdcwRT8Ll4RpU6w94POTxviP8WG2Bl5sO7BybFWzNUpPZHScud0U0FcmQDixTYKrKLb2lt0PNvP3\\\\\\\/Zz5Y0K6a6NgXtRAhEFlTHYC4A\\\\\\\/AOZVEDqlhXX1bK5G82IqVlmt0SvjEHePyFxIfDwlhHq6JLBu1LE3Y0T75U4GmDQE2+mS5ktzc44xwBpm2w8IaqLubkSWo+ZMBvzEFZ0glinDBUAG5J5HMiPDxR7+72vQtQy15kryLzae4oC2pGtsRhyoaa\\\\\\\/DKudLbwQZ9mITbhS0KAti3rj3V3u0Pb8TXTsJQ9vLD664GsckSk97DshSvMBNYVI\\\\\\\/NBEuRVPtoK6KtRx3fSwLRVaTm\\\\\\\/eHqUycn7VZ9pyNWW+pEOsve2SplcJn7ddmpFAexKjUZQ\\\\\\\/dA1cf8BrwhmepsZKkdKV0908rRNbdMtIGb0+cBhRkqnH7\\\\\\\/Ai4viE\\\\\\\/\\\\\\\/4lA8zRqzBerHGkD8xpCAjPIoJIH\\\\\\\/tejFhBp0P5GHwnYBqa7VGE+WdO59LFbdELs3Xw9mZmthOz3bsqx4g9+gL8k2G5Fp87QB0ZmkqEhAOaH\\\\\\\/wcktadTOJBbz5X4iRqhxoQrCLOZ4dsnycDasbJqoEWsqTA\\\\\\\/gYDGs6\\\\\\\/DOfyC+IT0lTqf+whYZm8uBY9vsWZ5n5Gp99oyIjqXDHm\\\\\\\/uzZRZo0VeV1gIHgbupuxDLIwe6LTNrkFKGzz8UeeGTDRN80zphVpbnTdTetK8nvXhykFG1CIwwilERJNgcXPy6ppaSb\\\\\\\/s2M6oln7KG4FKqZ3n\\\\\\\/XT4n6BIeHBrcsy04ffCwdkZWH\\\\\\\/VQeqqkJ1uPg+omuaCEHoivgM3XNwnqziqgILUWais4x3Mg5hlKwchD467fl2J4JSSRADCtiP41sYgPKxpfCWwHzGKKs0rf4oGRY5wXS1aX2jN1+6csaxiKOSQYjClePIn70oOHbZaFfwGqMSeR79YFTfjMgvRb4gBfiYgR1HcEGHJ03Djwcine5EYURCoN+qes5zWPDqiznH9aWiddqNIwJMOpaUxnyHaomBbqVRVzlFTRXxkXva8TLRL\\\\\\\/dstV4EOuOxmhjLeaTwyeqyKvw5BswLPrIYFwaomsL6a2whJW3nSnvx\\\\\\\/+4qOPWzIwLGce2u5xNDooh3jeEtq\\\\\\\/9SPqhpHN1KYLDzM6VBRs5rhEeNS\\\u0022,\\\u0022iv\\\u0022:\\\u00222lfjZ2ElpGw8ArCcFJDUHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221a8c64dab90a8cd09b6aa051943e20198ee36131df0e7fad3f22ec7516034586\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bp6ypa+BdmbXzcSvsWswk+nD8gpI04AevhZf2KUwK73bXtoouVstxShkY2O7GolQIiidnXzEjMO3d7ADZyZK+Qsb0EU\\\\\\\/WKZ2AvV\\\\\\\/6JVLT2JZLACjh7sFt+sN0LKrSIh1vXHsouWYCYP8Cja7ZZi58tDDKvzXRq60uAF8I64NQlPfn9jdAgWI743a0mm+xkGz+dyruzBMXF8aY0fsdqXDaQWVP1QvQUkzAaeuxU3d1Q8moHAONgDhbgbrxhANOIF\\\\\\\/dtiFBglL2BzATHHtVsCm8PSLXdSrjI+D1+23OYYKNUsprk3WgoAYS\\\\\\\/0cfihjcCfCUQ9aDFqXqaksW4f71ADZQhrSJ5VLHU5gL7dIU5SX3uaIVUkNKDEyRYL+LARpOEQ1KYkHUU5+xXiL1afryDaD4qgkK7Wmn2OqPca0rOQuZ+1vFVbM+lXQpiOd+6Oc3W2bUng3v5rlG9kZuuTnrXz6E9fpdzJAULtJ4jqw+RCJi+sIG3BSFvMNM9ogcTh6+MClz8hlV5n9YBGV5KCK3Y56wIlq6rjkKDdJw\\\\\\\/FT1KwOvuCRaLpWfecQwm+U8s9Xx2D78kk4VjGQsL0sxwKJ7mOBk\\\\\\\/ibPu0qKCL0wSPDmDt7XXR5UYDH8kQSRqwB4DsMzk3KhbUJ4kFjeOZwqlukKoutP6\\\\\\\/\\\\\\\/d5bNfpHGX3rR\\\\\\\/TTUju8h4T98qxHhoyXj3z+sM\\\\\\\/6j+dcJn7AOY3aZVJ3B+peWUfM+YpeGZ25\\\\\\\/\\\\\\\/eDMlQTLH87zDgW6qf\\\\\\\/ogmY9IHFTunE3CT6UdwxTaPGpEym4ZYQX1Gvhq9FF9g4ZXbID+qnsFo2NBmP0iJbO9dBXHELcnEB3oPDH8s\\\\\\\/askPTXo6lHT9HavmW0wi\\\\\\\/cUrb\\\\\\\/TBfK29ZQT9RREetrLRqA5ea10OzNoZZWp2j6UARPF8KbdhCz1WvAqbC33hSAH6rm8RY1kB7hUqp2utMnlkaYWTEzrWeBxj7r1KoCG5sj5WJWYn\\\\\\\/m6R3ZcAek0LMFjRDXUpk80lgtHEm5GGSIyQtBR0mDRnIToBbATgJRhlxxjfkiMfr+L10Yb3MD7cxn9hVKIMjvTk+llJZLb8749zSN+t1QcSmzXVk7h8\\\\\\\/Yqfaxy8BKQNJ0cYbfwX9SQqWkOASHfR9ti5v4OhTs3k\\\\\\\/BEEmzagh7zfpSjrCct5S26LWJS9pPpHbQ66tASSdgC7nyAu+aAG9Pn75AnkDjv4XpAhKlY+ICLgTVddsa2WBBF9BVdA7pLUjews2KcI1oig8AXdKrQYJvl7pP2cFWeH6OpZq6aTCihPN+RiSCKs6TOsa7LbfKTnw2guV0D0hHde8uF+WhuGuyZpk6rmghG4GwIV+cwtsF7phv1nVZrn+e8N5LKdIsd0kgd0k0lS0GkmvFpkKm3NNme8qYGY\\\\\\\/Fo\\\\\\\/+cWtiDWV5lbCXvQBWsnFNMSGwPBad0tdoAbZyBMm\\\\\\\/sd6FqPZq5BKJDRhuFDK7840UJ4RE5idMh\\\\\\\/7y4KeohbIwiAoiNFVtoBIq2QIcAfeRJyKYtCRLhYmMohKootrVDn24zs6uTwVuHQsGggKq0FMvgkOwnWK01Ja7cluH\\\\\\\/rhDwGvNhEGjbgQZhOAKaXri\\\\\\\/1j8wvJL9QU3wWiezLnp7deJnn26mhMsB9LuKdkAIn8nawuVKQ1\\\\\\\/vLgkOmWFK\\\\\\\/bigMDPHEH1zbGfd5vNlIeLUxT\\\\\\\/2XTCebvF0FgCEn4Szbc2NSPTOkxzuiDSMdW8SKOGkzl4zVh62O7x0OWWBYgoM6nvvNLbmip3iZjZ3RsMvy3NXHUxzFs1irfyGjfhrT3WGmUafHqfI2\\\\\\\/RN8LFQKiw2JTFJyYLJGhznNrRrHCPBSQGqleTULAfim3su87binwZQY0KQNVAhIokAqkMDdgL6MhpbMWptou3KlGyzzkRi3DBrjR6D5Mz1rjEoOMrWetByuGKRRnvljT7d7DMLuk1DyFNjOJssSHHrrhETtSccKXHd5GNnP07G330Y5gEpaI7xStRkPQTUv4N\\\\\\\/\\\\\\\/QTDE08tSEonu1abShdqrB2H2e8vtdnDfEUgwZGKCc6qeH4r7E\\\\\\\/NbDoVk\\\\\\\/ctmNFksd8B6Sc2Ole3sX\\\\\\\/EhHHDIAy3GuKea7XSVfgbn2OR5xy6HynFB67dTTdYMUhLDagsZ6c0o3ctvcJerEGKnwMN3Pts6tB+s3AJo6u+5ARcqD7DJnB4\\\\\\\/S4p\\\\\\\/11Tw4nooxoOjcYy3cnLOSdW5Rc2\\\\\\\/uX9ST6rAsDBlh6Dc2ihT\\\\\\\/d4zfptXllC+SplWB\\\\\\\/Eaxn+o8OBVvcz4nrvnnGy4kAEUPeoMuf+zJo8d1tFp1GPt0WVC4VCHGI60mayNJEqS65Dbw1LhDkPQAetedAFNnTUAYjrPoEH6suoHOpKjzwVz3uwJfR77LiFLHoTmJbEgAP8VDIsKlgD1k8o6TkIPca5N4K70kidBR7SpDbKsiXn29WapwZCIMU52CNAB8i1b1RsS8mCN6dqR79M3r\\\\\\\/vOFXVMxIgrdj2n+DoAbVB3o1STYQ3asooAreJ7RVXeJ7if1KfYuZWeNmZFW8ckCbIU3d2oXlFLcqLmyxgQKGjcfyiYo9cCofKAayj8bLKJGXuKt4aJZHbuRL3yPxe1Piu7gaH42Jb48F+039Cfc2yYWrwv2Kxoenpa3CYvOo0cPjB3QGapyWRM0FNdyBdIJs4thqMnvCKbRQFTQ6Nla5aBjHKl58NTUF5VhqVGQjzhqBONQQl+LmfVqXF+dNBkBrI++n8DsgAxcuNw25+m9WD79vFkhB9JeNm6i\\\\\\\/9P\\\\\\\/XM7WbyghH39W2FLG5ygj9xp3cMAck7zAzkDkjMhRHmWuOovYNcxtQkIydeM9f3V0yLQoEWV\\\\\\\/P0qEq0nB20rHyf3NnfPlbnFCVWqXXoLV3osk+KhssvESzLb2gOP2HMh1GyZj2JXqABmmaslCP0Pv3yUNXPMX1IqGhEIDUeLY0ZMP8TFAXLBa+mWx3zYss0TE4wWm125ZGJWa6XfBRXDuLfgsWCGqeLHjbHc9ZlPtPGzRl71+n\\\\\\\/C4t7aEigTASn0kVS7MwTrbpkzT8NbPyg3eEgrRvj12mXgA1lSsxaf6B0Hfk7SU8ysM\\\\\\\/wSCwrqDwaxI+1CPocm+sYUPcZTjHlGAg95Tt84kXH1kjSj7ds\\\\\\\/tVQoWAkC80tKohB8HzMBSqYqUdcqLM4\\\\\\\/yNxwG\\\\\\\/fWwmEAW6tQFoEDz7QeUa5Jnf13nnfbDJ2VsPl00Qig4SQwWZZ04S0doXt58JyCFdj\\\\\\\/onhEFSf0\\\\\\\/oxgNNujbF5nIQUm+OfZMekVmH5Ijw5gfFbMKIp7KhH6M7IiUmcFrwV+WG5UOEAvmo3zyiAaDM+imXa7W8qy9efmIa27Dx6s+Vn4FHphM\\\\\\\/VT5HvP+zY\\\\\\\/B6ExOIWInpPxBJMxdmnux\\\\\\\/Nc4q8fNOXda7rQ+XeJlcXV68rp518VDvfgfdvT2yMMSVXnAzgPFMMdZ2gvhGATP1qdIwS9WToQhNc6gr1zfmwsj4gpL5q2CPxyb4CZ+hsx2wr4EJ6EPe22qkqi1j5VEQTOisB5rVkAT5Xh4U1Jl+HXTsrPLQQi+BaKO7431PHbWlWdzklDKY\\\\\\\/OBqRbGyb2HC9neD9NH3+qpl5x0kfrWSR7Hq\\\\\\\/EANPpKv8GK1ponFz+CWzHNnR7pRDKQgALaS8w17jY+Fs+yscq8rGfZdy9uuWB3cSHOH6HgffLPlNKyojps7pNWEI1EbUaf89WNHtsjmsW7CA3LcSoDwJYyhI\\\\\\\/zFVjHdKTJRz+Yr64kTSV\\\\\\\/CLkN\\\\\\\/GkuHe2JI6BttmZdhr7XyeBRDVmkeEkPC9jMIaWOApLzT1cn5rJB9ySUQx\\\\\\\/QVwSV2Q3I6nuZBBvJvMecyns\\\\\\\/a9jZjDk0qHoFiSGxwFsJuYqf9WTgwGXJMcs72C0kEwqWmvaPXeSKMVTxSBG4dcEpn6l2qx4kuf3HEmArJ7fdjsEYMey+fUiIWNZLm6z2JQy6hL18WXCumPgmtu3BdrkHessQxi8zBv\\\\\\\/W\\\\\\\/Th5e7m\\\\\\\/t6DXzOxJ4paF9+l\\\\\\\/tDj9VPW97V7ByNi32T96ZuLgRFH2PjUs45NomiEIj4SK0xsjgynk6KswyOA3xHBd9x2gAbZvtbvfXg0W9xNNcX+nh5vhr\\\\\\\/AOmLsr7zQn6vKtUKByJPo0HpuDDK0902\\\\\\\/JuWOWdVy+l9P15ymo9KFaRRIOoCFl3jA37gM1GvM2dsO9HIVzUfNDPSgqi2OOorw4DFugVAYf0Dilfj5fj2m2mqWsf5HB3b9TC6pS0UchSfFSvTHC9yuiAgtzfQCJ2tem8PLwjrZaSXWMLnHh6g\\\\\\\/bnTRRAuXvyMH3txoC2MbDZcCUYS6RqlojmybYQdGH2oXXaIRLhZuDtrkQws\\\\\\\/xoGOkPIXy9OmFWBsqpYmcHQK9rp\\\\\\\/U\\\\\\\/JsC5Yt1BCiOBNRCngS\\\\\\\/K0JXxVNtNFHiEmsSJcmzUdhcUtviJ5lf2ApWuIRyjPq9YSw0PlAYElTx4O4KVJSY++K9brKJg8yhYTgxtWnmr+wkdzCqdII6RpWsNWJCHD8glxG1Pbk+xdBs6S8WMqGJt34l8akkpd4uOMjowl40zJ+Uzv9yMVranpOKmg8p12HCwLbLLFhBB19lk8rVau7irKynHInE5d5zzM0CUBHuWq0zLyP\\\\\\\/VjNWPiKUQbLkQLx2rEYsmi6hkJs58AxLEDH5+FF5InQWSWx018joEZRpTf5MurpSowswXKd91+QWZbzQmI5m8xqDy9c0g\\\\\\\/eN8\\\\\\\/fPp3bo73q6GSAFdy8WAS82p1hDRmY2i+W3tHcx9wRdBeN2lyH\\\\\\\/hRD4m6\\\\\\\/wHthF4yEUFBCOKdVJNMYjdZ4ZxVVgKCn5D+csYkkkuBN+hKXx2W68BbYO\\\\\\\/SKUyaLu3M97vpJoRTaPFhVHslOk4lVmUVl0JdOAE5vLlgDKoPf3kC6Hf+zSYF62aUFryaF8\\\\\\\/csZfsCkoVeBZ4+JLDPYDHVh2GwXFpZjamq4ShzbKEGf5e+WwU9R2YCso7YNAfyETActztQyLTUp9GTpE\\\\\\\/2qT7kZMt\\\\\\\/xku\\\\\\\/\\\\\\\/UOZ2Dt6hRQQLjtGReuLZ7uuEAY4qdlTMPlVf26iAs7ST9H9hvlyZHOyAP6fV0oKe\\\\\\\/yZgQHQopl6NWeAWSrAo0bFbqp2Veq2dMR1WHVbm2NTpuAi31uyF1cDXPFQCFD5e8pJi8s9i0QgAWV67\\\\\\\/CRW\\\\\\\/7fWx5bw66enRAEmRGIk5AmGeDJgQmfT6DODzaO9fuzE9HptFkgdVe005KWAXmYkgJyKOwxwoq8ghEdAWkFIzDfhUpaz+umgLIWCdkX+9mo3Hv3yGf0WyY\\\\\\\/xX68geD1bb2JXAd9F1pcpC+lO\\\\\\\/oPPy0Gk5V+mTWvds5N9J0AUglxYWpoS45jhEuehvFr4ifnyD1arcVbwXQm4dL1e70hVpPE6WeW6pii3hnvs7ZuS+JGL+pUgiGLKc6DI0mZnweUmlbf53yFSTV7+zs+yTs93L78tw6iwpXch4X2hcY26PQURfzTTNaow1mqFe8bhEjkvkpap1kCHrjqelimj1c03qC94smcoyAsmWbkrAf5QAsyUsZSiLp5zonZmgT9Q45Ah+iONMBpng+O\\\\\\\/4Kdjoka\\\\\\\/JWUD8ZLp+vcZLqdWYKbKvB9kd8bWq6AQg\\\\\\\/y0F5XDDP0a5yjO8IQOr+iPMclFIkVz1W0GO6U06tRc0tZ2pAE0YsG9vkP6xo\\\\\\\/4ZU2rvKkxSkCUmwo0skhrycIYEvcLQx8aFke87LTYMoksbkS3OfTXfQDc4X9+yLUSohnNfhnh7nhxwq5uVwKwyieYWePOBWIrcol6+iDTET+JdrIswqiBM1bjmexAjjhreGbzfQ\\\\\\\/X60jeDbeW\\\\\\\/7Gonb8TgTyRV8mHCe\\\\\\\/1bs1kdWWBCzwGoaMZIdc+ooKfoweRdBMysGp\\\\\\\/D41bZkbFLyvJwTcDvmpcDEa+\\\\\\\/hH6tvaPm1HAJPaN2KESpRk+OMpCVjalUP+CG+91nFv5n7k+0DdpOp15shPS3h9Gv2hBUC2MImdhTKop3O0JJWQgp8fb6BQSAwFcZqaYDCi7XQQqpPQ2qwR0GoEkJogHLR+nyDl1LK6qApzxJNfE69JC\\\\\\\/m04dhwFWtke9Zbu8wWcBZOvuOVa5L1voyn9mli4ebeA\\\\\\\/ie+Czi2dYMV3dDj58\\\\\\\/44dD8kfyUz5UdTySpdSocavvM9zoYJYyC8UqZiUWlLpQ5WB7vFCi7gLmw2q6sdnYyG\\\\\\\/eGmIj6n4e1eyWbG7kSB9blm+GToKA+oltSKsr9PorMuHSZv6beGOJYAnmbkwH\\\\\\\/YLzepkZBsNAJCOT7XSEugLkKXuvDixBwhiJGs2cHVIzPBJE8WGKCoIJDmV8KXSVrv4sDJa2m+XO5rJJ8e45SFX5Y0i3vOBTxAbFJ\\\\\\\/Zk1QKD6q1iIGE5bbFrhsqRGkImaumvqBa1RgLg998zdyIrcs4zh10IDhYl9CX06Z4g3bQoYuhMfIlOg9AgUdl5kkNDNNV+hUH0dj0IGMOR29\\\\\\\/9WgufI6AhOPM1QHOa3cX5m2cvEpbIkNW0mHOndOefq6o+F6TWhNPPBYC1Sg8OVMpX8KKSmb8j5NwVW8UHRtXhnNq0bvICho+s9GMVX22xr4V1X3TaotnBZAVGoslMhqlxzcA0l\\\\\\\/vuNIjuEfpYoQNcRNwtDJI0R+hrzRayINv55R6gAu3PuXyqnZfdb\\\\\\\/e5jHcOLZzAGS4B\\\\\\\/Ic4zk1Pj+XvOqyGduU63Im0Bgo\\\\\\\/U4jHlABgVj7TwWr3DllKCqlYVHsEGiBlC2BLDLLQkkLSDN7prsF5XkCoQd+KMYZE+lNLDzoRf1\\\\\\\/axMZdGZ0H8uM6SOWZJa46lHCiASqlIlvuIM520ftYxwrunv2YRhEDDvUwThTJ3WP\\\\\\\/ylg+TmtrC9YaZbQbkrHdQbzuRSZ4hbfofe+9\\\\\\\/apnfRgGYX6uAHFZ\\\\\\\/OGfRdhmCHoXVg24fnBHfshuovtxzu+sWI0XeuQaSkGKiYknjOwf+X\\\\\\\/HupeY3UDHpLlfMmQTmbi88dvVu1AbvDwcPFkjgxew7t249XHt2IEcaY+RrvBb2S42u4rfOclewXRfGCgG3Juaz6vBypVmHCctStafq+t9u272XyJPUFY\\\\\\\/5mycloaZJC2\\\\\\\/rwS2zUGB7M0mNnygx+Vep6BGRBmG0DZJxmVdypkDcGxnEPZvxLouOyUmtcgw5kBItRfyEfQRLlegSfd8dbVdy0y4YMUgWlwRxJCcWR3UXELLWDZGCRKVQNlOhA1cVWZ5yNH4sq0reSsYCkWsr8V4Z9y1g3MgDqzNvzI3hi4UPryg5CDabdVeYp\\\\\\\/MNA7vwejh43zSBYQjxt7hzgxE2nBMwsTUbWJcvIu3cf3bWt3uQvCdJ5KrwNJt5VOnxnVhpu4zz5H6QQBaL0bFhIfiSPZizx0VNU4R6p9wXSYZqkjvn1Z8ci6f+HxDQ838APypJmERCvy4KIgGp59Cu2sxmgEMuTs60qt0\\\\\\\/A+Pu89DWddSwn\\\\\\\/g2NZD\\\\\\\/zVnBLPXvwt4I9\\\\\\\/NzIy90PZPyGFEExOo1cHstYYiFEjzmJNn8q58Lia3+0NmeKjmfXcb26oPXop8qCxPs3SduAlu2u7BoVzoGQ5frAQ+LhFZ9uTXX2YHHuEKwI21m+gQxARue51hD\\\\\\\/TvcrtgSguN4Wwnx\\\\\\\/lB0B2JN4mRGwgRpNcsXRYbPEpzGCsrkzspsVk7HPVvDK1Sd75U9OSPEOLxH2AhwhkfWrZisvpSaHgMSKQzIamVQtycFQGkkceDARQR3xeD13+hSELaXOaw6SWPXKXZVsCuXji0YP4V+Lr8nEJ+TcXXYTD611tszZwOWSc3+ZY1yCjidB5eSQ\\\\\\\/aDejzTXYMXyLftvmLGcDFUAowTBfHDtTvTewRFTSu0EddeB\\\\\\\/W87EuYxe+IP5RkkS9xOMsE0MrRZCfFw7GgsSjgdathsH2WQawXwNhcKOv4xaLF4MF8tDd3tuTBx3IVJcOYJtPtvGzBg\\\\\\\/9F96gMRBsZ\\\\\\\/KeMBhsuAUh0NhMtl2zRljUFr9BdH6RIuCtIP0RwQMY7A31gHib1I9IeM59oO9Lz0erVd7an4sJ\\\\\\\/NB3ELJCtmZQ2TJ677ZhwNMREVWW9d2gDVWZBfcbf5W9\\\\\\\/oVTb4ML+1rAMnxkr+WsFR+nWQVf+JS9ndn8ncreChY3lU5nbLrVU\\\\\\\/WGsZ6w4yM6wfQNr5JPniwrK4wbT+GijOFteGW\\\\\\\/XHVLAOVBhKvhWToORCSQfzITbzCCYcyrtk7MysvOuMiyAOPhOK+AehKGb47NGW7B2VB5nQCojkQbXyyEcc3IzaJOCPbWpTC+DLHwUjpuvA96vXGfr3UkZ1k3LSjSSySEtmGD+mbq1ATsw3cyFNPr76AAMEkgvFSpKhOagIisSy49ISZAqhe2pSDUYIUrsv+3NNmpWTJ9Hn4MYyvRxwJh3FijdT1E+5BoJLoWsHpVYaKrAmtWSL2b71drcjo\\\\\\\/NWO9nNTJwACOW6ksz58f2q02fxg7clYW9L\\\\\\\/jESfnWQNA1MCuW6Zo6Ef4BywXnrZAFNCwMqx75SUaYtJPd3wbovfngH0i6hFsYtDP7XdS27Igl8xm7b\\\\\\\/OiVtIjVQrbq8MzsaawU5iQ+fpAuAVOo7XQaf6WN5Af5Xaj3W0FUbEk0m9w2M5aXEMt+KtnXD16pn1xW87knVXHIio9lZ8fPf\\\\\\\/4PNxcrgIDuoUTfUZjhu2kYqm3pmoZfALybBtZ8WeHoop6gYTCnjrhrdaZ8130JvNJ0ITHzbFxTveGvgVdASJpRbsy+tQLLnHF0plcVIoKZOsl+DWyD853glHSwl1t\\\\\\\/yRJ0nwp60BVEXmN\\\\\\\/W+NkOH7qzs9N49g0qNg\\\\\\\/aaiPZNInxVpmxLffbHJWiCAv8BjJhBnSqPMO3UO6PjQpM30TvilNMdmhGvIx8UV0YBJ1A0Twcmk6qoHWO7xRdy4khalUvboORyrR43w+U4vAdXUX\\\\\\\/XN9LQDp3pDmMcumqrQJTO03e\\\\\\\/ibKP7R7P4Jw5lyDOKo\\\\\\\/eBnMkHOAWwZ+94Qx3hVcQaB651UZNHmJuPHfK6KwDiv\\\\\\\/RUsN8Wy8uko5c5aUY30JjQhXRRw\\\\\\\/kdoupSdctklLkuOXKnIks+IDNWbbwf35aDtWJzltW7ZSqZm5M32lNNpY+9+4IzzwQQvOFHaDEwyyc\\\\\\\/4HeyR59eqdrnXTXtJzJ\\\\\\\/hD7qi2\\\\\\\/EmCXMq7dtR09xlW9A5FTikG5B1SmUDpQGLd\\\\\\\/GG3xH\\\\\\\/k2MITY\\\\\\\/BiNYJ0Bv4oZTwsbMq5CsxKNPeneItfrVIHvdDUoy\\\\\\\/rTkmp+g7c9whlTtgYKMGTU8pYMFpS8KlUma5ryGwiQEhKrORCIY3f8vChrw9RaKbCNgbEtvdO40cMnv8xrmnl\\\\\\\/nh+QUmretWH8aQqZVOeSzJW6OpNlGe8NJBPY8ehZ19C2eiv9f0xhJmNFky1kkUy2anfVKejk07aYFWf9zahVyGyG\\\\\\\/Y3BCtFj6P1p1vYJd6UZh7nf1NQJSVm++215n8Jsm6zPBcBDJo5FmWhbCfJ60MbXwYnCBabh9kdqz3Yx++17\\\\\\\/VfuXZwx9GFTPcsp9ldbAMjAYOAWIByn6xqWPmiMR9yXfLlrLdJOC02s0xJZHhy8rHrEZi5wQ4lGP+HDSeIu3Gw\\\\\\\/Dh8R8UVnhUgaZK+fn5YvFVvQIgKTqSdiNSwXtBA0Wj\\\\\\\/KpJ5UFNKiASj5\\\\\\\/yz49e1SjsE9yJ4uPRNf96VKH5CymPazh0YSKrCHc+acoD8PryCUNAn1J4oOfbFaVY0+WRTv7Os9RfFnOHnlM24QTu81gK\\\\\\\/xvrAbTORtwwzfdukpzei5\\\\\\\/BAvsajKAgkV2JGOP\\\\\\\/hRoMXXo2hmCVJ36aBHKso0AW5mcLSvlEm8+ZShApjfEwUiE0ZhsIj\\\\\\\/qja0R1kKqXmbTrLmsZg2uBpc+X\\\\\\\/6u17VetfWhlNaHDDnofQeh8+JLZoz7X6MQ63UACcnrAyCRekxLQ4eMcvD8Q453lO34ygvF1WGiwZo8ooKnfNlgf590yUnj2xtpfBKwUOBqWX5+yfaO7lcZnhksdRAH7HtNtHASdnu7q5xTQcuo4zroCMk5\\\\\\\/gV08BYX\\\\\\\/86ia08EZ1y9DmGrUnJPIYWl65paZzBaGBdHNGz4Yp\\\\\\\/Q8dOher6bsABV5whybS+4JMsgCRNuSHIUxZZmeYdhcLTOeOlbDh4XYeN97iuI9gakg6xG3Gwqd2bokdsJYh28UhYIqoZ84aOHI7yf\\\\\\\/pKGD1POaEbvLzbMuLUeldGNzZjdJe1oiIbPwKu0OxjdFnSH+xBVahNSbS3tX1g\\\\\\\/dsD6n+tWAG8WNO3kS9JAor9nQLVkvvOf4mCv6M1qUAOtQhyptkzzuLzSgnOgl21rRic966iP5T7m93CUxm6R7C175Z41AJ2LsYBhjmYTm09uHJWL6fs2I\\\\\\\/CTo8HpMsrRf2be8ydl1iR9+F\\\\\\\/EvPiM2NRPS8lKVrblxv15NJzFoEO5tbVk3aA8ppMGpaSRDw72rsSB8jaeCG5SLduXh83bhoGlfdqxBa3xkoHR1mH7uxqxnaREUptwykIkybwyGiHc1pHWRmeCIWrXHvjy96NYNfP6iGmBpj7mNpdqgkTWbmgM1KU5C3yUNl9aK7X60n9y9Ydyknb6umadvi1yB2T3fbR6eGhvJqof1G6uIcduP8IrzAPhvM9yorls2MSUll7Kj1NbBdErwOpyp53Fs\\\\\\\/\\\\\\\/Sr7SzEbvypycTYsR3lOvPQrt6fMEvUu+CzciRIrWSK10ZGXqcglFu2dbXmfE5LR9T05bQTTWC1YvfP+dMR9gTIIrhxhYXH8pCyLWehiLDhZ8wx2farxpR05sH4LQwwGPyczv5y+cU5rq+803AagXJ8Fb++WoXm7llvW2cBlEwR8iDPni3zEaeZFrRVL1IB7bwF+rn8gHx90lNqL3bJz3MCewl2rA0XmGQl1V58GFTugpz08vOfoghkOE15DQ\\\\\\\/iYWIaZ+mz41yP+j5niGdA0fLWxysUWHsE\\\\\\\/xeV02caohJmEOC3H+e415CtZXHbynBfS9opy4B2g\\\\\\\/LYIYJpDSC7SW1+G7LwzposjCfbfHJ\\\\\\\/HoSJ4NyQn8j\\\\\\\/s+\\\\\\\/ytXJdmKEBhPdgfaPBxtOs2Etxdl6OH7kFpveV6OmWFoYHKKzuMofBAi0F\\\\\\\/IaJCjeUuV7R+u7B4nbRcQaKVjmf4gmfu03JL0e+xTiBRy8cSy\\\\\\\/4NMZR3jtsUrZcGVFyuRtXUKJdgrQxcsKr+IH+Thvh22MTEtvnVZCsY1pmdiEx5mSAV+JwQdpVXcP0sMlHJmgeUML+Qk046RMLwZVBXf7ltYZPzmj53eO50tMlXxJaRXbJjZAZ7Hh2bMZE17lqPwvC9\\\\\\\/TMdCr9d+iAJZlBK7KjoG9C8D1Dk\\\\\\\/qdPYIno5FmVSpkbe09E+X7ePbWkZF2\\\\\\\/EZ1ZX+fmI28Tj\\\\\\\/vV0NQRIa9XLkAWk7G7s4ojrXYr\\\\\\\/xC1l6Rf6qGbxhYX1nD\\\\\\\/Xs229fSvKNjviDn\\\\\\\/BZt+700+1XeTRFGugT5Uv3dIsOdGYBwyN6FpUUDH6hVhEAQt2tE9QKEZ\\\\\\\/kujVghA7nUFay14j8REzZ0fzyB8b+\\\\\\\/BnQxIFUNoIPC1leV1+7kJLdBcTVnRoA1Z7ZfEqCasbQl+xRuU711c6oBcU+hnT93kAsIgvdDMoEeFqT1Hp95DGoWsHfOYiUFtVBuAXPPlrOW6tReqUu2t9FrK2vUMlK1ce3NH1hehRTmYT9Jr4jKMSHBwCUhMZW1IfP0nfyGF5oPR6yMXWnngBMVCun3AibOM8wfsDa6HynsqHb+45ApqOdiouhr3uTycI2Y5BtSr+UAbyLP4KyQ1SNvifEoJHT8G85I2tFhN20xFAZQmbV0+aaqv4+isQvrTs37UJsHBucBxB6Hd4uubiGw8BKtT\\\\\\\/VMKJ\\\\\\\/D9J8+YDbX8lTGt0WZK75RsfMUPsuWFZrDUful6epXrqLtfA5Zxxb1etJFJt4GN+0qOF\\\\\\\/2w1SBfNm+nSYFwi2cInisx5Viw5Gd6FNI47Ro\\\\\\\/2TQW1sH6nWBsoAPBd\\\\\\\/OlWzxjXZXXuMFs+Tf7jAMvjguBvf5XkB+4nSoTxG11j0UtFU6v2Ls+hI4TUsJZaEjegLfT9ysCwOv7azN+17pXhtSg\\\\\\\/6ZugFRwLPKnpcM\\\\\\\/eZ2q1hl8am2uuS535JsxPI+K\\\\\\\/lT1LucT70x6fbpi7Ueyg4mGgPUMg9c4rMtN256kvB6IMsNt\\\\\\\/SrRgeOBp37yYU\\\\\\\/DMPgrrI64Byp0A1hCTVjtqy5qfl5zpyu3bJUiQ+CHp38Mw4CaEpUK7FOk9WWWGX4hsoyTriQZidhUSbH4KZ3xYLIJNaSin\\\\\\\/3fdlWLjUk3+UhfSuhTY3u2YNjq693ghiMvJX2jPG\\\\\\\/pa+VfbVs43IuRAZHMOg0ViaAj2VIUsaiv+0q99YfmaSGQEt2WrUXU9r+Gau9gMgVSHp4aHBqDC7DO2APeTEHDHj7Bvlml3t\\\\\\\/3smOelk4cAvAVxbR9gBUMM\\\\\\\/OVwiLft3+iUXHUvUWecwHyiCL5xbZ+jlVD2de0b02UURKV7Cu\\\\\\\/V7RIatkGhm6T4T8vOxS9qicKCM1HPNQaiOLyAtiHyg0y\\\\\\\/Jp+4LgYJLwn0haOMJ0WS+ToJaRs7qIW34\\\\\\\/qL+D5z43HHlfhg9vS+HHmyPIQhisnBLXb8M0qMdhAls6bteg1v9GPK2CFIV4dgSirGsba\\\\\\\/B38MFHOkfECqy7rG3mWZYMAM57\\\\\\\/Bx4jyX6fx0mNWKAkXWUL\\\\\\\/a1XngrHudtgBVoBYZXJNsCeyVqNFz79NejvaA21UU4gkJPevqXK194oL0Ch5XDF6h6SyLDOyFO2vpoqTojPAiP1DUfCvKjdOgHkkc5j44RdW9GVcQuXuHLDzbcWrka2AgeEYmqQwtZsfz09+MhYdhL6Kv2+piiGDoB9Xb6Mhkrbnz0b2GKaxFDRLia9vLMgmcZWAf7zQaByJkKSg4YNzwdXGleUM49q4tbcrB514EkZ+7JJDcpYcn3kWfoN3NcE7PmIcV6q\\\\\\\/k4GBytS9hvQwd99oJdlt1xViZSMtGNxY6ia0mURmm3GFGr2b8+Alsm1ylm0wh1v5fX+BKPpR5t+XvynRC9Ne7ypIS+rliWe9jmMiUQAjp6dCukxRBKoipSidvKpOjyT0LJNF1+K2ln1ovYfiwC+tuQzCkIy5gR8XpjP9WBlSAwmkM\\\\\\\/9B2iIBCpPutGpGvpwnBvkthEI70lB1QH2OvdFB8EKuQGEsEpXhk2zL5i3ihT8NAKOjgaOHZHbNLUlGxxjx88gwaVF5LvBZdLHm7Jk2V7C99aXkHSX3rk+H5U3lN2Dkj5KhdOMVUQcSZmS5Jqy16yMLX3yhDO5sgT76gDbiHkMIfPlFa9ap+RcKHfZw8N+GqZed4DOjj6uO1+Ob7NWRYI58e2D+f4gUy+ymov0To9S52Hi61QdGbCX\\\\\\\/537mDyvjyX7NzoLDguU78CRHg7G9Lt3X671D2GFTI\\\\\\\/9oE88I3iedaqpXHMbh5KfzITnPBBZOTJ\\\\\\\/QI2Dn\\\\\\\/c2BSt9ZPG4lHHcXKP2hOHZkIZifRK5I6qsj7BgQcBpgBLWhlnqgvgPzwUm0n0W0IEPbnW1sNOpcZduZxrX+QcOjCRS06hCwKc99TEjgl2UcIFNgRp5NbBUi32m4aZgLeR4IsWe\\\\\\\/o5abJLofbNpiFEfnxDO26kTzH2JkSwTg8HfqDWBtYck+2OmXGMvSfiTQnIHaM51oOXmCM5wzQyYgFLJnDylGGrwmV\\\\\\\/pmiJ2UgQgY0H9\\\\\\\/FRk+bW3kK0BPdDyQgm3kgWKce+ubGKV3rGDwi4DSPwYP8ijgnIisJI9\\\\\\\/5qFrHYuPqdbGMpr9CVlJGvDMnTGELn55EdN1fOVoLWuOVLV1o\\\\\\\/9dF0rN7HKvZud2i7uSpKtPHu55GvYaCaxurmtPf9L+UQvsBho\\\\\\\/nz226KdkNLBtIloRL155660uBM3wY\\\\\\\/HgBEqOb9vDiOCIg1oEEDKf9lVvE3V3fqQ8Bke3NuYwEaHzMoDjQUT6nBLte1tlpLs684rQ6IhMNv1XBjnWnnR838ihC0beEcQmonoINBot9jY1VC7BkHjOP65KvUKg+vQl1edF+gx4b0F+gWtbPSq7HA0k5g1lw8BPlr8C82lTBxrvjNftcgj4Uw4YBSDuXLWqmXnX\\\\\\\/HzFA8jHR5m2gBXG7W\\\\\\\/ih4opSMGgs5+PimSpCLJeB1EjfFaseYTHdvRNjTWZtuOdb3J4gTgjZwKYA42Nz8vWZktylslUVhHo\\\\\\\/OxV+fLt3tyvx13ipitsimk\\\\\\\/03l0ZlR4XhpBIXn2xV7s9eVzxP2Q9Ibd3kmze3JN5fZbbpvzbr3gXmgPK\\\\\\\/DQYzQmj5uGg1VTw2mWNzgMTjGWYo61884svZ82vIVtPD0sSkA4eZ+LtPwzQkpnHOa7eMUI+BA9WSg58fG0+r69dHuW41SmxisQbjjV4LcTWM4GElWC1rwsAKovihQYFAcKmpGjYDxShNDy9NVJ6Sg3GRqKpBbd+R9gn5B2YnZSUxOcqsiklyVftEPJTptF06IHUgklh0t4RcLALGEJMPNNuA81RKuxXBfWu3STbDxdHQqVe0Q7V9Uk+YhKKhy4UVCVMgBskUmWHPdZrVRnBMtVDAabNyc7jdZXsaIbc4rDRtPxKIHaM5H43dfk2X8kqid2tSPkn2WPfaV+M1nM39Uz\\\\\\\/N3zuW0xeioipqLVjrPgD4hPcYdHGIAaRwWy7qgf\\\\\\\/QezT3qlR3dFofuBmPdFv1Uh2NaonRSWY6KtwzYfm4KXpC+MOEB3VtOdq6POwUVRuHD0IV5u1YlOj8DHmcAaivZDQzjLpJrpNqlnwy3dMNox9utw+hjNvc0Q6h\\\\\\\/GbWUkwnLVKIrCTGbhVsaQ8OPUCU9L7GEmAiiibnGU2oKThygWuB20TCocxHgAc1oITFOsxGlrV45QSN+0Vqnn3Epl\\\\\\\/FUiblSi\\\\\\\/5cI5mX7IJi7HKSB8\\\\\\\/A+NVDECWUFSuO+0aoP17VRjuR\\\\\\\/efxvNueJk10\\\\\\\/Hf7ZoJP8QoQHAEK2G\\\\\\\/MKSyw4eRj0l5s7vLZQ4NO9MzFmC8DVEiCYtOBYSawfr4eMKH72Sl0Lc0VGr8xH2XT37tRna+rA9+8UYAp\\\\\\\/F2+1w+J9Z\\\\\\\/tin9H9bC7Pkm3BdExQXivxjv0Y14NipJ\\\\\\\/LhvqnVHOhW1GBczRSJDXnimhyHB4uBT+TCK6RerRAjotfqMpLOMcxE0ZnKbxiBR3xY2widYfdsElOFu0xsVB+JvTQtgfe1pSdfsWSMXvcxcabNGnFD7AcbnEUhS3rQObHUYQlXBsJfgIpGnG96s3sUryBY7dZM0gZ1d18X8JAjf07dbhTsuKuytw4pQ1mCIyxX\\\\\\\/W7OpJK4egNvbitUKans3iaPTu9FPPD+PSPsHstx8W4PaTzkptozB0QXC8rnOGMaKJ95hRXinpd787tDllGL2OqGvxvdxMoOVubXeK2bSiZSxhEQ5QJ7AxG5RMw+2v10dVeHKIkfrxGHRt+0baXpBnewW0PDK35Uz7xMghnzuTr7UGpfDe35D2ZSyiMxytcWzn7TixFUQbj\\\\\\\/mhhzmiVC\\\\\\\/X0wSOFOMr1srEJfjiyjB7LDvzR5FLQpaIMqUeZc00pcdekOQyksurXjDUvZvCrm7woZI4a+7436TE59im3BlMwPPukxeZYl715f3GTY5KhtrWu98hPcysweYFEvojlC2\\\\\\\/k9WeuCa2g3jUI4sYvwGSXBNdSBTPWISs+xeZ9k+6f\\\\\\\/akfo9rtW4fJy92\\\\\\\/2OCj6tNXHr1Dv0EZtq2ro+xtG3zV+k7za9ECDsV\\\\\\\/kAEWPbYQHaSOfxget8E+6HeX0lnieU3jCHlgrHiEemq7SfwFN17UjzSR7ZuKOa8R0FBkiKvTr0ZZn5UEFCNIiY82f88CAu3nycRnEBbupI7WC4vpo5j9Fu08HVOk4UJo00F2+t0FrNr7SSKEx4j6TpgZLjvt14LT4SKwu5F3TICHYDUzCbcYu60DFKtuwY+FnjnauiQBxbJuB0EdK6yT2cl5Y2Ab25BekhpOUrmyoiDcierL4DfSWNIzwYhMTDHoW3Kh43XColcFCJq4QmTwxzptwZNnQ3KgRtoeG4IedvvfjYEVRvM2+5a++xA3ITtcyQOjH2Lj8h+ns3Fd3HG4ZQy\\\\\\\/J0XJ0r4d2gk7C+gza1Mz0qVfJ1jUQJEU8K1e+sVWIjNzis7lgS6vk4UCv4T\\\\\\\/YKruaBpbG9tI2k+vMjautPPqoaEI65SPPSgtsDxkvZLnFx9dRX\\\\\\\/8h1yQjAT0ZHdE3lVzrrNfCvsY2afnY5rCm2yOM+RTXw49ZV4ekWlMX+axvqidoZwAaSRxH97U5PkjibeFPSg5FX+sEze2r04vqFd4hUZkalYxR2FG5PIbDoKhhLjl8+3zCTdWKpYYKotixVqHyREsagmwGiVMj0hyf0skvwkBrKORk5SFFKmQtQnYeLDlmwsML9c9mc3gOX8CggId3dZ5tpsmN5n6gHXnAbtoYrzLcD2MP+srqdMXVyoKJc35soTN4U1Lnp8MEhn6cNU4qaprWyVXujBhMvIxi0Qg1Ry88uuqoLYX53AEK1Jbihe2ffgAgvptTyk92iLlodw8ejkEKlFP+z3lhbURxHGkkODTRuhM8y3rTw+owRybOhe9vYJUc+VyD9Y2UXcTqXWKK0UB+TuhRdnYlJuRUUFDDvRfxf0+kes2vvK+yM9+OKIjuX5xjhHojF+Jb04r6SCQeoEtIxN5nUaZtaaZbpHK6PobsGtpN3\\\\\\\/YwyeJs8FAELfpiycC2+ce5qzGpYMxnnQ1TxxzO06sMGD3dQlgDspZlSSClamSDkC9n1+rv\\\\\\\/l1H\\\\\\\/lbUh5ePtZPyQoGgH5c9gPPc7tKlSEaCSO7wJywrwrSUWSiiHA+GnduRfGfINT+nDbv3j9lQC+aQpg+KpsirBtj2Rz9YCtyvExaRnzv\\\\\\\/CeCqIXNmzri7dW72vte2wSwQ4Qvt+TAv98pwqhKe9hhmjtZbQQWSY\\\\\\\/Nb4uXEhEReEynKAjOfmi3ZnkN5HMhR3QlrT6etKNh1vgmPIBRy27cagDarAfOjCbWmOw3CrzpoURotiZl6xcZRORiYPIq\\\\\\\/zvG88AFCTOX9\\\\\\\/rfnSGtwCaHGoCf1t7dhzugr6Gin2Gki8FjH\\\\\\\/5+fnMkHIKEABgSjWbLj8HIEPPrOXRRYGU8rd7\\\\\\\/+S7uvJz5URgcvhkBQOkgE0H74zAhDskVkeuYMGZnnSIbOrAo8zog30JTSEIrqBycMfTc1ZVdcuBk6\\\\\\\/r\\\\\\\/Tnbp3OXIIzPTKOyKSrTI8C70SWizxdhFRCgx+BnqxhHD2wyWLDvMnVYiUOGrgWgE1Qghk7243wcqli9URX3Kj6xIBnC0k1PohF6FRLbtR86pau83bU1v\\\\\\\/Fw25qPZWwfDMqt5fB09lFCAnEAJ9qr1Gr6aE4aOU7UcZh0dfR+kFQ7g5wFvI+IjZpi5CWB0gsJJmjGDPzTfLbgtaeJ8PVDgU3IfX6M0gBlIZ5qioaoghRAKgQuPdlLYkztgpIkGK\\\\\\\/yNRv1SbZd\\\\\\\/\\\\\\\/KL5vjHDLa2tT0kGZPnC3mUXfcd\\\\\\\/dI40WjsbipL+oVjI\\\\\\\/uC7xhQ39SV4sMuS8F22f8VXBJi2e18nP5kTNbRObl1B24PbZ487UkrCuOQj1pPJGMmyw1+yroI4YySJFc2KJXQlEhtxOLCmGEbe\\\\\\\/5VLEFJQy4I3Fbb7i6bzJGKbvEhmfFJJDNJeZ21yBoCM58SMkbNMLOnN1Kk\\\\\\\/ej9OptMAc\\\\\\\/nvs87TrJ+MgyhSt5qDKKVfLkwEtlmY\\\\\\\/Uvts19iRxeDJfPfogcgvEwyv\\\\\\\/YoziHNgQt2Czg01Fx57lY49i+ngoEknMKXxsTYxqrzcHFwSvX81FrqljwWsFyL98dUO9V5Inm0NgxIGZJohhi97Hy2GjrOlZv6GRYavL66BHkK\\\\\\\/0BI56Irp+dtrQAKKAwyf7MSwZNdtE9fR9RK08i6i348evELtgJzS4yz4YySB6wNZ1ffrJDGyvGvmpP5NYheFAYTXRgb9db9pWA2r5jdL7pCUDYFiS67tq5SOgX+9DRxPUq1\\\\\\\/HOIVmx\\\\\\\/qoenXWiozXg7gLFzLuokBhE\\\\\\\/yghRcNQsrJhuoU6wExU+yyo85skiIFrzOStgmo0DGer5SbumTezh7cv+2bsdMSL38q3MjtmxvcIKEBBTBBHclrON0NJBQwFTiDdkewSbmEULTZJm1sMHrxOLK0dEnWzsBaGGpA5PM5SOYn6Ljq0ja+fKreBl+tUknkGrh4QIqIJztdsQa1buZ7xxAfHGQ9Z46GGUyI7jGTQ9gU+rFSziFxDI+vuT8VY\\\\\\\/YWxdXACRK6QrmxBp\\\\\\\/om2KCioKpBWpu8gCjvaO8OBWzUyql\\\\\\\/3iH+KquCwG\\\\\\\/OMIkC55AjwPJugtG5tGXSg0SNSM8Qa6oRCxazyoyEvUZHSTmIaIIZt8j5DL2t+7I+TTf30qRIh+KfC7YVDtvnFlW0rQvllIcjijZ18RErLpe1KYtc7k2jvmfRRPJxcWnqhH+Kc61B5EQY9ayAYmDQ0e2wMiRTYtJcQB2PLtNSWp361y87OeRNsPFQP3QHFxebmemmwBYP8mIoSJSRF8sqY74FzfP0NrUKwCRAIXXgnQlxAzJ3i4rOLJkRIG6UT7i4XlknOBxXwf5pwpTPyQmW1d72hjDvnp+QERqswAsPCA6ZM+HvHxE\\\\\\\/GQfJs6OzZ8eHrCb37rQqtUBGptYOi6VJYtqFoq6p8s3mwKkvAiZgl4GEnve2K5Lx2W\\\\\\\/2Zo\\\\\\\/XrVJekFbVFUKzdkkSAdWF6hOnPZrAToKI50AIwF+4AXE9v1EC8aNF9mnbrqtJaxJpqrNuhY07kjmhOIOrL27xmbFZRxFlUUXZitmnZt4wBNNCB77QBntcZtrwUei48VatvNOWRFwS1EWufVFrQRS+mZKXXTrU8\\\\\\\/I\\\\\\\/EmMjj7mZBnNLw8XWLg4MB2FqXPy3b5aMejpW56X90qy1vAXKtIE4PstqAXtqAezynpCg26QLbuMh\\\\\\\/r5pGPoOw0LljoGmw5fdb5Z9wvKUSA9HyHfDA1Tv7OzsA8Ed7a07NQexur+2KSNEAJpaOmmIUeU2TGFemqlFjyFbSiAp04qTu9qHPaIK6OiToGHE6Kx\\\\\\\/zB7GOhHznfbQamgHJgbew3k7Mvkq5XWd80IyQGY+f22FOdqyXPPBT1d6LBHVhkZaOeDnZzUvYi6ORrKSVJE8b7HPfvpB+84T\\\\\\\/S+fzecSPXXiji3DJUj64o0VPGNuRWPNJ8i00QN06sVi1CDI6Acy19jwK9BSs4uyH\\\\\\\/\\\\\\\/gwc\\\\\\\/KB4VvCuQ8QzeQsbkTFIbZxm\\\\\\\/ZJ08+z8oQ92Pz0XnXjS82XFhX\\\\\\\/aDdn2FzKZnt4LTZUQd1LrraamIlznmCk\\\\\\\/q0ymop+4QVu1gmdMmgzqXJAPGHIGQzURfGIhYZSb8o\\\\\\\/Xdk6SPvU+jDaC2XyWtVFknwOjnQ7v8E8NCw+A0tXk0XtI00tG8Vn\\\\\\\/v3KmHrpMDawVxn2zfODn+N5WUZnKdvG90gQqFcwZipR8d9Au2MJLnn0CP5Tovygn4dL0BIDeAuZ2gQhiu91I7hOzp2e3DvLuM8iOYsrcWcl4jM1zt7tAu\\\\\\\/0Wls597Kw21La7hlw5unHptopZNyR5jRQEHdg\\\\\\\/YpKOnrDwCHXp5T1S8KuG2W21CfpJ5wd9brMEEnwj0mPHUnTZCKJACbwbRy5BK77AEi+fVP01EnA6Bq2nfsEbWWueVqELprQW7ek3hIpe+EyEMAFLssHu9UoJsXH9HAjPEgst2CwGE56hhp3JQzJegLZhNR+7tGIMZLOCGCApVP6hfg1juJVxiinJKSGOdEiUxLcShlgHDjjtpvY\\\\\\\/A7biVUXtbCeai\\\\\\\/Zd+F4+sptNpASyAFJTOE6kQYG0rPAwLyrG2xJuXjKkF3Q2ZwnAyY5HH7n0YwgvUgIVPUa8Wb6+1mNLWXA8EtBN3l\\\\\\\/sXtgQbjE3vxRUMuyTIhtOBLl5mKKP0PKcL7MGkY0CVbkb4UraXG1SyCeJx65stcUEpwM3x9LkOQXnzo7UpILm7xjsQQTqjtz\\\\\\\/XJrQs1LPhJ6yVy5w4rdYiMDZENvPFA5jl9Uhe1rOOhaAIzv8XsCUNGhrQ8DbZNV1u6AdbSCmkp+EadC7UgbVJwhkn0CPcYqUla3PU0w1we8PVIAXsAevz2VtWaXMR83Y\\\\\\\/Rk0OQKAS8AgD57T2N5a+Sf8nQV0MhVmkDbhO3SL1v+4a4qRcZjyksA7TUBc4qRJSuuITxurSLmK6CnG1iio9YGWJRvkpx8LZgsdagl0oMMbogHhFRthOBYrxlhnBKfG4B050SS9hEI2ff75Mi4J4SyBxY4SBU7MnzNRFL+Omf\\\\\\\/AmCDzZ59mukPpNsjG2znznf2gEXLv89TALvdrHMK5MWbsPDYLb4uSjtj\\\\\\\/8uULqEeePg27m9AwECBkVsXRkKGlKRy4fztNreEKrDmmR5GsO9nYcRmPBD9sATwEJYDTjN0QwFe1JHbPZlxEBLDNW4\\\\\\\/BOKMnyUuN0OY1055ES1W3SILLxBUZV5PEWab7U11EFGnlH4BQku1uSWrdcIc51TTO\\\\\\\/STdvCAmeb3qxSXkHiubv4sn9DNy+7oGJVT9M1wzrlgydl6iS7oRKRyRo2TszZcrQHBq9wFT\\\\\\\/k35p4PwIK8aNPhNvozjdb8kgZhHYPMm7OJbiMeRVL4GaIBy619UshsLgyI1cKOUftbwdKbhbmqqEnqxkc3hjn7DjeASirbO6qxmRki6\\\\\\\/UoNcO+8Ixx7Yc4bFUxdvxFZI+mZQQO7uJ5wGi51NwwmKx3l8SNfHp\\\\\\\/MrR6Z21Qfvn\\\\\\\/JwYNvxjKKEMoA5w0mdPNaAN8CFSGnYi36gcJyJUtJWD3SUxIHwD2goS0oa6dJhKqTDwCoJKCV6qFlkpvz4dv\\\\\\\/zxZPMdMwbg1iAfQdjoHoltotoAfM+DjeuQCdTvh5R7jr7hhSU7VTeWnnr9fc+3DleKN3EeBUZT+2fL14HdF93HfnWZeVeui0f4dLR+cuwnE7bA7s+f6MvAZx43Feg6lw7BTcRqLgQuGSK4tNcwzqtgjmHmEwSUohgnb3IV9z3D3vxJYsoqVxDC43jl1JpdlGQaZydG954H9fnfXelPMH6TRpEBoObUFatoJA\\\\\\\/NH9wQa2ooghtkO+1y1rSXcWMskFo5EKxo45lmhov1eNDtS8rwQgmohuv1rTUOUS+yxhzom7cvveO01OAbIKsIeOqji6mjLIF7fWdbMDoM5rs2JZwo2G15+Hi\\\\\\\/MqmMG6C+eB3VJEqMScbqE3fv6GahYeQEOj0ML34aH+24+9Y9yzlE+dhPqNEmUYds\\\\\\\/6NrbN9YfWHHCy8W0LWJA+0TkmE6yR7nLWILbVzuj6\\\\\\\/VefAUPcDeugTBHafq5kqC8r0sK5voGXzxwonzwdBKIC8z0jMCpEUwTFMPc5v\\\\\\\/rYm0Mj3z6Ljh8Y+PjFSkL3f4lO+kfShNpPHRN6ZUXclLWdmkYZSo7RVUdV6yCtTCY5P9088DGcjj8qO+pF5S00cKiMcUk9EdArJNeralOiAdeaMEoW7QDQe81+1Lq7UruLOZeVmf9CUAnxVYqyqHV50f2amAKwgc5ftnUqStncz2jd30A3symjTvIkacGaerTO4z98yaZUXy6UCDw+O9559ZgwaTXHEQGTxPFrEzoCL2VChzCRPeLLH2cGc2COa1J2FRxI5BEoXhsN3XC1pKnmnXDp2LHz8CW5XxNx\\\\\\\/KzzexqmVfldxs6SaSMyblQ9p2Uaj0Ohkdz8mOfOpemXgV0c8I1MLquyqh4Lo37rkECnMLU+OovbCxgrgDU3Ht1OdCdUoA4B\\\\\\\/ljKHdQp5g2BXr2s7Z6843bX\\\\\\\/0QIKoqs3b5aIX+yZaDiMMJxkEvLpi8vBm3qorqW8H68++zpsor6MmnLLQRBHHudkEIGYUYwB+cWk4EokHhtdOIA8nYFYIvxzKW7B7fRafvb\\\\\\\/J3vS5+MLUg1CqfH98DltyKeg8LAe3w5pBvgQrqC45N9Fp6nuFpcuAfYM6ZP+JEynH4HePSmWQrWJWGaPfsen0uBg00JYduNHZEOYyClwaO6dSfrU+NgWmStPdlb5sn77W+JbCoD1ckCJi31+nKBkmcuzLtTNN\\\\\\\/7RPawmo\\\\\\\/E5odls4oivSykJ10+ivo4KJnJrV+nTJ0Fn3YuExfI157+ntZFRPFNFJxDIgta4sSbcGxDqwGNL3Beq+xPWqpT\\\\\\\/\\\\\\\/CHrXYQ1QrP5qynifBnvtyS3b2+XzK77uEcam3DGoEy6Ka4gt2n2ZXlEFDJCscY8f92lMC58\\\\\\\/4JWn+\\\\\\\/xQuow9+h3b8+BW5JTJJZaQXfrU0TffotkWZfREGRjt0f8x2BaZzXBzYTfXq7276jVKd33WO8EWcrPS3Mqa40LznP2XDSratOwrH8OC9xUmd26J8IgGysgWVUMWS9pPHE0KhLJr4YA+FGC0qRHLjUdjqd7l\\\\\\\/uOnGw7yAcNu+7LNM2sOQ2FvlZCuMyAxGJ3q\\\\\\\/Xp+tKHVvtjtRCqIqGLw1sOitqlbRFZQYcXf3PsDK5myBjknv5AnFwX37cmt4HTjY3+YK0G3+XO7W8HjlP5wBeziSqF9Krlpv1KMXyd2xZ7st6B9DkSYeXeY4GRgt0hi+fjwEbLWOQaT77GLFWajY04sdjBnZK5wZuA2l\\\\\\\/X+dfYr1aWHtNg\\\\\\\/cDFZlnelSpJ4FnaghQPQqzLUG7Pom1UkwYWnnNPUq1hlWzwsEcAvVKPxokyDM4huX81qp5H1mpN5pVmzgBIn3wGUKWea52FEYZ311MaZyD9tWfmyL5CgR5YsmlX1dNqik1HP7dJZRmmGHLdYgDl0vbB3txSD+GINo9OyQpV8qAdbrZu3VA9r5Cu1ktWbBHYwcSqk893goUYrEz\\\\\\\/\\\\\\\/dmqiHXUS5ANO6sAV4TNOI6GTPAGfiuSz1eXzy36WJ+BDXXuI\\\\\\\/NLfJU\\\\\\\/rRsDoJ2RT9N2BCY9fa11+JtnsFGT9ZEd\\\\\\\/aD35zpXl9Hi533hCZtqwXSwXAUY4z9T2fhkmlF0O1d3td0\\\\\\\/JzcJHprdqhliG8xqVwmPM4eN4LM3rz5BrCOlwn0SKwZ4Lyr8+Hkl6tXqSZJU4DsqYopjAcOWfVpvB79bPYAtLUpPZw3u1KNcD406VvMTHnyBfRX857kj60xUPQfuRM1fMK0KuRyB75wVZO5GyRNGX0fd+0oS0o2HlMkuoHNJ9Mqga9tSe84OY5niDHpoZZ+R7oTZSwCemLXuUFUiueOeR2nGhe0\\\\\\\/UrOfz1y0CjieL7ptJ4ylUJs9asC0jfXFxwBbHLUMHKmb2ayL2jD9G\\\\\\\/\\\\\\\/\\\\\\\/ylnYl3kioVZIz9RUppsZEfPCpE0ESWtDfmGrm2T8UHJv4sDLavb2iAM5UCVjYeCOJxEI0co1UsMR9IoaqM3AZNF+K5moC\\\\\\\/R0zuxwPuojN+npGDqM+MtsfApvHAdO0VRyPLPf1tb5+URjOuC1\\\\\\\/Iu+7vi8pdOjh7q7CEJNGkKgKD45reHOImEbGlRBCj1Xxk97L4unvJpQ1R\\\\\\\/WJn4RbI1lVQ2XAoMCqdr++4o4aWZO+t+l51pI5K\\\\\\\/o0N00mEyYg6RzC3QtKfgjKtWwFq2UdGn8Dxzeos9ticXQvnlXPvAb54tNcZTg+ds2dpsJO76w6TuxWAeIDQ+hkQ1IKRijIRdTOALpGApCN+asSf+4SW45PAWP3m5rPOPOxb2n0PiLtxb4+52cuhQmp+Dv4jol5KLyOBFjxDJ3YX0C\\\\\\\/RXhVdhGrXHAo1iAkushIPz8UFoF1nS4NzYkbWXl69QsnGi8FzR1C\\\\\\\/3R6U+BDXCBh0OUwWu0ayFoO6DSAXy6ZPRnqhqkNR+RKeaPza7lzgPKqL7h07qtXbHfYxMYqLggosFWVWPEkTBUJcSpyvxcFqQEPH4vzabcWzGy1AWuHB50gT4nJMuYPdHVi80XWOpOgfchWURzWRY8FEgxhbdXd0oXK5UtV875YWei\\\\\\\/VVvld8AStvBf8PnN+ASdL3Pm7yk\\\\\\\/4vADstRivvFwDsGgpDaO4K97MtKSXbBTaa0uG6MP7a13P3P4A1QuHO4RLKYRbGbVGnQueiGqPlVsoGDnQCQncTcQRSoAi1So\\\\\\\/EWMdoBAx912z7rCxQBqJHzmYeoVgGHt9\\\\\\\/KqR0CDJ+h9nt5nm2BUWSxDYLZ\\\\\\\/ffKvexc4j8zF\\\\\\\/u3AsFBtL8SsYMmeFgEScLseNUZIhsN\\\\\\\/kptALJGC7wBuhNKpFrJQf4Gm6LAbKfcZ3O8jPjWQVnDWAUnVVF\\\\\\\/VWsbLNWJXYdLGNQ8MGO3uxk1eTR\\\\\\\/v7MHgm8wqZbQMesik\\\\\\\/S2HvSp8OX7LOvxCKQNEoXPz9k995FXcNftW78e8WDxxnI3AIjV5o+1eqmNzRid+iuesrr0qJrG782cXu+hLhLYW6qf\\\\\\\/hYC4x9iwHcm26L4ol0cAWhWIDKU+FTA5cVE5D460ZCm+1TmR2bb3tUr\\\\\\\/zBVstpA80txUG8UtrMUg252ADV1ZtxDpfqNCmmHwMwg8gxJAun3dzf5g89TtCZhRQOPw8Im6PDq9NLDw3sWkutvZioQbBxIBHZRkZwJ6RfAYY\\\\\\\/qzrDpMtWvvwboYurxeWrUbc60KcqmUhGv52v8q27RCD8ky8c8pySnf42BMcfTQY3sqbYlYer5jihRmaz\\\\\\\/vBjv8V+sJZfzK3J4XBuesfNWZk5m0UctxmBgwm1bnl2uRmlTU7R27Am3hsWGLyQYyxg6tzx0iD7b9vCXtUktG8BX4cGmtbE2LFUqnxfzQKwPyyEQNXVM\\\\\\\/2tnb8I4z4YOp0fPxnz1LOB46fcZMUTl4AwC9zTZvCRlb2dIky3ZiS4PMqnCW8Dx+Flz4njnDPMOMm9tfz0p+Im2Pm888unO2F2A\\\\\\\/SEmrXLKY2ULI9SAGxLwEAgCvgKxL0KP\\\\\\\/DXx9D4KpfShKRouexYPSJi4Hrr9GKGntBfvBQiqGyTjlk20+Nx+mAU7NwskB4WZTyWaLZrFvbYnyAtm\\\\\\\/Vi9t\\\\\\\/q0SSiTyt37bQE73cxEsaICB6vcatWPOlnUtLJiBLSXtPWfdcw5MwifCUm43zpMmLD5ew2T4N1W0Gcc0dcvDw37ltEnxMOhbTtT4HrmUCedv9CHL023vg1eTsVVPoEsBOA5O8wV0bdEt\\\\\\\/7nhocvPTXv8qUyuOvWMyYEacu4bNpsKBsXa4FZqwdcS8kEg\\\\\\\/jn8py\\\\\\\/6lYiGn8mmvt8Z0POPJqV5AHQ1olXx5pTDOvQHR8CnNXThrxdZS5Qyyb9AAB6yY5VoBfiB+tpjsd1ui89O+PQWN6gmPuCWnoVsMEKLWiCVeF4xymgPi5Sz1eGoODuMUKxBAbCdrOIIYoGZAzjZjrNTiFP\\\\\\\/FCnbM1fIh1oAr+HMyyaSMzO25VlgLpcoRmoyCmegRygJDTAxx3+V1DRaIB\\\\\\\/YCVgBdj0SFokpynFf7PmaA34kA0X7NUACijcrWmDe8gTPf7hn6OWQ8thLYt7UEyJ7npkmqAmindWEaXjq1LlX5aXgHkZ7J\\\\\\\/AIVgWAvLl2zMN1iM9ZhsFnhpxDXnjDrqiKxCJMCTB1JsATu84oD+oKSCcYnnaJveowO6zv5KQ0zRoD3donwM2486iYq\\\\\\\/UcI7NMwQva0NnhgqWAujq0EAt2Rj94DztTOzb1jkV06U83zFWu8B0uNxyARKLzE986JoSJycCZnsKE6Q5oUJASzQKqoJ4NeACBi3OPQot7yqh2znWJMddsTn3i4BM+0HEzyu3OhjJC5ZyAK+\\\\\\\/POguyQZkxEgR2xG4Zn5VQsQDVqewydtSgyh0oIxtc+FX2sGMirj1sxebXyy5SJQUp1bJA0IHN3E9CHJ6p\\\\\\\/62B5fxaej1GKls4OpjeSit5A0K2ZxEjiL8i5tLLBoFHdDZX28l8oTL3WqZ66YKwVgxct8D\\\\\\\/Mabrcrwf0jawDiq3\\\\\\\/JHe84JpkFDfJ39tNjM6n3deBapOcRyGvj9q2+xxxAAH5lUShsNfvJYLpSjupzYHOFYXoNKVWKUDJUiVZ2rsUySQUcIGoLtd1\\\\\\\/hGywh3z8mk7Xa2WXY6rbZ\\\\\\\/jW+8d4uRMdunIhus0ybn\\\\\\\/zJPguWxeT9+nTN9QvTZTY0VB0BXa0xfmqNF1uksZA\\\\\\\/2hZRbjCLY0RFIjMd300vXZ4tm80WFVoZEkiylh3kM6da0hrQ0dFM0RfMNUHvtFMsE+t2Ty2Twkbrwe9p+mDSIJB7bXG4Z7e4KFfWd2nXTcG9iThoMpFtaV\\\\\\\/5Ee9jEiwzVIyiru7fP7+xkSKt5sl3ywE39O0UcUKAVTxWOyYFyts4d2QxcWStGvvoLgQohOpWCOPg9T2r7cTvsncW9cceHPZq8NCwK4n4Bc58YAHwV6zcsNwIV02O\\\\\\\/4kMgsiKgJRGtHXVUQleE8IHM8DcEPN++KBwZWhCjXqItf6A2pSn88o\\\\\\\/vLovcVzv4xa+QLaGd4mL6o0PbWTxNTa0Ez8wRYSurrkcUrBsv9q35JLeCGRSayqblQ+V3pxFk3uRGWgTbRUtX3ojN3bXwO+sHthxbUmFHkAn6nsknL3GToNqq0BSdK5N+zyqR0w6c8Qeh57UGcZNnKkoH2rIII2qR\\\\\\\/\\\\\\\/GMxFtbnLcoRk95z0uiKB9Mm2gawuODKL2g7IpMk8R7e3xagM8CyuO\\\\\\\/\\\\\\\/sBx0irrnoXgqbP57EZ1iMHDTXPVFd71uENEJZft0S+Y1qbd49V76X40lQQSzBANEdYsPjyDC+uX7RuY+OCYczY\\\\\\\/si1eKFBIlscFEWVqOWs2PSqQrZ8Cwgytx6+t\\\\\\\/UwVRE2od+Rh8pZKL+9RnsL+7j617HzYDxhQEcAflpmXBLmjQ1i2A+0dgjAgClXo3zjogMUVMNWW4VvUGbjEE1NFnX\\\\\\\/HRWWNAG0fsCE161Vvvp3pUoGcJv2LbDPNd+R91BdZNB30hAFm8VmPDl9ZJ7lCEZ29Y+bBEYSiBkfOBk98Th7oAtVUtD1C0J79V6ixNsPicNSvbwtk8HvQDzXor6IlQkKQCfFvn3qS5dI+om80aObzTGf91545t4pl1s3dQ7rt8KXjDYbFw3ONIz49+7hG22CTj\\\\\\\/HuZ4oCszg8Avi\\\\\\\/+D4uy1rLxlASLJEt4pb3Gdjd0ly7BDn8Pb7mxIRWlCu\\\\\\\/glRj5fSR1c\\\\\\\/W751tdRMp9q9MxnicuG1vrFALvNny+Ni3d5q6EtoAZbMXbAgyXDXYtgej\\\\\\\/e69CATxwSd2N2g5427QVN7q0UPDoYF197vZKxwdpTW74kKrzV2JvNGprimteoeCgDyg5gcOixMv1Y5xT+iYXX0qfSgQGqoJOftHfEsvDaQbx\\\\\\\/fZfZ5dK7CyoMfJXXl3ohF1hffDHY4VJDaLTOXVXP\\\\\\\/bKgXyqrhphbDXyeVnwtzAVpJl2Mutvdfwakxfu3wbGl50WOka6iX4NRZouT4aGXfm7uuaq3lMLhntO6MfX\\\\\\\/OLMhW5lPcJDSbGhtmtKfwgRji0OEOErM\\\\\\\/6MjT1t8If+FlNW6SH7Sk2fzAwb1rf3rgXNftQv9XCuGnplwsaHyqnH9jftLcsS04XTKhlWlIEEOWl7TXc5NOFys3319T8QgZoubkf6xlumGi4hRFmOv5\\\\\\\/j2xLA3aPHl3ITqlW1EverEMM79JPdtjSXsQ5RQQGKEni\\\\\\\/bg1aAjVfKWaQyj+v\\\\\\\/ZyyNgixJQ3XERr7QnXWGloLCHSl8MBQ3sbZqsRnDHwKvSXWbfPBwqaYsmRFwYk5pMyBNJ2u4bbMaelRdzMhdoaMskqdbozLfm3yiCDwoQcpdK5qAfam47hV1Buvx7pIhwJWNq6vAfosaDMG3by6pnZfkA\\\\\\\/8ut1Cd3dNmU8TiJY2Mpeabj0S5YiWOjEr8o6cwt1gl6IO6rafOqRZFcyidFyN6Fd4rMc9UDVuD6SlThxDMq1aQ9BSUKyRzmf3VeEWXuJAirmT0rMg6Z3hq5cDEiMJAugEkjbS6Z2QMijeBFuvsb6wBOxifUrbg4W4KBwOwjQppstEYh\\\\\\\/+uXFZnt\\\\\\\/4kM74dpYm51jBUd2CVED07ld2\\\\\\\/ggdC1E6uyjrI1VEhCxUSiLMX\\\\\\\/EeMPZU1KaDtPkB53GFf3M95mLLOKayIWZ9Z25ivVltjiVrbi59UAJDUHuxR7Fctr\\\\\\\/znLcn1quSt8u0akdQzk1eJMxBEQYGsj45FpixeVl6pBOq3gIFruCiLhAzKeRWvc90fahTsE7LNYfgXSxdK1Yt5QEXCRjDBS6WYJ0LN0key51V\\\\\\\/gYmA1VJu\\\\\\\/FmLimsEc4aC37n8r\\\\\\\/jaDyXbhP\\\\\\\/yLVsGPhTT0W3aW0pEBgxQodALy7r+Ie4UDBoumi5m0tUTqkNCzM\\\\\\\/iAKic0dWbdUid70Sk24a+Rf3jcDhgLTnK8ZFZdl0VcEWgVBc+tnH1IePmPoMMgHd3WcpykBPs0SUvEdCTuZZf95ezejQ\\\\\\\/vdL1rQA90vf26UZXa1GUJkooVFH8H7TrHZGZ2IZD18MeqmNQBpAAw8r4bdhqSjuJZDQZ+KYX5Zkaddwx9VZFZ\\\u0022,\\\u0022iv\\\u0022:\\\u0022d5F9ylqorOCqDFpTW6TPDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fdf059f4283d7d2b60c882e1959ee8405f60390a84ac4cb169c354e0b12a1ae0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jiAytiGE3R+aoaqRfkiLBGOI95c4xfnZNyZR5uMLXAhpnbO7D8KB2bVuVq0k\\\\\\\/JkgiiiqOp1krYx93G09nMg1\\\\\\\/+C0T707H5LIJdFMuOOY7dGUifuO2n908kO1ivvqHw+YwCu7dIl3d2VZOih9w5YK9u08tUOP9qqedmbBjJU5l7t4hJfiL6K4mAAbfidkaK4IJQoRnyJCjtOSsvGWnIRu6ylBKgoq1M5lnJW\\\\\\\/Te8VY\\\\\\\/pJMrmMSZkPrI6pSXRDX2rGvygDUzpHvQ2cYZVqyDe1WZnP1\\\\\\\/53OvusthtU7t+A8cA0DcAQZgC6MXEeqTyeOOLFb+OwbxYkRzS2Ua5ONeTFKdeDZqWzRbgJbagBuzyZarZ+F8bzkPsUEz5dEirX2oY4LKoRlHACBLIsa\\\\\\\/r4SE\\\\\\\/NlbbcUZeF7meHnXBbYf+P8cJ6Y\\\\\\\/ge7r5rqu2Ds\\\\\\\/T1L\\\\\\\/h1Iz\\\\\\\/AfIzO2cms4I1sWMA+YtIphaG0ZZqPxvDOY3VvY2DX13dFKj4ftYeei45nTSbsWN6yMC5n7wpY9Vw3PCuSJwvT+3w9GzhAIoL5k5k6WHRgiItHhJ7U4rcZsbldq\\\\\\\/ZC05A+wrz0QK1UDF23+k\\\\\\\/QOQl+TqAX7YjSOp85VvyEFVDqEzBzURF4ljlDyOmw7hAOTcGPjdNGpdJozm8AGDoGOHO3hmdPaXGAXH5svrz+dM79H79FPFlUy73TVX2QM74Vq2Ekt9PtThD8bBOqsk8J4gbXtgzHjv00bwykwmHbwD7\\\\\\\/\\\\\\\/Tf8FeId83g9pC1Gp\\\\\\\/paLbn3JFX5Pyxs4pceb55pHobbLBPZYArLD5k2NVCcfWN0bimx9\\\\\\\/ijW0t\\\\\\\/STS7+Ln0pMK4NFwNBS+k1b9Yp0OwWpUHHQHwZGW5lWOKLrRo+FRGvsnG3wCb5rS0CFDX3hVwgtKctX2X9cVSKBdNE2\\\\\\\/Z9uDEKNOdYFgiUrWyeEm5bpLGTtkLCU6n0OhdsK\\\\\\\/OKs0oj+K3f39bejTUr7pbXBEipRKtfHK4S\\\\\\\/ZBQPJuRfsPPAYeQw+72fAHYX8KRdzFgFEftrGWe5CB623XgBZvPEIHkr5nvHQCO5HJX4Isih9qjEPwGn0KBH6Lv2AHZIGN11wwzTu8EXJoI4K7cWG6ndMILQ6j9kB6a00UCNZylZ1YqULkX83fwpksGgwmo0fcGoIWXTPlvZQE9HHerFz0QSLbuOXiPNlMZ0n2O5K4\\\\\\\/TqUweNqt6AbwOvaJKHIjLfIpTFgIVW0KA9JcGErKQgn9UKOKZckFttPzlx4M5Y\\\\\\\/SGps0C1WNGWWe2Mz9ngt1pETbxugeZWJmtc4Tna9XDxlzQql1e9Oir6YHLLXjhppAjvmp1+BgQ0mHzuLKRosF3YjtukvW\\\\\\\/PbOcP8ELDah2q1O2r+f1DfqCytAxE7x2UJJ7iGu9LXcJwOU6oLVdcQyJ2tTxRGRXimhmWJZzwOloQeXbjJBHQz5ZNdm\\\\\\\/4oxQ8ItS5+Rd9CR84xmTnIwAtpk4W6CrDjrrm6de\\\\\\\/FsC8z1Ah0UCcRgn8xmTzEv91dn1WmkZjF\\\\\\\/JMeYYHinJImGntimG\\\\\\\/suOGGKMftBvSYPe8hZnmC2hPznCY2gbbctKaojhtvrVqQ2EOnVSpNAQNHLDCzgzNHdNNrTb4GiPUIR9+5Uct2c41nI6iexHaHVwaog9WrgPf5ezEe27pSgXmAWfGL5rnTkynBA8txTRKeGqQIy2J+3O14jBwUkdt3TmR9Oq96DfIv+\\\\\\\/\\\\\\\/yJUsPUv+cRULwSKeXSQoKYqfJGYAmXybVZNv1Cj1F8bNIO6HtT6unfsyrjR\\\\\\\/NL2HP+a76bpz7K4\\\\\\\/1dAXmOTFxLj5eNpFS0RvdH5Ai47nGDYFLRdM88M1CDG+Y11idEtI04hURvi1wzDjPWfZ\\\\\\\/leSpxqwvtZleJRWxckAEyQeSl9AKCZhzuWvI6SAdm5mHLFZrx8ROZv7TqpZ4p3VJlRTRZKANIUc\\\\\\\/9snkem1jhwAR4Lxy\\\\\\\/DISTBcGT3IDVpc+4Fr+PHswOEm5aiEAWitHROV72Ah5bbrpt6ZqLGxBGzNNYtQr1HWZZRdS75Of1JQz2OhMmZ2JLT5IO1AGQlRGbgEf\\\u0022,\\\u0022iv\\\u0022:\\\u0022GCnAD+iWQQb1B1oM9OEPXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022032814a4f47ee9080b49ca1b551b9bfd355d036078996854486cca1fda3c748f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494275 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3249" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vw-kWtsss0o6EOeLBcUqF3jYBzrUpUbHQPt3nLI_yEI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3249" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bwSXV9r8-w7KPJXoDtuue1qvi-lpB0r37wSmHM6rfLU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3239" title="Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3263" title="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494275" id="node494275" data-pisa="ggg;9/10/3249" data-pisa-master="ggg;g3.119.400445" data-apath="/ggg/9/10/3249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the <em>Anopheles gambiae</em> Complex</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8725-3929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Rebecca Love</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2653-760X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Seth N. Redmond</span>, <span class="highwire-citation-author" data-delta="2">Marco Pombi</span>, <span class="highwire-citation-author" data-delta="3">Beniamino Caputo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7939-6786" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vincenzo Petrarca</span>, <span class="highwire-citation-author" data-delta="5">Alessandra della Torre</span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-collab" data-hwp-id="collab-1">The <span class="nlm-italic" data-toggle="yes">Anopheles gambiae</span> 1000 Genomes Consortium</span></span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0646-0721" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nora J. Besansky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3249-3262;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400445 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494275" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. Rebecca Love </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Eck Institute for Global Health & Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BRebecca%2BLove%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Love%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BRebecca%2BLove%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8725-3929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Rebecca Love</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Seth N. Redmond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Infectious Disease and Microbiome Program, Broad Institute, Cambridge, MA 02142</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seth%2BN.%2BRedmond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Redmond%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeth%2BN.%2BRedmond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2653-760X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Seth N. Redmond</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marco Pombi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BPombi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pombi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BPombi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Beniamino Caputo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beniamino%2BCaputo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caputo%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeniamino%2BCaputo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vincenzo Petrarca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincenzo%2BPetrarca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrarca%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVincenzo%2BPetrarca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7939-6786" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vincenzo Petrarca</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alessandra della Torre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alessandra%2Bdella%2BTorre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=della%20Torre%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlessandra%2Bdella%2BTorre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nora J. Besansky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Eck Institute for Global Health & Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BJ.%2BBesansky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Besansky%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANora%2BJ.%2BBesansky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0646-0721" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nora J. Besansky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nbesansk@nd.edu" class="" data-icon-position="" data-hide-link-title="0">nbesansk@nd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3249" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494275" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3249.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494275" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3249.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494275" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3249.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494275" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Chromosomal inversion polymorphisms play an important role in adaptation to environmental heterogeneities. For mosquito species in the <em>Anopheles gambiae</em> complex that are significant vectors of human malaria, paracentric inversion polymorphisms are abundant and are associated with ecologically and epidemiologically important phenotypes. Improved understanding of these traits relies on determining mosquito karyotype, which currently depends upon laborious cytogenetic methods whose application is limited both by the requirement for specialized expertise and for properly preserved adult females at specific gonotrophic stages. To overcome this limitation, we developed sets of tag single nucleotide polymorphisms (SNPs) inside inversions whose biallelic genotype is strongly correlated with inversion genotype. We leveraged 1,347 fully sequenced <em>An. gambiae</em> and <em>Anopheles coluzzii</em> genomes in the Ag1000G database of natural variation. Beginning with principal components analysis (PCA) of population samples, applied to windows of the genome containing individual chromosomal rearrangements, we classified samples into three inversion genotypes, distinguishing homozygous inverted and homozygous uninverted groups by inclusion of the small subset of specimens in Ag1000G that are associated with cytogenetic metadata. We then assessed the correlation between candidate tag SNP genotypes and PCA-based inversion genotypes in our training sets, selecting those candidates with &gt;80% agreement. Our initial tests both in held-back validation samples from Ag1000G and in data independent of Ag1000G suggest that when used for <em>in silico</em> inversion genotyping of sequenced mosquitoes, these tags perform better than traditional cytogenetics, even for specimens where only a small subset of the tag SNPs can be successfully ascertained.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/anopheles-gambiae" class="hw-term hw-article-keyword hw-article-keyword-anopheles-gambiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Anopheles gambiae</em></a></li><li class="kwd"><a href="/keyword/chromosomal-inversion-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-chromosomal-inversion-polymorphism" rel="nofollow">chromosomal inversion polymorphism</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/inversion-genotyping" class="hw-term hw-article-keyword hw-article-keyword-inversion-genotyping" rel="nofollow">inversion genotyping</a></li><li class="kwd"><a href="/keyword/karyotype-analysis" class="hw-term hw-article-keyword hw-article-keyword-karyotype-analysis" rel="nofollow">karyotype analysis</a></li><li class="kwd"><a href="/keyword/malaria-vector" class="hw-term hw-article-keyword hw-article-keyword-malaria-vector" rel="nofollow">malaria vector</a></li><li class="kwd"><a href="/keyword/tag-snp" class="hw-term hw-article-keyword hw-article-keyword-tag-snp" rel="nofollow">tag SNP</a></li></ul><p id="p-5">A chromosomal inversion originates when a chromosome segment reverses end to end. Inversions maintained in plant and animal populations as structural polymorphisms tend to be large (several megabases) and contain hundreds of genes (reviewed in <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wellenreuther and Bernatchez 2018</a>). Long-term balancing selection can maintain these polymorphisms through millions of generations and multiple species radiations (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Wellenreuther and Bernatchez 2018</a>). Because recombination is greatly reduced between opposite orientations in inversion heterozygotes, inversions preserve selectively advantageous combinations of alleles despite homogenizing gene flow in collinear regions. Theory and mounting evidence implicate inversions in local adaptation, adaptive divergence, and range expansion, though the precise molecular mechanisms are rarely known (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hoffmann <em>et al.</em> 2004</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hoffmann and Rieseberg 2008</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schaeffer 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kirkpatrick 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lowry and Willis 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Joron <em>et al.</em> 2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jones <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kirkpatrick and Barrett 2015</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Twyford and Friedman 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kapun <em>et al.</em> 2016</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ayala <em>et al.</em> 2017</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Fuller <em>et al.</em> 2017</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wellenreuther <em>et al.</em> 2017</a>; <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Wellenreuther and Bernatchez 2018</a>). Importantly, because of occasional double-crossovers and gene conversion, the suppression of gene flux is not absolute. As long as inversion heterozygotes are formed in populations, any significant association between an inversion and an allele within its boundaries is subject to eventual erosion unless gene flux is countered by selection (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Navarro <em>et al.</em> 1997</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andolfatto <em>et al.</em> 2001</a>).</p><p id="p-6">The <em>Anopheles gambiae</em> complex is a medically important group of at least eight closely related and morphologically indistinguishable mosquito sibling species from sub-Saharan Africa (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">White <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Coetzee <em>et al.</em> 2013</a>). Three members of the complex (the eponymous <em>Anopheles gambiae</em>, <em>Anopheles coluzzii</em>, and <em>Anopheles arabiensis</em>) are among the most significant malaria vectors globally, responsible for a majority of the 435,000 malaria deaths in 2017 (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">World Health Organization 2018</a>). The ecological plasticity of these three species contributes greatly to their status as major human malaria vectors (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> 2002</a>). In contrast to the other five, these three species have wide distributions across diverse biomes of tropical Africa. Not coincidentally, they also segregate strikingly high numbers of paracentric inversion polymorphisms, which are implicated in adaptation to seasonal and spatial environmental heterogeneities related both to climatic variables and anthropogenic alterations of the landscape (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Coluzzi <em>et al.</em> 1979</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bryan <em>et al.</em> 1982</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Coluzzi <em>et al.</em> 1985</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Toure <em>et al.</em> 1998</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Manoukis <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Costantini <em>et al.</em> 2009</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Simard <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ayala <em>et al.</em> 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Caputo <em>et al.</em> 2014</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ayala <em>et al.</em> 2017</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cheng <em>et al.</em> 2018</a>). Some of these inversions also have been associated with ecologically relevant phenotypes, including desiccation and thermal tolerance (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gray <em>et al.</em> 2009</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rocca <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cassone <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fouet <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ayala <em>et al.</em> 2019</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cheng <em>et al.</em> 2018</a>).</p><p id="p-7">The sister taxa <em>An. gambiae</em> and <em>An. coluzzii</em>, the focus of the present investigation, are the most closely related species in the <em>An. gambiae</em> complex, sharing extensive nucleotide variation through both recent common ancestry and introgression (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fontaine <em>et al.</em> 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hanemaaijer <em>et al.</em> 2018</a>), while maintaining characteristic differences in ecology and behavior (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Costantini <em>et al.</em> 2009</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Diabaté <em>et al.</em> 2009</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Simard <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gimonneau <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gimonneau <em>et al.</em> 2012a</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gimonneau <em>et al.</em> 2012b</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dabire <em>et al.</em> 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Tene Fossog <em>et al.</em> 2015</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Ayala <em>et al.</em> 2017</a>). They also share four of six common chromosomal inversion polymorphisms on chromosomal arm 2R (<em>b</em>, <em>c</em>, <em>d</em>, <em>u</em>) and the only inversion polymorphism on chromosomal arm 2L (<em>a</em>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">della Torre <em>et al.</em> 2005</a>). These inversions range in size from ∼4Mb to 22Mb, and together span thousands of genes and a sizeable fraction of chromosome 2: ∼61% of 2R and ∼38% of 2L polytene (euchromatic) content (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> 2002</a>). Inversions 2L<em>a</em> and 2R<em>b</em> are found in populations throughout tropical Africa and are therefore cosmopolitan, while three other inversions on 2R (<em>c</em>, <em>d</em>, and <em>u</em>) are widespread in West, very rare in Central Africa, and absent from East Africa. The remaining two inversions, 2R<em>j</em> and 2R<em>k</em>, have more restricted geographic distributions (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> 2002</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Ayala <em>et al.</em> 2017</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagrammatic representation of the common polymorphic inversions (labeled brackets) on chromosome 2 in An. gambiae. Polytene chromosome map modified from Figure 1 and poster in Coluzzi et al. (2002). CT, centromere." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-41985999" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagrammatic representation of the common polymorphic inversions (labeled brackets) on chromosome 2 in &lt;em&gt;An. gambiae&lt;/em&gt;. Polytene chromosome map modified from Figure 1 and poster in Coluzzi &lt;em&gt;et al.&lt;/em&gt; (2002). CT, centromere.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3249/F1.medium.gif" width="440" height="38"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3249/F1.medium.gif" width="440" height="38"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Diagrammatic representation of the common polymorphic inversions (labeled brackets) on chromosome 2 in <em>An. gambiae</em>. Polytene chromosome map modified from <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and poster in <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> (2002)</a>. CT, centromere.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Early cytogenetic studies of <em>An. gambiae</em> and <em>An. coluzzii</em>, presumed at the time to be a single heterogenous species, uncovered genetic discontinuities that led to the designation of five presumed assortatively-mating ‘chromosomal forms’: FOREST, SAVANNA, MOPTI, BAMAKO, and BISSAU (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Coluzzi <em>et al.</em> 1985</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Toure <em>et al.</em> 1998</a>; <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> 2002</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">della Torre <em>et al.</em> 2005</a>). They were delineated based on stable non-random associations of different sets of chromosome 2R inversions in co-occurring populations, and differed in larval ecology. Subsequent DNA-based studies identified fixed differences in the ribosomal DNA (rDNA), located in the pericentromeric region of the X chromosome, leading to the definition of two assortatively mating M and S ‘molecular forms’ of <em>An. gambiae</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">della Torre <em>et al.</em> 2001</a>). The molecular forms, which were eventually given specific status as <em>An. coluzzii</em> (formerly M) and <em>An. gambiae sensu stricto</em> (formerly S) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Coetzee <em>et al.</em> 2013</a>), are incongruent with the chromosomal forms. Nearly all inversion associations segregate in both species albeit at different frequencies, and likely play similar roles in ecological specialization and adaptation in both <em>An. gambiae s.s</em>. (hereafter, <em>An. gambiae</em>) and <em>An. coluzzii</em> (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">della Torre <em>et al.</em> 2005</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Costantini <em>et al.</em> 2009</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Simard <em>et al.</em> 2009</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Ayala <em>et al.</em> 2017</a>). Hence, inversion associations are indicative of environmental heterogeneities more so than intrinsic reproductive boundaries.</p><p id="p-10">Beyond a role in ecological specialization, inversions in the <em>An. gambiae</em> complex are also associated with vector traits affecting malaria transmission intensity and control: biting and resting behavior (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Coluzzi <em>et al.</em> 1979</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Riehle <em>et al.</em> 2017</a>), seasonality (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rishikesh <em>et al.</em> 1985</a>), morphometric variation (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Petrarca <em>et al.</em> 1990</a>), and <em>Plasmodium</em> infection rates (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Petrarca and Beier 1992</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Riehle <em>et al.</em> 2017</a>). Although a robust molecular assay is available for genotyping inversion 2L<em>a</em> in natural populations (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">White <em>et al.</em> 2007</a>), 2R inversions with characterized breakpoint sequences (<em>j</em>, <em>b</em>, <em>c</em>, and <em>u</em>) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Coulibaly <em>et al.</em> 2007a</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sangare 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lobo <em>et al.</em> 2010</a>) proved difficult to genotype molecularly at the breakpoints (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Coulibaly <em>et al.</em> 2007b</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Lobo <em>et al.</em> 2010</a>), owing to extensive tracts of flanking repetitive DNA. The 2R<em>k</em> breakpoints have yet to be characterized, but recent localization of the 2R<em>d</em> breakpoints in the reference genome assembly using proximity-ligation sequencing (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Corbett-Detig <em>et al.</em> 2019</a>) also revealed high repeat content, suggesting that repetitive DNA at inversion breakpoints will pose a significant challenge both for breakpoint characterization and for molecular genotyping assays targeting breakpoint regions in these species.</p><p id="p-11">Failure to account for the presence of inversions is a barrier to a more comprehensive understanding of epidemiologically relevant mosquito behavior and physiology. Inversion-blind analysis of population data can mislead population genetic inference, and create spurious associations in genome-wide association studies (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Seich Al Basatena <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Houle and Marquez 2015</a>). Powerful genomic resources exist for <em>An. gambiae</em>, including a high-quality reference genome assembly (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Holt <em>et al.</em> 2002</a>) and a database of genomic variation (Ag1000G) based on deep genome re-sequencing of thousands of mosquitoes from natural populations across Africa (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">The <em>Anopheles gambiae</em> 1000 Genomes Consortium 2017</a>). Unfortunately, inversion genotypes are not automatically revealed by genome re-sequencing, as reads are mapped to their position in the reference genome assembly, not their position in the re-sequenced mosquito genome. Despite advancing genome technology, the only method currently available to determine the <em>An. gambiae</em> karyotype is a method perfected half a century ago (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Coluzzi 1968</a>) involving cytological analysis of ovarian nurse cell polytene chromosomes (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> 2002</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pombi <em>et al.</em> 2008</a>). At best, such cytological analysis is severely rate-limiting because it is laborious and requires highly specialized training. At worst, it is prohibitive because it requires proper preservation of chromosomes harvested only from ovaries of adult females at a specific gonotrophic stage; suitable polytenization is absent at other gonotrophic stages as well as in males (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">della Torre 1997</a>). While salivary glands of late fourth instar larvae also contain chromosomes with an adequate degree of polytenization, and the banding patterns of salivary and ovarian chromosomes are homologous in principle, most bands are difficult to homologize due to a different pattern of chromosome ‘puffing’ (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">della Torre 1997</a>), rendering this alternative impractical. To overcome these impediments, our goal is to develop broadly accessible computational and molecular methods of genotyping chromosomal inversions in individual specimens of <em>An. gambiae</em> and <em>An. coluzzii</em>.</p><p id="p-12">Here, we exploit the Ag1000G database and leverage the subset of cytologically karyotyped specimens within that database to develop a computational approach for karyotyping applicable to whole genome sequence data. We identify multiple tag single nucleotide polymorphisms (SNPs) significantly associated with inversions across geography that collectively predict with high confidence the genotypes of six common polymorphic inversions on chromosome 2 (<em>a</em>, <em>j</em>, <em>b</em>, <em>c</em>, <em>d</em>, <em>u</em>) in individually sequenced genomes of <em>An. coluzzii</em> and <em>An. gambiae</em>. We then apply this approach to data generated independently of Ag1000G to show that our approach has wider utility, even for specimens where only a small subset of the tag SNPs can be successfully ascertained.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mosquito genotype data</h3><p id="p-13">Variant call data used for the discovery of inversion tag SNPs were accessed from Ag1000G (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">The <em>Anopheles gambiae</em> 1000 Genomes Consortium 2017</a>) and Vector Observatory (VOBS; Table S1), projects of the Malaria Genomic Epidemiology Network (MalariaGEN; <a href="https://www.malariagen.net/">https://www.malariagen.net/</a>) that provide catalogs of genomic sequence variation based on individual wild-collected <em>An. gambiae</em> and <em>An. coluzzii</em> mosquitoes sampled from multiple African countries and the Mayotte archipelago. With the exception of four atypical samples (see next section), we verified species identifications as reported in Ag1000G and VOBS using principal component analysis (PCA) of biallelic SNPs on the X chromosome. We excluded any specimens with more than 50,000 missing genotypes on chromosomal arm 2R (N = 9), and any specimens subjected to whole genome amplification (WGA) prior to genomic sequencing (N = 44), as PCA revealed strong biases associated with WGA. After filtering, we retained variant call data from 1,347 mosquitoes (Table S2).</p></div><div id="sec-3" class="subsection"><h3>Karyotype imputation by local PCA</h3><p id="p-14">Cytological karyotype information derived from phase contrast microscopy of ovarian polytene chromosomes (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">della Torre 1997</a>) was available only for a relatively small subset of specimens (N = 373) in Ag1000G/VOBS (hereafter, Ag1000G for brevity). Thus, as a first step toward discovering SNPs putatively predictive of inversion status (tag SNPs), we imputed karyotypes computationally at each of six focal inversions (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), using local PCA (where ‘local’ refers to windows of the genome corresponding to chromosomal rearrangements). <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ma and Amos (2012)</a> showed that applying PCA to SNP genotypes in a window of the genome containing an inversion polymorphic in population genomic data (an approach that we call ‘PCA karyotyping’) produces a pattern of three equidistant clusters (stripes) in a plot of the first two principal components, assuming adequate numbers of each of three possible inversion genotypes: inverted and uninverted (standard) homokaryotypes, and heterokaryotypes. The two flanking stripes represent alternative homokaryotypes, and the middle stripe represents the inversion heterokaryotype, a 1:1 “admixture” between the two homokaryotype classes (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ma and Amos 2012</a>).</p><p id="p-15">To apply this approach, we combined specimens from both species (<em>An. gambiae</em> and <em>An. coluzzii</em>) and different geographic localities into a single metapopulation sample of 1,347 mosquitoes (Tables S1, S2). We identified a set of biallelic SNPs within inversion boundaries (Table S3) with potentially informative levels of polymorphism [minor allele count ≥3 and minimum alternate allele frequency (MAF) ≥0.15 for all inversions except 2R<em>d</em>, for which the MAF threshold was reduced to 0.03]. As 2R<em>d</em> overlaps 2R<em>u</em> in the genome (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>), we limited consideration to only those SNPs found outside (proximal to) 2R<em>u</em> for PCA karyotyping of 2R<em>d</em> (Table S3). Next, we converted mosquito genotypes at these SNPs to a count of the number of alternate alleles (‘0’ if both matched the reference allele, ‘1’ or ‘2’ if one or both matched the alternate allele, respectively). Using the scikit-allel Python package v1.1.9 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Miles and Harding 2017</a>), we then applied PCA to the resulting matrix of alternate allele counts, and represented the output as a scatter plot of the first two principal components for each mosquito in the population sample. The correct genotype corresponding to the two homokaryotype stripes was determined based on the inclusion in a given stripe of mosquitoes with cytologically determined karyotype. Based on this classification, mosquitoes without cytologically determined karyotypes were assigned a PCA karyotype.</p><p id="p-16">The distinction between stripes was not always sharp; the stripes could be diffuse and oblique rather than tightly clustered. In extreme cases, stripes were not initially discernable. Through an iterative process of ‘leave one population sample out’ followed by PCA, we determined that absence of a clear three-stripe pattern was attributable to some or all of the same four atypical source populations, in particular, those from Kenya, Mayotte, The Gambia, and Guinea Bissau. The Kenyan sample has been found to display signs of extreme inbreeding (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">The <em>Anopheles gambiae</em> 1000 Genomes Consortium 2017</a>), and Mayotte is an island whose mosquito population is plausibly subject both to inbreeding and a degree of isolation from mainland samples. The Gambia and Guinea Bissau are localities with unusually high degrees of hybridization and introgression between <em>An. gambiae</em> and <em>An. coluzzii</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Caputo <em>et al.</em> 2008</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Oliveira <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Caputo <em>et al.</em> 2011</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Marsden <em>et al.</em> 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Weetman <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Nwakanma <em>et al.</em> 2013</a>). Where necessary, we removed these population samples, as well as two <em>An. gambiae-An. coluzzii</em> hybrid specimens from Burkina Faso and Guinea Conakry, and repeated the PCA. In addition, successful PCA karyotyping of 2R<em>d</em> and 2R<em>j</em> required the removal of all <em>An. coluzzii</em> specimens owing to taxonomic structuring of variation. Accordingly, PCA karyotyping was successful on all (2L<em>a</em>) or subsets (all 2R inversions) of the 1,347 specimens (Table S4).</p></div><div id="sec-4" class="subsection"><h3>Discovery of SNPs predictive of inversion orientation</h3><p id="p-17">The PEST reference genome assembly for <em>An. gambiae</em> (AgamP4; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Giraldo-Calderón <em>et al.</em> 2015</a>) was derived from a colony whose karyotype was homozygous standard with respect to all common chromosomal inversions in this species. We therefore had the general expectation that an individual SNP might be a good predictor of chromosomal inversion orientation if the reference allele is strongly associated with the standard arrangement and the alternate allele is strongly associated with the inverted arrangement within and across population samples. As shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> in overview, we assessed SNP genotype-inversion genotype concordance for each inversion in individual mosquitoes, limiting our assessment to potentially more informative, higher frequency biallelic SNPs inside inversion boundaries (<em>i.e.</em>, those with MAF ≥ 5%). We converted both the SNP genotype and the corresponding mosquito’s PCA-based inversion genotype to single numbers, representing the count of alternate alleles (0, 1, or 2) in the case of SNP genotype, and the count of inverted chromosomes (0, 1, or 2) in the case of inversion genotype. Successfully performing tags are expected to have a SNP genotype that correlates strongly with the PCA-based inversion genotype.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of correspondence between SNP and inversion genotype in each mosquito. For each chromosomal rearrangement and mosquito, biallelic SNP genotypes inside rearrangement boundaries were converted to a number representing the count of alternative alleles (relative to the AgamP4 reference). We applied PCA to the resulting matrix to assign each individual mosquito an inversion genotype. The expectation is that the PCA-based genotype, expressed as the number of inverted chromosomes at the focal rearrangement, should match the number of alternative alleles at SNPs predictive of inversion status (tag SNPs)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-41985999" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Assessment of correspondence between SNP and inversion genotype in each mosquito. For each chromosomal rearrangement and mosquito, biallelic SNP genotypes inside rearrangement boundaries were converted to a number representing the count of alternative alleles (relative to the AgamP4 reference). We applied PCA to the resulting matrix to assign each individual mosquito an inversion genotype. The expectation is that the PCA-based genotype, expressed as the number of inverted chromosomes at the focal rearrangement, should match the number of alternative alleles at SNPs predictive of inversion status (tag SNPs).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3249/F2.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3249/F2.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494440" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Assessment of correspondence between SNP and inversion genotype in each mosquito. For each chromosomal rearrangement and mosquito, biallelic SNP genotypes inside rearrangement boundaries were converted to a number representing the count of alternative alleles (relative to the AgamP4 reference). We applied PCA to the resulting matrix to assign each individual mosquito an inversion genotype. The expectation is that the PCA-based genotype, expressed as the number of inverted chromosomes at the focal rearrangement, should match the number of alternative alleles at SNPs predictive of inversion status (tag SNPs).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">More formally, we sought to identify candidate tag SNPs using the procedure illustrated in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> (applied separately for each inversion). Specimens assigned a PCA-based karyotype for a focal inversion were divided into a training sample used for tag SNP discovery (75%) and a validation sample that was held in reserve until a later time (25%), using the model_selection module of the scikit-learn Python package (v0.19.2) (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pedregosa <em>et al.</em> 2011</a>). We ensured that both partitions were balanced with regard to inversion genotypes but randomized in all other respects. For robust identification of candidate tag SNPs within the training sample, we masked all SNP genotypes inside the inversion boundaries with a genotype quality (GQ) below 20.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of experimental design. For each inversion, the appropriate Ag1000G sample of mosquitoes that had been successfully karyotyped by PCA was partitioned into a training set (75%) and a validation set (25%). Ten bootstrap replicates of the training set were created from a random sample of 75% of the full training set. For each bootstrap replicate and each mosquito, higher frequency biallelic SNPs within inversion breakpoints were interrogated for genotypic concordance with the PCA-based genotype. Results were summarized across the ten replicates to create a set of candidate tag SNPs with concordance rates exceeding 80%. Candidate tags were used to genotype the held-out validation set, and the computational karyotype score computed across tags was compared to the PCA-based karyotype. Candidate tags were also used to interrogate mosquitoes sequenced independently of Ag1000G, and the computational karyotype score was compared to the associated cytogenetically determined karyotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-41985999" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of experimental design. For each inversion, the appropriate Ag1000G sample of mosquitoes that had been successfully karyotyped by PCA was partitioned into a training set (75%) and a validation set (25%). Ten bootstrap replicates of the training set were created from a random sample of 75% of the full training set. For each bootstrap replicate and each mosquito, higher frequency biallelic SNPs within inversion breakpoints were interrogated for genotypic concordance with the PCA-based genotype. Results were summarized across the ten replicates to create a set of candidate tag SNPs with concordance rates exceeding 80%. Candidate tags were used to genotype the held-out validation set, and the computational karyotype score computed across tags was compared to the PCA-based karyotype. Candidate tags were also used to interrogate mosquitoes sequenced independently of Ag1000G, and the computational karyotype score was compared to the associated cytogenetically determined karyotype.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3249/F3.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3249/F3.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494307" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Overview of experimental design. For each inversion, the appropriate Ag1000G sample of mosquitoes that had been successfully karyotyped by PCA was partitioned into a training set (75%) and a validation set (25%). Ten bootstrap replicates of the training set were created from a random sample of 75% of the full training set. For each bootstrap replicate and each mosquito, higher frequency biallelic SNPs within inversion breakpoints were interrogated for genotypic concordance with the PCA-based genotype. Results were summarized across the ten replicates to create a set of candidate tag SNPs with concordance rates exceeding 80%. Candidate tags were used to genotype the held-out validation set, and the computational karyotype score computed across tags was compared to the PCA-based karyotype. Candidate tags were also used to interrogate mosquitoes sequenced independently of Ag1000G, and the computational karyotype score was compared to the associated cytogenetically determined karyotype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Next, we created ten bootstrap replicates of the training sample (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Each of the ten replicates consisted of sub-samples of 75% of the full training sample, chosen at random with respect to all variables except inversion genotype balance. For each bootstrap replicate at each interrogated SNP (biallelic, MAF ≥ 5%), we calculated the SNP genotype-inversion genotype concordance for each mosquito in the sample, as described above (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Genotypic concordance at each SNP interrogated in a given bootstrap replicate was expressed as the percentage of mosquitoes for which the number of alternate SNP alleles matched the number of inverted chromosomes. Because an imbalance among inversion genotypes could lead to false-positive tag SNPs, we calculated concordance separately for the three inversion genotypes in each of the ten bootstrap replicates. We then averaged the concordance scores across the ten replicates, by inversion genotype. To generate a single, conservative tag SNP concordance statistic, we used the minimum of the three mean values. Note that because the mosquito composition differed among bootstrap replicates, some SNPs were not evaluated in all ten, if they did not pass our filters in one or more iterations. Finally, to eliminate SNP positions with high levels of missing genotypes, we also calculated for each inversion genotype in each bootstrap replicate the percentage of mosquitoes with SNP genotype calls at the candidate tag (the ‘call rate’), and averaged across the ten replicates.</p><p id="p-22">The procedure just described returned from 99 to 349 candidate tag SNPs for five inversions, but only two for 2R<em>c</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We therefore adopted a modified approach to control for suspected population structure. One possible source of structure was the haplotype configuration of 2R<em>c</em> with respect to the flanking inversions (2R<em>b</em> and 2R<em>u</em>) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). The inverted orientation of 2R<em>c</em> is in almost perfect linkage disequilibrium with the inverted orientation of either 2R<em>b</em> (as haplotype ‘2R<em>bc</em>’) or 2R<em>u</em> (as haplotype ‘2R<em>cu</em>’). In a ∼50-year cytogenetic database compiled from samples collected in many parts of sub-Saharan Africa (described in <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Pombi <em>et al.</em> 2008</a>), only four specimens were ever recorded as carrying the inverted orientation of 2R<em>c</em> unaccompanied by either 2R<em>b</em> or 2R<em>u</em> (V. Petrarca, unpublished data). A second source, not mutually exclusive, was population structure between <em>An. coluzzii</em>, <em>An. gambiae</em>, and the BAMAKO chromosomal form that is subsumed taxonomically within <em>An. gambiae</em> but is at least partially reproductively isolated and genetically differentiated (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Manoukis <em>et al.</em> 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Love <em>et al.</em> 2016</a>). Although 2R<em>c</em> occurs in all three taxa, there is a strong karyotype imbalance among them in natural populations and in Ag1000G. For example, of 70 <em>An. coluzzii</em> with 2R<em>c</em> in Ag1000G, at least 49 (70%) carried the 2R<em>bc</em> haplotype (haplotypes of the other specimens could not be inferred unambiguously). Similarly, of 64 non-BAMAKO <em>An. gambiae</em> with 2R<em>c</em>, 62 (97%) carried the 2R<em>bc</em> haplotype. On the other hand, all 45 BAMAKO, by definition, carried 2R<em>cu</em>. We initially partitioned our sample by species, but the inclusion of BAMAKO in the <em>An. gambiae</em> partition resulted in very few candidate tags concordant with inversion genotype (N = 17). Ultimately, we retained two data partitions (<em>An. coluzzii</em> and non-BAMAKO <em>An. gambiae</em>), eliminating a third BAMAKO partition due to the fixation of 2R<em>c</em> in this taxon (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Coluzzi <em>et al.</em> 1985</a>). From the non-BAMAKO <em>An. gambiae</em> partition (hereafter, <em>An. gambiae</em> for brevity), we omitted two of only three specimens carrying 2R<em>u</em> (AZ0267-C from Mali and AV0043 C from Guinea), guided by PCA. As described above, both data partitions were split into training (75%) and validation (25%) sets, and ten bootstrap replicates of each training set were analyzed.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494402/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Candidate tag SNPs predictive of inversion genotype in Ag1000G data</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">Ultimately, the candidate tag SNPs chosen (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) met the following three criteria: they were (i) analyzed in at least eight of the ten bootstrap replicates; (ii) called at a rate greater than 90% within each karyotype class; and (iii) concordant with karyotype more than 80% of the time within each karyotype class (99.5% for 2L<em>a</em>). Their approximate physical position relative to the span of each inversion is illustrated in Figure S1.</p></div><div id="sec-5" class="subsection"><h3>Validation of candidate tag SNPs in Ag1000G</h3><p id="p-24">We interrogated the candidate tag SNPs in the validation samples from Ag1000G that had been held aside during the discovery phase (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). For each mosquito in the validation set, we masked genotypes inside the focal inversion with GQ scores less than 20. Next, among the retained SNPs, we identified those corresponding to candidate tags and converted their diploid genotypes to a count of the number of alternate alleles. Finally, the number of alternate alleles at each tag SNP was summed across tags and averaged to provide an overall computational karyotype score. We compared this mean score to the PCA-based karyotype.</p></div><div id="sec-6" class="subsection"><h3>Testing tag SNPs in data independent of the Ag1000G pipeline</h3><p id="p-25">We also explored the efficacy of our tag SNPs for computational karyotyping in wild-caught mosquitoes subject to whole genome sequencing and variant calling by individual investigators, for which corresponding cytological karyotypes had been determined through phase microscopy (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). We used specimens originating from southern Mali, 8 <em>An. gambiae</em> BAMAKO chromosomal form (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Fontaine <em>et al.</em> 2015</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Love <em>et al.</em> 2016</a>) and 17 <em>An. coluzzii</em> (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Main <em>et al.</em> 2015</a>), whose variant calls and cytogenetic metadata are publicly accessible (Tables S5, S6). These data include specimens sequenced to much lower coverage than the standard adhered to by Ag1000G. We followed the same procedure described for the Ag1000G validation set to computationally karyotype these specimens, and compared their computational and cytologically determined karyotypes.</p></div><div id="sec-7" class="subsection"><h3>Genetic distance trees to assess inversion history</h3><p id="p-26">We compared patterns of relatedness near the breakpoints of all six inversions using unrooted neighbor-joining (NJ) trees. For each inversion, we used biallelic SNPs with a MAF of 0.01 found within 5 kb upstream and downstream of the distal and proximal breakpoints (15 kb for 2R<em>d</em>). Total numbers of SNPs for each inversion were: 2L<em>a</em>, 596; 2R<em>j</em>, 909; 2R<em>b</em>, 428; 2R<em>c</em>, 2141; 2R<em>d</em>, 955; 2R<em>u</em>, 1110. Using the python package <em>anhima</em>, we converted the number of alternate alleles at these SNPs into a Euclidean distance matrix, and then constructed neighbor-joining trees using all 1,347 specimens. To assess support for the nodes of the 2R<em>c</em> tree, we used the transfer bootstrap estimate (TBE; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lemoine <em>et al.</em> 2018</a>), a statistic that measures the number of taxa that must be transferred to make a given branch of a reference tree match the closest equivalent branch in a bootstrap tree. To calculate this statistic, we imported the matrix of alternate allele counts into R (v. 3.5.1, “Feather Spray”; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">R Core Team 2018</a>) and used the dist() function of base R to construct the Euclidean distance matrix. We then used the nj() function in the ape package (v. 5.2) to construct the neighbor joining tree, and the boot.phylo() function to generate 1,000 bootstrap trees. We used these trees as input to booster (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Lemoine <em>et al.</em> 2018</a>), which calculates the TBE for each node.</p></div><div id="sec-8" class="subsection"><h3>Code and data availability</h3><p id="p-27">All genomic sequence data and variant call files used in this study are located in open data repositories as specified in Tables S1 and S2. Phases 1 and 2 of Ag1000G are available from <a href="https://www.malariagen.net/data">https://www.malariagen.net/data</a>; variant calls and metadata for Ag1000G phase 3 and Vector Observatory data are available from <a href="https://figshare.com/projects/Data_for_In_silico_karyotyping_of_chromosomally_polymorphic_malaria_mosquitoes_in_the_Anopheles_gambiae_complex_/65522">https://figshare.com/projects/Data_for_In_silico_karyotyping_of_chromosomally_polymorphic_malaria_mosquitoes_in_the_Anopheles_gambiae_complex_/65522</a> under the Ag1000G terms of use: <a href="https://www.malariagen.net/data/terms-use/ag1000g-terms-use">https://www.malariagen.net/data/terms-use/ag1000g-terms-use</a>.</p><p id="p-28">The <em>An. gambiae</em> AgamP4 reference assembly is available through VectorBase (<a href="https://www.vectorbase.org">https://www.vectorbase.org</a>). All custom code necessary to reproduce this analysis can be found at <a href="https://github.com/rrlove/comp_karyo_notebooks">https://github.com/rrlove/comp_karyo_notebooks</a> and <a href="https://github.com/rrlove/ingenos">https://github.com/rrlove/ingenos</a>. Software packages used, and their versions, are summarized in Table S7. The complete set of tag SNPs, together with a custom script for computational karyotyping, which calculates the mean inversion genotype across the relevant tag SNPs, can be found at <a href="https://github.com/rrlove/compkaryo">https://github.com/rrlove/compkaryo</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.9159479">https://doi.org/10.25387/g3.9159479</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><p id="p-29">After filtering, we retained the genotype data from 1,347 individually sequenced <em>An. coluzzii</em> and <em>An. gambiae</em> mosquitoes from the Ag1000G repository of natural genomic sequence variation, representing population samples from 13 West, Central, and East African countries and the island of Mayotte (Tables S1, S2).</p><div id="sec-10" class="subsection"><h3>Patterns of genetic variation at inversion breakpoints</h3><p id="p-30">To gain insight into the relative roles of inversion history, taxonomic status, and geographic location in structuring genetic variation for each inversion, we reconstructed neighbor-joining trees based on SNPs in the immediate vicinity of the breakpoints (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The resulting dendrograms, color-coded by inversion genotype, taxon and African country, indicate little clustering on the basis of geographic location; outlier population samples are those with a history of inbreeding or hybridization (see Methods). On the other hand, with the notable exception of 2L<em>a</em>, taxonomic status is an important factor structuring inversion variation between <em>An. gambiae</em> and <em>An. coluzzii</em>. Moreover, BAMAKO specimens appear to comprise a differentiated outlier clade within the larger <em>An. gambiae</em> cluster. It is interesting to note that for inversion 2R<em>c</em>, taxonomic status appears to be a more decisive factor than inversion genotype. All three 2R<em>c</em> inversion genotypes cluster within their respective species (supported by bootstrap at 90%, or 98% if dendrograms are constructed after removing outlier samples from The Gambia, Guinea-Bissau and Kenya; not shown). Further investigation is required to determine whether this pattern results from a monophyletic inversion that subsequently differentiated between taxa, or from independent inversion events in the two taxa.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining dendrograms reconstructed from 1,347 An. gambiae and An. coluzzii mosquitoes from Ag1000G, using biallelic SNPs within 5 kb of inversion breakpoints (15 kb for 2Rd) having a minimum minor allele frequency of 0.01. Columns represent the same inversion dendrogram, alternately color-coded by inversion genotype as determined from PCA (first row), taxon (second row), or geographic source (third row). Some specimens that could not be karyotyped by PCA for inversions 2Rc, 2Rd, 2Rj, and 2Ru had cytogenetically determined karyotypes, which were used in place of PCA for color-coding the inversion genotype. &#x2018;None&#x2019; refers to mosquitoes that were not assigned an inversion genotype either by PCA or cytogenetically; &#x2018;Other&#x2019; refers to mosquitoes that were not identified taxonomically. Countries: AGO, Angola; BFA, Burkina Faso; CIV, Cote d&#x2019;Ivoire; CMR, Cameroon; GAB, Gabon; GHA, Ghana; GIN, Guinea; GMB, The Gambia; GNB, Guinea-Bissau; GNQ, Equatorial Guinea; KEN, Kenya; MLI, Mali; FRA, France (Mayotte Archipelago); UGA, Uganda." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-41985999" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Neighbor-joining dendrograms reconstructed from 1,347 &lt;em&gt;An. gambiae&lt;/em&gt; and &lt;em&gt;An. coluzzii&lt;/em&gt; mosquitoes from Ag1000G, using biallelic SNPs within 5 kb of inversion breakpoints (15 kb for 2R&lt;em&gt;d&lt;/em&gt;) having a minimum minor allele frequency of 0.01. Columns represent the same inversion dendrogram, alternately color-coded by inversion genotype as determined from PCA (first row), taxon (second row), or geographic source (third row). Some specimens that could not be karyotyped by PCA for inversions 2R&lt;em&gt;c&lt;/em&gt;, 2R&lt;em&gt;d&lt;/em&gt;, 2R&lt;em&gt;j&lt;/em&gt;, and 2R&lt;em&gt;u&lt;/em&gt; had cytogenetically determined karyotypes, which were used in place of PCA for color-coding the inversion genotype. &#x2018;None&#x2019; refers to mosquitoes that were not assigned an inversion genotype either by PCA or cytogenetically; &#x2018;Other&#x2019; refers to mosquitoes that were not identified taxonomically. Countries: AGO, Angola; BFA, Burkina Faso; CIV, Cote d&#x2019;Ivoire; CMR, Cameroon; GAB, Gabon; GHA, Ghana; GIN, Guinea; GMB, The Gambia; GNB, Guinea-Bissau; GNQ, Equatorial Guinea; KEN, Kenya; MLI, Mali; FRA, France (Mayotte Archipelago); UGA, Uganda.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3249/F4.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3249/F4.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494464" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Neighbor-joining dendrograms reconstructed from 1,347 <em>An. gambiae</em> and <em>An. coluzzii</em> mosquitoes from Ag1000G, using biallelic SNPs within 5 kb of inversion breakpoints (15 kb for 2R<em>d</em>) having a minimum minor allele frequency of 0.01. Columns represent the same inversion dendrogram, alternately color-coded by inversion genotype as determined from PCA (first row), taxon (second row), or geographic source (third row). Some specimens that could not be karyotyped by PCA for inversions 2R<em>c</em>, 2R<em>d</em>, 2R<em>j</em>, and 2R<em>u</em> had cytogenetically determined karyotypes, which were used in place of PCA for color-coding the inversion genotype. ‘None’ refers to mosquitoes that were not assigned an inversion genotype either by PCA or cytogenetically; ‘Other’ refers to mosquitoes that were not identified taxonomically. Countries: AGO, Angola; BFA, Burkina Faso; CIV, Cote d’Ivoire; CMR, Cameroon; GAB, Gabon; GHA, Ghana; GIN, Guinea; GMB, The Gambia; GNB, Guinea-Bissau; GNQ, Equatorial Guinea; KEN, Kenya; MLI, Mali; FRA, France (Mayotte Archipelago); UGA, Uganda.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Inversion karyotype imputation by PCA</h3><p id="p-32">Only 373 of the 1,347 mosquitoes were associated with metadata that included cytologically determined inversion karyotypes. As discovery of candidate tag SNPs requires provisional inversion genotype assignments, we applied local PCA to assign genotypes for individual inversions on chromosome 2, following <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Ma and Amos (2012)</a>. A recognized limitation to this population-level approach, beyond the fact that it cannot be applied to individual mosquitoes, is that its success depends upon the presence of all three inversion genotypes in the sample under study. For this reason, and with the goal of finding the most flexible solution to inversion genotyping across geography and taxa, we began with PCA based on the entire set of 1,347 mosquitoes, under the simplifying assumption that the expected ‘three-stripe’ signal on a PCA plot would not be overwhelmed by geographic or population structure. Only in the case of 2L<em>a</em> could genotype assignments be confidently inferred from the combination of all 1,347 specimens. For inversions on 2R, from one to four admixed (<em>An. gambiae-An. coluzzii</em>) or geographic outlier populations (highly inbred or island samples) had to be removed from analysis before a three-genotype pattern could be discerned on the PCA plot (Tables S2, S4; see Methods). Additionally, for 2R<em>d</em> and 2R<em>j</em>, <em>An. gambiae-An. coluzzii</em> population structure dominated the PCA. Taken together with the fact that 2R<em>j</em> has yet to be found in <em>An. coluzzii</em> (<a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> 2002</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">della Torre <em>et al.</em> 2005</a>), we removed all 341 <em>An. coluzzii</em> specimens (Tables S2, S4) prior to PCA karyotyping of 2R<em>d</em> and 2R<em>j</em> in <em>An. gambiae</em>. Ultimately, PCA karyotypes were imputed for 780-1,347 mosquitoes, depending upon the inversion (Table S4).</p></div><div id="sec-12" class="subsection"><h3>Tag SNP ascertainment and validation in Ag1000G</h3><p id="p-33">Dividing the Ag1000G samples into training (75%) and validation (25%) sets for each inversion, and working within the training sets using a bootstrapping procedure, we screened for candidate tag SNPs in the five 2R inversions and 2L<em>a</em> (see Methods for details). Candidate tag SNPs were those whose genotypes were concordant with the corresponding PCA genotypes, when averaged across ten bootstrap replicates, for more than 80% of the specimens that were scored (99.5% for 2L<em>a</em>). The number of candidate tags ranged from 99 (2R<em>j</em>) to 349 (2R<em>b</em>) excluding 2R<em>c</em>, in which only two candidates were found due to population structure between <em>An. gambiae</em> and <em>An. coluzzii</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Partitioning the 2R<em>c</em> sample by taxon (and omitting BAMAKO; see Methods) resulted in 49 and 57 tags for <em>An. gambiae</em> and <em>An. coluzzii</em>, respectively (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Notably, there was no overlap between the two sets of tags.</p><p id="p-34">To assess the performance of these candidate tags, we used them to predict karyotypes in the held-out validation sets of Ag1000G specimens. For each inversion and specimen, we calculated a computational karyotype score representing the average genotype inferred across all candidate tag SNPs ascertained (see Methods). Histograms of resulting computational karyotype scores generally showed tight clustering around the three theoretical genotypic optima (0, 1, 2), reflecting close agreement among specimens (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). For each specimen in a validation set, we then compared the computational karyotype score to its PCA karyotype, and tallied the number of disagreements (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). All except one specimen had matching PCA and computational karyotype scores. This exception, one of 254 (0.4%) assignments for 2R<em>c</em> in <em>An. gambiae</em>, involved a specimen carrying 2R<em>u</em> (AZ0267-C) already noted as an outlier (see Methods).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histograms of computational karyotyping scores calculated by interrogating tag SNPs in An. gambiae and An. coluzzii mosquitoes from the Ag1000G validation sets. Mean scores cluster around 0, 1, and 2. Note differences in Y-axis scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-41985999" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histograms of computational karyotyping scores calculated by interrogating tag SNPs in &lt;em&gt;An. gambiae&lt;/em&gt; and &lt;em&gt;An. coluzzii&lt;/em&gt; mosquitoes from the Ag1000G validation sets. Mean scores cluster around 0, 1, and 2. Note differences in Y-axis scale.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3249/F5.medium.gif" width="259" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3249/F5.medium.gif" width="259" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Histograms of computational karyotyping scores calculated by interrogating tag SNPs in <em>An. gambiae</em> and <em>An. coluzzii</em> mosquitoes from the Ag1000G validation sets. Mean scores cluster around 0, 1, and 2. Note differences in Y-axis scale.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494345/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494345/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mismatches between PCA and computational karyotypes in the Ag1000G validation sets</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Performance of tag SNPs in resequencing data independent of Ag1000G</h3><p id="p-37">Previous studies re-sequenced <em>An. gambiae</em> or <em>An. coluzzii</em> mosquitoes from Mali whose karyotypes had been determined from the polytene chromosome banding pattern (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Main <em>et al.</em> 2015</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Love <em>et al.</em> 2016</a>). Although sample size is limited, these data allow a direct comparison of cytogenetic and <em>in silico</em> karyotyping under less ideal conditions—lower sequencing depth, with variant calls made independently of the Ag1000G pipeline. For each specimen and inversion, we calculated computational karyotype scores (averaged across all tag SNPs that could be ascertained in a specimen) (Tables S5, S6). Histograms of these scores by inversion, similar to those based on Ag1000G validation sets, reveal clustering of scores around the three genotypic optima provided that taxon-specific tags (2R<em>c-coluzzii</em> and 2R<em>c-gambiae</em>) are applied to the conspecific taxon, and heterospecific applications (including use of 2R<em>c-gambiae</em> tags to genotype BAMAKO) are avoided (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, Figure S2).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histograms of computational karyotyping scores calculated by interrogating tag SNPs in An. gambiae and An. coluzzii mosquitoes re-sequenced independently of the Ag1000G pipeline, often at lower sequencing depth. Scores cluster near 0, 1, and 2 with little dispersion except when taxon-specific tag SNPs are applied to a different taxon (indicated by an asterisk)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-41985999" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histograms of computational karyotyping scores calculated by interrogating tag SNPs in &lt;em&gt;An. gambiae&lt;/em&gt; and &lt;em&gt;An. coluzzii&lt;/em&gt; mosquitoes re-sequenced independently of the Ag1000G pipeline, often at lower sequencing depth. Scores cluster near 0, 1, and 2 with little dispersion except when taxon-specific tag SNPs are applied to a different taxon (indicated by an asterisk).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3249/F6.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/10/3249/F6.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494472" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-38" class="first-child">Histograms of computational karyotyping scores calculated by interrogating tag SNPs in <em>An. gambiae</em> and <em>An. coluzzii</em> mosquitoes re-sequenced independently of the Ag1000G pipeline, often at lower sequencing depth. Scores cluster near 0, 1, and 2 with little dispersion except when taxon-specific tag SNPs are applied to a different taxon (indicated by an asterisk).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">In the BAMAKO sample of <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Love <em>et al.</em> (2016)</a> where mean sequencing depth ranged from 9-10x, there was concordance in karyotype assignments between cytogenetic and computational methods for five inversions including 2L<em>a</em>, even though only 10-12 2L<em>a</em> tags were ascertained (Tables S5, S6). However, as expected for BAMAKO, the <em>An. gambiae</em> 2R<em>c</em> tags failed. Due to the extreme geospatial restriction of BAMAKO, this specific problem is limited in scope.</p><p id="p-40">In the <em>An. coluzzii</em> sample of <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Main <em>et al.</em> (2015)</a>, mean sequencing coverage varied widely (4-66x; Table S5). The impact of very low sequencing coverage on the success of computational karyotyping is illustrated by specimens 04SEL021 and 04SEL02 (4x and 5x, respectively). For 04SEL021, there is no apparent disagreement between the cytogenetic and mean computational genotype scores for any of the six inversions. Nevertheless, for those inversions classified as heterozygotes both cytogenetically and computationally (2L<em>a</em>, 2R<em>b</em>, 2R<em>c</em>), the proportion of tags whose genotype matches the mean computational score drops drastically to ∼30% (Table S5), likely because true heterozygous sites are often scored as homozygous either for the reference or alternate allele (0 or 2) due to low sequencing coverage. (Indeed, using chromosome 3L, we confirmed the expected drop in the rate of heterozygosity with decreasing coverage in these 17 specimens; data not shown). Low coverage alone is less likely to bias computational scores toward zero or two. For 04SEL02 (5x coverage), where cytogenetic <em>vs.</em> computational discrepancies occur at 2R<em>b</em> and 2R<em>u</em>, the computational karyotype is supported respectively by 81% of 208 tags and &gt;94% of 57 tags, favoring the computational genotype by weight of evidence. The remaining six specimens with discordant inversion genotypes were sequenced to at least 10x coverage. In these cases, when the computational genotype score signaled ‘1’ in contradiction to a homokaryotypic cytogenetic genotype (02SEL85, 02SEL006, 02SEL009, 01Osel134), the proportion of tags supporting the computational genotype ranged from 65 to &gt;92%. For other types of genotypic disagreements between methods, the computational score was supported by &gt;80% of tags scored. Overall, these results suggest that computational karyotyping using tag SNPs can be successful in data derived independently of Ag1000G (Tables S5, S6), though care should be taken when this approach is applied to very low coverage samples.</p></div><div id="sec-14" class="subsection"><h3>Performance of tag SNPs against cytogenetically karyotyped Ag1000G specimens</h3><p id="p-41">We compared the cytogenetic karyotype assignments for 373 specimens in Ag1000G to their corresponding computational karyotype assignments (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Conflicts were few overall, and for every inversion, all but one conflict (involving specimen AZ0267-C, the exceptional <em>An. gambiae</em> carrier of the 2R<em>u</em> inversion) could be attributed to errors in the cytogenetically assigned scores, as genotypes imputed from both PCA and tag SNPs contradict the cytogenetic assignment. Visual reference back to the PCA plots clearly confirmed that for specimens whose cytogenetic and tag SNP assignments differed and for whom PCA karyotypes could be determined, their locations on the plot strongly agreed with the tag SNP genotype (Figure S3). Considering that we ascertained tens or hundreds of tags per specimen, and that the proportion of tags whose SNP genotype matched the computational score was greater than 83% in all except the unusual specimen AZ0267-C (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), the computational scores more confidently predict the true inversion genotype than traditional cytogenetics for these five inversions. The most dramatic example is with respect to inversion 2R<em>u</em>, where we noted an unusually large number of erroneous cytogenetic genotypes of ‘1’ (N = 18/29) conflicting with both PCA and computational assignments of ‘0’. It is not immediately clear what could lead to such an elevated rate of cytogenetic error (which otherwise is ∼4%), but it is possible that the 2R<em>u</em> heterozygous loop was mistaken either for a loop created by a rare inversion (sensu <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Pombi <em>et al.</em> 2008</a>) in the same chromosomal region, or for a 2R<em>d</em> loop in samples from regions where 2R<em>u</em> is rare (as supported by the fact all 11 cytogenetic errors in <em>An. gambiae</em> were found in samples from the same small region in Cameroon, six of which were scored computationally as ‘1’ for 2R<em>d</em>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494393/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494393/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Discrepancies between cytogenetic and computational karyotypes in Ag1000G mosquitoes analyzed</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">Our results also highlight the pitfalls of using taxon-specific tags to genotype other taxa, or populations with high levels of admixture between taxa (Table S8; Figure S4 summarizes the correct use of taxon-specific tags). As expected, we find elevated numbers of cytogenetic-computational disagreements when (i) 2R<em>c-gambiae</em> tags are applied to BAMAKO (60% of the 45 specimens), (ii) 2R<em>d-gambiae</em> tags are used to genotype <em>An. coluzzii</em>, and (iii) 2R<em>d-gambiae</em> tags are applied to admixed <em>An. gambiae-An. coluzzii</em> populations such as those from Guinea Bissau. These disagreements involve specimens carrying inverted arrangements according to cytogenetic analysis which are not tagged as inverted computationally, due to the lack of correlation between tags and the inverted orientation in the heterospecific genetic background.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-46">In recent years, numerous approaches have been developed for the discovery and detection of chromosomal inversions aside from traditional cytogenetics, including optical mapping, Hi-C, read cloud sequencing, and identification of discordantly mapping sequencing reads. Our method is not intended to substitute for these largely discovery-based approaches. Here, we are concerned with specific inversions that segregate at relatively high frequencies in natural populations, and have a long history of cytogenetic study. Our approach benefits from the fact that the chromosomal coordinates of their breakpoints already are known precisely or to a good approximation. The goal was to overcome the substantial limitations of cytogenetic analysis by identifying SNPs inside these rearrangements whose allelic state is predictive of inversion status, allowing for rapid <em>in silico</em> or molecular karyotyping of hundreds or thousands of specimens.</p><p id="p-47">Analysis of the Ag1000G database allowed us to develop the first standardized computational karyotyping of the six main polymorphic chromosomal inversions in the major malaria vectors <em>An. coluzzii</em> and <em>An. gambiae</em>, despite the fact that only a small subset of specimens in the database had cytogenetic karyotype assignments (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Direct comparison of computational karyotype scores with the cytogenetic assignment for the same specimen in Ag1000G suggests that computational karyotyping outperforms traditional cytogenetics in terms of accuracy, given that assignments are based on tens or hundreds of individual tags. Preliminary testing on specimens sequenced and computationally processed by individual laboratories outside of Ag1000G standards suggests that our tag SNPs have the potential to perform well, even on specimens sequenced to much lower depth. Our approach not only has a lower error rate compared to classical cytogenetics, but also is more widely applicable (regardless of mosquito gender, physiological status, or method of preservation), more widely accessible to those without specialized expertise, higher throughput, and therefore, ultimately cheaper to implement at scale. This method can now be used to predict inversion genotypes in previously sequenced data sets for which ecological and behavioral data may already be available. Even more important, easy large-scale adoption of this approach allows for new and properly powered association studies to be performed on ecologically and epidemiologically relevant mosquito phenotypes, studies that that would have been prohibitively ambitious when relying on cytogenetic karyotyping. In addition, this method can now facilitate sequencing experiments for which inversion karyotype is relevant at scale. Expanding the possibilities further, molecular assays based on these results that will allow inversion genotyping without whole genome sequencing are under development.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of the study area with the frequency of An. gambiae and An. coluzzii inversion genotypes inferred for up to six polymorphic chromosome 2 inversions summarized by country (and the island of Mayotte; see Table S2). Blue connecting lines point to An. coluzzii samples, while white connecting lines point to An. gambiae and hybrid/outlier populations. Image: Visible Earth, NASA. Produced with cartopy v.0.17.1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-41985999" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Map of the study area with the frequency of &lt;em&gt;An. gambiae&lt;/em&gt; and &lt;em&gt;An. coluzzii&lt;/em&gt; inversion genotypes inferred for up to six polymorphic chromosome 2 inversions summarized by country (and the island of Mayotte; see Table S2). Blue connecting lines point to &lt;em&gt;An. coluzzii&lt;/em&gt; samples, while white connecting lines point to &lt;em&gt;An. gambiae&lt;/em&gt; and hybrid/outlier populations. Image: Visible Earth, NASA. Produced with cartopy v.0.17.1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3249/F7.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/10/3249/F7.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3249/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494430" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-48" class="first-child">Map of the study area with the frequency of <em>An. gambiae</em> and <em>An. coluzzii</em> inversion genotypes inferred for up to six polymorphic chromosome 2 inversions summarized by country (and the island of Mayotte; see Table S2). Blue connecting lines point to <em>An. coluzzii</em> samples, while white connecting lines point to <em>An. gambiae</em> and hybrid/outlier populations. Image: Visible Earth, NASA. Produced with cartopy v.0.17.1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">However, some important limitations exist. Computational karyotyping is strictly dependent upon tag SNPs that are strongly correlated with inversion status, a contingency that depends upon representative sampling. Although Ag1000G is populated by samples derived from multiple countries in West, Central and East Africa, <em>An. coluzzii</em> is underrepresented, as is southern Africa (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">The <em>Anopheles gambiae</em> 1000 Genomes Consortium 2017</a>). Even more importantly, with the exception of the cosmopolitan inversions 2L<em>a</em> and 2R<em>b</em>, the inverted orientation of other rearrangements (2R<em>j</em>, 2R<em>c</em>, 2R<em>d</em>, and 2R<em>u</em>) is underrepresented in the Ag1000G data that was available at the time of our analysis. It is clear that population structure is an especially important factor in the application and further development of tags for 2R<em>c</em> and 2R<em>d</em>. The current taxon-specific tags should not be used to genotype heterospecific specimens (including BAMAKO) or samples from areas where high rates of interspecific hybridization are known. The presence of strong population structure means that correlations between tags and the inverted orientation characteristic of the target taxon cannot be assumed in a different taxon. The absence of correlation should downwardly bias the computational score, resulting in false negatives when genotyping true inverted homozygotes and heterozygotes. Finally, our inversion breakpoint dendrograms raise the possibility that at least one cytologically-recognized inversion, 2R<em>c</em>, may have arisen repeatedly at the molecular level, a result that requires further investigation beyond the scope of this study. With the exception of 2R<em>c</em>, 2R<em>d</em>, and 2R<em>j</em>, for which we developed taxon-specific tags, our approach implicitly assumed that inversions shared by <em>An. gambiae</em> and <em>An. coluzzii</em> are monophyletic, and may yield unexpected results if this assumption is violated. Accordingly, these tools should be applied with caution, and there is ample room for improvement as more data become available. Fortunately, our standardized approach makes it easy to accommodate improvements. The success of our method thus far suggests that the general approach may be suitable for studying inversions more broadly, in additional malaria vectors as well as other systems where inversions are implicated in local adaptation.</p><p id="p-50">Nearly twenty years ago, Coluzzi and colleagues predicted that the then-newly-assembled <em>An. gambiae</em> reference genome would facilitate our analyses of polymorphic chromosomal inversions in the <em>An. gambiae</em> complex (<a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Coluzzi <em>et al.</em> 2002</a>). Our work continues the realization of that prediction by providing, for the first time, cross-continent diagnostics for multiple inversions. These computational diagnostics, and the molecular diagnostics that they leverage, take us one step closer to understanding the contribution of chromosomal inversions to the deadly facility of <em>An. gambiae</em> and <em>An. coluzzii</em> for vectoring malaria.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank the Notre Dame Center for Research Computing for technical support, and C. Liu, C. Sweet, and J. Young for helpful discussions. We thank M. Kern and R. Montañez-Gonzalez for assistance with DNA extraction. This work was supported by the National Institutes of Health (R01 AI125360 awarded to NJB). During this work, NJB was supported by Target Malaria, which receives core funding from the Bill &amp; Melinda Gates Foundation and from the Open Philanthropy Project Fund, an advised fund of Silicon Valley Community Foundation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">2</span> <a href="https://www.malariagen.net/projects/ag1000g#people">https://www.malariagen.net/projects/ag1000g#people</a>.</p></li><li class="fn" id="fn-7"><p id="p-52">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.9159479">https://doi.org/10.25387/g3.9159479</a>.</p></li><li class="fn" id="fn-8"><p id="p-53"><em>Communicating editor: R. Anholt</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-13"><span class="received-label">Received </span>June 13, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-30"><span class="accepted-label">Accepted </span>July 30, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Love <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.1" data-doi="10.1017/S0016672301004955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Depaulis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Navarro</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Inversion polymorphisms and nucleotide variability in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672301004955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D8%26rft.atitle%253DInversion%2Bpolymorphisms%2Band%2Bnucleotide%2Bvariability%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672301004955%26rft_id%253Dinfo%253Apmid%252F11279826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672301004955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11279826&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168340200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.2" data-doi="10.1038/nature24995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Anopheles gambiae 1000 Genomes Consortium</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic diversity of the African malaria vector <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">552</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature24995</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D552%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24995%26rft_id%253Dinfo%253Apmid%252F29186111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29186111&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.3" data-doi="10.1111/evo.13176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Acevedo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pombi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Dia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Boccolini</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Chromosome inversions and ecological plasticity in the main African malaria mosquitoes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">701</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.13176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D71%26rft.spage%253D686%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.13176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.13176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.4" data-doi="10.3389/fgene.2014.00129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ullastres</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gonzalez</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Adaptation through chromosomal inversions in Anopheles.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00129</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00129%26rft_id%253Dinfo%253Apmid%252F24904633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24904633&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.5" data-doi="10.1111/mec.14880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chateau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fouet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Morlais</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Association mapping desiccation resistance within chromosomal inversions in the African malaria vector Anopheles gambiae.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1333</span>–<span class="cit-lpage">1342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D28%26rft.spage%253D1333%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.6" data-doi="10.1007/BF00056539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>,  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Di Deco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Petrarca</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coluzzi</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Inversion polymorphism and incipient speciation in <em>Anopheles gambiae s. str</em>. in The Gambia, West Africa.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00056539</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D59%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00056539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00056539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982PX49900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.7" data-doi="10.1111/mec.12866"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Nwakanma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. P.</span> <span class="cit-name-surname">Caputo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jawara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Oriero</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prominent intra-specific genetic divergence within <em>Anopheles gambiae</em> sibling species triggered by habitat discontinuities across a riverine landscape.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4574</span>–<span class="cit-lpage">4589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12866</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4574%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12866&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.8" data-doi="10.1186/1475-2875-7-182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Nwakanma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jawara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Adiamoh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Dia</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Anopheles gambiae</em> complex along The Gambia river, with particular reference to the molecular forms of <em>An. gambiae s.s</em>.</span> <abbr class="cit-jnl-abbrev">Malar. J.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">182</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1475-2875-7-182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalaria%2Bjournal%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DMalar%2BJ%26rft.aulast%253DCaputo%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.spage%253D182%26rft.epage%253D182%26rft.atitle%253DAnopheles%2Bgambiae%2Bcomplex%2Balong%2BThe%2BGambia%2Briver%252C%2Bwith%2Bparticular%2Breference%2Bto%2Bthe%2Bmolecular%2Bforms%2Bof%2BAn.%2Bgambiae%2Bs.s.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1475-2875-7-182%26rft_id%253Dinfo%253Apmid%252F18803885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1475-2875-7-182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18803885&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.9" data-doi="10.1371/journal.pone.0016415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Santolamazza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Vicente</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Nwakanma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jawara</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The “far-west” of <em>Anopheles gambiae</em> molecular forms.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e16415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0016415</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCaputo%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De16415%26rft.epage%253De16415%26rft.atitle%253DThe%2B%2522far-west%2522%2Bof%2BAnopheles%2Bgambiae%2Bmolecular%2Bforms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016415%26rft_id%253Dinfo%253Apmid%252F21347223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21347223&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.10" data-doi="10.1111/j.1365-294X.2011.05114.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cassone</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Molloy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Tan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Divergent transcriptional response to thermal stress by <em>Anopheles gambiae</em> larvae carrying alternative arrangements of inversion 2La.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2567</span>–<span class="cit-lpage">2580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2011.05114.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCassone%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D20%26rft.issue%253D12%26rft.spage%253D2567%26rft.epage%253D2580%26rft.atitle%253DDivergent%2Btranscriptional%2Bresponse%2Bto%2Bthermal%2Bstress%2Bby%2BAnopheles%2Bgambiae%2Blarvae%2Bcarrying%2Balternative%2Barrangements%2Bof%2Binversion%2B2La.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05114.x%26rft_id%253Dinfo%253Apmid%252F21535279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05114.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21535279&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.11" data-doi="10.1073/pnas.1806760115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Tan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Besansky</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A systems genetic analysis of inversion polymorphisms in the malaria mosquito <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">E7005</span>–<span class="cit-lpage">E7014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1806760115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1806760115%26rft_id%253Dinfo%253Apmid%252F29987007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzMwL0U3MDA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.12" data-doi="10.1534/genetics.111.137794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kamdem</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Mockaitis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Costantini</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ecological genomics of <em>Anopheles gambiae</em> along a latitudinal cline: a population-resequencing approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1417</span>–<span class="cit-lpage">1432</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.137794</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137794%26rft_id%253Dinfo%253Apmid%252F22209907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.13" data-doi="10.11646/zootaxa.3619.3.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coetzee</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Hunt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wilkerson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">della Torre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Coulibaly</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Anopheles coluzzii</em> and <em>Anopheles amharicus</em>, new members of the <em>Anopheles gambiae</em> complex.</span> <abbr class="cit-jnl-abbrev">Zootaxa</abbr> <span class="cit-vol">3619</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.11646/zootaxa.3619.3.2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZootaxa%26rft.volume%253D3619%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.11646%252Fzootaxa.3619.3.2%26rft_id%253Dinfo%253Apmid%252F26131476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.11646/zootaxa.3619.3.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26131476&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315436200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coluzzi</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Cromosomi politenici delle cellule nutrici ovariche nel complesso gambiae del genere <em>Anopheles</em>.</span> <abbr class="cit-jnl-abbrev">Parassitologia</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParassitologia%26rft.volume%253D10%26rft.spage%253D179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.15" data-doi="10.1080/11250008509440343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coluzzi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Petrarca</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">DiDeco</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Chromosomal inversion intergradation and incipient speciation in <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Boll. Zool.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/11250008509440343</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBoll.%2BZool.%26rft.volume%253D52%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1080%252F11250008509440343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/11250008509440343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.16" data-doi="10.1126/science.1077769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coluzzi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sabatini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">della Torre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Di Deco</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Petrarca</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A polytene chromosome analysis of the <em>Anopheles gambiae</em> species complex.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">1415</span>–<span class="cit-lpage">1418</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1077769</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1077769%26rft_id%253Dinfo%253Apmid%252F12364623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTU5Ny8xNDE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.17" data-doi="10.1016/0035-9203(79)90036-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coluzzi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sabatini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Petrarca</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Di Deco</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Chromosomal differentiation and adaptation to human environments in the <em>Anopheles gambiae</em> complex.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Trop. Med. Hyg.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0035-9203(79)90036-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BTrop.%2BMed.%2BHyg.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0035-9203%252879%252990036-1%26rft_id%253Dinfo%253Apmid%252F394408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0035-9203(79)90036-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=394408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.18" data-doi="10.1101/662114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Said</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Calzetta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Genetti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">McBroome</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Fine-mapping complex inversion breakpoints and investigating somatic pairing in the <em>Anopheles gambiae</em> species complex using proximity-ligation sequencing.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/662114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F662114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI2NjIxMTR2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.19" data-doi="10.1186/1472-6785-9-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costantini</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ayala</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Guelbeogo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pombi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. Y.</span> <span class="cit-name-surname">Some</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Living at the edge: biogeographic patterns of habitat segregation conform to speciation by niche expansion in <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">BMC Ecol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1472-6785-9-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Becology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEcol%26rft.aulast%253DCostantini%26rft.auinit1%253DC.%26rft.volume%253D9%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DLiving%2Bat%2Bthe%2Bedge%253A%2Bbiogeographic%2Bpatterns%2Bof%2Bhabitat%2Bsegregation%2Bconform%2Bto%2Bspeciation%2Bby%2Bniche%2Bexpansion%2Bin%2BAnopheles%2Bgambiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6785-9-16%26rft_id%253Dinfo%253Apmid%252F19460144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1472-6785-9-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19460144&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.20" data-doi="10.1371/journal.pone.0000849"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coulibaly</span>,  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. F.</span> <span class="cit-name-surname">Lobo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Fitzpatrick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kern</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Grushko</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Segmental duplication implicated in the genesis of inversion 2Rj of <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e849</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0000849</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCoulibaly%26rft.auinit1%253DM.%2BB.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De849%26rft.epage%253De849%26rft.atitle%253DSegmental%2Bduplication%2Bimplicated%2Bin%2Bthe%2Bgenesis%2Bof%2Binversion%2B2Rj%2Bof%2BAnopheles%2Bgambiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000849%26rft_id%253Dinfo%253Apmid%252F17786220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000849&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17786220&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.21" data-doi="10.1186/1475-2875-6-133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coulibaly</span>,  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pombi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Caputo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Nwakanma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jawara</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">PCR-based karyotyping of <em>Anopheles gambiae</em> inversion 2Rj identifies the BAMAKO chromosomal form.</span> <abbr class="cit-jnl-abbrev">Malar. J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1475-2875-6-133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalaria%2Bjournal%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DMalar%2BJ%26rft.aulast%253DCoulibaly%26rft.auinit1%253DM.%2BB.%26rft.volume%253D6%26rft.spage%253D133%26rft.epage%253D133%26rft.atitle%253DPCR-based%2Bkaryotyping%2Bof%2BAnopheles%2Bgambiae%2Binversion%2B2Rj%2Bidentifies%2Bthe%2BBAMAKO%2Bchromosomal%2Bform.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1475-2875-6-133%26rft_id%253Dinfo%253Apmid%252F17908310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1475-2875-6-133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17908310&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.22" data-doi="10.1111/j.1365-2915.2012.01049.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dabire</span>,  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sawadodgo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Diabate</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Toe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kengne</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assortative mating in mixed swarms of the mosquito <em>Anopheles gambiae s.s</em>. M and S molecular forms, in Burkina Faso, West Africa.</span> <abbr class="cit-jnl-abbrev">Med. Vet. Entomol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">298</span>–<span class="cit-lpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2915.2012.01049.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMed.%2BVet.%2BEntomol.%26rft.volume%253D27%26rft.spage%253D298%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2915.2012.01049.x%26rft_id%253Dinfo%253Apmid%252F23360106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2915.2012.01049.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23360106&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3249.23" data-doi="10.1007/978-94-009-1535-0_28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">della Torre</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Polytene chromosome preparation from anopheline mosquitoes</span>, pp. <span class="cit-fpage">329</span>–<span class="cit-lpage">336</span> in <span class="cit-source">Molecular Biology of Disease Vectors: A Methods Manual</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">J. M.</span><span class="cit-name-surname">Crampton</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Beard</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Louis</span></span>. <span class="cit-publ-name">Chapman &amp; Hall</span>, <span class="cit-publ-loc">London</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-94-009-1535-0_28</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2BDisease%2BVectors%253A%2BA%2BMethods%2BManual%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-94-009-1535-0_28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-94-009-1535-0_28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.24" data-doi="10.1046/j.1365-2583.2001.00235.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">della Torre</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fanello</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Akogbeto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dossou-yovo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Favia</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular evidence of incipient speciation within <em>Anopheles gambiae s.s</em>. in West Africa.</span> <abbr class="cit-jnl-abbrev">Insect Mol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2583.2001.00235.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2Bmolecular%2Bbiology%26rft.stitle%253DInsect%2BMol%2BBiol%26rft.aulast%253Ddella%2BTorre%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D18%26rft.atitle%253DMolecular%2Bevidence%2Bof%2Bincipient%2Bspeciation%2Bwithin%2BAnopheles%2Bgambiae%2Bs.s.%2Bin%2BWest%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2583.2001.00235.x%26rft_id%253Dinfo%253Apmid%252F11240632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2583.2001.00235.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11240632&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167235900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.25" data-doi="10.1016/j.ibmb.2005.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">della Torre</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Tu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Petrarca</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">On the distribution and genetic differentiation of <em>Anopheles gambiae s.s</em>. molecular forms.</span> <abbr class="cit-jnl-abbrev">Insect Biochem. Mol. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ibmb.2005.02.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2Bbiochemistry%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DInsect%2BBiochem%2BMol%2BBiol%26rft.aulast%253Ddella%2BTorre%26rft.auinit1%253DA.%26rft.volume%253D35%26rft.issue%253D7%26rft.spage%253D755%26rft.epage%253D769%26rft.atitle%253DOn%2Bthe%2Bdistribution%2Band%2Bgenetic%2Bdifferentiation%2Bof%2BAnopheles%2Bgambiae%2Bs.s.%2Bmolecular%2Bforms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ibmb.2005.02.006%26rft_id%253Dinfo%253Apmid%252F15894192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ibmb.2005.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15894192&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229447000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.26" data-doi="10.1098/rspb.2009.1167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diabaté</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Dao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Yaro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Adamou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gonzalez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Spatial swarm segregation and reproductive isolation between the molecular forms of <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">4215</span>–<span class="cit-lpage">4222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2009.1167</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2009.1167%26rft_id%253Dinfo%253Apmid%252F19734189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2009.1167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19734189&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.27" data-doi="10.1126/science.1258524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fontaine</span>,  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Pease</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Steele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Neafsey</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extensive introgression in a malaria vector species complex revealed by phylogenomics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">347</span>: <span class="cit-elocation-id">1258524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1258524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1258524%26rft_id%253Dinfo%253Apmid%252F25431491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDcvNjIxNy8xMjU4NTI0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.28" data-doi="10.1371/journal.pone.0034841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fouet</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Gray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Besansky</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Costantini</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Adaptation to aridity in the malaria mosquito <em>Anopheles gambiae</em>: chromosomal inversion polymorphism and body size influence resistance to desiccation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e34841</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0034841</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFouet%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De34841%26rft.epage%253De34841%26rft.atitle%253DAdaptation%2Bto%2Baridity%2Bin%2Bthe%2Bmalaria%2Bmosquito%2BAnopheles%2Bgambiae%253A%2Bchromosomal%2Binversion%2Bpolymorphism%2Band%2Bbody%2Bsize%2Binfluence%2Bresistance%2Bto%2Bdesiccation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0034841%26rft_id%253Dinfo%253Apmid%252F22514674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0034841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22514674&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.29" data-doi="10.1111/mec.14381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>,  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. D.</span> <span class="cit-name-surname">Haynes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Richards</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. W.</span> <span class="cit-name-surname">Schaeffer</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomics of natural populations: Evolutionary forces that establish and maintain gene arrangements in <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">6539</span>–<span class="cit-lpage">6562</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14381</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D6539%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.30" data-doi="10.1093/beheco/arq114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gimonneau</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bouyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Morand</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Besansky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Diabate</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A behavioral mechanism underlying ecological divergence in the malaria mosquito <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Behav. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1087</span>–<span class="cit-lpage">1092</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/beheco/arq114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbeheco%252Farq114%26rft_id%253Dinfo%253Apmid%252F22476108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/beheco/arq114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22476108&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.31" data-doi="10.1111/j.1365-2915.2011.00957.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gimonneau</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pombi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Choisy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Morand</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Dabire</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Larval habitat segregation between the molecular forms of the mosquito, <em>Anopheles gambiae</em> in a rice field area of Burkina Faso, West Africa.</span> <abbr class="cit-jnl-abbrev">Med. Vet. Entomol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2915.2011.00957.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMedical%2Band%2Bveterinary%2Bentomology%26rft.stitle%253DMed%2BVet%2BEntomol%26rft.aulast%253DGimonneau%26rft.auinit1%253DG.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D17%26rft.atitle%253DLarval%2Bhabitat%2Bsegregation%2Bbetween%2Bthe%2Bmolecular%2Bforms%2Bof%2Bthe%2Bmosquito%2BAnopheles%2Bgambiae%2Bin%2Ba%2Brice%2Bfield%2Barea%2Bof%2BBurkina%2BFaso%252C%2BWest%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2915.2011.00957.x%26rft_id%253Dinfo%253Apmid%252F21501199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2915.2011.00957.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21501199&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.32" data-doi="10.1186/1756-3305-5-65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gimonneau</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pombi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Dabire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Diabate</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Morand</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Behavioural responses of Anopheles gambiae sensu stricto M and S molecular form larvae to an aquatic predator in Burkina Faso.</span> <abbr class="cit-jnl-abbrev">Parasit. Vectors</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1756-3305-5-65</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DParasit%2BVectors%26rft.aulast%253DGimonneau%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.spage%253D65%26rft.epage%253D65%26rft.atitle%253DBehavioural%2Bresponses%2Bof%2BAnopheles%2Bgambiae%2Bsensu%2Bstricto%2BM%2Band%2BS%2Bmolecular%2Bform%2Blarvae%2Bto%2Ban%2Baquatic%2Bpredator%2Bin%2BBurkina%2BFaso.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-3305-5-65%26rft_id%253Dinfo%253Apmid%252F22463735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-3305-5-65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22463735&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.33" data-doi="10.1093/nar/gku1117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraldo-Calderón</span>,  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Emrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">MacCallum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Maslen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Dialynas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">VectorBase: an updated bioinformatics resource for invertebrate vectors and other organisms related with human diseases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D707</span>–<span class="cit-lpage">D713</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1117%26rft_id%253Dinfo%253Apmid%252F25510499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25510499&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.34" data-doi="10.1186/1475-2875-8-215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>,  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Rocca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Costantini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Besansky</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inversion 2La is associated with enhanced desiccation resistance in <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Malar. J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1475-2875-8-215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalaria%2Bjournal%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DMalar%2BJ%26rft.aulast%253DGray%26rft.auinit1%253DE.%2BM.%26rft.volume%253D8%26rft.spage%253D215%26rft.epage%253D215%26rft.atitle%253DInversion%2B2La%2Bis%2Bassociated%2Bwith%2Benhanced%2Bdesiccation%2Bresistance%2Bin%2BAnopheles%2Bgambiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1475-2875-8-215%26rft_id%253Dinfo%253Apmid%252F19772577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1475-2875-8-215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19772577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.35" data-doi="10.1111/mec.14947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanemaaijer</span>,  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Collier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Shott</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Houston</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The fate of genes that cross species boundaries after a major hybridization event in a natural mosquito population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">4978</span>–<span class="cit-lpage">4990</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D27%26rft.spage%253D4978%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.36" data-doi="10.1146/annurev.ecolsys.39.110707.173532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. H.</span> <span class="cit-name-surname">Rieseberg</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Revisiting the impact of inversions in evolution: from population genetic markers to drivers of adaptive shifts and speciation?</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.39.110707.173532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnnu%2BRev%2BEcol%2BEvol%2BSyst%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D39%26rft.spage%253D21%26rft.epage%253D42%26rft.atitle%253DRevisiting%2Bthe%2BImpact%2Bof%2BInversions%2Bin%2BEvolution%253A%2BFrom%2BPopulation%2BGenetic%2BMarkers%2Bto%2BDrivers%2Bof%2BAdaptive%2BShifts%2Band%2BSpeciation%253F%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173532%26rft_id%253Dinfo%253Apmid%252F20419035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20419035&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261725500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.37" data-doi="10.1016/j.tree.2004.06.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Sgro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Weeks</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Chromosomal inversion polymorphisms and adaptation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">482</span>–<span class="cit-lpage">488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2004.06.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D19%26rft.issue%253D9%26rft.spage%253D482%26rft.epage%253D488%26rft.atitle%253DChromosomal%2Binversion%2Bpolymorphisms%2Band%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2004.06.013%26rft_id%253Dinfo%253Apmid%252F16701311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2004.06.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16701311&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223899600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.38" data-doi="10.1126/science.1076181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Subramanian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Halpern</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. G.</span> <span class="cit-name-surname">Sutton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Charlab</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The genome sequence of the malaria mosquito <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1076181</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHolt%26rft.auinit1%253DR.%2BA.%26rft.volume%253D298%26rft.issue%253D5591%26rft.spage%253D129%26rft.epage%253D149%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2Bthe%2BMalaria%2BMosquito%2BAnopheles%2Bgambiae%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1076181%26rft_id%253Dinfo%253Apmid%252F12364791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTgvNTU5MS8xMjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.39" data-doi="10.1534/g3.115.019554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Marquez</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Linkage disequilibrium and inversion-typing of the <em>Drosophila melanogaster</em> genome reference panel.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1695</span>–<span class="cit-lpage">1701</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.019554</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019554%26rft_id%253Dinfo%253Apmid%252F26068573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvOC8xNjk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.40" data-doi="10.1038/nature10944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Grabherr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. F.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Russell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Mauceli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.41" data-doi="10.1038/nature10341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joron</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Frezal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. T.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Chamberlain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Lee</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chromosomal rearrangements maintain a polymorphic supergene controlling butterfly mimicry.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJoron%26rft.auinit1%253DM.%26rft.volume%253D477%26rft.issue%253D7363%26rft.spage%253D203%26rft.epage%253D206%26rft.atitle%253DChromosomal%2Brearrangements%2Bmaintain%2Ba%2Bpolymorphic%2Bsupergene%2Bcontrolling%2Bbutterfly%2Bmimicry.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10341%26rft_id%253Dinfo%253Apmid%252F21841803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21841803&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294603900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.42" data-doi="10.1093/molbev/msw016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapun</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Fabian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Goudet</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Flatt</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic evidence for adaptive inversion clines in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1317</span>–<span class="cit-lpage">1336</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw016%26rft_id%253Dinfo%253Apmid%252F26796550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26796550&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.43" data-doi="10.1371/journal.pbio.1000501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How and why chromosome inversions evolve.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1000501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D8%26rft.spage%253De1000501%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000501%26rft_id%253Dinfo%253Apmid%252F20927412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927412&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.44" data-doi="10.1111/mec.13074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Barrett</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromosome inversions, adaptive cassettes and the evolution of species’ ranges.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2046</span>–<span class="cit-lpage">2055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D2046%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.45" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.047985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.46" data-doi="10.1038/s41586-018-0043-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemoine</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Domelevo Entfellner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Wilkinson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Correia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Davila Felipe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Renewing Felsenstein’s phylogenetic bootstrap in the era of big data.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">556</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41586-018-0043-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D556%26rft.spage%253D452%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0043-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0043-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.47" data-doi="10.1186/1475-2875-9-293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lobo</span>,  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Sangare</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Regier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Reidenbach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Bretz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Breakpoint structure of the Anopheles gambiae 2Rb chromosomal inversion.</span> <abbr class="cit-jnl-abbrev">Malar. J.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1475-2875-9-293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalaria%2Bjournal%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DMalar%2BJ%26rft.aulast%253DLobo%26rft.auinit1%253DN.%2BF.%26rft.volume%253D9%26rft.spage%253D293%26rft.epage%253D293%26rft.atitle%253DBreakpoint%2Bstructure%2Bof%2Bthe%2BAnopheles%2Bgambiae%2B2Rb%2Bchromosomal%2Binversion.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1475-2875-9-293%26rft_id%253Dinfo%253Apmid%252F20974007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1475-2875-9-293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20974007&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.48" data-doi="10.1111/mec.13888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Steele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Coulibaly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Traore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Emrich</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chromosomal inversions and ecotypic differentiation in Anopheles gambiae: the perspective from whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">5889</span>–<span class="cit-lpage">5906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13888</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D5889%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.49" data-doi="10.1371/journal.pbio.1000500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>,  <span class="cit-name-given-names">D. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Willis</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A widespread chromosomal inversion polymorphism contributes to a major life-history transition, local adaptation, and reproductive isolation.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1000500</span> (erratum: PLoS Biol. 10: 10.1371/annotation/caa1b7dd-9b6d-44db-b6ce-666954903625).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D8%26rft.spage%253De1000500%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000500%26rft_id%253Dinfo%253Apmid%252F20927411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927411&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.50" data-doi="10.1371/journal.pone.0040224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. I.</span> <span class="cit-name-surname">Amos</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Investigation of inversion polymorphisms in the human genome using principal components analysis.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e40224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0040224</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMa%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40224%26rft.epage%253De40224%26rft.atitle%253DInvestigation%2Bof%2Binversion%2Bpolymorphisms%2Bin%2Bthe%2Bhuman%2Bgenome%2Busing%2Bprincipal%2Bcomponents%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040224%26rft_id%253Dinfo%253Apmid%252F22808122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22808122&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.51" data-doi="10.1111/mec.13382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Main</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Collier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Norris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Brisco</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Complex genome evolution in Anopheles coluzzii associated with increased insecticide usage in Mali.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5145</span>–<span class="cit-lpage">5157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D5145%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13382%26rft_id%253Dinfo%253Apmid%252F26359110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26359110&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.52" data-doi="10.1073/pnas.0709806105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manoukis</span>,  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Powell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Touré</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sacko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. E.</span> <span class="cit-name-surname">Edillo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A test of the chromosomal theory of ecotypic speciation in <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">2940</span>–<span class="cit-lpage">2945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0709806105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0709806105%26rft_id%253Dinfo%253Apmid%252F18287019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzgvMjk0MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.53" data-doi="10.1111/j.1365-294X.2011.05339.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsden</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Neimen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sanford</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dinis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Asymmetric introgression between the M and S forms of the malaria vector, <em>Anopheles gambiae</em>, maintains divergence despite extensive hybridization.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4983</span>–<span class="cit-lpage">4994</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2011.05339.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMarsden%26rft.auinit1%253DC.%2BD.%26rft.volume%253D20%26rft.issue%253D23%26rft.spage%253D4983%26rft.epage%253D4994%26rft.atitle%253DAsymmetric%2Bintrogression%2Bbetween%2Bthe%2BM%2Band%2BS%2Bforms%2Bof%2Bthe%2Bmalaria%2Bvector%252C%2BAnopheles%2Bgambiae%252C%2Bmaintains%2Bdivergence%2Bdespite%2Bextensive%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05339.x%26rft_id%253Dinfo%253Apmid%252F22059383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05339.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059383&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297915700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3249.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miles</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Harding</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> scikit-allel: A Python package for exploring and analysing genetic variation data. <a href="http://github.com/cggh/scikit-allel">http://github.com/cggh/scikit-allel</a>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5281/zenodo.597309</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Betran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Barbadilla</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ruiz</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Recombination and gene flux caused by gene conversion and crossing over in inversion heterokaryotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNavarro%26rft.auinit1%253DA.%26rft.volume%253D146%26rft.issue%253D2%26rft.spage%253D695%26rft.epage%253D709%26rft.atitle%253DRecombination%2Band%2BGene%2BFlux%2BCaused%2Bby%2BGene%2BConversion%2Band%2BCrossing%2BOver%2Bin%2BInversion%2BHeterokaryotypes%26rft_id%253Dinfo%253Apmid%252F9178017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzIvNjk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.56" data-doi="10.1534/genetics.112.148718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nwakanma</span>,  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Neafsey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jawara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Adiamoh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lund</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Breakdown in the process of incipient speciation in Anopheles gambiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.148718</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.148718%26rft_id%253Dinfo%253Apmid%252F23335339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEyMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.57" data-doi="10.1603/0022-2585(2008)45[1057:HLOHBM]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Salgueiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Palsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Vicente</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Arez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High levels of hybridization between molecular forms of <em>Anopheles gambiae</em> from Guinea Bissau.</span> <abbr class="cit-jnl-abbrev">J. Med. Entomol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1057</span>–<span class="cit-lpage">1063</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1603/0022-2585(2008)45[1057:HLOHBM]2.0.CO;2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedical%2BEntomology%26rft.stitle%253DJ%2BMed%2BEntomol%26rft.aulast%253DOliveira%26rft.auinit1%253DE.%26rft.volume%253D45%26rft.issue%253D6%26rft.spage%253D1057%26rft.epage%253D1063%26rft.atitle%253DHigh%2Blevels%2Bof%2Bhybridization%2Bbetween%2Bmolecular%2Bforms%2Bof%2BAnopheles%2Bgambiae%2Bfrom%2BGuinea%2BBissau.%26rft_id%253Dinfo%253Adoi%252F10.1603%252F0022-2585%25282008%252945%255B1057%253AHLOHBM%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F19058629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1603/0022-2585(2008)45[1057:HLOHBM]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19058629&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260661000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.58" data-doi="10.1016/j.cpc.2010.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Varoquaux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gramfort</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Michel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Thirion</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scikit-learn: Machine Learning in Python.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D12%26rft.spage%253D2825%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpc.2010.04.018%26rft_id%253Dinfo%253Apmid%252F23755062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpc.2010.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23755062&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.59" data-doi="10.4269/ajtmh.1992.46.229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petrarca</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Beier</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Intraspecific chromosomal polymorphism in the <em>Anopheles gambiae</em> complex as a factor affecting malaria transmission in the Kisumu area of Kenya.</span> <abbr class="cit-jnl-abbrev">Am. J. Trop. Med. Hyg.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4269/ajtmh.1992.46.229</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BTropical%2BMedicine%2Band%2BHygiene%26rft.stitle%253DAm%2BJ%2BTrop%2BMed%2BHyg%26rft.aulast%253DPetrarca%26rft.auinit1%253DV.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D229%26rft.epage%253D237%26rft.atitle%253DIntraspecific%2BChromosomal%2BPolymorphism%2Bin%2Bthe%2BAnopheles%2BGambiae%2BComplex%2Bas%2Ba%2BFactor%2BAffecting%2BMalaria%2BTransmission%2Bin%2Bthe%2BKisumu%2BArea%2Bof%2BKenya%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.1992.46.229%26rft_id%253Dinfo%253Apmid%252F1539757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo4OiI0Ni8yLzIyOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petrarca</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sabatinelli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Di Deco</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Papakay</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The <em>Anopheles gambiae</em> complex in the Federal Islamic Republic of Comoros (Indian Ocean): some cytogenetic and biometric data.</span> <abbr class="cit-jnl-abbrev">Parassitologia</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParassitologia%2B%2528Roma%2529%26rft.stitle%253DParassitologia%2B%2528Roma%2529%26rft.aulast%253DPetrarca%26rft.auinit1%253DV.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D371%26rft.epage%253D380%26rft.atitle%253DThe%2BAnopheles%2Bgambiae%2Bcomplex%2Bin%2Bthe%2BFederal%2BIslamic%2BRepublic%2Bof%2BComoros%2B%2528Indian%2BOcean%2529%253A%2Bsome%2Bcytogenetic%2Band%2Bbiometric%2Bdata.%26rft_id%253Dinfo%253Apmid%252F2132451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2132451&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.61" data-doi="10.1186/1471-2148-8-309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pombi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Caputo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Simard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Di Deco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coluzzi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chromosomal plasticity and evolutionary potential in the malaria vector <em>Anopheles gambiae sensu stricto</em>: insights from three decades of rare paracentric inversions.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-8-309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DPombi%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.spage%253D309%26rft.epage%253D309%26rft.atitle%253DChromosomal%2Bplasticity%2Band%2Bevolutionary%2Bpotential%2Bin%2Bthe%2Bmalaria%2Bvector%2BAnopheles%2Bgambiae%2Bsensu%2Bstricto%253A%2Binsights%2Bfrom%2Bthree%2Bdecades%2Bof%2Brare%2Bparacentric%2Binversions.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-309%26rft_id%253Dinfo%253Apmid%252F19000304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19000304&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3249.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> R: A language and environment for statistical computing, R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.63" data-doi="10.7554/eLife.25813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riehle</span>,  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bukhari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gneme</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Guelbeogo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Coulibaly</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The <em>Anopheles gambiae</em> 2La chromosome inversion is associated with susceptibility to <em>Plasmodium falciparum</em> in Africa.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.25813</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.25813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.25813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rishikesh</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Di Deco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Petrarca</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coluzzi</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Seasonal variations in indoor resting <em>Anopheles gambiae</em> and <em>Anopheles arabiensis</em> in Kaduna, Nigeria.</span> <abbr class="cit-jnl-abbrev">Acta Trop.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2Btropica%26rft.stitle%253DActa%2BTrop%26rft.aulast%253DRishikesh%26rft.auinit1%253DN.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D170%26rft.atitle%253DSeasonal%2Bvariations%2Bin%2Bindoor%2Bresting%2BAnopheles%2Bgambiae%2Band%2BAnopheles%2Barabiensis%2Bin%2BKaduna%252C%2BNigeria.%26rft_id%253Dinfo%253Apmid%252F2862779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2862779&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985ALA2000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.65" data-doi="10.1186/1475-2875-8-147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocca</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Gray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Costantini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Besansky</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">2La chromosomal inversion enhances thermal tolerance of <em>Anopheles gambiae</em> larvae.</span> <abbr class="cit-jnl-abbrev">Malar. J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1475-2875-8-147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalaria%2Bjournal%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DMalar%2BJ%26rft.aulast%253DRocca%26rft.auinit1%253DK.%2BA.%26rft.volume%253D8%26rft.spage%253D147%26rft.epage%253D147%26rft.atitle%253D2La%2Bchromosomal%2Binversion%2Benhances%2Bthermal%2Btolerance%2Bof%2BAnopheles%2Bgambiae%2Blarvae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1475-2875-8-147%26rft_id%253Dinfo%253Apmid%252F19573238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1475-2875-8-147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19573238&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.10.3249.66"><div class="cit-metadata unstructured">Sangare, D. M., 2007 Breakpoint analysis of the <em>Anopheles gambiae s.s</em>. chromosome 2Rb, 2Rc, and 2Ru inversions in <em>PhD Thesis</em>, <em>Graduate Program in Biological Sciences</em>, <em>University of Notre Dame</em>. University of Notre Dame, Notre Dame, IN.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.67" data-doi="10.1111/j.1558-5646.2008.00504.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>,  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection in heterogeneous environments maintains the gene arrangement polymorphism of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">3082</span>–<span class="cit-lpage">3099</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2008.00504.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DSchaeffer%26rft.auinit1%253DS.%2BW.%26rft.volume%253D62%26rft.issue%253D12%26rft.spage%253D3082%26rft.epage%253D3099%26rft.atitle%253DSelection%2Bin%2Bheterogeneous%2Benvironments%2Bmaintains%2Bthe%2Bgene%2Barrangement%2Bpolymorphism%2Bof%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00504.x%26rft_id%253Dinfo%253Apmid%252F18764919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00504.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18764919&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.68" data-doi="10.1534/genetics.112.145599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seich Al Basatena</span>,  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Hoggart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Coin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">O’Reilly</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of genomic inversions on estimation of population genetic parameters from SNP data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.145599</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145599%26rft_id%253Dinfo%253Apmid%252F23150602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzEvMjQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.69" data-doi="10.1186/1472-6785-9-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simard</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ayala</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Kamdem</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pombi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Etouna</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ecological niche partitioning between Anopheles gambiae molecular forms in Cameroon: the ecological side of speciation. </span><abbr class="cit-jnl-abbrev">BMC Ecol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1472-6785-9-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Becology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEcol%26rft.aulast%253DSimard%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.spage%253D17%26rft.epage%253D17%26rft.atitle%253DEcological%2Bniche%2Bpartitioning%2Bbetween%2BAnopheles%2Bgambiae%2Bmolecular%2Bforms%2Bin%2BCameroon%253A%2Bthe%2Becological%2Bside%2Bof%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6785-9-17%26rft_id%253Dinfo%253Apmid%252F19460146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1472-6785-9-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19460146&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.70" data-doi="10.1111/eva.12242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tene Fossog</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ayala</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Acevedo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kengne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ngomo Abeso Mebuy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Habitat segregation and ecological character displacement in cryptic African malaria mosquitoes.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D8%26rft.spage%253D326%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12242%26rft_id%253Dinfo%253Apmid%252F25926878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25926878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toure</span>,  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Petrarca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Traore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Coulibaly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Maiga</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The distribution and inversion polymorphism of chromosomally recognized taxa of the <em>Anopheles gambiae</em> complex in Mali, West Africa.</span> <abbr class="cit-jnl-abbrev">Parassitologia</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParassitologia%2B%2528Roma%2529%26rft.stitle%253DParassitologia%2B%2528Roma%2529%26rft.aulast%253DTour%25C3%25A9%26rft.auinit1%253DY.%2BT.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D477%26rft.epage%253D511%26rft.atitle%253DThe%2Bdistribution%2Band%2Binversion%2Bpolymorphism%2Bof%2Bchromosomally%2Brecognized%2Btaxa%2Bof%2Bthe%2BAnopheles%2Bgambiae%2Bcomplex%2Bin%2BMali%252C%2BWest%2BAfrica.%26rft_id%253Dinfo%253Apmid%252F10645562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10645562&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.72" data-doi="10.1111/evo.12663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Twyford</span>,  <span class="cit-name-given-names">A. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Friedman</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adaptive divergence in the monkey flower Mimulus guttatus is maintained by a chromosomal inversion.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1476</span>–<span class="cit-lpage">1486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12663</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12663%26rft_id%253Dinfo%253Apmid%252F25879251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25879251&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.73" data-doi="10.1093/molbev/msr199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weetman</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Wilding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Steen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pinto</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Donnelly</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene flow-dependent genomic divergence between <em>Anopheles gambiae</em> M and S forms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr199%26rft_id%253Dinfo%253Apmid%252F21836185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21836185&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298383900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.74" data-doi="10.1016/j.tree.2018.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellenreuther</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bernatchez</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Eco-Evolutionary Genomics of Chromosomal Inversions.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2018.04.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D33%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2018.04.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2018.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.75" data-doi="10.1111/jeb.13064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellenreuther</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Rosenquist</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jaksons</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Larson</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Local adaptation along an environmental cline in a species with an inversion polymorphism.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1077</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.13064</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1068%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.13064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.13064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.76" data-doi="10.1146/annurev-ecolsys-102710-145028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. H.</span> <span class="cit-name-surname">Collins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Besansky</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of <em>Anopheles gambiae</em> in relation to humans and malaria.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-ecolsys-102710-145028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D42%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-102710-145028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-102710-145028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3249.77" data-doi="10.4269/ajtmh.2007.76.334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Santolamazza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kamau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pombi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Grushko</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Molecular karyotyping of the 2La inversion in <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Am. J. Trop. Med. Hyg.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4269/ajtmh.2007.76.334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BTropical%2BMedicine%2Band%2BHygiene%26rft.stitle%253DAm%2BJ%2BTrop%2BMed%2BHyg%26rft.aulast%253DWHITE%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D334%26rft.epage%253D339%26rft.atitle%253DMOLECULAR%2BKARYOTYPING%2BOF%2BTHE%2B2LA%2BINVERSION%2BIN%2BANOPHELES%2BGAMBIAE%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.2007.76.334%26rft_id%253Dinfo%253Apmid%252F17297045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo4OiI3Ni8yLzMzNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3249.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">World Health Organization</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <em>World Malaria Report: 2018</em>, <a href="https://www.who.int/malaria/publications/world-malaria-report-2018/report/en/">https://www.who.int/malaria/publications/world-malaria-report-2018/report/en/</a>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3249.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3239" title="Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3263" title="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3249" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UhI_WrVa3WO2mVsnFHQhvSBUefVjE9zlpO6LBMveTwI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3249" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3249" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3249" class="active" data-icon-position="" data-hide-link-title="0">In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494275" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-oghWBEAmNDfBp_OqeDy1B-8H1xYBW824ytAclXmDeAQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3249.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3249" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-8wGApre3r2EGuGwzQQ2flTpQAWC0MqriCvM_ytU3Cy0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3249" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3249.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494275" id="node494275--2" data-pisa="ggg;9/10/3249" data-pisa-master="ggg;g3.119.400445" data-apath="/ggg/9/10/3249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the <em>Anopheles gambiae</em> Complex</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8725-3929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Rebecca Love</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2653-760X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Seth N. Redmond</span>, <span class="highwire-citation-author" data-delta="2">Marco Pombi</span>, <span class="highwire-citation-author" data-delta="3">Beniamino Caputo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7939-6786" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vincenzo Petrarca</span>, <span class="highwire-citation-author" data-delta="5">Alessandra della Torre</span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-collab" data-hwp-id="collab-1">The <span class="nlm-italic" data-toggle="yes">Anopheles gambiae</span> 1000 Genomes Consortium</span></span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0646-0721" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nora J. Besansky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3249-3262;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400445 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494275--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. Rebecca Love </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Eck Institute for Global Health & Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BRebecca%2BLove%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Love%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BRebecca%2BLove%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8725-3929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Rebecca Love</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Seth N. Redmond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Infectious Disease and Microbiome Program, Broad Institute, Cambridge, MA 02142</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seth%2BN.%2BRedmond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Redmond%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeth%2BN.%2BRedmond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2653-760X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Seth N. Redmond</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marco Pombi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BPombi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pombi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BPombi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Beniamino Caputo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beniamino%2BCaputo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caputo%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeniamino%2BCaputo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vincenzo Petrarca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincenzo%2BPetrarca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrarca%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVincenzo%2BPetrarca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7939-6786" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vincenzo Petrarca</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alessandra della Torre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alessandra%2Bdella%2BTorre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=della%20Torre%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlessandra%2Bdella%2BTorre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nora J. Besansky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Eck Institute for Global Health & Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BJ.%2BBesansky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Besansky%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANora%2BJ.%2BBesansky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0646-0721" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nora J. Besansky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nbesansk@nd.edu" class="" data-icon-position="" data-hide-link-title="0">nbesansk@nd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3249" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3249&amp;title=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3249&amp;title=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3249&amp;title=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3249&amp;text=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3249&amp;title=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3249&amp;t=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3249&amp;title=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3249&amp;title=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494275/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494275" id="node494275--3" data-pisa="ggg;9/10/3249" data-pisa-master="ggg;g3.119.400445" data-apath="/ggg/9/10/3249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the <em>Anopheles gambiae</em> Complex</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8725-3929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Rebecca Love</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2653-760X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Seth N. Redmond</span>, <span class="highwire-citation-author" data-delta="2">Marco Pombi</span>, <span class="highwire-citation-author" data-delta="3">Beniamino Caputo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7939-6786" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vincenzo Petrarca</span>, <span class="highwire-citation-author" data-delta="5">Alessandra della Torre</span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-collab" data-hwp-id="collab-1">The <span class="nlm-italic" data-toggle="yes">Anopheles gambiae</span> 1000 Genomes Consortium</span></span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0646-0721" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nora J. Besansky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3249-3262;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400445 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494275--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. Rebecca Love </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Eck Institute for Global Health & Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BRebecca%2BLove%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Love%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BRebecca%2BLove%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8725-3929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Rebecca Love</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Seth N. Redmond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Infectious Disease and Microbiome Program, Broad Institute, Cambridge, MA 02142</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seth%2BN.%2BRedmond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Redmond%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeth%2BN.%2BRedmond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2653-760X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Seth N. Redmond</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marco Pombi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BPombi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pombi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BPombi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Beniamino Caputo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beniamino%2BCaputo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caputo%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeniamino%2BCaputo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vincenzo Petrarca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincenzo%2BPetrarca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrarca%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVincenzo%2BPetrarca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7939-6786" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vincenzo Petrarca</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alessandra della Torre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dipartimento di Sanità Pubblica e Malattie Infettive, Istituto Pasteur Italia-Fondazione Cenci-Bolognetti, Università di Roma “La Sapienza,” Piazzale Aldo Moro, 5, 00185 Rome, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alessandra%2Bdella%2BTorre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=della%20Torre%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlessandra%2Bdella%2BTorre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nora J. Besansky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Eck Institute for Global Health & Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BJ.%2BBesansky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Besansky%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANora%2BJ.%2BBesansky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0646-0721" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nora J. Besansky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nbesansk@nd.edu" class="" data-icon-position="" data-hide-link-title="0">nbesansk@nd.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3249&amp;rft.epage=3262&amp;rft.atitle=In%20Silico%20Karyotyping%20of%20Chromosomally%20Polymorphic%20Malaria%20Mosquitoes%20in%20the%20Anopheles%20gambiae%20Complex&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Love%2C+R.+Rebecca&amp;rft.au=Redmond%2C+Seth+N.&amp;rft.au=Pombi%2C+Marco&amp;rft.au=Caputo%2C+Beniamino&amp;rft.au=Petrarca%2C+Vincenzo&amp;rft.au=della+Torre%2C+Alessandra&amp;rft.au=%2C+&amp;rft.au=Besansky%2C+Nora+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494275/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494275/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494275/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494275/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494275/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494275/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494275/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494275/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494275/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494275/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494275/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494275/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3249" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3249.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3249.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3249.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
