<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3079" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3079.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3079.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3079.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3079.atom" />
<meta name="HW.pisa" content="ggg;9/10/3079" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="De Novo Genome Sequence Assemblies of Gossypium raimondii and Gossypium turneri" />
<meta name="DC.Identifier" content="10.1534/g3.119.400392" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Udall et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cotton is an agriculturally important crop. Because of its importance, a genome sequence of a diploid cotton species ( Gossypium raimondii , D-genome) was first assembled using Sanger sequencing data in 2012. Improvements to DNA sequencing technology have improved accuracy and correctness of assembled genome sequences. Here we report a new de novo genome assembly of G. raimondii and its close relative G. turneri . The two genomes were assembled to a chromosome level using PacBio long-read technology, HiC, and Bionano optical mapping. This report corrects some minor assembly errors found in the Sanger assembly of G. raimondii . We also compare the genome sequences of these two species for gene composition, repetitive element composition, and collinearity. Most of the identified structural rearrangements between these two species are due to intra-chromosomal inversions. More inversions were found in the G. turneri genome sequence than the G. raimondii genome sequence. These findings and updates to the D-genome sequence will improve accuracy and translation of genomics to cotton breeding and genetics." />
<meta name="DC.Contributor" content="Joshua A. Udall" />
<meta name="DC.Contributor" content="Evan Long" />
<meta name="DC.Contributor" content="Chris Hanson" />
<meta name="DC.Contributor" content="Daojun Yuan" />
<meta name="DC.Contributor" content="Thiruvarangan Ramaraj" />
<meta name="DC.Contributor" content="Justin L. Conover" />
<meta name="DC.Contributor" content="Lei Gong" />
<meta name="DC.Contributor" content="Mark A. Arick" />
<meta name="DC.Contributor" content="Corrinne E. Grover" />
<meta name="DC.Contributor" content="Daniel G. Peterson" />
<meta name="DC.Contributor" content="Jonathan F. Wendel" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="De Novo Genome Sequence Assemblies of Gossypium raimondii and Gossypium turneri" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cotton is an agriculturally important crop. Because of its importance, a genome sequence of a diploid cotton species (&lt;i&gt;Gossypium raimondii&lt;/i&gt;, D-genome) was first assembled using Sanger sequencing data in 2012. Improvements to DNA sequencing technology have improved accuracy and correctness of assembled genome sequences. Here we report a new &lt;i&gt;de novo&lt;/i&gt; genome assembly of &lt;i&gt;G. raimondii&lt;/i&gt; and its close relative &lt;i&gt;G. turneri&lt;/i&gt;. The two genomes were assembled to a chromosome level using PacBio long-read technology, HiC, and Bionano optical mapping. This report corrects some minor assembly errors found in the Sanger assembly of &lt;i&gt;G. raimondii&lt;/i&gt;. We also compare the genome sequences of these two species for gene composition, repetitive element composition, and collinearity. Most of the identified structural rearrangements between these two species are due to intra-chromosomal inversions. More inversions were found in the &lt;i&gt;G. turneri&lt;/i&gt; genome sequence than the &lt;i&gt;G. raimondii&lt;/i&gt; genome sequence. These findings and updates to the D-genome sequence will improve accuracy and translation of genomics to cotton breeding and genetics.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3079" />
<meta name="citation_id" content="9/10/3079" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3079" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3079.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3079.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3079.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400392" />
<meta name="citation_pmid" content="31462444" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3079" />
<meta name="citation_lastpage" content="3085" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Joshua A. Udall" />
<meta name="citation_author_institution" content="USDA/Agricultural Research Service, Crop Germplasm Research Unit, College Station, TX 77845," />
<meta name="citation_author_email" content="Joshua.Udall@usda.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0978-4764" />
<meta name="citation_author" content="Evan Long" />
<meta name="citation_author_institution" content="Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042," />
<meta name="citation_author" content="Chris Hanson" />
<meta name="citation_author_institution" content="Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042," />
<meta name="citation_author" content="Daojun Yuan" />
<meta name="citation_author_institution" content="Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6007-5571" />
<meta name="citation_author" content="Thiruvarangan Ramaraj" />
<meta name="citation_author_institution" content="School of Computing, DePaul University, Chicago, IL 60604," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7333-1041" />
<meta name="citation_author" content="Justin L. Conover" />
<meta name="citation_author_institution" content="Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3558-6000" />
<meta name="citation_author" content="Lei Gong" />
<meta name="citation_author_institution" content="Key Laboratory of Molecular Epigenetics of the Ministry of Education, Northeast Normal University, Changchun, China, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6429-267X" />
<meta name="citation_author" content="Mark A. Arick" />
<meta name="citation_author_institution" content="Institute for Genomics, Biocomputing &amp; Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762" />
<meta name="citation_author" content="Corrinne E. Grover" />
<meta name="citation_author_institution" content="Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3878-5459" />
<meta name="citation_author" content="Daniel G. Peterson" />
<meta name="citation_author_institution" content="Institute for Genomics, Biocomputing &amp; Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0274-5968" />
<meta name="citation_author" content="Jonathan F. Wendel" />
<meta name="citation_author_institution" content="Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2258-5081" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=M. Bailly-Bechet;citation_author=A. Haudry;citation_author=E. Lerat;citation_title=“One code to find them all”: a perl tool to conveniently parse RepeatMasker output files.;citation_pages=13;citation_volume=5;citation_year=2014;citation_doi=10.1186/1759-8753-5-13" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=W. Bao;citation_author=K. K. Kojima;citation_author=O. Kohany;citation_title=Repbase Update, a database of repetitive elements in eukaryotic genomes.;citation_pages=11;citation_volume=6;citation_year=2015;citation_pmid=26045719;citation_doi=10.1186/s13100-015-0041-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. L. Byers;citation_author=D. B. Harker;citation_author=S. M. Yourstone;citation_author=P. J. Maughan;citation_author=J. A. Udall;citation_title=Development and mapping of SNP assays in allotetraploid cotton.;citation_pages=1201-1214;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22252442;citation_doi=10.1007/s00122-011-1780-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. L. Cantarel;citation_author=I. Korf;citation_author=S. M. C. Robb;citation_author=G. Parra;citation_author=E. Ross;citation_title=MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes.;citation_volume=18;citation_year=2008;citation_pmid=18025269;citation_doi=10.1101/gr.6743907" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=Y. Chen;citation_author=Y. Chen;citation_author=S. Feng;citation_author=T. Zhao;citation_author=B. Zhou;citation_title=Overcoming obstacles to interspecific hybridization between Gossypium hirsutum and G. turneri.;citation_pages=35;citation_volume=214;citation_year=2018;citation_doi=10.1007/s10681-018-2118-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Biol.;citation_author=Z. Chen;citation_author=H. Nie;citation_author=C. E. Grover;citation_author=Y. Wang;citation_author=P. Li;citation_title=Entire nucleotide sequences of Gossypium raimondii and G. arboreum mitochondrial genomes revealed A-genome species as cytoplasmic donor of the allotetraploid species.;citation_pages=484-493;citation_volume=19;citation_year=2017;citation_doi=10.1111/plb.12536" />
<meta name="citation_reference" content="citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=E. de Wit;citation_author=W. de Laat;citation_title=A decade of 3C technologies: insights into nuclear organization;citation_pages=11-24;citation_volume=26;citation_year=2012;citation_issue=1;citation_pmid=22215806;citation_doi=10.1101/gad.179804.111" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=X. Du;citation_author=G. Huang;citation_author=S. He;citation_author=Z. Yang;citation_author=G. Sun;citation_title=Resequencing of 243 diploid cotton accessions based on an updated A genome identifies the genetic basis of key agronomic traits.;citation_pages=796-802;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0116-x" />
<meta name="citation_reference" content="citation_journal_title=Cell Syst.;citation_author=N. C. Durand;citation_author=J. T. Robinson;citation_author=M. S. Shamim;citation_author=I. Machol;citation_author=J. P. Mesirov;citation_title=Juicebox Provides a Visualization System for Hi-C Contact Maps with Unlimited Zoom.;citation_pages=99-101;citation_volume=3;citation_year=2016;citation_pmid=27467250;citation_doi=10.1016/j.cels.2015.07.012" />
<meta name="citation_reference" content="citation_journal_title=Madrono;citation_author=P. A. Fryxell;citation_title=Gossypium turneri (Malvaceae), A New Species From Sonora, Mexico.;citation_pages=155-159;citation_volume=25;citation_year=1978" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=C. E. Grover;citation_author=M. A. Arick;citation_author=J. L. Conover;citation_author=A. Thrash;citation_author=G. Hu;citation_title=Comparative Genomics of an Unusual Biogeographic Disjunction in the Cotton Tribe (Gossypieae) Yields Insights into Genome Downsizing.;citation_pages=3328-3344;citation_volume=9;citation_year=2017;citation_doi=10.1093/gbe/evx248" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=C. E. Grover;citation_author=M. A. Arick;citation_author=A. Thrash;citation_author=J. L. Conover;citation_author=W. S. Sanders;citation_title=Insights into the Evolution of the New World Diploid Cottons (Gossypium, Subgenus Houzingenia) Based on Genome Sequencing.;citation_pages=53-71;citation_volume=11;citation_year=2019;citation_doi=10.1093/gbe/evy256" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=W. Z. Guo;citation_author=Z. Q. Sang;citation_author=B. L. Zhou;citation_author=T. Z. Zhang;citation_title=Genetic relationships of D-genome species based on two types of EST-SSR markers derived from G. arboreum and G. raimondii in Gossypium.;citation_pages=808-814;citation_volume=172;citation_year=2007;citation_doi=10.1016/j.plantsci.2006.12.012" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot.;citation_author=B. Hendrix;citation_author=J. M. Stewart;citation_title=Estimation of the nuclear DNA content of Gossypium species.;citation_volume=95;citation_year=2005;citation_pmid=15701660;citation_doi=10.1093/aob/mci078" />
<meta name="citation_reference" content="citation_title=Plant Genomes: Methods for Genetic and Physical Mapping;citation_title=Simple plant DNA isolation procedures;citation_pages=1-13;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: Scalable and accurate long-read assembly via adaptive κ-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=F. Li;citation_author=G. Fan;citation_author=C. Lu;citation_author=G. Xiao;citation_author=C. Zou;citation_title=Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution.;citation_pages=524-530;citation_volume=33;citation_year=2015;citation_pmid=25893780;citation_doi=10.1038/nbt.3208" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_title=Progress in Botany;citation_title=Sequencing Plant Genomes;citation_pages=109-193;citation_year=2018" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=N. Servant;citation_author=N. Varoquaux;citation_author=B. R. Lajoie;citation_author=E. Viara;citation_author=C.-J. Chen;citation_title=HiC-Pro: an optimized and flexible pipeline for Hi-C data processing.;citation_pages=259;citation_volume=16;citation_year=2015;citation_pmid=26619908;citation_doi=10.1186/s13059-015-0831-x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Botany;citation_author=M. Ulloa;citation_author=I. Y. Abdurakhmonov;citation_author=C. Perez-M;citation_author=R. Percy;citation_author=J. M. Stewart;citation_title=Genetic diversity and population structure of cotton (Gossypium spp.) of the New World assessed by SSR markers.;citation_pages=251-259;citation_volume=91;citation_year=2013;citation_doi=10.1139/cjb-2012-0192" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=L. Venturini;citation_author=S. Caim;citation_author=G. G. Kaithakottil;citation_author=D. L. Mapleson;citation_author=D. Swarbreck;citation_title=Leveraging multiple transcriptome assembly methods for improved gene structure annotation.;citation_volume=7;citation_year=2018;citation_doi=10.1093/gigascience/giy093" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Wang;citation_author=H. Tang;citation_author=J. D. Debarry;citation_author=X. Tan;citation_author=J. Li;citation_title=MCScanX: A toolkit for detection and evolutionary analysis of gene synteny and collinearity.;citation_volume=40;citation_year=2012;citation_pmid=22217600;citation_doi=10.1093/nar/gkr1293" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Wang;citation_author=L. Tu;citation_author=D. Yuan;citation_author=D. Zhu;citation_author=C. Shen;citation_title=Reference genome sequences of two cultivated allotetraploid cottons, Gossypium hirsutum and Gossypium barbadense.;citation_pages=224-229;citation_volume=51;citation_year=2019;citation_doi=10.1038/s41588-018-0282-x" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=T. Zhang;citation_author=Y. Hu;citation_author=W. Jiang;citation_author=L. Fang;citation_author=X. Guan;citation_title=Sequencing of allotetraploid cotton (Gossypium hirsutum L. acc. TM-1) provides a resource for fiber improvement.;citation_pages=531-537;citation_volume=33;citation_year=2015;citation_pmid=25893781;citation_doi=10.1038/nbt.3207" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="De Novo Genome Sequence Assemblies of Gossypium raimondii and Gossypium turneri" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Cotton is an agriculturally important crop. Because of its importance, a genome sequence of a diploid cotton species ( Gossypium raimondii , D-genome) was first assembled using Sanger sequencing data in 2012. Improvements to DNA sequencing technology have improved accuracy and correctness of assembled genome sequences. Here we report a new de novo genome assembly of G. raimondii and its close relative G. turneri . The two genomes were assembled to a chromosome level using PacBio long-read technology, HiC, and Bionano optical mapping. This report corrects some minor assembly errors found in the Sanger assembly of G. raimondii . We also compare the genome sequences of these two species for gene composition, repetitive element composition, and collinearity. Most of the identified structural rearrangements between these two species are due to intra-chromosomal inversions. More inversions were found in the G. turneri genome sequence than the G. raimondii genome sequence. These findings and updates to the D-genome sequence will improve accuracy and translation of genomics to cotton breeding and genetics." />
<meta name="og-title" property="og:title" content="De Novo Genome Sequence Assemblies of Gossypium raimondii and Gossypium turneri" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3079" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cotton is an agriculturally important crop. Because of its importance, a genome sequence of a diploid cotton species ( Gossypium raimondii , D-genome) was first assembled using Sanger sequencing data in 2012. Improvements to DNA sequencing technology have improved accuracy and correctness of assembled genome sequences. Here we report a new de novo genome assembly of G. raimondii and its close relative G. turneri . The two genomes were assembled to a chromosome level using PacBio long-read technology, HiC, and Bionano optical mapping. This report corrects some minor assembly errors found in the Sanger assembly of G. raimondii . We also compare the genome sequences of these two species for gene composition, repetitive element composition, and collinearity. Most of the identified structural rearrangements between these two species are due to intra-chromosomal inversions. More inversions were found in the G. turneri genome sequence than the G. raimondii genome sequence. These findings and updates to the D-genome sequence will improve accuracy and translation of genomics to cotton breeding and genetics." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>De Novo Genome Sequence Assemblies of Gossypium raimondii and Gossypium turneri | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"BEPk-v4jNphGtSAGnb2IPS-6pORwxYzlHiEC8QFW6Ac","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494290","apath":"\/ggg\/9\/10\/3079.atom","pisa":"ggg;9\/10\/3079","ac":{"\/ggg\/9\/10\/3079.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3079.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3079"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3079"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3079":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222BQCthtHrNxG\\\\\\\/2O9+I78qBz08dfK69kBYZyIzjEhuKBS+gjxzptv\\\\\\\/cY934W1EO0bKveBoLUtc4eJA5\\\\\\\/pvlU895JYfxcAZSc\\\\\\\/4v+NtAKs6TFzybZFZcTbt3nPiaP3dK6XhKBWNjzNHuMntGIL0PxwXLoBgqkElSJDvv8MdT3zr5e+bao9fHyvL5VIokg2QMMUTC72jsS2qKEBTfBFmxnXiBrSd0gvBp2jeJbgpUH8FVLkPNwuMWyePZAGrWNQ35DCGTrXdwZxXfMr6skESgmJnnwW9YwpUhnB6xGkXKOn1VjYv05qRkPNQGNI13nhEv88YFyU5wAjZreZ0skAURPCj8H4hTqMI62lgMcZisHUZ1Cp0Wuussa57zaWHPEHEiKsZcFnyRd5\\\\\\\/YYxl9xo2QSKKkvlAnjzA4ZY1gkV+SnhP4oyeFCtnLh0yDZRW9tb1re\\\\\\\/fzjwxp+yofhWJeXoNvQp2U8\\\\\\\/vtfkmCTiyCdsmeZqvjOdSAei4axeng0LoChsnjPwF4Vi7gihNWVJ534L2o+DAUC5srs3H6bY5af8EO+8TFpGEv2ofmJ\\\\\\\/f395TSJbiTk4PmXVIFinVpauvU09wlIg7bZd8JDuG7LXrmbRyvgfOk1ZrTOkgdKvLZtprQyHczfu4ZSnH8vwmV\\\\\\\/+rUa5ZzvU08yp94+A88shQN54TPXbfdtjYdkaw5u2QT0TTVj341SvIhG5NJdgPi2UAAMnJdvy9oau0sLa8f9LKgrxuy8BuFbYuAB2vOKkqh3029WzBd3qvZjS\\\\\\\/be+OOwvRPK52d2Dwti9mQ3yMMzv+qSKDPfs61YHLHdHOixYeALg5r1znq3iXcr7fdfXedgpe+BGMwl1uonYbokIRTmS9Ljk2Uf82hV0U4ktB\\\\\\\/dtZj3WkuAxPl55iPY5Dh2JfDhZUn+SJpJHn9AfMyyz+UiS5NCBhYsH4UmNhChFB\\\\\\\/0tWR3wC7tdyOjxwHVc3z+rZsvBAsXlpb\\\\\\\/64fLDsbIWbihnQnSiK+agnyRZ1HmwM321tZ23zkil0TfPEcWa8B726oo5PuhD8aiQvykSWRikEf5vZ+NxdP4A6up22\\\\\\\/QH6ckPUirqNHAS8u7aRY2IFoGTb8jIvwsVVk3yg4V\\\\\\\/1N53oSys9WACk6MYR8VG541bHwBT4lN+dwuDzJS5L3fjRW3eFOkGqf\\\\\\\/PF2K8MbUpEolRVH71SWfAZvTuvCZoNTPIi4kZY7Pi85sOXTLLUZQMwmmKiknn3\\\\\\\/I\\\\\\\/tTXa37ABXLRwNh1RBkEpb\\\\\\\/egUUn5RoNRXSME1vjSG6nwaC5R\\\u0022,\\\u0022iv\\\u0022:\\\u00227\\\\\\\/3juHnpiDRcuRtc7dFxcA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227dec41471c2ba5aaa197d775950beff88b2b7484e9679ecebc285b696a92250d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GzQVwQ+5N7vqvfNPMWcDF3XHh\\\\\\\/Pb+mcvZicdlP87C4wO0HkG1mahRWUPefGAkwT2OHKFcrfCjkpexiQPYVgNPClMyTfgPYX9mb3aOgmgRUbIGmoP7TI64mj1vkx3TU2p\\\\\\\/ixci6FScFRZR2s3NxCx\\\\\\\/QsbSmi02o8oreI2v53uR1NLMMOPs9GkLgK9I7Q3MH8XQcRTL4dcAf2jowMh8RE\\\\\\\/OQ3G7NsO3iaVfTLx2GmBXfoZP4esGKEGxn6VvFtsKUSs5DwVDDMRJwVbbtUVqY\\\\\\\/b0vkzUCXaarNwsLZTHZynHBmFIqOSTnPAIj+VRepkY9+XASA2DvA3ey1Swx+N7GMzQFmtOr\\\\\\\/UU31Qy4qcvwVz5nOgGWvnLbl25nxW+x5jpaGIqZaJRjN\\\\\\\/NYV3DoNfJ9phkDHideFWdwvf2c8OXlsIEhFiuWC5eVRKHXQrqFTTvZKIbjIDjQRKDtwqL+s1YfKjVgAY8hPYE0qtm2FHhPzKlLe3N6HrM4CSIrsS43tNbiqCGRfjeynnJQFbAXL3irSuYspr1mzJCZr9C0pqjs98EWgmKm1W3OjwAgMGRRkUAfrOOSmsIHrPOt94BeuhOftbmbqxuy8hVJkSePHkvOLUjrsBOiolHEzbxKr9as8glFw3TNCF0mRDVCadGN1TgqAm4ngF9Yc2vh7nYUYxbqf0j0FhjUZ+JCGf3ivqLRLO0hWpEngz\\\\\\\/dcU1NsQLbNqwKDKrAElFt1xp5yZbSBq4Q6Y5B3UyeTdgsXhydRXufU9i4lf7kp2aEd55pyXgrtl4SjxOiXJyx75I0TTt4m6eGZ+nVDhk4fb75dDHpnN5YADTOgr6WuMCtcw\\\\\\\/GUvpSpdntjVZ3K+LLT9iue1x\\\\\\\/TY\\\\\\\/tD2\\\\\\\/TVLP2YPe5MWE4JqxODrPSACS\\\\\\\/gyHnyArFb++c6isiQVhDrO81kEff0y+XXaVQNZSfRCmXAfK4tFecuamcBd564pZ80Hzre79WKQXlL3IYhuj2RM+c5+8R6bR79yNmWCdFCyteuRKpjeWwbbcbiZXTflgtbQp8vJgujhr3a3NUgdgAmZR97PZjwukoYlBKFOH1BJi\\\\\\\/hlJM\\\\\\\/nhi9F5r7lTG21hX7LtVl\\\\\\\/53P8zHi5+xowpksMnbzIf7gt53OWTIUdqumIS0yD2KuTMnu5wmlZOrEv2\\\\\\\/Ov4jd7eE1T2bFELmsqg0EFUsfiDlyvnOJjI9XKS3B\\\\\\\/wVWcRIX\\\\\\\/j7VWmUUQoTxi5Xwj4mr4YpViTeAwodELJm4Pd2Vec\\\\\\\/7psWRXcQ1QsZG6ePFFSn0hu2Hdrr6\\\\\\\/gPzFNOfA8yKLpdHhnZspu7NIMLKfHp8CX7DO7uuiSo\\\\\\\/5MQfF2\\\\\\\/O7ZphI3rZTN3t4qkWMtMzaJVMtJFgUea7YnsuIN7o4JIrf7xdWooAP5h6lHtnZ89MTbzW+i5gRrsa6Fcki8UbW1Xas+L+7dSHrzr4oV3sF4RQahk7uwWzDgyKPivV5XZHtwZvoXsRQZPur+ZWqvtKOj5zpzpFWARfXVEpnX7irs4v5PxDIPAUxJcXGe0cBw8KxkMuR8Bm2FMNSon0\\\\\\\/nVt6G\\\\\\\/SfUVygtP+MyNrfHcdA5gaGmP0m76wNS9s9QdrNh+ViP3qHI3GgzR6u8FiBuJb2AxRJL\\\\\\\/jG1AgYGrcrilR8P463jJlnX+GRKma5fjQc0egqyDAwTyHevvrTdZ75pMtPxyBB+G\\\\\\\/oyOfl\\\\\\\/bw03YBM\\\\\\\/E\\\\\\\/7y\\\\\\\/wmAUMrwrjRJwbWljKDJERV4gjpJOQRbHsIkN8Mr7Aj71nuPM2s02m8RGUXjhXoT6Qi3GDETNK6AeWBr8ll\\\\\\\/bF7WsHnNiy5wHksDJsNdBY3cuJMmqTWQlYKM7xSGbGg621c+C4fgD2kydxbNt8s73JEIBJCNGCw0c7WfOocGgFZDMffkOl68riuBUAMGywaazTiIevLMjCJM0ZhCSyKuzrFN+9JLYegxkx+QSOLi4jwOKZHJvar3pWOR8LH4NESsmajymgDLYPD37LM9Ki1NBRJcOCImNZjDttr0UX1Va1xUc6ylhESWL\\\\\\\/JJZJltUe9wj64nD9NjWEE9kDeIzWh4axdUpnHEDwK5OQQVT5AioR3TogrDks0o8XPJimDsTMfiZn3ghMqAldgF3yC5moFHFE6gjBQr5uTYloKkO\\\\\\\/dMinoQ0lwcoqUUo8njITepTD3\\\\\\\/eqorQP1S8OKSHSgGhZi6exYmjxRZALaMsuy9ztKLS86HcU2Fj4IcK74yqd+vI\\\\\\\/C3NwvbTGFvnhommd60mieGijuOqymkka1I+cY8481kIIas+pcr3ueQPNI93KlyCO\\\\\\\/91d658EjeP07NMjNBO1urg9eE9lK9RIfVv9pwCH354Hq7iEc8bRUto7opmUD3LvnXGcBaLfZIUGMy5M0wQCJ656+wCAKic4C0sCGqtnnppdk40OOzepV4ssP9nGKndCgYP4I0YIjwt1rFV\\\\\\\/jxoUmWqA3qbX3aVP4\\\\\\\/2405oStF4srzNO\\\\\\\/oSGcmS5KZbfVie4GFG5gYagJaPz7xqBVy\\\\\\\/vUg7rez+lXqYw4Z8rQu\\\\\\\/YxYOYGY\\\\\\\/xjIX4jFB5h6eKfY0o2j99TxF6r0xPg8XX9rAM93KWGORaN7VnFn\\\\\\\/kpUI4gx4fG5BM5Q1\\\\\\\/k9GrgaDWqjqaZNmttannX00cX+nioCz8fihYAN8jTfkWVPlQZy+rXmPSXaI4sjBwxXv3c3muzci+qVQTg\\\\\\\/Yb2A47QbX81SRaJ\\\\\\\/9AUkMBiXlSozxs9DpwhXEuqjPAfxXXiMOg5L8\\\\\\\/Bna+zHGRDdPIlS2sGROv4iOXPx1thrfftNvWgE5jq4n5aQdDUCebPW9CYt5Ha0ng26uQ\\\\\\\/Nv5gY\\\\\\\/2VH7MgCLhhp+yJNIUi8QUc++Cc\\\\\\\/OmnMwNcayxj\\\\\\\/2SqbApT1mvwPg4DcRMeG5C33Ue6xvJ+wyDqPY+5ekZHN97uRIp1ypl6Qu52euT8ZOD9pCwYbfKeEgfNPuxe764h58lo1qlcKEElKvyfsKEiwXnFwlFMysZX7hi4NsvbvyNsgjJDT2S\\\\\\\/l3Iojyt+AWwK+lslNFgTCrgSp5b0qbqIAfwu8IXTW\\\\\\\/K+gUhi\\\\\\\/N4MVDCltHZQrBS1fMvHLoLxRod0mYYvyMwflquJ0TkHHq4QTo0x1v7RDkm6qA8NHnB2adyPwm9q97XlcKxm68YSbeDzGanpRYP8nyweHGZLzzdDQ+N\\\\\\\/T2+KIu+sWON6M5phu8ZDXVX08FFloFsSNZqkxbhOfo\\\\\\\/m+OFqpvtTzMrleVmU\\\\\\\/eY4\\\\\\\/esi3aU6SsTVmC7jAXkH9fDjeeMVtm3m2CA11pn9PVAmkOHrfC0mQSAVfKqYP1rf5hxz2Fkr\\\\\\\/ANV6djsplK0KGz9zUuBV8D2JqWLxPcVr\\\\\\\/ph9fxfJGgQcUNKxJpsk5eZSohXULrgYRAPUCskEiq8FyOsQhjZ5aIaz6A8paZY7BGrn8zz8WjwKy5R8byLd\\\\\\\/bTPtQpRWAXSnKBfPn7HHV3jzZhuOBWBvrp\\\\\\\/W+UNOeibaYNWQuwvUybrdyJxRiKrDkpQo8WM6Qn82uydnJWReBGrzEav7Vi6myXbxjqfS1oG8EkfneGgkRQ\\\\\\\/0Rgys4etS\\\\\\\/YAo2AVC6KgVofjxUjlEeJiPclZaTSDdqWBX2iaoXggRAAAKd5ATKJyPQObQDRAAMytw25ybEnsaeR2y4aWr4pqqVt9edT0SEtD2OdABffcXpv+mnwr7HSIGI9oJgdqwW8fGx5PWzoJHdM4XSbYNYajLrNd+bHj7+0ED6C2n7HKwpcAKOaaxDZVix1I4ZksBxdrnxsU4o5uVS5c9Ere34K7e4a97yBicaDnpW+2fEhKIpnINMkY4B5tF6NVMVdXPqhirIWn3aezFvB4AVwJA9lAKkKh\\\\\\\/FsVUnqGu3bKdJa8zFILvKW06u+6\\\\\\\/JC5s0rkrO+Zent+Lc68GekWf5ogYzsSL0TckyG5uljPNOMr1E394dt0RCgP4ni84xalatJOPF7QXKECwOc15UUjq2KzitjjZaw0DtIe8HWx0mSbm1wLqVb72vCI1Abvn5hkjIGpg8fhy7F1J1w+2V3C+6KavgttR2QyiK0UYwhyMFUfT7E97mBORUA4vFuK7iOqrBgiWst5ueqTwjWN7fuUSb13JLmPVU9TwvvnxDJ+CBOhQupm2FBeG\\\\\\\/s88T6KPz1qPRb5Y5qvrJ+hReQ+kyXHZqmg67hfmEX2OywJYmxHTMZqcwGbevwqaBX8lDPR2aOor5KqWYTV2bA4ySnZcZsRmp4ThjxZeV6ToDCaRxFcGb208eIevjHuSPQFt51WBZ4hA2TjP1\\\\\\\/1X1pS\\\\\\\/4tfYD5+UOOn8IlIvDMYS0Z139qwxSPB428og1Hlqm\\\\\\\/87+JetIXM3k3RiaXcQ2O2h9iZ96VVNH2BFDBExKZ8WQJXGERfjonKWYYUBwhEOupdnw+KI3\\\\\\\/3OeSNRWPpBXzm0h0iNPQVlhcyWmxPfxJnG8bUBgyRtdEJXWYZj4z4e1eotD+7j2G0eg9XyYz7OhUhIbGWqDh+eaLBfmIrP4an+REr2BXTYiWDGzA2sNwILHbPyi6oXOq93Rp3sU5aOFEqY1oE98rHMAFuwgPc0jq3OGdNuptHUh6wZvXFAQmG8\\\\\\\/OccD2sDhY+GwhEQxqKP3Go3FM\\\\\\\/Acw6v1lMLYcxvTCG9GKMPdD1Ee4GasDIOowYVVXYwQitOXAxcT6qV5VbpSZbvA7ftN2LubHGJi48F9W3AlidJSSmaq8G2ophtPweVvddIm6AWxdG9bGEjXbrIWgDSL1GWf3vBZRGJ4TqhniThPB2RiLObYpBzgUO5JgTt3RyIoixR\\\\\\\/a4HoUmu3tHgvz03DizjK4IEKrHa1SswjmIJB8yUjveebqbTPropkcHNtzVKwbel5Im0DMxI3KmzwaVXjUTZHwWeAGfmlbo6wwt47NJv3ThIR6mta37cZRnuVKNJFcldms4KrTdb7XWSY2zDaK+3Nanf5sT\\\\\\\/UevJ5FDgKbEgQdjIrTrLzNsN8sz70sESwIT\\\\\\\/LG4vVKGdNi9z77irQMB4zbGFZ6TTkLwpQ2VqwYwl8VmuA5Va8s10ml+hyKb6lBnohvSIn9PKgl4F9QCtbzD4xwSRtlnAZSoO9WmQVy0fUGijvTDfpJMiT6NvwNogQXboqbsSfxBWmOsiF8Zgf3cGPdcILDwJXVdI2XSfTDgWZJUkVINZZXlVCoAD\\\\\\\/RVMVVK14Uze4j3AXgQM3iIZnXe+Ky2tbVpv8h1e\\\\\\\/IsQ0j8annf7fJWquZGFqqP0TNFV1wGTl\\\\\\\/+7NwBLPzLsaKL4qIkI9bVwV0UGRWNGO\\\\\\\/XOumKBeKw0zLNyncqWtm2rAsD4Llebr4amtHPK2zhp5aaHcwBYFi+kAYLpZxiPpSDVG\\\\\\\/qxadRJoZ\\\\\\\/htvjsSMdnXPVwDRWOACU2G9oVSzWomXPPVYWHHyXbA5+duW1JMfwEFWqf9QxKqVr\\\\\\\/v33S4po4+JLhOxmUK8SZT4yUDVO23TLp9SnVjxAFVJW5WMNTi8bLY7QSXIvCJURdjis3XNKjRtuM2nQ3wrzplmohWJPVsV5g3cAtLAXYkU3d1jRGWF9JPUqyGxcXQHr+jFeInvqQYLFSeLMGolYcVOUOrwkDz0hVzHgGX3Wx6cqQ4eaiiHBk8Gk6+csSaA7L0Pb\\\\\\\/6FM\\\\\\\/TaXdwSKfTaqnBY2LfLg+iUZ\\\\\\\/pXuNs5oaer\\\\\\\/fnympV98jEuva1zYwjzSxTEV\\\\\\\/X1Zv5q06sgtJt4a5uTGsmLWqhrCegQm7ry6LTIhsL54GSzhQ6pqdGm5h6JWxS1E2NAXPLonNLMj9tAR7Ea+4o\\\\\\\/SPXkb\\\\\\\/jQBS5eIMZ3fqMmeaqZs6\\\\\\\/n7wD9M\\\\\\\/6KflVAVJgkQxeBcxX7DIhs2gl+3iSIVkxsmOT1JSBcRaewa0whIZ6ujp1T7wH9OxbI8Fjb46yWbjy7mdURnRMVGs5jO4edKq\\\\\\\/8EEg2mjYHzgyJL3FZLgqIk4PLOkE1+S\\\\\\\/RRbOKMh15JO7qQakvNLDFDinRhUBGeqB6ulIH\\\\\\\/so78VMXK4laahfSUg9A\\\\\\\/9Xxc+M6ZOgUnGWJiyLpgvS1wWOEB9CG2dUwoZfTdT1PON85OwieeOwkaUD\\\\\\\/l06Fp+8NzIDtai0XGEK9WLlR46s\\\\\\\/8Pr79wYmF2A9S14LVB+y0BH8DV+sXf2srwnlJFBcAD6MUNN2X0CKPu4t3f7AbrfVLysyJ1ola03X83eL53nczNAHToPLYmWtCDAzQ4Pe\\\\\\\/ObczDLXjaRJZT5B7i4TzjRG48TmdD7aQIv8eT5rTptuEp4mMWnd66U1V5Z7nQ7ZvwRzst6CDEde\\\\\\\/lW050+WFgCnHAWP\\\\\\\/tHOs\\\\\\\/iNx75r7oYTn7rXaUtkpHFawTdoYhpMhZbZuCI6YcvOul3DzkkW3fR6DWbbC3ietkV9w2AyEr8NyL01Iudky06hz4vISHcePVd3RsvgQzT38tN4iB8C6DemywtRHMwBYEJ3zY0U+lxKigEfstvJlauwFJbrit5VRvAI1xWuNG6b1h\\\\\\\/T7A4YpQ7RVDvdDK8FQdr4NEAAb0KQ72QWWTMLBdhIgjGMhviJlwl2dIwzpOGtlmh5XUQVmurWD4h\\\\\\\/6szmIrQ1Sz7ctedaH3EH1uujUcEcfvfW7\\\\\\\/1Y\\\\\\\/vs9iBUI1NTCS6CMnibYAPGW\\\\\\\/MNmYOF9S2erqFrs1fPNVz5s6mc+qUdu+rdCstfpO6ZjfolYWfj7n7tyKBae8\\\\\\\/4vud4NeZBtxAj9YALinP2OLXmFhifOfS4yxSowyB2\\\\\\\/BZa6kjrFlCXR7p8rYgBvYdQrb5HYpeD0kXLt3tY2khd2RF36jAbYselQp5nFSSsj2Q+Pm+liO7ZlCoXXLTX4fcPGKKuuCTBAwrofjDAL+fppYW1U4Gj1vU90JmPIr7l5ok4TUoMpovdSVYu1avH\\\\\\\/4V69o1NtpRSYne5tKE3zZYkRzQjtO+dycExIbW7lD9yGpVGd5yp8B4J4EKauOnW0j0vz9oXU\\\\\\\/WdPH5krQ1y0ZB9T3UdCVwIanz25BiKC88c5rLIhW1tuBjww5QuLWJfER8xjUpqKhG++yW+72YD2A4M5VVrbTp8J9DHvBR0t260ACXcRcaPx9HEqnofOcW08CTrw3n+jAzLcm8m2E6AZgJxWRHCMz5oXlP2pzzQUnIBE4x7ZZJTNFrfNHosAO06biyvtmRHSePut4zff+6++Yxu3U\\\\\\\/54fu8DtaEBpWT3RAmtIqXWcxO0FW5X43QvAmO7KIr47HKVqD75R5ShVBHUaNpPdW8ZYAxj91PpoTHDMEL1zjLXXFH1NIuRbobvcrOvdUGqTBhLIlslYjhFG9ZQ+lF+tcRHgiW\\\\\\\/Pi2O5AgXRu2NZdgYpTVpET+8xpSNs0FExKFn96PpGVGJHD17KUruncwAavXrvv2o7GRb8aGTeWprVz2DAL2BJTLn\\\\\\\/OM27GZhjz1PM606xxIx7m3TG83idMWAxLptSDr7BrCSZ16N4sZ5FFMHB+6fWgnuPWDe4svZ45WaYWXd8aNChZVG0m1RH\\\\\\\/6rrR7WWia\\\\\\\/43\\\\\\\/n4s9bjC8EchACgla21AgNZT8mPX4LNYvsE1uJnLfA0LptRNr++EDwoXCPRi\\\\\\\/9BaHMelZOAtaEnCwEBS+v5JewvhQhKlx3ng1FeDhz+xP4k7cm2yeJJwh3qbecSY9aEFFvX8dxY6fAnDf3SOkNUlLTgYVTJH7SyIfEAdm2tfG7Dqv1CF97jn3WqR5J2t9sB3zyGLIRkyjUms0KMEBqqYaHR4R5PCvrsbYKcdlhCDseQyMxwwOllbDW9IeFk4g+3+QkVKbJW0GLlVYc1uJlUI89st23PC8sMdZJwmfmo2eVNo0lX3\\\\\\\/QK+SrY9\\\\\\\/KbfNVTe3tONFu6L7tuuI21L3lyVJ8tut3uG1BTPBPuRx6PN\\\\\\\/fxywrapj8RDfLlvnjjKalgOjVIxk90cFGC720oymxPJal46kL+PG5aGBFv4k7eHc5NhEGVxV9rhFoYoPA8KZX1nmbmgR6HbPqnMSODiQaQBqM1d1pTo22jPgoLE2NfLKTujWrKIJcJFWBVcLUjVvBxJkLBdCDsWENDCPPwwK1UPNvWsauRyxkoNXt4AY9eizSRRJ4z05kHRGL+JO22una5C5Ouq0kycUdjKAhVhbGvzuZfy9advsK+egzIu9AXQUTh78x\\\\\\\/P7yfI1TuOyKE38gjQ4yBg5U58HKaucNJ0LRfddwIALbCKrv3kTZdhkBGFWlgAe1bkzj+cmCArTHyM2zBgvq7qq+qf8Xmmdgpc12KS8Di459aGfwoG8k9MeTX\\\\\\\/PtRl+uQ\\\\\\\/PljNeY0+Q\\\\\\\/rnGcUjt7c5zw1UL42MUvmaqwDP0p50gk1BWPiyOIq7PYx8zS\\\\\\\/i8WKK28cBHPc+0BtHf0Lz+pgK+vekFXe84EbDiXd+old1D9y4ED9irkNmwaxKQPmpwQoVyARDm08w6sVQ5t6s\\\\\\\/ORfYc36wZHQ6ys3tFlxm0nUSD2ypzhOgs71MTP+H0Xlwzoc5H1wrsXQwnK7uHSmTg2joLU1TvtXlI+25UdkjKh6FfNVCGE2AWC2nXaTufQQBX6el8vl+HIoJ+lC+170\\\\\\\/CPxgoC1vGIP0G4dbxfusYKnBE\\\\\\\/Sq8tr7YjCBZ21xPEvzQvBBOojryQ6Cgefoo5QDxoJzX4+tGvv\\\\\\\/p8d6wrfiWCtqJy+cEzkecC39jazNrAsVkDaBAtgIJ35jxlJE2aoSSeUKRujWWXqgOFKC8CCc4kCZKQGJBNdUCZRL8eXcwKu5REJTSp1xnQxcNMpmbMa4kw0w+Rq3hrlBzXbVUlyAnWNH4GSoLjoE8hCO0jHbVPEQyijCgRzfov3Dw6OVRwzk5Rn42UDCg6A3oYcDX\\\\\\\/fg5w++Wsj+4lLfvLKxY8OEcifmBjDoUdgC9\\\\\\\/11ZQcVHGZQjwmfpDmP6vHps9CJxjyWposnlQqKIX2QrG5fZQVq9ddK1dpbQ1qxUtt\\\\\\\/s7RFH2xCEqtFgUPNZZLttdoF9XQKcikDtBOHlS7LW9HmhHLBUKjuFVAivfhLmFw2JlJW6qF92LOYif5dfERU8JTJhvxWlzH+aCvLDsemU2MgnVAoV38OXeUGKjkzPydFLqBhVPYcF6ekKuHJ+sEN6M2cvYFJHkpbuednsVD0Ufa5nAi9t\\\\\\\/gMwuXPaXqsVm7BBMkil0NaTGQrdHPPf7jZSuIOKtdGl9JImxwj8rc\\\\\\\/Yfk1FKPt\\\\\\\/\\\\\\\/awLg7JXSMsf7GS7\\\\\\\/QvD3RzDksNj0fb5rzvdyo5mVUv5hVCjEcsmQQqnqI3ULCkv89UOei1Uvx8efaYdL8yasGY9S4PfXN0PYvH0tq3A+7qi\\\\\\\/vPUT9wlGiy\\\\\\\/4wNfKcjK4rfBV\\\\\\\/02ib2uKPQfRt2chmj\\\\\\\/Ocf9zGd\\\\\\\/Bfrf1+f2jU3kkldOxy9\\\\\\\/UIQPPFd0ujFDBoX5hSzvsWkWQXEzoN1xcybhTBWIDvi8jdEnYTdcEPA+6L2Yb4ewFR48p8PaIbIagDUlNgpXVEt2hNV3sT\\\\\\\/qEfujVizaI+hBmx1mNom4iA+7O6gHV3AzPDcbJ0FvfqKuSE0kHP91P+YPtcqroi6x3rIPfHczcMHRnGEY2hzwWXEicxn2etNfC6d\\\\\\\/UhZ4UxCsOL45ZsGh6L4eAee41y8zdjN03MRUL\\\\\\\/DNgA6BYdNqdEha8XwaYyAdUisia09cydUfMllhEXO6j66\\\\\\\/1cvoPDvsnidb70V1yRKvATfU6xZzOKOdxS5ivLxYWW6UhSunwLUmLoJ8VS4CwDQyCYkRlKSRNJMmGMTwG9AbKo\\\\\\\/5hnGVzRyHgo6o+jo0y+7hjzBY2h3C13OH8EQNt+72MnTxlV\\\\\\\/oAlwbAXB0qpTyAbxXj\\\\\\\/boj3zurhLVEGU3EYsM34L0Ttv9AB3\\\\\\\/i3dfJmZfnb5L4xSXEa7tzyF8AGA7j7MIgo4PdVObyR52bUTT6tX2JJWtTF6KwO4XI3K46ZFvMoi5CoY6NU7C0amzBnFvhT7XOkIqEn+iyJbm0MiFQfHs\\\\\\\/CEWOOwoMH8EN4CQI9D0VZ64KoLh0S26Pm970vPEEUZ1ruhyBrbxzJ9N5wwVZV1ZtNGoGiJiH0zOPMA3e7SzgNyF5h50R3J02a\\\\\\\/r7t4yS6SnhNMHzsAZYx8MJRdLuf4o5nbbxXkr6UafXwydEC\\\\\\\/INTV9IesL7pZ7L2o+\\\\\\\/\\\\\\\/25Gx4sKoM0pDOp6YfLrESXhiFFCbWbSvlc1T51ZKfLkK8NZBMb5BBzo+I8NiErhAL5C4EVmjRzzyPtS3rYiL59ZaTaSGkSZRAWfI\\\\\\\/26D6HI012p7O1wpKM0\\\\\\\/opco2AMbGv4Sv9jFquZARBLt+jK4FOt4kGB+IrNvRNKwPPDDtJlXEcM8+mJcsolF37raHdD8m6OST\\\\\\\/RxyOhAen+WcskUK2zbquugRyswzlq+2dEskFIcrgxb1kCWYcFeArVWBp4XlgEgbUKfM5PijK0DnQFMEbKVyE7FmbVH\\\\\\\/Q\\\\\\\/axROhsWIHx0t+Wc5Dxr1xJcrTT6OvmZ7xbz3Ml+zMXSSUWmAbAGQ65ugDphSmLn5y2uzzWra+u\\\\\\\/1CO\\\\\\\/SUoPqRrjVVvzWENptAsm8TQiN9K4U5ROROEdUmfJ7FUBYSRFhjeNK1VrXvsIJ9gzfEm5AMfUPa\\\\\\\/qyAHFSKuQ2IXWFQmB8skHT9OZryTilnPwHnI7v1uobmE2aaxEvsDB6SvGybVcrQ9zLOf73U51WLNzLJbo10fYllwglDRdHr6zsBofkC88nivOfCMvSHHnSnAy9bGY4+IXL02vNSsLdbIXlgbUD++mNSCl9uIwMSH+Ox5+cfffPDoTa1TJwELK4x1YhjsXT1\\\\\\\/DwTU1UP13R2q2Rm2FRuFGi5DMkQxyZWsSCWa1qOpQlbVOCDOjh6Xl0Y0XIVZiFCCOxwLHF6F3AQbIm7+W7R2A7F9H1r\\\\\\\/\\\\\\\/rvT8rcyuOno44xSgnqT63\\\\\\\/OAoPew9i5LLywsr\\\\\\\/TheQgx9naRqOhvvt6n+YNkVNkT62kiIdMpZsYxRwEfMH40O1MVh0kHht1f6k+gRIOnJsx2TGQCbxBrUG6HSgf2rAjBOwArlq3kGzztwHpad2HywI8oeuJGra16OTU+qYjz7KbK5tMNpGRinbQ1Se\\\\\\\/ul2Q2HNXzBBITZFzEfSibu+IMkwB4lZjBmyrCKpBB0TG1ugmxiys0NwSQi9kmjUQYMK5dHBGZVC8ynn53KxszKg4060kC86c5gwgE7wNWe8mGf8kaaxMVDSMs7KizQpdTEh7JgFstTfP15nifjKnW7UJXrq4CE0KMi3XWsiWuFXgN+RA+8phudb+\\\\\\\/ZxxiPS4IuVGl+3EaHhrB2rxyF13A2x7TkLbE6fNedcHmx6HcvUlJQigmWAIgU9qS3RydZ+vxlB8+7gs42OgbuTnQdXW8QWJpxnMbacCFYpog7JXAeAL2IHsuR+gxIak7I8r2XKfTAybNF55T27ACD6wuHj8hkHWcr\\\\\\\/3Q94IR8bmPLg7caR0yrW+DxRsrChMwNhkXbUSg0TGklYGIWDZYTwck21TyMAzDtp7siEs4tzjqcT\\\\\\\/4nq9pa40VygeMNhptBsD93IHhFxhyEOZbnM1CqLQWm9jRXgx80gbzdhQV5v2HrEoiVyC2s8iFrx4AVxdqPHIKVEtgNVeRsX19h4oywibl4yaRwDhYl5+KFX5\\\\\\\/m1H2ebGjO6rLjvUQcr8G7SNNOT2uArArWcMMC1CazfHKy9Ser6lrkVjHOv0UN6dVYYIe5pFNnLJIIdqIosYnI8ZHEYeUHeeC2WXfl+5Zv4Bv\\\\\\\/+IFIJznPKlBoFI+NxAkxLIB8XZnWJIgbkY4eVqNBKIfBgb9snlRUPxwk6\\\\\\\/K522r95qfhknVFAyJG13fGms49xqOd+atrUjsddK+jZ29JZfCKzcw7SZw4\\\\\\\/NF+UugNlYudn9KSSvVLaQX5WD8nkkwpDx1FK0wOr62jipp+IaRbVfLVJwop0btO2Zvjv8u7YpWWN3QLUNeyeFGmdrJ2gGTzjFWYasel\\\\\\\/zWDtHU9ZuoO9AgKfAkfWxrULhzm0eUiUt5wJZ2f73Z9bZjx9V8mUp5qXv9uLqqeJ8mRusa4s80s8I\\\\\\\/BegdgLz+NRvXXlTemcUi8SW1rVC3Gn2iu+9VySWK1UvjmpZI\\\\\\\/lQMivuouhzhNbbgT3KewdxMQCt3llYGsIJI+iRibei5kSL+C6wV6M6+O8aBhWJrjo8SW9wv5jVLvByhB5vQwUpOJGzjntiCYJ2+X7nTSwy\\\\\\\/aGUXOC6Y\\\\\\\/CzPrHJibtbCBV0zKLoxKVb8zjuYVRLBoRBR88SfsXv0TWLdcrVbJK8WE8mxRM4LfFmVxaAxbkl5jZjfR50hoKK4HVJxlQ4SFZXg7J3mrWwzs1DypTLSqCPLME6wjXYipSilC\\\\\\\/Hqy73DcbZIdum8WgoQdT7r1lsBLqZKshirapVyUjkI7+KfBUFONxeLy4grN0lBy2VDMZL+0oitgboSDbu07KtXeiurpgXCwwTxx7YWj+6C\\\\\\\/97qrV3iupeSuL9\\\\\\\/B3L89RYCwB1ZYM\\\\\\\/dvVyHvq7UZjpd2xVp4QszFmzAo\\\\\\\/l\\\\\\\/oOyATGgkEEm4VIyEUdexIVrH+6V0ZXXJOQXFGrb9pSHRK2EdyFe\\\\\\\/zxkEHliKmo2cuSkkXQWxBDcH8+YS9gsa9ykVgTiBwV+B\\\\\\\/WO62At22ZsA\\\\\\\/5\\\\\\\/rEy+MjwMjH4eVJgGdgi5Ho4N+Ez+pSt3tbTCtdRLR8sKQs9QIm2LJQhGhlYkWtPdJYyuYK4ePfDDEGwjWn578stcwA6cbAXvvYlHHpXvWrhkdSNwYo46X+IGlX53fRxxbU\\\\\\\/lBXiHtXwSh4dcT9wxg8Sg6oIIfO1a\\\\\\\/MqTsyAXt9u8nSTBqz4sAuiq49Rt3teSQLKWuVbg3h409eeZWm+r1PsKGTjYYRsYkWm1r8D0IGdNfuNwwyMWM0HEdgnFLk7lbPOXXbWb4NiHkgqf54Dab4dcgeM+n9wehbt3+fYquir6TudNVM6h52VKc+\\\\\\\/DqWVDCdfldmPLTg7EL5x5C0RKvRKQpnS9A96fLacNWkAn9fhSXfZqHmM8if+QIUjDX174tZ01o+7CrdaXP9exaAlZtC55fAhaRW124dilPMujLzg8AfiaPxrjW8LaF2J36b32XwTofHSzry6zNFmsCE1HhNQw\\\\\\\/ppiWOlzMIwLbDIQFqWSc2N3PIu0VrIOP+s8jT71dzhkZIP\\\\\\\/z8wghHCILTVPuUCCjAcmEVBAx5JLG3p8GF92pihTFZi\\\\\\\/OCuWmXNaOVtNq7OGcWTI+pHUifexq9FXMn6vvbkXv1DCuO+XRf+iyr9mxRNuJeBJJHf7Ftr0UQWKu4jK9OLx77UcCrC0tQSzuP+s58khqpMODBMie6HMGe1z062Bxf7UFQtidBSTSwrBQx1YTNmk4x7eZnPSFDctZNy0AtuEmGdU+k4v5NVsy8qL+Qt11\\\\\\\/qamxOQ6kTNPjg50zNxE2jzaAp+zfqx2Nn1n\\\\\\\/azWXtXJBClzOaIgXQjKMaThBaa5wL4Q1KzPOVclNX9+ilImydsIkZHDMfsEV17Auja0s6TWdNJg\\\\\\\/WV62sI5zTe7EPU7uxSFXIFkQRhUe1jMwTxQIdHXexvO9vegaI3YC7bZAFn7sPD6C07yKcu9zc+Qb10sFCY4ykzUpQWQOIsm\\\\\\\/j8HaLR00nn4VpR5dI8niJZgobLdLt9sEpuaZsi4iuAztX6HcFiBXchJL5m6mBVHFVdWanWu1gL0DXToUcLMadOQ+64QwUQ9lwL48fsXpQVV4Kn54NEJbalJA9fd2kFJzGC8+8mzrAclFVYAi+qo6Xphp4WUtyjpB9ZzW47cOwBm9i4jTZOrmXNU4DldLsMt622X3fQIu5yZ5Pt9EM9wckoml1FvwXeMcMhQ74mq3RrqFHVxKuNn5hWDi9Nyz1BqvbdaBI\\\\\\\/EahnqoYulur2FDLZPvC18nVG8f\\\\\\\/izLojtQmDD\\\\\\\/7vIdb3ndu6P4cQMZmYeeMILjopzN17+Jrtk+rVvl6kNDsepE2T2JJ00U2C7BRyraeQXCgLdrNCbbpGn1rcSnSMmqvLATUQqKoZcLSjDs\\\\\\\/t87gVtXZp+irsnaCnPK+SjoHJ1fnunyS7RREIqgDDI1K9gG7iWRhoMAXQByvfL4l\\\\\\\/7zOPTFWPw6a2kOxsked1NRORf2izfo4MPcuApdz2WbobFXvA9mhXrK6uu4o1y+D2M77xn5XnInZbBD4S80t+VhQHD0Hj0pjUr09kFTpXJFj\\\\\\\/+MsWggGgHKI\\\\\\\/OBL\\\\\\\/Gt98y2WY\\\\\\\/IUgXIxLtauQxAmxia\\\\\\\/wqB9ZJExU7Gju0qeZwEhKqVvD0077tcz4wlDg9FiaRzf8dz7KRU+DisTeUbdqAYsxz8xrxA2v6XYfD24SX4d8IPdVIkS38Ek+YjEvP9X7EjkFegdhrExcMZpsEUkrJqE5FMzuL\\\\\\\/HH4\\\\\\\/NOrTuGcS2ki8TJTHt7464KzWfb++bOvu\\\\\\\/npiNt1bFBEbPvQfl7J7YfWauRgNVPa+37+Oy3NFB5L4k35xzHdbiOdHS4yeZ5+J2KiLwC8DtXFYXtIEwnlAEshuSs4csxfCHQRAi86hpmD5NA9wqf2KicJXfZGnaDtE8BD\\\\\\\/EfyUBeb8hKRH7VXjS+OwdiLYxP3unPyg59Pcg7I7LvU\\\\\\\/rtNWTCXPi4qYYbdrE5+VnYRPXXWoba44ZFxdDKQea1AhT1nYBqPBo+ISvbBheb9DQSVWGAWEuwwNzG+v6oZ6iv7aGCPvAA7LZakyw929NfU2qp4BxWfPN2byvdgsoCczhC\\\\\\\/RyUnIhlEvZFt8KdH0jdLxymIEVkC1g1Xxxdgr3CVdWkKprxoL+SZrJbzo3fvvccxpM+kGb5Noa1t4bN1OSVcwHb7Ryjdzi+hVoouVbG6xOFcECeulGzhe9HVeWfCjmexy\\\\\\\/BLq1G5JJF7nw8K8buSQ0jdRE38I0Tcx7HKP1o1gw4\\\\\\\/OxvNEcg4jioKDRk907nUCTks0\\\\\\\/E6NGAayFA7yB\\\\\\\/+9vwA1VW48EulzNQ1tOQRMDrueDMzhM7Qj5IcdMxbDYtFrnrXGH6ZIWokkfhDGDAu2LhMxG1OtLA+QiNzVFCG2Ub3U8aXayt2vOgDmUXatAz7v\\\\\\\/lLE7a53I\\\\\\\/zOSsii7gYWBtEvSW0Bme\\\\\\\/zVEiuk8oX5\\\\\\\/7HV\\\\\\\/3dmUEQ3FGCxcHYg5rifybFZjooTT8PtUnl6THi3cNdtkGuLHyy4YhitOQy2hWzSzHswI+QzcOG3HTVwNf1by1m2OsyOhqbLcrRbym+dt9V1ULl18fkxviGxBbZ7CHQnKqd8160\\\\\\\/PhZkGJuFG3Xj6hbvi8Tr\\\\\\\/InIPSoqwf+k19hJa2JpPu141OcaBbF4E5ZaBu7jtFTf4TXO+xFUIBIaXd7h0gBzDRVgjVHedr5NNurT2usoYhauhsod+A\\\\\\\/W6hx3u9+DiFZvhykNa2P9yAmEvF5EpVhN+WSZ4Ziv0GZZA6kdFfirNVK+vj06VDSbsVHXy6A6atdTgvLVOVtJ+Zdn2FvuJUOKeJnW1ZTcADQtWExbB+KWFG5ctWmfDolU8x\\\\\\\/YhDtWxCX4FGg3XHD1JEKU2d8RYAtpJD2uGhcH6ZhbeulAdPm6GPyH4QcUN5vrRxiCIfy3+FAG9ZfmhsBt15rQyWFLUAsP8TvY3sjlTBEbPSlZoMtYk+5vVczUhfMbFXQsODCLDDOFYDr5ZSulHlD2BYjImZH\\\\\\\/cnwaejsqh1LOUGEhHtb8F8nqtwcIR4UKdMAQdHKoaOXxL5w8iJ687CzfJQtO4Ydq82L2VTJmJzpgyhA7ytWwZtDGAWLM632qF430Et2ID6VHYqLCtJ3hJiB1LJ+ro\\\\\\\/VlxMJo2NTirtWpA6gBhMfHhTzOmkIB9TR225TehxNCHtl+wJwsL4sr1ObDFafYB3vyJN4wCZmfUdSrHmz5Vdy8KnZ9tAfDrfa3+ER4PSuOImZ4iXv+775xlkUubrnGcR8muY3wao33NN53pObXAN2g23vU4WnIEJX+wFBpQgoFmV5odh0N6+X\\\\\\\/K7lWPFd8aNFiJpuSJpkHt1m92sv9z\\\\\\\/M1p2KE\\\\\\\/YFuR9wGhmpmE\\\\\\\/4szLl8ZJ6T636uG9NcepKW8\\\\\\\/ralOx+fC6TznMQ7jI3pUGhGOoXjE5MuHxWgu7DC9SjiAv0o5AHPrV0z+sRlgjtbw0mW9EJOiG2cVADuWLkFFGuRI7K7As+xyG3HeoAAM\\\\\\\/IGSi3HbgkVMigSTCakJpt+xzIXcseVtCq9M5SRCCkexvfgPUQ7PBU1KTEHdsKS6lhSHPhc3bXvYyTNPbB8s0WDlDq2rKEFHhp6WmHwVWwM4W+ajXVkyE1iKLTE\\\\\\\/ACt80UUoDoW0QoZjuVM3Ac0frb10Z6BAWQxAjaEce++S4ktOQMVRP\\\\\\\/ybgwb43wtlMXwAhtvCJYbY+avkImH7WXdooOE87iq7GvyzuoQeKn6XosqKv6UCnD3VAsUARd62y8ltwYh0c6Or2aBlGFX8isBEcNFi\\\\\\\/mMWdCEYz8Lgb+G7XVJgMzIZgEHT6hrLR2c19+t80NE5Twy2CUzLcBvtmIqzGmaV7I4\\\\\\\/vs6HLUxgOyw4tlkvg0IXF+kfldEfUrYwdIZFYok0XkXEV\\\\\\\/Que3SBtzqRJm7BezU5LdgvL+jLlFEd4LxBlvNUDnXqbk5DNa0lz6OIbddj4mdvZhpufHOyZl0LocE9Gs2hjN7uzOPCmFmXhoaMUiJDeEpbWTkLntWol\\\\\\\/nZMkSTZ1Zt7dz4qRy\\\\\\\/75IsDh2CZdO1YRMAT6RH0bD3zN6in2HRgbOL0w+ZJesAYAnI3qdgnmrZwYBUREzUDNl8CV1w2mAMR9AqB7VXB80sHKMRmUnVjQqjNuGNEASa4cGzmYUCv+WPvwVR3Y1gAoIOsWtPOmBs\\\\\\\/cTIGoz6bnOO5D5JVHGcttxueg5BOdY5MAIKp0yNXRezrwWHG6PqCz99wpd068pBCpwfOc7lMcb4YM78H1U6olj81g71fMxa0gzWc2yDn7iSs7oafkayqMuj4f3RV9KdMlgP+pmTFHM7Rr\\\\\\\/pLtzF4inZNPerMTlzEVMxzCMmravudG29F9CgW+4WYLxNLxZib2uEO2WWsdCpyizh3efTNt8c92y8yM8cC2QyZApWjPchQWPwGnqZrbkt6II+1p19Syg9ovhdIoXV+SIBthgKETW1E9\\\\\\\/c1UFp1KpOHqPdkXyCRbItxsQ+Z3Colh5s44HqS4NwmgdhGgYh0H+KULrdxleIgZw8pwPFp2bWT5+GdZm0zYRrTg8jv54n7taokfluJeX2rx4ngH9lHidAvFmmqTPz7cuE3ojxXKDCXOIYs6utzh2VAnW1k6g1w1+ZlsZcOEgRWnvNw7GsYJ9c8lIbWtqDn5jDse4nTLrT263bhMJ7Z7ClHVsD2BSGVy93or09lfh3iu4of9IDZ9Q7ISO3phaQ9NRIBPL0Y\\\\\\\/636wq5tZ8cz+1VYQFHzB0OoHuxrZPCRUdIRv2ElY7iB+4JXOKhxC5Wn2VlcHOsoNb9Dtt1+6O2wisFFoOUhXlhUovu+x9L5P4HFgwecIdeKBxYA9QIjlVVNNw7IflUkIGqAEknsE6M8cOp3BSC3M85s3twRcgCDqTrw00hapzxofKLqYFRo7P+2TWWqDpxlT9+tCWK6IUL2ZGBOxbHJLrUvsW+xXHx+\\\\\\\/2IfHkPj9aTgn323HdjQFvkXe9bp6GZSg7YXJR\\\\\\\/gDGvwehljfcDZRBMk+6P0IXN6pY7mkTLuttLYCyYIpzNNipUbLiyx+zBv6vuIVxLX+mWJZ2+9UAC2OFADEabizP44qKHpNH+jv7xdbVUEjNpGAcdWorXq1EmNrpXySzGAjAaunigtF2BkqX6YJyGosfXiv8FRCL2BXhHo2saaOV2dlvNma1\\\\\\\/dUB\\\\\\\/ECTXH8R6nH3rDVYhluB+NSik6sa\\\\\\\/m8w7kuheUp\\\\\\\/aGrMkLyi6Wvs+aKGmXwSFynqxjZpxzKNN9lPZQABtdfRZTWDXQdYYsu9wv4p0CpPNoVYQIFLVrmAzeVYmLbKaChSuwh\\\\\\\/ty3ZC8Y1h0j1lQSjWv8+m\\\\\\\/vt4THqRXEw1xjkW6Yc4bsm6LZeArGsW5wAW+e\\\\\\\/6vSK1OFKYTzTIDukXSBTp7eUpRkuLu+JiJhadC9NjgT0X98XtKmAwU8Rii124hQkXkhCqdRgYzgzc0hr+CLsn2vIIBT4dDsCQRIeA98UYvelruWVnDHO1IAYQ\\\\\\\/EHAn5y0c0yrjT2yQwvvvnEeCHC49nHkU\\\\\\\/bKWSalK8HzDjZT+F9gs3ggQPSrOtIhFUV620cOUaj6LS1meXm2COvAZGsdpkEcizBDH7y73EXuzWe5ArG6qnAOuhGpkbIEmmvv\\\\\\\/fpe74icChcdK+m8BLPUWZ3yvhdN\\\\\\\/QqeOdbBs52kJ5rVbVtDIntfCCdOhjkjQBdBekpp12Llvq1FytQzF3ArE12SKH158NBv0sGozAGKgi3DWx0Ga+QB0vaf\\\\\\\/TaMVL2ugUt7UGOZMsNx+3B5Ce+sfadu9mZAFPGToK9Adax\\\\\\\/SgQul8s3wDOy7Ru\\\\\\\/y1RExYrFu5j2k0fLuO\\\\\\\/HBRBLuWwx4nPZZHZb\\\\\\\/L90BcCNzM22vO9CaRtzMjQgHhYvNF\\\\\\\/IS4D3hD4bxCZKpgM1\\\\\\\/G0Z6oyh29A2ArjvSqciEyX43PlCgSpA1J5oyq7Qp\\\\\\\/8Jo\\\\\\\/1Wnwnk6ZaEU3XXlwfTUl4ijLKA6EjS7CDm9oJCe\\\\\\\/Pd1LNZMBqjBEIKtv9xPhmdNTfxrIIfUYXg9tO\\\\\\\/wRdeG0g827YvcOtjjYAIoeoy8lIDhWt7dTXNhvEowKDsr41rDS88\\\\\\\/FUfBr5upP8WBR9YevqTZkfspEp+6tKVou0Aatl93oPn9F70LQ7NN8p71HimWOiNYgkiePDBLJcmOMzx50l0AG6mDRLLNjUZRmzcIg0bVxdPCj0S8wOn0Abzluq0KQrBiHAU3FNGexkQaYadygr6vg2lGOlvTQKqSXwvkTOA1Sx2isfr3Co0KvxElqiLoQPB\\\\\\\/4S7LL7\\\\\\\/+i6YFXOTmksLW3eQ\\\\\\\/IQUEESSDkjdBAIjHtH3wNwBMO8JGm1LeJXC3\\\\\\\/XFwSeAN9BHVo6i7eWE2fUm6gGDcp3C6HtbOo7ix0cS3I\\\\\\\/r7bNssrakf\\\\\\\/fDY6jVt2ZVy0qUYvlJR6YXrRSvWxV6MdeHzpFrwjhfY7a3U0hXTtYWSe\\\\\\\/uZ30IV2+DpJWX8pSY2fe90rw5QDdAJFYQNX8qa1ANlunCUvHHlQ8Nqm5t6pGWfD1Mbfwy2SKSxvf\\\\\\\/5hQlp0mF\\\\\\\/N9ISJb08F3Jmqq7pmZdblX9J4q1Anh3nOLPbsEaEhK8UbeEbnARp5\\\\\\\/A\\\\\\\/TW8vKZZ5tWZ2sC5YJ5d3Xg+dwCMMHr9zFuHYG9sxbJ89IGZczfPPRlpBRQBtcgFAyxor\\\\\\\/obfIm+FMshWqPAHdq32CbFUqdGhHmgPrPuNeJwsIA0Xutx+g\\\\\\\/NqsElqd45RQwC\\\\\\\/cR7DdqQqj1bbDvgVPiRpV+m1zs18fgo3bxDoTMmx28HzOXcrIy1AIb6VUFhqNjgjCce0dj+Bjat\\\\\\\/WS8KjDvhFhR2h6Z1g59aTcI+8TLvKrqIVM9rZ8qIJ36CuVzRTXVlMKbHA0V2m89yWL9RxyilZZKSU72dDi1B+0BRQC3nR7i2bYeqH5zggtTlis8rj4cgw0jlap21PhDOKDqWPO8b25w3i\\\\\\\/kShzvnX1NoTjOPEVIxgVNWLXuMGn4vdzfsb1uGLI+ZR1ET9b8LD9BEXJ6cfJlVgVrFr4MzlCOEKFzE57PhHp0n+eZUtiXQ2KWKx4hPer1iy+mFYiky1Zd86RLnoBpSc5MbtJAL69AiwIJi7DX+kCc1h+6tdhSX6trdpaxTrry7INK1xX47R714rtDPvf4+D+c+TSqBzjh6nczobYJlQ38a0rCB7o7wneQ6yUX5KANL99lFQhvtEUnyyenQ3loR+GrKb015+pVNRvnfjOCZuzEtgaVhIeeEndRXMVj4YV8d5LOby4pVyFvdqbdIhbFBeelZj7VuD1Izc2C348oiOy87a5Dsd+E4IUtvcP2S48z88HMTXIfXEkA435uB5GqIjLYc1E+hFL927gK4EzBlilEnHqybX\\\\\\\/iXWiPWOU3oF+d0fosjI\\\\\\\/e0m1dcc1uKph8XvHvTV8OFWQTYP9gLqeGaXdBhsFl8ayOrS3oqi9ycGHIwNZjotLgeHhJjJ0\\\\\\\/SKBfN0NyE9VjCQE6SVxIikracv4bNsgI1rYpGN1Anl3o0zRLkiXr48gHFekWC3a5OLEl9dNh0PFlE\\\\\\\/LdfTfjd3auwgWJMNP9qRfSkWqvK1QbdQU+H\\\\\\\/BucnlqQ2pCJmjqV1IgWqbIqKUOfamplneIeXP846lOWjdgeW7P2wnhmZ+HxyyOarzoSorbts5FhaJkKWkH2fBLpztLnWqzpw\\\\\\\/rEp34ONh9FpR+j7VvFp2I6Rmqix\\\\\\\/pfmZyR2cXblLSZh5a0U\\\\\\\/HVnHRYhC9qrpihk5c5Dhl4ffeEx7JYJkjsb+gMdmGlbCT8t\\\\\\\/eRN3ZpL1YpdPIE4URmhRdCpDpz7A3DJ7qN4Kbnile5eikbb5HWJY497kCnueQpikV11W4FFn0if5HXaGXk\\\\\\\/MlkUCV\\\\\\\/rLpEpehOL6Mvcwz\\\\\\\/EDkl+csyZahW2J9X6wtM+gTcZvjxW4OZ\\\\\\\/8crd6hNHHUJgajKS1fXgz6YsD7p364y1m7f7VPYEaI9AibOwIRZSLcGsAtIqRfU9J6zr709DDgHFDnvTm02fuKEAlJ4MFz2BZ0Yc8rjaegCkFGzIT155mnoUqUK5+GHoIWYJZoblLxz9Y18c65+CPleJe0KkMtdpTn\\\\\\\/3OtVsDsm58xcEplFkeGw9VO8T74nKQh2HbOSJbuSwEfv6Ur5EMg\\\\\\\/WwWvjS\\\\\\\/LRsuQJKSrcxPXxcJp2281lt6NvzKj6PNk8sAGLxBn9VRgroc0rjpLEoingwDJ8HFpMhOEOrTdmTBFsteacq6+UHTWnUkxjuyFaNnpok6F6+Q+W0MjR2Xr27oAcdRWcOJIYbG0AvqSm5tsNjg1ogiXglAEJVNT0SGMYcyg9ywHeUMRrBnTKn0kGBEmuf\\\\\\\/X\\\\\\\/84wHjAnno0CGknCH59K6x1\\\\\\\/JIY0MYNSu3fDTVdxhmBc9LOPs6TQAjLNKZw5PToBhM4CAxQMGau4FdSRY9ZPoF1IQIxx4K8WcM2DvC3BndQUZTxoVf9UG5c7xUFe0\\\\\\\/nUPgcR0l2XQmtoqpHtjU2i9H\\\\\\\/N6VCZ0KrHiMz0Ugo5rQLMAjQQJhZMCwfLxMSxWwVUlVAnhMsT0XMFUuOxXS6gfKj6J2sf0YkMh5DazkAAO2UXBV\\\\\\\/28Ma0SFu0g\\\\\\\/Nus12XrwVRUzKjweIcnOnLb6KIbvMyzl49Ds++P9eMJuSQ4C3ltR6QEI5PHpuoSZzIzLAlpS0P46LBxR8fLWy8MJn62RpT8kS+IMXSNwf2eTyYRyb6HIC4inrQiDyF7KDWhSXDfGfUsfd682fm3MSlMQlSRNu40D2a+KjtyaR86WVJet18IG\\\\\\\/TGr2WnLfFZx4EZK1AZbJ\\\\\\\/tfEDR2pCYSzx7RfVXQquqZRIGfag+QwOYJjr7TBemOMtQP\\\\\\\/zBQ8YLOUT3UvqXQpkWuGyoBwVCUx7OYxBH+40OGJCYbAkAi\\\\\\\/GBDQqJJmz9w1w\\\\\\\/dMb2G5GdereMG1zprAPeMiKaqGSJfUv\\\\\\\/AUkD4tIhEF4UEppyRd3uGJRB4MKWx4D+4LHQ5clJdZdvygszaVdQ3Nr5qTmDOURScj8MfKmxzLXJHjkwcfopnMzLVVWPa8VI6FPK9269QZmK1nVth4Xmo20QMve1O9msaU\\\\\\\/LHnaLungqoFf7W5YE3kpi\\\\\\\/HR0ePOAjehYQ0C1klcb4ptDnf1\\\\\\\/Fkxpq+HzXHi+Y4vZPQtQ+JvUkGOuDuHZ8rkRhIrIX+hVndDPloSgJxQcC\\\\\\\/AHpYY9Z55at7\\\\\\\/3pZwitlGHMiku\\\\\\\/5zlimUgnA7jeq6V6\\\\\\\/7xfn24a4+Bd1r4B6YsFGxujLbkqfrGg1Ao9htiGWYsJQJhqKw8HTLrl+k57H5YnqDHwXFAYOJPL68zAHV16XFq4pgxzq7pvtm4jlZM6QyXqFgQ9kTXMRCpwK+RaRsXIVLhljr7mBxM\\\\\\\/2vr\\\\\\\/kP4\\\\\\\/nCE49bMHGfLFVfw1PArMkiypfNo2fhVlgNsJJkt\\\\\\\/djbX08DBp6t6ZvQpJKUJWqKiY62C56AHNVks0QP\\\\\\\/vUUQr2pT+DBuFlgHFk9L7xWnocr4Ujxr+InZWIGPNSPrhZIC0gzWHt1g6DPVZkBpV4ddFNL6IKfalSqJJhBNTv4jYsH2oH+YRASg2PxWj3Lyg5e1hfnYpoN7eKVk2CLHxIwP46JX42Rdfht4sGdulDhg6bmzPOC0mSW+OFFgpuimiDnT3w7NVoxkx1s1GFqcnsqwj3pN2PlyBE9CHfhh8odlAKiwhtpBmT56zWh2hHLovCl0UEf3+9Y9ndKfA5in173032UctOd0jwNa+MvBUNBP2RlGSxpJvPVBFkvqKOjx0ngXIv7SGw0ZiA7gGtoDs17gTQbzE8\\\\\\\/rJ9wlJ3w1EVjBMYQpplO9bXKGmAMcIrhlyuxDVnH8ioG1sxdzGaw7LrtYqg4Yrv\\\\\\\/JqGvBbxQyVKo5rh7D5yZnKRxC1bULea9XBuJJDoffE4tDTzcpVnHXSmL58z8KaqxT8hka6XR72AOiAk4ltB+SJFLsDfwdw3MiKZFqVtc5f0gYfAXM2CsCi6CpUa8awybrEjfIs1U+OOXEo4nY7jkenfVEfN38Fn0waqPKJ\\\\\\\/jT2j3kLrG9+ZiW8S\\\\\\\/lCr9WMUWZDSN0D74j1H4kQXIy8Jxu8wS8M+GnoChg+KtomKkCko2rnm36IW7WN8Lado3YhQlUwihYagAUyy8O\\\\\\\/XhLTmM5knCe\\\\\\\/aJVEqa19ryIjRgJyVLF7ziRR+syBsmqKVTddWLsQDj+0uJc2jd6kSijEDhZkAJD0UFVzJFWcHgrifQ6ruKHR73RGy8E9YXRb0KSDbcrFKcJViRMnmfkC6q9c4g6Oz2+sKE+akuAtxqQ1khG2aHjME6FFiWoOcgy2oivqFtjjrSDSMRqtG8OyqQA1UrAmu+d+S\\\\\\\/h\\\\\\\/J+buMV3SW94zisoogWoFHDNy\\\\\\\/A+YLdj3Vx15myauwy0IhBK0PrIFsLSjzBmG\\\\\\\/mMDLPWuISaHfwNGTCw\\\\\\\/kESXcEFX9qFY3jKB2R4pqV44lMKNlxDGpTbjmOhBiXKbFGACpRl5WXU+U0iXyLQRMnCfFqNxWBofSxhrcOClvLnBY89T6qmDEB7JnjeT8hPH108K\\\\\\\/QPHrnff8kWb5pjjgmZ7VKbkNTVynuLPfQX6QdCOW84ihepKsSkwtw\\\\\\\/VrpF8yjRHd4LpJ4VkJJ640VIYBrW9FJQHJdiP3g09WdPeoRn6ShYC03TXdll39LJEtxTlxkZ9Uy0AaDKQflks38Y3t1b+gai5tltZcEhqBDQfUS8yEiYY7XVP9QFR0dc4PBK6iQbmTOFEnQapiaMikhxpj7aZf81pkk09HIGB3db1Cz9g2pOq\\\\\\\/X\\\\\\\/NTy+eEvV5dMiw2CEE48oRdrO2+HKmCQxzsksrjeEqd8E\\\\\\\/hn8gR3oOCrHNP52919GbIuF1XFUev4coldyuzI7TAoK55JcEHY4ozDHjaveQ6MdJOG7djCxvjbd\\\\\\\/WucivI6eVXsuPMFO5xZVUk1LeEyHOafFcqD4s57pQ8ZMO5JQXnbXyP\\\\\\\/l78xbRysQPtn7TYO+1VxCIsJqlTl6FaEgqpyhevliqvMo5LyQ8ENmLF4gBChNfogIcfy16dBAI861HLEaFuUDcE898N\\\\\\\/DmOHieXcpqsCUvVcf9vXyXOhcGmNxSyV5EfcLiXXMwLRX+XSa8MpyBXupk65mfPEibbTM6LIR+BAfAMfF7WYGRUoWEN3EHMcDHaHkqFxCvGtVoctDcJK96pGc8enUXLu\\\\\\\/2R1Wm0UR3HOLJMekJh5pcNaj7A8B2VNpFjpOO1zylEjcqH538qAiOVmdG+rimuMAEeomEqehbwlGeVvu+PeLGN0Ob8hjhQJaO9OBEunKf2h0iykWNHHt9fhJLDCQfuQD69H5XiwvZU+soW19BBvx5TdAcx+GHW54RS0jegMSIiYyGsFTxmbtoM9Tz5uNJHRDTSrN3ujIuMnAuGBWhI9rpJoMC3Mi\\\\\\\/p3HC1\\\\\\\/hAOb7pHJr8EMCT3stqBmnXiIqYoigLnFHUtMBUi0HZNA3Lg4FVW9a0oGyH20\\\\\\\/xdUOXFbeMJEQAck1WjtXoQv\\\\\\\/OlQew0JiXxfPhRyYp6GZIXL+Yfxaq1bXI6OqNGAoCeYlcj0uAflnqMCEeCSx0hgXEcskpzO9Sat3VKxg+eUfN95eO1PyQ3MhmEddSbeFX1yUHShKU\\\\\\\/9sLU8Keuy3go3rwg1q86Goz+FhK1NIkbxKjx2UW+f2o1wYBgM4S9urhCaR\\\\\\\/MMYC0JKGcZlN5CjpT963KqAJ9qWKGYUKSTEt5tpYDHneseUIQpJV2BSM1PWxkbcXDFqVzl6bM8fUh5Q52nqv\\\\\\\/GYpbdtV4wu3fD30dgiVecPuLSl7yIJ3iwphK+KU9BUaojlJ6raLAbVUEpG4bbUtA4FhoeOVQVjIHBEwu+PMgEVbr1RTk5r6EKE9dpkMKlyUEHT+K4qdQlbxj+0ALZf83P2TrYij0clOMwG4OmJ979LuUzoqn11m5S6dkJJcZl8u3GT6QSQRXn1mO7\\\\\\\/cD8TWZlHEAgSrZuNHu3W+sO0GNQqya4xwhKEjq\\\\\\\/dcK8J8WY\\\\\\\/0xSsax5xOXW479WeQv9YQ70puhDEXzM2mU4gsbQku2DM9JfPSmyjBKiMs4+AvA\\\\\\\/SoswgzpgJa1WVi3HAALQ2BKDu6UjxMNG115VLYegRbQR5wofjRmLUvCSXfxGsyrFbEXnZqWE\\\\\\\/g5Q9SfGXdbEcau7+j\\\\\\\/SZohrG1ssQt\\\\\\\/9FxFwlRKkLFSw8A0IgHZ\\\\\\\/KygbNz5ujoaEaZBzjUbfEwS23QGQmXHLuBIzz2edcEXNEIFN2ExVNK8KghUDAvfJzXghzKw9aoQC7TmkJnO\\\\\\\/cxMtBSGey7ITlakHdk5gmIMy6oB0mFjapgXgklaNWObmVDrnV7\\\\\\\/yw5fapnFb2cwaj54\\\\\\\/Bv\\\\\\\/38OW5Pw5SBb7IVUHsO4od4231D5wHETAHe83OrT60SkeBeLqByo83mCZkRVZbZQ8TNfi8h0UFMhf449uaUC9Jk3JNR4y9oYkvJlDA6Y7XcyAGiUrbkaPnEqyWFl21n9Aq3z403duVoDB1oy8cdmlzJ0QSCdNWVUb0a9AYsSvamf56o1CdXDaQPVynXPOoAEvuxZsDmy6Jfsfk6yZRF6g+TWG9X0UyrrQLw3CzjU7owATJ22UlY6m\\\\\\\/ZnZhPPGjN01gg2ul\\\\\\\/fiae4NTud40\\\\\\\/X+HC0G1mDOy+0kvysTMlqzezRBYHcjy4xPQ4rqoVn3pVHn0VSmi23JdNwIqjaT8gG9Vi9aWxg6j6BeJBH6FY1q0WDzZZB0MZDoS300VU2o\\\\\\\/7HM3HF6u9FVtWCYTE\\\\\\\/6+FdeNzmHfwKcxTj2J8iqNDrnZ7Vug60z\\\\\\\/JpIUthDICQWgvFfYdBR\\\\\\\/FuFuRCv9j84d\\\\\\\/T7EWylgpUo\\\\\\\/UnVnPFtHSwsmTRpARhdOaPIDToEQFgAJ3fQZAHbRHyOn9TvlpgcZZ1PLQNGQwU0exwWGiQtqdzDrcFdiTgsq\\\\\\\/uzHplLvPhEjhfir38VIkTCWAqg9\\\\\\\/HJ1sJ+QQ5ruOp0MdFD02b8lUnHnv1sPizZJLHlAFk0v9bKt+TvQT1wwpzIfTayJx+ohmZDEWQz4fgbY7qXNqy9WcqA\\\\\\\/kg9ALcYHFE7ddMFQeufyiGWv+c5E61PXUfGmKm++p8WOBr\\\\\\\/tAeP5k\\\\\\\/kNstnp1MEg5zKByY69AthdBCPVc\\\\\\\/9E4ujOwFAq1EB2AFGCYCVfVowfl63cufITC+a\\\\\\\/0SHlunyl\\\\\\\/ffHChg9SDuxvWHbvVKQnvXA5QlL8XzppvB6q4rfvHlTGIrmzTgJHVtYepkG86ci+2cfazqtqHim67M2QBaARTFSLx49yls7jaKJVTwZ44DH0uz4Qs2mAenXaRe2vvJZLFIIMy5KRHt+8RsOuPT8fpaz4v7gyOOIfZn8DH4+\\\\\\\/xegC9O657GvzWeVt+SPDllA3xUW+pxglxuqogo7Q\\\\\\\/RAvn3fkmB8uN7a+H6BOYCPkgIL83RoYMPN0BeC9AbunELtCjfR4mJ2DWFoKlAKulZBraFwCYTETVWkoqv7S9jBcBChHTGoHVwGuOHikmjTv4so4eNrbnBRnlScpg+MN5Rwwj5\\\\\\\/3aCZ8qgrLYVPkKbuOSDneELpz4CmMg19UaEZ+rEs1TuJbZJsY0DnjASBw2zckEvebgnd0CqamEB6Zcr3hd56iStSb5BMGxxHadvQwijf+bdKfyCMRmKzfSKLy+47Xlc8rdHusw9HMPtvyga9swCbW6bMwLC5wvdTfymZ6CVEx\\\\\\\/6UmqiOVC+YeGDR6rJ5tdRNLlQEwcdr8qKAUhnFBc\\\\\\\/9tXEpjeiMlUWdqDl+nuGI2U3HOhlshwVDSiX3698dk22xk6vygrbSaQ3uGRGq49RjIzKm7\\\\\\\/tpht8sFIuPEux4puBF2UvnCh9ZwzaR+y8URzEaVA9ltF0cz6ha62dduF7ApVF1i25BgUQbVil2sz0wZQ4KQXyyQR9aZO1hylzhP\\\\\\\/NVKWtZapE6+mAq4KQt6d3TYoEvcNhcCynagSTQph\\\\\\\/b6IJ6XIkGQzHA\\\\\\\/T8T3bfSiQ0yJtrbFReLERadfAFNHiLdPxAHCGk3AaRn9MEQD2rjQ\\\\\\\/L6EH3hFt6t\\\\\\\/Z6RCbNoDMWR5OpAL3YSGTv92ECF3R1YXuXDJ2D306xJb1mViRsDPsImY6sKrBu5Y4sN88bPQChBXUTf6Axyf1fJ2gA3XUbPfC11LAWQHRubPldTGPXi91S9ay5DCh2UtcO3q1A9UKSBNp\\\\\\\/ptjK3G8Epvadmw4G+EPzdHCqtM6XkWBwrUTLqol9ScNUwDlRmpntMEV2Neol4ZKWPyK1FAYljD\\\\\\\/qTdX87M9qciYJKawG5qrhb7IOb1K+WCakJLObXfQ\\\\\\\/1uOcgTr4MFVi5TznL82aIN3lZSdV7MMKp47zVUx3H0ZYd2OQXfw2BqBAPWzEt7n5QB6lZBPg1o5a20tyGoorYK4L4o1qceTomio3HKuHR8cs+Qx8sSoZDhbGCdGbNHriKv7gqVRbwp77JzmobEu3em9S8GUTBGD+AWhPOShuC53Xj\\\\\\\/NZG\\\\\\\/onM3xxqwXbI99qFG\\\\\\\/4BQbInNbyEceM1QrJT\\\\\\\/1O4RiGBQbQvuq0oyhYfncpoHxJxV4JS5N6G1NqnebP+Fyz3pcb92OY65medrxbZO3sfVBCQ4ZkB+R389cNO8D09zNsvlXYSq8dYFUhztxcQ7GtwR2pDhTIkfD+PqnCRjMQ4vkAB+ZeWQBVg2tvJKS5x4cdyg42IJq3Db6k86oHPttKvhljNqu5pAxSVbpLQ9Un2vYrDF+tuRAaCskmvagPV8qV5JCIsfsd+KNlQQpgddc+GOh73Tsk4lEHWE9NmWKPhhGBnaV+4jihHgBZtyQ9VsnpQp1D6pUDW1dZfhQ9H5PuvtYTDRIHUfISMBuSynE7BGnx496e+hWDm+PHMFfZruDI09xtaLCodbM8wxCDxULgg2kAI32dnksb0fj1+v4J+lBX29tgQ8PpyIPkdYIbD7oAk9BgoHHj49jnRIYLL75QPj2WtEZbgb6iOUmwl4sbCkapEaiixJpXT9vC7EfIr7daYUigTNU5ZLcJf6DuKAYyzGdxfuvvNX4HtwLpcRbysU6FSFXVlBHzpOxBh8BE5AWMqfJUza078L4pykqwCcz0PpEKqbqX1g4DvSEyxeKd1IRH00aKmmjudXdO5BHsch9DMW8PF4PaWIHGTgNjD278fu6k4oSgZcHrzw40qXm\\\\\\\/8ex3h5mote\\\\\\\/7eLCz\\\\\\\/wFiHE4XRT2AGY59Sbk9jQshEQ5\\\\\\\/XiWGxH4Cba2FL1Ny7aiTDyQR96W47FcD93XtO3+Y2jieG2lwKsUehjwYKmT6gnaWqJ3S5StmS457\\\\\\\/W8e6qRI2V3wukPsyxqBF8a68W3KJdFu+futUgptet78hzKp\\\\\\\/ixk911OFSr08+cz7pv2CsruaMDMWPt8mA4bTkC+0z\\\\\\\/u7+1CqqHhSCGGRN5yLySZ65HGrlpKYzDVjHmiVkXwUqiVHmk4h5UOaZzL5Yfz7CxxrDKD6TE2wnKbZN4av\\\\\\\/k6Dgr3gtgZmlclXT2KrbuyZv7h+uBViULR1GyenROZqqY4lUH3JoijbHieLUDLyrMB2yJCZJosg2noaOOSFX52yB4ebgb7yA373d2vht5XVNl7dBS8H2x68HP331wQjAUF3rGYOnrBAbbDoYomd4+twk4O+8yzItx15OYBOluIy3VDxXzQcxk2rSvAUGKlZSM2rumSwiqcNRCbPilRbkJTdqtUYAtXZgvfbAoJRImj1HdUPFwZT+EMj+XCmR8riUq2O0WoAqoTpc1G5sqSNCXBLRDWqiFtmo\\\u0022,\\\u0022iv\\\u0022:\\\u0022Nj7BKk2EYAQdRo8vvFBQrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c81cb022fa4844287001fb2ef2c01ceac21d6b9634bc59756687c903ad747ff4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jNamy3CHrV1LHoK6+ep0HZhRrK\\\\\\\/0D8XaL2OR2rE\\\\\\\/04i7f6mst7hTPInhFHzkhGg5ey+5TA2VcOPuYRHuIfN7vNHy1SMLJM\\\\\\\/SR6JxQGQK8SgLV7ryci4\\\\\\\/ZBGGCkKAQ92tuI\\\\\\\/KUrwA110P0WA0w7lM3407o\\\\\\\/xtY+AZrC7XB18zWaKE7PAqByh8a4rfabZPL+yxN\\\\\\\/DjtKYc0Pz++GziBsD32DtwRe4E4YTFk6JhntryoRdfTvXhfMVt4W8NTCWQk\\\\\\\/ESw77KlF0NmLRDRRVAkQdfdnTsuukoUoRkPYSGepIk7YLHHu7n2JU9XoomrjzR938rd3raOhXfoFjHh\\\\\\\/egFX30vVan9QnMshE8oYSk3WaRKX2pG05Q0knLY3du2PAFCNe6thofD8uXDpBj96CScHItLUWL6woxShrfJcwnG86oS5rn7BuNEb3JYCHZjCSurCslHVqmfF6LW2KD21IsbHJahXAMtspAhrueTsGKrYuL0AxiGeWhq50U2I9wSIy38r63kv2soIS9ySmZR4YGNbVPzYQBFBioQfVv3AdEQcwOuYzTUb\\\\\\\/k9H3GwJK+uf++L9HarT5F7dZsTRtedgaG3\\\\\\\/6fVt\\\\\\\/1QiVC3YcKaPPJhYld5RHetMihxOB2nRnBN7z8cLbZYYvaQ6e5lLjGGS6Cyt5Fszp8s0Sw0KRsWhzj9UfeJyMF50KmCMWAcsaNXklcleA5R1oYg4DyGXyjq7LVjVKNI8Hit7L7oSJ267u0HxBOoWtUDhaJ\\\\\\\/S1R7N9vb7GUhrVGB3AxRN0Xv6VDtaiPK+7xZIfw3qWnECLJJ75zFWPtWwbRi7RPgo9dQ9kndJ1iP+SSNp\\\\\\\/at+XnJd2drEsvetj1zp8yfVzHF3WkVbE45jMS2sQwBHVXRUNroGGAt5lVGkd3KluEF6uYTYGrRa4ioeG3hOudEppB0v8ErTs8cw+wRYKd8w0hZKnDPbT10MAvJUUptRi2jn\\\\\\\/FRKtoIIP\\\\\\\/fsLM\\\\\\\/z5sEmp1c27CFMOw5KbmVJ+YxdTHSZ6Fdo9xKtbzdlHky2bL249KrlYfx9FX6\\\\\\\/IUGkC1dtZbPyIsFwS+c143NHWJ8VaLab7oRMuZ3w2+Pk63OpZh+eD8OpqGjkruX9kIXjvdej1aJeUO9Y1tqhQ4KtUyI8ETQvWfJxkPFcd+EuyWrMytNcGIlNISP2A\\\\\\\/IRRgWkMG4dXlakQ+3+uwNUkQNUj1ste2JxVdNrRzn0F8rOYYnrQOX0MC\\\\\\\/2nUUkfDe0Cw9iPA9RhlPPrK\\\\\\\/YMHUGxq0F611WOlX0XY3r1wBKrMhGVJXb\\\\\\\/G2L1JkB3jG8OVoVIgm8j3IRiq8IQ\\\\\\\/GYJl\\\\\\\/tmc6EgugXqOXupTljtjUPn1p54OH8uMFaKkShrbLUVwq4WvZ6C0lBkp\\\\\\\/ZCMTHsqwKhs6sP7+SY1CXqiRsb7jwzTa9Hi09g4lfTQ268N691w9XvbaeMkoXQ4K4ekbb7SfHXs76RHIxntXMSeiSdESjSyZzY\\\\\\\/fDh\\\\\\\/fwL\\\\\\\/PnTGUSt2bcnzSeAxLAK9WKjSQ3To4wNVzs8+0iMTWMYvUD4mNe8GbXVDrtaG65GFSbf58uNTUScZ6BHOwcyj4Ma4\\\\\\\/ksPI1fdIhrYCgs5turfaZzR22ONiS4\\\\\\\/1oeONCHPf491cKQB3yDOW2sdUiZ3wCV8eHzhjdejXOtPD2kpMP4MflPcnqF9nJV44Bvi2WmsEjbuOFrOcPh\\\\\\\/NHw4c6Wfd05BuD1XlPeBwwhl7lpuyY5nTCZV+p1K+beQehmAg4InwfJmeFjpm29gh8tzRqn3UhZf1+yNbybJ1jpy2gumgmTIu8p57NTR8EAbsjadx\\\\\\\/bDocNtjtuGuk6xlY3NrAiuUmD8wDrrXwxci9WV1yxEzgp02pa19BTB8x4ZG26bKY0xQ6CXssE3PMkcvyfTd8Uuqj0KgOWYy0lNlRDHUZOXMDNku9\\\\\\\/ZSmjnNZopEDLIYtm07uK7dEgBE1f9WXN1GRu0CZMs0jAXnGDr922c9fbTXcY2uZFTbiN7Y50KO61I3FpbZJ\\\\\\\/avzq8d7UjHpaLkS1V8nDJ5cy7pw9wJos=\\\u0022,\\\u0022iv\\\u0022:\\\u0022S4+qIzZIkytGj\\\\\\\/B\\\\\\\/sPYjYA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e4c06b739b0cba8ba1860b3d4765b498041e97632342d39b5fc18c4933272e17\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494290 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3079" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-skZt89pia8i7A6dkpe_aL9uzHtSVxMEQD087SgoJlyc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3079" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qzxjskZuf5cIr0uCWPWVyUgZbwnAi90_CoUfczxwTeA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3067" title="Greenlip Abalone (Haliotis laevigata) Genome and Protein Analysis Provides Insights into Maturation and Spawning" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3087" title="An RNAi Screen for Genes Required for Growth of Drosophila Wing Tissue" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494290" id="node494290" data-pisa="ggg;9/10/3079" data-pisa-master="ggg;g3.119.400392" data-apath="/ggg/9/10/3079.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Sequence Assemblies of <em>Gossypium raimondii</em> and <em>Gossypium turneri</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0978-4764" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="1">Evan Long</span>, <span class="highwire-citation-author" data-delta="2">Chris Hanson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6007-5571" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daojun Yuan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7333-1041" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thiruvarangan Ramaraj</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-3558-6000" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Justin L. Conover</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6429-267X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lei Gong</span>, <span class="highwire-citation-author" data-delta="7">Mark A. Arick</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-3878-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corrinne E. Grover</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-0274-5968" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel G. Peterson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-2258-5081" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan F. Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3079-3085;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494290" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA/Agricultural Research Service, Crop Germplasm Research Unit, College Station, TX 77845,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0978-4764" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joshua A. Udall</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Joshua.Udall@usda.gov" class="" data-icon-position="" data-hide-link-title="0">Joshua.Udall@usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Evan Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chris Hanson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BHanson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hanson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BHanson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daojun Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daojun%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaojun%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6007-5571" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daojun Yuan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Thiruvarangan Ramaraj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Computing, DePaul University, Chicago, IL 60604,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thiruvarangan%2BRamaraj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramaraj%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThiruvarangan%2BRamaraj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7333-1041" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thiruvarangan Ramaraj</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justin L. Conover </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BL.%2BConover%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conover%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BL.%2BConover%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3558-6000" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Justin L. Conover</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lei Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Key Laboratory of Molecular Epigenetics of the Ministry of Education, Northeast Normal University, Changchun, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lei%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALei%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6429-267X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lei Gong</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mark A. Arick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomics, Biocomputing & Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BArick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arick%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BA.%2BArick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Corrinne E. Grover </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corrinne%2BE.%2BGrover%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grover%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorrinne%2BE.%2BGrover%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3878-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corrinne E. Grover</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Daniel G. Peterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomics, Biocomputing & Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BG.%2BPeterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peterson%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BG.%2BPeterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0274-5968" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel G. Peterson</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jonathan F. Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BF.%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BF.%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2258-5081" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan F. Wendel</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3079" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494290" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494290/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3079.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494290" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494290/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3079.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494290" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494290/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3079.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494290" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494290/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Cotton is an agriculturally important crop. Because of its importance, a genome sequence of a diploid cotton species (<em>Gossypium raimondii</em>, D-genome) was first assembled using Sanger sequencing data in 2012. Improvements to DNA sequencing technology have improved accuracy and correctness of assembled genome sequences. Here we report a new <em>de novo</em> genome assembly of <em>G. raimondii</em> and its close relative <em>G. turneri</em>. The two genomes were assembled to a chromosome level using PacBio long-read technology, HiC, and Bionano optical mapping. This report corrects some minor assembly errors found in the Sanger assembly of <em>G. raimondii</em>. We also compare the genome sequences of these two species for gene composition, repetitive element composition, and collinearity. Most of the identified structural rearrangements between these two species are due to intra-chromosomal inversions. More inversions were found in the <em>G. turneri</em> genome sequence than the <em>G. raimondii</em> genome sequence. These findings and updates to the D-genome sequence will improve accuracy and translation of genomics to cotton breeding and genetics.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/gossypium-raimondii" class="hw-term hw-article-keyword hw-article-keyword-gossypium-raimondii" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Gossypium raimondii</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/gossypium-turneri" class="hw-term hw-article-keyword hw-article-keyword-gossypium-turneri" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Gossypium turneri</em></a></li><li class="kwd"><a href="/keyword/cotton" class="hw-term hw-article-keyword hw-article-keyword-cotton" rel="nofollow">cotton</a></li><li class="kwd"><a href="/keyword/genome-sequence" class="hw-term hw-article-keyword hw-article-keyword-genome-sequence" rel="nofollow">genome sequence</a></li><li class="kwd"><a href="/keyword/pacbio" class="hw-term hw-article-keyword hw-article-keyword-pacbio" rel="nofollow">PacBio</a></li></ul><p id="p-3">In 2012, the first reference quality cotton genome was brought to fruition through a monumental, collaborative effort using a combination of next-generation sequencing technologies and targeted Sanger sequencing (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>). <em>Gossypium raimondii</em>, a Mesoamerican diploid species, was selected to represent the cotton genus for its small genome size and its relationship to the domesticated polyploid species (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2007</a>). Subsequently, this genome has been widely used by the cotton research community, garnering ∼500 citations from a wide spectrum of research. While this genome has been a reliable resource for over 7 years, increased read lengths have improved scaffolding and assembly quality, while development of chromosome conformation capture (3C) techniques have allowed association of sequences within the interphase nucleus but separated by thousands or millions of base pairs along the linear DNA strand (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de Wit and de Laat 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Peterson and Arick 2018</a>).</p><p id="p-4">The justification for the original <em>G. raimondii</em> sequence, <em>i.e.</em>, its phylogenetic relatedness to the domesticated allopolyploid species and the recruitment of genetic factors from that subgenome during domestication, make <em>G. raimondii</em> and its close relatives potential genetic sources for cotton breeding. <em>Gossypium turneri</em> is a species from Sonora, Mexico (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fryxell 1978</a>), that is closely related to <em>G. raimondii</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2007</a>). Like <em>G. raimondii</em>, fiber from <em>G. turneri</em> is unspinnable; however, <em>G</em>. <em>turneri</em> has phenotypic characters with agronomic potential, <em>e.g.</em>, caducous bracts, insect resistance, and abiotic stress tolerance (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2018</a>).The two species are generally similar, both having a haploid complement of 13 chromosomes and relatively small genome sizes (910 Mb <em>vs.</em> 880 Mb in <em>G. turneri</em> and <em>G</em>. <em>raimondii</em>, respectively; (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hendrix and Stewart 2005</a>)). The two species, however, are genetically distinct, as long recognized by taxonomists and their extreme allopatry (<em>G. raimondii</em> is from Peru, <em>G. turneri</em> from Baja California), as well as by genetic and phylogenetic data (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ulloa <em>et al.</em> 2013</a>), (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Grover <em>et al.</em> 2019</a>). Notably, a previously published draft genome suggests that gene gain and loss may be elevated in <em>G. turneri</em> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Grover <em>et al.</em> 2019</a>).</p><p id="p-5">Here we describe two <em>de novo</em> genome sequences, for <em>G. raimondii</em> (D5) and <em>G. turneri</em> (D10), which were assembled using newly generated PacBio, Hi-C, and Bionano (<em>G. raimondii</em> only) technologies. The <em>G. raimondii</em> genome sequence reported here represents an independent effort and identifies three significant assembly errors in the initial publication of <em>G</em>. <em>raimondii</em>, including a large assembly artifact on the original chromosome 1. We also report a high-quality sequence for <em>G</em>. <em>turneri</em> that is suitable for various comparative, genetic, and genomic analyses. Together, these genomes represent a useful resource for cotton breeding and for comparative genomics in general.</p><div class="section methods-materials" id="sec-1"><h2 class="">Methods &amp; Materials</h2><div id="sec-2" class="subsection"><h3>Plant material and sequencing</h3><p id="p-6">Leaf tissue of mature <em>G. raimondii</em> (accession D5-4) and <em>G. turneri</em> (accession D10-3) plants was collected at the Brigham Young University (BYU) greenhouse. DNA was extracted using CTAB techniques (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kidwell and Osborn 1992</a>). DNA concentration was measured by a Qubit Fluorometer (ThermoFisher, Inc.). The sequencing library was constructed according to PacBio recommendations at the BYU DNA Sequencing Center (DNASC). Fragments &gt;18 kb were selected for sequencing via BluePippen (Sage Science, LLC). Prior to sequencing, the size distribution of fragments in the libraries was evaluated using a Fragment Analyzer (Advanced Analytical Technologies, Inc). Eight and eleven PacBio cells were sequenced from a single library each for <em>G. raimondii</em> and <em>G. turneri</em>, respectively, on the Pacific Biosciences Sequel system. For both genomes, the raw PacBio sequencing reads were assembled using Canu V1.6 using default parameters (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Koren <em>et al.</em> 2017</a>).</p><p id="p-7">HiC libraries were constructed from <em>G. raimondii</em> leaf tissue at NorthEast Normal University, China. Sequencing was performed at Annoroad Gene Technology Co., Ltd (Beijing, China). The HiC data of <em>G. raimondii</em> was mapped to the previous genome sequence of <em>G. raimondii</em> using HiC-Pro (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Servant <em>et al.</em> 2015</a>), and to the newly assembled CANU contigs of <em>G. raimondii</em> PacBio reads by PhaseGenomics. The HiC interactions were used as evidence for contig proximity and in scaffolding contig sequences. An initial draft genome sequence of pseudochromosomes (PGA assembly) was created using a custom python script from PhaseGenomics.</p><p id="p-8">DNA was also extracted from young <em>G. raimondii</em> leaves following the Bionano Plant protocol for high-molecular weight DNA. DNA was purified, nicked, labeled, and repaired according to Bionano standard operating procedures for the Irys platform. Two optical maps of different enzymes (<em>Bsp</em>QI and <em>Bss</em>SI) were assembled using the IrysSolve pipeline on the BYU Fulton SuperComputing cluster. The optical maps were combined into a two-enzyme composite optical map and it was aligned to the PGA assembly using an <em>in silico</em> labeled reference sequence. Conflicts between the Bionano maps and the PGA assembly were manually identified in the Bionano Access software by comparing the mapped Bionano contigs to the CANU contigs along the draft genome sequence. Conflicts between datasets were resolved by repositioning and reorienting CANU contigs in PGA ordering files followed by reconstruction of the fasta sequence, provided there was supporting or no-conflict evidence from the optical map ((<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Durand <em>et al.</em> 2016</a>), Supp. Figure 1). Multiple iterations of mapping, conflict resolution, and draft sequence construction resulted in the final, new genome sequence of <em>G. raimondii</em>.</p><p id="p-9">Leaf tissue of <em>G. turneri</em> was shipped to DoveTail Genomics for DNA extraction and construction of HiC sequencing libraries. These HiC sequencing libraries were sequenced on the Illumina HiSeq 2500 (PE125 bp) at the BYU DNASC. Reads were mapped to the <em>G</em>. <em>raimondii</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>) reference genome, and a scaffolded assembly was created for <em>G. turneri</em> by Dovetail Genomics. Whole genome alignments identified <em>in-silico</em> assembly errors where a contiguous 25.7 Mb of Chromosome 9 (D10_09) was initially placed on D10_12, and the remainder of that chromosome was in smaller scaffolded pieces. Similar to the process above, manual iterations of scaffolding correctly assembled D10_09 and D10_11 using Juicebox (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Durand <em>et al.</em> 2016</a>). The final genome sequence of <em>G. turneri</em> was constructed using a custom python script developed by PhaseGenomics, LLC and consists of 13 assembled chromosomes.</p></div><div id="sec-3" class="subsection"><h3>Repeats and gene annotation</h3><p id="p-10">Repeats were identified using a combination of RepeatMasker (Smit <em>et al.</em>) and “One code to find them all” (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bailly-Bechet <em>et al.</em> 2014</a>), the latter used to assemble multiple adjacent RepeatMasker hits into complete transposable element (TE) copies. RepeatMasker was run for each genome with a custom library, which combines Repbase 23.04 repeats (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bao <em>et al.</em> 2015</a>) with cotton-specific repeats. Default parameters were run, except the run was “sensitive” and was set to mask only TEs (no low-complexity). Parameters are available at <a href="https://github.com/Wendellab/D5D10">https://github.com/Wendellab/D5D10</a>. “One code to find them all” was used to aggregate multiple hits into TE models using default parameters. The resulting output was aggregated and summarized in R/3.4.4 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">R Development Core Team 2008</a>) using <em>dplyr</em> /0.7.4 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Wickham <em>et al.</em> 2019</a>). All code can be found at <a href="https://github.com/Wendellab/D5D10">https://github.com/Wendellab/D5D10</a>.</p><p id="p-11">The MAKER-P pipeline (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cantarel <em>et al.</em> 2008</a>) was used to annotate <em>G. raimondii</em> and <em>G. turneri</em> genomes after masking repetitive elements with RepeatMasker (Smit <em>et al.</em>) using a custom database that enriched for cotton-specific repeat sequences.</p><p id="p-12"><em>Gossypium raimondii</em> was annotated using the iterative MAKER-P method previously described (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Grover <em>et al.</em> 2017</a>) with the following modifications: (1) assembly of RNA-seq data using Mikado (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Venturini <em>et al.</em> 2018</a>); (2) RNA-seq assembly provided as another prediction source instead of ESTs evidence; and (3) updated software versions. The raw RNA-seq reads are available from the SRA (PRJNA493521). The assembly and annotation quality for each genome was validated via the BUSCO (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Simão <em>et al.</em> 2015</a>) pipeline, which evaluates completeness by characterizing the presence, fragmentation, and/or duplication of highly conserved genes. Single-copy syntenic orthologs were inferred using MCScanX (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Wang <em>et al.</em> 2012</a>) with a minimum of 50 genes in a syntenic block and gap penalty of 2. Any gene belonging to two different syntenic groups was removed.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-13">The assembled genome sequences of <em>G. raimondii</em> (PRJNA493304) and <em>G. turneri</em> (PRJNA493521) are available in NCBI (CP032553-CP032565 and CP032571-CP032583, respectively). The raw data for <em>G. raimondii</em> and <em>G. turneri</em> are also available in NCBI (SRR6356446 and SRR7957402, respectively). Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9702299">https://doi.org/10.25387/g3.9702299</a>.</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><div id="sec-6" class="subsection"><h3>Genome assemblies</h3><p id="p-14">We report two <em>de novo</em> genome sequences for the genus <em>Gossypium</em>, a new and corrected assembly for <em>G. raimondii</em> (D5) and a new reference-quality assembly for the closely related <em>G. turneri</em> (D10). These new genomes integrate multiple sequencing technologies and provide a more accurate representation of each cotton genome. Notwithstanding the utility of the original <em>G. raimondii</em> sequence (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>), it has become evident that the genome sequence contained minor assembly errors.Our genome sequence reported here provides an improved <em>G. raimondii</em> assembly using PacBio long read sequencing technology and corrects some errors in the genome sequence that have been identified (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Du <em>et al.</em> 2018</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wang <em>et al.</em> 2019</a>).</p><p id="p-15">The <em>G. raimondii</em> genome was assembled from 43.7x PacBio coverage of raw sequence reads. The assembly consisted of 187 contigs with an N50 of 6.3Mb (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The contigs were scaffolded using HiC by PhaseGenomics and the pseudomolecules were manually adjusted using JuiceBox (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Durand <em>et al.</em> 2016</a>). The final scaffolded assembly was independently verified using a composite optical map of two different enzymes. A comparison of assembly metrics between the previous genome sequence and our new genome sequence of <em>G. raimondii</em> illustrates a 45x improvement in contig length and a 97x reduction in the number of gaps. The cumulative gap length of the new assembly (17.6 kb) was reduced by 647x compared to the assembled gaps of the previous genome sequence (11,391 kb). The final genome assembly size was 14.9 Mb smaller than the previous assembly, representing 98% of previously assembled genome sequence in length.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494511/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494511/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Assembly metrics of the <em>G. turneri</em> genome, the <em>G. raimondii</em> (our current assembly, D5), and the previous <em>G. raimondii</em> assembly (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">This is the first <em>de novo</em> genome sequence for <em>G. turneri</em>. The <em>G. turneri</em> genome was assembled from 73.2x PacBio of raw sequence reads. The assembly consisted of 220 contigs with an N50 of 7.9Mb (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Similar to the <em>G. raimondii</em> sequence, these contigs were scaffolded by Dovetail Genomics and the pseudomolecules were manually adjusted using JuiceBox. Bionano data were not collected for <em>G. turneri</em>. The <em>G. raimondii</em> Bionano data were uninformative when aligned to the <em>G. turneri</em> genome sequence (because the distances between labeled recognition sites were too different). After creation of the sequence assembly, the <em>G. raimondii</em> HiC sequence reads were also mapped to the <em>G. turneri</em> genome sequence (and <em>vice versa</em>). While the number of mapped reads was reduced significantly (29.90% and 12.67%, respectively), there were no association anomalies detected between genomes.</p><p id="p-17">The assembled genome sequences were also verified by alignments to the D<sub>T</sub>-genome of <em>G. hirsutum</em> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Wang <em>et al.</em> 2019</a>) and to the previous genome assembly of <em>G. raimondii</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The chromosomes had general agreement in their alignments between the four independently assembled sequences (old and new <em>G. raimondii</em>; <em>G. turneri</em>; D<sub>T</sub> of <em>G. hirsutum</em>). Such colinearity was also previously identified between cotton genomes. For example, genetic maps of <em>G. hirsutum</em> (<em>e.g.</em>, (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Byers <em>et al.</em> 2012</a>)) were used to previously verify and sometimes establish proper scaffolding between contigs (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome comparisons between G. raimondii (D5), G. turneri (D10), G. raimondii (2012), and the DT-genome of G. hirsutum (DT). A) Genome alignment between G. turneri (D10) and G. raimondii (D5). B) Genome alignment between DT and D10. C) Genome alignment between DT and D5. D) Genome alignment between D5 (2012) and D5 (new). Red circles indicate assembly errors in the 2012 sequence as identified by these alignments and independent HiC data (e.g., D5_13 &#x2013; Chr01, D5_11 &#x2013; Chr03, D5_04 &#x2013; Chr09, D5_02 &#x2013; Chr13)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1869801530" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome comparisons between &lt;em&gt;G. raimondii&lt;/em&gt; (D5), &lt;em&gt;G. turneri&lt;/em&gt; (D10), &lt;em&gt;G. raimondii&lt;/em&gt; (2012), and the D&lt;sub&gt;T&lt;/sub&gt;-genome of &lt;em&gt;G. hirsutum&lt;/em&gt; (D&lt;sub&gt;T&lt;/sub&gt;). A) Genome alignment between &lt;em&gt;G. turneri&lt;/em&gt; (D10) and &lt;em&gt;G. raimondii&lt;/em&gt; (D5). B) Genome alignment between D&lt;sub&gt;T&lt;/sub&gt; and D10. C) Genome alignment between D&lt;sub&gt;T&lt;/sub&gt; and D5. D) Genome alignment between D5 (2012) and D5 (new). Red circles indicate assembly errors in the 2012 sequence as identified by these alignments and independent HiC data (&lt;em&gt;e.g.&lt;/em&gt;, D5_13 &#x2013; Chr01, D5_11 &#x2013; Chr03, D5_04 &#x2013; Chr09, D5_02 &#x2013; Chr13).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3079/F1.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3079/F1.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494374" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Genome comparisons between <em>G. raimondii</em> (D5), <em>G. turneri</em> (D10), <em>G. raimondii</em> (2012), and the D<sub>T</sub>-genome of <em>G. hirsutum</em> (D<sub>T</sub>). A) Genome alignment between <em>G. turneri</em> (D10) and <em>G. raimondii</em> (D5). B) Genome alignment between D<sub>T</sub> and D10. C) Genome alignment between D<sub>T</sub> and D5. D) Genome alignment between D5 (2012) and D5 (new). Red circles indicate assembly errors in the 2012 sequence as identified by these alignments and independent HiC data (<em>e.g.</em>, D5_13 – Chr01, D5_11 – Chr03, D5_04 – Chr09, D5_02 – Chr13).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Error Correction in G. raimondii genome sequence</h3><p id="p-19">Errors were identified in the previous <em>G. raimondii</em> sequence (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>). In the previous genome sequence, the chromosomes were named to be consistent with previous genetic maps; however, a new chromosome naming convention has been used for diploid and allotetraploid cotton (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Zhang <em>et al.</em> 2015</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Du <em>et al.</em> 2018</a>), where homeologous chromosomes are organized in sequence pairs (<em>e.g.</em>, A<sub>T</sub>_01 - A<sub>T</sub>_13 [Chr. 01 - Chr. 13] are homeologs of D<sub>T</sub>_01 - D<sub>T</sub>_13 [Chr. 14 – Chr. 26], respectively). We have adopted this new naming convention for the homologous chromosomes of these two genomes. Structural errors in the previously published sequence were identified by genome alignments (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) and by mapping HiC reads to the genome sequence (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Supp Figure 1). The largest error was an assembly-derived translocation of D5_04 (previously Chr. 12) on D5_05 (previously Chr. 09) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Additional, smaller errors were found between Chr. 01 (now D5_07) and Chr. 13 (now D5_13); Chr. 02 (now D5_01) and Chr. 13 (now D5_13); Chr. 03 (now D5_02) and Chr. 13 (now D5_13); Chr. 02 (now D5_01) and Chr. 03 (now D5_02); Chr. 02 (now D5_01) and Chr. 07 (now D5_11); and Chr. 03 (now D5_02) and Chr. 07 (now D5_11) (Supp Figure 1). These corrections based on alignment and HiC data were also supported by the alignment of Bionano data.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HiC interactions detected in the previously published G. raimondii genome sequence (Paterson et al. 2012). A) Most interaction maps of chromosome sequences suggested that the genome sequence was assembled in the correct order. B) A sequence was incorrectly assembled within Chr. 9 (now D5_05) that created a large insertion (red box). Few interactions were found between the inserted segment and the remainder of Chr. 9. C) Corresponding interactions were identified in the HiC interaction plot between Chr. 9 and Chr. 12 (now D5_04), as well as &#x2018;pinch&#x2019; within the diagonal interaction map in Chr. 12, indicating the true position of the incorrectly assembled sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1869801530" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;HiC interactions detected in the previously published &lt;em&gt;G. raimondii&lt;/em&gt; genome sequence (Paterson &lt;em&gt;et al.&lt;/em&gt; 2012). A) Most interaction maps of chromosome sequences suggested that the genome sequence was assembled in the correct order. B) A sequence was incorrectly assembled within Chr. 9 (now D5_05) that created a large insertion (red box). Few interactions were found between the inserted segment and the remainder of Chr. 9. C) Corresponding interactions were identified in the HiC interaction plot between Chr. 9 and Chr. 12 (now D5_04), as well as &#x2018;pinch&#x2019; within the diagonal interaction map in Chr. 12, indicating the true position of the incorrectly assembled sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3079/F2.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3079/F2.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494489" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">HiC interactions detected in the previously published <em>G. raimondii</em> genome sequence (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>). A) Most interaction maps of chromosome sequences suggested that the genome sequence was assembled in the correct order. B) A sequence was incorrectly assembled within Chr. 9 (now D5_05) that created a large insertion (red box). Few interactions were found between the inserted segment and the remainder of Chr. 9. C) Corresponding interactions were identified in the HiC interaction plot between Chr. 9 and Chr. 12 (now D5_04), as well as ‘pinch’ within the diagonal interaction map in Chr. 12, indicating the true position of the incorrectly assembled sequence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We also inspected a reported nuclear mitochondrial genome insertion (NUMT) on D5_07 (previously Chr. 1, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) located between coordinates 23.1Mb and 25Mb (<a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>). This region appears to have been the result of assembly error. Alignment of the two genomes (previous D5 genome <em>vs.</em> new D5 genome) identified a 1.26Mb segment that was inserted into the old sequence and not found in our new <em>de novo</em> assembly. Bionano data also indicated an insertion in the old assembly while the ‘inserted’ Bionano contig was unmapped in the new assembly of D5 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic assembly data of the new G. raimondii sequence suggest that the previously reported mitochondrial insertion was likely due to an assembly error. A) Genome alignments between G. raimondii Chr. 01 (Paterson et al. 2012) and our new genome sequence of D5_07. The red circle indicates the putative position of the mitochondrial genome insertion in the previous G. raimondii sequence relative to the new assembly. B) Alignment of G. raimondii PacBio reads (Track 2) to the new reference genome of G. raimondii (Track 1). The multi-colored bars represent individual PacBio reads (Track 2). The previous reference genome of G. raimondii had a mitochondrial insertion somewhere in this 14kb region indicated by the blue bar of Track 3. There are no PacBio reads that span the gap between the flanking regions of the 6,071 repeat and the repeat itself. C) Bionano data mapped to the previous reference genome sequence of G. raimondii (Paterson et al. 2012) also suggest an insertion of a sequence that is non-contiguous in the flanking regions. The Ref1 track reference to the originally published genome sequence of G. raimondii with a mitochondrial insertion between &#x223C;23Mb and &#x223C;24Mb. Independently constructed Bionano contigs were aligned to the 2012 reference sequence. A Bionano contig matched the reference sequence in the mitochondria insertion region, but the flanking regions of the Bionano contig (yellow) did not match flanking Bionano contigs or the reference sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1869801530" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic assembly data of the new &lt;em&gt;G. raimondii&lt;/em&gt; sequence suggest that the previously reported mitochondrial insertion was likely due to an assembly error. A) Genome alignments between &lt;em&gt;G. raimondii&lt;/em&gt; Chr. 01 (Paterson &lt;em&gt;et al.&lt;/em&gt; 2012) and our new genome sequence of D5_07. The red circle indicates the putative position of the mitochondrial genome insertion in the previous &lt;em&gt;G. raimondii&lt;/em&gt; sequence relative to the new assembly. B) Alignment of &lt;em&gt;G. raimondii&lt;/em&gt; PacBio reads (Track 2) to the new reference genome of &lt;em&gt;G. raimondii&lt;/em&gt; (Track 1). The multi-colored bars represent individual PacBio reads (Track 2). The previous reference genome of &lt;em&gt;G. raimondii&lt;/em&gt; had a mitochondrial insertion somewhere in this 14kb region indicated by the blue bar of Track 3. There are no PacBio reads that span the gap between the flanking regions of the 6,071 repeat and the repeat itself. C) Bionano data mapped to the previous reference genome sequence of &lt;em&gt;G. raimondii&lt;/em&gt; (Paterson &lt;em&gt;et al.&lt;/em&gt; 2012) also suggest an insertion of a sequence that is non-contiguous in the flanking regions. The Ref1 track reference to the originally published genome sequence of &lt;em&gt;G. raimondii&lt;/em&gt; with a mitochondrial insertion between &#x223C;23Mb and &#x223C;24Mb. Independently constructed Bionano contigs were aligned to the 2012 reference sequence. A Bionano contig matched the reference sequence in the mitochondria insertion region, but the flanking regions of the Bionano contig (yellow) did not match flanking Bionano contigs or the reference sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3079/F3.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3079/F3.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3079/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494399" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Genomic assembly data of the new <em>G. raimondii</em> sequence suggest that the previously reported mitochondrial insertion was likely due to an assembly error. A) Genome alignments between <em>G. raimondii</em> Chr. 01 (<a id="xref-ref-19-9" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>) and our new genome sequence of D5_07. The red circle indicates the putative position of the mitochondrial genome insertion in the previous <em>G. raimondii</em> sequence relative to the new assembly. B) Alignment of <em>G. raimondii</em> PacBio reads (Track 2) to the new reference genome of <em>G. raimondii</em> (Track 1). The multi-colored bars represent individual PacBio reads (Track 2). The previous reference genome of <em>G. raimondii</em> had a mitochondrial insertion somewhere in this 14kb region indicated by the blue bar of Track 3. There are no PacBio reads that span the gap between the flanking regions of the 6,071 repeat and the repeat itself. C) Bionano data mapped to the previous reference genome sequence of <em>G. raimondii</em> (<a id="xref-ref-19-10" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>) also suggest an insertion of a sequence that is non-contiguous in the flanking regions. The Ref1 track reference to the originally published genome sequence of <em>G. raimondii</em> with a mitochondrial insertion between ∼23Mb and ∼24Mb. Independently constructed Bionano contigs were aligned to the 2012 reference sequence. A Bionano contig matched the reference sequence in the mitochondria insertion region, but the flanking regions of the Bionano contig (yellow) did not match flanking Bionano contigs or the reference sequence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Since NUMTs evolve more quickly than do functional mitochondrial gene sequences, we also inspected the sequence similarity of the NUMT to the mitochondrial genome sequence of <em>G. raimondii</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2017</a>). The NUMT exhibited high similarity to the published <em>G. raimondii</em> mitochondrial genome (99.8% PID over 94% of region between Chr01:23,100,000-25,000,000). On an individual gene basis, over half of the genes contained within the putative NUMT were over 99% identical to the published sequence in the <em>G. raimondii</em> mitochondrial genome, with an average of 95% similarity. Considering the D-genome alignments and Bionano data presented above, the NUMT was more likely an assembly artifact than a recent insertion event in the <em>G. raimondii</em> genome.</p></div><div id="sec-8" class="subsection"><h3>Structural Variations Between the D-genomes</h3><p id="p-24">Comparisons between <em>G. raimondii</em> and <em>G. turneri</em> revealed several structural differences between the two genomes (Supp Figure 2). The genomes were largely colinear and no significant duplicated segments (relative to the genome alignments) were found in either genome (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a><strong>)</strong>. The assembled sequence of the <em>G. turneri</em> genome was 20.3 Mb longer than the <em>G. raimondii</em> genome and the gene content was similar (see below). The largest number of structural variants between the two genomes were chromosomal inversions. We identified several relative inversions between the two <em>de novo</em> genomes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Inversions were manually identified in the genome alignment output file. A total of 64 Mb genome sequence had an inverted order between these two genomes.These regions included a total of 2,592 genes (∼6.4% of gene total number). The largest structural variant was an inversion on D10_08 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, Supp Figure 3 - 15). This inversion could have been the result of misassembly, but the putative break points had clear overlapping, individual PacBio reads in <em>G. raimondii</em> (Supp Figures 6 &amp; 9) and in <em>G. turneri</em> (Supp Figures 10 &amp; 13<strong>)</strong>. In addition, both genomes had consistent HiC patterns for the D8 chromosome where an inversion of ∼16 Mb would have been clearly identified had it been the result of assembly error in one of the two genome sequences (Supp Figure 1).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494360/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494360/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Inversions between the <em>de novo</em> genome assemblies of <em>G. turneri</em> and <em>G. raimondii</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-25">We also compared the <em>G. turneri</em> and <em>G. raimondii</em> genome sequence to other <em>Gossypium</em> genomes (Supp Figure 16 &amp; 17, (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Du <em>et al.</em> 2018</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Wang <em>et al.</em> 2019</a>)). If a large inversion between <em>G. turneri</em> and <em>G. raimondii</em> was 1) also present in the genome alignment between <em>G. arboreum</em> and <em>G. turneri</em> and 2) was not present in the genome alignment between <em>G. raimondii</em> and <em>G. arboreum</em> then it was considered as an inversion derived during the natural evolutionary history of the <em>G. turneri</em> genome (similar logic for inversions derived in <em>G. raimondii</em> or <em>G. arboreum</em>). The inversions need to be large (&gt;2 Mb) and present in only one genome to be confident about its description without further investigation. The largest inversions on chromosomes D10_03, D10_05, D10_07, and D10_08 appear to be specific to <em>G. turneri</em> (36% of the length of total inversions). Chromosome rearrangements (inversions and other events) specific for <em>G. arboreum</em> were found on A2_01, A2_02, A2_03, A2_07, and A2_11. Only one inversion (D5_13, 2.6 Mb) was found to be specific to the <em>G. raimondii</em> genome. Perhaps, these inversions were part of the speciation process between the different <em>Gossypium</em> genomes.</p></div><div id="sec-9" class="subsection"><h3>Gene annotations</h3><p id="p-26">Similar numbers of genes were found in the annotation of each genome. Annotation of the genomes of <em>G. turneri</em> and <em>G. raimondii</em> identified 38,489 and 40,743 gene models respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). BUSCO analysis reported &gt;90% completeness scores for both <em>G. turneri</em> and <em>G. raimondii</em> genome assemblies, indicating that the evolutionarily-conserved core gene set was present in both <em>de novo</em> assemblies (Supp. Figure 18). Using MCScanX, we were able to identify 23,499 syntenic orthologs shared between the two species, indicating that the gene order and gene compliment are largely conserved between these two species. Genes in 34 of these syntenic orthologs were inferred to have more than one syntenic ortholog; these genes were removed from the dataset, resulting in 23,465 high-confidence syntenic orthologs (Supp. File 1). While not every gene was categorized into syntenic relationships, this is not surprising given that genes present in tandem arrays were excluded from this analysis (a default setting of MCScanX), gene loss has likely occurred in both species since they last shared a common ancestor, and subtle differences in gene annotation in the two genome assemblies likely lead to slight differences in overall gene content.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494495/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494495/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Each of the <em>de novo</em> genome assemblies were annotated for gene content using Maker-P</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Repeats</h3><p id="p-27">Transposable element content was predicted for both <em>de novo</em> genomes and compared to the existing <em>G. raimondii</em> reference sequence (<a id="xref-ref-19-11" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>). As expected, the <em>de novo G. raimondii</em> genome had nearly identical predicted TE content with the previous <em>G. raimondii</em> genome sequence (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). This difference is not significant and can be attributed to slight differences in assembly of repetitive regions. Consistent with the larger size of <em>G. turneri</em> than <em>G. raimondii</em> (910 Mb <em>vs.</em> 880 Mb), the <em>G. turneri</em> genome assembled an additional 8.5 Mb and 10.6 Mb of repetitive sequence, relative to the previous and new <em>de novo G. raimondii</em> genome sequences, respectively. Generally, the <em>G. turneri</em> genome sequence has slightly fewer DNA TEs and more LTR retrotransposons than the two <em>G. raimondii</em> genomes, both with respect to absolute content and percent of genome (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). No non-LTR retrotransposons (<em>e.g.</em>, LINE/SINE) were detected. For all three genome assemblies, retrotransposons comprise approximately 36% of the genome sequence, whereas all DNA elements combined comprise just under 3% in each. These results are consistent with a previous analysis of low-coverage sequencing results of these two genomes (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Grover <em>et al.</em> 2019</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494342/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494342/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Repetitive content of the newly sequenced <em>G. turneri</em> and <em>G. raimondii</em> genomes, and the previously published <em>G. raimondii</em> (<a id="xref-ref-19-12" class="xref-bibr" href="#ref-19">Paterson <em>et al.</em> 2012</a>). No LINE or SINE elements were detected. The genome size of <em>G. turneri</em> is 910 Mb and <em>G. raimondii</em> is 880 Mb</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Conclusion</h3><p id="p-28">Genome sequences of many plants have been recently published, and in fact are too numerous to cite here. Many of these previously reported genome sequences are being revisited with long-read technology of PacBio or Oxford Nanopore. In this report, we present new <em>de novo</em> genome sequences for <em>G. raimondii</em> and <em>G. turneri</em> based on PacBio long-read sequence technology. Both of these genomes are closely related to the D<sub>T</sub>-genome of cultivated tetraploid cotton. These sequences provide an evolutionary perspective for comparative genomics of the <em>Gossypium</em> clades as well as providing useful resources for the genetic improvement of cotton. Because of the economic relevance of the <em>Gossypium</em> genus, additional genome sequences of related <em>Gossypium</em> species will continue to be studied and revised in the future.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-29">We thank the National Science Foundation Plant Genome Research Program (Grant #1339412) and Cotton Inc. for their financial support. This research was funded, in part, through USDA ARS Agreements 58-6066-6-046 and 58-6066-6-059.We thank BYU Fulton SuperComputer lab for their resources and generous support. We also thank ResearchIT for computational support at Iowa State University. We thank Rise Services for office accommodations in Orem, UT.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-30">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9702299">https://doi.org/10.25387/g3.9702299</a>.</p></li><li class="fn" id="fn-2"><p id="p-31"><em>Communicating editor: P. Morrell</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-29"><span class="received-label">Received </span>May 29, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-26"><span class="accepted-label">Accepted </span>August 26, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Udall <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.1" data-doi="10.1186/1759-8753-5-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailly-Bechet</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Haudry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lerat</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">“One code to find them all”: a perl tool to conveniently parse RepeatMasker output files.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1759-8753-5-13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D5%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1759-8753-5-13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1759-8753-5-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.2" data-doi="10.1186/s13100-015-0041-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Kojima</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Kohany</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Repbase Update, a database of repetitive elements in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13100-015-0041-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D6%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13100-015-0041-9%26rft_id%253Dinfo%253Apmid%252F26045719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13100-015-0041-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26045719&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.3" data-doi="10.1007/s00122-011-1780-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>,  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Harker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Yourstone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Maughan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Udall</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and mapping of SNP assays in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1780-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DByers%26rft.auinit1%253DR.%2BL.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1214%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSNP%2Bassays%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1780-8%26rft_id%253Dinfo%253Apmid%252F22252442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1780-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252442&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.4" data-doi="10.1101/gr.6743907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cantarel</span>,  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Korf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M. C.</span> <span class="cit-name-surname">Robb</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Parra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ross</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.6743907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6743907%26rft_id%253Dinfo%253Apmid%252F18025269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.5" data-doi="10.1007/s10681-018-2118-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Feng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zhao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Zhou</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Overcoming obstacles to interspecific hybridization between <em>Gossypium hirsutum</em> and <em>G. turneri</em>.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-018-2118-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D214%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-018-2118-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-018-2118-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.6" data-doi="10.1111/plb.12536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Grover</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Entire nucleotide sequences of <em>Gossypium raimondii</em> and <em>G. arboreum</em> mitochondrial genomes revealed A-genome species as cytoplasmic donor of the allotetraploid species.</span> <abbr class="cit-jnl-abbrev">Plant Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">484</span>–<span class="cit-lpage">493</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/plb.12536</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiol.%26rft.volume%253D19%26rft.spage%253D484%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fplb.12536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/plb.12536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. E.</span> <span class="cit-name-surname">Scheffler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Dennis</span></span></li></ol><cite>, B. a Triplett, T. Zhang <em>et al</em>., <span class="cit-pub-date">2007</span> Toward Sequencing Cotton (<em>Gossypium</em>) Genomes. PLANT Physiol. 145: 1303–1310.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.8" data-doi="10.1101/gad.179804.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">de Laat</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A decade of 3C technologies: insights into nuclear organization.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.179804.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253Dde%2BWit%26rft.auinit1%253DE.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D24%26rft.atitle%253DA%2Bdecade%2Bof%2B3C%2Btechnologies%253A%2Binsights%2Binto%2Bnuclear%2Borganization%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.179804.111%26rft_id%253Dinfo%253Apmid%252F22215806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMjYvMS8xMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzA3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.9" data-doi="10.1038/s41588-018-0116-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sun</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Resequencing of 243 diploid cotton accessions based on an updated A genome identifies the genetic basis of key agronomic traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">802</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0116-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D796%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0116-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0116-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.10" data-doi="10.1016/j.cels.2015.07.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>,  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Robinson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Shamim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Machol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mesirov</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Juicebox Provides a Visualization System for Hi-C Contact Maps with Unlimited Zoom.</span> <abbr class="cit-jnl-abbrev">Cell Syst.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cels.2015.07.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSyst.%26rft.volume%253D3%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cels.2015.07.012%26rft_id%253Dinfo%253Apmid%252F27467250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cels.2015.07.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27467250&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fryxell</span>,  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title"><em>Gossypium turneri</em> (Malvaceae), A New Species From Sonora, Mexico.</span> <abbr class="cit-jnl-abbrev">Madrono</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMadrono%26rft.volume%253D25%26rft.spage%253D155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.12" data-doi="10.1093/gbe/evx248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>,  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Arick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Conover</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Thrash</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative Genomics of an Unusual Biogeographic Disjunction in the Cotton Tribe (Gossypieae) Yields Insights into Genome Downsizing.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">3328</span>–<span class="cit-lpage">3344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evx248</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D3328%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.13" data-doi="10.1093/gbe/evy256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>,  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Arick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Thrash</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Conover</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Sanders</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Insights into the Evolution of the New World Diploid Cottons (<em>Gossypium</em>, Subgenus <em>Houzingenia</em>) Based on Genome Sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">71</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D11%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevy256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evy256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.14" data-doi="10.1016/j.plantsci.2006.12.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. Q.</span> <span class="cit-name-surname">Sang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Zhou</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. Z.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic relationships of D-genome species based on two types of EST-SSR markers derived from <em>G. arboreum</em> and <em>G. raimondii</em> in <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">814</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2006.12.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D172%26rft.spage%253D808%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2006.12.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2006.12.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.15" data-doi="10.1093/aob/mci078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hendrix</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Stewart</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of the nuclear DNA content of <em>Gossypium</em> species.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mci078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmci078%26rft_id%253Dinfo%253Apmid%252F15701660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mci078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15701660&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3079.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>,  <span class="cit-name-given-names">K. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Osborn</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Simple plant DNA isolation procedures</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span> in <span class="cit-source">Plant Genomes: Methods for Genetic and Physical Mapping</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dordrecht, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.17" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Walenz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Berlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Bergman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: Scalable and accurate long-read assembly via adaptive κ-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.18" data-doi="10.1038/nbt.3208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Fan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Xiao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome sequence of cultivated Upland cotton (<em>Gossypium hirsutum</em> TM-1) provides insights into genome evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">530</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3208</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3208%26rft_id%253Dinfo%253Apmid%252F25893780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25893780&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.19" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>,  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Wendel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gundlach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jenkins</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11798</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3079.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>,  <span class="cit-name-given-names">D. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Arick</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Sequencing Plant Genomes</span>, pp. <span class="cit-fpage">109</span>–<span class="cit-lpage">193</span> in <span class="cit-source">Progress in Botany</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Cánovas</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Lüttge</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Matyssek</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Pretzsch</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3079.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>. <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.22" data-doi="10.1186/s13059-015-0831-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Servant</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Varoquaux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Lajoie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Viara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-J.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HiC-Pro: an optimized and flexible pipeline for Hi-C data processing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0831-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0831-x%26rft_id%253Dinfo%253Apmid%252F26619908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0831-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26619908&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.23" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.10.3079.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>,  <span class="cit-name-given-names">A. F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hubley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Green</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> RepeatMasker Open-4.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.25" data-doi="10.1139/cjb-2012-0192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. Y.</span> <span class="cit-name-surname">Abdurakhmonov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Perez-M</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Percy</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Stewart</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic diversity and population structure of cotton (<em>Gossypium</em> spp.) of the New World assessed by SSR markers.</span> <abbr class="cit-jnl-abbrev">Botany</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/cjb-2012-0192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBotany%26rft.volume%253D91%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjb-2012-0192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjb-2012-0192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.26" data-doi="10.1093/gigascience/giy093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venturini</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Caim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. G.</span> <span class="cit-name-surname">Kaithakottil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Mapleson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Swarbreck</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Leveraging multiple transcriptome assembly methods for improved gene structure annotation.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giy093</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiy093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giy093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.27" data-doi="10.1093/nar/gkr1293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Debarry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Tan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MCScanX: A toolkit for detection and evolutionary analysis of gene synteny and collinearity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-elocation-id">e49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr1293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1293%26rft_id%253Dinfo%253Apmid%252F22217600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22217600&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.28" data-doi="10.1038/s41588-018-0282-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Tu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Yuan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Shen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Reference genome sequences of two cultivated allotetraploid cottons, <em>Gossypium hirsutum</em> and <em>Gossypium barbadense</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0282-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D51%26rft.spage%253D224%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0282-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0282-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Francois</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Henry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Muller</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> dplyr: A Grammar of Data Manipulation. R package version 0.7.4. <a href="https://CRAN.R-project.org/package=dplyr">https://CRAN.R-project.org/package=dplyr</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3079.30" data-doi="10.1038/nbt.3207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Jiang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Guan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequencing of allotetraploid cotton (<em>Gossypium hirsutum</em> L. acc. TM-1) provides a resource for fiber improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">537</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D531%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3207%26rft_id%253Dinfo%253Apmid%252F25893781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25893781&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3079.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3067" title="Greenlip Abalone (Haliotis laevigata) Genome and Protein Analysis Provides Insights into Maturation and Spawning" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3087" title="An RNAi Screen for Genes Required for Growth of Drosophila Wing Tissue" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3079" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-A3X3HY79dsox8ztywFNC46biUxa5dC8ASOwZoMZfr8I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3079" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3079" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3079" class="active" data-icon-position="" data-hide-link-title="0">De Novo Genome Sequence Assemblies of Gossypium raimondii and Gossypium turneri</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494290" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ISYw6F30vsiKPf6bhinIpBJ6KQiVJHQ5bX8Dmq5SP6s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3079.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3079" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-tBa5IP3Lk9WeQ0VdXTiNhJRQA715RMyU9mwcVmSYzhg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3079" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3079.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494290" id="node494290--2" data-pisa="ggg;9/10/3079" data-pisa-master="ggg;g3.119.400392" data-apath="/ggg/9/10/3079.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Sequence Assemblies of <em>Gossypium raimondii</em> and <em>Gossypium turneri</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0978-4764" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="1">Evan Long</span>, <span class="highwire-citation-author" data-delta="2">Chris Hanson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6007-5571" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daojun Yuan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7333-1041" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thiruvarangan Ramaraj</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-3558-6000" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Justin L. Conover</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6429-267X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lei Gong</span>, <span class="highwire-citation-author" data-delta="7">Mark A. Arick</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-3878-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corrinne E. Grover</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-0274-5968" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel G. Peterson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-2258-5081" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan F. Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3079-3085;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494290--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA/Agricultural Research Service, Crop Germplasm Research Unit, College Station, TX 77845,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0978-4764" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joshua A. Udall</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Joshua.Udall@usda.gov" class="" data-icon-position="" data-hide-link-title="0">Joshua.Udall@usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Evan Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chris Hanson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BHanson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hanson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BHanson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daojun Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daojun%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaojun%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6007-5571" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daojun Yuan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Thiruvarangan Ramaraj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Computing, DePaul University, Chicago, IL 60604,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thiruvarangan%2BRamaraj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramaraj%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThiruvarangan%2BRamaraj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7333-1041" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thiruvarangan Ramaraj</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justin L. Conover </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BL.%2BConover%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conover%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BL.%2BConover%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3558-6000" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Justin L. Conover</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lei Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Key Laboratory of Molecular Epigenetics of the Ministry of Education, Northeast Normal University, Changchun, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lei%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALei%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6429-267X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lei Gong</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mark A. Arick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomics, Biocomputing & Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BArick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arick%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BA.%2BArick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Corrinne E. Grover </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corrinne%2BE.%2BGrover%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grover%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorrinne%2BE.%2BGrover%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3878-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corrinne E. Grover</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Daniel G. Peterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomics, Biocomputing & Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BG.%2BPeterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peterson%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BG.%2BPeterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0274-5968" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel G. Peterson</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jonathan F. Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BF.%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BF.%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2258-5081" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan F. Wendel</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3079" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3079&amp;title=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3079&amp;title=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3079&amp;title=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3079&amp;text=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3079&amp;title=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3079&amp;t=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3079&amp;title=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3079&amp;title=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494290/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494290" id="node494290--3" data-pisa="ggg;9/10/3079" data-pisa-master="ggg;g3.119.400392" data-apath="/ggg/9/10/3079.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Sequence Assemblies of <em>Gossypium raimondii</em> and <em>Gossypium turneri</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0978-4764" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="1">Evan Long</span>, <span class="highwire-citation-author" data-delta="2">Chris Hanson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6007-5571" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daojun Yuan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7333-1041" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thiruvarangan Ramaraj</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-3558-6000" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Justin L. Conover</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6429-267X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lei Gong</span>, <span class="highwire-citation-author" data-delta="7">Mark A. Arick</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-3878-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corrinne E. Grover</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-0274-5968" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel G. Peterson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-2258-5081" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan F. Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3079-3085;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494290--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA/Agricultural Research Service, Crop Germplasm Research Unit, College Station, TX 77845,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0978-4764" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joshua A. Udall</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Joshua.Udall@usda.gov" class="" data-icon-position="" data-hide-link-title="0">Joshua.Udall@usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Evan Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chris Hanson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant and Wildlife Science Dept. Brigham Young University, Provo, UT 84042,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BHanson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hanson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BHanson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daojun Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daojun%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaojun%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6007-5571" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daojun Yuan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Thiruvarangan Ramaraj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Computing, DePaul University, Chicago, IL 60604,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thiruvarangan%2BRamaraj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramaraj%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThiruvarangan%2BRamaraj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7333-1041" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thiruvarangan Ramaraj</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justin L. Conover </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BL.%2BConover%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conover%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BL.%2BConover%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3558-6000" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Justin L. Conover</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lei Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Key Laboratory of Molecular Epigenetics of the Ministry of Education, Northeast Normal University, Changchun, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lei%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALei%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6429-267X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lei Gong</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mark A. Arick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomics, Biocomputing & Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BArick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arick%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BA.%2BArick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Corrinne E. Grover </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corrinne%2BE.%2BGrover%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grover%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorrinne%2BE.%2BGrover%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3878-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corrinne E. Grover</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Daniel G. Peterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomics, Biocomputing & Biotechnology, Mississippi State University, Mississippi State, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BG.%2BPeterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peterson%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BG.%2BPeterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0274-5968" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel G. Peterson</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jonathan F. Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ecology, Evolution, and Organismal Biology Dept., Iowa State University, Ames, IA 50010,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BF.%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BF.%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2258-5081" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan F. Wendel</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3079&amp;rft.epage=3085&amp;rft.atitle=De%20Novo%20Genome%20Sequence%20Assemblies%20of%20Gossypium%20raimondii%20and%20Gossypium%20turneri&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Udall%2C+Joshua+A.&amp;rft.au=Long%2C+Evan&amp;rft.au=Hanson%2C+Chris&amp;rft.au=Yuan%2C+Daojun&amp;rft.au=Ramaraj%2C+Thiruvarangan&amp;rft.au=Conover%2C+Justin+L.&amp;rft.au=Gong%2C+Lei&amp;rft.au=Arick%2C+Mark+A.&amp;rft.au=Grover%2C+Corrinne+E.&amp;rft.au=Peterson%2C+Daniel+G.&amp;rft.au=Wendel%2C+Jonathan+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494290/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494290/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494290/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494290/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494290/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494290/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494290/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494290/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494290/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494290/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494290/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494290/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496052" id="node496052" data-pisa="ggg;g3.120.401151v1" data-pisa-master="ggg;g3.120.401151" data-apath="/ggg/early/2020/03/26/g3.120.401151.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.120.401151" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Long-Read Genome Sequencing and Assembly of <em>Leptopilina</em> <em>boulardi</em>: A Specialist <em>Drosophila</em> Parasitoid</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3079" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods &amp; Materials</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3079.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3079.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3079.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
