<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3263" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3263.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3263.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3263.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3263.atom" />
<meta name="HW.pisa" content="ggg;9/10/3263" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" />
<meta name="DC.Identifier" content="10.1534/g3.119.400450" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Chen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Puccinia hordei ( Ph ) is a damaging pathogen of barley throughout the world. Despite its importance, almost nothing is known about the genomics of this pathogen, and a reference genome is lacking. In this study, the first reference genome was assembled for an Australian isolate of Ph (“ Ph 560”) using long-read SMRT sequencing. A total of 838 contigs were assembled, with a total size of 207 Mbp. This included both haplotype collapsed and separated regions, consistent with an estimated haploid genome size of about 150Mbp. An annotation pipeline that combined RNA-Seq of Ph -infected host tissues and homology to proteins from four other Puccinia species predicted 25,543 gene models of which 1,450 genes were classified as encoding secreted proteins based on the prediction of a signal peptide and no transmembrane domain. Genome resequencing using short-read technology was conducted for four additional Australian strains, Ph 612, Ph 626, Ph 608 and Ph 584, which are considered to be simple mutational derivatives of Ph 560 with added virulence to one or two of three barley leaf rust resistance genes ( viz. Rph3 , Rph13 and Rph19 ). To identify candidate genes for the corresponding avirulence genes AvrRph3 , AvrRph13 and AvrRph19 , genetic variation in predicted secreted protein genes between the strains was correlated to the virulence profiles of each, identifying 35, 29 and 46 candidates for AvrRph13 , AvrRph3 and AvrRph19 , respectively. The identification of these candidate genes provides a strong foundation for future efforts to isolate these three avirulence genes, investigate their biological properties, and develop new diagnostic tests for monitoring pathogen virulence." />
<meta name="DC.Contributor" content="Jiapeng Chen" />
<meta name="DC.Contributor" content="Jingqin Wu" />
<meta name="DC.Contributor" content="Peng Zhang" />
<meta name="DC.Contributor" content="Chongmei Dong" />
<meta name="DC.Contributor" content="Narayana M. Upadhyaya" />
<meta name="DC.Contributor" content="Qian Zhou" />
<meta name="DC.Contributor" content="Peter Dodds" />
<meta name="DC.Contributor" content="Robert F. Park" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Puccinia hordei&lt;/i&gt; (&lt;i&gt;Ph&lt;/i&gt;) is a damaging pathogen of barley throughout the world. Despite its importance, almost nothing is known about the genomics of this pathogen, and a reference genome is lacking. In this study, the first reference genome was assembled for an Australian isolate of &lt;i&gt;Ph&lt;/i&gt; (“&lt;i&gt;Ph&lt;/i&gt;560”) using long-read SMRT sequencing. A total of 838 contigs were assembled, with a total size of 207 Mbp. This included both haplotype collapsed and separated regions, consistent with an estimated haploid genome size of about 150Mbp. An annotation pipeline that combined RNA-Seq of &lt;i&gt;Ph&lt;/i&gt;-infected host tissues and homology to proteins from four other &lt;i&gt;Puccinia&lt;/i&gt; species predicted 25,543 gene models of which 1,450 genes were classified as encoding secreted proteins based on the prediction of a signal peptide and no transmembrane domain. Genome resequencing using short-read technology was conducted for four additional Australian strains, &lt;i&gt;Ph&lt;/i&gt;612, &lt;i&gt;Ph&lt;/i&gt;626, &lt;i&gt;Ph&lt;/i&gt;608 and &lt;i&gt;Ph&lt;/i&gt;584, which are considered to be simple mutational derivatives of &lt;i&gt;Ph&lt;/i&gt;560 with added virulence to one or two of three barley leaf rust resistance genes (&lt;i&gt;viz. Rph3&lt;/i&gt;, &lt;i&gt;Rph13&lt;/i&gt; and &lt;i&gt;Rph19&lt;/i&gt;). To identify candidate genes for the corresponding avirulence genes &lt;i&gt;AvrRph3&lt;/i&gt;, &lt;i&gt;AvrRph13&lt;/i&gt; and &lt;i&gt;AvrRph19&lt;/i&gt;, genetic variation in predicted secreted protein genes between the strains was correlated to the virulence profiles of each, identifying 35, 29 and 46 candidates for &lt;i&gt;AvrRph13&lt;/i&gt;, &lt;i&gt;AvrRph3&lt;/i&gt; and &lt;i&gt;AvrRph19&lt;/i&gt;, respectively. The identification of these candidate genes provides a strong foundation for future efforts to isolate these three avirulence genes, investigate their biological properties, and develop new diagnostic tests for monitoring pathogen virulence.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3263" />
<meta name="citation_id" content="9/10/3263" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3263" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3263.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3263.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3263.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400450" />
<meta name="citation_pmid" content="31444296" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3263" />
<meta name="citation_lastpage" content="3271" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jiapeng Chen" />
<meta name="citation_author_institution" content="Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9145-3018" />
<meta name="citation_author" content="Jingqin Wu" />
<meta name="citation_author_institution" content="Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia" />
<meta name="citation_author" content="Peng Zhang" />
<meta name="citation_author_institution" content="Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4191-1068" />
<meta name="citation_author" content="Chongmei Dong" />
<meta name="citation_author_institution" content="Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia" />
<meta name="citation_author" content="Narayana M. Upadhyaya" />
<meta name="citation_author_institution" content="Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3052-0416" />
<meta name="citation_author" content="Qian Zhou" />
<meta name="citation_author_institution" content="Agricultural Genomic Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" />
<meta name="citation_author" content="Peter Dodds" />
<meta name="citation_author_institution" content="Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and" />
<meta name="citation_author" content="Robert F. Park" />
<meta name="citation_author_institution" content="Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia" />
<meta name="citation_author_email" content="robert.park@sydney.edu.au" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9145-5371" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Anderson;citation_author=M. A. Khan;citation_author=A.-M. Catanzariti;citation_author=C. A. Jack;citation_author=A. Nemri;citation_title=Genome analysis and avirulence gene cloning using a high-density RADseq linkage map of the flax rust fungus, Melampsora lini.;citation_pages=667;citation_volume=17;citation_year=2016;citation_pmid=27550217;citation_doi=10.1186/s12864-016-3011-9" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. Cantu;citation_author=V. Segovia;citation_author=D. MacLean;citation_author=R. Bayles;citation_author=X. Chen;citation_title=Genome analyses of the wheat yellow (stripe) rust pathogen Puccinia striiformis f. sp. tritici reveal polymorphic and haustorial expressed secreted proteins as candidate effectors.;citation_pages=270;citation_volume=14;citation_year=2013;citation_pmid=23607900;citation_doi=10.1186/1471-2164-14-270" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A.-M. Catanzariti;citation_author=P. N. Dodds;citation_author=G. J. Lawrence;citation_author=M. A. Ayliffe;citation_author=J. G. Ellis;citation_title=Haustorially expressed secreted proteins from flax rust are highly enriched for avirulence elicitors.;citation_volume=18;citation_year=2006;citation_pmid=16326930;citation_doi=10.1105/tpc.105.035980" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Chen;citation_author=N. M. Upadhyaya;citation_author=D. Ortiz;citation_author=J. Sperschneider;citation_author=F. Li;citation_title=Loss of AvrSr50 by somatic exchange in stem rust leads to virulence for Sr50 resistance in wheat;citation_pages=1607-1610;citation_volume=358;citation_year=2017;citation_issue=6370;citation_pmid=29269475;citation_doi=10.1126/science.aao4810" />
<meta name="citation_reference" content="citation_journal_abbrev=Fly;citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. Wang le;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_issue=2;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_title=The Cereal Rusts II;citation_title=Barley leaf rust;citation_pages=173-205;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Exp. Agric.;citation_author=P. Cotterill;citation_author=R. G. Rees;citation_author=G. Platz;citation_author=R. Dill-Macky;citation_title=Effects of leaf rust on selected Australian barleys.;citation_pages=747-751;citation_volume=32;citation_year=1992;citation_doi=10.1071/EA9920747" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Exp. Agric.;citation_author=P. J. Cotterill;citation_author=R. F. Park;citation_author=R. G. Rees;citation_title=Pathogenic specialization of Puccinia hordei Otth. in Australia, 1966–1990.;citation_pages=127-134;citation_volume=46;citation_year=1995;citation_doi=10.1071/AR9950127" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. A. Cuomo;citation_author=G. Bakkeren;citation_author=H. B. Khalil;citation_author=V. Panwar;citation_author=D. Joly;citation_title=Comparative analysis highlights variable genome content of wheat rusts and divergence of the mating loci.;citation_volume=7;citation_year=2017;citation_pmid=27913634;citation_doi=10.1534/g3.116.032797" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. de Jonge;citation_author=P. H. van Esse;citation_author=K. Maruthachalam;citation_author=M. D. Bolton;citation_author=P. Santhanam;citation_title=Tomato immune receptor Ve1 recognizes effector of multiple fungal pathogens uncovered by genome and RNA sequencing.;citation_volume=109;citation_year=2012;citation_pmid=22416119;citation_doi=10.1073/pnas.1119623109" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=P. N. Dodds;citation_title=The Melampsora lini AvrL567 avirulence genes are expressed in haustoria and their products are recognized inside plant cells.;citation_volume=16;citation_year=2004;citation_pmid=14973158;citation_doi=10.1105/tpc.020040" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. N. Dodds;citation_author=J. P. Rathjen;citation_title=Plant immunity: towards an integrated view of plant-pathogen interactions.;citation_pages=539-548;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20585331;citation_doi=10.1038/nrg2812" />
<meta name="citation_reference" content="citation_author=C. Dong;citation_year=2017;citation_doi=10.17504/protocols.io.hbvb2n6" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=L. Du;citation_author=Y. Li;citation_author=X. Zhang;citation_author=B. Yue;citation_title=MSDB: a user-friendly program for reporting distribution and building databases of microsatellites from genome sequences.;citation_volume=104;citation_year=2012;citation_pmid=23144492;citation_doi=10.1093/jhered/ess082" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Duplessis;citation_author=C. A. Cuomo;citation_author=Y.-C. Lin;citation_author=A. Aerts;citation_author=E. Tisserant;citation_title=Obligate biotrophy features unraveled by the genomic analysis of rust fungi.;citation_volume=108;citation_year=2011;citation_pmid=21536894;citation_doi=10.1073/pnas.1019315108" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=T. Eilam;citation_author=W. R. Bushnell;citation_author=Y. Anikster;citation_title=Relative nuclear DNA content of rust fungi estimated by flow cytometry of propidium iodide-stained pycniospores.;citation_pages=728-735;citation_volume=84;citation_year=1994;citation_doi=10.1094/Phyto-84-728" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=H. H. Flor;citation_title=Current status of the gene-for-gene concept.;citation_pages=275-296;citation_volume=9;citation_year=1971;citation_doi=10.1146/annurev.py.09.090171.001423" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=D. Garnica;citation_author=A. Nemri;citation_author=N. Upadhyaya;citation_author=J. P. Rathjen;citation_author=P. N. Dodds;citation_title=The ins and outs of rust haustoria.;citation_pages=e1004329;citation_volume=10;citation_year=2014;citation_pmid=25211126;citation_doi=10.1371/journal.ppat.1004329" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. D. G. Jones;citation_author=J. L. Dangl;citation_title=The plant immune system.;citation_pages=323-329;citation_volume=444;citation_year=2006;citation_issue=7117;citation_pmid=17108957;citation_doi=10.1038/nature05286" />
<meta name="citation_reference" content="citation_journal_title=Australas. Plant Pathol.;citation_author=H. Karaoglu;citation_author=R. F. Park;citation_title=Isolation and characterization of microsatellite markers for the causal agent of barley leaf rust, Puccinia hordei.;citation_pages=47-52;citation_volume=43;citation_year=2013;citation_doi=10.1007/s13313-013-0244-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. L. Kitchen;citation_author=J. D. Moore;citation_author=S. A. Palmer;citation_author=R. G. Allaby;citation_title=MCMC-ODPR: Primer design optimization using Markov Chain Monte Carlo sampling.;citation_pages=287-287;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23126469;citation_doi=10.1186/1471-2105-13-287" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_year=2005" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=W. S. Lee;citation_author=K. E. Hammond-Kosack;citation_author=K. Kanyuka;citation_title=Barley stripe mosaic virus-mediated tools for investigating gene function in cereal plants and their pathogens: virus-induced gene silencing, host-mediated gene silencing, and virus-mediated overexpression of heterologous protein.;citation_volume=160;citation_year=2012;citation_pmid=22885938;citation_doi=10.1104/pp.112.203489" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Lu;citation_author=B. Kracher;citation_author=I. M. L. Saur;citation_author=S. Bauer;citation_author=S. R. Ellwood;citation_title=Allelic barley MLA immune receptors recognize sequence-unrelated avirulence effectors of the powdery mildew pathogen.;citation_volume=113;citation_year=2016;citation_pmid=27702901;citation_doi=10.1073/pnas.1612947113" />
<meta name="citation_reference" content="citation_journal_title=CUTADAPT REMOVES ADAPTER SEQUENCES FROM HIGHTHROUGHPUT SEQUENCING READS;citation_author=M. Martin;citation_pages=10;citation_volume=17;citation_year=2011;citation_pmid=25452271;citation_doi=10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Plant-Microbe Int.;citation_author=C. H. Mesarich;citation_author=S. A. Griffiths;citation_author=A. van der Burgt;citation_author=B. Ökmen;citation_author=H. G. Beenen;citation_title=Transcriptome sequencing uncovers the Avr5 avirulence gene of the tomato leaf mold pathogen Cladosporium fulvum. Mol.;citation_pages=846-857;citation_volume=27;citation_year=2014;citation_doi=10.1094/MPMI-02-14-0050-R" />
<meta name="citation_reference" content="citation_journal_title=Adv. Bioinforma.;citation_author=S. Metz;citation_author=J. M. Cabrera;citation_author=E. Rueda;citation_author=F. Giri;citation_author=P. Amavet;citation_title=FullSSR: Microsatellite Finder and Primer Designer.;citation_pages=6040124;citation_volume=2016;citation_year=2016;citation_doi=10.1155/2016/6040124" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=M. E. Miller;citation_author=Y. Zhang;citation_author=V. Omidvar;citation_author=J. Sperschneider;citation_author=B. Schwessinger;citation_title=De novo assembly and phasing of dikaryotic genomes from two isolates of Puccinia coronata f. sp. avenae, the causal agent of oat crown rust.;citation_volume=9;citation_year=2018;citation_pmid=29463655;citation_doi=10.1128/mBio.01650-17" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. Nemri;citation_author=D. G. O. Saunders;citation_author=C. Anderson;citation_author=N. M. Upadhyaya;citation_author=J. Win;citation_title=The genome sequence and effector complement of the flax rust pathogen Melampsora lini.;citation_pages=98;citation_volume=5;citation_year=2014;citation_pmid=24715894;citation_doi=10.3389/fpls.2014.00098" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=R. F. Park;citation_title=Pathogenic specialization and pathotype distribution of Puccinia hordei in Australia, 1992 to 2001.;citation_pages=1311-1316;citation_volume=87;citation_year=2003;citation_doi=10.1094/PDIS.2003.87.11.1311" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=R. F. Park;citation_title=Breeding cereals for rust resistance in Australia.;citation_pages=591-602;citation_volume=57;citation_year=2008;citation_doi=10.1111/j.1365-3059.2008.01836.x." />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Phytopathol.;citation_author=R. F. Park;citation_author=P. G. Golegaonkar;citation_author=L. Derevnina;citation_author=K. S. Sandhu;citation_author=H. Karaoglu;citation_title=Leaf rust of cultivated barley: pathology and control.;citation_pages=565-589;citation_volume=53;citation_year=2015;citation_doi=10.1146/annurev-phyto-080614-120324" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. N. Petersen;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions.;citation_pages=785-786;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21959131;citation_doi=10.1038/nmeth.1701" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=C. Plissonneau;citation_author=J. Benevenuto;citation_author=N. Mohd-Assaad;citation_author=S. Fouché;citation_author=F. E. Hartmann;citation_title=Using population and comparative genomics to understand the genetic basis of effector-driven fungal pathogen evolution.;citation_pages=1-15;citation_volume=8;citation_year=2017;citation_pmid=28220127;citation_doi=10.3389/fpls.2017.00023" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=C. R. Praz;citation_author=S. Bourras;citation_author=F. Zeng;citation_author=J. Sánchez-Martín;citation_author=F. Menardo;citation_title=AvrPm2 encodes an RNase-like avirulence effector which is conserved in the two different specialized forms of wheat and rye powdery mildew fungus.;citation_pages=1301-1314;citation_volume=213;citation_year=2017;citation_doi=10.1111/nph.14372" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Bot.;citation_author=S. Rogers;citation_author=S. Rehner;citation_author=C. Bledsoe;citation_title=Extraction of DNA from Basidiomycetes for ribosomal DNA hybridizations.;citation_pages=1235-1243;citation_volume=67;citation_year=1989;citation_doi=10.1139/b89-161" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Salcedo;citation_author=W. Rutter;citation_author=S. Wang;citation_author=A. Akhunova;citation_author=S. Bolus;citation_title=Variation in the AvrSr35 gene determines Sr35 resistance against wheat stem rust race Ug99;citation_pages=1604-1606;citation_volume=358;citation_year=2017;citation_issue=6370;citation_pmid=29269474;citation_doi=10.1126/science.aao7294" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. G. Saunders;citation_author=J. Win;citation_author=L. M. Cano;citation_author=L. J. Szabo;citation_author=S. Kamoun;citation_title=Using hierarchical clustering of secreted protein families to classify and rank candidate effectors of rust fungi.;citation_pages=e29847-e29847;citation_volume=7;citation_year=2012;citation_issue=1;citation_pmid=22238666;citation_doi=10.1371/journal.pone.0029847" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=B. Schwessinger;citation_author=J. Sperschneider;citation_author=W. S. Cuddy;citation_author=D. P. Garnica;citation_author=M. E. Miller;citation_title=A near-complete haplotype-phased genome of the dikaryotic wheat stripe rust fungus Puccinia striiformis f. sp. tritici reveals high interhaplotype diversity.;citation_volume=9;citation_year=2018;citation_pmid=29463659;citation_doi=10.1128/mBio.02275-17" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=H. Sonah;citation_author=R. K. Deshmukh;citation_author=R. R. Bélanger;citation_title=Computational prediction of effector proteins in fungi: Opportunities and challenges.;citation_pages=126;citation_volume=7;citation_year=2016;citation_doi=10.3389/fpls.2016.00126" />
<meta name="citation_reference" content="citation_journal_title=Proceedings / . International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology;citation_journal_abbrev=Proc Int Conf Intell Syst Mol Biol;citation_author=E. L. Sonnhammer;citation_author=G. von Heijne;citation_author=A. Krogh;citation_title=A hidden Markov model for predicting transmembrane helices in protein sequences.;citation_pages=175-182;citation_volume=6;citation_year=1998;citation_pmid=9783223" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. Sperschneider;citation_author=A. H. Williams;citation_author=J. K. Hane;citation_author=K. B. Singh;citation_author=J. M. Taylor;citation_title=Evaluation of secretion prediction highlights differing approaches needed for Oomycete and fungal effectors.;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.01168" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=N. M. Upadhyaya;citation_author=D. P. Garnica;citation_author=H. Karaoglu;citation_author=J. Sperschneider;citation_author=A. Nemri;citation_title=Comparative genomics of Australian isolates of the wheat stem rust pathogen Puccinia graminis f. sp. tritici reveals extensive polymorphism in candidate effector genes.;citation_pages=1-13;citation_volume=5;citation_year=2015;citation_doi=10.3389/fpls.2014.00759" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=H. Wallwork;citation_author=P. Preece;citation_author=P. J. Cotterill;citation_title=Puccinia hordei on barley and Ornithogalum umbellatum in South Australia. Austral.;citation_pages=95-97;citation_volume=21;citation_year=1992" />
<meta name="citation_reference" content="citation_title=Comparative genomics integrated with association analysis identifies candidate effector genes corresponding to Lr20 in phenotype-paired Puccinia triticina isolates from Australia. Front. Plant Sci;citation_pages=148;citation_volume=8;citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Puccinia hordei ( Ph ) is a damaging pathogen of barley throughout the world. Despite its importance, almost nothing is known about the genomics of this pathogen, and a reference genome is lacking. In this study, the first reference genome was assembled for an Australian isolate of Ph (“ Ph 560”) using long-read SMRT sequencing. A total of 838 contigs were assembled, with a total size of 207 Mbp. This included both haplotype collapsed and separated regions, consistent with an estimated haploid genome size of about 150Mbp. An annotation pipeline that combined RNA-Seq of Ph -infected host tissues and homology to proteins from four other Puccinia species predicted 25,543 gene models of which 1,450 genes were classified as encoding secreted proteins based on the prediction of a signal peptide and no transmembrane domain. Genome resequencing using short-read technology was conducted for four additional Australian strains, Ph 612, Ph 626, Ph 608 and Ph 584, which are considered to be simple mutational derivatives of Ph 560 with added virulence to one or two of three barley leaf rust resistance genes ( viz. Rph3 , Rph13 and Rph19 ). To identify candidate genes for the corresponding avirulence genes AvrRph3 , AvrRph13 and AvrRph19 , genetic variation in predicted secreted protein genes between the strains was correlated to the virulence profiles of each, identifying 35, 29 and 46 candidates for AvrRph13 , AvrRph3 and AvrRph19 , respectively. The identification of these candidate genes provides a strong foundation for future efforts to isolate these three avirulence genes, investigate their biological properties, and develop new diagnostic tests for monitoring pathogen virulence." />
<meta name="og-title" property="og:title" content="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3263" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Puccinia hordei ( Ph ) is a damaging pathogen of barley throughout the world. Despite its importance, almost nothing is known about the genomics of this pathogen, and a reference genome is lacking. In this study, the first reference genome was assembled for an Australian isolate of Ph (“ Ph 560”) using long-read SMRT sequencing. A total of 838 contigs were assembled, with a total size of 207 Mbp. This included both haplotype collapsed and separated regions, consistent with an estimated haploid genome size of about 150Mbp. An annotation pipeline that combined RNA-Seq of Ph -infected host tissues and homology to proteins from four other Puccinia species predicted 25,543 gene models of which 1,450 genes were classified as encoding secreted proteins based on the prediction of a signal peptide and no transmembrane domain. Genome resequencing using short-read technology was conducted for four additional Australian strains, Ph 612, Ph 626, Ph 608 and Ph 584, which are considered to be simple mutational derivatives of Ph 560 with added virulence to one or two of three barley leaf rust resistance genes ( viz. Rph3 , Rph13 and Rph19 ). To identify candidate genes for the corresponding avirulence genes AvrRph3 , AvrRph13 and AvrRph19 , genetic variation in predicted secreted protein genes between the strains was correlated to the virulence profiles of each, identifying 35, 29 and 46 candidates for AvrRph13 , AvrRph3 and AvrRph19 , respectively. The identification of these candidate genes provides a strong foundation for future efforts to isolate these three avirulence genes, investigate their biological properties, and develop new diagnostic tests for monitoring pathogen virulence." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fY3_qj77WaAT6zC-ZHWBxZIGuQNG75XJ5DW6ZYiUgKs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494283","apath":"\/ggg\/9\/10\/3263.atom","pisa":"ggg;9\/10\/3263","ac":{"\/ggg\/9\/10\/3263.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3263.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3263"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3263"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3263":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J\\\\\\\/jCqT7Y6EkjWsRguCNU\\\\\\\/vw9fg9qeCl1txdaRZ0mjqm3SmRfr\\\\\\\/wujRNwmUyT7yZKQcHimzP1oNJQ0fRZy6HlZEY0AkjcZ3Ugc4CijPQ4j9bjVwathNTdcelU4CF3jR9jHtA27rJ9bF03BJIOCvypKhXeOLrNvC4SbHKwYqE3y14zlAu0G1YTrrbadgdTBzMBSE9OAs0+iXfOluSQ5\\\\\\\/CDsEWpgjrwQrAUzzbR5jmLmhi5cS1cPUq8D9dhndhz81JX18ShyYht4aSFTrc3EEZmavSDj0ck73z7ZutA\\\\\\\/89niLNXBVianTUbpOCri+62zCDjrin8KmK2NOEtubnG9RIsh6R5K2wyUxtL0mqomgsXcWl9iNthZ5fXSGb6jK7isG6Turfy\\\\\\\/0FQKRYQxITen5kMXR0\\\\\\\/Ew5ZVTUbPrZor7Bqxw1meFgg\\\\\\\/FnnVF2R8UVXynyXHc5p6pEIN7DiqNDEGa6sefqnmBr8Dq4ZBRcE\\\\\\\/oroHSOyCFo5FxdA5\\\\\\\/rsR6kXOASC7QicYAqS598To2kQ8v1POxMN0Ig6kO1jSOU\\\\\\\/Q6R8rZ0CxeBOzZLFcj35ivMLBuPtTWsKb58AMNS9c42+aLT\\\\\\\/onqFVjz40JPp9KmrHP\\\\\\\/VFlNVFOGjBr4MiHDn341kh+irjVwcN9tfU0kXp\\\\\\\/m2laj7b7gyLKupZHQKYxBb1e49rFal\\\\\\\/M4dw446wbfjtQ9FHSR1KJ+JFrLDV5sH3tBD9i25SJ+z2yF8IJSuLyPGOji6cf8eNWoxWZD+P5UIyqZYXhpMjC7mncQgIG\\\\\\\/w7lSh+3RhHFi6zxUUHRnHw5mch2kzm8JjVWxSnb\\\\\\\/9aDsOQTPCEzOXlWkBNzioKcn4QggNgF57ZEpstwtQpf0n9KNM5e4KTbXaat6EHyEW+3bDHEX0ppPLQDAOkxdfd7AXd0lBHOjlkR\\\\\\\/YLPsxut\\\\\\\/CetTfQCPBGulWL2JUfr5zpcGi4Ry0GLV0x6kH8HpOQOGCnV6qI4Qs4eB\\\\\\\/KzX\\\\\\\/lLCUQUX6+LsCijZJ8QRBG\\\\\\\/AHiBskSWD60f1nJJMumQvB19YomXO+axFNFolJ6IN\\\\\\\/n9JJTXyRhgPk5jHjTcX4W5T3dBEZ\\\\\\\/pT55euJ0+1HyYD7XueByb3zD2lUXZNkWI3rEhpq2oCFqPICD\\\\\\\/gK9ab\\\\\\\/HeQwTPWLnxvWFUuhUOg5UoFLUb3kJpCIUwKqQGtJtDpUZQECISqSmiuwK69Eq1HlvDAiz5DQlZ\\\\\\\/cSExgs5pgugOpU3tbA5zB829l+LEWnqSJnGMcpdMQRMJXXA1i7JWJpV22udzMO0gcfLkH\\\u0022,\\\u0022iv\\\u0022:\\\u0022z00+\\\\\\\/24qgw+b5kCNQV70Bg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022242a4deb1cce0c953f487e764968dd5c17a4b5a7703c00c09f0b389d33fe9c53\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pBBLtA6RF8wBh4w7kTf4o3I1u1kZQor\\\\\\\/dHIOLTWI9IlRpspFekAJAI\\\\\\\/2aItz2wbtCnAGuh\\\\\\\/CfuCBngCFMTpjSZMppKELAaXjlpPTrMY1QiiXRAK2Os8MhFSCr0pJvcUGXKgO+BffppiKeLmKFVMCXsejYRLAh316WpPsa\\\\\\\/IJJhFx1jnXK4mMp3oqQRbxtMN1CXg1VhCfjJxNL4NFCgEy2yftYqTmBqpcccRyhPYLrkIYwlngWeL4JR74z5m8y0MK9LiFnepRWgOwMiqtT7mzvvAM9y2dpQPGUuETscwMfr\\\\\\\/NnxzEGO54s30jtB4NfuKB7zfLhrN4tT9LAzSbZ7TCqreYnfqIe3l6TfKg6FBaoeQPtPZq+YIm73WYT8FGnVEYRJsgqMrf4lOcV1Eu7sqOu5vGxoDHTeIFi2QC1Es2ET8C3kVH3OTpGIaInM7bp0qDUrizBCdtKFi0iC\\\\\\\/O1YSbz7VbFpPVMO4EvKWsJZ2fnjmI3t9OjqzP4gA9Opd+V+qNegQpEiIdkGFZAaAlVaVifID2l3hON8w\\\\\\\/5BwAXnKHeQagbsi7w9A6EhkIWpaj06fIdvbJlyYrYy+PheWeH72KsgDxw2F\\\\\\\/\\\\\\\/X63yheiCZubhpxyQOYXHuAO144EIZkFcMrbJh3pPyA6fTLiO7\\\\\\\/FTZFw6ruCUawO6SH0NrvWpeR\\\\\\\/r3S0D1ax9Rk4ofJ5ijpNv5PS6NmDeVU\\\\\\\/K0eNJIdy9y6W\\\\\\\/4M791Vr62ZpNI7SQs5qw2GwmfHiH3bwUrYKHGRBjnwxFNCRAJmIIOk3YUcYsJUwdgsxCw1Y7P8s33mTQV5e0KjmqOHcjmgCKPw9mPBU4mSnoQOoeGOWJjK8pjyxhTho8ka0iw+6YlnwckC697Ice87bQYEkGZhRZqs8ZoPeXdr7+uO9wDRvnZD9J9+X74xQGwZxJVvoB9nLbYKrAUzp9Vkgscsea5Xh7D7qPB38d4NVCcEagpBAOB3buiCLWbDq6a4Pjx4fFNXIyHrljNCeE9cv\\\\\\\/UqI9PJlUf7gl0Z41wneCz\\\\\\\/UlP+UASIPQlmQ3pAtX+jh6I7acPKcfMMucwnKjOTy8SUlREhPibpmVgmxlZTG34sdzpgg\\\\\\\/acq3qZWBrqlAS8i0ZbOkX9aEQo\\\\\\\/KovuZf0ZRMVL5n9FCR0JKIpC46i\\\\\\\/4lKaXfMg+\\\\\\\/qMxlTfXPU0zkhBeGcy8QrFFyQTiGdL2i\\\\\\\/4LKiCWlA\\\\\\\/qGpDmyYtbdAIZwlRmwJJP5J1bB7wR3PLE\\\\\\\/cai0JN3qnl\\\\\\\/Gm5qpm0KC1MAzOfstM0r4IWh4olECSZO3MqyHpiRK3xeey2Lt05fWH0hnCggKpnXmawJQTFyaIgQwVJ32WkfFIM4ebV3BGlmPOvlg\\\\\\\/VVjWBcnyOqSYqWqYOAJtNA81F6kcudMjPeMdxV4jHXKNoU1vUyXHb9NLr0Gw\\\\\\\/zXB4QCIKZY8csqfVPfMOCgm2voFBhhzTO7XCrlg\\\\\\\/MqWtrVHSFrZXLoOyEhp\\\\\\\/Eur2a1AtWHLtFnPTZjx9tPsS77eDdWw6L4xEmxytgvoc3dXiY0fYj0sakkigUPmr83g\\\\\\\/AHpPV47oVz+wEbEr41\\\\\\\/UfjPXD0zPOnFvlWxYt+ZgwSR8czwG6b0lqwM1Swj1bgsxImeSjA+aKLMQHx2YjJSPTlU+WEh4zrlf9mkKXzTXA0xre0udcrR\\\\\\\/lBVeRkF0NXHeA0Vz9S4FM\\\\\\\/3Yr66pl1aLk1gQQZG31yuiPUfyi+0DlZiPtCjjmA8o8wvvekXHIUTt8NGwzhh4u5eQn9Rllb9OT9aPIyqGrAUJ\\\\\\\/vNOcvZMuvuKGELv+PadDdaLhQ+i9lf+V1Dy9W\\\\\\\/bJssXPGmFMnU6k8zS+szJUqX64R69T98eILVXI\\\\\\\/uuETh9CeuysGKSQ7DTa7aqOVgavkJqBTesWhtc1IcJIfTWmjkXvXZ++s5aWCZpdJx9bXV1OIjC4L23NxwjX4jn6APhCwhUreWr4zVbovekmvMFFiB1pPmpTYhACg7RqlwHnympl8oBuJqZVSTKB9WdfhhUl013pJhaqas6IS7x+ZXqtruPuf9hki9uviHVKqKfSyI4Q5rpKsLaeDs0xvnrfm1qdbmT+ChpyaOb4qkGCRA20FFjlvsElu1eeMK1Cq3SzgOxCNY81QcYRZdlp5GOsfSBDnhq23pzk\\\\\\\/0hnabZ+uRYXTJqBLtw973qeaxQ0BURxZRk5xAv2lMGOIymDzgZHlpc+RnlJHSobx9Nob9STNbV76iXbzJdB50bD3\\\\\\\/KfFRTecG1T8L5vq+tVg+1BlhnMy39SayZKRv7IoCOTBkwhPK1jk8BLwuGykNZcF5\\\\\\\/KWdGDjWQzb1UxJEt2NlvRF2Z0\\\\\\\/9wluqIsvfwQCuyqmjs1Alsq97LvjAk6Y1XDhH3thmgcdaf+K1f5in28Rt8caO1AwC13J2lgHTWm1C6vFGwO+Nye6kp\\\\\\\/\\\\\\\/nFjRRkW86XxJLhQB5f2SDGqDx+gB5\\\\\\\/eUOmk0iwM6IS5StbtZBIpCtuW9TA075eAynEjggi3+IVXbnyKvsY07bMfgj2x5YGuGQ+3MdgbT\\\\\\\/gfb6hKcdweRJ3rCwOz1IkKDxKmnbgxMBwSlxk4shLbjcn0imT0HS0VUXEEWenx0yWh5elUqbWEnDgQsemCmBHdDp+JvF4XGwzk3XRtLS9dfll8hwD8byMHyHO\\\\\\\/qwkbPqXmxKQxmQjRhE3b6F4btWNgQBSXfNIyzF+elaWl4aNefSo6BSD8NrBixavb4avgStbFWVE80s1rVUzauicydLMLsjlhMjdvxWVzbs1CLeM6OHB0ytUmAyO7Is0C\\\\\\\/rNrVrTe\\\\\\\/s6ZYJsXZy7gjA6XRT4PBP9I\\\\\\\/FJ5gtLVVX+ikpxxzamhwcRDV90riJgFao\\\\\\\/VZB8L2WHSkyc9QI3qSfDNWs2lyLNYmHb9xBF6x74wgVZVE5yEcvPOVMC0IwSKR\\\\\\\/q4c5uYiySFbj0a8GJC4VVw7e8+yI+LFmqdD5Z5Egxc5Q\\\\\\\/rwRFUibpFwFwEDDyxU4nBHO1t0FRYFF+irMfbd+C\\\\\\\/3ugLXSqoW+PlGNxNbGum6Yf6MMGpgAlzSTlJHlUiJ1uXjaoqcHrQqHXJjRK+e6pczZIJ9eMiIlcHSEwMVkgTt\\\\\\\/6pqA2WBLzrVgid3ARwC\\\\\\\/zO+8qRN4uU9TgOrmLAh84EkaNVZGwuKiVJiXlbSAhPRQ4PckoK3WGhl5AVh2pBWKdPI3BiH9agIkPTlz7E76FcFXbWCiwKxYJsWkxUxdwQvZ+RRZzFSovzUw4fVDAL0x5dhmHvi+Op6hz34pghejFXhpjycCAhb2TvCSGq+SAKv8uhU6J8Vsj5NKNBfaEF7BNl\\\\\\\/tVCb\\\\\\\/Tj3bo2D9ErIhTmQc2TV753X56xf8IQIBQvR6Kmb4PpzsPjRHi\\\\\\\/eYYnc1yS8L3EXqjFrGoqMH\\\\\\\/wyGeXNIbkst7X0No1XhLRCnGGlIUPofqHJ3AqPB7qYvMN6S1TlYIhDC5ej4I0\\\\\\\/vSqPvKSV\\\\\\\/\\\\\\\/Q3Gr0nRMuzKWYk2iUrs5TzQbkpkcajVkGxQ1snJf54plcM7VH9S4lz0eWogv4bKtbHaYHiIp63zdeHq9r8NYUy1B9ZnaysTkRIFOQpObzrbi7bEmpDDo6kqY1FCl8PqKzZ3f6+PWHnFQ3v3bREIqtBqgjpN7vgZYZjJZBNB2T+luVwvrLSgQB4GLglbTtf0goWAx3GqFIgejxqVSaM6KqukyMQdjke3gn7qGcSplOzh6loyYDa64PmdQgni98OHza5uAJPtJO1U7bNZAM+PRqRlGDGM7BEEjRt29Jb3pt5UP8F1xfYtStfVhFqEbKCkB3AfRHSLUegVbCSiGj2p178Sa6oc+fY2k56gFywFWKMWyjiu9PzTNQ19JrzC8ArkLiOWAOr3RPh6vanZqAkpaCEdeSa2Rj08pT0GpEpx5AD8S9t32bHKoJ\\\\\\\/sGavQaTLjtkfuqGKe9TJ\\\\\\\/\\\\\\\/hDDXGEakHaql0tf83Mgp\\\\\\\/UslcGknYpwtO8L50iSF5o3wDSTyoDuDsrkBxpCMntjC5k3pKB0D+MGHjd1P1ot+jD2w3S8KHasWtY6UqwdECGeuNfCbSvzVq71lnQ888RpAQ4tfNBNmpy52aC5piGFF7jzYUGxyihIO88aVS8ZGPS4d4BHr+Z\\\\\\\/T0dB1xJg+vaDPEn5lbFKNRsJx33Y152JWFUg6e8ogyQjzNSH6MJ1QFxcPeOFP+qMMLSryWUcNS7Uiy7gVr+Fuwu3zRVoMaVD831BPcti3q6JiXrefaTjGgycPKqHBTWanYN1aSdacy3hP\\\\\\\/1FXWP0UtymFzYeCUSfTSScwLMch5HunxDwVe\\\\\\\/MObT16ELiJPbK0F67KCvsN\\\\\\\/1VYSfFJvHrFsFzQ+kChQjcbrzTqhZJwNrqODPUluyPu1sw6tb\\\\\\\/i6QClfgag\\\\\\\/D5kfDTT8HWIZNGc9IPKkG+RR5RYQJV\\\\\\\/yInIF3NhR7MWruZzfP68YO9+FWdLLlm+\\\\\\\/11QLUCelODZx4gIcJDDkrwCTAucQCl5DcKKBXB5FSY4Xem3Hv1pO1ksPvZjW\\\\\\\/Xk+ihUHkVnFZP7U7K5cneedLRY2ND+0KdWApuR2BwhKXwQKfm9WhRUYStu4mZ1AJsDCp+B9NnD0osJKG\\\\\\\/iy8ejHoFspQakFYbfssS2aCVjcCsM+ubmuf76e9E7z4VN7dDj9mrimSGskxMna5RmdrsGqlF3mmlwpV8DxfrV3og\\\\\\\/sc8GMFWztQ3SQP9q6ZJd6Sf6b1LzOTnnYPG1htyw7\\\\\\\/Z3qOE5fFG1puCpvtOti5u3dIWq4ZuraqP7A6xNskTTsAyCAlH5Q371y8IGTeqQyNDqnmAfIkIvR6ZTyLQ+nMsAEiq4JYZyUc1\\\\\\\/ay\\\\\\\/t+SbkzgaL4\\\\\\\/U1N\\\\\\\/Wd\\\\\\\/+VXLT2wsC3k2nNrTycIUOsYplWUwNoufa+trypJnVin6UsFlkynUC3g1G23hdYN9fKX+siu8x0jK0uxpAdYvcwKs9E4Wtts\\\\\\\/EWPpFn7dBPnp8FaZ7bsXRV6U168FuAa66hdS03gMGOQYJuqyF8NjCjlYPSCjV8MFKWCkyyleUyoxb+FF1fBdSuQkjlAx23pfgPGwATvbh9i\\\\\\\/ud\\\\\\\/nChjSWq1jodJlwLdhcHbeU9EtFWrbnj6SZqm1P4z8UumjQeUZe3GcdP9uJ3fjLM+7dtYzJCBzJ5sFZIKRq+5iCQcRv5pyLX0kfwkq8Wy2Hv87w4e3xWiYmfDzRWsry+lZRUqN\\\\\\\/TQP97J9EW\\\\\\\/5gbrFfWRZt+8wC2UKLn5H\\\\\\\/8vZSkzq8P7vOTbU7L0EFwY6XDMupNflJKj52hhfnBcZ\\\\\\\/fmwP4rcrxSnkUVk4qtWMauWv6W5a3Wm59T7ocW0ZL1rx+xb6yZeFkzHnLSO1amnXlpKPcIZkaHeIy0VdEDbCSh4d1aUH+PyC5DZtcooCog+6\\\\\\\/w7uyt8c+xcynUnwfUl8Y8PcAin0pGCZzX+ul4TgV6nvgsRRouq5jS1nLt93ErSY22qb7gQNC\\\\\\\/s5BRiJm+R+8nfOpRbjrXuQads58xR3\\\\\\\/+b0CUAMh4Cy5d8NtZq0e1FaWJqcBJdJOHrsS3Gg3mZBVARCuJ6fI9+Uuc3uDJCiqnriokD\\\\\\\/rv+yWt1ZtOQUMdtFewPN3sLlb8GKlBfXqiJ4iu9cxqd\\\\\\\/jsW+0WHktNi6ZRg2P8FHmVeMmK4IdSuYCUzK3rGg517TgYu7lsWcEJbhi5NdmbJGX\\\\\\\/PYjaalrI4xvOkF6a6dF3yTkzTm9Wn1CyeMEPaPHzPF+gA37+HI\\\\\\\/H3+\\\\\\\/gF+1KqSbgJY2p9nsbxSvNLMYbZmZzQRC1jKOdHT7yIr7SL2ZM5Rfo1oepu0n2p5AnyCeqssUCUWSWefSMoCjWxTp1ktPQRTkjVlLxDqXQtViN0JKvWhXKonf5j80OnwxlrHSRXt72Dl9EnX1MaVwgEJRHCMMFW23ktJa017AlRu+4r8D9VIzHNtPNd0tGKDZAUaVo1HFIW5R8Tjw72bv7Y2kbuhyhiucWTlrArZeXT9VM55WqaOC9DYWhiY4Pvxxl9P2USIgVELvSjtWhBK3Zf6VaN5IAN2LBvkjbp3Si5N6Ch6oSoDK2\\\\\\\/oUvapeMb+klz9ea3Na62RJPpI3J5Kj8P25SiI0FIeSjD1k2939FzhxD2pTMtDNrpBmNaNKYw7IkgH1C08XJtvtzcR\\\\\\\/QprYyVkOX1ZMEvkNEDDbsqQ0wrWV8Qea7Db6FWnCs7Or3kUA5lCzi2vPAPOmnYV3aDAZFVd1oTRnRSPq54YbKwL4JCp+zlqNEglyLrQ4CTTTjh9KJvqwJYNxFUqZcyKhZAainW8mmhg0XSiiG2PfL6aWP8xxoIv1hGfI01wr+yGMOqzkA5LMySULZYFCJGsQq5Vj3Yq+KLDkS5bEjT1OjGTF6WDaK8uT3LUALKoaRp4978MMmXJJTYxng6TtEXtJ5CBU\\\\\\\/UPz8X+9CEeS6TYQYFy9trbsuzl0NlzZtHYWptJJGStxMkppBJW0TC+GDFkpopzQXpq+qFUax1PfYGw\\\\\\\/t046QaMKt9cEz5EIQMKf1yP1dXH9QqBgRQXfWD4KkGqBPU71FUHM2gD9L+M9dG6ULEYuhHllf5kWjY0YhGmJYVSGKMLYq3alNYg8qCCFMeUl7SrJDYzsmWYHqhNeW8eYxkuFmkzVw\\\\\\\/t7fWRMWQCcgy4OqtkGLfag2575OZ3y7v2ApDHmfRqTKUg6ax1K9I4sp7GAX9Dnu+gS9VNL6Ov2BDSnkFAZQLhV0ikV610uXYCRmUx+YouZUbUAat9ywpfNN\\\\\\\/DT9OyJymaKrahGkEmtGjMCQ8fZDwq6sy7TLfrdl9rOOz5AWgClBinani+Sn52aEIJtiHLpLzTsUOJ9qMVfwh7g+XeAq6+qVezlC4HhtsYPWUePgdnruoZjiDXaTlOKyV3CEdc1XF7iElP6J2LANvm5ovVg0u+oxLwy4NTuDG1lv+xEpf4CTNeh05oG3FBuMfmOyet5F0nwhmXDnstkJeahBQpXiV8MDTv4LAVHKzMG03HdPmDCTftXeehBVt3DiO7F3LzzWMHRUMrrDi44q2NUY9ir+4KFufGQBUxdcCAW0VgOC8w10K5mku4zieMGiCoxCl6T9Dj9JyNvOwssFvTRnGxlym6HD89OJLhLjx0AklV8g8K23z+pwpdIpHa8qX6WnjMUvsih6EvKIJGbsFYaszIbxxVMFiW7Sy0O6hN7qry4WgAsznrx2K4He8Qv1oLbu3v2t6ORkgqCm6iWcxPPP5SlDOuYacr1UrKKBgPDKap3D6BUBwEHUl1a8Ak6wJssWwjNrrzcv5SH\\\\\\\/tAs0Zia0VoD3QjUMSlXHpMziVEefwV1uA\\\\\\\/CJ+kR2NyYC9nWVh9XkvWNd529Axiq1y86D8WdZasXB3TGdE6My+XkLBTy5FKsADwBwkZglZ\\\\\\\/avNLa+d221s9I83eCx\\\\\\\/bbuH7RAcDv\\\\\\\/7eFAx5BKRQHG5w0BPlQj+jCfpBYReIvPvBqeHN5fLfrrj+k1o5Rx46iQQDxs7ryR5wJaFuz2LWhb9puWmCxBA0zqo3Kt06kl\\\\\\\/YOL3zw4ZThX+S8bzcUmo4knJ\\\\\\\/eDcoFzsO4ZGPAqrQ2WiPcnyjxUYvIom6maKwbOm7cMjJgubzYOUdXr8w6g9iadm2Z+QirnOlPX+S\\\\\\\/TXMdSKnu+afeGfUG98KFMCD+UbBXGEIa+XVx2DqTa3s0b3TlPHagf2sADvIG1NlWGNe0mEe3xrLLNk9ogFN6mNWQWJdtrbH0CV6ZyXD3Fss90MuvSNAWXZtxX6CSQODnTNbbROjn\\\\\\\/n9YBWHrgay3y4lijwU35JSHEKpvR4P0Gr9Hky5cu0SD8KPEohXvi3VLL3Z2SoklVNCdD5Nuno9nQ9drI6ILW4KcYFN+VnsiTYSLlX1GuT7lUC\\\\\\\/rbH64nny+1a8mgRhxNeHw8q5S58QZnqz2tPNBkG+FoJ6nbG4U1qOFrY1IHnJkkA7lwpGXlj1nvsTFzWigGTuKgsVtkSYmRBgeSb2M54P1HUVHnkMyXKHyelh5v+ArUiWFb3Vc0fCl2PsHoSO9SAEqro2THHrz7RIV4t2UuGBfU6\\\\\\\/Bo6ulM+qLDScN0D2a3QEPTFA3RJcQfmzD2zWuYGk+wZXGFrBS\\\\\\\/jl1RafXVUBdZIOf1c\\\\\\\/WocU1Q0onSIrYW8R7k5zdgrYvHDU7kUatGzefdSxLHz8\\\\\\\/Jla13ZNeCFiGT4MzotqpLoJANDM0jbSXQ4AalqMTuqK3BYsrMnVzmfJEuQfEIAoqoVRQXGamte9cL3y6YxNm\\\\\\\/6TdYJe7fWNaqctqxSqC\\\\\\\/oFqJ8tOLnD\\\\\\\/igqnc009q41u5WceZ5Xiu+QgSKV4qdo7406\\\\\\\/4tlau+CW8Kzg\\\\\\\/nVBocnMPsLJ5eyNHHnKa8X6+pdsPXRWDXqNXgJmGtNox3jno04lPAYkT5F4dqTzNIAGio2m1cvZ8uwC\\\\\\\/BNH+Wd+\\\\\\\/yc2kp0sTEsuv7cdAP\\\\\\\/gf4B5kYGNnAvmvNhmGC3ahBEX5JMyeayzJGav\\\\\\\/6rDk4gEJiOqNSwjU0hJxUEkikwJ4Hm6VVbkR9EBgoLlzPPcKlDBHw0iS5wBQpDA0pav8YCF4mf3R5ChGh1DBogas3L0+qpYR2tOcDksm6fPIVY93PGE0vb\\\\\\\/FH2Oj80+nqr0H9Nbpo2Q5MBlVL2eBvBq5dHurLuZu69eYYLLlp67nf\\\\\\\/\\\\\\\/sTPNhW3FEo1fzROlVO0fJ7lAjSK5teqAJ4j1o0f0stqfgtQJgllvzRdFYbXQAzEpWgLuFnMhMnMNGBA6miemn+fE10eIINi5X1H\\\\\\\/1B0PjdXHvlOC4toILCwtQgLuYl34Ih\\\\\\\/D\\\\\\\/VaR\\\\\\\/Wx7aVGGUk5TpKh++Z5wJob8Ysn6qbfpoz8FhW6n2xOUDmyRGLxle\\\\\\\/7swNexJxKvvRBBujEpsUHxjC0AJyEZBu+lsQYIQnwa7YVxEquWfxBRVjZVU6E2I+zxaykzO3dfGZvdlr2DRaHLWJ4cd1930hAUs09DxmxBBGu6BCL7lYhpiSr6xUDOUX44X7cL1f8KseUSHZzMFiG+kuUQfwKBngkb69dxYSD4FPUFRohMelpE2sqIBg7CaCKvzz2tC2MAYSOtVLhBoNrKLC50VOsICZqRzWXr2081uDUDXSyp92DY72AF1oLvPH0JEVEz\\\\\\\/UhURyEGm3ezfb9lVB6bgbS2yav7V0iLJNDYyrjbI9D3UNe\\\\\\\/xKvgm6QhHmxJaSvTvGt55XceGlq9TJGh615TAysCtHNFMlKuf1xGrkyvGSuMkAadDbF8CcLrqt0mqlVjXdxfHBwSKzgZ2hKb0UuADCOjNgDgYdiG\\\\\\\/IrYTl9x2wi3Dn41WYSbZhRXLtPHncyduQhog6zbHpSbGfAfNip8qisYoJ4leTAPonGSPH3Cea6TTePso3HLDkNdMc68NNggue06r5hgy97IFlZTj2+CBlvYAFcoo1K6OGTU7iCl6GJXDJ3+ON09iA+etnaFn48RKbDerA1fZc+BSdn\\\\\\\/+A7t9YFduK42hv749hc+beZkZ5HQAqFzkmEoVmJbY7u9xG6TB24rAJkAgYOI6pKLqHwqv7VJHAmD5caEonf0ftMbCTh95QONb9QBVow14idN1aW+FeN7RbJNdj4IahgLSlu6J4Dfrx+kHVuqRfLR5H0aTHB6CSeaFQRUMkcSJZlvDiRC8yi1pVhRTNhJ63Rvi8VieDBpll8DrZtr1sveBb0M8\\\\\\\/\\\\\\\/svUwukBEuOId\\\\\\\/PYVZix5VISPG98m62s3EZB4JCJLRWYu\\\\\\\/7XXV+lvLFIagnwh5UL3aRLfP\\\\\\\/RA9E+dXG3BivdjelEmXP6MyWXs5LcSnPfmucgoJhZqpkVDQGT28VwXxFRNx8Dr3aaojOL7QsZqgfX3Zki4A2XAA6lNTf1E3Bo8MfqnFdOiJlXveBcux2f9pRogLoy5iKdYjbBHL5D8gZyAWLRPFQNMJG2gXhTtx1y9APJjGxZ\\\\\\\/HA3EXtNnfGqJZQhxPylpa28sYxKhZt23gjXyT6B9Dt4XwzkRCZGcP3EuyUoKFawZAud8cnmn8ncRI63PGNjiZFN0JTwhPN8\\\\\\\/R2EHyeneRn\\\\\\\/pWUlpiGcaRL98bd2INbpyrbaA8OUvbNvt76\\\\\\\/ooFnyjNqD0OZ7+fvjhSf053BSWnVYpwdNKzRSep7bRq3150NfBkpAHN0mf8KZ0UOpxye9uy\\\\\\\/3UDq\\\\\\\/ugmq6\\\\\\\/ik\\\\\\\/Xqcn5JZ+4wPvHeHQIjcHXZPtTN7Q3CjwjjziwzWUFbWfYKEYCDzlCbs+RS3fgAMo2BlRlANsIr\\\\\\\/p+u7ChoFENdUJBbqNrKrNKOleZo57ibiEYi3Ch+8klmYSfDpxILlJLBFnjO56qrYAfKJxJBo6YIAabNC79doRFs3ZVKlmBnYvpLDDJ828x9UFmLkUxRMMB4gNLQR2Q0J0Yz5Oz3Cxt3jYV8ma3f36XSLOa9TlvVA+64jjWbtJ0kpr4gMo0jH3XwIRrGmwAPSK\\\\\\\/JXw3dtBoqYLh5wqZNgR67FQJNDODQEf3eMJx1QGOBckrY3cbkNqgAk9KleoDlzgW1Wn7zD4OzMnwKGzD6xcOJZzzFpMi3RLnTaIrd29HMFaCXSGaPxxqp\\\\\\\/CRiJeJtyKUHCBKv7GLla0bCd\\\\\\\/iGYqxm+V\\\\\\\/L7hAT5L4KieKCB4p3HP7\\\\\\\/KdGbIWbQd06P4hCax9qUAZCwydga\\\\\\\/MWXCfhEAg0qEaO8jObUO8U6mO5j2r8xK3fbCVXlJzrmyOFOLsauIBIZFtGDuYcjHgDcpBcWHkRbciVuYKikt3AIHp4iXfjqfr1+xRKgxvWt\\\\\\\/MUZNv6mlTAmt2bjwInWGLCBhenhAnBA1QrQNSYfht6KYtzlnQyMtHqJDYYRnKU77uvh69qWB3O04cqIQ0wZlnfnkai33RoeQlnOfn5QBFB18NRz52yRe6rxYxQ8RU+fBkxLB9qJjJbqH1xYKtndJxRWcW7l38R5LnyvsmLMqTiPm5Q2kYEUA22CzDOHQLtxUQt5yW4iJGYHm8XF5Xw+TUByqHjuWhWwfmiPu13VXYm0Qz5n8jWRzV8CxpcJKOqqdc1xHIL86pVp\\\\\\\/rUjB6dBK3Axz0fl60VIseoUpqL2qdO5kMxg7Y4aIuC+Pv9R+6X7EY3+cYk2y8SOzwn1cMaAnNhsRmDx1bCTSrHa0BsDg3u7KNsDt4dQd0lXomt7YM6gOVDBVCSt+wfZhGcveRkCkZ4S5FM4og0ZkvRfQTx7eQ1wl7ydexMYFPFHbe3p2bBs3veCuuuotrQJjsgn4kxxyZeR9pfG8jW9MFoYxxxQ9VlvD56Oxlm7BenJ1vwTDVAo\\\\\\\/3Uq6+Q71fm5wKZP2\\\\\\\/WczQuZ1wgqwbim6efumiIGzqppibYyQk+Dcwr3kQPN61XlA6Hr\\\\\\\/bnpgociXFa0DW1A4YPxa9ehfQUhMNWc2Hwgx\\\\\\\/QPvDrgffPsQi0MXK+G+HsMjV+RlN9asMUazrA54E0dRu4OdQTWPIFj84xP+Xv\\\\\\\/EZbe1f6E+ZNp0TSErHqq3wTttYp6yHZphFr7vM68TFUp6fg6tv6aRqFbR5aJWiAaUd\\\\\\\/wdBOpNmO4hbCbQI6I+AEzo0p\\\\\\\/d\\\\\\\/LOPZqRy5YtdkEiBROBVqqqEFnZp\\\\\\\/f7\\\\\\\/B5mQs2OB8sNU7NeBvWeac7qn1w75rtqnTdPui7vKgIjqt2kEcpjnJX3+DeFqBngRaA\\\\\\\/bEecu+2n7ktwm423uIpDWyurQj3IMN6M78qXS0GDPbNoTA+rR5WWqAk1nrP8NaULTcwlogJulr7V+V7FL+Irhyd+mkl\\\\\\\/EWEdELTXbxwEj51AK6thBYLSGfC+4LjZ9Qswcvs7QbivACf1Up\\\\\\\/AuuY4YwSoB3b6TmDFu\\\\\\\/anMUsknd7\\\\\\\/\\\\\\\/QT\\\\\\\/m+vMsLF4Q9l7nJlADFmwjv7bEHYQbDiVDpSjO9bCSN3HbVZRYgfqKDAxvSPLY\\\\\\\/nfQGqxBjPLmgaeiopI8gojxHd0WMenp6eYl86bxgoNvNtqRh4bcn7GV++1Gt5vH0wbtVI8LLkXq1ch5YaYBqdqC8eIDzGJKootNAI09kon5FO39Re\\\\\\\/\\\\\\\/IDRJCQMqAoHdfW6stKy4Xd7k54soBB44P7IGVs0h+RQ\\\\\\\/wtV9vWDfjjNjr761bpqXH4+V4N2rFtTLEBKie0cFqPbD56Ma9SxufhK3zMrrbdXdFIckA5GIo0hwxS4VSBgabbuihlSNTobOmIoniPUs9eO3Mz2Nc1Kb4zpjBzG1upRdIxH3Qy1UXlbEDB88yRuNXxMtIbG3xqVw\\\\\\\/yRW0FQedFs650n98C7LN8syTeDiOaFmLe3+kUBio+MCxnaowRfT4EvfOfXl60VMU4A0Bnpoctfdqlbq7KA9v\\\\\\\/9CP8XAW7gTCjtpX8gEYi7GXfkjcMtU5dflN2JXtdG06kDjnds1VvfG0yD\\\\\\\/ojewGu\\\\\\\/rD3KyomsUS6vbAWZbRdUIDn6MUcW3qkR97Jg3JPlWlo4nGCn9c3tkZKhmlJmJqTehfTPbl3f\\\\\\\/nrckBmGn8ktZnqUl\\\\\\\/0GaSAKONS0NHKOFCmXqlO6fldKYU0AEUTnMGd0W2UExW6\\\\\\\/idOqREwgH4V2QZQPlKLpItWnfTE3Dtk7BNQEzgOeDldV8Ef6Z3ULlHZKCiEeDvTjlUjlj\\\\\\\/hCAnfHFjk9XuSaiPUacO3eThtUAdGrVDBeMB\\\\\\\/bI0FDeU9j\\\\\\\/1XWYU43mt2LZ5AsidvOWnNBwPKNIxYw9WPSU5igKiwRSqldjjMLcdAY\\\\\\\/wcwCUg9eCzbkq550D5YGfMHwDJdGvk4nSRQmj+mGVyzDvadOwKV2FLN5Cyg7K2j1dg2RVgivXIL0RFvfmJMrMqg1qym7iyG2QWQE058Pr5PSWOyaJNCJfC2iXLFK3Ibk71gt9ReJqeU9cbZQP+C7KZY4R1CbAEP6+6QRfaedJBl9Qq+PyTu8GEu7467LD+u8g4z7qaGSmpLr87paj4YoEqaEfBFXYin5NTqy\\\\\\\/SHz62mJR2z+wN1IB1tZZpQgzUmfwK9zkLH5bb7l5qSSPDdVtatj0i8vLATtfJ8SreWc59Ya9JRdl8Jq38crgjKfJh6drbbiWClFURLdv3NuMxwr9fs5ilSHP8olgvENlwW0Odn5Y\\\\\\\/Wz8gotuZd+6shLKFyx8XkEPDHYNlFBOqC5lzEmlOWaBsDVBX12injuYYHoER4u22eIy+Gsz9uFgWVEoFu+jQBmCt1vyRgKjgGX0I7Fj9ZFxCfQaJprW4xm8hCOnr1keSeDhJXQtimsTnNSn3fe+3brh8k5qC3DSsiIrFuGK6evB8Hb6mpET1YFnSymDczl6QziC7GrTQMBhgVHdjhVHaHsh\\\\\\\/yjyfdD\\\\\\\/bOUEFcpcvRcvgMzWbPiX3LIreZabMJCZBY3P4Tp01QGLSxPksbf2X9EUXRgmgZ+YFyZfv0jt9DZKdxjzOHjy6egHuFCta0ijZfXmGh92JdduhLOQYdrP36Gog+6M264xVgsaZJ7+Z+Epptub9B99CzGGSRfP4f8BQB\\\\\\\/RkdWRaq14rv0I+3kULDj12FXiet0IlXxqRIJFF3iNKDnCwuCbMxAeIrWjAhpMRG7ZEoepYlJ7lzxABqsUfsAYBGGJUGH9vGCDSgOOlGsHXNbCpi8M6FjbWIaccRG0duvR2ukXOPTYETovosun2cu74yUErqYE6k+wvCq2CkrpKzleajTPz9CkrqLy5JP6w+xHUd2Von1nMetFi\\\\\\\/f1TyJbw082kR3CDFKYGO7ut5ZsrW5ZDpZdUWA8Nfx8r6kaR40IYslaLniQN5CZhRS5GOWmGUeV81nOoCtabCNh+WG24v86SWB8pM7Y6MqJdaqjzDTWdAQtcnv1ZF6+KXj7V4rT\\\\\\\/m0crV6F\\\\\\\/PbFHVkiAGauCQX5khIYhNi6U8YPxaGK\\\\\\\/AlVGDlHcv45QBypCevvgfUVb2caYWeDmOLR0Xl6f6OPaTFt5\\\\\\\/1ckB8F9kYWR8TxL5PhvL9ep3SPCbiR4xTz8Gtom25oNQc3AIDxRYZ5M5FEvDcZh7+RuXqOv23HceQl7xle\\\\\\\/Jz04xbbx6DEWSfC3vcDizWybWCoydf3l5vdrH26r5i7C3uvzjJl15yByTOho9ohK9RLBiyJOoRctZGqA5i4+HfN3XWkhT3y1EKViSzUqbsSf9n2JuFYsbLE1aV62823Vj3NhMEoYSacEmRggwCpg6L+1yS5Qrx6nL9CRF5odBXuSCOgrJ6eJRDmg4Hx5zePbXI8VSxOGR6lOMCKqXAJ3Z3bDfO5m0kiTSVQYxU4Y\\\\\\\/SkL4VU5eaxcIkkSETZ5f5mOE3H5chN\\\\\\\/nX7Bd0e81wckZN\\\\\\\/\\\\\\\/VBTLsgNi+i\\\\\\\/A+dIFcZcgAmd2ZWHpJrH76wZ\\\\\\\/s1kpKOVgim2+6LF\\\\\\\/XV5fbi1urUPxAhX7Xuf+3KdeicBX\\\\\\\/T6YGiI0ncgxP4RB0WfEkE+lViXxpPpjJftWE+SX1H00Udzw5QfvoSjBE\\\\\\\/UpFtbCWCfZ90RaLqnbNoURON71Vcvo0gWsnmvwS+S1sjA1IIcuScY67T5m1B5fLrpB\\\\\\\/PX94zlVMbU5Ws+rvtvK9v8l3tmfMZdm+Hlhl0tI\\\\\\\/NH\\\\\\\/JRNcnsM4Poznck0XPUYzX6sMBJyfkl2ZpALJEU37xjuuThhcn+gC458frnfzJpf\\\\\\\/nQiV0okRgb3YVpBQ7bskffSrMuV8Kv5UDvKOR2kYm9NPnApUlTcyVseYLBdVaiMVpHDEj30lIAmZE7g6L8yaI2ZQycTyj6S9XlXpI6SLg5HmqiZFrCg3r69VD9iM3I7SUqQeCuy6SaJRdwuXz7+FLR\\\\\\\/7bQfSjBtCYDoWkxsIPIPqqcKBymP7Kis3T+ZLDC1ECYIvVX60JLOh+r0zp7vRTRfI6GD3YV9g7kY6sftZNMTzYkmfWRu2Njm5Z6ptYyrMhSigefC\\\\\\\/leueOTzwc7P\\\\\\\/7\\\\\\\/gDrSpHc+avIlH7Sp\\\\\\\/redeTDQuohIrAqDtv4sadFVuYefXkpzJcqEx8oIvYTsLMybPTugySRnxNPSmTLc2Dgyx4+\\\\\\\/swOsu7c86jWJ8JposEJ2EiZAQ\\\\\\\/ydqcVrcnfQp8j9bsdE1DnJh4s\\\\\\\/rM9X+Bjtp6fzXgqs6kwnrpsLJjy2C\\\\\\\/m79kyd+7GXtk1MPaSkSJLlzoUgrCFpjBfURrteXtDYXiNlkN2KGc\\\\\\\/ErcxG10iTz6gTol2nDOoOrQmCChi+L690Ligvt1IlkkuCRMt9grKOuE1+3oZrnRdTp4cyv9iUvI5WwSYv9d7OkSNp66l1y7P+1OYUKO33L1l1Lz2s9BPaLqb3ps1dsR7lOY4p5g03No6eYo8hYjcdTBPaWe2lP\\\\\\\/THUNjWSpfLZIK3BuX0vh2CkQd4rSNeRVdBtXP1spXPV2965CNBiz1Xb0hiaR7f8i6Q2AgW8nfXQGev+m0xWCbQARjsut5qZEW+H7gljMNBtvGFdVo+2iuVkjIjrwHnmD9NCzM6idWxi0ro\\\\\\\/ln\\\\\\\/r98KzyXjLB+jG\\\\\\\/foyNJ4A1444hVlYye\\\\\\\/Y5Fna\\\\\\\/SO8RICISvEaBU4\\\\\\\/rgvPLCamR7xKZNHxZPAZWQOsBmpZ5TqzeEkgdn89BMQp0E9n\\\\\\\/1u4BNmZQWRqIDle\\\\\\\/Nh8Awna2BmVXYvu\\\\\\\/0YuEkuI91YBNov8ezXdF\\\\\\\/3ZPbaCJF++79nW4oXgbiVxaadRllOgy8gM5I4\\\\\\\/ECCXqYNIq7sQ7VdShJrOQwDARxX5zKfma+TCmmijOUi9e+E7tQXzDoq6kUC+pQPbC7iBU6ZB1T2sxuq9GPcyME0ApdeYngj1yCzk+GLlTkksDfGOdra8R6EoLe2u3hLEbDR9+gk4reIE5nDYq5rnOMie\\\\\\\/dBL8svNVxs\\\\\\\/lf2wmTVhNV9ci9NwE8LLyNU3PeQrTXf0AZCEIAZA4juQMmlPqmzal4PrWiM5\\\\\\\/VYws2dbMkAYeCTeW0toC04INGE+5mUc6htaTiLlAiIR77W2\\\\\\\/BVDSVVafsEstGmqfiPmrsdF2lVQPHmpyUAT9bksCOc1+g1FujKmLLXAS4djUZaeqvqBs1cnHv9\\\\\\\/soyK8auAiHGcrFgk8ki9i2tiOtfp311ltEfTVvJOX+p6ef3ovzQoERyNZb1i9Hwf3VXG5udD8U\\\\\\\/nozLj3ex22aLo8k56FLlqeNDjPX\\\\\\\/0STJG3Jzi+0g6nxUf43cSwdIDEDE+ajujNPvaa5o78TqCexCKVV37Zkt6jZ4c3ohofauHgprWhq8d8T2rDt8ooayQiXBFFWjlDJgBVFMOy26vJcdbrsUGJJDRZ4hKnzN+KXzsEW3V5Ks4YYBDu2DEyZuyWmSMAk84sk1\\\\\\\/rd1G1\\\\\\\/JHIc+JTk7kABL4kREtkUdPIMajomi5es2q4r2Ie0VkQRfLnxnvl4ARKOympDJqk52M3OyjpsEhCClGSL2ud2MJ+9+QMW0M3G3Fxs+rCCEc\\\\\\\/gPj3uGedEi8BGRfFIJuaJWv49Y6axTTbbBHI0fmoRj2W1lqkxZ9BuWUIWcEvRIbqTm1uyFFz42j+ZSzVgwK3kP1K3Y1yE\\\\\\\/FJE1hcStnYrr70QXIPMz7wgJTnF5Mi+\\\\\\\/rIikoBcGHS2MuDrMdbD4vM2P5fh87p8VVWFXRV10UxvkPyHb+U36RvL7DZlfQF4C0r\\\\\\\/sOAPZL4yEOTj2RXDv1FMGV1e9sRcm4ryZJ7p5xnj0Db3UXM9sSk4W59XnZd2eNt7XzRS2OR6jpssa0W4sI8WyCKSzntsklgX7oY9jyAteLW6VXjiVjtLhWoBDkalu3twUREq3mW6EyBl5dKaWrUYxVxxLeLAEYlH5SEO1PxX+pqcsQKw7HTF8\\\\\\\/wLg5iZo7g7QbyEXMzK7KiaREVRKHVZRmpU\\\\\\\/yJg4tEAA0wFM+ktGSGMurQDIla\\\\\\\/ENnWdqWFWhaybU99UE\\\\\\\/9uoTrXLgjPb+L6rKLjqM4kwQUI5fEp1qNQQKSLGjejU6r\\\\\\\/nFmTTnXT2vy8voRarxX3ukbqzwgr8n2lKJ+z8hSuOyB1IygasKngDwTCGIRxB6WRI13qlcfuN2bXrF14yNB9FV4iH2BvlKheUmi7jPkihtOVp4UHzzZLnjKpQFjWVWepRF02L\\\\\\\/1XjJgyD0VXoISsmcnq+RwWHbKJZbM709cnyeZiZWtXfMzMh8SwO+8aVyiicWmCLqCdLRZlEFje5pNkhHo53t0UdAycvMOBsSQZcaYSdOxt9eBxBFFiHDu2qd0d09jK14p02PavoCAWSVQGFhpt4TpseqtIxEDlPFSUeyRg+HrVnfzhZ7pvobW0DB7gvTVn5ZJrPbL0hVogoB1fvZ4t6vZofJuL5k1AspbQGSj51PfhuqSjixDe8OTiUBYXlL\\\\\\\/IxH1BszOXwhF\\\\\\\/4Y37CST38CMlnUVTkSG\\\\\\\/GuhyUhO7Ky\\\\\\\/0yZcaCRYCalHoEQJ5vB6DyKcgi5\\\\\\\/oEAcGL85wtlVl3eYjzxWVLv5aBiluQ\\\\\\\/kpy0PIPUyQKxpX3QHLm0bniTK56\\\\\\\/8X95FTbMmMjtMrK6n4ig5soZzd8ATJG4gyDwh1xHshl50Ybsndk2pEH107vSyrart2nPYDKPzEJ6\\\\\\\/dvfN8OVSKhSlu84y+JP5YKFXJxYQLluauQOOD\\\\\\\/glFzNZMy1mGdis92uKYgDIwn56xIqVyWAETgT9feyvtWHTvcynakPj2W70r8pBs4LbrZ4oZydhqhbcww1Kjnv9PKWtxXZrPiicSSyz88ZPWgoL\\\\\\\/ca6pCbADp\\\\\\\/KfP111qoWktWILJTxu+6NHUfQIHlfaMhk7oKAyFvC3pRFK\\\\\\\/7chZB4lZPRYBSRmpfC2I\\\\\\\/NVyKXHTiC82EW5GmL5YaX8DKipSL3sPWY4WTqwmRBHDct1rFQYDGjGiz7Adt23E12KDfuDD3cKzvBlE514FD3zOUlC9GbICRdShvMb8OPLYsEcAMJP197xG8btbunPhBPlg58OaHttMJWWCPhQUywpzP4FUUuzFf64eioLzAVqEh0R\\\\\\\/a+Tc+MPmOfsc6QrjplZbcaeiuRGsYZn5iwuX8S9RTZWRcw840Pv9qKYV4uy0CsmYuh+dLFJv8\\\\\\\/7euNtWTPrlW0cGDRImRIbkeDq1iWf2z7KmdxfXLProdLqek+K48QXWhWgI6gYaGwlF5qm+Bq+HcLYv0TlTbs1wThx9G5BtqCyzQ++BdYOBeA+b9kaq+9uzfDO46G4BJc0f2CbGyi0E+6Dr10FX0uKn\\\\\\\/gODgim2b08pAmyjKlKI9OFwUk6Lp1L4clmF5FcNsxrovYNquHy6aynzpx9XOzCDv5luUKSgw1dst0SimBLhFBcaOl7cpbI5mHG3g4kAZPJJy9EhPm0MdyUO8gMyDnAQO7o4xbZHnRDAAA8t\\\\\\\/msBgiU98kDVSvJ7TViDhm44jvJu5EWAa1s3Zfcb5RMjFzk+NbU84NbsIXvgrFpYdQld+yXvdv4pBQ\\\\\\\/7oGBq7svE1hktQr5emDHcgojlqBRrZ3X5hdY81BV+wwK8kE3Uah2SYTmk66KNKiS1G06uudnWOfj3buhaYeStXNpe+3FcUA3QvJynv2UGWgmlkVz9t7aCWL4CNNI6hW0NIj0eciivKHxMDCbf9l2BCMeIGRVLqF0mOXOAlQcG6tI91eeLdo10I9TZ6BCFcOqJlftW75fweK4U2XyekwYC1ieladdV3\\\\\\\/SOiszGSqd6K3p8tSsNfTGfRUpUdvIM0JJCPAUOLIxuvjFmr0MeBDPDD+h\\\\\\\/jH0tAbvVZtD0CgXB8PXpP55\\\\\\\/Eb6FmPSQbL3tgHqCcjiFcSwj6CcFm3fPoYxbaDclZtkX\\\\\\\/3BNinuCc5rGXsD5g2RwhghT9xEKFZr76+o8Ow0IkwNFHudRHAzS8JqeNEKhOLit3C3Dj\\\\\\\/rkyOq2OBkWBLW\\\\\\\/PbyJZ2a\\\\\\\/4Kl\\\\\\\/GyJkanynY9l76W8+gNVrjqyxWM3JWtEXkJszsMCt1VeP9yDkUBzpMKaPqq0IcJgCHtx65sFJ3F3GvVYnGurSIOmxR6ZxoexFqb1bHGz7Pm96HLSbvCxPfVhIQOaZYrNbbpMVdnkJ7LnZO2IEQOYOdCzfLleMgNGwCB\\\\\\\/s5OT97I5ZlfeIWwDbr\\\\\\\/Em7g9Yo2WJWYQzY2qRSTRI2oX0Agvb461kFj+Xtxx0Pv6HHs0c1oFSkEzIbkT4llHrtIQs6106XiYgKBpmcmSJNlZMFBvE9pzb+Kguypku7NiHoR8gpEp+em79fDHjUVgzlrVksMpMu9mvHca99KJRLKZZoeE39Rf\\\\\\\/RiT0jVf8H7ZHTajtne0DJqUXB\\\\\\\/TXBEqt1Q5g\\\\\\\/Zal569XSUCqPazYdfEyH7SmHWbek2ovJzubz95GJK03tBajYNkEhgBKfZYMqqJuNufBUIapKk90yuvdbmcpuVGUTapZyGWtTKr46fgpL1YMfuJPlMMJUCt06m94iaj5Ic9bWwlFU4OQ7YfF0GSqG1LzJoCMfO8Rn3j9GH9p4sX4ky54PPOa5EACUUol3hit+Yq4uVthvbsd1qbPPV9jIAVOL5y863hZ70qNLsdQbov8mH9Gw+USL5klypiFtMEmL7cQU04WRZvFHeewIKjw+3ROOJByvsqkksER92PMemfSV1l\\\\\\\/+YnfstS6SnwK2tFI4EMKAb9DM\\\\\\\/QZ49fC4tid7mHOyQNKke4I6mtWPQhe8\\\\\\\/egMQkNFp9b2EDgaSbgleHSI2Kvm9SG7BlmceRPpL4Erwg8O5cl0LKGJPm5rSV1gx2TvbVVlSRYyNr+x4gxknyh+FtdP1KA0iJ2C5ARAAuGey2szpO76XUBYAPMCRRKWq7fnPawsijAuB0cAEYYQRBuc78ZbW1kVSE5eCe7Aljq7ddEJA25vEy9TyC0uZAfJ5aL1Ye2jEoRI4HjNQIVtPg\\\\\\\/gnheID\\\\\\\/E+V7uJHdOheW+w\\\\\\\/qj4aUEv8pP8EkAmdGt68o7owoCe8ivYMbUkn5GEGcEoCTtQZXRDxyEcTEtrlmSSRAjxEZpLLwxlqa\\\\\\\/ytZc4DKyq\\\\\\\/NSXubz3H35ifNHqZoIJwxmueDjnJiNK2TFjdl096KNoaYs1bady4QIzCevk06myxh9hJV5XwY5o2EU6IPJKgM6Dtb9L9yXogv3RkflTs1LuE1JbKdIPOez9bafg4l2d\\\\\\\/C7XSlGRA6jX2kmh0UqXzZCjSAkWAnSFTOIBpFy0Z90o91nfNlhn7S53S1U+8rZjWEKSyJZqeCsOLVslVg5DCFjsbsShmMSB1hy7nOASMNHGe\\\\\\\/DwDJOtnheXBEXilxfSG62p5Ir7eRYMeCNsRWQghAE0LLGvBCJlnvmjfAS9kFeIBA+VJD5eKebIeZEAfEJRVjOycTAPAhwmqltl1VK+XNOe+5JYodLvMQnm79Q5cCOvVM0RHovUHcFUIHe7wynWBajIgy3kF+CmRTZU8OTF0l5d2CL5JXHZ\\\\\\\/9gLuLYP4pGMWKfi8pPItMOwotpOF1\\\\\\\/atCpFkMTyBJCUOydbiu7UBpm8ijc7OJZgqBttYiVE\\\\\\\/7oR+GkFLFIt831IKYKKDN\\\\\\\/lDPr4iUonuU90jX5ph9kp6bBUQ8bYPX+USExMKMHEAFEmQ752QfFa6JwdchQviG7Ffyx5cTJYfydwAMFvJNL3BVUaxec4ZdBNGKlQOZunzu\\\\\\\/Es6b5mG4CvaRspIloVfHMaI6zeBojxOIMTc7xNWyOLlNN6fWyNtghODCPl08lynatQrIZzK05takU\\\\\\\/6btdW1ED\\\\\\\/TwvOlzpo75fyeNfjBBKOPTLxB3QOir83ORLbXqGZubFwt\\\\\\\/IDRtIe\\\\\\\/lpZjeXSfOGVHXhkrFH7vorrK1oHqOOk1KVPTGZYr9\\\\\\\/kX3MCyrI3IQ8bdPjHn9TBWli6fV7BHG7VP921Ywvd6vzwpFviwQ3nnX9jzYwEY+RpQJ4mSk1yT0eABKp7BBxVEOwrNnOuUE3CntjfiZdYT3UDT5tlje9olLObloYnpcyBnArzh1cXen\\\\\\\/4z7y8vI1+USO9mArPfaHgedF392e6ydFdKnFfe90xkRQAi2quoGGbZJcsyi3LE8YY94iRs7YQeIf0BUyVqfNOvckkDawAXklUZ+zOYY2jp6jkP1fcqWf380vETEaoaA21HinctylkoTj7EMtdXYbA\\\\\\\/zEUvqJYNPKwnrdycPfu9e9Xc1F9j0VtrkdSvJ+F8NpVQByKXovpskCoU\\\\\\\/3VioJh26sgDfpuQjTq38DL\\\\\\\/oKTO2V6rF1fbl9Umb5S8jz6tAsYj59vWPBwFIlMtUJp4CgLstSNduJLDPIyMmsiiNhY+6yluuuD4LXESmZYn5JETGhUiE6QPTHPUnnEoVymqJSoYedTlbhBYwR2hZh9M4KNXioNcO7UJLsWmaFfXcOsJvyF5RYM3xZ1sXXaASBMVU3KPbp1GRTeBjT4vw3j2HiAYQ422+rnKA5hA9bO5Xqk1cm2cYYRid9aynKm1yPjTaLg7vxN9flgrBfmL1QrUElo5UI5LsJ0ae73lQJ5MgatjiMNsI6n6xiGNTBsrjeBeWHZm86\\\\\\\/mhXAnpwbA0YDYtl4Xezi1h5uA1MhuUNoxzR4FVomTA\\\\\\\/zGTTHqUSlxGZMHCk1DHRXDGL1WIW38BmzsCvi9wTIpSMOMcot7vQUvL5fEpLieRGY+6OIut8OtGApB9A5C4v2qpwx5pHyJhdsiExe4MQleRgtCn9Dg0DuXAJJJFdWbP0qUBmRQZUL6cYFaePTwkmLMEzcWGrH1NngOYz0WugO1P3CNWUZL6gU3q9qqqCAiquHamRhilkEBs3MYUNpgGSt\\\\\\\/R+m7OJynGQbVZZmsV+CRJOUF5Mw52pgwRmmkV9Zy3ks4atCqK0Xi7iHHHtM9+tiixK0rd\\\\\\\/5chK+2owjR7Sca2hl0oiY+3lecvl4X7gVQAu2FAep6cNustRm5GJcquaZL0OqEooBQRLG6PnuiyrWrCH0n76XIa69Jy7tgwkOVbA9Zjy9CM2O3wZ5q4PPEIXNPQV0UKdtMXgF32ZOnvhEc67kAJLH2gAoJ2VP32CJCF4TReovJufNdmXp8DnFUkSgqzgjWAlM9N+J472skMREmmqJ1u4QiNuPlcxfJGe6llIrO2jC+2LrlYQ7FZXiBi3TplNuiKpzx\\\\\\\/+nIli2U1yunh4yb+hFQ1ck3gnDrAErKWVNayVDldDZevrMBqbXMZh+KiTP1iioLKURKRWLuC9UF0hTkOA\\\\\\\/1+viGRzwuA9fAvJEf\\\\\\\/yPcYIfpr1MTo45XCWFaJktBqTkoViNOn91h6Leif9KUW1678IYbA\\\\\\\/hG7pGjqafeMc9Kpg9XVwmg95KGRsmVPphU\\\\\\\/WVI1d7HrfEm8yxqd\\\\\\\/oQJIPcptJmjmblnXWDY\\\\\\\/pRuhfusE+5t9VnK+Vk36p80dw8xcZ1bum7m89ATisYnxXJCOqH4fAn2pYFS2SmfeZqRAQt18wTiKFGjP3qp5aNh4IzROfM95XgLIOxU0H0D5WxvzlN1AVIH1UwH8zane2fm2JaFgWcYJfSTB6mt3AqxZM\\\\\\\/jiM4fN\\\\\\\/XYWDSBEF9rpxS0rJCPs6fmCnQTH2b+P94jpCv\\\\\\\/B0JMy+iXzPM2yder64IlOAirIfzhzlIQTjsdZYBayDZONkkU9\\\\\\\/4tflHl58YKz6YtBXpfgLAyS6UG0pLjZ5KwcNlr8KOeDioKqpSKcwUb9QxYdHl1xtYXs1GGVArc+VaXH689cxKGiHaoFAchLQ\\\\\\\/HRs0ITVAIpArW3y0dpjQhoUuRekyIvdEK8TYVs64goys3yuhezEKIeTIQIdKAEOHTGgJOwfDm\\\\\\\/Bd\\\\\\\/Ifqvj830Zm8mXcd+iUADIoaqBYTbocPZgBJUGvtOVIIlj07omWVjBN+QXqm\\\\\\\/nBuZ01Nz0htHXbJZbEmz2NObDS6SEx4cvYgbbLw7JGPXY53sf\\\\\\\/s+dWrBRMWv8WXArM7Vk1W5Y3QzIsBjFxLTOLY7gflxz85BQ3AO\\\\\\\/wu4QFzMjtc2DMfNWwaR8jkIK8YCPPvUMTHVzrH0AyzqRYpB65MKoqYADDZVyEWGUuRcqLbUfrzZlPxp3SbgyHBQK24Fxjs2EvRi8xRNI4zjY2e0p90pSDW6mkFhEs9FauqLOgh9ACgxw\\\\\\\/cwY5NYAYCt3s9YGi2XlyANdv53ZBlTPjBAXTruvgRdd6Fr8QuOWkD5Ytk9CQ9gO33Wo7SZeal\\\\\\\/SOtuTFWSffHSFStzyAF71WQyBzuW5qKX9jB57I09DS3uh0fqEzcBwTtKQhFHtPTaTu0pWiY0XJg8i+i6Ad7OwMf3L9qDZZfvgl7SY166q8BhRlNsqy6jl1NPlH+\\\\\\\/dDtXUrJLuyxCSjqyHwmGdXCvoLq\\\\\\\/c\\\\\\\/aD7YkSJb40F\\\\\\\/9Geu89kBHQzjvioJkygDQfV4VsxD5zfVCp4ZnpQSWlzC3g4GTjM0lvs1o\\\\\\\/bmiFHBFtFtT8RhjckCVwmkHfDFJYQ9pWTt36VUvqptasHWSy4VxavWU8TgkX0ZhgnFAVTae33Z+3fUiNrWHKQkOkfGwrXxEmQ3rOKNycCrOADoVh3W6pkckS0FPw1LjiUjNU\\\\\\\/ub4ZC\\\\\\\/onSKDEWJm01Tb3uuktkfCePFdXaX0qrEpJfFO+2fAc3LvEIcEUoWh3veRvy9WPVCuwdnHDoOQ009gBzEF8TRXjfl2adN\\\\\\\/EVg\\\\\\\/BB54Nv443mEt8GHTw8PfhoEfdW9wDxF0tUDvtBgTSEdF+FyRVIuJoj+G+nggfCUF9mh5SAFDMHW296y1rOAP0smEa0fqDTgPxanuAA\\\\\\\/ADIPmdfUp6dCaFq9bQWb28CNdt4jvzatTRmkA62Mlm6GERWeCPs55vv6ZHvQi+1HnbqSvYM2cDcWvJibELF\\\\\\\/yPbp3g\\\\\\\/6SL207YdPe7PW4mNulzg2HjqoI1LqV6lliIZpplaQSE+Qc2frrfC289MahW1GlKnNcUe\\\\\\\/6XkrEmrZrYeLQY2d4UQ5p0ofL+ZGoa95fGfrjshTQQ\\\\\\\/nGD+uV5oz6+uJuICAVEH9xRb74h1ePnlHFzrJjd42o6Onbuf8MvOt9NwcuCG3BJKws+YLdD9U3u4wQ0SwvzKRv1+hzTJh9+TSkqC8qmaaSbHBbFm1IiXkfCueUA2qGpCBVM9JKLEFJ1tdrIRweOrOdRsP7gAiU0dFZqH9rUfQzVZJ9Ry6P9vDDnE1rSGgw4XnKB\\\\\\\/+6b+PPHUUAVVLlncElMLigXe14wMtXtoFVOthvdV5CQZAl2b1GRUtTKbWcAeUwjLYwdK6wMwuDgccG+JOn8Xenl1nqfPwFZ6JobUsFVizSUGtankHZRisSM2tfizgjN2TP4qqwY3iZ1D16M6cOVlMIxrzQUQa2XM2idxoT\\\\\\\/bndKsoGjwQpT37KkHCh4FTcW6X+llEXqzxGbeyZu9xrqH8z+bRHbrCO+YI\\\\\\\/6RW7t3KLF\\\\\\\/R4SC5yVyHQP3OSuikiuDNyEu3l\\\\\\\/7JytSGuOV4pys4bBezXtw\\\\\\\/hXLKyhagpSG1LwQRj6RuzVYnfKbrdvdPswZ9TXYxilzCrQP3IXsG7XGBdHov3RHEDvhR06dJo8\\\\\\\/8x0bgCjzmpqVYNrEKVWMNrYR0lkjx5xBCUxmUFThXoSw4PjFkTDOYGbV76USv8uwnIB6l8dTbyhi+1n2M8UlIbGX58dP0qQMVRoLE1pbjvvhPVQFlKLjumvXBZnSxh28bgal4myzbB5PbtFpZQon4ezRoZO3PekrnN1cbug5dOmGVIDtVQSkq5yjtFhQ2WEgRjffLynEYHQJe+22al4R5pK+CSuRz1hR82xG4wmvqwaIUru6OdJ9O6xEiYkQhZMfzGt\\\\\\\/saeJwO8tLWCXRnFsyEmy6hunS\\\\\\\/ofUljWxAR\\\\\\\/n7DWpH6GQ+iwUvfLolG0MusnbqkEUIdyNO6oX7vnm6eKURFZwLapvVBfD2IUCXBIdr450Pf4FRKR6Blc9p18mOWKpBa8p0EBBCp3etfbi+N7Q0mbf8U1zaRcet\\\\\\\/imTLVwIopCpK9uBmDChAwRLUs5kbnysxZ\\\\\\\/pEOEd5papXo+0Ldx4iLF2pAoK\\\\\\\/bejZYokOspw2T29SNqFDhJmQYOR2ctwINhjBFzHtkzUd4GIxfX5v3iS1MUg9QHw6\\\\\\\/q+EkxkFFP6gBez1pEYCoy3LTzDmveS6gj1eYXODisemlpLWJyEhMqNvzOK0MOjoWS2zykNZ1TKfT4DyfGCZLFaqDu\\\\\\\/oMqY90\\\\\\\/CEwB9WJqFEKSQZqNTbCW60cFa\\\\\\\/zKDqDUW6iSry9XpsZqUj0jYbqedAD9ZtlopYkNJGnYDnFGYgKbLqjVbEHN0D9rNrK6nqwqA\\\\\\\/9zOFZNPPeMuEhNO2Z0weexnF99N2Rnq6VH0stRYCxMvlVzpAKHWJhjWmEPFLF2MxwHkIMUclPKEzcP\\\\\\\/LYRLnoQCx7J\\\\\\\/8Q7hLm5noS3PIxs+\\\\\\\/6cynXNW8gDSB1\\\\\\\/+hWzZBzTt7FA+0mJBTeHbJvyrvVNx49qjkrpXIzwYVaaV+\\\\\\\/CyPCOd45yQNEazBuIbOz328fg4Kf8eGVuKvud+3jZgLecNAcIH++V286bmKsbrevWNpXabTlxJc3XlLM\\\\\\\/w4LIwNYI9TJB+Etm9tAwuLCwiNXcWzdYA6QaJ+48mmBmkWpU+12NRORiez7IpqEAcfwT1EMy2yv8rJ6P4YvKnSnZPQyE60k7vdVk4dLpmYCyDspv7L4zh+KKw9oFgXAWIpDwo4HinNmR+Q9AqJQhP9quOctT7\\\\\\\/\\\\\\\/PR\\\\\\\/NduVIlSjRKOvHtNDO116u7TaOJiHyenwed0RmCwbfVj568yheY\\\\\\\/GcsBF6CYL04LZPSeYIYyBwwNoQDQaECSeuAk3PBhnIzLSJ5xh2z+kV9zicgSwF7uYX+DUmXDTUDxvpa00ZIKW3exhel0FsUE4TAoHM11TxlbgYNh1ETbJJ9+3zFJXnrSW1cqyVhoZhsXULLJpMLG\\\\\\\/7tzdl9taFUt2CWiAuVgzjvWaLmw5kQAcLI8Dm931XvHfsYxEd6s3+NxyJzJpgBibNz\\\\\\\/O1yS6QC+p9\\\\\\\/L+CRQBbFjF3hyRiRJTG2v3hiUjRIZKS+5\\\\\\\/CbI8WrAyZb7p6n6pnZi8Ef4GZ2f9e0c22RAuoBomRYbqQmyU7fWOq+gOWyZrA1lu17adNSB\\\\\\\/5VNRaP9djk7W9sJjBxalUu8HaojHSxPiFyHywk1Kh2nMf6OfJT+svcW3QI5caOg85Npvz+gf9pSS+L6cDIVj+2PxbDCaA+mHqLBuV19CghxRNJWmAOrUuCiJODQT8k2wbEVFH3aWduwDwoOGR1ai2fUnlRM4ILNIGJw7eBvwA8Xb+2YpKwf7IyGlG0FI94SwSBc\\\\\\\/KoG\\\\\\\/Bc+Zh3uiT6oiMVJS0tH+FPdPWFj6jMNOOxnjDXDX1\\\\\\\/HassZfe6lHG7fu8fKwnoceRsbgmAQTllhPenOQGPGV10Ofb2qMOqhwo74YXgwOUMi2GY\\\\\\\/9kvXNPlNEThCuQtaEjSXKpNy5NFCqiB\\\\\\\/ZE9KXCG5d7zygGtYMsZa34A\\\\\\\/RIQs4HIkawbe6J6kGa\\\\\\\/CROoDRXQ\\\\\\\/Qymh0XHBMcxXXapihexxKr9wiPY\\\\\\\/ZEIhCgHUPTiCotru4UCGWDk3djd3yhdgbv2z+1ztMboIbvyYVe\\\\\\\/pgLCothe7Htmbf25i6LUiZ+II7+wZ4fwVDx6qFSx+VLoxYld\\\\\\\/F3DuXemJ5MY+01\\\\\\\/dzNzr3swoLqRcrQsSWyAcGk28YYd3tjTlwulRLezH4QElXWszkIA3jWP0HrqxTBD\\\\\\\/GxpnLhKEDVTMTbBm2to49ZPxnEb+A5B5ZVpkjJvv8kUBb09t5fwKUmeM3N4G1NF+LSlPK+IvUH0xueGCoJFvzri6O9OdxK84Us3U0d0QMSEhqGc2h\\\\\\\/lABE7MdO1DZooSs\\\\\\\/2K3K5XyOzfa8O4iXRhp6TGfGPNj7BJ2I0kRHH4dpHHFSBRgO1+WhJGXSVorfyg4gQ9cxzRojwkndNAjV+Z5J0yn7amKSdyoZHMDscyN0fYogRteOWd9A5xK0XONwudR\\\\\\\/uOUHeSe8GVo5Ssten7iUO2ny+S4NAkSFsJ6wc3kqGlne8c0+RFm+vs5Cmohq6xr38tUXQHYSLgT0JgFYriSyxKYWIFcpdYFK0S\\\\\\\/PSC\\\\\\\/4W7YmJMtcGEMVtut8pTyx0pmTvycdWPsgFyXaWGmAxWEzJoyta24s\\\\\\\/UM\\\\\\\/5bzw3529BrG3hqqBKfAg4Oatq1r88hQTv4UjgxvJCFJg8JAbaiadz4BG60P1R8eQqx0MKSqqQ20BhxCQZp9Z0OB4VCHt5OB5MtAeSKBqpg2Hqf8UMoo+7iH8xnTQtDXJ2ziytbd2EAfsuDWDUK6laULgDMxQZi4ajNsBr9glliZ1tIMCt3swXaFLZWVYEuFILfPypRzf9c53lrVY0lnuEhPqu7uD\\\\\\\/YcWhxPNbffunzkbdwxsMSUYTZin85C1GWovIXYkjurj\\\\\\\/6p4l+7zEJ+0av\\\\\\\/IdQEKlfB7xBkfAp+wHwArFHia58fkNQ18gkU8AswMY9duOoK69k0nI\\\\\\\/dVzwJ2IqKtut3eTkb3HoSu45JN6rBbVLbNIgA9z0mUGCEqwVf+uH2HgBbTqPN5Z2ZfeuoalNX6J\\\\\\\/sqn3021IsWa2YUw7cPrWorE\\\\\\\/kuVYBuaAkv+sy0B4DkKbfeWoWizdW\\\\\\\/hbiIkQ6B9dmYfVrjfquvjEjvnX7O5+J30ORi2lZ0eqmUFG6yhWqRNb2qsYEv6rxJz7yC2B9rFoNrQIsPcDXfpL1QCYGgKNVVdltidEmXons6SMk5TIRPrOZTOwhQtZCC3d5uqvse2RhNPz0kwlsLRL9en+oF6VBqfBT2NFj3yqbKmsd7SVnvH2FeSPwNl9lUscswPJFXK2siskXPP4sNbF7EmJJo77\\\\\\\/uPMT7c9QbsQwtuX5BXrwiD3qDrAgk5laCxb6ObFy7zCx1GKPYEfYWQowppiJyR0pCUb\\\\\\\/TJj3dpkbz4lntYt\\\\\\\/CEYpmnnCl+L6G\\\\\\\/aXKJl5ONi1WtRYNE+cl2V9OOQd+sbSEuZtGQnDo8z9YpPd2hEgODlETe+4icKiu5PyP9g6RdANCfcibIkpwRFHtwy27lCMQFR5JqJ27nMc1dXtNu6\\\\\\\/NqRV5ZitwZ\\\\\\\/nA0MAXkNPEpP9krTRCEfj3LfA\\\\\\\/nycIjDCAciiMplK5QS9T5yykPfMUPcQrS9ATMEyM7FtHeZrpFY\\\\\\\/WkNhwE7l36bHMc2ZGD9bkH+DB9+w7zMCX1zty15KZ+80IwrV5G61Xus\\\\\\\/yozM0RH4NGzWlRf7Y8PEGqQGAYlHGwOQLNFo4wamFlNqUEKaOhiCRdD5GTXHfrLt0wKpdEcshVhBD6goh7T6HXXlJyjT6Dsg83bYOLSGrS9S1XYGgJoQzbwgkXVBzIeVYTVfPm21bjgQLDYch7hOvu+peHG6iGLEQLW7ZtOEYacr6QpgkekWQVdCVrIWZYpqO\\\\\\\/rT5FFAYKl9\\\\\\\/LVWHJJKgAm1p7hbBIDHdyGOMu2Uufc9DAUWyRGD2LUI5HG0dKafD+IUaRIokq8AwinjcTqU4qgn4TR8SfEfYycdvq1Ned5uBqhaNHDUfZ3FNbpinFkW84uAAuawfv\\\\\\\/EISXsbtlZg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KLMIxJR7TGNZEovZZHgh9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022500a58d27d745299818580f5a31851e6a825bd93dc0c2af67f33771716cb9cae\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YhZJanMRZb9rwtuhDog1dffIZi\\\\\\\/rf8l+KRmj9idU82jh9\\\\\\\/PAr1P0qDDHKKa1Ldx+YYBiVRbgnv+QpziPSv8jeEznLqW7H7kLejHsA\\\\\\\/Gp4YR4rFgKygZUMDqdNyIZOdotFQGTOQ6YA11erMgvq\\\\\\\/zJDQwLTEU2qZwd\\\\\\\/UW6ULBVBAfrm0GA7ap1QiAjO5d52vIhljF\\\\\\\/Zfhb\\\\\\\/2AjAzvp9tDbv29knAaK2XkUfJhSCd81Ft6ij56pRKcCo8fxxhZPL0vYqFbBljw04HAeeESGQQ0pvDYRWDXMXxlRkyhuno33htjVAJlw2cPFmuRcfWzdmdU00qA2s\\\\\\\/G94GJT4WWD81JfuLXs3WOXfYjoKL6czFWLKWd3waIOvwRsF0WLk\\\\\\\/Lk0cpwawBY6XOn2HBMmoHe9Xr\\\\\\\/7SLofEIPFY14x+YX4BAEJkgKRfvL0NvDlbIVY9GzEz9CJBc4dq8T8aXLRk5JUkgCC2nZqA9b3rfjyt0INRQpZ6O8iHaZSjoPINuL3p1wEL4GpDkgizjQLGHqSdZ8wK4zhBos7AjGPNbN0zn\\\\\\\/n7N\\\\\\\/x5k3HELLjMUdlcHub9k4\\\\\\\/cwBlhP1IS8uKrSnmB6P2Gsm48nsJExOrBjVSNjP653KFw44auf6u\\\\\\\/wSIX6T6I7k4HEh+vnrKIltULKRLymr62W29Nvf582YenfU2kUp\\\\\\\/wCrVG5ECIJOVg1ViyK4doES3pogov45MPYLnEYNkbOVzv2EbCY0vbnyPYB+J5CDswGmQ7Oe\\\\\\\/m1sdnIFaUwO6fpOmpTGlb+Zkg8MyCHWCf8XTIiHkEptLG8inW2\\\\\\\/FXskrQ2kEdtCCl8qXbw5\\\\\\\/VrU3y9+0VDxcoTAaXnp\\\\\\\/7von0EOsob4osmEHClt3eVE+KpEIlul3fUKT1DPvwsk6wnXmUIT\\\\\\\/4sR8\\\\\\\/kccJBxIDHpEILEZhxhMk0keT7Gt3rpBnnaDTzW\\\\\\\/QudFBeCWJbxKvg6fUrJaDK8a+0InmxEOJqD3\\\\\\\/aXCZAUwANMZLr2vvQhRriwNdFiqw9YaG7thZspdXBTJRZuF1Z7u1IAuv0wP9WdfL47XlwkiNHPROY74v2GPTeHHZsihkjsqKWgQuAiTwPW8bdxBTxk2LOCnrBzdk7twt\\\\\\\/eRYT3KnqUh9Vw2Ux2wDrsUqphoMquxnViNwqv0tIlO6sWMAUDUqN5aTVtN\\\\\\\/QTb2K4CUJml4z6qPvcJ8qIbyy+\\\\\\\/S\\\\\\\/QySwSsbDHKE3HMMXs1un042QVqsP31xqcx2SwF2Je2ecWenUL2krO0FJP42L6zseeUg70QX186P7nCkVbyfDx2+vjwPPyXZhBc0HY1yjrToNRlTNklTNKp3NHiut3XcrJr\\\\\\\/9c+86r9i8wnrqxisIXcPoDsWvHy\\\\\\\/wwY\\\\\\\/SKU9pyKMOM+WJm4wTmjJsEdUWuafHpcgXL7j+fo9XnWFaa6MhsA90ht1XRQl0B\\\\\\\/Q9D7rtkQ53fjaRvjau0iCllkXr5rLlGPOxbw5MLGW+ex8qwxMBp4JMyZsx2dMxL8a2ohHSD6L+5R7HG8CJl\\\\\\\/Q9LcIqqAJ\\\\\\\/yeQ5SQAQF1ZEMhzTmDTEewO8VE0XGHJVyvgRGQoz97h04PFQRtfIFXGscNMbVZxoLcguJh1fhUgHUZrHZRRxUvL2VzUBGAETe51cWh9xSBG09wY1W2sNfgvqBCCXnBDLEYCUOuhrkiRZUhAWF5vp\\\\\\\/XfrPHAOXMmFiFKFAjcG+mz9+9p7egPH68N89kyQrvR2mOX0b\\\\\\\/QeLegfmmNAv4KBsYon\\\\\\\/imKqumIkPVZhEr1EnPa3eNoDOg+ZgyGVaTPp\\\\\\\/1MhSbnjoiRLUL3gvcuvRfU5upCGA1a\\\\\\\/o9VF\\\\\\\/WM9i54wtWXZ5EP\\\\\\\/KobLWh5GGdUN2SbPGKMrj\\\\\\\/M+sHaqv0QWu5ZUDhcom2zwUlGET9o2Z5bwsOhqQoWQyKO+d9lIm4nLFd1Lst4HkLpEjwLb9sxYROYc4cvqehK1HtZuaW3DxPvXPLZSVojOJ42z+mXMqYYX3Wd8iK0VJQZe5GbAg4Igz9BCx4fAEMe3RUz7yTGbDhyGnWzOyQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nDgQGg5zSYwcahLz984F+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022944ce9841be34c3571cf06348f12cd9608b3ccc656f9a3c14b650a5a9fa41831\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494283 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3263" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AqubSTS-wfPBQ7RzIU8zhmTpif5-Onz5aJ7zFUR01PM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-N4aEuaPZrD_4GSC8AELtDmVm0n6lm3LLtfGA5NELSuA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3249" title="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3273" title="IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494283" id="node494283" data-pisa="ggg;9/10/3263" data-pisa-master="ggg;g3.119.400450" data-apath="/ggg/9/10/3263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen <em>Puccinia hordei</em> Identifies Candidates for Three Avirulence Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9145-3018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jiapeng Chen</span>, <span class="highwire-citation-author" data-delta="1">Jingqin Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4191-1068" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peng Zhang</span>, <span class="highwire-citation-author" data-delta="3">Chongmei Dong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3052-0416" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Narayana M. Upadhyaya</span>, <span class="highwire-citation-author" data-delta="5">Qian Zhou</span>, <span class="highwire-citation-author" data-delta="6">Peter Dodds</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9145-5371" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert F. Park</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3263-3271;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400450 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494283" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiapeng Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiapeng%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiapeng%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9145-3018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jiapeng Chen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jingqin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingqin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingqin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peng Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4191-1068" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peng Zhang</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chongmei Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chongmei%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChongmei%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Narayana M. Upadhyaya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Narayana%2BM.%2BUpadhyaya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Upadhyaya%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANarayana%2BM.%2BUpadhyaya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3052-0416" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Narayana M. Upadhyaya</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qian Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Agricultural Genomic Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQian%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peter Dodds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BDodds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dodds%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BDodds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Robert F. Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BF.%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BF.%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9145-5371" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert F. Park</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:robert.park@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">robert.park@sydney.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3263" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494283" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494283/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3263.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494283" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494283/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3263.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494283" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494283/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3263.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494283" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494283/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2"><em>Puccinia hordei</em> (<em>Ph</em>) is a damaging pathogen of barley throughout the world. Despite its importance, almost nothing is known about the genomics of this pathogen, and a reference genome is lacking. In this study, the first reference genome was assembled for an Australian isolate of <em>Ph</em> (“<em>Ph</em>560”) using long-read SMRT sequencing. A total of 838 contigs were assembled, with a total size of 207 Mbp. This included both haplotype collapsed and separated regions, consistent with an estimated haploid genome size of about 150Mbp. An annotation pipeline that combined RNA-Seq of <em>Ph</em>-infected host tissues and homology to proteins from four other <em>Puccinia</em> species predicted 25,543 gene models of which 1,450 genes were classified as encoding secreted proteins based on the prediction of a signal peptide and no transmembrane domain. Genome resequencing using short-read technology was conducted for four additional Australian strains, <em>Ph</em>612, <em>Ph</em>626, <em>Ph</em>608 and <em>Ph</em>584, which are considered to be simple mutational derivatives of <em>Ph</em>560 with added virulence to one or two of three barley leaf rust resistance genes (<em>viz. Rph3</em>, <em>Rph13</em> and <em>Rph19</em>). To identify candidate genes for the corresponding avirulence genes <em>AvrRph3</em>, <em>AvrRph13</em> and <em>AvrRph19</em>, genetic variation in predicted secreted protein genes between the strains was correlated to the virulence profiles of each, identifying 35, 29 and 46 candidates for <em>AvrRph13</em>, <em>AvrRph3</em> and <em>AvrRph19</em>, respectively. The identification of these candidate genes provides a strong foundation for future efforts to isolate these three avirulence genes, investigate their biological properties, and develop new diagnostic tests for monitoring pathogen virulence.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/barley-leaf-rust" class="hw-term hw-article-keyword hw-article-keyword-barley-leaf-rust" rel="nofollow">barley leaf rust</a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/gene-gene" class="hw-term hw-article-keyword hw-article-keyword-gene-for-gene" rel="nofollow">gene-for-gene</a></li><li class="kwd"><a href="/keyword/avirulence-gene" class="hw-term hw-article-keyword hw-article-keyword-avirulence-gene" rel="nofollow">avirulence gene</a></li><li class="kwd"><a href="/keyword/resequencing" class="hw-term hw-article-keyword hw-article-keyword-resequencing" rel="nofollow">resequencing</a></li></ul><p id="p-3">Leaf rust, caused by the fungus <em>Puccinia hordei</em> (<em>Ph</em>), is a damaging disease of barley that has caused substantial yield losses as high as 62% (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cotterill <em>et al.</em> 1992</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">1995</a>). One of the most cost effective methods to manage this disease is the use of resistance genes (R genes) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Park 2008</a>). Many of the R genes deployed in agriculture to control rust are thought to encode immunoreceptors that recognize avirulence (Avr) gene products, or effector proteins, from invading rust pathogens. This recognition leads to the initiation of the host immune response to arrest rust development around the infection site, often leading to a localized hypersensitive reaction (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jones and Dangl 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dodds and Rathjen 2010</a>). In this model, the evolution of new rust strains is thought to involve modification or deletion of Avr genes, which allows the pathogen to evade detection by the R gene product and infect a previously resistant variety. Identifying these Avr genes in the pathogen and the corresponding R genes in the host is an important step in understanding genetic interactions in the <em>Ph</em>-barley pathosystem and in developing new sustainable approaches and diagnostics to reduce the threat posed by this pathogen.</p><p id="p-4">While the study of rust pathogen biology has been limited by their obligate biotrophic nature, Next-generation sequencing (NGS) technology has already resulted in the whole genome sequencing of several rust species and enabled significant advances in this field. To date, reference genomes have been assembled for four other cereal rust pathogens, namely <em>P. graminis</em> f. sp. <em>tritici</em> (<em>Pgt</em>), <em>P. triticina</em> (<em>Pt</em>), <em>P. coronata</em> f. sp. <em>avenae</em> (<em>Pca</em>), and <em>P. striiformis</em> f. sp. <em>tritici</em> (<em>Pst</em>) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Duplessis <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cuomo <em>et al.</em> 2017</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Miller <em>et al.</em> 2018</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schwessinger <em>et al.</em> 2018</a>). This has allowed initial cataloguing of genes that are believed to encode effector proteins that are secreted into the host and modulate the interaction between the pathogen and the host. Effector proteins are generally considered to contain a signal peptide for secretion, and no transmembrane domain (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Saunders <em>et al.</em> 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sperschneider <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sonah <em>et al.</em> 2016</a>). The up-regulation of a given candidate effector gene in haustoria, the fungal structures considered to be the major site of effector secretion, is considered further evidence that it is likely to encode an effector protein (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dodds 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Garnica <em>et al.</em> 2014</a>). A study of five Australian strains of <em>Pgt</em> identified 520 genes as candidate effectors based on their structure and up-regulation in haustoria compared to germinated urediniospores (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Upadhyaya <em>et al.</em> 2015</a>). In another comparative genomics study, non-synonymous nucleotide changes between two isolates of <em>Pst</em> led to the identification of five Avr candidates (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cantu <em>et al.</em> 2013</a>). Similarly, comparative genomics and statistical association analysis found 20 candidates for the Avr gene corresponding to R gene <em>Lr20</em> in <em>Pt</em> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wu <em>et al.</em> 2017</a>).</p><p id="p-5">With the availability of reference genomes, genome re-sequencing followed by variant calling has proven to be an effective method for Avr gene identification in plant pathogens. A typical workflow of re-sequencing analysis is to map sequencing reads from individual strains of different virulence profiles to a reference genome, and associate genotypic variations with virulence phenotypes based on mapping. For instance, re-sequencing of strains of the vascular wilt fungus <em>Verticillium dahliae</em> identified a 50 Kbp stretch of DNA present in four avirulent strains but absent in seven virulent strains, which led to the successful cloning of the Avr gene <em>Ave1</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de Jonge <em>et al.</em> 2012</a>). In another study, comparative genomics of two strains of <em>Cladosporium fulvum</em> (causal agent of tomato leaf mold) differing in virulence to resistance gene <em>Cf-5</em> identified one gene with a 2 bp deletion, and confirmed that the gene was the corresponding Avr gene <em>Avr5</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mesarich <em>et al.</em> 2014</a>). More recently, comparison of the genomes of mutant strains of the wheat stem rust pathogen with acquired virulence for the resistance genes <em>Sr50</em> or <em>Sr35</em> identified the matching avirulence genes <em>AvrSr50</em> and <em>AvrSr35</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2017</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Salcedo <em>et al.</em> 2017</a>). Taken together, these studies have demonstrated that the evolution of new virulence can be caused by non-synonymous mutations in Avr genes.</p><p id="p-6">In Australia, although sexual reproduction of <em>Ph</em> has been observed (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wallwork <em>et al.</em> 1992</a>), monitoring of strain (“pathotype”) evolution has suggested that the asexual cycle is dominant (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Park 2003</a>). One asexual (clonal) lineage within the pathogen population is thought to be derived from strain 5453P- (<em>Ph</em>560; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This progenitor strain was first discovered in Western Australia in 2001, being pathogenically distinct from all <em>Ph</em> strains detected previously in Australia (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Park <em>et al.</em> 2015</a>). A derivative of this lineage that had acquired virulence to <em>Rph19</em> was detected in 2003 (strain 5453P+, <em>Ph</em>584) (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Park <em>et al.</em> 2015</a>). Later in 2008 and 2009, two further strains with independent virulence gains to <em>Rph13</em> and <em>Rph3</em> were detected (strains 5453P+ +Rph13 (<em>Ph</em>608) and 5457P+ (<em>Ph</em>612), respectively (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Karaoglu and Park 2013</a>). Later, an isolate of strain 5457P- (<em>Ph</em>626) was detected in 2013, which likely arose from the progenitor strain 5453P- by gaining virulence to <em>Rph3</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Based on the gene-for-gene hypothesis (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Flor 1971</a>), it is assumed that one dominant Avr gene corresponds to each of the three resistance genes <em>Rph3</em>, <em>Rph13</em> and <em>Rph19</em>, <em>viz</em>. <em>AvrRph3</em>, <em>AvrRph13</em> and <em>AvrRph19</em>, respectively and that virulence evolution in this lineage resulted from single independent mutation events in these genes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proposed evolutionary pathways from the 5453P- lineage of Puccinia hordei based on pathogenicity testing. Nodes in the tree show culture numbers and strains in bracket, and the labels on tree branches show resistance genes overcome by the mutation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803642049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proposed evolutionary pathways from the 5453P- lineage of &lt;em&gt;Puccinia hordei&lt;/em&gt; based on pathogenicity testing. Nodes in the tree show culture numbers and strains in bracket, and the labels on tree branches show resistance genes overcome by the mutation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3263/F1.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3263/F1.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494397" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Proposed evolutionary pathways from the 5453P- lineage of <em>Puccinia hordei</em> based on pathogenicity testing. Nodes in the tree show culture numbers and strains in bracket, and the labels on tree branches show resistance genes overcome by the mutation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">In this study, whole genome sequencing was performed for five isolates of strains within the 5453P- lineage. The genome sequence of strain <em>Ph</em>560 was assembled from long read SMRT sequencing data to generate the first reference assembly for this pathogen species. A total of 25,543 gene models were predicted and those encoding secreted proteins (SPs) were identified. Re-sequencing data of isolates of the five strains were mapped to the reference genome to examine genetic variations that may account for their pathogenic differences on resistance genes <em>Rph3</em>, <em>Rph13</em> and <em>Rph19</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>DNA preparation and genome sequencing</h3><p id="p-9">The five strains used in this study were identified in annual Australia-wide national surveys of pathogenicity in <em>P. hordei</em> (R.F. Park, unpublished data), and are curated in the Plant Breeding Institute Rust Collection, The University of Sydney, Australia. Each isolate was established from a single pustule on an infected barley leaf to ensure purity. For PacBio sequencing of <em>Ph</em>560, a modified CTAB DNA extraction procedure (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Schwessinger 2016</a>) was used. The DNA solution obtained was then further purified to separate high molecular weight DNA from other impurities and low molecular weight DNA as described by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dong (2017)</a>. DNA concentration and purity was measured with a Qubit 3 (Invitrogen) and Nanodrop ND-1000 (Thermo Fisher Scientific). If the reading from the Qubit to Nanodrop ratio was smaller than 0.5, AMPure beads (Beckman, Coulter Inc.) were used to purify the DNA at the ratio of 0.45 beads to DNA (vol/vol) following the manufacturer’s protocol. DNA integrity was checked by pulsed-field gel electrophoresis. The sequencing library was prepared using SMRT cell Template Prep Kit 1.0-SPv3 with BluePippin Size-selection with 15-20 kb cutoff, and then sequenced using Sequel Sequencing Kit 2.0 at the Ramaciotti Centre for Genomics (Sydney, Australia).</p><p id="p-10">For Illumina sequencing of the five strains, the CTAB extraction method (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rogers <em>et al.</em> 1989</a>) was used to prepare DNA samples. For the genomic sequencing of <em>Ph</em>612, a library with insert size of 500 bp was sent to BGI Tech Solutions (Hong Kong) Co., Ltd. and was sequenced 90 bp paired-end on Illumina HiSeq 2000 platform. DNA extracted from <em>Ph</em>560 was sent to Novogene (HK) Ltd. for sequencing on Illumina HiSeq 2500 platform (PE 125bp), and the other three strains (<em>i.e.</em>, <em>Ph</em>626, <em>Ph</em>584 and <em>Ph</em>608) were sent to the Australian Genome Research Facility (AGRF) for sequencing on Illumina HiSeq 2500 (PE 125bp).</p></div><div id="sec-3" class="subsection"><h3>Genome assembly</h3><p id="p-11">The raw PacBio sequencing reads of <em>Ph</em>560 were first processed using the read correction module of Canu v1.6 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Koren <em>et al.</em> 2017</a>) with “genomeSize=220m” and the corrected reads were then assembled using SMARTdenovo version a411def (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ruan 2015</a>) with default parameters. To polish the assembly, Illumina reads were mapped to the contigs with BWA mem 0.7.5a-r405 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li 2013</a>) and the resulting mapping file was subjected to Pilon 1.22 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Walker <em>et al.</em> 2014</a>) to fix bases, fill gaps and correct local mis-assemblies. This mapping and fixing pipeline was run for three iterations.</p></div><div id="sec-4" class="subsection"><h3>Read mapping, variant calling and annotation</h3><p id="p-12">Illumina reads of the five strains were trimmed with Trim Galore v0.3.7 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Martin 2011</a>) with parameters “-quality 20 -phred33 -length 35”, and then mapped to Gn560 with BWA mem 0.7.5a-r405 (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Li 2013</a>). To minimize false positives of SNP calls around insertion/deletion (InDel) regions, poorly aligned reads around InDel were identified and realigned locally using RealignerTargetCreator and IndelRealigner in GATK package 3.8.1 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>). The subsequent mapping files were used to call SNPs and InDels using GATK HaplotypeCaller (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>). The identified SNPs and InDels were annotated with SnpEff v4.1 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cingolani <em>et al.</em> 2012</a>) using default parameters. To manually curate the SNP calls produced by GATK, reads were mapped to the reference genome using Bowtie2 v2.2.5 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Langmead and Salzberg 2012</a>) using parameters “-k 2–very-sensitive”. The parameter “-k 2” was selected to allow up to two mapping for a single read. The resulting read alignments were inspected using IGV to confirm the GATK SNP calls.</p></div><div id="sec-5" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-13">The evolutionary relationships between <em>Ph</em>560 and the four putative derivative strains identified in annual pathogenicity surveys (<em>Ph</em>584, <em>Ph</em>608, <em>Ph</em>612 and <em>Ph</em>626) were examined based on genome wide SNPs inferred using Poppr, an R package for population genetics analysis. Genetic distance was calculated using bitwise.dist function and 1,000 bootstrap replicates were performed.</p></div><div id="sec-6" class="subsection"><h3>RNA isolation and sequencing</h3><p id="p-14">Barley seedlings (cultivar Morex) were infected with <em>Ph</em> strain 612. Infected leaves were harvested at 4 and 7 days post inoculation (dpi) and stored at -80 ◦C. Total RNA was extracted from infected leaf tissue using TRIzol reagent (Life Technologies Australia Pty Ltd) according to the manufacturer’s instructions. The total RNA was then treated with RNase-free DNase I (New England BioLabs Inc.), and column purified using ISOLATE II RNA Mini Kit (Bioline Australia) according to the manufacturer’s instruction. The quantity and quality of the total RNA were examined by Nanodrop (Thermo Scientific) and Agilent 2100 Bioanalyzer (Agilent Technologies). Two micrograms of RNA from 4 dpi and 7 dpi were combined. Library was constructed using a TruSeq Stranded mRNA-Seq Library Prep kit and was sequenced on HiSeq 2000 (PE 100bp) at Ramaciotti Centre for Genomics.</p></div><div id="sec-7" class="subsection"><h3>Gene prediction</h3><p id="p-15">The raw RNA-Seq reads were trimmed with Trim Galore v0.3.7 (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Martin 2011</a>) with parameters “-quality 20 -phred33 -length 35”. The trimmed reads were aligned to Gn560 using TopHat v2.0.14 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kim <em>et al.</em> 2013</a>) with parameter setting “min-intron-length 10 -max-intron-length 5000 -mate-inner-dist 100 -mate-std-dev 100 -min-segment-intron 10 -max-segment-intron 5000”. Transcripts were assembled based on the mapping using Cufflinks v2.2.1 (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Trapnell <em>et al.</em> 2010</a>) with parameters “min-intron-length 10 -max-intron-length 5000 -minisoform-fraction 0.1”. To complement the RNA-Seq based prediction, protein sequences from <em>Pca</em>, <em>Pst</em> and <em>Pt</em> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cuomo <em>et al.</em> 2017</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Miller <em>et al.</em> 2018</a>;) were mapped to Gn560 with MAKER v2.31.8 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Holt and Yandell 2011</a>). Both transcript and protein mapping on Gn560 were used to predict gene structures with the MAKER pipeline.</p></div><div id="sec-8" class="subsection"><h3>Functional annotation</h3><p id="p-16">The putative protein sequences were searched against the non-redundant protein database v2016-02-29 and SwissProt database v2017-03-05 using BLASTP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) with parameters “-e 1e-5 -F F -a 4”. The BLASTP hits were filtered requiring a minimum 40% of the <em>Ph</em> query proteins matched target proteins and the match identities were not less than 30%. SignalP v4.1 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Petersen <em>et al.</em> 2011</a>) was used to predict signal peptide with parameter “-c 140” that examine the first 140 aa in given protein sequences. TMHMM v2.0c (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sonnhammer <em>et al.</em> 1998</a>) was used to predict transmembrane domains in all the resulting proteins with putative signal peptides. If the predicted transmembrane domain in a given protein did not overlap with its signal peptide, it was considered as a transmembrane protein and removed from the set of effector candidates.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-17">All raw sequencing data have been deposited in NCBI SRA under the BioProject ID PRJNA495764. The genome assembly has been deposited at GenBank under the accession RDRW00000000, and consists of sequences RDRW01000001-RDRW01000838. Gene structure prediction is recorded in a gff format file available at <a href="https://github.com/chjp/Phordei.git">https://github.com/chjp/Phordei.git</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7823987">https://doi.org/10.25387/g3.7823987</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3><em>De novo</em> assembly</h3><p id="p-18">The founder strain <em>Ph</em>560 was selected to assemble a reference genome for <em>P. hordei</em>. A total of 18.5 Gbp of PacBio SMRT sequencing data (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were generated for this strain. We used Canu to pre-correct the original reads and assembled the resulting data using SMARTdenovo. The resulting assembly was polished using three rounds of Pilon with 11.6 Gbp of 125bp paired-end sequencing data from the Illumina platform. The final assembly contained a total of 838 contigs with a size of 207 Mbp (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). This set of contigs is referred to as Gn560 henceforth. The alignment rate of Illumina reads of <em>Ph</em>560 to this assembly was 95.4%, indicating a high level of genome completeness. To examine gene space completeness in the assembly, BUSCO (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Simão <em>et al.</em> 2015</a>) was used to map 1,335 highly conserved single-copy orthologs in the Basidiomycota lineage to the assembly. This analysis showed 93.5% (1,249/1,335) of the conserved gene set was present in Gn560.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494537/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494537/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"> Sequencing information for five strains of <em>Puccinia hordei</em></span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494332/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494332/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"> <em>De novo</em> assembly statistics for <em>Puccinia hordei</em> strain <em>Ph</em>560</span><div class="sb-div caption-clear"></div></div></div><p id="p-19"><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Eilam <em>et al.</em> (1994)</a> used flow cytometry to estimate relative genome sizes of <em>Pgt</em> and <em>Pca</em> as 56% and 64% of that of <em>Ph</em>. This is consistent with their haploid genome assembly sizes of 88 Mbp (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Duplessis <em>et al.</em> 2011</a>) and 100 Mbp (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Miller <em>et al.</em> 2018</a>), respectively, and gives an estimated haploid genome size of 156 Mbp for <em>Ph</em>. We suspected that the larger assembly size was caused by high heterozygosity in some genomic regions preventing haploid assembly, resulting in assembly of two copies representing the two haplotypes of the uredinial stage of this organism. This suspicion was supported by Illumina read coverage analysis of the assembly, which showed a two-peak distribution of depth for both whole contigs and individual bases (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; Supplementary Table S1). This is expected when mixed haplo-separated and collapsed regions are present within the assembly (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Miller <em>et al.</em> 2018</a>), as collapsed regions will show twice the coverage of haplo-separated regions because reads from both alleles will map to the same position. Thus the first peak at ∼37X would represent the read depth at haplo-separated positions, while the second peak of ∼72X indicates the merged haploid assembly. In addition, 382 out of 1,249 (31%) complete genes identified in the BUSCO analysis were present in two copies (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), probably due to the allele separation on highly heterozygous regions. These numbers suggest that about two thirds of the haploid genome content of our assembly is represented in single copy collapsed sequences, while the remaining third is represented by duplicated contigs. This suggests a haploid genome size of about 150Mbp, consistent with the flow cytometry estimates relative to <em>Pgt</em> and <em>Pca</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coverage depth analysis of Illumina reads of Ph560 mapped back to the genome assembly. (A) Average coverage depth of contigs was calculated and the Y axis shows the number of contigs with a specific depth. Contigs with read depth higher than 75 are not shown. (B) Histogram of coverage depth over individual positions in all contigs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803642049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Coverage depth analysis of Illumina reads of &lt;em&gt;Ph&lt;/em&gt;560 mapped back to the genome assembly. (A) Average coverage depth of contigs was calculated and the Y axis shows the number of contigs with a specific depth. Contigs with read depth higher than 75 are not shown. (B) Histogram of coverage depth over individual positions in all contigs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3263/F2.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3263/F2.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494432" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Coverage depth analysis of Illumina reads of <em>Ph</em>560 mapped back to the genome assembly. (A) Average coverage depth of contigs was calculated and the Y axis shows the number of contigs with a specific depth. Contigs with read depth higher than 75 are not shown. (B) Histogram of coverage depth over individual positions in all contigs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Gene prediction</h3><p id="p-21">Under optimal conditions, <em>P. hordei</em> has colonized its host and produced infection structures by 2 days after infection, while the process of spore production commences within 6-8 days of infection (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Clifford 1985</a>). To guide gene annotation, RNA-Seq was therefore performed for infected barley leaf tissues collected at two time points (the 4<sup>th</sup> and 7<sup>th</sup> day) post inoculation with <em>Ph</em>612. The RNA-Seq reads were aligned to the draft genome Gn560 with TopHat2. A total of 38.6% of the reads were successfully aligned and retained, whereas the remaining 61.4% of reads were discarded as derived from the barley leaf transcriptome. The read alignment was then assembled to transcripts using Cufflink. These transcripts should capture a majority of the genes expressed <em>in planta</em> during active colonization and sporulation.</p><p id="p-22">To obtain a comprehensive gene repertoire, timepoints representing all life stages should be covered. Therefore, proteins predicted at more life stages than those sampled here in three other <em>Puccinia</em> species (<em>viz</em>. <em>Pt</em>, <em>Pca</em> and <em>Pst</em>) were used to complement the RNA-Seq data. These protein sets include genes expressed in dormant urediniospores, germinating urediniospores, isolated haustoria and during host infection. Both the <em>P. hordei</em> transcripts and the <em>Puccinia spp</em>. proteins were mapped to Gn560 using the MAKER pipeline, resulting in the prediction of 25,543 genes and 66,126 isoforms. The translated protein sequences from the gene isoforms were aligned to NCBI nr and Swissprot databases for functional annotation (Supplementary Table S2). The predicted genes were named with a common prefix “PH560” indicating their origin from <em>P. hordei</em> strain <em>Ph</em>560, and a unique suffix number derived from the prediction tool MAKER (<em>e.g.</em>, <em>PH560_12290</em>).</p></div><div id="sec-13" class="subsection"><h3>Effector prediction</h3><p id="p-23">Rust fungi secrete effector proteins into the plant apoplast or cytoplasm and secretome prediction based on the presence of secretion signal peptides at the N termini of proteins can be used to identify candidate effectors. A scan of the <em>Ph</em> proteome with SignalP4.1 identified 1,739 genes encoding a signal peptide (Supplementary Table S3). Of these, 289 were excluded because they contained a non-overlapping predicted transmembrane segment that is associated with membrane integration, leaving a total of 1,450 candidate genes predicted to encode secreted proteins.</p><p id="p-24">MAKER mapped the protein sequences and RNA-Seq reads to the genome and predicted gene structures by integrating evidence from both. With a cutoff of mapping coverage 90%: 121 SP genes were supported by RNAseq but not by protein mapping; 675 SP candidates were covered with &lt;90% by RNAseq but more than 90% by proteins; and the remaining 654 SP candidates were supported by both sources in integration.</p></div><div id="sec-14" class="subsection"><h3>Read mapping</h3><p id="p-25">In order to identify candidates for the three Avr genes in which mutations had occurred within the <em>Ph</em>560-derived lineage, we generated Illumina short read genomic DNA sequencing data for the four mutation-derived strains (<em>Ph</em>612, <em>Ph</em>608, <em>Ph</em>584, and <em>Ph</em>626) within this lineage. This yielded at least 11 Gbp (∼70X average coverage) data for each strain after quality trimming (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). To genotype the strains, the sequencing reads of each isolate were mapped to the reference genome Gn560 individually. For each strain, about 95% of reads could be mapped to the reference and the percentage of the reference coverage was over 99%, suggesting that almost all genes annotated in the reference could be genotyped.</p></div><div id="sec-15" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-26">To confirm the relatedness of the strains within the putative clonal lineage founded by strain 5453P, a phylogenetic tree including <em>Ph</em>560, the four presumed derivative strains, and three pathogenically distinct <em>P. hordei</em> strains collected in 1966 (<em>Ph</em>488, <em>Ph</em>489) and 1990 (<em>Ph</em>482) was constructed based on the genome-wide SNPs identified (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). While the three distant strains and <em>Ph</em>560 derived series formed two distinct clusters, within the <em>Ph</em>560 cluster, <em>Ph</em>584 and <em>Ph</em>608 formed a sister group and <em>Ph</em>626 and <em>Ph</em>612 formed distinct branches. The overall topology of the phylogenetic tree was consistent with the hypothesis that strains <em>Ph</em>584, <em>Ph</em>608, <em>Ph</em>612 and <em>Ph</em>626 were derived from <em>Ph</em>560 via simple mutational acquisition of virulence for the resistance genes <em>Rph3</em>, <em>Rph13</em>, and <em>Rph19</em>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dendrogram of eight Puccinia hordei strains based on the total identified SNPs inferred using Poppr. Genetic distance was calculated using bitwise.dist function. The numbers shown on the dendrogram branches are the percentage of bootstrap replicates (1,000) supporting the cluster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803642049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dendrogram of eight &lt;em&gt;Puccinia hordei&lt;/em&gt; strains based on the total identified SNPs inferred using Poppr. Genetic distance was calculated using bitwise.dist function. The numbers shown on the dendrogram branches are the percentage of bootstrap replicates (1,000) supporting the cluster.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3263/F3.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3263/F3.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494449" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Dendrogram of eight <em>Puccinia hordei</em> strains based on the total identified SNPs inferred using Poppr. Genetic distance was calculated using bitwise.dist function. The numbers shown on the dendrogram branches are the percentage of bootstrap replicates (1,000) supporting the cluster.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Genome wide polymorphism</h3><p id="p-28">To compare genotypes across the five strains, small variations including single nucleotide polymorphisms (SNPs) and insertion/deletions (InDels) between individual strains were examined based on mapping reads to the reference using GATK HaplotypeCaller (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>). For the putative progenitor strain <em>Ph</em>560, 516,533 variants were identified, occurring at a genome-wide frequency of 2.5/Kbp with over 90% present in a heterozygous state. Among the variants, SNPs and InDels showed a ratio of 4.8:1. The total number of sequence variants identified for the remaining four strains ranged from 552,081 to 554,627 and showed similar ratios of SNP/InDel (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494387/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494387/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"> Genomic variant calling in five <em>Puccinia hordei</em> strains</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">In order to relate genomic variants to gene structure, SnpEff (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cingolani <em>et al.</em> 2012</a>) was used to map all SNPs and InDels to exons and splice sites to evaluate their impact on protein coding (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). An InDel in a coding region can cause a reading frame shift if its size is not a multiple of three, whereas an InDel of one or several codons induces a less severe effect on protein coding. InDels of these two types were referred to as frameshift and inframe InDels, respectively. The average counts of frameshift and inframe mutations in the five strains were 5,409 and 1,339, respectively. In addition to InDels, sequence changes in the start and stop codons may also have a high impact on gene function, as loss of a start codon would abolish protein translation, gain of a stop codon would cause premature protein truncation, and a stop codon loss would result in additional amino acids (aa) that may change protein structure. These three types of mutations occurred on average 191, 2,629 and 779 times, respectively, in the five strains. Furthermore, aa changes caused by nonsynonymous (NSY) SNPs may also have direct functional implications, whereas synonymous (SYN) SNPs resulting in no aa change may not have such an effect. Across the five strains, more NSY variants were detected as compared to the SYN variants, and the average counts of these two types were 67,526 and 46,763, respectively.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494457/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494457/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title"> Coding impacts of genomic variants for five strains of <em>Puccinia hordei</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Genomic polymorphism in SP genes associated with avirulence/virulence</h3><p id="p-30">As effector proteins are most likely encoded by SP genes, our search for the three Avr genes <em>AvrRph3</em>, <em>AvrRph13</em>, and <em>AvrRph19</em> focused on the detection of mutations in candidate SP genes between the five <em>Ph</em> strains. A total of 714 SP genes contained protein coding polymorphisms in at least one of the five strains. The variants in these 714 genes were scanned manually for read count support in the read mapping to identify false calling. We found that 24 of these genes appeared to be multi-copy genes that had been collapsed into one locus in the assembly, because they contained sites with more than two alternative alleles and allele frequencies that diverged from the expected 1:1, and hence we were not able to genotype these genes. For the remainder, after manual curation, we confirmed the presence of 976, 1,023, 1,019, 1,240 and 1,023 polymorphic sites in SP genes of strains <em>Ph</em>560, <em>Ph</em>584, <em>Ph</em>608, <em>Ph</em>612 and <em>Ph</em>626, respectively (Supplementary Table S4). To identify candidates for the <em>AvrRph3</em>, <em>AvrRph13</em> and <em>AvrRph19</em> genes, we looked for amino-acid changing polymorphisms that were shared by isolates of the same virulence phenotype and that could therefore represent mutations determining these virulence changes in this lineage of <em>Ph</em>.</p><p id="p-31">To identify candidates for <em>AvrRph13</em>, we compared the variant calls for the SP genes between strains <em>Ph</em>584 and <em>Ph</em>608. The two strains displayed different genotypes at 41 positions in 35 SP genes (Supplementary Table S5) in either a homozygous or heterozygous condition.</p><p id="p-32">Based on pathogenicity plus geographic region and time of isolation, strains <em>Ph</em>626 and <em>Ph</em>612 are considered to have gained virulence to <em>Rph3</em> via independent single step mutations from their progenitor strains, <em>Ph</em>560 and <em>Ph</em>584, respectively. Therefore, comparisons between <em>Rph3</em>-virulent and avirulent strains were carried out, <em>i.e.</em>, <em>Ph</em>626 <em>vs.</em> <em>Ph</em>560 and <em>Ph</em>612 <em>vs.</em> <em>Ph</em>584, to detect the SP genes that showed functional variations. Because it was not known whether the gene had the same mutation for the two independent virulence gains, SP genes that showed protein polymorphism in both comparisons were considered as potential candidates. Within the SP genes, strains <em>Ph</em>560 and <em>Ph</em>626 showed differences in 123 sites distributed in 79 SP genes (Supplementary Table S5). Similar screening was performed to compare strains <em>Ph</em>584 and <em>Ph</em>612, which enabled the identification of 206 SP genes showing changes in protein coding ability. The gene sets resulting from the two comparisons shared a common panel of 29 genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4B</a> and Supplementary Table S5).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams for intersection and complement of genes encoding secreted proteins with non-synonymous mutations. (A), (B) and (C) demonstrate the candidates for Rph13 (35), Rph3 (29) and Rph19 (46), respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803642049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Venn diagrams for intersection and complement of genes encoding secreted proteins with non-synonymous mutations. (A), (B) and (C) demonstrate the candidates for &lt;em&gt;Rph13&lt;/em&gt; (35), &lt;em&gt;Rph3&lt;/em&gt; (29) and &lt;em&gt;Rph19&lt;/em&gt; (46), respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3263/F4.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3263/F4.medium.gif" width="440" height="156"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3263/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494328" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Venn diagrams for intersection and complement of genes encoding secreted proteins with non-synonymous mutations. (A), (B) and (C) demonstrate the candidates for <em>Rph13</em> (35), <em>Rph3</em> (29) and <em>Rph19</em> (46), respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The strains <em>Ph</em>584, <em>Ph</em>612 and <em>Ph</em>608 were virulent for <em>Rph19</em> and were derived sequentially from the progenitor <em>Ph</em>560. The snpEff annotation indicated that 59, 70 and 232 SP genes displayed protein polymorphisms when comparing <em>Ph</em>560 with <em>Ph</em>584, <em>Ph</em>560 with <em>Ph</em>608, and <em>Ph</em>560 with <em>Ph</em>612, respectively. As <em>AvrRph19</em> had to differ in all three comparisons, the three sets of SP genes that showed protein variations were compared and 46 genes in common were thus identified (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>). According to the putative genetic relationship of <em>Ph</em>584, <em>Ph</em>612 and <em>Ph</em>608, the mutations causing the <em>AvrRph19</em> loss in <em>Ph</em>584 should be retained in its two derivatives <em>Ph</em>612 and <em>Ph</em>608. These new mutations should also be specific to <em>Ph</em>584, <em>Ph</em>612 and <em>Ph</em>608, and absent in <em>Ph</em>626. There were eight such mutations distributed in seven genes that are therefore considered the most promising candidates for <em>AvrRph19</em> (Supplementary Table S5).</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-35">This study reports the first genome assembly of the economically important cereal rust pathogen <em>P. hordei</em>, providing a valuable genomic resource for the rust research community. As a high-quality reference genome, the assembly enables genomic and transcriptomic comparisons of isolates within and across different <em>P. hordei</em> lineages (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Park <em>et al.</em> 2015</a>), especially in studies targeting diversity in candidate effector genes. Genomes of the currently sequenced cereal rust fungi differ significantly in size, ranging from 83 Mbp in <em>Pst</em> to 135 Mbp in <em>Pt</em> (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Duplessis <em>et al.</em> 2011</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cuomo <em>et al.</em> 2017</a>; <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Miller <em>et al.</em> 2018</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Schwessinger <em>et al.</em> 2018</a>). A previous flow cytometry study estimated the <em>Ph</em> genome size to be 122 Mbp (Kullman <em>et al.</em> 2005). The larger size of assembly of 207 Mbp reported here may have resulted from high heterozygosity in some genomic regions between the two nuclei that have prevented haploid assembly and resulted in two separate allelic contig sequences (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Based on read coverage and the proportion of single copy and duplicated conserved genes, we estimated a total haploid genome size of about 150Mbp. The N50 of the <em>Ph</em> contigs generated in the present study is 405,324 bp, comparable to two recently published genome assemblies for rust species based on PacBio sequencing (<a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Miller <em>et al.</em> 2018</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Schwessinger <em>et al.</em> 2018</a>).</p><p id="p-36">Transcriptome sequencing of infected leaf tissues collected 4 and 7 days post inoculation was used to guide the prediction of <em>Ph</em> genes expressed in hyphae and haustoria at these two time points. In addition, the alignment of <em>Pgt</em>, <em>Pca</em>, <em>Pt</em> and <em>Pst</em> proteins to the <em>Ph</em> genome enabled the identification of conserved genes missing in the <em>Ph</em> RNA-Seq data due to their exclusive expression at other life stages or in other tissues. The combination of these two types of evidence allowed prediction of a comprehensive gene space for the damaging uredinial stage of <em>Ph</em> on barley. To build on the gene annotation work reported here, future efforts will focus more on biological functions, such as identifying secreted protein-encoding genes preferentially expressed in haustoria, the presumed sites for effector delivery into host shown to be enriched for effector genes in <em>Melampsora lini</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Catanzariti <em>et al.</em> 2006</a>).</p><p id="p-37">To identify candidate Avr genes in <em>Ph</em>, comparisons were made between the whole genome sequences of isolates of five <em>Ph</em> strains that comprised a progenitor and four presumed mutational derivatives (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) differing in virulence for three barley leaf rust resistance genes, <em>viz. Rph3</em>, <em>Rph13</em> and <em>Rph19</em>. The sequencing reads of the five strains were mapped to the reference genome Gn560 generated for the presumed progenitor <em>Ph</em>560. The mapping rates ranged from 85–95% with an average rate of 93%, similar to rates observed for Illumina mapping to the highly contiguous PacBio assemblies of <em>Pca</em> (<a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Miller <em>et al.</em> 2018</a>) and <em>Pst</em> (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Schwessinger <em>et al.</em> 2018</a>), and higher than observed for short read or Sanger based assemblies of other rust fungi (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Duplessis <em>et al.</em> 2011</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Cantu <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nemri <em>et al.</em> 2014</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Upadhyaya <em>et al.</em> 2015</a> <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Cuomo <em>et al.</em> 2017</a>). A phylogeny based on whole genome SNPs supported the clonal derivation of the five strains, indicating that the virulence differences between them should be the result of mutation.</p><p id="p-38">It is not known whether the progenitor strain <em>Ph</em>560 is homozygous or heterozygous at the three Avr loci <em>AvrRph3</em>, <em>AvrRph13</em> and <em>AvrRph19</em>. If homozygous, both alleles of each Avr gene would need to mutate in order for virulence gain; if heterozygous, mutation of the single Avr allele only is required. In this study, both homozygous and heterozygous polymorphisms in the virulent strains were analyzed. By focusing on 1,450 SP genes, this study identified 29, 35 and 46 genes that encode proteins polymorphic between strains with differential virulence profiles for the R genes <em>Rph3</em>, <em>Rph13</em> and <em>Rph19</em>, respectively (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) and seven genes containing common mutations in all three <em>Rph19</em> virulent strains. Given that the resistance genes <em>Rph3</em>, <em>Rph13</em>, and <em>Rph19</em> have not yet been isolated from barley, functional testing of the candidate Avr genes will require either an <em>in-planta</em> expression systems to express the genes in barley lines containing <em>Rph3</em>, <em>Rph13</em>, and <em>Rph19</em>, or testing in a heterologous system such as a viral overexpression system (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lee <em>et al.</em> 2012</a>) or a protoplast expression assay (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2016</a>). The functional study of these candidate genes will provide deeper understanding of the <em>Ph</em>-barley pathosystem.</p><p id="p-39">Compared with other cereal rust species present in Australia, <em>Ph</em> is the only one that undergoes sexual recombination (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wallwork <em>et al.</em> 1992</a>), providing an opportunity to use a map-based approach to help isolate Avr genes including those targeted in the current study. Sexual crosses of <em>Ph</em> isolates would permit assessments of co-segregation between the candidates identified in the present study and virulence among F<sub>2</sub> progeny. This research method has provided substantial knowledge of the flax rust-flax interaction, leading to the isolation of several Avr genes (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Dodds 2004</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Catanzariti <em>et al.</em> 2006</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2016</a>). Apart from traditional map-based cloning, genome wide association studies (GWAS) by sequencing could also serve as a powerful tool for Avr gene identification in isolates of <em>Ph</em> derived from sexual recombination. This has been shown to be efficient in several studies of plant pathogen populations with sexual recombination (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2016</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Plissonneau <em>et al.</em> 2017</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Praz <em>et al.</em> 2017</a>).</p><p id="p-40">The whole genome sequencing data of five strains reported here provides a valuable resource for developing new DNA markers that will be useful for genetic studies and population monitoring of the <em>Ph</em> 5453P- lineage in Australia and beyond. Diagnostic simple sequence repeat (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Du <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kitchen <em>et al.</em> 2012</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Karaoglu and Park 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Metz <em>et al.</em> 2016</a>) and single nucleotide polymorphism (SNP) markers have been and will be further developed for <em>P. hordei</em> based on genome re-sequencing data. Combined with these tools, the genome sequence data should prove very useful in contributing to the development of more sustainable rust control, surveillance and management strategies for this important barley pathogen.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">The authors wish to thank the Australian Grains Research and Development Corporation for financial support. JC acknowledges the financial support from the Judith and David Coffey Life Lab, Charles Perkins Centre, and the Faculty of Science, University of Sydney. JC also acknowledges the facilities and technical assistance of the Sydney Informatics Hub at the University of Sydney and, in particular, the access to the high performance computing facility Artemis.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7823987">https://doi.org/10.25387/g3.7823987</a>.</p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-18"><span class="received-label">Received </span>June 18, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-31"><span class="accepted-label">Accepted </span>July 31, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Chen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Madden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Schaffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.17.3389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.2" data-doi="10.1186/s12864-016-3011-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Khan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.-M.</span> <span class="cit-name-surname">Catanzariti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Jack</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nemri</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome analysis and avirulence gene cloning using a high-density RADseq linkage map of the flax rust fungus, <em>Melampsora lini</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">667</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-3011-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D667%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-3011-9%26rft_id%253Dinfo%253Apmid%252F27550217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-3011-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27550217&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.3" data-doi="10.1186/1471-2164-14-270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cantu</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Segovia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">MacLean</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bayles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome analyses of the wheat yellow (stripe) rust pathogen <em>Puccinia striiformis</em> f. sp. <em>tritici</em> reveal polymorphic and haustorial expressed secreted proteins as candidate effectors.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-270%26rft_id%253Dinfo%253Apmid%252F23607900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23607900&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.4" data-doi="10.1105/tpc.105.035980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catanzariti</span>,  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. N.</span> <span class="cit-name-surname">Dodds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Lawrence</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Ayliffe</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Ellis</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Haustorially expressed secreted proteins from flax rust are highly enriched for avirulence elicitors.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.105.035980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.105.035980%26rft_id%253Dinfo%253Apmid%252F16326930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMjQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.5" data-doi="10.1126/science.aao4810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Upadhyaya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ortiz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sperschneider</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Loss of <em>AvrSr50</em> by somatic exchange in stem rust leads to virulence for <em>Sr50</em> resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">358</span>: <span class="cit-fpage">1607</span>–<span class="cit-lpage">1610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aao4810</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D358%26rft.issue%253D6370%26rft.spage%253D1607%26rft.epage%253D1610%26rft.atitle%253DLoss%2Bof%2BAvrSr50%2Bby%2Bsomatic%2Bexchange%2Bin%2Bstem%2Brust%2Bleads%2Bto%2Bvirulence%2Bfor%2BSr50%2Bresistance%2Bin%2Bwheat%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aao4810%26rft_id%253Dinfo%253Apmid%252F29269475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTgvNjM3MC8xNjA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.6" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Platts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang le</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nguyen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/fly.19695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFly%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D80%26rft.epage%253D92%26rft.atitle%253DA%2Bprogram%2Bfor%2Bannotating%2Band%2Bpredicting%2Bthe%2Beffects%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%252C%2BSnpEff%253A%2BSNPs%2Bin%2Bthe%2Bgenome%2Bof%2BDrosophila%2Bmelanogaster%2Bstrain%2Bw1118%253B%2Biso-2%253B%2Biso-3.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305965500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3263.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>,  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Barley leaf rust</span>, pp. <span class="cit-fpage">173</span>–<span class="cit-lpage">205</span> in <span class="cit-source">The Cereal Rusts II</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Roelfs</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">W. R.</span> <span class="cit-name-surname">Bushnell</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.8" data-doi="10.1071/EA9920747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotterill</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Rees</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Platz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Dill-Macky</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Effects of leaf rust on selected Australian barleys.</span> <abbr class="cit-jnl-abbrev">Aust. J. Exp. Agric.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/EA9920747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BExp.%2BAgric.%26rft.volume%253D32%26rft.spage%253D747%26rft_id%253Dinfo%253Adoi%252F10.1071%252FEA9920747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/EA9920747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.9" data-doi="10.1071/AR9950127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotterill</span>,  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Park</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Rees</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Pathogenic specialization of <em>Puccinia hordei Otth</em>. in Australia, 1966–1990.</span> <abbr class="cit-jnl-abbrev">Aust. J. Exp. Agric.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/AR9950127</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BExp.%2BAgric.%26rft.volume%253D46%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR9950127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR9950127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.10" data-doi="10.1534/g3.116.032797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuomo</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bakkeren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. B.</span> <span class="cit-name-surname">Khalil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Panwar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Joly</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative analysis highlights variable genome content of wheat rusts and divergence of the mating loci.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032797</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032797%26rft_id%253Dinfo%253Apmid%252F27913634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi8zNjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.11" data-doi="10.1073/pnas.1119623109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Jonge</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">van Esse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Maruthachalam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Bolton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Santhanam</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tomato immune receptor <em>Ve1</em> recognizes effector of multiple fungal pathogens uncovered by genome and RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">5110</span>–<span class="cit-lpage">5115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1119623109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1119623109%26rft_id%253Dinfo%253Apmid%252F22416119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzEzLzUxMTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.12" data-doi="10.1105/tpc.020040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>,  <span class="cit-name-given-names">P. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The <em>Melampsora lini AvrL567</em> avirulence genes are expressed in haustoria and their products are recognized inside plant cells.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.020040</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.020040%26rft_id%253Dinfo%253Apmid%252F14973158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.13" data-doi="10.1038/nrg2812"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>,  <span class="cit-name-given-names">P. N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Rathjen</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant immunity: towards an integrated view of plant–pathogen interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">548</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2812</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDodds%26rft.auinit1%253DP.%2BN.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D539%26rft.epage%253D548%26rft.atitle%253DPlant%2Bimmunity%253A%2Btowards%2Ban%2Bintegrated%2Bview%2Bof%2Bplant-pathogen%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2812%26rft_id%253Dinfo%253Apmid%252F20585331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2812&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585331&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279988800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3263.14" data-doi="10.17504/protocols.io.hbvb2n6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Purification of HMW DNA from fungi for long read sequencing. <em>protocols.io</em><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.17504/protocols.io.hbvb2n6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.17504%252Fprotocols.io.hbvb2n6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.17504/protocols.io.hbvb2n6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.15" data-doi="10.1093/jhered/ess082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yue</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MSDB: a user-friendly program for reporting distribution and building databases of microsatellites from genome sequences.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/ess082</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fess082%26rft_id%253Dinfo%253Apmid%252F23144492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/ess082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23144492&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312885900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.16" data-doi="10.1073/pnas.1019315108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duplessis</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Cuomo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-C.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Aerts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Tisserant</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Obligate biotrophy features unraveled by the genomic analysis of rust fungi.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">9166</span>–<span class="cit-lpage">9171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1019315108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019315108%26rft_id%253Dinfo%253Apmid%252F21536894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzIyLzkxNjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.17" data-doi="10.1094/Phyto-84-728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eilam</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. R.</span> <span class="cit-name-surname">Bushnell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Anikster</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Relative nuclear DNA content of rust fungi estimated by flow cytometry of propidium iodide-stained pycniospores.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">735</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/Phyto-84-728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D84%26rft.spage%253D728%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPhyto-84-728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/Phyto-84-728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NY50000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.18" data-doi="10.1146/annurev.py.09.090171.001423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flor</span>,  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Current status of the gene-for-gene concept.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.py.09.090171.001423</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DFlor%26rft.auinit1%253DH.%26rft.volume%253D9%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.py.09.090171.001423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.py.09.090171.001423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971K729400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.19" data-doi="10.1371/journal.ppat.1004329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garnica</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nemri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Upadhyaya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Rathjen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. N.</span> <span class="cit-name-surname">Dodds</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The ins and outs of rust haustoria.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1004329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D10%26rft.spage%253De1004329%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004329%26rft_id%253Dinfo%253Apmid%252F25211126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25211126&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.20" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yandell</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Maker2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.21" data-doi="10.1038/nature05286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">J. D. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Dangl</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The plant immune system.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D444%26rft.issue%253D7117%26rft.spage%253D323%26rft.epage%253D329%26rft.atitle%253DThe%2Bplant%2Bimmune%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05286%26rft_id%253Dinfo%253Apmid%252F17108957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17108957&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242018300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.22" data-doi="10.1007/s13313-013-0244-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karaoglu</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Park</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Isolation and characterization of microsatellite markers for the causal agent of barley leaf rust, <em>Puccinia hordei</em>.</span> <abbr class="cit-jnl-abbrev">Australas. Plant Pathol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13313-013-0244-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAustralas.%2BPlant%2BPathol.%26rft.volume%253D43%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13313-013-0244-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13313-013-0244-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.23" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pertea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Trapnell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Pimentel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kelley</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-4-r36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.24" data-doi="10.1186/1471-2105-13-287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kitchen</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Moore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Palmer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Allaby</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MCMC-ODPR: primer design optimization using Markov Chain Monte Carlo sampling.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">287</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-287</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKitchen%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D287%26rft.epage%253D287%26rft.atitle%253DMCMC-ODPR%253A%2BPrimer%2Bdesign%2Boptimization%2Busing%2BMarkov%2BChain%2BMonte%2BCarlo%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-287%26rft_id%253Dinfo%253Apmid%252F23126469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23126469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.25" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Walenz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Berlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Bergman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-9.10.3263.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kullman</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tamm</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kullman</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> Fungal genome size database. <a href="http://www.zbi.ee/fungal-genomesize/">http://www.zbi.ee/fungal-genomesize/</a> [Accessed on June 2, 2018]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.27" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.28" data-doi="10.1104/pp.112.203489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Hammond-Kosack</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kanyuka</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Barley stripe mosaic virus-mediated tools for investigating gene function in cereal plants and their pathogens: virus-induced gene silencing, host-mediated gene silencing, and virus-mediated overexpression of heterologous protein.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">582</span>–<span class="cit-lpage">590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.112.203489</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.203489%26rft_id%253Dinfo%253Apmid%252F22885938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjAvMi81ODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3263.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint. arXiv:1303.3997</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.30" data-doi="10.1073/pnas.1612947113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kracher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M. L.</span> <span class="cit-name-surname">Saur</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Ellwood</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Allelic barley MLA immune receptors recognize sequence-unrelated avirulence effectors of the powdery mildew pathogen.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E6486</span>–<span class="cit-lpage">E6495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1612947113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1612947113%26rft_id%253Dinfo%253Apmid%252F27702901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQyL0U2NDg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.31" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet. J. 17: 10–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCUTADAPT%2BREMOVES%2BADAPTER%2BSEQUENCES%2BFROM%2BHIGHTHROUGHPUT%2BSEQUENCING%2BREADS%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft_id%253Dinfo%253Apmid%252F25452271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25452271&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.32" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivachenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cibulskis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.33" data-doi="10.1094/MPMI-02-14-0050-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mesarich</span>,  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Griffiths</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">van der Burgt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ökmen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. G.</span> <span class="cit-name-surname">Beenen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptome sequencing uncovers the <em>Avr5</em> avirulence gene of the tomato leaf mold pathogen <em>Cladosporium fulvum</em>. Mol.</span> <abbr class="cit-jnl-abbrev">Plant-Microbe Int.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">846</span>–<span class="cit-lpage">857</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/MPMI-02-14-0050-R</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant-Microbe%2BInt.%26rft.volume%253D27%26rft.spage%253D846%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-02-14-0050-R%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-02-14-0050-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.34" data-doi="10.1155/2016/6040124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metz</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Cabrera</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rueda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Giri</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Amavet</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">FullSSR: Microsatellite Finder and Primer Designer.</span> <abbr class="cit-jnl-abbrev">Adv. Bioinforma.</abbr> <span class="cit-vol">2016</span>: <span class="cit-fpage">6040124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2016/6040124</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BBioinforma.%26rft.volume%253D2016%26rft.spage%253D6040124%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2016%252F6040124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2016/6040124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.35" data-doi="10.1128/mBio.01650-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Omidvar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sperschneider</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schwessinger</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">De novo assembly and phasing of dikaryotic genomes from two isolates of <em>Puccinia coronata</em> f. sp. <em>avenae</em>, the causal agent of oat crown rust.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.01650-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01650-17%26rft_id%253Dinfo%253Apmid%252F29463655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiOS8xL2UwMTY1MC0xNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.36" data-doi="10.3389/fpls.2014.00098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nemri</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G. O.</span> <span class="cit-name-surname">Saunders</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Anderson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Upadhyaya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Win</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genome sequence and effector complement of the flax rust pathogen <em>Melampsora lini</em>.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2014.00098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2014.00098%26rft_id%253Dinfo%253Apmid%252F24715894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2014.00098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24715894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.37" data-doi="10.1094/PDIS.2003.87.11.1311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Pathogenic specialization and pathotype distribution of <em>Puccinia hordei</em> in Australia, 1992 to 2001.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1311</span>–<span class="cit-lpage">1316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS.2003.87.11.1311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D87%26rft.spage%253D1311%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS.2003.87.11.1311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS.2003.87.11.1311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.38" data-doi="10.1111/j.1365-3059.2008.01836.x."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Breeding cereals for rust resistance in Australia.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">602</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-3059.2008.01836.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D57%26rft.spage%253D591%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3059.2008.01836.x.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3059.2008.01836.x.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.39" data-doi="10.1146/annurev-phyto-080614-120324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. G.</span> <span class="cit-name-surname">Golegaonkar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Derevnina</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Sandhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Karaoglu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Leaf rust of cultivated barley: pathology and control.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-phyto-080614-120324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPhytopathol.%26rft.volume%253D53%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-phyto-080614-120324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-phyto-080614-120324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.40" data-doi="10.1038/nmeth.1701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>,  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brunak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">von Heijne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1701</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPetersen%26rft.auinit1%253DT.%2BN.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D785%26rft.epage%253D786%26rft.atitle%253DSignalP%2B4.0%253A%2Bdiscriminating%2Bsignal%2Bpeptides%2Bfrom%2Btransmembrane%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1701%26rft_id%253Dinfo%253Apmid%252F21959131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959131&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.41" data-doi="10.3389/fpls.2017.00023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plissonneau</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Benevenuto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Mohd-Assaad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fouché</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. E.</span> <span class="cit-name-surname">Hartmann</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Using population and comparative genomics to understand the genetic basis of effector-driven fungal pathogen evolution.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.00119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00023%26rft_id%253Dinfo%253Apmid%252F28220127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28220127&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.42" data-doi="10.1111/nph.14372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Praz</span>,  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bourras</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zeng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sánchez-Martín</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Menardo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Avr<em>Pm2</em> encodes an RNase-like avirulence effector which is conserved in the two different specialized forms of wheat and rye powdery mildew fungus.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1314</span> (erratum: New Phytol. 222: 2038).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14372</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D213%26rft.spage%253D1301%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.43" data-doi="10.1139/b89-161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rehner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bledsoe</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Extraction of DNA from Basidiomycetes for ribosomal DNA hybridizations.</span> <abbr class="cit-jnl-abbrev">Can. J. Bot.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1243</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/b89-161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BBot.%26rft.volume%253D67%26rft.spage%253D1235%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fb89-161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/b89-161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3263.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> Ultra-fast de novo assembler using long noisy reads. <a href="https://github.com/ruanjue/smartdenovo">https://github.com/ruanjue/smartdenovo</a> [Accessed February 25, 2018].</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.45" data-doi="10.1126/science.aao7294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salcedo</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Rutter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Akhunova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bolus</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in the <em>AvrSr35</em> gene determines <em>Sr35</em> resistance against wheat stem rust race Ug99.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">358</span>: <span class="cit-fpage">1604</span>–<span class="cit-lpage">1606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aao7294</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSalcedo%26rft.auinit1%253DA.%26rft.volume%253D358%26rft.issue%253D6370%26rft.spage%253D1604%26rft.epage%253D1606%26rft.atitle%253DVariation%2Bin%2Bthe%2BAvrSr35%2Bgene%2Bdetermines%2BSr35%2Bresistance%2Bagainst%2Bwheat%2Bstem%2Brust%2Brace%2BUg99%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aao7294%26rft_id%253Dinfo%253Apmid%252F29269474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTgvNjM3MC8xNjA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.46" data-doi="10.1371/journal.pone.0029847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>,  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Win</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Cano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Szabo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kamoun</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Using hierarchical clustering of secreted protein families to classify and rank candidate effectors of rust fungi.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e29847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0029847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSaunders%26rft.auinit1%253DD.%2BG.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253De29847%26rft.epage%253De29847%26rft.atitle%253DUsing%2Bhierarchical%2Bclustering%2Bof%2Bsecreted%2Bprotein%2Bfamilies%2Bto%2Bclassify%2Band%2Brank%2Bcandidate%2Beffectors%2Bof%2Brust%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0029847%26rft_id%253Dinfo%253Apmid%252F22238666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0029847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22238666&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3263.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwessinger</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite> <span class="cit-pub-date">2016</span> High quality DNA from fungi for long read sequencing <em>e.g.</em> PacBio. protocols.io<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.17504/protocols.io.ewtbfen</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.48" data-doi="10.1128/mBio.02275-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwessinger</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sperschneider</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Cuddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Garnica</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Miller</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A near-complete haplotype-phased genome of the dikaryotic wheat stripe rust fungus <em>Puccinia striiformis</em> f. sp. <em>tritici</em> reveals high interhaplotype diversity.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.02275-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.02275-17%26rft_id%253Dinfo%253Apmid%252F29463659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiOS8xL2UwMjI3NS0xNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.49" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.50" data-doi="10.3389/fpls.2016.00126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Deshmukh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Bélanger</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Computational prediction of effector proteins in fungi: Opportunities and challenges.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.00126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.00126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.00126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">von Heijne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Krogh</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A hidden Markov model for predicting transmembrane helices in protein sequences.</span> <abbr class="cit-jnl-abbrev">Proc. Int. Conf. Intell. Syst. Mol. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2B%252F%2B...%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%2B%253B%2BISMB.%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%26rft.stitle%253DProc%2BInt%2BConf%2BIntell%2BSyst%2BMol%2BBiol%26rft.aulast%253DSonnhammer%26rft.auinit1%253DE.%2BL.%26rft.volume%253D6%26rft.spage%253D175%26rft.epage%253D182%26rft.atitle%253DA%2Bhidden%2BMarkov%2Bmodel%2Bfor%2Bpredicting%2Btransmembrane%2Bhelices%2Bin%2Bprotein%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F9783223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9783223&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.52" data-doi="10.3389/fpls.2015.01168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sperschneider</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Hane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. B.</span> <span class="cit-name-surname">Singh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Taylor</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evaluation of secretion prediction highlights differing approaches needed for Oomycete and fungal effectors.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.01168</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.01168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.01168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.53" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pertea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mortazavi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kwan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.54" data-doi="10.3389/fpls.2014.00759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>,  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Garnica</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Karaoglu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sperschneider</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nemri</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative genomics of Australian isolates of the wheat stem rust pathogen <em>Puccinia graminis</em> f. sp. <em>tritici</em> reveals extensive polymorphism in candidate effector genes.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2014.00759</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2014.00759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2014.00759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.55" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Abeel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Priest</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Abouelliel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallwork</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Preece</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Cotterill</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title"><em>Puccinia hordei</em> on barley and <em>Ornithogalum umbellatum</em> in South Australia. Austral.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D21%26rft.spage%253D95%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3263.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sakthikumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Dong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Cuomo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative genomics integrated with association analysis identifies candidate effector genes corresponding to Lr20 in phenotype-paired <em>Puccinia triticina</em> isolates from Australia. Front. Plant Sci</span>. <span class="cit-vol">8</span>: <span class="cit-fpage">148</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3263.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3249" title="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3273" title="IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uGMjDHggK9ONKrCVRzeztcol4WmCWg1_2L9HzjcQYfA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3263" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3263" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3263" class="active" data-icon-position="" data-hide-link-title="0">De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494283" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cTPy9T7MB4V1GAY0Ss8GuEtWnjlKHLw0_siXBIJt6zc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3263.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3263" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-aOvOS0CudygI4tdMUsDZdH1JZRQdpsqV7AdxtTfqkJA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3263" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3263.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494283" id="node494283--2" data-pisa="ggg;9/10/3263" data-pisa-master="ggg;g3.119.400450" data-apath="/ggg/9/10/3263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen <em>Puccinia hordei</em> Identifies Candidates for Three Avirulence Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9145-3018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jiapeng Chen</span>, <span class="highwire-citation-author" data-delta="1">Jingqin Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4191-1068" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peng Zhang</span>, <span class="highwire-citation-author" data-delta="3">Chongmei Dong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3052-0416" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Narayana M. Upadhyaya</span>, <span class="highwire-citation-author" data-delta="5">Qian Zhou</span>, <span class="highwire-citation-author" data-delta="6">Peter Dodds</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9145-5371" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert F. Park</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3263-3271;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400450 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494283--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiapeng Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiapeng%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiapeng%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9145-3018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jiapeng Chen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jingqin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingqin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingqin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peng Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4191-1068" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peng Zhang</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chongmei Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chongmei%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChongmei%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Narayana M. Upadhyaya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Narayana%2BM.%2BUpadhyaya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Upadhyaya%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANarayana%2BM.%2BUpadhyaya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3052-0416" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Narayana M. Upadhyaya</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qian Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Agricultural Genomic Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQian%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peter Dodds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BDodds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dodds%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BDodds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Robert F. Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BF.%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BF.%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9145-5371" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert F. Park</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:robert.park@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">robert.park@sydney.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3263" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3263&amp;title=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3263&amp;title=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3263&amp;title=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3263&amp;text=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3263&amp;title=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3263&amp;t=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3263&amp;title=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3263&amp;title=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494283/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494283" id="node494283--3" data-pisa="ggg;9/10/3263" data-pisa-master="ggg;g3.119.400450" data-apath="/ggg/9/10/3263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen <em>Puccinia hordei</em> Identifies Candidates for Three Avirulence Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9145-3018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jiapeng Chen</span>, <span class="highwire-citation-author" data-delta="1">Jingqin Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4191-1068" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peng Zhang</span>, <span class="highwire-citation-author" data-delta="3">Chongmei Dong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3052-0416" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Narayana M. Upadhyaya</span>, <span class="highwire-citation-author" data-delta="5">Qian Zhou</span>, <span class="highwire-citation-author" data-delta="6">Peter Dodds</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9145-5371" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert F. Park</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3263-3271;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400450 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494283--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiapeng Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiapeng%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiapeng%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9145-3018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jiapeng Chen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jingqin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingqin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingqin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peng Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4191-1068" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peng Zhang</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chongmei Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chongmei%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChongmei%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Narayana M. Upadhyaya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Narayana%2BM.%2BUpadhyaya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Upadhyaya%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANarayana%2BM.%2BUpadhyaya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3052-0416" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Narayana M. Upadhyaya</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qian Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Agricultural Genomic Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQian%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peter Dodds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Commonwealth Scientific and Industrial Research Organisation, Agriculture and Food, Canberra, ACT, Australia, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BDodds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dodds%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BDodds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Robert F. Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding Institute, School of Life and Environmental Sciences, Faculty of Science, The University of Sydney, Cobbitty, NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BF.%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BF.%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9145-5371" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert F. Park</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:robert.park@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">robert.park@sydney.edu.au</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3263&amp;rft.epage=3271&amp;rft.atitle=De%20Novo%20Genome%20Assembly%20and%20Comparative%20Genomics%20of%20the%20Barley%20Leaf%20Rust%20Pathogen%20Puccinia%20hordei%20Identifies%20Candidates%20for%20Three%20Avirulence%20Genes&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chen%2C+Jiapeng&amp;rft.au=Wu%2C+Jingqin&amp;rft.au=Zhang%2C+Peng&amp;rft.au=Dong%2C+Chongmei&amp;rft.au=Upadhyaya%2C+Narayana+M.&amp;rft.au=Zhou%2C+Qian&amp;rft.au=Dodds%2C+Peter&amp;rft.au=Park%2C+Robert+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494283/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494283/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494283/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494283/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494283/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494283/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494283/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494283/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494283/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494283/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494283/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494283/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3263" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3263.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3263.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3263.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
