<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3201" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3201.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3201.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3201.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3201.atom" />
<meta name="HW.pisa" content="ggg;9/10/3201" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the Drosophila Phylogeny" />
<meta name="DC.Identifier" content="10.1534/g3.119.400280" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Brand et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Meiotic crossing over ensures proper segregation of homologous chromosomes and generates genotypic diversity. Despite these functions, little is known about the genetic factors and population genetic forces involved in the evolution of recombination rate differences among species. The dicistronic meiosis gene, mei-217/mei-218 , mediates most of the species differences in crossover rate and patterning during female meiosis between the closely related fruitfly species, Drosophila melanogaster and D. mauritiana . The MEI-218 protein is one of several meiosis-specific mini-chromosome maintenance (mei-MCM) proteins that form a multi-protein complex essential to crossover formation, whereas the BLM helicase acts as an anti-crossover protein. Here we study the molecular evolution of five genes— mei-218 , the other three known members of the mei-MCM complex, and Blm — over the phylogenies of three Drosophila species groups— melanogaster , obscura , and virilis . We then use transgenic assays in D. melanogaster to test if molecular evolution at mei-218 has functional consequences for crossing over using alleles from the distantly related species D. pseudoobscura and D. virilis . Our molecular evolutionary analyses reveal recurrent positive selection at two mei-MCM genes. Our transgenic assays show that sequence divergence among mei-218 alleles from D. melanogaster , D. pseudoobscura , and D. virilis has functional consequences for crossing over. In a D. melanogaster genetic background, the D. pseudoobscura mei-218 allele nearly rescues wildtype crossover rates but alters crossover patterning, whereas the D. virilis mei-218 allele conversely rescues wildtype crossover patterning but not crossover rates. These experiments demonstrate functional divergence at mei-218 and suggest that crossover rate and patterning are separable functions." />
<meta name="DC.Contributor" content="Cara L. Brand" />
<meta name="DC.Contributor" content="Lori Wright" />
<meta name="DC.Contributor" content="Daven C. Presgraves" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the Drosophila Phylogeny" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Meiotic crossing over ensures proper segregation of homologous chromosomes and generates genotypic diversity. Despite these functions, little is known about the genetic factors and population genetic forces involved in the evolution of recombination rate differences among species. The dicistronic meiosis gene, &lt;i&gt;mei-217/mei-218&lt;/i&gt;, mediates most of the species differences in crossover rate and patterning during female meiosis between the closely related fruitfly species, &lt;i&gt;Drosophila melanogaster&lt;/i&gt; and &lt;i&gt;D. mauritiana&lt;/i&gt;. The MEI-218 protein is one of several meiosis-specific mini-chromosome maintenance (mei-MCM) proteins that form a multi-protein complex essential to crossover formation, whereas the BLM helicase acts as an anti-crossover protein. Here we study the molecular evolution of five genes— &lt;i&gt;mei-218&lt;/i&gt;, the other three known members of the mei-MCM complex, and &lt;i&gt;Blm&lt;/i&gt;— over the phylogenies of three &lt;i&gt;Drosophila&lt;/i&gt; species groups— &lt;i&gt;melanogaster&lt;/i&gt;, &lt;i&gt;obscura&lt;/i&gt;, and &lt;i&gt;virilis&lt;/i&gt;. We then use transgenic assays in &lt;i&gt;D. melanogaster&lt;/i&gt; to test if molecular evolution at &lt;i&gt;mei-218&lt;/i&gt; has functional consequences for crossing over using alleles from the distantly related species &lt;i&gt;D. pseudoobscura&lt;/i&gt; and &lt;i&gt;D. virilis&lt;/i&gt;. Our molecular evolutionary analyses reveal recurrent positive selection at two mei-MCM genes. Our transgenic assays show that sequence divergence among &lt;i&gt;mei-218&lt;/i&gt; alleles from &lt;i&gt;D. melanogaster&lt;/i&gt;, &lt;i&gt;D. pseudoobscura&lt;/i&gt;, and &lt;i&gt;D. virilis&lt;/i&gt; has functional consequences for crossing over. In a &lt;i&gt;D. melanogaster&lt;/i&gt; genetic background, the &lt;i&gt;D. pseudoobscura mei-218&lt;/i&gt; allele nearly rescues wildtype crossover rates but alters crossover patterning, whereas the &lt;i&gt;D. virilis mei-218&lt;/i&gt; allele conversely rescues wildtype crossover patterning but not crossover rates. These experiments demonstrate functional divergence at &lt;i&gt;mei-218&lt;/i&gt; and suggest that crossover rate and patterning are separable functions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3201" />
<meta name="citation_id" content="9/10/3201" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3201" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3201.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3201.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3201.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400280" />
<meta name="citation_pmid" content="31362974" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3201" />
<meta name="citation_lastpage" content="3211" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Cara L. Brand" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York, 14627" />
<meta name="citation_author_email" content="cara.brand@rochester.edu" />
<meta name="citation_author_email" content="daven.presgraves@rochester.edu" />
<meta name="citation_author" content="Lori Wright" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York, 14627" />
<meta name="citation_author" content="Daven C. Presgraves" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York, 14627" />
<meta name="citation_author_email" content="cara.brand@rochester.edu" />
<meta name="citation_author_email" content="daven.presgraves@rochester.edu" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Allers;citation_author=M. Lichten;citation_title=Differential timing and control of noncrossover and crossover recombination during meiosis.;citation_pages=47-57;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461701;citation_doi=10.1016/S0092-8674(01)00416-0" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=W. K. Baker;citation_title=Crossing over in heterochromatin;citation_pages=59-60;citation_volume=92;citation_year=1958;citation_doi=10.1086/282010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. S. Baker;citation_author=A. T. C. Carpenter;citation_title=GENETIC ANALYSIS OF SEX CHROMOSOMAL MEIOTIC MUTANTS IN DROSOPHILA MELANOGASTER;citation_pages=255-286;citation_volume=71;citation_year=1972;citation_issue=2;citation_pmid=4625747" />
<meta name="citation_reference" content="citation_title=Genetics and Biology of Drosophila;citation_title=Meiotic mutants: genetic control of meiotic recombination and chromosome segregation;citation_pages=352-434;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=M. G. Barrón;citation_author=A.-S. Fiston-Lavier;citation_author=D. A. Petrov;citation_author=J. González;citation_title=Population Genomics of Transposable Elements in Drosophila;citation_pages=561-581;citation_volume=48;citation_year=2014;citation_pmid=25292358;citation_doi=10.1146/annurev-genet-120213-092359" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. H. Barton;citation_author=B. Charlesworth;citation_title=Why Sex and Recombination?;citation_pages=1986-1990;citation_volume=281;citation_year=1998;citation_issue=5385;citation_pmid=9748151;citation_doi=10.1126/science.281.5385.1986" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=F. Baudat;citation_author=J. Buard;citation_author=C. Grey;citation_author=A. Fledel-Alon;citation_author=C. Ober;citation_title=PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice;citation_volume=327;citation_year=2010;citation_pmid=20044539;citation_doi=10.1126/science.1183439" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=G. W. Beadle;citation_title=A Possible Influence of the Spindle Fibre on Crossing-Over in Drosophila;citation_pages=160-165;citation_volume=18;citation_year=1932;citation_issue=2;citation_pmid=16577442;citation_doi=10.1073/pnas.18.2.160" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=L. E. Berchowitz;citation_author=G. P. Copenhaver;citation_title=Genetic interference: don't stand so close to me.;citation_pages=91-102;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20885817;citation_doi=10.2174/138920210790886835" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=R. Bhagat;citation_author=E. A. Manheim;citation_author=D. E. Sherizen;citation_author=K. S. McKim;citation_title=Studies on crossover-specific mutants and the distribution of crossing over in Drosophila females.;citation_pages=160-171;citation_volume=107;citation_year=2004;citation_issue=3-4;citation_pmid=15467361;citation_doi=10.1159/000080594" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=C. L. Brand;citation_author=M. V. Cattani;citation_author=S. B. Kingan;citation_author=E. L. Landeen;citation_author=D. C. Presgraves;citation_title=Molecular evolution at a meiosis gene mediates species differences in the rate and patterning of recombination;citation_pages=1289-1295.e4;citation_volume=28;citation_year=2018;citation_doi=10.1016/j.cub.2018.02.056" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Brandvain;citation_author=G. Coop;citation_title=Scrambling Eggs: Meiotic Drive and the Evolution of Female Recombination Rates;citation_volume=190;citation_year=2012;citation_pmid=22143919;citation_doi=10.1534/genetics.111.136721" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. D. Brooks;citation_author=R. W. Marks;citation_title=THE ORGANIZATION OF GENETIC VARIATION FOR RECOMBINATION IN  DROSOPHILA MELANOGASTER;citation_pages=525-547;citation_volume=114;citation_year=1986;citation_issue=2;citation_pmid=3095185" />
<meta name="citation_reference" content="citation_title=The Evolution of Sex: An Examination of Current Ideas;citation_title=The evolution of recombination rates;citation_pages=87-105;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. T. C. Carpenter;citation_author=L. Sandler;citation_title=ON RECOMBINATION-DEFECTIVE MEIOTIC MUTANTS IN DROSOPHILA MELANOGASTER;citation_pages=453-475;citation_volume=76;citation_year=1974;citation_issue=3;citation_pmid=4208856" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_title=Genetic variation in recombination in Drosophila. I. Responses to selection and preliminary genetic analysis;citation_pages=71-83;citation_volume=54;citation_year=1985;citation_doi=10.1038/hdy.1985.10" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Charlesworth;citation_author=P. Sniegowski;citation_author=W. Stephan;citation_title=The evolutionary dynamics of repetitive DNA in eukaryotes.;citation_pages=215-220;citation_volume=371;citation_year=1994;citation_issue=6494;citation_pmid=8078581;citation_doi=10.1038/371215a0" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=N. H. Barton;citation_title=Recombination load associated with selection for increased recombination.;citation_pages=27-41;citation_volume=67;citation_year=1996;citation_issue=1;citation_pmid=8919888;citation_doi=10.1017/S0016672300033450" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=B. Charlesworth;citation_title=Evolution: Increased recombination caused by a single gene;citation_pages=R342-R344;citation_volume=28;citation_year=2018;citation_doi=10.1016/j.cub.2018.05.069" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Z.-X. Chen;citation_author=D. Sturgill;citation_author=J. Qu;citation_author=H. Jiang;citation_author=S. Park;citation_title=Comparative validation of the D. melanogaster modENCODE transcriptome annotation;citation_volume=24;citation_year=2014;citation_pmid=24985915;citation_doi=10.1101/gr.159384.113" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. R. Chintapalli;citation_author=J. Wang;citation_author=J. A. T. Dow;citation_title=Using FlyAtlas to identify better Drosophila melanogaster models of human disease.;citation_pages=715-720;citation_volume=39;citation_year=2007;citation_issue=6;citation_pmid=17534367;citation_doi=10.1038/ng2049" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. M. Comeron;citation_author=R. Ratnappan;citation_author=S. Bailin;citation_title=The Many Landscapes of Recombination in Drosophila melanogaster.;citation_pages=e1002905-e1002905;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071443;citation_doi=10.1371/journal.pgen.1002905" />
<meta name="citation_reference" content="Drosophila 12 Genomes Consortium et al., 2007 Evolution of genes and genomes on the Drosophila phylogeny. Nature 8: 203–218. https://doi.org/10.1038/nature06341" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=J. F. Crow;citation_title=An Advantage of Sexual Reproduction in a Rapidly Changing Environment;citation_pages=169-173;citation_volume=83;citation_year=1992;citation_issue=3;citation_pmid=1624761" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Davies;citation_author=E. Hatton;citation_author=N. Altemose;citation_author=J. G. Hussin;citation_author=F. Pratto;citation_title=Re-engineering the zinc fingers of PRDM9 reverses hybrid sterility in mice;citation_pages=171-176;citation_volume=530;citation_year=2016;citation_pmid=26840484;citation_doi=10.1038/nature16931" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_title=TRANSLOCATIONS INVOLVING THE THIRD AND THE FOURTH CHROMOSOMES OF DROSOPHILA MELANOGASTER;citation_pages=347-399;citation_volume=15;citation_year=1930;citation_issue=4;citation_pmid=17246603" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. P. Dowsett;citation_author=M. W. Young;citation_title=Differing levels of dispersed repetitive DNA among closely related species of Drosophila.;citation_pages=4570-4574;citation_volume=79;citation_year=1982;citation_issue=15;citation_pmid=6956880;citation_doi=10.1073/pnas.79.15.4570" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_title=The genetical theory of natural selection;citation_year=1930" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. L. Goldberg;citation_author=J.-Y. Sheen;citation_author=W. J. Gehring;citation_author=M. M. Green;citation_title=Unequal crossing-over associated with asymmetrical synapsis between nomadic elements in the Drosophila melanogaster genome;citation_pages=5017-5021;citation_volume=80;citation_year=1983;citation_issue=16;citation_pmid=16593354;citation_doi=10.1073/pnas.80.16.5017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. F. Grell;citation_title=TIME OF RECOMBINATION IN THE DROSOPHILA MELANOGASTER OOCYTE. III. SELECTION AND CHARACTERIZATION OF TEMPERATURE-SENSITIVE AND -INSENSITIVE, RECOMBINATION-DEFICIENT ALLELES IN DROSOPHILA;citation_pages=425-443;citation_volume=108;citation_year=1984;citation_issue=2;citation_pmid=17246234" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. T. Hamblin;citation_author=C. F. Aquadro;citation_title=DNA Sequence Variation and the Recombinational Landscape in Drosophila pseudoobscura: A Study of the Second Chromosome;citation_pages=859-869;citation_volume=153;citation_year=1999;citation_issue=2;citation_pmid=10511563" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=T. Hatkevich;citation_author=K. P. Kohl;citation_author=S. McMahan;citation_author=M. A. Hartmann;citation_author=A. M. Williams;citation_title=Bloom syndrome helicase promotes meiotic crossover patterning and homolog disjunction;citation_pages=96-102;citation_volume=27;citation_year=2017;citation_doi=10.1016/j.cub.2016.10.055" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=T. Hatkevich;citation_author=J. Sekelsky;citation_title=Bloom syndrome helicase in meiosis: Pro-crossover functions of an anti-crossover protein;citation_pages=1700073;citation_volume=39;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. S. S. Heil;citation_author=M. A. F. Noor;citation_title=Zinc finger binding motifs do not explain recombination rate variation within or between species of Drosophila.;citation_pages=e45055-e45055;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23028758;citation_doi=10.1371/journal.pone.0045055" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=The meiotic recombination landscape of Drosophila virilis is robust to mitotic damage during hybrid dysgenesis;citation_pages=342824;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=T. Hill;citation_author=A. J. Betancourt;citation_title=Extensive exchange of transposable elements in the Drosophila pseudoobscuragroup;citation_pages=20;citation_volume=9;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. M. Hunter;citation_author=W. Huang;citation_author=T. F. C. Mackay;citation_author=N. D. Singh;citation_title=The genetic architecture of natural variation in recombination rate in Drosophila melanogaster;citation_pages=e1005951;citation_volume=12;citation_year=2016;citation_pmid=27035832;citation_doi=10.1371/journal.pgen.1005951" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. H. Jones;citation_author=F. C. H. Franklin;citation_title=Meiotic crossing-over: obligation and interference.;citation_pages=246-248;citation_volume=126;citation_year=2006;citation_issue=2;citation_pmid=16873056;citation_doi=10.1016/j.cell.2006.07.010" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=T. V. Kent;citation_author=J. Uzunović;citation_author=S. I. Wright;citation_title=Coevolution between transposable elements and recombination;citation_volume=372;citation_year=2017;citation_pmid=29109221;citation_doi=10.1098/rstb.2016.0458" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. G. Kidwell;citation_title=GENETIC CHANGE OF RECOMBINATION VALUE IN DROSOPHILA MELANOGASTER. I. ARTIFICIAL SELECTION FOR HIGH AND LOW RECOMBINATION AND SOME PROPERTIES OF RECOMBINATION-MODIFYING GENES;citation_pages=419-432;citation_volume=70;citation_year=1972;citation_issue=3;citation_pmid=4623519" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=M. G. Kidwell;citation_author=D. R. Lisch;citation_title=Transposable elements and host genome evolution.;citation_pages=95-99;citation_volume=15;citation_year=2000;citation_issue=3;citation_pmid=10675923;citation_doi=10.1016/S0169-5347(99)01817-0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. P. Kohl;citation_author=C. D. Jones;citation_author=J. Sekelsky;citation_title=Evolution of an MCM Complex in Flies That Promotes Meiotic Crossovers by Blocking BLM Helicase;citation_pages=1363-1365;citation_volume=338;citation_year=2012;citation_issue=6112;citation_pmid=23224558;citation_doi=10.1126/science.1228190" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. P. Kohl;citation_author=J. Sekelsky;citation_title=Meiotic and Mitotic Recombination in Meiosis;citation_pages=327-334;citation_volume=194;citation_year=2013;citation_issue=2;citation_pmid=23733849;citation_doi=10.1534/genetics.113.150581" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=M. L. Frigge;citation_author=G. Masson;citation_author=D. F. Gudbjartsson;citation_title=Common and low-frequency variants associated with genome-wide recombination rate;citation_pages=11-16;citation_volume=46;citation_year=2014;citation_pmid=24270358;citation_doi=10.1038/ng.2833" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Kulathinal;citation_author=S. M. Bennett;citation_author=C. L. Fitzpatrick;citation_author=M. A. F. Noor;citation_title=Fine-scale mapping of recombination rate in Drosophila refines its correlation to diversity and divergence;citation_volume=105;citation_year=2008;citation_pmid=18621713;citation_doi=10.1073/pnas.0801848105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. M. Lake;citation_author=K. Teeter;citation_author=S. L. Page;citation_author=R. Nielsen;citation_author=R. S. Hawley;citation_title=A genetic analysis of the Drosophila mcm5 gene defines a domain specifically required for meiotic recombination;citation_volume=176;citation_year=2007;citation_pmid=17565942;citation_doi=10.1534/genetics.107.073551" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=A. M. Larracuente;citation_author=T. B. Sackton;citation_author=A. J. Greenberg;citation_author=A. Wong;citation_author=N. D. Singh;citation_title=Evolution of protein-coding genes in Drosophila.;citation_pages=114-123;citation_volume=24;citation_year=2008;citation_issue=3;citation_pmid=18249460;citation_doi=10.1016/j.tig.2007.12.001" />
<meta name="citation_reference" content="citation_journal_title=Sci.;citation_author=Y. C. G. Lee;citation_author=C. H. Langley;citation_title=Transposable elements in natural populations of Drosophila melanogaster. Philos. Trans. R. Soc. B Biol;citation_pages=1219-1228;citation_volume=365;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=D. L. Lindsley;citation_author=L. Sandler;citation_title=The genetic analysis of meiosis in female Drosophila melanogaster.;citation_pages=295-312;citation_volume=277;citation_year=1977;citation_issue=955;citation_pmid=16292" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Liu;citation_author=J. K. Jang;citation_author=J. Graham;citation_author=K. Nycz;citation_author=K. S. McKim;citation_title=Two Genes Required for Meiotic Recombination in Drosophila Are Expressed From a Dicistronic Message;citation_pages=1735-1746;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10747066" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=E. A. Manheim;citation_author=J. K. Jang;citation_author=D. Dominic;citation_author=K. S. McKim;citation_title=Cytoplasmic Localization and Evolutionary Conservation of MEI-218, a Protein Required for Meiotic Crossing-over in Drosophila;citation_volume=13;citation_year=2002;citation_pmid=11809824;citation_doi=10.1091/mbc.01-06-0318" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. S. McKim;citation_author=J. B. Dahmus;citation_author=R. S. Hawley;citation_title=Cloning of the Drosophila melanogaster Meiotic Recombination Gene mei-218: A Genetic and Molecular Analysis of Interval 15E;citation_pages=215-228;citation_volume=144;citation_year=1996;citation_issue=1;citation_pmid=8878687" />
<meta name="citation_reference" content="citation_title=Recombination and Meiosis, Genome Dynamics and Stability;citation_title=Synapsis, double-strand breaks, and domains of crossover control in Drosophila females;citation_pages=125-151;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. E. Miller;citation_author=C. B. Smith;citation_author=N. Y. Kazemi;citation_author=A. J. Cockrell;citation_author=A. V. Arvanitakis;citation_title=Whole-Genome Analysis of Individual Meiotic Events in Drosophila melanogaster Reveals That Noncrossover Gene Conversions Are Insensitive to Interference and the Centromere Effect;citation_volume=203;citation_year=2016;citation_pmid=26944917;citation_doi=10.1534/genetics.115.186486" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=E. Montgomery;citation_author=B. Charlesworth;citation_author=C. H. Langley;citation_title=A test for the role of natural selection in the stabilization of transposable element copy number in a population of Drosophila melanogaster.;citation_pages=31-41;citation_volume=49;citation_year=1987;citation_issue=1;citation_pmid=3032743;citation_doi=10.1017/S0016672300026707" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=H. J. Muller;citation_title=The mechanism of crossing-over;citation_pages=193-221;citation_volume=50;citation_year=1916;citation_doi=10.1086/279534" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Myers;citation_author=R. Bowden;citation_author=A. Tumian;citation_author=R. E. Bontrop;citation_author=C. Freeman;citation_title=Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination;citation_volume=327;citation_year=2010;citation_pmid=20044541;citation_doi=10.1126/science.1182363" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. L. Oliver;citation_author=L. Goodstadt;citation_author=J. J. Bayes;citation_author=Z. Birtle;citation_author=K. C. Roach;citation_title=Accelerated evolution of the Prdm9 speciation gene across diverse metazoan taxa.;citation_pages=e1000753-e1000753;citation_volume=5;citation_year=2009;citation_issue=12;citation_pmid=19997497;citation_doi=10.1371/journal.pgen.1000753" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. Ortiz-Barrientos;citation_author=A. S. Chang;citation_author=M. A. F. Noor;citation_title=A recombinational portrait of the Drosophila pseudoobscura genome.;citation_pages=23-31;citation_volume=87;citation_year=2006;citation_issue=1;citation_pmid=16545148;citation_doi=10.1017/S0016672306007932" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. D. Parvanov;citation_author=P. M. Petkov;citation_author=K. Paigen;citation_title=Prdm9 controls activation of mammalian recombination hotspots;citation_pages=835;citation_volume=12;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. E. Ptak;citation_author=D. A. Hinds;citation_author=K. Koehler;citation_author=B. Nickel;citation_author=N. Patil;citation_title=Fine-scale recombination patterns differ between chimpanzees and humans.;citation_pages=429-434;citation_volume=37;citation_year=2005;citation_issue=4;citation_pmid=15723063;citation_doi=10.1038/ng1529" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet. TIG;citation_author=K. R. Ritz;citation_author=M. A. F. Noor;citation_author=N. D. Singh;citation_title=Variation in Recombination Rate: Adaptive or Not?;citation_pages=364-374;citation_volume=33;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Sandor;citation_author=W. Li;citation_author=W. Coppieters;citation_author=T. Druet;citation_author=C. Charlier;citation_title=Genetic variants in REC8, RNF212, and PRDM9 influence male recombination in cattle.;citation_pages=e1002854-e1002854;citation_volume=8;citation_year=2012;citation_issue=7;citation_pmid=22844258;citation_doi=10.1371/journal.pgen.1002854" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=L. Ségurel;citation_author=E. M. Leffler;citation_author=M. Przeworski;citation_title=The case of the fickle fingers: how the PRDM9 zinc finger protein specifies meiotic recombination hotspots in humans.;citation_pages=e1001211-e1001211;citation_volume=9;citation_year=2011;citation_issue=12;citation_pmid=22162947;citation_doi=10.1371/journal.pbio.1001211" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=F. Smagulova;citation_author=K. Brick;citation_author=Y. Pu;citation_author=R. D. Camerini-Otero;citation_author=G. V. Petukhova;citation_title=The evolutionary turnover of recombination hot spots contributes to speciation in mice;citation_pages=266-280;citation_volume=30;citation_year=2016;citation_issue=3;citation_pmid=26833728;citation_doi=10.1101/gad.270009.115" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=J. Stapley;citation_author=P. G. D. Feulner;citation_author=S. E. Johnston;citation_author=A. W. Santure;citation_author=C. M. Smadja;citation_title=Variation in recombination frequency and distribution across eukaryotes: Patterns and processes;citation_volume=372;citation_year=2017;citation_pmid=29109219;citation_doi=10.1098/rstb.2016.0455" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. True;citation_author=J. M. Mercer;citation_author=C. C. Laurie;citation_title=Differences in Crossover Frequency and Distribution Among Three Sibling Species of Drosophila;citation_pages=507-523;citation_volume=142;citation_year=1996;citation_issue=2;citation_pmid=8852849" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=K. J. T. Venken;citation_author=Y. He;citation_author=R. A. Hoskins;citation_author=H. J. Bellen;citation_title=P[acman]: a BAC transgenic platform for targeted insertion of large DNA fragments in D. melanogaster;citation_volume=314;citation_year=2006;citation_pmid=17138868;citation_doi=10.1126/science.1134426" />
<meta name="citation_reference" content="citation_journal_title=Cell Cycle;citation_author=S. Wang;citation_author=D. Zickler;citation_author=N. Kleckner;citation_author=L. Zhang;citation_title=Meiotic crossover patterns: Obligatory crossover, interference and homeostasis in a single process;citation_pages=305-314;citation_volume=14;citation_year=2015;citation_pmid=25590558;citation_doi=10.4161/15384101.2014.991185" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Weinstein;citation_title=THE THEORY OF MULTIPLE-STRAND CROSSING OVER;citation_pages=155-199;citation_volume=21;citation_year=1936;citation_issue=3;citation_pmid=17246790" />
<meta name="citation_reference" content="citation_journal_title=Comput. Appl. Biosci. CABIOS;citation_author=Z. Yang;citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood;citation_pages=555-556;citation_volume=13;citation_year=1997;citation_pmid=9367129" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_author=W. S. W. Wong;citation_author=R. Nielsen;citation_title=Bayes empirical bayes inference of amino acid sites under positive selection;citation_volume=22;citation_year=2005;citation_pmid=15689528;citation_doi=10.1093/molbev/msi097" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Zalevsky;citation_author=A. J. MacQueen;citation_author=J. B. Duffy;citation_author=K. J. Kemphues;citation_author=A. M. Villeneuve;citation_title=Crossing Over During Caenorhabditis elegans Meiosis Requires a Conserved MutS-Based Pathway That Is Partially Dispensable in Budding Yeast;citation_pages=1271-1283;citation_volume=153;citation_year=1999;citation_issue=3;citation_pmid=10545458" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the Drosophila Phylogeny" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Meiotic crossing over ensures proper segregation of homologous chromosomes and generates genotypic diversity. Despite these functions, little is known about the genetic factors and population genetic forces involved in the evolution of recombination rate differences among species. The dicistronic meiosis gene, mei-217/mei-218 , mediates most of the species differences in crossover rate and patterning during female meiosis between the closely related fruitfly species, Drosophila melanogaster and D. mauritiana . The MEI-218 protein is one of several meiosis-specific mini-chromosome maintenance (mei-MCM) proteins that form a multi-protein complex essential to crossover formation, whereas the BLM helicase acts as an anti-crossover protein. Here we study the molecular evolution of five genes— mei-218 , the other three known members of the mei-MCM complex, and Blm — over the phylogenies of three Drosophila species groups— melanogaster , obscura , and virilis . We then use transgenic assays in D. melanogaster to test if molecular evolution at mei-218 has functional consequences for crossing over using alleles from the distantly related species D. pseudoobscura and D. virilis . Our molecular evolutionary analyses reveal recurrent positive selection at two mei-MCM genes. Our transgenic assays show that sequence divergence among mei-218 alleles from D. melanogaster , D. pseudoobscura , and D. virilis has functional consequences for crossing over. In a D. melanogaster genetic background, the D. pseudoobscura mei-218 allele nearly rescues wildtype crossover rates but alters crossover patterning, whereas the D. virilis mei-218 allele conversely rescues wildtype crossover patterning but not crossover rates. These experiments demonstrate functional divergence at mei-218 and suggest that crossover rate and patterning are separable functions." />
<meta name="og-title" property="og:title" content="Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the Drosophila Phylogeny" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3201" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Meiotic crossing over ensures proper segregation of homologous chromosomes and generates genotypic diversity. Despite these functions, little is known about the genetic factors and population genetic forces involved in the evolution of recombination rate differences among species. The dicistronic meiosis gene, mei-217/mei-218 , mediates most of the species differences in crossover rate and patterning during female meiosis between the closely related fruitfly species, Drosophila melanogaster and D. mauritiana . The MEI-218 protein is one of several meiosis-specific mini-chromosome maintenance (mei-MCM) proteins that form a multi-protein complex essential to crossover formation, whereas the BLM helicase acts as an anti-crossover protein. Here we study the molecular evolution of five genes— mei-218 , the other three known members of the mei-MCM complex, and Blm — over the phylogenies of three Drosophila species groups— melanogaster , obscura , and virilis . We then use transgenic assays in D. melanogaster to test if molecular evolution at mei-218 has functional consequences for crossing over using alleles from the distantly related species D. pseudoobscura and D. virilis . Our molecular evolutionary analyses reveal recurrent positive selection at two mei-MCM genes. Our transgenic assays show that sequence divergence among mei-218 alleles from D. melanogaster , D. pseudoobscura , and D. virilis has functional consequences for crossing over. In a D. melanogaster genetic background, the D. pseudoobscura mei-218 allele nearly rescues wildtype crossover rates but alters crossover patterning, whereas the D. virilis mei-218 allele conversely rescues wildtype crossover patterning but not crossover rates. These experiments demonstrate functional divergence at mei-218 and suggest that crossover rate and patterning are separable functions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the Drosophila Phylogeny | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"v5A8YPxtysKNZo2bO1afu51XXgqr1Y66YbfeCztyMTc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494293","apath":"\/ggg\/9\/10\/3201.atom","pisa":"ggg;9\/10\/3201","ac":{"\/ggg\/9\/10\/3201.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3201.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3201"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3201"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3201":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NjDP8gLE3y0q2CRRglkjWPcUaYNzHhNlvqDe2Mqu11flhfve4m6chjyu2x05nw7QttK3c7txwq34m7BP44dgGopY9ej3Z3l9krTOMXS1+MUU4ph9pUzxQziouc5d\\\\\\\/aRq7fq0Dw3vBU60aqsIwa6i1kTzurv3wB2IQacW60cWC2NHGUk9aIGToInyN6ysRZkdgbltU9hNapD6y\\\\\\\/E2gB0EPzULcTj6SFRfvy\\\\\\\/XyQqyMLRHDLg\\\\\\\/0y9Dp6zV3yAwyC2TW7BzNhqZGUb5YiW2BcpPaZZ1x50tlhwwuv67qN+unV9afIquymfKvXtHp8Z9hAYmfRQ2ISu5h3uvxYELkPJk6hGrFttHwe2iWVS+nHkGvvuqtSII7OIN8Q50Km3kjX9ewhQbGKkTWycl\\\\\\\/LrF3FzG++N0panWMSLfVWjSvTxCU6Bw3oMsvDXLt1E0JXwUyz3gMbxxQX8j\\\\\\\/xXqJJzE\\\\\\\/HoP\\\\\\\/fbC6wdBfdgJmbgGK2cmrmCN8HE9stU6l2EmMEhk6UCYaUtVJxfEuFyPXvu\\\\\\\/2XmxdK3\\\\\\\/fNC49DUXSWznK4wvEaGc+E4QZPBhYsQwTvueMLnOruWd7Zb0ixPbHhzvOc7b5934yxfyXt6PAkXa\\\\\\\/GVIUkWX\\\\\\\/7h2CXB6mjC9dEvv3EtOTgpTJ0KAuj2nORI7xByDHZgO5ecBLRg4bNViGnJFD7yBprEar2FrNRTwKv\\\\\\\/rnYzPGHXYJ199EWn6TjRdTaOiAvFN0PBoxqnfZjnCbyc12VRT+48A9PAn25Yx6v\\\\\\\/jvcQ7ler0uHwc2jiRby9uC\\\\\\\/DAfTD2taA7x\\\\\\\/yi2XtlECqjIRGdlcSsvPEMBcSllvBrvywe0EB0UvnwHqCj5Kl6QDiVktFvrg1pBjNbsCUPC835mlA+ILh+qMiMNun99qO+t1cqLnjDHToI9bq4WzhVaOtXWH5iIOaXR9SjE5\\\\\\\/qpDWFEbptTTHmIhAJhvz6IV3FugYQKJYg9FJvgrz60lbHS6w+yPPXcGfJNTrOGu5iMFX8MzkJq7yTLtWfiNbzPpLIfLDtyutbLsU9VZ+VcsC26nnpd58ehAqkT0LlcH3Sc6Ar9TY7S0xV2PemBf0vbZGfy0rw7o1QiqEmT9Sh1HEePuwYQMbHb73XnV0jB8oFrYoE1M\\\\\\\/xt10fMRjZ35odEDdqs9086TNmi8JzPMQZANRJS6UzEinzg1P6aJJcDfSJRy5j3pW7s4BqTM+tkYqekOjBE8Gdu0JOU2W3\\\\\\\/VBfCq57udxY\\\\\\\/fqSvXun0RuqF4Jvsoe\\\\\\\/ErACGeaV5CBFFLy9KnCF9qrj\\\u0022,\\\u0022iv\\\u0022:\\\u0022XZK2IJC+Pb+9f+p\\\\\\\/QbAnwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229aa58cd74c8ccdc5a4d0bba55936d066d1562d34512658c5d813a65436465d9d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HrNhb23CM6SCSN7eR3RfYdMkKFBmjaTWU0ZFAYXquoCDZ6n4Lp6r9dJ1QKVchPmS3IOGPgbdyb1XqKj9btZPdGtdpl7DiGbnd2hqSpWFkgTvGEezTqnsUlV+4IEEQCkrhIDIHFf\\\\\\\/FVTMNPi90YHzfNBnXjQ\\\\\\\/Jm0bHuSI4bWymktLRjYrSqu0TvYlSmtQyTyyWTM\\\\\\\/qng9IOu2FWDmFiSJ+NxdTtfw7Z97UIgyRzgMuzLwgB3Db6pfjMYRol1xpGid+Oii6ru5pe\\\\\\\/3WB3Q63OPr3f+aG+ua928D4XXVl8og\\\\\\\/HtzVjloxagUPEVJ2b0umYl+ako8sdRQvuafJ20K1+Fs\\\\\\\/eg52k64fHjEr0s\\\\\\\/CIaKsLS4roqm8OxGbVhas+\\\\\\\/GfWIp8xIXRH8eSHAYK8x0lRTUR1W7KUgaPHCB0Nn7yx406B28mEzDr1iwSP8v\\\\\\\/bIXG3ftzZovttADvb\\\\\\\/6CqpXlh5KD3HFNxjMJu8soEsXOHfEm9BRVCU+vOpIIWuMKgOS+UYrAsD4+aEHNJjEGaqW+P8her18\\\\\\\/Ze6hyZHd+diSyq7mcxFmxPlPiZM1Wfwwo\\\\\\\/80w0x3esdEh4tG0X4i8EN1PgiY4lUv6593Kch\\\\\\\/T+C\\\\\\\/d5LyIG5LR5uo6FQbHi1oqe12yrsgeWMGnTSuDH3hDpsjA2lIjEn8Y1uo2Ux3zpbMmG7aZ9Rug1LIa1lvr\\\\\\\/r1WfGLl17yH1vorxjfUovdQ94m2f5fmFJ5eBuJt5tXAYK+u+zXOhWr4OvAA0ya\\\\\\\/P55s\\\\\\\/l8OaG\\\\\\\/a8KIhnpUQ3GTBAHyNRzGE4WHRyiCMfkpIK01gHFXupDn20jNn1YL2terx3t85AezCCG3\\\\\\\/5noWYuaQcZzsTZqZrSHor+CJGt\\\\\\\/OWu6ddwCPFIttSpCuZ2RCuxzLdLRbQ5qIhqi950N1He7LAujVrC\\\\\\\/Udyf\\\\\\\/BBNYmSkvnEQs4SSzaAyZHxogVn0qRDq9krZl+W+3KZrtME1xxWDnagKK8BZq1uNTMor3lxipsjbtkXUm\\\\\\\/YUMSR5I59QE8cCzQ307Bpz8rRrYNFGgQTf9qN4zFzMQGBt78qmDEqh1+82KhtWAwPsqN7ReF5zG4mUGPKrKLfODDdLVrR33nQfF5dq6Tuw0iqT\\\\\\\/pCLBva4b2bRhf55Mkz9ytRvmPzxXpOV2Wlgq3VbCMS6VLlCxGsbI+SgODWl+AYy+gMG9WZ873CNNkNrRjz+muQiO8gwVE4QEXJQResQUnsLkhR0TJ90WNTdMSF5pDmpEm3UECVI2kAxY4f40\\\\\\\/P8KiXxt\\\\\\\/PpyrOnGxighvnjc8Jx\\\\\\\/xYJqykDrn3X5O+SyTM6QCFgVYS3+iak\\\\\\\/tiP\\\\\\\/UovwS\\\\\\\/Yv5+\\\\\\\/Dn9coQYEwBboC0QyINE\\\\\\\/gcauoV4G0Tagme5AZsJJTUHUq2QLVBnNfzcaicln2+DAF86F3fFCb+qBRZ4M6yWwc+2TOmesZeHCU\\\\\\\/c6\\\\\\\/WTIMfQACvu1lBTMGrkR4eN6TGFRTLs+I\\\\\\\/Y7Zhd2ZaK4wtwvQpvU18Z+Rxm3uzY4w4Roi8DMLNl19VClyZ5cmXBnJHlSfJP1In0Ol0+dQufd3FQnmMyfkmM0oSBBlpHSK\\\\\\\/KTo8bZvL8mgz\\\\\\\/DgzVqeeq4ZQkDKuldl5oQGI08wNBs1DHniLJh4dYu24tapE7twLFnYq9XwVKmAIii6XpARXNy9AAvTBV8ViEG\\\\\\\/TaWKeECvrmmRqo0tpf1YHQ30JME3vS2\\\\\\\/L49rIImi\\\\\\\/dG8c+jAv17caZccV8+GYoYB5DLH5E817vxt3\\\\\\\/jnpuNJAZK1z2nrnfx2vJtRbYPNrpXB\\\\\\\/14Xdz9R3sLL9Mz6W90i8ATJbgumz0EKD\\\\\\\/\\\\\\\/f53JyltNJ+Wmogd2cdneaHMOGXePZMo2YMCiaJ3w9i0DG+LOY6wApfevVedmHCn1VLWxAFwRIs+8bgM7DUGB\\\\\\\/\\\\\\\/wjpzPBx+3GkMcsrouXyja+mSMobmyC99TzkNcsVNtfAzpoBtDbNnVM7Y9t7wjfIboTHU78GFexVHD6pJ4u1aOeSKW6fv0jMn\\\\\\\/qgxjgy9Q69rNd8\\\\\\\/Y5KayeiieTCQZUjUwZRFaLLCgxsgvH+l48kGMg\\\\\\\/32JcMl1HK3uJA+pCAdmUNpOoZgHDJ8lQuymKuRA8rj2vzw8exT7XGwNQFIVMx1La79J0axWlT+YDW8iegWymYg20UeRfLvIAnzEjoB7+T7jSHvS7d0+GmyrAIOcc7OjAaD7kGAc80uLBBbWOVH8OAS9fK5LYpNNKGDnOgpNZskyswf8pdqxZWdD5X3\\\\\\\/OKydFYx4qcjsEGAIBTpWbg+lOP1PQyW0R57NgBvzRovWqIifPT1kB8LaPYYGaZZ+BFQsDX6WG3ejE1VGPtg46tmD+FcXfr1qVCngJzD1m6ACKRYY8h5AYGYJAu8\\\\\\\/iOZInq05UiI1kixEkLF9h9SZNoKK6D3Riu+DpdT1x\\\\\\\/tMcqGloAyuCB5zfCMMtF9o6B9fwG45HSKs71kUTPfx+Y9z7\\\\\\\/8qw9MLNuBkcKarff6GwMdpRzINKqpD8ZwWIG7As7OXrvksQHV\\\\\\\/\\\\\\\/uy+iGlMq7JBbLSCL5+MPE3hrm2QxjPqVBsNewS8zvCRA4W56eDI2PKEdKjH3YNtSDtlhPyx+\\\\\\\/6KIv\\\\\\\/xfj9cTZnq\\\\\\\/Ewleg1IvvvC1PHVCxsLlaZSgyl9ERQYqtGfZg00Uafm4UQj516uUJKYoBwShwYPQkouuzyllXVo9oXHrezMR7619YVb8mNzWfelqScFkBnljMqnoVZtcX2hF5zywGDBnNdLzVtWMvZwY8jZGprbTs1SfTcMEz6vkpQsPwpIwN6wKqB3eMIKFWY1p0LhiK5gHPRFIF72fy+O4Pe0UcdWfrk52WnoNjHEr1KPPxNMQPo+0gFQzDCSDXI6l59xdLFvVvX9aqL0+brnizNJu0tQXD9OBvSfLl63jicotRq+CdOlqBumPUg+taJG05V5MIHR67C9TC2TpX98gqmFQURYMB3H43OCFgebfjldB2eeCppei0kD53P2lA9RtpIGEz7p36QM8h5gQQ1MydFhKAzNrBz5TFcUzn5Rq22vMauRCC9fAwlfuADHxIMp9pdab8F\\\\\\\/8eH4iFO8opc0bx7\\\\\\\/FxnAhCDUT+0+0eqO42N8BOET5sRgbZGJCyGZ4rcyxSGJ9XhaO+1vOImLETOh5\\\\\\\/0xv7vfZ8\\\\\\\/DCi8YGPD0LeCof1RV+Wla0Fv03QTOuNAFqaWg8w+HQja0yVC+gIxtg8fHpLMuZgL6VH9zQ9uhcksmYmuu\\\\\\\/Xwtt6hGwBxIycADYp0VaAB65J5Y3nxB3ELHqmHFxhmfruiuVvclkpDpuPm9ATYO+xVJ7EwXKdGJdQ0WtaiwWs32TO75yfvm9mVzR+qESUNtoEEP0JbxJ9DZospgc1PlT0evoboWGLfVSuxswPybENhwlHUk+y7eUoq+vEmwFzIMy4TpfQpUwI+Z3MoBSIvV5gCbeyoBJKKr8p5w8JEPysmnGx1aop6kznJ0efZfsn3aAw6rz68JQYWqfu4S90mbAKo35HfZnMtdszHyHz+tiNNoAlDBCCddVPW64dLuCNMs\\\\\\\/2H6Ls898JWDvstPlrn3BOEDWz9cqZM4wCy9+wW4zna3oH7rK8bArB3etP+9nDuXkUcid6vBGLCx4J3LaT8+UN2Jizq8pWtUGmbFN\\\\\\\/DapnfzN7a4cF\\\\\\\/HW0KyL5e99YohOAbn5MS5NdzFEA6qT6aIrJT3+Ho87qsZftDAmMBbqT3YMKehPf2xe0cQgq6YtyfNFVkQBYNF\\\\\\\/0\\\\\\\/eWlbOuJ5xFh2ARW60k1mGOuqB6\\\\\\\/OC0Wyuu\\\\\\\/OpZ6fLfnPzFWAjLlFyv+o6i0PU60Y+gmlueRzCJGb6puWjB1r7OyFC2HfV0T\\\\\\\/c6AU8G+vz\\\\\\\/AoMCKqYn8Ca4+aumJ1ZPuN45iWj8ghlouQpN615wz7ebO8RRel91eitjMjk1yCKxYMRJVHOJFYkX+A0ayW+WJpigwILyYb6OGFWVFneLe7hkF6KlYoIk9VYr6PaAgIRyzD7G9MwGLsExkxkxbfrb8iQ5FWdiM00A3Tp2gX\\\\\\\/zLFag09j2csQfT\\\\\\\/7bazQjsAFt4nXkEqa1FOaMYWJCThiMojRayy0OwcmWsAk67pXOqR\\\\\\\/Ocz8KVFuc+RD2NzdbdxLEHoBiQUCkSkIqmW+Fu7AnFs8RynAUdAi8M4r6BnQNMcO4ftBAV\\\\\\\/km4df4Y3+a1u+oWGMHqiS+YsZnJh0H5+tLfvwXATF2ayZmlP1DK525ZZTOT+ZzXvxT2LTXPmTsiiurUeAvesxZX4dYAWxjdu947eCIec6XT+0jLPEFvH2XoCbXmjfChUNWTNOZ1IgwtKlUp9vwGi+WL0uJkpdCWAVMvnyorQt6Kfbr7nj+njOEZYL2QvJdGSokNGNwnIdeltLV4XxV4L5eW6QF8+9GBA6A7k6pJaIAfffbDBWxofrS61yb9iX7Zs\\\\\\\/NrqnZH8YweKpKk8K8FVznt2BLpOdd+InWMzlzXoDFsY1zOWz7Ky3k\\\\\\\/A08BF354W1LmjwInOmt9iSD0NmzQFIQXp+MJDNBnBVpoLOOnqGt1TIdO9HESucYOERl3kgv3E4FL8OkZQPc\\\\\\\/lngYhQirmwCqCHNM5JnVqwcw9r+vD592qDKFj7wPIVatZ2DTk+SUSLXJwZaHulFQ7xj9nVTBSwa1NYqS\\\\\\\/C5aTR8Whjs2SiT5VpBXGjGBrZ0w\\\\\\\/0z7Ddx1VdHBrDh4a+HVucrE1v2CMeRopz8XqLuo\\\\\\\/zFA16d43TRVB63IkCWIkwm9icXCjbhmOIriBK+EsmJqamZ6w2jOqZr1ZV7Va86EQSXiJkwJtguKcL9fY3HWcaGpEIBoIPKYgvsB9TGBWXixR3SinOs\\\\\\\/OioFVMubRzIKU4JIbkFOCyvd8\\\\\\\/UaBlme8JpPgLtn4wazFrR7+rOQF2xdT06UXLmqcAnJI6GjLUsx6Q\\\\\\\/ep\\\\\\\/T2I9NsRADpjnpcda3T0mEC7zKPMU0FL5yIfkz3rne1mrhg21p77nKNDh0kfx7SxlsNgec8zOeqxjLL0iBJbWZUE8NfzQZJ3ILW+z5MjlsFvfAUsS+97qX1AgXnOtzzK+a0fDRKA87f8ATC3xoO9TpT63JrFgE\\\\\\\/KF5XXurn39Qwjw+aUi5c6L6TJoOXcJj+NBuspkP3kq82MUq0p8y0pvQ6d6j9GGqLKMoWjoHvP4cfHzJfBo35W6ghOSyphzr6RMaVlh50mFjZ5c7LP61DbQXk+age7ZRpevKmsof6\\\\\\\/yeSzNe59y0BQDiQi62sIR8bkxDLiaB3Q+z5GxETbHakidzELvayxyaSjxYsnfygk9sgigVChRoPs\\\\\\\/X3CY1EpeaITz4PK7HC80gmanB4Hm3NW9Do2dVirwnWlll2byWisV8Sw73pkuixi825tFgYkapeURWeKCOiJp8kn742IgcWyw6t\\\\\\\/YS9YKZy+AejIpZHGf9E\\\\\\\/jlEzCjoYi1LZuT\\\\\\\/+q80+HQMgseGHFUzJH0iB\\\\\\\/f5FGxKw6GOmrYNrV5yoc+XkZn1bLzW1YJ3qvsietpG3kwqeBOB915VlbQU4K0PCrDe6t\\\\\\\/PMein0plf0zQMLu1OF1a5uIHeSP3bK9ocvelF9untYbe5VQsLFQpz\\\\\\\/T46xDU0F54hwVHB6NnRvi4QGDDoBWmVMHAE+OSCqwIdOjAIBBmPVOw7TjXkXgGLhLkSSpKyu3muQlZDHrZvZSEO1l6q9LClDcSpJuA28EvtS6DVxgMiaOrszWhuHmL0wWrAw\\\\\\\/NTWgzqT5M\\\\\\\/bRdNn2gvWHLUo8\\\\\\\/i3jzjVe5BBXp3d\\\\\\\/OP7wo2BZexUtpj7i+EhqGUsFzu9DBt8ljKXrnwDJbhUi76yjT5mmBS1dVWQ6c8vNujQBuXCyulGOja+PBie7tN9R50wgvZoWDGWHjUSEWVt52CLJJSrYtAwUW\\\\\\\/m8yIVc7YgP1HPqjJFx7xVY3kiBzebiXiUXDcExZfSOSOrT72NKgmgUpwcOVUkb6uL4Sa1pwceFpyk9AtfuslNXMBnmSh2+zMqzlLZjO9a9abJ52ownXn0ku9ACxoU7b3ZLSrLBIORZjAOzUdME5tgTLA1IJYUMh0sVNyI+ssqt9wlQOcWGfFA1WJPIrZIQs4BgOxxqi6xNX91ZpNSXv9BfFYaleutBRtXA27GeVYP1ry16ScGhjnaK32Sy8vPGWWAkyDL4F6lQ6Jjq3Z7ciectdNXHiSAPWwmJwkXD8IKntfkDVS0fpSPZE\\\\\\\/fJN0YT3hdfEGlkRWwGJEsBuHR1DaB0pYmG\\\\\\\/ePfoOIPMc5eXp861EY1Un2XIO\\\\\\\/UUkmRtnB8bBrIfDFEOi6FJt2Ef\\\\\\\/65uJw81D97WGdV7z+v7RFK8rCX\\\\\\\/nqZxD5s1zarrvppReI\\\\\\\/V1AI\\\\\\\/QAC7sKoYmlJOu+caPBpdj5TmLfcQ+3MMhNeiMuGwiKm2sxQC\\\\\\\/3E\\\\\\\/OFpVXW1FLcIMDfPDcXDaYbuFFLKl4HH04QMylpO0xcwT2SBSujWUTeHexaN7vlqkDlS7P63cOO0\\\\\\\/dXtq47Ka\\\\\\\/YUwz2mLHSbf9qOMnET1lUesDrCD5yUwgcUPmjt4DCk9l5Jb4TmjFf5MAigAKbF8fdRb9SNDPvUmeGLTlIGPhsQ9b5wUIcJ9Uuewa\\\\\\\/U1QFOt3eDKt4dWfqSF1Lcf6neN5Oqt7T32EiGFmuRoyzgn6iuzC0Lpp1HK4srSYlylfRNkmTsk6FNCTfCuGEj7QiOqQUESd2lTLZmKyLM+KupKqvtb2Jnratwgcb2UfAVnQo40dpQUadTyopAL8hHA3hGJFD2CyVVTkOvJphxXl7ScVCCONFP9cf2VcaAfqEG2fdrRbrUYtzBis03S++OplSk938qJXlQz6bTGfkMNm3HI68kl5+cOfnQ3IKgORGLCy3pqpeZpwEJP42auEd\\\\\\\/O8SXDvU59zK+6CYk0DafAP2SiZLWWmyu5P7n24Gz0iSdMG0O+hp2g0SHDTho\\\\\\\/fux0zn6DmuDiei5yioVOvSWPfwMduTOYQBxuEiVOLLgRtASEda97MI+eAhsVhQzX3YH6eUhnRzyfm192Q120yTKD+ZwxREHB\\\\\\\/ta7VkV0nchWtJXr5cFt5OjVwhWOnXLlpnKLWoENL9p\\\\\\\/2SFL\\\\\\\/6cFUJ+Aws1j2IvrkroXEAh9h0Uzxddc6T3JMtKaDVpRRsM2xYO+9emQIebxboHS3+ORrjprAmTzexWewp33Tix6fmd9aJCUGFGKsCVQlRsJlbXEiEqYcYfy5l+m4W6nsaLjDLuMAQy5VnjosKqM+ieevAQnCBt\\\\\\\/o\\\\\\\/bnGCV\\\\\\\/XaFLTGoFnSHgWizwh53xeUM9NAouFW99sDVCrEXa2xB1H9HTT\\\\\\\/KmUkVCol6II\\\\\\\/LUfKbvVnQOhVbzJG8JIbB6Sdf\\\\\\\/nMe7C59aZ9IvUpx0Dw7AsLoHWHw\\\\\\\/raIUaW1y9Mb5q94tbbGIxZbaCriTBizXFBGr\\\\\\\/WXxu\\\\\\\/0XQQgicSW9vSkE84tEjC4uFqnxKcVz\\\\\\\/RvB0EgVoESftHLnTZBwyxu+RWaTrB8oI1TiKGs6WE2gQPeX4ggiI6J47ZdcskbMHq\\\\\\\/ZxBSU7UeVnvrj\\\\\\\/xFmFFzKqvBHOOArpFhnUylEcQJrhNPINP4BUkalStk+EtRgk0SSnsQmZ\\\\\\\/rOKQRplb1jr0LyWnykxE3kUTm8QvGOTrQ9YPnPGIkit4DFjsHiswuj4sN\\\\\\\/MGFiy9j9qfG8WOrO8rNApEHThZSd\\\\\\\/ognZXKuAwS72+zqdBAFEy7if\\\\\\\/j+YiokkaqVsOTb7FdpEAqGuOefChznkG5mr1s+4oGz4CCaDZnYmmM\\\\\\\/G2XQ6yLV7bfOQk9OD+\\\\\\\/8p+bxSHFVpOm4ejKhYLalf\\\\\\\/fuMSiSitRvRe3eJsyBFAyVW+6Qrm4M7rqELKkoXL+ByBHcwJ\\\\\\\/Kta4HWLExUHmS\\\\\\\/11yi8EPxSgT4Np84KIqvFCHTxDAjMRsIgOtXKuFIkIULKF91fMPDTt0I2\\\\\\\/C3y36M\\\\\\\/FoLSGh7Dikna80qqBPpUQTm0+WAe\\\\\\\/RZ2ZpESwoXeCyytIrbj+e3dnl4T5Ps6ydKGCfdUosQLHaQLwZXqCQf8f2P2FOnd8U+pV471PKOF1HQsJpCh3frYUKjboWFt4ZQPsKmBDPjxMz6vn+pAhLfBMZR4P+AZ6R2i8yc6rvpp2VFq5X3bkJJhVGUMgx7IRx8SUHznq8VHUBb2\\\\\\\/Y9zf\\\\\\\/Y206Z3nmb8ZAc4xdLQBrFwzK2QbEhVDc23QfFgJQDqZm67YkRtn9Whc411bPWrMWkd7ri8q2j1cFEJVxBzMClk4HjyZuffjrglPY0JmvfGtifGJqEeu\\\\\\\/9YRDEl23P05rcq3RTzjiYVXZRjjAMOZ9snVnWD2Nxph0mDzXjhKIioGWpI+pZ1KC\\\\\\\/auwexavdqdD+mxw8Rz3uHj7h9wwwKpa2LccOIL81noYSZ7lL3EEqf0nt5aaMT1YiHn0S90yTe1QAVmY90Mx2XKPViRZCFXVqfLbrC9YVjJewQULN12jD3Vy5nJ4jTOor2n5qREcYIRjNwNm86zUmRL5HS3by6QMHaRRTEpRxPE2r2lhzV+60m2sOt8NRSZVTQZTG1Xzbw8Hq5EeSE8jvfLL6luNkqWhWdF8DeFSs51m8+fEY8XBHXBQVVkRV4ZVfq1SRLrzIQLlIBngdeOJNcusPcz8RaMSMDsZrfVvCcYu9bd+taoVlIwCMxifbvi5knMIan1itawWIbo7eKKvsM8y3xtkN\\\\\\\/PUZQm8v36wohM1SjkWGUoQhARedMEBVpEL3CvZ2RCIFFEMKKXkM1e7JCFBxk4o10MsWMUJRih0XEmWo1s54b3ww9y5lvvVL8KrTqbv4+9zyAiVVSosSD5DaoNAdMfrTnRHyF1dVBMQVjlYOm0mCs5L4bs6qkRbi4R5mt4gwc+OBmNev1bP6yE74XS+u0b4O8i2sY8Z88fZnPUlK3UuPcpJzeA\\\\\\\/e8INN+4X0Fz7XDUG59it3lCz9zuz5v\\\\\\\/t9iSpm1k4UZmPlT9NhXiomoc8tKPiM84uwGDxZBIl1XF+8w9rD3PM5UixUJmOdsOyy5jdrUcZ9CAZkpbMhj1wX\\\\\\\/qoVCeR3NfUVWiGkalL3zga2zurEQ2za3zxoEGGuA+3vg+3ymrwOFxPpGtRQxD+QE+C3wIfVYfZA7C1P6cjs7AQKoD7WXbKPH60TLsdXKfXbjU94eO9o\\\\\\\/UzJp0KD\\\\\\\/6MhSbxGB8tho2DVUBJD0U9d1So1PD3SsvJn+9JxJQQdfzW+3Pk9FuOBQDEP0r9t+xOyovxFrKVJljC5cGTpF2cT370v7K9kqJNTYdh7K9dtv4GjXszTZGUPZzCj7V\\\\\\\/HRFT\\\\\\\/kCCKTYtC0AJYba5vidmQxfgFwcghlxkyEEyl86C5XHtp01XaxT3ziEnpnZ7mqjkZD5YUODLUtk9f+0z1WCLNrKji+70valmUpi5pxCkyD\\\\\\\/GNIgWqNE4HxW\\\\\\\/rhdnlQ8h+bWlT25sgf0zYS\\\\\\\/lXGbRW5jdI8a\\\\\\\/ioLoQz1VovubIZ5wuv9mJRr3G3fLx5k\\\\\\\/BVe3ZpuNhQcJxhQQTXjbl4kRlyqqlf6QqXQPjBQMNRAX5XGHceG6wvvw6YYTfOYWvw4MK2lLHzsdqYoSFxCD4GeA95AIAMk7U7YxlTzCjBDz54kfvz\\\\\\\/fMlFi6DUBPliFR5O5D7yVrnHEcVUmKNLcQmqiVMNymcJLrT7v8\\\\\\\/F3e7\\\\\\\/KeunVA\\\\\\\/q5gTXx8wCglB6AdHwNM8reeKI+d1IpAE3C7IiIPEscNuymWfRmksrU+2Y6abjNPbR1VJ+syi4y2G6677r2vvVTizcK0\\\\\\\/1ez\\\\\\\/XUHllp4dIOptNC0WFgK6HwQk1ixdGAxlf5fpq5j3Njh\\\\\\\/ha5PUJTya7cC\\\\\\\/M7rPJsozzLrm4o5P1\\\\\\\/pac6dVTYnnUgYUrQN3vdrZWO9GprMt1bFPD+ypeTx00\\\\\\\/KOGGGrWlcYOsojkGt2gJlU2PcCzr1CvsYMczqQ+6\\\\\\\/jf8+PX6zOMbRod4Zu7xICUAhdBYsj1PIYVypP00dJNWpJ1caJivBJD9mu1gAyVYY6Lk0IvHktN8LUWymWqH4tOjaPTtu9P362ThGwsIHweJV++5T5FWY0yAEu4tpXCzAWvEAxox2odRhW1E62XL0JE4K1nODoudMdYeYVdwV3Z+rZqSUfQFP4v2M3kQsjAF3z6TfbETYBI\\\\\\\/iG3fFOKvwLq6aiYR6qZb1QgyFfql549SuWAxbyMte8jlDfdaP9AG3BS+\\\\\\\/dAsRuHgOeQzyA6I+BxUJCZcr9AtSKZb9EhdTWSksTfyrzmvES33i7yhYAkHyv+TdHgkOlwMYZgxJrwOYVHYY8Z+mX5Y8skf8mxg6WRncQzeeC2mCqyPgtaTQnWgo4rzVExrxyLBcvqAlSSTeUlSDBImbScgopH+Zy7MherXRGrfK0b2KMRWyGcNjMnxSw7Ph4xsoVHYgCsQ7L82jtOmIvX70W0ypxza3YPJ7SPHnZj5I5EcjisiuZ\\\\\\\/Mqrskfkl7vyef2RPaiNPMtm4RRyblH0qwQ3Z++ZMMC+n\\\\\\\/t8dOj4wTEeHiPGmIjIZjxhJigN1mZX4MHUutslmHEtmfL6e1goBhmYLDTHXvaStY\\\\\\\/abJSzqoDd0jEW9femjkB1lkF32EoI7mGfZR5bghNuVyyvBR8Tic8VB2DCag4OVlm9UkVHUBuiJR6+YGH\\\\\\\/RzJaxmC2XNPzitxFFeBwEi80DtFgVjfBZ6m6NB883SQAzy1K8skwOBNx4I2n0xNdlvmZuWTEjJjY2ffhyfMCNL6JnK\\\\\\\/QJGEcM0ExhBgK9ljbo9f3EJB11CWXMC83AWGioiktYvjXM728aTNS7iZIls2DLn4b3lcKMf1XQ9Tv61D4Lp94sRAJZfiA5rcJgHWF1b3GtM0Bi3lv3tVonVFn4VsUjHSnoDsgJf0qvxYEWIS+1j15o9BsRfj2fu\\\\\\\/W8i+x3sefXBRlsZHakWC3IsyWoQMc3LIY7e3ON7nIjpwtLVbsDyzvCHPLrgFvw\\\\\\\/Iq\\\\\\\/2EZ1vRUlF9iij4tD6xTgWh8I\\\\\\\/YSBa8nbGHTn9iRre7bwtOZfRYPIrH3P4uqhqU2EykO3365l5Q4dbJYLanTVOQghVCPV8IxeH6qy0MZpl2YNz9F2TtcTYV4\\\\\\\/irJ8YjdCxkZ4uYuPmBu5MYGzXFuqYRvslQOYiDucvptGFb+z\\\\\\\/qTdlcV4dqlt5bbLRecYkMz796wX\\\\\\\/yy2zWOioM0IxrWbXV6RrrA5anGxHjyCLyw4fmLPDGTaR4fLAqQZVuKzGM5eC2YhT4bu\\\\\\\/Hvm2ad8EMnmCvZTxwlijL1tFwf9BEmfiH9zvRQLUeSsoYDRjmsuTwE8zExBAR11RBRBzMpsVLtmNk4jCQ0CZ48hzwkIgpnHkCrm9\\\\\\\/+5\\\\\\\/sseapJRTNu67CFRfEyaHCuy9JudtwUelMWxCCAGjIiu8m7PWvczj2PqxAKFUp95mCd7W\\\\\\\/1IakBdgf1FkyLIuOo3AZZSIreDdz242lL0wmYMsli6L6TO5CfgCrDkFxvQRgUbekh1eE5orNxXocPQOfE5HMVvVa6n6AvlIwNvxuH25ZtNCfABhG9TBhxLxDAQBjLiiWZjdaMefOB4i5MnS8WuI5ilOeqzA5wmC2Ug7xwps7TkVkq\\\\\\\/6eyjhYuYHKHDo\\\\\\\/QIDRA+y0fP1CdpDzLERJ6w2pEbC5rtDqSH\\\\\\\/1UwnHVijB3yfHY+c9Tk9qUpxRCetX+CSiWi4vWReE+LkGJ0u7pQvroK9sDXWD6dqCbdMnM1XB\\\\\\\/ep+ebNrOiGAMPh+JjzhNYW\\\\\\\/\\\\\\\/1v7J0SiJvPPMwpOpdHfJVWyz4fcedrqPh5gH65rQ30kxXnfX\\\\\\\/jGm1oXZz5tkTK4DmXEpjOeJxcONE1YZT6M9jtNL0ShdekM+JHO6IRMnnMqq+MAbhhfiTMg9epAqDxVKjm35QTrb3l5e1\\\\\\\/TPwoQhBugzczdpTPeCpwe9WM2oif8578xk0ofYMnoafTHZ3gq4vbaaUihr3z5KdGCKwend7J6Yu0xJs3qEE50pFurOtAnX3SvwVYD0hHG50euBeQx7suNndp0x+misG0Licu6gLP3XYO6MzAVN1jfmyjaJlldGjCn5lkB4GchlGovoOMq3DpSgN4MPeCvDoClB1c\\\\\\\/KVb7o+Gch6LpGzDLBgL2GCPy2AKOSlZEwqLVk7d72cEBm+yaxjiLOVZeRjj5wF3BZLVFkw6MOEl4XiBDX\\\\\\\/Ls5451yP35oP6IeMefPlovYx4GbGro4mZYbAkeSoJWOCSSu1j2SMw3uVDiQ8YmOUG+g+anoJgxpKLYD1SNuCmwjPvtjvnUX4usj6YKM\\\\\\\/G46wHej7e6hL4kPNNdL\\\\\\\/vrSuVoAnsnVOiTrtvxvgVQKNd4CDLpIghRbV+3i08bJcURD3nVVAVX1Bt\\\\\\\/8l0sO2q83wHRNR3ZmxhQNuZGibEPNVvYTfY\\\\\\\/fF6boKS\\\\\\\/I2wjO\\\\\\\/t5yKN4feuIXXLMdczIXVrMYr2dx6UKZdyvFigSS6et8s2em2KuiyI52eq5hUW538I\\\\\\\/veYmnspWPeCWJl5UG1wNzuud1ceX9o6COY4bq+\\\\\\\/SNa5e1bFeEdqWpmRgcJIDB5wsdwuFgFDtXnsMFze9P5wveZzPvzf5SglCCYG4zWb32f8CC87tc2vqy1fRMBHLvi6q37Ag4pRYkt9\\\\\\\/8RmEDyjdQkg6cKh4IJ\\\\\\\/rKvtzJJ5ULmHt4st\\\\\\\/Ng\\\\\\\/mqhXAIlcmj+eG0WIP2whI3Zl+8zxbpmriBntMbqYwerIW\\\\\\\/dv16xbm8y8TkRMuoSvf7kUkKWxqI4z46XVHtjfZGWm1RSxwEYI36YBj9+pi\\\\\\\/5cY3FyqomI3d6LTdAHG7v3p1i00XaYRDcCe+7RinVOaBzmL2q+AcCZSAHpaQPj\\\\\\\/gWb5LvYh7W8FTgNaF5Mr7o8z7fRxGrse1kzhi07CRHWprIRrmUyMRUWkL9\\\\\\\/Y2Se2j2GDbAw2B0b9VIPT0nsKRgSboygLxchKpnd9NEWi31uvHGTd8k9Vq9PbOMNYqdjBI8gKlojL6ymv9PTFqr5x63AOdC1yaLX6is1IheMHtcDn6NQmAYO8MiKT0hy5FmQJg\\\\\\\/5xPIpB+jgTvhI\\\\\\\/W6oRgZLzEQf6MXU36ffhDLVvKb2liypMSDVyhwFzTSYaPLNcxQ95Ybki81RN5ch59yqL4YLGkXZjm1VBKD5disKHtJQRSqakQ3d9xitakqHNUWX+5HGWtXWaBm+7e8i5SoCACBkqkjw5AtVNAR3tlVPsEWfZDi6YgAmtb6nk678JjaCuiHZ2iPe9uoW8\\\\\\\/6Z8T2tlW+mNZUbursR5+11Q0yE2HoKBkfQx7xY3Xz9S9KvrXucReTDBF6o3SE6pY0MXG\\\\\\\/u5neOQFZ58LIq7PjL9WbQApfP2AkA66Ue1A+r0DikOeCijwp3+ni3PTieBWYK78LkFPKNkXa0y0uJxDSuUB0iVc7XLztq638w6Vhn8qwIDkLre\\\\\\\/yi3SoYaS+CRcrX1a4CPiCSl7bstc+eoC5YeI6lT5YQRk7yaQ4ti89I8goQ2srG1m9AbGuN6HCyNWt3cgmitKgHjzN2k\\\\\\\/rYRWv3a5zgkWRUs8Ks+lp2XY0m3kk5kPVAymv5OXbcv5nq3hz1oCal7RJnCwVS8T\\\\\\\/pqbW9skw5fN+XT+NNBPt3bVicwB2XFz0pct8xMaoty1\\\\\\\/NUctLUf7MP9CT7FiHRjsUmjRwiLBtzZSsgagXuvsmOKepFPn3KXQnUWRxjmiBMBzKdDCOCJS\\\\\\\/WbFRTI\\\\\\\/4nia8UEL1fm8qGK3J43NOdmCDaY\\\\\\\/h+6PA743uLn+NLlNBPj9q83dkCXTEXGhfgbax7uoe+hihLfB1pXiWESBgAfiCMuHBjd613ErUYo8HAna4TU8vIf1za8EJ5ZCuC+Hiu1Msd+osSSXL2LcSTy1PX53gLIARI3MbfbHzI4gqSvwb4JsIbgCwhehlFmihPsgx5BKeUDc4ThIyaagOGLGj6lgHbstTTmhIViMToA1UBs9avY8sOPUQf3XZHaaynRfATrUAawcpYnFegpa7ejrTOanFc9\\\\\\\/tefbwGPxk+U+GHKI4Z51CbD1w2oCfAjpFF+9TDNUpF1eOTTnu44Bqo+n0ViNFf+uNF9EwbJNPmFibSP\\\\\\\/I\\\\\\\/6BuaJBBiWVRWwgzyzB9dAuhMYQutxMLrYUNrL6+zbrbbl85XNN2g4KZZoW++z\\\\\\\/neCI8BBuZPTyobZT1zlUaZzQrtXScKEtT3cDOuPfrhPxzlciEjKy6ChpLCek0NSnbPCCetQGU1bjwvEP095LfuZWew831PGnJbSui+NROePkDfTD1sfIXF4WS8+YWecM0WRF9jrv+++p54phfkmRbb\\\\\\\/4iqoMFF22CjNA+7WXb4QhsMT8cM4jFpOm2OiM0EgCKZyOHcCPyH68Hy4XOrGQBmNWeLKIUVMu0iTj85XOESqULJd3bFSkpCzDqBVERXbHhcSinOx9IQFNk0VeLWTW2zjWi5octtvVKMHVXN6cMRUUcLNyYjtCclHy+nVY0fC2\\\\\\\/kgklU00OfrjQKhVbDUakYZSXgCa+lQOsS\\\\\\\/WIA5zFlSf4g5CPIssqINVFzGQryx67Bow4doswCItl+G9X6VnC1BhUeAicZWfloMA3Dkl0Su3RwzAASpo2wDcp4vwGv6sjLtyT89xP1uKuv5helaz2DuIYqF0\\\\\\\/zky+ocIuyY9metA6UPWxpdhZaB80ctxZwwja6k3MMwRlSwCDxYWq1LpT68+3QaC+49JM0L7o9l0L0vFDqR9NhFg62EfTMRyIToTHQw4QC2LAOnrV0eiL8FEJTi0mf1iOpD4OL76BcmlmC1U0zdJJLPNB8JbmVrpn2xK9p1mKAbT\\\\\\\/sZc6of8T4HopbLGsWrC5mISTPeI1LL7ZoeA0jzNTVdB5k4Zpodmloa7QyFk8kx8mhh8GxDxHHoVqzjJhRt67CFfqCCOnyfah5JO5qUVr3G\\\\\\\/5wBmf5wJd7+EnP1h5ghh74JRqXW5aZ1j2DA9y8n2Dtb+B5\\\\\\\/+cAeulQ+F0fAdUuV+l\\\\\\\/dJ5sKgZpnI5vwWyYZJR8hBEbKIuq+dEBcxjsj+JzX9lWSbhcmx2PNF1yW2SKgsCyTQrxreswY6PKgoXtCI\\\\\\\/eyMJAwhmDuMGv9TgmTlm3ZAw\\\\\\\/FAaWMrrnu4Ap7tsNEEEfHzf+Vto\\\\\\\/UEENc7hczUAkkJSLEBfOEz5llvPQ4h2jtb7NUMSM\\\\\\\/o7JjI0CMtsAuksFo0XoPalVJIGd4AYhxo2OQCVDMyJjX7qk3IXOL2jSdMSGERKhY1gBxPUMwV8rInA9InsIZ9ylm2fdW4co\\\\\\\/hCV\\\\\\\/Z1Wk28N\\\\\\\/IfU5f9UReH2itn+kYiWEACQG2AcvJWJEvfzhWfayp+tkZ2aBX\\\\\\\/lEOJc9jaTM0V\\\\\\\/celhtJOhLAR48cYGlz\\\\\\\/j6m+K8skDbp+PUczwfP3O4CVNLjFMKUWI3r65JTvc4rRrUEF+MkjC5xg\\\\\\\/+1C6mNmZXSxGXJnOfV8yqMFkDDekt7yKL3grXU9Z43nMBB\\\\\\\/KCKRad6rsc8YtAlXR6pU0IBLdiwLLr5PlYoQ2B5xICQH1xVebEmMLQSywHrLkx4VSKKAvd\\\\\\\/KBBpp3XSmSOOEVIuuMeQ8vLBjgqLKeQu7uRXRCzq41EWK7Bkk79q639GW1+lfZ0VyxZeNpFYhyI6SRrPT9zF6Bw7cYT\\\\\\\/MV8Uht5i8Pgg+oorhsF0e7MZDGQ9d95jyOEHFYo7RzSpdUc1weoH\\\\\\\/P6RjNprPJtcYI4Ms2zWnvYESjx8pctbmfwtwD1nk09X51WKSP0VxgV1wMYlNgTbNEKexQHMyE7VF58g0jCghqBK6ctqjAZk1IfETj1ZKSBRQswrP0QeO3tFpdMBiR\\\\\\\/WRxG4k1RG8bEgLsdKa\\\\\\\/Rs3NyWXRLpnSZtjxg7Ry8WYmKgpE0+H\\\\\\\/nOcksB1lQsck3RIEuag8EJWvHvANuWbhSQxL7wLSRhXWYMYTObXhn2ewb0Mw7Ubu0FJJkvUjnTTbFaBY9+wnViP0W7xXctgNDi0+zULXrSOTgtkr++mbllkipBpGlgzwv6ld9FJfb45yB2wXPJ7iCokIlLFvrswG6Iqnl6bwP7mkbKhCKWBAMwcd5bm0wZ+oF4lP+rq7n2nGCIv2+v7rfH+b\\\\\\\/fq31IIEWto7B7ZVlsjH+dpRsmDvv5JG1uaBRJXgsrHk5szUDYiZMzPa67kHAtrJrS5R5nly0hoMY9KWkpJSIQcQHAUsKm\\\\\\\/zmjwuLh5t9YtA0fmnkrErGRaqi4Xdd06FNWwFOvbQf04z6qJ48xRJh\\\\\\\/+lqQorAgjBiKTT4Pgbm72+w5UAwXXfXxqPUbVTqVG2JHPKw8u1vZG0pNR9DAJJ0YIwGZCjvVxSHpOAteYq9y6UCOdN5OU2mNnS4WPlSlR\\\\\\\/LCseAZjTsqnzU4x7aZIrEPD\\\\\\\/Atdf2xui74E7HpA+m4ytAVRqzkK7uvQAdJqbTRqrYQ3WZ6Z6qR8pLiFycaBak1aj+HGdqsSTKWYIq9UbT22aPx0u46y20\\\\\\\/KTv9nVWP4XsfYeE40Hczw16wqiTzPDOMwy0KdodSHb9FstBY3cmu\\\\\\\/1n7XIj9\\\\\\\/hnXSX0gCYrMn0YtkSyeuJ82SqEZ6tEzMS6iBY1xA0kIAPO37jyur8XTqcnZ4BAkHNnzQSrBcs3SnLLls1M1k+srmCW97nnvs6t5c\\\\\\\/jQlisgn8N3tKubwdiPlLVJtofsRTAAAmVQ1\\\\\\\/n4nVAEA1Nd7cFojT\\\\\\\/TYbGPqNzYKUsr8QhYTKhLLq1kDtLmGBlJgdM1PhJt1DFHbk+wJaeM6BQ\\\\\\\/zj\\\\\\\/Of3g6SGXMiKeQaN\\\\\\\/vRutMTMAxI4DeMw11q3XbJAWKzoJmevZ80lXt16nlSBHxS7NaMP5xHLtktsAEKSEb3Obk28omsk1c5K45AQsw92c970BYi+\\\\\\\/mWGwhCQU0XCk93Jn7dAZUVOUw4sGTy9kYjcVszxpFl+iEc9XN8rJ9j+99RpA9UaXlxa+6\\\\\\\/FmAHJi9qQwj+jR3w9P\\\\\\\/WpFRjNpeirqeJzbRd+ND++JaUWDDEKBJYHwQToDPc1mU0p4OOt47FElBt0rb6j0UYFiDcvL2kK8suCMz7tDpSC9HVaWtVXCezox9D7B9xZpfsZHSOMI5mQRoGcQFDPd+LvCGAIZl5o1MI6tOMlnoFlK7cXzNaQE8K1JcVQW3UVWX3YUQ2GUUpEfqZO1VhZrtLNt84odHvEz3x6PMjNMif3qfwqpYUPYelvhXVNlpcHQA1MkggRwST\\\\\\\/\\\\\\\/J0Up2+D\\\\\\\/km\\\\\\\/E4KHfHdMkIfm7hb0Dkgx09YkO392c9\\\\\\\/Vj5cHO5rykpxpXZBBWVEgJsJ2iDkx2vo63Idr0Lh36lllkupqBLs4jEtoy2L2bYK318W7xWqqqcsyeN7fBJB5P5ZE2Vcl5q8Pr3V6dhwGj+ywE9nDNgcf6TW1ej8MI93X7jc2Ky1MVyCd73fayjrwQYwUiFsJsz1tHLxaYP8dcBmaJlkUJvkvmEVTIjsnbffKbIC8Ve7ikyf9Cjj+lOQC9XDMtmBMQqjn+er\\\\\\\/TNXlmDc\\\\\\\/HEL+ei1T0aVyqoI4b6YBS4n3YECC2P2Iz\\\\\\\/Mr5OhkOhGEy0Yw7taDAR+03DcoVO5mNk8MN55sLdt9w8pKorA\\\\\\\/99z4t1Ix8a5bZhsIoinCQdfr0bRtutlqGSmbICk8BhjILreG+7PxOoXSI+L1KsxBXKdjLtmhgalqgqwuJwvxZV9VlvFMSPlNwj4zzoJG7FbJfjLxcKmYuMb3FbDkm5YE3s0zQQ90o7qsHV9zGxlT2PmIspKcyZ+J1KX3zi7jZ\\\\\\\/FLsgZO\\\\\\\/2be6bXnf\\\\\\\/RF51cCT4nrreVaAzrwckFD4OT8qmQtyqE+lMAJGig\\\\\\\/Ax9tuRoHQz88WIuJCN7YOg4\\\\\\\/R0A\\\\\\\/f8KUVX\\\\\\\/xSsIbQG+8eNxTcLQUTxRRbV1AjLAg0plm21ofiU+RIBwvL3IRU8WV2KAyCRo8x4YIfZLkiY4OnoAzpY9su+tYwwNBpgLGLkF1KdLkuuQsMutr+e31mWyB2L1ww4pJFiencwss9yVeJEiYyJbWZ2DqPmHTLtU2cGDyW9XwPdQSwcUFEv97EAhF1N1b5FqaoVXZfxCh8KV8gU+mySbzbFpLUXxzsXd8ayYlAVPCn0w1mRe92DCp3T6sJ5twm6loDvzX398+HIrIEthMpEA5noxGp1gvrdUprPGLGdMCeTR0T0hMefA017JxZ+Zq3YEeAkYTBL+kja1gWzW+IWUAyN58Gn2T3bfrksxDaMjVnvk0bgbLwOoV6wo+l69hmu\\\\\\\/SXC+nXhscxno7GBfxztPPKL7kSOz6ekKCs3oPq7VCD20hU3\\\\\\\/gA27I9LCw0oNEzvItMc7j+J9Lrow4WHsYNlKFAlz2+\\\\\\\/myFBLoWOIbk5OHBU\\\\\\\/wialFHCYOcHPX3K+QvBTXK+vmJUvjUO7XTLr+henVOu60iDGX9zQdh3xD8x8+AjGbu26nSRy2+\\\\\\\/RlsaOZ5rvk46TvVw21kBNvVluprOpWz8CUMKthl5saJwtpnhRApznkoGVSaI4wZSR79vjecbJ1Tk4hWV2CjyP3zPRZ+X2OOXdofhDTSdgeppy+vdNhMX3+WUrnGruPGkXPGMHdArmPHmCHCHA562Tousey2WhvAQpBxekMgqMiNiXSqtAUSKDjg+MwmgoDLpvIHrT\\\\\\\/PQUuGIvT\\\\\\\/bT9ma75Y8Qi2lml6acXF9j+G7EbpE32t2uUWK6OwwwAD74PJLKMf6UFKQXKDfllkTlCGCE2UWrc4nZcXmfHq0R4tHNcpW22NDddaGhpK3irY6aoj9eF0yNMUTAbdi2A98CuybeO+MknCFdpHQetTA7COKwk4v2FrRpuO65JydAaVagTEU2Z+91E2qaxf7ugFxvbINEHteYUQ8GzWrseCAcHY2wFihRuwoYm60GEIPD1IH\\\\\\\/RHqlgGIP+0NNa8dmeleilzIHdxOzeaDsOVvtzeMJlHnjCwDyGWVBt0gwGeQc04vCASS1QnIxtRKoBy5KmHogFvMYjyX0uheaONwW1LjL6FHpNPqGEhFK0+1aP1eUSvPlrOzRbdHMMikUGLPELchKANzJSEBZJieIifgAqBtp1bAHWarN5GoANFiarfkR0cQhxGZAyeA+kZIiokoJ96QpB88gNrZzMsR\\\\\\\/X42nDRyKG99k\\\\\\\/fY6Izk+tBkmTcCp1bXuBhlaa7WlYnULcSYHWcyvSH3kEH\\\\\\\/AGfyPmvajtbixSbVdmPwIxBnzEkUegYd1yiHwMoK6UkaIPe8GoMa1RPYtnjgmsrbvqhoJ8NRpCVUiN2z90rwtH8Bhf0MIHOjTAYqLKjVuoHclxmmWyIIIIpjfMqDnGghPghRv8nW8hfThirIbyQAiHsbS2Q1nJrdVfbVtEh\\\\\\\/xzZSuBnaXtRIUjUI+bgEAhh8OahUZ00mK8feHaBoJX0IhyeIl8b5m0PI4kQ9DXVwAD+2pnj820qvvPPwYXRiV2hCKxX+dexunir8M6uhnuxSJ\\\\\\\/2\\\\\\\/FCGNxRI1FouWytGBQsfOO0h8dE\\\\\\\/hEg3MA7Tmoh+wYdgV21GfaJhK2tcbiWPA9al4qzTQT7nWP95svJek+N8ICQTWRh5N\\\\\\\/bY0xTSYLLwGJqczMCHh5qj9PvhGNgehg28B60JOZZ3HFHskdlw4chPrSVcI\\\\\\\/AIpLxf4G5KtKu2wUZ6sPVE3Muy2wKJByOFX0iTWwjTqPe\\\\\\\/BDPpLv+M\\\\\\\/TkeA20N5cqO\\\\\\\/d61Y5+ubQju7hIKRiS7Hasm0\\\\\\\/XYNElT0ZKqL+YSbkiRikj1ob2kY65taC0glDEBuAKSvty1U6avmQYQqvsxruLtHIddMoxtSpwl31HUHir0lv11OYkvvxKmuCooaeXmyE2HoMOAlEISmJNQ6kWD9DsdDYDCVcJ77aCOq0sx4KE5iqgvb5erD9GvQfEVGFHUByCCYyTMXv3LlVkQL0TabKVW1UMU7Db8cRNvGBBELK\\\\\\\/DUiunwM3KRl302eSNOmyaPUo35pizr\\\\\\\/qsO\\\\\\\/UlzFwS3wWCUnGcUDQ7KiuE1xk2mywqCuL5gmyTZno6jLPn2VeEA5ZlvkDwbe3tWnLVf92920ogU77Vzqdt3hNr82YaewkXu+rywI\\\\\\\/II4e912GJULv8lvZMpi6ldBXvJbqrfutl4pFRQQXspWEHr5pH\\\\\\\/UBCNx8n6CRgiOIlzCHx6bqHYjPa6Mbiw82G2+g1J5rSQxIyINT1q2S2eDilIv0p8C21Aq19sYanbX78cd4AD9eTU3IP9g06eaJqOa1jLWMUyOMbwasx32Ed8hdgWHrDoG7ToJW6w3AyAEtq\\\\\\\/IQCym0qRyHDaOz8u0LRtFqdUtu+syLvhsOtpvYUZfuvholWo8pL2w+DmKwXMVtSNhkygZ+0FkeFF6KdsvOD6y9LwqJlJD\\\\\\\/gVJ6Aus4rBGqVyuKNOd6xpVOt96baFtv1W5gmLndvlHWPyxdnb2UlAi4W\\\\\\\/1rDvHKNtuuPPfFWWMozjPEU24\\\\\\\/1Pe2XsvX4GmJczqSbYdyOHJEWeDgZOPNukhc+91Mp0cgUqbD5VdRgjPi2Ga5GKkHFlZfPb1CEizWpvlm2RzCqkDH\\\\\\\/fNbvAmOgCIOtQlnZFPKQw5e1QHQry+lgsV5RkuTfoXC1MULPGOZp2mDUvihpX\\\\\\\/0HXYMIMXuiSQOcBW47jP4nAf6kz86TXADqoBNULRtjStzTTVL5P4joe7glzPwxVOqPk3prKZyDF77D7KdrciI+7e+GqvHQWZkHXh6c0eE0bD6naP3A6lMcEYKvoEDfYSeo1cPIotJSTDmq3n6eH6SwTlFpPMF3DAXmTXwWeiFMUQ6aoHSezEQ6kol60IzcRyAslUy\\\\\\\/d75cO38a6z\\\\\\\/v+mVbFwOBBm3Iqc\\\\\\\/q7fD6okAN02atS7DLyj5cDiWg+W8r5KK2gXDxhnJObFjgOoFx9E6aFpVHL51l5P8zq2K7qILamLNjx9dv1XDs+X+1aAUer5mw3LdY+xmhAhQqRJ\\\\\\\/onjPhBCiSNnceuKPBwZrdH7ggE8CbGIp3ZVj7SkkH+\\\\\\\/qHRGvh8YSvBrgyVw+amfrJSOx+8FGbJZ5Sf5ppxmYyEBZJ8ps5v+brEQHsldw1iVmdUPONBKFLrPHet2q6f326q+DT3ylpovT6MKPSkAdMs2PZt6I8axthWN\\\\\\\/MWxVMjh8jM2LqS+cZ3mGv1khaMM5hO0s2rkRoYVOOnZ9jEN3\\\\\\\/FXRykGwT4WxU5faUwHCz97Ow\\\\\\\/YG7A3Xs077\\\\\\\/IQ\\\\\\\/BMAemupXXGpbT+ysStY0BRt8yfhILGXcT01B6eScUFl6LO9v6LDtgSd7Vpk5hcXg7XAXN9B9HuqJ7CPEC7hT0KVLaMAh8P1BmWeRJOFwAHn7Kx4I3XVS3rimluA3tAYahXu3dA73ORKW9ecqZNPIEuhEvNhVYf9+st9cBtz9pnOdMi9V6Wxtrme+SmUIdC5q+7Lojnb\\\\\\\/kNAeHZBv0i4QZ5tosFHbvrx\\\\\\\/+h4IlvipHhuVmACCagsvHSuKy60\\\\\\\/lK8X93yC2NUK9Y0+ykdHlUGkkatVagKiSmBxWFwr0nGdwgFY7mG+NY384Nuibyx3EXAamxzRnn3EHzqPNGCXNuZ+RwphCYs3GBfcSUfApdAupbDZxJeQgD6WFAGwT+3bWZalBlPKqu+1hcEiMhuUlg2ZPGwwgqTXH34Yyw4NhgsDnqn7i7lsAHN5IXQkMLUbMPnnE1q4NdxpPWL7Slwk\\\\\\\/TvqBGNdpy5IDefyZ+mlGy\\\\\\\/HEKiubea4Laor0tUOzfoxlsPzVAfUBOjlL+O\\\\\\\/vfKNSrpeYCLoBalJdXtFchuQQjbadZ7e8w3XV77csDx6K2NeHga7g2dloAIIXZ4hBFFfW4rsmNA5Z259AvMaDjXZsm6d36uj+I22os851uic7vfT9mRV23mySJg1GZiOWsBf0eZMS1Nuu\\\\\\\/+KYmXmyP7yzU1BIrFGDuEy7+c0FD86O+lLn03y+4TEVU9ctEUigt\\\\\\\/vR3Sx8aTDAP\\\\\\\/YtEsQhUUMou8kWq+JrtzzWccCbISMZNb\\\\\\\/h8DEQOVRkOnM3i7eR6E7rSj5ELjrOX1HO9LMVfy3WLf+GzH7FWk9yHhO7JS9DZm2e6UEkIATLh7\\\\\\\/IYsdU27hmuIUMekAZs\\\\\\\/hkKBpW4kr2QeTseVjciZqojSPex5d1sU9d+c0rIxrLYdd0B1+2HZHSEs4fMNFvrWC181oo2f7ZDa4nAy7d7LocgBpiPYkUMoUGymDtfhOhyO43xO+tSa2MADkh991lQCA1RFw3IFkgqonwLM1v6gx3uB5N\\\\\\\/2jP0\\\\\\\/0rebXYbxY0h4EhdWe7YcqDwHPB5KR3AFsw1wOxGi4vQwa8U0ZElKAiFYJ9Dz0HtvtcrrKTpm9Q2eBDRVVPdoOLiqai0JjlVaxvNZ540lpUi0hJ8zK9bnk1Npm79ubtwXPhUKAxEkpSxfL1opW1qRWh0x6o\\\\\\\/4GZ4tUW4RfOZGoWWUmEtklFt7fp+WCby4lUo1nj+xFic5cEw+7rC3gifce\\\\\\\/NdVM68RJjOgo3NSJJu2tVOHVCY3G1Q93b5UMDGiPVFgK+jv3ufXiN\\\\\\\/lIc8ke7U4c4Pqq4RgkZByR\\\\\\\/ZIIUg8TmDWD5jyongvv74yNXNzQ72yGMAP2\\\\\\\/RJLeZ1oCHtfn7x8LEqYY+aQYzkHKDAPSvxNH+ux44oxyHSkZd+IEC6U6sWtisoWnlQ\\\\\\\/Xd+o6kyqFza4dBnCNtMxtwip0P1eZOcBZHmIAmL3xwjTsqr1EErkdNqULZXT54LSCuE1MeifeRhFn7GAXB9XDA1duDblPfbe2fAVbetD1mMVsWtKn35slsMMOlknw\\\\\\\/TECaUBWB8p0hzJ0r\\\\\\\/Fu0y5nIM0e3BSG6eo+2CmAuifTeG61VNyHpfUoSYuz3tdT1LmjU\\\\\\\/tbGlUDM7ej0GyFazX6QzryrU564ir7r5E6i9ntYJVBGQWEMMv4JDXo4dlXLNgpnomIt2n1Js2qiW34rOqUDp8hdxi1059WHUzpmF6qFylF9v8kAmiVnB1gnaFi4UzraqmthOckA1khj33ZQmsaIzzaesI6Ocwn1nNSUZDGSM6x52pYYU1E0H7M+pgVQbahWgd6VApJqZG4CPaD\\\\\\\/QnKrYgNqLpTvS1YlnxQ5MjBlhgNiuhSK9DEQAL9x\\\\\\\/crqfYgqhuq7oHRAlXgP9nKbEWS1setVMq8oWFau\\\\\\\/z4M7\\\\\\\/3KHflbVfZ0zcMdUOeuoYccllSwFIzSYTJHj6IigPWblOhKSJYgz+91E9ACr8VQMLq9sP9UhPyFcU2BBkdWP0WIYPnKZuFpZippkcDnm8ZiIvMNy4SqzBN6RANA0nbPuMQjsVuhrxfwQoVbiG1ZZG3MYZxicYA0fcZgpq7WNwqZfC+1fwmtrNuH9U1AudL4B+M0F9iADqrEq3BOzs9JNPJlGgWLv3SmZw3WHo3CYU+FRbS4qSo9XvSTCV0k2K09jmIdfEDwXIAxK0azC7huUdBH+NK8q+\\\\\\\/VrzKQnBxa8wUSxkKqN7PPTiUfAhg+oijktNHGaYq0grgvaVHLqoPBA1uiAjzO1QmhRtug5UYw5CgX0\\\\\\\/85nOxvqoqFSEZwCOkpfTP1i4hnbjcj\\\\\\\/OmL54mAdCCTIAjRWIW7yEQvjeP306eWTuSzn6hQ9kq\\\\\\\/4ZyXtfe20gPvJwDMwKwlazhHDasF2LFkgdeZYNBgNRheS778CFW8dduJTcF+gMRin6rvQ79cIEZBEu5yWcDMp\\\\\\\/9Tx4n1vxupJ+qyW\\\\\\\/FL9Q+\\\\\\\/zTnnCGoAA7+D+uM1npsrq2N6uw6AU5usrBjIJiiG869I6tY0Er8ajVWDcZYsTb5h1e0YH6PmI79xY0b9\\\\\\\/gVKsh7sA5qhDG+cYJ8fEy2aRH0kv7zzQ\\\\\\\/To5heinzP7EE67jghD\\\\\\\/Z8ZOsyINSdmuNCd1AkIj8UOo08qB1vlgxxZSAY\\\\\\\/F0YYtZpkJbPi\\\\\\\/fk3vdESqAb2VOtIh0zCMKPB57cUTiqCT52O2Cc3aIXUi1p9eH\\\\\\\/3MvTImujrJxqj\\\\\\\/5aUkhQMYTRvUKugKbbv+KMCsMZ5MrpdvxpZ3hw4S0ggLJvAdou5O3kKhxfvhGCugQl65vJR8ZmY8QAx1n78wjZPcRI6cu\\\\\\\/4MrUcMj3ybqzMJgZzs0lOaNeKuTQxdcQJ\\\\\\\/b+\\\\\\\/ahx9WCujudEcwN9aMVbODzInSI00Yz3C26yxUNamBZAquPngEPuIu3IhYnS6suPaqhXKezvbUgxaL+QAdbM4W9mkeSx\\\\\\\/LmA+zKaQwQyltlxIky4uXOePHb0482btfPJy\\\\\\\/rfLfYHa5h0fsTHBEs6hJjOymxMZYwk7M1+c4QAYDbPjZME0BNMMmzkCrF4Qi1tbC2oUi7bSidVGuU+N4qNKZ0ccobdX40OuTfm71sYabzhdYSxPH6+KAftzuWb6H4kn\\\\\\\/liZtLrnv+0NL+1Gq8WZ1TuCm6EE6q8b7J1jL0acHiyD0O0ytPevROI1QRegrQPQEj0zILF0iWNPw3uw9uilvyY3UgO1xHI108G5Tgvvl5nqTwtvAURCO5+ub8HE+kNwSp7IN5AAxFCGCWxcmwwtz5HbB+lUO0gQcBn+DsK8JYpxJRuvrJwWvUUC9+r1fqZyNKbdYxDOfTEfBVXkqHrq65lBFVjpc10Db2wfl\\\\\\\/fD2Sl\\\\\\\/0RY4FM7K2MJ8wTsw9NXehakn8awU3G6g4zEYrvXfOU3NKKhRoKJ7QxnHdFcc0h1gdLQz+uEAvndkmN7PXLLcFLfdejT1rBnVOa2xP9WPNjD8CPwl+Y716bTW2UtH1IzSUEEXeiss0TekT\\\\\\\/cEkZB6Za9nMM7K+vsWGTQehc9y5hteA0upE+vfjLbRST6BRYmo6BxBfYRcKCbD0nmS4prb\\\\\\\/xXa6SON8mRC6UbgpglQ+OpdKE0cD9HZpC72SpLrPEwX95wNVsHiL6FliNf2bujFdSS1fM7F7i9IpLHEwyNvqkVrx4c1XYMARjdHPhQdMQJ6gk7ZjFY2CSkePKIe+TawK9KDfRTbG3avxeh9ZTewgjyRuoPZYIG4El5ZolYztaP0+OhEohLTndtN6Yd9i6vAm\\\\\\\/+iGXgOqKJUHd7kBk0nLkJY2E+ianq1EjOKWe97mNoYctwfRYgPTangLL+eIBDDjuq4f4palWwQkPQz+JN6S\\\\\\\/kDthbQde\\\\\\\/puDVkuCXgT+tj1lFoNqZpem5k\\\\\\\/0LfyhuozWbuPdu9XmAkp1DZLHIsoE39D7pztbEUtXnjcYRbSh9\\\\\\\/gNmILSxzsEFJQxO3H6O2uJZp94WGVFqbVuPJmA9jjTUgPke48Mjp+J196b0XOXkz61bJxqQ9KAJtN0Oo7eVhPpCDewGla8i57m7a6sd\\\\\\\/kFFmtn+BBitXYrZNi0GP2jdC4ahVLx1oxJBM\\\\\\\/Y8t+ISSaD7KD8yWDXaYMON+Gj+5rtmZHhQyOFghlv4Bl5+MI9YpMcM24SDL3Q9XsJT0KdTLjguPTupLfFWrM2d\\\\\\\/zpbFzZGWGJCRwTXl6u9sewHhudd5avlUL5\\\\\\\/TX9dKGZf3HDDipsKhfU9t8Uj0bn616FosPmjWFJV2KX403o7RhMYarHY0BEb7\\\\\\\/w36NexkLlYyh\\\\\\\/nv5gqcvtjdTVciHJeyLT0hSJF13SymLN1VgwneltuS0kPp3EBNs8WsHUq7YG7u1g2gSr\\\\\\\/8EUO\\\\\\\/Pf\\\\\\\/VLYlM+GZ1fpR5nsqoHvFhn7ihl+3D8puvnQ\\\\\\\/77pFulYbyLvwW+6rNt1fPqtMXJGKqqt8JLAXBWewhGOj8s9JBU+bz+ydOpPQ3BQX9LUsCWER6s8abL+NziEsXMmpjs3z5XKlnf2F32ByVFkSHd+sme7F129U3uRSJn+L6Euj1hRl5DrJNMDyWaFukTzWsIvRl23mCrHuR\\\\\\\/ORYqAhOgA1nq9Rr\\\\\\\/dChc927ial8AU2wJ8waXcBjbPsh4QGqEhvftjMOZJhPtPVaPuqlGUxq4DarXGWwjNneT5n2HjlcqTKFDAWkSpDy2+u4yvCHK6D5xCjKvaNXrfBP\\\\\\\/zUfX0J5UQPXh3mgg4YihU7\\\\\\\/VTlG1xelKws0l6w52pJg3yuePWVY1eCIqc+gYVhwgNeStAuZsfVPzplwEKDmU3wMMDkGbjhnTNiuKjfTYCXBhTdTZ6OBzk\\\\\\\/KMcIDlwPSReme80IUEERhvq2tpLeN\\\\\\\/DQmwJ0vOqapdxn5pcjia3wImHNiIbIejielJ+BJGSxKrqjetk5YAokQmoY6VIV4chlPn4UFeWMOICu5thVDjedFTmGKfrzoC4SRZISV8XsTjwmxKAY+g9LRSYwQnAnweQN9PC0+3Lm+hY++VHKz5GaItKj3JVdeiSplumR0EsxSWBXmFpCLZUHwB\\\\\\\/jWgwBXfPtJEeUY83LtWkMPb277i5kt6t\\\\\\\/YVNrmvwY5vPmsZPygyongENhEW6UCSWMjjex7rqzzWno6N2aI\\\\\\\/6r9vjesvaEIfN\\\\\\\/vMi+bFf8NCdu8z22SzroDZPVxWpmCfHIUqIoq3gOCpTi9y4WegpwG8MLhtdOq9MXDmzow+bGMHJGu9cf1LIXoV0+pRhl\\\\\\\/n0p5o177vt+pXbFyA7hwiV2mo6bPunS8ZVvD5VsNFESs1eL1ShiThyyALl9PvU3Mp7mUOEWv4igXw8UoArcJ1j27+sxA0K6hozYfTNML2mqcLKKcbbF8tc9HIZBGj1RBEyeWRoIHAqfopdSTFn0Jpz92c0qEq0OHXQHM82RkjdRByGnjSKAwGLeDBMlG5e5u\\\\\\\/5\\\\\\\/XyLPGCXcjtbsEyqYsgZHHUM7Wc6\\\\\\\/V7MUO1DnNa5GBK5NFUJFdkV\\\\\\\/k3MAARIginq9reBpHr4+7y2T0GsPtFJkTRrh6JgMudg8o\\\\\\\/376r5NvfMJej8Xz37fwAx2EKBnCUWMvEh3so7MvgQuezePOp634GH\\\\\\\/4BoOv3IE6dZz7wX6ZOmjeKHzJl0b8yXLOxcgxmF+iDSdV2GQWb9QKjXMZIsNg81C5VWCNe5K1gHeGW5sNYNU04mn4D3InNUt2kuQI0fRhw6qO9o3sjXTlrtcMw4NAtVJ5RiagfU87tXxHuqDjCvavfk74NH0B3CF+uUyVIguvcWUdxblD50OCzfc9JidGS\\\\\\\/+dTJlS65Q1xu\\\\\\\/v77aVUmG2P4JgsxCJpC82Zgw2NVPu2gSz8uP0WL1vMlr7PQZkUB4GM5lb8QXc49L1beq\\\\\\\/ZZKmGi1XJ7qXKs8+G3LO5HAV0P4KXmmAAEss8tRDFpOIhuGO\\\\\\\/lEcEEB9OEbsLXk6RMHYW5Yv7cyOx09KAYCpsjDhV7aCxXB7yNHwcSR2E9c8uK7HhgD9bDhe\\\\\\\/wLJfwrEJkXfuIuVTi9VZt1qEZshEOZPqgQYyIys2XdrWVzYW1Pc0nA2LRV\\\\\\\/2L+Z0du\\\\\\\/hiZYSsgfjJfHz1KrsrhbIxB0Ew+LncGdYW789tz8lbNpxCQ8E0DDmIzBE\\\\\\\/jMrfVJSApTZVlVRWsLIPt7TpOnNR7eLX0kuVpSae5mZFAh1ndxQtm3Q\\\\\\\/ip8P6PLsoP+4HJbu4okdzG3LPsBnxg9NCJrWMogfbIKHhg73Ors8NyMNe1cgZV4kbRczpedFtIpIOePbeqsXcUJsk5Z\\\\\\\/mtErSxQjoknsFkyaCw2QxpBtg7Kg1gMuL+nuoWCJAYWc0Y2dRHo7xD\\\\\\\/jYrdqT+PIQrudR4G54fpPpIT4ZHOiXlXA912f5nslAalQdwhoTyDzUoNb1vWTTntfQbp570QB8rJmgk1vgW18BTG0T7WK\\\\\\\/+XkjFuNnJnhn8I3S+i6f48WA1Vp2Xqn2NLfVDcWKDIektTSwklklGqsjShBhMlQn2OhkCof5HvfAxrl+5AF4L2dIH5sLuxnxve2bxL+\\\\\\\/9SyPU\\\\\\\/yTP1aciEheZade+6hwFVpD8Fz2J7Kp8mbotw95Y6ZXhalUHcodeEAhaskyI3Af2u2AFevxH6m4iuAhKxmssmL3lnzpGqszQeOZzqEtNhv\\\\\\\/0cdVKeaIITYES9EjFQn+Il39\\\\\\\/6lPOArwEPEdNW4tVQUv+nGvxmoxzC4LFW4gnwN1bb6jNldFWxo279bXDQ3w4ACdkSJjIBtOX3R8GZwXpGwroFe57YtzpaSahUy51Yuope6lqc11N7wwPdrHiOKv7vvW8Aq7FA8ZdkQQbtOflVUBuNkbYbCQYK30RP9vAMms9Vm5zh+IngbAJtZdKoxqgFsLefDHoy8TQ3ldjabAW8XR1sz+AuEP7VwCJS1jlbo6qZN0b5s8CA3LTTOGvKl7\\\\\\\/v2ZcXw0md3xBwDde51CvE4esfsjxuyWnutECpydQHkSH4FQN4+tiOM7hXyXe77emovAmRZCYBnfwaKmi9rFyebWZczBN8UGFxby0tUAdR0bycFmKXQGQYte97IoIgIphKXd36ggpiNCfOcpeVcxhJzZcUW+eqKZHL1HT5lDUi3DmncFde1cvzUb8cDHqHBX6B6N767qNEd6+RrasCALTrXd2obAZRrJe05m7\\\\\\\/f1PEbyXN5HJCUJzSoZD0ocAlE4mbsgzgJaBEg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mjvDZqwRTtynfThrAGVQuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c0c68f6190bafc37263564f58e700b3aa93f36e2d14ef73baadc24a449d8412d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QTzM6KHEmfJbXE\\\\\\\/OaEqdpEGjiWCY5os2I+\\\\\\\/0Ew2rmNmbF05LbNrQr8GiaRg3UqUznIdtClmHszItxDvIaUmjs+h9fnhkPGrrANbhrDA0Nrjom6C5rUXCjjp74jApVhUEm5\\\\\\\/DqbdgnIl8sH7nY0nRBuQhKSjzhPZpPNXdq1IzPjFu6Y6U28xO2odrKCwlrq4iIDvFUsczilrOYPhePnfDp0pI09X29e++JcVzZDYeIP9V7LiaNTxKgIjLZt1U8JdhPArRJT9bWWdq6OxxxcvgVpOAF9cCJdllxs8Y1500uIALM671g5gY\\\\\\\/w88lof2bmizd\\\\\\\/YX6lShzpWA2kkvF5lHla7WUde0Ni8Fl5UVmmwqbwSVKr3\\\\\\\/7IfNk4n34JHc5Ci5ehfhqVIfNc0fHb4VaDD3OldvYgW8ctPLovS6cGZl4GGVb+\\\\\\\/72YB8XFVQbcRYtIoVsWuEcWMvdPeyEa0yUHJ75USACHiirlnmc0u3WyqiWFBQm2uVkC5exoyPeo93QTfmyTzUTBCM\\\\\\\/yC4D4dlzfKhXsT79vEHdQts8gaost7zoaPhRWHdmr0pOPu3LuKdsnGb+oCVZDPxqUduHtMqIaXAAM9ijHAe7JF+fDTw4\\\\\\\/1GFBrt4e9uyIxydve\\\\\\\/USpqcyeqcAcKYEYQjAp73nYYU08+ObceHaKfCkmWc0fOZ5xQhBTztTExZimTPa4gOKAdOMbLXhK8Dzz7OvUjhPG9BkxsT90xPAo+2ysnQiftspp5y\\\\\\\/3I5end6Xg4PiZ3QCyGApHw3y5wHjE7wNxkGcekTZxGvxx+6Ig9hbSNTnOFH\\\\\\\/GVShnaaR3M9xRRRx5skQqVTDNwYfdsAkse2e9AJR23AwttIRsaKpovuy\\\\\\\/+vN1wnaNp24xaUCGAcEo14I+XVZoeck1nXOM3rCdPlhrUaK3WpSIip\\\\\\\/HZxgky4iU1LQUsUQ78mIPyLO2NVWifredqM+QVWYmQTHG68H961glLE11MarFbpy0sf3e+zENmdW9NbRfkjAwmuXOnF\\\\\\\/WPDl\\\\\\\/gm+ETyzs63KY0VNgH9AVuKqzrFsLYkPaF0tp4iIF2PN6rsnMtF1f5W\\\\\\\/ROKr3DMCYs5L9pdx6JcjnOyO7WM9iYcoQwU7UxBvWY7bgj5vkEL0PyjBam3urnvfSi2cW+3vtHsOAqm8ycxGahbikWIBTAMe7AV87H1KQpX1ocv3OMBQxS1jQkmjKHvHKnK63ZJQpSHNHpZWAjRUiNP1xCJndOE92fR\\\\\\\/jboBccCkIDYjHI\\\\\\\/2gSLR2saa85IKSKULflkSSQM2Xu1ZtVlHBBO\\\\\\\/j6D+i8UyBhXGPeBAMa\\\\\\\/+MdRz\\\\\\\/B3NE59mBGMqVXjjT3W4hxplVbMjaS7p1b1L7s9BPXujz7SOalHbje0Bo2LvEbxvGTa60CleKHuWrU8+REsQSr8I2IVsUtNB3SODu315uFrt8Wgniv9Mt5LXadb6AwfAwYogPSf0D7a4xjtmTIci2WkrTPbSASUEQRjYX54pZvJXUSc9lRP1pL0OdYUSiIiZdY9\\\\\\\/9q8LbwivFyDb7H6smzZJj36g6pLfvAbFsXbQn7IX7Kbryrn0T10Yr4sNYhaL05A+4Hi5pKiHsNfqB3afuabtx3AnX7ryjC+YPmB5GHBePsUQ+9aPMqbD6ySB9ql911JwDKntzN4taqCExvEcHebtT\\\\\\\/7oNDoIiCTX4tp+uaIuJdLvd3nFr36P+v4PyLVp7CO81Csu8g3ckVV4Q4dPYOAfBBScLVD9APTWYPLmtSt0MyMGrvlklAumfCF6X1Glwr2zNFFsk4\\\\\\\/oB7opmGLyQS0AAvDKtpiMs8JYL7UYSz6TMZ8dq1f9ATQ3VW54u8yejwvfTsPW6W60liFGr6D4SNA6VpZHiss3PMwiV5QqcqWrw+YAGnqEalmXIouKZSi4xkyozTzk2vLc3RNjWc0KS7tOgqJyLtMerZqhzQcSTHArvHKuOTPm+VMQarlwYAECA9n5uJ1zWYyGVdypZqbaDoHvCKD7dICEikUaLdVVUeO6cqrGNWKXvVrA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022iG+BvN+GF5BXJvoYYfoWyA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c7bb6d35e78fce70ffee97ef88f43282b4c0cbf8ee02391773aa2d9831b48421\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494293 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3201" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Sz_YZiK5egPkoo5l4LER69C5MCgWcZw0H37X_rTe4qY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3201" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6mhD-flBe88krscOr3LhtsHHK-wlVJ9jxAbRaHszsjk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3181" title="Elucidating Drought Stress Tolerance in European Oaks Through Cross-Species Transcriptomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3213" title="Fine Mapping Using Whole-Genome Sequencing Confirms Anti-Müllerian Hormone as a Major Gene for Sex Determination in Farmed Nile Tilapia (Oreochromis niloticus L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494293" id="node494293" data-pisa="ggg;9/10/3201" data-pisa-master="ggg;g3.119.400280" data-apath="/ggg/9/10/3201.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the <em>Drosophila</em> Phylogeny</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cara L. Brand</span>, <span class="highwire-citation-author" data-delta="1">Lori Wright</span> and <span class="highwire-citation-author" data-delta="2">Daven C. Presgraves</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3201-3211;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400280 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494293" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cara L. Brand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cara%2BL.%2BBrand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brand%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACara%2BL.%2BBrand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cara.brand@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">cara.brand@rochester.edu</a>
<a href="mailto:daven.presgraves@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">daven.presgraves@rochester.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lori Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lori%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALori%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daven C. Presgraves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daven%2BC.%2BPresgraves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Presgraves%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaven%2BC.%2BPresgraves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cara.brand@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">cara.brand@rochester.edu</a>
<a href="mailto:daven.presgraves@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">daven.presgraves@rochester.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3201" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494293" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494293/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3201.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494293" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494293/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3201.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494293" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494293/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3201.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494293" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494293/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Meiotic crossing over ensures proper segregation of homologous chromosomes and generates genotypic diversity. Despite these functions, little is known about the genetic factors and population genetic forces involved in the evolution of recombination rate differences among species. The dicistronic meiosis gene, <em>mei-217/mei-218</em>, mediates most of the species differences in crossover rate and patterning during female meiosis between the closely related fruitfly species, <em>Drosophila melanogaster</em> and <em>D. mauritiana</em>. The MEI-218 protein is one of several meiosis-specific mini-chromosome maintenance (mei-MCM) proteins that form a multi-protein complex essential to crossover formation, whereas the BLM helicase acts as an anti-crossover protein. Here we study the molecular evolution of five genes— <em>mei-218</em>, the other three known members of the mei-MCM complex, and <em>Blm</em>— over the phylogenies of three <em>Drosophila</em> species groups— <em>melanogaster</em>, <em>obscura</em>, and <em>virilis</em>. We then use transgenic assays in <em>D. melanogaster</em> to test if molecular evolution at <em>mei-218</em> has functional consequences for crossing over using alleles from the distantly related species <em>D. pseudoobscura</em> and <em>D. virilis</em>. Our molecular evolutionary analyses reveal recurrent positive selection at two mei-MCM genes. Our transgenic assays show that sequence divergence among <em>mei-218</em> alleles from <em>D. melanogaster</em>, <em>D. pseudoobscura</em>, and <em>D. virilis</em> has functional consequences for crossing over. In a <em>D. melanogaster</em> genetic background, the <em>D. pseudoobscura mei-218</em> allele nearly rescues wildtype crossover rates but alters crossover patterning, whereas the <em>D. virilis mei-218</em> allele conversely rescues wildtype crossover patterning but not crossover rates. These experiments demonstrate functional divergence at <em>mei-218</em> and suggest that crossover rate and patterning are separable functions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/crossing-over" class="hw-term hw-article-keyword hw-article-keyword-crossing-over" rel="nofollow">crossing over</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li><li class="kwd"><a href="/keyword/positive-selection" class="hw-term hw-article-keyword hw-article-keyword-positive-selection" rel="nofollow">positive selection</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila</em></a></li></ul><p id="p-4" class="flushleft">During the early stages of meiosis, recombination occurs between homologous chromosomes, serving two functions. First, recombination repairs programmed DNA double-strand breaks (DSBs) and ensures proper Mendelian segregation of homologous chromosomes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baker and Hall 1976</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lindsley and Sandler 1977</a>). Second, recombination increases the efficacy of natural selection by reducing genetic linkage and creating novel genotypes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Fisher 1930</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Felsenstein 1974</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Crow 1992</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barton and Charlesworth 1998</a>). Despite these benefits, recombination has risks. Dispersed selfish repetitive DNA sequences— <em>e.g.</em>, transposons— introduce the risk of non-homologous ectopic exchange that generates chromosomal duplications and deletions (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Goldberg <em>et al.</em> 1983</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Charlesworth <em>et al.</em> 1994</a>). The rate and distribution of crossing over may therefore evolve to balance the benefits of recombination and the costs of ectopic exchange (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Montgomery <em>et al.</em> 1987</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Charlesworth and Barton 1996</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kent <em>et al.</em> 2017</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>).</p><p id="p-5">Recombination landscapes vary within and among taxa, but the genes, mechanisms, and evolutionary causes involved are still largely unknown (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ritz <em>et al.</em> 2017</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stapley <em>et al.</em> 2017</a>). In mammals, four loci are associated with intraspecific variation in recombination rates: RNF212, CPLX1, REC8, and PRDM9 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baudat <em>et al.</em> 2010</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sandor <em>et al.</em> 2012</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kong <em>et al.</em> 2014</a>). Best studied is the gene, <em>Prdm9</em>, which encodes a <em>trans</em>-acting major determinant of recombination distribution in most mammals (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Baudat <em>et al.</em> 2010</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Myers <em>et al.</em> 2010</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Parvanov <em>et al.</em> 2010</a>). The PRDM9 protein binds specific DNA sequence motifs and modifies local histones, initiating the formation of DSBs nearby (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Baudat <em>et al.</em> 2010</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Myers <em>et al.</em> 2010</a>). These DSBs are concentrated in the immediate vicinity of the motif, creating recombination “hotspots” once repaired. In rodents and primates, <em>Prdm9</em> shows signals of recurrent positive selection, particularly at sites encoding a zinc finger array that mediates DNA motif binding specificity (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Oliver <em>et al.</em> 2009</a>). The recurrent evolution at <em>Prdm9</em> alters the genomic distribution of recombination hotspots between closely related species and can, incidentally, cause sterility in species hybrids (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Ptak <em>et al.</em> 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Davies <em>et al.</em> 2016</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Smagulova <em>et al.</em> 2016</a>).</p><p id="p-6">In <em>Drosophila</em>, variation in the rate of recombination during female meiosis (males are achiasmate) exists along the lengths of chromosomes, among individuals, and between closely related species. Along chromosomes, rates of crossing over tend to be highest in medial euchromatic regions, lowest in centromere- and telomere-proximal regions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dobzhansky 1930</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beadle 1932</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baker and Carpenter 1972</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Lindsley and Sandler 1977</a>), and absent in heterochromatic regions where repetitive DNA sequences are abundant (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baker 1958</a>). Among individuals, natural genetic variation in crossover rates exists and responds to artificial selection (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kidwell 1972</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Charlesworth and Charlesworth 1985</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brooks and Marks 1986</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Brooks 1988</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Comeron <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hunter <em>et al.</em> 2016</a>). Between species, mean crossover frequencies vary more than two-fold (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ortiz-Barrientos <em>et al.</em> 2006</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Comeron <em>et al.</em> 2012</a>; L. Hemmer <em>et al</em>. unpublished). Despite these observations, few genetic loci are known that contribute to the observed variation in recombination rates (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Hunter <em>et al.</em> 2016</a>).</p><p id="p-7">The genetic basis for recombination differs between flies and mammals. For one, many genes identified in mammals, including <em>Prdm9</em>, appear to be absent from <em>Drosophila</em> (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Oliver <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Heil and Noor 2012</a>). Consistent with this, <em>Drosophila</em> lack comparably strong recombination hotspots (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Comeron <em>et al.</em> 2012</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Hunter <em>et al.</em> 2016</a>). <em>Drosophila</em> also lack <em>Msh4</em> and <em>Msh5</em>, the canonical proteins that promote crossover formation in most eukaryotes. Instead, flies have co-opted a meiosis-specific mini-chromosome maintenance (mei-MCM) complex encoded by the genes <em>mei-217</em>, <em>mei-218</em>, <em>rec</em>, and (presumably) <em>Mcm5</em> to promote the formation of class I crossovers (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zalevsky <em>et al.</em> 1999</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kohl <em>et al.</em> 2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kohl and Sekelsky 2013</a>). Class I crossovers are derived from heteroduplex DNA molecules (crossover intermediates) which are stabilized by the mei-MCM complex and ultimately resolved as crossover events (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Kohl <em>et al.</em> 2012</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Kohl and Sekelsky 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hatkevich <em>et al.</em> 2017</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hatkevich and Sekelsky 2017</a>). Class I crossovers are patterned by interference mechanisms that reduce the probability of a second crossover establishing nearby (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Muller 1916</a>) and by crossover suppression mechanisms that discourage crossover formation in telomere- and centromere-proximal regions (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Dobzhansky 1930</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Beadle 1932</a>). Mutations in mei-MCM genes result in a &gt;90% reduction in crossover frequency and a uniform chromosomal distribution of residual crossovers (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baker and Carpenter 1972</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Carpenter and Sandler 1974</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Grell 1984</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lake <em>et al.</em> 2007</a>). These residual crossovers, termed class II crossovers, are uniformly distributed with chromosome length and thus lack the spatial patterning that results from crossover interference or telomere- and centromere-proximal crossover suppression (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; reviewed in <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Kohl and Sekelsky 2013</a>). While the mei-MCMs promote class I crossover formation, the BLM helicase antagonizes crossover formation by dissolving heteroduplex DNA at multiple stages (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Hatkevich and Sekelsky 2017</a>). BLM unwinds the D-loops formed by strand invasion, leading to synthesis-dependent strand annealing (SDSA), the cause of most non-crossover gene conversion events (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allers and Lichten 2001</a>). BLM also dissolves crossover intermediates in the class II pathway, as the mei-MCMs do not act to stabilize these (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Kohl <em>et al.</em> 2012</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hatkevich <em>et al.</em> 2017</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Hatkevich and Sekelsky 2017</a>). Thus, BLM has anticrossover function, whereas the mei-MCMs are said to have “anti-anticrossover” function (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Kohl and Sekelsky 2013</a>). Given the antagonistic activities of the mei-MCM proteins and the BLM helicase, it seems plausible that molecular evolution at any of these proteins could contribute to phenotypic evolution of crossover rate and distribution.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Meiotic recombination in Drosophila. During meiotic recombination programmed DSBs are formed, the 5&#x2032; ends are recessed and the resulting 3&#x2032; single-stranded tails invade the homologous chromosome. After synthesis off the template, the BLM Helicase can unwind the structure which is then resolved via synthesis-dependent strand annealing (SDSA) into a non-crossover gene conversion. If the invading strand synthesizes far enough, second-end capture can occur, creating a crossover intermediate. Most crossover intermediates are processed via the class I pathway in which they are stabilized by the mei-MCM complex (green rings) and resolved into interfering crossovers. A smaller fraction of crossover intermediates enters the class II pathway in which they are resolved as either non-interfering crossovers or non-crossovers with equal probability. The BLM Helicase inhibits crossover intermediate processing though the class II pathway and therefore promotes the class I pathway." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386082795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Meiotic recombination in &lt;em&gt;Drosophila&lt;/em&gt;. During meiotic recombination programmed DSBs are formed, the 5&#x2032; ends are recessed and the resulting 3&#x2032; single-stranded tails invade the homologous chromosome. After synthesis off the template, the BLM Helicase can unwind the structure which is then resolved via synthesis-dependent strand annealing (SDSA) into a non-crossover gene conversion. If the invading strand synthesizes far enough, second-end capture can occur, creating a crossover intermediate. Most crossover intermediates are processed via the class I pathway in which they are stabilized by the mei-MCM complex (green rings) and resolved into interfering crossovers. A smaller fraction of crossover intermediates enters the class II pathway in which they are resolved as either non-interfering crossovers or non-crossovers with equal probability. The BLM Helicase inhibits crossover intermediate processing though the class II pathway and therefore promotes the class I pathway.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3201/F1.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3201/F1.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494362" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Meiotic recombination in <em>Drosophila</em>. During meiotic recombination programmed DSBs are formed, the 5′ ends are recessed and the resulting 3′ single-stranded tails invade the homologous chromosome. After synthesis off the template, the BLM Helicase can unwind the structure which is then resolved via synthesis-dependent strand annealing (SDSA) into a non-crossover gene conversion. If the invading strand synthesizes far enough, second-end capture can occur, creating a crossover intermediate. Most crossover intermediates are processed via the class I pathway in which they are stabilized by the mei-MCM complex (green rings) and resolved into interfering crossovers. A smaller fraction of crossover intermediates enters the class II pathway in which they are resolved as either non-interfering crossovers or non-crossovers with equal probability. The BLM Helicase inhibits crossover intermediate processing though the class II pathway and therefore promotes the class I pathway.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Recently, we showed that the dicistronic gene, <em>mei-217/mei-218</em> (hereafter <em>mei-217/-218</em>), is a major contributor to evolved species differences in recombination rate and patterning between <em>D. melanogaster</em> and <em>D. mauritiana</em> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>). The total genetic map of <em>D. mauritiana</em> is ∼1.8-fold longer than that of <em>D. melanogaster</em>, and the chromosomal distribution of crossover events differs between species (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">True <em>et al.</em> 1996</a>). This is most evident in telomere- and centromere-proximal regions where crossover formation is more suppressed in <em>D. melanogaster</em> than <em>D. mauritiana</em> (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">True <em>et al.</em> 1996</a>). When a wildtype <em>D. mauritiana</em> allele of <em>mei-217/-218</em> is transgenically introduced into mutant <em>D. melanogaster</em> females lacking <em>mei-218</em> function, a largely (∼82%) <em>D. mauritiana</em>-like genetic map is observed (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>). The <em>D. mauritiana</em> allele of <em>mei-217/-218</em> results in weaker telomeric and centromeric suppression of crossing over as well as reduced crossover interference in medial euchromatic regions (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>). Although <em>mei-217</em> and <em>mei-218</em> are encoded on a single transcript with two translation start sites that yield two distinct proteins (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Liu <em>et al.</em> 2000</a>), population genetic signals of recurrent positive selection in the <em>D. melanogaster</em> and <em>D. mauritiana</em> lineages localize exclusively to <em>mei-218</em> (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>). These findings imply that species differences in meiotic crossing over can be mediated by adaptive evolution at <em>mei-218</em>.</p><p id="p-10">In this paper, we study the long-term molecular evolution of MEI-218, and its interacting proteins, and we test for further evidence of its functional divergence in other <em>Drosophila</em> lineages. First, we ask if recurrent positive selection at <em>mei-218</em> is limited to the <em>D. melanogaster</em> and <em>D. mauritiana</em> lineages or instead extends to the broader <em>Drosophila</em> phylogeny, as seen with <em>Prdm9</em> in rodents and primates (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Oliver <em>et al.</em> 2009</a>). We survey protein-coding sequence evolution at genes encoding components of the mei-MCM complex (<em>mei-218</em>, <em>mei-217</em>, <em>rec</em>, <em>Mcm5</em>) and the <em>Blm</em> helicase among members of the <em>melanogaster</em>, <em>obscura</em>, and <em>virilis</em> species groups. Our analyses reveal evidence for recurrent positive selection at two genes— <em>mei-218</em> and <em>rec</em>— in the <em>melanogaster</em> and in the <em>obscura</em> species groups. Second, we ask if evolution at <em>mei-218</em> has functional consequences for meiotic crossing over. The genetic maps of <em>D. pseudoobscura</em> and <em>D. virilis</em> are ∼2 times longer than that of <em>D. melanogaster</em> (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Ortiz-Barrientos <em>et al.</em> 2006</a>; L. Hemmer <em>et al</em>. unpublished). While substitutions at <em>mei-218</em> might have affected crossing over, it is also possible that they were inconsequential (neutral) or mediated the evolution of some other (unknown) function (<em>e.g.</em>, in the male germline; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Chintapalli <em>et al.</em> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Chen <em>et al.</em> 2014</a>). To distinguish these possibilities, we assayed the meiotic crossover phenotypes of transgenes bearing wildtype <em>D. pseudoobscura</em> or <em>D. virilis</em> alleles in an otherwise <em>D. melanogaster</em> genetic background. Our transgene experiments show that both species’ alleles have functionally diverged from that of <em>D. melanogaster</em>: for the <em>D. pseudoobscura</em> allele, overall crossover rates are comparable to <em>D. melanogaster</em> whereas crossover patterning shifts to a more uniform distribution; for the <em>D. virilis</em> allele, crossover rates are aberrantly low compared to wildtype whereas crossover patterning is comparable to <em>D. melanogaster</em>. These observations suggest that crossover rate and crossover patterning may be separable functions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Alignments and PAML analyses</h3><p id="p-11" class="flushleft">We extracted and aligned the coding sequences of five genes, <em>mei-217</em>, <em>mei-218</em>, <em>rec</em>, <em>Mcm5</em>, and <em>Blm</em> from 23 species spread across the <em>melanogaster</em>, <em>obscura</em>, and <em>virilis</em> groups (Table S1). Coding sequences for <em>D. affinis</em> and the species in the <em>virilis</em> group were generously provided by Rob Unckless (University of Kansas) and Yasir Ahmed-Braimah (Cornell University), respectively. Once all coding sequences for each gene were compiled for each of the three species groups, we translated the coding sequence into predicted protein sequences, aligned the amino acid sequences using MUSCLE v3.8.425 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Edgar 2004</a>), then back-translated the alignment into the original nucleotide sequences. CDS alignments were assessed and gap-adjusted by hand to retain in-frame codons (alignments available by request). Each gene alignment was fit to an NSsites model, part of the CODEML package in PAML (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yang 1997</a>). We compared model 7 (M7), which does not allow <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> to exceed 1 for any codons, to model 8 (M8) which allows <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> &gt; 1 for a subset of codons. We used a likelihood ratio test to determine the best fit model. Sites identified as having experienced positive selection were those found to have posterior probabilities &gt;95% with Bayes Empirical Bayes (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yang <em>et al.</em> 2005</a>).</p></div><div id="sec-3" class="subsection"><h3>Generating transgenic flies</h3><p id="p-12" class="flushleft">To generate transgenic flies, we cloned the <em>D. pseudoobscura</em> and <em>D. virilis</em> alleles of <em>mei-217/-218</em> and used the Φ<em>C31</em> integrase to place the transgenes in (the same) desired chromosomal landing sites via site-specific integration (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Venken <em>et al.</em> 2006</a>). We amplified three sections of the <em>D. pseudoobscura mei-217/-218</em> extended gene region with associated 5′ and 3′ noncoding regions from the <em>D. pseudoobscura</em> reference genome strain using iProof polymerase (Bio-Rad, Hercules, CA) and sequentially reconstructed the fragments within a pBluescript KS+ vector in a three-step process using standard molecular techniques. The first fragment, containing the upstream region and most of <em>mei-217</em>, was amplified as a 2.2kb fragment, phosphorylated, and cloned into the <em>Spe</em>I site of KS+ generating an intermediate plasmid KS[pse<sup>frag1</sup>]. The second fragment, containing the 3′ end of <em>mei-217</em> and most of <em>mei-218</em>, was amplified as a 2.3kb fragment, phosphorylated, and cloned separately into the <em>Spe</em>I site of KS+ generating a second intermediate plasmid KS[pse<sup>frag2</sup>]. We then digested KS[pse<sup>frag1</sup>] with <em>XbaI</em> and <em>NotI</em> (New England Biolabs, Ipswich, MA), gel purified the resulting 2.2kb fragment, and cloned it upstream of KS[pse<sup>frag2</sup>] into <em>XbaI</em>/<em>NotI</em> sites. This generated a third intermediate plasmid KS[pse<sup>frag1+2</sup>]. The third fragment, containing the 3′ end of <em>mei-218</em> and downstream 3′ non-coding regions, was amplified and digested with <em>XhoI</em> and <em>NarI</em> (New England Biolabs, Ipswich, MA) resulting in a 1.4kb fragment which was subsequently cloned into the <em>XhoI</em>/<em>NarI</em> sites downstream of KS[pse<sup>frag1+2</sup>] generating the final plasmid KS[pse<sup>frag1+2+3</sup>]. This final recombinant plasmid, KS[pse<sup>frag1+2+3</sup>], reconstitutes the entire <em>D. pseudoobscura</em> 5.7kb <em>mei-217/-218</em> gene region. We confirmed the absence of introduced mutations in the cloned <em>mei-217/-218</em><sup>pse</sup> allele by direct Sanger sequencing of the KS[pse<sup>frag1+2+3</sup>] plasmid. We then cut the <em>mei-217/-218</em> insert from the KS+ vector with <em>NotI</em> and subcloned into an <em>attB</em>[Pacman]-Ap<sup>R</sup> vector obtained from the <em>Drosophila</em> Genomics Resource Center (Bloomington, IN).</p><p id="p-13">In an analogous manner, the <em>D. virilis mei-217/-218</em> extended gene region was amplified from the <em>D. virilis</em> reference genome strain and sequentially reconstructed within a pBluescript KS+ vector (hereafter KS+; Stratagene, La Jolla, CA) in a three-step process using standard molecular biology techniques. All bacterial transformations were performed at room temperature to enhance plasmid stability in One Shot TOP10 chemically competent <em>E. coli</em> (Invitrogen, Carlsbad, CA). First, the upstream region and most of <em>mei-217</em> was amplified as a 2.6 kb fragment, using a 5′ primer that contained a <em>SphI</em> site. The 5′ end of the resulting PCR product was digested with <em>SphI</em> (New England Biolabs, Ipswich, MA) while the 3′ end was made blunt using FastAp Thermosenstive Alkaline Phosphatase (Thermo Fisher, Waltham, MA). This fragment was cloned into the <em>SphI</em>/<em>SpeI</em> sites of KS+, generating intermediate plasmid KS[vir<sup>frag1</sup>]. Second, the 3′ end of the <em>mei-217</em> and most of <em>mei-218</em> was amplified as a 3.2kb fragment. The amplicon was phosphorylated with T4 Polynucleotide Kinase (Invitrogen, Carlsbad, CA) and ligated into the <em>Spe</em>I site of KS+ generating a second intermediate plasmid KS[vir<sup>frag2</sup>]. We then digested KS[vir<sup>frag2</sup>] with <em>AatII</em> and <em>NotI</em> (New England Biolabs, Ipswich, MA), gel purified the resulting 3.2kb fragment, and cloned it into the <em>SphI</em>/<em>SmaI</em> sites downstream of the vir<sup>frag1</sup> generating a third intermediate plasmid KS[vir<sup>frag1+2</sup>]. Third, the 3′ end of <em>mei-218</em> and downstream 3′ non-coding regions was amplified and digested with <em>AatII</em> and <em>SalI</em> (New England Biolabs, Ipswich, MA) resulting in a 1.1kb fragment which was subsequently cloned into the <em>AatII</em>/<em>SalI</em> sites downstream of KS[vir<sup>frag1+2</sup>] generating the final plasmid KS[vir<sup>frag1+2+3</sup>]. This final recombinant plasmid, KS[vir<sup>frag1+2+3</sup>], reconstitutes the entire <em>D. virilis</em> 6.9kb <em>mei-217/-218</em> gene region. We then cut the <em>mei-217/-218</em> insert from the KS+ vector with <em>NotI</em> and subcloned into an <em>attB</em>[Pacman]-Ap<sup>R</sup> vector obtained from the <em>Drosophila</em> Genomics Resource Center (Bloomington, IN). We confirmed the absence of introduced mutations in the cloned <em>mei-217/-218</em><sup>vir</sup> allele by Sanger sequencing.</p><p>Both <em>D. pseudoobscura</em> and <em>D. virilis</em> transgene constructs were introduced into <em>D. melanogaster y w</em>; <em>PBac</em>[<em>y</em><sup>+</sup>-attP-9A]VK00005 flies, which have an <em>attP</em> transgene landing site at cytological position 75A10 on chromosome arm <em>3L</em>, via injections performed by BestGene (Chino Hills, CA). The <em>attB-P</em>[<em>w</em><sup>+</sup> <em>mei-218</em><sup>pse</sup>]-Ap<sup>R</sup> and <em>attB-P</em>[<em>w</em><sup>+</sup> <em>mei-218</em><sup>vir</sup>]-Ap<sup>R</sup> transgenic flies (for simplicity, hereafter referred to as <em>P</em>[<em>mei-217/218</em><sup>pse</sup>] and <em>P</em>[<em>mei-217/-218</em><sup>vir</sup>], respectively) were then made homozygous and maintained as stocks. Following the crossing protocol in (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>), we estimated crossover rates for a multiply marked second chromosome in two female genotypes:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-15"><em>mei-218</em><sup>1</sup>; <em>net ho dp b pr cn</em>/ + + + + + +; <em>P</em>[<em>mei-217</em>/-<em>218</em><sup>pse</sup>]/ +; and</p></li><li id="list-item-2"><p id="p-16"><em>mei-218</em><sup>1</sup>; <em>net ho dp b pr cn</em>/ + + + + + +; <em>P</em>[<em>mei-217</em>/-<em>218</em><sup>vir</sup>]/ +.</p></li></ol><p>To estimate crossover frequencies, we crossed the female genotypes above to homozygous <em>net dpp</em><sup><em>d-ho</em></sup> <em>dp b pr cn</em> males and scored the progeny for all markers. (For clarity, we refer to <em>dpp</em><sup><em>d-ho</em></sup> throughout by its mutant synonym, <em>ho</em>.) We performed <em>n</em> = 13 and <em>n</em> = 14 crosses for <em>mei-217</em>/-<em>218</em><sup>pse</sup> and <em>mei-217</em>/-<em>218</em><sup>vir</sup>, respectively (see Table S2 for data). For each cross, we collected either ∼10 virgin <em>mei-218</em><sup>1</sup>; <em>net ho dp b pr cn/ + + + + + +</em>; <em>P</em>[<em>mei-217/-218</em><sup>pse</sup>]/+ females or <em>mei-218</em><sup>1</sup>; <em>net ho dp b p cn/ + + + + + +</em>; <em>P</em>[<em>mei-217/-218</em><sup>vir</sup>]/+ females, aged them for three to five days, and crossed them to ∼10 <em>net ho dp b pr cn</em> males that were aged for at least two days. After five days, parents were discarded, and the vials were hydrated with a solution of 0.5% propionic acid. All crosses were maintained in an incubator at 24C under a 12-hour light/dark cycle on standard corn-meal media. We estimated means and standard deviations of crossover frequency among the independent, replicate crosses and compared genotypes using standard <em>t</em>-tests (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3B</a>,<a id="xref-fig-4-1" class="xref-fig" href="#F4">4B</a>).</p><p id="p-17">To compare the spatial distribution of crossovers across the five intervals spanning the <em>net-cn</em> region (Table S3) while controlling for overall crossover rate differences, we compared standardized distributions among genotypes based on the proportion of crossovers occurring in each interval using <em>χ</em><sup>2</sup> tests. We also tested for genotypic differences in crossover event distributions among tetrads by inferring the frequencies of non-, single-, double- or triple crossovers (<em>E</em><sub>0</sub>, <em>E</em><sub>1</sub>, <em>E</em><sub>2</sub>, and <em>E</em><sub>3</sub>, respectively) using the algebraic methods of <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Weinstein (1936)</a>. We estimated the strength of Interference (<em>I</em>) as 1− (observed double crossovers / expected double crossovers). All statistical analyses were performed using <em>R</em> (<a href="http://www.R-project.org/">http://www.R-project.org/</a>). The <em>Drosophila</em> stocks and plasmids used in this study are available upon request. The authors affirm that all data necessary for confirming the conclusions of the article are present within the article, figures, and tables.</p></div><div id="sec-4" class="subsection"><h3>Data Availability</h3><p id="p-18" class="flushleft">Fly stocks and transgenic constructs are available upon request. The authors affirm that all data necessary for confirming the conclusions of the article are present within the article, figures, and tables. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9162431">https://doi.org/10.25387/g3.9162431</a>.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Positive selection at meiosis genes that regulate crossover formation and patterning</h3><p id="p-19" class="flushleft">We identified and extracted protein-coding sequences encoding members of the mei-MCM complex (<em>mei-218</em>, <em>mei-217</em>, <em>rec</em>, <em>Mcm5</em>) and, because it antagonizes the mei-MCMs, <em>Blm</em> (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Kohl <em>et al.</em> 2012</a>), from whole genome sequence data of 23 species (see Materials and Methods, Table S1). In some cases, we used Sanger sequencing to complement retrieved sequence data that had gaps and/or quality issues (Table S1). To investigate patterns of long-term molecular evolution, we used maximum likelihood methods to test for phylogenetic evidence of positive selection. We analyzed the <em>melanogaster</em> (<em>n</em> = 14), <em>pseudoobscura</em> (<em>n</em> = 5), and <em>virilis</em> (<em>n</em> = 4) species groups separately, as the phylogenetic distances among the three groups are so large that synonymous site divergence (<em>d</em><sub>S</sub>) is saturated (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Larracuente <em>et al.</em> 2008</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Stanley and Kulathinal 2016</a>). Using the codeml program in the PAML suite, we performed likelihood ratio tests to identify genes that have elevated rates of nonsynonymous substitution relative to synonymous substitution (<em>d</em><sub>N</sub><em>/d</em><sub>S</sub>) (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Yang 1997</a>). In particular, we compared the log-likelihood of a model for which the estimated values of <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> for individual codons is <em>β</em>-distributed between 0 and 1 (model 7) to that of an alternative model for which the distribution includes an additional class of codons with <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> &gt; 1 (model 8). Positive selection is inferred for cases in which model 8 provides a significantly better fit to the data (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Yang 1997</a>; see Materials and Methods).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Molecular evolution across the Drosophila phylogeny. (A) Phylogenetic relationship of the 23 species analyzed within the melanogaster, obscura, and virilis species group. (B) PAML analyses for the three species groups were performed separately because the phylogenetic distances among them is so large that ds is saturated. We report the log-likelihood estimates from a model 7 &#x2013; model 8 comparison (*P &lt; 0.05, **P &lt; 0.01, ***P &lt; 0.001). (C) A schematic of the structural domains in the five proteins analyzed. In the MEI-MCMs the AAA ATPase MCM domains are shaded in green and the MCM N-terminal domains are shaded in blue. In the BLM Helicase the RecQ DNA Helicase domain in shaded in purple. In MEI-218 and REC, the pins represent codons with evidence for positive selection in the melanogaster group (yellow circle) and the obscura group (red squares)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386082795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Molecular evolution across the &lt;em&gt;Drosophila&lt;/em&gt; phylogeny. (A) Phylogenetic relationship of the 23 species analyzed within the &lt;em&gt;melanogaster&lt;/em&gt;, &lt;em&gt;obscura&lt;/em&gt;, and &lt;em&gt;virilis&lt;/em&gt; species group. (B) PAML analyses for the three species groups were performed separately because the phylogenetic distances among them is so large that &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt; is saturated. We report the log-likelihood estimates from a model 7 &#x2013; model 8 comparison (*&lt;em&gt;P&lt;/em&gt; &lt; 0.05, **&lt;em&gt;P&lt;/em&gt; &lt; 0.01, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001). (C) A schematic of the structural domains in the five proteins analyzed. In the MEI-MCMs the AAA ATPase MCM domains are shaded in green and the MCM N-terminal domains are shaded in blue. In the BLM Helicase the RecQ DNA Helicase domain in shaded in purple. In MEI-218 and REC, the pins represent codons with evidence for positive selection in the &lt;em&gt;melanogaster&lt;/em&gt; group (yellow circle) and the &lt;em&gt;obscura&lt;/em&gt; group (red squares).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3201/F2.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3201/F2.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494416" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Molecular evolution across the <em>Drosophila</em> phylogeny. (A) Phylogenetic relationship of the 23 species analyzed within the <em>melanogaster</em>, <em>obscura</em>, and <em>virilis</em> species group. (B) PAML analyses for the three species groups were performed separately because the phylogenetic distances among them is so large that <em>d</em><sub>s</sub> is saturated. We report the log-likelihood estimates from a model 7 – model 8 comparison (*<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001). (C) A schematic of the structural domains in the five proteins analyzed. In the MEI-MCMs the AAA ATPase MCM domains are shaded in green and the MCM N-terminal domains are shaded in blue. In the BLM Helicase the RecQ DNA Helicase domain in shaded in purple. In MEI-218 and REC, the pins represent codons with evidence for positive selection in the <em>melanogaster</em> group (yellow circle) and the <em>obscura</em> group (red squares).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Two of the five genes show phylogenetic evidence of recurrent positive selection. First, we find that <em>mei-218</em> has a history of positive selection in both the <em>melanogaster</em> and <em>obscura</em> species groups (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). A Bayes Empirical Bayes (BEB) analysis (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Yang <em>et al.</em> 2005</a>) identified nine codons in the <em>melanogaster</em> group and three codons in the <em>obscura</em> group, all clustered in the second and third exons, with evidence of positive selection (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>; posterior probability &gt;0.95). The quality of the local sequence alignment is low for both species groups, compromising our confidence in the BEB-identified codons. To address this problem, we performed PAML analyses on each of the five exons, separately, to broadly localize the signal of positive selection within <em>mei-218</em>. Consistent with the codons identified by the BEB analysis, we find that the second and third exons, which encode a disordered protein region, show evidence of positive selection in both the <em>melanogaster</em> and the <em>obscura</em> species groups. We aligned coding sequences using MUSCLE (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Edgar 2004</a>), although other alignment algorithms (<em>i.e.</em>, ClustalW and Geneious) give qualitatively similar results. The alignment uncertainty results partly from high rates of indel evolution at <em>mei-218</em>. For example, compared to the <em>D. melanogaster</em> reference, sequences from the other species of the <em>melanogaster</em> group contain ≥13 indels ranging in size from 1 to 267 codons. This frequent insertion and/or deletion of codons limits the power of our PAML analysis of <em>mei-218</em>: of the 5,166 sites in the <em>mei-218 melanogaster</em> group alignment, 2,967 sites (57%) were not analyzed due to gaps in the multi-species alignment.</p><p id="p-22">Among the other four genes studied, only <em>rec</em> shows evidence of positive selection in the <em>melanogaster</em> and <em>obscura</em> species groups (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). BEB analyses identified two codons in the MCM N-terminal domain with histories of positive selection (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>; posterior probability &gt;0.95): one codon experienced positive selection in the <em>melanogaster</em> group and the other codon experienced positive selection in both the <em>melanogaster</em> and <em>obscura</em> groups. At the positively selected codon detected in the <em>melanogaster</em> group, eight different amino acid states are represented among 14 species. At the positively selected codon detected in both the <em>melanogaster</em> and <em>obscura</em> groups, different amino acid states are represented in 8/14 and 4/5 species, respectively. PAML analyses find no support for positive selection at the remaining three genes (<em>mei-217</em>, <em>Mcm5</em>, or <em>Blm</em>). In the <em>virilis</em> species group, none of the five genes tested show evidence for positive selection (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>).</p></div><div id="sec-7" class="subsection"><h3>Functional analysis of mei-217/-218 from D. pseudoobscura</h3><p id="p-23" class="flushleft"><em>D. pseudoobscura</em> diverged from <em>D. melanogaster</em> ∼30 mya, and its genetic map is ∼2-fold longer, with less expansive centromeric suppression of crossing over and a more uniform recombination landscape (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hamblin and Aquadro 1999</a>; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Ortiz-Barrientos <em>et al.</em> 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Drosophila 12 Genomes Consortium 2007</a>). The levels of coding sequence and length divergence between <em>D. pseudoobscura</em> and <em>D. melanogaster</em> are extraordinary, and different regions of the <em>mei-217/-218</em> sequence have experienced strikingly different rates of molecular evolution. Between <em>D. pseudoobscura</em> and <em>D. melanogaster</em>, pairwise amino acid identity for <em>MEI-218</em> is much lower for than for <em>MEI-217</em> (∼34% <em>vs.</em> 61%, respectively). The very low identity for <em>MEI-218</em> is attributable to the N-terminal disordered region— which shares only ∼20% identity and differs in length by 183 codons— not the C-terminal AAA ATPase MCM domain which shares 70% identity (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>). Given the extraordinary protein sequence (and indel) divergence at <em>MEI-218</em>, the statistical evidence for recurrent positive selection at <em>mei-218</em> in both species groups, and previous experimental evidence that <em>mei-217/-218</em> mediates species differences in crossing over (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>), we sought to test if molecular evolution at <em>mei-217/-218</em> between <em>D. pseudoobscura</em> and <em>D. melanogaster</em> has functional consequences for crossing over.</p><p id="p-24">To experimentally test for functional effects of molecular divergence at <em>mei-217/-218</em> between <em>D. pseudoobscura</em> and <em>D. melanogaster</em>, we cloned the entire <em>mei-217/-218</em> gene region, including all of the upstream and downstream noncoding regions, from <em>D. pseudoobscura</em> into an <em>attB-P</em>[acman] vector (hereafter <em>mei-217/-218</em><sup>pse</sup>; see Materials and Methods). We integrated this transgene construct into an <em>attP</em> site on chromosome arm <em>3L</em> of <em>D. melanogaster</em> (cytological position 75A10) and used genetic crosses to place the transgene in a <em>mei-218</em> loss-of-function genetic background resulting in the <em>D. melanogaster</em> stock, <em>mei-218</em><sup>1</sup>; <em>P</em>[<em>mei-217/-218</em><sup>pse</sup>] (see Materials and Methods). We then estimated crossover frequencies among six visible markers (<em>net ho dp b pr cn</em>) that span chromosome arm <em>2L</em> and the centromere for replicate crosses of <em>mei-218</em><sup>1</sup>; <em>net ho dp b pr cn</em>/+ + + + + +; <em>P</em>[<em>mei-217/-218</em><sup>pse</sup>]/+ females to <em>net ho dp b pr cn</em> males (<em>n</em> = 13 crosses, 2028 progeny; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Table S2). In this genotype, the <em>D. pseudoobscura</em> wildtype allele is the only source of <em>mei-218</em> function. As our transgenes include <em>mei-217/-218</em> coding and non-coding sequence, we are unable to attribute phenotypic effects to species differences in the protein sequence <em>vs.</em> expression level.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The mei-217/-218 allele of D. pseudoobscura alters the rate and patterning of crossing over in D. melanogaster. (A) D. melanogaster females containing a transgene of a D. pseudoobscura mei-217/-218 allele inserted on chromosome 3L (75A10) were assayed for crossing over. The endogenous mei-2181 allele contains a nonsense mutation. Crossover frequencies were estimated among the six visible markers spanning the left arm of chromosome 2 and the centromere: net (net), decapentaplegic (ho), dumpy (dp), black (b), purple (pr), and cinnabar (cn). mei-217/-218mel data re-produced from Brand et al. (2018). (B) For each genotype, the means and standard deviations [in brackets] of crossover frequency for the five genetic intervals measured in the two transgenic genotypes. The p-values are for unpaired t-tests (*P &lt; 0.05, **P &lt; 0.01, ***P &lt; 0.001). (C) The proportion of total crossovers distributed across the five intervals in the net-cn region in mei-217/-218mel (blue) and mei-217/-218pse (green) females. The total number of crossovers scored for mei-217/-218mel and mei-217/-218pse females is 956 and 786, respectively (see Table S2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386082795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;mei-217/-218&lt;/em&gt; allele of &lt;em&gt;D. pseudoobscura&lt;/em&gt; alters the rate and patterning of crossing over in &lt;em&gt;D. melanogaster&lt;/em&gt;. (A) &lt;em&gt;D. melanogaster&lt;/em&gt; females containing a transgene of a &lt;em&gt;D. pseudoobscura mei-217/-218&lt;/em&gt; allele inserted on chromosome &lt;em&gt;3L&lt;/em&gt; (75A10) were assayed for crossing over. The endogenous &lt;em&gt;mei-218&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt; allele contains a nonsense mutation. Crossover frequencies were estimated among the six visible markers spanning the left arm of chromosome &lt;em&gt;2&lt;/em&gt; and the centromere: &lt;em&gt;net&lt;/em&gt; (&lt;em&gt;net&lt;/em&gt;), &lt;em&gt;decapentaplegic&lt;/em&gt; (&lt;em&gt;ho&lt;/em&gt;), &lt;em&gt;dumpy&lt;/em&gt; (&lt;em&gt;dp&lt;/em&gt;), &lt;em&gt;black&lt;/em&gt; (&lt;em&gt;b&lt;/em&gt;), &lt;em&gt;purple&lt;/em&gt; (&lt;em&gt;pr&lt;/em&gt;), and &lt;em&gt;cinnabar&lt;/em&gt; (&lt;em&gt;cn&lt;/em&gt;). &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;mel&lt;/sup&gt; data re-produced from Brand &lt;em&gt;et al.&lt;/em&gt; (2018). (B) For each genotype, the means and standard deviations [in brackets] of crossover frequency for the five genetic intervals measured in the two transgenic genotypes. The &lt;em&gt;p&lt;/em&gt;-values are for unpaired &lt;em&gt;t&lt;/em&gt;-tests (*&lt;em&gt;P&lt;/em&gt; &lt; 0.05, **&lt;em&gt;P&lt;/em&gt; &lt; 0.01, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001). (C) The proportion of total crossovers distributed across the five intervals in the &lt;em&gt;net-cn&lt;/em&gt; region in &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;mel&lt;/sup&gt; (blue) and &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;pse&lt;/sup&gt; (green) females. The total number of crossovers scored for &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;mel&lt;/sup&gt; and &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;pse&lt;/sup&gt; females is 956 and 786, respectively (see Table S2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3201/F3.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3201/F3.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494325" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">The <em>mei-217/-218</em> allele of <em>D. pseudoobscura</em> alters the rate and patterning of crossing over in <em>D. melanogaster</em>. (A) <em>D. melanogaster</em> females containing a transgene of a <em>D. pseudoobscura mei-217/-218</em> allele inserted on chromosome <em>3L</em> (75A10) were assayed for crossing over. The endogenous <em>mei-218</em><sup>1</sup> allele contains a nonsense mutation. Crossover frequencies were estimated among the six visible markers spanning the left arm of chromosome <em>2</em> and the centromere: <em>net</em> (<em>net</em>), <em>decapentaplegic</em> (<em>ho</em>), <em>dumpy</em> (<em>dp</em>), <em>black</em> (<em>b</em>), <em>purple</em> (<em>pr</em>), and <em>cinnabar</em> (<em>cn</em>). <em>mei-217/-218</em><sup>mel</sup> data re-produced from <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> (2018)</a>. (B) For each genotype, the means and standard deviations [in brackets] of crossover frequency for the five genetic intervals measured in the two transgenic genotypes. The <em>p</em>-values are for unpaired <em>t</em>-tests (*<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001). (C) The proportion of total crossovers distributed across the five intervals in the <em>net-cn</em> region in <em>mei-217/-218</em><sup>mel</sup> (blue) and <em>mei-217/-218</em><sup>pse</sup> (green) females. The total number of crossovers scored for <em>mei-217/-218</em><sup>mel</sup> and <em>mei-217/-218</em><sup>pse</sup> females is 956 and 786, respectively (see Table S2).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">These experiments have three possible outcomes. First, sequence evolution at <em>mei-217/-218</em> may be of no functional consequence to crossing over: <em>mei-217/-218</em><sup>pse</sup> might rescue the <em>mei-218</em> mutant phenotype but produce wildtype <em>D. melanogaster</em>-like rates and patterning of crossing over. Second, sequence evolution at <em>mei-217/-218</em> may render it incompatible between species: the <em>mei-217/-218</em><sup>pse</sup> allele might be sub- or non-functional in <em>D. melanogaster</em> so that it cannot fully rescue the <em>mei-218</em> mutant phenotype. Last, sequence evolution at <em>mei-217/-218</em> may recapitulate some of the wildtype species differences in crossing over: the <em>mei-217/-218</em><sup>pse</sup> allele might rescue the null <em>mei-218</em> mutant phenotype but produce rates and/or patterning of crossing over that differ from <em>D. melanogaster</em> in a way similar to <em>D. pseudoobscura</em>.</p><p id="p-27">We find that, in a <em>D. melanogaster</em> genetic background lacking <em>mei-218</em> function, the <em>mei-217/-218</em><sup>pse</sup> rescues crossing over: the total <em>net-cn</em> genetic map length of 39.03 map units is smaller (∼1.13-fold) than, but comparable to <em>mei-217/-218</em><sup>mel</sup> controls (<em>t</em>-test, <em>P</em> = 0.044; <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). However, while the total lengths of the <em>mei-217/-218</em><sup>pse</sup> and <em>mei-217/-218</em><sup>mel</sup> genetic maps are comparable, we observe highly significant crossover rate heterogeneity among intervals between the transgenes. In <em>mei-217/-218</em><sup>pse</sup> females, the medial largest interval of chromosome arm <em>2L</em> (<em>dp-b</em>) experiences a 1.6-fold lower crossover frequency than the <em>mei-217/-218</em><sup>mel</sup> control (<em>t</em>-test, <em>P</em> &lt; 0.0001; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>) whereas the telomere-proximal (<em>net-ho</em>) and centromere (<em>pr-cn</em>) regions experience 2.13- and 1.31-fold higher crossover frequencies, respectively (<em>t</em>-test, <em>P</em> = 0.001 and <em>P</em> = 0.028, respectively; <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>). To distinguish crossover frequency and crossover patterning, we calculated the proportion of total crossovers that occurred in each genetic interval for each transgene. These values differ from genetic map distances (the proportion of recombinant progeny), for which crossover rate and patterning are confounded, and instead provide profiles of crossover patterning that are independent of crossover rate. We find that the patterning of crossovers differs significantly between <em>mei-217/-218</em><sup>mel</sup> and <em>mei-217/-218</em><sup>pse</sup> (<em>χ</em><sup>2</sup> test, <em>df</em> = 4, <em>P</em> &lt; 9.52e<sup>-28</sup>; <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>, Table S3). Specifically, crossovers in <em>mei-217/-218</em><sup>mel</sup> females are concentrated in the medial <em>ho-dp-b</em> regions and occur at lower frequencies in the telomeric <em>net-ho</em> and centromeric <em>pr-cn</em> regions, resulting in relatively larger variability in the proportion of crossovers among the genetic intervals (range between lowest <em>net-ho</em> and highest <em>dp-b</em> intervals = 0.44; <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B,C</a>; Table S3). In contrast, crossovers in <em>mei-217/-218</em><sup>pse</sup> females are distributed more uniformly across the genetic intervals, with less variability in the proportion of crossovers among the genetic intervals (range between lowest <em>net-ho</em> and highest <em>dp-b</em> intervals = 0.24; <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B,C</a>; Table S3). The <em>mei-217/-218</em><sup>pse</sup> transgene thus produces a total genetic map length that is nearly <em>D. melanogaster</em>-like but crossover patterning that differs from <em>D. melanogaster</em> in ways similar to <em>D. pseudoobscura</em>.</p><p id="p-28">The distribution of the number of crossovers per recovered chromosome— the number of non-crossover (NCO), single-crossover (SCO), double-crossover (DCO) chromosomes, and so on— differs significantly between <em>mei-217/-218</em><sup>pse</sup> and <em>mei-217/-218</em><sup>mel</sup> transgenes (<em>χ</em><sup>2</sup> test, <em>df</em> = 3, <em>P</em> &lt; 1.29e<sup>-16</sup>; Table S2). We used these data to infer the distribution of the number of crossovers per tetrad using the methods of <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Weinstein (1936)</a>. Two mechanisms constrain the distribution of the number of crossovers per meiosis: (1) crossover assurance encourages the formation of at least one obligate crossover per tetrad to guarantee proper segregation; and (2) crossover interference discourages the formation of multiple crossovers near one another (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jones and Franklin 2006</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wang <em>et al.</em> 2015</a>). As a result, the number of crossovers per tetrad is under-dispersed relative to Poisson expectations, with zero- and multiple-crossover classes under-represented in wildtype <em>D. melanogaster</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Mehrotra <em>et al.</em> 2007</a>). Consistent with regulation, the inferred number of crossovers per tetrad is similarly under-dispersed relative to Poisson expectation in <em>mei-217/-218</em><sup>pse</sup> females (<em>χ</em><sup>2</sup> test, <em>df</em> = 5, <em>P</em> &lt; 1.3e<sup>−237</sup>). However, the distribution of the number of crossovers per tetrad differs between the <em>mei-217/-218</em><sup>pse</sup> and <em>mei-217/-218</em><sup>mel</sup> females (<em>χ</em><sup>2</sup> test, <em>df</em> = 3, <em>P</em> = 2.2e<sup>−16</sup>; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The mean number of crossovers per tetrad is reduced in <em>mei-217/-218</em><sup>pse</sup> females compared to <em>mei-217/-218</em><sup>mel</sup> controls (0.78 <em>vs.</em> 0.91). This difference is largely attributable to a 1.9-fold increase in non-crossover (<em>E</em><sub>0</sub>) tetrads at the expense of single-crossovers (<em>E</em><sub>1</sub>), which are reduced 1.47-fold (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Crossover assurance thus appears weaker in <em>mei-217/-218</em><sup>pse</sup> females. At the same time, however, we find that multiple-crossover tetrads are more frequent in <em>mei-217/-218</em><sup>pse</sup> females: double- and triple-crossover tetrads are increased 1.16- and 5-fold, respectively (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). We tested whether this increase in multi-crossover tetrads is enabled by weaker crossover interference in <em>mei-217/-218</em><sup>pse</sup> females. Indeed, crossover interference for the two largest adjacent intervals (<em>ho-b-pr</em>) is ∼28% weaker in <em>mei-217/-218</em><sup>pse</sup> females, although the difference is not significant (<em>I</em> = 0.567 <em>vs.</em> 0.793, Mann-Whitney, <em>P</em> = 0.795; Table S2). Together, the weaker crossover assurance and crossover interference in <em>mei-217/-218</em><sup>pse</sup> females increase the relative variance (variance/mean) in the number of crossovers per tetrad (0.698) compared to that for in <em>mei-217/-218</em><sup>mel</sup> (0.350; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494371/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494371/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-29" class="first-child">The distribution of the inferred number of crossovers in the net-cn region per meiosis differs among genotypes</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">These results show that crossover assurance, interference, and centromeric and telomeric suppression are all weaker in <em>mei-217/-218</em><sup>pse</sup> females. These observations are qualitatively consistent with crossover patterning in wildtype <em>D. pseudoobscura</em>, which also shows reduced (or even possibly absent) centromeric suppression and a more uniform distribution of crossovers (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Hamblin and Aquadro 1999</a>; <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Ortiz-Barrientos <em>et al.</em> 2006</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kulathinal <em>et al.</em> 2008</a>). Alternatively, it is possible that the <em>mei-217/-218</em><sup>pse</sup> allele is incompatible with interactors from <em>D. melanogaster</em>, so that the functionally divergent <em>mei-217/-218</em><sup>pse</sup> allele is unable to fully receive and/or implement endogenous crossover patterning signals from <em>D. melanogaster</em>. Under this incompatibility hypothesis, the shift in crossover patterning toward one that is <em>D. pseudoobscura</em>-like would be coincidental rather than a reflection of the wildtype properties of the <em>D. pseudoobscura</em> allele. As with any heterologous transgene (or interspecific genetic) experiment, formally distinguishing between these two interpretations is difficult. However, under either interpretation, the results demonstrate that the effects of <em>D. pseudoobscura</em> and <em>D. melanogaster</em> alleles on meiotic crossing over have diverged and support the notion that <em>mei-217/-218</em> has two separable functions— crossover formation and crossover patterning.</p></div><div id="sec-8" class="subsection"><h3>Functional analysis of mei-217/-218 From D. virilis</h3><p id="p-36" class="flushleft">We also assayed a <em>mei-217/-218</em> wildtype allele from a species more distantly related to <em>D. melanogaster</em>. <em>D. virilis</em> diverged from <em>D. melanogaster</em> ∼50 mya and has a ∼2-fold longer genetic map (L. Hemmer <em>et al</em>. unpublished; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Drosophila 12 Genomes Consortium 2007</a>). Although our PAML analyses failed to detect evidence of positive selection at <em>mei-218</em> within the <em>virilis</em> species group, these analyses are uninformative about the possibility of positive selection in the lineages ancestral to the <em>virilis</em> and <em>melanogaster</em> species groups. The <em>MEI-217</em> protein shows ∼58% pairwise amino acid sequence identity between <em>D. melanogaster</em> and <em>D. virilis</em>. In contrast, the <em>MEI-218</em> protein shares only ∼34% pairwise amino acid sequence identity overall. Divergence at <em>mei-218</em> is not uniform across the protein: while the C-terminal AAA ATPase MCM domain shows ∼63% identity, the disordered N-terminal shows just ∼23% identity (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>; see also <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Manheim <em>et al.</em> 2002</a>). The <em>D. virilis</em> protein is 954 amino acids long, shorter than both <em>D. melanogaster</em> (1186 aa) and <em>D. pseudoobscura</em> (1002 aa). The difference in length is driven by indel evolution largely concentrated in the disordered region of <em>MEI-218</em>, as the N-terminal MCM domain has remained relatively unchanged between the three species (<em>D. melanogaster</em> 337 aa; <em>D. pseudoobscura</em> 336 aa; <em>D. virilis</em> 334 aa; see also <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Kohl <em>et al.</em> 2012</a>).</p><p id="p-37">To functionally assay the <em>D. virilis</em> allele of <em>mei-217/-218</em>, we followed the same strategy used to create transgenic flies with <em>mei-217/-218</em><sup>pse</sup> (see above) and <em>mei-217/-218</em><sup>mau</sup> (<a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>). We first cloned the entire <em>D. virilis mei-217/-218</em> gene region and flanking non-coding sequences into an <em>attB-P</em>[acman] vector (hereafter <em>mei-217/-218</em><sup>vir</sup>) and integrated it into an <em>attP</em> site on <em>3L</em> (75A10) in <em>D. melanogaster</em> (see Materials and Methods). We then used crosses to place the transgene into a <em>D. melanogaster mei-218</em> mutant background and measured crossover frequencies among visible markers spanning part of the second chromosome in replicate <em>mei-218</em><sup>1</sup>; <em>net ho dp b pr cn</em>/+ + + + + +; <em>P</em>[<em>mei-217/-218</em><sup>vir</sup>]/+ females (<em>n</em> = 14 crosses, 1304 progeny; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, Table S2; see Materials and Methods).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The mei-217/-218 allele of D. virilis alters the rate and patterning of crossing over in D. melanogaster. (A) D. melanogaster females containing a transgene of a D. virilis mei-217/-218 allele inserted on chromosome 3L (75A10) were assayed for crossing over. The endogenous mei-2181 allele contains a nonsense mutation. Crossover frequencies were estimated among the six visible markers spanning the left arm of chromosome 2 and the centromere: net (net), decapentaplegic (ho), dumpy (dp), black (b), purple (pr), and cinnabar (cn). mei-2181 and mei-217/-218mel data re-produced from Brand et al. (2018). (B) For each genotype, the means and standard deviations [in brackets] of crossover frequency for the five genetic intervals measured in the mei-218 mutant and two transgenic genotypes. p-values are derived for unpaired t-tests. *P &lt; 0.05, **P &lt; 0.01, ***P &lt; 0.001. (C) The proportion of total crossovers distributed across the five intervals in the net-cn region in mei-217/-218mel (blue), mei-217/-218vir (orange), and mei-2181 (gray) females. The total number of crossovers scored for the mei-217/-218mel, mei-217/-218vir, and mei-2181 females is 956, 93, and 7, respectively (see Table S2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386082795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;mei-217/-218&lt;/em&gt; allele of &lt;em&gt;D. virilis&lt;/em&gt; alters the rate and patterning of crossing over in &lt;em&gt;D. melanogaster&lt;/em&gt;. (A) &lt;em&gt;D. melanogaster&lt;/em&gt; females containing a transgene of a &lt;em&gt;D. virilis mei-217/-218&lt;/em&gt; allele inserted on chromosome &lt;em&gt;3L&lt;/em&gt; (75A10) were assayed for crossing over. The endogenous &lt;em&gt;mei-218&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt; allele contains a nonsense mutation. Crossover frequencies were estimated among the six visible markers spanning the left arm of chromosome &lt;em&gt;2&lt;/em&gt; and the centromere: &lt;em&gt;net&lt;/em&gt; (&lt;em&gt;net&lt;/em&gt;), &lt;em&gt;decapentaplegic&lt;/em&gt; (&lt;em&gt;ho&lt;/em&gt;), &lt;em&gt;dumpy&lt;/em&gt; (&lt;em&gt;dp&lt;/em&gt;), &lt;em&gt;black&lt;/em&gt; (&lt;em&gt;b&lt;/em&gt;), &lt;em&gt;purple&lt;/em&gt; (&lt;em&gt;pr&lt;/em&gt;), and &lt;em&gt;cinnabar&lt;/em&gt; (&lt;em&gt;cn&lt;/em&gt;). &lt;em&gt;mei-218&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt; and &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;mel&lt;/sup&gt; data re-produced from Brand &lt;em&gt;et al.&lt;/em&gt; (2018). (B) For each genotype, the means and standard deviations [in brackets] of crossover frequency for the five genetic intervals measured in the &lt;em&gt;mei-218&lt;/em&gt; mutant and two transgenic genotypes. &lt;em&gt;p&lt;/em&gt;-values are derived for unpaired &lt;em&gt;t&lt;/em&gt;-tests. *&lt;em&gt;P&lt;/em&gt; &lt; 0.05, **&lt;em&gt;P&lt;/em&gt; &lt; 0.01, ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001. (C) The proportion of total crossovers distributed across the five intervals in the &lt;em&gt;net-cn&lt;/em&gt; region in &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;mel&lt;/sup&gt; (blue), &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;vir&lt;/sup&gt; (orange), and &lt;em&gt;mei-218&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt; (gray) females. The total number of crossovers scored for the &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;mel&lt;/sup&gt;, &lt;em&gt;mei-217/-218&lt;/em&gt;&lt;sup&gt;vir&lt;/sup&gt;, and &lt;em&gt;mei-218&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt; females is 956, 93, and 7, respectively (see Table S2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3201/F4.medium.gif" width="440" height="418"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3201/F4.medium.gif" width="440" height="418"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3201/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494319" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">The <em>mei-217/-218</em> allele of <em>D. virilis</em> alters the rate and patterning of crossing over in <em>D. melanogaster</em>. (A) <em>D. melanogaster</em> females containing a transgene of a <em>D. virilis mei-217/-218</em> allele inserted on chromosome <em>3L</em> (75A10) were assayed for crossing over. The endogenous <em>mei-218</em><sup>1</sup> allele contains a nonsense mutation. Crossover frequencies were estimated among the six visible markers spanning the left arm of chromosome <em>2</em> and the centromere: <em>net</em> (<em>net</em>), <em>decapentaplegic</em> (<em>ho</em>), <em>dumpy</em> (<em>dp</em>), <em>black</em> (<em>b</em>), <em>purple</em> (<em>pr</em>), and <em>cinnabar</em> (<em>cn</em>). <em>mei-218</em><sup>1</sup> and <em>mei-217/-218</em><sup>mel</sup> data re-produced from <a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> (2018)</a>. (B) For each genotype, the means and standard deviations [in brackets] of crossover frequency for the five genetic intervals measured in the <em>mei-218</em> mutant and two transgenic genotypes. <em>p</em>-values are derived for unpaired <em>t</em>-tests. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001. (C) The proportion of total crossovers distributed across the five intervals in the <em>net-cn</em> region in <em>mei-217/-218</em><sup>mel</sup> (blue), <em>mei-217/-218</em><sup>vir</sup> (orange), and <em>mei-218</em><sup>1</sup> (gray) females. The total number of crossovers scored for the <em>mei-217/-218</em><sup>mel</sup>, <em>mei-217/-218</em><sup>vir</sup>, and <em>mei-218</em><sup>1</sup> females is 956, 93, and 7, respectively (see Table S2).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Rates of crossing over are strongly reduced in <em>mei-217/-218</em><sup>vir</sup> transgene-bearing females (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). While the control <em>mei-217/-218</em><sup>mel</sup> transgene produces a total genetic map length of 44.27 across the <em>net-cn</em> region, the <em>mei-217/-218</em><sup>vir</sup> transgene produces a genetic map that is ∼6-fold smaller (7.34; <em>t</em>-test, <em>P</em> &lt; 0.0001; <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). Two lines of evidence, however, indicate that <em>mei-217/-218</em><sup>vir</sup> does not behave like a null allele in <em>D. melanogaster</em>: compared to <em>mei-218</em><sup>1</sup> mutant females, <em>mei-217/-218</em><sup>vir</sup> females produce longer genetic maps (2.02 map units; <em>t</em>-test, <em>P</em> = 0.001; <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>) and show non-uniform spatial patterning of crossovers. In mutant <em>mei-218</em><sup>1</sup> females, ∼90% of crossovers are eliminated, and the residual crossovers fail to show crossover interference or centromeric suppression (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Baker and Carpenter 1972</a>); see also <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C</a>, Table S3). These observations suggest that most crossovers in wildtype females correspond to interfering class I crossovers, whereas residual crossovers in <em>mei-218</em><sup>1</sup> females correspond to non-interfering class II crossovers which originate via a different pathway (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Kohl and Sekelsky 2013</a>). To determine if the crossovers in <em>mei-217/-218</em><sup>vir</sup> behave like residual (presumed class II) crossovers of <em>mei-218</em><sup>1</sup> females, we tested whether crossover patterning across the genetic intervals is disrupted. The distribution of crossovers between <em>mei-217/-218</em><sup>vir</sup> and <em>mei-218</em><sup>1</sup> is significantly different (<em>χ</em><sup>2</sup> test, <em>df</em> = 4, <em>P</em> &lt; 1.2e<sup>-12</sup>; <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4C</a>, Table S3). Specifically, <em>mei-218</em><sup>1</sup> females show a non-uniform reduction in crossover frequency, with crossing over reduced in centromere-proximal regions (<em>pr-cn</em>) to only 20.3% of the <em>mei-217/-218</em><sup>mel</sup> value compared to 4.6% across the entire <em>2L</em> (<em>net-cn</em>) region (see also <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Carpenter and Sandler 1974</a>). In contrast, <em>mei-217/-218</em><sup>vir</sup> females show comparable reductions in crossover frequency among regions, with crossing over reduced in the centromere-proximal regions (<em>pr-cn</em>) to 11.9% of the <em>mei-217/-218</em><sup>mel</sup> value compared to 16.6% across the entire <em>2L</em> (<em>net-cn</em>) region (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4B</a>). As a result, crossover patterning in <em>mei-217/-218</em><sup>vir</sup> females is comparable to <em>mei-217/-218</em><sup>mel</sup> (<em>χ</em><sup>2</sup> test, <em>df</em> = 4, <em>P</em> = 0.05; <a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4C</a>). In <em>mei-217/-218</em><sup>vir</sup> females, then, overall crossover frequencies are reduced 83.4%, but gross crossover patterning is largely unchanged. These findings show that <em>mei-217/-218</em><sup>vir</sup> is unable to support wildtype (<em>D. melanogaster</em> or <em>D. virilis</em>) rates of crossing over in a <em>D. melanogaster</em> genetic background but appears able to integrate patterning information specified by <em>D. melanogaster</em>.</p><p id="p-40">The distributions of the observed number of crossover events per recovered chromosome (χ<sup>2</sup> test, <em>df</em> = 3 <em>P</em> = 7.07e<sup>-137</sup>; Table S2) and the estimated number of crossovers per tetrad helps to explain why the total <em>net-cn</em> map length is so much smaller in <em>mei-217/-218</em><sup>vir</sup> females than <em>mei-217/-218</em><sup>mel</sup> females. The estimated number of crossovers per tetrad is under-dispersed in females bearing the <em>mei-217/-218</em><sup>mel</sup> transgene (variance/mean = 0.350), which experience a mean of 0.91 crossovers per tetrad (<em>χ</em><sup>2</sup> test, <em>df</em> = 5, <em>P</em> &lt; e<sup>-200</sup>; <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">Brand <em>et al.</em> 2018</a>). In contrast, the estimated number of crossovers per tetrad is <em>over</em>-dispersed in females bearing the <em>mei-217/-218</em><sup>vir</sup> transgene (variance/mean = 1.33)— with a deficit of single-crossover tetrads (<em>E</em><sub>1</sub> = 0.094) and an excess of multiple-crossover tetrads (<em>E</em><sub>≥2</sub> = 0.021; <em>χ</em><sup>2</sup> test, <em>df</em> = 5, <em>P</em> = e<sup>-20</sup>; <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>)— which experience a mean of only 0.14 crossovers per tetrad. The reduced genetic map in <em>mei-217/-218</em><sup>vir</sup> females therefore occurs because most tetrads experience no crossovers during meiosis. Crossover assurance therefore appears strongly compromised in <em>mei-217/-218</em><sup>vir</sup> females implying that the <em>mei-217/-218</em><sup>vir</sup> allele is unable to ensure an obligate crossover in a <em>D. melanogaster</em> genetic background. Such achiasmate tetrads suffer elevated rates of mis-segregation and nondisjunction leading to production of aneuploid gametes and reduced fecundity (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Baker and Carpenter 1972</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bhagat <em>et al.</em> 2004</a>). Consistent with nondisjunction, <em>mei-217/-218</em><sup>vir</sup> females produce significantly fewer progeny (mean ± SD = 93.14 ± 17.3) than <em>mei-217/-218</em><sup>mel</sup> females (161.77 ± 52.76; <a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4B</a>; <em>t</em>-test <em>P</em> = 0.0005).</p><p id="p-41">The <em>mei-217/-218</em><sup>vir</sup> allele provides additional evidence that crossover rate and patterning are separable: in <em>D. melanogaster</em>, <em>mei-217/-218</em><sup>vir</sup> is hypomorphic with respect to crossover formation but not crossover patterning. The fact that <em>mei-217/-218</em><sup>vir</sup> cannot fully complement the <em>mei-218</em><sup>1</sup>-mediated loss of crossover formation in <em>D. melanogaster</em> suggests either of two possibilities. The wildtype function of <em>mei-217/-218</em><sup>vir</sup> may differ from <em>mei-217/-218</em><sup>mel</sup>: whereas <em>mei-217/-218</em><sup>mel</sup> functions in crossover formation and patterning in <em>D. melanogaster</em> (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Baker and Carpenter 1972</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">McKim <em>et al.</em> 1996</a>; <a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Kohl <em>et al.</em> 2012</a>), <em>mei-217/-218</em><sup>vir</sup> may be less essential to crossover formation but still essential to crossover patterning in <em>D. virilis</em>. Alternatively, <em>mei-217/-218</em><sup>vir</sup> may be genetically incompatible with factors from <em>D. melanogaster</em>: <em>mei-217/-218</em><sup>vir</sup> may fail to interact appropriately with <em>D. melanogaster</em>-encoded proteins such that crossover formation (but not patterning) is compromised. Under either model, the molecular divergence at <em>mei-217/-218</em> between <em>D. melanogaster</em> and <em>D. virilis</em> has functional consequences for female meiosis.</p></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-42" class="flushleft">Our phylogenetic analyses revealed that two mei-MCM genes, <em>mei-218</em> and <em>rec</em>, have histories of recurrent positive selection in the <em>melanogaster</em> and <em>obscura</em> species groups, and our transgenic assays show that the different species’ <em>mei-217/-218</em> alleles have functionally diverged with respect to crossover patterning (<em>mei-217/-218</em><sup>pse</sup>) and crossover formation (<em>mei-217/-218</em><sup>vir</sup>). These observations are superficially reminiscent of the recurrent positive selection at <em>Prdm9</em>, the major <em>trans</em>-acting factor that controls the distribution of recombination hotspots in mammals. The forces driving the rapid molecular evolution of <em>Prdm9</em> are reasonably well understood. During recombination-repair of DSBs, the DNA sequence motifs recognized by the PRDM9 zinc fingers tend to be replaced with non-motif sequence. As the number of recombination hotspots erodes over time, the overall frequency of recombination decreases to suboptimal levels, elevating the risk of chromosomal mis-segregation and/or breakage (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Ségurel <em>et al.</em> 2011</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Smagulova <em>et al.</em> 2016</a>). This model explains why there is selection for PRDM9 to acquire novel zinc fingers that recognize novel DNA sequence motifs, creating a new class of recombination hotspots, and thereby reestablishing appropriate recombination frequencies. This process can quickly lead to differences in the identity and distribution of recombination hotspots between closely related species and, incidentally, to sterility in species hybrids (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Ptak <em>et al.</em> 2005</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Davies <em>et al.</em> 2016</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Smagulova <em>et al.</em> 2016</a>). For the mei-MCMs, <em>mei-218</em> and <em>rec</em>, the causes of recurrent positive selection are unclear. In <em>Drosophila</em>, fine-scale heterogeneity in recombination rates exists, but recombination hotspots comparable to those in mammals do not (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Comeron <em>et al.</em> 2012</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Hunter <em>et al.</em> 2016</a>). Moreover, unlike PRDM9, the mei-MCMs do not have DNA binding domains known to recognize specific motifs. It therefore seems doubtful that the positive selection we have observed involves DNA motif turnover.</p><p id="p-43">The phylogenetic evidence for recurrent bouts of positive selection at <em>mei-218</em> and <em>rec</em> are similarly consistent with adaptation to moving fitness optima but the causes of selection are unclear. For instance, despite the absence of crossing over in <em>Drosophila</em> males, FlyAtlas and modEncode data show that both <em>mei-218</em> and <em>rec</em> are expressed in <em>D. melanogaster</em> testes (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Chintapalli <em>et al.</em> 2007</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Chen <em>et al.</em> 2014</a>). It is therefore possible that the history of positive selection at these genes reflects adaptation for male reproductive functions, although the functions of <em>mei-218</em> and <em>rec</em> in testes are unknown and mutant males are fertile. If, instead, recurrent positive selection at <em>mei-218</em> and <em>rec</em> has occurred to modulate crossing over, then we require a model in which the optimal rate and/or distribution of crossing over has changed repeatedly. Selfish genetic elements could provide one source of such fluctuating selection. First, meiotic drive in the female germline can generate selection for modifiers of crossing over. Depending on such details as whether drive occurs in meiosis I or II, or whether drive involves the centromere or telomere(s), selection can favor modifiers that increase or decrease rates of crossing over (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brandvain and Coop 2012</a>). Second, while crossing over provides important meiotic and evolutionary functions, it also entails the risk of ectopic non-homologous exchange between similar but dispersed sequences, like transposons. Ectopic exchange can generate deleterious duplications, deletions, and other chromosomal aberrations (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Goldberg <em>et al.</em> 1983</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barrón <em>et al.</em> 2014</a>). The optimal recombination rate should thus evolve to balance the benefits of crossing over against the costs. The requirement for at least one crossover per chromosome (arm) sets a minimum rate, whereas the risk of ectopic exchange may constrain the maximum rate. The risk of ectopic exchange depends on the abundance of dispersed repetitive DNA sequences with high similarity. In <em>D. melanogaster</em>, ≥2% of meioses yield aberrant chromosomes as a result of ectopic exchange between transposons (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016</a>). The rate of such ectopic exchange undoubtedly fluctuates over time, tracking with the load(s) of transposons of high sequence similarity. The typical evolutionary-demographic history of transposons involves invasion of a new host genome via horizontal transfer (or escape from suppression by the host surveillance system); a burst of proliferation; and eventual silencing upon capture by the host surveillance system (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Charlesworth <em>et al.</em> 1994</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kidwell and Lisch 2000</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barrón <em>et al.</em> 2014</a>). Under this scenario, the risk of ectopic exchange due to any particular transposon will spike with transposon proliferation, as genomes come to harbor a high number of highly similar transposon sequences, and then fade as the sequences of silenced transposons diverge from one another and degenerate. The response to fluctuating selection pressures on crossover rates could be mediated by meiosis genes like <em>mei-218</em> and <em>rec</em>. Consistent with this hypothesis, in both the <em>melanogaster</em> and <em>obscura</em> groups, <em>D. mauritiana</em> and <em>D. pseudoobscura</em> have higher mean rates of crossing over and smaller transposon loads compared to their respective sister species, <em>D. melanogaster</em> and <em>D. persimilis</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Dowsett and Young 1982</a>; <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">True <em>et al.</em> 1996</a>; <a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Ortiz-Barrientos <em>et al.</em> 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hill and Betancourt 2018</a>). Similarly, within <em>D. melanogaster</em>, transposon densities are highest in chromosomal regions that experience little or no crossing over (reviewed in <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lee and Langley 2010</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Barrón <em>et al.</em> 2014</a>). It is important to note, however, that strong alternative models exist in which the presence of transposons favors <em>increased</em> rates of crossing over (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Charlesworth and Barton 1996</a>). While distinguishing among these hypotheses will be challenging (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Charlesworth 2018</a>), it is clear that selfish genetic elements present a ubiquitous, powerful, and perhaps underappreciated source of selection on rates of recombination.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Jan Spence for comments on an earlier draft of the manuscript; Rob Unckless and Yasir Ahmed-Braimah for providing sequence data; Christina Muirhead for statistical advice; and Jeff Sekelsky for sharing fly stocks. This work was supported by funds from an NSF-DDIG (DEB 1403154) to CLB, and the David and Lucile Packard Foundation, the University of Rochester, and the NIH (R01 GM111380) to DCP.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-45">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9162431">https://doi.org/10.25387/g3.9162431</a>.</p></li><li class="fn" id="fn-8"><p id="p-46"><em>Communicating editor: J. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-23"><span class="received-label">Received </span>April 23, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-27"><span class="accepted-label">Accepted </span>July 27, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Brand <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.1" data-doi="10.1016/S0092-8674(01)00416-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lichten</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Differential timing and control of noncrossover and crossover recombination during meiosis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">57</span>. <a href="https://doi.org/10.1016/S0092-8674(01)00416-0">https://doi.org/10.1016/S0092-8674(01)00416-0</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAllers%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D57%26rft.atitle%253DDifferential%2Btiming%2Band%2Bcontrol%2Bof%2Bnoncrossover%2Band%2Bcrossover%2Brecombination%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900416-0%26rft_id%253Dinfo%253Apmid%252F11461701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00416-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461701&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.2" data-doi="10.1086/282010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Crossing over in heterochromatin</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">60</span>. <a href="https://doi.org/10.1086/282010">https://doi.org/10.1086/282010</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D92%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1958WJ57400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">B. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. T. C.</span> <span class="cit-name-surname">Carpenter</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic analysis of sex chromosomal meiotic mutants in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBaker%26rft.auinit1%253DB.%2BS.%26rft.volume%253D71%26rft.issue%253D2%26rft.spage%253D255%26rft.epage%253D286%26rft.atitle%253DGENETIC%2BANALYSIS%2BOF%2BSEX%2BCHROMOSOMAL%2BMEIOTIC%2BMUTANTS%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F4625747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzEvMi8yNTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3201.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">B. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Meiotic mutants: genetic control of meiotic recombination and chromosome segregation</span>, pp. <span class="cit-fpage">352</span>–<span class="cit-lpage">434</span> in <span class="cit-source"><em>Genetics and Biology of Drosophila</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Novitski</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ashburner</span></span>. Academic Press, New York.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.5" data-doi="10.1146/annurev-genet-120213-092359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrón</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.-S.</span> <span class="cit-name-surname">Fiston-Lavier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">González</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population Genomics of Transposable Elements in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">581</span>. <a href="https://doi.org/10.1146/annurev-genet-120213-092359">https://doi.org/10.1146/annurev-genet-120213-092359</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D48%26rft.spage%253D561%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-120213-092359%26rft_id%253Dinfo%253Apmid%252F25292358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-120213-092359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25292358&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.6" data-doi="10.1126/science.281.5385.1986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Why sex and recombination?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">1986</span>–<span class="cit-lpage">1990</span>. <a href="https://doi.org/10.1126/science.281.5385.1986">https://doi.org/10.1126/science.281.5385.1986</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D281%26rft.issue%253D5385%26rft.spage%253D1986%26rft.epage%253D1990%26rft.atitle%253DWhy%2BSex%2Band%2BRecombination%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.281.5385.1986%26rft_id%253Dinfo%253Apmid%252F9748151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODEvNTM4NS8xOTg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.7" data-doi="10.1126/science.1183439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudat</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Buard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Grey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fledel-Alon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ober</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">840</span>. Erratum 328: 690. <a href="https://doi.org/10.1126/science.1183439">https://doi.org/10.1126/science.1183439</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183439%26rft_id%253Dinfo%253Apmid%252F20044539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.8" data-doi="10.1073/pnas.18.2.160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beadle</span>,  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1932</span> <span class="cit-article-title">A possible influence of the spindle fibre on crossing-over in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">165</span>. <a href="https://doi.org/10.1073/pnas.18.2.160">https://doi.org/10.1073/pnas.18.2.160</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBeadle%26rft.auinit1%253DG.%2BW.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D165%26rft.atitle%253DA%2BPossible%2BInfluence%2Bof%2Bthe%2BSpindle%2BFibre%2Bon%2BCrossing-Over%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.18.2.160%26rft_id%253Dinfo%253Apmid%252F16577442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiIxOC8yLzE2MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.9" data-doi="10.2174/138920210790886835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berchowitz</span>,  <span class="cit-name-given-names">L. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Copenhaver</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic interference: Don’t stand so close to me</span>. <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">102</span>. <a href="https://doi.org/10.2174/138920210790886835">https://doi.org/10.2174/138920210790886835</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DBerchowitz%26rft.auinit1%253DL.%2BE.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D91%26rft.epage%253D102%26rft.atitle%253DGenetic%2Binterference%253A%2Bdon%2527t%2Bstand%2Bso%2Bclose%2Bto%2Bme.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920210790886835%26rft_id%253Dinfo%253Apmid%252F20885817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920210790886835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20885817&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275150100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.10" data-doi="10.1159/000080594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhagat</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Manheim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Sherizen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">McKim</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Studies on crossover-specific mutants and the distribution of crossing over in <em>Drosophila</em> females</span>. <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">171</span>. <a href="https://doi.org/10.1159/000080594">https://doi.org/10.1159/000080594</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DBhagat%26rft.auinit1%253DR.%26rft.volume%253D107%26rft.issue%253D3-4%26rft.spage%253D160%26rft.epage%253D171%26rft.atitle%253DStudies%2Bon%2Bcrossover-specific%2Bmutants%2Band%2Bthe%2Bdistribution%2Bof%2Bcrossing%2Bover%2Bin%2BDrosophila%2Bfemales.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000080594%26rft_id%253Dinfo%253Apmid%252F15467361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000080594&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15467361&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.11" data-doi="10.1016/j.cub.2018.02.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brand</span>,  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Cattani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. B.</span> <span class="cit-name-surname">Kingan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Landeen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Presgraves</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Molecular evolution at a meiosis gene mediates species differences in the rate and patterning of recombination</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1295.e4</span>. <a href="https://doi.org/10.1016/j.cub.2018.02.056">https://doi.org/10.1016/j.cub.2018.02.056</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D28%26rft.spage%253D1289%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2018.02.056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2018.02.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.12" data-doi="10.1534/genetics.111.136721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Scrambling Eggs: Meiotic Drive and the Evolution of Female Recombination Rates</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">723</span>. <a href="https://doi.org/10.1534/genetics.111.136721">https://doi.org/10.1534/genetics.111.136721</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136721%26rft_id%253Dinfo%253Apmid%252F22143919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNzA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>,  <span class="cit-name-given-names">L. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Marks</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">The organization of genetic variation for recombination in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrooks%26rft.auinit1%253DL.%2BD.%26rft.volume%253D114%26rft.issue%253D2%26rft.spage%253D525%26rft.epage%253D547%26rft.atitle%253DTHE%2BORGANIZATION%2BOF%2BGENETIC%2BVARIATION%2BFOR%2BRECOMBINATION%2BIN%2B%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F3095185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE0LzIvNTI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3201.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>,  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The evolution of recombination rates</span>, pp. <span class="cit-fpage">87</span>–<span class="cit-lpage">105</span> in <span class="cit-source"><em>The Evolution of Sex: An Examination of Current Ideas</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Michod</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Levin</span></span>. <span class="cit-publ-name">Sinauer Associates</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>,  <span class="cit-name-given-names">A. T. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sandler</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">On recombination-defective meiotic mutants in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCarpenter%26rft.auinit1%253DA.%2BT.%2BC.%26rft.volume%253D76%26rft.issue%253D3%26rft.spage%253D453%26rft.epage%253D475%26rft.atitle%253DON%2BRECOMBINATION-DEFECTIVE%2BMEIOTIC%2BMUTANTS%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F4208856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzYvMy80NTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.16" data-doi="10.1038/hdy.1985.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Genetic variation in recombination in <em>Drosophila</em>. I. Responses to selection and preliminary genetic analysis</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">83</span>. <a href="https://doi.org/10.1038/hdy.1985.10">https://doi.org/10.1038/hdy.1985.10</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D54%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1985.10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1985.10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AGR4000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.17" data-doi="10.1038/371215a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sniegowski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The evolutionary dynamics of repetitive DNA in eukaryotes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">371</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">220</span>. <a href="https://doi.org/10.1038/371215a0">https://doi.org/10.1038/371215a0</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D371%26rft.issue%253D6494%26rft.spage%253D215%26rft.epage%253D220%26rft.atitle%253DThe%2Bevolutionary%2Bdynamics%2Bof%2Brepetitive%2BDNA%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F371215a0%26rft_id%253Dinfo%253Apmid%252F8078581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/371215a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8078581&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PG29000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.18" data-doi="10.1017/S0016672300033450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Recombination load associated with selection for increased recombination</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">41</span>. <a href="https://doi.org/10.1017/S0016672300033450">https://doi.org/10.1017/S0016672300033450</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D41%26rft.atitle%253DRecombination%2Bload%2Bassociated%2Bwith%2Bselection%2Bfor%2Bincreased%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033450%26rft_id%253Dinfo%253Apmid%252F8919888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8919888&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UE29800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.19" data-doi="10.1016/j.cub.2018.05.069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Evolution: Increased recombination caused by a single gene</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">R342</span>–<span class="cit-lpage">R344</span>. <a href="https://doi.org/10.1016/j.cub.2018.02.072">https://doi.org/10.1016/j.cub.2018.02.072</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D28%26rft.spage%253DR342%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2018.05.069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2018.05.069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.20" data-doi="10.1101/gr.159384.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Z.-X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sturgill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Qu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jiang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Park</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative validation of the <em>D. melanogaster</em> modENCODE transcriptome annotation</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1223</span>. <a href="https://doi.org/10.1101/gr.159384.113">https://doi.org/10.1101/gr.159384.113</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.159384.113%26rft_id%253Dinfo%253Apmid%252F24985915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTIwOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.21" data-doi="10.1038/ng2049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chintapalli</span>,  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A. T.</span> <span class="cit-name-surname">Dow</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using FlyAtlas to identify better <em>Drosophila melanogaster</em> models of human disease</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">720</span>. <a href="https://doi.org/10.1038/ng2049">https://doi.org/10.1038/ng2049</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChintapalli%26rft.auinit1%253DV.%2BR.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D715%26rft.epage%253D720%26rft.atitle%253DUsing%2BFlyAtlas%2Bto%2Bidentify%2Bbetter%2BDrosophila%2Bmelanogaster%2Bmodels%2Bof%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2049%26rft_id%253Dinfo%253Apmid%252F17534367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17534367&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246859100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.22" data-doi="10.1371/journal.pgen.1002905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ratnappan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bailin</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The many landscapes of recombination in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002905</span>. <a href="https://doi.org/10.1371/journal.pgen.1002905">https://doi.org/10.1371/journal.pgen.1002905</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1002905%26rft.epage%253De1002905%26rft.atitle%253DThe%2BMany%2BLandscapes%2Bof%2BRecombination%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002905%26rft_id%253Dinfo%253Apmid%252F23071443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.23"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Drosophila 12 Genomes Consortium et al., 2007</span> Evolution of genes and genomes on the Drosophila phylogeny. Nature 8: 203–218. <a href="https://doi.org/10.1038/nature06341">https://doi.org/10.1038/nature06341</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>,  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">An advantage of sexual reproduction in a rapidly changing environment</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">173</span>. <a href="https://doi.org/10.1093/oxfordjournals.jhered.a111187">https://doi.org/10.1093/oxfordjournals.jhered.a111187</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DCrow%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D169%26rft.epage%253D173%26rft.atitle%253DAn%2BAdvantage%2Bof%2BSexual%2BReproduction%2Bin%2Ba%2BRapidly%2BChanging%2BEnvironment%26rft_id%253Dinfo%253Apmid%252F1624761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1624761&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JA24300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.25" data-doi="10.1038/nature16931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hatton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Altemose</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Hussin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Pratto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Re-engineering the zinc fingers of PRDM9 reverses hybrid sterility in mice</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">530</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">176</span>. <a href="https://doi.org/10.1038/nature16931">https://doi.org/10.1038/nature16931</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D530%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16931%26rft_id%253Dinfo%253Apmid%252F26840484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26840484&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">Translocations Involving the Third and the Fourth Chromosomes of <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DT.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D347%26rft.epage%253D399%26rft.atitle%253DTRANSLOCATIONS%2BINVOLVING%2BTHE%2BTHIRD%2BAND%2BTHE%2BFOURTH%2BCHROMOSOMES%2BOF%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F17246603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxNS80LzM0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.27" data-doi="10.1073/pnas.79.15.4570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dowsett</span>,  <span class="cit-name-given-names">A. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Young</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Differing levels of dispersed repetitive DNA among closely related species of <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">4570</span>–<span class="cit-lpage">4574</span>. <a href="https://doi.org/10.1073/pnas.79.15.4570">https://doi.org/10.1073/pnas.79.15.4570</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDowsett%26rft.auinit1%253DA.%2BP.%26rft.volume%253D79%26rft.issue%253D15%26rft.spage%253D4570%26rft.epage%253D4574%26rft.atitle%253DDiffering%2Blevels%2Bof%2Bdispersed%2Brepetitive%2BDNA%2Bamong%2Bclosely%2Brelated%2Bspecies%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.15.4570%26rft_id%253Dinfo%253Apmid%252F6956880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMTUvNDU3MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.28" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>. <a href="https://doi.org/10.1093/nar/gkh340">https://doi.org/10.1093/nar/gkh340</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3201.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source"><em>The genetical theory of natural selection</em></span>, <span class="cit-publ-name">Oxford Clarnedon Press. Oxford, UK</span>. <a href="https://doi.org/10.5962/bhl.title.27468">https://doi.org/10.5962/bhl.title.27468</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.31" data-doi="10.1073/pnas.80.16.5017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-Y.</span> <span class="cit-name-surname">Sheen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Gehring</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Green</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Unequal crossing-over associated with asymmetrical synapsis between nomadic elements in the <em>Drosophila melanogaster</em> genome</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">5017</span>–<span class="cit-lpage">5021</span>. <a href="https://doi.org/10.1073/pnas.80.16.5017">https://doi.org/10.1073/pnas.80.16.5017</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGoldberg%26rft.auinit1%253DM.%2BL.%26rft.volume%253D80%26rft.issue%253D16%26rft.spage%253D5017%26rft.epage%253D5021%26rft.atitle%253DUnequal%2Bcrossing-over%2Bassociated%2Bwith%2Basymmetrical%2Bsynapsis%2Bbetween%2Bnomadic%2Belements%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.16.5017%26rft_id%253Dinfo%253Apmid%252F16593354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMTYvNTAxNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grell</span>,  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Time of recombination in the <em>Drosophila melanogaster</em> oocyte. III. Selection and characterization of temperature-sensitive and -insensitive, recombination-deficient alleles in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrell%26rft.auinit1%253DR.%2BF.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D425%26rft.epage%253D443%26rft.atitle%253DTIME%2BOF%2BRECOMBINATION%2BIN%2BTHE%2BDROSOPHILA%2BMELANOGASTER%2BOOCYTE.%2BIII.%2BSELECTION%2BAND%2BCHARACTERIZATION%2BOF%2BTEMPERATURE-SENSITIVE%2BAND%2B-INSENSITIVE%252C%2BRECOMBINATION-DEFICIENT%2BALLELES%2BIN%2BDROSOPHILA%26rft_id%253Dinfo%253Apmid%252F17246234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA4LzIvNDI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>,  <span class="cit-name-given-names">M. T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Aquadro</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">DNA sequence variation and the recombinational landscape in <em>Drosophila pseudoobscura</em>: A study of the second chromosome</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D153%26rft.issue%253D2%26rft.spage%253D859%26rft.epage%253D869%26rft.atitle%253DDNA%2BSequence%2BVariation%2Band%2Bthe%2BRecombinational%2BLandscape%2Bin%2BDrosophila%2Bpseudoobscura%253A%2BA%2BStudy%2Bof%2Bthe%2BSecond%2BChromosome%26rft_id%253Dinfo%253Apmid%252F10511563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzIvODU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.34" data-doi="10.1016/j.cub.2016.10.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hatkevich</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Kohl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">McMahan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Hartmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Williams</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bloom syndrome helicase promotes meiotic crossover patterning and homolog disjunction</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">102</span>. <a href="https://doi.org/10.1016/j.cub.2016.10.055">https://doi.org/10.1016/j.cub.2016.10.055</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D27%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2016.10.055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2016.10.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hatkevich</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sekelsky</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bloom syndrome helicase in meiosis: Pro-crossover functions of an anti-crossover protein</span>. <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1700073</span>. <a href="https://doi.org/10.1002/bies.201700073">https://doi.org/10.1002/bies.201700073</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D39%26rft.spage%253D1700073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.36" data-doi="10.1371/journal.pone.0045055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heil</span>,  <span class="cit-name-given-names">C. S. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. F.</span> <span class="cit-name-surname">Noor</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Zinc finger binding motifs do not explain recombination rate variation within or between species of <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45055</span>. <a href="https://doi.org/10.1371/journal.pone.0045055">https://doi.org/10.1371/journal.pone.0045055</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHeil%26rft.auinit1%253DC.%2BS.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45055%26rft.epage%253De45055%26rft.atitle%253DZinc%2Bfinger%2Bbinding%2Bmotifs%2Bdo%2Bnot%2Bexplain%2Brecombination%2Brate%2Bvariation%2Bwithin%2Bor%2Bbetween%2Bspecies%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045055%26rft_id%253Dinfo%253Apmid%252F23028758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028758&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.10.3201.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hemmer</span>,  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Dias</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Van Vaerenberghe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Howard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The meiotic recombination landscape of <em>Drosophila virilis</em> is robust to mitotic damage during hybrid dysgenesis</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">342824</span>; <a href="https://doi.org/10.1101/342824">https://doi.org/10.1101/342824</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Betancourt</span></span></li></ol><cite>, <span class="cit-pub-date">2018 </span> <span class="cit-article-title">Extensive exchange of transposable elements in the <em>Drosophila</em> pseudoobscuragroup</span>. <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">20</span>. </cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D9%26rft.spage%253D20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.39" data-doi="10.1371/journal.pgen.1005951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>,  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Singh</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic architecture of natural variation in recombination rate in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005951</span>. <a href="https://doi.org/10.1371/journal.pgen.1005951">https://doi.org/10.1371/journal.pgen.1005951</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005951%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005951%26rft_id%253Dinfo%253Apmid%252F27035832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27035832&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.40" data-doi="10.1016/j.cell.2006.07.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">G. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C. H.</span> <span class="cit-name-surname">Franklin</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Meiotic crossing-over: Obligation and interference</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">248</span>. <a href="https://doi.org/10.1016/j.cell.2006.07.010">https://doi.org/10.1016/j.cell.2006.07.010</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DJones%26rft.auinit1%253DG.%2BH.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D246%26rft.epage%253D248%26rft.atitle%253DMeiotic%2Bcrossing-over%253A%2Bobligation%2Band%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.07.010%26rft_id%253Dinfo%253Apmid%252F16873056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16873056&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239552600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.41" data-doi="10.1098/rstb.2016.0458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Uzunović</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Wright</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Coevolution between transposable elements and recombination</span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">372</span>: <span class="cit-fpage">20160458</span>. <a href="https://doi.org/10.1098/rstb.2016.0458">https://doi.org/10.1098/rstb.2016.0458</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0458%26rft_id%253Dinfo%253Apmid%252F29109221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2016.0458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29109221&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>,  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic change of recombination value in <em>Drosophila melanogaster</em>. I. Artificial selection for high and low recombination and some properties of recombination-modifying genes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKidwell%26rft.auinit1%253DM.%2BG.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D419%26rft.epage%253D432%26rft.atitle%253DGENETIC%2BCHANGE%2BOF%2BRECOMBINATION%2BVALUE%2BIN%2BDROSOPHILA%2BMELANOGASTER.%2BI.%2BARTIFICIAL%2BSELECTION%2BFOR%2BHIGH%2BAND%2BLOW%2BRECOMBINATION%2BAND%2BSOME%2BPROPERTIES%2BOF%2BRECOMBINATION-MODIFYING%2BGENES%26rft_id%253Dinfo%253Apmid%252F4623519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzAvMy80MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.43" data-doi="10.1016/S0169-5347(99)01817-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>,  <span class="cit-name-given-names">M. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Lisch</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Transposable elements and host genome evolution</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">99</span>. <a href="https://doi.org/10.1016/S0169-5347(99)01817-0">https://doi.org/10.1016/S0169-5347(99)01817-0</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DKidwell%26rft.auinit1%253DM.%2BG.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D95%26rft.epage%253D99%26rft.atitle%253DTransposable%2Belements%2Band%2Bhost%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252899%252901817-0%26rft_id%253Dinfo%253Apmid%252F10675923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(99)01817-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10675923&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085405300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.44" data-doi="10.1126/science.1228190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohl</span>,  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Jones</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sekelsky</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of an MCM complex in flies that promotes meiotic crossovers by blocking BLM helicase</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">338</span>: <span class="cit-fpage">1363</span>–<span class="cit-lpage">1365</span>. <a href="https://doi.org/10.1126/science.1228190">https://doi.org/10.1126/science.1228190</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKohl%26rft.auinit1%253DK.%2BP.%26rft.volume%253D338%26rft.issue%253D6112%26rft.spage%253D1363%26rft.epage%253D1365%26rft.atitle%253DEvolution%2Bof%2Ban%2BMCM%2BComplex%2Bin%2BFlies%2BThat%2BPromotes%2BMeiotic%2BCrossovers%2Bby%2BBlocking%2BBLM%2BHelicase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1228190%26rft_id%253Dinfo%253Apmid%252F23224558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzgvNjExMi8xMzYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.45" data-doi="10.1534/genetics.113.150581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohl</span>,  <span class="cit-name-given-names">K. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sekelsky</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Meiotic and mitotic recombination in meiosis</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">334</span>. <a href="https://doi.org/10.1534/genetics.113.150581">https://doi.org/10.1534/genetics.113.150581</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKohl%26rft.auinit1%253DK.%2BP.%26rft.volume%253D194%26rft.issue%253D2%26rft.spage%253D327%26rft.epage%253D334%26rft.atitle%253DMeiotic%2Band%2BMitotic%2BRecombination%2Bin%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150581%26rft_id%253Dinfo%253Apmid%252F23733849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.46" data-doi="10.1038/ng.2833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Thorleifsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Frigge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Masson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Gudbjartsson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Common and low-frequency variants associated with genome-wide recombination rate</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">16</span>. <a href="https://doi.org/10.1038/ng.2833">https://doi.org/10.1038/ng.2833</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2833%26rft_id%253Dinfo%253Apmid%252F24270358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24270358&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.47" data-doi="10.1073/pnas.0801848105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Bennett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Fitzpatrick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. F.</span> <span class="cit-name-surname">Noor</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Fine-scale mapping of recombination rate in <em>Drosophila</em> refines its correlation to diversity and divergence</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">10051</span>–<span class="cit-lpage">10056</span>. <a href="https://doi.org/10.1073/pnas.0801848105">https://doi.org/10.1073/pnas.0801848105</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0801848105%26rft_id%253Dinfo%253Apmid%252F18621713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzI5LzEwMDUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.48" data-doi="10.1534/genetics.107.073551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lake</span>,  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Teeter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Page</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Hawley</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A genetic analysis of the <em>Drosophila</em> mcm5 gene defines a domain specifically required for meiotic recombination</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2151</span>–<span class="cit-lpage">2163</span>. <a href="https://doi.org/10.1534/genetics.107.073551">https://doi.org/10.1534/genetics.107.073551</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.073551%26rft_id%253Dinfo%253Apmid%252F17565942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzIxNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.49" data-doi="10.1016/j.tig.2007.12.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larracuente</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. B.</span> <span class="cit-name-surname">Sackton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Greenberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Singh</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution of protein-coding genes in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">123</span>. <a href="https://doi.org/10.1016/j.tig.2007.12.001">https://doi.org/10.1016/j.tig.2007.12.001</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLarracuente%26rft.auinit1%253DA.%2BM.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D114%26rft.epage%253D123%26rft.atitle%253DEvolution%2Bof%2Bprotein-coding%2Bgenes%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.12.001%26rft_id%253Dinfo%253Apmid%252F18249460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18249460&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254058200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y. C. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transposable elements in natural populations of <em>Drosophila melanogaster</em>. Philos. Trans. R. Soc. B Biol</span>. <abbr class="cit-jnl-abbrev">Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1228</span>. <a href="https://doi.org/10.1098/rstb.2009.0318">https://doi.org/10.1098/rstb.2009.0318</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%26rft.volume%253D365%26rft.spage%253D1219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsley</span>,  <span class="cit-name-given-names">D. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sandler</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">The genetic analysis of meiosis in female <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">312</span>. <a href="https://doi.org/10.1098/rstb.1977.0019">https://doi.org/10.1098/rstb.1977.0019</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DLindsley%26rft.auinit1%253DD.%2BL.%26rft.volume%253D277%26rft.issue%253D955%26rft.spage%253D295%26rft.epage%253D312%26rft.atitle%253DThe%2Bgenetic%2Banalysis%2Bof%2Bmeiosis%2Bin%2Bfemale%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F16292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16292&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977CZ41100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Jang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Graham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nycz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">McKim</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Two genes required for meiotic recombination in <em>Drosophila</em> are expressed from a dicistronic message</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1735</span>–<span class="cit-lpage">1746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1735%26rft.epage%253D1746%26rft.atitle%253DTwo%2BGenes%2BRequired%2Bfor%2BMeiotic%2BRecombination%2Bin%2BDrosophila%2BAre%2BExpressed%2BFrom%2Ba%2BDicistronic%2BMessage%26rft_id%253Dinfo%253Apmid%252F10747066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE3MzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.53" data-doi="10.1091/mbc.01-06-0318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manheim</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Jang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dominic</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">McKim</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Cytoplasmic Localization and Evolutionary Conservation of MEI-218, a Protein Required for Meiotic Crossing-over in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">95</span>. <a href="https://doi.org/10.1091/mbc.01-06-0318">https://doi.org/10.1091/mbc.01-06-0318</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.01-06-0318%26rft_id%253Dinfo%253Apmid%252F11809824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjEzLzEvODQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKim</span>,  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Dahmus</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Hawley</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Cloning of the <em>Drosophila melanogaster</em> meiotic recombination gene mei-218: a genetic and molecular analysis of interval 15E</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcKim%26rft.auinit1%253DK.%2BS.%26rft.volume%253D144%26rft.issue%253D1%26rft.spage%253D215%26rft.epage%253D228%26rft.atitle%253DCloning%2Bof%2Bthe%2BDrosophila%2Bmelanogaster%2BMeiotic%2BRecombination%2BGene%2Bmei-218%253A%2BA%2BGenetic%2Band%2BMolecular%2BAnalysis%2Bof%2BInterval%2B15E%26rft_id%253Dinfo%253Apmid%252F8878687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzEvMjE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3201.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehrotra</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Hawley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">McKim</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synapsis, double-strand breaks, and domains of crossover control in <em>Drosophila</em> females</span>, pp. <span class="cit-fpage">125</span>–<span class="cit-lpage">151</span> in <span class="cit-source">Recombination and Meiosis, <em>Genome Dynamics and Stability</em></span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Egel</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">D.-H.</span> <span class="cit-name-surname">Lankenau</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.56" data-doi="10.1534/genetics.115.186486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. Y.</span> <span class="cit-name-surname">Kazemi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Cockrell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Arvanitakis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole-Genome Analysis of Individual Meiotic Events in <em>Drosophila melanogaster</em> Reveals That Noncrossover Gene Conversions Are Insensitive to Interference and the Centromere Effect</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">171</span>. <a href="https://doi.org/10.1534/genetics.115.186486">https://doi.org/10.1534/genetics.115.186486</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.186486%26rft_id%253Dinfo%253Apmid%252F26944917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvMTU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.57" data-doi="10.1017/S0016672300026707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A test for the role of natural selection in the stabilization of transposable element copy number in a population of <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">41</span>. <a href="https://doi.org/10.1017/S0016672300026707">https://doi.org/10.1017/S0016672300026707</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DMontgomery%26rft.auinit1%253DE.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D41%26rft.atitle%253DA%2Btest%2Bfor%2Bthe%2Brole%2Bof%2Bnatural%2Bselection%2Bin%2Bthe%2Bstabilization%2Bof%2Btransposable%2Belement%2Bcopy%2Bnumber%2Bin%2Ba%2Bpopulation%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300026707%26rft_id%253Dinfo%253Apmid%252F3032743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300026707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3032743&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G654900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.58" data-doi="10.1086/279534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>,  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1916</span> <span class="cit-article-title">The mechanism of crossing-over</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">221</span>. <a href="https://doi.org/10.1086/279534">https://doi.org/10.1086/279534</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D50%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200875900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.59" data-doi="10.1126/science.1182363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bowden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tumian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Bontrop</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Freeman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">876</span>–<span class="cit-lpage">879</span>. <a href="https://doi.org/10.1126/science.1182363">https://doi.org/10.1126/science.1182363</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1182363%26rft_id%253Dinfo%253Apmid%252F20044541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84NzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.60" data-doi="10.1371/journal.pgen.1000753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>,  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Goodstadt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Bayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Birtle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. C.</span> <span class="cit-name-surname">Roach</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accelerated evolution of the Prdm9 speciation gene across diverse metazoan taxa</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000753</span>. <a href="https://doi.org/10.1371/journal.pgen.1000753">https://doi.org/10.1371/journal.pgen.1000753</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DOliver%26rft.auinit1%253DP.%2BL.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De1000753%26rft.epage%253De1000753%26rft.atitle%253DAccelerated%2Bevolution%2Bof%2Bthe%2BPrdm9%2Bspeciation%2Bgene%2Bacross%2Bdiverse%2Bmetazoan%2Btaxa.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000753%26rft_id%253Dinfo%253Apmid%252F19997497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000753&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997497&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.61" data-doi="10.1017/S0016672306007932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ortiz-Barrientos</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Chang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. F.</span> <span class="cit-name-surname">Noor</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A recombinational portrait of the <em>Drosophila pseudoobscura</em> genome</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">31</span>. <a href="https://doi.org/10.1017/S0016672306007932">https://doi.org/10.1017/S0016672306007932</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DOrtiz-Barrientos%26rft.auinit1%253DD.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D31%26rft.atitle%253DA%2Brecombinational%2Bportrait%2Bof%2Bthe%2BDrosophila%2Bpseudoobscura%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672306007932%26rft_id%253Dinfo%253Apmid%252F16545148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672306007932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16545148&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236536100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parvanov</span>,  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Petkov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Paigen</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prdm9 controls activation of mammalian recombination hotspots</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">835</span>. <a href="https://doi.org/10.1126/science.1181495">https://doi.org/10.1126/science.1181495</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D12%26rft.spage%253D835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.63" data-doi="10.1038/ng1529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ptak</span>,  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Hinds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Koehler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nickel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Patil</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Fine-scale recombination patterns differ between chimpanzees and humans</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">434</span>. Erratum: 445. <a href="https://doi.org/10.1038/ng1529">https://doi.org/10.1038/ng1529</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPtak%26rft.auinit1%253DS.%2BE.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D429%26rft.epage%253D434%26rft.atitle%253DFine-scale%2Brecombination%2Bpatterns%2Bdiffer%2Bbetween%2Bchimpanzees%2Band%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1529%26rft_id%253Dinfo%253Apmid%252F15723063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15723063&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228040000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritz</span>,  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. F.</span> <span class="cit-name-surname">Noor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Singh</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in Recombination Rate: Adaptive or Not?</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">364</span>–<span class="cit-lpage">374</span>. <a href="https://doi.org/10.1016/j.tig.2017.03.003">https://doi.org/10.1016/j.tig.2017.03.003</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%2BTIG%26rft.volume%253D33%26rft.spage%253D364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.65" data-doi="10.1371/journal.pgen.1002854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandor</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Coppieters</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Druet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Charlier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic variants in REC8, RNF212, and PRDM9 influence male recombination in cattle</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002854</span>. <a href="https://doi.org/10.1371/journal.pgen.1002854">https://doi.org/10.1371/journal.pgen.1002854</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSandor%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002854%26rft.epage%253De1002854%26rft.atitle%253DGenetic%2Bvariants%2Bin%2BREC8%252C%2BRNF212%252C%2Band%2BPRDM9%2Binfluence%2Bmale%2Brecombination%2Bin%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002854%26rft_id%253Dinfo%253Apmid%252F22844258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002854&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22844258&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.66" data-doi="10.1371/journal.pbio.1001211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ségurel</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Leffler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Przeworski</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The case of the fickle fingers: How the PRDM9 zinc finger protein specifies meiotic recombination hotspots in humans</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1001211</span>. <a href="https://doi.org/10.1371/journal.pbio.1001211">https://doi.org/10.1371/journal.pbio.1001211</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSegurel%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253De1001211%26rft.epage%253De1001211%26rft.atitle%253DThe%2Bcase%2Bof%2Bthe%2Bfickle%2Bfingers%253A%2Bhow%2Bthe%2BPRDM9%2Bzinc%2Bfinger%2Bprotein%2Bspecifies%2Bmeiotic%2Brecombination%2Bhotspots%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001211%26rft_id%253Dinfo%253Apmid%252F22162947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22162947&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.67" data-doi="10.1101/gad.270009.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smagulova</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Brick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Pu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Camerini-Otero</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. V.</span> <span class="cit-name-surname">Petukhova</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The evolutionary turnover of recombination hot spots contributes to speciation in mice</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">266</span>–<span class="cit-lpage">280</span>. <a href="https://doi.org/10.1101/gad.270009.115">https://doi.org/10.1101/gad.270009.115</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSmagulova%26rft.auinit1%253DF.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D266%26rft.epage%253D280%26rft.atitle%253DThe%2Bevolutionary%2Bturnover%2Bof%2Brecombination%2Bhot%2Bspots%2Bcontributes%2Bto%2Bspeciation%2Bin%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.270009.115%26rft_id%253Dinfo%253Apmid%252F26833728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMzAvMy8yNjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanley</span>  <span class="cit-name-given-names">C. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Kulathinal</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> flyDIVaS: A Comparative Genomics Resource for <em>Drosophila</em> Divergence and Selection. G3(Bethesda). 6: 2355–2363. <a href="https://doi.org/10.1534/g3.116.031138">https://doi.org/10.1534/g3.116.031138</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.69" data-doi="10.1098/rstb.2016.0455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. G. D.</span> <span class="cit-name-surname">Feulner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Johnston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Santure</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Smadja</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in recombination frequency and distribution across eukaryotes: Patterns and processes</span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">372</span>: <span class="cit-fpage">20160455</span>. Erratum 373: 20170360. <a href="https://doi.org/10.1098/rstb.2016.0455">https://doi.org/10.1098/rstb.2016.0455</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0455%26rft_id%253Dinfo%253Apmid%252F29109219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2016.0455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29109219&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">True</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Mercer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Laurie</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Differences in crossover frequency and distribution among three sibling species of <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTrue%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D142%26rft.issue%253D2%26rft.spage%253D507%26rft.epage%253D523%26rft.atitle%253DDifferences%2Bin%2BCrossover%2BFrequency%2Band%2BDistribution%2BAmong%2BThree%2BSibling%2BSpecies%2Bof%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F8852849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzIvNTA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.71" data-doi="10.1126/science.1134426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venken</span>,  <span class="cit-name-given-names">K. J. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Hoskins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Bellen</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">P[acman]: a BAC transgenic platform for targeted insertion of large DNA fragments in <em>D. melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">1747</span>–<span class="cit-lpage">1751</span>. <a href="https://doi.org/10.1126/science.1134426">https://doi.org/10.1126/science.1134426</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1134426%26rft_id%253Dinfo%253Apmid%252F17138868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTQvNTgwNi8xNzQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.72" data-doi="10.4161/15384101.2014.991185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zickler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kleckner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meiotic crossover patterns: Obligatory crossover, interference and homeostasis in a single process</span>. <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">314</span>. <a href="https://doi.org/10.4161/15384101.2014.991185">https://doi.org/10.4161/15384101.2014.991185</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D14%26rft.spage%253D305%26rft_id%253Dinfo%253Adoi%252F10.4161%252F15384101.2014.991185%26rft_id%253Dinfo%253Apmid%252F25590558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/15384101.2014.991185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25590558&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinstein</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">The theory of multiple-strand crossing over</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWeinstein%26rft.auinit1%253DA.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D155%26rft.epage%253D199%26rft.atitle%253DTHE%2BTHEORY%2BOF%2BMULTIPLE-STRAND%2BCROSSING%2BOVER%26rft_id%253Dinfo%253Apmid%252F17246790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyMS8zLzE1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">PAML: a program package for phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Comput. Appl. Biosci. CABIOS</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BAppl.%2BBiosci.%2BCABIOS%26rft.volume%253D13%26rft.spage%253D555%26rft_id%253Dinfo%253Apmid%252F9367129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9367129&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.75" data-doi="10.1093/molbev/msi097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S. W.</span> <span class="cit-name-surname">Wong</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bayes empirical bayes inference of amino acid sites under positive selection</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1118</span>. <a href="https://doi.org/10.1093/molbev/msi097">https://doi.org/10.1093/molbev/msi097</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi097%26rft_id%253Dinfo%253Apmid%252F15689528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15689528&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228139400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3201.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zalevsky</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">MacQueen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Duffy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Kemphues</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Villeneuve</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Crossing over during <em>Caenorhabditis elegans</em> meiosis requires a conserved MutS-based pathway rhat Is partially dispensable in budding yeast</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1271</span>–<span class="cit-lpage">1283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZalevsky%26rft.auinit1%253DJ.%26rft.volume%253D153%26rft.issue%253D3%26rft.spage%253D1271%26rft.epage%253D1283%26rft.atitle%253DCrossing%2BOver%2BDuring%2BCaenorhabditis%2Belegans%2BMeiosis%2BRequires%2Ba%2BConserved%2BMutS-Based%2BPathway%2BThat%2BIs%2BPartially%2BDispensable%2Bin%2BBudding%2BYeast%26rft_id%253Dinfo%253Apmid%252F10545458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1My8zLzEyNzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3201.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3181" title="Elucidating Drought Stress Tolerance in European Oaks Through Cross-Species Transcriptomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3213" title="Fine Mapping Using Whole-Genome Sequencing Confirms Anti-Müllerian Hormone as a Major Gene for Sex Determination in Farmed Nile Tilapia (Oreochromis niloticus L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3201" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XK-a6_tDjQrYQT-NWVlWLmOh8itHocAk-ms1dBf2XQo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3201" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3201" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3201" class="active" data-icon-position="" data-hide-link-title="0">Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the Drosophila Phylogeny</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494293" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3qJiJ8Ut5DKTDMYSzVD-xtgI89hLOyZDgTRObiPcF-w" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3201.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3201" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-aYAToG4fgCHSO9onENJfLtuHcWYQKsFh857fBJWZoIk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3201" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3201.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494293" id="node494293--2" data-pisa="ggg;9/10/3201" data-pisa-master="ggg;g3.119.400280" data-apath="/ggg/9/10/3201.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the <em>Drosophila</em> Phylogeny</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cara L. Brand</span>, <span class="highwire-citation-author" data-delta="1">Lori Wright</span> and <span class="highwire-citation-author" data-delta="2">Daven C. Presgraves</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3201-3211;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400280 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494293--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cara L. Brand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cara%2BL.%2BBrand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brand%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACara%2BL.%2BBrand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cara.brand@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">cara.brand@rochester.edu</a>
<a href="mailto:daven.presgraves@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">daven.presgraves@rochester.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lori Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lori%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALori%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daven C. Presgraves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daven%2BC.%2BPresgraves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Presgraves%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaven%2BC.%2BPresgraves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cara.brand@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">cara.brand@rochester.edu</a>
<a href="mailto:daven.presgraves@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">daven.presgraves@rochester.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3201" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3201&amp;title=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3201&amp;title=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3201&amp;title=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3201&amp;text=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3201&amp;title=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3201&amp;t=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3201&amp;title=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3201&amp;title=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494293/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494293" id="node494293--3" data-pisa="ggg;9/10/3201" data-pisa-master="ggg;g3.119.400280" data-apath="/ggg/9/10/3201.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Positive Selection and Functional Divergence at Meiosis Genes That Mediate Crossing Over Across the <em>Drosophila</em> Phylogeny</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cara L. Brand</span>, <span class="highwire-citation-author" data-delta="1">Lori Wright</span> and <span class="highwire-citation-author" data-delta="2">Daven C. Presgraves</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3201-3211;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400280 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494293--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cara L. Brand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cara%2BL.%2BBrand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brand%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACara%2BL.%2BBrand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cara.brand@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">cara.brand@rochester.edu</a>
<a href="mailto:daven.presgraves@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">daven.presgraves@rochester.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lori Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lori%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALori%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daven C. Presgraves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Rochester, Rochester, New York, 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daven%2BC.%2BPresgraves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Presgraves%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaven%2BC.%2BPresgraves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cara.brand@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">cara.brand@rochester.edu</a>
<a href="mailto:daven.presgraves@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">daven.presgraves@rochester.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3201&amp;rft.epage=3211&amp;rft.atitle=Positive%20Selection%20and%20Functional%20Divergence%20at%20Meiosis%20Genes%20That%20Mediate%20Crossing%20Over%20Across%20the%20Drosophila%20Phylogeny&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Brand%2C+Cara+L.&amp;rft.au=Wright%2C+Lori&amp;rft.au=Presgraves%2C+Daven+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494293/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494293/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494293/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494293/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494293/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494293/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494293/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494293/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494293/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494293/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494293/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494293/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3201" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3201.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3201.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3201.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
