<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3409" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3409.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3409.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3409.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3409.atom" />
<meta name="HW.pisa" content="ggg;9/10/3409" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris" />
<meta name="DC.Identifier" content="10.1534/g3.119.400357" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Ojeda et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Compared to angiosperms, gymnosperms lag behind in the availability of assembled and annotated genomes. Most genomic analyses in gymnosperms, especially conifer tree species, rely on the use of de novo assembled transcriptomes. However, the level of allelic redundancy and transcript fragmentation in these assembled transcriptomes, and their effect on downstream applications have not been fully investigated. Here, we assessed three assembly strategies for short-reads data, including the utility of haploid megagametophyte tissue during de novo assembly as single-allele guides, for six individuals and five different tissues in Pinus sylvestris . We then contrasted haploid and diploid tissue genotype calls obtained from the assembled transcriptomes to evaluate the extent of paralog mapping. The use of the haploid tissue during assembly increased its completeness without reducing the number of assembled transcripts. Our results suggest that current strategies that rely on available genomic resources as guidance to minimize allelic redundancy are less effective than the application of strategies that cluster redundant assembled transcripts. The strategy yielding the lowest levels of allelic redundancy among the assembled transcriptomes assessed here was the generation of SuperTranscripts with Lace followed by CD-HIT clustering. However, we still observed some levels of heterozygosity (multiple gene fragments per transcript reflecting allelic redundancy) in this assembled transcriptome on the haploid tissue, indicating that further filtering is required before using these assemblies for downstream applications. We discuss the influence of allelic redundancy when these reference transcriptomes are used to select regions for probe design of exome capture baits and for estimation of population genetic diversity." />
<meta name="DC.Contributor" content="Dario I. Ojeda" />
<meta name="DC.Contributor" content="Tiina M. Mattila" />
<meta name="DC.Contributor" content="Tom Ruttink" />
<meta name="DC.Contributor" content="Sonja T. Kujala" />
<meta name="DC.Contributor" content="Katri Kärkkäinen" />
<meta name="DC.Contributor" content="Jukka-Pekka Verta" />
<meta name="DC.Contributor" content="Tanja Pyhäjärvi" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Compared to angiosperms, gymnosperms lag behind in the availability of assembled and annotated genomes. Most genomic analyses in gymnosperms, especially conifer tree species, rely on the use of &lt;i&gt;de novo&lt;/i&gt; assembled transcriptomes. However, the level of allelic redundancy and transcript fragmentation in these assembled transcriptomes, and their effect on downstream applications have not been fully investigated. Here, we assessed three assembly strategies for short-reads data, including the utility of haploid megagametophyte tissue during &lt;i&gt;de novo&lt;/i&gt; assembly as single-allele guides, for six individuals and five different tissues in &lt;i&gt;Pinus sylvestris&lt;/i&gt;. We then contrasted haploid and diploid tissue genotype calls obtained from the assembled transcriptomes to evaluate the extent of paralog mapping. The use of the haploid tissue during assembly increased its completeness without reducing the number of assembled transcripts. Our results suggest that current strategies that rely on available genomic resources as guidance to minimize allelic redundancy are less effective than the application of strategies that cluster redundant assembled transcripts. The strategy yielding the lowest levels of allelic redundancy among the assembled transcriptomes assessed here was the generation of SuperTranscripts with Lace followed by CD-HIT clustering. However, we still observed some levels of heterozygosity (multiple gene fragments per transcript reflecting allelic redundancy) in this assembled transcriptome on the haploid tissue, indicating that further filtering is required before using these assemblies for downstream applications. We discuss the influence of allelic redundancy when these reference transcriptomes are used to select regions for probe design of exome capture baits and for estimation of population genetic diversity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3409" />
<meta name="citation_id" content="9/10/3409" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3409" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3409.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3409.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3409.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400357" />
<meta name="citation_pmid" content="31427456" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3409" />
<meta name="citation_lastpage" content="3421" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dario I. Ojeda" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics," />
<meta name="citation_author_institution" content="Norwegian Institute of Bioeconomy Research, 1433, Ås, Norway," />
<meta name="citation_author_email" content="dario.alayon@nibio.no" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8181-4804" />
<meta name="citation_author" content="Tiina M. Mattila" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1298-7370" />
<meta name="citation_author" content="Tom Ruttink" />
<meta name="citation_author_institution" content="Plant Sciences Unit, Flanders Research Institute for Agriculture, Fisheries and Food (ILVO), 9090 Melle, Belgium," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1012-9399" />
<meta name="citation_author" content="Sonja T. Kujala" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics," />
<meta name="citation_author_institution" content="Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0949-6156" />
<meta name="citation_author" content="Katri Kärkkäinen" />
<meta name="citation_author_institution" content="Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3799-3013" />
<meta name="citation_author" content="Jukka-Pekka Verta" />
<meta name="citation_author_institution" content="Organismal and Evolutionary Biology. University of Helsinki, P.O. Box 3, 00014, Helsinki, Finland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1701-6124" />
<meta name="citation_author" content="Tanja Pyhäjärvi" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics," />
<meta name="citation_author_institution" content="Biocenter Oulu, University of Oulu, P.O.Box 8000, FI-90014, Oulu, Finland," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6958-5172" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Armero;citation_author=L. Baudouin;citation_author=S. Bocs;citation_author=D. This;citation_title=Improving transcriptome de novo assembly by using a reference genome of a related species: Translational genomics from oil palm to coconut.;citation_pages=e0173300;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0173300" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Bao;citation_author=T. Jiang;citation_author=T. Girke;citation_title=BRANCH: Boosting RNA-Seq assemblies with partial or related genomic sequences.;citation_volume=29;citation_year=2013;citation_pmid=23493323;citation_doi=10.1093/bioinformatics/btt127" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Birol;citation_author=A. Raymond;citation_author=S. D. Jackman;citation_author=S. Pleasance;citation_author=R. Coope;citation_title=Assembling the 20 Gb white spruce (Picea glauca) genome from whole-genome shotgun sequencing data.;citation_volume=29;citation_year=2013;citation_pmid=23698863;citation_doi=10.1093/bioinformatics/btt178" />
<meta name="citation_reference" content="citation_journal_title=Comptes rendus biologies;citation_journal_abbrev=C R Biol;citation_author=B. Boeckmann;citation_author=M. C. Blatter;citation_author=L. Famiglietti;citation_author=U. Hinz;citation_author=L. Lane;citation_title=Protein variety and functional diversity: Swiss-Prot annotation in its biological context.;citation_pages=882-899;citation_volume=328;citation_year=2005;citation_issue=10-11;citation_pmid=16286078;citation_doi=10.1016/j.crvi.2005.06.001" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=A. Bolger;citation_author=B. Arsova;citation_author=B. Usadel;citation_title=Plant genome and transcriptome annotations: from misconceptions to simple solutions.;citation_pages=437-449;citation_volume=19;citation_year=2018;citation_doi=10.1093/bib/bbw135" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=D. M. Bryant;citation_author=K. Johnson;citation_author=T. DiTommaso;citation_author=T. Tickle;citation_author=M. B. Couger;citation_title=A tissue-mapped axolotl de novo transcriptome enables identification of limb regeneration factors.;citation_pages=762-776;citation_volume=18;citation_year=2017;citation_pmid=28099853;citation_doi=10.1016/j.celrep.2016.12.063" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Bushmanova;citation_author=D. Antipov;citation_author=A. Lapidus;citation_author=V. Suvorov;citation_author=A. D. Prjibelski;citation_title=rnaQUAST: a quality assessment tool for de novo transcriptome assemblies.;citation_volume=32;citation_year=2016;citation_pmid=27153654;citation_doi=10.1093/bioinformatics/btw218" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=J. Canales;citation_author=R. Bautista;citation_author=P. Label;citation_author=J. Gómez-Maldonado;citation_author=I. Lesur;citation_title=De novo assembly of maritime pine transcriptome: Implications for forest breeding and biotechnology.;citation_pages=286-299;citation_volume=12;citation_year=2014;citation_pmid=24256179;citation_doi=10.1111/pbi.12136" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. M. Celedon;citation_author=M. M. S. Yuen;citation_author=A. Chiang;citation_author=H. Henderson;citation_author=K. E. Reid;citation_title=Cell-type- and tissue-specific transcriptomes of the white spruce (Picea glauca) bark unmask fine-scale spatial patterns of constitutive and induced conifer defense.;citation_pages=710-726;citation_volume=92;citation_year=2017;citation_doi=10.1111/tpj.13673" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=F. Chen;citation_author=W. Dong;citation_author=J. Zhang;citation_author=X. Guo;citation_author=J. Chen;citation_title=The sequenced angiosperm genomes and genome databases.;citation_pages=418;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00418" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=N. M. Davidson;citation_author=A. D. K. Hawkins;citation_author=A. Oshlack;citation_title=SuperTranscripts: A data driven reference for analysis and visualisation of transcriptomes.;citation_pages=148;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1284-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=N. M. Davidson;citation_author=A. Oshlack;citation_title=Corset: enabling differential gene expression analysis for de novo assembled transcriptomes.;citation_pages=410;citation_volume=15;citation_year=2014;citation_pmid=25063469;citation_doi=10.1186/s13059-014-0410-6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: Ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=A. Bateman;citation_author=J. Clements;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_title=Pfam: The protein families database.;citation_volume=42;citation_year=2014;citation_pmid=24288371;citation_doi=10.1093/nar/gkt1223" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Fu;citation_author=Y. Ma;citation_author=H. Yao;citation_author=Z. Xu;citation_author=S. Chen;citation_title=IDP-denovo: De novo transcriptome assembly and isoform annotation by hybrid sequencing.;citation_pages=2168-2176;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty098" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Fu;citation_author=B. Niu;citation_author=Z. Zhu;citation_author=S. Wu;citation_author=W. Li;citation_title=CD-HIT: accelerated for clustering the next generation sequencing data.;citation_volume=28;citation_year=2012;citation_pmid=23060610;citation_doi=10.1093/bioinformatics/bts565" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=P. Gayral;citation_author=J. Melo-Ferreira;citation_author=S. Glémin;citation_author=N. Bierne;citation_author=M. Carneiro;citation_title=Reference-free population genomics from next-generation transcriptome data and the vertebrate-invertebrate gap.;citation_pages=e1003457;citation_volume=9;citation_year=2013;citation_pmid=23593039;citation_doi=10.1371/journal.pgen.1003457" />
<meta name="citation_reference" content="citation_journal_title=Curr. Plant Biol.;citation_author=M. Geniza;citation_author=P. Jaiswal;citation_title=Tools for building de novo transcriptome assembly.;citation_pages=41-45;citation_volume=11–12;citation_year=2017;citation_doi=10.1016/j.cpb.2017.12.004" />
<meta name="citation_reference" content="citation_title=Pine transcriptomics - RNA-Seq data analysis of Scots pine (Pinus sylvestris) seedlings subjected to a wounding experiment;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=Z. Gompert;citation_author=K. Mock;citation_title=Detection of individual ploidy levels with genotyping-by-sequencing (GBS) analysis.;citation_pages=1156-1167;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12657" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=D. Gonzales-Ibeas;citation_author=P. J. Martinez-Garcia;citation_author=R. Famula;citation_author=A. Delfino-Mix;citation_title=Assessing the gene content of the megagenome: sugar pine (Pinus lambertiana). Genes, Genomes;citation_pages=3787-3802;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. Grivet;citation_author=K. Avia;citation_author=A. Vaattovaara;citation_author=A. J. Eckert;citation_author=D. B. Neale;citation_title=High rate of adaptive evolution in two widespread European pines.;citation_pages=6857-6870;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14402" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. A. Hodgins;citation_author=S. Yeaman;citation_author=K. A. Nurkowski;citation_author=L. H. Rieseberg;citation_author=S. N. Aitken;citation_title=Expression divergence is correlated with sequence evolution but not positive selection in conifers.;citation_volume=33;citation_year=2016;citation_pmid=26873578;citation_doi=10.1093/molbev/msw032" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mol. Sci.;citation_author=B. Höllbacher;citation_author=A. O. Schmitt;citation_author=H. Hofer;citation_author=F. Ferreira;citation_author=P. Lackner;citation_title=Identification of proteases and protease inhibitors in allergenic and non-allergenic pollen.;citation_pages=1199;citation_volume=18;citation_year=2017;citation_doi=10.3390/ijms18061199" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=M. Hölzer;citation_author=M. Marz;citation_title=De novo transcriptome assembly: A comprehensive cross-species comparison of short-read RNA-Seq assemblers.;citation_pages=giz039;citation_volume=8;citation_year=2019;citation_doi=10.1093/gigascience/giz039" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=G. T. Howe;citation_author=J. Yu;citation_author=B. Knaus;citation_author=R. Cronn;citation_author=S. Kolpak;citation_title=A SNP resource for Douglas-fir: de novo transcriptome assembly and SNP detection and validation.;citation_pages=137-137;citation_volume=14;citation_year=2013;citation_issue=1;citation_pmid=23445355;citation_doi=10.1186/1471-2164-14-137" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. G. Hu;citation_author=H. Liu;citation_author=Y. Q. Jin;citation_author=Y. Q. Sun;citation_author=Y. Li;citation_title=De novo transcriptome assembly and characterization for the widespread and stress-tolerant conifer Platycladus orientalis.;citation_pages=e0148985;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0148985" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT — The BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. Kujala;citation_author=O. Savolainen;citation_title=Sequence variation patterns along a latitudinal cline in Scots pine (Pinus sylvestris): Signs of clinal adaptation?;citation_pages=1451-1467;citation_volume=8;citation_year=2012;citation_doi=10.1007/s11295-012-0532-5" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. R. De La Torre;citation_author=Z. Li;citation_author=Y. Van De Peer;citation_author=P. K. Ingvarsson;citation_title=Contrasting rates of molecular evolution and patterns of selection among gymnosperms and flowering plants.;citation_pages=1363-1377;citation_volume=34;citation_year=2017;citation_doi=10.1093/molbev/msx069" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. R. De La Torre;citation_author=Y. C. Lin;citation_author=Y. Van De Peer;citation_author=P. K. Ingvarsson;citation_title=Genome-wide analysis reveals diverged patterns of codon bias, gene expression, and rates of sequence evolution in Picea gene families.;citation_volume=7;citation_year=2015;citation_pmid=25747252;citation_doi=10.1093/gbe/evv044" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=Z. Li;citation_author=A. E. Baniaga;citation_author=E. B. Sessa;citation_author=M. Scascitelli;citation_author=S. W. Graham;citation_title=Early genome duplications in conifers and other seed plants;citation_pages=e1501084-e1501084;citation_volume=1;citation_year=2015;citation_issue=10;citation_pmid=26702445;citation_doi=10.1126/sciadv.1501084" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Li;citation_author=A. Godzik;citation_title=Cd-hit: A fast program for clustering and comparing large sets of protein or nucleotide sequences.;citation_volume=22;citation_year=2006;citation_pmid=16731699;citation_doi=10.1093/bioinformatics/btl158" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence alignment/map (SAM) format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=W. Li;citation_author=L. Jaroszewski;citation_author=A. Godzik;citation_title=Clustering of highly homologous sequences to reduce the size of  large protein databases;citation_pages=282-283;citation_volume=17;citation_year=2001;citation_issue=3;citation_pmid=11294794;citation_doi=10.1093/bioinformatics/17.3.282" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Z. Li;citation_author=A. R. De La Torre;citation_author=L. Sterck;citation_author=F. M. Cánovas;citation_author=C. Avila;citation_title=Single-copy genes as molecular markers for phylogenomic studies in seed plants.;citation_pages=1130-1147;citation_volume=9;citation_year= 2017;citation_doi=10.1093/gbe/evx070" />
<meta name="citation_reference" content="citation_title=Scots pine (Pinus sylvestris L.) heartwood formation and wounding stress: A view from the transcriptome;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=K. Lim;citation_author=T. Paasela;citation_author=A. Harju;citation_author=M. Venäläinen;citation_author=L. Paulin;citation_title=Developmental changes in scots pine transcriptome during heartwood formation.;citation_volume=172;citation_year=2016;citation_pmid=27600814;citation_doi=10.1104/pp.16.01082" />
<meta name="citation_reference" content="citation_journal_title=Comput. Biol. J.;citation_author=M. Liu;citation_author=Z. Adelman;citation_author=K. Myles;citation_author=L. Zhang;citation_title=A transcriptome post-scaffolding method for assembling high quality contigs.;citation_pages=1-4;citation_volume=2014;citation_year=2014;citation_doi=10.1155/2014/961823" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=U. López de Heredia;citation_author=J. L. Vázquez-Poletti;citation_title=RNA-seq analysis in forest tree species: bioinformatic problems and solutions.;citation_pages=30;citation_volume=12;citation_year=2016;citation_doi=10.1007/s11295-016-0995-x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Marchet;citation_author=L. Lecompte;citation_author=C. D. Silva;citation_author=C. Cruaud;citation_author=J. M. Aury;citation_title=De novo clustering of long reads by gene from transcriptomics data.;citation_pages=e2;citation_volume=47;citation_year=2019;citation_doi=10.1093/nar/gky834" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=G. R. A. Margarido;citation_author=D. Heckerman;citation_title=ConPADE : Genome assembly ploidy estimation from next-generation sequencing data.;citation_pages=e1004229;citation_volume=11;citation_year=2015;citation_doi=10.1371/journal.pcbi.1004229" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=G. J. McKinney;citation_author=R. K. Waples;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Paralogs are revealed by proportion of heterozygotes and deviations in read ratios in genotyping-by-sequencing data from natural populations.;citation_pages=656-669;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12613" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Glutton: large-scale integration of non-model organism transcriptome data for comparative analysis.;citation_pages=077511;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=I. Merino;citation_author=M. Abrahamsson;citation_author=L. Sterck;citation_author=B. Craven-Bartle;citation_author=F. Canovas;citation_title=Transcript profiling for early stages during embryo development in Scots pine.;citation_pages=255;citation_volume=16;citation_year=2016;citation_doi=10.1186/s12870-016-0939-5" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=T. Müller;citation_author=F. Freund;citation_author=H. Wildhagen;citation_author=K. J. Schmid;citation_title=Targeted re-sequencing of five Douglas-fir provenances reveals population structure and putative target genes of positive selection.;citation_pages=816;citation_volume=11;citation_year=2015;citation_doi=10.1007/s11295-014-0816-z" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. B. Neale;citation_author=P. E. McGuire;citation_author=N. C. Wheeler;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_title=The Douglas-fir genome sequence reveals specialization of the photosynthetic apparatus in Pinaceae.;citation_volume=7;citation_year=2017;citation_pmid=28751502;citation_doi=10.1534/g3.117.300078" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. B. Neale;citation_author=J. L. Wegrzyn;citation_author=K. A. Stevens;citation_author=A. V. Zimin;citation_author=D. Puiu;citation_title=DECODING THE MASSIVE GENOME OF LOBLOLLY PINE USING HAPLOID DNA AND NOVEL ASSEMBLY STRATEGIES.;citation_pages=R59;citation_volume=15;citation_year=2014;citation_pmid=24647006;citation_doi=10.1186/gb-2014-15-3-r59" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=L. G. Neves;citation_author=J. M. Davis;citation_author=W. B. Barbazuk;citation_author=M. Kirst;citation_title=Whole-exome targeted sequencing of the uncharacterized pine genome.;citation_pages=146-156;citation_volume=75;citation_year=2013;citation_pmid=23551702;citation_doi=10.1111/tpj.12193" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Nystedt;citation_author=N. R. Street;citation_author=A. Wetterbom;citation_author=A. Zuccolo;citation_author=Y. C. Lin;citation_title=The Norway spruce genome sequence and conifer genome evolution.;citation_pages=579-584;citation_volume=497;citation_year=2013;citation_pmid=23698360;citation_doi=10.1038/nature12211" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Ono;citation_author=K. Ishii;citation_author=T. Kozaki;citation_author=I. Ogiwara;citation_author=M. Kanekatsu;citation_title=Removal of redundant contigs from de novo RNA-Seq assemblies via homology search improves accurate detection of differentially expressed genes.;citation_pages=1031;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2247-0" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. L. Parchman;citation_author=K. S. Geist;citation_author=J. A. Grahnen;citation_author=C. W. Benkman;citation_author=C. A. Buerkle;citation_title=Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery.;citation_pages=180-180;citation_volume=11;citation_year=2010;citation_pmid=20233449;citation_doi=10.1186/1471-2164-11-180" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=G. Pertea;citation_author=X. Huang;citation_author=F. Liang;citation_author=V. Antonescu;citation_author=R. Sultana;citation_title=TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets;citation_pages=651-652;citation_volume=19;citation_year=2003;citation_issue=5;citation_pmid=12651724;citation_doi=10.1093/bioinformatics/btg034" />
<meta name="citation_reference" content="citation_journal_title=Mill. Mol. Ecol. Resour.;citation_author=S. Pinosio;citation_author=S. C. González-Martínez;citation_author=F. Bagnoli;citation_author=F. Cattonaro;citation_author=D. Grivet;citation_title=First insights into the transcriptome and development of new genomic tools of a widespread circum-Mediterranean tree species, Pinus halepensis;citation_pages=846-856;citation_volume=14;citation_year=2014;citation_pmid=24450970;citation_doi=10.1111/1755-0998.12232" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=I. Porth;citation_author=R. White;citation_author=B. Jaquish;citation_author=K. Ritland;citation_title=Partial correlation analysis of transcriptomes helps detangle the growth and defense network in spruce.;citation_pages=1349-1359;citation_volume=218;citation_year=2018;citation_doi=10.1111/nph.15075" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Pyhäjärvi;citation_author=M. R. García-Gil;citation_author=T. Knürr;citation_author=M. Mikkonen;citation_author=W. Wachowiak;citation_title=Demographic history has influenced nucleotide diversity in European Pinus sylvestris populations.;citation_pages=1713-1724;citation_volume=177;citation_year=2007;citation_pmid=18039881;citation_doi=10.1534/genetics.107.077099" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=T. Pyhäjärvi;citation_author=S. T. Kujala;citation_author=O. Savolainen;citation_title=Revisiting protein heterozygosity in plants-nucleotide diversity in allozyme coding genes of conifer Pinus sylvestris.;citation_pages=385-397;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-010-0340-8" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Raherison;citation_author=P. Rigault;citation_author=S. Caron;citation_author=P. L. Poulin;citation_author=B. Boyle;citation_title=Transcriptome profiling in conifers and the PiceaGenExpress database show patterns of diversification within gene families and interspecific conservation in vascular gene expression.;citation_pages=434-434;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22931377;citation_doi=10.1186/1471-2164-13-434" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=A. Roberts;citation_author=L. Pachter;citation_title=Streaming fragment assignment for real-time analysis of sequencing experiments.;citation_pages=71-73;citation_volume=10;citation_year=2013;citation_pmid=23160280;citation_doi=10.1038/nmeth.2251" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Romiguier;citation_author=P. Gayral;citation_author=M. Ballenghien;citation_author=A. Bernard;citation_author=V. Cahais;citation_title=Comparative population genomics in animals uncovers the determinants of genetic diversity.;citation_pages=261-263;citation_volume=515;citation_year=2014;citation_pmid=25141177;citation_doi=10.1038/nature13685" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=T. Ruttink;citation_author=L. Sterck;citation_author=A. Rohde;citation_author=C. Bendixen;citation_author=P. Rouzé;citation_title=Orthology guided assembly in highly heterozygous crops: creating a reference transcriptome to uncover genetic diversity in Lolium perenne.;citation_pages=605-617;citation_volume=11;citation_year=2013;citation_pmid=23433242;citation_doi=10.1111/pbi.12051" />
<meta name="citation_reference" content="citation_journal_title=Evol. Lett.;citation_author=G. Sandler;citation_author=F. E. G. Beaudry;citation_author=S. C. H. Barrett;citation_author=S. I. Wright;citation_title=The effects of haploid selection on Y chromosome evolution in two closely related dioecious plants.;citation_pages=368-377;citation_volume=2;citation_year=2018;citation_doi=10.1002/evl3.60" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Smith-Unna;citation_author=C. Boursnell;citation_author=R. Patro;citation_author=J. M. Hibberd;citation_author=S. Kelly;citation_title=TransRate: reference-free quality assessment of de novo transcriptome assemblies.;citation_volume=26;citation_year=2016;citation_pmid=27252236;citation_doi=10.1101/gr.196469.115" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=S. A. Smith;citation_author=M. J. Moore;citation_author=J. W. Brown;citation_author=Y. Yang;citation_title=Analysis of phylogenomic datasets reveals conflict, concordance, and gene duplications with examples from animals and plants.;citation_pages=150;citation_volume=15;citation_year=2015;citation_pmid=26239519;citation_doi=10.1186/s12862-015-0423-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. A. Stevens;citation_author=J. L. Wegrzyn;citation_author=A. Zimin;citation_author=D. Puiu;citation_author=M. Crepeau;citation_title=Sequence of the sugar pine megagenome.;citation_volume=204;citation_year=2016;citation_pmid=27794028;citation_doi=10.1534/genetics.116.193227" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=Š. Stočes;citation_author=T. Ruttink;citation_author=J. Bartoš;citation_author=B. Studer;citation_author=S. Yates;citation_title=Orthology guided transcriptome assembly of Italian ryegrass and meadow fescue for single-nucleotide polymorphism discovery.;citation_pages=1-14;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2015.04.0025" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=H. Suren;citation_author=K. A. Hodgins;citation_author=S. Yeaman;citation_author=K. A. Nurkowski;citation_author=P. Smets;citation_title=Exome capture from the spruce and pine giga-genomes.;citation_pages=1136-1146;citation_volume=16;citation_year=2016;citation_doi=10.1111/1755-0998.12570" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Surget-Groba;citation_author=J. I. Montoya-Burgos;citation_title=Optimization of de novo transcriptome assembly from next-generation sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20693479;citation_doi=10.1101/gr.103846.109" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. V. Syring;citation_author=J. A. Tennessen;citation_author=T. N. Jennings;citation_author=J. Wegrzyn;citation_author=C. Scelfo-Dalbey;citation_title=Targeted capture sequencing in whitebark pine reveals range-wide demographic and adaptive patterns despite challenges of a large, repetitive genome.;citation_pages=484;citation_volume=7;citation_year=2016;citation_doi=10.3389/fpls.2016.00484" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_pmid=2513255;citation_doi=PMC1203831" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=UniProt: a hub for protein information.;citation_volume=43;citation_year=2015;citation_pmid=25348405;citation_doi=10.1093/nar/gku989" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=D. Uddenberg;citation_author=S. Akhter;citation_author=P. Ramachandran;citation_author=J. F. Sundström;citation_author=A. Carlsbecker;citation_title=Sequenced genomes and rapidly emerging technologies pave the way for conifer evolutionary developmental biology.;citation_pages=00970;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.00970" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=S. Ueno;citation_author=Y. Nakamura;citation_author=M. Kobayashi;citation_author=S. Terashima;citation_author=W. Ishizuka;citation_title=TodoFirGene: Developing transcriptome resources for genetic analysis of Abies sachalinensis.;citation_pages=1276-1284;citation_volume=59;citation_year=2018;citation_doi=10.1093/pcp/pcy058" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. P. Verta;citation_author=C. R. Landry;citation_author=J. Mackay;citation_title=Dissection of expression‐quantitative trait locus and allele specificity using a haploid/diploid plant system – insights into compensatory evolution of transcriptional regulation within populations.;citation_pages=159-171;citation_volume=211;citation_year=2016;citation_doi=10.1111/nph.13888" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=E. A. Visser;citation_author=J. L. Wegrzyn;citation_author=A. A. Myburg;citation_author=S. Naidoo;citation_title=Defence transcriptome assembly and pathogenesis related gene family analysis in Pinus tecunumanii (low elevation).;citation_pages=632;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-5015-0" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=W. Wachowiak;citation_author=U. Trivedi;citation_author=A. Perry;citation_author=S. Cavers;citation_title=Comparative transcriptomics of a complex of four European pine species.;citation_pages=234;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1401-z" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=T. Wan;citation_author=Z. M. Liu;citation_author=L. F. Li;citation_author=A. R. Leitch;citation_author=I. J. Leitch;citation_title=A genome for gnetophytes and early evolution of seed plants.;citation_pages=82-89;citation_volume=4;citation_year=2018;citation_doi=10.1038/s41477-017-0097-2" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Q. Xu;citation_author=C. Zhu;citation_author=Y. Fan;citation_author=Z. Song;citation_author=S. Xing;citation_title=Population transcriptomics uncovers the regulation of gene expression variation in adaptation to changing environment.;citation_pages=25536;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep25536" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=T. Yahav;citation_author=E. Privman;citation_title=A comparative analysis of methods for de novo assembly of hymenopteran genomes using either haploid or diploid samples.;citation_pages=6480;citation_volume=9;citation_year=2019;citation_doi=10.1038/s41598-019-42795-6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Yan;citation_author=Z. Song;citation_author=Q. Xu;citation_author=L. Kang;citation_author=C. Zhu;citation_title=Population transcriptomic characterization of the genetic and expression variation of a candidate progenitor of Miscanthus energy crops.;citation_pages=5911-5922;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14338" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Yang;citation_author=S. A. Smith;citation_title=Optimizing de novo assembly of short-read RNA-seq data for phylogenomics.;citation_pages=328;citation_volume=14;citation_year=2013;citation_pmid=23672450;citation_doi=10.1186/1471-2164-14-328" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. Yang;citation_author=S. A. Smith;citation_title=Orthology inference in nonmodel organisms using transcriptomes and low-coverage genomes: Improving accuracy and matrix occupancy for phylogenomics.;citation_volume=31;citation_year=2014;citation_pmid=25158799;citation_doi=10.1093/molbev/msu245" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=L. Zhao;citation_author=H. Zhang;citation_author=M. V. Kohnen;citation_author=K. V. S. K. Prasad;citation_author=L. Gu;citation_title=Analysis of transcriptome and epitranscriptome in plants using PacBio Iso-Seq and nanopore-based direct RNA sequencing.;citation_pages=253;citation_volume=10;citation_year=2019;citation_doi=10.3389/fgene.2019.00253" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Zimin;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_author=A. Holtz-Morris;citation_author=M. Koriabine;citation_title=Sequencing and Assembly of the 22-Gb Loblolly Pine Genome;citation_pages=875-890;citation_volume=196;citation_year=2014;citation_issue=3;citation_pmid=24653210;citation_doi=10.1534/genetics.113.159715" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Compared to angiosperms, gymnosperms lag behind in the availability of assembled and annotated genomes. Most genomic analyses in gymnosperms, especially conifer tree species, rely on the use of de novo assembled transcriptomes. However, the level of allelic redundancy and transcript fragmentation in these assembled transcriptomes, and their effect on downstream applications have not been fully investigated. Here, we assessed three assembly strategies for short-reads data, including the utility of haploid megagametophyte tissue during de novo assembly as single-allele guides, for six individuals and five different tissues in Pinus sylvestris . We then contrasted haploid and diploid tissue genotype calls obtained from the assembled transcriptomes to evaluate the extent of paralog mapping. The use of the haploid tissue during assembly increased its completeness without reducing the number of assembled transcripts. Our results suggest that current strategies that rely on available genomic resources as guidance to minimize allelic redundancy are less effective than the application of strategies that cluster redundant assembled transcripts. The strategy yielding the lowest levels of allelic redundancy among the assembled transcriptomes assessed here was the generation of SuperTranscripts with Lace followed by CD-HIT clustering. However, we still observed some levels of heterozygosity (multiple gene fragments per transcript reflecting allelic redundancy) in this assembled transcriptome on the haploid tissue, indicating that further filtering is required before using these assemblies for downstream applications. We discuss the influence of allelic redundancy when these reference transcriptomes are used to select regions for probe design of exome capture baits and for estimation of population genetic diversity." />
<meta name="og-title" property="og:title" content="Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3409" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Compared to angiosperms, gymnosperms lag behind in the availability of assembled and annotated genomes. Most genomic analyses in gymnosperms, especially conifer tree species, rely on the use of de novo assembled transcriptomes. However, the level of allelic redundancy and transcript fragmentation in these assembled transcriptomes, and their effect on downstream applications have not been fully investigated. Here, we assessed three assembly strategies for short-reads data, including the utility of haploid megagametophyte tissue during de novo assembly as single-allele guides, for six individuals and five different tissues in Pinus sylvestris . We then contrasted haploid and diploid tissue genotype calls obtained from the assembled transcriptomes to evaluate the extent of paralog mapping. The use of the haploid tissue during assembly increased its completeness without reducing the number of assembled transcripts. Our results suggest that current strategies that rely on available genomic resources as guidance to minimize allelic redundancy are less effective than the application of strategies that cluster redundant assembled transcripts. The strategy yielding the lowest levels of allelic redundancy among the assembled transcriptomes assessed here was the generation of SuperTranscripts with Lace followed by CD-HIT clustering. However, we still observed some levels of heterozygosity (multiple gene fragments per transcript reflecting allelic redundancy) in this assembled transcriptome on the haploid tissue, indicating that further filtering is required before using these assemblies for downstream applications. We discuss the influence of allelic redundancy when these reference transcriptomes are used to select regions for probe design of exome capture baits and for estimation of population genetic diversity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0eoqsRxq9fQYkfJzzxRlHU2X5sijqOncPMp66ik_DHs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494261","apath":"\/ggg\/9\/10\/3409.atom","pisa":"ggg;9\/10\/3409","ac":{"\/ggg\/9\/10\/3409.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3409.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3409"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3409"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3409.figures-only":true,"\/content\/9\/10\/3409":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SZi6u5WOG42hc8pPgdbbbpyuiiPy71agUv8ccL18np5zpR3Z2UC25XpJ3VotymYBV+NmvCqe\\\\\\\/tsOIgx0UhWYiO8Y5SIeU3Lv8lkBd9MALQKpHCmcqLr9q1n8wxb5qmCSqPTCXQhtHxVu4COV4p5u6Ys6vKSWGtx1iy4PAcjr3Q3q4b72VCszbNMwYvySjFTL4ZWvezKqLQu0L\\\\\\\/vZ6Gk2AZpqMqvq+3LqSVFJZZJevo72GYSR9XsBIACcV3hyTtVG3RiEqIgL7+wiWeKdSSgBi09pmq57lGfQxy+x6YVK99kGIg9eUCaPmXDdCPnbUVzLlRh31m7gQhPuGgxFWrHihF2E+Qtf726DnurSRBF\\\\\\\/ugxB4BMedVrbEcdyNkXqb5WNkb4UT8V90laGd9A6lE7RNBZkXxy118JdisyZvanMOV78Cn\\\\\\\/nZfIwWutbhWeWbBlRfA+GZMX3Z+HDLZgIOlFDHF7x09fqLFF4+WdP2iRXjPcl1xDfdEgC5FPIAg1d9WyRhCYn7b8MwE8tjn1M5uwYztYDN4NN7gGW\\\\\\\/IyzUvrERGr\\\\\\\/9GIPg4\\\\\\\/F8SMVqAXd9fzSa0Om6SqPEhpnN86Y96wUAHdOpOjJ8Uwjwupf7uyotcbzdWzAnzvw3AZ96JTv3r2AlSZlb+YwU8RqelXtYedS\\\\\\\/0El7DcU75cBO836Uia8btxTAca2ims6lRKhmO6l8K09Z72iyWraVptkxhgT1Xt8kltoaBlIqrktD\\\\\\\/DY6utXq+SAbn46aONPZ2mCHURrA7IMUBx8Nlsp62hZFk\\\\\\\/AEuY4NfDTb9i\\\\\\\/rQcQPvupHKJbQkG2sfICqWnIGSwoheAT7o3WE68FugbcQtrOz6DroGhwL3wVQ7dNHodJt5NkZNUq1h8+5G9FD7wygLYSGmba0K7o5dGCNRy4mhP0Bn9cip+tj+7PjJz2AFmu3wk0+fk0xSE5quWgm0fs12eS5pYJQxBdyT0WHdRRur7CIBZ67pX\\\\\\\/RPV0Qq5aBYnz4ffIQ25IJKiu0NEgS69tCZDqPPSkNzszwOgkE2psT+HptmBeYdf43MUfHUU\\\\\\\/BkEkBt0lhBHKdxS4O1DjZHWh+at8ofCAxB8rkCa6tW2J+3dlMjlnCGrkHvmn00Jscg\\\\\\\/J36jv+ijO846AY3I\\\\\\\/CLOxR\\\\\\\/98AvSoFprPpzfmFMQS+kLYsVxyZRXEv6pawa3htmaf9dMygEI1G8PBeb259WbG1HkxVsHXcUMbpUtalBNDxTLJPAiZsKCYj33T83LXsb4HAmRJg6BVL6uSyioBoQ1RmAwYhpOCvkT6\\\u0022,\\\u0022iv\\\u0022:\\\u0022+a83CRx1k0QjL3qEBwMgpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022acb6b9fd9836641f2db720084beecebced2d60cf2bd3431c8d59fabf46a13180\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vy5eaezHZMu+yiryIfzQoz4EvqZTWJK4NqxO3E4SgjzBKWVSrEso7XNfihUAopUo6aJ9MSrh9bxlteEUaFxr9vPMpXmmPUb08U67Up6\\\\\\\/kXTYHasD2ath5gsq7KR1VP2haF7aw7S6PvkFnXUZewsVZTdaxS1Nhw5\\\\\\\/nIB0rR76sjq1rhgRhpNT15gIvQKTrvM0EmbEUXeugC6hum6ltor49o7W7fn9ImA3VJO+THow5zknCOXUfJ0wagoXpL3+W4ZZkmst0KC8+zJ9u0cv2ddXbNKN4xEG5PFKVPWxBoniMM97I80WlbH\\\\\\\/N1cYB3LXQs4t3H6HwENSTCiv9Yp07I9xpRVI\\\\\\\/g+DAj9rZHF2y0+lyMmRI65DOU0XbIUdpfZ\\\\\\\/FIzxKTe2aadUKJxkjt0oNeluquBx5ndPvcMijrvjPtsXSYgOaKGjLKm2LdMqWjFuW8NuXuPYI9XNetQPviizbHf5eO7Gbg9lObOyYgK9HpP3L9bk+PHujyfx3jhMFQgp03iB2F3eNSPYN+yh2HhoEG4CfPhGNiCj9iUK6qOx65bjtLnx3aDMvma77QQWI8b9eVLxEbOzoba8S2riSNd1X6PA2xs0qU+CRHLnPYPstI8szqBXh8vYbjS7u7uk8bHoVeOqkVn2NDyvuodWj0PDzuu1TsEZJ0PjKw+K\\\\\\\/HHMO0pUSCQIbtr1sVXjuGHXcAzYbMooB9RCJPdNbwAAs5NvKK1q\\\\\\\/c0RiKL50olNcPwrcms5RGbM014K124LByiAWeQ\\\\\\\/ioinNHpveeHHEbtfZ\\\\\\\/eDkoQ6i4iL7l+p6OoO+YgX711oB8g41Jfrr6G9NcVkDWJ37EFspp3SXp7lF8oadm4oirGspaSAwdfzHMSwThbFK93AjGzcmpdsCaF\\\\\\\/bi2lbxgEDmf8vNUpLVyNsktw69b170X7U02k8BbT+ITzXZ6AyIJxqUwpXXyrX6vOzIvov7PnrfLG4lC0CAzdCiKkDquw+gEvz1kfee+4PMa8aP3v\\\\\\\/xgDn\\\\\\\/teqEYP2AQp8MO9ojwwRtHUC6EjxrE\\\\\\\/cWS+Kv1uy944zEFAQrnz1OIU1262yrH7K4dwvkpGnGUmLCpSMbMrny8\\\\\\\/GDNw20ZV4i1VZRxOvto+PorIr0kQE07MUO6Wivh9dcO+NJaEnd0FTll4fh8n1+VDRUbf5y5uRPjyTgH6Y5LV75ovdKtFkOWnta3FU8eZuIXoDJZDWbMH1gAoOX+PwsLl5n1c5pOi6KjXp\\\\\\\/jJO6hjc+aJ26m8WfN+hb1cG4vjZVBuF8iBNfV89m6tZsK079515HVvh34A4E2uQGqukNflHVK2czmTyNVjJaEKj2MLt2\\\\\\\/+ZW4aByVbemy2IRF\\\\\\\/x6SMT1goIwl6icg2zT\\\\\\\/j4BE0cN7ADF3U02AoPjERL\\\\\\\/xP97jhgmnO2xn4IGNxYiG1FwSCAXt\\\\\\\/eMzCXWngfMQ8X\\\\\\\/Tdt9nUtdHGq3965cZpzKe5hKQw1DkSAfntOATVff1qk87LaRMk3pGEF+\\\\\\\/YVt+LALqveNMtrQQlVdLW0mBRMe1aw3BPJ2zEP8jc\\\\\\\/zOC3UtvGlwFyDh2M127zmlJCadcWh3MWrO7nb05ZBvcrq2Ig0xuOGcezpH3F2olkzr+lNQARDS7e6NNjN6SjL3rd+GiOaNpAUgS0d\\\\\\\/LO6KkQkgEtKBIUySdGREhQnshGjfF4cp9FLIRT\\\\\\\/Ah550Xx1IemldbbGEa6cUeKwyOrryug\\\\\\\/mgV4t07prVhnmT17T94Qwr1JwNKZCck8qVjsIMUFiDg2Fn+JqA4R4NFIlJZuOloveHzylw4cNYc0lQ3RaHjQxEYfgd63Cn\\\\\\\/Ah17NahC4IrrIGkluumS4dBlwgBs+10jVnS\\\\\\\/FWWBn8IfE5P0h2JGD2P70UjytbNufyCj0H2tCrtqNb+B8v1ewk8ApP4YHfW+3Daj9vFRmh3AZsa\\\\\\\/9MKM2gxOYORFJWDplTx5UH0o6J5OOMPmIjf7dwOD907siZHJPLDSRrEpauKY8Uv9gGug6ZXmN3QtNlcqIwQc+\\\\\\\/ERKxMlETNOhbdwVBtsCpBaKT9GotEi07\\\\\\\/M2DXCKyMshD8VpJWZmQDXAExRu+AGT+KY+UZv6id50Ngz2MP520K1BFPq7Of6BBU7VCNKUXszWrV4n9\\\\\\\/e7xbQHiCr5+WI1apLrhWa2do1zGJ8AGncR5LkInlHZbduvKfRQDmCucUqGLl77s+5QUGltAAXmgkg6fOFaDe\\\\\\\/lUTKI\\\\\\\/d+\\\\\\\/63YBJRSGlBLZpgXxszXyb2Hpfyhodo8hmbTYKNZ7+B2Al7ilqFv4Q6OqEoQmkz1ozZEb1X0bKKi5prAm4ihj25Xld291YWBeud8OqQcelyltS7ehulH0CkuqvMFr5ofJP6iwBQOCu1\\\\\\\/MUfzPil4ukewPzSh5VVngCwvuRjvrIZI8Ah8Ij+AxUOMZmSyxxKbpBzWJXpbKdz7eVG31B95YwEfKfjruCUDhNte3RSOriHVNPSr9SGmVjVjD0eVEbx5nNNjnhzJW9bwGpcAV+xQJLpPiWSL4cuduVREUDFM6xpOczVgyPWDJtk5ttMLIoRQVGiFXgxd6fP2V+JXMWsEXJCG\\\\\\\/naNDzI0hb223P3zNkAgOvMF0eu1HKmEPSj4wZVS2Zfc6bIXOfA\\\\\\\/gNd+5XgjjSsa28dpCwNVwsAbHKrD+Dimr9L9p46a7VDc1jmuP0kE9mFW3a2PN55beGYb\\\\\\\/A\\\\\\\/vnMFmOIpyZCb+aWbJX2pvqL9E4Ubn1rnzbxcwAdOoSkvT\\\\\\\/gH+Wr60zo8A5hanvYtX0ofp8D7HyvKXP1X2dTOhPHA6n6qniaC7vyl4RQ8ROb2M4Su6YUUbb\\\\\\\/h21Kk3Nx9OSSRB9srSXL+bfxFI6RkIdEESqVHYwv\\\\\\\/watxY4VpsE5IYJ32IX\\\\\\\/JqAjCEs31ji67io3tm9luMyWD4uO4TX8yg0R\\\\\\\/eVFWQCRgUGcHCdxCHF9ruQmeS1UGYq6lAwI\\\\\\\/MAMKqUDoaOYJxBvLfFSlnN7mwz48Ng9bbqfyrmkceBSFgfyDqhA\\\\\\\/FwYwJqYnYN2U1DaC6JHH4JF66CiHeE7+Hucl8gegl2BLV7LpaaTIrjX3LVQrM2MO7GOGubMbqw3RPuNhwPBMjATmfyi61p0JPLA6urJiTcMj7Iz6m\\\\\\\/sdd38x1ySMbh14JbRO6lpw4rMl6jOT0wk7WqW3JNBe1dIvJQkyna4KAJHg7CGosKU9PvSB7Nm0\\\\\\\/ikJ3nkyVCWmSkuAOnjhlME6sV+aVNk+qnwjwIIoTK9jZSPphbFONsLxboQFQtsoshLwPPzZfQvSQh\\\\\\\/CH6FaF\\\\\\\/JoBtip8MLrX1sysuZIsu9+HzK7YuCW83HARjEAryXnm1H7JXF+EMmcX7t8+E+PjfHVmGOlFfxmXElwRKXontjvTtLttLRlahjhTkt7z0YVSISrwhbl8U7+tl\\\\\\\/AklgpJwO0IiBA\\\\\\\/gGTA6fBa+f+dP4J5WMrbxV+4OsMGS7BtaPIwuHnSficy\\\\\\\/fIidSyKcQgMANJRFpaLmj3zFG5hOwJKoYfZ9sLG8B241NgpQOdzso0QxW6v3998lFL16nZMQ1WLReP\\\\\\\/uxMBF3dQCDpliUaVy4NUn\\\\\\\/KCbNHqwS1yNwMnlQ9r4BzcAgjB56iz9Sx8Qr0O6hQZAawwdD4i0EkV0ZtPr7SX\\\\\\\/yHrwF13uR8HFjuB+cVgCktlHIov+VY8fCv7cfhSMVDvUIOpWs4ZKlXtEbuNDDYMzxA4dDxAE21tr14yXoVulQve2thAQyI2kbvPzUMgSt1FO7IhlVtKG1VZNfApgcZ73Qa++w5BBKgsaY8YuVRI1NxR7E5hE75WG43k+MZVFxl+SgQqSamr3kwnwzIzBrhXg6WDib\\\\\\\/wcsr2n02NG1WMRTbnCmcDf19Xe7y3hgQspRDQKvy3Fndo+YvaVztuA69AL6FXfgWB6Ekgeu4KDoyfVI0xFTw8rdoKCUFaEj4Bh3btbY6i3Fbeq8cgJzHPOJxxLOLVpG25wvY9BlltGmDbHiphDfxjeu+PR6bThPao7gd1f1w5DoFmXyK+z9lKk9yYbwNj7Kv3H7GTBdFH+41xSiqddQMmgmZ5ODAqdD6YQa06faVm5jsNmLOw8CV5nrApjish+H0jx7mUCUm1Jj3LPR1h1XaC949rRn03magroYzXNqTWpYMwcT6Hgfhn0Gpr3kK\\\\\\\/XQxt6bqTePGsH9kox5j4Uw8V9LWfZIovlIupiq8CIoQBw8c2YQI3jlMJ\\\\\\\/7YKM32CcntJXm7VQnFMI12\\\\\\\/1yYuZWAb9G1Rc9nSLbpXaOujFq2enDRlhIt171Jo1SfcO4T5\\\\\\\/Yw\\\\\\\/ODj9kaHar40JIYq3WfiFYp98KADofG\\\\\\\/dPNSOHifnoMOUKoP6z4sa2cwfVGdc5FbFVpjWO6ZBzP8qFvyS8q21kelrMZPzod65rXIG6z5737gq8PUbmp8dcUk2sN3ADvTJIYV8NbT6WCJLcySj9ofohVNztlIMxlIL84AICeH4E8b63deCbp5YAy1a9xxY52oaAsow4djO+9sTRqHvfkyiZo3PvKiLYoXyue2CZN1tiWoAP\\\\\\\/iz\\\\\\\/x5EdJlV1MnWYA6fd6lj8v5HgjtSn2LVgtd\\\\\\\/sTBeiLtht5rc77vta9LdYmMOKTaPKK71X3ykMMLpqeNWvfQ+uR+eqM0sYEqJNiF1bcRSfkZnA8R2V\\\\\\\/veAKzT+qkMU6Wyn3MKE+nJBt3tpF0x\\\\\\\/wwLLJw0hja3DuYZwFEjdFgvip30JvLT+TkkRvPE9YqgSvPEpfIfMX2\\\\\\\/2gRLfnah6jX9EqSZNE9OniZgjtGJ2Kny1dIOi28lk\\\\\\\/ZW98h9GbdFy8+mj6x5pkYtktaAJJlZX+Vk9EHHF9YApZcbVaCiLT1a3YXxtHQCY2n6CwqEsr1DTNntFA2sp5SMMLyJRP+ThHdvXYxFB9uwlKKdHXqKG67SgHz8tMSHSbgxBwdZ0aK3lGmVzljtQdt8pft\\\\\\\/URwPjX6TxLRoJLHMN3ElzZmK\\\\\\\/zQu+Y6\\\\\\\/\\\\\\\/86D+xtFgvzldOh76k+9+Kw7\\\\\\\/CXDkeYXXkY0RLctG\\\\\\\/\\\\\\\/jpXHctcZGmLcv7KoQJBsardq4\\\\\\\/LNHKgZ52p1szpEtEaIV4mGZNDavGMAxgr7SjnJ0ceEz1LnKeLbZluT3hoW5tILcWMzB1\\\\\\\/+udjw8hLTmW1uOgVTpwK4jQI1+g+4Z1awQbmTDOUrm\\\\\\\/lXCM8a1vAnpGhGFDYhYU7t6SD6bmaM8PKb6Zkasorhpo0vrGg2\\\\\\\/yqt+N0DPft6yVFvsPTJxNIOtFIEEcuzFpY7iGuKQD0uYlOl9GdcMhRNaskYnCMKFqdC0kZ1E6u5kDInshUOt1VVMNWwdswC1o0es1onatIod5fJLCVBr5d\\\\\\\/kW3h1ww+QZgBqFE4d4m8H539oG2TtnvkZO4iMIQSvyzu6uWn2XmjpNoyAdwueHYdZ9mfQ7sugAbOxoLm9IXzsyKPpDRpgQ\\\\\\\/E0ZKNQgLXEVcLbMuseqWGT+CTEkAIbJZ8ygJjcgblQP143K0HELOOvwQvW\\\\\\\/P9+rKewT963Mc5ekNTbSwTq3U3uUs+dDmB5ACztTrmsDmtA+XrrrBAWi2+\\\\\\\/ZZUAsrOVE766ur1tbU6pklCYu0gLCxmvrtxYkhpj9Or9u6OrlKsOLnI1SUqyH8oD3QSIllO27WsvY26ej4+bdje\\\\\\\/ZjfjvkwTfnSLkbgRpFqo7m62qS\\\\\\\/ynt1gIzDcQdOBD\\\\\\\/2kJPZSHkS+zvLRdv6H1e0y4OtBxQcyFIicSjd5WdTVtnbxcPvTHW5pnDpSuwbYsqVzAV3vzfuMEYsvPdilJ0rAsR2663kIDeU9AyPZePKFIR8g2fzoS3MmjkcB8yE1u5iG4SA6qqwrUFBLxn308fsfkbhy24S3etJw8DAtUjD5zgDzeNLp\\\\\\\/GPtk99RZPXjcCCcR2F+3E9F\\\\\\\/dhivjvvTDkTDjW8mgPU9XFZQZw6OHq644VW17seBBsYOo\\\\\\\/XM991VML3pjkVJSS8AFp\\\\\\\/RQvVHGSYefaMSyHKgNb0aL45cObrF8AeR5NVwK5pypGCbqVK6Byg8bBmpVqNP8AmROkFA9Ei7zl2nWa\\\\\\\/z8kVvX8nxlgolGuXtI\\\\\\\/KVIabXaWFKQ9cnVrUvvNsWYbVX8xaSQgza+qnYovXXFLUXvId9iRGS5ZVmIRn\\\\\\\/x7JXSWh0GQIswLJzEe+Y+OX8MKEgPl1QqI1zeAY\\\\\\\/OEGTcIr5v+OhkrO0tLUkGn6O\\\\\\\/tjaVNwC5XBiyL\\\\\\\/OjyPU1in6u\\\\\\\/cc2ODtg3xRYV3Aeld9VpuDT4TnJjY\\\\\\\/EihsoaahbIJDSu\\\\\\\/jb4TiTD\\\\\\\/LDdvx75bBNOWqzv35CD8pUmdTJ7ZpnBzXpxIwFLdJtpQOF5zyDdok\\\\\\\/8ciE9FO7L0tCYuSefq3tpzpdah7ESfJIwZmnB3My21zHYuWMGPpFg5yd2ACCj\\\\\\\/r5SvC8Z5hKtbWor7CSFQT+4DgmkkU8\\\\\\\/yhGdHRpo++EhODP4zuwcRDoVwKADFeIBH+J8axEplYKYxa\\\\\\\/40SSS1m6aGbSfOLDEdIKcnFAMwiUMNTe9AXqYx2Nz\\\\\\\/IzJ07H6OrOIDWqI7m4D7ys9T9vJy2ZdkJywW4lrdn8T9VHbNOH64hVhZlo3\\\\\\\/mQiUIM4MCg3\\\\\\\/wL\\\\\\\/lhbaVMzG3BgK98NBLADLbuCg2zwpYck+UZu9NfqTcQQ+DiDF6qcBMJfGWr\\\\\\\/DYt0leyQ9mesxfKxOBuHtKS22V1h3uauQXyaAmsx+RNnf+TcoywPXWSqSkTmAKCvqbSp41VZ+MvaP4iasRE9Dxian3m5xuJf7\\\\\\\/LiMt+IuQRHHvNwz7wuqboJ9mX2qNvOWgn3ExGRhr91vuf1piZ2PrkcGpXkf8NYoBXpyyHDbRxHdz9wy7gAmndU06C285pow4XTVdwUxE9E5P48FafTUcEwxSPM1733r0ol+UkM00ob8\\\\\\\/j0kkISD7JJtHFBY7p1\\\\\\\/0smQCs8AMqcUHCfCjyrg7s+kUA4+zgKGeggJqZUi02tpVM8GsCjLr6Wnkrwnu9rizznHYvNb6AHvfb2iw9paP1z7ZxdEjcEShG990cN6sSwJNky38nQTLctZQupxvrtMdzacCrHYas8M1nVlYswHU6Yn5So34XVPI8ZZsDA0yYS84usxmJBfwsi8QrvffsuxV9FppmpiTcY6U+opIC7e6N1nW7uio1pxg2Gz6M3wse1UV6mTm5vgTcT0Y89yXkvUi8OxM3D1s8yclytIP8cfzgnUz09ZoY3Xt4B5jOEGHnJxpjA\\\\\\\/Wty9Hlt\\\\\\\/uXPFmE3wdGoCnh\\\\\\\/G2+qvO2j4avQXBGCfROrC0\\\\\\\/6ISMFqocXFd6u4zdd4HFXDncuHGnmq0M7Q7RNCEaj5vGm3ZdxvFT2wm0JQBcH13hC4XuTX11RcJPV73q2UjE5oN0YZah7jWra7EVxd1LasIPULh0zTaX4irAZow8BiT7qfD83M2\\\\\\\/Dgi7moyEMgpVpzLjTdft7Busnpkw2NOi2YhnNwpeCDVIzxOCqRFwucjFRHEn+z1R25\\\\\\\/hbJxkC0L+mBQTK8XPN35NiQwJhaoqqlfBbChLMV92y+agewfQSU663wJVhZMhbrW9Yw0nt2CkK6VUrdw8AduuDxZ2uigmstgMnjOStwoKVi4kFB6MvdrKDyCVd3ekwTM9fgxWZIgF0cvECMDsqSKbvgi2fNCQEDun9RqxWVMNR4JEsp8+e4aptCSxReoLbcqox1Hst+PsNZ3qQQb8SXabbiNj2HO6bphGKIw4iSMT0Aq83kPyvAlteDtWHZo+4pHFH4AlpawbJXIMW0ejOTmAmXcHXofFIDDkCjV3aqo450326\\\\\\\/JxCsJslrTAW5pwSjfS+ru77xihpI85bRLHBDPYeDEVL3v3asmZczf0MWB0ea2QuSZ5lNzG8oWrAnpUkTm623fNaVcHoLiN\\\\\\\/nkyw2BOQNb8UXYftScCN6ao2vtVyzYKHQ0qCCClgB78DRrYKxQcQHonfOU0NPrFgUaw9wxe\\\\\\\/hrkPFD84vYz0JkMTr5hWJt4g4Jxwg+Q2ZOf3RitQfHy54jSNimPkZ\\\\\\\/363k1gcJMuPRxId3kOxf7Rx3VwDBsUNpbcOK10J6LxOvRwdcFlXZDENJ6T3lMSAk+Zrk7ATS2sT\\\\\\\/JJOUoIZSQQEhOtbQXt\\\\\\\/2t64nmk9ScfKtsuhxy1iHAOOWQvBnwlr858jZsuagNdMp6fyhuBXNkM6Bi17nd3uQQiO2I9Vr3mInaE5g0JTV6yJFnmu7EEGcZnLY84BmOOjlWrZ4nkRR0y2o4r8jZ5fs+\\\\\\\/0HtjVPm8L7nNyQe4YmWLoe2fESqc07wnyByfbkXBQ9JBb7sk+SzWvdHjdrXrZagD0JIoG9RFuZnI7plwtRm5g8hieL4vQ3Skf23QgVMH30qAEzjEwKAKzIoM4EjeEnFM0LaMPF\\\\\\\/8XCdFJyiciUOTJyhVLDA50WBPnhd7Srt9tJjnVgN4l86\\\\\\\/Sa1PlQ6\\\\\\\/dExdLqyFNpQ+OlDfWNX1t2bWHkyN9Z5+maZWklUsV7suCR2rSjtjTOsoJK4KIfs8UaGTj8nfE4Y3k6UYfWb9GT7n3X6Vp9EmNqpFpbFyn\\\\\\\/zVBTHA5b5Wl9yzY28VbzSzOdfPZoDoYBNc2\\\\\\\/o0G4B6PmP6giKGQWR9KmxbBQoLWOCstORUrzigqTvCyf2spi9yGccsRNZlGKuOngXpzSO2eIgt2CFFDCkWN2OAzGEz4yFO2RDBGfP+b5PrzffVMX0GpEsC8pzyR4sfmvuN7I22NvCAB6ji3Z8IrixmodHj+lvTzBXwHI+0CVVu6lcprHld6R55P8SM93cSePiCJQ9DPNBYWtnt0F94SLfcGJcGAaltLIDtPiVPDPgCKIaiq+x\\\\\\\/YrSVsIqefrNx+gmStaiIxzEUaj7cQ2cIXGuaS9APZhP3fKJggUFW7iZ6PtD3fLRlek5YM19jpyJMA5\\\\\\\/Hpm\\\\\\\/QjWW5wqUVmdIjO5fQ+PVM0p\\\\\\\/J+I38r4yaAjaXtKiBEtfJHbHlravv3UhkOjYcD\\\\\\\/2z9vwRbeM9YP\\\\\\\/hOi2o9eE6aY27NRTAyV8HRkyD2cFcvx3DvIzIZp\\\\\\\/48SsUM4e+b3bT7fCkHT44ip794MqY0YnUnsrZgnVSNotvxP\\\\\\\/DOPo6c3VZLGqtIlclZT9\\\\\\\/GYCC\\\\\\\/z\\\\\\\/2J9asRUeIHawJpn6j2lFLmaJK\\\\\\\/NnmQgFf3DS4MAldyXczYkCJ7N5NoX7clVlmhxkHsWK3iGtVbscHFTxRls46AUt682bemEeGrIjFTXedF+1m0gcsbta2BbPy7+t46braWMLjfDyorQZfB4awvb7sVDbJohWjyyy7lLDKM44kUL4FkbbxeSct9IYdPia2G7ra8HMfut4MjEvAAtd+d0X5RvClJOJnbSiImt8BN9x2UDFb2YqU8Mjzg\\\\\\\/5LtBvG64ct77y3rW+MjR80+Z47rMuz+QK4+PfuKCdTV0phGjBJhmSwAi\\\\\\\/r+H4gnSvmDWBszzpCq\\\\\\\/O4j67E\\\\\\\/TN1zTXce1QGfyc4An8kakNQtqNVAKrdXdRAJAnfoA3XgbwOLQxDfUWPunUXrMVmkqh6bTIfKUM\\\\\\\/3Be4d\\\\\\\/dZAL8AMB+ShpHaIoieWGCHbD6OpGxMXj+xa0I8QDeN4PBcEhPti1m82TbwqEFtA3XLDVNyjdyXb6lghaG142eJbilPnfU6Xk3pF3bWj3y1CLUL+5i42APzRH25TJmVkRt7rC9c6UPQN+SOdqq0PyNFeHBvqt793R8NsEovoJzuQsu1GFus7ELLvoeknyuZf7RmV\\\\\\\/gVcZE\\\\\\\/k\\\\\\\/KBebliIdppf6QSvvbZKfAV4Q8WB3LC1LJMF6nbJlm+Aoo689b63lStNzZ5XNl59Qz1RmtaundolACLex2fUVocYceCQCjWUArcqVWF7Nw2eYH+PEHq5SHs9SjAAoJKNh+ix9AZNaMvH4OIvd8GCYOj6WllqG5KqhhrEjMvN3YGgYPeWalNpR2smKPTO7+Odcf7F0GBu3+t+xHT6ed3lVIvEn3mUszDJ52pUQxB\\\\\\\/RPMfYQid0oXQJlDUouw8RjznbIXxk4Sm6aBxlIxfhCnLySZHAO4GEVJtKy+RfYuv76I1BwXeckLEFB53RR90o6wmqtbdgF\\\\\\\/UkygO8K6MCUPJFiGPi6gj6DCm2anlFbWHe\\\\\\\/AQK9HuO695766rrCWkt70upWjKmnjfySraFnSN1HxFlI4\\\\\\\/i06RNv6U3KPRPjQGSq2URTQY5oBV79Nw0OUl80BTosw+KVxSSsSpjV2IoyfSGjnttipU5TbVEdgD70kEbndRtDgZ9yMxvVWwASaadzTAjxTWIoHucYIrPNt4asm4qs9SvQMP5AZX\\\\\\\/FWG07UrjzPtaagsIuQS1B8\\\\\\\/sUS3+XXaGzTd8zH3w0SeDW7xe7H5f6G2yMhTLvuH5KbY8\\\\\\\/nEBcNuTCxLd3HI3Ilm+Hgu0hx6cf5O6s2tffolfSwr+PgJs6hsYiJDe5C7D0oZeKT6V+IrNFjJq2O+43Uq5J96jlCReL4dX4OH34PMwoUOKd0DyEd1GAb0lBzHSBViDKfu0EOfAJovSCDSvYitzoMWl4KD4U3IN1M4C3hOozT7dK\\\\\\\/LgZoyL4FjuX0XC39buYr6xSdstFkHf6hLmqlZo2UOPpAKF\\\\\\\/JUmgLwru3\\\\\\\/zCOTp\\\\\\\/7NdWcBuzD5vcYHmaPS1BqGD8oOXWNDEjX7uJJcJSJAy6Ue3wii4KbCAAe\\\\\\\/dnBb93p3UByU2zY8L5F00sPNQFKFps4\\\\\\\/4G3kvnBzw53jXzHJBD2BRyBtr\\\\\\\/ryZ0V60dlXxZ5grk2QJoV0+4pvBS6\\\\\\\/7+gcWmgHuEiLw9HTC9u\\\\\\\/OC44q2B1esLdGjFA\\\\\\\/ciV6OlVSaxJCTT5NvrI3KpK7fMy7IZFWXxNNv40jFNVVb9ydBfxuxPb\\\\\\\/2FYllbC1k5QZsXC5n2RmVQXoRuhZ1HC8EoN15QsjQVCg\\\\\\\/TVkVCOpfvsupLAOYIRnxZblWkizrYF26JZ\\\\\\\/vDdtUIUh1SKnhIrCsocW8W6zTQM4Q7YXjITN4GcBBOAvnYDelFt9SbotZejWnJ\\\\\\\/C8VavpOB0GawwrGl8Dkl3qgWz6JWr2rfCO59Zi1Fqg7mGsfGI6rhIreiJBptvg9LrTDN5mD+xKPUJVBugs2zlAimyAG+7jNWj7lODHyRpdymVA6HQ7nTvj7N79Dxk0ChXggTmm4PTaMq9tr\\\\\\\/jltifkzNU7QUkwnRQ6fm80fW+Kqra8fvgh2FKLfVzVXCEh16cop9GxUiH7UTYMOd62ReKXGV0pcN7gi2wjMbelBhh11kqGVJUxRPxhRXn2lUhJHtLDUkRYEibFALWmWLsg\\\\\\\/qYJZqjxpyVyCrRYb+imuE99lb0U4x\\\\\\\/PZo95t5bfzrO0zmv3w5ErMbCpkHCxxbF\\\\\\\/+UoWJWK\\\\\\\/r785EKrsct0f9QowPFWNtPi5bTbHgR5r+pDk+RjHWJD5Naj7RaORAYAoLfTpJ1YZ7xhExeyK0zZ2kPgKzgoP\\\\\\\/8oD36qgWOHMK29P0pXDC7QH6Gs+cR\\\\\\\/AkOZ9LT6AbIMoerdG7UHaeXSX1l6K7Urd9Kfb1kC+8dhSoeu5nMZsJOf2wuT5NL2qXntCsxXy1NdBeA6kqUVdse9j73WmfnudSMixzJ2BFMwYOa9Wc5eLb53N1srznlgG14Ln4bUB6ugkfnHKL\\\\\\\/n2thC8KrHOxxDJ\\\\\\\/uZvBT2mWDAUW4i+elLcJr5tHAj4kD9g7HFmfIh52UxW\\\\\\\/Bu79J\\\\\\\/DanM+4NsoYeOrQrQi5sXxooBV78vStt7KTQ44xe2BFnlaQ4a94mGrNQi63\\\\\\\/XU7yNSSaMYCYYYQnNJkV66er\\\\\\\/YkXUO1PmaMBhHCfuTa1ZBGCTh3LtyMWZ0iu\\\\\\\/f9zAfukWnH9q6fo5x5liH5TH5DrWerywIDCkALPvWH6ClXd9iUcuRmL7S0CWc6Kd3OnwIMHWEB1vCi\\\\\\\/KWI2jZGxH2y1yYEtZD7oIe229rLcs9NRC+PToMRa9rn067\\\\\\\/B4CLmA05ZWy5GO7ZwM6CdkSlaG+C6g431UAizM2wcqLayrtLEbycVsqTxLLY4ZQtr83F9AQKeQLXJzMYV2uhpqKqUtZKGZ7UR5DBDib8ntp2NQl\\\\\\\/YkGSeGz8kAXlgPgr9NkDvMPH70JIDtKI0i18oTXG1kq852UM5g6dX8Ha66nmcxHhNymHSJnQEFcke2xaKVAgqg1vH9Fx5QFqJml61oqyhrZBW2ty3db\\\\\\\/Nc5yMEFKfHrx6PTMWM+M6QIH947PmvHxdNJnJAsBdKprfYze1I\\\\\\\/3Yu7h2PnP8DE7kmEDUOYkwanpL8T\\\\\\\/4h8EndKtPlphMqlWFqR9YkwzlKP55oG\\\\\\\/2cXyFLuaLTw99AzZYB8uyBwqZaH2yc0WnVgfmcLwpbWidUIw3ylx9DAhl5d7bTTon3gDqXhxGZBnQj3M7aB6\\\\\\\/4MQr3ARoj5d+bTyVy4FyEhIWNVakicHcgBnMSfp2y+zqZzu0Z7e+zbz73WL8NfMl8WKKEzCnPt6OHp3Dq6tywvegZ+j+hwH4QDUuamoJBmMnFB8RlHdHRxamhrHCKhCUnyNSr+xL7AJf+I+MvNMpYhl\\\\\\\/\\\\\\\/cBno67R1rswej8MuH0g4ckiwiTy51rGE+eTMyjGrnlf3+so7RCL8ShhINsW0Oa3A0Hy++abATB6b7cEYYllxtKT7XtjWtYJ7eICCg\\\\\\\/y2+Ix69OHK7PX8fRksRc94BRM9T\\\\\\\/woVcImuYDZYQeJcYgl0nos\\\\\\\/CYj+pTXcHtkz84ZjBPxiVTeV+SNCS1cjBU4i6QNh3uByiiYLiaF9rlyjnYGS+B5e0PafmhMj7AFYIjCqBBuuYugd6ooGFRIP9rRHGoodRWqpV1KengbFaoKTT2Gu3M2KNq24FIWgwv77GG4kucxPvwFnpGp88FrfWtm6NedGtaENEjrh3QEGNRSM1bWu0xBQxR+dx2csJFPzUwsFO5sHhg+dFxTDnzsDlXqy2Jwgkbh9Jo04+ool3cBHT3LgG7fSBWgBUowlwO3dUiSKgE10lZPueTRKUTRC57u0EjHMfWXm4NVsi3htp0ECVhq\\\\\\\/HvtbNk0C9KBTtCRNpnBlj\\\\\\\/TbsNhP1smXK+ig\\\\\\\/5vLnHAyfz199Un8yBmDEjV6pvHqyVL3qw6WeSGUI3zQweiTzwvjhiP1Ji9eqoqjpEkX4vyb2o\\\\\\\/69E\\\\\\\/kqV1o9WVvihvsUgSRQgk+Dbo7Afv+DRk3d1yw2grnqs5yxcRPUEJQN6SnMogebu4fveQim8TcS2CHc6dgZRYzd+JLtxl0J2MG0BaudQAGkuyDWi6\\\\\\\/xQr0C9Vpv+w+0y9OI927j3kr\\\\\\\/4TZL+OyQ2B5YNv3CM7Ga8ebDgjifV8AoqjW16A5cJrLN35ncuOqP8LQjJTl3BuWHj7J9CMIoj8wiS6PFn11krrAVHJGQ85f5nOUNVvGI7g+iGZ0xD9R6k17FVzbFOduL\\\\\\\/SEGCSX62d0i272TSfFEIa1\\\\\\\/LA5ldJQztYyWvmKO6tYy3pI\\\\\\\/FcUzVr2GP8n4OuU43gpgrWElQDfEyWhW7fFiyjGQ11AoK7xldrtN5hmaWxxFOdw+8sbjy+OYN52pQV\\\\\\\/8J1qDLfyscoSwZzRG5U4p+5luOJ4e+OjQ4lhQfY+fO8rnLo3lRfGLVRt4WEL0lkZc4vws1O0UMnE5CwHb\\\\\\\/eP2ePisTP6aQ6o9X9\\\\\\\/wHEvGmeyPBXMachiFN0O1D04N\\\\\\\/xJUngeoNIS15Sm34Cd0JmbcxAOFLHAFGjuZkSUFjZpgleefMGgpbaisAI4xBib2HZj4CyklEw8Q\\\\\\\/araKPZnabA4n5y1RsqX4FvuKqD05xS6LjedWKTbKTKQoQgptuCRdGqnAvMfhNDB4zcD9E1gOmtOJxnfetYifdQA4iD\\\\\\\/SOEQ77d4dmuhQsBzvhvLruxPmrmIYn7uDvQgyIhPR2B+nmer10ELNeb4t04oVSjTGU6C1qKnb2Qa1o8y6yDkW3hTZr4nzD28j\\\\\\\/eQNK11CRyuXq\\\\\\\/cYGpt7S6gK5lJfgVjy6MGSUp++LEKGvFgfUz5TcLU75n0nWgXxePnysyawKDdJ\\\\\\\/IzeIzqPWi1OQmNkOrwNdZdl6p0IOYeBKCfqVF3WABT0Q1oaLoDUsFTmrkwy7Zl\\\\\\\/X7PoMNTMDDs9Xk+t8CVg0UsUT6Kjf\\\\\\\/RgB7qHK+hrhy9MJPFWQSXFFpKnplLf0fcmErts1flt\\\\\\\/BMcibEFmkd\\\\\\\/Zzk5h0E9NUOW7TJMmKi4rB\\\\\\\/m3WbQiy24ykVzHk8K9w4mpvzBhtjg5vrmnvXrYUWLk+s2wWj6WTWMDpgf5dqYeaySbO0sg9kUGitdggZDvWjwfxO+TiPM2nO6AD7lOOjtjr3+20\\\\\\\/1v\\\\\\\/ISUrQpN38oerXM\\\\\\\/Om4ERxfYN6I2pCfHRwCsJi33HGuFHdnPrruD8NYFQGADIk6jqpfd0YODhoxQE217jDAmnZbIK\\\\\\\/3FA5wq7mVbjbfNVO1pfQCtg8IXiBe0HJIHKESnF27pqTALjivq8xWGIC+nriqfIkhWxUCdJFHvrzkk5U6LHQ+B\\\\\\\/qGqKQ0OKbq51d7iDfqQwQYogxcVCwHAwB3qxm3rnMW1D1DNXK7Ku25y83UjhoiSMD\\\\\\\/DgxGoajhzkYJj1xhnkNe7CEr1APYL\\\\\\\/592HtUYzOM+NvBLSYCw4WAKtQlqCA2RVwTQB\\\\\\\/n79CICem+bwnITtSzstvMhingQmKp7f6xv3IbALVihrGGceAn0rQDdQpudup6Ncu2hFsTO+FrTmprKobwIHLCx4tCYAdLmD5ie7TUvKklL5oW5M1CnOs2liaZ1nTgGJLrwFXle8J4qKcv\\\\\\\/Xw9ZxgaeaMFlZZPbyjrs7o+\\\\\\\/hZoTSJb36\\\\\\\/Bc1cNecRNGFlm7PfFIj7YV04Pg5YQSZfgKnhSlmW+slcKp6hZLR2XUWXr86jliIJuO+ndoBkXSDPyYBtfvmHUAkr6R8j0NTrNM5dcvahWL6Foh240kEf3s\\\\\\\/sLa7+MDJKRU0tXMoKVC5YG0zfHZUlnT00auw5KPIomm99FsQIxWMwtxcRN980w8XFEFySmbKqcsJ5a8aNWdLMbvPEaFqCGGzGXraohndvFBuXhvDZkb2kqVBs52xzMfAdKFGCqiq4ptnOHVprHR\\\\\\\/AInZJ8mJeyi8Ajk0jtcb7C0LTiuHGF2vXmYTp8PFCUkRZZHf27XEz5Wvn\\\\\\\/kRYpAbsE9Pctpf3X8D+8+eAEk3bQSEE5mU1Qsl5bTgPL4tx35jOzTXJ6DvxkXbzZJbpyYeodPyTV6Nu17atDZoWROjKYG2bAqrOORsmaF4O410xSbK\\\\\\\/YdXVsisTn0HYKWSoGQg0OjLK0BDlHNhr4kXAqDaSL8lzBLX7x3\\\\\\\/ZwwO3G28IBkT97Bof814Fd7EIOqPsz\\\\\\\/5YsSbGBMfedySZfgxy63qChfjvq2FZabQy7mJT5taHN\\\\\\\/44PhrIkyBJGZsVN6LhUW99E65EgDddr8rJGtyvfbBAvmcRPkr+IOEDX1cLiy56xKNTlYqU1KmiNXcuM3c8kqhwChIHzDH5rcjwrVXVe9teGID1xlfGqbcExHDXpEXaQr8xGRas68RtYVhielMUqYKq+yAQ6XgwHVtsEmgWJd+0tz3euiFNAja2BBWCMuhIYe4blmhmB0SC6hirv8gMzOy8\\\\\\\/exuy4qsXjgcFcbG7kpfjMKXtdl0Pe+nkd\\\\\\\/fjmK9cz+cmLrlENquBG8b4cSDmqbxdq82LEUscuc7h4gmXj6jkabaO0wDHoY31FmXYIPvCE2XPPbNDiHbiCvhJVABULHdYDLqbg9W3dZa6REGt4bTs352YX2Vn3cfgzuuIcaqzeOFnsBj0Q5n05lh5d\\\\\\\/i52H+\\\\\\\/b8sZNFlcCRzPGlBKRVZzuodZ+XZamqBhmCO4E0vttgVFfx29qWpM\\\\\\\/HK8LXlH+b2ztWfQwvT6C0qcyXHmrA6KknbJ\\\\\\\/EeH0FYuEvvZezx5KNxXYDGvyqCoRNEMxmd8UlyllOYccLKb4uEg35kVouqsceErQObTGFTpkoWvSesAugu0HS6mwok4K4QMov90mRzZLmIfb5p6ZAG9iemUtQ4MaQOidrAFy6DsDFn6YJlJX8lbfvDUafCtDxMR39TVXBluNpZ8xUBt98r4lARkevJ4Q72Sw862T5fYNAo5l42pvSjZKUqFEd8rpbHkvFw\\\\\\\/SD1abP15wnWMqEHSrwYlB6CuCAcxNsdIp\\\\\\\/yxKvkUaPmEH5pfk2lYFytoa9AdOJAcpC1qKUTaCIskiaK3XsuZFyoSW992\\\\\\\/Q+7j3Ow\\\\\\\/3ji91KmsubIJ38t4shtEm5\\\\\\\/9qg1DtoXYjGxzTrcKJ+nDdZfOn5ZPhXd5N\\\\\\\/aTP\\\\\\\/ZhOiA9U\\\\\\\/1utRmrmtWskVpKOkR5u+IMSOxZ\\\\\\\/HePw7Nmrd9PmoegykIJwkVjSNsAzLHDRcxgYP5dMeFgobzczJnhv0JG9GhRoAhBxS6Ekl9cybDEs1Pwmtt\\\\\\\/ZD9j8KNx\\\\\\\/w1i3TJl3raSXLDycE1q7jYlP5B\\\\\\\/yhCVPHawANIoqewA41gIuG7woJvbsblkbFOvMQM+ffAiLWNCjcQinAuXX3z2AOcjAchgFHPS\\\\\\\/JlcZj0KLf1maeMy\\\\\\\/amVHHSh8jOx1wOnmCjY65e21+SU2UFkNZF+io8oJ3UvV6Tb10GH3\\\\\\\/rSukuWjovgrncDobtv8VXt8M6L7UAlqzMUyXkj6zn3VvMJSKDfaOlUU5sqx9Dw4BxBG9TtsvIAOW3Aewx6p\\\\\\\/Sqy0Ay1adZ2pidL++MMnVjmhmSWMBHBVB0l2rxk0jPgkShFj2u6DKocSrToxNQy+4sqy4VkEuiDpwV\\\\\\\/juCDwLfk\\\\\\\/bJ0gLom5d\\\\\\\/Ge\\\\\\\/mpET9rWUMyZU6XHgCaVM7W9HEe07KU5EFcFtynf\\\\\\\/p\\\\\\\/sAN3Y7ZAMvpNsbrtFZpZA6G02flEvsB+eN3k0QhsQfiT9JSXtax5iijLvG2oOyhO87MxMEg\\\\\\\/4ADiL7417tp2pTp2uokI1tQNoxBKyGnGJ7Ac31vneFYpqFjctTkXaXm\\\\\\\/AFk3Ipoq4BN2PnZHiD3knMkLJAKcIq4kxGEON8AO7qlpB8Sp5qfSQozFLwFlhvUePlGLhyGEgueIAogyXHRI\\\\\\\/F+mRi3boVFVP0jFiYHe2Brw68MZcSzwHOynyZ8jVuDf9+yAfnJePCPl3kbjRbeivOaKk8xolE2RnOg1GnOept7RQQdBVnBQlWdMEkIgqO5VBmxR4vTsO+\\\\\\\/bPGhjNdbSpAqBhIbkHVGqblgH9ZxW0+AJXCYtCGJ6dL0zHdon+Pp0mffPO+vHalZGEKAp2YWSK+ifGuBadWk4lquiPPJP38CWo+76EDLw8gP0U28ipe1QCziaXszgPGxVlZsDvcWuRA8nnAWv3O7I82HhmenvLfEA6L9g\\\\\\\/1Pmsp+nu1CVrdgFq6ZqMOOkOa1g2ljnch8BdRwZCbXyQVn6V\\\\\\\/ndV4c9DjrO32WZtD9ASTM7Js7l59hpcrJWG6jSWFrZdXOuDnNUNjkZDwUioMZGB98eJBzV7KaP9d7jgA1QcSYQjZhQ94CKPht3qlbYmKwQMMgkQNmqsZtqIi5D4SvMq3tivmub4MhVhpcmI\\\\\\\/ptH8Cybfh62KoJAQTPVzBLHRzXqlKw32U9jcd7thKkh7LI7aUhfPBzS1U1hA3gwZwrmHQNWInXobMlh3Q3IUVVjymv\\\\\\\/nKsLv+mOSnaG5MlgHDlF6JxNT77DfcUqcFq\\\\\\\/8Ca3C\\\\\\\/nGGuVNY7P5Wx8IECc9qcen0ld4vrEWKU6DZgoEr3ahtvL5BQUsOLvHtT089LsUdxAT2Mx1eolUnGBKgtwgmsC5m\\\\\\\/W5TaD9i0p+5LNsdVEixqtunZ6M9jZNxdt+MYRviqt3bLgdesNAXxgsdGquLkOe7TZQ5Jqc+j3uDfZfTarNHjYIONgQuUP4hog48\\\\\\\/yRH5YfiwQKisVfRPTXhfiVsKbSjt8zvmW6a4kWzRPY7zp6\\\\\\\/nG3se\\\\\\\/5PaZVwUrtDtF2rl6Ybk5RQpfKN3Fy8JDARDDzDakYWz09F4RL3VMovwt+ZXy9PYimA3A0KkZNmlhvOIx+7J9nrsizLSJHyzdOBaJdtypdQTRFTFWXRgE6vwdV15HY+06tZ+95W5qbfECmAqUWHRWCVj86\\\\\\\/ZSyh74g48jCBDez0NTdfuZpzs29LdEaolF0RSNMKlN6ImCgH+HdPZIn4EfOVDyeECcg5yqMqv\\\\\\\/kescCIx+Y6\\\\\\\/YirwTfh5QBzdJPmn1nVcnLtRakcYkTBfBdjdZvmRFUyodcqgUi2zuA3QrQd0zmTzWOQ6t7ViqmNaYIzSloqXFD36bnHAPe4TLu39DW9tIk7Uu0BXOraau3EoMqdy3Imphm3+i1L4KKb2iZCevwHXzBIO6W1JufexUm6raNFD6+s1UR+wroFtUw9PteMFTHWWVbzJQXf2BGnKMnOr4Oq0+JrhLprm\\\\\\\/oQdbdLWuN8TelLtp1AM6qvEnSSw6Y8vcyCH4tVwRNC30oQ+2TW3dtYRPnVADIXp1GAWltc10V0HJySfigb4aToEutNNPBRvrdlZzY0PJ+X\\\\\\\/IylpjDcz7G2xvJOBOKeJ0CDXGPnK9o6xRh2tClJWa63ghkPiGLzm1voaJhywKb2dNgrYfBrPtg8DSSKdJh\\\\\\\/hemOXRY9CYk5\\\\\\\/YiBsXnamxO8dMW9dUZctSDv4woCDscW3Y\\\\\\\/Qw9gRJrhuLQHGxOzbK2BoqCT\\\\\\\/f0bhBH1BCbG5Gnfxxl0HiL2mooLYfVsipfVZxIGjp0wOQljmytWShFr1yOLXJsyRk+zuEof79DBXWgUdzlYamVFUvUTabyWX8gL\\\\\\\/h6Ro8O2NBgskT7Uh5atpT3dH7hzSUCsrC61tqlJqbMTp2H67FvwJXeh01pCyhrS4d1CknTU6qmZKdLiq8o28NAjaqqwTkg4qXPsToEFqsRSdEnVJL6rFCJGwBu2olUoyV4Yq0vje9crnO8Vn5UvaUdza41mKoud+QiHHQTZKdYcVpJmNWV8Exgnj7JocHGHHmUlLucfaThZNqR\\\\\\\/Ze8MQmkNxVLNvuLG+\\\\\\\/1lttiQ9LuSXHzzfvCvHY65upe+8F3ao11+CSIcWUxK5Y4VCF6U\\\\\\\/EqOSZlMCTTBZ2ScRZhoJwsMWJEPH+gJOYlnVJ6wgBBqRC3EiPX\\\\\\\/8sLYn3thje3x9RZdyZsT3idECet0Ps5huWooFJpALpjQMcYE1LkhFOA7hgCmYZSSOvxOGs4rwqb\\\\\\\/waL1TDPb\\\\\\\/iE55R3uQdgMfO\\\\\\\/FlHro896DvHbC7VeRu7Enft7au6qIoX1LylWnfzS4qztuM4i42093v1+6WpCDbXGTiWPMTTp4Z\\\\\\\/EbzcsDRvQCxnCNPvVyxpI+8xjn3pH4rUy95+mcbIztjbtkIRV+BMjodrnmKs2nKoKI++j\\\\\\\/3gtktPVgXVS8pUL3nXtZGtWCgz+EpToBLFugc66MeTEuXJET8DFveTGEDvyI7bWSXeXWv2AhYgLYWoa0VrEy0TMeLde47XvvAs4HD3EDcwTP863ivIp6mPW\\\\\\\/dX3YDUWIn1xavSxTgoW0Xu\\\\\\\/m6F2FKu14irJvm+VMs9zok\\\\\\\/IK3CzjOfwhd7omSL9nor\\\\\\\/oyDuEGPXdKAnY0Oydz39x\\\\\\\/FHzkD3bw1jt3L5EVNzu6d05ozS5VP4LwCB3JoafZuUupL\\\\\\\/ev+5qxGA8CBzYbx+9QtFKBsIOe\\\\\\\/G7k+SfW6liMeS9NKynay\\\\\\\/I4Zg087zkbvMuyT1T2n3k\\\\\\\/H3W8CsbLwP5zem9qybfTpHF\\\\\\\/oG6pNcOv6LpRJ8lUxOFnkaDR4Z1tAkeqasO8efEU4IGfQ8MV7FwdnsYeXGX5h6yjMpuqaZNvZEWF2GnTAIzsgcGAUZ4XEUMy9Ca3bMOqCr\\\\\\\/y7MXShLpIwIxbVix\\\\\\\/Xwqv+k8OLO7BJ91qf05Nc5RBopw9yVXHjGDC70M5XLDb5Y+n9PMjE5M9yJqylotxyWYPXEoNV\\\\\\\/sOcJU1zG84jODPu29Px2hpyTWrV6KWJw+CuWwPv1nbe9sav1GakWd+\\\\\\\/SfS3UQDpJpsJrczthE3WKsmo6q8HAcQp4Pj+960HawzYI2egZKXCnp+I4MXvzNJM4Ogx0iLky6pobUpZnLK90HOn0+cYGNp2A6BgasDJErzx1XZyuqZJ9uGY1YlIsC3RY4A0nY1uwrc\\\\\\\/apm5i1QEkcwhui3WwoHTjvZqL8XqSoUNIUDIK5StQ3eQ\\\\\\\/TZqv27zfBkSX3Qc+\\\\\\\/XVLe9C3pf0HBRYAWenxX9cbmFVP0Za99FxNMbeY97oSbsniq3ppsM\\\\\\\/SfseEPJ43uY7l+2InpQg7SGzSiizKGKWpb8JMpxu5H4uIIVbTmAG+q6h9hoB8l64yDXPNR+4Bxck82Uh5NMfheXGoFGqh01o29Gzqz184zFpgo21xQv8DKeG0KokBK+q8JGsQGdlcJjE6rVnn1wtrZF\\\\\\\/ZPWhyzqYDbgZZdVnYRHs9a6UQW0m8dEbuSKm+Rlf5kQmDjXn3DegHvtokvbqBjMrP5EYK08EN5\\\\\\\/5G393CDt3xnmI+BCKMHMQq7X9qT8Dxg+5Vk9CAGx46M9S0tLDCwlQnjjNvJ\\\\\\\/Qzu08GB+LGxOXjezKxh83HBwNkuUvX1Kbjp3iVK2a7YsYpLJ9NX87AtsNmx6Nlm0qaRzUv65a\\\\\\\/lOXVtJN9hKOM50eiSRkXopq0vNvtiQQ0Zhmrm9uXc5VDrrGW6VHfwGyY7ouzfhVHThvnithbCyqYnZZtZ\\\\\\\/utXRNejp2Bqj2RLQs+LIQxerPEb15+PrnOes+GBa+FU0NyePQfnS4gasYuv2eZFFrMAd7CFNnBqZCGk1CeCLK0SnTH3rV+pE5QN8xKygMOmX3akBhQ4Ee1qrxSdFMBzYoA+NseMRQYmoNXLLx3x9Q4MtP4SSVqLn9yACnvEdfVMhwDNmaptkN\\\\\\\/oJ4wOCF1A5P+DvK6g\\\\\\\/goazuRXlskX7JPxQWyuSXwFMw1nEraB6iefGF8nn5g3E730dv244EHg2e76FVms+ZzpC6N51Q1ZxrigPWIiFBkYK6sLgYJTqCwfz5chuilOo\\\\\\\/CsOl7Zb3PyjauznB8KdUpKkcaf0nf55clrXHhPRe+yK5GWgT9ajycdTa8bNtsFwT0MKarYALzBMQlNXN3RXzkFfJNX35hThEe1LfakFGdG9t4tlTvvUGYosxijdNF8JsmZYtiDZ4YajZr6dOoASrsqjvl4oNwUtgmVZKJrZJGNO\\\\\\\/dJyqmZ76+SYn2QCqFxfpIhwykIdhYcI22wnxV4WAvej3ZA8iNrInBEXtpdGVQj7\\\\\\\/ErvSYl4Jhf3exBdh5eQG89xGa+jaQAGs4vPvOHr8uMOd7bK93SeHVwe6hg9LHbGcNgbWKkCF2nFav47P8k6HhqrsAt8+aVKZwsR\\\\\\\/2LcGciOWDJ403B8u97IfjU3gTVhlCp4jCIFoCoJX2wFK9aEWGx+kKoSCWu494YDqXVfI+6YZriKmUdD8HYXqaAjp4Zn2hBUEugjvlJewnJU\\\\\\\/rwIFkhrpJPpidxGc9NU\\\\\\\/VPfAG2AU+Hc2ODEAu0FSSgYioWMva2+ByIEiNeJd0e0h+xGkZLvRUYx8h7MUFFen5Rx1DEBRx+9C94VCWMaFvRz\\\\\\\/b\\\\\\\/Xu64Qfsqj426SNfeGKFu0r+T4MbCMBIISaLuoduiYN0bchoj\\\\\\\/t8RicEJ914wV\\\\\\\/CRlGCyn8RdDOcEHczOcckhwnSdly6YxYrarEUq\\\\\\\/P41VhJBZTHKt+nGntRHQ92A8UGfF7TSgQmtaBN5xOLfWTRjaTvuwVq9pyZvg26BEbxQILlRfez53js4bBRl48eKvjsvJ\\\\\\\/A62hGcPMBjs2iwZ694plLK1WCp44ObyaY4wHYMEVt2CVBbovWaX25yrs6pmVRELl90UeU6zcRRMwqcgenH00KZr890jpEisEX\\\\\\\/rlWWJ48v6lDLvTp7DZY\\\\\\\/uZFdSqeqwgZn4b4LMfB3Lz6g1ZkH46KXCWtJ3a3u891wO0YubAT1nz5GaM5qacTdO8INmhPpBDN\\\\\\\/aYNIVr8e5JjCVvfJLHlmenS8k2YNE8jrS3sWdAAYtWEJXkSo9M1CL\\\\\\\/HEcuq+nndqf5Dzk9EXJmsOgBe7JfUqwGjW3Yzbd+um1o1aaUYS6N+G0DGuA77U8j50BzKliRcFfjTtOXNdxw6A6LW948sKdHJ+nRDqr6GrCVszUxSca9ro0TD4A3RW\\\\\\\/DrUxUFd\\\\\\\/V+ujTA6lUcB8ond7lcZHnHEri7B2mkwgsS6NOCMBc2BLkpPHdqqxHbnK\\\\\\\/ruXX40XyHAu62vVoXQjuEngf\\\\\\\/Ak0mAgx+2LmnIe7hhhVQPtk6BhpJmXFv4B5WJMsQKvjM9+DAkO7OJUHdRVLER2xUV9FJ04YIWHX8KudHWzwxpXh7xtfhKbBNoISd+r7PfmHmdvi\\\\\\\/RzPL4VCQBnPrhpqv5GWLeCAAwISSzg6u4O5MHcvZS\\\\\\\/xOl10g2CDp6jSlG9men9oID5R0OCMAcEYWrBwtVnvsLAIE0+B+nLBVv\\\\\\\/YTYLx5LgGFqnCFvZnqQoj765gG45u6iLiuxmHqZA5n157CiBAp5SKDE2XcOzWGihUtLlz7tnT3G1D0yOiDG52HWLB0h0IAXRILVYjEqkObb1EnCIBfvnjYimyNNvETwjTySw2t9pdjkpQKf6IfO\\\\\\\/HQBsVTIGUbltxU7Zb+O5D8gM4M0ye7Htq0ffAXUqHPYu39hvq2\\\\\\\/zPhZxbIE66byseBkbMb5xMNEhUKbaHvkvt2hV4otXL08Z4H8NAj31SU0VV2EQWrmS03yYLTktscko753aoujlkFpjDc6hu0ZgHFLb+Ti\\\\\\\/tK9dcdeX1tBYmkwzoMk+6OfpDXxVpfeBI309tCCCXu6AbAhdoUQ2i5tEyHHQcq013hz\\\\\\\/yEVo2G4Yisi6RUkqr5E8CLNl+QF7ewuZkI+G56z\\\\\\\/5yzbjzYfVcm2VsEv6s7uoQw5RzrwSwInlvS5GRiqBn85OxlE3+BkVAK3+vX7NmvFw71sqx6Cuy\\\\\\\/+S1Hh5PeRCNHWC6lliPSJM76TdHX+rZWipB9hcb3\\\\\\\/ozHvcVRissY7hX+I6HLyfWVpO7Jj6Sdu13A\\\\\\\/XtBstF2Xd1Ffok+kd5gLYhY1L+5k8gdeoZ4xAJVc4qDn3Cvkd8Pyk3QDif2okMrAIG9iwrk4Xy7kmhnNrJ+gywbuPtV9rxrMxlhdbYKYJvWVW\\\\\\\/GBsBZSBAvOhb7qN2MzNQA9PB10GOmm2W7Bh1dmx4swW4nNnVamTHS1l1VWbRtP93Z5MEeq6PqVOba2+HhOY0WZrWxyVULf\\\\\\\/WI2xVySRDjVtVbrq0dey9HMGhxJzkKSHHnW9wuY3LCUYjd+yf3Ee9RoilKlzTNAVCnAmauWK4Sq5StzF9IVnLShZuzUk2FlxeQzGEauFayqkh1PLYfS\\\\\\\/1JA8UfkTui8nelvGozXW5pxQo5EC7Pjt3mpQ4\\\\\\\/cHWMWW+dRJhDv1TMoYGKrsV6CfoWw9IBW1gvRAaC418PqATJszGi4ScpOWH3fIQ7BBbLQ7\\\\\\\/+E7ynFZrbzppwda+vRppX5AEO+VcYgdigt5EtXQHXCabnABUt2\\\\\\\/1QuUKOd3aW9vte37mPftcp+XiRhGXzgyvLyUdEWtFQBSgcQK3LVD7hzILOuAHKGX8lni2imrXm3p\\\\\\\/hsc\\\\\\\/aSlV4t0LHMiU89zyYl2naAcRx9+\\\\\\\/PhPiJH3db6FVvKP5vHaOKBQJMtQyv9b4YT7HH5FW9HBtzrfH\\\\\\\/IqihEobNgwhb1t1VGescTsimVLMp33tGrPpenBpSJPSbxySHUbiEIAdUo\\\\\\\/na7kCma3yE\\\\\\\/IA73d+ESkodN2oaSiZkdJpyWGKM80K4vm2geQoP4P7QctHNYb+Q081De+bHT7ieMn205kaaJ6smNiXA5Z3K1YaCgeHXaXnJbpMFa9524NEDxWFuhSJxNx4BBKQ3dW3qmMupI56A2tEzvUWpXiQIx\\\\\\\/ZoVT7pWiFG8HqmBmvGpHvZ68lVIiUthRSWShENdLcIE8LNeAIyGJKMCkMCpSw1SF41lWE4X7sAL56BnHZ4JwyYjmEpYooMJCgp6gHrP\\\\\\\/Y3Eo6GvYumCnHs+IJYsec6HErEJblSrCxK5dFzYraT6uc4Oo9FUO46T6jZdU7NVDHAJT\\\\\\\/Fvw6lmd37z6OgvGVX+VRTJ4qgA0NvL\\\\\\\/Xgi4brxZSBw\\\\\\\/+FtJpt6wAV3vboMrOT\\\\\\\/vGz04HCdORh6j5aP66m6up8t2nXfx1yNzmJKTDomn\\\\\\\/H0mzLO08Rc1neYeLoWABKxh4a6rObTVyVdk5t+GNqyr90V0afa9fTPGNaZZ9tOXMZbuUESL8m\\\\\\\/HYcxEnu5YGDp8A6RY8MMry8seZ7AOGybKu5p2RHByK39YLtnaKIH4y+S65CkSzn\\\\\\\/gIMCCpTShdzz\\\\\\\/ur8aVVKBgNCipXEa+6+gXrjbT72yW2TdpTjY9+szsv2oV7wTbzUIuTKD0fJ2GEgTQGN1xyjFS1X7AAmZyljtfRXNXDnb20L7YrH7TvU25gzjIG1FcRSy29qTEtPkjTSlWTDi4G6rrrrUQxa+BRnRpKUGe7RQvF8Ngn9pk5eGTlNJ0ulouqG5UDqNa1kfUHdPYai8V6pukNSxlxBOHFxZaSPlMKOqE+DpyFQkG4PBo4ZVvp20uJs0v58N7TscBA0GqZiGAz2peIoGd27agTsTBwdDN40R93xUSEnT2+9OT78HLVyJMQEbkny3knKUY1jD1dF50UmTkplPkCZqAu3F2h2JCZeOH4dunvi0dNRYZeXSHVo76LcqdDrjHSZjgppdH0B\\\\\\\/5iC5+Xf4t6dX\\\\\\\/UHW9QyV\\\\\\\/zXGJsDtN1ngMHW0F8k2stqg5wuOaQ7fzpn3eSkObKv+1MmpzF7Jg5OdTjnufwFbABuGaIWWhddfbieldOQ0S6RMM05fR2urth4XATRpPZANB6Jiuyoy+Aiw0AjDXGjNtpBKz\\\\\\\/757GYu7sNuwKhiIjI+EMPyl2H90YJp7GIjHKwz9Z\\\\\\\/gFMf31JgvAFxvdsBAaXh3C9KT3OgL95kM0Sz7AxlL4j+tM7Oulaaa\\\\\\\/JkObIstuxIzy9+LuBnxhRJeBpj34qylezUJDbDfXNcrj2loA6UurDxW1HZ\\\\\\\/PQPflmayVBO7pdtVs39sjcZ05pU1OacHTBvxSbBd+73JlaUbG195bnP1L2w2\\\\\\\/RhU15GT9eB595y003vVR9edblP0ZKswicR7v6GXwiWkSj3qi+q+bkVBCCGQDvQfFL+M+NEAs5UWVFVmhfd8F3at2cSITgwZCaDreAnY9I0ZPHUgSlik8oKvPA41oi9lzJCVJPmR49LTKOVpZri\\\\\\\/tOeImEWBWxsCP4GQ+op39Dx7lVfiL0VhFkDn6NSG8wqa5w4sPnDoKUpCKfrRmxldhuPBPFLr9SG1tdFwsgQIKPV5J+VDv8rNV2KV5ngT03reY9QaGTPzM8Bi6t+MJcRXGiLv9VSuHcwWK8JRLC61fJpDmUlbADZfDzWzTQ6gZAu8dPhHyF0864FDQyd4bomzNmpzxtdBJL6kUZtIjs62jyhptYLJdHenLmslr0LB1o6W0kW8q63my3jSZW2G8QLcyJmmqgcKWVsFlu5g50EPliQqS008QFiggT0k0AJplVuZSUwDKL\\\\\\\/ywaUUAYuEa3w4rIowxhONvXmnimrpEzM327oBBVYLqhLGzO8zrqZXtXrMc0kpz1p8xQICGj+yxcCqOwQUzwV\\\\\\\/lRKjLn3p2U\\\\\\\/OOWWrftPoOs20B6HujHOHG1x84s3GKRuLPIP4jG1u476S3lA2don8jYiO+QC0\\\\\\\/kZesteeZ2X3DUncdJ8xpA5Oi9j3Nz4nju7JxjsqUXunqBLhx3p\\\\\\\/lyWw1OwKFHOM\\\\\\\/swwIQEhe4U9R8+dTake1Ep+\\\\\\\/3wS5bZYBa+ojGpRucWAAQ9sV2qbsCc82YuSyqkp1MkZFZwLW0iEqR9Wwq7MkcuOiiM9seX1d4MHFzKQlrS1jpHbvGj4K2forCwkPyfV9hnN+KaRJ9M3ICaYGxIKoSp6zYZtV\\\\\\\/OJeWf1eRo2wze\\\\\\\/p7+Jji33x9iuPHeFVMxFKnQ6mq2BoKeJpt\\\\\\\/x8HeQ1j0sVB2d+m8FEjL6QHSbfo6wrhQ4IsxrX+2i+rra64UxD1ytRY+gboHmu\\\\\\\/B74sa7SxAlealUWKgaoOfnZHGIMEW\\\\\\\/j5WyLl5y1r01GesTxULKDY0+fC4X6ZbL69K5CagPOnJ8+PJLbB1Kt1sM27BBUxxfvqNUG9GGN+E4wk+j3OnbXT9d9cKCXOMsVNywl7K\\\\\\\/FarA5aU51VnyZlx1EsbaKeK4vGTgSxsNsQH8ojXfTJFUfFhoD1m+1YBhC2ZmSvGoyOuzZi\\\\\\\/FDm6CZXXasCqQ7KyD1igAQH+tu5yeWGuVzMDhGR+Dbn4ru6v3WoH4WzHU9YLjoQpBlZDPGqqahMmBB2ltAxZG5ex3Z4reVzlYLtXhkp2t2C4mpaETyPN7vLmyptJsmXuLkvoDASiJ3nyr\\\\\\\/9wvocIbaoLD33fkSx2yqBZ8EnmaXNURQJfPtJXbJohmRaoKkuTRbwoBYnjsk0DTtI88T78He5m7Jegx8rRlc1JJ9q7xiqopQXBtP6EjbQkWQdDxbJPKc1YXryQGZq2S\\\\\\\/4pOI8N6xrAS96PJLeVT2O\\\\\\\/rM2MRVqYEWHpQzoECI98QqGdWhaiZrk3gKr8TsIOxza7WR\\\\\\\/dE02aiWYeTJ2adyEPRlxmW1lyT8n1S0GebaSk9l3lYBzhi6GpvYtDxA1ioZJ06Fo5ByEL3kLCP59WcHL2v8PZHGsIK5fZ6KeOVILBX6AtwadME1\\\\\\\/S8yCLC+PLmSAm0z2eH1TS6ziXh0d\\\\\\\/VxGu77chqfP0UHG5DnJwtNUMwLOE2Ib0Vgnw5oNiIuJPMgj2mpu8Nx5iHlukusZsOkY4oeS4lU3xxtWhDUp5IO\\\\\\\/9siSdr3drVW1Y\\\\\\\/m1qjOyFthbuisqP5hpArJMYsdCgAfDp2vsP3TabqOmJn6p+CzXYoeNDWNznxCVgbaSwIciTCuCpoO35b6tJPvZ\\\\\\\/D5ucru9zqDRmbOplkSmTVjwtiTUbdNgSmrNd2bjeRo6XeNcXknFhd0l4xzMXIQEXY8dW\\\\\\\/Rjn8hWq+eBo3YAEhFJRS1LOWW2bHcj3AggilwscqR2jlfFzlalU5jL7Zpiwy3X9VuBn1Mdm811MUDMTkaCwCMj\\\\\\\/e8pOzb+19Hl3xm9\\\\\\\/hMm1PToYEsK3sMvIeVsMIjnwYtmn8eMtYEXJGzH\\\\\\\/s49IEqn\\\\\\\/hWIdhVKl+ift9g5FronGQMLMYh65gigUU1QyuGeUuDQcpzG8j3OK5VFUd+NMvy+YgtmSoaT7VfZ7Q2CBJhE7csvhyYFUbRz+R2ojmzUdZUEOorYSSudhTibegTcnEU99SwNiAjclpBNu18EE5jTgq5MOAj0o05HR5SssOAv5dBaa5Cvj8IxQOT9ll0W\\\\\\\/zxJNxftIVUurQvyG2k1nQO06McIYG1xwKntf8ZanmbhBHcxY47dhpJ2iS3S5ElMx0Amm26jXp3P8EVnhl9gzKcUOufezAPvFAL+llZBrX2apqj\\\\\\\/dKQaBg1a2ZgRakv\\\\\\\/uWzkSsiiFVv6P8GaiGTHGfgPxHmjFRjUd5okhAewQapbnd30kF8u4Weyci8ZV0\\\\\\\/0Ay\\\\\\\/m8n17HA\\\\\\\/mHISZ\\\\\\\/NMb7IDxJ014QCUip4dwRWtZMcQ2Z9411Qoi9UgJkiYzdAc6p2uDwZs\\\\\\\/bV8pG2S1I8qk4tEf9lBew80K4W9FebdBrwTB3eI5LMKALKzPPoYZjuswoKLX7JFjwl7eB74sx0ccUHWk7NfN1zy1kBTd2PsuQpJPaC\\\\\\\/Mjk3NwnE6j8Su6v7MmvZzQNFU+0+cB2Uz4P44mw5U53PK8hxMoi+8eP0ENsKN6ivdE1ArMOlVz8LstuMms6XkpvyI4ZbeA31AD9cQCiaskrH9m47\\\\\\\/419DLRQmdsReTXwMSZI7e6OE7Kt9QUKXK7DV5JfEnEwWuXLAprO0672OjUr0zcIKmxONRXf0baqdPNSZcxViRCbOO+cQ83m59hganX6LT9AOlO2omtKetAajlaWOBfWX42Kmx3aOiwmos6sX1ay8UmPKCLN\\\\\\\/GUq7RHANK4Grhcy8nUMKFgXM4sKuIPf6H6lGXXrqcRaa7c1qBzfdRjCwF0gTXC30iSVt9hKoWckGMPwaRvEz6ZRyNFVjPJxgx+AyZN+HXYgvhtT7l23LFsoatCmLtyavVGCLk1z04lT7G8jwnvc9unbcKE7ixX477qGxfp6HP055cPWiv0mOFH4w8T4GWsrA3hWssEZdDNair8NiOxuzHtpXFvmy6pBp2\\\\\\\/e6Gd\\\\\\\/nb0szDtXfS\\\\\\\/TE4wi3oLMr2fmCphLVAEc+OVTnH3ppsV1RLx5mmP4uURo56095CExqMCkmgxdQijryFzHoGDl4sRvLrYg1AXOCi6U98GblWEG2Vl597e0zQrGWqPXZ0rCrz7TozKkqwqM4T\\\\\\\/otpHOeqkl7GdrBKAK14AJsYfTE5COx7b8OjIsbOdXXpGKTzn5Ju\\\\\\\/oa+0aHBgY\\\\\\\/7L9W8aqFNtVoshWg55q9kspl\\\\\\\/DKM8xPOs\\\u0022,\\\u0022iv\\\u0022:\\\u0022vq6YurK0TyHe+894qXOnyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022687a88f5a0a66116c0c9d846858a8bb9a51f1869826c25f732374062062fc2fe\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Vl0wSX\\\\\\\/lONihxVb2ynvCvBem9IVeJvdMbtR9FIDShjxv0sntF6yH6EqCBkm4n3Vvkhj+5ry3d3Metygut+L\\\\\\\/P8hyvsbmi84Toy5imeB4FF5GFthmVUrHOVRzyRgyr\\\\\\\/QmInW5tQIgrPCN88CrIpojZV99a29Uvf5rKPqgQI8OCCCwQADs7UDSYrwE\\\\\\\/Mgu6EU77qathP4uRacXnAsbeM+YiUYaZ6aAbFdC5DJvVELkBnJgI7bsU+yrqTwcQDkCLSx0Qj1dL+LxxVGsJrQes2HGgGU32rfr1mh2jHOsUe7u+LRBRlH5m+MLpundnElBZVkhs1H4Rh4UkTl\\\\\\\/B7O5lNU+7GbLdS9XNdgJfYuHo0rHY+vAB0PKN6ytXPK3ruB+g2t77I+qRoSsfnXoNfXsx0z2WQbuarZ4mcw\\\\\\\/gN1icCXy8LJ2T6nuN0LnkXvF6s59fqBb9VK1HNqXnP272KBtmAKLKUCSOQoG2Bd37jELPMccUhbK2P9164fyuh8uANv2b\\\\\\\/LY38dcZXMJXTOW5APeAHLDsw1H4h0n0EXHDjoLPK5lhhT4GSHwgfPJYY9C6LUlWcvo8XiiLzb1h7t83c8AtWpf9mzxL7gfdv7CUzPGuRFnjLPs5nnIQ0mA2nQxa6fD6IR1xkh6UvP8\\\\\\\/QWaqyEHJQXI5gdJZEUauqAs5hg\\\\\\\/yys23WCUKOdCf1LVURdp50jj6v3avFs7lyR7yba0zYmh61XZ9Slq78zfJygz8HJB1HUhGt+SMm6ZLfyNaG2VWdeP+Tc\\\\\\\/GqArOZyb1CBsxnue7LbrGBrWpIYcJtPYpAyzwHkoXRAz8PYvMwVUaoQZjKZ+OhOAjhy\\\\\\\/MNH1SRaXrLicbigkbB6p6GbflqsgmkWqrF88+JbNM80J+LzOWsJHNZaxB4wNID5Xxl3xFJcz1+nO+ovRavHkL4kmkJn35D9+7CBbpCMS+p5X3YT2lDl0qbeM4auxf\\\\\\\/aAqo9xyYDshkowzSpn7nvxWVlqyHv6UoAhUyC7kv0BNhBpu0nlL2Z\\\\\\\/ESTOu2arb0LZed67RNEiPdv1GK2Ww2eSIA6dZcIHeQS8PoNU1XKWuBTJbWOUnam2KzrCXlxjP7WMz7wwkuInvSUR27ZZZPd2AevnriC13LnAWCNYllsGnSfXBu63KlhJwekHryKewiG3r2RHzdThSF59qLW\\\\\\\/sGdXEOSf0BnndiH254pmvPvFiwjs\\\\\\\/PdAbYJPsbnV20sO90jrQRvMMv6jyvc6oCeTiSA8Oi6vZPYaRWMM3YP2Vw5JbuPz6dJMDXlJtVIpugMp+hs1AF2P0YOkMUj\\\\\\\/8VFYe1LtvHwz5PjMXl4ZASXkMkxsqWaCHNxdB3RrcFrG1FCL9hLMcGXwHjWGYS1uKBJmxY6RH2V3hEncPYZy9iPYbE2jXe07\\\\\\\/wnS3uZyMKeLxMnCBlNb0z9\\\\\\\/x5O+MYTQs\\\\\\\/EUxLg3ECmgV2ehg4t8UexkQ7E95MWJd43Oiwgj0UbYTt33sclb0nlHysZP+g\\\\\\\/+Npf1mj8MY9VOQ5qQVzicbHosg0qyBGYsu\\\\\\\/yaAbHOvKc48Mp+vfw0Dptk5xBg5N5hJqA9+oAgpMGm88R0PaSVf3UsB0vo7wDjwH\\\\\\\/2DMkLl\\\\\\\/R52YniP4Fo9cV+0k57mPbEXXtnYc2aatPDRIFz8Jw29vGVLju94F1+xgnglMKndOT8uaoipIt+NdAwonkGD6l2tEiBltHl16aoQETEdGcQfmWuCMkFJfmm9G14c4uHqrywOZ6ghKSxUpRHsvQhhoeoTHla7pnBKyyf\\\\\\\/RtFPVjJK9Hkw4oTrs4hYL8+1yzSlfwiZq6dIpdj7chuZCvAqOwO2u\\\\\\\/a7TRl5FpAs5gnuR83HngO66tDdbh\\\\\\\/jTO+CQq0Jrv30bTRG6OxaD2gxuPibtndP6FjAobVbjawfD78JP3lG8peNhmAVLBtyFlVyJ8J2J6oTxy7v\\\\\\\/+zb7p8xPHoTRr8+weBoNBteR+aV+ovhChWpIiNsFkw2F7OHfOmVj7ygTMsqcnesYeEpYSoMg8lJX\\\\\\\/aHSfkQdCu+Af\\\\\\\/b8lBZI21wH33eOc442ceHqsiQ\\\\\\\/JsBk+FkUgC\\\u0022,\\\u0022iv\\\u0022:\\\u0022rNFB8oS3AcpQuGLtnJRMyg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ad62a11e5e02c64e0becc609972b49d502726c79e7554c80ee2a24a34bedc317\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494261 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3409" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mXsWScr6B5ataKlvfpCsI1ghL31s6rYo6TZY1FPeY7k" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3409" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4ZC7f4bmo_1ceQOtF8oT5XOr3yfAjXMIsXaX8lqXFPs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3395" title="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3423" title="Environmental Association Identifies Candidates for Tolerance to Low Temperature and Drought" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494261" id="node494261" data-pisa="ggg;9/10/3409" data-pisa-master="ggg;g3.119.400357" data-apath="/ggg/9/10/3409.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Utilization of Tissue Ploidy Level Variation in <em>de Novo</em> Transcriptome Assembly of <em>Pinus sylvestris</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8181-4804" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dario I. Ojeda</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1298-7370" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tiina M. Mattila</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1012-9399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tom Ruttink</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0949-6156" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonja T. Kujala</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3799-3013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katri Kärkkäinen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1701-6124" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jukka-Pekka Verta</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6958-5172" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanja Pyhäjärvi</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3409-3421;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400357 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494261" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dario I. Ojeda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Norwegian Institute of Bioeconomy Research, 1433, Ås, Norway,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BI.%2BOjeda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ojeda%20DI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADario%2BI.%2BOjeda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8181-4804" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dario I. Ojeda</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dario.alayon@nibio.no" class="" data-icon-position="" data-hide-link-title="0">dario.alayon@nibio.no</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tiina M. Mattila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiina%2BM.%2BMattila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mattila%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATiina%2BM.%2BMattila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1298-7370" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tiina M. Mattila</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tom Ruttink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Sciences Unit, Flanders Research Institute for Agriculture, Fisheries and Food (ILVO), 9090 Melle, Belgium,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tom%2BRuttink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruttink%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATom%2BRuttink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1012-9399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tom Ruttink</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sonja T. Kujala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonja%2BT.%2BKujala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kujala%20ST&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonja%2BT.%2BKujala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0949-6156" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonja T. Kujala</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katri Kärkkäinen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katri%2BK%C3%A4rkk%C3%A4inen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4rkk%C3%A4inen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatri%2BK%25C3%25A4rkk%25C3%25A4inen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3799-3013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katri Kärkkäinen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jukka-Pekka Verta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Organismal and Evolutionary Biology. University of Helsinki, P.O. Box 3, 00014, Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jukka-Pekka%2BVerta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verta%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJukka-Pekka%2BVerta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1701-6124" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jukka-Pekka Verta</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tanja Pyhäjärvi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Biocenter Oulu, University of Oulu, P.O.Box 8000, FI-90014, Oulu, Finland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BPyh%C3%A4j%C3%A4rvi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pyh%C3%A4j%C3%A4rvi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanja%2BPyh%25C3%25A4j%25C3%25A4rvi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6958-5172" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanja Pyhäjärvi</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3409" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494261" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3409.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494261" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3409.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494261" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3409.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494261" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Compared to angiosperms, gymnosperms lag behind in the availability of assembled and annotated genomes. Most genomic analyses in gymnosperms, especially conifer tree species, rely on the use of <em>de novo</em> assembled transcriptomes. However, the level of allelic redundancy and transcript fragmentation in these assembled transcriptomes, and their effect on downstream applications have not been fully investigated. Here, we assessed three assembly strategies for short-reads data, including the utility of haploid megagametophyte tissue during <em>de novo</em> assembly as single-allele guides, for six individuals and five different tissues in <em>Pinus sylvestris</em>. We then contrasted haploid and diploid tissue genotype calls obtained from the assembled transcriptomes to evaluate the extent of paralog mapping. The use of the haploid tissue during assembly increased its completeness without reducing the number of assembled transcripts. Our results suggest that current strategies that rely on available genomic resources as guidance to minimize allelic redundancy are less effective than the application of strategies that cluster redundant assembled transcripts. The strategy yielding the lowest levels of allelic redundancy among the assembled transcriptomes assessed here was the generation of SuperTranscripts with Lace followed by CD-HIT clustering. However, we still observed some levels of heterozygosity (multiple gene fragments per transcript reflecting allelic redundancy) in this assembled transcriptome on the haploid tissue, indicating that further filtering is required before using these assemblies for downstream applications. We discuss the influence of allelic redundancy when these reference transcriptomes are used to select regions for probe design of exome capture baits and for estimation of population genetic diversity.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/haploid-tissue" class="hw-term hw-article-keyword hw-article-keyword-haploid-tissue" rel="nofollow">Haploid tissue</a></li><li class="kwd"><a href="/keyword/megagametophyte" class="hw-term hw-article-keyword hw-article-keyword-megagametophyte" rel="nofollow">megagametophyte</a></li><li class="kwd"><a href="/keyword/paralogy" class="hw-term hw-article-keyword hw-article-keyword-paralogy" rel="nofollow">paralogy</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/pinus-sylvestris" class="hw-term hw-article-keyword hw-article-keyword-pinus-sylvestris" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Pinus sylvestris</em></a></li><li class="kwd"><a href="/keyword/allelic-redundancy" class="hw-term hw-article-keyword hw-article-keyword-allelic-redundancy" rel="nofollow">allelic redundancy</a></li><li class="kwd"><a href="/keyword/short-reads" class="hw-term hw-article-keyword hw-article-keyword-short-reads" rel="nofollow">short-reads</a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-Seq</a></li></ul><p id="p-3">Coniferous trees are a dominant member of boreal forests worldwide. From an ecological, genetic, and evolutionary point of view, they represent an interesting group for comparative analyses to other seed plant groups. Due to their phylogenetic position (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2015</a>), variation in genome size, highly repetitive structure and organization rich in pseudogenes (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Wan <em>et al.</em> 2018</a>), they can serve as a useful contrasting group for comparative genomic analyses with angiosperms. However, the limited availability of genomic resources, particularly whole genome reference sequences (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Birol <em>et al.</em> 2013</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Neale <em>et al.</em> 2014</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">2017</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Zimin <em>et al.</em> 2014</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Uddenberg <em>et al.</em> 2015</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Stevens <em>et al.</em> 2016</a>; <a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Wan <em>et al.</em> 2018</a>), has slowed down comparative genomic analyses in conifers with angiosperms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chen <em>et al.</em> 2018</a>).</p><p id="p-4">Given this limited availability of whole genome reference sequences, the vast majority of current genomic analyses in conifers rely on <em>de novo</em> assembled reference transcriptomes obtained with next generation sequencing of RNA (RNA-Seq) (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2015</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">López de Heredia and Vázquez-Poletti 2016</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baker <em>et al.</em> 2018</a>). The use of these assembled references in conifer trees, therefore, has expanded to other applications in addition to the identification of differentially expressed genes (DEGs). These applications include comparative genomic analyses (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Wachowiak <em>et al.</em> 2015</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">De La Torre <em>et al.</em> 2017</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baker <em>et al.</em> 2018</a>) and expression-QTL mapping (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Verta <em>et al.</em> 2016</a>). It also includes phylogenomic analyses (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Li <em>et al.</em> 2017</a>), SNP discovery and molecular marker development (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Parchman <em>et al.</em> 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Canales <em>et al.</em> 2014</a>), and gene identification for exome-capture bait development (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Howe <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Müller <em>et al.</em> 2015</a>). However, the generation of reliable and complete reference transcriptomes still faces several challenges. Because many non-model species are outbreeding, their genome displays high levels of heterozygosity that hampers <em>de novo</em> assembly algorithms and causes allelic redundancy (the presence of alleles of the same gene on different transcripts) and transcript fragmentation (splitting of portions of the same gene). Hence, these reference transcriptomes usually contain a larger number of contigs (transcripts) than the number of expressed genes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gayral <em>et al.</em> 2013</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ono <em>et al.</em> 2015</a>). Thus, downstream applications, such as the design of an exome capture, or estimation of population genetic parameters, based on these assembled transcriptomes could lead to biased estimations, and cost-ineffective experiments.</p><p id="p-5">Several strategies have been employed to handle allelic redundancy and transcript fragmentation in <em>de novo</em> assembled transcriptomes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Davidson and Oshlack 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Davidson <em>et al.</em> 2017</a>). These approaches include scaffolding translation mapping (STM) during the assembly (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Surget-Groba and Montoya-Burgos 2010</a>), post-scaffolding methods after assembly (TransPs) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Liu <em>et al.</em> 2014</a>), and Orthology Guided Assembly (OGA) (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Ruttink <em>et al.</em> 2013</a>). It includes as well the identification of orthologous contigs using partial or complete genome sequences (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bao <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Armero <em>et al.</em> 2017</a>), and phylogeny-informed identification of orthologous sequences (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Medlar <em>et al.</em> 2016</a>). Although some of these strategies have been successfully applied to some crop species (<em>e.g.</em>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Stočes <em>et al.</em>, 2016</a>), these methods rely on the quality of assembled and annotated genomic resources from closely related species to cluster allelic sequences and scaffold fragmented transcripts. In conifers, these methods have not been applied so far and allelic redundancy of assembled transcriptomes in this group is commonly reduced with CD-HIT clustering and the selection of the longest representative, with CAP3 clustering (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Suren <em>et al.</em> 2016</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Li <em>et al.</em> 2017</a>), or with a combination of mapping to a reference genome and redundancy reduction with the pipeline EvidentialGene (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Visser <em>et al.</em> 2018</a>). Thus, the levels of allelic redundancy of previous published transcriptomes in conifers has not yet been determined and it is unclear to what extent allelic redundancy, and transcript fragmentation influences downstream applications. Considering the diverse range of applications of <em>de novo</em> reference transcriptomes in conifer trees (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Raherison <em>et al.</em> 2012</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Canales <em>et al.</em> 2014</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pinosio <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">De La Torre <em>et al.</em> 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hu <em>et al.</em> 2016</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Celedon <em>et al.</em> 2017</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Porth <em>et al.</em> 2018</a>), there is currently a need for additional strategies to assess the suitability of a transcriptome assembly for some of these diverse applications.</p><p id="p-6">Paralog sequence collapse (PSC), the over-clustering of highly similar sequences leads to another type of problems in assembly and its downstream applications. These occur in the original assembly when different paralogs are assembled to create for instance mosaics (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Ruttink <em>et al.</em> 2013</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Yang and Smith 2014</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Smith <em>et al.</em> 2015</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Stočes <em>et al.</em> 2016</a>). Allelic redundancy reduction will increase the frequency of PSC. However, most plant genomes contain several closely related gene copies (paralogs) due to whole genome or single gene duplications, making PSC and consequential paralog mapping (the mapping of reads originating from different paralogs on the same location of the reference) a common problem across species. In evolutionary and population genetic analyses, it is desirable to keep even highly similar gene copies separate, since after the duplication event they obtained distinct genomic locations, gene genealogies and evolutionary histories. Collapsing of paralogous gene copies will lead to paralog mapping and false variant calls. If paralogous sequences are collapsed into a single representative sequence in the reference during redundancy reduction, reads originating from those different gene copies will map together and variation between two paralogs will appear as polymorphism. One solution for this problem is the identification and exclusion of paralogs after the SNP calling with model based approaches or by identification of excess heterozygosity (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gayral <em>et al.</em> 2013</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">McKinney <em>et al.</em> 2017</a>), which may lead to considerable loss of data in cases when PSC is common. Further, similar collapse takes place in polyploid genome assemblies, and model-based methods have also been developed to estimate ploidy level of individual contigs and variants (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Margarido and Heckerman 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gompert and Mock 2017</a>). However, these methods are based on DNA reads randomly sampled from each segment and thus cannot be directly applied to RNA-derived sequence data without additional modification of the models.</p><p id="p-7">Here, we take advantage of the availability of different tissue ploidy levels in conifers to determine the effects of allelic redundancy and PSC on downstream analyses. Using the single-seed haploid megagametophyte tissue in <em>Pinus sylvestris</em> L., we first tested three strategies to generate <em>de novo</em> assembled transcriptome references and second, used a novel approach to estimate levels of paralog mapping (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). PSC and the resulting paralog read mismapping in the context of this study was studied by utilizing the expected lack of heterozygosity in samples obtained from haploid megagametophyte tissues. The amount of observed haploid heterozygosity (H<sub>o</sub>) in various assembled transcriptomes in relation to observed and expected heterozygosity (H<sub>E</sub>) in diploid tissues was then used to assess the levels of PSC and its effects on estimates of genetic diversity in each assembly. We further discuss the effect of each strategy used to generate a reference transcriptome and how the levels of PSC can influence downstream applications. Particularly, we focused on the effect of PSC on reference transcriptomes for studies aiming for: 1) marker development (especially for probe design and the availability of a non-redundant reference sequence for downstream mapping analyses), and 2) use of the assembled transcriptome as a reference for population genetic analyses and genetic diversity estimates.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strategies used to generate and evaluate the reference transcriptome for P. sylvestris. A) Individual assemblies, B) combined assembly of all reads per sample, and C) assembly of all megagametophyte (ME) per sample; retaining only &gt; 500 bp transcripts (1) and then all different tissues per sample combined using the ME assembly as guidance sequences during the de novo assembly (2). Trinity and CLCbio Workbench assemblers were used on all three strategies. The secondary clustering consisted of the Orthology Guided Approach (OGA), construction of Lace SuperTranscripts followed by CD-HIT reduction of allelic redundancy. Assemblies marked with an * were assessed for levels of paralog mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1195211802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Strategies used to generate and evaluate the reference transcriptome for &lt;em&gt;P. sylvestris&lt;/em&gt;. A) Individual assemblies, B) combined assembly of all reads per sample, and C) assembly of all megagametophyte (ME) per sample; retaining only &gt; 500 bp transcripts (1) and then all different tissues per sample combined using the ME assembly as guidance sequences during the &lt;em&gt;de novo&lt;/em&gt; assembly (2). Trinity and CLCbio Workbench assemblers were used on all three strategies. The secondary clustering consisted of the Orthology Guided Approach (OGA), construction of Lace SuperTranscripts followed by CD-HIT reduction of allelic redundancy. Assemblies marked with an * were assessed for levels of paralog mapping.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3409/F1.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3409/F1.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494441" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Strategies used to generate and evaluate the reference transcriptome for <em>P. sylvestris</em>. A) Individual assemblies, B) combined assembly of all reads per sample, and C) assembly of all megagametophyte (ME) per sample; retaining only &gt; 500 bp transcripts (1) and then all different tissues per sample combined using the ME assembly as guidance sequences during the <em>de novo</em> assembly (2). Trinity and CLCbio Workbench assemblers were used on all three strategies. The secondary clustering consisted of the Orthology Guided Approach (OGA), construction of Lace SuperTranscripts followed by CD-HIT reduction of allelic redundancy. Assemblies marked with an * were assessed for levels of paralog mapping.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-9">We collected five tissues (needle, phloem, vegetative bud, embryo and megagametophyte) from six non-related individuals of <em>P. sylvestris</em> growing in a forest study site located in the Punkaharju, Southern Finland on May 26<sup>th</sup> -27<sup>th</sup>, 2016 (Table S1). At the time of collection, male strobili were shedding pollen and the vegetative growth for the year had already started. The sampled trees grow in two close-by naturally regenerated locations (Mäkrä and Ranta-Halola, Finland). Needles (NE), phloem (PH) and vegetative buds (VB) were dissected in the field and stored immediately in RNAlater (Thermo Fisher Scientific). Samples were transported on dry ice and kept in -80° or -20° (samples in RNAlater) until RNA extraction. Megagametophytes (ME) and embryos (EM) were sampled from a single germinating seed of the abovementioned mother trees. Before germination, seeds were stored in the dark at 4°. Germination was initiated by keeping the seeds in moist paper, under constant light (300 umol/m<sup>2</sup>/s) and in 23° degrees for 48 hr. Each seed was carefully dissected by removing first the seed coat, the nucellar cap and layers, and taking care of separating the diploid embryo and haploid megagametophyte tissues. From each seed, megagametophyte tissue and embryo were collected, and rinsed with 70% ethanol during the dissection.</p></div><div id="sec-3" class="subsection"><h3>RNA isolation, library preparation and sequencing</h3><p id="p-10">mRNA was directly extracted from the embryo (EM) and megagametophyte (ME) using Dynabeads mRNA Direct Micro Kit according to manufacturer’s instructions, except for a minor modification (using 200 µl of lysis buffer). Total RNA was extracted from needles (NE), vegetative buds (VB), and phloem (PH) with the Spectrum Plant Total RNA Kit (Protocol B, Sigma). After total RNA extraction, mRNA was captured using NEBNext Poly(A) mRNA Magnetic Isolation Module (New England Biolabs Inc.). mRNA extractions were treated with Turbo DNA-free Kit (Thermo Fisher Scientific). RNA concentration was quantified using Qubit 2.0 (Invitrogen) and Qubit RNA HS Assay Kit (Thermo Fisher Scientific). All libraries (6 trees × 5 tissues) were prepared using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (New England Biolabs Inc.) with a fragmentation time of 5-12 min. An insert size selection of 300 bp was targeted using a concentration of 40-45 µl per 20 µl AMPure XP (Agencourt) and between 12-15 cycles of PCR during library preparation. Libraries were indexed using NEBNext Multiplex Oligos for Illumina, Single Index Set 1. RNA library concentrations were quantified using NEBNext Library Quant Kit for Illumina and LightCycler 480 (Roche). Fragment size distributions of mRNA, total RNA and libraries were verified with 2100 Bioanalyzer RNA 6000 Pico Kit and DNA 1000 kits (Agilent). 6-12 libraries were pooled in 5 runs of an Illumina NextSeq500 instrument using pair-end 2 × 150 bp and sequenced with Mid-Output Kit (Illumina) in the Biocenter Oulu Sequencing Center.</p></div><div id="sec-4" class="subsection"><h3>Strategies to generate de novo reference transcriptomes: primary assemblies</h3><p id="p-11">Raw read quality was analyzed with FastQC (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andrews 2010</a>), and adapters and low quality reads were removed with Trimmomatic 0.33 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bolger <em>et al.</em> 2014</a>) using the following parameters: “TruSeq3-PE-2.fa:2:30:10:1:true LEADING:3 TRAILING:3 SLIDINGWINDOW:10:20”. We used three main strategies to perform primary reference assembly for <em>P. sylvestris</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The first primary assembly strategy is based on individual assemblies, using all reads per sample per genotype using CLC Genomic Workbench (<a href="http://www.clcbio.com">www.clcbio.com</a>) ver. 10.1 with default settings or Trinity ver.2.4.0 with default settings (with the - group-pairs-distance 1000) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Haas <em>et al.</em> 2013</a>). This resulted in 30 separate assemblies per algorithm (here called TRINITY<sub>individual</sub> and CLCbio<sub>individual</sub>) (strategy A in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The second strategy first combined all reads from all samples (700 × 10<sup>6</sup> reads), normalized the reads (default 50 coverage) and performed <em>de novo</em> assembly using CLCbio and TRINITY with “not guided” mode (strategy B in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, further referred to as CLCbio<sub>notguided</sub> and TRINITY<sub>notguided</sub>). The third strategy first performed individual <em>de novo</em> assemblies, using CLCbio and TRINITY, for each of the six haploid megagametophyte tissues where only a single allele per gene is expected. We retained only transcripts &gt; 500 bp, thus generating pseudo long reads for subsequent guided assembly. These &gt;500 bp transcripts from the single-seed megagametophyte tissue are used for resolving isoforms, and improving assembly of complex transcripts, but they are not incorporated into the final assembly (<a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki">https://github.com/trinityrnaseq/trinityrnaseq/wiki</a>) (strategy C in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>, referred to as CLCbio<sub>guided</sub> and TRINITY<sub>guided</sub>).</p></div><div id="sec-5" class="subsection"><h3>Quality assessment and completeness of primary assemblies</h3><p id="p-12">The assemblies obtained from the three above described strategies were evaluated in terms of several statistics, including number of contigs, N50, average contig size and number of predicted ORFs. These statistics were obtained from TrinityStats, rnaQuast (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bushmanova <em>et al.</em> 2016</a>), and TransRate (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Smith-Unna <em>et al.</em> 2016</a>). Additionally, a metric of gene completeness of all these assemblies was determined with BUSCO ver 2.0 using the embryophyta_odb9 database (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Simão <em>et al.</em> 2015</a>) (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). We opted for these four applications in order to obtain complementary information and also to further compare similar outputs from different approaches (biological/reference based <em>vs.</em> statistical/ reference-free based metrics) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Geniza and Jaiswal 2017</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hölzer and Marz 2019</a>). We further calculated the N50 considering only the transcripts with an expression value that represented 90% of the total expression data (E90N50). This was done for the Trinity assembled transcriptomes only (TRINITY<sub>guided</sub> and TRINITY<sub>notguided</sub>) based on their levels of expression. First, the individual reads were separately mapped to each reference using eXpress (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Roberts and Pachter 2013</a>), then the matrix of counts was generated using the scripts of the Trinity RNAseq pipeline (Supplementary Data S1).</p></div><div id="sec-6" class="subsection"><h3>Secondary clustering: the Orthology Guided Assembly (OGA) approach</h3><p id="p-13">After the assessment of the individual transcriptomes obtained in strategy A, we selected the reference from one assembler (CLCbio<sub>individual</sub>) and combined all the contigs generated from the individual assemblies. This combined set of <em>P. sylvestris</em> contigs was used for secondary assembly using OGA (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Ruttink <em>et al.</em> 2013</a>) with previously published proteomes from <em>Pinus taeda</em> and <em>Pinus lambertiana</em> (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>) to guide the assembly. We either used 1) all annotated proteins (ALL) from <em>P. taeda</em> v1.0; or 2), all annotated proteins (ALL) from <em>P. taeda</em> v2.01 (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Neale <em>et al.</em> 2014</a>); or 3) all annotated proteins from <em>P. lambertiana</em> v1.0 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gonzales-Ibeas <em>et al.</em> 2016</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Stevens <em>et al.</em> 2016</a>); or 4) only the high quality curated proteins (HQ) from <em>P. taeda</em>; or 5) from <em>P. lambertiana</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Briefly, OGA first uses sequence similarity (tBLASTn e-value cut-off 1e-5 and up to 250 hits allowed) against the proteomes of the reference species to select allelic and fragmented contigs from all genotypes (assembled individually) per reference protein, then applies CAP3 clustering using default parameters for assembly (minimal overlap length of 40 bases and per cent identity of 90%) on a gene-by-gene basis (<a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Ruttink <em>et al.</em> 2013</a>), and finally selects the most likely orthologous CAP3 contigs per protein of the reference species. With this procedure it is possible to resolve transcript fragmentation and allelic redundancy across the individual assemblies, while generating a transcriptome reference sequence for <em>P. sylvestris</em> representing orthologs of a closely related species.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494496/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494496/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Proteomes used as reference for Orthology Guided Assembly (OGA) of <em>P. sylvestris</em> transcriptomes. Number of input proteins used per reference species and the number of orthologs identified in <em>P. sylvestris</em> with the OGA approach</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Secondary clustering: Construction of SuperTranscripts with Lace and CD-HIT</h3><p id="p-14">Currently, there are several strategies to reduce redundant transcripts in <em>de novo</em> assembled transcriptomes for species that lack a reference genome. The generation of SuperTranscripts (representation of all isoforms by a single non-redundant contig) has been reported as an alternative for a reference genome (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Davidson <em>et al.</em> 2017</a>) and this has been implemented in the Lace software. This strategy reduces the amount of contigs by generating one single contig containing all exons arranged on different isoforms. In population genetic analyses, it is desirable to avoid the mapping of reads separately for different isoforms and information collected from different isoforms may be closer to the actual collection of exons of the gene in genomic sequence, thus the generation of SuperTranscripts with LACE it is a reasonable strategy to avoid this bias. In contrast, CD-HIT and CAP3 reduces the number of contigs based on homology percentage, and both have been reported as the most effective at decreasing redundancy in <em>de novo</em> assembled transcriptome from short reads (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Yang and Smith 2013</a>). Additional approaches include the TIGR strategy (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pertea <em>et al.</em> 2003</a>) and the Evidential Gene pipeline (<a href="http://arthropods.eugenes.org/EvidentialGene/about/EvidentialGene_trassembly_pipe.html">http://arthropods.eugenes.org/EvidentialGene/about/EvidentialGene_trassembly_pipe.html</a>). Here, we opted for a combination of SuperTranscript representation followed by a reduction by CD-HIT, which in combination reduced the number of transcripts based on different perspectives. We selected the latter based on its wide application in several groups, including conifers. First, the different isoforms assembled for each gene in the TRINITY<sub>guided</sub> reference were clustered into SuperTranscripts using the Lace software version 1.00 (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Davidson <em>et al.</em> 2017</a>). Briefly, we first divided the data into separate files with Lace based on cluster information from the TRINITY assembly and then generated multiple alignments for each cluster using BLAT v.35 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kent 2002</a>). Based on the multiple alignment of the isoforms, we applied a graph-based algorithm to generate a single sequence (SuperTranscript) containing combined information from all isoforms (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Davidson <em>et al.</em> 2017</a>). To avoid spurious isoforms with very low support, we excluded short (≤ 300 bp) and low expressed (total effective count ≤ 10 read count) isoforms prior to running Lace. In cases where all isoforms per cluster were filtered out after these exclusion criteria, the longest isoform was kept in the reference to avoid total exclusion of this sequence from the resulting reference transcriptome. We further identified SuperTranscripts that appeared to contain similar exonic sequences merged consecutively, as these kinds of “mosaics” could arise due to allelic variation and PSC. These were identified by self-blasting the SuperTranscripts. All SuperTranscripts that had blast hits within itself (other than the obvious 100% self-match) were identified as potential mosaics.</p><p id="p-15">Additionally, in order to inspect the effect on commonly applied clustering procedures that are used to decrease the allelic redundancy of assembled transcriptomes, the Trinity assembly selected (TRINITY<sub>guided</sub>) was clustered using CD-HIT-EST version 4.7 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li and Godzik 2006</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Fu <em>et al.</em> 2012</a>) with sequence identity cut-off (-c) 0.95, a commonly used threshold (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Wachowiak <em>et al.</em> 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hodgins <em>et al.</em> 2016</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Li <em>et al.</em> 2017</a>).</p></div><div id="sec-8" class="subsection"><h3>Quality assessment and completeness of secondary clustering references</h3><p id="p-16">Finally, we evaluated the reference transcriptome assemblies obtained from the secondary clustering using the same metrics used on the primary assemblies, obtained from TrinityStats, rnaQuast (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Bushmanova <em>et al.</em> 2016</a>) and Transrate (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Smith-Unna <em>et al.</em> 2016</a>) and estimated their completeness with BUSCO using the embryophyta_odb9 database (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Simão <em>et al.</em> 2015</a>) (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>). After this assessment, we selected four of the references obtained with the secondary clustering (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>, references marked with an *), which were used as reference transcriptomes to map the haploid (megagametophyte) and diploid tissues for assessment of the levels of PSC.</p></div><div id="sec-9" class="subsection"><h3>Assessment of the levels of PSC in references transcriptomes using haploid and diploid tissues</h3><p id="p-17">In order to evaluate the level of PSC on the obtained reference transcriptomes, we utilized the known ploidy level of two tissue types, the haploid (megagametophyte) and diploid tissues. In total, we employed six independent individuals, representing three different genotypes per individual: the vegetative tissues needle, phloem and vegetative bud (pooled reads of NE, PH, VB per individual) representing the diploid maternal genotype. The megagametophyte (ME) material containing a haploid maternal genome while the embryo (EM), being diploid but representing the next generation. One of the haploid genomes of the embryo is the same as the seed megagametophyte genome. The other haploid genome comes from an unknown pollinating father. Individual reads from ME, EM and NE+PH+VB of these six individuals were mapped against the seven selected reference transcriptomes obtained, four from the primary assemblies and another three obtained with the secondary clustering (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>, assemblies with an *). Mapping of the reads was performed with STAR aligner (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dobin <em>et al.</em> 2013</a>) given its suitability to allow spliced mapping of reads originating from RNA sequencing, <em>e.g.</em>, super transcriptome reference combining multiple isoforms (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Dobin <em>et al.</em> 2013</a>). We used default settings, with the exception of allowing reads to map to only one locus in the reference (–outFilterMultimapNmax 1). Otherwise, the read was considered unmapped. We also modified the default setting for filtering alignments with a mismatch of 0.025 (–outFilterMismatchNoverLmax 0.025). We used the two-pass mapping strategy (–twopassMode Basic) where STAR first performs the first pass mapping, then extracts junctions, inserts them into the genome index, and finally uses this information during the remapping of all reads in the second mapping step. Duplicates were removed with SAMtools (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2009</a>) and read groups added with the picard tool AddOrReplaceReadGroups. We generated three vcf files per reference transcriptome according to the ploidy of the tissues: one vcf for the ME (haploid), one for the EM (diploid), and one combining NE+PH+VB (diploid). Monomorphic and polymorphic sites were called with FreeBayes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Garrison and Marth 2012</a>) using default parameters, with exceptions of using a mutation rate (-T) of 0.005, excluding indels (-i), ignoring complex events (-u), and allowing no MNPs (multi-nucleotide polymorphisms) (-X). Each vcf file with both monomorphic and polymorphic sites was filtered with vcftools (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Danecek <em>et al.</em> 2011</a>) with a minimum depth per sample set to 10, and maximum amount of missing data 0.5 per site. Further filters were applied to polymorphic sites keeping only bi-allelic SNP sites with a quality &gt; 20. Number of heterozygous and homozygous variant calls were determined with vcftools–hardy option. Number of callable sites, sites with sufficient depth and amount of missing data per variant was determined for each transcript to allow comparable, per bp level nucleotide diversity estimates. Both observed and expected heterozygosity (π) (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Tajima 1989</a>) per nucleotide was calculated for each transcript individually. Given that the expected value of π is equal to θ (=4N<sub>e</sub>μ), we contrasted our observations to earlier, independent θ estimates for <em>P. sylvestris</em> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Pyhäjärvi <em>et al.</em> 2007</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kujala and Savolainen 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Grivet <em>et al.</em> 2017</a>). Heterozygous calls from reads originating from haploid tissues indicate paralog mapping or sequencing/genotype calling errors and we used this to compare the level of PSC among the reference transcriptomes.</p></div><div id="sec-10" class="subsection"><h3>Functional annotation and identification of contaminants on the assembled transcriptomes</h3><p id="p-18">The assembled transcriptomes selected as references for differential gene expression (TRINITY<sub>guided</sub>, see criteria later) was annotated using Trinotate, a pipeline for functional annotation of transcriptomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bolger <em>et al.</em> 2018</a>). First, similarities to known proteins were detected by a BLASTX search (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Camacho <em>et al.</em> 2009</a>) (e ≤ 1e−5) against two comprehensive protein databases: Swiss-Prot (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boeckmann <em>et al.</em> 2005</a>) and UniRef90 (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">The UniProt Consortium 2015</a>) obtained from UniProt (available on Mar 8, 2018). Coding regions within transcripts were predicted using TransDecoder (<span class="underline"><a href="http://transdecoder.github.io">http://transdecoder.github.io</a></span>). The protein products identified from TransDecoder were searched for sequence similarities against the Swiss-Prot and UniRef90 protein database and for conserved protein domains using Hmmer (<a href="http://hmmer.org/">http://hmmer.org/</a>) and PFam (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Finn <em>et al.</em> 2014</a>). All results were parsed by the Trinotate pipeline (<span class="underline"><a href="https://trinotate.github.io">https://trinotate.github.io</a></span>) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bryant <em>et al.</em> 2017</a>), stored in a SQLite relational database, and reported as a tab-delimited transcript annotation file.</p><p id="p-19">We also examined whether possible contaminants were present in the final assemblies. In order to identify contaminant sequences (not belonging to <em>P. sylvestris</em>), we performed a BLASTx search of the assembled reference (TRINITY<sub>guided</sub>) against Swiss-Prot limited to sequences classified as bacteria, viruses, metazoan, alveolata and/or fungi. We identified transcripts that had a BLASTx hit e ≤ 1e−5 and sequence similarity of at least 65%. Transcripts potentially originating from organelle genomes were identified by BLAST against <em>P. taeda</em> and <em>P. lambertiana</em> mitochondrial genomes and <em>P. sylvestris</em> and <em>P. mugo</em> chloroplast genomes (NCBI GenBank IDs <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JN854158.1&amp;atom=%2Fggg%2F9%2F10%2F3409.atom">JN854158.1</a> and KX833097.1) with e-value cutoff 5e-2, identity cutoff 80% and word size = 60. BLAST search of the transcripts against the <em>P. taeda</em> (v.2.01) (<a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Zimin <em>et al.</em>, 2014</a>) and <em>P. lambertiana</em> (v.1.0,) (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Stevens <em>et al.</em> 2016</a>) reference genome sequences was used as an additional method to identify single-copy and multi-copy transcripts. If any region of a given transcript had multiple BLAST hits with more than 85% identity, the transcript was assigned as multi-copy. We excluded 10 bp from each edge of the alignments to avoid random alignment in the edges. In addition, we required that 50% of the transcript length had sequence similarity in the corresponding reference.</p></div><div id="sec-11" class="subsection"><h3>Completeness of assembled transcriptomes and comparisons with published transcriptomes of P. sylvestris</h3><p id="p-20">In order to determine the completeness of the transcriptome references obtained for this species, we used the core set of genes (embryophyta_odb9) dataset in BUSCO (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Simão <em>et al.</em> 2015</a>). Although this core set is only based on angiosperm taxa, it provides an estimate for the completeness of a core set of genes, also for gymnosperms. We evaluated three of the references obtained here (TRINITY<sub>guided</sub>, TRINITY<sub>CD-HIT</sub>, and TRINITY<sub>Lace</sub>), and compared them to previously published assemblies for <em>P. sylvestris</em>. Transcriptomes for this species have been assembled from heartwood of wounded and unwounded seedlings (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Godfrey 2012</a>), needles of two-year-old seedlings collected from five individuals (genotypes) (<a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">Wachowiak <em>et al.</em> 2015</a>), embryos and megagametophytes at different developmental stages from a single individual (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Merino <em>et al.</em> 2016</a>). References are also available from wood cores collected from four individual mature trees (35 to 46-years-old) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lim <em>et al.</em> 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lim 2017</a>) and pollen (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Höllbacher <em>et al.</em> 2017</a>).</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-21">The workflow with all the commands used is deposited in <a href="https://github.com/DI-Ojeda/Pin_syl_transcriptome">https://github.com/DI-Ojeda/Pin_syl_transcriptome</a>. Raw reads were deposited in the NCBI SRA repository, under BioProject PRJNA531617 (SRR8996768-SRR8996761). Assemblies obtained from both programs using the three strategies (A, B and C), the set of <em>P. sylvestris</em> orthologs identified with the OGA approach with all five data sets, and the assemblies obtained using the two additional secondary clustering strategies are deposited in the NCBI TSA repository, under BioProject PRJNA531617. Files S1-S8 contains the supplementary figures and files. Tables S1-S6 contain the supplementary tables, supplemental material available at figshare: <a href="https://doi.org/10.6084/m9.figshare.7623746">https://doi.org/10.6084/m9.figshare.7623746</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Primary assemblies</h3><p id="p-22">Individual assemblies (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a>, strategy A) obtained with the CLCbio Genomic Workbench (CLCbio<sub>individual</sub>) contained overall less contigs (average 84,181) than TRINITY<sub>individual</sub> assemblies (average 149,943) (Table S2) and displayed slightly (on average 5%) lower BUSCO completeness scores (Figures S1 and S2). We obtained similar results in terms of average contig size, longest assembled contig, and N50 among the four assemblies obtained with the pooled assembly strategies (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1</a>, strategies B and C) with Trinity and CLCbio (Table S3). A higher number of contigs was obtained with Trinity regardless if strategy B or C was used, and this was also reflected in the number of predicted ORFs. Using the megagametophyte &gt; 500 bp contigs (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1</a>, strategy C) as a guide during assembly increased the total number of contigs obtained in the second assembly on both algorithms. A large proportion of transcripts assembled with Trinity have low expression values. For instance, we obtained 52,208 genes with a &gt;10 TPM level of expression for the TRINITY<sub>guided</sub> assembly (Figure S4). We obtained a slightly larger number of genes with the TRINITY<sub>notguided</sub> strategy (64,986 genes, E90N50 = 2,158, the N50 value obtained with only a level of expression of 90%) than with the TRINITY<sub>guided</sub> strategy (50,760 genes, E90N50 = 2,388) when a cut-off expression value of &gt;90% was used (Figure S5). BUSCO completeness scores were also higher on these latter assemblies compared with strategy B. The highest BUSCO completeness scores and the lowest number of fragmented transcripts was obtained with the TRINITY<sub>guided</sub> assembly (Figure S6 A).</p></div><div id="sec-15" class="subsection"><h3>Secondary clustering: Orthology Guided Assembly (OGA)</h3><p id="p-23">We used the individual assemblies obtained with the CLCbio (CLCbio<sub>individual</sub>), which contained overall considerably lower complexity (measured by the number of unique contigs per sample, as function of the total number of reads available) (Figure S3). Given its reduced size compared to TRINITY assemblies, the identification of orthologs with the proteomes of <em>P. lambertiana</em> and <em>P. taeda</em> required less computational time. We combined the individual transcripts for each of the 30 samples resulting in a total of 2,495,509 transcripts (average length = 642.76 bp, N50-length = 930 bp) and used them as an input for the identification of orthologs with the OGA approach. Overall, we obtained a higher number of orthologs of <em>P. sylvestris</em> using the ALL proteome datasets for both reference species, with higher numbers with <em>P. taeda</em> (27,241 orthologs) than <em>P. lambertiana</em> (22,807 orthologs) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and S5). This was also reflected in the BUSCO completeness scores, where a more complete core gene set was recovered with the ALL proteome datasets for <em>P. taeda</em> (42.6%) than for <em>P. lambertiana</em> (42.0%) (Table S4 and Figure S6B). The ability to recover a full-length <em>P. sylvestris</em> transcript per gene with the OGA approach was determined by plotting the distribution of the fraction of the assembled transcript length <em>vs.</em> the respective <em>P. taeda</em> or <em>P. lambertiana</em> reference sequence length. We found that the majority of the transcripts obtained using the five reference proteomes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) were fragmented (broken or partially assembled transcripts) (Figure S7). For instance, using the latest version of the <em>P. taeda</em> protein set (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, ALL dataset ver. 2.01) resulted in the recovery of 13,131 <em>P. sylvestris</em> transcripts (out of the 36,732 <em>P. taeda</em> reference proteins), but only one-third of the <em>P. sylvestris</em> transcripts encodes a full-length protein with a similar length as the <em>P. taeda</em> reference protein (ratio close to 1) (Figure S7).</p></div><div id="sec-16" class="subsection"><h3>Secondary clustering: Reduction of allelic redundancy with Lace and CD-HIT</h3><p id="p-24">After the previous assessments, the TRINITY<sub>guided</sub> reference was further analyzed to reduce its redundancy with the construction of SuperTranscrips (Lace) and with CD-HIT. From the original set of 1,307,499 contigs in the TRINITY<sub>guided</sub> assembly, a total of 787,820 SuperTranscrips were obtained with Lace, of which 71,344 (9%) included multiple isoforms. Out of the multi-isoform clusters, most had a few (&lt;5) isoforms (Table S5). With CD-HIT, we further reduced the number of transcripts by 9%, resulting in a set of 717,762 transcripts.</p></div><div id="sec-17" class="subsection"><h3>Assessment of levels of paralog sequence collapse (PSC)</h3><p id="p-25">To evaluate the level of paralog read mapping due to PSC on the reference transcriptomes obtained, patterns of heterozygosity among haploid and diploid tissues were compared on the seven selected reference transcriptomes (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1</a>). The summary of SNP and genotype calling results is presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. It is noteworthy that both the size of the transcriptome and the number of callable sites (monomorphic and polymorphic sites passing the filters) are very different among assemblies, the latter varying from 6.7 × 10<sup>6</sup> callable sites in OGA<sub>PilaALL</sub> to 102 × 10<sup>6</sup> callable sites in the TRINITY<sub>CD-HIT</sub>. Larger references resulted in lower average depth and vice versa, because smaller assemblies force the same amount of reads to map to smaller number of locations. In addition, the number of callable sites will act as a denominator in per nucleotide estimates, thus reducing the apparent level of diversity in the reference transcriptomes with more callable sites. Therefore, for evaluating the behavior of different assemblers, we also considered the haploid/diploid ratio of H<sub>o</sub> in addition to the absolute number of variants per nucleotide, as the ratio is less affected by the assembly size.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494523/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494523/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of the assessment of PSC levels utilizing observed heterozygosity patterns on the seven reference transcriptomes selected. NA = not analyzed. *Only including genes where there are &gt;100 callable sites. <sup>1</sup>Rough estimate for total sites from <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Grivet <em>et al.</em> (2017)</a>, assuming that one-third of coding sites are non-synonymous and two-third are synonymous. <sup>2</sup>Assuming Hardy-Weinberg equilibrium</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">As expected, observed heterozygosity per nucleotide was lower in megagametophytes than in diploid tissues on all the assemblies assessed. However, some observed heterozygosity in haploid tissue in all assemblies suggest that all the assessed transcriptomes still result in considerable amount of false heterozygous calls, either originating from paralog read mapping or from errors during mapping and variant calling (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). We found that the relative amount of false heterozygous calls (ratio of H<sub>o</sub>) was lowest in TRINITY<sub>CD-HIT</sub> (TRINITY<sub>guided</sub> assembly, followed by creation of SuperTranscripts and clustering redundant contigs with CD-HIT, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). In contrast, haploid <em>vs.</em> diploid observed H<sub>O</sub> ratio was the highest among all assemblies in OGA assemblies. Also, the H<sub>o</sub>/H<sub>E</sub> ratio was considerably higher than one for both OGA assemblies in diploid tissues suggesting excess heterozygote calls likely due to PSC.</p><p id="p-27">As a further validation of our methods of identifying levels of paralog read mapping based on haploid heterozygosity, we compared the heterozygosity level in single and multi-copy genes identified in the TRINITY<sub>Lace</sub> (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1</a>). We expected the paralog mapping and hence haploid observed heterozygosity to be higher in multi-copy genes that are prone to PSC. In diploid genotypes, multi-copy genes had slightly higher heterozygosity than single copy genes. In haploid genotypes, the majority (93%) of genes with H<sub>o</sub> &gt; 0 were found in multi-copy genes (Figure S8). On the other hand, 30% of the multi-copy genes, but only 14% of the single-copy genes (where we obtained sufficient data for SNP calling) have H<sub>o</sub> &gt; 0. Thus, the two independent methods yield consistent but not completely overlapping methods to identify loci prone to PSC.</p></div><div id="sec-18" class="subsection"><h3>BUSCO completeness and functional annotation of the assembled transcriptome</h3><p id="p-28">Our final assemblies ranged between 61.0% and 82.7% completeness, with the highest level recovered in the TRINITY<sub>guided</sub> assembly (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In contrast to previous published assembled reference transcriptomes of <em>P. sylvestris</em> that used only one or two different tissues (<a id="xref-ref-84-4" class="xref-bibr" href="#ref-84">Wachowiak <em>et al.</em> 2015</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Lim <em>et al.</em> 2016</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Merino <em>et al.</em> 2016</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Höllbacher <em>et al.</em> 2017</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lim 2017</a>), our assemblies were based on five different tissues (needle, phloem, vegetative bud, embryo and megagametophyte) from six individuals. The number of full-length genes captured in the TRINITY<sub>guided</sub> assembly is also reflected in the number of ORFs predicted with TransRate, which was larger than the previously available reference for <em>P. sylvestris</em> (Table S6). The TRINITY<sub>guided</sub> reference therefore represents the most complete available resource for this species to date. Using the Trinotate pipeline we annotated the 1.3 million transcripts of this assembly and found a large number (50,760) of likely full-length transcripts enriched in the E90 set (Figure S5a) and at least 64,260 complete ORFs predicted with Transdecoder. In addition, Trinotate found 24,780 genes matching Pfam Tracheophyta protein hit (Supplementary Data S1).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of completeness on the core set of genes in BUSCO of the assemblies obtained in this study in comparison with published transcriptomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1195211802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percentage of completeness on the core set of genes in BUSCO of the assemblies obtained in this study in comparison with published transcriptomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3409/F2.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3409/F2.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494450" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Percentage of completeness on the core set of genes in BUSCO of the assemblies obtained in this study in comparison with published transcriptomes.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-30">Considering the large and complex genomes of conifers, transcriptomes are increasingly being used as a reference resource in a variety of applications (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Müller <em>et al.</em> 2015</a>; <a id="xref-ref-84-5" class="xref-bibr" href="#ref-84">Wachowiak <em>et al.</em> 2015</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Suren <em>et al.</em> 2016</a>). Particularly in Pinaceae, where the vast majority of transcriptomes have been generated (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">López de Heredia and Vázquez-Poletti 2016</a>), the number of assembled contigs (transcripts) is always larger than the actual number of genes estimated based on their genome sequence (Table S6) (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gonzales-Ibeas <em>et al.</em> 2016</a>). This is not a unique to conifers, but it is a common output in assembled transcriptomes of other organisms. Splitting of alleles originating from heterozygote material, pooling of multiple genotypes, contaminants and rare transcripts accumulating with increased sequence depth all have potential contribution to the increase the number of contigs in <em>de novo</em> assembled transcriptomes. On the other hand, paralogous gene sequences are erroneously collapsed into a single reference sequence, causing paralog sequence collapse (PSC). This is one of the main limitations and potential source of error is the level PSC in the reference transcriptome. There are currently several strategies to deal with redundancy, but most rely on the availability of available genomes and/or high quality annotated references.</p><p id="p-31">One characteristic of conifers (and other gymnosperms) is the presence of easily accessible megagametophyte tissue, and their amenity to extract haploid genome information from it. Here we employed two popular assemblers with contrasting features (in terms of availability and output), Trinity and CLC Genomic Workbench for our primary assemblies with the purpose to provide reference to benchmarked data on <em>de novo</em> assembly. Then we employed alternative strategies to reduce paralogs with the secondary clustering. Our selection of secondary clustering strategies was based on strategies previously employed on model and non-model systems and that could be a useful comparison for <em>P. sylvestris</em> (and most gymnosperms) with limited available genomic resources. We used the variation of ploidy in <em>P. sylvestris</em> tissues to assess their utility during the <em>de novo</em> assembly and employ a novel strategy to estimate levels of PSC on the assembled transcriptomes. The same approach could be utilized in any species with haploid material available (<em>e.g.</em>, social insects, haplontic plants and fungi) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sandler <em>et al.</em> 2018</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Yahav and Privman 2019</a>).</p><div id="sec-20" class="subsection"><h3>Improving de novo assembly transcriptomes with the haploid megagametophyte tissue</h3><p id="p-32">The most common strategy applied in <em>de novo</em> transcriptome assembly involves combining read data from several genotypes (individuals), developmental stages, and tissues of the target species. This is justified as a mean to capture most of the genes expressed under a variety of conditions and individuals. However, this also causes high levels of allelic redundancy and transcript fragmentation due to the high levels of heterozygosity and the genetic diversity across several genotypes (<a id="xref-ref-68-5" class="xref-bibr" href="#ref-68">Ruttink <em>et al.</em> 2013</a>). In conifers, this is the strategy routinely employed to generate <em>de novo</em> reference transcriptomes (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">López de Heredia and Vázquez-Poletti 2016</a>), and it invariably leads to a large number of transcripts. Among the three strategies used here (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1</a>), combining all tissues and genotypes (strategy B) resulted in an assembly with the highest number of assembled contigs and predicted ORFs, but without necessarily being the most complete assembly (Table S3). On the other hand, the inclusion of the megagametophyte (haploid) during the <em>de novo</em> assembly as long read guidance (<a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1C</a>), increased the BUSCO completeness scores for both assemblers (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Contrary to <em>de novo</em> transcriptome hybrid assembly, where long reads (such as PacBio) are aligned to the short reads during the assembly (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fu <em>et al.</em> 2018</a>), the long reads we used here from the megagametophyte (&gt;500 bp) were not incorporated into the final assembly, and were only used to resolve isoforms during assembly. An additional strategy for generating more complete transcriptomes is the utilization of long read technologies (<em>e.g.</em>, Iso-Seq from PacBio or direct RNA from Nanopore), which allows to obtain full-length splice isoforms and several other post-transcriptional events (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Zhao <em>et al.</em> 2019</a>), with a few available approaches to cluster transcripts based on gene sequences (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Marchet <em>et al.</em> 2019</a>). The main benefit obtained from incorporating haploid tissue reads during the assembly stage is the increase of transcriptome completeness. The inclusion of haploid tissue did not considerably affect the number of transcripts or the ORFs predicted, regardless of the assembler used (Table S3), which were still considerably larger than the estimated number of genes in conifer genomes (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-92-3" class="xref-bibr" href="#ref-92">Zimin <em>et al.</em> 2014</a>; <a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Stevens <em>et al.</em> 2016</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Neale <em>et al.</em> 2017</a>).</p></div><div id="sec-21" class="subsection"><h3>Levels of paralog sequence collapse in the assembled transcriptomes and the effect of redundancy reducing strategies</h3><p id="p-33">In conifers, CD-HIT clustering is the most common strategy used to reduce allelic redundancy, along with CAP3, with similar levels of effectiveness using either strategy (<a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Yang and Smith 2013</a>). In contrast, generation of gene-based consensus transcript using isoform alignment (Lace) has been rarely applied in conifer trees (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Ueno <em>et al.</em> 2018</a>). The CD-HIT algorithm was originally designed to reduce large protein data sets into representative sequences based on homology (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Li <em>et al.</em> 2001</a>), not to reduce redundancy in transcriptomes. In our study, CD-HIT reduced the number of transcripts and predicted ORFs more than the generation of SuperTranscripts with Lace alone (Table S4). Both Lace and CD-HIT reduced the percentage of duplicated genes in the BUSCO set to about 5% of the genes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). As a side effect, applying these clustering steps reduce the BUSCO completeness scores of the transcriptomes by nearly 20% in <em>P. sylvestris</em> (Table S4). In addition, Lace resulted in the inclusion of duplicated exons on the generated sequences, resulting in the generation of artificial mosaic transcripts. We identified a total of 44,535 mosaics on this assembly before applying the CD-HIT clustering. Contrary to CD-HIT and Lace, the OGA approach was less successful in the reduction of false heterozygous calls. For OGA assemblies, the observed heterozygosity was 25% higher than the expected heterozygosity, indicating a higher number of false heterozygous calls (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). This was mainly due to the lack of completeness of the <em>P. taeda</em> and <em>P. lambertiana</em> reference genome annotations. As better and more complete reference genomes become available in conifers, this approach might be an effective strategy to reduce allelic redundancy, PSC and paralog read mapping.</p></div><div id="sec-22" class="subsection"><h3>Effect of allelic redundancy on the development of exome capture probes and strategies to mitigate its effect</h3><p id="p-34">Appropriate selection of target genes is a crucial step for an effective target exome capture experiment, as the presence of paralog sequences, misassembled regions, mosaics, cpDNA and mtDNA decrease the efficiency of the recovery of the captured regions. When the design is entirely based on an assembled transcriptome, the most important step is to select the most appropriate contigs for bait design. For those references obtained with programs that group assembled contigs by graph component (multiple isoforms) such as Trinity, a representative sequence must be selected for bait design. This is usually accomplished by selecting the isoform with a minimum length (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Suren <em>et al.</em> 2016</a>), or combining information from isoform size and their expression level (<a id="xref-ref-89-3" class="xref-bibr" href="#ref-89">Yang and Smith 2013</a>). Based on our estimates of PSC, collapsing paralogs with Lace and CD-HIT after a haploid-guided assembly is a reasonable strategy to select suitable candidate genes. However, additional considerations should be added to further select the most suitable regions for bait design (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Additional steps must include the removal of contaminant regions (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Howe <em>et al.</em> 2013</a>), genes from the organelle genomes (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Syring <em>et al.</em> 2016</a>), exclusion of potential mosaics produced by Lace, and the exclusion of transcripts with low level of expression. The exclusion of these regions will increase the efficiency in exome capture experiments, as well as of more accurate estimates of genetic diversity in population genetic analyses. In our case, 0.4% of the transcripts had a significant hit to chloroplast or mitochondrial <em>Pinus</em> genomes in the TRINITY<sub>Lace</sub> assembly. Excluding genes with low levels of expression (cut-off of &gt;10 TPM) reduced the number of genes (including their isoforms) to 108,860 after the TRINITY<sub>guided</sub> assembly (Figure S5B). These low expressed genes are probably enriched with assembly artifacts and therefore not desirable in the bait design. For future reference and to help exome capture bait design of <em>P. sylvestris</em>, information on copy number, organelle genome match, isoform number, expression, putative contaminants and mosaics at gene level for Trinity<sub>Lace</sub> gene level assembly is provided in Data S1.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strategies to employ the availability of haploid tissue in conifers for de novo transcriptome assembly and their application to assess the amount of allelic redundancy and paralog sequence collapse (PSC). Green boxes indicate additional steps recommended when additional genomic resources are available from a related species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1195211802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Strategies to employ the availability of haploid tissue in conifers for &lt;em&gt;de novo&lt;/em&gt; transcriptome assembly and their application to assess the amount of allelic redundancy and paralog sequence collapse (PSC). Green boxes indicate additional steps recommended when additional genomic resources are available from a related species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3409/F3.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3409/F3.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3409/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494384" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Strategies to employ the availability of haploid tissue in conifers for <em>de novo</em> transcriptome assembly and their application to assess the amount of allelic redundancy and paralog sequence collapse (PSC). Green boxes indicate additional steps recommended when additional genomic resources are available from a related species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">One aspect that has been rarely explored in the assembled transcriptomes of conifers is the proportion of chimaeras, which has been estimated between 4–9% in the assembled transcriptomes evaluated so far (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Ueno <em>et al.</em> 2018</a>). There are currently available strategies to identify chimeras in assembled transcriptomes (<em>e.g.</em>, <a id="xref-ref-89-4" class="xref-bibr" href="#ref-89">Yang and Smith 2013</a>), but these rely on the quality of annotated proteomes from closely related taxa. Thus, how effective this strategy is in conifers remains to be determined. Finally, the intron-exon boundaries of the selected genes can be inferred using annotated genome references of related species, and excluded from the target regions of the bait design, as it has been shown to improve capture efficiency due to the hybridization on the probe-exon regions (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Neves <em>et al.</em> 2013</a>; <a id="xref-ref-75-4" class="xref-bibr" href="#ref-75">Suren <em>et al.</em> 2016</a>). However, the success of predicting these boundaries rely on the quality of available reference genomes from closely related taxa or low coverage shotgun sequencing of the target species. Depending on the availability of genomic resources from closely related taxa, we suggest several tailored steps that can be integrated to select suitable target genes for bait design (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p></div><div id="sec-23" class="subsection"><h3>Influence of transcriptome assembly strategies on estimates of genetic diversity</h3><p id="p-37">The use of transcriptomes as a reference for obtaining SNPs and estimating genetic diversity is becoming a common strategy for non-model species that lack a reference genome (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Gayral <em>et al.</em> 2013</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Romiguier <em>et al.</em> 2014</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Xu <em>et al.</em> 2016</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Yan <em>et al.</em> 2017</a>). However, several factors can introduce errors, and careful assessment of the assembled transcriptome is required before employing this downstream application. These include the presence of alleles, isoforms, and paralogs on the assembled transcriptome contigs. For instance, allelic redundancy allows reads to map to their respective allele without mismatches, and because variant calling algorithms can report no polymorphisms, this will cause an underestimation of genetic diversity. On the other hand, too greedy clustering of transcripts will result in PSC, the collapse of paralogous sequences and the removal of some of these from the reference. As a result, reads derived from the genes that were removed from the reference may map onto the remaining sequence and variant calling algorithms will report nucleotide substitutions that differentiate the paralogous genes as SNPs (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Gayral <em>et al.</em> 2013</a>).</p><p id="p-38">Overall, the H<sub>e</sub> per nucleotide for the assembled transcriptomes we assessed was lower than expected based on previous studies (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Grivet <em>et al.</em> 2017</a>), ranging from 1.2 × 10<sup>−4</sup> to 8.3 × 10<sup>−4</sup> (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>), an order of magnitude lower than previously reported in <em>P. sylvestris</em> (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Pyhäjärvi <em>et al.</em> 2011</a>). The minimum required read depth and missing data threshold we applied to our data set were relatively lenient, which has probably increased the number of callable sites relative to variant calls. In addition, allele-specific expression can also reduce the observed genetic diversity measured using RNA-seq data. Note that in this study the measures of diversity among haploid and diploid samples were used to evaluate different assemblies and were based on relative levels of heterozygosity. For precise estimates of genetic diversity, datasets with reads originating from DNA, deeper sequencing, more stringent filters and larger number of individuals should be used. We additionally suggest careful analysis of genetic diversity from multi-copy genes as they are especially prone to paralog mapping. Our estimation of heterozygosity on the haploid tissues were higher for multi-copy than single-copy genes (Figure S8), and the exclusion of multi-copy genes will decrease false SNPs in downstream analysis. This approach, however, comes with a caveat of possible bias resulting from ignoring fast evolving gene families from the analysis (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-39">In conclusion, we found that the use of individual assemblies obtained from the haploid tissue as a guide improves <em>de novo</em> assembly and can be also be employed to assess the levels of paralogy; thus, the availability of haploid tissue could be also exploited on other organisms that lack available genome references. Strategies employed to decrease redundancy cause paralog sequence collapse and the reduction of transcriptome completeness. We suggest that collapsing paralogs with Lace and CD-HIT after a haploid-guided assembly is a reasonable strategy to generate reference transcriptomes with reduced levels of paralogy. Precautions should be taken to inspect the output from Lace, as we found that it sometimes produces mosaic transcripts. Elimination of lowly expressed genes, contaminants, and multi-copy genes will decrease false SNPs in downstream analysis. The reference transcriptomes obtained using these strategies should provide less biased estimates on population genetic analyses, as well as to select more suitable regions for target enrichment.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We acknowledge the advice and help on bioinformatics from Lieven Sterck, John Liechty, Sam Yeaman, Henrik Hedberg, Tuomas Jormola, Jaakko Tyrmi, Witold Wachowiak and Nader Aryamanesh. Special thanks to Brian Haas for his helpful suggestions regarding the assemblies with Trinity. We thank Outi Savolainen and Komlan Avia for providing access to their experimental material. Soile Alatalo and Sami Saarenpää are acknowledged for skilled field and laboratory work. This work was funded by the Academy of Finland (287431 and 293819 to TP, GENOWOOD no. 307582), and the European Commission (EVOLTREE <a href="http://www.evoltree.eu/">http://www.evoltree.eu/</a>). We thank the CSC-IT Center for Science, Finland, for computational resources.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-41">Supplemental material available at figshare: <a href="https://doi.org/10.6084/m9.figshare.7623746">https://doi.org/10.6084/m9.figshare.7623746</a>.</p></li><li class="fn" id="fn-2"><p id="p-42"><em>Communicating editor: A. Eckert</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-17"><span class="received-label">Received </span>May 17, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-16"><span class="accepted-label">Accepted </span>August 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Ojeda <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> FastQC: a quality control tool for high throughput sequence data.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.2" data-doi="10.1371/journal.pone.0173300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Armero</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Baudouin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bocs</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">This</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improving transcriptome de novo assembly by using a reference genome of a related species: Translational genomics from oil palm to coconut.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e0173300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0173300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0173300%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0173300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0173300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wegrzyn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Sezen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Falk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Maloney</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> Comparative transcriptomics among four white pine species. G3 (Bethesda) 8: 1461–1474.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.4" data-doi="10.1093/bioinformatics/btt127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Jiang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Girke</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">BRANCH: Boosting RNA-Seq assemblies with partial or related genomic sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1250</span>–<span class="cit-lpage">1259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt127</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt127%26rft_id%253Dinfo%253Apmid%252F23493323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23493323&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.5" data-doi="10.1093/bioinformatics/btt178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Raymond</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Jackman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pleasance</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Coope</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembling the 20 Gb white spruce (<em>Picea glauca</em>) genome from whole-genome shotgun sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1492</span>–<span class="cit-lpage">1497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt178%26rft_id%253Dinfo%253Apmid%252F23698863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698863&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320065400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.6" data-doi="10.1016/j.crvi.2005.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeckmann</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Blatter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Famiglietti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Hinz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lane</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Protein variety and functional diversity: Swiss-Prot annotation in its biological context.</span> <abbr class="cit-jnl-abbrev">C. R. Biol.</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">882</span>–<span class="cit-lpage">899</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.crvi.2005.06.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComptes%2Brendus%2Bbiologies%26rft.stitle%253DC%2BR%2BBiol%26rft.aulast%253DBoeckmann%26rft.auinit1%253DB.%26rft.volume%253D328%26rft.issue%253D10-11%26rft.spage%253D882%26rft.epage%253D899%26rft.atitle%253DProtein%2Bvariety%2Band%2Bfunctional%2Bdiversity%253A%2BSwiss-Prot%2Bannotation%2Bin%2Bits%2Bbiological%2Bcontext.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.crvi.2005.06.001%26rft_id%253Dinfo%253Apmid%252F16286078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.crvi.2005.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16286078&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233663500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.7" data-doi="10.1093/bib/bbw135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Arsova</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Plant genome and transcriptome annotations: from misconceptions to simple solutions.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbw135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft.volume%253D19%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbw135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbw135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.8" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.9" data-doi="10.1016/j.celrep.2016.12.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>,  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">DiTommaso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Tickle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Couger</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A tissue-mapped axolotl <em>de novo</em> transcriptome enables identification of limb regeneration factors.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2016.12.063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D18%26rft.spage%253D762%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2016.12.063%26rft_id%253Dinfo%253Apmid%252F28099853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2016.12.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28099853&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.10" data-doi="10.1093/bioinformatics/btw218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bushmanova</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Antipov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lapidus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Suvorov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Prjibelski</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">rnaQUAST: a quality assessment tool for <em>de novo</em> transcriptome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2210</span>–<span class="cit-lpage">2212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw218</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw218%26rft_id%253Dinfo%253Apmid%252F27153654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153654&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.11" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coulouris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Avagyan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Papadopoulos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: Architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.12" data-doi="10.1111/pbi.12136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Canales</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bautista</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Label</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gómez-Maldonado</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Lesur</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">De novo assembly of maritime pine transcriptome: Implications for forest breeding and biotechnology.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D12%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12136%26rft_id%253Dinfo%253Apmid%252F24256179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24256179&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333027500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.13" data-doi="10.1111/tpj.13673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Celedon</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M. S.</span> <span class="cit-name-surname">Yuen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chiang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Henderson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Reid</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Cell-type- and tissue-specific transcriptomes of the white spruce (<em>Picea glauca</em>) bark unmask fine-scale spatial patterns of constitutive and induced conifer defense.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D92%26rft.spage%253D710%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.14" data-doi="10.3389/fpls.2018.00418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Dong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The sequenced angiosperm genomes and genome databases.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">418</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00418</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D418%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.15" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Auton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Abecasis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Albers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.16" data-doi="10.1186/s13059-017-1284-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D. K.</span> <span class="cit-name-surname">Hawkins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Oshlack</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">SuperTranscripts: A data driven reference for analysis and visualisation of transcriptomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1284-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1284-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1284-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.17" data-doi="10.1186/s13059-014-0410-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">N. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Oshlack</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Corset: enabling differential gene expression analysis for <em>de novo</em> assembled transcriptomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D410%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0410-6%26rft_id%253Dinfo%253Apmid%252F25063469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25063469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.18" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Schlesinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drenkow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zaleski</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: Ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.19" data-doi="10.1093/nar/gkt1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>,  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bateman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Clements</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Coggill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. Y.</span> <span class="cit-name-surname">Eberhardt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pfam: The protein families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D222</span>–<span class="cit-lpage">D230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1223</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1223%26rft_id%253Dinfo%253Apmid%252F24288371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24288371&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.20" data-doi="10.1093/bioinformatics/bty098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">IDP-denovo: <em>De novo</em> transcriptome assembly and isoform annotation by hybrid sequencing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2168</span>–<span class="cit-lpage">2176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D2168%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.21" data-doi="10.1093/bioinformatics/bts565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Niu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CD-HIT: accelerated for clustering the next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3150</span>–<span class="cit-lpage">3152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts565</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts565%26rft_id%253Dinfo%253Apmid%252F23060610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060610&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311902700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Marth</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Haplotype-based variant detection from short-read sequencing. arXiv 1207.3907.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.23" data-doi="10.1371/journal.pgen.1003457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gayral</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Melo-Ferreira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Glémin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Bierne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Carneiro</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reference-free population genomics from next-generation transcriptome data and the vertebrate-invertebrate gap.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003457</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003457%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003457%26rft_id%253Dinfo%253Apmid%252F23593039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23593039&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.24" data-doi="10.1016/j.cpb.2017.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geniza</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jaiswal</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Tools for building de novo transcriptome assembly.</span> <abbr class="cit-jnl-abbrev">Curr. Plant Biol.</abbr> <span class="cit-vol">11–12</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cpb.2017.12.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BPlant%2BBiol.%26rft.volume%253D1112%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpb.2017.12.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpb.2017.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3409.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Godfrey</span>,  <span class="cit-name-given-names">O. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Pine transcriptomics - RNA-Seq data analysis of Scots pine (Pinus sylvestris) seedlings subjected to a wounding experiment</span>, <span class="cit-publ-name">University of Helsinki, Helsinki, Finland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.26" data-doi="10.1111/1755-0998.12657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gompert</span>,  <span class="cit-name-given-names">Z.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Mock</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Detection of individual ploidy levels with genotyping-by-sequencing (GBS) analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1156</span>–<span class="cit-lpage">1167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12657</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D1156%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzales-Ibeas</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Martinez-Garcia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Famula</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Delfino-Mix</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessing the gene content of the megagenome: sugar pine (<em>Pinus lambertiana</em>). Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3787</span>–<span class="cit-lpage">3802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D6%26rft.spage%253D3787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.28" data-doi="10.1111/mec.14402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grivet</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Avia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vaattovaara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Eckert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Neale</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High rate of adaptive evolution in two widespread European pines.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">6857</span>–<span class="cit-lpage">6870</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14402</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D6857%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Papanicolaou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yassour</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Grabherr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Blood</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> De novo transcript sequence reconstruction from RNA-Seq: reference generation and analysis with Trinity. Nat. Protoc. 8: 10.1038.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.30" data-doi="10.1093/molbev/msw032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgins</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yeaman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Nurkowski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. H.</span> <span class="cit-name-surname">Rieseberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Aitken</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Expression divergence is correlated with sequence evolution but not positive selection in conifers.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1502</span>–<span class="cit-lpage">1516</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw032%26rft_id%253Dinfo%253Apmid%252F26873578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26873578&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.31" data-doi="10.3390/ijms18061199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Höllbacher</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. O.</span> <span class="cit-name-surname">Schmitt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hofer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ferreira</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lackner</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Identification of proteases and protease inhibitors in allergenic and non-allergenic pollen.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/ijms18061199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D18%26rft.spage%253D1199%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms18061199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms18061199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.32" data-doi="10.1093/gigascience/giz039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hölzer</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marz</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title"><em>De novo</em> transcriptome assembly: A comprehensive cross-species comparison of short-read RNA-Seq assemblers.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">giz039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giz039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D8%26rft.spage%253Dgiz039%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiz039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giz039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.33" data-doi="10.1186/1471-2164-14-137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>,  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Knaus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cronn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kolpak</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A SNP resource for Douglas-fir: <em>De novo</em> transcriptome assembly and SNP detection and validation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-137</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHowe%26rft.auinit1%253DG.%2BT.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D137%26rft.epage%253D137%26rft.atitle%253DA%2BSNP%2Bresource%2Bfor%2BDouglas-fir%253A%2Bde%2Bnovo%2Btranscriptome%2Bassembly%2Band%2BSNP%2Bdetection%2Band%2Bvalidation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-137%26rft_id%253Dinfo%253Apmid%252F23445355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23445355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.34" data-doi="10.1371/journal.pone.0148985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">X. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. Q.</span> <span class="cit-name-surname">Jin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">De novo transcriptome assembly and characterization for the widespread and stress-tolerant conifer <em>Platycladus orientalis</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0148985</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0148985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0148985%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0148985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0148985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.35" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT — The BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zNDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.36" data-doi="10.1007/s11295-012-0532-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kujala</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Savolainen</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequence variation patterns along a latitudinal cline in Scots pine (Pinus sylvestris): Signs of clinal adaptation?</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-012-0532-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D1451%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-012-0532-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-012-0532-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.37" data-doi="10.1093/molbev/msx069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De La Torre</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Van De Peer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. K.</span> <span class="cit-name-surname">Ingvarsson</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Contrasting rates of molecular evolution and patterns of selection among gymnosperms and flowering plants.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1363</span>–<span class="cit-lpage">1377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D1363%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.38" data-doi="10.1093/gbe/evv044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De La Torre</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. C.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Van De Peer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. K.</span> <span class="cit-name-surname">Ingvarsson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide analysis reveals diverged patterns of codon bias, gene expression, and rates of sequence evolution in <em>Picea</em> gene families.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1002</span>–<span class="cit-lpage">1015</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv044%26rft_id%253Dinfo%253Apmid%252F25747252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25747252&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.39" data-doi="10.1126/sciadv.1501084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Baniaga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. B.</span> <span class="cit-name-surname">Sessa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Scascitelli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. W.</span> <span class="cit-name-surname">Graham</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Early genome duplications in conifers and other seed plants.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">e1501084</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/sciadv.1501084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D1%26rft.issue%253D10%26rft.spage%253De1501084%26rft.epage%253De1501084%26rft.atitle%253DEarly%2Bgenome%2Bduplications%2Bin%2Bconifers%2Band%2Bother%2Bseed%2Bplants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1501084%26rft_id%253Dinfo%253Apmid%252F26702445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiMS8xMC9lMTUwMTA4NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzQwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.40" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Godzik</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cd-hit: A fast program for clustering and comparing large sets of protein or nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1658</span>–<span class="cit-lpage">1659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btl158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16731699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238905700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.41" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence alignment/map (SAM) format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.42" data-doi="10.1093/bioinformatics/17.3.282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Jaroszewski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Godzik</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Clustering of highly homologous sequences to reduce the size of large protein databases.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.3.282</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D282%26rft.epage%253D283%26rft.atitle%253DClustering%2Bof%2Bhighly%2Bhomologous%2Bsequences%2Bto%2Breduce%2Bthe%2Bsize%2Bof%2B%2Blarge%2Bprotein%2Bdatabases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.3.282%26rft_id%253Dinfo%253Apmid%252F11294794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.3.282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11294794&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168053800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.43" data-doi="10.1093/gbe/evx070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">De La Torre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sterck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. M.</span> <span class="cit-name-surname">Cánovas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Avila</span></span> <span class="cit-etal">et al.</span></li></ol><cite>,<span class="cit-pub-date"> 2017</span> <span class="cit-article-title">Single-copy genes as molecular markers for phylogenomic studies in seed plants.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1130</span>–<span class="cit-lpage">1147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evx070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D1130%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3409.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Scots pine (Pinus sylvestris L.) heartwood formation and wounding stress: A view from the transcriptome</span>, <span class="cit-publ-name">University of Helsinki, Helsinki, Finland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.45" data-doi="10.1104/pp.16.01082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Paasela</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Harju</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Venäläinen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Paulin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Developmental changes in scots pine transcriptome during heartwood formation.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.16.01082</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.01082%26rft_id%253Dinfo%253Apmid%252F27600814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTcyLzMvMTQwMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzQwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.46" data-doi="10.1155/2014/961823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Adelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Myles</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A transcriptome post-scaffolding method for assembling high quality contigs.</span> <abbr class="cit-jnl-abbrev">Comput. Biol. J.</abbr> <span class="cit-vol">2014</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2014/961823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BBiol.%2BJ.%26rft.volume%253D2014%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2014%252F961823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2014/961823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.47" data-doi="10.1007/s11295-016-0995-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López de Heredia</span>,  <span class="cit-name-given-names">U.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Vázquez-Poletti</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">RNA-seq analysis in forest tree species: bioinformatic problems and solutions.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-016-0995-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D12%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-016-0995-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-016-0995-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.48" data-doi="10.1093/nar/gky834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchet</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lecompte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cruaud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Aury</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title"><em>De novo</em> clustering of long reads by gene from transcriptomics data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>: <span class="cit-elocation-id">e2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky834</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253De2%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.49" data-doi="10.1371/journal.pcbi.1004229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>,  <span class="cit-name-given-names">G. R. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Heckerman</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ConPADE : Genome assembly ploidy estimation from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1004229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004229</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D11%26rft.spage%253De1004229%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.50" data-doi="10.1111/1755-0998.12613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>,  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Waples</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. W.</span> <span class="cit-name-surname">Seeb</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Seeb</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Paralogs are revealed by proportion of heterozygotes and deviations in read ratios in genotyping-by-sequencing data from natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">669</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12613</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D656%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Medlar</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Laakso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Miraldo</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Löytynoja</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Glutton: large-scale integration of non-model organism transcriptome data for comparative analysis.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">077511</span>. <span class="cit-comment">doi: 10.1101/077511</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.52" data-doi="10.1186/s12870-016-0939-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merino</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abrahamsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sterck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Craven-Bartle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Canovas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transcript profiling for early stages during embryo development in Scots pine.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-016-0939-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D16%26rft.spage%253D255%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-016-0939-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-016-0939-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.53" data-doi="10.1007/s11295-014-0816-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Freund</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wildhagen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Schmid</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Targeted re-sequencing of five Douglas-fir provenances reveals population structure and putative target genes of positive selection.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">816</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-014-0816-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D11%26rft.spage%253D816%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-014-0816-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-014-0816-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.54" data-doi="10.1534/g3.117.300078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. E.</span> <span class="cit-name-surname">McGuire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. C.</span> <span class="cit-name-surname">Wheeler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Crepeau</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Douglas-fir genome sequence reveals specialization of the photosynthetic apparatus in Pinaceae.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3157</span>–<span class="cit-lpage">3167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300078%26rft_id%253Dinfo%253Apmid%252F28751502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOS8zMTU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zNDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.55" data-doi="10.1186/gb-2014-15-3-r59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Wegrzyn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Zimin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Puiu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2014-15-3-r59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR59%26rft.atitle%253DDECODING%2BTHE%2BMASSIVE%2BGENOME%2BOF%2BLOBLOLLY%2BPINE%2BUSING%2BHAPLOID%2BDNA%2BAND%2BNOVEL%2BASSEMBLY%2BSTRATEGIES.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r59%26rft_id%253Dinfo%253Apmid%252F24647006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24647006&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.56" data-doi="10.1111/tpj.12193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>,  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. B.</span> <span class="cit-name-surname">Barbazuk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kirst</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-exome targeted sequencing of the uncharacterized pine genome.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D75%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12193%26rft_id%253Dinfo%253Apmid%252F23551702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23551702&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320708300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.57" data-doi="10.1038/nature12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Street</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wetterbom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zuccolo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. C.</span> <span class="cit-name-surname">Lin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Norway spruce genome sequence and conifer genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12211%26rft_id%253Dinfo%253Apmid%252F23698360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698360&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319556100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.58" data-doi="10.1186/s12864-015-2247-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ishii</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kozaki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ogiwara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kanekatsu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Removal of redundant contigs from <em>de novo</em> RNA-Seq assemblies via homology search improves accurate detection of differentially expressed genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2247-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1031%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2247-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2247-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.59" data-doi="10.1186/1471-2164-11-180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parchman</span>,  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Geist</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Grahnen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Benkman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Buerkle</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcriptome sequencing in an ecologically important tree species: Assembly, annotation, and marker discovery.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DParchman%26rft.auinit1%253DT.%2BL.%26rft.volume%253D11%26rft.spage%253D180%26rft.epage%253D180%26rft.atitle%253DTranscriptome%2Bsequencing%2Bin%2Ban%2Becologically%2Bimportant%2Btree%2Bspecies%253A%2Bassembly%252C%2Bannotation%252C%2Band%2Bmarker%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-180%26rft_id%253Dinfo%253Apmid%252F20233449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20233449&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.60" data-doi="10.1093/bioinformatics/btg034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Antonescu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Sultana</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">TIGR gene indices clustering tools (TGICL): A software system for fast clustering of large EST datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DPertea%26rft.auinit1%253DG.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D651%26rft.epage%253D652%26rft.atitle%253DTIGR%2BGene%2BIndices%2Bclustering%2Btools%2B%2528TGICL%2529%253A%2Ba%2Bsoftware%2Bsystem%2Bfor%2Bfast%2Bclustering%2Bof%2Blarge%2BEST%2Bdatasets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg034%26rft_id%253Dinfo%253Apmid%252F12651724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12651724&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181964600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.61" data-doi="10.1111/1755-0998.12232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinosio</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">González-Martínez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Bagnoli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Cattonaro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Grivet</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">First insights into the transcriptome and development of new genomic tools of a widespread circum-Mediterranean tree species, <em>Pinus halepensis</em></span>. <abbr class="cit-jnl-abbrev">Mill. Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">846</span>–<span class="cit-lpage">856</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMill.%2BMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D846%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12232%26rft_id%253Dinfo%253Apmid%252F24450970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24450970&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.62" data-doi="10.1111/nph.15075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porth</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jaquish</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ritland</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Partial correlation analysis of transcriptomes helps detangle the growth and defense network in spruce.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">218</span>: <span class="cit-fpage">1349</span>–<span class="cit-lpage">1359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.15075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D218%26rft.spage%253D1349%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.15075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.15075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.63" data-doi="10.1534/genetics.107.077099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">García-Gil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Knürr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mikkonen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wachowiak</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Demographic history has influenced nucleotide diversity in European <em>Pinus sylvestris</em> populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1713</span>–<span class="cit-lpage">1724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.077099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D1713%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.077099%26rft_id%253Dinfo%253Apmid%252F18039881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.077099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18039881&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251368800035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.64" data-doi="10.1007/s11295-010-0340-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Kujala</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Savolainen</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revisiting protein heterozygosity in plants-nucleotide diversity in allozyme coding genes of conifer <em>Pinus sylvestris</em>.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-010-0340-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D385%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0340-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0340-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.65" data-doi="10.1186/1471-2164-13-434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raherison</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rigault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Caron</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Poulin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Boyle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome profiling in conifers and the PiceaGenExpress database show patterns of diversification within gene families and interspecific conservation in vascular gene expression.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRaherison%26rft.auinit1%253DE.%2BS.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D434%26rft.epage%253D434%26rft.atitle%253DTranscriptome%2Bprofiling%2Bin%2Bconifers%2Band%2Bthe%2BPiceaGenExpress%2Bdatabase%2Bshow%2Bpatterns%2Bof%2Bdiversification%2Bwithin%2Bgene%2Bfamilies%2Band%2Binterspecific%2Bconservation%2Bin%2Bvascular%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-434%26rft_id%253Dinfo%253Apmid%252F22931377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22931377&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.66" data-doi="10.1038/nmeth.2251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pachter</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Streaming fragment assignment for real-time analysis of sequencing experiments.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2251%26rft_id%253Dinfo%253Apmid%252F23160280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160280&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312810100040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.67" data-doi="10.1038/nature13685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romiguier</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gayral</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ballenghien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bernard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Cahais</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative population genomics in animals uncovers the determinants of genetic diversity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">515</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">263</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13685</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D515%26rft.spage%253D261%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13685%26rft_id%253Dinfo%253Apmid%252F25141177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25141177&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.68" data-doi="10.1111/pbi.12051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruttink</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sterck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rohde</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bendixen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rouzé</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Orthology guided assembly in highly heterozygous crops: creating a reference transcriptome to uncover genetic diversity in Lolium perenne.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D605%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12051%26rft_id%253Dinfo%253Apmid%252F23433242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23433242&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.69" data-doi="10.1002/evl3.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. E. G.</span> <span class="cit-name-surname">Beaudry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C. H.</span> <span class="cit-name-surname">Barrett</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Wright</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The effects of haploid selection on Y chromosome evolution in two closely related dioecious plants.</span> <abbr class="cit-jnl-abbrev">Evol. Lett.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">368</span>–<span class="cit-lpage">377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/evl3.60</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BLett.%26rft.volume%253D2%26rft.spage%253D368%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevl3.60%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evl3.60&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.70" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.71" data-doi="10.1101/gr.196469.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith-Unna</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Boursnell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Patro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hibberd</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kelly</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">TransRate: reference-free quality assessment of de novo transcriptome assemblies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1134</span>–<span class="cit-lpage">1144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.196469.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.196469.115%26rft_id%253Dinfo%253Apmid%252F27252236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI2LzgvMTEzNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzQwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.72" data-doi="10.1186/s12862-015-0423-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Moore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Brown</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yang</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analysis of phylogenomic datasets reveals conflict, concordance, and gene duplications with examples from animals and plants.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12862-015-0423-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D15%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-015-0423-0%26rft_id%253Dinfo%253Apmid%252F26239519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-015-0423-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26239519&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.73" data-doi="10.1534/genetics.116.193227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Wegrzyn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zimin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Puiu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Crepeau</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sequence of the sugar pine megagenome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">1613</span>–<span class="cit-lpage">1626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.193227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.193227%26rft_id%253Dinfo%253Apmid%252F27794028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC80LzE2MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzM0MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.74" data-doi="10.3835/plantgenome2015.04.0025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stočes</span>,  <span class="cit-name-given-names">Š.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ruttink</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bartoš</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Studer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yates</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Orthology guided transcriptome assembly of Italian ryegrass and meadow fescue for single-nucleotide polymorphism discovery.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.04.0025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.04.0025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.75" data-doi="10.1111/1755-0998.12570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suren</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Hodgins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yeaman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Nurkowski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Smets</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Exome capture from the spruce and pine giga-genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1136</span>–<span class="cit-lpage">1146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12570</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D1136%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.76" data-doi="10.1101/gr.103846.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Surget-Groba</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. I.</span> <span class="cit-name-surname">Montoya-Burgos</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimization of de novo transcriptome assembly from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1432</span>–<span class="cit-lpage">1440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.103846.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.103846.109%26rft_id%253Dinfo%253Apmid%252F20693479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMC8xNDMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zNDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.77" data-doi="10.3389/fpls.2016.00484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Syring</span>,  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Tennessen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. N.</span> <span class="cit-name-surname">Jennings</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wegrzyn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Scelfo-Dalbey</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Targeted capture sequencing in whitebark pine reveals range-wide demographic and adaptive patterns despite challenges of a large, repetitive genome.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.00484</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D484%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.00484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.00484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.78" data-doi="PMC1203831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>,  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D123%26rft.spage%253D585%26rft_id%253Dinfo%253Adoi%252FPMC1203831%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=PMC1203831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2513255&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AX26700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.79" data-doi="10.1093/nar/gku989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The UniProt Consortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">UniProt: a hub for protein information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D204</span>–<span class="cit-lpage">D212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku989</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku989%26rft_id%253Dinfo%253Apmid%252F25348405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25348405&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.80" data-doi="10.3389/fpls.2015.00970"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uddenberg</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Akhter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ramachandran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Sundström</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Carlsbecker</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequenced genomes and rapidly emerging technologies pave the way for conifer evolutionary developmental biology.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">00970</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.00970</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D00970%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.00970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.00970&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.81" data-doi="10.1093/pcp/pcy058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ueno</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Nakamura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kobayashi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Terashima</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ishizuka</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">TodoFirGene: Developing transcriptome resources for genetic analysis of <em>Abies sachalinensis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">1276</span>–<span class="cit-lpage">1284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/pcp/pcy058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft.volume%253D59%26rft.spage%253D1276%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcy058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcy058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.82" data-doi="10.1111/nph.13888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verta</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Landry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Dissection of expression‐quantitative trait locus and allele specificity using a haploid/diploid plant system – insights into compensatory evolution of transcriptional regulation within populations.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13888</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D211%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.83" data-doi="10.1186/s12864-018-5015-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Wegrzyn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Myburg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Naidoo</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Defence transcriptome assembly and pathogenesis related gene family analysis in <em>Pinus tecunumanii</em> (low elevation).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-5015-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D632%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-5015-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-5015-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.84" data-doi="10.1186/s12864-015-1401-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wachowiak</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Trivedi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Perry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cavers</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative transcriptomics of a complex of four European pine species.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1401-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D234%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1401-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1401-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.85" data-doi="10.1038/s41477-017-0097-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. M.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Leitch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. J.</span> <span class="cit-name-surname">Leitch</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A genome for gnetophytes and early evolution of seed plants.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41477-017-0097-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D4%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41477-017-0097-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41477-017-0097-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.86" data-doi="10.1038/srep25536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Fan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Xing</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population transcriptomics uncovers the regulation of gene expression variation in adaptation to changing environment.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">25536</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep25536</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D25536%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep25536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep25536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.87" data-doi="10.1038/s41598-019-42795-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yahav</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Privman</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">A comparative analysis of methods for <em>de novo</em> assembly of hymenopteran genomes using either haploid or diploid samples.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">6480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-019-42795-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D9%26rft.spage%253D6480%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-019-42795-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-019-42795-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.88" data-doi="10.1111/mec.14338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Population transcriptomic characterization of the genetic and expression variation of a candidate progenitor of Miscanthus energy crops.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">5911</span>–<span class="cit-lpage">5922</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14338</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D5911%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14338&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.89" data-doi="10.1186/1471-2164-14-328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Optimizing de novo assembly of short-read RNA-seq data for phylogenomics.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-328</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-328%26rft_id%253Dinfo%253Apmid%252F23672450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23672450&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.90" data-doi="10.1093/molbev/msu245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Orthology inference in nonmodel organisms using transcriptomes and low-coverage genomes: Improving accuracy and matrix occupancy for phylogenomics.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3081</span>–<span class="cit-lpage">3092</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu245%26rft_id%253Dinfo%253Apmid%252F25158799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25158799&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3409.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.91" data-doi="10.3389/fgene.2019.00253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Kohnen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V. S. K.</span> <span class="cit-name-surname">Prasad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Analysis of transcriptome and epitranscriptome in plants using PacBio Iso-Seq and nanopore-based direct RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2019.00253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D10%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2019.00253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2019.00253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3409.92" data-doi="10.1534/genetics.113.159715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Crepeau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Holtz-Morris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Koriabine</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing and assembly of the 22-Gb loblolly pine genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.159715</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZimin%26rft.auinit1%253DA.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D875%26rft.epage%253D890%26rft.atitle%253DSequencing%2Band%2BAssembly%2Bof%2Bthe%2B22-Gb%2BLoblolly%2BPine%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159715%26rft_id%253Dinfo%253Apmid%252F24653210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zNDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3409.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3395" title="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3423" title="Environmental Association Identifies Candidates for Tolerance to Low Temperature and Drought" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3409.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TYvIRGXDYF-YfPU4o82DUGxg24_uu1acfrfTi_Op6SM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3409" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3409" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3409" class="active" data-icon-position="" data-hide-link-title="0">Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494261" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-L2C8CicFk_pb4ok24itkGsyoWmcsr7t9siTj-wR-syg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3409.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3409" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VjBCo76UO46XAmyCw-8G99FdEZtjlDJVMlBVy7ICdSU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3409" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3409.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494261" id="node494261--2" data-pisa="ggg;9/10/3409" data-pisa-master="ggg;g3.119.400357" data-apath="/ggg/9/10/3409.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Utilization of Tissue Ploidy Level Variation in <em>de Novo</em> Transcriptome Assembly of <em>Pinus sylvestris</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8181-4804" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dario I. Ojeda</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1298-7370" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tiina M. Mattila</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1012-9399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tom Ruttink</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0949-6156" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonja T. Kujala</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3799-3013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katri Kärkkäinen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1701-6124" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jukka-Pekka Verta</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6958-5172" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanja Pyhäjärvi</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3409-3421;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400357 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494261--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dario I. Ojeda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Norwegian Institute of Bioeconomy Research, 1433, Ås, Norway,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BI.%2BOjeda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ojeda%20DI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADario%2BI.%2BOjeda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8181-4804" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dario I. Ojeda</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dario.alayon@nibio.no" class="" data-icon-position="" data-hide-link-title="0">dario.alayon@nibio.no</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tiina M. Mattila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiina%2BM.%2BMattila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mattila%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATiina%2BM.%2BMattila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1298-7370" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tiina M. Mattila</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tom Ruttink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Sciences Unit, Flanders Research Institute for Agriculture, Fisheries and Food (ILVO), 9090 Melle, Belgium,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tom%2BRuttink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruttink%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATom%2BRuttink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1012-9399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tom Ruttink</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sonja T. Kujala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonja%2BT.%2BKujala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kujala%20ST&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonja%2BT.%2BKujala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0949-6156" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonja T. Kujala</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katri Kärkkäinen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katri%2BK%C3%A4rkk%C3%A4inen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4rkk%C3%A4inen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatri%2BK%25C3%25A4rkk%25C3%25A4inen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3799-3013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katri Kärkkäinen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jukka-Pekka Verta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Organismal and Evolutionary Biology. University of Helsinki, P.O. Box 3, 00014, Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jukka-Pekka%2BVerta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verta%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJukka-Pekka%2BVerta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1701-6124" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jukka-Pekka Verta</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tanja Pyhäjärvi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Biocenter Oulu, University of Oulu, P.O.Box 8000, FI-90014, Oulu, Finland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BPyh%C3%A4j%C3%A4rvi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pyh%C3%A4j%C3%A4rvi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanja%2BPyh%25C3%25A4j%25C3%25A4rvi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6958-5172" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanja Pyhäjärvi</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3409" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3409&amp;title=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3409&amp;title=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3409&amp;title=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3409&amp;text=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3409&amp;title=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3409&amp;t=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3409&amp;title=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3409&amp;title=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494261/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494261" id="node494261--3" data-pisa="ggg;9/10/3409" data-pisa-master="ggg;g3.119.400357" data-apath="/ggg/9/10/3409.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Utilization of Tissue Ploidy Level Variation in <em>de Novo</em> Transcriptome Assembly of <em>Pinus sylvestris</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8181-4804" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dario I. Ojeda</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1298-7370" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tiina M. Mattila</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1012-9399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tom Ruttink</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0949-6156" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonja T. Kujala</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3799-3013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katri Kärkkäinen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1701-6124" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jukka-Pekka Verta</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6958-5172" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanja Pyhäjärvi</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3409-3421;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400357 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494261--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dario I. Ojeda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Norwegian Institute of Bioeconomy Research, 1433, Ås, Norway,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BI.%2BOjeda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ojeda%20DI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADario%2BI.%2BOjeda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8181-4804" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dario I. Ojeda</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dario.alayon@nibio.no" class="" data-icon-position="" data-hide-link-title="0">dario.alayon@nibio.no</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tiina M. Mattila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiina%2BM.%2BMattila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mattila%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATiina%2BM.%2BMattila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1298-7370" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tiina M. Mattila</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tom Ruttink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Sciences Unit, Flanders Research Institute for Agriculture, Fisheries and Food (ILVO), 9090 Melle, Belgium,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tom%2BRuttink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruttink%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATom%2BRuttink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1012-9399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tom Ruttink</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sonja T. Kujala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonja%2BT.%2BKujala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kujala%20ST&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonja%2BT.%2BKujala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0949-6156" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonja T. Kujala</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katri Kärkkäinen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Natural Resources Institute Finland (Luke), 90570, Oulu, Finland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katri%2BK%C3%A4rkk%C3%A4inen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4rkk%C3%A4inen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatri%2BK%25C3%25A4rkk%25C3%25A4inen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3799-3013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katri Kärkkäinen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jukka-Pekka Verta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Organismal and Evolutionary Biology. University of Helsinki, P.O. Box 3, 00014, Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jukka-Pekka%2BVerta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verta%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJukka-Pekka%2BVerta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1701-6124" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jukka-Pekka Verta</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tanja Pyhäjärvi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics,</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Biocenter Oulu, University of Oulu, P.O.Box 8000, FI-90014, Oulu, Finland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BPyh%C3%A4j%C3%A4rvi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pyh%C3%A4j%C3%A4rvi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanja%2BPyh%25C3%25A4j%25C3%25A4rvi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6958-5172" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanja Pyhäjärvi</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3409&amp;rft.epage=3421&amp;rft.atitle=Utilization%20of%20Tissue%20Ploidy%20Level%20Variation%20in%20de%20Novo%20Transcriptome%20Assembly%20of%20Pinus%20sylvestris&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ojeda%2C+Dario+I.&amp;rft.au=Mattila%2C+Tiina+M.&amp;rft.au=Ruttink%2C+Tom&amp;rft.au=Kujala%2C+Sonja+T.&amp;rft.au=K%C3%A4rkk%C3%A4inen%2C+Katri&amp;rft.au=Verta%2C+Jukka-Pekka&amp;rft.au=Pyh%C3%A4j%C3%A4rvi%2C+Tanja"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494261/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494261/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494261/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494261/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494261/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494261/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494261/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494261/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494261/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494261/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494261/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494261/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3409" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3409.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3409.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3409.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
