<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3297" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3297.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3297.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3297.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3297/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3297.atom" />
<meta name="HW.pisa" content="ggg;9/10/3297" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models" />
<meta name="DC.Identifier" content="10.1534/g3.119.400378" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Mollinari, Garcia. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Modern SNP genotyping technologies allow measurement of the relative abundance of different alleles for a given locus and consequently estimation of their allele dosage, opening a new road for genetic studies in autopolyploids. Despite advances in genetic linkage analysis in autotetraploids, there is a lack of statistical models to perform linkage analysis in organisms with higher ploidy levels. In this paper, we present a statistical method to estimate recombination fractions and infer linkage phases in full-sib populations of autopolyploid species with even ploidy levels for a set of SNP markers using hidden Markov models. Our method uses efficient two-point procedures to reduce the search space for the best linkage phase configuration and reestimate the final parameters by maximizing the likelihood of the Markov chain. To evaluate the method, and demonstrate its properties, we rely on simulations of autotetraploid, autohexaploid and autooctaploid populations and on a real tetraploid potato data set. The results show the reliability of our approach, including situations with complex linkage phase scenarios in hexaploid and octaploid populations." />
<meta name="DC.Contributor" content="Marcelo Mollinari" />
<meta name="DC.Contributor" content="Antonio Augusto Franco Garcia" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Modern SNP genotyping technologies allow measurement of the relative abundance of different alleles for a given locus and consequently estimation of their allele dosage, opening a new road for genetic studies in autopolyploids. Despite advances in genetic linkage analysis in autotetraploids, there is a lack of statistical models to perform linkage analysis in organisms with higher ploidy levels. In this paper, we present a statistical method to estimate recombination fractions and infer linkage phases in full-sib populations of autopolyploid species with even ploidy levels for a set of SNP markers using hidden Markov models. Our method uses efficient two-point procedures to reduce the search space for the best linkage phase configuration and reestimate the final parameters by maximizing the likelihood of the Markov chain. To evaluate the method, and demonstrate its properties, we rely on simulations of autotetraploid, autohexaploid and autooctaploid populations and on a real tetraploid potato data set. The results show the reliability of our approach, including situations with complex linkage phase scenarios in hexaploid and octaploid populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3297" />
<meta name="citation_id" content="9/10/3297" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3297" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3297.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3297.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3297.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400378" />
<meta name="citation_pmid" content="31405891" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3297" />
<meta name="citation_lastpage" content="3314" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marcelo Mollinari" />
<meta name="citation_author_institution" content="Department of Horticultural Science, Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7001-8498" />
<meta name="citation_author" content="Antonio Augusto Franco Garcia" />
<meta name="citation_author_institution" content="Department of Genetics, University of São Paulo/ESALQ, Piracicaba, São Paulo, Brazil" />
<meta name="citation_author_email" content="augusto.garcia@usp.br" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0634-3277" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=K. Aitken;citation_author=P. Jackson;citation_author=C. McIntyre;citation_title=Construction of a genetic linkage map for Saccharum officinarum incorporating both simplex and duplex markers to increase genome coverage.;citation_pages=742-756;citation_volume=50;citation_year=2007;citation_issue=8;citation_pmid=17893734;citation_doi=10.1139/G07-056" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=C. M. Arizio;citation_author=S. M. Costa Tártara;citation_author=M. M. Manifesto;citation_title=Carotenoids gene markers for sweetpotato (Ipomoea batatas l. lam): applications in genetic mapping, diversity evaluation and cross-species transference.;citation_pages=237-251;citation_volume=289;citation_year=2014;citation_pmid=24384928;citation_doi=10.1007/s00438-013-0803-3" />
<meta name="citation_reference" content="citation_title=Statistical Modeling in Biostatistics and Bioinformatics;citation_title=Finite mixture model clustering of SNP data;citation_pages=139-157;citation_year=2014" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_title=Polyploidy and Genome Evolution;citation_author=J. A. Birchler;citation_title=Genetic Consequences of Polyploidy in Plants;citation_year=2012;citation_doi=10.1007/978-3-642-31442-1_2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. M. Bourke;citation_author=G. van Geest;citation_author=R. E. Voorrips;citation_author=J. Jansen;citation_author=T. Kranenburg;citation_title=polymapR–linkage analysis and genetic map construction from F1 populations of outcrossing polyploids.;citation_pages=3496-3502;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty371" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=T. Kranenburg;citation_author=J. Jansen;citation_author=R. G. F. Visser;citation_title=Integrating haplotype-specific linkage maps in tetraploid species using SNP markers.;citation_pages=2211-2226;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2768-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=R. G. Visser;citation_author=C. Maliepaard;citation_title=The double-reduction landscape in tetraploid potato as revealed by a high-density linkage map.;citation_volume=201;citation_year=2015;citation_pmid=26377683;citation_doi=10.1534/genetics.115.181008" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. Broman;citation_author=S. Sen;citation_year=2009;citation_doi=10.1007/978-0-387-92125-9" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. H. Buetow;citation_author=A. Chakravarti;citation_title=Multipoint gene mapping using seriation. I. General methods.;citation_pages=180-188;citation_volume=41;citation_year=1987;citation_issue=2;citation_pmid=3475978" />
<meta name="citation_reference" content="citation_title=Discussions in cytogenetics;citation_year=1962" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=J. Quant. Trait Loci;citation_author=R. W. Doerge;citation_title=Constructing genetic maps by rapid chain delineation.;citation_pages=1-14;citation_volume=2;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Doerge;citation_author=B. A. Craig;citation_title=Model selection for quantitative trait locus analysis in polyploids;citation_pages=7951-7956;citation_volume=97;citation_year=2000;citation_issue=14;citation_pmid=10884425;citation_doi=10.1073/pnas.97.14.7951" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=R. A. Fisher;citation_title=The theory of linkage in polysomic inheritance.;citation_volume=233;citation_year=1947;citation_doi=10.1098/rstb.1947.0006" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=R. A. Fisher;citation_title=Allowance for double reduction in the calculation of genotype frequencies with polysomic inheritance.;citation_pages=169-171;citation_volume=12;citation_year=1943;citation_doi=10.1111/j.1469-1809.1943.tb02320.x" />
<meta name="citation_reference" content="citation_title=Quantitative genetics and breeding methods in autopolyploids plants;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. A. F. Garcia;citation_author=M. Mollinari;citation_author=T. G. Marconi;citation_author=O. R. Serang;citation_author=R. R. Silva;citation_title=SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids.;citation_pages=3399;citation_volume=3;citation_year=2013;citation_pmid=24292365;citation_doi=10.1038/srep03399" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=F. Grandke;citation_author=S. Ranganathan;citation_author=N. van Bers;citation_author=J. R. de Haan;citation_author=D. Metzler;citation_title=PERGOLA: Fast and deterministic linkage mapping of polyploids.;citation_pages=12;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12859-016-1416-8" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=C. A. Hackett;citation_title=A comment on Xie and Xu: 'mapping quantitative trait loci in tetraploid species'.;citation_pages=187-189;citation_volume=78;citation_year=2001;citation_issue=2;citation_pmid=11732096;citation_doi=10.1017/S0016672301005262" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Hackett;citation_author=J. E. Bradshaw;citation_author=G. J. Bryan;citation_title=QTL mapping in autotetraploids using snp dosage information.;citation_pages=1885-1904;citation_volume=127;citation_year=2014;citation_pmid=24981609;citation_doi=10.1007/s00122-014-2347-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. A. Hackett;citation_author=J. E. Bradshaw;citation_author=J. W. McNicol;citation_title=Interval Mapping of Quantitative Trait Loci in Autotetraploid Species;citation_pages=1819-1832;citation_volume=159;citation_year=2001;citation_issue=4;citation_pmid=11779817" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. A. Hackett;citation_author=K. McLean;citation_author=G. J. Bryan;citation_title=Linkage analysis and QTL mapping using SNP dosage data in a tetraploid potato mapping population.;citation_pages=e63939;citation_volume=8;citation_year=2013;citation_pmid=23704960;citation_doi=10.1371/journal.pone.0063939" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. Haldane;citation_title=Theoretical Genetics of Autopolyploids.;citation_pages=359-372;citation_volume=22;citation_year=1930;citation_doi=10.1007/BF02984197" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=C. Jiang;citation_author=Z.-B. Zeng;citation_title=Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines.;citation_pages=47-58;citation_volume=101;citation_year=1997;citation_issue=1;citation_pmid=9465409;citation_doi=10.1023/A:1018394410659" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=A. Kriegner;citation_author=J. C. Cervantes;citation_author=K. Burg;citation_author=R. O. Mwanga;citation_author=D. Zhang;citation_title=A genetic linkage map of sweetpotato [Ipomoea batatas(l.) lam.] based on aflp markers.;citation_pages=169-185;citation_volume=11;citation_year=2003;citation_doi=10.1023/A:1022870917230" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. S. Lander;citation_author=P. Green;citation_title=Construction of multilocus genetic linkage maps in humans.;citation_pages=2363-2367;citation_volume=84;citation_year=1987;citation_issue=8;citation_pmid=3470801;citation_doi=10.1073/pnas.84.8.2363" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=P. Green;citation_author=J. Abrahamson;citation_author=A. Barlow;citation_author=M. J. Daly;citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.;citation_pages=174-181;citation_volume=1;citation_year=1987;citation_issue=2;citation_pmid=3692487;citation_doi=10.1016/0888-7543(87)90010-3" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. J. Leach;citation_author=L. Wang;citation_author=M. J. Kearsey;citation_author=Z. Luo;citation_title=Multilocus tetrasomic linkage analysis using hidden markov chain model.;citation_volume=107;citation_year=2010;citation_pmid=20142473;citation_doi=10.1073/pnas.0908477107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. H. Lemmon;citation_author=J. F. Doebley;citation_title=Genetic dissection of a genomic region with pleiotropic effects on domestication traits in maize reveals multiple linked QTL.;citation_volume=198;citation_year=2014;citation_pmid=24950893;citation_doi=10.1534/genetics.114.165845" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. A. Lewin;citation_author=D. M. Larkin;citation_author=J. Pontius;citation_author=S. J. O’Brien;citation_title=Every genome sequence needs a good map.;citation_volume=19;citation_year=2009;citation_pmid=19596977;citation_doi=10.1101/gr.094557.109" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Plant Genomics;citation_author=J. Li;citation_author=K. Das;citation_author=G. Fu;citation_author=C. Tong;citation_author=Y. Li;citation_title=Em algorithm for mapping quantitative trait Loci in multivalent tetraploids.;citation_pages=216547-216547;citation_volume=2010;citation_year=2010;citation_pmid=21317976;citation_doi=10.1155/2010/216547" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M.-C. Luo;citation_author=Y. Q. Gu;citation_author=F. M. You;citation_author=K. R. Deal;citation_author=Y. Ma;citation_title=A 4-gigabase physical map unlocks the structure and evolution of the complex genome of aegilops tauschii, the wheat d-genome progenitor.;citation_volume=110;citation_year=2013;citation_pmid=23610408;citation_doi=10.1073/pnas.1219082110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. W. Luo;citation_author=R. M. Zhang;citation_author=M. J. Kearsey;citation_title=Theoretical basis for genetic linkage analysis in autotetraploid species.;citation_volume=101;citation_year=2004;citation_pmid=15100415;citation_doi=10.1073/pnas.0304482101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. W. Luo;citation_author=Z. Zhang;citation_author=L. Leach;citation_author=R. M. Zhang;citation_author=J. E. Bradshaw;citation_title=Constructing genetic linkage maps under a tetrasomic model.;citation_volume=172;citation_year=2006;citation_pmid=16415363;citation_doi=10.1534/genetics.105.052449" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. N. Massa;citation_author=N. C. Manrique-Carpintero;citation_author=J. J. Coombs;citation_author=D. G. Zarka;citation_author=A. E. Boone;citation_title=Genetic linkage mapping of economically important traits in cultivated tetraploid potato (solanum tuberosum l.). G3: Genes, Genomes;citation_pages=2357-2364;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=K. Mather;citation_title=Segregation and linkage in autotetraploids.;citation_pages=287-314;citation_volume=32;citation_year=1936;citation_doi=10.1007/BF02982683" />
<meta name="citation_reference" content="citation_title=The mesurement of linkage in heredity;citation_year=1957" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. Mollinari;citation_author=G. R. A. Margarido;citation_author=R. Vencovsky;citation_author=A. A. F. Garcia;citation_title=Evaluation of algorithms used to order markers on genetic maps.;citation_pages=494-502;citation_volume=103;citation_year=2009;citation_issue=6;citation_pmid=19639011;citation_doi=10.1038/hdy.2009.96" />
<meta name="citation_reference" content="citation_title=Plant genotyping: methods and protocols;citation_author=M. Mollinari;citation_author=O. Serang;citation_title=Quantitative SNP Genotyping of Polyploids with MassARRAY and Other Platforms;citation_year=2015;citation_doi=10.1007/978-1-4939-1966-6_17" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=H. J. Muller;citation_title=A New Mode of Segregation in Gregory’s Tetraploid Primulas.;citation_pages=508-512;citation_volume=48;citation_year=1914;citation_doi=10.1086/279426" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. C. Osborn;citation_author=J. C. Pires;citation_author=J. A. Birchler;citation_author=D. L. Auger;citation_author=Z. J. Chen;citation_title=Understanding mechanisms of novel gene expression in polyploids.;citation_pages=141-147;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12615008;citation_doi=10.1016/S0168-9525(03)00015-5" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. P. Otto;citation_author=J. Whitton;citation_title=POLYPLOID INCIDENCE AND EVOLUTION;citation_pages=401-437;citation_volume=34;citation_year=2000;citation_issue=1;citation_pmid=11092833;citation_doi=10.1146/annurev.genet.34.1.401" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. F. Preedy;citation_author=C. A. Hackett;citation_title=A rapid marker ordering approach for high-density genetic linkage maps in experimental autotetraploid populations using multidimensional scaling.;citation_pages=2117-2132;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2761-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=L. Rabiner;citation_title=A tutorial on hidden markov models and selected applications in speech recognition.;citation_pages=257-286;citation_volume=77;citation_year=1989;citation_doi=10.1109/5.18626" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=K. Rak;citation_author=P. C. Bethke;citation_author=J. P. Palta;citation_title=Qtl mapping of potato chip color and tuber traits within an autotetraploid family.;citation_pages=15;citation_volume=37;citation_year=2017;citation_doi=10.1007/s11032-017-0619-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Rehmsmeier;citation_title=A computational approach to developing mathematical models of polyploid meiosis.;citation_volume=193;citation_year=2013;citation_pmid=23335332;citation_doi=10.1534/genetics.112.145581" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=M. I. Ripol;citation_author=G. A. Churchill;citation_author=J. A. G. D. Silva;citation_author=M. Sorrells;citation_title=Statistical aspects of genetic mapping in autopolyploids.;citation_pages=31-41;citation_volume=235;citation_year=1999;citation_issue=1-2;citation_pmid=10415330;citation_doi=10.1016/S0378-1119(99)00218-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=S. Schie;citation_author=R. Chaudhary;citation_author=T. Debener;citation_title=Analysis of a Complex Polyploid Plant Genome using Molecular Markers: Strong Evidence for Segmental Allooctoploidy in Garden Dahlias.;citation_volume=7;citation_year=2014;citation_doi=10.3835/plantgenome2014.01.0002" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Serang;citation_author=M. Mollinari;citation_author=A. A. Garcia;citation_title=Efficient exact maximum a posteriori computation for bayesian SNP genotyping in polyploids.;citation_pages=e30906-e30906;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363513;citation_doi=10.1371/journal.pone.0030906" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. K. Sharma;citation_author=D. Bolser;citation_author=J. de Boer;citation_author=M. Sønderkær;citation_author=W. Amoros;citation_title=Construction of Reference Chromosome-Scale Pseudomolecules for Potato: Integrating the Potato Genome with Genetic and Physical Maps. G3: Genes, Genomes;citation_pages=2031-2047;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=K. Shirasawa;citation_author=M. Tanaka;citation_author=Y. Takahata;citation_author=D. Ma;citation_author=Q. Cao;citation_title=A high-density SNP genetic map consisting of a complete set of homologous groups in autohexaploid sweetpotato (Ipomoea batatas).;citation_pages=44207;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep44207" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=D. E. Soltis;citation_author=M. C. Segovia-Salcedo;citation_author=I. Jordon-Thaden;citation_author=L. Majure;citation_author=N. M. Miles;citation_title=Are polyploids really evolutionary dead-ends (again)? A critical reappraisal of Mayrose et al. ();citation_pages=1105-1117;citation_volume=202;citation_year=2014;citation_pmid=24754325;citation_doi=10.1111/nph.12756" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_author=L. H. Rieseberg;citation_title=Molecular Data and the Dynamic Nature of Polyploidy.;citation_pages=243-273;citation_volume=12;citation_year=1993;citation_doi=10.1080/713608048" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=D. E. Soltis;citation_author=C. J. Visger;citation_author=P. S. Soltis;citation_title=The polyploidy revolution then…and now: Stebbins revisited.;citation_volume=101;citation_year=2014;citation_pmid=25049267;citation_doi=10.3732/ajb.1400178" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. E. Sorrells;citation_title=Development and Application of RFLPs in Polyploids;citation_pages=1086-1091;citation_volume=32;citation_year=1992;citation_issue=5;citation_doi=10.2135/cropsci1992.0011183X003200050003x" />
<meta name="citation_reference" content="citation_title=Meiotic configurations;citation_author=J. Sybenga;citation_year=1975;citation_doi=10.1007/978-3-642-80960-6" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. van Geest;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=A. Marasek-Ciolakowska;citation_author=Y. Liao;citation_title=An ultra-dense integrated linkage map for hexaploid chrysanthemum enables multi-allelic QTL analysis.;citation_pages=2527-2541;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2974-5" />
<meta name="citation_reference" content="citation_title=Genetic Mapping in Experimental Populations;citation_author=J. W. Van Ooijen;citation_author=J. Jansen;citation_year=2013;citation_doi=10.1017/CBO9781139003889" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Van Os;citation_author=P. Stam;citation_author=R. G. Visser;citation_author=H. J. Van Eck;citation_title=RECORD: a novel method for ordering loci on a genetic linkage map.;citation_pages=30-40;citation_volume=112;citation_year=2005;citation_issue=1;citation_pmid=16228189;citation_doi=10.1007/s00122-005-0097-x" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=G. Gort;citation_author=B. Vosman;citation_title=Genotype calling in tetraploid species from bi-allelic marker data using mixture models.;citation_pages=172-172;citation_volume=12;citation_year=2011;citation_pmid=21595880;citation_doi=10.1186/1471-2105-12-172" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=C. Maliepaard;citation_title=The simulation of meiosis in diploid and tetraploid organisms using various genetic models.;citation_pages=248-248;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23013469;citation_doi=10.1186/1471-2105-13-248" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. Wang;citation_author=F. A. van Eeuwijk;citation_author=J. Jansen;citation_title=The potential of probabilistic graphical models in linkage map construction.;citation_pages=1-12;citation_volume=130;citation_year=2016;citation_doi=10.1007/s00122-016-2806-z" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Wang;citation_author=B. Roe;citation_author=S. Macmil;citation_author=Q. Yu;citation_author=J. E. Murray;citation_title=Microcollinearity between autopolyploid sugarcane and diploid sorghum genomes.;citation_pages=261-261;citation_volume=11;citation_year=2010;citation_pmid=20416060;citation_doi=10.1186/1471-2164-11-261" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. K. Wu;citation_author=W. Burnquist;citation_author=M. E. Sorrells;citation_author=T. L. Tew;citation_author=P. H. Moore;citation_title=The detection and estimation of linkage in polyploids using single-dose restriction fragments.;citation_pages=294-300;citation_volume=83;citation_year=1992;citation_pmid=24202510;citation_doi=10.1007/BF00224274" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Wu;citation_author=C.-X. Ma;citation_author=G. Casella;citation_title=A Bivalent Polyploid Model for Mapping Quantitative Trait Loci in Outcrossing Tetraploids;citation_pages=581-595;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020446;citation_doi=10.1534/genetics.166.1.581" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. Wu;citation_author=C.-X. Ma;citation_author=I. Painter;citation_author=Z.-B. Zeng;citation_title=Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.;citation_pages=349-363;citation_volume=61;citation_year=2002;citation_issue=3;citation_pmid=12027621;citation_doi=10.1006/tpbi.2002.1577" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=F. Xu;citation_author=Y. Lyu;citation_author=C. Tong;citation_author=W. Wu;citation_author=X. Zhu;citation_title=A statistical model for QTL mapping in polysomic autotetraploids underlying double reduction.;citation_volume=15;citation_year=2013;citation_pmid=24177379;citation_doi=10.1093/bib/bbt073" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zheng;citation_author=R. E. Voorrips;citation_author=J. Jansen;citation_author=C. A. Hackett;citation_author=J. Ho;citation_title=Probabilistic multilocus haplotype reconstruction in outcrossing tetraploids.;citation_volume=203;citation_year=2016;citation_pmid=26920758;citation_doi=10.1534/genetics.115.185579" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Modern SNP genotyping technologies allow measurement of the relative abundance of different alleles for a given locus and consequently estimation of their allele dosage, opening a new road for genetic studies in autopolyploids. Despite advances in genetic linkage analysis in autotetraploids, there is a lack of statistical models to perform linkage analysis in organisms with higher ploidy levels. In this paper, we present a statistical method to estimate recombination fractions and infer linkage phases in full-sib populations of autopolyploid species with even ploidy levels for a set of SNP markers using hidden Markov models. Our method uses efficient two-point procedures to reduce the search space for the best linkage phase configuration and reestimate the final parameters by maximizing the likelihood of the Markov chain. To evaluate the method, and demonstrate its properties, we rely on simulations of autotetraploid, autohexaploid and autooctaploid populations and on a real tetraploid potato data set. The results show the reliability of our approach, including situations with complex linkage phase scenarios in hexaploid and octaploid populations." />
<meta name="og-title" property="og:title" content="Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3297" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Modern SNP genotyping technologies allow measurement of the relative abundance of different alleles for a given locus and consequently estimation of their allele dosage, opening a new road for genetic studies in autopolyploids. Despite advances in genetic linkage analysis in autotetraploids, there is a lack of statistical models to perform linkage analysis in organisms with higher ploidy levels. In this paper, we present a statistical method to estimate recombination fractions and infer linkage phases in full-sib populations of autopolyploid species with even ploidy levels for a set of SNP markers using hidden Markov models. Our method uses efficient two-point procedures to reduce the search space for the best linkage phase configuration and reestimate the final parameters by maximizing the likelihood of the Markov chain. To evaluate the method, and demonstrate its properties, we rely on simulations of autotetraploid, autohexaploid and autooctaploid populations and on a real tetraploid potato data set. The results show the reliability of our approach, including situations with complex linkage phase scenarios in hexaploid and octaploid populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WCqNSaveck61Ox2GynJTg65nF1IUuc0FzNedsoeSG_0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494271","apath":"\/ggg\/9\/10\/3297.atom","pisa":"ggg;9\/10\/3297","ac":{"\/ggg\/9\/10\/3297.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3297.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3297"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3297"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3297":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uDPr7vGw+JJPl5rhcoL5aBin83sByDgGmWhjHouZh7CGkwDu3znQUoigAXvGOQ1BkLuqH4Ca7J91yJL62HlfThOJqp1NaLIjODRYD0dHS76RiQGWdSYtZ7w3s40TOSO6MZJWuZM8u9hJQij9AfcG+CycCQRqsHHbI+YsbZ\\\\\\\/DPDx2t5Abefi1h0MiE6adidnVeYqGPYz329RFRYJzJg6rnMJQXjuRhKIPuV9d3KCd83MUizelF0jZZAgCChA\\\\\\\/Ie10s+SeQOkyRoMVob8e6bSVksjz8pnYRsIs+K7LggY86UKrdhj+jKAAN4F9mgqe1Fnqp8mFNuVFCevywE4U0Oo1iuQh1KTSyg+n\\\\\\\/xp4jUcYyXKxS2XIflwOzvLCdE4s9\\\\\\\/GTz+WRYSOuBSg6phWkSw9rhsqmaYdGbFWmhnvZM9LTUzjcOGGItacyz02kCxgXHc66m0EtgUtxSucA5IkZK9ovuG2OxvnwADPm3nPIyKOBnr+qB3nuWit7vavZr6RmEQNonCY0bq1wez6CF\\\\\\\/daD6IIW3VGxYGR8ukPFuW1IBSHvCO8oGr\\\\\\\/tH+Vq+Vp+Nls6Z\\\\\\\/fGWg7O8zTxxWAg6Pb\\\\\\\/3w9XEzDl12eBcWkp8HBQD+FVQy9WHVELEnoCAaEUbZMFuhpCLtg+VdgfcBzirDG39j+bcyJqC\\\\\\\/PslV9lyB50bqoop0kOwAWp51Hg9piRulQT6uvhyz7rXZvNraVk0w0wKRECvJvV5ENAFNk7zkSti+ZGDG2Rha6qfC3LL2BlX\\\\\\\/VCzvA34xnahiR97mHsKgotVI7lLN6D9fVt41WMxRzO8uU+xDO7qaC\\\\\\\/mcI5gb1ceJPvRjsi9io5rtEyhEJ+ki\\\\\\\/lHa3AHce94s8ByaiLsmJFlP1HvwUCr0Fw1BYS1UtDZJE1jFxsHBxn1a1MrL76qmstoNSN+Sarntnn4\\\\\\\/hPfiJEWxpXeAhTs3okkuUNjT6+GrMRVQyss8k+Gfc\\\\\\\/+GctQf1R53EZFmh9yQPVW+vyRSytRjEg7ExUFCaqaumB3iORe\\\\\\\/M2MMttfsJ85oq3MM\\\\\\\/KSHAxT163AJP55uv\\\\\\\/tGXs\\\\\\\/JJ\\\\\\\/JKwZVJflDRnyXoU4NrjnXg\\\\\\\/BnVEjHqcmEIUZwFTpl1D+jAJ6+7KayNnQrs2iIjZqFA5NhYGC5GDmaTp9IG4BvOSDmtrxdKQNnRb3LLgRBihG5FhFUjrvVpH1juFY1IWmGPVPlqeG7nIx2f9tRAhogv1FI8EtVaPKJRijOW7SfU9REoDS+2rB\\\\\\\/JlyraXDcA+XHxV2PRw8C3Xmr\\\\\\\/\\\\\\\/HdfEBkTBnXdH\\\u0022,\\\u0022iv\\\u0022:\\\u0022nGG\\\\\\\/q\\\\\\\/vHVczadzpa75iIBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223ed039a54b944ad7400b23bac248945c939f784fae1f1e9af15ba50864cfac7f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225UXxpRDR8F2qTMm4pnXnmXJUZO9lHytKw3ZM6+5yB34Tw+1BCerrM06CGsrW1vswRnAmrjco\\\\\\\/+D08FYZoNDdGIz46zNo8Fe6436qjwEMRh01ZF2beA2KU\\\\\\\/w1JmUyR6XOO4O2nKCafy\\\\\\\/zw5TKGOFJX\\\\\\\/fGmYru5x4DzVDOM13rV1tOdOpuMH6bSoRZFwWTvYoXACuXhmGBkJ2zd8ANEPjqs5yAjGLXcGg6sEQ5VhJTXnmWac1JMgr8reJrq3G\\\\\\\/7fcurLEZXrHqBC2fM\\\\\\\/ls4tN+U0GQpHOMgtRyhvt9u+NXFccJnipa322n1NEWiTk4vHIDSaVSzJ3EKX4D7tpTwL24ZFiDsHFzs+sSyWYaXH\\\\\\\/9GlEJ33isT4NBrOM0dfnY1lvqeKFiPu5jtMO+s79gpxIvkM6A52WDu1EP2m7l7ngo5oWPTQHflFs\\\\\\\/RKsGV4cz6v55jS4kiAig9oZK5WlJfBdy2HkpcvrSlEkV2PtAegfwe6XOb3UpAE6sBCdOE7LsE0yPClNlqhxLcK3C5TKoXvlxXY7g2kcbJH1TJ8jv+ZNNc0Bk2kisZVyNS\\\\\\\/2OhHYx8LAkPIu+7NRGjR0rs3xrEBByGuhjThoRpov2zSB\\\\\\\/5KqWHUYl5EEPQX62bGAtHD+VQj95ycdiBUka\\\\\\\/L7RcBHR3+FC\\\\\\\/E3TcGIrTdhD5njNXKS5SAkovPt4VtSc38B2vzWqyWTE0n50KQ\\\\\\\/wCbhSck\\\\\\\/L+ec5ZpSg+K91DlTZxyBLaw8EQdT7pBWXEvjiL1auxDPoiZuJTqqgx68LkfdbIlJRtbBRpsb903HlgXRd7ldS2P+YADxjY8BlUePdAS44iZNerDJvWz4pZB9QtRxVFldyDgfzQomUYX3NysstR+n\\\\\\\/n8pJUfrH0yLZfkGs5RxuC8MY5UzY3+sjxvcw0msSDPjADpbEtYuN9mUcslTe1PPKqlvp2sgxg4CuNzykMYTm5JcxYVvAqmD3aWR42U9e1ftQAP4l9sBvum0SWoTRWfyggXES3pxfocjQYdD0SKfZaCRaYJz1u19XUiEDqRvjFioDplh0p+ZNuf5BBxJNxIQmJ+FX1iaOeqXTCmeuovMw97uexfkssIzvYwlJqB2vN++X+Xb5fqTlMzolh3TyFPh1LqveVqFFhOAr3Go4dbiz6isRN7v37VzhapEFdegpVRdjiNTDnFD6qTI+j0yBBixsYxDbHolepGGo70Xozp0VOPLKZV9Hoh1zebk39LlZD\\\\\\\/L3YhgFPYla1W3oWqwujdjnH2+5Zz+0Rf1Nty6S2qLQC80GnzADHAMl84bDIjZtc63KOe0KglcYW3+PY6iHLlNoCGuc+Fb6\\\\\\\/CSobjQwnxPrVPJh\\\\\\\/s1dAhqFNNB+LSIpEg4E2cCrD0\\\\\\\/v8gG+esR4QCQrSOklrWP+P+\\\\\\\/t+0\\\\\\\/ANHPXgUl3rG6JPVgCx0XDY8I4BrGe9w\\\\\\\/+5mAxBSNLVHrngSjFhtCPVeNn6jZxW7+c8a7tW6m6aeAhTUC8wnvAGL8lF+rh1AtIHALP+ouCvzuqVZ2A8vd0hWeElfkZBI6goEWhqjqWnRhvbCPgsxiU8t6NRROt\\\\\\\/aj+SX5pA98rTiduNU8OMA0HwA3osAg2fqtO5jD9xmS6iIsoH3EhKql8qGTTgWp1J00cYam5KWrE7hZORhIJQhn2Tocxq\\\\\\\/OllcZ2FtlkRqNVQoUmzN7tbZ+7\\\\\\\/BUOtkBRgH512CkQ0giJaDbwy\\\\\\\/6J7lVfpkLfLAleEqfsQLsBN3p+MPZjeRK5wriRqhmVK3dXvUFDXG+GPiRMK1ROV5bcmOv4xecT0unYYM14ndfeUWxhU4tWDEZE4uW1u\\\\\\\/5So\\\\\\\/qLrXQ57N1QLLp5xmEW3MT3C8VWcFp8Q7SJU14BiPK+J7Rq\\\\\\\/OfdyXaFTOl1hP4O2oMefx8UhzkOHcdcUFxe3+GpgjUtvNszlqcmaRux3TG56rAvrMQCS+hhpgkCHgI8xS1pdlHaYXkgRC6T+eeScX7zRqJjxu9fY4LQtvRZ7DLcmzLKvqPEwWsqgINbGbHm\\\\\\\/UUuEA4frVkzAm7ja2eKaTXHoRu45yR9P+yy2dza26651oVJBnFoW6ORWmVB4hRDeEQV4187elvxXt1ecCXMPWNrNsXZr9GYNKN6aBSmwbsjBGw7RBCoSc6uV81MOaZ3YtkpT5AMvma9+pgJ3UWLqF+x2iPy4ofQGNIIzOm2W0cJs3xyosQkyfoIxRiHrY7sPVYIv7WV8xXkQ2eYNcCYlCPgXdJOBreDF0kMJiM5VojF1dasmsfGDoQlqD+6tlE5HJpJ2wB0HzNM7XNTRR9rne3H\\\\\\\/XRc0Z7oxZKQdOO05UYmHjt6XEFhxLAUokIUpkbat5YovsCxGBa0ms8P4deEI1Jeje8PqbMkB5jZLBMJO7hxZGHizVMkj6tQUBvB4MrVFUGx\\\\\\\/PhQgoRgkuncNRMVEWvmT\\\\\\\/D\\\\\\\/RDH21Fqm16Nx1XRExUPGiQlyhBfHCxBGtEmXYkkY7J+FTJFpuzSBfgJ5jQ6YDsVoMKRAs4GkoNmr5lZW1G1RzsPrU46BV9ulVFaV+tcWmJXwTXrsRRpyA9qRMpdVGxl+R6u4wRW0q8zvVzaggaE0AH5ufIBixFesVrf0QAG7bPJtK6kX5HZBuCyyB3R5NUpTZ6LW8ClaZAiahJ8kGT1ARr3kE+ZzhJykJuuz2L5z8OatMtcoVfAfcOC9\\\\\\\/3cM\\\\\\\/y3lQEk3SCvr0w93MyGNaldQvBieTjERy4IcDzM0bPgGOfLZEs6YSSR32KFF0I4th6w1vIek1t6mCa2kn0zpCtPqi+Hm4nb1EJVcSVXs8OvrUCP2xjJdf+O1jYT3JK6OQKeZJaqlFKaEBdCf+ml+YHeO7x2ZdeJVNUrJfetkz+99sFC5hk6fbmN0m6Rl7EorE5WlEpwC\\\\\\\/H4+SeSaHX8Eo4RxaB1mVCjAvj1u3WFols+gweZaqyuJI1v9R3AridWXT4FMaOzQTqUw23f4QiachHY6KCfNn5caD9OSVmegNJvzU5BbbaBi\\\\\\\/MHsJYApoGrwmWu6xDJxzsXrdgTMQAUMXsmciOLHzYqlL+XYsIKfurwGnVyQllD5w6p+R0noZ\\\\\\\/R7Hac7xOzE795WKY6LQ2q5QJmH+kUo8yWNhTksWtiA7szjjI32p0sodMuU06o9OhohDeo7gQKvK5beVSMiE42sulxv1KSbkyzC6+mTkl3pJPnKIHPuZlXpaxFtbL99XMzSm\\\\\\\/BmCdswCf+HcRijfWldV2xFa7Cg4bdoiXcI4mmQMQc65Vg3Dqz7gdE3imop9394oSuQTyNDhRxUr8qGJOaIMmx7d5RsVoybpU6n8CV+newSvX8LSx+1foUrvvB+SnCJsBeM\\\\\\\/UqhX0puhvdACyVuveV49YJVEWjuXQ9A6r71G+ev1WIgggBIL01D1rMMvSoGQ48yvwPSLRQzP8x4fNIymUtTJs7OE85vrCUmtfXalCcgYO2fUlPKW3uyKtUdkBGLfiFEul+JXaFRK5ZpfsPbtJ8iz8\\\\\\\/JA6vhXw5q8XXmA2ATTZjslbHs6Y6G+8FNeMP1iAm\\\\\\\/PDvc7jsQHrZUxSRFqKQl99EgtuYf1D5ZfpwrivI59k+yjutMR737WBA528squwUARjfnKnvCYnzfDNECMJCc6F2jEDy0V2BwGMSRYMmAz9KpYkwC2NcuJ2ripWZ2STyLu2R1d9sSq\\\\\\\/wUWD\\\\\\\/7815BVbNWa4PAcSK5N3U+CmS8NCSk0XZ44jX0m8HD\\\\\\\/7RMl5OAIPxzv0MVhv6KcQfE1Dw5TCfbGQ\\\\\\\/xiq4aluJvBX4SHTqG3cstmQrTMOJbVnicKrsBC+XSR4y3E5VzjYCrYyQ1wb35of5B14QQHwiEP1ksNomGmf5oBdo4RvE9SmcmUcb86ALxhGYMjqk5kHTWbgDURq7W6wLg7TD57GmK0BpFp3t8KEm2kCcYnumS4FceoR2h8QsGM4Ua0DIfQES4j3xUKWXr7rt7ZoPSNz\\\\\\\/DVL4gQO12zphKf1CYfKKe3VJWfBnFwyFQ\\\\\\\/c7T2TpQHg2koKI2b160C\\\\\\\/r9RrT5oTQ3ghNz4wk598Yw4OSatJEbrpdscV4mM+RJeDPPlBQ27eCBdD\\\\\\\/mv3yM49dCVl58pjJxMumIPP1wq64hxjIPg6sZA1iadd5ZJ3H6se6++EDPhmwWjw\\\\\\\/1N5OO6n5NGVQAyu9UDS\\\\\\\/P3VHd0j84j12hkN+wDuhVoZPBfrXboRKHzrxRG1XHbmOl5EDGHMItWqSmb7GXAAd2LiRJE9UCkmx\\\\\\\/VpZQu+WiFFuDxozSIgyesz2IDu3XoRvBqE2t\\\\\\\/5CncfgWLDWNl9f1DXb\\\\\\\/uBwhYtPOfXf3k\\\\\\\/YoiaLkOb1PJAi2jPEVaF4mxmzpwQvMdt782EdV65AHnwSERu1bK3ZBE3X5VnvES0mEPOMMXTLiFbA0+jCtGNQCaWjUlsg6pPC6tBFtDj+8rw2\\\\\\\/riieaETHX3yXXfBDVX3ak7aVGCzn8auO8jk2IshBYbXn8v+1dHs6nHm\\\\\\\/ylif1ram\\\\\\\/a14xd8TjFbQw7a+NkCqv07JPxxlX0SToL8exl9TecXQSfiH5gqhbd5atRtxvdFMYd+FUjPP+7kQqTd5XzgmN76RSLSzWd292\\\\\\\/GP0teRV9BsySR5to\\\\\\\/Zjc37ue7SHP2SAsozdYYn3pob5lyQkmle3jAN8sjJIiYlF8aTf1RCM1vrh8IOYx0Z\\\\\\\/wwRnIPMeB+7CY5WXEwfPqzGwvjkfMTSmVnVO4qTHUS8SeOfsKQ7eK6YB\\\\\\\/seiDiAl9e0rpFqRGrIs+2Iv4Zcns\\\\\\\/s6NH50aCh87iBkA3yAJKellzenMCrT9ASfM7IQNuCaAdS\\\\\\\/Bz3Ignaoz+u\\\\\\\/bzWNYqmZ2isjHE61mvoYgBoKwsBrCv36mskxbaOg9kVXPRvgAFxLhOx5n1HLLr1O0cEcwy5dkx3Q4v8gTq6kaELmZcBdjjcDKHTu2IX2s6Z9c5ta6X6AzvqwwnzeBK0oTHM5u9uUKQFJZthLXimsLTpXcuGRYLodM+TXNzjaU5pVhbEMKcyPR4cA6zS3ioQNU+oNw4gw1Z2KUiU0l4EbU\\\\\\\/62R5QA4mE4jN0OTM0s\\\\\\\/IXC+kjGjAdx6IzeQAqTSVF3yAuerOXT\\\\\\\/StD8cn2ps0w0DyceeiNwL79HY4TxUimiRhO0Leg9Ru6D06+YDXHQ5ty5JxpihuAOqJD8EK3kZ9WJaiN242OI3znqJM1inOHqzE3NlcHyGAPaEoAd2DpCyArdlJNQ9n4FmjbUUuuoLwFpRibNEPWqPnQVgKnH41OBePhkwnk71idpFpAi2ybGKFILqQutT4o+IA6I6pdvQTe5co69Tz2OiFSwZLgeey12zPztkCwP2\\\\\\\/1SmD44mUNxSWsN2jyAn7qFe2XxwJIGtWqoWTXrlkzJDNDdQAqP88kCYEIt12qqpW8xUJoKwHbHribuFaoJSOIOSkEx6UHp5oY+MzDncrX7AMYJBksnNEQA55tCVB3iktfb4wE6a9wcahKpiQDgyBWq8SP75FEgrmWQNe+\\\\\\\/1fQCeHzsmECdUpmLGEUb\\\\\\\/5QmtxSrGZzq3Jcrznz7T2DLd2DRizxn+vgKdfED99SBMHvJbv+FF0xmJ73FOUISNGbstkGi0lh3OmjwfEfArTYGMAUT\\\\\\\/dBqdK1DoI6CgUd3G1895sQB+qfvLrxilqdrYQBTthfgHwxwrC6TXeJ+boShQSqzzHVLmN\\\\\\\/3rlo3YCJoAwIl8lztAvUdINsf8fe8Sk\\\\\\\/jeKBoX2TAoIZ1nUOtHmD1lQ4pMsNiqLs7\\\\\\\/6TfQLpsMfV0EQIfs8UhSjQo434iimOM8qRtTy33ZhjdwiGwtM3JbwKhuq+lCs9GhEFJ+qDwHphz9EVe1tYiPuGtOmLWH4YM8NOTJEbEQDrJfQl8prAXEZWK5i53DMxu9KGHnMy5YfQdX9ANo6SQCFDkXlPfm9MouXlwHQxGQmY6JnEC5zO6DzMHHymANIoJX\\\\\\\/lOGEWHJdcxQAEYit5v4YyqJaCDBqQ22Vz6goft\\\\\\\/geAnX+zpDVTdBDSeu8sI+hmSYmJTwNguAqnPQNd0tjluI9MJfYNvzXANVFbUjfFGG6pmRTCdgKcN5LaiZyKBXvyNWQMe9ZY6slhgRfCFqykSmkHzrlGRDIAKC412abtdWmGgH1VZev409Ijke5+\\\\\\\/dU6\\\\\\\/E8rAkk3Twgsm4W7gXblvPWLZvCQCmnqz706ZZ6XRcf0XakcDxgY7uLORqTApzEAPtx3uRQr6OhcW3yy2tW+N4MqhU9QRLNTzJXmgm0W5xpTNzWWQCn8JoOC0Inj1y\\\\\\\/p8fz1FTZgRGpT7R6rC5\\\\\\\/lcS5AIkuoKRyDohmZlDa2tMLgd+5vetPLF9JZ7X9CrktP6BFoobPokCAYTlC7ZR0NjRCc2lBCKKt3ch6k16w9ogFvpOsiPQQ0WnRw7eJyBhp\\\\\\\/8YikXXZ+UtBAy7guXq8LAP+dAix8cPEB1tAxXlsKzJGIokN+hzxKSg0EDTax0FdOBbEErx20VLNIDv6zploxpbEsr2UYUtcLhw86U76TdU8YXn+7I3H4pCSKxuapLsg6m5XbTWzYwqGBZZPjE5XiLUAUjMoE2pB5dCGgzzT\\\\\\\/HCmW9QfBQ1xfQ5MPUSFYtgRsbhZl29oP3lxUU8MgEWmu2AJSjJIszr0v+rSLvh0KoFzS4Bb3eohDhjrmSy+B2FQUG7\\\\\\\/EsnyBMSDx37wp23jmmhyFojAzUzp4\\\\\\\/WOgO1BEr48\\\\\\\/a9oX9yISsDRNUmSURm+4gECXnYGFP\\\\\\\/Ge6Hv6qXAAKCFNsPIMfe8FP6c6Qj\\\\\\\/jZ+7tZnc7dUGf5a5ywaonpRE7BuoTRfiJVT5ohTF8LH6F4N6JQKyaVZhzeJy4JVP6rurud4lMSFFHf0knaX0H2A766Kj8P7MPjDoO0gb46AelbP5NQEsAHjwYm+1UpCK4\\\\\\\/n1yl+dU2aMJh8bItvjjppHCwUckEs+CkQ0y+GArGxStRm429J\\\\\\\/2EXskrBY2CQkf4mqYBiGflMDkRlJFOWpSb+DGH45pkm9RZRqS+6P4Zrt7bTFJqaD4BAB7VOHIhbdysBiwxSAMPNON4wr3PVebfwwdWbmH7Rtqqd4SYb1hwNddSKRHZfqRKtPu++gSX1sFFOpwcIKJRUiNmWqomne9kzjSxXgrD98pzx5cF0tOEcBsOHirpaQ+EQo6b3qK7rTFcqtXwLTBNAkhzSFfAHEpq+Lb8D60wqiCgvXaSc1kvFeF2jt1yp+Et1mLTp+sXpf4BStSg84SjBB8\\\\\\\/7cQmgvZ25zyHLQRQo8kXcN6gBuxHElgzJwHSkXBT43EOHZHwYbIx4lVZNhzwFg0UHSXr0yZqXPp9+4u6wLPjqt0HghpQb6N7ikPkO+Eli3ycnwhaBBV1ESFNAzdM28Cvjw3GKLoH9fg5x0qMb2GTvzfFsCV7ABvDobCvSGHk8iHndDvomfPwvNEVFdHd2HcZ533\\\\\\\/ECJoGkdC3TCDcsAo2DXlAwTxQKwcV9dWwkZZ5jW8MTfL9jbuO1uoHS3i2qSVr9NXiGmp6OQz\\\\\\\/UlMcAp20j6RViv9Fo\\\\\\\/IufedTUvLFOIjRzhJ1yQIGkzE6aMr1+11s9J+Ma9DreMAWGpIFY99IoDjIQqwCTwLkMHLRrdwwuLC2wxOjYcvvgohJK8tiZyO+nMceglKIzQYfZP2QT5BtEaCPVi2quhLnML6IonGZoKbrIfQXK93OlydDeCi0baCJFpMgE9isbUjEdK0HdDA1okqWUgmnXFMESqapKlPH1fy9UEDpptfYGeoPxCshwUfPMJOvnn6efFOhAY4TOndRjrcrx2KXVwEdaaQk09cEYxXr4QCKSiyf8DTXGOd4zjhbPBEn0t+nh2LV3ldaOiKbFXFc\\\\\\\/5m5lkwtT8gvJiwN\\\\\\\/D7zEgBtC9hTniWvdj+kcPto5kOEwZTHok+Gh\\\\\\\/qe++iA+fhxkPXg4ANh\\\\\\\/PU0QcBIWPQMsCw49RNiLrF2QdkoNxWJcmJTLIy0o8HTXLF2DMQWe7T0nEJCgexw4iuACHyPu6y7FucCy9BXUbb\\\\\\\/ZsQAee1EahA8B5k7RzwOtsQMPgXtE7gS0mlPEWVzeZ\\\\\\\/YQdvAnndQdO+g06JSMhY5Vnp6WVFRGj74ZbaGvhDuoz4tFcyFK8HaCZcyFr1DHgX43bHKx1Py2LwuBuAxKSNu8iEJz25W4iVyi6Ymzo1pcr5jVAussDoFYgN3Rrs8pyXYR6cvie\\\\\\\/S9h8HYxJeAmi0+djpHng9Kax+6m0V7gt5UpsqCv23FaLbNfhKDO\\\\\\\/DQ3BbY3o908F55jIu8573oBUo7u8Gmph6l+Sb0Zq5vNkbd3vNRrjMnzQKtMKAxdYZuhhYEgls9D2b2Tx+jZ+EKtjjUdbQRBNaa7lUtkLQcJVTspYDLpRq0zvFwZwzY7ZsBQUr1LdGN1kUQPqlACXkfqf4609T6mPQkrrg8X\\\\\\\/o16wo4Do93XxSQqqwncooz0bZv9Kpi2k4c8a67Qm4bcjjoz4N9XIlab4MIvWQTdbHg\\\\\\\/mkbPRbbZV3NgoROLSscSPgyYkihI\\\\\\\/XXvZoZbYlPDvOYkpTNuXqHAe\\\\\\\/sI9ExKktLTZlna+boaYc0juZg9sSXAuRwlP9JF\\\\\\\/nxG4lohMHdM6Xhk+hwofHLGGFWAHCSH8H+J51F2wUUlyVfMQ6gAH2\\\\\\\/U\\\\\\\/PXddwex08OpZNcNH6ATpxApqj8sylbOBGCWAXa4aY81jYWV1JyM8ZI3XR9J8By4RztfBPU\\\\\\\/azIx5FtDb5AQk1fxpIYnj6uz0K89ntNiG4EhVH6jL2okLRYnWnwQLEAiO0Ijddxv9ifR13GYCBW7rsbMi0JuJDiJXrvsX29pMJiqnl\\\\\\\/eI5mzg2FiUKvlOiN0In9KWRwwQy7HltJ4+oyWKnTHlhtDrBznjjK6ASPUZm\\\\\\\/aC040NH8kBVZgjZeRWzcX99cyrgHhnD+nh86ybsXdqUOeyuJkCdvR2HNOdWKS\\\\\\\/5LNozWHJxb61f68NSN+fwM\\\\\\\/Nz9uYixV44FFxk53kzIVheXMjXu3UkFL5sctrGcO8xrX5LHXf9OnwUiVUGmUyfiCNkTUxPVLjHJB1sqZlsNjgIoqOQLbKcFGF0I3HPHlg+LzDWKPHdsy0CNmAERiaUCF+tnsvE+MdZJGuq35D739Eu61PXjVqZRuJGTfed3cJ+MEbl4mPiooXYIEc4GjcepSQFSXdEXpf5emFJ68BUxSvNWvP6ctbkD7eXCCBs3m6ujLI02L5Z5R4ANFcGLB9XIYXFlYPfQldh9\\\\\\\/TGVGTPity0CtR86juAu\\\\\\\/4UeyGD8MvD9L2C3XrhUkWcs64uR7GxJy\\\\\\\/O837OCyiP1CrnVxVAosTtOcYAzv3VbDAcLmhZjQOmw9dL58gzQkKfgQ09HWVsbqFjN6drHyDhw+R7hs21OUYS5EPBN0SlF7F7lpnoV+HTwD8Zix7De1Tanw6vYNd1zFGSgOZFrx9AJ32w3oPVoTq0W0oFTlOzUzn6XOTxO9UQ3sEbwB3HWosFmNxzAHme6QLbOYasQXF2R+efIWV2pG8U+QeO6p1fJ0avARrOEwV\\\\\\\/lHnt1GAf3lGLdtLdXsgAuvfX3+hYieN4bVg1i2Fbe7OMibZJT7QPgNWquJfYKn3UTJ1ReoWWp\\\\\\\/wtQne0BGGhcaTU8tGtYmW1QvoNJGa4WE8WhWB3a7Sy031ZWH6qtG0lok09JTg5wOvuEiTN+B013Y1qxh0tlyHE9PgRlMqPIMDnZ90c\\\\\\\/DC+6s8iqfDzJjUPGcnzOQgH00gc5jrfl5BFHQiv\\\\\\\/TqH9neZVpPRisKK39DyWgQ9mcP8kKytaXGtg31uMzLB2SaG+qp0xwBAzId2N+fV4cY0qiwSdVdy55JenSXBU7F0+iuXrUmW7znOwd4OYLjvWE1LMa6DNiM5ycI33PHQjL8MRaWAqBu3Jhy3Eoa2LlhPC0o8Q4DjCLaQfJxfYflBCLRtewL23\\\\\\\/hTzB6zEVI\\\\\\\/bjPXECxzkJEc\\\\\\\/ayG5z5auvIJKy4dCDktmZM4KTPL22AY0LQ2lr7qr2xfytIFSL2MG9AnfHxlRoV3m9ik1SQcF+aRWhvDvJBG0XLC3moCFrRVlVj7LBe4PANJSg1Y+7vEG9ocpx9iTTeHDCqPvKiRfpviMBn1qyY\\\\\\\/Futkuhb6Cl6Kid7RcIxo6iKf6fbMJ3fSaBuJHonqxo11Mt3GbnG\\\\\\\/8D\\\\\\\/TRgtPtAkazef4cAjrtrfXMVeltKRm+MEaulAjAWiNiFXjnah3VMKLm3miMWdEFeQ+xKrjCfwmP+DPEFUeIjVWWQBAUFVB9lgvVuQwjbwP+zJ4SfgA1LHwsS\\\\\\\/fsUci6thlMTc8WFyYuW1CqOHTzPcQgiQws18bWFy8UPgnAq5GZsHj5X011IFwzTy\\\\\\\/ppVeoODNa7u7a+vWckJ6u+mFkdOrpMLfvSEfm3zhR\\\\\\\/s8UPIxffpg88sfNdnlwUb817+qOrue+J05b+U6yWlD6dTY6WJxk\\\\\\\/4XIw\\\\\\\/D\\\\\\\/ececUMSbyJftg5LF38Zh3QeQajeGYTfX8DFg6Nq26ltxB7CA8F7Wazv\\\\\\\/SLyGJ7suYSuK4dnn2AtHPxLJpGbs1w69E6VxlS2ZyI6fbEA+mjsUi46eYec8mIicSBsSquXdGi447EvPSx7qhdZQ\\\\\\\/td51IUXPZ+fF2X\\\\\\\/DUGGfMnrtxH2PnmtMiwl6chw2U0t+UkDS\\\\\\\/iUoB4d3U6VTfl2LOUGDcMfAUyzUDksa3WPbHAf4iuyAuRwTlnIsU9Ud+9CoJ9m2aYz31fjsoUGt91Z+2YqNu3QeCtnX5zJoaSb09j5WjDFcvfyAobwdEZ7fSzXglJ1ttx1i0TBKk1Ve22sVTHi5xHp1wEnUyXNqJzgvsQIibmSSySMzTnkenkLOn0L+Ituc\\\\\\\/zrSpfZR\\\\\\\/7eFWAq3nC1Q4cTBi+bokn8B7KxVQfwLbQepQIm42CaBwG8alDqH5oLqExFSZQHlhqCjXl9VOMpym8Etduilxad9AMionqB7BkQyjtI4cONYbrxaIPqphmhR2ChHgtA6+hoJur3M1lJNFwH+A08SnwnK1EaXUv3PtrX3ijr0ATN2oXURBK3FzxB3kSkqsVyv\\\\\\\/8GPfQxEnVDapoIYeTLNRFvq6GXMUqjhjfrDczRVGrJwI5ZP+4g\\\\\\\/kIQsK4d6YQcpF+iY79Uzp5R9LqXvAv5FxqaYWuuq4stypfCwNSaX3g7Yr69bftmCH78P8CJH\\\\\\\/yEbKbl2TqpDGF7tU6AluRPydr\\\\\\\/gco+ZGBWJI+wqnchOhScPpmc6RVCAeVgeVILmcHmSIXSzraMYF4tLMefY2LMzgLkkt9TsWDf6QAYphtr+xv+vWKtgnyoNJ2jTaY2+E8IWHHW7z37p8GCmZDqEUVW8SxdK9I+ez3Q+YfHubKbeCgM809Ers8Gpt\\\\\\\/OYfRyn4py2fcpkY0nt4aa8lOSI61n996IgRzuzVu2tivCtekvcOeqOTnKKwfKBAlfQJAXDDxPlcaPlaS31VZNXEnh\\\\\\\/1bhotOUY0m9vmomqtT6nL9Uu7exAF46CIzZ+f7Tk33RIzRZKnPiTtOgPX+C818umDtRlgjIl02UK+ToKrNbHcVgj8L3xbqj7vtfW6HVyhe3uvw57Im3wfhftscqF5uXtGeatcf4C4zX3c1R6GP2BI3HLsApEYdTLE1KPnvhwKK99b4HrglgUU1OTV7qV05GWF3Gu3LEPwA76qb3Gl32RdxsqTnKvy3dsMIXMFl67pPSRm3OTlYvuLMCcj8nOg\\\\\\\/5a7wsHOr4ZpwZljQESgaVDEbVqtfe77MkaN\\\\\\\/IrSeRabdXdhAz+dzXXxyy5jaWEHL8PbABSHqp7XWobDWgdjMrd+yvb\\\\\\\/Xaxk1tITrxYd0DLFm4sMujwb4gze5W82hAgcUa1wP1\\\\\\\/BHmlnXQb6N49SPtyuk2MUvDuyZaWyaXAyuPee165Bl1qGRBJZ0K8O+ax5Wg3uQzmp9Xmv6qUElt4+DehwL3XxeeOgQJqVlHG5h7TelXIRblGRTPvbgowz1QeIJr5ZWJsYYCBpkWmV2Y7YS6BnJCZ7lqwRVKkiGnW0zbmqTmvMs9X+Ge7LnDZz7sRLn5nTrr0I4XVLsruPkifAcuyLo72HB1ry4CUUPPmkZ4wSwdIZZ\\\\\\\/DqTzmGD0uzLUitKwUAxnSJhuOV\\\\\\\/MmTqdGtlJYfw7h967Di3MeGH17OL\\\\\\\/eZuZ16GACgQAbnRSXbb4BzsKW7QSnuBQifBuZgd2d0YqAPRvc6h5WeOkP9Zu6fKfWK8sMGHEaSQdmcbMBNic2\\\\\\\/eub1kpYL3Nq4IT5Wn5JVmwL3Jlh5Z0GJ8YG+jzS1ScYLdgF4oP+EZgdNByUzyl3AfCVWUuB02DZx0FvoQxAqECLxpwbyEIJpkwN\\\\\\\/6DFBt+OmKgo9PSdf\\\\\\\/3jVHDmSiKeU05c6sTSBhU\\\\\\\/t7sclpW+QzahsWjVdzOjTrmhxAYKMbKVU5z+aZowpzMTqklccVqJ+jWg2JN+j6DA\\\\\\\/f+EZ1kr9bl2ErX8VsS9MnMfeA5\\\\\\\/1+qnEnjLKN0tv8zKfxIgPN7+qZnTHXSQqYzmFLipVGNgZeFkooODuKohyFWWBaDodgpX\\\\\\\/SA\\\\\\\/koD5RURmaA2KRytgigJfWKevyrx07pqsAUr0GF9YsVIYx+72W9D2nxUHG4M\\\\\\\/4mEa3SnWguBTGvf8Ji+GOEUKxbEbuSfNl7xTlJbyjcflS+ZdLL1VEfLJhwH6pCZpKFg\\\\\\\/CAEw8q0jSaIUnqSpjn972epUOaVTy50Yz7yUPtlvantkGltlXY1g9t3Prlw0JSnm2PAvmvTYk85+ln77RxQ\\\\\\\/mQZKEdBXAE\\\\\\\/1F2Hmz\\\\\\\/GlKimJEYY08qGIj\\\\\\\/BACXbfTds0elrugO25YWXPlQKthM8mjRaEP7HDfszvmuZvhWxmtxIYn3JsELrNh7gp\\\\\\\/y9nuGdc7OPx8SGhGVrp2yRJfkApMrZ2rbr1rMf5uuPpUfJDNuMLo8xU\\\\\\\/EIzt4d3HG7f3\\\\\\\/HQwKLODAtr3mDIaavV\\\\\\\/6Wvlxf9uFz\\\\\\\/\\\\\\\/B8fZiCvDtjwgNzqytX6ufaMbs9GMNwZpJyuMd9GrHo21ieWI2weHfHpZRww0TeSxMiJ47kYMbd64fyKgP9OTfY+QpPMqLFg5TKRFyMEJNBLq\\\\\\\/XJHVaZow3c84ESJwgn8+NIbXhUiRr8fJYwUUGrysJv7Y0lTAZlmA3W73e1oF6kHGLs37+GguLNVxo60m0gCCm2w0bmsTPgBxm5v6rEJfpXkxxrifJ09J7S+sZ\\\\\\\/8lmeGWv4yL\\\\\\\/Udka6BZk2dTGo+Dhrl9r2JN0Hw8TU\\\\\\\/renvP8GNKCOyBRNrE6Che4\\\\\\\/n5jXCuoX9dg3WbmJYSM4C1hXabKnzsqqohRRCh\\\\\\\/grTUDl1aTfEwL+1bsTACfjbq6t2Z9XkgqRK4HYZM3SIq7xE2R958x+Y+NLZ+1J7XjOj4iPGsOClmsfbEVaGo34qOhm9zU35X8kol2bm+sFrcJ\\\\\\\/PcTR7l4zaRA599\\\\\\\/iEw4pcPh\\\\\\\/bWJvURC8uanXjpVBmNQ6\\\\\\\/gRGnNaJyAGz2u7rzDcXbTf0FhX\\\\\\\/FEG5lA\\\\\\\/rMZKHub4bFh01yybfEboG+9ZYvwb7mwUebwuL8qOxHS2++hhCuRhorvROEihk\\\\\\\/8jrBTB1zY\\\\\\\/9JJWKlsd7IcuKG9dIRmv2DovFO8KGisU3bCGTWv5YyEbVtawUrmdFJlZzWckZEIwl3pr8Lh7ig32yrloNn8sWuE3JhPsGoqDa1sEmIu4B42XJskmlWpyhG2aehNHbHL1uVl1v56nvB8HMPar2wVSJeFN8TBEe8g6U4fvxL5\\\\\\\/4Ou3GfrZ76MoJuv+CnVFSYgik0wjktPwFBY7Y5QZYZakfZjAdmjjohDRcmJIv7auVN49NHlD1bxRRVfugn7Gnxi4LMa85rypHu0IOxs76y+8TAdjJhFuF8rvCMcUU0CI5vRw2h7Rp+zTHQUoXpxsziC\\\\\\\/8BkXS9SUG9OpsGGoO2XiMAdyczbokM4OcsT8qaPztLyNw9j9O4coEbVZEr945nGqZmPLiDTRfd89OC7cdhWYb1QQRwMH8Bn1bK2X4QARns\\\\\\\/yl32Iz6e6lJiqUv\\\\\\\/IAVTqjG2\\\\\\\/zV\\\\\\\/zmmuvaFeFen60KI0wsJfJPH8uxUCcze7qVBio\\\\\\\/v+VgI4I1xWBOGtTN\\\\\\\/e72iUJp1b6zeAnsGsisiy6M2+6QCNIcKn2qhezJpacVvdQ3na2yCEcFGtgEAVmJQ+ohFRJbq+agBCmc0899xeoCxnvF8EewlA\\\\\\\/GkUpkgkekA0NFEEi0SyaQkrBA5+cVFo8uOshOcXE3SkfnU5c2cH9fX6Ietu\\\\\\\/jxn1nh+ah3EpSbZ1pJuhefA1XtbEXTg6JN+SGGZpe7L6K1zRoyqV52Qdjm4CGHN7L5yu8rJH68Isk2wGx4lWb0DEzWCMGdYZoTx\\\\\\\/t8lnAKWPISAeeTuhv\\\\\\\/Gi9alE\\\\\\\/eNoQWvI7CO+YetlQz4m0SDx3aIP0N\\\\\\\/B6S+oJ00t98LHDxaQUb6dzgcSWQqjINcK+Wep\\\\\\\/Fhk6KUnrHTlQvsUFOfQ3vWmM56uS78kfn15oSOCmUGkNSkbMDiQeHf0YySEPTW9Hggb7IMYFeDa2AzgcKsvKy3jlMMe9OjlFqlqYmfSNpSPiE5g32Trqdfjl59QdQ5dxLSWfnoplYwDaAsTRIGCWB1CtpbAgbl7jtz3S7vS2V\\\\\\\/QOalyvfCMzHsPsi\\\\\\\/imYrL43SoELwKBOvunbYKqe1INdb29iK5A5hHpEYpuDK5eKGC5YaH\\\\\\\/kwH0TlV2icpkw4GFk0gmZD9MIQolOAclGChR\\\\\\\/kINpZRgcEPtHKkXynqtZZYy2BjrZ4ZfC3ajWtPV4q06IGvGv5lrSUhygCdmPkD2P+hAizWtVL4PjB9DsTLFbmQURQ1FEbjLlfLcuHRJgRGT8SCjW+pUsSB8qKuDIfE86MN3zp1H5Qyq6Eo56Hn0yH7SDElnAjLgO3gOqfqw\\\\\\\/t7zLOSBSHBbbDhMQJth1HGHOq2fCgT3F3MWmID9wRXGvAaNON+p\\\\\\\/Eocqq6xBotuPJQncmx3TQ9zBAhhk17vhB2e+7Vm8HKPmKOrdZ\\\\\\\/qxDh3VVd7coO4+6BnBUPBQnxFb0nr8oXcuCNhdaM2NTTrYv94yNyVQENaTvIckTlaQbc+wlfm3V5IS1YYO8\\\\\\\/wJKf5uWdO5LqfswdyEclrL6XaN\\\\\\\/cOBBm1Ie4R9T2ENa4OmvSR7sVR7bELS\\\\\\\/JkpRldT2x6BL3kVgTiyzo+ZFUl3PBBtIhCPQp2G00UVFmXeuI6qxLuucv+048EI4csVJU5wJE6WUUjX7sAxyEAPUJykLtvggvVjrHkZ1j1qZZT7QGBwlOfwnAXi+F0pjGDtCY\\\\\\\/rS06LqckMBM8TgLrI9V0esr2EEX\\\\\\\/otFFqpa9NgAd3NxYc14R3IvTeku7ol3TgtcldV78KLFGudG9NALorpEbtO1GUasIBHMQv\\\\\\\/ivJ6FCzzb1x4gJlIYnsleOZDnS5aJ8\\\\\\\/S5mYsEAA2mbqIBjQYvON5\\\\\\\/ydRBbpBToHEJOYk\\\\\\\/2p7k8l7+3Y\\\\\\\/kCsWB4j9k1jHk79924TxF57oOavFWb0JNYxLVjOXLRt2n74GlfqFwfBJS7NjS22vIX8CUuFB19S8IKAa9xltiWnOqBqFtOTD6vWKFhg05ybfjvDpvdEr0O2G57lNXZ1ice2hSrwJh2EnIrebTf4304JsglmOVhhApSbBOZJlQG+Ru+XAwVCWrMd\\\\\\\/GA93jtXv4xc2Uxril4RPZUosJ6RVNO1M+cjw9Kgo\\\\\\\/rqdnkqy6rqjabuTYnIAgO5DGxyG7fPFXQm6dLk9LlvGsCJCnyNeY59EVOudD8ry31+ZU36Wzq7jsdThFnDoRB80K7p0\\\\\\\/sDlcnHddpXpRdHeGuxwfn6EOy5H9A0+Co5PvkTGu6ux2hV1SKtndhYi58YJhfBb9UgxZnl9QQ8mUeI\\\\\\\/fQVMf+PS8ikdvzogCJCoEGa912N\\\\\\\/OODOSt1R3YhPX6gTrqSud6KFELYlDRysmfIb\\\\\\\/YjCmSdgyyY1rppPCsw96tJNx09E0kOsNmvGX8AwHr73sRKGRvJilEoy8P5rpYSLh+KLANNwFuQmWwFL9J1rbGFj\\\\\\\/vZZ8RAT\\\\\\\/NwTTkmmhNL2CPLviZ3J5tXehYBHdazpZ0a\\\\\\\/brqt\\\\\\\/fxhUr3jIMQztFA\\\\\\\/yny6pilWQCsxh8Ygkq3lgsJLZNwB870ucBb7QVg6\\\\\\\/4o1nOR5c2T4wbXD65puyB4vQLhTIsDB2BU0p10yto0s8mxPjpQ72Q+bezwjZgQ2ylt1\\\\\\\/4gcWXe64rPj6vvVF0P40shYXKpiNcICna1dYsg2EKaPZLercmRMs7OZmoUJidV+ci3la21GuONxqPmNU4J14vRwW8fVUgbvjlZ2fsA4GXCioqGO7fEWKbj\\\\\\\/QEwTKrZRADJl04rynfW+uoT9CLhis7g392MlqXlG0Rr2xJf7AcEXAYyvVv7hVnMyZDY789fKwcYf12EDCLZPAp2w4\\\\\\\/T2ozxCkDPKsbD+dn5DfZ1O9eemg2qO2m8Isx+j5q+k94u4GJJsbFV8roEutIZbaDoYPFAy4MRktknMOhdD1sukqje3HMI8zrtkVh1ZPmWmCmWo+GpHvgwBniavf5SxgaGUPgjTQ9\\\\\\\/4Yt8W9hNMBffuyfXsGbkRCAF0S+UH4FpuI8G0TapERRHIA6WJ\\\\\\\/zLJjXxCnwkjLd\\\\\\\/CY3iC\\\\\\\/KRsDnQofCWWq4QeicuVWompjjJTGUkCmpIHquja\\\\\\\/W5PYlYxrmnBfKsTtIW2WmukvHIRaeSQuNZB3JVDHXcEvQ0xUPPiBs47ECARYmDqhQW45pZQnOcFzZyfDbeoV4aauypQ+u4V2TfjXdeeQmpF1eg4mC9SW9IHZF6PDPHlNprLSD3BRw\\\\\\\/TsmLsEyLCSOqY4djSofw\\\\\\\/J93bybHW2crYeVSfqT0ZGNIXKcH1aGEbjgZ12vlimNNRy0Y8rU9wIWvVC5Z7OJjOrqh6ObO3VvN+x8CyNM81Z\\\\\\\/iD77CYK68EQqC4mgwvCzDASxHichJ5Bv0iwT8oT1d\\\\\\\/t39E\\\\\\\/6+TJ++7nX3KLGJXk7EaoZ0hOP1SzRkQuI6uqPkjL9TdqWGRtsqaOEmMZNodUw\\\\\\\/vyAbK\\\\\\\/v42UI\\\\\\\/+P3u6QOtBt2CI20s64HhdhyAcLiAvV1msg7cR5A26nq3hU4vC2yjlMtzw1e1M3c6qB6Nu24XdBkp13HIq2e8aFddewJenqXDMdIu4ogKgFA1AjUCelE9dfubv6MCRr4brzaEFV7lBOsgcBJmllaRtWPG8sxjtY11w0HSJSBEOaMH8FRy2GA7UUa9D1m82u7fUd+J07E6AkjnNilIOhhhgwcNBFsxCFXOOhw4RzSFLtrPWlew856jIz+yg3X8zrKSaG80qWMxEhcBYEa+Lr\\\\\\\/OvO4ZDKBQVe1eugYMYw2gLJ9LyME4MEu1y8rZYGkqGSC6+ZJUeUykN0TlzdSB1Fa+coodq+saTTBJwe4N3Z3a1MakOK2hS2S5OYpRWjHXtoh9RhmflKqq1THLDbHxmdSSRqJk+EOd\\\\\\\/fPbHPk7bjFjySeXSz9cJNWBUZN+3J7lVmjVV162fNtnVvgIkidMlm1np5l7+KmuhPhr1Q2GzAllRon\\\\\\\/fPoaQZwRrx4eLGDilyfGc9iDR+4sCQYNRiClKPvP+uBQcDJa55sl\\\\\\\/6yuWXOGgCtX3o63sr6QLFjKpOPXA29zMClla40pES7Dgnp1j59sg7eEjXZS62gw7jVWHGsJAQU\\\\\\\/T2jl15E5SmXC8RgqZ2qF3D9wQOaeQm+iaBBsNmAd41zqGoNB7eS\\\\\\\/369ptwWwwXggubgd8cTZflt90jK+FBjb8P4Poeag\\\\\\\/Vvjt1qtdvv8CkEEi8JhOJaUk+Zn1WblFEDf3Iks4Zi0rJuahoBzPI7aslsatJ+r\\\\\\\/HiauXmTHwLAfk86ujtAM0Va+3+cVmrb0\\\\\\\/FY2vWs\\\\\\\/hLsR62+vR3KT6fFG69ErFCJIq1qKFwbbwqBuMehKf2Zz6K4wPsTZbI0zNAX+\\\\\\\/SZRk6BK7ACTMvhFN\\\\\\\/76pRhGKRT1Jh6c5lTMQ4b0DAD+F5xdXEG+QqUj0MQrQpgtfIOvmvGlyxzCFg2GtSwFZQ+9XZ9C9Jvl562Z\\\\\\\/ZFRR6RLI433Sl1dfqfWmiLqYwjod6adv2wCdrRZ5lbSnRFwwWnwX1IUI6cMrotWq9aJ7QxoFzK0xJKaclHMgAf0OS+PUpddtXniHbRMLfVZ2nL7JV+nhhcx92Ip9rqNgWBxWA3eJE4ULmPh1mrUqh9mw7u3oLSRwf6KnAx9iWSjPrk6NkAiNrjj\\\\\\\/6NRxTJrhrcKIlRNk+TXq9MF9XiPINpzIElPnkUiINcaFKMqc3nTvyaSpStdpAhfbUlDy+245xRthUcJP383lU0tBLdrzSnzx7ZU3NnuBqGgfxGqxm37NspWwqfCJKuTzdszIpLiL3iYn2W2mLMhqpiZvzSyPBCUWDsoTHgpEBv4OJIAwgyFr+LKziKIFNfOSj4bDsB9DSnoebUII7qK6VppAUbF8v5RweSheY26yfvggGpI90t7kfQ36iyBfNVP10XSMmp5oEPpNY9exfio2VxosQz7lKljGF08WyagNYfJUugBL0UTt7bqj8EkJTfGO3WwAGDFh9e1iyPYWt4sP68iSLuu2IamDLj0\\\\\\\/gyO2zyhhK2hiN\\\\\\\/vNWc5NzI8rpkBjuImIqdgNPGXM42Xzrcz7SzJ1kXAa8+QMAEYSbNnD2RIAPa7Wp50XaBEAHKtglF9NjnJl9CmQk7bjS\\\\\\\/El8Q\\\\\\\/f53N22SH1q79Ou3mq8Qwj6ph2eb9v9htVlX1ahzVVaew+nXwUp90ziYwwThgiFjxW89a75Fhg9jv6eV8sST\\\\\\\/mZoUyD2P3Y1wn1qC0ZKlcCRxITUYlLEmnokSJcpctgnHYfQz1SX4qI3kwUHuy2aSFi04fMFnvk7T1lvr7dP3uoUAMn+GjXHSLbSpgUhw36WpSEEORIGRmtwybk7P8xI7eeCyUPA7WkzuG8kEemL6p0rbF6mgucEA9V6GYN521Nxf0NAb+YkAPLHXlC5hfrvjxdpA6vRDRNaz0PHPxLw4blwNs7Rqb96GBuD0hn9tbrVkkE4xnCpVPN0R2i\\\\\\\/IefoNARgzF0qdt2TEjCU8HfW1ZwTPTjGbcAUOrTGPdl12xxz9CHEBKGvC\\\\\\\/zUDyXvY0ckoab25mmbsmxlSPc0Alqf2X21bUSuver7nHgREMcJOoZydn3Q0e2ye52HqVXG\\\\\\\/Km0QYtbXhLokxCGXN59og6XtE2B8sYqjstt5AWn12oMHPK+hHB61p0F7MtZVJnRHL548iWNkWWSwoPj7eFfNQV85j5PQ7pB7SXYSlKT5TlKVQzjVwZ6ZTk3o7Ntr9rCwBPpk9NQBiRhLSWn\\\\\\\/Tmd6NyvR0UF7T4hVbDKoCsppcXQhegD4juFukMBpFtDT4h0mzJZhqOB8KEy1poZ9WFp\\\\\\\/5eoyPwT2XddLfjAsgQe3im5SqyMdvF75M3quhHbXRt7p\\\\\\\/yHVgk7bBQu75Le2mHrZySiYPjPwmJ8sm+xNnLzXG2a21Krp+7ghQqI\\\\\\\/ZG5ioLWz6T7ttosNG+\\\\\\\/MB0uJ6V4ynEDWHbqZlsGq\\\\\\\/VAjaWNsgANt2h0Nng9\\\\\\\/\\\\\\\/HNL3o+zDFwWfoc+WSN4KbfWyh0v7aNOMVDmV\\\\\\\/2h4fFy9a7sTcwQxxaQGHMqlFHM4F3XXYp1sa5tkOskSyVa19nZBjMusve2gU5Gy9B12ADhh2goWmZUMCULCsygWCPNeugGui9D\\\\\\\/EnPI+7uEeCFLys+EJ9962+kXx8UInj2KqDabIUUSO1HAuG+aUeoZT5VmHuiMu1GvEO+6u2ijvNmnNQxIQHrY3WDceZyNgdBiDU+x04vgZkdWzjqIwl5s\\\\\\\/v5EXC5CUAaD0gp7P3i++ynhorUL342\\\\\\\/ft\\\\\\\/FTT3rBmDuz2bBBC8AcCGsWeIgn8Jjnxgsk0TJ0fsbZ3kpRDfczXqu387KeiYJDxAeOhNACf1SRhh8xkFHWoBMsHzjcLIrBpJSU2n4Ve\\\\\\\/+1iP+P\\\\\\\/hKzaUfLJ+iyUDfSvZgDalB3de3aknF4gsceYHE4Hsazv3Xnj4iqX\\\\\\\/txjMbxCNRdGf0yhHnAa8N0XVKfHUOQ1p0wc84NsUyPy0eZ3VRfOZBYRYqlDuqsEWOtvGW2yjta6k9Oz+DSK6XAJLhEKuOkd3sKUpB6jjB95ED1xoxZHAcZlggYj9zUmjE9vMllGFimYWvHclMa3w8IpEIYG71o6R8ADNlHpNA07hLdNCtwpCuiZp5iXl7RPrVdeSQRC7kwNrNat+MpTT6BF2qS8FDNu22AMeDECNqnfmT2acXFU1gpgpvHGTxgCRQeKwoup+xfIWwUvY58PfyA8C5OUFTM87Ap9YhFvgBSWFY1ddiYyACOM3KoBhMVFeINwLt5+vymc3mgm0q\\\\\\\/srdABkgAuyCxv4hnrfP054rTDfw6eHUE8MGqwRSaCXvjmMwj4fLR0dsf8M4kMQmT0tib2pziSvhJF+dOd9xTfA8lX8BMYCoUDiypAAyDa5ifIgOSQWnWRlHu2smVYdOYjSUd3Hg4Yyb5CCJRhLNcmEmTyVs3UEzX\\\\\\\/kVDg5mEjLFlQNsC5o\\\\\\\/QOYiQxMCT6G9B4lqDpRZvyBSUHa4ne9O\\\\\\\/31isGYLAoUzN+YfvGWcMUUrPDKriY5BHPG7CX\\\\\\\/tpOJ4NTE0Cy+wkY7AoiN2Kk0fraR+tko5WlNUXDkhj8VpLxhGPsi02Dza4+7NL5ASKJ+tUA0XtkDQW2XMn5mTM7vHbsLFpUx+s8bke2iwnXfk9rBxc49WT3U4lgoQkfxkfIiCb7aOLhjxYV3zGdaCBJG4baG+cr1V\\\\\\\/aaYsFTlE2oqI2FuYEeeLxlgrGrRaZA13Sk2mo7FtZKOjUJ9343TasRSSH2YLYEwxvwfDncghpcfjQXSTIf0G0\\\\\\\/aKrCD3ouUyjg6yPr2gsjH8F+g2yKfoEyqKlgY1lz8JDCjhpnzPTHtdU+ao2ovUf6ts8tMyCaaisq+wWhLbeJZjL8ATO8fxfUIyeTQMvkJ1cXdSqux7Yl5CxbBrAtZcXm7MTd\\\\\\\/n9xvCMpZIykLAf+uH+kmvp2rtotiznyODyngA4FSIQjjYZ+Dl766lAJUNbfPZawHe2FhUIhWElKXKZV6w\\\\\\\/pFeV3emfulEsZkheKkzHZHuZrJr3yKkyNaaUSW05fxA0zpfNYpQQ0Z4nzAOAPSXzeI300aKkFLOIAbRz3oOz22tlK689AoneeBObDZ23WNp19SS6NOtq6jQJjWT\\\\\\\/6xUNphgiNPV6ooTcgfFrlNdSudAE8u7zJoYNAa7fxhblu0SkDBkUVFJuAMMnmCkP2XxjmVzRIYeqAs\\\\\\\/2c0nPWHx8jGeABtaOyMWpJhyL+oOie+l7\\\\\\\/GMB1iTRuksgw8IFllGIfZsFrn6ftl5uri1g7Scgv8hYNtzT5q\\\\\\\/sgquqG+V1gCMNNPQIgR6vuXvdkpx1kaQJ5SGyDn89Zv80buoJvTbStfDeNfE17+lUBx7vitziGxaUwIA0VrauvyzWnw6lyv317Az21\\\\\\\/eKpBj7LA6ddioQYtw+iZZwn+8YVxLqAdWxcv7LSaMnsYgFGnWLtVsQcmu0HhN11zBM3cr46IQyu5vzcNkz0H71ElwPAazyPr9isamZ8CNSyIEUc8P0ZZBNx7QZkz5wDDc+spDvqpD8S7UAEf1NeBIr2ZuJai2rGPlZN1dEfj+fWDmAL6pw4qVdSLHXYgGnHUJ5C\\\\\\\/q3qlT0WQtf0d7x7CBWhHY+PljLm\\\\\\\/uf\\\\\\\/mIUMmaq80EYqGvzr1CVKjnxT+Hb8DQA8qJWfxcBZFKyAjboVg3uWq801Bm2t82NwlZeXADjENsYYPhmz+sKfGETFPNTCqlblb\\\\\\\/mbE62Bo6mFhfC08nr6mwCl2s1WQpB9MJud\\\\\\\/+3qG2ihg8ZR5LtsRcUpD4HOnOCi3mN9AelA16fUB62CbY9R4oUk1EIMPXtDDvJoezaAm\\\\\\\/okbDHzX21bD7raU5wVSsv0hdTkdlGyobwJHMUTC6wq5kvwGnhtrX3yNRf7tHVIaboY7NpLDWODsFuNrVf7Fa81Pj1Tyk+vK1Tjf5Dl4fG9OyWfgPBl6cZu1Mkanq01dMEKVT8mKJ\\\\\\\/NfrTjoOIIr4Cj5nTUR7YxvMTTVLytQB9nLnl0UgRZomLsDbpfvr\\\\\\\/\\\\\\\/8VsvTqm4pM5m7nbsVpb5BuJ7qhIqR\\\\\\\/aabsTcsEhh1hqvqlYs05J9WfSqN3rMdLCWcK0O9XGfJiRi9ZZXUkqD+cUuQohLI7qafX+oBxEbISELB65l9g3zziX+x\\\\\\\/B+2NFlaUtuqEmtyWQcxFiJlcQdh9mq00e9IfRBQR9vz30cPvY0TYJ0vgVK6m7Kx8qW8i6j5yQ1EmzwujXAXGKm52XZklUmdp5gzT5FIlreJP+WebeST0om3kUP9ieeu7oxUn6Ab3QJkXbAemxXVserL1TU80aCwhoVEGqVIyvK6a0SX7yBYDHdCx1Dey2+phISw\\\\\\\/bNIrZTg0y4J6snnZYkcUCLKqDBcALPp\\\\\\\/YwlMYCHFjTwYIn1fw4netFt+AoCKovoA6BUtj2MAtpOV3VtvpsswfHUppmiPeWwTHWDTBioMRobCQEC9jaZUWlQDr2LutZJJrXL+Vow8Uq0wZ40cbS82p4qcFEsJH7UANxMAjU\\\\\\\/CSHJR35wEiXTaPU19U2fPblFWDPzLlz\\\\\\\/AhAEhnQVdQGCL+5zVBU1aLYZCI5q5l9J8\\\\\\\/EU8bnFjF4yvI6zFFR8gbqw6qdHa1t4Zz\\\\\\\/HGsEZlL9iSscvOgRAl2\\\\\\\/fwlN1w+BsOXH5UI7f7gMotHG\\\\\\\/pLQhwPmakerlsB8emOt0BhVuQSZ9nHDQ61kb1ijTfLJd7i2xFuSkiR0dcRsvsVDs356MoGdEoi5lHQ9j89z98yXFxuZrDvqDCOtmz2lgVboOpX7jkeHdROhTkI5cWyKmslCMGeHFC6CYJlgNgsw9QA0vJYWXRhgXDCSvZsrEGmKEmk8VF\\\\\\\/zAxV910JSNYPeCEBb29WhEtyx1pgSaXpWDpMcBobdxWmocH8k8Xwk6fgnNMTsuBo6Gmb8FwSjfJF9PGVRo4r4j6RxErqNab7LkupOSw1B7uvP8HKKthqXkvFg60sAbTraFn5WfulUcw2I6APO1hNv53atgrqbnbt26KmiOYwi0sxIMrthkcZGXJ9IrmRSs\\\\\\\/9IEOp3544kG+ODYZcY4\\\\\\\/j5RFoJMyC+\\\\\\\/9pU4rPVWDcc7l+UxqOBWJR6M5T+7vSc68RZtk1433paJfe7ntjMe6dwPKfLQSWDm6hafBOZ6gwevSwiIV4f5q1foVNvI1eCiyPg18g53lCgxR5RD6jz+W5mwEAUnuWMiXlTIaGZSfWjQT3V96MnF0DKKEMi6ljGDRtWBCyRdIclkhFNz2y9Y1KLDhmcowUyWSiR+ZdDdV2+\\\\\\\/rKldlI+9L1C4cwlm+eu09SilZa2yq8urhotYXoNtcns275DeC3V4ZGf\\\\\\\/\\\\\\\/P30QXAeMSLNFd6SL\\\\\\\/H7yPtu08FbKCzlQwBk\\\\\\\/a4O9tPwTY75DMa304Z0yLpLuGHlJ8qDMQvnA6OuyIihvVz1OMtsfunWt7ryun\\\\\\\/Nr4VRBnc3bylOXIVf5ng3Qi2Zi7pffsJtd5sZCDuAAejxCOyPK842ZTarWj6zu69CDLKGSDFijzN7Mzc06Xw58MVp3cZuJcKzZx0uGy9\\\\\\\/MivSNnXp6kc4xj8JyNIDkEHz2fG03SX5oRI6jflOTlR4Z8H1OzCfIUj9JiNwFP5W2gM9Qnomh+sem5\\\\\\\/jHhccC8NO9C\\\\\\\/Ux8fCFqLRWYSg\\\\\\\/Tn72dZO4tl32N72jYzpmGkNoonOTpnP4iAnQ9C0kHhVDXYi9wlrBxQEOFhLDdWqZu5+45FyiYMpo4tym4I5Vo9jKE4CCxidvbBNJSZ++BfRBoLu5utd5ryTrukjQg51ECNJENHZUxKbyhcuACj9vh6aMZz46mnsGKS61BM1fLzZ+gqAxxcLFKTAcs6njlUi1qH\\\\\\\/di+q0dTOfA9ZOliX8lOyWCnawPXgRPlC79+7DxsYMVegQWfXHdkKY1NxqHBoz68ffUHpIe18iHbxqS\\\\\\\/kUmRZY6xdxZWR8TLwEgaUy3RJcXuJ3VMykxACFAZyFzI\\\\\\\/HVugnUflGHjNoWHS2GGMROdRpdHhreNcG45+5+\\\\\\\/1l2pCE+9oxdsJUppCMAvJwOLMe3JIPoHRGYUXkHmSvdQRcUJWUIrplxiI0UmdXSXQZ1c5Dw3sE9DCT+Q8OW4y4d8BFoL7JWSG3bi8Oy1HJtzro+nb4ymjepvyLwAXogekMIU0YA\\\\\\\/XpfoDE\\\\\\\/X3yLGzbeDBCQ8vkpSCccIylhPU\\\\\\\/W\\\\\\\/Bm8yVap\\\\\\\/JgPmmxYw9vIWmsqJGMaaUmVdpouuiBAfJzgKMvQCJsBedQGMnmKPB0RdDZxb00EJQTKPEoSyIZGZxOkvphuI9+liA3uE+6BcTleuPsn79IisX98w0y3AtzKJpUQ\\\\\\\/avnUotUMtH038fHCdGKq2S+CEVwzZXnAixxAu8o6YU\\\\\\\/hR39UK60na8MCUnMCEkV1KZphLkc1qqGJZdVCXvcewH0nP12IbCYBtEl9NxTeB0tbmtUvepDdyIKHCaJcjlEzy+f4GyQjBP3+ogZThq5bCsueDrUSff6+5vQrdk5sGkkYnPjO8GA6dWt7L76CzoC6Vl+ENsDXxNnjcLTRYTtO3ZQ1vgU0oIV7CUaWNB0kHfCXHkR7JiAOIaP6LsfFvZHrN6Ecf+il2G0WXlqIgmk0fMdSHWoq05MMDo3gDpeTU71E51zLN1hfHHIJ6MMVOPS8AKQJ4yF0JoMScHQoL66Eosjq03p8lToaddebpS+wTrQnihjWGte4EXovhXxbm6kLBfirm5MFUSIGwMLtYQxQb5scMmxgVo+gb\\\\\\\/q3Piz6ClhKmbpJtJfYukvp7UaQAGC+fdsTpS3dkccnJNHLo\\\\\\\/wFBTIkmx6fTtnRrkcbs2UwP+S1xb3zlQ+x1YjxtxZ9Q9s92i4+MyA\\\\\\\/UFzdah0Kxys2rqPnoA6JZn9NUZL71s0LsnHuEjOwxHUJin8WhvRHLAoLDVmT4cpnvvmMlUQ4mSRtCbZLLK7mdjKMfjnG971jYN9rUDjPQFusxExE8HD5FYYR1bZVkHrp6IM7kKVw\\\\\\\/mKBXAxVq\\\\\\\/ydiDLs90tn3VcFuZDEfkY4FClXtAFRUUKxuuU3JceNdwjgG+7iQ532+IZjY8kEErQvjKcUj16GkeTlIFol\\\\\\\/tNvAqAIZSBGlhD1psyCxy20L2qTX7bN01qZZM\\\\\\\/4sp5QZkQYkMRDkQPptJNRXOCgagG+lpu7x2LbGC3XtlJ7cSixFe\\\\\\\/Ks2FFV89YNFIPpMXcjH\\\\\\\/6vIv78yjY86Odpsj\\\\\\\/DYghJn9Rbn5Y3IhLme8GXDOEhuBAaonaYRPXBv\\\\\\\/nJtUh5fVh87XwCG1OUswSk6tdN2C2vZafsVXTZuzOu+eR9JCHY131VQdFljRU1PhxpYO3jr+yJVshD\\\\\\\/ZePwXDQihpFHwvkSkYiANrRohZpC+fdsY6nQyO5Utb2E8vgU0lMXVjc9bUhmuRKT2eEJ6B36vyHKB0FMDb4XN61emLgSp\\\\\\\/Ha7PaPDqqylFBYAooQ7oJc6Of2t80T0HspjniivP08wWepKeII3Mvpd5J\\\\\\\/7HTt14XqQMvwgOUw1aIp1FUBJQiFieE61L6P0lVsy2nCbEgc0mx3vztSyl45LulYMZJXqFInEM9vi+xUJfa1QTmybItW+GrfEtqnd6A1iS3jyBrAAnGZAsI8v7MzikoH03YziIJvbhR\\\\\\\/i5dwg0qfdSdrdgpbikrFEXt\\\\\\\/pbCuOenca\\\\\\\/I6QJh3U9lfENvVHb1qzQ3Gd0sh1FcLzldmt3qaxrBt+nCYzk7XyjrPj8WZtA9l2vPnpNuuEMHDtlWydEMKCgVgGmBbXBL8ym8XD3WYQ8rPTubSJOVv2fuGlKTL0JpXhfT\\\\\\\/IrgBhplmF\\\\\\\/CvqAozmZBMefoFpY3Pwqi4\\\\\\\/6YK9JJtTSNjQEGpHqMzA1D7dep7KSpwZcqtucji\\\\\\\/3dCrYF7LWwa8bHAdWRp1zqQjdpijZynoGzmYkVaN7rAjcNpiwOjYSiRuEiDIHfBjJ\\\\\\\/eTdSRKw4eoWTyJmLWuC1sZ6wIxoBgY47mdMOSwKq5Hgv918QMgcdud+kwAv+NI2mNuOiqwGNMrr\\\\\\\/rrBV4912acUipEwZLG0gmUjme02aCowOw\\\\\\\/l\\\\\\\/u43tbnW+7XwdQGMz6TGbVbwI0FaOrCygbok9m+UvX0WPtX4vXMUyAKXHux8IvsxP+rfr3HKmzG1IuYDgNxrpyCg0i5S3H\\\\\\\/xA20qtT6LSAosPp\\\\\\\/CIaCh5rjz2bBWGoAAmuGAoK6wuk06RRyf+NxaTSQEvqbitz8Qr7a2JxGQwOa0s4n1+UmWvRAnJY3jdQzU93Mhgiymp3hmfRJtFkBNbHzNNcfMCjYAsMYXAiak52B7RGEiZnuldpUigHIcudC+iBMpQ0+g5Y5yyoJ1XfuFO\\\\\\\/f00Ag7yTpcqiYsmbckS3wMcH2PQuVq3qWzCbhF6VJeXR\\\\\\\/+zniBmPl4xvTbELQGR51NJC0weJMwOhmsKV3nkyRHA\\\\\\\/kLzMMFBE9DuXFjsRSe3gJrEUMqT9VTNy4jQfxdYspzGpbS07YVM7P581vI9E84epzjjhXwhHeCnrRus3PQR4k6IuN1f8v+776qDx+tIHj631E1Ol4b56wMv1LUSt81xvsb9UBrASbQYZDaN8fqtk8hmszGSU8hZHFcC1XY3PLxCIxKqnoJRjLe7SPvQMJHsjbPh41EBLDCeW9XKY6Laly0m\\\\\\\/iJxiEPyBQcEuVEc2bTp1oBFvfG29mvqS175zymi2sXgFLhlTazXw9ULhIwuq+8Dl1YDmTkscxqPWkemRCnLPLLH75RTyKoxGYg6xxvLssilSEsCo7me8eYUslYBWQZWB9oL23Nd7ifVp6OZW05z104Qq1sdHrifq12OS1aQbZL9oHBBSrakhoEBb9NeN5c5KphI5+ktWtPBkEIHVpozAfSeEFnmu4inSckTPqTvFR4V+Ds5Do38\\\\\\\/LSIw9lvDLrTSIm1c003YYCK7vMNWfLDe4P4hD6a\\\\\\\/dQzMGM8PJ7oS6OoBfX717nRClZvVUib54qT+vvlvK7TeS2BSt2JEKZHOPtABjlVesfloFZGCWvI8FCnT9ocwkbecddBKF\\\\\\\/QMWzf6ktvf9e4cPTKjOJyG2+aUDX7CoOcq\\\\\\\/hdyzvIydxquc+qcQ38K6Gd7IN6LrBXq4XiiZpxN3Myv2mn46OnnjBYNjQR8WBubbrdKqAUlYfIkEa+b60Z9WpBTNyVtJ3LxRvnXZJ6vKp64SMhbHFyzKoePrfo5tbm+OF3+nyRQ6+uLKsMsM+fDk75RB470nyoXmoA5PcoJr3TqttXbXey\\\\\\\/jlVoCpAybg6ZCMMNzK+japEkC7aLi4oRCxYveom\\\\\\\/hLrOa+iLO6134ij6\\\\\\\/quhNKWlFiwsrHHdT6lZhp71q85RWNGRBriBedvQ16ekPwzv\\\\\\\/piZ\\\\\\\/aRW50SfODqb7UwP2Q4a5Rp4yHJPWoSWje3j3KgZ4mCprMj2egClFoRuj5gnxubN6vRIOiVaoWdFe3g02mHK2BQfLZQ4jk5plwhADz1agBH4P6HlJySa9y63hCZs6rhiC\\\\\\\/1F\\\\\\\/MlhtAcF\\\\\\\/89HjBnr6Ve1gER3XBBhLZM+pNeuZFRxHmZWSt3L9nj04UvDoNAYuGjlCTRf\\\\\\\/8nw8eEWtSNefYdmQWiSUb5N9qrRoAMlAHK9DlsSi5wdSgZDGy3m5ggqxiaXK+U8FgOdj5rTJgbW1Q0ggEOq2G2VsnuTRG0cSS+WOI0Vruj6BeXolUkg1ygFDvKyZO91ZVlvrvBPYFJHHw5IA586nQMzS3QwwsJBY2qVWtOn+M9ddPnTL5Ke1zLKPQEL6nFx+MvRV+2gh5C1crCpELjjJewyg90RYaPmDKHkXzpL+ydnOqilehY0nNzhi12EVdWZGJt9PZ\\\\\\\/BfMMik2qVbROU0v0KFVUPNPnsAaxa5EIuz\\\\\\\/iEE3dCsypliAOTF28sRgPydX5TYlv8gMDks35jhCzUEFkn5d\\\\\\\/OJgUvU509pQve2tv9Dd5oSZ8vQo67rbpSwrW2rDCZKhBu99TWVUTmdedCpyi4vCzmwajEhjIuJ9TP0OAaX91D6pj2\\\\\\\/ngI6Hw6G3Wzv\\\\\\\/YfeVj+VwOZpdzCwzK9DLSMOamEcixvYAWw9wS847u0yrkT8ZT0dbLYpojigig2A+HunvVSvXCZFg+f4nEQbiGZW0zZe6df66h8oF+IK72zD5BpKUFd62OXQxFad95a\\\\\\\/XCuggW69eOasmkHDBBYke7qUud3qlvfT\\\\\\\/iigTInyaOxDG8QOisV0\\\\\\\/Rz3mQEbQbjQthVeWk8NOebRa\\\\\\\/JCo9BWNK7wlkJgWpAyXghygZ6sDU4j3g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ykS2wdkA\\\\\\\/r2\\\\\\\/L33k+lPkAg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228ac4819271b48118db989f900c5dc2c3e30bc18e22b5393a72acab0725bd4a40\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227Z33hEKzf\\\\\\\/SjfYm9lHnBCt0a5ZsIVmKb8S1gK8LwouOJlzbtnELPKTHzDFcBtVUX4RctJ3mYLT0d7+9UNRrzTSi0anMpgPdquHPLbcb\\\\\\\/G4OA9clMekQunP6hHKU\\\\\\\/jN8Xyw5I8EaEb8nTOJs2aBXWGUmjxWRpqgbV4w2DAPoKSfEnIJDAu\\\\\\\/jzxm2mYI69eEi0KICvHtw12qm3sBxGWtNTqYlar9g4oZJHmLI4v7Q9SDIKH0dzp5l\\\\\\\/4miJP2QU525vA1AMy2PTZVocbT2aUI4ckj8FXuMi561OASVFwH9bXxw60orF+QBfeCyYJzzPQSESS0s630RCC5f6C2BnvHlnARQRDpDFCDUPGtpUI9iVW+Km+7zYA1BqAS4z5sh8DYgojgxaDrSDLy3eAGYg9Oah6HB2yl1msBlIDSGO4g1XIw7r6ICs22wANoe7gOEsCrOCEl9IqG2yu5y+v5ZRIUCbg38\\\\\\\/UlLstkGNS3j4y+F0EM07o0Ga2jIxXU49bs01e0mLebugalUEMhBIQlR+kXeKz2bGX21XjzSzB2SZcVKtnnT4UZtbQO+6\\\\\\\/YU9S+WPs666V7HxZ20g1Fkl8QzQa34oQ\\\\\\\/IROQzlp272RQrgXtgWdA5kFJWRyYpFtwdZGip+AWtitAuCzhCtREgh4mYpi4G3xRTsVvh6fqndx8Ytz1fsxT+cqCXEct5l\\\\\\\/3Xqvd35kdE5sucLcpQnwFR+pdNKGpBce6Xe\\\\\\\/4x\\\\\\\/c9cACirEZ8qtTLiN4zctdOsfTcjIPt0XJbOAUS\\\\\\\/BM3Exi2UXLUPbiJh86VCxmzo\\\\\\\/QCgSGml6kJBt08C8j5r6jtm6XdqS\\\\\\\/+Ih+N5udZe3P\\\\\\\/+UX3XUuFPuEN7Me+aR+vT59hBTfeVpfi4jZMqtA1iQc6xdme62\\\\\\\/SW\\\\\\\/PH0POJUR6pXcvlAP36\\\\\\\/it0DyZAP\\\\\\\/n\\\\\\\/OkeuOyKd+jDu2Qc5AjVVmEM2tWLKbp\\\\\\\/F00JTjJORT7T98ZRoHh3VjhcUV6Q0mHFQCGDza8uUb8RVRXO0gAYY5SS0mLJRXlnjiH+iC6cantqcnaVo1inG3ZYuHtVnQ49fKZWY34x22bDPZ3QgZrnVJPMKgpezLezu1a9bFcDIBH63BF\\\\\\\/03jSlTakf9CeCmhu1o8E6tvzREeWzDjPmj+ZDyI\\\\\\\/qWfs6araOKEIbuQPnKDGc8e+CeFrCqz\\\\\\\/RS3eIFGCbXlRFuGo8c1j5KLafhDF6JupKDDAdTCCsRq+h1FuP9ebVuyBPjczrwVWmjCeCwP78mIiHGz1GV3jIqXpxmexkkmSvuUSLneIgZS076AuQD95iGG6T1AN3hAwii3Ic9gyl6kOGpXp4OFh26kHgK\\\\\\\/HYjGEF8gLHxMIn4z6PSgrC4XfHQxXffX15\\\\\\\/J0yq12eIyJ83jzFY8K9WFS0AdO\\\\\\\/onu8oKDQfPQfAU4jrSOoqhdHNtpjL3Bm+D1N3tQWzq+IGofu\\\\\\\/IBdC8DJ9k6e+oFmXPdj3ZFeMOkjQuOkjWozfTdUNgO3AJhRZh6UtiC802XOLLStoEgJxELTBXI7K9VBhZ9JT3DaOqSAgtpHnYqCpyIfNQG9g8KV1Bi39iI\\\\\\\/oCzwnIJg3+Gc7HZZsBhqxYWPixP2Cp3H4\\\\\\\/LDWojK5gPzgJyKcW11coYSOhQNynyUUlf2Q6keBxqXk53Cc35NUVpbQVHH\\\\\\\/8T0JJPPlew8pnFBTcI8435woGKkB9WqabLigxl0z+xwAHPts+dkNyDALVzATocr\\\\\\\/OuO8Dmj35wxN09oFMwHYo6wQBb5gHwEllQuDIlRmXdfyFDN\\\\\\\/pjtzAsSyBFDn74AGZ4VL7xIZBcw6mrIxIY4eGpUOSXS2fIpFf4Kq5F8UgVrkjL84N0AhkrxHiOOIJ6sY0THvpxRKjUU1cVezo17VyFNps+eTTn2B7yKYsbpVZvGeSLV0V2elAEGJYoYBlNkds3ceHOMvQavGy+IgfKMS42B3qdM35aRWJ2v4WN2ORgCKCio86OGxCbFotEc82F16x2Ge3+U4i2nObdDfI0Y\\\\\\\/rciyRNSMiAP6bs8wQfQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/U0RVZardbH4QrsrJIYU6w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f6b6452814970f4d4d82086671d613762eecbe02e5db9c333e45966122e91cae\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494271 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3297" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0VJYtzCil0wCcWHqeNNSVi_5oizxc-GJDjZViJS6ZR0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3297" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zimxxrLgnq1wIRVQBXZ_UoVkvwcI-_tW09JBfPEuxfo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3287" title="Comparative Transcriptomic Analysis Reveals Molecular Profiles of Central Nervous System in Maternal Diapause Induction of Locusta migratoria" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3315" title="Disease Resistance Genetics and Genomics in Octoploid Strawberry" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494271" id="node494271" data-pisa="ggg;9/10/3297" data-pisa-master="ggg;g3.119.400378" data-apath="/ggg/9/10/3297.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcelo Mollinari</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio Augusto Franco Garcia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3297-3314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400378 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494271" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcelo Mollinari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticultural Science, Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcelo%2BMollinari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mollinari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcelo%2BMollinari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcelo Mollinari</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonio Augusto Franco Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of São Paulo/ESALQ, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BAugusto%2BFranco%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BAugusto%2BFranco%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio Augusto Franco Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3297" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494271" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494271/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3297.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494271" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494271/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3297.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494271" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494271/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3297.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494271" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494271/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Modern SNP genotyping technologies allow measurement of the relative abundance of different alleles for a given locus and consequently estimation of their allele dosage, opening a new road for genetic studies in autopolyploids. Despite advances in genetic linkage analysis in autotetraploids, there is a lack of statistical models to perform linkage analysis in organisms with higher ploidy levels. In this paper, we present a statistical method to estimate recombination fractions and infer linkage phases in full-sib populations of autopolyploid species with even ploidy levels for a set of SNP markers using hidden Markov models. Our method uses efficient two-point procedures to reduce the search space for the best linkage phase configuration and reestimate the final parameters by maximizing the likelihood of the Markov chain. To evaluate the method, and demonstrate its properties, we rely on simulations of autotetraploid, autohexaploid and autooctaploid populations and on a real tetraploid potato data set. The results show the reliability of our approach, including situations with complex linkage phase scenarios in hexaploid and octaploid populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">Polyploidy</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">Recombination</a></li><li class="kwd"><a href="/keyword/fraction" class="hw-term hw-article-keyword hw-article-keyword-fraction" rel="nofollow">Fraction</a></li><li class="kwd"><a href="/keyword/bivalent-pairing" class="hw-term hw-article-keyword hw-article-keyword-bivalent-pairing" rel="nofollow">Bivalent Pairing</a></li><li class="kwd"><a href="/keyword/multilocus" class="hw-term hw-article-keyword hw-article-keyword-multilocus" rel="nofollow">Multilocus</a></li><li class="kwd"><a href="/keyword/analysis" class="hw-term hw-article-keyword hw-article-keyword-analysis" rel="nofollow">Analysis</a></li></ul><p id="p-3">Polyploids are organisms with more than two sets of chromosomes. They are very important in agriculture and play a fundamental role in evolutionary processes, such as differentiation of species (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Soltis <em>et al.</em> 2014a</a>). The number of sets of chromosomes in an organism is called <em>ploidy level</em>. These multiple chromosome sets can originate from the combination of genomes from different, but related species, or from duplicated genomes from the same species (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Birchler 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Comai 2005</a>). In the first scenario, they are called <em>allopolyploids</em>; in the second, <em>autopolyploids</em>. Polyploid organisms are also characterized according to their pattern of inheritance. In general, allopolyploids exhibit diploid-like (or <em>disomic</em>) segregation, since homologous chromosomes, or homologs, tend to form bivalents within each sub-genome. Autopolyploids, however, have more than two homologs per homology group, forming either random bivalents or multivalents during meiosis, resulting in <em>polysomic segregation</em> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sybenga 1975</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Soltis <em>et al.</em> 1993</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Osborn <em>et al.</em> 2003</a>). Since the molecular mechanics of polyploid organisms are quite complex, this dichotomy is often broken, and polyploids can display intermediate modes of inheritance (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Otto and Whitton 2000</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Osborn <em>et al.</em> 2003</a>). Throughout this paper, the term autopolyploid (or autotetraploid, autohexaploid, etc.) will refer to polyploid organisms that exhibit polysomic segregation.</p><p id="p-4">Despite advances in genetic studies in autotetraploids, (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mather 1936</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fisher 1943</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">1947</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hackett <em>et al.</em> 2001</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Luo <em>et al.</em> 2004</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">2006</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wu <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Leach <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Xu <em>et al.</em> 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rehmsmeier 2013</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zheng <em>et al.</em> 2016</a>), there is still a shortage of statistical methods to address organisms with higher ploidy levels, such as sweet potato (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kriegner <em>et al.</em> 2003</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arizio <em>et al.</em> 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Shirasawa <em>et al.</em> 2017</a>), sugarcane (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wang <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Garcia <em>et al.</em> 2013</a>), some ornamental flowers and forage crops (reviewed in (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Soltis <em>et al.</em> 2014b</a>)). In this work, we denote as <em>high-level autopolyploids</em> those autopolyploid organisms with ploidy level greater than four. A fundamental class of statistical methods that have lagged behind in high-level autopolyploid studies is the construction of genetic maps. A reliable genetic map is a crucial step in quantitative trait loci (QTL) analysis, as well as the assembly of reference genomes and the study of evolutionary processes (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lewin <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Luo <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lemmon and Doebley 2014</a>). Although understanding the concept of genetic mapping is rather easy, the construction of such maps in high-level autopolyploids is challenging. Even under bivalent pairing, there are many possible configurations during meiosis, and the number of possibilities gets exponentially larger as the ploidy level increases. Denoting <em>m</em> as the ploidy level, it is possible to find up to <em>m</em> different alleles for a locus in one individual. Furthermore, if some of those alleles are not distinguishable, it is necessary to consider the number of copies of each different allelic form, also known as <em>dosage</em>.</p><p id="p-5">The construction of a genetic map in a full-sib population can be summarized in five basic steps: <em>i</em>) estimation of pairwise recombination fractions and associated statistical tests; <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-1.gif"/></noscript></span></span>) separation of markers into linkage groups; <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-2.gif"/></noscript></span></span>) ordering of markers within each linkage group using an optimization technique; <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-3.gif"/></noscript></span></span>) parental phasing, recombination fraction updating and likelihood computation (or other objective function) and <em>v</em>) if the order is optimal, the map is complete, otherwise, return to step <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-4.gif"/></noscript></span></span>. Historically, genetic maps in high-level autopolyploids have been constructed using only alleles present in one homolog, called <em>single-dose</em> or <em>simplex</em> markers (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wu <em>et al.</em> 1992</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sorrells 1992</a>). In a full-sib population, these markers segregate in a 1:1 ratio (if they are present only in one parent), or in a 1:2:1 ratio (if present in both parents, also called <em>double simplex</em>). Given this level of simplification, it is possible to use the five-step procedure coupled with a standard software suitable for diploid populations. Nevertheless, it is well accepted that the use of single-dose markers imposes limitations on the construction of adequate genetic maps. These approaches sub-sample the genome (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Garcia <em>et al.</em> 2013</a>), which precludes further consideration of multiallelic effects in models for QTL mapping and subsequent studies. Moreover, there is low statistical power to detect linkage when markers are in repulsion phase configurations (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Wu <em>et al.</em> 1992</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ripol <em>et al.</em> 1999</a>). Although some authors have addressed this problem by including multi-dose (or multiplex) markers when constructing genetic maps and performing QTL mapping (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Ripol <em>et al.</em> 1999</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Doerge and Craig 2000</a>), the limitations on the genotyping technologies at the time required that the allelic dosage had to be inferred based on expected segregation rates. Because of the high amount of hidden information imposed by marker systems on those studies (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Wu <em>et al.</em> 1992</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Ripol <em>et al.</em> 1999</a>), the estimation of recombination fraction between multi-dose markers was highly impaired.</p><p id="p-6">Quantitative genotyping technologies for single nucleotide polymorphism (SNPs) evaluation have opened the door for further genetic mapping studies in high-level autopolyploids. It is now possible to measure the abundance of specific alleles within a locus in a polyploid genome (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Voorrips <em>et al.</em> 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Serang <em>et al.</em> 2012</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Garcia <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bargary <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mollinari and Serang 2015</a>). This technology, combined with the genotypic distribution in the population, makes it possible to infer the allelic dosage by using the ratio between the abundances of the two alternative alleles (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Serang <em>et al.</em> 2012</a>). Once the dosage of the markers is estimated, the construction of linkage maps can be significantly improved by taking this information into account, as previously done in autotetraploids by <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hackett <em>et al.</em> (2013</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2014)</a>.</p><p id="p-7">Genetic linkage maps can be constructed based on two-point or multipoint estimates of the recombination fraction. Two-point methods use information on pairs of markers, and even though they are less computationally demanding than multipoint methods, they require a higher amount of information in the markers to provide reliable results. Only recently, using a two-point-based method, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">van Geest <em>et al.</em> (2017)</a> published an integrated hexaploid chrysanthemum genetic map using scripts implemented in the R package polymapR (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>). Multipoint approaches, instead, use information of multiple markers present in a linkage group, increasing the statistical efficiency of the analysis (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lander and Green 1987</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jiang and Zeng 1997</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mollinari <em>et al.</em> 2009</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Leach <em>et al.</em> 2010</a>). This feature is particularly important in polyploid linkage analysis, where markers are mostly partially informative. One widely used procedure to obtain multipoint estimates is the hidden Markov model (HMM) (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lander and Green 1987</a>). The construction of the genetic map using this method provides the estimates of the recombination fractions between all adjacent markers in a linkage group, as well as the multipoint likelihood, which has been shown to be an excellent criterion to evaluate and compare linkage phase configurations and orders of markers (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Mollinari <em>et al.</em> 2009</a>). <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Leach <em>et al.</em> (2010)</a> presented a statistical framework in which HMMs were applied to reconstruct genetic linkage maps, but it was limited to autotetraploids. Recently, software packages such as polymapR (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>), pergola (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grandke <em>et al.</em> 2017</a>) and netgwas (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Behrouzi and Wit 2017</a>), have been developed to build genetic maps in high-level autopolyploids. However, only polymapR is capable of estimating recombination fractions and inferring parental linkage phases in outcrossing populations, though it does not use multipoint procedures to perform those tasks.</p><p id="p-8">The main challenges we address in this paper are the inference of the haplotypes of the multiple homologs and the multipoint estimation of recombination fractions in high-level autopolyploids. Although <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Zheng <em>et al.</em> (2016)</a> proposed a probabilistic multilocus haplotype reconstruction model for autotetraploids considering double reduction, this remains as an open question for organisms with higher ploidy levels. Our method relies on an HMM and is developed for species with even ploidy levels under random chromosome segregation (complete polysomic inheritance). We also present a two-point method which is capable of dealing with hundreds of markers even in high ploidy level scenarios. Hence, we are proposing solutions for steps <em>i</em> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-5.gif"/></noscript></span></span> in high-level autopolyploids. Step <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-6.gif"/></noscript></span></span> is straightforward from step <em>i</em> using clustering algorithms, as proposed by (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Van Ooijen and Jansen 2013</a>). Even though step <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-7.gif"/></noscript></span></span> is a challenging task in genetic mapping, it can be addressed using pairwise recombination fractions or the resulting likelihood of the Markov model as it has been proposed by several studies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lander <em>et al.</em> 1987</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Buetow and Chakravarti 1987</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Doerge 1996</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Van Os <em>et al.</em> 2005</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wu <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Preedy and Hackett 2016</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wang <em>et al.</em> 2016</a>). To evaluate our method, and to show its properties, we rely on simulations of autotetraploid, autohexaploid, and autooctaploid data and on a real tetraploid potato data set. We also perform a set of hexaploid simulations to compare our method to the one implemented in polymapR (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>). The R computer codes to reproduce all simulations and analysis are publicly available.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">In this section, we define the notation used throughout this article and present the probabilistic model for the gamete formation in autopolyploids. The mathematical derivation of the HMM, including the estimation of the model parameters, is based on the work of <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rabiner (1989)</a>, which presents the hidden Markovian process using tree major elements, namely, the <em>transition probability</em> function (Equation 6), the <em>initial state</em> function (Equation 7), the <em>emission probability</em> function (Equations 8 and 9). In the genetic mapping context, the first connects adjacent marker loci in function of their recombination fraction, the second is the prior probability of the genotypes in the mapping population, and the third connects the observed marker dosage to the complete multi-allelic hidden genotypic states. While these ideas are widespread in the genetic mapping literature, for instance in <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Lander and Green (1987)</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jiang and Zeng (1997)</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hackett and Broadfoot (2003)</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Hackett <em>et al.</em> (2013)</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Leach <em>et al.</em> (2010)</a>, here we present a generalization to any even ploidy level. We avoid using matrix notation throughout our model derivation since their high dimensionality would precludes the application of HMM techniques in polyploids with high ploidy level. We conclude this section by explaining the complexity of estimating linkage phases between markers, presenting an efficient two-point algorithm that simplifies the problem in a way that allows the phasing to be inferred using real data.</p><div id="sec-2" class="subsection"><h3>Notation</h3><p id="p-10">Consider a mapping population derived from a cross between two autopolyploid individuals <em>P</em> and <em>Q</em> with the same ploidy level (full-sib family). The ploidy level is denoted by <em>m</em>, and can be any even number greater than zero. Let the vectors <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-9.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-11.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-12.gif"/></noscript></span></span>, denote the genotype of two adjacent multiallelic loci <em>k</em> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-13.gif"/></noscript></span></span> in <em>P</em> and <em>Q</em>, respectively. The superscript <em>i</em> indicates one of the possible alleles for the loci, and each locus has <em>m</em> different alleles in each parent. For example, for a cross between two autohexaploid individuals, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-14.gif"/></noscript></span></span>; similarly, this can be done for <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-15.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-17.gif"/></noscript></span></span>. All alleles denoted by the same superscript number are in the same homologs (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-18.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-19.gif"/></noscript></span></span> are in homolog 1, etc). The following assumptions are made to ensure random chromosome segregation (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Muller 1914</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Haldane 1930</a>) and no double reduction (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Burnham 1962</a>): <em>i</em>) there is only formation of bivalents during the meiosis; <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-20.gif"/></noscript></span></span>) there is no preferential pairing during the formation of bivalents; <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-21.gif"/></noscript></span></span>) all bivalents have the same recombination fraction between loci <em>k</em> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-22.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-23.gif"/></noscript></span></span>) bivalents are independent; <em>v</em>) there is separation of sister chromatids during meiosis II and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-24.gif"/></noscript></span></span>) there is no chromatid interference. Consequences of violations of these assumptions will be addressed later using simulations. Although each bivalent is composed by a pair of chromosomes with two sister chromatids each, given assumption <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-25.gif"/></noscript></span></span>, we will consider only one sister chromatid per homolog during the derivation of our model.</p></div><div id="sec-3" class="subsection"><h3>Bivalent formation</h3><p id="p-11">Bivalent formation occurs during meiosis I (more specifically, at the pachytene stage of prophase). In diploid cells, there is only one possible pairing configuration: two duplicated homologs from a homology group pair to form one bivalent. However, in autopolyploid cells, given the previous assumptions, the expected number of possible pairing configurations, <em>i.e.</em>, the number of possible bivalent chromosomal pairings for a given homology group during meiosis can be obtained by sequentially choosing pairs out of <em>m</em> homologs without replacement, divided by all possible permutations of the chosen pairs<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>The orientation of the bivalents does not affect the expected frequencies of each gamete type, and therefore will not be considered. For example, as showed by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hackett (2001)</a> in autotetraploids, there are two bivalents and three possible bivalent configurations: homolog pair as 1 with 2, and 3 with 4; or, 1 with 3 and 2 with 4; or 1 with 4 and 2 with 3. We denote <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-27.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-28.gif"/></noscript></span></span> a set of all bivalent configurations for a given ploidy level.</p></div><div id="sec-4" class="subsection"><h3>Expected gametic frequency for a given bivalent configuration</h3><p id="p-12">We will present the expected gametic frequencies considering parent <em>P</em>. Since parent <em>Q</em> undergoes a similar process, it is possible to combine the expected gametic frequencies to obtain the expected genotypic frequency in the full-sib population. Each of the bivalents obtained for a given configuration <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-29.gif"/></noscript></span></span> can result in two types of chromosomes for loci <em>k</em> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-30.gif"/></noscript></span></span>: <em>parental</em>, which results from bivalents with zero or any other even number of recombinations between <em>k</em> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-31.gif"/></noscript></span></span>; and <em>recombinants</em>, which results from bivalents with any odd number of recombinations. As presented by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Doerge and Craig (2000)</a>, the probabilities of all chromosome types for any single bivalent can be represented always as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-32.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-33.gif"/></noscript></span></span> is the recombination fraction between <em>k</em> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-34.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-35.gif"/></noscript></span></span>. For a given configuration <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-36.gif"/></noscript></span></span>, the expected frequencies for all possible gametes derived from that configuration is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-37.gif"/></noscript></span></span>where ⊗ denotes the Kronecker product of matrices and subscripts in V indicate the corresponding bivalent. All elements of this product are of the form<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-38.gif"/></noscript></span></span>where <em>l</em> denotes the number of total recombinant bivalents between loci <em>k</em> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-39.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-40.gif"/></noscript></span></span>. From this, we can define the probability of observing any gamete (for two loci) given a bivalent configuration <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-41.gif"/></noscript></span></span> as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where vectors <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-43.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-44.gif"/></noscript></span></span> denote a subset of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-45.gif"/></noscript></span></span> alleles present in <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-47.gif"/></noscript></span></span>, respectively; <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-48.gif"/></noscript></span></span> indicates a gamete for loci <em>k</em> and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-49.gif"/></noscript></span></span> from parent <em>P</em>. <em>Consistent</em> means that the gamete can be produced from bivalent configuration <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-50.gif"/></noscript></span></span>. Notice that some gametes cannot be obtained from <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-51.gif"/></noscript></span></span> once the bivalents are formed.</p><p id="p-13">Since we assume that alleles with the same superscript are in the same homolog, <em>l</em> can be obtained by a simple examination of superscripts of elements contained in <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-52.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-53.gif"/></noscript></span></span>. Consider, for example, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-54.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-55.gif"/></noscript></span></span>, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). If one observes <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-56.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-57.gif"/></noscript></span></span>, the number of recombinant chromosomes is <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-58.gif"/></noscript></span></span>. Therefore, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-59.gif"/></noscript></span></span>. On the other hand, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-60.gif"/></noscript></span></span>, since it is impossible to obtain this gamete from configuration <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-61.gif"/></noscript></span></span>, <em>i.e.</em>, it is not <em>consistent</em> with <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-62.gif"/></noscript></span></span>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="One possible pairing configuration in an autohexaploid, namely .  denotes one allele present in homolog i for locus k in parent P. Notice that some allelic configurations, such as , are impossible to be obtained in this bivalent pairing. In this case, the homologs containing alleles  and  will migrate to opposite poles of the cell during meiosis I. Therefore,  and  will not be present in the same gamete." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1714010710" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;One possible pairing configuration in an autohexaploid, namely &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-63&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-64&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; denotes one allele present in homolog &lt;em&gt;i&lt;/em&gt; for locus &lt;em&gt;k&lt;/em&gt; in parent &lt;em&gt;P&lt;/em&gt;. Notice that some allelic configurations, such as &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-65&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, are impossible to be obtained in this bivalent pairing. In this case, the homologs containing alleles &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-66&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-67&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; will migrate to opposite poles of the cell during meiosis I. Therefore, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-68&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-69&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; will not be present in the same gamete.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3297/F1.medium.gif" width="440" height="70"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3297/F1.medium.gif" width="440" height="70"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494433" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">One possible pairing configuration in an autohexaploid, namely <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-63.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-64.gif"/></noscript></span></span> denotes one allele present in homolog <em>i</em> for locus <em>k</em> in parent <em>P</em>. Notice that some allelic configurations, such as <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-65.gif"/></noscript></span></span>, are impossible to be obtained in this bivalent pairing. In this case, the homologs containing alleles <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-67.gif"/></noscript></span></span> will migrate to opposite poles of the cell during meiosis I. Therefore, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-68.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F1/embed/mml-math-69.gif"/></noscript></span></span> will not be present in the same gamete.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Expected gametic frequency unconditional to bivalent configurations</h3><p id="p-15">In reality <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-70.gif"/></noscript></span></span> is unknown, thus the conditional probability given by Equation (2) must be considered for all possible <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-71.gif"/></noscript></span></span>. The probability of observing a gamete <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-72.gif"/></noscript></span></span>, unconditional to <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-73.gif"/></noscript></span></span>, can be expressed as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-74.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>It is important to notice that only a subset of Ψ is <em>consistent</em> with the observed gamete, and consequently <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-75.gif"/></noscript></span></span> only for some <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-76.gif"/></noscript></span></span>’s. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows a graphical representation of Equations 2 and 3 for autohexaploid gametes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of Equations 2 and 3 for autohexaploid gametes. The first 15 tables represent the gametic probabilities given different bivalent configurations  (Equation 2). The rows and the columns indicate gametic configurations for loci k and , respectively. For simplification, only the superscripts of the gametic configurations were presented. For example, row 123, column 123, represent the gamete . Colored cells indicate the probability of gametic configurations consistent with the bivalent configuration . The color scale indicates the number of recombinant bivalents associated to the gametic probability varying from 0 (dark blue) to 3 (light blue). Blank cells indicate non-consistent configurations. The far right full table represents the sum over all &#x3C8; configurations, weighted by their probability (Equation 3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1714010710" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Graphical representation of Equations 2 and 3 for autohexaploid gametes. The first 15 tables represent the gametic probabilities given different bivalent configurations &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-77&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (Equation 2). The rows and the columns indicate gametic configurations for loci &lt;em&gt;k&lt;/em&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-78&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, respectively. For simplification, only the superscripts of the gametic configurations were presented. For example, row 123, column 123, represent the gamete &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-79&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Colored cells indicate the probability of gametic configurations &lt;em&gt;consistent&lt;/em&gt; with the bivalent configuration &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-80&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The color scale indicates the number of recombinant bivalents associated to the gametic probability varying from 0 (dark blue) to 3 (light blue). Blank cells indicate &lt;em&gt;non-consistent&lt;/em&gt; configurations. The far right full table represents the sum over all &lt;em&gt;&#x3C8;&lt;/em&gt; configurations, weighted by their probability (Equation 3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3297/F2.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3297/F2.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494310" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-16" class="first-child">Graphical representation of Equations 2 and 3 for autohexaploid gametes. The first 15 tables represent the gametic probabilities given different bivalent configurations <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-77.gif"/></noscript></span></span> (Equation 2). The rows and the columns indicate gametic configurations for loci <em>k</em> and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-78.gif"/></noscript></span></span>, respectively. For simplification, only the superscripts of the gametic configurations were presented. For example, row 123, column 123, represent the gamete <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-79.gif"/></noscript></span></span>. Colored cells indicate the probability of gametic configurations <em>consistent</em> with the bivalent configuration <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F2/embed/mml-math-80.gif"/></noscript></span></span>. The color scale indicates the number of recombinant bivalents associated to the gametic probability varying from 0 (dark blue) to 3 (light blue). Blank cells indicate <em>non-consistent</em> configurations. The far right full table represents the sum over all <em>ψ</em> configurations, weighted by their probability (Equation 3).</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The probability of observing a specific gamete is always the same for each <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-81.gif"/></noscript></span></span> in this consistent subset (Equation 2). Therefore, under random pairing (assumption <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-82.gif"/></noscript></span></span>), our task reduces to finding the number of elements in this subset that are consistent with the observed gamete and multiply <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-83.gif"/></noscript></span></span> by this number. The result is the probability of observing a gamete unconditional to the bivalent configuration.</p><p id="p-18">For every gamete, <em>l</em> can change from zero to <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-84.gif"/></noscript></span></span> recombinant homologs. The observed gamete is the result of homologs that migrate to one pole of the cell at anaphase I with a subsequent migration to opposite poles at anaphase II. Since we are assuming that there is separation of sister chromatids during anaphase II, if <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-85.gif"/></noscript></span></span> (all chromosomes are of parental type), there is no information about the pairing configuration of the homologs that migrate to the opposite pole of the cell. In this situation, there are <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-86.gif"/></noscript></span></span> possible pairing configurations, and the number of possible <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-87.gif"/></noscript></span></span> that can produce gametes with <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-88.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-89.gif"/></noscript></span></span>. Therefore, for <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-90.gif"/></noscript></span></span>, there are <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-91.gif"/></noscript></span></span> possible pairing configurations of parental chromosomes. For the remaining <em>l</em> recombinant chromosomes, the number of possible pairing configurations is <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-92.gif"/></noscript></span></span>. Thus, the total number of possible pairing configurations that can produce a specific gamete is <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-93.gif"/></noscript></span></span>. This is precisely the number of elements in the subset of Ψ consistent with the observed gamete. Given the assumption of no preferential pairing during the formation of bivalents, <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-94.gif"/></noscript></span></span>, the probability of a gamete <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-95.gif"/></noscript></span></span>, unconditional to <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-96.gif"/></noscript></span></span>, can be simplified to</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-97.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span></div><div id="sec-6" class="subsection"><h3>Map reconstruction via hidden Markov model</h3><p id="p-19">The construction of a genetic map involves the estimation of the genetic distance and order between markers within linkage groups. If the origin of the haplotypes (<em>i.e.</em>, linkage phase) for the parents of the mapping population is unknown, it also needs to be estimated. For several years, hidden Markov models have been proven to be an excellent avenue for obtaining these estimates (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Lander and Green 1987</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Jiang and Zeng 1997</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Mollinari <em>et al.</em> 2009</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Leach <em>et al.</em> 2010</a>). The multipoint likelihood obtained using HMMs is employable as a criterion to compare marker orders and judge which one is best, and also to provide a reliable estimation of recombination fraction and linkage phases. (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Rabiner 1989</a>) defines an HMM as a generative process composed of three well-defined probability distributions: <em>transition</em>, <em>initial state</em> and <em>emission</em>. In genetic mapping context, the transition probability distribution is defined as the probability of having a particular genotype at position <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-98.gif"/></noscript></span></span>, given the genotype at position <em>k</em>. Using Equation (4) the gametic transition probabilities <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-99.gif"/></noscript></span></span>, or the conditional probability of a gamete genotype at locus <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-100.gif"/></noscript></span></span> given the gamete genotype at locus <em>k</em>, is simply<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-101.gif"/></noscript></span></span>Under random chromosome segregation, both <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-102.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-103.gif"/></noscript></span></span> can have <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-104.gif"/></noscript></span></span> different genotypes. Let <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-105.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-106.gif"/></noscript></span></span> denote all possible genotypes that <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-107.gif"/></noscript></span></span> can assume for locus <em>k</em>. Also, assume that genotypes in <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-108.gif"/></noscript></span></span> are arranged according to the lexicographical order of their superscripts. For example, in an autotetraploid, <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-109.gif"/></noscript></span></span> for locus <em>k</em>. After some simplifications (see Supplementary Information, File S1) the transition probability, <em>i.e.</em>, the conditional probability of a gametic genotype <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-110.gif"/></noscript></span></span> in locus <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-111.gif"/></noscript></span></span> given the gametic genotype <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-112.gif"/></noscript></span></span> in locus <em>k</em>, is<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-113.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-114.gif"/></noscript></span></span>. The initial state and the emission probability distributions will be addressed in the next section (Equations 7 to 9).</p></div><div id="sec-7" class="subsection"><h3>Including information from both parents</h3><p id="p-20">Any given individual in a full-sib population is formed by the union of gametes from both parents, <em>P</em> and <em>Q</em>. Each parent can form <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-115.gif"/></noscript></span></span> different gametes for locus <em>k</em>. Since the formation of gametes in both parents is independent, the genotypic transition probability distribution can be written as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-116.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-117.gif"/></noscript></span></span> denotes the genotype of an individual derived from the union of gametes <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-118.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-119.gif"/></noscript></span></span> at locus <em>k</em>. The same reasoning applies to <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-120.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-121.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-122.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-123.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-124.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-125.gif"/></noscript></span></span> denote the number of recombinant bivalents between loci <em>k</em> and <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-126.gif"/></noscript></span></span> in parents <em>P</em> and <em>Q</em>, respectively. Let <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-127.gif"/></noscript></span></span> denote the number of possible genotypes derived from the cross between individuals <em>P</em> and <em>Q</em>. For simplification and without loss of generality, let <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-128.gif"/></noscript></span></span>. For a comprehensive autotetraploid example of the transition probabilities (similar to that presented by <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hackett (2001)</a>) and the indexation used in Equation 6, see Table S3.8, File S3 in Supplementary Information.</p><p id="p-21">Given a ploidy level <em>m</em> and a recombination fraction <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-129.gif"/></noscript></span></span>, the only information required to obtain <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-130.gif"/></noscript></span></span> in Equation (6) is <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-131.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-132.gif"/></noscript></span></span>. Since the genotypes in <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-133.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-134.gif"/></noscript></span></span> are arranged according to the lexicographical order of their superscripts, it is possible to obtain <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-135.gif"/></noscript></span></span> for any given pair <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-136.gif"/></noscript></span></span> using the algorithm presented in Supplementary Information, File S2. Although the number of possible transitions between positions <em>k</em> and <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-137.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-138.gif"/></noscript></span></span>, which can be a very large number even for modest ploidy levels, it is possible to obtain the transition between any specific genotypes in <em>j</em> and <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-139.gif"/></noscript></span></span> without computing the entirety of the transition space.</p><p id="p-22">The initial state distribution is the probability of observing a specific genotype. Given the assumption that there is no preferential pairing during the formation of bivalents, a uniform probability density function can be employed as the initial state probability function<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-140.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>To this point, both transition and initial state distributions consider different allelic variants for all <em>m</em> homologs in both parents. This scenario can only be achieved when using fully informative markers. In reality, autopolyploid species may have the same allelic variant in some homologs. Besides, even if a particular locus have different allelic form in all homologs, modern genotyping platforms are usually capable of detecting polymorphisms at the nucleotide level (SNPs), which are essentially biallelic. Due to this lack of identity between the observed data and the full transition space, we make use of the emission function, which is defined as the probability of observing a molecular phenotype given a genotype <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-141.gif"/></noscript></span></span>.</p><p id="p-23">The detection of the allelic variants in modern genotyping platforms is based on the abundance of different alternative nucleotides. In the autopolyploid setting, this can be translated as the <em>dosage</em> of a SNP at a specific locus. The dosage of a SNP can be estimated using the ratio between the abundance of its two allelic forms. Several methods were proposed to perform this task including (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Voorrips <em>et al.</em> 2011</a>), (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Serang <em>et al.</em> 2012</a>) and (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bargary <em>et al.</em> 2014</a>). Here we introduce a biallelic derivation of the emission probability distribution. Although the function presented here use biallelic information, other distributions can be derived for partial informative multiallelic marker systems following the same reasoning.</p><p id="p-24">Let <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-142.gif"/></noscript></span></span> denote the <em>observed dosage</em> of one allelic form in locus <em>k</em> for parents <em>P</em> and <em>Q</em>, respectively. The choice of the allelic form denoted by <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-143.gif"/></noscript></span></span> is arbitrary, as long as the same allelic form is used in <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-144.gif"/></noscript></span></span>. The dosage observed in parent <em>P</em> can be originated from alleles present in <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-145.gif"/></noscript></span></span> of the <em>m</em> homologs. Let <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-146.gif"/></noscript></span></span> denote a set of size <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-147.gif"/></noscript></span></span> containing all possible subsets in <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-148.gif"/></noscript></span></span> that originate the observed dosage <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-149.gif"/></noscript></span></span>. The operator <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-150.gif"/></noscript></span></span> is the cardinality of a set. The same reasoning applies for <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-151.gif"/></noscript></span></span>. For instance, in an autotetraploid, if <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-152.gif"/></noscript></span></span>, the three doses present in locus <em>k</em> can be derived from four distinct subsets <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-153.gif"/></noscript></span></span>. Given two particular subsets <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-154.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-155.gif"/></noscript></span></span> in <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-156.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-157.gif"/></noscript></span></span>, each one of the <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-158.gif"/></noscript></span></span> genotypic states in the full transition space can be associated to a dosage. The observed dosage <em>O</em> associated to the <em>j</em>-th state is obtained by counting the number of alleles present in the intersection between the parental allelic set <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-159.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-160.gif"/></noscript></span></span>. Thus, the emission function can be defined as<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-161.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-162.gif"/></noscript></span></span> and <em>ϵ</em> denotes the global genotype error rate. In addition to the point estimate of the dosage, the genotyping calling methods cited above also provide the probability distribution of the dosages for a particular marker for all individuals of the biparental population. If this information is available, a more general emission function can be derived. Instead of modeling a global error rate <em>ϵ</em>, we use the prior information provided by the genotyping calling procedure. Let <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-163.gif"/></noscript></span></span> denote the probability distribution vector associated to the dosages <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-164.gif"/></noscript></span></span> at position <em>k</em> for a particular individual in the biparental population. For example, <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-165.gif"/></noscript></span></span> denotes a tetraploid individual with probabilities <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-166.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-167.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-168.gif"/></noscript></span></span> of having one, two and three doses, respectively, and zero for the remaining ones. Then, the emission probability function can be written as<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-169.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>In this case, the observation <em>O</em> can be any dosage from 0 to <em>m</em> and the information about the genotypes will be contained in the probability distribution of the dosages <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-170.gif"/></noscript></span></span>. Thus, the probability of observing any dosage given a genotype <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-171.gif"/></noscript></span></span> associated to a particular dosage <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-172.gif"/></noscript></span></span> can be obtained by simply assessing the corresponding value in the probability distribution provided by the genotype calling procedure. Notice that Equation 8 can be reduced to Equation 9 using the appropriate <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-173.gif"/></noscript></span></span>. For example, in autotetraploids, when the observed dosage for locus <em>k</em> is one, <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-174.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-175.gif"/></noscript></span></span>. Moreover, for missing values, it is possible to use the probability distribution of the genotypic classes under polysomic segregation, as presented by <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Serang <em>et al.</em> (2012)</a>.</p></div><div id="sec-8" class="subsection"><h3>Multipoint likelihood and the estimation of recombination fraction</h3><p>Suppose there are <em>z</em> markers in a homology group in a known order represented by <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-176.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-177.gif"/></noscript></span></span> denote the recombination fraction vector between all marker intervals in this sequence. Also, assume linkage phase configurations in parents <em>P</em> and <em>Q</em> denoted respectively by <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-178.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-179.gif"/></noscript></span></span>. The sequence of observations for the <em>z</em> markers is denoted by <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-180.gif"/></noscript></span></span> and its underlying probability distributions are denoted by <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-181.gif"/></noscript></span></span>. The likelihood of <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-182.gif"/></noscript></span></span> can be obtained using Equations (6), (7) and (9) following the classical <em>forward procedure</em> (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Rabiner 1989</a>). Let <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-183.gif"/></noscript></span></span> denote the probability of the partial observation sequence <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-184.gif"/></noscript></span></span> and genotype <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-185.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-186.gif"/></noscript></span></span> given the sequence of recombination fractions r, the linkage phase configurations <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-187.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-188.gif"/></noscript></span></span> and the probability distributions for the sequence of observations Π. The forward procedure follows the steps below:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-26">Initialization:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-189.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span></p></li><li id="list-item-2"><p id="p-27">Induction:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-190.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span></p><p id="p-28">where <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-191.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-192.gif"/></noscript></span></span></p></li><li id="list-item-3"><p id="p-29">Termination:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-193.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span></p></li></ol><p>Then, the likelihood of the model is defined as<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-194.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>where <em>n</em> is the number of individuals in the full-sib population, <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-195.gif"/></noscript></span></span> is the sequence of marker observations for individual <em>i</em> and <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-196.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-197.gif"/></noscript></span></span> matrix where the <em>k</em>-th column denotes the probability distributions associated to the marker <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-198.gif"/></noscript></span></span>, individual <em>i</em>. The multipoint maximum likelihood estimate of r can be obtained using the <em>forward-backward</em> procedure coupled with the EM algorithm (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Rabiner 1989</a>). For the backward procedure, consider the variable <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-199.gif"/></noscript></span></span> as the probability of the partial observation sequence from <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-200.gif"/></noscript></span></span> to <em>z</em>, given the genotype <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-201.gif"/></noscript></span></span>, the recombination fraction vector r, the linkage phase configurations <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-202.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-203.gif"/></noscript></span></span> and the probability distributions for the sequence of observations Π. The solution to <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-204.gif"/></noscript></span></span> was also described by (<a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Rabiner 1989</a>) as follows:</p><ol class="list-ord " id="list-2"><li id="list-item-4"><p id="p-30">Initialization:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-205.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span></p></li><li id="list-item-5"><p id="p-31">Induction:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-206.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span></p></li></ol><p>where <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-207.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-208.gif"/></noscript></span></span></p><p id="p-32">To estimate the recombination fraction for all intervals in the marker sequence we need to define <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-209.gif"/></noscript></span></span> as the probability of state <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-210.gif"/></noscript></span></span> at position <em>k</em> and state <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-211.gif"/></noscript></span></span> at position <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-212.gif"/></noscript></span></span> given the sequence of observations <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-213.gif"/></noscript></span></span> and their underlying probability distributions Π, the recombination fraction vector r and the linkage phase configurations <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-214.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-215.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-216.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>The recombination frequency <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-217.gif"/></noscript></span></span> can be estimated through an iterative process using<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-218.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span>where <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-219.gif"/></noscript></span></span> is calculated for individual <em>i</em>, <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-220.gif"/></noscript></span></span> is the proportion of recombinations between markers <em>k</em> and <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-221.gif"/></noscript></span></span> for individuals with genotypes <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-222.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-223.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-224.gif"/></noscript></span></span> is the vector of recombination fractions in the iteration <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-225.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-226.gif"/></noscript></span></span> is the updated recombination fraction vector (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Broman and Sen 2009</a>).</p></div><div id="sec-9" class="subsection"><h3>Estimation of linkage phase</h3><p id="p-33">Let the Cartesian product <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-227.gif"/></noscript></span></span> denotes a set containing all possible linkage phase configurations in parent <em>P</em>. Also, let <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-228.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-229.gif"/></noscript></span></span>, denote a set containing all possible linkage phase configurations in both parents. The probability of the linkage phase configurations can be obtained using Bayes’ rule<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-230.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span>where O is an array containing the observation for <em>z</em> markers in <em>n</em> individuals, and Π is the underlying probability distribution for all marker observations. Since the prior probability <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-231.gif"/></noscript></span></span> can be assumed to be uniform, the posterior probability is proportional to the likelihood of the model, which can be used to select the best linkage phase configuration. Depending on the dosage and number of markers, some of these configurations are equivalent and will result in the same likelihood. The search space for the best linkage phase configuration can be unwieldy depending on the ploidy level, dosage and number of markers. Also, the transition space on the HMM gets larger as the ploidy level increases. To circumvent these problems, we propose a very efficient two-point procedure to reduce the search space for linkage phases. With the estimates in hands, it is possible to compare two-point likelihoods of alternative linkage phase configurations, eliminating those that do not meet a given threshold. The remaining configurations will be evaluated using the multipoint approach. The adequate threshold level will be discussed in the Simulations section.</p></div><div id="sec-10" class="subsection"><h3>Two-point algorithm for high-level autopolyploids</h3><p id="p-34">When the linkage analysis is conducted only between two markers (two-point analysis), the information contained in these markers does not propagate into the rest of the chain. Thus, based on the dosage and linkage phase configuration of the markers involved in the analysis, the <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-232.gif"/></noscript></span></span> genotypic states present in the full transition space can be collapsed into a small number of states, and a straightforward likelihood function can be derived. It is worthwhile to mention that the estimates obtained using the two-point procedure are the same as those obtained using the multipoint algorithm for two markers. However, the two-point computation is extremely faster.</p><p id="p-35">Consider a biallelic marker in an autopolyploid biparental cross with ploidy <em>m</em>. The number of possible genotypic states in the progeny for a given locus at position <em>k</em> is <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-233.gif"/></noscript></span></span>, where the operator <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-234.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-235.gif"/></noscript></span></span> denotes module. For example, in an autohexaploid biparental cross, if the dosage of the marker at position <em>k</em> in parent <em>P</em> is two (<span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-236.gif"/></noscript></span></span>) and in parent <em>Q</em> is three (<span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-237.gif"/></noscript></span></span>), the number of possible genotypic classes expected in the progeny is six. Depending on the linkage phase configuration, each of the <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-238.gif"/></noscript></span></span> genotypic states in the full transition space corresponds to one of these expected genotypic classes, as presented in the emission function (Equations 8 and 9). Thus, in the previous example, all the <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-239.gif"/></noscript></span></span> states could be collapsed into six different classes. To perform this reduction of dimensionality, let <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-240.gif"/></noscript></span></span> denote one of the possible genotypes based on the observed dosage of one individual in the progeny of an autopolyploid biparental cross for position <em>k</em> with ploidy <em>m</em>. The joint probability of <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-241.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-242.gif"/></noscript></span></span>, for a given genotypic configuration at positions <em>k</em> and <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-243.gif"/></noscript></span></span> can be written as<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-244.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>where <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-245.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-246.gif"/></noscript></span></span> was defined in Equation 8; the same applies to <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-247.gif"/></noscript></span></span>. Since in a two-point analysis the probability distribution of the genotypic states in locus <em>k</em> can be assumed to be uniform, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-248.gif"/></noscript></span></span>, Equation (19) can be rewritten as a sum of weighted terms from Equation (6)<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-249.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span>where<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-250.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-251.gif"/></noscript></span></span> is 1 if <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-252.gif"/></noscript></span></span> corresponds to <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-253.gif"/></noscript></span></span> according to the procedure described in Supplementary Information, File S2, and zero otherwise. Equation 20 can be expressed in matrix form as<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-254.gif"/></noscript></span><span class="disp-formula-label">(21)</span></span>where <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-255.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-256.gif"/></noscript></span></span> matrix. Yet, in a two-point analysis with biallelic markers, the linkage phase configuration can be summarized in an ordered pair <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-257.gif"/></noscript></span></span> indicating the number of homologs that share allelic variants for loci <em>k</em> and <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-258.gif"/></noscript></span></span> in parents <em>P</em> and <em>Q</em>, respectively. For a given pair <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-259.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-260.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-261.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-262.gif"/></noscript></span></span> denote the set of homologs inherited by parent <em>P</em> in positions <em>k</em> and <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-263.gif"/></noscript></span></span>, which can be assessed using the superscripts in <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-264.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-265.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-266.gif"/></noscript></span></span> indicates the cardinality of the set. Notice that <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-267.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-268.gif"/></noscript></span></span> can assume several linkage phase configurations resulting in the same <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-269.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-270.gif"/></noscript></span></span> denote a set containing all possible pairs <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-271.gif"/></noscript></span></span> for a given pair <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-272.gif"/></noscript></span></span>. In this set, there are <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-273.gif"/></noscript></span></span> partitions, each one corresponding to a different <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-274.gif"/></noscript></span></span>. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows an example of <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-275.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-276.gif"/></noscript></span></span> in an autotetraploid homology group. The size of the set is 36, and it can be subdivided into three partitions where <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-277.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-278.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-279.gif"/></noscript></span></span>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of  for an autotetraploid homology group with observed dosages  and  homologs sharing alleles. In this case,  denotes a set of size six, containing all possible subsets of size two in . The same reasoning applies to . The horizontal bars represent homologs forming a homology group and the dots represent allelic variations of a biallelic marker. The number below each homology group represents the number of homologs that share allelic variants (). This defines three partitions: ,  and . Notice that, from a homology group within a specific partition, it is possible to obtain the same linkage phase configuration observed in another homology group within that partition by permuting the its homologs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1714010710" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-280&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for an autotetraploid homology group with observed dosages &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-281&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-282&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; homologs sharing alleles. In this case, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-283&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; denotes a set of size six, containing all possible subsets of size two in &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-284&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The same reasoning applies to &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-285&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The horizontal bars represent homologs forming a homology group and the dots represent allelic variations of a biallelic marker. The number below each homology group represents the number of homologs that share allelic variants (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-286&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). This defines three partitions: &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-287&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-288&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-289&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Notice that, from a homology group within a specific partition, it is possible to obtain the same linkage phase configuration observed in another homology group within that partition by permuting the its homologs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3297/F3.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3297/F3.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494349" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Example of <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-280.gif"/></noscript></span></span> for an autotetraploid homology group with observed dosages <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-281.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-282.gif"/></noscript></span></span> homologs sharing alleles. In this case, <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-283.gif"/></noscript></span></span> denotes a set of size six, containing all possible subsets of size two in <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-284.gif"/></noscript></span></span>. The same reasoning applies to <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-285.gif"/></noscript></span></span>. The horizontal bars represent homologs forming a homology group and the dots represent allelic variations of a biallelic marker. The number below each homology group represents the number of homologs that share allelic variants (<span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-286.gif"/></noscript></span></span>). This defines three partitions: <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-287.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-288.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F3/embed/mml-math-289.gif"/></noscript></span></span>. Notice that, from a homology group within a specific partition, it is possible to obtain the same linkage phase configuration observed in another homology group within that partition by permuting the its homologs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">In a two-point context, the likelihood function derived from any of the configurations belonging to the same partition (same <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-290.gif"/></noscript></span></span>) will be the same. Thus, any of them can be used to obtain the likelihood function for a given <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-291.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-292.gif"/></noscript></span></span> denote one of the possible pairs <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-293.gif"/></noscript></span></span> that correspond to <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-294.gif"/></noscript></span></span>. The same reasoning applies to parent <em>Q</em>. Without loss of generality, the two-point likelihood function of biallelic observed molecular phenotypes for markers <em>k</em> and <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-295.gif"/></noscript></span></span> given <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-296.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-296.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-297.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-297.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-298.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-298.gif"/></noscript></span><span class="disp-formula-label">(22)</span></span>where <em>n</em> is the number of individuals and <em>T</em> denotes transposition of a vector. In Equation (22), <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-299.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-299.gif"/></noscript></span></span> can be estimated using iterative procedures such as EM or Newton-Raphson. As in Equation (18), it is possible to list all linkage phase configurations and evaluate them based on their likelihood. Here we use the LOD Score (base-10 logarithm of likelihood ratios) in relation to the highest likelihood. Thus, models with high likelihoods will yield LOD Scores close to zero. We also use the LOD Score to assess the evidence for linkage between the two markers using the ratio between the model under <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-300.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-300.gif"/></noscript></span></span> and under the null hypothesis of no linkage <span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-301.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-301.gif"/></noscript></span></span>, given a linkage phase configuration.</p><p id="p-38">As previously shown, it is possible to enumerate all linkage phase configurations for parent <em>P</em> using the Cartesian product <span class="inline-formula" id="inline-formula-275"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-302.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-302.gif"/></noscript></span></span>. To reduce this Cartesian space based on two-point analysis, we add a restriction where all pairs <span class="inline-formula" id="inline-formula-276"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-303.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-303.gif"/></noscript></span></span> in a sequence of configurations <span class="inline-formula" id="inline-formula-277"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-304.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-304.gif"/></noscript></span></span> must be contained in <span class="inline-formula" id="inline-formula-278"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-305.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-305.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-279"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-306.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-306.gif"/></noscript></span></span> is a subset of all partitions in <span class="inline-formula" id="inline-formula-280"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-307.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-307.gif"/></noscript></span></span> in which the associated LOD Score is smaller than <em>η</em>. Thus, a reduced subset of linkage phases in parent <em>P</em> based on two-point analysis can be obtained using<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-308.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-308.gif"/></noscript></span><span class="disp-formula-label">(23)</span></span>It is important to note that it is not necessary to represent the whole Cartesian space <span class="inline-formula" id="inline-formula-281"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-309.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-309.gif"/></noscript></span></span> to restrict the linkage phase configurations to the condition <span class="inline-formula" id="inline-formula-282"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-310.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-310.gif"/></noscript></span></span>. This procedure can be done through the sequential addition of markers from <span class="inline-formula" id="inline-formula-283"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-311.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-284"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-312.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-312.gif"/></noscript></span></span>. For each marker <span class="inline-formula" id="inline-formula-285"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-313.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-313.gif"/></noscript></span></span> added to the end of the chain, the ordered pair <span class="inline-formula" id="inline-formula-286"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-314.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-314.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-287"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-315.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-315.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-288"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-316.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-316.gif"/></noscript></span></span>, is evaluated and only linkage phase configurations that meet the condition <span class="inline-formula" id="inline-formula-289"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-317.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-317.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-290"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-318.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-318.gif"/></noscript></span></span> are considered.</p><p id="p-39">Some of the configurations selected using the previous procedure can be equivalent once they are products of a permutation of the same set of homologs. In order to remove this redundancy, let each one of the selected configurations be represented as a binary matrix of dimensions <span class="inline-formula" id="inline-formula-291"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-319.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-319.gif"/></noscript></span></span> such as<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-320.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-320.gif"/></noscript></span><span class="disp-formula-label">(24)</span></span>where <span class="inline-formula" id="inline-formula-292"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-321.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-321.gif"/></noscript></span></span>, U is the number of selected linkage phase configurations, and <span class="inline-formula" id="inline-formula-293"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-322.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-322.gif"/></noscript></span></span> indicates that <span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-323.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-323.gif"/></noscript></span></span> was the last marker inserted in the chain. The rows of matrix <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-324.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-324.gif"/></noscript></span></span> represent the homologs for the <em>u</em>-th linkage phase configuration with the insertion of the <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-325.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-325.gif"/></noscript></span></span>-th marker at the end of chain; 1 denotes the presence of an allelic variation, and 0 denotes its absence. If a matrix <span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-326.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-326.gif"/></noscript></span></span> could be obtained from a matrix <span class="inline-formula" id="inline-formula-298"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-327.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-327.gif"/></noscript></span></span> just by permuting the rows (permuting the order of the homologs), these two linkage configurations yield the same likelihood. Thus, one of the configurations should be excluded from consideration. The same reasoning applies to parent <em>Q</em>. This procedure can be done recursively until all redundancy is eliminated. The reduced linkage phase configurations search space considering both parents is obtained using <span class="inline-formula" id="inline-formula-299"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-328.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-328.gif"/></noscript></span></span>, such as <span class="inline-formula" id="inline-formula-300"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-329.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-329.gif"/></noscript></span></span>, combined with the redundancy elimination for homology groups. This sequential procedure results in a set of linkage phase configurations containing markers up to <span class="inline-formula" id="inline-formula-301"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-330.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-330.gif"/></noscript></span></span>, which are evaluated using the HMM likelihood. A LOD Score threshold in relation to the most likely configuration is assumed to determine which configurations should be taken into consideration in the next round of marker inclusion (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Additionally, it is possible to limit the two-point search space reduction to a window of SNPs in the terminal part of the chain to speed up the phasing process. Finally, with all markers inserted, the multipoint likelihood of the whole map is used to find the best configuration among the remaining ones, and the recombination fractions are reestimated. To demonstrate the mechanics of the two-point analysis coupled with the multipoint procedure, a simple example is presented in Supplementary Information, File S3.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of linkage phase configuration estimation using sequential search space reduction and HMM evaluation. Only one parent is presented. The two-point search reduction is composed of two parts: the first one evaluates the LOD Scores obtained through pairwise recombination fraction likelihoods. The second detects equivalent configurations by performing all possible permutations of the homologs. The remaining configurations are evaluated using the HMM-based likelihood. In the first step, linkage phase configurations of  and  are evaluated using the two-point analysis. Color shades indicate different linkage phase configurations provided by the two-point analysis. In this example, there are two possible linkage phases represented by two shades of red. In the second step, we evaluate the linkage phases between markers  and , and  and . Configurations with LOD Scores smaller than &#x3B7; are maintained to be evaluated by HMM. There are two possible linkage phases given a certain &#x3B7;, represented by two shades of blue. These two configurations are combined with the configurations from the previous step, resulting in four configurations evaluated using HMM likelihood. Given a likelihood threshold, only configurations 1 and 4 are eligible for the next step. The same reasoning applies for the remaining markers. A final linkage phase configuration is obtained after inserting the last marker and choosing the one that yields the highest HMM-based likelihood." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1714010710" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Example of linkage phase configuration estimation using sequential search space reduction and HMM evaluation. Only one parent is presented. The two-point search reduction is composed of two parts: the first one evaluates the LOD Scores obtained through pairwise recombination fraction likelihoods. The second detects equivalent configurations by performing all possible permutations of the homologs. The remaining configurations are evaluated using the HMM-based likelihood. In the first step, linkage phase configurations of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-331&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-332&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are evaluated using the two-point analysis. Color shades indicate different linkage phase configurations provided by the two-point analysis. In this example, there are two possible linkage phases represented by two shades of red. In the second step, we evaluate the linkage phases between markers &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-333&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-334&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-335&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-336&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Configurations with LOD Scores smaller than &lt;em&gt;&#x3B7;&lt;/em&gt; are maintained to be evaluated by HMM. There are two possible linkage phases given a certain &lt;em&gt;&#x3B7;&lt;/em&gt;, represented by two shades of blue. These two configurations are combined with the configurations from the previous step, resulting in four configurations evaluated using HMM likelihood. Given a likelihood threshold, only configurations 1 and 4 are eligible for the next step. The same reasoning applies for the remaining markers. A final linkage phase configuration is obtained after inserting the last marker and choosing the one that yields the highest HMM-based likelihood.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3297/F4.medium.gif" width="440" height="116"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3297/F4.medium.gif" width="440" height="116"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494509" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">Example of linkage phase configuration estimation using sequential search space reduction and HMM evaluation. Only one parent is presented. The two-point search reduction is composed of two parts: the first one evaluates the LOD Scores obtained through pairwise recombination fraction likelihoods. The second detects equivalent configurations by performing all possible permutations of the homologs. The remaining configurations are evaluated using the HMM-based likelihood. In the first step, linkage phase configurations of <span class="inline-formula" id="inline-formula-302"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-331.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-331.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-303"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-332.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-332.gif"/></noscript></span></span> are evaluated using the two-point analysis. Color shades indicate different linkage phase configurations provided by the two-point analysis. In this example, there are two possible linkage phases represented by two shades of red. In the second step, we evaluate the linkage phases between markers <span class="inline-formula" id="inline-formula-304"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-333.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-333.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-305"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-334.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-334.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-306"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-335.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-335.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-307"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-336.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/F4/embed/mml-math-336.gif"/></noscript></span></span>. Configurations with LOD Scores smaller than <em>η</em> are maintained to be evaluated by HMM. There are two possible linkage phases given a certain <em>η</em>, represented by two shades of blue. These two configurations are combined with the configurations from the previous step, resulting in four configurations evaluated using HMM likelihood. Given a likelihood threshold, only configurations 1 and 4 are eligible for the next step. The same reasoning applies for the remaining markers. A final linkage phase configuration is obtained after inserting the last marker and choosing the one that yields the highest HMM-based likelihood.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-41">All the methods and procedures described here are available in the R package <em>MAPpoly</em>, which is freely available from <a href="https://github.com/mmollina/mappoly">https://github.com/mmollina/mappoly</a>. R scripts to perform the simulations and the potato map construction presented in this article can be accessed at <a href="https://github.com/mmollina/Autopolyploid_Linkage">https://github.com/mmollina/Autopolyploid_Linkage</a>. The tetraploid potato data set is available through the Solanaceae Coordinated Agricultural Project at <a href="http://solcap.msu.edu/potato_infinium.shtml">http://solcap.msu.edu/potato_infinium.shtml</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8218325">https://doi.org/10.25387/g3.8218325</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Simulations</h3><div id="sec-14" class="subsection"><h4>Simulation 1 - local performance under random bivalent pairing:</h4><p id="p-42">the aim of this simulation study was to evaluate the local performance of the algorithm considering three ploidy levels (<span class="inline-formula" id="inline-formula-308"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-337.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-337.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-309"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-338.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-338.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-310"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-339.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-339.gif"/></noscript></span></span>) under the mapping model assumptions (<em>i.e.</em>, random pairing and bivalent formation). To be in accordance with molecular data that have been made available through sequence technologies, we simulated bi-allelic markers that can be observed in terms of dosage in parents and progeny. Three different linkage phase scenarios were simulated. In scenario A, for each marker, one of the allelic variants was assigned to the first homolog in the homology group and the remaining variants of the same type were assigned to the subsequent homologs. In scenario B, the allelic variant was randomly assigned to one of the first <span class="inline-formula" id="inline-formula-311"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-340.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-340.gif"/></noscript></span></span> homolog and the remaining were assigned to the subsequent homologs. In scenario C, the allelic variants were randomly assigned to the <em>m</em> homologs. Thus, it is expected an increasing difficulty to detect recombination events from scenario A, where the allelic variants were concentrated in the same homologs, to scenario C, where they were randomly distributed.</p><p id="p-43">For each combination of ploidy level and linkage phase scenario, we simulated five different parental haplotypes. In total, 45 parental configurations were considered (<span class="inline-formula" id="inline-formula-312"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-341.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-341.gif"/></noscript></span></span>, Supplementary Information, Figure S4). For autotetraploid and autohexaploid configurations, we simulated 1000 full-sib populations. For autooctaploids, this number was reduced to 200 due to the high demand of computer processing required to reconstruct such maps. Each population was comprised of 200 individuals with one linkage group containing 10 markers positioned at a fixed distance of 1 centimorgan (cM) between them. For each combination, the percentage of correctly estimated linkage phase configuration in each parent was recorded. Also, for the cases where the linkage phases were correctly estimated, we calculated the average Euclidean distance between the distances of the estimated and simulated maps using <span class="inline-formula" id="inline-formula-313"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-342.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-342.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-314"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-343.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-343.gif"/></noscript></span></span> is the vector of distances for a estimated map, d is the vector of distances for the simulated map, <em>z</em> is the number of markers and <em>T</em> indicates vector transposition. For example, a value of 1 cM indicates that the maps differ 1 cM in average from each other (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Mollinari <em>et al.</em> 2009</a>). We used the sequential two-point procedure to reduce the search space assuming that linkage phase configurations with associated <span class="inline-formula" id="inline-formula-315"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-344.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-344.gif"/></noscript></span></span> should be investigated using HMM multipoint strategies (<span class="inline-formula" id="inline-formula-316"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-345.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-345.gif"/></noscript></span></span>). For the remaining configurations evaluated using HMM, we kept those with <span class="inline-formula" id="inline-formula-317"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-346.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-346.gif"/></noscript></span></span> to be evaluated in the next round of marker insertion.</p></div><div id="sec-15" class="subsection"><h4>Simulation 2 - chromosome-wide performance under preferential pairing and multivalent formation:</h4><p id="p-44">In this simulation study, we evaluated the performance of the algorithm in dense maps, allowing for multivalent formation and preferential pairing. We used Scenario C from the previous study as a template to simulate five tetraploid and five hexaploid parental haplotypic configurations, each one comprising 200 equally spaced markers with a final length of 100.0 cM (Supplementary Information, Figure S5). For each parental configuration, we simulated 200 full-sib populations of 200 offspring considering a combination of three levels of preferential pairing (0.00, 0.25 and 0.50) and three levels of cross-like quadrivalent formation proportion (0.00, 0.25 and 0.50) with the position of the pairing partner switch varying across simulations. No hexavalents were simulated in this study. For autohexaploids, the multivalent configurations were always composed by a cross-like quadrivalent plus a bivalent. The centromere was positioned at 20.0 cM from the beginning of the chromosome (subtelocentric centromere with arms ratio 1:4) to study the effect of the double reduction which is more pronounced at the end of both chromosome arms. All simulations were conducted using the software PedigreeSim (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Voorrips and Maliepaard 2012</a>). In addition to the statistics recorded in Simulation 1, we computed the rate of double reduction observed in each marker for all constructed maps using the “founderalleles” file provided by PedigreeSim. We also evaluate two values for the LOD Score threshold associated to the two-point analysis (<span class="inline-formula" id="inline-formula-318"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-347.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-347.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-319"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-348.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-348.gif"/></noscript></span></span>). We used a multipoint LOD Score threshold of 10.0 and also limited the two-point search to a 50 SNP window in the terminal part of the map. Markers presenting higher doses than the sum of the doses in both parents, originated from double reduced gametes were filtered-out and assumed as missing data.</p></div></div><div id="sec-16" class="subsection"><h3>Simulation results</h3><div id="sec-17" class="subsection"><h4>Simulation 1:</h4><p id="p-45"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the percentage of data sets where the linkage phase configuration was correctly estimated in both parents <em>P</em> and <em>Q</em>. In scenario (A) the method was capable of recovering the correct linkage phase configuration in all situations for all ploidy levels. In scenarios (B) and (C) there was a slight decrease in the ability to correctly estimate the linkage phase configuration, especially for <span class="inline-formula" id="inline-formula-320"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-349.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-349.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-321"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-350.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-350.gif"/></noscript></span></span>. Although in these cases the percentages of correctly estimated linkage phases was lower, they were nevertheless high, varying from 100 to 88.8%. This indicates a very good performance to estimate the linkage phase configurations, even using the two-point procedure to narrow the search space.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494405/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494405/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Percentage of data sets where linkage phase configuration was correctly estimated for five different parental P and Q haplotypes in simulation 1</span><div class="sb-div caption-clear"></div></div></div><p id="p-46"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows the distributions of the average Euclidean distances between the estimated and simulated distance vectors for the correctly estimated linkage phase configuration. In all cases, the majority of the recombination fractions were consistently estimated once the medians of all distributions are very close 0.5 cM, with no practical problems in terms of mapping construction. These results show that, apart from a relatively small percentage of entangled linkage phase configurations, the method successfully performed the phasing and managed to estimate the recombination fraction of 10 markers in all situations evaluated.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of the average Euclidean distances between the estimated and simulated distance vectors considering correctly estimated linkage phase configurations. The order of boxplots is the same as the order of haplotypes in Figure S4. Each column indicates the results for different linkage phase configuration scenarios, namely, A, B and C, and each row indicates a different haplotypic configuration within three ploidy levels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1714010710" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distributions of the average Euclidean distances between the estimated and simulated distance vectors considering correctly estimated linkage phase configurations. The order of boxplots is the same as the order of haplotypes in Figure S4. Each column indicates the results for different linkage phase configuration scenarios, namely, A, B and C, and each row indicates a different haplotypic configuration within three ploidy levels.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3297/F5.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3297/F5.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494373" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-47" class="first-child">Distributions of the average Euclidean distances between the estimated and simulated distance vectors considering correctly estimated linkage phase configurations. The order of boxplots is the same as the order of haplotypes in Figure S4. Each column indicates the results for different linkage phase configuration scenarios, namely, A, B and C, and each row indicates a different haplotypic configuration within three ploidy levels.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Simulation 2:</h4><p id="p-48">The proportion of correctly estimated linkage phase configurations for the dense chromosome-wise map is shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. In general, results for tetraploid maps were superior when compared to results for hexaploid maps. It is also possible to observe a better performance for the threshold level <span class="inline-formula" id="inline-formula-325"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-354.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-354.gif"/></noscript></span></span> in comparison to <span class="inline-formula" id="inline-formula-326"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-355.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-355.gif"/></noscript></span></span>. Similarly to Simulation 1, maps resulting from configurations with no preferential pairing or quadrivalent formation showed a high proportion of correctly estimated linkage phase configurations. Results ranged from 100 to 99% for tetraploid maps and from 100 to 84% for hexaploid maps. Different levels of quadrivalent formation rate had no substantial influence in estimating the correct linkage phase configurations in tetraploids. Within the preferential pairing level 0.0, the percentage of maps with correct linkage phases varied from 100 to 90%. For hexaploids, there was a decrease in this percentage as the quadrivalent formation increases from 0.0 to 0.50, with proportions varying from 100 to 70.5%. Especially for autohexaploids, there was considerable variation between the five simulated configurations. This occurred because the effect of the quadrivalent formation can be more pronounced depending on the level of information contained in a particular configuration. Also, the use of a more stringent two-point threshold <span class="inline-formula" id="inline-formula-327"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-356.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-356.gif"/></noscript></span></span>, improved the performance of the phasing algorithm.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494315/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494315/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Percentage of data sets where linkage phase configuration was correctly estimated for parents <strong><em>P</em></strong> and <strong><em>Q</em></strong> in simulation 2</span><div class="sb-div caption-clear"></div></div></div><p id="p-49">Within the preferential pairing level 0.25, results showed decay of correctly estimated linkage phases, which was more pronounced for hexaploid cases with threshold level <span class="inline-formula" id="inline-formula-332"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-361.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-361.gif"/></noscript></span></span>, reaching a minimum value of 52.5% for parent <em>Q</em> in configuration 1. Again, the use of a higher two-point threshold level, <span class="inline-formula" id="inline-formula-333"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-362.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-362.gif"/></noscript></span></span>, helped to improve this number to 68.5%. For preferential pairing level 0.50, there was a clear distinction between the results in tetraploid and hexaploid cases. In the former, the effect was not as pronounced as it was in the latter, where in several cases, the proportion of correctly estimated linkage phases was close to zero. As expected, the usage of a higher threshold level of <span class="inline-formula" id="inline-formula-334"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-363.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-363.gif"/></noscript></span></span> helped to improve the number of corrected estimated linkage phase configurations. Interestingly, for both cases with preferential pairing (0.25 and 0.50), the formation of quadrivalents had an overall tendency to improve the algorithm’s performance. This improvement was expected because when a quadrivalent is formed, each chromosome involved can exchange segments with two others, providing more information regarding their phase configuration.</p><p id="p-50">Given a correctly estimated linkage phase, the recombination fractions were consistently estimated for all levels of preferential pairing with no quadrivalent formation (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). However, they were overestimated in the presence of quadrivalent formation. This effect was mainly observed at the terminal regions of the chromosome, especially in the long arm, where double reduction is more pronounced. In this case, tetraploid maps were the most affected. This is in agreement with our expectations since in autohexaploid simulations, there was always the formation of a bivalent which was not involved in the double reduction process (although the rates of double reduction were very similar in both ploidy levels, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). In addition to the quadrivalent, the bivalent serves as an extra source of information to access the recombination events. The average Euclidean distances reflect the overestimation of recombination fractions in cases with quadrivalent formation, showing distributions with higher medians and interquartile ranges in tetraploid cases when compared to hexaploids (Supplementary Information, Figure S6). Nevertheless, all the Euclidean distances distributions were located relatively close to zero, with a maximum value of 1.4 cM, indicating that although we observed overestimated recombination fractions toward the terminal ends of the chromosome, they were equally distributed, causing no severe disturbances in the final map. Figure S7, in Supplementary Information, shows an example of the effect of increasing quadrivalent formation rate in autotetraploid and autohexaploid maps. As the markers get further away from the centromere, the recombination fractions become overestimated.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of estimated vs. simulated maps given a correct estimation of linkage phases in simulation 2. Smoothed conditional means of the observed average rate of double reduction is presented along with the simulated chromosome. The centromere was positioned at 20 cM from its beginning (vertical dashed line). Upper panels show the results for tetraploid simulations while lower panels show the results for hexaploid simulations. Three levels of preferential pairing (0.00, 0.25, 0.50) and three levels of quadrivalent formation rate (0.00, 0.25, 0.50) were simulated. The lines superimposed to the scatter plots are smoothed conditional means of the distances using a generalized additive model. Both two-point thresholds were considered since they only affect the phasing procedure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1714010710" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of estimated &lt;em&gt;vs.&lt;/em&gt; simulated maps given a correct estimation of linkage phases in simulation 2. Smoothed conditional means of the observed average rate of double reduction is presented along with the simulated chromosome. The centromere was positioned at 20 cM from its beginning (vertical dashed line). Upper panels show the results for tetraploid simulations while lower panels show the results for hexaploid simulations. Three levels of preferential pairing (0.00, 0.25, 0.50) and three levels of quadrivalent formation rate (0.00, 0.25, 0.50) were simulated. The lines superimposed to the scatter plots are smoothed conditional means of the distances using a generalized additive model. Both two-point thresholds were considered since they only affect the phasing procedure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3297/F6.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/10/3297/F6.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494330" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-51" class="first-child">Comparison of estimated <em>vs.</em> simulated maps given a correct estimation of linkage phases in simulation 2. Smoothed conditional means of the observed average rate of double reduction is presented along with the simulated chromosome. The centromere was positioned at 20 cM from its beginning (vertical dashed line). Upper panels show the results for tetraploid simulations while lower panels show the results for hexaploid simulations. Three levels of preferential pairing (0.00, 0.25, 0.50) and three levels of quadrivalent formation rate (0.00, 0.25, 0.50) were simulated. The lines superimposed to the scatter plots are smoothed conditional means of the distances using a generalized additive model. Both two-point thresholds were considered since they only affect the phasing procedure.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-19" class="subsection"><h3>Analysis of real tetraploid potato SNP data set</h3><p id="p-52">We applied our method to construct a genetic map of the B2721 population which is a cross between tetraploid potato varieties <em>Atlantic</em> and <em>B1829-5</em>. The population comprises 160 offsprings genotyped with the SolCAP Infinium 8303 potato array. The genotype calling was performed using fitTetra R package (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Voorrips <em>et al.</em> 2011</a>). We obtained 4017 SNPs and computed all the pairwise recombination fraction between them for all possible linkage phase configurations. For each pair, we selected the configuration that yields the higher likelihood and applied the Unweighted Pair Group Method with Arithmetic Mean (UPGMA) clustering algorithm to assign markers into 12 linkage groups. Within each linkage group, we ordered the markers using the MDSMap R package (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Preedy and Hackett 2016</a>). We applied the unconstrained multidimensional scaling algorithm (MDS) to the pairwise marker distance obtained using Haldane’s mapping function and <span class="inline-formula" id="inline-formula-335"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-364.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-364.gif"/></noscript></span></span> weighting. We performed two rounds of marker removal by inspecting the nearest neighbor measure scatter plot and also the monotony of the resulting recombination fraction matrix. Given the marker order obtained for each group, we applied our algorithm using <span class="inline-formula" id="inline-formula-336"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-365.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-365.gif"/></noscript></span></span>. For each round of marker inclusion, we limited our phase search to the last 100 markers inserted at the end of the map and eliminated markers that caused map inflation greater than 10 cM. The resulting map consisted of 3348 SNPs (58% simplex and double simplex markers and 42% multiplex) distributed in 12 linkage groups with lengths varying from 165.2 cM to 332.5 cM with no visible gaps between markers (Supplementary Information, Table S8.1). These values seem inflated compared to tetraploid potato maps available in the literature (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sharma <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Massa <em>et al.</em> 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bourke <em>et al.</em> 2015</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rak <em>et al.</em> 2017</a>). The map expansion observed here was mainly caused by two sources of error, namely, local marker misplacement and genotyping errors. Both factors cause the detection of spurious recombination events which are propagated through the HMM causing a global overestimation of recombination fractions.</p><p id="p-53">While obtaining a <em>de novo</em> order based on multiploint likelihood is not feasible in our current implementation, we used the multipoint likelihood as an objective function to compare the map obtained using the genomic order from the <em>Solanum tuberosum</em> genome version 4.03 (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Sharma <em>et al.</em> 2013</a>) and the MDS based order. When using the genomic order, the length of all linkage groups are smaller, and the likelihoods were substantially superior when compared to the <em>de novo</em> MDS-based order (Supplementary Information, Table S8.1 and Figure S8.1). Furthermore, our algorithm estimated the same linkage phase in both cases, indicating the robustness of the phasing method to local marker misplacement. To address the genotyping errors, we used the approaches presented in Equations 8 and 9, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-337"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-366.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-366.gif"/></noscript></span></span> use the probability distribution provided by the mixture proportions of the doses from fitTetra software and <span class="inline-formula" id="inline-formula-338"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-367.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-367.gif"/></noscript></span></span> assuming a global genotyping error; in this case we assumed an <em>ad hoc</em> error rate of 5%. We applied this prior information in both <em>de novo</em> MDS-based and the genome-based orders. The result also can be observed in Table S8.1 and Figure S8.1. Both approaches produced smaller maps when compared to their relative original maps. However, since <span class="inline-formula" id="inline-formula-339"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-368.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-368.gif"/></noscript></span></span> relies on the dosage proportions based in single SNPs, the adjustment of the map was not as flexible as observed in <span class="inline-formula" id="inline-formula-340"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-369.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-369.gif"/></noscript></span></span>, which assumes an equal global error for all markers. Thus the usage of global error allowed genotypes to conform to a global chromosomal structure, rather than restrain the markers in certain genotypic classes proposed by the classification method. Furthermore, the usage of a global error mitigates the effect of the local marker misplacement caused by the MDS algorithm by clustering markers into linkage disequilibrium blocks. This effect can be observed in linkage groups 1, 5, 8, 10, 11, and 12, where the difference between <em>de novo</em> MDS-based and the genome-based order when modeling a global error was less than 10 cM.</p></div><div id="sec-20" class="subsection"><h3>Comparison with polymapR software</h3><p id="p-54">Among the available methods to construct maps in high-dose autopolyploids, namely, pergola (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Grandke <em>et al.</em> 2017</a>), netgwas (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Behrouzi and Wit 2017</a>), and polymapR (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>), only the latter is capable of inferring parental haplotypes and estimating recombination fraction in outcrossing populations. Thus, we limited our comparison to polymapR software. To assess the performance of the methods, we simulated 50 full-sib hexaploid populations with 200 each individuals in five marker density scenarios: 200, 400, 600, 800, and 1000 equally spaced markers in a 100 cM linkage group. Similarly to Simulation 2 described before, we randomly assigned allelic variants, from 0 to 3 doses, to the six homologs. Additionally, in this study, we considered two different dosage proportions scenarios: the first one considers a higher proportion of simplex and double simplex markers, with 40% of the simulated markers being nulliplex, 40% simplex, 10% duplex and 10% triplex in both parents; the second, considers equal proportions for all doses, with 25% for all dosage types, from nulliplex to triplex (Supplementary Information, Figure S9). In total, 500 populations were simulated (5 × 2 × 50) and for each, we obtained the phased map using polymapR and our HMM-based procedure. All simulations were performed using the software PedigreeSim (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Voorrips and Maliepaard 2012</a>) considering no preferential pairing and no quadrivalent formation. For both methods, we recorded the percentage of correctly phased markers, the final map length, and the number of markers inserted in each phased map.</p><p id="p-55">To employ our HMM-based method, we ordered the markers using the unconstrained MDS algorithm (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Preedy and Hackett 2016</a>) weighted by <span class="inline-formula" id="inline-formula-341"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-370.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-370.gif"/></noscript></span></span> with no rounds of marker removal. Given the MDS order, two levels of <em>η</em> were used: 3 and 5; the same levels were used for the multilocus LOD threshold. The phase search was limited to the last 50 markers inserted at the end of the map. To construct maps using polymapR, we first applied the function cluster_SN_markers to perform a grid search from LOD Scores 1 to 20 and chose the lowest one that yields six homologs based on simplex markers in coupling linkage phase. In the next step, we assigned double simplex and duplex markers to the linkage group using the function assign_linkage_group, and the remaining marker types were assigned using the function homolog_lg_assignment. This procedure was performed for both parents assuming LOD Score thresholds of 3 and 5. All remaining pairwise recombination fractions were computed, and the MDS algorithm (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Preedy and Hackett 2016</a>) was used to order the markers. Marker positions were estimated using the projection of the MDS result onto a single dimension principal curve. Finally, a phased map was created using the function create_phased_maplist. We used polymapR version 1.0.19. Differently from the HMM-based method, where the <em>η</em> indicates the LOD threshold from which the multipoint likelihood should be used to chose the best phase configuration, polymapR uses LOD thresholds to make decisions whether a marker should be used in a certain mapping context, such as, clustering homologs or to assign it into assembled linkage groups. Thus, they are not directly comparable.</p><p id="p-56">Table S10.1 in Supplementary Information, shows the results obtained using both methods. Overall, both methods recovered the vast majority of the linkage phases of the markers across all simulations. In the presence of a high number of single-dose markers both methods recovered the correct linkage phase from 99.7 to 100% of the included markers with polymapR positioning in average 94.3% of the markers when using <span class="inline-formula" id="inline-formula-342"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-371.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-371.gif"/></noscript></span></span> and 85.0% when using <span class="inline-formula" id="inline-formula-343"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-372.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-372.gif"/></noscript></span></span> and our HMM-based method positioned in average 100.0% of the markers when using <span class="inline-formula" id="inline-formula-344"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-373.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-373.gif"/></noscript></span></span> and 99.8% when using <span class="inline-formula" id="inline-formula-345"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-374.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-374.gif"/></noscript></span></span>. In cases where the dosages were uniformly assigned, 96.2–100% of the markers were correctly phased when using polymapR and from 99.1 to 100% when using the HMM-based method. In this case, the number of positioned markers varied from 16.9 to 77.5% for <span class="inline-formula" id="inline-formula-346"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-375.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-375.gif"/></noscript></span></span> and from 14.4 to 57.0% for <span class="inline-formula" id="inline-formula-347"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-376.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-376.gif"/></noscript></span></span> for polymapR and from 99.3 to 99.9% for <span class="inline-formula" id="inline-formula-348"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-377.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-377.gif"/></noscript></span></span> and from 97.8 to 99.9% for <span class="inline-formula" id="inline-formula-349"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-378.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-378.gif"/></noscript></span></span> when using our HMM-based method. As pointed out by <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">van Geest <em>et al.</em> (2017)</a>, when using polymapR’s method, the presence of a sufficiently large number of simplex markers uniformly distributed throughout the genome is essential to define each homologs and be able to assign multiple dose markers to the framework map. It is important to mention that using modern genotyping technologies, most data sets have high proportions of single dose markers. While this is generally true, single dose markers can be absent in certain chromosome regions or even along entire homologs due to recent duplication events.</p><p id="p-57">Table S10.2, Supplementary Information, shows the average map length and the associated standard deviation obtained in all simulations. In cases with a high number of single-dose markers, the average map length produced by polymapR ranged from 87.0 to 89.9 cM, while in the case where the dosages where uniformly assigned, map lengths ranged from 76.7 and 80.1 cM. These results confirm the underestimation tendency in the MDS algorithm when using when using <span class="inline-formula" id="inline-formula-350"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-379.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-379.gif"/></noscript></span></span> as weighting function, as observed by <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Preedy and Hackett (2016)</a>. In our HMM-based method, considering the MDS order, the maps lengths were highly overestimated, ranging from 143.0 cM to 548.0 cM. Since we did not introduce errors in our simulation procedures, the observed map inflation is exclusively due to local marker misplacement caused by the MDS algorithm. Nonetheless, even with local marker misplacements, the linkage phase configuration was correctly estimated in the vast majority of the cases. To accommodate the local marker misplacements, we used an <em>ad hoc</em> global error of 5% in the HMM emission function. As observed in the tetraploid potato analysis, this strategy allowed markers in the wrong order, but in the same linkage disequilibrium block, being positioned closely together through the HMM estimation process. The resulting maps lengths were close to the simulated 100 cM (Supplementary Information, Table S10.2 and Figure S10.1). Although in general, our method yielded denser maps, it is worthwhile to mention that, polymapR’s method is substantially faster than ours, notably when our method uses high values for <em>η</em>, in which case the HMM computations play a significant role in the phasing procedures (Supplementary Information, Table S10.3). Nonetheless, it was precisely the multipoint procedure that allowed our method to position more markers when compared to polymapR.</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-58">Although the concept of linkage mapping is relatively simple, the combinatorial properties and increasingly missing information that arise from the multiple sets of chromosomes make the construction of genetic maps in high-level autopolyploids challenging. In this work, we frame and solve two fundamental steps toward the construction of such maps, namely multipoint recombination fraction estimation and linkage phase estimation. We showed that, combined with standard grouping and ordering procedures (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Preedy and Hackett 2016</a>), these maps could be reliably constructed. Our method can be applied to biallelic codominant markers and, due to the flexibility of the HMM framework upon which it was derived, it is extendable to any codominant molecular marker. The HMM used in this work takes into account the linkage phase configuration of the whole linkage group to estimate the recombination fractions between adjacent markers. An efficient two-point approach was also presented to reduce the search space of linkage phase configurations. As a result, our method provides the likelihood of the model, which can be used as an objective function to compare different map configurations, including linkage phases and marker orders. When considering experimental populations, our method is a generalization, for any even ploidy level, of well established genetic linkage mapping methods. For diploid (<span class="inline-formula" id="inline-formula-351"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-380.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-380.gif"/></noscript></span></span>) populations derived from biparental crosses, our method is equivalent to the influential Lander and Green algorithm (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Lander and Green 1987</a>); considering full-sib phase-unknown crosses, it is equivalent to <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> (2002)</a>. For tetraploids (<span class="inline-formula" id="inline-formula-352"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-381.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-381.gif"/></noscript></span></span>) the method is equivalent to <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Leach <em>et al.</em> (2010)</a>, disregarding double reduction.</p><p id="p-59">To assess the statistical power of our method, we conducted two simulation studies. In simulation 1, we demonstrated that our model was capable of correctly estimating the majority of parental linkage phase configurations and recombination fractions in a limited number of markers, even for complex linkage phase configurations and high ploidy levels. Since other methods are based on single-dose markers to assemble homology groups, to the best of our knowledge, this is the only method capable of phasing markers in high-dose autopolyploid genomes in small regions. These well-assembled regions could function as multiallelic codominant markers which propagate their information through the HMM to the rest of the chain, improving the quality of the final map. In simulation 2, we analyzed a sequence of 200 markers in combinations of different levels of preferential pairing and rates of quadrivalent formation. In this situation, quadrivalent formation rate had a marginal effect on the phasing procedure, whereas preferential pairing reduced its performance, especially for autohexaploids. The usage of a higher two-point threshold (<em>η</em>) improved the linkage phase estimation in all cases. This fact indicates that the haplotype phasing is more accurate when HMM-based likelihood is used as objective function to evaluate linkage phases. We also observed that quadrivalent formation yield overestimated recombination fractions between adjacent markers located further away from the centromere. Interestingly, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bourke <em>et al.</em> (2015</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2016)</a> found that higher quadrivalent rates had little effect on the recombination fractions. Most likely, the source of this discrepancy was the different usage of information in both approaches. While in both studies (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Bourke <em>et al.</em> 2015</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2016</a>) used exclusively two-point recombination fraction estimates, here the information of the markers propagates along the linkage group and the recombination fraction between two adjacent markers depends on the behavior of the whole chromosome. Moreover, the overestimation of the recombination fraction was expected since our model disregards double reduction and, consequently, was not able to correctly assess the number of crossing over events when this phenomenon was present.</p><p id="p-60">Although our model is robust enough to cope with certain levels of preferential pairing and tetravalent rate formation, it is possible to include both phenomena in specific points of its derivation. Preferential paring can be included in Equation 4 by not considering <span class="inline-formula" id="inline-formula-353"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-382.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-382.gif"/></noscript></span></span> as uniformly distributed. Double reduction can be included in the definition of the genotypic states in the full transition space (Equation 5). These two phenomena add extra layers of complexity to the genetic mapping of polyploid organisms with high ploidy levels and should be addressed in future studies.</p><p id="p-61">We also build a tetraploid map using the ideas presented in this study coupled with standard grouping and ordering procedures. While the choice of the genotype error rate depends on specific characteristics of the data set, we demonstrate that it is possible to use prior information on the HMM framework, including the probability distribution of the marker dosages for each SNP and a global error to avoid map inflation caused by local marker misplacement and genotyping errors. Finally, we compared our HMM-based method to the polymapR two-point based method and, as already pointed out by (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">van Geest <em>et al.</em> 2017</a>), we concluded that with a number sufficiently large of single-dose markers uniformly distributed across the homologs, both methods performed well. However, when those markers are absent in a specific homologs or chromosome region, our method was able to build denser maps when compared to polymapR. Moreover, some autopolyploids with ploidy level higher than six, such as sugarcane (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aitken <em>et al.</em> 2007</a>) and garden dahlias (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schie <em>et al.</em> 2014</a>), could benefit only from our method, since polymapR is limited only to tetraploid and hexaploid species. The difficulty in correctly estimating linkage phase configurations where multi-dose allelic variants are spread randomly in all homologous chromosomes lies in two significant aspects of the experiments studied here: (<em>i</em>) the outbred nature of the experimental crosses and (<em>ii</em>) the incomplete information of the markers based on dosage (<em>i.e.</em>, by not being multiallelic). In experimental population derived from inbred lines, the origin of the haplotypes can be easily inferred from the genetic design. However, obtaining pure inbred lines in high-level autopolyploids has been proven to be impractical due to the high number of crosses and generations necessary to achieve homozygous genotypes and to the inbreeding depression which some species undergo (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gallais 2003</a>). In our method, the linkage phase configuration is obtained by comparing the likelihood of a set of models with different linkage phase configurations (Equation 18). The capability of estimating the correct configuration is directly related to the information contained in the marker data. Some of these limitations are overcome through the use of HMMs which take into account the information of the linkage group as a whole.</p><p id="p-62">HMMs provide an excellent avenue to assemble genetic maps in complex scenarios, but they are remarkably computational demanding and, in some cases, unfeasible to use. Apart from parallel computing, which can greatly speed up the estimation process and is ubiquitous nowadays, the usage of two-point approaches is a viable option to reduce the dimension of the original problem efficiently. The dimension reduction is achieved by collapsing genotypic states in the full transition space according to the marker information. However, in several cases, the two-point based method can result in low statistical power which is related to the amount of information contained in markers in certain combinations of allelic dosage and linkage phase configurations. This lack of information is exacerbated as markers get distant from each other. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows nine possible configurations of pairs of markers in one autohexaploid parent. Considering one of the parents non-informative, we computed the Fisher’s information equations based on the likelihood Equation (22) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mather 1957</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Ripol <em>et al.</em> 1999</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Luo <em>et al.</em> 2004</a>). The equations were plotted as a function of the recombination fraction. The information profiles are related to the number of different haplotypes present on the parental configuration for a given marker dosage. For instance, for two single-dose markers (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, panel A), when the alleles share the same homolog (<span class="inline-formula" id="inline-formula-354"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-383.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-383.gif"/></noscript></span></span>), it is always possible to detect if the gamete contains at least one recombinant chromosome. However, when the alleles are in different homologs (<span class="inline-formula" id="inline-formula-355"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-384.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3297/embed/mml-math-384.gif"/></noscript></span></span>), the detection of recombination events is limited to meiotic configurations containing a bivalent where these chromosomes paired with each other. Intermediate situations involving multi-dose markers can be observed in the other panels in <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>. Additionally, the model proposed here contemplates both parents on the analyses, leading to more complicated linkage phase configurations and information equations. The lack of information for some phase configurations in two-point procedures is essentially caused by the biallelic nature of the dosage-based markers. However, in several situations, genomic and transcriptomic references are available for related diploid species and often provide the physical order of the SNPs in small regions. Our phasing procedure could be applied in these regions to obtain local haplotypes, which could function as multiallelic markers improving the information in a two-point analysis. Moreover, in a multipoint context, when using multiallelic markers, the number of visited states in the Markov model can be significantly reduced, making the HMM procedure much more efficient. Ideally, in a full-sib population, the number of different alleles should be as high as two times the ploidy level (fully informative). In this case, the Markov model would be fully observed and, the task of estimating recombination fraction reduces to count the number of recombinant events given a linkage phase configuration. Since our algorithm does not need the entire transition space to work, only a subset of states should be visited, making the calculation much faster when compared to the biallelic case.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fisher&#x2019;s information for the two-point maximum likelihood estimators in different combinations of dosages and linkage phases configurations considering one informative hexaploid parent. (I) single-dose markers; alleles share 1 and 0 homologs. (II) double-dose markers; alleles share 2, 1 and 0 homologs. (III) triple dose markers; alleles share 3, 2, 1 and 0 homologs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1714010710" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fisher&#x2019;s information for the two-point maximum likelihood estimators in different combinations of dosages and linkage phases configurations considering one informative hexaploid parent. (I) single-dose markers; alleles share 1 and 0 homologs. (II) double-dose markers; alleles share 2, 1 and 0 homologs. (III) triple dose markers; alleles share 3, 2, 1 and 0 homologs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3297/F7.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/10/3297/F7.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3297/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-63" class="first-child">Fisher’s information for the two-point maximum likelihood estimators in different combinations of dosages and linkage phases configurations considering one informative hexaploid parent. (I) single-dose markers; alleles share 1 and 0 homologs. (II) double-dose markers; alleles share 2, 1 and 0 homologs. (III) triple dose markers; alleles share 3, 2, 1 and 0 homologs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">Once the map is assembled, given the HMM framework, it is a trivial exercise to obtain the probability of a specific genotype at any map position, conditioned on the whole linkage group and to compute the probability of any unobserved genotype given the genetic map using this information. These conditional probabilities are the basis for answering a series of fundamental questions about quantitative trait loci analysis in high-level autopolyploids, such as the effect of the dosage level on the variation of quantitative traits, the interaction of the alleles within (dominance effects) and between loci (epistatic effects). Therefore, the present study provides a sound basis for unveiling the complex structure of autopolyploid genomes through genetic mapping.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">The authors wish to thank Dr. Guilherme da Silva Pereira and Dr. Zhao-Bang Zeng for their invaluable suggestions for elaboration of the manuscript. We also thank Dr. Peter Bourke for consulting with us on the comparison between polymapR and our HMM-based method. This work was supported by the Bill and Melinda Gates Foundation [OPP1052983] and is part of the Genomic Tools for Sweetpotato Improvement project (GT4SP); MM was also supported by Fundação de Amparo à Pesquisa do Estado de São Paulo (FAPESP) [2012/17009-8, 2013/12245-8]; AAFG was supported by FAPESP [08/52197-4]; AAFG has a productivity scholarship from CNPq.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-66">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8218325">https://doi.org/10.25387/g3.8218325</a>.</p></li><li class="fn" id="fn-2"><p id="p-67"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-03"><span class="received-label">Received </span>June 3, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-08"><span class="accepted-label">Accepted </span>August 8, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Mollinari, Garcia</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.1" data-doi="10.1139/G07-056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jackson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">McIntyre</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Construction of a genetic linkage map for <em>Saccharum officinarum</em> incorporating both simplex and duplex markers to increase genome coverage.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">742</span>–<span class="cit-lpage">756</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/G07-056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DAitken%26rft.auinit1%253DK.%2BS.%26rft.volume%253D50%26rft.issue%253D8%26rft.spage%253D742%26rft.epage%253D756%26rft.atitle%253DConstruction%2Bof%2Ba%2Bgenetic%2Blinkage%2Bmap%2Bfor%2BSaccharum%2Bofficinarum%2Bincorporating%2Bboth%2Bsimplex%2Band%2Bduplex%2Bmarkers%2Bto%2Bincrease%2Bgenome%2Bcoverage.%26rft_id%253Dinfo%253Adoi%252F10.1139%252FG07-056%26rft_id%253Dinfo%253Apmid%252F17893734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/G07-056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17893734&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.2" data-doi="10.1007/s00438-013-0803-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arizio</span>,  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Costa Tártara</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Manifesto</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Carotenoids gene markers for sweetpotato (<em>Ipomoea batatas</em> l. lam): applications in genetic mapping, diversity evaluation and cross-species transference.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-013-0803-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D289%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-013-0803-3%26rft_id%253Dinfo%253Apmid%252F24384928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-013-0803-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24384928&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bargary</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hinde</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Finite mixture model clustering of SNP data</span>, pp. <span class="cit-fpage">139</span>–<span class="cit-lpage">157</span> in <span class="cit-source">Statistical Modeling in Biostatistics and Bioinformatics</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Switzerland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behrouzi</span>,  <span class="cit-name-given-names">P.</span></span> and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Wit</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> De novo construction of polyploid linkage maps using discrete graphical models. arXiv .</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.5" data-doi="10.1007/978-3-642-31442-1_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>,  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic Consequences of Polyploidy in Plants</span>, pp. <span class="cit-fpage">21</span>–<span class="cit-lpage">32</span> in <span class="cit-source">Polyploidy and Genome Evolution</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Soltis</span>,  <span class="cit-name-given-names">P. S.</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Soltis</span></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin, Germany</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-31442-1_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPolyploidy%2Band%2BGenome%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-31442-1_2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-31442-1_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.6" data-doi="10.1093/bioinformatics/bty371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">van Geest</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kranenburg</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">polymapR–linkage analysis and genetic map construction from F1 populations of outcrossing polyploids.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">3496</span>–<span class="cit-lpage">3502</span>. <span class="cit-comment">Erratum: 540</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3496%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.7" data-doi="10.1007/s00122-016-2768-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kranenburg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G. F.</span> <span class="cit-name-surname">Visser</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Integrating haplotype-specific linkage maps in tetraploid species using SNP markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2211</span>–<span class="cit-lpage">2226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2768-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2211%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2768-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2768-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.8" data-doi="10.1534/genetics.115.181008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Visser</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maliepaard</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The double-reduction landscape in tetraploid potato as revealed by a high-density linkage map.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.181008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.181008%26rft_id%253Dinfo%253Apmid%252F26377683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzMvODUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.9" data-doi="10.1007/978-0-387-92125-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sen</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">A Guide to QTL Mapping with R/qtl</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-92125-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2BGuide%2Bto%2BQTL%2BMapping%2Bwith%2BR%252Fqtl%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-92125-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-92125-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buetow</span>,  <span class="cit-name-given-names">K. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chakravarti</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Multipoint gene mapping using seriation. I. General methods.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBuetow%26rft.auinit1%253DK.%2BH.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D180%26rft.epage%253D188%26rft.atitle%253DMultipoint%2Bgene%2Bmapping%2Busing%2Bseriation.%2BI.%2BGeneral%2Bmethods.%26rft_id%253Dinfo%253Apmid%252F3475978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3475978&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J757500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burnham</span>,  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-source">Discussions in cytogenetics</span>, <span class="cit-publ-name">Burgess Publishing</span>, <span class="cit-publ-loc">Mineapolis</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.12" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>,  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Constructing genetic maps by rapid chain delineation.</span> <abbr class="cit-jnl-abbrev">J. Quant. Trait Loci</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BQuant.%2BTrait%2BLoci%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.14" data-doi="10.1073/pnas.97.14.7951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>,  <span class="cit-name-given-names">R. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Craig</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Model selection for quantitative trait locus analysis in polyploids.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7951</span>–<span class="cit-lpage">7956</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.97.14.7951</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D97%26rft.issue%253D14%26rft.spage%253D7951%26rft.epage%253D7956%26rft.atitle%253DModel%2Bselection%2Bfor%2Bquantitative%2Btrait%2Blocus%2Banalysis%2Bin%2Bpolyploids%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.14.7951%26rft_id%253Dinfo%253Apmid%252F10884425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTQvNzk1MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.15" data-doi="10.1098/rstb.1947.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">The theory of linkage in polysomic inheritance.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">233</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.1947.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1947.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.1947.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.16" data-doi="10.1111/j.1469-1809.1943.tb02320.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">Allowance for double reduction in the calculation of genotype frequencies with polysomic inheritance.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1943.tb02320.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02320.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02320.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Quantitative genetics and breeding methods in autopolyploids plants</span>, <span class="cit-publ-name">INRA</span>, <span class="cit-publ-loc">Paris</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.18" data-doi="10.1038/srep03399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>,  <span class="cit-name-given-names">A. A. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mollinari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Marconi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. R.</span> <span class="cit-name-surname">Serang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Silva</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">3399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep03399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D3399%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep03399%26rft_id%253Dinfo%253Apmid%252F24292365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep03399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24292365&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.19" data-doi="10.1186/s12859-016-1416-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grandke</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ranganathan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">van Bers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">de Haan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Metzler</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">PERGOLA: Fast and deterministic linkage mapping of polyploids.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1416-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1416-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1416-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.20" data-doi="10.1017/S0016672301005262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A comment on Xie and Xu: ‘Mapping quantitative trait loci in tetraploid species’.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672301005262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D187%26rft.epage%253D189%26rft.atitle%253DA%2Bcomment%2Bon%2BXie%2Band%2BXu%253A%2B%2527mapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Btetraploid%2Bspecies%2527.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672301005262%26rft_id%253Dinfo%253Apmid%252F11732096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672301005262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11732096&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.21" data-doi="10.1007/s00122-014-2347-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Bradshaw</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Bryan</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL mapping in autotetraploids using snp dosage information.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1885</span>–<span class="cit-lpage">1904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2347-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1885%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2347-2%26rft_id%253Dinfo%253Apmid%252F24981609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2347-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24981609&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Bradshaw</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">McNicol</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Interval mapping of quantitative trait loci in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D159%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1832%26rft.atitle%253DInterval%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2Bin%2BAutotetraploid%2BSpecies%26rft_id%253Dinfo%253Apmid%252F11779817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.23" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Broadfoot</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.24" data-doi="10.1371/journal.pone.0063939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">McLean</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Bryan</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Linkage analysis and QTL mapping using SNP dosage data in a tetraploid potato mapping population.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e63939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0063939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De63939%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0063939%26rft_id%253Dinfo%253Apmid%252F23704960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0063939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23704960&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.25" data-doi="10.1007/BF02984197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">Theoretical Genetics of Autopolyploids.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02984197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D22%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02984197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02984197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200610100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.26" data-doi="10.1023/A:1018394410659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.-B.</span> <span class="cit-name-surname">Zeng</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1018394410659</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DJiang%26rft.auinit1%253DC.%26rft.volume%253D101%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D58%26rft.atitle%253DMapping%2Bquantitative%2Btrait%2Bloci%2Bwith%2Bdominant%2Band%2Bmissing%2Bmarkers%2Bin%2Bvarious%2Bcrosses%2Bfrom%2Btwo%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1018394410659%26rft_id%253Dinfo%253Apmid%252F9465409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1018394410659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9465409&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071700000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.27" data-doi="10.1023/A:1022870917230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kriegner</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Cervantes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Burg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. O.</span> <span class="cit-name-surname">Mwanga</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A genetic linkage map of sweetpotato [<em>Ipomoea batatas</em>(l.) lam.] based on aflp markers.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1022870917230</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D11%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1022870917230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1022870917230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.28" data-doi="10.1073/pnas.84.8.2363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">E. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Green</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Construction of multilocus genetic linkage maps in humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">2363</span>–<span class="cit-lpage">2367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.84.8.2363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D84%26rft.issue%253D8%26rft.spage%253D2363%26rft.epage%253D2367%26rft.atitle%253DConstruction%2Bof%2Bmultilocus%2Bgenetic%2Blinkage%2Bmaps%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.8.2363%26rft_id%253Dinfo%253Apmid%252F3470801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NC84LzIzNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.29" data-doi="10.1016/0888-7543(87)90010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Green</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Abrahamson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Barlow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Daly</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Mapmaker: An interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">181</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0888-7543(87)90010-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D181%26rft.atitle%253DMAPMAKER%253A%2Ban%2Binteractive%2Bcomputer%2Bpackage%2Bfor%2Bconstructing%2Bprimary%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bexperimental%2Band%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990010-3%26rft_id%253Dinfo%253Apmid%252F3692487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3692487&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.30" data-doi="10.1073/pnas.0908477107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leach</span>,  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Kearsey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Luo</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multilocus tetrasomic linkage analysis using hidden markov chain model.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">4270</span>–<span class="cit-lpage">4274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0908477107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0908477107%26rft_id%253Dinfo%253Apmid%252F20142473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzkvNDI3MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.31" data-doi="10.1534/genetics.114.165845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemmon</span>,  <span class="cit-name-given-names">Z. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Doebley</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic dissection of a genomic region with pleiotropic effects on domestication traits in maize reveals multiple linked QTL.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.165845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165845%26rft_id%253Dinfo%253Apmid%252F24950893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMzQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.32" data-doi="10.1101/gr.094557.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewin</span>,  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Larkin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pontius</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">O’Brien</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Every genome sequence needs a good map.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1925</span>–<span class="cit-lpage">1928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094557.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094557.109%26rft_id%253Dinfo%253Apmid%252F19596977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8xOTI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.33" data-doi="10.1155/2010/216547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Das</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Fu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">EM algorithm for mapping quantitative trait loci in multivalent tetraploids.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2010</span>: <span class="cit-elocation-id">216547</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2010/216547</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BPlant%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D2010%26rft.spage%253D216547%26rft.epage%253D216547%26rft.atitle%253DEm%2Balgorithm%2Bfor%2Bmapping%2Bquantitative%2Btrait%2BLoci%2Bin%2Bmultivalent%2Btetraploids.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2010%252F216547%26rft_id%253Dinfo%253Apmid%252F21317976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2010/216547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21317976&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.34" data-doi="10.1073/pnas.1219082110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. Q.</span> <span class="cit-name-surname">Gu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. M.</span> <span class="cit-name-surname">You</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Deal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ma</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A 4-gigabase physical map unlocks the structure and evolution of the complex genome of aegilops tauschii, the wheat d-genome progenitor.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">7940</span>–<span class="cit-lpage">7945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1219082110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1219082110%26rft_id%253Dinfo%253Apmid%252F23610408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc5NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.35" data-doi="10.1073/pnas.0304482101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Kearsey</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Theoretical basis for genetic linkage analysis in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">7040</span>–<span class="cit-lpage">7045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0304482101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0304482101%26rft_id%253Dinfo%253Apmid%252F15100415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE4LzcwNDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.36" data-doi="10.1534/genetics.105.052449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Leach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Bradshaw</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Constructing genetic linkage maps under a tetrasomic model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2635</span>–<span class="cit-lpage">2645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.052449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.052449%26rft_id%253Dinfo%253Apmid%252F16415363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI2MzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>,  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. C.</span> <span class="cit-name-surname">Manrique-Carpintero</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Coombs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Zarka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Boone</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic linkage mapping of economically important traits in cultivated tetraploid potato (solanum tuberosum l.). G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2357</span>–<span class="cit-lpage">2364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D5%26rft.spage%253D2357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.38" data-doi="10.1007/BF02982683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">Segregation and linkage in autotetraploids.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02982683</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D32%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02982683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02982683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200612400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-source">The mesurement of linkage in heredity</span>, <span class="cit-publ-name">Methuen &amp; Co</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.40" data-doi="10.1038/hdy.2009.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R. A.</span> <span class="cit-name-surname">Margarido</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vencovsky</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evaluation of algorithms used to order markers on genetic maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2009.96</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMollinari%26rft.auinit1%253DM.%26rft.volume%253D103%26rft.issue%253D6%26rft.spage%253D494%26rft.epage%253D502%26rft.atitle%253DEvaluation%2Bof%2Balgorithms%2Bused%2Bto%2Border%2Bmarkers%2Bon%2Bgenetic%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.96%26rft_id%253Dinfo%253Apmid%252F19639011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.96&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19639011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.41" data-doi="10.1007/978-1-4939-1966-6_17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Serang</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative SNP Genotyping of Polyploids with MassARRAY and Other Platforms</span>, pp. <span class="cit-fpage">215</span>–<span class="cit-lpage">241</span> in <span class="cit-source">Plant genotyping: methods and protocols</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Batley</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-1966-6_17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bgenotyping%253A%2Bmethods%2Band%2Bprotocols%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-1966-6_17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-1966-6_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.42" data-doi="10.1086/279426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>,  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1914</span> <span class="cit-article-title">A New Mode of Segregation in Gregory’s Tetraploid Primulas.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/279426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D48%26rft.spage%253D508%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200856200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.43" data-doi="10.1016/S0168-9525(03)00015-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>,  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Pires</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Birchler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Auger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. J.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Understanding mechanisms of novel gene expression in polyploids.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(03)00015-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DOsborn%26rft.auinit1%253DT.%2BC.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D141%26rft.epage%253D147%26rft.atitle%253DUnderstanding%2Bmechanisms%2Bof%2Bnovel%2Bgene%2Bexpression%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252803%252900015-5%26rft_id%253Dinfo%253Apmid%252F12615008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(03)00015-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12615008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181584500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.44" data-doi="10.1146/annurev.genet.34.1.401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>,  <span class="cit-name-given-names">S. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Whitton</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Polyploid incidence and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">437</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.34.1.401</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D401%26rft.epage%253D437%26rft.atitle%253DPOLYPLOID%2BINCIDENCE%2BAND%2BEVOLUTION%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.34.1.401%26rft_id%253Dinfo%253Apmid%252F11092833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.34.1.401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11092833&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166299400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.45" data-doi="10.1007/s00122-016-2761-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preedy</span>,  <span class="cit-name-given-names">K. F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Hackett</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A rapid marker ordering approach for high-density genetic linkage maps in experimental autotetraploid populations using multidimensional scaling.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2117</span>–<span class="cit-lpage">2132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2761-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2117%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2761-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2761-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.46" data-doi="10.1109/5.18626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabiner</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A tutorial on hidden markov models and selected applications in speech recognition.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/5.18626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D77%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.18626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/5.18626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.47" data-doi="10.1007/s11032-017-0619-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rak</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Bethke</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Palta</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Qtl mapping of potato chip color and tuber traits within an autotetraploid family.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-017-0619-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D37%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-017-0619-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-017-0619-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.48" data-doi="10.1534/genetics.112.145581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rehmsmeier</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A computational approach to developing mathematical models of polyploid meiosis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1094</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.145581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145581%26rft_id%253Dinfo%253Apmid%252F23335332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEwODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.49" data-doi="10.1016/S0378-1119(99)00218-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ripol</span>,  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A. G. D.</span> <span class="cit-name-surname">Silva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Statistical aspects of genetic mapping in autopolyploids.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">235</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0378-1119(99)00218-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DRipol%26rft.auinit1%253DM.%2BI.%26rft.volume%253D235%26rft.issue%253D1-2%26rft.spage%253D31%26rft.epage%253D41%26rft.atitle%253DStatistical%2Baspects%2Bof%2Bgenetic%2Bmapping%2Bin%2Bautopolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900218-8%26rft_id%253Dinfo%253Apmid%252F10415330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00218-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10415330&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081851700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.50" data-doi="10.3835/plantgenome2014.01.0002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schie</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chaudhary</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Debener</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Analysis of a Complex Polyploid Plant Genome using Molecular Markers: Strong Evidence for Segmental Allooctoploidy in Garden Dahlias.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.01.0002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.01.0002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.01.0002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.51" data-doi="10.1371/journal.pone.0030906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serang</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mollinari</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient exact maximum a posteriori computation for bayesian snp genotyping in polyploids.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e30906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0030906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSerang%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30906%26rft.epage%253De30906%26rft.atitle%253DEfficient%2Bexact%2Bmaximum%2Ba%2Bposteriori%2Bcomputation%2Bfor%2Bbayesian%2BSNP%2Bgenotyping%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030906%26rft_id%253Dinfo%253Apmid%252F22363513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363513&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>,  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bolser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">de Boer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sønderkær</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Amoros</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Construction of Reference Chromosome-Scale Pseudomolecules for Potato: Integrating the Potato Genome with Genetic and Physical Maps. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2031</span>–<span class="cit-lpage">2047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D3%26rft.spage%253D2031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.53" data-doi="10.1038/srep44207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tanaka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Takahata</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Cao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A high-density SNP genetic map consisting of a complete set of homologous groups in autohexaploid sweetpotato (<em>Ipomoea batatas</em>).</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">44207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep44207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D44207%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep44207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep44207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.54" data-doi="10.1111/nph.12756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Segovia-Salcedo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Jordon-Thaden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Majure</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Miles</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Are polyploids really evolutionary dead-ends (again)? A critical reappraisal of Mayrose <em>et al.</em> ()</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12756</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D202%26rft.spage%253D1105%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12756%26rft_id%253Dinfo%253Apmid%252F24754325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12756&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24754325&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335470200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.55" data-doi="10.1080/713608048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Soltis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. H.</span> <span class="cit-name-surname">Rieseberg</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Molecular Data and the Dynamic Nature of Polyploidy.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">273</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07352689309701903</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D12%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.1080%252F713608048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/713608048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LJ92200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.56" data-doi="10.3732/ajb.1400178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Visger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Soltis</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">The polyploidy revolution then…and now: Stebbins revisited.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1057</span>–<span class="cit-lpage">1078</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.1400178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1400178%26rft_id%253Dinfo%253Apmid%252F25049267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDEvNy8xMDU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.57" data-doi="10.2135/cropsci1992.0011183X003200050003x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>,  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Development and application of rflps in polyploids.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1086</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1992.0011183X003200050003x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSorrells%26rft.auinit1%253DM.%2BE.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1086%26rft.epage%253D1091%26rft.atitle%253DDevelopment%2Band%2BApplication%2Bof%2BRFLPs%2Bin%2BPolyploids%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1992.0011183X003200050003x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1992.0011183X003200050003x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.58" data-doi="10.1007/978-3-642-80960-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sybenga</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-source">Meiotic configurations</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-80960-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMeiotic%2Bconfigurations%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-80960-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-80960-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.59" data-doi="10.1007/s00122-017-2974-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Geest</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Bourke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Marasek-Ciolakowska</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An ultra-dense integrated linkage map for hexaploid chrysanthemum enables multi-allelic QTL analysis.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2527</span>–<span class="cit-lpage">2541</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2974-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2527%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2974-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2974-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3297.60" data-doi="10.1017/CBO9781139003889"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>,  <span class="cit-name-given-names">J. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jansen</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Genetic Mapping in Experimental Populations</span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/CBO9781139003889</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BMapping%2Bin%2BExperimental%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1017%252FCBO9781139003889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/CBO9781139003889&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.61" data-doi="10.1007/s00122-005-0097-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Os</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Stam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Visser</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Van Eck</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Record: a novel method for ordering loci on a genetic linkage map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-005-0097-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVan%2BOs%26rft.auinit1%253DH.%26rft.volume%253D112%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D40%26rft.atitle%253DRECORD%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Bordering%2Bloci%2Bon%2Ba%2Bgenetic%2Blinkage%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0097-x%26rft_id%253Dinfo%253Apmid%252F16228189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0097-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16228189&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.62" data-doi="10.1186/1471-2105-12-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>,  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gort</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vosman</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype calling in tetraploid species from bi-allelic marker data using mixture models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D12%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DGenotype%2Bcalling%2Bin%2Btetraploid%2Bspecies%2Bfrom%2Bbi-allelic%2Bmarker%2Bdata%2Busing%2Bmixture%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-172%26rft_id%253Dinfo%253Apmid%252F21595880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21595880&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.63" data-doi="10.1186/1471-2105-13-248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>,  <span class="cit-name-given-names">R. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maliepaard</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The simulation of meiosis in diploid and tetraploid organisms using various genetic models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-248</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D248%26rft.epage%253D248%26rft.atitle%253DThe%2Bsimulation%2Bof%2Bmeiosis%2Bin%2Bdiploid%2Band%2Btetraploid%2Borganisms%2Busing%2Bvarious%2Bgenetic%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-248%26rft_id%253Dinfo%253Apmid%252F23013469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23013469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.64" data-doi="10.1007/s00122-016-2806-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">van Eeuwijk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jansen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The potential of probabilistic graphical models in linkage map construction.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2806-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2806-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.65" data-doi="10.1186/1471-2164-11-261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Roe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Macmil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Murray</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Microcollinearity between autopolyploid sugarcane and diploid sorghum genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">261</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-261</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.spage%253D261%26rft.epage%253D261%26rft.atitle%253DMicrocollinearity%2Bbetween%2Bautopolyploid%2Bsugarcane%2Band%2Bdiploid%2Bsorghum%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-261%26rft_id%253Dinfo%253Apmid%252F20416060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20416060&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.66" data-doi="10.1007/BF00224274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Burnquist</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Tew</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Moore</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The detection and estimation of linkage in polyploids using single-dose restriction fragments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">294</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00224274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00224274%26rft_id%253Dinfo%253Apmid%252F24202510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00224274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24202510&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HB49800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.67" data-doi="10.1534/genetics.166.1.581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-X.</span> <span class="cit-name-surname">Ma</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Casella</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A bivalent polyploid model for mapping quantitative trait loci in outcrossing tetraploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.166.1.581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D581%26rft.epage%253D595%26rft.atitle%253DA%2BBivalent%2BPolyploid%2BModel%2Bfor%2BMapping%2BQuantitative%2BTrait%2BLoci%2Bin%2BOutcrossing%2BTetraploids%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.581%26rft_id%253Dinfo%253Apmid%252F15020446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvNTgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.68" data-doi="10.1006/tpbi.2002.1577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-X.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Painter</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.-B.</span> <span class="cit-name-surname">Zeng</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Simultaneous Maximum Likelihood Estimation of Linkage and Linkage Phases in Outcrossing Species.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/tpbi.2002.1577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D363%26rft.atitle%253DSimultaneous%2Bmaximum%2Blikelihood%2Bestimation%2Bof%2Blinkage%2Band%2Blinkage%2Bphases%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1577%26rft_id%253Dinfo%253Apmid%252F12027621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027621&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176540100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.69" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Bhat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Close</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lonardi</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1000212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000212</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.70" data-doi="10.1093/bib/bbt073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lyu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A statistical model for QTL mapping in polysomic autotetraploids underlying double reduction.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1044</span>–<span class="cit-lpage">1056</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbt073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbt073%26rft_id%253Dinfo%253Apmid%252F24177379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbt073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24177379&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3297.71" data-doi="10.1534/genetics.115.185579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Hackett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ho</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Probabilistic multilocus haplotype reconstruction in outcrossing tetraploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.185579</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185579%26rft_id%253Dinfo%253Apmid%252F26920758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvMTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3297.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3287" title="Comparative Transcriptomic Analysis Reveals Molecular Profiles of Central Nervous System in Maternal Diapause Induction of Locusta migratoria" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3315" title="Disease Resistance Genetics and Genomics in Octoploid Strawberry" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3297" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IIN0E5ehvMUwAYhmgTsVtP2SVl-aHdqYyjyPjPROIbI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3297" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3297" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3297" class="active" data-icon-position="" data-hide-link-title="0">Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494271" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-43tG2wXqG3RA3_60ylAQ-lMGGBJMzP4FCUqjqCn-1yY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3297.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3297" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xGUht0f4lAjgJbUMtchv9mQ6tDIvYcUIOPSz8Ody0tE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3297" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3297.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494271" id="node494271--2" data-pisa="ggg;9/10/3297" data-pisa-master="ggg;g3.119.400378" data-apath="/ggg/9/10/3297.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcelo Mollinari</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio Augusto Franco Garcia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3297-3314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400378 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494271--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcelo Mollinari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticultural Science, Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcelo%2BMollinari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mollinari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcelo%2BMollinari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcelo Mollinari</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonio Augusto Franco Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of São Paulo/ESALQ, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BAugusto%2BFranco%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BAugusto%2BFranco%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio Augusto Franco Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3297" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3297&amp;title=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3297&amp;title=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3297&amp;title=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3297&amp;text=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3297&amp;title=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3297&amp;t=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3297&amp;title=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3297&amp;title=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494271/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494271" id="node494271--3" data-pisa="ggg;9/10/3297" data-pisa-master="ggg;g3.119.400378" data-apath="/ggg/9/10/3297.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Haplotype Phasing in Experimental Autopolyploid Populations with High Ploidy Level Using Hidden Markov Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcelo Mollinari</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio Augusto Franco Garcia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3297-3314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400378 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494271--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcelo Mollinari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticultural Science, Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcelo%2BMollinari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mollinari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcelo%2BMollinari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcelo Mollinari</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonio Augusto Franco Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of São Paulo/ESALQ, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BAugusto%2BFranco%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BAugusto%2BFranco%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio Augusto Franco Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3297&amp;rft.epage=3314&amp;rft.atitle=Linkage%20Analysis%20and%20Haplotype%20Phasing%20in%20Experimental%20Autopolyploid%20Populations%20with%20High%20Ploidy%20Level%20Using%20Hidden%20Markov%20Models&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mollinari%2C+Marcelo&amp;rft.au=Garcia%2C+Antonio+Augusto+Franco"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494271/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494271/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494271/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494271/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494271/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494271/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494271/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494271/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494271/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494271/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494271/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494271/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3297" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3297.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3297.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3297.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
