<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3345" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3345.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3345.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3345.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3345.atom" />
<meta name="HW.pisa" content="ggg;9/10/3345" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii" />
<meta name="DC.Identifier" content="10.1534/g3.119.400428" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Cohn et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The enzyme telomerase ensures the integrity of linear chromosomes by maintaining telomere length. As a hallmark of cancer, cell immortalization and unlimited proliferation is gained by reactivation of telomerase. However, a significant fraction of cancer cells instead uses alternative telomere lengthening mechanisms to ensure telomere function, collectively known as Alternative Lengthening of Telomeres (ALT). Although the budding yeast Naumovozyma castellii ( Saccharomyces castellii ) has a proficient telomerase activity, we demonstrate here that telomeres in N. castellii are efficiently maintained by a novel ALT mechanism after telomerase knockout. Remarkably, telomerase-negative cells proliferate indefinitely without any major growth crisis and display wild-type colony morphology. Moreover, ALT cells maintain linear chromosomes and preserve a wild-type DNA organization at the chromosome termini, including a short stretch of terminal telomeric sequence. Notably, ALT telomeres are elongated by the addition of ∼275 bp repeats containing a short telomeric sequence and the subtelomeric DNA located just internally (TelKO element). Although telomeres may be elongated by several TelKO repeats, no dramatic genome-wide amplification occurs, thus indicating that the repeat addition may be regulated. Intriguingly, a short interstitial telomeric sequence (ITS) functions as the initiation point for the addition of the TelKO element. This implies that N. castellii telomeres are structurally predisposed to efficiently switch to the ALT mechanism as a response to telomerase dysfunction." />
<meta name="DC.Contributor" content="Marita Cohn" />
<meta name="DC.Contributor" content="Ahu Karademir Andersson" />
<meta name="DC.Contributor" content="Raquel Quintilla Mateo" />
<meta name="DC.Contributor" content="Mirja Carlsson Möller" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The enzyme telomerase ensures the integrity of linear chromosomes by maintaining telomere length. As a hallmark of cancer, cell immortalization and unlimited proliferation is gained by reactivation of telomerase. However, a significant fraction of cancer cells instead uses alternative telomere lengthening mechanisms to ensure telomere function, collectively known as Alternative Lengthening of Telomeres (ALT). Although the budding yeast &lt;i&gt;Naumovozyma castellii&lt;/i&gt; (&lt;i&gt;Saccharomyces castellii&lt;/i&gt;) has a proficient telomerase activity, we demonstrate here that telomeres in &lt;i&gt;N. castellii&lt;/i&gt; are efficiently maintained by a novel ALT mechanism after telomerase knockout. Remarkably, telomerase-negative cells proliferate indefinitely without any major growth crisis and display wild-type colony morphology. Moreover, ALT cells maintain linear chromosomes and preserve a wild-type DNA organization at the chromosome termini, including a short stretch of terminal telomeric sequence. Notably, ALT telomeres are elongated by the addition of ∼275 bp repeats containing a short telomeric sequence and the subtelomeric DNA located just internally (TelKO element). Although telomeres may be elongated by several TelKO repeats, no dramatic genome-wide amplification occurs, thus indicating that the repeat addition may be regulated. Intriguingly, a short interstitial telomeric sequence (ITS) functions as the initiation point for the addition of the TelKO element. This implies that &lt;i&gt;N. castellii&lt;/i&gt; telomeres are structurally predisposed to efficiently switch to the ALT mechanism as a response to telomerase dysfunction.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3345" />
<meta name="citation_id" content="9/10/3345" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3345" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3345.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3345.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3345.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400428" />
<meta name="citation_pmid" content="31427453" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3345" />
<meta name="citation_lastpage" content="3358" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marita Cohn" />
<meta name="citation_author_institution" content="Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden" />
<meta name="citation_author_email" content="marita.cohn@biol.lu.se" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3370-2864" />
<meta name="citation_author" content="Ahu Karademir Andersson" />
<meta name="citation_author_institution" content="Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden" />
<meta name="citation_author" content="Raquel Quintilla Mateo" />
<meta name="citation_author_institution" content="Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden" />
<meta name="citation_author" content="Mirja Carlsson Möller" />
<meta name="citation_author_institution" content="Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=E. Astromskas;citation_author=M. Cohn;citation_title=Tools and methods for genetic analysis of Saccharomyces castellii.;citation_pages=499-509;citation_volume=24;citation_year=2007;citation_issue=6;citation_pmid=17431926;citation_doi=10.1002/yea.1488" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=E. Astromskas;citation_author=M. Cohn;citation_title=Ends-in vs. ends-out targeted insertion mutagenesis in Saccharomyces castellii.;citation_pages=339-347;citation_volume=55;citation_year=2009;citation_issue=3;citation_pmid=19437021;citation_doi=10.1007/s00294-009-0248-8" />
<meta name="citation_reference" content="citation_journal_title=Nature medicine;citation_journal_abbrev=Nat Med;citation_author=T. M. Bryan;citation_author=A. Englezou;citation_author=L. Dalla-Pozza;citation_author=M. A. Dunham;citation_author=R. R. Reddel;citation_title=Evidence for an alternative mechanism for maintaining telomere length in human tumors and tumor-derived cell lines.;citation_pages=1271-1274;citation_volume=3;citation_year=1997;citation_issue=11;citation_pmid=9359704;citation_doi=10.1038/nm1197-1271" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. J. Cesare;citation_author=R. R. Reddel;citation_title=Alternative lengthening of telomeres: models, mechanisms and implications.;citation_pages=319-330;citation_volume=11;citation_year=2010;citation_issue=5;citation_pmid=20351727;citation_doi=10.1038/nrg2763" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Q. Chen;citation_author=A. Ijpma;citation_author=C. W. Greider;citation_title=Two Survivor Pathways That Allow Growth in the Absence of Telomerase Are Generated by Distinct Telomere Recombination Events;citation_pages=1819-1827;citation_volume=21;citation_year=2001;citation_issue=5;citation_pmid=11238918;citation_doi=10.1128/MCB.21.5.1819-1827.2001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. F. Cliften;citation_author=R. S. Fulton;citation_author=R. K. Wilson;citation_author=M. Johnston;citation_title=After the duplication: gene loss and adaptation in Saccharomyces genomes.;citation_volume=172;citation_year=2006;citation_pmid=16322519;citation_doi=10.1534/genetics.105.048900" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Cohn;citation_author=E. H. Blackburn;citation_title=Telomerase in yeast;citation_pages=396-400;citation_volume=269;citation_year=1995;citation_issue=5222;citation_pmid=7618104;citation_doi=10.1126/science.7618104" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=M. Cohn;citation_author=M. J. McEachern;citation_author=E. H. Blackburn;citation_title=Telomeric sequence diversity within the genus Saccharomyces.;citation_pages=83-91;citation_volume=33;citation_year=1998;citation_issue=2;citation_pmid=9506895;citation_doi=10.1007/s002940050312" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Forstemann;citation_author=M. Hoss;citation_author=J. Lingner;citation_title=Telomerase-dependent repeat divergence at the 3' ends of yeast telomeres;citation_pages=2690-2694;citation_volume=28;citation_year=2000;citation_issue=14;citation_pmid=10908324;citation_doi=10.1093/nar/28.14.2690" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Fridholm;citation_author=E. Astromskas;citation_author=M. Cohn;citation_title=Telomerase-dependent generation of 70-nt-long telomeric single-stranded 3′ overhangs in yeast.;citation_volume=41;citation_year=2013;citation_pmid=23125369;citation_doi=10.1093/nar/gks1028" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. L. Gordon;citation_author=D. Armisén;citation_author=E. Proux-Wéra;citation_author=S. S. ÓhÉigeartaigh;citation_author=K. P. Byrne;citation_title=Evolutionary erosion of yeast sex chromosomes by mating-type switching accidents.;citation_volume=108;citation_year=2011;citation_pmid=22123960;citation_doi=10.1073/pnas.1112808108" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. W. Greider;citation_title=Telomerase is processive.;citation_pages=4572-4580;citation_volume=11;citation_year=1991;citation_issue=9;citation_pmid=1875940;citation_doi=10.1128/MCB.11.9.4572" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Groff-Vindman;citation_author=A. J. Cesare;citation_author=S. Natarajan;citation_author=J. D. Griffith;citation_author=M. J. McEachern;citation_title=Recombination at Long Mutant Telomeres Produces Tiny Single- and Double-Stranded Telomeric Circles;citation_pages=4406-4412;citation_volume=25;citation_year=2005;citation_issue=11;citation_pmid=15899847;citation_doi=10.1128/MCB.25.11.4406-4412.2005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. B. Harley;citation_author=A. B. Futcher;citation_author=C. W. Greider;citation_title=Telomeres shorten during ageing of human fibroblasts.;citation_pages=458-460;citation_volume=345;citation_year=1990;citation_issue=6274;citation_pmid=2342578;citation_doi=10.1038/345458a0" />
<meta name="citation_reference" content="citation_journal_title=Experimental cell research;citation_journal_abbrev=Exp Cell Res;citation_author=L. Hayflick;citation_author=P. S. Moorhead;citation_title=The serial cultivation of human diploid cell strains.;citation_pages=585-621;citation_volume=25;citation_year=1961;citation_pmid=13905658;citation_doi=10.1016/0014-4827(61)90192-6" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Jain;citation_author=A. K. Hebden;citation_author=T. M. Nakamura;citation_author=K. M. Miller;citation_author=J. P. Cooper;citation_title=HAATI survivors replace canonical telomeres with blocks of generic heterochromatin.;citation_pages=223-227;citation_volume=467;citation_year=2010;citation_issue=7312;citation_pmid=20829796;citation_doi=10.1038/nature09374" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=A. Karademir Andersson;citation_author=M. Cohn;citation_title=Naumovozyma castellii: an alternative model for budding yeast molecular biology.;citation_pages=95-109;citation_volume=34;citation_year=2017;citation_doi=10.1002/yea.3218" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=A. Karademir Andersson;citation_author=C. Gustafsson;citation_author=R. Krishnankutty;citation_author=M. Cohn;citation_title=Multiple DNA Interactions Contribute to the Initiation of Telomerase Elongation.;citation_pages=2109-2123;citation_volume=429;citation_year=2017;citation_doi=10.1016/j.jmb.2017.04.023" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=A. Karademir Andersson;citation_author=S. Oredsson;citation_author=M. Cohn;citation_title=Development of stable haploid strains and molecular genetic tools for Naumovozyma castellii (Saccharomyces castellii).;citation_pages=633-646;citation_volume=33;citation_year=2016;citation_doi=10.1002/yea.3213" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=A. Kass-Eisler;citation_author=C. W. Greider;citation_title=Recombination in telomere-length maintenance.;citation_pages=200-204;citation_volume=25;citation_year=2000;citation_issue=4;citation_pmid=10754555;citation_doi=10.1016/S0968-0004(00)01557-7" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. W. Kim;citation_author=M. A. Piatyszek;citation_author=K. R. Prowse;citation_author=C. B. Harley;citation_author=M. D. West;citation_title=Specific association of human telomerase activity with immortal cells and cancer;citation_pages=2011-2015;citation_volume=266;citation_year=1994;citation_issue=5193;citation_pmid=7605428;citation_doi=10.1126/science.7605428" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. LeBel;citation_author=E. Rosonina;citation_author=D. C. Sealey;citation_author=F. Pryde;citation_author=D. Lydall;citation_title=Telomere maintenance and survival in Saccharomyces cerevisiae in the absence of telomerase and RAD52.;citation_volume=182;citation_year=2009;citation_pmid=19380905;citation_doi=10.1534/genetics.109.102939" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. S. Lendvay;citation_author=D. K. Morris;citation_author=J. Sah;citation_author=B. Balasubramanian;citation_author=V. Lundblad;citation_title=Senescence mutants of Saccharomyces cerevisiae with a defect in telomere replication identify three additional EST genes.;citation_pages=1399-1412;citation_volume=144;citation_year=1996;citation_pmid=8978029" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=V. Lundblad;citation_author=E. H. Blackburn;citation_title=An alternative pathway for yeast telomere maintenance rescues est1- senescence.;citation_pages=347-360;citation_volume=73;citation_year=1993;citation_issue=2;citation_pmid=8477448;citation_doi=10.1016/0092-8674(93)90234-H" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=V. Lundblad;citation_author=J. W. Szostak;citation_title=A mutant with a defect in telomere elongation leads to senescence in yeast.;citation_pages=633-643;citation_volume=57;citation_year=1989;citation_issue=4;citation_pmid=2655926;citation_doi=10.1016/0092-8674(89)90132-3" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=L. Maringele;citation_author=D. Lydall;citation_title=Telomerase- and recombination-independent immortalization of budding yeast.;citation_volume=18;citation_year=2004;citation_pmid=15489288;citation_doi=10.1101/gad.316504" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. J. McEachern;citation_author=E. H. Blackburn;citation_title=Runaway telomere elongation caused by telomerase RNA gene mutations.;citation_pages=403-409;citation_volume=376;citation_year=1995;citation_issue=6539;citation_pmid=7630414;citation_doi=10.1038/376403a0" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. J. McEachern;citation_author=E. H. Blackburn;citation_title=Cap-prevented recombination between terminal telomeric repeat arrays (telomere CPR) maintains telomeres in Kluyveromyces lactis lacking telomerase.;citation_pages=1822-1834;citation_volume=10;citation_year=1996;citation_issue=14;citation_pmid=8698241;citation_doi=10.1101/gad.10.14.1822" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=M. J. McEachern;citation_author=J. E. Haber;citation_title=Break-induced replication and recombinational telomere elongation in yeast.;citation_pages=111-135;citation_volume=75;citation_year=2006;citation_pmid=16756487;citation_doi=10.1146/annurev.biochem.74.082803.133234" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. M. Nakamura;citation_author=J. P. Cooper;citation_author=T. R. Cech;citation_title=Two Modes of Survival of Fission Yeast Without Telomerase;citation_pages=493-496;citation_volume=282;citation_year=1998;citation_issue=5388;citation_pmid=9774280;citation_doi=10.1126/science.282.5388.493" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Natarajan;citation_author=M. J. McEachern;citation_title=Recombinational Telomere Elongation Promoted by DNA Circles;citation_pages=4512-4521;citation_volume=22;citation_year=2002;citation_issue=13;citation_pmid=12052861;citation_doi=10.1128/MCB.22.13.4512-4521.2002" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=A. M. Olovnikov;citation_title=A theory of marginotomy. The incomplete copying of template margin in enzymic synthesis of polynucleotides and biological significance of the phenomenon.;citation_pages=181-190;citation_volume=41;citation_year=1973;citation_issue=1;citation_pmid=4754905;citation_doi=10.1016/0022-5193(73)90198-7" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Systematic and Evolutionary Microbiology;citation_journal_abbrev=Int. J. Syst. Evol. Microbiol.;citation_author=R. F. Petersen;citation_author=T. Nilsson-Tillgren;citation_author=J. Piskur;citation_title=Karyotypes of Saccharomyces sensu lato species;citation_pages=1925-1931;citation_volume=49;citation_year=1999;citation_issue=4;citation_pmid=10555377;citation_doi=10.1099/00207713-49-4-1925" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=H. A. Pickett;citation_author=R. R. Reddel;citation_title=Molecular mechanisms of activity and derepression of alternative lengthening of telomeres.;citation_pages=875-880;citation_volume=22;citation_year=2015;citation_pmid=26581522;citation_doi=10.1038/nsmb.3106" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=R. J. Reid;citation_author=I. Sunjevaric;citation_author=M. Keddache;citation_author=R. Rothstein;citation_title=Efficient PCR-based gene disruption in Saccharomyces strains using intergenic primers.;citation_pages=319-328;citation_volume=19;citation_year=2002;citation_issue=4;citation_pmid=11870855;citation_doi=10.1002/yea.817" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=J. Rhodin Edsö;citation_author=R. Tati;citation_author=M. Cohn;citation_title=Highly sequence-specific binding is retained within the DNA-binding domain of the Saccharomyces castellii Cdc13 telomere-binding protein.;citation_volume=8;citation_year=2008;citation_pmid=18759744;citation_doi=10.1111/j.1567-1364.2008.00431.x" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cancer biology;citation_journal_abbrev=Semin Cancer Biol;citation_author=J. W. Shay;citation_author=W. E. Wright;citation_title=Role of telomeres and telomerase in cancer.;citation_pages=349-353;citation_volume=21;citation_year=2011;citation_issue=6;citation_pmid=22015685;citation_doi=10.1016/j.semcancer.2011.10.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. S. Singer;citation_author=D. E. Gottschling;citation_title=TLC1: template RNA component of Saccharomyces cerevisiae telomerase;citation_pages=404-409;citation_volume=266;citation_year=1994;citation_issue=5184;citation_pmid=7545955;citation_doi=10.1126/science.7545955" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=O. Steinberg-Neifach;citation_author=N. F. Lue;citation_title=Modulation of telomere terminal structure by telomerase components in Candida albicans.;citation_pages=2710-2722;citation_volume=34;citation_year=2006;citation_issue=9;citation_pmid=16714448;citation_doi=10.1093/nar/gkl345" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Topcu;citation_author=K. Nickles;citation_author=C. Davis;citation_author=M. J. McEachern;citation_title=Abrupt disruption of capping and a single source for recombinationally elongated telomeres in Kluyveromyces lactis.;citation_volume=102;citation_year=2005;citation_pmid=15713803;citation_doi=10.1073/pnas.0408770102" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. Wahlin;citation_author=M. Cohn;citation_title=Analysis of the RAP1 protein binding to homogeneous telomeric repeats in Saccharomyces castellii.;citation_pages=241-256;citation_volume=19;citation_year=2002;citation_issue=3;citation_pmid=11816032;citation_doi=10.1002/yea.816" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. Wahlin;citation_author=M. Rosen;citation_author=M. Cohn;citation_title=DNA binding and telomere length regulation of yeast RAP1 homologues.;citation_pages=821-833;citation_volume=332;citation_year=2003;citation_issue=4;citation_pmid=12972254;citation_doi=10.1016/S0022-2836(03)00850-7" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Wang;citation_author=S. Rizvydeen;citation_author=M. Vahedi;citation_author=D. M. Vargas Gonzalez;citation_author=A. L. Allred;citation_title=Novel telomere-anchored PCR approach for studying sexual stage telomeres in Aspergillus nidulans.;citation_pages=e99491;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0099491" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=J. Xu;citation_author=M. J. McEachern;citation_title=Maintenance of very long telomeres by recombination in the Kluyveromyces lactis stn1–M1 mutant involves extreme telomeric turnover, telomeric circles, and concerted telomeric amplification.;citation_volume=32;citation_year=2012;citation_pmid=22645309;citation_doi=10.1128/MCB.00430-12" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="The enzyme telomerase ensures the integrity of linear chromosomes by maintaining telomere length. As a hallmark of cancer, cell immortalization and unlimited proliferation is gained by reactivation of telomerase. However, a significant fraction of cancer cells instead uses alternative telomere lengthening mechanisms to ensure telomere function, collectively known as Alternative Lengthening of Telomeres (ALT). Although the budding yeast Naumovozyma castellii ( Saccharomyces castellii ) has a proficient telomerase activity, we demonstrate here that telomeres in N. castellii are efficiently maintained by a novel ALT mechanism after telomerase knockout. Remarkably, telomerase-negative cells proliferate indefinitely without any major growth crisis and display wild-type colony morphology. Moreover, ALT cells maintain linear chromosomes and preserve a wild-type DNA organization at the chromosome termini, including a short stretch of terminal telomeric sequence. Notably, ALT telomeres are elongated by the addition of ∼275 bp repeats containing a short telomeric sequence and the subtelomeric DNA located just internally (TelKO element). Although telomeres may be elongated by several TelKO repeats, no dramatic genome-wide amplification occurs, thus indicating that the repeat addition may be regulated. Intriguingly, a short interstitial telomeric sequence (ITS) functions as the initiation point for the addition of the TelKO element. This implies that N. castellii telomeres are structurally predisposed to efficiently switch to the ALT mechanism as a response to telomerase dysfunction." />
<meta name="og-title" property="og:title" content="Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3345" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The enzyme telomerase ensures the integrity of linear chromosomes by maintaining telomere length. As a hallmark of cancer, cell immortalization and unlimited proliferation is gained by reactivation of telomerase. However, a significant fraction of cancer cells instead uses alternative telomere lengthening mechanisms to ensure telomere function, collectively known as Alternative Lengthening of Telomeres (ALT). Although the budding yeast Naumovozyma castellii ( Saccharomyces castellii ) has a proficient telomerase activity, we demonstrate here that telomeres in N. castellii are efficiently maintained by a novel ALT mechanism after telomerase knockout. Remarkably, telomerase-negative cells proliferate indefinitely without any major growth crisis and display wild-type colony morphology. Moreover, ALT cells maintain linear chromosomes and preserve a wild-type DNA organization at the chromosome termini, including a short stretch of terminal telomeric sequence. Notably, ALT telomeres are elongated by the addition of ∼275 bp repeats containing a short telomeric sequence and the subtelomeric DNA located just internally (TelKO element). Although telomeres may be elongated by several TelKO repeats, no dramatic genome-wide amplification occurs, thus indicating that the repeat addition may be regulated. Intriguingly, a short interstitial telomeric sequence (ITS) functions as the initiation point for the addition of the TelKO element. This implies that N. castellii telomeres are structurally predisposed to efficiently switch to the ALT mechanism as a response to telomerase dysfunction." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"HOt209GQ5TTxSwpHgE5liZLUUGgoMoeBqBUY3TtYfKI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494286","apath":"\/ggg\/9\/10\/3345.atom","pisa":"ggg;9\/10\/3345","ac":{"\/ggg\/9\/10\/3345.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3345.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3345"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3345"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3345":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F59wkZwuJc\\\\\\\/kkxVXH6keqV17qZjYX+r4MPd7WMZHViTVKjHQwo05qRq9R67b8+0CrqJ4W379uIblG5yNElsdFVjtiPVpxGbJnmTBHj\\\\\\\/APUHfPuphPLwvFXz4iqHDjM0nnb\\\\\\\/LsT2pFZToJJyk6s1lD+4b3V2OZ\\\\\\\/06W6gJjy3ozuf3DphwczbXM+yC0X4gsAsDdqArECFQc6POe5yQlWFfD7itk7GjYMWbBJCBhAn3LR9jWL8877h46KWQegQR4QdeApSbnA7456w98oHukJlrVpZBfTzcEoO0gjQVs4yiT+TPkpmdHi6vlZsEMW3WIlTt8i6gWkZ8qdkgS0eknBmKVeG+3FNEbsFby7khjqtMSEz1Iq9o3ez\\\\\\\/B1JUJ4V1QW00vXmbWOSBgUXJKD0spH6okRTd9CxRwUSdmuTTi8jsyQNFqbTLY6\\\\\\\/HLswMJkwh5C8vboc1LEk0iJcRIBCKtxIylMe61zz9DPOXf4lqHCYM0ekcfRXwiF8udhEqCV8pfXN7mtI5X\\\\\\\/gXt9SisHDBfkf2e8Ps0NOuJDYQ911OvB4Qv7OlzGYFISLe8fCUuVMbeBCS3OQaz2O0wtLmrMlKKscalCQ3cjaupHYAx44rQ8LlMuoBceXHOgEge0c2lS5Za+XLK6swdpjQwi7kMSMdei71JeMrLUW\\\\\\\/L\\\\\\\/lAxly1H3bpTr4oc5\\\\\\\/luUV+lnIDX93Ndsyl4dcPqZsLa5mFFC8vO\\\\\\\/hQKcDLW+lcKc940taZHINRv66k1Bx5fQHL40IE4euRWaIuYYbW0aQjBLvq4me4dtM5xb0Dz57YdYM19WEe49dytRfao8eT5Ej2rIj2XIVHCpFQk3C3N+deBZJPaSndSj\\\\\\\/Ckxh\\\\\\\/g2XSIzjyUdJcPZn0nn66niNZvOfelsj7N+vM4cjEIFQhqi+bCFgTpqCl8f2os5RP2Iu1CxQmubM14DaiIQ6JIz5jYwg9XaWbVbyL5+gal6wRzb7cHtdMHKuqnFhoemE5lsUhgdkc18nP+RaXXhlr9lAf+6yr\\\\\\\/SYGKUCXfrs2pk9\\\\\\\/d+JUYoN57WM6F\\\\\\\/OCZNkbfmhkamXKEUx3fFmB+bjOro0shLAUBNzPXF\\\\\\\/0rWeLtrRsXuYWDqRQwReSmFiIIbGHd33habzv7Q\\\\\\\/Ks0Vxww5ojFylt80JaZZLh0iZ8stgu73C3T7ChCybtaPFyTaQ0p5PqVATkwB\\\\\\\/aU4\\\\\\\/fTCbeUHBIq6xxF1BxLr\\\\\\\/zGkug4iEcJPRaERdLsB3IXLkURQTCZwtN53QE2eA1swCVMU+NgOs4lUZ0C9W79BJxWjIivNO\\\u0022,\\\u0022iv\\\u0022:\\\u0022zRPfc00eA+QHaOSMVnEspg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022444c7e1130df13abb20bae94dd8e1d82be7a6de1d1f5bee7b05019f2d56f4c35\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RLY7DWYj5f73L6aDMngNDRj2Fu80fAOZG+GA3lXwWNguTAZW7760eEYoCORK0qQfLXNUnO3RxoIkmG5P4ioOMOR5664IxIv4lUP\\\\\\\/+zWoVeQkshqhMV6QflKd+xbrV\\\\\\\/CazVgzMyJWRS5ZFiJ6NE4PPUq1WypCsS49GH4FjhJKVSAz9Izr3fKLPywB914Lt1BRaTV\\\\\\\/Z7sIosorTMTQQKBWuF0vdJL1BaI7fgMWUQTWo\\\\\\\/xZxaGy7+tTptYqWOpIpOcUt+ExoL7G2vyJqdavcqpP0f+QpyLQoDc5FUqwOtsZiODFjEF2sdFGnf9EIVl3qSIYbp\\\\\\\/bPQO0seKENy4qH8QYK10xbFJt3KSh\\\\\\\/nK0lW5+8fuNMKXTj0wfzK2E20cmbJSG\\\\\\\/HjTHXEfjfOoPrbFfK4lSXBlBnjCHlyeGBaixjdznxm6l9mDrCPc1D6CLwkuqw6ojjAkhrpnWNWXcol9hNi4xqf1t1hL3denCSKmh63E6zZ4sevy8KGVCU7OULtBlrE8voZTkjpYOEiqZp+w99gAZAgkNcyASrH\\\\\\\/nsj7OASmwYhHTRy6LQWp8CNg1wfe2Q7XkkK9Uq8T76kvq4LqwTu4uliMb1EJ+xj4E7O1arngMjGr2c\\\\\\\/0mKIjiW6yAYKF4+AOxNGk01FKBKuMQ005e+jfUFp0uE8ltH5ZyQ6s04zHbQDrBhaVqUJtvJQ2u31zceNm8egb81pmAVgExdAqqfyuLLa43yS8Mht7IIVIY\\\\\\\/ui+qwrL75n1AVaE8GrTiCede03SDwbx6+9cIRYCuVhGWWCkLE2yLQKw48KJotte\\\\\\\/ZbdJqJScYfbO61DrAneFG\\\\\\\/0P3jtI+D\\\\\\\/vjel2VEBoou9G6LP4QrHvz7+bi5HFmKsNZBUFitbxXCBIXaemaEaiJC0JNGIki2+C21dLUBjczrhZZaOYnMvGb8CCkC4QBh9kpQJcM8NhKx4SiE+oM9w6mJt0vkW\\\\\\\/OGFnCUbF7RJn1wa9w7oE5S2w9JkCKx9LgNxDaiykhTq1q+sCc5VuG4F\\\\\\\/3okQ\\\\\\\/pRgKSctunKOwBpzMb4hZcsq2p170biMZhNUvelHGJua0YXHLFU1Hfou2gBpQSTI9sNAPbPs9Zx8EOJF05GmKVUdids81uyK5wrsWcTHX2TV57H0jteL1dObyvsekGGRocyDOLsYpG6+SUJvDyHLouZFEfnykQYmvzpQDVGlXFMloyLloi+7EFFZmsfjTDUzFZw+b40Z+X7hk7m8eSJHmp1XODoS8tCjcZGOHIC05ecGLKAee4PGfoVEN8AC0trzdL8ZhPU7JnxD\\\\\\\/XfbRpSbAuDbsxlP07t3gVDlAWRkg2bwUGHBceOIWOt\\\\\\\/0+xXuv8Uf9txu2kly+TdB2omTcm0dYk2bfD\\\\\\\/Kjqf9OjLCub6K6SUnP1ruz+rviTYyuJzXfFsMQ05dviVHH6RjYGOT+ZXuCx+7PrgjzfBgf7UT\\\\\\\/ZQTWEFJhbHw17V5UPuqJQdIW1feOMCoptRrKBSr\\\\\\\/K6g4pOt7Lz+kaIygUNe8QCZZ8jJ9etUDhz4dYxp60dYJ+ag8C4\\\\\\\/NQkMiTzKrzR5xdfEsGzrPerVhLGE3yHOTRhi\\\\\\\/J9r092bWhF1PbmEQmGjyh5HU4C2iEgiAnbE8OtY8oF5iDNF0JrxjUn6HyQh6+dQ7UWBu6wJYZyxRpwopL1mmDOujwhjssvdo6JJcSL1bdChWe7Hr0Q0QPuA9GsTTWFpvzeAKe1LrJGgiYSLLSFkJm+K09B4sCNugdznSy0R4Ii8UzTD\\\\\\\/XdjSW5+8jGmrv0JinTag8JHomDN3ZYF6k1cYzWRMaIFL66UVbzwrZYL\\\\\\\/4BNUJvHt1z\\\\\\\/kNNAAhQeV3gTwtVhtGDqjRn5lHGGRdurvjtD2e25uDS+k9T8CEIeuPk4lq3uvtlY+vnNNbVsE1wJ\\\\\\\/DnuHb9xelbfXHjpXF81nk8o\\\\\\\/IMQrl\\\\\\\/mIK2OBLAI+6FC1lJGKrNaQChUNx1k0tClRGnS3jwtyOu1XDDfAcSYFekf\\\\\\\/VF6m69aCC0U7yxGUexvDm9zY0Sm2hsfZAoWWSakQOowGvbo06BucSVh2SFInzK2TdDMdbOfgLSaO2C3g+nA51IkaDKew0ja9uEELqNaKpm1p7YnR46AxldPx6o7Dba+t9m\\\\\\\/vjvT48ZELeOVhDNScqmfjFsHTer8XZrqaWp\\\\\\\/o\\\\\\\/fTNMPMCTn2rLLJRvW74fdN3E7YZNBX5GZ83SkRXlLvvj0LKkVr5uroEJo0JBIPcwDt7wyCmnNVTVYZQiWtXwSSZA6YG71PSDcgkMWc0xtupL2QeKFTo9FboSRTZ31J8JXhVwUp92Q4ePix6YIngZttj6lnO200WZVn3W5NJZlN8iTC6v3Du3kCjscpyaBhcp+vgALkz3zRIwxtIqv4HLeolLlQ1+YkQ+fWA5uMgMa9MgipnwA0QVWvwAujush+2xAjg0JKqd5eoyFRETMKrm6c7VaQSNbqhEKAi+m3YxuwirFk8moP8CiP+lqGUImlAwxQTgO8kge0576t8JcpUt4DdwbwVnfR7ThJCnjNRlsStQuh3yh8ol2tqtlIB05sfFr\\\\\\\/MFG+EXHTtaT1YiqoN55XXeJM7b75k5f\\\\\\\/OVlaQttVaG4WV1rrYMdFbT7LMUU+9teNI7+iF5Suq1tmQqd0wY1a3gIgqm1SLXC7x885sfUcONLLt8jcDl+bDOl7kDpvSLAX\\\\\\\/bhTN4xP1C5lp47vPpOxQzOHB\\\\\\\/Ytg5251mpg4PE30wBi6L+GXyOSyFGLWXJA81CCB9LZ0HQNz1yJ8k5cfqmDmfNJM6TS\\\\\\\/5ZiQGkGzpztRIYblUibquqD4oUidk10YiQI7az6t9cwR5D\\\\\\\/7KdQtGYdu3gZJeEelHpqXgrDF+3oI2wfkQrLbmSZ5zAOWzPOuI5K2gbStAv5uq1iRPQ9voL\\\\\\\/yhkbd3BSK\\\\\\\/WU8kyNfwmdBFIOpjo\\\\\\\/ohF3J2vldqMz8YCsTk2Gi4Z87QNf8G2E3IGl\\\\\\\/u9h5N4hmj6JaxPxdMjcmU1uybkLdO5QxV164Qc\\\\\\\/M0fYcJqUdqKwbJHtQoS6F9ytJX51MFUleYfs3HeaPezLr+QmLDtm+Ydosk3SxQJscvGJdYX+FLyONlvezkSLA129mFxTkStJBfTcwPhBPVkeyV47GcSfRYsUVQWetZAzbS2kTsFGZacmCFfGTOsrdFbt3lz77H1q4PolqhbBIXB60+2SwMyaMje\\\\\\\/Dbv\\\\\\\/FQc2ynvhCIO6K0I0LmxqQfwTRmYfcPUhH0ikgvfX5eDcZI29294rZMKxjYUsw9YHuu8FPDAAEP5jj0C9Iz3jFLjweeBTd0Qd2\\\\\\\/mLU6VLI1lQe2TWnFldQfzhya+70j83chX6mJuFBkR7sUeUOKqmQf7fFcrgAjPNs2lbt6A3K73z1m6yPG6Cs2IPaF0BPqvyX2U2PJMpmoPYKkiYXe9BRUizoFgLobnIdFY9FyC9GISx4AJaWSRrLiXvoR6wKMnXuaY+wesv6QiUaGLN8peKuTYbCd91WttlyDF7hFk0IRHskiKXpF+JFfIxzlT+Gz04MfaGL0DDq5qbEC7Xx4L+ZGsfzCn2sj9v1E21Gg\\\\\\\/rd3yWq1vtcfEC+bFanmR6KZ8tjDC+fmKK1yBT\\\\\\\/d\\\\\\\/ZVA4H+XsEWUkDH9Ie64fg+Hmgokmnzkd3g8Ol+MqskI0eR4dT6ttuYoj5cGY+FEedydLx1lfyNHtIPt9f22n\\\\\\\/gHi02FyH9Z4sPyTbQsdirV6Daj3aFIgQ0NbwsK\\\\\\\/L\\\\\\\/Pcu2xEoumogICR8i+KCbhxNQmHdKEyIOIJAxpZO4d\\\\\\\/avA9RNg2UduNcPA5uL19+qt2TIgHe1dUC1MBEYXk1sUKJVBalptbKKDro8jE\\\\\\\/sxa2Sdm2WZH3\\\\\\\/aK+SZ4Hwg2PNskhLjAN47aQpgbx0VOhnDh7n9SdKbmX9PmgjuSD5jWyOZQAL9MKmAtWRSjstF+J7FibLiwMIM\\\\\\\/+kUiH0PCp7D4ih8SU06UmNpN9DhOJN9VP91KTUpFGd\\\\\\\/3TD+m4KtlimoOlIQNXbjijbE2T5qeHtqMVmLRgXVW9hCw\\\\\\\/B9FmAWqiiaqcwY78guk+QkI5Qw8hqwgFIRAobd5IhGMLJhtqV663Mx9mLJgSf2L5ezYXZwm3eVW+UJmccbuTsRgqLejuJLHRLuSg8huv52fQobbjVF9V2zdL5CY7xx6onEyrzOzf6WFx5p3U5B2KhJvEHacl8bD7Ay4BcQ1jDgzjPtxTw6fSmXP2XkIVjnyD95grfxutiGYUJVKgCnObFDVCo\\\\\\\/NU\\\\\\\/Mo66AzWSAJp6ZbZWSy0QIey5vfWy1gZSXl2cOqPAipK5HR4Ig3h+kHFXOh5\\\\\\\/TpSE307bKagBrYT8O59\\\\\\\/Jeo+Jdo2uTa69Q3VFDzmTzd1yta8OuE1MQ8BrbjeK4pF5U2gL2qm0Zx4Jz1QM37KQjGxGGYs8mRcNu2H2sX2cMRQyxVIWt4H5Amolv5x9bFzPTuvX9jiqkDuANCnqKCJs8pBMi8E3hzP8NUIwoHcDF8aQI7Z3\\\\\\\/gfmSgvc8KbtOGrWjhgEQgnHOlFK\\\\\\\/mXMcfAtB9\\\\\\\/6LCbV3xuApHhMAt1RYRKzrXZunAeB1rFB34KVWK8wdU0Wui33tS4yzPAxUAaik08boNDFofpGPblwUMVFpa2REZMwJ004IJqd6lAkwYEyiwJ6nNEUvUHfU9SwbE4D27ZrNeLuSW0um63wpRFVRbX+xYkADxSUyCBF4M9iDcXdFaIgzVQegvFHOoAYAYX6yI9xR0JbjTprOWJ7qmuSf\\\\\\\/s3MakNoxghnchI3b\\\\\\\/ULISBGmuSTGNVNxg3n9xcUqodDTevT4pSfAfSAqDF0975iSNUgk7GLQ6S2nYHLAB2c5izziqwMP5HEtYgqHpAOdFOy\\\\\\\/VnVsolfxBMdXT7Nh+v8cgmbTDrqTztkI0Mjb\\\\\\\/zMUcSeNSDKbWM3Y+lJ1BoCE14R72rLlwpbiBo\\\\\\\/hJfC1HYcKm2p8yEI4OR4YQgF\\\\\\\/bhmLdHm2BtTBzjbB+625Ft19LoClrHitm++TaJnH1Syisu6KyvdAwiOmp9SHYgl47ZpkAooeqBA03CchwcIVGxjM6V4KIn6WPqnXjIFKEMgmppsVM3ZTcdAqprYzYXhkX+Wzjz9i64N3wKbQboASTpz0+0vUk+v3FU0s3YYcT3Rj5Rc1uci4U8jauKBpeAIC2T2eg8OeeYCZ2hxVdiV7G0Rro\\\\\\\/6Z5suFlDgxYCK4wd6vY1inb0jZDtHNpJaIs\\\\\\\/bwsEj947Ee25PO1OPYmTqfqMSA1ZcyPU\\\\\\\/7m3aauXqPy2hrcSRDoPKixFh59rw65ffBxPkLjPUJ1\\\\\\\/mn2mASjmuJpoYnXIPF7HFUxdmUmHNm7f9pg71PTEG6GDsIEmHvb6YKShNEqKfHosiYUMP9SpsIvY4Drtw43a0+9xyG1Vf1gAbh2Bl3iANvsZkLjdSjVxWGdjw9Z4THdmArTZG6MPsX2sIZCC1q83iELocDjj0HNquK6rwZEQIO92ht5JAiFAhl\\\\\\\/br1pz5l5XhQpoe8gA+9vpMq9liEiwA7akjrWYDbbc+dppxAzWA2RUfo0r3cvixNcsg0vOnzIIe4h4uN6aXwVmX5X6ZCotyK4hTFqMckkleiLG0fYM3RdthKuelcTYj8NvWw\\\\\\\/u3KAziH9IBUsxOSQ0En18x6s1wIVxGFqoSmijNdImRD\\\\\\\/GE\\\\\\\/oUA2mdQixLwFhYJr78rY13FzXz6xs1qEhzR9sHUH997QRyN319cfpcZCVkpNp+MJLlPDq4u732jSaaLoYStGQDec37P8zro9PyRlnA8m01gwnFUuLhZGpeyPV4qLCl059iJGibPttUZgaXNEwqIJlJ92WwnQcCe3fTom+c7nnDWbbqC1\\\\\\\/f4GH81BTyaclQTQ2OL7H6po+aqEzLIPCluodQ0S2tme9GC6Ua4ru\\\\\\\/W47AKJzSoLMoKOrU2WlUSjLorOI1PzhREb71ZlRsFbgUkjdOg9bFMd4eB8SrFXH4e\\\\\\\/WpX6YUgiXga+DKOjMZV328psPNM7SUOVpjFsblh1ap8UxNop7wKXdWM1TYYwcSf\\\\\\\/LZ2SpmCUxlE34stPPspQj9j1vKUkVNKd6wxtok1nadjWvdgnogemHewImfdZ6NL2h9288OyYJuMu9bcMSojRT+ungfATSK9uPNTP1jYfFdkY\\\\\\\/oPjJK3aIA0vGtKrKAB5ZA2i8JKEnym9zEikWPUe4ZJQsXWF\\\\\\\/\\\\\\\/tx8ZsL+7Hf1QS3r5p75YrrPOlniDJZ4+CeuJlVg5yLZyCfr+psir9po+Ku25RHaXrSnmnxHB\\\\\\\/OQcI1FiDY1puxq1lV8BnuEtguCy9C6+W2v+AQCUXBV0Raft5TXu1TCQ5Dfh3cQjg1fSo9jRs\\\\\\\/INiAWRTrJCxCulR\\\\\\\/7EuTf2BpEVGU0HPa12dLzn\\\\\\\/n7Rg4hpxb3OpxUqwO+6DI3zfXTXqkJ8apLVJodAiXPScY+cf8YwxX7qIlwGE1xuqd3HfkPk60Q3Sx\\\\\\\/Y+UKZ2YQrkIci8jBfUv9a\\\\\\\/Cqqw0nimJw1BdRkS7sTBQhiT7Ml7c50zKWvBZI5OvwDoIxK+jj9i0N3N5rnKnjfv0\\\\\\\/i2zSr52etUI7s3Z3kCMqFSTn2nQXA7VI6IFeZZKGjsgbw2QNtvwbLjYURaNrVFsmBx1s99pzdkmpKVFKK056+UKLpsoIXVtaVWiIygEtA6vHxpf0XOYrM+Ol4epEpzCdVeDl1qr9yPOsEiWuwTJ0cTL5qkG7DFIGtUBWIiRdrVoyheNxXBfFOe+XKZMciyquETnIkdOUsF47r2n2JN\\\\\\\/I52U+NERc+Ir5L1TFm6PNhwdlLkfKJQBcP2Pgb8lJOV6cDCveOaIclIia20nHykfKEBhwIcaz7daGvqyt1nR8YOq+ke3x1bwkMsf6R1ujxt8KKZC9JhRuTK3dU1lTvST3\\\\\\\/2WgwLko5QbTKV3jia1JGOKaPLmPg2EOOQZF3VWRk25\\\\\\\/oazj3IdyTAx41NEdtDfdl1OR7r4c86nvlHkVHURy1INxtwFSYl8MNMkGfOs8EUFGqx8tEBQwAzwOBXJKuEZWJ2KQqpevj9TYBiBa4sPPY7LAeQNA8YL8GsgsjnwA3xZrCbR3GqXzz+paceTf0PRLNO+66WqUIsK\\\\\\\/svMGe\\\\\\\/QjTRYKYiqCZOVSNHDt2akqENTFiuCdC3SBUDJHqs7AnB\\\\\\\/7iRXoRDwWQKeJXHsR6vuXVQkuj5QKIGTg4E1GCGxv6XfAoyuKD6lNafIH86UPYOkrjFYFfe+hkxaV5CPHMTqwJMYkkwyk2YFv+nhfgI1rFtp3i4zSXWYjPMCtZeTplyuQTs3ofcWck4c+5iMpdT7QvdR2fg\\\\\\\/nGvs7ps5+Ke6hpu0+OQbHW4yeMmlDr+nOw4kpVbw5spPqXnz+BQX9NwJyYJQSDtu1Uvs+CvY4DTgTrZ49LmmSlgUk0OdsGJPu54M\\\\\\\/JQWlZfOlGBClFImeS8zdSY9Lez95vsIyUliVsQ56bayr0g0fRmKO2+W8VALJqCxriEFL+ATG4sHcDqzMxq5UO\\\\\\\/83zhF5OhipBjwfnYiOKn\\\\\\\/OxXWgmyIx71HNYhu8uORTspVx24Y2I+8gPn2hsuTkAQ9glh1A8CF4BMRe1LjsIvQpxOGBTf2F+j2wEQqnyK8ObtdmN033bxk86pRLM7aPIf\\\\\\\/kjRnX9FzVbZT7IU3Mg+c95lvBR4\\\\\\\/6IqHyV+CsR2g4XjQd0k+uKmytoNZRbAuGjaIkykBrXBZdo77Uc5HTKyBUOlF28LNIrlYo+jcZF2fHJgJCGBc5Ra9Ku1O4tiH8AxeyJD\\\\\\\/1t864Mdhfl6GifBau0VgSmsS5n4nRBrqFsVt1LGjEqcWOfQmuIyUpecq8ykc2Z97t7Mn5m1HppTNlruSYzpU+a\\\\\\\/1xjCMY+vR8BZ9y6UlsIl+M6dzvz6\\\\\\\/N402I0wN7Z7I+K7eBVPmpGh0w7a7vCNK6qKMcmemz7iJ5Q6iw972qxoEZZt1uQvpGvIZ\\\\\\\/pT4qzTeP6aIQJ1l2g+09aVx93ll0bjDFFlGHP2VQSbDEVpBTIamveCmK3rUm9MQYu1F8q9\\\\\\\/sjKPgAtBcq\\\\\\\/9IxeOvP+ebrOXsTfsyRwW4++bQHaFs3u9Xh4qx6kuzE3CdCJaeMtn2Bb1u7rjti72dbpxFD8L6LMjmjVwTf199yDNEt9V54OW49p4w+9TYiMwRQ5173bznbx1b8pr8ZL\\\\\\\/\\\\\\\/OsiaEoHXENkdtvFN1BkZYDP998rQhJ+pfulUM46rp21lD6XI6mbmLEFNJwTyI5HsRdEuaKOUglcpSIR9d1UStCvTMXbegTvkh6SnkqNd+xRiHdfyixu45YGRuJK93cc\\\\\\\/1Bxxy9sGqnEVrXB20lNmo55xay8DxVCTKsxIHTV9rne0hmVAggFI53AJdKb+WtQo7pF6bRzRguUUj0N7Ce2l1Co\\\\\\\/YL18IEh6WoCRQ3+9d8D2d9wrLrD+BghI3QnJ0Ww8PgPi\\\\\\\/fxeGdU4dOjJsxJ86rBjEe83T4k1C5aO3WksMhBwpTEBNVq8U9kjhzJ4sRG1zAbAU+bU6MYlTU7uFJsfT0lRVx3a4BXihjQjkUo6Vggd2hpGFMS0aOIi9ycDJGjJBLc6Iqo8wk8zmM3QvdqbNI2Z3BFzUHS05SxjUyz4l12IJbp857y4g1sUY1YyHP8cnWKk5zNHDP1JltDzdOt63rRpOjNDNUrUj9SUb+pPX\\\\\\\/6vN4pxnd0DKNloWvVmNmUg14f2d84iMR5DBmWngMYXWxqIggPC\\\\\\\/G9EE\\\\\\\/OdQrkGnNwYUbZjp8PynUsY7+dwDFF9Uz4B\\\\\\\/yxfG96yYVDXkRYYYJ8hk2UuzSMMF\\\\\\\/hAR303FTgZdJwULxjhvwskMEc6avHGzHJdy0gMPKRZlUCygHpZdYs47QEb7XBYQXGdkM36dSQ1fzdNShqegNDbC1D3YESG+cNL+9ckJsCKNlTHIowifjDyGcEmwXD2GZRjaeu37CxkbMxfRSrKaV1M6G2perOqXIEf1kdOl3ECB90Mrjriolg40wGh+RtfQV2dNLnvRVcuPJPmoaTvNuI2G5jAQ6JmFGuxfiGVhI3LJW9nj\\\\\\\/IDRNN5RJ63mekKgsih2+QQdh4qJoV2LxDFl8itR20HVQfPoVFYC+girKm9uAFYR4M8sCzvuldNRr14TLjHt1TMUUfw7U3UaoUDQ98NzD\\\\\\\/EKuRIkpIf6a7if75zVG5nqdFhEv0nlJEzRnu9NRzHaETWgBrw35q0iBwr8sBSGvh33\\\\\\\/7nnQKGI1q33LxcjWiMCEyNhT3aRNxTRJOJcVUyf\\\\\\\/2R3UDu2GkuwmVRo9cr4qGu\\\\\\\/dnwu6YldMdkJtwJtDU9x1EHNeTd8wpx0Wr7GslCCBOfJwhTmy9NHztALgctXok0HabZ1Pxd0xXDlPze5CB9CLoCzkenBPEfZWdszlIoQjLwR5lqXXrWo2AljyceKu\\\\\\\/jp0Uecrg2RupQG9rCqSVHYmg1aOmg4DJ1BLsRzgkrsGSldFA1q1A8pV0j4bkbiC6YRUcZZr5TEQvftIvS3A08QkxzSSglwKhe1SPDOJFRxo1lTSNJ1+LK3afjoRtMrN2LIYoHeTh7dQtLRM+bPZ+\\\\\\\/HwVGoY3sZaUSEsIdOvhk42cGzxGdcwK8T3WXUbwabPrYdHy4f\\\\\\\/F9JKoh6XQPxsEkhZOMvR8syyMW+JD9HKf\\\\\\\/Gg+JIaccVcWcwaZFfBjfzzZhjQJH4SWX\\\\\\\/Tk5MoCVhin\\\\\\\/7ojDQAfPR51tvcaIRoCDfFgvV481h6V7LPPZD7WZmw\\\\\\\/z5eZgYuFnBNA80+o4Pw2wUcKyImRdrPw\\\\\\\/iI6fI6jzNbwll26MNtS+ejg02IbQoAxHS9XMsZk6SqPxxUbJCwAPIiNclzJjjUHSloYUHi3BDEpI5Trn1ErwchiIk4OfN0yX6JlY85k4KqVi3kZXQ1VOeNGbDTXpo2D6Q2kEnFb11kZK+d\\\\\\\/YWbVewVyeFX0sW8B5lKKIHusmKdiHWvnVGP65xPwB+fOpZTkeJ6oSp\\\\\\\/UW3ehJQvPbXRkhfpfpGvtchW7UIMP5x7Vkjg01BIqmrUIQY3\\\\\\\/gQUstMn0KAKY7HV9DqY\\\\\\\/qK2ZmdWcLjM94OtCprVcx8AzeTZyeWXTklW6+UQoWeugfio\\\\\\\/5EI7Nb9odc+W+jKfJGE5dQyxswZzpVMUvaDZsYUYnYlIy3ncQ\\\\\\\/cJFActrFB6zDWru2p+EKVe67+UoKklS9tGONf3GrXlWM7gi3Ejc9htCGZcLHdT9iLbeo8EUrpBUAZTcIQZep8y5mI4H4BohenhakUunuDHAeyjEcfvyop05JWBbr0sD+L4p5xXHKH+qu+51lmUigQn4qWXP0Ym+Du+RsFSgHUvCx4odhjWeEExUYPSkrOqpIPva1vz7bh34AWtnwa3fwZMmH\\\\\\\/IJHLL+3xqjpe8nMNOHjb9a9qtpENueyRTDzvB7mtFvgXyGWnYYDx7TJ8dt9mtVWZWHOVbd1A6pEhDDkRSCcih+2+pesuBih4IMpYT2\\\\\\\/HXII9CvN67Hh3nU6GPEbh3rstr2meX4\\\\\\\/1A2hhk3YovOy8Mb9lJ2gpdUnneHr5E2e3uWHAWWYFIusjCEZyYzv+HEw\\\\\\\/B5cRkIKLwbLHu4vfcgF6JKZLz79CyBBXqwnSedWePy9gkjwD8lMsuqiQJlmMYD9K4hGfel8gVjZA22HGpRYkfXFsM7n54F7MsB2+HPIgAJsQlHAhuVzpWdlafIGKBVjj2Ruxtpj3j3BtLoFYJS9z1Jk34NJMhAbrCIEjLC7DTV3MIZLekwaVKsBkpwBg2n5ZMs2Cn6yAKDOLiFlISBc5Nt8GrCtEtDSH6GsqW2oqJuK64+DhUCEB0CCwHV0euIW92FGj+wtntqY4CmRjhBBriEZqZCvb+DVFJWda0VEjccLQ7iud44m\\\\\\\/ZgJRNIeohpvRj7p5eksS1EGD4npB4nyiCNb4pnMJEgHe8Vs1emTvoYdFt7yg3ey6FMAlY7kMi+2e4572w8E9tUmfUruHdJ1i\\\\\\\/Nb792YaoowItgX7grsIlWpk++DMBp41+gYtZZhGpdVBiKk6R8yi16ioGBJOAU6CrD21fzYpovTYyHx9IzQJOAqOSG6EWRV2rJqG1FylEEK2gJe+RAHTmlJqTGLyMR+7+3\\\\\\\/s6pw84\\\\\\\/s2YvjjMkiriImk36zAiBTgN3U1pYjpprpsxkLKVSuD2fj4o7e8XM9kXo6x0\\\\\\\/tSa0xst7g5W78xo3EFcTVE9Czavt1gVaNMXpTvxw0oRpYTt4mWZNG5m6IuhqllOjAOKjtwZ0gBVxonT92z5dTMtTwtyLcxT121UZA1c4JSRsd0a1JMdlQdyoFeMKLYN2yS79e39\\\\\\\/vCvrSuzCuYpNdpShcLCGq1GFxaREPF9GS2YVleYVW\\\\\\\/wLdjaFKotX5tXjsqmHOOK84CX97Id0ptjWNPug5Z\\\\\\\/VqhCJ8o0Tqpob5z86J623C9oGSCNyWEOW3zDinxGHQDzj5VjaKo9j1vN4bPOSbAsXNY8Qva71WmQhIc4bIDOlJB8RTuOmfLnXRWhojZ0b+8gOboG+21V+BejrkJjF6AU1aXZM5phc7qVFAUD0E6YURhtTFagx0XtQwBCrLxX\\\\\\\/unwcWVs\\\\\\\/VZqFDCNzcmmL7AAj8zLOk1hJDwKb9qGdDo6id5+od6k7jdvqEZAcmsIfu1F\\\\\\\/i3Hj1tJg3vjw5nsRRYiWY9anZDBCjUQdYE0HjGjRQS8S4Osf8mx1NrDVTQyW\\\\\\\/TN2W6yP6rxQFNnDmAt7MrVGpCT9IU+0U+jQS0epWdigO66obmnBTRGIbibssTHKbKq4UXO+MyIVSk3NdRaXyh5fwdkzG13\\\\\\\/13QYFfd2h454ICasAsItMoN4ka3oLK7fXRuzKF5qvOwgl36ksNCR9QKjdvV2UHh1uDnpxYPwFZX3CPWj1tfGjjZlP6v8cbgdFwMb0ROPfBpXVAba\\\\\\\/Qqg2M6NVnRTwURcr0UzlfCFafhHTb44q9C5kyjoSahLQbPmWovY4vosXW85rZQRFMIceQxfk4+VM3glkbA3QoOYPJ4wOAR8YKT7SoyjvPR40A72MA09+tRh9gnhJhEEff2uEtwxhQBgk+KOMqvyf3vHdZvF4SdTzCa9PKi+9gD+pCrwOUfqjDEPeKyiS4H7SYsx99BuSm1bOSBpVMJuUxATNicBAaYNPKlNQPm9crCxvc10+SLvoRizC8l4uYy4VekJJ5Sv2V+r23WKpzArWGQv2\\\\\\\/cvy6xtJSz2vuBn4PpgREyYrAhNWxZKmxySdLPumfHU+onyxKCUioXg0y7cen78b4rH2jSX2m5Y0mrZau2UfEXUi8Q8rFjZmOd5WGbGuZ5Kq4j\\\\\\\/eeU+TCDGCJ6EBZ\\\\\\\/T\\\\\\\/XgeKQf0a+Rv4NR5e\\\\\\\/HNlcRm9eVSq1njxah9oxHzXuqcsnhNhcoNzTuoypbPP5wbo2H8pCR5hNbbjogwDGFuFl1kYZpY527P8F2ohCqRB2gOg6zENGjkDO\\\\\\\/RwjVGyXJL3xHLIJ3PoDP\\\\\\\/r6Y\\\\\\\/YqwP\\\\\\\/J9tBJvO6gnTSwJhzjoMQ+5EW4A5RE3FAC89Fl8aaUHcnHGLvxCpUpfRG69IFv3DZPpu8zUUFdv7byM1CKNqR3QgWip9RvcONCjtmge843Zjuctj1WUXBV6DVND5GdYC78O4JeYo2GuDHTljIa1inV6Fa7tAPSaKPcpGAvgaVNd1LahO0NA2TqRPQL50uN2vFdPC+m4TWhu5FjBPxEveRWm+Yly4GPTTl8BXpaVlFal2aZUbxwzhVoEI9H6UrgGcd3RofoapKETvj0xe+soPXHADKImIGQYx30OFrgCs7mCclsHJCUBWyS9fOAG6ADDyfUOJ4G5NGGTekKwHqWviYdAjM2rFYVsvyePlB0EkzjRwZRhBHDReeG\\\\\\\/R7hkx\\\\\\\/SbTLHZ0T4B2FCLzfJRa4thm17NvA5A2E3jBg9jdXCblumEfbJFBjhZ2XkQQwJX3vdMzBSrE8o6UCZpcXP0e59KjX6hbCSvmS7VHh4JLSOYhzRE52+syQJXunywNJI\\\\\\\/cuQdsY5DHXKsxLw2hxVKRifAYup2e0y\\\\\\\/ODCMZwKd5npXm\\\\\\\/AHU3jN84JSuxIsSy5z9b305ln3sKjlLrCHLMuXWwF1HDzDMIfnI+8ZxZIWuM4ZZBrXe1277tyGzb3VXrqTSNLHEzpO0y+G7eotlC4wrfla7ixTPTZSfkvI1uAJfrDVBcYjRC6KjCk1aRGjfPDHG971FRgUilxiuL7AOAstqg5vvhLVQ1PlnDjQ23izWXhXgRy2SSDQuZcR+WGivCYTFG7mCjaffxT\\\\\\\/tqqNj28smqohIzv1VtgKuVTvUCwEPT9EffdmrGN44sSUphGj\\\\\\\/lGiOmZgHB\\\\\\\/LTAimYIhPD2Cp0V3n+XyR6XkTxETZn770AJeMdf9haAlGwOGuBQU8hAmI1m\\\\\\\/OO8vIJS3bGYVvn\\\\\\\/M3jwfbc3ZYpiU6tS4k2T+OFJwi5oPnU266631RTlAYcDSAnAHT\\\\\\\/kzmaTJRq6yccb5segCIx7G+l2daNKcCkScvOIIiKFYfTh0qyGUqLn+2FssuY3mUh0AFCqZIy6Ol\\\\\\\/F3PrAhOVEx7QxB+YtPTRmp7R2gcJwM2AxBMe0VMtlrZxb1fa9PqCYyTiRjsPUHPzX7Oyv\\\\\\\/oJ+G\\\\\\\/F5j6M7ypQlLxinfRGvrz8I6GIFDai9NLVGs8t5rlrDt\\\\\\\/zfp4GNiU\\\\\\\/yR1LUNkhXoYJGkt+vXzVwzR3y7rW9VxeCWdh37af+Oq+B\\\\\\\/rOGTLjw1UrOSU0nP+zaWcoTy9x9e6HumK+AwtwyHEhw1dKiFySHuwDARl2dWvNOs8oVV7p1jVcw5AjHJQbJS2AlBcoOh2P6mJHJ7gJOgYAzw\\\\\\\/KzJyfNwCL6kYBNke3g7ee6IHq+0DljeRHgEaJgAJuE6s2JmGf8M+Da1FWnP0YFSyXIRH09VUuj37\\\\\\\/6goiliXXbkNUipQKNN4cUtOoUuOAVZpnVIWN+TRCU6pmiDEmZGbxTsSGRkohRVS6wDENzOHXKS+CfXK7pIYdoAY0soDSX34CkdexmLdof\\\\\\\/2cWj8FQZ0xXy9w+91RS4OSKizpTyiCo+3Aeal8xdbclkFutrrnEg0gZfloVnBIz11NbMtkmeKLdUy45SCjnzixwc6cpL9jsB4hJpvv+q5I9abHwvbiZ2\\\\\\\/EV8EQ9Cs94MH9GBo+OEuunYVt8mvZeRVDfuTXlrSbvEHtwOJX9SbIQ4MsFm0wFAjeGCrBXezWBAP75Vl9P8ryKzSaV9WGdz6WQFn4BPfYbsGMMvziRby\\\\\\\/Fn4mEH8EnzzqYgn\\\\\\\/itEpuOZ7\\\\\\\/CEohF5j2E\\\\\\\/T4M714c1KVk0MP0cXy0H3OYQ2zFsDQDhOq41oQoLjv8IGo\\\\\\\/eu0bTNmgF0tp0O+7RrPMSJhO99wbUB3u\\\\\\\/YOmjr11K7WKtnmsxDlO+49mBWWXS8EvivQaaUkswuYHd0Ti1hz9HDfYTf82shKX2fviTvvtWHHUtCEAj3yz\\\\\\\/o43eACUY\\\\\\\/VyoEznlE4XaWNcjUEf41iSpexFGC\\\\\\\/PwkbIAsCzXNFIdDGOrN2ZOss\\\\\\\/K8x34z2bVhyUsGYy9sIgTZUC2Ks2XyGx684l1wbt9fJcLycRa+FdRPbhX98RJX\\\\\\\/hHBcxtyMgiyfcBTSDyGbsi\\\\\\\/VDlJb2Ng1PZQIkN\\\\\\\/Eed\\\\\\\/2Xom0cSa3zJMQODiUZ\\\\\\\/iFLKqr\\\\\\\/\\\\\\\/43Z6EqB\\\\\\\/y\\\\\\\/8Mwb8MnXc+2reFwCP7LgMNfi+LkW0YhFhatWaiouV0Wi6EZZgL2hREspPpQD2I0+8w8N1fJLXsr9ljtLLuaKiRM49kYImNeFxdmNKTk+PMIXt1o3S6RM+PPmZiwdMgjLkdTTBPkDWM6OxzX8nk0SWx2tepxgn2HicnEH\\\\\\\/9XemcxOqBd0zsGwMB5hWJ7PZIyLlVYbYQjUwbvEQxePUKdgYvp5q1+ge9DGsXS3yWJki+VxJbkx93xaqH4tuHuGhAksBiku\\\\\\\/BIFWh95YddiMWhtlHWSmR6WZBSyIf1qfxgKxpFqjdIgbvvrG5uKCX0La4fb5DimMQkZNf+mdXb9Ebgh8Mo8ct4YWf7YpkiAQuNl+PBDpY7biESU2Aqk+CRRoj9UMnw798FJdVMG6I60evQCvI6jwg0ZhkuC5Gg0CUGafr2XTCEA\\\\\\\/Fq6OTkHtjJY1bAoZbEXG+XucfSI0lq88WyOfsY\\\\\\\/XwA0Nzuhi48PusFX6idTbsLCnF65yN\\\\\\\/Nejgnfyyb4PBjjU6jQq7ub9BmKTTD\\\\\\\/Rvbz6UErjCHs6SA8Kfi2erRkXdr7kX+kdz6DmE2e6khaet2vFXCYok\\\\\\\/Yz3hGU1Nb\\\\\\\/nH981vW1GsKbsBYVqYvNPn7mg4omSfFVa2BLxU9AUhiW3A1HMsrx9XMrPEViFPTMD6f1t52iD8LOSsl11RfTwBW+hwozQjPwJ9ZBTEruqB\\\\\\\/5iZ3DoQ9QNvzAm9BC+nSmUKCjYHcm38kbULeMYX24MZaFnJEv2rmcz14qTYCwAm7TplhKs5Sw\\\\\\\/Hs7ZGShwFhpnJw2oGVWWe+RFtQpsWoYEHUg5HveegdNvGD\\\\\\\/o9OcVoNHtFb5EMW4Bbjc2fu0x+qChIDJG6wi6SSJk6gR07FtRU69S+oSza8K2uVaGDb2aONz3EYbSSj8X18Ftknr67diA5+UqJVyZHfGTiXAG0lCuYt4VU6M6En1pXVn4APc83OM\\\\\\\/QWJcmsxbDyjFvdhbXCXWJ8UFPwLZQwjsqYR720ulbc\\\\\\\/qQ+rgMM07+JRcoS7MbqzTBJkEdowuHVQHA7TxAipEZGZLEuIKUd0Vs0NFdOXtnC43cpsTyDoDeUg2jvMuYNNj10irSV81iBz8hERqALX3mFDHQ7g0eDwxpKQ\\\\\\\/XRy5FIUxbGtQRws4FDEF5rWeATLM0I7doOXKzn1K413c78Zwsf6GETMjTNyk0BOUEq21R2hNolt01sVXLdkkXaJzdJWTJ18jwi9sWS0JSm6jVVqLik47BlJ0qsOmp+3Jovt0QfAWNXHsclBukTqoBNfZESU\\\\\\\/8zCZiS\\\\\\\/VBSFzeHAcx16hnCRVAdSZREyqV5ZsZWpS4qvjNgLheDtqPDGFfvAYzkSXfTpgcBgOh8AVpsDFvCgtdctd9mbPljkkNf0nEX750C5oKwaK3GroCdzFsw0c\\\\\\\/o3fCQgB5ynQFAjO6LfFXZGHuOBDEdRxn0M69KQ4UROdypdkfpK9W6ugWg9ZlLu3a3zdaQag7UAl2SLAmD6a9ZoY1uKCSOsprG\\\\\\\/4m3jpK4GMmVWfVZ6q+5TlImZX8R19TJU1d0KeLtNeUTnUEmS02QdhnY9SauEyqrYPpiKOyOeHJd9IehCymDbNJzfRVwOTfQetceb6117dLY7ISk0TEEDt+R4Iik9mM1qExH+7HHmB36y0hmhs\\\\\\\/YM7TuUW1uSGtpjRXPMxyWRzWWre\\\\\\\/qz7OTMMRkOurpKpPb3AYXqa1Qkm+3+Ld1LRg9M+wZkS4akpZa9cHUJmmcDyJG1T6aJppKaBOJMm9ad5sRuOzJIxfVEz13ZoOFgteVFvvUF7Cg5oMyrdP6xlg12\\\\\\\/gZfpa1j7jhI3lY2m5ZL9kTCI\\\\\\\/twhl5ZAZ\\\\\\\/n5qfcI0wJkRxvb\\\\\\\/3lkGHaSw33VvEmdFKm8yzihP89iH1of16ety3PYyoVb4jSDfyJFUUr1upLFmROjTYWhXw35gGiLKziuhDnfhK7X1LH0d5XdKajCL6kKRR2xnc6sLlVyzuReYTS1E9J1zFE9VWTas\\\\\\\/orsDBez32W0TkHb6n5b2h0nKpoE91cK+Z5zLOh9awZC7DxY4LZ4A\\\\\\\/HQGTaZKA75tRJRhjpeozw0E6opCI8WcorysRS1labkL\\\\\\\/nKNnp6CgeY8n0yNtVZYJ3mSRRgHcHvBuLTZy0Tna+KUPSS1WrPbFpmpnN3hqAFFBPf5fMcH0o9Snw5z8eHl34p\\\\\\\/UCM04mLpzhJCCVA56U3tT8GIfYnQkNsxeD2Dam742Nr0wFyjPtCLLI\\\\\\\/agDx4WY2SHiMbeOA6+L\\\\\\\/dXGMkEC0q9WGUSnXRSJONWsiQHF46Rz8U6LQ6J7ibqSHB0IVrRL7f0YP+ea3cx4WppJBKLizMb+1dJ78JLM5Sm6YyLc9pIsN5NHF3DTiVWUur1JxuOFrqnNAWpvZQVvUp65nbQ7p49Ks5IdXAM8L86yQtBLsB0AQrKIKiHfQ2HWklVY\\\\\\\/1HG422GyrKARKFmD7wHaa2gKK43Qh15EP1FA4WpBE45vZyQdiZ5AM70gyNwaU4WqxQHVmxSCk++c1odXqUhmxxceCUWgYHWv+vgno3t8Ybt0dyIKVP7t65zUDKYhLZV\\\\\\\/SsGFqHlhwhzMg9UoZItBg3KbsN833WMsQ17RhedeZbTcXuRmevCkXQSs25jjJr+U0I4Ma8tI\\\\\\\/m4BjTbjvYLfLMg7t9K\\\\\\\/IWWufcBJxcPLrPxBhRyk9vPZbI4Di8uRRZlOIZYRXtj68VfdI7YVOCoy0gNIKmWS8HKN3mB+wiWbG0lOavfJfAeBg3UJxcIGESKNsRFOnGczMj4\\\\\\\/ZjDG4+AUc0k\\\\\\\/vFhipAtljUk2FgD3UdujR+oJkX00VJgVG44zm5jxqRBzA2B82XG2TAO6CaL1pCAFTTXCzk9F11qQIrGBhemYQQl5ucq4hpK21KSXyo6r7ym1lH8IXPy5YCCqO19Ts9pPz\\\\\\\/se0u3imJKbs0W7E420pMCE7PE1d2JL\\\\\\\/auBd4dwthnvOS3Bv9J356bzbuQH2fQd21ZGTw+deQebZD5oRjH1XKJ3JDBVlfyVJ1Y52o8Yu39xgdh\\\\\\\/qq4Bhru8Gjyk2P92VFhMEeyV2zq2jMqUAQLxs7gL5ZQQzBQRlAv+68FBC+nkPB+UQHoqYEJywdmXlp4ziiwgfT9CEqP1Jd8SnTHaVek0RIVZxoEZ\\\\\\\/1bvcSmVgaonDZTYhYeZN5Om2Wt2TnyMqfdc7+MQezvRW3+fqYDy\\\\\\\/T5waNhuYlEXimoC6qI14vQeE4qImwoZIfxpvX5tblt+SIGa4+O56XEOgoQsSk+DKMhiVqAh9GD09HaoOOWEhvsL1orIjWNeJ\\\\\\\/p7Ma94HQVMwSAeODROHGX9Htg94O2+LEomhQEstLElD8erYbfElwHh3rJoWR3cO6C3uC\\\\\\\/jIrpRTAeUNFhv3p4VZUsQMkCgTO0lcami+smax90PybdLYMEehyB9A2PCtMtGEUqfKWYgRmCoaOU04OPwtoKlQP4iqqDfswWmWR5ELlmRM95dUbG4sFke2llTbrGlQRqWDxfP3YGZyQPSvjxs83wbo8uTrO+DtOSl3zpzcu9qDNuq\\\\\\\/HWpiRdM\\\\\\\/5DV0K3CkzPUiY5rvlLXurAJQNCOGqu1VJ7+EvZ75zQ03N0Pd3moHz5KRhRYsf0RW6mV2d6tqTpJoLq2Lsz2PpF5TCuGlrbp\\\\\\\/1PyQGBbQXtg3hdbFnPjQGePrFyibqlLn\\\\\\\/q7rqla2NDdpfBmz1xVmRdaTMbIvY2Y7xT7V8HcC\\\\\\\/Ah1LCD9XSiZjPDqfC\\\\\\\/5FJ7yUcBjcU4Y63uw5x0NOzbW+BtLUXwVbclFcovdN44UT1Wuk\\\\\\\/E0fk5Ohm0qYuWKDX1HFzxST6xAtBfOqri9LwTszrffSK5ru6YMe8eFpjQkUM4b04tW6jRhWBW0I1bcEdn+M3EIfrvRcW9P0TaPmk8PY2X\\\\\\\/9fYr0eJ98p+B2r6YLPWQu770wLa+7XrljnbwMhrHAUjPVFAnilqbC9GKuKjM6ouWNycn+uWDZfAVSnOjiJZBNkIvRouDRN99y3VAUtF5D3IZ9kWV3xANQd7WK2pnDm1tQw2WyfXAw3tMzboAtL\\\\\\\/jbIzNAP\\\\\\\/sGT2FqwfHQUvlqsFeUPKCynqaFXjoxHK69wCHOgQI9f+Ld38TT1l0tniKPYsiVB31u1ER+SPJGGQpY4CkxAKbPVa+jvXBgEJgfVn3twBwuqNFwYrkT\\\\\\\/5+HSyfiq+0H7eXKzIe9SL4lDXFUyAkx6Jz4FCoOagmU3i7ehHm25geV5pyadHSHl+Mu4ko1g6ZGOOA4Nz7vpuhUZyxXtQk9jWalYCggQp3dwfnTzVfm4QVskmBnnbb90H8JvGnxSn8PdChFOMczWhw+NyuRrHV3lVUHSvesB5b1kLvzy6AL4z\\\\\\\/fVw47G\\\\\\\/3XW4rKROH5RoD3W9G6IFjKpwS3vvbz1nxX8NXotXNL4zSBKRLQPko0kEp7MkS8sFT4ACLfYaGd66c0lUlYsa4caE6FpRsnbwCg1wGWvNcJ1VJjnNtI1Z6qCrqkL3VSv132yQUYAXYZKMOFOMOtpk2udWfIly9S976gmb5MNM77EkQqSJ+5Y+uyS5ltwZB53WI3Uha9sMnqEZoixLZdlv\\\\\\\/VltNvNgZCOTl+Q9xRKcd9EjDMYUtqMclq1jwerypljbvnZ9E4BHYncuoGvAVp8ubD8f3u68uya9BZtdMqf20ntMELW\\\\\\\/uh+vYeid0\\\\\\\/qn0w+4ApOuDU64EINqYISWCzskxiTGlFuK3VEQhtFKckR6AQqDa3Le+3a8zopWyFYR9pLdx\\\\\\\/+6JeyTCfTG8i+9tSzbcnFcWjR9\\\\\\\/WH6LnZ+FHIllVFcaCehZkbgUK7h1l5mjDBopy2ZmbDNjQd\\\\\\\/iQ9wLe18DGrESnRGL\\\\\\\/tUC52o9YLKik4lOaP2nXYkaCNyyJ3ma\\\\\\\/mNagdbp6jVqQeRn6Gf7XJ\\\\\\\/fzWwn2tGeO1GKzk0BSWM5Pg\\\\\\\/BwQgq9dMpJf2bUuYLsW6zKT+mc0RuUCTaSnMSSrsxSDq5yFXU+oxobd6A5GcIzmb6R\\\\\\\/N6+vOCRVdD9IWj7AGmGK3EDrrLCrTy45ytLWQ6K2L1YYgau34wtuqx0q3DaMnAj9zbt4lAL\\\\\\\/K5SFXu7dTNYSqsxP2zmlQyKmhPlcv9MJE5y5ffmSCaBGMdFuJ6o1TskjA\\\\\\\/80xy2j9p3+zdsVJ8ja41hgf01bqm\\\\\\\/+VJud5yEnyAGASTv9mVzn5FOVtEiNAEp6fEtPn6kk7pp8mAKdHfDfrf87BNiOsDVh6yIoQAyvlvCpfOdLuV+joLISw4iZgeEblo0lFEt70Mds2C5Hf0FwiG1HKw9FrW+1Kjihc4GKgNT3Ptf4imWi6Z6cGYc58SL9zD3kf9Y9AdSgevqVHUx9\\\\\\\/SrBb3feIE1K88xvihqLLjpv11EIudVZiyALRy1XP8zQL1hpe+Q8LpWbGOaY2Gr1KZ1luARwZYSHRQcFZzQdM6b8yuoLUdI2NhonPIu80CofCzUCYpHa3vQ6f+G3VTY0pTk84wKkrFqohiikN7uLbWqGTs5VmxazKy3KRbZQyLnv5+84d1b8RhTciBUlonXQFCzCb7RIcIOi64IqTkvtRPROrWVxC82cPsPJWuT\\\\\\\/W\\\\\\\/qVGqVEMZhIhz5SVbQ2Y90Zql0OHmfZM4AllzraZRJuawwOjHFIuI8Aec9u9\\\\\\\/yI1WUVxnt+8ygEyu+NrvDmKsnESE7nlnzlkY\\\\\\\/SZguvBD2NLiAoJ1pdzodhmS+y4WiH5g8+I9avdUnT8txfc\\\\\\\/zeQuH1KE0NYk6Yjo3u0mih5uyoL1xtJzbqsZsX9wFC9TW8JDUGm2T4BAA5NgVJ1fcSzarXkPXmMp9QcF1CXaqWfvHFRcO45dpXlgZ9+r1H1dRX9zcjC3qhOoMJQjbxrUxYp+CFf8ONAlVX+QBxXD7\\\\\\\/R4yBx+mqDNNJgVZrj4s2m8zDiLzZgFkLzL11LeOpGZwsEDX9wl5CzSBLeEF1OchtlopaicgFhTZVu7wFooVJTnicpDvpvrqVmgeoz7j3R6PWZnZALv7+SRL9EVA9FOwkq6i5x3\\\\\\\/P\\\\\\\/9f12OjdbfUGYLrAzrGjNiXvNzt+CMO7l6061zpPKUyz0Kx6PSxjpgKR9ApbTk12VXvyCwvfPBXz8pOSVydP87g04MItpUdLK4pdXxd6o\\\\\\\/NEexZdND5u+EAU1cwgp\\\\\\\/MlpU518BUUXtuBdra\\\\\\\/oAcNvU6wTMrIOeg\\\\\\\/3xeSyAjxp2emq73ZeUrDNc4XBPRgd0d9q2kSlYH9pj5iNIBhzMuzVdQTR7RbCbPEVV9Ks22XG+LivW8Q7I061dF4xq\\\\\\\/Wn9\\\\\\\/7MsdM8QU2Y6ugBK\\\\\\\/WletKeAA6+\\\\\\\/HWG1PTJ6wgx8nqgHZ\\\\\\\/yptREitmbcoz4UOfQ1bkAZGaj7+xa1magC6ey9IB3ynax3z4yV1YIhfUYiWKkv9Tow1hVZChTnpzK0l0x\\\\\\\/uq94x1ACgJF\\\\\\\/PDU2m6VvamX9C3qD4XHbGkcsI2l0T86EPKPTdgwbwG2mwdnB\\\\\\\/OVOy9rp9PTP37Kvar33+pXuYdB22Y7q9RiRhexB5WWdu8IlFu5zgSpwnCzUq0a1pytjmZl8LwR+t5UWKS9CTXamCpgFzaXpUvoRlJA31VUHtl1O2Q2ELZdB+IFfGPMA\\\\\\\/aRaw7Gpetzm1jUQRlyFTdAi\\\\\\\/t4+wPa3V\\\\\\\/GU7TCmwFZX9aIWPIE4C1c4xFC0eArNlzL4Qxl1huv1fqNk3Kf1Me23frDetOfvG0keKuAacTbtAa8qNq1d+ePEc1z8NB1gzkvbXxCi11pJ+lgMlJNr+uDXR755J2xdd3pAxrGT0L\\\\\\\/UMwEmQkyBoU+F3MtD25BCZGfTjIbQXZAfENuWHhC2pEN\\\\\\\/pm3sarrXJctudgSa4YvRwH6s9i+IY3lR6jlE7y1hGJUMKZ4mXIVkA8cU67VyXAWAy4T3YyPdpq4YxD5WECGfWZlL\\\\\\\/NEFF2Zf7xN67vux4ZUBT0fZSV\\\\\\\/5hX5+HTWBK\\\\\\\/K9q674b6ckuEJnMtJlVMEJPeo91s25BCPZwXR+DX+4sEjeJvO+S3rG4Rivjv24Xj8TokazvEWLdplJJTnkk6pWzerFsDhg6qWWJeVv7XsrcafkHqactwjQR8vrBz7rFrXVGRrXNVQ4OCkHXowlXN9K9hYE6Lox8PfTmgOQU39SevgGRs9W3znQbFMfv3ixrc\\\\\\\/4vwAzMAPIAsTTH2H04ciHw6V5BvoEhtYts4on8Tv3BK94ZnrYJVv+WG9HgjrlAywARQVT5S2x8JC4rIm5qeVV47CICKVvzdNKHn8exoXZQLwws4VF8LrHrDcnNiDdTwv62TSPFcyH5V6e37a5mgrgCGOsXUm1eKjh59iIo5kxQmFJI9v49xbYWApfzz7zfSSlD6Pz9aPog90AW4QACnj3TcLO28lK7Wr+eiJ\\\\\\\/9wWFmpqRPJWtb\\\\\\\/gHT8QTZy7zTY4N56FlOc8DfSkVyH8ZoegIgM7SvwqvV2EPHR5mPL578wNQHKLJ+nsjJ5E0JaMksBF+l71PqADHiAyfoSvLKKHhcm31VWY9\\\\\\\/QUIsg0dpvCRCf7PudfL5WLAQTghbi9hTa6gbvwsCzMEQS6HE+fqSBJ7y6G4a\\\\\\\/g6hKIowIqUvZHJlKD2ty1iQTtgfbAbkMYmw958dKpr6HG\\\\\\\/PwSqJqHjJ0muUM9aAvTb0Zs7OibfUaJoGQv69oXYfQ22Y6AGy3zqRc6EK481o3npIR+H9Boq3nb5CpgwLmUgY1iZJDve7PFNpcB4T4Ag91rRbs5mNF3ZA7g2vj6OHIqgtI+jjP0k2RiLGAbBKk1oaSq1dbCtg+i8uYDfDzO6Ij4yBDW9naxuEkFGKUnq0M21zlWuWwZfJTcUrgM0ilGx7AURKUkmJxaRbZSigc0ku3q2xgWfZMzlE7E7jM7pxZqMdaGOJH99Mx7KoYYtXblaU8Dp\\\\\\\/i\\\\\\\/jCuB7VbX8ARoHaC7OalNirGe5hmRwLQgTXiGYwyLfmUjk6jJwq4dmxrZQam\\\\\\\/++R1eyo\\\\\\\/HIvup9G6EgSdX5aa0ylhe\\\\\\\/8UO02sosp+Fopne9g9CK3d\\\\\\\/S1pwysxs866vbKEXBl0eZMXbFz9JpR0joWqYHKTvbaqWYS53w+L7k\\\\\\\/tJJG77aPjGvulNBaqwxMwiwkMCQNhMwX5gZCKqYJHj66\\\\\\\/yEPVb9x1qw9X+O6ujSVnVt5ZEQr5HdiVNF0nYf\\\\\\\/GN8SNDzyMtx8+DbgYObTAGgH5DeRhMn8jM30U0p9DzffjMz\\\\\\\/Mk2KE8Wemz7NP3kwKO3r01My7QeM2oDxj6XAzp3pi\\\\\\\/Gs+SN6QEvbOcSPbhJAtGWYSJUaBwG2ka35hek7knnRjHEQPNek17HatJE1IyqIFjMni9lM\\\\\\\/3PGHO0jZLT8TKL7bSxwCcv19PmIoRJxOL9\\\\\\\/5qlG0bhVbpjQl11p\\\\\\\/5YmAFue6jCysL0TO320JnIZNTjfOVYE7T7oaMu7wbXLPFqAMAbDSX9136m2D1GGX+seCgSDaq6FHBvuGWJTuz5zSg\\\\\\\/z+\\\\\\\/bwazSL4zHUfVIsmPSJhVcYeSi9K35LGPEg+4o+ts74tvt0M1aOaZGS\\\\\\\/jSwc1NpbdY+nQuBEKOTkjdt7bQihC8ZafjGWyVxfL5MgTZtuQZvwuqSIwRCN37T77jmGu+57DgCoRXmWfkR0ybWFeSJmozX\\\\\\\/XfPyzV0Z7O0WNOV3MU72OzHNn8amlL\\\\\\\/qWZ8l55EVo7aa3bHJwKGYiItzHiAaPewTz6KG8d8KvOCshXaO5dVcmaH0wsBba\\\\\\\/cVZBa\\\\\\\/KdvN\\\\\\\/OdZ7\\\\\\\/vRI1duLI1BGr\\\\\\\/P3E3WPVTg5TBSImlx6SLgHMy+vzWQITbCSPaXatYjc5XB4yB3rnzU441lBdouOgKDd3LVQmTiaffBMO4j4PhDXaYBhTVjaikomxSQe4AFeYT2WutUhPml6ftRKBJ2wZ59blDy2gJYXR8ner9MQuM6sOXw6bqjP6WW4gPji3tVmvBK7Dw\\\\\\\/nrgvWlzVJZ26wQVZIZ\\\\\\\/B2R4uAId3AQC2t11n5zGHe7YJ\\\\\\\/O\\\\\\\/cXg5Aotxed7El0o1N00x3W5xVHNYLo43dpSeFif4bFp35yVD00F2xrK9PD3O9MoBo5tvxw0X+7USgHCoYnv4PtY4H0h8DnNemJxSuNysyNZjtBCnRAzBvmSHbcvEMEedWGTaFf\\\\\\\/F\\\\\\\/yLeAnuNzE4txlZZJzCYefsbt7xnudrY9IQYx2F1sCqS5rfEwY6isJxkDIh+MS9RBeiMbhoFEjAgfdi0MDc6B6BMx1hSbgLZ3UYdoGFOdEIbMPbw+nwcvAOdUpl\\\\\\\/0eLuGP\\\\\\\/fXAaloKvIpDlAB33h+2ClaQ4TtzE4E8vvpVG+8ms\\\\\\\/NSpkwUKZuR5p+KDX79+c52HuSG8o\\\\\\\/nFIDlgiJP6H6bLXnZROZ4X7CagWncOatzuJgdnXSa1Bi04BOMzJwSN68VdVQ2Y0rwj9heYmrnspQ9CBhiqRdwiPzCTkZL0WjC7pEwxjxZBFPKLUT5H\\\\\\\/Lt7BCI\\\\\\\/6DoQFav7Unq\\\\\\\/O0QgcsRy20954YhpI7LVMjQfdxoNCQu+3+CkAZhkF7RthV+zOd31d+oLBAJMuKgCiixkJU8rCPrZiQ78859AGV6ng1VyCZsAZEXVQvGyJusJPqpkME1Q8m48\\\\\\\/2sxuWhTpVdL\\\\\\\/uJo1ZoTielPJFvm2N6U2VN7Gga9+4dwEsCxx4TkntvFGzQ30lRBfchwLx5tcQa+nQ+OFDX\\\\\\\/vViok7S67Xaf\\\\\\\/KFTf2OqUQbee\\\\\\\/LMjk1\\\\\\\/xo1+SZwNyRbbcoT36FvhsydVhXWNaMgqqEzmueGZyLVK5RGqM9tjYGHS0Moo3FPTuxImmowxkGkdPoKuGeyUaVy8NykHMDjYOV2VpB3VHDsOIvhL2ZEcJx5rmSJsCchbS0eNuPEK3gE5KJWItkudu1USBa0Lk8uHfZ0bDSunErGb0XFjeMxT8yQrk9YdvdIhafuUeiApRs3h55dNw3KZiBn3wU2i1iPioQcA4KeuHgkoZeMQCpskwzzgR11\\\\\\\/RkLjLCvc6pPKxX8xCuh3aXrewNwG1\\\\\\\/58jmebtrKKs7HvdxYy1pFHxa4DUWKKQeYh40\\\\\\\/rkvFS2j0NBa0rJmED0dOWNogu7UwIlqQmVbxWsqiopgHq+NjEKue6VvXnAESFqJxJ1eZJOuByO0rDvY\\\\\\\/PFbTeWwJVBqkJTQm4JDUlX1jsKJ1ZeeGo6B+UuhsczSU0tr8yL\\\\\\\/5gJW3xvNb9JotK2d+Aycg7x6GkPSktb9d\\\\\\\/MQqP0cqHBoiFhaEIDwKLzKWGVhBtyz3NTkdftwj2MLgDH1R4GAqvG4HRnjf9mPYS3Do6phILVCEXBOLZK1Ctf\\\\\\\/zZmDXxunihsilC9ClqlghW43TYLOwmy56tgdTxH0PwhE1wkeslQR6+Seom94qNNA1SwoLVvaYQ76ZARgFpurBwAEJP8Y6Mz2FeJTdZLXD9X0QGLcm6GXvCfu9FuJ3U0B6PGYdtX90ww0wxiC1d2RDxcpsECRqjSAWUx4e\\\\\\\/5ZcLiSUZYooc+nIv0An47K\\\\\\\/Wy+aIj2EPS8nd8G7nvej9J9ADkU6T+zhHA+oVJkmiMB1HvY0qXHx\\\\\\\/ezYoUYYPprA4dFgA+wMCQV8wIT3wuBXiGHRlE9nvGLJGlFP4aGoDt8UZvypUXLyJck1FK6mM\\\\\\\/jYqmMzhfhBOAKlJpjoLx5c8cgBkwcrF8t9XO06A6Av\\\\\\\/T0qL3skNn6HVBBwwmGeGKmaHRXQR5cnMEo0atbyJQ3WbwGxGOLTXnzDani\\\\\\\/X2LYh6FvFqu78xINfsYPCcdyfcLqUPgaErcdc0fulrKjD2n3xOt+MkdPm+Y\\\\\\\/8q46ccGUrwHPr+X3TE8y1zlP\\\\\\\/y3deaSinwfpKXQJBgvhn4JNizeAxT0cXZ57zf+WC8WuLQNWzfMuqLpIR0d40Ao79yzbIhMswOQa0t5N+PkiH9rnfPTRiCb7iKw7HB5hNh1tYkJOpwl9UnRbrhfuvCyH5v\\\\\\\/VTjs57tUoK3WjIYuY+y6EtjQog62mIyd7nxBMKK9PZEyzUjHNGehOX9ylK8nOuACVxxP3lSbP7eK8\\\\\\\/RsMCAl1Nntt4ESWRqGjKJpTOr9V5oqzp117KeMnlBfy1YSuvB\\\\\\\/LUkduu3fl+ke6rZVYWvIO9ZQJA7hIngR\\\\\\\/10Wdp747EyOTWjRFlqYCKGi3ihbR3e\\\\\\\/\\\\\\\/zAjoUp02OI6cobzibUSpu5TmT3kuWLBD6mpnTiXWCp2mEfWhepnbhhqN3VW10ShkL2CHDWGGeRLk\\\\\\\/g1pSmM2jtSQ+ssa+V4UQVIQ3Znb0L2AlNKK7Iq4jdTQVUaESLfa\\\\\\\/t+YBVI\\\\\\\/QvL5lykMvIx4XDYsMjX00X2QtTlD2Q563CB668EnW\\\\\\\/THNSTDGpdO8kUIwTv7xJ8JmRiuNzkjfYb3LAHJitmsaocEQtMlSMDu7Be9hBuqUcJYZTBM9aUdG8R3h2R2YhoX4k2vHRZRtalxOV9mr617NTd\\\\\\\/icnc8S+WTTdiySD67UUtZMMkXdH+NPQ3n22FuCZhYDgWs0b6Kbsx8dA2J63Om6G+z6Tx45n2KvXoMazS\\\\\\\/4L\\\\\\\/xKxoBW9VUh7H7klnHMd8PUsEoBoYf+98Cpq5bSRqFoKKft4e+7XZg1A+V4FAuPQpqGG1vdow9bv1eQ1mIX3imLt\\\\\\\/KyGOOxwFOhD8aMnF300UIK9Ft1KeOJ4wF9uAR1HTn60Ij501dAU32sp+or6lMlKmmbqMfwc4TSnC5OGjWwmScc6gYqjvqL2bxME4iNpwbXLVt02OmLlJbSLAA9dpLreqwO5Cq0c7k4gDjOcRh7FGZkeaoscHuwV5YBNZWgc+MTIiabwc5F\\\\\\\/xbysaKlacMy9+quW2NaWBEvjt9D8d2HNhxf2kRLpMeCRG4AC8CnTJciCwDxq6BSeWgTnYu3PfFYugU7RpX9vb85izbLc1n5s1PkJhXTzjnsW7NLNoJ5LfHTm8KAN+vn29mn5XKVnF2RBNsimjfMd59IRuUMWOCNEmNNQIDbiVXzfaePL3InnOb1e1FN\\\\\\\/3Yw5Hkoc\\\\\\\/jIZb\\\\\\\/5lxZDX0zUx3U90YiOindVmONFdMUHduQ7CoVQZytf2WUthKBts0+F\\\\\\\/6itntH8ap4o2yvFbKcfLtRg7PgXNpmsa39pvHfZ32zTnh00vejwR+0mVC6Ir2efbpmmdP2L80XOKy1Thej\\\\\\\/JQStUzDH3k8+zAzFdE3QqaThHx9zXUwAkcao21bk2MS3OLYn0rT8CxmoNuwu2AkDtoQTN+2r4l7UVh6MX9ABkWcXEVC9QO20RLnw6kbKLAB+KqAQqR3Gn4l76PHdpQskj3Xa4Fd18haJIERBJ8eOBKsmBW\\\\\\\/IVpvC1q2aH001O+\\\\\\\/EwAGgYRz5rCXYJ7A65rysh0INQwKUmDBHvfwkx3dYm9hx6zsaTO4ZOxoiYNHIIXXFd1tdDESPfswm3G1jHjk+ZX6UMqnwpv0oQYh+a7mi5QlA4TkCWk8AYSGW7mGfdpq5XMw5nfBsWkHM2FpSon2M0iIPMwuLK91wju7pXUlHwQXkC7XkW8ViIszaZu7T02L8rA6JVZjrx4v39oEBIsm4jzKqqizw6+2bxpBm3xLYwIcDHXhiwQ9UUitd7V+SiRj9NG4v5yVNxjlEBoZrXm2Opi6js1SebHkhH5vIhqOrZm+Pd3OsU5AX9FFCx\\\\\\\/F72lTNrZcAy7c0MUNrjt8LzBYe1lQQOQjeJqC3h3NI2hbvsg1O3RaqokS90mtCz2l+RRuxXhZzuQalaIyGYU3colYgYXox4DpJXvlADsdaHT8ZnRZuWN57fYQ9nKA9MUAgs9V7t6noTa8pPmnKmF8d5dhJLnDUNWXuFIG+7F\\\\\\\/hnH8gNRHQpz5Ak4xglHdVROvzvogl024oT2aQ\\\\\\\/O7XKM3GdIyKrXxicrSfmgsBPKXv1PXeXvu6uo2YB7iVKM01E0lXJ+s6VJ68vActsqWiPO1aMP8QR9XABO79SfbCZUYue55HKQVrzV5b1W0ICBRueKNlOxHNZOi0LnXXSFc+j87UsCluh0vlT6bg3n6Tbkc0LAtHXkY1O3yQ4OqX4UDridZE351udQjwJJx\\\\\\\/dDZlU0LcyMls\\\\\\\/mjWA9Xg0q2owGEcyN8vsnZa+vRtogL11+DUZ++ei2HXtwkeWjx+qsW3\\\\\\\/Gf0N6ZNc178EyHBvwOv9TqCuC3lFEHsaZrjHsgTaw+H+3PXY1ULXbLUIwVWZf9rf64A30K8mShhnSHgX9PYHlK2S4I8FD68BzkB8h875grbRp2IIBQ7oxev6fzGn8d+tIrl5JPN\\\\\\\/vpoRnmpJi\\\\\\\/h91K3fKLx+t2gRFekCSbnAEr85oIfsdKS4CZsp1mYDTwaSiOKElb+JTK32FgsYvbceyBMrIAwy4pl+7zSn+OElrkvJNyTanDFcH6xn9OxCXCrCBvd3kfvAZdQMj+6GsX66ns6R2l05xaRz9l26kWm+4QqBhyGTUiRHgTMTOqWrVZPzUZDz1BQmGRuYkAIrzCXwuh3pOlNOjqou7pbdegv0Mzdigc4GdMwsIKjyvDuwlsW33\\\\\\\/yJPr9WhZkMUSEeijwfS3CVBAYSNYkWT8B\\\\\\\/eVrOu+8FxpZduMPyj03bUqeOGQubloKzm+9Hdz20W4AIRxlROFyWXymO2nXElOyoPHSzFp8G1FUmwoy3Ddj3kLSxkkNsy70gtUySZq6sbMKXkuxSsxUB9mZOUpRCQeAtF08jyJXANQ0+g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022irRfT6I89ooMtkuogFMf4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022536e9453bb785367cdd07e9e575d63f3b3ba52f23b6e27e92c132b8aacf353e7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kJkh4EUNGr2slxXJYwkcYx4qLYCX2L4KYYTWfm9+dVgtJWhdAOY7hqnC+sFTymqtNvGLX9ejQrxSjmnIVFJ8JM\\\\\\\/DD9hndh5Yci4Xu1E4S2xAQS+mJXQm2swJwlWqT5UBZSRng9eo3mo43xTxmInHbSWoEpR+9aiq1NiWkrnNUbu76Eh5nh1p8dA\\\\\\\/WG0GCnT\\\\\\\/5LdfFYEc0mCbwG2sfE\\\\\\\/RsQBWxQMCsfsxUfebAaHwPRk3V6SY3L7TRjhiuaU+cRgdqvbERDDDpyoxLJtxChf+ZbBtDYm6XoMB5ZjmNQehY608S06xOqIShQvnf2Bk7dgaDdhZd1I2kI5i2BOm5wQHLwONQZRX4HavD6I8c+LCgJi1SozrxneP349cLwWSlNpomlthZUnjcqvnVB5Lb49hvOAbTv9ZDmUKIigsEYlNMn0MwIzeupKEU9RrWOHRka42IM1maq6ET3e8cKzak8ln6gfG5Daqo2tqeTpuRCYOP0M7kBF8XaP8Lo6D\\\\\\\/FU1U6sZWdXYkhR84\\\\\\\/ZjElge1Q57DpQ49N0q1RG9+AXs\\\\\\\/qzpIw5bS53wI57Q0MVm8qS7rYSrDgdOmIebIt4ryRCGmFLaNrNUfgWh4kirmM34Y\\\\\\\/u6w0AdT1DwsNh43tXeff0ernANCknkjGwXc0idBYD3s+cJ9ux0QAQJyzvJPS\\\\\\\/CJZ5t+912oeHd4AfPXDQIlYfF8ui00+hNvCCsnnO7++YtSZ4t423PayMu+26Cgf7Q7Yd4DQ9vmvz6CHuIAv3PC+Trrk1g1EvsRkVnZQ\\\\\\\/nsr0i8FX2DdcqemkFWsEJFeQR0TmAT63lArVaPeUT7UTaiF+k6LLolR\\\\\\\/A4MP4PIp\\\\\\\/qnEPpEeXjhmTwtUf6oE+9V9f4he2TyKBUWIOiwqqJ458D\\\\\\\/0ZaUY4srZl4R9kVVGiaY0\\\\\\\/JxTaa0fdWH\\\\\\\/z92KiCYdngZW2JhOMiEvgJakw41AVaPfS1rsOpa0LWwvhvUTNCvkLSu61yjSaXguH5exPWGPUZ6noTJ156jr+Y0Dnc\\\\\\\/78BiNW01eYEBmZo8omQQucwK2P5+3XxTCc7+YxVpOvo7K4mMQxwZn4kt30sG2tOIQoHyBLPy3sIWXoRRoi+dwVMS9M3ewZQUP9BnmOyf6AVb0OyTI4Uax+7pi+B0zObcy7vHmLp7Qs1lZ+M25G6gB1YqHllBgasgWYwmG2ergTmNSWT5aWhVQZL77lJNHxOlQgpmnk\\\\\\\/dbWzNEPtB6TT4zN7idOoh8mlefFXQNvbjr8vFE66insKa2C6Ws39qJVuO56+86J6yMONYlyQTDEyCY5JfyWa4PgQUJNPkeGiLLPQjr\\\\\\\/qRWj6p8PHLydegxRX1KVPmEmxsUXbIJnsSOPQR8UbtIKdQ5P\\\\\\\/DDmihWUjydXc\\\\\\\/U3vriQOgho6BcuVTbdAQdnr1Y+lXus\\\\\\\/yK\\\\\\\/NQDIvYB35MaozPqthnE0Je47TDZQN9KBqpZ8XAhU2DhNmDE2EsWxMlbFuLMebzHKSNyp0eL0t5PtJDmexHGkBxM74ptOGfQvxyJiCyy1oE0DX\\\\\\\/oVU02JW3JXagdf3SHYzvocYglpy63MaoKBvzkPPxSFg86Vd6iU87DowHFLj+H1L7JKVlLt15NMPbg6sYMtQ3qk\\\\\\\/HyG72RL0oFkIElduPV6NTfx\\\\\\\/rwUg6KbIW0KNxgrMCbMhNCYme7H\\\\\\\/v37iZy93YPcXuZlSTG1rKoNGbwKYdv0HpA910RS7gX0uob5GotxIPWyC33g4STCe4YOq55AcFR7rLtX1Dhfk0yJCEa0vrslsVJI5aV1gMXsmrWCZhGyUPwv8s4xEWHYtixrJhYI6zLe9AZQxUqdd9AYMIrbgxErKXXwTf87mwNOgNDhEco9XptLcJm6StN7cK3qPiKXaTPNCaIwInyySyjxp7ACHmJjwSQdsmGnZB0GN1ekzws7RQfM\\\\\\\/JMWvjHfsHd0Ai3zts3ueL9m3JZ\\\\\\\/\\\\\\\/2cpq\\\\\\\/UOn0l\\\\\\\/K1MTXtRGCMC+U9cTk3pb9pig4ylJihbj51+OUM73oy1Y9\\\\\\\/rPODgY\\\\\\\/GyqWw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022bmuotHflglZYIhJHFj5rfQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c8f94540f9dac18316319ed11e69e4c302cd473aa614b67fd35e922f0e1403fe\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494286 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3345" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bWaFeENqZCE2y01r9NVd8yxAcrUDMBnonQ5_H0sSm8U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3345" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eScWMNkl9Dr1ik3sR2IqV_9dLooSaNiPXBcNZc_lVNA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3333" title="Genomic Prediction Including SNP-Specific Variance Predictors" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3359" title="Exon 3 of the NUMB Gene Emerged in the Chordate Lineage Coopting the NUMB Protein to the Regulation of MDM2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494286" id="node494286" data-pisa="ggg;9/10/3345" data-pisa-master="ggg;g3.119.400428" data-apath="/ggg/9/10/3345.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alternative Lengthening of Telomeres in the Budding Yeast <em>Naumovozyma castellii</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3370-2864" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marita Cohn</span>, <span class="highwire-citation-author" data-delta="1">Ahu Karademir Andersson</span>, <span class="highwire-citation-author" data-delta="2">Raquel Quintilla Mateo</span> and <span class="highwire-citation-author" data-delta="3">Mirja Carlsson Möller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3345-3358;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400428 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494286" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marita Cohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marita%2BCohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cohn%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarita%2BCohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3370-2864" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marita Cohn</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marita.cohn@biol.lu.se" class="" data-icon-position="" data-hide-link-title="0">marita.cohn@biol.lu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ahu Karademir Andersson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ahu%2BKarademir%2BAndersson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersson%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAhu%2BKarademir%2BAndersson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Raquel Quintilla Mateo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raquel%2BQuintilla%2BMateo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mateo%20RQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaquel%2BQuintilla%2BMateo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mirja Carlsson Möller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mirja%2BCarlsson%2BM%C3%B6ller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%B6ller%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMirja%2BCarlsson%2BM%25C3%25B6ller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3345" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494286" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3345.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494286" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3345.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494286" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3345.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494286" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The enzyme telomerase ensures the integrity of linear chromosomes by maintaining telomere length. As a hallmark of cancer, cell immortalization and unlimited proliferation is gained by reactivation of telomerase. However, a significant fraction of cancer cells instead uses alternative telomere lengthening mechanisms to ensure telomere function, collectively known as Alternative Lengthening of Telomeres (ALT). Although the budding yeast <em>Naumovozyma castellii</em> (<em>Saccharomyces castellii</em>) has a proficient telomerase activity, we demonstrate here that telomeres in <em>N. castellii</em> are efficiently maintained by a novel ALT mechanism after telomerase knockout. Remarkably, telomerase-negative cells proliferate indefinitely without any major growth crisis and display wild-type colony morphology. Moreover, ALT cells maintain linear chromosomes and preserve a wild-type DNA organization at the chromosome termini, including a short stretch of terminal telomeric sequence. Notably, ALT telomeres are elongated by the addition of ∼275 bp repeats containing a short telomeric sequence and the subtelomeric DNA located just internally (TelKO element). Although telomeres may be elongated by several TelKO repeats, no dramatic genome-wide amplification occurs, thus indicating that the repeat addition may be regulated. Intriguingly, a short interstitial telomeric sequence (ITS) functions as the initiation point for the addition of the TelKO element. This implies that <em>N. castellii</em> telomeres are structurally predisposed to efficiently switch to the ALT mechanism as a response to telomerase dysfunction.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/alt" class="hw-term hw-article-keyword hw-article-keyword-alt" rel="nofollow">ALT</a></li><li class="kwd"><a href="/keyword/alternative-lengthening-telomeres" class="hw-term hw-article-keyword hw-article-keyword-alternative-lengthening-of-telomeres" rel="nofollow">Alternative Lengthening of Telomeres</a></li><li class="kwd"><a href="/keyword/telomere" class="hw-term hw-article-keyword hw-article-keyword-telomere" rel="nofollow">telomere</a></li><li class="kwd"><a href="/keyword/genome-integrity" class="hw-term hw-article-keyword hw-article-keyword-genome-integrity" rel="nofollow">genome integrity</a></li><li class="kwd"><a href="/keyword/telomerase-independent-telomere-maintenance" class="hw-term hw-article-keyword hw-article-keyword-telomerase-independent-telomere-maintenance" rel="nofollow">telomerase-independent telomere maintenance</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/naumovozyma-castellii" class="hw-term hw-article-keyword hw-article-keyword-naumovozyma-castellii" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Naumovozyma castellii</em></a></li></ul><p id="p-3">The ends of linear chromosomes are protected by special DNA-protein complexes, telomeres. Telomeres are essential for both genome integrity and cell proliferation. Every time a cell divides, a piece of DNA is lost due to the “end replication problem’’ (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Olovnikov 1973</a>). This progressive attrition of telomeres triggers DNA damage response at chromosome ends, which are not able to distinguish themselves from DNA double-strand breaks. At the crisis point, cellular proliferation will stop and cells will eventually die. Conversely, cancer cells escape from cell cycle arrest and telomere dysfunction triggers genome instability (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kim <em>et al.</em> 1994</a>). Hence, telomere-induced cellular senescence could be considered as a “tumor-suppressing mechanism” (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Shay and Wright 2011</a>).</p><p id="p-4">Most eukaryotic organisms, including yeasts, counteract this progressive telomere shortening by employing the enzyme telomerase, which is a ribonucleoprotein complex with a specific RNA subunit. Telomerase adds telomeric repeats onto chromosomal ends by using its intrinsic RNA template (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Greider 1991</a>). Due to insufficient telomerase activity, human somatic cells undergo a limited number of cell divisions (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hayflick and Moorhead 1961</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Harley <em>et al.</em> 1990</a>). In contrast, most cancer cells overcome replicative limitations to gain cellular immortality by reactivating telomerase. Even though telomerase is a key component for unlimited proliferation, 10–15% of cancer cells maintain their chromosome ends over many population doublings in the absence of telomerase. These cells use alternative mechanisms rather than re-activating the telomerase. Collectively, the telomerase-independent mechanisms are known as Alternative Lengthening of Telomeres (ALT) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bryan <em>et al.</em> 1997</a>). Although telomerase is an attractive target for anti-cancer therapies, development of an effective treatment will benefit by addressing the ALT pathways (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cesare and Reddel 2010</a>).</p><p id="p-5">Budding yeasts express telomerase and proliferate indefinitely. When telomerase is disabled in <em>Saccharomyces cerevisiae</em>, cells divide for a limited number of generations, like human somatic cells. Their telomeres progressively shorten and very short telomeres inhibit cell proliferation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lundblad and Szostak 1989</a>). Telomerase-negative cells display a decline in growth rate and most cells stop dividing and senesce. A few of them overcome the replicative senescence, improve their growth rate and “survivors” arise (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lundblad and Blackburn 1993</a>). In order to restore telomere lengths they amplify telomeric and subtelomeric sequences via recombination-mediated ALT pathways (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kass-Eisler and Greider 2000</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2001</a>).</p><p id="p-6">Recombinational telomere extension has been described also in <em>Kluyveromyces lactis</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McEachern and Blackburn 1996</a>). It is suggested that telomerase deletion mutants use the “roll and spread” mechanism where a telomeric circle (t-circle) is used as a template to elongate an eroded telomere. This newly elongated telomere is then used as a template to elongate other telomeres through break-induced replication (BIR) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Natarajan and McEachern 2002</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Groff-Vindman <em>et al.</em> 2005</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Topcu <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McEachern and Haber 2006</a>).</p><p id="p-7">Recombination is actively used but not essential for formation of all types of survivors. A subset of <em>S. cerevisiae</em> cells still proliferate in the absence of both telomerase and recombination through formation of palindromes (PAL) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Maringele and Lydall 2004</a>). PAL survivors have been observed when ExoI (exonuclease I) is disabled in <em>S. cerevisiae</em> cells lacking both telomerase and recombination. They contain linear but abnormally sized chromosomes without telomeric sequences (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Maringele and Lydall 2004</a>). In contrast, when telomeres of <em>S. cerevisiae</em> strains were first lengthened, rare survivors exhibiting telomeric DNA amplification appeared in telomerase- and <a href="https://identifiers.org/bioentitylink/SGD:S000004494?doi=10.1534/g3.119.400428"><em>RAD52</em></a>-deficient cells (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">LeBel <em>et al.</em> 2009</a>).</p><p id="p-8">The budding yeast <em>Naumovozyma castellii</em> has been used as a model organism in telomere studies owing to its human-like telomere structure and maintenance (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Karademir Andersson and Cohn 2017</a>). <em>N. castellii</em> belongs to an evolutionary lineage that underwent whole genome duplication followed by a massive gene loss (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cliften <em>et al.</em> 2006</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Karademir Andersson <em>et al.</em> 2017</a>). Its genome is sequenced and has been used extensively in comparative genomics (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Karademir Andersson and Cohn 2017</a>). <em>N. castellii</em> has ten chromosomes and a genome size of 11.2 Mb (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cliften <em>et al.</em> 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gordon <em>et al.</em> 2011</a>). Its processive telomerase enzyme is an excellent tool to study molecular mechanisms of telomerase-dependent telomere maintenance (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cohn and Blackburn 1995</a>). The regular octamer repeats, TCTGGGTG, have enabled the determination of the minimal binding sites for the telomere proteins Rap1 and Cdc13 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cohn <em>et al.</em> 1998</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wahlin <em>et al.</em> 2003</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rhodin Edsö <em>et al.</em> 2008</a>). Recently, we showed that the single-stranded overhang varies in length between 14-200 nt (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fridholm <em>et al.</em> 2013</a>).</p><p id="p-9">Here, we report that telomerase-negative <em>N. castellii</em> cells avoid replicative senescence, and proliferate without any major growth crisis, by using an ALT mechanism with novel characteristics. The efficient ALT mechanism is observed in both diploid and haploid <em>tlc1</em><sup>-</sup> mutants, as well as in diploid <em>est1</em>Δ mutants. <em>N. castellii</em> ALT cells maintain linear chromosomes and lack any gross chromosomal rearrangements. Interestingly, ALT cells maintain a similar chromosomal end structure as the wild-type, but with a shorter telomeric region. Intriguingly, we discovered that ALT telomeres are lengthened by uniform ∼275 bp repeats containing a subtelomeric region and the shorter telomeric region. We propose that telomere lengthening occurs by inter-telomeric copying, starting from a short interstitial telomeric sequence in the subtelomeric element. The actual structure of the <em>N. castellii</em> subtelomeric DNA may therefore form the foundation for the efficient activation of this novel type of ALT maintenance mechanism.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media</h3><p id="p-10">The yeast <em>Naumovozyma castellii</em> was previously called <em>Saccharomyces castellii</em> or <em>Naumovia castellii</em>. The <em>N. castellii</em> strains used in this study were NRRL Y-12 630 (type strain), Y235 (<em>MATa/MATα</em>, <em>ura3/ura3)</em>, YMC121, YMC124, YMC125 and YMC130 (<em>MATa/MATα</em>, <em>ura3/ura3</em>, <em>tlc1</em>::<em>kanMX3/tlc1</em>::<em>kanMX3</em>); YMC122 and YMC123 (<em>MATa/MATα</em>, <em>ura3/ura3</em>, <em>TLC1/TLC1</em>); YMC300 (<em>MATa/MATα</em>, <em>ura3/ura3</em>, <em>EST1/est1Δ</em>::<em>URA3</em>), YMC48 and YMC63 (<em>MATα</em>, <em>hoΔ</em>::<em>hphMX4</em>, <em>ura3</em>); YMC481, YMC482 and YMC631 (<em>MATα</em>, <em>hoΔ</em>::<em>hphMX4</em>, <em>ura3</em>, <em>tlc1</em>::<em>kanMX3</em>) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Astromskas and Cohn 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>). YMC300 (<em>MATa/MATα</em>, <em>ura3/ura3</em>, <em>EST1/est1Δ</em>::<em>URA3</em>), YMC303, YMC305 (<em>ura3/ ura3</em>, <em>est1Δ</em>::<em>URA3/est1Δ</em>::<em>URA3</em>). <em>Escherichia coli</em> strain DH5α was used in the cloning experiment.</p><p id="p-11">All yeast strains were grown in YPD medium containing 1% (w/v) yeast extract, 2% (w/v) peptone and 2% (w/v) glucose. Ura<sup>-</sup> drop out plates contained 26.7 g/l minimal SD base, 0.77 g/l ura<sup>-</sup> DO supplement and 2% agar. Geneticin and hygromycin resistant cells were grown in YPD medium containing 75 mg/l geneticin and 200 mg/l hygromycin B (Duchefa), respectively. Yeast cells were grown at 25°.</p><p id="p-12">The <em>TLC1</em> gene was disrupted by using the TLC1-kanMX3 construct as previously described (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>), with the kanMX3 cassette inserted into the <em>Hpa</em>I site of the <em>N. castellii TLC1</em> gene. PCR amplification of the plasmid template was performed with primers 5′- ATTTAGTATTTTGTTTTCCCG -3′ and 5′- TGAATGATTTACTTGTCGTCGC -3′ and annealing at 57° for 30 s. To confirm insertion into the correct genomic DNA site of transformants, PCR amplicon size was estimated by using primers annealing 0.5-0.6 kb up- and downstream regions of the <em>TLC1</em> gene, respectively: 5′- GTAGTTTCTTCGTCAGCC -′3 and 5′- TTGGCTCCAGAAGTCGC -3′. To further confirm the <em>tlc1</em><sup>-</sup> genotype, <em>Pvu</em>I-treated genomic DNA (gDNA) extracted from spores of five tetrads was analyzed with Southern blot hybridization (previously described in (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>)). Using the <em>TLC1</em> gene as the hybridization probe, the fragment size difference confirmed the correct insertion of the kanMX3 marker within the <em>TLC1</em> gene (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>).</p><p id="p-13">The <em>EST1</em> gene was deleted in the <em>N. castellii</em> strain Y235, leading to strain YMC300. The deletion cassette was constructed by using the plasmid pWJ1042 containing the <em>K. lactis URA3</em> gene flanked by 143 nt long direct repeats, allowing the subsequent pop-out of the <em>URA3</em> gene (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Reid <em>et al.</em> 2002</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>). The deletion cassette was PCR amplified in two separate fragments with primers: 1) 5′ – <span class="underline">TAGTTAAACATATTTACGACACCAACGGTGACCCGATTCTGG-</span> GAATTGGGTACCGGGCCC -3′ and 5′- CTGATATCACCAACGCCC -3′, 2) 5′- GAGCGCTGATTCTCTTTTGG -3′ and 5′- <span class="underline">TCAAGATTGATCTAGTTGGATT-ATTGAACATTTTCTGAACAGTGA</span> ATGGCAATTCCCGGGGATC -3′. The 45 nt long up- and downstream regions of the <em>EST1</em> gene are underlined and the rest correspond to complementary sequences up- and downstream of the direct repeats in pWJ1042. PCR with Phusion DNA polymerase was performed according to manufacturer’s instruction, with annealing at 50° for 30 s. Correct insertion into transformants was determined by PCR, using primers annealing 0.5 kb up- and downstream of the <em>EST1</em> gene, respectively: 5′- ATCTCGGTGCTCAAAGGTGC -3′ and 5′-ACAGCCTTGGCAGATATCGAAG -3′. Verification of knockouts was also performed by Southern blot hybridization of <em>Bam</em>HI cleaved genomic DNA, using an oligonucleotide probe specific for the <em>klURA3</em> marker gene (5′- CTGATATCACCAACGCCC -3′) which was 5′ end-labeled with [γ<sup>32</sup>P]-ATP by T4 Polynucleotide kinase (NEB). Hybridization conditions as previously described at 40° (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>).</p><p id="p-14">Both the deletion and the disruption cassettes were transformed using lithium acetate method and yeast cells were spread on appropriate selective plates, as previously described (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>). Diploid cells were sporulated and tetrads were dissected on YPD plates with the Singer MSM microdissection system according to the manufacturers manual, as previously described (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>; <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>).</p></div><div id="sec-3" class="subsection"><h3>Preparation of Naumovozyma castellii telomerase extract</h3><p id="p-15">The <em>tlc1</em><sup>-</sup> cells (&gt;75 generations) were grown in YPD medium at 25° overnight (ON) and harvested at OD<sub>600</sub> = 1-1.5. The cells were resuspended in TMG buffer (10 mM Tris-HCl pH 8, 1.2 mM MgCI<sub>2</sub>, 0.1 mM EDTA, 0.1 mM EGTA, 1.5 mM DTT, 40U/ml RiboLock RNasin (Thermo Scientific), 1x Complete Protease inhibitor cocktail (Roche) and disrupted using a bead beater. The protein fraction was ultracentrifuged in a 50.2 Ti rotor (Beckman) at 40 000 rpm, 4° for 90 min. The crude extract was partially purified on a DEAE-agarose column, which was equilibrated with TMG buffer (10% glycerol) and washed with 0.4 M and 0.5 M sodium acetate in TMG buffer (10% glycerol) respectively. The active telomerase enzyme was eluted using 0.6 M sodium acetate in TMG buffer (10% glycerol). Desalting and concentrating was done using Amicon Ultra-15 centrifugal filters (Millipore, 30K MWCO). Aliquots were stored in -80°. All solutions were prepared with Diethylpyrocarbonate treated water to prevent RNase contamination.</p></div><div id="sec-4" class="subsection"><h3>Telomerase primer extension assay</h3><p id="p-16">Telomerase activity was assayed in a total volume of 20 μl at 18° for 30 min, with 10 μl of the telomerase extract in a 1x telomerase reaction buffer (0.05 M Tris-HCl pH 8.0, 1mM spermidine, 1mM DTT), 50 μM dTTP, 50 μM dCTP, 200 mM K-Glu, 0.083 μM [α<sup>32</sup>P]-dGTP (PerkinElmer), 0.54 μM dGTP, and a gel purified oligonucleotide, (GGGTGTCT)<sub>2</sub>, with a final concentration 1.65 μM. The primer extension reactions were stopped with 200 μl 10 mM Tris-HCl (pH 7.5) and 21 mM EDTA. An 11-mer telomeric primer, which was labeled with [γ<sup>32</sup>P]-ATP by T4 Polynucleotide kinase, was added into the stop solution as loading control. Extraction was made with an equal volume of phenol/chloroform:isoamyl alcohol (24:1). An equal volume of 5 M ammonium acetate with 0.1 mg/ml glycogen was added before DNA precipitation with 1 ml 95% ethanol in room temperature for 60 min. The samples were washed with 70% ethanol and dried. Pellets were resuspended in 4 μl Xylene-cyanol dye mix, incubated at 90° for 2 min and loaded on a 10% polyacrylamide/7 M Urea sequencing gel. The gel was run at 1800 V for 1h 40 min in 0.6 x TBE buffer (26.7 mM Tris-Borate and 0.6 mM EDTA), dried and analyzed for radioactive signal with a BioRad Molecular Imager FX PhosphorImager.</p></div><div id="sec-5" class="subsection"><h3>Colony streaking assay</h3><p id="p-17">After sporulation and microdissection of asci, tetrad spores were germinated and grown into colonies at 25° for 48 hr (passage 1). Colonies were then restreaked onto fresh YPD plates (passage 2). After 48 hr incubation at 25°, single colonies were picked and restreaked again (passage 3). Restreaking was repeated to obtain up to 12-16 subsequent passages. Each passage corresponds to 20-25 generations (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Reid <em>et al.</em> 2002</a>; <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>). Several separate streaks were then assembled onto the same plate for the figure preparation.</p></div><div id="sec-6" class="subsection"><h3>Yeast spot test assays</h3><p id="p-18">Whole colonies from passage 2-15 (p2-p15) of the diploid <em>tlc1</em><sup>-</sup> ALT strain, and the isogenic WT strain, were grown overnight in 2 ml YPD, at 25°. Undiluted, and serial 10-fold dilutions of the cultures were spotted onto YPD plates and incubated for 2 days at 25°.</p></div><div id="sec-7" class="subsection"><h3>Liquid growth assays</h3><p id="p-19">Colonies from the diploid <em>tlc1</em><sup>-</sup> strain, passage 3 (p3) and passage 16 (p16), as well as wild-type (WT) strains obtained from the same sporulation tetrad, were inoculated into liquid YPD media and grown overnight at 25°. Next day, cultures were inoculated to a final concentration of 5 × 10<sup>5</sup> cells/ml. After 24 hr of growth at 25° the cell density was counted in a Bürker chamber. The mean value of ten independent experiments was calculated and plotted together with the standard deviation.</p></div><div id="sec-8" class="subsection"><h3>DNA isolation and terminal restriction fragment assay (TRF assay)</h3><p id="p-20">One single colony from each passage was inoculated in 2-10 ml YPD liquid media and grown for ON at 25°, 200 rpm. Cells were mechanically broken using glass beads. DNA extraction was done with phenol/chloroform:isoamyl alcohol and then with ethanol. After RNase A incubation at a final concentration of 75 mg/l, genomic DNA (gDNA) was precipitated with 25 mM NH<sub>4</sub>Ac followed by ethanol. Appropriate amount of gDNA was digested by either <em>Hind</em>III or <em>Dde</em>I (ThermoScientific). 300 ng of digested gDNA was separated on a 0.8% agarose gel, 0.5 x TBE, and was transferred to a Hybond-XL membrane (Amersham). Hybridization probes were 5′end-labeled by T4 polynucleotide kinase and [γ<sup>32</sup>P]-ATP. The membrane was hybridized with a telomeric probe, (TGTCTGGG)<sub>2</sub>, at 40° for ON. The membrane was washed for 5 min and then washed at 45° for 15 min, twice in 100 mM Na<sub>2</sub>HPO<sub>4</sub> and 2% SDS. Signals were visualized using Typoon FLA 9500 (GE Healthcare).</p></div><div id="sec-9" class="subsection"><h3>Bal31 assay</h3><p id="p-21">1.4 μg gDNA was incubated with 0.3U of Bal31 nuclease (NEB) for increasing time at 30°. In each time point, 1/10 volume of the reaction was taken out and Bal31 was heat-inactivated at 65° for 10 min in the presence of 33 mM EGTA. DNA was phenol-chloroform-extracted, ethanol-precipitated, and treated by either <em>Hind</em>III or <em>Dde</em>I. Digested DNA was separated on a 0.8% agarose gel (140 ng DNA per well) and subjected to Southern blot analysis as described above. As a control, linear plasmid DNA was incubated with Bal31 and digestion was assessed by a decrease in size of the linear plasmid DNA on an agarose gel (data not shown).</p></div><div id="sec-10" class="subsection"><h3>Pulsed field gel electrophoresis (PFGE)</h3><p id="p-22">Yeast cells were grown at 25°, ON, one single colony inoculated into 4 ml YPD. Cells from 1-2 ml samples were pelleted, washed twice in 1 ml Tris/EDTA-solution (10 mM Tris-HCl pH 7.5, 50 nM EDTA) and dissolved in 100 μl Tris/EDTA-solution. Cells were placed at 42° for 30 s and mixed with 950 μl Low-melting-point (LMP) agarose (1% w/v in 125 mM EDTA pH 7.5, 42°). The cell suspension was pipetted into plug molds and allowed to solidify at 4°. Agarose plugs were collected into 2 ml microtubes with 1.5 ml 10 mM Tris-HCl pH 7.5, 0.5 M EDTA. Protoplasts were prepared by adding 40 μl 10 mg/ml Zymolyase (20T, US Biological) and incubated at 37° ON. Next day the buffer was removed and chromosomes were prepared by addition of 1 ml 10 mM Tris-HCl pH 7.5, 0.5 M EDTA, 1% N-lauroylsarcosine pH 9.5, 2 mg/ml Proteinase K and incubation ON at 50°. The plugs were washed five times with 2 ml Tris/EDTA-solution for 1 hr at room temperature and finally stored in Tris/EDTA-solution at 4°. The plugs were inserted in the wells of a 1% agarose gel (Bio-Rad Pulse field certified agarose) in 0.5x TBE and fixed with LMP agarose. Chromosome size marker: <em>S. cerevisiae strain</em> YNN295 (Pharmacia). Chromosomes were separated by pulsed field gel electrophoresis (PFGE) employing a CHEF Mapper XA (Bio-Rad), 0.5x TBE at 14°, and then stained in 0.5 μg/ml ethidium bromide, 0.5x TBE, for 2-4 h. Separation program: Block 1, pulse-time 240 s for 8 h; block 2, pulse-time 160 s for 10 h, block 3, pulse-time 90 s for 14 h, block 4, pulse-time 60 s for 8 h; angle 60°, 4.5 V/cm.</p></div><div id="sec-11" class="subsection"><h3>Cloning of telomeric fragments</h3><p id="p-23">Genomic DNA of passage 11 (∼275 generations) was digested with <em>Dde</em>I and then separated on a 0.8% agarose gel. <em>Dde</em>I-treated fragments shorter than 350 bp were gel-extracted. Purified fragments were blunt-end treated with T4 DNA polymerase (Fermentas) according to manufacturer’s instruction. These fragments were spin-purified and then ligated into dephosphorylated <em>Sma</em>I-treated pUC18. The ligation mixture was transformed into <em>E. coli</em> cells and transformants were selected on LB-ampicillin plates containing X-gal and IPTG. Transformants were passaged onto both a master plate and a Hybond N membrane (Amersham). In total, 10 202 colonies were screened for the presence of telomere sequences by colony hybridization with the telomeric probe. A total of 448 colonies giving hybridization signal were subjected to plasmid DNA (pDNA) purification and were further screened by slot blot hybridization, where the clones showing hybridization signal were selected for sequencing using the universal primer M13 (17-mer). Three colonies (ID 382, ID 406, and ID 448) showed a strong hybridization signal, indicating telomere-containing insert.</p></div><div id="sec-12" class="subsection"><h3>Slot blot analysis</h3><p id="p-24">Genomic DNA from ALT cells with and without the ALT ladder pattern was spotted on the membrane. Negative control: λ DNA. Positive control: pDNA of the clone ID 406, which contains both the TelKO element and 27 bp long telomeric sequence. The WT type strain Y-12630 was used for normalization. Twofold serial dilution of gDNA was prepared in TE buffer and boiled in 200 μl of denaturation buffer (0.4 M NaOH, 10 mM EDTA pH 8.2) for 10 min. After snap cooling on ice, DNA was vacuum-spotted using a Bio-Dot SF blotting system (BioRad) onto a Hybond-XL membrane pre-equilibrated with sterile H<sub>2</sub>O. Wells were washed with 0.4 M NaOH. The membrane was rinsed in 100 ml of 2x SSC buffer for 5 min, UV-crosslinked and hybridized with probes specific for telomeric sequences, and the TelKO element (20-mer TelKO-A: 5′- TGGGGTACGAGAAAATGTTG -3′). Hybridization was done at 45° ON and was followed by washing at 50° for 15 min, twice as described above. The signals were quantified using QuantityOne software (BioRad), normalized against λ DNA and then the value of WT DNA (WT DNA was set as 100%). Relative percentage values were plotted.</p></div><div id="sec-13" class="subsection"><h3>DNA isolation and telomere-PCR</h3><p id="p-25">Cells were harvested at OD<sub>600</sub> = 0.8, resuspended in 4 ml of SEB buffer (1 M sorbitol, 0.1 M EDTA and 14 mM β-mercaptoethanol) and incubated with 50 μL Zymolyase 20T at 37°, 60 rpm. The spheroplasts were harvested at 3 000 xg for 5 min and then incubated with 0.25 mg/ml proteinase K (Roche) in EDS buffer (50 mM EDTA, 0.2% SDS) at 37° for 1 h. Then 2 ml 3 M potassium acetate was added and incubated on ice for 1h. DNA was precipitated by adding an equal volume of isopropanol, rinsed in 2 ml 70% ethanol and resuspended in 0.5 ml TE buffer for ON. After RNase A incubation DNA was phenol-chloroform-extracted. DNA was precipitated by adding 1/10 volume of 5M NH<sub>4</sub>Ac and then 2.5 volume of 95% ethanol. Finally DNA was washed in 1 ml 70% ethanol and dissolved in 10 mM Tris-HCI, pH 8.0 at 4°.</p><p id="p-26">To C-tail the genomic DNA (gDNA), 200 ng gDNA was denatured in 50 mM potassium acetate, 20 mM Tris acetate, 20 mM magnesium acetate, 0.25 mM cobalt chloride and 0.1 mM dCTP at 98° for 10 min, tailed with 0.2 U Terminal Transferase (NEB) at 37° for 30 min and heat-denatured at 70° for 10 min. Non-tailed DNA was prepared the same as tailing but terminal transferase was replaced with H<sub>2</sub>O. For the PCR reactions, 25 μl of 1 μM forward primer (Subtel-F3), 1 μM reverse primer (poly(G)<sub>18</sub>), 8 ng C-tailed or non-tailed gDNA, was mixed with 25 μl JumpStart REDTaq Ready Mix Reaction Mix (SigmaAldrich). PCR amplification was initiated at 95° for 2 min, then carried out using 45 cycles of 95° for 30 s, 64° for 30 s, 72° for 3 min, and final extension 72° for 5 min. Primers used in telomere PCR:</p><p id="p-27">Subtel-F3, 5′-TGGTTGTAATATTGGAATTATTTTGTAATTCTAGTTCCCCGTTGGG-3′; poly(G)<sub>18</sub>, 5′- TGCTCCATACATTACTTAT(G)<sub>18</sub> -3′. PCR products were separated on a 0.8% agarose gel, gel extracted and sequenced.</p></div><div id="sec-14" class="subsection"><h3>Data and reagent availability</h3><p id="p-28">Strains are available upon request. The authors affirm that all data necessary for confirming the conclusions of this article are represented fully within the article and its tables and figures. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9632294">https://doi.org/10.25387/g3.9632294</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Telomerase-negative cells proliferate without a growth crisis</h3><p id="p-29">Telomere elongation by telomerase is the most common way to maintain telomeres, in order to sustain genome integrity and thus cell proliferation. In <em>S. cerevisiae</em>, telomerase is expressed constitutively and deletion of any sub-component of telomerase leads to a gradual telomere shortening (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lundblad and Szostak 1989</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Singer and Gottschling 1994</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lendvay <em>et al.</em> 1996</a>). The telomere attrition leads to replicative senescence within 50-100 cell divisions, as observed by a severe decrease in cell proliferation and small colonies with rough edges. However, a low amount of cells may survive this crisis, and are termed “survivors” (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Lundblad and Szostak 1989</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lundblad and Blackburn 1993</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">McEachern and Blackburn 1995</a>). With the aim to investigate telomere length maintenance in the absence of telomerase in <em>N. castellii</em>, we disrupted the <em>TLC1</em> gene, encoding the telomerase RNA subunit that provides the template for telomerase elongation. We previously inserted the kanMX3 marker gene to disrupt the <em>TLC1</em> in the diploid <em>N. castellii</em> strain Y235 (described in (<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>)). Five heterozygous transformants (<em>TLC1/tlc1</em>::<em>kanMX3</em>) were sporulated and ten asci were microdissected into four haploid spores (tetrad) (Figure S1A). Replica-plating onto YPD-geneticin plates showed that each tetrad contained two wild-type spores (WT; <em>TLC1</em>) and two knockouts (KO; <em>tlc1</em>::<em>kanMX3</em>) (Figure S1A). The genotype of the knockout strains was verified by PCR amplicon size evaluation, using primers annealing up- and downstream of the <em>TLC1</em> gene, respectively, and the <em>tlc1</em><sup>-</sup> genotype was further confirmed by Southern blot hybridization (previously described in (<a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Astromskas and Cohn 2009</a>)).</p><p id="p-30">To verify the loss of telomerase activity, we performed <em>in vitro</em> telomerase assays of extracts made from both WT and <em>tlc1</em><sup>-</sup> strains (Figure S1B). We tested telomerase activity of all four spores of two tetrads, using the primer extension assay with a 16 nt long telomeric DNA primer. The primer extension assay is highly sensitive and displays each nucleotide added onto the primer (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cohn and Blackburn 1995</a>). Telomerase extracted from WT spores elongated the primer by multiple additions of the telomeric repeat, showing the characteristic 8 nt repeat pattern (Figure S1B, lanes 1 and 4). Since the catalytic activity of telomerase is dependent on the <em>TLC1</em> RNA subcomponent, disruption of the <em>TLC1</em> gene would eliminate the telomerase activity. As expected, extracts from the KO spores did not show any extension of the primer, thus verifying the knockout of telomerase activity (Figure S1B, lanes 2-3).</p><p id="p-31">Immediately after germination of the spores, the <em>tlc1</em><sup>-</sup> KO colonies (correspond to p1) did not exhibit any difference in growth rate and colony morphology when compared to isogenic WT colonies (Figure S1A). Both WT and <em>tlc1</em><sup>-</sup> KO colonies formed bright and smooth colonies on the germination plates. In <em>S. cerevisiae</em>, telomerase-negative strains initially have WT-like colony shape and growth rate, but after continued propagation on plates they display senescence, typically at 80-100 generations (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Lundblad and Szostak 1989</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Lundblad and Blackburn 1993</a>). To detect any senescence phenotype of our <em>N. castellii</em> telomerase-negative strains, we performed passages of the <em>tlc1</em><sup>-</sup> strains on YPD plates, re-streaking plates every two days. In total 16 <em>tlc1</em><sup>-</sup> strains were streaked for 8 passages (p8 &gt;160 generations), and four of these strains were streaked for a total of 15 passages (p15 &gt;300 generations). Unexpectedly, we did not observe any senescence, neither any severe growth defect, nor any major colony morphology changes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A and B</a>, Figure S1C). A spot test assay was performed on serial dilutions of cell cultures from passages 2 to 15 (p2-p15), which confirmed that all passages showed a very similar growth capacity (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a>). However, in spite of a generally good growth ability, a slightly slower growth rate was observed during the passaging on plates, where <em>tlc1</em><sup>-</sup> strains required around half a day longer incubation time than WT cells to reach roughly the same colony size. To further investigate this, we performed a growth assay in liquid YPD media. Cultures of cells from passage 3 (p3) and 16 (p16) were inoculated to a concentration of 5 × 10<sup>5</sup> cells/ml and the cell density was counted after 24 hr of incubation. The results showed that <em>tlc1</em><sup>-</sup> cells exhibit a slightly lower overall growth rate compared to isogenic WT cells (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Telomerase-deficient N. castellii strains do not display any senescence phenotype. (A) Growth analysis of a diploid knockout strain (tlc1-/tlc1-) on YPD plates. The knockout strain and the isogenic wild-type (WT) strain from the same tetrad were continuously streaked onto YPD plates for &#x223C;300 generations. The different passage numbers are indicated. The initially haploid spores switch into diploid cells already in p1. (B) Close-up of colonies from passages 3 vs. 10 (p3, p10). (C) Yeast spot test assay of passages 2-15 (p2-p15) and isogenic WT, 10-fold dilutions. (D) Growth assays in liquid YPD media. Colonies from p3 and p16 were inoculated into liquid YPD media at a final concentration of 5 &#xD7; 105 cells/ml and the cell density was counted after 24 hr. Each column represents the mean value of ten independent experiments. Bars denote standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Telomerase-deficient &lt;em&gt;N. castellii&lt;/em&gt; strains do not display any senescence phenotype. (A) Growth analysis of a diploid knockout strain (&lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;/&lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) on YPD plates. The knockout strain and the isogenic wild-type (WT) strain from the same tetrad were continuously streaked onto YPD plates for &#x223C;300 generations. The different passage numbers are indicated. The initially haploid spores switch into diploid cells already in p1. (B) Close-up of colonies from passages 3 &lt;em&gt;vs.&lt;/em&gt; 10 (p3, p10). (C) Yeast spot test assay of passages 2-15 (p2-p15) and isogenic WT, 10-fold dilutions. (D) Growth assays in liquid YPD media. Colonies from p3 and p16 were inoculated into liquid YPD media at a final concentration of 5 &#xD7; 10&lt;sup&gt;5&lt;/sup&gt; cells/ml and the cell density was counted after 24 hr. Each column represents the mean value of ten independent experiments. Bars denote standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F1.medium.gif" width="440" height="421"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3345/F1.medium.gif" width="440" height="421"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494439" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">Telomerase-deficient <em>N. castellii</em> strains do not display any senescence phenotype. (A) Growth analysis of a diploid knockout strain (<em>tlc1</em><sup>-</sup>/<em>tlc1</em><sup>-</sup>) on YPD plates. The knockout strain and the isogenic wild-type (WT) strain from the same tetrad were continuously streaked onto YPD plates for ∼300 generations. The different passage numbers are indicated. The initially haploid spores switch into diploid cells already in p1. (B) Close-up of colonies from passages 3 <em>vs.</em> 10 (p3, p10). (C) Yeast spot test assay of passages 2-15 (p2-p15) and isogenic WT, 10-fold dilutions. (D) Growth assays in liquid YPD media. Colonies from p3 and p16 were inoculated into liquid YPD media at a final concentration of 5 × 10<sup>5</sup> cells/ml and the cell density was counted after 24 hr. Each column represents the mean value of ten independent experiments. Bars denote standard deviation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Even though we did not detect any widespread colony morphology defects, we noticed that the <em>tlc1</em><sup>-</sup> strains showed a population of small size colonies (0.5-1 mm) that differed noticeably from the WT colony size (1-3 mm). The colony size was not stable, since an ancestor with WT colony size generated both big and small colonies in next generations, and vice versa. Only rare colonies showed an uneven colony shape (“rough-edged”). Since the Y235 strain is able to switch mating type, the strains will eventually be able to mate and become diploid. Flow cytometry analysis confirmed that the majority of the cells in a culture became diploid already in p1, both KO and WT strains (data not shown). Since the mating occurs between isogenic cells, the strains become homozygous for the knockout allele (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>). In conclusion, our results show that the proliferative capacity of diploid <em>N. castellii</em> strains is only mildly affected by the knockout of telomerase.</p></div><div id="sec-17" class="subsection"><h3>Haploid N. castellii <em>tlc1</em><sup>-</sup> strains do not show any senescence phenotype</h3><p id="p-34">To investigate whether haploid telomerase-negative yeast strains would show the same survival capacity as the diploid, we used the same kanMX3 marker gene strategy to disrupt the <em>TLC1</em> gene in two stable haploid <em>N. castellii</em> strains (YMC48 and YMC63)(<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>). Three knockout transformants were further analyzed, and the successful disruption was verified by PCR amplification using primers annealing 0.5-0.6 kb up- and downstream of the <em>TLC1</em> gene, respectively (Figure S2).</p><p id="p-35">To analyze the growth ability, we passaged all three haploid <em>tlc1</em><sup>-</sup> strains on YPD plates for a total of 16 times, re-streaking every two days (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Similar to the diploid strain, we did not observe any senescence or severe abnormalities in colony morphology. However, haploid <em>tlc1</em><sup>-</sup> strains grew slightly slower than both their parental WT strains and the diploid <em>tlc1</em><sup>-</sup> strains, and required 1-2 days longer than the WT strain to form roughly the same colony size. Even though most colonies had WT colony morphology, we detected a small number of colonies with a slight uneven edge (rough-edged) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The rough-edged colonies appeared mostly in passage 6-7 (p6-7) and subsequently decreased. Overall, colonies had WT morphology and only very few rough-edged colonies could be detected. Although small size colonies appeared, they also occurred in WT streaks, and the numbers did not significantly differ.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haploid tlc1- N. castellii strains do not display any senescence phenotype. (A) Growth analysis of the haploid knockout strain (tlc1-) on YPD plates. The knockout strain YMC481 and the wild-type (WT) parental strain (YMC48) were continuously streaked onto YPD plates. The different passage numbers are indicated. (B) Close-up of colonies from p7, p8 and p14. Arrows indicate colonies showing a minor rough-edge phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haploid &lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt; N. castellii&lt;/em&gt; strains do not display any senescence phenotype. (A) Growth analysis of the haploid knockout strain (&lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) on YPD plates. The knockout strain YMC481 and the wild-type (WT) parental strain (YMC48) were continuously streaked onto YPD plates. The different passage numbers are indicated. (B) Close-up of colonies from p7, p8 and p14. Arrows indicate colonies showing a minor rough-edge phenotype.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F2.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3345/F2.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494528" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">Haploid <em>tlc1<sup>-</sup> N. castellii</em> strains do not display any senescence phenotype. (A) Growth analysis of the haploid knockout strain (<em>tlc1</em><sup>-</sup>) on YPD plates. The knockout strain YMC481 and the wild-type (WT) parental strain (YMC48) were continuously streaked onto YPD plates. The different passage numbers are indicated. (B) Close-up of colonies from p7, p8 and p14. Arrows indicate colonies showing a minor rough-edge phenotype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Together, our results show that neither diploid, nor haploid telomerase negative <em>N. castellii tlc1</em><sup>-</sup> strains exhibit any severe growth crisis when passaged for extensive numbers of generations. Furthermore, the lack of any marked senescence indicates a high frequency of telomerase-independent survivors.</p></div><div id="sec-18" class="subsection"><h3>Telomerase-negative cells exhibit rapid telomere loss followed by dramatic incremental lengthening</h3><p id="p-38">To study the telomere lengths in our telomerase-negative <em>N. castellii</em> strains, we performed Terminal Restriction Fragment (TRF) assays on colonies from each passage of the serial plate streaking procedure. Genomic DNA was extracted from each passage of five diploid homozygotes (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) and digested with <em>Hind</em>III enzyme. After electrophoresis and Southern blotting, telomeres were visualized by hybridization of the membrane with a 16-mer telomeric sequence probe (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). <em>Hind</em>III-treatment of WT DNA produced several hybridizing fragments ranging from 1 kb to 6 kb, where some strongly hybridizing fragments indicate that several telomeres are collected together in the same size range (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, lanes 1 and 17). The signal at ∼1.5 kb showed the smeariness that is characteristic of telomere fragments, which is due to the length variations between individual telomeres in the cell, as well as in a population of cells. In the <em>tlc1</em><sup>-</sup> strains, the same hybridizing bands were visible in the first few passages (p1-p3), but were observed to fade in signal with increasing number of passages. Moreover, the bands were gradually decreasing in size, hence indicating a gradual telomere shortening. Strikingly, most of the bands above 2 kb totally disappeared after p3 (corresponding to ∼75 generations) and the only remaining telomere signals in p4-p6 were found in two residual bands of approximately 1.2 and 1.5 kb length (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, lanes 5-8). Interestingly, a dramatically changed banding pattern appeared in later passages (typically in p7-9), constituted by several bands of increasing size, like a ladder (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, lane 9). The respective fragments of this ladder pattern showed an incremental size increase of approximately 300 bp, indicating that the ladder pattern may be formed by a repeated element. In combination with the appearance of the ladder pattern, the two residual 1.2/1.5 kb bands disappeared. It is interesting to note, that although the ladder pattern in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a> changes somewhat in the relative strength of the bands, it is present in all of the following passages (except in p9) (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A</a>, lanes 9-16). However, the ladder pattern occurred stochastically in the different strains (Figure S2C), and even varied in appearance between different preparations of the same strain.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Telomere attrition is followed by generation of a novel type of ALT pattern. Terminal restriction fragment (TRF) assays of diploid (tlc1-/tlc1-) and haploid (tlc1-) survivors. Genomic DNA was digested with HindIII, separated on a 0.8% agarose gel and analyzed by Southern blot hybridization with a 16-mer telomeric sequence probe. (A) Diploid knockout survivors (tlc1-/tlc1-) from passage 1-15 (lanes 2-16) of YMC130, and WT DNA (lanes 1 and 17). In lane 10 (p9), a possible sample mix-up cannot be excluded. (B) Haploid knockout survivors (YMC481, tlc1-) from passage 1-11 (lanes 2-12), and WT DNA (YMC48) (lanes 1 and 13)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Telomere attrition is followed by generation of a novel type of ALT pattern. Terminal restriction fragment (TRF) assays of diploid (&lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt;/tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) and haploid (&lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) survivors. Genomic DNA was digested with &lt;em&gt;Hind&lt;/em&gt;III, separated on a 0.8% agarose gel and analyzed by Southern blot hybridization with a 16-mer telomeric sequence probe. (A) Diploid knockout survivors (&lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt;/tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) from passage 1-15 (lanes 2-16) of YMC130, and WT DNA (lanes 1 and 17). In lane 10 (p9), a possible sample mix-up cannot be excluded. (B) Haploid knockout survivors (YMC481, &lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) from passage 1-11 (lanes 2-12), and WT DNA (YMC48) (lanes 1 and 13).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F3.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3345/F3.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494417" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Telomere attrition is followed by generation of a novel type of ALT pattern. Terminal restriction fragment (TRF) assays of diploid (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) and haploid (<em>tlc1</em><sup>-</sup>) survivors. Genomic DNA was digested with <em>Hind</em>III, separated on a 0.8% agarose gel and analyzed by Southern blot hybridization with a 16-mer telomeric sequence probe. (A) Diploid knockout survivors (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) from passage 1-15 (lanes 2-16) of YMC130, and WT DNA (lanes 1 and 17). In lane 10 (p9), a possible sample mix-up cannot be excluded. (B) Haploid knockout survivors (YMC481, <em>tlc1</em><sup>-</sup>) from passage 1-11 (lanes 2-12), and WT DNA (YMC48) (lanes 1 and 13).</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The incremental ladder pattern exhibits an interesting profile. To investigate its characteristics, we hybridized the same membrane from above with a telomeric probe of the complementary strand, <em>i.e.</em>, the AC-rich strand (data not shown). This revealed the exact same TRF pattern as with the TG-rich telomeric probe. Since hybridization to both telomeric strands produced the same hybridizing fragments, we conclude that the <em>Hind</em>III-fragments of the ladder pattern contain double-stranded telomeric DNA.</p><p id="p-41">Next, we wanted to investigate telomere attrition in the haploid <em>tlc1</em><sup>-</sup> strains. We made TRF assays of genomic DNA extracted from the three haploid <em>tlc1</em><sup>-</sup> strains using the same <em>Hind</em>III digestion (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a> and S2D). Consistent with the diploid, the haploid WT DNA exhibits several strongly hybridizing fragments between 1-6 kb. In the first three passages a gradual telomere shortening is visible by the shortening of the fragments, and eventually the disappearance of most of them (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B</a>, lanes 2-4). Notably, most of the remaining signal then appeared in a short defined fragment of ∼1.2 kb that remains through all the subsequent passages. Strikingly, a similar ladder pattern as previously seen in the diploid <em>tlc1</em><sup>-</sup> strains was observed in p9 (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a>, lane 10). However, in a longer exposure, a faint ladder pattern can be visualized also in p6-p11, thus indicating a similar ladder pattern signal profile as in the diploid. In contrast to the diploid strain, however, the short fragment of ∼1.2 kb is still visible and has a strong hybridization signal through all passages in the haploid <em>tlc1</em><sup>-</sup> strain (see also Figure S2D).</p><p id="p-42">In summary, both haploid and diploid <em>tlc1</em><sup>-</sup> knockout strains of <em>N. castellii</em> experience a rapid progressive shortening of telomeres. However, in later generation survivors, the TRF assay reveals a dramatic telomere phenomenon, where a ladder pattern is generated by band sizes increasing by ∼300 bp increments. The fact that the ladder pattern includes double-stranded telomeric sequences suggests that it is produced by an alternative telomere lengthening mechanism. Intriguingly, this is a novel type of ALT lengthening pattern, differing from the Type I and Type II patterns previously described in <em>S. cerevisiae</em>.</p></div><div id="sec-19" class="subsection"><h3>EST1 knockout confirms that lack of a major growth crisis is a general response when telomerase is inactivated in N. castellii cells</h3><p id="p-43">The lack of a detectable major growth crisis in our <em>tlc1</em><sup>-</sup> strains of <em>N. castellii</em> is in great contrast to the phenotype previously found in <em>S. cerevisiae</em> (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Lundblad and Szostak 1989</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">McEachern and Blackburn 1996</a>). We wanted to investigate whether this high survival frequency is a general mechanism in response to a telomerase lengthening deficiency. Therefore, we aimed to delete the <em>EST1</em> gene, which is encoding a protein sub-component of the telomerase holoenzyme. Notably, the discovery of the characteristic delayed senescence phenotype of telomerase-deficient strains was made in <em>S. cerevisiae </em><a href="https://identifiers.org/bioentitylink/SGD:S000004223?doi=10.1534/g3.119.400428"><em>est1</em></a>Δ mutants and was termed the <em>est</em> (ever shortening telomere) phenotype (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Lundblad and Szostak 1989</a>).</p><p id="p-44">We constructed an <em>EST1</em> deletion cassette with 45 bp homologous sequences (Figure S3A). In order to facilitate the correct targeting and gene replacement, we used the “split-marker” approach, in which three-way recombination occurs to simultaneously insert and assemble the <em>K. lactis URA3</em> marker gene (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Reid <em>et al.</em> 2002</a>; <a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Karademir Andersson <em>et al.</em> 2016</a>). The two cassettes overlap with a 452 bp complementary sequence within the <em>klURA3</em> marker gene. The deletion construct was transformed into a diploid <em>ura3</em><sup>-</sup> strain (Y235) and six transformants were selected on drop-out plates lacking uracil. Deletion of the <em>EST1</em> gene was verified by PCR, using primers annealing 0.5 kb up- and downstream of the <em>EST1</em> gene, respectively. Subsequently, four correct transformants were sporulated and one ascus of each transformant was micro-dissected into tetrads (Figure S3B). The genotype of all germinated spores was verified by PCR amplification, where the replacement of the wild-type (WT) 3.2 kb band with a 2.5 kb knockout (KO) band showed that each tetrad contained two WT spores (<em>EST1</em>) and two KO spores (<em>est1</em>Δ::<em>URA3</em>) (Figure S3C). In order to further confirm the correct <em>EST1</em> deletion, we analyzed all four tetrads by Southern blot hybridization (Figure S3D). Hybridization with a <em>klURA3</em> probe, showed the expected 3 kb <em>BamH</em>I-generated band in the KO strains (<em>est1</em>Δ::<em>URA3</em>), which was lacking in the WT strains (<em>EST1</em>).</p><p id="p-45">The <em>est1</em>Δ spores were viable immediately after germination and they did not exhibit any major difference in growth rate and colony morphology when compared to isogenic WT spores (Figure S3B). Flow cytometry analysis showed that the majority of the cells in p1 cultures of both KO and WT strains are diploid (data not shown). We passaged all four tetrads 16 times on YPD plates every two days (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). In accordance with the <em>tlc1</em><sup>-</sup> strains, we did not observe any senescence or any major abnormalities in colony morphology, and we could not detect any rough-edged colonies. Moreover, <em>est1</em>Δ knockouts had a similar growth rate on YPD plates and they showed a WT colony size after two days incubation. Taken together, these results show that bypassing the cellular senescence without a major growth crisis is a general phenotypic response in telomerase-negative <em>N. castellii</em> strains.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deletion of EST1 leads to the generation of the same type of ALT survivors as for the tlc1- knockouts. (A) Growth analysis by continuous passaging of diploid (est1&#x394;/est1&#x394;) survivors and the wild-type (WT) strain onto YPD plates. Passage numbers are indicated. (B) TRF assay by HindIII digestion and Southern blot hybridization with a telomeric probe. Diploid (est1&#x394;/est1&#x394;) survivors from p2-p16 (lanes 2-16), and WT (lanes 1 and 17) YMC303 from tetrad 1 (Figure S3E)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deletion of &lt;em&gt;EST1&lt;/em&gt; leads to the generation of the same type of ALT survivors as for the &lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt; knockouts. (A) Growth analysis by continuous passaging of diploid (&lt;em&gt;est1&lt;/em&gt;&#x394;/&lt;em&gt;est1&lt;/em&gt;&#x394;) survivors and the wild-type (WT) strain onto YPD plates. Passage numbers are indicated. (B) TRF assay by &lt;em&gt;Hind&lt;/em&gt;III digestion and Southern blot hybridization with a telomeric probe. Diploid (&lt;em&gt;est1&lt;/em&gt;&#x394;/&lt;em&gt;est1&lt;/em&gt;&#x394;) survivors from p2-p16 (lanes 2-16), and WT (lanes 1 and 17) YMC303 from tetrad 1 (Figure S3E).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F4.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3345/F4.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494301" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Deletion of <em>EST1</em> leads to the generation of the same type of ALT survivors as for the <em>tlc1</em><sup>-</sup> knockouts. (A) Growth analysis by continuous passaging of diploid (<em>est1</em>Δ/<em>est1</em>Δ) survivors and the wild-type (WT) strain onto YPD plates. Passage numbers are indicated. (B) TRF assay by <em>Hind</em>III digestion and Southern blot hybridization with a telomeric probe. Diploid (<em>est1</em>Δ/<em>est1</em>Δ) survivors from p2-p16 (lanes 2-16), and WT (lanes 1 and 17) YMC303 from tetrad 1 (Figure S3E).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>The ladder pattern is a general phenomenon in N. castellii ALT cells</h3><p id="p-47">In order to determine whether telomerase-negative <em>N. castellii</em> cells use a common ALT pathway to maintain telomeres, we performed TRF assays of two of the diploid <em>est1</em>Δ strains derived from separate tetrads (Figure S3B, tetrad 1 and 2). DNA extracted from several passages was analyzed in the same way as described above for the <em>tlc1</em><sup>-</sup> strains (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a> and Figure S3E). Intriguingly, we observed a similar rapid telomere attrition, which was then followed by generation of the ladder pattern phenomenon in later passages. As before, most of the original 1-6 kb telomeric bands disappeared after passage 4, leaving two very short bands at ∼1.2 kb and ∼1.4 kb that are kept throughout a majority of the passages (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>, lane 6). The ladder pattern appears strongly in p9-p10, but can be observed as faint signals also in p7-p8 (as well as p12 in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a> and p11 in Figure S3E). Strikingly, in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>, p11 exhibits an extremely high signal, which is collected into a high molecular weight smear (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>, lane 11). Since some signal is also trapped in the well, it indicates the presence of extremely long DNA fragments that are not resolved in the gel. It is therefore tempting to speculate that the telomeres in the p11 cell population are super-elongated.</p><p id="p-48">Taken together, the telomere analyses of <em>N. castellii</em> telomerase-negative cells reveal overall similar results for the <em>tlc1</em><sup>-</sup> and <em>est1</em>Δ strains. Both knockout mutants underwent a very rapid progressive telomere shortening, followed by telomere rearrangement and lengthening. Two different TRF assay patterns are generated, that show distinct and characteristic profiles. The pattern appearing in early generations constitutes mainly short hybridizing fragments of ∼1.2 kb and ∼1.4 kb. In contrast, in the later generations a striking ladder pattern emerges with incrementally longer fragments reaching sizes even beyond 10 kb. The ladder pattern appears transiently, and may stay on or disappear in the following passages, indicating stochastic ALT lengthening by specific repeated elements. In conclusion, a similar maintenance of telomeres was observed in both <em>tlc1</em><sup>-</sup> and <em>est1</em>Δ mutants, indicating a general ALT response in the absence of a functional telomerase enzyme. Moreover, the absence of a major growth crisis suggests that this ALT mechanism is quickly and easily activated in <em>N. castellii</em> telomerase-negative cells.</p></div><div id="sec-21" class="subsection"><h3>Telomerase-negative cells maintain linear chromosomes</h3><p id="p-49">The fission yeast <em>Schizosaccharomyces pombe</em> has been shown to produce telomerase-negative survivors by using chromosome circularization (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nakamura <em>et al.</em> 1998</a>). To investigate whether <em>N. castellii tlc1</em><sup>-</sup> cells still maintain linear chromosomes, we performed Pulsed Field Gel Electrophoresis (PFGE), as only linear chromosomes will enter the PFGE gel (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). To analyze clearly established survivors, we chose <em>tlc1</em><sup>-</sup> survivors from passage 9. As positive controls for linear chromosomes, we analyzed karyotypes of the parental strain and two isogenic WT strains obtained from the same tetrad, which all showed eight chromosomal bands in the PFGE (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>, lanes 1 and 3-4). As <em>N. castellii</em> was previously shown to have 10 chromosomes in the whole genome sequencing approach, probably some of the chromosomes are not resolving on the PFGE gel (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Cliften <em>et al.</em> 2006</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Gordon <em>et al.</em> 2011</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Karademir Andersson and Cohn 2017</a>). This notion is supported by the variation in signal intensity of the bands, suggesting that some bands include overlapping chromosomes. This is further supported by previous karyotyping experiments, where PFGE displayed eight or nine bands on the gel (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Petersen <em>et al.</em> 1999</a>). Notably, the chromosomes of the <em>tlc1</em><sup>-</sup> strains were resolved into the same eight bands as WT, thus showing that <em>tlc1</em><sup>-</sup> survivors maintain linear chromosomes (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>, lanes 2 and 5).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="N. castellii ALT cells maintain linear chromosomes. Separation of whole chromosomes by pulsed field gel electrophoresis. (A, B) Diploid (tlc1-/tlc1-) strains. Two diploid knockouts (tlc1-/tlc1-) from p9 (lanes 2 and 5), and two isogenic wild-type (WT) strains obtained from the same tetrad (YMC121-124) (lanes 3-4). Y235 is the parental strain for the diploid KO (lane 1). (C, D) Haploid (tlc1-) strain YMC481 from p5 (lane 2), and its WT parental strain YMC48 (lanes 1 and 3). (A and C) Gels stained with EtBr. (B and D) Southern blot and hybridization of the membranes with a telomeric probe." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;N. castellii&lt;/em&gt; ALT cells maintain linear chromosomes. Separation of whole chromosomes by pulsed field gel electrophoresis. (A, B) Diploid (&lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt;/tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) strains. Two diploid knockouts (&lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt;/tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) from p9 (lanes 2 and 5), and two isogenic wild-type (WT) strains obtained from the same tetrad (YMC121-124) (lanes 3-4). Y235 is the parental strain for the diploid KO (lane 1). (C, D) Haploid (&lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) strain YMC481 from p5 (lane 2), and its WT parental strain YMC48 (lanes 1 and 3). (A and C) Gels stained with EtBr. (B and D) Southern blot and hybridization of the membranes with a telomeric probe.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F5.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3345/F5.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494518" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child"><em>N. castellii</em> ALT cells maintain linear chromosomes. Separation of whole chromosomes by pulsed field gel electrophoresis. (A, B) Diploid (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) strains. Two diploid knockouts (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) from p9 (lanes 2 and 5), and two isogenic wild-type (WT) strains obtained from the same tetrad (YMC121-124) (lanes 3-4). Y235 is the parental strain for the diploid KO (lane 1). (C, D) Haploid (<em>tlc1</em><sup>-</sup>) strain YMC481 from p5 (lane 2), and its WT parental strain YMC48 (lanes 1 and 3). (A and C) Gels stained with EtBr. (B and D) Southern blot and hybridization of the membranes with a telomeric probe.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">Next, we investigated the presence of telomeric DNA in survivor chromosomes, by performing a Southern blot of the PFGE gel and hybridization with the telomeric probe (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). All chromosome bands of the WT strains strongly hybridized with the telomeric probe (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>, lanes 1 and 3-4). Similarly, all the bands of the knockouts (KO) exhibited hybridization signal, showing that all chromosomes indeed contain telomeric DNA. However, as expected, a lower signal intensity was obtained in the KO chromosomes, thus confirming the loss of telomere sequences.</p><p id="p-52">PFGE analysis of haploid <em>tlc1</em><sup>-</sup> KO cells (passage 5) showed the same result as for the diploid cells (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>). The same eight chromosomal bands were visible both in the haploid parental WT strain and the KO strain. Similarly, a telomeric hybridization signal was observed for all bands, although the KO gives a lower signal intensity (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5D</a>). Notably, since the Ethidium Bromide stained PFGE gel showed a comparable signal intensity for the WT and KO strains, the lower hybridization signal indicates a quite high loss of telomeric DNA sequences in this KO strain. Interestingly, the smallest chromosome was observed to shift upwards in the KO strains. However, we were not able to correllate this phenomenon to the survivors, since we have observed variations in the migration of this chromosome.</p><p id="p-53">In conclusion, our results show that telomerase-deficient <em>N. castellii</em> survivors are able to maintain linear chromosomes, and they do not exhibit any gross chromosomal changes. Notably, even though a loss of telomeric DNA is observed, all chromosome bands still show some remaining telomeric sequences, indicating that at least one of the telomeres of each chromosome contains some telomeric DNA.</p></div><div id="sec-22" class="subsection"><h3>The ALT mechanism generates distinct repeats that are terminally located</h3><p id="p-54">The ladder pattern that was observed in the late passages of all the strains, implies that long repeats of a specific length are added to the telomeres in the survivors. To address whether the DNA fragments in the ladder pattern are terminally located on the chromosomes, we performed assays with the exonuclease Bal31, which cleaves dsDNA from the terminal ends (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). In WT control DNA, cleavage with Bal31 exonuclease leads to a decreasing size of telomeric fragments with increasing incubation time, thus indicating their terminal location (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fridholm <em>et al.</em> 2013</a>). Here, gDNA from a diploid <em>tlc1</em><sup>-</sup> strain (p11) was incubated with Bal31 for increasing periods of time, followed by the TRF assay procedure, <em>i.e.</em>, digested with <em>Hind</em>III, blotting and hybridization with a telomeric probe. The telomeres of this strain showed a clear ladder pattern (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>, lanes 1 and 10). Intriguingly, these ladder pattern fragments were sensitive to Bal31 treatment, as shown by a progressive shortening in size and eventually disappearance (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a>, lanes 2-9). Thus, we could conclude that the ladder pattern fragments are terminally positioned on the chromosomes, which indicates that they are products of the ALT lengthening mechanism.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bal31 assays show that fragments of the ladder pattern are terminally located. (A) Diploid (tlc1-/tlc1-) survivors from p11. DNA was cleaved with Bal31 for increasing periods of time and then treated as in the TRF assay, using HindIII. (B) Bal31 assay using DdeI of diploid (tlc1-/tlc1-) survivors from p11 having the ladder pattern (lanes 2-9). DdeI generates two bands of &#x223C;275 bp and &#x223C;120 bp, respectively (lanes 1 and 11; 0 min). WT DNA shows a &#x223C;400 bp smear (lane 10). (C) TRF assay using DdeI of four independent colonies of diploid (tlc1-/tlc1-) survivors from p11, which do not generate the ladder pattern (lanes 1-4), and WT (lane 5). These survivors lack the &#x223C;275 bp band previously observed in the ladder pattern survivors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bal31 assays show that fragments of the ladder pattern are terminally located. (A) Diploid (&lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt;/tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) survivors from p11. DNA was cleaved with Bal31 for increasing periods of time and then treated as in the TRF assay, using &lt;em&gt;Hind&lt;/em&gt;III. (B) Bal31 assay using &lt;em&gt;Dde&lt;/em&gt;I of diploid (&lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt;/tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) survivors from p11 having the ladder pattern (lanes 2-9). &lt;em&gt;Dde&lt;/em&gt;I generates two bands of &#x223C;275 bp and &#x223C;120 bp, respectively (lanes 1 and 11; 0 min). WT DNA shows a &#x223C;400 bp smear (lane 10). (C) TRF assay using &lt;em&gt;Dde&lt;/em&gt;I of four independent colonies of diploid (&lt;em&gt;tlc1&lt;sup&gt;-&lt;/sup&gt;/tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt;) survivors from p11, which do not generate the ladder pattern (lanes 1-4), and WT (lane 5). These survivors lack the &#x223C;275 bp band previously observed in the ladder pattern survivors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F6.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/10/3345/F6.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494507" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-55" class="first-child">Bal31 assays show that fragments of the ladder pattern are terminally located. (A) Diploid (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) survivors from p11. DNA was cleaved with Bal31 for increasing periods of time and then treated as in the TRF assay, using <em>Hind</em>III. (B) Bal31 assay using <em>Dde</em>I of diploid (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) survivors from p11 having the ladder pattern (lanes 2-9). <em>Dde</em>I generates two bands of ∼275 bp and ∼120 bp, respectively (lanes 1 and 11; 0 min). WT DNA shows a ∼400 bp smear (lane 10). (C) TRF assay using <em>Dde</em>I of four independent colonies of diploid (<em>tlc1<sup>-</sup>/tlc1</em><sup>-</sup>) survivors from p11, which do not generate the ladder pattern (lanes 1-4), and WT (lane 5). These survivors lack the ∼275 bp band previously observed in the ladder pattern survivors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">To analyze the structural change in the telomeres in more detail, we wanted to use an enzyme that cleaves more closely to the terminus than <em>Hind</em>III, to release a shorter and potentially more homogeneous terminal fragment. <em>Dde</em>I was found to collect all of the WT telomeres into fragments centered around ∼400 bp (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>, lane 10). Due to the variation in telomere length, the Southern blot hybridization shows an expected smeary pattern, ranging between 250-600 bp. Thus, the <em>Dde</em>I target site is situated very close to the ∼250 bp telomeric sequences. Remarkably, the ALT strain that previously showed the <em>Hind</em>III ladder pattern generated a very different result using <em>Dde</em>I, with two distinct telomeric fragments appearing at ∼275 bp and ∼120 bp, respectively (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>, lanes 1 and 11). Thus, all the previously observed <em>Hind</em>III ladder pattern fragments are collected into two distinct bands by <em>Dde</em>I. This indicates that the ALT telomeres exhibit a total reorganization where the telomeric sequences are incorporated into longer repeated units.</p><p id="p-57">To determine whether these two distinct <em>Dde</em>I-fragments originate from sequences located terminally on the chromosomes, we performed a Bal31 assay (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6B</a>). The ∼120 bp fragment was sensitive to Bal31 treatment, as it gradually decreased in signal and totally disappeared after two hours of incubation (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6B</a>, lanes 2-9). Hence, we conclude that the ∼120 bp <em>Dde</em>I-fragment is terminally located. On the other hand, the hybridizing fragment of ∼275 bp was more resistant to Bal31 digestion. Its signal was only slightly diminished after two hours, which indicates more internally located fragments or repeated elements that are present in long arrays (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6B</a>, lanes 2-9).</p><p id="p-58">Since the total telomeric signal is collected into the two <em>Dde</em>I fragments, it suggested that at least one of them would also be part of the <em>Hind</em>III ladder pattern. In order to test this hypothesis, we wanted to analyze gDNA extracted from ALT cells that are not producing any <em>Hind</em>III ladder pattern. Indeed, the ladder pattern is observed to occur stochastically and appears at a low frequency, and even two colonies from the same streak may differ regarding the presence of the ladder pattern. The <em>Hind</em>III-digested gDNA of four independent colonies from the same plate streak of the diploid <em>tlc1</em><sup>-</sup> strain (p11) were analyzed by TRF assay and the absence of the ladder pattern was confirmed (data not shown). We then performed the TRF assay using <em>Dde</em>I on these particular gDNA. As previously, the telomeric fragments of the WT DNA were collected into a smeary region at ∼400 bp with the <em>Dde</em>I digestion (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6C</a>, lane 5). In contrast, the telomerase KO survivors generated a single prominent hybridizing fragment of ∼120 bp. Notably, the ∼275 bp band previously observed in the ladder pattern survivors was absent in these samples (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6C</a>, lanes 1-4). Therefore, we conclude that the ∼275 bp fragment is clearly coupled to the ladder pattern, and each distinct telomeric fragment in the ladder pattern should contain at least one copy of this particular element.</p></div><div id="sec-23" class="subsection"><h3>Telomerase-negative cells maintain the telomere-adjacent subtelomeric region and short telomeric sequences at chromosome ends</h3><p id="p-59">Next, we wanted to reveal what DNA sequence is present at the terminus of the ALT telomeres. To that end, we performed two separate approaches. First, a PCR based technique, the so-called telomere-PCR method (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Forstemann <em>et al.</em> 2000</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wahlin <em>et al.</em> 2003</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2014</a>), and second a direct telomere cloning procedure. In telomere-PCR, the ends of genomic DNA was C-tailed using terminal transferase and dCTP. PCR amplification was performed using a forward primer targeting a previously identified subtelomeric region and a reverse poly(G)<sub>18</sub> primer annealing to the C-tail. On an agarose gel, PCR amplification of both haploid and diploid WT strains yielded a broad band of ∼850 bp (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>, lanes 3 and 6). In addition, a weak and smeared band of ∼1.6 kb could be observed. An additional weak ∼150 bp band was determined as a background since it also appeared in the non-tailed control gDNA (data not shown).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ALT cells maintain the same overall terminal DNA structure as WT cells, but have shorter telomeric DNA sequences. (A) Telomere-PCR analysis of C-tailed genomic DNA, WT diploid (lane 3), WT haploid (lane 6), diploid vs. haploid tlc1- survivors (lanes 4-5 vs. 7). KO DNA both with (lane 4) and without (lane 5) a ladder pattern was analyzed. DNA size markers (lanes 1, 2, 8). The respective &#x223C;850 bp vs. &#x223C;650 bp bands of the WT and tlc1- samples were gel-extracted and sequenced. (B and C) Schematic representation of the terminal telomere sequences in WT cells and tlc1- ALT cells (KO). (D) Schematic showing the terminal structure of cells with ALT elongation (a ladder pattern strain). ITS, denotes the 9 bp interstitial telomeric sequences. TelKO, denotes the telomere-adjacent subtelomeric region flanked by ITS. The HinfI and DdeI recognition sites are indicated, together with the fragment sizes that DdeI generates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;ALT cells maintain the same overall terminal DNA structure as WT cells, but have shorter telomeric DNA sequences. (A) Telomere-PCR analysis of C-tailed genomic DNA, WT diploid (lane 3), WT haploid (lane 6), diploid &lt;em&gt;vs.&lt;/em&gt; haploid &lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt; survivors (lanes 4-5 &lt;em&gt;vs.&lt;/em&gt; 7). KO DNA both with (lane 4) and without (lane 5) a ladder pattern was analyzed. DNA size markers (lanes 1, 2, 8). The respective &#x223C;850 bp &lt;em&gt;vs.&lt;/em&gt; &#x223C;650 bp bands of the WT and &lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt; samples were gel-extracted and sequenced. (B and C) Schematic representation of the terminal telomere sequences in WT cells and &lt;em&gt;tlc1&lt;/em&gt;&lt;sup&gt;-&lt;/sup&gt; ALT cells (KO). (D) Schematic showing the terminal structure of cells with ALT elongation (a ladder pattern strain). ITS, denotes the 9 bp interstitial telomeric sequences. TelKO, denotes the telomere-adjacent subtelomeric region flanked by ITS. The &lt;em&gt;Hinf&lt;/em&gt;I and &lt;em&gt;Dde&lt;/em&gt;I recognition sites are indicated, together with the fragment sizes that &lt;em&gt;Dde&lt;/em&gt;I generates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F7.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/10/3345/F7.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494448" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-60" class="first-child">ALT cells maintain the same overall terminal DNA structure as WT cells, but have shorter telomeric DNA sequences. (A) Telomere-PCR analysis of C-tailed genomic DNA, WT diploid (lane 3), WT haploid (lane 6), diploid <em>vs.</em> haploid <em>tlc1</em><sup>-</sup> survivors (lanes 4-5 <em>vs.</em> 7). KO DNA both with (lane 4) and without (lane 5) a ladder pattern was analyzed. DNA size markers (lanes 1, 2, 8). The respective ∼850 bp <em>vs.</em> ∼650 bp bands of the WT and <em>tlc1</em><sup>-</sup> samples were gel-extracted and sequenced. (B and C) Schematic representation of the terminal telomere sequences in WT cells and <em>tlc1</em><sup>-</sup> ALT cells (KO). (D) Schematic showing the terminal structure of cells with ALT elongation (a ladder pattern strain). ITS, denotes the 9 bp interstitial telomeric sequences. TelKO, denotes the telomere-adjacent subtelomeric region flanked by ITS. The <em>Hinf</em>I and <em>Dde</em>I recognition sites are indicated, together with the fragment sizes that <em>Dde</em>I generates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">Telomere-PCR of the <em>tlc1</em><sup>-</sup> strains similarly produced two bands, but with a smaller size, ∼650 bp and ∼1.4 kb (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>, lanes 4, 5 and 7). Notably, the ∼650 bp band appears quite distinct in both the KO strain containing, and the one lacking a ladder pattern (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a>, lanes 4 and 5, respectively). It is worth noting that the size difference of the most prominent bands in the WT and KO strains is 200 bp, which would correspond to the shortening of the telomere sequences. To analyze whether these products contain terminal sequences of the chromosome ends, we gel-extracted the ∼850 bp and ∼650 bp bands from WT and KO strains, respectively.</p><p id="p-62">Sequencing by using various primers targeting the subtelomeric region revealed that both WT and <em>tlc1</em><sup>-</sup> KO strains contain the same 220 bp subtelomeric region, which we termed the TelKO element (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7B</a>, Figure S4A). In the WT telomeres, the TelKO element is residing just interior of the terminal telomeric DNA sequences. Thus, we have here determined the TelKO region to be the most telomere-proximal subtelomeric sequence. Notably, it contains a <em>Dde</em>I recognition site, which is the site that releases the ∼400 bp telomere smear (<a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6B</a>). Moreover, the TelKO element is flanked by a short 9 bp interstitial telomeric sequence (ITS), defining a boundary to the continued subtelomeric sequences. Strikingly, also in the ALT strains, the TelKO element is located just internally of a telomeric sequence that is terminally located. However, the stretch of telomere sequence is shorter, around 50 bp (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7C</a>, Figure S4A).</p><p id="p-63">The telomere-PCR of the survivors was performed on gDNA extracted from p11, and thus represent clearly established ALT strains. Since we could identify the C-tail in our sequencing results, we could determine that the telomere sequence stretch of ∼50 bp was indeed terminally located. Even though the exact number of basepairs in the telomere sequence differed in the separate sequencing samples, the TelKO element was invariably located interior of it (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7C</a>). We also obtained the same sequence organization from other primer combinations in the telomere-PCR, thus confirming our results (data not shown). Therefore, we conclude that the ALT cells indeed contain a short telomeric stretch (∼50 bp) at the very end of the chromosome, and that the telomere-adjacent region (TelKO element) is also maintained.</p><p id="p-64">Second, we performed a direct cloning of telomeres. Since we concluded that the <em>Dde</em>I fragments reside very close to the terminus, we set out to clone telomeres by constructing a sub-genomic library of <em>Dde</em>I-fragments. DNA extracted from survivors was digested with <em>Dde</em>I, separated on an agarose gel and the fragments with appropriate sizes were gel-extracted and cloned. <em>E. coli</em> clones were screened by colony hybridization with a telomeric probe and positive clones were analyzed further by plasmid purification and slot blot hybridization, resulting in three clones with a strong signal. Sequencing showed that the clones contained short stretches of telomeric sequences of different lengths (27 bp, 55 bp and 56 bp) (Figure S4B and S4C). Moreover, they all contained a part of the TelKO element (64-65 bp, termed TelKO-A). Two of the clones additionally shared another part of the TelKO element (TelKO-B), flanking the other side of the telomeric sequence. The organization of these cloned <em>Dde</em>I fragments, with telomeric sequences flanked by upstream and downstream parts of the TelKO element, indicate that they emanated from internal positions in the ALT-extended telomeres.</p><p id="p-65">In summary, the two isolation approaches together revealed important features of the survivor telomeres. First, the knockouts still maintain telomeric sequences at the extreme termini, although drastically shortened. Second, the presence of ∼50 bp internally positioned telomeric sequences, which are flanked by the most terminal subtelomeric element (TelKO), indicates that telomeres are lengthened by repeated elements containing both telomeric sequences and the telomere-adjacent TelKO element. The assembly of the sequence information allowed us to construct a schematic model for the elongated array structure of the ALT telomeres (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7D</a>). The situation where two consecutive TelKO elements are flanking a ∼50 bp telomeric sequence creates two <em>Dde</em>I fragments containing telomeric sequences; the ∼275 bp internal fragment and the ∼120 bp terminal fragment (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7D</a>). These fragments correspond to the previously observed <em>Dde</em>I bands in the TRF assay, which therefore confirms the structure (<a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6B</a>). Significantly, this data suggests that the ladder pattern observed in the TRF assay is generated by the addition of ∼275 bp elements onto the ends of ALT chromosomes.</p></div><div id="sec-24" class="subsection"><h3>ALT telomeres are extended with a specific ∼275 bp element containing both subtelomeric and telomeric DNA</h3><p id="p-66">The sequence analyses of the cloned telomere fragments suggested that the <em>N. castellii</em> ALT chromosomes are extended with ∼275 bp repeats containing the subtelomeric TelKO element and telomeric sequences. To further confirm the proposed structure and organization of the ALT telomeres, we made a TRF assay using various restriction enzymes (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). We selected an ALT strain exhibiting the characteristic ladder pattern when digested with <em>Hind</em>III, where the size of the bands is increasing in ∼275 bp increments (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>, lane 1). The <em>Hinf</em>I site is located more terminally than the <em>Hind</em>III site, and is therefore expected to release shorter terminal fragments. As expected, <em>Hinf</em>I digestion is shifting down the ladder pattern into smaller fragment, although maintaining the same incremental steps, hence confirming that the repeats are terminally located (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>, lane 3). In stark contrast, <em>Dde</em>I digestion of the ALT strain collects the telomeric signal into the two distinct bands of ∼275 bp and ∼120 bp, as shown previously (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a>, lane 2). Double digestions, further confirm this structure, since the more terminally located <em>Dde</em>I and <em>Hinf</em>I sites overrule the more internally located <em>Hind</em>III site (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8</a>, lanes 4-5).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="TRF assay using HinfI and DdeI confirms that ALT cell telomeres are elongated by &#x223C;275 bp TelKO elements. An ALT strain exhibiting the HindIII ladder pattern was digested with the HinfI and DdeI enzymes in single vs. double digestions, as indicated. Note that HinfI cuts more terminally than HindIII, thus shifting down the ladder pattern. The sizes of some bands in the HinfI ladder are indicated, as well as the specific &#x223C;275 bp DdeI band that is only observed in ALT strains having the ladder pattern, i.e., that are elongated by TelKO elements." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-578091624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;TRF assay using &lt;em&gt;Hinf&lt;/em&gt;I and &lt;em&gt;Dde&lt;/em&gt;I confirms that ALT cell telomeres are elongated by &#x223C;275 bp TelKO elements. An ALT strain exhibiting the &lt;em&gt;Hind&lt;/em&gt;III ladder pattern was digested with the &lt;em&gt;Hinf&lt;/em&gt;I and &lt;em&gt;Dde&lt;/em&gt;I enzymes in single &lt;em&gt;vs.&lt;/em&gt; double digestions, as indicated. Note that &lt;em&gt;Hinf&lt;/em&gt;I cuts more terminally than &lt;em&gt;Hind&lt;/em&gt;III, thus shifting down the ladder pattern. The sizes of some bands in the &lt;em&gt;Hinf&lt;/em&gt;I ladder are indicated, as well as the specific &#x223C;275 bp &lt;em&gt;Dde&lt;/em&gt;I band that is only observed in ALT strains having the ladder pattern, &lt;em&gt;i.e.&lt;/em&gt;, that are elongated by TelKO elements.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3345/F8.medium.gif" width="163" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/10/3345/F8.medium.gif" width="163" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3345/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494484" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-67" class="first-child">TRF assay using <em>Hinf</em>I and <em>Dde</em>I confirms that ALT cell telomeres are elongated by ∼275 bp TelKO elements. An ALT strain exhibiting the <em>Hind</em>III ladder pattern was digested with the <em>Hinf</em>I and <em>Dde</em>I enzymes in single <em>vs.</em> double digestions, as indicated. Note that <em>Hinf</em>I cuts more terminally than <em>Hind</em>III, thus shifting down the ladder pattern. The sizes of some bands in the <em>Hinf</em>I ladder are indicated, as well as the specific ∼275 bp <em>Dde</em>I band that is only observed in ALT strains having the ladder pattern, <em>i.e.</em>, that are elongated by TelKO elements.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">To further confirm the subtelomeric structure, we rehybridized the membrane with a probe annealing to the TelKO element (Figure S5). As expected, the TelKO element hybridized to the ∼275 bp <em>Dde</em>I fragment of the ALT strain. Taken together, the specific band patterns obtained with various restriction enzymes in the TRF assay confirm the structure obtained from the telomere-PCR sequence data. Please see the Supplementary Figure S5 for the full description of the confirmation.</p><p id="p-69">The addition of the long arrays of repeated ∼275 bp elements appears rather dramatic in the TRF assay, but may in theory be derived from a few lengthening events. To gain information regarding the extent of the elongation events, we quantified the amount of TelKO elements in the genome by slot blot hybridization. Serially diluted genomic DNA, both from ALT cells with a ladder pattern profile and without ladder profile was put on the membrane, together with the WT <em>N. castellii</em> Y-12630 (type strain) and hybridization controls (Figure S6). First, we performed hybridization of the membrane with the telomere probe, which showed that the amount of telomeric sequence is markedly decreased in the ALT cells compared to the WT. Moreover, a slight difference between the two types of ALT cells was observed (Figure S6). As in the TRF analysis, the ALT cells without a ladder profile showed a lower signal than the ladder profile cells, indicating that a slightly lower amount of telomere sequence is present in those cells that are not being elongated with the ∼275 bp element.</p><p id="p-70">Next, we hybridized the membrane with a probe directed toward the TelKO region. Compared to WT, the TelKO hybridization signal was almost twofold higher in the ALT cells having the ladder pattern (Figure S6). In contrast, ALT cells without ladder profile showed a slightly lower content of the TelKO element compared to WT. Hence, although TelKO elements are added onto the ALT cell telomeres, the overall increase of elements is rather modest, thus indicating a limited elongation activity.</p><p id="p-71">Taken together, our results show that late passages of <em>N. castellii</em> ALT strains contain ∼275 bp repeated elements including both telomeric sequences and the internal subtelomeric region just adjacent to it, the TelKO element (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7D</a>). The TelKO element is a part of the WT telomere structure, but is increasing in numbers in the ALT strains due to addition of several consecutive repeats. When several repeats are added to a single telomere, forming an array of repeats, this is visualized as a ladder pattern in the TRF assay. We suggest that the elongation event may be accomplished by the shortened telomere invading the subtelomeric region of another chromosome, to use it as a template to copy toward the end. The interstitial telomeric sequence (ITS) that is flanking the TelKO element would be able to act as a starting point for this event. In fact, the sequences from the cloned internal telomeric segments end in exactly the same permutation as the ITS, thus implying the alignment of the shortened telomere with the ITS in the initiation of the elongation process.</p></div></div><div class="section discussion" id="sec-25"><h2 class="">Discussion</h2><p id="p-72">Genome stability in eukaryotes relies on the structural and the functional integrity of telomeres. The most widely adopted way to maintain telomeres is by utilizing the enzyme telomerase. However certain eukaryotic organisms can restore telomere length and thus chromosomal linearity in the absence of a functional telomerase (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Lundblad and Blackburn 1993</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">McEachern and Blackburn 1996</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kass-Eisler and Greider 2000</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Maringele and Lydall 2004</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jain <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Xu and McEachern 2012</a>). When telomerase fails to maintain telomeres, alternative mechanisms may be activated, so-called Alternative Lengthening of Telomeres (ALT). ALT pathways are well characterized in certain yeast species, including <em>S. cerevisiae</em>, <em>K</em>, <em>lactis</em>, <em>and S. pombe</em> (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Lundblad and Blackburn 1993</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">McEachern and Blackburn 1996</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Nakamura <em>et al.</em> 1998</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kass-Eisler and Greider 2000</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Natarajan and McEachern 2002</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Maringele and Lydall 2004</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Jain <em>et al.</em> 2010</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Xu and McEachern 2012</a>). These studies demonstrate the presence of various ALT pathways that are activated in a context-dependent manner.</p><p id="p-73">In this study, we investigated telomerase-independent telomere maintenance in the budding yeast <em>N. castellii</em>. Intriguingly, we found a novel ALT mechanism that provides a very efficient backup mechanism for telomere maintenance when telomerase is inactivated. In contrast to <em>S. cerevisiae</em> and <em>K. lactis</em>, telomerase-negative <em>N. castellii</em> cells do not display any major growth crisis and they maintain WT colony morphology (<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Lundblad and Szostak 1989</a>; <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">McEachern and Blackburn 1996</a>). This undetectable growth crisis is a general response to loss of telomerase activity, as it occurs in both <em>tlc1</em><sup>-</sup> strains and <em>est1</em>Δ strains. Moreover, it is not dependent on the cell type, since both diploid and haploid <em>tlc1</em><sup>-</sup> strains proliferate over 300 generations without displaying any growth senescence. Thus, <em>N. castellii</em> tolerates loss of telomerase surprisingly well, and conveniently avoids cellular replicative senescence. Notably, lack of a prominent senescence has also been observed in <em>Candida albicans</em> telomerase KO mutants (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Steinberg-Neifach and Lue 2006</a>).</p><p id="p-74"><em>N. castellii</em> ALT cells experience a rapid progressive telomere shortening and exhibit a reorganization of the telomeric sequences (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>). Interestingly, in later generations they use a unique way to counteract telomere attrition. This novel ALT pathway elongates telomeres by addition of ∼275 bp repeated DNA elements that contain both subtelomeric DNA and telomeric sequences, termed TelKO elements. The elongation is observed as a striking ladder pattern in the TRF assay, showing the incremental lengthening by uniform TelKO repeats. It is noteworthy that these particular features differ from the Type I and Type II ALT mechanisms that were previously reported in <em>S. cerevisiae</em>. However, some conceptual similarity to the Type I amplification of subtelomeric Y’ elements can be noted.</p><p id="p-75">Although the repeat addition generates a spectacular ladder pattern with a relatively high signal in the TRF assay, the quantification does not show any drastic increase in the amount of TelKO elements in the KO genome. This may indicate that the number of TelKO repeats added to each telomere is low, or alternatively, only a few telomeres may be elongated at each specific time point. Potentially, the TelKO arrays may be present only in a subset of the cells in a population. We further hypothesize that the array may be rather dynamic, with alternating lengthening and shortening of the repeat arrays.</p><p id="p-76">The repeated TelKO elements contain the most telomere-proximal subtelomeric region, as well as a ∼50 bp stretch of telomeric sequence that are internally located in the repeat array. A homology search of the TelKO element in the <em>N. castellii</em> genome database did not generate any match. However, even though the whole <em>N. castellii</em> genome is sequenced, it lacks telomeric and terminal subtelomeric sequences (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Cliften <em>et al.</em> 2006</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Gordon <em>et al.</em> 2011</a>). This further supports that the location of the TelKO elements is restricted to the distal subtelomeres. In accordance with previously proposed models for homology-directed ALT maintenance, the addition of TelKO elements may form via inter-telomeric replication where a single-stranded 3′ overhang of an eroded telomere would invade the duplex DNA of another telomere to use it as a copy template (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pickett and Reddel 2015</a>). Intriguingly, we found evidence that the TelKO element sequences start at a 9 bp region of interstitial telomeric sequences (ITS) present within the subtelomere. This feature implies that the ITS is the target site for the invasion of a shortened telomere and hence functions as the specific initation site for the copying. We therefore propose that this specific subtelomere structure forms the fundament for the rescue of eroded telomeres, by forming the template for the elongation of the shortened telomeres with repeated TelKO elements.</p><p id="p-77">On the other hand, we cannot exclude the possibility that the TelKO elements may be added by the “roll and spread mechanism”, which was demonstrated in telomerase-negative <em>K. lactis</em> strains (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Natarajan and McEachern 2002</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">McEachern and Haber 2006</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Xu and McEachern 2012</a>). The repeated element would in such a case be templated by a telomeric circle (t-circle) that is generated by excision of a t-loop. In the <em>N. castellii</em> survivors, such a t-circle would however need to contain also the subtelomeric region. <em>K. lactis</em> telomerase KO cells transformed with DNA circles containing both telomeric repeats and non-telomeric sequences are indeed able to add alternating regions to telomeres. Interestingly, such alternating terminal structures are also present in <em>S. cerevisiae</em> Type I survivors, where subtelomeric Y’ elements are insterspersed with telomeric sequences. It is tempting to speculate about the possibility that different types of lengthening mechanisms may actually take place, which could be depending on the actual structure being present in individual telomeres at specific time points. It is interesting to note that the later passages show a maintained short telomere structure. Hypothetically, this could suggest an efficient copying of telomeric sequences from one short end onto another short end. Possibly, the short uniform telomeric repeats of <em>N. castellii</em>, in contrast to the irregular telomeric sequences in <em>S. cerevisiae</em>, could be allowing for this process. Moreover, the short length of the ITS may imply that the initial annealing of homologous sequences are rare events, while subsequent array extensions could be promoted by using the full TelKO element for annealing. Thus, the intitial process may be replaced by other subsequent mechanisms at individual telomeres where a TelKO element has been added.</p><p id="p-78">Despite the telomeric erosion, telomerase-negative <em>N. castellii</em> cells maintain their linear chromosomes and lack abnormally sized chromosomes. Interestingly, our data show that they maintain a WT structural organization at the very ends of the chromosomes and a short stretch of telomeric sequences at the terminus. The short stretch of terminal telomeric sequence might avoid extensive chromosomal rearrangement through its association with telomere-specific proteins. We previously showed that <em>tlc1</em><sup>-</sup> KO cells do indeed have short 3′ overhangs that would provide binding sites for the Cdc13 protein (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Rhodin Edsö <em>et al.</em> 2008</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Fridholm <em>et al.</em> 2013</a>). The Rap1 protein binds every 16 nt of duplex DNA and thus the 50 bp telomeric sequence of the mutant telomeres would be bound by 2-3 Rap1 molecules (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wahlin and Cohn 2002</a>). Therefore, the telomere binding proteins may shield mutant telomeres from end to end fusions and thus inhibit massive chromosomal rearrangements.</p><p id="p-79">In conclusion, our results show that <em>N. castellii</em> bypasses cellular senescence very proficiently and effectively after telomerase is disabled. The telomerase-negative cells proliferate without neither growth crisis nor severe growth defects. We propose that the specific subtelomere structure present in some telomeres forms the fundament for the rescue of eroded telomeres, by forming the copy template for the addition of repeated elements containing both telomere sequences and the adjacent subtelomere region onto shortened telomeres. This uniquely efficient ALT mechanism is a general response to telomerase dysfunction and is activated in both diploid and haploid cells. In essence, it provides the ability to maintain linear chromosomes and WT structural organization at the terminal ends of chromosomes. ALT cells preserve short telomeric tracts at their chromosomal ends, which might provide the necessary minimal protection and thus help to overcome the replicative limitations. As this ALT pathway displays novel characteristics, it will be highly interesting to further elucidate the underlying molecular mechanisms.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-80">The work was supported by the Swedish Cancer Society; the Royal Physiographic Society in Lund; the Jörgen Lindström Foundation; the Erik Philip-Sörensen Foundation and the Carl Trygger Foundation. We thank the following students for technical assistance: Arnau Sellarès Rubio, Nicole Rietzschel, Sandra Keiper, Klaudia Karwowska, Oriol Roqué Perelló, Emilie Skoog, Stephanie Trittel, Evelien Jochems, Gema Valera Vázquez, Patrick Scicluna, Danni Wang, Arezoo Mohajeri, Brittany Banis, Andrea Comasio, Humberto Itriago, Zaneta Zalomskyte, Miroslav Bobrik and Zubaida Marufee Islam.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-81">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9632294">https://doi.org/10.25387/g3.9632294</a>.</p></li><li class="fn" id="fn-2"><p id="p-82"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-24"><span class="received-label">Received </span>April 24, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-10"><span class="accepted-label">Accepted </span>August 10, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Cohn <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.1" data-doi="10.1002/yea.1488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Astromskas</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Tools and methods for genetic analysis of <em>Saccharomyces castellii</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">509</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.1488</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DAstromskas%26rft.auinit1%253DE.%26rft.volume%253D24%26rft.issue%253D6%26rft.spage%253D499%26rft.epage%253D509%26rft.atitle%253DTools%2Band%2Bmethods%2Bfor%2Bgenetic%2Banalysis%2Bof%2BSaccharomyces%2Bcastellii.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1488%26rft_id%253Dinfo%253Apmid%252F17431926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1488&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17431926&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247759800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.2" data-doi="10.1007/s00294-009-0248-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Astromskas</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ends-in <em>vs.</em> ends-out targeted insertion mutagenesis in <em>Saccharomyces castellii</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00294-009-0248-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DAstromskas%26rft.auinit1%253DE.%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D339%26rft.epage%253D347%26rft.atitle%253DEnds-in%2Bvs.%2Bends-out%2Btargeted%2Binsertion%2Bmutagenesis%2Bin%2BSaccharomyces%2Bcastellii.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-009-0248-8%26rft_id%253Dinfo%253Apmid%252F19437021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-009-0248-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19437021&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.3" data-doi="10.1038/nm1197-1271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>,  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Englezou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Dalla-Pozza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Dunham</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Reddel</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Evidence for an alternative mechanism for maintaining telomere length in human tumors and tumor-derived cell lines.</span> <abbr class="cit-jnl-abbrev">Nat. Med.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1271</span>–<span class="cit-lpage">1274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nm1197-1271</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DBryan%26rft.auinit1%253DT.%2BM.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253D1271%26rft.epage%253D1274%26rft.atitle%253DEvidence%2Bfor%2Ban%2Balternative%2Bmechanism%2Bfor%2Bmaintaining%2Btelomere%2Blength%2Bin%2Bhuman%2Btumors%2Band%2Btumor-derived%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1197-1271%26rft_id%253Dinfo%253Apmid%252F9359704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1197-1271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9359704&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YD89600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.4" data-doi="10.1038/nrg2763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cesare</span>,  <span class="cit-name-given-names">A. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Reddel</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Alternative lengthening of telomeres: models, mechanisms and implications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2763</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCesare%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D319%26rft.epage%253D330%26rft.atitle%253DAlternative%2Blengthening%2Bof%2Btelomeres%253A%2Bmodels%252C%2Bmechanisms%2Band%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2763%26rft_id%253Dinfo%253Apmid%252F20351727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20351727&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276771400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.5" data-doi="10.1128/MCB.21.5.1819-1827.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ijpma</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Greider</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Two survivor pathways that allow growth in the absence of telomerase are generated by distinct telomere recombination events.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1827</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.21.5.1819-1827.2001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DChen%26rft.auinit1%253DQ.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D1819%26rft.epage%253D1827%26rft.atitle%253DTwo%2BSurvivor%2BPathways%2BThat%2BAllow%2BGrowth%2Bin%2Bthe%2BAbsence%2Bof%2BTelomerase%2BAre%2BGenerated%2Bby%2BDistinct%2BTelomere%2BRecombination%2BEvents%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.5.1819-1827.2001%26rft_id%253Dinfo%253Apmid%252F11238918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjIxLzUvMTgxOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzM0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.6" data-doi="10.1534/genetics.105.048900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cliften</span>,  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Fulton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Wilson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Johnston</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">After the duplication: gene loss and adaptation in Saccharomyces genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">872</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.048900</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048900%26rft_id%253Dinfo%253Apmid%252F16322519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzIvODYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.7" data-doi="10.1126/science.7618104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohn</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Blackburn</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Telomerase in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.7618104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCohn%26rft.auinit1%253DM%26rft.volume%253D269%26rft.issue%253D5222%26rft.spage%253D396%26rft.epage%253D400%26rft.atitle%253DTelomerase%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7618104%26rft_id%253Dinfo%253Apmid%252F7618104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvNTIyMi8zOTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzNDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.8" data-doi="10.1007/s002940050312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohn</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">McEachern</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Blackburn</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Telomeric sequence diversity within the genus Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s002940050312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DCohn%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D83%26rft.epage%253D91%26rft.atitle%253DTelomeric%2Bsequence%2Bdiversity%2Bwithin%2Bthe%2Bgenus%2BSaccharomyces.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050312%26rft_id%253Dinfo%253Apmid%252F9506895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9506895&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072679900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.9" data-doi="10.1093/nar/28.14.2690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forstemann</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hoss</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lingner</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Telomerase-dependent repeat divergence at the 3′ ends of yeast telomeres.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2690</span>–<span class="cit-lpage">2694</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/28.14.2690</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DForstemann%26rft.auinit1%253DK.%26rft.volume%253D28%26rft.issue%253D14%26rft.spage%253D2690%26rft.epage%253D2694%26rft.atitle%253DTelomerase-dependent%2Brepeat%2Bdivergence%2Bat%2Bthe%2B3%2527%2Bends%2Bof%2Byeast%2Btelomeres%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.14.2690%26rft_id%253Dinfo%253Apmid%252F10908324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.14.2690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10908324&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088427500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.10" data-doi="10.1093/nar/gks1028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fridholm</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Astromskas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Telomerase-dependent generation of 70-nt-long telomeric single-stranded 3′ overhangs in yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks1028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1028%26rft_id%253Dinfo%253Apmid%252F23125369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23125369&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.11" data-doi="10.1073/pnas.1112808108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Armisén</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Proux-Wéra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">ÓhÉigeartaigh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Byrne</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolutionary erosion of yeast sex chromosomes by mating-type switching accidents.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">20024</span>–<span class="cit-lpage">20029</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1112808108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1112808108%26rft_id%253Dinfo%253Apmid%252F22123960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzUwLzIwMDI0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.12" data-doi="10.1128/MCB.11.9.4572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greider</span>,  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Telomerase is processive.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">4572</span>–<span class="cit-lpage">4580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.11.9.4572</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGreider%26rft.auinit1%253DC%2BW%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D4572%26rft.epage%253D4580%26rft.atitle%253DTelomerase%2Bis%2Bprocessive.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.9.4572%26rft_id%253Dinfo%253Apmid%252F1875940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzkvNDU3MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzM0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.13" data-doi="10.1128/MCB.25.11.4406-4412.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groff-Vindman</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Cesare</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Natarajan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Griffith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">McEachern</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination at long mutant telomeres produces tiny single- and double-stranded telomeric circles.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4406</span>–<span class="cit-lpage">4412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.25.11.4406-4412.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGroff-Vindman%26rft.auinit1%253DC.%26rft.volume%253D25%26rft.issue%253D11%26rft.spage%253D4406%26rft.epage%253D4412%26rft.atitle%253DRecombination%2Bat%2BLong%2BMutant%2BTelomeres%2BProduces%2BTiny%2BSingle-%2Band%2BDouble-Stranded%2BTelomeric%2BCircles%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.11.4406-4412.2005%26rft_id%253Dinfo%253Apmid%252F15899847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS80NDA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.14" data-doi="10.1038/345458a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harley</span>,  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Futcher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Greider</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Telomeres shorten during ageing of human fibroblasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">458</span>–<span class="cit-lpage">460</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/345458a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHarley%26rft.auinit1%253DC.%2BB.%26rft.volume%253D345%26rft.issue%253D6274%26rft.spage%253D458%26rft.epage%253D460%26rft.atitle%253DTelomeres%2Bshorten%2Bduring%2Bageing%2Bof%2Bhuman%2Bfibroblasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F345458a0%26rft_id%253Dinfo%253Apmid%252F2342578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/345458a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2342578&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DG08200070&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.15" data-doi="10.1016/0014-4827(61)90192-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayflick</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Moorhead</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">The serial cultivation of human diploid cell strains.</span> <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0014-4827(61)90192-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bcell%2Bresearch%26rft.stitle%253DExp%2BCell%2BRes%26rft.aulast%253DHayflick%26rft.auinit1%253DL.%26rft.volume%253D25%26rft.spage%253D585%26rft.epage%253D621%26rft.atitle%253DThe%2Bserial%2Bcultivation%2Bof%2Bhuman%2Bdiploid%2Bcell%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-4827%252861%252990192-6%26rft_id%253Dinfo%253Apmid%252F13905658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-4827(61)90192-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13905658&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19611433B00011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.16" data-doi="10.1038/nature09374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Hebden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Nakamura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Miller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Cooper</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">HAATI survivors replace canonical telomeres with blocks of generic heterochromatin.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09374</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJain%26rft.auinit1%253DD.%26rft.volume%253D467%26rft.issue%253D7312%26rft.spage%253D223%26rft.epage%253D227%26rft.atitle%253DHAATI%2Bsurvivors%2Breplace%2Bcanonical%2Btelomeres%2Bwith%2Bblocks%2Bof%2Bgeneric%2Bheterochromatin.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09374%26rft_id%253Dinfo%253Apmid%252F20829796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20829796&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281616300038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.17" data-doi="10.1002/yea.3218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karademir Andersson</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title"><em>Naumovozyma castellii</em>: an alternative model for budding yeast molecular biology.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.3218</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D34%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.3218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.3218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.18" data-doi="10.1016/j.jmb.2017.04.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karademir Andersson</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gustafsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Krishnankutty</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multiple DNA Interactions Contribute to the Initiation of Telomerase Elongation.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">429</span>: <span class="cit-fpage">2109</span>–<span class="cit-lpage">2123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2017.04.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D429%26rft.spage%253D2109%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2017.04.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2017.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.19" data-doi="10.1002/yea.3213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karademir Andersson</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Oredsson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development of stable haploid strains and molecular genetic tools for <em>Naumovozyma castellii</em> (<em>Saccharomyces castellii</em>).</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.3213</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D33%26rft.spage%253D633%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.3213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.3213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.20" data-doi="10.1016/S0968-0004(00)01557-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass-Eisler</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Greider</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Recombination in telomere-length maintenance.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0968-0004(00)01557-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DKass-Eisler%26rft.auinit1%253DA.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D200%26rft.epage%253D204%26rft.atitle%253DRecombination%2Bin%2Btelomere-length%2Bmaintenance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0968-0004%252800%252901557-7%26rft_id%253Dinfo%253Apmid%252F10754555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0968-0004(00)01557-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10754555&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086583100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.21" data-doi="10.1126/science.7605428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Piatyszek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Prowse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Harley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">West</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Specific association of human telomerase activity with immortal cells and cancer.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">2011</span>–<span class="cit-lpage">2015</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.7605428</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKim%26rft.auinit1%253DN.%26rft.volume%253D266%26rft.issue%253D5193%26rft.spage%253D2011%26rft.epage%253D2015%26rft.atitle%253DSpecific%2Bassociation%2Bof%2Bhuman%2Btelomerase%2Bactivity%2Bwith%2Bimmortal%2Bcells%2Band%2Bcancer%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7605428%26rft_id%253Dinfo%253Apmid%252F7605428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjYvNTE5My8yMDExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.22" data-doi="10.1534/genetics.109.102939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LeBel</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rosonina</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Sealey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Pryde</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lydall</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Telomere maintenance and survival in <em>Saccharomyces cerevisiae</em> in the absence of telomerase and RAD52.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.102939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.102939%26rft_id%253Dinfo%253Apmid%252F19380905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzMvNjcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lendvay</span>,  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Morris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Balasubramanian</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Lundblad</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Senescence mutants of <em>Saccharomyces cerevisiae</em> with a defect in telomere replication identify three additional EST genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1399</span>–<span class="cit-lpage">1412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D144%26rft.spage%253D1399%26rft_id%253Dinfo%253Apmid%252F8978029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8978029&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VY93100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.24" data-doi="10.1016/0092-8674(93)90234-H"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Blackburn</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">An alternative pathway for yeast telomere maintenance rescues est1- senescence.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(93)90234-H</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLundblad%26rft.auinit1%253DV.%26rft.volume%253D73%26rft.issue%253D2%26rft.spage%253D347%26rft.epage%253D360%26rft.atitle%253DAn%2Balternative%2Bpathway%2Bfor%2Byeast%2Btelomere%2Bmaintenance%2Brescues%2Best1-%2Bsenescence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990234-H%26rft_id%253Dinfo%253Apmid%252F8477448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90234-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8477448&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KY50500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.25" data-doi="10.1016/0092-8674(89)90132-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Szostak</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A mutant with a defect in telomere elongation leads to senescence in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(89)90132-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLundblad%26rft.auinit1%253DV.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D633%26rft.epage%253D643%26rft.atitle%253DA%2Bmutant%2Bwith%2Ba%2Bdefect%2Bin%2Btelomere%2Belongation%2Bleads%2Bto%2Bsenescence%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990132-3%26rft_id%253Dinfo%253Apmid%252F2655926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90132-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2655926&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U743600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.26" data-doi="10.1101/gad.316504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maringele</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lydall</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Telomerase- and recombination-independent immortalization of budding yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2663</span>–<span class="cit-lpage">2675</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.316504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.316504%26rft_id%253Dinfo%253Apmid%252F15489288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzIxLzI2NjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzNDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.27" data-doi="10.1038/376403a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McEachern</span>,  <span class="cit-name-given-names">M. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Blackburn</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Runaway telomere elongation caused by telomerase RNA gene mutations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">376</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/376403a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcEachern%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D376%26rft.issue%253D6539%26rft.spage%253D403%26rft.epage%253D409%26rft.atitle%253DRunaway%2Btelomere%2Belongation%2Bcaused%2Bby%2Btelomerase%2BRNA%2Bgene%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F376403a0%26rft_id%253Dinfo%253Apmid%252F7630414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/376403a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7630414&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RM63900045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.28" data-doi="10.1101/gad.10.14.1822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McEachern</span>,  <span class="cit-name-given-names">M. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Blackburn</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Cap-prevented recombination between terminal telomeric repeat arrays (telomere CPR) maintains telomeres in <em>Kluyveromyces lactis</em> lacking telomerase.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1822</span>–<span class="cit-lpage">1834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.10.14.1822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMcEachern%26rft.auinit1%253DM%2BJ%26rft.volume%253D10%26rft.issue%253D14%26rft.spage%253D1822%26rft.epage%253D1834%26rft.atitle%253DCap-prevented%2Brecombination%2Bbetween%2Bterminal%2Btelomeric%2Brepeat%2Barrays%2B%2528telomere%2BCPR%2529%2Bmaintains%2Btelomeres%2Bin%2BKluyveromyces%2Blactis%2Blacking%2Btelomerase.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.14.1822%26rft_id%253Dinfo%253Apmid%252F8698241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEwLzE0LzE4MjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzNDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.29" data-doi="10.1146/annurev.biochem.74.082803.133234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McEachern</span>,  <span class="cit-name-given-names">M. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Haber</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Break-induced replication and recombinational telomere elongation in yeast.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.biochem.74.082803.133234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.volume%253D75%26rft.spage%253D111%26rft.epage%253D135%26rft.atitle%253DBreak-induced%2Breplication%2Band%2Brecombinational%2Btelomere%2Belongation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.74.082803.133234%26rft_id%253Dinfo%253Apmid%252F16756487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.74.082803.133234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16756487&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239807600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.30" data-doi="10.1126/science.282.5388.493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>,  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Cooper</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Cech</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Two modes of survival of fission yeast without telomerase.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.282.5388.493</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNakamura%26rft.auinit1%253DT.%2BM.%26rft.volume%253D282%26rft.issue%253D5388%26rft.spage%253D493%26rft.epage%253D496%26rft.atitle%253DTwo%2BModes%2Bof%2BSurvival%2Bof%2BFission%2BYeast%2BWithout%2BTelomerase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5388.493%26rft_id%253Dinfo%253Apmid%252F9774280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTM4OC80OTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzNDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.31" data-doi="10.1128/MCB.22.13.4512-4521.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natarajan</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">McEachern</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recombinational telomere elongation promoted by DNA circles.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">4512</span>–<span class="cit-lpage">4521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.22.13.4512-4521.2002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNatarajan%26rft.auinit1%253DS.%26rft.volume%253D22%26rft.issue%253D13%26rft.spage%253D4512%26rft.epage%253D4521%26rft.atitle%253DRecombinational%2BTelomere%2BElongation%2BPromoted%2Bby%2BDNA%2BCircles%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.13.4512-4521.2002%26rft_id%253Dinfo%253Apmid%252F12052861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMy80NTEyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.32" data-doi="10.1016/0022-5193(73)90198-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olovnikov</span>,  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">A theory of marginotomy. The incomplete copying of template margin in enzymic synthesis of polynucleotides and biological significance of the phenomenon.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-5193(73)90198-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DOlovnikov%26rft.auinit1%253DA.%2BM.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D181%26rft.epage%253D190%26rft.atitle%253DA%2Btheory%2Bof%2Bmarginotomy.%2BThe%2Bincomplete%2Bcopying%2Bof%2Btemplate%2Bmargin%2Bin%2Benzymic%2Bsynthesis%2Bof%2Bpolynucleotides%2Band%2Bbiological%2Bsignificance%2Bof%2Bthe%2Bphenomenon.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-5193%252873%252990198-7%26rft_id%253Dinfo%253Apmid%252F4754905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-5193(73)90198-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4754905&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973Q722300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.33" data-doi="10.1099/00207713-49-4-1925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>,  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nilsson-Tillgren</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Piskur</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Karyotypes of Saccharomyces sensu lato species.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Bacteriol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1925</span>–<span class="cit-lpage">1931</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/00207713-49-4-1925</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BSystematic%2Band%2BEvolutionary%2BMicrobiology%26rft.stitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft.aulast%253DPetersen%26rft.auinit1%253DR.%2BF.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1925%26rft.epage%253D1931%26rft.atitle%253DKaryotypes%2Bof%2BSaccharomyces%2Bsensu%2Blato%2Bspecies%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00207713-49-4-1925%26rft_id%253Dinfo%253Apmid%252F10555377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00207713-49-4-1925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10555377&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.34" data-doi="10.1038/nsmb.3106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pickett</span>,  <span class="cit-name-given-names">H. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Reddel</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Molecular mechanisms of activity and derepression of alternative lengthening of telomeres.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">880</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nsmb.3106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D22%26rft.spage%253D875%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3106%26rft_id%253Dinfo%253Apmid%252F26581522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26581522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.35" data-doi="10.1002/yea.817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Sunjevaric</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Keddache</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rothstein</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Efficient PCR-based gene disruption in Saccharomyces strains using intergenic primers.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.817</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DReid%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D319%26rft.epage%253D328%26rft.atitle%253DEfficient%2BPCR-based%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bstrains%2Busing%2Bintergenic%2Bprimers.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.817%26rft_id%253Dinfo%253Apmid%252F11870855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11870855&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.36" data-doi="10.1111/j.1567-1364.2008.00431.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rhodin Edsö</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tati</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Highly sequence-specific binding is retained within the DNA-binding domain of the <em>Saccharomyces castellii</em> Cdc13 telomere-binding protein.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1567-1364.2008.00431.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00431.x%26rft_id%253Dinfo%253Apmid%252F18759744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2008.00431.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18759744&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260537300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.37" data-doi="10.1016/j.semcancer.2011.10.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shay</span>,  <span class="cit-name-given-names">J. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. E.</span> <span class="cit-name-surname">Wright</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Role of telomeres and telomerase in cancer.</span> <abbr class="cit-jnl-abbrev">Semin. Cancer Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.semcancer.2011.10.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcancer%2Bbiology%26rft.stitle%253DSemin%2BCancer%2BBiol%26rft.aulast%253DShay%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D349%26rft.epage%253D353%26rft.atitle%253DRole%2Bof%2Btelomeres%2Band%2Btelomerase%2Bin%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcancer.2011.10.001%26rft_id%253Dinfo%253Apmid%252F22015685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcancer.2011.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22015685&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298571300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.38" data-doi="10.1126/science.7545955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>,  <span class="cit-name-given-names">M. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Gottschling</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title"><em>TLC1</em>: template RNA component of <em>Saccharomyces cerevisiae</em> telomerase.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.7545955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSinger%26rft.auinit1%253DM.%26rft.volume%253D266%26rft.issue%253D5184%26rft.spage%253D404%26rft.epage%253D409%26rft.atitle%253DTLC1%253A%2Btemplate%2BRNA%2Bcomponent%2Bof%2BSaccharomyces%2Bcerevisiae%2Btelomerase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7545955%26rft_id%253Dinfo%253Apmid%252F7545955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjYvNTE4NC80MDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzNDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.39" data-doi="10.1093/nar/gkl345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinberg-Neifach</span>,  <span class="cit-name-given-names">O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. F.</span> <span class="cit-name-surname">Lue</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Modulation of telomere terminal structure by telomerase components in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2710</span>–<span class="cit-lpage">2722</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl345</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSteinberg-Neifach%26rft.auinit1%253DO.%26rft.volume%253D34%26rft.issue%253D9%26rft.spage%253D2710%26rft.epage%253D2722%26rft.atitle%253DModulation%2Bof%2Btelomere%2Bterminal%2Bstructure%2Bby%2Btelomerase%2Bcomponents%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl345%26rft_id%253Dinfo%253Apmid%252F16714448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16714448&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238763700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.40" data-doi="10.1073/pnas.0408770102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Topcu</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nickles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Davis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">McEachern</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Abrupt disruption of capping and a single source for recombinationally elongated telomeres in <em>Kluyveromyces lactis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">3348</span>–<span class="cit-lpage">3353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0408770102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0408770102%26rft_id%253Dinfo%253Apmid%252F15713803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzkvMzM0OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzM0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.41" data-doi="10.1002/yea.816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahlin</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Analysis of the RAP1 protein binding to homogeneous telomeric repeats in <em>Saccharomyces castellii</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.816</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWahlin%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D256%26rft.atitle%253DAnalysis%2Bof%2Bthe%2BRAP1%2Bprotein%2Bbinding%2Bto%2Bhomogeneous%2Btelomeric%2Brepeats%2Bin%2BSaccharomyces%2Bcastellii.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.816%26rft_id%253Dinfo%253Apmid%252F11816032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11816032&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174109200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.42" data-doi="10.1016/S0022-2836(03)00850-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahlin</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rosen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohn</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">DNA binding and telomere length regulation of yeast RAP1 homologues.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">833</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(03)00850-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWahlin%26rft.auinit1%253DJ.%26rft.volume%253D332%26rft.issue%253D4%26rft.spage%253D821%26rft.epage%253D833%26rft.atitle%253DDNA%2Bbinding%2Band%2Btelomere%2Blength%2Bregulation%2Bof%2Byeast%2BRAP1%2Bhomologues.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252803%252900850-7%26rft_id%253Dinfo%253Apmid%252F12972254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(03)00850-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12972254&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3345.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.43" data-doi="10.1371/journal.pone.0099491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rizvydeen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vahedi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Vargas Gonzalez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Allred</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel telomere-anchored PCR approach for studying sexual stage telomeres in <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e99491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0099491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De99491%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0099491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0099491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3345.44" data-doi="10.1128/MCB.00430-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">McEachern</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maintenance of very long telomeres by recombination in the <em>Kluyveromyces lactis</em> stn1–M1 mutant involves extreme telomeric turnover, telomeric circles, and concerted telomeric amplification.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2992</span>–<span class="cit-lpage">3008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.00430-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00430-12%26rft_id%253Dinfo%253Apmid%252F22645309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNS8yOTkyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3345.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3333" title="Genomic Prediction Including SNP-Specific Variance Predictors" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3359" title="Exon 3 of the NUMB Gene Emerged in the Chordate Lineage Coopting the NUMB Protein to the Regulation of MDM2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3345" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--nWFuIHEke66Ue4EcX5xnDa8HkXp12oyZMfP8Ay6ek0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3345" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3345" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3345" class="active" data-icon-position="" data-hide-link-title="0">Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494286" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-k9N_9U2J1cTYpnAORbQ4Ut3yTh785DGf1-z88mdkqwk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3345.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3345" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9HsaL2ACUMEx3qF4PQd86ACVYpMBbVmd0M12KcvVWhc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3345" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3345.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494286" id="node494286--2" data-pisa="ggg;9/10/3345" data-pisa-master="ggg;g3.119.400428" data-apath="/ggg/9/10/3345.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alternative Lengthening of Telomeres in the Budding Yeast <em>Naumovozyma castellii</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3370-2864" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marita Cohn</span>, <span class="highwire-citation-author" data-delta="1">Ahu Karademir Andersson</span>, <span class="highwire-citation-author" data-delta="2">Raquel Quintilla Mateo</span> and <span class="highwire-citation-author" data-delta="3">Mirja Carlsson Möller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3345-3358;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400428 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494286--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marita Cohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marita%2BCohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cohn%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarita%2BCohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3370-2864" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marita Cohn</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marita.cohn@biol.lu.se" class="" data-icon-position="" data-hide-link-title="0">marita.cohn@biol.lu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ahu Karademir Andersson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ahu%2BKarademir%2BAndersson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersson%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAhu%2BKarademir%2BAndersson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Raquel Quintilla Mateo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raquel%2BQuintilla%2BMateo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mateo%20RQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaquel%2BQuintilla%2BMateo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mirja Carlsson Möller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mirja%2BCarlsson%2BM%C3%B6ller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%B6ller%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMirja%2BCarlsson%2BM%25C3%25B6ller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3345" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3345&amp;title=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3345&amp;title=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3345&amp;title=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3345&amp;text=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3345&amp;title=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3345&amp;t=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3345&amp;title=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3345&amp;title=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494286/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494286" id="node494286--3" data-pisa="ggg;9/10/3345" data-pisa-master="ggg;g3.119.400428" data-apath="/ggg/9/10/3345.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alternative Lengthening of Telomeres in the Budding Yeast <em>Naumovozyma castellii</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3370-2864" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marita Cohn</span>, <span class="highwire-citation-author" data-delta="1">Ahu Karademir Andersson</span>, <span class="highwire-citation-author" data-delta="2">Raquel Quintilla Mateo</span> and <span class="highwire-citation-author" data-delta="3">Mirja Carlsson Möller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3345-3358;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400428 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494286--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marita Cohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marita%2BCohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cohn%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarita%2BCohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3370-2864" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marita Cohn</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marita.cohn@biol.lu.se" class="" data-icon-position="" data-hide-link-title="0">marita.cohn@biol.lu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ahu Karademir Andersson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ahu%2BKarademir%2BAndersson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersson%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAhu%2BKarademir%2BAndersson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Raquel Quintilla Mateo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raquel%2BQuintilla%2BMateo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mateo%20RQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaquel%2BQuintilla%2BMateo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mirja Carlsson Möller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Genetics group, Lund University, 223 62 Lund, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mirja%2BCarlsson%2BM%C3%B6ller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%B6ller%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMirja%2BCarlsson%2BM%25C3%25B6ller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3345&amp;rft.epage=3358&amp;rft.atitle=Alternative%20Lengthening%20of%20Telomeres%20in%20the%20Budding%20Yeast%20Naumovozyma%20castellii&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cohn%2C+Marita&amp;rft.au=Andersson%2C+Ahu+Karademir&amp;rft.au=Mateo%2C+Raquel+Quintilla&amp;rft.au=M%C3%B6ller%2C+Mirja+Carlsson"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494286/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494286/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494286/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494286/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494286/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494286/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494286/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494286/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494286/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494286/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494286/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494286/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3345" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-25" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3345.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3345.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3345.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
